
<html lang="en"     class="pb-page"  data-request-id="3d750e2a-c5ad-4fc7-a8aa-7cc3bc63e545"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2017.60.issue-22;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.7b01313;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Heterocyclic Analogues of Modafinil as Novel, Atypical Dopamine Transporter Inhibitors" /></meta><meta name="dc.Creator" content="Predrag  Kalaba" /></meta><meta name="dc.Creator" content="Nilima Y.  Aher" /></meta><meta name="dc.Creator" content="Marija  Ilić" /></meta><meta name="dc.Creator" content="Vladimir  Dragačević" /></meta><meta name="dc.Creator" content="Marcus  Wieder" /></meta><meta name="dc.Creator" content="Andras G.  Miklosi" /></meta><meta name="dc.Creator" content="Martin  Zehl" /></meta><meta name="dc.Creator" content="Judith  Wackerlig" /></meta><meta name="dc.Creator" content="Alexander  Roller" /></meta><meta name="dc.Creator" content="Tetyana  Beryozkina" /></meta><meta name="dc.Creator" content="Bojana  Radoman" /></meta><meta name="dc.Creator" content="Sivaprakasam R.  Saroja" /></meta><meta name="dc.Creator" content="Wolfgang  Lindner" /></meta><meta name="dc.Creator" content="Eduardo Perez  Gonzalez" /></meta><meta name="dc.Creator" content="Vasiliy  Bakulev" /></meta><meta name="dc.Creator" content="Johann Jakob  Leban" /></meta><meta name="dc.Creator" content="Harald H.  Sitte" /></meta><meta name="dc.Creator" content="Ernst  Urban" /></meta><meta name="dc.Creator" content="Thierry  Langer" /></meta><meta name="dc.Creator" content="Gert  Lubec" /></meta><meta name="dc.Description" content="Modafinil is a wake promoting compound with high potential for cognitive enhancement. It is targeting the dopamine transporter (DAT) with moderate selectivity, thereby leading to reuptake inhibitio..." /></meta><meta name="Description" content="Modafinil is a wake promoting compound with high potential for cognitive enhancement. It is targeting the dopamine transporter (DAT) with moderate selectivity, thereby leading to reuptake inhibitio..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 9, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01313" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01313" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01313" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01313" /></link>
        
    
    

<title>Heterocyclic Analogues of Modafinil as Novel, Atypical Dopamine Transporter Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01313" /></meta><meta property="og:title" content="Heterocyclic Analogues of Modafinil as Novel, Atypical Dopamine Transporter Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0013.jpeg" /></meta><meta property="og:description" content="Modafinil is a wake promoting compound with high potential for cognitive enhancement. It is targeting the dopamine transporter (DAT) with moderate selectivity, thereby leading to reuptake inhibition and increased dopamine levels in the synaptic cleft. A series of modafinil analogues have been reported so far, but more target-specific analogues remain to be discovered. It was the aim of this study to synthesize and characterize such analogues and, indeed, a series of compounds were showing higher activities on the DAT and a higher selectivity toward DAT versus serotonin and norepinephrine transporters than modafinil. This was achieved by substituting the amide moiety by five- and six-membered aromatic heterocycles. In vitro studies indicated binding to the cocaine pocket on DAT, although molecular dynamics revealed binding different from that of cocaine. Moreover, no release of dopamine was observed, ruling out amphetamine-like effects. The absence of neurotoxicity of a representative analogue may encourage further preclinical studies of the above-mentioned compounds." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01313"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01313">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01313&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01313&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01313&amp;href=/doi/10.1021/acs.jmedchem.7b01313" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 9330-9348</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01295" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01320" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Heterocyclic Analogues of Modafinil as Novel, Atypical Dopamine Transporter Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Predrag++Kalaba">Predrag Kalaba</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>◆</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nilima+Y.++Aher">Nilima Y. Aher</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>◆</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marija++Ili%C4%87">Marija Ilić</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Vladimir++Draga%C4%8Devi%C4%87">Vladimir Dragačević</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marcus++Wieder">Marcus Wieder</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-2631-8415" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andras+G.++Miklosi">Andras G. Miklosi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Zehl">Martin Zehl</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Judith++Wackerlig">Judith Wackerlig</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alexander++Roller">Alexander Roller</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tetyana++Beryozkina">Tetyana Beryozkina</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Bojana++Radoman">Bojana Radoman</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sivaprakasam+R.++Saroja">Sivaprakasam R. Saroja</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wolfgang++Lindner">Wolfgang Lindner</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Eduardo+Perez++Gonzalez">Eduardo Perez Gonzalez</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Vasiliy++Bakulev">Vasiliy Bakulev</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-3312-7783" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Johann+Jakob++Leban">Johann Jakob Leban</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Harald+H.++Sitte">Harald H. Sitte</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ernst++Urban">Ernst Urban</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thierry++Langer">Thierry Langer</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gert++Lubec">Gert Lubec</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>○</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-6333-9461" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Pharmaceutical Chemistry, Faculty of Life Sciences, University of Vienna, Althanstraße 14, 1090 Vienna, Austria</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Ural Federal University named after the first President of Russia B. N. Yeltsin, 19 Mira St., Yekaterinburg 620002, Russia</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Department of Analytical Chemistry, Faculty of Chemistry, University of Vienna, Währinger Straße 38, 1090 Vienna, Austria</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">X-ray Structure Analysis Centre, Faculty of Chemistry, University of Vienna, Währinger Straße 38, 1090 Vienna, Austria</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Department of Pediatrics, Medical University of Vienna, 1090 Vienna, Austria</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">Laboratory of Fine Organic Chemistry, Department of Chemistry and Biochemistry, Faculty of Science and Technology, University of Sao Paulo State, Roberto Simonsen 305, CEP 19060-900 Presidente Prudente, Sao Paolo, Brazil</span></div><div class="aff-info" id="aff7"><span class="aff-symbol">∇</span> <span class="aff-text">Institute of Pharmacology, Centre of Physiology and Pharmacology, Medical University of Vienna, 1090, Vienna, Austria</span></div><div class="aff-info" id="aff8"><span class="aff-symbol">○</span> <span class="aff-text">Neuroscience Laboratory, Paracelsus Medical University, 5020 Salzburg, Austria</span></div><div class="corresp-info"><strong>*</strong>For G.L.: phone, ++43-676-569-48-16; E-mail, <a href="/cdn-cgi/l/email-protection#aacdcfd8de84c6dfc8cfc9eac6dfc8cfc9c6cbc884c9c5c7"><span class="__cf_email__" data-cfemail="a4c3c1d6d08ac8d1c6c1c7e4c8d1c6c1c7c8c5c68ac7cbc9">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>For T.L.: E-mail, <a href="/cdn-cgi/l/email-protection#97e3fffef2e5e5eeb9fbf6f9f0f2e5d7e2f9fee1fef2b9f6f4b9f6e3"><span class="__cf_email__" data-cfemail="41352928243333386f2d202f26243301342f283728246f20226f2035">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01313&amp;href=/doi/10.1021%2Facs.jmedchem.7b01313" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 9330–9348</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 1, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>6 September 2017</li><li><span class="item_label"><b>Published</b> online</span>9 November 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 November 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01313" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01313</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9330%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPredrag%2BKalaba%252C%2BNilima%2BY.%2BAher%252C%2BMarija%2BIli%25C4%2587%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D22%26contentID%3Dacs.jmedchem.7b01313%26title%3DHeterocyclic%2BAnalogues%2Bof%2BModafinil%2Bas%2BNovel%252C%2BAtypical%2BDopamine%2BTransporter%2BInhibitors%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9348%26publicationDate%3DNovember%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01313"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1599</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">11</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01313" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Heterocyclic Analogues of Modafinil as Novel, Atypical Dopamine Transporter Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Predrag&quot;,&quot;last_name&quot;:&quot;Kalaba&quot;},{&quot;first_name&quot;:&quot;Nilima&quot;,&quot;last_name&quot;:&quot;Y. Aher&quot;},{&quot;first_name&quot;:&quot;Marija&quot;,&quot;last_name&quot;:&quot;Ilić&quot;},{&quot;first_name&quot;:&quot;Vladimir&quot;,&quot;last_name&quot;:&quot;Dragačević&quot;},{&quot;first_name&quot;:&quot;Marcus&quot;,&quot;last_name&quot;:&quot;Wieder&quot;},{&quot;first_name&quot;:&quot;Andras&quot;,&quot;last_name&quot;:&quot;G. Miklosi&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Zehl&quot;},{&quot;first_name&quot;:&quot;Judith&quot;,&quot;last_name&quot;:&quot;Wackerlig&quot;},{&quot;first_name&quot;:&quot;Alexander&quot;,&quot;last_name&quot;:&quot;Roller&quot;},{&quot;first_name&quot;:&quot;Tetyana&quot;,&quot;last_name&quot;:&quot;Beryozkina&quot;},{&quot;first_name&quot;:&quot;Bojana&quot;,&quot;last_name&quot;:&quot;Radoman&quot;},{&quot;first_name&quot;:&quot;Sivaprakasam&quot;,&quot;last_name&quot;:&quot;R. Saroja&quot;},{&quot;first_name&quot;:&quot;Wolfgang&quot;,&quot;last_name&quot;:&quot;Lindner&quot;},{&quot;first_name&quot;:&quot;Eduardo&quot;,&quot;last_name&quot;:&quot;Perez Gonzalez&quot;},{&quot;first_name&quot;:&quot;Vasiliy&quot;,&quot;last_name&quot;:&quot;Bakulev&quot;},{&quot;first_name&quot;:&quot;Johann&quot;,&quot;last_name&quot;:&quot;Jakob Leban&quot;},{&quot;first_name&quot;:&quot;Harald&quot;,&quot;last_name&quot;:&quot;H. Sitte&quot;},{&quot;first_name&quot;:&quot;Ernst&quot;,&quot;last_name&quot;:&quot;Urban&quot;},{&quot;first_name&quot;:&quot;Thierry&quot;,&quot;last_name&quot;:&quot;Langer&quot;},{&quot;first_name&quot;:&quot;Gert&quot;,&quot;last_name&quot;:&quot;Lubec&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;09&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;9330-9348&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01313&quot;},&quot;abstract&quot;:&quot;Modafinil is a wake promoting compound with high potential for cognitive enhancement. It is targeting the dopamine transporter (DAT) with moderate selectivity, thereby leading to reuptake inhibition and increased dopamine levels in the synaptic cleft. A series of modafinil analogues have been reported so far, but more target-specific analogues remain to be discovered. It was the aim of this study to synthesize and characterize such analogues and, indeed, a series of compounds were showing higher activities on the DAT and a higher selectivity toward DAT versus serotonin and norepinephrine transporters than modafinil. This was achieved by substituting the amide moiety by five- and six-membered aromatic heterocycles. In vitro studies indicated binding to the cocaine pocket on DAT, although molecular dynamics revealed binding different from that of cocaine. Moreover, no release of dopamine was observed, ruling out amphetamine-like effects. The absence of neurotoxicity of a representative analogue may encourage fu&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01313&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01313" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01313&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01313" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01313&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01313" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01313&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01313&amp;href=/doi/10.1021/acs.jmedchem.7b01313" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01313" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01313" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01313%26sid%3Dliteratum%253Aachs%26pmid%3D29091428%26genre%3Darticle%26aulast%3DKalaba%26date%3D2017%26atitle%3DHeterocyclic%2BAnalogues%2Bof%2BModafinil%2Bas%2BNovel%252C%2BAtypical%2BDopamine%2BTransporter%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D22%26spage%3D9330%26epage%3D9348%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291012" title="Organic acids">Organic acids</a>,</li><li><a href="/action/doSearch?ConceptID=290723" title="Solutions">Solutions</a>,</li><li><a href="/action/doSearch?ConceptID=290860" title="Solution chemistry">Solution chemistry</a>,</li><li><a href="/action/doSearch?ConceptID=291594" title="Silica">Silica</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/jmcmar.2017.60.issue-22/20171122/jmcmar.2017.60.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Modafinil is a wake promoting compound with high potential for cognitive enhancement. It is targeting the dopamine transporter (DAT) with moderate selectivity, thereby leading to reuptake inhibition and increased dopamine levels in the synaptic cleft. A series of modafinil analogues have been reported so far, but more target-specific analogues remain to be discovered. It was the aim of this study to synthesize and characterize such analogues and, indeed, a series of compounds were showing higher activities on the DAT and a higher selectivity toward DAT versus serotonin and norepinephrine transporters than modafinil. This was achieved by substituting the amide moiety by five- and six-membered aromatic heterocycles. In vitro studies indicated binding to the cocaine pocket on DAT, although molecular dynamics revealed binding different from that of cocaine. Moreover, no release of dopamine was observed, ruling out amphetamine-like effects. The absence of neurotoxicity of a representative analogue may encourage further preclinical studies of the above-mentioned compounds.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53580" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53580" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Modafinil, i.e., 2-[(diphenylmethyl)sulfinyl]acetamide, (<b>1</b>) (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) is an FDA-approved drug for treatment of narcolepsy and other sleep disorders.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> It is a mild psychostimulant-like agent that increases wakefulness, improves attention, and enhances performance in a variety of cognitive tasks.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> It has been shown to reduce jet lag and improve mood among shift workers. It has also been studied as an alternative to amphetamines for military usage and is currently also approved for Air Force missions in the U.S.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0001.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of 2-[(diphenylmethyl)sulfinyl]acetamide (modafinil).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">It is mode of action remained unknown for long time despite of several years of preclinical research.<a onclick="showRef(event, 'ref3 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref5">(3, 5)</a> Using radioligand binding assays, Mignot et al. (1994) have demonstrated that modafinil binds to the dopamine transporter (DAT) and not to any other transporters and receptors tested in their study.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Zolkowska et al. (2009) performed an in vitro binding study and concluded that the DAT was the exclusive target at which modafinil displayed relevant binding.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Loland et al. (2012) showed that <i>R</i>- and <i>S</i>-modafinil bind to the DAT less potently than cocaine, with <i>R</i>-modafinil having an in vitro profile different from cocaine.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p">Modafinil’s pharmacological profile can be clearly distinguished from those of other classical psychostimulants.<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2-4)</a> It binds to DAT in a different way as compared to cocaine, which may lead to its unique pharmacological profile and, unlike amphetamine, modafinil does not cause efflux of dopamine.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">Modafinil has been intensively studied, and many of its derivatives have been synthesized and tested for biological activity. Cao et al. (2010) have synthesized a series of modafinil analogues by adding <i>para</i>-halo-substituents to the aryl rings, modifying the sulfoxide function and replacing the primary amide group with secondary and tertiary amides and amines.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Okonula-Bakare et al. (2014) synthesized a series of thioacetamide and sulfinylacetamide analogues and further investigated the role of the terminal amide or substituted amine functions on DAT vs serotonin transporter (SERT) and norepinephrine transporter (NET) binding.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Cao et al. (2016) further extended the SAR of modafinil analogues by chemical modifications of the oxidation states of sulfoxide and the amide functional groups, by halogenating the phenyl rings, and/or by functionalizing the terminal nitrogen with substituted piperazines.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><div class="NLM_p last">The aim of the current study was to generate a novel series of modafinil analogues with higher dopamine reuptake inhibition activity and higher selectivity at DAT vs SERT and NET by substituting the amide moiety with five- and six-membered aromatic heterocycles. The most potent inhibitors from this series were used in release studies to show that these compounds are not causing efflux of dopamine (amphetamine-like effect). Furthermore, binding studies were carried out to determine whether these compounds are targeting the same binding pocket as cocaine. Molecular docking was performed using a homology model of the DAT in order to propose the binding site of two compounds that showed significant dopamine-reuptake inhibition in vitro. Subsequently, molecular dynamics (MD) simulations with the highest scoring ligand poses were performed. To mimic the physiological system, the DAT structural model was placed in a double lipid membrane. The obtained binding poses for the compounds from molecular docking and from MD simulations were compared with already established binding sites for the cocaine analogue CFT<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and mephedrone analogues.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Furthermore, we aimed to demonstrate that a selected compound from the series shows no general in vivo neurotoxicity that may render it suitable for further preclinical development.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45000" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45000" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Synthesis of novel 2-[(diphenylmethyl)sulfinyl]thiadiazole and [(benzhydrylsulfinyl)methyl]oxadiazole analogues (<b>4a</b>–<b>e</b>) was achieved as depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Nucleophilic substitution of commercially available substituted chloro-methyl-thiodiazoles and oxadiazole analogues with diphenylmethanethiol (<b>2</b>) in the presence of <i>t</i>-BuOK in DMF or diisopropylethylamine (DIPEA) in 1,4-dioxane provided substituted [(benzhydrylsulfenyl)methyl]-thiodiazole and -oxadiazole intermediates (<b>3a</b>–<b>e</b>) in 35–95% yield. The oxidation reaction in CH<sub>3</sub>COOH with H<sub>2</sub>O<sub>2</sub> gave final products (<b>4a</b>–<b>e</b>) in 30–70% yield.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0010.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>3a</b>–<b>e</b> and <b>4a</b>–<b>e</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) DMF, <i>t</i>-BuOK, rt, overnight; (b) 1,4-dioxane, DIPEA, rt, 3 h; (c) CH<sub>3</sub>COOH, H<sub>2</sub>O<sub>2</sub>, rt, 24 h.</p></p></figure><div class="NLM_p">Substituted [(benzhydrylsulfinyl)methyl]-pyridinyl, -isoxazolyl, -pyrimidinyl, -oxadiazolyl, and -thiophenyl analogues were generated via another synthetic route depicted in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Alkylation of diphenylmethyl-<i>S</i>-isothiuronium hydrobromide (<b>5</b>) with plain or substituted chloro-methyl-heterocycles with K<sub>2</sub>CO<sub>3</sub> in MeOH provided intermediates (<b>6a</b>–<b>q</b>) in 34–97% yield. Oxidation of the sulfenyl moiety gave the desired sulfoxide-containing analogues (<b>7a</b>–<b>q</b>) in 27–90% yield.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0011.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>6a</b>–<b>q</b> and <b>7a</b>–<b>q</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) MeOH, K<sub>2</sub>CO<sub>3</sub>, rt, 48 h; (b) CH<sub>3</sub>COOH, H<sub>2</sub>O<sub>2</sub>, rt, overnight.</p></p></figure><div class="NLM_p">Unsubstituted and substituted 2-((benzhydrylsulfinyl)methyl)thiophene analogues were obtained as depicted in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Condensation of commercially available benzhydrols (<b>8a</b>–<b>f</b>) with thiophen-2-ylmethanethiol (<b>9</b>) and BF<sub>3</sub>·Et<sub>2</sub>O in CH<sub>3</sub>COOH provided analogues with sulfenyl moiety (<b>10a</b>–<b>f</b>) in 66–94% yield. As in the previous synthetic routes, oxidation of sulfenyl intermediates with H<sub>2</sub>O<sub>2</sub> in CH<sub>3</sub>COOH generated sulfoxide containing products (<b>11a</b>–<b>f</b>) in 19–85% yield.</div><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0012.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>10a</b>–<b>f</b> and <b>11a</b>–<b>f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) BF<sub>3</sub>·Et<sub>2</sub>O, CH<sub>3</sub>COOH, rt, overnight; (b) CH<sub>3</sub>COOH, H<sub>2</sub>O<sub>2</sub>, rt, overnight.</p></p></figure><div class="NLM_p last">Compound <b>11c</b> was separated further into two mixtures, <b>11g</b> and <b>11h</b>, both containing two pairs of stereoisomers. Each mixture was screened on Chiralpack IA HPLC column (Daicel Inc., Tokyo, Japan) to separate into individual enantiomers. For compound <b>11g</b>, this was achieved using 90% hexane:10% 2-propanol as the mobile phase. Biological data is not available because they were not soluble in stock solution. Compound <b>11h</b> was separated by HPLC into compounds <b>11i</b> and <b>11j</b>, using 60% hexane:40% ethyl acetate as the mobile phase.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68521" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68521" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> SARs at DAT, NET, and SERT</h3><div class="NLM_p">All final compounds (<b>4a</b>–<b>e</b>, <b>7a</b>–<b>q</b>, and <b>11a</b>–<b>f</b>) were evaluated for uptake inhibition at the monoamine transporters (DAT, SERT, and NET) in HEK293 cells stably expressing human isoforms of DAT, NET, and SERT.</div><div class="NLM_p">The inhibition of the uptake of [<sup>3</sup>H]-dopamine (for DAT), [<sup>3</sup>H]-1-methyl-4-phenylpyridinium (for NET), and [<sup>3</sup>H]-serotonin (for SERT) was measured as described in detail in the Experimental Procedures.</div><div class="NLM_p">Changes in DAT inhibition and selectivity as the result of modifications compared to modafinil (<b>1</b>) as a parent compound were evaluated; SAR analyses explored the effect of substituting the amide moiety with heterocycles as well as the influence of introducing of halogen atoms and methyl groups on phenyl ring(s) in analogues containing the thiophene part. Results are presented in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Uptake Inhibition Data in DAT WT, NET WT, and SERT WT<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">IC<sub>50</sub> (μM) ± SD</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">DAT</th><th class="colsep0 rowsep0" align="center">NET</th><th class="colsep0 rowsep0" align="center">SERT</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="center">11.5 ± 4.3</td><td class="colsep0 rowsep0" align="center">283.6 ± 23.7</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4a</b></td><td class="colsep0 rowsep0" align="center">38.1 ± 2.3</td><td class="colsep0 rowsep0" align="center">85.7 ± 33.5</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4b</b></td><td class="colsep0 rowsep0" align="center">107.6 ± 1.5</td><td class="colsep0 rowsep0" align="center">145.8 ± 6.0</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4c</b></td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4d</b></td><td class="colsep0 rowsep0" align="center">43.7 ± 1.4</td><td class="colsep0 rowsep0" align="center">71.0 ± 27.2</td><td class="colsep0 rowsep0" align="center">209.5 ± 13.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4e</b></td><td class="colsep0 rowsep0" align="center">69.1 ± 4.1</td><td class="colsep0 rowsep0" align="center">30.0 ± 8.0</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7a</b></td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="center">63.5 ± 6.2</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7c</b></td><td class="colsep0 rowsep0" align="center">56.7 ± 0.2</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7d</b></td><td class="colsep0 rowsep0" align="center">3.1 ± 1.7</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7e</b></td><td class="colsep0 rowsep0" align="center">38.6 ± 11.3</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7f</b></td><td class="colsep0 rowsep0" align="center">40.4 ± 1.4</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7g</b></td><td class="colsep0 rowsep0" align="center">109.0 ± 14.4</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7h</b></td><td class="colsep0 rowsep0" align="center">90.5 ± 9.8</td><td class="colsep0 rowsep0" align="center">227.5 ± 47.0</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7i</b></td><td class="colsep0 rowsep0" align="center">82.6 ± 3.9</td><td class="colsep0 rowsep0" align="center">767.5 ± 77.9</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7j</b></td><td class="colsep0 rowsep0" align="center">155.7 ± 10.0</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7k</b></td><td class="colsep0 rowsep0" align="center">3.0 ± 0.3</td><td class="colsep0 rowsep0" align="center">57.6 ± 21.9</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7l</b></td><td class="colsep0 rowsep0" align="center">29.3 ± 5.1</td><td class="colsep0 rowsep0" align="center">144.2 ± 33.2</td><td class="colsep0 rowsep0" align="center">59.6 ± 2.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7m</b></td><td class="colsep0 rowsep0" align="center">4.9 ± 0.3</td><td class="colsep0 rowsep0" align="center">188.1 ± 85.6</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7n</b></td><td class="colsep0 rowsep0" align="center">283.6 ± 66.4</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7o</b></td><td class="colsep0 rowsep0" align="center">16.6 ± 0.7</td><td class="colsep0 rowsep0" align="center">67.2 ± 32.5</td><td class="colsep0 rowsep0" align="center">44.7 ± 9.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7p</b></td><td class="colsep0 rowsep0" align="center">12.8 ± 1.0</td><td class="colsep0 rowsep0" align="center">68.4 ± 33.0</td><td class="colsep0 rowsep0" align="center">57.2 ± 28.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7q</b></td><td class="colsep0 rowsep0" align="center">13.7 ± 1.8</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">70.9 ± 38.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11a</b></td><td class="colsep0 rowsep0" align="center">7.2 ± 1.3</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11b</b></td><td class="colsep0 rowsep0" align="center">1.4 ± 0.4</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11c</b></td><td class="colsep0 rowsep0" align="center">4.2 ± 0.3</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11d</b></td><td class="colsep0 rowsep0" align="center">2.0 ± 0.5</td><td class="colsep0 rowsep0" align="center">152.5 ± 15.6</td><td class="colsep0 rowsep0" align="center">118.4 ± 42.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11e</b></td><td class="colsep0 rowsep0" align="center">4.0 ± 0.6</td><td class="colsep0 rowsep0" align="center">112.6 ± 24.7</td><td class="colsep0 rowsep0" align="center">120.5 ± 58.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11f</b></td><td class="colsep0 rowsep0" align="center">5.8 ± 1.7</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11g</b></td><td class="colsep0 rowsep0" align="center">3.7 ± 0.4</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11h</b></td><td class="colsep0 rowsep0" align="center">2.4 ± 0.8</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11i</b></td><td class="colsep0 rowsep0" align="center">4.1 ± 0.4</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11j</b></td><td class="colsep0 rowsep0" align="center">1.5 ± 0.5</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">IC<sub>50</sub> values represents data from at least three independent experiments, each performed in triplicate. NA: Not active in used concentration range.</p></div></div></div><div class="NLM_p">Substitution of the amide moiety with substituted (and in some cases partially reduced) thiadiazoles and oxadiazoles led to a decrease of DAT reuptake inhibition compared to modafinil (<b>4a</b>,<b>b</b>, <b>4d</b>,<b>e</b>, and <b>7c</b>), with compound <b>4c</b> being nonspecific to all three transporters under the same experimental conditions. Modifications with pyridines (<b>7a</b>,<b>b</b>) and 5-cyclopropyl-isoxazole (<b>7e</b>) also led to the decrease of DAT reuptake inhibition. However, when the amide part was replaced with 2-methylpyrimidine-4-ol (<b>7d</b>), an analogue with higher DAT inhibition potency was obtained. Modifications of the substituent pattern on the pyrimidine moiety (<b>7f</b>–<b>j</b>) led to a decrease in DAT reuptake inhibition potency.</div><div class="NLM_p">Substitution with a 2-thiophenyl-group (<b>11b</b>) provided an analogue with higher in vitro activity on DAT (IC<sub>50</sub> = 1.4 μM) and higher selectivity compared with modafinil (<b>1</b>). An analogue containing a 3-thiophenyl-group (<b>7k</b>) also showed higher activity on DAT (IC<sub>50</sub> = 3.0 μM) than modafinil but in contrast to <b>11b</b> also displayed some affinity toward NET (IC<sub>50</sub> = 57.6 μM). Further modifications of compounds <b>7k</b> and <b>11b</b>, by introduction of halogen atoms and methyl groups on the thiophene ring, yielded analogues <b>7l</b>–<b>q</b>. With the exception of <b>7n</b> (IC<sub>50</sub> = 283.6 μM on DAT), those analogues showed only moderately lower activity than <b>7k</b> and <b>11b</b>.</div><div class="NLM_p">Compound <b>11b</b> was further modified by introducing substitutions on the phenyl rings. Introduction of an electron-donating methyl group on a single phenyl in <i>para</i>- position yielded compound <b>11d</b> (IC<sub>50</sub> = 2.0 μM on DAT), while introduction of a methyl group in <i>para</i>- position of both phenyl rings yielded compound <b>11f</b> (IC<sub>50</sub> = 5.8 μM on DAT). Introduction of electron-withdrawing groups, such as chlorine and fluorine in <i>para</i>- positions on both phenyl rings, generated compounds <b>11a</b> and <b>11e</b>, respectively, with higher activity on DAT than modafinil (IC<sub>50</sub> = 7.2 and 4.0 μM, respectively). Introduction of one bromine in <i>para</i>- position yielded analogue <b>11c</b> (IC<sub>50</sub> = 4.2 μM on DAT).</div><div class="NLM_p">Compounds <b>11i</b> and <b>11j</b> show better activity on DAT than modafinil (IC<sub>50</sub> = 4.1 and 1.6 μM, respectively). Absolute configuration was attributed by crystallographic data and was determined to be <i>S</i>,<i>S</i> and <i>R</i>,<i>R</i> for <b>11i</b> and <b>11j</b>, respectively.</div><div class="NLM_p last">Compounds <b>7d</b> and <b>11b</b>, are enantiomeric compounds. Separation of individual enantiomers into <i>R</i>- and <i>S</i>-form could lead to a further gain in binding to DAT.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Examination of Amphetamine-Like Releasing Properties</h3><div class="NLM_p">Amphetamines are substrates of DAT which cause the release of dopamine from neuronal cells.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> To investigate whether our novel dopamine reuptake inhibitors cause release of dopamine, release assays of two analogues of choice (<b>7d</b>, <b>11b</b>) have been performed.</div><div class="NLM_p">As depicted in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>, neither 10 μM <b>7d</b> nor 10 μM <b>11b</b> did alter [<sup>3</sup>H]MPP<sup>+</sup> release opposed to the positive control (10 μM <span class="smallcaps smallerCapital">d</span>-amphetamine). Elevation of intracellular Na<sup>+</sup> levels accelerates substrate efflux, hence we applied 10 μM monensin, an ionophore that promotes electroneutral Na<sup>+</sup>/H<sup>+</sup> exchange, to unambiguously confirm the absence of <b>7d</b> and/or <b>11b</b>-induced substrate release. The observations from HEK-DAT cells therefore clearly indicate that <b>7d</b> and <b>11b</b> specifically block DAT without acting as a substrate and can be clearly distinguished from amphetamine.</div><figure id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. A release assay was performed to examine whether compounds <b>7d</b> and <b>11b</b> have any ability to release the neurotransmitter from inside the cells, like amphetamine. HEK-DAT cells were grown on PDL-coated coverslips, treated with 0.1 μM [<sup>3</sup>H]MPP<sup>+</sup> (methyl-4-phenylpyridinium) at 37 °C for 20 min and washed with KHB for 40 min in superfusion chambers. Four experiments were compared, and all four started with three fractions under untreated conditions (baseline). The next four fractions were either again untreated or treated with 10 μM monensin. The final five fractions were either treated with 10 μM test compound (<b>7d</b> or <b>11b</b>) or 10 μM <span class="smallcaps smallerCapital">d</span>-amphetamine. Nonlinear regression analysis was carried out by using GraphPad Prism 6. Values are presented as mean ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> [<sup>3</sup>H]WIN 35,428 Binding Assays</h3><div class="NLM_p">Dixon plots (in which the reciprocal of bound radioactivity is plotted as a function of one inhibitor’s concentration at a fixed concentration of the second inhibitor) allow reading out whether two inhibitors can occupy the same binding site simultaneously or whether their binding is mutually exclusive.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Dixon plots showed a nearly parallel shift for <b>7d</b> in the presence of cocaine, indicating mutually exclusive binding of analogue <b>7d</b> and cocaine as shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> The same result was obtained for compound <b>11b</b> as depicted in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B.</div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Mutually exclusive binding of cocaine in the presence of analogues <b>7d</b> and <b>11b</b>. Intact HEK-DAT cells were incubated with the radioligand (∼10 nM [<sup>3</sup>H]WIN 35,428), the indicated concentrations of <b>7d</b> or <b>11b</b>, and increasing concentrations of cocaine in binding buffer. Nonspecific binding was determined in the presence of 30 μM mazindol (control). The binding data from the competition experiments is transformed into a Dixon plot by expressing the reciprocal of [<sup>3</sup>H]WIN 35,428 bound (measured in femtomole) as a function of the concentration of cocaine at a fixed concentration of <b>7d</b> or <b>11b</b>. Data are means ± SD of three experiments performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Molecular Docking and MD Simulation</h3><div class="NLM_p">The highest scored ligand positions are shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>. The docked ligand positions of compound <b>7d</b> and <b>11b</b> are very similar to each other, and both resemble the putative binding sites of cocaine and the cocaine analogue CFT<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> as well as the binding site for mephedrone analogues.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. For compound <b>7d</b> and <b>11b</b> the highest scored docking pose (A) and the final pose after the MD simulation (B) is shown. The labeled amino acids are shown explicitly on the protein backbone. The ligands move during the MD simulations from their initial position (A) to their final position (B), however, they were found not to leave the dopamine binding site. The homology model was based on the crystallized Drosophila DAT, deposited in the Protein Data Bank with the PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M48">4M48</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Molecular docking is a fast and often reliable tool to investigate possible ligand poses in the binding site of a protein, but receptor flexibility is still a challenge for docking protocols.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> This can be circumvented using MD simulations to evaluate and refine the predicted poses.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a></div><div class="NLM_p">While the simulation time of the MD simulations is sufficient to give a basis for a validation of the ligand stability at the specific binding site, it cannot be used to validate the binding and its affinity. Especially motion and/or rearrangement of the protein backbone happen on a much longer time scale and are not regarded in MD simulations at the nanosecond scale.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><div class="NLM_p">Visual inspection and the monitored RMSD values (provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b01313/suppl_file/jm7b01313_si_001.pdf" class="ext-link">Supporting Information</a>) indicate that the systems were stable during the MD simulation (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01313/suppl_file/jm7b01313_si_001.pdf" class="ext-link">Figure S-1</a> in Supporting Information). The RMSD analysis shows that the ligand is stable in the binding site, but both <b>7d</b> and <b>11b</b> changed their conformation in the binding site during the first 500 ps from the initial, docked position shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A to the conformation shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B.</div><div class="NLM_p">The used docking protocol did not regard the dynamics of the amino acids in the binding site, but it could still reproduce the binding position previously described for mephedrone analogues.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Using MD simulations the flexibility of all simulated compounds was explicitly regarded, and the binding position of both compounds changed significantly. Especially for a subsequent structure-based molecular modeling approach, these proposed binding poses are an important addition to the already established interaction patterns.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p">The interaction maps, shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>, for both simulated compounds reveal that the sulfoxide group is not involved in any interaction with the protein binding site, but interactions are governed by hydrophobic contacts and π–π and π–cation interactions.</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0005.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Different interactions between parts of the ligand and amino acids are shown as heatmap ranging from dark violet (lowest interactions), green (medium interactions) to bright yellow (highest interactions). Rows indicate part of the ligands and columns the different amino acids. The first letters before the column indicate the type of interaction, H for hydrophobic interaction, HBA for hydrogen bond acceptor, HBD for hydrogen bond donor, AR for aromatic interaction, and PI for positively ionizable interaction.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Most prominently for <b>7d</b>, hydrophobic contacts to Phe326 and Phe76 as well as Tyr156 are shown. Hydrogen bond interactions with Ala480, Arg85, and Asp476 occur also throughout the simulation. In addition to the hydrophobic contact, Arg85 has a stable π–cation interaction with <b>7d</b>.</div><div class="NLM_p">The interaction pattern for <b>11b</b> shows hydrophobic interactions between Phe326, Phe76, Phe320, Arg85, and Tyr156. There are no hydrogen bond interactions present during the simulation other than hydrophobic interactions there are weak aromatic interactions and a π–cation interaction between Arg85 and <b>11b</b>. Both compounds have similar hydrophobic interaction partners (Phe326, Phe76, Tyr156) and π–cation interactions with Arg85.</div><div class="NLM_p last">Comparing the interaction pattern to <i>R</i>- and <i>S</i>-modafinil as described by Loland et al.,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> some similarities and differences are striking. The reported interaction of the amide moiety of <i>S</i>- and <i>R</i>-modafinil with Phe76 is also present during the MD simulations for <b>7d</b> and <b>11b</b>, but the interaction with Ser321 is not observed. The reported interaction between <i>R</i>-modafinil and Tyr156 could be observed for <b>11b</b> but not for <b>7d</b>. These dissimilarities are most likely a result of the different methods used. Loland et al. utilized induced-fit docking and obtained the interactions based on the docked pose, while herein MD simulation and analysis of the interactions during the simulation were carried out.</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Neurotoxicity of Compound <b>11b</b></h3><div class="NLM_p">To evaluate basic neurotoxicity, the open field, elevated plus maze, rota rod studies, and the forced swim test were carried out. Young male rats, age 10–12 week,s were divided in two groups: 10 compound <b>11b</b> treated (treated group) and 10 DMSO-treated animals (control group). Rats were handled during 3 days before beginning of the experiment. Ten mg/kg body weight of <b>11b</b> was administered by intraperitoneal injection 30 min before each behavioral test.</div><div class="NLM_p">As shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>, there were no significant differences between treated and untreated animals for the parameter’s total distance traveled, resting time, local and large movements, average velocity, number of crossings of the center, frequency of spontaneous changes of direction, and time spent at the margin. Analogue results were obtained in the elevated plus maze test, where no significant differences between treated and untreated rats were observed regarding the parameters time spent in the left open arm, left closed arm, total distance traveled, distance traveled in open and closed arms, time spent in the open and closed arm, and the resting time (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>).</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0006.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Open field study: Total distance traveled, resting time, local and large movements, average velocity, number of crossings of the center, frequency of spontaneous changes of direction, and time spent at the margin are measured as parameters between control and <b>11b</b>-treated group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0007.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Elevated plus maze study: Time spent in the left open arm, left closed arm, total distance traveled, distance traveled in open and closed arms, time spend in the open and closed arm, and the resting time between groups are measured as determining parameters. There is no significant difference between control and treated group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Likewise, the <b>11b</b>-treated group did not show significantly altered behavior in the rotarod (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>) and forced swim test (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>) compared to the control group.</div><figure id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0008.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Rotarod study of <b>11b</b>: Times spent on the revolving rod were measured for the control and <b>11b</b>-treated group. There is no significant difference between control and treated group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig9" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0009.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Forced swim test study: Immobility times in the water were measured. There is no significant difference between control and treated group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">An observational battery was used to check all important neurological functions as listed in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>. There were no significant differences between controls and the group treated with <b>11b</b>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Results of the Neurological Observational Battery Study</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameter</th><th class="colsep0 rowsep0" align="center" char="±">control group</th><th class="colsep0 rowsep0" align="center" char="±">treated group</th><th class="colsep0 rowsep0" align="center" char="."><i>P</i> value (<i>t</i> test)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">body position</td><td class="colsep0 rowsep0" align="char" char="±">1.8 ± 0.4216</td><td class="colsep0 rowsep0" align="char" char="±">1.9 ± 0.3162</td><td class="colsep0 rowsep0" align="char" char=".">0.5566</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">locomotor activity</td><td class="colsep0 rowsep0" align="char" char="±">2.600 ± 0.9661</td><td class="colsep0 rowsep0" align="char" char="±">2.600 ± 1.350</td><td class="colsep0 rowsep0" align="char" char=".">>0.9999</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">transfer arousal</td><td class="colsep0 rowsep0" align="char" char="±">3.100 ± 0.3162</td><td class="colsep0 rowsep0" align="char" char="±">3.200 ± 0.4216</td><td class="colsep0 rowsep0" align="char" char=".">0.5566</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">piloerection</td><td class="colsep0 rowsep0" align="char" char="±">2.100 ± 0.3162</td><td class="colsep0 rowsep0" align="char" char="±">2.200 ± 0.4216</td><td class="colsep0 rowsep0" align="char" char=".">0.5566</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">limb rotation</td><td class="colsep0 rowsep0" align="char" char="±">3.300 ± 0.4830</td><td class="colsep0 rowsep0" align="char" char="±">3.400 ± 0.6992</td><td class="colsep0 rowsep0" align="char" char=".">0.7147</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">pelvic elevation</td><td class="colsep0 rowsep0" align="char" char="±">3.800 ± 0.6325</td><td class="colsep0 rowsep0" align="char" char="±">3.600 ± 0.8433</td><td class="colsep0 rowsep0" align="char" char=".">0.5566</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tail elevation</td><td class="colsep0 rowsep0" align="char" char="±">2.200 ± 0.6325</td><td class="colsep0 rowsep0" align="char" char="±">1.800 ± 0.6325</td><td class="colsep0 rowsep0" align="char" char=".">0.1744</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">finger approach</td><td class="colsep0 rowsep0" align="char" char="±">1.400 ± 1.350</td><td class="colsep0 rowsep0" align="char" char="±">1.400 ± 1.350</td><td class="colsep0 rowsep0" align="char" char=".">>0.9999</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">touch escape</td><td class="colsep0 rowsep0" align="char" char="±">1.200 ± 1.033</td><td class="colsep0 rowsep0" align="char" char="±">1.200 ± 1.033</td><td class="colsep0 rowsep0" align="char" char=".">>0.9999</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">visual placing</td><td class="colsep0 rowsep0" align="char" char="±">1.900 ± 0.3162</td><td class="colsep0 rowsep0" align="char" char="±">1.700 ± 0.4830</td><td class="colsep0 rowsep0" align="char" char=".">0.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">grip strength</td><td class="colsep0 rowsep0" align="char" char="±">2.400 ± 1.265</td><td class="colsep0 rowsep0" align="char" char="±">3.000 ± 1.700</td><td class="colsep0 rowsep0" align="char" char=".">0.3833</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">pinna reflex</td><td class="colsep0 rowsep0" align="char" char="±">1.200 ± 1.033</td><td class="colsep0 rowsep0" align="char" char="±">1.400 ± 0.9661</td><td class="colsep0 rowsep0" align="char" char=".">0.6601</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">toe pinch</td><td class="colsep0 rowsep0" align="char" char="±">1.800 ± 0.6325</td><td class="colsep0 rowsep0" align="char" char="±">1.600 ± 1.578</td><td class="colsep0 rowsep0" align="char" char=".">0.7164</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">limb tone</td><td class="colsep0 rowsep0" align="char" char="±">3.300 ± 0.6749</td><td class="colsep0 rowsep0" align="char" char="±">3.500 ± 0.8498</td><td class="colsep0 rowsep0" align="char" char=".">0.5676</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tail pinch</td><td class="colsep0 rowsep0" align="char" char="±">0.8000 ± 0.7888</td><td class="colsep0 rowsep0" align="char" char="±">0.9000 ± 0.8756</td><td class="colsep0 rowsep0" align="char" char=".">0.7915</td></tr></tbody></table></div></div><div class="NLM_p last">Behavioral assays did not show any behavioral/toxic changes following daily intraperitoneal administrations of <b>11b</b> at an intraperitoneal dose of 10 mg/kg body weight. This makes <b>11b</b> a promising compound for further development, and indeed, first behavioral studies of this compound class showed cognitive enhancement that challenged synthesis and further development.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26458" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26458" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In the current study, a novel series of modafinil analogues containing five- and six-membered aromatic heterocycles have been synthesized. Analogues were tested for dopamine reuptake inhibition, and several compounds showed similar or improved activity and selectivity (DAT vs NET and SERT) compared to modafinil. Compounds <b>7d</b> and <b>11b</b> have been selected for further testing. When tested in dopamine release assay, both compounds caused no efflux of dopamine (amphetamine-like properties). Binding studies have demonstrated that analogues <b>7d</b> and <b>11b</b> are competing for binding with cocaine on DAT. This result was further supported by a docking study of the compounds to a homology model of human DAT. While molecular docking experiments support the finding that <b>7d</b> and <b>11b</b> are binding into the same putative binding pocket of cocaine, molecular dynamics simulations indicate a different binding spot to that of cocaine, i.e., different amino acids are involved in binding of <b>7d</b> and <b>11b</b> within the pocket. As compound <b>11b</b> showed the highest reuptake inhibition, it was therefore the compound of choice for neurotoxicity studies. Results indicate no general neurotoxicity of <b>11b</b>. Therefore, by introducing heterocyclic modifications to the modafinil skeleton, a novel series of atypical DAT inhibitors has been synthesized. Several compounds showed improved activity and specificity compared to modafinil, which together with absence of neurotoxicity for at least one tested compound, makes them suitable for further development.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Procedures</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68716" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68716" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Synthesis</h3><div class="NLM_p"><sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on a Bruker Avance 500 NMR spectrometer (UltraShield) using a 5 mm switchable probe (PA BBO 500SB BBF-H-D-05-Z, 1H, BB = <sup>19</sup>F and <sup>31</sup>P, <sup>15</sup>N) with <i>z</i> axis gradients and automatic tuning and matching accessory (Bruker BioSpin). The resonance frequency for <sup>1</sup>H NMR was 500.13 MHz and for <sup>13</sup>C NMR 125.75 MHz. All measurements were performed for a solution in fully deuterated chloroform or DMSO at 298 K. Standard 1D and gradient-enhanced (ge) 2D experiments, like double quantum filtered (DQF) COSY, NOESY, HSQC, and HMBC, were used as supplied by the manufacturer. Chemical shifts are referenced internally to the residual, nondeuterated solvent signal for chloroform <sup>1</sup>H (δ 7.26 ppm) or DMSO <sup>1</sup>H (δ 2.50 ppm) and to the carbon signal of the solvent for chloroform <sup>13</sup>C (δ 77.00 ppm) or DMSO <sup>13</sup>C (δ 39.57 ppm).</div><div class="NLM_p">HRESIMS spectra were obtained on a maXis HD ESI-Qq-TOF mass spectrometer (Bruker Daltonics, Bremen, Germany). Samples were dissolved to 20 μg/mL in MeOH and directly infused into the ESI source at a flow rate of 3 μL/min with a syringe pump. The ESI ion source was operated as follows: capillary voltage, 0.9 to 4.0 kV (individually optimized); nebulizer, 0.4 bar (N<sub>2</sub>); dry gas flow, 4 L/min (N<sub>2</sub>); dry temperature, 200 °C. Mass spectra were recorded in the range of <i>m</i>/<i>z</i> 50–1550 in the positive-ion mode. The sum formulas were determined using Bruker Compass DataAnalysis 4.2 based on the mass accuracy (Δ<i>m</i>/<i>z</i> ≤ 2 ppm) and isotopic pattern matching (SmartFormula algorithm).</div><div class="NLM_p">The purity of the compounds was determined by HPLC either on an UltiMate 3000 series system equipped with VWD detector (Dionex/Thermo Fisher Scientific, Germering, Germany) or on LC-2010A HT liquid chromatograph device (Shimadzu Corporation, Tokyo, Japan). Separation was carried out on an Acclaim 120 C18, 2.1 mm × 150 mm, 3 μm HPLC column (Thermo Fisher Scientific) using LC-MS-grade water and acetonitrile as mobile phases A and B, respectively. The sample components were separated and eluted with a linear gradient from 10% to 90% B in 25 min followed by an isocratic column cleaning and re-equilibration step. The flow rate was 0.2 mL/min and the column oven temperature was set to 25 °C. The purity was determined from the UV chromatogram (254 nm) as the ratio of the peak area of the compound to the total peak area (i.e., the sum of the areas of all peaks that were not present in the solvent blank). On the basis of the HPLC data, all final compounds are ≥95% pure. Melting points were measured on Leica Galen III apparatus (Leica Biosystems, Germany).</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> 5-((Benzhydrylthio)methyl)-1,3,4-thiadiazol-2(3<i>H</i>)-one (<b>3a</b>) and 5-((Benzhydrylthio)methyl)-2-methoxy-2,3-dihydro-1,3,4-thiadiazole (<b>3b</b>)</h4><div class="NLM_p last">To a solution of diphenylmethanethiol (1.35 g, 6.75 mmol) in DMF (15 mL) was added <i>t</i>-BuOK (1.34 g) and 2-(chloromethyl)-5-methoxy-1, 3, 4-thiadiazole (0.89, 5.4 mmol), and the reaction mixture was stirred at room temperature overnight. Then, saturated aqueous NH<sub>4</sub>Cl solution was added, and the reaction mixture was extracted with EtOAc. After removal of the solvent, the residue was purified by silica gel column chromatography (hexane:CH<sub>2</sub>Cl<sub>2</sub> 1:1), affording 0.20 g of compound <b>3b</b> (yield 11%) and 0.60 g of compound <b>3a</b> (yield 35%).</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> 2-((Benzhydrylthio)methyl)-5-methyl-1,3,4-thiadiazole (<b>3c</b>)</h4><div class="NLM_p last">To a solution of diphenylmethanethiol (2.00 g, 10.0 mmol) in DMF (20 mL) was added <i>t</i>-BuOK (1.0 g) and 2-(chloromethyl)-5-methyl-1,3,4-thiadiazole (1.22 g, 8.2 mmol), and the reaction mixture was stirred overnight at room temperature. Then, a saturated aqueous NH<sub>4</sub>Cl solution was added and the reaction mixture was extracted with EtOAc. After removal of the solvent, the residue was purified by column chromatography on silica gel (hexane:CH<sub>2</sub>Cl<sub>2,</sub> 1:1), to give 1.14 g of compound <b>3c</b> (yield 44%).</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 5-((Benzhydrylthio)methyl)-3-(trifluoromethyl)-1,2,4-oxadiazole (<b>3d</b>)</h4><div class="NLM_p last">To a solution of diphenylmethanethiol (1.50 g, 7.5 mmol) in dioxane (20 mL) was added DIPEA (1.4 mL) and 5-(chloromethyl)-3-(trifluoromethyl)-1,2,4-oxadiazole (1.33 g, 7.1 mmol), and the reaction mixture was stirred at room temperature overnight. Then, the reaction mixture was quenched with water and extracted with EtOAc. After removal of the solvent, the residue was purified by flash chromatography on silica gel (CH<sub>2</sub>Cl<sub>2</sub>) to give 2.47 g of compound <b>3d (</b>yield >95%).</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 5-(Benzhydrylthiomethyl)-4-isopropyl-1,2,3-thiadiazole (<b>3e</b>)</h4><div class="NLM_p last">To the solution of diphenylmethanethiol (1.87 g, 9.30 mmol) in anhydrous 1,4-dioxane (5 mL), 5-(chloromethyl)-4-isopropyl-1,2,3-thiadiazole (1.31 g, 7.44 mmol) and (1.54 mL) DIPEA (1.2 g, 9.30 mmol) were added dropwise. The resulting mixture was stirred at room temperature for 3 h and evaporated in vacuum until dryness. The crude product was purified by column chromatography on silica gel (EtOAc:hexane, 1:20), affording 1.91 g of the colorless crystals (yield 47%).</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 5-((Benzhydrylthio)methyl)-2-methoxy-2,3-dihydro-1,3,4-thiadiazole (<b>4a</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (27.6%, 1.46 mL) was added to the solution of <b>3a</b> (1.5 g) in glacial acetic acid (35 mL). The reaction mixture was stirred at room temperature during 24 h. The solvent was evaporated in vacuum until dryness. The residue was diluted with water, and the precipitate was filtered and purified by flash chromatography on silica gel (hexane:EtOAc, 3:1) to give 1.0 g of compound <b>4a</b> as white solid (yield 64%); mp 142–145 °C; retention time 16.96 min; purity 95.5%. HRESIMS <i>m</i>/<i>z</i> 353.0393 [M + Na]<sup>+</sup> (calcd for C<sub>16</sub>H<sub>14</sub>N<sub>2</sub>NaO<sub>2</sub>S<sub>2</sub><sup>+</sup>, 353.0389, Δ = −1.0 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 10.17 (br s, 1H), 7.44–7.39 (m, 10H), 5.10 (s, 1H), 3.89 (d, <i>J</i> = 15 Hz, 1H), 3.74 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 172.2 (C═O), 145.1 (C), 134.3 (C), 133.6 (C), 129.6 (2CH), 129.5 (2CH), 128.9 (2CH + 2<i>p</i>-CH), 128.8 (2CH), 71.7 (CH), 51.5 (CH<sub>2</sub>).</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 5-((Benzhydrylsulfinyl)methyl)-2-methoxy-2,3-dihydro-1,3,4-thiadiazole (<b>4b</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (27.6%, 0.07 mL) was added to the solution of <b>3b</b> (6.1 mmol, 0.2 g) in glacial acetic acid (10 mL). The reaction mixture was stirred at room temperature during 24 h. Two new compounds were detected on TLC. The solvent was evaporated in vacuum until dryness. The residue was diluted with water, and the precipitate was filtered off and purified by column chromatography on silica gel (CH<sub>2</sub>Cl<sub>2</sub>) to give 0.07 g of compound <b>4b</b> as white solid (yield 33%); mp 127–129 °C; retention time 18.61 min; purity 98.8%. HRESIMS <i>m</i>/<i>z</i> 367.0549 [M + Na]<sup>+</sup> (calcd for C<sub>17</sub>H<sub>16</sub>N<sub>2</sub>NaO<sub>2</sub>S<sub>2</sub><sup>+</sup>, 367.0545, Δ = −1.1 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.44–7.38 (m, 10H), 5.10 (s, 1H), 3.89 (d, <i>J</i> = 15 Hz, 1H), 3.7 (d, <i>J</i> = 15 Hz, 1H), 3.56 (s, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 169.8 (C), 142.1 (C), 134.6 (C), 133.6 (C), 129.5 (4CH), 128.9 (2CH), 128.8 (4CH + 2<i>p</i>-CH), 71.9 (CH), 52.0 (CH<sub>2</sub>), 34.3 (CH<sub>3</sub>).</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 2-((Benzhydrylsulfinyl)methyl)-5-methyl-1,3,4-thiadiazole (<b>4c</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (27.6%, 0.04 mL) was added to the solution of <b>3c</b> (3.65 mmol, 1.14 g) in glacial acetic acid (15 mL). The reaction mixture was stirred during 24 h at room temperature. The solvent was evaporated in vacuum until dryness, and the residue was diluted with water and filtered off to separate the precipitate, which was purified by column chromatography on silica gel (CH<sub>2</sub>Cl<sub>2</sub>) to give 0.85 g of <b>4c</b> as white solid (yield 71%); mp 155–156 °C; retention time 17.14 min; purity 99.2%. HRESIMS <i>m</i>/<i>z</i> 351.0601 [M + Na]<sup>+</sup> (calcd for C<sub>17</sub>H<sub>16</sub>N<sub>2</sub>NaOS<sub>2</sub><sup>+</sup>, 351.0596, Δ = −1.4 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.43–7.34 (m, 10H), 5.0 (s, 1H), 4.32 (d, <i>J</i> = 15 Hz, 1H), 4.11 (d, <i>J</i> = 15 Hz, 1H), 2.83 (3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 167.53 (C), 158.2 (C), 134.3 (C), 133.7 (C), 129.5 (4CH), 129.1 (2CH), 128.8 (2CH + 2<i>p</i>-CH), 128.5 (2CH), 70.4 (CH), 48.0 (CH<sub>2</sub>), 15.7 (CH<sub>3</sub>).</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 5-((Benzhydrylsulfinyl)methyl)-3-(trifluoromethyl)-1,2,4-oxadiazole (<b>4d</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (27.6%, 0.78 mL) was added to the solution of <b>3d</b> (7.05 mmol, 2.47 g) in glacial acetic acid (20 mL). The reaction mixture was stirred during 24 h at room temperature. The solvent was evaporated in vacuum until dryness. The residue was diluted with water and extracted with CH<sub>2</sub>Cl<sub>2</sub> then evaporated and purified by column chromatography on silica gel (CH<sub>2</sub>Cl<sub>2</sub>) to give 2.11 g of a white solid (yield 81%) of <b>4d</b>; mp 90–92 °C; retention time 22.72 min; purity 99.6%. HRESIMS <i>m</i>/<i>z</i> 389.0542 [M + Na]<sup>+</sup> (calcd for C<sub>17</sub>H<sub>13</sub>F<sub>3</sub>N<sub>2</sub>NaO<sub>2</sub>S<sup>+</sup>, 389.0542, Δ = 0.1 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.50–7.39 (m, 10H), 5.31 (s, 1H), 4.29 (d, <i>J</i> = 15 Hz, 1H), 4.01 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 173.7 (C), 161.9 (q, <sup>2</sup><i>J</i>(<sup>13</sup>C, <sup>19</sup>F) = 40.6 Hz, C), 133.9 (C), 133.2 (C), 129.7 (2CH), 129.4 (2CH), 129.1 (CH), 129.0 (2 <i>m</i>-CH + <i>p</i>-CH), 117.6 (q, <sup>1</sup><i>J(</i><sup>13</sup>C, <sup>19</sup>F) = 270.8 Hz, CF<sub>3</sub>), 72.5 (CH), 46.0 (CH<sub>2</sub>).</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 5-(Benzhydrylsulfinylmethyl)-4-isopropyl-1,2,3-thiadiazole (<b>4e</b>)</h4><div class="NLM_p last">To the solution of <b>3e</b> (2.08 g, 6.10 mmol) in glacial HOAc (10 mL), the solution of 27.6% H<sub>2</sub>O<sub>2</sub> in water (6.10 mmol, 0.75 mL) was added. The reaction mixture was stirred at room temperature for 24 h and evaporated in vacuum until dryness. The resulting oil was dissolved in EtOAc and washed with water (3 × 50 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and evaporated until dryness. The crude product was purified with column chromatography on silica gel (EtOAc/hexanes 1:3 →1:1 → EtOAc). Pure product <b>4e</b> was isolated as a white solid (yield 65%, 1.41 g); mp 107–108 °C; retention time 21.98 min; purity 99.9%. HRESIMS <i>m</i>/<i>z</i> 379.0910 [M + Na]<sup>+</sup> (calcd for C<sub>19</sub>H<sub>20</sub>N<sub>2</sub>NaOS<sub>2</sub><sup>+</sup>, 379.0909, Δ = −0.2 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.42–7.38 (m, 10H), 4.14 (d, <i>J</i> = 15 Hz, 1H), 3.96 (d, <i>J</i> = 15 Hz, 1H), 2.75 (m, 1H), 1.46 (d, <i>J</i> = 7 Hz, 3H), 1.28 (d, <i>J</i> = 7 Hz, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 167.2 (C), 136.1 (C), 134.4 (C), 133.7 (C), 129.7 (2CH), 129.2 (2CH), 129.0 (CH), 128.9 (2CH), 128.8 (CH), 128.4 (2CH), 71.0 (CH), 45.8 (CH<sub>2</sub>), 27.6 (CH), 23.4 (CH<sub>3</sub>), 22.4 (CH<sub>3</sub>).</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> [(Diphenylmethyl)sulfanyl]methanimideamide (<b>5</b>)</h4><div class="NLM_p last">Diphenylmethanol (30.9 g, 168 mmol) and thiourea (14.7 g, 194 mmol) were mixed in a 1 L three-neck round bottomed flask and dissolved in 150 mL of MeOH. The reaction mixture was refluxed for 0.5 h and 48% HBr (87.4 mL, 772 mmol) was added dropwise over 1 h and the reaction mixture was stirred under reflux for additional 2.5 h. After cooling to the normal temperature, MeOH was removed in vacuum. A pale-yellow product was obtained and suspended in 100 mL of DCM under stirring for 0.5 h at room temperature. Then solution was filtered and washed with cold DCM. The solid residue was suspended in 100 mL of water and stirred for additional 0.5 h at room temperature. Finally, the product was filtered under reduced pressure, washed with cold water, and dried under high vacuum to give 38.4 g of a white, powdered solid (yield 71%).</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 2-((Benzhydrylthio)methyl)-3,4-dimetohoxypyridine (<b>6a</b>)</h4><div class="NLM_p last">In a round-bottom flask 3.33 g (10.30 mmol) of <b>5</b> was dissolved in 75 mL of MeOH. Afterward, 2.24 g (10.3 mmol) of 2-(chloromethyl)-3,4-dimethoxypyridine and 6.9 g (5.0 equiv) of potassium carbonate were also added to the solution. Then, the reaction mixture was left under stirring for 24 h at room temperature. The MeOH was evaporated, and the residual mixture was diluted in water. Then, the reaction products were extracted with EtOAc and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Then, the solvent was removed by rotary evaporation. The crude product was purified by flash column chromatography on silica gel (5% MeOH in DCM), affording 2.60 g of the isolated solid (yield 59%).</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 4-((Benzhydrylthio)methyl)pyridine (<b>6b</b>)</h4><div class="NLM_p last">In a 250 mL round bottomed flask, 3.23 g (10 mmol) of <b>5</b> is dissolved in 100 mL of MeOH. Afterward, 2.53 g (10 mmol) of 4-(bromomethyl)pyridine hydrochloride and 6.9 g (5 equiv) of potassium carbonate were added to the mixture. The mixture was left to stir at room temperature for 24 h. MeOH was evaporated followed of addition of water. Reaction mixture was extracted with EtOAc, and organic extracts were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and filtered off, and EtOAc was removed by rotary evaporation. The crude product was purified by flash column chromatography on silica gel (5% MeOH in DCM), affording 2.0 g of the solid product (yield 68%).</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 5-((Benzhydrylthio)methyl)-3-methyl-1,2,4-oxadiazole (<b>6c</b>)</h4><div class="NLM_p last">To a suspension of thiouronium salt 5 (2 g, 6.2 mmol) in MeOH (30 mL) was added solid K<sub>2</sub>CO<sub>3</sub> (5 equiv, 4.28 g) and 5-(chloromethyl)-3-methyl-1,2,4-oxadiazole (0.83 g, 6.2 mmol) in several portions. The reaction mixture was stirred at room temperature for 24 h. MeOH was evaporated and water was added, then dilute reaction mixture was extracted with CH<sub>2</sub>Cl<sub>2</sub> and purified by flash chromatography on silica gel (CH<sub>2</sub>Cl<sub>2</sub>:petroleum ether, 1:1), to give 1.32 g of compound <b>6c</b> (yield 72%).</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 6-((Benzhydrylthio)methyl)-2-methylpyrimidin-4-ol (<b>6d</b>)</h4><div class="NLM_p last">To a suspension of thiouronium salt <b>5</b> (3.0 g, 9.3 mmol) in MeOH (30 mL) was added solid K<sub>2</sub>CO<sub>3</sub> (5 equiv, 6.42 g) and 6-(chloromethyl)-2-methylpyrimidin-4-ol (1.47, 9.3 mmol) in several portions. The reaction mixture was stirred at room temperature for 24 h. Then, the MeOH was evaporated, and residual was diluted with water and extracted with EtOAc. Organic layer was separated and acidified with 12 M HCl solution under stirring. The precipitate was filtered to give 2.2 g of compound <b>6d</b> (yield 73%).</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 3-((Benzhydrylthio)methyl)-5-cyclopropylisoxazole (<b>6e</b>)</h4><div class="NLM_p last">To a suspension of thiouronium salt <b>5</b> (3.0 g, 9.3 mmol) in MeOH (30 mL) was added solid K<sub>2</sub>CO<sub>3</sub> (5 equiv, 6.42 g) and 3-(chloromethyl)-5-cyclopropylisoxazole (1.46 g, 9.3 mmol) in portions. The reaction mixture was stirred at room temperature for 24 h. Then, the MeOH was evaporated and residual was diluted with water. The product was extracted with CH<sub>2</sub>Cl<sub>2</sub> and purified by flash chromatography on silica gel (CH<sub>2</sub>Cl<sub>2</sub>:petroleum ether, 3:5), to give 2.89 g of the isolated <b>6e</b> (yield 97%).</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 2-((Benzhydrylthio)methyl)-6-(trifluoromethyl)pyrimidin-4-ol (<b>6f</b>)</h4><div class="NLM_p last">To a suspension of thiouronium salt <b>5</b> (303 mg, 0.93 mmol) in MeOH (30 mL) was added solid K<sub>2</sub>CO<sub>3</sub> (5 equiv, 0.64 g) and 2-(chloromethyl)-6-(trifluoromethyl)pyrimidin-4-ol (200 mg, 0.93 mmol) in several portions. The reaction mixture was stirred at room temperature for 24 h. The MeOH was evaporated and the residual products were diluted with water and aqueous solution was extracted with CHCl<sub>3</sub> and then acidified with 12 M HCl. The precipitate was filtered to give 0.22 g of isolated compound <b>6f</b> (yield 63%).</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 2-((Benzhydrylthio)methyl)-6-methylpyrimidin-4-ol (<b>6g</b>)</h4><div class="NLM_p last">Same procedure as for <b>6f</b>. Final isolated product gave 178 mg of <b>6g</b> (yield 59%).</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 2-((Benzhydrylthio)methyl)-4-methoxy-6-methylpyrimidine (<b>6h</b>)</h4><div class="NLM_p last">Same procedure as for <b>6f</b>. Reaction mixture was extracted with CH<sub>2</sub>Cl<sub>2</sub>, and extracts were purified by flash chromatography on silica gel (CH<sub>2</sub>Cl<sub>2</sub>), to give 0.25 g of isolated <b>6h</b> (yield 80%).</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 4-((Benzhydrylthio)methyl)-2,6-dimethylpyrimidine (<b>6i</b>)</h4><div class="NLM_p last">The synthesis was carried out by the same procedure used for <b>6h</b>, affording 0.26 g of the isolated <b>6i</b> (yield 88%).</div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 2-((Benzhydrylthio)methyl)-5-methylpyrimidine (<b>6j</b>)</h4><div class="NLM_p last">To a suspension of thiouronium salt <b>5</b> (0.35 mg, 1.08 mmol) in MeOH (30 mL) was added solid K<sub>2</sub>CO<sub>3</sub> (5 equiv, 0.74 g) and 2-(chloromethyl)-5-methylpyrimidine hydrochloride (0.194 g, 1.08 mmol) in several portions. The reaction mixture was stirred at room temperature for 24 h. The MeOH was evaporated, and the residual products were extracted with CH<sub>2</sub>Cl<sub>2</sub> and purified by flash chromatography on silica gel (CH<sub>2</sub>Cl<sub>2</sub>) to give 0.30 mg of isolated <b>6j</b> (yield 90%).</div></div><div id="sec5_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 3-((Benzhydrylthio)methyl)thiophene (<b>6k</b>)</h4><div class="NLM_p last">To a suspension of thiouronium salt <b>5</b> (4.9 g, 15.0 mmol) in MeOH (50 mL) was added solid K<sub>2</sub>CO<sub>3</sub> (5 equiv, 10.35 g) and 3-(chloromethyl)-thiophene (2.0 g, 15.0 mmol) in several portions. The reaction mixture was stirred at room temperature for 24 h. The MeOH was evaporated, and the residual products were extracted with CH<sub>2</sub>Cl<sub>2</sub> and purified by flash chromatography on silica gel (CH<sub>2</sub>Cl<sub>2</sub>), affording 3.26 g of isolated <b>6k</b> (yield 73%).</div></div><div id="sec5_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 3-((Benzhydrylthio)methyl)-2-bromothiophene (<b>6l</b>)</h4><div class="NLM_p last">To a suspension of thiouronium salt <b>5</b> (0.18 g, 0.6 mmol) in MeOH (25 mL) was added solid K<sub>2</sub>CO<sub>3</sub> (5 equiv, 0.385 g) and 3-(chloromethyl)-thiophene (0.211 g, 0.6 mmol) in several portions. The reaction mixture was stirred at room temperature for 24 h. The MeOH was evaporated and the residual products were extracted with EtOAc and purified by flash chromatography on silica gel (petroleum ether:ethyl acetate, 1:1), affording 0.15 g of isolated <b>6l</b> (yield 67%).</div></div><div id="sec5_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 2-((Benzhydrylthio)methyl)-3-chlorothiophene (<b>6m</b>)</h4><div class="NLM_p last">To a suspension of thiouronium salt <b>5</b> (0.96 g, 2.9 mmol) in MeOH (50 mL) was added solid K<sub>2</sub>CO<sub>3</sub> (5 equiv, 2.06 g) and 3-chloro-2-(chloromethyl) thiophene (0.50 g, 2.9 mmol) in several portions. The reaction mixture was stirred at room temperature for 24 h. The MeOH was evaporated and the residual products were extracted with EtOAc and purified by flash chromatography on silica gel (petroleum ether:ethyl acetate, 4:1), to give 0.33 g of isolated <b>6m</b> (yield 33%).</div></div><div id="sec5_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 4-((Benzhydrylthio)methyl)-2,3-dimethylthiophene (<b>6n</b>)</h4><div class="NLM_p last">To a suspension of thiouronium salt <b>5</b> (0.582 g, 1.8 mmol) in MeOH (25 mL) was added solid K<sub>2</sub>CO<sub>3</sub> (5 equiv, 1.24 g) and 4-(chloromethyl)-2,3-dimethylthiophene (0.289 g, 1.8 mmol) in several portions. The reaction mixture was stirred at room temperature overnight. The MeOH was evaporated and the residual products were extracted with EtOAc and concentrated under reduced pressure to give 0.53 g of isolated <b>6n</b> (yield 89%).</div></div><div id="sec5_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 4-((Benzhydrylthio)methyl)-2-bromothiophene (<b>6o</b>)</h4><div class="NLM_p last">To a suspension of thiouronium salt <b>5</b> (0. 306 g, 0.9 mmol) in MeOH (30 mL) was added solid K<sub>2</sub>CO<sub>3</sub> (5 equiv, 0.62 g) and 2-bromo-4-(chloromethyl) thiophene (0.211 g, 0.9 mmol) in portions. The reaction mixture was stirred at room temperature overnight. The MeOH was evaporated, and the residual products were extracted with EtOAc and concentrated under reduced pressure to give 0.30 g of isolated <b>6o</b> (yield 89%).</div></div><div id="sec5_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 2-((Benzhydrylthio)methyl)-5-chlorothiophene (<b>6p</b>)</h4><div class="NLM_p last">To a suspension of thiouronium salt <b>5</b> (0.71 g, 2.2 mmol) in MeOH (30 mL) was added solid K<sub>2</sub>CO<sub>3</sub> (5 equiv, 1.51 g) and 2-chloro-5-(chloromethyl) thiophene (0.30 mg, 1.8 mmol) in portions. The reaction mixture was stirred at room temperature overnight. The MeOH was evaporated and the residual products were extracted with EtOAc and concentrated under reduced pressure to give 0. 50 mg of isolated <b>6p</b> (yield 84%).</div></div><div id="sec5_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 2-((Benzhydrylthio)methyl)-4-bromothiophene (<b>6q</b>)</h4><div class="NLM_p last">To a suspension of thiouronium salt <b>5</b> (0.73 g, 2.3 mmol) in MeOH (30 mL) was added solid K<sub>2</sub>CO<sub>3</sub> (5 equiv, 1.59 g) and 4-bromo-2-(chloromethyl) thiophene (0.40 g, 1.9 mmol) in portions. The reaction mixture was stirred at room temperature overnight. The MeOH was evaporated and the residual products were extracted with EtOAc and concentrated under reduced pressure to give 0.61 mg of isolated <b>6p</b> (yield 85%).</div></div><div id="sec5_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 2-((Benzhydrylsulfinyl)methyl)-3,4-dimethoxypyridine (<b>7a</b>)</h4><div class="NLM_p last">In a 100 mL round-bottom flask, 1.917 g (5.45 mmol) of <b>6a</b> was dissolved in 10 mL of glacial acetic acid. Then 0.625 mL (5.51 mmol) of 30% H<sub>2</sub>O<sub>2</sub> was dropped into the solution and the reaction mixture was stirred for 12 h at room temperature. Then the reaction mixture was neutralized with cold 5% sodium bicarbonate solution. Reaction products were extracted with EtOAC, and the organic extracts were collected, combined and dried over anhydrous Na<sub>2</sub>SO<sub>4.</sub> Then the organic solution was concentrated and the residual material was purified by flash column chromatography on silica gel (5% MeOH in DCM), to yield 1.49 g of the isolated <b>7a</b> as white powder (yield 78%); mp 131–132 °C; retention time 18.81 min; purity 95.3%. HRESIMS <i>m</i>/<i>z</i> 390.1133 [M + Na]<sup>+</sup> (calcd for C<sub>21</sub>H<sub>21</sub>NNaO<sub>3</sub>S<sup>+</sup>, 390.1134, Δ = 0.3 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 8.25 (d, <i>J</i> = 5 Hz, 1H), 7.53–7.31 (m, 10H), 6.80 (d, <i>J</i> = 5 Hz, 1H), 5.31 (s, 1H), 4.06 (d, <i>J</i> = 15 Hz, 1H), 4.0 (d, <i>J</i> = 15 Hz, 1H), 3.90 (s, 3H), 3.80 (s, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 158.6 (C), 146.0 (CH), 145.8 (C), 145.3 (C), 136.6 (C), 134.3 (C), 130.0 (2CH), 129.0 (2CH + 2 <i>m</i>-CH), 128.5 (2 <i>m</i>-CH), 128.1 (CH), 128.0 (CH), 107.3 (CH), 70.4 (CH), 61.5 (CH<sub>3</sub>), 55.7 (CH<sub>3</sub>), 51.5 (CH<sub>3</sub>), 52.5 (CH<sub>2</sub>).</div></div><div id="sec5_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 4-((Benzhydrylsulfinyl)methyl)pyridine (<b>7b</b>)</h4><div class="NLM_p last">In a 100 mL round-bottom flask, (2.0 g, 6.87 mmol) of <b>6b</b> was dissolved in 10 mL of glacial acetic acid. Then 30% H<sub>2</sub>O<sub>2</sub> (0.79 mL, 6.97 mmol)) was dropped into the solution, and the reaction mixture was stirred for 12 h at room temperature. Then, the reaction mixture was neutralized with cold 5% sodium bicarbonate solution. The aqueous neutral solution was extracted (3 × 50 mL) with ethyl acetate. Organic extracts were collected, combined, and dried over anhydrous Na<sub>2</sub>SO<sub>4.</sub> The organic solution was concentrated, and residual was purified by flash column chromatography on silica gel (5% MeOH in DCM) affording 1.9 g of the isolated <b>7b</b> as yellow crystals (yield 90%); mp 139–140 °C; retention time 17.06 min; purity 98.2%. HRESIMS <i>m</i>/<i>z</i> 330.0919 [M + Na]<sup>+</sup> (calcd for C<sub>19</sub>H<sub>17</sub>NNaOS<sup>+</sup>, 330.0923, Δ = 1.3 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 8.59 (m, 2H), 7.42–7.35 (m, 10H), 7.15 (m, 2H), 4.76 (s, 1H), 3.86 (d, <i>J</i> = 15 Hz, 1H), 3.67 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 149.6 (2CH), 139.8 (C), 135.10 (C), 134.2 (C), 129.5 (2CH), 129.3 (2CH), 128.8 (2CH), 128.7 (CH), 128.6 (2CH + <i>p</i>-CH), 125.3 (2CH), 71.4 (CH), 55.1 (CH<sub>2</sub>).</div></div><div id="sec5_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 5-[(Benzhydrylsulfinyl)methyl]-3-methyl-1,2,4-oxadiazole (<b>7c</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (27.6%, 0.4 mL) was added to the solution of <b>6c</b> (1.32 g, 4.46 mmol) in glacial acetic acid (15 mL). The reaction mixture was stirred during 24 h at room temperature. The solvent was evaporated in vacuum until dryness. The residue was diluted with water and extracted with CH<sub>2</sub>Cl<sub>2</sub> then organic extracts were evaporated and the solid product was purified by column chromatography on silica gel (hexane, hexane:CH<sub>2</sub>Cl<sub>2</sub> 1:1, CH<sub>2</sub>Cl<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>:EtOAc 1:1), affording 1.20 g of the isolated <b>7c</b> as white crystals (yield 86%); mp 109–110 °C; retention time 18.88 min; purity 97.5%. HRESIMS <i>m</i>/<i>z</i> 351.0568 [M + Na]<sup>+</sup> (calcd for C<sub>17</sub>H<sub>16</sub>KN<sub>2</sub>O<sub>2</sub>S<sup>+</sup>, 351.0564, Δ = −1.3 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.51–7.37 (m, 9H), 5.29 (s, 1H), 4.11 (d, <i>J</i> = 15 Hz, 1H), 3.90 (d, <i>J</i> = 15 Hz, 1H), 2.44 (s, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 170.4 (C), 167.8 (C), 134.7 (C), 133.3 (C), 129.6 (2CH), 129.5 (2CH), 128.9 (2CH), 128.7 (2CH + 2 <i>p</i>-CH), 71.5 (CH), 45.9 (CH<sub>2</sub>), 11.7 (CH<sub>3</sub>).</div></div><div id="sec5_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 6-[(Benzhydrylsulfinyl)methyl]-2-methylpyrimidin-4-ol (<b>7d</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 0.55 mL) was added to the solution of <b>6d</b> (1.93 g, 6.0 mmol) in glacial acetic acid (20 mL). The reaction mixture was stirred during 24 h at room temperature. The solvent was evaporated in vacuum until dryness. The residue was diluted with water and extracted with CH<sub>2</sub>Cl<sub>2</sub> then organic extracts were evaporated and the residual product was purified by column chromatography on silica gel (hexane, hexane:CH<sub>2</sub>Cl<sub>2</sub> 1:1, CH<sub>2</sub>Cl<sub>2</sub>), affording a solid product that was finally crystallized from EtOH to give 1.40 g of the isolated <b>7d</b> as white crystals (yield 69%); mp 181–182 °C; retention time 14.56 min; purity 99.4%. HRESIMS <i>m</i>/<i>z</i> 361.0984 [M + Na]<sup>+</sup> (calcd for C<sub>19</sub>H<sub>18</sub>N<sub>2</sub>NaO<sub>2</sub>S<sup>+</sup>, 361.0981, Δ = −0.7 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 12.9 (br s, OH), 7.52–7.35 (m, 10H), 6.25 (s, 1H), 5.19 (s, 1H), 3.71 (d, <i>J</i> = 15 Hz, 1H), 3.57 (d, <i>J</i> = 15 Hz, 1H), 2.45 (s, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 164.6 (C), 159.1 (C), 159.0 (C), 135.6 (C), 134.2 (C), 129.6 (2CH), 128.8 (2CH), 128.7 (2CH), 128.5 (2CH), 113.7 (CH), 71.7 (CH), 57.9 (CH<sub>2</sub>), 21.7 (CH<sub>3</sub>).</div></div><div id="sec5_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 3-[(Benzhydrylsulfinyl)methyl]-5-cyclopropylisoxazole (<b>7e</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (27.6%, 0.8 mL) was added to the solution of <b>6e</b> (2.8 g, 8.7 mmol) in glasial acetic acid (20 mL). The reaction mixture was stirred during 24 h at room temperature. The solvent was evaporated in vacuum until dryness. The residue was diluted with water and extracted with CH<sub>2</sub>Cl<sub>2</sub> then organic extracts were evaporated and the residual product was purified by column chromatography on silica gel (EtOAc:petroleum ether 1:3), affording 1.76 g of the isolated <b>7e</b> as white powder (yield 60%); mp 68–70 °C; retention time 21.28 min; purity 99.3%. HRESIMS <i>m</i>/<i>z</i> 360.1030 [M + Na]<sup>+</sup> (calcd for C<sub>20</sub>H<sub>19</sub>NNaO<sub>2</sub>S<sup>+</sup>, 360.1029, Δ = −0.4 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.48–7.35 (m, 10H), 6.05 (s, 1H), 4.99 (s, 1H), 3.91 (d, <i>J</i> = 15 Hz, 1H), 3.63 (d, <i>J</i> = 15 Hz, 1H), 2.04 (m, 1H), 1.08 (m, 2H), 0.99 (m, 2H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 175.8 (C), 155.0 (C), 135.1 (C), 133.8 (C), 129.7 (2CH), 129.3 (2CH), 128.9 (2CH), 128.7 (2CH), 128.5 (CH), 128.4 (CH), 100.4 (CH), 70.4 (CH), 45.7 (CH<sub>2</sub>), 8.7 (CH<sub>2</sub>), 8.6 (CH<sub>2</sub>), 8.2 (CH).</div></div><div id="sec5_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 2-[(Benzhydrylsulfinyl)methyl]-6-(trifluoromethyl)pyrimidin-4-ol (<b>7f</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (27.6%, 0.052 mL) was added to the solution of <b>6f</b> (0.197 mg, 0.55 mmol) in glacial acetic acid (4 mL). The reaction mixture was stirred during 24 h at room temperature. The solvent was evaporated in vacuum until dryness. The product was crystallized from ethanol to give 0.136 g of <b>7f</b>. Additional portions of the product were obtained from the filtrate using column chromatography on silica gel (petroleum ether:EtOAc 3:1), to give 41 mg, yielding 86% of <b>7f</b> as white powder; mp 173–174 °C; retention time 3.16 min; purity 99.4%. HRESIMS <i>m</i>/<i>z</i> 415.0696 [M + Na]<sup>+</sup> (calcd for C<sub>19</sub>H<sub>15</sub>F<sub>3</sub>N<sub>2</sub>NaO<sub>2</sub>S<sup>+</sup>, 415.0699, Δ = 0.6 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 11.95 (br s, OH), 7.47–7.37 (m, 10H), 6.76 (s, 1H), 5.37 (s, 1H), 3.94 (d, <i>J</i> = 15 Hz, 1H), 3.62 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 161.6 (C), 155.2 (C), 152.5 (q, <sup>2</sup><i>J</i>(<sup>13</sup>C,<sup>19</sup>F) = 35.4 Hz, C), 133.8 (C), 133.5 (C), 129 5 (2 <i>o</i>-CH + 2-<i>m</i> CH), 129.0 (2 <i>o</i>-CH + 2 <i>m</i>-CH + <i>p</i>-CH), 128.8 (CH), 120.4 (q, <sup>2</sup><i>J</i>(<sup>13</sup>C,<sup>19</sup>F) = 35.4 Hz, CF<sub>3</sub>), 113.2 (CH), 71.1 (CH), 51.2 (CH<sub>2</sub>).</div></div><div id="sec5_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 2-[(Benzhydrylsulfinyl)methyl]-6-methylpyrimidin-4-ol (<b>7g</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (27.6%, 0.052 mL) was added to the solution of <b>6g</b> (0.178 mg, 0.55 mmol) in glacial acetic acid (4 mL). The reaction mixture was stirred during 24 h at room temperature. The solvent was evaporated in vacuum until dryness. The product was crystallized from EtOH to yield 114 mg of <b>7g</b>. Additional portions of the product were obtained from the filtrate using column chromatography on silica gel (petroleum ether:EtOAc, 3:1) to give 40 mg, yielding 83% of the isolated <b>7g</b> as white powder; mp 150–152 °C; retention time 14.99 min; purity 98.2%. HRESIMS <i>m</i>/<i>z</i> 361.0981 [M + Na]<sup>+</sup> (calcd for C<sub>19</sub>H<sub>18</sub>N<sub>2</sub>NaO<sub>2</sub>S<sup>+</sup>, 361.0981, Δ = 0.0 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 12.1 (br s, OH), 7.46–7.35 (m, 10H), 6.21 (s, 1H), 5.27 (s, 1H), 3.82 (d, <i>J</i> = 15 Hz, 1H), 3.65 (d, <i>J</i> = 15 Hz, 1H), 2.30 (s, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 165.3 (C), 163.3 (C), 152.5 (C), 134.5 (C), 133.7 (C), 129.6 (2CH), 129.4 (2CH), 128.9 (2 <i>o</i>-CH + 2 <i>m</i>-CH), 128.8 (CH), 128.7 CH), 112.3 (CH), 71.9 (CH), 53.3 (CH<sub>2</sub>), 24.0 (CH<sub>3</sub>).</div></div><div id="sec5_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 2-[(Benzhydrylsulfinyl)methyl]-4-methoxy-6-methylpyrimidine (<b>7h</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (27.6%, 0.074 mL) was added to the solution of <b>6h</b> (0.250 mg,0.74 mmol) in glacial acetic acid (4 mL). The reaction mixture was stirred at room temperature during 24 h. The solvent was evaporated in vacuum until dryness. The residue was diluted with water and extracted with CH<sub>2</sub>Cl<sub>2</sub> then organic extracts were evaporated and the residual product was purified by column chromatography (petroleum ether:EtOAc 3:1, EtOAc), affording 0.197 g of the isolated <b>7h</b> as white powder (yield 75%); mp 128 °C; retention time 19.5 min; purity 99.9%. HRESIMS <i>m</i>/<i>z</i> 375.1136 [M + Na]<sup>+</sup> (calcd for C<sub>20</sub>H<sub>20</sub>N<sub>2</sub>NaO<sub>2</sub>S<sup>+</sup>, 375.1138, Δ = 0.5 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.54–7.30 (m, 10H), 6.45 (s, 1H), 5.38 (s, 1H), 4.06 (d, <i>J</i> = 15 Hz, 1H), 3.95 (d, <i>J</i> = 15 Hz, 1H), 3.97 (s, 3H), 2.42 (s, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 170.0 (C), 167.6 (C), 161.2 (C), 136.3 (C), 133.9 (C), 130.0 (2CH), 129.0 (2CH), 128.8 (2CH), 128.6 (2CH), 128.3 (CH), 128.0 (9CH), 105.3 (CH), 70.4 (CH), 59.1 (CH<sub>2</sub>), 53.9 (s, 3H), 23.7 (CH<sub>3</sub>).</div></div><div id="sec5_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 4-[(Benzhydrylsulfinyl)methyl]-2,6-dimethylpyrimidine (<b>7i</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (27.6%, 0.080 mL) was added to the solution of <b>6i</b> (0.261 mg,0.81 mmol) in glacial acetic acid (4 mL). The reaction mixture was stirred during 24 h at room temperature. The solvent was evaporated in vacuum until dryness. The residue was diluted with water and extracted with CH<sub>2</sub>Cl<sub>2</sub> then organic extracts were evaporated and the residual product was purified by column chromatography on silica gel (petroleum ether:EtOAc 3:1), to yield 0.20 g of yellow oil <b>7i</b> (yield 73%); retention time 16.77 min; purity 98.8%. HRESIMS <i>m</i>/<i>z</i> 359.1185 [M + Na]<sup>+</sup> (calcd for C<sub>20</sub>H<sub>20</sub>N<sub>2</sub>NaOS<sup>+</sup>, 359.1189, Δ = 1.0 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.51–7.35 (m, 10H), 6.96 (s, 1H), 5.16 (s, 1H), 3.88 (d, <i>J</i> = 15 Hz, 1H), 3.68 (d, <i>J</i> = 15 Hz, 1H), 2.71 (s, 3H), 2.46 (s, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 167.8 (C), 167.5 (C), 159.3 (C), 135.5 (C), 134.0 (C), 129.6 (2CH), 129.2 (2CH), 128.9 (2CH), 128.7 (2CH), 128.4 (2CH), 119.2 (CH), 71.3 CH), 57.3 (CH<sub>2</sub>), 26.0 (CH<sub>3</sub>), 24.1 (CH<sub>3</sub>).</div></div><div id="sec5_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 2-[(Benzhydrylsulfinyl)methyl]-5-methylpyrimidine (<b>7j</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (27.6%, 0.080 mL) was added to the solution of <b>6j</b> (0.255 mg, 0.83 mmol) in glacial acetic acid (4 mL). The reaction mixture was stirred at room temperature during 24 h. The solvent was evaporated in vacuum until dryness. The residue was diluted with water, extracted with CH<sub>2</sub>Cl<sub>2</sub>, evaporated, and purified by column chromatography on silica gel (petroleum ether:EtOAc 3:1, EtOAc), to give 0.153 g of the isolated <b>7j</b> as white crystals (yield 57%); mp 138–140 °C; retention time 17.38 min; purity 99.9%. HRESIMS <i>m</i>/<i>z</i> 345.1030 [M + Na]<sup>+</sup> (calcd for C<sub>19</sub>H<sub>18</sub>N<sub>2</sub>NaOS<sup>+</sup>, 345.1032, Δ = 0.5 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 8.5 (s, 2H), 7.52–7.32 (m, 10H), 5.30 (s, 1H), 4.14 (d, <i>J</i> = 15 Hz, 1H), 4.05 (d, <i>J</i> = 15 Hz, 1H), 2.31(s, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 159.6 (C), 157.6 (CH), 136.0 (C), 134.1 (C), 129.9 (2CH), 129.2 (C), 129.1 (2CH), 128.9 (2CH), 128.6 (2CH), 128.3 (CH), 128.1 (CH), 70.8 (CH), 58.7 (CH<sub>2</sub>), 15.4 (CH<sub>3</sub>).</div></div><div id="sec5_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 3-((Benzhydrylsulfinyl)methyl)thiophene (<b>7k</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 1.96 mL) was added to the solution of <b>6k</b> (17 mmol, 5.26 g) in glacial acetic acid (35 mL). The reaction mixture was stirred during 24 h at room temperature. Then the reaction mixture was neutralized with cold 5% sodium bicarbonate solution. The aqueous neutral solution was extracted (3 × 50 mL) with ethyl acetate, and the organic extracts were collected, combined, and dried over anhydrous Na<sub>2</sub>SO<sub>4.</sub> The organic solution was concentrated under reduced pressure and residual product was purified by flash column chromatography on silica gel (5% MeOH in DCM) to yield 1.49 g of the isolated <b>7k</b> as pale-yellow powder (yield 28%); mp 92–96 °C; retention time 3.52 min; purity 97.4%. HRESIMS <i>m</i>/<i>z</i> 335.0535 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>16</sub>NaOS<sub>2</sub><sup>+</sup>, 335.0535, Δ = −0.0 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.42–7.33 (m, 11H), 7.18 (m, 1H), 7.02 (m, 1H), 4.66 (s, 1H), 3.94 (d, <i>J</i> = 15 Hz, 1H), 3.73 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 135.8 (C), 134.5 (C), 129.7 (C), 129.5 (2CH), 129.2 (2CH), 128.8 (CH), 128.7 (2CH), 128.6 (2CH), 128.3 (2CH), 126.2 (CH), 125.2 (CH), 69.7 (CH), 50.4 (CH<sub>2</sub>)</div></div><div id="sec5_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 3-((Benzhydrylsulfinyl)methyl)-2-bromothiophene (<b>7l</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 0.04 mL) was added to the solution of <b>6l</b> (0.15 g, 0.4 mmol) in glacial acetic acid (10 mL). The reaction mixture was stirred during 24 h at room temperature. Then the reaction mixture was neutralized with cold 5% sodium bicarbonate solution. The aqueous neutral solution was extracted (3 × 50 mL) with ethyl acetate, and the organic extracts were collected, combined, and dried over anhydrous Na<sub>2</sub>SO<sub>4.</sub> The organic solution was concentrated under reduced pressure, and residual was purified by flash column chromatography on silica gel (hexane:EtOAc, 1:1). Product was dried in high vacuum to give 0.05 g of isolated <b>7l</b> was obtained as brownish powder (yield 32%); mp 99–102 °C; retention time 23.01 min; purity 95.0%. HRESIMS <i>m</i>/<i>z</i> 412.9644 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>15</sub>BrNaOS<sub>2</sub><sup>+</sup>, 412.9640, Δ = −0.9 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.44–7.26 (m, 10H), 7.00 (d, <i>J</i> = 10 Hz), 7.02 (m, 1H), 4.87 (s, 1H), 3.90 (d, <i>J</i> = 15 Hz, 1H), 3.72 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 135.5 (C), 134.5 (C), 130.9 (C), 129.4 (2CH), 129.3 (2CH), 128.9 CH), 128.7 (2CH), 128.4 (2CH), 126.3 (CH), 113.3 (CH), 71.8 (CH), 51.0 (CH<sub>2</sub>).</div></div><div id="sec5_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 2-((Benzhydrylsulfinyl)methyl)-3-chlorothiophene (<b>7m</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 0.11 mL) was added to the solution of <b>6m</b> (0.33 mg, 0.9 mmol) in glacial acetic acid (15 mL). The reaction mixture was stirred at room temperature during 24 h. Then the reaction mixture was neutralized with cold 5% sodium bicarbonate solution. The aqueous neutral solution was extracted (3 × 50 mL) with ethyl acetate, and the organic extracts were collected, combined, and dried over anhydrous Na<sub>2</sub>SO<sub>4.</sub> The organic solution was concentrated under reduced pressure and residual product was purified by flash column chromatography on silica gel (hexane:EtOAc, 1:1) and dried in a high vacuum to give 0.115 g of <b>7m</b> as gray powder (yield 37%); mp 108–110 °C; retention time 27.6 min; purity 98.6%. HRESIMS <i>m</i>/<i>z</i> 369.0144 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>15</sub>ClNaOS<sub>2</sub><sup>+</sup>, 369.0145, Δ = 0.3 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.48–7.36 (m, 10H), 7.30 (d, <i>J</i> = 5 Hz, 1H), 6.95 (d, <i>J</i> = 5 Hz, 1H), 4.90 (s, 1H), 4.12 (d, <i>J</i> = 15 Hz, 1H), 3.84 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 135.2 (C), 134.2 (C), 129.5 (2CH), 129.3 (2CH), 128.7 (2 <i>o</i>-CH + 2 <i>m</i>-CH), 128.4 (2CH), 127.5 (CH), 126.1 (CH), 125.9 (C), 124.6 (C), 71.0 (CH), 49.1 (CH<sub>2</sub>).</div></div><div id="sec5_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 4-((Benzhydrylsulfinyl)methyl)-2,3-dimethylthiophene (<b>7n</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 0.17 mL) was added to the solution of <b>6n</b> (0.531 g 1.6 mmol) in glacial acetic acid (20 mL). The reaction mixture was stirred at room temperature during 24 h. Then, the reaction mixture was neutralized with cold 5% sodium bicarbonate solution. The aqueous neutral solution was extracted (3 × 50 mL) with ethyl acetate, and the organic extracts were collected, combined, and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The organic solution was concentrated under reduced pressure, and residual was purified by flash column chromatography on silica gel (5% MeOH in DCM). Product was dried in high vacuum to give 0.14 g of <b>7n</b> as brownish crystals (yield 27%); mp 114–118 °C; retention. time 23.82 min; purity 96.7%. HRESIMS 363.0853 [M + Na]<sup>+</sup> (calcd for C<sub>20</sub>H<sub>20</sub>NaOS<sub>2</sub><sup>+</sup>, 363.0848, Δ = −1.4 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.50–7.34 (m, 10H), 6.92 (s, 1H), 4.83 (s, 1H), 3.82 (d, <i>J</i> = 15 Hz, 1H), 3.66 (d, <i>J</i> = 15 Hz, 1H), 2.32 (s, 3H), 1.93 (s, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 134.6 (C), 134.2 (C), 132.0 (C), 132.3 (CH), 130.4 (C), 129.6 (2CH), 129.2 (2CH), 128.8 (2CH), 128.7 (2CH), 128.3 (2CH), 121.4 (CH), 70.5 (CH), 51.5 (CH<sub>2</sub>), 13.6 (CH<sub>3</sub>), 12.2 (CH<sub>3</sub>).</div></div><div id="sec5_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 4-((Benzhydrylsulfinyl)methyl)-2-bromothiophene (<b>7o</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 0.05 mL) was added to the solution of <b>6o</b> (0.20 g, 0.5 mmol) in glacial acetic acid (15 mL). The reaction mixture was stirred during 24 h at room temperature. Then the reaction mixture was neutralized with cold 5% sodium bicarbonate solution. The aqueous neutral solution was extracted (3 × 50 mL) with ethyl acetate, and the organic extracts were collected, combined, and dried over anhydrous Na<sub>2</sub>SO<sub>4.</sub> The organic solution was concentrated under reduced pressure, and residual was purified by flash column chromatography on silica gel (2.5% MeOH in DCM). Product was dried in high vacuum to give 0.90 g of <b>7o</b> was obtained as brown powder (yield 43%); mp 91–94 °C; retention time 4.21 min; purity 98.9%. HRESIMS 412.9646 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>15</sub>BrNaOS<sub>2</sub><sup>+</sup>, 412.9640, Δ = −1.6 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.44–7.36 (m, 10H), 7.06 (s, 1H), 6.95 (s, 1H), 4.70 (s, 1H), 3.85 (d, <i>J</i> = 15 Hz, 1H), 3.62 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 135.5 (C), 134.5 (C), 131.3 (CH), 130.6 (C), 129.4 (2CH), 129.3 (2CH), 128.8 (2CH), 128.6 (2CH), 128.5 (CH), 128.4 (CH), 126.7 (CH), 112.9 (C), 70.4 (CH), 50.4 (CH<sub>2</sub>).</div></div><div id="sec5_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 2-((Benzhydrylsulfinyl)methyl)-5-chlorothiophene (<b>7p</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 0.18 mL) was added to the solution of <b>6p</b> (0.60 g, 1.8 mmol) in glacial acetic acid (20 mL). The reaction mixture was stirred at room temperature during 24 h. Then, the reaction mixture was neutralized with cold 5% sodium bicarbonate solution. The aqueous neutral solution was extracted (3 × 50 mL) with ethyl acetate, and the organic extracts were collected, combined, and dried over anhydrous Na<sub>2</sub>SO<sub>4.</sub> The organic solution was concentrated under reduced pressure, and residual was purified by flash column chromatography on silica gel (1% MeOH in DCM). Product was dried in high vacuum to give 0.74 g of <b>7p</b> as dark-yellow powder (yield 43%); mp 89–92 °C; retention time 23.33 min; purity 99.5%. HRESIMS 369.0146 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>15</sub>ClNaOS<sub>2</sub><sup>+</sup>, 369.0145, Δ = −0.2 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.44–7.36 (m, 10H), 6.85 (m, 1H), 6.70 (m, 1H), 4.77 (s, 1H), 3.99 (d, <i>J</i> = 15 Hz, 1H), 3.75 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 135.1 (C), 134.1 (C), 131.0 (C), 129.5 (2CH), 129.4 (2CH), 128.8 (4CH), 128.6 (CH), 128.5 (2CH), 126.1 (CH), 69.7 (CH), 50.1 (CH<sub>2</sub>).</div></div><div id="sec5_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 2-((Benzhydrylsulfinyl)methyl)-4-bromothiophene (<b>7q</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 0.2 mL) was added to the solution of <b>6q</b> (0.71 g, 1.9 mmol) in glacial acetic acid (20 mL). The reaction mixture was stirred at room temperature during 24 h. Then the reaction mixture was neutralized with cold 5% sodium bicarbonate solution. The aqueous neutral solution was extracted (3 × 50 mL) with ethyl acetate, and the organic extracts were collected, combined, and dried over anhydrous Na<sub>2</sub>SO<sub>4.</sub> The organic solution was concentrated under reduced pressure, and residual was purified by flash column chromatography on silica gel (1% MeOH in DCM). Product was dried in high vacuum to give 0.22 g of <b>7q</b> as yellow powder (yield 30%); mp 89–91 °C; retention time 23.44 min; purity 98.6%. HRESIMS 412.9638 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>15</sub>BrNaOS<sub>2</sub><sup>+</sup>, 412.9640, Δ = 0.4 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.44–7.36 (m, 10H), 7.22 (s, 1H), 6.84 (s, 1H), 4.76 (s, 1H), 4.03 (d, <i>J</i> = 15 Hz, 1H), 3.80 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 135.1 (C), 134.1 (C), 132.0 (C), 131.3 (CH), 129.5 (2CH), 129.4 (2CH), 128.8 (2CH), 128.7 (2CH), 128.6 (CH), 128.5 (CH), 124.1 (CH), 109.9 (C), 70.1 (CH), 50.1 (CH<sub>2</sub>).</div></div><div id="sec5_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 2-(((Bis(4-chlorophenyl)methyl)thio)methyl)thiophene (<b>10a</b>)</h4><div class="NLM_p last">Bis(4-chlorophenyl) methanol (0.96 g, 3.8 mmol) and (0.5 g, 3.8 mmol) of thiophen-2-ylmethanethiol were dissolved in glacial acetic acid. Then 1.1 equiv (1.1 mL, 4.2 mmol) of BF<sub>3</sub>·Et<sub>2</sub>O was added to a solution, and reaction mixture was left under stirring overnight at room temperature. Reaction mixture was then poured over ice, a small amount of water was added, and acetic acid was neutralized with addition of solid NaHCO<sub>3</sub>. Product was then extracted (3×) with 50 mL of ethyl acetate. Organic layers were collected, combined, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. Crude product was then purified by flash column chromatography on silica gel (2.5% MeOH in DCM) affording 1.23 g of the oily product (yield 89%).</div></div><div id="sec5_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 2-((Benzhydrylthio)methyl)thiophene (<b>10b</b>)</h4><div class="NLM_p last">Diphenylmethanol (1.4 g, 7.67 mmol) and thiophen-2-yl-methanethiol (1 g, 7.67 mmol) were dissolved in glacial acetic acid and 1.1 equiv (2.2 mL, 8.45 mmol) of BF<sub>3</sub>·Et<sub>2</sub>O was added to the solution and the reaction mixture was left under stirring overnight at room temperature. The reaction mixture was then poured over ice and a small amount of water was added to dilute the excess of acetic acid, then the resulting solution was neutralized with addition of solid NaHCO<sub>3</sub>. Product was extracted (3 × 50 mL) with ethyl acetate and the organic extracts were collected, combined, and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The organic solution was concentrated, and the crude residual material was purified via flash column chromatography on silica gel (2.5% MeOH in DCM), to give 2.07 g of the oily product (yield 91%).</div></div><div id="sec5_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 2-((((4-Bromophenyl)(phenyl)methyl)thio)methyl)thiophene (<b>10c</b>)</h4><div class="NLM_p last">4-Bromobenzhydrol (2.02 g, 7.67 mmol) and thiophen-2-yl-methanethiol (1.0 g, 7.67 mmol) were dissolved in glacial acetic acid. Then 1.1 equiv (2.2 mL, 8.45 mmol) of BF<sub>3</sub>·Et<sub>2</sub>O was added to the solution and the reaction mixture was left under stirring overnight at room temperature. Reaction mixture was then poured over ice, a small amount (1–2 mL) of water was added, and acetic acid excess was neutralized with addition of solid NaHCO<sub>3</sub>. Product was then extracted (3 × 15 mL) with ethyl acetate. Organic layers were collected, combined, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. Crude product was purified via flash column chromatography on silica gel (2.5% MeOH in DCM) affording 2.66 g of the oily product (yield 92%).</div></div><div id="sec5_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 2-(((Phenyl(<i>p</i>-tolyl)methyl)thio)methyl)thiophene (<b>10d</b>)</h4><div class="NLM_p last">4-Methylbenzhydrol (1.52 g, 7.67 mmol) and thiophen-2-yl-methanethiol 1 g (7.67 mmol) of were dissolved in glacial acetic acid. Then 1.1 equiv (2.2 mL, 8.45 mmol) of BF<sub>3</sub>·Et<sub>2</sub>O was added to a solution and the reaction mixture was left under stirring overnight at room temperature. Reaction mixture was then poured over ice, a small amount (1–2 mL) of water was added, and the acetic acid excess was neutralized with addition of solid NaHCO<sub>3</sub>. Product was then extracted (3 × 15 mL) with ethyl acetate. Organic layers were collected, combined, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered off, and concentrated under reduced pressure. Crude product was purified by flash column chromatography on silica gel (2.5% MeOH in DCM), affording 1.9 g of the oily product (yield 79%).</div></div><div id="sec5_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 2-(((Bis(4-fluorophenyl)methyl)thio)methyl)thiophene (<b>10e</b>)</h4><div class="NLM_p last">Bis(4-fluorophenyl) methanol (0.84 g, 3.8 mmol) and (0.5 g, 3.8 mmol) of thiophen-2-ylmethanethiol were dissolved in glacial acetic acid. Then 1.1 equiv (1.1 mL, 4.2 mmol) of BF<sub>3</sub>·Et<sub>2</sub>O was added to a solution and the reaction mixture was left under stirring overnight at room temperature. Reaction mixture was then poured over ice, a small amount (1–2 mL) of water was added, and acetic acid excess was neutralized with addition of solid NaHCO<sub>3</sub>. Product was then extracted (3 × 15 mL) of ethyl acetate. Organic layers were collected, combined, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered off, and concentrated under reduced pressure. Crude product was purified by flash column chromatography on silica gel (2.5% MeOH in DCM) affording 0.82 g of the oily product (yield 65%).</div></div><div id="sec5_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 2-(((Di-<i>p</i>-tolylmethyl)thio)methyl)thiophene (<b>10f</b>)</h4><div class="NLM_p last">Di-<i>p</i>-tolylmethanol (0.6 g, 2.8 mmol) and thiophen-2-yl-methanethiol (0.37 g, 2.8 mmol) were dissolved in glacial acetic acid. Then 1.1 equiv (0.8 mL, 3.1 mmol) of BF<sub>3</sub>·Et<sub>2</sub>O was added to solution and the reaction mixture was left under stirring overnight at room temperature. Reaction mixture was then poured over ice, and a small amount of water (1–2 mL) was added, and acetic acid excess was neutralized with addition of solid NaHCO<sub>3</sub>. Product was then extracted (3 × 50 mL) with ethyl acetate. Organic extracts were collected, combined, and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, Organic solution was concentrated under reduced pressure, and the crude product was purified via flash column chromatography on silica gel (2.5% MeOH in DCM) affording 0.72 g of the oily product (yield 86%).</div></div><div id="sec5_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 2-(((Bis(4-chlorophenyl)methyl)sulfinyl)methyl)thiophene (<b>11a</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 1.1 mL) was added to the solution of <b>10a</b> (1.23 g, 3.4 mmol) in glacial acetic acid (50 mL). The reaction mixture was stirred during 24 h at room temperature. Acid was neutralized with 5% sodium bicarbonate and ice, and the product was extracted with EtOAc, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. Then, product was purified by flash column chromatography on silica gel (hexane:EtOAC, 1:1) and dried in high vacuum to give 0.74 g of <b>11a</b> as yellow crystals (yield 57%); mp 112–114 °C; retention time 24.24 min; purity 98.7%. HRESIMS 402.9756 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>14</sub>Cl<sub>2</sub>NaOS<sub>2</sub><sup>+</sup>, 402.9755, Δ = −0.1 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.40–7.35 (m, 9H), 7.06 (m, 1H), 6.96 (m, 1H), 4.67 (s, 1h), 4.09 (d, <i>J</i> = 15 Hz, 1H), 3.90 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 134.8 (C), 134.6 (C), 133.6 (C), 131.9 (C), 130.9 (2CH), 130.1 (2CH), 129.7 (C), 129.6 (2CH), 129.1 (CH), 128.9 (2CH), 127.4 (CH), 127.0 (CH), 66.9 (CH), 50.3 (CH<sub>2</sub>).</div></div><div id="sec5_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 2-((Benzhydrylsulfinyl)methyl)thiophene (<b>11b</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 0.8 mL) was added to the solution of <b>10b</b> (2.07 g, 7.0 mmol) in glacial acetic acid (50 mL). The reaction mixture was stirred overnight at room temperature. Acid was neutralized with 5% sodium bicarbonate and ice, and the product was extracted with EtOAc, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. Then, product was purified by flash column chromatography on silica gel (2.5% MeOH in DCM) and dried in high vacuum to give 1.49 g of <b>11b</b> as yellow powder (yield 68%); mp 116–119 °C; retention time 21.45 min; purity 99.5%. HRESIMS 335.0537 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>16</sub>NaOS<sub>2</sub><sup>+</sup>, 335.0535, Δ = −0.6 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.42–7.34 (m, 11H), 7.05 (m, 1H), 6.96 (m, 1H), 4.75 (s, 1H), 4.10 (d, <i>J</i> = 15 Hz, 1H), 3.88 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 135.5 (C), 134.1 (C), 130.2 (C), 129.6 (2CH), 129.3 (2CH), 129.0 (CH), 128.7 (2 <i>o</i>-CH + 2 <i>m</i>-CH), 128.4 (2CH), 127.3 (CH), 126.8 (CH), 69.2 (CH), 50.1 (CH<sub>2</sub>).</div></div><div id="sec5_1_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 2-((((4-Bromophenyl)(phenyl)methyl)sulfinyl)methyl)thiophene (<b>11c</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 0.82 mL) was added to the solution of <b>10c</b> (7.1 mmol, 2.66 g) in glacial acetic acid (50 mL). The reaction mixture was stirred at room temperature overnight. Acid was neutralized with 5% sodium bicarbonate and ice. Then product was extracted with EtOAc, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. Product was purified via flash column chromatography on silica gel (5% MeOH in DCM) and dried in high vacuum to give 2.36 g of <b>11c</b> as brown solid (yield 85%); mp 95–98 °C; retention time 23.71 min; purity 98.8%. HRESIMS 412.9644 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>15</sub>BrNaOS<sub>2</sub><sup>+</sup>, 412.9640, Δ = −1.1 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.54–7.31 (m, 10H), 7.05 (m, 1H), 6.96 (m, 1H), 4.69 (s, 1H), 4.09 (d, <i>J</i> = 15 Hz, 1H), 3.89 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 134.8 (C), 133.4 (C), 132.3 (2CH), 130.4 (2CH), 130.1 (C), 129.6 (2CH), 129.0 (CH), 128.8 (2CH), 128.6 (CH), 127.4 (CH), 126.9 (CH), 122.5 (C), 68.0 (CH), 50.3 (CH<sub>2</sub>).</div></div><div id="sec5_1_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 2-(((Phenyl(<i>p</i>-tolyl)methyl)sulfinyl)methyl)thiophene (<b>11d</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 0.7 mL) was added to the solution of <b>10d</b> (1.9 g, 6.1 mmol) in glacial acetic acid (40 mL). The reaction mixture was stirred overnight at room temperature. Acid was neutralized with 5% sodium bicarbonate and ice. Product was extracted with EtOAc, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. Then, product was purified via flash column chromatography on silica gel (2.5% MeOH in DCM) and dried in high vacuum to give 0.64 g of <b>11d</b> as orange powder (yield 33%); mp 82–86 °C; retention time 22.55 min; purity 96.2%. HRESIMS 349.0690 [M + Na]<sup>+</sup> (calcd for C<sub>19</sub>H<sub>18</sub>NaOS<sub>2</sub><sup>+</sup>, 349.0691, Δ = 0.3 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.32–6.96 (m, 12H), 4.70 (1H), 4.08 (d, <i>J</i> = 15 Hz, 1H), 3.87 (d, <i>J</i> = 15 Hz, 1H), 2.36 (s, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 138.2 (C), 138.1 (C), 132.6 (C), 131.3 (C), 130.5 (C), 129.9 (2CH), 129.4 (2CH), 129.3 (2CH), 128.9 (CH), 128.6 (CH), 127.2 (CH), 126.6 (CH), 68.8 (CH), 50.0 (CH<sub>2</sub>) 21.1 (CH<sub>3</sub>).</div></div><div id="sec5_1_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 2-(((Bis(4-fluorophenyl)methyl)sulfinyl)methyl)thiophene (<b>11e</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 0.25 mL) was added to the solution of <b>10e</b> (0.82 g, 2.5 mmol) in glacial acetic acid (20 mL). The reaction mixture was stirred overnight at room temperature. Acid was neutralized with 5% sodium bicarbonate and ice. Product was extracted with EtOAc, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. Then product was purified by flash column chromatography on silica gel (hexane:EtOAC, 1:1) and dried in high vacuum to give 0.40 g of <b>11e</b> as yellow powder (yield 47.0%); mp 103–106 °C; retention time 21.22 min; purity 98.5%. HRESIMS 371.0349 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>14</sub>F<sub>2</sub>NaOS<sub>2</sub><sup>+</sup>, 371.0346, Δ = −0.7 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.38–9.96 (m, 11H), 4.70 (s, 1H), 4.08 (d, <i>J</i> = 15 Hz, 1H), 3.88 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 162, 9 (q, <sup>1</sup><i>J</i>(<sup>13</sup>C,<sup>19</sup>F) = 246,9 Hz, C), 162.6 (q, <sup>1</sup><i>J</i>(<sup>13</sup>C,<sup>19</sup>F) = 247,6 Hz, C), 131.3 (q, <sup>3</sup><i>J</i>(<sup>13</sup>C,<sup>19</sup>F) = 8.2 Hz, 2CH), 131.1 (q, <sup>4</sup><i>J</i>(<sup>13</sup>C,<sup>19</sup>F) = 3.1 Hz, C), 130.5 (q, <sup>3</sup><i>J</i>(<sup>13</sup>C,<sup>19</sup>F) = 8.1 Hz, 2CH), 129.8 (C), 129.5 (q, <sup>4</sup><i>J</i>(<sup>13</sup>C,<sup>19</sup>F) = 2.9 Hz, C), 127.4 (CH), 127.0 (CH), 116.4 (q, <sup>2</sup><i>J</i>(<sup>13</sup>C,<sup>19</sup>F) = 21.6 Hz, CH), 115.7(q, <sup>2</sup><i>J</i>(<sup>13</sup>C,<sup>19</sup>F) = 21.7 Hz, CH), 66.8 (CH), 50.0 (CH<sub>2</sub>).</div></div><div id="sec5_1_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 2-(((Di-<i>p</i>-tolylmethyl)sulfinyl)methyl)thiophene (<b>11f</b>)</h4><div class="NLM_p last">H<sub>2</sub>O<sub>2</sub> (30%, 0.32 mL) was added to the solution of <b>10f</b> (0. 92 mg, 3.1 mmol) in glacial acetic acid (20 mL). The reaction mixture was stirred at room temperature overnight and neutralized with 5% sodium bicarbonate solution and ice. The reaction mixture was extracted with EtOAc, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. Then, product was purified by flash column chromatography on silica gel (1% MeOH in DCM) and dried in high vacuum to give 0.20 g of <b>11f</b> (yield 19%); mp 113–115 °C; retention time 24.40 min; purity 99.9%. HRESIMS 363.0848 [M + Na]<sup>+</sup> (calcd for C<sub>20</sub>H<sub>20</sub>NaOS<sub>2</sub><sup>+</sup>, 363.0848, Δ = −0.0 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.32–7.17 (m, 9H), 7.04 (m, 1H), 6.96 (m, 1H), 4.70 (s, 1H), 4.08 (d, <i>J</i> = 15 Hz, 1H), 3.87 (d, <i>J</i> = 15 Hz, 1H), 2.36 (s, 3H), 2.32 (s, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 138.1 (2C), 132.7 (C), 131.3 (C), 130.5 (C), 129.9 (2CH), 129.4 (2CH), 129.3 (2CH), 128.9 (CH), 128.6 (CH), 127.2 (CH), 126.6 (CH), 68.8 (CH), 50.0 (CH2), 21.1 (2CH<sub>3</sub>).</div></div><div id="sec5_1_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 2-((((4-Bromophenyl)(phenyl)methyl)sulfinyl)methyl)thiophene (<b>11g</b>)</h4><div class="NLM_p last">From <b>11c</b> by separation on silica gel column (mobile phase EtOH/toluene = 60/40), two mixtures of stereoisomer compounds were obtained: <b>11g</b> and <b>11h</b>. HRESIMS 412.9639 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>15</sub>BrNaOS<sub>2</sub><sup>+</sup>, 412.9640, Δ = 0.3 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.54–7.36 (m, 7H), 7.34 (m, 1H), 7.31 (m, 2H), 7.05 (m, 1H), 6,96 (m, 1H), 4.69 (s, 1H), 4.09 (d, <i>J</i> = 15 Hz, 1H), 3.89 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 134.8 (C), 133.4 (C), 132.4 (2CH), 130.4 (2CH), 130.1 (C), 129.6 (2CH), 129.0 (CH), 128.8 (2CH), 128.6 (CH), 127.4 (CH), 126.9 (CH), 122.6 (C), 68.0 (CH), 50.3 (CH<sub>2</sub>).</div></div><div id="sec5_1_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 2-((((4-Bromophenyl)(phenyl)methyl)sulfinyl)methyl)thiophene (<b>11h</b>)</h4><div class="NLM_p last">Same as <b>11g</b>. HRESIMS 412.9643 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>15</sub>BrNaOS<sub>2</sub><sup>+</sup>, 412.9640, Δ = −0.9 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.54–7.31 (m, 10H), 7.05 (m, 1H), 6.96 (m, 1H), 4.69 (s, 1H), 4.09 (d, <i>J</i> = 15 Hz, 1H), 3.89 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 134.8 (C), 133.4 (C), 132.3 (2CH), 130.4 (2CH), 130.1 (C), 129.6 (2CH), 129.0 (CH), 128.8 (2CH), 128.6 (CH), 127.4 (CH), 126.9 (CH), 122.5 (C), 68.0 (CH), 50.3 (CH<sub>2</sub>).</div></div><div id="sec5_1_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 2-((((4-Bromophenyl)(phenyl)methyl)sulfinyl)methyl)thiophene (<b>11i</b>)</h4><div class="NLM_p last">From <b>11h</b>, by separation on Chiralpack IA column using 60% hexane/40% ethyl acetate as the mobile phase, two individual enantiomers were obtained: <b>11i</b> (less retained enantiomer) and <b>11j</b> (more retained enantiomer), respectively; retention time (chiral column) 17.4 and 29.7 min for <b>11i</b> and <b>11j</b>, respectively. Yield >95%. HRESIMS 412.9640 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>15</sub>BrNaOS<sub>2</sub><sup>+</sup>, 412.9640, Δ = 0.0 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.49–7.38 (m, 7H), 7.34 (m, 1H), 7.29 (m, 2H), 7.05 (m, 1H), 6,96 (m, 1H), 4.70 (s, 1H), 4.09 (d, <i>J</i> = 15 Hz, 1H), 3.89 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 135.0 (C), 133.2 (C), 131.8 (2CH), 131.3 (2CH), 129.9 (C), 129.4 (2CH), 129.1 (CH), 128.8 (2CH), 128.7 (CH), 127.3 (CH), 126.9 (CH), 122.8 (C), 68.4 (CH), 50.1 (CH<sub>2</sub>). Retention time 28.28 min; relative peak area 98.4%; CCDC code 1569748.</div></div><div id="sec5_1_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 2-((((4-Bromophenyl)(phenyl)methyl)sulfinyl)methyl)thiophene (<b>11j</b>)</h4><div class="NLM_p last">Same as <b>11i</b>; retention time (chiral column) 17.4 and 29.7 min for <b>11i</b> and <b>11j</b>, respectively. Yield >95%. HRESIMS 412.9640 [M + Na]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>15</sub>BrNaOS<sub>2</sub><sup>+</sup>, 412.9640, Δ = 0.0 ppm). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ = 7.49–7.38 (m, 7H), 7.34 (m, 1H), 7.29 (m,2H), 7.05 (m, 1H), 6,95 (m,1H), 4.70 (s, 1H), 4.09 (d, <i>J</i> = 15 Hz, 1H), 3.89 (d, <i>J</i> = 15 Hz, 1H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ = 135.0 (C), 133.2 (C), 131.8 (2CH), 131.3 (2CH), 129.9 (C), 129.4 (2CH), 129.1 (CH), 128.8 (2CH), 128.7 (CH), 127.3 (CH), 126.9 (CH), 122.8 (C), 68.4 (CH), 50.1 (CH<sub>2</sub>). Retention time 28.30 min; relative peak area 97.1%; CCDC code 1569749.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> X-ray Measurement</h3><div class="NLM_p last">The X-ray intensity data were measured on Bruker X8 APEX2 diffractometer equipped with a multilayer monochromator, a Mo Kα INCOATEC micro focus sealed tube, and a Kryoflex cooling device. The structures were solved by direct methods and refined by full-matrix least-squares techniques. Non-hydrogen atoms were refined with anisotropic displacement parameters. Hydrogen atoms were inserted at calculated positions and refined with a riding model. The following software was used: Bruker SAINT software package<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> using a narrow-frame algorithm for frame integration, SADABS<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> for absorption correction, OLEX2<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> for structure solution, refinement, molecular diagrams, and graphical user-interface, Shelxle<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> for refinement and graphical user-interface, SHELXS-2013<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> for structure solution, SHELXL-2013<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> for refinement, and Platon<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> for symmetry check. Experimental data and CCDC codes can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01313/suppl_file/jm7b01313_si_001.pdf" class="ext-link">Supporting Information</a>. Crystal data, data collection parameters, and structure refinement details as well as crystal structures visualized are available in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01313/suppl_file/jm7b01313_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> DAT, NET, SERT Reuptake Inhibition Assay</h3><div class="NLM_p">Assays were performed as previously described by Sucic et al.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Dulbecco’s Modified Eagle’s Medium (DMEM), trypsin, and fetal calf serum (FCS) were purchased from Sigma-Aldrich Handels GmbH (Austria). [<sup>3</sup>H]5-HT (hydroxytryptamine creatinine sulfate; 5-[1,2-<sup>3</sup>H[N]]; 27.8 Ci/mmol), [<sup>3</sup>H]DA (dihydroxyphenylethylamine; 3,4-[ring-2,5,6-<sup>3</sup>[H]]-dopamine; 36.6 Ci/mmol), and [<sup>3</sup>H]MPP<sup>+</sup> (methyl-4-phenylpyridinium iodide; 1-[methyl-3H]; 80 Ci/mmol) were purchased from PerkinElmer, Boston, MA.</div><div class="NLM_p">HEK293 cells stably expressing human isoforms of the dopamine transporter (DAT), norepinephrine transporter (NET), and serotonin transporter (SERT) were used for reuptake inhibition assays. All cell lines were seeded in 96-well plates precoated with poly-<span class="smallcaps smallerCapital">d</span>-lysine (PDL) (5 × 10<sup>4</sup> cells/well) 24 h prior to the experiment. Each well was washed with 0.1 mL of Krebs–HEPES buffer (KHB; 10 mM HEPES, 120 mM NaCl, 3 mM KCl, 2 mM CaCl<sub>2</sub>·2H<sub>2</sub>O, 2 mM MgCl<sub>2</sub>·6H<sub>2</sub>O, 5 mM <span class="smallcaps smallerCapital">d</span>-(+)-glucose monohydrate, pH 7.3). Cells were preincubated 5 min in KHB containing increasing concentrations (0.001 μM to 1 mM) of test compound. Compounds were dissolved first in 99.9% dimethyl sulfoxide (DMSO) and subsequently diluted in KHB. Cells were incubated in KHB containing increasing concentrations of compound with addition of 0.2 μM [<sup>3</sup>H]-dopamine (for HEK-DAT), 0.05 μM [<sup>3</sup>H]MPP<sup>+</sup> (for HEK-NET), and 0.4 μM [<sup>3</sup>H]5-HT (for HEK-SERT). Incubation times were 1 min for HEK-DAT and HEK-SERT and 3 min for HEK-NET. For determination of unspecific uptake in HEK-DAT and HEK-NET, 10 μM mazindol was used and 10 μM paroxetine was used for HEK-SERT. After incubation at room temperature, reactions were stopped by the addition of 0.1 mL of ice-cold KHB. Finally, cells were lysed with 0.3 mL of 1% SDS and released radioactivity was measured by a liquid scintillation counter (Tricarb-2300TR, PerkinElmer).</div><div class="NLM_p last">Nonlinear regression analysis was carried out for reuptake assays. All calculations were performed using GraphPad Prism version 6.00 for Windows, GraphPad Software,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> San Diego, CA, USA.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Dopamine Release Assay</h3><div class="NLM_p">Monensin sodium salt and <span class="smallcaps smallerCapital">d</span>-amphetamine hemisulfate salt were purchased from Sigma-Aldrich Co.</div><div class="NLM_p">For release studies, HEK-DAT cells were grown overnight on 5 mm diameter PDL-coated round glass coverslips placed in a 96-well plate. Cells were incubated with 0.1 μM [<sup>3</sup>H]MPP<sup>+</sup> at 37 °C for 20 min. The coverslips were transferred into superfusion chambers, and excess radioactivity was washed out with KHB for 40 min (0.7 mL/min) at 25 °C to obtain stable baselines as described before.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The experiment was started with the collection of fractions (2 min) as depicted in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>. During the experiments, the buffer was switched either to monensin or remained at control buffer after the collection of three baseline fractions for another four fractions. Subsequently, <b>7d</b> (or <b>11b</b>) or <span class="smallcaps smallerCapital">d</span>-amphetamine was added for another five fractions as indicated in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>. Finally, cells were lysed in 1% SDS and the released radioactivity was quantified by liquid scintillation counting.</div><div class="NLM_p last">Nonlinear regression analysis was carried out for release assays. All calculations were performed using GraphPad Prism version 6.00 for Windows, GraphPad Software,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> San Diego, CA, USA.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> [<sup>3</sup>H]WIN 35,428 Binding Assays</h3><div class="NLM_p">Zinc chloride 0.1 M solution was purchased from Sigma-Aldrich Co. WIN, 35,428, [<i>N</i>-methyl-<sup>3</sup>H] (84.0 Ci/mmol) was purchased from PerkinElmer, Boston, USA.</div><div class="NLM_p last">For binding studies, HEK-DAT cells were grown overnight in 48-well plates precoated with PDL (0.1 × 10<sup>6</sup> cells/well) 24 h prior to the experiment. Each well was washed out with 0.5 mL of binding buffer (10 mM HEPES, 120 mM NaCl, 3 mM KCl, 2 mM CaCl<sub>2</sub>·2H<sub>2</sub>O, 2 mM MgCl<sub>2</sub>·6H<sub>2</sub>O, 10 μM ZnCl<sub>2</sub>, 5 mM <span class="smallcaps smallerCapital">d</span>-(+)-glucose monohydrate, pH 7,3). Cells were incubated with the radioligand (∼10 nM [<sup>3</sup>H]WIN 35,428), the indicated concentrations of <b>7d</b> and <b>11b</b>, and increasing concentrations of cocaine (0.001 μM to 1 μM). The reaction volume was 0.1 mL. Nonspecific binding was determined in the presence of 30 μM mazindol. Binding was allowed to proceed for 30 min at 25 °C and terminated by washing the cells two times with 0.5 mL of binding buffer. Cells were lysed in 1% SDS, and the radioactivity was quantified by liquid scintillation counting.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> Homology Modeling</h3><div class="NLM_p last">The protein homology model was kindly provided by the group of Gerhard Ecker from the University of Vienna. The homology model was based on the crystallized Drosophila DAT, as described in Saha et al.,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> in the outward facing conformation deposited in the Protein Data Bank with the PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M48">4M48</a>. The DAT structure was in complex with nortriptylin.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> Molecular Docking</h3><div class="NLM_p last">Molecular docking was performed using Autodock Vina 1.1.2.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The ligands were prepared using the Maestro Modeling<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> suite, and double bonds and hydrogen atoms were assigned as shown in Figure B. Flexible bonds were added to the ligands using the flexible bond detection of AutoDockTools 1.5.6.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> For the molecular docking run, the standard parameters were used; protein flexibility was not regarded and the exhaustiveness level was set to 16. Waters were removed and the binding site was defined as the center of mass of nortriptyline. The search space was restricted to the protein structure of the homology model. Docking analysis was performed using compound <b>7d</b> and <b>11b</b>. Autodock Vina used the AMBER force field to evaluate the interaction and energy of the different poses.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> MD Simulations</h3><div class="NLM_p">The protein–ligand systems were oriented and optimized using the position of proteins in membrane (PMM 2.0) method provided by the Orientation of Proteins in Membranes (OPM) web service.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The obtained orientation of the protein–ligand system were subsequently used. The CHARMM-GUI web interface<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and the membrane builder<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> were used to set up the membrane–protein–ligand systems. The protein was placed in a bilayer containing 600 phosphatidylcholine (POPC). The membrane spanned an area of 150 × 150 Å<sup>2</sup>, leading to a rectangle with 150 × 150 × 105 Å<sup>3</sup>. The membrane–protein–ligand systems were solvated using TIP3P water molecules. MD simulations were carried out using CHARMM,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> utilizing the CHARMM/OpenMM coupling.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37, 38)</a> Parameters and molecular topologies for the ligands were generated based on the CGenFF force field<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> using the ParamChem server (<a href="http://cgenff.paramchem.org" class="extLink">cgenff.paramchem.org</a>). The membrane–protein–ligand complexes were solvated in rectangular boxes of TIP3P water.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Ions were added to compensate for any net charge of the solute and to set the ion concentration to 0.15 M KCl. Electrostatic interactions were computed using the particle-mesh-Ewald method.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> SHAKE was used to keep all bonds involving hydrogen atoms rigid. After initial equilibration using the equilibration steps provided by CHARMM-GUI for membrane–protein systems, the two complexes were simulated using Langevin dynamics at 303.15 K; the pressure was kept around 1 atm by a Monte Carlo barostat.</div><div class="NLM_p">The system containing <b>7d</b> was simulated for 80 ns and the system containing <b>11b</b> for 75 ns.</div><div class="NLM_p">The stability of the simulations was monitored by computing root-mean-square deviations for the protein and ligand, using the MDAnalysis package,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> as well as visual inspection of the trajectories.</div><div class="NLM_p">Additionally, the interactions of the ligand (ionic interactions, hydrophobic interactions, and aromatic and hydrogen bond interactions) with the protein were analyzed during the MD simulations. This was done by generating a structure-based pharmacophore model at every saved coordinate set of the MD simulation and subsequently analyzing the frequency of the features.</div><div class="NLM_p">Structure-based pharmacophore models were generated using an in-house developed open-source chemoinformatics toolkit (<a href="https://github.com/aglanger/cdpkit" class="extLink">https://github.com/aglanger/cdpkit</a>). The maximum possible frequency value corresponds to the length of the simulation, i.e., for 80 ns the maximum number a specific feature can appear is 8000.</div><div class="NLM_p last">An interaction matrix for the different MD simulations was generated and shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>. The columns of the interaction matrix indicate all amino acid residues that are involved in a pharmacophore feature at some point during the MD simulations, the rows designate moieties of the ligand and the color in the matrix indicate how often a specific amino acid was involved in a specific pharmacophore feature: the colors range from violet (zero interaction) to bright yellow (interaction at every time step), with green as intermediate. To distinguish between different interaction types between ligand and protein, the first letters of the column and row names indicate the interaction type: H for hydrophobic interactions, HBD for hydrogen bond donors, HBA for hydrogen bond acceptors, AR for aromatic interactions, and PI for positive ionizable interactions. The amino acid residue numbers correspond to the residue number in the PDB file and are consistent with earlier publication in the field (e.g., Loland et al).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Ligand atoms were divided in three ring regions for each ligand (ringA1, ringA2, and ringB). The ligand ring naming corresponds to Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.7b01313/suppl_file/jm7b01313_si_001.pdf" class="ext-link">Figure S-3</a>. The interaction map was generated using the Python package matplotlib.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Using this work-flow, it is possible to analyze the number of interaction partners and also their statistical frequency and identify amino acids that are important for binding.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> Animals</h3><div class="NLM_p last">Sixty male Sprague–Dawley rats, aged between 12 and 14 weeks, were used in all experiments. They were bred and maintained in cages made of Makrolon and filled with autoclaved woodchips in the CoreUnit of Biomedical Research, Division of Laboratory Animal Science and Genetics, Medical University of Vienna. Food and water in bottles was available ad libitum. The room was illuminated with artificial light at an intensity of about 200 lx in 2 m from 5 a.m. to 7 p.m.. Experiments were carried out between 8 a.m. and 2 p.m. All procedures were carried out according to the guidelines of the European Communities Council Directive of 24 November 1986 (86/609/EEC) and Ethics committee, Medical University of Vienna, and were approved by Federal Ministry of Education, Science and Culture, Austria (BMWFW-66.009/0114-WF/II/3b/2014). All efforts were made to minimize animal suffering and to reduce the number of animals used.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Behavioral Studies</h3><div class="NLM_p">To study the behavioral effects of <b>11b</b> treatment, multiple experiments were carried out by treating rats (10 per group) with an intraperitoneal dose of <b>11b</b> 10 mg/kg body weight for 10 days and thereafter on the days of each test evaluation. The sequence of tests used was as follows: open field, elevated plus maze, rota rod, neurological observational battery, and forced swim test. Simultaneously a group of 10 rats with DMSO treatment was studied as vehicle control.</div><div id="sec5_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> Open Field (OF)</h4><div class="NLM_p last">Rats were observed by a video monitoring system consisting of a video camcorder coupled to the computational tracking system in an arena (100 cm × 100 cm long, with 40 cm high walls) for 10 min. Each rat was placed in the center, and the following parameters were measured: (a) path length, (b) resting time, (c) percent local movements, (d) percent large movements, (e) speed, (f) times crossing the center, and (g) reversals.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> Elevated Plus Maze (EPM)</h4><div class="NLM_p last">Rats were observed for anxiety-like behavior. The EPM was made out of black PVC and consisted of four arms (each 50 cm long and 10 cm wide) arranged in the shape of a plus sign and elevated 70 cm above the floor. Two opposite arranged arms were open, and the other two arms had 40 cm walls. All arms were interconnected by a 10 cm × 10 cm wide central area. Rats were observed with a video camcorder coupled to a computational tracking system in an arena. Rats were placed on the central area. Following parameters were recorded: (a) the time spent in closed and open arms, (b) number of entries into the closed and open arms, and (c) path length in closed and open arms. An entry was defined as having the rat placing all four legs into the box.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> Rota Rod</h4><div class="NLM_p last">The rota rod (Rota Rod “Economex”, Columbus Instruments, Ohio, USA) tests balance and coordination and is comprised of a rotating drum which accelerates from 4 to 40 rpm over the course of 5 min. The time at which each rat fell from the drum was recorded. Each rat received three pretraining trials. Subsequently, each rat completed three more consecutive trials and the longest time on the drum was used for analysis.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> Neurological Observational Battery</h4><div class="NLM_p last">The procedure follows the set up by Irwin.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> A battery of tests was applied to reveal defects in gait or posture, changes in muscle tone, grip strength, visual acuity, and temperature. To complete the assessment, vitally important reflexes were scored. In addition, during the manipulations, incidences of abnormal behavior, fear, irritability, aggression, excitability, salivation, and lacrimation were recorded.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> Forced Swim Test (FST)</h4><div class="NLM_p last">The test was performed as described previously.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The procedure consisted of two sessions, the pretest session and the test session, using the same apparatus and conditions (diameter 18 cm, height 40 cm, containing 23 cm of water maintained at 25 °C). During the pretest session, rats were forced to swim for 15 min; 24 h later, rats were placed in the same apparatus for 5 min, which wa designated as a test session. The duration of immobility during 5 min was recorded.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i91"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01313">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17652" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17652" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01313" class="ext-link">10.1021/acs.jmedchem.7b01313</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">MD simulation data, structure of CE-103, reuptake inhibition curves, X-ray measurements data, high-resolution mass spectrometry spectra, <sup>1</sup>H and <sup>13</sup>C NMR spectra, HPLC-determined purity data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01313/suppl_file/jm7b01313_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01313/suppl_file/jm7b01313_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01313/suppl_file/jm7b01313_si_001.pdf">jm7b01313_si_001.pdf (17.63 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01313/suppl_file/jm7b01313_si_002.csv">jm7b01313_si_002.csv (2.38 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b01313" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33385" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33385" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thierry Langer</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Life Sciences, University of Vienna, Althanstraße 14, 1090 Vienna, Austria</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#86f2eeefe3f4f4ffa8eae7e8e1e3f4c6f3e8eff0efe3a8e7e5a8e7f2"><span class="__cf_email__" data-cfemail="483c20212d3a3a31662429262f2d3a083d26213e212d66292b66293c">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gert Lubec</span> - <span class="hlFld-Affiliation affiliation">Neuroscience
Laboratory, Paracelsus Medical University, 5020 Salzburg, Austria</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6333-9461" title="Orcid link">http://orcid.org/0000-0002-6333-9461</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2740425553094b52454244674b524542444b46450944484a"><span class="__cf_email__" data-cfemail="81e6e4f3f5afedf4e3e4e2c1edf4e3e4e2ede0e3afe2eeec">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Predrag Kalaba</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Life Sciences, University of Vienna, Althanstraße 14, 1090 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nilima Y. Aher</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Life Sciences, University of Vienna, Althanstraße 14, 1090 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marija Ilić</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Life Sciences, University of Vienna, Althanstraße 14, 1090 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vladimir Dragačević</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Life Sciences, University of Vienna, Althanstraße 14, 1090 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcus Wieder</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Life Sciences, University of Vienna, Althanstraße 14, 1090 Vienna, Austria</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2631-8415" title="Orcid link">http://orcid.org/0000-0003-2631-8415</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andras G. Miklosi</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Life Sciences, University of Vienna, Althanstraße 14, 1090 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Zehl</span> - <span class="hlFld-Affiliation affiliation">Department
of Analytical Chemistry, Faculty of Chemistry, University of Vienna, Währinger Straße 38, 1090 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Judith Wackerlig</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Life Sciences, University of Vienna, Althanstraße 14, 1090 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexander Roller</span> - <span class="hlFld-Affiliation affiliation">X-ray
Structure Analysis Centre, Faculty of Chemistry, University of Vienna, Währinger Straße 38, 1090 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tetyana Beryozkina</span> - <span class="hlFld-Affiliation affiliation">Ural
Federal University named after the first President of Russia B. N.
Yeltsin, 19 Mira St., Yekaterinburg 620002, Russia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bojana Radoman</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Life Sciences, University of Vienna, Althanstraße 14, 1090 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sivaprakasam R. Saroja</span> - <span class="hlFld-Affiliation affiliation">Department
of Pediatrics, Medical University of Vienna, 1090 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wolfgang Lindner</span> - <span class="hlFld-Affiliation affiliation">Department
of Analytical Chemistry, Faculty of Chemistry, University of Vienna, Währinger Straße 38, 1090 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eduardo Perez Gonzalez</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Fine Organic Chemistry, Department of Chemistry and Biochemistry,
Faculty of Science and Technology, University
of Sao Paulo State, Roberto
Simonsen 305, CEP 19060-900 Presidente Prudente, Sao Paolo, Brazil</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vasiliy Bakulev</span> - <span class="hlFld-Affiliation affiliation">Ural
Federal University named after the first President of Russia B. N.
Yeltsin, 19 Mira St., Yekaterinburg 620002, Russia</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3312-7783" title="Orcid link">http://orcid.org/0000-0002-3312-7783</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johann Jakob Leban</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Life Sciences, University of Vienna, Althanstraße 14, 1090 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Harald H. Sitte</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmacology, Centre of Physiology and Pharmacology, Medical University of Vienna, 1090, Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ernst Urban</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Life Sciences, University of Vienna, Althanstraße 14, 1090 Vienna, Austria</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Predrag Kalaba and Nilima Y. Aher have contributed equally to this manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d87e5809-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i93">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56103" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56103" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We are indebted to Dr. Hubert Gstach and Prof. Thomas Erker from the University of Vienna (Austria) for allowing us to use the space and instruments in their laboratories and Marion Holey for her guidance to perform the uptake inhibition assays. Furthermore, we are very grateful to Prof. Gerhard Ecker from the University of Vienna (Austria) for providing us the protein homology model and to FAPESP by conceding a postdoctoral grant (2016/10149-0) to Prof. Eduardo R. P. Gonzalez.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i94" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i94"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i95" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i95"> Abbreviations Used</h2><tr><td class="NLM_term">NET</td><td class="NLM_def"><p class="first last">norepinephrine transporter</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">potassium <i>tert</i>-butoxide</p></td></tr><tr><td class="NLM_term">BF<sub>3</sub>·Et<sub>2</sub>O</td><td class="NLM_def"><p class="first last">boron trifluoride diethyl etherate</p></td></tr><tr><td class="NLM_term">[<sup>3</sup>H]MPP<sup>+</sup></td><td class="NLM_def"><p class="first last">methyl-4-phenylpyridinium</p></td></tr><tr><td class="NLM_term">KHB</td><td class="NLM_def"><p class="first last">Krebs–Henseleit buffer</p></td></tr><tr><td class="NLM_term">PDL</td><td class="NLM_def"><p class="first last">poly <span class="smallcaps smallerCapital">d</span>-lysine</p></td></tr><tr><td class="NLM_term">CDCl<sub>3</sub></td><td class="NLM_def"><p class="first last">deuterated chloroform</p></td></tr><tr><td class="NLM_term">DMSO-<i>d</i><sub>6</sub></td><td class="NLM_def"><p class="first last">deuterated dimethyl sulfoxide</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i96">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72731" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72731" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 47 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Urban, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, W.-J.</span><span> </span><span class="NLM_article-title">Performance enhancement at the cost of potential brain plasticity: neural ramifications of nootropic drugs in the healthy developing brain</span> <span class="citation_source-journal">Front. Syst. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">38</span><span class="refDoi"> DOI: 10.3389/fnsys.2014.00038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.3389%2Ffnsys.2014.00038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=24860437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A280%3ADC%252BC2cflvFamsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=38&author=K.+R.+Urbanauthor=W.-J.+Gao&title=Performance+enhancement+at+the+cost+of+potential+brain+plasticity%3A+neural+ramifications+of+nootropic+drugs+in+the+healthy+developing+brain&doi=10.3389%2Ffnsys.2014.00038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Performance enhancement at the cost of potential brain plasticity: neural ramifications of nootropic drugs in the healthy developing brain</span></div><div class="casAuthors">Urban Kimberly R; Gao Wen-Jun</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in systems neuroscience</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">38</span>
        ISSN:<span class="NLM_cas:issn">1662-5137</span>.
    </div><div class="casAbstract">Cognitive enhancement is perhaps one of the most intriguing and controversial topics in neuroscience today.  Currently, the main classes of drugs used as potential cognitive enhancers include psychostimulants (methylphenidate (MPH), amphetamine), but wakefulness-promoting agents (modafinil) and glutamate activators (ampakine) are also frequently used.  Pharmacologically, substances that enhance the components of the memory/learning circuits-dopamine, glutamate (neuronal excitation), and/or norepinephrine-stand to improve brain function in healthy individuals beyond their baseline functioning.  In particular, non-medical use of prescription stimulants such as MPH and illicit use of psychostimulants for cognitive enhancement have seen a recent rise among teens and young adults in schools and college campuses.  However, this enhancement likely comes with a neuronal, as well as ethical, cost.  Altering glutamate function via the use of psychostimulants may impair behavioral flexibility, leading to the development and/or potentiation of addictive behaviors.  Furthermore, dopamine and norepinephrine do not display linear effects; instead, their modulation of cognitive and neuronal function maps on an inverted-U curve.  Healthy individuals run the risk of pushing themselves beyond optimal levels into hyperdopaminergic and hypernoradrenergic states, thus vitiating the very behaviors they are striving to improve.  Finally, recent studies have begun to highlight potential damaging effects of stimulant exposure in healthy juveniles.  This review explains how the main classes of cognitive enhancing drugs affect the learning and memory circuits, and highlights the potential risks and concerns in healthy individuals, particularly juveniles and adolescents.  We emphasize the performance enhancement at the potential cost of brain plasticity that is associated with the neural ramifications of nootropic drugs in the healthy developing brain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9mMH7P4hI_--MdeMhXAHkfW6udTcc2ea-kV2KCip-jLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cflvFamsg%253D%253D&md5=d552cabbeece23291f10b62d98deb3fc</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3389%2Ffnsys.2014.00038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnsys.2014.00038%26sid%3Dliteratum%253Aachs%26aulast%3DUrban%26aufirst%3DK.%2BR.%26aulast%3DGao%26aufirst%3DW.-J.%26atitle%3DPerformance%2520enhancement%2520at%2520the%2520cost%2520of%2520potential%2520brain%2520plasticity%253A%2520neural%2520ramifications%2520of%2520nootropic%2520drugs%2520in%2520the%2520healthy%2520developing%2520brain%26jtitle%3DFront.%2520Syst.%2520Neurosci.%26date%3D2014%26volume%3D8%26spage%3D38%26doi%3D10.3389%2Ffnsys.2014.00038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Murphy, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekstrand, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarchick, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wideman, C. H.</span><span> </span><span class="NLM_article-title">Modafinil as a cognitive enhancer of spatial working memory in rats</span> <span class="citation_source-journal">Physiol. Behav.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">142</span><span class="NLM_x">, </span> <span class="NLM_fpage">126</span><span class="NLM_x">–</span> <span class="NLM_lpage">130</span><span class="refDoi"> DOI: 10.1016/j.physbeh.2015.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1016%2Fj.physbeh.2015.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=25656691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitl2lu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2015&pages=126-130&author=H.+M.+Murphyauthor=D.+Ekstrandauthor=M.+Tarchickauthor=C.+H.+Wideman&title=Modafinil+as+a+cognitive+enhancer+of+spatial+working+memory+in+rats&doi=10.1016%2Fj.physbeh.2015.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Modafinil as a cognitive enhancer of spatial working memory in rats</span></div><div class="casAuthors">Murphy, Helen M.; Ekstrand, Dylan; Tarchick, Matthew; Wideman, Cyrilla H.</div><div class="citationInfo"><span class="NLM_cas:title">Physiology & Behavior</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">126-130</span>CODEN:
                <span class="NLM_cas:coden">PHBHA4</span>;
        ISSN:<span class="NLM_cas:issn">0031-9384</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The present expt. examd. the influence of modafinil on working memory in rats.  Control and exptl. rats were placed in cages equipped with a running wheel.  The study was divided into three periods: 1. habituation, 2. exptl. - in which modafinil was orally administered to exptl. animals, and 3. withdrawal.  Spatial working memory was tested utilizing the Morris Water Maze.  Animals were given two trials in each session: 1. a sample trial in which they discovered the location of the platform and 2. a test trial in which they recalled the location of the platform.  Platform placement and starting place location of the rats were changed every session.  Performance of control animals during sample trials and test trials showed no difference in time to reach the platform; whereas, exptl. animals demonstrated faster attainment of the goal during the test trial.  In withdrawal, there was no difference in time between the two trials for exptl. animals.  Wheel running activity of the exptl. group was lower than that of the control group during the exptl. period.  This study supports the hypothesis that modafinil has a pos. effect on working memory in rats.  It is suggested that benefits of modafinil can extend beyond its prescribed usage; however, these advantages of modafinil use should be addressed in discussions related to ethical concerns assocd. with neuroenhancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBq28Wmv2ueLVg90H21EOLACvtfcHk0ljhGvIc2fbn6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitl2lu7k%253D&md5=b6b9fb7ea0d0af6d21fcb3129d074638</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.physbeh.2015.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.physbeh.2015.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DMurphy%26aufirst%3DH.%2BM.%26aulast%3DEkstrand%26aufirst%3DD.%26aulast%3DTarchick%26aufirst%3DM.%26aulast%3DWideman%26aufirst%3DC.%2BH.%26atitle%3DModafinil%2520as%2520a%2520cognitive%2520enhancer%2520of%2520spatial%2520working%2520memory%2520in%2520rats%26jtitle%3DPhysiol.%2520Behav.%26date%3D2015%26volume%3D142%26spage%3D126%26epage%3D130%26doi%3D10.1016%2Fj.physbeh.2015.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Gerrard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malcolm, R.</span><span> </span><span class="NLM_article-title">Mechanisms of modafinil: A review of current research</span> <span class="citation_source-journal">Neuropsychiatr. Dis. Treat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">349</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19300566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD2sXps1WnsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=349-364&author=P.+Gerrardauthor=R.+Malcolm&title=Mechanisms+of+modafinil%3A+A+review+of+current+research"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of modafinil: a review of current research</span></div><div class="casAuthors">Gerrard, Paul; Malcolm, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Neuropsychiatric Disease and Treatment</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">349-364</span>CODEN:
                <span class="NLM_cas:coden">NDTEAZ</span>;
        ISSN:<span class="NLM_cas:issn">1176-6328</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press (NZ) Ltd.</span>)
        </div><div class="casAbstract">A review.  The novel wake-promoting agent modafinil has been in use for the treatment of several sleep disorders for a few years and is now undergoing clin. trials for its use in the treatment of stimulant addiction, but its primary mechanism of action remains elusive.  Previous lab. studies have shown that modafinil has antioxidative and neuroprotective effects, which have not previously been suggested to be related to its wake-promoting effects.  However, recent research indicates that free radicals may be related to sleep induction as well as cellular damage, suggesting that a common target of action may mediate modafinil's ability to oppose both of these effects.  In this review we summarize and discuss previously published research on modafinil's neural, cytoprotective, and cognitive effects, and we propose possible primary biochem. targets that could underlie the effects of modafinil obsd. in these studies.  We also suggest neurocognitive mechanisms responsible for modafinil's cognitive enhancing effects and its therapeutic potential in the treatment of stimulant addiction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbxuHCZ8GqFbVg90H21EOLACvtfcHk0ljhGvIc2fbn6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXps1WnsLw%253D&md5=3b9d1c81db02be3d8ecf8c370208bddf</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGerrard%26aufirst%3DP.%26aulast%3DMalcolm%26aufirst%3DR.%26atitle%3DMechanisms%2520of%2520modafinil%253A%2520A%2520review%2520of%2520current%2520research%26jtitle%3DNeuropsychiatr.%2520Dis.%2520Treat.%26date%3D2007%26volume%3D3%26spage%3D349%26epage%3D364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Schmitt, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reith, M. E. A.</span><span> </span><span class="NLM_article-title">The atypical stimulant and nootropic modafinil interacts with the dopamine transporter in a different manner than classical cocaine-like inhibitors</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e25790</span><span class="refDoi"> DOI: 10.1371/journal.pone.0025790</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1371%2Fjournal.pone.0025790" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=22043293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVCku73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e25790&author=K.+C.+Schmittauthor=M.+E.+A.+Reith&title=The+atypical+stimulant+and+nootropic+modafinil+interacts+with+the+dopamine+transporter+in+a+different+manner+than+classical+cocaine-like+inhibitors&doi=10.1371%2Fjournal.pone.0025790"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The atypical stimulant and nootropic modafinil interacts with the dopamine transporter in a different manner than classical cocaine-like inhibitors</span></div><div class="casAuthors">Schmitt, Kyle C.; Reith, Maarten E. A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e25790</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Modafinil is a mild psychostimulant with pro-cognitive and antidepressant effects.  Unlike many conventional stimulants, modafinil has little appreciable potential for abuse, making it a promising therapeutic agent for cocaine addiction.  The chief mol. target of modafinil is the dopamine transporter (DAT); however, the mechanistic details underlying modafinil's unique effects remain unknown.  Recent studies suggest that the conformational effects of a given DAT ligand influence the magnitude of the ligand's reinforcing properties.  For example, the atypical DAT inhibitors benztropine and GBR12909 do not share cocaine's notorious addictive liability, despite having greater binding affinity.  Here, we show that the binding mechanism of modafinil is different than cocaine and similar to other atypical inhibitors.  We previously established two mutations (W84L and D313N) that increase the likelihood that the DAT will adopt an outward-facing conformational state - these mutations increase the affinity of cocaine-like inhibitors considerably, but have little or opposite effect on atypical inhibitor binding.  Thus, a compd.'s WT/mutant affinity ratio can indicate whether the compd. preferentially interacts with a more outward- or inward-facing conformational state.  Modafinil displayed affinity ratios similar to those of benztropine, GBR12909 and bupropion (which lack cocaine-like effects in humans), but far different than those of cocaine, β-CFT or methylphenidate.  Whereas treatment with zinc (known to stabilize an outward-facing transporter state) increased the affinity of cocaine and methylphenidate two-fold, it had little or no effect on the binding of modafinil, benztropine, bupropion or GBR12909.  Addnl., computational modeling of inhibitor binding indicated that while β-CFT and methylphenidate stabilize an "open-to-out" conformation, binding of either modafinil or bupropion gives rise to a more closed conformation.  Our findings highlight a mechanistic difference between modafinil and cocaine-like stimulants and further demonstrate that the conformational effects of a given DAT inhibitor influence its phenomenol. effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk_0tcjHjbqbVg90H21EOLACvtfcHk0lhYL5optloOrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVCku73L&md5=6599e43f3f518b78bf834fb6c6d72010</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025790&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025790%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DK.%2BC.%26aulast%3DReith%26aufirst%3DM.%2BE.%2BA.%26atitle%3DThe%2520atypical%2520stimulant%2520and%2520nootropic%2520modafinil%2520interacts%2520with%2520the%2520dopamine%2520transporter%2520in%2520a%2520different%2520manner%2520than%2520classical%2520cocaine-like%2520inhibitors%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De25790%26doi%3D10.1371%2Fjournal.pone.0025790" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Volkow, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fowler, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alexoff, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Telang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayne, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hooker, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shea, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muench, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apelskog-Torres, K.</span><span> </span><span class="NLM_article-title">Effects of modafinil on dopamine and dopamine transporters in the male human brain: clinical implications</span> <span class="citation_source-journal">JAMA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">301</span><span class="NLM_x">, </span> <span class="NLM_fpage">1148</span><span class="NLM_x">–</span> <span class="NLM_lpage">1154</span><span class="refDoi"> DOI: 10.1001/jama.2009.351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1001%2Fjama.2009.351" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19293415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsVOlsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=301&publication_year=2009&pages=1148-1154&author=N.+D.+Volkowauthor=J.+S.+Fowlerauthor=J.+Loganauthor=D.+Alexoffauthor=W.+Zhuauthor=F.+Telangauthor=G.+J.+Wangauthor=M.+Jayneauthor=J.+M.+Hookerauthor=C.+Wongauthor=+Hubbardauthor=P.+Carterauthor=D.+Warnerauthor=P.+Kingauthor=C.+Sheaauthor=Y.+Xuauthor=L.+Muenchauthor=K.+Apelskog-Torres&title=Effects+of+modafinil+on+dopamine+and+dopamine+transporters+in+the+male+human+brain%3A+clinical+implications&doi=10.1001%2Fjama.2009.351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of modafinil on dopamine and dopamine transporters in the male human brain. Clinical implications</span></div><div class="casAuthors">Volkow, Nora D.; Fowler, Joanna S.; Logan, Jean; Alexoff, David; Zhu, Wei; Telang, Frank; Wang, Gene-Jack; Jayne, Millard; Hooker, Jacob M.; Wong, Christopher; Hubbard, Barbara; Carter, Pauline; Warner, Donald; King, Payton; Shea, Colleen; Xu, Youwen; Muench, Lisa; Apelskog-Torres, Karen</div><div class="citationInfo"><span class="NLM_cas:title">JAMA, the Journal of the American Medical Association</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">301</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1148-1154</span>CODEN:
                <span class="NLM_cas:coden">JAMAAP</span>;
        ISSN:<span class="NLM_cas:issn">0098-7484</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">Modafinil, a wake-promoting drug used to treat narcolepsy, is increasingly being used as a cognitive enhancer.  Although initially launched as distinct from stimulants that increase extracellular dopamine by targeting dopamine transporters, recent preclin. studies suggest otherwise.  Objective was to measure the acute effects of modafinil at doses used therapeutically (200 mg and 400 mg given orally) on extracellular dopamine and on dopamine transporters in the male human brain.  Design, Setting, and Participants Positron emission tomog. with [11C]raclopride (D2/D3 radioligand sensitive to changes in endogenous dopamine) and [11C]cocaine (dopamine transporter radioligand) was used to measure the effects of modafinil on extracellular dopamine and on dopamine transporters in 10 healthy male participants.  The study took place over an 8-mo period (2007-2008) at Brookhaven National Lab.  Main Outcome Measures Primary outcomes were changes in dopamine D2/D3 receptor and dopamine transporter availability (measured by changes in binding potential) after modafinil when compared with after placebo.  Modafinil decreased mean (SD) [11C]raclopride binding potential in caudate (6.1% [6.5%]; 95% confidence interval [CI], 1.5% to 10.8%; P =.02), putamen (6.7% [4.9%]; 95% CI, 3.2% to 10.3%; P =.002), and nucleus accumbens (19.4% [20%]; 95% CI, 5% to 35%; P =.02), reflecting increases in extracellular dopamine.  Modafinil also decreased [11C]cocaine binding potential in caudate (53.8% [13.8%]; 95% CI, 43.9% to 63.6%; P <.001), putamen (47.2% [11.4%]; 95% CI, 39.1% to 55.4%; P <.001), and nucleus accumbens (39.3% [10%]; 95% CI, 30% to 49%; P =.001), reflecting occupancy of dopamine transporters.  In this pilot study, modafinil blocked dopamine transporters and increased dopamine in the human brain (including the nucleus accumbens).  Because drugs that increase dopamine in the nucleus accumbens have the potential for abuse, and considering the increasing use of modafinil, these results highlight the need for heightened awareness for potential abuse of and dependence on modafinil in vulnerable populations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2VFxKbKbpArVg90H21EOLACvtfcHk0lhYL5optloOrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsVOlsbY%253D&md5=a8f5a22cc75df00230c83eb666f7e9de</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1001%2Fjama.2009.351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.2009.351%26sid%3Dliteratum%253Aachs%26aulast%3DVolkow%26aufirst%3DN.%2BD.%26aulast%3DFowler%26aufirst%3DJ.%2BS.%26aulast%3DLogan%26aufirst%3DJ.%26aulast%3DAlexoff%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DTelang%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DG.%2BJ.%26aulast%3DJayne%26aufirst%3DM.%26aulast%3DHooker%26aufirst%3DJ.%2BM.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DHubbard%26aulast%3DCarter%26aufirst%3DP.%26aulast%3DWarner%26aufirst%3DD.%26aulast%3DKing%26aufirst%3DP.%26aulast%3DShea%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DMuench%26aufirst%3DL.%26aulast%3DApelskog-Torres%26aufirst%3DK.%26atitle%3DEffects%2520of%2520modafinil%2520on%2520dopamine%2520and%2520dopamine%2520transporters%2520in%2520the%2520male%2520human%2520brain%253A%2520clinical%2520implications%26jtitle%3DJAMA%26date%3D2009%26volume%3D301%26spage%3D1148%26epage%3D1154%26doi%3D10.1001%2Fjama.2009.351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Mignot, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishino, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guilleminault, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dement, W. C.</span><span> </span><span class="NLM_article-title">Modafinil binds to the dopamine uptake carrier site with low affinity</span> <span class="citation_source-journal">Sleep</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">437</span><span class="refDoi"> DOI: 10.1093/sleep/17.5.436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1093%2Fsleep%2F17.5.436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=7991954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A280%3ADyaK2M%252FosVSmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1994&pages=436-437&author=E.+Mignotauthor=S.+Nishinoauthor=C.+Guilleminaultauthor=W.+C.+Dement&title=Modafinil+binds+to+the+dopamine+uptake+carrier+site+with+low+affinity&doi=10.1093%2Fsleep%2F17.5.436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Modafinil binds to the dopamine uptake carrier site with low affinity</span></div><div class="casAuthors">Mignot E; Nishino S; Guilleminault C; Dement W C</div><div class="citationInfo"><span class="NLM_cas:title">Sleep</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">436-7</span>
        ISSN:<span class="NLM_cas:issn">0161-8105</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTIGv6cQ4HX-Rjv2lLSLOxFfW6udTcc2eatuD3CidXXWLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2M%252FosVSmsA%253D%253D&md5=888ee2341f4b1a375f0ffa25f3d8523d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fsleep%2F17.5.436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fsleep%252F17.5.436%26sid%3Dliteratum%253Aachs%26aulast%3DMignot%26aufirst%3DE.%26aulast%3DNishino%26aufirst%3DS.%26aulast%3DGuilleminault%26aufirst%3DC.%26aulast%3DDement%26aufirst%3DW.%2BC.%26atitle%3DModafinil%2520binds%2520to%2520the%2520dopamine%2520uptake%2520carrier%2520site%2520with%2520low%2520affinity%26jtitle%3DSleep%26date%3D1994%26volume%3D17%26spage%3D436%26epage%3D437%26doi%3D10.1093%2Fsleep%2F17.5.436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Zolkowska, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothman, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Partilla, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prisinzano, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumann, M. H.</span><span> </span><span class="NLM_article-title">Evidence for the involvement of dopamine transporters in the behavioral stimulant effects of modafinil</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">329</span><span class="NLM_x">, </span> <span class="NLM_fpage">738</span><span class="NLM_x">–</span> <span class="NLM_lpage">746</span><span class="refDoi"> DOI: 10.1124/jpet.108.146142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1124%2Fjpet.108.146142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19197004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlt1OlsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=329&publication_year=2009&pages=738-746&author=D.+Zolkowskaauthor=R.+Jainauthor=R.+B.+Rothmanauthor=J.+S.+Partillaauthor=B.+L.+Rothauthor=V.+Setolaauthor=T.+E.+Prisinzanoauthor=M.+H.+Baumann&title=Evidence+for+the+involvement+of+dopamine+transporters+in+the+behavioral+stimulant+effects+of+modafinil&doi=10.1124%2Fjpet.108.146142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for the involvement of dopamine transporters in behavioral stimulant effects of modafinil</span></div><div class="casAuthors">Zolkowska, Dorota; Jain, Raka; Rothman, Richard B.; Partilla, John S.; Roth, Bryan L.; Setola, Vincent; Prisinzano, Thomas E.; Baumann, Michael H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">329</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">738-746</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Modafinil is prescribed for numerous medical conditions, but the drug's mechanism of action is unclear.  Here, we examd. the interaction of modafinil with receptors and transporters in vitro and compared pharmacol. effects of the drug with those produced by indirect dopamine (DA) agonists 1-[2-[bis(4-fluorophenyl)methoxy]ethyl]-4-(3-phenylpropyl)piperazine (GBR12909) and (+)-methamphetamine (METH).  Modafinil was screened at various receptors and transporters using binding assays.  Transporter-mediated uptake and release were examd. in rat brain synaptosomes.  Effects of modafinil on motor activity and neurochem. were detd. in rats undergoing in vivo microdialysis in nucleus accumbens.  Of the receptors and transporters assayed, modafinil displayed measurable potency only at DA transporters (DAT), inhibiting [3H]DA uptake, with an IC50 value of 4.0 μM.  Accordingly, modafinil pretreatment (10 μM) antagonized METH-induced release of the DAT substrate [3H]1-methyl-4-phenylpyridinium.  I.v. modafinil (20 and 60 mg/kg) produced dose-dependent increases in motor activity and extracellular DA, without affecting serotonin (5-HT).  Analogous results were obsd. for GBR12909 (1 and 3 mg/kg), whereas METH (0.3 and 1 mg/kg) increased DA and 5-HT.  Locomotor effects of all drugs were pos. correlated with dialyzate DA (P < 0.001).  Interestingly, modafinil pretreatment reduced METH-induced ambulation and DA release.  Our data show that modafinil interacts with DAT sites in rat brain, a property shared with agonist medications under investigation for treating cocaine dependence.  Nondopaminergic mechanisms may also contribute to the pharmacol. of modafinil.  Finally, the results suggest that modafinil should be tested as an adjunct for treating METH addiction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhab522ik-NrVg90H21EOLACvtfcHk0lhDayhNblcIwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlt1OlsL4%253D&md5=0c5af6285ed23382c46a0d35bafbaa3b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1124%2Fjpet.108.146142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.108.146142%26sid%3Dliteratum%253Aachs%26aulast%3DZolkowska%26aufirst%3DD.%26aulast%3DJain%26aufirst%3DR.%26aulast%3DRothman%26aufirst%3DR.%2BB.%26aulast%3DPartilla%26aufirst%3DJ.%2BS.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DPrisinzano%26aufirst%3DT.%2BE.%26aulast%3DBaumann%26aufirst%3DM.%2BH.%26atitle%3DEvidence%2520for%2520the%2520involvement%2520of%2520dopamine%2520transporters%2520in%2520the%2520behavioral%2520stimulant%2520effects%2520of%2520modafinil%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2009%26volume%3D329%26spage%3D738%26epage%3D746%26doi%3D10.1124%2Fjpet.108.146142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Loland, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mereu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okunola, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prisinzano, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopajtic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katz, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">R-Modafinil (Armodafinil): a unique dopamine uptake inhibitor and potential medication for psychostimulant abuse</span> <span class="citation_source-journal">Biol. Psychiatry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">413</span><span class="refDoi"> DOI: 10.1016/j.biopsych.2012.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1016%2Fj.biopsych.2012.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=22537794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFelt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=405-413&author=J.+C.+Lolandauthor=M.+Mereuauthor=O.+Okunolaauthor=J.+Caoauthor=T.+E.+Prisinzanoauthor=S.+Mazierauthor=T.+Kopajticauthor=L.+Shiauthor=J.+L.+Katzauthor=G.+Tandaauthor=A.+H.+Newman&title=R-Modafinil+%28Armodafinil%29%3A+a+unique+dopamine+uptake+inhibitor+and+potential+medication+for+psychostimulant+abuse&doi=10.1016%2Fj.biopsych.2012.03.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">R-Modafinil (Armodafinil): A Unique Dopamine Uptake Inhibitor and Potential Medication for Psychostimulant Abuse</span></div><div class="casAuthors">Loland, Claus J.; Mereu, Maddalena; Okunola, Oluyomi M.; Cao, Jianjing; Prisinzano, Thomas E.; Mazier, Sonia; Kopajtic, Theresa; Shi, Lei; Katz, Jonathan L.; Tanda, Gianluigi; Newman, Amy Hauck</div><div class="citationInfo"><span class="NLM_cas:title">Biological Psychiatry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">405-413</span>CODEN:
                <span class="NLM_cas:coden">BIPCBF</span>;
        ISSN:<span class="NLM_cas:issn">0006-3223</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">(±)-Modafinil has piqued interest as a treatment for attention-deficit/hyperactivity disorder and stimulant dependence.  The R-enantiomer of modafinil might have unique pharmacol. properties that should be further investigated.  (±)-Modafinil and its R-(-)- and S-(+)-enantiomers were synthesized and tested for inhibition of [3H] dopamine (DA) uptake and [3H]WIN 35428 binding in human dopamine transporter (DAT) wild-type and mutants with altered conformational equil.  Data were compared with cocaine and the atypical DA uptake inhibitor, JHW 007.  R- and S-modafinil were also evaluated in microdialysis studies in the mouse nucleus accumbens shell and in a cocaine discrimination procedure.  (±)-, R-, and S-modafinil bind to the DAT and inhibit DA uptake less potently than cocaine, with R-modafinil having approx. threefold higher affinity than its S-enantiomer.  Mol. docking studies revealed subtle differences in binding modes for the enantiomers.  R-modafinil was significantly less potent in the DAT Y156F mutant compared with wild-type DAT, whereas S-modafinil was affected less.  Studies with the Y335A DAT mutant showed that the R- and S-enantiomers tolerated the inward-facing conformation better than cocaine, which was further supported by [2-(trimethylammonium)ethyl]-methanethiosulfonate reactivity on the DAT E2C I159C.  Microdialysis studies demonstrated that both R- and S-modafinil produced increases in extracellular DA concns. in the nucleus accumbens shell less efficaciously than cocaine and with a longer duration of action.  Both enantiomers fully substituted in mice trained to discriminate cocaine from saline.R-modafinil displays an in vitro profile different from cocaine.  Future trials with R-modafinil as a substitute therapy with the potential benefit of cognitive enhancement for psychostimulant addiction are warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjL4KiRs4PUbVg90H21EOLACvtfcHk0lhDayhNblcIwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFelt74%253D&md5=78808210fc68d2fbcdd97190f03b63d0</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.biopsych.2012.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopsych.2012.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DLoland%26aufirst%3DJ.%2BC.%26aulast%3DMereu%26aufirst%3DM.%26aulast%3DOkunola%26aufirst%3DO.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DPrisinzano%26aufirst%3DT.%2BE.%26aulast%3DMazier%26aufirst%3DS.%26aulast%3DKopajtic%26aufirst%3DT.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DKatz%26aufirst%3DJ.%2BL.%26aulast%3DTanda%26aufirst%3DG.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DR-Modafinil%2520%2528Armodafinil%2529%253A%2520a%2520unique%2520dopamine%2520uptake%2520inhibitor%2520and%2520potential%2520medication%2520for%2520psychostimulant%2520abuse%26jtitle%3DBiol.%2520Psychiatry%26date%3D2012%26volume%3D72%26spage%3D405%26epage%3D413%26doi%3D10.1016%2Fj.biopsych.2012.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prisinzano, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okunola, O. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopajtic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shook, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katz, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">SARs at the monoamine transporters for a novel series of modafinil analogues</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">48</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span><span class="refDoi"> DOI: 10.1021/ml1002025</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml1002025" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12lt7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=48-52&author=J.+Caoauthor=T.+E.+Prisinzanoauthor=O.+M.+Okunolaauthor=T.+Kopajticauthor=M.+Shookauthor=J.+L.+Katzauthor=A.+H.+Newman&title=SARs+at+the+monoamine+transporters+for+a+novel+series+of+modafinil+analogues&doi=10.1021%2Fml1002025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">SARs at the Monoamine Transporters for a Novel Series of Modafinil Analogues</span></div><div class="casAuthors">Cao, Jianjing; Prisinzano, Thomas E.; Okunola, Oluyomi M.; Kopajtic, Theresa; Shook, Matthew; Katz, Jonathan L.; Hauck Newman, Amy</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-52</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of modafinil (1) analogs were synthesized wherein (1) para-halo-substitutents were added to the aryl rings, (2) the sulfoxide function was removed, and (3) the primary amide group was replaced with secondary and tertiary amides and amines to investigate the effects of these chem. modifications on dopamine transporter, serotonin transporter, and norepinephrine transporter binding.  In addn., the locomotor-stimulant effects in mice of (±)-modafinil (1) (I), its R- and S-enantiomers, and its para-chloro sulfinylacetamide analog (5c) (II) were compared to those of cocaine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDr9ac-J7nqbVg90H21EOLACvtfcHk0lhDayhNblcIwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12lt7bN&md5=660e546e8524b2e5e8b448ee5bf0e567</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fml1002025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml1002025%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DPrisinzano%26aufirst%3DT.%2BE.%26aulast%3DOkunola%26aufirst%3DO.%2BM.%26aulast%3DKopajtic%26aufirst%3DT.%26aulast%3DShook%26aufirst%3DM.%26aulast%3DKatz%26aufirst%3DJ.%2BL.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DSARs%2520at%2520the%2520monoamine%2520transporters%2520for%2520a%2520novel%2520series%2520of%2520modafinil%2520analogues%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D2%26spage%3D48%26epage%3D52%26doi%3D10.1021%2Fml1002025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Okunola-Bakare, O. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopajtic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katz, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loland, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Elucidation of structural elements for selectivity across monoamine transporters: novel 2-[(diphenylmethyl)sulfynil]acetamide (modafinil) analogues</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1000</span><span class="NLM_x">–</span> <span class="NLM_lpage">1013</span><span class="refDoi"> DOI: 10.1021/jm401754x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401754x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1000-1013&author=O.+M.+Okunola-Bakareauthor=J.+Caoauthor=T.+Kopajticauthor=L.+J.+Katzauthor=C.+J.+Lolandauthor=L.+Shiauthor=A.+H.+Newman&title=Elucidation+of+structural+elements+for+selectivity+across+monoamine+transporters%3A+novel+2-%5B%28diphenylmethyl%29sulfynil%5Dacetamide+%28modafinil%29+analogues&doi=10.1021%2Fjm401754x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm401754x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401754x%26sid%3Dliteratum%253Aachs%26aulast%3DOkunola-Bakare%26aufirst%3DO.%2BM.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DKopajtic%26aufirst%3DT.%26aulast%3DKatz%26aufirst%3DL.%2BJ.%26aulast%3DLoland%26aufirst%3DC.%2BJ.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DElucidation%2520of%2520structural%2520elements%2520for%2520selectivity%2520across%2520monoamine%2520transporters%253A%2520novel%25202-%255B%2528diphenylmethyl%2529sulfynil%255Dacetamide%2520%2528modafinil%2529%2520analogues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1000%26epage%3D1013%26doi%3D10.1021%2Fjm401754x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slack, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bakare, O. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burzynski, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rais, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slusher, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopajtic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonifazi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellenberger, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yano, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bi, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xi, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loland, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Novel and high affinity 2-[(diphenylmethyl)sulfynil]acetamide (modafinil) analogues as atypical dopamine transporter inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">10676</span><span class="NLM_x">–</span> <span class="NLM_lpage">10691</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFelu7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10676-10691&author=J.+Caoauthor=R.+D.+Slackauthor=O.+M.+Bakareauthor=C.+Burzynskiauthor=R.+Raisauthor=B.+S.+Slusherauthor=T.+Kopajticauthor=A.+Bonifaziauthor=M.+P.+Ellenbergerauthor=H.+Yanoauthor=Y.+Heauthor=G.+H.+Biauthor=Z.+H.+Xiauthor=C.+J.+Lolandauthor=A.+H.+Newman&title=Novel+and+high+affinity+2-%5B%28diphenylmethyl%29sulfynil%5Dacetamide+%28modafinil%29+analogues+as+atypical+dopamine+transporter+inhibitors&doi=10.1021%2Facs.jmedchem.6b01373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Novel and High Affinity 2-[(Diphenylmethyl)sulfinyl]acetamide (Modafinil) Analogues as Atypical Dopamine Transporter Inhibitors</span></div><div class="casAuthors">Cao, Jianjing; Slack, Rachel D.; Bakare, Oluyomi M.; Burzynski, Caitlin; Rais, Rana; Slusher, Barbara S.; Kopajtic, Theresa; Bonifazi, Alessandro; Ellenberger, Michael P.; Yano, Hideaki; He, Yi; Bi, Guo-Hua; Xi, Zheng-Xiong; Loland, Claus J.; Newman, Amy Hauck</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10676-10691</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of pharmacotherapeutic treatments of psychostimulant abuse has remained a challenge, despite significant efforts made toward relevant mechanistic targets, such as the dopamine transporter (DAT).  The atypical DAT inhibitors have received attention due to their promising pharmacol. profiles in animal models of cocaine and methamphetamine abuse.  Herein the authors report a series of modafinil analogs that have an atypical DAT inhibitor profile.  The authors extended SAR by chem. manipulating the oxidn. states of the sulfoxide and the amide functional groups, halogenating the Ph rings, and/or functionalizing the terminal nitrogen with substituted piperazines, resulting in several novel leads such as 1-(4-(2-((bis(4-fluorophenyl)methyl)sulfinyl)ethyl)piperazin-1-yl)-3-phenylpropan-2-ol (compd. 11b), which demonstrated high DAT affinity (Ki = 2.5 nM) and selectivity without producing concomitant locomotor stimulation in mice, as compared to cocaine.  These results are consistent with an atypical DAT inhibitor profile and suggest that 11b may be a potential lead for development as a psychostimulant abuse medication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL0BwESHNuRbVg90H21EOLACvtfcHk0liagJwx-uHo-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFelu7bP&md5=e912f256c764249ad606d83ac9c570fb</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01373%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DSlack%26aufirst%3DR.%2BD.%26aulast%3DBakare%26aufirst%3DO.%2BM.%26aulast%3DBurzynski%26aufirst%3DC.%26aulast%3DRais%26aufirst%3DR.%26aulast%3DSlusher%26aufirst%3DB.%2BS.%26aulast%3DKopajtic%26aufirst%3DT.%26aulast%3DBonifazi%26aufirst%3DA.%26aulast%3DEllenberger%26aufirst%3DM.%2BP.%26aulast%3DYano%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DBi%26aufirst%3DG.%2BH.%26aulast%3DXi%26aufirst%3DZ.%2BH.%26aulast%3DLoland%26aufirst%3DC.%2BJ.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DNovel%2520and%2520high%2520affinity%25202-%255B%2528diphenylmethyl%2529sulfynil%255Dacetamide%2520%2528modafinil%2529%2520analogues%2520as%2520atypical%2520dopamine%2520transporter%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10676%26epage%3D10691%26doi%3D10.1021%2Facs.jmedchem.6b01373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Beuming, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kniazeff, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergmann, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gracia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raniszewska, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Javitch, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gether, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loland, C. J.</span><span> </span><span class="NLM_article-title">The binding sites for cocaine and dopamine in the dopamine transporter overlap</span> <span class="citation_source-journal">Nat. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">780</span><span class="NLM_x">–</span> <span class="NLM_lpage">789</span><span class="refDoi"> DOI: 10.1038/nn.2146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1038%2Fnn.2146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=18568020" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1cXns1Omsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2008&pages=780-789&author=T.+Beumingauthor=J.+Kniazeffauthor=M.+L.+Bergmannauthor=L.+Shiauthor=L.+Graciaauthor=K.+Raniszewskaauthor=A.+H.+Newmanauthor=J.+A.+Javitchauthor=H.+Weinsteinauthor=U.+Getherauthor=C.+J.+Loland&title=The+binding+sites+for+cocaine+and+dopamine+in+the+dopamine+transporter+overlap&doi=10.1038%2Fnn.2146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The binding sites for cocaine and dopamine in the dopamine transporter overlap</span></div><div class="casAuthors">Beuming, Thijs; Kniazeff, Julie; Bergmann, Marianne L.; Shi, Lei; Gracia, Luis; Raniszewska, Klaudia; Newman, Amy Hauck; Javitch, Jonathan A.; Weinstein, Harel; Gether, Ulrik; Loland, Claus J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">780-789</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Beuming and colleagues detd. that the binding site for cocaine overlaps with that of dopamine on the dopamine transporter.  Detailed modeling and mutagenesis expts. revealed that this site is deeply buried amongst several transmembrane domains.  Cocaine is a widely abused substance with psychostimulant effects that are attributed to inhibition of the dopamine transporter (DAT).  We present mol. models for DAT binding of cocaine and cocaine analogs constructed from the high-resoln. structure of the bacterial transporter homolog LeuT.  Our models suggest that the binding site for cocaine and cocaine analogs is deeply buried between transmembrane segments 1, 3, 6 and 8, and overlaps with the binding sites for the substrates dopamine and amphetamine, as well as for benztropine-like DAT inhibitors.  We validated our models by detailed mutagenesis and by trapping the radiolabeled cocaine analog [3H]CFT in the transporter, either by crosslinking engineered cysteines or with an engineered Zn2+-binding site that was situated extracellularly to the predicted common binding pocket.  Our data demonstrate the mol. basis for the competitive inhibition of dopamine transport by cocaine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogCWenpzRZ47Vg90H21EOLACvtfcHk0liagJwx-uHo-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXns1Omsrw%253D&md5=7f96344e92478939540d84a37eaf3a70</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnn.2146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.2146%26sid%3Dliteratum%253Aachs%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DKniazeff%26aufirst%3DJ.%26aulast%3DBergmann%26aufirst%3DM.%2BL.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DGracia%26aufirst%3DL.%26aulast%3DRaniszewska%26aufirst%3DK.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DJavitch%26aufirst%3DJ.%2BA.%26aulast%3DWeinstein%26aufirst%3DH.%26aulast%3DGether%26aufirst%3DU.%26aulast%3DLoland%26aufirst%3DC.%2BJ.%26atitle%3DThe%2520binding%2520sites%2520for%2520cocaine%2520and%2520dopamine%2520in%2520the%2520dopamine%2520transporter%2520overlap%26jtitle%3DNat.%2520Neurosci.%26date%3D2008%26volume%3D11%26spage%3D780%26epage%3D789%26doi%3D10.1038%2Fnn.2146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Saha, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Partilla, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehner, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seddik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandtner, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitte, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumann, M. H.</span><span> </span><span class="NLM_article-title">Second-generation” mephedrone analogs, 4-MEC and 4-MePPP, differentially affect monoamine transporter function</span> <span class="citation_source-journal">Neuropsychopharmacology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">1321</span><span class="NLM_x">–</span> <span class="NLM_lpage">1331</span><span class="refDoi"> DOI: 10.1038/npp.2014.325</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1038%2Fnpp.2014.325" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2015&pages=1321-1331&author=K.+Sahaauthor=J.+S.+Partillaauthor=K.+R.+Lehnerauthor=A.+Seddikauthor=T.+Stocknerauthor=M.+Holyauthor=W.+Sandtnerauthor=G.+F.+Eckerauthor=H.+H.+Sitteauthor=M.+H.+Baumann&title=Second-generation%E2%80%9D+mephedrone+analogs%2C+4-MEC+and+4-MePPP%2C+differentially+affect+monoamine+transporter+function&doi=10.1038%2Fnpp.2014.325"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnpp.2014.325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnpp.2014.325%26sid%3Dliteratum%253Aachs%26aulast%3DSaha%26aufirst%3DK.%26aulast%3DPartilla%26aufirst%3DJ.%2BS.%26aulast%3DLehner%26aufirst%3DK.%2BR.%26aulast%3DSeddik%26aufirst%3DA.%26aulast%3DStockner%26aufirst%3DT.%26aulast%3DHoly%26aufirst%3DM.%26aulast%3DSandtner%26aufirst%3DW.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DSitte%26aufirst%3DH.%2BH.%26aulast%3DBaumann%26aufirst%3DM.%2BH.%26atitle%3DSecond-generation%25E2%2580%259D%2520mephedrone%2520analogs%252C%25204-MEC%2520and%25204-MePPP%252C%2520differentially%2520affect%2520monoamine%2520transporter%2520function%26jtitle%3DNeuropsychopharmacology%26date%3D2015%26volume%3D40%26spage%3D1321%26epage%3D1331%26doi%3D10.1038%2Fnpp.2014.325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Sitte, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reither, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boehm, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singer, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pifl, C.</span><span> </span><span class="NLM_article-title">Carrier-mediated release, transport rates, and charge transfer induced by amphetamine, tyramine, and dopamine in mammalian cells transfected with the human dopamine transporter</span> <span class="citation_source-journal">J. Neurochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">1289</span><span class="NLM_x">–</span> <span class="NLM_lpage">1297</span><span class="refDoi"> DOI: 10.1046/j.1471-4159.1998.71031289.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1046%2Fj.1471-4159.1998.71031289.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=9721755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADyaK1cXls1Khtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=1998&pages=1289-1297&author=H.+H.+Sitteauthor=S.+Huckauthor=H.+Reitherauthor=S.+Boehmauthor=E.+A.+Singerauthor=C.+Pifl&title=Carrier-mediated+release%2C+transport+rates%2C+and+charge+transfer+induced+by+amphetamine%2C+tyramine%2C+and+dopamine+in+mammalian+cells+transfected+with+the+human+dopamine+transporter&doi=10.1046%2Fj.1471-4159.1998.71031289.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Carrier-mediated release, transport rates, and charge transfer induced by amphetamine, tyramine, and dopamine in mammalian cells transfected with the human dopamine transporter</span></div><div class="casAuthors">Sitte, H. H.; Huck, S.; Reither, H.; Boehm, S.; Singer, E. A.; Pifl, C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1289-1297</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Lippincott-Raven Publishers</span>)
        </div><div class="casAbstract">Amphetamine and related substances induce dopamine release.  According to a traditional explanation, this dopamine release occurs in exchange for amphetamine by the dopamine transporter (DAT).  The authors tested this hypothesis in human embryonic kidney 293 cells stably transfected with the human DAT by measuring the uptake of dopamine, tyramine, and D- and L-amphetamine as well as substrate-induced release of preloaded N-methyl-4-[3H]phenylpyridinium ([3H]MPP+).  The uptake of substrates was sodium-dependent and was inhibited by ouabain and cocaine, which also prevented substrate-induced release of MPP+.  Patch-clamp recordings revealed that all four substrates elicited voltage-dependent inward currents (on top of constitutive leak currents) that were prevented by cocaine.  Whereas individual substrates had similar affinities in release, uptake, and patch-clamp expts., maximal effects displayed remarkable differences.  Hence, maximal effects in release and current induction were ∼25% higher for D-amphetamine as compared with the other substrates.  By contrast, dopamine was the most efficacious substrate in uptake expts., with its maximal initial uptake rate exceeding those of amphetamine and tyramine by factors of 20 and 4, resp.  The authors' expts. indicate a poor correlation between substrate-induced release and the transport of substrates, whereas the ability of substrates to induce currents correlates well with their releasing action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYoVL6P5Eh27Vg90H21EOLACvtfcHk0ljZGhybg5P2CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXls1Khtbs%253D&md5=89cf153f292293927320301b2ec49b70</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1046%2Fj.1471-4159.1998.71031289.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1471-4159.1998.71031289.x%26sid%3Dliteratum%253Aachs%26aulast%3DSitte%26aufirst%3DH.%2BH.%26aulast%3DHuck%26aufirst%3DS.%26aulast%3DReither%26aufirst%3DH.%26aulast%3DBoehm%26aufirst%3DS.%26aulast%3DSinger%26aufirst%3DE.%2BA.%26aulast%3DPifl%26aufirst%3DC.%26atitle%3DCarrier-mediated%2520release%252C%2520transport%2520rates%252C%2520and%2520charge%2520transfer%2520induced%2520by%2520amphetamine%252C%2520tyramine%252C%2520and%2520dopamine%2520in%2520mammalian%2520cells%2520transfected%2520with%2520the%2520human%2520dopamine%2520transporter%26jtitle%3DJ.%2520Neurochem.%26date%3D1998%26volume%3D71%26spage%3D1289%26epage%3D1297%26doi%3D10.1046%2Fj.1471-4159.1998.71031289.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Sarker, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissensteiner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steiner, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitte, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freissmuth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sucic, S.</span><span> </span><span class="NLM_article-title">The high-affinity binding site for tricyclic antidepressants resides in the outer vestibule of the serotonin transporter</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">1026</span><span class="NLM_x">–</span> <span class="NLM_lpage">1035</span><span class="refDoi"> DOI: 10.1124/mol.110.067538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1124%2Fmol.110.067538" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2010&pages=1026-1035&author=S.+Sarkerauthor=R.+Weissensteinerauthor=I.+Steinerauthor=H.+H.+Sitteauthor=G.+F.+Eckerauthor=M.+Freissmuthauthor=S.+Sucic&title=The+high-affinity+binding+site+for+tricyclic+antidepressants+resides+in+the+outer+vestibule+of+the+serotonin+transporter&doi=10.1124%2Fmol.110.067538"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1124%2Fmol.110.067538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.110.067538%26sid%3Dliteratum%253Aachs%26aulast%3DSarker%26aufirst%3DS.%26aulast%3DWeissensteiner%26aufirst%3DR.%26aulast%3DSteiner%26aufirst%3DI.%26aulast%3DSitte%26aufirst%3DH.%2BH.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DFreissmuth%26aufirst%3DM.%26aulast%3DSucic%26aufirst%3DS.%26atitle%3DThe%2520high-affinity%2520binding%2520site%2520for%2520tricyclic%2520antidepressants%2520resides%2520in%2520the%2520outer%2520vestibule%2520of%2520the%2520serotonin%2520transporter%26jtitle%3DMol.%2520Pharmacol.%26date%3D2010%26volume%3D78%26spage%3D1026%26epage%3D1035%26doi%3D10.1124%2Fmol.110.067538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Segel, I. H.</span> <span class="citation_source-book">Enzyme Kinetics: Behavior and Analysis of Rapid Equilibrium and Steady-State Enzyme Systems</span>; <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1975</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1975&author=I.+H.+Segel&title=Enzyme+Kinetics%3A+Behavior+and+Analysis+of+Rapid+Equilibrium+and+Steady-State+Enzyme+Systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSegel%26aufirst%3DI.%2BH.%26btitle%3DEnzyme%2520Kinetics%253A%2520Behavior%2520and%2520Analysis%2520of%2520Rapid%2520Equilibrium%2520and%2520Steady-State%2520Enzyme%2520Systems%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D1975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Meng, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mezei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, M.</span><span> </span><span class="NLM_article-title">Molecular Docking: A powerful approach for structure-based drug discovery</span> <span class="citation_source-journal">Curr. Comput.-Aided Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">157</span><span class="refDoi"> DOI: 10.2174/157340911795677602</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.2174%2F157340911795677602" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=21534921" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsFyrsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=146-157&author=Y.+H.+Mengauthor=H.+X.+Zhangauthor=M.+Mezeiauthor=M.+Cui&title=Molecular+Docking%3A+A+powerful+approach+for+structure-based+drug+discovery&doi=10.2174%2F157340911795677602"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking: A powerful approach for structure-based drug discovery</span></div><div class="casAuthors">Meng, Xuan-Yu; Zhang, Hong-Xing; Mezei, Mihaly; Cui, Meng</div><div class="citationInfo"><span class="NLM_cas:title">Current Computer-Aided Drug Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-157</span>CODEN:
                <span class="NLM_cas:coden">CCDDAS</span>;
        ISSN:<span class="NLM_cas:issn">1573-4099</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Mol. docking has become an increasingly important tool for drug discovery.  In this review, we present a brief introduction of the available mol. docking methods, and their development and applications in drug discovery.  The relevant basic theories, including sampling algorithms and scoring functions, are summarized.  The differences in and performance of available docking software are also discussed.  Flexible receptor mol. docking approaches, esp. those including backbone flexibility in receptors, are a challenge for available docking methods.  A recently developed Local Move Monte Carlo (LMMC) based approach is introduced as a potential soln. to flexible receptor docking problems.  Three application examples of mol. docking approaches for drug discovery are provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKdxlSun_gubVg90H21EOLACvtfcHk0ljm1PeckAKP6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsFyrsLY%253D&md5=b1712e0c9091fae4c71c280b296b61f4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F157340911795677602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157340911795677602%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DY.%2BH.%26aulast%3DZhang%26aufirst%3DH.%2BX.%26aulast%3DMezei%26aufirst%3DM.%26aulast%3DCui%26aufirst%3DM.%26atitle%3DMolecular%2520Docking%253A%2520A%2520powerful%2520approach%2520for%2520structure-based%2520drug%2520discovery%26jtitle%3DCurr.%2520Comput.-Aided%2520Drug%2520Des.%26date%3D2011%26volume%3D7%26spage%3D146%26epage%3D157%26doi%3D10.2174%2F157340911795677602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Liu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kokubo, H.</span><span> </span><span class="NLM_article-title">Exploring the stability of ligand binding modes to proteins by molecular dynamics simulations</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">201</span><span class="NLM_x">–</span> <span class="NLM_lpage">211</span><span class="refDoi"> DOI: 10.1007/s10822-016-0005-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1007%2Fs10822-016-0005-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=28074360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsl2jsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=201-211&author=K.+Liuauthor=E.+Watanabeauthor=H.+Kokubo&title=Exploring+the+stability+of+ligand+binding+modes+to+proteins+by+molecular+dynamics+simulations&doi=10.1007%2Fs10822-016-0005-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the stability of ligand binding modes to proteins by molecular dynamics simulations</span></div><div class="casAuthors">Liu, Kai; Watanabe, Etsurou; Kokubo, Hironori</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-211</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The binding mode prediction is of great importance to structure-based drug design.  The discrimination of various binding poses of ligand generated by docking is a great challenge not only to docking score functions but also to the relatively expensive free energy calcn. methods.  Here we systematically analyzed the stability of various ligand poses under mol. dynamics (MD) simulation.  First, a data set of 120 complexes was built based on the typical physicochem. properties of drug-like ligands.  Three potential binding poses (one correct pose and two decoys) were selected for each ligand from self-docking in addn. to the exptl. pose.  Then, five independent MD simulations for each pose were performed with different initial velocities for the statistical anal.  Finally, the stabilities of ligand poses under MD were evaluated and compared with the native one from crystal structure.  We found that about 94% of the native poses were maintained stable during the simulations, which suggests that MD simulations are accurate enough to judge most exptl. binding poses as stable properly.  Interestingly, incorrect decoy poses were maintained much less and 38-44% of decoys could be excluded just by performing equil. MD simulations, though 56-62% of decoys were stable.  The computationally-heavy binding free energy calcn. can be performed only for these survived poses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfcZT8myW_ZbVg90H21EOLACvtfcHk0ljm1PeckAKP6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsl2jsQ%253D%253D&md5=26e962221ef86fa1cab0ec25bf89a16d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2Fs10822-016-0005-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-016-0005-2%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DWatanabe%26aufirst%3DE.%26aulast%3DKokubo%26aufirst%3DH.%26atitle%3DExploring%2520the%2520stability%2520of%2520ligand%2520binding%2520modes%2520to%2520proteins%2520by%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2017%26volume%3D31%26spage%3D201%26epage%3D211%26doi%3D10.1007%2Fs10822-016-0005-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Sakano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahamood, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujitani, H.</span><span> </span><span class="NLM_article-title">Molecular dynamics analysis to evaluate docking pose prediction</span> <span class="citation_source-journal">Biophys. Physicobiology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">194</span><span class="refDoi"> DOI: 10.2142/biophysico.13.0_181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.2142%2Fbiophysico.13.0_181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=27924273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A280%3ADC%252BC2sjktFOlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=181-194&author=T.+Sakanoauthor=M.+I.+Mahamoodauthor=T.+Yamashitaauthor=H.+Fujitani&title=Molecular+dynamics+analysis+to+evaluate+docking+pose+prediction&doi=10.2142%2Fbiophysico.13.0_181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics analysis to evaluate docking pose prediction</span></div><div class="casAuthors">Sakano Takako; Mahamood Md Iqbal; Yamashita Takefumi; Fujitani Hideaki</div><div class="citationInfo"><span class="NLM_cas:title">Biophysics and physicobiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">181-194</span>
        ISSN:<span class="NLM_cas:issn">2189-4779</span>.
    </div><div class="casAbstract">The accurate prediction of a ligand-protein complex structure is important for computer-assisted drug development.  Although many docking methods have been developed over the last three decades, the success of binding structure prediction remains greatly limited.  The purpose of this study was to demonstrate the usefulness of molecular dynamics (MD) simulation in assessing a docking pose predicted using a docking program.  If the predicted pose is not unstable in an aqueous environment, MD simulation equilibrates the system and removes the ligand from the predicted position.  Here we investigated two proteins that are important potential therapeutic targets: β2 adrenergic receptor (β2AR) and PR-Set7.  While β2AR is rigid and its ligands are very similar to the template ligand (carazolol), PR-Set7 is very flexible and its ligands vary greatly from the template ligand (histone H4 tail peptide).  On an empirical basis, we usually expect that the docking prediction is accurate when the protein is rigid and its ligands are similar to the template ligand.  The MD analyses in this study clearly suggested such a tendency.  Furthermore, we discuss the possibility that the MD simulation can predict the binding pose of a ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQbbAkhVKypBrYtC3U5jeOifW6udTcc2ebXYYiC5gtVfLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjktFOlsA%253D%253D&md5=fbdf9e7fa6a1b8f4803b93a393f6b2c9</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2142%2Fbiophysico.13.0_181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2142%252Fbiophysico.13.0_181%26sid%3Dliteratum%253Aachs%26aulast%3DSakano%26aufirst%3DT.%26aulast%3DMahamood%26aufirst%3DM.%2BI.%26aulast%3DYamashita%26aufirst%3DT.%26aulast%3DFujitani%26aufirst%3DH.%26atitle%3DMolecular%2520dynamics%2520analysis%2520to%2520evaluate%2520docking%2520pose%2520prediction%26jtitle%3DBiophys.%2520Physicobiology%26date%3D2016%26volume%3D13%26spage%3D181%26epage%3D194%26doi%3D10.2142%2Fbiophysico.13.0_181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Khelashvili, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanley, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahai, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeVine, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeFabritiis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, H.</span><span> </span><span class="NLM_article-title">Spontaneous inward opening of the dopamine transporter is triggered by PIP2-regulated dynamics of the N-terminus</span> <span class="citation_source-journal">ACS Chem. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1825</span><span class="NLM_x">–</span> <span class="NLM_lpage">1837</span><span class="refDoi"> DOI: 10.1021/acschemneuro.5b00179</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschemneuro.5b00179" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht12nu73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1825-1837&author=G.+Khelashviliauthor=N.+Stanleyauthor=M.+A.+Sahaiauthor=J.+Medinaauthor=M.+V.+LeVineauthor=L.+Shiauthor=G.+DeFabritiisauthor=H.+Weinstein&title=Spontaneous+inward+opening+of+the+dopamine+transporter+is+triggered+by+PIP2-regulated+dynamics+of+the+N-terminus&doi=10.1021%2Facschemneuro.5b00179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Spontaneous Inward Opening of the Dopamine Transporter Is Triggered by PIP2-Regulated Dynamics of the N-Terminus</span></div><div class="casAuthors">Khelashvili, George; Stanley, Nathaniel; Sahai, Michelle A.; Medina, Jaime; LeVine, Michael V.; Shi, Lei; De Fabritiis, Gianni; Weinstein, Harel</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1825-1837</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the dynamic mechanism of concerted motions in a full-length mol. model of the human dopamine transporter (hDAT), a member of the neurotransmitter/sodium symporter (NSS) family, involved in state-to-state transitions underlying function.  The findings result from an anal. of unbiased atomistic mol. dynamics simulation trajectories (totaling >14 μs) of the hDAT mol. immersed in lipid membrane environments with or without phosphatidylinositol 4,5-biphosphate (PIP2) lipids.  The N-terminal region of hDAT (N-term) is shown to have an essential mechanistic role in correlated rearrangements of specific structural motifs relevant to state-to-state transitions in the hDAT.  The mechanism involves PIP2-mediated electrostatic interactions between the N-term and the intracellular loops of the transporter mol.  Quant. analyses of collective motions in the trajectories reveal that these interactions correlate with the inward-opening dynamics of hDAT and are allosterically coupled to the known functional sites of the transporter.  The obsd. large-scale motions are enabled by specific reconfiguration of the network of ionic interactions at the intracellular end of the protein.  The isomerization to the inward-facing state in hDAT is accompanied by concomitant movements in the extracellular vestibule and results in the release of an Na+ ion from the Na2 site and destabilization of the substrate dopamine in the primary substrate binding S1 site.  The dynamic mechanism emerging from the findings highlights the involvement of the PIP2-regulated interactions between the N-term and the intracellular loop 4 in the functionally relevant conformational transitions that are also similar to those found to underlie state-to-state transitions in the leucine transporter (LeuT), a prototypical bacterial homolog of the NSS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodF_0sIAe1vbVg90H21EOLACvtfcHk0lg4RaU97IxgmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht12nu73P&md5=daf0a8ccf0d1b6a8a5782917a3ceb9d1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facschemneuro.5b00179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschemneuro.5b00179%26sid%3Dliteratum%253Aachs%26aulast%3DKhelashvili%26aufirst%3DG.%26aulast%3DStanley%26aufirst%3DN.%26aulast%3DSahai%26aufirst%3DM.%2BA.%26aulast%3DMedina%26aufirst%3DJ.%26aulast%3DLeVine%26aufirst%3DM.%2BV.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DDeFabritiis%26aufirst%3DG.%26aulast%3DWeinstein%26aufirst%3DH.%26atitle%3DSpontaneous%2520inward%2520opening%2520of%2520the%2520dopamine%2520transporter%2520is%2520triggered%2520by%2520PIP2-regulated%2520dynamics%2520of%2520the%2520N-terminus%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2015%26volume%3D6%26spage%3D1825%26epage%3D1837%26doi%3D10.1021%2Facschemneuro.5b00179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Sase, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aher, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saroja, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sase, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Höger, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bakulev, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitte, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leban, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lubec, G.</span><span> </span><span class="NLM_article-title">A heterocyclic compound CE-103 inhibits dopamine reuptake and modulates dopamine transporter and dopamine D1-D3 containing receptor complexes</span> <span class="citation_source-journal">Neuropharmacology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">186</span><span class="NLM_x">–</span> <span class="NLM_lpage">196</span><span class="refDoi"> DOI: 10.1016/j.neuropharm.2015.07.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1016%2Fj.neuropharm.2015.07.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=26407764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWltLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2016&pages=186-196&author=A.+Saseauthor=Y.+D.+Aherauthor=S.+R.+Sarojaauthor=M.+K.+Ganesanauthor=S.+Saseauthor=M.+Holyauthor=H.+H%C3%B6gerauthor=V.+Bakulevauthor=G.+F.+Eckerauthor=T.+Langerauthor=H.+H.+Sitteauthor=J.+Lebanauthor=G.+Lubec&title=A+heterocyclic+compound+CE-103+inhibits+dopamine+reuptake+and+modulates+dopamine+transporter+and+dopamine+D1-D3+containing+receptor+complexes&doi=10.1016%2Fj.neuropharm.2015.07.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A heterocyclic compound CE-103 inhibits dopamine reuptake and modulates dopamine transporter and dopamine D1-D3 containing receptor complexes</span></div><div class="casAuthors">Sase, Ajinkya; Aher, Yogesh D.; Saroja, Sivaprakasam R.; Ganesan, Minu Karthika; Sase, Sunetra; Holy, Marion; Hoeger, Harald; Bakulev, Vasiliy; Ecker, Gerhard F.; Langer, Thierry; Sitte, Harald H.; Leban, Johann; Lubec, Gert</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">186-196</span>CODEN:
                <span class="NLM_cas:coden">NEPHBW</span>;
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of compds. have been reported to enhance memory via the DA system and herein a heterocyclic compd. was tested for working memory (WM) enhancement.  2-((Benzhydrylsulfinyl)methyl)thiazole (CE-103) was synthesized in a six-step synthesis.  Binding of CE-103 to the dopamine (DAT), serotonin (SERT) and norepinephrine (NET) transporters and dopamine reuptake inhibition was tested as well as blood brain permeation and a screen for GPCR targets. 60 male Sprague Dawley rats were divided into six groups: CE-103 treated 1-10 mg/kg body wt., trained (TDI) and yoked (YDI) and vehicle treated, trained (TVI) and yoked (YVI) rats.  Daily single i.p. injections for a period of 10 days were administered and rats were tested in a radial arm maze (RAM).  Hippocampi were taken 6 h following the last day of training and complexes contg. the unphosphorylated or phosphorylated dopamine transporter (DAT) and complexes contg. the D1-3 dopamine receptor subunits were detd.  CE-103 was binding to the DAT but insignificantly to SERT or NET and dopamine reuptake was blocked specifically (IC50 = 14.73 μM).  From day eight the compd. was decreasing WM errors in the RAM significantly at both doses tested as compared to the vehicle controls.  In the trained CE-103-treated group levels of the complex contg. the phosphorylated dopamine transporter (pDAT) as well as D1R were decreased while levels of complexes contg. D2R and D3R were significantly increased.  CE-103 was shown to enhance spatial WM and DA reuptake inhibition with subsequent modulation of D1-3 receptors is proposed as a possible mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGok9mWVCsTDErVg90H21EOLACvtfcHk0lg4RaU97IxgmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWltLvL&md5=294205b7102eaf775a56b012c1c1434c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2015.07.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2015.07.039%26sid%3Dliteratum%253Aachs%26aulast%3DSase%26aufirst%3DA.%26aulast%3DAher%26aufirst%3DY.%2BD.%26aulast%3DSaroja%26aufirst%3DS.%2BR.%26aulast%3DGanesan%26aufirst%3DM.%2BK.%26aulast%3DSase%26aufirst%3DS.%26aulast%3DHoly%26aufirst%3DM.%26aulast%3DH%25C3%25B6ger%26aufirst%3DH.%26aulast%3DBakulev%26aufirst%3DV.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DSitte%26aufirst%3DH.%2BH.%26aulast%3DLeban%26aufirst%3DJ.%26aulast%3DLubec%26aufirst%3DG.%26atitle%3DA%2520heterocyclic%2520compound%2520CE-103%2520inhibits%2520dopamine%2520reuptake%2520and%2520modulates%2520dopamine%2520transporter%2520and%2520dopamine%2520D1-D3%2520containing%2520receptor%2520complexes%26jtitle%3DNeuropharmacology%26date%3D2016%26volume%3D102%26spage%3D186%26epage%3D196%26doi%3D10.1016%2Fj.neuropharm.2015.07.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span>Bruker SAINT v7.68A<span class="NLM_publisher-name">Bruker AXS</span>,<span class="NLM_x"> </span><span class="NLM_year">2005–2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bruker+SAINT+v7.68ABruker+AXS%2C+2005%E2%80%932016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DBruker%2520SAINT%2520v7.68A%26pub%3DBruker%2520AXS%26date%3D2005%25E2%2580%25932016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Sheldrick, G. M.</span>SADABS; <span class="NLM_publisher-name">University of Göttingen</span>: <span class="NLM_publisher-loc">Göttingen, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">1996</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=G.+M.+Sheldrick&title=SADABS"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26jtitle%3DSADABS%26pub%3DUniversity%2520of%2520G%25C3%25B6ttingen%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Dolomanov, O. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourhis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gildea, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, J. A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puschmann, H.</span><span> </span><span class="NLM_article-title">OLEX2</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">341</span><span class="refDoi"> DOI: 10.1107/S0021889808042726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1107%2FS0021889808042726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsFSnsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2009&pages=339-341&author=O.+V.+Dolomanovauthor=L.+J.+Bourhisauthor=R.+J.+Gildeaauthor=J.+A.+K.+Howardauthor=H.+Puschmann&title=OLEX2&doi=10.1107%2FS0021889808042726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">OLEX2: a complete structure solution, refinement and analysis program</span></div><div class="casAuthors">Dolomanov, Oleg V.; Bourhis, Luc J.; Gildea, Richard J.; Howard, Judith A. K.; Puschmann, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">339-341</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">New software, OLEX2, was developed for the detn., visualization and anal. of mol. crystal structures.  The software has a portable mouse-driven workflow-oriented and fully comprehensive graphical user interface for structure soln., refinement and report generation, as well as novel tools for structure anal.  OLEX2 seamlessly links all aspects of the structure soln., refinement and publication process and presents them in a single workflow-driven package, with the ultimate goal of producing an application which will be useful to both chemists and crystallographers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIYSUbyzIgxbVg90H21EOLACvtfcHk0lg4RaU97IxgmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsFSnsbg%253D&md5=7a2f9697b13b8e76de0e7dc5074ab229</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0021889808042726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889808042726%26sid%3Dliteratum%253Aachs%26aulast%3DDolomanov%26aufirst%3DO.%2BV.%26aulast%3DBourhis%26aufirst%3DL.%2BJ.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DHoward%26aufirst%3DJ.%2BA.%2BK.%26aulast%3DPuschmann%26aufirst%3DH.%26atitle%3DOLEX2%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2009%26volume%3D42%26spage%3D339%26epage%3D341%26doi%3D10.1107%2FS0021889808042726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Huebschle, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheldrick, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dittrich, B.</span><span> </span><span class="NLM_article-title">ShelXle: a Qt graphical user interface for SHELXL</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1281</span><span class="NLM_x">–</span> <span class="NLM_lpage">1284</span><span class="refDoi"> DOI: 10.1107/S0021889811043202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1107%2FS0021889811043202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=22477785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFSisrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2011&pages=1281-1284&author=C.+B.+Huebschleauthor=G.+M.+Sheldrickauthor=B.+Dittrich&title=ShelXle%3A+a+Qt+graphical+user+interface+for+SHELXL&doi=10.1107%2FS0021889811043202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">ShelXle: a Qt graphical user interface for SHELXL</span></div><div class="casAuthors">Huebschle, Christian B.; Sheldrick, George M.; Dittrich, Birger</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1281-1284</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">ShelXle is a user-friendly graphical user interface for SHELXL.  It combines an editor with syntax highlighting for SHELXL-assocd. files with an interactive graphical display for visualization of a three-dimensional structure.  ShelXle is a graphical user interface for SHELXL [Sheldrick, G. M. (2008). Acta Cryst. A64, 112-122], currently the most widely used program for small-mol. structure refinement.  It combines an editor with syntax highlighting for the SHELXL-assocd. (input) and (output) files with an interactive graphical display for visualization of a three-dimensional structure including the electron d. (Fo) and difference d. (Fo-Fc) maps.  Special features of ShelXle include intuitive atom (re-)naming, a strongly coupled editor, structure visualization in various mono and stereo modes, and a novel way of displaying disorder extending over special positions.  ShelXle is completely compatible with all features of SHELXL and is written entirely in C++ using the Qt4 and FFTW libraries.  It is available at no cost for Windows, Linux and Mac-OS X and as source code.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow_d-mylopt7Vg90H21EOLACvtfcHk0lh7QMVNaqEv3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFSisrrF&md5=b9c11983c90116eaf0720fcdbf061ff3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0021889811043202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889811043202%26sid%3Dliteratum%253Aachs%26aulast%3DHuebschle%26aufirst%3DC.%2BB.%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26aulast%3DDittrich%26aufirst%3DB.%26atitle%3DShelXle%253A%2520a%2520Qt%2520graphical%2520user%2520interface%2520for%2520SHELXL%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2011%26volume%3D44%26spage%3D1281%26epage%3D1284%26doi%3D10.1107%2FS0021889811043202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Sheldrick, G. M.</span> <span class="citation_source-book">SHELXS</span>; <span class="NLM_publisher-name">University of Göttingen</span>: <span class="NLM_publisher-loc">Göttingen, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">1996</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=G.+M.+Sheldrick&title=SHELXS"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26btitle%3DSHELXS%26pub%3DUniversity%2520of%2520G%25C3%25B6ttingen%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Sheldrick, G. M.</span> <span class="citation_source-book">SHELXL</span>; <span class="NLM_publisher-name">University of Göttingen</span>: <span class="NLM_publisher-loc">Göttingen, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">1996</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=G.+M.+Sheldrick&title=SHELXL"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26btitle%3DSHELXL%26pub%3DUniversity%2520of%2520G%25C3%25B6ttingen%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Spek, A. L.</span><span> </span><span class="NLM_article-title">Structure validation in chemical crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">148</span><span class="NLM_x">–</span> <span class="NLM_lpage">155</span><span class="refDoi"> DOI: 10.1107/S090744490804362X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1107%2FS090744490804362X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19171970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=148-155&author=A.+L.+Spek&title=Structure+validation+in+chemical+crystallography&doi=10.1107%2FS090744490804362X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure validation in chemical crystallography</span></div><div class="casAuthors">Spek, Anthony L.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">148-155</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Automated structure validation was introduced in chem. crystallog. about 12 years ago as a tool to assist practitioners with the exponential growth in crystal structure analyses.  Validation has since evolved into an easy-to-use checkCIF/PLATON web-based IUCr service.  The result of a crystal structure detn. has to be supplied as a CIF-formatted computer-readable file.  The checking software tests the data in the CIF for completeness, quality, and consistency.  In addn., the reported structure is checked for incomplete anal., errors in the anal., and relevant issues to be verified.  A validation report is generated in the form of a list of ALERTS on the issues to be cor., checked, or commented on.  Structure validation has largely eliminated obvious problems with structure reports published in IUCr journals, such as refinement in a space group of too low symmetry.  This paper reports on the current status of structure validation and possible future extensions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8nVxgyQYutbVg90H21EOLACvtfcHk0lh7QMVNaqEv3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWhtL8%253D&md5=1cc05cd996f846a98b55ea09289af6a9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS090744490804362X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS090744490804362X%26sid%3Dliteratum%253Aachs%26aulast%3DSpek%26aufirst%3DA.%2BL.%26atitle%3DStructure%2520validation%2520in%2520chemical%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D148%26epage%3D155%26doi%3D10.1107%2FS090744490804362X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Sucic, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dallinger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zdrazil, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissensteiner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kudlacek, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cha, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gether, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freissmuth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitte, H. H.</span><span> </span><span class="NLM_article-title">The N-terminus of monoamine transporters is a lever required for the action of amphetamines</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">10924</span><span class="NLM_x">–</span> <span class="NLM_lpage">10938</span><span class="refDoi"> DOI: 10.1074/jbc.M109.083154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1074%2Fjbc.M109.083154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=20118234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFaisL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=10924-10938&author=S.+Sucicauthor=S.+Dallingerauthor=B.+Zdrazilauthor=R.+Weissensteinerauthor=T.+N.+Jorgensenauthor=M.+Holyauthor=O.+Kudlacekauthor=S.+Seidelauthor=J.+H.+Chaauthor=U.+Getherauthor=A.+H.+Newmanauthor=G.+F.+Eckerauthor=M.+Freissmuthauthor=H.+H.+Sitte&title=The+N-terminus+of+monoamine+transporters+is+a+lever+required+for+the+action+of+amphetamines&doi=10.1074%2Fjbc.M109.083154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The N Terminus of Monoamine Transporters Is a Lever Required for the Action of Amphetamines</span></div><div class="casAuthors">Sucic, Sonja; Dallinger, Stefan; Zdrazil, Barbara; Weissensteiner, Rene; Jorgensen, Trine N.; Holy, Marion; Kudlacek, Oliver; Seidel, Stefan; Cha, Joo Hwan; Gether, Ulrik; Newman, Amy H.; Ecker, Gerhard F.; Freissmuth, Michael; Sitte, Harald H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">10924-10938</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The serotonin transporter (SERT) terminates neurotransmission by removing serotonin from the synaptic cleft.  In addn., it is the site of action of antidepressants (which block the transporter) and of amphetamines (which induce substrate efflux).  We explored the functional importance of the N terminus in mediating the action of amphetamines by focusing initially on the highly conserved threonine residue at position 81, a candidate site for phosphorylation by protein kinase C.  Mol. dynamics simulations of the wild type SERT, compared with its mutations SERTT81A and SERTT81D, suggested structural changes in the inner vestibule indicative of an opening of the inner vestibule.  Predictions from this model (e.g. the preferential accumulation of SERTT81A in the inward conformation, its reduced turnover no., and a larger distance between its N and C termini) were verified.  Most importantly, SERTT81A (and the homologous mutations in noradrenaline and dopamine) failed to support amphetamine-induced efflux, and this was not remedied by aspartate at this position.  Amphetamine-induced currents through SERTT81A were comparable with those through the wild type transporter.  Both abundant Na+ entry and accumulation of SERTT81A in the inward facing conformation ought to favor amphetamine-induced efflux.  Thus, we surmised that the N terminus must play a direct role in driving the transporter into a state that supports amphetamine-induced efflux.  This hypothesis was verified by truncating the first 64 amino acids and by tethering the N terminus to an addnl. transmembrane helix.  Either modification abolished amphetamine-induced efflux.  We therefore conclude that the N terminus of monoamine transporters acts as a lever that sustains reverse transport.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBoS2MkOsfgLVg90H21EOLACvtfcHk0lh7QMVNaqEv3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFaisL4%253D&md5=4e93e69edd41b5eab9a8b3d6502a6b0d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.083154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.083154%26sid%3Dliteratum%253Aachs%26aulast%3DSucic%26aufirst%3DS.%26aulast%3DDallinger%26aufirst%3DS.%26aulast%3DZdrazil%26aufirst%3DB.%26aulast%3DWeissensteiner%26aufirst%3DR.%26aulast%3DJorgensen%26aufirst%3DT.%2BN.%26aulast%3DHoly%26aufirst%3DM.%26aulast%3DKudlacek%26aufirst%3DO.%26aulast%3DSeidel%26aufirst%3DS.%26aulast%3DCha%26aufirst%3DJ.%2BH.%26aulast%3DGether%26aufirst%3DU.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DFreissmuth%26aufirst%3DM.%26aulast%3DSitte%26aufirst%3DH.%2BH.%26atitle%3DThe%2520N-terminus%2520of%2520monoamine%2520transporters%2520is%2520a%2520lever%2520required%2520for%2520the%2520action%2520of%2520amphetamines%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D10924%26epage%3D10938%26doi%3D10.1074%2Fjbc.M109.083154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goehring, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penmatsa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ressler, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gouaux, E.</span><span> </span><span class="NLM_article-title">Structural basis for action by diverse antidepressants on biogenic amine transporters</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">503</span><span class="NLM_x">, </span> <span class="NLM_fpage">141</span><span class="NLM_x">–</span> <span class="NLM_lpage">145</span><span class="refDoi"> DOI: 10.1038/nature12648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1038%2Fnature12648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=24121440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSnsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=141-145&author=H.+Wangauthor=A.+Goehringauthor=K.+H.+Wangauthor=A.+Penmatsaauthor=R.+Resslerauthor=E.+Gouaux&title=Structural+basis+for+action+by+diverse+antidepressants+on+biogenic+amine+transporters&doi=10.1038%2Fnature12648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for action by diverse antidepressants on biogenic amine transporters</span></div><div class="casAuthors">Wang, Hui; Goehring, April; Wang, Kevin H.; Penmatsa, Aravind; Ressler, Ryan; Gouaux, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7474</span>),
    <span class="NLM_cas:pages">141-145</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The biogenic amine transporters (BATs) regulate endogenous neurotransmitter concns. and are targets for a broad range of therapeutic agents including selective serotonin reuptake inhibitors (SSRIs), serotonin-noradrenaline reuptake inhibitors (SNRIs) and tricyclic antidepressants (TCAs).  Because eukaryotic BATs are recalcitrant to crystallog. anal., the authors' understanding of the mechanism of these inhibitors and antidepressants is limited.  LeuT is a bacterial homolog of BATs and has proven to be a valuable paradigm for understanding relationships between their structure and function.  However, because only approx. 25% of the amino acid sequence of LeuT is in common with that of BATs, and as LeuT is a promiscuous amino acid transporter, it does not recapitulate the pharmacol. properties of BATs.  Indeed, SSRIs and TCAs bind in the extracellular vestibule of LeuT and act as non-competitive inhibitors of transport.  By contrast, multiple studies demonstrate that both TCAs and SSRIs are competitive inhibitors for eukaryotic BATs and bind to the primary binding pocket.  Here the authors engineered LeuT to harbor human BAT-like pharmacol. by mutating key residues around the primary binding pocket.  The final LeuBAT mutant binds the SSRI sertraline with a binding const. of 18 nM and displays high-affinity binding to a range of SSRIs, SNRIs and a TCA.  The authors detd. 12 crystal structures of LeuBAT in complex with four classes of antidepressants.  The chem. diverse inhibitors have a remarkably similar mode of binding in which they straddle transmembrane helix (TM) 3, wedge between TM3/TM8 and TM1/TM6, and lock the transporter in a sodium- and chloride-bound outward-facing open conformation.  Together, these studies define common and simple principles for the action of SSRIs, SNRIs and TCAs on BATs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8AMnfh2-Af7Vg90H21EOLACvtfcHk0liRQgisNXv6xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSnsLrE&md5=26cc6cf173fdf20ec20d928d02f05193</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnature12648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12648%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DGoehring%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DK.%2BH.%26aulast%3DPenmatsa%26aufirst%3DA.%26aulast%3DRessler%26aufirst%3DR.%26aulast%3DGouaux%26aufirst%3DE.%26atitle%3DStructural%2520basis%2520for%2520action%2520by%2520diverse%2520antidepressants%2520on%2520biogenic%2520amine%2520transporters%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D141%26epage%3D145%26doi%3D10.1038%2Fnature12648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Trott, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">461</span><span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+Improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0liRQgisNXv6xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520Improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Morris, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huey, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindstrom, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belew, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">Software news and updates AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">2785</span><span class="NLM_x">–</span> <span class="NLM_lpage">2791</span><span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=Software+news+and+updates+AutoDock4+and+AutoDockTools4%3A+Automated+docking+with+selective+receptor+flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0liRQgisNXv6xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DSoftware%2520news%2520and%2520updates%2520AutoDock4%2520and%2520AutoDockTools4%253A%2520Automated%2520docking%2520with%2520selective%2520receptor%2520flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Lomize, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pogozheva, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosberg, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lomize, A. L.</span><span> </span><span class="NLM_article-title">OPM database and PPM web server: Resources for positioning of proteins in membranes</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x"> (</span><span class="NLM_issue">D1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D370</span><span class="NLM_x">–</span> <span class="NLM_lpage">D376</span><span class="refDoi"> DOI: 10.1093/nar/gkr703</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1093%2Fnar%2Fgkr703" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=21890895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12hurzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D370-D376&issue=D1&author=M.+A.+Lomizeauthor=I.+D.+Pogozhevaauthor=H.+Jooauthor=H.+I.+Mosbergauthor=A.+L.+Lomize&title=OPM+database+and+PPM+web+server%3A+Resources+for+positioning+of+proteins+in+membranes&doi=10.1093%2Fnar%2Fgkr703"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">OPM database and PPM web server: resources for positioning of proteins in membranes</span></div><div class="casAuthors">Lomize, Mikhail A.; Pogozheva, Irina D.; Joo, Hyeon; Mosberg, Henry I.; Lomize, Andrei L.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D370-D376</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Orientations of Proteins in Membranes (OPM) database is a curated web resource that provides spatial positions of membrane-bound peptides and proteins of known three-dimensional structure in the lipid bilayer, together with their structural classification, topol. and intracellular localization.  OPM currently contains more than 1200 transmembrane and peripheral proteins and peptides from approx. 350 organisms that represent approx. 3800 Protein Data Bank entries.  Proteins are classified into classes, superfamilies and families and assigned to 21 distinct membrane types.  Spatial positions of proteins with respect to the lipid bilayer are optimized by the PPM 2.0 method that accounts for the hydrophobic, hydrogen bonding and electrostatic interactions of the proteins with the anisotropic water-lipid environment described by the dielec. const. and hydrogen-bonding profiles.  The OPM database is freely accessible at http://opm.phar.umich.edu.  Data can be sorted, searched or retrieved using the hierarchical classification, source organism, localization in different types of membranes.  The database offers downloadable coordinates of proteins and peptides with membrane boundaries.  A gallery of protein images and several visualization tools are provided.  The database is supplemented by the PPM server (http://opm.phar.umich.edu/server.php) which can be used for calcg. spatial positions in membranes of newly detd. proteins structures or theor. models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo76Y6X6BHLRbVg90H21EOLACvtfcHk0ljQ3vMj89ySIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12hurzJ&md5=b931126c56d227ab0eb6c5762c1dae6b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr703%26sid%3Dliteratum%253Aachs%26aulast%3DLomize%26aufirst%3DM.%2BA.%26aulast%3DPogozheva%26aufirst%3DI.%2BD.%26aulast%3DJoo%26aufirst%3DH.%26aulast%3DMosberg%26aufirst%3DH.%2BI.%26aulast%3DLomize%26aufirst%3DA.%2BL.%26atitle%3DOPM%2520database%2520and%2520PPM%2520web%2520server%253A%2520Resources%2520for%2520positioning%2520of%2520proteins%2520in%2520membranes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26issue%3DD1%26spage%3DD370%26epage%3DD376%26doi%3D10.1093%2Fnar%2Fgkr703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Jo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iyer, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Im, W.</span><span> </span><span class="NLM_article-title">CHARMM-GUI: A web-based graphical user interface for CHARMM</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1859</span><span class="NLM_x">–</span> <span class="NLM_lpage">1865</span><span class="refDoi"> DOI: 10.1002/jcc.20945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1002%2Fjcc.20945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=18351591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1cXosVKksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=1859-1865&author=S.+Joauthor=T.+Kimauthor=V.+G.+Iyerauthor=W.+Im&title=CHARMM-GUI%3A+A+web-based+graphical+user+interface+for+CHARMM&doi=10.1002%2Fjcc.20945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM-GUI: a web-based graphical user interface for CHARMM</span></div><div class="casAuthors">Jo, Sunhwan; Kim, Taehoon; Iyer, Vidyashankara G.; Im, Wonpil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1859-1865</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">CHARMM is an academic research program used widely for macromol. mechanics and dynamics with versatile anal. and manipulation tools of at. coordinates and dynamics trajectories.  CHARMM-GUI, http://www.charmm-gui.org, has been developed to provide a web-based graphical user interface to generate various input files and mol. systems to facilitate and standardize the usage of common and advanced simulation techniques in CHARMM.  The web environment provides an ideal platform to build and validate a mol. model system in an interactive fashion such that, if a problem is found through visual inspection, one can go back to the previous setup and regenerate the whole system again.  In this article, we describe the currently available functional modules of CHARMM-GUI Input Generator that form a basis for the advanced simulation techniques.  Future directions of the CHARMM-GUI development project are also discussed briefly together with other features in the CHARMM-GUI website, such as Archive and Movie Gallery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7gpU-1N3WOLVg90H21EOLACvtfcHk0ljQ3vMj89ySIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXosVKksbc%253D&md5=112a3dd61d792b040f9f716b32220d7e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20945%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DIyer%26aufirst%3DV.%2BG.%26aulast%3DIm%26aufirst%3DW.%26atitle%3DCHARMM-GUI%253A%2520A%2520web-based%2520graphical%2520user%2520interface%2520for%2520CHARMM%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2008%26volume%3D29%26spage%3D1859%26epage%3D1865%26doi%3D10.1002%2Fjcc.20945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Jo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Im, W.</span><span> </span><span class="NLM_article-title">Automated builder and database of protein/membrane complexes for molecular dynamics simulations</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e880</span><span class="refDoi"> DOI: 10.1371/journal.pone.0000880</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1371%2Fjournal.pone.0000880" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=17849009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A280%3ADC%252BD2srjsFyjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=e880&author=S.+Joauthor=T.+Kimauthor=W.+Im&title=Automated+builder+and+database+of+protein%2Fmembrane+complexes+for+molecular+dynamics+simulations&doi=10.1371%2Fjournal.pone.0000880"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Automated builder and database of protein/membrane complexes for molecular dynamics simulations</span></div><div class="casAuthors">Jo Sunhwan; Kim Taehoon; Im Wonpil</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e880</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Molecular dynamics simulations of membrane proteins have provided deeper insights into their functions and interactions with surrounding environments at the atomic level.  However, compared to solvation of globular proteins, building a realistic protein/membrane complex is still challenging and requires considerable experience with simulation software.  Membrane Builder in the CHARMM-GUI website (http://www.charmm-gui.org) helps users to build such a complex system using a web browser with a graphical user interface.  Through a generalized and automated building process including system size determination as well as generation of lipid bilayer, pore water, bulk water, and ions, a realistic membrane system with virtually any kinds and shapes of membrane proteins can be generated in 5 minutes to 2 hours depending on the system size.  Default values that were elaborated and tested extensively are given in each step to provide reasonable options and starting points for both non-expert and expert users.  The efficacy of Membrane Builder is illustrated by its applications to 12 transmembrane and 3 interfacial membrane proteins, whose fully equilibrated systems with three different types of lipid molecules (DMPC, DPPC, and POPC) and two types of system shapes (rectangular and hexagonal) are freely available on the CHARMM-GUI website.  One of the most significant advantages of using the web environment is that, if a problem is found, users can go back and re-generate the whole system again before quitting the browser.  Therefore, Membrane Builder provides the intuitive and easy way to build and simulate the biologically important membrane system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAsncNNJLqIBfKb6ELK8t0fW6udTcc2eaUwNvmUYb-0rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2srjsFyjsQ%253D%253D&md5=d2384741fc1d18b33cff16b254fc006b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0000880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0000880%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DIm%26aufirst%3DW.%26atitle%3DAutomated%2520builder%2520and%2520database%2520of%2520protein%252Fmembrane%2520complexes%2520for%2520molecular%2520dynamics%2520simulations%26jtitle%3DPLoS%2520One%26date%3D2007%26volume%3D2%26spage%3De880%26doi%3D10.1371%2Fjournal.pone.0000880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Brooks, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrella, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Won, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archontis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartels, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boresch, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caves, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinner, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feig, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodoscek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Im, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuczera, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lazaridis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ovchinnikov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paci, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pastor, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Post, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pu, J. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tidor, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venable, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodcock, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">York, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">CHARMM: The biomolecular simulation program</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1545</span><span class="NLM_x">–</span> <span class="NLM_lpage">1614</span><span class="refDoi"> DOI: 10.1002/jcc.21287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1002%2Fjcc.21287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19444816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms1Ciu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=1545-1614&author=B.+R.+Brooksauthor=C.+L.+Brooksauthor=A.+D.+Mackerellauthor=L.+Nilssonauthor=R.+J.+Petrellaauthor=B.+Rouxauthor=Y.+Wonauthor=G.+Archontisauthor=C.+Bartelsauthor=S.+Boreschauthor=A.+Caflischauthor=L.+Cavesauthor=Q.+Cuiauthor=A.+R.+Dinnerauthor=M.+Feigauthor=S.+Fischerauthor=J.+Gaoauthor=M.+Hodoscekauthor=W.+Imauthor=K.+Kuczeraauthor=T.+Lazaridisauthor=J.+Maauthor=V.+Ovchinnikovauthor=E.+Paciauthor=R.+W.+Pastorauthor=C.+B.+Postauthor=J.+Z.+Puauthor=M.+Schaeferauthor=B.+Tidorauthor=R.+M.+Venableauthor=H.+L.+Woodcockauthor=X.+Wuauthor=W.+Yangauthor=D.+M.+Yorkauthor=M.+Karplus&title=CHARMM%3A+The+biomolecular+simulation+program&doi=10.1002%2Fjcc.21287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM: The biomolecular simulation program</span></div><div class="casAuthors">Brooks, B. R.; Brooks, C. L., III; Mackerell, A. D., Jr.; Nilsson, L.; Petrella, R. J.; Roux, B.; Won, Y.; Archontis, G.; Bartels, C.; Boresch, S.; Caflisch, A.; Caves, L.; Cui, Q.; Dinner, A. R.; Feig, M.; Fischer, S.; Gao, J.; Hodoscek, M.; Im, W.; Kuczera, K.; Lazaridis, T.; Ma, J.; Ovchinnikov, V.; Paci, E.; Pastor, R. W.; Post, C. B.; Pu, J. Z.; Schaefer, M.; Tidor, B.; Venable, R. M.; Woodcock, H. L.; Wu, X.; Yang, W.; York, D. M.; Karplus, M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1545-1614</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  CHARMM (Chem. at HARvard Mol. Mechanics) is a highly versatile and widely used mol. simulation program.  It has been developed over the last three decades with a primary focus on mols. of biol. interest, including proteins, peptides, lipids, nucleic acids, carbohydrates, and small mol. ligands, as they occur in soln., crystals, and membrane environments.  For the study of such systems, the program provides a large suite of computational tools that include numerous conformational and path sampling methods, free energy estimators, mol. minimization, dynamics, and anal. techniques, and model-building capabilities.  The CHARMM program is applicable to problems involving a much broader class of many-particle systems.  Calcns. with CHARMM can be performed using a no. of different energy functions and models, from mixed quantum mech.-mol. mech. force fields, to all-atom classical potential energy functions with explicit solvent and various boundary conditions, to implicit solvent and membrane models.  The program has been ported to numerous platforms in both serial and parallel architectures.  This article provides an overview of the program as it exists today with an emphasis on developments since the publication of the original CHARMM article in 1983.  © 2009 Wiley Periodicals, Inc. J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOXZCtNIqoCbVg90H21EOLACvtfcHk0ljQ3vMj89ySIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms1Ciu70%253D&md5=2c6a2be869362d7131f5aea8411c1552</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21287%26sid%3Dliteratum%253Aachs%26aulast%3DBrooks%26aufirst%3DB.%2BR.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26aulast%3DNilsson%26aufirst%3DL.%26aulast%3DPetrella%26aufirst%3DR.%2BJ.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DWon%26aufirst%3DY.%26aulast%3DArchontis%26aufirst%3DG.%26aulast%3DBartels%26aufirst%3DC.%26aulast%3DBoresch%26aufirst%3DS.%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DCaves%26aufirst%3DL.%26aulast%3DCui%26aufirst%3DQ.%26aulast%3DDinner%26aufirst%3DA.%2BR.%26aulast%3DFeig%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DHodoscek%26aufirst%3DM.%26aulast%3DIm%26aufirst%3DW.%26aulast%3DKuczera%26aufirst%3DK.%26aulast%3DLazaridis%26aufirst%3DT.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DOvchinnikov%26aufirst%3DV.%26aulast%3DPaci%26aufirst%3DE.%26aulast%3DPastor%26aufirst%3DR.%2BW.%26aulast%3DPost%26aufirst%3DC.%2BB.%26aulast%3DPu%26aufirst%3DJ.%2BZ.%26aulast%3DSchaefer%26aufirst%3DM.%26aulast%3DTidor%26aufirst%3DB.%26aulast%3DVenable%26aufirst%3DR.%2BM.%26aulast%3DWoodcock%26aufirst%3DH.%2BL.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DYork%26aufirst%3DD.%2BM.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DCHARMM%253A%2520The%2520biomolecular%2520simulation%2520program%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D1545%26epage%3D1614%26doi%3D10.1002%2Fjcc.21287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Eastman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radmer, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruns, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ku, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beauchamp, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shukla, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tye, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houston, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stich, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">OpenMM 4: A reusable, extensible, hardware independent library for high performance molecular simulation</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">461</span><span class="NLM_x">–</span> <span class="NLM_lpage">469</span><span class="refDoi"> DOI: 10.1021/ct300857j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300857j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFektbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=461-469&author=P.+Eastmanauthor=M.+S.+Friedrichsauthor=J.+D.+Choderaauthor=R.+J.+Radmerauthor=C.+M.+Brunsauthor=J.+P.+Kuauthor=K.+A.+Beauchampauthor=T.+J.+Laneauthor=L.+P.+Wangauthor=D.+Shuklaauthor=T.+Tyeauthor=M.+Houstonauthor=T.+Stichauthor=C.+Kleinauthor=M.+R.+Shirtsauthor=V.+S.+Pande&title=OpenMM+4%3A+A+reusable%2C+extensible%2C+hardware+independent+library+for+high+performance+molecular+simulation&doi=10.1021%2Fct300857j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">OpenMM 4: A Reusable, Extensible, Hardware Independent Library for High Performance Molecular Simulation</span></div><div class="casAuthors">Eastman, Peter; Friedrichs, Mark S.; Chodera, John D.; Radmer, Randall J.; Bruns, Christopher M.; Ku, Joy P.; Beauchamp, Kyle A.; Lane, Thomas J.; Wang, Lee-Ping; Shukla, Diwakar; Tye, Tony; Houston, Mike; Stich, Timo; Klein, Christoph; Shirts, Michael R.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">461-469</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">OpenMM is a software toolkit for performing mol. simulations on a range of high performance computing architectures.  It is based on a layered architecture: the lower layers function as a reusable library that can be invoked by any application, while the upper layers form a complete environment for running mol. simulations.  The library API hides all hardware-specific dependencies and optimizations from the users and developers of simulation programs: they can be run without modification on any hardware on which the API has been implemented.  The current implementations of OpenMM include support for graphics processing units using the OpenCL and CUDA frameworks.  In addn., OpenMM was designed to be extensible, so new hardware architectures can be accommodated and new functionality (e.g., energy terms and integrators) can be easily added.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJZt8HqGLFN7Vg90H21EOLACvtfcHk0liOUkddo1UHdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFektbbP&md5=484edfd2b97123d2e26550778b0edaaf</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fct300857j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300857j%26sid%3Dliteratum%253Aachs%26aulast%3DEastman%26aufirst%3DP.%26aulast%3DFriedrichs%26aufirst%3DM.%2BS.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DRadmer%26aufirst%3DR.%2BJ.%26aulast%3DBruns%26aufirst%3DC.%2BM.%26aulast%3DKu%26aufirst%3DJ.%2BP.%26aulast%3DBeauchamp%26aufirst%3DK.%2BA.%26aulast%3DLane%26aufirst%3DT.%2BJ.%26aulast%3DWang%26aufirst%3DL.%2BP.%26aulast%3DShukla%26aufirst%3DD.%26aulast%3DTye%26aufirst%3DT.%26aulast%3DHouston%26aufirst%3DM.%26aulast%3DStich%26aufirst%3DT.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DOpenMM%25204%253A%2520A%2520reusable%252C%2520extensible%252C%2520hardware%2520independent%2520library%2520for%2520high%2520performance%2520molecular%2520simulation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D461%26epage%3D469%26doi%3D10.1021%2Fct300857j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swails, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeom, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastman, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemkul, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeong, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klauda, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Im, W.</span><span> </span><span class="NLM_article-title">CHARMM-GUI Input Generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">413</span><span class="refDoi"> DOI: 10.1021/acs.jctc.5b00935</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00935" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVWru7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=405-413&author=J.+Leeauthor=Y.+Chengauthor=J.+M.+Swailsauthor=M.+S.+Yeomauthor=P.+K.+Eastmanauthor=J.+A.+Lemkulauthor=S.+Weiauthor=J.+Bucknerauthor=J.+C.+Jeongauthor=Y.+Qiauthor=S.+Joauthor=V.+S.+Pandeauthor=D.+A.+Caseauthor=C.+L.+Brooksauthor=A.+D.+MacKerellauthor=J.+B.+Klaudaauthor=W.+Im&title=CHARMM-GUI+Input+Generator+for+NAMD%2C+GROMACS%2C+AMBER%2C+OpenMM%2C+and+CHARMM%2FOpenMM+simulations+using+the+CHARMM36+additive+force+field&doi=10.1021%2Facs.jctc.5b00935"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM-GUI Input Generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM Simulations Using the CHARMM36 Additive Force Field</span></div><div class="casAuthors">Lee, Jumin; Cheng, Xi; Swails, Jason M.; Yeom, Min Sun; Eastman, Peter K.; Lemkul, Justin A.; Wei, Shuai; Buckner, Joshua; Jeong, Jong Cheol; Qi, Yifei; Jo, Sunhwan; Pande, Vijay S.; Case, David A.; Brooks, Charles L.; MacKerell, Alexander D.; Klauda, Jeffery B.; Im, Wonpil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">405-413</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proper treatment of nonbonded interactions is essential for the accuracy of mol. dynamics (MD) simulations, esp. in studies of lipid bilayers.  The use of the CHARMM36 force field (C36 FF) in different MD simulation programs can result in disagreements with published simulations performed with CHARMM due to differences in the protocols used to treat the long-range and 1-4 nonbonded interactions.  In this study, we systematically test the use of the C36 lipid FF in NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM.  A wide range of Lennard-Jones (LJ) cutoff schemes and integrator algorithms were tested to find the optimal simulation protocol to best match bilayer properties of six lipids with varying acyl chain satn. and head groups.  MD simulations of a 1,2-dipalmitoyl-sn-phosphatidylcholine (DPPC) bilayer were used to obtain the optimal protocol for each program.  MD simulations with all programs were found to reasonably match the DPPC bilayer properties (surface area per lipid, chain order parameters, and area compressibility modulus) obtained using the std. protocol used in CHARMM as well as from expts.  The optimal simulation protocol was then applied to the other five lipid simulations and resulted in excellent agreement between results from most simulation programs as well as with exptl. data.  AMBER compared least favorably with the expected membrane properties, which appears to be due to its use of the hard-truncation in the LJ potential vs. a force-based switching function used to smooth the LJ potential as it approaches the cutoff distance.  The optimal simulation protocol for each program has been implemented in CHARMM-GUI.  This protocol is expected to be applicable to the remainder of the additive C36 FF including the proteins, nucleic acids, carbohydrates, and small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrioKQ5MoVZkrVg90H21EOLACvtfcHk0liOUkddo1UHdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVWru7nI&md5=1e986f2c205aca8fb80442bc9c95b229</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00935&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00935%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DYeom%26aufirst%3DM.%2BS.%26aulast%3DEastman%26aufirst%3DP.%2BK.%26aulast%3DLemkul%26aufirst%3DJ.%2BA.%26aulast%3DWei%26aufirst%3DS.%26aulast%3DBuckner%26aufirst%3DJ.%26aulast%3DJeong%26aufirst%3DJ.%2BC.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DJo%26aufirst%3DS.%26aulast%3DPande%26aufirst%3DV.%2BS.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DKlauda%26aufirst%3DJ.%2BB.%26aulast%3DIm%26aufirst%3DW.%26atitle%3DCHARMM-GUI%2520Input%2520Generator%2520for%2520NAMD%252C%2520GROMACS%252C%2520AMBER%252C%2520OpenMM%252C%2520and%2520CHARMM%252FOpenMM%2520simulations%2520using%2520the%2520CHARMM36%2520additive%2520force%2520field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D405%26epage%3D413%26doi%3D10.1021%2Facs.jctc.5b00935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatcher, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kundu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darian, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guvench, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopes, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorobyov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">671</span><span class="NLM_x">–</span> <span class="NLM_lpage">690</span><span class="refDoi"> DOI: 10.1002/jcc.21367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1002%2Fjcc.21367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19575467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=671-690&author=K.+Vanommeslaegheauthor=E.+Hatcherauthor=C.+Acharyaauthor=S.+Kunduauthor=S.+Zhongauthor=J.+Shimauthor=E.+Darianauthor=O.+Guvenchauthor=P.+Lopesauthor=I.+Vorobyovauthor=A.+D.+MacKerell&title=CHARMM+general+force+field%3A+A+force+field+for+drug-like+molecules+compatible+with+the+CHARMM+all-atom+additive+biological+force+fields&doi=10.1002%2Fjcc.21367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span></div><div class="casAuthors">Vanommeslaeghe, K.; Hatcher, E.; Acharya, C.; Kundu, S.; Zhong, S.; Shim, J.; Darian, E.; Guvench, O.; Lopes, P.; Vorobyov, I.; Mackerell, A. D., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">671-690</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The widely used CHARMM additive all-atom force field includes parameters for proteins, nucleic acids, lipids, and carbohydrates.  In the present article, an extension of the CHARMM force field to drug-like mols. is presented.  The resulting CHARMM General Force Field (CGenFF) covers a wide range of chem. groups present in biomols. and drug-like mols., including a large no. of heterocyclic scaffolds.  The parametrization philosophy behind the force field focuses on quality at the expense of transferability, with the implementation concg. on an extensible force field.  Statistics related to the quality of the parametrization with a focus on exptl. validation are presented.  Addnl., the parametrization procedure, described fully in the present article in the context of the model systems, pyrrolidine, and 3-phenoxymethyl-pyrrolidine will allow users to readily extend the force field to chem. groups that are not explicitly covered in the force field as well as add functional groups to and link together mols. already available in the force field.  CGenFF thus makes it possible to perform "all-CHARMM" simulations on drug-target interactions thereby extending the utility of CHARMM force fields to medicinally relevant systems. © 2009 Wiley Periodicals, Inc.J Comput Chem, 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCuh0CLdmxN7Vg90H21EOLACvtfcHk0ljBrlWDK1VCxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D&md5=26212e0e4f73bded0c89d4b411cd3833</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21367%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DAcharya%26aufirst%3DC.%26aulast%3DKundu%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DDarian%26aufirst%3DE.%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DLopes%26aufirst%3DP.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DCHARMM%2520general%2520force%2520field%253A%2520A%2520force%2520field%2520for%2520drug-like%2520molecules%2520compatible%2520with%2520the%2520CHARMM%2520all-atom%2520additive%2520biological%2520force%2520fields%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D671%26epage%3D690%26doi%3D10.1002%2Fjcc.21367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">MacKerell, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bashford, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunbrack, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evanseck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph-McCarthy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuchnir, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuczera, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, F. T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michnick, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prodhom, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reiher, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlenkrich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stote, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Straub, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiorkiewicz-Kuczera, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">All-atom empirical potential for molecular modeling and dynamics studies of proteins</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">3586</span><span class="NLM_x">–</span> <span class="NLM_lpage">3616</span><span class="refDoi"> DOI: 10.1021/jp973084f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp973084f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADyaK1cXivVOlsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1998&pages=3586-3616&author=A.+MacKerellauthor=D.+Bashfordauthor=M.+Bellottauthor=R.+Dunbrackauthor=J.+Evanseckauthor=M.+Fieldauthor=S.+Fischerauthor=J.+Gaoauthor=H.+Guoauthor=S.+Haauthor=D.+Joseph-McCarthyauthor=L.+Kuchnirauthor=K.+Kuczeraauthor=F.+T.+K.+Lauauthor=C.+Mattosauthor=S.+Michnickauthor=T.+Ngoauthor=D.+T.+Nguyenauthor=B.+Prodhomauthor=W.+E.+Reiherauthor=B.+Rouxauthor=M.+Schlenkrichauthor=J.+C.+Smithauthor=R.+Stoteauthor=J.+Straubauthor=M.+Watanabeauthor=J.+Wiorkiewicz-Kuczeraauthor=D.+Yinauthor=M.+Karplus&title=All-atom+empirical+potential+for+molecular+modeling+and+dynamics+studies+of+proteins&doi=10.1021%2Fjp973084f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">All-Atom Empirical Potential for Molecular Modeling and Dynamics Studies of Proteins</span></div><div class="casAuthors">MacKerell, A. D., Jr.; Bashford, D.; Bellott, M.; Dunbrack, R. L.; Evanseck, J. D.; Field, M. J.; Fischer, S.; Gao, J.; Guo, H.; Ha, S.; Joseph-McCarthy, D.; Kuchnir, L.; Kuczera, K.; Lau, F. T. K.; Mattos, C.; Michnick, S.; Ngo, T.; Nguyen, D. T.; Prodhom, B.; Reiher, W. E., III; Roux, B.; Schlenkrich, M.; Smith, J. C.; Stote, R.; Straub, J.; Watanabe, M.; Wiorkiewicz-Kuczera, J.; Yin, D.; Karplus, M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3586-3616</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New protein parameters are reported for the all-atom empirical energy function in the CHARMM program.  The parameter evaluation was based on a self-consistent approach designed to achieve a balance between the internal (bonding) and interaction (nonbonding) terms of the force field and among the solvent-solvent, solvent-solute, and solute-solute interactions.  Optimization of the internal parameters used exptl. gas-phase geometries, vibrational spectra, and torsional energy surfaces supplemented with ab initio results.  The peptide backbone bonding parameters were optimized with respect to data for N-methylacetamide and the alanine dipeptide.  The interaction parameters, particularly the at. charges, were detd. by fitting ab initio interaction energies and geometries of complexes between water and model compds. that represented the backbone and the various side chains.  In addn., dipole moments, exptl. heats and free energies of vaporization, solvation and sublimation, mol. vols., and crystal pressures and structures were used in the optimization.  The resulting protein parameters were tested by applying them to noncyclic tripeptide crystals, cyclic peptide crystals, and the proteins crambin, bovine pancreatic trypsin inhibitor, and carbonmonoxy myoglobin in vacuo and in a crystal.  A detailed anal. of the relationship between the alanine dipeptide potential energy surface and calcd. protein φ, χ angles was made and used in optimizing the peptide group torsional parameters.  The results demonstrate that use of ab initio structural and energetic data by themselves are not sufficient to obtain an adequate backbone representation for peptides and proteins in soln. and in crystals.  Extensive comparisons between mol. dynamics simulation and exptl. data for polypeptides and proteins were performed for both structural and dynamic properties.  Calcd. data from energy minimization and dynamics simulations for crystals demonstrate that the latter are needed to obtain meaningful comparisons with exptl. crystal structures.  The presented parameters, in combination with the previously published CHARMM all-atom parameters for nucleic acids and lipids, provide a consistent set for condensed-phase simulations of a wide variety of mols. of biol. interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor1-3zB5qPwrVg90H21EOLACvtfcHk0ljBrlWDK1VCxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXivVOlsb4%253D&md5=ebb5100dafd0daeee60ca2fa66c1324a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjp973084f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp973084f%26sid%3Dliteratum%253Aachs%26aulast%3DMacKerell%26aufirst%3DA.%26aulast%3DBashford%26aufirst%3DD.%26aulast%3DBellott%26aufirst%3DM.%26aulast%3DDunbrack%26aufirst%3DR.%26aulast%3DEvanseck%26aufirst%3DJ.%26aulast%3DField%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DHa%26aufirst%3DS.%26aulast%3DJoseph-McCarthy%26aufirst%3DD.%26aulast%3DKuchnir%26aufirst%3DL.%26aulast%3DKuczera%26aufirst%3DK.%26aulast%3DLau%26aufirst%3DF.%2BT.%2BK.%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DMichnick%26aufirst%3DS.%26aulast%3DNgo%26aufirst%3DT.%26aulast%3DNguyen%26aufirst%3DD.%2BT.%26aulast%3DProdhom%26aufirst%3DB.%26aulast%3DReiher%26aufirst%3DW.%2BE.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DSchlenkrich%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26aulast%3DStote%26aufirst%3DR.%26aulast%3DStraub%26aufirst%3DJ.%26aulast%3DWatanabe%26aufirst%3DM.%26aulast%3DWiorkiewicz-Kuczera%26aufirst%3DJ.%26aulast%3DYin%26aufirst%3DD.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DAll-atom%2520empirical%2520potential%2520for%2520molecular%2520modeling%2520and%2520dynamics%2520studies%2520of%2520proteins%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1998%26volume%3D102%26spage%3D3586%26epage%3D3616%26doi%3D10.1021%2Fjp973084f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Essmann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perera, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berkowitz, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, L. G.</span><span> </span><span class="NLM_article-title">A smooth particle mesh Ewald method</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">8577</span><span class="NLM_x">–</span> <span class="NLM_lpage">8593</span><span class="refDoi"> DOI: 10.1063/1.470117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1063%2F1.470117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1995&pages=8577-8593&author=U.+Essmannauthor=L.+Pereraauthor=M.+L.+Berkowitzauthor=T.+Dardenauthor=H.+Leeauthor=L.+G.+Pedersen&title=A+smooth+particle+mesh+Ewald+method&doi=10.1063%2F1.470117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth particle mesh Ewald method</span></div><div class="casAuthors">Essmann, Ulrich; Perera, Lalith; Berkowitz, Max L.; Darden, Tom; Lee, Hsing; Pedersen, Lee G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8577-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The previously developed particle mesh Ewald method is reformulated in terms of efficient B-spline interpolation of the structure factors.  This reformulation allows a natural extension of the method to potentials of the form 1/rp with p ≥ 1.  Furthermore, efficient calcn. of the virial tensor follows.  Use of B-splines in the place of Lagrange interpolation leads to analytic gradients as well as a significant improvement in the accuracy.  The authors demonstrate that arbitrary accuracy can be achieved, independent of system size N, at a cost that scales as N log(N).  For biomol. systems with many thousands of atoms and this method permits the use of Ewald summation at a computational cost comparable to that of a simple truncation method of 10 Å or less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Bp1XlSY32LVg90H21EOLACvtfcHk0ljnXquZWNVY2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D&md5=092a679dd3bee08da28df41e302383a7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1063%2F1.470117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.470117%26sid%3Dliteratum%253Aachs%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DBerkowitz%26aufirst%3DM.%2BL.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPedersen%26aufirst%3DL.%2BG.%26atitle%3DA%2520smooth%2520particle%2520mesh%2520Ewald%2520method%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D103%26spage%3D8577%26epage%3D8593%26doi%3D10.1063%2F1.470117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Michaud-Agrawal, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denning, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolf, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beckstein, O.</span><span> </span><span class="NLM_article-title">MDAnalysis: A toolkit for the analysis of molecular dynamics simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">2319</span><span class="NLM_x">–</span> <span class="NLM_lpage">2327</span><span class="refDoi"> DOI: 10.1002/jcc.21787</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1002%2Fjcc.21787" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=21500218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvFalsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=2319-2327&author=N.+Michaud-Agrawalauthor=E.+J.+Denningauthor=T.+B.+Woolfauthor=O.+Beckstein&title=MDAnalysis%3A+A+toolkit+for+the+analysis+of+molecular+dynamics+simulations&doi=10.1002%2Fjcc.21787"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">MDAnalysis: A toolkit for the analysis of molecular dynamics simulations</span></div><div class="casAuthors">Michaud-Agrawal, Naveen; Denning, Elizabeth J.; Woolf, Thomas B.; Beckstein, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2319-2327</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">MDAnal. is an object-oriented library for structural and temporal anal. of mol. dynamics (MD) simulation trajectories and individual protein structures.  It is written in the Python language with some performance-crit. code in C.  It uses the powerful NumPy package to expose trajectory data as fast and efficient NumPy arrays.  It has been tested on systems of millions of particles.  Many common file formats of simulation packages including CHARMM, Gromacs, Amber, and NAMD and the Protein Data Bank format can be read and written.  Atoms can be selected with a syntax similar to CHARMM's powerful selection commands.  MDAnal. enables both novice and experienced programmers to rapidly write their own anal. tools and access data stored in trajectories in an easily accessible manner that facilitates interactive explorative anal.  MDAnal. has been tested on and works for most Unix-based platforms such as Linux and Mac OS X.  It is freely available under the GNU General Public License from http://mdanal.googlecode.com.  © 2011 Wiley Periodicals, Inc. J Comput Chem 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgUxkDoU25-bVg90H21EOLACvtfcHk0ljnXquZWNVY2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvFalsr8%253D&md5=d567042c65cfdc1c81336a29137654bf</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21787%26sid%3Dliteratum%253Aachs%26aulast%3DMichaud-Agrawal%26aufirst%3DN.%26aulast%3DDenning%26aufirst%3DE.%2BJ.%26aulast%3DWoolf%26aufirst%3DT.%2BB.%26aulast%3DBeckstein%26aufirst%3DO.%26atitle%3DMDAnalysis%253A%2520A%2520toolkit%2520for%2520the%2520analysis%2520of%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2011%26volume%3D32%26spage%3D2319%26epage%3D2327%26doi%3D10.1002%2Fjcc.21787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Hunter, J. D.</span><span> </span><span class="NLM_article-title">Matplotlib: A 2D graphics environment</span> <span class="citation_source-journal">Comput. Sci. Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">90</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span><span class="refDoi"> DOI: 10.1109/MCSE.2007.55</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1109%2FMCSE.2007.55" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=90-95&author=J.+D.+Hunter&title=Matplotlib%3A+A+2D+graphics+environment&doi=10.1109%2FMCSE.2007.55"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1109%2FMCSE.2007.55&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FMCSE.2007.55%26sid%3Dliteratum%253Aachs%26aulast%3DHunter%26aufirst%3DJ.%2BD.%26atitle%3DMatplotlib%253A%2520A%25202D%2520graphics%2520environment%26jtitle%3DComput.%2520Sci.%2520Eng.%26date%3D2007%26volume%3D9%26spage%3D90%26epage%3D95%26doi%3D10.1109%2FMCSE.2007.55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Weitzdoerfer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstl, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeger, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dreher, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lubec, G.</span><span> </span><span class="NLM_article-title">Long-term influence of perinatal asphyxia on the social behavior in aging rats</span> <span class="citation_source-journal">Gerontology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">–</span> <span class="NLM_lpage">205</span><span class="refDoi"> DOI: 10.1159/000078348</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1159%2F000078348" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2004&pages=200-205&author=R.+Weitzdoerferauthor=N.+Gerstlauthor=D.+Pollakauthor=H.+Hoegerauthor=W.+Dreherauthor=G.+Lubec&title=Long-term+influence+of+perinatal+asphyxia+on+the+social+behavior+in+aging+rats&doi=10.1159%2F000078348"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1159%2F000078348&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000078348%26sid%3Dliteratum%253Aachs%26aulast%3DWeitzdoerfer%26aufirst%3DR.%26aulast%3DGerstl%26aufirst%3DN.%26aulast%3DPollak%26aufirst%3DD.%26aulast%3DHoeger%26aufirst%3DH.%26aulast%3DDreher%26aufirst%3DW.%26aulast%3DLubec%26aufirst%3DG.%26atitle%3DLong-term%2520influence%2520of%2520perinatal%2520asphyxia%2520on%2520the%2520social%2520behavior%2520in%2520aging%2520rats%26jtitle%3DGerontology%26date%3D2004%26volume%3D50%26spage%3D200%26epage%3D205%26doi%3D10.1159%2F000078348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Weitzdoerfer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeger, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engidawork, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singewald, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lubec, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lubec, B.</span><span> </span><span class="NLM_article-title">Neuronal nitric oxide synthase knock-out mice show impaired cognitive performance</span> <span class="citation_source-journal">Nitric Oxide</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">130</span><span class="NLM_x">–</span> <span class="NLM_lpage">140</span><span class="refDoi"> DOI: 10.1016/j.niox.2004.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1016%2Fj.niox.2004.03.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=130-140&author=R.+Weitzdoerferauthor=H.+Hoegerauthor=E.+Engidaworkauthor=M.+Engelmannauthor=N.+Singewaldauthor=G.+Lubecauthor=B.+Lubec&title=Neuronal+nitric+oxide+synthase+knock-out+mice+show+impaired+cognitive+performance&doi=10.1016%2Fj.niox.2004.03.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.niox.2004.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.niox.2004.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DWeitzdoerfer%26aufirst%3DR.%26aulast%3DHoeger%26aufirst%3DH.%26aulast%3DEngidawork%26aufirst%3DE.%26aulast%3DEngelmann%26aufirst%3DM.%26aulast%3DSingewald%26aufirst%3DN.%26aulast%3DLubec%26aufirst%3DG.%26aulast%3DLubec%26aufirst%3DB.%26atitle%3DNeuronal%2520nitric%2520oxide%2520synthase%2520knock-out%2520mice%2520show%2520impaired%2520cognitive%2520performance%26jtitle%3DNitric%2520Oxide%26date%3D2004%26volume%3D10%26spage%3D130%26epage%3D140%26doi%3D10.1016%2Fj.niox.2004.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Irwin, S.</span><span> </span><span class="NLM_article-title">Comprehensive observational assessment: Ia. A systematic, quantitative procedure for assessing the behavioral and physiologic state of the mouse</span> <span class="citation_source-journal">Psychopharmacologia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1968</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">222</span><span class="NLM_x">–</span> <span class="NLM_lpage">257</span><span class="refDoi"> DOI: 10.1007/BF00401402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1007%2FBF00401402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=5679627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A280%3ADyaF1M%252FgtlCiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1968&pages=222-257&author=S.+Irwin&title=Comprehensive+observational+assessment%3A+Ia.+A+systematic%2C+quantitative+procedure+for+assessing+the+behavioral+and+physiologic+state+of+the+mouse&doi=10.1007%2FBF00401402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive observational assessment: Ia. A systematic, quantitative procedure for assessing the behavioral and physiologic state of the mouse</span></div><div class="casAuthors">Irwin S</div><div class="citationInfo"><span class="NLM_cas:title">Psychopharmacologia</span>
        (<span class="NLM_cas:date">1968</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">222-57</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQae8F3n8PRtf6AExegM5LLfW6udTcc2ebVDmKe74hjnbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaF1M%252FgtlCiuw%253D%253D&md5=37ab1a9fa105ba776dbd68650fd9edbf</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1007%2FBF00401402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00401402%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DS.%26atitle%3DComprehensive%2520observational%2520assessment%253A%2520Ia.%2520A%2520systematic%252C%2520quantitative%2520procedure%2520for%2520assessing%2520the%2520behavioral%2520and%2520physiologic%2520state%2520of%2520the%2520mouse%26jtitle%3DPsychopharmacologia%26date%3D1968%26volume%3D13%26spage%3D222%26epage%3D257%26doi%3D10.1007%2FBF00401402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Porsolt, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anton, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blavet, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jalfre, M.</span><span> </span><span class="NLM_article-title">Behavioural despair in rats: a new model sensitive to antidepressant treatments</span> <span class="citation_source-journal">Eur. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1978</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">379</span><span class="NLM_x">–</span> <span class="NLM_lpage">391</span><span class="refDoi"> DOI: 10.1016/0014-2999(78)90118-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1016%2F0014-2999%2878%2990118-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=204499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADyaE1cXhvVagsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1978&pages=379-391&author=R.+D.+Porsoltauthor=G.+Antonauthor=N.+Blavetauthor=M.+Jalfre&title=Behavioural+despair+in+rats%3A+a+new+model+sensitive+to+antidepressant+treatments&doi=10.1016%2F0014-2999%2878%2990118-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Behavioral despair in rats:  a new model sensitive to antidepressant treatments</span></div><div class="casAuthors">Porsolt, Roger D.; Anton, Guy; Blavet, Nadine; Jalfre, Maurice</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">379-91</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    </div><div class="casAbstract">Rats when forced to swim in a cylinder from which they cannot escape will, after an initial period of vigorous activity, adopt a characteristic immobile posture which can be readily identified.  Immobility was decreased by various clin. effective antidepressant drugs at doses which otherwise decreased spontaneous motor activity in an open field.  Antidepressants could thus be distinguished from psychostimulants which decreased immobility at doses which increased general activity.  Anxiolytic compds. did not affect immobility, whereas major tranquilizers enhanced it.  Thus, immobility reflects a state of lowered mood in the rat which is selectively sensitive to antidepressant treatments.  Pos. findings with atypical antidepressant drugs such as iprindole and mianserin suggest that the method may be capable of discovering new antidepressants hitherto undetectable with classical pharmacol. tests.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0oUdFHAF1sLVg90H21EOLACvtfcHk0lgymWbFu6uytw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXhvVagsr0%253D&md5=293e1fe8766c76315e86bc375c810a58</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2F0014-2999%2878%2990118-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-2999%252878%252990118-8%26sid%3Dliteratum%253Aachs%26aulast%3DPorsolt%26aufirst%3DR.%2BD.%26aulast%3DAnton%26aufirst%3DG.%26aulast%3DBlavet%26aufirst%3DN.%26aulast%3DJalfre%26aufirst%3DM.%26atitle%3DBehavioural%2520despair%2520in%2520rats%253A%2520a%2520new%2520model%2520sensitive%2520to%2520antidepressant%2520treatments%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1978%26volume%3D47%26spage%3D379%26epage%3D391%26doi%3D10.1016%2F0014-2999%2878%2990118-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 11 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Rachel D. Slack, Therese C. Ku, Jianjing Cao, JoLynn B. Giancola, Alessandro Bonifazi, Claus J. Loland, Alexandra Gadiano, Jenny Lam, Rana Rais, Barbara S. Slusher, Mark Coggiano, Gianluigi Tanda, <span class="NLM_string-name hlFld-ContribAuthor">Amy Hauck Newman</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–Activity Relationships for a Series of (Bis(4-fluorophenyl)methyl)sulfinyl Alkyl Alicyclic Amines at the Dopamine Transporter: Functionalizing the Terminal Nitrogen Affects Affinity, Selectivity, and Metabolic Stability. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (5)
                                     , 2343-2357. <a href="https://doi.org/10.1021/acs.jmedchem.9b01188" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01188</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01188%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure%2525E2%252580%252593Activity%252BRelationships%252Bfor%252Ba%252BSeries%252Bof%252B%252528Bis%2525284-fluorophenyl%252529methyl%252529sulfinyl%252BAlkyl%252BAlicyclic%252BAmines%252Bat%252Bthe%252BDopamine%252BTransporter%25253A%252BFunctionalizing%252Bthe%252BTerminal%252BNitrogen%252BAffects%252BAffinity%25252C%252BSelectivity%25252C%252Band%252BMetabolic%252BStability%26aulast%3DSlack%26aufirst%3DRachel%2BD.%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D23072019%26date%3D14112019%26date%3D29102019%26volume%3D63%26issue%3D5%26spage%3D2343%26epage%3D2357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Predrag Kalaba, Marija Ilić, Nilima Y. Aher, Vladimir Dragačević, Marcus Wieder, Martin Zehl, Judith Wackerlig, Stanislav Beyl, Simone B. Sartori, Karl Ebner, Alexander Roller, Natalie Lukic, Tetyana Beryozkina, Eduardo Rene Perez Gonzalez, Philip Neill, Jawad Akbar Khan, Vasiliy Bakulev, Johann Jakob Leban, Steffen Hering, Christian Pifl, Nicolas Singewald, Jana Lubec, Ernst Urban, Harald H. Sitte, Thierry Langer, <span class="NLM_string-name hlFld-ContribAuthor">Gert Lubec</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–Activity Relationships of Novel Thiazole-Based Modafinil Analogues Acting at Monoamine Transporters. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (1)
                                     , 391-417. <a href="https://doi.org/10.1021/acs.jmedchem.9b01938" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01938</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01938%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure%2525E2%252580%252593Activity%252BRelationships%252Bof%252BNovel%252BThiazole-Based%252BModafinil%252BAnalogues%252BActing%252Bat%252BMonoamine%252BTransporters%26aulast%3DKalaba%26aufirst%3DPredrag%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D23112019%26date%3D30122019%26date%3D16122019%26volume%3D63%26issue%3D1%26spage%3D391%26epage%3D417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jana  Lubec</span>, <span class="hlFld-ContribAuthor ">Predrag  Kalaba</span>, <span class="hlFld-ContribAuthor ">Ahmed M.  Hussein</span>, <span class="hlFld-ContribAuthor ">Daniel Daba  Feyissa</span>, <span class="hlFld-ContribAuthor ">Mohamed H.  Kotob</span>, <span class="hlFld-ContribAuthor ">Rasha R.  Mahmmoud</span>, <span class="hlFld-ContribAuthor ">Oliver  Wieder</span>, <span class="hlFld-ContribAuthor ">Arthur  Garon</span>, <span class="hlFld-ContribAuthor ">Claudia  Sagheddu</span>, <span class="hlFld-ContribAuthor ">Marija  Ilic</span>, <span class="hlFld-ContribAuthor ">Vladimir  Dragačević</span>, <span class="hlFld-ContribAuthor ">Anita  Cybulska-Klosowicz</span>, <span class="hlFld-ContribAuthor ">Martin  Zehl</span>, <span class="hlFld-ContribAuthor ">Judith  Wackerlig</span>, <span class="hlFld-ContribAuthor ">Simone B.  Sartori</span>, <span class="hlFld-ContribAuthor ">Karl  Ebner</span>, <span class="hlFld-ContribAuthor ">Shima  Kouhnavardi</span>, <span class="hlFld-ContribAuthor ">Alexander  Roller</span>, <span class="hlFld-ContribAuthor ">Natalie  Gajic</span>, <span class="hlFld-ContribAuthor ">Marco  Pistis</span>, <span class="hlFld-ContribAuthor ">Nicolas  Singewald</span>, <span class="hlFld-ContribAuthor ">Johann Jakob  Leban</span>, <span class="hlFld-ContribAuthor ">Volker  Korz</span>, <span class="hlFld-ContribAuthor ">Jovana  Malikovic</span>, <span class="hlFld-ContribAuthor ">Roberto  Plasenzotti</span>, <span class="hlFld-ContribAuthor ">Harald H.  Sitte</span>, <span class="hlFld-ContribAuthor ">Francisco J.  Monje</span>, <span class="hlFld-ContribAuthor ">Thierry  Langer</span>, <span class="hlFld-ContribAuthor ">Ernst  Urban</span>, <span class="hlFld-ContribAuthor ">Christian  Pifl</span>, <span class="hlFld-ContribAuthor ">Gert  Lubec</span>. </span><span class="cited-content_cbyCitation_article-title">Reinstatement of synaptic plasticity in the aging brain through specific dopamine transporter inhibition. </span><span class="cited-content_cbyCitation_journal-name">Molecular Psychiatry</span><span> <strong>2021,</strong> <em>55 </em><a href="https://doi.org/10.1038/s41380-021-01214-x" title="DOI URL">https://doi.org/10.1038/s41380-021-01214-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41380-021-01214-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41380-021-01214-x%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Psychiatry%26atitle%3DReinstatement%252Bof%252Bsynaptic%252Bplasticity%252Bin%252Bthe%252Baging%252Bbrain%252Bthrough%252Bspecific%252Bdopamine%252Btransporter%252Binhibition%26aulast%3DLubec%26aufirst%3DJana%26date%3D2021%26date%3D2021%26volume%3D55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gianluigi  Tanda</span>, <span class="hlFld-ContribAuthor ">Melinda  Hersey</span>, <span class="hlFld-ContribAuthor ">Briana  Hempel</span>, <span class="hlFld-ContribAuthor ">Zheng-Xiong  Xi</span>, <span class="hlFld-ContribAuthor ">Amy Hauck  Newman</span>. </span><span class="cited-content_cbyCitation_article-title">Modafinil and its structural analogs as atypical dopamine uptake inhibitors and potential medications for psychostimulant use disorder. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Pharmacology</span><span> <strong>2021,</strong> <em>56 </em>, 13-21. <a href="https://doi.org/10.1016/j.coph.2020.07.007" title="DOI URL">https://doi.org/10.1016/j.coph.2020.07.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.coph.2020.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.coph.2020.07.007%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Pharmacology%26atitle%3DModafinil%252Band%252Bits%252Bstructural%252Banalogs%252Bas%252Batypical%252Bdopamine%252Buptake%252Binhibitors%252Band%252Bpotential%252Bmedications%252Bfor%252Bpsychostimulant%252Buse%252Bdisorder%26aulast%3DTanda%26aufirst%3DGianluigi%26date%3D2021%26volume%3D56%26spage%3D13%26epage%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amy Hauck  Newman</span>, <span class="hlFld-ContribAuthor ">Therese  Ku</span>, <span class="hlFld-ContribAuthor ">Chloe J.  Jordan</span>, <span class="hlFld-ContribAuthor ">Alessandro  Bonifazi</span>, <span class="hlFld-ContribAuthor ">Zheng-Xiong  Xi</span>. </span><span class="cited-content_cbyCitation_article-title">New Drugs, Old Targets: Tweaking the Dopamine System to Treat Psychostimulant Use Disorders. </span><span class="cited-content_cbyCitation_journal-name">Annual Review of Pharmacology and Toxicology</span><span> <strong>2021,</strong> <em>61 </em>
                                    (1)
                                     , 609-628. <a href="https://doi.org/10.1146/annurev-pharmtox-030220-124205" title="DOI URL">https://doi.org/10.1146/annurev-pharmtox-030220-124205</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1146/annurev-pharmtox-030220-124205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1146%2Fannurev-pharmtox-030220-124205%26sid%3Dliteratum%253Aachs%26jtitle%3DAnnual%2520Review%2520of%2520Pharmacology%2520and%2520Toxicology%26atitle%3DNew%252BDrugs%25252C%252BOld%252BTargets%25253A%252BTweaking%252Bthe%252BDopamine%252BSystem%252Bto%252BTreat%252BPsychostimulant%252BUse%252BDisorders%26aulast%3DNewman%26aufirst%3DAmy%2BHauck%26date%3D2021%26volume%3D61%26issue%3D1%26spage%3D609%26epage%3D628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elias  Ranjbari</span>, <span class="hlFld-ContribAuthor ">Mai H.  Philipsen</span>, <span class="hlFld-ContribAuthor ">Zhaoying  Wang</span>, <span class="hlFld-ContribAuthor ">Andrew G.  Ewing</span>. </span><span class="cited-content_cbyCitation_article-title">Combined electrochemistry and mass spectrometry imaging to interrogate the mechanism of action of modafinil, a cognition-enhancing drug, at the cellular and sub-cellular level. </span><span class="cited-content_cbyCitation_journal-name">QRB Discovery</span><span> <strong>2021,</strong> <em>2 </em><a href="https://doi.org/10.1017/qrd.2021.4" title="DOI URL">https://doi.org/10.1017/qrd.2021.4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1017/qrd.2021.4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1017%2Fqrd.2021.4%26sid%3Dliteratum%253Aachs%26jtitle%3DQRB%2520Discovery%26atitle%3DCombined%252Belectrochemistry%252Band%252Bmass%252Bspectrometry%252Bimaging%252Bto%252Binterrogate%252Bthe%252Bmechanism%252Bof%252Baction%252Bof%252Bmodafinil%25252C%252Ba%252Bcognition-enhancing%252Bdrug%25252C%252Bat%252Bthe%252Bcellular%252Band%252Bsub-cellular%252Blevel%26aulast%3DRanjbari%26aufirst%3DElias%26date%3D2021%26date%3D2021%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">JoLynn B.  Giancola</span>, <span class="hlFld-ContribAuthor ">Alessandro  Bonifazi</span>, <span class="hlFld-ContribAuthor ">Jianjing  Cao</span>, <span class="hlFld-ContribAuthor ">Therese  Ku</span>, <span class="hlFld-ContribAuthor ">Alexandra J.  Haraczy</span>, <span class="hlFld-ContribAuthor ">Jenny  Lam</span>, <span class="hlFld-ContribAuthor ">Rana  Rais</span>, <span class="hlFld-ContribAuthor ">Mark A.  Coggiano</span>, <span class="hlFld-ContribAuthor ">Gianluigi  Tanda</span>, <span class="hlFld-ContribAuthor ">Amy Hauck  Newman</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-activity relationships for a series of (Bis(4-fluorophenyl)methyl)sulfinylethyl-aminopiperidines and -piperidine amines at the dopamine transporter: Bioisosteric replacement of the piperazine improves metabolic stability. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>208 </em>, 112674. <a href="https://doi.org/10.1016/j.ejmech.2020.112674" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112674</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112674%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-activity%252Brelationships%252Bfor%252Ba%252Bseries%252Bof%252B%252528Bis%2525284-fluorophenyl%252529methyl%252529sulfinylethyl-aminopiperidines%252Band%252B-piperidine%252Bamines%252Bat%252Bthe%252Bdopamine%252Btransporter%25253A%252BBioisosteric%252Breplacement%252Bof%252Bthe%252Bpiperazine%252Bimproves%252Bmetabolic%252Bstability%26aulast%3DGiancola%26aufirst%3DJoLynn%2BB.%26date%3D2020%26volume%3D208%26spage%3D112674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Renee A.  Rotolo</span>, <span class="hlFld-ContribAuthor ">Predrag  Kalaba</span>, <span class="hlFld-ContribAuthor ">Vladimir  Dragacevic</span>, <span class="hlFld-ContribAuthor ">Rose E.  Presby</span>, <span class="hlFld-ContribAuthor ">Julia  Neri</span>, <span class="hlFld-ContribAuthor ">Emily  Robertson</span>, <span class="hlFld-ContribAuthor ">Jen-Hau  Yang</span>, <span class="hlFld-ContribAuthor ">Merce  Correa</span>, <span class="hlFld-ContribAuthor ">Vasiliy  Bakulev</span>, <span class="hlFld-ContribAuthor ">Natalia N.  Volkova</span>, <span class="hlFld-ContribAuthor ">Christian  Pifl</span>, <span class="hlFld-ContribAuthor ">Gert  Lubec</span>, <span class="hlFld-ContribAuthor ">John D.  Salamone</span>. </span><span class="cited-content_cbyCitation_article-title">Behavioral and dopamine transporter binding properties of the modafinil analog (S, S)-CE-158: reversal of the motivational effects of tetrabenazine and enhancement of progressive ratio responding. </span><span class="cited-content_cbyCitation_journal-name">Psychopharmacology</span><span> <strong>2020,</strong> <em>237 </em>
                                    (11)
                                     , 3459-3470. <a href="https://doi.org/10.1007/s00213-020-05625-6" title="DOI URL">https://doi.org/10.1007/s00213-020-05625-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00213-020-05625-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00213-020-05625-6%26sid%3Dliteratum%253Aachs%26jtitle%3DPsychopharmacology%26atitle%3DBehavioral%252Band%252Bdopamine%252Btransporter%252Bbinding%252Bproperties%252Bof%252Bthe%252Bmodafinil%252Banalog%252B%252528S%25252C%252BS%252529-CE-158%25253A%252Breversal%252Bof%252Bthe%252Bmotivational%252Beffects%252Bof%252Btetrabenazine%252Band%252Benhancement%252Bof%252Bprogressive%252Bratio%252Bresponding%26aulast%3DRotolo%26aufirst%3DRenee%2BA.%26date%3D2020%26date%3D2020%26volume%3D237%26issue%3D11%26spage%3D3459%26epage%3D3470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ana  Sousa</span>, <span class="hlFld-ContribAuthor ">Ricardo Jorge  Dinis-Oliveira</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacokinetic and pharmacodynamic of the cognitive enhancer modafinil: Relevant clinical and forensic aspects. </span><span class="cited-content_cbyCitation_journal-name">Substance Abuse</span><span> <strong>2020,</strong> <em>41 </em>
                                    (2)
                                     , 155-173. <a href="https://doi.org/10.1080/08897077.2019.1700584" title="DOI URL">https://doi.org/10.1080/08897077.2019.1700584</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/08897077.2019.1700584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F08897077.2019.1700584%26sid%3Dliteratum%253Aachs%26jtitle%3DSubstance%2520Abuse%26atitle%3DPharmacokinetic%252Band%252Bpharmacodynamic%252Bof%252Bthe%252Bcognitive%252Benhancer%252Bmodafinil%25253A%252BRelevant%252Bclinical%252Band%252Bforensic%252Baspects%26aulast%3DSousa%26aufirst%3DAna%26date%3D2020%26date%3D2020%26volume%3D41%26issue%3D2%26spage%3D155%26epage%3D173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mary Hongying  Cheng</span>, <span class="hlFld-ContribAuthor ">Ivet  Bahar</span>. </span><span class="cited-content_cbyCitation_article-title">Monoamine transporters: structure, intrinsic dynamics and allosteric regulation. </span><span class="cited-content_cbyCitation_journal-name">Nature Structural & Molecular Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (7)
                                     , 545-556. <a href="https://doi.org/10.1038/s41594-019-0253-7" title="DOI URL">https://doi.org/10.1038/s41594-019-0253-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41594-019-0253-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41594-019-0253-7%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Structural%2520%2526%2520Molecular%2520Biology%26atitle%3DMonoamine%252Btransporters%25253A%252Bstructure%25252C%252Bintrinsic%252Bdynamics%252Band%252Ballosteric%252Bregulation%26aulast%3DCheng%26aufirst%3DMary%2BHongying%26date%3D2019%26date%3D2019%26volume%3D26%26issue%3D7%26spage%3D545%26epage%3D556" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Judith  Camats-Perna</span>, <span class="hlFld-ContribAuthor ">Predrag  Kalaba</span>, <span class="hlFld-ContribAuthor ">Karl  Ebner</span>, <span class="hlFld-ContribAuthor ">Simone B.  Sartori</span>, <span class="hlFld-ContribAuthor ">Harish  Vuyyuru</span>, <span class="hlFld-ContribAuthor ">Nilima Y.  Aher</span>, <span class="hlFld-ContribAuthor ">Vladimir  Dragačević</span>, <span class="hlFld-ContribAuthor ">Nicolas  Singewald</span>, <span class="hlFld-ContribAuthor ">Mario  Engelmann</span>, <span class="hlFld-ContribAuthor ">Gert  Lubec</span>. </span><span class="cited-content_cbyCitation_article-title">Differential Effects of Novel Dopamine Reuptake Inhibitors on Interference With Long-Term Social Memory in Mice. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Behavioral Neuroscience</span><span> <strong>2019,</strong> <em>13 </em><a href="https://doi.org/10.3389/fnbeh.2019.00063" title="DOI URL">https://doi.org/10.3389/fnbeh.2019.00063</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fnbeh.2019.00063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffnbeh.2019.00063%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Behavioral%2520Neuroscience%26atitle%3DDifferential%252BEffects%252Bof%252BNovel%252BDopamine%252BReuptake%252BInhibitors%252Bon%252BInterference%252BWith%252BLong-Term%252BSocial%252BMemory%252Bin%252BMice%26aulast%3DCamats-Perna%26aufirst%3DJudith%26date%3D2019%26date%3D2019%26volume%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0001.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of 2-[(diphenylmethyl)sulfinyl]acetamide (modafinil).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0010.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>3a</b>–<b>e</b> and <b>4a</b>–<b>e</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) DMF, <i>t</i>-BuOK, rt, overnight; (b) 1,4-dioxane, DIPEA, rt, 3 h; (c) CH<sub>3</sub>COOH, H<sub>2</sub>O<sub>2</sub>, rt, 24 h.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0011.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>6a</b>–<b>q</b> and <b>7a</b>–<b>q</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) MeOH, K<sub>2</sub>CO<sub>3</sub>, rt, 48 h; (b) CH<sub>3</sub>COOH, H<sub>2</sub>O<sub>2</sub>, rt, overnight.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0012.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>10a</b>–<b>f</b> and <b>11a</b>–<b>f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) BF<sub>3</sub>·Et<sub>2</sub>O, CH<sub>3</sub>COOH, rt, overnight; (b) CH<sub>3</sub>COOH, H<sub>2</sub>O<sub>2</sub>, rt, overnight.</p></p></figure><figure data-id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. A release assay was performed to examine whether compounds <b>7d</b> and <b>11b</b> have any ability to release the neurotransmitter from inside the cells, like amphetamine. HEK-DAT cells were grown on PDL-coated coverslips, treated with 0.1 μM [<sup>3</sup>H]MPP<sup>+</sup> (methyl-4-phenylpyridinium) at 37 °C for 20 min and washed with KHB for 40 min in superfusion chambers. Four experiments were compared, and all four started with three fractions under untreated conditions (baseline). The next four fractions were either again untreated or treated with 10 μM monensin. The final five fractions were either treated with 10 μM test compound (<b>7d</b> or <b>11b</b>) or 10 μM <span class="smallcaps smallerCapital">d</span>-amphetamine. Nonlinear regression analysis was carried out by using GraphPad Prism 6. Values are presented as mean ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Mutually exclusive binding of cocaine in the presence of analogues <b>7d</b> and <b>11b</b>. Intact HEK-DAT cells were incubated with the radioligand (∼10 nM [<sup>3</sup>H]WIN 35,428), the indicated concentrations of <b>7d</b> or <b>11b</b>, and increasing concentrations of cocaine in binding buffer. Nonspecific binding was determined in the presence of 30 μM mazindol (control). The binding data from the competition experiments is transformed into a Dixon plot by expressing the reciprocal of [<sup>3</sup>H]WIN 35,428 bound (measured in femtomole) as a function of the concentration of cocaine at a fixed concentration of <b>7d</b> or <b>11b</b>. Data are means ± SD of three experiments performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. For compound <b>7d</b> and <b>11b</b> the highest scored docking pose (A) and the final pose after the MD simulation (B) is shown. The labeled amino acids are shown explicitly on the protein backbone. The ligands move during the MD simulations from their initial position (A) to their final position (B), however, they were found not to leave the dopamine binding site. The homology model was based on the crystallized Drosophila DAT, deposited in the Protein Data Bank with the PDB-ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M48">4M48</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0005.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Different interactions between parts of the ligand and amino acids are shown as heatmap ranging from dark violet (lowest interactions), green (medium interactions) to bright yellow (highest interactions). Rows indicate part of the ligands and columns the different amino acids. The first letters before the column indicate the type of interaction, H for hydrophobic interaction, HBA for hydrogen bond acceptor, HBD for hydrogen bond donor, AR for aromatic interaction, and PI for positively ionizable interaction.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0006.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Open field study: Total distance traveled, resting time, local and large movements, average velocity, number of crossings of the center, frequency of spontaneous changes of direction, and time spent at the margin are measured as parameters between control and <b>11b</b>-treated group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0007.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Elevated plus maze study: Time spent in the left open arm, left closed arm, total distance traveled, distance traveled in open and closed arms, time spend in the open and closed arm, and the resting time between groups are measured as determining parameters. There is no significant difference between control and treated group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0008.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Rotarod study of <b>11b</b>: Times spent on the revolving rod were measured for the control and <b>11b</b>-treated group. There is no significant difference between control and treated group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/medium/jm-2017-01313b_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0009.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Forced swim test study: Immobility times in the water were measured. There is no significant difference between control and treated group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01313/20171117/images/large/jm-2017-01313b_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01313&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i96">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71472" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71472" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 47 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Urban, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, W.-J.</span><span> </span><span class="NLM_article-title">Performance enhancement at the cost of potential brain plasticity: neural ramifications of nootropic drugs in the healthy developing brain</span> <span class="citation_source-journal">Front. Syst. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">38</span><span class="refDoi"> DOI: 10.3389/fnsys.2014.00038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.3389%2Ffnsys.2014.00038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=24860437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A280%3ADC%252BC2cflvFamsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=38&author=K.+R.+Urbanauthor=W.-J.+Gao&title=Performance+enhancement+at+the+cost+of+potential+brain+plasticity%3A+neural+ramifications+of+nootropic+drugs+in+the+healthy+developing+brain&doi=10.3389%2Ffnsys.2014.00038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Performance enhancement at the cost of potential brain plasticity: neural ramifications of nootropic drugs in the healthy developing brain</span></div><div class="casAuthors">Urban Kimberly R; Gao Wen-Jun</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in systems neuroscience</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">38</span>
        ISSN:<span class="NLM_cas:issn">1662-5137</span>.
    </div><div class="casAbstract">Cognitive enhancement is perhaps one of the most intriguing and controversial topics in neuroscience today.  Currently, the main classes of drugs used as potential cognitive enhancers include psychostimulants (methylphenidate (MPH), amphetamine), but wakefulness-promoting agents (modafinil) and glutamate activators (ampakine) are also frequently used.  Pharmacologically, substances that enhance the components of the memory/learning circuits-dopamine, glutamate (neuronal excitation), and/or norepinephrine-stand to improve brain function in healthy individuals beyond their baseline functioning.  In particular, non-medical use of prescription stimulants such as MPH and illicit use of psychostimulants for cognitive enhancement have seen a recent rise among teens and young adults in schools and college campuses.  However, this enhancement likely comes with a neuronal, as well as ethical, cost.  Altering glutamate function via the use of psychostimulants may impair behavioral flexibility, leading to the development and/or potentiation of addictive behaviors.  Furthermore, dopamine and norepinephrine do not display linear effects; instead, their modulation of cognitive and neuronal function maps on an inverted-U curve.  Healthy individuals run the risk of pushing themselves beyond optimal levels into hyperdopaminergic and hypernoradrenergic states, thus vitiating the very behaviors they are striving to improve.  Finally, recent studies have begun to highlight potential damaging effects of stimulant exposure in healthy juveniles.  This review explains how the main classes of cognitive enhancing drugs affect the learning and memory circuits, and highlights the potential risks and concerns in healthy individuals, particularly juveniles and adolescents.  We emphasize the performance enhancement at the potential cost of brain plasticity that is associated with the neural ramifications of nootropic drugs in the healthy developing brain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9mMH7P4hI_--MdeMhXAHkfW6udTcc2eZSzp3vLqVeJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cflvFamsg%253D%253D&md5=d552cabbeece23291f10b62d98deb3fc</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3389%2Ffnsys.2014.00038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnsys.2014.00038%26sid%3Dliteratum%253Aachs%26aulast%3DUrban%26aufirst%3DK.%2BR.%26aulast%3DGao%26aufirst%3DW.-J.%26atitle%3DPerformance%2520enhancement%2520at%2520the%2520cost%2520of%2520potential%2520brain%2520plasticity%253A%2520neural%2520ramifications%2520of%2520nootropic%2520drugs%2520in%2520the%2520healthy%2520developing%2520brain%26jtitle%3DFront.%2520Syst.%2520Neurosci.%26date%3D2014%26volume%3D8%26spage%3D38%26doi%3D10.3389%2Ffnsys.2014.00038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Murphy, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekstrand, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarchick, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wideman, C. H.</span><span> </span><span class="NLM_article-title">Modafinil as a cognitive enhancer of spatial working memory in rats</span> <span class="citation_source-journal">Physiol. Behav.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">142</span><span class="NLM_x">, </span> <span class="NLM_fpage">126</span><span class="NLM_x">–</span> <span class="NLM_lpage">130</span><span class="refDoi"> DOI: 10.1016/j.physbeh.2015.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1016%2Fj.physbeh.2015.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=25656691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitl2lu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2015&pages=126-130&author=H.+M.+Murphyauthor=D.+Ekstrandauthor=M.+Tarchickauthor=C.+H.+Wideman&title=Modafinil+as+a+cognitive+enhancer+of+spatial+working+memory+in+rats&doi=10.1016%2Fj.physbeh.2015.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Modafinil as a cognitive enhancer of spatial working memory in rats</span></div><div class="casAuthors">Murphy, Helen M.; Ekstrand, Dylan; Tarchick, Matthew; Wideman, Cyrilla H.</div><div class="citationInfo"><span class="NLM_cas:title">Physiology & Behavior</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">126-130</span>CODEN:
                <span class="NLM_cas:coden">PHBHA4</span>;
        ISSN:<span class="NLM_cas:issn">0031-9384</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The present expt. examd. the influence of modafinil on working memory in rats.  Control and exptl. rats were placed in cages equipped with a running wheel.  The study was divided into three periods: 1. habituation, 2. exptl. - in which modafinil was orally administered to exptl. animals, and 3. withdrawal.  Spatial working memory was tested utilizing the Morris Water Maze.  Animals were given two trials in each session: 1. a sample trial in which they discovered the location of the platform and 2. a test trial in which they recalled the location of the platform.  Platform placement and starting place location of the rats were changed every session.  Performance of control animals during sample trials and test trials showed no difference in time to reach the platform; whereas, exptl. animals demonstrated faster attainment of the goal during the test trial.  In withdrawal, there was no difference in time between the two trials for exptl. animals.  Wheel running activity of the exptl. group was lower than that of the control group during the exptl. period.  This study supports the hypothesis that modafinil has a pos. effect on working memory in rats.  It is suggested that benefits of modafinil can extend beyond its prescribed usage; however, these advantages of modafinil use should be addressed in discussions related to ethical concerns assocd. with neuroenhancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBq28Wmv2ueLVg90H21EOLACvtfcHk0lh3YIq817USWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitl2lu7k%253D&md5=b6b9fb7ea0d0af6d21fcb3129d074638</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.physbeh.2015.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.physbeh.2015.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DMurphy%26aufirst%3DH.%2BM.%26aulast%3DEkstrand%26aufirst%3DD.%26aulast%3DTarchick%26aufirst%3DM.%26aulast%3DWideman%26aufirst%3DC.%2BH.%26atitle%3DModafinil%2520as%2520a%2520cognitive%2520enhancer%2520of%2520spatial%2520working%2520memory%2520in%2520rats%26jtitle%3DPhysiol.%2520Behav.%26date%3D2015%26volume%3D142%26spage%3D126%26epage%3D130%26doi%3D10.1016%2Fj.physbeh.2015.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Gerrard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malcolm, R.</span><span> </span><span class="NLM_article-title">Mechanisms of modafinil: A review of current research</span> <span class="citation_source-journal">Neuropsychiatr. Dis. Treat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">349</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19300566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD2sXps1WnsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=349-364&author=P.+Gerrardauthor=R.+Malcolm&title=Mechanisms+of+modafinil%3A+A+review+of+current+research"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of modafinil: a review of current research</span></div><div class="casAuthors">Gerrard, Paul; Malcolm, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Neuropsychiatric Disease and Treatment</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">349-364</span>CODEN:
                <span class="NLM_cas:coden">NDTEAZ</span>;
        ISSN:<span class="NLM_cas:issn">1176-6328</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press (NZ) Ltd.</span>)
        </div><div class="casAbstract">A review.  The novel wake-promoting agent modafinil has been in use for the treatment of several sleep disorders for a few years and is now undergoing clin. trials for its use in the treatment of stimulant addiction, but its primary mechanism of action remains elusive.  Previous lab. studies have shown that modafinil has antioxidative and neuroprotective effects, which have not previously been suggested to be related to its wake-promoting effects.  However, recent research indicates that free radicals may be related to sleep induction as well as cellular damage, suggesting that a common target of action may mediate modafinil's ability to oppose both of these effects.  In this review we summarize and discuss previously published research on modafinil's neural, cytoprotective, and cognitive effects, and we propose possible primary biochem. targets that could underlie the effects of modafinil obsd. in these studies.  We also suggest neurocognitive mechanisms responsible for modafinil's cognitive enhancing effects and its therapeutic potential in the treatment of stimulant addiction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbxuHCZ8GqFbVg90H21EOLACvtfcHk0lh3YIq817USWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXps1WnsLw%253D&md5=3b9d1c81db02be3d8ecf8c370208bddf</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGerrard%26aufirst%3DP.%26aulast%3DMalcolm%26aufirst%3DR.%26atitle%3DMechanisms%2520of%2520modafinil%253A%2520A%2520review%2520of%2520current%2520research%26jtitle%3DNeuropsychiatr.%2520Dis.%2520Treat.%26date%3D2007%26volume%3D3%26spage%3D349%26epage%3D364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Schmitt, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reith, M. E. A.</span><span> </span><span class="NLM_article-title">The atypical stimulant and nootropic modafinil interacts with the dopamine transporter in a different manner than classical cocaine-like inhibitors</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e25790</span><span class="refDoi"> DOI: 10.1371/journal.pone.0025790</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1371%2Fjournal.pone.0025790" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=22043293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVCku73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e25790&author=K.+C.+Schmittauthor=M.+E.+A.+Reith&title=The+atypical+stimulant+and+nootropic+modafinil+interacts+with+the+dopamine+transporter+in+a+different+manner+than+classical+cocaine-like+inhibitors&doi=10.1371%2Fjournal.pone.0025790"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The atypical stimulant and nootropic modafinil interacts with the dopamine transporter in a different manner than classical cocaine-like inhibitors</span></div><div class="casAuthors">Schmitt, Kyle C.; Reith, Maarten E. A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e25790</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Modafinil is a mild psychostimulant with pro-cognitive and antidepressant effects.  Unlike many conventional stimulants, modafinil has little appreciable potential for abuse, making it a promising therapeutic agent for cocaine addiction.  The chief mol. target of modafinil is the dopamine transporter (DAT); however, the mechanistic details underlying modafinil's unique effects remain unknown.  Recent studies suggest that the conformational effects of a given DAT ligand influence the magnitude of the ligand's reinforcing properties.  For example, the atypical DAT inhibitors benztropine and GBR12909 do not share cocaine's notorious addictive liability, despite having greater binding affinity.  Here, we show that the binding mechanism of modafinil is different than cocaine and similar to other atypical inhibitors.  We previously established two mutations (W84L and D313N) that increase the likelihood that the DAT will adopt an outward-facing conformational state - these mutations increase the affinity of cocaine-like inhibitors considerably, but have little or opposite effect on atypical inhibitor binding.  Thus, a compd.'s WT/mutant affinity ratio can indicate whether the compd. preferentially interacts with a more outward- or inward-facing conformational state.  Modafinil displayed affinity ratios similar to those of benztropine, GBR12909 and bupropion (which lack cocaine-like effects in humans), but far different than those of cocaine, β-CFT or methylphenidate.  Whereas treatment with zinc (known to stabilize an outward-facing transporter state) increased the affinity of cocaine and methylphenidate two-fold, it had little or no effect on the binding of modafinil, benztropine, bupropion or GBR12909.  Addnl., computational modeling of inhibitor binding indicated that while β-CFT and methylphenidate stabilize an "open-to-out" conformation, binding of either modafinil or bupropion gives rise to a more closed conformation.  Our findings highlight a mechanistic difference between modafinil and cocaine-like stimulants and further demonstrate that the conformational effects of a given DAT inhibitor influence its phenomenol. effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk_0tcjHjbqbVg90H21EOLACvtfcHk0lgzvK67G6w_yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVCku73L&md5=6599e43f3f518b78bf834fb6c6d72010</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025790&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025790%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DK.%2BC.%26aulast%3DReith%26aufirst%3DM.%2BE.%2BA.%26atitle%3DThe%2520atypical%2520stimulant%2520and%2520nootropic%2520modafinil%2520interacts%2520with%2520the%2520dopamine%2520transporter%2520in%2520a%2520different%2520manner%2520than%2520classical%2520cocaine-like%2520inhibitors%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De25790%26doi%3D10.1371%2Fjournal.pone.0025790" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Volkow, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fowler, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alexoff, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Telang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayne, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hooker, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shea, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muench, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apelskog-Torres, K.</span><span> </span><span class="NLM_article-title">Effects of modafinil on dopamine and dopamine transporters in the male human brain: clinical implications</span> <span class="citation_source-journal">JAMA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">301</span><span class="NLM_x">, </span> <span class="NLM_fpage">1148</span><span class="NLM_x">–</span> <span class="NLM_lpage">1154</span><span class="refDoi"> DOI: 10.1001/jama.2009.351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1001%2Fjama.2009.351" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19293415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsVOlsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=301&publication_year=2009&pages=1148-1154&author=N.+D.+Volkowauthor=J.+S.+Fowlerauthor=J.+Loganauthor=D.+Alexoffauthor=W.+Zhuauthor=F.+Telangauthor=G.+J.+Wangauthor=M.+Jayneauthor=J.+M.+Hookerauthor=C.+Wongauthor=+Hubbardauthor=P.+Carterauthor=D.+Warnerauthor=P.+Kingauthor=C.+Sheaauthor=Y.+Xuauthor=L.+Muenchauthor=K.+Apelskog-Torres&title=Effects+of+modafinil+on+dopamine+and+dopamine+transporters+in+the+male+human+brain%3A+clinical+implications&doi=10.1001%2Fjama.2009.351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of modafinil on dopamine and dopamine transporters in the male human brain. Clinical implications</span></div><div class="casAuthors">Volkow, Nora D.; Fowler, Joanna S.; Logan, Jean; Alexoff, David; Zhu, Wei; Telang, Frank; Wang, Gene-Jack; Jayne, Millard; Hooker, Jacob M.; Wong, Christopher; Hubbard, Barbara; Carter, Pauline; Warner, Donald; King, Payton; Shea, Colleen; Xu, Youwen; Muench, Lisa; Apelskog-Torres, Karen</div><div class="citationInfo"><span class="NLM_cas:title">JAMA, the Journal of the American Medical Association</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">301</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1148-1154</span>CODEN:
                <span class="NLM_cas:coden">JAMAAP</span>;
        ISSN:<span class="NLM_cas:issn">0098-7484</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">Modafinil, a wake-promoting drug used to treat narcolepsy, is increasingly being used as a cognitive enhancer.  Although initially launched as distinct from stimulants that increase extracellular dopamine by targeting dopamine transporters, recent preclin. studies suggest otherwise.  Objective was to measure the acute effects of modafinil at doses used therapeutically (200 mg and 400 mg given orally) on extracellular dopamine and on dopamine transporters in the male human brain.  Design, Setting, and Participants Positron emission tomog. with [11C]raclopride (D2/D3 radioligand sensitive to changes in endogenous dopamine) and [11C]cocaine (dopamine transporter radioligand) was used to measure the effects of modafinil on extracellular dopamine and on dopamine transporters in 10 healthy male participants.  The study took place over an 8-mo period (2007-2008) at Brookhaven National Lab.  Main Outcome Measures Primary outcomes were changes in dopamine D2/D3 receptor and dopamine transporter availability (measured by changes in binding potential) after modafinil when compared with after placebo.  Modafinil decreased mean (SD) [11C]raclopride binding potential in caudate (6.1% [6.5%]; 95% confidence interval [CI], 1.5% to 10.8%; P =.02), putamen (6.7% [4.9%]; 95% CI, 3.2% to 10.3%; P =.002), and nucleus accumbens (19.4% [20%]; 95% CI, 5% to 35%; P =.02), reflecting increases in extracellular dopamine.  Modafinil also decreased [11C]cocaine binding potential in caudate (53.8% [13.8%]; 95% CI, 43.9% to 63.6%; P <.001), putamen (47.2% [11.4%]; 95% CI, 39.1% to 55.4%; P <.001), and nucleus accumbens (39.3% [10%]; 95% CI, 30% to 49%; P =.001), reflecting occupancy of dopamine transporters.  In this pilot study, modafinil blocked dopamine transporters and increased dopamine in the human brain (including the nucleus accumbens).  Because drugs that increase dopamine in the nucleus accumbens have the potential for abuse, and considering the increasing use of modafinil, these results highlight the need for heightened awareness for potential abuse of and dependence on modafinil in vulnerable populations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2VFxKbKbpArVg90H21EOLACvtfcHk0lgzvK67G6w_yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsVOlsbY%253D&md5=a8f5a22cc75df00230c83eb666f7e9de</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1001%2Fjama.2009.351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.2009.351%26sid%3Dliteratum%253Aachs%26aulast%3DVolkow%26aufirst%3DN.%2BD.%26aulast%3DFowler%26aufirst%3DJ.%2BS.%26aulast%3DLogan%26aufirst%3DJ.%26aulast%3DAlexoff%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DTelang%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DG.%2BJ.%26aulast%3DJayne%26aufirst%3DM.%26aulast%3DHooker%26aufirst%3DJ.%2BM.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DHubbard%26aulast%3DCarter%26aufirst%3DP.%26aulast%3DWarner%26aufirst%3DD.%26aulast%3DKing%26aufirst%3DP.%26aulast%3DShea%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DMuench%26aufirst%3DL.%26aulast%3DApelskog-Torres%26aufirst%3DK.%26atitle%3DEffects%2520of%2520modafinil%2520on%2520dopamine%2520and%2520dopamine%2520transporters%2520in%2520the%2520male%2520human%2520brain%253A%2520clinical%2520implications%26jtitle%3DJAMA%26date%3D2009%26volume%3D301%26spage%3D1148%26epage%3D1154%26doi%3D10.1001%2Fjama.2009.351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Mignot, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishino, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guilleminault, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dement, W. C.</span><span> </span><span class="NLM_article-title">Modafinil binds to the dopamine uptake carrier site with low affinity</span> <span class="citation_source-journal">Sleep</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">437</span><span class="refDoi"> DOI: 10.1093/sleep/17.5.436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1093%2Fsleep%2F17.5.436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=7991954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A280%3ADyaK2M%252FosVSmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1994&pages=436-437&author=E.+Mignotauthor=S.+Nishinoauthor=C.+Guilleminaultauthor=W.+C.+Dement&title=Modafinil+binds+to+the+dopamine+uptake+carrier+site+with+low+affinity&doi=10.1093%2Fsleep%2F17.5.436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Modafinil binds to the dopamine uptake carrier site with low affinity</span></div><div class="casAuthors">Mignot E; Nishino S; Guilleminault C; Dement W C</div><div class="citationInfo"><span class="NLM_cas:title">Sleep</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">436-7</span>
        ISSN:<span class="NLM_cas:issn">0161-8105</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTIGv6cQ4HX-Rjv2lLSLOxFfW6udTcc2eakjgIbBAMZZ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2M%252FosVSmsA%253D%253D&md5=888ee2341f4b1a375f0ffa25f3d8523d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fsleep%2F17.5.436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fsleep%252F17.5.436%26sid%3Dliteratum%253Aachs%26aulast%3DMignot%26aufirst%3DE.%26aulast%3DNishino%26aufirst%3DS.%26aulast%3DGuilleminault%26aufirst%3DC.%26aulast%3DDement%26aufirst%3DW.%2BC.%26atitle%3DModafinil%2520binds%2520to%2520the%2520dopamine%2520uptake%2520carrier%2520site%2520with%2520low%2520affinity%26jtitle%3DSleep%26date%3D1994%26volume%3D17%26spage%3D436%26epage%3D437%26doi%3D10.1093%2Fsleep%2F17.5.436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Zolkowska, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothman, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Partilla, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prisinzano, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumann, M. H.</span><span> </span><span class="NLM_article-title">Evidence for the involvement of dopamine transporters in the behavioral stimulant effects of modafinil</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">329</span><span class="NLM_x">, </span> <span class="NLM_fpage">738</span><span class="NLM_x">–</span> <span class="NLM_lpage">746</span><span class="refDoi"> DOI: 10.1124/jpet.108.146142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1124%2Fjpet.108.146142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19197004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlt1OlsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=329&publication_year=2009&pages=738-746&author=D.+Zolkowskaauthor=R.+Jainauthor=R.+B.+Rothmanauthor=J.+S.+Partillaauthor=B.+L.+Rothauthor=V.+Setolaauthor=T.+E.+Prisinzanoauthor=M.+H.+Baumann&title=Evidence+for+the+involvement+of+dopamine+transporters+in+the+behavioral+stimulant+effects+of+modafinil&doi=10.1124%2Fjpet.108.146142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for the involvement of dopamine transporters in behavioral stimulant effects of modafinil</span></div><div class="casAuthors">Zolkowska, Dorota; Jain, Raka; Rothman, Richard B.; Partilla, John S.; Roth, Bryan L.; Setola, Vincent; Prisinzano, Thomas E.; Baumann, Michael H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">329</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">738-746</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Modafinil is prescribed for numerous medical conditions, but the drug's mechanism of action is unclear.  Here, we examd. the interaction of modafinil with receptors and transporters in vitro and compared pharmacol. effects of the drug with those produced by indirect dopamine (DA) agonists 1-[2-[bis(4-fluorophenyl)methoxy]ethyl]-4-(3-phenylpropyl)piperazine (GBR12909) and (+)-methamphetamine (METH).  Modafinil was screened at various receptors and transporters using binding assays.  Transporter-mediated uptake and release were examd. in rat brain synaptosomes.  Effects of modafinil on motor activity and neurochem. were detd. in rats undergoing in vivo microdialysis in nucleus accumbens.  Of the receptors and transporters assayed, modafinil displayed measurable potency only at DA transporters (DAT), inhibiting [3H]DA uptake, with an IC50 value of 4.0 μM.  Accordingly, modafinil pretreatment (10 μM) antagonized METH-induced release of the DAT substrate [3H]1-methyl-4-phenylpyridinium.  I.v. modafinil (20 and 60 mg/kg) produced dose-dependent increases in motor activity and extracellular DA, without affecting serotonin (5-HT).  Analogous results were obsd. for GBR12909 (1 and 3 mg/kg), whereas METH (0.3 and 1 mg/kg) increased DA and 5-HT.  Locomotor effects of all drugs were pos. correlated with dialyzate DA (P < 0.001).  Interestingly, modafinil pretreatment reduced METH-induced ambulation and DA release.  Our data show that modafinil interacts with DAT sites in rat brain, a property shared with agonist medications under investigation for treating cocaine dependence.  Nondopaminergic mechanisms may also contribute to the pharmacol. of modafinil.  Finally, the results suggest that modafinil should be tested as an adjunct for treating METH addiction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhab522ik-NrVg90H21EOLACvtfcHk0lg7BOO6dFhZ-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlt1OlsL4%253D&md5=0c5af6285ed23382c46a0d35bafbaa3b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1124%2Fjpet.108.146142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.108.146142%26sid%3Dliteratum%253Aachs%26aulast%3DZolkowska%26aufirst%3DD.%26aulast%3DJain%26aufirst%3DR.%26aulast%3DRothman%26aufirst%3DR.%2BB.%26aulast%3DPartilla%26aufirst%3DJ.%2BS.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DPrisinzano%26aufirst%3DT.%2BE.%26aulast%3DBaumann%26aufirst%3DM.%2BH.%26atitle%3DEvidence%2520for%2520the%2520involvement%2520of%2520dopamine%2520transporters%2520in%2520the%2520behavioral%2520stimulant%2520effects%2520of%2520modafinil%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2009%26volume%3D329%26spage%3D738%26epage%3D746%26doi%3D10.1124%2Fjpet.108.146142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Loland, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mereu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okunola, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prisinzano, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopajtic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katz, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">R-Modafinil (Armodafinil): a unique dopamine uptake inhibitor and potential medication for psychostimulant abuse</span> <span class="citation_source-journal">Biol. Psychiatry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">413</span><span class="refDoi"> DOI: 10.1016/j.biopsych.2012.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1016%2Fj.biopsych.2012.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=22537794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFelt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=405-413&author=J.+C.+Lolandauthor=M.+Mereuauthor=O.+Okunolaauthor=J.+Caoauthor=T.+E.+Prisinzanoauthor=S.+Mazierauthor=T.+Kopajticauthor=L.+Shiauthor=J.+L.+Katzauthor=G.+Tandaauthor=A.+H.+Newman&title=R-Modafinil+%28Armodafinil%29%3A+a+unique+dopamine+uptake+inhibitor+and+potential+medication+for+psychostimulant+abuse&doi=10.1016%2Fj.biopsych.2012.03.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">R-Modafinil (Armodafinil): A Unique Dopamine Uptake Inhibitor and Potential Medication for Psychostimulant Abuse</span></div><div class="casAuthors">Loland, Claus J.; Mereu, Maddalena; Okunola, Oluyomi M.; Cao, Jianjing; Prisinzano, Thomas E.; Mazier, Sonia; Kopajtic, Theresa; Shi, Lei; Katz, Jonathan L.; Tanda, Gianluigi; Newman, Amy Hauck</div><div class="citationInfo"><span class="NLM_cas:title">Biological Psychiatry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">405-413</span>CODEN:
                <span class="NLM_cas:coden">BIPCBF</span>;
        ISSN:<span class="NLM_cas:issn">0006-3223</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">(±)-Modafinil has piqued interest as a treatment for attention-deficit/hyperactivity disorder and stimulant dependence.  The R-enantiomer of modafinil might have unique pharmacol. properties that should be further investigated.  (±)-Modafinil and its R-(-)- and S-(+)-enantiomers were synthesized and tested for inhibition of [3H] dopamine (DA) uptake and [3H]WIN 35428 binding in human dopamine transporter (DAT) wild-type and mutants with altered conformational equil.  Data were compared with cocaine and the atypical DA uptake inhibitor, JHW 007.  R- and S-modafinil were also evaluated in microdialysis studies in the mouse nucleus accumbens shell and in a cocaine discrimination procedure.  (±)-, R-, and S-modafinil bind to the DAT and inhibit DA uptake less potently than cocaine, with R-modafinil having approx. threefold higher affinity than its S-enantiomer.  Mol. docking studies revealed subtle differences in binding modes for the enantiomers.  R-modafinil was significantly less potent in the DAT Y156F mutant compared with wild-type DAT, whereas S-modafinil was affected less.  Studies with the Y335A DAT mutant showed that the R- and S-enantiomers tolerated the inward-facing conformation better than cocaine, which was further supported by [2-(trimethylammonium)ethyl]-methanethiosulfonate reactivity on the DAT E2C I159C.  Microdialysis studies demonstrated that both R- and S-modafinil produced increases in extracellular DA concns. in the nucleus accumbens shell less efficaciously than cocaine and with a longer duration of action.  Both enantiomers fully substituted in mice trained to discriminate cocaine from saline.R-modafinil displays an in vitro profile different from cocaine.  Future trials with R-modafinil as a substitute therapy with the potential benefit of cognitive enhancement for psychostimulant addiction are warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjL4KiRs4PUbVg90H21EOLACvtfcHk0lg7BOO6dFhZ-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFelt74%253D&md5=78808210fc68d2fbcdd97190f03b63d0</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.biopsych.2012.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopsych.2012.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DLoland%26aufirst%3DJ.%2BC.%26aulast%3DMereu%26aufirst%3DM.%26aulast%3DOkunola%26aufirst%3DO.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DPrisinzano%26aufirst%3DT.%2BE.%26aulast%3DMazier%26aufirst%3DS.%26aulast%3DKopajtic%26aufirst%3DT.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DKatz%26aufirst%3DJ.%2BL.%26aulast%3DTanda%26aufirst%3DG.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DR-Modafinil%2520%2528Armodafinil%2529%253A%2520a%2520unique%2520dopamine%2520uptake%2520inhibitor%2520and%2520potential%2520medication%2520for%2520psychostimulant%2520abuse%26jtitle%3DBiol.%2520Psychiatry%26date%3D2012%26volume%3D72%26spage%3D405%26epage%3D413%26doi%3D10.1016%2Fj.biopsych.2012.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prisinzano, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okunola, O. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopajtic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shook, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katz, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">SARs at the monoamine transporters for a novel series of modafinil analogues</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">48</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span><span class="refDoi"> DOI: 10.1021/ml1002025</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml1002025" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12lt7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=48-52&author=J.+Caoauthor=T.+E.+Prisinzanoauthor=O.+M.+Okunolaauthor=T.+Kopajticauthor=M.+Shookauthor=J.+L.+Katzauthor=A.+H.+Newman&title=SARs+at+the+monoamine+transporters+for+a+novel+series+of+modafinil+analogues&doi=10.1021%2Fml1002025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">SARs at the Monoamine Transporters for a Novel Series of Modafinil Analogues</span></div><div class="casAuthors">Cao, Jianjing; Prisinzano, Thomas E.; Okunola, Oluyomi M.; Kopajtic, Theresa; Shook, Matthew; Katz, Jonathan L.; Hauck Newman, Amy</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-52</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of modafinil (1) analogs were synthesized wherein (1) para-halo-substitutents were added to the aryl rings, (2) the sulfoxide function was removed, and (3) the primary amide group was replaced with secondary and tertiary amides and amines to investigate the effects of these chem. modifications on dopamine transporter, serotonin transporter, and norepinephrine transporter binding.  In addn., the locomotor-stimulant effects in mice of (±)-modafinil (1) (I), its R- and S-enantiomers, and its para-chloro sulfinylacetamide analog (5c) (II) were compared to those of cocaine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDr9ac-J7nqbVg90H21EOLACvtfcHk0lg7BOO6dFhZ-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12lt7bN&md5=660e546e8524b2e5e8b448ee5bf0e567</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fml1002025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml1002025%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DPrisinzano%26aufirst%3DT.%2BE.%26aulast%3DOkunola%26aufirst%3DO.%2BM.%26aulast%3DKopajtic%26aufirst%3DT.%26aulast%3DShook%26aufirst%3DM.%26aulast%3DKatz%26aufirst%3DJ.%2BL.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DSARs%2520at%2520the%2520monoamine%2520transporters%2520for%2520a%2520novel%2520series%2520of%2520modafinil%2520analogues%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D2%26spage%3D48%26epage%3D52%26doi%3D10.1021%2Fml1002025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Okunola-Bakare, O. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopajtic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katz, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loland, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Elucidation of structural elements for selectivity across monoamine transporters: novel 2-[(diphenylmethyl)sulfynil]acetamide (modafinil) analogues</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1000</span><span class="NLM_x">–</span> <span class="NLM_lpage">1013</span><span class="refDoi"> DOI: 10.1021/jm401754x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401754x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1000-1013&author=O.+M.+Okunola-Bakareauthor=J.+Caoauthor=T.+Kopajticauthor=L.+J.+Katzauthor=C.+J.+Lolandauthor=L.+Shiauthor=A.+H.+Newman&title=Elucidation+of+structural+elements+for+selectivity+across+monoamine+transporters%3A+novel+2-%5B%28diphenylmethyl%29sulfynil%5Dacetamide+%28modafinil%29+analogues&doi=10.1021%2Fjm401754x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm401754x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401754x%26sid%3Dliteratum%253Aachs%26aulast%3DOkunola-Bakare%26aufirst%3DO.%2BM.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DKopajtic%26aufirst%3DT.%26aulast%3DKatz%26aufirst%3DL.%2BJ.%26aulast%3DLoland%26aufirst%3DC.%2BJ.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DElucidation%2520of%2520structural%2520elements%2520for%2520selectivity%2520across%2520monoamine%2520transporters%253A%2520novel%25202-%255B%2528diphenylmethyl%2529sulfynil%255Dacetamide%2520%2528modafinil%2529%2520analogues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1000%26epage%3D1013%26doi%3D10.1021%2Fjm401754x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slack, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bakare, O. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burzynski, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rais, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slusher, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopajtic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonifazi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellenberger, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yano, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bi, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xi, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loland, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span> </span><span class="NLM_article-title">Novel and high affinity 2-[(diphenylmethyl)sulfynil]acetamide (modafinil) analogues as atypical dopamine transporter inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">10676</span><span class="NLM_x">–</span> <span class="NLM_lpage">10691</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFelu7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10676-10691&author=J.+Caoauthor=R.+D.+Slackauthor=O.+M.+Bakareauthor=C.+Burzynskiauthor=R.+Raisauthor=B.+S.+Slusherauthor=T.+Kopajticauthor=A.+Bonifaziauthor=M.+P.+Ellenbergerauthor=H.+Yanoauthor=Y.+Heauthor=G.+H.+Biauthor=Z.+H.+Xiauthor=C.+J.+Lolandauthor=A.+H.+Newman&title=Novel+and+high+affinity+2-%5B%28diphenylmethyl%29sulfynil%5Dacetamide+%28modafinil%29+analogues+as+atypical+dopamine+transporter+inhibitors&doi=10.1021%2Facs.jmedchem.6b01373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Novel and High Affinity 2-[(Diphenylmethyl)sulfinyl]acetamide (Modafinil) Analogues as Atypical Dopamine Transporter Inhibitors</span></div><div class="casAuthors">Cao, Jianjing; Slack, Rachel D.; Bakare, Oluyomi M.; Burzynski, Caitlin; Rais, Rana; Slusher, Barbara S.; Kopajtic, Theresa; Bonifazi, Alessandro; Ellenberger, Michael P.; Yano, Hideaki; He, Yi; Bi, Guo-Hua; Xi, Zheng-Xiong; Loland, Claus J.; Newman, Amy Hauck</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10676-10691</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of pharmacotherapeutic treatments of psychostimulant abuse has remained a challenge, despite significant efforts made toward relevant mechanistic targets, such as the dopamine transporter (DAT).  The atypical DAT inhibitors have received attention due to their promising pharmacol. profiles in animal models of cocaine and methamphetamine abuse.  Herein the authors report a series of modafinil analogs that have an atypical DAT inhibitor profile.  The authors extended SAR by chem. manipulating the oxidn. states of the sulfoxide and the amide functional groups, halogenating the Ph rings, and/or functionalizing the terminal nitrogen with substituted piperazines, resulting in several novel leads such as 1-(4-(2-((bis(4-fluorophenyl)methyl)sulfinyl)ethyl)piperazin-1-yl)-3-phenylpropan-2-ol (compd. 11b), which demonstrated high DAT affinity (Ki = 2.5 nM) and selectivity without producing concomitant locomotor stimulation in mice, as compared to cocaine.  These results are consistent with an atypical DAT inhibitor profile and suggest that 11b may be a potential lead for development as a psychostimulant abuse medication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL0BwESHNuRbVg90H21EOLACvtfcHk0lj8RVF31n003g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFelu7bP&md5=e912f256c764249ad606d83ac9c570fb</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01373%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DSlack%26aufirst%3DR.%2BD.%26aulast%3DBakare%26aufirst%3DO.%2BM.%26aulast%3DBurzynski%26aufirst%3DC.%26aulast%3DRais%26aufirst%3DR.%26aulast%3DSlusher%26aufirst%3DB.%2BS.%26aulast%3DKopajtic%26aufirst%3DT.%26aulast%3DBonifazi%26aufirst%3DA.%26aulast%3DEllenberger%26aufirst%3DM.%2BP.%26aulast%3DYano%26aufirst%3DH.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DBi%26aufirst%3DG.%2BH.%26aulast%3DXi%26aufirst%3DZ.%2BH.%26aulast%3DLoland%26aufirst%3DC.%2BJ.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26atitle%3DNovel%2520and%2520high%2520affinity%25202-%255B%2528diphenylmethyl%2529sulfynil%255Dacetamide%2520%2528modafinil%2529%2520analogues%2520as%2520atypical%2520dopamine%2520transporter%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10676%26epage%3D10691%26doi%3D10.1021%2Facs.jmedchem.6b01373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Beuming, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kniazeff, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergmann, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gracia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raniszewska, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Javitch, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gether, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loland, C. J.</span><span> </span><span class="NLM_article-title">The binding sites for cocaine and dopamine in the dopamine transporter overlap</span> <span class="citation_source-journal">Nat. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">780</span><span class="NLM_x">–</span> <span class="NLM_lpage">789</span><span class="refDoi"> DOI: 10.1038/nn.2146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1038%2Fnn.2146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=18568020" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1cXns1Omsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2008&pages=780-789&author=T.+Beumingauthor=J.+Kniazeffauthor=M.+L.+Bergmannauthor=L.+Shiauthor=L.+Graciaauthor=K.+Raniszewskaauthor=A.+H.+Newmanauthor=J.+A.+Javitchauthor=H.+Weinsteinauthor=U.+Getherauthor=C.+J.+Loland&title=The+binding+sites+for+cocaine+and+dopamine+in+the+dopamine+transporter+overlap&doi=10.1038%2Fnn.2146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The binding sites for cocaine and dopamine in the dopamine transporter overlap</span></div><div class="casAuthors">Beuming, Thijs; Kniazeff, Julie; Bergmann, Marianne L.; Shi, Lei; Gracia, Luis; Raniszewska, Klaudia; Newman, Amy Hauck; Javitch, Jonathan A.; Weinstein, Harel; Gether, Ulrik; Loland, Claus J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">780-789</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Beuming and colleagues detd. that the binding site for cocaine overlaps with that of dopamine on the dopamine transporter.  Detailed modeling and mutagenesis expts. revealed that this site is deeply buried amongst several transmembrane domains.  Cocaine is a widely abused substance with psychostimulant effects that are attributed to inhibition of the dopamine transporter (DAT).  We present mol. models for DAT binding of cocaine and cocaine analogs constructed from the high-resoln. structure of the bacterial transporter homolog LeuT.  Our models suggest that the binding site for cocaine and cocaine analogs is deeply buried between transmembrane segments 1, 3, 6 and 8, and overlaps with the binding sites for the substrates dopamine and amphetamine, as well as for benztropine-like DAT inhibitors.  We validated our models by detailed mutagenesis and by trapping the radiolabeled cocaine analog [3H]CFT in the transporter, either by crosslinking engineered cysteines or with an engineered Zn2+-binding site that was situated extracellularly to the predicted common binding pocket.  Our data demonstrate the mol. basis for the competitive inhibition of dopamine transport by cocaine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogCWenpzRZ47Vg90H21EOLACvtfcHk0lj8RVF31n003g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXns1Omsrw%253D&md5=7f96344e92478939540d84a37eaf3a70</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnn.2146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.2146%26sid%3Dliteratum%253Aachs%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DKniazeff%26aufirst%3DJ.%26aulast%3DBergmann%26aufirst%3DM.%2BL.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DGracia%26aufirst%3DL.%26aulast%3DRaniszewska%26aufirst%3DK.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DJavitch%26aufirst%3DJ.%2BA.%26aulast%3DWeinstein%26aufirst%3DH.%26aulast%3DGether%26aufirst%3DU.%26aulast%3DLoland%26aufirst%3DC.%2BJ.%26atitle%3DThe%2520binding%2520sites%2520for%2520cocaine%2520and%2520dopamine%2520in%2520the%2520dopamine%2520transporter%2520overlap%26jtitle%3DNat.%2520Neurosci.%26date%3D2008%26volume%3D11%26spage%3D780%26epage%3D789%26doi%3D10.1038%2Fnn.2146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Saha, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Partilla, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehner, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seddik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockner, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandtner, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitte, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumann, M. H.</span><span> </span><span class="NLM_article-title">Second-generation” mephedrone analogs, 4-MEC and 4-MePPP, differentially affect monoamine transporter function</span> <span class="citation_source-journal">Neuropsychopharmacology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">1321</span><span class="NLM_x">–</span> <span class="NLM_lpage">1331</span><span class="refDoi"> DOI: 10.1038/npp.2014.325</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1038%2Fnpp.2014.325" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2015&pages=1321-1331&author=K.+Sahaauthor=J.+S.+Partillaauthor=K.+R.+Lehnerauthor=A.+Seddikauthor=T.+Stocknerauthor=M.+Holyauthor=W.+Sandtnerauthor=G.+F.+Eckerauthor=H.+H.+Sitteauthor=M.+H.+Baumann&title=Second-generation%E2%80%9D+mephedrone+analogs%2C+4-MEC+and+4-MePPP%2C+differentially+affect+monoamine+transporter+function&doi=10.1038%2Fnpp.2014.325"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnpp.2014.325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnpp.2014.325%26sid%3Dliteratum%253Aachs%26aulast%3DSaha%26aufirst%3DK.%26aulast%3DPartilla%26aufirst%3DJ.%2BS.%26aulast%3DLehner%26aufirst%3DK.%2BR.%26aulast%3DSeddik%26aufirst%3DA.%26aulast%3DStockner%26aufirst%3DT.%26aulast%3DHoly%26aufirst%3DM.%26aulast%3DSandtner%26aufirst%3DW.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DSitte%26aufirst%3DH.%2BH.%26aulast%3DBaumann%26aufirst%3DM.%2BH.%26atitle%3DSecond-generation%25E2%2580%259D%2520mephedrone%2520analogs%252C%25204-MEC%2520and%25204-MePPP%252C%2520differentially%2520affect%2520monoamine%2520transporter%2520function%26jtitle%3DNeuropsychopharmacology%26date%3D2015%26volume%3D40%26spage%3D1321%26epage%3D1331%26doi%3D10.1038%2Fnpp.2014.325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Sitte, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reither, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boehm, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singer, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pifl, C.</span><span> </span><span class="NLM_article-title">Carrier-mediated release, transport rates, and charge transfer induced by amphetamine, tyramine, and dopamine in mammalian cells transfected with the human dopamine transporter</span> <span class="citation_source-journal">J. Neurochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">1289</span><span class="NLM_x">–</span> <span class="NLM_lpage">1297</span><span class="refDoi"> DOI: 10.1046/j.1471-4159.1998.71031289.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1046%2Fj.1471-4159.1998.71031289.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=9721755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADyaK1cXls1Khtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=1998&pages=1289-1297&author=H.+H.+Sitteauthor=S.+Huckauthor=H.+Reitherauthor=S.+Boehmauthor=E.+A.+Singerauthor=C.+Pifl&title=Carrier-mediated+release%2C+transport+rates%2C+and+charge+transfer+induced+by+amphetamine%2C+tyramine%2C+and+dopamine+in+mammalian+cells+transfected+with+the+human+dopamine+transporter&doi=10.1046%2Fj.1471-4159.1998.71031289.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Carrier-mediated release, transport rates, and charge transfer induced by amphetamine, tyramine, and dopamine in mammalian cells transfected with the human dopamine transporter</span></div><div class="casAuthors">Sitte, H. H.; Huck, S.; Reither, H.; Boehm, S.; Singer, E. A.; Pifl, C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1289-1297</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Lippincott-Raven Publishers</span>)
        </div><div class="casAbstract">Amphetamine and related substances induce dopamine release.  According to a traditional explanation, this dopamine release occurs in exchange for amphetamine by the dopamine transporter (DAT).  The authors tested this hypothesis in human embryonic kidney 293 cells stably transfected with the human DAT by measuring the uptake of dopamine, tyramine, and D- and L-amphetamine as well as substrate-induced release of preloaded N-methyl-4-[3H]phenylpyridinium ([3H]MPP+).  The uptake of substrates was sodium-dependent and was inhibited by ouabain and cocaine, which also prevented substrate-induced release of MPP+.  Patch-clamp recordings revealed that all four substrates elicited voltage-dependent inward currents (on top of constitutive leak currents) that were prevented by cocaine.  Whereas individual substrates had similar affinities in release, uptake, and patch-clamp expts., maximal effects displayed remarkable differences.  Hence, maximal effects in release and current induction were ∼25% higher for D-amphetamine as compared with the other substrates.  By contrast, dopamine was the most efficacious substrate in uptake expts., with its maximal initial uptake rate exceeding those of amphetamine and tyramine by factors of 20 and 4, resp.  The authors' expts. indicate a poor correlation between substrate-induced release and the transport of substrates, whereas the ability of substrates to induce currents correlates well with their releasing action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYoVL6P5Eh27Vg90H21EOLACvtfcHk0lgnwwEZjWknqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXls1Khtbs%253D&md5=89cf153f292293927320301b2ec49b70</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1046%2Fj.1471-4159.1998.71031289.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1471-4159.1998.71031289.x%26sid%3Dliteratum%253Aachs%26aulast%3DSitte%26aufirst%3DH.%2BH.%26aulast%3DHuck%26aufirst%3DS.%26aulast%3DReither%26aufirst%3DH.%26aulast%3DBoehm%26aufirst%3DS.%26aulast%3DSinger%26aufirst%3DE.%2BA.%26aulast%3DPifl%26aufirst%3DC.%26atitle%3DCarrier-mediated%2520release%252C%2520transport%2520rates%252C%2520and%2520charge%2520transfer%2520induced%2520by%2520amphetamine%252C%2520tyramine%252C%2520and%2520dopamine%2520in%2520mammalian%2520cells%2520transfected%2520with%2520the%2520human%2520dopamine%2520transporter%26jtitle%3DJ.%2520Neurochem.%26date%3D1998%26volume%3D71%26spage%3D1289%26epage%3D1297%26doi%3D10.1046%2Fj.1471-4159.1998.71031289.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Sarker, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissensteiner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steiner, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitte, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freissmuth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sucic, S.</span><span> </span><span class="NLM_article-title">The high-affinity binding site for tricyclic antidepressants resides in the outer vestibule of the serotonin transporter</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">1026</span><span class="NLM_x">–</span> <span class="NLM_lpage">1035</span><span class="refDoi"> DOI: 10.1124/mol.110.067538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1124%2Fmol.110.067538" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2010&pages=1026-1035&author=S.+Sarkerauthor=R.+Weissensteinerauthor=I.+Steinerauthor=H.+H.+Sitteauthor=G.+F.+Eckerauthor=M.+Freissmuthauthor=S.+Sucic&title=The+high-affinity+binding+site+for+tricyclic+antidepressants+resides+in+the+outer+vestibule+of+the+serotonin+transporter&doi=10.1124%2Fmol.110.067538"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1124%2Fmol.110.067538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.110.067538%26sid%3Dliteratum%253Aachs%26aulast%3DSarker%26aufirst%3DS.%26aulast%3DWeissensteiner%26aufirst%3DR.%26aulast%3DSteiner%26aufirst%3DI.%26aulast%3DSitte%26aufirst%3DH.%2BH.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DFreissmuth%26aufirst%3DM.%26aulast%3DSucic%26aufirst%3DS.%26atitle%3DThe%2520high-affinity%2520binding%2520site%2520for%2520tricyclic%2520antidepressants%2520resides%2520in%2520the%2520outer%2520vestibule%2520of%2520the%2520serotonin%2520transporter%26jtitle%3DMol.%2520Pharmacol.%26date%3D2010%26volume%3D78%26spage%3D1026%26epage%3D1035%26doi%3D10.1124%2Fmol.110.067538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Segel, I. H.</span> <span class="citation_source-book">Enzyme Kinetics: Behavior and Analysis of Rapid Equilibrium and Steady-State Enzyme Systems</span>; <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1975</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1975&author=I.+H.+Segel&title=Enzyme+Kinetics%3A+Behavior+and+Analysis+of+Rapid+Equilibrium+and+Steady-State+Enzyme+Systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSegel%26aufirst%3DI.%2BH.%26btitle%3DEnzyme%2520Kinetics%253A%2520Behavior%2520and%2520Analysis%2520of%2520Rapid%2520Equilibrium%2520and%2520Steady-State%2520Enzyme%2520Systems%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D1975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Meng, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mezei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, M.</span><span> </span><span class="NLM_article-title">Molecular Docking: A powerful approach for structure-based drug discovery</span> <span class="citation_source-journal">Curr. Comput.-Aided Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">157</span><span class="refDoi"> DOI: 10.2174/157340911795677602</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.2174%2F157340911795677602" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=21534921" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsFyrsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=146-157&author=Y.+H.+Mengauthor=H.+X.+Zhangauthor=M.+Mezeiauthor=M.+Cui&title=Molecular+Docking%3A+A+powerful+approach+for+structure-based+drug+discovery&doi=10.2174%2F157340911795677602"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking: A powerful approach for structure-based drug discovery</span></div><div class="casAuthors">Meng, Xuan-Yu; Zhang, Hong-Xing; Mezei, Mihaly; Cui, Meng</div><div class="citationInfo"><span class="NLM_cas:title">Current Computer-Aided Drug Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-157</span>CODEN:
                <span class="NLM_cas:coden">CCDDAS</span>;
        ISSN:<span class="NLM_cas:issn">1573-4099</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Mol. docking has become an increasingly important tool for drug discovery.  In this review, we present a brief introduction of the available mol. docking methods, and their development and applications in drug discovery.  The relevant basic theories, including sampling algorithms and scoring functions, are summarized.  The differences in and performance of available docking software are also discussed.  Flexible receptor mol. docking approaches, esp. those including backbone flexibility in receptors, are a challenge for available docking methods.  A recently developed Local Move Monte Carlo (LMMC) based approach is introduced as a potential soln. to flexible receptor docking problems.  Three application examples of mol. docking approaches for drug discovery are provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKdxlSun_gubVg90H21EOLACvtfcHk0lgnwwEZjWknqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsFyrsLY%253D&md5=b1712e0c9091fae4c71c280b296b61f4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F157340911795677602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157340911795677602%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DY.%2BH.%26aulast%3DZhang%26aufirst%3DH.%2BX.%26aulast%3DMezei%26aufirst%3DM.%26aulast%3DCui%26aufirst%3DM.%26atitle%3DMolecular%2520Docking%253A%2520A%2520powerful%2520approach%2520for%2520structure-based%2520drug%2520discovery%26jtitle%3DCurr.%2520Comput.-Aided%2520Drug%2520Des.%26date%3D2011%26volume%3D7%26spage%3D146%26epage%3D157%26doi%3D10.2174%2F157340911795677602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Liu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kokubo, H.</span><span> </span><span class="NLM_article-title">Exploring the stability of ligand binding modes to proteins by molecular dynamics simulations</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">201</span><span class="NLM_x">–</span> <span class="NLM_lpage">211</span><span class="refDoi"> DOI: 10.1007/s10822-016-0005-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1007%2Fs10822-016-0005-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=28074360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsl2jsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=201-211&author=K.+Liuauthor=E.+Watanabeauthor=H.+Kokubo&title=Exploring+the+stability+of+ligand+binding+modes+to+proteins+by+molecular+dynamics+simulations&doi=10.1007%2Fs10822-016-0005-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the stability of ligand binding modes to proteins by molecular dynamics simulations</span></div><div class="casAuthors">Liu, Kai; Watanabe, Etsurou; Kokubo, Hironori</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-211</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The binding mode prediction is of great importance to structure-based drug design.  The discrimination of various binding poses of ligand generated by docking is a great challenge not only to docking score functions but also to the relatively expensive free energy calcn. methods.  Here we systematically analyzed the stability of various ligand poses under mol. dynamics (MD) simulation.  First, a data set of 120 complexes was built based on the typical physicochem. properties of drug-like ligands.  Three potential binding poses (one correct pose and two decoys) were selected for each ligand from self-docking in addn. to the exptl. pose.  Then, five independent MD simulations for each pose were performed with different initial velocities for the statistical anal.  Finally, the stabilities of ligand poses under MD were evaluated and compared with the native one from crystal structure.  We found that about 94% of the native poses were maintained stable during the simulations, which suggests that MD simulations are accurate enough to judge most exptl. binding poses as stable properly.  Interestingly, incorrect decoy poses were maintained much less and 38-44% of decoys could be excluded just by performing equil. MD simulations, though 56-62% of decoys were stable.  The computationally-heavy binding free energy calcn. can be performed only for these survived poses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfcZT8myW_ZbVg90H21EOLACvtfcHk0lgnwwEZjWknqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsl2jsQ%253D%253D&md5=26e962221ef86fa1cab0ec25bf89a16d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2Fs10822-016-0005-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-016-0005-2%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DWatanabe%26aufirst%3DE.%26aulast%3DKokubo%26aufirst%3DH.%26atitle%3DExploring%2520the%2520stability%2520of%2520ligand%2520binding%2520modes%2520to%2520proteins%2520by%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2017%26volume%3D31%26spage%3D201%26epage%3D211%26doi%3D10.1007%2Fs10822-016-0005-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Sakano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahamood, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujitani, H.</span><span> </span><span class="NLM_article-title">Molecular dynamics analysis to evaluate docking pose prediction</span> <span class="citation_source-journal">Biophys. Physicobiology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">194</span><span class="refDoi"> DOI: 10.2142/biophysico.13.0_181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.2142%2Fbiophysico.13.0_181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=27924273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A280%3ADC%252BC2sjktFOlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=181-194&author=T.+Sakanoauthor=M.+I.+Mahamoodauthor=T.+Yamashitaauthor=H.+Fujitani&title=Molecular+dynamics+analysis+to+evaluate+docking+pose+prediction&doi=10.2142%2Fbiophysico.13.0_181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics analysis to evaluate docking pose prediction</span></div><div class="casAuthors">Sakano Takako; Mahamood Md Iqbal; Yamashita Takefumi; Fujitani Hideaki</div><div class="citationInfo"><span class="NLM_cas:title">Biophysics and physicobiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">181-194</span>
        ISSN:<span class="NLM_cas:issn">2189-4779</span>.
    </div><div class="casAbstract">The accurate prediction of a ligand-protein complex structure is important for computer-assisted drug development.  Although many docking methods have been developed over the last three decades, the success of binding structure prediction remains greatly limited.  The purpose of this study was to demonstrate the usefulness of molecular dynamics (MD) simulation in assessing a docking pose predicted using a docking program.  If the predicted pose is not unstable in an aqueous environment, MD simulation equilibrates the system and removes the ligand from the predicted position.  Here we investigated two proteins that are important potential therapeutic targets: β2 adrenergic receptor (β2AR) and PR-Set7.  While β2AR is rigid and its ligands are very similar to the template ligand (carazolol), PR-Set7 is very flexible and its ligands vary greatly from the template ligand (histone H4 tail peptide).  On an empirical basis, we usually expect that the docking prediction is accurate when the protein is rigid and its ligands are similar to the template ligand.  The MD analyses in this study clearly suggested such a tendency.  Furthermore, we discuss the possibility that the MD simulation can predict the binding pose of a ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQbbAkhVKypBrYtC3U5jeOifW6udTcc2eaK0aeLblwxU7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjktFOlsA%253D%253D&md5=fbdf9e7fa6a1b8f4803b93a393f6b2c9</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2142%2Fbiophysico.13.0_181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2142%252Fbiophysico.13.0_181%26sid%3Dliteratum%253Aachs%26aulast%3DSakano%26aufirst%3DT.%26aulast%3DMahamood%26aufirst%3DM.%2BI.%26aulast%3DYamashita%26aufirst%3DT.%26aulast%3DFujitani%26aufirst%3DH.%26atitle%3DMolecular%2520dynamics%2520analysis%2520to%2520evaluate%2520docking%2520pose%2520prediction%26jtitle%3DBiophys.%2520Physicobiology%26date%3D2016%26volume%3D13%26spage%3D181%26epage%3D194%26doi%3D10.2142%2Fbiophysico.13.0_181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Khelashvili, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanley, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahai, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeVine, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeFabritiis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, H.</span><span> </span><span class="NLM_article-title">Spontaneous inward opening of the dopamine transporter is triggered by PIP2-regulated dynamics of the N-terminus</span> <span class="citation_source-journal">ACS Chem. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1825</span><span class="NLM_x">–</span> <span class="NLM_lpage">1837</span><span class="refDoi"> DOI: 10.1021/acschemneuro.5b00179</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschemneuro.5b00179" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht12nu73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1825-1837&author=G.+Khelashviliauthor=N.+Stanleyauthor=M.+A.+Sahaiauthor=J.+Medinaauthor=M.+V.+LeVineauthor=L.+Shiauthor=G.+DeFabritiisauthor=H.+Weinstein&title=Spontaneous+inward+opening+of+the+dopamine+transporter+is+triggered+by+PIP2-regulated+dynamics+of+the+N-terminus&doi=10.1021%2Facschemneuro.5b00179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Spontaneous Inward Opening of the Dopamine Transporter Is Triggered by PIP2-Regulated Dynamics of the N-Terminus</span></div><div class="casAuthors">Khelashvili, George; Stanley, Nathaniel; Sahai, Michelle A.; Medina, Jaime; LeVine, Michael V.; Shi, Lei; De Fabritiis, Gianni; Weinstein, Harel</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1825-1837</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the dynamic mechanism of concerted motions in a full-length mol. model of the human dopamine transporter (hDAT), a member of the neurotransmitter/sodium symporter (NSS) family, involved in state-to-state transitions underlying function.  The findings result from an anal. of unbiased atomistic mol. dynamics simulation trajectories (totaling >14 μs) of the hDAT mol. immersed in lipid membrane environments with or without phosphatidylinositol 4,5-biphosphate (PIP2) lipids.  The N-terminal region of hDAT (N-term) is shown to have an essential mechanistic role in correlated rearrangements of specific structural motifs relevant to state-to-state transitions in the hDAT.  The mechanism involves PIP2-mediated electrostatic interactions between the N-term and the intracellular loops of the transporter mol.  Quant. analyses of collective motions in the trajectories reveal that these interactions correlate with the inward-opening dynamics of hDAT and are allosterically coupled to the known functional sites of the transporter.  The obsd. large-scale motions are enabled by specific reconfiguration of the network of ionic interactions at the intracellular end of the protein.  The isomerization to the inward-facing state in hDAT is accompanied by concomitant movements in the extracellular vestibule and results in the release of an Na+ ion from the Na2 site and destabilization of the substrate dopamine in the primary substrate binding S1 site.  The dynamic mechanism emerging from the findings highlights the involvement of the PIP2-regulated interactions between the N-term and the intracellular loop 4 in the functionally relevant conformational transitions that are also similar to those found to underlie state-to-state transitions in the leucine transporter (LeuT), a prototypical bacterial homolog of the NSS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodF_0sIAe1vbVg90H21EOLACvtfcHk0ligEnh2bMr86Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht12nu73P&md5=daf0a8ccf0d1b6a8a5782917a3ceb9d1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facschemneuro.5b00179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschemneuro.5b00179%26sid%3Dliteratum%253Aachs%26aulast%3DKhelashvili%26aufirst%3DG.%26aulast%3DStanley%26aufirst%3DN.%26aulast%3DSahai%26aufirst%3DM.%2BA.%26aulast%3DMedina%26aufirst%3DJ.%26aulast%3DLeVine%26aufirst%3DM.%2BV.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DDeFabritiis%26aufirst%3DG.%26aulast%3DWeinstein%26aufirst%3DH.%26atitle%3DSpontaneous%2520inward%2520opening%2520of%2520the%2520dopamine%2520transporter%2520is%2520triggered%2520by%2520PIP2-regulated%2520dynamics%2520of%2520the%2520N-terminus%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2015%26volume%3D6%26spage%3D1825%26epage%3D1837%26doi%3D10.1021%2Facschemneuro.5b00179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Sase, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aher, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saroja, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sase, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Höger, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bakulev, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitte, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leban, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lubec, G.</span><span> </span><span class="NLM_article-title">A heterocyclic compound CE-103 inhibits dopamine reuptake and modulates dopamine transporter and dopamine D1-D3 containing receptor complexes</span> <span class="citation_source-journal">Neuropharmacology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">186</span><span class="NLM_x">–</span> <span class="NLM_lpage">196</span><span class="refDoi"> DOI: 10.1016/j.neuropharm.2015.07.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1016%2Fj.neuropharm.2015.07.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=26407764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWltLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2016&pages=186-196&author=A.+Saseauthor=Y.+D.+Aherauthor=S.+R.+Sarojaauthor=M.+K.+Ganesanauthor=S.+Saseauthor=M.+Holyauthor=H.+H%C3%B6gerauthor=V.+Bakulevauthor=G.+F.+Eckerauthor=T.+Langerauthor=H.+H.+Sitteauthor=J.+Lebanauthor=G.+Lubec&title=A+heterocyclic+compound+CE-103+inhibits+dopamine+reuptake+and+modulates+dopamine+transporter+and+dopamine+D1-D3+containing+receptor+complexes&doi=10.1016%2Fj.neuropharm.2015.07.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A heterocyclic compound CE-103 inhibits dopamine reuptake and modulates dopamine transporter and dopamine D1-D3 containing receptor complexes</span></div><div class="casAuthors">Sase, Ajinkya; Aher, Yogesh D.; Saroja, Sivaprakasam R.; Ganesan, Minu Karthika; Sase, Sunetra; Holy, Marion; Hoeger, Harald; Bakulev, Vasiliy; Ecker, Gerhard F.; Langer, Thierry; Sitte, Harald H.; Leban, Johann; Lubec, Gert</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">186-196</span>CODEN:
                <span class="NLM_cas:coden">NEPHBW</span>;
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of compds. have been reported to enhance memory via the DA system and herein a heterocyclic compd. was tested for working memory (WM) enhancement.  2-((Benzhydrylsulfinyl)methyl)thiazole (CE-103) was synthesized in a six-step synthesis.  Binding of CE-103 to the dopamine (DAT), serotonin (SERT) and norepinephrine (NET) transporters and dopamine reuptake inhibition was tested as well as blood brain permeation and a screen for GPCR targets. 60 male Sprague Dawley rats were divided into six groups: CE-103 treated 1-10 mg/kg body wt., trained (TDI) and yoked (YDI) and vehicle treated, trained (TVI) and yoked (YVI) rats.  Daily single i.p. injections for a period of 10 days were administered and rats were tested in a radial arm maze (RAM).  Hippocampi were taken 6 h following the last day of training and complexes contg. the unphosphorylated or phosphorylated dopamine transporter (DAT) and complexes contg. the D1-3 dopamine receptor subunits were detd.  CE-103 was binding to the DAT but insignificantly to SERT or NET and dopamine reuptake was blocked specifically (IC50 = 14.73 μM).  From day eight the compd. was decreasing WM errors in the RAM significantly at both doses tested as compared to the vehicle controls.  In the trained CE-103-treated group levels of the complex contg. the phosphorylated dopamine transporter (pDAT) as well as D1R were decreased while levels of complexes contg. D2R and D3R were significantly increased.  CE-103 was shown to enhance spatial WM and DA reuptake inhibition with subsequent modulation of D1-3 receptors is proposed as a possible mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGok9mWVCsTDErVg90H21EOLACvtfcHk0ligEnh2bMr86Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWltLvL&md5=294205b7102eaf775a56b012c1c1434c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2015.07.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2015.07.039%26sid%3Dliteratum%253Aachs%26aulast%3DSase%26aufirst%3DA.%26aulast%3DAher%26aufirst%3DY.%2BD.%26aulast%3DSaroja%26aufirst%3DS.%2BR.%26aulast%3DGanesan%26aufirst%3DM.%2BK.%26aulast%3DSase%26aufirst%3DS.%26aulast%3DHoly%26aufirst%3DM.%26aulast%3DH%25C3%25B6ger%26aufirst%3DH.%26aulast%3DBakulev%26aufirst%3DV.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DSitte%26aufirst%3DH.%2BH.%26aulast%3DLeban%26aufirst%3DJ.%26aulast%3DLubec%26aufirst%3DG.%26atitle%3DA%2520heterocyclic%2520compound%2520CE-103%2520inhibits%2520dopamine%2520reuptake%2520and%2520modulates%2520dopamine%2520transporter%2520and%2520dopamine%2520D1-D3%2520containing%2520receptor%2520complexes%26jtitle%3DNeuropharmacology%26date%3D2016%26volume%3D102%26spage%3D186%26epage%3D196%26doi%3D10.1016%2Fj.neuropharm.2015.07.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span>Bruker SAINT v7.68A<span class="NLM_publisher-name">Bruker AXS</span>,<span class="NLM_x"> </span><span class="NLM_year">2005–2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bruker+SAINT+v7.68ABruker+AXS%2C+2005%E2%80%932016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DBruker%2520SAINT%2520v7.68A%26pub%3DBruker%2520AXS%26date%3D2005%25E2%2580%25932016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Sheldrick, G. M.</span>SADABS; <span class="NLM_publisher-name">University of Göttingen</span>: <span class="NLM_publisher-loc">Göttingen, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">1996</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=G.+M.+Sheldrick&title=SADABS"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26jtitle%3DSADABS%26pub%3DUniversity%2520of%2520G%25C3%25B6ttingen%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Dolomanov, O. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourhis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gildea, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, J. A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puschmann, H.</span><span> </span><span class="NLM_article-title">OLEX2</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">341</span><span class="refDoi"> DOI: 10.1107/S0021889808042726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1107%2FS0021889808042726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsFSnsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2009&pages=339-341&author=O.+V.+Dolomanovauthor=L.+J.+Bourhisauthor=R.+J.+Gildeaauthor=J.+A.+K.+Howardauthor=H.+Puschmann&title=OLEX2&doi=10.1107%2FS0021889808042726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">OLEX2: a complete structure solution, refinement and analysis program</span></div><div class="casAuthors">Dolomanov, Oleg V.; Bourhis, Luc J.; Gildea, Richard J.; Howard, Judith A. K.; Puschmann, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">339-341</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">New software, OLEX2, was developed for the detn., visualization and anal. of mol. crystal structures.  The software has a portable mouse-driven workflow-oriented and fully comprehensive graphical user interface for structure soln., refinement and report generation, as well as novel tools for structure anal.  OLEX2 seamlessly links all aspects of the structure soln., refinement and publication process and presents them in a single workflow-driven package, with the ultimate goal of producing an application which will be useful to both chemists and crystallographers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIYSUbyzIgxbVg90H21EOLACvtfcHk0lgcHkDI4WfTYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsFSnsbg%253D&md5=7a2f9697b13b8e76de0e7dc5074ab229</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0021889808042726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889808042726%26sid%3Dliteratum%253Aachs%26aulast%3DDolomanov%26aufirst%3DO.%2BV.%26aulast%3DBourhis%26aufirst%3DL.%2BJ.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DHoward%26aufirst%3DJ.%2BA.%2BK.%26aulast%3DPuschmann%26aufirst%3DH.%26atitle%3DOLEX2%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2009%26volume%3D42%26spage%3D339%26epage%3D341%26doi%3D10.1107%2FS0021889808042726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Huebschle, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheldrick, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dittrich, B.</span><span> </span><span class="NLM_article-title">ShelXle: a Qt graphical user interface for SHELXL</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1281</span><span class="NLM_x">–</span> <span class="NLM_lpage">1284</span><span class="refDoi"> DOI: 10.1107/S0021889811043202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1107%2FS0021889811043202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=22477785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFSisrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2011&pages=1281-1284&author=C.+B.+Huebschleauthor=G.+M.+Sheldrickauthor=B.+Dittrich&title=ShelXle%3A+a+Qt+graphical+user+interface+for+SHELXL&doi=10.1107%2FS0021889811043202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">ShelXle: a Qt graphical user interface for SHELXL</span></div><div class="casAuthors">Huebschle, Christian B.; Sheldrick, George M.; Dittrich, Birger</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1281-1284</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">ShelXle is a user-friendly graphical user interface for SHELXL.  It combines an editor with syntax highlighting for SHELXL-assocd. files with an interactive graphical display for visualization of a three-dimensional structure.  ShelXle is a graphical user interface for SHELXL [Sheldrick, G. M. (2008). Acta Cryst. A64, 112-122], currently the most widely used program for small-mol. structure refinement.  It combines an editor with syntax highlighting for the SHELXL-assocd. (input) and (output) files with an interactive graphical display for visualization of a three-dimensional structure including the electron d. (Fo) and difference d. (Fo-Fc) maps.  Special features of ShelXle include intuitive atom (re-)naming, a strongly coupled editor, structure visualization in various mono and stereo modes, and a novel way of displaying disorder extending over special positions.  ShelXle is completely compatible with all features of SHELXL and is written entirely in C++ using the Qt4 and FFTW libraries.  It is available at no cost for Windows, Linux and Mac-OS X and as source code.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow_d-mylopt7Vg90H21EOLACvtfcHk0lgcHkDI4WfTYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFSisrrF&md5=b9c11983c90116eaf0720fcdbf061ff3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0021889811043202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889811043202%26sid%3Dliteratum%253Aachs%26aulast%3DHuebschle%26aufirst%3DC.%2BB.%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26aulast%3DDittrich%26aufirst%3DB.%26atitle%3DShelXle%253A%2520a%2520Qt%2520graphical%2520user%2520interface%2520for%2520SHELXL%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2011%26volume%3D44%26spage%3D1281%26epage%3D1284%26doi%3D10.1107%2FS0021889811043202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Sheldrick, G. M.</span> <span class="citation_source-book">SHELXS</span>; <span class="NLM_publisher-name">University of Göttingen</span>: <span class="NLM_publisher-loc">Göttingen, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">1996</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=G.+M.+Sheldrick&title=SHELXS"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26btitle%3DSHELXS%26pub%3DUniversity%2520of%2520G%25C3%25B6ttingen%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Sheldrick, G. M.</span> <span class="citation_source-book">SHELXL</span>; <span class="NLM_publisher-name">University of Göttingen</span>: <span class="NLM_publisher-loc">Göttingen, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">1996</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=G.+M.+Sheldrick&title=SHELXL"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26btitle%3DSHELXL%26pub%3DUniversity%2520of%2520G%25C3%25B6ttingen%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Spek, A. L.</span><span> </span><span class="NLM_article-title">Structure validation in chemical crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">148</span><span class="NLM_x">–</span> <span class="NLM_lpage">155</span><span class="refDoi"> DOI: 10.1107/S090744490804362X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1107%2FS090744490804362X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19171970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=148-155&author=A.+L.+Spek&title=Structure+validation+in+chemical+crystallography&doi=10.1107%2FS090744490804362X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure validation in chemical crystallography</span></div><div class="casAuthors">Spek, Anthony L.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">148-155</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Automated structure validation was introduced in chem. crystallog. about 12 years ago as a tool to assist practitioners with the exponential growth in crystal structure analyses.  Validation has since evolved into an easy-to-use checkCIF/PLATON web-based IUCr service.  The result of a crystal structure detn. has to be supplied as a CIF-formatted computer-readable file.  The checking software tests the data in the CIF for completeness, quality, and consistency.  In addn., the reported structure is checked for incomplete anal., errors in the anal., and relevant issues to be verified.  A validation report is generated in the form of a list of ALERTS on the issues to be cor., checked, or commented on.  Structure validation has largely eliminated obvious problems with structure reports published in IUCr journals, such as refinement in a space group of too low symmetry.  This paper reports on the current status of structure validation and possible future extensions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8nVxgyQYutbVg90H21EOLACvtfcHk0lgcHkDI4WfTYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWhtL8%253D&md5=1cc05cd996f846a98b55ea09289af6a9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS090744490804362X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS090744490804362X%26sid%3Dliteratum%253Aachs%26aulast%3DSpek%26aufirst%3DA.%2BL.%26atitle%3DStructure%2520validation%2520in%2520chemical%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D148%26epage%3D155%26doi%3D10.1107%2FS090744490804362X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Sucic, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dallinger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zdrazil, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissensteiner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kudlacek, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cha, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gether, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freissmuth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitte, H. H.</span><span> </span><span class="NLM_article-title">The N-terminus of monoamine transporters is a lever required for the action of amphetamines</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">10924</span><span class="NLM_x">–</span> <span class="NLM_lpage">10938</span><span class="refDoi"> DOI: 10.1074/jbc.M109.083154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1074%2Fjbc.M109.083154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=20118234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFaisL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=10924-10938&author=S.+Sucicauthor=S.+Dallingerauthor=B.+Zdrazilauthor=R.+Weissensteinerauthor=T.+N.+Jorgensenauthor=M.+Holyauthor=O.+Kudlacekauthor=S.+Seidelauthor=J.+H.+Chaauthor=U.+Getherauthor=A.+H.+Newmanauthor=G.+F.+Eckerauthor=M.+Freissmuthauthor=H.+H.+Sitte&title=The+N-terminus+of+monoamine+transporters+is+a+lever+required+for+the+action+of+amphetamines&doi=10.1074%2Fjbc.M109.083154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The N Terminus of Monoamine Transporters Is a Lever Required for the Action of Amphetamines</span></div><div class="casAuthors">Sucic, Sonja; Dallinger, Stefan; Zdrazil, Barbara; Weissensteiner, Rene; Jorgensen, Trine N.; Holy, Marion; Kudlacek, Oliver; Seidel, Stefan; Cha, Joo Hwan; Gether, Ulrik; Newman, Amy H.; Ecker, Gerhard F.; Freissmuth, Michael; Sitte, Harald H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">10924-10938</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The serotonin transporter (SERT) terminates neurotransmission by removing serotonin from the synaptic cleft.  In addn., it is the site of action of antidepressants (which block the transporter) and of amphetamines (which induce substrate efflux).  We explored the functional importance of the N terminus in mediating the action of amphetamines by focusing initially on the highly conserved threonine residue at position 81, a candidate site for phosphorylation by protein kinase C.  Mol. dynamics simulations of the wild type SERT, compared with its mutations SERTT81A and SERTT81D, suggested structural changes in the inner vestibule indicative of an opening of the inner vestibule.  Predictions from this model (e.g. the preferential accumulation of SERTT81A in the inward conformation, its reduced turnover no., and a larger distance between its N and C termini) were verified.  Most importantly, SERTT81A (and the homologous mutations in noradrenaline and dopamine) failed to support amphetamine-induced efflux, and this was not remedied by aspartate at this position.  Amphetamine-induced currents through SERTT81A were comparable with those through the wild type transporter.  Both abundant Na+ entry and accumulation of SERTT81A in the inward facing conformation ought to favor amphetamine-induced efflux.  Thus, we surmised that the N terminus must play a direct role in driving the transporter into a state that supports amphetamine-induced efflux.  This hypothesis was verified by truncating the first 64 amino acids and by tethering the N terminus to an addnl. transmembrane helix.  Either modification abolished amphetamine-induced efflux.  We therefore conclude that the N terminus of monoamine transporters acts as a lever that sustains reverse transport.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBoS2MkOsfgLVg90H21EOLACvtfcHk0ljsbI46-43EvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFaisL4%253D&md5=4e93e69edd41b5eab9a8b3d6502a6b0d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.083154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.083154%26sid%3Dliteratum%253Aachs%26aulast%3DSucic%26aufirst%3DS.%26aulast%3DDallinger%26aufirst%3DS.%26aulast%3DZdrazil%26aufirst%3DB.%26aulast%3DWeissensteiner%26aufirst%3DR.%26aulast%3DJorgensen%26aufirst%3DT.%2BN.%26aulast%3DHoly%26aufirst%3DM.%26aulast%3DKudlacek%26aufirst%3DO.%26aulast%3DSeidel%26aufirst%3DS.%26aulast%3DCha%26aufirst%3DJ.%2BH.%26aulast%3DGether%26aufirst%3DU.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DFreissmuth%26aufirst%3DM.%26aulast%3DSitte%26aufirst%3DH.%2BH.%26atitle%3DThe%2520N-terminus%2520of%2520monoamine%2520transporters%2520is%2520a%2520lever%2520required%2520for%2520the%2520action%2520of%2520amphetamines%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D10924%26epage%3D10938%26doi%3D10.1074%2Fjbc.M109.083154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goehring, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penmatsa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ressler, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gouaux, E.</span><span> </span><span class="NLM_article-title">Structural basis for action by diverse antidepressants on biogenic amine transporters</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">503</span><span class="NLM_x">, </span> <span class="NLM_fpage">141</span><span class="NLM_x">–</span> <span class="NLM_lpage">145</span><span class="refDoi"> DOI: 10.1038/nature12648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1038%2Fnature12648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=24121440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSnsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=141-145&author=H.+Wangauthor=A.+Goehringauthor=K.+H.+Wangauthor=A.+Penmatsaauthor=R.+Resslerauthor=E.+Gouaux&title=Structural+basis+for+action+by+diverse+antidepressants+on+biogenic+amine+transporters&doi=10.1038%2Fnature12648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for action by diverse antidepressants on biogenic amine transporters</span></div><div class="casAuthors">Wang, Hui; Goehring, April; Wang, Kevin H.; Penmatsa, Aravind; Ressler, Ryan; Gouaux, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7474</span>),
    <span class="NLM_cas:pages">141-145</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The biogenic amine transporters (BATs) regulate endogenous neurotransmitter concns. and are targets for a broad range of therapeutic agents including selective serotonin reuptake inhibitors (SSRIs), serotonin-noradrenaline reuptake inhibitors (SNRIs) and tricyclic antidepressants (TCAs).  Because eukaryotic BATs are recalcitrant to crystallog. anal., the authors' understanding of the mechanism of these inhibitors and antidepressants is limited.  LeuT is a bacterial homolog of BATs and has proven to be a valuable paradigm for understanding relationships between their structure and function.  However, because only approx. 25% of the amino acid sequence of LeuT is in common with that of BATs, and as LeuT is a promiscuous amino acid transporter, it does not recapitulate the pharmacol. properties of BATs.  Indeed, SSRIs and TCAs bind in the extracellular vestibule of LeuT and act as non-competitive inhibitors of transport.  By contrast, multiple studies demonstrate that both TCAs and SSRIs are competitive inhibitors for eukaryotic BATs and bind to the primary binding pocket.  Here the authors engineered LeuT to harbor human BAT-like pharmacol. by mutating key residues around the primary binding pocket.  The final LeuBAT mutant binds the SSRI sertraline with a binding const. of 18 nM and displays high-affinity binding to a range of SSRIs, SNRIs and a TCA.  The authors detd. 12 crystal structures of LeuBAT in complex with four classes of antidepressants.  The chem. diverse inhibitors have a remarkably similar mode of binding in which they straddle transmembrane helix (TM) 3, wedge between TM3/TM8 and TM1/TM6, and lock the transporter in a sodium- and chloride-bound outward-facing open conformation.  Together, these studies define common and simple principles for the action of SSRIs, SNRIs and TCAs on BATs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8AMnfh2-Af7Vg90H21EOLACvtfcHk0ljsbI46-43EvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSnsLrE&md5=26cc6cf173fdf20ec20d928d02f05193</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnature12648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12648%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DGoehring%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DK.%2BH.%26aulast%3DPenmatsa%26aufirst%3DA.%26aulast%3DRessler%26aufirst%3DR.%26aulast%3DGouaux%26aufirst%3DE.%26atitle%3DStructural%2520basis%2520for%2520action%2520by%2520diverse%2520antidepressants%2520on%2520biogenic%2520amine%2520transporters%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D141%26epage%3D145%26doi%3D10.1038%2Fnature12648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Trott, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">461</span><span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+Improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0ljsbI46-43EvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520Improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Morris, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huey, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindstrom, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belew, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">Software news and updates AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">2785</span><span class="NLM_x">–</span> <span class="NLM_lpage">2791</span><span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=Software+news+and+updates+AutoDock4+and+AutoDockTools4%3A+Automated+docking+with+selective+receptor+flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0lhNf7dNOZSfiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DSoftware%2520news%2520and%2520updates%2520AutoDock4%2520and%2520AutoDockTools4%253A%2520Automated%2520docking%2520with%2520selective%2520receptor%2520flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Lomize, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pogozheva, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosberg, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lomize, A. L.</span><span> </span><span class="NLM_article-title">OPM database and PPM web server: Resources for positioning of proteins in membranes</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x"> (</span><span class="NLM_issue">D1</span><span class="NLM_x">) </span> <span class="NLM_fpage">D370</span><span class="NLM_x">–</span> <span class="NLM_lpage">D376</span><span class="refDoi"> DOI: 10.1093/nar/gkr703</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1093%2Fnar%2Fgkr703" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=21890895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12hurzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D370-D376&issue=D1&author=M.+A.+Lomizeauthor=I.+D.+Pogozhevaauthor=H.+Jooauthor=H.+I.+Mosbergauthor=A.+L.+Lomize&title=OPM+database+and+PPM+web+server%3A+Resources+for+positioning+of+proteins+in+membranes&doi=10.1093%2Fnar%2Fgkr703"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">OPM database and PPM web server: resources for positioning of proteins in membranes</span></div><div class="casAuthors">Lomize, Mikhail A.; Pogozheva, Irina D.; Joo, Hyeon; Mosberg, Henry I.; Lomize, Andrei L.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D370-D376</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Orientations of Proteins in Membranes (OPM) database is a curated web resource that provides spatial positions of membrane-bound peptides and proteins of known three-dimensional structure in the lipid bilayer, together with their structural classification, topol. and intracellular localization.  OPM currently contains more than 1200 transmembrane and peripheral proteins and peptides from approx. 350 organisms that represent approx. 3800 Protein Data Bank entries.  Proteins are classified into classes, superfamilies and families and assigned to 21 distinct membrane types.  Spatial positions of proteins with respect to the lipid bilayer are optimized by the PPM 2.0 method that accounts for the hydrophobic, hydrogen bonding and electrostatic interactions of the proteins with the anisotropic water-lipid environment described by the dielec. const. and hydrogen-bonding profiles.  The OPM database is freely accessible at http://opm.phar.umich.edu.  Data can be sorted, searched or retrieved using the hierarchical classification, source organism, localization in different types of membranes.  The database offers downloadable coordinates of proteins and peptides with membrane boundaries.  A gallery of protein images and several visualization tools are provided.  The database is supplemented by the PPM server (http://opm.phar.umich.edu/server.php) which can be used for calcg. spatial positions in membranes of newly detd. proteins structures or theor. models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo76Y6X6BHLRbVg90H21EOLACvtfcHk0lhNf7dNOZSfiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12hurzJ&md5=b931126c56d227ab0eb6c5762c1dae6b</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr703%26sid%3Dliteratum%253Aachs%26aulast%3DLomize%26aufirst%3DM.%2BA.%26aulast%3DPogozheva%26aufirst%3DI.%2BD.%26aulast%3DJoo%26aufirst%3DH.%26aulast%3DMosberg%26aufirst%3DH.%2BI.%26aulast%3DLomize%26aufirst%3DA.%2BL.%26atitle%3DOPM%2520database%2520and%2520PPM%2520web%2520server%253A%2520Resources%2520for%2520positioning%2520of%2520proteins%2520in%2520membranes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26issue%3DD1%26spage%3DD370%26epage%3DD376%26doi%3D10.1093%2Fnar%2Fgkr703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Jo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iyer, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Im, W.</span><span> </span><span class="NLM_article-title">CHARMM-GUI: A web-based graphical user interface for CHARMM</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1859</span><span class="NLM_x">–</span> <span class="NLM_lpage">1865</span><span class="refDoi"> DOI: 10.1002/jcc.20945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1002%2Fjcc.20945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=18351591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1cXosVKksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=1859-1865&author=S.+Joauthor=T.+Kimauthor=V.+G.+Iyerauthor=W.+Im&title=CHARMM-GUI%3A+A+web-based+graphical+user+interface+for+CHARMM&doi=10.1002%2Fjcc.20945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM-GUI: a web-based graphical user interface for CHARMM</span></div><div class="casAuthors">Jo, Sunhwan; Kim, Taehoon; Iyer, Vidyashankara G.; Im, Wonpil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1859-1865</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">CHARMM is an academic research program used widely for macromol. mechanics and dynamics with versatile anal. and manipulation tools of at. coordinates and dynamics trajectories.  CHARMM-GUI, http://www.charmm-gui.org, has been developed to provide a web-based graphical user interface to generate various input files and mol. systems to facilitate and standardize the usage of common and advanced simulation techniques in CHARMM.  The web environment provides an ideal platform to build and validate a mol. model system in an interactive fashion such that, if a problem is found through visual inspection, one can go back to the previous setup and regenerate the whole system again.  In this article, we describe the currently available functional modules of CHARMM-GUI Input Generator that form a basis for the advanced simulation techniques.  Future directions of the CHARMM-GUI development project are also discussed briefly together with other features in the CHARMM-GUI website, such as Archive and Movie Gallery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7gpU-1N3WOLVg90H21EOLACvtfcHk0lhNf7dNOZSfiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXosVKksbc%253D&md5=112a3dd61d792b040f9f716b32220d7e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20945%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DIyer%26aufirst%3DV.%2BG.%26aulast%3DIm%26aufirst%3DW.%26atitle%3DCHARMM-GUI%253A%2520A%2520web-based%2520graphical%2520user%2520interface%2520for%2520CHARMM%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2008%26volume%3D29%26spage%3D1859%26epage%3D1865%26doi%3D10.1002%2Fjcc.20945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Jo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Im, W.</span><span> </span><span class="NLM_article-title">Automated builder and database of protein/membrane complexes for molecular dynamics simulations</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e880</span><span class="refDoi"> DOI: 10.1371/journal.pone.0000880</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1371%2Fjournal.pone.0000880" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=17849009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A280%3ADC%252BD2srjsFyjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=e880&author=S.+Joauthor=T.+Kimauthor=W.+Im&title=Automated+builder+and+database+of+protein%2Fmembrane+complexes+for+molecular+dynamics+simulations&doi=10.1371%2Fjournal.pone.0000880"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Automated builder and database of protein/membrane complexes for molecular dynamics simulations</span></div><div class="casAuthors">Jo Sunhwan; Kim Taehoon; Im Wonpil</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e880</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Molecular dynamics simulations of membrane proteins have provided deeper insights into their functions and interactions with surrounding environments at the atomic level.  However, compared to solvation of globular proteins, building a realistic protein/membrane complex is still challenging and requires considerable experience with simulation software.  Membrane Builder in the CHARMM-GUI website (http://www.charmm-gui.org) helps users to build such a complex system using a web browser with a graphical user interface.  Through a generalized and automated building process including system size determination as well as generation of lipid bilayer, pore water, bulk water, and ions, a realistic membrane system with virtually any kinds and shapes of membrane proteins can be generated in 5 minutes to 2 hours depending on the system size.  Default values that were elaborated and tested extensively are given in each step to provide reasonable options and starting points for both non-expert and expert users.  The efficacy of Membrane Builder is illustrated by its applications to 12 transmembrane and 3 interfacial membrane proteins, whose fully equilibrated systems with three different types of lipid molecules (DMPC, DPPC, and POPC) and two types of system shapes (rectangular and hexagonal) are freely available on the CHARMM-GUI website.  One of the most significant advantages of using the web environment is that, if a problem is found, users can go back and re-generate the whole system again before quitting the browser.  Therefore, Membrane Builder provides the intuitive and easy way to build and simulate the biologically important membrane system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAsncNNJLqIBfKb6ELK8t0fW6udTcc2eZyOs-Om6wPmLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2srjsFyjsQ%253D%253D&md5=d2384741fc1d18b33cff16b254fc006b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0000880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0000880%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DIm%26aufirst%3DW.%26atitle%3DAutomated%2520builder%2520and%2520database%2520of%2520protein%252Fmembrane%2520complexes%2520for%2520molecular%2520dynamics%2520simulations%26jtitle%3DPLoS%2520One%26date%3D2007%26volume%3D2%26spage%3De880%26doi%3D10.1371%2Fjournal.pone.0000880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Brooks, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrella, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Won, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archontis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartels, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boresch, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caflisch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caves, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinner, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feig, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodoscek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Im, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuczera, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lazaridis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ovchinnikov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paci, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pastor, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Post, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pu, J. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tidor, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venable, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodcock, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">York, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">CHARMM: The biomolecular simulation program</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1545</span><span class="NLM_x">–</span> <span class="NLM_lpage">1614</span><span class="refDoi"> DOI: 10.1002/jcc.21287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1002%2Fjcc.21287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19444816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms1Ciu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=1545-1614&author=B.+R.+Brooksauthor=C.+L.+Brooksauthor=A.+D.+Mackerellauthor=L.+Nilssonauthor=R.+J.+Petrellaauthor=B.+Rouxauthor=Y.+Wonauthor=G.+Archontisauthor=C.+Bartelsauthor=S.+Boreschauthor=A.+Caflischauthor=L.+Cavesauthor=Q.+Cuiauthor=A.+R.+Dinnerauthor=M.+Feigauthor=S.+Fischerauthor=J.+Gaoauthor=M.+Hodoscekauthor=W.+Imauthor=K.+Kuczeraauthor=T.+Lazaridisauthor=J.+Maauthor=V.+Ovchinnikovauthor=E.+Paciauthor=R.+W.+Pastorauthor=C.+B.+Postauthor=J.+Z.+Puauthor=M.+Schaeferauthor=B.+Tidorauthor=R.+M.+Venableauthor=H.+L.+Woodcockauthor=X.+Wuauthor=W.+Yangauthor=D.+M.+Yorkauthor=M.+Karplus&title=CHARMM%3A+The+biomolecular+simulation+program&doi=10.1002%2Fjcc.21287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM: The biomolecular simulation program</span></div><div class="casAuthors">Brooks, B. R.; Brooks, C. L., III; Mackerell, A. D., Jr.; Nilsson, L.; Petrella, R. J.; Roux, B.; Won, Y.; Archontis, G.; Bartels, C.; Boresch, S.; Caflisch, A.; Caves, L.; Cui, Q.; Dinner, A. R.; Feig, M.; Fischer, S.; Gao, J.; Hodoscek, M.; Im, W.; Kuczera, K.; Lazaridis, T.; Ma, J.; Ovchinnikov, V.; Paci, E.; Pastor, R. W.; Post, C. B.; Pu, J. Z.; Schaefer, M.; Tidor, B.; Venable, R. M.; Woodcock, H. L.; Wu, X.; Yang, W.; York, D. M.; Karplus, M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1545-1614</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  CHARMM (Chem. at HARvard Mol. Mechanics) is a highly versatile and widely used mol. simulation program.  It has been developed over the last three decades with a primary focus on mols. of biol. interest, including proteins, peptides, lipids, nucleic acids, carbohydrates, and small mol. ligands, as they occur in soln., crystals, and membrane environments.  For the study of such systems, the program provides a large suite of computational tools that include numerous conformational and path sampling methods, free energy estimators, mol. minimization, dynamics, and anal. techniques, and model-building capabilities.  The CHARMM program is applicable to problems involving a much broader class of many-particle systems.  Calcns. with CHARMM can be performed using a no. of different energy functions and models, from mixed quantum mech.-mol. mech. force fields, to all-atom classical potential energy functions with explicit solvent and various boundary conditions, to implicit solvent and membrane models.  The program has been ported to numerous platforms in both serial and parallel architectures.  This article provides an overview of the program as it exists today with an emphasis on developments since the publication of the original CHARMM article in 1983.  © 2009 Wiley Periodicals, Inc. J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOXZCtNIqoCbVg90H21EOLACvtfcHk0lh_pCmW7nGoxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms1Ciu70%253D&md5=2c6a2be869362d7131f5aea8411c1552</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21287%26sid%3Dliteratum%253Aachs%26aulast%3DBrooks%26aufirst%3DB.%2BR.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26aulast%3DNilsson%26aufirst%3DL.%26aulast%3DPetrella%26aufirst%3DR.%2BJ.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DWon%26aufirst%3DY.%26aulast%3DArchontis%26aufirst%3DG.%26aulast%3DBartels%26aufirst%3DC.%26aulast%3DBoresch%26aufirst%3DS.%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DCaves%26aufirst%3DL.%26aulast%3DCui%26aufirst%3DQ.%26aulast%3DDinner%26aufirst%3DA.%2BR.%26aulast%3DFeig%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DHodoscek%26aufirst%3DM.%26aulast%3DIm%26aufirst%3DW.%26aulast%3DKuczera%26aufirst%3DK.%26aulast%3DLazaridis%26aufirst%3DT.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DOvchinnikov%26aufirst%3DV.%26aulast%3DPaci%26aufirst%3DE.%26aulast%3DPastor%26aufirst%3DR.%2BW.%26aulast%3DPost%26aufirst%3DC.%2BB.%26aulast%3DPu%26aufirst%3DJ.%2BZ.%26aulast%3DSchaefer%26aufirst%3DM.%26aulast%3DTidor%26aufirst%3DB.%26aulast%3DVenable%26aufirst%3DR.%2BM.%26aulast%3DWoodcock%26aufirst%3DH.%2BL.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DYork%26aufirst%3DD.%2BM.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DCHARMM%253A%2520The%2520biomolecular%2520simulation%2520program%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D1545%26epage%3D1614%26doi%3D10.1002%2Fjcc.21287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Eastman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chodera, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radmer, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruns, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ku, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beauchamp, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shukla, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tye, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houston, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stich, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirts, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span> </span><span class="NLM_article-title">OpenMM 4: A reusable, extensible, hardware independent library for high performance molecular simulation</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">461</span><span class="NLM_x">–</span> <span class="NLM_lpage">469</span><span class="refDoi"> DOI: 10.1021/ct300857j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300857j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFektbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=461-469&author=P.+Eastmanauthor=M.+S.+Friedrichsauthor=J.+D.+Choderaauthor=R.+J.+Radmerauthor=C.+M.+Brunsauthor=J.+P.+Kuauthor=K.+A.+Beauchampauthor=T.+J.+Laneauthor=L.+P.+Wangauthor=D.+Shuklaauthor=T.+Tyeauthor=M.+Houstonauthor=T.+Stichauthor=C.+Kleinauthor=M.+R.+Shirtsauthor=V.+S.+Pande&title=OpenMM+4%3A+A+reusable%2C+extensible%2C+hardware+independent+library+for+high+performance+molecular+simulation&doi=10.1021%2Fct300857j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">OpenMM 4: A Reusable, Extensible, Hardware Independent Library for High Performance Molecular Simulation</span></div><div class="casAuthors">Eastman, Peter; Friedrichs, Mark S.; Chodera, John D.; Radmer, Randall J.; Bruns, Christopher M.; Ku, Joy P.; Beauchamp, Kyle A.; Lane, Thomas J.; Wang, Lee-Ping; Shukla, Diwakar; Tye, Tony; Houston, Mike; Stich, Timo; Klein, Christoph; Shirts, Michael R.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">461-469</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">OpenMM is a software toolkit for performing mol. simulations on a range of high performance computing architectures.  It is based on a layered architecture: the lower layers function as a reusable library that can be invoked by any application, while the upper layers form a complete environment for running mol. simulations.  The library API hides all hardware-specific dependencies and optimizations from the users and developers of simulation programs: they can be run without modification on any hardware on which the API has been implemented.  The current implementations of OpenMM include support for graphics processing units using the OpenCL and CUDA frameworks.  In addn., OpenMM was designed to be extensible, so new hardware architectures can be accommodated and new functionality (e.g., energy terms and integrators) can be easily added.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJZt8HqGLFN7Vg90H21EOLACvtfcHk0lhL0j3RigoI0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFektbbP&md5=484edfd2b97123d2e26550778b0edaaf</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fct300857j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300857j%26sid%3Dliteratum%253Aachs%26aulast%3DEastman%26aufirst%3DP.%26aulast%3DFriedrichs%26aufirst%3DM.%2BS.%26aulast%3DChodera%26aufirst%3DJ.%2BD.%26aulast%3DRadmer%26aufirst%3DR.%2BJ.%26aulast%3DBruns%26aufirst%3DC.%2BM.%26aulast%3DKu%26aufirst%3DJ.%2BP.%26aulast%3DBeauchamp%26aufirst%3DK.%2BA.%26aulast%3DLane%26aufirst%3DT.%2BJ.%26aulast%3DWang%26aufirst%3DL.%2BP.%26aulast%3DShukla%26aufirst%3DD.%26aulast%3DTye%26aufirst%3DT.%26aulast%3DHouston%26aufirst%3DM.%26aulast%3DStich%26aufirst%3DT.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DOpenMM%25204%253A%2520A%2520reusable%252C%2520extensible%252C%2520hardware%2520independent%2520library%2520for%2520high%2520performance%2520molecular%2520simulation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D461%26epage%3D469%26doi%3D10.1021%2Fct300857j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swails, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeom, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eastman, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemkul, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeong, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pande, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klauda, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Im, W.</span><span> </span><span class="NLM_article-title">CHARMM-GUI Input Generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">413</span><span class="refDoi"> DOI: 10.1021/acs.jctc.5b00935</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00935" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVWru7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=405-413&author=J.+Leeauthor=Y.+Chengauthor=J.+M.+Swailsauthor=M.+S.+Yeomauthor=P.+K.+Eastmanauthor=J.+A.+Lemkulauthor=S.+Weiauthor=J.+Bucknerauthor=J.+C.+Jeongauthor=Y.+Qiauthor=S.+Joauthor=V.+S.+Pandeauthor=D.+A.+Caseauthor=C.+L.+Brooksauthor=A.+D.+MacKerellauthor=J.+B.+Klaudaauthor=W.+Im&title=CHARMM-GUI+Input+Generator+for+NAMD%2C+GROMACS%2C+AMBER%2C+OpenMM%2C+and+CHARMM%2FOpenMM+simulations+using+the+CHARMM36+additive+force+field&doi=10.1021%2Facs.jctc.5b00935"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM-GUI Input Generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM Simulations Using the CHARMM36 Additive Force Field</span></div><div class="casAuthors">Lee, Jumin; Cheng, Xi; Swails, Jason M.; Yeom, Min Sun; Eastman, Peter K.; Lemkul, Justin A.; Wei, Shuai; Buckner, Joshua; Jeong, Jong Cheol; Qi, Yifei; Jo, Sunhwan; Pande, Vijay S.; Case, David A.; Brooks, Charles L.; MacKerell, Alexander D.; Klauda, Jeffery B.; Im, Wonpil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">405-413</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proper treatment of nonbonded interactions is essential for the accuracy of mol. dynamics (MD) simulations, esp. in studies of lipid bilayers.  The use of the CHARMM36 force field (C36 FF) in different MD simulation programs can result in disagreements with published simulations performed with CHARMM due to differences in the protocols used to treat the long-range and 1-4 nonbonded interactions.  In this study, we systematically test the use of the C36 lipid FF in NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM.  A wide range of Lennard-Jones (LJ) cutoff schemes and integrator algorithms were tested to find the optimal simulation protocol to best match bilayer properties of six lipids with varying acyl chain satn. and head groups.  MD simulations of a 1,2-dipalmitoyl-sn-phosphatidylcholine (DPPC) bilayer were used to obtain the optimal protocol for each program.  MD simulations with all programs were found to reasonably match the DPPC bilayer properties (surface area per lipid, chain order parameters, and area compressibility modulus) obtained using the std. protocol used in CHARMM as well as from expts.  The optimal simulation protocol was then applied to the other five lipid simulations and resulted in excellent agreement between results from most simulation programs as well as with exptl. data.  AMBER compared least favorably with the expected membrane properties, which appears to be due to its use of the hard-truncation in the LJ potential vs. a force-based switching function used to smooth the LJ potential as it approaches the cutoff distance.  The optimal simulation protocol for each program has been implemented in CHARMM-GUI.  This protocol is expected to be applicable to the remainder of the additive C36 FF including the proteins, nucleic acids, carbohydrates, and small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrioKQ5MoVZkrVg90H21EOLACvtfcHk0lhL0j3RigoI0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVWru7nI&md5=1e986f2c205aca8fb80442bc9c95b229</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00935&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00935%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DYeom%26aufirst%3DM.%2BS.%26aulast%3DEastman%26aufirst%3DP.%2BK.%26aulast%3DLemkul%26aufirst%3DJ.%2BA.%26aulast%3DWei%26aufirst%3DS.%26aulast%3DBuckner%26aufirst%3DJ.%26aulast%3DJeong%26aufirst%3DJ.%2BC.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DJo%26aufirst%3DS.%26aulast%3DPande%26aufirst%3DV.%2BS.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DKlauda%26aufirst%3DJ.%2BB.%26aulast%3DIm%26aufirst%3DW.%26atitle%3DCHARMM-GUI%2520Input%2520Generator%2520for%2520NAMD%252C%2520GROMACS%252C%2520AMBER%252C%2520OpenMM%252C%2520and%2520CHARMM%252FOpenMM%2520simulations%2520using%2520the%2520CHARMM36%2520additive%2520force%2520field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D405%26epage%3D413%26doi%3D10.1021%2Facs.jctc.5b00935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatcher, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kundu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darian, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guvench, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopes, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorobyov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">671</span><span class="NLM_x">–</span> <span class="NLM_lpage">690</span><span class="refDoi"> DOI: 10.1002/jcc.21367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1002%2Fjcc.21367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=19575467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=671-690&author=K.+Vanommeslaegheauthor=E.+Hatcherauthor=C.+Acharyaauthor=S.+Kunduauthor=S.+Zhongauthor=J.+Shimauthor=E.+Darianauthor=O.+Guvenchauthor=P.+Lopesauthor=I.+Vorobyovauthor=A.+D.+MacKerell&title=CHARMM+general+force+field%3A+A+force+field+for+drug-like+molecules+compatible+with+the+CHARMM+all-atom+additive+biological+force+fields&doi=10.1002%2Fjcc.21367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span></div><div class="casAuthors">Vanommeslaeghe, K.; Hatcher, E.; Acharya, C.; Kundu, S.; Zhong, S.; Shim, J.; Darian, E.; Guvench, O.; Lopes, P.; Vorobyov, I.; Mackerell, A. D., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">671-690</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The widely used CHARMM additive all-atom force field includes parameters for proteins, nucleic acids, lipids, and carbohydrates.  In the present article, an extension of the CHARMM force field to drug-like mols. is presented.  The resulting CHARMM General Force Field (CGenFF) covers a wide range of chem. groups present in biomols. and drug-like mols., including a large no. of heterocyclic scaffolds.  The parametrization philosophy behind the force field focuses on quality at the expense of transferability, with the implementation concg. on an extensible force field.  Statistics related to the quality of the parametrization with a focus on exptl. validation are presented.  Addnl., the parametrization procedure, described fully in the present article in the context of the model systems, pyrrolidine, and 3-phenoxymethyl-pyrrolidine will allow users to readily extend the force field to chem. groups that are not explicitly covered in the force field as well as add functional groups to and link together mols. already available in the force field.  CGenFF thus makes it possible to perform "all-CHARMM" simulations on drug-target interactions thereby extending the utility of CHARMM force fields to medicinally relevant systems. © 2009 Wiley Periodicals, Inc.J Comput Chem, 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCuh0CLdmxN7Vg90H21EOLACvtfcHk0lhL0j3RigoI0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D&md5=26212e0e4f73bded0c89d4b411cd3833</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21367%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DAcharya%26aufirst%3DC.%26aulast%3DKundu%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DDarian%26aufirst%3DE.%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DLopes%26aufirst%3DP.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DCHARMM%2520general%2520force%2520field%253A%2520A%2520force%2520field%2520for%2520drug-like%2520molecules%2520compatible%2520with%2520the%2520CHARMM%2520all-atom%2520additive%2520biological%2520force%2520fields%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D671%26epage%3D690%26doi%3D10.1002%2Fjcc.21367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">MacKerell, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bashford, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunbrack, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evanseck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph-McCarthy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuchnir, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuczera, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, F. T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michnick, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prodhom, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reiher, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlenkrich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stote, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Straub, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiorkiewicz-Kuczera, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">All-atom empirical potential for molecular modeling and dynamics studies of proteins</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">3586</span><span class="NLM_x">–</span> <span class="NLM_lpage">3616</span><span class="refDoi"> DOI: 10.1021/jp973084f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp973084f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADyaK1cXivVOlsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1998&pages=3586-3616&author=A.+MacKerellauthor=D.+Bashfordauthor=M.+Bellottauthor=R.+Dunbrackauthor=J.+Evanseckauthor=M.+Fieldauthor=S.+Fischerauthor=J.+Gaoauthor=H.+Guoauthor=S.+Haauthor=D.+Joseph-McCarthyauthor=L.+Kuchnirauthor=K.+Kuczeraauthor=F.+T.+K.+Lauauthor=C.+Mattosauthor=S.+Michnickauthor=T.+Ngoauthor=D.+T.+Nguyenauthor=B.+Prodhomauthor=W.+E.+Reiherauthor=B.+Rouxauthor=M.+Schlenkrichauthor=J.+C.+Smithauthor=R.+Stoteauthor=J.+Straubauthor=M.+Watanabeauthor=J.+Wiorkiewicz-Kuczeraauthor=D.+Yinauthor=M.+Karplus&title=All-atom+empirical+potential+for+molecular+modeling+and+dynamics+studies+of+proteins&doi=10.1021%2Fjp973084f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">All-Atom Empirical Potential for Molecular Modeling and Dynamics Studies of Proteins</span></div><div class="casAuthors">MacKerell, A. D., Jr.; Bashford, D.; Bellott, M.; Dunbrack, R. L.; Evanseck, J. D.; Field, M. J.; Fischer, S.; Gao, J.; Guo, H.; Ha, S.; Joseph-McCarthy, D.; Kuchnir, L.; Kuczera, K.; Lau, F. T. K.; Mattos, C.; Michnick, S.; Ngo, T.; Nguyen, D. T.; Prodhom, B.; Reiher, W. E., III; Roux, B.; Schlenkrich, M.; Smith, J. C.; Stote, R.; Straub, J.; Watanabe, M.; Wiorkiewicz-Kuczera, J.; Yin, D.; Karplus, M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3586-3616</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New protein parameters are reported for the all-atom empirical energy function in the CHARMM program.  The parameter evaluation was based on a self-consistent approach designed to achieve a balance between the internal (bonding) and interaction (nonbonding) terms of the force field and among the solvent-solvent, solvent-solute, and solute-solute interactions.  Optimization of the internal parameters used exptl. gas-phase geometries, vibrational spectra, and torsional energy surfaces supplemented with ab initio results.  The peptide backbone bonding parameters were optimized with respect to data for N-methylacetamide and the alanine dipeptide.  The interaction parameters, particularly the at. charges, were detd. by fitting ab initio interaction energies and geometries of complexes between water and model compds. that represented the backbone and the various side chains.  In addn., dipole moments, exptl. heats and free energies of vaporization, solvation and sublimation, mol. vols., and crystal pressures and structures were used in the optimization.  The resulting protein parameters were tested by applying them to noncyclic tripeptide crystals, cyclic peptide crystals, and the proteins crambin, bovine pancreatic trypsin inhibitor, and carbonmonoxy myoglobin in vacuo and in a crystal.  A detailed anal. of the relationship between the alanine dipeptide potential energy surface and calcd. protein φ, χ angles was made and used in optimizing the peptide group torsional parameters.  The results demonstrate that use of ab initio structural and energetic data by themselves are not sufficient to obtain an adequate backbone representation for peptides and proteins in soln. and in crystals.  Extensive comparisons between mol. dynamics simulation and exptl. data for polypeptides and proteins were performed for both structural and dynamic properties.  Calcd. data from energy minimization and dynamics simulations for crystals demonstrate that the latter are needed to obtain meaningful comparisons with exptl. crystal structures.  The presented parameters, in combination with the previously published CHARMM all-atom parameters for nucleic acids and lipids, provide a consistent set for condensed-phase simulations of a wide variety of mols. of biol. interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor1-3zB5qPwrVg90H21EOLACvtfcHk0liI2vp00BQRww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXivVOlsb4%253D&md5=ebb5100dafd0daeee60ca2fa66c1324a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjp973084f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp973084f%26sid%3Dliteratum%253Aachs%26aulast%3DMacKerell%26aufirst%3DA.%26aulast%3DBashford%26aufirst%3DD.%26aulast%3DBellott%26aufirst%3DM.%26aulast%3DDunbrack%26aufirst%3DR.%26aulast%3DEvanseck%26aufirst%3DJ.%26aulast%3DField%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DHa%26aufirst%3DS.%26aulast%3DJoseph-McCarthy%26aufirst%3DD.%26aulast%3DKuchnir%26aufirst%3DL.%26aulast%3DKuczera%26aufirst%3DK.%26aulast%3DLau%26aufirst%3DF.%2BT.%2BK.%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DMichnick%26aufirst%3DS.%26aulast%3DNgo%26aufirst%3DT.%26aulast%3DNguyen%26aufirst%3DD.%2BT.%26aulast%3DProdhom%26aufirst%3DB.%26aulast%3DReiher%26aufirst%3DW.%2BE.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DSchlenkrich%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26aulast%3DStote%26aufirst%3DR.%26aulast%3DStraub%26aufirst%3DJ.%26aulast%3DWatanabe%26aufirst%3DM.%26aulast%3DWiorkiewicz-Kuczera%26aufirst%3DJ.%26aulast%3DYin%26aufirst%3DD.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DAll-atom%2520empirical%2520potential%2520for%2520molecular%2520modeling%2520and%2520dynamics%2520studies%2520of%2520proteins%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1998%26volume%3D102%26spage%3D3586%26epage%3D3616%26doi%3D10.1021%2Fjp973084f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Essmann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perera, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berkowitz, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, L. G.</span><span> </span><span class="NLM_article-title">A smooth particle mesh Ewald method</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">8577</span><span class="NLM_x">–</span> <span class="NLM_lpage">8593</span><span class="refDoi"> DOI: 10.1063/1.470117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1063%2F1.470117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1995&pages=8577-8593&author=U.+Essmannauthor=L.+Pereraauthor=M.+L.+Berkowitzauthor=T.+Dardenauthor=H.+Leeauthor=L.+G.+Pedersen&title=A+smooth+particle+mesh+Ewald+method&doi=10.1063%2F1.470117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth particle mesh Ewald method</span></div><div class="casAuthors">Essmann, Ulrich; Perera, Lalith; Berkowitz, Max L.; Darden, Tom; Lee, Hsing; Pedersen, Lee G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8577-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The previously developed particle mesh Ewald method is reformulated in terms of efficient B-spline interpolation of the structure factors.  This reformulation allows a natural extension of the method to potentials of the form 1/rp with p ≥ 1.  Furthermore, efficient calcn. of the virial tensor follows.  Use of B-splines in the place of Lagrange interpolation leads to analytic gradients as well as a significant improvement in the accuracy.  The authors demonstrate that arbitrary accuracy can be achieved, independent of system size N, at a cost that scales as N log(N).  For biomol. systems with many thousands of atoms and this method permits the use of Ewald summation at a computational cost comparable to that of a simple truncation method of 10 Å or less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Bp1XlSY32LVg90H21EOLACvtfcHk0liI2vp00BQRww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D&md5=092a679dd3bee08da28df41e302383a7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1063%2F1.470117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.470117%26sid%3Dliteratum%253Aachs%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DBerkowitz%26aufirst%3DM.%2BL.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPedersen%26aufirst%3DL.%2BG.%26atitle%3DA%2520smooth%2520particle%2520mesh%2520Ewald%2520method%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D103%26spage%3D8577%26epage%3D8593%26doi%3D10.1063%2F1.470117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Michaud-Agrawal, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denning, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolf, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beckstein, O.</span><span> </span><span class="NLM_article-title">MDAnalysis: A toolkit for the analysis of molecular dynamics simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">2319</span><span class="NLM_x">–</span> <span class="NLM_lpage">2327</span><span class="refDoi"> DOI: 10.1002/jcc.21787</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1002%2Fjcc.21787" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=21500218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvFalsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=2319-2327&author=N.+Michaud-Agrawalauthor=E.+J.+Denningauthor=T.+B.+Woolfauthor=O.+Beckstein&title=MDAnalysis%3A+A+toolkit+for+the+analysis+of+molecular+dynamics+simulations&doi=10.1002%2Fjcc.21787"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">MDAnalysis: A toolkit for the analysis of molecular dynamics simulations</span></div><div class="casAuthors">Michaud-Agrawal, Naveen; Denning, Elizabeth J.; Woolf, Thomas B.; Beckstein, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2319-2327</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">MDAnal. is an object-oriented library for structural and temporal anal. of mol. dynamics (MD) simulation trajectories and individual protein structures.  It is written in the Python language with some performance-crit. code in C.  It uses the powerful NumPy package to expose trajectory data as fast and efficient NumPy arrays.  It has been tested on systems of millions of particles.  Many common file formats of simulation packages including CHARMM, Gromacs, Amber, and NAMD and the Protein Data Bank format can be read and written.  Atoms can be selected with a syntax similar to CHARMM's powerful selection commands.  MDAnal. enables both novice and experienced programmers to rapidly write their own anal. tools and access data stored in trajectories in an easily accessible manner that facilitates interactive explorative anal.  MDAnal. has been tested on and works for most Unix-based platforms such as Linux and Mac OS X.  It is freely available under the GNU General Public License from http://mdanal.googlecode.com.  © 2011 Wiley Periodicals, Inc. J Comput Chem 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgUxkDoU25-bVg90H21EOLACvtfcHk0liI2vp00BQRww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvFalsr8%253D&md5=d567042c65cfdc1c81336a29137654bf</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21787%26sid%3Dliteratum%253Aachs%26aulast%3DMichaud-Agrawal%26aufirst%3DN.%26aulast%3DDenning%26aufirst%3DE.%2BJ.%26aulast%3DWoolf%26aufirst%3DT.%2BB.%26aulast%3DBeckstein%26aufirst%3DO.%26atitle%3DMDAnalysis%253A%2520A%2520toolkit%2520for%2520the%2520analysis%2520of%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2011%26volume%3D32%26spage%3D2319%26epage%3D2327%26doi%3D10.1002%2Fjcc.21787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Hunter, J. D.</span><span> </span><span class="NLM_article-title">Matplotlib: A 2D graphics environment</span> <span class="citation_source-journal">Comput. Sci. Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">90</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span><span class="refDoi"> DOI: 10.1109/MCSE.2007.55</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1109%2FMCSE.2007.55" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=90-95&author=J.+D.+Hunter&title=Matplotlib%3A+A+2D+graphics+environment&doi=10.1109%2FMCSE.2007.55"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1109%2FMCSE.2007.55&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FMCSE.2007.55%26sid%3Dliteratum%253Aachs%26aulast%3DHunter%26aufirst%3DJ.%2BD.%26atitle%3DMatplotlib%253A%2520A%25202D%2520graphics%2520environment%26jtitle%3DComput.%2520Sci.%2520Eng.%26date%3D2007%26volume%3D9%26spage%3D90%26epage%3D95%26doi%3D10.1109%2FMCSE.2007.55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Weitzdoerfer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstl, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeger, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dreher, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lubec, G.</span><span> </span><span class="NLM_article-title">Long-term influence of perinatal asphyxia on the social behavior in aging rats</span> <span class="citation_source-journal">Gerontology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">–</span> <span class="NLM_lpage">205</span><span class="refDoi"> DOI: 10.1159/000078348</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1159%2F000078348" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2004&pages=200-205&author=R.+Weitzdoerferauthor=N.+Gerstlauthor=D.+Pollakauthor=H.+Hoegerauthor=W.+Dreherauthor=G.+Lubec&title=Long-term+influence+of+perinatal+asphyxia+on+the+social+behavior+in+aging+rats&doi=10.1159%2F000078348"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1159%2F000078348&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000078348%26sid%3Dliteratum%253Aachs%26aulast%3DWeitzdoerfer%26aufirst%3DR.%26aulast%3DGerstl%26aufirst%3DN.%26aulast%3DPollak%26aufirst%3DD.%26aulast%3DHoeger%26aufirst%3DH.%26aulast%3DDreher%26aufirst%3DW.%26aulast%3DLubec%26aufirst%3DG.%26atitle%3DLong-term%2520influence%2520of%2520perinatal%2520asphyxia%2520on%2520the%2520social%2520behavior%2520in%2520aging%2520rats%26jtitle%3DGerontology%26date%3D2004%26volume%3D50%26spage%3D200%26epage%3D205%26doi%3D10.1159%2F000078348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Weitzdoerfer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeger, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engidawork, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singewald, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lubec, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lubec, B.</span><span> </span><span class="NLM_article-title">Neuronal nitric oxide synthase knock-out mice show impaired cognitive performance</span> <span class="citation_source-journal">Nitric Oxide</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">130</span><span class="NLM_x">–</span> <span class="NLM_lpage">140</span><span class="refDoi"> DOI: 10.1016/j.niox.2004.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1016%2Fj.niox.2004.03.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=130-140&author=R.+Weitzdoerferauthor=H.+Hoegerauthor=E.+Engidaworkauthor=M.+Engelmannauthor=N.+Singewaldauthor=G.+Lubecauthor=B.+Lubec&title=Neuronal+nitric+oxide+synthase+knock-out+mice+show+impaired+cognitive+performance&doi=10.1016%2Fj.niox.2004.03.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.niox.2004.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.niox.2004.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DWeitzdoerfer%26aufirst%3DR.%26aulast%3DHoeger%26aufirst%3DH.%26aulast%3DEngidawork%26aufirst%3DE.%26aulast%3DEngelmann%26aufirst%3DM.%26aulast%3DSingewald%26aufirst%3DN.%26aulast%3DLubec%26aufirst%3DG.%26aulast%3DLubec%26aufirst%3DB.%26atitle%3DNeuronal%2520nitric%2520oxide%2520synthase%2520knock-out%2520mice%2520show%2520impaired%2520cognitive%2520performance%26jtitle%3DNitric%2520Oxide%26date%3D2004%26volume%3D10%26spage%3D130%26epage%3D140%26doi%3D10.1016%2Fj.niox.2004.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Irwin, S.</span><span> </span><span class="NLM_article-title">Comprehensive observational assessment: Ia. A systematic, quantitative procedure for assessing the behavioral and physiologic state of the mouse</span> <span class="citation_source-journal">Psychopharmacologia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1968</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">222</span><span class="NLM_x">–</span> <span class="NLM_lpage">257</span><span class="refDoi"> DOI: 10.1007/BF00401402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1007%2FBF00401402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=5679627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A280%3ADyaF1M%252FgtlCiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1968&pages=222-257&author=S.+Irwin&title=Comprehensive+observational+assessment%3A+Ia.+A+systematic%2C+quantitative+procedure+for+assessing+the+behavioral+and+physiologic+state+of+the+mouse&doi=10.1007%2FBF00401402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive observational assessment: Ia. A systematic, quantitative procedure for assessing the behavioral and physiologic state of the mouse</span></div><div class="casAuthors">Irwin S</div><div class="citationInfo"><span class="NLM_cas:title">Psychopharmacologia</span>
        (<span class="NLM_cas:date">1968</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">222-57</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQae8F3n8PRtf6AExegM5LLfW6udTcc2eZi-TcILCWU07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaF1M%252FgtlCiuw%253D%253D&md5=37ab1a9fa105ba776dbd68650fd9edbf</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1007%2FBF00401402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00401402%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DS.%26atitle%3DComprehensive%2520observational%2520assessment%253A%2520Ia.%2520A%2520systematic%252C%2520quantitative%2520procedure%2520for%2520assessing%2520the%2520behavioral%2520and%2520physiologic%2520state%2520of%2520the%2520mouse%26jtitle%3DPsychopharmacologia%26date%3D1968%26volume%3D13%26spage%3D222%26epage%3D257%26doi%3D10.1007%2FBF00401402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Porsolt, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anton, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blavet, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jalfre, M.</span><span> </span><span class="NLM_article-title">Behavioural despair in rats: a new model sensitive to antidepressant treatments</span> <span class="citation_source-journal">Eur. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1978</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">379</span><span class="NLM_x">–</span> <span class="NLM_lpage">391</span><span class="refDoi"> DOI: 10.1016/0014-2999(78)90118-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=10.1016%2F0014-2999%2878%2990118-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=204499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;key=1%3ACAS%3A528%3ADyaE1cXhvVagsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1978&pages=379-391&author=R.+D.+Porsoltauthor=G.+Antonauthor=N.+Blavetauthor=M.+Jalfre&title=Behavioural+despair+in+rats%3A+a+new+model+sensitive+to+antidepressant+treatments&doi=10.1016%2F0014-2999%2878%2990118-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Behavioral despair in rats:  a new model sensitive to antidepressant treatments</span></div><div class="casAuthors">Porsolt, Roger D.; Anton, Guy; Blavet, Nadine; Jalfre, Maurice</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">379-91</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    </div><div class="casAbstract">Rats when forced to swim in a cylinder from which they cannot escape will, after an initial period of vigorous activity, adopt a characteristic immobile posture which can be readily identified.  Immobility was decreased by various clin. effective antidepressant drugs at doses which otherwise decreased spontaneous motor activity in an open field.  Antidepressants could thus be distinguished from psychostimulants which decreased immobility at doses which increased general activity.  Anxiolytic compds. did not affect immobility, whereas major tranquilizers enhanced it.  Thus, immobility reflects a state of lowered mood in the rat which is selectively sensitive to antidepressant treatments.  Pos. findings with atypical antidepressant drugs such as iprindole and mianserin suggest that the method may be capable of discovering new antidepressants hitherto undetectable with classical pharmacol. tests.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0oUdFHAF1sLVg90H21EOLACvtfcHk0lgsnxyU3XlPzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXhvVagsr0%253D&md5=293e1fe8766c76315e86bc375c810a58</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2F0014-2999%2878%2990118-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-2999%252878%252990118-8%26sid%3Dliteratum%253Aachs%26aulast%3DPorsolt%26aufirst%3DR.%2BD.%26aulast%3DAnton%26aufirst%3DG.%26aulast%3DBlavet%26aufirst%3DN.%26aulast%3DJalfre%26aufirst%3DM.%26atitle%3DBehavioural%2520despair%2520in%2520rats%253A%2520a%2520new%2520model%2520sensitive%2520to%2520antidepressant%2520treatments%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1978%26volume%3D47%26spage%3D379%26epage%3D391%26doi%3D10.1016%2F0014-2999%2878%2990118-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M48" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M48','PDB','4M48'); return false;">PDB: 4M48</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i91"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01313">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_73177"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01313">10.1021/acs.jmedchem.7b01313</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">MD simulation data, structure of CE-103, reuptake inhibition curves, X-ray measurements data, high-resolution mass spectrometry spectra, <sup>1</sup>H and <sup>13</sup>C NMR spectra, HPLC-determined purity data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01313/suppl_file/jm7b01313_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01313/suppl_file/jm7b01313_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01313/suppl_file/jm7b01313_si_001.pdf">jm7b01313_si_001.pdf (17.63 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01313/suppl_file/jm7b01313_si_002.csv">jm7b01313_si_002.csv (2.38 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01313&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-22%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01313%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01313" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67991864de1cd93e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
