
<html lang="en"     class="pb-page"  data-request-id="ef312b57-03bf-4dc0-8aa0-0af6a73f1916"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2021.64.issue-11;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.1c00234;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Computational Bioactivity Fingerprint Similarities To Navigate the Discovery of Novel Scaffolds" /></meta><meta name="dc.Creator" content="Guo-Li  Xiong" /></meta><meta name="dc.Creator" content="Yue  Zhao" /></meta><meta name="dc.Creator" content="Lu  Liu" /></meta><meta name="dc.Creator" content="Zhong-Ye  Ma" /></meta><meta name="dc.Creator" content="Ai-Ping  Lu" /></meta><meta name="dc.Creator" content="Yan  Cheng" /></meta><meta name="dc.Creator" content="Ting-Jun  Hou" /></meta><meta name="dc.Creator" content="Dong-Sheng  Cao" /></meta><meta name="dc.Description" content="As one of the central tasks of modern medicinal chemistry, scaffold hopping is expected to lead to the discovery of structural novel biological active compounds and broaden the chemical space of kn..." /></meta><meta name="Description" content="As one of the central tasks of modern medicinal chemistry, scaffold hopping is expected to lead to the discovery of structural novel biological active compounds and broaden the chemical space of kn..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 19, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00234" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00234" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00234" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00234" /></link>
        
    
    

<title>Computational Bioactivity Fingerprint Similarities To Navigate the Discovery of Novel Scaffolds | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00234" /></meta><meta property="og:title" content="Computational Bioactivity Fingerprint Similarities To Navigate the Discovery of Novel Scaffolds" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0008.jpeg" /></meta><meta property="og:description" content="As one of the central tasks of modern medicinal chemistry, scaffold hopping is expected to lead to the discovery of structural novel biological active compounds and broaden the chemical space of known active compounds. Here, we report the computational bioactivity fingerprint (CBFP) for easier scaffold hopping, where the predicted activities in multiple quantitative structure–activity relationship models are integrated to characterize the biological space of a molecule. In retrospective benchmarks, the CBFP representation shows outstanding scaffold hopping potential relative to other chemical descriptors. In the prospective validation for the discovery of novel inhibitors of poly [ADP-ribose] polymerase 1, 35 predicted compounds with diverse structures are tested, 25 of which show detectable growth-inhibitory activity; beyond this, the most potent (compound 6) has an IC50 of 0.263 nM. These results support the use of CBFP representation as the bioactivity proxy of molecules to explore uncharted chemical space and discover novel compounds." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00234"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00234">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00234&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00234&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00234&amp;href=/doi/10.1021/acs.jmedchem.1c00234" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 7544-7554</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00228" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00256" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Computational Bioactivity Fingerprint Similarities To Navigate the Discovery of Novel Scaffolds</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Guo-Li Xiong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guo-Li Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Xiangya School of Pharmaceutical Sciences, Central South University, Changsha, Hunan 410003, China</div><div class="loa-info-affiliations-info">Hunan Key Laboratory of Diagnostic and Therapeutic Drug Research for Chronic Diseases, Central South University, Changsha, Hunan 410013, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guo-Li++Xiong">Guo-Li Xiong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yue Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yue Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Xiangya School of Pharmaceutical Sciences, Central South University, Changsha, Hunan 410003, China</div><div class="loa-info-affiliations-info">Hunan Key Laboratory of Diagnostic and Therapeutic Drug Research for Chronic Diseases, Central South University, Changsha, Hunan 410013, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yue++Zhao">Yue Zhao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lu Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lu Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Xiangya School of Pharmaceutical Sciences, Central South University, Changsha, Hunan 410003, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lu++Liu">Lu Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhong-Ye Ma</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhong-Ye Ma</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Xiangya School of Pharmaceutical Sciences, Central South University, Changsha, Hunan 410003, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhong-Ye++Ma">Zhong-Ye Ma</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ai-Ping Lu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ai-Ping Lu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute for Advancing Translational Medicine in Bone and Joint Diseases, School of Chinese Medicine, Hong Kong Baptist University, 999077 Hong Kong SAR, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ai-Ping++Lu">Ai-Ping Lu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yan Cheng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yan Cheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacy, The Second Xiangya Hospital, Central South University, Changsha, Hunan 410003, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yan++Cheng">Yan Cheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ting-Jun Hou</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ting-Jun Hou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Innovation Institute for Artificial Intelligence in Medicine of Zhejiang University, College of Pharmaceutical Sciences, Zhejiang University, Hangzhou, Zhejiang 310058, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#e3978a8d8489968d8b8c96a3998996cd868796cd808d"><span class="__cf_email__" data-cfemail="3a4e53545d504f5452554f7a40504f145f5e4f145954">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ting-Jun++Hou">Ting-Jun Hou</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Dong-Sheng Cao</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dong-Sheng Cao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Xiangya School of Pharmaceutical Sciences, Central South University, Changsha, Hunan 410003, China</div><div class="loa-info-affiliations-info">Hunan Key Laboratory of Diagnostic and Therapeutic Drug Research for Chronic Diseases, Central South University, Changsha, Hunan 410013, China</div><div class="loa-info-affiliations-info">Institute for Advancing Translational Medicine in Bone and Joint Diseases, School of Chinese Medicine, Hong Kong Baptist University, 999077 Hong Kong SAR, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#1b7469727e756f7a7736787f685b2a2d2835787476"><span class="__cf_email__" data-cfemail="18776a717d766c7974357b7c6b58292e2b367b7775">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dong-Sheng++Cao">Dong-Sheng Cao</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3604-3785" title="Orcid link">https://orcid.org/0000-0003-3604-3785</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00234&amp;href=/doi/10.1021%2Facs.jmedchem.1c00234" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 7544–7554</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 19, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>6 February 2021</li><li><span class="item_label"><b>Published</b> online</span>19 May 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00234" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00234</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7544%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DGuo-Li%2BXiong%252C%2BYue%2BZhao%252C%2BLu%2BLiu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D11%26contentID%3Dacs.jmedchem.1c00234%26title%3DComputational%2BBioactivity%2BFingerprint%2BSimilarities%2BTo%2BNavigate%2Bthe%2BDiscovery%2Bof%2BNovel%2BScaffolds%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7554%26publicationDate%3DJune%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00234"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">980</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00234" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Computational Bioactivity Fingerprint Similarities To Navigate the Discovery of Novel Scaffolds&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Guo-Li&quot;,&quot;last_name&quot;:&quot;Xiong&quot;},{&quot;first_name&quot;:&quot;Yue&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Lu&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Zhong-Ye&quot;,&quot;last_name&quot;:&quot;Ma&quot;},{&quot;first_name&quot;:&quot;Ai-Ping&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;Cheng&quot;},{&quot;first_name&quot;:&quot;Ting-Jun&quot;,&quot;last_name&quot;:&quot;Hou&quot;},{&quot;first_name&quot;:&quot;Dong-Sheng&quot;,&quot;last_name&quot;:&quot;Cao&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;7544-7554&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00234&quot;},&quot;abstract&quot;:&quot;As one of the central tasks of modern medicinal chemistry, scaffold hopping is expected to lead to the discovery of structural novel biological active compounds and broaden the chemical space of known active compounds. Here, we report the computational bioactivity fingerprint (CBFP) for easier scaffold hopping, where the predicted activities in multiple quantitative structure–activity relationship models are integrated to characterize the biological space of a molecule. In retrospective benchmarks, the CBFP representation shows outstanding scaffold hopping potential relative to other chemical descriptors. In the prospective validation for the discovery of novel inhibitors of poly [ADP-ribose] polymerase 1, 35 predicted compounds with diverse structures are tested, 25 of which show detectable growth-inhibitory activity; beyond this, the most potent (compound 6) has an IC50 of 0.263 nM. These results support the use of CBFP representation as the bioactivity proxy of molecules to explore uncharted chemical space&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00234&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00234" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00234&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00234" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00234&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00234" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00234&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00234&amp;href=/doi/10.1021/acs.jmedchem.1c00234" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00234" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00234" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00234%26sid%3Dliteratum%253Aachs%26pmid%3D34008979%26genre%3Darticle%26aulast%3DXiong%26date%3D2021%26atitle%3DComputational%2BBioactivity%2BFingerprint%2BSimilarities%2BTo%2BNavigate%2Bthe%2BDiscovery%2Bof%2BNovel%2BScaffolds%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D11%26spage%3D7544%26epage%3D7554%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a>,</li><li><a href="/action/doSearch?ConceptID=291321" title="Bioactivity">Bioactivity</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/jmcmar.2021.64.issue-11/20210610/jmcmar.2021.64.issue-11.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">As one of the central tasks of modern medicinal chemistry, scaffold hopping is expected to lead to the discovery of structural novel biological active compounds and broaden the chemical space of known active compounds. Here, we report the computational bioactivity fingerprint (CBFP) for easier scaffold hopping, where the predicted activities in multiple quantitative structure–activity relationship models are integrated to characterize the biological space of a molecule. In retrospective benchmarks, the CBFP representation shows outstanding scaffold hopping potential relative to other chemical descriptors. In the prospective validation for the discovery of novel inhibitors of poly [ADP-ribose] polymerase 1, 35 predicted compounds with diverse structures are tested, 25 of which show detectable growth-inhibitory activity; beyond this, the most potent (compound 6) has an IC<sub>50</sub> of 0.263 nM. These results support the use of CBFP representation as the bioactivity proxy of molecules to explore uncharted chemical space and discover novel compounds.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55533" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55533" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Despite the widespread and extensive applications of combinatorial and parallel synthesis in drug discovery, the discovered small druglike molecules only represent a small proportion of the chemical space and most regions remain unexplored.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> The number of possible small molecules populating the chemical space has been estimated to exceed 10<sup>60</sup>; nevertheless, newly identified drug candidates are always close to known ligands in chemical space.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Some reasons may explain the importance of identifying bioactive compounds with novel chemotypes, including improving potency and drug-likeness, providing standby molecules to cope with potential attrition, and breaking the patent barriers of counterparts.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> To set up the core competitiveness, many pharmaceutical companies are investing heavily in research for diverse bioactive molecules.</div><div class="NLM_p">Scaffold hopping, a concept derived from computational chemistry, is a widely used technique for the exploration of new chemotypes.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> It is the detection of new compounds that share similar activity and molecular properties with known active compounds but possess different core structures. These novel structures can serve as starting points for lead optimization efforts. Many marketed drugs and drug candidates were identified by scaffold hopping methods. For example, the centrally acting analgesic agent, tramadol, was generated from the ring-opening operation of morphine;<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> a novel 5-lipoxygenase inhibitor with an entirely new core structure was found by topological pharmacophore searching.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> In 2017, Bajorath et al. presented a mini-perspective on the recent advances in scaffold hopping,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> where they summarized the exemplary computational methods of this field, including pharmacophore searching,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> shape searching,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> chemical similarity,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> structure-based similarity searching (molecular docking)<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> methods, and machine learning (ML) methods.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Essentially, these methods follow the “similar property principle,” in which structure similarity is applied to infer biological activity.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> It appears that the essence of these similarity-based methods is contrary to that of scaffold hopping, that is, they have intrinsic limitations in discovering novel structures. Does this mean that these methods are not suitable for scaffold hopping? It should be noted that chemical structures can be characterized by different descriptors with varying degrees of simplification, from simple binary topological fingerprints to complicated multipoint 3D pharmacophore fingerprints. With the decrease in the dependence on basic chemical structures, the abstract nature of these representations has increased significantly. Therefore, pharmacophore-based methods have gained preference due to relatively good scaffold hopping potentials.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> However, they still more or less depend on structural information, thus limiting their practical capability in searching for new chemical scaffolds. There is therefore an urgent need to develop more straightforward and systematic approaches for identifying structurally novel compounds.</div><div class="NLM_p">In the context of scaffold hopping, moving from chemical structures to the bioactivity space seems blocked. Conversely, searching for novel chemotypes by using bioactivity space similarities may be more rational. Here, an important question is raised: how to characterize the bioactivity space of compounds. The expertise accumulated in chemical genomics provides a new idea wherein the activities against a panel of reference targets can be combined to represent the biological position of the ligand in the bioactivity space.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> In practice, the compound activity profiles derived from multiple assay results are used to constitute the bioactivity signatures. Two ligands with similar biological effects on the targets of interest should share similar bioactivity signatures. Such bioactivity representation has been widely applied in many scenarios including mechanism-of-action elucidation,<a onclick="showRef(event, 'ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24">(21−24)</a> virtual screening and potency prediction,<a onclick="showRef(event, 'ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref27">(25−27)</a> toxicity prediction,<a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28−30)</a> etc. As for scaffold hopping, there are some meaningful explorations. Petrone et al. compared 1.5 million in-house compounds by using the biological “high-throughput screening fingerprints” (HTS-FPs) derived from 195 assays<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and concluded that the HTS-FP was of an enormous value in virtual screening and scaffold hopping in particular. Recently, researchers of AstraZeneca combined the HTS-FP and the ECFP4 fingerprint for scaffold hopping,<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> and the results indicated that there was a synergistic action between the two fingerprints with the HTS-FP contributing the most to the overall performance.</div><div class="NLM_p">However, traditional assay-based biological representation has two inherent drawbacks which may hinder its application in the drug discovery domain.<a onclick="showRef(event, 'ref20 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref20 ref34 ref35">(20,34,35)</a> One is data paucity. Although many large-scale biochemical and cellar assays are collected and serve as compound representation, a large proportion of molecules have only been tested in a few assays, resulting in a sparse bioactivity matrix. Moreover, due to inadequate sampling of biological space, some additional noise data may be introduced. Another challenge is the limited application range. Only molecules that have been tested in enough assays are qualified to generate their own bioactivity representations, thereby being predicted by the established models. Nevertheless, we require predictions to be made on unknown molecules rather than those that have been tested in multiple assays in most cases.</div><div class="NLM_p last">A significant advancement that coped with these drawbacks described here was the computational bioactivity fingerprint (CBFP), where the predicted activities in multiple quantitative structure–activity relationship (QSAR) models were integrated to characterize the biological space of a molecule. Of particular note was that both tested and untested small molecules could obtain their own fixed-length biological representations. A retrospective scaffold hopping study was conducted to (1) evaluate the scaffold hopping potential of the CBFP through similarity searching, (2) compare the performance of the CBFP with those of other baseline chemical descriptors, and (3) determine the similarity value distribution of the CBFP to provide more guidance for virtual screening. Finally, the CBFP representation was used in a prospective experiment to find novel inhibitors of poly [ADP-ribose] polymerase 1 (PARP1) to confirm if it could be successfully employed in practical drug design applications.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15878" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15878" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  Global Scaffold Hopping Potential</h3><div class="NLM_p">We performed similarity searching to explore the scaffold hopping potential of the CBFP and other baseline representations. These comparisons were based on the global scaffold hopping performance without distinguishing the targets. We used scaffold recall rata (SRR) to evaluate the scaffold hopping potential of different representations. Of note, the potential described here means the ability to discover novel chemotypes rather than hopping magnitudes. The results have been presented in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Box plots of the SRR values within top 100 (A) and top 5000 (B) for different screening data sets. The horizontal lines indicate the median, and the plus signs represent the mean SRR values of the 1660 query molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Global SRR (%) Results of 10 Molecular Representations for Different Screening Data Sets<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" rowspan="2" align="center">molecule representation</th><th class="rowsep1 colsep0" colspan="2" align="center">ChEMBL</th><th class="rowsep1 colsep0" colspan="2" align="center">Natural Products</th><th class="rowsep1 colsep0" colspan="2" align="center">ChemBlock</th><th class="rowsep1 colsep0" colspan="2" align="center">SPECS</th><th class="rowsep1 colsep0" colspan="2" align="center">ASINEX</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">top 100</th><th class="colsep0 rowsep0" align="center">top 5000</th><th class="colsep0 rowsep0" align="center">top 100</th><th class="colsep0 rowsep0" align="center">top 5000</th><th class="colsep0 rowsep0" align="center">top 100</th><th class="colsep0 rowsep0" align="center">top 5000</th><th class="colsep0 rowsep0" align="center">top 100</th><th class="colsep0 rowsep0" align="center">top 5000</th><th class="colsep0 rowsep0" align="center">top 100</th><th class="colsep0 rowsep0" align="center">top 5000</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AtomPair</td><td class="colsep0 rowsep0" align="left">23.9 ± 10.1</td><td class="colsep0 rowsep0" align="left">54.5 ± 11.3</td><td class="colsep0 rowsep0" align="left">27.5 ± 11.0</td><td class="colsep0 rowsep0" align="left">54.8 ± 13.3</td><td class="colsep0 rowsep0" align="left">27.4 ± 11.9</td><td class="colsep0 rowsep0" align="left">51.2 ± 16.3</td><td class="colsep0 rowsep0" align="left"><b>26.3 ± 11.7</b></td><td class="colsep0 rowsep0" align="left">49.8 ± 17.0</td><td class="colsep0 rowsep0" align="left">20.8 ± 9.0</td><td class="colsep0 rowsep0" align="left">35.3 ± 12.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MACCS</td><td class="colsep0 rowsep0" align="left">21.0 ± 10.4</td><td class="colsep0 rowsep0" align="left">54.4 ± 16.4</td><td class="colsep0 rowsep0" align="left">29.2 ± 14.6</td><td class="colsep0 rowsep0" align="left">60.8 ± 18.7</td><td class="colsep0 rowsep0" align="left">15.0 ± 8.7</td><td class="colsep0 rowsep0" align="left">44.3 ± 16.6</td><td class="colsep0 rowsep0" align="left">20.2 ± 9.8</td><td class="colsep0 rowsep0" align="left">48.4 ± 16.7</td><td class="colsep0 rowsep0" align="left">18.0 ± 8.8</td><td class="colsep0 rowsep0" align="left">35.2 ± 13.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ECFP4</td><td class="colsep0 rowsep0" align="left">28.5 ± 16.2</td><td class="colsep0 rowsep0" align="left">63.2 ± 18.7</td><td class="colsep0 rowsep0" align="left">34.8 ± 18.2</td><td class="colsep0 rowsep0" align="left">63.5 ± 17.1</td><td class="colsep0 rowsep0" align="left">27.5 ± 12.8</td><td class="colsep0 rowsep0" align="left">53.0 ± 20.2</td><td class="colsep0 rowsep0" align="left">25.7 ± 12.6</td><td class="colsep0 rowsep0" align="left">50.1 ± 20.3</td><td class="colsep0 rowsep0" align="left">25.0 ± 12.2</td><td class="colsep0 rowsep0" align="left">44.5 ± 21.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FP2</td><td class="colsep0 rowsep0" align="left">16.1 ± 8.0</td><td class="colsep0 rowsep0" align="left">33.6 ± 15.1</td><td class="colsep0 rowsep0" align="left">15.9 ± 8.8</td><td class="colsep0 rowsep0" align="left">23.1 ± 10.9</td><td class="colsep0 rowsep0" align="left">15.3 ± 8.6</td><td class="colsep0 rowsep0" align="left">22.6 ± 10.5</td><td class="colsep0 rowsep0" align="left">15.4 ± 8.7</td><td class="colsep0 rowsep0" align="left">23.5 ± 10.9</td><td class="colsep0 rowsep0" align="left">14.5 ± 7.9</td><td class="colsep0 rowsep0" align="left">24.7 ± 12.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Gpi_DAPH3</td><td class="colsep0 rowsep0" align="left">16.5 ± 10.3</td><td class="colsep0 rowsep0" align="left">47.1 ± 17.7</td><td class="colsep0 rowsep0" align="left">20.2 ± 12.8</td><td class="colsep0 rowsep0" align="left">44.7 ± 18.3</td><td class="colsep0 rowsep0" align="left"><b>27.6 ± 12.9</b></td><td class="colsep0 rowsep0" align="left">49.0 ± 19.2</td><td class="colsep0 rowsep0" align="left">15.8 ± 10.1</td><td class="colsep0 rowsep0" align="left">36.8 ± 18.0</td><td class="colsep0 rowsep0" align="left">14.5 ± 8.6</td><td class="colsep0 rowsep0" align="left">31.3 ± 16.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TGD</td><td class="colsep0 rowsep0" align="left">16.8 ± 12.4</td><td class="colsep0 rowsep0" align="left">42.6 ± 15.2</td><td class="colsep0 rowsep0" align="left">18.6 ± 11.4</td><td class="colsep0 rowsep0" align="left">42.4 ± 15.1</td><td class="colsep0 rowsep0" align="left">19.7 ± 9.9</td><td class="colsep0 rowsep0" align="left">41.2 ± 14.5</td><td class="colsep0 rowsep0" align="left">13.2 ± 9.6</td><td class="colsep0 rowsep0" align="left">32.9 ± 14.5</td><td class="colsep0 rowsep0" align="left">14.5 ± 8.6</td><td class="colsep0 rowsep0" align="left">29.0 ± 15.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TGT</td><td class="colsep0 rowsep0" align="left">13.6 ± 11.4</td><td class="colsep0 rowsep0" align="left">41.1 ± 17.7</td><td class="colsep0 rowsep0" align="left">16.0 ± 14.3</td><td class="colsep0 rowsep0" align="left">39.5 ± 17.2</td><td class="colsep0 rowsep0" align="left">20.2 ± 13.6</td><td class="colsep0 rowsep0" align="left">42.0 ± 18.3</td><td class="colsep0 rowsep0" align="left">14.2 ± 13.3</td><td class="colsep0 rowsep0" align="left">34.0 ± 18.0</td><td class="colsep0 rowsep0" align="left">11.5 ± 10.1</td><td class="colsep0 rowsep0" align="left">25.0 ± 15.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CATS</td><td class="colsep0 rowsep0" align="left">21.1 ± 11.8</td><td class="colsep0 rowsep0" align="left">43.4 ± 13.5</td><td class="colsep0 rowsep0" align="left">22.7 ± 10.6</td><td class="colsep0 rowsep0" align="left">44.3 ± 13.4</td><td class="colsep0 rowsep0" align="left">20.5 ± 9.9</td><td class="colsep0 rowsep0" align="left">41.5 ± 14.2</td><td class="colsep0 rowsep0" align="left">13.4 ± 8.7</td><td class="colsep0 rowsep0" align="left">32.8 ± 13.6</td><td class="colsep0 rowsep0" align="left">17.4 ± 9.4</td><td class="colsep0 rowsep0" align="left">33.5 ± 14.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MOE2D</td><td class="colsep0 rowsep0" align="left">29.7 ± 17.2</td><td class="colsep0 rowsep0" align="left">53.3 ± 16.2</td><td class="colsep0 rowsep0" align="left">31.4 ± 15.7</td><td class="colsep0 rowsep0" align="left">54.2 ± 15.0</td><td class="colsep0 rowsep0" align="left">25.5 ± 13.1</td><td class="colsep0 rowsep0" align="left">51.3 ± 15.8</td><td class="colsep0 rowsep0" align="left">19.2 ± 13.1</td><td class="colsep0 rowsep0" align="left">44.2 ± 16.9</td><td class="colsep0 rowsep0" align="left">29.3 ± 18.8</td><td class="colsep0 rowsep0" align="left">46.3 ± 22.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CBFP</td><td class="colsep0 rowsep0" align="left"><b>40.6 ± 21.8</b></td><td class="colsep0 rowsep0" align="left"><b>70.3 ± 18.1</b></td><td class="colsep0 rowsep0" align="left"><b>46.9 ± 23.1</b></td><td class="colsep0 rowsep0" align="left"><b>66.5 ± 19.3</b></td><td class="colsep0 rowsep0" align="left">26.7 ± 13.0</td><td class="colsep0 rowsep0" align="left"><b>53.4 ± 17.0</b></td><td class="colsep0 rowsep0" align="left">25.6 ± 13.1</td><td class="colsep0 rowsep0" align="left"><b>51.7 ± 17.5</b></td><td class="colsep0 rowsep0" align="left"><b>42.6 ± 26.3</b></td><td class="colsep0 rowsep0" align="left"><b>60.9 ± 24.6</b></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The bold values represent the maximum value of the column.</p></div></div></div><div class="NLM_p">Focusing on the top 100 molecules, it can be observed from <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> that most structural descriptors yielded mediocre performances across the screening datasets, suggesting that these descriptors exhibited a certain ability of scaffold hopping. The CBFP representation outperformed other chemical descriptors for three screening datasets (ChEMBL, Natural Products, and ASINEX), though a slightly less performant than the Gpi_DAPH3 and AtomPair descriptors for the ChemBlock and SPECS datasets, respectively. Improvements in the SRR results of the CBFP over those of the chemical descriptors were statistically significant (95% confidence interval, CI) according to the paired <i>t</i>-test, except for the ECFP4 descriptors. When the top 5000 compounds were taken into consideration, the SRR values for all representations improved remarkably. Particularly, the CBFP representation was superior to other chemical descriptors with its average SRR values above 50% across datasets, suggesting that the query molecules could recall more than 50% scaffolds on average within the top 5000 compounds. Another significant characteristic is the large SD across all the datasets. The significant differences among the query molecules of distinct scaffolds and targets may explain this dispersibility. Regarding the performance degradation on ChemBlock and SPECS, the high similarity between the screening compounds and the query molecules seems to make the higher ranking of some active molecules difficult. Moreover, with the enlargement of the screening data, simply employing the top 100/5000 as the threshold is more and more stringent. For example, the threshold of top 100 means the top 0.11% for the ChEMBL data set but the top 0.047% for the SPECS data set, with the former being more than twice the latter.</div><div class="NLM_p">Interestingly, the chemical descriptors, as illustrated in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, tended to show a similar trend across the screening datasets. The ECFP4 descriptors showed relatively good scaffold hopping potential in the four low-level representations (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). Its average SRR values exceeded 25 and 50% in most datasets within the top 100 and 5000 compounds, respectively. This was consistent with the conclusions of previous studies that reported ECFP4 as the best chemical structure fingerprint in 2D fingerprints for performing scaffold hopping.<a onclick="showRef(event, 'ref32 ref36'); return false;" href="javascript:void(0);" class="ref ref32 ref36">(32,36)</a> The least scaffold hopping potential was obtained by FP2. Even in the top 5000 compounds, its average SRR values rarely exceeded 25%. Additionally, it is difficult to overlook the unexpected performance of the MOE2D descriptors, especially within the top 100 compounds. Its median SRR values were usually high followed by those of the CBFP representation in most cases. The MOE2D representation, classified as a high-level representation, comprises 206 descriptors reflecting different properties of molecules, such as physical properties, drug-likeness, topologies, subdivided surface areas, etc. Such multiperspective, complementary, and comprehensive representation is less reliant on chemical details but focuses more on the whole molecule level. Therefore, we believe that MOE2D is a good choice in practical applications of scaffold hopping in the future. Lastly, it seems that pharmacophore fingerprints (median-level representation) have no significant advantage over the atom and structural fingerprints (low-level representation), which is not consistent with the original assumption that less dependence on chemical structures will lead to superior scaffold hopping performance, suggesting a significant gap in our understanding. However, it would be improper to deny their scaffold hopping potentials based on other approaches except simple similarity-based ranking. Anyway, the effectiveness of pharmacophore-based approaches has been proven in practice.</div><div class="NLM_p last">The active molecule recall rate (AMRR) results were also systematically analyzed and the results are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_001.pdf" class="ext-link">Table S1 and Figure S1</a>. They are closely related to the SRR values, reflecting the virtual screening performance. Normally, AMRR values are smaller than the corresponding SRR values, as not all the compounds sharing the same scaffold can be recalled within a predefined threshold. For specific molecule representations, the AMRR and SRR results share highly similar trends. Therefore, we will not show more details herein and the AMRR results will not be included in our subsequent discussions.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">2.2.  Target-Based Scaffold Hopping Potential Analysis</h3><div class="NLM_p">There were many differences among the 17 query targets, reflected by the large SDs in all the molecular representations and screening datasets. Some target-specific characteristics may be concealed by the average-value-based statistics, though they could represent the average level of scaffold hopping potential. For this reason, a detailed scaffold hopping potential analysis across the targets is shown in this part.</div><div class="NLM_p">To compare the performance of the tested methods, SSR results within the top 100 compounds of the 17 query targets were plotted using the CBFP representation as a function of other chemical descriptors (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The points above the diagonal are the specific query targets for which the CBFP representation outperformed the chemical descriptors (higher SRR value). The higher performance of the CBFP in scaffold hopping is evident in these comparisons. We applied the Wilcoxon matched-pairs signed-ranks test to critically evaluate these representations. At 95% CI, the CBFP representation significantly outperformed eight of the nine chemical descriptors. The CBFP representation may have slightly lower performance than that of ECFP4 in some cases, leading to no significant difference. Additionally, a heatmap reflecting the results of individual targets on different screening datasets was plotted to show the landscape of the scaffold hopping potential (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). To some targets, like DRD2, GRHR, MCR4, and SERT, CBFP representation exhibited strong scaffold hopping potential up to 90%. We also observed that the same molecular representation exhibited different scaffold hopping abilities across targets. Even the CBFP representation could not yield high SRR values for all protein targets. With the results for ChEMBL as an example, the SSR value for the melanocortin 4 receptor (MC4R) reached 86.2% but dropped to 11.1% for HIV-1 reverse transcriptase (HIVRT), showing a significant fluctuation. The results of the TGD and TDT descriptors were highly variable across the query targets; the AtomPair and ECFP4 descriptors yielded a relatively more stable performance. This indicates that it is critical and necessary to make a comprehensive assessment of molecular representations before implementing similarity searching on a given target. The results are not unexpected because, to some extent, such target-specific results run through the whole length of the drug discovery process. Despite the different performance across targets in general, the CBFP is considered a comprehensive and powerful biological representation suitable for scaffold hopping applications.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Comparison of the CBFP representation and the other chemical descriptors for the five screening datasets in terms of the SRR results within the top 100 compounds. For each chemical descriptor, there are 17 points representing 17 different query datasets. Points above the diagonal are the specific query targets for which the CBFP representation outperformed the chemical descriptors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Heatmap of the average SRR values (top 100 compounds) of the individual targets for the five different screening datasets. Annotations on the left of the heatmap show the corresponding targets. The capital letters at the bottom refer to different molecular representations: A, AtomPair; M, MACCS; E, ECFP4; F, FP2; G, Gpi_DAPH3; D, TGD; T, TGT; C, CATS; O, MOE; P, CBFP. Colors represent a three gradient from the worst (red) to the best (blue).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Among the 17 query targets, DHFR and HIVRT seemed to be different from the others (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). The 10 molecular representations could consistently yield satisfactory scaffold hopping performances (blue) on DHFR but poor performances (red) on HIVRT. To understand this phenomenon, we closely scrutinized the active molecules of the two targets. The Tanimoto similarity calculation was implemented between each molecule and the rest of the molecules with distinct scaffolds. The molecule–molecule Tanimoto values are plotted in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A to visualize the chemical distance of scaffolds/molecules. As plotted here, the molecules toward DHFR had a higher degree of structural similarity compared to those toward HIVRT. Most similarity values for DHFR were distributed within the range of 0.18–0.28 while most intermolecular similarity values for HIVPR ranged from 0.08 to 0.15. At first glance, such similarity differences seem negligible, but they could indeed improve the rankings of active molecules and recall diverse scaffolds within the threshold. Moreover, the nearest neighbor Tanimoto similarity values of the query molecules are compared in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B. In terms of the median, HIVPR had higher data diversity compared to DHFR: more than 50% of the nearest neighbor similarity values for DHFR were higher than the worst 75% values for HIVPR, implying that the quality of the validation data could also affect scaffold recalling, especially for molecular representations that are heavily dependent on the structural details of molecules. However, under the same validation conditions, the comparison of molecular representations was reliable and meaningful.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of DHFR and HIVRT in terms of chemical diversity. The Tanimoto similarity between the molecules with different scaffolds was calculated based on the ECFP4 descriptors to represent the chemical diversity of the query datasets. (A) Frequent distribution curves of the Tanimoto similarity values of DHFR and HIVRT. (B) Box plot of the nearest neighbor Tanimoto similarity of query molecules. HIVPR: red, DHFR: blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7">2.3.  Guidelines for Virtual Screening: Successful Scaffold Hopping and Similarity Thresholds</h3><div class="NLM_p">Both integral and separate analyses have confirmed that the CBFP performed well in searching for novel scaffolds. However, these qualitative analyses may not do much to guide real drug screening programs. Therefore, successful scaffold hopping and similarity thresholds will be discussed in this part.</div><div class="NLM_p">Generally, if the scaffold recall rate within the top 1% of compounds is greater than or equal to 25%, the trial is considered as a successful scaffold hopping. The average SRR values of the CBFP representations are illustrated in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>. For comparison purposes, ECFP4 and MOE2D, two excellent baseline chemical descriptors, are also included. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, all the three molecular representations could realize successful scaffold hopping for the five screening datasets. Particularly, the SRR results of the CBFP for ChEMBL, Natural Products, and ASINEX reached up to 50%, which was about twice the minimum of a successful scaffold hopping, suggesting that the CBFP had sufficient representation ability to navigate the discovery of novel scaffolds. Additionally, MOE2D could attain SRR values of more than 40% in four of the five screening datasets. Furthermore, the threshold was tightened to the top 0.5% for more rigorously evaluation (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). As expected, the SRR values of the three representations decreased at different degrees, although their rankings remained unchanged. The average decrease of ECFP4 and MOE2D reached 12.5 and 10.2%, respectively, while the average decline for CBFP representation was 8.9%, implying that most recalled scaffolds of the top 1% compounds are mainly located in the top 0.5% region and that CBFP representation could improve the rankings of molecules with novel active chemotypes.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Scaffold hopping performance within 1% (A) and 0.5% (B) for the five screening datasets comparing CBFP with the ECFP4 and MOE2D representations. CBFP: green, ECFP4: blue, MOE2D: yellow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We also calculated the similarity threshold for the CBFP representation. When 25% scaffolds were recalled, the mean Tanimoto similarity value was about 0.70. Meanwhile, under the same conditions, the similarity value for ECFP4 was about 0.38. Additionally, the similarity values of molecules near the top 0.5 and 1% were about 0.63 and 0.58, respectively. These data could be employed as a reference for virtual screening.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">2.4.  Prospective Validation</h3><div class="NLM_p">In addition to the retrospective comparison and validation of the 17 benchmarks and 5 screening datasets, it is essential to test the performance of the novel bioactivity representation in practice. Therefore, a prospective experiment aiming at searching for novel PARP1 inhibitors was designed and implemented. As a hot target in cancer therapy, PARP1 is tremendously important for a vast variety of cellular functions, such as DNA damage repair, gene transcription, and cell apoptosis.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Until now, four PARP1 inhibitors (olaparib, niraparib, rucaparib, and talazoparib, see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_001.pdf" class="ext-link">Table S2</a>) have been approved by FDA, mainly for the treatment of BRCA-mutated advanced ovarian and breast cancer, with their applications in other cancers in clinical evaluation.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> From the perspective of chemotype, most inhibitors fall into three categories: phthalazinone derivatives, benzimidazole formamide derivatives, and tricyclic lactam indole derivatives. Considering the clinical importance and limited chemical space of anticancer drugs, searching for novel chemotypes may provide alternative starting points for lead optimization and improve adverse reactions commonly found in marketed drugs.</div><div class="NLM_p">Olaparib, the first approved PARP1 inhibitor launched by AstraZeneca, was chosen as the query molecular in the similarity searching with CBFP representation. A library containing 0.5 M commercially available synthetic small molecules downloaded from ChemBridge was used as the screening data set. Due to the computational complexity and storage space for these compounds, a two-step filter operation was carried out before similarity searching to retain only promising compounds that conformed to historic expert experience. First, we curated the whole screening data set and removed those compounds violating Lipinski’s rule of five. Then, the washed screening data set was docked to the prepared protein 4L6S using the standard precision (SP) mode of Glide.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> The compounds were ranked according to their Glide docking scores. We retained a total of 12,763 molecules with docking scores lower than −8 and calculated their 831-bit CBFP (the PARP-1 protein was removed from the CBFP). The Tanimoto similarities between the query molecule and the screened molecules were calculated and used as the ranking criteria. A similarity value of 1.0 indicates that the molecule has an identical position in the bioactivity space as the query molecule, whereas a value of 0 means it is far from the query molecule in the bioactivity space. The 50 top-ranked compounds spanning 46 distinct molecular scaffolds were collected. Considering the potential redundancy of chemotypes, for the two compounds sharing the same Murcko scaffold, only the higher-ranked one was maintained to represent its scaffold. The same operation was carried out on analogues. All the compounds successfully passed the PAINS (pan assay interference compounds) prediction. Ultimately, 35 compounds (see the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_002.xlsx" class="ext-link">Supporting Information</a>) were retained and purchased from the commercial supplier (ChemBridge) with a minimum of 95% purity as determined by HPLC.</div><div class="NLM_p">Guided by the diverse chemotypes and the high biological similarities with olaparib, these compounds were evaluated in cellular assays using MDA-MB-436 cancer cell lines (BRCA1-deficient breast cancer cells) and the antiproliferation activities of these hit compounds were demonstrated as shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A. Surprisingly, most compounds selected by the CBFP-based similarity searching exhibited antiproliferative activity against BRCA1-deficient cells: 25 of the 35 compounds attained statistical difference compared to dimethyl sulfoxide (DMSO) alone (<i>P</i> < 0.05, one-tailed <i>t</i>-test). Among them, eight compounds showed more than 50% inhibition, suggesting that their IC<sub>50</sub> were lower than 10 μM. These potent compounds had diverse chemical scaffolds although they shared some fragments with approved drugs. Additionally, olaparib displayed strong antiproliferative activity with 84% inhibition, which is consistent with previous reports. Unexpectedly, hit compounds 6, 9, and 24 showed more potent antiproliferative activity than olaparib, and their dose–response curves are shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B. Compound 6 had an IC<sub>50</sub> of 161.2 nM, far lower than those of compounds 24 and 9. For this promising growth-inhibition activity, it was submitted to the PARP1 inhibitory study using chemiluminescence assay to confirm our assumption that its antiproliferative activity was indeed caused by its PARP1 inhibitory capability. Compound 6 displayed excellent PARP1 inhibitory activity with an IC<sub>50</sub> value of 0.263 nM, while that of olaparib was 1.719 nM (the value provided by the kit manufacturer was 1.8 nM). It is worth mentioning that the novel hit compound 6 possessed a distinct scaffold, which never appeared in the PARP-1 binders of ChEMBL27. The reverse docking simulation in the PARP1 active site showed that the quinazolin-4(3<i>H</i>)-one fragment formed two hydrogen bonds with Ser904 and Gly863, π–π stacking interactions with Tyr907, and the single benzene could form π–H stacking interactions with Asp766, reconfirming the favorable binding of compound 6 to PARP-1. Additionally, reverse target fishing was carried out using four different web services: Similarity Ensemble Approach (SEA),<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> SwissTargetPrediction,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> TargetNet,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> and PharmMapper.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The SMILES of hit compound 6 was submitted to these servers to search for the potential targets based on the similar property principle (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_001.pdf" class="ext-link">Table S3</a>). Among the three methods, SwissTargetPrediction and TargetNet could not find PARP1 in their top 10 predictive targets, and SEA could only predict PARP-1 as the tenth likely target. However, when olaparib was used as the query molecule, these methods could easily find PARP1 as its potential target. Regretfully, the 3D pharmacophore similarity-based PharmMapper could not find PARP-1 for both compound 6 and olaparib (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_001.pdf" class="ext-link">Table S4</a>). These results demonstrated that traditional similarity methods could not identify our hit compound 6 as a potential PARP1 inhibitor. In conclusion, utilizing bioactivity space similarity leads to the discovery of compound 6, demonstrating that it is completely feasible for use in finding novel active scaffolds by CBFP-based similarity searching.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Prospective evaluation for finding novel PARP1 inhibitors. (A) Cellular growth-inhibitory activities of the 35 hit compounds. (B) Dose–response curves for the most potent hit compounds 6, 24, and 39.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">3.  Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30613" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30613" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this study, we introduced the CBFP, an informative, high-dimensional, resource-conserving, and universal molecular representation that incorporates the predictive bioactivities of multiple QSAR models to characterize the bioactivity space of compounds for the discovery of novel scaffolds.</div><div class="NLM_p">The scaffold hopping abilities of CBFP representation and nine other chemical descriptors have been systematically benchmarked in 17 different query datasets paired with five screening datasets by ranking large databases in a decreasing order of Tanimoto similarity values with the query molecule. The CBFP representation could highly discriminate structural descriptors in terms of its efficacy for improving the rankings of molecules with novel scaffolds. In the prospective experiment, the identified eight hit compounds exhibit novel active scaffolds, and the most potent hit compound 6 could show more efficacious inhibitory activity against PARP-1 and breast cancer cells with respect to olaparib, suggesting it has the potential to show activity in subsequent in vivo and cellular mechanistic studies. Both compound 6 and the other two hits (Compounds 9 and 24) with good cellular inhibition could be regarded as starting points for medicinal chemistry optimization. These results support the use of the CBFP for discovering novel active scaffolds and broadening the chemical space of current drugs. It is worth reemphasizing that these hit compounds exhibit different chemical scaffolds from current active compounds, and very few of these compounds could be found in the final candidate lists of other chemical descriptors.</div><div class="NLM_p">Despite the superiority in generality and financial expenditure, there are two important concepts that require consideration in employing the predictions of QSAR models to characterize the bioactivity space of compounds. One is the reliability of models. In our CBFP construction, the accuracy of all the 832 × 9 classification models was in the range of 70.0–99.8%, providing adequate separation of high-active and low-active compounds. Even so, we understand that the reliability of predictive results of cheminformatics models is not equal to that of biochemical assays. However, it is demanding to require it to possess good generality and reliability simultaneously, as an increase in one quality would imply a decrease in the other. We can minimize the distance between predictive activity and real interactions by optimizing these classification models to the best of our ability. Another concept is the weak correlation with the chemical structure. CBFP representation is defined as a kind of bioactivity space characterization and the dependency on chemical structures is kept as low as possible. However, the generation of CBFP representation is essentially based on QSAR models and chemical descriptors (MOE2D, CATS, and MACCS), thereby having certain distant connections with structural information of compounds. In any case, the superiority of the CBFP relative to other pure chemical descriptors in scaffold hopping has been confirmed by our retrospective and prospective validations.</div><div class="NLM_p last">Finally, it is necessary to stress the importance of prospective validation. Up to now, many strategies of ligand- or structure-based virtual screening have reported impressive performance in retrospective validation without proving useful beyond this. Once applied in practice, their results are usually not up to expectation. To determine the transferability of the performance in real practice and rule out inappropriate models, it is recommended that proper prospective validation be implemented alongside retrospective validation.<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">4.  Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75141" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75141" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11">4.1.  Definition of Molecular Scaffolds</h3><div class="NLM_p last">The term “molecular scaffold” is designed to depict the core structure of a molecule. In this study, the Bemis and Murcko scaffold<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> was used for scaffold hopping analysis. It is defined as pure ring systems and ring systems connected by linkers. The side-chains are usually excluded in scaffold calculation and chain compounds usually do not have Murcko scaffolds. From the perspective of the study endpoint, the Murcko scaffold is qualified to cover all types of scaffold hopping, including heterocycle replacement, ring opening and closure, pseudopeptides and peptidomimetics, and topological/shape-based scaffold hopping.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In this study, the Murcko scaffold calculation was implemented in the KNIME platform (version 4.1).<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">4.2.  Data Set and Preparation</h3><div class="NLM_p">The query data set was downloaded from the article reported by Vogt et al.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The whole data set consists of 1675 unique small molecules for 17 different targets collected from the ChEMBL database. A target subset contains a minimum of 10 distinct scaffolds and each of which has 5 corresponding molecules. All the active compounds satisfy three rules: (1) the potency (<i>K</i><sub>i</sub> or IC<sub>50</sub> value) should be at least 1 μM; (2) the number of the total nonhydrogen atoms is not more than 50; and (3) the number of the nonhydrogen atoms of the side-chains should be less than that of the scaffold. We later removed molecules with unidentified SMILES, leading to a total of 1660 molecules belonging to 332 scaffolds. The information of the query data set is listed in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_001.pdf" class="ext-link">Table S5</a>.</div><div class="NLM_p last">To better simulate a real scenario, five molecule datasets with different sizes and sources were collected as the screening datasets, including ChEMBL (85,644, from ChEMBL), Natural Products (120,686, from ZINC), ChemBlock (198,643, from Chemical Block), SPECS (212,863, from SPECS) and ASINEX (567,106, from ASINEX). All the screening molecules were washed to ensure that the screening datasets were truly nonredundant from the query data set. As a bias control, all the screened molecules follow the same ranges of heavy atoms of the query molecules.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">4.3.  CBFP Construction</h3><div class="NLM_p">The CBFP of a compound is designed as a binary encoding of the predicted bioactivities toward 832 human proteins. The workflow of the construction of the CBFP is illustrated in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_001.pdf" class="ext-link">Figure S3</a> and elaborated below.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14">4.3.1.  Data Collection and Preparation</h4><div class="NLM_p last">The drug giant AstraZeneca has ever analyzed the in vitro bioactivity data extracted from the literature and patents, where 1736 human proteins were considered to be candidate drug targets.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> In this study, these proteins were selected as the candidate targets for CBFP construction. Two publicly available databases, ChEMBL<a onclick="showRef(event, 'ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref52 ref53">(52,53)</a> and BindingDB,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> were selected as the sources of small molecules for model training. Taking the names of proteins as the keywords, we collected both biochemical and cellular assay data of each target, which was the usual practice of QSAR modeling. Then, a series of preparations were applied: (1) the compounds without concrete bioassay annotations (IC<sub>50</sub>, EC<sub>50</sub>, Ki values) or structures were deleted; (2) duplicated molecules were identified and filtered off based on the molecular InChIkey format; (3) if a molecule was reported with two or more bioactivity entries, the arithmetic mean of these values was employed provided that the fluctuation was less than 0.3, otherwise discarded; and (4) target datasets with less than 100 compounds were removed from the candidate protein list due to limited chemical space. Finally, 0.81 M molecules spanning 832 protein targets were collected, with an average of 980 compounds per target data set. The total data set contained 0.54 M distinct compounds and 0.48 M Murcko scaffolds, indicating a diverse chemical space. The detailed information on these QSAR datasets has been provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_003.csv" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15">4.3.2.  Model Training and Evaluation</h4><div class="NLM_p">In this study, three different types of descriptors, CATS, MACCS, and MOE2D, were used to describe these compounds. The fingerprint calculation was implemented in Molecular Operating Environment software (MOE, <i>version 2018.01</i>).<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Three popular ML algorithms, including random forest (RF),<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Adaptive Boosting (AdaBoost),<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> and gradient boosting machine (GBM),<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> were used to construct the QSAR classification models with R packages, randomForest,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> adabag,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> and gbm.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a></div><div class="NLM_p last">For each target, three molecular representations and three ML algorithms were combined, thereby producing nine classification models. To ensure the balance between positive and negative samples and avoid potential data bias, for each subset, we employed the median bioactivity value as the cutoff point, dividing the compounds into two equal parts. The high bioactivity part was considered as the positive set (label = 1) and the low bioactivity part as the negative set (label = 0). Then the data set was split into training set:test set = 8:2. Hyperparameter tuning was performed via the grid search method to optimize the area under the receiver operating characteristic curve (AUC) value. The modeling process was repeated 10 times and the final prediction of each compound was the arithmetic mean of the 10 predictions. Finally, a total of 832 × 9 SAR models were established and the accuracy values on the test sets were mainly in the range of 75.9–98.5% (see the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_003.csv" class="ext-link">Supporting Information</a>). The R code implementing the CBFP and the default model.zip file has been deposited in GitHub (<a href="https://github.com/xiongguoli/Computational-Bioactivity-Fingreprint" class="extLink">https://github.com/xiongguoli/Computational-Bioactivity-Fingreprint</a>).</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16">4.3.3.  CBFP Calculation</h4><div class="NLM_p last">Molecule descriptors (CATS, MACCS, and MOE2D) were generated for the whole query data set and five screening datasets using MOE. Then, these molecules were predicted by the 832 × 9 SAR models one by one. To achieve more accurate predictions, we employed the consensus score to represent the potential activity against the protein target by averaging the predicted values from nine individual models. The consensus score of all the proteins were assembled as an 832-bit vector and each bit is in a scale of 0∼1. Then, this feature vector was translated into a standard binary fingerprint by using a cutoff, where 1 means that this molecule has a strong possibility to be high active to this protein and 0 means low active. In this study, we employed 0.5 as the appropriate cutoff, which was derived from the comparison of many thresholds (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_001.pdf" class="ext-link">Figure S2</a>). This 832-bit fingerprint was named computational bioactivity fingerprint (CBFP) for the bioactivity characterization of small molecules.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17">4.4.  Baseline Descriptors</h3><div class="NLM_p last">The scaffold hopping potential of the CBFP representation was benchmarked against nine chemical descriptors, which were divided into three groups by focusing on their degrees of structure simplification. Basic structure recordings, including atoms, atom connections, and structural keys, were classified as low-level representations (AtomPairs, MACCS, FP2, and ECFP4). More extensive recordings, including pharmacophore, pharmacophore combinations, and distances, were classified as median-level representation (Gpi-DAPH3, TGD, TGT, and CATS). Finally, comprehensive molecular property descriptors, such as molecular weight and logP, are classified as high-level representations (MOE2D). These molecular descriptors were calculated by the RDKit package (<i>version 2019.03.1,</i> windows platform) and MOE (<i>version 2018.1</i>).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18">4.5.  Similarity Searching</h3><div class="NLM_p">We used traditional similarity searching method to investigate the scaffold hopping ability of the CBFP and the other baseline chemical descriptors. As previously described, there were five different molecules of each scaffold. The five compounds were employed as the query molecules in turn and the remaining compounds of the other scaffolds within the same target were added to the screening data set. Therefore, a total of 1660 searching trials were carried out and analyzed. The similarities between a query molecule and the screened molecules were measured by the Tanimoto similarity coefficients paired with different molecular representations.</div><div class="NLM_p last">Here, two statistical parameters, SRR and AMRR,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> were utilized to evaluate the scaffold hopping potential and virtual screening performance, respectively. The SRR and AMRR values are defined as the ratio between the distinct scaffolds/active compounds retrieved within the threshold and the total active scaffolds/active compounds added to the screening data set. The thresholds were set to the top 100 and 5000 compounds for all the screening datasets. It should be clarified that the potential of scaffold hopping described in this work does not refer to the magnitude of hopping but the ability to discover novel chemotypes. The similarity searching and evaluation workflow were implemented in the KNIME platform (version 4.1).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19">4.6.  In Vitro Antiproliferation Assay and PARP-1 Inhibition Assays</h3><div class="NLM_p">The human-cancer-cell line MDA-MB-436 was maintained in Leibovitz’s L-15 medium containing 10 μg/mL insulin, 16 μg/mL glutathione, and 10% (v/v) fetal bovine serum at 37 °C in 100% air incubator. Cell proliferation assay was determined by Cell Counting-Kit 8 (CCK8) assay. MDA-MB-436 cells were passaged the day before dosing into a 96-well plate, allowed to grow for 12 h, and treated with different concentrations for 11 days. After incubation, 10 μL of CCK8 was added to each well, which was incubated in an incubator for 3 h, and the absorbance at 450 nM was measured with a microplate reader. The inhibition rate (%) = [1–(OD drug – OD blank)]/(OD DMSO – OD blank) × 100%. The IC<sub>50</sub> values were calculated using nonlinear regression with a normalized dose–response fit using the Prism GraphPad software.</div><div class="NLM_p last">PARP1 inhibition assays were carried out according to chemiluminescence procedures. The PARP1 enzymatic activity of compounds in a cell-free system was determined by PARP1 chemiluminescence activity assay in 384-well plates. Each well was coated with histone (25 μL) diluted in 125 μL of PBS buffer and incubated at 4 °C overnight. After incubation, the plate was washed three times using 100 μL PBST buffer per well and blocked the wells by adding 100 μL of blocking buffer at room temperature for 60–90 min. The next step is the ribosylation reaction. The master mixture including 10× PARP buffer (1.25 μL), 10× PARP assay mixture (1.25 μL), activated DNA (2.5 μL), and distilled water (7.5 μL) should be added to every well followed by adding 2.5 μL of the compound at varying concentrations or the inhibitor buffer (no inhibitor). The reaction was initiated by adding 10 μL of 2 ng/μL PARP1 enzyme to the wells positive control and test inhibitor, and 10 μL of PARP buffer was added to the blank wells. The last step is detecting chemiluminescence. Diluted streptavidin-HRP (25 μL) was added to each well and incubated for 30 min at room temperature. Just before use, 25 μL HRP chemiluminescent substrate A and 25 μL HRP chemiluminescent substrate B were mixed on ice and added to each well. Immediately, the plate should be read in a luminometer or microtiter-plate reader capable of reading chemiluminescence. The inhibition rate of PARP1 enzymatic activity was calculated as (inhibitor – control)/(positive – control) × 100%. The concentration required for 50% inhibition of PARP1 enzymatic activity (IC<sub>50</sub>) was calculated using nonlinear regression with a normalized dose–response fit using Prism GraphPad software.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00234" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87380" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87380" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00234?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00234</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">AMRR results for different screening datasets (Table S1 and Figure S1); PARP-1 inhibitors approved by FDA (Table S2); reverse target fishing of compound 6 and olaparib (Tables S3 and S4); information of the query data set (Table S5); the threshold of CBFP representation (Figure S2); workflow for CBFP construction (Figure S3) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_001.pdf" class="ext-link">PDF</a>).</p></li><li><p class="inline">Molecular formula strings of all the compounds tested and associated biochemical data (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_002.xlsx" class="ext-link">XLSX</a>).</p></li><li><p class="inline">Information on the QSAR models of CBFP representation (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_003.csv" class="ext-link">CSV</a>).</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_001.pdf">jm1c00234_si_001.pdf (635.56 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_002.xlsx">jm1c00234_si_002.xlsx (144.81 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_003.csv">jm1c00234_si_003.csv (2.44 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00234" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87008" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87008" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ting-Jun Hou</span> - <span class="hlFld-Affiliation affiliation">Innovation
Institute for Artificial Intelligence in Medicine of Zhejiang University,
College of Pharmaceutical Sciences, Zhejiang
University, Hangzhou, Zhejiang 310058, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b1c5d8dfd6dbc4dfd9dec4f1cbdbc49fd4d5c49fd2df"><span class="__cf_email__" data-cfemail="9eeaf7f0f9f4ebf0f6f1ebdee4f4ebb0fbfaebb0fdf0">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dong-Sheng Cao</span> - <span class="hlFld-Affiliation affiliation">Xiangya
School of Pharmaceutical Sciences, Central
South University, Changsha, Hunan 410003, China</span>; 
    <span class="hlFld-Affiliation affiliation">Hunan
Key Laboratory of Diagnostic and Therapeutic Drug Research for Chronic
Diseases, Central South University, Changsha, Hunan 410013, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
for Advancing Translational Medicine in Bone and Joint Diseases, School
of Chinese Medicine, Hong Kong Baptist University, 999077 Hong Kong
SAR, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3604-3785" title="Orcid link">https://orcid.org/0000-0003-3604-3785</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ef809d868a819b8e83c28c8b9cafded9dcc18c8082"><span class="__cf_email__" data-cfemail="147b667d717a60757839777067542522273a777b79">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guo-Li Xiong</span> - <span class="hlFld-Affiliation affiliation">Xiangya
School of Pharmaceutical Sciences, Central
South University, Changsha, Hunan 410003, China</span>; 
    <span class="hlFld-Affiliation affiliation">Hunan
Key Laboratory of Diagnostic and Therapeutic Drug Research for Chronic
Diseases, Central South University, Changsha, Hunan 410013, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yue Zhao</span> - <span class="hlFld-Affiliation affiliation">Xiangya
School of Pharmaceutical Sciences, Central
South University, Changsha, Hunan 410003, China</span>; 
    <span class="hlFld-Affiliation affiliation">Hunan
Key Laboratory of Diagnostic and Therapeutic Drug Research for Chronic
Diseases, Central South University, Changsha, Hunan 410013, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lu Liu</span> - <span class="hlFld-Affiliation affiliation">Xiangya
School of Pharmaceutical Sciences, Central
South University, Changsha, Hunan 410003, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhong-Ye Ma</span> - <span class="hlFld-Affiliation affiliation">Xiangya
School of Pharmaceutical Sciences, Central
South University, Changsha, Hunan 410003, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ai-Ping Lu</span> - <span class="hlFld-Affiliation affiliation">Institute
for Advancing Translational Medicine in Bone and Joint Diseases, School
of Chinese Medicine, Hong Kong Baptist University, 999077 Hong Kong
SAR, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan Cheng</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, The Second Xiangya Hospital, Central South University, Changsha, Hunan 410003, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>D.-S.C., T.-J.H., and G.-L.X. designed the study. G.-L.X. and L.L. constructed the fingerprints and performed the retrospective analysis. Y.Z. and Z.-Y.M. performed the prospective analysis and elaborated the results. G.-L.X. and Y.Z. wrote the manuscript and prepared the diagrams and figures. T.-J.H., A.-P.L., and Y.C. helped check and improve the manuscript. All authors contributed to manuscript revision and approved the final version. G.-L.X. and Y.Z. contributed equally to this paper.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74313" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74313" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by Changsha Municipal Natural Science Foundation (kq2014144), Changsha Science and Technology Bureau Project (kq2001034), Key R&D Program of Zhejiang Province (2020C03010), National Science Foundation of China (81773632), Zhejiang Provincial Natural Science Foundation of China (LZ19H300001), and HKBU Strategic Development Fund Project (SDF19-0402-P02). The study was approved by the University’s Review Board.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AMRR</td><td class="NLM_def"><p class="first last">active molecule recall rate</p></td></tr><tr><td class="NLM_term">CBFP</td><td class="NLM_def"><p class="first last">computational bioactivity fingerprint</p></td></tr><tr><td class="NLM_term">CI</td><td class="NLM_def"><p class="first last">confidence interval</p></td></tr><tr><td class="NLM_term">HTS-FP</td><td class="NLM_def"><p class="first last">high-throughput screening fingerprints</p></td></tr><tr><td class="NLM_term">ML</td><td class="NLM_def"><p class="first last">machine learning</p></td></tr><tr><td class="NLM_term">MOE</td><td class="NLM_def"><p class="first last">molecular operating environment software</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan assay interference compounds</p></td></tr><tr><td class="NLM_term">PARP1</td><td class="NLM_def"><p class="first last">poly (ADP-ribose) polymerase 1</p></td></tr><tr><td class="NLM_term">QSAR</td><td class="NLM_def"><p class="first last">quantitative structure–activity relationship</p></td></tr><tr><td class="NLM_term">SRR</td><td class="NLM_def"><p class="first last">scaffold recall rata</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40112" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40112" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 61 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruddigkeit, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Deursen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J. L.</span></span> <span> </span><span class="NLM_article-title">Enumeration of 166 billion organic small molecules in the chemical universe database GDB-17</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2864</span>– <span class="NLM_lpage">2875</span>, <span class="refDoi"> DOI: 10.1021/ci300415d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300415d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2864-2875&author=L.+Ruddigkeitauthor=R.+van+Deursenauthor=L.+C.+Blumauthor=J.+L.+Reymond&title=Enumeration+of+166+billion+organic+small+molecules+in+the+chemical+universe+database+GDB-17&doi=10.1021%2Fci300415d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fci300415d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300415d%26sid%3Dliteratum%253Aachs%26aulast%3DRuddigkeit%26aufirst%3DL.%26aulast%3Dvan%2BDeursen%26aufirst%3DR.%26aulast%3DBlum%26aufirst%3DL.%2BC.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DEnumeration%2520of%2520166%2520billion%2520organic%2520small%2520molecules%2520in%2520the%2520chemical%2520universe%2520database%2520GDB-17%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2012%26volume%3D52%26spage%3D2864%26epage%3D2875%26doi%3D10.1021%2Fci300415d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wallach, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heifets, A.</span></span> <span> </span><span class="NLM_article-title">Most Ligand-Based Classification Benchmarks Reward Memorization Rather than Generalization</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">916</span>– <span class="NLM_lpage">932</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00403</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00403" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=916-932&author=I.+Wallachauthor=A.+Heifets&title=Most+Ligand-Based+Classification+Benchmarks+Reward+Memorization+Rather+than+Generalization&doi=10.1021%2Facs.jcim.7b00403"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00403%26sid%3Dliteratum%253Aachs%26aulast%3DWallach%26aufirst%3DI.%26aulast%3DHeifets%26aufirst%3DA.%26atitle%3DMost%2520Ligand-Based%2520Classification%2520Benchmarks%2520Reward%2520Memorization%2520Rather%2520than%2520Generalization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2018%26volume%3D58%26spage%3D916%26epage%3D932%26doi%3D10.1021%2Facs.jcim.7b00403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gastreich, M.</span></span> <span> </span><span class="NLM_article-title">The next level in chemical space navigation: going far beyond enumerable compound libraries</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1148</span>– <span class="NLM_lpage">1156</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1016%2Fj.drudis.2019.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=30851414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkslaksb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=1148-1156&author=T.+Hoffmannauthor=M.+Gastreich&title=The+next+level+in+chemical+space+navigation%3A+going+far+beyond+enumerable+compound+libraries"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The next level in chemical space navigation: going far beyond enumerable compound libraries</span></div><div class="casAuthors">Hoffmann, Torsten; Gastreich, Marcus</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1148-1156</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Recent innovations have brought pharmacophore-driven methods for navigating virtual chem. spaces, the size of which can reach into the billions of mols., to the fingertips of every chemist.  There has been a paradigm shift in the underlying computational chem. that drives chem. space search applications, incorporating intelligent reaction knowledge into their core so that they can readily deliver com. available mols. as nearest neighbor hits from within giant virtual spaces.  These vast resources enable medicinal chemists to execute rapid scaffold-hopping expts., rapid hit expansion, and structure-activity relationship (SAR) exploitation in largely intellectual property (IP)-free territory and at unparalleled low cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodA_WnK9Z3jbVg90H21EOLACvtfcHk0lh0lMWz_jv8qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkslaksb4%253D&md5=d707299b8c6e0ace79f27172147517e3</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2019.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2019.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHoffmann%26aufirst%3DT.%26aulast%3DGastreich%26aufirst%3DM.%26atitle%3DThe%2520next%2520level%2520in%2520chemical%2520space%2520navigation%253A%2520going%2520far%2520beyond%2520enumerable%2520compound%2520libraries%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26spage%3D1148%26epage%3D1156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, C. M.</span></span> <span> </span><span class="NLM_article-title">Chemical space and biology</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>432</i></span>,  <span class="NLM_fpage">824</span>– <span class="NLM_lpage">828</span>, <span class="refDoi"> DOI: 10.1038/nature03192</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1038%2Fnature03192" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=15602547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=824-828&author=C.+M.+Dobson&title=Chemical+space+and+biology&doi=10.1038%2Fnature03192"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical space and biology</span></div><div class="casAuthors">Dobson, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">824-828</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. space - which encompasses all possible small org. mols., including those present in biol. systems - is vast.  So vast, in fact, that so far only a tiny fraction of it has been explored.  Nevertheless, these explorations have greatly enhanced our understanding of biol., and have led to the development of many of today's drugs.  The discovery of new bioactive mols., facilitated by a deeper understanding of the nature of the regions of chem. space that are relevant to biol., will advance our knowledge of biol. processes and lead to new strategies to treat disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolNmRTVWoSDrVg90H21EOLACvtfcHk0lh0lMWz_jv8qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jN&md5=b951fff4aa4877682756fb1831bff09d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature03192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03192%26sid%3Dliteratum%253Aachs%26aulast%3DDobson%26aufirst%3DC.%2BM.%26atitle%3DChemical%2520space%2520and%2520biology%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D824%26epage%3D828%26doi%3D10.1038%2Fnature03192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bohacek, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMartin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span> <span> </span><span class="NLM_article-title">The art and practice of structure-based drug design: a molecular modeling perspective</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1098-1128(199601)16:1<3::AID-MED1>3.0.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=8788213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1996&pages=3-50&author=R.+S.+Bohacekauthor=C.+McMartinauthor=W.+C.+Guida&title=The+art+and+practice+of+structure-based+drug+design%3A+a+molecular+modeling+perspective&doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The art and practice of structure-based drug design: a molecular modeling perspective</span></div><div class="casAuthors">Bohacek, Regine S.; McMartin, Colin; Guida, Wayne C.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-50</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review with 120 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmfrWdtPosdrVg90H21EOLACvtfcHk0ljtyOzMCp58rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D&md5=edfa592791c111479117feed600dead6</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291098-1128%2528199601%252916%253A1%253C3%253A%253AAID-MED1%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DBohacek%26aufirst%3DR.%2BS.%26aulast%3DMcMartin%26aufirst%3DC.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26atitle%3DThe%2520art%2520and%2520practice%2520of%2520structure-based%2520drug%2520design%253A%2520a%2520molecular%2520modeling%2520perspective%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D1996%26volume%3D16%26spage%3D3%26epage%3D50%26doi%3D10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renner, S.</span></span> <span> </span><span class="NLM_article-title">Scaffold-Hopping: How Far Can You Jump?</span>. <i>QSAR Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1162</span>– <span class="NLM_lpage">1171</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1002%2Fqsar.200610091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsFCjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=1162-1171&author=G.+Schneiderauthor=P.+Schneiderauthor=S.+Renner&title=Scaffold-Hopping%3A+How+Far+Can+You+Jump%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold-hopping: how far can you jump?</span></div><div class="casAuthors">Schneider, Gisbert; Schneider, Petra; Renner, Steffen</div><div class="citationInfo"><span class="NLM_cas:title">QSAR & Combinatorial Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1162-1171</span>CODEN:
                <span class="NLM_cas:coden">QCSSAU</span>;
        ISSN:<span class="NLM_cas:issn">1611-020X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Finding new isofunctional chemotypes with the aim of identifying new lead candidates remains a challenging task in medicinal chem.  Different virtual screening techniques were shown to be useful for this scaffold-hopping process.  We have compiled recent examples of scaffold-hops that were achieved by virtual screening.  Most of these techniques rely on some sort of similarity estn. between known ref. mols. and screening compds., some include receptor-structure information and scoring of the ligand - receptor interaction.  In this review, we discuss current scaffold-hopping strategies and pinpoint potential future directions.  Challenges for future research lie in the identification of an appropriate level of abstraction from an atomistic mol. representation to allow for the detection of isofunctional chemotypes, the rational design of chemotype diversity in screening libraries, and an understanding of the distribution of similarity scores for a given screening library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRGMZFw4ah2rVg90H21EOLACvtfcHk0ljtyOzMCp58rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsFCjtg%253D%253D&md5=d859a9eeec334ee140d2cf6423bf3b21</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fqsar.200610091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fqsar.200610091%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DRenner%26aufirst%3DS.%26atitle%3DScaffold-Hopping%253A%2520How%2520Far%2520Can%2520You%2520Jump%253F%26jtitle%3DQSAR%2520Comb.%2520Sci.%26date%3D2006%26volume%3D25%26spage%3D1162%26epage%3D1171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funatsu, K.</span></span> <span> </span><span class="NLM_article-title">Exploring Topological Pharmacophore Graphs for Scaffold Hopping</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>2073–2081</i></span>,  <span class="NLM_fpage">60</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2073%E2%80%932081&publication_year=2020&pages=60&author=H.+Nakanoauthor=T.+Miyaoauthor=K.+Funatsu&title=Exploring+Topological+Pharmacophore+Graphs+for+Scaffold+Hopping"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNakano%26aufirst%3DH.%26aulast%3DMiyao%26aufirst%3DT.%26aulast%3DFunatsu%26aufirst%3DK.%26atitle%3DExploring%2520Topological%2520Pharmacophore%2520Graphs%2520for%2520Scaffold%2520Hopping%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D2073%25E2%2580%25932081%26spage%3D60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neidhart, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, G.</span></span> <span> </span><span class="NLM_article-title">“Scaffold-Hopping” by Topological Pharmacophore Search: A Contribution to Virtual Screening</span>. <i>Angew. Chem. Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">2894</span>– <span class="NLM_lpage">2896</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1002%2F%28SICI%291521-3773%2819991004%2938%3A19%3C2894%3A%3AAID-ANIE2894%3E3.0.CO%3B2-F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10540384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A280%3ADC%252BC2sbhtVertA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=2894-2896&author=G.+Schneiderauthor=W.+Neidhartauthor=T.+Gillerauthor=G.+Schmid&title=%E2%80%9CScaffold-Hopping%E2%80%9D+by+Topological+Pharmacophore+Search%3A+A+Contribution+to+Virtual+Screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">"Scaffold-Hopping" by Topological Pharmacophore Search: A Contribution to Virtual Screening</span></div><div class="casAuthors">Schneider; Neidhart; Giller; Schmid</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2894-2896</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A chemically advanced template search (CATS) based on topological pharmacophore models has been developed as a technique for virtual screening.  This technique has successfully identified novel potent Ca(2+) antagonists (such as 2) that have a similar activity to 1 (a known T-channel blocking agent) in a library of several hundred thousand compounds on the basis of a correlation vector representation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRWkOLBNK38cedfST00W0BcfW6udTcc2ebAvu_fvpBQLLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbhtVertA%253D%253D&md5=c2d88f2b56ea8af78fb167641e254384</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819991004%2938%3A19%3C2894%3A%3AAID-ANIE2894%3E3.0.CO%3B2-F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819991004%252938%253A19%253C2894%253A%253AAID-ANIE2894%253E3.0.CO%253B2-F%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DNeidhart%26aufirst%3DW.%26aulast%3DGiller%26aufirst%3DT.%26aulast%3DSchmid%26aufirst%3DG.%26atitle%3D%25E2%2580%259CScaffold-Hopping%25E2%2580%259D%2520by%2520Topological%2520Pharmacophore%2520Search%253A%2520A%2520Contribution%2520to%2520Virtual%2520Screening%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%2520Engl.%26date%3D1999%26volume%3D38%26spage%3D2894%26epage%3D2896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bravo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berrocoso, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and development of tramadol for the treatment of pain</span>. <i>Expert. Opin. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1281</span>– <span class="NLM_lpage">1291</span>, <span class="refDoi"> DOI: 10.1080/17460441.2017.1377697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1080%2F17460441.2017.1377697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=28920461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFajtbfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1281-1291&author=L.+Bravoauthor=J.+A.+Micoauthor=E.+Berrocoso&title=Discovery+and+development+of+tramadol+for+the+treatment+of+pain&doi=10.1080%2F17460441.2017.1377697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and development of tramadol for the treatment of pain</span></div><div class="casAuthors">Bravo, Lidia; Mico, Juan Antonio; Berrocoso, Esther</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1281-1291</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">: Tramadol is an opioid drug that, unlike classic opioids, also modulates the monoaminergic system by inhibiting noradrenergic and serotoninergic reuptake.  For this reason, tramadol is considered an atypical opioid.  These special pharmacol. characteristics have made tramadol one of the most commonly prescribed analgesic drugs to treat moderate to severe pain.: The aim of this review is to provide a historical description of the biochem., pharmacokinetics and particularly, the mechanisms of action of tramadol.  In addn., a summary is offered of the analgesic effects of tramadol in a variety of animal models of acute and chronic pain.  Finally, clin. studies that demonstrate the efficacy and safety of tramadol in the treatment of pain are also assessed.: The discovery that tramadol combines opioid and monoaminergic effects represented a milestone in the evolution of pain treatment.  Given its 'mild effect' on opioid receptors, tramadol induces fewer side effects than classic opioids.  Tramadol produces satisfactory analgesia against various types of pain and it is currently approved for the treatment of moderate to severe pain.  Thus, the combination of monoamine and opioid mechanisms opens new avenues for the design of innovative analgesics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLiCyb6lVHW7Vg90H21EOLACvtfcHk0lggeITFmVym5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFajtbfP&md5=ebb4c6df453ff1a37096ed155d26fae5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1080%2F17460441.2017.1377697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2017.1377697%26sid%3Dliteratum%253Aachs%26aulast%3DBravo%26aufirst%3DL.%26aulast%3DMico%26aufirst%3DJ.%2BA.%26aulast%3DBerrocoso%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520development%2520of%2520tramadol%2520for%2520the%2520treatment%2520of%2520pain%26jtitle%3DExpert.%2520Opin.%2520Drug%2520Discov.%26date%3D2017%26volume%3D12%26spage%3D1281%26epage%3D1291%26doi%3D10.1080%2F17460441.2017.1377697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Kuhrt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoernig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanrikulu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinhilber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Identification of natural-product-derived inhibitors of 5-lipoxygenase activity by ligand-based virtual screening</span>. <i>J. Med. Pharma. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">2640</span>– <span class="NLM_lpage">2646</span>, <span class="refDoi"> DOI: 10.1021/jm060655w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060655w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2640-2646&author=L.+Frankeauthor=O.+Schwarzauthor=L.+Muller-Kuhrtauthor=C.+Hoernigauthor=L.+Fischerauthor=S.+Georgeauthor=Y.+Tanrikuluauthor=P.+Schneiderauthor=O.+Werzauthor=D.+Steinhilberauthor=G.+Schneider&title=Identification+of+natural-product-derived+inhibitors+of+5-lipoxygenase+activity+by+ligand-based+virtual+screening&doi=10.1021%2Fjm060655w"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm060655w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060655w%26sid%3Dliteratum%253Aachs%26aulast%3DFranke%26aufirst%3DL.%26aulast%3DSchwarz%26aufirst%3DO.%26aulast%3DMuller-Kuhrt%26aufirst%3DL.%26aulast%3DHoernig%26aufirst%3DC.%26aulast%3DFischer%26aufirst%3DL.%26aulast%3DGeorge%26aufirst%3DS.%26aulast%3DTanrikulu%26aufirst%3DY.%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DWerz%26aufirst%3DO.%26aulast%3DSteinhilber%26aufirst%3DD.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DIdentification%2520of%2520natural-product-derived%2520inhibitors%2520of%25205-lipoxygenase%2520activity%2520by%2520ligand-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Pharma.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2640%26epage%3D2646%26doi%3D10.1021%2Fjm060655w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Scaffold Hopping</span>. <i>J. Med. Pharma. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1238</span>– <span class="NLM_lpage">1246</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01437</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01437" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1238-1246&author=Y.+Huauthor=D.+Stumpfeauthor=J.+Bajorath&title=Recent+Advances+in+Scaffold+Hopping&doi=10.1021%2Facs.jmedchem.6b01437"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01437%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DRecent%2520Advances%2520in%2520Scaffold%2520Hopping%26jtitle%3DJ.%2520Med.%2520Pharma.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1238%26epage%3D1246%26doi%3D10.1021%2Facs.jmedchem.6b01437" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baringhaus, K.-H.</span></span> <span> </span><span class="NLM_article-title">The scaffold hopping potential of pharmacophores</span>. <i>Drug Discovery Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e263</span>– <span class="NLM_lpage">e269</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1016%2Fj.ddtec.2010.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFyjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=e263-e269&author=G.+Hesslerauthor=K.-H.+Baringhaus&title=The+scaffold+hopping+potential+of+pharmacophores"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The scaffold hopping potential of pharmacophores</span></div><div class="casAuthors">Hessler, Gerhard; Baringhaus, Karl-Heinz</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">E263-E269</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The goal of scaffold hopping is to replace the chem. core structure by a novel chem. motif while keeping the biol. activity of the mol.  As pharmacophores define chem. features essential for biol. activity, they can be successfully employed to guide scaffold replacements.  To this end, various novel approaches have recently been developed and applied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1Rwswgk1y1bVg90H21EOLACvtfcHk0ljSHS4l7GDG6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFyjsL4%253D&md5=2621bdad53855b24427db53dd368a127</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2010.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2010.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DHessler%26aufirst%3DG.%26aulast%3DBaringhaus%26aufirst%3DK.-H.%26atitle%3DThe%2520scaffold%2520hopping%2520potential%2520of%2520pharmacophores%26jtitle%3DDrug%2520Discovery%2520Today%2520Technol.%26date%3D2010%26volume%3D7%26spage%3De263%26epage%3De269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yue, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Identification of novel small-molecule inhibitors targeting menin-MLL interaction, repurposing the antidiarrheal loperamide</span>. <i>Perkin 2</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">8503</span>– <span class="NLM_lpage">8519</span>, <span class="refDoi"> DOI: 10.1039/c6ob01248e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1039%2Fc6ob01248e" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=8503-8519&author=L.+Yueauthor=J.+Duauthor=F.+Yeauthor=Z.+Chenauthor=L.+Liauthor=F.+Lianauthor=B.+Zhangauthor=Y.+Zhangauthor=H.+Jiangauthor=K.+Chenauthor=Y.+Liauthor=B.+Zhouauthor=N.+Zhangauthor=Y.+Yangauthor=C.+Luo&title=Identification+of+novel+small-molecule+inhibitors+targeting+menin-MLL+interaction%2C+repurposing+the+antidiarrheal+loperamide&doi=10.1039%2Fc6ob01248e"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2Fc6ob01248e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc6ob01248e%26sid%3Dliteratum%253Aachs%26aulast%3DYue%26aufirst%3DL.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520novel%2520small-molecule%2520inhibitors%2520targeting%2520menin-MLL%2520interaction%252C%2520repurposing%2520the%2520antidiarrheal%2520loperamide%26jtitle%3DPerkin%25202%26date%3D2016%26volume%3D14%26spage%3D8503%26epage%3D8519%26doi%3D10.1039%2Fc6ob01248e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span>; <span class="NLM_string-name">Lu, W. C.</span>; <span class="NLM_string-name">Hu, J. C.</span>; <span class="NLM_string-name">Liu, Y. C.</span>; <span class="NLM_string-name">Zhang, C. H.</span>; <span class="NLM_string-name">Lian, F. L.</span>; <span class="NLM_string-name">Zhang, N. X.</span>; <span class="NLM_string-name">Meng, F. W.</span>; <span class="NLM_string-name">Luo, C.</span>; <span class="NLM_string-name">Chen, K. X.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterizations of Novel, Selective Histone Methyltransferase SET7 Inhibitors by Scaffold Hopping- and 2D-Molecular Fingerprint-Based Similarity Search</span>.  <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume">23</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.3390%2Fmolecules23030567" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&author=H.+Ding&author=W.+C.+Lu&author=J.+C.+Hu&author=Y.+C.+Liu&author=C.+H.+Zhang&author=F.+L.+Lian&author=N.+X.+Zhang&author=F.+W.+Meng&author=C.+Luo&author=K.+X.+Chen&title=Identification+and+Characterizations+of+Novel%2C+Selective+Histone+Methyltransferase+SET7+Inhibitors+by+Scaffold+Hopping-+and+2D-Molecular+Fingerprint-Based+Similarity+Search"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23030567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23030567%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DH.%26atitle%3DIdentification%2520and%2520Characterizations%2520of%2520Novel%252C%2520Selective%2520Histone%2520Methyltransferase%2520SET7%2520Inhibitors%2520by%2520Scaffold%2520Hopping-%2520and%25202D-Molecular%2520Fingerprint-Based%2520Similarity%2520Search%26jtitle%3DMolecules%26date%3D2018%26volume%3D23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totrov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span> <span> </span><span class="NLM_article-title">Docking to multiple pockets or ligand fields for screening, activity prediction and scaffold hopping</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1741</span>– <span class="NLM_lpage">1755</span>, <span class="refDoi"> DOI: 10.4155/fmc.14.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.4155%2Ffmc.14.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=25407367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFCit7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1741-1755&author=Y.+C.+Chenauthor=M.+Totrovauthor=R.+Abagyan&title=Docking+to+multiple+pockets+or+ligand+fields+for+screening%2C+activity+prediction+and+scaffold+hopping&doi=10.4155%2Ffmc.14.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Docking to multiple pockets or ligand fields for screening, activity prediction and scaffold hopping</span></div><div class="casAuthors">Chen, Yu-Chen; Totrov, Max; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1741-1755</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Background: Two recent technol. advances dramatically reducing the rate of false-negatives in activity prediction by docking flexible 3D models of compds. include multi-conformational docking (mPockDock) and the docking of candidates to at. property fields derived by co-crystd. ligands (mApfDock).  Results: The mApfDock and mPockDock provide the AUC of 90.4 and 83.8%, resp.  The mApfDock gave better performance when compds. required large induced-fit pocket changes unseen in crystallog., whereas the mPockDock is superior when the co-crystd. ligands do not represent sufficient chem. and binding location diversity.  Conclusion: Both approaches proved to be efficient for scaffold hopping; they are complementary when the coverage of the co-crystd. complexes is poor but become convergent when the complexes are diverse enough.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobPjvRDSmjvrVg90H21EOLACvtfcHk0lj5V9UE5aTLng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFCit7vE&md5=4e9d30a6ad9f1c68b05ac617e8463821</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.113%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%2BC.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DDocking%2520to%2520multiple%2520pockets%2520or%2520ligand%2520fields%2520for%2520screening%252C%2520activity%2520prediction%2520and%2520scaffold%2520hopping%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D1741%26epage%3D1755%26doi%3D10.4155%2Ffmc.14.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilton, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acklin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azzaoui, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span> <span> </span><span class="NLM_article-title">New methods for ligand-based virtual screening: use of data fusion and machine learning to enhance the effectiveness of similarity searching</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">470</span>, <span class="refDoi"> DOI: 10.1021/ci050348j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050348j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvV2h" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=462-470&author=J.+Hertauthor=P.+Willettauthor=D.+J.+Wiltonauthor=P.+Acklinauthor=K.+Azzaouiauthor=E.+Jacobyauthor=A.+Schuffenhauer&title=New+methods+for+ligand-based+virtual+screening%3A+use+of+data+fusion+and+machine+learning+to+enhance+the+effectiveness+of+similarity+searching&doi=10.1021%2Fci050348j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">New Methods for Ligand-Based Virtual Screening: Use of Data Fusion and Machine Learning to Enhance the Effectiveness of Similarity Searching</span></div><div class="casAuthors">Hert, Jerome; Willett, Peter; Wilton, David J.; Acklin, Pierre; Azzaoui, Kamal; Jacoby, Edgar; Schuffenhauer, Ansgar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-470</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Similarity searching using a single bioactive ref. structure is a well-established technique for accessing chem. structure databases.  This paper describes two extensions of the basic approach.  First, we discuss the use of group fusion to combine the results of similarity searches when multiple ref. structures are available.  We demonstrate that this technique is notably more effective than conventional similarity searching in scaffold-hopping searches for structurally diverse sets of active mols.; conversely, the technique will do little to improve the search performance if the actives are structurally homogeneous.  Second, we make the assumption that the nearest neighbors resulting from a similarity search, using a single bioactive ref. structure, are also active and use this assumption to implement approx. forms of group fusion, substructural anal., and binary kernel discrimination.  This approach, called turbo similarity searching, is notably more effective than conventional similarity searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9JTJUYOKnZrVg90H21EOLACvtfcHk0lj5V9UE5aTLng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvV2h&md5=cc24b6dff94f70a6e5ece86f8b14b673</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci050348j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050348j%26sid%3Dliteratum%253Aachs%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DWilton%26aufirst%3DD.%2BJ.%26aulast%3DAcklin%26aufirst%3DP.%26aulast%3DAzzaoui%26aufirst%3DK.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26atitle%3DNew%2520methods%2520for%2520ligand-based%2520virtual%2520screening%253A%2520use%2520of%2520data%2520fusion%2520and%2520machine%2520learning%2520to%2520enhance%2520the%2520effectiveness%2520of%2520similarity%2520searching%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2006%26volume%3D46%26spage%3D462%26epage%3D470%26doi%3D10.1021%2Fci050348j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muegge, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, P.</span></span> <span> </span><span class="NLM_article-title">An overview of molecular fingerprint similarity search in virtual screening</span>. <i>Expert. Opin. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">148</span>, <span class="refDoi"> DOI: 10.1517/17460441.2016.1117070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1517%2F17460441.2016.1117070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=26558489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=137-148&author=I.+Mueggeauthor=P.+Mukherjee&title=An+overview+of+molecular+fingerprint+similarity+search+in+virtual+screening&doi=10.1517%2F17460441.2016.1117070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">An overview of molecular fingerprint similarity search in virtual screening</span></div><div class="casAuthors">Muegge, Ingo; Mukherjee, Prasenjit</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">137-148</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A central premise of medicinal chem. is that structurally similar mols. exhibit similar biol. activities.  Mol. fingerprints encode properties of small mols. and assess their similarities computationally through bit string comparisons.  Based on the similarity to a biol. active template, mol. fingerprint methods allow for identifying addnl. compds. with a higher chance of displaying similar biol. activities against the same target - a process commonly referred to as virtual screening (VS).  This article focuses on fingerprint similarity searches in the context of compd. selection for enhancing hit sets, comparing compd. decks, and VS.  In addn., the authors discuss the application of fingerprints in predictive modeling.  Fingerprint similarity search methods are esp. useful in VS if only a few unrelated ligands are known for a given target and therefore more complex and information rich methods such as pharmacophore searches or structure-based design are not applicable.  In addn., fingerprint methods are used in characterizing properties of compd. collections such as chem. diversity, d. in chem. space, and content of biol. active mols. (biodiversity).  Such assessments are important for deciding what compds. to exptl. screen, to purchase, or to assemble in a virtual compd. deck for in silico screening or de novo design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCfZIE_oEtR7Vg90H21EOLACvtfcHk0lj9kPLGaqQvEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtLbF&md5=de18b3dbbf5ad1974ca1150193ee77aa</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1517%2F17460441.2016.1117070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2016.1117070%26sid%3Dliteratum%253Aachs%26aulast%3DMuegge%26aufirst%3DI.%26aulast%3DMukherjee%26aufirst%3DP.%26atitle%3DAn%2520overview%2520of%2520molecular%2520fingerprint%2520similarity%2520search%2520in%2520virtual%2520screening%26jtitle%3DExpert.%2520Opin.%2520Drug%2520Discov.%26date%3D2016%26volume%3D11%26spage%3D137%26epage%3D148%26doi%3D10.1517%2F17460441.2016.1117070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Molecular similarity in medicinal chemistry</span>. <i>J. Med. Pharma. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">3186</span>– <span class="NLM_lpage">3204</span>, <span class="refDoi"> DOI: 10.1021/jm401411z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401411z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=3186-3204&author=G.+Maggioraauthor=M.+Vogtauthor=D.+Stumpfeauthor=J.+Bajorath&title=Molecular+similarity+in+medicinal+chemistry&doi=10.1021%2Fjm401411z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm401411z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401411z%26sid%3Dliteratum%253Aachs%26aulast%3DMaggiora%26aufirst%3DG.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DMolecular%2520similarity%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Med.%2520Pharma.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D3186%26epage%3D3204%26doi%3D10.1021%2Fjm401411z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tetko, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filimonov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poroikov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varnek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roitberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isayev, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtarolo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourches, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspuru-Guzik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrafiotis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherkasov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">QSAR without borders</span>. <i>RIC Rev.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">3525</span>– <span class="NLM_lpage">3564</span>, <span class="refDoi"> DOI: 10.1039/d0cs00098a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1039%2Fd0cs00098a" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2020&pages=3525-3564&author=E.+N.+Muratovauthor=J.+Bajorathauthor=R.+P.+Sheridanauthor=I.+V.+Tetkoauthor=D.+Filimonovauthor=V.+Poroikovauthor=T.+I.+Opreaauthor=I.+I.+Baskinauthor=A.+Varnekauthor=A.+Roitbergauthor=O.+Isayevauthor=S.+Curtaroloauthor=D.+Fourchesauthor=Y.+Cohenauthor=A.+Aspuru-Guzikauthor=D.+A.+Winklerauthor=D.+Agrafiotisauthor=A.+Cherkasovauthor=A.+Tropsha&title=QSAR+without+borders&doi=10.1039%2Fd0cs00098a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2Fd0cs00098a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fd0cs00098a%26sid%3Dliteratum%253Aachs%26aulast%3DMuratov%26aufirst%3DE.%2BN.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26aulast%3DFilimonov%26aufirst%3DD.%26aulast%3DPoroikov%26aufirst%3DV.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DBaskin%26aufirst%3DI.%2BI.%26aulast%3DVarnek%26aufirst%3DA.%26aulast%3DRoitberg%26aufirst%3DA.%26aulast%3DIsayev%26aufirst%3DO.%26aulast%3DCurtarolo%26aufirst%3DS.%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DCohen%26aufirst%3DY.%26aulast%3DAspuru-Guzik%26aufirst%3DA.%26aulast%3DWinkler%26aufirst%3DD.%2BA.%26aulast%3DAgrafiotis%26aufirst%3DD.%26aulast%3DCherkasov%26aufirst%3DA.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DQSAR%2520without%2520borders%26jtitle%3DRIC%2520Rev.%26date%3D2020%26volume%3D49%26spage%3D3525%26epage%3D3564%26doi%3D10.1039%2Fd0cs00098a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camargo, L. M.</span></span> <span> </span><span class="NLM_article-title">The opportunities of mining historical and collective data in drug discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">434</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1016%2Fj.drudis.2014.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=25463034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A280%3ADC%252BC2MzjsF2mug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=422-434&author=A.+M.+Wassermannauthor=E.+Lounkineauthor=J.+W.+Daviesauthor=M.+Glickauthor=L.+M.+Camargo&title=The+opportunities+of+mining+historical+and+collective+data+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The opportunities of mining historical and collective data in drug discovery</span></div><div class="casAuthors">Wassermann Anne Mai; Camargo L Miguel; Lounkine Eugen; Davies John W; Glick Meir</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">422-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Vast amounts of bioactivity data have been generated for small molecules across public and corporate domains.  Biological signatures, either derived from systematic profiling efforts or from existing historical assay data, have been successfully employed for small molecule mechanism-of-action elucidation, drug repositioning, hit expansion and screening subset design.  This article reviews different types of biological descriptors and applications, and we demonstrate how biological data can outlive the original purpose or project for which it was generated.  By comparing 150 HTS campaigns run at Novartis over the past decade on the basis of their active and inactive chemical matter, we highlight the opportunities and challenges associated with cross-project learning in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR2wZT2paLsIg20zIbnCCI3fW6udTcc2eZs1y4-Whl7MLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzjsF2mug%253D%253D&md5=44e61022e1b87335e0f43101e086cb4b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DCamargo%26aufirst%3DL.%2BM.%26atitle%3DThe%2520opportunities%2520of%2520mining%2520historical%2520and%2520collective%2520data%2520in%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D422%26epage%3D434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fliri, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loging, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thadeio, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmann, R. A.</span></span> <span> </span><span class="NLM_article-title">Biological spectra analysis: Linking biological activity profiles to molecular structure</span>. <i>Proc. Nat. Acad ScI. U S A</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1073/pnas.0407790101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1073%2Fpnas.0407790101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=15625110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD2MXptFSmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=261-266&author=A.+F.+Fliriauthor=W.+T.+Logingauthor=P.+F.+Thadeioauthor=R.+A.+Volkmann&title=Biological+spectra+analysis%3A+Linking+biological+activity+profiles+to+molecular+structure&doi=10.1073%2Fpnas.0407790101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Biological spectra analysis: Linking biological activity profiles to molecular structure</span></div><div class="casAuthors">Fliri, Anton F.; Loging, William T.; Thadeio, Peter F.; Volkmann, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-266</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Establishing quant. relationships between mol. structure and broad biol. effects has been a longstanding challenge in science.  Currently, no method exists for forecasting broad biol. activity profiles of medicinal agents even within narrow boundaries of structurally similar mols.  Starting from the premise that biol. activity results from the capacity of small org. mols. to modulate the activity of the proteome, we set out to investigate whether descriptor sets could be developed for measuring and quantifying this mol. property.  Using a 1,567-compd. database, we show that percent inhibition values, detd. at single high drug concn. in a battery of in vitro assays representing a cross section of the proteome, provide precise mol. property descriptors that identify the structure of mols.  When broad biol. activity of mols. is represented in spectra form, org. mols. can be sorted by quantifying differences between biol. spectra.  Unlike traditional structure-activity relationship methods, sorting of mols. by using biospectra comparisons does not require knowledge of a mol.'s putative drug targets.  To illustrate this finding, we selected as starting point the biol. activity spectra of clotrimazole and tioconazole because their putative target, lanosterol demethylase (CYP51), was not included in the bioassay array.  Spectra similarity obtained through profile similarity measurements and hierarchical clustering provided an unbiased means for establishing quant. relationships between chem. structures and biol. activity spectra.  This methodol., which we have termed biol. spectra anal., provides the capability not only of sorting mols. on the basis of biospectra similarity but also of predicting simultaneous interactions of new mols. with multiple proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocRkPiQrArMLVg90H21EOLACvtfcHk0lidqdrqoZmXow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXptFSmsQ%253D%253D&md5=ff5faffa569df4c61389d6baef5800c3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0407790101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0407790101%26sid%3Dliteratum%253Aachs%26aulast%3DFliri%26aufirst%3DA.%2BF.%26aulast%3DLoging%26aufirst%3DW.%2BT.%26aulast%3DThadeio%26aufirst%3DP.%2BF.%26aulast%3DVolkmann%26aufirst%3DR.%2BA.%26atitle%3DBiological%2520spectra%2520analysis%253A%2520Linking%2520biological%2520activity%2520profiles%2520to%2520molecular%2520structure%26jtitle%3DProc.%2520Nat.%2520Acad%2520ScI.%2520U%2520S%2520A%26date%3D2005%26volume%3D102%26spage%3D261%26epage%3D266%26doi%3D10.1073%2Fpnas.0407790101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fliri, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loging, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thadeio, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmann, R. A.</span></span> <span> </span><span class="NLM_article-title">Biospectra analysis: model proteome characterizations for linking molecular structure and biological response</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">6918</span>– <span class="NLM_lpage">6925</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050494g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVGisrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6918-6925&author=A.+F.+Fliriauthor=W.+T.+Logingauthor=P.+F.+Thadeioauthor=R.+A.+Volkmann&title=Biospectra+analysis%3A+model+proteome+characterizations+for+linking+molecular+structure+and+biological+response"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Biospectra Analysis: Model Proteome Characterizations for Linking Molecular Structure and Biological Response</span></div><div class="casAuthors">Fliri, Anton F.; Loging, William T.; Thadeio, Peter F.; Volkmann, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6918-6925</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Establishing quant. relationships between mol. structure and broad biol. effects has been a long-standing goal in drug discovery.  Evaluation of the capacity of mols. to modulate protein functions is a prerequisite for understanding the relationship between mol. structure and in vivo biol. response.  A particular challenge in these investigations is to derive quant. measurements of a mol.'s functional activity pattern across different proteins.  Herein we describe an operationally simple probabilistic structure-activity relationship (SAR) approach, termed biospectra anal., for identifying agonist and antagonist effect profiles of medicinal agents by using pattern similarity between biol. activity spectra (biospectra) of mols. as the determinant.  Accordingly, in vitro binding data (percent inhibition values of mols. detd. at single high drug concn. in a battery of assays representing a cross section of the proteome) are useful for identifying functional effect profile similarity between medicinal agents.  To illustrate this finding, the relationship between biospectra similarity of 24 mols., identified by hierarchical clustering of a 1567 mol. dataset as being most closely aligned with the neurotransmitter dopamine, and their agonist or antagonist properties was probed.  Distinguishing the results described in this study from those obtained with affinity-based methods, the obsd. assocn. between biospectra and biol. response profile similarity remains intact even upon removal of putative drug targets from the dataset (four dopaminergic [D1/D2/D3/D4] and two adrenergic [α1 and α2] receptors).  These findings indicate that biospectra anal. provides an unbiased new tool for forecasting structure-response relationships and for translating broad biol. effect information into chem. structure design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJa5_zvZqHGLVg90H21EOLACvtfcHk0lidqdrqoZmXow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVGisrjE&md5=e899a8db1f2d66dcb7b7c5d656816111</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm050494g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050494g%26sid%3Dliteratum%253Aachs%26aulast%3DFliri%26aufirst%3DA.%2BF.%26aulast%3DLoging%26aufirst%3DW.%2BT.%26aulast%3DThadeio%26aufirst%3DP.%2BF.%26aulast%3DVolkmann%26aufirst%3DR.%2BA.%26atitle%3DBiospectra%2520analysis%253A%2520model%2520proteome%2520characterizations%2520for%2520linking%2520molecular%2520structure%2520and%2520biological%2520response%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6918%26epage%3D6925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campillos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bork, P.</span></span> <span> </span><span class="NLM_article-title">Drug target identification using side-effect similarity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1126/science.1158140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1126%2Fscience.1158140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=18621671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD1cXot1entbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2008&pages=263-266&author=M.+Campillosauthor=M.+Kuhnauthor=A.+C.+Gavinauthor=L.+J.+Jensenauthor=P.+Bork&title=Drug+target+identification+using+side-effect+similarity&doi=10.1126%2Fscience.1158140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Target Identification Using Side-Effect Similarity</span></div><div class="casAuthors">Campillos, Monica; Kuhn, Michael; Gavin, Anne-Claude; Jensen, Lars Juhl; Bork, Peer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">5886</span>),
    <span class="NLM_cas:pages">263-266</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Targets for drugs have so far been predicted on the basis of mol. or cellular features, for example, by exploiting similarity in chem. structure or in activity across cell lines.  We used phenotypic side-effect similarities to infer whether two drugs share a target.  Applied to 746 marketed drugs, a network of 1018 side effect-driven drug-drug relations became apparent, 261 of which are formed by chem. dissimilar drugs from different therapeutic indications.  We exptl. tested 20 of these unexpected drug-drug relations and validated 13 implied drug-target relations by in vitro binding assays, of which 11 reveal inhibition consts. equal to less than 10 micromolar.  Nine of these were tested and confirmed in cell assays, documenting the feasibility of using phenotypic information to infer mol. interactions and hinting at new uses of marketed drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbiFcs_camL7Vg90H21EOLACvtfcHk0lhPMEy-oqNdWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXot1entbY%253D&md5=d8fe131adeb39750cb97ee4e147de355</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fscience.1158140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1158140%26sid%3Dliteratum%253Aachs%26aulast%3DCampillos%26aufirst%3DM.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DGavin%26aufirst%3DA.%2BC.%26aulast%3DJensen%26aufirst%3DL.%2BJ.%26aulast%3DBork%26aufirst%3DP.%26atitle%3DDrug%2520target%2520identification%2520using%2520side-effect%2520similarity%26jtitle%3DScience%26date%3D2008%26volume%3D321%26spage%3D263%26epage%3D266%26doi%3D10.1126%2Fscience.1158140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blat, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrobel, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunet, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hieronymus, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>313</i></span>,  <span class="NLM_fpage">1929</span>– <span class="NLM_lpage">1935</span>, <span class="refDoi"> DOI: 10.1126/science.1132939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1126%2Fscience.1132939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=17008526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVSnsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&author=J.+Lambauthor=E.+D.+Crawfordauthor=D.+Peckauthor=J.+W.+Modellauthor=I.+C.+Blatauthor=M.+J.+Wrobelauthor=J.+Lernerauthor=J.+P.+Brunetauthor=A.+Subramanianauthor=K.+N.+Rossauthor=M.+Reichauthor=H.+Hieronymusauthor=G.+Weiauthor=S.+A.+Armstrongauthor=S.+J.+Haggartyauthor=P.+A.+Clemonsauthor=R.+Weiauthor=S.+A.+Carrauthor=E.+S.+Landerauthor=T.+R.+Golub&title=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease&doi=10.1126%2Fscience.1132939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The connectivity map: Using gene-expression signatures to connect small molecules, genes, and disease</span></div><div class="casAuthors">Lamb, Justin; Crawford, Emily D.; Peck, David; Modell, Joshua W.; Blat, Irene C.; Wrobel, Matthew J.; Lerner, Jim; Brunet, Jean-Philippe; Subramanian, Aravind; Ross, Kenneth N.; Reich, Michael; Hieronymus, Haley; Wei, Guo; Armstrong, Scott A.; Haggarty, Stephen J.; Clemons, Paul A.; Wei, Ru; Carr, Steven A.; Lander, Eric S.; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">313</span>
        (<span class="NLM_cas:issue">5795</span>),
    <span class="NLM_cas:pages">1929-1935</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">To pursue a systematic approach to the discovery of functional connections among diseases, genetic perturbation, and drug action, we have created the first installment of a ref. collection of gene-expression profiles from cultured human cells treated with bioactive small mols., together with pattern-matching software to mine these data.  We demonstrate that this "Connectivity Map" resource can be used to find connections among small mols. sharing a mechanism of action, chems. and physiol. processes, and diseases and drugs.  These results indicate the feasibility of the approach and suggest the value of a large-scale community Connectivity Map project.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtu0M2BUO2w7Vg90H21EOLACvtfcHk0lhPMEy-oqNdWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVSnsrbN&md5=1fad887ad515ffdd8064e526ff25a4db</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1126%2Fscience.1132939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1132939%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DCrawford%26aufirst%3DE.%2BD.%26aulast%3DPeck%26aufirst%3DD.%26aulast%3DModell%26aufirst%3DJ.%2BW.%26aulast%3DBlat%26aufirst%3DI.%2BC.%26aulast%3DWrobel%26aufirst%3DM.%2BJ.%26aulast%3DLerner%26aufirst%3DJ.%26aulast%3DBrunet%26aufirst%3DJ.%2BP.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DHieronymus%26aufirst%3DH.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DWei%26aufirst%3DR.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DThe%2520Connectivity%2520Map%253A%2520using%2520gene-expression%2520signatures%2520to%2520connect%2520small%2520molecules%252C%2520genes%252C%2520and%2520disease%26jtitle%3DScience%26date%3D2006%26volume%3D313%26spage%3D1929%26epage%3D1935%26doi%3D10.1126%2Fscience.1132939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wosikowski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuurhuis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paull, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, S. E.</span></span> <span> </span><span class="NLM_article-title">Identification of epidermal growth factor receptor and c-erbB2 pathway inhibitors by correlation with gene expression patterns</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">1505</span>– <span class="NLM_lpage">1515</span>, <span class="refDoi"> DOI: 10.1093/jnci/89.20.1505</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1093%2Fjnci%2F89.20.1505" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=9337347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADyaK2sXntVGgsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1997&pages=1505-1515&author=K.+Wosikowskiauthor=D.+Schuurhuisauthor=K.+Johnsonauthor=K.+D.+Paullauthor=T.+G.+Myersauthor=J.+N.+Weinsteinauthor=S.+E.+Bates&title=Identification+of+epidermal+growth+factor+receptor+and+c-erbB2+pathway+inhibitors+by+correlation+with+gene+expression+patterns&doi=10.1093%2Fjnci%2F89.20.1505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of epidermal growth factor receptor and c-erbB2 pathway inhibitors by correlation with gene expression patterns</span></div><div class="casAuthors">Wosikowski, Katja; Schuurhuis, Danita; Johnson, Kathryn; Paull, Kenneth D.; Myers, Timothy G.; Weinstein, John N.; Bates, Susan E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">1505-1515</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Growth factor receptor-signaling pathways are potentially important targets for anticancer therapy.  The interaction of anticancer agents with specific mol. targets can be identified by correlating target expression patterns with cytotoxicity patterns.  The authors sought to identify new agents that target and inhibit the activity of the epidermal growth factor (EGF) receptor and of c-erbB2 (also called HER2 or neu), by correlating EGF receptor, transforming growth factor (TGF)-α (a ligand for EGF receptor), and c-erbB2 mRNA expression levels with the results of cytotoxicity assays of the 49 000 compds. in the National Cancer Institute (NCI) drug screen database.  The levels of mRNAs were measured and used to generate a mol. target database for the 60 cell lines of the NCI anticancer drug screen.  The computer anal. program, COMPARE, was used to search for cytotoxicity patterns in the NCI drug screen database that were highly correlated with EGF receptor, TGF-α, or c-erbB2 mRNA expression patterns.  The putative EGF receptor-inhibiting compds. were tested for effects on basal tyrosine phosphorylation, in vitro EGF receptor tyrosine kinase activity, and EGF-dependent growth.  Putative ErbB2-inhibiting compds. were tested for effects on antibody-induced ErbB2 tyrosine kinase activity.  EGF receptor mRNA and TGF-α mRNA levels were highest in cell lines derived from renal cancers, and c-erbB2 mRNA levels were highest in cells derived from breast, ovarian, and colon cancers.  Twenty-five compds. with high correlation coeffs. (for cytotoxicity and levels of the measured mRNAs) were tested as inhibitors of the EGF receptor or c-erbB2 signaling pathways; 14 compds. were identified as inhibitors of these pathways.  The most potent compd., B4, inhibited autophosphorylation (which occurs following activation) of ErbB2 by 50% in whole cells at 7.7 μM.  Novel EGF receptor or c-erbB2 pathway inhibitors can be identified in the NCI drug screen by correlation of cytotoxicity patterns with EGF receptor or c-erbB2 mRNA expression levels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokhKMwhdDk8bVg90H21EOLACvtfcHk0lhCYlrAe9k6UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXntVGgsLk%253D&md5=2b98161f612d31a02522fc4c5bcbe561</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F89.20.1505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F89.20.1505%26sid%3Dliteratum%253Aachs%26aulast%3DWosikowski%26aufirst%3DK.%26aulast%3DSchuurhuis%26aufirst%3DD.%26aulast%3DJohnson%26aufirst%3DK.%26aulast%3DPaull%26aufirst%3DK.%2BD.%26aulast%3DMyers%26aufirst%3DT.%2BG.%26aulast%3DWeinstein%26aufirst%3DJ.%2BN.%26aulast%3DBates%26aufirst%3DS.%2BE.%26atitle%3DIdentification%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520and%2520c-erbB2%2520pathway%2520inhibitors%2520by%2520correlation%2520with%2520gene%2520expression%2520patterns%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D1997%26volume%3D89%26spage%3D1505%26epage%3D1515%26doi%3D10.1093%2Fjnci%2F89.20.1505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hassane, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbett, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abboud, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liesveld, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, C. T.</span></span> <span> </span><span class="NLM_article-title">Discovery of agents that eradicate leukemia stem cells using an in silico screen of public gene expression data</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">5654</span>– <span class="NLM_lpage">5662</span>, <span class="refDoi"> DOI: 10.1182/blood-2007-11-126003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1182%2Fblood-2007-11-126003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=18305216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsVOktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2008&pages=5654-5662&author=D.+C.+Hassaneauthor=M.+L.+Guzmanauthor=C.+Corbettauthor=X.+Liauthor=R.+Abboudauthor=F.+Youngauthor=J.+L.+Liesveldauthor=M.+Carrollauthor=C.+T.+Jordan&title=Discovery+of+agents+that+eradicate+leukemia+stem+cells+using+an+in+silico+screen+of+public+gene+expression+data&doi=10.1182%2Fblood-2007-11-126003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of agents that eradicate leukemia stem cells using an in silico screen of public gene expression data</span></div><div class="casAuthors">Hassane, Duane C.; Guzman, Monica L.; Corbett, Cheryl; Li, Xiaojie; Abboud, Ramzi; Young, Fay; Liesveld, Jane L.; Carroll, Martin; Jordan, Craig T.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5654-5662</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Increasing evidence indicates that malignant stem cells are important for the pathogenesis of acute myelogenous leukemia (AML) and represent a reservoir of cells that drive the development of AML and relapse.  Therefore, new treatment regimens are necessary to prevent relapse and improve therapeutic outcomes.  Previous studies have shown that the sesquiterpene lactone, parthenolide (PTL), ablates bulk, progenitor, and stem AML cells while causing no appreciable toxicity to normal hematopoietic cells.  Thus, PTL must evoke cellular responses capable of mediating AML selective cell death.  Given recent advances in chem. genomics such as gene expression-based high-throughput screening (GE-HTS) and the Connectivity Map, we hypothesized that the gene expression signature resulting from treatment of primary AML with PTL could be used to search for similar signatures in publicly available gene expression profiles deposited into the Gene Expression Omnibus (GEO).  We therefore devised a broad in silico screen of the GEO database using the PTL gene expression signature as a template and discovered 2 new agents, celastrol and 4-hydroxy-2-nonenal, that effectively eradicate AML at the bulk, progenitor, and stem cell level.  These findings suggest the use of multicenter collections of high-throughput data to facilitate discovery of leukemia drugs and drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi4ise6u_curVg90H21EOLACvtfcHk0lhCYlrAe9k6UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsVOktb8%253D&md5=ccd65463c8a0aba9b5f204970f5c267d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1182%2Fblood-2007-11-126003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2007-11-126003%26sid%3Dliteratum%253Aachs%26aulast%3DHassane%26aufirst%3DD.%2BC.%26aulast%3DGuzman%26aufirst%3DM.%2BL.%26aulast%3DCorbett%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DAbboud%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DF.%26aulast%3DLiesveld%26aufirst%3DJ.%2BL.%26aulast%3DCarroll%26aufirst%3DM.%26aulast%3DJordan%26aufirst%3DC.%2BT.%26atitle%3DDiscovery%2520of%2520agents%2520that%2520eradicate%2520leukemia%2520stem%2520cells%2520using%2520an%2520in%2520silico%2520screen%2520of%2520public%2520gene%2520expression%2520data%26jtitle%3DBlood%26date%3D2008%26volume%3D111%26spage%3D5654%26epage%3D5662%26doi%3D10.1182%2Fblood-2007-11-126003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span>; <span class="NLM_string-name">Xu, M.</span>; <span class="NLM_string-name">Zhu, H.</span>; <span class="NLM_string-name">Chen, C. Z.</span>; <span class="NLM_string-name">Zhu, W.</span>; <span class="NLM_string-name">Lee, E. M.</span>; <span class="NLM_string-name">He, S.</span>; <span class="NLM_string-name">Zhang, L.</span>; <span class="NLM_string-name">Zhao, J.</span>; <span class="NLM_string-name">Shamim, K.</span>; <span class="NLM_string-name">Bougie, D.</span>; <span class="NLM_string-name">Huang, W.</span>; <span class="NLM_string-name">Xia, M.</span>; <span class="NLM_string-name">Hall, M. D.</span>; <span class="NLM_string-name">Lo, D.</span>; <span class="NLM_string-name">Simeonov, A.</span>; <span class="NLM_string-name">Austin, C. P.</span>; <span class="NLM_string-name">Qiu, X.</span>; <span class="NLM_string-name">Tang, H.</span>; <span class="NLM_string-name">Zheng, W.</span></span> <span> </span><span class="NLM_article-title">Biological activity-based modeling identifies antiviral leads against SARS-CoV-2</span>.  <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1038%2Fs41587-021-00839-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&author=R.+Huang&author=M.+Xu&author=H.+Zhu&author=C.+Z.+Chen&author=W.+Zhu&author=E.+M.+Lee&author=S.+He&author=L.+Zhang&author=J.+Zhao&author=K.+Shamim&author=D.+Bougie&author=W.+Huang&author=M.+Xia&author=M.+D.+Hall&author=D.+Lo&author=A.+Simeonov&author=C.+P.+Austin&author=X.+Qiu&author=H.+Tang&author=W.+Zheng&title=Biological+activity-based+modeling+identifies+antiviral+leads+against+SARS-CoV-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fs41587-021-00839-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41587-021-00839-1%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DBiological%2520activity-based%2520modeling%2520identifies%2520antiviral%2520leads%2520against%2520SARS-CoV-2%26jtitle%3DNat.%2520Biotechnol.%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sipes, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kothiya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reif, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Judson, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houck, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dix, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavlock, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, T. B.</span></span> <span> </span><span class="NLM_article-title">Profiling 976 ToxCast chemicals across 331 enzymatic and receptor signaling assays</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">878</span>– <span class="NLM_lpage">895</span>, <span class="refDoi"> DOI: 10.1021/tx400021f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx400021f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtlWhu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=878-895&author=N.+S.+Sipesauthor=M.+T.+Martinauthor=P.+Kothiyaauthor=D.+M.+Reifauthor=R.+S.+Judsonauthor=A.+M.+Richardauthor=K.+A.+Houckauthor=D.+J.+Dixauthor=R.+J.+Kavlockauthor=T.+B.+Knudsen&title=Profiling+976+ToxCast+chemicals+across+331+enzymatic+and+receptor+signaling+assays&doi=10.1021%2Ftx400021f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling 976 ToxCast Chemicals across 331 Enzymatic and Receptor Signaling Assays</span></div><div class="casAuthors">Sipes, Nisha S.; Martin, Matthew T.; Kothiya, Parth; Reif, David M.; Judson, Richard S.; Richard, Ann M.; Houck, Keith A.; Dix, David J.; Kavlock, Robert J.; Knudsen, Thomas B.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">878-895</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding potential health risks is a significant challenge due to the large nos. of diverse chems. with poorly characterized exposures and mechanisms of toxicities.  The present study analyzes 976 chems. (including failed pharmaceuticals, alternative plasticizers, food additives, and pesticides) in Phases I and II of the U.S. EPA's ToxCast project across 331 cell-free enzymic and ligand-binding high-throughput screening (HTS) assays.  Half-maximal activity concns. (AC50) were identified for 729 chems. in 256 assays (7135 chem.-assay pairs).  Some of the most commonly affected assays were CYPs (CYP2C9 and CYP2C19), transporters (mitochondrial TSPO, norepinephrine, and dopaminergic), and GPCRs (aminergic).  Heavy metals, surfactants, and dithiocarbamate fungicides showed promiscuous but distinctly different patterns of activity, whereas many of the pharmaceutical compds. showed promiscuous activity across GPCRs.  Literature anal. confirmed >50% of the activities for the most potent chem.-assay pairs (54) but also revealed 10 missed interactions.  Twenty-two chems. with known estrogenic activity were correctly identified for the majority (77%), missing only the weaker interactions.  In many cases, novel findings for previously unreported chem.-target combinations clustered with known chem.-target interactions.  Results from this large inventory of chem.-biol. interactions can inform read-across methods as well as link potential targets to mol. initiating events in adverse outcome pathways for diverse toxicities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1kBIOo_1MCbVg90H21EOLACvtfcHk0lglnXAWw-r9hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtlWhu7s%253D&md5=473da27aad42b986ef3dc0dde8acf14c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Ftx400021f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx400021f%26sid%3Dliteratum%253Aachs%26aulast%3DSipes%26aufirst%3DN.%2BS.%26aulast%3DMartin%26aufirst%3DM.%2BT.%26aulast%3DKothiya%26aufirst%3DP.%26aulast%3DReif%26aufirst%3DD.%2BM.%26aulast%3DJudson%26aufirst%3DR.%2BS.%26aulast%3DRichard%26aufirst%3DA.%2BM.%26aulast%3DHouck%26aufirst%3DK.%2BA.%26aulast%3DDix%26aufirst%3DD.%2BJ.%26aulast%3DKavlock%26aufirst%3DR.%2BJ.%26aulast%3DKnudsen%26aufirst%3DT.%2BB.%26atitle%3DProfiling%2520976%2520ToxCast%2520chemicals%2520across%2520331%2520enzymatic%2520and%2520receptor%2520signaling%2520assays%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2013%26volume%3D26%26spage%3D878%26epage%3D895%26doi%3D10.1021%2Ftx400021f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span>; <span class="NLM_string-name">Wang, G.</span></span> <span> </span><span class="NLM_article-title">Machine Learning Based Toxicity Prediction: From Chemical Structural Description to Transcriptome Analysis</span>.  <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume">19</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.3390%2Fijms19082358" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&author=Y.+Wu&author=G.+Wang&title=Machine+Learning+Based+Toxicity+Prediction%3A+From+Chemical+Structural+Description+to+Transcriptome+Analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.3390%2Fijms19082358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms19082358%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26atitle%3DMachine%2520Learning%2520Based%2520Toxicity%2520Prediction%253A%2520From%2520Chemical%2520Structural%2520Description%2520to%2520Transcriptome%2520Analysis%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2018%26volume%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muthas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, S.</span></span> <span> </span><span class="NLM_article-title">Exploiting Pharmacological Similarity to Identify Safety Concerns - Listen to What the Data Tells You</span>. <i>Mol. Inform.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1002/minf.201200088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1002%2Fminf.201200088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=27481022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFKruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=37-45&author=D.+Muthasauthor=S.+Boyer&title=Exploiting+Pharmacological+Similarity+to+Identify+Safety+Concerns+-+Listen+to+What+the+Data+Tells+You&doi=10.1002%2Fminf.201200088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting Pharmacological Similarity to Identify Safety Concerns - Listen to What the Data Tells You</span></div><div class="casAuthors">Muthas, Daniel; Boyer, Scott</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-45</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">While most new drugs are designed to act on a single target or a small no. of targets, many do show broad pharmacol. activity.  In some cases this can be beneficial and necessary for efficacy and in others it can be detrimental, leading to increased safety liability.  To probe off-target pharmacol. most drug discovery programs include screening against a broad panel of targets that represent known troublesome pharmacol.  Hits against any one of these targets can then be subjected to a risk assessment for potential safety problems in preclin. or clin. studies.  In addn., the secondary pharmacol. profile can also be thought of as an alternative description of the compd. and as such can be used as a method for assessing similarity'.  Consequently, inspection of the in vivo findings of pharmacol. neighbors can give important insights into potential safety liabilities that are neither identified by pure chem. similarity searches nor by risk assessment on individual targets.  Here we show that the pharmacol. profile contains addnl. information as compared to chem. similarity, and also demonstrate how this can be used in the hazard assessment done during drug discovery and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofUE1Sy23AN7Vg90H21EOLACvtfcHk0lglnXAWw-r9hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFKruw%253D%253D&md5=af95ecc021c2227be4f043bd467d6008</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fminf.201200088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201200088%26sid%3Dliteratum%253Aachs%26aulast%3DMuthas%26aufirst%3DD.%26aulast%3DBoyer%26aufirst%3DS.%26atitle%3DExploiting%2520Pharmacological%2520Similarity%2520to%2520Identify%2520Safety%2520Concerns%2520-%2520Listen%2520to%2520What%2520the%2520Data%2520Tells%2520You%26jtitle%3DMol.%2520Inform.%26date%3D2013%26volume%3D32%26spage%3D37%26epage%3D45%26doi%3D10.1002%2Fminf.201200088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrone, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simms, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigsch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Rethinking molecular similarity: comparing compounds on the basis of biological activity</span>. <i>Am. Chem. Soc. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1399</span>– <span class="NLM_lpage">1409</span>, <span class="refDoi"> DOI: 10.1021/cb3001028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3001028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1WrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1399-1409&author=P.+M.+Petroneauthor=B.+Simmsauthor=F.+Nigschauthor=E.+Lounkineauthor=P.+Kutchukianauthor=A.+Cornettauthor=Z.+Dengauthor=J.+W.+Daviesauthor=J.+L.+Jenkinsauthor=M.+Glick&title=Rethinking+molecular+similarity%3A+comparing+compounds+on+the+basis+of+biological+activity&doi=10.1021%2Fcb3001028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Rethinking Molecular Similarity: Comparing Compounds on the Basis of Biological Activity</span></div><div class="casAuthors">Petrone, Paula M.; Simms, Benjamin; Nigsch, Florian; Lounkine, Eugen; Kutchukian, Peter; Cornett, Allen; Deng, Zhan; Davies, John W.; Jenkins, Jeremy L.; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1399-1409</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Since the advent of high-throughput screening (HTS), there has been an urgent need for methods that facilitate the interrogation of large-scale chem. biol. data to build a mode of action (MoA) hypothesis.  This can be done either prior to the HTS by subset design of compds. with known MoA or post HTS by data annotation and mining.  To enable this process, we developed a tool that compares compds. solely on the basis of their bioactivity: the chem. biol. descriptor high-throughput screening fingerprint (HTS-FP).  In the current embodiment, data are aggregated from 195 biochem. and cell-based assays developed at Novartis and can be used to identify bioactivity relationships among the inhouse collection comprising ∼1.5 million compds.  We demonstrate the value of the HTS-FP for virtual screening and in particular scaffold hopping.  HTS-FP outperforms state of the art methods in several aspects, retrieving bioactive compds. with remarkable chem. dissimilarity to a probe structure.  We also apply HTS-FP for the design of screening subsets in HTS.  Using retrospective data, we show that a biodiverse selection of plates performs significantly better than a chem. diverse selection of plates, both in terms of no. of hits and diversity of chemotypes retrieved.  This is also true in the case of hit expansion predictions using HTS-FP similarity.  Sets of compds. clustered with HTS-FP are biol. meaningful, in the sense that these clusters enrich for genes and gene ontol. (GO) terms, showing that compds. that are bioactively similar also tend to target proteins that operate together in the cell.  HTS-FP are valuable not only because of their predictive power but mainly because they relate compds. solely on the basis of bioactivity, harnessing the accumulated knowledge of a high-throughput screening facility toward the understanding of how compds. interact with the proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFjchAxW_TSLVg90H21EOLACvtfcHk0liVPsRKJ1Lf7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1WrtLY%253D&md5=66ecd494f9200a0fdc1c142402ff255f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fcb3001028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3001028%26sid%3Dliteratum%253Aachs%26aulast%3DPetrone%26aufirst%3DP.%2BM.%26aulast%3DSimms%26aufirst%3DB.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKutchukian%26aufirst%3DP.%26aulast%3DCornett%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DRethinking%2520molecular%2520similarity%253A%2520comparing%2520compounds%2520on%2520the%2520basis%2520of%2520biological%2520activity%26jtitle%3DAm.%2520Chem.%2520Soc.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1399%26epage%3D1409%26doi%3D10.1021%2Fcb3001028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sturm, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandriessche, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klambauer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engkvist, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">Application of Bioactivity Profile-Based Fingerprints for Building Machine Learning Models</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">962</span>– <span class="NLM_lpage">972</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00550</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00550" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=962-972&author=N.+Sturmauthor=J.+Sunauthor=Y.+Vandriesscheauthor=A.+Mayrauthor=G.+Klambauerauthor=L.+Carlssonauthor=O.+Engkvistauthor=H.+Chen&title=Application+of+Bioactivity+Profile-Based+Fingerprints+for+Building+Machine+Learning+Models&doi=10.1021%2Facs.jcim.8b00550"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00550%26sid%3Dliteratum%253Aachs%26aulast%3DSturm%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DVandriessche%26aufirst%3DY.%26aulast%3DMayr%26aufirst%3DA.%26aulast%3DKlambauer%26aufirst%3DG.%26aulast%3DCarlsson%26aufirst%3DL.%26aulast%3DEngkvist%26aufirst%3DO.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DApplication%2520of%2520Bioactivity%2520Profile-Based%2520Fingerprints%2520for%2520Building%2520Machine%2520Learning%2520Models%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2019%26volume%3D59%26spage%3D962%26epage%3D972%26doi%3D10.1021%2Facs.jcim.8b00550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laufkotter, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturm, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engkvist, O.</span></span> <span> </span><span class="NLM_article-title">Combining structural and bioactivity-based fingerprints improves prediction performance and scaffold hopping capability</span>. <i>Aust. J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">54</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=54&author=O.+Laufkotterauthor=N.+Sturmauthor=J.+Bajorathauthor=H.+Chenauthor=O.+Engkvist&title=Combining+structural+and+bioactivity-based+fingerprints+improves+prediction+performance+and+scaffold+hopping+capability"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLaufkotter%26aufirst%3DO.%26aulast%3DSturm%26aufirst%3DN.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DEngkvist%26aufirst%3DO.%26atitle%3DCombining%2520structural%2520and%2520bioactivity-based%2520fingerprints%2520improves%2520prediction%2520performance%2520and%2520scaffold%2520hopping%2520capability%26jtitle%3DAust.%2520J.%2520Chem.%26date%3D2019%26volume%3D11%26spage%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helal, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciejewski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregori-Puigjane, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span> <span> </span><span class="NLM_article-title">Public Domain HTS Fingerprints: Design and Evaluation of Compound Bioactivity Profiles from PubChem’s Bioassay Repository</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">390</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00498</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00498" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=390-398&author=K.+Y.+Helalauthor=M.+Maciejewskiauthor=E.+Gregori-Puigjaneauthor=M.+Glickauthor=A.+M.+Wassermann&title=Public+Domain+HTS+Fingerprints%3A+Design+and+Evaluation+of+Compound+Bioactivity+Profiles+from+PubChem%E2%80%99s+Bioassay+Repository&doi=10.1021%2Facs.jcim.5b00498"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00498%26sid%3Dliteratum%253Aachs%26aulast%3DHelal%26aufirst%3DK.%2BY.%26aulast%3DMaciejewski%26aufirst%3DM.%26aulast%3DGregori-Puigjane%26aufirst%3DE.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26atitle%3DPublic%2520Domain%2520HTS%2520Fingerprints%253A%2520Design%2520and%2520Evaluation%2520of%2520Compound%2520Bioactivity%2520Profiles%2520from%2520PubChem%25E2%2580%2599s%2520Bioassay%2520Repository%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2016%26volume%3D56%26spage%3D390%26epage%3D398%26doi%3D10.1021%2Facs.jcim.5b00498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cortes
Cabrera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrone, P. M.</span></span> <span> </span><span class="NLM_article-title">Optimal HTS Fingerprint Definitions by Using a Desirability Function and a Genetic Algorithm</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">641</span>– <span class="NLM_lpage">646</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00447</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00447" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=641-646&author=A.+Cortes%0ACabreraauthor=P.+M.+Petrone&title=Optimal+HTS+Fingerprint+Definitions+by+Using+a+Desirability+Function+and+a+Genetic+Algorithm&doi=10.1021%2Facs.jcim.7b00447"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00447%26sid%3Dliteratum%253Aachs%26aulast%3DCortes%2BCabrera%26aufirst%3DA.%26aulast%3DPetrone%26aufirst%3DP.%2BM.%26atitle%3DOptimal%2520HTS%2520Fingerprint%2520Definitions%2520by%2520Using%2520a%2520Desirability%2520Function%2520and%2520a%2520Genetic%2520Algorithm%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2018%26volume%3D58%26spage%3D641%26epage%3D646%26doi%3D10.1021%2Facs.jcim.7b00447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gardiner, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holliday, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Dowd, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span> <span> </span><span class="NLM_article-title">Effectiveness of 2D fingerprints for scaffold hopping</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">414</span>, <span class="refDoi"> DOI: 10.4155/fmc.11.4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.4155%2Ffmc.11.4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=21452977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVClu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=405-414&author=E.+J.+Gardinerauthor=J.+D.+Hollidayauthor=C.+O%E2%80%99Dowdauthor=P.+Willett&title=Effectiveness+of+2D+fingerprints+for+scaffold+hopping&doi=10.4155%2Ffmc.11.4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Effectiveness of 2D fingerprints for scaffold hopping</span></div><div class="casAuthors">Gardiner, Eleanor J.; Holliday, John D.; O'Dowd, Caroline; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">405-414</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Background: It has been suggested that similarity searching using 2D fingerprints may not be suitable for scaffold hopping.  Methods: This article reports a detailed evaluation of the effectiveness of six common types of 2D fingerprints when they are used for scaffold-hopping similarity searches of the Mol. Design Limited Drug Data Report database, World of Mol. Bioactivity database and Maximum Unbiased Validation database.  Results: The results demonstrate that 2D fingerprints can be used for scaffold hopping, with novel scaffolds being identified in nearly every search that was carried out.  The degree of enrichment depends on the structural diversity of the actives that are being sought, with the greatest enrichments often being obtained using the extended connectivity fingerprint encoding a circular substructure of diam. four bonds (ECFP4) fingerprint.  Conclusion: 2D fingerprints provide a simple and computationally efficient way of identifying novel chemotypes in lead-discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhWHRgpKIC5LVg90H21EOLACvtfcHk0lhIzKNza63fbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVClu7w%253D&md5=502004660048eb4960f671e51c135264</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.4155%2Ffmc.11.4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.11.4%26sid%3Dliteratum%253Aachs%26aulast%3DGardiner%26aufirst%3DE.%2BJ.%26aulast%3DHolliday%26aufirst%3DJ.%2BD.%26aulast%3DO%25E2%2580%2599Dowd%26aufirst%3DC.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DEffectiveness%2520of%25202D%2520fingerprints%2520for%2520scaffold%2520hopping%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D405%26epage%3D414%26doi%3D10.4155%2Ffmc.11.4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, S.</span></span> <span> </span><span class="NLM_article-title">PARP inhibitors in pancreatic cancer: molecular mechanisms and clinical applications</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">49</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1186%2Fs12943-020-01167-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=32122376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvVaksbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=49&author=H.+Zhuauthor=M.+Weiauthor=J.+Xuauthor=J.+Huaauthor=C.+Liangauthor=Q.+Mengauthor=Y.+Zhangauthor=J.+Liuauthor=B.+Zhangauthor=X.+Yuauthor=S.+Shi&title=PARP+inhibitors+in+pancreatic+cancer%3A+molecular+mechanisms+and+clinical+applications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">PARP inhibitors in pancreatic cancer: molecular mechanisms and clinical applications</span></div><div class="casAuthors">Zhu, Heng; Wei, Miaoyan; Xu, Jin; Hua, Jie; Liang, Chen; Meng, Qingcai; Zhang, Yiyin; Liu, Jiang; Zhang, Bo; Yu, Xianjun; Shi, Si</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Abstr.: Pancreatic cancer is a highly lethal disease with a poor prognosis, and existing therapies offer only limited effectiveness.  Mutation gene sequencing has shown several gene assocns. that may account for its carcinogenesis, revealing a promising research direction.  Poly (ADP-ribose) polymerase (PARP) inhibitors target tumor cells with a homologous recombination repair (HRR) deficiency based on the concept of synthetic lethality.  The most prominent target gene is BRCA, in which mutations were first identified in breast cancer and ovarian cancer.  PARP inhibitors can trap the PARP-1 protein at a single-stranded break/DNA lesion and disrupt its catalytic cycle, ultimately leading to replication fork progression and consequent double-strand breaks.  For tumor cells with BRCA mutations, HRR loss would result in cell death.  Pancreatic cancer has also been reported to have a strong relationship with BRCA gene mutations, which indicates that pancreatic cancer patients may benefit from PARP inhibitors.  Several clin. trials are being conducted and have begun to yield results.  For example, the POLO (Pancreatic Cancer Olaparib Ongoing) trial has demonstrated that the median progression-free survival was observably longer in the olaparib group than in the placebo group.  However, PARP inhibitor resistance has partially precluded their use in clin. applications, and the major mechanism underlying this resistance is the restoration of HRR.  Therefore, detg. how to use PARP inhibitors in more clin. applications and how to avoid adverse effects, as well as prognosis and treatment response biomarkers, require addnl. research.  This review elaborates on future prospects for the application of PARP inhibitors in pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpds8_DZ3Aiu7Vg90H21EOLACvtfcHk0lhIzKNza63fbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvVaksbs%253D&md5=e147882c753e7c0116a64117255e71c5</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1186%2Fs12943-020-01167-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-020-01167-9%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DWei%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DHua%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DMeng%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DS.%26atitle%3DPARP%2520inhibitors%2520in%2520pancreatic%2520cancer%253A%2520molecular%2520mechanisms%2520and%2520clinical%2520applications%26jtitle%3DMol.%2520Cancer%26date%3D2020%26volume%3D19%26spage%3D49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D. S.</span></span> <span> </span><span class="NLM_article-title">The ups and downs of Poly(ADP-ribose) Polymerase-1 inhibitors in cancer therapy-Current progress and future direction</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>203</i></span>, <span class="NLM_elocation-id">112570</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1016%2Fj.ejmech.2020.112570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=32717529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVOqsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=203&publication_year=2020&author=Y.+Zhaoauthor=L.+X.+Zhangauthor=T.+Jiangauthor=J.+Longauthor=Z.+Y.+Maauthor=A.+P.+Luauthor=Y.+Chengauthor=D.+S.+Cao&title=The+ups+and+downs+of+Poly%28ADP-ribose%29+Polymerase-1+inhibitors+in+cancer+therapy-Current+progress+and+future+direction"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The ups and downs of Poly(ADP-ribose) Polymerase-1 inhibitors in cancer therapy-Current progress and future direction</span></div><div class="casAuthors">Zhao, Yue; Zhang, Liu-Xia; Jiang, Ting; Long, Jing; Ma, Zhong-Ye; Lu, Ai-Ping; Cheng, Yan; Cao, Dong-Sheng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">203</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112570</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) Polymerase 1 (PARP1), one of the most investigated 18 membered PARP family enzymes, is involved in a variety of cellular functions including DNA damage repair, gene transcription and cell apoptosis.  PARP1 can form a PARP1(ADP-ribose) polymers, then bind to the DNA damage gap to recruit DNA repair proteins, and repair the break to maintain genomic stability.  PARP1 is highly expressed in tumor cells, so the inhibition of PARP1 can block DNA repair, promote tumor cell apoptosis, and exert antitumor activity.  To date, four PARP1 inhibitors namely olaparib, rucaparib, niraparib and talazoparib, have been approved by Food and Drug Administration (FDA) for treating ovarian cancer and breast cancer with BRCA1/2 mutation.  These drugs have showed super advantages over conventional chemotherapeutic drugs with low hematol. toxicity and slowly developed drug resistance.  In this article, we summarize and analyze the structure features of PARP1, the biol. functions and antitumor mechanisms of PARP1 inhibitors.  Importantly, we suggest that establishing a new structure-activity relationship of developed PARP1 inhibitors via substructural searching and the matched mol. pair anal. would accelerate the process in finding more potent and safer PARP1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPrtemJ1WT_LVg90H21EOLACvtfcHk0ljIl2lEd8nfFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVOqsb3E&md5=d999be53f0d108423d532411a475194f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112570%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DL.%2BX.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DLong%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DZ.%2BY.%26aulast%3DLu%26aufirst%3DA.%2BP.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%2BS.%26atitle%3DThe%2520ups%2520and%2520downs%2520of%2520Poly%2528ADP-ribose%2529%2520Polymerase-1%2520inhibitors%2520in%2520cancer%2520therapy-Current%2520progress%2520and%2520future%2520direction%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span>. <i>J. Med. Pharma. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Pharma.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span>. <i>J. Med. Pharma. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+Halgren&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Pharma.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keiser, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armbruster, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernsberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span>. <i>Biotechnology</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">197</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0lhP9L7ViVVXAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DBiotechnology%26date%3D2007%26volume%3D25%26spage%3D197%26epage%3D206%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoete, V.</span></span> <span> </span><span class="NLM_article-title">SwissTargetPrediction: updated data and new features for efficient prediction of protein targets of small molecules</span>. <i>Nucleic Acids Symp. Ser.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">W357</span>– <span class="NLM_lpage">W364</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1093%2Fnar%2Fgkz382" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W357-W364&author=A.+Dainaauthor=O.+Michielinauthor=V.+Zoete&title=SwissTargetPrediction%3A+updated+data+and+new+features+for+efficient+prediction+of+protein+targets+of+small+molecules&doi=10.1093%2Fnar%2Fgkz382"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz382%26sid%3Dliteratum%253Aachs%26aulast%3DDaina%26aufirst%3DA.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DZoete%26aufirst%3DV.%26atitle%3DSwissTargetPrediction%253A%2520updated%2520data%2520and%2520new%2520features%2520for%2520efficient%2520prediction%2520of%2520protein%2520targets%2520of%2520small%2520molecules%26jtitle%3DNucleic%2520Acids%2520Symp.%2520Ser.%26date%3D2019%26volume%3D47%26spage%3DW357%26epage%3DW364%26doi%3D10.1093%2Fnar%2Fgkz382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D. S.</span></span> <span> </span><span class="NLM_article-title">TargetNet: a web service for predicting potential drug-target interaction profiling via multi-target SAR models</span>. <i>Perspect. Drug Discovery Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1007/s10822-016-9915-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1007%2Fs10822-016-9915-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=413-424&author=Z.+J.+Yaoauthor=J.+Dongauthor=Y.+J.+Cheauthor=M.+F.+Zhuauthor=M.+Wenauthor=N.+N.+Wangauthor=S.+Wangauthor=A.+P.+Luauthor=D.+S.+Cao&title=TargetNet%3A+a+web+service+for+predicting+potential+drug-target+interaction+profiling+via+multi-target+SAR+models&doi=10.1007%2Fs10822-016-9915-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1007%2Fs10822-016-9915-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-016-9915-2%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DZ.%2BJ.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DChe%26aufirst%3DY.%2BJ.%26aulast%3DZhu%26aufirst%3DM.%2BF.%26aulast%3DWen%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DN.%2BN.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DA.%2BP.%26aulast%3DCao%26aufirst%3DD.%2BS.%26atitle%3DTargetNet%253A%2520a%2520web%2520service%2520for%2520predicting%2520potential%2520drug-target%2520interaction%2520profiling%2520via%2520multi-target%2520SAR%2520models%26jtitle%3DPerspect.%2520Drug%2520Discovery%2520Des.%26date%3D2016%26volume%3D30%26spage%3D413%26epage%3D424%26doi%3D10.1007%2Fs10822-016-9915-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H. P. M.</span></span> <span> </span><span class="NLM_article-title">Update: a web server for potential drug target identification with a comprehensive target pharmacophore database</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2017</i></span>,  <span class="NLM_fpage">W356</span>– <span class="NLM_lpage">W360</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1093%2Fnar%2Fgkx374" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2017&publication_year=2017&pages=W356-W360&author=X.+Wangauthor=Y.+Shenauthor=S.+Wangauthor=S.+Liauthor=W.+Zhangauthor=X.+Liuauthor=L.+Laiauthor=J.+Peiauthor=H.+P.+M.+Li&title=Update%3A+a+web+server+for+potential+drug+target+identification+with+a+comprehensive+target+pharmacophore+database"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx374%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLai%26aufirst%3DL.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%2BP.%2BM.%26atitle%3DUpdate%253A%2520a%2520web%2520server%2520for%2520potential%2520drug%2520target%2520identification%2520with%2520a%2520comprehensive%2520target%2520pharmacophore%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D2017%26spage%3DW356%26epage%3DW360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grisoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Scaffold-Hopping from Synthetic Drugs by Holistic Molecular Representation</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">16469</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1038%2Fs41598-018-34677-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=30405170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A280%3ADC%252BB3cvnslSmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&author=F.+Grisoniauthor=D.+Merkauthor=R.+Byrneauthor=G.+Schneider&title=Scaffold-Hopping+from+Synthetic+Drugs+by+Holistic+Molecular+Representation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold-Hopping from Synthetic Drugs by Holistic Molecular Representation</span></div><div class="casAuthors">Grisoni Francesca; Merk Daniel; Byrne Ryan; Schneider Gisbert; Grisoni Francesca</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16469</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The discovery of novel ligand chemotypes allows to explore uncharted regions in chemical space, thereby potentially improving synthetic accessibility, potency, and the drug-likeness of molecules.  Here, we demonstrate the scaffold-hopping ability of the new Weighted Holistic Atom Localization and Entity Shape (WHALES) molecular descriptors compared to seven state-of-the-art molecular representations on 30,000 compounds and 182 biological targets.  In a prospective application, we apply WHALES to the discovery of novel retinoid X receptor (RXR) modulators.  WHALES descriptors identified four agonists with innovative molecular scaffolds, populating uncharted regions of the chemical space.  One of the agonists, possessing a rare non-acidic chemotype, revealed high selectivity on 12 nuclear receptors and comparable efficacy as bexarotene on induction of ATP-binding cassette transporter A1, angiopoietin like protein 4 and apolipoprotein E.  The outcome of this research supports WHALES as an innovative tool to explore novel regions of the chemical space and to detect novel bioactive chemotypes by straightforward similarity searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSbrRUzliurTk2yPDyEFiigfW6udTcc2ea9q0dWfiLYbrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvnslSmuw%253D%253D&md5=79373c2f6568169c54ed622d2e69a633</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-34677-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-34677-0%26sid%3Dliteratum%253Aachs%26aulast%3DGrisoni%26aufirst%3DF.%26aulast%3DMerk%26aufirst%3DD.%26aulast%3DByrne%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DScaffold-Hopping%2520from%2520Synthetic%2520Drugs%2520by%2520Holistic%2520Molecular%2520Representation%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grisoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Consonni, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiss, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tagliabue, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todeschini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Scaffold hopping from natural products to synthetic mimetics by holistic molecular similarity</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvVyqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=1-9&author=F.+Grisoniauthor=D.+Merkauthor=V.+Consonniauthor=J.+A.+Hissauthor=S.+G.+Tagliabueauthor=R.+Todeschiniauthor=G.+Schneider&title=Scaffold+hopping+from+natural+products+to+synthetic+mimetics+by+holistic+molecular+similarity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold hopping from natural products to synthetic mimetics by holistic molecular similarity</span></div><div class="casAuthors">Grisoni, Francesca; Merk, Daniel; Consonni, Viviana; Hiss, Jan A.; Tagliabue, Sara Giani; Todeschini, Roberto; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Communications Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>CODEN:
                <span class="NLM_cas:coden">CCOHCT</span>;
        ISSN:<span class="NLM_cas:issn">2399-3669</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Natural products offer unexplored mol. frameworks for the development of chem. leads and innovative drugs.  However, the structural complexity of natural products compared with synthetic drug-like mols. often limits the scaffold hopping potential of natural-product-inspired mol. design.  Here we introduce a holistic mol. representation incorporating pharmacophore and shape patterns, which facilitates scaffold hopping from natural products to isofunctional synthetic compds.  This computational approach captures simultaneously the partial charge, atom distributions and mol. shape.  In a prospective application, we use four natural cannabinoids as queries in a chem. database search for novel synthetic modulators of human cannabinoid receptors.  Of the synthetic compds. selected by the new method, 35% are exptl. confirmed as active.  These cannabinoid receptor modulators are structurally less complex than their resp. natural product templates.  The results of this study validate this holistic mol. representation for hit and lead finding in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovwYjU-7sJ17Vg90H21EOLACvtfcHk0ljhNi6ugRBnNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvVyqtLk%253D&md5=1b28699db14d8c63b992561efa73821d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGrisoni%26aufirst%3DF.%26aulast%3DMerk%26aufirst%3DD.%26aulast%3DConsonni%26aufirst%3DV.%26aulast%3DHiss%26aufirst%3DJ.%2BA.%26aulast%3DTagliabue%26aufirst%3DS.%2BG.%26aulast%3DTodeschini%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DScaffold%2520hopping%2520from%2520natural%2520products%2520to%2520synthetic%2520mimetics%2520by%2520holistic%2520molecular%2520similarity%26jtitle%3DCommun.%2520Chem.%26date%3D2018%26volume%3D1%26spage%3D1%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bemis, G.</span></span> <span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span>. <i>J. Med. Pharm. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2893</span>, <span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+Bemis&title=The+properties+of+known+drugs.+1.+Molecular+frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Pharm.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fillbrunn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeuffer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrum, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthold, M. R.</span></span> <span> </span><span class="NLM_article-title">KNIME for reproducible cross-domain analysis of life science data</span>. <i>J. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>261</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1016/j.jbiotec.2017.07.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1016%2Fj.jbiotec.2017.07.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=28757290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1yrs73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=2017&pages=149-156&author=A.+Fillbrunnauthor=C.+Dietzauthor=J.+Pfeufferauthor=R.+Rahnauthor=G.+A.+Landrumauthor=M.+R.+Berthold&title=KNIME+for+reproducible+cross-domain+analysis+of+life+science+data&doi=10.1016%2Fj.jbiotec.2017.07.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">KNIME for reproducible cross-domain analysis of life science data</span></div><div class="casAuthors">Fillbrunn, Alexander; Dietz, Christian; Pfeuffer, Julianus; Rahn, Rene; Landrum, Gregory A.; Berthold, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biotechnology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">149-156</span>CODEN:
                <span class="NLM_cas:coden">JBITD4</span>;
        ISSN:<span class="NLM_cas:issn">0168-1656</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Expts. in the life sciences often involve tools from a variety of domains such as mass spectrometry, next generation sequencing, or image processing.  Passing the data between those tools often involves complex scripts for controlling data flow, data transformation, and statistical anal.  Such scripts are not only prone to be platform dependent, they also tend to grow as the expt. progresses and are seldomly well documented, a fact that hinders the reproducibility of the expt.  Workflow systems such as KNIME Analytics Platform aim to solve these problems by providing a platform for connecting tools graphically and guaranteeing the same results on different operating systems.  As an open source software, KNIME allows scientists and programmers to provide their own extensions to the scientific community.  In this review paper we present selected extensions from the life sciences that simplify data exploration, anal., and visualization and are interoperable due to KNIME's unified data model.  Addnl., we name other workflow systems that are commonly used in the life sciences and highlight their similarities and differences to KNIME.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK1RhJJLnEJ7Vg90H21EOLACvtfcHk0ljuxny3_IH7BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1yrs73I&md5=ab6ae9e93da9fe972abe5f99903b33cc</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.jbiotec.2017.07.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbiotec.2017.07.028%26sid%3Dliteratum%253Aachs%26aulast%3DFillbrunn%26aufirst%3DA.%26aulast%3DDietz%26aufirst%3DC.%26aulast%3DPfeuffer%26aufirst%3DJ.%26aulast%3DRahn%26aufirst%3DR.%26aulast%3DLandrum%26aufirst%3DG.%2BA.%26aulast%3DBerthold%26aufirst%3DM.%2BR.%26atitle%3DKNIME%2520for%2520reproducible%2520cross-domain%2520analysis%2520of%2520life%2520science%2520data%26jtitle%3DJ.%2520Biotechnol.%26date%3D2017%26volume%3D261%26spage%3D149%26epage%3D156%26doi%3D10.1016%2Fj.jbiotec.2017.07.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berthold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cebron, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dill, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kötter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieb, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiswedel, B.</span></span> <span> </span><span class="NLM_article-title">KNIME: The Konstanz Information Miner</span>. <i>ACM Sigkdd Explor. Newsl.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">31</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1145%2F1656274.1656280" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=26-31&author=M.+R.+Bertholdauthor=N.+Cebronauthor=F.+Dillauthor=T.+R.+Gabrielauthor=T.+K%C3%B6tterauthor=T.+Meinlauthor=P.+Ohlauthor=C.+Siebauthor=K.+Thielauthor=B.+Wiswedel&title=KNIME%3A+The+Konstanz+Information+Miner"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1145%2F1656274.1656280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1145%252F1656274.1656280%26sid%3Dliteratum%253Aachs%26aulast%3DBerthold%26aufirst%3DM.%2BR.%26aulast%3DCebron%26aufirst%3DN.%26aulast%3DDill%26aufirst%3DF.%26aulast%3DGabriel%26aufirst%3DT.%2BR.%26aulast%3DK%25C3%25B6tter%26aufirst%3DT.%26aulast%3DMeinl%26aufirst%3DT.%26aulast%3DOhl%26aufirst%3DP.%26aulast%3DSieb%26aufirst%3DC.%26aulast%3DThiel%26aufirst%3DK.%26aulast%3DWiswedel%26aufirst%3DB.%26atitle%3DKNIME%253A%2520The%2520Konstanz%2520Information%2520Miner%26jtitle%3DACM%2520Sigkdd%2520Explor.%2520Newsl.%26date%3D2006%26volume%3D11%26spage%3D26%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geppert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Scaffold hopping using two-dimensional fingerprints: true potential, black magic, or a hopeless endeavor? Guidelines for virtual screening</span>. <i>J. Med. Pharm. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5707</span>– <span class="NLM_lpage">5715</span>, <span class="refDoi"> DOI: 10.1021/jm100492z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100492z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5707-5715&author=M.+Vogtauthor=D.+Stumpfeauthor=H.+Geppertauthor=J.+Bajorath&title=Scaffold+hopping+using+two-dimensional+fingerprints%3A+true+potential%2C+black+magic%2C+or+a+hopeless+endeavor%3F+Guidelines+for+virtual+screening&doi=10.1021%2Fjm100492z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm100492z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100492z%26sid%3Dliteratum%253Aachs%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DGeppert%26aufirst%3DH.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DScaffold%2520hopping%2520using%2520two-dimensional%2520fingerprints%253A%2520true%2520potential%252C%2520black%2520magic%252C%2520or%2520a%2520hopeless%2520endeavor%253F%2520Guidelines%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Pharm.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5707%26epage%3D5715%26doi%3D10.1021%2Fjm100492z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Southan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boppana, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagarlapudi, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muresan, S.</span></span> <span> </span><span class="NLM_article-title">Analysis of in vitro bioactivity data extracted from drug discovery literature and patents: Ranking 1654 human protein targets by assayed compounds and molecular scaffolds</span>. <i>Aust. J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">14</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=14&author=C.+Southanauthor=K.+Boppanaauthor=S.+A.+Jagarlapudiauthor=S.+Muresan&title=Analysis+of+in+vitro+bioactivity+data+extracted+from+drug+discovery+literature+and+patents%3A+Ranking+1654+human+protein+targets+by+assayed+compounds+and+molecular+scaffolds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSouthan%26aufirst%3DC.%26aulast%3DBoppana%26aufirst%3DK.%26aulast%3DJagarlapudi%26aufirst%3DS.%2BA.%26aulast%3DMuresan%26aufirst%3DS.%26atitle%3DAnalysis%2520of%2520in%2520vitro%2520bioactivity%2520data%2520extracted%2520from%2520drug%2520discovery%2520literature%2520and%2520patents%253A%2520Ranking%25201654%2520human%2520protein%2520targets%2520by%2520assayed%2520compounds%2520and%2520molecular%2520scaffolds%26jtitle%3DAust.%2520J.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">ChEMBL web services: streamlining access to drug discovery data and utilities</span>. <i>Nucleic Acids Symp. Ser.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">W612</span>– <span class="NLM_lpage">W620</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1093%2Fnar%2Fgkv352" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=W612-W620&author=M.+Daviesauthor=M.+Nowotkaauthor=G.+Papadatosauthor=N.+Dedmanauthor=A.+Gaultonauthor=F.+Atkinsonauthor=L.+Bellisauthor=J.+P.+Overington&title=ChEMBL+web+services%3A+streamlining+access+to+drug+discovery+data+and+utilities&doi=10.1093%2Fnar%2Fgkv352"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv352%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DDedman%26aufirst%3DN.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DAtkinson%26aufirst%3DF.%26aulast%3DBellis%26aufirst%3DL.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%2520web%2520services%253A%2520streamlining%2520access%2520to%2520drug%2520discovery%2520data%2520and%2520utilities%26jtitle%3DNucleic%2520Acids%2520Symp.%2520Ser.%26date%3D2015%26volume%3D43%26spage%3DW612%26epage%3DW620%26doi%3D10.1093%2Fnar%2Fgkv352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Veij, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felix, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magarinos, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosquera, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutowo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordillo-Maranon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mugumbate, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Lopez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radoux, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura-Cabrera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span> <span> </span><span class="NLM_article-title">ChEMBL: towards direct deposition of bioassay data</span>. <i>Nucleic Acids Symp. Ser.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">D930</span>– <span class="NLM_lpage">D940</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1093%2Fnar%2Fgky1075" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D930-D940&author=D.+Mendezauthor=A.+Gaultonauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+De+Veijauthor=E.+Felixauthor=M.+P.+Magarinosauthor=J.+F.+Mosqueraauthor=P.+Mutowoauthor=M.+Nowotkaauthor=M.+Gordillo-Maranonauthor=F.+Hunterauthor=L.+Juncoauthor=G.+Mugumbateauthor=M.+Rodriguez-Lopezauthor=F.+Atkinsonauthor=N.+Boscauthor=C.+J.+Radouxauthor=A.+Segura-Cabreraauthor=A.+Herseyauthor=A.+R.+Leach&title=ChEMBL%3A+towards+direct+deposition+of+bioassay+data&doi=10.1093%2Fnar%2Fgky1075"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1075%26sid%3Dliteratum%253Aachs%26aulast%3DMendez%26aufirst%3DD.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDe%2BVeij%26aufirst%3DM.%26aulast%3DFelix%26aufirst%3DE.%26aulast%3DMagarinos%26aufirst%3DM.%2BP.%26aulast%3DMosquera%26aufirst%3DJ.%2BF.%26aulast%3DMutowo%26aufirst%3DP.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DGordillo-Maranon%26aufirst%3DM.%26aulast%3DHunter%26aufirst%3DF.%26aulast%3DJunco%26aufirst%3DL.%26aulast%3DMugumbate%26aufirst%3DG.%26aulast%3DRodriguez-Lopez%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DF.%26aulast%3DBosc%26aufirst%3DN.%26aulast%3DRadoux%26aufirst%3DC.%2BJ.%26aulast%3DSegura-Cabrera%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26atitle%3DChEMBL%253A%2520towards%2520direct%2520deposition%2520of%2520bioassay%2520data%26jtitle%3DNucleic%2520Acids%2520Symp.%2520Ser.%26date%3D2019%26volume%3D47%26spage%3DD930%26epage%3DD940%26doi%3D10.1093%2Fnar%2Fgky1075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorissen, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">BindingDB: a web-accessible database of experimentally determined protein-ligand binding affinities</span>. <i>Nucleic Acids Symp. Ser.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">D198</span>– <span class="NLM_lpage">D201</span>, <span class="refDoi"> DOI: 10.1093/nar/gkl999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1093%2Fnar%2Fgkl999" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=D198-D201&author=T.+Liuauthor=Y.+Linauthor=X.+Wenauthor=R.+N.+Jorissenauthor=M.+K.+Gilson&title=BindingDB%3A+a+web-accessible+database+of+experimentally+determined+protein-ligand+binding+affinities&doi=10.1093%2Fnar%2Fgkl999"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl999%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DJorissen%26aufirst%3DR.%2BN.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520a%2520web-accessible%2520database%2520of%2520experimentally%2520determined%2520protein-ligand%2520binding%2520affinities%26jtitle%3DNucleic%2520Acids%2520Symp.%2520Ser.%26date%3D2007%26volume%3D35%26spage%3DD198%26epage%3DD201%26doi%3D10.1093%2Fnar%2Fgkl999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span> <i>Molecular
Operating Environment (MOE), 2018.01</i>; <span class="NLM_publisher-name">Chemical
Computing Group ULC</span>, <span class="NLM_publisher-loc">Montreal,
QC</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular%0AOperating+Environment+%28MOE%29%2C+2018.01%3B+Chemical%0AComputing+Group+ULC%2C+Montreal%2C%0AQC%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMolecular%250AOperating%2520Environment%2520%2528MOE%2529%252C%25202018.01%26pub%3DChemical%250AComputing%2520Group%2520ULC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Breiman, L.</span></span> <span> </span><span class="NLM_article-title">Random forests</span>. <i>Mach. Learn.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">32</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1023%2FA%3A1010933404324" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2001&pages=5-32&author=L.+Breiman&title=Random+forests"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1023%2FA%3A1010933404324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1010933404324%26sid%3Dliteratum%253Aachs%26aulast%3DBreiman%26aufirst%3DL.%26atitle%3DRandom%2520forests%26jtitle%3DMach.%2520Learn.%26date%3D2001%26volume%3D45%26spage%3D5%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Freund, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapire, R. E.</span></span> <span> </span><span class="NLM_article-title">A decision-theoretic generalization of on-line learning and an application to boosting</span>. <i>J. Comput. Syst. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">139</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1006%2Fjcss.1997.1504" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1997&pages=119-139&author=Y.+Freundauthor=R.+E.+Schapire&title=A+decision-theoretic+generalization+of+on-line+learning+and+an+application+to+boosting"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1006%2Fjcss.1997.1504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjcss.1997.1504%26sid%3Dliteratum%253Aachs%26aulast%3DFreund%26aufirst%3DY.%26aulast%3DSchapire%26aufirst%3DR.%2BE.%26atitle%3DA%2520decision-theoretic%2520generalization%2520of%2520on-line%2520learning%2520and%2520an%2520application%2520to%2520boosting%26jtitle%3DJ.%2520Comput.%2520Syst.%2520Sci.%26date%3D1997%26volume%3D55%26spage%3D119%26epage%3D139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, J. H.</span></span> <span> </span><span class="NLM_article-title">Greedy function approximation: a gradient boosting machine</span>. <i>Ann. Stat.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_fpage">1189</span>– <span class="NLM_lpage">1232</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&pages=1189-1232&author=J.+H.+Friedman&title=Greedy+function+approximation%3A+a+gradient+boosting+machine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFriedman%26aufirst%3DJ.%2BH.%26atitle%3DGreedy%2520function%2520approximation%253A%2520a%2520gradient%2520boosting%2520machine%26jtitle%3DAnn.%2520Stat.%26date%3D2001%26spage%3D1189%26epage%3D1232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiener, M.</span></span> <span> </span><span class="NLM_article-title">Classification and regression by randomForest</span>. <i>R News</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">22</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=18-22&author=A.+Liawauthor=M.+Wiener&title=Classification+and+regression+by+randomForest"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DWiener%26aufirst%3DM.%26atitle%3DClassification%2520and%2520regression%2520by%2520randomForest%26jtitle%3DR%2520News%26date%3D2002%26volume%3D2%26spage%3D18%26epage%3D22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alfaro, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gámez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, N.</span></span> <span> </span><span class="NLM_article-title">adabag: An R package for classification with boosting and bagging</span>. <i>J. Stat. Soft.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">35</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.18637%2Fjss.v054.i02" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2013&pages=1-35&author=E.+Alfaroauthor=M.+G%C3%A1mezauthor=N.+Garcia&title=adabag%3A+An+R+package+for+classification+with+boosting+and+bagging"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.18637%2Fjss.v054.i02&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18637%252Fjss.v054.i02%26sid%3Dliteratum%253Aachs%26aulast%3DAlfaro%26aufirst%3DE.%26aulast%3DG%25C3%25A1mez%26aufirst%3DM.%26aulast%3DGarcia%26aufirst%3DN.%26atitle%3Dadabag%253A%2520An%2520R%2520package%2520for%2520classification%2520with%2520boosting%2520and%2520bagging%26jtitle%3DJ.%2520Stat.%2520Soft.%26date%3D2013%26volume%3D54%26spage%3D1%26epage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ridgeway, G.</span></span> <i>gbm: Generalized boosted regression models</i>. R package version <span class="NLM_year">2006</span>,  <span class="NLM_volume">1</span>,  <span class="NLM_fpage">55</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=G.+Ridgeway&title=gbm%3A+Generalized+boosted+regression+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DRidgeway%26aufirst%3DG.%26jtitle%3Dgbm%253A%2520Generalized%2520boosted%2520regression%2520models%26date%3D2006%26volume%3D1%26spage%3D55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Box plots of the SRR values within top 100 (A) and top 5000 (B) for different screening data sets. The horizontal lines indicate the median, and the plus signs represent the mean SRR values of the 1660 query molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Comparison of the CBFP representation and the other chemical descriptors for the five screening datasets in terms of the SRR results within the top 100 compounds. For each chemical descriptor, there are 17 points representing 17 different query datasets. Points above the diagonal are the specific query targets for which the CBFP representation outperformed the chemical descriptors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Heatmap of the average SRR values (top 100 compounds) of the individual targets for the five different screening datasets. Annotations on the left of the heatmap show the corresponding targets. The capital letters at the bottom refer to different molecular representations: A, AtomPair; M, MACCS; E, ECFP4; F, FP2; G, Gpi_DAPH3; D, TGD; T, TGT; C, CATS; O, MOE; P, CBFP. Colors represent a three gradient from the worst (red) to the best (blue).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of DHFR and HIVRT in terms of chemical diversity. The Tanimoto similarity between the molecules with different scaffolds was calculated based on the ECFP4 descriptors to represent the chemical diversity of the query datasets. (A) Frequent distribution curves of the Tanimoto similarity values of DHFR and HIVRT. (B) Box plot of the nearest neighbor Tanimoto similarity of query molecules. HIVPR: red, DHFR: blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Scaffold hopping performance within 1% (A) and 0.5% (B) for the five screening datasets comparing CBFP with the ECFP4 and MOE2D representations. CBFP: green, ECFP4: blue, MOE2D: yellow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/medium/jm1c00234_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Prospective evaluation for finding novel PARP1 inhibitors. (A) Cellular growth-inhibitory activities of the 35 hit compounds. (B) Dose–response curves for the most potent hit compounds 6, 24, and 39.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00234/20210602/images/large/jm1c00234_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00234&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i24">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22366" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22366" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 61 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruddigkeit, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Deursen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J. L.</span></span> <span> </span><span class="NLM_article-title">Enumeration of 166 billion organic small molecules in the chemical universe database GDB-17</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2864</span>– <span class="NLM_lpage">2875</span>, <span class="refDoi"> DOI: 10.1021/ci300415d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300415d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2864-2875&author=L.+Ruddigkeitauthor=R.+van+Deursenauthor=L.+C.+Blumauthor=J.+L.+Reymond&title=Enumeration+of+166+billion+organic+small+molecules+in+the+chemical+universe+database+GDB-17&doi=10.1021%2Fci300415d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fci300415d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300415d%26sid%3Dliteratum%253Aachs%26aulast%3DRuddigkeit%26aufirst%3DL.%26aulast%3Dvan%2BDeursen%26aufirst%3DR.%26aulast%3DBlum%26aufirst%3DL.%2BC.%26aulast%3DReymond%26aufirst%3DJ.%2BL.%26atitle%3DEnumeration%2520of%2520166%2520billion%2520organic%2520small%2520molecules%2520in%2520the%2520chemical%2520universe%2520database%2520GDB-17%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2012%26volume%3D52%26spage%3D2864%26epage%3D2875%26doi%3D10.1021%2Fci300415d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wallach, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heifets, A.</span></span> <span> </span><span class="NLM_article-title">Most Ligand-Based Classification Benchmarks Reward Memorization Rather than Generalization</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">916</span>– <span class="NLM_lpage">932</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00403</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00403" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=916-932&author=I.+Wallachauthor=A.+Heifets&title=Most+Ligand-Based+Classification+Benchmarks+Reward+Memorization+Rather+than+Generalization&doi=10.1021%2Facs.jcim.7b00403"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00403%26sid%3Dliteratum%253Aachs%26aulast%3DWallach%26aufirst%3DI.%26aulast%3DHeifets%26aufirst%3DA.%26atitle%3DMost%2520Ligand-Based%2520Classification%2520Benchmarks%2520Reward%2520Memorization%2520Rather%2520than%2520Generalization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2018%26volume%3D58%26spage%3D916%26epage%3D932%26doi%3D10.1021%2Facs.jcim.7b00403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gastreich, M.</span></span> <span> </span><span class="NLM_article-title">The next level in chemical space navigation: going far beyond enumerable compound libraries</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1148</span>– <span class="NLM_lpage">1156</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1016%2Fj.drudis.2019.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=30851414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkslaksb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=1148-1156&author=T.+Hoffmannauthor=M.+Gastreich&title=The+next+level+in+chemical+space+navigation%3A+going+far+beyond+enumerable+compound+libraries"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The next level in chemical space navigation: going far beyond enumerable compound libraries</span></div><div class="casAuthors">Hoffmann, Torsten; Gastreich, Marcus</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1148-1156</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Recent innovations have brought pharmacophore-driven methods for navigating virtual chem. spaces, the size of which can reach into the billions of mols., to the fingertips of every chemist.  There has been a paradigm shift in the underlying computational chem. that drives chem. space search applications, incorporating intelligent reaction knowledge into their core so that they can readily deliver com. available mols. as nearest neighbor hits from within giant virtual spaces.  These vast resources enable medicinal chemists to execute rapid scaffold-hopping expts., rapid hit expansion, and structure-activity relationship (SAR) exploitation in largely intellectual property (IP)-free territory and at unparalleled low cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodA_WnK9Z3jbVg90H21EOLACvtfcHk0lgJ0MQRYMktVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkslaksb4%253D&md5=d707299b8c6e0ace79f27172147517e3</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2019.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2019.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHoffmann%26aufirst%3DT.%26aulast%3DGastreich%26aufirst%3DM.%26atitle%3DThe%2520next%2520level%2520in%2520chemical%2520space%2520navigation%253A%2520going%2520far%2520beyond%2520enumerable%2520compound%2520libraries%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26spage%3D1148%26epage%3D1156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobson, C. M.</span></span> <span> </span><span class="NLM_article-title">Chemical space and biology</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>432</i></span>,  <span class="NLM_fpage">824</span>– <span class="NLM_lpage">828</span>, <span class="refDoi"> DOI: 10.1038/nature03192</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1038%2Fnature03192" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=15602547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=824-828&author=C.+M.+Dobson&title=Chemical+space+and+biology&doi=10.1038%2Fnature03192"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical space and biology</span></div><div class="casAuthors">Dobson, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">824-828</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. space - which encompasses all possible small org. mols., including those present in biol. systems - is vast.  So vast, in fact, that so far only a tiny fraction of it has been explored.  Nevertheless, these explorations have greatly enhanced our understanding of biol., and have led to the development of many of today's drugs.  The discovery of new bioactive mols., facilitated by a deeper understanding of the nature of the regions of chem. space that are relevant to biol., will advance our knowledge of biol. processes and lead to new strategies to treat disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolNmRTVWoSDrVg90H21EOLACvtfcHk0lhKlw4Nsd65jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jN&md5=b951fff4aa4877682756fb1831bff09d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature03192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03192%26sid%3Dliteratum%253Aachs%26aulast%3DDobson%26aufirst%3DC.%2BM.%26atitle%3DChemical%2520space%2520and%2520biology%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D824%26epage%3D828%26doi%3D10.1038%2Fnature03192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bohacek, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMartin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span> <span> </span><span class="NLM_article-title">The art and practice of structure-based drug design: a molecular modeling perspective</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1098-1128(199601)16:1<3::AID-MED1>3.0.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=8788213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1996&pages=3-50&author=R.+S.+Bohacekauthor=C.+McMartinauthor=W.+C.+Guida&title=The+art+and+practice+of+structure-based+drug+design%3A+a+molecular+modeling+perspective&doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The art and practice of structure-based drug design: a molecular modeling perspective</span></div><div class="casAuthors">Bohacek, Regine S.; McMartin, Colin; Guida, Wayne C.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-50</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review with 120 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmfrWdtPosdrVg90H21EOLACvtfcHk0lhKlw4Nsd65jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D&md5=edfa592791c111479117feed600dead6</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291098-1128%2528199601%252916%253A1%253C3%253A%253AAID-MED1%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DBohacek%26aufirst%3DR.%2BS.%26aulast%3DMcMartin%26aufirst%3DC.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26atitle%3DThe%2520art%2520and%2520practice%2520of%2520structure-based%2520drug%2520design%253A%2520a%2520molecular%2520modeling%2520perspective%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D1996%26volume%3D16%26spage%3D3%26epage%3D50%26doi%3D10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renner, S.</span></span> <span> </span><span class="NLM_article-title">Scaffold-Hopping: How Far Can You Jump?</span>. <i>QSAR Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1162</span>– <span class="NLM_lpage">1171</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1002%2Fqsar.200610091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsFCjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=1162-1171&author=G.+Schneiderauthor=P.+Schneiderauthor=S.+Renner&title=Scaffold-Hopping%3A+How+Far+Can+You+Jump%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold-hopping: how far can you jump?</span></div><div class="casAuthors">Schneider, Gisbert; Schneider, Petra; Renner, Steffen</div><div class="citationInfo"><span class="NLM_cas:title">QSAR & Combinatorial Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1162-1171</span>CODEN:
                <span class="NLM_cas:coden">QCSSAU</span>;
        ISSN:<span class="NLM_cas:issn">1611-020X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Finding new isofunctional chemotypes with the aim of identifying new lead candidates remains a challenging task in medicinal chem.  Different virtual screening techniques were shown to be useful for this scaffold-hopping process.  We have compiled recent examples of scaffold-hops that were achieved by virtual screening.  Most of these techniques rely on some sort of similarity estn. between known ref. mols. and screening compds., some include receptor-structure information and scoring of the ligand - receptor interaction.  In this review, we discuss current scaffold-hopping strategies and pinpoint potential future directions.  Challenges for future research lie in the identification of an appropriate level of abstraction from an atomistic mol. representation to allow for the detection of isofunctional chemotypes, the rational design of chemotype diversity in screening libraries, and an understanding of the distribution of similarity scores for a given screening library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRGMZFw4ah2rVg90H21EOLACvtfcHk0ljaAg7yDYEKdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsFCjtg%253D%253D&md5=d859a9eeec334ee140d2cf6423bf3b21</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fqsar.200610091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fqsar.200610091%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DRenner%26aufirst%3DS.%26atitle%3DScaffold-Hopping%253A%2520How%2520Far%2520Can%2520You%2520Jump%253F%26jtitle%3DQSAR%2520Comb.%2520Sci.%26date%3D2006%26volume%3D25%26spage%3D1162%26epage%3D1171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funatsu, K.</span></span> <span> </span><span class="NLM_article-title">Exploring Topological Pharmacophore Graphs for Scaffold Hopping</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>2073–2081</i></span>,  <span class="NLM_fpage">60</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2073%E2%80%932081&publication_year=2020&pages=60&author=H.+Nakanoauthor=T.+Miyaoauthor=K.+Funatsu&title=Exploring+Topological+Pharmacophore+Graphs+for+Scaffold+Hopping"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNakano%26aufirst%3DH.%26aulast%3DMiyao%26aufirst%3DT.%26aulast%3DFunatsu%26aufirst%3DK.%26atitle%3DExploring%2520Topological%2520Pharmacophore%2520Graphs%2520for%2520Scaffold%2520Hopping%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D2073%25E2%2580%25932081%26spage%3D60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neidhart, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, G.</span></span> <span> </span><span class="NLM_article-title">“Scaffold-Hopping” by Topological Pharmacophore Search: A Contribution to Virtual Screening</span>. <i>Angew. Chem. Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">2894</span>– <span class="NLM_lpage">2896</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1002%2F%28SICI%291521-3773%2819991004%2938%3A19%3C2894%3A%3AAID-ANIE2894%3E3.0.CO%3B2-F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10540384" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A280%3ADC%252BC2sbhtVertA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=2894-2896&author=G.+Schneiderauthor=W.+Neidhartauthor=T.+Gillerauthor=G.+Schmid&title=%E2%80%9CScaffold-Hopping%E2%80%9D+by+Topological+Pharmacophore+Search%3A+A+Contribution+to+Virtual+Screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">"Scaffold-Hopping" by Topological Pharmacophore Search: A Contribution to Virtual Screening</span></div><div class="casAuthors">Schneider; Neidhart; Giller; Schmid</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2894-2896</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A chemically advanced template search (CATS) based on topological pharmacophore models has been developed as a technique for virtual screening.  This technique has successfully identified novel potent Ca(2+) antagonists (such as 2) that have a similar activity to 1 (a known T-channel blocking agent) in a library of several hundred thousand compounds on the basis of a correlation vector representation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRWkOLBNK38cedfST00W0BcfW6udTcc2eaOYmoh4BSnxbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbhtVertA%253D%253D&md5=c2d88f2b56ea8af78fb167641e254384</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819991004%2938%3A19%3C2894%3A%3AAID-ANIE2894%3E3.0.CO%3B2-F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819991004%252938%253A19%253C2894%253A%253AAID-ANIE2894%253E3.0.CO%253B2-F%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DNeidhart%26aufirst%3DW.%26aulast%3DGiller%26aufirst%3DT.%26aulast%3DSchmid%26aufirst%3DG.%26atitle%3D%25E2%2580%259CScaffold-Hopping%25E2%2580%259D%2520by%2520Topological%2520Pharmacophore%2520Search%253A%2520A%2520Contribution%2520to%2520Virtual%2520Screening%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%2520Engl.%26date%3D1999%26volume%3D38%26spage%3D2894%26epage%3D2896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bravo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berrocoso, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and development of tramadol for the treatment of pain</span>. <i>Expert. Opin. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1281</span>– <span class="NLM_lpage">1291</span>, <span class="refDoi"> DOI: 10.1080/17460441.2017.1377697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1080%2F17460441.2017.1377697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=28920461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFajtbfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1281-1291&author=L.+Bravoauthor=J.+A.+Micoauthor=E.+Berrocoso&title=Discovery+and+development+of+tramadol+for+the+treatment+of+pain&doi=10.1080%2F17460441.2017.1377697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and development of tramadol for the treatment of pain</span></div><div class="casAuthors">Bravo, Lidia; Mico, Juan Antonio; Berrocoso, Esther</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1281-1291</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">: Tramadol is an opioid drug that, unlike classic opioids, also modulates the monoaminergic system by inhibiting noradrenergic and serotoninergic reuptake.  For this reason, tramadol is considered an atypical opioid.  These special pharmacol. characteristics have made tramadol one of the most commonly prescribed analgesic drugs to treat moderate to severe pain.: The aim of this review is to provide a historical description of the biochem., pharmacokinetics and particularly, the mechanisms of action of tramadol.  In addn., a summary is offered of the analgesic effects of tramadol in a variety of animal models of acute and chronic pain.  Finally, clin. studies that demonstrate the efficacy and safety of tramadol in the treatment of pain are also assessed.: The discovery that tramadol combines opioid and monoaminergic effects represented a milestone in the evolution of pain treatment.  Given its 'mild effect' on opioid receptors, tramadol induces fewer side effects than classic opioids.  Tramadol produces satisfactory analgesia against various types of pain and it is currently approved for the treatment of moderate to severe pain.  Thus, the combination of monoamine and opioid mechanisms opens new avenues for the design of innovative analgesics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLiCyb6lVHW7Vg90H21EOLACvtfcHk0ljaAg7yDYEKdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFajtbfP&md5=ebb4c6df453ff1a37096ed155d26fae5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1080%2F17460441.2017.1377697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2017.1377697%26sid%3Dliteratum%253Aachs%26aulast%3DBravo%26aufirst%3DL.%26aulast%3DMico%26aufirst%3DJ.%2BA.%26aulast%3DBerrocoso%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520development%2520of%2520tramadol%2520for%2520the%2520treatment%2520of%2520pain%26jtitle%3DExpert.%2520Opin.%2520Drug%2520Discov.%26date%3D2017%26volume%3D12%26spage%3D1281%26epage%3D1291%26doi%3D10.1080%2F17460441.2017.1377697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Kuhrt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoernig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanrikulu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinhilber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Identification of natural-product-derived inhibitors of 5-lipoxygenase activity by ligand-based virtual screening</span>. <i>J. Med. Pharma. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">2640</span>– <span class="NLM_lpage">2646</span>, <span class="refDoi"> DOI: 10.1021/jm060655w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060655w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2640-2646&author=L.+Frankeauthor=O.+Schwarzauthor=L.+Muller-Kuhrtauthor=C.+Hoernigauthor=L.+Fischerauthor=S.+Georgeauthor=Y.+Tanrikuluauthor=P.+Schneiderauthor=O.+Werzauthor=D.+Steinhilberauthor=G.+Schneider&title=Identification+of+natural-product-derived+inhibitors+of+5-lipoxygenase+activity+by+ligand-based+virtual+screening&doi=10.1021%2Fjm060655w"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm060655w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060655w%26sid%3Dliteratum%253Aachs%26aulast%3DFranke%26aufirst%3DL.%26aulast%3DSchwarz%26aufirst%3DO.%26aulast%3DMuller-Kuhrt%26aufirst%3DL.%26aulast%3DHoernig%26aufirst%3DC.%26aulast%3DFischer%26aufirst%3DL.%26aulast%3DGeorge%26aufirst%3DS.%26aulast%3DTanrikulu%26aufirst%3DY.%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DWerz%26aufirst%3DO.%26aulast%3DSteinhilber%26aufirst%3DD.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DIdentification%2520of%2520natural-product-derived%2520inhibitors%2520of%25205-lipoxygenase%2520activity%2520by%2520ligand-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Pharma.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2640%26epage%3D2646%26doi%3D10.1021%2Fjm060655w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Scaffold Hopping</span>. <i>J. Med. Pharma. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1238</span>– <span class="NLM_lpage">1246</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01437</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01437" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1238-1246&author=Y.+Huauthor=D.+Stumpfeauthor=J.+Bajorath&title=Recent+Advances+in+Scaffold+Hopping&doi=10.1021%2Facs.jmedchem.6b01437"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01437%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DRecent%2520Advances%2520in%2520Scaffold%2520Hopping%26jtitle%3DJ.%2520Med.%2520Pharma.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1238%26epage%3D1246%26doi%3D10.1021%2Facs.jmedchem.6b01437" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baringhaus, K.-H.</span></span> <span> </span><span class="NLM_article-title">The scaffold hopping potential of pharmacophores</span>. <i>Drug Discovery Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e263</span>– <span class="NLM_lpage">e269</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1016%2Fj.ddtec.2010.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFyjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=e263-e269&author=G.+Hesslerauthor=K.-H.+Baringhaus&title=The+scaffold+hopping+potential+of+pharmacophores"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The scaffold hopping potential of pharmacophores</span></div><div class="casAuthors">Hessler, Gerhard; Baringhaus, Karl-Heinz</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">E263-E269</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The goal of scaffold hopping is to replace the chem. core structure by a novel chem. motif while keeping the biol. activity of the mol.  As pharmacophores define chem. features essential for biol. activity, they can be successfully employed to guide scaffold replacements.  To this end, various novel approaches have recently been developed and applied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1Rwswgk1y1bVg90H21EOLACvtfcHk0lj-UpLPzW79ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFyjsL4%253D&md5=2621bdad53855b24427db53dd368a127</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2010.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2010.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DHessler%26aufirst%3DG.%26aulast%3DBaringhaus%26aufirst%3DK.-H.%26atitle%3DThe%2520scaffold%2520hopping%2520potential%2520of%2520pharmacophores%26jtitle%3DDrug%2520Discovery%2520Today%2520Technol.%26date%3D2010%26volume%3D7%26spage%3De263%26epage%3De269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yue, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Identification of novel small-molecule inhibitors targeting menin-MLL interaction, repurposing the antidiarrheal loperamide</span>. <i>Perkin 2</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">8503</span>– <span class="NLM_lpage">8519</span>, <span class="refDoi"> DOI: 10.1039/c6ob01248e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1039%2Fc6ob01248e" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=8503-8519&author=L.+Yueauthor=J.+Duauthor=F.+Yeauthor=Z.+Chenauthor=L.+Liauthor=F.+Lianauthor=B.+Zhangauthor=Y.+Zhangauthor=H.+Jiangauthor=K.+Chenauthor=Y.+Liauthor=B.+Zhouauthor=N.+Zhangauthor=Y.+Yangauthor=C.+Luo&title=Identification+of+novel+small-molecule+inhibitors+targeting+menin-MLL+interaction%2C+repurposing+the+antidiarrheal+loperamide&doi=10.1039%2Fc6ob01248e"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2Fc6ob01248e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc6ob01248e%26sid%3Dliteratum%253Aachs%26aulast%3DYue%26aufirst%3DL.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520novel%2520small-molecule%2520inhibitors%2520targeting%2520menin-MLL%2520interaction%252C%2520repurposing%2520the%2520antidiarrheal%2520loperamide%26jtitle%3DPerkin%25202%26date%3D2016%26volume%3D14%26spage%3D8503%26epage%3D8519%26doi%3D10.1039%2Fc6ob01248e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span>; <span class="NLM_string-name">Lu, W. C.</span>; <span class="NLM_string-name">Hu, J. C.</span>; <span class="NLM_string-name">Liu, Y. C.</span>; <span class="NLM_string-name">Zhang, C. H.</span>; <span class="NLM_string-name">Lian, F. L.</span>; <span class="NLM_string-name">Zhang, N. X.</span>; <span class="NLM_string-name">Meng, F. W.</span>; <span class="NLM_string-name">Luo, C.</span>; <span class="NLM_string-name">Chen, K. X.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterizations of Novel, Selective Histone Methyltransferase SET7 Inhibitors by Scaffold Hopping- and 2D-Molecular Fingerprint-Based Similarity Search</span>.  <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume">23</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.3390%2Fmolecules23030567" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&author=H.+Ding&author=W.+C.+Lu&author=J.+C.+Hu&author=Y.+C.+Liu&author=C.+H.+Zhang&author=F.+L.+Lian&author=N.+X.+Zhang&author=F.+W.+Meng&author=C.+Luo&author=K.+X.+Chen&title=Identification+and+Characterizations+of+Novel%2C+Selective+Histone+Methyltransferase+SET7+Inhibitors+by+Scaffold+Hopping-+and+2D-Molecular+Fingerprint-Based+Similarity+Search"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23030567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23030567%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DH.%26atitle%3DIdentification%2520and%2520Characterizations%2520of%2520Novel%252C%2520Selective%2520Histone%2520Methyltransferase%2520SET7%2520Inhibitors%2520by%2520Scaffold%2520Hopping-%2520and%25202D-Molecular%2520Fingerprint-Based%2520Similarity%2520Search%26jtitle%3DMolecules%26date%3D2018%26volume%3D23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totrov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span> <span> </span><span class="NLM_article-title">Docking to multiple pockets or ligand fields for screening, activity prediction and scaffold hopping</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1741</span>– <span class="NLM_lpage">1755</span>, <span class="refDoi"> DOI: 10.4155/fmc.14.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.4155%2Ffmc.14.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=25407367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFCit7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1741-1755&author=Y.+C.+Chenauthor=M.+Totrovauthor=R.+Abagyan&title=Docking+to+multiple+pockets+or+ligand+fields+for+screening%2C+activity+prediction+and+scaffold+hopping&doi=10.4155%2Ffmc.14.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Docking to multiple pockets or ligand fields for screening, activity prediction and scaffold hopping</span></div><div class="casAuthors">Chen, Yu-Chen; Totrov, Max; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1741-1755</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Background: Two recent technol. advances dramatically reducing the rate of false-negatives in activity prediction by docking flexible 3D models of compds. include multi-conformational docking (mPockDock) and the docking of candidates to at. property fields derived by co-crystd. ligands (mApfDock).  Results: The mApfDock and mPockDock provide the AUC of 90.4 and 83.8%, resp.  The mApfDock gave better performance when compds. required large induced-fit pocket changes unseen in crystallog., whereas the mPockDock is superior when the co-crystd. ligands do not represent sufficient chem. and binding location diversity.  Conclusion: Both approaches proved to be efficient for scaffold hopping; they are complementary when the coverage of the co-crystd. complexes is poor but become convergent when the complexes are diverse enough.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobPjvRDSmjvrVg90H21EOLACvtfcHk0liMaW0U4XnmJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFCit7vE&md5=4e9d30a6ad9f1c68b05ac617e8463821</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.113%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%2BC.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DDocking%2520to%2520multiple%2520pockets%2520or%2520ligand%2520fields%2520for%2520screening%252C%2520activity%2520prediction%2520and%2520scaffold%2520hopping%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D1741%26epage%3D1755%26doi%3D10.4155%2Ffmc.14.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilton, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acklin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azzaoui, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span> <span> </span><span class="NLM_article-title">New methods for ligand-based virtual screening: use of data fusion and machine learning to enhance the effectiveness of similarity searching</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">470</span>, <span class="refDoi"> DOI: 10.1021/ci050348j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050348j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvV2h" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=462-470&author=J.+Hertauthor=P.+Willettauthor=D.+J.+Wiltonauthor=P.+Acklinauthor=K.+Azzaouiauthor=E.+Jacobyauthor=A.+Schuffenhauer&title=New+methods+for+ligand-based+virtual+screening%3A+use+of+data+fusion+and+machine+learning+to+enhance+the+effectiveness+of+similarity+searching&doi=10.1021%2Fci050348j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">New Methods for Ligand-Based Virtual Screening: Use of Data Fusion and Machine Learning to Enhance the Effectiveness of Similarity Searching</span></div><div class="casAuthors">Hert, Jerome; Willett, Peter; Wilton, David J.; Acklin, Pierre; Azzaoui, Kamal; Jacoby, Edgar; Schuffenhauer, Ansgar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-470</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Similarity searching using a single bioactive ref. structure is a well-established technique for accessing chem. structure databases.  This paper describes two extensions of the basic approach.  First, we discuss the use of group fusion to combine the results of similarity searches when multiple ref. structures are available.  We demonstrate that this technique is notably more effective than conventional similarity searching in scaffold-hopping searches for structurally diverse sets of active mols.; conversely, the technique will do little to improve the search performance if the actives are structurally homogeneous.  Second, we make the assumption that the nearest neighbors resulting from a similarity search, using a single bioactive ref. structure, are also active and use this assumption to implement approx. forms of group fusion, substructural anal., and binary kernel discrimination.  This approach, called turbo similarity searching, is notably more effective than conventional similarity searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9JTJUYOKnZrVg90H21EOLACvtfcHk0liMaW0U4XnmJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvV2h&md5=cc24b6dff94f70a6e5ece86f8b14b673</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci050348j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050348j%26sid%3Dliteratum%253Aachs%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DWilton%26aufirst%3DD.%2BJ.%26aulast%3DAcklin%26aufirst%3DP.%26aulast%3DAzzaoui%26aufirst%3DK.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DSchuffenhauer%26aufirst%3DA.%26atitle%3DNew%2520methods%2520for%2520ligand-based%2520virtual%2520screening%253A%2520use%2520of%2520data%2520fusion%2520and%2520machine%2520learning%2520to%2520enhance%2520the%2520effectiveness%2520of%2520similarity%2520searching%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2006%26volume%3D46%26spage%3D462%26epage%3D470%26doi%3D10.1021%2Fci050348j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muegge, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, P.</span></span> <span> </span><span class="NLM_article-title">An overview of molecular fingerprint similarity search in virtual screening</span>. <i>Expert. Opin. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">148</span>, <span class="refDoi"> DOI: 10.1517/17460441.2016.1117070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1517%2F17460441.2016.1117070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=26558489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=137-148&author=I.+Mueggeauthor=P.+Mukherjee&title=An+overview+of+molecular+fingerprint+similarity+search+in+virtual+screening&doi=10.1517%2F17460441.2016.1117070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">An overview of molecular fingerprint similarity search in virtual screening</span></div><div class="casAuthors">Muegge, Ingo; Mukherjee, Prasenjit</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">137-148</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A central premise of medicinal chem. is that structurally similar mols. exhibit similar biol. activities.  Mol. fingerprints encode properties of small mols. and assess their similarities computationally through bit string comparisons.  Based on the similarity to a biol. active template, mol. fingerprint methods allow for identifying addnl. compds. with a higher chance of displaying similar biol. activities against the same target - a process commonly referred to as virtual screening (VS).  This article focuses on fingerprint similarity searches in the context of compd. selection for enhancing hit sets, comparing compd. decks, and VS.  In addn., the authors discuss the application of fingerprints in predictive modeling.  Fingerprint similarity search methods are esp. useful in VS if only a few unrelated ligands are known for a given target and therefore more complex and information rich methods such as pharmacophore searches or structure-based design are not applicable.  In addn., fingerprint methods are used in characterizing properties of compd. collections such as chem. diversity, d. in chem. space, and content of biol. active mols. (biodiversity).  Such assessments are important for deciding what compds. to exptl. screen, to purchase, or to assemble in a virtual compd. deck for in silico screening or de novo design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCfZIE_oEtR7Vg90H21EOLACvtfcHk0lgQRhH9j6nupw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtLbF&md5=de18b3dbbf5ad1974ca1150193ee77aa</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1517%2F17460441.2016.1117070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2016.1117070%26sid%3Dliteratum%253Aachs%26aulast%3DMuegge%26aufirst%3DI.%26aulast%3DMukherjee%26aufirst%3DP.%26atitle%3DAn%2520overview%2520of%2520molecular%2520fingerprint%2520similarity%2520search%2520in%2520virtual%2520screening%26jtitle%3DExpert.%2520Opin.%2520Drug%2520Discov.%26date%3D2016%26volume%3D11%26spage%3D137%26epage%3D148%26doi%3D10.1517%2F17460441.2016.1117070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Molecular similarity in medicinal chemistry</span>. <i>J. Med. Pharma. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">3186</span>– <span class="NLM_lpage">3204</span>, <span class="refDoi"> DOI: 10.1021/jm401411z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401411z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=3186-3204&author=G.+Maggioraauthor=M.+Vogtauthor=D.+Stumpfeauthor=J.+Bajorath&title=Molecular+similarity+in+medicinal+chemistry&doi=10.1021%2Fjm401411z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm401411z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401411z%26sid%3Dliteratum%253Aachs%26aulast%3DMaggiora%26aufirst%3DG.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DMolecular%2520similarity%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Med.%2520Pharma.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D3186%26epage%3D3204%26doi%3D10.1021%2Fjm401411z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tetko, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filimonov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poroikov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varnek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roitberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isayev, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtarolo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourches, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspuru-Guzik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrafiotis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherkasov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">QSAR without borders</span>. <i>RIC Rev.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">3525</span>– <span class="NLM_lpage">3564</span>, <span class="refDoi"> DOI: 10.1039/d0cs00098a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1039%2Fd0cs00098a" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2020&pages=3525-3564&author=E.+N.+Muratovauthor=J.+Bajorathauthor=R.+P.+Sheridanauthor=I.+V.+Tetkoauthor=D.+Filimonovauthor=V.+Poroikovauthor=T.+I.+Opreaauthor=I.+I.+Baskinauthor=A.+Varnekauthor=A.+Roitbergauthor=O.+Isayevauthor=S.+Curtaroloauthor=D.+Fourchesauthor=Y.+Cohenauthor=A.+Aspuru-Guzikauthor=D.+A.+Winklerauthor=D.+Agrafiotisauthor=A.+Cherkasovauthor=A.+Tropsha&title=QSAR+without+borders&doi=10.1039%2Fd0cs00098a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2Fd0cs00098a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fd0cs00098a%26sid%3Dliteratum%253Aachs%26aulast%3DMuratov%26aufirst%3DE.%2BN.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26aulast%3DFilimonov%26aufirst%3DD.%26aulast%3DPoroikov%26aufirst%3DV.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DBaskin%26aufirst%3DI.%2BI.%26aulast%3DVarnek%26aufirst%3DA.%26aulast%3DRoitberg%26aufirst%3DA.%26aulast%3DIsayev%26aufirst%3DO.%26aulast%3DCurtarolo%26aufirst%3DS.%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DCohen%26aufirst%3DY.%26aulast%3DAspuru-Guzik%26aufirst%3DA.%26aulast%3DWinkler%26aufirst%3DD.%2BA.%26aulast%3DAgrafiotis%26aufirst%3DD.%26aulast%3DCherkasov%26aufirst%3DA.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DQSAR%2520without%2520borders%26jtitle%3DRIC%2520Rev.%26date%3D2020%26volume%3D49%26spage%3D3525%26epage%3D3564%26doi%3D10.1039%2Fd0cs00098a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camargo, L. M.</span></span> <span> </span><span class="NLM_article-title">The opportunities of mining historical and collective data in drug discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">434</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1016%2Fj.drudis.2014.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=25463034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A280%3ADC%252BC2MzjsF2mug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=422-434&author=A.+M.+Wassermannauthor=E.+Lounkineauthor=J.+W.+Daviesauthor=M.+Glickauthor=L.+M.+Camargo&title=The+opportunities+of+mining+historical+and+collective+data+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The opportunities of mining historical and collective data in drug discovery</span></div><div class="casAuthors">Wassermann Anne Mai; Camargo L Miguel; Lounkine Eugen; Davies John W; Glick Meir</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">422-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Vast amounts of bioactivity data have been generated for small molecules across public and corporate domains.  Biological signatures, either derived from systematic profiling efforts or from existing historical assay data, have been successfully employed for small molecule mechanism-of-action elucidation, drug repositioning, hit expansion and screening subset design.  This article reviews different types of biological descriptors and applications, and we demonstrate how biological data can outlive the original purpose or project for which it was generated.  By comparing 150 HTS campaigns run at Novartis over the past decade on the basis of their active and inactive chemical matter, we highlight the opportunities and challenges associated with cross-project learning in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR2wZT2paLsIg20zIbnCCI3fW6udTcc2eYTN8d--c5gQbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzjsF2mug%253D%253D&md5=44e61022e1b87335e0f43101e086cb4b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DCamargo%26aufirst%3DL.%2BM.%26atitle%3DThe%2520opportunities%2520of%2520mining%2520historical%2520and%2520collective%2520data%2520in%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D422%26epage%3D434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fliri, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loging, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thadeio, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmann, R. A.</span></span> <span> </span><span class="NLM_article-title">Biological spectra analysis: Linking biological activity profiles to molecular structure</span>. <i>Proc. Nat. Acad ScI. U S A</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1073/pnas.0407790101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1073%2Fpnas.0407790101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=15625110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD2MXptFSmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=261-266&author=A.+F.+Fliriauthor=W.+T.+Logingauthor=P.+F.+Thadeioauthor=R.+A.+Volkmann&title=Biological+spectra+analysis%3A+Linking+biological+activity+profiles+to+molecular+structure&doi=10.1073%2Fpnas.0407790101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Biological spectra analysis: Linking biological activity profiles to molecular structure</span></div><div class="casAuthors">Fliri, Anton F.; Loging, William T.; Thadeio, Peter F.; Volkmann, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-266</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Establishing quant. relationships between mol. structure and broad biol. effects has been a longstanding challenge in science.  Currently, no method exists for forecasting broad biol. activity profiles of medicinal agents even within narrow boundaries of structurally similar mols.  Starting from the premise that biol. activity results from the capacity of small org. mols. to modulate the activity of the proteome, we set out to investigate whether descriptor sets could be developed for measuring and quantifying this mol. property.  Using a 1,567-compd. database, we show that percent inhibition values, detd. at single high drug concn. in a battery of in vitro assays representing a cross section of the proteome, provide precise mol. property descriptors that identify the structure of mols.  When broad biol. activity of mols. is represented in spectra form, org. mols. can be sorted by quantifying differences between biol. spectra.  Unlike traditional structure-activity relationship methods, sorting of mols. by using biospectra comparisons does not require knowledge of a mol.'s putative drug targets.  To illustrate this finding, we selected as starting point the biol. activity spectra of clotrimazole and tioconazole because their putative target, lanosterol demethylase (CYP51), was not included in the bioassay array.  Spectra similarity obtained through profile similarity measurements and hierarchical clustering provided an unbiased means for establishing quant. relationships between chem. structures and biol. activity spectra.  This methodol., which we have termed biol. spectra anal., provides the capability not only of sorting mols. on the basis of biospectra similarity but also of predicting simultaneous interactions of new mols. with multiple proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocRkPiQrArMLVg90H21EOLACvtfcHk0lgDuBdBF8J70w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXptFSmsQ%253D%253D&md5=ff5faffa569df4c61389d6baef5800c3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0407790101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0407790101%26sid%3Dliteratum%253Aachs%26aulast%3DFliri%26aufirst%3DA.%2BF.%26aulast%3DLoging%26aufirst%3DW.%2BT.%26aulast%3DThadeio%26aufirst%3DP.%2BF.%26aulast%3DVolkmann%26aufirst%3DR.%2BA.%26atitle%3DBiological%2520spectra%2520analysis%253A%2520Linking%2520biological%2520activity%2520profiles%2520to%2520molecular%2520structure%26jtitle%3DProc.%2520Nat.%2520Acad%2520ScI.%2520U%2520S%2520A%26date%3D2005%26volume%3D102%26spage%3D261%26epage%3D266%26doi%3D10.1073%2Fpnas.0407790101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fliri, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loging, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thadeio, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmann, R. A.</span></span> <span> </span><span class="NLM_article-title">Biospectra analysis: model proteome characterizations for linking molecular structure and biological response</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">6918</span>– <span class="NLM_lpage">6925</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050494g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVGisrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6918-6925&author=A.+F.+Fliriauthor=W.+T.+Logingauthor=P.+F.+Thadeioauthor=R.+A.+Volkmann&title=Biospectra+analysis%3A+model+proteome+characterizations+for+linking+molecular+structure+and+biological+response"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Biospectra Analysis: Model Proteome Characterizations for Linking Molecular Structure and Biological Response</span></div><div class="casAuthors">Fliri, Anton F.; Loging, William T.; Thadeio, Peter F.; Volkmann, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6918-6925</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Establishing quant. relationships between mol. structure and broad biol. effects has been a long-standing goal in drug discovery.  Evaluation of the capacity of mols. to modulate protein functions is a prerequisite for understanding the relationship between mol. structure and in vivo biol. response.  A particular challenge in these investigations is to derive quant. measurements of a mol.'s functional activity pattern across different proteins.  Herein we describe an operationally simple probabilistic structure-activity relationship (SAR) approach, termed biospectra anal., for identifying agonist and antagonist effect profiles of medicinal agents by using pattern similarity between biol. activity spectra (biospectra) of mols. as the determinant.  Accordingly, in vitro binding data (percent inhibition values of mols. detd. at single high drug concn. in a battery of assays representing a cross section of the proteome) are useful for identifying functional effect profile similarity between medicinal agents.  To illustrate this finding, the relationship between biospectra similarity of 24 mols., identified by hierarchical clustering of a 1567 mol. dataset as being most closely aligned with the neurotransmitter dopamine, and their agonist or antagonist properties was probed.  Distinguishing the results described in this study from those obtained with affinity-based methods, the obsd. assocn. between biospectra and biol. response profile similarity remains intact even upon removal of putative drug targets from the dataset (four dopaminergic [D1/D2/D3/D4] and two adrenergic [α1 and α2] receptors).  These findings indicate that biospectra anal. provides an unbiased new tool for forecasting structure-response relationships and for translating broad biol. effect information into chem. structure design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJa5_zvZqHGLVg90H21EOLACvtfcHk0lgDuBdBF8J70w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVGisrjE&md5=e899a8db1f2d66dcb7b7c5d656816111</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm050494g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050494g%26sid%3Dliteratum%253Aachs%26aulast%3DFliri%26aufirst%3DA.%2BF.%26aulast%3DLoging%26aufirst%3DW.%2BT.%26aulast%3DThadeio%26aufirst%3DP.%2BF.%26aulast%3DVolkmann%26aufirst%3DR.%2BA.%26atitle%3DBiospectra%2520analysis%253A%2520model%2520proteome%2520characterizations%2520for%2520linking%2520molecular%2520structure%2520and%2520biological%2520response%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6918%26epage%3D6925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campillos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bork, P.</span></span> <span> </span><span class="NLM_article-title">Drug target identification using side-effect similarity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1126/science.1158140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1126%2Fscience.1158140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=18621671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD1cXot1entbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2008&pages=263-266&author=M.+Campillosauthor=M.+Kuhnauthor=A.+C.+Gavinauthor=L.+J.+Jensenauthor=P.+Bork&title=Drug+target+identification+using+side-effect+similarity&doi=10.1126%2Fscience.1158140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Target Identification Using Side-Effect Similarity</span></div><div class="casAuthors">Campillos, Monica; Kuhn, Michael; Gavin, Anne-Claude; Jensen, Lars Juhl; Bork, Peer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">5886</span>),
    <span class="NLM_cas:pages">263-266</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Targets for drugs have so far been predicted on the basis of mol. or cellular features, for example, by exploiting similarity in chem. structure or in activity across cell lines.  We used phenotypic side-effect similarities to infer whether two drugs share a target.  Applied to 746 marketed drugs, a network of 1018 side effect-driven drug-drug relations became apparent, 261 of which are formed by chem. dissimilar drugs from different therapeutic indications.  We exptl. tested 20 of these unexpected drug-drug relations and validated 13 implied drug-target relations by in vitro binding assays, of which 11 reveal inhibition consts. equal to less than 10 micromolar.  Nine of these were tested and confirmed in cell assays, documenting the feasibility of using phenotypic information to infer mol. interactions and hinting at new uses of marketed drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbiFcs_camL7Vg90H21EOLACvtfcHk0liHAR6CvNCr3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXot1entbY%253D&md5=d8fe131adeb39750cb97ee4e147de355</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fscience.1158140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1158140%26sid%3Dliteratum%253Aachs%26aulast%3DCampillos%26aufirst%3DM.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DGavin%26aufirst%3DA.%2BC.%26aulast%3DJensen%26aufirst%3DL.%2BJ.%26aulast%3DBork%26aufirst%3DP.%26atitle%3DDrug%2520target%2520identification%2520using%2520side-effect%2520similarity%26jtitle%3DScience%26date%3D2008%26volume%3D321%26spage%3D263%26epage%3D266%26doi%3D10.1126%2Fscience.1158140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blat, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrobel, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunet, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hieronymus, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>313</i></span>,  <span class="NLM_fpage">1929</span>– <span class="NLM_lpage">1935</span>, <span class="refDoi"> DOI: 10.1126/science.1132939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1126%2Fscience.1132939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=17008526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVSnsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&author=J.+Lambauthor=E.+D.+Crawfordauthor=D.+Peckauthor=J.+W.+Modellauthor=I.+C.+Blatauthor=M.+J.+Wrobelauthor=J.+Lernerauthor=J.+P.+Brunetauthor=A.+Subramanianauthor=K.+N.+Rossauthor=M.+Reichauthor=H.+Hieronymusauthor=G.+Weiauthor=S.+A.+Armstrongauthor=S.+J.+Haggartyauthor=P.+A.+Clemonsauthor=R.+Weiauthor=S.+A.+Carrauthor=E.+S.+Landerauthor=T.+R.+Golub&title=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease&doi=10.1126%2Fscience.1132939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The connectivity map: Using gene-expression signatures to connect small molecules, genes, and disease</span></div><div class="casAuthors">Lamb, Justin; Crawford, Emily D.; Peck, David; Modell, Joshua W.; Blat, Irene C.; Wrobel, Matthew J.; Lerner, Jim; Brunet, Jean-Philippe; Subramanian, Aravind; Ross, Kenneth N.; Reich, Michael; Hieronymus, Haley; Wei, Guo; Armstrong, Scott A.; Haggarty, Stephen J.; Clemons, Paul A.; Wei, Ru; Carr, Steven A.; Lander, Eric S.; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">313</span>
        (<span class="NLM_cas:issue">5795</span>),
    <span class="NLM_cas:pages">1929-1935</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">To pursue a systematic approach to the discovery of functional connections among diseases, genetic perturbation, and drug action, we have created the first installment of a ref. collection of gene-expression profiles from cultured human cells treated with bioactive small mols., together with pattern-matching software to mine these data.  We demonstrate that this "Connectivity Map" resource can be used to find connections among small mols. sharing a mechanism of action, chems. and physiol. processes, and diseases and drugs.  These results indicate the feasibility of the approach and suggest the value of a large-scale community Connectivity Map project.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtu0M2BUO2w7Vg90H21EOLACvtfcHk0liHAR6CvNCr3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVSnsrbN&md5=1fad887ad515ffdd8064e526ff25a4db</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1126%2Fscience.1132939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1132939%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DCrawford%26aufirst%3DE.%2BD.%26aulast%3DPeck%26aufirst%3DD.%26aulast%3DModell%26aufirst%3DJ.%2BW.%26aulast%3DBlat%26aufirst%3DI.%2BC.%26aulast%3DWrobel%26aufirst%3DM.%2BJ.%26aulast%3DLerner%26aufirst%3DJ.%26aulast%3DBrunet%26aufirst%3DJ.%2BP.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DHieronymus%26aufirst%3DH.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DWei%26aufirst%3DR.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DThe%2520Connectivity%2520Map%253A%2520using%2520gene-expression%2520signatures%2520to%2520connect%2520small%2520molecules%252C%2520genes%252C%2520and%2520disease%26jtitle%3DScience%26date%3D2006%26volume%3D313%26spage%3D1929%26epage%3D1935%26doi%3D10.1126%2Fscience.1132939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wosikowski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuurhuis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paull, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, S. E.</span></span> <span> </span><span class="NLM_article-title">Identification of epidermal growth factor receptor and c-erbB2 pathway inhibitors by correlation with gene expression patterns</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">1505</span>– <span class="NLM_lpage">1515</span>, <span class="refDoi"> DOI: 10.1093/jnci/89.20.1505</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1093%2Fjnci%2F89.20.1505" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=9337347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADyaK2sXntVGgsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1997&pages=1505-1515&author=K.+Wosikowskiauthor=D.+Schuurhuisauthor=K.+Johnsonauthor=K.+D.+Paullauthor=T.+G.+Myersauthor=J.+N.+Weinsteinauthor=S.+E.+Bates&title=Identification+of+epidermal+growth+factor+receptor+and+c-erbB2+pathway+inhibitors+by+correlation+with+gene+expression+patterns&doi=10.1093%2Fjnci%2F89.20.1505"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of epidermal growth factor receptor and c-erbB2 pathway inhibitors by correlation with gene expression patterns</span></div><div class="casAuthors">Wosikowski, Katja; Schuurhuis, Danita; Johnson, Kathryn; Paull, Kenneth D.; Myers, Timothy G.; Weinstein, John N.; Bates, Susan E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">1505-1515</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Growth factor receptor-signaling pathways are potentially important targets for anticancer therapy.  The interaction of anticancer agents with specific mol. targets can be identified by correlating target expression patterns with cytotoxicity patterns.  The authors sought to identify new agents that target and inhibit the activity of the epidermal growth factor (EGF) receptor and of c-erbB2 (also called HER2 or neu), by correlating EGF receptor, transforming growth factor (TGF)-α (a ligand for EGF receptor), and c-erbB2 mRNA expression levels with the results of cytotoxicity assays of the 49 000 compds. in the National Cancer Institute (NCI) drug screen database.  The levels of mRNAs were measured and used to generate a mol. target database for the 60 cell lines of the NCI anticancer drug screen.  The computer anal. program, COMPARE, was used to search for cytotoxicity patterns in the NCI drug screen database that were highly correlated with EGF receptor, TGF-α, or c-erbB2 mRNA expression patterns.  The putative EGF receptor-inhibiting compds. were tested for effects on basal tyrosine phosphorylation, in vitro EGF receptor tyrosine kinase activity, and EGF-dependent growth.  Putative ErbB2-inhibiting compds. were tested for effects on antibody-induced ErbB2 tyrosine kinase activity.  EGF receptor mRNA and TGF-α mRNA levels were highest in cell lines derived from renal cancers, and c-erbB2 mRNA levels were highest in cells derived from breast, ovarian, and colon cancers.  Twenty-five compds. with high correlation coeffs. (for cytotoxicity and levels of the measured mRNAs) were tested as inhibitors of the EGF receptor or c-erbB2 signaling pathways; 14 compds. were identified as inhibitors of these pathways.  The most potent compd., B4, inhibited autophosphorylation (which occurs following activation) of ErbB2 by 50% in whole cells at 7.7 μM.  Novel EGF receptor or c-erbB2 pathway inhibitors can be identified in the NCI drug screen by correlation of cytotoxicity patterns with EGF receptor or c-erbB2 mRNA expression levels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokhKMwhdDk8bVg90H21EOLACvtfcHk0lhFz77ZV8hOOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXntVGgsLk%253D&md5=2b98161f612d31a02522fc4c5bcbe561</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F89.20.1505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F89.20.1505%26sid%3Dliteratum%253Aachs%26aulast%3DWosikowski%26aufirst%3DK.%26aulast%3DSchuurhuis%26aufirst%3DD.%26aulast%3DJohnson%26aufirst%3DK.%26aulast%3DPaull%26aufirst%3DK.%2BD.%26aulast%3DMyers%26aufirst%3DT.%2BG.%26aulast%3DWeinstein%26aufirst%3DJ.%2BN.%26aulast%3DBates%26aufirst%3DS.%2BE.%26atitle%3DIdentification%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520and%2520c-erbB2%2520pathway%2520inhibitors%2520by%2520correlation%2520with%2520gene%2520expression%2520patterns%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D1997%26volume%3D89%26spage%3D1505%26epage%3D1515%26doi%3D10.1093%2Fjnci%2F89.20.1505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hassane, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbett, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abboud, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liesveld, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, C. T.</span></span> <span> </span><span class="NLM_article-title">Discovery of agents that eradicate leukemia stem cells using an in silico screen of public gene expression data</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">5654</span>– <span class="NLM_lpage">5662</span>, <span class="refDoi"> DOI: 10.1182/blood-2007-11-126003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1182%2Fblood-2007-11-126003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=18305216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsVOktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2008&pages=5654-5662&author=D.+C.+Hassaneauthor=M.+L.+Guzmanauthor=C.+Corbettauthor=X.+Liauthor=R.+Abboudauthor=F.+Youngauthor=J.+L.+Liesveldauthor=M.+Carrollauthor=C.+T.+Jordan&title=Discovery+of+agents+that+eradicate+leukemia+stem+cells+using+an+in+silico+screen+of+public+gene+expression+data&doi=10.1182%2Fblood-2007-11-126003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of agents that eradicate leukemia stem cells using an in silico screen of public gene expression data</span></div><div class="casAuthors">Hassane, Duane C.; Guzman, Monica L.; Corbett, Cheryl; Li, Xiaojie; Abboud, Ramzi; Young, Fay; Liesveld, Jane L.; Carroll, Martin; Jordan, Craig T.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5654-5662</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Increasing evidence indicates that malignant stem cells are important for the pathogenesis of acute myelogenous leukemia (AML) and represent a reservoir of cells that drive the development of AML and relapse.  Therefore, new treatment regimens are necessary to prevent relapse and improve therapeutic outcomes.  Previous studies have shown that the sesquiterpene lactone, parthenolide (PTL), ablates bulk, progenitor, and stem AML cells while causing no appreciable toxicity to normal hematopoietic cells.  Thus, PTL must evoke cellular responses capable of mediating AML selective cell death.  Given recent advances in chem. genomics such as gene expression-based high-throughput screening (GE-HTS) and the Connectivity Map, we hypothesized that the gene expression signature resulting from treatment of primary AML with PTL could be used to search for similar signatures in publicly available gene expression profiles deposited into the Gene Expression Omnibus (GEO).  We therefore devised a broad in silico screen of the GEO database using the PTL gene expression signature as a template and discovered 2 new agents, celastrol and 4-hydroxy-2-nonenal, that effectively eradicate AML at the bulk, progenitor, and stem cell level.  These findings suggest the use of multicenter collections of high-throughput data to facilitate discovery of leukemia drugs and drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi4ise6u_curVg90H21EOLACvtfcHk0lhFz77ZV8hOOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsVOktb8%253D&md5=ccd65463c8a0aba9b5f204970f5c267d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1182%2Fblood-2007-11-126003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2007-11-126003%26sid%3Dliteratum%253Aachs%26aulast%3DHassane%26aufirst%3DD.%2BC.%26aulast%3DGuzman%26aufirst%3DM.%2BL.%26aulast%3DCorbett%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DAbboud%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DF.%26aulast%3DLiesveld%26aufirst%3DJ.%2BL.%26aulast%3DCarroll%26aufirst%3DM.%26aulast%3DJordan%26aufirst%3DC.%2BT.%26atitle%3DDiscovery%2520of%2520agents%2520that%2520eradicate%2520leukemia%2520stem%2520cells%2520using%2520an%2520in%2520silico%2520screen%2520of%2520public%2520gene%2520expression%2520data%26jtitle%3DBlood%26date%3D2008%26volume%3D111%26spage%3D5654%26epage%3D5662%26doi%3D10.1182%2Fblood-2007-11-126003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span>; <span class="NLM_string-name">Xu, M.</span>; <span class="NLM_string-name">Zhu, H.</span>; <span class="NLM_string-name">Chen, C. Z.</span>; <span class="NLM_string-name">Zhu, W.</span>; <span class="NLM_string-name">Lee, E. M.</span>; <span class="NLM_string-name">He, S.</span>; <span class="NLM_string-name">Zhang, L.</span>; <span class="NLM_string-name">Zhao, J.</span>; <span class="NLM_string-name">Shamim, K.</span>; <span class="NLM_string-name">Bougie, D.</span>; <span class="NLM_string-name">Huang, W.</span>; <span class="NLM_string-name">Xia, M.</span>; <span class="NLM_string-name">Hall, M. D.</span>; <span class="NLM_string-name">Lo, D.</span>; <span class="NLM_string-name">Simeonov, A.</span>; <span class="NLM_string-name">Austin, C. P.</span>; <span class="NLM_string-name">Qiu, X.</span>; <span class="NLM_string-name">Tang, H.</span>; <span class="NLM_string-name">Zheng, W.</span></span> <span> </span><span class="NLM_article-title">Biological activity-based modeling identifies antiviral leads against SARS-CoV-2</span>.  <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1038%2Fs41587-021-00839-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&author=R.+Huang&author=M.+Xu&author=H.+Zhu&author=C.+Z.+Chen&author=W.+Zhu&author=E.+M.+Lee&author=S.+He&author=L.+Zhang&author=J.+Zhao&author=K.+Shamim&author=D.+Bougie&author=W.+Huang&author=M.+Xia&author=M.+D.+Hall&author=D.+Lo&author=A.+Simeonov&author=C.+P.+Austin&author=X.+Qiu&author=H.+Tang&author=W.+Zheng&title=Biological+activity-based+modeling+identifies+antiviral+leads+against+SARS-CoV-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fs41587-021-00839-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41587-021-00839-1%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DBiological%2520activity-based%2520modeling%2520identifies%2520antiviral%2520leads%2520against%2520SARS-CoV-2%26jtitle%3DNat.%2520Biotechnol.%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sipes, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kothiya, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reif, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Judson, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houck, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dix, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavlock, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, T. B.</span></span> <span> </span><span class="NLM_article-title">Profiling 976 ToxCast chemicals across 331 enzymatic and receptor signaling assays</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">878</span>– <span class="NLM_lpage">895</span>, <span class="refDoi"> DOI: 10.1021/tx400021f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx400021f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtlWhu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=878-895&author=N.+S.+Sipesauthor=M.+T.+Martinauthor=P.+Kothiyaauthor=D.+M.+Reifauthor=R.+S.+Judsonauthor=A.+M.+Richardauthor=K.+A.+Houckauthor=D.+J.+Dixauthor=R.+J.+Kavlockauthor=T.+B.+Knudsen&title=Profiling+976+ToxCast+chemicals+across+331+enzymatic+and+receptor+signaling+assays&doi=10.1021%2Ftx400021f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling 976 ToxCast Chemicals across 331 Enzymatic and Receptor Signaling Assays</span></div><div class="casAuthors">Sipes, Nisha S.; Martin, Matthew T.; Kothiya, Parth; Reif, David M.; Judson, Richard S.; Richard, Ann M.; Houck, Keith A.; Dix, David J.; Kavlock, Robert J.; Knudsen, Thomas B.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">878-895</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding potential health risks is a significant challenge due to the large nos. of diverse chems. with poorly characterized exposures and mechanisms of toxicities.  The present study analyzes 976 chems. (including failed pharmaceuticals, alternative plasticizers, food additives, and pesticides) in Phases I and II of the U.S. EPA's ToxCast project across 331 cell-free enzymic and ligand-binding high-throughput screening (HTS) assays.  Half-maximal activity concns. (AC50) were identified for 729 chems. in 256 assays (7135 chem.-assay pairs).  Some of the most commonly affected assays were CYPs (CYP2C9 and CYP2C19), transporters (mitochondrial TSPO, norepinephrine, and dopaminergic), and GPCRs (aminergic).  Heavy metals, surfactants, and dithiocarbamate fungicides showed promiscuous but distinctly different patterns of activity, whereas many of the pharmaceutical compds. showed promiscuous activity across GPCRs.  Literature anal. confirmed >50% of the activities for the most potent chem.-assay pairs (54) but also revealed 10 missed interactions.  Twenty-two chems. with known estrogenic activity were correctly identified for the majority (77%), missing only the weaker interactions.  In many cases, novel findings for previously unreported chem.-target combinations clustered with known chem.-target interactions.  Results from this large inventory of chem.-biol. interactions can inform read-across methods as well as link potential targets to mol. initiating events in adverse outcome pathways for diverse toxicities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1kBIOo_1MCbVg90H21EOLACvtfcHk0li-qPIeiXzxXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtlWhu7s%253D&md5=473da27aad42b986ef3dc0dde8acf14c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Ftx400021f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx400021f%26sid%3Dliteratum%253Aachs%26aulast%3DSipes%26aufirst%3DN.%2BS.%26aulast%3DMartin%26aufirst%3DM.%2BT.%26aulast%3DKothiya%26aufirst%3DP.%26aulast%3DReif%26aufirst%3DD.%2BM.%26aulast%3DJudson%26aufirst%3DR.%2BS.%26aulast%3DRichard%26aufirst%3DA.%2BM.%26aulast%3DHouck%26aufirst%3DK.%2BA.%26aulast%3DDix%26aufirst%3DD.%2BJ.%26aulast%3DKavlock%26aufirst%3DR.%2BJ.%26aulast%3DKnudsen%26aufirst%3DT.%2BB.%26atitle%3DProfiling%2520976%2520ToxCast%2520chemicals%2520across%2520331%2520enzymatic%2520and%2520receptor%2520signaling%2520assays%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2013%26volume%3D26%26spage%3D878%26epage%3D895%26doi%3D10.1021%2Ftx400021f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span>; <span class="NLM_string-name">Wang, G.</span></span> <span> </span><span class="NLM_article-title">Machine Learning Based Toxicity Prediction: From Chemical Structural Description to Transcriptome Analysis</span>.  <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume">19</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.3390%2Fijms19082358" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&author=Y.+Wu&author=G.+Wang&title=Machine+Learning+Based+Toxicity+Prediction%3A+From+Chemical+Structural+Description+to+Transcriptome+Analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.3390%2Fijms19082358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms19082358%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26atitle%3DMachine%2520Learning%2520Based%2520Toxicity%2520Prediction%253A%2520From%2520Chemical%2520Structural%2520Description%2520to%2520Transcriptome%2520Analysis%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2018%26volume%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muthas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, S.</span></span> <span> </span><span class="NLM_article-title">Exploiting Pharmacological Similarity to Identify Safety Concerns - Listen to What the Data Tells You</span>. <i>Mol. Inform.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1002/minf.201200088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1002%2Fminf.201200088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=27481022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFKruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=37-45&author=D.+Muthasauthor=S.+Boyer&title=Exploiting+Pharmacological+Similarity+to+Identify+Safety+Concerns+-+Listen+to+What+the+Data+Tells+You&doi=10.1002%2Fminf.201200088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting Pharmacological Similarity to Identify Safety Concerns - Listen to What the Data Tells You</span></div><div class="casAuthors">Muthas, Daniel; Boyer, Scott</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-45</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">While most new drugs are designed to act on a single target or a small no. of targets, many do show broad pharmacol. activity.  In some cases this can be beneficial and necessary for efficacy and in others it can be detrimental, leading to increased safety liability.  To probe off-target pharmacol. most drug discovery programs include screening against a broad panel of targets that represent known troublesome pharmacol.  Hits against any one of these targets can then be subjected to a risk assessment for potential safety problems in preclin. or clin. studies.  In addn., the secondary pharmacol. profile can also be thought of as an alternative description of the compd. and as such can be used as a method for assessing similarity'.  Consequently, inspection of the in vivo findings of pharmacol. neighbors can give important insights into potential safety liabilities that are neither identified by pure chem. similarity searches nor by risk assessment on individual targets.  Here we show that the pharmacol. profile contains addnl. information as compared to chem. similarity, and also demonstrate how this can be used in the hazard assessment done during drug discovery and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofUE1Sy23AN7Vg90H21EOLACvtfcHk0li-qPIeiXzxXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFKruw%253D%253D&md5=af95ecc021c2227be4f043bd467d6008</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fminf.201200088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201200088%26sid%3Dliteratum%253Aachs%26aulast%3DMuthas%26aufirst%3DD.%26aulast%3DBoyer%26aufirst%3DS.%26atitle%3DExploiting%2520Pharmacological%2520Similarity%2520to%2520Identify%2520Safety%2520Concerns%2520-%2520Listen%2520to%2520What%2520the%2520Data%2520Tells%2520You%26jtitle%3DMol.%2520Inform.%26date%3D2013%26volume%3D32%26spage%3D37%26epage%3D45%26doi%3D10.1002%2Fminf.201200088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrone, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simms, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigsch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Rethinking molecular similarity: comparing compounds on the basis of biological activity</span>. <i>Am. Chem. Soc. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1399</span>– <span class="NLM_lpage">1409</span>, <span class="refDoi"> DOI: 10.1021/cb3001028</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3001028" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1WrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1399-1409&author=P.+M.+Petroneauthor=B.+Simmsauthor=F.+Nigschauthor=E.+Lounkineauthor=P.+Kutchukianauthor=A.+Cornettauthor=Z.+Dengauthor=J.+W.+Daviesauthor=J.+L.+Jenkinsauthor=M.+Glick&title=Rethinking+molecular+similarity%3A+comparing+compounds+on+the+basis+of+biological+activity&doi=10.1021%2Fcb3001028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Rethinking Molecular Similarity: Comparing Compounds on the Basis of Biological Activity</span></div><div class="casAuthors">Petrone, Paula M.; Simms, Benjamin; Nigsch, Florian; Lounkine, Eugen; Kutchukian, Peter; Cornett, Allen; Deng, Zhan; Davies, John W.; Jenkins, Jeremy L.; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1399-1409</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Since the advent of high-throughput screening (HTS), there has been an urgent need for methods that facilitate the interrogation of large-scale chem. biol. data to build a mode of action (MoA) hypothesis.  This can be done either prior to the HTS by subset design of compds. with known MoA or post HTS by data annotation and mining.  To enable this process, we developed a tool that compares compds. solely on the basis of their bioactivity: the chem. biol. descriptor high-throughput screening fingerprint (HTS-FP).  In the current embodiment, data are aggregated from 195 biochem. and cell-based assays developed at Novartis and can be used to identify bioactivity relationships among the inhouse collection comprising ∼1.5 million compds.  We demonstrate the value of the HTS-FP for virtual screening and in particular scaffold hopping.  HTS-FP outperforms state of the art methods in several aspects, retrieving bioactive compds. with remarkable chem. dissimilarity to a probe structure.  We also apply HTS-FP for the design of screening subsets in HTS.  Using retrospective data, we show that a biodiverse selection of plates performs significantly better than a chem. diverse selection of plates, both in terms of no. of hits and diversity of chemotypes retrieved.  This is also true in the case of hit expansion predictions using HTS-FP similarity.  Sets of compds. clustered with HTS-FP are biol. meaningful, in the sense that these clusters enrich for genes and gene ontol. (GO) terms, showing that compds. that are bioactively similar also tend to target proteins that operate together in the cell.  HTS-FP are valuable not only because of their predictive power but mainly because they relate compds. solely on the basis of bioactivity, harnessing the accumulated knowledge of a high-throughput screening facility toward the understanding of how compds. interact with the proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFjchAxW_TSLVg90H21EOLACvtfcHk0lgS8FD54Daouw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1WrtLY%253D&md5=66ecd494f9200a0fdc1c142402ff255f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fcb3001028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3001028%26sid%3Dliteratum%253Aachs%26aulast%3DPetrone%26aufirst%3DP.%2BM.%26aulast%3DSimms%26aufirst%3DB.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DKutchukian%26aufirst%3DP.%26aulast%3DCornett%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DRethinking%2520molecular%2520similarity%253A%2520comparing%2520compounds%2520on%2520the%2520basis%2520of%2520biological%2520activity%26jtitle%3DAm.%2520Chem.%2520Soc.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1399%26epage%3D1409%26doi%3D10.1021%2Fcb3001028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sturm, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandriessche, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klambauer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engkvist, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">Application of Bioactivity Profile-Based Fingerprints for Building Machine Learning Models</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">962</span>– <span class="NLM_lpage">972</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00550</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00550" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=962-972&author=N.+Sturmauthor=J.+Sunauthor=Y.+Vandriesscheauthor=A.+Mayrauthor=G.+Klambauerauthor=L.+Carlssonauthor=O.+Engkvistauthor=H.+Chen&title=Application+of+Bioactivity+Profile-Based+Fingerprints+for+Building+Machine+Learning+Models&doi=10.1021%2Facs.jcim.8b00550"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00550%26sid%3Dliteratum%253Aachs%26aulast%3DSturm%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DVandriessche%26aufirst%3DY.%26aulast%3DMayr%26aufirst%3DA.%26aulast%3DKlambauer%26aufirst%3DG.%26aulast%3DCarlsson%26aufirst%3DL.%26aulast%3DEngkvist%26aufirst%3DO.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DApplication%2520of%2520Bioactivity%2520Profile-Based%2520Fingerprints%2520for%2520Building%2520Machine%2520Learning%2520Models%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2019%26volume%3D59%26spage%3D962%26epage%3D972%26doi%3D10.1021%2Facs.jcim.8b00550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laufkotter, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturm, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engkvist, O.</span></span> <span> </span><span class="NLM_article-title">Combining structural and bioactivity-based fingerprints improves prediction performance and scaffold hopping capability</span>. <i>Aust. J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">54</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=54&author=O.+Laufkotterauthor=N.+Sturmauthor=J.+Bajorathauthor=H.+Chenauthor=O.+Engkvist&title=Combining+structural+and+bioactivity-based+fingerprints+improves+prediction+performance+and+scaffold+hopping+capability"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLaufkotter%26aufirst%3DO.%26aulast%3DSturm%26aufirst%3DN.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DEngkvist%26aufirst%3DO.%26atitle%3DCombining%2520structural%2520and%2520bioactivity-based%2520fingerprints%2520improves%2520prediction%2520performance%2520and%2520scaffold%2520hopping%2520capability%26jtitle%3DAust.%2520J.%2520Chem.%26date%3D2019%26volume%3D11%26spage%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helal, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciejewski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregori-Puigjane, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span> <span> </span><span class="NLM_article-title">Public Domain HTS Fingerprints: Design and Evaluation of Compound Bioactivity Profiles from PubChem’s Bioassay Repository</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">390</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00498</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00498" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=390-398&author=K.+Y.+Helalauthor=M.+Maciejewskiauthor=E.+Gregori-Puigjaneauthor=M.+Glickauthor=A.+M.+Wassermann&title=Public+Domain+HTS+Fingerprints%3A+Design+and+Evaluation+of+Compound+Bioactivity+Profiles+from+PubChem%E2%80%99s+Bioassay+Repository&doi=10.1021%2Facs.jcim.5b00498"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00498%26sid%3Dliteratum%253Aachs%26aulast%3DHelal%26aufirst%3DK.%2BY.%26aulast%3DMaciejewski%26aufirst%3DM.%26aulast%3DGregori-Puigjane%26aufirst%3DE.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26atitle%3DPublic%2520Domain%2520HTS%2520Fingerprints%253A%2520Design%2520and%2520Evaluation%2520of%2520Compound%2520Bioactivity%2520Profiles%2520from%2520PubChem%25E2%2580%2599s%2520Bioassay%2520Repository%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2016%26volume%3D56%26spage%3D390%26epage%3D398%26doi%3D10.1021%2Facs.jcim.5b00498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cortes
Cabrera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrone, P. M.</span></span> <span> </span><span class="NLM_article-title">Optimal HTS Fingerprint Definitions by Using a Desirability Function and a Genetic Algorithm</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">641</span>– <span class="NLM_lpage">646</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00447</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00447" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=641-646&author=A.+Cortes%0ACabreraauthor=P.+M.+Petrone&title=Optimal+HTS+Fingerprint+Definitions+by+Using+a+Desirability+Function+and+a+Genetic+Algorithm&doi=10.1021%2Facs.jcim.7b00447"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00447%26sid%3Dliteratum%253Aachs%26aulast%3DCortes%2BCabrera%26aufirst%3DA.%26aulast%3DPetrone%26aufirst%3DP.%2BM.%26atitle%3DOptimal%2520HTS%2520Fingerprint%2520Definitions%2520by%2520Using%2520a%2520Desirability%2520Function%2520and%2520a%2520Genetic%2520Algorithm%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2018%26volume%3D58%26spage%3D641%26epage%3D646%26doi%3D10.1021%2Facs.jcim.7b00447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gardiner, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holliday, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Dowd, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span> <span> </span><span class="NLM_article-title">Effectiveness of 2D fingerprints for scaffold hopping</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">414</span>, <span class="refDoi"> DOI: 10.4155/fmc.11.4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.4155%2Ffmc.11.4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=21452977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVClu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=405-414&author=E.+J.+Gardinerauthor=J.+D.+Hollidayauthor=C.+O%E2%80%99Dowdauthor=P.+Willett&title=Effectiveness+of+2D+fingerprints+for+scaffold+hopping&doi=10.4155%2Ffmc.11.4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Effectiveness of 2D fingerprints for scaffold hopping</span></div><div class="casAuthors">Gardiner, Eleanor J.; Holliday, John D.; O'Dowd, Caroline; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">405-414</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Background: It has been suggested that similarity searching using 2D fingerprints may not be suitable for scaffold hopping.  Methods: This article reports a detailed evaluation of the effectiveness of six common types of 2D fingerprints when they are used for scaffold-hopping similarity searches of the Mol. Design Limited Drug Data Report database, World of Mol. Bioactivity database and Maximum Unbiased Validation database.  Results: The results demonstrate that 2D fingerprints can be used for scaffold hopping, with novel scaffolds being identified in nearly every search that was carried out.  The degree of enrichment depends on the structural diversity of the actives that are being sought, with the greatest enrichments often being obtained using the extended connectivity fingerprint encoding a circular substructure of diam. four bonds (ECFP4) fingerprint.  Conclusion: 2D fingerprints provide a simple and computationally efficient way of identifying novel chemotypes in lead-discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhWHRgpKIC5LVg90H21EOLACvtfcHk0liTdLk9sET24A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVClu7w%253D&md5=502004660048eb4960f671e51c135264</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.4155%2Ffmc.11.4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.11.4%26sid%3Dliteratum%253Aachs%26aulast%3DGardiner%26aufirst%3DE.%2BJ.%26aulast%3DHolliday%26aufirst%3DJ.%2BD.%26aulast%3DO%25E2%2580%2599Dowd%26aufirst%3DC.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DEffectiveness%2520of%25202D%2520fingerprints%2520for%2520scaffold%2520hopping%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D405%26epage%3D414%26doi%3D10.4155%2Ffmc.11.4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, S.</span></span> <span> </span><span class="NLM_article-title">PARP inhibitors in pancreatic cancer: molecular mechanisms and clinical applications</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">49</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1186%2Fs12943-020-01167-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=32122376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvVaksbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=49&author=H.+Zhuauthor=M.+Weiauthor=J.+Xuauthor=J.+Huaauthor=C.+Liangauthor=Q.+Mengauthor=Y.+Zhangauthor=J.+Liuauthor=B.+Zhangauthor=X.+Yuauthor=S.+Shi&title=PARP+inhibitors+in+pancreatic+cancer%3A+molecular+mechanisms+and+clinical+applications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">PARP inhibitors in pancreatic cancer: molecular mechanisms and clinical applications</span></div><div class="casAuthors">Zhu, Heng; Wei, Miaoyan; Xu, Jin; Hua, Jie; Liang, Chen; Meng, Qingcai; Zhang, Yiyin; Liu, Jiang; Zhang, Bo; Yu, Xianjun; Shi, Si</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Abstr.: Pancreatic cancer is a highly lethal disease with a poor prognosis, and existing therapies offer only limited effectiveness.  Mutation gene sequencing has shown several gene assocns. that may account for its carcinogenesis, revealing a promising research direction.  Poly (ADP-ribose) polymerase (PARP) inhibitors target tumor cells with a homologous recombination repair (HRR) deficiency based on the concept of synthetic lethality.  The most prominent target gene is BRCA, in which mutations were first identified in breast cancer and ovarian cancer.  PARP inhibitors can trap the PARP-1 protein at a single-stranded break/DNA lesion and disrupt its catalytic cycle, ultimately leading to replication fork progression and consequent double-strand breaks.  For tumor cells with BRCA mutations, HRR loss would result in cell death.  Pancreatic cancer has also been reported to have a strong relationship with BRCA gene mutations, which indicates that pancreatic cancer patients may benefit from PARP inhibitors.  Several clin. trials are being conducted and have begun to yield results.  For example, the POLO (Pancreatic Cancer Olaparib Ongoing) trial has demonstrated that the median progression-free survival was observably longer in the olaparib group than in the placebo group.  However, PARP inhibitor resistance has partially precluded their use in clin. applications, and the major mechanism underlying this resistance is the restoration of HRR.  Therefore, detg. how to use PARP inhibitors in more clin. applications and how to avoid adverse effects, as well as prognosis and treatment response biomarkers, require addnl. research.  This review elaborates on future prospects for the application of PARP inhibitors in pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpds8_DZ3Aiu7Vg90H21EOLACvtfcHk0lg_URB9icxubA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvVaksbs%253D&md5=e147882c753e7c0116a64117255e71c5</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1186%2Fs12943-020-01167-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-020-01167-9%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DWei%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DHua%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DMeng%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DS.%26atitle%3DPARP%2520inhibitors%2520in%2520pancreatic%2520cancer%253A%2520molecular%2520mechanisms%2520and%2520clinical%2520applications%26jtitle%3DMol.%2520Cancer%26date%3D2020%26volume%3D19%26spage%3D49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D. S.</span></span> <span> </span><span class="NLM_article-title">The ups and downs of Poly(ADP-ribose) Polymerase-1 inhibitors in cancer therapy-Current progress and future direction</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>203</i></span>, <span class="NLM_elocation-id">112570</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1016%2Fj.ejmech.2020.112570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=32717529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVOqsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=203&publication_year=2020&author=Y.+Zhaoauthor=L.+X.+Zhangauthor=T.+Jiangauthor=J.+Longauthor=Z.+Y.+Maauthor=A.+P.+Luauthor=Y.+Chengauthor=D.+S.+Cao&title=The+ups+and+downs+of+Poly%28ADP-ribose%29+Polymerase-1+inhibitors+in+cancer+therapy-Current+progress+and+future+direction"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The ups and downs of Poly(ADP-ribose) Polymerase-1 inhibitors in cancer therapy-Current progress and future direction</span></div><div class="casAuthors">Zhao, Yue; Zhang, Liu-Xia; Jiang, Ting; Long, Jing; Ma, Zhong-Ye; Lu, Ai-Ping; Cheng, Yan; Cao, Dong-Sheng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">203</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112570</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) Polymerase 1 (PARP1), one of the most investigated 18 membered PARP family enzymes, is involved in a variety of cellular functions including DNA damage repair, gene transcription and cell apoptosis.  PARP1 can form a PARP1(ADP-ribose) polymers, then bind to the DNA damage gap to recruit DNA repair proteins, and repair the break to maintain genomic stability.  PARP1 is highly expressed in tumor cells, so the inhibition of PARP1 can block DNA repair, promote tumor cell apoptosis, and exert antitumor activity.  To date, four PARP1 inhibitors namely olaparib, rucaparib, niraparib and talazoparib, have been approved by Food and Drug Administration (FDA) for treating ovarian cancer and breast cancer with BRCA1/2 mutation.  These drugs have showed super advantages over conventional chemotherapeutic drugs with low hematol. toxicity and slowly developed drug resistance.  In this article, we summarize and analyze the structure features of PARP1, the biol. functions and antitumor mechanisms of PARP1 inhibitors.  Importantly, we suggest that establishing a new structure-activity relationship of developed PARP1 inhibitors via substructural searching and the matched mol. pair anal. would accelerate the process in finding more potent and safer PARP1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPrtemJ1WT_LVg90H21EOLACvtfcHk0lg_URB9icxubA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVOqsb3E&md5=d999be53f0d108423d532411a475194f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112570%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DL.%2BX.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DLong%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DZ.%2BY.%26aulast%3DLu%26aufirst%3DA.%2BP.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%2BS.%26atitle%3DThe%2520ups%2520and%2520downs%2520of%2520Poly%2528ADP-ribose%2529%2520Polymerase-1%2520inhibitors%2520in%2520cancer%2520therapy-Current%2520progress%2520and%2520future%2520direction%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span>. <i>J. Med. Pharma. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Pharma.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span>. <i>J. Med. Pharma. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+Halgren&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Pharma.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keiser, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armbruster, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernsberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Relating protein pharmacology by ligand chemistry</span>. <i>Biotechnology</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">197</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1038/nbt1284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1038%2Fnbt1284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=17287757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=197-206&author=M.+J.+Keiserauthor=B.+L.+Rothauthor=B.+N.+Armbrusterauthor=P.+Ernsbergerauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Relating+protein+pharmacology+by+ligand+chemistry&doi=10.1038%2Fnbt1284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Relating protein pharmacology by ligand chemistry</span></div><div class="casAuthors">Keiser, Michael J.; Roth, Bryan L.; Armbruster, Blaine N.; Ernsberger, Paul; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">197-206</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The identification of protein function based on biol. information is an area of intense research.  Here the authors consider a complementary technique that quant. groups and relates proteins based on the chem. similarity of their ligands.  The authors began with 65,000 ligands annotated into sets for hundreds of drug targets.  The similarity score between each set was calcd. using ligand topol.  A statistical model was developed to rank the significance of the resulting similarity scores, which are expressed as a min. spanning tree to map the sets together.  Although these maps are connected solely by chem. similarity, biol. sensible clusters nevertheless emerged.  Links among unexpected targets also emerged, among them that methadone, emetine and loperamide (Imodium) may antagonize muscarinic M3, α2 adrenergic and neurokinin NK2 receptors, resp.  These predictions were subsequently confirmed exptl.  Relating receptors by ligand chem. organizes biol. to reveal unexpected relationships that may be assayed using the ligands themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKEMguhmluLVg90H21EOLACvtfcHk0lhYJ_5Cp4rgsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOrsLo%253D&md5=1b7373d52563fca5fe1e893d85f70573</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnbt1284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1284%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DArmbruster%26aufirst%3DB.%2BN.%26aulast%3DErnsberger%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRelating%2520protein%2520pharmacology%2520by%2520ligand%2520chemistry%26jtitle%3DBiotechnology%26date%3D2007%26volume%3D25%26spage%3D197%26epage%3D206%26doi%3D10.1038%2Fnbt1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoete, V.</span></span> <span> </span><span class="NLM_article-title">SwissTargetPrediction: updated data and new features for efficient prediction of protein targets of small molecules</span>. <i>Nucleic Acids Symp. Ser.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">W357</span>– <span class="NLM_lpage">W364</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1093%2Fnar%2Fgkz382" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W357-W364&author=A.+Dainaauthor=O.+Michielinauthor=V.+Zoete&title=SwissTargetPrediction%3A+updated+data+and+new+features+for+efficient+prediction+of+protein+targets+of+small+molecules&doi=10.1093%2Fnar%2Fgkz382"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz382%26sid%3Dliteratum%253Aachs%26aulast%3DDaina%26aufirst%3DA.%26aulast%3DMichielin%26aufirst%3DO.%26aulast%3DZoete%26aufirst%3DV.%26atitle%3DSwissTargetPrediction%253A%2520updated%2520data%2520and%2520new%2520features%2520for%2520efficient%2520prediction%2520of%2520protein%2520targets%2520of%2520small%2520molecules%26jtitle%3DNucleic%2520Acids%2520Symp.%2520Ser.%26date%3D2019%26volume%3D47%26spage%3DW357%26epage%3DW364%26doi%3D10.1093%2Fnar%2Fgkz382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D. S.</span></span> <span> </span><span class="NLM_article-title">TargetNet: a web service for predicting potential drug-target interaction profiling via multi-target SAR models</span>. <i>Perspect. Drug Discovery Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1007/s10822-016-9915-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1007%2Fs10822-016-9915-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=413-424&author=Z.+J.+Yaoauthor=J.+Dongauthor=Y.+J.+Cheauthor=M.+F.+Zhuauthor=M.+Wenauthor=N.+N.+Wangauthor=S.+Wangauthor=A.+P.+Luauthor=D.+S.+Cao&title=TargetNet%3A+a+web+service+for+predicting+potential+drug-target+interaction+profiling+via+multi-target+SAR+models&doi=10.1007%2Fs10822-016-9915-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1007%2Fs10822-016-9915-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-016-9915-2%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DZ.%2BJ.%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DChe%26aufirst%3DY.%2BJ.%26aulast%3DZhu%26aufirst%3DM.%2BF.%26aulast%3DWen%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DN.%2BN.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DA.%2BP.%26aulast%3DCao%26aufirst%3DD.%2BS.%26atitle%3DTargetNet%253A%2520a%2520web%2520service%2520for%2520predicting%2520potential%2520drug-target%2520interaction%2520profiling%2520via%2520multi-target%2520SAR%2520models%26jtitle%3DPerspect.%2520Drug%2520Discovery%2520Des.%26date%3D2016%26volume%3D30%26spage%3D413%26epage%3D424%26doi%3D10.1007%2Fs10822-016-9915-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H. P. M.</span></span> <span> </span><span class="NLM_article-title">Update: a web server for potential drug target identification with a comprehensive target pharmacophore database</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2017</i></span>,  <span class="NLM_fpage">W356</span>– <span class="NLM_lpage">W360</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1093%2Fnar%2Fgkx374" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2017&publication_year=2017&pages=W356-W360&author=X.+Wangauthor=Y.+Shenauthor=S.+Wangauthor=S.+Liauthor=W.+Zhangauthor=X.+Liuauthor=L.+Laiauthor=J.+Peiauthor=H.+P.+M.+Li&title=Update%3A+a+web+server+for+potential+drug+target+identification+with+a+comprehensive+target+pharmacophore+database"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx374%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLai%26aufirst%3DL.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%2BP.%2BM.%26atitle%3DUpdate%253A%2520a%2520web%2520server%2520for%2520potential%2520drug%2520target%2520identification%2520with%2520a%2520comprehensive%2520target%2520pharmacophore%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D2017%26spage%3DW356%26epage%3DW360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grisoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Scaffold-Hopping from Synthetic Drugs by Holistic Molecular Representation</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">16469</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1038%2Fs41598-018-34677-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=30405170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A280%3ADC%252BB3cvnslSmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&author=F.+Grisoniauthor=D.+Merkauthor=R.+Byrneauthor=G.+Schneider&title=Scaffold-Hopping+from+Synthetic+Drugs+by+Holistic+Molecular+Representation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold-Hopping from Synthetic Drugs by Holistic Molecular Representation</span></div><div class="casAuthors">Grisoni Francesca; Merk Daniel; Byrne Ryan; Schneider Gisbert; Grisoni Francesca</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16469</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The discovery of novel ligand chemotypes allows to explore uncharted regions in chemical space, thereby potentially improving synthetic accessibility, potency, and the drug-likeness of molecules.  Here, we demonstrate the scaffold-hopping ability of the new Weighted Holistic Atom Localization and Entity Shape (WHALES) molecular descriptors compared to seven state-of-the-art molecular representations on 30,000 compounds and 182 biological targets.  In a prospective application, we apply WHALES to the discovery of novel retinoid X receptor (RXR) modulators.  WHALES descriptors identified four agonists with innovative molecular scaffolds, populating uncharted regions of the chemical space.  One of the agonists, possessing a rare non-acidic chemotype, revealed high selectivity on 12 nuclear receptors and comparable efficacy as bexarotene on induction of ATP-binding cassette transporter A1, angiopoietin like protein 4 and apolipoprotein E.  The outcome of this research supports WHALES as an innovative tool to explore novel regions of the chemical space and to detect novel bioactive chemotypes by straightforward similarity searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSbrRUzliurTk2yPDyEFiigfW6udTcc2eaMsWifbSlCvrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvnslSmuw%253D%253D&md5=79373c2f6568169c54ed622d2e69a633</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-34677-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-34677-0%26sid%3Dliteratum%253Aachs%26aulast%3DGrisoni%26aufirst%3DF.%26aulast%3DMerk%26aufirst%3DD.%26aulast%3DByrne%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DScaffold-Hopping%2520from%2520Synthetic%2520Drugs%2520by%2520Holistic%2520Molecular%2520Representation%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grisoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Consonni, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiss, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tagliabue, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todeschini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Scaffold hopping from natural products to synthetic mimetics by holistic molecular similarity</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvVyqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=1-9&author=F.+Grisoniauthor=D.+Merkauthor=V.+Consonniauthor=J.+A.+Hissauthor=S.+G.+Tagliabueauthor=R.+Todeschiniauthor=G.+Schneider&title=Scaffold+hopping+from+natural+products+to+synthetic+mimetics+by+holistic+molecular+similarity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold hopping from natural products to synthetic mimetics by holistic molecular similarity</span></div><div class="casAuthors">Grisoni, Francesca; Merk, Daniel; Consonni, Viviana; Hiss, Jan A.; Tagliabue, Sara Giani; Todeschini, Roberto; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Communications Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>CODEN:
                <span class="NLM_cas:coden">CCOHCT</span>;
        ISSN:<span class="NLM_cas:issn">2399-3669</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Natural products offer unexplored mol. frameworks for the development of chem. leads and innovative drugs.  However, the structural complexity of natural products compared with synthetic drug-like mols. often limits the scaffold hopping potential of natural-product-inspired mol. design.  Here we introduce a holistic mol. representation incorporating pharmacophore and shape patterns, which facilitates scaffold hopping from natural products to isofunctional synthetic compds.  This computational approach captures simultaneously the partial charge, atom distributions and mol. shape.  In a prospective application, we use four natural cannabinoids as queries in a chem. database search for novel synthetic modulators of human cannabinoid receptors.  Of the synthetic compds. selected by the new method, 35% are exptl. confirmed as active.  These cannabinoid receptor modulators are structurally less complex than their resp. natural product templates.  The results of this study validate this holistic mol. representation for hit and lead finding in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovwYjU-7sJ17Vg90H21EOLACvtfcHk0lhFueEaGU2FGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvVyqtLk%253D&md5=1b28699db14d8c63b992561efa73821d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGrisoni%26aufirst%3DF.%26aulast%3DMerk%26aufirst%3DD.%26aulast%3DConsonni%26aufirst%3DV.%26aulast%3DHiss%26aufirst%3DJ.%2BA.%26aulast%3DTagliabue%26aufirst%3DS.%2BG.%26aulast%3DTodeschini%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DScaffold%2520hopping%2520from%2520natural%2520products%2520to%2520synthetic%2520mimetics%2520by%2520holistic%2520molecular%2520similarity%26jtitle%3DCommun.%2520Chem.%26date%3D2018%26volume%3D1%26spage%3D1%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bemis, G.</span></span> <span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span>. <i>J. Med. Pharm. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2893</span>, <span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+Bemis&title=The+properties+of+known+drugs.+1.+Molecular+frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Pharm.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fillbrunn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeuffer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrum, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthold, M. R.</span></span> <span> </span><span class="NLM_article-title">KNIME for reproducible cross-domain analysis of life science data</span>. <i>J. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>261</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1016/j.jbiotec.2017.07.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1016%2Fj.jbiotec.2017.07.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=28757290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1yrs73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=2017&pages=149-156&author=A.+Fillbrunnauthor=C.+Dietzauthor=J.+Pfeufferauthor=R.+Rahnauthor=G.+A.+Landrumauthor=M.+R.+Berthold&title=KNIME+for+reproducible+cross-domain+analysis+of+life+science+data&doi=10.1016%2Fj.jbiotec.2017.07.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">KNIME for reproducible cross-domain analysis of life science data</span></div><div class="casAuthors">Fillbrunn, Alexander; Dietz, Christian; Pfeuffer, Julianus; Rahn, Rene; Landrum, Gregory A.; Berthold, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biotechnology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">149-156</span>CODEN:
                <span class="NLM_cas:coden">JBITD4</span>;
        ISSN:<span class="NLM_cas:issn">0168-1656</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Expts. in the life sciences often involve tools from a variety of domains such as mass spectrometry, next generation sequencing, or image processing.  Passing the data between those tools often involves complex scripts for controlling data flow, data transformation, and statistical anal.  Such scripts are not only prone to be platform dependent, they also tend to grow as the expt. progresses and are seldomly well documented, a fact that hinders the reproducibility of the expt.  Workflow systems such as KNIME Analytics Platform aim to solve these problems by providing a platform for connecting tools graphically and guaranteeing the same results on different operating systems.  As an open source software, KNIME allows scientists and programmers to provide their own extensions to the scientific community.  In this review paper we present selected extensions from the life sciences that simplify data exploration, anal., and visualization and are interoperable due to KNIME's unified data model.  Addnl., we name other workflow systems that are commonly used in the life sciences and highlight their similarities and differences to KNIME.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK1RhJJLnEJ7Vg90H21EOLACvtfcHk0lhnwu0dywDiNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1yrs73I&md5=ab6ae9e93da9fe972abe5f99903b33cc</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.jbiotec.2017.07.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbiotec.2017.07.028%26sid%3Dliteratum%253Aachs%26aulast%3DFillbrunn%26aufirst%3DA.%26aulast%3DDietz%26aufirst%3DC.%26aulast%3DPfeuffer%26aufirst%3DJ.%26aulast%3DRahn%26aufirst%3DR.%26aulast%3DLandrum%26aufirst%3DG.%2BA.%26aulast%3DBerthold%26aufirst%3DM.%2BR.%26atitle%3DKNIME%2520for%2520reproducible%2520cross-domain%2520analysis%2520of%2520life%2520science%2520data%26jtitle%3DJ.%2520Biotechnol.%26date%3D2017%26volume%3D261%26spage%3D149%26epage%3D156%26doi%3D10.1016%2Fj.jbiotec.2017.07.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berthold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cebron, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dill, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kötter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieb, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiswedel, B.</span></span> <span> </span><span class="NLM_article-title">KNIME: The Konstanz Information Miner</span>. <i>ACM Sigkdd Explor. Newsl.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">31</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1145%2F1656274.1656280" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=26-31&author=M.+R.+Bertholdauthor=N.+Cebronauthor=F.+Dillauthor=T.+R.+Gabrielauthor=T.+K%C3%B6tterauthor=T.+Meinlauthor=P.+Ohlauthor=C.+Siebauthor=K.+Thielauthor=B.+Wiswedel&title=KNIME%3A+The+Konstanz+Information+Miner"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1145%2F1656274.1656280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1145%252F1656274.1656280%26sid%3Dliteratum%253Aachs%26aulast%3DBerthold%26aufirst%3DM.%2BR.%26aulast%3DCebron%26aufirst%3DN.%26aulast%3DDill%26aufirst%3DF.%26aulast%3DGabriel%26aufirst%3DT.%2BR.%26aulast%3DK%25C3%25B6tter%26aufirst%3DT.%26aulast%3DMeinl%26aufirst%3DT.%26aulast%3DOhl%26aufirst%3DP.%26aulast%3DSieb%26aufirst%3DC.%26aulast%3DThiel%26aufirst%3DK.%26aulast%3DWiswedel%26aufirst%3DB.%26atitle%3DKNIME%253A%2520The%2520Konstanz%2520Information%2520Miner%26jtitle%3DACM%2520Sigkdd%2520Explor.%2520Newsl.%26date%3D2006%26volume%3D11%26spage%3D26%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geppert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Scaffold hopping using two-dimensional fingerprints: true potential, black magic, or a hopeless endeavor? Guidelines for virtual screening</span>. <i>J. Med. Pharm. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5707</span>– <span class="NLM_lpage">5715</span>, <span class="refDoi"> DOI: 10.1021/jm100492z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100492z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5707-5715&author=M.+Vogtauthor=D.+Stumpfeauthor=H.+Geppertauthor=J.+Bajorath&title=Scaffold+hopping+using+two-dimensional+fingerprints%3A+true+potential%2C+black+magic%2C+or+a+hopeless+endeavor%3F+Guidelines+for+virtual+screening&doi=10.1021%2Fjm100492z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm100492z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100492z%26sid%3Dliteratum%253Aachs%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DGeppert%26aufirst%3DH.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DScaffold%2520hopping%2520using%2520two-dimensional%2520fingerprints%253A%2520true%2520potential%252C%2520black%2520magic%252C%2520or%2520a%2520hopeless%2520endeavor%253F%2520Guidelines%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Pharm.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5707%26epage%3D5715%26doi%3D10.1021%2Fjm100492z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Southan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boppana, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagarlapudi, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muresan, S.</span></span> <span> </span><span class="NLM_article-title">Analysis of in vitro bioactivity data extracted from drug discovery literature and patents: Ranking 1654 human protein targets by assayed compounds and molecular scaffolds</span>. <i>Aust. J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">14</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=14&author=C.+Southanauthor=K.+Boppanaauthor=S.+A.+Jagarlapudiauthor=S.+Muresan&title=Analysis+of+in+vitro+bioactivity+data+extracted+from+drug+discovery+literature+and+patents%3A+Ranking+1654+human+protein+targets+by+assayed+compounds+and+molecular+scaffolds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSouthan%26aufirst%3DC.%26aulast%3DBoppana%26aufirst%3DK.%26aulast%3DJagarlapudi%26aufirst%3DS.%2BA.%26aulast%3DMuresan%26aufirst%3DS.%26atitle%3DAnalysis%2520of%2520in%2520vitro%2520bioactivity%2520data%2520extracted%2520from%2520drug%2520discovery%2520literature%2520and%2520patents%253A%2520Ranking%25201654%2520human%2520protein%2520targets%2520by%2520assayed%2520compounds%2520and%2520molecular%2520scaffolds%26jtitle%3DAust.%2520J.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">ChEMBL web services: streamlining access to drug discovery data and utilities</span>. <i>Nucleic Acids Symp. Ser.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">W612</span>– <span class="NLM_lpage">W620</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1093%2Fnar%2Fgkv352" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=W612-W620&author=M.+Daviesauthor=M.+Nowotkaauthor=G.+Papadatosauthor=N.+Dedmanauthor=A.+Gaultonauthor=F.+Atkinsonauthor=L.+Bellisauthor=J.+P.+Overington&title=ChEMBL+web+services%3A+streamlining+access+to+drug+discovery+data+and+utilities&doi=10.1093%2Fnar%2Fgkv352"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv352%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DDedman%26aufirst%3DN.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DAtkinson%26aufirst%3DF.%26aulast%3DBellis%26aufirst%3DL.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%2520web%2520services%253A%2520streamlining%2520access%2520to%2520drug%2520discovery%2520data%2520and%2520utilities%26jtitle%3DNucleic%2520Acids%2520Symp.%2520Ser.%26date%3D2015%26volume%3D43%26spage%3DW612%26epage%3DW620%26doi%3D10.1093%2Fnar%2Fgkv352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Veij, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felix, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magarinos, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosquera, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutowo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordillo-Maranon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mugumbate, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Lopez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radoux, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura-Cabrera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span> <span> </span><span class="NLM_article-title">ChEMBL: towards direct deposition of bioassay data</span>. <i>Nucleic Acids Symp. Ser.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">D930</span>– <span class="NLM_lpage">D940</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1093%2Fnar%2Fgky1075" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D930-D940&author=D.+Mendezauthor=A.+Gaultonauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+De+Veijauthor=E.+Felixauthor=M.+P.+Magarinosauthor=J.+F.+Mosqueraauthor=P.+Mutowoauthor=M.+Nowotkaauthor=M.+Gordillo-Maranonauthor=F.+Hunterauthor=L.+Juncoauthor=G.+Mugumbateauthor=M.+Rodriguez-Lopezauthor=F.+Atkinsonauthor=N.+Boscauthor=C.+J.+Radouxauthor=A.+Segura-Cabreraauthor=A.+Herseyauthor=A.+R.+Leach&title=ChEMBL%3A+towards+direct+deposition+of+bioassay+data&doi=10.1093%2Fnar%2Fgky1075"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1075%26sid%3Dliteratum%253Aachs%26aulast%3DMendez%26aufirst%3DD.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDe%2BVeij%26aufirst%3DM.%26aulast%3DFelix%26aufirst%3DE.%26aulast%3DMagarinos%26aufirst%3DM.%2BP.%26aulast%3DMosquera%26aufirst%3DJ.%2BF.%26aulast%3DMutowo%26aufirst%3DP.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DGordillo-Maranon%26aufirst%3DM.%26aulast%3DHunter%26aufirst%3DF.%26aulast%3DJunco%26aufirst%3DL.%26aulast%3DMugumbate%26aufirst%3DG.%26aulast%3DRodriguez-Lopez%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DF.%26aulast%3DBosc%26aufirst%3DN.%26aulast%3DRadoux%26aufirst%3DC.%2BJ.%26aulast%3DSegura-Cabrera%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26atitle%3DChEMBL%253A%2520towards%2520direct%2520deposition%2520of%2520bioassay%2520data%26jtitle%3DNucleic%2520Acids%2520Symp.%2520Ser.%26date%3D2019%26volume%3D47%26spage%3DD930%26epage%3DD940%26doi%3D10.1093%2Fnar%2Fgky1075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorissen, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span> <span> </span><span class="NLM_article-title">BindingDB: a web-accessible database of experimentally determined protein-ligand binding affinities</span>. <i>Nucleic Acids Symp. Ser.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">D198</span>– <span class="NLM_lpage">D201</span>, <span class="refDoi"> DOI: 10.1093/nar/gkl999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1093%2Fnar%2Fgkl999" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=D198-D201&author=T.+Liuauthor=Y.+Linauthor=X.+Wenauthor=R.+N.+Jorissenauthor=M.+K.+Gilson&title=BindingDB%3A+a+web-accessible+database+of+experimentally+determined+protein-ligand+binding+affinities&doi=10.1093%2Fnar%2Fgkl999"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl999%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DJorissen%26aufirst%3DR.%2BN.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520a%2520web-accessible%2520database%2520of%2520experimentally%2520determined%2520protein-ligand%2520binding%2520affinities%26jtitle%3DNucleic%2520Acids%2520Symp.%2520Ser.%26date%3D2007%26volume%3D35%26spage%3DD198%26epage%3DD201%26doi%3D10.1093%2Fnar%2Fgkl999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span> <i>Molecular
Operating Environment (MOE), 2018.01</i>; <span class="NLM_publisher-name">Chemical
Computing Group ULC</span>, <span class="NLM_publisher-loc">Montreal,
QC</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular%0AOperating+Environment+%28MOE%29%2C+2018.01%3B+Chemical%0AComputing+Group+ULC%2C+Montreal%2C%0AQC%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMolecular%250AOperating%2520Environment%2520%2528MOE%2529%252C%25202018.01%26pub%3DChemical%250AComputing%2520Group%2520ULC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Breiman, L.</span></span> <span> </span><span class="NLM_article-title">Random forests</span>. <i>Mach. Learn.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">32</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1023%2FA%3A1010933404324" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2001&pages=5-32&author=L.+Breiman&title=Random+forests"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1023%2FA%3A1010933404324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1010933404324%26sid%3Dliteratum%253Aachs%26aulast%3DBreiman%26aufirst%3DL.%26atitle%3DRandom%2520forests%26jtitle%3DMach.%2520Learn.%26date%3D2001%26volume%3D45%26spage%3D5%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Freund, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapire, R. E.</span></span> <span> </span><span class="NLM_article-title">A decision-theoretic generalization of on-line learning and an application to boosting</span>. <i>J. Comput. Syst. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">139</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.1006%2Fjcss.1997.1504" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1997&pages=119-139&author=Y.+Freundauthor=R.+E.+Schapire&title=A+decision-theoretic+generalization+of+on-line+learning+and+an+application+to+boosting"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1006%2Fjcss.1997.1504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjcss.1997.1504%26sid%3Dliteratum%253Aachs%26aulast%3DFreund%26aufirst%3DY.%26aulast%3DSchapire%26aufirst%3DR.%2BE.%26atitle%3DA%2520decision-theoretic%2520generalization%2520of%2520on-line%2520learning%2520and%2520an%2520application%2520to%2520boosting%26jtitle%3DJ.%2520Comput.%2520Syst.%2520Sci.%26date%3D1997%26volume%3D55%26spage%3D119%26epage%3D139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, J. H.</span></span> <span> </span><span class="NLM_article-title">Greedy function approximation: a gradient boosting machine</span>. <i>Ann. Stat.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_fpage">1189</span>– <span class="NLM_lpage">1232</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&pages=1189-1232&author=J.+H.+Friedman&title=Greedy+function+approximation%3A+a+gradient+boosting+machine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFriedman%26aufirst%3DJ.%2BH.%26atitle%3DGreedy%2520function%2520approximation%253A%2520a%2520gradient%2520boosting%2520machine%26jtitle%3DAnn.%2520Stat.%26date%3D2001%26spage%3D1189%26epage%3D1232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiener, M.</span></span> <span> </span><span class="NLM_article-title">Classification and regression by randomForest</span>. <i>R News</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">22</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=18-22&author=A.+Liawauthor=M.+Wiener&title=Classification+and+regression+by+randomForest"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DWiener%26aufirst%3DM.%26atitle%3DClassification%2520and%2520regression%2520by%2520randomForest%26jtitle%3DR%2520News%26date%3D2002%26volume%3D2%26spage%3D18%26epage%3D22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alfaro, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gámez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, N.</span></span> <span> </span><span class="NLM_article-title">adabag: An R package for classification with boosting and bagging</span>. <i>J. Stat. Soft.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">35</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;key=10.18637%2Fjss.v054.i02" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2013&pages=1-35&author=E.+Alfaroauthor=M.+G%C3%A1mezauthor=N.+Garcia&title=adabag%3A+An+R+package+for+classification+with+boosting+and+bagging"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.18637%2Fjss.v054.i02&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18637%252Fjss.v054.i02%26sid%3Dliteratum%253Aachs%26aulast%3DAlfaro%26aufirst%3DE.%26aulast%3DG%25C3%25A1mez%26aufirst%3DM.%26aulast%3DGarcia%26aufirst%3DN.%26atitle%3Dadabag%253A%2520An%2520R%2520package%2520for%2520classification%2520with%2520boosting%2520and%2520bagging%26jtitle%3DJ.%2520Stat.%2520Soft.%26date%3D2013%26volume%3D54%26spage%3D1%26epage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ridgeway, G.</span></span> <i>gbm: Generalized boosted regression models</i>. R package version <span class="NLM_year">2006</span>,  <span class="NLM_volume">1</span>,  <span class="NLM_fpage">55</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=G.+Ridgeway&title=gbm%3A+Generalized+boosted+regression+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DRidgeway%26aufirst%3DG.%26jtitle%3Dgbm%253A%2520Generalized%2520boosted%2520regression%2520models%26date%3D2006%26volume%3D1%26spage%3D55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i20"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00234">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_12655"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00234?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00234</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">AMRR results for different screening datasets (Table S1 and Figure S1); PARP-1 inhibitors approved by FDA (Table S2); reverse target fishing of compound 6 and olaparib (Tables S3 and S4); information of the query data set (Table S5); the threshold of CBFP representation (Figure S2); workflow for CBFP construction (Figure S3) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_001.pdf">PDF</a>).</p></li><li><p class="inline">Molecular formula strings of all the compounds tested and associated biochemical data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_002.xlsx">XLSX</a>).</p></li><li><p class="inline">Information on the QSAR models of CBFP representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_003.csv">CSV</a>).</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_001.pdf">jm1c00234_si_001.pdf (635.56 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_002.xlsx">jm1c00234_si_002.xlsx (144.81 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00234/suppl_file/jm1c00234_si_003.csv">jm1c00234_si_003.csv (2.44 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00234&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-11%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00234%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00234" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679941d759a63d42","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
