
<html lang="en"     class="pb-page"  data-request-id="863561a4-7981-4931-9f76-b22804de4146"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-16;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.9b00869;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Design, Synthesis, and Characterization of Ogerin-Based Positive Allosteric Modulators for G Protein-Coupled Receptor 68 (GPR68)" /></meta><meta name="dc.Creator" content="Xufen  Yu" /></meta><meta name="dc.Creator" content="Xi-Ping  Huang" /></meta><meta name="dc.Creator" content="Terry P.  Kenakin" /></meta><meta name="dc.Creator" content="Samuel T.  Slocum" /></meta><meta name="dc.Creator" content="Xin  Chen" /></meta><meta name="dc.Creator" content="Michael L.  Martini" /></meta><meta name="dc.Creator" content="Jing  Liu" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="G protein-coupled receptor 68 (GPR68) is an understudied orphan G protein-coupled receptor (GPCR). It is expressed most abundantly in the brain, potentially playing important roles in learning and ..." /></meta><meta name="Description" content="G protein-coupled receptor 68 (GPR68) is an understudied orphan G protein-coupled receptor (GPCR). It is expressed most abundantly in the brain, potentially playing important roles in learning and ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 12, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00869" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00869" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00869" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00869" /></link>
        
    
    

<title>Design, Synthesis, and Characterization of Ogerin-Based Positive Allosteric Modulators for G Protein-Coupled Receptor 68 (GPR68) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00869" /></meta><meta property="og:title" content="Design, Synthesis, and Characterization of Ogerin-Based Positive Allosteric Modulators for G Protein-Coupled Receptor 68 (GPR68)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0015.jpeg" /></meta><meta property="og:description" content="G protein-coupled receptor 68 (GPR68) is an understudied orphan G protein-coupled receptor (GPCR). It is expressed most abundantly in the brain, potentially playing important roles in learning and memory. Pharmacological studies with GPR68 have been hindered by lack of chemical tools that can selectively modulate its activity. We previously reported the first small-molecule positive allosteric modulator (PAM), ogerin (1), and showed that 1 can potentiate proton activity at the GPR68–Gs pathway. Here, we report the first comprehensive structure–activity relationship (SAR) study on the scaffold of 1. Our lead compound resulted from this study, MS48107 (71), displayed 33-fold increased allosteric activity compared to 1. Compound 71 demonstrated high selectivity over closely related proton GPCRs and 48 common drug targets, and was bioavailable and brain-penetrant in mice. Thus, our SAR study has resulted in an improved GPR68 PAM for investigating the physiological and pathophysiological roles of GPR68 in vitro and in vivo." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00869"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00869">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00869&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00869&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00869&amp;href=/doi/10.1021/acs.jmedchem.9b00869" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 16</span><span class="cit-fg-pageRange">, 7557-7574</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/16" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00819" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00871" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design, Synthesis, and Characterization of Ogerin-Based Positive Allosteric Modulators for G Protein-Coupled Receptor 68 (GPR68)</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Xufen Yu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xufen Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xufen++Yu">Xufen Yu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xi-Ping Huang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xi-Ping Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology,  The National Institute of Mental Health Psychoactive Drug Screening Program (NIMH PDSP), Department of Pharmacology, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#0d757d65786c636a4d78636e23686978"><span class="__cf_email__" data-cfemail="027a726a77636c6542776c612c676677">[email protected]</span></a> (X.-P.H.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xi-Ping++Huang">Xi-Ping Huang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Terry P. Kenakin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Terry P. Kenakin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Terry+P.++Kenakin">Terry P. Kenakin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Samuel T. Slocum</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Samuel T. Slocum</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Samuel+T.++Slocum">Samuel T. Slocum</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xin Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xin Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Chen">Xin Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael L. Martini</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael L. Martini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael+L.++Martini">Michael L. Martini</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jing Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jing Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#03696a6d642d6f6a76436e70706e2d666776"><span class="__cf_email__" data-cfemail="a0cac9cec78eccc9d5e0cdd3d3cd8ec5c4d5">[email protected]</span></a> (J.L.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Liu">Jing Liu</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jian Jin</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#dcb6b5bdb2f2b6b5b29cb1afafb1f2b9b8a9"><span class="__cf_email__" data-cfemail="7d17141c13531714133d100e0e1053181908">[email protected]</span></a> (J.J.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00869&amp;href=/doi/10.1021%2Facs.jmedchem.9b00869" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 16</span><span class="cit-pageRange">, 7557–7574</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 12, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>29 May 2019</li><li><span class="item_label"><b>Published</b> online</span>12 July 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 August 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00869" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00869</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7557%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DXufen%2BYu%252C%2BXi-Ping%2BHuang%252C%2BTerry%2BP.%2BKenakin%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D16%26contentID%3Dacs.jmedchem.9b00869%26title%3DDesign%252C%2BSynthesis%252C%2Band%2BCharacterization%2Bof%2BOgerin-Based%2BPositive%2BAllosteric%2BModulators%2Bfor%2BG%2BProtein-Coupled%2BReceptor%2B68%2B%2528GPR68%2529%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7574%26publicationDate%3DAugust%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00869"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2110</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00869" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design, Synthesis, and Characterization of Ogerin-Based Positive Allosteric Modulators for G Protein-Coupled Receptor 68 (GPR68)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Xufen&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Xi-Ping&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Terry&quot;,&quot;last_name&quot;:&quot;P. Kenakin&quot;},{&quot;first_name&quot;:&quot;Samuel&quot;,&quot;last_name&quot;:&quot;T. Slocum&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;L. Martini&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;16&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;7557-7574&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00869&quot;},&quot;abstract&quot;:&quot;G protein-coupled receptor 68 (GPR68) is an understudied orphan G protein-coupled receptor (GPCR). It is expressed most abundantly in the brain, potentially playing important roles in learning and memory. Pharmacological studies with GPR68 have been hindered by lack of chemical tools that can selectively modulate its activity. We previously reported the first small-molecule positive allosteric modulator (PAM), ogerin (1), and showed that 1 can potentiate proton activity at the GPR68–Gs pathway. Here, we report the first comprehensive structure–activity relationship (SAR) study on the scaffold of 1. Our lead compound resulted from this study, MS48107 (71), displayed 33-fold increased allosteric activity compared to 1. Compound 71 demonstrated high selectivity over closely related proton GPCRs and 48 common drug targets, and was bioavailable and brain-penetrant in mice. Thus, our SAR study has resulted in an improved GPR68 PAM for investigating the physiological and pathophysiological roles of GPR68 in vitro an&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00869&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00869" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00869&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00869" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00869&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00869" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00869&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00869&amp;href=/doi/10.1021/acs.jmedchem.9b00869" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00869" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00869" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00869%26sid%3Dliteratum%253Aachs%26pmid%3D31298539%26genre%3Darticle%26aulast%3DYu%26date%3D2019%26atitle%3DDesign%252C%2BSynthesis%252C%2Band%2BCharacterization%2Bof%2BOgerin-Based%2BPositive%2BAllosteric%2BModulators%2Bfor%2BG%2BProtein-Coupled%2BReceptor%2B68%2B%2528GPR68%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D16%26spage%3D7557%26epage%3D7574%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291713" title="Polyacrylamide">Polyacrylamide</a>,</li><li><a href="/action/doSearch?ConceptID=291631" title="Modulators">Modulators</a>,</li><li><a href="/action/doSearch?ConceptID=291188" title="Cooperativity">Cooperativity</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/16" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/jmcmar.2019.62.issue-16/20190822/jmcmar.2019.62.issue-16.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">G protein-coupled receptor 68 (GPR68) is an understudied orphan G protein-coupled receptor (GPCR). It is expressed most abundantly in the brain, potentially playing important roles in learning and memory. Pharmacological studies with GPR68 have been hindered by lack of chemical tools that can selectively modulate its activity. We previously reported the first small-molecule positive allosteric modulator (PAM), ogerin (<b>1</b>), and showed that <b>1</b> can potentiate proton activity at the GPR68–G<sub>s</sub> pathway. Here, we report the first comprehensive structure–activity relationship (SAR) study on the scaffold of <b>1</b>. Our lead compound resulted from this study, MS48107 (<b>71</b>), displayed 33-fold increased allosteric activity compared to <b>1</b>. Compound <b>71</b> demonstrated high selectivity over closely related proton GPCRs and 48 common drug targets, and was bioavailable and brain-penetrant in mice. Thus, our SAR study has resulted in an improved GPR68 PAM for investigating the physiological and pathophysiological roles of GPR68 in vitro and in vivo.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41606" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41606" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">G protein-coupled receptor 68 (GPR68) was originally discovered as a clinically related receptor in an ovarian cancer cell line as “OGR1” for <i>o</i>varian cancer <i>G</i> protein-coupled <i>r</i>eceptor.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> While it is widely expressed in most cells and tissues, including spleen, testes, small intestine, peripheral blood leukocytes, heart, lung, placenta, kidney, and bone,<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> it is most abundantly expressed in the brain, particularly in the hippocampus<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a>—the brain region that is critical for learning and memory. Despite being known for over two decades, GPR68 remains an understudied orphan G protein-coupled receptor (GPCR) according to IUPHAR.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Generally known as a proton-sensing GPCR, it couples to multiple signaling pathways through various G proteins (G<sub>s</sub>, G<sub>q</sub>, G<sub>i</sub>, and G<sub>12/13</sub>) in different cells.<a onclick="showRef(event, 'ref2 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref2 ref5 ref6 ref7">(2,5−7)</a> However, it remains to be established whether GPR68 couples to the β-arrestin signaling pathway. Consistent with its capacity to couple to multiple signaling pathways and its wide expression pattern, GPR68 has been implicated in many biological processes, including pH homeostasis,<a onclick="showRef(event, 'ref2 ref8'); return false;" href="javascript:void(0);" class="ref ref2 ref8">(2,8)</a> tumor growth and metastasis,<a onclick="showRef(event, 'ref6 ref7 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref9 ref10 ref11">(6,7,9−11)</a> inflammation,<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12−14)</a> osteoclast development,<a onclick="showRef(event, 'ref9 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref9 ref15 ref16">(9,15,16)</a> insulin secretion,<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> and epithelial barrier function.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> Interestingly, the receptor also functions as a mechanotransducer that senses blood flow (shear stress) in arteries, thus modulating vascular physiology.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Thus, GPR68, a potentially important therapeutic target, is one of the National Institutes of Health (NIH)’s Illuminating the Druggable Genome GPCRs.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Presently, the underlying molecular mechanisms have not been determined between proton-sensing and these physiological or pathological processes. Pharmacological studies of GPR68 have previously been hindered by the lack of small-molecule ligands. It is clear that potent and selective GPR68 ligands that can be used to activate or block its activity are urgently needed to elucidate GPR68 pharmacology and clarify its potential roles in the aforementioned biological processes.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div><div class="NLM_p">Previous studies have shown that receptor activity is reduced or completely lost if extracellular histidine residues are mutated, presumably by affecting protonation states responding to extracellular pH changes.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Therefore, it was proposed that alkaline pH conditions stabilize GPR68 in an inactive conformation by hydrogen-bonding interactions involving extracellular histidine residues, while acidification and histidine protonation break hydrogen-bond interactions, initiating receptor activation.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The putative histidine-binding sites for protons are scattered throughout different extracellular loops rendering challenges to directly activate or block this receptor with orthosteric ligands. This, in turn, makes allosteric binding site more attractive to target as with many other GPCRs.<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> Allosteric modulators rely on endogenous protons to produce function and can preserve the spatial and temporal nature of the proton-mediated activation of GPR68. Allosteric modulators bind to GPR68 at a site different from that of protons and function by fine-tuning proton activity through increasing or reducing proton affinity (α-cooperativity), efficacy (β-cooperativity), or both.</div><div class="NLM_p">For conventional GPCRs, such as metabotropic glutamate receptors (mGluRs), binding and functional assays can be carried out in biological systems that the concentration of the orthosteric ligand (i.e., glutamate) can be controlled. However, for GPR68, the orthosteric ligand is proton, which always presents in the receptor compartment. Therefore, the allosteric binding affinity (<i>K</i><sub>B</sub>), defined as the dissociation equilibrium constant of allosteric modulator in the absence of orthosteric ligand (i.e., proton), does not have corresponding biological meaning and cannot be measured in binding assays. Instead, we can estimate the allosteric affinity cooperativity α, efficacy cooperativity β, and allosteric binding affinity <i>K</i><sub>B</sub> from functional assays according to the standard allosteric operational model.<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26−28)</a> The observed binding affinity of allosteric modulators can be calculated by the equation <i>K</i><sub>B</sub> = (1 + [A]/<i>K</i><sub>A</sub>)/(1 + α[A]/<i>K</i><sub>A</sub>), where [A] is the proton concentration and <i>K</i><sub>A</sub> is the equilibrium dissociation constant of protons at GPR68 in the absence of modulators. In structure–activity relationship (SAR) studies, we actually can optimize the potency of the modulators in proton-mediated signaling by optimizing α, β, and/or <i>K</i><sub>B</sub>. In contrast to traditional orthosteric agonists or antagonists which rely on binding affinity (<i>K</i><sub>i</sub>) for selectivity, allosteric modulators achieve high selectivity through allosteric αβ cooperativity in addition to allosteric binding affinity (<i>K</i><sub>B</sub>).<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> The overall allosteric activity can be quantified with a characteristic allosteric index, Log(αβ/<i>K</i><sub>B</sub>),<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> which contains information about allosteric cooperativity and can be calculated and used as a guiding parameter for SAR studies and optimization.<a onclick="showRef(event, 'ref29 ref32'); return false;" href="javascript:void(0);" class="ref ref29 ref32">(29,32)</a></div><div class="NLM_p">Development of small-molecule ligands for GPR68 remains a challenging task. A small number of isoxazole compounds have been reported to possess GPR68 agonist activity.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> However, several reports suggested that the weak activity of isoxazoles could be nonspecific.<a onclick="showRef(event, 'ref12 ref34'); return false;" href="javascript:void(0);" class="ref ref12 ref34">(12,34)</a> Divalent metal ions, such as Cu<sup>2+</sup> and Zn<sup>2+</sup>, were initially used to inhibit GPR68 activity.<a onclick="showRef(event, 'ref2 ref19'); return false;" href="javascript:void(0);" class="ref ref2 ref19">(2,19)</a> Recently, these divalent ions, in addition to Co<sup>2+</sup> and Mn<sup>2+</sup>, have been reported to be GPR68 agonists.<a onclick="showRef(event, 'ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37">(35−37)</a> However, these recent reports neither achieved full pH control nor used a broad enough pH range to address potential allosteric mechanisms. In addition, Zn<sup>2+</sup> is a well-known allosteric modulator for the orphan receptor GPR39.<a onclick="showRef(event, 'ref38 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref38 ref39 ref40">(38−40)</a> Moreover, these divalent ions are biologically important metals or trace elements and play critical roles in many biological processes.<a onclick="showRef(event, 'ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref41 ref42 ref43">(41−43)</a> Therefore, these divalent metals are unlikely GPR68 selective tools.</div><div class="NLM_p">In 2015, we reported the discovery of ogerin (<b>1</b>) as the first positive allosteric modulator (PAM) of GPR68.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Compound <b>1</b> potentiates proton activity in GPR68-mediated G<sub>s</sub>–cAMP production, but has no activity at related proton-sensing GPCRs, including GPR4 and GPR65. This selective ligand led to the discovery that GPR68 activation suppresses contextual- but not cue-dependent learning and memory, and does so in wild-type but not GPR68 knockout mice.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> These data provide the first evidence for a role of GPR68 in cognition. Specifically, sustained hippocampal GPR68 activation may contribute to memory deficits. Compound <b>1</b>, which is now commercially available, has been used by the research community, and its PAM activity has been confirmed by other groups.<a onclick="showRef(event, 'ref21 ref45'); return false;" href="javascript:void(0);" class="ref ref21 ref45">(21,45)</a> Compound <b>1</b> was identified from a ZINC sublibrary of about 3.1 million compounds through a virtual screening campaign targeting GPR68.<a onclick="showRef(event, 'ref44 ref46'); return false;" href="javascript:void(0);" class="ref ref44 ref46">(44,46)</a> Thorough SAR studies on the scaffold of compound <b>1</b> have not been reported to date. Here, we report a comprehensive SAR study at the following three regions of compound <b>1</b>: (1) the aminotriazine core, (2) the benzyl alcohol moiety, and (3) the benzylamine moiety (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). We describe design, synthesis, and biological characterization of various analogues. This study has resulted in the identification of significantly improved GPR68 PAMs.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0001.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structure of <b>1</b> and three highlighted regions for SAR study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18207" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18207" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Biological Assay and Data Analysis</h3><div class="NLM_p last">We used a GloSensor cAMP assay to characterize all of the synthesized putative GPR68 PAMs. More specifically, HEK293T cells were transiently transfected with receptor and GloSensor cAMP plasmids for proton-mediated receptor activation of G<sub>s</sub>–cAMP production. To determine allosteric modulator activity, proton concentration responses in the absence and presence of increasing concentrations of putative PAMs were quantified and results were analyzed using a standard allosteric operational model<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26−28)</a> in GraphPad Prism. Several previously reported compounds<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> (<b>1</b>, <b>20</b>, <b>30</b>, <b>55</b>, <b>61</b>, <b>62</b>, and <b>63</b>) were included in this study for comparison.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Modifications of the Aminotriazine Core</h3><div class="NLM_p">To investigate the SAR of the aminotriazine core, we replaced the free amino group on the triazine core with a few small alkylamino groups. We also switched the triazine core to a pyrimidine core (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). These designed compounds were prepared using the synthetic routes outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Intermediates <b>4</b> and <b>9</b> were prepared according to the previously reported procedures<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> from 2,4,6-trichloro-1,3,5-triazine (<b>2</b>) and 2,6-dichlorpyrimidin-4-amine (<b>8</b>). Briefly, alkylamine substitution of one chloro group on trichlorotriazine <b>1</b> at low temperature provided alkylamino-substituted dichlorotriazines <b>3</b>. Benzylamine substitution of one chloro group on dichlorotriazines <b>3</b> or dicholoropyrimidine <b>8</b> at an elevated temperature yielded intermediates <b>4</b> or <b>9</b>, respectively. Suzuki coupling between intermediates <b>4</b> or <b>9</b> and commercially available (2-(hydroxymethyl)phenyl)boronic acid provided the designed compounds <b>5</b>–<b>7</b> and <b>10</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0005.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>5</b>–<b>7</b> and <b>10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) tetrahydrofuran (THF), 0 °C to room temperature (rt), 48–67%; (b) dioxane, reflux, 1 h, 56–96%; (c) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O = 5:3, 120 °C, microwave, 20 min, 42–98%.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Effects of Modifications to the Triazine Core on Allosteric Activity<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0008.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0009.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The structure of <b>10</b> is shown.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Values represent means ± standard error of the mean (SEM) from a minimum of three independent assays, each in triplicate or quadruplicate. Δ stands for the difference of Log(αβ/<i>K</i><sub>B</sub>) from that of <b>1</b>, positive values indicated increased allosteric activity and negative values indicated reduced allosteric activity. Proton concentration–response curves are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 1</a>.</p></div></div><div></div></div><div class="NLM_p last">As indicated in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, a small monosubstituted amino group on the triazine core is well tolerated. Either methylamino (<b>5</b>) or ethylamino (<b>6</b>) group has a minimal effect on allosteric parameters or binding affinity. However, dimethylamino group (<b>7</b>) completely eliminates the allosteric activity. Replacing the triazine core with a pyrimidine (<b>10</b>) also greatly reduces allosteric activity to barely detectable levels. These results suggest that the amino-substituted triazine core is critical for the PAM activity. Therefore, the aminotriazine core is maintained in the following SAR studies.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Modifications of the Benzyl Alcohol Moiety</h3><div class="NLM_p">For the benzyl alcohol moiety, we explored a variety of substituents on the phenyl ring (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The synthesis of these analogues started with 2-amino-4,6-dichloro-1,3,5-triazine (<b>11</b>). Benzylamine substitution of one chloro group on <b>11</b> provided the common intermediate <b>12</b>. Suzuki coupling between intermediate <b>12</b> and a set of commercially available phenylboronic acids provided compounds <b>13</b>–<b>15</b>, <b>21</b>, <b>22</b>–<b>29</b>, <b>31</b>, and <b>32</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>A,D). Compound <b>16</b> was obtained by reducing acetophenone <b>15</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>B). Compounds <b>18</b> and <b>19</b> were synthesized through reductive aminations of benzaldehyde <b>17</b> with different amines (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>C). Compounds <b>37</b> and <b>38</b> were prepared through Suzuki coupling between intermediate <b>12</b> and synthesized phenylboronic acids <b>36</b> bearing ethoxy and isopropoxy groups, respectively, followed by formal group reduction (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>E).</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>13</b>–<b>15</b>, <b>21</b>, <b>16</b>, <b>18</b>, <b>19</b>, <b>22</b>–<b>29</b>, <b>31</b>, <b>32</b>, <b>37</b>, and <b>38</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) dioxane, reflux, 1 h, 78–94%; (b) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O = 5:3, 120 °C, microwave, 20 min, 56–99%; (c) NaBH<sub>4,</sub> MeOH, 0 °C to rt, 42%; (d) NH<sub>3</sub>·H<sub>2</sub>O or NH<sub>2</sub>Me, MgSO<sub>4</sub>, NaBH<sub>4</sub>, THF, rt, 2 days, 45–67%. (e) Ethylene glycol, toluene, reflux, 4–8 h, 82–96%; (f) alkyl bromide, K<sub>2</sub>CO<sub>3</sub>, dimethylformamide, 85 °C, 34–78%; (g) i. <i>n</i>-BuLi, THF, −78 °C, 30 min; ii. B(OMe)<sub>3</sub>, −78 °C, 1 h, then rt, overnight; iii. HCl (3 N), rt, 1 h, 26-57%; (h) <b>12</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O = 5:3, 120 °C, microwave, 20 min; (i) NaBH<sub>4</sub>, THF, 0 °C to rt, 23–97%.</p></p></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Effects of Modifications of the Benzylmethanol Moiety on Allosteric Activity<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0010.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0011.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Values represent means ± SEM from a minimum of three independent assays, each in triplicate or quadruplicate. Δ stands for the difference of log(αβ/<i>K</i><sub>B</sub>) from that of <b>1</b>; positive values indicated increased allosteric activity, and negative values indicated reduced allosteric activity. Proton concentration–response curves are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 1</a>.</p></div></div><div></div></div><div class="NLM_p last">Switching the hydroxymethyl group (<b>1</b>) to hydrogen (<b>13</b>), methyl (<b>14</b>), methyl ketone (<b>15</b>), and 1-hydroxylethyl (<b>16</b>) groups is not tolerated and leads to either weak or inactive analogues. Replacing the hydroxymethyl group with an aminomethyl group (<b>18</b>) retains affinity cooperativity and efficacy cooperativity but with reduced allosteric binding affinity, rendering a less effective GPR68 PAM compared to <b>1</b>. The <i>N</i>-methyl-substituted aminomethyl (<b>19</b>) group further reduces allosteric activities and results in a weak PAM. Previously, we have demonstrated the importance of the position of the hydroxymethyl group: the <i>ortho</i>-hydroxymethyl isomer was much more effective than the <i>meta</i>-hydroxymethyl isomer to activate GPR68 G<sub>s</sub>–cAMP pathway.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> We have confirmed this trend by retesting <b>20</b> and by comparing the activities of <b>1</b> and <b>20</b>. This trend has also been confirmed by comparing the activities of <b>15</b> and <b>21</b> (two methyl ketone analogues). Based on these SAR results, we kept the ortho-positioned hydroxymethyl group in subsequent SAR studies and explored additional substituents, such as halides and alkoxy groups on the phenyl ring. First, we installed fluoro groups at 3- (<b>22</b>), 4- (<b>23</b>), 5- (<b>24</b>), and 6- (<b>25</b>) positions on the phenyl ring. Compared to <b>1</b>, the 3-fluoro analogue <b>22</b> is a less effective GPR68 PAM due to the decreased efficacy cooperativity and allosteric binding affinity. The 4-fluoro analogue <b>23</b> shows slightly increased PAM activity compared to compound <b>1</b>. Interestingly, the 6-fluoro analogue <b>25</b> has significantly improved affinity cooperativity (log(α) = 1.15 ± 0.14). However, its binding affinity (−log(<i>K</i><sub>B</sub>) = 3.69 ± 0.12) is greatly reduced, resulting in an overall slightly worse PAM (log(αβ/<i>K</i><sub>B</sub>) = 5.19 ± 0.03) than compound <b>1</b> (log(αβ/<i>K</i><sub>B</sub>) = 5.31 ± 0.05). The 5-fluoro analogue <b>24</b> is a slightly better PAM (log(αβ/<i>K</i><sub>B</sub>) = 5.69 ± 0.04) than <b>1</b> due to its slightly improved affinity cooperativity (log(α) = 0.91 ± 0.07) and allosteric binding affinity (−log(<i>K</i><sub>B</sub>) = 4.58 ± 0.08). Next, we explored compounds with a chloro group at 4- (<b>26</b>), 5- (<b>27</b>), and 6- (<b>28</b>) positions on the phenyl ring. Similar to <b>25</b>, all of these chloro analogues show improved affinity cooperativities (log(α) = 1.01 ± 0.08 (<b>26</b>); 1.20 ± 0.11 (<b>27</b>); and 1.25 ± 0.19 (<b>28</b>)). In addition, these compounds display improved allosteric binding affinities (−log(<i>K</i><sub>B</sub>) = 4.76 ± 0.14 (<b>26</b>); 4.49 ± 0.19 (<b>27</b>); and 4.43 ± 0.20 (<b>28</b>)) albeit reduced efficacy cooperativities, resulting in slightly better PAMs (2- to 4-fold) than <b>1</b>. We also explored analogues bearing a methoxy group at different substitution positions (<b>29</b>–<b>32</b>). A methoxy group in general leads to reduced allosteric index compared to <b>1</b>. Among the four substitution positions, the 4-position (<b>30</b>) is more tolerated while the 6-position (<b>32</b>) shows least PAM activity. Consequently, we installed ethoxy (<b>37</b>) and isopropoxy (<b>38</b>) groups at the 4-position to explore steric effects. The 4-ethoxy group (<b>37</b>) slightly improves affinity cooperativity (log(α) = 0.87 ± 0.11), maintains allosteric binding affinities, and slightly decreases efficacy cooperativity, resulting in a comparable allosteric index to <b>1</b>. The 4-isopropoxy group (<b>38</b>) shows slightly improved allosteric activity (log(αβ/<i>K</i><sub>B</sub>) = 5.74 ± 0.07) compared to <b>1</b> due to the increased allosteric binding affinities (−log(<i>K</i><sub>B</sub>) = 4.91 ± 0.12). These SAR results indicate that additional substituents on the benzyl alcohol moiety are tolerated, and appropriate substitutions on the phenyl ring could lead to enhanced allosteric activity. Several substituents were identified to improve the modulators’ affinity cooperativities and/or allosteric binding affinities. However, none of the substituents increased the modulators’ efficacy cooperativities.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Modifications of the Benzylamine Moiety</h3><div class="NLM_p">To evaluate the effects of the benzylamine moiety on allosteric activity, we designed a variety of compounds with differentially substituted benzylamines. In addition, we also explored different carbon linker length between the phenyl and amino groups (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). These compounds were prepared using nucleophilic aromatic substitution and Suzuki coupling reactions described in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>.</div><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0007.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>41</b>–<b>54</b>, <b>56</b>–<b>60</b>, and <b>64</b>–<b>81</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) dioxane, reflux, 1 h, 75–96 %; (b) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O = 5:3, 120 °C, microwave, 20 min, 16–99%.</p></p></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Effects of Modifications of the Benzylamine Side Chain on Allosteric Activity<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0012.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0013.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Values represent means ± SEM from a minimum of three independent assays, each in triplicate or quadruplicate. Δ stands for the difference of log(αβ/<i>K</i><sub>B</sub>) from that of <b>1</b>; positive values indicate increased allosteric activity, and negative values indicate reduced allosteric activity. Proton concentration–response curves are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 1</a>.</p></div></div><div></div></div><div class="NLM_p last">Initially, a series of electron-withdrawing groups (EWGs) such as fluoro (<b>41</b>–<b>43</b>), chloro (<b>44</b>–<b>46</b>), and trifluoromethyl (<b>47</b>–<b>49</b>) were introduced to the phenyl moiety to explore their effects at different substitution positions. 2-, 3-, and 4-Fluoro groups lead to slightly improved allosteric indexes mainly due to the increased affinity cooperativity with log(α) values at 0.98 ± 0.15, 0.70 ± 0.13, and 1.09 ± 0.30, respectively. Compared to the fluoro substitutions, the chloro substitutions provide more significant allosteric activity improvement because of the higher allosteric binding affinities (−log(<i>K</i><sub>B</sub>) = 5.07 ± 0.08 (<b>44</b>), 4.97 ± 0.19 (<b>45</b>), and 5.22 ± 0.07 (<b>46</b>)). Similarly, the trifluoromethyl analogues show improved allosteric index due to improved allosteric binding affinities (−log(<i>K</i><sub>B</sub>) = 5.05 ± 0.10 (<b>47</b>), 5.11 ± 0.12 (<b>48</b>), and 5.03 ± 0.08 (<b>49</b>)). Thus, EWGs are able to improve the modulators’ allosteric activities by increased affinity cooperativities or allosteric binding affinities. However, the substitution positions only have moderate effects on the allosteric indexes. Next, we explored a couple of electron-donating groups: methyl (<b>50</b>–<b>52</b>) and methoxy (<b>52</b>–<b>55</b>). The methyl analogues show slightly improved allosteric indexes mainly due to slightly higher allosteric binding affinity (<b>50</b>) or affinity cooperativity (<b>51</b> and <b>52</b>). Although the 2-, 3-, and 4-methoxy groups improve the affinity cooperativities (log(α) = 0.98 ± 0.17 (<b>53</b>), 0.89 ± 0.18 (<b>54</b>), and 0.78 ± 0.11 (<b>55</b>)), they reduce the allosteric binding affinities, resulting in analogues with reduced allosteric indexes. We subsequently explored a few larger alkoxy groups, such as ethoxy (<b>56</b>), isopropoxy (<b>57</b>), cyclopropoxy (<b>58</b>), and phenoxy (<b>59</b>), at the 4-position. Interestingly, most of these larger ether substituents diminish the efficacy cooperativities, leading to GPR68 α-PAMs, positive allosteric modulators with affinity cooperativities (α) greater than 1 and efficacy cooperativities (β) equal or close to 1. Compared to 4-methoxy group (<b>55</b>), these larger alkoxy groups enhance allosteric binding affinities (−Log(<i>K</i><sub>B</sub>) = 4.77 ± 0.03 (<b>56</b>), 4.99 ± 0.09 (<b>57</b>), 5.07 ± 0.12 (<b>58</b>), and 5.19 ± 0.14 (<b>59</b>)), leading to improved allosteric activities. In particular, the 4-phenoxy analogue (<b>59</b>) is an α-PAM with the most improved allosteric activity compared to compound <b>1</b>. Previously, we have shown that longer carbon linkers between the phenyl and amino groups resulted in analogues with higher PAM activity.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> We have now confirmed this SAR trend by testing compounds <b>60</b>–<b>63</b>. While a shorter linker (<b>60</b>) results in reduced allosteric index mainly due to the reduced allosteric binding affinity (−log(<i>K</i><sub>B</sub>) = 3.92 ± 0.10 (<b>60</b>)), longer linkers (<b>61</b>–<b>63</b>) enhance allosteric binding affinities (−log(<i>K</i><sub>B</sub>) = 4.50 ± 0.21 (<b>61</b>), 5.22 ± 0.04 (<b>62</b>), and 5.52 ± 0.12 (<b>63</b>)), leading to improved allosteric indexes. We also extended the carbon linker length of the 4-phenoxy-substituted benzylamine analogue <b>59</b>, leading to three analogues <b>64</b>–<b>66</b>. These analogues, however, do not show enhanced allosteric activities compared to <b>59</b>. Instead, longer linkers lead to reduced allosteric activities. Interestingly, these analogues all retain high affinity cooperativity α values. Thus, extending the carbon linker length between the phenyl group and amino group does not always lead to better PAMs. Importantly, these SAR results suggest that similar to the benzyl alcohol moiety, modifications of the benzylamine moiety could improve the modulators’ affinity cooperativities and/or allosteric binding affinities, but not their efficacy cooperativities.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Combined Modifications of Different Moieties</h3><div class="NLM_p">The SAR study described above indicates that several functional groups, such as 6-fluoro (<b>25</b>)- and 5-chloro (<b>27</b>)-substituted hydroxymethylphenyl moieties and 2-fluoro (<b>41</b>)-, 4-fluoro (<b>43</b>)-, 3-methyl (<b>51</b>)-, 2-methoxy (<b>53</b>)-, and 4-phenoxy (<b>59</b>)-substituted benzylamine, lead to high affinity cooperativities and/or improved allosteric binding affinities. Using these preferred moieties, we designed a number of combination compounds, to further improve the allosteric activity and obtain potent GPR68 α-PAMs. We also explored different linker lengths between the phenyl group and amino group (<i>n</i> = 1–4). Synthesis of these compounds is summarized in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, and SAR results of these compounds are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Effects of Combined Modifications on Allosteric Activity<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0014.gif" alt="" id="fx7" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th><th class="colsep0 rowsep0" align="center">R<sup>4</sup></th><th class="colsep0 rowsep0" align="center">R<sup>5</sup></th><th class="colsep0 rowsep0" align="center" char="±">Log(α)</th><th class="colsep0 rowsep0" align="center" char="±">Log(β)</th><th class="colsep0 rowsep0" align="center" char="±">–Log(<i>K</i><sub>B</sub>)</th><th class="colsep0 rowsep0" align="center" char="±">Log(αβ/<i>K</i><sub>B</sub>)</th><th class="colsep0 rowsep0" align="center" char=".">Δ</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" rowspan="6" align="char" valign="middle" char=".">1</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char="±">0.60 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">0.36 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">4.35 ± 0.12</td><td class="colsep0 rowsep0" align="char" char="±">5.31 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">0.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>67</b></td><td class="colsep0 rowsep0" rowspan="10" align="left" valign="middle">6-F</td><td class="colsep0 rowsep0" align="left">2-F</td><td class="colsep0 rowsep0" align="char" char="±">1.09 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">0.09 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">4.22 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">5.40 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">0.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>68</b></td><td class="colsep0 rowsep0" align="left">4-F</td><td class="colsep0 rowsep0" align="char" char="±">1.01 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">0.09 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">4.45 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">5.55 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">0.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>69</b></td><td class="colsep0 rowsep0" align="left">3-Me</td><td class="colsep0 rowsep0" align="char" char="±">0.84 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">0.14 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">4.44 ± 0.12</td><td class="colsep0 rowsep0" align="char" char="±">5.42 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">0.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>70</b></td><td class="colsep0 rowsep0" align="left">2-OMe</td><td class="colsep0 rowsep0" align="char" char="±">1.12 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">–0.12 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">4.11 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">5.11 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">–0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>71</b></td><td class="colsep0 rowsep0" align="left">4-OPh</td><td class="colsep0 rowsep0" align="char" char="±">0.90 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">0.06 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">5.85 ± 0.12</td><td class="colsep0 rowsep0" align="char" char="±">6.83 ± 0.08</td><td class="colsep0 rowsep0" align="char" char=".">1.52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>72</b></td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" rowspan="3" align="left" valign="middle">H</td><td class="colsep0 rowsep0" align="char" char="±">1.06 ± 0.15</td><td class="colsep0 rowsep0" align="char" char="±">0.03 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">5.04 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">6.13 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">0.82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>73</b></td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char="±">1.19 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">0.02 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">4.47 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">5.67 ± 0.10</td><td class="colsep0 rowsep0" align="char" char=".">0.36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>74</b></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char="±">1.24 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">0.06 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">4.04 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">5.34 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>75</b></td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" rowspan="2" align="left" valign="middle">4-OPh</td><td class="colsep0 rowsep0" align="char" char="±">1.14 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">–0.06 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">5.05 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">6.14 ± 0.10</td><td class="colsep0 rowsep0" align="char" char=".">0.82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>76</b></td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char="±">1.15 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">0.01 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">4.94 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">6.10 ± 0.11</td><td class="colsep0 rowsep0" align="char" char=".">0.81</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>77</b></td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" rowspan="5" align="left" valign="middle">5-Cl</td><td class="colsep0 rowsep0" rowspan="3" align="left" valign="middle">H</td><td class="colsep0 rowsep0" align="char" char="±">1.09 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">0.03 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">5.14 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">6.26 ± 0.09</td><td class="colsep0 rowsep0" align="char" char=".">0.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>78</b></td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char="±">1.03 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">0.01 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">5.35 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">6.38 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">1.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>79</b></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char="±">1.09 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">0.06 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">4.98 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">6.14 ± 0.10</td><td class="colsep0 rowsep0" align="char" char=".">0.83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>80</b></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" rowspan="2" align="left" valign="middle">4-OPh</td><td class="colsep0 rowsep0" align="char" char="±">1.05 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">0.03 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">4.70 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">5.78 ± 0.11</td><td class="colsep0 rowsep0" align="char" char=".">0.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>81</b></td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char="±">1.39 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">–0.04 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">4.59 ± 0.18</td><td class="colsep0 rowsep0" align="char" char="±">5.87 ± 0.14</td><td class="colsep0 rowsep0" align="char" char=".">0.56</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Values represent means ± SEM from a minimum of three independent assays, each in triplicate or quadruplicate. Δ stands for the difference of Log(αβ/<i>K</i><sub>B</sub>) from that of <b>1</b>; positive values indicate increased allosteric activity, and negative values indicate reduced allosteric activity. Proton concentration–response curves are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 1</a>.</p></div></div></div><div class="NLM_p">Compared to <b>25</b>, 2-fluoro (<b>67</b>)-, 4-fluoro (<b>68</b>)-, 3-methyl (<b>69</b>)-, 2-methoxy (<b>70</b>)-, and 4-phenoxy (<b>71</b>)-substituted benzyl groups retain relatively high affinity cooperativity values and show low efficacy cooperativity values. Most strikingly, the 4-phenoxy analogue <b>71</b> displays greatly enhanced allosteric binding affinity (−log(<i>K</i><sub>B</sub>) = 5.85 ± 0.12 vs 3.69 ± 0.12 for compound <b>25</b>), resulting in a very potent GPR68 α-PAM. Compared to <b>1</b>, compound <b>71</b> has 33-fold better allosteric activity (log(αβ/<i>K</i><sub>B</sub>) = 6.83 ± 0.08 vs 5.31 ± 0.05 for compound <b>1</b>). We have shown that the linker length between the phenyl and amino groups has different effects on the allosteric activities, depending on different R5 groups (<b>60</b>–<b>63</b> and <b>59</b>, <b>64</b> and <b>65</b> (see above)). Extending the carbon linker length of <b>25</b> to 2–4 results in compounds <b>72</b>–<b>74</b>. Compared to <b>25</b>, all of these longer linker analogues maintain high affinity cooperativity values and show low efficacy cooperativity values. For allosteric binding affinity, the 2-carbon linker (<b>72</b>, −Log(<i>K</i><sub>B</sub>) = 5.04 ± 0.09) is optimal among the 4 linker lengths explored. Extending the carbon linker length of <b>71</b> to 2–3 leads to α-PAMs <b>75</b> and <b>76</b> with retained high affinity cooperativities, low efficacy cooperativities, but reduced allosteric binding affinities.</div><div class="NLM_p">To improve the allosteric activity of <b>27</b>, we also increased the linker length between the phenyl group and the amino group. The longer linker analogues <b>77</b>–<b>79</b> maintain high affinity cooperativities and low efficacy cooperativities and enhance allosteric binding affinities (−log(<i>K</i><sub>B</sub>) = 5.14 ± 0.11 (<b>77</b>), 5.35 ± 0.06 (<b>78</b>), and 4.98 ± 0.10 (<b>79</b>)) with the 3-carbon linker (<b>78</b>) being optimal. Finally, we replaced the benzylamine moiety of <b>27</b> with a 4-phenoxy substituted benzylamine moiety (<b>80</b>) or a 4-phenoxy-substituted homobenzylamine moiety (<b>81</b>). Compounds <b>80</b> and <b>81</b> preserve the α-PAM activity. Compared to <b>27</b>, both compounds show similar allosteric binding affinities (−log(<i>K</i><sub>B</sub>) = 4.70 ± 0.14 (<b>80</b>) and 4.59 ± 0.18 (<b>81</b>)) and similar overall allosteric activities (i.e., log(αβ/<i>K</i><sub>B</sub>) values).</div><div class="NLM_p last">Our SAR study has identified several GPR68 PAMs with better allosteric activities than compound <b>1</b>. For example, <b>46</b>, an αβ-PAM (α and β > 1), has shown 6-fold higher allosteric activity (log(αβ/<i>K</i><sub>B</sub>) value) than <b>1</b>. α-PAM <b>71</b> has the highest log(αβ/<i>K</i><sub>B</sub>) value, which is 33-fold higher than that of <b>1</b>. Compounds <b>77</b> and <b>78</b>, α-PAMs, have approximately 10-fold better allosteric activity than <b>1</b>. We selected these four GPR68 PAMs for further characterization.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Selectivity Profiles of Selected GPR68 PAMs</h3><div class="NLM_p">To determine selectivities of <b>46</b>, <b>71</b>, <b>77</b>, and <b>78</b> for GPR68 over other closely related proton GPCRs, such as GPR4 and GPR65, we tested these GPR68 PAMs against GPR4 and GPR65 using GloSensor cAMP assays. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A–D, GPR4 and GPR65 have different response profiles to proton stimulation compared to GPR68 in the absence of modulators. The proton potency values (pEC<sub>50</sub>) were pH 8.04 (Hill slope of 5.25) for GPR4, pH 7.47 (Hill slope of 4.57) for GPR65, and pH 6.73 (Hill slope of 3.38) for GPR68. In the presence of the lead PAM <b>71</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B) as well as <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), proton activity at GPR68 is potentiated in a PAM concentration-dependent manner (proton concentration–response curves were shifted leftward (affinity cooperativity α > 1) and upward (efficacy cooperativity β > 1)), while compound <b>71</b> has no PAM activity at GPR4 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C) and weak potentiation of proton activity at GPR65 only at high concentrations (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). These data indicate that our lead PAM <b>71</b> is selective for GPR68 over the functionally related receptors GPR4 and GPR65. Similarly, as illustrated in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 2</a>, compounds <b>46</b>, <b>77</b>, and <b>78</b> show no effect or weak activity only at high concentrations at GPR4 and GPR65.</div><figure id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Selectivity of the GPR68 PAM 71. Proton concentration–response curves in the absence and presence of increasing concentrations of <b>1</b> at human GPR68 (A), <b>71</b> at human GPR68 (B), GPR4 (C), and GPR65 (D). Proton-mediated G<sub>s</sub>–cAMP production was measured in HEK293T cells transiently transfected with GPR68, GPR4, or GPR65, and GloSensor cAMP plasmids. Results were normalized to the proton activity in the absence of modulator and represented means ± SEM from a minimum of three independent assays, each in quadruplicate. Curves were analyzed in Prism using the built-in four parameter logistic functions. (E) Selectivity of <b>71</b> over 48 common GPCRs, ion channels, and transporters in radioligand binding assays. Values represent average binding affinity (p<i>K</i><sub>i</sub>) from a minimum of three independent assays, each in triplicate. Targets with less than 50% inhibition at 10 μM (in quadruplicate) are not shown. BZP for benzodiazepine receptor; DAT for dopamine transporter; DOR for delta (δ) opioid receptor; hERG for human ether-a-go-go-related potassium channel; KOR for kappa (κ) opioid receptor; MOR for mu (μ) opioid receptor; NET for noradrenaline transporter; PBR for peripheral benzodiazepine binding site; SERT for serotonin transporter. (F) Off-target agonist activity of <b>71</b> at human GPCRome. <b>71</b> was screened across the human GPCRome (318 GPCRs) for off-target agonist activity at 1 μM using the PRESTO-Tango screening platform. Results (fold of basal) represent means of four replicates. Dopamine receptor D<sub>2</sub> with 100 nM quinpirole served as an assay control. The dashed lines indicate an arbitrary cutoff line at 3× fold of basal.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compounds <b>46</b>, <b>71</b>, <b>77</b>, and <b>78</b> were further screened in radioligand binding assays against a panel of 48 common drug targets, including GPCRs, neurotransmitter transporters, and hERG ion channels. As indicated in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 3</a>, most of these compounds have affinity values higher than 300 nM (p<i>K</i><sub>i</sub> = 6.50) at all targets tested. Among the tested GPCRs, 5-HT<sub>2B</sub> has moderate binding affinity (p<i>K</i><sub>i</sub> = 6.50–7.00) to <b>71</b>, <b>77</b>, and <b>78</b>, with <i>K</i><sub>i</sub> values of 219, 209, and 295 nM, respectively. In addition, adenosine 2A (A<sub>2A</sub>) receptor has moderate binding affinity to <b>46</b> with <i>K</i><sub>i</sub> value of 300 nM. The compounds with binding affinities between 300 and 100 nM (p<i>K</i><sub>i</sub> = 6.50–7.00) were further tested in functional assays to determine if they acted as agonists or antagonists. At 5-HT<sub>2B</sub> receptors, <b>71</b>, <b>77</b>, and <b>78</b> show no agonist activity but display weak antagonist activity (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 4A,B</a>) with functional affinity (<i>K</i><sub>i</sub>) values of 310, 678, and 212 nM, respectively, which are comparable to their corresponding binding affinities. At A<sub>2A</sub> receptors (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 4C</a>), <b>46</b> displays no agonist activity, but shows weak inverse agonist activity with potency of 1700 nM in the absence of reference agonist and antagonist activity with functional affinity of 6700 nM.</div><div class="NLM_p last">For lead PAMs with the potential to serve as molecular probes for chemical biology studies, it is also desirable to determine their off-target agonist activities at other human GPCRs. Therefore, <b>46</b>, <b>71</b>, <b>77</b>, and <b>78</b> were tested against all human GPCRome in our PRESTO-Tango screening assays.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> As illustrated in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F, across the whole human GPCRome, compound <b>71</b> has agonist activity only at the MT<sub>1</sub> and MT<sub>2</sub> receptors. Compound <b>46</b> shows no agonist activity, while compounds <b>77</b> and <b>78</b> display agonist activity at an atypical chemokine receptor 3 (ACKR3) (previously known as CXCR7)<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 5</a>). Subsequent concentration–response assays indicate that <b>71</b> is a weak full agonist at the MT<sub>1</sub> receptor (EC<sub>50</sub> = 320 nM) and a weak partial agonist activity at the MT<sub>2</sub> receptor (EC<sub>50</sub> = 540 nM; <i>E</i><sub>max</sub> = 24%) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 6</a>). Consistently, <b>71</b> displays low binding affinities to MT<sub>1</sub> (5900 nM) and MT<sub>2</sub> (1100 nM) receptors. Compounds <b>77</b> and <b>78</b> show full agonist activities at ACKR3 with EC<sub>50</sub> values of 1700 and 690 nM, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 6</a>).</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Characterization of Selected Lead Modulators at Mouse GPR68</h3><div class="NLM_p">To assess whether selected human GPR68 PAMs also function as PAMs at mouse GPR68 receptor, we determined the PAM activity at mouse GPR68 in the absence and presence of lead modulators for proton-mediated G<sub>s</sub>–cAMP production. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 7</a>, in the absence of modulators, mouse GPR68 responds to proton stimulation in a very similar way as human GPR68 with a potency of pH 6.68 (Hill slope of 2.95). As with human GPR68, the proton activity at mouse GPR68 is also potentiated by the lead PAMs <b>71</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), <b>46</b>, <b>77</b>, and <b>78</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 7</a>) in a concentration-dependent manner—shifting proton response curves to the left with increased maxima. Extracted allosteric parameters determined by fitting the curves to the operational allosteric model are listed in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. Compared to the allosteric activity of these PAMs at human GPCR68, the compounds display 2- to 3-fold less allosteric activity at mouse GPR68, suggesting a potential small species difference in sensitivity to GPR68 PAMs.</div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Allosteric potentiation of proton activity by compound <b>71</b> at mouse GPR68. G<sub>s</sub> GloSensor cAMP assays were carried out in HEK293T cells transiently transfected with mouse GPR68 as with human GPR68. Proton concentration responses were measured in the absence and presence of increasing concentrations of modulator. Results were normalized to proton activity in the absence of modulator and represented means ± SEM from a minimum of three independent assays, each in quadruplicate. Curves were analyzed in Prism. Allosteric parameters are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Allosteric Parameters of Selected PAMs at Mouse GPR68<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">compd</th><th class="colsep0 rowsep0" align="center" char="±">Log(α)</th><th class="colsep0 rowsep0" align="center" char="±">Log(β)</th><th class="colsep0 rowsep0" align="center" char="±">–Log(<i>K</i><sub>B</sub>)</th><th class="colsep0 rowsep0" align="center" char="±">Log(αβ/<i>K</i><sub>B</sub>)</th><th class="colsep0 rowsep0" align="center" char=".">Δ</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>1</b></td><td class="colsep0 rowsep0" align="char" char="±">0.80 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">0.04 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">4.13 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">4.97 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">–0.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>46</b></td><td class="colsep0 rowsep0" align="char" char="±">1.41 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">–0.0 2 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">4.30 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">5.68 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">–0.43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>63</b></td><td class="colsep0 rowsep0" align="char" char="±">0.91 ± 0.18</td><td class="colsep0 rowsep0" align="char" char="±">0.08 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">4.91 ± 0.17</td><td class="colsep0 rowsep0" align="char" char="±">5.89 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">–0.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>71</b></td><td class="colsep0 rowsep0" align="char" char="±">1.26 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">0.09 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">5.12 ± 0.19</td><td class="colsep0 rowsep0" align="char" char="±">6.46 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">–0.43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>77</b></td><td class="colsep0 rowsep0" align="char" char="±">1.06 ± 0.12</td><td class="colsep0 rowsep0" align="char" char="±">0.13 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">4.72 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">5.92 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">–0.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>78</b></td><td class="colsep0 rowsep0" align="char" char="±">0.99 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">0.13 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">4.86 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">5.98 ± 0.04</td><td class="colsep0 rowsep0" align="char" char=".">–0.40</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Values represent means ± SEM from a minimum of three independent assays, each in quadruplicate. Δ stands for the difference of Log(αβ/<i>K</i><sub>B</sub>) from the corresponding values at human GPR68 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>); positive values indicate increased allosteric activity, and negative values indicate reduced allosteric activity. Proton concentration–response curves are presented in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 7</a>.</p></div></div></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> In Vivo Pharmacokinetic (PK) Properties of Compound <b>71</b></h3><div class="NLM_p">After obtaining the promising in vitro potency and selectivity results for compound <b>71</b>, we assessed in vivo pharmacokinetic properties of this lead compound in mice. Since GPR68 is most abundantly expressed in the brain and is associated with learning and memory, it is necessary to test whether this compound can penetrate the blood–brain barrier (BBB) in animals. A single intraperitoneal injection at the dose of 25 mg/kg led to high exposure levels (above 10 μM) in both plasma and brain at 0.5 h in mice (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The high compound exposure levels in both plasma and brain were maintained for 2 h. These results have demonstrated that compound <b>71</b> is bioavailable and can readily cross the BBB in mice. Furthermore, there were no clinical signs observed in test mice. Taken together, these results suggest that compound <b>71</b> is a valuable chemical tool for investigating biological function of GPR68 in vivo.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Mouse pharmacokinetic profile of compound <b>71</b>. Compound concentrations were quantified in both plasma (red curve) and brain (blue curve) at 0.5, 1.0, and 2.0 h after a single 25 mg/kg intraperitoneal (IP) administration. Compound concentrations in plasma and brain at each time point are average values from three test mice. The error bars represent ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Characterization of Selected Inactive Analogues</h3><div class="NLM_p last">Compounds <b>7</b>, <b>20</b> (a known inactive analogue of <b>1</b>),<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and <b>21</b> do not show PAM activity at GPR68, GPR4, and/or GPR65 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 8</a>), proton responses are left-shifted by <b>71</b> as expected and are minimally affected by the presence of the inactive analogues, suggesting that the inactive analogues do not bind to the allosteric site of GPR68 to antagonize the PAM activity of <b>71</b>. In addition, inactive analogues <b>7</b> and <b>21</b> show minimal binding affinities to common drug targets (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 3</a>). Moreover, <b>7</b> and <b>21</b> display no off-target agonist activity across the human GPCRome (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Figure 5</a>). Therefore, compounds <b>7</b>, <b>20</b>, and <b>21</b> are potentially useful inactive controls for studying GPR68 pharmacology.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35878" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35878" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">It is worth noting the unique nature of GPR68 pharmacology as the co-binding ligand for the allosteric effect, namely, hydrogen ion, is always present. Therefore, GPR68 allosteric modulators will likely have direct consequences to cellular signaling: PAMs will produce direct agonism, while negative allosteric modulators will produce inverse agonism. In addition, differences exist for PAMs depending on whether the allosterism primarily affects affinity (α-effect) or efficacy (β-effect). Specifically, α-PAMs (α > 1 and β ≈ 1) will produce greater agonism at neutral pH conditions while β-PAMs (α ≈ 1 and β > 1) will produce relatively more agonism in acidic environments. The differences in signaling produced by α- versus β-modulators are described in detail in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">Supporting Information</a>. In the present study, we are able to achieve potent and selective GPR68 α-PAMs (e.g., compounds <b>71</b>, <b>77</b>, and <b>78</b>) and αβ-PAMs (e.g., compound <b>46</b>), but not β-PAMs.<a onclick="showRef(event, 'ref50 ref51 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref50 ref51 ref52 ref53">(50−53)</a></div><div class="NLM_p">There are examples that precipitous changes in allosteric activity may come from apparently small changes in molecular structure, known specifically as “activity switching” in the PAMs for mGluRs.<a onclick="showRef(event, 'ref54 ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref54 ref55 ref56">(54−56)</a> In this SAR study, with compound <b>1</b> as a scaffold, we also observed large changes in allosteric activity (from active PAMs to inactive analogues) with small structural modifications to the aminotriazine core and benzyl alcohol group. Further SAR and mutational studies are needed to elucidate underlying molecular mechanisms, which will aid in designing better modulators.</div><div class="NLM_p">Depending on the p<i>K</i><sub>a</sub> values, the physiochemical properties of modulators, such as net charges and H-bonding potential, could change under different pH conditions. Correspondingly, binding interactions of modulators with GPR68 could be pH-dependent, leading to different levels of receptor activation in both pH- and concentration-dependent manner. Therefore, our functional assay is designed to use a wide pH range and is done in the absence and presence of increasing concentrations of modulators to generate at least six sets of 16-point proton concentration–response curves (each in quadruplicate) in a single 384-well plate for proper comparison.</div><div class="NLM_p">To achieve novel brain-penetrant GPR68 PAMs, we tried to maintain low tPSA values by utilizing lipophilic substituents on the triazine scaffold. While this strategy successfully led to BBB permeable compound <b>71</b>, these lipophilic groups could also reduce the aqueous solubility of the compounds. Indeed, we have observed bell-shaped proton concentration–response curves at high compound concentrations (10 and/or 30 μM) for compounds <b>57</b>–<b>59</b>, <b>63</b>–<b>66</b>, <b>75</b>–<b>78</b>, and <b>81</b>. The assay signals waned when modulators were tested at pH < 7.40. This is likely due to the reduced aqueous solubility of these compounds under acidic conditions or aggregation.<a onclick="showRef(event, 'ref57 ref58'); return false;" href="javascript:void(0);" class="ref ref57 ref58">(57,58)</a> Addition of 1 mg/mL bovine serum albumin (BSA) in assay buffers may have improved solubility but did not affect the bell-shaped curves. Under these circumstances, we may have slightly underestimated the allosteric parameters of these compounds, especially with regard to the efficacy cooperativity β. In any case, this effect prevented high concentrations of modulators from being included in the functional assays. This, in turn, prevented observation of the characteristic saturation (termed the “ceiling effect”) of allosteric modulation, further leading to increased levels of variation in estimated allosteric parameters.</div><div class="NLM_p last">In summary, we have performed a comprehensive SAR study by exploring three regions of the scaffold represented by compound <b>1</b>. While structural changes on the aminotriazine core did not improve the allosteric activity, modifications to the benzyl alcohol and benzylamino groups did lead to slightly more active GPR68 PAMs with increased affinity cooperativities and allosteric binding affinities. The subsequently designed combination compounds with the substituents showing best α and <i>K</i><sub>B</sub> values from each region ultimately resulted in much improved GPR68 α-PAMs, including compound <b>71</b>, which has 33-fold increased allosteric activity compared to <b>1</b>. We also identified a significantly improved αβ-PAM, compound <b>46</b>. We further determined that compounds <b>71</b> and <b>46</b> and two additional GPR68 PAMs were selective for GPR68 over GPR4 and GPR65, two closely related proton GPCRs. In addition, these four compounds had no or low off-target agonist activity across the human GPCRome and had weak to moderate binding affinity to 48 common GPCRs, ion channels, and transporters. Moreover, we demonstrated that our lead compound, <b>71</b>, is bioavailable in mice and has excellent brain penetration. Finally, we identified several potentially useful inactive controls. These novel GPR68 PAMs and inactive controls are valuable chemical tools for studying GPR68 biology and pharmacology. Furthermore, our study paved the way for developing the next generation of GPR68 PAMs and allosteric modulators for other GPCRs.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12678" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12678" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Chemistry General Procedures</h3><div class="NLM_p last">All chemical reagents were purchased from commercial vendors and used without further purification. Dry solvents were purchased from commercial vendors with a verified anhydrous standard. Microwave-heated reactions were performed with a Discover SP microwave system with an Explorer 12 Hybrid Autosampler by CEM (Buckingham, U.K.). EMD Millipore 210–270 μm 60-F<sub>254</sub> silica gel plates were used to perform analytical thin-layer chromatography (TLC). The plates were examined under a UV Lamp. The flash column chromatography was conducted using a Teledyne ISCO CombiFlash Rf<sup>+</sup> instrument. This instrument was also equipped with a variable-wavelength UV detector and a fraction collector. RediSep Rf normal phase silica columns were used for purification. Preparative high-performance liquid chromatography (HPLC) was used to purify all final compounds. HPLC was conducted on an Agilent Prep 1200 series with the UV detector set to 254 nm. Samples were injected into a Phenomenex Luna 750 × 30 mm<sup>2</sup>, 5 μm C18 column. Separation was performed at room temperature with a flow rate of 40 mL/min. The linear gradient program was set as follows: 10% methanol (A) in H<sub>2</sub>O containing 0.1% trifluoroacetic acid (B) to 100% methanol (A). All compounds for biological evaluation were determined to have >95% purity after the purification using the aforementioned prep-HPLC methods. An Agilent 1200 series system with diode array detector and a 2.1 mm × 150 mm Zorbax 300SB-C18 5 μm column for chromatography were used to obtain HPLC spectra for all compounds. Samples (0.5 μL) were injected onto a C18 column at room temperature. The column flow rate was set to 0.4 mL/min. Chromatography was performed with water containing 0.1% formic acid as Solvent A and acetonitrile-containing 0.1% formic acid as solvent B. The linear gradient was set as follows: 1% B (0–1 min), 1–99% B (1–4 min), and 99% B (4–8 min). An Agilent G1969A atmospheric pressure interface time of flight with an electrospray ionization source was used to acquire high-resolution mass spectra data in positive-ion mode. All compounds were also characterized using a Bruker (Billerica, MA) DRX-600 nuclear magnetic resonance (NMR) spectrometer (600 MHz, <sup>1</sup>H NMR). Chemical shifts are reported in parts per million (δ) relative to residual solvent peaks. <sup>1</sup>H NMR data are reported as follows: chemical shift, multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet), coupling constant, and integration.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> General Procedures for Preparation of Compounds <b>5</b>–<b>7</b></h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (2-(4-(Benzylamino)-6-(methylamino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>5</b>)<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a></h4><div class="NLM_p last">Intermediate <b>3</b> was prepared according to previously published procedures starting with 4,6-dichloro-<i>N</i>-methyl-1,3,5-triazin-2-amine (<b>2</b>) and benzylamine. The white solid <b>3</b> (24.9 mg, 0.1 mmol) was suspended in dioxane (0.8 mL) and water (0.2 mL) in a microwave tube. Then, tetrakis(triphenylphosphine)palladium(0) (5.8 mg, 5 mol %), potassium carbonate (27.6 mg, 0.2 mmol), and (2-(hydroxymethyl)phenyl)boronic acid (30.4 mg, 0.2 mmol) were added. The mixture was heated to 120 °C for 20 min under microwave irradiation. The resulting mixture was filtered through Celite with ethyl acetate as eluent solvent. The filter was washed with water and extracted with ethyl acetate. The combined organic layers were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The resulting residue was dissolved in 1.5 mL of methanol and purified by preparative HPLC to yield <b>7</b> as a white solid (27 mg, 84%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.81 (d, <i>J</i> = 7.8 Hz, 1H), 7.60 (dd, <i>J</i> = 20.0, 7.4 Hz, 3H), 7.50 (t, <i>J</i> = 7.6 Hz, 1H), 7.43 (d, <i>J</i> = 7.6 Hz, 1H), 7.35 (t, <i>J</i> = 7.8 Hz, 2H), 7.31–7.24 (m, 1H), 4.81 (s, 2H), 4.72 (d, <i>J</i> = 5.1 Hz, 2H), 3.05 (d, <i>J</i> = 5.3 Hz, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>20</sub>N<sub>5</sub>O<sup>+</sup> 322.1662, found 322.1678.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (2-(4-(Benzylamino)-6-(ethylamino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>6</b>)</h4><div class="NLM_p last">White solid (15.2 mg, 98%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.83 (d, <i>J</i> = 7.8 Hz, 1H), 7.69–7.57 (m, 2H), 7.51 (t, <i>J</i> = 7.6 Hz, 1H), 7.35 (ddt, <i>J</i> = 41.4, 34.0, 7.3 Hz, 5H), 4.81 (s, 2H), 4.70 (s, 2H), 3.54 (q, <i>J</i> = 7.2 Hz, 2H), 1.22 (t, <i>J</i> = 7.2 Hz, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>22</sub>N<sub>5</sub>O<sup>+</sup> 336.1819, found 336.1873.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (2-(4-(Benzylamino)-6-(dimethylamino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>7</b>)</h4><div class="NLM_p last">White solid (32 mg, 42%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.90 (d, <i>J</i> = 7.8 Hz, 1H), 7.62 (dd, <i>J</i> = 24.7, 7.5 Hz, 2H), 7.52 (t, <i>J</i> = 7.6 Hz, 1H), 7.42 (d, <i>J</i> = 7.5 Hz, 2H), 7.35 (t, <i>J</i> = 7.4 Hz, 2H), 7.28 (t, <i>J</i> = 7.3 Hz, 1H), 4.84 (s, 2H), 4.70 (s, 2H), 3.32 (d, <i>J</i> = 24.3 Hz, 6H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>22</sub>N<sub>5</sub>O<sup>+</sup> 336.1819, found 336.1832.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (2-(6-Amino-2-(benzylamino)pyrimidin-4-yl)phenyl)methanol (<b>10</b>)</h4><div class="NLM_p last">Compound <b>10</b> was synthesized following the same procedure for preparing <b>5</b> starting with 2,6-dichloropyrimidin-4-amine <b>8</b> and benzylamine. The title compound <b>10</b> was obtained as a white solid (27.2 mg, 92%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.57 (d, <i>J</i> = 5.2 Hz, 2H), 7.53–7.43 (m, 2H), 7.39 (d, <i>J</i> = 7.6 Hz, 2H), 7.34 (t, <i>J</i> = 7.6 Hz, 2H), 7.27 (t, <i>J</i> = 7.3 Hz, 1H), 6.20 (s, 1H), 4.66 (s, 2H), 4.62 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>19</sub>N<sub>4</sub>O<sup>+</sup> 307.1553, found 307.1542.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> General Procedures for Preparation of Compounds <b>13</b>–<b>15</b></h3><div class="NLM_p">Intermediate <b>12</b> was prepared according to previously published procedures.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>N</i><sup>2</sup>-Benzyl-6-phenyl-1,3,5-triazine-2,4-diamine (<b>13</b>)</h4><div class="NLM_p last">To a dried microwave tube were added intermediate <b>12</b> (23.6 mg, 0.1 mmol), phenylboronic acid (24.4 mg, 0.2 mmol), tetrakis(triphenylphosphine)palladium (5.8 mg, 5 mol %), potassium carbonate (27.6 mg, 0.2 mmol) dioxane (0.8 mL), and water (0.2 mL). The mixture was heated to 120 °C for 20 min under microwave irradiation. The resulting mixture was filtered through Celite with ethyl acetate as eluent solvent. The filter was washed with water and extracted with ethyl acetate. The combined organic layers were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The resulting residue was dissolved in 1.5 mL of methanol and purified by preparative HPLC to yield <b>13</b> as a white solid (26 mg, 94%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.15 (t, <i>J</i> = 6.3 Hz, 2H), 7.69 (q, <i>J</i> = 8.9, 8.2 Hz, 1H), 7.57 (dt, <i>J</i> = 11.2, 7.5 Hz, 2H), 7.42–7.30 (m, 4H), 7.27 (q, <i>J</i> = 6.9 Hz, 1H), 4.69 (d, <i>J</i> = 3.8 Hz, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>16</sub>N<sub>5</sub><sup>+</sup> 278.1400, found 278.1415.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i><sup>2</sup>-Benzyl-6-(<i>o</i>-tolyl)-1,3,5-triazine-2,4-diamine (<b>14</b>)</h4><div class="NLM_p last">White solid (29 mg, 98%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.60 (dd, <i>J</i> = 12.1, 7.8 Hz, 1H), 7.51 (t, <i>J</i> = 7.6 Hz, 1H), 7.43–7.31 (m, 6H), 7.28 (q, <i>J</i> = 7.2, 5.7 Hz, 1H), 4.69 (s, 2H), 2.52 (s, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>18</sub>N<sub>5</sub><sup>+</sup> 292.1557, found 292.1539.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 1-(2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)phenyl)ethan-1-one (<b>15</b>)</h4><div class="NLM_p last">White solid (48 mg, 54%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.06 (d, <i>J</i> = 7.7 Hz, 1H), 7.89 (t, <i>J</i> = 7.9 Hz, 1H), 7.80 (dd, <i>J</i> = 7.7, 3.2 Hz, 1H), 7.73 (dt, <i>J</i> = 11.0, 7.6 Hz, 1H), 7.44–7.23 (m, 5H), 4.71 (s, 2H), 2.01 (d, <i>J</i> = 2.4 Hz, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>N<sub>5</sub>O<sup>+</sup> 320.1506, found 320.1531.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 1-(2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)phenyl)ethan-1-ol (<b>16</b>)</h4><div class="NLM_p last">Compound <b>15</b> (15 mg, 0.05 mmol) was dissolved in THF (mL). After the mixture was cooled to 0 °C, sodium borohydride (4 mg, 0.1 mmol) was added slowly. The reaction was monitored by TLC until <b>15</b> disappeared. Methanol (1 mL) was added slowly to quench the reaction. The mixture was filtered and concentrated in vacuo. The resulting residue was dissolved in 1.5 mL of methanol and purified by preparative HPLC to yield <b>16</b> as a white solid (16 mg, 51%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.76–7.59 (m, 3H), 7.52–7.44 (m, 2H), 7.43–7.31 (m, 4H), 7.28 (dd, <i>J</i> = 9.2, 5.7 Hz, 1H), 5.27–5.18 (m, 1H), 4.70 (s, 2H), 1.53 (d, <i>J</i> = 6.5 Hz, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>20</sub>N<sub>5</sub>O<sup>+</sup> 322.1662, found 322.1669.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 6-(2-(Aminomethyl)phenyl)-<i>N</i><sup>2</sup>-benzyl-1,3,5-triazine-2,4-diamine (<b>18</b>)</h4><div class="NLM_p last">Intermediate <b>17</b> (123 mg, 86%) was prepared following the same procedures that were used to prepare <b>13</b> starting with 2-formylphenyl boronic acid and intermediate <b>12</b>. To a suspension of intermediate <b>17</b> (47 mg, 0.15 mmol), magnesium sulfate (200 mg) and NH<sub>3</sub>·H<sub>2</sub>O (0.75 mmol, 107 μL) in THF (2 mL) was added sodium borohydride (5.7 mg, 0.15 mmol) slowly. After stirring for 2 days at room temperature, methanol (1 mL) was added slowly to quench the reaction. The mixture was filtered and concentrated in vacuo. The resulting residue was dissolved in 1.5 mL of methanol and purified by preparative HPLC to yield <b>18</b> as yellow oil (21 mg, 45%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.83 (d, <i>J</i> = 7.8 Hz, 1H), 7.64 (t, <i>J</i> = 7.5 Hz, 1H), 7.60 (d, <i>J</i> = 7.5 Hz, 1H), 7.51 (q, <i>J</i> = 7.0, 6.5 Hz, 1H), 7.40 (d, <i>J</i> = 7.6 Hz, 1H), 7.37–7.31 (m, 3H), 7.29 (q, <i>J</i> = 7.9, 7.2 Hz, 1H), 4.82 (s, 2H), 4.69 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>19</sub>N<sub>6</sub><sup>+</sup> 307.1666, found 307.1676.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i><sup>2</sup>-Benzyl-6-(2-((methylamino)methyl)phenyl)-1,3,5-triazine-2,4-diamine (<b>19</b>)</h4><div class="NLM_p last">Compound <b>19</b> was prepared following the general procedure for preparing compound <b>18</b> from intermediate <b>17</b> and methylamine (2 M in THF). Yellow oil (22 mg, 67%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.69 (dd, <i>J</i> = 7.6, 3.7 Hz, 1H), 7.66–7.49 (m, 3H), 7.39–7.15 (m, 5H), 4.62 (s, 2H), 4.39 (s, 2H), 2.79 (s, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>21</sub>N<sub>6</sub><sup>+</sup> 321.1882, found 321.1877.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> General Procedures for Preparation of Compounds <b>21</b>–<b>29</b>, <b>31</b>, and <b>32</b></h3><div class="NLM_p">Compounds <b>21</b>–<b>19</b>, <b>31</b> and <b>32</b> were prepared following the general procedure for preparing compound <b>13</b> from intermediate <b>12</b> and commercially available substituted phenylboronic acids.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 1-(3-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)phenyl)ethan-1-one (<b>21</b>)</h4><div class="NLM_p last">White solid (28 mg, 87%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 8.80 (s, 1H), 8.44 (d, <i>J</i> = 7.8 Hz, 1H), 8.32–8.20 (m, 1H), 7.78–7.63 (m, 1H), 7.47–7.19 (m, 5H), 4.69 (s, 2H), 2.67 (s, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>N<sub>5</sub>O<sup>+</sup> 320.1506, found 320.1522.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)-6-fluorophenyl)methanol (<b>22</b>)</h4><div class="NLM_p last">White solid (29 mg, 89%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.63 (d, <i>J</i> = 7.8 Hz, 1H), 7.53 (td, <i>J</i> = 8.0, 5.1 Hz, 1H), 7.44–7.32 (m, 5H), 7.28 (t, <i>J</i> = 7.5 Hz, 1H), 4.83 (s, 2H), 4.69 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>FN<sub>5</sub>O<sup>+</sup> 326.1412, found 326.1435.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)-5-fluorophenyl)methanol (<b>23</b>)</h4><div class="NLM_p last">White solid (30 mg, 91%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.86 (dt, <i>J</i> = 10.6, 5.3 Hz, 1H), 7.43–7.30 (m, 5H), 7.30–7.15 (m, 2H), 4.84 (d, <i>J</i> = 5.1 Hz, 2H), 4.67 (d, <i>J</i> = 5.0 Hz, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>FN<sub>5</sub>O<sup>+</sup> 326.1412, found 326.1426.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)-4-fluorophenyl)methanol (<b>24</b>)</h4><div class="NLM_p last">White solid (28 mg, 86%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.66–7.57 (m, 2H), 7.42–7.31 (m, 5H), 7.30–7.24 (m, 1H), 4.78 (s, 2H), 4.69 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>FN<sub>5</sub>O<sup>+</sup> 326.1412, found 326.1407.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)-3-fluorophenyl)methanol (<b>25</b>)</h4><div class="NLM_p last">White solid (31 mg, 94%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.63–7.54 (m, 2H), 7.43–7.31 (m, 5H), 7.28–7.23 (m, 1H), 4.76 (s, 2H), 4.69 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>FN<sub>5</sub>O<sup>+</sup> 326.1412, found 326.1419.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)-5-chlorophenyl)methanol (<b>26</b>)</h4><div class="NLM_p last">White solid (16 mg, 64%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.81 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.65 (s, 1H), 7.48 (dd, <i>J</i> = 21.4, 8.4 Hz, 1H), 7.41–7.30 (m, 4H), 7.27 (t, <i>J</i> = 7.5 Hz, 1H), 4.81 (s, 2H), 4.68 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>ClN<sub>5</sub>O<sup>+</sup> 342.1116, found 342.1111.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)-4-chlorophenyl)methanol (<b>27</b>)</h4><div class="NLM_p last">White solid (27 mg, 93%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.86 (d, <i>J</i> = 2.3 Hz, 1H), 7.65–7.56 (m, 2H), 7.42–7.32 (m, 4H), 7.28 (t, <i>J</i> = 7.3 Hz, 1H), 4.79 (s, 2H), 4.69 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>ClN<sub>5</sub>O<sup>+</sup> 342.1116, found 342.1126.</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)-3-chlorophenyl)methanol (<b>28</b>)</h4><div class="NLM_p last">White solid (6.8 mg, 78%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.42–7.30 (m, 3H), 7.26–7.09 (m, 5H), 4.53 (s, 2H), 4.49 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>ClN<sub>5</sub>O<sup>+</sup> 342.1116, found 342.1123.</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)-6-methoxyphenyl)methanol (<b>29</b>)</h4><div class="NLM_p last">White solid (13 mg, 84%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.48 (t, <i>J</i> = 8.1 Hz, 1H), 7.40 (d, <i>J</i> = 7.2 Hz, 1H), 7.37–7.31 (m, 4H), 7.29 (dd, <i>J</i> = 7.8, 3.5 Hz, 2H), 4.82 (s, 2H), 4.69 (s, 2H), 3.91 (s, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>20</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 338.1612, found 338.1618.</div></div><div id="sec4_4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)-4-methoxyphenyl)methanol (<b>31</b>)</h4><div class="NLM_p last">White solid (28 mg, 86%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.47 (dd, <i>J</i> = 8.6, 5.9 Hz, 1H), 7.44 (d, <i>J</i> = 2.8 Hz, 1H), 7.40 (d, <i>J</i> = 7.2 Hz, 1H), 7.35 (t, <i>J</i> = 7.6 Hz, 3H), 7.31–7.24 (m, 1H), 7.17 (dd, <i>J</i> = 8.5, 2.7 Hz, 1H), 4.72 (d, <i>J</i> = 2.8 Hz, 2H), 4.69 (d, <i>J</i> = 2.9 Hz, 2H), 3.86 (s, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>20</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 338.1612, found 338.1614.</div></div><div id="sec4_4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)-3-methoxyphenyl)methanol (<b>32</b>)</h4><div class="NLM_p last">White solid (16 mg, 48%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.54 (td, <i>J</i> = 8.2, 3.0 Hz, 1H), 7.42–7.32 (m, 4H), 7.29 (tt, <i>J</i> = 8.5, 4.8 Hz, 1H), 7.13 (dd, <i>J</i> = 7.9, 5.6 Hz, 2H), 4.68 (s, 2H), 4.68 (s, 2H), 3.88 (d, <i>J</i> = 1.6 Hz, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>20</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 338.1612, found 338.1625.</div></div><div id="sec4_4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> General Procedures for Preparation of Compounds <b>37</b> and <b>38</b></h4><div class="NLM_p last">Intermediate <b>36</b> was prepared according to previously published procedures.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a></div></div><div id="sec4_4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)-5-ethoxyphenyl)methanol (<b>37</b>)</h4><div class="NLM_p last">To a dried microwave tube were added intermediate <b>12</b> (23.6 mg, 0.1 mmol), intermediate <b>36</b> (39.2 mg, 0.2 mmol), tetrakis(triphenylphosphine)palladium(0) (5.8 mg, 5 mol %), potassium carbonate (27.6 mg, 0.2 mmol %), dioxane (0.8 mL), and water (0.2 mL). The mixture was heated to 120 °C for 20 min under microwave. The resulting mixture was filtered through Celite with ethyl acetate as eluent solvent. The filter was washed with water and extracted with ethyl acetate. The combined organic layers were washed brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The resulting residue was dissolved in 5 mL of THF. After the mixture was cooled to 0 °C, sodium borohydride (11 mg, 0.3 mmol) was added slowly. The reaction was warmed to room temperature and stirred for 10 min before methanol was added slowly to quench the reaction. The mixture was washed with water and extracted with ethyl acetate. Combined organic layers were washed with water, brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The resulting residue was dissolved in 1.5 mL of methanol and purified by preparative HPLC to yield <b>37</b> as a white solid (3.7 mg, 23%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.68 (d, <i>J</i> = 7.9 Hz, 2H), 7.60 (d, <i>J</i> = 8.1 Hz, 2H), 7.35–7.30 (m, 3H), 6.29 (s, 1H), 4.10 (s, 2H), 3.65 (s, 2H), 2.65–2.51 (q, <i>J</i> = 7.2 Hz, 2H), 1.49 (t, <i>J</i> = 7.2 Hz, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>22</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 352.1768, found 352.1762.</div></div><div id="sec4_4_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (2-(4-Amino-6-(benzylamino)-1,3,5-triazin-2-yl)-5-isopropoxyphenyl)methanol (<b>38</b>)</h4><div class="NLM_p last">White solid (32 mg, 88%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.86 (d, <i>J</i> = 8.7 Hz, 1H), 7.39 (d, <i>J</i> = 7.3 Hz, 1H), 7.37–7.30 (m, 3H), 7.27 (t, <i>J</i> = 7.3 Hz, 1H), 7.14 (d, <i>J</i> = 2.6 Hz, 1H), 7.00 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 4.81 (s, 2H), 4.78–4.72 (m, 1H), 4.68 (s, 2H), 1.35 (d, <i>J</i> = 6.0 Hz, 6H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>24</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 366.1925, found 366.1934.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> General Procedures for Preparation of Compounds <b>41</b>–<b>54</b>, <b>56</b>–<b>60</b>, and <b>64</b>–<b>81</b></h3><div class="NLM_p">Intermediate <b>40</b> was prepared from intermediates <b>3</b> and <b>11</b> and commercially available amines (intermediate <b>39</b>) according to the previously reported procedures.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Compounds <b>41</b>–<b>54</b>, <b>56</b>–<b>60</b>, and <b>64</b>–<b>66</b> were prepared following the general procedure for preparing compound <b>13</b> from intermediate <b>40</b> and (2-(hydroxymethyl)phenyl)boronic acid. Compounds <b>67</b>–<b>81</b> were prepared following the general procedure for preparing compound <b>13</b> from intermediate <b>40</b> and substituted (2-(hydroxymethyl)phenyl)boronic acids.</div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (2-(4-Amino-6-((2-fluorobenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>41</b>)</h4><div class="NLM_p last">White solid (27 mg, 82%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.82 (d, <i>J</i> = 7.7 Hz, 1H), 7.65–7.54 (m, 2H), 7.47 (dt, <i>J</i> = 26.2, 7.6 Hz, 2H), 7.33–7.25 (m, 1H), 7.13 (t, <i>J</i> = 7.7 Hz, 1H), 7.07 (t, <i>J</i> = 9.3 Hz, 1H), 4.79 (d, <i>J</i> = 3.9 Hz, 2H), 4.74 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>FN<sub>5</sub>O<sup>+</sup> 326.1412, found 326.1443.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (2-(4-Amino-6-((3-fluorobenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>42</b>)</h4><div class="NLM_p last">White solid (27 mg, 83%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.82 (d, <i>J</i> = 7.8 Hz, 1H), 7.65–7.56 (m, 2H), 7.49 (t, <i>J</i> = 7.9 Hz, 1H), 7.32 (td, <i>J</i> = 8.0, 5.7 Hz, 1H), 7.21–7.10 (m, 2H), 6.98 (td, <i>J</i> = 8.5, 2.6 Hz, 1H), 4.81 (s, 2H), 4.68 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>FN<sub>5</sub>O<sup>+</sup> 326.1412, found 326.1435.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> (2-(4-Amino-6-((4-fluorobenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>43</b>)</h4><div class="NLM_p last">White solid (26 mg, 80%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.81 (dt, <i>J</i> = 7.9, 4.7 Hz, 1H), 7.72–7.53 (m, 3H), 7.49 (dd, <i>J</i> = 7.9, 4.8 Hz, 1H), 7.39 (dt, <i>J</i> = 9.7, 5.0 Hz, 1H), 7.03 (tt, <i>J</i> = 8.7, 4.5 Hz, 2H), 4.80 (t, <i>J</i> = 4.9 Hz, 2H), 4.64 (d, <i>J</i> = 4.9 Hz, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>FN<sub>5</sub>O<sup>+</sup> 326.1412, found 326.1438.</div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (2-(4-Amino-6-((2-chlorobenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>44</b>)</h4><div class="NLM_p last">White solid (32 mg, 94%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.84 (d, <i>J</i> = 7.7 Hz, 1H), 7.68–7.58 (m, 2H), 7.54–7.47 (m, 2H), 7.41 (dd, <i>J</i> = 7.2, 2.0 Hz, 1H), 7.29 (ddt, <i>J</i> = 9.4, 6.9, 4.0 Hz, 2H), 4.83 (s, 2H), 4.79 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>ClN<sub>5</sub>O<sup>+</sup> 342.1116, found 342.1113.</div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (2-(4-Amino-6-((3-chlorobenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>45</b>)</h4><div class="NLM_p last">White solid (26 mg, 78%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.83 (d, <i>J</i> = 7.4 Hz, 1H), 7.70–7.57 (m, 2H), 7.51 (td, <i>J</i> = 7.5, 1.5 Hz, 1H), 7.43 (d, <i>J</i> = 2.1 Hz, 1H), 7.36–7.19 (m, 3H), 4.83 (s, 2H), 4.69 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>ClN<sub>5</sub>O<sup>+</sup> 342.1116, found 342.1102.</div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (2-(4-Amino-6-((4-chlorobenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>46</b>)</h4><div class="NLM_p last">White solid (18 mg, 52%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.83 (dd, <i>J</i> = 7.9, 1.2 Hz, 1H), 7.67–7.58 (m, 2H), 7.51 (td, <i>J</i> = 7.5, 1.5 Hz, 1H), 7.38 (d, <i>J</i> = 8.4 Hz, 2H), 7.33 (d, <i>J</i> = 9.0 Hz, 2H), 4.82 (s, 2H), 4.67 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>17</sub>ClN<sub>5</sub>O<sup>+</sup> 342.1116, found 342.1108.</div></div><div id="sec4_5_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> (2-(4-Amino-6-((2-(trifluoromethyl)benzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>47</b>)</h4><div class="NLM_p last">White solid (35 mg, 93%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.86 (d, <i>J</i> = 7.7 Hz, 1H), 7.72 (t, <i>J</i> = 6.7 Hz, 1H), 7.67–7.55 (m, 4H), 7.53 (td, <i>J</i> = 7.5, 1.5 Hz, 1H), 7.47 (td, <i>J</i> = 7.4, 2.2 Hz, 1H), 4.93 (s, 2H), 4.85 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>17</sub>F<sub>3</sub>N<sub>5</sub>O<sup>+</sup> 376.1380, found 376.1388.</div></div><div id="sec4_5_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> (2-(4-Amino-6-((3-(trifluoromethyl)benzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>48</b>)</h4><div class="NLM_p last">White solid (35 mg, 94%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.83 (d, <i>J</i> = 7.6 Hz, 1H), 7.73–7.46 (m, 7H), 4.83 (s, 2H), 4.78 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>17</sub>F<sub>3</sub>N<sub>5</sub>O<sup>+</sup> 376.1380, found 376.1367.</div></div><div id="sec4_5_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> (2-(4-Amino-6-((4-(trifluoromethyl)benzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>49</b>)</h4><div class="NLM_p last">White solid (36 mg, 96%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.83 (dd, <i>J</i> = 7.8, 1.3 Hz, 1H), 7.67–7.55 (m, 3H), −6.76 (m, 1H), 4.82 (s, 2H), 4.66 (s, 2H), 3.78 (s, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>17</sub>F<sub>3</sub>N<sub>5</sub>O<sup>+</sup> 376.1380, found 376.1374.</div></div><div id="sec4_5_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (2-(4-Amino-6-((2-methylbenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>50</b>)</h4><div class="NLM_p last">White solid (28 mg, 88%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.82 (d, <i>J</i> = 7.7 Hz, 1H), 7.64–7.54 (m, 2H), 7.54–7.44 (m, 1H), 7.35–7.22 (m, 1H), 7.19–7.12 (m, 3H), 4.80 (s, 2H), 4.68 (s, 2H), 2.35 (s, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>20</sub>N<sub>5</sub>O<sup>+</sup> 322.1662, found 322.1654.</div></div><div id="sec4_5_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (2-(4-Amino-6-((3-methylbenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>51</b>)</h4><div class="NLM_p last">White solid (29 mg, 92%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.81 (d, <i>J</i> = 7.7 Hz, 1H), 7.68–7.55 (m, 2H), 7.50 (td, <i>J</i> = 7.5, 1.6 Hz, 1H), 7.25–7.13 (m, 3H), 7.08 (d, <i>J</i> = 7.3 Hz, 1H), 4.81 (s, 2H), 4.65 (s, 2H), 2.31 (s, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>20</sub>N<sub>5</sub>O<sup>+</sup> 322.1662, found 322.1657.</div></div><div id="sec4_5_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> (2-(4-Amino-6-((4-methylbenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>52</b>)</h4><div class="NLM_p last">White solid (22 mg, 69%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.81 (d, <i>J</i> = 7.7 Hz, 1H), 7.70–7.57 (m, 2H), 7.55–7.47 (m, 1H), 7.27 (d, <i>J</i> = 7.8 Hz, 2H), 7.15 (d, <i>J</i> = 7.7 Hz, 2H), 4.81 (s, 2H), 4.64 (s, 2H), 2.29 (s, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>20</sub>N<sub>5</sub>O<sup>+</sup> 322.1662, found 322.1672.</div></div><div id="sec4_5_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> (2-(4-Amino-6-((2-methoxybenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>53</b>)</h4><div class="NLM_p last">White solid (27 mg, 79%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.80 (d, <i>J</i> = 7.8 Hz, 1H), 7.66–7.58 (m, 2H), 7.49 (t, <i>J</i> = 7.6 Hz, 1H), 7.38–7.18 (m, 2H), 7.04–6.74 (m, 2H), 4.80 (s, 2H), 4.67 (s, 2H), 3.85 (s, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>20</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 338.1612, found 338.1623.</div></div><div id="sec4_5_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> (2-(4-Amino-6-((3-methoxybenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>54</b>)</h4><div class="NLM_p last">White solid (29 mg, 87%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.83 (dd, <i>J</i> = 7.8, 1.3 Hz, 1H), 7.67–7.55 (m, 3H), −6.76 (m, 1H), 4.82 (s, 2H), 4.66 (s, 2H), 3.78 (s, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>20</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 338.1612, found 338.1618.</div></div><div id="sec4_5_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (2-(4-Amino-6-((4-ethoxybenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>56</b>)</h4><div class="NLM_p last">White solid (32 mg, 93%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.81 (t, <i>J</i> = 6.4 Hz, 1H), 7.69–7.55 (m, 2H), 7.55–7.46 (m, 1H), 7.31 (td, <i>J</i> = 7.8, 7.0, 3.6 Hz, 1H), 6.95–6.79 (m, 3H), 4.81 (s, 2H), 4.60 (s, 2H), 4.00 (q, <i>J</i> = 6.7 Hz, 2H), 1.36 (t, <i>J</i> = 6.7 Hz, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>22</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 352.1768, found 352.1765.</div></div><div id="sec4_5_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (2-(4-Amino-6-((4-isopropoxybenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>57</b>)</h4><div class="NLM_p last">White solid (3.2 mg, 52%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.83 (dd, <i>J</i> = 7.8, 1.3 Hz, 1H), 7.70–7.55 (m, 2H), 7.51 (td, <i>J</i> = 7.6, 1.5 Hz, 1H), 7.36–7.16 (m, 1H), 6.98–6.73 (m, 3H), 4.81 (s, 2H), 4.61 (s, 2H), 4.60–4.53 (m, 1H), 1.29 (d, <i>J</i> = 6.0 Hz, 6H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>24</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 366.1925, found 366.1928.</div></div><div id="sec4_5_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> (2-(4-Amino-6-((4-cyclopropoxybenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>58</b>)</h4><div class="NLM_p last">White solid (30 mg, 84%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.83 (d, <i>J</i> = 7.8 Hz, 1H), 7.62 (dt, <i>J</i> = 21.8, 7.4 Hz, 2H), 7.51 (t, <i>J</i> = 7.5 Hz, 1H), 7.33 (d, <i>J</i> = 8.2 Hz, 2H), 7.02 (d, <i>J</i> = 8.2 Hz, 2H), 4.81 (s, 2H), 4.62 (s, 2H), 3.80–3.71 (m, 1H), 0.77 (t, <i>J</i> = 6.3 Hz, 2H), 0.66 (t, <i>J</i> = 2.9 Hz, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>22</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 364.1768, found 364.1776.</div></div><div id="sec4_5_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> (2-(4-Amino-6-((4-phenoxybenzyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>59</b>)</h4><div class="NLM_p last">White solid (13.4 mg, 84%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.82 (dd, <i>J</i> = 7.8, 1.2 Hz, 1H), 7.65–7.56 (m, 3H), 7.56–7.46 (m, 2H), 7.40 (d, <i>J</i> = 8.6 Hz, 1H), 7.32 (dd, <i>J</i> = 8.6, 7.3 Hz, 2H), 7.09 (t, <i>J</i> = 7.4 Hz, 1H), 6.95 (d, <i>J</i> = 8.5 Hz, 3H), 4.81 (s, 2H), 4.65 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>22</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 400.1768, found 400.1743.</div></div><div id="sec4_5_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> (2-(4-Amino-6-(phenylamino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>60</b>)</h4><div class="NLM_p last">White solid (28 mg, 96%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.88 (d, <i>J</i> = 7.6 Hz, 1H), 7.75 (d, <i>J</i> = 7.9 Hz, 2H), 7.64 (dt, <i>J</i> = 17.2, 7.7 Hz, 2H), 7.53 (t, <i>J</i> = 7.5 Hz, 1H), 7.39 (t, <i>J</i> = 7.9 Hz, 2H), 7.21 (t, <i>J</i> = 7.4 Hz, 1H), 4.87 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>16</sub>N<sub>5</sub>O<sup>+</sup> 294.1349, found 294.1341.</div></div><div id="sec4_5_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> (2-(4-Amino-6-((4-phenoxyphenethyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>64</b>)</h4><div class="NLM_p last">White solid (32 mg, 77%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.82 (dd, <i>J</i> = 7.7, 1.3 Hz, 1H), 7.67–7.57 (m, 3H), 7.51 (td, <i>J</i> = 7.6, 1.4 Hz, 1H), 7.35–7.19 (m, 4H), 7.08 (t, <i>J</i> = 7.3 Hz, 1H), 6.93 (dd, <i>J</i> = 8.6, 6.8 Hz, 3H), 4.79 (s, 2H), 3.73 (t, <i>J</i> = 7.3 Hz, 2H), 2.94 (t, <i>J</i> = 7.4 Hz, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>24</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 414.1925, found 414.1945.</div></div><div id="sec4_5_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> (2-(4-Amino-6-((3-(4-phenoxyphenyl)propyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>65</b>)</h4><div class="NLM_p last">White solid (18 mg, 87%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.83 (d, <i>J</i> = 7.8 Hz, 1H), 7.70–7.56 (m, 3H), 7.51 (t, <i>J</i> = 7.3 Hz, 1H), 7.31 (t, <i>J</i> = 7.7 Hz, 1H), 7.21 (d, <i>J</i> = 8.2 Hz, 2H), 7.07 (t, <i>J</i> = 7.4 Hz, 1H), 6.97–6.78 (m, 4H), 4.81 (s, 2H), 3.52 (t, <i>J</i> = 7.0 Hz, 2H), 2.71 (t, <i>J</i> = 6.1 Hz, 2H), 2.08–1.86 (m, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 428.2081, found 428.2076.</div></div><div id="sec4_5_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> (2-(4-Amino-6-((4-(4-phenoxyphenyl)butyl)amino)-1,3,5-triazin-2-yl)phenyl)methanol (<b>66</b>)</h4><div class="NLM_p last">White solid (17 mg, 73%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.82 (d, <i>J</i> = 7.8 Hz, 1H), 7.63 (dt, <i>J</i> = 22.8, 7.7 Hz, 3H), 7.52 (t, <i>J</i> = 7.5 Hz, 1H), 7.30 (q, <i>J</i> = 6.2, 4.7 Hz, 1H), 7.19 (d, <i>J</i> = 8.2 Hz, 2H), 7.06 (t, <i>J</i> = 7.4 Hz, 1H), 6.91 (dd, <i>J</i> = 23.1, 8.2 Hz, 4H), 4.81 (s, 2H), 3.52 (t, <i>J</i> = 6.4 Hz, 2H), 2.66 (q, <i>J</i> = 6.4 Hz, 2H), 1.70 (dt, <i>J</i> = 7.9, 3.9 Hz, 4H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>28</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> 442.2238, found 442.2234.</div></div><div id="sec4_5_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> (2-(4-Amino-6-((2-fluorobenzyl)amino)-1,3,5-triazin-2-yl)-3-fluorophenyl)methanol (<b>67</b>)</h4><div class="NLM_p last">White solid (33 mg, 96%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.54 (td, <i>J</i> = 8.0, 5.5 Hz, 1H), 7.45 (td, <i>J</i> = 7.6, 1.7 Hz, 1H), 7.38 (d, <i>J</i> = 7.8 Hz, 1H), 7.34–7.26 (m, 1H), 7.23–7.13 (m, 2H), 7.13–7.04 (m, 1H), 4.71 (s, 2H), 4.68 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>16</sub>F<sub>2</sub>N<sub>5</sub>O<sup>+</sup> 344.1317, found 344.1321.</div></div><div id="sec4_5_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> (2-(4-Amino-6-((4-fluorobenzyl)amino)-1,3,5-triazin-2-yl)-3-fluorophenyl)methanol (<b>68</b>)</h4><div class="NLM_p last">White solid (32 mg, 92%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.58 (td, <i>J</i> = 8.0, 5.5 Hz, 1H), 7.44–7.31 (m, 3H), 7.25–7.19 (m, 1H), 7.08 (t, <i>J</i> = 8.6 Hz, 2H), 4.72 (s, 2H), 4.65 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>16</sub>F<sub>2</sub>N<sub>5</sub>O<sup>+</sup> 344.1317, found 344.1323.</div></div><div id="sec4_5_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> (2-(4-Amino-6-((3-methylbenzyl)amino)-1,3,5-triazin-2-yl)-3-fluorophenyl)methanol (<b>69</b>)</h4><div class="NLM_p last">White solid (32 mg, 94%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.61–7.53 (m, 1H), 7.39 (d, <i>J</i> = 7.7 Hz, 1H), 7.26–7.15 (m, 4H), 7.14–7.05 (m, 1H), 4.73 (s, 2H), 4.64 (s, 2H), 2.34 (s, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>19</sub>FN<sub>5</sub>O<sup>+</sup> 340.1568, found 340.1562.</div></div><div id="sec4_5_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> (2-(4-Amino-6-((2-methoxybenzyl)amino)-1,3,5-triazin-2-yl)-3-fluorophenyl)methanol (<b>70</b>)</h4><div class="NLM_p last">White solid (32 mg, 91%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.62–7.54 (m, 1H), 7.42–7.37 (m, 1H), 7.33 (dd, <i>J</i> = 7.4, 2.2 Hz, 1H), 7.31–7.26 (m, 1H), 7.26–7.20 (m, 1H), 6.98 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 6.96–6.88 (m, 1H), 4.73 (s, 2H), 4.66 (s, 2H), 3.86 (s, 3H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>19</sub>FN<sub>5</sub>O<sub>2</sub><sup>+</sup> 356.1517, found 356.1523.</div></div><div id="sec4_5_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> (2-(4-Amino-6-((4-phenoxybenzyl)amino)-1,3,5-triazin-2-yl)-3-fluorophenyl)methanol (<b>71</b>)</h4><div class="NLM_p last">White solid (16 mg, 75%). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ 7.59 (td, <i>J</i> = 8.1, 5.6 Hz, 1H), 7.43–7.37 (m, 3H), 7.36–7.30 (m, 2H), 7.27–7.21 (m, 1H), 7.13–7.07 (m, 1H), 6.99–6.95 (m, 4H), 4.73 (s, 2H), 4.65 (s, 2H); <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD) δ 163.70, 163.17, 162.58, 160.59, 158.72, 158.44, 144.72, 134.40, 133.79, 131.03, 130.77, 125.63, 124.64, 120.03, 120.01, 116.34, 116.17, 62.66, 45.33. HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>21</sub>FN<sub>5</sub>O<sub>2</sub><sup>+</sup> 418.1674, found 418.1665.</div></div><div id="sec4_5_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> (2-(4-Amino-6-(phenethylamino)-1,3,5-triazin-2-yl)-3-fluorophenyl)methanol (<b>72</b>)</h4><div class="NLM_p last">White solid (30 mg, 89%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.60 (td, <i>J</i> = 8.2, 5.5 Hz, 1H), 7.41 (t, <i>J</i> = 8.8 Hz, 1H), 7.35–7.14 (m, 6H), 4.73 (d, <i>J</i> = 2.9 Hz, 2H), 3.71 (t, 2H), 2.95 (t, <i>J</i> = 7.5 Hz, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>19</sub>FN<sub>5</sub>O<sup>+</sup> 340.1568, found 340.1577.</div></div><div id="sec4_5_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> (2-(4-Amino-6-((3-phenylpropyl)amino)-1,3,5-triazin-2-yl)-3-fluorophenyl)methanol (<b>73</b>)</h4><div class="NLM_p last">White solid (25 mg, 71%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.60 (td, <i>J</i> = 8.1, 5.6 Hz, 1H), 7.40 (d, <i>J</i> = 7.8 Hz, 1H), 7.29–7.23 (m, 2H), 7.23–7.10 (m, 4H), 4.73 (s, 2H), 3.50 (t, <i>J</i> = 7.2 Hz, 2H), 2.71 (t, <i>J</i> = 7.2 Hz, 2H), 2.04–1.89 (m, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>21</sub>FN<sub>5</sub>O<sup>+</sup> 354.1725, found 354.1735.</div></div><div id="sec4_5_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> (2-(4-Amino-6-((4-phenylbutyl)amino)-1,3,5-triazin-2-yl)-3-fluorophenyl)methanol (<b>74</b>)</h4><div class="NLM_p last">White solid (23 mg, 64%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.65–7.56 (m, 1H), 7.40 (t, <i>J</i> = 6.6 Hz, 1H), 7.24 (ddt, <i>J</i> = 8.9, 5.1, 2.6 Hz, 3H), 7.21–7.17 (m, 2H), 7.14 (tt, <i>J</i> = 7.2, 1.5 Hz, 1H), 4.74 (s, 2H), 3.50 (t, 2H), 2.67 (t, <i>J</i> = 7.0 Hz, 2H), 1.81–1.53 (m, 4H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>23</sub>FN<sub>5</sub>O<sup>+</sup> 368.1881, found 368.1883.</div></div><div id="sec4_5_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> (2-(4-Amino-6-((4-phenoxyphenethyl)amino)-1,3,5-triazin-2-yl)-3-fluorophenyl)methanol (<b>75</b>)</h4><div class="NLM_p last">White solid (19 mg, 88%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.75–7.49 (m, 4H), 7.41 (t, <i>J</i> = 8.4 Hz, 1H), 7.32 (ddd, <i>J</i> = 8.6, 6.0, 2.1 Hz, 1H), 7.29–7.18 (m, 2H), 7.07 (t, <i>J</i> = 7.4 Hz, 1H), 6.99–6.85 (m, 3H), 4.73 (s, 2H), 3.72 (t, <i>J</i> = 7.4 Hz, 2H), 2.94 (t, <i>J</i> = 7.3 Hz, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>23</sub>FN<sub>5</sub>O<sub>2</sub><sup>+</sup> 432.1830, found 432.1832.</div></div><div id="sec4_5_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> (2-(4-Amino-6-((3-(4-phenoxyphenyl)propyl)amino)-1,3,5-triazin-2-yl)-3-fluorophenyl)methanol (<b>76</b>)</h4><div class="NLM_p last">White solid (16 mg, 70%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.64–7.46 (m, 1H), 7.39 (d, <i>J</i> = 7.4 Hz, 1H), 7.31 (ddd, <i>J</i> = 8.7, 7.4, 1.5 Hz, 2H), 7.26–7.13 (m, 4H), 7.07 (t, <i>J</i> = 7.4 Hz, 1H), 7.01–6.77 (m, 3H), 4.68 (s, 2H), 3.48 (t, <i>J</i> = 7.1 Hz, 2H), 2.70 (t, <i>J</i> = 7.7 Hz, 2H), 2.06–1.83 (m, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>25</sub>FN<sub>5</sub>O<sub>2</sub><sup>+</sup> 446.1987, found 446.1978.</div></div><div id="sec4_5_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> (2-(4-Amino-6-(phenethylamino)-1,3,5-triazin-2-yl)-4-chlorophenyl)methanol (<b>77</b>)</h4><div class="NLM_p last">White solid (15 mg, 43%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.84 (d, <i>J</i> = 2.2 Hz, 1H), 7.70–7.52 (m, 3H), 7.36–7.08 (m, 4H), 4.77 (s, 2H), 3.72 (t, <i>J</i> = 7.5 Hz, 2H), 2.95 (t, <i>J</i> = 7.6 Hz, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>19</sub>ClN<sub>5</sub>O<sup>+</sup> 356.1273, found 356.1278.</div></div><div id="sec4_5_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> (2-(4-Amino-6-((3-phenylpropyl)amino)-1,3,5-triazin-2-yl)-4-chlorophenyl)methanol (<b>78</b>)</h4><div class="NLM_p last">White solid (7 mg, 19%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.86 (t, <i>J</i> = 1.7 Hz, 1H), 7.68–7.54 (m, 3H), 7.35–7.05 (m, 4H), 4.77 (s, 2H), 3.50 (t, <i>J</i> = 7.0 Hz, 2H), 2.71 (t, <i>J</i> = 7.7 Hz, 2H), 2.06–1.86 (m, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>21</sub>ClN<sub>5</sub>O<sup>+</sup> 370.1429, found 370.1423.</div></div><div id="sec4_5_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> (2-(4-Amino-6-((4-phenylbutyl)amino)-1,3,5-triazin-2-yl)-4-chlorophenyl)methanol (<b>79</b>)</h4><div class="NLM_p last">White solid (12 mg, 32%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.86 (t, <i>J</i> = 3.4 Hz, 1H), 7.69–7.52 (m, 3H), 7.33–7.00 (m, 4H), 4.76 (s, 2H), 3.58–3.44 (m, 2H), 2.67 (q, <i>J</i> = 7.5, 6.9 Hz, 2H), 1.70 (tdd, <i>J</i> = 9.8, 5.3, 2.7 Hz, 4H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>23</sub>ClN<sub>5</sub>O<sup>+</sup> 384.1586, found 384.1589.</div></div><div id="sec4_5_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> (2-(4-Amino-6-((4-phenoxybenzyl)amino)-1,3,5-triazin-2-yl)-4-chlorophenyl)methanol (<b>80</b>)</h4><div class="NLM_p last">White solid (8 mg, 18%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.91–7.87 (m, 1H), 7.63–7.53 (m, 2H), 7.39 (d, <i>J</i> = 8.5 Hz, 2H), 7.35–7.30 (m, 2H), 7.10 (t, <i>J</i> = 7.6 Hz, 1H), 6.99–6.93 (m, 4H), 4.75 (s, 2H), 4.64 (s, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>21</sub>ClN<sub>5</sub>O<sub>2</sub><sup>+</sup> 434.1387, found 434.1376.</div></div><div id="sec4_5_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> (2-(4-Amino-6-((4-phenoxyphenethyl)amino)-1,3,5-triazin-2-yl)-4-chlorophenyl)methanol (<b>81</b>)</h4><div class="NLM_p last">White solid (16 mg, 36%). <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD) δ 7.86 (d, <i>J</i> = 2.2 Hz, 1H), 7.57 (dt, <i>J</i> = 10.6, 8.4 Hz, 2H), 7.38–7.19 (m, 4H), 7.13–7.04 (m, 1H), 7.01–6.81 (m, 4H), 4.74 (s, 2H), 3.78–3.65 (m, 2H), 2.93 (t, <i>J</i> = 7.5 Hz, 2H). HRMS <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>23</sub>ClN<sub>5</sub>O<sub>2</sub><sup>+</sup> 448.1535, found 448.1537.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> Calcium Release Assays</h3><div class="NLM_p last">Calcium release assays for G<sub>q</sub>-GPCRs were performed according to published protocol<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> with minor modifications. In brief, HEK293 cells stably expressing 5-HT<sub>2B</sub> receptors<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> were plated in 384-well black clear bottom plates (coated with poly-<span class="smallcaps smallerCapital">l</span>-Lys) at a density of 10 000 cells/well in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 1% dialyzed fetal bovine serum (dFBS) overnight (16–20 h). The cells were loaded with Fluo-4 Direct Calcium dye (20 μL/well) for 1 h at 37 °C in FLIPR buffer [1× Hank’s balanced salt solution (HBSS), 20 mM 4-(2-hydroxyethyl)-1-piperazine ethanesulfonic acid (HEPES), and 2.5 mM probenecid, pH 7.40, room temperature]. Drug working dilutions (3× of final) were prepared in the FLIPR buffer supplemented with 1 mg/mL BSA and 0.1 mg/mL ascorbic acid. The FLIPR<sup>TETRA</sup> was programmed to transfer 10 μL/well of drug solution, and fluorescence was recorded for a total of 2 min with 1 read/s and 10 s before drug addition. The maximum fluorescence intensity (fold of basal) within the first minute of drug addition was exported and analyzed in Prism.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> GloSensor cAMP Assays</h3><div class="NLM_p last">Proton-mediated GPR68 G<sub>s</sub> activation was determined by measuring cAMP production using GloSensor cAMP assays as reported previously<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> with modifications as detailed below. Calcium- and magnesium-free Hank’s balanced salt solution (1× HBSS) was used as the base for all pH buffers. Different zwitterionic organic buffer agents were used for different pH ranges, 1× HBSS with 20 mM 2-(<i>N</i>-morpholino)ethanesulfonic acid for pH 5.60–6.70, 1× HBSS with 20 mM 4-(2-hydroxyethyl)-1-piperazine ethanesulfonic acid (HEPES) for pH 6.80–8.20, and 1× HBSS with 20 mM (tris(hydroxymethyl)methylamino)propanesulfonic acid (TAPS) for pH 8.30–8.60. The pH was adjusted with KOH and fresh buffers were made in the morning to use within 24 h. The phosphodiesterase (PDE) inhibitor Ro 20-1724 (10 μM) was included in assay buffer to eliminate potential effect of PDE inhibition. BSA (1 mg/mL) was added to buffer for samples with solubility issues. Addition of BSA does not affect GloSensor cAMP assays and is routinely applied in cell-based GPCR functional assays.<a onclick="showRef(event, 'ref63 ref64'); return false;" href="javascript:void(0);" class="ref ref63 ref64">(63,64)</a> Briefly, HEK293T cells were co-transfected with a modified polyethylenimine (PEI) transfection method.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> For a 10 cm dish of HEK293T cells with 12 mL growth medium, 4 μg of receptor DNA and 4 μg of GloSensor DNA were mixed in 0.4 mL of Opti-MEM medium, followed by addition of 40 μL PEI reagent (increase proportionally when larger dishes are used). The mixture was incubated for 20 min at room temperature and then added to cells for incubation overnight (16–20 h). Transfected cells were plated in poly-<span class="smallcaps smallerCapital">l</span>-Lys-coated 384-well white clear bottom plates using DMEM + 1% dFBS for a minimum of 6 h (up to 24 h) before being used for assays. After culture medium was removed, the cells were incubated with 20 μL/well of 3.5 mM luciferin prepared in assay buffer (1× HBSS, 20 mM TAPS, pH 8.40) for 1 h at 37 °C. Ligand stock was made in dimethyl sulfoxide at 10 mM and diluted with pH buffers to desired concentrations. To stimulate cells with premixed drug solutions at desired pH conditions, luciferin-loading solution was removed before addition of drug solution (25 μL/well). Luminescence was counted at 20–30 min after drug application, and data were analyzed in the GraphPad Prism.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> GPCRome Off-Target Agonist Assays</h3><div class="NLM_p">Off-target agonist activity was carried out according to previously published procedure using the PRESTO-Tango assay platform<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> with modifications as outlined below. The assay was designed and modified such that 40 receptors were tested in each 384-well plate, each in quadruplicate wells for basal (with medium vehicle) and in quadruplicate for sample. Each plate also contained 16 wells of dopamine D<sub>2</sub> receptors stimulated with 100 nM quinpirole as assay controls and 16 wells of D<sub>2</sub> with vehicle alone as D<sub>2</sub> basal activity. In addition, there were two columns of wells (32 wells) of untransfected cells as negative controls. The whole procedure takes 3 days from plating cells to reading plates. In brief, HTLA cells were plated in poly-<span class="smallcaps smallerCapital">l</span>-Lys-coated 384-well white clear bottom plates in DMEM supplemented with 2% dFBS (10 000 cells per 40 μL per well) overnight, transfected with receptor Tango constructs (15 ng/well) overnight using an in-plate PEI transfection method,<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> in which receptor DNA was resuspended in Opti-MEM and hybridized with PEI reagent prior to dilution and distribution to 384-well plates and subsequent addition to cells. After overnight incubation, cells received drugs without replacement of the medium and were incubated overnight. Drug solutions and medium were removed and Bright-Glo reagent (20 μL/well) were added. Luminescence was counted on a luminescence reader after 20 min incubation at room temperature. Results were presented in the form of fold of basal for each receptor and analyzed in the Prism.</div><div class="NLM_p last">For Tango arrestin recruitment concentration responses with individual GPCRs, HTLA cells were transfected with receptor DNA overnight in DMEM with 10% FBS and plated into poly-<span class="smallcaps smallerCapital">l</span>-Lys-coated 384-well plates as above using DMEM supplemented with 1% dFBS. Drugs were added 6 h later and incubated overnight (16–20 h). Medium and drug solutions were removed, and Bright-Glo reagent (20 μL/well) was added. Luminescence was counted and results were analyzed in Prism.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> Radioligand Binding Assays</h3><div class="NLM_p last">Ligand binding profiling was performed by the NIMH PDSP. The NIMH PDSP is directed by Bryan L Roth, MD, PhD, at the University of North Carolina at Chapel Hill, North Carolina, and Program Officer Jamie Driscoll at NIMH, Bethesda, MD. Details about binding assay procedures and protocols are available online at: <a href="https://pdspdb.unc.edu/pdspWeb/?site=assays" class="extLink">https://pdspdb.unc.edu/pdspWeb/?site=assays</a>.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> Data Analysis</h3><div class="NLM_p">Proton-mediated GPR68 G<sub>s</sub> activity, as measured as relative luminescence unit, in the absence and presence of increasing concentrations of ligands was fitted to the standard allosteric operational model to obtain allosteric parameters, affinity cooperativity α and efficacy cooperativity β, and allosteric ligand binding affinity <i>K</i><sub>B</sub>.<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26−28)</a> Briefly, affinity cooperativity α defines the reciprocal effect between allosteric modulator and orthosteric agonist (protons in this case) with α > 1 for increasing affinity and 0 < α < 1 for reducing affinity. Efficacy cooperativity β defines the effect of allosteric modulator on the efficacy of the orthosteric agonist (protons in this case) with β > 1 for increasing proton efficacy and 0 < β < 1 for reducing proton efficacy. The <i>K</i><sub>B</sub> defines the equilibrium binding affinity of test allosteric ligands in the absence of orthosteric agonist (H<sup>+</sup>). Since orthosteric agonist protons always exist in the receptor compartment, <i>K</i><sub>B</sub> in this case has no physical meaning. We applied the comprehensive index log(αβ/<i>K</i><sub>B</sub>) to quantify the overall allosteric activity of test allosteric ligand and the difference from that of reference ligand, ΔLog(αβ/<i>K</i><sub>B</sub>), to represent the net change of allosteric activity.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> For summary, results were normalized to the percentages, pooled together, and analyzed in the Prism. Normalization was performed using the built-in normalization function in the Prism, where proton basal activity in the absence of test allosteric ligand was set as 0% and the maximum proton activity in the absence of test allosteric ligand was set as 100%. In functional antagonist assays, functional affinity (<i>K</i><sub>i</sub>) values were calculated from IC<sub>50</sub> values using the Cheng–Prusoff equation <i>K</i><sub>i</sub> = IC<sub>50</sub>/(1 + <i>L</i>/EC<sub>50</sub>),<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> where IC<sub>50</sub> is the antagonist concentration when 50% inhibition is achieved against reference agonist at concentration [L], EC<sub>50</sub> is the potency of the reference agonist.</div><div id="sec4_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> Mouse Pharmacokinetic Study</h4><div class="NLM_p last">A standard PK study was performed. Compound <b>71</b> was dissolved in 5% <i>N</i>-methyl-2-pyrrolidone, 5% Solutol HS-15, and 90% normal saline. Nine male Swiss Albino mice were treated with compound <b>71</b> intraperitoneally at the 25 mg/kg dose. Plasma and brain samples were collected at 0.5, 1.0, and 2.0 h. Compound concentrations in plasma and brain at each time point are average values from three test mice. The error bars represent ±SEM.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i95"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00869" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74264" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74264" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00869" class="ext-link">10.1021/acs.jmedchem.9b00869</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline"><sup>1</sup>H NMR spectra of compounds <b>46</b>, <b>71</b>, <b>77</b>, and <b>78</b>; <sup>13</sup>C NMR spectra of compound <b>71</b>; proton concentration–response curves at human GPR68 in the absence and presence of increasing concentrations of test compounds; proton concentration–response curves of compounds <b>20</b>, <b>46</b>, <b>77</b>, and <b>78</b> at GPR4 and GPR65; heat map showing off-target binding activities of selected lead compounds at 48 common GPCRs, ion channels, and transporters; functional activities of compounds with binding affinity values lower than 300 nM (p<i>K</i><sub>i</sub> > 6.50); off-target agonist activity at human GPCRome; functional characterization of the compounds identified from the GPCRome screening; proton concentration–response curves of compounds <b>1</b>, <b>46</b>, <b>63</b>, <b>71</b>, <b>77</b>, and <b>78</b> at mouse GPR68; effects of inactive analogues on the GPR68 PAM <b>71</b>; allosteric modulation of GPR68 proton activity: and implications for pH-dependent agonism and inverse agonism (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf">jm9b00869_si_001.pdf (6.05 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_002.csv">jm9b00869_si_002.csv (6.7 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00869" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24806" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24806" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xi-Ping Huang</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmacology, 
    The National Institute of Mental
Health Psychoactive
Drug Screening Program (NIMH PDSP), Department of Pharmacology, 
     and , University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c3bbb3abb6a2ada483b6ada0eda6a7b6"><span class="__cf_email__" data-cfemail="8ef6fee6fbefe0e9cefbe0eda0ebeafb">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Liu</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#85efecebe2abe9ecf0c5e8f6f6e8abe0e1f0"><span class="__cf_email__" data-cfemail="bbd1d2d5dc95d7d2cefbd6c8c8d695dedfce">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1a70737b74347073745a77696977347f7e6f"><span class="__cf_email__" data-cfemail="bbd1d2dad595d1d2d5fbd6c8c8d695dedfce">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xufen Yu</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Terry P. Kenakin</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmacology and , University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Samuel T. Slocum</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmacology and , University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Chen</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael L. Martini</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>X.Y. and X.-P.H. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e6290-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i97">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36853" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36853" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The research described here was in part supported by the grants R56MH111789 (to X.-P.H. and T.P.K.) and U24DK116195 (to J.J.) from the U.S. National Institutes of Health, and by the PDSP contract HHSN-271-2013-00017-C from the U.S. National Institute of Mental Health. M.L.M. acknowledges the support by the NIGMS-funded Integrated Pharmacological Sciences Training Program T32GM062754 and the MSTP training grants at the Icahn School of Mount Sinai (NIH T32GM007280).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G protein-coupled receptor</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">G<sub>S</sub></td><td class="NLM_def"><p class="first last">stimulatory G protein</p></td></tr><tr><td class="NLM_term">cAMP</td><td class="NLM_def"><p class="first last">cyclic adenosine monophosphate</p></td></tr><tr><td class="NLM_term">PAM</td><td class="NLM_def"><p class="first last">positive allosteric modulator</p></td></tr><tr><td class="NLM_term">MT<sub>1</sub></td><td class="NLM_def"><p class="first last">melatonin receptor 1</p></td></tr><tr><td class="NLM_term">MT<sub>2</sub></td><td class="NLM_def"><p class="first last">melatonin receptor 2</p></td></tr><tr><td class="NLM_term">ACKR3</td><td class="NLM_def"><p class="first last">atypical chemokine receptor 3</p></td></tr><tr><td class="NLM_term">5-HT<sub>2B</sub></td><td class="NLM_def"><p class="first last">5-hydroxytryptamine receptor 2B</p></td></tr><tr><td class="NLM_term">mGLuR5</td><td class="NLM_def"><p class="first last">metabotropic glutamate receptor 5</p></td></tr><tr><td class="NLM_term">D<sub>2</sub></td><td class="NLM_def"><p class="first last">dopamine D<sub>2</sub> receptor</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">Na<sub>2</sub>SO<sub>4</sub></td><td class="NLM_def"><p class="first last">sodium sulfate</p></td></tr><tr><td class="NLM_term">dioxane</td><td class="NLM_def"><p class="first last">1,4-dioxane</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">K<sub>2</sub>CO<sub>3</sub></td><td class="NLM_def"><p class="first last">potassium carbonate</p></td></tr><tr><td class="NLM_term">Pd(PPh<sub>3</sub>)<sub>4</sub></td><td class="NLM_def"><p class="first last">tetrakis(triphenylphosphine)palladium(0)</p></td></tr><tr><td class="NLM_term">MgSO<sub>4</sub></td><td class="NLM_def"><p class="first last">magnesium sulfate</p></td></tr><tr><td class="NLM_term">NaBH<sub>4</sub></td><td class="NLM_def"><p class="first last">sodium borohydride</p></td></tr><tr><td class="NLM_term"><sup><i>n</i></sup>BuLi</td><td class="NLM_def"><p class="first last"><i>n</i>-butyllithium solution</p></td></tr><tr><td class="NLM_term">B(OMe)<sub>3</sub></td><td class="NLM_def"><p class="first last">trimethyl borate</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i99">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84552" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84552" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 66 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casey, G.</span></span> <span> </span><span class="NLM_article-title">Identification of Human OGR1, a Novel G Protein-Coupled Receptor That Maps to Chromosome 14</span>. <i>Genomics</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">397</span>– <span class="NLM_lpage">402</span>, <span class="refDoi"> DOI: 10.1006/geno.1996.0377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1006%2Fgeno.1996.0377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=8661159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADyaK28Xks1aqsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1996&pages=397-402&author=Y.+Xuauthor=G.+Casey&title=Identification+of+Human+OGR1%2C+a+Novel+G+Protein-Coupled+Receptor+That+Maps+to+Chromosome+14&doi=10.1006%2Fgeno.1996.0377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of human OGR1, a novel G protein-coupled receptor that maps to chromosome 14</span></div><div class="casAuthors">Xu, Yan; Casey, Graham</div><div class="citationInfo"><span class="NLM_cas:title">Genomics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">397-402</span>CODEN:
                <span class="NLM_cas:coden">GNMCEP</span>;
        ISSN:<span class="NLM_cas:issn">0888-7543</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">We describe the cloning of a novel G protein-coupled receptor, termed ovarian cancer G protein-coupled receptor 1 (OGR1), from an ovarian cancer cell line that maps to chromosome 14q31.  The predicted open reading frame of OGR1 encodes a protein of 365 amino acids.  OGR1 is expressed as a single 3.0-kb transcript in several tissues, including spleen, testis, small intestine, peripheral blood leukocytes, brain, heart, lung, placenta, and kidney, with no detectable OGR1 expression in thymus, prostate, ovary, colon, liver, skeletal muscle, or pancreas.  OGR1 shares strongest homol. (49-54%) with the human orphan receptor GPR4.  The structural features of OGR1 suggest that the ligand for OGR1 is likely to be a small peptide or a small bioactive mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0WdsDcxdHO7Vg90H21EOLACvtfcHk0ljLpEfCG2MZNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xks1aqsro%253D&md5=a311f68e33b2fe0ef84d7fe704d888a4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1006%2Fgeno.1996.0377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fgeno.1996.0377%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DCasey%26aufirst%3DG.%26atitle%3DIdentification%2520of%2520Human%2520OGR1%252C%2520a%2520Novel%2520G%2520Protein-Coupled%2520Receptor%2520That%2520Maps%2520to%2520Chromosome%252014%26jtitle%3DGenomics%26date%3D1996%26volume%3D35%26spage%3D397%26epage%3D402%26doi%3D10.1006%2Fgeno.1996.0377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, M.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasser, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junker, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofstetter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seuwen, K.</span></span> <span> </span><span class="NLM_article-title">Proton-Sensing G-Protein-Coupled Receptors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>425</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">98</span>, <span class="refDoi"> DOI: 10.1038/nature01905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fnature01905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=12955148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFansrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2003&pages=93-98&author=M.-G.+Ludwigauthor=M.+Vanekauthor=D.+Gueriniauthor=J.+A.+Gasserauthor=C.+E.+Jonesauthor=U.+Junkerauthor=H.+Hofstetterauthor=R.+M.+Wolfauthor=K.+Seuwen&title=Proton-Sensing+G-Protein-Coupled+Receptors&doi=10.1038%2Fnature01905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Proton-sensing G-protein-coupled receptors</span></div><div class="casAuthors">Ludwig, Marie-Gabrielle; Vanek, Miroslava; Guerini, Danilo; Gasser, Juerg A.; Jones, Carol E.; Junker, Uwe; Hofstetter, Hans; Wolf, Romain M.; Seuwen, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">425</span>
        (<span class="NLM_cas:issue">6953</span>),
    <span class="NLM_cas:pages">93-98</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Blood pH is maintained in a narrow range around pH 7.4 mainly through regulation of respiration and renal acid extrusion.  The mol. mechanisms involved in pH homeostasis are not completely understood.  Here we show that ovarian cancer G-protein-coupled receptor 1 (OGR1), previously described as a receptor for sphingosylphosphorylcholine, acts as a proton-sensing receptor stimulating inositol phosphate formation.  The receptor is inactive at pH 7.8, and fully activated at pH 6.8-site-directed mutagenesis shows that histidines at the extracellular surface are involved in pH sensing.  We find that GPR4, a close relative of OGR1, also responds to pH changes, but elicits cAMP formation.  It is known that the skeleton participates in pH homeostasis as a buffering organ, and that osteoblasts respond to pH changes in the physiol. range, but the pH-sensing mechanism operating in these cells was hitherto not known.  We detect expression of OGR1 in osteosarcoma cells and primary human osteoblast precursors, and show that these cells exhibit strong pH-dependent inositol phosphate formation.  Immunohistochem. on rat tissue sections confirms the presence of OGR1 in osteoblasts and osteocytes.  We propose that OGR1 and GPR4 are proton-sensing receptors involved in pH homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfpVd_U0kyY7Vg90H21EOLACvtfcHk0ljLpEfCG2MZNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFansrw%253D&md5=131775a2f98871f4cbe6519b53679a0d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature01905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01905%26sid%3Dliteratum%253Aachs%26aulast%3DLudwig%26aufirst%3DM.-G.%26aulast%3DVanek%26aufirst%3DM.%26aulast%3DGuerini%26aufirst%3DD.%26aulast%3DGasser%26aufirst%3DJ.%2BA.%26aulast%3DJones%26aufirst%3DC.%2BE.%26aulast%3DJunker%26aufirst%3DU.%26aulast%3DHofstetter%26aufirst%3DH.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DSeuwen%26aufirst%3DK.%26atitle%3DProton-Sensing%2520G-Protein-Coupled%2520Receptors%26jtitle%3DNature%26date%3D2003%26volume%3D425%26spage%3D93%26epage%3D98%26doi%3D10.1038%2Fnature01905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Regard, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coughlin, S. R.</span></span> <span> </span><span class="NLM_article-title">Anatomical Profiling of G Protein-Coupled Receptor Expression</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">561</span>– <span class="NLM_lpage">571</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2008.08.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.cell.2008.08.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=18984166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlyqtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2008&pages=561-571&author=J.+B.+Regardauthor=I.+T.+Satoauthor=S.+R.+Coughlin&title=Anatomical+Profiling+of+G+Protein-Coupled+Receptor+Expression&doi=10.1016%2Fj.cell.2008.08.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Anatomical profiling of G protein-coupled receptor expression</span></div><div class="casAuthors">Regard, Jean B.; Sato, Isaac T.; Coughlin, Shaun R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">561-571</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) comprise the largest family of transmembrane signaling mols. and regulate a host of physiol. and disease processes.  To better understand the functions of GPCRs in vivo, we quantified transcript levels of 353 nonodorant GPCRs in 41 adult mouse tissues.  Cluster anal. placed many GPCRs into anticipated anatomical and functional groups and predicted previously unidentified roles for less-studied receptors.  From one such prediction, we showed that the Gpr91 ligand succinate can regulate lipolysis in white adipose tissue, suggesting that signaling by this citric acid cycle intermediate may regulate energy homeostasis.  We also showed that pairwise anal. of GPCR expression across tissues may help predict drug side effects.  This resource will aid studies to understand GPCR function in vivo and may assist in the identification of therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiJtFrCT3H77Vg90H21EOLACvtfcHk0lhrmrM7_GNBZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlyqtr3J&md5=dbe3c2c9c295050ec4ba86124c8aa2b4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2008.08.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2008.08.040%26sid%3Dliteratum%253Aachs%26aulast%3DRegard%26aufirst%3DJ.%2BB.%26aulast%3DSato%26aufirst%3DI.%2BT.%26aulast%3DCoughlin%26aufirst%3DS.%2BR.%26atitle%3DAnatomical%2520Profiling%2520of%2520G%2520Protein-Coupled%2520Receptor%2520Expression%26jtitle%3DCell%26date%3D2008%26volume%3D135%26spage%3D561%26epage%3D571%26doi%3D10.1016%2Fj.cell.2008.08.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davenport, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marrion, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faccenda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. A.</span></span> <span> </span><span class="NLM_article-title">CGTP Collaborators. The Concise Guide to Pharmacology 2017/18: G Protein-Coupled Receptors</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">S17</span>– <span class="NLM_lpage">S129</span>, <span class="refDoi"> DOI: 10.1111/bph.13878</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1111%2Fbph.13878" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=29055040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs12ls7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2017&pages=S17-S129&author=S.+P.+Alexanderauthor=A.+Christopoulosauthor=A.+P.+Davenportauthor=E.+Kellyauthor=N.+V.+Marrionauthor=J.+A.+Petersauthor=E.+Faccendaauthor=S.+D.+Hardingauthor=A.+J.+Pawsonauthor=J.+L.+Sharmanauthor=C.+Southanauthor=J.+A.+Davies&title=CGTP+Collaborators.+The+Concise+Guide+to+Pharmacology+2017%2F18%3A+G+Protein-Coupled+Receptors&doi=10.1111%2Fbph.13878"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">THE CONCISE GUIDE TO PHARMACOLOGY 2017/18: G protein-coupled receptors</span></div><div class="casAuthors">Alexander, Stephen P. H.; Christopoulos, Arthur; Davenport, Anthony P.; Kelly, Eamonn; Marrion, Neil V.; Peters, John A.; Faccenda, Elena; Harding, Simon D.; Pawson, Adam J.; Sharman, Joanna L.; Southan, Christopher; Davies, Jamie A.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue">S1</span>),
    <span class="NLM_cas:pages">S17-S129</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Concise Guide to PHARMACOl. 2017/18 provides concise overviews of the key properties of nearly 1800 human drug targets with an emphasis on selective pharmacol. (where available), plus links to an open access knowledgebase of drug targets and their ligands (), which provides more detailed views of target and ligand properties.  Although the Concise Guide represents approx. 400 pages, the material presented is substantially reduced compared to information and links presented on the website.  It provides a permanent, citable, point-in-time record that will survive database updates.  The full contents of this section can be found at .  G protein-coupled receptors are one of the eight major pharmacol. targets into which the Guide is divided, with the others being: ligand-gated ion channels, voltage-gated ion channels, other ion channels, nuclear hormone receptors, catalytic receptors, enzymes and transporters.  These are presented with nomenclature guidance and summary information on the best available pharmacol. tools, alongside key refs. and suggestions for further reading.  The landscape format of the Concise Guide is designed to facilitate comparison of related targets from material contemporary to mid-2017, and supersedes data presented in the 2015/16 and 2013/14 Concise Guides and previous Guides to Receptors and Channels.  It is produced in close conjunction with the Nomenclature Committee of the Union of Basic and Clin. Pharmacol. (NC-IUPHAR), therefore, providing official IUPHAR classification and nomenclature for human drug targets, where appropriate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrktrD3wjuPZ7Vg90H21EOLACvtfcHk0lhrmrM7_GNBZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs12ls7%252FO&md5=2d4bc1acc92b6aa659365aa70ba50983</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1111%2Fbph.13878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.13878%26sid%3Dliteratum%253Aachs%26aulast%3DAlexander%26aufirst%3DS.%2BP.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DDavenport%26aufirst%3DA.%2BP.%26aulast%3DKelly%26aufirst%3DE.%26aulast%3DMarrion%26aufirst%3DN.%2BV.%26aulast%3DPeters%26aufirst%3DJ.%2BA.%26aulast%3DFaccenda%26aufirst%3DE.%26aulast%3DHarding%26aufirst%3DS.%2BD.%26aulast%3DPawson%26aufirst%3DA.%2BJ.%26aulast%3DSharman%26aufirst%3DJ.%2BL.%26aulast%3DSouthan%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DJ.%2BA.%26atitle%3DCGTP%2520Collaborators.%2520The%2520Concise%2520Guide%2520to%2520Pharmacology%25202017%252F18%253A%2520G%2520Protein-Coupled%2520Receptors%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2017%26volume%3D174%26spage%3DS17%26epage%3DS129%26doi%3D10.1111%2Fbph.13878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mogi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damirin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komachi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span> <span> </span><span class="NLM_article-title">Sphingosylphosphorylcholine Antagonizes Proton-Sensing Ovarian Cancer G-Protein-Coupled Receptor 1 (OGR1)-Mediated Inositol Phosphate Production and cAMP Accumulation</span>. <i>J. Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">160</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1254/jphs.FP0050599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1254%2Fjphs.FP0050599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=16210776" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFOns7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2005&pages=160-167&author=C.+Mogiauthor=H.+Tomuraauthor=M.+Toboauthor=J.-Q.+Wangauthor=A.+Damirinauthor=J.+Konauthor=M.+Komachiauthor=K.+Hashimotoauthor=K.+Satoauthor=F.+Okajima&title=Sphingosylphosphorylcholine+Antagonizes+Proton-Sensing+Ovarian+Cancer+G-Protein-Coupled+Receptor+1+%28OGR1%29-Mediated+Inositol+Phosphate+Production+and+cAMP+Accumulation&doi=10.1254%2Fjphs.FP0050599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Sphingosylphosphorylcholine antagonizes proton-sensing ovarian cancer G-protein-coupled receptor 1 (OGR1)-mediated inositol phosphate production and cAMP accumulation</span></div><div class="casAuthors">Mogi, Chihiro; Tomura, Hideaki; Tobo, Masayuki; Wang, Ju-Qiang; Damirin, Alatangaole; Kon, Junko; Komachi, Mayumi; Hashimoto, Kinji; Sato, Koichi; Okajima, Fumikazu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacological Sciences (Tokyo, Japan)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">160-167</span>CODEN:
                <span class="NLM_cas:coden">JPSTGJ</span>;
        ISSN:<span class="NLM_cas:issn">1347-8613</span>.
    
            (<span class="NLM_cas:orgname">Japanese Pharmacological Society</span>)
        </div><div class="casAbstract">Ovarian cancer G-protein-coupled receptor 1 (OGR1), previously proposed as a receptor for sphingosylphosphorylcholine (SPC), has recently been identified as a proton-sensing or extracellular pH-responsive G-protein-coupled receptor stimulating inositol phosphate prodn., reflecting the activation of phospholipase C.  In the present study, we found that acidic pH stimulated cAMP accumulation, reflecting the activation of adenylyl cyclase, in addn. to inositol phosphate prodn. in OGR1-expressing cells.  The cAMP response was hardly affected by the inhibition of phospholipase C.  SPC inhibited the acidification-induced actions in a pH-dependent manner, while no OGR1-dependent agonistic action of SPC was obsd.  Thus, the dose-response curves of the proton-induced actions were shifted to the right in the presence of SPC regardless of stereoisoform.  The antagonistic property was also obsd. for psychosine and glucosylsphingosine.  In conclusion, OGR1 stimulation may lead to the activation of adenylyl cyclase in addn. to phospholipase C in response to extracellular acidification but not to SPC.  However, SPC and related lysolipids antagonize the proton-induced and OGR1-mediated actions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7WOLALesDkrVg90H21EOLACvtfcHk0liCiKMLHx9kGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFOns7vO&md5=8fdb013385f4cdf2b187a9c9172bdec8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1254%2Fjphs.FP0050599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1254%252Fjphs.FP0050599%26sid%3Dliteratum%253Aachs%26aulast%3DMogi%26aufirst%3DC.%26aulast%3DTomura%26aufirst%3DH.%26aulast%3DTobo%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.-Q.%26aulast%3DDamirin%26aufirst%3DA.%26aulast%3DKon%26aufirst%3DJ.%26aulast%3DKomachi%26aufirst%3DM.%26aulast%3DHashimoto%26aufirst%3DK.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DOkajima%26aufirst%3DF.%26atitle%3DSphingosylphosphorylcholine%2520Antagonizes%2520Proton-Sensing%2520Ovarian%2520Cancer%2520G-Protein-Coupled%2520Receptor%25201%2520%2528OGR1%2529-Mediated%2520Inositol%2520Phosphate%2520Production%2520and%2520cAMP%2520Accumulation%26jtitle%3DJ.%2520Pharmacol.%2520Sci.%26date%3D2005%26volume%3D99%26spage%3D160%26epage%3D167%26doi%3D10.1254%2Fjphs.FP0050599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oates, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirmani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Ovarian Cancer G Protein-Coupled Receptor 1, a New Metastasis Suppressor Gene in Prostate Cancer</span>. <i>JNCI, J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">1313</span>– <span class="NLM_lpage">1327</span>, <span class="refDoi"> DOI: 10.1093/jnci/djm107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1093%2Fjnci%2Fdjm107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=17728215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWqtbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2007&pages=1313-1327&author=L.+S.+Singhauthor=M.+Berkauthor=R.+Oatesauthor=Z.+Zhaoauthor=H.+Tanauthor=Y.+Jiangauthor=A.+Zhouauthor=K.+Kirmaniauthor=R.+Steinmetzauthor=D.+Lindnerauthor=Y.+Xu&title=Ovarian+Cancer+G+Protein-Coupled+Receptor+1%2C+a+New+Metastasis+Suppressor+Gene+in+Prostate+Cancer&doi=10.1093%2Fjnci%2Fdjm107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Ovarian Cancer G Protein-Coupled Receptor 1, a New Metastasis Suppressor Gene in Prostate Cancer</span></div><div class="casAuthors">Singh, Lisam Shanjukumar; Berk, Michael; Oates, Rhonda; Zhao, Zhenwen; Tan, Haiyan; Jiang, Ying; Zhou, Aimin; Kirmani, Kashif; Steinmetz, Rosemary; Lindner, Daniel; Xu, Yan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1313-1327</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background: Metastasis is a process by which tumors spread from primary organs to other sites in the body and is the major cause of death for cancer patients.  The ovarian cancer G protein-coupled receptor 1 (OGR1) gene has been shown to be expressed at lower levels in metastatic compared with primary prostate cancer tissues.  Methods: We used an orthotopic mouse metastasis model, in which we injected PC3 metastatic human prostate cancer cells stably transfected with empty vector (vector-PC3) or OGR1-expressing vector (OGR1-PC3) into the prostate lobes of athymic or NOD/SCID mice (n = 3-8 mice per group).  Migration of PC3 cells transiently transfected with vector control or with OGR1- or GPR4 (a G protein-coupled receptor with the highest homol. to OGR1)-expressing vectors was measured in vitro by Boyden chamber assays.  G protein alpha-inhibitory subunit 1 (Gαi1) expression after treatment with pertussis toxin (PTX) was measured using immunoblotting anal.  The inhibitory factor present in the conditioned medium was extd. using org. solvents and analyzed by mass spectrometry.  Results: In vivo, all 26 mice carrying tumors that were derived from vector-PC3 cells developed prostate cancer metastases (mean = 100%, 95% confidence interval [Cl] = 83.97% to 100%) but few (4 of 32) mice carrying tumors derived from OGR1-expressing PC3 cells (mean = 12.50%, 95% Cl = 4.08% to 29.93%) developed metastases.  However, exogenous OGR1 overexpression had no effect on primary prostate tumor growth in vivo.  In vitro, expression of OGR1, but not GPR4, inhibited cell migration (mean percentage of cells migrated, 30.2% vs. 100%, difference = 69.8%, 95% Cl = 63.0% to 75.9%; P < .001) via increased expression of Gαi1 and the secretion fo a chloroform/methanol-extractable heat-insensitive factor into the conditioned medium through a PTX-sensitive pathway.  Conclusion: OGR1 is a novel metastasis suppressor gene for prostate cancer.  OGR1's constitutive activity via Gαi contributes to its inhibitory effect on cell migration in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrB2_OCs3OF67Vg90H21EOLACvtfcHk0liCiKMLHx9kGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWqtbjF&md5=2b9d5ff8e805fb22665665412ac3cc1c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjm107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjm107%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DL.%2BS.%26aulast%3DBerk%26aufirst%3DM.%26aulast%3DOates%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DA.%26aulast%3DKirmani%26aufirst%3DK.%26aulast%3DSteinmetz%26aufirst%3DR.%26aulast%3DLindner%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DOvarian%2520Cancer%2520G%2520Protein-Coupled%2520Receptor%25201%252C%2520a%2520New%2520Metastasis%2520Suppressor%2520Gene%2520in%2520Prostate%2520Cancer%26jtitle%3DJNCI%252C%2520J.%2520Natl.%2520Cancer%2520Inst.%26date%3D2007%26volume%3D99%26spage%3D1313%26epage%3D1327%26doi%3D10.1093%2Fjnci%2Fdjm107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, J.</span></span> <span> </span><span class="NLM_article-title">Ovarian Cancer G Protein Coupled Receptor 1 Suppresses Cell Migration of MCF7 Breast Cancer Cells via a Gα12/13-Rho-Rac1 Pathway</span>. <i>J. Mol. Signal.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">6</span> <span class="refDoi"> DOI: 10.1186/1750-2187-8-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1186%2F1750-2187-8-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=23663350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BC3snivV2ntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=J.+Liauthor=B.+Guoauthor=J.+Wangauthor=X.+Chengauthor=Y.+Xuauthor=J.+Sang&title=Ovarian+Cancer+G+Protein+Coupled+Receptor+1+Suppresses+Cell+Migration+of+MCF7+Breast+Cancer+Cells+via+a+G%CE%B112%2F13-Rho-Rac1+Pathway&doi=10.1186%2F1750-2187-8-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Ovarian cancer G protein coupled receptor 1 suppresses cell migration of MCF7 breast cancer cells via a Gα12/13-Rho-Rac1 pathway</span></div><div class="casAuthors">Li Jing; Guo Bin; Wang Jing; Cheng Xiaoyan; Xu Yan; Sang Jianli</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular signaling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6</span>
        ISSN:<span class="NLM_cas:issn">1750-2187</span>.
    </div><div class="casAbstract">BACKGROUND:  Ovarian cancer G protein coupled receptor 1 (OGR1) mediates inhibitory effects on cell migration in human prostate and ovarian cancer cells.  However, the mechanisms and signaling pathways that mediate these inhibitory effects are essentially unknown.  METHODS:  MCF7 cell line was chosen as a model system to study the mechanisms by which OGR1 regulates cell migration, since it expresses very low levels of endogenous OGR1.  Cell migratory activities were assessed using both wound healing and transwell migration assays.  The signaling pathways involved were studied using pharmacological inhibitors and genetic forms of the relevant genes, as well as small G protein pull-down activity assays.  The expression levels of various signaling molecules were analyzed by Western blot and quantitative PCR analysis.  RESULTS:  Over-expression of OGR1 in MCF7 cells substantially enhanced activation of Rho and inhibition of Rac1, resulting in inhibition of cell migration.  In addition, expression of the Gα12/13 specific regulator of G protein signaling (RGS) domain of p115RhoGEF, but not treatment with pertussis toxin (PTX, a Gαi specific inhibitor), could abrogate OGR1-dependent Rho activation, Rac1 inactivation, and inhibition of migration in MCF7 cells.  The bioactive lipids tested had no effect on OGR1 function in cell migration.  CONCLUSION:  Our data suggest, for the first time, that OGR1 inhibits cell migration through a Gα12/13 -Rho-Rac1 signaling pathway in MCF7 cells.  This pathway was not significantly affected by bioactive lipids and all the assays were conducted at constant pH, suggesting a constitutive activity of OGR1.  This is the first clear delineation of an OGR1-mediated cell signaling pathway involved in migration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSAWumsAcuPOgHqQV89nF-2fW6udTcc2eZvM9GnIwy8Hrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3snivV2ntg%253D%253D&md5=9524ed612a2f1db66a9f1db5ab771e3b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2F1750-2187-8-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1750-2187-8-6%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DSang%26aufirst%3DJ.%26atitle%3DOvarian%2520Cancer%2520G%2520Protein%2520Coupled%2520Receptor%25201%2520Suppresses%2520Cell%2520Migration%2520of%2520MCF7%2520Breast%2520Cancer%2520Cells%2520via%2520a%2520G%25CE%25B112%252F13-Rho-Rac1%2520Pathway%26jtitle%3DJ.%2520Mol.%2520Signal.%26date%3D2013%26volume%3D8%26doi%3D10.1186%2F1750-2187-8-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Damaghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtkowiak, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillies, R. J.</span></span> <span> </span><span class="NLM_article-title">pH Sensing and Regulation in Cancer</span>. <i>Front. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">370</span>, <span class="refDoi"> DOI: 10.3389/fphys.2013.00370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.3389%2Ffphys.2013.00370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=24381558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BC2cflsVymtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=370&author=M.+Damaghiauthor=J.+W.+Wojtkowiakauthor=R.+J.+Gillies&title=pH+Sensing+and+Regulation+in+Cancer&doi=10.3389%2Ffphys.2013.00370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">pH sensing and regulation in cancer</span></div><div class="casAuthors">Damaghi Mehdi; Wojtkowiak Jonathan W; Gillies Robert J</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in physiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">370</span>
        ISSN:<span class="NLM_cas:issn">1664-042X</span>.
    </div><div class="casAbstract">Cells maintain intracellular pH (pHi) within a narrow range (7.1-7.2) by controlling membrane proton pumps and transporters whose activity is set by intra-cytoplasmic pH sensors.  These sensors have the ability to recognize and induce cellular responses to maintain the pHi, often at the expense of acidifying the extracellular pH.  In turn, extracellular acidification impacts cells via specific acid-sensing ion channels (ASICs) and proton-sensing G-protein coupled receptors (GPCRs).  In this review, we will discuss some of the major players in proton sensing at the plasma membrane and their downstream consequences in cancer cells and how these pH-mediated changes affect processes such as migration and metastasis.  The complex mechanisms by which they transduce acid pH signals to the cytoplasm and nucleus are not well understood.  However, there is evidence that expression of proton-sensing GPCRs such as GPR4, TDAG8, and OGR1 can regulate aspects of tumorigenesis and invasion, including cofilin and talin regulated actin (de-)polymerization.  Major mechanisms for maintenance of pHi homeostasis include monocarboxylate, bicarbonate, and proton transporters.  Notably, there is little evidence suggesting a link between their activities and those of the extracellular H(+)-sensors, suggesting a mechanistic disconnect between intra- and extracellular pH.  Understanding the mechanisms of pH sensing and regulation may lead to novel and informed therapeutic strategies that can target acidosis, a common physical hallmark of solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRiMyC_tQmaj-3MPKNEWB5RfW6udTcc2eZvM9GnIwy8Hrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cflsVymtQ%253D%253D&md5=3a46626baf76f257c63c9d97c73aaa0c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3389%2Ffphys.2013.00370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphys.2013.00370%26sid%3Dliteratum%253Aachs%26aulast%3DDamaghi%26aufirst%3DM.%26aulast%3DWojtkowiak%26aufirst%3DJ.%2BW.%26aulast%3DGillies%26aufirst%3DR.%2BJ.%26atitle%3DpH%2520Sensing%2520and%2520Regulation%2520in%2520Cancer%26jtitle%3DFront.%2520Physiol.%26date%3D2013%26volume%3D4%26spage%3D370%26doi%3D10.3389%2Ffphys.2013.00370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirmani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Abnormalities in Osteoclastogenesis and Decreased Tumorigenesis in Mice Deficient for Ovarian Cancer G Protein-Coupled Receptor 1</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e5705</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0005705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1371%2Fjournal.pone.0005705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=19479052" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&author=H.+Liauthor=D.+Wangauthor=L.+S.+Singhauthor=M.+Berkauthor=H.+Tanauthor=Z.+Zhaoauthor=R.+Steinmetzauthor=K.+Kirmaniauthor=G.+Weiauthor=Y.+Xu&title=Abnormalities+in+Osteoclastogenesis+and+Decreased+Tumorigenesis+in+Mice+Deficient+for+Ovarian+Cancer+G+Protein-Coupled+Receptor+1&doi=10.1371%2Fjournal.pone.0005705"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0005705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0005705%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DSingh%26aufirst%3DL.%2BS.%26aulast%3DBerk%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DSteinmetz%26aufirst%3DR.%26aulast%3DKirmani%26aufirst%3DK.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DAbnormalities%2520in%2520Osteoclastogenesis%2520and%2520Decreased%2520Tumorigenesis%2520in%2520Mice%2520Deficient%2520for%2520Ovarian%2520Cancer%2520G%2520Protein-Coupled%2520Receptor%25201%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26doi%3D10.1371%2Fjournal.pone.0005705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">Effects of Ovarian Cancer G Protein Coupled Receptor 1 on the Proliferation, Migration, and Adhesion of Human Ovarian Cancer Cells</span>. <i>Chin. Med. J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">1327</span>– <span class="NLM_lpage">1332</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=21740742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BC3MnmvFCqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2011&pages=1327-1332&author=J.+Renauthor=L.+Zhang&title=Effects+of+Ovarian+Cancer+G+Protein+Coupled+Receptor+1+on+the+Proliferation%2C+Migration%2C+and+Adhesion+of+Human+Ovarian+Cancer+Cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of ovarian cancer G protein coupled receptor 1 on the proliferation, migration, and adhesion of human ovarian cancer cells</span></div><div class="casAuthors">Ren Juan; Zhang Long</div><div class="citationInfo"><span class="NLM_cas:title">Chinese medical journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1327-32</span>
        ISSN:<span class="NLM_cas:issn">0366-6999</span>.
    </div><div class="casAbstract">BACKGROUND:  OGR1 was found as a G-protein coupled receptor (GPCR) and proton sensor.  Our previous studies have found that OGR1 has inhibitory effect on the metastasis of prostate cancer.  In order to investigate the roles of OGR1 gene in the biological activities of ovarian cancer, we studied the OGR1 effects on ovarian cancer cells, HEY cells.  METHODS:  OGR1 gene was transfected into HEY cell, in which endogenous expression is low.  OGR1-overxepressed cells and vector-transfected cells were compared in different assays.  Western blotting was employed to confirm the high expression level of OGR1.  Cell proliferation was determined by MTT assay and cell doubling time assay.  Cell migration assay (transwell assay) and cell adhesion assay were performed to determine the migration and adhesion potential of cells.  Student's t test was employed for statistical analysis.  RESULTS:  Proliferation of OGR1-overexpressed cells was significantly reduced (P < 0.01); cell migration was significantly inhibited in the OGR1-transfected cells (P < 0.01); cell adhesion to extracellular matrix including fibronectin, vitronectin, collagen I/IV was significantly increased (P < 0.01).  CONCLUSIONS:  OGR1 expression in human ovarian cancer cells significantly inhibited the cell proliferation and migration, but significantly enhanced cell adhesion to the extracellular matrix.  It indicated that OGR1 may be a tumor suppressor gene for ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR0VQIsxKz7Z--MdeMhXAHkfW6udTcc2ebDtzklEKYWy7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MnmvFCqsg%253D%253D&md5=585e6f6b286f6be37c4534b7eb00d83f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DEffects%2520of%2520Ovarian%2520Cancer%2520G%2520Protein%2520Coupled%2520Receptor%25201%2520on%2520the%2520Proliferation%252C%2520Migration%252C%2520and%2520Adhesion%2520of%2520Human%2520Ovarian%2520Cancer%2520Cells%26jtitle%3DChin.%2520Med.%2520J.%26date%3D2011%26volume%3D124%26spage%3D1327%26epage%3D1332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiley, S. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sriram, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicklick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowy, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insel, P. A.</span></span> <span> </span><span class="NLM_article-title">GPR68, a Proton-Sensing GPCR, Mediates Interaction of Cancer-Associated Fibroblasts and Cancer Cells</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1170</span>– <span class="NLM_lpage">1183</span>, <span class="refDoi"> DOI: 10.1096/fj.201700834R</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1096%2Ffj.201700834R" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=29092903" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWnt7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=1170-1183&author=S.+Z.+Wileyauthor=K.+Sriramauthor=W.+Liangauthor=S.+E.+Changauthor=R.+Frenchauthor=T.+McCannauthor=J.+Sicklickauthor=H.+Nishiharaauthor=A.+M.+Lowyauthor=P.+A.+Insel&title=GPR68%2C+a+Proton-Sensing+GPCR%2C+Mediates+Interaction+of+Cancer-Associated+Fibroblasts+and+Cancer+Cells&doi=10.1096%2Ffj.201700834R"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">GPR68, a proton-sensing GPCR, mediates interaction of cancer-associated fibroblasts and cancer cells</span></div><div class="casAuthors">Wiley, Shu Z.; Sriram, Krishna; Liang, Wenjing; Chang, Sarah E.; French, Randall; McCann, Thalia; Sicklick, Jason; Nishihara, Hiroshi; Lowy, Andrew M.; Insel, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1170-1183</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">The microenvironment of pancreatic ductal adenocarcinoma (PDAC) is characterized by a dense fibrotic stroma (desmoplasia) generated by pancreatic cancer-assocd. fibroblasts (CAFs) derived from pancreatic stellate cells (PSCs) and pancreatic fibroblasts (PFs).  Using an unbiased GPCRomic array approach, we identified 82 G-protein-coupled receptors (GPCRs) commonly expressed by CAFs derived from 5 primary PDAC tumors.  Compared with PSCs and PFs, CAFs have increased expression of GPR68 (a proton-sensing GPCR), with the results confirmed by immunoblotting, The Cancer Genome Atlas data, and immunohistochem. of PDAC tumors.  Coculture of PSCs with PDAC cells, or incubation with TNF-α, induced GPR68 expression.  GPR68 activation (by decreasing the extracellular pH) enhanced IL-6 expression via a cAMP/PKA/cAMP response element binding protein signaling pathway.  Knockdown of GPR68 by short interfering RNA diminished low pH-induced prodn. of IL-6 and enhancement of PDAC cell proliferation by CAF conditioned media.  CAFs from other gastrointestinal cancers also express GPR68.  PDAC cells thus induce expression by CAFs of GPR68, which senses the acidic microenvironment, thereby increasing prodn. of fibrotic markers and IL-6 and promoting PDAC cell proliferation.  CAF-expressed GPR68 is a mediator of low-pH-promoted regulation of the tumor microenvironments, in particular to PDAC cell-CAF interaction and may be a novel therapeutic target for pancreatic and perhaps other types of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLeNoINb1PMrVg90H21EOLACvtfcHk0lj6zYI9jpOXrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWnt7nE&md5=d5c11ef52e6ec43e166c9a7d400e4caa</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1096%2Ffj.201700834R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.201700834R%26sid%3Dliteratum%253Aachs%26aulast%3DWiley%26aufirst%3DS.%2BZ.%26aulast%3DSriram%26aufirst%3DK.%26aulast%3DLiang%26aufirst%3DW.%26aulast%3DChang%26aufirst%3DS.%2BE.%26aulast%3DFrench%26aufirst%3DR.%26aulast%3DMcCann%26aufirst%3DT.%26aulast%3DSicklick%26aufirst%3DJ.%26aulast%3DNishihara%26aufirst%3DH.%26aulast%3DLowy%26aufirst%3DA.%2BM.%26aulast%3DInsel%26aufirst%3DP.%2BA.%26atitle%3DGPR68%252C%2520a%2520Proton-Sensing%2520GPCR%252C%2520Mediates%2520Interaction%2520of%2520Cancer-Associated%2520Fibroblasts%2520and%2520Cancer%2520Cells%26jtitle%3DFASEB%2520J.%26date%3D2018%26volume%3D32%26spage%3D1170%26epage%3D1183%26doi%3D10.1096%2Ffj.201700834R" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span> <span> </span><span class="NLM_article-title">Regulation of Inflammation by Extracellular Acidification and Proton-Sensing GPCRs</span>. <i>Cell. Signal.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2263</span>– <span class="NLM_lpage">2271</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2013.07.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.cellsig.2013.07.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=23917207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlyrtLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2013&pages=2263-2271&author=F.+Okajima&title=Regulation+of+Inflammation+by+Extracellular+Acidification+and+Proton-Sensing+GPCRs&doi=10.1016%2Fj.cellsig.2013.07.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of inflammation by extracellular acidification and proton-sensing GPCRs</span></div><div class="casAuthors">Okajima, Fumikazu</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2263-2271</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Under ischemic and inflammatory circumstances, such as allergic airway asthma, rheumatoid arthritis, atherosclerosis, and tumors, extracellular acidification occurs due to the stimulation of anaerobic glycolysis.  An acidic microenvironment has been shown to modulate pro-inflammatory or anti-inflammatory responses, including cyclooxygenase-2 (COX-2) expression, prostaglandin synthesis, and cytokine expression, in a variety of cell types, and thereby to exacerbate or ameliorate inflammation.  However, mol. mechanisms underlying extracellular acidic pH-induced actions have not been fully understood.  Recent studies have shown that ovarian cancer G protein-coupled receptor 1 (OGR1)-family G protein-coupled receptors (GPCRs) can sense extracellular pH or protons, which in turn stimulates intracellular signaling pathways and subsequent diverse cellular responses.  In the present review, I discuss extracellular acidic pH-induced inflammatory responses and related responses in inflammatory cells, such as macrophages and neutrophils, and non-inflammatory cells, such as smooth muscle cells and endothelial cells, focusing esp. on proton-sensing GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoViaL3lpdAL7Vg90H21EOLACvtfcHk0lhCAznSwnXqjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlyrtLrF&md5=7bd4b1d6cf1bac0a5aba1c4bf2619ed9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2013.07.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2013.07.022%26sid%3Dliteratum%253Aachs%26aulast%3DOkajima%26aufirst%3DF.%26atitle%3DRegulation%2520of%2520Inflammation%2520by%2520Extracellular%2520Acidification%2520and%2520Proton-Sensing%2520GPCRs%26jtitle%3DCell.%2520Signal.%26date%3D2013%26volume%3D25%26spage%3D2263%26epage%3D2271%26doi%3D10.1016%2Fj.cellsig.2013.07.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, W.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.-H.</span></span> <span> </span><span class="NLM_article-title">Expression and Function of Proton-Sensing G-Protein-Coupled Receptors in Inflammatory Pain</span>. <i>Mol. Pain</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">S39</span>, <span class="refDoi"> DOI: 10.1016/j.jpain.2009.01.167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.jpain.2009.01.167" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=S39&author=Y.-J.+Chenauthor=C.-W.+Huangauthor=C.-S.+Linauthor=W.-H.+Changauthor=W.-H.+Sun&title=Expression+and+Function+of+Proton-Sensing+G-Protein-Coupled+Receptors+in+Inflammatory+Pain&doi=10.1016%2Fj.jpain.2009.01.167"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.jpain.2009.01.167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jpain.2009.01.167%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.-J.%26aulast%3DHuang%26aufirst%3DC.-W.%26aulast%3DLin%26aufirst%3DC.-S.%26aulast%3DChang%26aufirst%3DW.-H.%26aulast%3DSun%26aufirst%3DW.-H.%26atitle%3DExpression%2520and%2520Function%2520of%2520Proton-Sensing%2520G-Protein-Coupled%2520Receptors%2520in%2520Inflammatory%2520Pain%26jtitle%3DMol.%2520Pain%26date%3D2009%26volume%3D10%26spage%3DS39%26doi%3D10.1016%2Fj.jpain.2009.01.167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Vallière, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eloranta, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clay, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spalinger, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tcymbarevich, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terhalle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, M.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suply, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fried, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kullak-Ublick, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frey-Wagner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seuwen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogler, G.</span></span> <span> </span><span class="NLM_article-title">G Protein-Coupled pH-Sensing Receptor OGR1 Is a Regulator of Intestinal Inflammation</span>. <i>Inflammatory Bowel Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1269</span>– <span class="NLM_lpage">1281</span>, <span class="refDoi"> DOI: 10.1097/MIB.0000000000000375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1097%2FMIB.0000000000000375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=25856770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BC2MjitlOisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1269-1281&author=C.+de%0AValli%C3%A8reauthor=Y.+Wangauthor=J.+J.+Elorantaauthor=S.+Vidalauthor=I.+Clayauthor=M.+R.+Spalingerauthor=I.+Tcymbarevichauthor=A.+Terhalleauthor=M.-G.+Ludwigauthor=T.+Suplyauthor=M.+Friedauthor=G.+A.+Kullak-Ublickauthor=I.+Frey-Wagnerauthor=M.+Scharlauthor=K.+Seuwenauthor=C.+A.+Wagnerauthor=G.+Rogler&title=G+Protein-Coupled+pH-Sensing+Receptor+OGR1+Is+a+Regulator+of+Intestinal+Inflammation&doi=10.1097%2FMIB.0000000000000375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">G Protein-coupled pH-sensing Receptor OGR1 Is a Regulator of Intestinal Inflammation</span></div><div class="casAuthors">de Valliere Cheryl; Wang Yu; Eloranta Jyrki J; Vidal Solange; Clay Ieuan; Spalinger Marianne R; Tcymbarevich Irina; Terhalle Anne; Ludwig Marie-Gabrielle; Suply Thomas; Fried Michael; Kullak-Ublick Gerd A; Frey-Wagner Isabelle; Scharl Michael; Seuwen Klaus; Wagner Carsten A; Rogler Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Inflammatory bowel diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1269-81</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  A novel family of proton-sensing G protein-coupled receptors, including OGR1, GPR4, and TDAG8, was identified to be important for physiological pH homeostasis and inflammation.  Thus, we determined the function of proton-sensing OGR1 in the intestinal mucosa.  MTEHODS:  OGR1 expression in colonic tissues was investigated in controls and patients with IBD.  Expression of OGR1 upon cell activation was studied in the Mono Mac 6 (MM6) cell line and primary human and murine monocytes by real-time PCR.  Ogr1 knockout mice were crossbred with Il-10 deficient mice and studied for more than 200 days.  Microarray profiling was performed using Ogr1 and Ogr1 (WT) residential peritoneal macrophages.  RESULTS:  Patients with IBD expressed higher levels of OGR1 in the mucosa than non-IBD controls.  Treatment of MM6 cells with TNF, led to significant upregulation of OGR1 expression, which could be reversed by the presence of NF-κB inhibitors.  Kaplan-Meier survival analysis showed a significantly delayed onset and progression of rectal prolapse in female Ogr1/Il-10 mice.  These mice displayed significantly less rectal prolapses.  Upregulation of gene expression, mediated by OGR1, in response to extracellular acidification in mouse macrophages was enriched for inflammation and immune response, actin cytoskeleton, and cell-adhesion gene pathways.  CONCLUSIONS:  OGR1 expression is induced in cells of human macrophage lineage and primary human monocytes by TNF.  NF-κB inhibition reverses the induction of OGR1 expression by TNF.  OGR1 deficiency protects from spontaneous inflammation in the Il-10 knockout model.  Our data indicate a pathophysiological role for pH-sensing receptor OGR1 during the pathogenesis of mucosal inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTevE0e4oFxjZdYoSR9eyvsfW6udTcc2eakI4qRK2XXdbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjitlOisQ%253D%253D&md5=a530ecb419d883b77581519a74c33ebd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1097%2FMIB.0000000000000375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FMIB.0000000000000375%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BValli%25C3%25A8re%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DEloranta%26aufirst%3DJ.%2BJ.%26aulast%3DVidal%26aufirst%3DS.%26aulast%3DClay%26aufirst%3DI.%26aulast%3DSpalinger%26aufirst%3DM.%2BR.%26aulast%3DTcymbarevich%26aufirst%3DI.%26aulast%3DTerhalle%26aufirst%3DA.%26aulast%3DLudwig%26aufirst%3DM.-G.%26aulast%3DSuply%26aufirst%3DT.%26aulast%3DFried%26aufirst%3DM.%26aulast%3DKullak-Ublick%26aufirst%3DG.%2BA.%26aulast%3DFrey-Wagner%26aufirst%3DI.%26aulast%3DScharl%26aufirst%3DM.%26aulast%3DSeuwen%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DC.%2BA.%26aulast%3DRogler%26aufirst%3DG.%26atitle%3DG%2520Protein-Coupled%2520pH-Sensing%2520Receptor%2520OGR1%2520Is%2520a%2520Regulator%2520of%2520Intestinal%2520Inflammation%26jtitle%3DInflammatory%2520Bowel%2520Dis.%26date%3D2015%26volume%3D21%26spage%3D1269%26epage%3D1281%26doi%3D10.1097%2FMIB.0000000000000375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mailhot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birnbaum, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKay, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason-Savas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odgren, P. R.</span></span> <span> </span><span class="NLM_article-title">Expression of and Role for Ovarian Cancer G-Protein-Coupled Receptor 1 (OGR1) During Osteoclastogenesis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">23598</span>– <span class="NLM_lpage">23605</span>, <span class="refDoi"> DOI: 10.1074/jbc.M602191200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1074%2Fjbc.M602191200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=16787916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD28XotVCmtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=23598-23605&author=M.+Yangauthor=G.+Mailhotauthor=M.+J.+Birnbaumauthor=C.+A.+MacKayauthor=A.+Mason-Savasauthor=P.+R.+Odgren&title=Expression+of+and+Role+for+Ovarian+Cancer+G-Protein-Coupled+Receptor+1+%28OGR1%29+During+Osteoclastogenesis&doi=10.1074%2Fjbc.M602191200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of and Role for Ovarian Cancer G-protein-coupled Receptor 1 (OGR1) during Osteoclastogenesis</span></div><div class="casAuthors">Yang, Meiheng; Mailhot, Genevieve; Birnbaum, Mark J.; MacKay, Carole A.; Mason-Savas, April; Odgren, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">23598-23605</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Osteoclasts differentiate from hematopoietic mononuclear precursor cells under the control of both colony stimulating factor-1 (CSF-1, or M-CSF) and receptor activator of NF-κB ligand (RANKL, or TRANCE, TNFSF11) to carry out bone resorption.  Using high d. gene microarrays, we followed gene expression changes in long bone RNA when CSF-1 injections were used to restore osteoclast populations in the CSF-1-null toothless (csf1tl/csf1tl) osteopetrotic rat.  We found that ovarian cancer G-protein-coupled receptor 1 (OGR1, or GPR68) was strongly up-regulated, rising >6-fold in vivo after 2 days of CSF-1 treatments.  OGR1 is a dual membrane receptor for both protons (extracellular pH) and lysolipids.  Strong induction of OGR1 mRNA was also obsd. by microarray, real-time RT-PCR, and immunoblotting when mouse bone marrow mononuclear cells and RAW 264.7 pre-osteoclast-like cells were treated with RANKL to induce osteoclast differentiation.  Anti-OGR1 immunofluorescence showed intense labeling of RANKL-treated RAW cells.  The time course of OGR1 mRNA expression suggests that OGR1 induction is early but not immediate, peaking 2 days after inducing osteoclast differentiation both in vivo and in vitro.  Specific inhibition of OGR1 by anti-OGR1 antibody and by small inhibitory RNA inhibited RANKL-induced differentiation of both mouse bone marrow mononuclear cells and RAW cells in vitro, as evidenced by a decrease in tartrate-resistant acid phosphatase-pos. osteoclasts.  Taken together, these data indicate that OGR1 is expressed early during osteoclastogenesis both in vivo and in vitro and plays a role in osteoclast differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr18tKXf0aByLVg90H21EOLACvtfcHk0li6d0Vj-qqpSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XotVCmtb4%253D&md5=138ea5bea27a2a00c0cb218401187f53</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M602191200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M602191200%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DM.%26aulast%3DMailhot%26aufirst%3DG.%26aulast%3DBirnbaum%26aufirst%3DM.%2BJ.%26aulast%3DMacKay%26aufirst%3DC.%2BA.%26aulast%3DMason-Savas%26aufirst%3DA.%26aulast%3DOdgren%26aufirst%3DP.%2BR.%26atitle%3DExpression%2520of%2520and%2520Role%2520for%2520Ovarian%2520Cancer%2520G-Protein-Coupled%2520Receptor%25201%2520%2528OGR1%2529%2520During%2520Osteoclastogenesis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D23598%26epage%3D23605%26doi%3D10.1074%2Fjbc.M602191200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pereverzev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komarova, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korcok, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tremblay, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, S. M.</span></span> <span> </span><span class="NLM_article-title">Extracellular Acidification Enhances Osteoclast Survival through an NFAT-Independent, Protein Kinase C-Dependent Pathway</span>. <i>Bone</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1016/j.bone.2007.08.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.bone.2007.08.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=17964236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOrtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2008&pages=150-161&author=A.+Pereverzevauthor=S.+V.+Komarovaauthor=J.+Korcokauthor=S.+Armstrongauthor=G.+B.+Tremblayauthor=S.+J.+Dixonauthor=S.+M.+Sims&title=Extracellular+Acidification+Enhances+Osteoclast+Survival+through+an+NFAT-Independent%2C+Protein+Kinase+C-Dependent+Pathway&doi=10.1016%2Fj.bone.2007.08.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular acidification enhances osteoclast survival through an NFAT-independent, protein kinase C-dependent pathway</span></div><div class="casAuthors">Pereverzev, Alexey; Komarova, Svetlana V.; Korcok, Jasminka; Armstrong, Souzan; Tremblay, Gilles B.; Dixon, S. Jeffrey; Sims, Stephen M.</div><div class="citationInfo"><span class="NLM_cas:title">Bone (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">150-161</span>CODEN:
                <span class="NLM_cas:coden">BONEDL</span>;
        ISSN:<span class="NLM_cas:issn">8756-3282</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Systemic acidosis has detrimental effects on the skeleton and local acidosis is assocd. with bone destruction in inflammatory and neoplastic diseases.  However, the mechanisms by which acidosis enhances osteoclastic bone resorption are poorly understood.  Our aim was to examine the effects of acid on osteoclast survival and the involvement of cytosolic Ca2+ in mediating these effects.  Osteoclasts were isolated from long bones of newborn rats, and multinucleated osteoclast-like cells were generated from RAW 264.7 cells.  Cytosolic free Ca2+ concn. ([Ca2+]i) was monitored using fura-2.  Survival of rat osteoclasts over a period of 18 h was significantly enhanced by acidification of the medium from 40 ± 10% at pH 7.6 to 83 ± 4% at pH 7.0.  Consistent with its effects on survival, acidosis suppressed osteoclast apoptosis at 6 h.  We examd. the possible involvement of the proton-sensing receptor ovarian cancer G protein-coupled receptor 1 (OGR1) in mediating the effects of acid.  Acid-induced rise of [Ca2+]i was inhibited by the OGR1 antagonist Cu2+ and was suppressed in osteoclast-like cells in which OGR1 transcripts were depleted using RNA interference.  These findings support an essential role for OGR1 in acid-induced Ca2+ signaling in osteoclasts.  Addn. of Cu2+ or chelation of cytosolic Ca2+ with BAPTA abolished the ability of acidification to enhance osteoclast survival.  Inhibition of NFAT activation with the cell-permeable peptide 11R-VIVIT did not alter the ability of acid to promote survival; however, it suppressed the increase in survival induced by RANKL.  In contrast, inhibition of protein kinase C (PKC) blocked the effect of acid on osteoclast survival.  Thus, this study reveals that extracellular acidification enhances osteoclast survival through an NFAT-independent, PKC-dependent pathway.  Increased osteoclast survival may contribute to bone loss in systemic and local acidosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhDkL1S7QxArVg90H21EOLACvtfcHk0li6d0Vj-qqpSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOrtrrO&md5=e5ce8afa2e263ab2126dddf8e923551f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bone.2007.08.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bone.2007.08.044%26sid%3Dliteratum%253Aachs%26aulast%3DPereverzev%26aufirst%3DA.%26aulast%3DKomarova%26aufirst%3DS.%2BV.%26aulast%3DKorcok%26aufirst%3DJ.%26aulast%3DArmstrong%26aufirst%3DS.%26aulast%3DTremblay%26aufirst%3DG.%2BB.%26aulast%3DDixon%26aufirst%3DS.%2BJ.%26aulast%3DSims%26aufirst%3DS.%2BM.%26atitle%3DExtracellular%2520Acidification%2520Enhances%2520Osteoclast%2520Survival%2520through%2520an%2520NFAT-Independent%252C%2520Protein%2520Kinase%2520C-Dependent%2520Pathway%26jtitle%3DBone%26date%3D2008%26volume%3D42%26spage%3D150%26epage%3D161%26doi%3D10.1016%2Fj.bone.2007.08.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hisada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakakura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamide, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimonji, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsurumaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishizuka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span> <span> </span><span class="NLM_article-title">Proton-Sensing Ovarian Cancer G Protein-Coupled Receptor 1 on Dendritic Cells is Required for Airway Responses in a Murine Asthma Model</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e79985</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0079985</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1371%2Fjournal.pone.0079985" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=24244587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsl2jtLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=H.+Aokiauthor=C.+Mogiauthor=T.+Hisadaauthor=T.+Nakakuraauthor=Y.+Kamideauthor=I.+Ichimonjiauthor=H.+Tomuraauthor=M.+Toboauthor=K.+Satoauthor=H.+Tsurumakiauthor=K.+Dobashiauthor=T.+Moriauthor=A.+Haradaauthor=M.+Yamadaauthor=M.+Moriauthor=T.+Ishizukaauthor=F.+Okajima&title=Proton-Sensing+Ovarian+Cancer+G+Protein-Coupled+Receptor+1+on+Dendritic+Cells+is+Required+for+Airway+Responses+in+a+Murine+Asthma+Model&doi=10.1371%2Fjournal.pone.0079985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Proton-sensing ovarian cancer G protein-coupled receptor 1 on dendritic cells is required for airway responses in a murine asthma model</span></div><div class="casAuthors">Aoki, Haruka; Mogi, Chihiro; Hisada, Takeshi; Nakakura, Takashi; Kamide, Yosuke; Ichimonji, Isao; Tomura, Hideaki; Tobo, Masayuki; Sato, Koichi; Tsurumaki, Hiroaki; Dobashi, Kunio; Mori, Tetsuya; Harada, Akihiro; Yamada, Masanobu; Mori, Masatomo; Ishizuka, Tamotsu; Okajima, Fumikazu</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e79985</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Ovarian cancer G protein-coupled receptor 1 (OGR1) stimulation by extracellular protons causes the activation of G proteins and subsequent cellular functions.  However, the physiol. and pathophysiol. roles of OGR1 in airway responses remain largely unknown.  In the present study, we show that OGR1-deficient mice are resistant to the cardinal features of asthma, including airway eosinophilia, airway hyperresponsiveness (AHR), and goblet cell metaplasia, in assocn. with a remarkable inhibition of Th2 cytokine and IgE prodn., in an ovalbumin (OVA)-induced asthma model.  Intratracheal transfer to wild-type mice of OVA-primed bone marrow-derived dendritic cells (DCs) from OGR1-deficient mice developed lower AHR and eosinophilia after OVA inhalation compared with the transfer of those from wild-type mice.  Migration of OVA-pulsed DCs to peribronchial lymph nodes was also inhibited by OGR1 deficiency in the adoption expts.  The presence of functional OGR1 in DCs was confirmed by the expression of OGR1 mRNA and the OGR1-sensitive Ca2+ response.  OVA-induced expression of CCR7, a mature DC chemokine receptor, and migration response to CCR7 ligands in an in vitro Transwell assay were attenuated by OGR1 deficiency.  We conclude that OGR1 on DCs is crit. for migration to draining lymph nodes, which, in turn, stimulates Th2 phenotype change and subsequent induction of airway inflammation and AHR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNZZP8vtaesrVg90H21EOLACvtfcHk0lj-ergRqZaRsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsl2jtLjN&md5=c42fdcd789144205cf6f5f50c01b6a79</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0079985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0079985%26sid%3Dliteratum%253Aachs%26aulast%3DAoki%26aufirst%3DH.%26aulast%3DMogi%26aufirst%3DC.%26aulast%3DHisada%26aufirst%3DT.%26aulast%3DNakakura%26aufirst%3DT.%26aulast%3DKamide%26aufirst%3DY.%26aulast%3DIchimonji%26aufirst%3DI.%26aulast%3DTomura%26aufirst%3DH.%26aulast%3DTobo%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DTsurumaki%26aufirst%3DH.%26aulast%3DDobashi%26aufirst%3DK.%26aulast%3DMori%26aufirst%3DT.%26aulast%3DHarada%26aufirst%3DA.%26aulast%3DYamada%26aufirst%3DM.%26aulast%3DMori%26aufirst%3DM.%26aulast%3DIshizuka%26aufirst%3DT.%26aulast%3DOkajima%26aufirst%3DF.%26atitle%3DProton-Sensing%2520Ovarian%2520Cancer%2520G%2520Protein-Coupled%2520Receptor%25201%2520on%2520Dendritic%2520Cells%2520is%2520Required%2520for%2520Airway%2520Responses%2520in%2520a%2520Murine%2520Asthma%2520Model%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0079985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakakura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnishi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wayama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span> <span> </span><span class="NLM_article-title">Deficiency of Proton-Sensing Ovarian Cancer G Protein-Coupled Receptor 1 Attenuates Glucose-Stimulated Insulin Secretion</span>. <i>Endocrinology</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">4171</span>– <span class="NLM_lpage">4180</span>, <span class="refDoi"> DOI: 10.1210/en.2012-1164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1210%2Fen.2012-1164" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=22733973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlagtL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2012&pages=4171-4180&author=T.+Nakakuraauthor=C.+Mogiauthor=M.+Toboauthor=H.+Tomuraauthor=K.+Satoauthor=M.+Kobayashiauthor=H.+Ohnishiauthor=S.+Tanakaauthor=M.+Wayamaauthor=T.+Sugiyamaauthor=T.+Kitamuraauthor=A.+Haradaauthor=F.+Okajima&title=Deficiency+of+Proton-Sensing+Ovarian+Cancer+G+Protein-Coupled+Receptor+1+Attenuates+Glucose-Stimulated+Insulin+Secretion&doi=10.1210%2Fen.2012-1164"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Deficiency of proton-sensing ovarian cancer G protein-coupled receptor 1 attenuates glucose-stimulated insulin secretion</span></div><div class="casAuthors">Nakakura, Takashi; Mogi, Chihiro; Tobo, Masayuki; Tomura, Hideaki; Sato, Koichi; Kobayashi, Masaki; Ohnishi, Hiroshi; Tanaka, Shigeyasu; Wayama, Mitsutoshi; Sugiyama, Tetsuya; Kitamura, Tadahiro; Harada, Akihiro; Okajima, Fumikazu</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4171-4180</span>CODEN:
                <span class="NLM_cas:coden">ENDOAO</span>;
        ISSN:<span class="NLM_cas:issn">0013-7227</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Ovarian cancer G protein-coupled receptor 1 (OGR1) has been shown as a receptor for protons.  In the present study, we aimed to know whether OGR1 plays a role in insulin secretion and, if so, the manner in which it does.  To this end, we created OGR1-deficient mice and examd. insulin secretion activity in vivo and in vitro.  OGR1 deficiency reduced insulin secretion induced by glucose administered i.p., although it was not assocd. with glucose intolerance in vivo.  Increased insulin sensitivity and reduced plasma glucagon level may explain, in part, the unusual normal glucose tolerance.  In vitro islet expts. revealed that glucose-stimulated insulin secretion was dependent on extracellular pH and sensitive to OGR1; insulin secretion at pH 7.4 to 7.0, but not 8.0, was significantly suppressed by OGR1 deficiency and inhibition of Gq/11 proteins.  Insulin secretion induced by KCl and tolbutamide was also significantly inhibited, whereas that induced by several insulin secretagogues, including vasopressin, a glucagon-like peptide 1 receptor agonist, and forskolin, was not suppressed by OGR1 deficiency.  The inhibition of insulin secretion was assocd. with the redn. of glucose-induced increase in intracellular Ca2+ concn.  In conclusion, the OGR1/Gq/11 protein pathway is activated by extracellular protons existing under the physiol. extracellular pH of 7.4 and further stimulated by acidification, resulting in the enhancement of insulin secretion in response to high glucose concns. and KCl.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6HbzjeKWwcbVg90H21EOLACvtfcHk0lg1XWolO2j7UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlagtL7L&md5=289021f9a87e717f8261188faf0194ca</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1210%2Fen.2012-1164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2012-1164%26sid%3Dliteratum%253Aachs%26aulast%3DNakakura%26aufirst%3DT.%26aulast%3DMogi%26aufirst%3DC.%26aulast%3DTobo%26aufirst%3DM.%26aulast%3DTomura%26aufirst%3DH.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DKobayashi%26aufirst%3DM.%26aulast%3DOhnishi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DWayama%26aufirst%3DM.%26aulast%3DSugiyama%26aufirst%3DT.%26aulast%3DKitamura%26aufirst%3DT.%26aulast%3DHarada%26aufirst%3DA.%26aulast%3DOkajima%26aufirst%3DF.%26atitle%3DDeficiency%2520of%2520Proton-Sensing%2520Ovarian%2520Cancer%2520G%2520Protein-Coupled%2520Receptor%25201%2520Attenuates%2520Glucose-Stimulated%2520Insulin%2520Secretion%26jtitle%3DEndocrinology%26date%3D2012%26volume%3D153%26spage%3D4171%26epage%3D4180%26doi%3D10.1210%2Fen.2012-1164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohebbi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benabbas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daryadel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourgeois, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, M.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seuwen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, C. A.</span></span> <span> </span><span class="NLM_article-title">The Proton-Activated G Protein Coupled Receptor OGR1 Acutely Regulates the Activity of Epithelial Proton Transport Proteins</span>. <i>Cell. Physiol. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1159/000338486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1159%2F000338486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=22508039" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls12isr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=313-324&author=N.+Mohebbiauthor=C.+Benabbasauthor=S.+Vidalauthor=A.+Daryadelauthor=S.+Bourgeoisauthor=A.+Velicauthor=M.-G.+Ludwigauthor=K.+Seuwenauthor=C.+A.+Wagner&title=The+Proton-Activated+G+Protein+Coupled+Receptor+OGR1+Acutely+Regulates+the+Activity+of+Epithelial+Proton+Transport+Proteins&doi=10.1159%2F000338486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The Proton-activated G Protein Coupled Receptor OGR1 Acutely Regulates the Activity of Epithelial Proton Transport Proteins</span></div><div class="casAuthors">Mohebbi, Nilufar; Benabbas, Chahira; Vidal, Solange; Daryadel, Arezoo; Bourgeois, Soline; Velic, Ana; Ludwig, Marie-Gabrielle; Seuwen, Klaus; Wagner, Carsten A.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Physiology and Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">313-324</span>CODEN:
                <span class="NLM_cas:coden">CEPBEW</span>;
        ISSN:<span class="NLM_cas:issn">1015-8987</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">The Ovarian cancer G protein-coupled Receptor 1 (OGR1; GPR68) is proton-sensitive in the pH range of 6.8 - 7.8.  However, its physiol. function is not defined to date.  OGR1 signals via inositol trisphosphate and intracellular calcium, albeit downstream events are unclear.  To elucidate OGR1 function further, we transfected HEK293 cells with active OGR1 receptor or a mutant lacking 5 histidine residues (H5Phe-OGR1).  An acute switch of extracellular pH from 8 to 7.1 (10 nmol/l vs 90 nmol/l protons) stimulated NHE and H+-ATPase activity in OGR1-transfected cells, but not in H5Phe-OGR1-transfected cells.  ZnCl2 and CuCl2 that both inhibit OGR1 reduced the stimulatory effect.  The activity was blocked by chelerythrine, whereas the ERK1/2 inhibitor PD 098059 had no inhibitory effect.  OGR1 activation increased intracellular calcium in transfected HEK293 cells.  We next isolated proximal tubules from kidneys of wild-type and OGR1-deficient mice and measured the effect of extracellular pH on NHE activity in vitro.  Deletion of OGR1 affected the pH-dependent proton extrusion, however, in the opposite direction as expected from cell culture expts.  Upregulated expression of the pH-sensitive kinase Pyk2 in OGR1 KO mouse proximal tubule cells may compensate for the loss of OGR1.  Thus, we present the first evidence that OGR1 modulates the activity of two major plasma membrane proton transport systems.  OGR1 may be involved in the regulation of plasma membrane transport proteins and intra- and/or extracellular pH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUmm5Hg76aDLVg90H21EOLACvtfcHk0lg1XWolO2j7UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls12isr0%253D&md5=bd394fbc4729548e3a1f6f09a14c3ef6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1159%2F000338486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000338486%26sid%3Dliteratum%253Aachs%26aulast%3DMohebbi%26aufirst%3DN.%26aulast%3DBenabbas%26aufirst%3DC.%26aulast%3DVidal%26aufirst%3DS.%26aulast%3DDaryadel%26aufirst%3DA.%26aulast%3DBourgeois%26aufirst%3DS.%26aulast%3DVelic%26aufirst%3DA.%26aulast%3DLudwig%26aufirst%3DM.-G.%26aulast%3DSeuwen%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DC.%2BA.%26atitle%3DThe%2520Proton-Activated%2520G%2520Protein%2520Coupled%2520Receptor%2520OGR1%2520Acutely%2520Regulates%2520the%2520Activity%2520of%2520Epithelial%2520Proton%2520Transport%2520Proteins%26jtitle%3DCell.%2520Physiol.%2520Biochem.%26date%3D2012%26volume%3D29%26spage%3D313%26epage%3D324%26doi%3D10.1159%2F000338486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Vallière, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clay, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jurisic, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tcymbarevich, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, M.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okoniewski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eloranta, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kullak-Ublick, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seuwen, K.</span></span> <span> </span><span class="NLM_article-title">The pH-Sensing Receptor OGR1 Improves Barrier Function of Epithelial Cells and Inhibits Migration in an Acidic Environment</span>. <i>Am. J. Physiol.: Gastrointest. Liver Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>309</i></span>,  <span class="NLM_fpage">G475</span>– <span class="NLM_lpage">G490</span>, <span class="refDoi"> DOI: 10.1152/ajpgi.00408.2014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1152%2Fajpgi.00408.2014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=26206859" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=309&publication_year=2015&pages=G475-G490&author=C.+de%0AValli%C3%A8reauthor=S.+Vidalauthor=I.+Clayauthor=G.+Jurisicauthor=I.+Tcymbarevichauthor=S.+Langauthor=M.-G.+Ludwigauthor=M.+Okoniewskiauthor=J.+J.+Elorantaauthor=G.+A.+Kullak-Ublickauthor=C.+A.+Wagnerauthor=G.+Roglerauthor=K.+Seuwen&title=The+pH-Sensing+Receptor+OGR1+Improves+Barrier+Function+of+Epithelial+Cells+and+Inhibits+Migration+in+an+Acidic+Environment&doi=10.1152%2Fajpgi.00408.2014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1152%2Fajpgi.00408.2014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpgi.00408.2014%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BValli%25C3%25A8re%26aufirst%3DC.%26aulast%3DVidal%26aufirst%3DS.%26aulast%3DClay%26aufirst%3DI.%26aulast%3DJurisic%26aufirst%3DG.%26aulast%3DTcymbarevich%26aufirst%3DI.%26aulast%3DLang%26aufirst%3DS.%26aulast%3DLudwig%26aufirst%3DM.-G.%26aulast%3DOkoniewski%26aufirst%3DM.%26aulast%3DEloranta%26aufirst%3DJ.%2BJ.%26aulast%3DKullak-Ublick%26aufirst%3DG.%2BA.%26aulast%3DWagner%26aufirst%3DC.%2BA.%26aulast%3DRogler%26aufirst%3DG.%26aulast%3DSeuwen%26aufirst%3DK.%26atitle%3DThe%2520pH-Sensing%2520Receptor%2520OGR1%2520Improves%2520Barrier%2520Function%2520of%2520Epithelial%2520Cells%2520and%2520Inhibits%2520Migration%2520in%2520an%2520Acidic%2520Environment%26jtitle%3DAm.%2520J.%2520Physiol.%253A%2520Gastrointest.%2520Liver%2520Physiol.%26date%3D2015%26volume%3D309%26spage%3DG475%26epage%3DG490%26doi%3D10.1152%2Fajpgi.00408.2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vessières, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nonomura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimaud, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francisco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, F.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainquist, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cahalan, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orth, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordone, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laffitte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrion, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patapoutian, A.</span></span> <span> </span><span class="NLM_article-title">GPR68 Senses Flow and Is Essential for Vascular Physiology</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">762.e16</span>– <span class="NLM_lpage">775.e16</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.03.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.cell.2018.03.076" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=762.e16-775.e16&author=J.+Xuauthor=J.+Mathurauthor=E.+Vessi%C3%A8resauthor=S.+Hammackauthor=K.+Nonomuraauthor=J.+Favreauthor=L.+Grimaudauthor=M.+Petrusauthor=A.+Franciscoauthor=J.+Liauthor=V.+Leeauthor=F.-L.+Xiangauthor=J.+K.+Mainquistauthor=S.+M.+Cahalanauthor=A.+P.+Orthauthor=J.+R.+Walkerauthor=S.+Maauthor=V.+Lukacsauthor=L.+Bordoneauthor=M.+Bandellauthor=B.+Laffitteauthor=Y.+Xuauthor=S.+Chienauthor=D.+Henrionauthor=A.+Patapoutian&title=GPR68+Senses+Flow+and+Is+Essential+for+Vascular+Physiology&doi=10.1016%2Fj.cell.2018.03.076"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.03.076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.03.076%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DMathur%26aufirst%3DJ.%26aulast%3DVessi%25C3%25A8res%26aufirst%3DE.%26aulast%3DHammack%26aufirst%3DS.%26aulast%3DNonomura%26aufirst%3DK.%26aulast%3DFavre%26aufirst%3DJ.%26aulast%3DGrimaud%26aufirst%3DL.%26aulast%3DPetrus%26aufirst%3DM.%26aulast%3DFrancisco%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DV.%26aulast%3DXiang%26aufirst%3DF.-L.%26aulast%3DMainquist%26aufirst%3DJ.%2BK.%26aulast%3DCahalan%26aufirst%3DS.%2BM.%26aulast%3DOrth%26aufirst%3DA.%2BP.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DLukacs%26aufirst%3DV.%26aulast%3DBordone%26aufirst%3DL.%26aulast%3DBandell%26aufirst%3DM.%26aulast%3DLaffitte%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DChien%26aufirst%3DS.%26aulast%3DHenrion%26aufirst%3DD.%26aulast%3DPatapoutian%26aufirst%3DA.%26atitle%3DGPR68%2520Senses%2520Flow%2520and%2520Is%2520Essential%2520for%2520Vascular%2520Physiology%26jtitle%3DCell%26date%3D2018%26volume%3D173%26spage%3D762.e16%26epage%3D775.e16%26doi%3D10.1016%2Fj.cell.2018.03.076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span> <i>Illuminating the Druggable Genome | NIH Common
Fund</i>, <span class="NLM_year">2019</span>. <a href="https://www.commonfund.nih.gov/idg/" class="extLink">https://www.commonfund.nih.gov/idg/</a> (accessed May 7, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Illuminating+the+Druggable+Genome+%7C+NIH+Common%0AFund%2C+2019.+https%3A%2F%2Fwww.commonfund.nih.gov%2Fidg%2F+%28accessed+May+7%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DIlluminating%2520the%2520Druggable%2520Genome%2520%257C%2520NIH%2520Common%250AFund%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiley, S. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sriram, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmerón, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insel, P. A.</span></span> <span> </span><span class="NLM_article-title">GPR68: An Emerging Drug Target in Cancer</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>, <span class="NLM_elocation-id">559</span> <span class="refDoi"> DOI: 10.3390/ijms20030559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.3390%2Fijms20030559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlSrs7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&author=S.+Z.+Wileyauthor=K.+Sriramauthor=C.+Salmer%C3%B3nauthor=P.+A.+Insel&title=GPR68%3A+An+Emerging+Drug+Target+in+Cancer&doi=10.3390%2Fijms20030559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">GPR68: an emerging drug target in cancer</span></div><div class="casAuthors">Wiley, Shu Z.; Sriram, Krishna; Salmeron, Cristina; Insel, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">559</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  GPR68 (or ovarian cancer G protein-coupled receptor 1, OGR1) is a proton-sensing G-protein-coupled receptor (GPCR) that responds to extracellular acidity and regulates a variety of cellular functions.  Acidosis is considered a defining hallmark of the tumor microenvironment (TME).  GPR68 expression is highly upregulated in numerous types of cancer.  Emerging evidence has revealed that GPR68 may play crucial roles in tumor biol., including tumorigenesis, tumor growth, and metastasis.  This review summarizes current knowledge regarding GPR68-its expression, regulation, signaling pathways, physiol. roles, and functions it regulates in human cancers (including prostate, colon and pancreatic cancer, melanoma, medulloblastoma, and myelodysplastic syndrome).  The findings provide evidence for GPR68 as a potentially novel therapeutic target but in addn., we note challenges in developing drugs that target GPR68.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo73vTsxafymbVg90H21EOLACvtfcHk0lhUsNnItZdkVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlSrs7zI&md5=c0c4080c5c2dfd0f9c32c3fa8bd74702</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3390%2Fijms20030559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20030559%26sid%3Dliteratum%253Aachs%26aulast%3DWiley%26aufirst%3DS.%2BZ.%26aulast%3DSriram%26aufirst%3DK.%26aulast%3DSalmer%25C3%25B3n%26aufirst%3DC.%26aulast%3DInsel%26aufirst%3DP.%2BA.%26atitle%3DGPR68%253A%2520An%2520Emerging%2520Drug%2520Target%2520in%2520Cancer%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26doi%3D10.3390%2Fijms20030559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Allosteric Modulators of G-Protein-Coupled Receptors: Drug-Target Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01844</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01844" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFCgtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=24-45&author=S.+Luauthor=J.+Zhang&title=Small+Molecule+Allosteric+Modulators+of+G-Protein-Coupled+Receptors%3A+Drug-Target+Interactions&doi=10.1021%2Facs.jmedchem.7b01844"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Allosteric Modulators of G-Protein-Coupled Receptors: Drug-Target Interactions</span></div><div class="casAuthors">Lu, Shaoyong; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-45</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are the largest class of signaling receptors that are most frequently targeted by therapeutic drugs.  Allosteric modulators bound to GPCRs at allosteric sites provide the potential for differential selectivity and improved safety compared with traditional orthosteric ligands.  The recent breakthroughs in GPCR structural biol. have made structures of GPCRs from classes A, B, C, and F complexed with small-mol. allosteric modulators available.  Knowledge of the detailed receptor-modulator interactions at the allosteric sites is useful for structure-based GPCR drug design of novel therapeutics.  This Perspective comprehensively summarizes the current status of structural complexes between GPCRs and their small-mol. allosteric modulators, particularly the key receptor-modulator interactions at the allosteric sites.  Then, the structural diversity of allosteric sites across four GPCR subfamilies is compared.  This study is expected to contribute to the design of GPCR allosteric drugs with an improved therapeutic action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVDf1DtcU6ZLVg90H21EOLACvtfcHk0lhUsNnItZdkVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFCgtb0%253D&md5=7a41f4c00dadecd0661d135811644ad8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01844%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DSmall%2520Molecule%2520Allosteric%2520Modulators%2520of%2520G-Protein-Coupled%2520Receptors%253A%2520Drug-Target%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D24%26epage%3D45%26doi%3D10.1021%2Facs.jmedchem.7b01844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coughlin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopper, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanco, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirunagaru, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robichaud, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doller, D.</span></span> <span> </span><span class="NLM_article-title">Allosteric Modalities for Membrane-Bound Receptors: Insights from Drug Hunting for Brain Diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">5979</span>– <span class="NLM_lpage">6002</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01651</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01651" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisFyhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=5979-6002&author=Q.+Coughlinauthor=A.+T.+Hopperauthor=M.-J.+Blancoauthor=V.+Tirunagaruauthor=A.+J.+Robichaudauthor=D.+Doller&title=Allosteric+Modalities+for+Membrane-Bound+Receptors%3A+Insights+from+Drug+Hunting+for+Brain+Diseases&doi=10.1021%2Facs.jmedchem.8b01651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Modalities for Membrane-Bound Receptors: Insights from Drug Hunting for Brain Diseases</span></div><div class="casAuthors">Coughlin, Quinn; Hopper, Allen T.; Blanco, Maria-Jesus; Tirunagaru, Vijaya; Robichaud, Albert J.; Doller, Dario</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5979-6002</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Medicinal chemists are accountable for embedding the appropriate drug target profile into the mol. architecture of a clin. candidate.  An accurate characterization of the functional effects following binding of a drug to its biol. target is a fundamental step in the discovery of new medicines, informing the translation of preclin. efficacy and safety observations into human trials.  Membrane-bound proteins, particularly ion channels and G protein-coupled receptors (GPCRs), are biol. targets prone to allosteric modulation.  Investigations using allosteric drug candidates and chem. tools suggest that their functional effects may be tailored with a high degree of translational alignment, making them mol. tools to correct pathophysiol. functional tone and enable personalized medicine when a causative target-to-disease link is known.  We present select examples of functional mol. fine-tuning of allosterism and discuss consequences relevant to drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG7wCMbhExHrVg90H21EOLACvtfcHk0lgEketYbpVpRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisFyhs70%253D&md5=38fbd2cbc62615804f1f812b27fae13f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01651%26sid%3Dliteratum%253Aachs%26aulast%3DCoughlin%26aufirst%3DQ.%26aulast%3DHopper%26aufirst%3DA.%2BT.%26aulast%3DBlanco%26aufirst%3DM.-J.%26aulast%3DTirunagaru%26aufirst%3DV.%26aulast%3DRobichaud%26aufirst%3DA.%2BJ.%26aulast%3DDoller%26aufirst%3DD.%26atitle%3DAllosteric%2520Modalities%2520for%2520Membrane-Bound%2520Receptors%253A%2520Insights%2520from%2520Drug%2520Hunting%2520for%2520Brain%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26spage%3D5979%26epage%3D6002%26doi%3D10.1021%2Facs.jmedchem.8b01651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T.</span></span> <span> </span><span class="NLM_article-title">New Concepts in Drug Discovery: Collateral Efficacy and Permissive Antagonism</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">919</span>– <span class="NLM_lpage">927</span>, <span class="refDoi"> DOI: 10.1038/nrd1875</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fnrd1875" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=16264435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=919-927&author=T.+Kenakin&title=New+Concepts+in+Drug+Discovery%3A+Collateral+Efficacy+and+Permissive+Antagonism&doi=10.1038%2Fnrd1875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">New Concepts in Drug Discovery: Collateral Efficacy and Permissive Antagonism</span></div><div class="casAuthors">Kenakin, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">919-927</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  New perspectives on the complexity of G-protein-coupled receptor (GPCR) signaling and the increased resoln. of existing tools for studying GPCR behavior has led to the conception of new hypotheses that affect the discovery of drugs acting at GPCRs.  Taking into consideration the novel concepts of collateral efficacy and permissive antagonism in the search for synthetic agonists and antagonists, resp., will be essential in the search for drugs with unique therapeutic profiles.  Here, the design of drugs against HIV is used as an example of how these concepts might be taken into consideration for GPCR-targeted drugs in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp26TMWCy0fUrVg90H21EOLACvtfcHk0lgEketYbpVpRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsLfO&md5=b7e09bcb4f73d52b4ab80f521ab78a64</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnrd1875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1875%26sid%3Dliteratum%253Aachs%26aulast%3DKenakin%26aufirst%3DT.%26atitle%3DNew%2520Concepts%2520in%2520Drug%2520Discovery%253A%2520Collateral%2520Efficacy%2520and%2520Permissive%2520Antagonism%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2005%26volume%3D4%26spage%3D919%26epage%3D927%26doi%3D10.1038%2Fnrd1875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ehlert, F. J.</span></span> <span> </span><span class="NLM_article-title">Analysis of Allosterism in Functional Assays</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>315</i></span>,  <span class="NLM_fpage">740</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1124/jpet.105.090886</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fjpet.105.090886" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=16046613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFyhtr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=315&publication_year=2005&pages=740-754&author=F.+J.+Ehlert&title=Analysis+of+Allosterism+in+Functional+Assays&doi=10.1124%2Fjpet.105.090886"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of allosterism in functional assays</span></div><div class="casAuthors">Ehlert, Frederick J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">315</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">740-754</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The theor. basis for analyzing the effects of an allosteric modulator on the response to an agonist is described.  The effects of an allosteric modulator on the concn.-response curve to an agonist can be attributed to changes in the obsd. dissocn. const. and intrinsic efficacy of the agonist-receptor complex.  Each of these two changes can be represented by a coeff. or factor.  It is possible to est. the ratio of the coeff. of change in agonist efficacy divided by that for the agonist dissocn. const.  This ratio is designated as the relative activity (RA) of the agonist in the presence of the allosteric modulator.  The RA value can be estd. for each concn. of allosteric modulator by nonlinear regression anal., regardless of the shape of the concn.-response curve.  Regression anal. of the RA values against the concn. of allosteric modulator yields ests. of the dissocn. const. (KA) of the allosteric modulator and the maximal RA value.  If the concn.-response curve to the agonist obeys a logistic function and the allosteric modulator influences the maximal response, it is possible to distinguish between the maximal change in affinity from that of efficacy.  If the agonist concn.-response curve obeys a logistic equation with a Hill slope of 1, the RA values can be estd. easily from the agonist EC50 and Emax values.  This anal. illustrates the utility of the RA value in quantifying allosteric effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKvK9-uIW607Vg90H21EOLACvtfcHk0lgC-KaEBge_4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFyhtr%252FO&md5=0cb5a02f65f93990511d12fb06f5da12</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1124%2Fjpet.105.090886&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.105.090886%26sid%3Dliteratum%253Aachs%26aulast%3DEhlert%26aufirst%3DF.%2BJ.%26atitle%3DAnalysis%2520of%2520Allosterism%2520in%2520Functional%2520Assays%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2005%26volume%3D315%26spage%3D740%26epage%3D754%26doi%3D10.1124%2Fjpet.105.090886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntosh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marrs, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pertwee, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, R. A.</span></span> <span> </span><span class="NLM_article-title">Allosteric Modulation of the Cannabinoid CB1 Receptor</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">1484</span>– <span class="NLM_lpage">1495</span>, <span class="refDoi"> DOI: 10.1124/mol.105.016162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fmol.105.016162" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=16113085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtF2it7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2005&pages=1484-1495&author=M.+R.+Priceauthor=G.+L.+Baillieauthor=A.+Thomasauthor=L.+A.+Stevensonauthor=M.+Eassonauthor=R.+Goodwinauthor=A.+McLeanauthor=L.+McIntoshauthor=G.+Goodwinauthor=G.+Walkerauthor=P.+Westwoodauthor=J.+Marrsauthor=F.+Thomsonauthor=P.+Cowleyauthor=A.+Christopoulosauthor=R.+G.+Pertweeauthor=R.+A.+Ross&title=Allosteric+Modulation+of+the+Cannabinoid+CB1+Receptor&doi=10.1124%2Fmol.105.016162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric modulation of the cannabinoid CB1 receptor</span></div><div class="casAuthors">Price, Martin R.; Baillie, Gemma L.; Thomas, Adele; Stevenson, Lesley A.; Easson, Morag; Goodwin, Richard; McLean, Adele; McIntosh, Lorraine; Goodwin, Gillian; Walker, Glenn; Westwood, Paul; Marrs, Julia; Thomson, Fiona; Cowley, Phillip; Christopoulos, Arthur; Pertwee, Roger G.; Ross, Ruth A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1484-1495</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The authors investigated the pharmacol. of three novel compds., Org 27569, Org 27759 and Org 29647 at the cannabinoid CB1 receptor.  In equil. binding assays, the Org compds. significantly increased the binding of the CB1 receptor agonist [3H]CP 55,940 [(1R,3R,4R)-3-[2-hydroxy-4-(1,1-dimethylheptyl)phenyl]-4-(3-hydroxypropyl) cyclohexan-1-ol], indicative of a pos. cooperative allosteric effect.  The same compds. caused a significant, but incomplete, decrease in the specific binding of the CB1 receptor inverse agonist [3H]SR 141716A [N-(piperidin-1-yl)-5-(4-chlorophenyl)-1-(2,4-dichlorophenyl)-4-methyl-1H -pyrazole-3-carboximide hydrochloride], indicative of a limited neg. binding cooperativity.  Anal. of the data according to an allosteric ternary complex model revealed that the estd. affinity of each Org compd. was not significantly different when the radioligand was [3H]CP 55,940 or [3H]SR 141716A.  However, the estd. cooperatively factor for the interaction between modulator and radioligand was greater than 1 when detd. against [3H]CP 55,940 and less than 1 when detd. against [3H]SR 141716A.  [3H]CP 55,940 dissocn. kinetic studies also validated the allosteric nature of the Org compds., because they all significantly decreased radioligand dissocn.  These data suggest that the Org compds. bind allosterically to the CB1 receptor and elicit a conformational change that increases agonist affinity for the orthosteric binding site.  In contrast to the binding assays, however, the Org compds. behaved as insurmountable antagonists of receptor function; in the reporter gene assay, the guanosine 5'-O-(3-[35S]thio)triphosphate binding assay and the mouse vas deferens assay they elicited a significant redn. in the Emax value for CB1 receptor agonists.  The data presented clearly demonstrate, for the first time, that the cannabinoid CB1 receptor contains an allosteric binding site that can be recognized by synthetic small mol. ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq98nUa2ffQ7bVg90H21EOLACvtfcHk0lgC-KaEBge_4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtF2it7nM&md5=a342d304a8157b1e7b962755a775154d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1124%2Fmol.105.016162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.105.016162%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DM.%2BR.%26aulast%3DBaillie%26aufirst%3DG.%2BL.%26aulast%3DThomas%26aufirst%3DA.%26aulast%3DStevenson%26aufirst%3DL.%2BA.%26aulast%3DEasson%26aufirst%3DM.%26aulast%3DGoodwin%26aufirst%3DR.%26aulast%3DMcLean%26aufirst%3DA.%26aulast%3DMcIntosh%26aufirst%3DL.%26aulast%3DGoodwin%26aufirst%3DG.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DWestwood%26aufirst%3DP.%26aulast%3DMarrs%26aufirst%3DJ.%26aulast%3DThomson%26aufirst%3DF.%26aulast%3DCowley%26aufirst%3DP.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DPertwee%26aufirst%3DR.%2BG.%26aulast%3DRoss%26aufirst%3DR.%2BA.%26atitle%3DAllosteric%2520Modulation%2520of%2520the%2520Cannabinoid%2520CB1%2520Receptor%26jtitle%3DMol.%2520Pharmacol.%26date%3D2005%26volume%3D68%26spage%3D1484%26epage%3D1495%26doi%3D10.1124%2Fmol.105.016162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuano, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Pharmacological Profiling of Analogues of Benzyl Quinolone Carboxylic Acid (BQCA) as Allosteric Modulators of the M1 Muscarinic Receptor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5151</span>– <span class="NLM_lpage">5172</span>, <span class="refDoi"> DOI: 10.1021/jm400540b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400540b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVaqtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=5151-5172&author=S.+N.+Mistryauthor=C.+Valantauthor=P.+M.+Sextonauthor=B.+Capuanoauthor=A.+Christopoulosauthor=P.+J.+Scammells&title=Synthesis+and+Pharmacological+Profiling+of+Analogues+of+Benzyl+Quinolone+Carboxylic+Acid+%28BQCA%29+as+Allosteric+Modulators+of+the+M1+Muscarinic+Receptor&doi=10.1021%2Fjm400540b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Pharmacological Profiling of Analogues of Benzyl Quinolone Carboxylic Acid (BQCA) as Allosteric Modulators of the M1 Muscarinic Receptor</span></div><div class="casAuthors">Mistry, Shailesh N.; Valant, Celine; Sexton, Patrick M.; Capuano, Ben; Christopoulos, Arthur; Scammells, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5151-5172</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Established therapy in Alzheimer's disease involves potentiation of the endogenous orthosteric ligand, acetylcholine, at the M1 muscarinic receptors found in higher concns. in the cortex and hippocampus.  Adverse effects, due to indiscriminate activation of other muscarinic receptor subtypes, are common.  M1 muscarinic pos. allosteric modulators/allosteric agonists such as BQCA [1-(4-methoxybenzyl)-4-oxo-1,4-dihydroquinoline-3-carboxylic acid, I] offer an attractive soln., being exquisitely M1-selective over other muscarinic subtypes.  A common difficulty with allosteric ligands is interpreting SAR, based on composite potency values derived in the presence of fixed concn. of agonist.  In reality these values encompass multiple pharmacol. parameters, each potentially and differentially sensitive to structural modification of the ligand.  We report novel BQCA analogs which appear to augment ligand affinity for the receptor (pKB), intrinsic efficacy (τB), and both binding (α) and functional (β) cooperativity with acetylcholine.  Ultimately, development of such enriched SAR surrounding allosteric modulators will provide insight into their mode of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoigST7OJR7vbVg90H21EOLACvtfcHk0li3XCyDGCoiXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVaqtbs%253D&md5=ab134e0b5400aa825916695eaf1203b1</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm400540b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400540b%26sid%3Dliteratum%253Aachs%26aulast%3DMistry%26aufirst%3DS.%2BN.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DCapuano%26aufirst%3DB.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26atitle%3DSynthesis%2520and%2520Pharmacological%2520Profiling%2520of%2520Analogues%2520of%2520Benzyl%2520Quinolone%2520Carboxylic%2520Acid%2520%2528BQCA%2529%2520as%2520Allosteric%2520Modulators%2520of%2520the%2520M1%2520Muscarinic%2520Receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D5151%26epage%3D5172%26doi%3D10.1021%2Fjm400540b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourgognon, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanger, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verity, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogg, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butcher, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molloy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macedo-Hatch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawlak, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broad, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinert, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallucci, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobin, A. B.</span></span> <span> </span><span class="NLM_article-title">M1 Muscarinic Allosteric Modulators Slow Prion Neurodegeneration and Restore Memory Loss</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">487</span>– <span class="NLM_lpage">499</span>, <span class="refDoi"> DOI: 10.1172/JCI87526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1172%2FJCI87526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=27991860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BC1c%252FmtF2hug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=487-499&author=S.+J.+Bradleyauthor=J.-M.+Bourgognonauthor=H.+E.+Sangerauthor=N.+Verityauthor=A.+J.+Moggauthor=D.+J.+Whiteauthor=A.+J.+Butcherauthor=J.+A.+Morenoauthor=C.+Molloyauthor=T.+Macedo-Hatchauthor=J.+M.+Edwardsauthor=J.+Wessauthor=R.+Pawlakauthor=D.+J.+Readauthor=P.+M.+Sextonauthor=L.+M.+Broadauthor=J.+R.+Steinertauthor=G.+R.+Mallucciauthor=A.+Christopoulosauthor=C.+C.+Felderauthor=A.+B.+Tobin&title=M1+Muscarinic+Allosteric+Modulators+Slow+Prion+Neurodegeneration+and+Restore+Memory+Loss&doi=10.1172%2FJCI87526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">M1 muscarinic allosteric modulators slow prion neurodegeneration and restore memory loss</span></div><div class="casAuthors">Bradley Sophie J; Bourgognon Julie-Myrtille; Sanger Helen E; Verity Nicholas; Mogg Adrian J; White David J; Butcher Adrian J; Moreno Julie A; Molloy Colin; Macedo-Hatch Timothy; Edwards Jennifer M; Wess Jurgen; Pawlak Robert; Read David J; Sexton Patrick M; Broad Lisa M; Steinert Joern R; Mallucci Giovanna R; Christopoulos Arthur; Felder Christian C; Tobin Andrew B</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">487-499</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The current frontline symptomatic treatment for Alzheimer's disease (AD) is whole-body upregulation of cholinergic transmission via inhibition of acetylcholinesterase.  This approach leads to profound dose-related adverse effects.  An alternative strategy is to selectively target muscarinic acetylcholine receptors, particularly the M1 muscarinic acetylcholine receptor (M1 mAChR), which was previously shown to have procognitive activity.  However, developing M1 mAChR-selective orthosteric ligands has proven challenging.  Here, we have shown that mouse prion disease shows many of the hallmarks of human AD, including progressive terminal neurodegeneration and memory deficits due to a disruption of hippocampal cholinergic innervation.  The fact that we also show that muscarinic signaling is maintained in both AD and mouse prion disease points to the latter as an excellent model for testing the efficacy of muscarinic pharmacological entities.  The memory deficits we observed in mouse prion disease were completely restored by treatment with benzyl quinolone carboxylic acid (BQCA) and benzoquinazoline-12 (BQZ-12), two highly selective positive allosteric modulators (PAMs) of M1 mAChRs.  Furthermore, prolonged exposure to BQCA markedly extended the lifespan of diseased mice.  Thus, enhancing hippocampal muscarinic signaling using M1 mAChR PAMs restored memory loss and slowed the progression of mouse prion disease, indicating that this ligand type may have clinical benefit in diseases showing defective cholinergic transmission, such as AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ6VBjSjCSsKp0d-O_J5C0AfW6udTcc2eaWKrpxWjegNrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c%252FmtF2hug%253D%253D&md5=b8384618de37b3f83676ea77911a7625</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1172%2FJCI87526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI87526%26sid%3Dliteratum%253Aachs%26aulast%3DBradley%26aufirst%3DS.%2BJ.%26aulast%3DBourgognon%26aufirst%3DJ.-M.%26aulast%3DSanger%26aufirst%3DH.%2BE.%26aulast%3DVerity%26aufirst%3DN.%26aulast%3DMogg%26aufirst%3DA.%2BJ.%26aulast%3DWhite%26aufirst%3DD.%2BJ.%26aulast%3DButcher%26aufirst%3DA.%2BJ.%26aulast%3DMoreno%26aufirst%3DJ.%2BA.%26aulast%3DMolloy%26aufirst%3DC.%26aulast%3DMacedo-Hatch%26aufirst%3DT.%26aulast%3DEdwards%26aufirst%3DJ.%2BM.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DPawlak%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DD.%2BJ.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DBroad%26aufirst%3DL.%2BM.%26aulast%3DSteinert%26aufirst%3DJ.%2BR.%26aulast%3DMallucci%26aufirst%3DG.%2BR.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DFelder%26aufirst%3DC.%2BC.%26aulast%3DTobin%26aufirst%3DA.%2BB.%26atitle%3DM1%2520Muscarinic%2520Allosteric%2520Modulators%2520Slow%2520Prion%2520Neurodegeneration%2520and%2520Restore%2520Memory%2520Loss%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2017%26volume%3D127%26spage%3D487%26epage%3D499%26doi%3D10.1172%2FJCI87526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T.</span></span> <span> </span><span class="NLM_article-title">A Scale of Agonism and Allosteric Modulation for Assessment of Selectivity, Bias, and Receptor Mutation</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">414</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1124/mol.117.108787</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fmol.117.108787" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=28679508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVyiurzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2017&pages=414-424&author=T.+Kenakin&title=A+Scale+of+Agonism+and+Allosteric+Modulation+for+Assessment+of+Selectivity%2C+Bias%2C+and+Receptor+Mutation&doi=10.1124%2Fmol.117.108787"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A scale of agonism and allosteric modulation for assessment of selectivity, bias, and receptor mutation</span></div><div class="casAuthors">Kenakin, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">414-424</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">An index of agonism is described that can be used to quantify agonist receptor selectivity, bias, cell-based agonism, and the effects of receptor mutation on signaling.  The parameter is derived from agonist concn.-response curves and comprises the maximal response to the agonist (max) and the EC50 in the form of Δlog(max/EC50).  This parameter is derived from equations describing agonists as pos. allosteric facilitators of receptor-signaling protein interaction.  A similar index is also derived to quantify the potentiating effects of pos. allosteric modulators, which can be used to quantify in situ pos. allosteric modulator activity in vivo.  These indexes lend themselves to statistical anal. and are system-independent in that the effects of the system processing of agonist response and differences in assay sensitivity and receptor expression are cancelled.  The various applications of the Δlog(max/EC50) scale are described for each pharmacol. application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5mhzE_5MqrLVg90H21EOLACvtfcHk0lj_bjj-CYuE4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVyiurzP&md5=f1e35110d4347fe685cb776be30b5d79</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1124%2Fmol.117.108787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.117.108787%26sid%3Dliteratum%253Aachs%26aulast%3DKenakin%26aufirst%3DT.%26atitle%3DA%2520Scale%2520of%2520Agonism%2520and%2520Allosteric%2520Modulation%2520for%2520Assessment%2520of%2520Selectivity%252C%2520Bias%252C%2520and%2520Receptor%2520Mutation%26jtitle%3DMol.%2520Pharmacol.%26date%3D2017%26volume%3D92%26spage%3D414%26epage%3D424%26doi%3D10.1124%2Fmol.117.108787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jörg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinh, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuano, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span> <span> </span><span class="NLM_article-title">4-Phenylpyridin-2-one Derivatives: A Novel Class of Positive Allosteric Modulator of the M1 Muscarinic Acetylcholine Receptor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">388</span>– <span class="NLM_lpage">409</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01562</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01562" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFegu7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=388-409&author=S.+N.+Mistryauthor=M.+J%C3%B6rgauthor=H.+Limauthor=N.+B.+Vinhauthor=P.+M.+Sextonauthor=B.+Capuanoauthor=A.+Christopoulosauthor=J.+R.+Laneauthor=P.+J.+Scammells&title=4-Phenylpyridin-2-one+Derivatives%3A+A+Novel+Class+of+Positive+Allosteric+Modulator+of+the+M1+Muscarinic+Acetylcholine+Receptor&doi=10.1021%2Facs.jmedchem.5b01562"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">4-Phenylpyridin-2-one Derivatives: A Novel Class of Positive Allosteric Modulator of the M1 Muscarinic Acetylcholine Receptor</span></div><div class="casAuthors">Mistry, Shailesh N.; Jorg, Manuela; Lim, Herman; Vinh, Natalie B.; Sexton, Patrick M.; Capuano, Ben; Christopoulos, Arthur; Lane, J. Robert; Scammells, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">388-409</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pos. allosteric modulators (PAMs) of the M1 muscarinic acetylcholine receptor (M1 mAChR) are a promising strategy for the treatment of the cognitive deficits assocd. with diseases including Alzheimer's and schizophrenia.  Herein, we report the design, synthesis, and characterization of a novel family of M1 mAChR PAMs.  The most active compds. of the 4-phenylpyridin-2-one series exhibited comparable binding affinity to the ref. compd., 1-(4-methoxybenzyl)-4-oxo-1,4-dihydroquinoline-3-carboxylic acid (BQCA), but markedly improved pos. cooperativity with acetylcholine, and retained exquisite selectivity for the M1 mAChR.  Furthermore, the pharmacol. characterization revealed ligands with a diverse range of activities, including modulators that displayed both high intrinsic efficacy and PAM activity, those that showed no detectable agonism but robust PAM activity and ligands that displayed robust allosteric agonism but little modulatory activity.  Compd. I was found to have the best pharmacol. profile.  Thus, the 4-phenylpyridin-2-one scaffold offers an attractive starting point for further lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMlFmtgndXrbVg90H21EOLACvtfcHk0lj_bjj-CYuE4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFegu7zP&md5=315362c43e2b6c204f038f30c6ade642</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01562&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01562%26sid%3Dliteratum%253Aachs%26aulast%3DMistry%26aufirst%3DS.%2BN.%26aulast%3DJ%25C3%25B6rg%26aufirst%3DM.%26aulast%3DLim%26aufirst%3DH.%26aulast%3DVinh%26aufirst%3DN.%2BB.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DCapuano%26aufirst%3DB.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26atitle%3D4-Phenylpyridin-2-one%2520Derivatives%253A%2520A%2520Novel%2520Class%2520of%2520Positive%2520Allosteric%2520Modulator%2520of%2520the%2520M1%2520Muscarinic%2520Acetylcholine%2520Receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D388%26epage%3D409%26doi%3D10.1021%2Facs.jmedchem.5b01562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Russell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetsch, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Rooij, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frantz, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, J. W.</span></span> <span> </span><span class="NLM_article-title">Regulated Expression of pH Sensing G Protein-Coupled Receptor-68 Identified through Chemical Biology Defines a New Drug Target for Ischemic Heart Disease</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1077</span>– <span class="NLM_lpage">1083</span>, <span class="refDoi"> DOI: 10.1021/cb300001m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300001m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVWls7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1077-1083&author=J.+L.+Russellauthor=S.+C.+Goetschauthor=H.+R.+Aguilarauthor=H.+Coeauthor=X.+Luoauthor=N.+Liuauthor=E.+van+Rooijauthor=D.+E.+Frantzauthor=J.+W.+Schneider&title=Regulated+Expression+of+pH+Sensing+G+Protein-Coupled+Receptor-68+Identified+through+Chemical+Biology+Defines+a+New+Drug+Target+for+Ischemic+Heart+Disease&doi=10.1021%2Fcb300001m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Regulated Expression of pH Sensing G Protein-Coupled Receptor-68 Identified through Chemical Biology Defines a New Drug Target for Ischemic Heart Disease</span></div><div class="casAuthors">Russell, Jamie L.; Goetsch, Sean C.; Aguilar, Hector R.; Coe, Helen; Luo, Xiang; Liu, Ning; van Rooij, Eva; Frantz, Doug E.; Schneider, Jay W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1077-1083</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. biol. promises discovery of new and unexpected mechanistic pathways, protein functions and disease targets.  Here, we probed the mechanism-of-action and protein targets of 3,5-disubstituted isoxazoles (Isx), cardiomyogenic small mols. that target Notch-activated epicardium-derived cells (NECs) in vivo and promote functional recovery after myocardial infarction (MI).  Mechanistic studies in NECs led to an Isx-activated Gq protein-coupled receptor (GqPCR) hypothesis tested in a cell-based functional target screen for GPCRs regulated by Isx.  This screen identified one agonist hit, the extracellular proton/pH-sensing GPCR GPR68, confirmed through genetic gain- and loss-of-function.  Overlooked until now, GPR68 expression and localization were highly regulated in early post-natal and adult post-infarct mouse heart, where GPR68-expressing cells accumulated subepicardially.  Remarkably, GPR68-expressing cardiomyocytes established a proton-sensing cellular buffer zone surrounding the MI.  Isx pharmacol. regulated gene expression (mRNAs and miRs) in this GPR68-enriched border zone, driving cardiomyogenic and pro-survival transcriptional programs in vivo.  In conclusion, we tracked a (micromolar) bioactive small mol. mechanism-of-action to a candidate target protein, GPR68, and validated this target as a previously unrecognized regulator of myocardial cellular responses to tissue acidosis, setting the stage for future (nanomolar) target-based drug lead discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQLtpNAH6aiLVg90H21EOLACvtfcHk0lg0iuPzlWKYmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVWls7o%253D&md5=751e2e1833518df9da9253f0516ba4e5</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fcb300001m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300001m%26sid%3Dliteratum%253Aachs%26aulast%3DRussell%26aufirst%3DJ.%2BL.%26aulast%3DGoetsch%26aufirst%3DS.%2BC.%26aulast%3DAguilar%26aufirst%3DH.%2BR.%26aulast%3DCoe%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3Dvan%2BRooij%26aufirst%3DE.%26aulast%3DFrantz%26aufirst%3DD.%2BE.%26aulast%3DSchneider%26aufirst%3DJ.%2BW.%26atitle%3DRegulated%2520Expression%2520of%2520pH%2520Sensing%2520G%2520Protein-Coupled%2520Receptor-68%2520Identified%2520through%2520Chemical%2520Biology%2520Defines%2520a%2520New%2520Drug%2520Target%2520for%2520Ischemic%2520Heart%2520Disease%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1077%26epage%3D1083%26doi%3D10.1021%2Fcb300001m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mogi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakakura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span> <span> </span><span class="NLM_article-title">Role of Extracellular Proton-Sensing OGR1 in Regulation of Insulin Secretion and Pancreatic to Cell Functions</span>. <i>Endocr. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">101</span>, <span class="refDoi"> DOI: 10.1507/endocrj.EJ13-0380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1507%2Fendocrj.EJ13-0380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=24088601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFWqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2014&pages=101&author=C.+Mogiauthor=T.+Nakakuraauthor=F.+Okajima&title=Role+of+Extracellular+Proton-Sensing+OGR1+in+Regulation+of+Insulin+Secretion+and+Pancreatic+to+Cell+Functions&doi=10.1507%2Fendocrj.EJ13-0380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Role of extracellular proton-sensing OGR1 in regulation of insulin secretion and pancreatic β-cell functions</span></div><div class="casAuthors">Mogi, Chihiro; Nakakura, Takashi; Okajima, Fumikazu</div><div class="citationInfo"><span class="NLM_cas:title">Endocrine Journal (Kyoto, Japan)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-110</span>CODEN:
                <span class="NLM_cas:coden">ENJOEO</span>;
        ISSN:<span class="NLM_cas:issn">0918-8959</span>.
    
            (<span class="NLM_cas:orgname">Japan Endocrine Society</span>)
        </div><div class="casAbstract">A review.  Insulin secretion with respect to pH environments has been investigated for a long time but its mechanism remains largely unknown.  Extracellular pH is usually maintained at around 7.4 and, its change has been thought to occur in non-physiol. situations.  Acidification takes place under ischemic and inflammatory microenvironments, where stimulation of anaerobic glycolysis results in the prodn. of lactic acid.  In addn. to ionotropic ion channels, such as transient receptor potential V1 (TRPV1) and acid-sensing ion channels (ASICs), metabotropic proton-sensing G protein-coupled receptors (GPCRs) have also been identified recently as proton-sensing machineries.  While ionotropic ion channels usually sense strong acidic pH, proton-sensing GPCRs sense pH of 7.6 to 6.0 and have been shown to mediate a variety of biol. actions in neutral and mildly acidic pH environments.  Studies with receptor knockout mice have revealed that proton-sensing receptors, including ovarian cancer G protein-coupled receptor 1 (OGR1), a proton-sensing GPCRs, play a role in the regulation of insulin secretion and glucose metab. under physiol. conditions.  Small mol. 3,5-disubstituted isoxazoles have recently been identified as OGR1 agonists working at neutral pH and have been shown to stimulate pancreatic β-cell differentiation and insulin synthesis.  Thus, proton-sensing OGR1 may be an important player for insulin secretion and a potential target for improving β-cell function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2FjKKiqNrX7Vg90H21EOLACvtfcHk0lg0iuPzlWKYmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFWqtb0%253D&md5=a59172fd753ef3455cf0a62415c19ede</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1507%2Fendocrj.EJ13-0380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1507%252Fendocrj.EJ13-0380%26sid%3Dliteratum%253Aachs%26aulast%3DMogi%26aufirst%3DC.%26aulast%3DNakakura%26aufirst%3DT.%26aulast%3DOkajima%26aufirst%3DF.%26atitle%3DRole%2520of%2520Extracellular%2520Proton-Sensing%2520OGR1%2520in%2520Regulation%2520of%2520Insulin%2520Secretion%2520and%2520Pancreatic%2520to%2520Cell%2520Functions%26jtitle%3DEndocr.%2520J.%26date%3D2014%26volume%3D61%26spage%3D101%26doi%3D10.1507%2Fendocrj.EJ13-0380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abe-Ohya, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiozawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, F.</span></span> <span> </span><span class="NLM_article-title">Identification of Metals from Osteoblastic ST-2 Cell Supernatants as Novel OGR1 Agonists</span>. <i>J. Recept. Signal Transduction</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">492</span>, <span class="refDoi"> DOI: 10.3109/10799893.2015.1015736</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.3109%2F10799893.2015.1015736" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=26053506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlSqurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=485-492&author=R.+Abe-Ohyaauthor=T.+Ishikawaauthor=H.+Shiozawaauthor=K.+Sudaauthor=F.+Nara&title=Identification+of+Metals+from+Osteoblastic+ST-2+Cell+Supernatants+as+Novel+OGR1+Agonists&doi=10.3109%2F10799893.2015.1015736"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of metals from osteoblastic ST-2 cell supernatants as novel OGR1 agonists</span></div><div class="casAuthors">Abe-Ohya, Rie; Ishikawa, Tomio; Shiozawa, Hideyuki; Suda, Koji; Nara, Futoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Receptors and Signal Transduction</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">485-492</span>CODEN:
                <span class="NLM_cas:coden">JRSTCT</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Ovarian cancer G-protein-coupled receptor 1 (OGR1) is a G-protein-coupled receptor (GPCR), which has previously been identified as a receptor for protons.  It has been reported in this and previous studies that OGR1 expression was markedly up-regulated during osteoclast differentiation.  We predicted the possibility of other mols. activating OGR1 in neutral pH, and that osteoblasts might release OGR1 agonistic mols. and activate OGR1 expressed in osteoclasts such as RANKL.  We screened for cell supernatants and organ exts. and discovered OGR1 agonistic activity in ST-2 osteoblastic cell supernatants and pancreatic tissues.  Finally, we partially purified and identified essential metals, Fe, Zn, Co, Ni and Mn, as novel OGR1 agonists.  These OGR1 agonistic metals induce intracellular Gq-coupled inositol phosphate signals in OGR1-expressing cells and primary osteoclasts through OGR1.  We also confirmed that these OGR1 agonistic metals activated OGR1 through the same residues which act with protons.  Here, we demonstrate that metals, Fe, Zn, Co, Ni and Mn are the novel OGR1 agonists, which can singly activate OGR1 in neutral pH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpen1-rQPSfgbVg90H21EOLACvtfcHk0ljHMIN2b3sQuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlSqurk%253D&md5=eba16e9d6e1fa3c3b40f879425f7bf73</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.3109%2F10799893.2015.1015736&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10799893.2015.1015736%26sid%3Dliteratum%253Aachs%26aulast%3DAbe-Ohya%26aufirst%3DR.%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DShiozawa%26aufirst%3DH.%26aulast%3DSuda%26aufirst%3DK.%26aulast%3DNara%26aufirst%3DF.%26atitle%3DIdentification%2520of%2520Metals%2520from%2520Osteoblastic%2520ST-2%2520Cell%2520Supernatants%2520as%2520Novel%2520OGR1%2520Agonists%26jtitle%3DJ.%2520Recept.%2520Signal%2520Transduction%26date%3D2015%26volume%3D35%26spage%3D485%26epage%3D492%26doi%3D10.3109%2F10799893.2015.1015736" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Negishi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omori, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takanashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagayama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirayama, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishina, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakakura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochimaru, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomura, H.</span></span> <span> </span><span class="NLM_article-title">Manganese and Cobalt Activate Zebrafish Ovarian Cancer G-protein-Coupled Receptor 1 but Not GPR4</span>. <i>J. Recept. Signal Transduction</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">408</span>, <span class="refDoi"> DOI: 10.1080/10799893.2017.1298130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1080%2F10799893.2017.1298130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=28270026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvFyjtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&pages=401-408&author=J.+Negishiauthor=Y.+Omoriauthor=M.+Shindoauthor=H.+Takanashiauthor=S.+Mushaauthor=S.+Nagayamaauthor=J.+Hirayamaauthor=H.+Nishinaauthor=T.+Nakakuraauthor=C.+Mogiauthor=K.+Satoauthor=F.+Okajimaauthor=Y.+Mochimaruauthor=H.+Tomura&title=Manganese+and+Cobalt+Activate+Zebrafish+Ovarian+Cancer+G-protein-Coupled+Receptor+1+but+Not+GPR4&doi=10.1080%2F10799893.2017.1298130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Manganese and cobalt activate zebrafish ovarian cancer G-protein-coupled receptor 1 but not GPR4</span></div><div class="casAuthors">Negishi, Jun; Omori, Yuka; Shindo, Mami; Takanashi, Hayate; Musha, Shiori; Nagayama, Suminori; Hirayama, Jun; Nishina, Hiroshi; Nakakura, Takashi; Mogi, Chihiro; Sato, Koichi; Okajima, Fumikazu; Mochimaru, Yuta; Tomura, Hideaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Receptors and Signal Transduction</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">401-408</span>CODEN:
                <span class="NLM_cas:coden">JRSTCT</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Mammalian ovarian G-protein-coupled receptor 1 (OGR1) is activated by some metals in addn. to extracellular protons and coupling to multiple intracellular signaling pathways.  In the present study, we examd. whether zebrafish OGR1, zebrafish GPR4, and human GPR4 (zOGR1, zGPR4, and hGPR4, resp.) could sense the metals and activate the intracellular signaling pathways.  On one hand, we found that only manganese and cobalt of the tested metals stimulated SRE-promoter activities in zOGR1-overexpressed HEK293T cells.  On the other hand, none of the metals tested stimulated the promoter activities in zGPR4- and hGPR4-overexpressed cells.  The OGR1 mutant (H4F), which is lost to activation by extracellular protons, did not stimulate metal-induced SRE-promoter activities.  These results suggest that zOGR1, but not GPR4, is also a metal-sensing G-protein-coupled receptor in addn. to a proton-sensing G-protein-coupled receptor, although not all metals that activate hOGR1 activated zOGR1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgUPJELCG8T7Vg90H21EOLACvtfcHk0ljHMIN2b3sQuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvFyjtLk%253D&md5=b4bb47d9fb82ef15ebfdc9c4f2c4bf02</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1080%2F10799893.2017.1298130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10799893.2017.1298130%26sid%3Dliteratum%253Aachs%26aulast%3DNegishi%26aufirst%3DJ.%26aulast%3DOmori%26aufirst%3DY.%26aulast%3DShindo%26aufirst%3DM.%26aulast%3DTakanashi%26aufirst%3DH.%26aulast%3DMusha%26aufirst%3DS.%26aulast%3DNagayama%26aufirst%3DS.%26aulast%3DHirayama%26aufirst%3DJ.%26aulast%3DNishina%26aufirst%3DH.%26aulast%3DNakakura%26aufirst%3DT.%26aulast%3DMogi%26aufirst%3DC.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DOkajima%26aufirst%3DF.%26aulast%3DMochimaru%26aufirst%3DY.%26aulast%3DTomura%26aufirst%3DH.%26atitle%3DManganese%2520and%2520Cobalt%2520Activate%2520Zebrafish%2520Ovarian%2520Cancer%2520G-protein-Coupled%2520Receptor%25201%2520but%2520Not%2520GPR4%26jtitle%3DJ.%2520Recept.%2520Signal%2520Transduction%26date%3D2017%26volume%3D37%26spage%3D401%26epage%3D408%26doi%3D10.1080%2F10799893.2017.1298130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mochimaru, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negishi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomura, H.</span></span> <span> </span><span class="NLM_article-title">Metals Differentially Activate Ovarian Cancer G Protein-Coupled Receptor 1 in Various Species</span>. <i>Zool. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.2108/zs170145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.2108%2Fzs170145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=29623784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVCitbrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2018&pages=109-114&author=Y.+Mochimaruauthor=J.+Negishiauthor=S.+Murakamiauthor=S.+Mushaauthor=K.+Satoauthor=F.+Okajimaauthor=H.+Tomura&title=Metals+Differentially+Activate+Ovarian+Cancer+G+Protein-Coupled+Receptor+1+in+Various+Species&doi=10.2108%2Fzs170145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Metals differentially activate ovarian cancer G protein-coupled receptor 1 in various species</span></div><div class="casAuthors">Mochimaru, Yuta; Negishi, Jun; Murakami, Syo; Musha, Shiori; Sato, Koichi; Okajima, Fumikazu; Tomura, Hideaki</div><div class="citationInfo"><span class="NLM_cas:title">Zoological Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">109-114</span>CODEN:
                <span class="NLM_cas:coden">ZOSCEX</span>;
        ISSN:<span class="NLM_cas:issn">0289-0003</span>.
    
            (<span class="NLM_cas:orgname">Zoological Society of Japan</span>)
        </div><div class="casAbstract">Human, mouse, and zebrafish ovarian cancer G protein-coupled receptors (OGR1s) are activated by both metals and extracellular protons.  In the present study, we examd. whether pig, rat, chicken, and Xenopus OGR1 homologs could sense and be activated by protons and metals.  We found that all homologs stimulated serum response element (SRE)-driven promoter activities when they are stimulated by protons.  On the other hand, metals differentially activated the homologs.  The results using chimeric receptors of human and zebrafish OGR1s indicate that the specificity of the metal-induced activation lies in the extracellular region.  These results suggest that protons are an evolutionally conserved agonist of OGR1.  However, the types of metals that activated the receptor differed among the homologs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGc3vXJ0Zx0rVg90H21EOLACvtfcHk0lgGHJQJ2bzChw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVCitbrK&md5=9e69ce405f2ddd7e49197f120d4ad2bd</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.2108%2Fzs170145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2108%252Fzs170145%26sid%3Dliteratum%253Aachs%26aulast%3DMochimaru%26aufirst%3DY.%26aulast%3DNegishi%26aufirst%3DJ.%26aulast%3DMurakami%26aufirst%3DS.%26aulast%3DMusha%26aufirst%3DS.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DOkajima%26aufirst%3DF.%26aulast%3DTomura%26aufirst%3DH.%26atitle%3DMetals%2520Differentially%2520Activate%2520Ovarian%2520Cancer%2520G%2520Protein-Coupled%2520Receptor%25201%2520in%2520Various%2520Species%26jtitle%3DZool.%2520Sci.%26date%3D2018%26volume%3D35%26spage%3D109%26epage%3D114%26doi%3D10.2108%2Fzs170145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyazaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munechika, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamizono, A.</span></span> <span> </span><span class="NLM_article-title">Isolation of Zn2+ as an Endogenous Agonist of GPR39 from Fetal Bovine Serum</span>. <i>J. Recept. Signal Transduction</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">246</span>, <span class="refDoi"> DOI: 10.1080/10799890701506147</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1080%2F10799890701506147" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=17885920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVGrs7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=235-246&author=S.+Yasudaauthor=T.+Miyazakiauthor=K.+Munechikaauthor=M.+Yamashitaauthor=Y.+Ikedaauthor=A.+Kamizono&title=Isolation+of+Zn2%2B+as+an+Endogenous+Agonist+of+GPR39+from+Fetal+Bovine+Serum&doi=10.1080%2F10799890701506147"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of Zn2+ as an Endogenous Agonist of GPR39 from Fetal Bovine Serum</span></div><div class="casAuthors">Yasuda, Shin-Ichiro; Miyazaki, Takahiro; Munechika, Kouji; Yamashita, Masami; Ikeda, Yoshitaka; Kamizono, Akihito</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Receptors and Signal Transduction</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-246</span>CODEN:
                <span class="NLM_cas:coden">JRSTCT</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">The authors attempted to det. natural agonists of GPR39 in fetal bovine serum (FBS).  FBS was conditioned to ext. peptides and fractionated by two types of HPLC.  The activity of each fraction was monitored by intracellular calcium mobilization.  Then the purified active ingredient was analyzed by inductively coupled plasma mass spectrometry.  In this fashion, Zn2+ was identified as an agonist of GPR39, although no peptidergic mols. were found.  The calcium-mobilizing activity of Zn2+ was not abolished by pertussis toxin but was by a phospholipase C (PLC) inhibitor, U73122, indicating that the activity of GPR39 is mediated through the Gqα -PLC pathway.  In addn., Zn2+ also activated mouse and rat GPR39, showing that the function of GPR39 as a Zn2+ receptor is conserved across species.  This study is the first exploration of GPR39 agonists in FBS and indicates that GPR39 functions as a Gq-coupled Zn2+-sensing receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYWj8fr2kSX7Vg90H21EOLACvtfcHk0lgGHJQJ2bzChw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVGrs7fL&md5=f6fdd40e52ad9380e3b88cbdd9152bdf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1080%2F10799890701506147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10799890701506147%26sid%3Dliteratum%253Aachs%26aulast%3DYasuda%26aufirst%3DS.%26aulast%3DMiyazaki%26aufirst%3DT.%26aulast%3DMunechika%26aufirst%3DK.%26aulast%3DYamashita%26aufirst%3DM.%26aulast%3DIkeda%26aufirst%3DY.%26aulast%3DKamizono%26aufirst%3DA.%26atitle%3DIsolation%2520of%2520Zn2%252B%2520as%2520an%2520Endogenous%2520Agonist%2520of%2520GPR39%2520from%2520Fetal%2520Bovine%2520Serum%26jtitle%3DJ.%2520Recept.%2520Signal%2520Transduction%26date%3D2007%26volume%3D27%26spage%3D235%26epage%3D246%26doi%3D10.1080%2F10799890701506147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holst, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egerod, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schild, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vickers, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheetham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, L.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storjohann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stidsen, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, T. W.</span></span> <span> </span><span class="NLM_article-title">GPR39 Signaling Is Stimulated by Zinc Ions but Not by Obestatin</span>. <i>Endocrinology</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>148</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1210/en.2006-0933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1210%2Fen.2006-0933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=16959833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsV2kuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2007&pages=13-20&author=B.+Holstauthor=K.+L.+Egerodauthor=E.+Schildauthor=S.+P.+Vickersauthor=S.+Cheethamauthor=L.-O.+Gerlachauthor=L.+Storjohannauthor=C.+E.+Stidsenauthor=R.+Jonesauthor=A.+G.+Beck-Sickingerauthor=T.+W.+Schwartz&title=GPR39+Signaling+Is+Stimulated+by+Zinc+Ions+but+Not+by+Obestatin&doi=10.1210%2Fen.2006-0933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">GPR39 signaling is stimulated by zinc ions but not by Obestatin</span></div><div class="casAuthors">Holst, Birgitte; Egerod, Kristoffer L.; Schild, Enrico; Vickers, Steve P.; Cheetham, Sharon; Gerlach, Lars-Ole; Storjohann, Laura; Stidsen, Carsten E.; Jones, Rob; Beck-Sickinger, Annette G.; Schwartz, Thue W.</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">148</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-20</span>CODEN:
                <span class="NLM_cas:coden">ENDOAO</span>;
        ISSN:<span class="NLM_cas:issn">0013-7227</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">GPR39 is an orphan member of the ghrelin receptor family that recently was suggested to be the receptor for Obestatin, a peptide derived from the ghrelin precursor.  Here, we compare the effect of Obestatin to the effect of Zn2+ on signal transduction and study the effect of Obestatin on food intake.  Although Zn2+ stimulated inositol phosphate turnover, cAMP prodn., arrestin mobilization, as well as cAMP response element-dependent and serum response element-dependent transcriptional activity in GPR39-expressing cells as opposed to mock-transfected cells, no reproducible effect was obtained with Obestatin in the GPR39-expressing cells.  Moreover, no specific binding of Obestatin could be detected in two different types of GPR39-expressing cells using three different radio-iodinated forms of Obestatin.  By quant. PCR anal., GPR39 expression was readily detected in peripheral organs such as duodenum and kidney but not in the pituitary and hypothalamus, i.e. presumed central target organs for Obestatin.  Obestatin had no significant and reproducible effect on acute food intake in either freely fed or fasted lean mice.  It is concluded that GPR39 is probably not the Obestatin receptor.  In contrast, the potency and efficacy of Zn2+ in respect of activating signaling indicates that this metal ion could be a physiol. relevant agonist or modulator of GPR39.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozz858uBcTxLVg90H21EOLACvtfcHk0lhI-BtBNeV21g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsV2kuw%253D%253D&md5=d9809707fa39525ca2c0392b66efc7c4</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1210%2Fen.2006-0933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2006-0933%26sid%3Dliteratum%253Aachs%26aulast%3DHolst%26aufirst%3DB.%26aulast%3DEgerod%26aufirst%3DK.%2BL.%26aulast%3DSchild%26aufirst%3DE.%26aulast%3DVickers%26aufirst%3DS.%2BP.%26aulast%3DCheetham%26aufirst%3DS.%26aulast%3DGerlach%26aufirst%3DL.-O.%26aulast%3DStorjohann%26aufirst%3DL.%26aulast%3DStidsen%26aufirst%3DC.%2BE.%26aulast%3DJones%26aufirst%3DR.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DSchwartz%26aufirst%3DT.%2BW.%26atitle%3DGPR39%2520Signaling%2520Is%2520Stimulated%2520by%2520Zinc%2520Ions%2520but%2520Not%2520by%2520Obestatin%26jtitle%3DEndocrinology%26date%3D2007%26volume%3D148%26spage%3D13%26epage%3D20%26doi%3D10.1210%2Fen.2006-0933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroeze, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Novel GPR39 Agonists Allosterically Modulated by Zinc</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">726</span>– <span class="NLM_lpage">737</span>, <span class="refDoi"> DOI: 10.1124/mol.116.106112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fmol.116.106112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=27754899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitValu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2016&pages=726-737&author=S.+Satoauthor=X.-P.+Huangauthor=W.+K.+Kroezeauthor=B.+L.+Roth&title=Discovery+and+Characterization+of+Novel+GPR39+Agonists+Allosterically+Modulated+by+Zinc&doi=10.1124%2Fmol.116.106112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of novel GPR39 agonists allosterically modulated by zinc</span></div><div class="casAuthors">Sato, Seiji; Huang, Xi-Ping; Kroeze, Wesley K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">726-737</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">In this study,we identified two previously described kinase inhibitors- 3-(4-chloro-2-fluorobenzyl)-2-methyl-N-(3-methyl-1H-pyrazol- 5-yl)-8-(morpholinomethyl)imidazo[1,2-b]pyridazin-6-amine (LY2784544) and 1H-benzimidazole-4-carboxylic acid, 2-methyl- 1-[[2-methyl-3-(trifluoromethyl)phenyl]methyl]-6-(4-morpholinyl)- (GSK2636771)-as novel GPR39 agonists by unbiased small-mol.-based screening using a β-arrestin recruitment screening approach (PRESTO-Tango).  We characterized the signaling of LY2784544 and GSK2636771 and compared their signaling patterns with a previously described "GPR39-selective" agonist N-[3-chloro-4-[[[2-(methylamino)-6-(2-pyridinyl)-4- pyrimidinyl]amino]methyl]phenyl]methanesulfonamide (GPR39-C3) at both canonical and noncanonical signaling pathways.  Unexpectedly, all three compds. displayed probe-dependent and pathway-dependent allosteric modulation by concns. of zinc reported to be physiol.  LY2784544 and GS2636771 at GPR39 in the presence of zinc were generally as potent or more potent than their reported activities against kinases in whole-cell assays.  These findings reveal an unexpected role of zinc as an allosteric potentiator of small-mol.-induced activation of GPR39 and expand the list of potential kinase off-targets to include understudied G protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-OZkkmVxuTbVg90H21EOLACvtfcHk0lhI-BtBNeV21g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitValu7Y%253D&md5=947d12ea0d4932d63cb83d0b76f1b09e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1124%2Fmol.116.106112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.116.106112%26sid%3Dliteratum%253Aachs%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Novel%2520GPR39%2520Agonists%2520Allosterically%2520Modulated%2520by%2520Zinc%26jtitle%3DMol.%2520Pharmacol.%26date%3D2016%26volume%3D90%26spage%3D726%26epage%3D737%26doi%3D10.1124%2Fmol.116.106112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ayton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bush, A. I.</span></span> <span> </span><span class="NLM_article-title">Biometals and Their Therapeutic Implications in Alzheimer’s Disease</span>. <i>Neurotherapeutics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1007/s13311-014-0312-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1007%2Fs13311-014-0312-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=25354461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFWqtrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=109-120&author=S.+Aytonauthor=P.+Leiauthor=A.+I.+Bush&title=Biometals+and+Their+Therapeutic+Implications+in+Alzheimer%E2%80%99s+Disease&doi=10.1007%2Fs13311-014-0312-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Biometals and Their Therapeutic Implications in Alzheimer's Disease</span></div><div class="casAuthors">Ayton, Scott; Lei, Peng; Bush, Ashley I.</div><div class="citationInfo"><span class="NLM_cas:title">Neurotherapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-120</span>CODEN:
                <span class="NLM_cas:coden">NEURNV</span>;
        ISSN:<span class="NLM_cas:issn">1878-7479</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">No disease modifying therapy exists for Alzheimer's disease (AD).  The growing burden of this disease to our society necessitates continued investment in drug development.  Over the last decade, multiple phase 3 clin. trials testing drugs that were designed to target established disease mechanisms of AD have all failed to benefit patients.  There is, therefore, a need for new treatment strategies.  Changes to the transition metals, zinc, copper, and iron, in AD impact on the mol. mechanisms of disease, and targeting these metals might be an alternative approach to treat the disease.  Here we review how metals feature in mol. mechanisms of AD, and we describe preclin. and clin. data that demonstrate the potential for metal-based drug therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6czoQwd24qLVg90H21EOLACvtfcHk0lhI-BtBNeV21g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFWqtrbL&md5=8ece64c6a8b6a144e190f4f3685f5b5f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2Fs13311-014-0312-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13311-014-0312-z%26sid%3Dliteratum%253Aachs%26aulast%3DAyton%26aufirst%3DS.%26aulast%3DLei%26aufirst%3DP.%26aulast%3DBush%26aufirst%3DA.%2BI.%26atitle%3DBiometals%2520and%2520Their%2520Therapeutic%2520Implications%2520in%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DNeurotherapeutics%26date%3D2015%26volume%3D12%26spage%3D109%26epage%3D120%26doi%3D10.1007%2Fs13311-014-0312-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nordberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordberg, G. F.</span></span> <span> </span><span class="NLM_article-title">Trace Element Research-Historical and Future Aspects</span>. <i>J. Trace Elem. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">46</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1016/j.jtemb.2016.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.jtemb.2016.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=27238729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC28XptFeku74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2016&pages=46-52&author=M.+Nordbergauthor=G.+F.+Nordberg&title=Trace+Element+Research-Historical+and+Future+Aspects&doi=10.1016%2Fj.jtemb.2016.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Trace element research-historical and future aspects</span></div><div class="casAuthors">Nordberg, Monica; Nordberg, Gunnar F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Trace Elements in Medicine and Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">46-52</span>CODEN:
                <span class="NLM_cas:coden">JTEBFO</span>;
        ISSN:<span class="NLM_cas:issn">0946-672X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">During the last 30 years the International Society for Trace Element Research and the Nordic Trace Element Society has been active .  During this period the importance of these elements for human diseases has been increasingly recognized, including their contribution to the global burden of disease.  New anal. methods allow biomonitoring data to be related to health outcome.  Future research using modern chem. methods will focus more on elemental speciation and on measuring lower concns. leading to further identifying adverse effects and crit. organs.  Extensive knowledge about essentiality and toxicity of trace elements in humans has emerged during the last two decades and at present the difficulties in defining a range of acceptable oral intakes for essential elements has largely been overcome.  Biol. monitoring of trace element concns. in various media such as blood or urine is of great importance and an overview is given.  As an example, a more detailed description of biol. monitoring of cadmium is given, explaining biokinetics including the role of metallothionein in modifying kinetics and toxicity.  Finally future challenges related to risk assessment of newly developed metallic nanomaterials and metal contg. medical devices are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5vHXI3RTH5LVg90H21EOLACvtfcHk0ljiZRmWuURhjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptFeku74%253D&md5=6693cc9bd708986e10c529e317c92103</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.jtemb.2016.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jtemb.2016.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DNordberg%26aufirst%3DM.%26aulast%3DNordberg%26aufirst%3DG.%2BF.%26atitle%3DTrace%2520Element%2520Research-Historical%2520and%2520Future%2520Aspects%26jtitle%3DJ.%2520Trace%2520Elem.%2520Med.%2520Biol.%26date%3D2016%26volume%3D38%26spage%3D46%26epage%3D52%26doi%3D10.1016%2Fj.jtemb.2016.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prakash, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhaliwal, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majeed, A. B. A.</span></span> <span> </span><span class="NLM_article-title">Brain Biometals and Alzheimer’s Disease - Boon or Bane?</span>. <i>Int. J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">108</span>, <span class="refDoi"> DOI: 10.3109/00207454.2016.1174118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.3109%2F00207454.2016.1174118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=27044501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvFyrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=99-108&author=A.+Prakashauthor=G.+K.+Dhaliwalauthor=P.+Kumarauthor=A.+B.+A.+Majeed&title=Brain+Biometals+and+Alzheimer%E2%80%99s+Disease+-+Boon+or+Bane%3F&doi=10.3109%2F00207454.2016.1174118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Brain biometals and Alzheimer's disease - boon or bane?</span></div><div class="casAuthors">Prakash, Atish; Dhaliwal, Gagandeep Kaur; Kumar, Puneet; Abdul Majeed, Abu Bakar</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">99-108</span>CODEN:
                <span class="NLM_cas:coden">IJNUB7</span>;
        ISSN:<span class="NLM_cas:issn">0020-7454</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) is the most common form of dementia.  Several hypotheses have been put forward to explain the basis of disease onset and progression.  A complicated array of mol. events has been implicated in the pathogenesis of AD.  It is attributed to a variety of pathol. conditions that share similar crit. processes, such as oxidative stress, proteinaceous aggregations, mitochondrial dysfunctions and energy failure.  There is increasing evidence suggesting that metal homeostasis is dysregulated in the pathol. of AD.  Biometals play an important role in the normal body functioning but AD may be mediated or triggered by disproportion of metal ions leading to changes in crit. biol. systems and initiating a cascade of events finally leading to neurodegeneration and cell death.  The link is multifactorial, and although the source of the shift in oxidative homeostasis is still unclear, current evidence points to changes in the balance of redox transition metals, esp. iron, copper (Cu) and other trace metals.  Their levels in the brain are found to be elevated in AD.  In other neurodegenerative disorders, Cu, zinc, aluminum and manganese are involved.  This paper is a review of recent advances of the role of metals in the pathogenesis and pathophysiol. of AD and related neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzMxteJ9Bvu7Vg90H21EOLACvtfcHk0ljiZRmWuURhjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvFyrtr0%253D&md5=61752befa2170b316304992907793696</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.3109%2F00207454.2016.1174118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00207454.2016.1174118%26sid%3Dliteratum%253Aachs%26aulast%3DPrakash%26aufirst%3DA.%26aulast%3DDhaliwal%26aufirst%3DG.%2BK.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMajeed%26aufirst%3DA.%2BB.%2BA.%26atitle%3DBrain%2520Biometals%2520and%2520Alzheimer%25E2%2580%2599s%2520Disease%2520-%2520Boon%2520or%2520Bane%253F%26jtitle%3DInt.%2520J.%2520Neurosci.%26date%3D2017%26volume%3D127%26spage%3D99%26epage%3D108%26doi%3D10.3109%2F00207454.2016.1174118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpiak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroeze, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saddoris, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolova, V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangano, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penn, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koller, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">Allosteric Ligands for the Pharmacologically Dark Receptors GPR68 and GPR65</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>527</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1038/nature15699</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fnature15699" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=26550826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVWmsrjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=527&publication_year=2015&pages=477-483&author=X.-P.+Huangauthor=J.+Karpiakauthor=W.+K.+Kroezeauthor=H.+Zhuauthor=X.+Chenauthor=S.+S.+Moyauthor=K.+A.+Saddorisauthor=V.+D.+Nikolovaauthor=M.+S.+Farrellauthor=S.+Wangauthor=T.+J.+Manganoauthor=D.+A.+Deshpandeauthor=A.+Jiangauthor=R.+B.+Pennauthor=J.+Jinauthor=B.+H.+Kollerauthor=T.+Kenakinauthor=B.+K.+Shoichetauthor=B.+L.+Roth&title=Allosteric+Ligands+for+the+Pharmacologically+Dark+Receptors+GPR68+and+GPR65&doi=10.1038%2Fnature15699"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric ligands for the pharmacologically dark receptors GPR68 and GPR65</span></div><div class="casAuthors">Huang, Xi-Ping; Karpiak, Joel; Kroeze, Wesley K.; Zhu, Hu; Chen, Xin; Moy, Sheryl S.; Saddoris, Kara A.; Nikolova, Viktoriya D.; Farrell, Martilias S.; Wang, Sheng; Mangano, Thomas J.; Deshpande, Deepak A.; Jiang, Alice; Penn, Raymond B.; Jin, Jian; Koller, Beverly H.; Kenakin, Terry; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">527</span>
        (<span class="NLM_cas:issue">7579</span>),
    <span class="NLM_cas:pages">477-483</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">At least 120 non-olfactory G-protein-coupled receptors in the human genome are 'orphans' for which endogenous ligands are unknown, and many have no selective ligands, hindering the detn. of their biol. functions and clin. relevance.  Among these is GPR68, a proton receptor that lacks small mol. modulators for probing its biol.  Using yeast-based screens against GPR68, here the authors identify the benzodiazepine drug lorazepam as a non-selective GPR68 pos. allosteric modulator.  More than 3000 GPR68 homol. models were refined to recognize lorazepam in a putative allosteric site.  Docking 3.1 million mols. predicted new GPR68 modulators, many of which were confirmed in functional assays.  One potent GPR68 modulator, ogerin, suppressed recall in fear conditioning in wild-type but not in GPR68-knockout mice.  The same approach led to the discovery of allosteric agonists and neg. allosteric modulators for GPR65.  Combining phys. and structure-based screening may be broadly useful for ligand discovery for understudied and orphan GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDB_IiCoSwVrVg90H21EOLACvtfcHk0lgKlU5VJxGbNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVWmsrjJ&md5=a23d2d68c06b96cfbe7b53f2b17f3895</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature15699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15699%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DKarpiak%26aufirst%3DJ.%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DMoy%26aufirst%3DS.%2BS.%26aulast%3DSaddoris%26aufirst%3DK.%2BA.%26aulast%3DNikolova%26aufirst%3DV.%2BD.%26aulast%3DFarrell%26aufirst%3DM.%2BS.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DDeshpande%26aufirst%3DD.%2BA.%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DPenn%26aufirst%3DR.%2BB.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKoller%26aufirst%3DB.%2BH.%26aulast%3DKenakin%26aufirst%3DT.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DAllosteric%2520Ligands%2520for%2520the%2520Pharmacologically%2520Dark%2520Receptors%2520GPR68%2520and%2520GPR65%26jtitle%3DNature%26date%3D2015%26volume%3D527%26spage%3D477%26epage%3D483%26doi%3D10.1038%2Fnature15699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McAleer, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Primerano, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denvir, J.</span></span> <span> </span><span class="NLM_article-title">Cytokine Regulation in Human CD4 T Cells by the Aryl Hydrocarbon Receptor and Gq-Coupled Receptors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">10954</span> <span class="refDoi"> DOI: 10.1038/s41598-018-29262-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fs41598-018-29262-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=30026493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BB3c7gtVWgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&author=J.+P.+McAleerauthor=J.+Fanauthor=B.+Roarauthor=D.+A.+Primeranoauthor=J.+Denvir&title=Cytokine+Regulation+in+Human+CD4+T+Cells+by+the+Aryl+Hydrocarbon+Receptor+and+Gq-Coupled+Receptors&doi=10.1038%2Fs41598-018-29262-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Cytokine Regulation in Human CD4 T Cells by the Aryl Hydrocarbon Receptor and Gq-Coupled Receptors</span></div><div class="casAuthors">McAleer Jeremy P; Roar Bryanna; Fan Jun; Primerano Donald A; Denvir James</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10954</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Th17 cells contribute to host defense on mucosal surfaces but also provoke autoimmune diseases when directed against self-antigens.  Identifying therapeutic targets that regulate Th17 cell differentiation and/or cytokine production has considerable value.  Here, we study the aryl hydrocarbon receptor (AhR)-dependent transcriptome in human CD4 T cells treated with Th17-inducing cytokines.  We show that the AhR reciprocally regulates IL-17 and IL-22 production in human CD4 T cells.  Global gene expression analysis revealed that AhR ligation decreased IL21 expression, correlating with delayed upregulation of RORC during culture with Th17-inducing cytokines.  Several of the AhR-dependent genes have known roles in cellular assembly, organization, development, growth and proliferation.  We further show that expression of GPR15, GPR55 and GPR68 positively correlates with IL-22 production in the presence of the AhR agonist FICZ.  Activation of GPR68 with the lorazepam derivative ogerin resulted in suppression of IL-22 and IL-10 secretion by T cells, with no effect on IL-17.  Under neutral Th0 conditions, ogerin and the Gq/11 receptor inhibitor YM254890 blunted IL-22 induction by FICZ.  These data reveal the AhR-dependent transcriptome in human CD4 T cells and suggest the mechanism through which the AhR regulates T cell function may be partially dependent on Gq-coupled receptors including GPR68.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTbp0LpAwT-1jdblLE_E9PWfW6udTcc2eYx0NN7jyj1F7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7gtVWgtQ%253D%253D&md5=35b3e312e63edda984be2e40bf232c9d</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-29262-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-29262-4%26sid%3Dliteratum%253Aachs%26aulast%3DMcAleer%26aufirst%3DJ.%2BP.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DRoar%26aufirst%3DB.%26aulast%3DPrimerano%26aufirst%3DD.%2BA.%26aulast%3DDenvir%26aufirst%3DJ.%26atitle%3DCytokine%2520Regulation%2520in%2520Human%2520CD4%2520T%2520Cells%2520by%2520the%2520Aryl%2520Hydrocarbon%2520Receptor%2520and%2520Gq-Coupled%2520Receptors%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26doi%3D10.1038%2Fs41598-018-29262-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mysinger, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolstad, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. G.</span></span> <span> </span><span class="NLM_article-title">ZINC: a Free Tool to Discover Chemistry for Biology</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1757</span>– <span class="NLM_lpage">1768</span>, <span class="refDoi"> DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+a+Free+Tool+to+Discover+Chemistry+for+Biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0ljsihavYSGpyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520a%2520Free%2520Tool%2520to%2520Discover%2520Chemistry%2520for%2520Biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kroeze, W.
K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giguère, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciaky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">PRESTO-Tango as an Open-Source Resource for Interrogation of the Druggable Human GPCRome</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">369</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fnsmb.3014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=25895059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnslOgtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=362-369&author=W.%0AK.+Kroezeauthor=M.+F.+Sassanoauthor=X.-P.+Huangauthor=K.+Lansuauthor=J.+D.+McCorvyauthor=P.+M.+Gigu%C3%A8reauthor=N.+Sciakyauthor=B.+L.+Roth&title=PRESTO-Tango+as+an+Open-Source+Resource+for+Interrogation+of+the+Druggable+Human+GPCRome&doi=10.1038%2Fnsmb.3014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">PRESTO-Tango as an open-source resource for interrogation of the druggable human GPCRome</span></div><div class="casAuthors">Kroeze, Wesley K.; Sassano, Maria F.; Huang, Xi-Ping; Lansu, Katherine; McCorvy, John D.; Giguere, Patrick M.; Sciaky, Noah; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">362-369</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are essential mediators of cellular signaling and are important targets of drug action.  Of the approx. 350 nonolfactory human GPCRs, more than 100 are still considered to be 'orphans' because their endogenous ligands remain unknown.  Here, we describe a unique open-source resource that allows interrogation of the druggable human GPCRome via a G protein-independent β-arrestin-recruitment assay.  We validate this unique platform at more than 120 nonorphan human GPCR targets, demonstrate its utility for discovering new ligands for orphan human GPCRs and describe a method (parallel receptorome expression and screening via transcriptional output, with transcriptional activation following arrestin translocation (PRESTO-Tango)) for the simultaneous and parallel interrogation of the entire human nonolfactory GPCRome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVbl7caV9gUrVg90H21EOLACvtfcHk0ljsihavYSGpyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnslOgtLg%253D&md5=2a78b74a5f1425fa9e8b79b730c16be6</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3014%26sid%3Dliteratum%253Aachs%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DLansu%26aufirst%3DK.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DGigu%25C3%25A8re%26aufirst%3DP.%2BM.%26aulast%3DSciaky%26aufirst%3DN.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DPRESTO-Tango%2520as%2520an%2520Open-Source%2520Resource%2520for%2520Interrogation%2520of%2520the%2520Druggable%2520Human%2520GPCRome%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D362%26epage%3D369%26doi%3D10.1038%2Fnsmb.3014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bachelerie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Baruch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkhardt, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Combadiere, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farber, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horuk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparre-Ulrich, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luster, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantovani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsushima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nibbs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Power, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proudfoot, A. E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenkilde, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sozzani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thelen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshie, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnik, A.</span></span> <span> </span><span class="NLM_article-title">International Union of Basic and Clinical Pharmacology. [corrected]. LXXXIX. Update on the Extended Family of Chemokine Receptors and Introducing a New Nomenclature for Atypical Chemokine Receptors</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1124/pr.113.007724</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fpr.113.007724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=24218476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptVyktLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2014&pages=1-79&author=F.+Bachelerieauthor=A.+Ben-Baruchauthor=A.+M.+Burkhardtauthor=C.+Combadiereauthor=J.+M.+Farberauthor=G.+J.+Grahamauthor=R.+Horukauthor=A.+H.+Sparre-Ulrichauthor=M.+Locatiauthor=A.+D.+Lusterauthor=A.+Mantovaniauthor=K.+Matsushimaauthor=P.+M.+Murphyauthor=R.+Nibbsauthor=H.+Nomiyamaauthor=C.+A.+Powerauthor=A.+E.+I.+Proudfootauthor=M.+M.+Rosenkildeauthor=A.+Rotauthor=S.+Sozzaniauthor=M.+Thelenauthor=O.+Yoshieauthor=A.+Zlotnik&title=International+Union+of+Basic+and+Clinical+Pharmacology.+%5Bcorrected%5D.+LXXXIX.+Update+on+the+Extended+Family+of+Chemokine+Receptors+and+Introducing+a+New+Nomenclature+for+Atypical+Chemokine+Receptors&doi=10.1124%2Fpr.113.007724"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">International union of basic and clinical pharmacology LXXXIX. Update on the extended family of chemokine receptors and introducing a new nomenclature for atypical chemokine receptors</span></div><div class="casAuthors">Bachelerie, Francoise; Ben-Baruch, Adit; Burkhardt, Amanda M.; Combadiere, Christophe; Farber, Joshua M.; Graham, Gerard J.; Horuk, Richard; Sparre-Ulrich, Alexander Hovard; Locati, Massimo; Luster, Andrew D.; Mantovani, Alberto; Matsushima, Kouji; Murphy, Philip M.; Nibbs, Robert; Nomiyama, Hisayuki; Power, Christine A.; Proudfoot, Amanda E. I.; Rosenkilde, Mette M.; Rot, Antal; Sozzani, Silvano; Thelen, Marcus; Yoshie, Osamu; Zlotnik, Albert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-79, 80 pp.</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Sixteen years ago, the Nomenclature Committee of the International Union of Pharmacol. approved a system for naming human seven-transmembrane (7TM) G protein-coupled chemokine receptors, the large family of leukocyte chemoattractant receptors that regulates immune system development and function, in large part by mediating leukocyte trafficking.  This was announced in Pharmacol. Reviews in a major overview of the first decade of research in this field.  Since then, several new receptors have been discovered, and major advances have been made for the others in many areas, including structural biol., signal transduction mechanisms, biol., and pharmacol.  New and diverse roles have been identified in infection, immunity, inflammation, development, cancer, and other areas.  The first two drugs acting at chemokine receptors have been approved by the U.S. Food and Drug Administration (FDA), maraviroc targeting CCR5 in human immunodeficiency virus (HIV)/AIDS, and plerixafor targeting CXCR4 for stem cell mobilization for transplantation in cancer, and other candidates are now undergoing pivotal clin. trials for diverse disease indications.  In addn., a subfamily of atypical chemokine receptors has emerged that may signal through arrestins instead of G proteins to act as chemokine scavengers, and many microbial and invertebrate G protein-coupled chemokine receptors and sol. chemokine-binding proteins have been described.  Here, we review this extended family of chemokine receptors and chemokine-binding proteins at the basic, translational, and clin. levels, including an update on drug development.  We also introduce a new nomenclature for atypical chemokine receptors with the stem ACKR (atypical chemokine receptor) approved by the Nomenclature Committee of the International Union of Pharmacol. and the Human Genome Nomenclature Committee.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7EhMCJlz-zLVg90H21EOLACvtfcHk0lgnvAptenalHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptVyktLg%253D&md5=44a2c14ddf2508ada59121610c5682cf</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1124%2Fpr.113.007724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.113.007724%26sid%3Dliteratum%253Aachs%26aulast%3DBachelerie%26aufirst%3DF.%26aulast%3DBen-Baruch%26aufirst%3DA.%26aulast%3DBurkhardt%26aufirst%3DA.%2BM.%26aulast%3DCombadiere%26aufirst%3DC.%26aulast%3DFarber%26aufirst%3DJ.%2BM.%26aulast%3DGraham%26aufirst%3DG.%2BJ.%26aulast%3DHoruk%26aufirst%3DR.%26aulast%3DSparre-Ulrich%26aufirst%3DA.%2BH.%26aulast%3DLocati%26aufirst%3DM.%26aulast%3DLuster%26aufirst%3DA.%2BD.%26aulast%3DMantovani%26aufirst%3DA.%26aulast%3DMatsushima%26aufirst%3DK.%26aulast%3DMurphy%26aufirst%3DP.%2BM.%26aulast%3DNibbs%26aufirst%3DR.%26aulast%3DNomiyama%26aufirst%3DH.%26aulast%3DPower%26aufirst%3DC.%2BA.%26aulast%3DProudfoot%26aufirst%3DA.%2BE.%2BI.%26aulast%3DRosenkilde%26aufirst%3DM.%2BM.%26aulast%3DRot%26aufirst%3DA.%26aulast%3DSozzani%26aufirst%3DS.%26aulast%3DThelen%26aufirst%3DM.%26aulast%3DYoshie%26aufirst%3DO.%26aulast%3DZlotnik%26aufirst%3DA.%26atitle%3DInternational%2520Union%2520of%2520Basic%2520and%2520Clinical%2520Pharmacology.%2520%255Bcorrected%255D.%2520LXXXIX.%2520Update%2520on%2520the%2520Extended%2520Family%2520of%2520Chemokine%2520Receptors%2520and%2520Introducing%2520a%2520New%2520Nomenclature%2520for%2520Atypical%2520Chemokine%2520Receptors%26jtitle%3DPharmacol.%2520Rev.%26date%3D2014%26volume%3D66%26spage%3D1%26epage%3D79%26doi%3D10.1124%2Fpr.113.007724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bachelerie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantovani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nibbs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sozzani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thelen, M.</span></span> <span> </span><span class="NLM_article-title">An Atypical Addition to the Chemokine Receptor Nomenclature: IUPHAR Review 15</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">3945</span>– <span class="NLM_lpage">3949</span>, <span class="refDoi"> DOI: 10.1111/bph.13182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1111%2Fbph.13182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=25958743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOhurjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2015&pages=3945-3949&author=F.+Bachelerieauthor=G.+J.+Grahamauthor=M.+Locatiauthor=A.+Mantovaniauthor=P.+M.+Murphyauthor=R.+Nibbsauthor=A.+Rotauthor=S.+Sozzaniauthor=M.+Thelen&title=An+Atypical+Addition+to+the+Chemokine+Receptor+Nomenclature%3A+IUPHAR+Review+15&doi=10.1111%2Fbph.13182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">An atypical addition to the chemokine receptor nomenclature: IUPHAR review 15</span></div><div class="casAuthors">Bachelerie, Francoise; Graham, Gerard J.; Locati, Massimo; Mantovani, Alberto; Murphy, Philip M.; Nibbs, Robert; Rot, Antal; Sozzani, Silvano; Thelen, Marcus</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3945-3949</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Chemokines and their receptors are essential regulators of in vivo leukocyte migration and, some years ago, a systematic nomenclature system was developed for the chemokine receptor family.  Chemokine receptor biol. and biochem. was recently extensively reviewed.  In this review, we also highlighted a new component to the nomenclature system that incorporates receptors previously known as 'scavenging', or 'decoy', chemokine receptors on the basis of their lack of classical signalling responses to ligand binding and their general ability to scavenge, or sequester, their cognate chemokine ligands.  These mols. are now collectively referred to as 'atypical chemokine receptors', or ACKRs, and play fundamental roles in regulating in vivo responses to chemokines.  This commentary highlights this new addn. to the chemokine receptor nomenclature system and provides brief information on the four receptors currently covered by this nomenclature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroBDigTqM-rrVg90H21EOLACvtfcHk0lgnvAptenalHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOhurjE&md5=978dce7741d3af741ca16cb2635d0b7a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1111%2Fbph.13182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.13182%26sid%3Dliteratum%253Aachs%26aulast%3DBachelerie%26aufirst%3DF.%26aulast%3DGraham%26aufirst%3DG.%2BJ.%26aulast%3DLocati%26aufirst%3DM.%26aulast%3DMantovani%26aufirst%3DA.%26aulast%3DMurphy%26aufirst%3DP.%2BM.%26aulast%3DNibbs%26aufirst%3DR.%26aulast%3DRot%26aufirst%3DA.%26aulast%3DSozzani%26aufirst%3DS.%26aulast%3DThelen%26aufirst%3DM.%26atitle%3DAn%2520Atypical%2520Addition%2520to%2520the%2520Chemokine%2520Receptor%2520Nomenclature%253A%2520IUPHAR%2520Review%252015%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2015%26volume%3D172%26spage%3D3945%26epage%3D3949%26doi%3D10.1111%2Fbph.13182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nickols, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conn, P. J.</span></span> <span> </span><span class="NLM_article-title">Development of Allosteric Modulators of GPCRs for Treatment of CNS Disorders</span>. <i>Neurobiol. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">71</span>, <span class="refDoi"> DOI: 10.1016/j.nbd.2013.09.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.nbd.2013.09.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=24076101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BC2c%252FitVejsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2014&pages=55-71&author=H.+H.+Nickolsauthor=P.+J.+Conn&title=Development+of+Allosteric+Modulators+of+GPCRs+for+Treatment+of+CNS+Disorders&doi=10.1016%2Fj.nbd.2013.09.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Development of allosteric modulators of GPCRs for treatment of CNS disorders</span></div><div class="casAuthors">Nickols Hilary Highfield; Conn P Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Neurobiology of disease</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">55-71</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The discovery of allosteric modulators of G protein-coupled receptors (GPCRs) provides a promising new strategy with potential for developing novel treatments for a variety of central nervous system (CNS) disorders.  Traditional drug discovery efforts targeting GPCRs have focused on developing ligands for orthosteric sites which bind endogenous ligands.  Allosteric modulators target a site separate from the orthosteric site to modulate receptor function.  These allosteric agents can either potentiate (positive allosteric modulator, PAM) or inhibit (negative allosteric modulator, NAM) the receptor response and often provide much greater subtype selectivity than orthosteric ligands for the same receptors.  Experimental evidence has revealed more nuanced pharmacological modes of action of allosteric modulators, with some PAMs showing allosteric agonism in combination with positive allosteric modulation in response to endogenous ligand (ago-potentiators) as well as "bitopic" ligands that interact with both the allosteric and orthosteric sites.  Drugs targeting the allosteric site allow for increased drug selectivity and potentially decreased adverse side effects.  Promising evidence has demonstrated potential utility of a number of allosteric modulators of GPCRs in multiple CNS disorders, including neurodegenerative diseases such as Alzheimer's disease, Parkinson's disease, and Huntington's disease, as well as psychiatric or neurobehavioral diseases such as anxiety, schizophrenia, and addiction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT5AaOVGoQQ2DEC-75aqa5AfW6udTcc2eY1kzo_SdzG3rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c%252FitVejsA%253D%253D&md5=a1535b4448734bcae63e9797d2009737</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.nbd.2013.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nbd.2013.09.013%26sid%3Dliteratum%253Aachs%26aulast%3DNickols%26aufirst%3DH.%2BH.%26aulast%3DConn%26aufirst%3DP.%2BJ.%26atitle%3DDevelopment%2520of%2520Allosteric%2520Modulators%2520of%2520GPCRs%2520for%2520Treatment%2520of%2520CNS%2520Disorders%26jtitle%3DNeurobiol.%2520Dis.%26date%3D2014%26volume%3D61%26spage%3D55%26epage%3D71%26doi%3D10.1016%2Fj.nbd.2013.09.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burford, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wehrman, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerritz, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traynor, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, A.</span></span> <span> </span><span class="NLM_article-title">Discovery of Positive Allosteric Modulators and Silent Allosteric Modulators of the μ-Opioid Receptor</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">10830</span>– <span class="NLM_lpage">10835</span>, <span class="refDoi"> DOI: 10.1073/pnas.1300393110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1073%2Fpnas.1300393110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=23754417" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFejs7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=10830-10835&author=N.+T.+Burfordauthor=M.+J.+Clarkauthor=T.+S.+Wehrmanauthor=S.+W.+Gerritzauthor=M.+Banksauthor=J.+O%E2%80%99Connellauthor=J.+R.+Traynorauthor=A.+Alt&title=Discovery+of+Positive+Allosteric+Modulators+and+Silent+Allosteric+Modulators+of+the+%CE%BC-Opioid+Receptor&doi=10.1073%2Fpnas.1300393110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of positive allosteric modulators and silent allosteric modulators of the μ-opioid receptor</span></div><div class="casAuthors">Burford, Neil T.; Clark, Mary J.; Wehrman, Tom S.; Gerritz, Samuel W.; Banks, Martyn; O'Connell, Jonathan; Traynor, John R.; Alt, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10830-10835,S10830/1-S10830/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">μ-Opioid receptors are among the most studied G protein-coupled receptors because of the therapeutic value of agonists, such as morphine, that are used to treat chronic pain.  However, these drugs have significant side effects, such as respiratory suppression, constipation, allodynia, tolerance, and dependence, as well as abuse potential.  Efforts to fine tune pain control while alleviating the side effects of drugs, both physiol. and psychol., have led to the development of a wide variety of structurally diverse agonist ligands for the μ-opioid receptor, as well as compds. that target κ- and δ-opioid receptors.  In recent years, the identification of allosteric ligands for some G protein-coupled receptors has provided breakthroughs in obtaining receptor subtype-selectivity that can reduce the overall side effect profiles of a potential drug.  However, pos. allosteric modulators (PAMs) can also have the specific advantage of only modulating the activity of the receptor when the orthosteric agonist occupies the receptor, thus maintaining spatial and temporal control of receptor signaling in vivo.  This second advantage of allosteric modulators may yield breakthroughs in opioid receptor research and could lead to drugs with improved side-effect profiles or fewer tolerance and dependence issues compared with orthosteric opioid receptor agonists.  Here, the authors describe the discovery and characterization of μ-opioid receptor PAMs and silent allosteric modulators, identified from high-throughput screening using a β-arrestin-recruitment assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdiTL5ko3pr7Vg90H21EOLACvtfcHk0lg2bHsZFASJ_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFejs7nJ&md5=5fd64e5ae56607fa2396fdd3f32caf54</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1300393110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1300393110%26sid%3Dliteratum%253Aachs%26aulast%3DBurford%26aufirst%3DN.%2BT.%26aulast%3DClark%26aufirst%3DM.%2BJ.%26aulast%3DWehrman%26aufirst%3DT.%2BS.%26aulast%3DGerritz%26aufirst%3DS.%2BW.%26aulast%3DBanks%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DJ.%26aulast%3DTraynor%26aufirst%3DJ.%2BR.%26aulast%3DAlt%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520Positive%2520Allosteric%2520Modulators%2520and%2520Silent%2520Allosteric%2520Modulators%2520of%2520the%2520%25CE%25BC-Opioid%2520Receptor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D10830%26epage%3D10835%26doi%3D10.1073%2Fpnas.1300393110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Livingston, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanczyk, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burford, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canals, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traynor, J. R.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic Evidence for a Putative Conserved Allosteric Site on Opioid Receptors</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1124/mol.117.109561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fmol.117.109561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=29233847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsF2rtb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2018&pages=157-167&author=K.+E.+Livingstonauthor=M.+A.+Stanczykauthor=N.+T.+Burfordauthor=A.+Altauthor=M.+Canalsauthor=J.+R.+Traynor&title=Pharmacologic+Evidence+for+a+Putative+Conserved+Allosteric+Site+on+Opioid+Receptors&doi=10.1124%2Fmol.117.109561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic evidence for a putative conserved allosteric site on opioid receptors</span></div><div class="casAuthors">Livingston, Kathryn E.; Stanczyk, M. Alexander; Burford, Neil T.; Alt, Andrew; Canals, Meritxell; Traynor, John R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-167</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Allosteric modulators of G protein-coupled receptors, including opioid receptors, have been proposed as possible therapeutic agents with enhanced selectivity.  BMS-986122 is a pos. allosteric modulator (PAM) of the μ-opioid receptor (μ-OR).  BMS-986187 is a structurally distinct PAM for the δ-opioid receptor (δ-OR) that has been reported to exhibit 100-fold selectivity in promoting δ-OR over μ-OR agonism.  We used ligand binding and second-messenger assays to show that BMS-986187 is an effective PAM at the μ-OR and at the k-opioid receptor (κ-OR), but it is ineffective at the nociceptin receptor.  The affinity of BMS-986187 for δ-ORs and κ-ORs is approx. 20- to 30-fold higher than for μ-ORs, detd. using an allosteric ternary complex model.  Moreover, we provide evidence, using a silent allosteric modulator as an allosteric antagonist, that BMS-986187 and BMS-986122 bind to a similar region on all three traditional opioid receptor types (μ-OR, δ-OR, and κ-OR).  In contrast to the dogma surrounding allosteric modulators, the results indicate a possible conserved allosteric binding site across the opioid receptor family that can accommodate structurally diverse mols.  These findings have implications for the development of selective allosteric modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPZysO_yHQu7Vg90H21EOLACvtfcHk0lhL8xlsj-U2dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsF2rtb3F&md5=8b2e08fa5e7036428c428b97f4f6ab90</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1124%2Fmol.117.109561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.117.109561%26sid%3Dliteratum%253Aachs%26aulast%3DLivingston%26aufirst%3DK.%2BE.%26aulast%3DStanczyk%26aufirst%3DM.%2BA.%26aulast%3DBurford%26aufirst%3DN.%2BT.%26aulast%3DAlt%26aufirst%3DA.%26aulast%3DCanals%26aufirst%3DM.%26aulast%3DTraynor%26aufirst%3DJ.%2BR.%26atitle%3DPharmacologic%2520Evidence%2520for%2520a%2520Putative%2520Conserved%2520Allosteric%2520Site%2520on%2520Opioid%2520Receptors%26jtitle%3DMol.%2520Pharmacol.%26date%3D2018%26volume%3D93%26spage%3D157%26epage%3D167%26doi%3D10.1124%2Fmol.117.109561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haas, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salazar, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herber, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degnan, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balakrishnan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macor, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albright, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strittmatter, S. M.</span></span> <span> </span><span class="NLM_article-title">Silent Allosteric Modulation of mGluR5 Maintains Glutamate Signaling While Rescuing Alzheimer’s Mouse Phenotypes</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.06.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.celrep.2017.06.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=28683325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFartbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2017&pages=76-88&author=L.+T.+Haasauthor=S.+V.+Salazarauthor=L.+M.+Smithauthor=H.+R.+Zhaoauthor=T.+O.+Coxauthor=C.+S.+Herberauthor=A.+P.+Degnanauthor=A.+Balakrishnanauthor=J.+E.+Macorauthor=C.+F.+Albrightauthor=S.+M.+Strittmatter&title=Silent+Allosteric+Modulation+of+mGluR5+Maintains+Glutamate+Signaling+While+Rescuing+Alzheimer%E2%80%99s+Mouse+Phenotypes&doi=10.1016%2Fj.celrep.2017.06.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Silent Allosteric Modulation of mGluR5 Maintains Glutamate Signaling while Rescuing Alzheimer's Mouse Phenotypes</span></div><div class="casAuthors">Haas, Laura T.; Salazar, Santiago V.; Smith, Levi M.; Zhao, Helen R.; Cox, Timothy O.; Herber, Charlotte S.; Degnan, Andrew P.; Balakrishnan, Anand; Macor, John E.; Albright, Charles F.; Strittmatter, Stephen M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-88</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Metabotropic glutamate receptor 5 (mGluR5) has been implicated in Alzheimer's disease (AD) pathol.  We sought to understand whether mGluR5's role in AD requires glutamate signaling.  We used a potent mGluR5 silent allosteric modulator (SAM, BMS-984923) to sep. its well-known physiol. role in glutamate signaling from a pathol. role in mediating amyloid-β oligomer (Aβo) action.  Binding of the SAM to mGluR5 does not change glutamate signaling but strongly reduces mGluR5 interaction with cellular prion protein (PrPC) bound to Aβo.  The SAM compd. prevents Aβo-induced signal transduction in brain slices and in an AD transgenic mouse model, the APPswe/PS1ΔE9 strain.  Critically, 4 wk of SAM treatment rescues memory deficits and synaptic depletion in the APPswe/PS1ΔE9 transgenic mouse brain.  Our data show that mGluR5's role in Aβo-dependent AD phenotypes is sep. from its role in glutamate signaling and silent allosteric modulation of mGluR5 has promise as a disease-modifying AD intervention with a broad therapeutic window.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiq45bkzh9ebVg90H21EOLACvtfcHk0lhL8xlsj-U2dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFartbnN&md5=3e3e1a787fb58397ac649068f0ccd07e</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.06.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.06.023%26sid%3Dliteratum%253Aachs%26aulast%3DHaas%26aufirst%3DL.%2BT.%26aulast%3DSalazar%26aufirst%3DS.%2BV.%26aulast%3DSmith%26aufirst%3DL.%2BM.%26aulast%3DZhao%26aufirst%3DH.%2BR.%26aulast%3DCox%26aufirst%3DT.%2BO.%26aulast%3DHerber%26aufirst%3DC.%2BS.%26aulast%3DDegnan%26aufirst%3DA.%2BP.%26aulast%3DBalakrishnan%26aufirst%3DA.%26aulast%3DMacor%26aufirst%3DJ.%2BE.%26aulast%3DAlbright%26aufirst%3DC.%2BF.%26aulast%3DStrittmatter%26aufirst%3DS.%2BM.%26atitle%3DSilent%2520Allosteric%2520Modulation%2520of%2520mGluR5%2520Maintains%2520Glutamate%2520Signaling%2520While%2520Rescuing%2520Alzheimer%25E2%2580%2599s%2520Mouse%2520Phenotypes%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D20%26spage%3D76%26epage%3D88%26doi%3D10.1016%2Fj.celrep.2017.06.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conn, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span> <span> </span><span class="NLM_article-title">Synthesis and SAR of a mGluR5 Allosteric Partial Antagonist Lead: Unexpected Modulation of Pharmacology with Slight Structural Modifications to a 5-(Phenylethynyl)pyrimidine Scaffold</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">4098</span>– <span class="NLM_lpage">4101</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.05.091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.bmcl.2008.05.091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=18550372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslequro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=4098-4101&author=S.+Sharmaauthor=A.+L.+Rodriguezauthor=P.+J.+Connauthor=C.+W.+Lindsley&title=Synthesis+and+SAR+of+a+mGluR5+Allosteric+Partial+Antagonist+Lead%3A+Unexpected+Modulation+of+Pharmacology+with+Slight+Structural+Modifications+to+a+5-%28Phenylethynyl%29pyrimidine+Scaffold&doi=10.1016%2Fj.bmcl.2008.05.091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and SAR of a mGluR5 allosteric partial antagonist lead: Unexpected modulation of pharmacology with slight structural modifications to a 5-(phenylethynyl)pyrimidine scaffold</span></div><div class="casAuthors">Sharma, Sameer; Rodriguez, Alice L.; Conn, P. Jeffrey; Lindsley, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4098-4101</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">This Letter describes the synthesis and SAR, developed through an iterative analog library approach, of a mGluR5 allosteric partial antagonist lead based on a 5-(phenylethynyl)pyrimidine scaffold I (R = H, 3-Me, 3,5-F2, etc.).  With slight structural modifications to the distal Ph ring, analogs demonstrated a range of pharmacol. activities from mGluR5 partial antagonism to full antagonism/neg. allosteric modulation to pos. allosteric modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQBeYocKgYurVg90H21EOLACvtfcHk0lhPd4uwNqNCUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslequro%253D&md5=323d229f189185ed5d237e0a6e37e9a8</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.05.091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.05.091%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DRodriguez%26aufirst%3DA.%2BL.%26aulast%3DConn%26aufirst%3DP.%2BJ.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3DSynthesis%2520and%2520SAR%2520of%2520a%2520mGluR5%2520Allosteric%2520Partial%2520Antagonist%2520Lead%253A%2520Unexpected%2520Modulation%2520of%2520Pharmacology%2520with%2520Slight%2520Structural%2520Modifications%2520to%2520a%25205-%2528Phenylethynyl%2529pyrimidine%2520Scaffold%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D4098%26epage%3D4101%26doi%3D10.1016%2Fj.bmcl.2008.05.091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brogan, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conn, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span> <span> </span><span class="NLM_article-title">Molecular Switches” on mGluR Allosteric Ligands That Modulate Modes of Pharmacology</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">2403</span>– <span class="NLM_lpage">2410</span>, <span class="refDoi"> DOI: 10.1021/bi200129s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi200129s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3MXislGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=2403-2410&author=M.+R.+Woodauthor=C.+R.+Hopkinsauthor=J.+T.+Broganauthor=P.+J.+Connauthor=C.+W.+Lindsley&title=Molecular+Switches%E2%80%9D+on+mGluR+Allosteric+Ligands+That+Modulate+Modes+of+Pharmacology&doi=10.1021%2Fbi200129s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">"Molecular Switches" on mGluR Allosteric Ligands that Modulate Modes of Pharmacology</span></div><div class="casAuthors">Wood, Michael R.; Hopkins, Corey R.; Brogan, John T.; Conn, P. Jeffrey; Lindsley, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2403-2410</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  G-Protein-coupled receptors (GPCRs) represent the largest class of drug targets, accounting for more than 40% of marketed drugs; however, discovery efforts for many GPCRs have failed to provide viable drug candidates.  Historically, drug discovery efforts have focused on developing ligands that act at the orthosteric site of the endogenous agonist.  Recently, efforts have focused on functional assay paradigms and the discovery of ligands that act at allosteric sites to modulate receptor function in either a pos., neg., or neutral manner.  Allosteric modulators have numerous advantages over orthosteric ligands, including high subtype selectivity; the ability to mimic physiol. conditions; the lack of desensitization, downregulation, and internalization; and reduced side effects.  Despite these virtues, challenging issues have now arisen for allosteric modulators of metabotropic glutamate receptors (mGluRs): Shallow SAR, ligand-directed trafficking, and the identification of subtle "mol. switches" that modulate the modes of pharmacol.  Here, we will discuss the impact of modest structural changes to multiple mGluR allosteric ligands scaffolds that unexpectedly modulate pharmacol. and raise concerns over metab. and the pharmacol. of metabolites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf_Kg5p51i47Vg90H21EOLACvtfcHk0lhPd4uwNqNCUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXislGntbg%253D&md5=adc945bc7ea29582144a60687a9aaf9d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fbi200129s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi200129s%26sid%3Dliteratum%253Aachs%26aulast%3DWood%26aufirst%3DM.%2BR.%26aulast%3DHopkins%26aufirst%3DC.%2BR.%26aulast%3DBrogan%26aufirst%3DJ.%2BT.%26aulast%3DConn%26aufirst%3DP.%2BJ.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3DMolecular%2520Switches%25E2%2580%259D%2520on%2520mGluR%2520Allosteric%2520Ligands%2520That%2520Modulate%2520Modes%2520of%2520Pharmacology%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D2403%26epage%3D2410%26doi%3D10.1021%2Fbi200129s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conn, P. J.</span></span> <span> </span><span class="NLM_article-title">Molecular Insights into Metabotropic Glutamate Receptor Allosteric Modulation</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1124/mol.114.097220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fmol.114.097220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=25808929" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVGgt7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2015&pages=188-202&author=K.+J.+Gregoryauthor=P.+J.+Conn&title=Molecular+Insights+into+Metabotropic+Glutamate+Receptor+Allosteric+Modulation&doi=10.1124%2Fmol.114.097220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular insights into metabotropic glutamate receptor allosteric modulation</span></div><div class="casAuthors">Gregory, Karen J.; Conn, P. Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">188-202</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The metabotropic glutamate (mGlu) receptors are a group of eight family C G protein-coupled receptors that are expressed throughout the central nervous system (CNS) and periphery.  Within the CNS the different subtypes are found in neurons, both pre- and/or postsynaptically, where they mediate modulatory roles and in glial cells.  The mGlu receptor family provides attractive targets for numerous psychiatric and neurol. disorders, with the majority of discovery programs focused on targeting allosteric sites, with allosteric ligands now available for all mGlu receptor subtypes.  However, the development of allosteric ligands remains challenging.  Biased modulation, probe dependence, and mol. switches all contribute to the complex mol. pharmacol. exhibited by mGlu receptor allosteric ligands.  In recent years we have made significant progress in our understanding of this mol. complexity coupled with an increased understanding of the structural basis of mGlu allosteric modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzyYqss3Nb4rVg90H21EOLACvtfcHk0lis4nBF-WupRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVGgt7jF&md5=273ed67eab526a2df6cc9d473a8de427</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1124%2Fmol.114.097220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.114.097220%26sid%3Dliteratum%253Aachs%26aulast%3DGregory%26aufirst%3DK.%2BJ.%26aulast%3DConn%26aufirst%3DP.%2BJ.%26atitle%3DMolecular%2520Insights%2520into%2520Metabotropic%2520Glutamate%2520Receptor%2520Allosteric%2520Modulation%26jtitle%3DMol.%2520Pharmacol.%26date%3D2015%26volume%3D88%26spage%3D188%26epage%3D202%26doi%3D10.1124%2Fmol.114.097220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Colloidal Aggregation Causes Inhibition of G Protein-Coupled Receptors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2406</span>– <span class="NLM_lpage">2414</span>, <span class="refDoi"> DOI: 10.1021/jm301749y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301749y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2406-2414&author=M.+F.+Sassanoauthor=A.+K.+Doakauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Colloidal+Aggregation+Causes+Inhibition+of+G+Protein-Coupled+Receptors&doi=10.1021%2Fjm301749y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Colloidal aggregation causes inhibition of G protein-coupled receptors</span></div><div class="casAuthors">Sassano, Maria F.; Doak, Allison K.; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2406-2414</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Colloidal aggregation is the dominant mechanism for artifactual inhibition of sol. proteins, and controls against it are now widely deployed.  Conversely, investigating this mechanism for membrane-bound receptors has proven difficult.  Here, the authors investigated the activity of 4 well-characterized aggregators against 3 G protein-coupled receptors (GPCRs) recognizing peptide and protein ligands.  Each of the aggregators was active at micromolar concns. against the 3 GPCRs in cell-based assays.  This activity could be attenuated by either centrifugation of the inhibitor stock soln. or by the addn. of Tween-80 detergent.  In the absence of agonist, the aggregators acted as inverse agonists, consistent with a direct receptor interaction.  Meanwhile, several literature GPCR ligands that resembled aggregators themselves formed colloids, by both phys. and enzymol. tests.  These observations suggest that some GPCRs may be artifactually antagonized by colloidal aggregates, an effect that merits the attention of investigators in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojG9CcuOYoXrVg90H21EOLACvtfcHk0lis4nBF-WupRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCgt7s%253D&md5=84fa59f9635330739bf836f8085be5e0</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm301749y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301749y%26sid%3Dliteratum%253Aachs%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DColloidal%2520Aggregation%2520Causes%2520Inhibition%2520of%2520G%2520Protein-Coupled%2520Receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2406%26epage%3D2414%26doi%3D10.1021%2Fjm301749y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Owen, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesh, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nedyalkova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, M. S.</span></span> <span> </span><span class="NLM_article-title">Colloidal Drug Formulations Can Explain “Bell-shaped” Concentration-Response Curves</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">777</span>– <span class="NLM_lpage">784</span>, <span class="refDoi"> DOI: 10.1021/cb4007584</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4007584" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVCgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=777-784&author=S.+C.+Owenauthor=A.+K.+Doakauthor=A.+N.+Ganeshauthor=L.+Nedyalkovaauthor=C.+K.+McLaughlinauthor=B.+K.+Shoichetauthor=M.+S.+Shoichet&title=Colloidal+Drug+Formulations+Can+Explain+%E2%80%9CBell-shaped%E2%80%9D+Concentration-Response+Curves&doi=10.1021%2Fcb4007584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Colloidal Drug Formulations Can Explain "Bell-Shaped" Concentration-Response Curves</span></div><div class="casAuthors">Owen, Shawn C.; Doak, Allison K.; Ganesh, Ahil N.; Nedyalkova, Lyudmila; McLaughlin, Christopher K.; Shoichet, Brian K.; Shoichet, Molly S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">777-784</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug efficacy does not always increase sigmoidally with concn., which has puzzled the community for decades.  Unlike std. sigmoidal curves, bell-shaped concn.-response curves suggest more complex biol. effects, such as multiple-binding sites or multiple targets.  Here, we investigate a phys. property-based mechanism for bell-shaped curves.  Beginning with the observation that some drugs form colloidal aggregates at relevant concns., we detd. concn.-response curves for three aggregating anticancer drugs, formulated both as colloids and as free monomer.  Colloidal formulations exhibited bell-shaped curves, losing activity at higher concns., while monomeric formulations gave typical sigmoidal curves, sustaining a plateau of max. activity.  Inverting the question, we next asked if mols. with bell-shaped curves, reported in the literature, form colloidal aggregates at relevant concns.  We selected 12 mols. reported to have bell-shaped concn.-response curves and found that five of these formed colloids.  To understand the mechanism behind the loss of activity at concns. where colloids are present, we investigated the diffusion of colloid-forming dye Evans blue into cells.  We found that colloidal species are excluded from cells, which may explain the mechanism behind toxicol. screens that use Evans blue, Trypan blue, and related dyes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg5AlTuU9YnrVg90H21EOLACvtfcHk0lh2-MKQsEmc2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVCgtA%253D%253D&md5=8cb90ab5f6348326d08ccfcf5377b1f3</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fcb4007584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4007584%26sid%3Dliteratum%253Aachs%26aulast%3DOwen%26aufirst%3DS.%2BC.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DGanesh%26aufirst%3DA.%2BN.%26aulast%3DNedyalkova%26aufirst%3DL.%26aulast%3DMcLaughlin%26aufirst%3DC.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DM.%2BS.%26atitle%3DColloidal%2520Drug%2520Formulations%2520Can%2520Explain%2520%25E2%2580%259CBell-shaped%25E2%2580%259D%2520Concentration-Response%2520Curves%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D777%26epage%3D784%26doi%3D10.1021%2Fcb4007584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vijayaraghavan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajan, S.</span></span> <span> </span><span class="NLM_article-title">Docking, Synthesis and Antimalarial Activity of Novel 4-Anilinoquinoline Derivatives</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1693</span>– <span class="NLM_lpage">1697</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.bmcl.2017.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=28318947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslKrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=1693-1697&author=S.+Vijayaraghavanauthor=S.+Mahajan&title=Docking%2C+Synthesis+and+Antimalarial+Activity+of+Novel+4-Anilinoquinoline+Derivatives&doi=10.1016%2Fj.bmcl.2017.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Docking, synthesis and antimalarial activity of novel 4-anilinoquinoline derivatives</span></div><div class="casAuthors">Vijayaraghavan, Shilpa; Mahajan, Supriya</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1693-1697</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of 4-anilinoquinoline triazine derivs. were designed, synthesized and screened for in vivo antimalarial activity against a chloroquine-sensitive strain of Plasmodium berghei.  The compds. were further subjected to in vitro antimalarial activity against chloroquine-resistant W2 strain of Plasmodium falciparum and β-haematin inhibition studies.  All the compds. exhibited in vivo antimalarial activity better than that shown by the std. drug, chloroquine.  Twelve out of fifteen compds. showed better inhibition than that of chloroquine against chloroquine-resistant W2 strain of Plasmodium falciparum.  Ten compds. showed β-haematin inhibition, better than that of chloroquine, with IC50 values in the range of 18-25 μM.  One compd., 3k, was better than artemisinin against W2 strain of Plasmodium falciparum and also displayed the best β-haematin inhibitory activity, thereby becoming eligible to be explored as a potential lead for antimalarial chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb2Xtd3d_0F7Vg90H21EOLACvtfcHk0lh2-MKQsEmc2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslKrsb4%253D&md5=c1edcedab3fdfcf4adfe30bbb3bb02f6</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DVijayaraghavan%26aufirst%3DS.%26aulast%3DMahajan%26aufirst%3DS.%26atitle%3DDocking%252C%2520Synthesis%2520and%2520Antimalarial%2520Activity%2520of%2520Novel%25204-Anilinoquinoline%2520Derivatives%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D1693%26epage%3D1697%26doi%3D10.1016%2Fj.bmcl.2017.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span> <span> </span><span class="NLM_article-title">Cationic Palladium Complex Catalyzed Diastereo- and Enantioselective Tandem Annulation of 2-Formylarylboronic Acids with Allenoates</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4366</span>– <span class="NLM_lpage">4369</span>, <span class="refDoi"> DOI: 10.1021/ol901714d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol901714d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVOjurnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=4366-4369&author=X.+Yuauthor=X.+Lu&title=Cationic+Palladium+Complex+Catalyzed+Diastereo-+and+Enantioselective+Tandem+Annulation+of+2-Formylarylboronic+Acids+with+Allenoates&doi=10.1021%2Fol901714d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Cationic Palladium Complex Catalyzed Diastereo- and Enantioselective Tandem Annulation of 2-Formylarylboronic Acids with Allenoates</span></div><div class="casAuthors">Yu, Xufen; Lu, Xiyan</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4366-4369</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cationic palladium complex [(dppp)Pd(H2O)2]2+(BF4-)2 catalyzed tandem annulation reactions of 2-formylarylboronic acids and allenoates produce the indenol derivs., e.g., I, diastereo- and enantioselectively in a highly efficient way.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiunsyzciEG7Vg90H21EOLACvtfcHk0lhm6DPjg-ps_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVOjurnK&md5=3307677788b9c8e46f3d7dddf90b0c1a</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fol901714d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol901714d%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26atitle%3DCationic%2520Palladium%2520Complex%2520Catalyzed%2520Diastereo-%2520and%2520Enantioselective%2520Tandem%2520Annulation%2520of%25202-Formylarylboronic%2520Acids%2520with%2520Allenoates%26jtitle%3DOrg.%2520Lett.%26date%3D2009%26volume%3D11%26spage%3D4366%26epage%3D4369%26doi%3D10.1021%2Fol901714d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revankar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">Parallel Functional Activity Profiling Reveals Valvulopathogens Are Potent 5-Hydroxytryptamine(2B) Receptor Agonists: Implications for Drug Safety Assessment</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">710</span>– <span class="NLM_lpage">722</span>, <span class="refDoi"> DOI: 10.1124/mol.109.058057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fmol.109.058057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=19570945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1eitrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2009&pages=710-722&author=X.-P.+Huangauthor=V.+Setolaauthor=P.+N.+Yadavauthor=J.+A.+Allenauthor=S.+C.+Roganauthor=B.+J.+Hansonauthor=C.+Revankarauthor=M.+Robersauthor=C.+Doucetteauthor=B.+L.+Roth&title=Parallel+Functional+Activity+Profiling+Reveals+Valvulopathogens+Are+Potent+5-Hydroxytryptamine%282B%29+Receptor+Agonists%3A+Implications+for+Drug+Safety+Assessment&doi=10.1124%2Fmol.109.058057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Parallel functional activity profiling reveals valvulopathogens are potent 5-hydroxytryptamine2B receptor agonists: implications for drug safety assessment</span></div><div class="casAuthors">Huang, Xi-Ping; Setola, Vincent; Yadav, Prem N.; Allen, John A.; Rogan, Sarah C.; Hanson, Bonnie J.; Revankar, Chetana; Robers, Matt; Doucette, Chris; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">710-722</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Drug-induced valvular heart disease (VHD) is a serious side effect of a few medications, including some that are on the market.  Pharmacol. studies of VHD-assocd. medications (e.g., fenfluramine, pergolide, methysergide, and cabergoline) have revealed that they and/or their metabolites are potent 5-hydroxytryptamine2B (5-HT2B) receptor agonists.  We have shown that activation of 5-HT2B receptors on human heart valve interstitial cells in vitro induces a proliferative response reminiscent of the fibrosis that typifies VHD.  To identify current or future drugs that might induce VHD, we screened approx. 2200 U.S. Food and Drug Administration (FDA)-approved or investigational medications to identify 5-HT2B receptor agonists, using calcium-based high-throughput screening.  Of these 2200 compds., 27 were 5-HT2B receptor agonists (hits); 14 of these had previously been identified as 5-HT2B receptor agonists, including seven bona fide valvulopathogens.  Six of the hits (guanfacine, quinidine, xylometazoline, oxymetazoline, fenoldopam, and ropinirole) are approved medications.  Twenty-three of the hits were then "functionally profiled" (i.e., assayed in parallel for 5-HT2B receptor agonism using multiple readouts to test for functional selectivity).  In these assays, the known valvulopathogens were efficacious at concns. as low as 30 nM, whereas the other compds. were less so.  Hierarchical clustering anal. of the pEC50 data revealed that ropinirole (which is not assocd. with valvulopathy) was clearly segregated from known valvulopathogens.  Taken together, our data demonstrate that patterns of 5-HT2B receptor functional selectivity might be useful for identifying compds. likely to induce valvular heart disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6tCEte6PXtLVg90H21EOLACvtfcHk0lhm6DPjg-ps_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1eitrvF&md5=48f3f3a220c542bc42b7378cc79d73d0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1124%2Fmol.109.058057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.109.058057%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DYadav%26aufirst%3DP.%2BN.%26aulast%3DAllen%26aufirst%3DJ.%2BA.%26aulast%3DRogan%26aufirst%3DS.%2BC.%26aulast%3DHanson%26aufirst%3DB.%2BJ.%26aulast%3DRevankar%26aufirst%3DC.%26aulast%3DRobers%26aufirst%3DM.%26aulast%3DDoucette%26aufirst%3DC.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DParallel%2520Functional%2520Activity%2520Profiling%2520Reveals%2520Valvulopathogens%2520Are%2520Potent%25205-Hydroxytryptamine%25282B%2529%2520Receptor%2520Agonists%253A%2520Implications%2520for%2520Drug%2520Safety%2520Assessment%26jtitle%3DMol.%2520Pharmacol.%26date%3D2009%26volume%3D76%26spage%3D710%26epage%3D722%26doi%3D10.1124%2Fmol.109.058057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agegnehu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tribo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, R. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">Structural Determinants of 5-HT2B Receptor Activation and Biased Agonism</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">787</span>– <span class="NLM_lpage">796</span>, <span class="refDoi"> DOI: 10.1038/s41594-018-0116-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fs41594-018-0116-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=30127358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFGjtLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=787-796&author=J.+D.+McCorvyauthor=D.+Wackerauthor=S.+Wangauthor=B.+Agegnehuauthor=J.+Liuauthor=K.+Lansuauthor=A.+R.+Triboauthor=R.+H.+J.+Olsenauthor=T.+Cheauthor=J.+Jinauthor=B.+L.+Roth&title=Structural+Determinants+of+5-HT2B+Receptor+Activation+and+Biased+Agonism&doi=10.1038%2Fs41594-018-0116-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Structural determinants of 5-HT2B receptor activation and biased agonism</span></div><div class="casAuthors">McCorvy, John D.; Wacker, Daniel; Wang, Sheng; Agegnehu, Bemnat; Liu, Jing; Lansu, Katherine; Tribo, Alexandra R.; Olsen, Reid H. J.; Che, Tao; Jin, Jian; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">787-796</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Serotonin (5-hydroxytryptamine; 5-HT) receptors modulate a variety of physiol. processes ranging from perception, cognition and emotion to vascular and smooth muscle contraction, platelet aggregation, gastrointestinal function and reprodn.  Drugs that interact with 5-HT receptors effectively treat diseases as diverse as migraine headaches, depression and obesity.  Here we present four structures of a prototypical serotonin receptor-the human 5-HT2B receptor-in complex with chem. and pharmacol. diverse drugs, including methysergide, methylergonovine, lisuride and LY266097.  A detailed anal. of these structures complemented by comprehensive interrogation of signaling illuminated key structural determinants essential for activation.  Addnl. structure-guided mutagenesis expts. revealed binding pocket residues that were essential for agonist-mediated biased signaling and β-arrestin2 translocation.  Given the importance of 5-HT receptors for a large no. of therapeutic indications, insights derived from these studies should accelerate the design of safer and more effective medications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwC5QTOdEZhLVg90H21EOLACvtfcHk0lgin25oEr38pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFGjtLjM&md5=93be9401bc680249499fa3af5d40f395</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fs41594-018-0116-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-018-0116-7%26sid%3Dliteratum%253Aachs%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DAgegnehu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLansu%26aufirst%3DK.%26aulast%3DTribo%26aufirst%3DA.%2BR.%26aulast%3DOlsen%26aufirst%3DR.%2BH.%2BJ.%26aulast%3DChe%26aufirst%3DT.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DStructural%2520Determinants%2520of%25205-HT2B%2520Receptor%2520Activation%2520and%2520Biased%2520Agonism%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D787%26epage%3D796%26doi%3D10.1038%2Fs41594-018-0116-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stauch, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gati, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batyuk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slocum, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brehm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madsen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandner, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiriaeva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, R. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tribo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yous, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weierstall, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Ligand Recognition at the Human MT1 Melatonin Receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>569</i></span>,  <span class="NLM_fpage">284</span>– <span class="NLM_lpage">288</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1141-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fs41586-019-1141-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=31019306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVSkur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=569&publication_year=2019&pages=284-288&author=B.+Stauchauthor=L.+C.+Johanssonauthor=J.+D.+McCorvyauthor=N.+Patelauthor=G.+W.+Hanauthor=X.-P.+Huangauthor=C.+Gatiauthor=A.+Batyukauthor=S.+T.+Slocumauthor=A.+Ishchenkoauthor=W.+Brehmauthor=T.+A.+Whiteauthor=N.+Michaelianauthor=C.+Madsenauthor=L.+Zhuauthor=T.+D.+Grantauthor=J.+M.+Grandnerauthor=A.+Shiriaevaauthor=R.+H.+J.+Olsenauthor=A.+R.+Triboauthor=S.+Yousauthor=R.+C.+Stevensauthor=U.+Weierstallauthor=V.+Katritchauthor=B.+L.+Rothauthor=W.+Liuauthor=V.+Cherezov&title=Structural+Basis+of+Ligand+Recognition+at+the+Human+MT1+Melatonin+Receptor&doi=10.1038%2Fs41586-019-1141-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of ligand recognition at the human MT1 melatonin receptor</span></div><div class="casAuthors">Stauch, Benjamin; Johansson, Linda C.; McCorvy, John D.; Patel, Nilkanth; Han, Gye Won; Huang, Xi-Ping; Gati, Cornelius; Batyuk, Alexander; Slocum, Samuel T.; Ishchenko, Andrii; Brehm, Wolfgang; White, Thomas A.; Michaelian, Nairie; Madsen, Caleb; Zhu, Lan; Grant, Thomas D.; Grandner, Jessica M.; Shiriaeva, Anna; Olsen, Reid H. J.; Tribo, Alexandra R.; Yous, Said; Stevens, Raymond C.; Weierstall, Uwe; Katritch, Vsevolod; Roth, Bryan L.; Liu, Wei; Cherezov, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">569</span>
        (<span class="NLM_cas:issue">7755</span>),
    <span class="NLM_cas:pages">284-288</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Melatonin (N-acetyl-5-methoxytryptamine) is a neurohormone that maintains circadian rhythms by synchronization to environmental cues and is involved in diverse physiol. processes such as the regulation of blood pressure and core body temp., oncogenesis, and immune function.  Melatonin is formed in the pineal gland in a light-regulated manner by enzymic conversion from 5-hydroxytryptamine (5-HT or serotonin), and modulates sleep and wakefulness by activating two high-affinity G-protein-coupled receptors, type 1A (MT1) and type 1B (MT2).  Shift work, travel, and ubiquitous artificial lighting can disrupt natural circadian rhythms; as a result, sleep disorders affect a substantial population in modern society and pose a considerable economic burden.  Over-the-counter melatonin is widely used to alleviate jet lag and as a safer alternative to benzodiazepines and other sleeping aids, and is one of the most popular supplements in the United States.  Here, we present high-resoln. room-temp. X-ray free electron laser (XFEL) structures of MT1 in complex with four agonists: the insomnia drug ramelteon, two melatonin analogs, and the mixed melatonin-serotonin antidepressant agomelatine.  The structure of MT2 is described in an accompanying paper.  Although the MT1 and 5-HT receptors have similar endogenous ligands, and agomelatine acts on both receptors, the receptors differ markedly in the structure and compn. of their ligand pockets; in MT1, access to the ligand pocket is tightly sealed from solvent by extracellular loop 2, leaving only a narrow channel between transmembrane helixes IV and V that connects it to the lipid bilayer.  The binding site is extremely compact, and ligands interact with MT1 mainly by strong arom. stacking with Phe179 and auxiliary hydrogen bonds with Asn162 and Gln181.  Our structures provide an unexpected example of atypical ligand entry for a non-lipid receptor, lay the mol. foundation of ligand recognition by melatonin receptors, and will facilitate the design of future tool compds. and therapeutic agents, while their comparison to 5-HT receptors yields insights into the evolution and polypharmacol. of G-protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplFhqB4FkpoLVg90H21EOLACvtfcHk0lgin25oEr38pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVSkur0%253D&md5=61cecfc7a75118449d015c5b695d2f8f</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1141-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1141-3%26sid%3Dliteratum%253Aachs%26aulast%3DStauch%26aufirst%3DB.%26aulast%3DJohansson%26aufirst%3DL.%2BC.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DGati%26aufirst%3DC.%26aulast%3DBatyuk%26aufirst%3DA.%26aulast%3DSlocum%26aufirst%3DS.%2BT.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DBrehm%26aufirst%3DW.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DMichaelian%26aufirst%3DN.%26aulast%3DMadsen%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DGrant%26aufirst%3DT.%2BD.%26aulast%3DGrandner%26aufirst%3DJ.%2BM.%26aulast%3DShiriaeva%26aufirst%3DA.%26aulast%3DOlsen%26aufirst%3DR.%2BH.%2BJ.%26aulast%3DTribo%26aufirst%3DA.%2BR.%26aulast%3DYous%26aufirst%3DS.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DStructural%2520Basis%2520of%2520Ligand%2520Recognition%2520at%2520the%2520Human%2520MT1%2520Melatonin%2520Receptor%26jtitle%3DNature%26date%3D2019%26volume%3D569%26spage%3D284%26epage%3D288%26doi%3D10.1038%2Fs41586-019-1141-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauch, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batyuk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gati, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slocum, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandner, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, R. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tribo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaare, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zatsepin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weierstall, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yous, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span> <span> </span><span class="NLM_article-title">XFEL Structures of the Human MT2 Melatonin Receptor Reveal the Basis of Subtype Selectivity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>569</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1144-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fs41586-019-1144-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=31019305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVSkuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=569&publication_year=2019&pages=289-292&author=L.+C.+Johanssonauthor=B.+Stauchauthor=J.+D.+McCorvyauthor=G.+W.+Hanauthor=N.+Patelauthor=X.-P.+Huangauthor=A.+Batyukauthor=C.+Gatiauthor=S.+T.+Slocumauthor=C.+Liauthor=J.+M.+Grandnerauthor=S.+Haoauthor=R.+H.+J.+Olsenauthor=A.+R.+Triboauthor=S.+Zaareauthor=L.+Zhuauthor=N.+A.+Zatsepinauthor=U.+Weierstallauthor=S.+Yousauthor=R.+C.+Stevensauthor=W.+Liuauthor=B.+L.+Rothauthor=V.+Katritchauthor=V.+Cherezov&title=XFEL+Structures+of+the+Human+MT2+Melatonin+Receptor+Reveal+the+Basis+of+Subtype+Selectivity&doi=10.1038%2Fs41586-019-1144-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">XFEL structures of the human MT2 melatonin receptor reveal the basis of subtype selectivity</span></div><div class="casAuthors">Johansson, Linda C.; Stauch, Benjamin; McCorvy, John D.; Han, Gye Won; Patel, Nilkanth; Huang, Xi-Ping; Batyuk, Alexander; Gati, Cornelius; Slocum, Samuel T.; Li, Chufeng; Grandner, Jessica M.; Hao, Shuming; Olsen, Reid H. J.; Tribo, Alexandra R.; Zaare, Sahba; Zhu, Lan; Zatsepin, Nadia A.; Weierstall, Uwe; Yous, Said; Stevens, Raymond C.; Liu, Wei; Roth, Bryan L.; Katritch, Vsevolod; Cherezov, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">569</span>
        (<span class="NLM_cas:issue">7755</span>),
    <span class="NLM_cas:pages">289-292</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The human MT1 and MT2 melatonin receptors1,2 are G-protein-coupled receptors (GPCRs) that help to regulate circadian rhythm and sleep patterns3.  Drug development efforts have targeted both receptors for the treatment of insomnia, circadian rhythm and mood disorders, and cancer3, and MT2 has also been implicated in type 2 diabetes4,5.  Here we report X-ray free electron laser (XFEL) structures of the human MT2 receptor in complex with the agonists 2-phenylmelatonin (2-PMT) and ramelteon6 at resolns. of 2.8 Å and 3.3 Å, resp., along with two structures of function-related mutants: H2085.46A (superscripts represent the Ballesteros-Weinstein residue numbering nomenclature7) and N862.50D, obtained in complex with 2-PMT.  Comparison of the structures of MT2 with a published structure8 of MT1 reveals that, despite conservation of the orthosteric ligand-binding site residues, there are notable conformational variations as well as differences in [3H]melatonin dissocn. kinetics that provide insights into the selectivity between melatonin receptor subtypes.  A membrane-buried lateral ligand entry channel is obsd. in both MT1 and MT2, but in addn. the MT2 structures reveal a narrow opening towards the solvent in the extracellular part of the receptor.  We provide functional and kinetic data that support a prominent role for intramembrane ligand entry in both receptors, and suggest that there might also be an extracellular entry path in MT2.  Our findings contribute to a mol. understanding of melatonin receptor subtype selectivity and ligand access modes, which are essential for the design of highly selective melatonin tool compds. and therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrePfxnJ1rXDbVg90H21EOLACvtfcHk0ljAOIH9KpPNLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVSkuro%253D&md5=6627fc1e26c873f6a14a7a4d4bf3a346</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1144-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1144-0%26sid%3Dliteratum%253Aachs%26aulast%3DJohansson%26aufirst%3DL.%2BC.%26aulast%3DStauch%26aufirst%3DB.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DBatyuk%26aufirst%3DA.%26aulast%3DGati%26aufirst%3DC.%26aulast%3DSlocum%26aufirst%3DS.%2BT.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DGrandner%26aufirst%3DJ.%2BM.%26aulast%3DHao%26aufirst%3DS.%26aulast%3DOlsen%26aufirst%3DR.%2BH.%2BJ.%26aulast%3DTribo%26aufirst%3DA.%2BR.%26aulast%3DZaare%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DZatsepin%26aufirst%3DN.%2BA.%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DYous%26aufirst%3DS.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DXFEL%2520Structures%2520of%2520the%2520Human%2520MT2%2520Melatonin%2520Receptor%2520Reveal%2520the%2520Basis%2520of%2520Subtype%2520Selectivity%26jtitle%3DNature%26date%3D2019%26volume%3D569%26spage%3D289%26epage%3D292%26doi%3D10.1038%2Fs41586-019-1144-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Longo, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavran, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leahy, D. J.</span></span> <span> </span><span class="NLM_article-title">Transient Mammalian Cell Transfection with Polyethylenimine (PEI)</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">227</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1016/b978-0-12-418687-3.00018-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fb978-0-12-418687-3.00018-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=24011049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslGgt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2013&pages=227-240&author=P.+A.+Longoauthor=J.+M.+Kavranauthor=M.-S.+Kimauthor=D.+J.+Leahy&title=Transient+Mammalian+Cell+Transfection+with+Polyethylenimine+%28PEI%29&doi=10.1016%2Fb978-0-12-418687-3.00018-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Transient mammalian cell transfection with polyethyylenimine (PEI)</span></div><div class="casAuthors">Longo, Patti A.; Kavran, Jennifer M.; Kim, Min-Sung; Leahy, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">Laboratory Methods in Enzymology: DNA</span>),
    <span class="NLM_cas:pages">227-240</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Std. protein expression systems, such as E. coli, often fail to produce folded, monodisperse, or functional eukaryotic proteins (see Small-scale Expression of Proteins in E. coli).  The expression of these proteins is greatly benefited by using a eukaryotic system, such as mammalian cells, that contains the appropriate folding and posttranslational machinery.  Here, we describe methods for both small- and large-scale transient expression in mammalian cells using polyethylenimine (PEI).  We find this procedure to be more cost-effective and quicker than the more traditional route of generating stable cell lines.  First, optimal transfection conditions are detd. on a small-scale, using adherent cells.  These conditions are then translated for use in large-scale suspension cultures.  For further details on generating stable cell lines please (see Rapid creation of stable mammalian cell lines for regulated expression of proteins using the Gateway Recombination Cloning Technol. and Flp-InT-REx lines or Generating mammalian stable cell lines by electroporation).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDaeKbQiaGD7Vg90H21EOLACvtfcHk0lh31NZsk0g0aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslGgt7zP&md5=0548d9c7c95a04d40aeb7a2ac5e14d45</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fb978-0-12-418687-3.00018-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fb978-0-12-418687-3.00018-5%26sid%3Dliteratum%253Aachs%26aulast%3DLongo%26aufirst%3DP.%2BA.%26aulast%3DKavran%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DM.-S.%26aulast%3DLeahy%26aufirst%3DD.%2BJ.%26atitle%3DTransient%2520Mammalian%2520Cell%2520Transfection%2520with%2520Polyethylenimine%2520%2528PEI%2529%26jtitle%3DMethods%2520Enzymol.%26date%3D2013%26volume%3D529%26spage%3D227%26epage%3D240%26doi%3D10.1016%2Fb978-0-12-418687-3.00018-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship Between the Inhibition Constant (K1) and the Concentration of Inhibitor Which Causes 50 Per Cent Inhibition (I50) of an Enzymatic Reaction</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>– <span class="NLM_lpage">3108</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=C.+Chengauthor=W.+H.+Prusoff&title=Relationship+Between+the+Inhibition+Constant+%28K1%29+and+the+Concentration+of+Inhibitor+Which+Causes+50+Per+Cent+Inhibition+%28I50%29+of+an+Enzymatic+Reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0lhfduriBeRzIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DC.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520Between%2520the%2520Inhibition%2520Constant%2520%2528K1%2529%2520and%2520the%2520Concentration%2520of%2520Inhibitor%2520Which%2520Causes%252050%2520Per%2520Cent%2520Inhibition%2520%2528I50%2529%2520of%2520an%2520Enzymatic%2520Reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xi-Ping Huang, Terrence P. Kenakin, Shuo Gu, Brian K. Shoichet, <span class="NLM_string-name hlFld-ContribAuthor">Bryan L. Roth</span>. </span><span class="cited-content_cbyCitation_article-title">Differential Roles of Extracellular Histidine Residues of GPR68 for Proton-Sensing and Allosteric Modulation by Divalent Metal Ions. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2020,</strong> <em>59 </em>
                                    (38)
                                     , 3594-3614. <a href="https://doi.org/10.1021/acs.biochem.0c00576" title="DOI URL">https://doi.org/10.1021/acs.biochem.0c00576</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.0c00576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.0c00576%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DDifferential%252BRoles%252Bof%252BExtracellular%252BHistidine%252BResidues%252Bof%252BGPR68%252Bfor%252BProton-Sensing%252Band%252BAllosteric%252BModulation%252Bby%252BDivalent%252BMetal%252BIons%26aulast%3DHuang%26aufirst%3DXi-Ping%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14072020%26date%3D28082020%26date%3D10092020%26date%3D31082020%26volume%3D59%26issue%3D38%26spage%3D3594%26epage%3D3614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Muhammad Syafiq  Bin Shahari</span>, <span class="hlFld-ContribAuthor ">Ahmad  Junaid</span>, <span class="hlFld-ContribAuthor ">Edward R. T.  Tiekink</span>, <span class="hlFld-ContribAuthor ">Anton V.  Dolzhenko</span>. </span><span class="cited-content_cbyCitation_article-title">A New One-Pot Three-Component Synthesis of 4-Aryl-6-cycloamino-1,3,5-triazin-2-amines under Microwave Irradiation. </span><span class="cited-content_cbyCitation_journal-name">Synthesis</span><span> <strong>2021,</strong> <em>53 </em>
                                    (14)
                                     , 2457-2468. <a href="https://doi.org/10.1055/a-1401-2795" title="DOI URL">https://doi.org/10.1055/a-1401-2795</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1055/a-1401-2795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1055%2Fa-1401-2795%26sid%3Dliteratum%253Aachs%26jtitle%3DSynthesis%26atitle%3DA%252BNew%252BOne-Pot%252BThree-Component%252BSynthesis%252Bof%252B4-Aryl-6-cycloamino-1%25252C3%25252C5-triazin-2-amines%252Bunder%252BMicrowave%252BIrradiation%26aulast%3DBin%2BShahari%26aufirst%3DMuhammad%2BSyafiq%26date%3D2021%26date%3D2021%26volume%3D53%26issue%3D14%26spage%3D2457%26epage%3D2468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ab  Majeed Ganai</span>, <span class="hlFld-ContribAuthor ">Tabasum  Khan Pathan</span>, <span class="hlFld-ContribAuthor ">Girish A.  Hampannavar</span>, <span class="hlFld-ContribAuthor ">Chandrakant  Pawar</span>, <span class="hlFld-ContribAuthor ">Vincent A.  Obakachi</span>, <span class="hlFld-ContribAuthor ">Babita  Kushwaha</span>, <span class="hlFld-ContribAuthor ">Narva  Deshwar Kushwaha</span>, <span class="hlFld-ContribAuthor ">Rajshekhar  Karpoormath</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances on the s‐Triazine Scaffold with Emphasis on Synthesis, Structure‐Activity and Pharmacological Aspects: A Concise Review. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2021,</strong> <em>6 </em>
                                    (7)
                                     , 1616-1660. <a href="https://doi.org/10.1002/slct.202004591" title="DOI URL">https://doi.org/10.1002/slct.202004591</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.202004591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.202004591%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DRecent%252BAdvances%252Bon%252Bthe%252Bs%2525E2%252580%252590Triazine%252BScaffold%252Bwith%252BEmphasis%252Bon%252BSynthesis%25252C%252BStructure%2525E2%252580%252590Activity%252Band%252BPharmacological%252BAspects%25253A%252BA%252BConcise%252BReview%26aulast%3DMajeed%2BGanai%26aufirst%3DAb%26date%3D2021%26date%3D2021%26volume%3D6%26issue%3D7%26spage%3D1616%26epage%3D1660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marie-Aude  Hiebel</span>, <span class="hlFld-ContribAuthor ">Franck  Suzenet</span>. </span><span class="cited-content_cbyCitation_article-title">Triazines, tetrazines, and fused ring polyaza systems. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 467-499. <a href="https://doi.org/10.1016/B978-0-323-89812-6.00014-6" title="DOI URL">https://doi.org/10.1016/B978-0-323-89812-6.00014-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-323-89812-6.00014-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-323-89812-6.00014-6%26sid%3Dliteratum%253Aachs%26atitle%3DTriazines%25252C%252Btetrazines%25252C%252Band%252Bfused%252Bring%252Bpolyaza%252Bsystems%26aulast%3DHiebel%26aufirst%3DMarie-Aude%26date%3D2021%26spage%3D467%26epage%3D499%26pub%3DElsevier%26date%3D2021%26volume%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Simon R.  Foster</span>, <span class="hlFld-ContribAuthor ">Alexander S.  Hauser</span>, <span class="hlFld-ContribAuthor ">Line  Vedel</span>, <span class="hlFld-ContribAuthor ">Ryan T.  Strachan</span>, <span class="hlFld-ContribAuthor ">Xi-Ping  Huang</span>, <span class="hlFld-ContribAuthor ">Ariana C.  Gavin</span>, <span class="hlFld-ContribAuthor ">Sushrut D.  Shah</span>, <span class="hlFld-ContribAuthor ">Ajay P.  Nayak</span>, <span class="hlFld-ContribAuthor ">Linda M.  Haugaard-Kedström</span>, <span class="hlFld-ContribAuthor ">Raymond B.  Penn</span>, <span class="hlFld-ContribAuthor ">Bryan L.  Roth</span>, <span class="hlFld-ContribAuthor ">Hans  Bräuner-Osborne</span>, <span class="hlFld-ContribAuthor ">David E.  Gloriam</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Human Signaling Systems: Pairing Peptides to G Protein-Coupled Receptors. </span><span class="cited-content_cbyCitation_journal-name">Cell</span><span> <strong>2019,</strong> <em>179 </em>
                                    (4)
                                     , 895-908.e21. <a href="https://doi.org/10.1016/j.cell.2019.10.010" title="DOI URL">https://doi.org/10.1016/j.cell.2019.10.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cell.2019.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cell.2019.10.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%26atitle%3DDiscovery%252Bof%252BHuman%252BSignaling%252BSystems%25253A%252BPairing%252BPeptides%252Bto%252BG%252BProtein-Coupled%252BReceptors%26aulast%3DFoster%26aufirst%3DSimon%2BR.%26date%3D2019%26volume%3D179%26issue%3D4%26spage%3D895%26epage%3D908.e21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0001.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structure of <b>1</b> and three highlighted regions for SAR study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0005.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>5</b>–<b>7</b> and <b>10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) tetrahydrofuran (THF), 0 °C to room temperature (rt), 48–67%; (b) dioxane, reflux, 1 h, 56–96%; (c) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O = 5:3, 120 °C, microwave, 20 min, 42–98%.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>13</b>–<b>15</b>, <b>21</b>, <b>16</b>, <b>18</b>, <b>19</b>, <b>22</b>–<b>29</b>, <b>31</b>, <b>32</b>, <b>37</b>, and <b>38</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) dioxane, reflux, 1 h, 78–94%; (b) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O = 5:3, 120 °C, microwave, 20 min, 56–99%; (c) NaBH<sub>4,</sub> MeOH, 0 °C to rt, 42%; (d) NH<sub>3</sub>·H<sub>2</sub>O or NH<sub>2</sub>Me, MgSO<sub>4</sub>, NaBH<sub>4</sub>, THF, rt, 2 days, 45–67%. (e) Ethylene glycol, toluene, reflux, 4–8 h, 82–96%; (f) alkyl bromide, K<sub>2</sub>CO<sub>3</sub>, dimethylformamide, 85 °C, 34–78%; (g) i. <i>n</i>-BuLi, THF, −78 °C, 30 min; ii. B(OMe)<sub>3</sub>, −78 °C, 1 h, then rt, overnight; iii. HCl (3 N), rt, 1 h, 26-57%; (h) <b>12</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O = 5:3, 120 °C, microwave, 20 min; (i) NaBH<sub>4</sub>, THF, 0 °C to rt, 23–97%.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0007.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>41</b>–<b>54</b>, <b>56</b>–<b>60</b>, and <b>64</b>–<b>81</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) dioxane, reflux, 1 h, 75–96 %; (b) Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O = 5:3, 120 °C, microwave, 20 min, 16–99%.</p></p></figure><figure data-id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Selectivity of the GPR68 PAM 71. Proton concentration–response curves in the absence and presence of increasing concentrations of <b>1</b> at human GPR68 (A), <b>71</b> at human GPR68 (B), GPR4 (C), and GPR65 (D). Proton-mediated G<sub>s</sub>–cAMP production was measured in HEK293T cells transiently transfected with GPR68, GPR4, or GPR65, and GloSensor cAMP plasmids. Results were normalized to the proton activity in the absence of modulator and represented means ± SEM from a minimum of three independent assays, each in quadruplicate. Curves were analyzed in Prism using the built-in four parameter logistic functions. (E) Selectivity of <b>71</b> over 48 common GPCRs, ion channels, and transporters in radioligand binding assays. Values represent average binding affinity (p<i>K</i><sub>i</sub>) from a minimum of three independent assays, each in triplicate. Targets with less than 50% inhibition at 10 μM (in quadruplicate) are not shown. BZP for benzodiazepine receptor; DAT for dopamine transporter; DOR for delta (δ) opioid receptor; hERG for human ether-a-go-go-related potassium channel; KOR for kappa (κ) opioid receptor; MOR for mu (μ) opioid receptor; NET for noradrenaline transporter; PBR for peripheral benzodiazepine binding site; SERT for serotonin transporter. (F) Off-target agonist activity of <b>71</b> at human GPCRome. <b>71</b> was screened across the human GPCRome (318 GPCRs) for off-target agonist activity at 1 μM using the PRESTO-Tango screening platform. Results (fold of basal) represent means of four replicates. Dopamine receptor D<sub>2</sub> with 100 nM quinpirole served as an assay control. The dashed lines indicate an arbitrary cutoff line at 3× fold of basal.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Allosteric potentiation of proton activity by compound <b>71</b> at mouse GPR68. G<sub>s</sub> GloSensor cAMP assays were carried out in HEK293T cells transiently transfected with mouse GPR68 as with human GPR68. Proton concentration responses were measured in the absence and presence of increasing concentrations of modulator. Results were normalized to proton activity in the absence of modulator and represented means ± SEM from a minimum of three independent assays, each in quadruplicate. Curves were analyzed in Prism. Allosteric parameters are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/medium/jm-2019-00869k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Mouse pharmacokinetic profile of compound <b>71</b>. Compound concentrations were quantified in both plasma (red curve) and brain (blue curve) at 0.5, 1.0, and 2.0 h after a single 25 mg/kg intraperitoneal (IP) administration. Compound concentrations in plasma and brain at each time point are average values from three test mice. The error bars represent ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00869/20190815/images/large/jm-2019-00869k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00869&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i99">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50205" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50205" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 66 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casey, G.</span></span> <span> </span><span class="NLM_article-title">Identification of Human OGR1, a Novel G Protein-Coupled Receptor That Maps to Chromosome 14</span>. <i>Genomics</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">397</span>– <span class="NLM_lpage">402</span>, <span class="refDoi"> DOI: 10.1006/geno.1996.0377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1006%2Fgeno.1996.0377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=8661159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADyaK28Xks1aqsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1996&pages=397-402&author=Y.+Xuauthor=G.+Casey&title=Identification+of+Human+OGR1%2C+a+Novel+G+Protein-Coupled+Receptor+That+Maps+to+Chromosome+14&doi=10.1006%2Fgeno.1996.0377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of human OGR1, a novel G protein-coupled receptor that maps to chromosome 14</span></div><div class="casAuthors">Xu, Yan; Casey, Graham</div><div class="citationInfo"><span class="NLM_cas:title">Genomics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">397-402</span>CODEN:
                <span class="NLM_cas:coden">GNMCEP</span>;
        ISSN:<span class="NLM_cas:issn">0888-7543</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">We describe the cloning of a novel G protein-coupled receptor, termed ovarian cancer G protein-coupled receptor 1 (OGR1), from an ovarian cancer cell line that maps to chromosome 14q31.  The predicted open reading frame of OGR1 encodes a protein of 365 amino acids.  OGR1 is expressed as a single 3.0-kb transcript in several tissues, including spleen, testis, small intestine, peripheral blood leukocytes, brain, heart, lung, placenta, and kidney, with no detectable OGR1 expression in thymus, prostate, ovary, colon, liver, skeletal muscle, or pancreas.  OGR1 shares strongest homol. (49-54%) with the human orphan receptor GPR4.  The structural features of OGR1 suggest that the ligand for OGR1 is likely to be a small peptide or a small bioactive mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0WdsDcxdHO7Vg90H21EOLACvtfcHk0lip0KHWi8dt8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xks1aqsro%253D&md5=a311f68e33b2fe0ef84d7fe704d888a4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1006%2Fgeno.1996.0377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fgeno.1996.0377%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DCasey%26aufirst%3DG.%26atitle%3DIdentification%2520of%2520Human%2520OGR1%252C%2520a%2520Novel%2520G%2520Protein-Coupled%2520Receptor%2520That%2520Maps%2520to%2520Chromosome%252014%26jtitle%3DGenomics%26date%3D1996%26volume%3D35%26spage%3D397%26epage%3D402%26doi%3D10.1006%2Fgeno.1996.0377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, M.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasser, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junker, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofstetter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seuwen, K.</span></span> <span> </span><span class="NLM_article-title">Proton-Sensing G-Protein-Coupled Receptors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>425</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">98</span>, <span class="refDoi"> DOI: 10.1038/nature01905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fnature01905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=12955148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFansrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2003&pages=93-98&author=M.-G.+Ludwigauthor=M.+Vanekauthor=D.+Gueriniauthor=J.+A.+Gasserauthor=C.+E.+Jonesauthor=U.+Junkerauthor=H.+Hofstetterauthor=R.+M.+Wolfauthor=K.+Seuwen&title=Proton-Sensing+G-Protein-Coupled+Receptors&doi=10.1038%2Fnature01905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Proton-sensing G-protein-coupled receptors</span></div><div class="casAuthors">Ludwig, Marie-Gabrielle; Vanek, Miroslava; Guerini, Danilo; Gasser, Juerg A.; Jones, Carol E.; Junker, Uwe; Hofstetter, Hans; Wolf, Romain M.; Seuwen, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">425</span>
        (<span class="NLM_cas:issue">6953</span>),
    <span class="NLM_cas:pages">93-98</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Blood pH is maintained in a narrow range around pH 7.4 mainly through regulation of respiration and renal acid extrusion.  The mol. mechanisms involved in pH homeostasis are not completely understood.  Here we show that ovarian cancer G-protein-coupled receptor 1 (OGR1), previously described as a receptor for sphingosylphosphorylcholine, acts as a proton-sensing receptor stimulating inositol phosphate formation.  The receptor is inactive at pH 7.8, and fully activated at pH 6.8-site-directed mutagenesis shows that histidines at the extracellular surface are involved in pH sensing.  We find that GPR4, a close relative of OGR1, also responds to pH changes, but elicits cAMP formation.  It is known that the skeleton participates in pH homeostasis as a buffering organ, and that osteoblasts respond to pH changes in the physiol. range, but the pH-sensing mechanism operating in these cells was hitherto not known.  We detect expression of OGR1 in osteosarcoma cells and primary human osteoblast precursors, and show that these cells exhibit strong pH-dependent inositol phosphate formation.  Immunohistochem. on rat tissue sections confirms the presence of OGR1 in osteoblasts and osteocytes.  We propose that OGR1 and GPR4 are proton-sensing receptors involved in pH homeostasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfpVd_U0kyY7Vg90H21EOLACvtfcHk0lip0KHWi8dt8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFansrw%253D&md5=131775a2f98871f4cbe6519b53679a0d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature01905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01905%26sid%3Dliteratum%253Aachs%26aulast%3DLudwig%26aufirst%3DM.-G.%26aulast%3DVanek%26aufirst%3DM.%26aulast%3DGuerini%26aufirst%3DD.%26aulast%3DGasser%26aufirst%3DJ.%2BA.%26aulast%3DJones%26aufirst%3DC.%2BE.%26aulast%3DJunker%26aufirst%3DU.%26aulast%3DHofstetter%26aufirst%3DH.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DSeuwen%26aufirst%3DK.%26atitle%3DProton-Sensing%2520G-Protein-Coupled%2520Receptors%26jtitle%3DNature%26date%3D2003%26volume%3D425%26spage%3D93%26epage%3D98%26doi%3D10.1038%2Fnature01905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Regard, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coughlin, S. R.</span></span> <span> </span><span class="NLM_article-title">Anatomical Profiling of G Protein-Coupled Receptor Expression</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">561</span>– <span class="NLM_lpage">571</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2008.08.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.cell.2008.08.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=18984166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlyqtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2008&pages=561-571&author=J.+B.+Regardauthor=I.+T.+Satoauthor=S.+R.+Coughlin&title=Anatomical+Profiling+of+G+Protein-Coupled+Receptor+Expression&doi=10.1016%2Fj.cell.2008.08.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Anatomical profiling of G protein-coupled receptor expression</span></div><div class="casAuthors">Regard, Jean B.; Sato, Isaac T.; Coughlin, Shaun R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">561-571</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) comprise the largest family of transmembrane signaling mols. and regulate a host of physiol. and disease processes.  To better understand the functions of GPCRs in vivo, we quantified transcript levels of 353 nonodorant GPCRs in 41 adult mouse tissues.  Cluster anal. placed many GPCRs into anticipated anatomical and functional groups and predicted previously unidentified roles for less-studied receptors.  From one such prediction, we showed that the Gpr91 ligand succinate can regulate lipolysis in white adipose tissue, suggesting that signaling by this citric acid cycle intermediate may regulate energy homeostasis.  We also showed that pairwise anal. of GPCR expression across tissues may help predict drug side effects.  This resource will aid studies to understand GPCR function in vivo and may assist in the identification of therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiJtFrCT3H77Vg90H21EOLACvtfcHk0ljLFpIuyhpynA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlyqtr3J&md5=dbe3c2c9c295050ec4ba86124c8aa2b4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2008.08.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2008.08.040%26sid%3Dliteratum%253Aachs%26aulast%3DRegard%26aufirst%3DJ.%2BB.%26aulast%3DSato%26aufirst%3DI.%2BT.%26aulast%3DCoughlin%26aufirst%3DS.%2BR.%26atitle%3DAnatomical%2520Profiling%2520of%2520G%2520Protein-Coupled%2520Receptor%2520Expression%26jtitle%3DCell%26date%3D2008%26volume%3D135%26spage%3D561%26epage%3D571%26doi%3D10.1016%2Fj.cell.2008.08.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davenport, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marrion, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faccenda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. A.</span></span> <span> </span><span class="NLM_article-title">CGTP Collaborators. The Concise Guide to Pharmacology 2017/18: G Protein-Coupled Receptors</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">S17</span>– <span class="NLM_lpage">S129</span>, <span class="refDoi"> DOI: 10.1111/bph.13878</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1111%2Fbph.13878" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=29055040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs12ls7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2017&pages=S17-S129&author=S.+P.+Alexanderauthor=A.+Christopoulosauthor=A.+P.+Davenportauthor=E.+Kellyauthor=N.+V.+Marrionauthor=J.+A.+Petersauthor=E.+Faccendaauthor=S.+D.+Hardingauthor=A.+J.+Pawsonauthor=J.+L.+Sharmanauthor=C.+Southanauthor=J.+A.+Davies&title=CGTP+Collaborators.+The+Concise+Guide+to+Pharmacology+2017%2F18%3A+G+Protein-Coupled+Receptors&doi=10.1111%2Fbph.13878"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">THE CONCISE GUIDE TO PHARMACOLOGY 2017/18: G protein-coupled receptors</span></div><div class="casAuthors">Alexander, Stephen P. H.; Christopoulos, Arthur; Davenport, Anthony P.; Kelly, Eamonn; Marrion, Neil V.; Peters, John A.; Faccenda, Elena; Harding, Simon D.; Pawson, Adam J.; Sharman, Joanna L.; Southan, Christopher; Davies, Jamie A.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue">S1</span>),
    <span class="NLM_cas:pages">S17-S129</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Concise Guide to PHARMACOl. 2017/18 provides concise overviews of the key properties of nearly 1800 human drug targets with an emphasis on selective pharmacol. (where available), plus links to an open access knowledgebase of drug targets and their ligands (), which provides more detailed views of target and ligand properties.  Although the Concise Guide represents approx. 400 pages, the material presented is substantially reduced compared to information and links presented on the website.  It provides a permanent, citable, point-in-time record that will survive database updates.  The full contents of this section can be found at .  G protein-coupled receptors are one of the eight major pharmacol. targets into which the Guide is divided, with the others being: ligand-gated ion channels, voltage-gated ion channels, other ion channels, nuclear hormone receptors, catalytic receptors, enzymes and transporters.  These are presented with nomenclature guidance and summary information on the best available pharmacol. tools, alongside key refs. and suggestions for further reading.  The landscape format of the Concise Guide is designed to facilitate comparison of related targets from material contemporary to mid-2017, and supersedes data presented in the 2015/16 and 2013/14 Concise Guides and previous Guides to Receptors and Channels.  It is produced in close conjunction with the Nomenclature Committee of the Union of Basic and Clin. Pharmacol. (NC-IUPHAR), therefore, providing official IUPHAR classification and nomenclature for human drug targets, where appropriate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrktrD3wjuPZ7Vg90H21EOLACvtfcHk0ljLFpIuyhpynA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs12ls7%252FO&md5=2d4bc1acc92b6aa659365aa70ba50983</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1111%2Fbph.13878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.13878%26sid%3Dliteratum%253Aachs%26aulast%3DAlexander%26aufirst%3DS.%2BP.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DDavenport%26aufirst%3DA.%2BP.%26aulast%3DKelly%26aufirst%3DE.%26aulast%3DMarrion%26aufirst%3DN.%2BV.%26aulast%3DPeters%26aufirst%3DJ.%2BA.%26aulast%3DFaccenda%26aufirst%3DE.%26aulast%3DHarding%26aufirst%3DS.%2BD.%26aulast%3DPawson%26aufirst%3DA.%2BJ.%26aulast%3DSharman%26aufirst%3DJ.%2BL.%26aulast%3DSouthan%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DJ.%2BA.%26atitle%3DCGTP%2520Collaborators.%2520The%2520Concise%2520Guide%2520to%2520Pharmacology%25202017%252F18%253A%2520G%2520Protein-Coupled%2520Receptors%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2017%26volume%3D174%26spage%3DS17%26epage%3DS129%26doi%3D10.1111%2Fbph.13878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mogi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damirin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komachi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span> <span> </span><span class="NLM_article-title">Sphingosylphosphorylcholine Antagonizes Proton-Sensing Ovarian Cancer G-Protein-Coupled Receptor 1 (OGR1)-Mediated Inositol Phosphate Production and cAMP Accumulation</span>. <i>J. Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">160</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1254/jphs.FP0050599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1254%2Fjphs.FP0050599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=16210776" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFOns7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2005&pages=160-167&author=C.+Mogiauthor=H.+Tomuraauthor=M.+Toboauthor=J.-Q.+Wangauthor=A.+Damirinauthor=J.+Konauthor=M.+Komachiauthor=K.+Hashimotoauthor=K.+Satoauthor=F.+Okajima&title=Sphingosylphosphorylcholine+Antagonizes+Proton-Sensing+Ovarian+Cancer+G-Protein-Coupled+Receptor+1+%28OGR1%29-Mediated+Inositol+Phosphate+Production+and+cAMP+Accumulation&doi=10.1254%2Fjphs.FP0050599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Sphingosylphosphorylcholine antagonizes proton-sensing ovarian cancer G-protein-coupled receptor 1 (OGR1)-mediated inositol phosphate production and cAMP accumulation</span></div><div class="casAuthors">Mogi, Chihiro; Tomura, Hideaki; Tobo, Masayuki; Wang, Ju-Qiang; Damirin, Alatangaole; Kon, Junko; Komachi, Mayumi; Hashimoto, Kinji; Sato, Koichi; Okajima, Fumikazu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacological Sciences (Tokyo, Japan)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">160-167</span>CODEN:
                <span class="NLM_cas:coden">JPSTGJ</span>;
        ISSN:<span class="NLM_cas:issn">1347-8613</span>.
    
            (<span class="NLM_cas:orgname">Japanese Pharmacological Society</span>)
        </div><div class="casAbstract">Ovarian cancer G-protein-coupled receptor 1 (OGR1), previously proposed as a receptor for sphingosylphosphorylcholine (SPC), has recently been identified as a proton-sensing or extracellular pH-responsive G-protein-coupled receptor stimulating inositol phosphate prodn., reflecting the activation of phospholipase C.  In the present study, we found that acidic pH stimulated cAMP accumulation, reflecting the activation of adenylyl cyclase, in addn. to inositol phosphate prodn. in OGR1-expressing cells.  The cAMP response was hardly affected by the inhibition of phospholipase C.  SPC inhibited the acidification-induced actions in a pH-dependent manner, while no OGR1-dependent agonistic action of SPC was obsd.  Thus, the dose-response curves of the proton-induced actions were shifted to the right in the presence of SPC regardless of stereoisoform.  The antagonistic property was also obsd. for psychosine and glucosylsphingosine.  In conclusion, OGR1 stimulation may lead to the activation of adenylyl cyclase in addn. to phospholipase C in response to extracellular acidification but not to SPC.  However, SPC and related lysolipids antagonize the proton-induced and OGR1-mediated actions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7WOLALesDkrVg90H21EOLACvtfcHk0ljLFpIuyhpynA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFOns7vO&md5=8fdb013385f4cdf2b187a9c9172bdec8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1254%2Fjphs.FP0050599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1254%252Fjphs.FP0050599%26sid%3Dliteratum%253Aachs%26aulast%3DMogi%26aufirst%3DC.%26aulast%3DTomura%26aufirst%3DH.%26aulast%3DTobo%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.-Q.%26aulast%3DDamirin%26aufirst%3DA.%26aulast%3DKon%26aufirst%3DJ.%26aulast%3DKomachi%26aufirst%3DM.%26aulast%3DHashimoto%26aufirst%3DK.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DOkajima%26aufirst%3DF.%26atitle%3DSphingosylphosphorylcholine%2520Antagonizes%2520Proton-Sensing%2520Ovarian%2520Cancer%2520G-Protein-Coupled%2520Receptor%25201%2520%2528OGR1%2529-Mediated%2520Inositol%2520Phosphate%2520Production%2520and%2520cAMP%2520Accumulation%26jtitle%3DJ.%2520Pharmacol.%2520Sci.%26date%3D2005%26volume%3D99%26spage%3D160%26epage%3D167%26doi%3D10.1254%2Fjphs.FP0050599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oates, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirmani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Ovarian Cancer G Protein-Coupled Receptor 1, a New Metastasis Suppressor Gene in Prostate Cancer</span>. <i>JNCI, J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">1313</span>– <span class="NLM_lpage">1327</span>, <span class="refDoi"> DOI: 10.1093/jnci/djm107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1093%2Fjnci%2Fdjm107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=17728215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWqtbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2007&pages=1313-1327&author=L.+S.+Singhauthor=M.+Berkauthor=R.+Oatesauthor=Z.+Zhaoauthor=H.+Tanauthor=Y.+Jiangauthor=A.+Zhouauthor=K.+Kirmaniauthor=R.+Steinmetzauthor=D.+Lindnerauthor=Y.+Xu&title=Ovarian+Cancer+G+Protein-Coupled+Receptor+1%2C+a+New+Metastasis+Suppressor+Gene+in+Prostate+Cancer&doi=10.1093%2Fjnci%2Fdjm107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Ovarian Cancer G Protein-Coupled Receptor 1, a New Metastasis Suppressor Gene in Prostate Cancer</span></div><div class="casAuthors">Singh, Lisam Shanjukumar; Berk, Michael; Oates, Rhonda; Zhao, Zhenwen; Tan, Haiyan; Jiang, Ying; Zhou, Aimin; Kirmani, Kashif; Steinmetz, Rosemary; Lindner, Daniel; Xu, Yan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1313-1327</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background: Metastasis is a process by which tumors spread from primary organs to other sites in the body and is the major cause of death for cancer patients.  The ovarian cancer G protein-coupled receptor 1 (OGR1) gene has been shown to be expressed at lower levels in metastatic compared with primary prostate cancer tissues.  Methods: We used an orthotopic mouse metastasis model, in which we injected PC3 metastatic human prostate cancer cells stably transfected with empty vector (vector-PC3) or OGR1-expressing vector (OGR1-PC3) into the prostate lobes of athymic or NOD/SCID mice (n = 3-8 mice per group).  Migration of PC3 cells transiently transfected with vector control or with OGR1- or GPR4 (a G protein-coupled receptor with the highest homol. to OGR1)-expressing vectors was measured in vitro by Boyden chamber assays.  G protein alpha-inhibitory subunit 1 (Gαi1) expression after treatment with pertussis toxin (PTX) was measured using immunoblotting anal.  The inhibitory factor present in the conditioned medium was extd. using org. solvents and analyzed by mass spectrometry.  Results: In vivo, all 26 mice carrying tumors that were derived from vector-PC3 cells developed prostate cancer metastases (mean = 100%, 95% confidence interval [Cl] = 83.97% to 100%) but few (4 of 32) mice carrying tumors derived from OGR1-expressing PC3 cells (mean = 12.50%, 95% Cl = 4.08% to 29.93%) developed metastases.  However, exogenous OGR1 overexpression had no effect on primary prostate tumor growth in vivo.  In vitro, expression of OGR1, but not GPR4, inhibited cell migration (mean percentage of cells migrated, 30.2% vs. 100%, difference = 69.8%, 95% Cl = 63.0% to 75.9%; P < .001) via increased expression of Gαi1 and the secretion fo a chloroform/methanol-extractable heat-insensitive factor into the conditioned medium through a PTX-sensitive pathway.  Conclusion: OGR1 is a novel metastasis suppressor gene for prostate cancer.  OGR1's constitutive activity via Gαi contributes to its inhibitory effect on cell migration in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrB2_OCs3OF67Vg90H21EOLACvtfcHk0li0L9GOTu6TfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWqtbjF&md5=2b9d5ff8e805fb22665665412ac3cc1c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjm107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjm107%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DL.%2BS.%26aulast%3DBerk%26aufirst%3DM.%26aulast%3DOates%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DA.%26aulast%3DKirmani%26aufirst%3DK.%26aulast%3DSteinmetz%26aufirst%3DR.%26aulast%3DLindner%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DOvarian%2520Cancer%2520G%2520Protein-Coupled%2520Receptor%25201%252C%2520a%2520New%2520Metastasis%2520Suppressor%2520Gene%2520in%2520Prostate%2520Cancer%26jtitle%3DJNCI%252C%2520J.%2520Natl.%2520Cancer%2520Inst.%26date%3D2007%26volume%3D99%26spage%3D1313%26epage%3D1327%26doi%3D10.1093%2Fjnci%2Fdjm107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, J.</span></span> <span> </span><span class="NLM_article-title">Ovarian Cancer G Protein Coupled Receptor 1 Suppresses Cell Migration of MCF7 Breast Cancer Cells via a Gα12/13-Rho-Rac1 Pathway</span>. <i>J. Mol. Signal.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">6</span> <span class="refDoi"> DOI: 10.1186/1750-2187-8-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1186%2F1750-2187-8-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=23663350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BC3snivV2ntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=J.+Liauthor=B.+Guoauthor=J.+Wangauthor=X.+Chengauthor=Y.+Xuauthor=J.+Sang&title=Ovarian+Cancer+G+Protein+Coupled+Receptor+1+Suppresses+Cell+Migration+of+MCF7+Breast+Cancer+Cells+via+a+G%CE%B112%2F13-Rho-Rac1+Pathway&doi=10.1186%2F1750-2187-8-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Ovarian cancer G protein coupled receptor 1 suppresses cell migration of MCF7 breast cancer cells via a Gα12/13-Rho-Rac1 pathway</span></div><div class="casAuthors">Li Jing; Guo Bin; Wang Jing; Cheng Xiaoyan; Xu Yan; Sang Jianli</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular signaling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6</span>
        ISSN:<span class="NLM_cas:issn">1750-2187</span>.
    </div><div class="casAbstract">BACKGROUND:  Ovarian cancer G protein coupled receptor 1 (OGR1) mediates inhibitory effects on cell migration in human prostate and ovarian cancer cells.  However, the mechanisms and signaling pathways that mediate these inhibitory effects are essentially unknown.  METHODS:  MCF7 cell line was chosen as a model system to study the mechanisms by which OGR1 regulates cell migration, since it expresses very low levels of endogenous OGR1.  Cell migratory activities were assessed using both wound healing and transwell migration assays.  The signaling pathways involved were studied using pharmacological inhibitors and genetic forms of the relevant genes, as well as small G protein pull-down activity assays.  The expression levels of various signaling molecules were analyzed by Western blot and quantitative PCR analysis.  RESULTS:  Over-expression of OGR1 in MCF7 cells substantially enhanced activation of Rho and inhibition of Rac1, resulting in inhibition of cell migration.  In addition, expression of the Gα12/13 specific regulator of G protein signaling (RGS) domain of p115RhoGEF, but not treatment with pertussis toxin (PTX, a Gαi specific inhibitor), could abrogate OGR1-dependent Rho activation, Rac1 inactivation, and inhibition of migration in MCF7 cells.  The bioactive lipids tested had no effect on OGR1 function in cell migration.  CONCLUSION:  Our data suggest, for the first time, that OGR1 inhibits cell migration through a Gα12/13 -Rho-Rac1 signaling pathway in MCF7 cells.  This pathway was not significantly affected by bioactive lipids and all the assays were conducted at constant pH, suggesting a constitutive activity of OGR1.  This is the first clear delineation of an OGR1-mediated cell signaling pathway involved in migration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSAWumsAcuPOgHqQV89nF-2fW6udTcc2ebkYi0fymD-r7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3snivV2ntg%253D%253D&md5=9524ed612a2f1db66a9f1db5ab771e3b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2F1750-2187-8-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1750-2187-8-6%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DSang%26aufirst%3DJ.%26atitle%3DOvarian%2520Cancer%2520G%2520Protein%2520Coupled%2520Receptor%25201%2520Suppresses%2520Cell%2520Migration%2520of%2520MCF7%2520Breast%2520Cancer%2520Cells%2520via%2520a%2520G%25CE%25B112%252F13-Rho-Rac1%2520Pathway%26jtitle%3DJ.%2520Mol.%2520Signal.%26date%3D2013%26volume%3D8%26doi%3D10.1186%2F1750-2187-8-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Damaghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtkowiak, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillies, R. J.</span></span> <span> </span><span class="NLM_article-title">pH Sensing and Regulation in Cancer</span>. <i>Front. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">370</span>, <span class="refDoi"> DOI: 10.3389/fphys.2013.00370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.3389%2Ffphys.2013.00370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=24381558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BC2cflsVymtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=370&author=M.+Damaghiauthor=J.+W.+Wojtkowiakauthor=R.+J.+Gillies&title=pH+Sensing+and+Regulation+in+Cancer&doi=10.3389%2Ffphys.2013.00370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">pH sensing and regulation in cancer</span></div><div class="casAuthors">Damaghi Mehdi; Wojtkowiak Jonathan W; Gillies Robert J</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in physiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">370</span>
        ISSN:<span class="NLM_cas:issn">1664-042X</span>.
    </div><div class="casAbstract">Cells maintain intracellular pH (pHi) within a narrow range (7.1-7.2) by controlling membrane proton pumps and transporters whose activity is set by intra-cytoplasmic pH sensors.  These sensors have the ability to recognize and induce cellular responses to maintain the pHi, often at the expense of acidifying the extracellular pH.  In turn, extracellular acidification impacts cells via specific acid-sensing ion channels (ASICs) and proton-sensing G-protein coupled receptors (GPCRs).  In this review, we will discuss some of the major players in proton sensing at the plasma membrane and their downstream consequences in cancer cells and how these pH-mediated changes affect processes such as migration and metastasis.  The complex mechanisms by which they transduce acid pH signals to the cytoplasm and nucleus are not well understood.  However, there is evidence that expression of proton-sensing GPCRs such as GPR4, TDAG8, and OGR1 can regulate aspects of tumorigenesis and invasion, including cofilin and talin regulated actin (de-)polymerization.  Major mechanisms for maintenance of pHi homeostasis include monocarboxylate, bicarbonate, and proton transporters.  Notably, there is little evidence suggesting a link between their activities and those of the extracellular H(+)-sensors, suggesting a mechanistic disconnect between intra- and extracellular pH.  Understanding the mechanisms of pH sensing and regulation may lead to novel and informed therapeutic strategies that can target acidosis, a common physical hallmark of solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRiMyC_tQmaj-3MPKNEWB5RfW6udTcc2ebkYi0fymD-r7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cflsVymtQ%253D%253D&md5=3a46626baf76f257c63c9d97c73aaa0c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3389%2Ffphys.2013.00370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphys.2013.00370%26sid%3Dliteratum%253Aachs%26aulast%3DDamaghi%26aufirst%3DM.%26aulast%3DWojtkowiak%26aufirst%3DJ.%2BW.%26aulast%3DGillies%26aufirst%3DR.%2BJ.%26atitle%3DpH%2520Sensing%2520and%2520Regulation%2520in%2520Cancer%26jtitle%3DFront.%2520Physiol.%26date%3D2013%26volume%3D4%26spage%3D370%26doi%3D10.3389%2Ffphys.2013.00370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirmani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">Abnormalities in Osteoclastogenesis and Decreased Tumorigenesis in Mice Deficient for Ovarian Cancer G Protein-Coupled Receptor 1</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e5705</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0005705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1371%2Fjournal.pone.0005705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=19479052" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&author=H.+Liauthor=D.+Wangauthor=L.+S.+Singhauthor=M.+Berkauthor=H.+Tanauthor=Z.+Zhaoauthor=R.+Steinmetzauthor=K.+Kirmaniauthor=G.+Weiauthor=Y.+Xu&title=Abnormalities+in+Osteoclastogenesis+and+Decreased+Tumorigenesis+in+Mice+Deficient+for+Ovarian+Cancer+G+Protein-Coupled+Receptor+1&doi=10.1371%2Fjournal.pone.0005705"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0005705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0005705%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DSingh%26aufirst%3DL.%2BS.%26aulast%3DBerk%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DSteinmetz%26aufirst%3DR.%26aulast%3DKirmani%26aufirst%3DK.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DAbnormalities%2520in%2520Osteoclastogenesis%2520and%2520Decreased%2520Tumorigenesis%2520in%2520Mice%2520Deficient%2520for%2520Ovarian%2520Cancer%2520G%2520Protein-Coupled%2520Receptor%25201%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26doi%3D10.1371%2Fjournal.pone.0005705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">Effects of Ovarian Cancer G Protein Coupled Receptor 1 on the Proliferation, Migration, and Adhesion of Human Ovarian Cancer Cells</span>. <i>Chin. Med. J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">1327</span>– <span class="NLM_lpage">1332</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=21740742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BC3MnmvFCqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2011&pages=1327-1332&author=J.+Renauthor=L.+Zhang&title=Effects+of+Ovarian+Cancer+G+Protein+Coupled+Receptor+1+on+the+Proliferation%2C+Migration%2C+and+Adhesion+of+Human+Ovarian+Cancer+Cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of ovarian cancer G protein coupled receptor 1 on the proliferation, migration, and adhesion of human ovarian cancer cells</span></div><div class="casAuthors">Ren Juan; Zhang Long</div><div class="citationInfo"><span class="NLM_cas:title">Chinese medical journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1327-32</span>
        ISSN:<span class="NLM_cas:issn">0366-6999</span>.
    </div><div class="casAbstract">BACKGROUND:  OGR1 was found as a G-protein coupled receptor (GPCR) and proton sensor.  Our previous studies have found that OGR1 has inhibitory effect on the metastasis of prostate cancer.  In order to investigate the roles of OGR1 gene in the biological activities of ovarian cancer, we studied the OGR1 effects on ovarian cancer cells, HEY cells.  METHODS:  OGR1 gene was transfected into HEY cell, in which endogenous expression is low.  OGR1-overxepressed cells and vector-transfected cells were compared in different assays.  Western blotting was employed to confirm the high expression level of OGR1.  Cell proliferation was determined by MTT assay and cell doubling time assay.  Cell migration assay (transwell assay) and cell adhesion assay were performed to determine the migration and adhesion potential of cells.  Student's t test was employed for statistical analysis.  RESULTS:  Proliferation of OGR1-overexpressed cells was significantly reduced (P < 0.01); cell migration was significantly inhibited in the OGR1-transfected cells (P < 0.01); cell adhesion to extracellular matrix including fibronectin, vitronectin, collagen I/IV was significantly increased (P < 0.01).  CONCLUSIONS:  OGR1 expression in human ovarian cancer cells significantly inhibited the cell proliferation and migration, but significantly enhanced cell adhesion to the extracellular matrix.  It indicated that OGR1 may be a tumor suppressor gene for ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR0VQIsxKz7Z--MdeMhXAHkfW6udTcc2ebbO1qrCOp_Ybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MnmvFCqsg%253D%253D&md5=585e6f6b286f6be37c4534b7eb00d83f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DEffects%2520of%2520Ovarian%2520Cancer%2520G%2520Protein%2520Coupled%2520Receptor%25201%2520on%2520the%2520Proliferation%252C%2520Migration%252C%2520and%2520Adhesion%2520of%2520Human%2520Ovarian%2520Cancer%2520Cells%26jtitle%3DChin.%2520Med.%2520J.%26date%3D2011%26volume%3D124%26spage%3D1327%26epage%3D1332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiley, S. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sriram, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicklick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowy, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insel, P. A.</span></span> <span> </span><span class="NLM_article-title">GPR68, a Proton-Sensing GPCR, Mediates Interaction of Cancer-Associated Fibroblasts and Cancer Cells</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1170</span>– <span class="NLM_lpage">1183</span>, <span class="refDoi"> DOI: 10.1096/fj.201700834R</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1096%2Ffj.201700834R" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=29092903" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWnt7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=1170-1183&author=S.+Z.+Wileyauthor=K.+Sriramauthor=W.+Liangauthor=S.+E.+Changauthor=R.+Frenchauthor=T.+McCannauthor=J.+Sicklickauthor=H.+Nishiharaauthor=A.+M.+Lowyauthor=P.+A.+Insel&title=GPR68%2C+a+Proton-Sensing+GPCR%2C+Mediates+Interaction+of+Cancer-Associated+Fibroblasts+and+Cancer+Cells&doi=10.1096%2Ffj.201700834R"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">GPR68, a proton-sensing GPCR, mediates interaction of cancer-associated fibroblasts and cancer cells</span></div><div class="casAuthors">Wiley, Shu Z.; Sriram, Krishna; Liang, Wenjing; Chang, Sarah E.; French, Randall; McCann, Thalia; Sicklick, Jason; Nishihara, Hiroshi; Lowy, Andrew M.; Insel, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1170-1183</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">The microenvironment of pancreatic ductal adenocarcinoma (PDAC) is characterized by a dense fibrotic stroma (desmoplasia) generated by pancreatic cancer-assocd. fibroblasts (CAFs) derived from pancreatic stellate cells (PSCs) and pancreatic fibroblasts (PFs).  Using an unbiased GPCRomic array approach, we identified 82 G-protein-coupled receptors (GPCRs) commonly expressed by CAFs derived from 5 primary PDAC tumors.  Compared with PSCs and PFs, CAFs have increased expression of GPR68 (a proton-sensing GPCR), with the results confirmed by immunoblotting, The Cancer Genome Atlas data, and immunohistochem. of PDAC tumors.  Coculture of PSCs with PDAC cells, or incubation with TNF-α, induced GPR68 expression.  GPR68 activation (by decreasing the extracellular pH) enhanced IL-6 expression via a cAMP/PKA/cAMP response element binding protein signaling pathway.  Knockdown of GPR68 by short interfering RNA diminished low pH-induced prodn. of IL-6 and enhancement of PDAC cell proliferation by CAF conditioned media.  CAFs from other gastrointestinal cancers also express GPR68.  PDAC cells thus induce expression by CAFs of GPR68, which senses the acidic microenvironment, thereby increasing prodn. of fibrotic markers and IL-6 and promoting PDAC cell proliferation.  CAF-expressed GPR68 is a mediator of low-pH-promoted regulation of the tumor microenvironments, in particular to PDAC cell-CAF interaction and may be a novel therapeutic target for pancreatic and perhaps other types of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLeNoINb1PMrVg90H21EOLACvtfcHk0lg2MPeJxuRgyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWnt7nE&md5=d5c11ef52e6ec43e166c9a7d400e4caa</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1096%2Ffj.201700834R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.201700834R%26sid%3Dliteratum%253Aachs%26aulast%3DWiley%26aufirst%3DS.%2BZ.%26aulast%3DSriram%26aufirst%3DK.%26aulast%3DLiang%26aufirst%3DW.%26aulast%3DChang%26aufirst%3DS.%2BE.%26aulast%3DFrench%26aufirst%3DR.%26aulast%3DMcCann%26aufirst%3DT.%26aulast%3DSicklick%26aufirst%3DJ.%26aulast%3DNishihara%26aufirst%3DH.%26aulast%3DLowy%26aufirst%3DA.%2BM.%26aulast%3DInsel%26aufirst%3DP.%2BA.%26atitle%3DGPR68%252C%2520a%2520Proton-Sensing%2520GPCR%252C%2520Mediates%2520Interaction%2520of%2520Cancer-Associated%2520Fibroblasts%2520and%2520Cancer%2520Cells%26jtitle%3DFASEB%2520J.%26date%3D2018%26volume%3D32%26spage%3D1170%26epage%3D1183%26doi%3D10.1096%2Ffj.201700834R" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span> <span> </span><span class="NLM_article-title">Regulation of Inflammation by Extracellular Acidification and Proton-Sensing GPCRs</span>. <i>Cell. Signal.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2263</span>– <span class="NLM_lpage">2271</span>, <span class="refDoi"> DOI: 10.1016/j.cellsig.2013.07.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.cellsig.2013.07.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=23917207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlyrtLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2013&pages=2263-2271&author=F.+Okajima&title=Regulation+of+Inflammation+by+Extracellular+Acidification+and+Proton-Sensing+GPCRs&doi=10.1016%2Fj.cellsig.2013.07.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of inflammation by extracellular acidification and proton-sensing GPCRs</span></div><div class="casAuthors">Okajima, Fumikazu</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Signalling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2263-2271</span>CODEN:
                <span class="NLM_cas:coden">CESIEY</span>;
        ISSN:<span class="NLM_cas:issn">0898-6568</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Under ischemic and inflammatory circumstances, such as allergic airway asthma, rheumatoid arthritis, atherosclerosis, and tumors, extracellular acidification occurs due to the stimulation of anaerobic glycolysis.  An acidic microenvironment has been shown to modulate pro-inflammatory or anti-inflammatory responses, including cyclooxygenase-2 (COX-2) expression, prostaglandin synthesis, and cytokine expression, in a variety of cell types, and thereby to exacerbate or ameliorate inflammation.  However, mol. mechanisms underlying extracellular acidic pH-induced actions have not been fully understood.  Recent studies have shown that ovarian cancer G protein-coupled receptor 1 (OGR1)-family G protein-coupled receptors (GPCRs) can sense extracellular pH or protons, which in turn stimulates intracellular signaling pathways and subsequent diverse cellular responses.  In the present review, I discuss extracellular acidic pH-induced inflammatory responses and related responses in inflammatory cells, such as macrophages and neutrophils, and non-inflammatory cells, such as smooth muscle cells and endothelial cells, focusing esp. on proton-sensing GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoViaL3lpdAL7Vg90H21EOLACvtfcHk0lg2MPeJxuRgyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlyrtLrF&md5=7bd4b1d6cf1bac0a5aba1c4bf2619ed9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cellsig.2013.07.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellsig.2013.07.022%26sid%3Dliteratum%253Aachs%26aulast%3DOkajima%26aufirst%3DF.%26atitle%3DRegulation%2520of%2520Inflammation%2520by%2520Extracellular%2520Acidification%2520and%2520Proton-Sensing%2520GPCRs%26jtitle%3DCell.%2520Signal.%26date%3D2013%26volume%3D25%26spage%3D2263%26epage%3D2271%26doi%3D10.1016%2Fj.cellsig.2013.07.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, W.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.-H.</span></span> <span> </span><span class="NLM_article-title">Expression and Function of Proton-Sensing G-Protein-Coupled Receptors in Inflammatory Pain</span>. <i>Mol. Pain</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">S39</span>, <span class="refDoi"> DOI: 10.1016/j.jpain.2009.01.167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.jpain.2009.01.167" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=S39&author=Y.-J.+Chenauthor=C.-W.+Huangauthor=C.-S.+Linauthor=W.-H.+Changauthor=W.-H.+Sun&title=Expression+and+Function+of+Proton-Sensing+G-Protein-Coupled+Receptors+in+Inflammatory+Pain&doi=10.1016%2Fj.jpain.2009.01.167"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.jpain.2009.01.167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jpain.2009.01.167%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.-J.%26aulast%3DHuang%26aufirst%3DC.-W.%26aulast%3DLin%26aufirst%3DC.-S.%26aulast%3DChang%26aufirst%3DW.-H.%26aulast%3DSun%26aufirst%3DW.-H.%26atitle%3DExpression%2520and%2520Function%2520of%2520Proton-Sensing%2520G-Protein-Coupled%2520Receptors%2520in%2520Inflammatory%2520Pain%26jtitle%3DMol.%2520Pain%26date%3D2009%26volume%3D10%26spage%3DS39%26doi%3D10.1016%2Fj.jpain.2009.01.167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Vallière, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eloranta, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clay, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spalinger, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tcymbarevich, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terhalle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, M.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suply, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fried, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kullak-Ublick, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frey-Wagner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seuwen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogler, G.</span></span> <span> </span><span class="NLM_article-title">G Protein-Coupled pH-Sensing Receptor OGR1 Is a Regulator of Intestinal Inflammation</span>. <i>Inflammatory Bowel Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1269</span>– <span class="NLM_lpage">1281</span>, <span class="refDoi"> DOI: 10.1097/MIB.0000000000000375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1097%2FMIB.0000000000000375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=25856770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BC2MjitlOisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1269-1281&author=C.+de%0AValli%C3%A8reauthor=Y.+Wangauthor=J.+J.+Elorantaauthor=S.+Vidalauthor=I.+Clayauthor=M.+R.+Spalingerauthor=I.+Tcymbarevichauthor=A.+Terhalleauthor=M.-G.+Ludwigauthor=T.+Suplyauthor=M.+Friedauthor=G.+A.+Kullak-Ublickauthor=I.+Frey-Wagnerauthor=M.+Scharlauthor=K.+Seuwenauthor=C.+A.+Wagnerauthor=G.+Rogler&title=G+Protein-Coupled+pH-Sensing+Receptor+OGR1+Is+a+Regulator+of+Intestinal+Inflammation&doi=10.1097%2FMIB.0000000000000375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">G Protein-coupled pH-sensing Receptor OGR1 Is a Regulator of Intestinal Inflammation</span></div><div class="casAuthors">de Valliere Cheryl; Wang Yu; Eloranta Jyrki J; Vidal Solange; Clay Ieuan; Spalinger Marianne R; Tcymbarevich Irina; Terhalle Anne; Ludwig Marie-Gabrielle; Suply Thomas; Fried Michael; Kullak-Ublick Gerd A; Frey-Wagner Isabelle; Scharl Michael; Seuwen Klaus; Wagner Carsten A; Rogler Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Inflammatory bowel diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1269-81</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  A novel family of proton-sensing G protein-coupled receptors, including OGR1, GPR4, and TDAG8, was identified to be important for physiological pH homeostasis and inflammation.  Thus, we determined the function of proton-sensing OGR1 in the intestinal mucosa.  MTEHODS:  OGR1 expression in colonic tissues was investigated in controls and patients with IBD.  Expression of OGR1 upon cell activation was studied in the Mono Mac 6 (MM6) cell line and primary human and murine monocytes by real-time PCR.  Ogr1 knockout mice were crossbred with Il-10 deficient mice and studied for more than 200 days.  Microarray profiling was performed using Ogr1 and Ogr1 (WT) residential peritoneal macrophages.  RESULTS:  Patients with IBD expressed higher levels of OGR1 in the mucosa than non-IBD controls.  Treatment of MM6 cells with TNF, led to significant upregulation of OGR1 expression, which could be reversed by the presence of NF-κB inhibitors.  Kaplan-Meier survival analysis showed a significantly delayed onset and progression of rectal prolapse in female Ogr1/Il-10 mice.  These mice displayed significantly less rectal prolapses.  Upregulation of gene expression, mediated by OGR1, in response to extracellular acidification in mouse macrophages was enriched for inflammation and immune response, actin cytoskeleton, and cell-adhesion gene pathways.  CONCLUSIONS:  OGR1 expression is induced in cells of human macrophage lineage and primary human monocytes by TNF.  NF-κB inhibition reverses the induction of OGR1 expression by TNF.  OGR1 deficiency protects from spontaneous inflammation in the Il-10 knockout model.  Our data indicate a pathophysiological role for pH-sensing receptor OGR1 during the pathogenesis of mucosal inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTevE0e4oFxjZdYoSR9eyvsfW6udTcc2eaZFWzLTwoPwrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjitlOisQ%253D%253D&md5=a530ecb419d883b77581519a74c33ebd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1097%2FMIB.0000000000000375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FMIB.0000000000000375%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BValli%25C3%25A8re%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DEloranta%26aufirst%3DJ.%2BJ.%26aulast%3DVidal%26aufirst%3DS.%26aulast%3DClay%26aufirst%3DI.%26aulast%3DSpalinger%26aufirst%3DM.%2BR.%26aulast%3DTcymbarevich%26aufirst%3DI.%26aulast%3DTerhalle%26aufirst%3DA.%26aulast%3DLudwig%26aufirst%3DM.-G.%26aulast%3DSuply%26aufirst%3DT.%26aulast%3DFried%26aufirst%3DM.%26aulast%3DKullak-Ublick%26aufirst%3DG.%2BA.%26aulast%3DFrey-Wagner%26aufirst%3DI.%26aulast%3DScharl%26aufirst%3DM.%26aulast%3DSeuwen%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DC.%2BA.%26aulast%3DRogler%26aufirst%3DG.%26atitle%3DG%2520Protein-Coupled%2520pH-Sensing%2520Receptor%2520OGR1%2520Is%2520a%2520Regulator%2520of%2520Intestinal%2520Inflammation%26jtitle%3DInflammatory%2520Bowel%2520Dis.%26date%3D2015%26volume%3D21%26spage%3D1269%26epage%3D1281%26doi%3D10.1097%2FMIB.0000000000000375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mailhot, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birnbaum, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKay, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason-Savas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odgren, P. R.</span></span> <span> </span><span class="NLM_article-title">Expression of and Role for Ovarian Cancer G-Protein-Coupled Receptor 1 (OGR1) During Osteoclastogenesis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">23598</span>– <span class="NLM_lpage">23605</span>, <span class="refDoi"> DOI: 10.1074/jbc.M602191200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1074%2Fjbc.M602191200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=16787916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD28XotVCmtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=23598-23605&author=M.+Yangauthor=G.+Mailhotauthor=M.+J.+Birnbaumauthor=C.+A.+MacKayauthor=A.+Mason-Savasauthor=P.+R.+Odgren&title=Expression+of+and+Role+for+Ovarian+Cancer+G-Protein-Coupled+Receptor+1+%28OGR1%29+During+Osteoclastogenesis&doi=10.1074%2Fjbc.M602191200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of and Role for Ovarian Cancer G-protein-coupled Receptor 1 (OGR1) during Osteoclastogenesis</span></div><div class="casAuthors">Yang, Meiheng; Mailhot, Genevieve; Birnbaum, Mark J.; MacKay, Carole A.; Mason-Savas, April; Odgren, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">23598-23605</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Osteoclasts differentiate from hematopoietic mononuclear precursor cells under the control of both colony stimulating factor-1 (CSF-1, or M-CSF) and receptor activator of NF-κB ligand (RANKL, or TRANCE, TNFSF11) to carry out bone resorption.  Using high d. gene microarrays, we followed gene expression changes in long bone RNA when CSF-1 injections were used to restore osteoclast populations in the CSF-1-null toothless (csf1tl/csf1tl) osteopetrotic rat.  We found that ovarian cancer G-protein-coupled receptor 1 (OGR1, or GPR68) was strongly up-regulated, rising >6-fold in vivo after 2 days of CSF-1 treatments.  OGR1 is a dual membrane receptor for both protons (extracellular pH) and lysolipids.  Strong induction of OGR1 mRNA was also obsd. by microarray, real-time RT-PCR, and immunoblotting when mouse bone marrow mononuclear cells and RAW 264.7 pre-osteoclast-like cells were treated with RANKL to induce osteoclast differentiation.  Anti-OGR1 immunofluorescence showed intense labeling of RANKL-treated RAW cells.  The time course of OGR1 mRNA expression suggests that OGR1 induction is early but not immediate, peaking 2 days after inducing osteoclast differentiation both in vivo and in vitro.  Specific inhibition of OGR1 by anti-OGR1 antibody and by small inhibitory RNA inhibited RANKL-induced differentiation of both mouse bone marrow mononuclear cells and RAW cells in vitro, as evidenced by a decrease in tartrate-resistant acid phosphatase-pos. osteoclasts.  Taken together, these data indicate that OGR1 is expressed early during osteoclastogenesis both in vivo and in vitro and plays a role in osteoclast differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr18tKXf0aByLVg90H21EOLACvtfcHk0lgOiPhd3lOrhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XotVCmtb4%253D&md5=138ea5bea27a2a00c0cb218401187f53</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M602191200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M602191200%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DM.%26aulast%3DMailhot%26aufirst%3DG.%26aulast%3DBirnbaum%26aufirst%3DM.%2BJ.%26aulast%3DMacKay%26aufirst%3DC.%2BA.%26aulast%3DMason-Savas%26aufirst%3DA.%26aulast%3DOdgren%26aufirst%3DP.%2BR.%26atitle%3DExpression%2520of%2520and%2520Role%2520for%2520Ovarian%2520Cancer%2520G-Protein-Coupled%2520Receptor%25201%2520%2528OGR1%2529%2520During%2520Osteoclastogenesis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D23598%26epage%3D23605%26doi%3D10.1074%2Fjbc.M602191200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pereverzev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komarova, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korcok, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tremblay, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, S. M.</span></span> <span> </span><span class="NLM_article-title">Extracellular Acidification Enhances Osteoclast Survival through an NFAT-Independent, Protein Kinase C-Dependent Pathway</span>. <i>Bone</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1016/j.bone.2007.08.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.bone.2007.08.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=17964236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOrtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2008&pages=150-161&author=A.+Pereverzevauthor=S.+V.+Komarovaauthor=J.+Korcokauthor=S.+Armstrongauthor=G.+B.+Tremblayauthor=S.+J.+Dixonauthor=S.+M.+Sims&title=Extracellular+Acidification+Enhances+Osteoclast+Survival+through+an+NFAT-Independent%2C+Protein+Kinase+C-Dependent+Pathway&doi=10.1016%2Fj.bone.2007.08.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular acidification enhances osteoclast survival through an NFAT-independent, protein kinase C-dependent pathway</span></div><div class="casAuthors">Pereverzev, Alexey; Komarova, Svetlana V.; Korcok, Jasminka; Armstrong, Souzan; Tremblay, Gilles B.; Dixon, S. Jeffrey; Sims, Stephen M.</div><div class="citationInfo"><span class="NLM_cas:title">Bone (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">150-161</span>CODEN:
                <span class="NLM_cas:coden">BONEDL</span>;
        ISSN:<span class="NLM_cas:issn">8756-3282</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Systemic acidosis has detrimental effects on the skeleton and local acidosis is assocd. with bone destruction in inflammatory and neoplastic diseases.  However, the mechanisms by which acidosis enhances osteoclastic bone resorption are poorly understood.  Our aim was to examine the effects of acid on osteoclast survival and the involvement of cytosolic Ca2+ in mediating these effects.  Osteoclasts were isolated from long bones of newborn rats, and multinucleated osteoclast-like cells were generated from RAW 264.7 cells.  Cytosolic free Ca2+ concn. ([Ca2+]i) was monitored using fura-2.  Survival of rat osteoclasts over a period of 18 h was significantly enhanced by acidification of the medium from 40 ± 10% at pH 7.6 to 83 ± 4% at pH 7.0.  Consistent with its effects on survival, acidosis suppressed osteoclast apoptosis at 6 h.  We examd. the possible involvement of the proton-sensing receptor ovarian cancer G protein-coupled receptor 1 (OGR1) in mediating the effects of acid.  Acid-induced rise of [Ca2+]i was inhibited by the OGR1 antagonist Cu2+ and was suppressed in osteoclast-like cells in which OGR1 transcripts were depleted using RNA interference.  These findings support an essential role for OGR1 in acid-induced Ca2+ signaling in osteoclasts.  Addn. of Cu2+ or chelation of cytosolic Ca2+ with BAPTA abolished the ability of acidification to enhance osteoclast survival.  Inhibition of NFAT activation with the cell-permeable peptide 11R-VIVIT did not alter the ability of acid to promote survival; however, it suppressed the increase in survival induced by RANKL.  In contrast, inhibition of protein kinase C (PKC) blocked the effect of acid on osteoclast survival.  Thus, this study reveals that extracellular acidification enhances osteoclast survival through an NFAT-independent, PKC-dependent pathway.  Increased osteoclast survival may contribute to bone loss in systemic and local acidosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhDkL1S7QxArVg90H21EOLACvtfcHk0lgOiPhd3lOrhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOrtrrO&md5=e5ce8afa2e263ab2126dddf8e923551f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bone.2007.08.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bone.2007.08.044%26sid%3Dliteratum%253Aachs%26aulast%3DPereverzev%26aufirst%3DA.%26aulast%3DKomarova%26aufirst%3DS.%2BV.%26aulast%3DKorcok%26aufirst%3DJ.%26aulast%3DArmstrong%26aufirst%3DS.%26aulast%3DTremblay%26aufirst%3DG.%2BB.%26aulast%3DDixon%26aufirst%3DS.%2BJ.%26aulast%3DSims%26aufirst%3DS.%2BM.%26atitle%3DExtracellular%2520Acidification%2520Enhances%2520Osteoclast%2520Survival%2520through%2520an%2520NFAT-Independent%252C%2520Protein%2520Kinase%2520C-Dependent%2520Pathway%26jtitle%3DBone%26date%3D2008%26volume%3D42%26spage%3D150%26epage%3D161%26doi%3D10.1016%2Fj.bone.2007.08.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hisada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakakura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamide, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimonji, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsurumaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishizuka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span> <span> </span><span class="NLM_article-title">Proton-Sensing Ovarian Cancer G Protein-Coupled Receptor 1 on Dendritic Cells is Required for Airway Responses in a Murine Asthma Model</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e79985</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0079985</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1371%2Fjournal.pone.0079985" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=24244587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsl2jtLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=H.+Aokiauthor=C.+Mogiauthor=T.+Hisadaauthor=T.+Nakakuraauthor=Y.+Kamideauthor=I.+Ichimonjiauthor=H.+Tomuraauthor=M.+Toboauthor=K.+Satoauthor=H.+Tsurumakiauthor=K.+Dobashiauthor=T.+Moriauthor=A.+Haradaauthor=M.+Yamadaauthor=M.+Moriauthor=T.+Ishizukaauthor=F.+Okajima&title=Proton-Sensing+Ovarian+Cancer+G+Protein-Coupled+Receptor+1+on+Dendritic+Cells+is+Required+for+Airway+Responses+in+a+Murine+Asthma+Model&doi=10.1371%2Fjournal.pone.0079985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Proton-sensing ovarian cancer G protein-coupled receptor 1 on dendritic cells is required for airway responses in a murine asthma model</span></div><div class="casAuthors">Aoki, Haruka; Mogi, Chihiro; Hisada, Takeshi; Nakakura, Takashi; Kamide, Yosuke; Ichimonji, Isao; Tomura, Hideaki; Tobo, Masayuki; Sato, Koichi; Tsurumaki, Hiroaki; Dobashi, Kunio; Mori, Tetsuya; Harada, Akihiro; Yamada, Masanobu; Mori, Masatomo; Ishizuka, Tamotsu; Okajima, Fumikazu</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e79985</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Ovarian cancer G protein-coupled receptor 1 (OGR1) stimulation by extracellular protons causes the activation of G proteins and subsequent cellular functions.  However, the physiol. and pathophysiol. roles of OGR1 in airway responses remain largely unknown.  In the present study, we show that OGR1-deficient mice are resistant to the cardinal features of asthma, including airway eosinophilia, airway hyperresponsiveness (AHR), and goblet cell metaplasia, in assocn. with a remarkable inhibition of Th2 cytokine and IgE prodn., in an ovalbumin (OVA)-induced asthma model.  Intratracheal transfer to wild-type mice of OVA-primed bone marrow-derived dendritic cells (DCs) from OGR1-deficient mice developed lower AHR and eosinophilia after OVA inhalation compared with the transfer of those from wild-type mice.  Migration of OVA-pulsed DCs to peribronchial lymph nodes was also inhibited by OGR1 deficiency in the adoption expts.  The presence of functional OGR1 in DCs was confirmed by the expression of OGR1 mRNA and the OGR1-sensitive Ca2+ response.  OVA-induced expression of CCR7, a mature DC chemokine receptor, and migration response to CCR7 ligands in an in vitro Transwell assay were attenuated by OGR1 deficiency.  We conclude that OGR1 on DCs is crit. for migration to draining lymph nodes, which, in turn, stimulates Th2 phenotype change and subsequent induction of airway inflammation and AHR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNZZP8vtaesrVg90H21EOLACvtfcHk0ljzVpOZbg2KWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsl2jtLjN&md5=c42fdcd789144205cf6f5f50c01b6a79</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0079985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0079985%26sid%3Dliteratum%253Aachs%26aulast%3DAoki%26aufirst%3DH.%26aulast%3DMogi%26aufirst%3DC.%26aulast%3DHisada%26aufirst%3DT.%26aulast%3DNakakura%26aufirst%3DT.%26aulast%3DKamide%26aufirst%3DY.%26aulast%3DIchimonji%26aufirst%3DI.%26aulast%3DTomura%26aufirst%3DH.%26aulast%3DTobo%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DTsurumaki%26aufirst%3DH.%26aulast%3DDobashi%26aufirst%3DK.%26aulast%3DMori%26aufirst%3DT.%26aulast%3DHarada%26aufirst%3DA.%26aulast%3DYamada%26aufirst%3DM.%26aulast%3DMori%26aufirst%3DM.%26aulast%3DIshizuka%26aufirst%3DT.%26aulast%3DOkajima%26aufirst%3DF.%26atitle%3DProton-Sensing%2520Ovarian%2520Cancer%2520G%2520Protein-Coupled%2520Receptor%25201%2520on%2520Dendritic%2520Cells%2520is%2520Required%2520for%2520Airway%2520Responses%2520in%2520a%2520Murine%2520Asthma%2520Model%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0079985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakakura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnishi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wayama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span> <span> </span><span class="NLM_article-title">Deficiency of Proton-Sensing Ovarian Cancer G Protein-Coupled Receptor 1 Attenuates Glucose-Stimulated Insulin Secretion</span>. <i>Endocrinology</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">4171</span>– <span class="NLM_lpage">4180</span>, <span class="refDoi"> DOI: 10.1210/en.2012-1164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1210%2Fen.2012-1164" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=22733973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlagtL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2012&pages=4171-4180&author=T.+Nakakuraauthor=C.+Mogiauthor=M.+Toboauthor=H.+Tomuraauthor=K.+Satoauthor=M.+Kobayashiauthor=H.+Ohnishiauthor=S.+Tanakaauthor=M.+Wayamaauthor=T.+Sugiyamaauthor=T.+Kitamuraauthor=A.+Haradaauthor=F.+Okajima&title=Deficiency+of+Proton-Sensing+Ovarian+Cancer+G+Protein-Coupled+Receptor+1+Attenuates+Glucose-Stimulated+Insulin+Secretion&doi=10.1210%2Fen.2012-1164"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Deficiency of proton-sensing ovarian cancer G protein-coupled receptor 1 attenuates glucose-stimulated insulin secretion</span></div><div class="casAuthors">Nakakura, Takashi; Mogi, Chihiro; Tobo, Masayuki; Tomura, Hideaki; Sato, Koichi; Kobayashi, Masaki; Ohnishi, Hiroshi; Tanaka, Shigeyasu; Wayama, Mitsutoshi; Sugiyama, Tetsuya; Kitamura, Tadahiro; Harada, Akihiro; Okajima, Fumikazu</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4171-4180</span>CODEN:
                <span class="NLM_cas:coden">ENDOAO</span>;
        ISSN:<span class="NLM_cas:issn">0013-7227</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Ovarian cancer G protein-coupled receptor 1 (OGR1) has been shown as a receptor for protons.  In the present study, we aimed to know whether OGR1 plays a role in insulin secretion and, if so, the manner in which it does.  To this end, we created OGR1-deficient mice and examd. insulin secretion activity in vivo and in vitro.  OGR1 deficiency reduced insulin secretion induced by glucose administered i.p., although it was not assocd. with glucose intolerance in vivo.  Increased insulin sensitivity and reduced plasma glucagon level may explain, in part, the unusual normal glucose tolerance.  In vitro islet expts. revealed that glucose-stimulated insulin secretion was dependent on extracellular pH and sensitive to OGR1; insulin secretion at pH 7.4 to 7.0, but not 8.0, was significantly suppressed by OGR1 deficiency and inhibition of Gq/11 proteins.  Insulin secretion induced by KCl and tolbutamide was also significantly inhibited, whereas that induced by several insulin secretagogues, including vasopressin, a glucagon-like peptide 1 receptor agonist, and forskolin, was not suppressed by OGR1 deficiency.  The inhibition of insulin secretion was assocd. with the redn. of glucose-induced increase in intracellular Ca2+ concn.  In conclusion, the OGR1/Gq/11 protein pathway is activated by extracellular protons existing under the physiol. extracellular pH of 7.4 and further stimulated by acidification, resulting in the enhancement of insulin secretion in response to high glucose concns. and KCl.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6HbzjeKWwcbVg90H21EOLACvtfcHk0ljzVpOZbg2KWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlagtL7L&md5=289021f9a87e717f8261188faf0194ca</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1210%2Fen.2012-1164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2012-1164%26sid%3Dliteratum%253Aachs%26aulast%3DNakakura%26aufirst%3DT.%26aulast%3DMogi%26aufirst%3DC.%26aulast%3DTobo%26aufirst%3DM.%26aulast%3DTomura%26aufirst%3DH.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DKobayashi%26aufirst%3DM.%26aulast%3DOhnishi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DWayama%26aufirst%3DM.%26aulast%3DSugiyama%26aufirst%3DT.%26aulast%3DKitamura%26aufirst%3DT.%26aulast%3DHarada%26aufirst%3DA.%26aulast%3DOkajima%26aufirst%3DF.%26atitle%3DDeficiency%2520of%2520Proton-Sensing%2520Ovarian%2520Cancer%2520G%2520Protein-Coupled%2520Receptor%25201%2520Attenuates%2520Glucose-Stimulated%2520Insulin%2520Secretion%26jtitle%3DEndocrinology%26date%3D2012%26volume%3D153%26spage%3D4171%26epage%3D4180%26doi%3D10.1210%2Fen.2012-1164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohebbi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benabbas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daryadel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourgeois, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, M.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seuwen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, C. A.</span></span> <span> </span><span class="NLM_article-title">The Proton-Activated G Protein Coupled Receptor OGR1 Acutely Regulates the Activity of Epithelial Proton Transport Proteins</span>. <i>Cell. Physiol. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1159/000338486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1159%2F000338486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=22508039" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls12isr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=313-324&author=N.+Mohebbiauthor=C.+Benabbasauthor=S.+Vidalauthor=A.+Daryadelauthor=S.+Bourgeoisauthor=A.+Velicauthor=M.-G.+Ludwigauthor=K.+Seuwenauthor=C.+A.+Wagner&title=The+Proton-Activated+G+Protein+Coupled+Receptor+OGR1+Acutely+Regulates+the+Activity+of+Epithelial+Proton+Transport+Proteins&doi=10.1159%2F000338486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The Proton-activated G Protein Coupled Receptor OGR1 Acutely Regulates the Activity of Epithelial Proton Transport Proteins</span></div><div class="casAuthors">Mohebbi, Nilufar; Benabbas, Chahira; Vidal, Solange; Daryadel, Arezoo; Bourgeois, Soline; Velic, Ana; Ludwig, Marie-Gabrielle; Seuwen, Klaus; Wagner, Carsten A.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular Physiology and Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">313-324</span>CODEN:
                <span class="NLM_cas:coden">CEPBEW</span>;
        ISSN:<span class="NLM_cas:issn">1015-8987</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">The Ovarian cancer G protein-coupled Receptor 1 (OGR1; GPR68) is proton-sensitive in the pH range of 6.8 - 7.8.  However, its physiol. function is not defined to date.  OGR1 signals via inositol trisphosphate and intracellular calcium, albeit downstream events are unclear.  To elucidate OGR1 function further, we transfected HEK293 cells with active OGR1 receptor or a mutant lacking 5 histidine residues (H5Phe-OGR1).  An acute switch of extracellular pH from 8 to 7.1 (10 nmol/l vs 90 nmol/l protons) stimulated NHE and H+-ATPase activity in OGR1-transfected cells, but not in H5Phe-OGR1-transfected cells.  ZnCl2 and CuCl2 that both inhibit OGR1 reduced the stimulatory effect.  The activity was blocked by chelerythrine, whereas the ERK1/2 inhibitor PD 098059 had no inhibitory effect.  OGR1 activation increased intracellular calcium in transfected HEK293 cells.  We next isolated proximal tubules from kidneys of wild-type and OGR1-deficient mice and measured the effect of extracellular pH on NHE activity in vitro.  Deletion of OGR1 affected the pH-dependent proton extrusion, however, in the opposite direction as expected from cell culture expts.  Upregulated expression of the pH-sensitive kinase Pyk2 in OGR1 KO mouse proximal tubule cells may compensate for the loss of OGR1.  Thus, we present the first evidence that OGR1 modulates the activity of two major plasma membrane proton transport systems.  OGR1 may be involved in the regulation of plasma membrane transport proteins and intra- and/or extracellular pH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUmm5Hg76aDLVg90H21EOLACvtfcHk0liTnPwGmcZiVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls12isr0%253D&md5=bd394fbc4729548e3a1f6f09a14c3ef6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1159%2F000338486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000338486%26sid%3Dliteratum%253Aachs%26aulast%3DMohebbi%26aufirst%3DN.%26aulast%3DBenabbas%26aufirst%3DC.%26aulast%3DVidal%26aufirst%3DS.%26aulast%3DDaryadel%26aufirst%3DA.%26aulast%3DBourgeois%26aufirst%3DS.%26aulast%3DVelic%26aufirst%3DA.%26aulast%3DLudwig%26aufirst%3DM.-G.%26aulast%3DSeuwen%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DC.%2BA.%26atitle%3DThe%2520Proton-Activated%2520G%2520Protein%2520Coupled%2520Receptor%2520OGR1%2520Acutely%2520Regulates%2520the%2520Activity%2520of%2520Epithelial%2520Proton%2520Transport%2520Proteins%26jtitle%3DCell.%2520Physiol.%2520Biochem.%26date%3D2012%26volume%3D29%26spage%3D313%26epage%3D324%26doi%3D10.1159%2F000338486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Vallière, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clay, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jurisic, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tcymbarevich, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, M.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okoniewski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eloranta, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kullak-Ublick, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seuwen, K.</span></span> <span> </span><span class="NLM_article-title">The pH-Sensing Receptor OGR1 Improves Barrier Function of Epithelial Cells and Inhibits Migration in an Acidic Environment</span>. <i>Am. J. Physiol.: Gastrointest. Liver Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>309</i></span>,  <span class="NLM_fpage">G475</span>– <span class="NLM_lpage">G490</span>, <span class="refDoi"> DOI: 10.1152/ajpgi.00408.2014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1152%2Fajpgi.00408.2014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=26206859" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=309&publication_year=2015&pages=G475-G490&author=C.+de%0AValli%C3%A8reauthor=S.+Vidalauthor=I.+Clayauthor=G.+Jurisicauthor=I.+Tcymbarevichauthor=S.+Langauthor=M.-G.+Ludwigauthor=M.+Okoniewskiauthor=J.+J.+Elorantaauthor=G.+A.+Kullak-Ublickauthor=C.+A.+Wagnerauthor=G.+Roglerauthor=K.+Seuwen&title=The+pH-Sensing+Receptor+OGR1+Improves+Barrier+Function+of+Epithelial+Cells+and+Inhibits+Migration+in+an+Acidic+Environment&doi=10.1152%2Fajpgi.00408.2014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1152%2Fajpgi.00408.2014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpgi.00408.2014%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BValli%25C3%25A8re%26aufirst%3DC.%26aulast%3DVidal%26aufirst%3DS.%26aulast%3DClay%26aufirst%3DI.%26aulast%3DJurisic%26aufirst%3DG.%26aulast%3DTcymbarevich%26aufirst%3DI.%26aulast%3DLang%26aufirst%3DS.%26aulast%3DLudwig%26aufirst%3DM.-G.%26aulast%3DOkoniewski%26aufirst%3DM.%26aulast%3DEloranta%26aufirst%3DJ.%2BJ.%26aulast%3DKullak-Ublick%26aufirst%3DG.%2BA.%26aulast%3DWagner%26aufirst%3DC.%2BA.%26aulast%3DRogler%26aufirst%3DG.%26aulast%3DSeuwen%26aufirst%3DK.%26atitle%3DThe%2520pH-Sensing%2520Receptor%2520OGR1%2520Improves%2520Barrier%2520Function%2520of%2520Epithelial%2520Cells%2520and%2520Inhibits%2520Migration%2520in%2520an%2520Acidic%2520Environment%26jtitle%3DAm.%2520J.%2520Physiol.%253A%2520Gastrointest.%2520Liver%2520Physiol.%26date%3D2015%26volume%3D309%26spage%3DG475%26epage%3DG490%26doi%3D10.1152%2Fajpgi.00408.2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vessières, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nonomura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimaud, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francisco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, F.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainquist, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cahalan, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orth, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordone, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laffitte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrion, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patapoutian, A.</span></span> <span> </span><span class="NLM_article-title">GPR68 Senses Flow and Is Essential for Vascular Physiology</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">762.e16</span>– <span class="NLM_lpage">775.e16</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.03.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.cell.2018.03.076" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=762.e16-775.e16&author=J.+Xuauthor=J.+Mathurauthor=E.+Vessi%C3%A8resauthor=S.+Hammackauthor=K.+Nonomuraauthor=J.+Favreauthor=L.+Grimaudauthor=M.+Petrusauthor=A.+Franciscoauthor=J.+Liauthor=V.+Leeauthor=F.-L.+Xiangauthor=J.+K.+Mainquistauthor=S.+M.+Cahalanauthor=A.+P.+Orthauthor=J.+R.+Walkerauthor=S.+Maauthor=V.+Lukacsauthor=L.+Bordoneauthor=M.+Bandellauthor=B.+Laffitteauthor=Y.+Xuauthor=S.+Chienauthor=D.+Henrionauthor=A.+Patapoutian&title=GPR68+Senses+Flow+and+Is+Essential+for+Vascular+Physiology&doi=10.1016%2Fj.cell.2018.03.076"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.03.076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.03.076%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DMathur%26aufirst%3DJ.%26aulast%3DVessi%25C3%25A8res%26aufirst%3DE.%26aulast%3DHammack%26aufirst%3DS.%26aulast%3DNonomura%26aufirst%3DK.%26aulast%3DFavre%26aufirst%3DJ.%26aulast%3DGrimaud%26aufirst%3DL.%26aulast%3DPetrus%26aufirst%3DM.%26aulast%3DFrancisco%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DV.%26aulast%3DXiang%26aufirst%3DF.-L.%26aulast%3DMainquist%26aufirst%3DJ.%2BK.%26aulast%3DCahalan%26aufirst%3DS.%2BM.%26aulast%3DOrth%26aufirst%3DA.%2BP.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DLukacs%26aufirst%3DV.%26aulast%3DBordone%26aufirst%3DL.%26aulast%3DBandell%26aufirst%3DM.%26aulast%3DLaffitte%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DChien%26aufirst%3DS.%26aulast%3DHenrion%26aufirst%3DD.%26aulast%3DPatapoutian%26aufirst%3DA.%26atitle%3DGPR68%2520Senses%2520Flow%2520and%2520Is%2520Essential%2520for%2520Vascular%2520Physiology%26jtitle%3DCell%26date%3D2018%26volume%3D173%26spage%3D762.e16%26epage%3D775.e16%26doi%3D10.1016%2Fj.cell.2018.03.076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span> <i>Illuminating the Druggable Genome | NIH Common
Fund</i>, <span class="NLM_year">2019</span>. <a href="https://www.commonfund.nih.gov/idg/" class="extLink">https://www.commonfund.nih.gov/idg/</a> (accessed May 7, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Illuminating+the+Druggable+Genome+%7C+NIH+Common%0AFund%2C+2019.+https%3A%2F%2Fwww.commonfund.nih.gov%2Fidg%2F+%28accessed+May+7%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DIlluminating%2520the%2520Druggable%2520Genome%2520%257C%2520NIH%2520Common%250AFund%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiley, S. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sriram, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmerón, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insel, P. A.</span></span> <span> </span><span class="NLM_article-title">GPR68: An Emerging Drug Target in Cancer</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>, <span class="NLM_elocation-id">559</span> <span class="refDoi"> DOI: 10.3390/ijms20030559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.3390%2Fijms20030559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlSrs7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&author=S.+Z.+Wileyauthor=K.+Sriramauthor=C.+Salmer%C3%B3nauthor=P.+A.+Insel&title=GPR68%3A+An+Emerging+Drug+Target+in+Cancer&doi=10.3390%2Fijms20030559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">GPR68: an emerging drug target in cancer</span></div><div class="casAuthors">Wiley, Shu Z.; Sriram, Krishna; Salmeron, Cristina; Insel, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">559</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  GPR68 (or ovarian cancer G protein-coupled receptor 1, OGR1) is a proton-sensing G-protein-coupled receptor (GPCR) that responds to extracellular acidity and regulates a variety of cellular functions.  Acidosis is considered a defining hallmark of the tumor microenvironment (TME).  GPR68 expression is highly upregulated in numerous types of cancer.  Emerging evidence has revealed that GPR68 may play crucial roles in tumor biol., including tumorigenesis, tumor growth, and metastasis.  This review summarizes current knowledge regarding GPR68-its expression, regulation, signaling pathways, physiol. roles, and functions it regulates in human cancers (including prostate, colon and pancreatic cancer, melanoma, medulloblastoma, and myelodysplastic syndrome).  The findings provide evidence for GPR68 as a potentially novel therapeutic target but in addn., we note challenges in developing drugs that target GPR68.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo73vTsxafymbVg90H21EOLACvtfcHk0liTnPwGmcZiVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlSrs7zI&md5=c0c4080c5c2dfd0f9c32c3fa8bd74702</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3390%2Fijms20030559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20030559%26sid%3Dliteratum%253Aachs%26aulast%3DWiley%26aufirst%3DS.%2BZ.%26aulast%3DSriram%26aufirst%3DK.%26aulast%3DSalmer%25C3%25B3n%26aufirst%3DC.%26aulast%3DInsel%26aufirst%3DP.%2BA.%26atitle%3DGPR68%253A%2520An%2520Emerging%2520Drug%2520Target%2520in%2520Cancer%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26doi%3D10.3390%2Fijms20030559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Allosteric Modulators of G-Protein-Coupled Receptors: Drug-Target Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01844</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01844" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFCgtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=24-45&author=S.+Luauthor=J.+Zhang&title=Small+Molecule+Allosteric+Modulators+of+G-Protein-Coupled+Receptors%3A+Drug-Target+Interactions&doi=10.1021%2Facs.jmedchem.7b01844"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Allosteric Modulators of G-Protein-Coupled Receptors: Drug-Target Interactions</span></div><div class="casAuthors">Lu, Shaoyong; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-45</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are the largest class of signaling receptors that are most frequently targeted by therapeutic drugs.  Allosteric modulators bound to GPCRs at allosteric sites provide the potential for differential selectivity and improved safety compared with traditional orthosteric ligands.  The recent breakthroughs in GPCR structural biol. have made structures of GPCRs from classes A, B, C, and F complexed with small-mol. allosteric modulators available.  Knowledge of the detailed receptor-modulator interactions at the allosteric sites is useful for structure-based GPCR drug design of novel therapeutics.  This Perspective comprehensively summarizes the current status of structural complexes between GPCRs and their small-mol. allosteric modulators, particularly the key receptor-modulator interactions at the allosteric sites.  Then, the structural diversity of allosteric sites across four GPCR subfamilies is compared.  This study is expected to contribute to the design of GPCR allosteric drugs with an improved therapeutic action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVDf1DtcU6ZLVg90H21EOLACvtfcHk0lgoXyp7y-Oxlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFCgtb0%253D&md5=7a41f4c00dadecd0661d135811644ad8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01844%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DSmall%2520Molecule%2520Allosteric%2520Modulators%2520of%2520G-Protein-Coupled%2520Receptors%253A%2520Drug-Target%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D24%26epage%3D45%26doi%3D10.1021%2Facs.jmedchem.7b01844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coughlin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopper, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanco, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirunagaru, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robichaud, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doller, D.</span></span> <span> </span><span class="NLM_article-title">Allosteric Modalities for Membrane-Bound Receptors: Insights from Drug Hunting for Brain Diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">5979</span>– <span class="NLM_lpage">6002</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01651</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01651" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisFyhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=5979-6002&author=Q.+Coughlinauthor=A.+T.+Hopperauthor=M.-J.+Blancoauthor=V.+Tirunagaruauthor=A.+J.+Robichaudauthor=D.+Doller&title=Allosteric+Modalities+for+Membrane-Bound+Receptors%3A+Insights+from+Drug+Hunting+for+Brain+Diseases&doi=10.1021%2Facs.jmedchem.8b01651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Modalities for Membrane-Bound Receptors: Insights from Drug Hunting for Brain Diseases</span></div><div class="casAuthors">Coughlin, Quinn; Hopper, Allen T.; Blanco, Maria-Jesus; Tirunagaru, Vijaya; Robichaud, Albert J.; Doller, Dario</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5979-6002</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Medicinal chemists are accountable for embedding the appropriate drug target profile into the mol. architecture of a clin. candidate.  An accurate characterization of the functional effects following binding of a drug to its biol. target is a fundamental step in the discovery of new medicines, informing the translation of preclin. efficacy and safety observations into human trials.  Membrane-bound proteins, particularly ion channels and G protein-coupled receptors (GPCRs), are biol. targets prone to allosteric modulation.  Investigations using allosteric drug candidates and chem. tools suggest that their functional effects may be tailored with a high degree of translational alignment, making them mol. tools to correct pathophysiol. functional tone and enable personalized medicine when a causative target-to-disease link is known.  We present select examples of functional mol. fine-tuning of allosterism and discuss consequences relevant to drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG7wCMbhExHrVg90H21EOLACvtfcHk0lgoXyp7y-Oxlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisFyhs70%253D&md5=38fbd2cbc62615804f1f812b27fae13f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01651%26sid%3Dliteratum%253Aachs%26aulast%3DCoughlin%26aufirst%3DQ.%26aulast%3DHopper%26aufirst%3DA.%2BT.%26aulast%3DBlanco%26aufirst%3DM.-J.%26aulast%3DTirunagaru%26aufirst%3DV.%26aulast%3DRobichaud%26aufirst%3DA.%2BJ.%26aulast%3DDoller%26aufirst%3DD.%26atitle%3DAllosteric%2520Modalities%2520for%2520Membrane-Bound%2520Receptors%253A%2520Insights%2520from%2520Drug%2520Hunting%2520for%2520Brain%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26spage%3D5979%26epage%3D6002%26doi%3D10.1021%2Facs.jmedchem.8b01651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T.</span></span> <span> </span><span class="NLM_article-title">New Concepts in Drug Discovery: Collateral Efficacy and Permissive Antagonism</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">919</span>– <span class="NLM_lpage">927</span>, <span class="refDoi"> DOI: 10.1038/nrd1875</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fnrd1875" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=16264435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=919-927&author=T.+Kenakin&title=New+Concepts+in+Drug+Discovery%3A+Collateral+Efficacy+and+Permissive+Antagonism&doi=10.1038%2Fnrd1875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">New Concepts in Drug Discovery: Collateral Efficacy and Permissive Antagonism</span></div><div class="casAuthors">Kenakin, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">919-927</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  New perspectives on the complexity of G-protein-coupled receptor (GPCR) signaling and the increased resoln. of existing tools for studying GPCR behavior has led to the conception of new hypotheses that affect the discovery of drugs acting at GPCRs.  Taking into consideration the novel concepts of collateral efficacy and permissive antagonism in the search for synthetic agonists and antagonists, resp., will be essential in the search for drugs with unique therapeutic profiles.  Here, the design of drugs against HIV is used as an example of how these concepts might be taken into consideration for GPCR-targeted drugs in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp26TMWCy0fUrVg90H21EOLACvtfcHk0lgoXyp7y-Oxlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsLfO&md5=b7e09bcb4f73d52b4ab80f521ab78a64</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnrd1875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1875%26sid%3Dliteratum%253Aachs%26aulast%3DKenakin%26aufirst%3DT.%26atitle%3DNew%2520Concepts%2520in%2520Drug%2520Discovery%253A%2520Collateral%2520Efficacy%2520and%2520Permissive%2520Antagonism%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2005%26volume%3D4%26spage%3D919%26epage%3D927%26doi%3D10.1038%2Fnrd1875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ehlert, F. J.</span></span> <span> </span><span class="NLM_article-title">Analysis of Allosterism in Functional Assays</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>315</i></span>,  <span class="NLM_fpage">740</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1124/jpet.105.090886</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fjpet.105.090886" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=16046613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFyhtr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=315&publication_year=2005&pages=740-754&author=F.+J.+Ehlert&title=Analysis+of+Allosterism+in+Functional+Assays&doi=10.1124%2Fjpet.105.090886"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of allosterism in functional assays</span></div><div class="casAuthors">Ehlert, Frederick J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">315</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">740-754</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The theor. basis for analyzing the effects of an allosteric modulator on the response to an agonist is described.  The effects of an allosteric modulator on the concn.-response curve to an agonist can be attributed to changes in the obsd. dissocn. const. and intrinsic efficacy of the agonist-receptor complex.  Each of these two changes can be represented by a coeff. or factor.  It is possible to est. the ratio of the coeff. of change in agonist efficacy divided by that for the agonist dissocn. const.  This ratio is designated as the relative activity (RA) of the agonist in the presence of the allosteric modulator.  The RA value can be estd. for each concn. of allosteric modulator by nonlinear regression anal., regardless of the shape of the concn.-response curve.  Regression anal. of the RA values against the concn. of allosteric modulator yields ests. of the dissocn. const. (KA) of the allosteric modulator and the maximal RA value.  If the concn.-response curve to the agonist obeys a logistic function and the allosteric modulator influences the maximal response, it is possible to distinguish between the maximal change in affinity from that of efficacy.  If the agonist concn.-response curve obeys a logistic equation with a Hill slope of 1, the RA values can be estd. easily from the agonist EC50 and Emax values.  This anal. illustrates the utility of the RA value in quantifying allosteric effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKvK9-uIW607Vg90H21EOLACvtfcHk0lhUOedPRJqDIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFyhtr%252FO&md5=0cb5a02f65f93990511d12fb06f5da12</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1124%2Fjpet.105.090886&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.105.090886%26sid%3Dliteratum%253Aachs%26aulast%3DEhlert%26aufirst%3DF.%2BJ.%26atitle%3DAnalysis%2520of%2520Allosterism%2520in%2520Functional%2520Assays%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2005%26volume%3D315%26spage%3D740%26epage%3D754%26doi%3D10.1124%2Fjpet.105.090886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntosh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marrs, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pertwee, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, R. A.</span></span> <span> </span><span class="NLM_article-title">Allosteric Modulation of the Cannabinoid CB1 Receptor</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">1484</span>– <span class="NLM_lpage">1495</span>, <span class="refDoi"> DOI: 10.1124/mol.105.016162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fmol.105.016162" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=16113085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtF2it7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2005&pages=1484-1495&author=M.+R.+Priceauthor=G.+L.+Baillieauthor=A.+Thomasauthor=L.+A.+Stevensonauthor=M.+Eassonauthor=R.+Goodwinauthor=A.+McLeanauthor=L.+McIntoshauthor=G.+Goodwinauthor=G.+Walkerauthor=P.+Westwoodauthor=J.+Marrsauthor=F.+Thomsonauthor=P.+Cowleyauthor=A.+Christopoulosauthor=R.+G.+Pertweeauthor=R.+A.+Ross&title=Allosteric+Modulation+of+the+Cannabinoid+CB1+Receptor&doi=10.1124%2Fmol.105.016162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric modulation of the cannabinoid CB1 receptor</span></div><div class="casAuthors">Price, Martin R.; Baillie, Gemma L.; Thomas, Adele; Stevenson, Lesley A.; Easson, Morag; Goodwin, Richard; McLean, Adele; McIntosh, Lorraine; Goodwin, Gillian; Walker, Glenn; Westwood, Paul; Marrs, Julia; Thomson, Fiona; Cowley, Phillip; Christopoulos, Arthur; Pertwee, Roger G.; Ross, Ruth A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1484-1495</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The authors investigated the pharmacol. of three novel compds., Org 27569, Org 27759 and Org 29647 at the cannabinoid CB1 receptor.  In equil. binding assays, the Org compds. significantly increased the binding of the CB1 receptor agonist [3H]CP 55,940 [(1R,3R,4R)-3-[2-hydroxy-4-(1,1-dimethylheptyl)phenyl]-4-(3-hydroxypropyl) cyclohexan-1-ol], indicative of a pos. cooperative allosteric effect.  The same compds. caused a significant, but incomplete, decrease in the specific binding of the CB1 receptor inverse agonist [3H]SR 141716A [N-(piperidin-1-yl)-5-(4-chlorophenyl)-1-(2,4-dichlorophenyl)-4-methyl-1H -pyrazole-3-carboximide hydrochloride], indicative of a limited neg. binding cooperativity.  Anal. of the data according to an allosteric ternary complex model revealed that the estd. affinity of each Org compd. was not significantly different when the radioligand was [3H]CP 55,940 or [3H]SR 141716A.  However, the estd. cooperatively factor for the interaction between modulator and radioligand was greater than 1 when detd. against [3H]CP 55,940 and less than 1 when detd. against [3H]SR 141716A.  [3H]CP 55,940 dissocn. kinetic studies also validated the allosteric nature of the Org compds., because they all significantly decreased radioligand dissocn.  These data suggest that the Org compds. bind allosterically to the CB1 receptor and elicit a conformational change that increases agonist affinity for the orthosteric binding site.  In contrast to the binding assays, however, the Org compds. behaved as insurmountable antagonists of receptor function; in the reporter gene assay, the guanosine 5'-O-(3-[35S]thio)triphosphate binding assay and the mouse vas deferens assay they elicited a significant redn. in the Emax value for CB1 receptor agonists.  The data presented clearly demonstrate, for the first time, that the cannabinoid CB1 receptor contains an allosteric binding site that can be recognized by synthetic small mol. ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq98nUa2ffQ7bVg90H21EOLACvtfcHk0lhUOedPRJqDIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtF2it7nM&md5=a342d304a8157b1e7b962755a775154d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1124%2Fmol.105.016162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.105.016162%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DM.%2BR.%26aulast%3DBaillie%26aufirst%3DG.%2BL.%26aulast%3DThomas%26aufirst%3DA.%26aulast%3DStevenson%26aufirst%3DL.%2BA.%26aulast%3DEasson%26aufirst%3DM.%26aulast%3DGoodwin%26aufirst%3DR.%26aulast%3DMcLean%26aufirst%3DA.%26aulast%3DMcIntosh%26aufirst%3DL.%26aulast%3DGoodwin%26aufirst%3DG.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DWestwood%26aufirst%3DP.%26aulast%3DMarrs%26aufirst%3DJ.%26aulast%3DThomson%26aufirst%3DF.%26aulast%3DCowley%26aufirst%3DP.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DPertwee%26aufirst%3DR.%2BG.%26aulast%3DRoss%26aufirst%3DR.%2BA.%26atitle%3DAllosteric%2520Modulation%2520of%2520the%2520Cannabinoid%2520CB1%2520Receptor%26jtitle%3DMol.%2520Pharmacol.%26date%3D2005%26volume%3D68%26spage%3D1484%26epage%3D1495%26doi%3D10.1124%2Fmol.105.016162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuano, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Pharmacological Profiling of Analogues of Benzyl Quinolone Carboxylic Acid (BQCA) as Allosteric Modulators of the M1 Muscarinic Receptor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5151</span>– <span class="NLM_lpage">5172</span>, <span class="refDoi"> DOI: 10.1021/jm400540b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400540b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVaqtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=5151-5172&author=S.+N.+Mistryauthor=C.+Valantauthor=P.+M.+Sextonauthor=B.+Capuanoauthor=A.+Christopoulosauthor=P.+J.+Scammells&title=Synthesis+and+Pharmacological+Profiling+of+Analogues+of+Benzyl+Quinolone+Carboxylic+Acid+%28BQCA%29+as+Allosteric+Modulators+of+the+M1+Muscarinic+Receptor&doi=10.1021%2Fjm400540b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Pharmacological Profiling of Analogues of Benzyl Quinolone Carboxylic Acid (BQCA) as Allosteric Modulators of the M1 Muscarinic Receptor</span></div><div class="casAuthors">Mistry, Shailesh N.; Valant, Celine; Sexton, Patrick M.; Capuano, Ben; Christopoulos, Arthur; Scammells, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5151-5172</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Established therapy in Alzheimer's disease involves potentiation of the endogenous orthosteric ligand, acetylcholine, at the M1 muscarinic receptors found in higher concns. in the cortex and hippocampus.  Adverse effects, due to indiscriminate activation of other muscarinic receptor subtypes, are common.  M1 muscarinic pos. allosteric modulators/allosteric agonists such as BQCA [1-(4-methoxybenzyl)-4-oxo-1,4-dihydroquinoline-3-carboxylic acid, I] offer an attractive soln., being exquisitely M1-selective over other muscarinic subtypes.  A common difficulty with allosteric ligands is interpreting SAR, based on composite potency values derived in the presence of fixed concn. of agonist.  In reality these values encompass multiple pharmacol. parameters, each potentially and differentially sensitive to structural modification of the ligand.  We report novel BQCA analogs which appear to augment ligand affinity for the receptor (pKB), intrinsic efficacy (τB), and both binding (α) and functional (β) cooperativity with acetylcholine.  Ultimately, development of such enriched SAR surrounding allosteric modulators will provide insight into their mode of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoigST7OJR7vbVg90H21EOLACvtfcHk0lhUOedPRJqDIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVaqtbs%253D&md5=ab134e0b5400aa825916695eaf1203b1</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm400540b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400540b%26sid%3Dliteratum%253Aachs%26aulast%3DMistry%26aufirst%3DS.%2BN.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DCapuano%26aufirst%3DB.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26atitle%3DSynthesis%2520and%2520Pharmacological%2520Profiling%2520of%2520Analogues%2520of%2520Benzyl%2520Quinolone%2520Carboxylic%2520Acid%2520%2528BQCA%2529%2520as%2520Allosteric%2520Modulators%2520of%2520the%2520M1%2520Muscarinic%2520Receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D5151%26epage%3D5172%26doi%3D10.1021%2Fjm400540b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourgognon, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanger, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verity, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogg, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butcher, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molloy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macedo-Hatch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawlak, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broad, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinert, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallucci, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobin, A. B.</span></span> <span> </span><span class="NLM_article-title">M1 Muscarinic Allosteric Modulators Slow Prion Neurodegeneration and Restore Memory Loss</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">487</span>– <span class="NLM_lpage">499</span>, <span class="refDoi"> DOI: 10.1172/JCI87526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1172%2FJCI87526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=27991860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BC1c%252FmtF2hug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=487-499&author=S.+J.+Bradleyauthor=J.-M.+Bourgognonauthor=H.+E.+Sangerauthor=N.+Verityauthor=A.+J.+Moggauthor=D.+J.+Whiteauthor=A.+J.+Butcherauthor=J.+A.+Morenoauthor=C.+Molloyauthor=T.+Macedo-Hatchauthor=J.+M.+Edwardsauthor=J.+Wessauthor=R.+Pawlakauthor=D.+J.+Readauthor=P.+M.+Sextonauthor=L.+M.+Broadauthor=J.+R.+Steinertauthor=G.+R.+Mallucciauthor=A.+Christopoulosauthor=C.+C.+Felderauthor=A.+B.+Tobin&title=M1+Muscarinic+Allosteric+Modulators+Slow+Prion+Neurodegeneration+and+Restore+Memory+Loss&doi=10.1172%2FJCI87526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">M1 muscarinic allosteric modulators slow prion neurodegeneration and restore memory loss</span></div><div class="casAuthors">Bradley Sophie J; Bourgognon Julie-Myrtille; Sanger Helen E; Verity Nicholas; Mogg Adrian J; White David J; Butcher Adrian J; Moreno Julie A; Molloy Colin; Macedo-Hatch Timothy; Edwards Jennifer M; Wess Jurgen; Pawlak Robert; Read David J; Sexton Patrick M; Broad Lisa M; Steinert Joern R; Mallucci Giovanna R; Christopoulos Arthur; Felder Christian C; Tobin Andrew B</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">487-499</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The current frontline symptomatic treatment for Alzheimer's disease (AD) is whole-body upregulation of cholinergic transmission via inhibition of acetylcholinesterase.  This approach leads to profound dose-related adverse effects.  An alternative strategy is to selectively target muscarinic acetylcholine receptors, particularly the M1 muscarinic acetylcholine receptor (M1 mAChR), which was previously shown to have procognitive activity.  However, developing M1 mAChR-selective orthosteric ligands has proven challenging.  Here, we have shown that mouse prion disease shows many of the hallmarks of human AD, including progressive terminal neurodegeneration and memory deficits due to a disruption of hippocampal cholinergic innervation.  The fact that we also show that muscarinic signaling is maintained in both AD and mouse prion disease points to the latter as an excellent model for testing the efficacy of muscarinic pharmacological entities.  The memory deficits we observed in mouse prion disease were completely restored by treatment with benzyl quinolone carboxylic acid (BQCA) and benzoquinazoline-12 (BQZ-12), two highly selective positive allosteric modulators (PAMs) of M1 mAChRs.  Furthermore, prolonged exposure to BQCA markedly extended the lifespan of diseased mice.  Thus, enhancing hippocampal muscarinic signaling using M1 mAChR PAMs restored memory loss and slowed the progression of mouse prion disease, indicating that this ligand type may have clinical benefit in diseases showing defective cholinergic transmission, such as AD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ6VBjSjCSsKp0d-O_J5C0AfW6udTcc2eYsYFv0IbiM0Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c%252FmtF2hug%253D%253D&md5=b8384618de37b3f83676ea77911a7625</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1172%2FJCI87526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI87526%26sid%3Dliteratum%253Aachs%26aulast%3DBradley%26aufirst%3DS.%2BJ.%26aulast%3DBourgognon%26aufirst%3DJ.-M.%26aulast%3DSanger%26aufirst%3DH.%2BE.%26aulast%3DVerity%26aufirst%3DN.%26aulast%3DMogg%26aufirst%3DA.%2BJ.%26aulast%3DWhite%26aufirst%3DD.%2BJ.%26aulast%3DButcher%26aufirst%3DA.%2BJ.%26aulast%3DMoreno%26aufirst%3DJ.%2BA.%26aulast%3DMolloy%26aufirst%3DC.%26aulast%3DMacedo-Hatch%26aufirst%3DT.%26aulast%3DEdwards%26aufirst%3DJ.%2BM.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DPawlak%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DD.%2BJ.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DBroad%26aufirst%3DL.%2BM.%26aulast%3DSteinert%26aufirst%3DJ.%2BR.%26aulast%3DMallucci%26aufirst%3DG.%2BR.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DFelder%26aufirst%3DC.%2BC.%26aulast%3DTobin%26aufirst%3DA.%2BB.%26atitle%3DM1%2520Muscarinic%2520Allosteric%2520Modulators%2520Slow%2520Prion%2520Neurodegeneration%2520and%2520Restore%2520Memory%2520Loss%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2017%26volume%3D127%26spage%3D487%26epage%3D499%26doi%3D10.1172%2FJCI87526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T.</span></span> <span> </span><span class="NLM_article-title">A Scale of Agonism and Allosteric Modulation for Assessment of Selectivity, Bias, and Receptor Mutation</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">414</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1124/mol.117.108787</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fmol.117.108787" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=28679508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVyiurzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2017&pages=414-424&author=T.+Kenakin&title=A+Scale+of+Agonism+and+Allosteric+Modulation+for+Assessment+of+Selectivity%2C+Bias%2C+and+Receptor+Mutation&doi=10.1124%2Fmol.117.108787"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A scale of agonism and allosteric modulation for assessment of selectivity, bias, and receptor mutation</span></div><div class="casAuthors">Kenakin, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">414-424</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">An index of agonism is described that can be used to quantify agonist receptor selectivity, bias, cell-based agonism, and the effects of receptor mutation on signaling.  The parameter is derived from agonist concn.-response curves and comprises the maximal response to the agonist (max) and the EC50 in the form of Δlog(max/EC50).  This parameter is derived from equations describing agonists as pos. allosteric facilitators of receptor-signaling protein interaction.  A similar index is also derived to quantify the potentiating effects of pos. allosteric modulators, which can be used to quantify in situ pos. allosteric modulator activity in vivo.  These indexes lend themselves to statistical anal. and are system-independent in that the effects of the system processing of agonist response and differences in assay sensitivity and receptor expression are cancelled.  The various applications of the Δlog(max/EC50) scale are described for each pharmacol. application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5mhzE_5MqrLVg90H21EOLACvtfcHk0lhB_dmwxujFcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVyiurzP&md5=f1e35110d4347fe685cb776be30b5d79</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1124%2Fmol.117.108787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.117.108787%26sid%3Dliteratum%253Aachs%26aulast%3DKenakin%26aufirst%3DT.%26atitle%3DA%2520Scale%2520of%2520Agonism%2520and%2520Allosteric%2520Modulation%2520for%2520Assessment%2520of%2520Selectivity%252C%2520Bias%252C%2520and%2520Receptor%2520Mutation%26jtitle%3DMol.%2520Pharmacol.%26date%3D2017%26volume%3D92%26spage%3D414%26epage%3D424%26doi%3D10.1124%2Fmol.117.108787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jörg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinh, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuano, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span> <span> </span><span class="NLM_article-title">4-Phenylpyridin-2-one Derivatives: A Novel Class of Positive Allosteric Modulator of the M1 Muscarinic Acetylcholine Receptor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">388</span>– <span class="NLM_lpage">409</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01562</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01562" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFegu7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=388-409&author=S.+N.+Mistryauthor=M.+J%C3%B6rgauthor=H.+Limauthor=N.+B.+Vinhauthor=P.+M.+Sextonauthor=B.+Capuanoauthor=A.+Christopoulosauthor=J.+R.+Laneauthor=P.+J.+Scammells&title=4-Phenylpyridin-2-one+Derivatives%3A+A+Novel+Class+of+Positive+Allosteric+Modulator+of+the+M1+Muscarinic+Acetylcholine+Receptor&doi=10.1021%2Facs.jmedchem.5b01562"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">4-Phenylpyridin-2-one Derivatives: A Novel Class of Positive Allosteric Modulator of the M1 Muscarinic Acetylcholine Receptor</span></div><div class="casAuthors">Mistry, Shailesh N.; Jorg, Manuela; Lim, Herman; Vinh, Natalie B.; Sexton, Patrick M.; Capuano, Ben; Christopoulos, Arthur; Lane, J. Robert; Scammells, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">388-409</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pos. allosteric modulators (PAMs) of the M1 muscarinic acetylcholine receptor (M1 mAChR) are a promising strategy for the treatment of the cognitive deficits assocd. with diseases including Alzheimer's and schizophrenia.  Herein, we report the design, synthesis, and characterization of a novel family of M1 mAChR PAMs.  The most active compds. of the 4-phenylpyridin-2-one series exhibited comparable binding affinity to the ref. compd., 1-(4-methoxybenzyl)-4-oxo-1,4-dihydroquinoline-3-carboxylic acid (BQCA), but markedly improved pos. cooperativity with acetylcholine, and retained exquisite selectivity for the M1 mAChR.  Furthermore, the pharmacol. characterization revealed ligands with a diverse range of activities, including modulators that displayed both high intrinsic efficacy and PAM activity, those that showed no detectable agonism but robust PAM activity and ligands that displayed robust allosteric agonism but little modulatory activity.  Compd. I was found to have the best pharmacol. profile.  Thus, the 4-phenylpyridin-2-one scaffold offers an attractive starting point for further lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMlFmtgndXrbVg90H21EOLACvtfcHk0lhB_dmwxujFcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFegu7zP&md5=315362c43e2b6c204f038f30c6ade642</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01562&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01562%26sid%3Dliteratum%253Aachs%26aulast%3DMistry%26aufirst%3DS.%2BN.%26aulast%3DJ%25C3%25B6rg%26aufirst%3DM.%26aulast%3DLim%26aufirst%3DH.%26aulast%3DVinh%26aufirst%3DN.%2BB.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DCapuano%26aufirst%3DB.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26atitle%3D4-Phenylpyridin-2-one%2520Derivatives%253A%2520A%2520Novel%2520Class%2520of%2520Positive%2520Allosteric%2520Modulator%2520of%2520the%2520M1%2520Muscarinic%2520Acetylcholine%2520Receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D388%26epage%3D409%26doi%3D10.1021%2Facs.jmedchem.5b01562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Russell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetsch, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Rooij, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frantz, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, J. W.</span></span> <span> </span><span class="NLM_article-title">Regulated Expression of pH Sensing G Protein-Coupled Receptor-68 Identified through Chemical Biology Defines a New Drug Target for Ischemic Heart Disease</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1077</span>– <span class="NLM_lpage">1083</span>, <span class="refDoi"> DOI: 10.1021/cb300001m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300001m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVWls7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1077-1083&author=J.+L.+Russellauthor=S.+C.+Goetschauthor=H.+R.+Aguilarauthor=H.+Coeauthor=X.+Luoauthor=N.+Liuauthor=E.+van+Rooijauthor=D.+E.+Frantzauthor=J.+W.+Schneider&title=Regulated+Expression+of+pH+Sensing+G+Protein-Coupled+Receptor-68+Identified+through+Chemical+Biology+Defines+a+New+Drug+Target+for+Ischemic+Heart+Disease&doi=10.1021%2Fcb300001m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Regulated Expression of pH Sensing G Protein-Coupled Receptor-68 Identified through Chemical Biology Defines a New Drug Target for Ischemic Heart Disease</span></div><div class="casAuthors">Russell, Jamie L.; Goetsch, Sean C.; Aguilar, Hector R.; Coe, Helen; Luo, Xiang; Liu, Ning; van Rooij, Eva; Frantz, Doug E.; Schneider, Jay W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1077-1083</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. biol. promises discovery of new and unexpected mechanistic pathways, protein functions and disease targets.  Here, we probed the mechanism-of-action and protein targets of 3,5-disubstituted isoxazoles (Isx), cardiomyogenic small mols. that target Notch-activated epicardium-derived cells (NECs) in vivo and promote functional recovery after myocardial infarction (MI).  Mechanistic studies in NECs led to an Isx-activated Gq protein-coupled receptor (GqPCR) hypothesis tested in a cell-based functional target screen for GPCRs regulated by Isx.  This screen identified one agonist hit, the extracellular proton/pH-sensing GPCR GPR68, confirmed through genetic gain- and loss-of-function.  Overlooked until now, GPR68 expression and localization were highly regulated in early post-natal and adult post-infarct mouse heart, where GPR68-expressing cells accumulated subepicardially.  Remarkably, GPR68-expressing cardiomyocytes established a proton-sensing cellular buffer zone surrounding the MI.  Isx pharmacol. regulated gene expression (mRNAs and miRs) in this GPR68-enriched border zone, driving cardiomyogenic and pro-survival transcriptional programs in vivo.  In conclusion, we tracked a (micromolar) bioactive small mol. mechanism-of-action to a candidate target protein, GPR68, and validated this target as a previously unrecognized regulator of myocardial cellular responses to tissue acidosis, setting the stage for future (nanomolar) target-based drug lead discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQLtpNAH6aiLVg90H21EOLACvtfcHk0lhTA6Bu4mJWmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVWls7o%253D&md5=751e2e1833518df9da9253f0516ba4e5</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fcb300001m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300001m%26sid%3Dliteratum%253Aachs%26aulast%3DRussell%26aufirst%3DJ.%2BL.%26aulast%3DGoetsch%26aufirst%3DS.%2BC.%26aulast%3DAguilar%26aufirst%3DH.%2BR.%26aulast%3DCoe%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3Dvan%2BRooij%26aufirst%3DE.%26aulast%3DFrantz%26aufirst%3DD.%2BE.%26aulast%3DSchneider%26aufirst%3DJ.%2BW.%26atitle%3DRegulated%2520Expression%2520of%2520pH%2520Sensing%2520G%2520Protein-Coupled%2520Receptor-68%2520Identified%2520through%2520Chemical%2520Biology%2520Defines%2520a%2520New%2520Drug%2520Target%2520for%2520Ischemic%2520Heart%2520Disease%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1077%26epage%3D1083%26doi%3D10.1021%2Fcb300001m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mogi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakakura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span> <span> </span><span class="NLM_article-title">Role of Extracellular Proton-Sensing OGR1 in Regulation of Insulin Secretion and Pancreatic to Cell Functions</span>. <i>Endocr. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">101</span>, <span class="refDoi"> DOI: 10.1507/endocrj.EJ13-0380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1507%2Fendocrj.EJ13-0380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=24088601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFWqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2014&pages=101&author=C.+Mogiauthor=T.+Nakakuraauthor=F.+Okajima&title=Role+of+Extracellular+Proton-Sensing+OGR1+in+Regulation+of+Insulin+Secretion+and+Pancreatic+to+Cell+Functions&doi=10.1507%2Fendocrj.EJ13-0380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Role of extracellular proton-sensing OGR1 in regulation of insulin secretion and pancreatic β-cell functions</span></div><div class="casAuthors">Mogi, Chihiro; Nakakura, Takashi; Okajima, Fumikazu</div><div class="citationInfo"><span class="NLM_cas:title">Endocrine Journal (Kyoto, Japan)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-110</span>CODEN:
                <span class="NLM_cas:coden">ENJOEO</span>;
        ISSN:<span class="NLM_cas:issn">0918-8959</span>.
    
            (<span class="NLM_cas:orgname">Japan Endocrine Society</span>)
        </div><div class="casAbstract">A review.  Insulin secretion with respect to pH environments has been investigated for a long time but its mechanism remains largely unknown.  Extracellular pH is usually maintained at around 7.4 and, its change has been thought to occur in non-physiol. situations.  Acidification takes place under ischemic and inflammatory microenvironments, where stimulation of anaerobic glycolysis results in the prodn. of lactic acid.  In addn. to ionotropic ion channels, such as transient receptor potential V1 (TRPV1) and acid-sensing ion channels (ASICs), metabotropic proton-sensing G protein-coupled receptors (GPCRs) have also been identified recently as proton-sensing machineries.  While ionotropic ion channels usually sense strong acidic pH, proton-sensing GPCRs sense pH of 7.6 to 6.0 and have been shown to mediate a variety of biol. actions in neutral and mildly acidic pH environments.  Studies with receptor knockout mice have revealed that proton-sensing receptors, including ovarian cancer G protein-coupled receptor 1 (OGR1), a proton-sensing GPCRs, play a role in the regulation of insulin secretion and glucose metab. under physiol. conditions.  Small mol. 3,5-disubstituted isoxazoles have recently been identified as OGR1 agonists working at neutral pH and have been shown to stimulate pancreatic β-cell differentiation and insulin synthesis.  Thus, proton-sensing OGR1 may be an important player for insulin secretion and a potential target for improving β-cell function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2FjKKiqNrX7Vg90H21EOLACvtfcHk0lhTA6Bu4mJWmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFWqtb0%253D&md5=a59172fd753ef3455cf0a62415c19ede</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1507%2Fendocrj.EJ13-0380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1507%252Fendocrj.EJ13-0380%26sid%3Dliteratum%253Aachs%26aulast%3DMogi%26aufirst%3DC.%26aulast%3DNakakura%26aufirst%3DT.%26aulast%3DOkajima%26aufirst%3DF.%26atitle%3DRole%2520of%2520Extracellular%2520Proton-Sensing%2520OGR1%2520in%2520Regulation%2520of%2520Insulin%2520Secretion%2520and%2520Pancreatic%2520to%2520Cell%2520Functions%26jtitle%3DEndocr.%2520J.%26date%3D2014%26volume%3D61%26spage%3D101%26doi%3D10.1507%2Fendocrj.EJ13-0380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abe-Ohya, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiozawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, F.</span></span> <span> </span><span class="NLM_article-title">Identification of Metals from Osteoblastic ST-2 Cell Supernatants as Novel OGR1 Agonists</span>. <i>J. Recept. Signal Transduction</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">492</span>, <span class="refDoi"> DOI: 10.3109/10799893.2015.1015736</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.3109%2F10799893.2015.1015736" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=26053506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlSqurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=485-492&author=R.+Abe-Ohyaauthor=T.+Ishikawaauthor=H.+Shiozawaauthor=K.+Sudaauthor=F.+Nara&title=Identification+of+Metals+from+Osteoblastic+ST-2+Cell+Supernatants+as+Novel+OGR1+Agonists&doi=10.3109%2F10799893.2015.1015736"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of metals from osteoblastic ST-2 cell supernatants as novel OGR1 agonists</span></div><div class="casAuthors">Abe-Ohya, Rie; Ishikawa, Tomio; Shiozawa, Hideyuki; Suda, Koji; Nara, Futoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Receptors and Signal Transduction</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">485-492</span>CODEN:
                <span class="NLM_cas:coden">JRSTCT</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Ovarian cancer G-protein-coupled receptor 1 (OGR1) is a G-protein-coupled receptor (GPCR), which has previously been identified as a receptor for protons.  It has been reported in this and previous studies that OGR1 expression was markedly up-regulated during osteoclast differentiation.  We predicted the possibility of other mols. activating OGR1 in neutral pH, and that osteoblasts might release OGR1 agonistic mols. and activate OGR1 expressed in osteoclasts such as RANKL.  We screened for cell supernatants and organ exts. and discovered OGR1 agonistic activity in ST-2 osteoblastic cell supernatants and pancreatic tissues.  Finally, we partially purified and identified essential metals, Fe, Zn, Co, Ni and Mn, as novel OGR1 agonists.  These OGR1 agonistic metals induce intracellular Gq-coupled inositol phosphate signals in OGR1-expressing cells and primary osteoclasts through OGR1.  We also confirmed that these OGR1 agonistic metals activated OGR1 through the same residues which act with protons.  Here, we demonstrate that metals, Fe, Zn, Co, Ni and Mn are the novel OGR1 agonists, which can singly activate OGR1 in neutral pH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpen1-rQPSfgbVg90H21EOLACvtfcHk0lhTA6Bu4mJWmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlSqurk%253D&md5=eba16e9d6e1fa3c3b40f879425f7bf73</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.3109%2F10799893.2015.1015736&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10799893.2015.1015736%26sid%3Dliteratum%253Aachs%26aulast%3DAbe-Ohya%26aufirst%3DR.%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DShiozawa%26aufirst%3DH.%26aulast%3DSuda%26aufirst%3DK.%26aulast%3DNara%26aufirst%3DF.%26atitle%3DIdentification%2520of%2520Metals%2520from%2520Osteoblastic%2520ST-2%2520Cell%2520Supernatants%2520as%2520Novel%2520OGR1%2520Agonists%26jtitle%3DJ.%2520Recept.%2520Signal%2520Transduction%26date%3D2015%26volume%3D35%26spage%3D485%26epage%3D492%26doi%3D10.3109%2F10799893.2015.1015736" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Negishi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omori, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takanashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagayama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirayama, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishina, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakakura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochimaru, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomura, H.</span></span> <span> </span><span class="NLM_article-title">Manganese and Cobalt Activate Zebrafish Ovarian Cancer G-protein-Coupled Receptor 1 but Not GPR4</span>. <i>J. Recept. Signal Transduction</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">408</span>, <span class="refDoi"> DOI: 10.1080/10799893.2017.1298130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1080%2F10799893.2017.1298130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=28270026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvFyjtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&pages=401-408&author=J.+Negishiauthor=Y.+Omoriauthor=M.+Shindoauthor=H.+Takanashiauthor=S.+Mushaauthor=S.+Nagayamaauthor=J.+Hirayamaauthor=H.+Nishinaauthor=T.+Nakakuraauthor=C.+Mogiauthor=K.+Satoauthor=F.+Okajimaauthor=Y.+Mochimaruauthor=H.+Tomura&title=Manganese+and+Cobalt+Activate+Zebrafish+Ovarian+Cancer+G-protein-Coupled+Receptor+1+but+Not+GPR4&doi=10.1080%2F10799893.2017.1298130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Manganese and cobalt activate zebrafish ovarian cancer G-protein-coupled receptor 1 but not GPR4</span></div><div class="casAuthors">Negishi, Jun; Omori, Yuka; Shindo, Mami; Takanashi, Hayate; Musha, Shiori; Nagayama, Suminori; Hirayama, Jun; Nishina, Hiroshi; Nakakura, Takashi; Mogi, Chihiro; Sato, Koichi; Okajima, Fumikazu; Mochimaru, Yuta; Tomura, Hideaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Receptors and Signal Transduction</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">401-408</span>CODEN:
                <span class="NLM_cas:coden">JRSTCT</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Mammalian ovarian G-protein-coupled receptor 1 (OGR1) is activated by some metals in addn. to extracellular protons and coupling to multiple intracellular signaling pathways.  In the present study, we examd. whether zebrafish OGR1, zebrafish GPR4, and human GPR4 (zOGR1, zGPR4, and hGPR4, resp.) could sense the metals and activate the intracellular signaling pathways.  On one hand, we found that only manganese and cobalt of the tested metals stimulated SRE-promoter activities in zOGR1-overexpressed HEK293T cells.  On the other hand, none of the metals tested stimulated the promoter activities in zGPR4- and hGPR4-overexpressed cells.  The OGR1 mutant (H4F), which is lost to activation by extracellular protons, did not stimulate metal-induced SRE-promoter activities.  These results suggest that zOGR1, but not GPR4, is also a metal-sensing G-protein-coupled receptor in addn. to a proton-sensing G-protein-coupled receptor, although not all metals that activate hOGR1 activated zOGR1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgUPJELCG8T7Vg90H21EOLACvtfcHk0lhTA6Bu4mJWmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvFyjtLk%253D&md5=b4bb47d9fb82ef15ebfdc9c4f2c4bf02</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1080%2F10799893.2017.1298130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10799893.2017.1298130%26sid%3Dliteratum%253Aachs%26aulast%3DNegishi%26aufirst%3DJ.%26aulast%3DOmori%26aufirst%3DY.%26aulast%3DShindo%26aufirst%3DM.%26aulast%3DTakanashi%26aufirst%3DH.%26aulast%3DMusha%26aufirst%3DS.%26aulast%3DNagayama%26aufirst%3DS.%26aulast%3DHirayama%26aufirst%3DJ.%26aulast%3DNishina%26aufirst%3DH.%26aulast%3DNakakura%26aufirst%3DT.%26aulast%3DMogi%26aufirst%3DC.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DOkajima%26aufirst%3DF.%26aulast%3DMochimaru%26aufirst%3DY.%26aulast%3DTomura%26aufirst%3DH.%26atitle%3DManganese%2520and%2520Cobalt%2520Activate%2520Zebrafish%2520Ovarian%2520Cancer%2520G-protein-Coupled%2520Receptor%25201%2520but%2520Not%2520GPR4%26jtitle%3DJ.%2520Recept.%2520Signal%2520Transduction%26date%3D2017%26volume%3D37%26spage%3D401%26epage%3D408%26doi%3D10.1080%2F10799893.2017.1298130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mochimaru, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negishi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okajima, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomura, H.</span></span> <span> </span><span class="NLM_article-title">Metals Differentially Activate Ovarian Cancer G Protein-Coupled Receptor 1 in Various Species</span>. <i>Zool. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.2108/zs170145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.2108%2Fzs170145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=29623784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVCitbrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2018&pages=109-114&author=Y.+Mochimaruauthor=J.+Negishiauthor=S.+Murakamiauthor=S.+Mushaauthor=K.+Satoauthor=F.+Okajimaauthor=H.+Tomura&title=Metals+Differentially+Activate+Ovarian+Cancer+G+Protein-Coupled+Receptor+1+in+Various+Species&doi=10.2108%2Fzs170145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Metals differentially activate ovarian cancer G protein-coupled receptor 1 in various species</span></div><div class="casAuthors">Mochimaru, Yuta; Negishi, Jun; Murakami, Syo; Musha, Shiori; Sato, Koichi; Okajima, Fumikazu; Tomura, Hideaki</div><div class="citationInfo"><span class="NLM_cas:title">Zoological Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">109-114</span>CODEN:
                <span class="NLM_cas:coden">ZOSCEX</span>;
        ISSN:<span class="NLM_cas:issn">0289-0003</span>.
    
            (<span class="NLM_cas:orgname">Zoological Society of Japan</span>)
        </div><div class="casAbstract">Human, mouse, and zebrafish ovarian cancer G protein-coupled receptors (OGR1s) are activated by both metals and extracellular protons.  In the present study, we examd. whether pig, rat, chicken, and Xenopus OGR1 homologs could sense and be activated by protons and metals.  We found that all homologs stimulated serum response element (SRE)-driven promoter activities when they are stimulated by protons.  On the other hand, metals differentially activated the homologs.  The results using chimeric receptors of human and zebrafish OGR1s indicate that the specificity of the metal-induced activation lies in the extracellular region.  These results suggest that protons are an evolutionally conserved agonist of OGR1.  However, the types of metals that activated the receptor differed among the homologs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGc3vXJ0Zx0rVg90H21EOLACvtfcHk0ljNxDnfv7ehMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVCitbrK&md5=9e69ce405f2ddd7e49197f120d4ad2bd</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.2108%2Fzs170145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2108%252Fzs170145%26sid%3Dliteratum%253Aachs%26aulast%3DMochimaru%26aufirst%3DY.%26aulast%3DNegishi%26aufirst%3DJ.%26aulast%3DMurakami%26aufirst%3DS.%26aulast%3DMusha%26aufirst%3DS.%26aulast%3DSato%26aufirst%3DK.%26aulast%3DOkajima%26aufirst%3DF.%26aulast%3DTomura%26aufirst%3DH.%26atitle%3DMetals%2520Differentially%2520Activate%2520Ovarian%2520Cancer%2520G%2520Protein-Coupled%2520Receptor%25201%2520in%2520Various%2520Species%26jtitle%3DZool.%2520Sci.%26date%3D2018%26volume%3D35%26spage%3D109%26epage%3D114%26doi%3D10.2108%2Fzs170145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyazaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munechika, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamizono, A.</span></span> <span> </span><span class="NLM_article-title">Isolation of Zn2+ as an Endogenous Agonist of GPR39 from Fetal Bovine Serum</span>. <i>J. Recept. Signal Transduction</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">246</span>, <span class="refDoi"> DOI: 10.1080/10799890701506147</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1080%2F10799890701506147" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=17885920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVGrs7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=235-246&author=S.+Yasudaauthor=T.+Miyazakiauthor=K.+Munechikaauthor=M.+Yamashitaauthor=Y.+Ikedaauthor=A.+Kamizono&title=Isolation+of+Zn2%2B+as+an+Endogenous+Agonist+of+GPR39+from+Fetal+Bovine+Serum&doi=10.1080%2F10799890701506147"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of Zn2+ as an Endogenous Agonist of GPR39 from Fetal Bovine Serum</span></div><div class="casAuthors">Yasuda, Shin-Ichiro; Miyazaki, Takahiro; Munechika, Kouji; Yamashita, Masami; Ikeda, Yoshitaka; Kamizono, Akihito</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Receptors and Signal Transduction</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-246</span>CODEN:
                <span class="NLM_cas:coden">JRSTCT</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">The authors attempted to det. natural agonists of GPR39 in fetal bovine serum (FBS).  FBS was conditioned to ext. peptides and fractionated by two types of HPLC.  The activity of each fraction was monitored by intracellular calcium mobilization.  Then the purified active ingredient was analyzed by inductively coupled plasma mass spectrometry.  In this fashion, Zn2+ was identified as an agonist of GPR39, although no peptidergic mols. were found.  The calcium-mobilizing activity of Zn2+ was not abolished by pertussis toxin but was by a phospholipase C (PLC) inhibitor, U73122, indicating that the activity of GPR39 is mediated through the Gqα -PLC pathway.  In addn., Zn2+ also activated mouse and rat GPR39, showing that the function of GPR39 as a Zn2+ receptor is conserved across species.  This study is the first exploration of GPR39 agonists in FBS and indicates that GPR39 functions as a Gq-coupled Zn2+-sensing receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYWj8fr2kSX7Vg90H21EOLACvtfcHk0ljNxDnfv7ehMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVGrs7fL&md5=f6fdd40e52ad9380e3b88cbdd9152bdf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1080%2F10799890701506147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10799890701506147%26sid%3Dliteratum%253Aachs%26aulast%3DYasuda%26aufirst%3DS.%26aulast%3DMiyazaki%26aufirst%3DT.%26aulast%3DMunechika%26aufirst%3DK.%26aulast%3DYamashita%26aufirst%3DM.%26aulast%3DIkeda%26aufirst%3DY.%26aulast%3DKamizono%26aufirst%3DA.%26atitle%3DIsolation%2520of%2520Zn2%252B%2520as%2520an%2520Endogenous%2520Agonist%2520of%2520GPR39%2520from%2520Fetal%2520Bovine%2520Serum%26jtitle%3DJ.%2520Recept.%2520Signal%2520Transduction%26date%3D2007%26volume%3D27%26spage%3D235%26epage%3D246%26doi%3D10.1080%2F10799890701506147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holst, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egerod, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schild, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vickers, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheetham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, L.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storjohann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stidsen, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck-Sickinger, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, T. W.</span></span> <span> </span><span class="NLM_article-title">GPR39 Signaling Is Stimulated by Zinc Ions but Not by Obestatin</span>. <i>Endocrinology</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>148</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1210/en.2006-0933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1210%2Fen.2006-0933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=16959833" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsV2kuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2007&pages=13-20&author=B.+Holstauthor=K.+L.+Egerodauthor=E.+Schildauthor=S.+P.+Vickersauthor=S.+Cheethamauthor=L.-O.+Gerlachauthor=L.+Storjohannauthor=C.+E.+Stidsenauthor=R.+Jonesauthor=A.+G.+Beck-Sickingerauthor=T.+W.+Schwartz&title=GPR39+Signaling+Is+Stimulated+by+Zinc+Ions+but+Not+by+Obestatin&doi=10.1210%2Fen.2006-0933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">GPR39 signaling is stimulated by zinc ions but not by Obestatin</span></div><div class="casAuthors">Holst, Birgitte; Egerod, Kristoffer L.; Schild, Enrico; Vickers, Steve P.; Cheetham, Sharon; Gerlach, Lars-Ole; Storjohann, Laura; Stidsen, Carsten E.; Jones, Rob; Beck-Sickinger, Annette G.; Schwartz, Thue W.</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">148</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-20</span>CODEN:
                <span class="NLM_cas:coden">ENDOAO</span>;
        ISSN:<span class="NLM_cas:issn">0013-7227</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">GPR39 is an orphan member of the ghrelin receptor family that recently was suggested to be the receptor for Obestatin, a peptide derived from the ghrelin precursor.  Here, we compare the effect of Obestatin to the effect of Zn2+ on signal transduction and study the effect of Obestatin on food intake.  Although Zn2+ stimulated inositol phosphate turnover, cAMP prodn., arrestin mobilization, as well as cAMP response element-dependent and serum response element-dependent transcriptional activity in GPR39-expressing cells as opposed to mock-transfected cells, no reproducible effect was obtained with Obestatin in the GPR39-expressing cells.  Moreover, no specific binding of Obestatin could be detected in two different types of GPR39-expressing cells using three different radio-iodinated forms of Obestatin.  By quant. PCR anal., GPR39 expression was readily detected in peripheral organs such as duodenum and kidney but not in the pituitary and hypothalamus, i.e. presumed central target organs for Obestatin.  Obestatin had no significant and reproducible effect on acute food intake in either freely fed or fasted lean mice.  It is concluded that GPR39 is probably not the Obestatin receptor.  In contrast, the potency and efficacy of Zn2+ in respect of activating signaling indicates that this metal ion could be a physiol. relevant agonist or modulator of GPR39.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozz858uBcTxLVg90H21EOLACvtfcHk0ljNxDnfv7ehMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsV2kuw%253D%253D&md5=d9809707fa39525ca2c0392b66efc7c4</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1210%2Fen.2006-0933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2006-0933%26sid%3Dliteratum%253Aachs%26aulast%3DHolst%26aufirst%3DB.%26aulast%3DEgerod%26aufirst%3DK.%2BL.%26aulast%3DSchild%26aufirst%3DE.%26aulast%3DVickers%26aufirst%3DS.%2BP.%26aulast%3DCheetham%26aufirst%3DS.%26aulast%3DGerlach%26aufirst%3DL.-O.%26aulast%3DStorjohann%26aufirst%3DL.%26aulast%3DStidsen%26aufirst%3DC.%2BE.%26aulast%3DJones%26aufirst%3DR.%26aulast%3DBeck-Sickinger%26aufirst%3DA.%2BG.%26aulast%3DSchwartz%26aufirst%3DT.%2BW.%26atitle%3DGPR39%2520Signaling%2520Is%2520Stimulated%2520by%2520Zinc%2520Ions%2520but%2520Not%2520by%2520Obestatin%26jtitle%3DEndocrinology%26date%3D2007%26volume%3D148%26spage%3D13%26epage%3D20%26doi%3D10.1210%2Fen.2006-0933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroeze, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Novel GPR39 Agonists Allosterically Modulated by Zinc</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">726</span>– <span class="NLM_lpage">737</span>, <span class="refDoi"> DOI: 10.1124/mol.116.106112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fmol.116.106112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=27754899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitValu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2016&pages=726-737&author=S.+Satoauthor=X.-P.+Huangauthor=W.+K.+Kroezeauthor=B.+L.+Roth&title=Discovery+and+Characterization+of+Novel+GPR39+Agonists+Allosterically+Modulated+by+Zinc&doi=10.1124%2Fmol.116.106112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of novel GPR39 agonists allosterically modulated by zinc</span></div><div class="casAuthors">Sato, Seiji; Huang, Xi-Ping; Kroeze, Wesley K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">726-737</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">In this study,we identified two previously described kinase inhibitors- 3-(4-chloro-2-fluorobenzyl)-2-methyl-N-(3-methyl-1H-pyrazol- 5-yl)-8-(morpholinomethyl)imidazo[1,2-b]pyridazin-6-amine (LY2784544) and 1H-benzimidazole-4-carboxylic acid, 2-methyl- 1-[[2-methyl-3-(trifluoromethyl)phenyl]methyl]-6-(4-morpholinyl)- (GSK2636771)-as novel GPR39 agonists by unbiased small-mol.-based screening using a β-arrestin recruitment screening approach (PRESTO-Tango).  We characterized the signaling of LY2784544 and GSK2636771 and compared their signaling patterns with a previously described "GPR39-selective" agonist N-[3-chloro-4-[[[2-(methylamino)-6-(2-pyridinyl)-4- pyrimidinyl]amino]methyl]phenyl]methanesulfonamide (GPR39-C3) at both canonical and noncanonical signaling pathways.  Unexpectedly, all three compds. displayed probe-dependent and pathway-dependent allosteric modulation by concns. of zinc reported to be physiol.  LY2784544 and GS2636771 at GPR39 in the presence of zinc were generally as potent or more potent than their reported activities against kinases in whole-cell assays.  These findings reveal an unexpected role of zinc as an allosteric potentiator of small-mol.-induced activation of GPR39 and expand the list of potential kinase off-targets to include understudied G protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-OZkkmVxuTbVg90H21EOLACvtfcHk0lhDJkFATZG7vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitValu7Y%253D&md5=947d12ea0d4932d63cb83d0b76f1b09e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1124%2Fmol.116.106112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.116.106112%26sid%3Dliteratum%253Aachs%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Novel%2520GPR39%2520Agonists%2520Allosterically%2520Modulated%2520by%2520Zinc%26jtitle%3DMol.%2520Pharmacol.%26date%3D2016%26volume%3D90%26spage%3D726%26epage%3D737%26doi%3D10.1124%2Fmol.116.106112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ayton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bush, A. I.</span></span> <span> </span><span class="NLM_article-title">Biometals and Their Therapeutic Implications in Alzheimer’s Disease</span>. <i>Neurotherapeutics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1007/s13311-014-0312-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1007%2Fs13311-014-0312-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=25354461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFWqtrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=109-120&author=S.+Aytonauthor=P.+Leiauthor=A.+I.+Bush&title=Biometals+and+Their+Therapeutic+Implications+in+Alzheimer%E2%80%99s+Disease&doi=10.1007%2Fs13311-014-0312-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Biometals and Their Therapeutic Implications in Alzheimer's Disease</span></div><div class="casAuthors">Ayton, Scott; Lei, Peng; Bush, Ashley I.</div><div class="citationInfo"><span class="NLM_cas:title">Neurotherapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-120</span>CODEN:
                <span class="NLM_cas:coden">NEURNV</span>;
        ISSN:<span class="NLM_cas:issn">1878-7479</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">No disease modifying therapy exists for Alzheimer's disease (AD).  The growing burden of this disease to our society necessitates continued investment in drug development.  Over the last decade, multiple phase 3 clin. trials testing drugs that were designed to target established disease mechanisms of AD have all failed to benefit patients.  There is, therefore, a need for new treatment strategies.  Changes to the transition metals, zinc, copper, and iron, in AD impact on the mol. mechanisms of disease, and targeting these metals might be an alternative approach to treat the disease.  Here we review how metals feature in mol. mechanisms of AD, and we describe preclin. and clin. data that demonstrate the potential for metal-based drug therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6czoQwd24qLVg90H21EOLACvtfcHk0lhDJkFATZG7vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFWqtrbL&md5=8ece64c6a8b6a144e190f4f3685f5b5f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2Fs13311-014-0312-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13311-014-0312-z%26sid%3Dliteratum%253Aachs%26aulast%3DAyton%26aufirst%3DS.%26aulast%3DLei%26aufirst%3DP.%26aulast%3DBush%26aufirst%3DA.%2BI.%26atitle%3DBiometals%2520and%2520Their%2520Therapeutic%2520Implications%2520in%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DNeurotherapeutics%26date%3D2015%26volume%3D12%26spage%3D109%26epage%3D120%26doi%3D10.1007%2Fs13311-014-0312-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nordberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordberg, G. F.</span></span> <span> </span><span class="NLM_article-title">Trace Element Research-Historical and Future Aspects</span>. <i>J. Trace Elem. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">46</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1016/j.jtemb.2016.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.jtemb.2016.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=27238729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC28XptFeku74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2016&pages=46-52&author=M.+Nordbergauthor=G.+F.+Nordberg&title=Trace+Element+Research-Historical+and+Future+Aspects&doi=10.1016%2Fj.jtemb.2016.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Trace element research-historical and future aspects</span></div><div class="casAuthors">Nordberg, Monica; Nordberg, Gunnar F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Trace Elements in Medicine and Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">46-52</span>CODEN:
                <span class="NLM_cas:coden">JTEBFO</span>;
        ISSN:<span class="NLM_cas:issn">0946-672X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">During the last 30 years the International Society for Trace Element Research and the Nordic Trace Element Society has been active .  During this period the importance of these elements for human diseases has been increasingly recognized, including their contribution to the global burden of disease.  New anal. methods allow biomonitoring data to be related to health outcome.  Future research using modern chem. methods will focus more on elemental speciation and on measuring lower concns. leading to further identifying adverse effects and crit. organs.  Extensive knowledge about essentiality and toxicity of trace elements in humans has emerged during the last two decades and at present the difficulties in defining a range of acceptable oral intakes for essential elements has largely been overcome.  Biol. monitoring of trace element concns. in various media such as blood or urine is of great importance and an overview is given.  As an example, a more detailed description of biol. monitoring of cadmium is given, explaining biokinetics including the role of metallothionein in modifying kinetics and toxicity.  Finally future challenges related to risk assessment of newly developed metallic nanomaterials and metal contg. medical devices are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5vHXI3RTH5LVg90H21EOLACvtfcHk0lhDJkFATZG7vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptFeku74%253D&md5=6693cc9bd708986e10c529e317c92103</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.jtemb.2016.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jtemb.2016.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DNordberg%26aufirst%3DM.%26aulast%3DNordberg%26aufirst%3DG.%2BF.%26atitle%3DTrace%2520Element%2520Research-Historical%2520and%2520Future%2520Aspects%26jtitle%3DJ.%2520Trace%2520Elem.%2520Med.%2520Biol.%26date%3D2016%26volume%3D38%26spage%3D46%26epage%3D52%26doi%3D10.1016%2Fj.jtemb.2016.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prakash, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhaliwal, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majeed, A. B. A.</span></span> <span> </span><span class="NLM_article-title">Brain Biometals and Alzheimer’s Disease - Boon or Bane?</span>. <i>Int. J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">108</span>, <span class="refDoi"> DOI: 10.3109/00207454.2016.1174118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.3109%2F00207454.2016.1174118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=27044501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvFyrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=99-108&author=A.+Prakashauthor=G.+K.+Dhaliwalauthor=P.+Kumarauthor=A.+B.+A.+Majeed&title=Brain+Biometals+and+Alzheimer%E2%80%99s+Disease+-+Boon+or+Bane%3F&doi=10.3109%2F00207454.2016.1174118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Brain biometals and Alzheimer's disease - boon or bane?</span></div><div class="casAuthors">Prakash, Atish; Dhaliwal, Gagandeep Kaur; Kumar, Puneet; Abdul Majeed, Abu Bakar</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">99-108</span>CODEN:
                <span class="NLM_cas:coden">IJNUB7</span>;
        ISSN:<span class="NLM_cas:issn">0020-7454</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) is the most common form of dementia.  Several hypotheses have been put forward to explain the basis of disease onset and progression.  A complicated array of mol. events has been implicated in the pathogenesis of AD.  It is attributed to a variety of pathol. conditions that share similar crit. processes, such as oxidative stress, proteinaceous aggregations, mitochondrial dysfunctions and energy failure.  There is increasing evidence suggesting that metal homeostasis is dysregulated in the pathol. of AD.  Biometals play an important role in the normal body functioning but AD may be mediated or triggered by disproportion of metal ions leading to changes in crit. biol. systems and initiating a cascade of events finally leading to neurodegeneration and cell death.  The link is multifactorial, and although the source of the shift in oxidative homeostasis is still unclear, current evidence points to changes in the balance of redox transition metals, esp. iron, copper (Cu) and other trace metals.  Their levels in the brain are found to be elevated in AD.  In other neurodegenerative disorders, Cu, zinc, aluminum and manganese are involved.  This paper is a review of recent advances of the role of metals in the pathogenesis and pathophysiol. of AD and related neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzMxteJ9Bvu7Vg90H21EOLACvtfcHk0ljQtL1D_QPgLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvFyrtr0%253D&md5=61752befa2170b316304992907793696</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.3109%2F00207454.2016.1174118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00207454.2016.1174118%26sid%3Dliteratum%253Aachs%26aulast%3DPrakash%26aufirst%3DA.%26aulast%3DDhaliwal%26aufirst%3DG.%2BK.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMajeed%26aufirst%3DA.%2BB.%2BA.%26atitle%3DBrain%2520Biometals%2520and%2520Alzheimer%25E2%2580%2599s%2520Disease%2520-%2520Boon%2520or%2520Bane%253F%26jtitle%3DInt.%2520J.%2520Neurosci.%26date%3D2017%26volume%3D127%26spage%3D99%26epage%3D108%26doi%3D10.3109%2F00207454.2016.1174118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpiak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroeze, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saddoris, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolova, V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangano, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penn, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koller, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenakin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">Allosteric Ligands for the Pharmacologically Dark Receptors GPR68 and GPR65</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>527</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1038/nature15699</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fnature15699" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=26550826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVWmsrjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=527&publication_year=2015&pages=477-483&author=X.-P.+Huangauthor=J.+Karpiakauthor=W.+K.+Kroezeauthor=H.+Zhuauthor=X.+Chenauthor=S.+S.+Moyauthor=K.+A.+Saddorisauthor=V.+D.+Nikolovaauthor=M.+S.+Farrellauthor=S.+Wangauthor=T.+J.+Manganoauthor=D.+A.+Deshpandeauthor=A.+Jiangauthor=R.+B.+Pennauthor=J.+Jinauthor=B.+H.+Kollerauthor=T.+Kenakinauthor=B.+K.+Shoichetauthor=B.+L.+Roth&title=Allosteric+Ligands+for+the+Pharmacologically+Dark+Receptors+GPR68+and+GPR65&doi=10.1038%2Fnature15699"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric ligands for the pharmacologically dark receptors GPR68 and GPR65</span></div><div class="casAuthors">Huang, Xi-Ping; Karpiak, Joel; Kroeze, Wesley K.; Zhu, Hu; Chen, Xin; Moy, Sheryl S.; Saddoris, Kara A.; Nikolova, Viktoriya D.; Farrell, Martilias S.; Wang, Sheng; Mangano, Thomas J.; Deshpande, Deepak A.; Jiang, Alice; Penn, Raymond B.; Jin, Jian; Koller, Beverly H.; Kenakin, Terry; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">527</span>
        (<span class="NLM_cas:issue">7579</span>),
    <span class="NLM_cas:pages">477-483</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">At least 120 non-olfactory G-protein-coupled receptors in the human genome are 'orphans' for which endogenous ligands are unknown, and many have no selective ligands, hindering the detn. of their biol. functions and clin. relevance.  Among these is GPR68, a proton receptor that lacks small mol. modulators for probing its biol.  Using yeast-based screens against GPR68, here the authors identify the benzodiazepine drug lorazepam as a non-selective GPR68 pos. allosteric modulator.  More than 3000 GPR68 homol. models were refined to recognize lorazepam in a putative allosteric site.  Docking 3.1 million mols. predicted new GPR68 modulators, many of which were confirmed in functional assays.  One potent GPR68 modulator, ogerin, suppressed recall in fear conditioning in wild-type but not in GPR68-knockout mice.  The same approach led to the discovery of allosteric agonists and neg. allosteric modulators for GPR65.  Combining phys. and structure-based screening may be broadly useful for ligand discovery for understudied and orphan GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDB_IiCoSwVrVg90H21EOLACvtfcHk0ljQtL1D_QPgLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVWmsrjJ&md5=a23d2d68c06b96cfbe7b53f2b17f3895</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature15699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15699%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DKarpiak%26aufirst%3DJ.%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DMoy%26aufirst%3DS.%2BS.%26aulast%3DSaddoris%26aufirst%3DK.%2BA.%26aulast%3DNikolova%26aufirst%3DV.%2BD.%26aulast%3DFarrell%26aufirst%3DM.%2BS.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DDeshpande%26aufirst%3DD.%2BA.%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DPenn%26aufirst%3DR.%2BB.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKoller%26aufirst%3DB.%2BH.%26aulast%3DKenakin%26aufirst%3DT.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DAllosteric%2520Ligands%2520for%2520the%2520Pharmacologically%2520Dark%2520Receptors%2520GPR68%2520and%2520GPR65%26jtitle%3DNature%26date%3D2015%26volume%3D527%26spage%3D477%26epage%3D483%26doi%3D10.1038%2Fnature15699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McAleer, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Primerano, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denvir, J.</span></span> <span> </span><span class="NLM_article-title">Cytokine Regulation in Human CD4 T Cells by the Aryl Hydrocarbon Receptor and Gq-Coupled Receptors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">10954</span> <span class="refDoi"> DOI: 10.1038/s41598-018-29262-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fs41598-018-29262-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=30026493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BB3c7gtVWgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&author=J.+P.+McAleerauthor=J.+Fanauthor=B.+Roarauthor=D.+A.+Primeranoauthor=J.+Denvir&title=Cytokine+Regulation+in+Human+CD4+T+Cells+by+the+Aryl+Hydrocarbon+Receptor+and+Gq-Coupled+Receptors&doi=10.1038%2Fs41598-018-29262-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Cytokine Regulation in Human CD4 T Cells by the Aryl Hydrocarbon Receptor and Gq-Coupled Receptors</span></div><div class="casAuthors">McAleer Jeremy P; Roar Bryanna; Fan Jun; Primerano Donald A; Denvir James</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10954</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Th17 cells contribute to host defense on mucosal surfaces but also provoke autoimmune diseases when directed against self-antigens.  Identifying therapeutic targets that regulate Th17 cell differentiation and/or cytokine production has considerable value.  Here, we study the aryl hydrocarbon receptor (AhR)-dependent transcriptome in human CD4 T cells treated with Th17-inducing cytokines.  We show that the AhR reciprocally regulates IL-17 and IL-22 production in human CD4 T cells.  Global gene expression analysis revealed that AhR ligation decreased IL21 expression, correlating with delayed upregulation of RORC during culture with Th17-inducing cytokines.  Several of the AhR-dependent genes have known roles in cellular assembly, organization, development, growth and proliferation.  We further show that expression of GPR15, GPR55 and GPR68 positively correlates with IL-22 production in the presence of the AhR agonist FICZ.  Activation of GPR68 with the lorazepam derivative ogerin resulted in suppression of IL-22 and IL-10 secretion by T cells, with no effect on IL-17.  Under neutral Th0 conditions, ogerin and the Gq/11 receptor inhibitor YM254890 blunted IL-22 induction by FICZ.  These data reveal the AhR-dependent transcriptome in human CD4 T cells and suggest the mechanism through which the AhR regulates T cell function may be partially dependent on Gq-coupled receptors including GPR68.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTbp0LpAwT-1jdblLE_E9PWfW6udTcc2ebpidbD0vV6g7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7gtVWgtQ%253D%253D&md5=35b3e312e63edda984be2e40bf232c9d</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-29262-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-29262-4%26sid%3Dliteratum%253Aachs%26aulast%3DMcAleer%26aufirst%3DJ.%2BP.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DRoar%26aufirst%3DB.%26aulast%3DPrimerano%26aufirst%3DD.%2BA.%26aulast%3DDenvir%26aufirst%3DJ.%26atitle%3DCytokine%2520Regulation%2520in%2520Human%2520CD4%2520T%2520Cells%2520by%2520the%2520Aryl%2520Hydrocarbon%2520Receptor%2520and%2520Gq-Coupled%2520Receptors%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26doi%3D10.1038%2Fs41598-018-29262-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mysinger, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolstad, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. G.</span></span> <span> </span><span class="NLM_article-title">ZINC: a Free Tool to Discover Chemistry for Biology</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1757</span>– <span class="NLM_lpage">1768</span>, <span class="refDoi"> DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+a+Free+Tool+to+Discover+Chemistry+for+Biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0ljIh832m4j_bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520a%2520Free%2520Tool%2520to%2520Discover%2520Chemistry%2520for%2520Biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kroeze, W.
K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giguère, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciaky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">PRESTO-Tango as an Open-Source Resource for Interrogation of the Druggable Human GPCRome</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">369</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fnsmb.3014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=25895059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnslOgtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=362-369&author=W.%0AK.+Kroezeauthor=M.+F.+Sassanoauthor=X.-P.+Huangauthor=K.+Lansuauthor=J.+D.+McCorvyauthor=P.+M.+Gigu%C3%A8reauthor=N.+Sciakyauthor=B.+L.+Roth&title=PRESTO-Tango+as+an+Open-Source+Resource+for+Interrogation+of+the+Druggable+Human+GPCRome&doi=10.1038%2Fnsmb.3014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">PRESTO-Tango as an open-source resource for interrogation of the druggable human GPCRome</span></div><div class="casAuthors">Kroeze, Wesley K.; Sassano, Maria F.; Huang, Xi-Ping; Lansu, Katherine; McCorvy, John D.; Giguere, Patrick M.; Sciaky, Noah; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">362-369</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are essential mediators of cellular signaling and are important targets of drug action.  Of the approx. 350 nonolfactory human GPCRs, more than 100 are still considered to be 'orphans' because their endogenous ligands remain unknown.  Here, we describe a unique open-source resource that allows interrogation of the druggable human GPCRome via a G protein-independent β-arrestin-recruitment assay.  We validate this unique platform at more than 120 nonorphan human GPCR targets, demonstrate its utility for discovering new ligands for orphan human GPCRs and describe a method (parallel receptorome expression and screening via transcriptional output, with transcriptional activation following arrestin translocation (PRESTO-Tango)) for the simultaneous and parallel interrogation of the entire human nonolfactory GPCRome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVbl7caV9gUrVg90H21EOLACvtfcHk0ljIh832m4j_bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnslOgtLg%253D&md5=2a78b74a5f1425fa9e8b79b730c16be6</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3014%26sid%3Dliteratum%253Aachs%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DLansu%26aufirst%3DK.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DGigu%25C3%25A8re%26aufirst%3DP.%2BM.%26aulast%3DSciaky%26aufirst%3DN.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DPRESTO-Tango%2520as%2520an%2520Open-Source%2520Resource%2520for%2520Interrogation%2520of%2520the%2520Druggable%2520Human%2520GPCRome%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D362%26epage%3D369%26doi%3D10.1038%2Fnsmb.3014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bachelerie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Baruch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkhardt, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Combadiere, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farber, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horuk, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparre-Ulrich, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luster, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantovani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsushima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nibbs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Power, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proudfoot, A. E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenkilde, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sozzani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thelen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshie, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnik, A.</span></span> <span> </span><span class="NLM_article-title">International Union of Basic and Clinical Pharmacology. [corrected]. LXXXIX. Update on the Extended Family of Chemokine Receptors and Introducing a New Nomenclature for Atypical Chemokine Receptors</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1124/pr.113.007724</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fpr.113.007724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=24218476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptVyktLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2014&pages=1-79&author=F.+Bachelerieauthor=A.+Ben-Baruchauthor=A.+M.+Burkhardtauthor=C.+Combadiereauthor=J.+M.+Farberauthor=G.+J.+Grahamauthor=R.+Horukauthor=A.+H.+Sparre-Ulrichauthor=M.+Locatiauthor=A.+D.+Lusterauthor=A.+Mantovaniauthor=K.+Matsushimaauthor=P.+M.+Murphyauthor=R.+Nibbsauthor=H.+Nomiyamaauthor=C.+A.+Powerauthor=A.+E.+I.+Proudfootauthor=M.+M.+Rosenkildeauthor=A.+Rotauthor=S.+Sozzaniauthor=M.+Thelenauthor=O.+Yoshieauthor=A.+Zlotnik&title=International+Union+of+Basic+and+Clinical+Pharmacology.+%5Bcorrected%5D.+LXXXIX.+Update+on+the+Extended+Family+of+Chemokine+Receptors+and+Introducing+a+New+Nomenclature+for+Atypical+Chemokine+Receptors&doi=10.1124%2Fpr.113.007724"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">International union of basic and clinical pharmacology LXXXIX. Update on the extended family of chemokine receptors and introducing a new nomenclature for atypical chemokine receptors</span></div><div class="casAuthors">Bachelerie, Francoise; Ben-Baruch, Adit; Burkhardt, Amanda M.; Combadiere, Christophe; Farber, Joshua M.; Graham, Gerard J.; Horuk, Richard; Sparre-Ulrich, Alexander Hovard; Locati, Massimo; Luster, Andrew D.; Mantovani, Alberto; Matsushima, Kouji; Murphy, Philip M.; Nibbs, Robert; Nomiyama, Hisayuki; Power, Christine A.; Proudfoot, Amanda E. I.; Rosenkilde, Mette M.; Rot, Antal; Sozzani, Silvano; Thelen, Marcus; Yoshie, Osamu; Zlotnik, Albert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-79, 80 pp.</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Sixteen years ago, the Nomenclature Committee of the International Union of Pharmacol. approved a system for naming human seven-transmembrane (7TM) G protein-coupled chemokine receptors, the large family of leukocyte chemoattractant receptors that regulates immune system development and function, in large part by mediating leukocyte trafficking.  This was announced in Pharmacol. Reviews in a major overview of the first decade of research in this field.  Since then, several new receptors have been discovered, and major advances have been made for the others in many areas, including structural biol., signal transduction mechanisms, biol., and pharmacol.  New and diverse roles have been identified in infection, immunity, inflammation, development, cancer, and other areas.  The first two drugs acting at chemokine receptors have been approved by the U.S. Food and Drug Administration (FDA), maraviroc targeting CCR5 in human immunodeficiency virus (HIV)/AIDS, and plerixafor targeting CXCR4 for stem cell mobilization for transplantation in cancer, and other candidates are now undergoing pivotal clin. trials for diverse disease indications.  In addn., a subfamily of atypical chemokine receptors has emerged that may signal through arrestins instead of G proteins to act as chemokine scavengers, and many microbial and invertebrate G protein-coupled chemokine receptors and sol. chemokine-binding proteins have been described.  Here, we review this extended family of chemokine receptors and chemokine-binding proteins at the basic, translational, and clin. levels, including an update on drug development.  We also introduce a new nomenclature for atypical chemokine receptors with the stem ACKR (atypical chemokine receptor) approved by the Nomenclature Committee of the International Union of Pharmacol. and the Human Genome Nomenclature Committee.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7EhMCJlz-zLVg90H21EOLACvtfcHk0ljIh832m4j_bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptVyktLg%253D&md5=44a2c14ddf2508ada59121610c5682cf</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1124%2Fpr.113.007724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.113.007724%26sid%3Dliteratum%253Aachs%26aulast%3DBachelerie%26aufirst%3DF.%26aulast%3DBen-Baruch%26aufirst%3DA.%26aulast%3DBurkhardt%26aufirst%3DA.%2BM.%26aulast%3DCombadiere%26aufirst%3DC.%26aulast%3DFarber%26aufirst%3DJ.%2BM.%26aulast%3DGraham%26aufirst%3DG.%2BJ.%26aulast%3DHoruk%26aufirst%3DR.%26aulast%3DSparre-Ulrich%26aufirst%3DA.%2BH.%26aulast%3DLocati%26aufirst%3DM.%26aulast%3DLuster%26aufirst%3DA.%2BD.%26aulast%3DMantovani%26aufirst%3DA.%26aulast%3DMatsushima%26aufirst%3DK.%26aulast%3DMurphy%26aufirst%3DP.%2BM.%26aulast%3DNibbs%26aufirst%3DR.%26aulast%3DNomiyama%26aufirst%3DH.%26aulast%3DPower%26aufirst%3DC.%2BA.%26aulast%3DProudfoot%26aufirst%3DA.%2BE.%2BI.%26aulast%3DRosenkilde%26aufirst%3DM.%2BM.%26aulast%3DRot%26aufirst%3DA.%26aulast%3DSozzani%26aufirst%3DS.%26aulast%3DThelen%26aufirst%3DM.%26aulast%3DYoshie%26aufirst%3DO.%26aulast%3DZlotnik%26aufirst%3DA.%26atitle%3DInternational%2520Union%2520of%2520Basic%2520and%2520Clinical%2520Pharmacology.%2520%255Bcorrected%255D.%2520LXXXIX.%2520Update%2520on%2520the%2520Extended%2520Family%2520of%2520Chemokine%2520Receptors%2520and%2520Introducing%2520a%2520New%2520Nomenclature%2520for%2520Atypical%2520Chemokine%2520Receptors%26jtitle%3DPharmacol.%2520Rev.%26date%3D2014%26volume%3D66%26spage%3D1%26epage%3D79%26doi%3D10.1124%2Fpr.113.007724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bachelerie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantovani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nibbs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sozzani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thelen, M.</span></span> <span> </span><span class="NLM_article-title">An Atypical Addition to the Chemokine Receptor Nomenclature: IUPHAR Review 15</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">3945</span>– <span class="NLM_lpage">3949</span>, <span class="refDoi"> DOI: 10.1111/bph.13182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1111%2Fbph.13182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=25958743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOhurjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2015&pages=3945-3949&author=F.+Bachelerieauthor=G.+J.+Grahamauthor=M.+Locatiauthor=A.+Mantovaniauthor=P.+M.+Murphyauthor=R.+Nibbsauthor=A.+Rotauthor=S.+Sozzaniauthor=M.+Thelen&title=An+Atypical+Addition+to+the+Chemokine+Receptor+Nomenclature%3A+IUPHAR+Review+15&doi=10.1111%2Fbph.13182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">An atypical addition to the chemokine receptor nomenclature: IUPHAR review 15</span></div><div class="casAuthors">Bachelerie, Francoise; Graham, Gerard J.; Locati, Massimo; Mantovani, Alberto; Murphy, Philip M.; Nibbs, Robert; Rot, Antal; Sozzani, Silvano; Thelen, Marcus</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3945-3949</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Chemokines and their receptors are essential regulators of in vivo leukocyte migration and, some years ago, a systematic nomenclature system was developed for the chemokine receptor family.  Chemokine receptor biol. and biochem. was recently extensively reviewed.  In this review, we also highlighted a new component to the nomenclature system that incorporates receptors previously known as 'scavenging', or 'decoy', chemokine receptors on the basis of their lack of classical signalling responses to ligand binding and their general ability to scavenge, or sequester, their cognate chemokine ligands.  These mols. are now collectively referred to as 'atypical chemokine receptors', or ACKRs, and play fundamental roles in regulating in vivo responses to chemokines.  This commentary highlights this new addn. to the chemokine receptor nomenclature system and provides brief information on the four receptors currently covered by this nomenclature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroBDigTqM-rrVg90H21EOLACvtfcHk0lgbyPMQtv3qSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOhurjE&md5=978dce7741d3af741ca16cb2635d0b7a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1111%2Fbph.13182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.13182%26sid%3Dliteratum%253Aachs%26aulast%3DBachelerie%26aufirst%3DF.%26aulast%3DGraham%26aufirst%3DG.%2BJ.%26aulast%3DLocati%26aufirst%3DM.%26aulast%3DMantovani%26aufirst%3DA.%26aulast%3DMurphy%26aufirst%3DP.%2BM.%26aulast%3DNibbs%26aufirst%3DR.%26aulast%3DRot%26aufirst%3DA.%26aulast%3DSozzani%26aufirst%3DS.%26aulast%3DThelen%26aufirst%3DM.%26atitle%3DAn%2520Atypical%2520Addition%2520to%2520the%2520Chemokine%2520Receptor%2520Nomenclature%253A%2520IUPHAR%2520Review%252015%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2015%26volume%3D172%26spage%3D3945%26epage%3D3949%26doi%3D10.1111%2Fbph.13182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nickols, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conn, P. J.</span></span> <span> </span><span class="NLM_article-title">Development of Allosteric Modulators of GPCRs for Treatment of CNS Disorders</span>. <i>Neurobiol. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">71</span>, <span class="refDoi"> DOI: 10.1016/j.nbd.2013.09.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.nbd.2013.09.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=24076101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A280%3ADC%252BC2c%252FitVejsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2014&pages=55-71&author=H.+H.+Nickolsauthor=P.+J.+Conn&title=Development+of+Allosteric+Modulators+of+GPCRs+for+Treatment+of+CNS+Disorders&doi=10.1016%2Fj.nbd.2013.09.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Development of allosteric modulators of GPCRs for treatment of CNS disorders</span></div><div class="casAuthors">Nickols Hilary Highfield; Conn P Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Neurobiology of disease</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">55-71</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The discovery of allosteric modulators of G protein-coupled receptors (GPCRs) provides a promising new strategy with potential for developing novel treatments for a variety of central nervous system (CNS) disorders.  Traditional drug discovery efforts targeting GPCRs have focused on developing ligands for orthosteric sites which bind endogenous ligands.  Allosteric modulators target a site separate from the orthosteric site to modulate receptor function.  These allosteric agents can either potentiate (positive allosteric modulator, PAM) or inhibit (negative allosteric modulator, NAM) the receptor response and often provide much greater subtype selectivity than orthosteric ligands for the same receptors.  Experimental evidence has revealed more nuanced pharmacological modes of action of allosteric modulators, with some PAMs showing allosteric agonism in combination with positive allosteric modulation in response to endogenous ligand (ago-potentiators) as well as "bitopic" ligands that interact with both the allosteric and orthosteric sites.  Drugs targeting the allosteric site allow for increased drug selectivity and potentially decreased adverse side effects.  Promising evidence has demonstrated potential utility of a number of allosteric modulators of GPCRs in multiple CNS disorders, including neurodegenerative diseases such as Alzheimer's disease, Parkinson's disease, and Huntington's disease, as well as psychiatric or neurobehavioral diseases such as anxiety, schizophrenia, and addiction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT5AaOVGoQQ2DEC-75aqa5AfW6udTcc2eZqrr1cLievurntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c%252FitVejsA%253D%253D&md5=a1535b4448734bcae63e9797d2009737</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.nbd.2013.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nbd.2013.09.013%26sid%3Dliteratum%253Aachs%26aulast%3DNickols%26aufirst%3DH.%2BH.%26aulast%3DConn%26aufirst%3DP.%2BJ.%26atitle%3DDevelopment%2520of%2520Allosteric%2520Modulators%2520of%2520GPCRs%2520for%2520Treatment%2520of%2520CNS%2520Disorders%26jtitle%3DNeurobiol.%2520Dis.%26date%3D2014%26volume%3D61%26spage%3D55%26epage%3D71%26doi%3D10.1016%2Fj.nbd.2013.09.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burford, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wehrman, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerritz, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traynor, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, A.</span></span> <span> </span><span class="NLM_article-title">Discovery of Positive Allosteric Modulators and Silent Allosteric Modulators of the μ-Opioid Receptor</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">10830</span>– <span class="NLM_lpage">10835</span>, <span class="refDoi"> DOI: 10.1073/pnas.1300393110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1073%2Fpnas.1300393110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=23754417" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFejs7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=10830-10835&author=N.+T.+Burfordauthor=M.+J.+Clarkauthor=T.+S.+Wehrmanauthor=S.+W.+Gerritzauthor=M.+Banksauthor=J.+O%E2%80%99Connellauthor=J.+R.+Traynorauthor=A.+Alt&title=Discovery+of+Positive+Allosteric+Modulators+and+Silent+Allosteric+Modulators+of+the+%CE%BC-Opioid+Receptor&doi=10.1073%2Fpnas.1300393110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of positive allosteric modulators and silent allosteric modulators of the μ-opioid receptor</span></div><div class="casAuthors">Burford, Neil T.; Clark, Mary J.; Wehrman, Tom S.; Gerritz, Samuel W.; Banks, Martyn; O'Connell, Jonathan; Traynor, John R.; Alt, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10830-10835,S10830/1-S10830/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">μ-Opioid receptors are among the most studied G protein-coupled receptors because of the therapeutic value of agonists, such as morphine, that are used to treat chronic pain.  However, these drugs have significant side effects, such as respiratory suppression, constipation, allodynia, tolerance, and dependence, as well as abuse potential.  Efforts to fine tune pain control while alleviating the side effects of drugs, both physiol. and psychol., have led to the development of a wide variety of structurally diverse agonist ligands for the μ-opioid receptor, as well as compds. that target κ- and δ-opioid receptors.  In recent years, the identification of allosteric ligands for some G protein-coupled receptors has provided breakthroughs in obtaining receptor subtype-selectivity that can reduce the overall side effect profiles of a potential drug.  However, pos. allosteric modulators (PAMs) can also have the specific advantage of only modulating the activity of the receptor when the orthosteric agonist occupies the receptor, thus maintaining spatial and temporal control of receptor signaling in vivo.  This second advantage of allosteric modulators may yield breakthroughs in opioid receptor research and could lead to drugs with improved side-effect profiles or fewer tolerance and dependence issues compared with orthosteric opioid receptor agonists.  Here, the authors describe the discovery and characterization of μ-opioid receptor PAMs and silent allosteric modulators, identified from high-throughput screening using a β-arrestin-recruitment assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdiTL5ko3pr7Vg90H21EOLACvtfcHk0lgbyPMQtv3qSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFejs7nJ&md5=5fd64e5ae56607fa2396fdd3f32caf54</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1300393110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1300393110%26sid%3Dliteratum%253Aachs%26aulast%3DBurford%26aufirst%3DN.%2BT.%26aulast%3DClark%26aufirst%3DM.%2BJ.%26aulast%3DWehrman%26aufirst%3DT.%2BS.%26aulast%3DGerritz%26aufirst%3DS.%2BW.%26aulast%3DBanks%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DJ.%26aulast%3DTraynor%26aufirst%3DJ.%2BR.%26aulast%3DAlt%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520Positive%2520Allosteric%2520Modulators%2520and%2520Silent%2520Allosteric%2520Modulators%2520of%2520the%2520%25CE%25BC-Opioid%2520Receptor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D10830%26epage%3D10835%26doi%3D10.1073%2Fpnas.1300393110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Livingston, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanczyk, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burford, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canals, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traynor, J. R.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic Evidence for a Putative Conserved Allosteric Site on Opioid Receptors</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1124/mol.117.109561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fmol.117.109561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=29233847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsF2rtb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2018&pages=157-167&author=K.+E.+Livingstonauthor=M.+A.+Stanczykauthor=N.+T.+Burfordauthor=A.+Altauthor=M.+Canalsauthor=J.+R.+Traynor&title=Pharmacologic+Evidence+for+a+Putative+Conserved+Allosteric+Site+on+Opioid+Receptors&doi=10.1124%2Fmol.117.109561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic evidence for a putative conserved allosteric site on opioid receptors</span></div><div class="casAuthors">Livingston, Kathryn E.; Stanczyk, M. Alexander; Burford, Neil T.; Alt, Andrew; Canals, Meritxell; Traynor, John R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-167</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Allosteric modulators of G protein-coupled receptors, including opioid receptors, have been proposed as possible therapeutic agents with enhanced selectivity.  BMS-986122 is a pos. allosteric modulator (PAM) of the μ-opioid receptor (μ-OR).  BMS-986187 is a structurally distinct PAM for the δ-opioid receptor (δ-OR) that has been reported to exhibit 100-fold selectivity in promoting δ-OR over μ-OR agonism.  We used ligand binding and second-messenger assays to show that BMS-986187 is an effective PAM at the μ-OR and at the k-opioid receptor (κ-OR), but it is ineffective at the nociceptin receptor.  The affinity of BMS-986187 for δ-ORs and κ-ORs is approx. 20- to 30-fold higher than for μ-ORs, detd. using an allosteric ternary complex model.  Moreover, we provide evidence, using a silent allosteric modulator as an allosteric antagonist, that BMS-986187 and BMS-986122 bind to a similar region on all three traditional opioid receptor types (μ-OR, δ-OR, and κ-OR).  In contrast to the dogma surrounding allosteric modulators, the results indicate a possible conserved allosteric binding site across the opioid receptor family that can accommodate structurally diverse mols.  These findings have implications for the development of selective allosteric modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPZysO_yHQu7Vg90H21EOLACvtfcHk0lghkRuBWHz37Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsF2rtb3F&md5=8b2e08fa5e7036428c428b97f4f6ab90</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1124%2Fmol.117.109561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.117.109561%26sid%3Dliteratum%253Aachs%26aulast%3DLivingston%26aufirst%3DK.%2BE.%26aulast%3DStanczyk%26aufirst%3DM.%2BA.%26aulast%3DBurford%26aufirst%3DN.%2BT.%26aulast%3DAlt%26aufirst%3DA.%26aulast%3DCanals%26aufirst%3DM.%26aulast%3DTraynor%26aufirst%3DJ.%2BR.%26atitle%3DPharmacologic%2520Evidence%2520for%2520a%2520Putative%2520Conserved%2520Allosteric%2520Site%2520on%2520Opioid%2520Receptors%26jtitle%3DMol.%2520Pharmacol.%26date%3D2018%26volume%3D93%26spage%3D157%26epage%3D167%26doi%3D10.1124%2Fmol.117.109561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haas, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salazar, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herber, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degnan, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balakrishnan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macor, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albright, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strittmatter, S. M.</span></span> <span> </span><span class="NLM_article-title">Silent Allosteric Modulation of mGluR5 Maintains Glutamate Signaling While Rescuing Alzheimer’s Mouse Phenotypes</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.06.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.celrep.2017.06.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=28683325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFartbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2017&pages=76-88&author=L.+T.+Haasauthor=S.+V.+Salazarauthor=L.+M.+Smithauthor=H.+R.+Zhaoauthor=T.+O.+Coxauthor=C.+S.+Herberauthor=A.+P.+Degnanauthor=A.+Balakrishnanauthor=J.+E.+Macorauthor=C.+F.+Albrightauthor=S.+M.+Strittmatter&title=Silent+Allosteric+Modulation+of+mGluR5+Maintains+Glutamate+Signaling+While+Rescuing+Alzheimer%E2%80%99s+Mouse+Phenotypes&doi=10.1016%2Fj.celrep.2017.06.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Silent Allosteric Modulation of mGluR5 Maintains Glutamate Signaling while Rescuing Alzheimer's Mouse Phenotypes</span></div><div class="casAuthors">Haas, Laura T.; Salazar, Santiago V.; Smith, Levi M.; Zhao, Helen R.; Cox, Timothy O.; Herber, Charlotte S.; Degnan, Andrew P.; Balakrishnan, Anand; Macor, John E.; Albright, Charles F.; Strittmatter, Stephen M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-88</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Metabotropic glutamate receptor 5 (mGluR5) has been implicated in Alzheimer's disease (AD) pathol.  We sought to understand whether mGluR5's role in AD requires glutamate signaling.  We used a potent mGluR5 silent allosteric modulator (SAM, BMS-984923) to sep. its well-known physiol. role in glutamate signaling from a pathol. role in mediating amyloid-β oligomer (Aβo) action.  Binding of the SAM to mGluR5 does not change glutamate signaling but strongly reduces mGluR5 interaction with cellular prion protein (PrPC) bound to Aβo.  The SAM compd. prevents Aβo-induced signal transduction in brain slices and in an AD transgenic mouse model, the APPswe/PS1ΔE9 strain.  Critically, 4 wk of SAM treatment rescues memory deficits and synaptic depletion in the APPswe/PS1ΔE9 transgenic mouse brain.  Our data show that mGluR5's role in Aβo-dependent AD phenotypes is sep. from its role in glutamate signaling and silent allosteric modulation of mGluR5 has promise as a disease-modifying AD intervention with a broad therapeutic window.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiq45bkzh9ebVg90H21EOLACvtfcHk0lghkRuBWHz37Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFartbnN&md5=3e3e1a787fb58397ac649068f0ccd07e</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.06.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.06.023%26sid%3Dliteratum%253Aachs%26aulast%3DHaas%26aufirst%3DL.%2BT.%26aulast%3DSalazar%26aufirst%3DS.%2BV.%26aulast%3DSmith%26aufirst%3DL.%2BM.%26aulast%3DZhao%26aufirst%3DH.%2BR.%26aulast%3DCox%26aufirst%3DT.%2BO.%26aulast%3DHerber%26aufirst%3DC.%2BS.%26aulast%3DDegnan%26aufirst%3DA.%2BP.%26aulast%3DBalakrishnan%26aufirst%3DA.%26aulast%3DMacor%26aufirst%3DJ.%2BE.%26aulast%3DAlbright%26aufirst%3DC.%2BF.%26aulast%3DStrittmatter%26aufirst%3DS.%2BM.%26atitle%3DSilent%2520Allosteric%2520Modulation%2520of%2520mGluR5%2520Maintains%2520Glutamate%2520Signaling%2520While%2520Rescuing%2520Alzheimer%25E2%2580%2599s%2520Mouse%2520Phenotypes%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D20%26spage%3D76%26epage%3D88%26doi%3D10.1016%2Fj.celrep.2017.06.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conn, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span> <span> </span><span class="NLM_article-title">Synthesis and SAR of a mGluR5 Allosteric Partial Antagonist Lead: Unexpected Modulation of Pharmacology with Slight Structural Modifications to a 5-(Phenylethynyl)pyrimidine Scaffold</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">4098</span>– <span class="NLM_lpage">4101</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.05.091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.bmcl.2008.05.091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=18550372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslequro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=4098-4101&author=S.+Sharmaauthor=A.+L.+Rodriguezauthor=P.+J.+Connauthor=C.+W.+Lindsley&title=Synthesis+and+SAR+of+a+mGluR5+Allosteric+Partial+Antagonist+Lead%3A+Unexpected+Modulation+of+Pharmacology+with+Slight+Structural+Modifications+to+a+5-%28Phenylethynyl%29pyrimidine+Scaffold&doi=10.1016%2Fj.bmcl.2008.05.091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and SAR of a mGluR5 allosteric partial antagonist lead: Unexpected modulation of pharmacology with slight structural modifications to a 5-(phenylethynyl)pyrimidine scaffold</span></div><div class="casAuthors">Sharma, Sameer; Rodriguez, Alice L.; Conn, P. Jeffrey; Lindsley, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4098-4101</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">This Letter describes the synthesis and SAR, developed through an iterative analog library approach, of a mGluR5 allosteric partial antagonist lead based on a 5-(phenylethynyl)pyrimidine scaffold I (R = H, 3-Me, 3,5-F2, etc.).  With slight structural modifications to the distal Ph ring, analogs demonstrated a range of pharmacol. activities from mGluR5 partial antagonism to full antagonism/neg. allosteric modulation to pos. allosteric modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQBeYocKgYurVg90H21EOLACvtfcHk0lghkRuBWHz37Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslequro%253D&md5=323d229f189185ed5d237e0a6e37e9a8</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.05.091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.05.091%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DRodriguez%26aufirst%3DA.%2BL.%26aulast%3DConn%26aufirst%3DP.%2BJ.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3DSynthesis%2520and%2520SAR%2520of%2520a%2520mGluR5%2520Allosteric%2520Partial%2520Antagonist%2520Lead%253A%2520Unexpected%2520Modulation%2520of%2520Pharmacology%2520with%2520Slight%2520Structural%2520Modifications%2520to%2520a%25205-%2528Phenylethynyl%2529pyrimidine%2520Scaffold%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D4098%26epage%3D4101%26doi%3D10.1016%2Fj.bmcl.2008.05.091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brogan, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conn, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span> <span> </span><span class="NLM_article-title">Molecular Switches” on mGluR Allosteric Ligands That Modulate Modes of Pharmacology</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">2403</span>– <span class="NLM_lpage">2410</span>, <span class="refDoi"> DOI: 10.1021/bi200129s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi200129s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3MXislGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=2403-2410&author=M.+R.+Woodauthor=C.+R.+Hopkinsauthor=J.+T.+Broganauthor=P.+J.+Connauthor=C.+W.+Lindsley&title=Molecular+Switches%E2%80%9D+on+mGluR+Allosteric+Ligands+That+Modulate+Modes+of+Pharmacology&doi=10.1021%2Fbi200129s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">"Molecular Switches" on mGluR Allosteric Ligands that Modulate Modes of Pharmacology</span></div><div class="casAuthors">Wood, Michael R.; Hopkins, Corey R.; Brogan, John T.; Conn, P. Jeffrey; Lindsley, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2403-2410</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  G-Protein-coupled receptors (GPCRs) represent the largest class of drug targets, accounting for more than 40% of marketed drugs; however, discovery efforts for many GPCRs have failed to provide viable drug candidates.  Historically, drug discovery efforts have focused on developing ligands that act at the orthosteric site of the endogenous agonist.  Recently, efforts have focused on functional assay paradigms and the discovery of ligands that act at allosteric sites to modulate receptor function in either a pos., neg., or neutral manner.  Allosteric modulators have numerous advantages over orthosteric ligands, including high subtype selectivity; the ability to mimic physiol. conditions; the lack of desensitization, downregulation, and internalization; and reduced side effects.  Despite these virtues, challenging issues have now arisen for allosteric modulators of metabotropic glutamate receptors (mGluRs): Shallow SAR, ligand-directed trafficking, and the identification of subtle "mol. switches" that modulate the modes of pharmacol.  Here, we will discuss the impact of modest structural changes to multiple mGluR allosteric ligands scaffolds that unexpectedly modulate pharmacol. and raise concerns over metab. and the pharmacol. of metabolites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf_Kg5p51i47Vg90H21EOLACvtfcHk0li_-VX7I0zl9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXislGntbg%253D&md5=adc945bc7ea29582144a60687a9aaf9d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fbi200129s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi200129s%26sid%3Dliteratum%253Aachs%26aulast%3DWood%26aufirst%3DM.%2BR.%26aulast%3DHopkins%26aufirst%3DC.%2BR.%26aulast%3DBrogan%26aufirst%3DJ.%2BT.%26aulast%3DConn%26aufirst%3DP.%2BJ.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3DMolecular%2520Switches%25E2%2580%259D%2520on%2520mGluR%2520Allosteric%2520Ligands%2520That%2520Modulate%2520Modes%2520of%2520Pharmacology%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D2403%26epage%3D2410%26doi%3D10.1021%2Fbi200129s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conn, P. J.</span></span> <span> </span><span class="NLM_article-title">Molecular Insights into Metabotropic Glutamate Receptor Allosteric Modulation</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1124/mol.114.097220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fmol.114.097220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=25808929" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVGgt7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2015&pages=188-202&author=K.+J.+Gregoryauthor=P.+J.+Conn&title=Molecular+Insights+into+Metabotropic+Glutamate+Receptor+Allosteric+Modulation&doi=10.1124%2Fmol.114.097220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular insights into metabotropic glutamate receptor allosteric modulation</span></div><div class="casAuthors">Gregory, Karen J.; Conn, P. Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">188-202</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The metabotropic glutamate (mGlu) receptors are a group of eight family C G protein-coupled receptors that are expressed throughout the central nervous system (CNS) and periphery.  Within the CNS the different subtypes are found in neurons, both pre- and/or postsynaptically, where they mediate modulatory roles and in glial cells.  The mGlu receptor family provides attractive targets for numerous psychiatric and neurol. disorders, with the majority of discovery programs focused on targeting allosteric sites, with allosteric ligands now available for all mGlu receptor subtypes.  However, the development of allosteric ligands remains challenging.  Biased modulation, probe dependence, and mol. switches all contribute to the complex mol. pharmacol. exhibited by mGlu receptor allosteric ligands.  In recent years we have made significant progress in our understanding of this mol. complexity coupled with an increased understanding of the structural basis of mGlu allosteric modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzyYqss3Nb4rVg90H21EOLACvtfcHk0li_-VX7I0zl9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVGgt7jF&md5=273ed67eab526a2df6cc9d473a8de427</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1124%2Fmol.114.097220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.114.097220%26sid%3Dliteratum%253Aachs%26aulast%3DGregory%26aufirst%3DK.%2BJ.%26aulast%3DConn%26aufirst%3DP.%2BJ.%26atitle%3DMolecular%2520Insights%2520into%2520Metabotropic%2520Glutamate%2520Receptor%2520Allosteric%2520Modulation%26jtitle%3DMol.%2520Pharmacol.%26date%3D2015%26volume%3D88%26spage%3D188%26epage%3D202%26doi%3D10.1124%2Fmol.114.097220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Colloidal Aggregation Causes Inhibition of G Protein-Coupled Receptors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2406</span>– <span class="NLM_lpage">2414</span>, <span class="refDoi"> DOI: 10.1021/jm301749y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301749y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVCgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2406-2414&author=M.+F.+Sassanoauthor=A.+K.+Doakauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Colloidal+Aggregation+Causes+Inhibition+of+G+Protein-Coupled+Receptors&doi=10.1021%2Fjm301749y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Colloidal aggregation causes inhibition of G protein-coupled receptors</span></div><div class="casAuthors">Sassano, Maria F.; Doak, Allison K.; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2406-2414</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Colloidal aggregation is the dominant mechanism for artifactual inhibition of sol. proteins, and controls against it are now widely deployed.  Conversely, investigating this mechanism for membrane-bound receptors has proven difficult.  Here, the authors investigated the activity of 4 well-characterized aggregators against 3 G protein-coupled receptors (GPCRs) recognizing peptide and protein ligands.  Each of the aggregators was active at micromolar concns. against the 3 GPCRs in cell-based assays.  This activity could be attenuated by either centrifugation of the inhibitor stock soln. or by the addn. of Tween-80 detergent.  In the absence of agonist, the aggregators acted as inverse agonists, consistent with a direct receptor interaction.  Meanwhile, several literature GPCR ligands that resembled aggregators themselves formed colloids, by both phys. and enzymol. tests.  These observations suggest that some GPCRs may be artifactually antagonized by colloidal aggregates, an effect that merits the attention of investigators in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojG9CcuOYoXrVg90H21EOLACvtfcHk0li_-VX7I0zl9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVCgt7s%253D&md5=84fa59f9635330739bf836f8085be5e0</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm301749y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301749y%26sid%3Dliteratum%253Aachs%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DColloidal%2520Aggregation%2520Causes%2520Inhibition%2520of%2520G%2520Protein-Coupled%2520Receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2406%26epage%3D2414%26doi%3D10.1021%2Fjm301749y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Owen, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesh, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nedyalkova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, M. S.</span></span> <span> </span><span class="NLM_article-title">Colloidal Drug Formulations Can Explain “Bell-shaped” Concentration-Response Curves</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">777</span>– <span class="NLM_lpage">784</span>, <span class="refDoi"> DOI: 10.1021/cb4007584</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4007584" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVCgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=777-784&author=S.+C.+Owenauthor=A.+K.+Doakauthor=A.+N.+Ganeshauthor=L.+Nedyalkovaauthor=C.+K.+McLaughlinauthor=B.+K.+Shoichetauthor=M.+S.+Shoichet&title=Colloidal+Drug+Formulations+Can+Explain+%E2%80%9CBell-shaped%E2%80%9D+Concentration-Response+Curves&doi=10.1021%2Fcb4007584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Colloidal Drug Formulations Can Explain "Bell-Shaped" Concentration-Response Curves</span></div><div class="casAuthors">Owen, Shawn C.; Doak, Allison K.; Ganesh, Ahil N.; Nedyalkova, Lyudmila; McLaughlin, Christopher K.; Shoichet, Brian K.; Shoichet, Molly S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">777-784</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug efficacy does not always increase sigmoidally with concn., which has puzzled the community for decades.  Unlike std. sigmoidal curves, bell-shaped concn.-response curves suggest more complex biol. effects, such as multiple-binding sites or multiple targets.  Here, we investigate a phys. property-based mechanism for bell-shaped curves.  Beginning with the observation that some drugs form colloidal aggregates at relevant concns., we detd. concn.-response curves for three aggregating anticancer drugs, formulated both as colloids and as free monomer.  Colloidal formulations exhibited bell-shaped curves, losing activity at higher concns., while monomeric formulations gave typical sigmoidal curves, sustaining a plateau of max. activity.  Inverting the question, we next asked if mols. with bell-shaped curves, reported in the literature, form colloidal aggregates at relevant concns.  We selected 12 mols. reported to have bell-shaped concn.-response curves and found that five of these formed colloids.  To understand the mechanism behind the loss of activity at concns. where colloids are present, we investigated the diffusion of colloid-forming dye Evans blue into cells.  We found that colloidal species are excluded from cells, which may explain the mechanism behind toxicol. screens that use Evans blue, Trypan blue, and related dyes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg5AlTuU9YnrVg90H21EOLACvtfcHk0li_-VX7I0zl9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVCgtA%253D%253D&md5=8cb90ab5f6348326d08ccfcf5377b1f3</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fcb4007584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4007584%26sid%3Dliteratum%253Aachs%26aulast%3DOwen%26aufirst%3DS.%2BC.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DGanesh%26aufirst%3DA.%2BN.%26aulast%3DNedyalkova%26aufirst%3DL.%26aulast%3DMcLaughlin%26aufirst%3DC.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DM.%2BS.%26atitle%3DColloidal%2520Drug%2520Formulations%2520Can%2520Explain%2520%25E2%2580%259CBell-shaped%25E2%2580%259D%2520Concentration-Response%2520Curves%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D777%26epage%3D784%26doi%3D10.1021%2Fcb4007584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vijayaraghavan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajan, S.</span></span> <span> </span><span class="NLM_article-title">Docking, Synthesis and Antimalarial Activity of Novel 4-Anilinoquinoline Derivatives</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1693</span>– <span class="NLM_lpage">1697</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fj.bmcl.2017.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=28318947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslKrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=1693-1697&author=S.+Vijayaraghavanauthor=S.+Mahajan&title=Docking%2C+Synthesis+and+Antimalarial+Activity+of+Novel+4-Anilinoquinoline+Derivatives&doi=10.1016%2Fj.bmcl.2017.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Docking, synthesis and antimalarial activity of novel 4-anilinoquinoline derivatives</span></div><div class="casAuthors">Vijayaraghavan, Shilpa; Mahajan, Supriya</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1693-1697</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of 4-anilinoquinoline triazine derivs. were designed, synthesized and screened for in vivo antimalarial activity against a chloroquine-sensitive strain of Plasmodium berghei.  The compds. were further subjected to in vitro antimalarial activity against chloroquine-resistant W2 strain of Plasmodium falciparum and β-haematin inhibition studies.  All the compds. exhibited in vivo antimalarial activity better than that shown by the std. drug, chloroquine.  Twelve out of fifteen compds. showed better inhibition than that of chloroquine against chloroquine-resistant W2 strain of Plasmodium falciparum.  Ten compds. showed β-haematin inhibition, better than that of chloroquine, with IC50 values in the range of 18-25 μM.  One compd., 3k, was better than artemisinin against W2 strain of Plasmodium falciparum and also displayed the best β-haematin inhibitory activity, thereby becoming eligible to be explored as a potential lead for antimalarial chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb2Xtd3d_0F7Vg90H21EOLACvtfcHk0lizWmL6guaCrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslKrsb4%253D&md5=c1edcedab3fdfcf4adfe30bbb3bb02f6</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DVijayaraghavan%26aufirst%3DS.%26aulast%3DMahajan%26aufirst%3DS.%26atitle%3DDocking%252C%2520Synthesis%2520and%2520Antimalarial%2520Activity%2520of%2520Novel%25204-Anilinoquinoline%2520Derivatives%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D1693%26epage%3D1697%26doi%3D10.1016%2Fj.bmcl.2017.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span> <span> </span><span class="NLM_article-title">Cationic Palladium Complex Catalyzed Diastereo- and Enantioselective Tandem Annulation of 2-Formylarylboronic Acids with Allenoates</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4366</span>– <span class="NLM_lpage">4369</span>, <span class="refDoi"> DOI: 10.1021/ol901714d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol901714d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVOjurnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=4366-4369&author=X.+Yuauthor=X.+Lu&title=Cationic+Palladium+Complex+Catalyzed+Diastereo-+and+Enantioselective+Tandem+Annulation+of+2-Formylarylboronic+Acids+with+Allenoates&doi=10.1021%2Fol901714d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Cationic Palladium Complex Catalyzed Diastereo- and Enantioselective Tandem Annulation of 2-Formylarylboronic Acids with Allenoates</span></div><div class="casAuthors">Yu, Xufen; Lu, Xiyan</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4366-4369</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cationic palladium complex [(dppp)Pd(H2O)2]2+(BF4-)2 catalyzed tandem annulation reactions of 2-formylarylboronic acids and allenoates produce the indenol derivs., e.g., I, diastereo- and enantioselectively in a highly efficient way.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiunsyzciEG7Vg90H21EOLACvtfcHk0lizWmL6guaCrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVOjurnK&md5=3307677788b9c8e46f3d7dddf90b0c1a</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fol901714d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol901714d%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26atitle%3DCationic%2520Palladium%2520Complex%2520Catalyzed%2520Diastereo-%2520and%2520Enantioselective%2520Tandem%2520Annulation%2520of%25202-Formylarylboronic%2520Acids%2520with%2520Allenoates%26jtitle%3DOrg.%2520Lett.%26date%3D2009%26volume%3D11%26spage%3D4366%26epage%3D4369%26doi%3D10.1021%2Fol901714d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revankar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">Parallel Functional Activity Profiling Reveals Valvulopathogens Are Potent 5-Hydroxytryptamine(2B) Receptor Agonists: Implications for Drug Safety Assessment</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">710</span>– <span class="NLM_lpage">722</span>, <span class="refDoi"> DOI: 10.1124/mol.109.058057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1124%2Fmol.109.058057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=19570945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1eitrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2009&pages=710-722&author=X.-P.+Huangauthor=V.+Setolaauthor=P.+N.+Yadavauthor=J.+A.+Allenauthor=S.+C.+Roganauthor=B.+J.+Hansonauthor=C.+Revankarauthor=M.+Robersauthor=C.+Doucetteauthor=B.+L.+Roth&title=Parallel+Functional+Activity+Profiling+Reveals+Valvulopathogens+Are+Potent+5-Hydroxytryptamine%282B%29+Receptor+Agonists%3A+Implications+for+Drug+Safety+Assessment&doi=10.1124%2Fmol.109.058057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Parallel functional activity profiling reveals valvulopathogens are potent 5-hydroxytryptamine2B receptor agonists: implications for drug safety assessment</span></div><div class="casAuthors">Huang, Xi-Ping; Setola, Vincent; Yadav, Prem N.; Allen, John A.; Rogan, Sarah C.; Hanson, Bonnie J.; Revankar, Chetana; Robers, Matt; Doucette, Chris; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">710-722</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Drug-induced valvular heart disease (VHD) is a serious side effect of a few medications, including some that are on the market.  Pharmacol. studies of VHD-assocd. medications (e.g., fenfluramine, pergolide, methysergide, and cabergoline) have revealed that they and/or their metabolites are potent 5-hydroxytryptamine2B (5-HT2B) receptor agonists.  We have shown that activation of 5-HT2B receptors on human heart valve interstitial cells in vitro induces a proliferative response reminiscent of the fibrosis that typifies VHD.  To identify current or future drugs that might induce VHD, we screened approx. 2200 U.S. Food and Drug Administration (FDA)-approved or investigational medications to identify 5-HT2B receptor agonists, using calcium-based high-throughput screening.  Of these 2200 compds., 27 were 5-HT2B receptor agonists (hits); 14 of these had previously been identified as 5-HT2B receptor agonists, including seven bona fide valvulopathogens.  Six of the hits (guanfacine, quinidine, xylometazoline, oxymetazoline, fenoldopam, and ropinirole) are approved medications.  Twenty-three of the hits were then "functionally profiled" (i.e., assayed in parallel for 5-HT2B receptor agonism using multiple readouts to test for functional selectivity).  In these assays, the known valvulopathogens were efficacious at concns. as low as 30 nM, whereas the other compds. were less so.  Hierarchical clustering anal. of the pEC50 data revealed that ropinirole (which is not assocd. with valvulopathy) was clearly segregated from known valvulopathogens.  Taken together, our data demonstrate that patterns of 5-HT2B receptor functional selectivity might be useful for identifying compds. likely to induce valvular heart disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6tCEte6PXtLVg90H21EOLACvtfcHk0lizWmL6guaCrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1eitrvF&md5=48f3f3a220c542bc42b7378cc79d73d0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1124%2Fmol.109.058057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.109.058057%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DYadav%26aufirst%3DP.%2BN.%26aulast%3DAllen%26aufirst%3DJ.%2BA.%26aulast%3DRogan%26aufirst%3DS.%2BC.%26aulast%3DHanson%26aufirst%3DB.%2BJ.%26aulast%3DRevankar%26aufirst%3DC.%26aulast%3DRobers%26aufirst%3DM.%26aulast%3DDoucette%26aufirst%3DC.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DParallel%2520Functional%2520Activity%2520Profiling%2520Reveals%2520Valvulopathogens%2520Are%2520Potent%25205-Hydroxytryptamine%25282B%2529%2520Receptor%2520Agonists%253A%2520Implications%2520for%2520Drug%2520Safety%2520Assessment%26jtitle%3DMol.%2520Pharmacol.%26date%3D2009%26volume%3D76%26spage%3D710%26epage%3D722%26doi%3D10.1124%2Fmol.109.058057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agegnehu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tribo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, R. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">Structural Determinants of 5-HT2B Receptor Activation and Biased Agonism</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">787</span>– <span class="NLM_lpage">796</span>, <span class="refDoi"> DOI: 10.1038/s41594-018-0116-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fs41594-018-0116-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=30127358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFGjtLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=787-796&author=J.+D.+McCorvyauthor=D.+Wackerauthor=S.+Wangauthor=B.+Agegnehuauthor=J.+Liuauthor=K.+Lansuauthor=A.+R.+Triboauthor=R.+H.+J.+Olsenauthor=T.+Cheauthor=J.+Jinauthor=B.+L.+Roth&title=Structural+Determinants+of+5-HT2B+Receptor+Activation+and+Biased+Agonism&doi=10.1038%2Fs41594-018-0116-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Structural determinants of 5-HT2B receptor activation and biased agonism</span></div><div class="casAuthors">McCorvy, John D.; Wacker, Daniel; Wang, Sheng; Agegnehu, Bemnat; Liu, Jing; Lansu, Katherine; Tribo, Alexandra R.; Olsen, Reid H. J.; Che, Tao; Jin, Jian; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">787-796</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Serotonin (5-hydroxytryptamine; 5-HT) receptors modulate a variety of physiol. processes ranging from perception, cognition and emotion to vascular and smooth muscle contraction, platelet aggregation, gastrointestinal function and reprodn.  Drugs that interact with 5-HT receptors effectively treat diseases as diverse as migraine headaches, depression and obesity.  Here we present four structures of a prototypical serotonin receptor-the human 5-HT2B receptor-in complex with chem. and pharmacol. diverse drugs, including methysergide, methylergonovine, lisuride and LY266097.  A detailed anal. of these structures complemented by comprehensive interrogation of signaling illuminated key structural determinants essential for activation.  Addnl. structure-guided mutagenesis expts. revealed binding pocket residues that were essential for agonist-mediated biased signaling and β-arrestin2 translocation.  Given the importance of 5-HT receptors for a large no. of therapeutic indications, insights derived from these studies should accelerate the design of safer and more effective medications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwC5QTOdEZhLVg90H21EOLACvtfcHk0litG7rQccNZDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFGjtLjM&md5=93be9401bc680249499fa3af5d40f395</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fs41594-018-0116-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-018-0116-7%26sid%3Dliteratum%253Aachs%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DAgegnehu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLansu%26aufirst%3DK.%26aulast%3DTribo%26aufirst%3DA.%2BR.%26aulast%3DOlsen%26aufirst%3DR.%2BH.%2BJ.%26aulast%3DChe%26aufirst%3DT.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DStructural%2520Determinants%2520of%25205-HT2B%2520Receptor%2520Activation%2520and%2520Biased%2520Agonism%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D787%26epage%3D796%26doi%3D10.1038%2Fs41594-018-0116-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stauch, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gati, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batyuk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slocum, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brehm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madsen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandner, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiriaeva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, R. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tribo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yous, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weierstall, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Ligand Recognition at the Human MT1 Melatonin Receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>569</i></span>,  <span class="NLM_fpage">284</span>– <span class="NLM_lpage">288</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1141-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fs41586-019-1141-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=31019306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVSkur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=569&publication_year=2019&pages=284-288&author=B.+Stauchauthor=L.+C.+Johanssonauthor=J.+D.+McCorvyauthor=N.+Patelauthor=G.+W.+Hanauthor=X.-P.+Huangauthor=C.+Gatiauthor=A.+Batyukauthor=S.+T.+Slocumauthor=A.+Ishchenkoauthor=W.+Brehmauthor=T.+A.+Whiteauthor=N.+Michaelianauthor=C.+Madsenauthor=L.+Zhuauthor=T.+D.+Grantauthor=J.+M.+Grandnerauthor=A.+Shiriaevaauthor=R.+H.+J.+Olsenauthor=A.+R.+Triboauthor=S.+Yousauthor=R.+C.+Stevensauthor=U.+Weierstallauthor=V.+Katritchauthor=B.+L.+Rothauthor=W.+Liuauthor=V.+Cherezov&title=Structural+Basis+of+Ligand+Recognition+at+the+Human+MT1+Melatonin+Receptor&doi=10.1038%2Fs41586-019-1141-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of ligand recognition at the human MT1 melatonin receptor</span></div><div class="casAuthors">Stauch, Benjamin; Johansson, Linda C.; McCorvy, John D.; Patel, Nilkanth; Han, Gye Won; Huang, Xi-Ping; Gati, Cornelius; Batyuk, Alexander; Slocum, Samuel T.; Ishchenko, Andrii; Brehm, Wolfgang; White, Thomas A.; Michaelian, Nairie; Madsen, Caleb; Zhu, Lan; Grant, Thomas D.; Grandner, Jessica M.; Shiriaeva, Anna; Olsen, Reid H. J.; Tribo, Alexandra R.; Yous, Said; Stevens, Raymond C.; Weierstall, Uwe; Katritch, Vsevolod; Roth, Bryan L.; Liu, Wei; Cherezov, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">569</span>
        (<span class="NLM_cas:issue">7755</span>),
    <span class="NLM_cas:pages">284-288</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Melatonin (N-acetyl-5-methoxytryptamine) is a neurohormone that maintains circadian rhythms by synchronization to environmental cues and is involved in diverse physiol. processes such as the regulation of blood pressure and core body temp., oncogenesis, and immune function.  Melatonin is formed in the pineal gland in a light-regulated manner by enzymic conversion from 5-hydroxytryptamine (5-HT or serotonin), and modulates sleep and wakefulness by activating two high-affinity G-protein-coupled receptors, type 1A (MT1) and type 1B (MT2).  Shift work, travel, and ubiquitous artificial lighting can disrupt natural circadian rhythms; as a result, sleep disorders affect a substantial population in modern society and pose a considerable economic burden.  Over-the-counter melatonin is widely used to alleviate jet lag and as a safer alternative to benzodiazepines and other sleeping aids, and is one of the most popular supplements in the United States.  Here, we present high-resoln. room-temp. X-ray free electron laser (XFEL) structures of MT1 in complex with four agonists: the insomnia drug ramelteon, two melatonin analogs, and the mixed melatonin-serotonin antidepressant agomelatine.  The structure of MT2 is described in an accompanying paper.  Although the MT1 and 5-HT receptors have similar endogenous ligands, and agomelatine acts on both receptors, the receptors differ markedly in the structure and compn. of their ligand pockets; in MT1, access to the ligand pocket is tightly sealed from solvent by extracellular loop 2, leaving only a narrow channel between transmembrane helixes IV and V that connects it to the lipid bilayer.  The binding site is extremely compact, and ligands interact with MT1 mainly by strong arom. stacking with Phe179 and auxiliary hydrogen bonds with Asn162 and Gln181.  Our structures provide an unexpected example of atypical ligand entry for a non-lipid receptor, lay the mol. foundation of ligand recognition by melatonin receptors, and will facilitate the design of future tool compds. and therapeutic agents, while their comparison to 5-HT receptors yields insights into the evolution and polypharmacol. of G-protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplFhqB4FkpoLVg90H21EOLACvtfcHk0litG7rQccNZDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVSkur0%253D&md5=61cecfc7a75118449d015c5b695d2f8f</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1141-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1141-3%26sid%3Dliteratum%253Aachs%26aulast%3DStauch%26aufirst%3DB.%26aulast%3DJohansson%26aufirst%3DL.%2BC.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DGati%26aufirst%3DC.%26aulast%3DBatyuk%26aufirst%3DA.%26aulast%3DSlocum%26aufirst%3DS.%2BT.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DBrehm%26aufirst%3DW.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DMichaelian%26aufirst%3DN.%26aulast%3DMadsen%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DGrant%26aufirst%3DT.%2BD.%26aulast%3DGrandner%26aufirst%3DJ.%2BM.%26aulast%3DShiriaeva%26aufirst%3DA.%26aulast%3DOlsen%26aufirst%3DR.%2BH.%2BJ.%26aulast%3DTribo%26aufirst%3DA.%2BR.%26aulast%3DYous%26aufirst%3DS.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DStructural%2520Basis%2520of%2520Ligand%2520Recognition%2520at%2520the%2520Human%2520MT1%2520Melatonin%2520Receptor%26jtitle%3DNature%26date%3D2019%26volume%3D569%26spage%3D284%26epage%3D288%26doi%3D10.1038%2Fs41586-019-1141-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauch, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batyuk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gati, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slocum, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandner, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, R. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tribo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaare, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zatsepin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weierstall, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yous, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span> <span> </span><span class="NLM_article-title">XFEL Structures of the Human MT2 Melatonin Receptor Reveal the Basis of Subtype Selectivity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>569</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1144-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1038%2Fs41586-019-1144-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=31019305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVSkuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=569&publication_year=2019&pages=289-292&author=L.+C.+Johanssonauthor=B.+Stauchauthor=J.+D.+McCorvyauthor=G.+W.+Hanauthor=N.+Patelauthor=X.-P.+Huangauthor=A.+Batyukauthor=C.+Gatiauthor=S.+T.+Slocumauthor=C.+Liauthor=J.+M.+Grandnerauthor=S.+Haoauthor=R.+H.+J.+Olsenauthor=A.+R.+Triboauthor=S.+Zaareauthor=L.+Zhuauthor=N.+A.+Zatsepinauthor=U.+Weierstallauthor=S.+Yousauthor=R.+C.+Stevensauthor=W.+Liuauthor=B.+L.+Rothauthor=V.+Katritchauthor=V.+Cherezov&title=XFEL+Structures+of+the+Human+MT2+Melatonin+Receptor+Reveal+the+Basis+of+Subtype+Selectivity&doi=10.1038%2Fs41586-019-1144-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">XFEL structures of the human MT2 melatonin receptor reveal the basis of subtype selectivity</span></div><div class="casAuthors">Johansson, Linda C.; Stauch, Benjamin; McCorvy, John D.; Han, Gye Won; Patel, Nilkanth; Huang, Xi-Ping; Batyuk, Alexander; Gati, Cornelius; Slocum, Samuel T.; Li, Chufeng; Grandner, Jessica M.; Hao, Shuming; Olsen, Reid H. J.; Tribo, Alexandra R.; Zaare, Sahba; Zhu, Lan; Zatsepin, Nadia A.; Weierstall, Uwe; Yous, Said; Stevens, Raymond C.; Liu, Wei; Roth, Bryan L.; Katritch, Vsevolod; Cherezov, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">569</span>
        (<span class="NLM_cas:issue">7755</span>),
    <span class="NLM_cas:pages">289-292</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The human MT1 and MT2 melatonin receptors1,2 are G-protein-coupled receptors (GPCRs) that help to regulate circadian rhythm and sleep patterns3.  Drug development efforts have targeted both receptors for the treatment of insomnia, circadian rhythm and mood disorders, and cancer3, and MT2 has also been implicated in type 2 diabetes4,5.  Here we report X-ray free electron laser (XFEL) structures of the human MT2 receptor in complex with the agonists 2-phenylmelatonin (2-PMT) and ramelteon6 at resolns. of 2.8 Å and 3.3 Å, resp., along with two structures of function-related mutants: H2085.46A (superscripts represent the Ballesteros-Weinstein residue numbering nomenclature7) and N862.50D, obtained in complex with 2-PMT.  Comparison of the structures of MT2 with a published structure8 of MT1 reveals that, despite conservation of the orthosteric ligand-binding site residues, there are notable conformational variations as well as differences in [3H]melatonin dissocn. kinetics that provide insights into the selectivity between melatonin receptor subtypes.  A membrane-buried lateral ligand entry channel is obsd. in both MT1 and MT2, but in addn. the MT2 structures reveal a narrow opening towards the solvent in the extracellular part of the receptor.  We provide functional and kinetic data that support a prominent role for intramembrane ligand entry in both receptors, and suggest that there might also be an extracellular entry path in MT2.  Our findings contribute to a mol. understanding of melatonin receptor subtype selectivity and ligand access modes, which are essential for the design of highly selective melatonin tool compds. and therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrePfxnJ1rXDbVg90H21EOLACvtfcHk0litG7rQccNZDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVSkuro%253D&md5=6627fc1e26c873f6a14a7a4d4bf3a346</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1144-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1144-0%26sid%3Dliteratum%253Aachs%26aulast%3DJohansson%26aufirst%3DL.%2BC.%26aulast%3DStauch%26aufirst%3DB.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DBatyuk%26aufirst%3DA.%26aulast%3DGati%26aufirst%3DC.%26aulast%3DSlocum%26aufirst%3DS.%2BT.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DGrandner%26aufirst%3DJ.%2BM.%26aulast%3DHao%26aufirst%3DS.%26aulast%3DOlsen%26aufirst%3DR.%2BH.%2BJ.%26aulast%3DTribo%26aufirst%3DA.%2BR.%26aulast%3DZaare%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DZatsepin%26aufirst%3DN.%2BA.%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DYous%26aufirst%3DS.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DXFEL%2520Structures%2520of%2520the%2520Human%2520MT2%2520Melatonin%2520Receptor%2520Reveal%2520the%2520Basis%2520of%2520Subtype%2520Selectivity%26jtitle%3DNature%26date%3D2019%26volume%3D569%26spage%3D289%26epage%3D292%26doi%3D10.1038%2Fs41586-019-1144-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Longo, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavran, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leahy, D. J.</span></span> <span> </span><span class="NLM_article-title">Transient Mammalian Cell Transfection with Polyethylenimine (PEI)</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">227</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1016/b978-0-12-418687-3.00018-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2Fb978-0-12-418687-3.00018-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=24011049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslGgt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2013&pages=227-240&author=P.+A.+Longoauthor=J.+M.+Kavranauthor=M.-S.+Kimauthor=D.+J.+Leahy&title=Transient+Mammalian+Cell+Transfection+with+Polyethylenimine+%28PEI%29&doi=10.1016%2Fb978-0-12-418687-3.00018-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Transient mammalian cell transfection with polyethyylenimine (PEI)</span></div><div class="casAuthors">Longo, Patti A.; Kavran, Jennifer M.; Kim, Min-Sung; Leahy, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">Laboratory Methods in Enzymology: DNA</span>),
    <span class="NLM_cas:pages">227-240</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Std. protein expression systems, such as E. coli, often fail to produce folded, monodisperse, or functional eukaryotic proteins (see Small-scale Expression of Proteins in E. coli).  The expression of these proteins is greatly benefited by using a eukaryotic system, such as mammalian cells, that contains the appropriate folding and posttranslational machinery.  Here, we describe methods for both small- and large-scale transient expression in mammalian cells using polyethylenimine (PEI).  We find this procedure to be more cost-effective and quicker than the more traditional route of generating stable cell lines.  First, optimal transfection conditions are detd. on a small-scale, using adherent cells.  These conditions are then translated for use in large-scale suspension cultures.  For further details on generating stable cell lines please (see Rapid creation of stable mammalian cell lines for regulated expression of proteins using the Gateway Recombination Cloning Technol. and Flp-InT-REx lines or Generating mammalian stable cell lines by electroporation).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDaeKbQiaGD7Vg90H21EOLACvtfcHk0lhE_2ougwyA9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslGgt7zP&md5=0548d9c7c95a04d40aeb7a2ac5e14d45</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fb978-0-12-418687-3.00018-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fb978-0-12-418687-3.00018-5%26sid%3Dliteratum%253Aachs%26aulast%3DLongo%26aufirst%3DP.%2BA.%26aulast%3DKavran%26aufirst%3DJ.%2BM.%26aulast%3DKim%26aufirst%3DM.-S.%26aulast%3DLeahy%26aufirst%3DD.%2BJ.%26atitle%3DTransient%2520Mammalian%2520Cell%2520Transfection%2520with%2520Polyethylenimine%2520%2528PEI%2529%26jtitle%3DMethods%2520Enzymol.%26date%3D2013%26volume%3D529%26spage%3D227%26epage%3D240%26doi%3D10.1016%2Fb978-0-12-418687-3.00018-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship Between the Inhibition Constant (K1) and the Concentration of Inhibitor Which Causes 50 Per Cent Inhibition (I50) of an Enzymatic Reaction</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>– <span class="NLM_lpage">3108</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=C.+Chengauthor=W.+H.+Prusoff&title=Relationship+Between+the+Inhibition+Constant+%28K1%29+and+the+Concentration+of+Inhibitor+Which+Causes+50+Per+Cent+Inhibition+%28I50%29+of+an+Enzymatic+Reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0lhE_2ougwyA9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DC.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520Between%2520the%2520Inhibition%2520Constant%2520%2528K1%2529%2520and%2520the%2520Concentration%2520of%2520Inhibitor%2520Which%2520Causes%252050%2520Per%2520Cent%2520Inhibition%2520%2528I50%2529%2520of%2520an%2520Enzymatic%2520Reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i95"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00869">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_14635"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00869">10.1021/acs.jmedchem.9b00869</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline"><sup>1</sup>H NMR spectra of compounds <b>46</b>, <b>71</b>, <b>77</b>, and <b>78</b>; <sup>13</sup>C NMR spectra of compound <b>71</b>; proton concentration–response curves at human GPR68 in the absence and presence of increasing concentrations of test compounds; proton concentration–response curves of compounds <b>20</b>, <b>46</b>, <b>77</b>, and <b>78</b> at GPR4 and GPR65; heat map showing off-target binding activities of selected lead compounds at 48 common GPCRs, ion channels, and transporters; functional activities of compounds with binding affinity values lower than 300 nM (p<i>K</i><sub>i</sub> > 6.50); off-target agonist activity at human GPCRome; functional characterization of the compounds identified from the GPCRome screening; proton concentration–response curves of compounds <b>1</b>, <b>46</b>, <b>63</b>, <b>71</b>, <b>77</b>, and <b>78</b> at mouse GPR68; effects of inactive analogues on the GPR68 PAM <b>71</b>; allosteric modulation of GPR68 proton activity: and implications for pH-dependent agonism and inverse agonism (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_001.pdf">jm9b00869_si_001.pdf (6.05 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00869/suppl_file/jm9b00869_si_002.csv">jm9b00869_si_002.csv (6.7 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00869&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-16%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00869%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00869" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67991fbadc4c3c9a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
