
<html lang="en"     class="pb-page"  data-request-id="75c38a95-55ff-4878-8b83-8a5cfd61392c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2017.60.issue-12;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.6b01862;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Design and Discovery of N-(2-Methyl-5′-morpholino-6′-((tetrahydro-2H-pyran-4-yl)oxy)-[3,3′-bipyridin]-5-yl)-3-(trifluoromethyl)benzamide (RAF709): A Potent, Selective, and Efficacious RAF Inhibitor Targeting RAS Mutant Cancers" /></meta><meta name="dc.Creator" content="Gisele A.  Nishiguchi" /></meta><meta name="dc.Creator" content="Alice  Rico" /></meta><meta name="dc.Creator" content="Huw  Tanner" /></meta><meta name="dc.Creator" content="Robert J.  Aversa" /></meta><meta name="dc.Creator" content="Benjamin R.  Taft" /></meta><meta name="dc.Creator" content="Sharadha  Subramanian" /></meta><meta name="dc.Creator" content="Lina  Setti" /></meta><meta name="dc.Creator" content="Matthew T.  Burger" /></meta><meta name="dc.Creator" content="Lifeng  Wan" /></meta><meta name="dc.Creator" content="Victoriano  Tamez" /></meta><meta name="dc.Creator" content="Aaron  Smith" /></meta><meta name="dc.Creator" content="Yan  Lou" /></meta><meta name="dc.Creator" content="Paul A.  Barsanti" /></meta><meta name="dc.Creator" content="Brent A.  Appleton" /></meta><meta name="dc.Creator" content="Mulugeta  Mamo" /></meta><meta name="dc.Creator" content="Laura  Tandeske" /></meta><meta name="dc.Creator" content="Ina  Dix" /></meta><meta name="dc.Creator" content="John E.  Tellew" /></meta><meta name="dc.Creator" content="Shenlin  Huang" /></meta><meta name="dc.Creator" content="Lesley A.  Mathews Griner" /></meta><meta name="dc.Creator" content="Vesselina G.  Cooke" /></meta><meta name="dc.Creator" content="Anne  Van Abbema" /></meta><meta name="dc.Creator" content="Hanne  Merritt" /></meta><meta name="dc.Creator" content="Sylvia  Ma" /></meta><meta name="dc.Creator" content="Kalyani  Gampa" /></meta><meta name="dc.Creator" content="Fei  Feng" /></meta><meta name="dc.Creator" content="Jing  Yuan" /></meta><meta name="dc.Creator" content="Yingyun  Wang" /></meta><meta name="dc.Creator" content="Jacob R.  Haling" /></meta><meta name="dc.Creator" content="Sepideh  Vaziri" /></meta><meta name="dc.Creator" content="Mohammad  Hekmat-Nejad" /></meta><meta name="dc.Creator" content="Johanna M.  Jansen" /></meta><meta name="dc.Creator" content="Valery  Polyakov" /></meta><meta name="dc.Creator" content="Richard  Zang" /></meta><meta name="dc.Creator" content="Vijay  Sethuraman" /></meta><meta name="dc.Creator" content="Payman  Amiri" /></meta><meta name="dc.Creator" content="Mallika  Singh" /></meta><meta name="dc.Creator" content="Emma  Lees" /></meta><meta name="dc.Creator" content="Wenlin  Shao" /></meta><meta name="dc.Creator" content="Darrin D.  Stuart" /></meta><meta name="dc.Creator" content="Michael P.  Dillon" /></meta><meta name="dc.Creator" content="Savithri  Ramurthy" /></meta><meta name="dc.Description" content="RAS oncogenes have been implicated in &gt;30% of human cancers, all representing high unmet medical need. The exquisite dependency on CRAF kinase in KRAS mutant tumors has been established in genetica..." /></meta><meta name="Description" content="RAS oncogenes have been implicated in &gt;30% of human cancers, all representing high unmet medical need. The exquisite dependency on CRAF kinase in KRAS mutant tumors has been established in genetica..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 8, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01862" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01862" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01862" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01862" /></link>
        
    
    

<title>Design and Discovery of N-(2-Methyl-5′-morpholino-6′-((tetrahydro-2H-pyran-4-yl)oxy)-[3,3′-bipyridin]-5-yl)-3-(trifluoromethyl)benzamide (RAF709): A Potent, Selective, and Efficacious RAF Inhibitor Targeting RAS Mutant Cancers | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01862" /></meta><meta property="og:title" content="Design and Discovery of N-(2-Methyl-5′-morpholino-6′-((tetrahydro-2H-pyran-4-yl)oxy)-[3,3′-bipyridin]-5-yl)-3-(trifluoromethyl)benzamide (RAF709): A Potent, Selective, and Efficacious RAF Inhibitor Targeting RAS Mutant Cancers" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0017.jpeg" /></meta><meta property="og:description" content="RAS oncogenes have been implicated in &gt;30% of human cancers, all representing high unmet medical need. The exquisite dependency on CRAF kinase in KRAS mutant tumors has been established in genetically engineered mouse models and human tumor cells. To date, many small molecule approaches are under investigation to target CRAF, yet kinase-selective and cellular potent inhibitors remain challenging to identify. Herein, we describe 14 (RAF709) [Aversa,  Biaryl amide compounds as kinase inhibitors and their preparation. WO 2014151616, 2014], a selective B/C RAF inhibitor, which was developed through a hypothesis-driven approach focusing on drug-like properties. A key challenge encountered in the medicinal chemistry campaign was maintaining a balance between good solubility and potent cellular activity (suppression of pMEK and proliferation) in KRAS mutant tumor cell lines. We investigated the small molecule crystal structure of lead molecule 7 and hypothesized that disruption of the crystal packing would improve solubility, which led to a change from N-methylpyridone to a tetrahydropyranyl oxy-pyridine derivative. 14 proved to be soluble, kinase selective, and efficacious in a KRAS mutant xenograft model." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01862"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01862">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01862&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01862&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01862&amp;href=/doi/10.1021/acs.jmedchem.6b01862" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 4869-4881</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b00060" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b00095" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design and Discovery of <i>N</i>-(2-Methyl-5′-morpholino-6′-((tetrahydro-2<i>H</i>-pyran-4-yl)oxy)-[3,3′-bipyridin]-5-yl)-3-(trifluoromethyl)benzamide (RAF709): A Potent, Selective, and Efficacious RAF Inhibitor Targeting RAS Mutant Cancers</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gisele+A.++Nishiguchi">Gisele A. Nishiguchi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alice++Rico">Alice Rico</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Huw++Tanner">Huw Tanner</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Aversa">Robert J. Aversa</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-9873-4381" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Benjamin+R.++Taft">Benjamin R. Taft</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sharadha++Subramanian">Sharadha Subramanian</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lina++Setti">Lina Setti</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+T.++Burger">Matthew T. Burger</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lifeng++Wan">Lifeng Wan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Victoriano++Tamez">Victoriano Tamez</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Aaron++Smith">Aaron Smith</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yan++Lou">Yan Lou</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paul+A.++Barsanti">Paul A. Barsanti</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Brent+A.++Appleton">Brent A. Appleton</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mulugeta++Mamo">Mulugeta Mamo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Laura++Tandeske">Laura Tandeske</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ina++Dix">Ina Dix</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John+E.++Tellew">John E. Tellew</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shenlin++Huang">Shenlin Huang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lesley+A.++Mathews+Griner">Lesley A. Mathews Griner</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Vesselina+G.++Cooke">Vesselina G. Cooke</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anne++Van+Abbema">Anne Van Abbema</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hanne++Merritt">Hanne Merritt</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sylvia++Ma">Sylvia Ma</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kalyani++Gampa">Kalyani Gampa</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fei++Feng">Fei Feng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Yuan">Jing Yuan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yingyun++Wang">Yingyun Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jacob+R.++Haling">Jacob R. Haling</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sepideh++Vaziri">Sepideh Vaziri</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mohammad++Hekmat-Nejad">Mohammad Hekmat-Nejad</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Johanna+M.++Jansen">Johanna M. Jansen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Valery++Polyakov">Valery Polyakov</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Richard++Zang">Richard Zang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Vijay++Sethuraman">Vijay Sethuraman</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Payman++Amiri">Payman Amiri</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mallika++Singh">Mallika Singh</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Emma++Lees">Emma Lees</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wenlin++Shao">Wenlin Shao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Darrin+D.++Stuart">Darrin D. Stuart</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+P.++Dillon">Michael P. Dillon</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Savithri++Ramurthy">Savithri Ramurthy</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-2444-5309" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Global Discovery Chemistry, Novartis Institutes for BioMedical Research, Novartis Pharma AG, Werk Klybeck, Postfach, CH-4002 Basel, Switzerland</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Genomics Institute of the Novartis Research Foundation, 10675 John Hopkins Drive, San Diego, California 92121, United States</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Oncology, Novartis Institutes for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">Oncology, Novartis Institutes for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: 510-879-9463. Fax: 510-923-3360. E-mail: <a href="/cdn-cgi/l/email-protection#b6c5d7c0dfc2dec4df98c4d7dbc3c4c2decff6d8d9c0d7c4c2dfc598d5d9db"><span class="__cf_email__" data-cfemail="4c3f2d3a2538243e25623e2d21393e3824350c22233a2d3e38253f622f2321">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01862&amp;href=/doi/10.1021%2Facs.jmedchem.6b01862" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 4869–4881</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 30, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>24 January 2017</li><li><span class="item_label"><b>Published</b> online</span>8 June 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 June 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01862" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01862</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4869%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DGisele%2BA.%2BNishiguchi%252C%2BAlice%2BRico%252C%2BHuw%2BTanner%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D12%26contentID%3Dacs.jmedchem.6b01862%26title%3DDesign%2Band%2BDiscovery%2Bof%2BN-%25282-Methyl-5%25E2%2580%25B2-morpholino-6%25E2%2580%25B2-%2528%2528tetrahydro-2H-pyran-4-yl%2529oxy%2529-%255B3%252C3%25E2%2580%25B2-bipyridin%255D-5-yl%2529-3-%2528trifluoromethyl%2529benzamide%2B%2528RAF709%2529%253A%2BA%2BPotent%252C%2BSelective%252C%2Band%2BEfficacious%2BRAF%2BInhibitor%2BTargeting%2BRAS%2BMutant%2BCancers%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4881%26publicationDate%3DJune%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01862"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3648</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">21</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01862" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design and Discovery of N-(2-Methyl-5′-morpholino-6′-((tetrahydro-2H-pyran-4-yl)oxy)-[3,3′-bipyridin]-5-yl)-3-(trifluoromethyl)benzamide (RAF709): A Potent, Selective, and Efficacious RAF Inhibitor Targeting RAS Mutant Cancers&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Gisele&quot;,&quot;last_name&quot;:&quot;A. Nishiguchi&quot;},{&quot;first_name&quot;:&quot;Alice&quot;,&quot;last_name&quot;:&quot;Rico&quot;},{&quot;first_name&quot;:&quot;Huw&quot;,&quot;last_name&quot;:&quot;Tanner&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Aversa&quot;},{&quot;first_name&quot;:&quot;Benjamin&quot;,&quot;last_name&quot;:&quot;R. Taft&quot;},{&quot;first_name&quot;:&quot;Sharadha&quot;,&quot;last_name&quot;:&quot;Subramanian&quot;},{&quot;first_name&quot;:&quot;Lina&quot;,&quot;last_name&quot;:&quot;Setti&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;T. Burger&quot;},{&quot;first_name&quot;:&quot;Lifeng&quot;,&quot;last_name&quot;:&quot;Wan&quot;},{&quot;first_name&quot;:&quot;Victoriano&quot;,&quot;last_name&quot;:&quot;Tamez&quot;},{&quot;first_name&quot;:&quot;Aaron&quot;,&quot;last_name&quot;:&quot;Smith&quot;},{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;Lou&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;A. Barsanti&quot;},{&quot;first_name&quot;:&quot;Brent&quot;,&quot;last_name&quot;:&quot;A. Appleton&quot;},{&quot;first_name&quot;:&quot;Mulugeta&quot;,&quot;last_name&quot;:&quot;Mamo&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;Tandeske&quot;},{&quot;first_name&quot;:&quot;Ina&quot;,&quot;last_name&quot;:&quot;Dix&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;E. Tellew&quot;},{&quot;first_name&quot;:&quot;Shenlin&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Lesley&quot;,&quot;last_name&quot;:&quot;A. Mathews Griner&quot;},{&quot;first_name&quot;:&quot;Vesselina&quot;,&quot;last_name&quot;:&quot;G. Cooke&quot;},{&quot;first_name&quot;:&quot;Anne&quot;,&quot;last_name&quot;:&quot;Van Abbema&quot;},{&quot;first_name&quot;:&quot;Hanne&quot;,&quot;last_name&quot;:&quot;Merritt&quot;},{&quot;first_name&quot;:&quot;Sylvia&quot;,&quot;last_name&quot;:&quot;Ma&quot;},{&quot;first_name&quot;:&quot;Kalyani&quot;,&quot;last_name&quot;:&quot;Gampa&quot;},{&quot;first_name&quot;:&quot;Fei&quot;,&quot;last_name&quot;:&quot;Feng&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Yuan&quot;},{&quot;first_name&quot;:&quot;Yingyun&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Jacob&quot;,&quot;last_name&quot;:&quot;R. Haling&quot;},{&quot;first_name&quot;:&quot;Sepideh&quot;,&quot;last_name&quot;:&quot;Vaziri&quot;},{&quot;first_name&quot;:&quot;Mohammad&quot;,&quot;last_name&quot;:&quot;Hekmat-Nejad&quot;},{&quot;first_name&quot;:&quot;Johanna&quot;,&quot;last_name&quot;:&quot;M. Jansen&quot;},{&quot;first_name&quot;:&quot;Valery&quot;,&quot;last_name&quot;:&quot;Polyakov&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;Zang&quot;},{&quot;first_name&quot;:&quot;Vijay&quot;,&quot;last_name&quot;:&quot;Sethuraman&quot;},{&quot;first_name&quot;:&quot;Payman&quot;,&quot;last_name&quot;:&quot;Amiri&quot;},{&quot;first_name&quot;:&quot;Mallika&quot;,&quot;last_name&quot;:&quot;Singh&quot;},{&quot;first_name&quot;:&quot;Emma&quot;,&quot;last_name&quot;:&quot;Lees&quot;},{&quot;first_name&quot;:&quot;Wenlin&quot;,&quot;last_name&quot;:&quot;Shao&quot;},{&quot;first_name&quot;:&quot;Darrin&quot;,&quot;last_name&quot;:&quot;D. Stuart&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;P. Dillon&quot;},{&quot;first_name&quot;:&quot;Savithri&quot;,&quot;last_name&quot;:&quot;Ramurthy&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;4869-4881&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01862&quot;},&quot;abstract&quot;:&quot;RAS oncogenes have been implicated in &gt;30% of human cancers, all representing high unmet medical need. The exquisite dependency on CRAF kinase in KRAS mutant tumors has been established in genetically engineered mouse models and human tumor cells. To date, many small molecule approaches are under investigation to target CRAF, yet kinase-selective and cellular potent inhibitors remain challenging to identify. Herein, we describe 14 (RAF709) [ Aversa , Biaryl amide compounds as kinase inhibitors and their preparation . WO 2014151616, 2014 ], a selective B/C RAF inhibitor, which was developed through a hypothesis-driven approach focusing on drug-like properties. A key challenge encountered in the medicinal chemistry campaign was maintaining a balance between good solubility and potent cellular activity (suppression of pMEK and proliferation) in KRAS mutant tumor cell lines. We investigated the small molecule crystal structure of lead molecule 7 and hypothesized that disruption of the crystal packing would improv&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01862&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01862" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01862&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01862" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01862&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01862" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01862&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01862&amp;href=/doi/10.1021/acs.jmedchem.6b01862" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01862" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01862" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01862%26sid%3Dliteratum%253Aachs%26pmid%3D28557458%26genre%3Darticle%26aulast%3DNishiguchi%26date%3D2017%26atitle%3DDesign%2Band%2BDiscovery%2Bof%2BN-%25282-Methyl-5%25E2%2580%25B2-morpholino-6%25E2%2580%25B2-%2528%2528tetrahydro-2H-pyran-4-yl%2529oxy%2529-%255B3%252C3%25E2%2580%25B2-bipyridin%255D-5-yl%2529-3-%2528trifluoromethyl%2529benzamide%2B%2528RAF709%2529%253A%2BA%2BPotent%252C%2BSelective%252C%2Band%2BEfficacious%2BRAF%2BInhibitor%2BTargeting%2BRAS%2BMutant%2BCancers%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D12%26spage%3D4869%26epage%3D4881%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291454" title="Solubility">Solubility</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/jmcmar.2017.60.issue-12/20170622/jmcmar.2017.60.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">RAS oncogenes have been implicated in >30% of human cancers, all representing high unmet medical need. The exquisite dependency on CRAF kinase in KRAS mutant tumors has been established in genetically engineered mouse models and human tumor cells. To date, many small molecule approaches are under investigation to target CRAF, yet kinase-selective and cellular potent inhibitors remain challenging to identify. Herein, we describe <b>14</b> (RAF709) [<contrib-group person-group-type="allauthors">Aversa</contrib-group>, Biaryl amide compounds as kinase inhibitors and their preparation. WO 2014151616, <span class="NLM_year">2014</span>], a selective B/C RAF inhibitor, which was developed through a hypothesis-driven approach focusing on drug-like properties. A key challenge encountered in the medicinal chemistry campaign was maintaining a balance between good solubility and potent cellular activity (suppression of pMEK and proliferation) in KRAS mutant tumor cell lines. We investigated the small molecule crystal structure of lead molecule <b>7</b> and hypothesized that disruption of the crystal packing would improve solubility, which led to a change from <i>N</i>-methylpyridone to a tetrahydropyranyl oxy-pyridine derivative. <b>14</b> proved to be soluble, kinase selective, and efficacious in a KRAS mutant xenograft model.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32703" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32703" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The mitogen-activated protein kinase (MAPK) signaling pathway plays a major role in the regulation of multiple cellular functions. The pathway is activated through extracellular signals that in turn induce the RAS proteins (K-, N-, HRAS) to exchange GDP for GTP, resulting in activation of the RAF/MEK/ERK cascade. The MAPK pathway is often constitutively activated in human cancers through activating mutations in the RAS proteins or BRAF. In fact, the RAS genes are the most frequently mutated oncogenes in human tumors, however, targeting RAS directly as a cancer therapeutic has been challenging. Recent studies have described approaches to selectively inhibit KRAS<sup>G12C</sup> through covalent targeting, resulting in a cellular active inhibitor in the low micromolar range.<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(1-3)</a> While such covalent inhibitors demonstrate proof-of-concept for targeting KRAS<sup>G12C</sup>-driven tumors, other RAS mutant alleles remain undruggable.</div><div class="NLM_p">In considering targets downstream of mutant RAS, emerging biology from genetically engineered mouse models and tumor cell lines has begun to uncover which effectors are critical for tumorigenesis, providing new opportunities for therapeutic development. For example, CRAF was shown to be the critical mediator of mutant KRAS-driven cell proliferation and tumor development in both cellular and genetically engineered mouse tumor models.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(4, 5)</a> CRAF was also shown to play a critical role in feedback-mediated pathway reactivation following MEK inhibitor<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(6)</a> treatment in KRAS mutant cancers. In addition, CRAF plays an essential role in mediating paradoxical activation following BRAF inhibitor treatment.<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(7-9)</a> Thus, selective inhibitors that potently inhibit the activity of B/CRAF could be both effective in blocking mutant RAS-driven tumorigenesis and preventing feedback mediated reactivation While selective RAF inhibitors such as vemurafenib (<b>1</b>) have been developed to treat BRAF<sup>V600E</sup> mutant tumors, these inhibitors are ineffective against tumors with wild-type BRAF and in fact can accelerate the growth of RAS mutant tumors through paradoxical activation of RAF dimers.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(10, 11)</a> LY3009120 (<b>2</b>) has recently been described as an effective inhibitor of RAF dimers and demonstrates efficacy against some preclinical models of RAS mutant tumors; however, it appears to inhibit several other kinases, including those that have important roles in cancer such as ephrin receptors, JNK and SRC family members (<a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(12)</a> Thus, a potent inhibitor of RAF dimers with an improved selectivity profile will be invaluable in assessing the therapeutic utility and therapeutic index in RAS mutant tumors.</div><figure id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04625" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04625" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The major medicinal chemistry goal of this project was to identify highly selective RAF inhibitors that would potently suppress the RAF–MEK–ERK pathway in KRAS mutant tumor cells without causing paradoxical activation. For this purpose, we developed a full-length CRAF biochemical assay as well as Calu-6 (KRAS<sup>Q61K</sup>) cell proliferation and pMEK target modulation assays in which inhibition and activation of the pathway could be measured. Herein we describe our hit finding approach, the design and the synthesis of analogues that led to the discovery of <b>14</b> (<b>RAF709</b>), a RAF inhibitor that is well-tolerated and efficacious in KRAS mutant xenograft models.</div><div class="NLM_p">To assess whether any existing RAF kinase inhibitor chemotypes were able to suppress pMEK in KRAS mutant tumor cells without inducing paradoxical activation, we pursued an accelerated hit finding strategy that centered on leveraging our existing RAF inhibitor knowledgebase. A data set from 28k compounds that had shown evidence of RAF inhibition in historical biochemical or cellular assays was assembled. Subsets of this collection were selected for the CRAF biochemical assay and the pMEK target modulation assay in Calu-6 (KRAS<sup>Q61K</sup>) cells. The overarching selection criterion was to ensure sampling of all chemotypes for which there were analogues with good cellular potency in order to ensure we could assess the potential for paradoxical activation. A set of 116 compounds with evidence of pMEK suppression in Calu-6 cells was selected for assessment of paradoxical activation.</div><div class="NLM_p">From the analysis of this data set, there was a consistent trend toward type 2 inhibitors (DFG out, αC-helix in) exhibiting less paradoxical activation compared to type 1 (DFG in, αC-helix in) and 1.5 (DFG in, αC-helix out)<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(13, 14)</a> modes of inhibition (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). Compound <b>1</b> did not meet the criteria (pMEK > 30 μM) to be considered in the data set. On the basis of this observation, the team focused on type 2 inhibitors to further progress toward our medicinal chemistry goal. In depth data mining and further profiling of the hits uncovered several issues associated with type 2 inhibitors, including poor kinase selectivity, low aqueous solubility, and high lipophilicity. With these liabilities in mind, we hypothesized that increased levels of saturation could be an approach to improve solubility and enhance kinase selectivity. It has been documented that an increase in the fraction of sp<sup>3</sup> atoms (Fsp<sup>3</sup>), defined as the ratio of sp<sup>3</sup> atoms to the total number of non-hydrogen atoms in the molecule, is associated with decrease in melting point and improved solubility for small molecules.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(15)</a> At the same time, higher sp<sup>3</sup> character within a given molecule can also offer the opportunity for higher specificity in receptor/ligand complementarity and potentially mitigate off-targets binding.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(16)</a> This hypothesis led the team to further probe our type 2 hits, leading to the prioritization of compound <b>3</b> (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>), a starting point that has measurable solubility and an attractive kinase selectivity profile relative to other hits. In a KINOME<i>scan</i> binding displacement assay,<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(17, 18)</a><b>3</b> had a selectivity score <i>S</i>(35) of 0.076 when screened at 10 μM, indicative of high kinase selectivity. The morpholine moiety as a potential hinge binder was also viewed as a unique feature within RAF kinase inhibitors.</div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Box-plot of the distribution of fold-activation of pMEK in Calu-6 cells for each of the annotated types (mechanisms) of kinase inhibition<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(19)</a> based on medicinal chemistry scaffold matches. The 116 compounds in this analysis had a CRAF IC<sub>50</sub> varying from 42 nM to 30 pM and an EC<sub>50</sub> in the pMEK target modulation assay in Calu-6 cells <0.3 μM. The solid line at 2-fold-activation indicates the team’s assessment of a relevant cutoff.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray structure of <b>3</b> in the BRAF construct (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAL">5VAL</a>)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(21)</a> illustrating key features of protein–ligand interactions depiction of the crystallized ligand illustrating most important features of protein ligand interactions, labeled key residues, and a ligand pocket surface. All distances are in Å.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To elucidate the binding mode, we obtained an X-ray cocrystal structure of <b>3</b> complexed with the kinase domain BRAF. As illustrated in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>, the oxygen of the morpholine makes a H-bond interaction to the backbone NH of residue C532 in the hinge region (O···N distance is approximately 2.7 Å). The pyrimidine core is situated in a hydrophobic region, approximately 3.6 Å away from F595, which could be consistent with an edge to face or T-shaped π-interaction. The exact type of this interaction is difficult to ascertain due to the 2.4 Å resolution of the crystal structure. Because the two nitrogen atoms of the pyrimidine core are facing the solvent, they are unlikely to adversely affect the potency of the compound but are beneficial for lowering lipophilicity, and probably, enhancing its solubility. The central phenyl is twisted out of the plane of the molecule due to the presence of the “flag” methyl, allowing the amide moiety to engage the protein in two hydrogen bond interactions typical for type 2 inhibitors. The carbonyl makes a H-bond to D594 (O···N distance is approximately 2.7 Å) and the NH is 2.9 Å away from E501 without destroying the salt bridge between K483 and E501. The <i>t</i>-butyl phenyl occupies a hydrophobic region referred to here as the induced fit pocket. The protein adopted an inactive conformation characteristic for type 2 inhibitors with the DFG out and the αC-helix in.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(20)</a></div><div class="NLM_p">We initiated hit optimization with <b>3</b> (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>) aiming to improve the solubility and cellular potency. Through hit expansion from our compound collection, we identified analogue <b>4</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), with a lower cLogP (4.1) and similar cellular potency as <b>3</b>. From this compound, we synthesized the corresponding reverse-amide match <b>5</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) in order to assess if there is a favorable amide orientation for the series. Compound <b>5</b> had comparable biochemical potency and antiproliferative effect relative to <b>4</b> with improved target modulation, measurable solubility, and reasonable clearance (rat and human liver microsomes Cl<sub>int</sub>: 79 and 43 μL/min/mg), making this a preferred amide isomer for subsequent analogues.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biochemical and Cellular Activity, Permeability, and Solubility Data for Compounds <b>3</b>, <b>4</b>, <b>5</b>, and <b>6</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0013.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="/" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">CRAF IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">pMEK Calu-6 EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">CP Calu-6 EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="/">Caco-2 <i>P</i><sub>app</sub> AB/BA<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a> (× 10<sup>–6</sup> cm/s)</th><th class="colsep0 rowsep0" align="center" char=".">Sol<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">cLogP<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">0.0005</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="char" char=".">6.4</td><td class="colsep0 rowsep0" align="char" char="/">15/15</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">4.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">0.002</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char="/">15/8</td><td class="colsep0 rowsep0" align="char" char="."><2</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">0.0001</td><td class="colsep0 rowsep0" align="char" char=".">0.08</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td><td class="colsep0 rowsep0" align="char" char="/">19/38</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">4.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">0.0002</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="char" char="/"><a class="ref internalNav" href="#t1fn4" aria-label="d">d</a>/1.8</td><td class="colsep0 rowsep0" align="char" char="."><2</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Forward and reverse flux were measured via transit across 21-day cultured Caco-2 monolayers at pH7.4 with quantification via LC/MS/MS.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">Solubility (shake–flask assay) was measured with Cl-free PBS at pH 6.8.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">cLogP was calculated using Biobyte software.</p></div><div class="footnote" id="t1fn4"><sup>d</sup><p class="last">Poor recovery.</p></div></div></div><div class="NLM_p">On the basis of the BRAF cocrystal structure with <b>3</b>, we postulated that removal of the pyrimidine nitrogen adjacent to the central phenyl should not affect biochemical potency because both nitrogens in the pyrimidine ring face the solvent. The pyridine analogue <b>6</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) was found to have similar potency as <b>5</b> in the biochemical assay. The resulted torsional angle increase between the pyridine ring and the central phenyl ring relative to the pyrimidine–central phenyl ring led to the hypothesis that there is some flexibility (around the hydrophobic pocket and the gate keeper residue T529) that would be tolerated. While the poor solubility of compound <b>6</b> still remained a concern, the potent cellular activity provided confidence in the series and led to the hypothesis that core modifications could be utilized to modulate cell potency as well as the properties of the molecules. This turned our attention to the lipophilicity of the cores, and we focused on designing analogues with lower cLogP.</div><div class="NLM_p">In an effort to decrease the cLogP, we investigated core modifications with more polar heterocycles in conjunction with central and induced fit pyridyl modifications. Four representative examples of our SAR set are illustrated in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>. We quickly learned from compounds <b>7</b>–<b>10</b> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>) that intrinsic clearance and solubility were consistently improved but the increased polarity eroded cellular potency in many cases. We hypothesized that optimal polarity might be essential to balance potency and drug-like properties and strategized that making different core modifications would allow us to find the right balance. The balanced profile of solubility, target modulation, and antiproliferative effect of pyridone <b>7</b> led us to select this compound over the other analogues for further profiling. Because kinase selectivity is one of the liabilities with type 2 inhibitors, we screened this compound using the KINOME<i>scan</i> binding displacement assay against 456 human kinases and their relevant mutants at 1 μM (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). Of the 456 kinases tested, it showed a high level of selectivity, demonstrating greater than 99% binding to BRAF, BRAF<sup>V600E</sup>, and 90% binding to CRAF with binding to only one off-target at >80% (PDGFRβ,89%) and a selectivity score (<i>S</i>-score) of <i>S</i>(35) = 0.015. In cellular target modulation, it was potent and showed minimal paradoxical activation in KRAS mutant cell lines.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Potency, ADME, and Solubility Data for Analogues with a Lower cLogP</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0014.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="/" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">Y</th><th class="colsep0 rowsep0" align="center">Z</th><th class="colsep0 rowsep0" align="center" char=".">CRAF IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">pMEK Calu-6 EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">CP Calu-6 EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="/">Caco-2 <i>P</i><sub>app</sub> AB/BA (× 10<sup>–6</sup> cm/s)</th><th class="colsep0 rowsep0" align="center" char=".">Cl<sub>int</sub> (human microsomal μL/min/mg)</th><th class="colsep0 rowsep0" align="center" char=".">Sol (μM)</th><th class="colsep0 rowsep0" align="center" char=".">cLogP</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="char" char=".">0.0004</td><td class="colsep0 rowsep0" align="char" char=".">0.076</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char="/">20/22</td><td class="colsep0 rowsep0" align="char" char=".">51</td><td class="colsep0 rowsep0" align="char" char=".">70</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="char" char=".">0.0003</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td><td class="colsep0 rowsep0" align="char" char="/">21/16</td><td class="colsep0 rowsep0" align="char" char=".">110</td><td class="colsep0 rowsep0" align="char" char=".">190</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="char" char=".">0.0008</td><td class="colsep0 rowsep0" align="char" char=".">0.46</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char="/">7/43</td><td class="colsep0 rowsep0" align="char" char="."><7.7</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char=".">0.001</td><td class="colsep0 rowsep0" align="char" char=".">3.96</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char="/">3/39</td><td class="colsep0 rowsep0" align="char" char="."><7.7</td><td class="colsep0 rowsep0" align="char" char=".">1040</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td></tr></tbody></table></div></div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. KINOME<i>scan</i> profile of <b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compound <b>7</b> also demonstrated good permeability, minimum efflux based on Caco-2 assays, and moderate in vitro clearance measured in rat (Cl<sub>int</sub>: 46 μL/min/mg) and human liver microsomes (Cl<sub>int</sub>: 51 μL/min/mg). Given its favorable in vitro ADME profile, the pharmacokinetics of <b>7</b> was evaluated in rats (Sprague–Dawley) at 1 mg/kg IV and 3 mg/kg PO. The results indicated that <b>7</b> had an oral total exposure (AUC) of 0.438 μM·h, high in vivo clearance (53 mL/min/kg), <i>C</i><sub>max</sub> of 0.15 μM, and low-moderate bioavailability of 22%. Formulating <b>7</b> proved challenging, potentially due to its solubility, and this hindered further in vivo studies, especially at higher doses where a lack of dose-proportional exposure was observed. More in depth physicochemical characterization of <b>7</b> revealed a melting point >230 °C (decomposed), which clearly suggested that tight crystal packing in the solid phase might also have contributed to the poor solubility and this was further supported by measuring the solubility from crystalline material which gave a value of 4 μM. Therefore, lead optimization centered on further addressing the solubility of the series without compromising the cellular potency.</div><div class="NLM_p">To further understand the chemical modifications required to disrupt crystal packing or lowering the melting point, we investigated the small molecule X-ray structure of <b>7</b>.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(21)</a> The molecules are arranged in pairs oriented head-to-tail so that the amide nitrogen of each molecule (N22) forms a moderately strong H-bond to the carbonyl oxygen (O12) of the pyridone ring of the second molecule (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). The distance between N and O atoms is 2.99 Å. Additionally, the same pyridone rings are oriented in the parallel planes with the distance of 4.21 Å between them, which is consistent with the parallel displaced π-interaction.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0005.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Small molecule X-ray structure of compound <b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">On the basis of this analysis, we hypothesized that removal of the carbonyl would disrupt the intermolecular hydrogen bond network and furthermore the introduction of sp<sup>3</sup> character in close proximity to the core could potentially disrupt the parallel displaced π-stacking interaction. To test our hypothesis, the 3-morpholino-2-oxypyridine core emerged as a valuable scaffold (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). In this series, the pyridone carbonyl was replaced by the ether oxygen and the directionality of the methyl provided a solvent exposed vector. This core was initially derived from the pyridone alkylation reaction (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) to provide the minor O-alkylated product, exemplified by <b>11</b> (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). It is noteworthy to highlight the serendipitous nature of the discovery of the O-alkylated core “<i>as a</i> reaction <i>byproduct</i>”.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Effect of Increased Fsp<sup>3</sup> on Solubility, Permeability, and Activity in the Oxy-pyridine Core</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0015.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0016.gif" alt="" id="fx4" /></img><div></div></div><div class="NLM_p">While compound <b>11</b> was moderately potent in cellular assays and remained poorly soluble, we hypothesized that the methoxy group was satisfying the disruption of the intermolecular hydrogen bonding network but was not large enough to disrupt the parallel π-interaction. Given the Fsp<sup>3</sup> of compound <b>11</b> is 0.29, we envisioned that an increase in Fsp<sup>3</sup> around the methoxy group might address the solubility liability. As shown in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>, by increasing the size of the R<sub>1</sub> substituent from methyl to isopropyl, compound <b>12</b> was obtained. Although its solubility was still low, the fact that cell potency was maintained was encouraging and larger substituents were investigated. A glycol moiety <b>13</b> (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) was tolerated, and it provided a lower cLogP and higher Fsp<sup>3</sup>, with solubility trending in a favorable direction at 24 μM, however, an efflux liability was introduced. By exploring larger R<sub>1</sub> substituents such as the tetrahydropyran in <b>14</b> high throughput solubility was improved to 200 μM and the solubility of the highly stable crystalline material was measured to be 120 μM (melting point: 196 °C). Moreover, pMEK cellular potency was preserved and low efflux was maintained. Overall, <b>14</b> provided the most favorable balance of physicochemical properties and cellular potency, and hence it was selected for further profiling.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> X-ray Structure of <b>14</b></h3><div class="NLM_p">The X-ray cocrystal structure of <b>14</b> in BRAF illustrated in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> shows a binding mode similar to <b>3</b> with respect to the morpholine and induced fit moiety. The central pyridine occupies the space of the corresponding central toluyl tightly filling up a narrow hydrophobic pocket formed by the side chains of K483, I527, T529, and A481. The CF<sub>3</sub> phenyl occupies the hydrophobic induced-fit pocket similar to the structure of compound <b>3</b>. The tetrahydropyran faces a solvent opening in the ATP binding pocket which can accommodate such a large group. The protein maintains the DFG-out αC-Helix-in inactive conformation consistent with a type 2 mode of inhibition.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0006.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray structure of <b>14</b> in BRAF (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAM">5VAM</a>)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(21)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> In Vitro Profiling of <b>14</b><a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(22)</a></h3><div class="NLM_p">The biochemical activity of <b>14</b> was further characterized against purified full-length BRAF where the IC<sub>50</sub> measured 0.0004 μM, which is in line with the potency against CRAF. Off-rate is an important attribute of potency, impacting both the cellular activity in the face of feedback and the PK/PD relationship in vivo. Therefore, we attempted to measure the dissociation rate constant for <b>14</b> using the rapid dilution method and it appeared to have very slow dissociation kinetics (<i>T</i><sub>1/2</sub> > 6.5 h). In cellular assays, the dose–response of pMEK and pERK were measured in Calu-6 cells with EC<sub>50</sub> = 0.02 and 0.1 μM with minimal paradoxical activation (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>A) and inhibition of proliferation with EC<sub>50</sub> = 0.95 μM. Compound <b>1</b> clearly demonstrated paradoxical activation in the pMEK assay (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>A) with maximum activity at 3 μM and no resultant inhibition up to the highest concentration tested (30 μM). The ability of <b>14</b> to stabilize RAF dimerization was measured using the CRAF:BRAF NanoBiT luciferase complementation assay (Promega). Consistent with most other RAF inhibitors, <b>14</b> stabilized BRAF–CRAF dimers with an EC<sub>50</sub> = 0.8 μM (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>B) while <b>1</b> demonstrated modest stabilization of dimers in this assay. Additional supporting data was revealed in the X-ray crystal structure of <b>14</b> (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>), where each BRAF protomer in the dimer structure was loaded with compound. This also demonstrated that (at least under the conditions of crystallization) binding of <b>14</b> to one protomer did not preclude binding to the other protomer in the dimer structure. Altogether, these data demonstrate that <b>14</b> stabilizes RAF dimers but is relatively equipotent at inhibiting both protomers because it induces minimal paradoxical activation but effectively suppresses signaling.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0007.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) pMEK inhibition and activation of <b>1</b> and <b>14</b> in Calu-6 RAS mutant cells. (B) Effect of <b>1</b> and <b>14</b> on the stimulation of dimerization in HCT116 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Kinase Selectivity of <b>14</b></h3><div class="NLM_p">Consistent with this chemical series, <b>14</b> was found to be highly selective when evaluated using the KINOME<i>scan</i> screening platform. Of the 456 kinases tested,<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(17, 18)</a><b>14</b> showed a high level of selectivity, demonstrating greater than 99% on-target binding to BRAF, BRAF<sup>V600E</sup>, and CRAF at 1 μM and very few off-targets with DDR1 (>99%), DDR2 (86%), FRK (92%), and PDGFRb (96%), the only kinases with binding >80% at 1 μM. To the best of our knowledge, this compound exhibits the highest level of kinase selectivity among known type 2 RAF inhibitors with a selectivity score of <i>S</i>(35) = 0.018.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(18)</a></div><div class="NLM_p last">Compounds such as <b>14</b> and <b>7</b> with a morpholine as the hinge binding moiety appeared to be remarkably selective inhibitors of RAF kinases often exhibiting more than 10000-fold difference in the binding affinities even toward other structurally similar proteins like p38 and EGFR. The combination of a morpholine, a central core attached to the perpendicular central phenyl and an appropriate group to fill the induced-fit hydrophobic pocket (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>), allowed us to fine-tune inhibitors to accommodate the unique geometries of RAF kinases ensuring suboptimal binding to other kinases.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Pharmacokinetics of <b>14</b></h3><div class="NLM_p"><b>14</b> was reasonably stable in plasma after a 3 h incubation at 37 °C across species [plasma stability (%remaining): rat 85%, mouse 82%, dog 95%, human 101%], and plasma protein binding was measured to be 98% across species. In pharmacokinetic experiments (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>), <b>14</b> had moderate clearance in mouse (35 mL/min/kg) and dog (14 mL/min/kg) and high clearance in rat (50 mL/min/kg). <i>C</i><sub>max</sub> in mouse (1 μM), dog (0.5 μM), and rat (0.5 μM) reached pharmacologically active concentrations, and acceptable oral availability was observed in mouse (68%), rat (24%), and dog (48%). Compared to <b>7</b>, <b>14</b> was less challenging to formulate due to higher intrinsic solubility which enabled progression into PK/PD and efficacy studies at higher doses.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. PK Profile of <b>14</b> across Species</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="/" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center" char="/">dose (iv/po, mg/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>max</sub> (po, μM)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>inf</sub> (po, μM·h)</th><th class="colsep0 rowsep0" align="center" char=".">Cl (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>ss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center"><i>F</i> (po) (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mouse</td><td class="colsep0 rowsep0" align="char" char="/">1/3</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="center">68<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="char" char="/">3/10</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="center">24<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog</td><td class="colsep0 rowsep0" align="char" char="/">0.3/1</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="center">48<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Formulation (iv/po): 20% PEG300 + 5% solutol.</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last">Formulation (iv/po): 25% PEG300 + 5% solutol.</p></div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> PK/PD of <b>14</b></h3><div class="NLM_p">The PK/PD relationship of <b>14</b> was examined in the Calu-6 xenograft model in nude mice. The plasma exposure of <b>14</b>, and levels of pERK in tumor tissue were determined following a single oral administration across a dose range of 10, 30, and 200 mg/kg. <b>14</b> showed dose-proportional increases in plasma exposure and a corresponding dose-dependent inhibition of pERK in Calu-6 tumors (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>).</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0008.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. PK/PD analysis of <b>14</b> in the human Calu-6 NSCLC xenograft in mice. Percent pERK inhibition (left axis) and free exposure (right axis) following a single dose of <b>14</b>, at time points indicated, in Calu-6 tumor bearing mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">At 24 h postdose of 10 mg/kg <b>14</b>, plasma levels were below the lower limit of quantitation and there was a complete lack of pERK inhibition. At the highest dose level of 200 mg/kg, free plasma concentrations were maintained above the cellular in vitro pERK IC<sub>50</sub> value for the entire 24 h, which was consistent with sustained inhibition of pERK ≥ 70%. It is interesting to note that at the 10 and 30 mg/kg doses, the pathway seemed to be hyperactivated compared to baseline at the 24 h time point. The most likely explanation for this is that reactivation of the pathway occurred following the loss of negative feedback when pERK was briefly suppressed by <b>14</b> at earlier time points. However, it may also be possible that either lower concentrations of <b>14</b> or its slow dissociation rate could be contributing to the pathway hyperactivation. Dose-dependent modulation of pMEK, DUSP6, and SPRY4 were similar to those described for pERK (data not shown).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(22)</a></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Efficacy and Tolerability of <b>14</b> in the Calu-6 Xenograft Model</h3><div class="NLM_p">The antitumor efficacy and tolerability of <b>14</b> were determined in the Calu-6 xenograft nude mouse model. Mice were treated with vehicle or <b>14</b> at 10, 30, or 200 mg/kg po daily beginning 12 days after tumor implantation and continued until day 27. The antitumor activity was determined by assessing %T/C or % regression on day 27 postimplant (15 days of treatment). Treatment with <b>14</b> resulted in dose-dependent antitumor activity with 10 mg/kg being subefficacious (%T/C = 92%), 30 mg/kg resulted in measurable antitumor activity (%T/C = 46%), and 200 mg/kg resulted in mean tumor regression of 92%, while the same high dose was not efficacious in the PC3, KRAS WT model (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_001.pdf" class="ext-link">Figure 1A</a>). All doses were well tolerated, with no significant body weight loss and no signs of toxicity or mortalities observed (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a> and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_001.pdf" class="ext-link">Figure 1B</a>).</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0009.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Efficacy of <b>14</b> in Calu-6 xenograft in mice. Tumor volumes (A) or percent body weight change from initial (B) treatment groups vs vehicle control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Chemistry</h3><div class="NLM_p">Compounds from <a class="ref internalNav" href="#tbl2" aria-label="Tables 2">Tables 2</a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> were synthesized according to <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>, <a class="ref internalNav" href="#sch2" aria-label="2">2</a>, and <a class="ref internalNav" href="#sch3" aria-label="3">3</a>. The synthesis of morpholine–core bromide moieties are illustrated in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Starting with commercially available <b>15</b>, formation of the morpholine was obtained by reaction with 1-bromo-2-(2-bromoethoxy)ethane in the presence of DIEA at 120 °C to provide product <b>16</b> in 70% yield (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). By reacting <b>16</b> with HCl in dioxane at reflux, the corresponding pyridone was obtained which upon treatment with K<sub>2</sub>CO<sub>3</sub> and iodomethane an approximately 5:1 ratio of the N-alkylated and O-alkylated products (<b>17</b> and <b>16</b>) was obtained and easily separable by column chromatography.</div><figure id="sch1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Core/Morpholine Bromides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1-bromo-2-(2-bromoethoxy)ethane, DMF, DIEA, 120 °C, 70–76%; (b) HCl, dioxane, 110 °C, 93%; (c) K<sub>2</sub>CO<sub>3</sub>, DMF, MeI, 76–91%; (d) morpholine, 100 °C, 61%.</p></p></figure><div class="NLM_p">The pyrazinone bromide <b>19</b> was synthesized by starting with commercially available bis-bromide <b>18</b> and treating with morpholine at 100 °C. Upon isolation of the product (<b>19</b>), <i>N</i>-methylation was accomplished as described above to give a major isomer as the desired product <b>20</b>.</div><div class="NLM_p">Bromide <b>22</b> was synthesized by starting with <b>21</b> and utilizing the procedure for the ring closure of the morpholine as described above for the synthesis of <b>16</b> to give intermediate <b>22</b> in 76% yield.</div><div class="NLM_p">Intermediates containing the central phenyl or central pyridine and induced fit motifs were synthesized according to <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Starting with <b>23</b>, reaction with 3-(trifluoromethyl)benzoyl chloride in DCM and DIEA afforded intermediate <b>24</b>. Upon borylation of <b>24</b>, boronic ester intermediate <b>25</b> was isolated in 71% yield. Compound <b>28</b> was obtained by treating the commercially available bromide <b>26</b> with bis(pinacolato)diboron, KOAc, and Pd(dppf)Cl<sub>2</sub>–DCM in dioxane to give boronate ester <b>27</b>. Reduction of the nitro functionality to the corresponding aniline <b>28</b> was accomplished with hydrogen and Pd/C in 99% yield. Boronate ester <b>30</b> was easily prepared by reacting <b>29</b> with 3-(trifluoromethyl)benzoyl chloride in DCM and DIEA.</div><figure id="sch2" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0011.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Boronic Ester Intermediates<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) ArCOCl, DCM, DIEA, 96–100%; (b) Pd(dppf)Cl<sub>2</sub>–DCM, dioxane, bis(pinacolato)diboron, KOAc, 70–80%; (c) H<sub>2</sub>, Pd/C, 99%.</p></p></figure><div class="NLM_p">Final compounds (<b>7</b>–<b>10</b>) were obtained by the Suzuki–Miyaura reaction between the core bromides and the appropriate boronic esters as illustrated in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Alternatively, final compounds were also obtained via a Suzuki–Miyaura reaction with the core bromides and the boronic ester <b>28</b>, followed by amide bond formation with the appropriate carboxylic acid in the presence of EDC and HOAt.</div><figure id="sch3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0012.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Final Compounds Synthesis<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Pd(dppf)Cl<sub>2</sub>–DCM, DME, 2 M Na<sub>2</sub>CO<sub>3</sub>, 120 °C; (b) Pd(dppf)Cl<sub>2</sub>–DCM, DME, 2 M Na<sub>2</sub>CO<sub>3</sub>, 120 °C, boronic ester, then DMF, EDC, HOAt, carboxylic acid; (c) R<sub>1</sub>-OH, NaH, dioxane or DMF, 80–105 °C.</p></p></figure><div class="NLM_p last">To enable last-step diversification and obtain final compounds exemplified in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>, intermediate <b>31</b> was synthesized via Suzuki–Miyaura coupling reaction between <b>22</b> and <b>25</b>. Treatment of this intermediate with the desired preformed alkoxide in THF allowed for isolation of final products (<b>12</b>–<b>14</b>) upon HPLC purification (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77315" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77315" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Despite the clinical activity of RAF and MEK inhibitors in BRAF<sup>V600</sup> mutant melanoma, they are relatively ineffective in RAS mutant tumors, leaving a significant unmet medical need. The class of RAF inhibitors approved for the treatment of BRAF<sup>V600mut</sup> melanoma, including vemurafenib and dabrafenib, induce paradoxical activation in BRAF wild-type cells and can induce growth of RAS<sup>mut</sup> tumors. However, given the central role of CRAF in driving mutant KRAS-driven tumorigenesis, there remains a significant interest in developing RAF inhibitors that will be efficacious in RAS mutant tumors. Our work demonstrates that type 2 RAF inhibitors have the potential to be effective in RAS mutant tumors because they induce minimal paradoxical activation compared to RAF inhibitors with other binding modes (i.e., type 1 or type 1.5). However, previous type 2 RAF inhibitors have suffered from a lack of selectivity, potency, and solubility. We set out to address these key issues in the design of a next-generation type 2 RAF inhibitor, and in this article we describe the discovery of <b>14</b>. By considering saturation as a characteristic to modulate kinase selectivity during hit selection, we identified our starting point, a morpholine analogue <b>3</b> which demonstrated good selectivity. A strategy focused on lowering lipophilicity to address solubility enabled the identification of <b>7</b>, a potent type 2 RAF inhibitor in KRAS mutant tumor cells with minimal paradoxical activation. Limitations associated with solubility and formulation of <b>7</b> and challenges in balancing physicochemical properties with cellular potency directed the effort toward increasing the Fsp<sup>3</sup> character as a strategy to improve solubility by disrupting crystal packing. This approach led to the identification of <b>14</b>, a compound shown to be highly kinase selective and cellularly potent in a KRAS mutant cell line (Calu-6) with minimal paradoxical activation which we believe would enable the compound to be combined with other known linear and/or lateral pathway inhibitors. In vivo evaluation of <b>14</b> indicated that it has a reasonable rodent PK profile suitable for PK/PD/efficacy and tolerability studies. <b>14</b> was shown to modulate pERK/pMEK<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(22)</a> in vivo, and it demonstrated dose-dependent tumor regressions in the KRAS mutant Calu-6 xenograft model with no significant body weight loss.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65805" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65805" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Methods</h3><div class="NLM_p last">The compounds and/or intermediates were characterized by high performance liquid chromatography (HPLC) using a Waters Millennium chromatography system with a 2695 separation module (Milford, MA). The analytical columns were Alltima C-18 reversed phase, 4.6 mm × 50 mm, flow 2.5 mL/min, from Alltech (Deerfield, IL). A gradient elution was used, typically starting with 5% acetonitrile/95% water and progressing to 100% acetonitrile over a period of 10 min. All solvents contained 0.1% trifluoroacetic acid (TFA). Compounds were detected by ultraviolet light (UV) absorption at either 220 or 254 nm. HPLC solvents were from Burdick and Jackson (Muskegan, MI) or Fisher Scientific (Pittsburgh, PA). Mass spectrometric analysis was performed on an LCMS instrument: a Waters system (Alliance HT HPLC and a Micromass ZQ mass spectrometer, Eclipse XDB-C18, 2.1 mm × 50 mm; solvent system, 5–95% acetonitrile in water with 0.1% TFA; flow rate 0.8 mL/min; molecular weight range 200–800; cone voltage 20 V; column temperature 40 °C). All masses were reported as those of the protonated parent ions. <sup>1</sup>H nuclear magnetic resonance (NMR) analyses described herein were performed on some of the compounds with a Varian 400-MR MHz NMR (Palo Alto, CA) spectrometer operating at a frequency of 399.89 MHz for <sup>1</sup>H or Bruker DRX-500 NMR spectrometer operating at a frequency of 500.13 MHz for <sup>1</sup>H. The spectral reference was either TMS or the known chemical shift of the solvent. The spectra were recorded at a temperature of 298 K. Preparative separations were carried out using a Teledyne ISCO chromatography system, or by HPLC using a Waters 2767 sample manager, C-18 reversed phase column, 30 mm × 50 mm, flow 75 mL/min. Typical solvents employed for the Teledyne ISCO chromatography system were dichloromethane, methanol, ethyl acetate, and heptane. Typical solvents employed for the reverse phase HPLC were varying concentrations of acetonitrile and water with 0.1% trifluoroacetic acid. The purity of all compounds screened in the biological assays was examined by LC-MS analysis and were found to be ≥95%. Experimental details for the synthesis and characterization of compounds <b>3</b> and <b>4</b> are described in the <a href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General Procedure for the Suzuki–Miyaura Reaction</h3><div class="NLM_p">To the appropriate heteroaryl halide (0.10 mmol) and the appropriate boronic ester (0.12 mmol) in DME (1 mL) was added PdCl<sub>2</sub> (dppf)·CH<sub>2</sub>Cl<sub>2</sub> adduct (0.010 mmol) and 2 M aqueous sodium carbonate (0.50 mmol). The reaction mixture was irradiated at 120 °C in a Biotage Initiator microwave for 10–12 min. The cooled reaction mixture was diluted with 2:1 DCM:MeOH (15 mL) and filtered. The filtrate was concentrated and purified by reverse phase HPLC to give the desired product as its TFA salt.</div><div class="NLM_p">The following compounds were prepared by the above method starting from the appropriate halide and boronic ester.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> <i>N</i>-(4-Methyl-3-(6-morpholinopyrimidin-4-yl)phenyl)-3-(trifluoromethyl)benzamide (<b>5</b>)</h4><div class="NLM_p last">Obtained in 51% yield. LCMS (<i>m</i>/<i>z</i>) (M + H) = 443.3, <i>R</i><sub>t</sub> = 0.75 min. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.70 (s, 1H), 8.86 (s, 1H), 8.26–8.38 (m, 2H), 7.93–8.06 (m, 2H), 7.74–7.90 (m, 2H), 7.43 (d, <i>J</i> = 8.5 Hz, 1H), 7.22 (s, 1H), 3.89 (s, 4H), 3.64–3.79 (m, 4H), 2.33 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.52, 161.50, 153.48, 137.38, 135.90, 132.32, 131.93, 131.66, 130.32, 129.83, 129.57, 128.82, 125.51, 124.66, 124.63, 123.35, 122.87, 121.72, 103.55, 66.15, 45.40,19.42. HRMS <i>m</i>/<i>z</i> (M<sup>+</sup>+ 1) calcd 443.1695, obsd 443.1694.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> <i>N</i>-(4-Methyl-3-(2-morpholinopyridin-4-yl)phenyl)-3-(trifluoromethyl)benzamide (<b>6</b>)</h4><div class="NLM_p last">Obtained in 28% yield as a white solid. LCMS (<i>m</i>/<i>z</i>) (M + H) = 442.3, <i>R</i><sub>t</sub> = 0.76 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.40–10.61 (m, 1 H), 8.30 (s, 1 H), 8.26 (d, <i>J</i> = 7.83 Hz, 1 H), 8.17 (d, <i>J</i> = 5.48 Hz, 1 H), 7.90–8.03 (m, 1 H), 7.67–7.82 (m, 3 H), 7.27–7.41 (m, 1H), 7.00 (s, 1 H), 6.82 (d, <i>J</i> = 5.48 Hz, 1 H), 3.44–3.59 (m, 8 H), 2.25 (s, 3 H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.39, 139.29, 137.33, 136.06, 132.28, 131.37, 130.74, 130.25, 130.22, 129.78, 129.53, 128.69, 125.52, 124.64, 123.35, 121.36, 121.14, 114.59, 109.70, 66.08, 45.95, 19.83. HRMS <i>m</i>/<i>z</i> (M<sup>+</sup> + 1) calcd 442.1742, obsd 442.1742.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> <i>N</i>-(4-Methyl-3-(1-methyl-5-morpholino-6-oxo-1,6-dihydropyridin-3-yl)phenyl)-3-(trifluoromethyl)benzamide (<b>7</b>)</h4><div class="NLM_p last">Obtained in 15% yield as an off-white solid. LCMS (<i>m</i>/<i>z</i>) (M + H) = 472.3, <i>R</i><sub>t</sub> = 0.88 min. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 8.27 (s, 1H), 8.22 (d, <i>J</i> = 8.2 Hz, 1H), 7.91 (d, <i>J</i> = 7.8 Hz, 1H), 7.71–7.79 (m, 1H), 7.56–7.66 (m, 2H), 7.36 (d, <i>J</i> = 1.6 Hz, 1H), 7.31 (d, <i>J</i> = 8.2 Hz, 1H), 6.93 (d, <i>J</i> = 2.0 Hz, 1H), 3.82–3.92 (m, 4H), 3.65 (s, 3H), 3.09–3.19 (m, 4H), 2.32 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.28, 157.93, 140.77, 138.05, 137.20, 136.17, 132.27, 131.49, 131.10, 130.64, 130.23, 129.77, 129.52, 128.60, 125.54, 124.60, 123.38, 122.06, 121.93, 119.95, 118.61, 66.60, 49.38,37.66, 20.16. HRMS <i>m</i>/<i>z</i> (M<sup>+</sup> + 1) calcd 472.1848, obsd 472.1844.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>N</i>-(4-Methyl-3-(4-methyl-6-morpholino-5-oxo-4,5-dihydropyrazin-2-yl)phenyl)-3-(trifluoromethyl)benzamide (<b>8</b>)</h4><div class="NLM_p last">Obtained in 13% yield as a solid. LCMS <i>m</i>/<i>z</i> (M + H) = 473.1, <i>R</i><sub>t</sub> = 0.92 min. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 8.25 (s, 1 H), 8.20 (d, <i>J</i> = 7.83 Hz, 1 H), 7.89 (d, <i>J</i> = 7.83 Hz, 1 H), 7.67–7.80 (m, 2 H), 7.56 (dd, <i>J</i> = 8.22, 2.35 Hz, 1 H), 7.26 (d, <i>J</i> = 8.22 Hz, 1 H), 7.18 (s, 1 H), 3.80 (s, 8 H), 3.56 (s, 3 H), 2.39 (s, 3 H). HRMS <i>m</i>/<i>z</i> (M<sup>+</sup> + 1) calcd 473.1801, obsd 473.1801.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> <i>N</i>-(1′,2-Dimethyl-5′-morpholino-6′-oxo-1′,6′-dihydro-[3,3′-bipyridin]-5-yl)-3-(trifluoromethyl)benzamide (<b>9</b>)</h4><div class="NLM_p last">Obtained in 22% yield as a solid. LCMS <i>m</i>/<i>z</i> (M + H) = 473.0, <i>R</i><sub>t</sub> = 0.56 min. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 8.77 (d, <i>J</i> = 2.35 Hz, 1 H), 8.29 (s, 1 H), 8.23 (d, <i>J</i> = 7.83 Hz, 1 H), 8.12 (d, <i>J</i> = 2.35 Hz, 1 H), 7.92 (d, <i>J</i> = 8.22 Hz, 1 H), 7.68–7.80 (m, 1 H), 7.43 (d, <i>J</i> = 2.35 Hz, 1 H), 6.93 (d, <i>J</i> = 2.35 Hz, 1 H), 3.77–3.94 (m, 4 H), 3.64 (s, 3 H), 3.07–3.20 (m, 4 H), 2.52 (s, 3 H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.70, 157.97, 151.42, 141.01, 140.49, 135.61, 133.84, 132.72, 132.35, 131.10, 130.34, 129.06, 124.70, 121.43, 116.76, 66.56, 49.34, 37.71, 23.21. HRMS <i>m</i>/<i>z</i> (M<sup>+</sup> + 1) calcd 473.1801, obsd 473.1804; mp 242 °C.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i>-(1′,2-Dimethyl-5′-morpholino-6′-oxo-1′,6′-dihydro-[3,3′-bipyridin]-5-yl)-2-(trifluoromethyl)isonicotinamide (<b>10</b>)</h4><div class="NLM_p last">To 5-bromo-1-methyl-3-morpholinopyridin-2(1<i>H</i>)-one (546 mg, 2.0 mmol) and 6-methyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridin-3-amine (492 mg, 2.1 mmol) in DME (volume: 11 mL) was added PdCl<sub>2</sub> (dppf)·CH<sub>2</sub>Cl<sub>2</sub> adduct (163 mg, 0.2 mmol) and 2 M aqueous sodium carbonate (3.00 mL, 6 mmol). The reaction mixture was irradiated at 125 °C for 20 min. LC-MS showed primarily conversion to product. The cooled reaction mixture was diluted with 2:1 DCM:MeOH (15 mL) and filtered. The filtrate was concentrated with silica gel and purified by flash chromatography over silica gel (ethyl acetate with 0–15% methanol gradient) to give 5′-amino-1,2′-dimethyl-5-morpholino-[3,3′-bipyridin]-6(1<i>H</i>)-one (450 mg, 1.498 mmol, 74.9% yield) as a brown solid. LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 301.0. To a solution of 2-(trifluoromethyl)pyridine-4-carboxylic acid (0.021 g, 0.110 mmol), HOAt (0.015 g, 0.110 mmol), and EDC·HCl (0.021 g, 0.110 mmol) in DMF (0.5 mL) was added 5′-amino-1,2′-dimethyl-5-morpholino-[3,3′-bipyridin]-6(1<i>H</i>)-one (0.030 g, 0.100 mmol). The mixture was stirred overnight at ambient temperature. The reaction mixture was diluted with DMSO, filtered, and purified by HPLC. The clean fractions were lyophilized to give <i>N</i>-(1′,2-dimethyl-5′-morpholino-6′-oxo-1′,6′-dihydro-[3,3′-bipyridin]-5-yl)-2-(trifluoromethyl)isonicotinamide (13 mg, 0.022 mmol, 21.93% yield) as a light-yellow solid. LCMS (<i>m</i>/<i>z</i>) (M + H) = 474.0, <i>R</i><sub>t</sub> = 0.55 min. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 9.23 (d, <i>J</i> = 2.35 Hz, 1 H), 8.99 (d, <i>J</i> = 5.14 Hz, 1 H), 8.43 (d, <i>J</i> = 2.35 Hz, 1 H), 8.37 (s, 1 H), 8.19 (dd, <i>J</i> = 5.01, 1.54 Hz, 1 H), 7.54 (d, <i>J</i> = 2.25 Hz, 1 H), 6.97 (d, <i>J</i> = 2.30 Hz, 1 H), 3.83–3.91 (m, 4 H), 3.67 (s, 3 H), 3.14–3.22 (m, 4 H), 2.72 (s, 3 H). HRMS <i>m</i>/<i>z</i> (M<sup>+</sup> + 1) calcd 474.1753, obsd 474.1755.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i>-(6′-Methoxy-2-methyl-5′-morpholino-[3,3′-bipyridin]-5-yl)-3-(trifluoromethyl)benzamide (<b>11</b>)</h4><div class="NLM_p last">Obtained in 28% yield as a light-yellow solid by following the general procedure for the Suzuki–Miyaura reaction. LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 473.3, <i>R</i><sub>t</sub> = 0.73 min. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 9.32 (d, <i>J</i> = 2.4 Hz, 1H), 8.43 (d, <i>J</i> = 2.4 Hz, 1H), 8.34 (s, 1H), 8.28 (d, <i>J</i> = 8.2 Hz, 1H), 7.96 (d, <i>J</i> = 7.8 Hz, 1H), 7.88 (d, <i>J</i> = 2.0 Hz, 1H), 7.78 (t, <i>J</i> = 7.8 Hz, 1H), 7.31 (d, <i>J</i> = 2.0 Hz, 1H), 4.05 (s, 3H), 3.80–3.91 (m, 4H), 3.09–3.18 (m, 4H), 2.67 (s, 3H). HRMS <i>m</i>/<i>z</i> (M<sup>+</sup> + 1) calcd 473.1801, obsd 473.1801.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Procedure for S<sub>N</sub>Ar</h3><div class="NLM_p">To a mixture of sodium hydride, 60% in mineral oil (3.0–5.2 mmol) in 1,4-dioxane or DMF (7 mL) at ambient temperature was added the appropriate alcohol (3.0–5.0 mmol). The mixture was stirred for 15–30 min. <i>N</i>-(6′-Fluoro-2-methyl-5′-morpholino-[3,3′-bipyridin]-5-yl)-3-(trifluoromethyl)benzamide (1.0 mmol) was added, and the reaction was stirred at 80–105 °C for 1–4 h. The cooled reaction mixture was poured into water (50 mL) and extracted with EtOAc (2 × 50 mL). The combined organics were washed with brine (25 mL), dried over sodium sulfate, filtered, and concentrated onto silica gel. The mixture was purified by flash chromatography (24 g silica gel, DCM with a 0–10% MeOH gradient over 20 min). Pure product fractions were concentrated to give the desired product. The products could be purified further by reverse phase HPLC or recrystallization from acetonitrile.</div><div class="NLM_p">The following compounds were prepared by the above method starting from the appropriate alcohol.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i>-(6′-Isopropoxy-2-methyl-5′-morpholino-[3,3′-bipyridin]-5-yl)-3-(trifluoromethyl)benzamide Trifluoroacetic Acid Salt (<b>12</b>)</h4><div class="NLM_p last">Obtained in 56% yield as an off-white solid. LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 501.1, <i>R</i><sub>t</sub> = 0.87 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.92 (s, 1H), 9.03 (s, 1H), 8.33 (s, 1H), 8.21–8.31 (m, 2H), 8.02 (d, <i>J</i> = 7.8 Hz, 1H), 7.78–7.86 (m, 2H), 7.24 (d, <i>J</i> = 2.0 Hz, 1H), 5.35 (dt, <i>J</i> = 12.2, 6.2 Hz, 1H), 3.67–3.78 (m, 4H), 3.06 (m, 4H), 2.52–2.56 (s, 3H), 1.34 (d, <i>J</i> = 6.3 Hz, 6H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.99, 159.01, 158.72, 155.56, 149.30, 138.29, 135.95, 135.72, 135.31, 135.03, 133.37, 132.48, 130.51, 129.96, 129.70, 129.33, 126.64, 125.56, 125.45, 124.82, 123.28, 68.51, 66.54,50.27, 22.41, 20.85. HRMS <i>m</i>/<i>z</i> (M<sup>+</sup> + 1) calcd 501.2114, obsd 501.212; mp 204 °C.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>N</i>-(6′-(2-Hydroxyethoxy)-2-methyl-5′-morpholino-[3,3′-bipyridin]-5-yl)-3-(trifluoromethyl)benzamide (<b>13</b>)</h4><div class="NLM_p last">Obtained in 60% yield as a white solid. LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 503.2, <i>R</i><sub>t</sub> = 0.66 min. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 8.63 (d, <i>J</i> = 2.4 Hz, 1H), 8.13–8.19 (m, 2H), 8.09 (d, <i>J</i> = 7.8 Hz, 1H), 7.95 (br s, 1H), 7.85 (d, <i>J</i> = 7.8 Hz, 1H), 7.76 (d, <i>J</i> = 1.6 Hz, 1H), 7.67 (t, <i>J</i> = 7.8 Hz, 1H), 7.11 (d, <i>J</i> = 2.0 Hz, 1H), 4.59–4.67 (m, 2H), 3.97–4.06 (m, 2H), 3.85–3.95 (m, 4H), 3.11–3.20 (m, 4H), 2.52 (s, 3H). HRMS <i>m</i>/<i>z</i> (M<sup>+</sup> + 1) calcd 503.1906, obsd 503.1907.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i>-(2-Methyl-5′-morpholino-6′-((tetrahydro-2<i>H</i>-pyran-4-yl)oxy)-[3,3′-bipyridin]-5-yl)-3-(trifluoromethyl)benzamide (<b>14</b>)</h4><div class="NLM_p last">Obtained in 66% yield as an off-white solid. LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 543.1, <i>R</i><sub>t</sub> = 0.77 min. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 8.62 (s, 1H), 8.16 (br s, 2H), 8.10 (d, <i>J</i> = 7.8 Hz, 1H), 7.86 (d, <i>J</i> = 7.8 Hz, 2H), 7.77 (d, <i>J</i> = 2.0 Hz, 1H), 7.63–7.73 (m, 1H), 7.07 (d, <i>J</i> = 2.4 Hz, 1H), 5.43 (dt, <i>J</i> = 8.1, 4.0 Hz, 1H), 3.95–4.06 (m, 2H), 3.83–3.94 (m, 4H), 3.70 (ddd, <i>J</i> = 11.5, 8.4, 3.1 Hz, 2H), 3.10–3.25 (m, 4H), 2.54 (s, 3H), 2.17 (dt, <i>J</i> = 8.7, 4.5 Hz, 2H), 1.89 (dtd, <i>J</i> = 12.8, 8.49, 8.5, 3.9 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.73, 154.68, 151.28, 140.72, 138.01, 135.84, 133.88, 133.20, 132.37, 130.33, 129.84, 129.59, 129.07, 128.89, 125.89, 125.51, 124.74, 123.34, 69.80, 66.64, 65.03, 50.40,32.19,23.22. HRMS <i>m</i>/<i>z</i> (M<sup>+</sup> + 1) calcd 543.2219, obsd 543.2218; mp 196 °C.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 4-(5-Bromo-2-methoxypyridin-3-yl)morpholine (<b>16</b>)</h4><div class="NLM_p last">To a solution of 5-bromo-2-methoxypyridin-3-amine (3.00 g, 14.8 mmol, 1.00 equiv) in DMF (40 mL) was added 1-bromo-2-(2-bromoethoxy)ethane (4.11 g, 17.7 mmol, 1.20 equiv) and DIEA (7.74 mL, 44.3 mmol, 3.00 equiv). The reaction mixture was stirred at 120 °C for 24 h. Upon cooling to room temperature, the reaction was partitioned between water and EtOAc, the aqueous phase was extracted three times with EtOAc, and the organics were combined, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude material was purified via silica gel flash chromatography eluting with heptanes and ethyl acetate (0–20%) to give 4-(5-bromo-2-methoxypyridin-3-yl)morpholine (2.80 g, 70% yield). LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 273.0/275.0. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.84 (d, <i>J</i> = 2.0 Hz, 1H), 7.14 (d, <i>J</i> = 2.0 Hz, 1H), 3.97 (s, 3H), 3.76–3.91 (m, 4H), 2.90–3.18 (m, 4H).</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 5-Bromo-1-methyl-3-morpholinopyridin-2(1<i>H</i>)-one (<b>17</b>)</h4><div class="NLM_p last">A 0.3 M solution of 4-(5-bromo-2-methoxypyridin-3-yl)morpholine (2.5 g, 9.15 mmol, 1.00 equiv) in 4 M HCl in dioxane (3.8 mL, 45.8 mmol, 5 equiv) was heated to 100 °C for 1 h. Upon cooling to room temperature, the reaction was concentrated to dryness under low pressure. The residue was suspended in water (20 mL) and neutralized with solid sodium bicarbonate. The solid was filtered, rinsed with water (2 × 10 mL), and dried to give 5-bromo-3-morpholinopyridin-2-ol in 93% yield (2.2 g). LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 258.9/260.9. To a 0.3 M solution of 5-bromo-3-morpholinopyridin-2-ol (500 mg, 1.9 mmol, 1.00 equiv) in DMF (7 mL) was added K<sub>2</sub>CO<sub>3</sub> (560 mg, 4.1 mmol, 2.10 equiv) and iodomethane (0.139 mL, 2.2 mmol, 1.1 equiv). The reaction mixture was stirred at ambient temperature overnight. LC-MS showed conversion to pyridone product and about 15–20% of O-alkylated byproduct. The reaction mixture was diluted with ethyl acetate and filtered. The filtrate was concentrated under reduced pressure with silica gel. The crude mixture was purified by flash chromatography eluting with heptanes and ethyl acetate (50–100%, then 10:1 ethyl acetate:methanol gradient) to give 5-bromo-1-methyl-3-morpholinopyridin-2(1<i>H</i>)-one (400 mg, 76% yield). LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 272.8/274.9. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.11 (d, <i>J</i> = 2.4 Hz, 1H), 6.64 (d, <i>J</i> = 2.4 Hz, 1H), 3.76–3.97 (m, 4H), 3.52 (s, 3H), 3.05–3.27 (m, 4H).</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 5-Bromo-3-morpholinopyrazin-2(1<i>H</i>)-one (<b>19</b>)</h4><div class="NLM_p last">A solution of 3,5-dibromopyrazin-2-ol (500 mg, 1.97 mmol, 1.0 equiv) was heated in morpholine (0.9 mL, 9.85 mmol, 5 equiv) at 100 °C for 18 h. Upon overnight stirring, it was cooled to room temperature and the precipitate filtered. The filtrate was partitioned between water and ethyl acetate, and the organic phase was dried with sodium sulfate, filtered, and concentrated. The yellow solid was used without further purification (310 mg). LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 259.9/261.9.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 5-Bromo-1-methyl-3-morpholinopyrazin-2(1<i>H</i>)-one (<b>20</b>)</h4><div class="NLM_p last">To a solution of 5-bromo-3-morpholinopyrazin-2(1<i>H</i>)-one (22 mg, 0.085 mmol, 1.0 equiv) in DMF (1 mL, 0.085 M) at 0 °C was added potassium carbonate (23 mg, 0.169 mmol, 2.0 equiv), followed by iodomethane (0.005 mL, 0.085 mmol, 1.0 equiv). The reaction was allowed to warm to room temperature and stirred for 2 h. Partitioned between ethyl acetate and water, the organic phase was dried with sodium sulfate, filtered, and concentrated to dryness to give 5-bromo-1-methyl-3-morpholinopyrazin-2(1<i>H</i>)-one in 91% yield as a major isomer. LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 274.0/276.0. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 6.77 (s, 1 H), 3.85–4.00 (m, 4 H), 3.66–3.83 (m, 4 H), 3.42 (s, 3 H).</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 4-(5-Bromo-2-fluoropyridin-3-yl)morpholine (<b>22</b>)</h4><div class="NLM_p last">To a stirred solution of 3-amino-5-bromo-2-fluoropyridine (10.0 g, 52.5 mmol, 1.00 equiv) in DMF (100 mL) at 0 °C was slowly added NaH (3.78 g, 157 mmol, 3.00 equiv). The mixture was allowed to warm to room temperature over 15 min and treated with bis(2-bromoethyl) ether (9.89 mL, 79.0 mmol, 1.50 equiv). The mixture was heated to 80 °C and stirred for 3 h. The cooled reaction mixture was poured into water (500 mL) and stirred for 20 min. The mixture was filtered. The filter cake was washed with water (2 × 50 mL), allowed to air-dry for a few hours, and then washed with heptane (2 × 50 mL). The resulting solid was dried to give <b>20</b> (10.5 g, 76.4% yield) as a light-brown solid which was used without further purification. LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 261.0/263.0. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 7.81 (t, <i>J</i> = 2.0 Hz, 1H), 7.32 (dd, <i>J</i> = 9.0, 2.0 Hz, 1H), 3.84–3.91 (m, 4H), 3.11–3.16 (m, 4H).</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>N</i>-(5-Bromo-6-methylpyridin-3-yl)-3-(trifluoromethyl)benzamide (<b>24</b>)</h4><div class="NLM_p last">To a solution of 3-bromo-2-methylpyridin-5-amine (510 mg, 2.73 mmol, 1.00 equiv) in DCM (10 mL) was added DIEA (0.476 mL, 2.73 mmol, 1.00 equiv) and 3-(trifluoromethyl)benzoyl chloride (569 mg, 2.73 mmol, 1.00 equiv). The mixture was stirred at ambient temperature for 2 h. The reaction mixture was washed sequentially with water (10 mL) and brine (10 mL), dried over sodium sulfate, filtered, and concentrated to give <i>N</i>-(5-bromo-6-methylpyridin-3-yl)-3-(trifluoromethyl)benzamide (980 mg, 100% yield) as an off-white solid which was used without further purification. LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 359.0/361.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.70 (m, 1H), 8.78 (m, 1H), 8.46 (m, 1H), 8.28 (m, 2H), 7.98 (m, 1H), 7.78 (m, 1H), 2.55 (m, 3H).</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> <i>N</i>-(6-Methyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridin-3-yl)-3-(trifluoromethyl)benzamide (<b>25</b>)</h4><div class="NLM_p last">To a solution of <i>N</i>-(5-bromo-6-methylpyridin-3-yl)-3-(trifluoromethyl)benzamide (10.5 g, 29.2 mmol, 1.00 equiv) in 1,4-dioxane (9 mL) was added bis(pinacolato)diboron (11.1 g, 43.9 mmol, 1.50 equiv), potassium acetate (5.74 g, 58.5 mmol, 2.00 equiv), and PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub> adduct (2.39 g, 2.92 mmol, 0.100 equiv). The reaction was stirred at reflux for 2 h. The cooled reaction mixture was diluted with ethyl acetate (20 mL) and filtered through Celite. The filtrate was concentrated and purified by flash chromatography over silica gel (220 g, heptanes with 0–100% ethyl acetate gradient) to give <i>N</i>-(6-methyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridin-3-yl)-3-(trifluoromethyl)benzamide (6.69 g, 70.7% yield) as an off-white solid. LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 325.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.43–10.60 (m, 1H), 8.88–9.01 (m, 1H), 8.22–8.42 (m, 3H), 7.90–8.05 (m, 1H), 7.68–7.84 (m, 1H), 2.53–2.65 (m, 3H), 1.31 (s, 12H).</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 2-Methyl-5-nitro-3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridine (<b>27</b>)</h4><div class="NLM_p last">To a degassed mixture of 3-bromo-2-methyl-5-nitropyridine (12.67 g, 58.4 mmol, 1.0 equiv), bis(pinacolato)diboron (17.79 g, 70.1 mmol, 1.2 equiv), and KOAc (17.19 g, 175 mmol, 3.0 equiv) in dioxane (volume: 160 mL) was added PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub> adduct (2.384 g, 2.92 mmol, 0.05 equiv). The resulting mixture was stirred at 100 °C for 3 h. The cooled mixture was diluted with ethyl acetate and filtered through Celite. The filtrate was adsorbed onto Celite and purified by flash chromatography (heptanes with 0–20% ethyl acetate gradient). The clean fractions were concentrated to give 2-methyl-5-nitro-3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridine (12.7 g, 48.1 mmol, 82% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 9.33 (d, <i>J</i> = 2.74 Hz, 1 H), 8.79 (d, <i>J</i> = 2.74 Hz, 1 H), 2.87 (s, 3 H), 1.39 (s, 12 H).</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 6-Methyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridin-3-amine (<b>28</b>)</h4><div class="NLM_p last">A degassed solution of 2-methyl-5-nitro-3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridine (12.7 g, 48.1 mmol) in MeOH (volume: 160 mL) was treated with 10% palladium on carbon (Degussa type) (0.614 g, 0.577 mmol). The mixture was purged and flushed with hydrogen three times and stirred under a hydrogen balloon overnight at ambient temperature for 17 h. The degassed mixture was filtered through Celite. The filter cake was rinsed with additional methanol. The combined filtrates were concentrated and then dried under high vacuum to give 6-methyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridin-3-amine (11.1 g, 47.4 mmol, 99% yield) as an off-white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm 8.05 (d, <i>J</i> = 3.13 Hz, 1 H), 7.38 (d, <i>J</i> = 3.13 Hz, 1 H), 2.63 (s, 3 H), 1.35 (s, 12 H).</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i>-(4-Methyl-3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl)-3-(trifluoromethyl)benzamide (<b>30</b>)</h4><div class="NLM_p last">To a solution of 4-methyl-3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)aniline (7.59 g, 32.6 mmol, 1.0 equiv) in THF (300 mL, 0.1 M) at 0 °C was added 3-trifluoromethylbenzoyl chloride (4.8 mL, 32.6 mmol, 1.0 equiv) and the reaction mixture was brought to room temperature. After 3 h, the reaction was concentrated under low pressure and dried under vacuum to give 12.6 g of the desired product in 96% yield. LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 406.2, <i>R</i><sub>t</sub> = 1.24 min.</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>N</i>-(6′-Fluoro-2-methyl-5′-morpholino-[3,3′-bipyridin]-5-yl)-3-(trifluoromethyl)benzamide (<b>31</b>)</h4><div class="NLM_p last">To a solution of 4-(5-bromo-2-fluoropyridin-3-yl)morpholine (4.65 g, 16.03 mmol) and <i>N</i>-(6-methyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridin-3-yl)-3-(trifluoromethyl)benzamide (7.81 g, 19.23 mmol) in DME (90 mL) was added 2 M aqueous sodium carbonate (24.04 mL, 48.1 mmol). Argon was bubbled through the mixture for 5 min. PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub> adduct (0.785 g, 0.962 mmol) was added, and the mixture was purged with argon briefly, fitted with an air-cooled condenser, and then heated at 100 °C for 1.5 h. The cooled reaction mixture was poured into water (80 mL) and extracted with EtOAc (2 × 120 mL). The combined organics were washed with brine (50 mL), dried over sodium sulfate, filtered, and concentrated with silica gel. The mixture was purified by flash chromatography (120 g silica gel, DCM with 0–10% MeOH gradient). Product fractions were concentrated to give <i>N</i>-(6′-fluoro-2-methyl-5′-morpholino-[3,3′-bipyridin]-5-yl)-3-(trifluoromethyl)benzamide (6.38 g, 77.7% yield) as a beige solid. LCMS <i>m</i>/<i>z</i> [M + H<sup>+</sup>] = 461.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 10.67 (s, 1H), 8.87 (d, <i>J</i> = 2.4 Hz, 1H), 8.31 (s, 1H), 8.27 (d, <i>J</i> = 7.8 Hz, 1H), 8.06 (d, <i>J</i> = 2.4 Hz, 1H), 7.98 (d, <i>J</i> = 7.8 Hz, 1H), 7.73–7.87 (m, 2H), 7.52 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 3.65–3.83 (m, 4H), 3.03–3.16 (m, 4H), 2.41 (s, 3H).</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Data Mining</h3><div class="NLM_p last">We created a tailored knowledgebase, drawing from historical RAF biochemical IC<sub>50</sub>s (all isoforms, mutant and wild-type) and cellular EC<sub>50</sub>s (target modulation or cell proliferation) from assays using cells driven by mutant RAF or mutant RAS. Compounds were selected for this knowledgebase if they had scored at least one IC<sub>50</sub> or EC<sub>50</sub> value <10 uM in one of the historical assays. The knowledgebase of 28k compounds was further annotated with scaffold information, comprised of both substructure matches from scaffolds as defined by medicinal chemists in the historical programs as well as canonical level03 and level04 scaffolds as determined by the scaffold tree algorithm.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(23)</a> The substructure-based scaffolds were used to categorize compounds as type 1, type 2, or type 1.5 inhibitors, whereas the scaffold tree scaffolds were used in the selection protocols to ensure coverage of available chemotypes. Subset selection was a combination of cheminformatics protocols and chemistry review to focus on chemical matter with historical potency (especially in target modulation assays), SAR tractability (small series, not singletons), and physical chemical properties. The cRAF biochemical assay was a filter for entry into the activation assay (requiring <1 uM IC<sub>50</sub>), resulting in a data set of around 200 compounds with cRAF IC<sub>50</sub> and cellular data; a 116 compound subset had a pMEK EC<sub>50</sub> ≤ 0.3 uM, which made them suitable for assessing activation.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Biochemical Assay</h3><div class="NLM_p last">The CRAF kinase assay was carried out using 10 nM kinase-dead MEK1 protein substrate (carrying a K97R mutation), 3 μM ATP, and 10 pM CRAF Y340E/Y341E. The reaction buffer contained 50 mM Tris pH 7.5, 10 mM MgCl<sub>2</sub>, 0.05% BSA, 50 mM NaCl, 0.01% Tween-20, and 1 mM DTT. The reactions were carried out at room temperature in a volume of 10 μL in white 384-shallow-well plates (PorxiPlate, PerkinElmer, Waltham, MA) for 40 min and stopped by adding 5 μL/well quench solution (50 mM Tris pH 7.5, 50 mM EDTA). Terminated reactions received 5 μL/well detection reagents consisting of 50 mM Tris pH 7.5, 0.01% Tween-20, 1:1000 diluted antiphospho MEK1/2 S217/S221 antibody (Cell Signaling Technology, Danvers, MA), 0.01 mg/mL each of AlphaScreen Protein A-coated acceptor beads, and streptavidin-coated donor beads (PerkinElmer). Plates were read in an EnVision plate reader (PerkinElmer) after overnight incubation at room temperature. In compound inhibition studies, compounds were tested over a concentration range of 25 μM to 1.74 × 10<sup>–6</sup> μM in 16-point, 3-fold format. DMSO was at a final concentration of 0.5%. Compounds were preincubated with CRAF for 30 min before adding substrates to start the reaction. Inhibition data were fit to a four-parameter logistic equation to calculate the IC<sub>50</sub> of the compounds <b>3</b>–<b>14</b></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Biochemical Kinase Specificity Profile of <b>7</b></h3><div class="NLM_p last">The kinase specificity profile for compound 7 reported in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> was determined as previously described.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(17, 18)</a> Compound 7 was assessed in the DiscoverX KINOME<i>scan</i> binding assay at 1 μM with activity presented as % of control = [(test compound signal – positive control signal)/(negative control signal – positive control signal]] × 100. Data for the 457 kinase KINOME<i>scan</i> are in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> In Vivo PK/PD and Efficacy</h3><div class="NLM_p">Mouse studies were undertaken in accordance with the Novartis Institutes for Biomedical Research Animal Care and Use Committee protocols and regulations. Mice were housed in a temperature- and humidity-controlled animal facility with ad libitum access to food and water and acclimated for at least 3 days before experimental procedures.</div><div class="NLM_p">For tumor PD measurements in the Calu-6 model, tumor bearing mice were randomized into treatment groups and treated with vehicle or a single oral administration of <b>14</b> at 10, 30, or 200 mg/kg. Tumor samples were collected at 1, 3, 8, 16, and 24 h post single dose of <b>14</b> (<i>n</i> = 3/time point) and analyzed for levels of phosphorylated ERK1/2 (Thr202/Tyr204; Thr185/Tyr187) and total ERK using the MesoScale Discovery (MSD) platform. Briefly, the pulverized tumor was lysed in Tris lysis buffer (supplied by MesoScale Discovery Multi-Spot assay system) and centrifuged at 11000 rpm for 20 min. (4 °C). Supernatants were collected for protein lysates. Relative amounts of phosphorylated, and total ERK were determined using the Multi-Spot Assay System K151CWD-2 and K151DUD-2 from MSD. Plasma samples were collected at 0.25, 0.5, 1, 3, 8, 16, and 24 post single dose of <b>14</b> (<i>n</i> = 3/time point), and concentrations of <b>14</b> were measured by LC-MS.</div><div class="NLM_p last">For efficacy in Calu-6 xenograft tumors, 10 × 10<sup>6</sup> Calu-6 cells in 50% Matrigel were implanted subcutaneously into the right flank of female nude mice (6–8 weeks old). Mice were randomized into treatment groups on day 12 post implantation, when the average tumor volume was ∼400 mm<sup>3</sup>. Mice were grouped (<i>n</i> = 8) and treated with vehicle (20% Captisol) or <b>14</b> at 10, 30, or 200 mg/kg qd. Treatments began on day 12 and continued until day 27 post implantation. Tumor volume and body weights were collected at the time of randomization and twice per week for the study duration. Tumor volume was determined by measurement with calipers and calculated using a modified ellipsoid formula, where tumor volume (TV) (mm<sup>3</sup>) = [((<i>l</i> × <i>w</i><sup>2</sup>) × 3.14159))/6], where <i>l</i> is the longest axis of the tumor and <i>w</i> is perpendicular to <i>l</i>. Mice were monitored for tumor growth, body weight, and body condition. Animal well-being and behavior were monitored twice weekly. General health of mice was monitored daily.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Crystallization of Wild-Type BRAF</h3><div class="NLM_p last">The kinase domain of wild-type BRAF (residues 445–723) was expressed using the baculovirus-expression system, purified by Ni-NTA chromatography, and concentrated to 2 mg/mL in a buffer composed of 20 mM Bis-Tris propane pH 7.0, 15% glycerol, and 1 mM TCEP plus 0.1% CHAPS. Crystals with wild-type BRAF were grown by the vapor diffusion method at 291 K in hanging drops. Crystals were grown at 18 °C by mixing equal volumes of protein incubated with 0.3 mM of an internal control compound and the following well solution: 100 mM Tris pH 8.4, 100 mM NaCl, and 12% PEG8000. The internal control compound was then displaced by soaking the crystal overnight in cryosolution of 100 mM Tris pH 8.4, 15% PEG8000, and 20% glycerol supplemented with 300 μM of the desired target compound. However, cocrystals can also be obtained by complexing wild-type BRAF protein directly with the target compound and performing cocrystallization experiments as described above.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Data Processing and Refinement</h3><div class="NLM_p last">Data were collected at beamlines 5.0.1 and 5.0.2 of the Advanced Light Source at Lawrence Berkeley National Laboratory (Berkeley, CA) and processed with autoPROC.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(24)</a> Structures were solved by molecular replacement using the 5CT7 BRAF structure as search model in PHASER<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(25)</a> and refined with BUSTER.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(26, 27)</a> All models were built with COOT.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Small Molecule Crystal Structure</h3><div class="NLM_p">Single crystals of <b>7</b> suitable for X-ray structural investigation were obtained from a methanol solution by slowly evaporating the solvents. Diffraction data were collected at 100 K on a Bruker AXS MicroStar diffractometer using a SMART 6000 CCD detector on a three-circle platform goniometer with Cu Kα radiation (λ = 1.54178 Å) from a microfocus rotating generator equipped with Incoatec multilayer optics (Helios confocal mirror system). Sixteen ω-scans at different ϕ-positions were performed to ensure appropriate data redundancy (11.1, Friedel pairs merged).</div><div class="NLM_p">Collected images were processed using SAINT software (SAINT V7.36A, Bruker AXS 2006). Scaling and absorption correction (multiscan method) of the raw data were performed with program SADABS (SADABS V2008/1, Sheldrick G.M. 2008). The semiempirical absorption correction is based on the intensities of symmetry-related reflections measured at different angular settings (minimum and maximum transmission 0.931 and 0.973; ratio of minimum to maximum transmission is 0.957). Assignment of space group and setup of files for structure solution was done using program XPREP (XPREP V2008/2, Sheldrick, G. M., 2008).</div><div class="NLM_p">The structure was solved by dual-space recycling methods and refined on <i>F</i><sup>2</sup> with the SHELXTL suite of programs (SHELXTL V6.12, Sheldrick, G. M., 2001). Anisotropic displacement parameters were used for all non-hydrogen atoms. Hydrogen atoms were unambiguously located in difference Fourier maps and refined in idealized positions using rigid methyl or hydroxyl groups or a riding model. The structure consists of two independent molecules. In molecule 2, all numbers used for molecule 1 have been increased by 40. Data are deposited under CCDC 1508642.</div><div class="NLM_p last">Final data: C<sub>25</sub>H<sub>24</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub>, <i>M</i><sub>r</sub> = 471.47, crystal size 0.08 × 0.06 × 0.03 mm<sup>3</sup> (grown from methanol), monoclinic, space group <i>P</i>21/<i>n</i> (no. 14) with <i>a</i> = 16.334 (3), <i>b</i> = 13.727 (3), <i>c</i> = 21.539(4) Å, β = 108.839(11)°, <i>V</i> = 4570.7(16) Å<sup>3</sup>, <i>Z</i> = 8, <i>D</i><sub>c</sub> = 1.370 g.cm<sup>–3</sup>, μ = 0.908 mm<sup>–1</sup>, <i>F</i>(000) = 1968, 94814 reflections measured, 8382 independent, <i>R</i><sub>int</sub> = 0.0508, 2.98° < θ < 68.22°, <i>T</i> = 100(2) K, 617 parameters, <i>R</i><sub>1</sub> = 0.0354, <i>wR</i><sub>2</sub> = 0.0846 for 8382 reflections with <i>I</i> > 2σ (<i>I</i>), <i>R</i><sub>1</sub> = 0.0452, <i>wR</i><sub>2</sub> = 0.0910 for all 8382 data, GoF = 1.034, restrained GoF = 1.034, res el dens = +0.30/–0.34 e·Å<sup>–3</sup>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01862">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38783" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38783" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01862" class="ext-link">10.1021/acs.jmedchem.6b01862</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Cellular assay conditions, compound <b>7</b> data for the 457 kinase KINOME<i>scan</i>, X-ray data table for <b>3</b> and <b>14</b>, extended data for the small molecule crystal structure <b>7</b>, PC3 xenograft study data for <b>14</b> with experimental details (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Machine-readable representation of structures with key data (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_001.pdf">jm6b01862_si_001.pdf (843.38 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_002.csv">jm6b01862_si_002.csv (0.84 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b01862" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00457" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00457" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Savithri Ramurthy</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2444-5309" title="Orcid link">http://orcid.org/0000-0002-2444-5309</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#95e6f4e3fce1fde7fcbbe7f4f8e0e7e1fdecd5fbfae3f4e7e1fce6bbf6faf8"><span class="__cf_email__" data-cfemail="95e6f4e3fce1fde7fcbbe7f4f8e0e7e1fdecd5fbfae3f4e7e1fce6bbf6faf8">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gisele A. Nishiguchi</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alice Rico</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huw Tanner</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Aversa</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9873-4381" title="Orcid link">http://orcid.org/0000-0002-9873-4381</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Benjamin R. Taft</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sharadha Subramanian</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lina Setti</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew T. Burger</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lifeng Wan</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Victoriano Tamez</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aaron Smith</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan Lou</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul A. Barsanti</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brent A. Appleton</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mulugeta Mamo</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura Tandeske</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ina Dix</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, Novartis Pharma AG, Werk Klybeck, Postfach, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John E. Tellew</span> - <span class="hlFld-Affiliation affiliation">Genomics
Institute of the Novartis Research Foundation, 10675 John Hopkins Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shenlin Huang</span> - <span class="hlFld-Affiliation affiliation">Genomics
Institute of the Novartis Research Foundation, 10675 John Hopkins Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lesley A. Mathews Griner</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vesselina G. Cooke</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anne Van Abbema</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hanne Merritt</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sylvia Ma</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kalyani Gampa</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fei Feng</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Yuan</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yingyun Wang</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacob R. Haling</span> - <span class="hlFld-Affiliation affiliation">Genomics
Institute of the Novartis Research Foundation, 10675 John Hopkins Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sepideh Vaziri</span> - <span class="hlFld-Affiliation affiliation">Genomics
Institute of the Novartis Research Foundation, 10675 John Hopkins Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mohammad Hekmat-Nejad</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johanna M. Jansen</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Valery Polyakov</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard Zang</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vijay Sethuraman</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Payman Amiri</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mallika Singh</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emma Lees</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenlin Shao</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darrin D. Stuart</span> - <span class="hlFld-Affiliation affiliation">Oncology, Novartis Institutes for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael P. Dillon</span> - <span class="hlFld-Affiliation affiliation">Global
Discovery Chemistry, Novartis Institutes
for BioMedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d90e3623-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i51">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27357" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27357" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dazhi Tang, Elaine Ginn, Colin Lorentzen, Kent Wong and Linda Xiao for generating solubility, in vitro ADME data as well as Shengtian Yang for running NMR structural elucidation of <b>14</b>.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i52" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i52"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i53" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i53"> Abbreviations Used</h2><tr><td class="NLM_term">Fsp<sup>3</sup></td><td class="NLM_def"><p class="first last">fraction sp<sup>3</sup></p></td></tr><tr><td class="NLM_term">Sol</td><td class="NLM_def"><p class="first last">solubility</p></td></tr><tr><td class="NLM_term">CP</td><td class="NLM_def"><p class="first last">cell proliferation</p></td></tr><tr><td class="NLM_term">Cl</td><td class="NLM_def"><p class="first last">clearance</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>ss</sub></td><td class="NLM_def"><p class="first last">volume of distribution</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-(dimethylamino)propyl)carbodiimide</p></td></tr><tr><td class="NLM_term">HOAt</td><td class="NLM_def"><p class="first last">1-hydroxy-7-azabenzotriazole</p></td></tr><tr><td class="NLM_term">SNAr</td><td class="NLM_def"><p class="first last">nucleophilic aromatic substitution</p></td></tr><tr><td class="NLM_term"><i>R</i><sub>t</sub></td><td class="NLM_def"><p class="first last">retention time</p></td></tr><tr><td class="NLM_term">TCEP</td><td class="NLM_def"><p class="first last">(tris(2-carbosyethyl) phosphine)</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i54">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42176" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42176" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 27 other publications.
        </p><ol id="references" class="useLabel"><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Ostrem, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sos, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">503</span><span class="NLM_x"> (</span><span class="NLM_issue">7477</span><span class="NLM_x">) </span> <span class="NLM_fpage">548</span><span class="NLM_x">–</span> <span class="NLM_lpage">551</span><span class="refDoi"> DOI: 10.1038/nature12796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1038%2Fnature12796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=24256730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=548-551&issue=7477&author=J.+M.+Ostremauthor=U.+Petersauthor=M.+L.+Sosauthor=J.+A.+Wellsauthor=K.+M.+Shokat&title=K-Ras%28G12C%29+inhibitors+allosterically+control+GTP+affinity+and+effector+interactions&doi=10.1038%2Fnature12796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span></div><div class="casAuthors">Ostrem, Jonathan M.; Peters, Ulf; Sos, Martin L.; Wells, James A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7477</span>),
    <span class="NLM_cas:pages">548-551</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Somatic mutations in the small GTPase K-Ras are the most common activating lesions found in human cancer, and are generally assocd. with poor response to std. therapies.  Efforts to target this oncogene directly have faced difficulties owing to its picomolar affinity for GTP/GDP and the absence of known allosteric regulatory sites.  Oncogenic mutations result in functional activation of Ras family proteins by impairing GTP hydrolysis.  With diminished regulation by GTPase activity, the nucleotide state of Ras becomes more dependent on relative nucleotide affinity and concn.  This gives GTP an advantage over GDP and increases the proportion of active GTP-bound Ras.  Here we report the development of small mols. that irreversibly bind to a common oncogenic mutant, K-Ras(G12C).  These compds. rely on the mutant cysteine for binding and therefore do not affect the wild-type protein.  Crystallog. studies reveal the formation of a new pocket that is not apparent in previous structures of Ras, beneath the effector binding switch-II region.  Binding of these inhibitors to K-Ras(G12C) disrupts both switch-I and switch-II, subverting the native nucleotide preference to favor GDP over GTP and impairing binding to Raf.  Our data provide structure-based validation of a new allosteric regulatory site on Ras that is targetable in a mutant-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokkgcVXuBn_rVg90H21EOLACvtfcHk0lj4W6785dn8UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO&md5=27c1e7d8ef6b7e4b2919e9f00a073923</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature12796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12796%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DK-Ras%2528G12C%2529%2520inhibitors%2520allosterically%2520control%2520GTP%2520affinity%2520and%2520effector%2520interactions%26jtitle%3DNature%26date%3D2013%26volume%3D503%26issue%3D7477%26spage%3D548%26epage%3D551%26doi%3D10.1038%2Fnature12796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Lito, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span> </span><span class="NLM_article-title">Allele-specific inhibitors inactivate mutant KRAS G12C by a trapping mechanism</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">351</span><span class="NLM_x"> (</span><span class="NLM_issue">6273</span><span class="NLM_x">) </span> <span class="NLM_fpage">604</span><span class="NLM_x">–</span> <span class="NLM_lpage">608</span><span class="refDoi"> DOI: 10.1126/science.aad6204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1126%2Fscience.aad6204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=26841430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2jt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=604-608&issue=6273&author=P.+Litoauthor=M.+Solomonauthor=L.+S.+Liauthor=R.+Hansenauthor=N.+Rosen&title=Allele-specific+inhibitors+inactivate+mutant+KRAS+G12C+by+a+trapping+mechanism&doi=10.1126%2Fscience.aad6204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Allele-specific inhibitors inactivate mutant KRAS G12C by a trapping mechanism</span></div><div class="casAuthors">Lito, Piro; Solomon, Martha; Li, Lian-Sheng; Hansen, Rasmus; Rosen, Neal</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6273</span>),
    <span class="NLM_cas:pages">604-608</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">It is thought that KRAS oncoproteins are constitutively active because their guanosine triphosphatase (GTPase) activity is disabled.  Consequently, drugs targeting the inactive or GDP-bound conformation are not expected to be effective.  We describe a mechanism that enables such drugs to inhibit KRASG12C signaling and cancer cell growth.  Inhibition requires intact GTPase activity and occurs because drug-bound KRASG12C is insusceptible to nucleotide exchange factors and thus trapped in its inactive state.  Indeed, mutants completely lacking GTPase activity and those promoting exchange reduced the potency of the drug.  Suppressing nucleotide exchange activity downstream of various tyrosine kinases enhanced KRASG12C inhibition, whereas its potentiation had the opposite effect.  These findings reveal that KRASG12C undergoes nucleotide cycling in cancer cells and provide a basis for developing effective therapies to treat KRASG12C-driven cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqK3AbqEZO6k7Vg90H21EOLACvtfcHk0lj4W6785dn8UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2jt78%253D&md5=22a47908ac0146822ecf19f5d0c952de</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad6204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad6204%26sid%3Dliteratum%253Aachs%26aulast%3DLito%26aufirst%3DP.%26aulast%3DSolomon%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DL.%2BS.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DAllele-specific%2520inhibitors%2520inactivate%2520mutant%2520KRAS%2520G12C%2520by%2520a%2520trapping%2520mechanism%26jtitle%3DScience%26date%3D2016%26volume%3D351%26issue%3D6273%26spage%3D604%26epage%3D608%26doi%3D10.1126%2Fscience.aad6204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Patricelli, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janes, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kucharski, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ely, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Firdaus, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babbar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span> </span><span class="NLM_article-title">Selective inhibition of oncogenic KRAS output with small molecules targeting the inactive state</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">316</span><span class="NLM_x">–</span> <span class="NLM_lpage">329</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-1105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1158%2F2159-8290.CD-15-1105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=26739882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVyru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=316-329&issue=3&author=M.+P.+Patricelliauthor=M.+R.+Janesauthor=L.+S.+Liauthor=R.+Hansenauthor=U.+Petersauthor=L.+V.+Kesslerauthor=Y.+Chenauthor=J.+M.+Kucharskiauthor=J.+Fengauthor=T.+Elyauthor=J.+H.+Chenauthor=S.+J.+Firdausauthor=A.+Babbarauthor=P.+Renauthor=Y.+Liu&title=Selective+inhibition+of+oncogenic+KRAS+output+with+small+molecules+targeting+the+inactive+state&doi=10.1158%2F2159-8290.CD-15-1105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Oncogenic KRAS Output with Small Molecules Targeting the Inactive State</span></div><div class="casAuthors">Patricelli, Matthew P.; Janes, Matthew R.; Li, Lian-Sheng; Hansen, Rasmus; Peters, Ulf; Kessler, Linda V.; Chen, Yuching; Kucharski, Jeff M.; Feng, Jun; Ely, Tess; Chen, Jeffrey H.; Firdaus, Sarah J.; Babbar, Anjali; Ren, Pingda; Liu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">316-329</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">KRAS gain-of-function mutations occur in approx. 30% of all human cancers.  Despite more than 30 years of KRAS-focused research and development efforts, no targeted therapy has been discovered for cancers with KRAS mutations.  Here, we describe ARS-853, a selective, covalent inhibitor of KRASG12C that inhibits mutant KRAS-driven signaling by binding to the GDP-bound oncoprotein and preventing activation.  Based on the rates of engagement and inhibition obsd. for ARS-853, along with a mutant-specific mass spectrometry-based assay for assessing KRAS activation status, we show that the nucleotide state of KRASG12C is in a state of dynamic flux that can be modulated by upstream signaling factors.  These studies provide convincing evidence that the KRASG12C mutation generates a "hyperexcitable" rather than a "statically active" state and that targeting the inactive, GDP-bound form is a promising approach for generating novel anti-RAS therapeutics.  Significance: A cell-active, mutant-specific, covalent inhibitor of KRASG12C is described that targets the GDP-bound, inactive state and prevents subsequent activation.  Using this novel compd., we demonstrate that KRASG12C oncoprotein rapidly cycles bound nucleotide and responds to upstream signaling inputs to maintain a highly active state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJJA-9IIHeN7Vg90H21EOLACvtfcHk0ljUSbXsppTCng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVyru7s%253D&md5=41db2f02006e8f0be7a9bb47354addcd</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-1105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-1105%26sid%3Dliteratum%253Aachs%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DLi%26aufirst%3DL.%2BS.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DKessler%26aufirst%3DL.%2BV.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKucharski%26aufirst%3DJ.%2BM.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DEly%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DFirdaus%26aufirst%3DS.%2BJ.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520oncogenic%2520KRAS%2520output%2520with%2520small%2520molecules%2520targeting%2520the%2520inactive%2520state%26jtitle%3DCancer%2520Discovery%26date%3D2016%26volume%3D6%26issue%3D3%26spage%3D316%26epage%3D329%26doi%3D10.1158%2F2159-8290.CD-15-1105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Karreth, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frese, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeNicola, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baccarini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuveson, D. A.</span><span> </span><span class="NLM_article-title">C-Raf is required for the initiation of lung cancer by K-Ras (G12D)</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">128</span><span class="NLM_x">–</span> <span class="NLM_lpage">136</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-10-0044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1158%2F2159-8290.CD-10-0044" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2011&pages=128-136&issue=2&author=F.+A.+Karrethauthor=K.+K.+Freseauthor=G.+M.+DeNicolaauthor=M.+Baccariniauthor=D.+A.+Tuveson&title=C-Raf+is+required+for+the+initiation+of+lung+cancer+by+K-Ras+%28G12D%29&doi=10.1158%2F2159-8290.CD-10-0044"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-10-0044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-10-0044%26sid%3Dliteratum%253Aachs%26aulast%3DKarreth%26aufirst%3DF.%2BA.%26aulast%3DFrese%26aufirst%3DK.%2BK.%26aulast%3DDeNicola%26aufirst%3DG.%2BM.%26aulast%3DBaccarini%26aufirst%3DM.%26aulast%3DTuveson%26aufirst%3DD.%2BA.%26atitle%3DC-Raf%2520is%2520required%2520for%2520the%2520initiation%2520of%2520lung%2520cancer%2520by%2520K-Ras%2520%2528G12D%2529%26jtitle%3DCancer%2520Discovery%26date%3D2011%26volume%3D1%26issue%3D2%26spage%3D128%26epage%3D136%26doi%3D10.1158%2F2159-8290.CD-10-0044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Blasco, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francoz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santamaria, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canamero, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubus, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charron, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baccarini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbacid, M.</span><span> </span><span class="NLM_article-title">C-Raf, but not B-Raf, is essential for development of K-Ras oncogene-driven on-small cell lung carcinoma</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">652</span><span class="NLM_x">–</span> <span class="NLM_lpage">663</span><span class="refDoi"> DOI: 10.1016/j.ccr.2011.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1016%2Fj.ccr.2011.04.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=652-663&issue=5&author=R.+B.+Blascoauthor=S.+Francozauthor=D.+Santamariaauthor=M.+Canameroauthor=P.+Dubusauthor=J.+Charronauthor=M.+Baccariniauthor=M.+Barbacid&title=C-Raf%2C+but+not+B-Raf%2C+is+essential+for+development+of+K-Ras+oncogene-driven+on-small+cell+lung+carcinoma&doi=10.1016%2Fj.ccr.2011.04.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DBlasco%26aufirst%3DR.%2BB.%26aulast%3DFrancoz%26aufirst%3DS.%26aulast%3DSantamaria%26aufirst%3DD.%26aulast%3DCanamero%26aufirst%3DM.%26aulast%3DDubus%26aufirst%3DP.%26aulast%3DCharron%26aufirst%3DJ.%26aulast%3DBaccarini%26aufirst%3DM.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DC-Raf%252C%2520but%2520not%2520B-Raf%252C%2520is%2520essential%2520for%2520development%2520of%2520K-Ras%2520oncogene-driven%2520on-small%2520cell%2520lung%2520carcinoma%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D19%26issue%3D5%26spage%3D652%26epage%3D663%26doi%3D10.1016%2Fj.ccr.2011.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Caunt, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sale, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, S. J.</span><span> </span><span class="NLM_article-title">MEK1 and MEK2 inhibitors and cancer therapy: the long and winding road</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">577</span><span class="NLM_x">–</span> <span class="NLM_lpage">592</span><span class="refDoi"> DOI: 10.1038/nrc4000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1038%2Fnrc4000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=26399658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGhsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=577-592&issue=10&author=C.+J.+Cauntauthor=M.+J.+Saleauthor=P.+D.+Smithauthor=S.+J.+Cook&title=MEK1+and+MEK2+inhibitors+and+cancer+therapy%3A+the+long+and+winding+road&doi=10.1038%2Fnrc4000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">MEK1 and MEK2 inhibitors and cancer therapy: the long and winding road</span></div><div class="casAuthors">Caunt, Christopher J.; Sale, Matthew J.; Smith, Paul D.; Cook, Simon J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">577-592</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The role of the ERK signaling pathway in cancer is thought to be most prominent in tumors in which mutations in the receptor tyrosine kinases RAS, BRAF, CRAF, MEK1 or MEK2 drive growth factor-independent ERK1 and ERK2 activation and thence inappropriate cell proliferation and survival.  New drugs that inhibit RAF or MEK1 and MEK2 have recently been approved or are currently undergoing late-stage clin. evaluation.  In this Review, the authors consider the ERK pathway, focusing particularly on the role of MEK1 and MEK2, the 'gatekeepers' of ERK1/2 activity.  The authors discuss their validation as drug targets, the merits of targeting MEK1 and MEK2 vs. BRAF and the mechanisms of action of different inhibitors of MEK1 and MEK2.  The authors also consider how some of the systems-level properties (intrapathway regulatory loops and wider signaling network connections) of the ERK pathway present a challenge for the success of MEK1 and MEK2 inhibitors, discuss mechanisms of resistance to these inhibitors, and review their clin. progress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnTcSz8CvRcLVg90H21EOLACvtfcHk0ljKJp2WkNBayA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGhsb%252FO&md5=b1fd44051024a59608dd61310a437c65</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrc4000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc4000%26sid%3Dliteratum%253Aachs%26aulast%3DCaunt%26aufirst%3DC.%2BJ.%26aulast%3DSale%26aufirst%3DM.%2BJ.%26aulast%3DSmith%26aufirst%3DP.%2BD.%26aulast%3DCook%26aufirst%3DS.%2BJ.%26atitle%3DMEK1%2520and%2520MEK2%2520inhibitors%2520and%2520cancer%2520therapy%253A%2520the%2520long%2520and%2520winding%2520road%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26issue%3D10%26spage%3D577%26epage%3D592%26doi%3D10.1038%2Fnrc4000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Joseph, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pratilas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poulikakos, P. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halilovic, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Persaud, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xing, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viale, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solit, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span> </span><span class="NLM_article-title">The RAF inhibitor PLX4032 inhibits ERK signaling and tumor cell proliferation in a V600E BRAF-selective manner</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x"> (</span><span class="NLM_issue">33</span><span class="NLM_x">) </span> <span class="NLM_fpage">14903</span><span class="NLM_x">–</span> <span class="NLM_lpage">14908</span><span class="refDoi"> DOI: 10.1073/pnas.1008990107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1073%2Fpnas.1008990107" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=14903-14908&issue=33&author=E.+W.+Josephauthor=C.+A.+Pratilasauthor=P.+I.+Poulikakosauthor=M.+Tadiauthor=W.+Wangauthor=B.+S.+Taylorauthor=E.+Halilovicauthor=Y.+Persaudauthor=F.+Xingauthor=A.+Vialeauthor=J.+Tsaiauthor=P.+B.+Chapmanauthor=G.+Bollagauthor=D.+B.+Solitauthor=N.+Rosen&title=The+RAF+inhibitor+PLX4032+inhibits+ERK+signaling+and+tumor+cell+proliferation+in+a+V600E+BRAF-selective+manner&doi=10.1073%2Fpnas.1008990107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1008990107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1008990107%26sid%3Dliteratum%253Aachs%26aulast%3DJoseph%26aufirst%3DE.%2BW.%26aulast%3DPratilas%26aufirst%3DC.%2BA.%26aulast%3DPoulikakos%26aufirst%3DP.%2BI.%26aulast%3DTadi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DTaylor%26aufirst%3DB.%2BS.%26aulast%3DHalilovic%26aufirst%3DE.%26aulast%3DPersaud%26aufirst%3DY.%26aulast%3DXing%26aufirst%3DF.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DChapman%26aufirst%3DP.%2BB.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DSolit%26aufirst%3DD.%2BB.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DThe%2520RAF%2520inhibitor%2520PLX4032%2520inhibits%2520ERK%2520signaling%2520and%2520tumor%2520cell%2520proliferation%2520in%2520a%2520V600E%2520BRAF-selective%2520manner%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26issue%3D33%26spage%3D14903%26epage%3D14908%26doi%3D10.1073%2Fpnas.1008990107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Hatzivassiliou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandhuber, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarado, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludlam, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stokoe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gloor, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vigers, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morales, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aliagas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sideris, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeflich, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaiswal, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seshagiri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeppen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedman, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malek, S.</span><span> </span><span class="NLM_article-title">RAF inhibitors prime wild-type RAF to activate the MAPK pathway and enhance growth</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">464</span><span class="NLM_x"> (</span><span class="NLM_issue">7287</span><span class="NLM_x">) </span> <span class="NLM_fpage">431</span><span class="NLM_x">–</span> <span class="NLM_lpage">435</span><span class="refDoi"> DOI: 10.1038/nature08833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1038%2Fnature08833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=20130576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVygtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=431-435&issue=7287&author=G.+Hatzivassiliouauthor=K.+Songauthor=I.+Yenauthor=B.+J.+Brandhuberauthor=D.+J.+Andersonauthor=R.+Alvaradoauthor=M.+J.+Ludlamauthor=D.+Stokoeauthor=S.+L.+Gloorauthor=G.+Vigersauthor=T.+Moralesauthor=I.+Aliagasauthor=B.+Liuauthor=S.+Siderisauthor=K.+P.+Hoeflichauthor=B.+S.+Jaiswalauthor=S.+Seshagiriauthor=H.+Koeppenauthor=M.+Belvinauthor=L.+S.+Friedmanauthor=S.+Malek&title=RAF+inhibitors+prime+wild-type+RAF+to+activate+the+MAPK+pathway+and+enhance+growth&doi=10.1038%2Fnature08833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">RAF inhibitors prime wild-type RAF to activate the MAPK pathway and enhance growth</span></div><div class="casAuthors">Hatzivassiliou, Georgia; Song, Kyung; Yen, Ivana; Brandhuber, Barbara J.; Anderson, Daniel J.; Alvarado, Ryan; Ludlam, Mary J. C.; Stokoe, David; Gloor, Susan L.; Vigers, Guy; Morales, Tony; Aliagas, Ignacio; Liu, Bonnie; Sideris, Steve; Hoeflich, Klaus P.; Jaiswal, Bijay S.; Seshagiri, Somasekar; Koeppen, Hartmut; Belvin, Marcia; Friedman, Lori S.; Malek, Shiva</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7287</span>),
    <span class="NLM_cas:pages">431-435</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activating mutations in KRAS and BRAF are found in more than 30% of all human tumors and 40% of melanoma, resp., thus targeting this pathway could have broad therapeutic effects.  Small mol. ATP-competitive RAF kinase inhibitors have potent antitumor effects on mutant BRAF(V600E) tumors but, in contrast to mitogen-activated protein kinase kinase (MEK) inhibitors, are not potent against RAS mutant tumor models, despite RAF functioning as a key effector downstream of RAS and upstream of MEK.  Here we show that ATP-competitive RAF inhibitors have two opposing mechanisms of action depending on the cellular context.  In BRAF(V600E) tumors, RAF inhibitors effectively block the mitogen-activated protein kinase (MAPK) signalling pathway and decrease tumor growth.  Notably, in KRAS mutant and RAS/RAF wild-type tumors, RAF inhibitors activate the RAF-MEK-ERK pathway in a RAS-dependent manner, thus enhancing tumor growth in some xenograft models.  Inhibitor binding activates wild-type RAF isoforms by inducing dimerization, membrane localization and interaction with RAS-GTP.  These events occur independently of kinase inhibition and are, instead, linked to direct conformational effects of inhibitors on the RAF kinase domain.  On the basis of these findings, we demonstrate that ATP-competitive kinase inhibitors can have opposing functions as inhibitors or activators of signalling pathways, depending on the cellular context.  Furthermore, this work provides new insights into the therapeutic use of ATP-competitive RAF inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGounpcMETzVoLVg90H21EOLACvtfcHk0ljKJp2WkNBayA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVygtb8%253D&md5=eb48bdf352ee9c9e182e63e38fd820a5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature08833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08833%26sid%3Dliteratum%253Aachs%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DBrandhuber%26aufirst%3DB.%2BJ.%26aulast%3DAnderson%26aufirst%3DD.%2BJ.%26aulast%3DAlvarado%26aufirst%3DR.%26aulast%3DLudlam%26aufirst%3DM.%2BJ.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DGloor%26aufirst%3DS.%2BL.%26aulast%3DVigers%26aufirst%3DG.%26aulast%3DMorales%26aufirst%3DT.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DJaiswal%26aufirst%3DB.%2BS.%26aulast%3DSeshagiri%26aufirst%3DS.%26aulast%3DKoeppen%26aufirst%3DH.%26aulast%3DBelvin%26aufirst%3DM.%26aulast%3DFriedman%26aufirst%3DL.%2BS.%26aulast%3DMalek%26aufirst%3DS.%26atitle%3DRAF%2520inhibitors%2520prime%2520wild-type%2520RAF%2520to%2520activate%2520the%2520MAPK%2520pathway%2520and%2520enhance%2520growth%26jtitle%3DNature%26date%3D2010%26volume%3D464%26issue%3D7287%26spage%3D431%26epage%3D435%26doi%3D10.1038%2Fnature08833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Heidorn, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milagre, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittaker, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nourry, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niculescu-Duvas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhomen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reis-Filho, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Springer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marais, R.</span><span> </span><span class="NLM_article-title">Kinase-dead BRAF and oncogenic RAS cooperate to drive tumor progression through CRAF</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">140</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">209</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1016/j.cell.2009.12.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1016%2Fj.cell.2009.12.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=20141835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvVWitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2010&pages=209-221&issue=2&author=S.+J.+Heidornauthor=C.+Milagreauthor=S.+Whittakerauthor=A.+Nourryauthor=I.+Niculescu-Duvasauthor=N.+Dhomenauthor=J.+Hussainauthor=J.+S.+Reis-Filhoauthor=C.+J.+Springerauthor=C.+Pritchardauthor=R.+Marais&title=Kinase-dead+BRAF+and+oncogenic+RAS+cooperate+to+drive+tumor+progression+through+CRAF&doi=10.1016%2Fj.cell.2009.12.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase-dead BRAF and oncogenic RAS cooperate to drive tumor progression through CRAF</span></div><div class="casAuthors">Heidorn, Sonja J.; Milagre, Carla; Whittaker, Steven; Nourry, Arnaud; Niculescu-Duvas, Ion; Dhomen, Nathalie; Hussain, Jahan; Reis-Filho, Jorge S.; Springer, Caroline J.; Pritchard, Catrin; Marais, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">209-221</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We describe a mechanism of tumorigenesis mediated by kinase-dead BRAF in the presence of oncogenic RAS.  We show that drugs that selectively inhibit BRAF drive RAS-dependent BRAF binding to CRAF, CRAF activation, and MEK-ERK signaling.  This does not occur when oncogenic BRAF is inhibited, demonstrating that BRAF inhibition per se does not drive pathway activation; it only occurs when BRAF is inhibited in the presence of oncogenic RAS.  Kinase-dead BRAF mimics the effects of the BRAF-selective drugs and kinase-dead Braf and oncogenic Ras cooperate to induce melanoma in mice.  Our data reveal another paradigm of BRAF-mediated signaling that promotes tumor progression.  They highlight the importance of understanding pathway signaling in clin. practice and of genotyping tumors prior to administering BRAF-selective drugs, to identify patients who are likely to respond and also to identify patients who may experience adverse effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMSb6ckHpR07Vg90H21EOLACvtfcHk0lh6ej5fuTmx_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvVWitL0%253D&md5=80af840cb978f5956b3d998b185115f4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.12.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.12.040%26sid%3Dliteratum%253Aachs%26aulast%3DHeidorn%26aufirst%3DS.%2BJ.%26aulast%3DMilagre%26aufirst%3DC.%26aulast%3DWhittaker%26aufirst%3DS.%26aulast%3DNourry%26aufirst%3DA.%26aulast%3DNiculescu-Duvas%26aufirst%3DI.%26aulast%3DDhomen%26aufirst%3DN.%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DReis-Filho%26aufirst%3DJ.%2BS.%26aulast%3DSpringer%26aufirst%3DC.%2BJ.%26aulast%3DPritchard%26aufirst%3DC.%26aulast%3DMarais%26aufirst%3DR.%26atitle%3DKinase-dead%2520BRAF%2520and%2520oncogenic%2520RAS%2520cooperate%2520to%2520drive%2520tumor%2520progression%2520through%2520CRAF%26jtitle%3DCell%26date%3D2010%26volume%3D140%26issue%3D2%26spage%3D209%26epage%3D221%26doi%3D10.1016%2Fj.cell.2009.12.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Lito, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saborowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gadal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saborowski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastenhuber, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fellmann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohara, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikami, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lukacs, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishii, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span> </span><span class="NLM_article-title">Disruption of CRAF-mediated MEK activation is required for effective MEK inhibition in KRAS mutant tumours</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">697</span><span class="NLM_x">–</span> <span class="NLM_lpage">710</span><span class="refDoi"> DOI: 10.1016/j.ccr.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1016%2Fj.ccr.2014.03.011" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=697-710&issue=5&author=P.+Litoauthor=A.+Saborowskiauthor=J.+Yueauthor=M.+Solomonauthor=E.+Josephauthor=S.+Gadalauthor=M.+Saborowskiauthor=E.+Kastenhuberauthor=C.+Fellmannauthor=K.+Oharaauthor=K.+Morikamiauthor=T.+Miuraauthor=C.+Lukacsauthor=N.+Ishiiauthor=S.+Loweauthor=N.+Rosen&title=Disruption+of+CRAF-mediated+MEK+activation+is+required+for+effective+MEK+inhibition+in+KRAS+mutant+tumours&doi=10.1016%2Fj.ccr.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DLito%26aufirst%3DP.%26aulast%3DSaborowski%26aufirst%3DA.%26aulast%3DYue%26aufirst%3DJ.%26aulast%3DSolomon%26aufirst%3DM.%26aulast%3DJoseph%26aufirst%3DE.%26aulast%3DGadal%26aufirst%3DS.%26aulast%3DSaborowski%26aufirst%3DM.%26aulast%3DKastenhuber%26aufirst%3DE.%26aulast%3DFellmann%26aufirst%3DC.%26aulast%3DOhara%26aufirst%3DK.%26aulast%3DMorikami%26aufirst%3DK.%26aulast%3DMiura%26aufirst%3DT.%26aulast%3DLukacs%26aufirst%3DC.%26aulast%3DIshii%26aufirst%3DN.%26aulast%3DLowe%26aufirst%3DS.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DDisruption%2520of%2520CRAF-mediated%2520MEK%2520activation%2520is%2520required%2520for%2520effective%2520MEK%2520inhibition%2520in%2520KRAS%2520mutant%2520tumours%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26issue%3D5%26spage%3D697%26epage%3D710%26doi%3D10.1016%2Fj.ccr.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Lamba, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lazzari, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cancelliere, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grernrum, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieftink, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernards, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiNicolantonio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardelli, A.</span><span> </span><span class="NLM_article-title">RAF suppression synergizes with MEK inhibition in KRAS mutant cancer cells</span> <span class="citation_source-journal">Cell Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1475</span><span class="NLM_x">–</span> <span class="NLM_lpage">1483</span><span class="refDoi"> DOI: 10.1016/j.celrep.2014.07.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1016%2Fj.celrep.2014.07.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=25199829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWltrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=1475-1483&issue=5&author=S.+Lambaauthor=M.+Russoauthor=C.+Sunauthor=L.+Lazzariauthor=C.+Cancelliereauthor=W.+Grernrumauthor=C.+Lieftinkauthor=R.+Bernardsauthor=F.+DiNicolantonioauthor=A.+Bardelli&title=RAF+suppression+synergizes+with+MEK+inhibition+in+KRAS+mutant+cancer+cells&doi=10.1016%2Fj.celrep.2014.07.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">RAF Suppression Synergizes with MEK Inhibition in KRAS Mutant Cancer Cells</span></div><div class="casAuthors">Lamba, Simona; Russo, Mariangela; Sun, Chong; Lazzari, Luca; Cancelliere, Carlotta; Grernrum, Wipawadee; Lieftink, Cor; Bernards, Rene; Di Nicolantonio, Federica; Bardelli, Alberto</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1475-1483</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">KRAS is the most frequently mutated oncogene in human cancer, yet no therapies are available to treat KRAS mutant cancers.  We used two independent reverse genetic approaches to identify components of the RAS-signaling pathways required for growth of KRAS mutant tumors.  Small interfering RNA (siRNA) screening of 37 KRAS mutant colorectal cancer cell lines showed that RAF1 suppression was synthetic lethal with MEK inhibition.  An unbiased kinome short hairpin RNA (shRNA)-based screen confirmed this synthetic lethal interaction in colorectal as well as in lung cancer cells bearing KRAS mutations.  Compds. targeting RAF kinases can reverse resistance to the MEK inhibitor selumetinib.  MEK inhibition induces RAS activation and BRAF-RAF1 dimerization and sustains MEK-ERK signaling, which is responsible for intrinsic resistance to selumetinib.  Prolonged dual blockade of RAF and MEK leads to persistent ERK suppression and efficiently induces apoptosis.  Our data underlie the relevance of developing combinatorial regimens of drugs targeting the RAF-MEK pathway in KRAS mutant tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokGXz97UHPObVg90H21EOLACvtfcHk0lh6ej5fuTmx_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWltrfO&md5=1ef6459ec755190ea299c71d07035e28</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.07.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.07.033%26sid%3Dliteratum%253Aachs%26aulast%3DLamba%26aufirst%3DS.%26aulast%3DRusso%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DLazzari%26aufirst%3DL.%26aulast%3DCancelliere%26aufirst%3DC.%26aulast%3DGrernrum%26aufirst%3DW.%26aulast%3DLieftink%26aufirst%3DC.%26aulast%3DBernards%26aufirst%3DR.%26aulast%3DDiNicolantonio%26aufirst%3DF.%26aulast%3DBardelli%26aufirst%3DA.%26atitle%3DRAF%2520suppression%2520synergizes%2520with%2520MEK%2520inhibition%2520in%2520KRAS%2520mutant%2520cancer%2520cells%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26issue%3D5%26spage%3D1475%26epage%3D1483%26doi%3D10.1016%2Fj.celrep.2014.07.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Peng, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henry, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogeti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rutkoski, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wise, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Horn, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchanan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mochalkin, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kays, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donoho, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walgren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conti, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plowman, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starling, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, D. L.</span><span> </span><span class="NLM_article-title">Inhibition of RAF isoforms and active dimers by LY3009120 leads to anti-tumor activities in RAS or BRAF mutant cancers</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">398</span><span class="refDoi"> DOI: 10.1016/j.ccell.2015.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1016%2Fj.ccell.2015.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=26343583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVKgsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=384-398&issue=3&author=S.+B.+Pengauthor=J.+R.+Henryauthor=M.+D.+Kaufmanauthor=W.+P.+Luauthor=B.+D.+Smithauthor=S.+Vogetiauthor=T.+J.+Rutkoskiauthor=S.+Wiseauthor=L.+Chunauthor=+Zhangauthor=R.+D.+Van+Hornauthor=T.+Yinauthor=X.+Zhangauthor=V.+Yadavauthor=S.+H.+Chenauthor=X.+Gongauthor=X.+Maauthor=Y.+Websterauthor=S.+Buchananauthor=I.+Mochalkinauthor=L.+Huberauthor=L.+Kaysauthor=G.+P.+Donohoauthor=J.+Walgrenauthor=D.+McCannauthor=P.+Patelauthor=I.+Contiauthor=G.+D.+Plowmanauthor=J.+J.+Starlingauthor=D.+L.+Flynn&title=Inhibition+of+RAF+isoforms+and+active+dimers+by+LY3009120+leads+to+anti-tumor+activities+in+RAS+or+BRAF+mutant+cancers&doi=10.1016%2Fj.ccell.2015.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of RAF Isoforms and Active Dimers by LY3009120 Leads to Anti-tumor Activities in RAS or BRAF Mutant Cancers</span></div><div class="casAuthors">Peng, Sheng-Bin; Henry, James R.; Kaufman, Michael D.; Lu, Wei-Ping; Smith, Bryan D.; Vogeti, Subha; Rutloski, Thomas J.; Wise, Scott; Chun, Lawrence; Zhang, Youyan; Van Horn, Robert D.; Yin, Tinggui; Zhang, Xiaoyi; Yadav, Vipin; Chen, Shih-Hsun; Gong, Xueqian; Ma, Xiwen; Webster, Yue; Buchanan, Sean; Mochalkin, Igor; Huber, Lysiane; Kays, Lisa; Donoho, Gregory P.; Walgren, Jennie; McCann, Denis; Patel, Phenil; Conti, Ilaria; Plowman, Gregory D.; Starling, James J.; Flynn, Daniel L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">384-398</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">LY3009120 is a pan-RAF and RAF dimer inhibitor that inhibits all RAF isoforms and occupies both protomers in RAF dimers.  Biochem. and cellular analyses revealed that LY3009120 inhibits ARAF, BRAF, and CRAF isoforms with similar affinity, while vemurafenib or dabrafenib have little or modest CRAF activity compared to their BRAF activities.  LY3009120 induces BRAF-CRAF dimerization but inhibits the phosphorylation of downstream MEK and ERK, suggesting that it effectively inhibits the kinase activity of BRAF-CRAF heterodimers.  Further analyses demonstrated that LY3009120 also inhibits various forms of RAF dimers including BRAF or CRAF homodimers.  Due to these unique properties, LY3009120 demonstrates minimal paradoxical activation, inhibits MEK1/2 phosphorylation, and exhibits anti-tumor activities across multiple models carrying KRAS, NRAS, or BRAF mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJMiHK-Gub5bVg90H21EOLACvtfcHk0lh6ej5fuTmx_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVKgsr7O&md5=c167caa6e5a9b47a6ec1ee2ab6d9e0d1</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DS.%2BB.%26aulast%3DHenry%26aufirst%3DJ.%2BR.%26aulast%3DKaufman%26aufirst%3DM.%2BD.%26aulast%3DLu%26aufirst%3DW.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BD.%26aulast%3DVogeti%26aufirst%3DS.%26aulast%3DRutkoski%26aufirst%3DT.%2BJ.%26aulast%3DWise%26aufirst%3DS.%26aulast%3DChun%26aufirst%3DL.%26aulast%3DZhang%26aulast%3DVan%2BHorn%26aufirst%3DR.%2BD.%26aulast%3DYin%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DYadav%26aufirst%3DV.%26aulast%3DChen%26aufirst%3DS.%2BH.%26aulast%3DGong%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DWebster%26aufirst%3DY.%26aulast%3DBuchanan%26aufirst%3DS.%26aulast%3DMochalkin%26aufirst%3DI.%26aulast%3DHuber%26aufirst%3DL.%26aulast%3DKays%26aufirst%3DL.%26aulast%3DDonoho%26aufirst%3DG.%2BP.%26aulast%3DWalgren%26aufirst%3DJ.%26aulast%3DMcCann%26aufirst%3DD.%26aulast%3DPatel%26aufirst%3DP.%26aulast%3DConti%26aufirst%3DI.%26aulast%3DPlowman%26aufirst%3DG.%2BD.%26aulast%3DStarling%26aufirst%3DJ.%2BJ.%26aulast%3DFlynn%26aufirst%3DD.%2BL.%26atitle%3DInhibition%2520of%2520RAF%2520isoforms%2520and%2520active%2520dimers%2520by%2520LY3009120%2520leads%2520to%2520anti-tumor%2520activities%2520in%2520RAS%2520or%2520BRAF%2520mutant%2520cancers%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26issue%3D3%26spage%3D384%26epage%3D398%26doi%3D10.1016%2Fj.ccell.2015.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Rational design of inhibitors that bind to inactive kinase conformations</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span><span class="refDoi"> DOI: 10.1038/nchembio799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1038%2Fnchembio799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=16783341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=358-364&author=Y.+Liuauthor=N.+S.+Gray&title=Rational+design+of+inhibitors+that+bind+to+inactive+kinase+conformations&doi=10.1038%2Fnchembio799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design of inhibitors that bind to inactive kinase conformations</span></div><div class="casAuthors">Liu, Yi; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The majority of kinase inhibitors, known as type I inhibitors that have been developed so far target the ATP binding site of the kinase in its active conformation, in which the activation loop is phosphorylated.  Recently, crystal structures of inhibitors such as imatinib, BIRB796, and sorafenib (BAY43-9006), known as type II inhibitors, have revealed a new binding mode that exploits an addnl. binding site immediately adjacent to the region occupied by ATP.  This pocket is made accessible by an activation-loop rearrangement that is characteristic of kinases in an inactive conformation.  The examn. of binding features of known type II inhibitors is discussed and a pharmacophore model currently used in the design of a new generation of structurally diverse type II inhibitors is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxahsMPlhgDbVg90H21EOLACvtfcHk0lideifodsDyFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D&md5=4a9347a44a2cf4a412c9741fdcc5696b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio799%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DRational%2520design%2520of%2520inhibitors%2520that%2520bind%2520to%2520inactive%2520kinase%2520conformations%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D358%26epage%3D364%26doi%3D10.1038%2Fnchembio799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Angiolini, M.</span><span> </span><span class="NLM_article-title">Targeting the DFG-in kinase conformation: a new trend emerging from patent analysis</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">309</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span><span class="refDoi"> DOI: 10.4155/fmc.10.294</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.4155%2Ffmc.10.294" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=21446845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVCltb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=309-337&author=M.+Angiolini&title=Targeting+the+DFG-in+kinase+conformation%3A+a+new+trend+emerging+from+patent+analysis&doi=10.4155%2Ffmc.10.294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the DFG-in kinase conformation: a new trend emerging from a patent analysis</span></div><div class="casAuthors">Angiolini, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">309-337</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Aberrant kinase signaling leads to a multitude of disease states.  The clin. and com. success of agents typified by imatinib or dasatinib in the treatment of hematol. malignancies has further validated kinase inhibition as a useful clin. strategy.  This increased interest in kinases as therapeutic targets is evidenced by the rapidly increasing no. of patent applications and peer-reviewed articles.  This article discusses recent Patent that describe small mols. targeting the DFG-in active kinase conformation, by the so-called type I1/21/2'' inhibitor, against a small set of clin. relevant targets such as B-Raf, p38αα, Jak2 and EphB4.  Preclin. and clin. data are also highlighted for the most promising new mol. entities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGWPNwqVr3XrVg90H21EOLACvtfcHk0lideifodsDyFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVCltb8%253D&md5=7c6cbe986141be40b6ae2109170fc41b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4155%2Ffmc.10.294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.10.294%26sid%3Dliteratum%253Aachs%26aulast%3DAngiolini%26aufirst%3DM.%26atitle%3DTargeting%2520the%2520DFG-in%2520kinase%2520conformation%253A%2520a%2520new%2520trend%2520emerging%2520from%2520patent%2520analysis%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D309%26epage%3D337%26doi%3D10.4155%2Ffmc.10.294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Ishikawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, Y.</span><span> </span><span class="NLM_article-title">Improvement in aqueous solubility in small molecule drug discovery programs by disruption of molecular planarity and symmetry</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1539</span><span class="NLM_x">–</span> <span class="NLM_lpage">1554</span><span class="refDoi"> DOI: 10.1021/jm101356p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101356p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisVWltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1539-1554&author=M.+Ishikawaauthor=Y.+Hashimoto&title=Improvement+in+aqueous+solubility+in+small+molecule+drug+discovery+programs+by+disruption+of+molecular+planarity+and+symmetry&doi=10.1021%2Fjm101356p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Improvement in Aqueous Solubility in Small Molecule Drug Discovery Programs by Disruption of Molecular Planarity and Symmetry</span></div><div class="casAuthors">Ishikawa, Minoru; Hashimoto, Yuichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1539-1554</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKbtbfvqKZjrVg90H21EOLACvtfcHk0lideifodsDyFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisVWltLw%253D&md5=a88804b3850625ef9c3f7424366acb91</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm101356p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101356p%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DM.%26aulast%3DHashimoto%26aufirst%3DY.%26atitle%3DImprovement%2520in%2520aqueous%2520solubility%2520in%2520small%2520molecule%2520drug%2520discovery%2520programs%2520by%2520disruption%2520of%2520molecular%2520planarity%2520and%2520symmetry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1539%26epage%3D1554%26doi%3D10.1021%2Fjm101356p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Lovering, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bikker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humblet, C.</span><span> </span><span class="NLM_article-title">Escape from flatland: increasing saturation as an approach to improving clinical success</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6752</span><span class="NLM_x">–</span> <span class="NLM_lpage">6756</span><span class="refDoi"> DOI: 10.1021/jm901241e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901241e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6752-6756&author=F.+Loveringauthor=J.+Bikkerauthor=C.+Humblet&title=Escape+from+flatland%3A+increasing+saturation+as+an+approach+to+improving+clinical+success&doi=10.1021%2Fjm901241e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Escape from Flatland: Increasing Saturation as an Approach to Improving Clinical Success</span></div><div class="casAuthors">Lovering, Frank; Bikker, Jack; Humblet, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6752-6756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The medicinal chem. community has become increasingly aware of the value of tracking calcd. phys. properties such as mol. wt., topol. polar surface area, rotatable bonds, and hydrogen bond donors and acceptors.  The authors hypothesized that the shift to high-throughput synthetic practices over the past decade may be another factor that may predispose mols. to fail by steering discovery efforts toward achiral, arom. compds.  The authors have proposed two simple and interpretable measures of the complexity of mols. prepd. as potential drug candidates.  The first is carbon bond satn. as defined by fraction Sp3 (Fsp3) where Fsp3 = (no. of Sp3 hybridized carbons/total carbon count).  The second is simply whether a chiral carbon exists in the mol.  The authors demonstrate that both complexity (as measured by Fsp3) and the presence of chiral centers correlate with success as compds. transition from discovery, through clin. testing, to drugs.  To explain these observations, the authors further demonstrate that satn. correlates with soly., an exptl. phys. property important to success in the drug discovery setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC1f3Ed7nUBbVg90H21EOLACvtfcHk0lhJvvLhJGKuQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN&md5=4ca92c30c17c53d77ad376719bad951e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm901241e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901241e%26sid%3Dliteratum%253Aachs%26aulast%3DLovering%26aufirst%3DF.%26aulast%3DBikker%26aufirst%3DJ.%26aulast%3DHumblet%26aufirst%3DC.%26atitle%3DEscape%2520from%2520flatland%253A%2520increasing%2520saturation%2520as%2520an%2520approach%2520to%2520improving%2520clinical%2520success%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6752%26epage%3D6756%26doi%3D10.1021%2Fjm901241e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelias, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.-M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lhJvvLhJGKuQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.-M.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Karaman, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faraoni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lhJvvLhJGKuQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Fabbro, D.</span><span> </span><span class="NLM_article-title">25 years of small molecular weight kinase inhibitors: potentials and limitations</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">766</span><span class="NLM_x">–</span> <span class="NLM_lpage">775</span><span class="refDoi"> DOI: 10.1124/mol.114.095489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1124%2Fmol.114.095489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=25549667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVKltLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2015&pages=766-775&issue=5&author=D.+Fabbro&title=25+years+of+small+molecular+weight+kinase+inhibitors%3A+potentials+and+limitations&doi=10.1124%2Fmol.114.095489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">25 years of small molecular weight kinase inhibitors: potentials and limitations</span></div><div class="casAuthors">Fabbro, Doriano</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">766-775</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Deregulation of protein and lipid kinase activities leads to a variety of pathologies, ranging from cancer inflammatory diseases, diabetes, infectious diseases, and cardiovascular disorders.  Protein kinases and lipid kinases represent, therefore, an important target for the pharmaceutical industry.  In fact, approx. one-third of all protein targets under investigation in the pharmaceutical industry are protein or lipid kinases.  To date, 30 kinase inhibitors have been approved, which, with few exceptions, are mainly for oncol. indications and directed against only a handful of protein and lipid kinases, leaving 70% of the kinome untapped.  Despite these successes in kinase drug discovery, the development of kinase inhibitors with outstanding selectivity, identification and validation of driver kinase(s) in diseases, and the emerging problem of resistance to the inhibition of key target kinases remain major challenges.  This mini review provides an insight into protein and lipid kinase drug discovery with respect to achievements, binding modes of inhibitors, and novel avenues for the generation of second-generation kinase inhibitors to treat cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZhF-9NQJx7bVg90H21EOLACvtfcHk0liW8VZpMoZD5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVKltLk%253D&md5=6422b036e4e447379ac6c608662954c2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1124%2Fmol.114.095489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.114.095489%26sid%3Dliteratum%253Aachs%26aulast%3DFabbro%26aufirst%3DD.%26atitle%3D25%2520years%2520of%2520small%2520molecular%2520weight%2520kinase%2520inhibitors%253A%2520potentials%2520and%2520limitations%26jtitle%3DMol.%2520Pharmacol.%26date%3D2015%26volume%3D87%26issue%3D5%26spage%3D766%26epage%3D775%26doi%3D10.1124%2Fmol.114.095489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Vijayan, R. S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Modi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duong-Ly, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunbrack, R. L.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, R. M.</span><span> </span><span class="NLM_article-title">Conformational analysis of the DFG-Out kinase motif and biochemical profiling of structurally validated Type II inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">466</span><span class="NLM_x">–</span> <span class="NLM_lpage">479</span><span class="refDoi"> DOI: 10.1021/jm501603h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501603h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCku7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=466-479&author=R.+S.+K.+Vijayanauthor=P.+Heauthor=V.+Modiauthor=K.+C.+Duong-Lyauthor=H.+Maauthor=J.+R.+Petersonauthor=R.+L.+Dunbrackauthor=R.+M.+Levy&title=Conformational+analysis+of+the+DFG-Out+kinase+motif+and+biochemical+profiling+of+structurally+validated+Type+II+inhibitors&doi=10.1021%2Fjm501603h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of the DFG-Out Kinase Motif and Biochemical Profiling of Structurally Validated Type II Inhibitors</span></div><div class="casAuthors">Vijayan, R. S. K.; He, Peng; Modi, Vivek; Duong-Ly, Krisna C.; Ma, Haiching; Peterson, Jeffrey R.; Dunbrack, Roland L.; Levy, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">466-479</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structural coverage of the human kinome has been steadily increasing over time.  The structures provide valuable insights into the mol. basis of kinase function and also provide a foundation for understanding the mechanisms of kinase inhibitors.  There are a large no. of kinase structures in the PDB for which the Asp and Phe of the DFG motif on the activation loop swap positions, resulting in the formation of a new allosteric pocket.  The authors refer to these structures as "classical DFG-out" conformations in order to distinguish them from conformations that have also been referred to as DFG-out in the literature but that do not have a fully formed allosteric pocket.  The authors have completed a structural anal. of almost 200 small mol. inhibitors bound to classical DFG-out conformations; the authors find that they are recognized by both type I and type II inhibitors.  In contrast, the authors find that nonclassical DFG-out conformations strongly select against type II inhibitors because these structures have not formed a large enough allosteric pocket to accommodate this type of binding mode.  In the course of this study the authors discovered that the no. of structurally validated type II inhibitors that can be found in the PDB and that are also represented in publicly available biochem. profiling studies of kinase inhibitors is very small.  The authors have obtained new profiling results for several addnl. structurally validated type II inhibitors identified through the conformational anal.  Although the available profiling data for type II inhibitors is still much smaller than for type I inhibitors, a comparison of the two data sets supports the conclusion that type II inhibitors are more selective than type I.  The authors comment on the possible contribution of the DFG-in to DFG-out conformational reorganization to the selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb9E5JKOmDZ7Vg90H21EOLACvtfcHk0liW8VZpMoZD5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCku7rP&md5=6d729997f242d582a4adff9463fd585c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm501603h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501603h%26sid%3Dliteratum%253Aachs%26aulast%3DVijayan%26aufirst%3DR.%2BS.%2BK.%26aulast%3DHe%26aufirst%3DP.%26aulast%3DModi%26aufirst%3DV.%26aulast%3DDuong-Ly%26aufirst%3DK.%2BC.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26aulast%3DDunbrack%26aufirst%3DR.%2BL.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DConformational%2520analysis%2520of%2520the%2520DFG-Out%2520kinase%2520motif%2520and%2520biochemical%2520profiling%2520of%2520structurally%2520validated%2520Type%2520II%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D466%26epage%3D479%26doi%3D10.1021%2Fjm501603h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="note"><p class="first last">Experimental detail for the small molecule crystal structure of <b>7</b>, and X-ray cocrystal structure data table of <b>3</b> and <b>14</b> and kinase profile of <b>7</b> are given in the <a href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_001.pdf" class="ext-link">Supporting Information</a></p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="note"><p class="first last">Manuscript discussing the entire biology of <b>14</b> has been submitted for publication.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Schuffenhauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roggo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetzel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">The scaffold tree – visualization of the scaffold universe by hierarchical scaffold classification</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span><span class="refDoi"> DOI: 10.1021/ci600338x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600338x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=47-58&author=A.+Schuffenhauerauthor=P.+Ertlauthor=S.+Roggoauthor=S.+Wetzelauthor=M.+A.+Kochauthor=H.+Waldmann&title=The+scaffold+tree+%E2%80%93+visualization+of+the+scaffold+universe+by+hierarchical+scaffold+classification&doi=10.1021%2Fci600338x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The Scaffold Tree - Visualization of the Scaffold Universe by Hierarchical Scaffold Classification</span></div><div class="casAuthors">Schuffenhauer, Ansgar; Ertl, Peter; Roggo, Silvio; Wetzel, Stefan; Koch, Marcus A.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A hierarchical classification of chem. scaffolds (mol. framework, which is obtained by pruning all terminal side chains) has been introduced.  The mol. frameworks form the leaf nodes in the hierarchy trees.  By an iterative removal of rings, scaffolds forming the higher levels in the hierarchy tree are obtained.  Prioritization rules ensure that less characteristic, peripheral rings are removed first.  All scaffolds in the hierarchy tree are well-defined chem. entities making the classification chem. intuitive.  The classification is deterministic, data-set-independent, and scales linearly with the no. of compds. included in the data set.  The application of the classification is demonstrated on two data sets extd. from the PubChem database, namely, pyruvate kinase binders and a collection of pesticides.  The examples shown demonstrate that the classification procedure handles robustly synthetic structures and natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8XfR2xvGasbVg90H21EOLACvtfcHk0liW8VZpMoZD5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO&md5=89cec5a01a37802ab0826ecababfaf0a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fci600338x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600338x%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRoggo%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DKoch%26aufirst%3DM.%2BA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DThe%2520scaffold%2520tree%2520%25E2%2580%2593%2520visualization%2520of%2520the%2520scaffold%2520universe%2520by%2520hierarchical%2520scaffold%2520classification%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D47%26epage%3D58%26doi%3D10.1021%2Fci600338x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Vonrhein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flensburg, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smart, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paciorek, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Womack, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bricogne, G.</span><span> </span><span class="NLM_article-title">Data processing and analysis with the <i>autoPROC</i> toolbox</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">302</span><span class="refDoi"> DOI: 10.1107/S0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+processing+and+analysis+with+the+autoPROC+toolbox&doi=10.1107%2FS0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0lhsCjp5CLwMoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520processing%2520and%2520analysis%2520with%2520the%2520autoPROC%2520toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2FS0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title"><i>Phaser</i> crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhsCjp5CLwMoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Smart, O. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Womack, T. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flensburg, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paciorek, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vonrhein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bricogne, G.</span><span> </span><span class="NLM_article-title">Exploiting structure similarity in refinement: automated NCS and target-structure restraints in <i>BUSTER</i></span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">368</span><span class="NLM_x">–</span> <span class="NLM_lpage">380</span><span class="refDoi"> DOI: 10.1107/S0907444911056058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1107%2FS0907444911056058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=22505257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Gns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=368-380&author=O.+S.+Smartauthor=T.+O.+Womackauthor=C.+Flensburgauthor=P.+Kellerauthor=W.+Paciorekauthor=A.+Sharffauthor=C.+Vonrheinauthor=G.+Bricogne&title=Exploiting+structure+similarity+in+refinement%3A+automated+NCS+and+target-structure+restraints+in+BUSTER&doi=10.1107%2FS0907444911056058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting structure similarity in refinement: automated NCS and target-structure restraints in BUSTER</span></div><div class="casAuthors">Smart, Oliver S.; Womack, Thomas O.; Flensburg, Claus; Keller, Peter; Paciorek, Wlodek; Sharff, Andrew; Vonrhein, Clemens; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">368-380</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Maximum-likelihood X-ray macromol. structure refinement in BUSTER has been extended with restraints facilitating the exploitation of structural similarity.  The similarity can be between two or more chains within the structure being refined, thus favoring NCS, or to a distinct 'target' structure that remains fixed during refinement.  The local structural similarity restraints (LSSR) approach considers all distances less than 5.5 Å between pairs of atoms in the chain to be restrained.  For each, the difference from the distance between the corresponding atoms in the related chain is found.  LSSR applies a restraint penalty on each difference.  A functional form that reaches a plateau for large differences is used to avoid the restraints distorting parts of the structure that are not similar.  Because LSSR are local, there is no need to sep. out domains.  Some restraint pruning is still necessary, but this has been automated.  LSSR have been available to academic users of BUSTER since 2009 with the easy-to-use and options.  The use of LSSR is illustrated in the re-refinement of PDB entries , where enables the correct ligand-binding structure to be found, and , where contributes to the location of an addnl. copy of the cyclic peptide ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu-qo0DGjn3bVg90H21EOLACvtfcHk0lhsCjp5CLwMoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Gns78%253D&md5=bea89f86f489e8dbd00a9866a3d7dab9</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444911056058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911056058%26sid%3Dliteratum%253Aachs%26aulast%3DSmart%26aufirst%3DO.%2BS.%26aulast%3DWomack%26aufirst%3DT.%2BO.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DExploiting%2520structure%2520similarity%2520in%2520refinement%253A%2520automated%2520NCS%2520and%2520target-structure%2520restraints%2520in%2520BUSTER%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D368%26epage%3D380%26doi%3D10.1107%2FS0907444911056058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhLzIT3WjKAYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref14':['cit14'],'ref21':['cit21'],'ref22':[],'ref23':[],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 21 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Malcolm P. Huestis, Darlene Dela Cruz, Antonio G. DiPasquale, Matthew R. Durk, Charles Eigenbrot, Paul Gibbons, Alberto Gobbi, Thomas L. Hunsaker, Hank La, Dennis H. Leung, Wendy Liu, Shiva Malek, Mark Merchant, John G. Moffat, Christine S. Muli, Christine J. Orr, Brendan T. Parr, Frances Shanahan, Christopher J. Sneeringer, Weiru Wang, Ivana Yen, Jianping Yin, Michael Siu, <span class="NLM_string-name hlFld-ContribAuthor">Joachim Rudolph</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting KRAS Mutant Cancers via Combination Treatment: Discovery of a 5-Fluoro-4-(3H)-quinazolinone Aryl Urea pan-RAF Kinase Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (7)
                                     , 3940-3955. <a href="https://doi.org/10.1021/acs.jmedchem.0c02085" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02085</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02085%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BKRAS%252BMutant%252BCancers%252Bvia%252BCombination%252BTreatment%25253A%252BDiscovery%252Bof%252Ba%252B5-Fluoro-4-%2525283H%252529-quinazolinone%252BAryl%252BUrea%252Bpan-RAF%252BKinase%252BInhibitor%26aulast%3DHuestis%26aufirst%3DMalcolm%2BP.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D01122020%26date%3D29032021%26volume%3D64%26issue%3D7%26spage%3D3940%26epage%3D3955" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Savithri Ramurthy, Benjamin R. Taft, Robert J. Aversa, Paul A. Barsanti, Matthew T. Burger, Yan Lou, Gisele A. Nishiguchi, Alice Rico, Lina Setti, Aaron Smith, Sharadha Subramanian, Victoriano Tamez, Huw Tanner, Lifeng Wan, Cheng Hu, Brent A. Appleton, Mulugeta Mamo, Laura Tandeske, John E. Tellew, Shenlin Huang, Qin Yue, Apurva Chaudhary, Hung Tian, Raman Iyer, A. Quamrul Hassan, Lesley A. Mathews Griner, Laura R. La Bonte, Vesselina G. Cooke, Anne Van Abbema, Hanne Merritt, Kalyani Gampa, Fei Feng, Jing Yuan, Yuji Mishina, Yingyun Wang, Jacob R. Haling, Sepideh Vaziri, Mohammad Hekmat-Nejad, Valery Polyakov, Richard Zang, Vijay Sethuraman, Payman Amiri, Mallika Singh, William R. Sellers, Emma Lees, Wenlin Shao, Michael P. Dillon, <span class="NLM_string-name hlFld-ContribAuthor">Darrin D. Stuart</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Discovery of N-(3-(2-(2-Hydroxyethoxy)-6-morpholinopyridin-4-yl)-4-methylphenyl)-2-(trifluoromethyl)isonicotinamide, a Selective, Efficacious, and Well-Tolerated RAF Inhibitor Targeting RAS Mutant Cancers: The Path to the Clinic. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (5)
                                     , 2013-2027. <a href="https://doi.org/10.1021/acs.jmedchem.9b00161" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00161</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00161%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252BDiscovery%252Bof%252BN-%2525283-%2525282-%2525282-Hydroxyethoxy%252529-6-morpholinopyridin-4-yl%252529-4-methylphenyl%252529-2-%252528trifluoromethyl%252529isonicotinamide%25252C%252Ba%252BSelective%25252C%252BEfficacious%25252C%252Band%252BWell-Tolerated%252BRAF%252BInhibitor%252BTargeting%252BRAS%252BMutant%252BCancers%25253A%252BThe%252BPath%252Bto%252Bthe%252BClinic%26aulast%3DRamurthy%26aufirst%3DSavithri%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D26012019%26date%3D16052019%26date%3D06052019%26volume%3D63%26issue%3D5%26spage%3D2013%26epage%3D2027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Quan-Xing Zi, Sheng-Jiao Yan, Chang-Long Yang, Kun Li, <span class="NLM_string-name hlFld-ContribAuthor">Jun Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Three-Component Cascade Reaction of 1,1-Enediamines, N,N-Dimethylformamide Dimethyl Acetal, and 1,3-Dicarbonyl Compounds: Selective Synthesis of Diverse 2-Aminopyridine Derivatives. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2019,</strong> <em>4 </em>
                                    (2)
                                     , 2863-2873. <a href="https://doi.org/10.1021/acsomega.8b03284" title="DOI URL">https://doi.org/10.1021/acsomega.8b03284</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.8b03284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.8b03284%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DThree-Component%252BCascade%252BReaction%252Bof%252B1%25252C1-Enediamines%25252C%252BN%25252CN-Dimethylformamide%252BDimethyl%252BAcetal%25252C%252Band%252B1%25252C3-Dicarbonyl%252BCompounds%25253A%252BSelective%252BSynthesis%252Bof%252BDiverse%252B2-Aminopyridine%252BDerivatives%26aulast%3DZi%26aufirst%3DQuan-Xing%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D25112018%26date%3D25012019%26volume%3D4%26issue%3D2%26spage%3D2863%26epage%3D2873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Qin Luo, Rong Huang, Qiang Xiao, Yuan Yao, Jun Lin, <span class="NLM_string-name hlFld-ContribAuthor">Sheng-Jiao Yan</span>. </span><span class="cited-content_cbyCitation_article-title">Highly Selective Synthesis of 2-Amino-4,6-diarylpyridine Derivatives by the Cascade Reaction of 1,1-Enediamines with α,β-Unsaturated Ketones. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>84 </em>
                                    (4)
                                     , 1999-2011. <a href="https://doi.org/10.1021/acs.joc.8b03008" title="DOI URL">https://doi.org/10.1021/acs.joc.8b03008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.8b03008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.8b03008%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DHighly%252BSelective%252BSynthesis%252Bof%252B2-Amino-4%25252C6-diarylpyridine%252BDerivatives%252Bby%252Bthe%252BCascade%252BReaction%252Bof%252B1%25252C1-Enediamines%252Bwith%252B%2525CE%2525B1%25252C%2525CE%2525B2-Unsaturated%252BKetones%26aulast%3DLuo%26aufirst%3DQin%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D25112018%26date%3D31012019%26date%3D17012019%26volume%3D84%26issue%3D4%26spage%3D1999%26epage%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Bogos Agianian, <span class="NLM_string-name hlFld-ContribAuthor">Evripidis Gavathiotis</span>. </span><span class="cited-content_cbyCitation_article-title">Current Insights of BRAF Inhibitors in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (14)
                                     , 5775-5793. <a href="https://doi.org/10.1021/acs.jmedchem.7b01306" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01306</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01306%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCurrent%252BInsights%252Bof%252BBRAF%252BInhibitors%252Bin%252BCancer%26aulast%3DAgianian%26aufirst%3DBogos%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D01092017%26date%3D06032018%26date%3D20022018%26volume%3D61%26issue%3D14%26spage%3D5775%26epage%3D5793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vikram  Jeet Singh</span>, <span class="hlFld-ContribAuthor ">Bharti  Sharma</span>, <span class="hlFld-ContribAuthor ">Pooja A  Chawla</span>. </span><span class="cited-content_cbyCitation_article-title">Recent developments in mitogen activated protein kinase inhibitors as potential anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>17 </em>, 105161. <a href="https://doi.org/10.1016/j.bioorg.2021.105161" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.105161</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.105161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.105161%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DRecent%252Bdevelopments%252Bin%252Bmitogen%252Bactivated%252Bprotein%252Bkinase%252Binhibitors%252Bas%252Bpotential%252Banticancer%252Bagents%26aulast%3DJeet%2BSingh%26aufirst%3DVikram%26date%3D2021%26volume%3D17%26spage%3D105161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Masamitsu  Tsukamoto</span>, <span class="hlFld-ContribAuthor ">Tadashi  Nakamura</span>, <span class="hlFld-ContribAuthor ">Hirohiko  Kimura</span>, <span class="hlFld-ContribAuthor ">Hitoshi  Nakayama</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and application of trifluoromethylpyridines as a key structural motif in active agrochemical and pharmaceutical ingredients. </span><span class="cited-content_cbyCitation_journal-name">Journal of Pesticide Science</span><span> <strong>2021,</strong> <em>46 </em>
                                    (2)
                                     , 125-142. <a href="https://doi.org/10.1584/jpestics.D21-012" title="DOI URL">https://doi.org/10.1584/jpestics.D21-012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1584/jpestics.D21-012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1584%2Fjpestics.D21-012%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Pesticide%2520Science%26atitle%3DSynthesis%252Band%252Bapplication%252Bof%252Btrifluoromethylpyridines%252Bas%252Ba%252Bkey%252Bstructural%252Bmotif%252Bin%252Bactive%252Bagrochemical%252Band%252Bpharmaceutical%252Bingredients%26aulast%3DTsukamoto%26aufirst%3DMasamitsu%26date%3D2021%26volume%3D46%26issue%3D2%26spage%3D125%26epage%3D142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rahim  Ullah</span>, <span class="hlFld-ContribAuthor ">Qing  Yin</span>, <span class="hlFld-ContribAuthor ">Aidan H.  Snell</span>, <span class="hlFld-ContribAuthor ">Lixin  Wan</span>. </span><span class="cited-content_cbyCitation_article-title">RAF-MEK-ERK pathway in cancer evolution and treatment. </span><span class="cited-content_cbyCitation_journal-name">Seminars in Cancer Biology</span><span> <strong>2021,</strong> <em>84 </em><a href="https://doi.org/10.1016/j.semcancer.2021.05.010" title="DOI URL">https://doi.org/10.1016/j.semcancer.2021.05.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.semcancer.2021.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.semcancer.2021.05.010%26sid%3Dliteratum%253Aachs%26jtitle%3DSeminars%2520in%2520Cancer%2520Biology%26atitle%3DRAF-MEK-ERK%252Bpathway%252Bin%252Bcancer%252Bevolution%252Band%252Btreatment%26aulast%3DUllah%26aufirst%3DRahim%26date%3D2021%26volume%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tingting  Lu</span>, <span class="hlFld-ContribAuthor ">Jiangyan  Cao</span>, <span class="hlFld-ContribAuthor ">Fengming  Zou</span>, <span class="hlFld-ContribAuthor ">Xixiang  Li</span>, <span class="hlFld-ContribAuthor ">Aoli  Wang</span>, <span class="hlFld-ContribAuthor ">Wenliang  Wang</span>, <span class="hlFld-ContribAuthor ">Huamin  Liang</span>, <span class="hlFld-ContribAuthor ">Qingwang  Liu</span>, <span class="hlFld-ContribAuthor ">Chen  Hu</span>, <span class="hlFld-ContribAuthor ">Cheng  Chen</span>, <span class="hlFld-ContribAuthor ">Zhenquan  Hu</span>, <span class="hlFld-ContribAuthor ">Wenchao  Wang</span>, <span class="hlFld-ContribAuthor ">Lili  Li</span>, <span class="hlFld-ContribAuthor ">Jian  Ge</span>, <span class="hlFld-ContribAuthor ">Yang  Shen</span>, <span class="hlFld-ContribAuthor ">Tao  Ren</span>, <span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Ruixiang  Xia</span>, <span class="hlFld-ContribAuthor ">Qingsong  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a highly potent kinase inhibitor capable of overcoming multiple imatinib-resistant ABL mutants for chronic myeloid leukemia (CML). </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmacology</span><span> <strong>2021,</strong> <em>897 </em>, 173944. <a href="https://doi.org/10.1016/j.ejphar.2021.173944" title="DOI URL">https://doi.org/10.1016/j.ejphar.2021.173944</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejphar.2021.173944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejphar.2021.173944%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmacology%26atitle%3DDiscovery%252Bof%252Ba%252Bhighly%252Bpotent%252Bkinase%252Binhibitor%252Bcapable%252Bof%252Bovercoming%252Bmultiple%252Bimatinib-resistant%252BABL%252Bmutants%252Bfor%252Bchronic%252Bmyeloid%252Bleukemia%252B%252528CML%252529%26aulast%3DLu%26aufirst%3DTingting%26date%3D2021%26volume%3D897%26spage%3D173944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kelvin F.  Cho</span>, <span class="hlFld-ContribAuthor ">Taylur P.  Ma</span>, <span class="hlFld-ContribAuthor ">Christopher M.  Rose</span>, <span class="hlFld-ContribAuthor ">Donald S.  Kirkpatrick</span>, <span class="hlFld-ContribAuthor ">Kebing  Yu</span>, <span class="hlFld-ContribAuthor ">Robert A.  Blake</span>. </span><span class="cited-content_cbyCitation_article-title">Chaperone mediated detection of small molecule target binding in cells. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2020,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-019-14033-0" title="DOI URL">https://doi.org/10.1038/s41467-019-14033-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-019-14033-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-019-14033-0%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DChaperone%252Bmediated%252Bdetection%252Bof%252Bsmall%252Bmolecule%252Btarget%252Bbinding%252Bin%252Bcells%26aulast%3DCho%26aufirst%3DKelvin%2BF.%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaojuan  Wang</span>, <span class="hlFld-ContribAuthor ">Fan  Wu</span>, <span class="hlFld-ContribAuthor ">Han  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaoyuan  Duan</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>, <span class="hlFld-ContribAuthor ">Amannisa  Tuersuntuoheti</span>, <span class="hlFld-ContribAuthor ">Luying  Su</span>, <span class="hlFld-ContribAuthor ">Shida  Yan</span>, <span class="hlFld-ContribAuthor ">Yuechao  Zhao</span>, <span class="hlFld-ContribAuthor ">Yan  Lu</span>, <span class="hlFld-ContribAuthor ">Kai  Li</span>, <span class="hlFld-ContribAuthor ">Jinjie  Yao</span>, <span class="hlFld-ContribAuthor ">Zhiwen  Luo</span>, <span class="hlFld-ContribAuthor ">Lei  Guo</span>, <span class="hlFld-ContribAuthor ">Jianmei  Liu</span>, <span class="hlFld-ContribAuthor ">Xiao  Chen</span>, <span class="hlFld-ContribAuthor ">Yalan  Lu</span>, <span class="hlFld-ContribAuthor ">Hanjie  Hu</span>, <span class="hlFld-ContribAuthor ">Xingchen  Li</span>, <span class="hlFld-ContribAuthor ">Mandula  Bao</span>, <span class="hlFld-ContribAuthor ">Xinyu  Bi</span>, <span class="hlFld-ContribAuthor ">Boyu  Du</span>, <span class="hlFld-ContribAuthor ">Shiying  Miao</span>, <span class="hlFld-ContribAuthor ">Jianqiang  Cai</span>, <span class="hlFld-ContribAuthor ">Linfang  Wang</span>, <span class="hlFld-ContribAuthor ">Haitao  Zhou</span>, <span class="hlFld-ContribAuthor ">Jianming  Ying</span>, <span class="hlFld-ContribAuthor ">Wei  Song</span>, <span class="hlFld-ContribAuthor ">Hong  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">PDCD6 cooperates with C-Raf to facilitate colorectal cancer progression via Raf/MEK/ERK activation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Experimental & Clinical Cancer Research</span><span> <strong>2020,</strong> <em>39 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13046-020-01632-9" title="DOI URL">https://doi.org/10.1186/s13046-020-01632-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13046-020-01632-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13046-020-01632-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Experimental%2520%2526%2520Clinical%2520Cancer%2520Research%26atitle%3DPDCD6%252Bcooperates%252Bwith%252BC-Raf%252Bto%252Bfacilitate%252Bcolorectal%252Bcancer%252Bprogression%252Bvia%252BRaf%25252FMEK%25252FERK%252Bactivation%26aulast%3DWang%26aufirst%3DXiaojuan%26date%3D2020%26date%3D2020%26volume%3D39%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Regina  Martínez</span>, <span class="hlFld-ContribAuthor ">Bruno  Di Geronimo</span>, <span class="hlFld-ContribAuthor ">Miryam  Pastor</span>, <span class="hlFld-ContribAuthor ">José María  Zapico</span>, <span class="hlFld-ContribAuthor ">Claire  Coderch</span>, <span class="hlFld-ContribAuthor ">Rostyslav  Panchuk</span>, <span class="hlFld-ContribAuthor ">Nadia  Skorokhyd</span>, <span class="hlFld-ContribAuthor ">Maciej  Maslyk</span>, <span class="hlFld-ContribAuthor ">Ana  Ramos</span>, <span class="hlFld-ContribAuthor ">Beatriz  de Pascual-Teresa</span>. </span><span class="cited-content_cbyCitation_article-title">Multitarget Anticancer Agents Based on Histone Deacetylase and Protein Kinase CK2 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (7)
                                     , 1497. <a href="https://doi.org/10.3390/molecules25071497" title="DOI URL">https://doi.org/10.3390/molecules25071497</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25071497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25071497%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMultitarget%252BAnticancer%252BAgents%252BBased%252Bon%252BHistone%252BDeacetylase%252Band%252BProtein%252BKinase%252BCK2%252BInhibitors%26aulast%3DMart%25C3%25ADnez%26aufirst%3DRegina%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D7%26spage%3D1497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ling‐Bin  Kong</span>, <span class="hlFld-ContribAuthor ">Sheng‐Jiao  Yan</span>, <span class="hlFld-ContribAuthor ">Yuan  Yao</span>, <span class="hlFld-ContribAuthor ">Qiang  Xiao</span>, <span class="hlFld-ContribAuthor ">Jun  Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Cascade Reactions Utilizing the Nucleophilic Properties of 1,1‐Enediamines for the Regioselective Synthesis of 4‐Aryl‐2‐aminopyridines. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2019,</strong> <em>4 </em>
                                    (11)
                                     , 3083-3087. <a href="https://doi.org/10.1002/slct.201900026" title="DOI URL">https://doi.org/10.1002/slct.201900026</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.201900026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.201900026%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DCascade%252BReactions%252BUtilizing%252Bthe%252BNucleophilic%252BProperties%252Bof%252B1%25252C1%2525E2%252580%252590Enediamines%252Bfor%252Bthe%252BRegioselective%252BSynthesis%252Bof%252B4%2525E2%252580%252590Aryl%2525E2%252580%2525902%2525E2%252580%252590aminopyridines%26aulast%3DKong%26aufirst%3DLing%25E2%2580%2590Bin%26date%3D2019%26date%3D2019%26volume%3D4%26issue%3D11%26spage%3D3083%26epage%3D3087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xuan-Xuan  Du</span>, <span class="hlFld-ContribAuthor ">Quan-Xing  Zi</span>, <span class="hlFld-ContribAuthor ">Yu-Meng  Wu</span>, <span class="hlFld-ContribAuthor ">Yi  Jin</span>, <span class="hlFld-ContribAuthor ">Jun  Lin</span>, <span class="hlFld-ContribAuthor ">Sheng-Jiao  Yan</span>. </span><span class="cited-content_cbyCitation_article-title">An environmentally benign multi-component reaction: regioselective synthesis of fluorinated 2-aminopyridines using diverse properties of the nitro group. </span><span class="cited-content_cbyCitation_journal-name">Green Chemistry</span><span> <strong>2019,</strong> <em>21 </em>
                                    (6)
                                     , 1505-1516. <a href="https://doi.org/10.1039/C8GC03698E" title="DOI URL">https://doi.org/10.1039/C8GC03698E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8GC03698E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8GC03698E%26sid%3Dliteratum%253Aachs%26jtitle%3DGreen%2520Chemistry%26atitle%3DAn%252Benvironmentally%252Bbenign%252Bmulti-component%252Breaction%25253A%252Bregioselective%252Bsynthesis%252Bof%252Bfluorinated%252B2-aminopyridines%252Busing%252Bdiverse%252Bproperties%252Bof%252Bthe%252Bnitro%252Bgroup%26aulast%3DDu%26aufirst%3DXuan-Xuan%26date%3D2019%26date%3D2019%26volume%3D21%26issue%3D6%26spage%3D1505%26epage%3D1516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chih-Shia  Lee</span>, <span class="hlFld-ContribAuthor ">Liam C.  Lee</span>, <span class="hlFld-ContribAuthor ">Tina L.  Yuan</span>, <span class="hlFld-ContribAuthor ">Sirisha  Chakka</span>, <span class="hlFld-ContribAuthor ">Christof  Fellmann</span>, <span class="hlFld-ContribAuthor ">Scott W.  Lowe</span>, <span class="hlFld-ContribAuthor ">Natasha J.  Caplen</span>, <span class="hlFld-ContribAuthor ">Frank  McCormick</span>, <span class="hlFld-ContribAuthor ">Ji  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">MAP kinase and autophagy pathways cooperate to maintain RAS mutant cancer cell survival. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2019,</strong> <em>116 </em>
                                    (10)
                                     , 4508-4517. <a href="https://doi.org/10.1073/pnas.1817494116" title="DOI URL">https://doi.org/10.1073/pnas.1817494116</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1817494116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1817494116%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DMAP%252Bkinase%252Band%252Bautophagy%252Bpathways%252Bcooperate%252Bto%252Bmaintain%252BRAS%252Bmutant%252Bcancer%252Bcell%252Bsurvival%26aulast%3DLee%26aufirst%3DChih-Shia%26date%3D2019%26date%3D2019%26volume%3D116%26issue%3D10%26spage%3D4508%26epage%3D4517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lu  Wang</span>, <span class="hlFld-ContribAuthor ">Yanmin  Zhang</span>, <span class="hlFld-ContribAuthor ">Qing  Zhang</span>, <span class="hlFld-ContribAuthor ">Gaoyuan  Zhu</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Chunqi  Duan</span>, <span class="hlFld-ContribAuthor ">Tao  Lu</span>, <span class="hlFld-ContribAuthor ">Weifang  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of potent Pan-Raf inhibitors with increased solubility to overcome drug resistance. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>163 </em>, 243-255. <a href="https://doi.org/10.1016/j.ejmech.2018.11.033" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.11.033</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.11.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.11.033%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bpotent%252BPan-Raf%252Binhibitors%252Bwith%252Bincreased%252Bsolubility%252Bto%252Bovercome%252Bdrug%252Bresistance%26aulast%3DWang%26aufirst%3DLu%26date%3D2019%26volume%3D163%26spage%3D243%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hui-Yan  Wang</span>, <span class="hlFld-ContribAuthor ">Ying  Shen</span>, <span class="hlFld-ContribAuthor ">Hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Hei</span>, <span class="hlFld-ContribAuthor ">Hong-Yi  Zhao</span>, <span class="hlFld-ContribAuthor ">Minhang  Xin</span>, <span class="hlFld-ContribAuthor ">She-Min  Lu</span>, <span class="hlFld-ContribAuthor ">San-Qi  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of 2-(aminopyrimidin-5-yl)-4-(morpholin-4-yl)-6- substituted triazine as PI3K and BRAF dual inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2018,</strong> <em>10 </em>
                                    (20)
                                     , 2445-2455. <a href="https://doi.org/10.4155/fmc-2018-0145" title="DOI URL">https://doi.org/10.4155/fmc-2018-0145</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0145%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B2-%252528aminopyrimidin-5-yl%252529-4-%252528morpholin-4-yl%252529-6-%252Bsubstituted%252Btriazine%252Bas%252BPI3K%252Band%252BBRAF%252Bdual%252Binhibitor%26aulast%3DWang%26aufirst%3DHui-Yan%26date%3D2018%26volume%3D10%26issue%3D20%26spage%3D2445%26epage%3D2455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Jin  Wang</span>, <span class="hlFld-ContribAuthor ">Hong-Yi  Zhao</span>, <span class="hlFld-ContribAuthor ">Xue-Yan  Yang</span>, <span class="hlFld-ContribAuthor ">Hao  Lei</span>, <span class="hlFld-ContribAuthor ">Minhang  Xin</span>, <span class="hlFld-ContribAuthor ">Yong-Xiao  Cao</span>, <span class="hlFld-ContribAuthor ">San-Qi  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and biological evaluation of irreversible EGFR tyrosine kinase inhibitors containing pyrido[3,4-d]pyrimidine scaffold. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (12)
                                     , 3619-3633. <a href="https://doi.org/10.1016/j.bmc.2018.05.039" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.05.039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.05.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.05.039%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Birreversible%252BEGFR%252Btyrosine%252Bkinase%252Binhibitors%252Bcontaining%252Bpyrido%25255B3%25252C4-d%25255Dpyrimidine%252Bscaffold%26aulast%3DZhang%26aufirst%3DHao%26date%3D2018%26volume%3D26%26issue%3D12%26spage%3D3619%26epage%3D3633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenlin  Shao</span>, <span class="hlFld-ContribAuthor ">Yuji M.  Mishina</span>, <span class="hlFld-ContribAuthor ">Yun  Feng</span>, <span class="hlFld-ContribAuthor ">Giordano  Caponigro</span>, <span class="hlFld-ContribAuthor ">Vesselina G.  Cooke</span>, <span class="hlFld-ContribAuthor ">Stacy  Rivera</span>, <span class="hlFld-ContribAuthor ">Yingyun  Wang</span>, <span class="hlFld-ContribAuthor ">Fang  Shen</span>, <span class="hlFld-ContribAuthor ">Joshua M.  Korn</span>, <span class="hlFld-ContribAuthor ">Lesley A.  Mathews Griner</span>, <span class="hlFld-ContribAuthor ">Gisele  Nishiguchi</span>, <span class="hlFld-ContribAuthor ">Alice  Rico</span>, <span class="hlFld-ContribAuthor ">John  Tellew</span>, <span class="hlFld-ContribAuthor ">Jacob R.  Haling</span>, <span class="hlFld-ContribAuthor ">Robert  Aversa</span>, <span class="hlFld-ContribAuthor ">Valery  Polyakov</span>, <span class="hlFld-ContribAuthor ">Richard  Zang</span>, <span class="hlFld-ContribAuthor ">Mohammad  Hekmat-Nejad</span>, <span class="hlFld-ContribAuthor ">Payman  Amiri</span>, <span class="hlFld-ContribAuthor ">Mallika  Singh</span>, <span class="hlFld-ContribAuthor ">Nicholas  Keen</span>, <span class="hlFld-ContribAuthor ">Michael P.  Dillon</span>, <span class="hlFld-ContribAuthor ">Emma  Lees</span>, <span class="hlFld-ContribAuthor ">Savithri  Ramurthy</span>, <span class="hlFld-ContribAuthor ">William R.  Sellers</span>, <span class="hlFld-ContribAuthor ">Darrin D.  Stuart</span>. </span><span class="cited-content_cbyCitation_article-title">Antitumor Properties of RAF709, a Highly Selective and Potent Inhibitor of RAF Kinase Dimers, in Tumors Driven by Mutant RAS or BRAF. </span><span class="cited-content_cbyCitation_journal-name">Cancer Research</span><span> <strong>2018,</strong> <em>78 </em>
                                    (6)
                                     , 1537-1548. <a href="https://doi.org/10.1158/0008-5472.CAN-17-2033" title="DOI URL">https://doi.org/10.1158/0008-5472.CAN-17-2033</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/0008-5472.CAN-17-2033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F0008-5472.CAN-17-2033%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Research%26atitle%3DAntitumor%252BProperties%252Bof%252BRAF709%25252C%252Ba%252BHighly%252BSelective%252Band%252BPotent%252BInhibitor%252Bof%252BRAF%252BKinase%252BDimers%25252C%252Bin%252BTumors%252BDriven%252Bby%252BMutant%252BRAS%252Bor%252BBRAF%26aulast%3DShao%26aufirst%3DWenlin%26date%3D2018%26date%3D2018%26volume%3D78%26issue%3D6%26spage%3D1537%26epage%3D1548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Jin  Wang</span>, <span class="hlFld-ContribAuthor ">Ying  Shen</span>, <span class="hlFld-ContribAuthor ">Hui-Yan  Wang</span>, <span class="hlFld-ContribAuthor ">Wei-Ming  Duan</span>, <span class="hlFld-ContribAuthor ">Hong-Yi  Zhao</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Hei</span>, <span class="hlFld-ContribAuthor ">Minhang  Xin</span>, <span class="hlFld-ContribAuthor ">Yong-Xiao  Cao</span>, <span class="hlFld-ContribAuthor ">San-Qi  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of 2,4,6-trisubstitued pyrido[3,4-d]pyrimidine derivatives as new EGFR-TKIs. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>148 </em>, 221-237. <a href="https://doi.org/10.1016/j.ejmech.2018.02.051" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.02.051</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.02.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.02.051%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B2%25252C4%25252C6-trisubstitued%252Bpyrido%25255B3%25252C4-d%25255Dpyrimidine%252Bderivatives%252Bas%252Bnew%252BEGFR-TKIs%26aulast%3DZhang%26aufirst%3DHao%26date%3D2018%26volume%3D148%26spage%3D221%26epage%3D237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Viola  Previtali</span>, <span class="hlFld-ContribAuthor ">Cristina  Trujillo</span>, <span class="hlFld-ContribAuthor ">Jean-Charles  Boisson</span>, <span class="hlFld-ContribAuthor ">Hassan  Khartabil</span>, <span class="hlFld-ContribAuthor ">Eric  Hénon</span>, <span class="hlFld-ContribAuthor ">Isabel  Rozas</span>. </span><span class="cited-content_cbyCitation_article-title">Development of the first model of a phosphorylated, ATP/Mg
              2+
              -containing B-Raf monomer by molecular dynamics simulations: a tool for structure-based design. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2017,</strong> <em>19 </em>
                                    (46)
                                     , 31177-31185. <a href="https://doi.org/10.1039/C7CP05038K" title="DOI URL">https://doi.org/10.1039/C7CP05038K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7CP05038K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7CP05038K%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DDevelopment%252Bof%252Bthe%252Bfirst%252Bmodel%252Bof%252Ba%252Bphosphorylated%25252C%252BATP%25252FMg%252B2%25252B%252B-containing%252BB-Raf%252Bmonomer%252Bby%252Bmolecular%252Bdynamics%252Bsimulations%25253A%252Ba%252Btool%252Bfor%252Bstructure-based%252Bdesign%26aulast%3DPrevitali%26aufirst%3DViola%26date%3D2017%26date%3D2017%26volume%3D19%26issue%3D46%26spage%3D31177%26epage%3D31185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Box-plot of the distribution of fold-activation of pMEK in Calu-6 cells for each of the annotated types (mechanisms) of kinase inhibition<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(19)</a> based on medicinal chemistry scaffold matches. The 116 compounds in this analysis had a CRAF IC<sub>50</sub> varying from 42 nM to 30 pM and an EC<sub>50</sub> in the pMEK target modulation assay in Calu-6 cells <0.3 μM. The solid line at 2-fold-activation indicates the team’s assessment of a relevant cutoff.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray structure of <b>3</b> in the BRAF construct (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAL">5VAL</a>)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(21)</a> illustrating key features of protein–ligand interactions depiction of the crystallized ligand illustrating most important features of protein ligand interactions, labeled key residues, and a ligand pocket surface. All distances are in Å.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. KINOME<i>scan</i> profile of <b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0005.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Small molecule X-ray structure of compound <b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0006.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray structure of <b>14</b> in BRAF (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAM">5VAM</a>)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(21)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0007.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) pMEK inhibition and activation of <b>1</b> and <b>14</b> in Calu-6 RAS mutant cells. (B) Effect of <b>1</b> and <b>14</b> on the stimulation of dimerization in HCT116 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0008.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. PK/PD analysis of <b>14</b> in the human Calu-6 NSCLC xenograft in mice. Percent pERK inhibition (left axis) and free exposure (right axis) following a single dose of <b>14</b>, at time points indicated, in Calu-6 tumor bearing mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0009.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Efficacy of <b>14</b> in Calu-6 xenograft in mice. Tumor volumes (A) or percent body weight change from initial (B) treatment groups vs vehicle control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Core/Morpholine Bromides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1-bromo-2-(2-bromoethoxy)ethane, DMF, DIEA, 120 °C, 70–76%; (b) HCl, dioxane, 110 °C, 93%; (c) K<sub>2</sub>CO<sub>3</sub>, DMF, MeI, 76–91%; (d) morpholine, 100 °C, 61%.</p></p></figure><figure data-id="sch2" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0011.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Boronic Ester Intermediates<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) ArCOCl, DCM, DIEA, 96–100%; (b) Pd(dppf)Cl<sub>2</sub>–DCM, dioxane, bis(pinacolato)diboron, KOAc, 70–80%; (c) H<sub>2</sub>, Pd/C, 99%.</p></p></figure><figure data-id="sch3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/medium/jm-2016-01862v_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0012.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Final Compounds Synthesis<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01862/20170616/images/large/jm-2016-01862v_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01862&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Pd(dppf)Cl<sub>2</sub>–DCM, DME, 2 M Na<sub>2</sub>CO<sub>3</sub>, 120 °C; (b) Pd(dppf)Cl<sub>2</sub>–DCM, DME, 2 M Na<sub>2</sub>CO<sub>3</sub>, 120 °C, boronic ester, then DMF, EDC, HOAt, carboxylic acid; (c) R<sub>1</sub>-OH, NaH, dioxane or DMF, 80–105 °C.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i54">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74132" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74132" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 27 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Ostrem, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sos, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">503</span><span class="NLM_x"> (</span><span class="NLM_issue">7477</span><span class="NLM_x">) </span> <span class="NLM_fpage">548</span><span class="NLM_x">–</span> <span class="NLM_lpage">551</span><span class="refDoi"> DOI: 10.1038/nature12796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1038%2Fnature12796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=24256730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=548-551&issue=7477&author=J.+M.+Ostremauthor=U.+Petersauthor=M.+L.+Sosauthor=J.+A.+Wellsauthor=K.+M.+Shokat&title=K-Ras%28G12C%29+inhibitors+allosterically+control+GTP+affinity+and+effector+interactions&doi=10.1038%2Fnature12796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span></div><div class="casAuthors">Ostrem, Jonathan M.; Peters, Ulf; Sos, Martin L.; Wells, James A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7477</span>),
    <span class="NLM_cas:pages">548-551</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Somatic mutations in the small GTPase K-Ras are the most common activating lesions found in human cancer, and are generally assocd. with poor response to std. therapies.  Efforts to target this oncogene directly have faced difficulties owing to its picomolar affinity for GTP/GDP and the absence of known allosteric regulatory sites.  Oncogenic mutations result in functional activation of Ras family proteins by impairing GTP hydrolysis.  With diminished regulation by GTPase activity, the nucleotide state of Ras becomes more dependent on relative nucleotide affinity and concn.  This gives GTP an advantage over GDP and increases the proportion of active GTP-bound Ras.  Here we report the development of small mols. that irreversibly bind to a common oncogenic mutant, K-Ras(G12C).  These compds. rely on the mutant cysteine for binding and therefore do not affect the wild-type protein.  Crystallog. studies reveal the formation of a new pocket that is not apparent in previous structures of Ras, beneath the effector binding switch-II region.  Binding of these inhibitors to K-Ras(G12C) disrupts both switch-I and switch-II, subverting the native nucleotide preference to favor GDP over GTP and impairing binding to Raf.  Our data provide structure-based validation of a new allosteric regulatory site on Ras that is targetable in a mutant-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokkgcVXuBn_rVg90H21EOLACvtfcHk0liQv3pa2JGuSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO&md5=27c1e7d8ef6b7e4b2919e9f00a073923</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature12796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12796%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DK-Ras%2528G12C%2529%2520inhibitors%2520allosterically%2520control%2520GTP%2520affinity%2520and%2520effector%2520interactions%26jtitle%3DNature%26date%3D2013%26volume%3D503%26issue%3D7477%26spage%3D548%26epage%3D551%26doi%3D10.1038%2Fnature12796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Lito, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span> </span><span class="NLM_article-title">Allele-specific inhibitors inactivate mutant KRAS G12C by a trapping mechanism</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">351</span><span class="NLM_x"> (</span><span class="NLM_issue">6273</span><span class="NLM_x">) </span> <span class="NLM_fpage">604</span><span class="NLM_x">–</span> <span class="NLM_lpage">608</span><span class="refDoi"> DOI: 10.1126/science.aad6204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1126%2Fscience.aad6204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=26841430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2jt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=604-608&issue=6273&author=P.+Litoauthor=M.+Solomonauthor=L.+S.+Liauthor=R.+Hansenauthor=N.+Rosen&title=Allele-specific+inhibitors+inactivate+mutant+KRAS+G12C+by+a+trapping+mechanism&doi=10.1126%2Fscience.aad6204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Allele-specific inhibitors inactivate mutant KRAS G12C by a trapping mechanism</span></div><div class="casAuthors">Lito, Piro; Solomon, Martha; Li, Lian-Sheng; Hansen, Rasmus; Rosen, Neal</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6273</span>),
    <span class="NLM_cas:pages">604-608</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">It is thought that KRAS oncoproteins are constitutively active because their guanosine triphosphatase (GTPase) activity is disabled.  Consequently, drugs targeting the inactive or GDP-bound conformation are not expected to be effective.  We describe a mechanism that enables such drugs to inhibit KRASG12C signaling and cancer cell growth.  Inhibition requires intact GTPase activity and occurs because drug-bound KRASG12C is insusceptible to nucleotide exchange factors and thus trapped in its inactive state.  Indeed, mutants completely lacking GTPase activity and those promoting exchange reduced the potency of the drug.  Suppressing nucleotide exchange activity downstream of various tyrosine kinases enhanced KRASG12C inhibition, whereas its potentiation had the opposite effect.  These findings reveal that KRASG12C undergoes nucleotide cycling in cancer cells and provide a basis for developing effective therapies to treat KRASG12C-driven cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqK3AbqEZO6k7Vg90H21EOLACvtfcHk0liQv3pa2JGuSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2jt78%253D&md5=22a47908ac0146822ecf19f5d0c952de</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad6204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad6204%26sid%3Dliteratum%253Aachs%26aulast%3DLito%26aufirst%3DP.%26aulast%3DSolomon%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DL.%2BS.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DAllele-specific%2520inhibitors%2520inactivate%2520mutant%2520KRAS%2520G12C%2520by%2520a%2520trapping%2520mechanism%26jtitle%3DScience%26date%3D2016%26volume%3D351%26issue%3D6273%26spage%3D604%26epage%3D608%26doi%3D10.1126%2Fscience.aad6204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Patricelli, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janes, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kucharski, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ely, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Firdaus, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babbar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span> </span><span class="NLM_article-title">Selective inhibition of oncogenic KRAS output with small molecules targeting the inactive state</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">316</span><span class="NLM_x">–</span> <span class="NLM_lpage">329</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-1105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1158%2F2159-8290.CD-15-1105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=26739882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVyru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=316-329&issue=3&author=M.+P.+Patricelliauthor=M.+R.+Janesauthor=L.+S.+Liauthor=R.+Hansenauthor=U.+Petersauthor=L.+V.+Kesslerauthor=Y.+Chenauthor=J.+M.+Kucharskiauthor=J.+Fengauthor=T.+Elyauthor=J.+H.+Chenauthor=S.+J.+Firdausauthor=A.+Babbarauthor=P.+Renauthor=Y.+Liu&title=Selective+inhibition+of+oncogenic+KRAS+output+with+small+molecules+targeting+the+inactive+state&doi=10.1158%2F2159-8290.CD-15-1105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Oncogenic KRAS Output with Small Molecules Targeting the Inactive State</span></div><div class="casAuthors">Patricelli, Matthew P.; Janes, Matthew R.; Li, Lian-Sheng; Hansen, Rasmus; Peters, Ulf; Kessler, Linda V.; Chen, Yuching; Kucharski, Jeff M.; Feng, Jun; Ely, Tess; Chen, Jeffrey H.; Firdaus, Sarah J.; Babbar, Anjali; Ren, Pingda; Liu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">316-329</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">KRAS gain-of-function mutations occur in approx. 30% of all human cancers.  Despite more than 30 years of KRAS-focused research and development efforts, no targeted therapy has been discovered for cancers with KRAS mutations.  Here, we describe ARS-853, a selective, covalent inhibitor of KRASG12C that inhibits mutant KRAS-driven signaling by binding to the GDP-bound oncoprotein and preventing activation.  Based on the rates of engagement and inhibition obsd. for ARS-853, along with a mutant-specific mass spectrometry-based assay for assessing KRAS activation status, we show that the nucleotide state of KRASG12C is in a state of dynamic flux that can be modulated by upstream signaling factors.  These studies provide convincing evidence that the KRASG12C mutation generates a "hyperexcitable" rather than a "statically active" state and that targeting the inactive, GDP-bound form is a promising approach for generating novel anti-RAS therapeutics.  Significance: A cell-active, mutant-specific, covalent inhibitor of KRASG12C is described that targets the GDP-bound, inactive state and prevents subsequent activation.  Using this novel compd., we demonstrate that KRASG12C oncoprotein rapidly cycles bound nucleotide and responds to upstream signaling inputs to maintain a highly active state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJJA-9IIHeN7Vg90H21EOLACvtfcHk0liQv3pa2JGuSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVyru7s%253D&md5=41db2f02006e8f0be7a9bb47354addcd</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-1105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-1105%26sid%3Dliteratum%253Aachs%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DLi%26aufirst%3DL.%2BS.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DKessler%26aufirst%3DL.%2BV.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKucharski%26aufirst%3DJ.%2BM.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DEly%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DFirdaus%26aufirst%3DS.%2BJ.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520oncogenic%2520KRAS%2520output%2520with%2520small%2520molecules%2520targeting%2520the%2520inactive%2520state%26jtitle%3DCancer%2520Discovery%26date%3D2016%26volume%3D6%26issue%3D3%26spage%3D316%26epage%3D329%26doi%3D10.1158%2F2159-8290.CD-15-1105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Karreth, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frese, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeNicola, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baccarini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuveson, D. A.</span><span> </span><span class="NLM_article-title">C-Raf is required for the initiation of lung cancer by K-Ras (G12D)</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">128</span><span class="NLM_x">–</span> <span class="NLM_lpage">136</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-10-0044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1158%2F2159-8290.CD-10-0044" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2011&pages=128-136&issue=2&author=F.+A.+Karrethauthor=K.+K.+Freseauthor=G.+M.+DeNicolaauthor=M.+Baccariniauthor=D.+A.+Tuveson&title=C-Raf+is+required+for+the+initiation+of+lung+cancer+by+K-Ras+%28G12D%29&doi=10.1158%2F2159-8290.CD-10-0044"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-10-0044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-10-0044%26sid%3Dliteratum%253Aachs%26aulast%3DKarreth%26aufirst%3DF.%2BA.%26aulast%3DFrese%26aufirst%3DK.%2BK.%26aulast%3DDeNicola%26aufirst%3DG.%2BM.%26aulast%3DBaccarini%26aufirst%3DM.%26aulast%3DTuveson%26aufirst%3DD.%2BA.%26atitle%3DC-Raf%2520is%2520required%2520for%2520the%2520initiation%2520of%2520lung%2520cancer%2520by%2520K-Ras%2520%2528G12D%2529%26jtitle%3DCancer%2520Discovery%26date%3D2011%26volume%3D1%26issue%3D2%26spage%3D128%26epage%3D136%26doi%3D10.1158%2F2159-8290.CD-10-0044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Blasco, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francoz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santamaria, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canamero, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubus, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charron, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baccarini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbacid, M.</span><span> </span><span class="NLM_article-title">C-Raf, but not B-Raf, is essential for development of K-Ras oncogene-driven on-small cell lung carcinoma</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">652</span><span class="NLM_x">–</span> <span class="NLM_lpage">663</span><span class="refDoi"> DOI: 10.1016/j.ccr.2011.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1016%2Fj.ccr.2011.04.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=652-663&issue=5&author=R.+B.+Blascoauthor=S.+Francozauthor=D.+Santamariaauthor=M.+Canameroauthor=P.+Dubusauthor=J.+Charronauthor=M.+Baccariniauthor=M.+Barbacid&title=C-Raf%2C+but+not+B-Raf%2C+is+essential+for+development+of+K-Ras+oncogene-driven+on-small+cell+lung+carcinoma&doi=10.1016%2Fj.ccr.2011.04.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DBlasco%26aufirst%3DR.%2BB.%26aulast%3DFrancoz%26aufirst%3DS.%26aulast%3DSantamaria%26aufirst%3DD.%26aulast%3DCanamero%26aufirst%3DM.%26aulast%3DDubus%26aufirst%3DP.%26aulast%3DCharron%26aufirst%3DJ.%26aulast%3DBaccarini%26aufirst%3DM.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DC-Raf%252C%2520but%2520not%2520B-Raf%252C%2520is%2520essential%2520for%2520development%2520of%2520K-Ras%2520oncogene-driven%2520on-small%2520cell%2520lung%2520carcinoma%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D19%26issue%3D5%26spage%3D652%26epage%3D663%26doi%3D10.1016%2Fj.ccr.2011.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Caunt, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sale, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, S. J.</span><span> </span><span class="NLM_article-title">MEK1 and MEK2 inhibitors and cancer therapy: the long and winding road</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">577</span><span class="NLM_x">–</span> <span class="NLM_lpage">592</span><span class="refDoi"> DOI: 10.1038/nrc4000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1038%2Fnrc4000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=26399658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGhsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=577-592&issue=10&author=C.+J.+Cauntauthor=M.+J.+Saleauthor=P.+D.+Smithauthor=S.+J.+Cook&title=MEK1+and+MEK2+inhibitors+and+cancer+therapy%3A+the+long+and+winding+road&doi=10.1038%2Fnrc4000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">MEK1 and MEK2 inhibitors and cancer therapy: the long and winding road</span></div><div class="casAuthors">Caunt, Christopher J.; Sale, Matthew J.; Smith, Paul D.; Cook, Simon J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">577-592</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The role of the ERK signaling pathway in cancer is thought to be most prominent in tumors in which mutations in the receptor tyrosine kinases RAS, BRAF, CRAF, MEK1 or MEK2 drive growth factor-independent ERK1 and ERK2 activation and thence inappropriate cell proliferation and survival.  New drugs that inhibit RAF or MEK1 and MEK2 have recently been approved or are currently undergoing late-stage clin. evaluation.  In this Review, the authors consider the ERK pathway, focusing particularly on the role of MEK1 and MEK2, the 'gatekeepers' of ERK1/2 activity.  The authors discuss their validation as drug targets, the merits of targeting MEK1 and MEK2 vs. BRAF and the mechanisms of action of different inhibitors of MEK1 and MEK2.  The authors also consider how some of the systems-level properties (intrapathway regulatory loops and wider signaling network connections) of the ERK pathway present a challenge for the success of MEK1 and MEK2 inhibitors, discuss mechanisms of resistance to these inhibitors, and review their clin. progress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnTcSz8CvRcLVg90H21EOLACvtfcHk0lilq2bg5EpyBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGhsb%252FO&md5=b1fd44051024a59608dd61310a437c65</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrc4000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc4000%26sid%3Dliteratum%253Aachs%26aulast%3DCaunt%26aufirst%3DC.%2BJ.%26aulast%3DSale%26aufirst%3DM.%2BJ.%26aulast%3DSmith%26aufirst%3DP.%2BD.%26aulast%3DCook%26aufirst%3DS.%2BJ.%26atitle%3DMEK1%2520and%2520MEK2%2520inhibitors%2520and%2520cancer%2520therapy%253A%2520the%2520long%2520and%2520winding%2520road%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26issue%3D10%26spage%3D577%26epage%3D592%26doi%3D10.1038%2Fnrc4000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Joseph, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pratilas, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poulikakos, P. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halilovic, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Persaud, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xing, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viale, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chapman, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollag, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solit, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span> </span><span class="NLM_article-title">The RAF inhibitor PLX4032 inhibits ERK signaling and tumor cell proliferation in a V600E BRAF-selective manner</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x"> (</span><span class="NLM_issue">33</span><span class="NLM_x">) </span> <span class="NLM_fpage">14903</span><span class="NLM_x">–</span> <span class="NLM_lpage">14908</span><span class="refDoi"> DOI: 10.1073/pnas.1008990107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1073%2Fpnas.1008990107" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=14903-14908&issue=33&author=E.+W.+Josephauthor=C.+A.+Pratilasauthor=P.+I.+Poulikakosauthor=M.+Tadiauthor=W.+Wangauthor=B.+S.+Taylorauthor=E.+Halilovicauthor=Y.+Persaudauthor=F.+Xingauthor=A.+Vialeauthor=J.+Tsaiauthor=P.+B.+Chapmanauthor=G.+Bollagauthor=D.+B.+Solitauthor=N.+Rosen&title=The+RAF+inhibitor+PLX4032+inhibits+ERK+signaling+and+tumor+cell+proliferation+in+a+V600E+BRAF-selective+manner&doi=10.1073%2Fpnas.1008990107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1008990107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1008990107%26sid%3Dliteratum%253Aachs%26aulast%3DJoseph%26aufirst%3DE.%2BW.%26aulast%3DPratilas%26aufirst%3DC.%2BA.%26aulast%3DPoulikakos%26aufirst%3DP.%2BI.%26aulast%3DTadi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DTaylor%26aufirst%3DB.%2BS.%26aulast%3DHalilovic%26aufirst%3DE.%26aulast%3DPersaud%26aufirst%3DY.%26aulast%3DXing%26aufirst%3DF.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DChapman%26aufirst%3DP.%2BB.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DSolit%26aufirst%3DD.%2BB.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DThe%2520RAF%2520inhibitor%2520PLX4032%2520inhibits%2520ERK%2520signaling%2520and%2520tumor%2520cell%2520proliferation%2520in%2520a%2520V600E%2520BRAF-selective%2520manner%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26issue%3D33%26spage%3D14903%26epage%3D14908%26doi%3D10.1073%2Fpnas.1008990107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Hatzivassiliou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandhuber, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarado, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludlam, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stokoe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gloor, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vigers, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morales, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aliagas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sideris, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeflich, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaiswal, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seshagiri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeppen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedman, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malek, S.</span><span> </span><span class="NLM_article-title">RAF inhibitors prime wild-type RAF to activate the MAPK pathway and enhance growth</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">464</span><span class="NLM_x"> (</span><span class="NLM_issue">7287</span><span class="NLM_x">) </span> <span class="NLM_fpage">431</span><span class="NLM_x">–</span> <span class="NLM_lpage">435</span><span class="refDoi"> DOI: 10.1038/nature08833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1038%2Fnature08833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=20130576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVygtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=431-435&issue=7287&author=G.+Hatzivassiliouauthor=K.+Songauthor=I.+Yenauthor=B.+J.+Brandhuberauthor=D.+J.+Andersonauthor=R.+Alvaradoauthor=M.+J.+Ludlamauthor=D.+Stokoeauthor=S.+L.+Gloorauthor=G.+Vigersauthor=T.+Moralesauthor=I.+Aliagasauthor=B.+Liuauthor=S.+Siderisauthor=K.+P.+Hoeflichauthor=B.+S.+Jaiswalauthor=S.+Seshagiriauthor=H.+Koeppenauthor=M.+Belvinauthor=L.+S.+Friedmanauthor=S.+Malek&title=RAF+inhibitors+prime+wild-type+RAF+to+activate+the+MAPK+pathway+and+enhance+growth&doi=10.1038%2Fnature08833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">RAF inhibitors prime wild-type RAF to activate the MAPK pathway and enhance growth</span></div><div class="casAuthors">Hatzivassiliou, Georgia; Song, Kyung; Yen, Ivana; Brandhuber, Barbara J.; Anderson, Daniel J.; Alvarado, Ryan; Ludlam, Mary J. C.; Stokoe, David; Gloor, Susan L.; Vigers, Guy; Morales, Tony; Aliagas, Ignacio; Liu, Bonnie; Sideris, Steve; Hoeflich, Klaus P.; Jaiswal, Bijay S.; Seshagiri, Somasekar; Koeppen, Hartmut; Belvin, Marcia; Friedman, Lori S.; Malek, Shiva</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7287</span>),
    <span class="NLM_cas:pages">431-435</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activating mutations in KRAS and BRAF are found in more than 30% of all human tumors and 40% of melanoma, resp., thus targeting this pathway could have broad therapeutic effects.  Small mol. ATP-competitive RAF kinase inhibitors have potent antitumor effects on mutant BRAF(V600E) tumors but, in contrast to mitogen-activated protein kinase kinase (MEK) inhibitors, are not potent against RAS mutant tumor models, despite RAF functioning as a key effector downstream of RAS and upstream of MEK.  Here we show that ATP-competitive RAF inhibitors have two opposing mechanisms of action depending on the cellular context.  In BRAF(V600E) tumors, RAF inhibitors effectively block the mitogen-activated protein kinase (MAPK) signalling pathway and decrease tumor growth.  Notably, in KRAS mutant and RAS/RAF wild-type tumors, RAF inhibitors activate the RAF-MEK-ERK pathway in a RAS-dependent manner, thus enhancing tumor growth in some xenograft models.  Inhibitor binding activates wild-type RAF isoforms by inducing dimerization, membrane localization and interaction with RAS-GTP.  These events occur independently of kinase inhibition and are, instead, linked to direct conformational effects of inhibitors on the RAF kinase domain.  On the basis of these findings, we demonstrate that ATP-competitive kinase inhibitors can have opposing functions as inhibitors or activators of signalling pathways, depending on the cellular context.  Furthermore, this work provides new insights into the therapeutic use of ATP-competitive RAF inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGounpcMETzVoLVg90H21EOLACvtfcHk0lilq2bg5EpyBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVygtb8%253D&md5=eb48bdf352ee9c9e182e63e38fd820a5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature08833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08833%26sid%3Dliteratum%253Aachs%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DBrandhuber%26aufirst%3DB.%2BJ.%26aulast%3DAnderson%26aufirst%3DD.%2BJ.%26aulast%3DAlvarado%26aufirst%3DR.%26aulast%3DLudlam%26aufirst%3DM.%2BJ.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DGloor%26aufirst%3DS.%2BL.%26aulast%3DVigers%26aufirst%3DG.%26aulast%3DMorales%26aufirst%3DT.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DJaiswal%26aufirst%3DB.%2BS.%26aulast%3DSeshagiri%26aufirst%3DS.%26aulast%3DKoeppen%26aufirst%3DH.%26aulast%3DBelvin%26aufirst%3DM.%26aulast%3DFriedman%26aufirst%3DL.%2BS.%26aulast%3DMalek%26aufirst%3DS.%26atitle%3DRAF%2520inhibitors%2520prime%2520wild-type%2520RAF%2520to%2520activate%2520the%2520MAPK%2520pathway%2520and%2520enhance%2520growth%26jtitle%3DNature%26date%3D2010%26volume%3D464%26issue%3D7287%26spage%3D431%26epage%3D435%26doi%3D10.1038%2Fnature08833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Heidorn, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milagre, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittaker, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nourry, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niculescu-Duvas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhomen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hussain, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reis-Filho, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Springer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marais, R.</span><span> </span><span class="NLM_article-title">Kinase-dead BRAF and oncogenic RAS cooperate to drive tumor progression through CRAF</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">140</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">209</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1016/j.cell.2009.12.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1016%2Fj.cell.2009.12.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=20141835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvVWitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2010&pages=209-221&issue=2&author=S.+J.+Heidornauthor=C.+Milagreauthor=S.+Whittakerauthor=A.+Nourryauthor=I.+Niculescu-Duvasauthor=N.+Dhomenauthor=J.+Hussainauthor=J.+S.+Reis-Filhoauthor=C.+J.+Springerauthor=C.+Pritchardauthor=R.+Marais&title=Kinase-dead+BRAF+and+oncogenic+RAS+cooperate+to+drive+tumor+progression+through+CRAF&doi=10.1016%2Fj.cell.2009.12.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase-dead BRAF and oncogenic RAS cooperate to drive tumor progression through CRAF</span></div><div class="casAuthors">Heidorn, Sonja J.; Milagre, Carla; Whittaker, Steven; Nourry, Arnaud; Niculescu-Duvas, Ion; Dhomen, Nathalie; Hussain, Jahan; Reis-Filho, Jorge S.; Springer, Caroline J.; Pritchard, Catrin; Marais, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">209-221</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We describe a mechanism of tumorigenesis mediated by kinase-dead BRAF in the presence of oncogenic RAS.  We show that drugs that selectively inhibit BRAF drive RAS-dependent BRAF binding to CRAF, CRAF activation, and MEK-ERK signaling.  This does not occur when oncogenic BRAF is inhibited, demonstrating that BRAF inhibition per se does not drive pathway activation; it only occurs when BRAF is inhibited in the presence of oncogenic RAS.  Kinase-dead BRAF mimics the effects of the BRAF-selective drugs and kinase-dead Braf and oncogenic Ras cooperate to induce melanoma in mice.  Our data reveal another paradigm of BRAF-mediated signaling that promotes tumor progression.  They highlight the importance of understanding pathway signaling in clin. practice and of genotyping tumors prior to administering BRAF-selective drugs, to identify patients who are likely to respond and also to identify patients who may experience adverse effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMSb6ckHpR07Vg90H21EOLACvtfcHk0lgZayaJrbebgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvVWitL0%253D&md5=80af840cb978f5956b3d998b185115f4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.12.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.12.040%26sid%3Dliteratum%253Aachs%26aulast%3DHeidorn%26aufirst%3DS.%2BJ.%26aulast%3DMilagre%26aufirst%3DC.%26aulast%3DWhittaker%26aufirst%3DS.%26aulast%3DNourry%26aufirst%3DA.%26aulast%3DNiculescu-Duvas%26aufirst%3DI.%26aulast%3DDhomen%26aufirst%3DN.%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DReis-Filho%26aufirst%3DJ.%2BS.%26aulast%3DSpringer%26aufirst%3DC.%2BJ.%26aulast%3DPritchard%26aufirst%3DC.%26aulast%3DMarais%26aufirst%3DR.%26atitle%3DKinase-dead%2520BRAF%2520and%2520oncogenic%2520RAS%2520cooperate%2520to%2520drive%2520tumor%2520progression%2520through%2520CRAF%26jtitle%3DCell%26date%3D2010%26volume%3D140%26issue%3D2%26spage%3D209%26epage%3D221%26doi%3D10.1016%2Fj.cell.2009.12.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Lito, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saborowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gadal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saborowski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastenhuber, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fellmann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohara, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morikami, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lukacs, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishii, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span> </span><span class="NLM_article-title">Disruption of CRAF-mediated MEK activation is required for effective MEK inhibition in KRAS mutant tumours</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">697</span><span class="NLM_x">–</span> <span class="NLM_lpage">710</span><span class="refDoi"> DOI: 10.1016/j.ccr.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1016%2Fj.ccr.2014.03.011" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=697-710&issue=5&author=P.+Litoauthor=A.+Saborowskiauthor=J.+Yueauthor=M.+Solomonauthor=E.+Josephauthor=S.+Gadalauthor=M.+Saborowskiauthor=E.+Kastenhuberauthor=C.+Fellmannauthor=K.+Oharaauthor=K.+Morikamiauthor=T.+Miuraauthor=C.+Lukacsauthor=N.+Ishiiauthor=S.+Loweauthor=N.+Rosen&title=Disruption+of+CRAF-mediated+MEK+activation+is+required+for+effective+MEK+inhibition+in+KRAS+mutant+tumours&doi=10.1016%2Fj.ccr.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DLito%26aufirst%3DP.%26aulast%3DSaborowski%26aufirst%3DA.%26aulast%3DYue%26aufirst%3DJ.%26aulast%3DSolomon%26aufirst%3DM.%26aulast%3DJoseph%26aufirst%3DE.%26aulast%3DGadal%26aufirst%3DS.%26aulast%3DSaborowski%26aufirst%3DM.%26aulast%3DKastenhuber%26aufirst%3DE.%26aulast%3DFellmann%26aufirst%3DC.%26aulast%3DOhara%26aufirst%3DK.%26aulast%3DMorikami%26aufirst%3DK.%26aulast%3DMiura%26aufirst%3DT.%26aulast%3DLukacs%26aufirst%3DC.%26aulast%3DIshii%26aufirst%3DN.%26aulast%3DLowe%26aufirst%3DS.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DDisruption%2520of%2520CRAF-mediated%2520MEK%2520activation%2520is%2520required%2520for%2520effective%2520MEK%2520inhibition%2520in%2520KRAS%2520mutant%2520tumours%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26issue%3D5%26spage%3D697%26epage%3D710%26doi%3D10.1016%2Fj.ccr.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Lamba, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lazzari, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cancelliere, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grernrum, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieftink, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernards, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiNicolantonio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardelli, A.</span><span> </span><span class="NLM_article-title">RAF suppression synergizes with MEK inhibition in KRAS mutant cancer cells</span> <span class="citation_source-journal">Cell Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1475</span><span class="NLM_x">–</span> <span class="NLM_lpage">1483</span><span class="refDoi"> DOI: 10.1016/j.celrep.2014.07.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1016%2Fj.celrep.2014.07.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=25199829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWltrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=1475-1483&issue=5&author=S.+Lambaauthor=M.+Russoauthor=C.+Sunauthor=L.+Lazzariauthor=C.+Cancelliereauthor=W.+Grernrumauthor=C.+Lieftinkauthor=R.+Bernardsauthor=F.+DiNicolantonioauthor=A.+Bardelli&title=RAF+suppression+synergizes+with+MEK+inhibition+in+KRAS+mutant+cancer+cells&doi=10.1016%2Fj.celrep.2014.07.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">RAF Suppression Synergizes with MEK Inhibition in KRAS Mutant Cancer Cells</span></div><div class="casAuthors">Lamba, Simona; Russo, Mariangela; Sun, Chong; Lazzari, Luca; Cancelliere, Carlotta; Grernrum, Wipawadee; Lieftink, Cor; Bernards, Rene; Di Nicolantonio, Federica; Bardelli, Alberto</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1475-1483</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">KRAS is the most frequently mutated oncogene in human cancer, yet no therapies are available to treat KRAS mutant cancers.  We used two independent reverse genetic approaches to identify components of the RAS-signaling pathways required for growth of KRAS mutant tumors.  Small interfering RNA (siRNA) screening of 37 KRAS mutant colorectal cancer cell lines showed that RAF1 suppression was synthetic lethal with MEK inhibition.  An unbiased kinome short hairpin RNA (shRNA)-based screen confirmed this synthetic lethal interaction in colorectal as well as in lung cancer cells bearing KRAS mutations.  Compds. targeting RAF kinases can reverse resistance to the MEK inhibitor selumetinib.  MEK inhibition induces RAS activation and BRAF-RAF1 dimerization and sustains MEK-ERK signaling, which is responsible for intrinsic resistance to selumetinib.  Prolonged dual blockade of RAF and MEK leads to persistent ERK suppression and efficiently induces apoptosis.  Our data underlie the relevance of developing combinatorial regimens of drugs targeting the RAF-MEK pathway in KRAS mutant tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokGXz97UHPObVg90H21EOLACvtfcHk0lgZayaJrbebgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWltrfO&md5=1ef6459ec755190ea299c71d07035e28</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.07.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.07.033%26sid%3Dliteratum%253Aachs%26aulast%3DLamba%26aufirst%3DS.%26aulast%3DRusso%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DLazzari%26aufirst%3DL.%26aulast%3DCancelliere%26aufirst%3DC.%26aulast%3DGrernrum%26aufirst%3DW.%26aulast%3DLieftink%26aufirst%3DC.%26aulast%3DBernards%26aufirst%3DR.%26aulast%3DDiNicolantonio%26aufirst%3DF.%26aulast%3DBardelli%26aufirst%3DA.%26atitle%3DRAF%2520suppression%2520synergizes%2520with%2520MEK%2520inhibition%2520in%2520KRAS%2520mutant%2520cancer%2520cells%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26issue%3D5%26spage%3D1475%26epage%3D1483%26doi%3D10.1016%2Fj.celrep.2014.07.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Peng, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henry, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaufman, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogeti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rutkoski, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wise, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Horn, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchanan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mochalkin, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kays, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donoho, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walgren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conti, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plowman, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starling, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, D. L.</span><span> </span><span class="NLM_article-title">Inhibition of RAF isoforms and active dimers by LY3009120 leads to anti-tumor activities in RAS or BRAF mutant cancers</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">398</span><span class="refDoi"> DOI: 10.1016/j.ccell.2015.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1016%2Fj.ccell.2015.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=26343583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVKgsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=384-398&issue=3&author=S.+B.+Pengauthor=J.+R.+Henryauthor=M.+D.+Kaufmanauthor=W.+P.+Luauthor=B.+D.+Smithauthor=S.+Vogetiauthor=T.+J.+Rutkoskiauthor=S.+Wiseauthor=L.+Chunauthor=+Zhangauthor=R.+D.+Van+Hornauthor=T.+Yinauthor=X.+Zhangauthor=V.+Yadavauthor=S.+H.+Chenauthor=X.+Gongauthor=X.+Maauthor=Y.+Websterauthor=S.+Buchananauthor=I.+Mochalkinauthor=L.+Huberauthor=L.+Kaysauthor=G.+P.+Donohoauthor=J.+Walgrenauthor=D.+McCannauthor=P.+Patelauthor=I.+Contiauthor=G.+D.+Plowmanauthor=J.+J.+Starlingauthor=D.+L.+Flynn&title=Inhibition+of+RAF+isoforms+and+active+dimers+by+LY3009120+leads+to+anti-tumor+activities+in+RAS+or+BRAF+mutant+cancers&doi=10.1016%2Fj.ccell.2015.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of RAF Isoforms and Active Dimers by LY3009120 Leads to Anti-tumor Activities in RAS or BRAF Mutant Cancers</span></div><div class="casAuthors">Peng, Sheng-Bin; Henry, James R.; Kaufman, Michael D.; Lu, Wei-Ping; Smith, Bryan D.; Vogeti, Subha; Rutloski, Thomas J.; Wise, Scott; Chun, Lawrence; Zhang, Youyan; Van Horn, Robert D.; Yin, Tinggui; Zhang, Xiaoyi; Yadav, Vipin; Chen, Shih-Hsun; Gong, Xueqian; Ma, Xiwen; Webster, Yue; Buchanan, Sean; Mochalkin, Igor; Huber, Lysiane; Kays, Lisa; Donoho, Gregory P.; Walgren, Jennie; McCann, Denis; Patel, Phenil; Conti, Ilaria; Plowman, Gregory D.; Starling, James J.; Flynn, Daniel L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">384-398</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">LY3009120 is a pan-RAF and RAF dimer inhibitor that inhibits all RAF isoforms and occupies both protomers in RAF dimers.  Biochem. and cellular analyses revealed that LY3009120 inhibits ARAF, BRAF, and CRAF isoforms with similar affinity, while vemurafenib or dabrafenib have little or modest CRAF activity compared to their BRAF activities.  LY3009120 induces BRAF-CRAF dimerization but inhibits the phosphorylation of downstream MEK and ERK, suggesting that it effectively inhibits the kinase activity of BRAF-CRAF heterodimers.  Further analyses demonstrated that LY3009120 also inhibits various forms of RAF dimers including BRAF or CRAF homodimers.  Due to these unique properties, LY3009120 demonstrates minimal paradoxical activation, inhibits MEK1/2 phosphorylation, and exhibits anti-tumor activities across multiple models carrying KRAS, NRAS, or BRAF mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJMiHK-Gub5bVg90H21EOLACvtfcHk0lgELroUp9sX1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVKgsr7O&md5=c167caa6e5a9b47a6ec1ee2ab6d9e0d1</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DS.%2BB.%26aulast%3DHenry%26aufirst%3DJ.%2BR.%26aulast%3DKaufman%26aufirst%3DM.%2BD.%26aulast%3DLu%26aufirst%3DW.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BD.%26aulast%3DVogeti%26aufirst%3DS.%26aulast%3DRutkoski%26aufirst%3DT.%2BJ.%26aulast%3DWise%26aufirst%3DS.%26aulast%3DChun%26aufirst%3DL.%26aulast%3DZhang%26aulast%3DVan%2BHorn%26aufirst%3DR.%2BD.%26aulast%3DYin%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DYadav%26aufirst%3DV.%26aulast%3DChen%26aufirst%3DS.%2BH.%26aulast%3DGong%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DWebster%26aufirst%3DY.%26aulast%3DBuchanan%26aufirst%3DS.%26aulast%3DMochalkin%26aufirst%3DI.%26aulast%3DHuber%26aufirst%3DL.%26aulast%3DKays%26aufirst%3DL.%26aulast%3DDonoho%26aufirst%3DG.%2BP.%26aulast%3DWalgren%26aufirst%3DJ.%26aulast%3DMcCann%26aufirst%3DD.%26aulast%3DPatel%26aufirst%3DP.%26aulast%3DConti%26aufirst%3DI.%26aulast%3DPlowman%26aufirst%3DG.%2BD.%26aulast%3DStarling%26aufirst%3DJ.%2BJ.%26aulast%3DFlynn%26aufirst%3DD.%2BL.%26atitle%3DInhibition%2520of%2520RAF%2520isoforms%2520and%2520active%2520dimers%2520by%2520LY3009120%2520leads%2520to%2520anti-tumor%2520activities%2520in%2520RAS%2520or%2520BRAF%2520mutant%2520cancers%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26issue%3D3%26spage%3D384%26epage%3D398%26doi%3D10.1016%2Fj.ccell.2015.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Rational design of inhibitors that bind to inactive kinase conformations</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span><span class="refDoi"> DOI: 10.1038/nchembio799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1038%2Fnchembio799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=16783341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=358-364&author=Y.+Liuauthor=N.+S.+Gray&title=Rational+design+of+inhibitors+that+bind+to+inactive+kinase+conformations&doi=10.1038%2Fnchembio799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design of inhibitors that bind to inactive kinase conformations</span></div><div class="casAuthors">Liu, Yi; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The majority of kinase inhibitors, known as type I inhibitors that have been developed so far target the ATP binding site of the kinase in its active conformation, in which the activation loop is phosphorylated.  Recently, crystal structures of inhibitors such as imatinib, BIRB796, and sorafenib (BAY43-9006), known as type II inhibitors, have revealed a new binding mode that exploits an addnl. binding site immediately adjacent to the region occupied by ATP.  This pocket is made accessible by an activation-loop rearrangement that is characteristic of kinases in an inactive conformation.  The examn. of binding features of known type II inhibitors is discussed and a pharmacophore model currently used in the design of a new generation of structurally diverse type II inhibitors is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxahsMPlhgDbVg90H21EOLACvtfcHk0lgELroUp9sX1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGjs7g%253D&md5=4a9347a44a2cf4a412c9741fdcc5696b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio799%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DRational%2520design%2520of%2520inhibitors%2520that%2520bind%2520to%2520inactive%2520kinase%2520conformations%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D358%26epage%3D364%26doi%3D10.1038%2Fnchembio799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Angiolini, M.</span><span> </span><span class="NLM_article-title">Targeting the DFG-in kinase conformation: a new trend emerging from patent analysis</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">309</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span><span class="refDoi"> DOI: 10.4155/fmc.10.294</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.4155%2Ffmc.10.294" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=21446845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVCltb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=309-337&author=M.+Angiolini&title=Targeting+the+DFG-in+kinase+conformation%3A+a+new+trend+emerging+from+patent+analysis&doi=10.4155%2Ffmc.10.294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the DFG-in kinase conformation: a new trend emerging from a patent analysis</span></div><div class="casAuthors">Angiolini, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">309-337</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Aberrant kinase signaling leads to a multitude of disease states.  The clin. and com. success of agents typified by imatinib or dasatinib in the treatment of hematol. malignancies has further validated kinase inhibition as a useful clin. strategy.  This increased interest in kinases as therapeutic targets is evidenced by the rapidly increasing no. of patent applications and peer-reviewed articles.  This article discusses recent Patent that describe small mols. targeting the DFG-in active kinase conformation, by the so-called type I1/21/2'' inhibitor, against a small set of clin. relevant targets such as B-Raf, p38αα, Jak2 and EphB4.  Preclin. and clin. data are also highlighted for the most promising new mol. entities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGWPNwqVr3XrVg90H21EOLACvtfcHk0lgELroUp9sX1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVCltb8%253D&md5=7c6cbe986141be40b6ae2109170fc41b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4155%2Ffmc.10.294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.10.294%26sid%3Dliteratum%253Aachs%26aulast%3DAngiolini%26aufirst%3DM.%26atitle%3DTargeting%2520the%2520DFG-in%2520kinase%2520conformation%253A%2520a%2520new%2520trend%2520emerging%2520from%2520patent%2520analysis%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D309%26epage%3D337%26doi%3D10.4155%2Ffmc.10.294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Ishikawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, Y.</span><span> </span><span class="NLM_article-title">Improvement in aqueous solubility in small molecule drug discovery programs by disruption of molecular planarity and symmetry</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1539</span><span class="NLM_x">–</span> <span class="NLM_lpage">1554</span><span class="refDoi"> DOI: 10.1021/jm101356p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101356p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisVWltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1539-1554&author=M.+Ishikawaauthor=Y.+Hashimoto&title=Improvement+in+aqueous+solubility+in+small+molecule+drug+discovery+programs+by+disruption+of+molecular+planarity+and+symmetry&doi=10.1021%2Fjm101356p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Improvement in Aqueous Solubility in Small Molecule Drug Discovery Programs by Disruption of Molecular Planarity and Symmetry</span></div><div class="casAuthors">Ishikawa, Minoru; Hashimoto, Yuichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1539-1554</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKbtbfvqKZjrVg90H21EOLACvtfcHk0li-gyXhOO3QEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisVWltLw%253D&md5=a88804b3850625ef9c3f7424366acb91</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm101356p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101356p%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DM.%26aulast%3DHashimoto%26aufirst%3DY.%26atitle%3DImprovement%2520in%2520aqueous%2520solubility%2520in%2520small%2520molecule%2520drug%2520discovery%2520programs%2520by%2520disruption%2520of%2520molecular%2520planarity%2520and%2520symmetry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1539%26epage%3D1554%26doi%3D10.1021%2Fjm101356p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Lovering, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bikker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humblet, C.</span><span> </span><span class="NLM_article-title">Escape from flatland: increasing saturation as an approach to improving clinical success</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6752</span><span class="NLM_x">–</span> <span class="NLM_lpage">6756</span><span class="refDoi"> DOI: 10.1021/jm901241e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901241e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6752-6756&author=F.+Loveringauthor=J.+Bikkerauthor=C.+Humblet&title=Escape+from+flatland%3A+increasing+saturation+as+an+approach+to+improving+clinical+success&doi=10.1021%2Fjm901241e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Escape from Flatland: Increasing Saturation as an Approach to Improving Clinical Success</span></div><div class="casAuthors">Lovering, Frank; Bikker, Jack; Humblet, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6752-6756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The medicinal chem. community has become increasingly aware of the value of tracking calcd. phys. properties such as mol. wt., topol. polar surface area, rotatable bonds, and hydrogen bond donors and acceptors.  The authors hypothesized that the shift to high-throughput synthetic practices over the past decade may be another factor that may predispose mols. to fail by steering discovery efforts toward achiral, arom. compds.  The authors have proposed two simple and interpretable measures of the complexity of mols. prepd. as potential drug candidates.  The first is carbon bond satn. as defined by fraction Sp3 (Fsp3) where Fsp3 = (no. of Sp3 hybridized carbons/total carbon count).  The second is simply whether a chiral carbon exists in the mol.  The authors demonstrate that both complexity (as measured by Fsp3) and the presence of chiral centers correlate with success as compds. transition from discovery, through clin. testing, to drugs.  To explain these observations, the authors further demonstrate that satn. correlates with soly., an exptl. phys. property important to success in the drug discovery setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC1f3Ed7nUBbVg90H21EOLACvtfcHk0li-gyXhOO3QEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN&md5=4ca92c30c17c53d77ad376719bad951e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm901241e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901241e%26sid%3Dliteratum%253Aachs%26aulast%3DLovering%26aufirst%3DF.%26aulast%3DBikker%26aufirst%3DJ.%26aulast%3DHumblet%26aufirst%3DC.%26atitle%3DEscape%2520from%2520flatland%253A%2520increasing%2520saturation%2520as%2520an%2520approach%2520to%2520improving%2520clinical%2520success%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6752%26epage%3D6756%26doi%3D10.1021%2Fjm901241e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Fabian, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biggs, W. H.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benedetti, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ford, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grotzfeld, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Insko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelias, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velasco, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.-M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0li-gyXhOO3QEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.-M.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Karaman, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faraoni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lgC7TGkrrU0iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Fabbro, D.</span><span> </span><span class="NLM_article-title">25 years of small molecular weight kinase inhibitors: potentials and limitations</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">766</span><span class="NLM_x">–</span> <span class="NLM_lpage">775</span><span class="refDoi"> DOI: 10.1124/mol.114.095489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1124%2Fmol.114.095489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=25549667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVKltLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2015&pages=766-775&issue=5&author=D.+Fabbro&title=25+years+of+small+molecular+weight+kinase+inhibitors%3A+potentials+and+limitations&doi=10.1124%2Fmol.114.095489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">25 years of small molecular weight kinase inhibitors: potentials and limitations</span></div><div class="casAuthors">Fabbro, Doriano</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">766-775</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Deregulation of protein and lipid kinase activities leads to a variety of pathologies, ranging from cancer inflammatory diseases, diabetes, infectious diseases, and cardiovascular disorders.  Protein kinases and lipid kinases represent, therefore, an important target for the pharmaceutical industry.  In fact, approx. one-third of all protein targets under investigation in the pharmaceutical industry are protein or lipid kinases.  To date, 30 kinase inhibitors have been approved, which, with few exceptions, are mainly for oncol. indications and directed against only a handful of protein and lipid kinases, leaving 70% of the kinome untapped.  Despite these successes in kinase drug discovery, the development of kinase inhibitors with outstanding selectivity, identification and validation of driver kinase(s) in diseases, and the emerging problem of resistance to the inhibition of key target kinases remain major challenges.  This mini review provides an insight into protein and lipid kinase drug discovery with respect to achievements, binding modes of inhibitors, and novel avenues for the generation of second-generation kinase inhibitors to treat cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZhF-9NQJx7bVg90H21EOLACvtfcHk0lgC7TGkrrU0iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVKltLk%253D&md5=6422b036e4e447379ac6c608662954c2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1124%2Fmol.114.095489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.114.095489%26sid%3Dliteratum%253Aachs%26aulast%3DFabbro%26aufirst%3DD.%26atitle%3D25%2520years%2520of%2520small%2520molecular%2520weight%2520kinase%2520inhibitors%253A%2520potentials%2520and%2520limitations%26jtitle%3DMol.%2520Pharmacol.%26date%3D2015%26volume%3D87%26issue%3D5%26spage%3D766%26epage%3D775%26doi%3D10.1124%2Fmol.114.095489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Vijayan, R. S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Modi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duong-Ly, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunbrack, R. L.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, R. M.</span><span> </span><span class="NLM_article-title">Conformational analysis of the DFG-Out kinase motif and biochemical profiling of structurally validated Type II inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">466</span><span class="NLM_x">–</span> <span class="NLM_lpage">479</span><span class="refDoi"> DOI: 10.1021/jm501603h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501603h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCku7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=466-479&author=R.+S.+K.+Vijayanauthor=P.+Heauthor=V.+Modiauthor=K.+C.+Duong-Lyauthor=H.+Maauthor=J.+R.+Petersonauthor=R.+L.+Dunbrackauthor=R.+M.+Levy&title=Conformational+analysis+of+the+DFG-Out+kinase+motif+and+biochemical+profiling+of+structurally+validated+Type+II+inhibitors&doi=10.1021%2Fjm501603h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of the DFG-Out Kinase Motif and Biochemical Profiling of Structurally Validated Type II Inhibitors</span></div><div class="casAuthors">Vijayan, R. S. K.; He, Peng; Modi, Vivek; Duong-Ly, Krisna C.; Ma, Haiching; Peterson, Jeffrey R.; Dunbrack, Roland L.; Levy, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">466-479</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structural coverage of the human kinome has been steadily increasing over time.  The structures provide valuable insights into the mol. basis of kinase function and also provide a foundation for understanding the mechanisms of kinase inhibitors.  There are a large no. of kinase structures in the PDB for which the Asp and Phe of the DFG motif on the activation loop swap positions, resulting in the formation of a new allosteric pocket.  The authors refer to these structures as "classical DFG-out" conformations in order to distinguish them from conformations that have also been referred to as DFG-out in the literature but that do not have a fully formed allosteric pocket.  The authors have completed a structural anal. of almost 200 small mol. inhibitors bound to classical DFG-out conformations; the authors find that they are recognized by both type I and type II inhibitors.  In contrast, the authors find that nonclassical DFG-out conformations strongly select against type II inhibitors because these structures have not formed a large enough allosteric pocket to accommodate this type of binding mode.  In the course of this study the authors discovered that the no. of structurally validated type II inhibitors that can be found in the PDB and that are also represented in publicly available biochem. profiling studies of kinase inhibitors is very small.  The authors have obtained new profiling results for several addnl. structurally validated type II inhibitors identified through the conformational anal.  Although the available profiling data for type II inhibitors is still much smaller than for type I inhibitors, a comparison of the two data sets supports the conclusion that type II inhibitors are more selective than type I.  The authors comment on the possible contribution of the DFG-in to DFG-out conformational reorganization to the selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb9E5JKOmDZ7Vg90H21EOLACvtfcHk0lgC7TGkrrU0iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCku7rP&md5=6d729997f242d582a4adff9463fd585c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm501603h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501603h%26sid%3Dliteratum%253Aachs%26aulast%3DVijayan%26aufirst%3DR.%2BS.%2BK.%26aulast%3DHe%26aufirst%3DP.%26aulast%3DModi%26aufirst%3DV.%26aulast%3DDuong-Ly%26aufirst%3DK.%2BC.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26aulast%3DDunbrack%26aufirst%3DR.%2BL.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DConformational%2520analysis%2520of%2520the%2520DFG-Out%2520kinase%2520motif%2520and%2520biochemical%2520profiling%2520of%2520structurally%2520validated%2520Type%2520II%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D466%26epage%3D479%26doi%3D10.1021%2Fjm501603h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">21</a></strong><div class="note"><p class="first last">Experimental detail for the small molecule crystal structure of <b>7</b>, and X-ray cocrystal structure data table of <b>3</b> and <b>14</b> and kinase profile of <b>7</b> are given in the <a href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_001.pdf" class="ext-link">Supporting Information</a></p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">22</a></strong><div class="note"><p class="first last">Manuscript discussing the entire biology of <b>14</b> has been submitted for publication.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Schuffenhauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roggo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetzel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">The scaffold tree – visualization of the scaffold universe by hierarchical scaffold classification</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span><span class="refDoi"> DOI: 10.1021/ci600338x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600338x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=47-58&author=A.+Schuffenhauerauthor=P.+Ertlauthor=S.+Roggoauthor=S.+Wetzelauthor=M.+A.+Kochauthor=H.+Waldmann&title=The+scaffold+tree+%E2%80%93+visualization+of+the+scaffold+universe+by+hierarchical+scaffold+classification&doi=10.1021%2Fci600338x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The Scaffold Tree - Visualization of the Scaffold Universe by Hierarchical Scaffold Classification</span></div><div class="casAuthors">Schuffenhauer, Ansgar; Ertl, Peter; Roggo, Silvio; Wetzel, Stefan; Koch, Marcus A.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A hierarchical classification of chem. scaffolds (mol. framework, which is obtained by pruning all terminal side chains) has been introduced.  The mol. frameworks form the leaf nodes in the hierarchy trees.  By an iterative removal of rings, scaffolds forming the higher levels in the hierarchy tree are obtained.  Prioritization rules ensure that less characteristic, peripheral rings are removed first.  All scaffolds in the hierarchy tree are well-defined chem. entities making the classification chem. intuitive.  The classification is deterministic, data-set-independent, and scales linearly with the no. of compds. included in the data set.  The application of the classification is demonstrated on two data sets extd. from the PubChem database, namely, pyruvate kinase binders and a collection of pesticides.  The examples shown demonstrate that the classification procedure handles robustly synthetic structures and natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8XfR2xvGasbVg90H21EOLACvtfcHk0lixt_Mf4tW7hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO&md5=89cec5a01a37802ab0826ecababfaf0a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fci600338x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600338x%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRoggo%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DKoch%26aufirst%3DM.%2BA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DThe%2520scaffold%2520tree%2520%25E2%2580%2593%2520visualization%2520of%2520the%2520scaffold%2520universe%2520by%2520hierarchical%2520scaffold%2520classification%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D47%26epage%3D58%26doi%3D10.1021%2Fci600338x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Vonrhein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flensburg, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smart, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paciorek, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Womack, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bricogne, G.</span><span> </span><span class="NLM_article-title">Data processing and analysis with the <i>autoPROC</i> toolbox</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">302</span><span class="refDoi"> DOI: 10.1107/S0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+processing+and+analysis+with+the+autoPROC+toolbox&doi=10.1107%2FS0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0lixt_Mf4tW7hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520processing%2520and%2520analysis%2520with%2520the%2520autoPROC%2520toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2FS0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title"><i>Phaser</i> crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lixt_Mf4tW7hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Smart, O. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Womack, T. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flensburg, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paciorek, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vonrhein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bricogne, G.</span><span> </span><span class="NLM_article-title">Exploiting structure similarity in refinement: automated NCS and target-structure restraints in <i>BUSTER</i></span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">368</span><span class="NLM_x">–</span> <span class="NLM_lpage">380</span><span class="refDoi"> DOI: 10.1107/S0907444911056058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1107%2FS0907444911056058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=22505257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Gns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=368-380&author=O.+S.+Smartauthor=T.+O.+Womackauthor=C.+Flensburgauthor=P.+Kellerauthor=W.+Paciorekauthor=A.+Sharffauthor=C.+Vonrheinauthor=G.+Bricogne&title=Exploiting+structure+similarity+in+refinement%3A+automated+NCS+and+target-structure+restraints+in+BUSTER&doi=10.1107%2FS0907444911056058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting structure similarity in refinement: automated NCS and target-structure restraints in BUSTER</span></div><div class="casAuthors">Smart, Oliver S.; Womack, Thomas O.; Flensburg, Claus; Keller, Peter; Paciorek, Wlodek; Sharff, Andrew; Vonrhein, Clemens; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">368-380</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Maximum-likelihood X-ray macromol. structure refinement in BUSTER has been extended with restraints facilitating the exploitation of structural similarity.  The similarity can be between two or more chains within the structure being refined, thus favoring NCS, or to a distinct 'target' structure that remains fixed during refinement.  The local structural similarity restraints (LSSR) approach considers all distances less than 5.5 Å between pairs of atoms in the chain to be restrained.  For each, the difference from the distance between the corresponding atoms in the related chain is found.  LSSR applies a restraint penalty on each difference.  A functional form that reaches a plateau for large differences is used to avoid the restraints distorting parts of the structure that are not similar.  Because LSSR are local, there is no need to sep. out domains.  Some restraint pruning is still necessary, but this has been automated.  LSSR have been available to academic users of BUSTER since 2009 with the easy-to-use and options.  The use of LSSR is illustrated in the re-refinement of PDB entries , where enables the correct ligand-binding structure to be found, and , where contributes to the location of an addnl. copy of the cyclic peptide ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu-qo0DGjn3bVg90H21EOLACvtfcHk0liujzIQsGSIgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Gns78%253D&md5=bea89f86f489e8dbd00a9866a3d7dab9</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444911056058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911056058%26sid%3Dliteratum%253Aachs%26aulast%3DSmart%26aufirst%3DO.%2BS.%26aulast%3DWomack%26aufirst%3DT.%2BO.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DExploiting%2520structure%2520similarity%2520in%2520refinement%253A%2520automated%2520NCS%2520and%2520target-structure%2520restraints%2520in%2520BUSTER%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D368%26epage%3D380%26doi%3D10.1107%2FS0907444911056058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liujzIQsGSIgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAL','PDB','5VAL'); return false;">PDB: 5VAL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAM','PDB','5VAM'); return false;">PDB: 5VAM</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i50"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01862">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_65046"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01862">10.1021/acs.jmedchem.6b01862</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Cellular assay conditions, compound <b>7</b> data for the 457 kinase KINOME<i>scan</i>, X-ray data table for <b>3</b> and <b>14</b>, extended data for the small molecule crystal structure <b>7</b>, PC3 xenograft study data for <b>14</b> with experimental details (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Machine-readable representation of structures with key data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_001.pdf">jm6b01862_si_001.pdf (843.38 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01862/suppl_file/jm6b01862_si_002.csv">jm6b01862_si_002.csv (0.84 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01862&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-12%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01862%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01862" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a804dca5ad18b","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
