
<html lang="en"     class="pb-page"  data-request-id="28a026b2-8e7b-498d-98eb-91db36e0d5d1"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.7b01342;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2018.61.issue-1;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Structure-Based Design of 6-Chloro-4-aminoquinazoline-2-carboxamide Derivatives as Potent and Selective p21-Activated Kinase 4 (PAK4) Inhibitors" /></meta><meta name="dc.Creator" content="Chenzhou  Hao" /></meta><meta name="dc.Creator" content="Fan  Zhao" /></meta><meta name="dc.Creator" content="Hongyan  Song" /></meta><meta name="dc.Creator" content="Jing  Guo" /></meta><meta name="dc.Creator" content="Xiaodong  Li" /></meta><meta name="dc.Creator" content="Xiaolin  Jiang" /></meta><meta name="dc.Creator" content="Ran  Huan" /></meta><meta name="dc.Creator" content="Shuai  Song" /></meta><meta name="dc.Creator" content="Qiaoling  Zhang" /></meta><meta name="dc.Creator" content="Ruifeng  Wang" /></meta><meta name="dc.Creator" content="Kai  Wang" /></meta><meta name="dc.Creator" content="Yu  Pang" /></meta><meta name="dc.Creator" content="Tongchao  Liu" /></meta><meta name="dc.Creator" content="Tianqi  Lu" /></meta><meta name="dc.Creator" content="Wanxu  Huang" /></meta><meta name="dc.Creator" content="Jian  Wang" /></meta><meta name="dc.Creator" content="Bin  Lin" /></meta><meta name="dc.Creator" content="Zhonggui  He" /></meta><meta name="dc.Creator" content="Haitao  Li" /></meta><meta name="dc.Creator" content="Feng  Li" /></meta><meta name="dc.Creator" content="Dongmei  Zhao" /></meta><meta name="dc.Creator" content="Maosheng  Cheng" /></meta><meta name="dc.Description" content="Herein, we report the discovery and characterization of a novel class of PAK4 inhibitors with a quinazoline scaffold. Based on the shape and chemical composition of the ATP-binding pocket of PAKs, ..." /></meta><meta name="Description" content="Herein, we report the discovery and characterization of a novel class of PAK4 inhibitors with a quinazoline scaffold. Based on the shape and chemical composition of the ATP-binding pocket of PAKs, ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 30, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01342" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01342" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01342" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01342" /></link>
        
    
    

<title>Structure-Based Design of 6-Chloro-4-aminoquinazoline-2-carboxamide Derivatives as Potent and Selective p21-Activated Kinase 4 (PAK4) Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01342" /></meta><meta property="og:title" content="Structure-Based Design of 6-Chloro-4-aminoquinazoline-2-carboxamide Derivatives as Potent and Selective p21-Activated Kinase 4 (PAK4) Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0022.jpeg" /></meta><meta property="og:description" content="Herein, we report the discovery and characterization of a novel class of PAK4 inhibitors with a quinazoline scaffold. Based on the shape and chemical composition of the ATP-binding pocket of PAKs, we chose a 2,4-diaminoquinazoline series of inhibitors as a starting point. Guided by X-ray crystallography and a structure-based drug design (SBDD) approach, a series of novel 4-aminoquinazoline-2-carboxamide PAK4 inhibitors were designed and synthesized. The inhibitors’ selectivity, therapeutic potency, and pharmaceutical properties were optimized. One of the best compounds, 31 (CZh226), showed remarkable PAK4 selectivity (346-fold vs PAK1) and favorable kinase selectivity profile. Moreover, this compound potently inhibited the migration and invasion of A549 tumor cells by regulating the PAK4-directed downstream signaling pathways in vitro. Taken together, these data support the further development of 31 as a lead compound for PAK4-targeted anticancer drug discovery and as a valuable research probe for the further biological investigation of group II PAKs." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01342"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01342">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01342&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01342&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01342&amp;href=/doi/10.1021/acs.jmedchem.7b01342" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 265-285</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01323" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01345" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Design of 6-Chloro-4-aminoquinazoline-2-carboxamide Derivatives as Potent and Selective p21-Activated Kinase 4 (PAK4) Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chenzhou++Hao">Chenzhou Hao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>§</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-6803-7270" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fan++Zhao">Fan Zhao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol "><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hongyan++Song">Hongyan Song</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span><span class="author-xref-symbol "><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Guo">Jing Guo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaodong++Li">Xiaodong Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaolin++Jiang">Xiaolin Jiang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ran++Huan">Ran Huan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shuai++Song">Shuai Song</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Qiaoling++Zhang">Qiaoling Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ruifeng++Wang">Ruifeng Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kai++Wang">Kai Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yu++Pang">Yu Pang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tongchao++Liu">Tongchao Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tianqi++Lu">Tianqi Lu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wanxu++Huang">Wanxu Huang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Wang">Jian Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Bin++Lin">Bin Lin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhonggui++He">Zhonggui He</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Haitao++Li">Haitao Li</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Feng++Li">Feng Li</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dongmei++Zhao">Dongmei Zhao</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Maosheng++Cheng">Maosheng Cheng</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Key Laboratory of Structure-Based Drug Design & Discovery of Ministry of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Cell Biology, Key Laboratory of Cell Biology, Ministry of Public Health, and Key Laboratory of Medical Cell Biology, Ministry of Education, China Medical University, Shenyang 110001, China</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">∥</span> <span class="aff-text">Tsinghua-Peking Joint Center for Life Sciences, Beijing Advanced Innovation Center for Structural Biology, Department of Basic Medical Sciences, School of Medicine, Tsinghua University, Beijing 100084, China</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">⊥</span> <span class="aff-text">Institute of Materials Research and Engineering, A*STAR (Agency for Science, Technology and Research), 2 Fusionopolis Way, Innovis, #08-03, Singapore 138634</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">#</span> <span class="aff-text">School of Pharmacy, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div><div class="corresp-info"><strong>*</strong>(H.L.) Phone: +86 10 62771392. E-mail: <a href="/cdn-cgi/l/email-protection#dfb3b7ab9fabacb6b1b8b7aabef1babbaaf1bcb1"><span class="__cf_email__" data-cfemail="204c4854605453494e474855410e4544550e434e">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>(F.L.) Phone: +86 24 23256666. E-mail: <a href="/cdn-cgi/l/email-protection#2543494c6548444c490b4648500b4041500b464b"><span class="__cf_email__" data-cfemail="0c6a60654c616d6560226f617922696879226f62">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>(D.Z.) Phone: +86 24 43520219. E-mail: <a href="/cdn-cgi/l/email-protection#9afef5f4fdf7fff3e0b7acaddaabaca9b4f9f5f7"><span class="__cf_email__" data-cfemail="a0c4cfcec7cdc5c9da8d9697e09196938ec3cfcd">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>(M.C.) Phone: +86 24 43520019. E-mail: <a href="/cdn-cgi/l/email-protection#563b25353e33383116252f263e2378333223783538"><span class="__cf_email__" data-cfemail="91fce2f2f9f4fff6d1e2e8e1f9e4bff4f5e4bff2ff">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01342&amp;href=/doi/10.1021%2Facs.jmedchem.7b01342" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 265–285</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 30, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 September 2017</li><li><span class="item_label"><b>Published</b> online</span>30 December 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 January 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01342" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01342</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ec-1612971054143.svg" alt="ACS Editors’ Choice" /></img><span>ACS Editors’ Choice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01342"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">8284</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">18</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01342" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Design of 6-Chloro-4-aminoquinazoline-2-carboxamide Derivatives as Potent and Selective p21-Activated Kinase 4 (PAK4) Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Chenzhou&quot;,&quot;last_name&quot;:&quot;Hao&quot;},{&quot;first_name&quot;:&quot;Fan&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Hongyan&quot;,&quot;last_name&quot;:&quot;Song&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Guo&quot;},{&quot;first_name&quot;:&quot;Xiaodong&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Xiaolin&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Ran&quot;,&quot;last_name&quot;:&quot;Huan&quot;},{&quot;first_name&quot;:&quot;Shuai&quot;,&quot;last_name&quot;:&quot;Song&quot;},{&quot;first_name&quot;:&quot;Qiaoling&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Ruifeng&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Kai&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Yu&quot;,&quot;last_name&quot;:&quot;Pang&quot;},{&quot;first_name&quot;:&quot;Tongchao&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Tianqi&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Wanxu&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Bin&quot;,&quot;last_name&quot;:&quot;Lin&quot;},{&quot;first_name&quot;:&quot;Zhonggui&quot;,&quot;last_name&quot;:&quot;He&quot;},{&quot;first_name&quot;:&quot;Haitao&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Feng&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Dongmei&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Maosheng&quot;,&quot;last_name&quot;:&quot;Cheng&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;265-285&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01342&quot;},&quot;abstract&quot;:&quot;Herein, we report the discovery and characterization of a novel class of PAK4 inhibitors with a quinazoline scaffold. Based on the shape and chemical composition of the ATP-binding pocket of PAKs, we chose a 2,4-diaminoquinazoline series of inhibitors as a starting point. Guided by X-ray crystallography and a structure-based drug design (SBDD) approach, a series of novel 4-aminoquinazoline-2-carboxamide PAK4 inhibitors were designed and synthesized. The inhibitors’ selectivity, therapeutic potency, and pharmaceutical properties were optimized. One of the best compounds, 31 (CZh226), showed remarkable PAK4 selectivity (346-fold vs PAK1) and favorable kinase selectivity profile. Moreover, this compound potently inhibited the migration and invasion of A549 tumor cells by regulating the PAK4-directed downstream signaling pathways in vitro. Taken together, these data support the further development of 31 as a lead compound for PAK4-targeted anticancer drug discovery and as a valuable research probe for the further&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01342&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01342" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01342&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01342" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01342&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01342" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01342&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01342&amp;href=/doi/10.1021/acs.jmedchem.7b01342" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01342" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01342" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01342%26sid%3Dliteratum%253Aachs%26pmid%3D29190083%26genre%3Darticle%26aulast%3DHao%26date%3D2018%26atitle%3DStructure-Based%2BDesign%2Bof%2B6-Chloro-4-aminoquinazoline-2-carboxamide%2BDerivatives%2Bas%2BPotent%2Band%2BSelective%2Bp21-Activated%2BKinase%2B4%2B%2528PAK4%2529%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D1%26spage%3D265%26epage%3D285%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/jmcmar.2018.61.issue-1/20180111/jmcmar.2018.61.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Herein, we report the discovery and characterization of a novel class of PAK4 inhibitors with a quinazoline scaffold. Based on the shape and chemical composition of the ATP-binding pocket of PAKs, we chose a 2,4-diaminoquinazoline series of inhibitors as a starting point. Guided by X-ray crystallography and a structure-based drug design (SBDD) approach, a series of novel 4-aminoquinazoline-2-carboxamide PAK4 inhibitors were designed and synthesized. The inhibitors’ selectivity, therapeutic potency, and pharmaceutical properties were optimized. One of the best compounds, <b>31</b> (CZh226), showed remarkable PAK4 selectivity (346-fold vs PAK1) and favorable kinase selectivity profile. Moreover, this compound potently inhibited the migration and invasion of A549 tumor cells by regulating the PAK4-directed downstream signaling pathways <i>in vitro</i>. Taken together, these data support the further development of <b>31</b> as a lead compound for PAK4-targeted anticancer drug discovery and as a valuable research probe for the further biological investigation of group II PAKs.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75155" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75155" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The p21-activated kinases (PAKs) are serine/threonine (Ser/Thr) protein kinases that have been identified as downstream signaling effectors of Rho-family GTPases.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> The six mammalian PAK isoforms are categorized into two groups: PAK1–3 (group I) and PAK4–6 (group II), based on their structural homologies and biochemical features.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> As key components of the Ras-Rac/Cdc42-PAK pathway, PAKs have pivotal roles in many fundamental cellular processes, including cytoskeletal reorganization, focal adhesion, cell motility, morphological changes, cell-cycle progression, etc.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Moreover, the overexpression, amplification, and mutational activation of PAK isoforms, in particular, PAK1 and PAK4, have been linked to many human diseases, including breast cancer, lung cancer, prostate cancer, colon cancer, and human head and neck squamous cell carcinoma.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Consequently, PAKs have emerged as attractive targets for new anticancer therapies and have been the subject of extensive drug discovery efforts.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">Although the two groups of PAK proteins are similar in overall sequence and structure, they are differentiated by their tissue expression profiles,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> subcellular localization,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> GTPase specificity,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> activation mechanism,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and downstream substrate specificity.<a onclick="showRef(event, 'ref2 ref7'); return false;" href="javascript:void(0);" class="ref ref2 ref7">(2, 7)</a> Studies using knockout mice lacking one or more specific PAK isoforms revealed the role of each isoform in normal tissue development, with phenotypes that range from no apparent effect to early embryonic death.<a onclick="showRef(event, 'ref2 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref2 ref9 ref10">(2, 9, 10)</a> Among all of the PAKs, PAK4 is the most studied group II PAK member, and it has a place at critical nodal points in multiple signaling pathways that are associated with cell growth, cytoskeletal dynamics, cell polarity, survival, and development.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> PAK4 is particularly highly expressed in prostate, testis, lung, heart, brain, and liver.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> It has attracted considerable interest because of its role in cancer invasion, metastasis, and proliferation of BRAF- or KRAS-driven cancers.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> In addition to PAK4, there is emerging evidence for the roles for PAK5 and PAK6 in cancer progression.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Moreover, a recent study revealed that PAK2 inhibition correlates with increased acute cardiovascular toxicity, which may be enhanced by PAK1 inhibition.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Thus, the development of specific and potent PAK4 inhibitors is highly desirable for minimizing the risk of the potential side effects associated with the inhibition of normal function of group I PAKs and will also shed further light on its role in cancer progression.</div><div class="NLM_p">Over the past two decades, several PAK inhibitors have been developed that exhibit different levels of selectivity across the PAK isoforms.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5, 6)</a> On the basis of their binding modes, PAK inhibitors can be categorized into several types (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> PF-3758309 (<b>2</b>),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> a “Pan-PAK inhibitor”, was the first compound advanced to phase I trials (NCT00932126) by Pfizer, targeting both group I and II PAKs. However, further development of <b>2</b> was called to a stop because of unsatisfactory pharmacokinetic data in phase I testing.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Compound <b>4</b><a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and <b>5</b> (GNE-2861),<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> a type I 1/2 kinase inhibitor, show significant group I and group II PAK selectivity, respectively. Most recently, a new dual PAK4/NAMPT modulator <b>8</b> (KPT-9274)<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> advanced to phase I human clinical trials (NCT02702492) for the treatment of solid malignancies or non-Hodgkin’s lymphoma (NHL).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0001.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Classification of representative PAK inhibitors by mechanism of function.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5, 6)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Our research group has focused on the discovery and optimization of PAK4 inhibitors as new therapeutic agents for over a decade,<a onclick="showRef(event, 'ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24">(22-24)</a> and we identified quinazoline-based compound <b>9</b> (LCH-7749944) as a moderate PAK4 inhibitor via structure-based virtual screening in 2012.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Compound <b>9</b> was notable in that it possessed higher potency against PAK4 than PAK1 and significantly inhibited the migration and invasion of human gastric cancer cells in conjunction with the concomitant blockage of the PAK4/LIMK1/cofilin and PAK4/MEK-1/ERK1/2/MMP2 pathways. Herein, we describe our structure-based design efforts toward the discovery of potent group II PAK inhibitors specifically targeting PAK4 and the structure–activity relationship for this series. Our goal for optimization was to improve the group II inhibitory activity of early lead <b>10a</b> while maintaining high selectivity over group I PAKs (PAK1 and PAK4 as representatives for group I and group II PAKs, respectively). The binding mode revealed in the X-ray crystallographic studies of <b>10a</b>, combined with analysis of the residue differences among PAK isoforms, shed light on some of the selectivity determinants within the ATP binding site. The following optimization led to the discovery of compound <b>31</b>, a potent PAK4 inhibitor with substantially lower inhibition for group I PAKs (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0002.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design and modification strategies of novel and selective PAK4 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61342" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61342" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Identification and Characterization of Early Lead <b>10a</b></h3><div class="NLM_p">Inspired by our previous study of the PAK4 inhibitor <b>9</b>,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> we identified compound <b>10a</b> as a moderately potent biochemical inhibitor of PAK4 (PAK4 <i>K</i><sub>i</sub> = 0.710 μM) without measurable binding to PAK1 (PAK1 <i>K</i><sub>i</sub> > 10 μM) from a collection of quinazoline derivatives. For an independent confirmation of activity and to address potential secondary effects in the biochemical assay, such as PAK4 conformation change induced by peptide substrate effect or fluorescent false signal, two direct binding assays were used to identify the binding between <b>10a</b> and the truncated PAK4. Differential scanning fluorimetry (DSF)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> showed a melting temperature shift of 0.5 °C for the compound <b>10a</b>–PAK4 complex compared to the truncated PAK4 protein (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>, Table S4). Surface plasmon resonance (SPR)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> assay resolved the dissociation constant of <b>10a</b> with PAK4 to be 0.156 μM, by multiconcentration kinetic analysis and fitted by 1:1 binding mode. The results demonstrate that compound <b>10a</b> binds directly to the truncated PAK4 kinase domain.</div><div class="NLM_p">To gain insight into the origin of the PAK4 selectivity (PAK1 <i>K</i><sub>i</sub>/PAK4 <i>K</i><sub>i</sub>) displayed by compound <b>10a</b> (>14-fold), the crystal structure of PAK4 kinase domain in complex with compound <b>10a</b> was obtained. The cocrystal structure determined at 2.65 Å resolution (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>, Table S1) clearly shows that PAK4 adopts an active conformation (DFG-in) with a salt bridge (2.3 Å) between Lys350 and Glu366 in helix αC, and <b>10a</b> is located in the ATP binding cleft between the N- and C-lobes (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). The imidazole ring of <b>10a</b> aligns in the hinge region, forming two H-bonds with the main chain of Leu398, and an additional nonclassical H-bond (3.0 Å) is formed between Glu396 and the C2–H of the amino imidazole.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The N-1 methyl group makes effective hydrophobic interactions with Met395, the gatekeeper residue. A favorable electrostatic quinazoline N3 to imidazole C5 interaction was proposed to orient these rings in a coplanar manner. The piperidine NH<sub>2</sub> group makes a hydrogen bond to the backbone carbonyl of Glu329 (glycine rich loop/P-loop) of 2.9 Å. The cocrystal structure confirmed that <b>10a</b> is a typical type I inhibitor that binds to the hinge region without perturbing the DFG-in state. The simplicity of the molecule (MW = 357.5) and its reasonable binding efficiency<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> (LE = 0.31) made it an attractive starting point for further optimization.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of PAK4 in complex with <b>10a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVF">5XVF</a>). Note: Asp458 is oriented toward the NH<sub>2</sub> in the piperidine of <b>10a</b> compared to its counterpart Asp407 in PAK1 as in the structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O0R">4O0R</a> (cyan stick). The distance between the piperidine NH<sub>2</sub> of <b>10a</b> and the carboxyl group (OD2 atom) of Asp458<sup>PAK4</sup> and Asp407<sup>PAK1</sup> is 4.7 and 6.0 Å, respectively (green line). The ∠O<sub>(Asp458)</sub>–N<sub>(piperidine N)</sub>–N<sub>(piperidine NH2)</sub> angle is approximately 51.7° (cyan line).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further improve the binding affinity of compound <b>10a</b>, SAR at the hinge binding motif was explored based on the structural insight provided by the crystal structure above. As shown in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, the role of the N-1 group in imidazole analogues is critical, as cyclopropyl analog <b>10b</b> and isopropyl analog <b>10c</b> exhibited similar inhibitory activity with methyl <b>10a</b>, while the activity of bulky phenyl <b>10d</b> was only half of <b>10a</b>. Pyrazole analogues <b>11a</b> [PAK4 <i>K</i><sub>i</sub> = 0.099 μM, PAK4 <i>K</i><sub>d</sub> (SPR) = 0.043 μM] and <b>11b</b> [PAK4 <i>K</i><sub>i</sub> = 0.016 μM, PAK4 <i>K</i><sub>d</sub> (SPR) = 0.006 μM] were found to exhibit significantly improved biochemical and cellular potency (data not shown) when the donor C–H hydrogen bond to the hinge in <b>10a</b> is replaced by a N–H in <b>11</b>. Such results clearly showed that the nonclassical C–H hydrogen bond is not as good as the classical N–H hydrogen bond in this case, and the change of the hydrogen bond from the nonclassical type to the classical type led to significant improvement. The 1<i>H</i>-pyrazol-3-amine as the hinge binding motif was maintained in the following optimization. Although the pyrazole analogues showed dramatically improved inhibitory activities against PAK4, they unfortunately did not lead to any improvement in their PAK4/1 selectivity relative to <b>10a</b> (<b>11a</b>, 18-fold; <b>11b</b>, 19-fold). To discover PAK4 inhibitors with better selectivity over PAK1, next we sought to exploit the subtle differences between PAK4 and PAK1 crystallographic data available in the PDB database.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of the Initial Optimization of the Hinge Binding Motif</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0015.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0016.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">PAK4 and PAK1 kinase inhibition was determined using a FRET-based Z′-Lyte assay according to the manufacturer’s instructions (Invitrogen, Carlsbad, USA). The <i>K</i><sub>i</sub> values are the average of at least two duplicate experiments. SEM < ±20%.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">PAK1 <i>K</i><sub>i</sub>/ PAK4 <i>K</i><sub>i</sub>, <i>x</i> = fold.</p></div><div class="footnote" id="t1fn3"><sup>Table c</sup><p class="last">The clogP values were calculated by the Qikprop software with default settings (pH = 7.0).</p></div><div class="footnote" id="t1fn4"><sup>Table d</sup><p class="last">van der Waals surface area of polar nitrogen and oxygen atoms. Calculated by the Qikprop software.</p></div><div class="footnote" id="t1fn5"><sup>Table e</sup><p class="last">Ligand efficiency = (−1.4 log <i>K</i><sub>i</sub>)/(<i>n</i> heavy atoms).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></p></div><div class="footnote" id="t1fn6"><sup>Table f</sup><p class="last">Ligand lipophilicity efficiency = (−log <i>K</i><sub>i</sub>) – (cLogP).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></p></div><div class="footnote" id="t1fn7"><sup>Table g</sup><p class="last">ND = not determined.</p></div><div class="footnote" id="t1fn8"><sup>Table h</sup><p class="last">Reported data.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></p></div><div class="footnote" id="t1fn9"><sup>Table i</sup><p class="last">Reported data.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></p></div></div><div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Active-Site Sequences and Structures of Group I and II PAKs: Rational Design of a New Series of PAK4 Inhibitors by Targeting the DFG Motif</h3><div class="NLM_p">Like all other typical protein kinases, the kinase domain of PAKs consists of a N-terminal domain and a C-terminal domain that are connected via a hinge.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> As shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A, the ATP binding pocket is located between the two lobes. The selective inhibition of PAK4 over group I PAKs has proven to be a challenging endeavor, in part because of the high sequence homology within the ATP binding site of the PAKs (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>, Figure S1). To address this challenge, we exploited the three-dimensional structures of PAKs and defined the ATP binding pocket as a set of 40 residues (P1 through P40, starting at the N-terminus, in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A) able to interact with ATP-competitive inhibitors (checked using KLIFS, a structural kinase–ligand interaction database).<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31, 32)</a> The direct comparison of the residues set at the primary sequence level revealed that more than 65% (26/40) of residues are conserved in the respective ATP pockets of PAK1 and PAK4 (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C). We believe that sequence variations and unique active site features among PAKs will provide opportunities for the selective targeting of these family members.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0004.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of the ATP-binding pocket between group I and II PAKs. (A) Structure of the PAK kinase domain (generated from the PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X4Z">2X4Z</a>). Residues contributing to the binding cleft (P1–P40) are depicted as spheres. (B) Alignment (main chain atoms only) of the DFG–aspartate of PAK1 and PAK4 in the active state. An updated survey of all PDB PAK1 structures in the active DFG-in state indicated that 74% (26/35) of the DFG–aspartate side chain is pointing away from the ATP-competitive ligand. For the corresponding residue in PAK4, 71% (22/31) of the DFG–aspartate side chain is pointing toward the ligand. Figures were generated using PyMOL (The PyMOL Molecular Graphics System, Version 1.7.2 Schrödinger, LLC.). The definition of the dihedral angles χ1(chi): N–Cα–Cβ–Cγ. (C) Sequence alignment of the residues surrounding the ATP binding site in PAK1 and PAK4. Differential residues between the two kinases are highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Most protein kinases share a strictly conserved DFG (Asp-Phe-Gly, P38–P40) motif in the ATP site that can adopt two distinct conformations, the active DFG-in and the inactive DFG-out states.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Further alignment of the protein crystal structures available in the Protein Data Bank (PDB) revealed that the orientation of the PAK4 DFG–aspartate (Asp458<sup>PAK4</sup>) side chain is different from the corresponding residue in PAK1 (Asp407<sup>PAK1</sup>).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> As a proof of concept, Crawford et al. proved that taking advantage of differing orientations of the DFG Asp motif serves as a viable means to design Group I selective PAK inhibitors.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> An updated survey of all PDB PAK1 structures in the active DFG-in state indicated that 74% (26/35) of the DFG–aspartate side chain is pointing away from the ATP-competitive ligand, and for the corresponding residue in PAK4, 71% (22/31) of the DFG–aspartate side chain is pointing toward the ligand (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B; see the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details). Presumably, this can be explained by a key residue difference in the DFG-1 residue (P36, Thr<sup>406</sup> in PAK1 and Ser<sup>457</sup> in PAK4): there is potentially an unfavorable interaction of the larger, branched Thr residue with the Asp residue in PAK1. In the active DFG-in conformation, the aspartate is required to chelate Mg<sup>2+</sup> and helps to orient the γ-phosphate for its transfer.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The important biological function of the DFG motif and orientation differences in the DFG Asp (Asp458<sup>PAK4</sup>/Asp407<sup>PAK1</sup>) suggested to us that the modification of the early lead compounds by specifically targeting the DFG motif would enhance PAK4 inhibitory activity and selectivity.</div><div class="NLM_p">Indeed, the PAK4/<b>10a</b> crystal structure showed that the distance from piperidine NH<sub>2</sub> to the side chain of Asp458 is 4.7 Å, and the ∠O<sub>(Asp458)</sub>–N<sub>(piperidine N)</sub>–N<sub>(piperidine NH2)</sub> angle is 51.7° (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>), which could allow the installation of an appropriate moiety to interact with the DFG motif. Furthermore, to generate electrostatic interactions or hydrogen binding with the negatively charged residue Asp458<sup>PAK4</sup> through the quinazoline C2 position, the side chain should adopt an appropriate angle to ensure that it could make favorable interactions with the DFG motif. In light of this, we incorporated a putative DFG binding moiety to the central core of <b>10a</b> by tethering a cationic amino group or hydrogen bond donor (HBD) group through an amide carbonyl linkage in consideration of its sp<sup>2</sup> hybridization. In doing so, a novel 4-((1<i>H</i>-pyrazol-3-yl)-amino)quinazoline-2-carboxamide scaffold was designed by inserting a carbonyl group at the C2 side chain of our initial set of compounds (see <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of Carboxamide Substituents</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0017.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0018.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">PAK4 kinase inhibition was determined using a FRET-based Z′-Lyte assay according to the manufacturer’s instructions (Invitrogen, Carlsbad, USA). The <i>K</i><sub>i</sub> values are the average of at least two duplicate experiments. SEM < ±20%.</p></div><div class="footnote" id="t2fn2"><sup>Table b</sup><p class="last">Racemic mixture.</p></div></div><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Structural Modification and SAR Interpretation</h3><div class="NLM_p">The inhibitory activities of the designed compounds for PAK1 and PAK4 were first evaluated via a well-established fluorescence resonance energy transfer (FRET)-based Z′-Lyte assay (Invitrogen, Carlsbad, USA). Two well characterized PAK inhibitors, <b>1</b><a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and <b>2</b>,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> were used as positive controls.</div><div class="NLM_p">Following the rational design approach above, the incorporation of various tail amide groups led to compounds <b>12</b>–<b>17</b> with diverse activities (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). The 4-amino-piperidines <b>12</b> and <b>14</b> displayed a decrease in PAK4 affinity compared to <b>11a</b>, with <i>K</i><sub>i</sub> values of 0.354 and 0.674 μM, respectively. Meanwhile, alcohol analogues <b>13</b> and <b>15</b> lost an order of magnitude in potency, suggesting that the amino group at the tail position is preferable presumably to interact with the side chain of Asp458<sup>PAK4</sup> via forming a hydrogen bond or ionic interactions. The methylation of the NH<sub>2</sub> as in <b>16</b> resulted in a decrease by half. The overall trend of decreasing activity is probably due to steric hindrance with the introduction of a slightly bulkier functional group at the terminal position of the piperidine ring and, more importantly, the introduction of the carbonyl group between the quinazoline moiety and the piperidine moiety (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>, Figure S2C,D). Such evidence prompted us to try to contract the exo nitrogen atom into an endo one, which led to compound <b>17</b>. As expected, compound <b>17</b> showed dramatically improved binding affinities [PAK4 <i>K</i><sub>i</sub> = 0.045 μM, PAK4 <i>K</i><sub>d</sub> (SPR) = 0.066 μM] over those of <b>12</b>. Compared to the 2,4-diaminoquinazoline analogue <b>11a</b>, the introduction of an amide side chain in compound <b>17</b> was clearly essential in improving its PAK4 selectivity (18-fold vs >222-fold, as shown in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). A molecular docking simulation predicted that the two compounds adopted very similar conformations, and their aromatic rings superimposed very well. The most significant difference between these two inhibitors is the additional interactions between the piperazine moiety and the DFG binding motif of PAK4, which likely provided a charge-assisted hydrogen bond<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (CAHB, traditional hydrogen bonding is accompanied by Coulombic interactions) interaction with the deprotonated carboxylic acid side chain of Asp458<sup>PAK4</sup> (2.3 Å, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A). To obtain more insight into the interactions critical to the excellent activity of compound <b>17</b>, compounds <b>18</b>–<b>20</b> were designed, synthesized, and evaluated. Unsurprisingly, the importance of this electrostatic interaction is elucidated by the fact that compound <b>18</b> [PAK4 <i>K</i><sub>i</sub> = 2.172 μM, PAK4 <i>K</i><sub>d</sub> (SPR) = 1.940 μM], in which the positively charged nitrogen atom was replaced with an oxygen atom, is a very poor inhibitor (<a class="ref internalNav" href="#notes-1" aria-label="Figure S2E,F">Figure S2E,F</a>). Compound <b>19</b>, an <i>N</i>-methylated derivative of <b>17</b>, was 17-fold less active, highlighting the importance of the amine proton acting as a hydrogen bond donor. When the amine proton was replaced by an amide proton (<b>20</b>, PAK4 <i>K</i><sub>i</sub> = 2.408 μM), the activity was decreased by 54-fold (<a class="ref internalNav" href="#notes-1" aria-label="Figure S2G,H">Figure S2G,H</a>). The dramatic decrease in activity could be attributed to the fact that the amide group could not form salt bridge interactions with the side chain of Asp458<sup>PAK4</sup>. Furthermore, changing the piperazine ring to other heterocyclic secondary amines was not tolerable, as indicated by the activities of compounds <b>21</b>–<b>24</b>. This indicated that the conformation of the amide side chain is critical to the activity of the compounds. The cyclopropyl analog <b>25</b> [PAK4 <i>K</i><sub>i</sub> = 0.016 μM, PAK4 <i>K</i><sub>d</sub> (SPR) = 0.016 μM, PAK1 <i>K</i><sub>i</sub> = 2.750 μM] showed improved activity, but its selectivity was not improved (172-fold, shown in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). In summary, an additional hydrogen bond conjoins with strong salt bridge interactions, and the correct orientation of the piperazine ring likely explains the enhanced activity and selectivity of <b>17</b> and <b>25</b>.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0005.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Predicted binding mode of <b>17</b>. (A) Superimposition of structures of <b>17</b> (green) with <b>10a</b> from the crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVF">5XVF</a>). PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X4Z">2X4Z</a> was used for molecular docking simulation with Autodock4. Note: Asp458 in PAK4 overlaid with a key residue different in PAK1 (cyan, Asp407 from structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O0R">4O0R</a>). (B) Overlay of PAK4 and PAK1 with <b>17</b> (generated from the PDB structures <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X4Z">2X4Z</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O0R">4O0R</a>, respectively).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. <i>In Vitro</i> Biological and Physical Properties of Piperazine Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0019.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0020.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">PAK4 and PAK1 kinase inhibition was determined using a FRET-based Z′-Lyte assay according to the manufacturer’s instructions (Invitrogen, Carlsbad, USA). The <i>K</i><sub>i</sub> values are the average of at least two duplicate experiments. SEM < ±20%.</p></div><div class="footnote" id="t3fn2"><sup>Table b</sup><p class="last">PAK1 <i>K</i><sub>i</sub>/PAK4 <i>K</i><sub>i</sub>, <i>x</i> = fold.</p></div><div class="footnote" id="t3fn3"><sup>Table c</sup><p class="last">The clogP values were calculated by the Qikprop software with default settings (pH = 7.0).</p></div><div class="footnote" id="t3fn4"><sup>Table d</sup><p class="last">The p<i>K</i><sub>a</sub> values were calculated using ChemAxon’s MarvinSketch (version 5.4.11).</p></div><div class="footnote" id="t3fn5"><sup>Table e</sup><p class="last">The van der Waals surface area of polar nitrogen and oxygen atoms. Calculated by the Qikprop software.</p></div><div class="footnote" id="t3fn6"><sup>Table f</sup><p class="last">Ligand efficiency = (−1.4 log <i>K</i><sub>i</sub>)/(<i>n</i> heavy atoms).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></p></div><div class="footnote" id="t3fn7"><sup>Table g</sup><p class="last">Ligand lipophilicity efficiency = (−log <i>K</i><sub>i</sub>) – (cLogP).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></p></div><div class="footnote" id="t3fn8"><sup>Table h</sup><p class="last">ND = not determined.</p></div><div class="footnote" id="t3fn9"><sup>Table i</sup><p class="last">Mixture of diastereomers.</p></div></div><div></div></div><div class="NLM_p">Beginning with the new scaffold of compounds <b>17</b> and <b>25</b>, further structural optimization was undertaken to enhance the binding affinity of the inhibitors by increasing the spatial occupancy of the kinase ATP pocket. Meanwhile, the “floor pocket” on the bottom of the ATP binding pocket near the DFG motif and the “upper pocket” beneath the glycine-rich loop (P-loop) also captured our attention (shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B). In contrast, PAK1 has a much wider and shallower “floor pocket”, and PAK4 has a deeper and overall smaller subpocket. This difference in size was attributed to the distinct conformations of DFG Asp, as mentioned above, and key residues differences inside the subpocket (Ser457<sup>PAK4</sup> vs Thr406<sup>PAK1</sup>; Ser445<sup>PAK4</sup> vs Asn394<sup>PAK1</sup>). Because of the subtle conformation differences in the P-loop (shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B), the specific features of the “upper pocket” in different PAK isoforms were also observed. It was apparent from the modeling structures that these hydrophobic pockets are located around the piperazine ring of <b>17</b> and could be accessed via an axial substituent on the piperazine ring. We hypothesized that an improvement in the binding affinity and selectivity might be achieved by targeting these subpockets. <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a> shows our efforts at potency optimization through piperazine substitution. The analogues with the methyl group positioned next to the amide nitrogen (compounds <b>26</b>–<b>29</b>) were proven unsuccessful at improving potency and selectivity. Among them, the compounds with a (<i>R</i>)-configuration at the methyl stereocenter (<b>28</b>, <b>29</b>) exhibited better potency and selectivity than their corresponding (<i>S</i>)-analogues (<b>26</b>, <b>27</b>). A significant improvement in PAK4 selectivity was obtained by incorporating a (<i>R</i>)-methyl group next to the amine nitrogen on the piperazine ring (<b>30</b>, <b>31</b>), which most likely contributes to their shape complementarity to maximize van der Waals contacts. In the end, compound <b>31</b> had robust PAK4 affinity (PAK4 <i>K</i><sub>i</sub> = 0.009 μM) and excellent selectivity at 346-fold over PAK1 (PAK1 <i>K</i><sub>i</sub> = 3.112 μM), comparable to that of <b>5</b> (PAK1 <i>K</i><sub>i</sub> = 2.9 μM<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a>). Interestingly, the (<i>S</i>)-methyl substituent (<b>32</b>, <b>33</b>) was not tolerated, suggesting that chirality is a crucial element that influences van der Waals interactions. The bulky (<i>R</i>)-ethyl analogue, <b>36</b> (PAK4 <i>K</i><sub>i</sub> = 0.006 μM, PAK1 <i>K</i><sub>i</sub> = 0.368 μM), displayed similar potency against PAK4 in the single digit nanomolar range, but it had little selectivity against PAK4 (61-fold). Further substitution of the piperazine ring, however, resulted in a significant decrease in activity (dimethyl analogues <b>34</b> and <b>35</b>).</div><div class="NLM_p">To better understand the origin of the excellent PAK4 selectivity, the crystal structures of the PAK4 kinase domain in complex with <b>31</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVG">5XVG</a>) and its highly similar methyl analogue <b>30</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVA">5XVA</a>) were solved to 2.10 and 1.85 Å, respectively (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> Table S1). The binding mode observed for <b>31</b> was consistent with that observed with <b>30</b> (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>), and the key interactions are shown in <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>. The aminopyrazole of <b>31</b> forms classic donor–acceptor–donor H-bond interactions with the hinge region (carbonyl of Glu396<sup>PAK4</sup>; carbonyl and NH of Leu398<sup>PAK4</sup>), which is similar to that observed in the crystal structure of PAK4 in complex with <b>10a</b>. The cyclopropyl moiety makes effective hydrophobic interactions with Met395<sup>PAK4</sup>, the gatekeeper residue, and Val335<sup>PAK4</sup>. Consistent with our hypothesis, the NH on the piperazine ring forms a charge-assisted hydrogen bond with the negatively charged residue Asp458<sup>PAK4</sup> through the amide carbonyl linkage, with a distance of 2.8 Å. It is noteworthy that the carboxylic side chain of Asp444<sup>PAK4</sup> induces conformational changes to form electrostatic interactions with the tail of <b>31</b> (3.4 Å). In comparison with the putative structure of PAK1/<b>31</b>, the side chain of Asp407<sup>PAK1</sup> is pointing away from compound <b>31</b> and hence not able to form electrostatic interactions with the piperazine ring (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B). Additionally, the piperazine ring of <b>31</b> makes lipophilic contact with the “floor pocket” of PAK4 mentioned above, and the (<i>R</i>)-methyl group of the piperazine ring fills a hydrophobic dimple in the glycine rich loop above, which also explains why an additional (<i>R</i>)-methyl group increased the binding affinity and selectivity for PAK4. The differences in the binding modes for <b>31</b> to PAK4 and PAK1 are also directly linked to the differences in docking scores (X-score:<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> −8.95 for PAK4 and −6.75 for PAK1) and to the experimentally observed selectivity. The beneficial van der Waals interactions resulting from shape complementarity as well as the anticipated CAHB with Asp458<sup>PAK4</sup> contribute to the increased binding affinity of <b>31</b>, demonstrating the rationale of our optimization hypothesis, thereby providing a molecular explanation for the subtype selectivity.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0006.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Crystal structure of PAK4 in complex with <b>30</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVA">5XVA</a>). Superimposition of the structures of <b>30</b> (green) with <b>10a</b> (cyan) from the crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVF">5XVF</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0007.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) Crystal structure of PAK4 in complex with <b>31</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVG">5XVG</a>). (B) Predicted binding mode of <b>31</b> in PAK1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O0R">4O0R</a>). A schematic representation of the contacts between <b>31</b> and the binding site residues of PAK4 (C) and PAK1 (D) was visualized by LigandScout.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Note: the piperazine ring of <b>31</b> provides charge-assisted hydrogen bond (CAHB) interactions with Asp458<sup>PAK4</sup> and Asp444<sup>PAK4</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Lastly, the role of the substituent at the C-6 position of the quinazoline core was investigated, and the data are summarized in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>. As shown in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>, the 6-chloro group lies in the cliff-like entrance around the hinge area and leads to van der Waals interactions with Phe397<sup>PAK4</sup>, Glu399<sup>PAK4</sup>, and Ile327<sup>PAK4</sup> (P24, P26, and P1 in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B, respectively). To probe this distinctive entrance region, the removal of the chloro group (compound <b>37</b>, PAK4 <i>K</i><sub>i</sub> = 0.017 μM) caused a slight decrease in the potency for PAK4 and a significant reduction in PAK4/1 selectivity (<b>37</b>, PAK4 selectivity: 63-fold). In addition, the position of the chloro group was also found to affect the selectivity profile. Although the 7-chloro derivative <b>38</b> (PAK4 <i>K</i><sub>i</sub> = 0.006 μM) displayed better potency than that of <b>31</b>, it suffered from reduced PAK4 selectivity (<b>38</b>, PAK4 selectivity: 57-fold). Further investigation suggested that the 6-chloro group in <b>31</b> could be replaced by electron-withdrawing substituents, such as F (<b>39</b>) and Br (<b>40</b>), and still retain its strong PAK4 affinity, but the PAK4 selectivity was reduced by variable degrees (PAK4 selectivity: 31- and 190-fold, respectively). The compound bearing a large electron-donating group, such as a methoxyl group (<b>41</b>), suffered from reduced potency for both PAK1 and PAK4 (PAK4 <i>K</i><sub>i</sub> = 0.036 μM; PAK1 <i>K</i><sub>i</sub> > 4.52 μM). These key findings highlighted the importance of both shape and electrostatic complementarity at the 6-position of the quinazoline core because the residues around the 6-position between PAK4 and PAK1 are not conserved (Phe397<sup>PAK4</sup> vs Tyr346<sup>PAK1</sup>; Glu399<sup>PAK4</sup> vs Ala348<sup>PAK1</sup>, <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>). It may be possible to further exploit such sequence differences for the future design of inhibitors that distinguish PAK4 and PAK1 to a larger extent.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0008.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Microenvironment of the 6-substituent of the inhibitors. The X-ray structure of <b>31</b> in complex with PAK4 (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVG">5XVG</a>) overlaid with key residue differences with PAK1 (cyan, Tyr346 and Ala348 from structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O0R">4O0R</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of the Quinazoline Core</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0021.gif" alt="" id="fx7" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Compound</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center">PAK4<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a><i>K</i><sub>i</sub> (<i>μ</i>M)</th><th class="colsep0 rowsep0" align="center">PAK1<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a><i>K</i><sub>i</sub> (<i>μ</i>M)</th><th class="colsep0 rowsep0" align="center">PAK4 select. index<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">clogP<a class="ref internalNav" href="#t4fn4" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">p<i>K</i><sub>a</sub><a class="ref internalNav" href="#t4fn5" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">PSA<a class="ref internalNav" href="#t4fn6" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">LE<a class="ref internalNav" href="#t4fn7" aria-label="f">f</a>/LLE<a class="ref internalNav" href="#t4fn8" aria-label="g">g</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">0.009</td><td class="colsep0 rowsep0" align="left">3.112</td><td class="colsep0 rowsep0" align="left">346×</td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">7.9</td><td class="colsep0 rowsep0" align="left">96.2</td><td class="colsep0 rowsep0" align="left">0.39/5.49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">0.017</td><td class="colsep0 rowsep0" align="left">1.072</td><td class="colsep0 rowsep0" align="left">63×</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">7.9</td><td class="colsep0 rowsep0" align="left">95.6</td><td class="colsep0 rowsep0" align="left">0.39/5.69</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">0.007</td><td class="colsep0 rowsep0" align="left">0.401</td><td class="colsep0 rowsep0" align="left">57×</td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">7.9</td><td class="colsep0 rowsep0" align="left">95.6</td><td class="colsep0 rowsep0" align="left">0.39/5.60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">0.016</td><td class="colsep0 rowsep0" align="left">0.503</td><td class="colsep0 rowsep0" align="left">31×</td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">7.9</td><td class="colsep0 rowsep0" align="left">96.2</td><td class="colsep0 rowsep0" align="left">0.38/5.48</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">0.011</td><td class="colsep0 rowsep0" align="left">2.085</td><td class="colsep0 rowsep0" align="left">190×</td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">7.9</td><td class="colsep0 rowsep0" align="left">95.6</td><td class="colsep0 rowsep0" align="left">0.38/5.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">0.036</td><td class="colsep0 rowsep0" align="left">>4.52</td><td class="colsep0 rowsep0" align="left">>126×</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">7.9</td><td class="colsep0 rowsep0" align="left">104.6</td><td class="colsep0 rowsep0" align="left">0.35/5.26</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">PAK4 and PAK1 kinase inhibition was determined using a FRET-based Z′-Lyte assay according to the manufacturer’s instructions (Invitrogen, Carlsbad, USA). The <i>K</i><sub>i</sub> values are the average of at least two duplicate experiments. SEM < ±20%.</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last">PAK1 <i>K</i><sub>i</sub>/PAK4 <i>K</i><sub>i</sub>, <i>x</i> = fold.</p></div><div class="footnote" id="t4fn4"><sup>c</sup><p class="last">The clogP values were calculated using the Qikprop software with the default settings (pH = 7.0).</p></div><div class="footnote" id="t4fn5"><sup>d</sup><p class="last">The p<i>K</i><sub>a</sub> values were calculated using ChemAxon’s MarvinSketch (version 5.4.11).</p></div><div class="footnote" id="t4fn6"><sup>e</sup><p class="last">The van der Waals surface area of polar nitrogen and oxygen atoms. Calculated using the Qikprop software.</p></div><div class="footnote" id="t4fn7"><sup>f</sup><p class="last">Ligand efficiency = (−1.4 log <i>K</i><sub>i</sub>)/(<i>n</i> heavy atoms).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></p></div><div class="footnote" id="t4fn8"><sup>g</sup><p class="last">Ligand lipophilicity efficiency = (−log <i>K</i><sub>i</sub>) – (cLogP).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></p></div></div></div><div class="NLM_p">After several rounds of lead optimization aided by X-ray crystallography and molecular docking simulations, several compounds were identified with significant inhibitory potency against PAK4 with <i>K</i><sub>i</sub> values higher than or compatible to the reference compound <b>2</b>. Over the course of the optimization, a high degree of ligand efficiency was maintained, with LE ≥ 0.38 and LLE ≥ 5.3 for the majority of sub-20 nM <i>K</i><sub>i</sub> compounds. Among them, compound <b>31</b> demonstrated the best balance of potency, selectivity, and desirable physicochemical properties (p<i>K</i><sub>a</sub> = 7.93, clogP = 2.54, PSA = 96.1). Encouraged by the potency of <b>31</b> and its selectivity over PAK1, its kinome selectivity profile was investigated by determining % inhibition at 0.1 and 1.0 μM (concentration equal to 11-fold and 110-fold above PAK4 <i>K</i><sub>i</sub>, respectively) against a panel of 54 kinases at the ATP <i>K</i><sub>m</sub> concentration, and the full data set is available in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>. Good kinase selectivity was observed for <b>31</b>, with only PAK4 producing greater than 80% inhibition at the screening concentration of 0.1 μM (PAK4 IC<sub>50</sub> = 0.0111 μM, <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>). These data demonstrated that compound <b>31</b> is a potent PAK4 inhibitor with good kinase spectrum selectivity. Although the DFG motif is highly conserved among protein kinases, the proper spatial arrangement of <b>31</b> with the DFG motif of PAK4 in combination with other residue differences within the ATP binding site would likely explains the selectivity of <b>31</b>.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0009.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Selectivity profile of compound <b>31</b> measured at a concentration of (A) 0.1 μM and (B) 1.0 μM in a panel of 54 kinases generated with the SelectScreen Profiling Service from Life Technologies (red columns denote >80%, yellow columns between 40 and 80%, and green columns <40% inhibition).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Cellular Effects of Compound <b>31</b></h3><div class="NLM_p">Previous studies have shown that PAK4 is overexpressed in several nonsmall cell lung cancer (NSCLC) cell lines and human NSCLC tissues.<a onclick="showRef(event, 'ref12 ref38'); return false;" href="javascript:void(0);" class="ref ref12 ref38">(12, 38)</a> Increased expression of PAK4 was correlated with metastasis, shorter overall survival, and advanced-stage NSCLC. Furthermore, selective suppression of PAK4 with siRNA has been shown to substantially suppress the migration and invasion of the PAK4-dependent A549 cells (human lung adenocarcinoma epithelial cell).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Considering the potent PAK4 inhibition of compound <b>31</b>, we first investigated its antiproliferative effects on A549 cells. Meanwhile, the tumor cell line NCI-H460 (human large cell lung cancer cell), whose growth was not dependent on PAK4, and HEK-293 cells (human embryonic kidney 293 cell) were used to test the potential off-target effects of potent PAK4 inhibitors. It was shown that compound <b>31</b> has moderate antiproliferative activities against A549 cells (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>A), which is similar to the previous observation of <b>5</b> on MDA-MB-436 and MCF10A PIK3CA cells.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The relatively low cell growth inhibitory activities suggested that PAK4 might not be the “driving force” for proliferation of these cancer cells. Next, the effects of compound <b>31</b> on tumor cell migration and invasion were assessed using wound healing and transwell assays. Microphotographs showed that untreated A549 cells filled most of the wounded area 2 days after scratching the cell monolayer, whereas treatment with indicated doses of compound <b>31</b> significantly suppressed the wound healing (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>B). Compound <b>31</b> treatment also significantly inhibited the A549 cell migration (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>C,E) and invasion (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>D,F) by transwell assay. In contrast, the decrease in cell migration and invasion was not pronounced in NCI-H460 and HEK-293 cells (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>G–J), suggesting that inhibition of cell motility by compound <b>31</b> is in fact related to PAK4 inhibition. To further illustrate the mechanism, the effects of compound <b>31</b> on the PAK4/LIMK1/cofilin, PAK4/MMP2, or PAK4/GEF-H1 signaling pathways were examined by Western blot analysis. As shown in <a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>K, phosphorylation of PAK4, LIMK1, and cofilin were inhibited by compound <b>31</b> treatment in a dose-dependent manner. Furthermore, phosphorylation of Ser810 on GEF-H1 and expression of MMP-2 were also inhibited by compound <b>31</b>, which may contribute to its antimetastasis effect of A549 lung cancer cells.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0010.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Compound <b>31</b> suppresses the migratory and invasive potential of A549 lung cancer cells via inhibition of PAK4 kinase activity and its downstream pathways. (A) MTT assay showed the antiproliferative effect of compound <b>31</b> on A549, HEK-293, and NCI-H460 cells. (B) Compound <b>31</b> inhibits migration of A549 lung cancer cells in a wound healing assay. (C–F) Transwell assay [−(D)/+(C) matrigel] was performed to show the effect of indicated concentrations of <b>31</b> on A549 lung cancer cell migration (C) and invasion (D), and migrated cells were counted, respectively (E,F). The results are presented as the mean ± standard deviation (*, <i>p</i> < 0.05; **, <i>p</i> < 0.01; ***, <i>p</i> < 0.001 vs control). (G,H) Effect of <b>31</b> on HEK-293 cell migration (G) and invasion (H). (I,J) Effect of <b>31</b> on NCI-H460 cell migration (I) and invasion (J). Scale bar, 500 μm. (K) Western blot assay showed the effect of compound <b>31</b> on the phospho-PAK4 Ser<sup>474</sup> level and on the phosphorylation levels of PAK4 downstream targets LIMK1 and cofilin. The expression of MMP-2 and the phosphorylation of GEF-H1 were also downregulated by compound <b>31</b> treatment in a dose-dependent manner. A549 cells were cultured with vehicle or the indicated concentrations of compound <b>31</b> for 24 h, and then proteins were extracted and subjected to analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> <i>In Vitro</i> Metabolic Stability, CYP450 Inhibition, hERG Inhibition, and <i>in Vivo</i> Pharmacokinetics Properties</h3><div class="NLM_p">A preliminary ADMET properties assessment of <b>31</b> was conducted to evaluate its potential for further development. The <i>in vitro</i> metabolic stability of the compound was assessed in liver microsomes (rat and human) and rat blood plasma. As shown in <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>, compound <b>31</b> possessed moderate microsomal stability and excellent blood plasma stability (for more information, see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). Further CYP450 inhibition determination showed that <b>31</b> possessed favorable metabolic properties, as the inhibition ratios for the five main CYPs (CYP1A2, 2C9, 2C19, 2D6, and 3A4) were less than 20% at 10 μM. Previous studies demonstrated that a basic nitrogen (or generally the positive charge feature) would lead to greater activity against human ether-a-go-go-related gene (hERG) channels (Kv11.1).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Gratifyingly, the p<i>K</i><sub>a</sub> of compound <b>31</b> (calculated p<i>K</i><sub>a</sub> = 7.9) was reduced by greater than two units than the early lead (calculated p<i>K</i><sub>a</sub> = 10.0 for <b>10a</b>) and was found to be safe (IC<sub>50</sub> > 40 μM) when tested in a hERG patch clamp assay for assessing hERG-associated cardiotoxicity. On the basis of these favorable <i>in vitro</i> profiles and excellent aqueous solubility (>10 mg/mL at pH 7.4 and pH 2.0), preliminary pharmacokinetic studies of <b>31</b> were performed by administering Sprague–Dawley (SD) rats a 2 mg/kg intravenous (iv) dose of the compound. After the intravenous administration at a dose of 2 mg/kg, <b>31</b> exhibited a moderate half-life of 1.46 ± 0.07 h and achieved a maximum concentration (<i>C</i><sub>max</sub>) of 736 ± 170 ng/mL. The area under the curve (AUC<sub>(0–∞)</sub>) was 472 ± 23.4 ng·h/mL. It is worth noting that compound <b>31</b> exhibited relatively high plasma clearance (70.7 ± 3.53 mL/min/kg), which may be due to its high polarity. Further in-depth studies are under way to elucidate the mechanism of action and assess the therapeutic potential of <b>31</b> as a new anticancer agent, and the results will be reported in due course.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Key Data for Compound <b>31</b> (Corresponding Hydrochloride Salt)</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="6" align="center">physiochemical properties</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">LogP</th><th class="colsep0 rowsep0" align="center">LogD<sub>7.4</sub></th><th class="colsep0 rowsep0" align="center">p<i>K</i><sub>a</sub></th><th class="colsep0 rowsep0" align="center">DMSO solubility</th><th class="colsep0 rowsep0" align="center">aqueous solubility at pH 7.4</th><th class="colsep0 rowsep0" align="center">aqueous solubility at pH 2.0</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2.54<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a> (2.6<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a>)</td><td class="colsep0 rowsep0" align="left">1.90<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">7.93<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a> (7.9<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a>)</td><td class="colsep0 rowsep0" align="left">>30 mM</td><td class="colsep0 rowsep0" align="left">>10 mg/mL</td><td class="colsep0 rowsep0" align="left">>10 mg/mL</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="6" align="center">liver microsomal stability</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameters</th><th class="colsep0 rowsep0" align="center"><i>T</i><sub>1/2</sub> (min)</th><th class="colsep0 rowsep0" align="center">remaining (%) <i>T</i> = 1 h</th><th class="colsep0 rowsep0" align="center">remaining (%)<a class="ref internalNav" href="#t5fn4" aria-label="d">d</a><i>T</i> = 1 h</th><th class="colsep0 rowsep0" align="center">CL<sub>int(mic)</sub>(μL/min/mg)</th><th class="colsep0 rowsep0" align="center">CL<sub>int(liver)</sub>(mL/min/kg)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">in rat</td><td class="colsep0 rowsep0" align="left">41.8</td><td class="colsep0 rowsep0" align="left">36.9</td><td class="colsep0 rowsep0" align="left">97.0</td><td class="colsep0 rowsep0" align="left">33.2</td><td class="colsep0 rowsep0" align="left">59.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">in human</td><td class="colsep0 rowsep0" align="left">48.4</td><td class="colsep0 rowsep0" align="left">41.9</td><td class="colsep0 rowsep0" align="left">102.7</td><td class="colsep0 rowsep0" align="left">28.7</td><td class="colsep0 rowsep0" align="left">25.8</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="6" align="center">cytochrome P450 inhibition<a class="ref internalNav" href="#t5fn5" aria-label="e">e</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">isozyme</th><th class="colsep0 rowsep0" align="center">CYP1A2</th><th class="colsep0 rowsep0" align="center">CYP2C9</th><th class="colsep0 rowsep0" align="center">CYP2C19</th><th class="colsep0 rowsep0" align="center">CYP2D6</th><th class="colsep0 rowsep0" align="center">CYP3A4</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">% inhibition at 10 μM</td><td class="colsep0 rowsep0" align="left">17.2</td><td class="colsep0 rowsep0" align="left">15.1</td><td class="colsep0 rowsep0" align="left">12.2</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">2.3</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="5" align="center">stability in rat blood plasma</th><th class="rowsep1 colsep0" colspan="2" align="center">hERG inhibition<a class="ref internalNav" href="#t5fn6" aria-label="f">f</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameters</th><th class="colsep0 rowsep0" align="center">time point (min)</th><th class="colsep0 rowsep0" align="center">% remaining</th><th class="colsep0 rowsep0" align="center">time point (min)</th><th class="colsep0 rowsep0" align="center">% remaining</th><th class="colsep0 rowsep0" align="center">parameters</th><th class="colsep0 rowsep0" align="center">42.0% inhibition at 40 μM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">compd <b>31</b></td><td class="colsep0 rowsep0" align="left">60</td><td class="colsep0 rowsep0" align="left">110.1</td><td class="colsep0 rowsep0" align="left">120</td><td class="colsep0 rowsep0" align="left">115.3</td><td class="colsep0 rowsep0" align="left">compd <b>31</b></td><td class="colsep0 rowsep0" align="left">IC<sub>50</sub> > 40 μM</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="7" align="center">pharmacokinetic parameters (iv administration in SD Rats)<a class="ref internalNav" href="#t5fn7" aria-label="g">g</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">dose (mg/kg)</th><th class="colsep0 rowsep0" align="center"><i>C</i><sub>0</sub> (ng/mL)</th><th class="colsep0 rowsep0" align="center">Cl (mL/min/kg)</th><th class="colsep0 rowsep0" align="center">Vss (L/kg)</th><th class="colsep0 rowsep0" align="center"><i>T</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center">AUC<sub>(0–t)</sub> (ng·h/mL)</th><th class="colsep0 rowsep0" align="center">AUC <sub>(0–∞)</sub> (ng·h/mL)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2 mg/kg</td><td class="colsep0 rowsep0" align="left">736 ± 170</td><td class="colsep0 rowsep0" align="left">70.7 ± 3.53</td><td class="colsep0 rowsep0" align="left">6.43 ± 0.68</td><td class="colsep0 rowsep0" align="left">1.46 ± 0.07</td><td class="colsep0 rowsep0" align="left">465 ± 21.7</td><td class="colsep0 rowsep0" align="left">472 ± 23.4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">Experimental data.</p></div><div class="footnote" id="t5fn2"><sup>b</sup><p class="last">The clogP values were calculated using the Qikprop software with default settings (pH = 7.0).</p></div><div class="footnote" id="t5fn3"><sup>c</sup><p class="last">Calculated using the Qikprop software.</p></div><div class="footnote" id="t5fn4"><sup>d</sup><p class="last">No NADPH regenerating system was added to the sample (replaced by buffer) during the 60 min incubation.</p></div><div class="footnote" id="t5fn5"><sup>e</sup><p class="last">Performed at 10 μM concentration. α-Naphthoflavone (CYP1A2), sulfaphenazole (CYP2C9), ticlopidine (CYP2C19), quinidine (CYP2D6), and ketoconazole (CYP3A4) were used as the positive controls.</p></div><div class="footnote" id="t5fn6"><sup>f</sup><p class="last">Measured in hERG-expressing CHO cells using a Qpatch 16× assay.</p></div><div class="footnote" id="t5fn7"><sup>g</sup><p class="last">The data are the averages of three independent determinations and reported as the mean ± SD (standard deviation).</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65233" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65233" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The synthesis of 4-aminoimidazole analogs was carried out as shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Briefly, <i>N</i>1-alkylation of commercially available nitro-imidazole <b>42</b> with the appropriate alkyl halides (methyl iodide or 2-bromopropane) proceeded smoothly followed by palladium-mediated hydrogenation to yield the amino imidazole hydrochloride salts <b>44a</b> and <b>44c</b> in good yields.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Meanwhile, for analogues <b>44b</b> and <b>44d</b>, an alternate synthetic route was used. 1,4-Dinitro-1<i>H</i>-imidazole was obtained by the nitration of <b>42</b> following a known general procedure.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The treatment of 1,4-dinitroimidazole <b>46</b> with primary amines afforded the <i>N</i>1-substituted 4-nitroimidazole <b>47</b>, followed by hydrogenation to provide <b>44b</b> and <b>44d</b>.</div><figure id="sch1" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0011.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of 4-Aminoimidazole Analogs<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) CH<sub>3</sub>I, K<sub>2</sub>CO<sub>3</sub>, MeCN, 65 °C, 12 h; (b) (i) H<sub>2</sub>, Pd/C, rt, 8 h; (ii) HCl–EtOH, 0 °C, 1 h; (c) <i>i-</i>PrBr, DMF, K<sub>2</sub>CO<sub>3</sub>, 65 °C, 12 h; (d) HNO<sub>3</sub>, Ac<sub>2</sub>O, H<sub>2</sub>SO<sub>4</sub>, 0 °C to rt, 1 h; (e) RNH<sub>2</sub>, MeOH, H<sub>2</sub>O, 24 h.</p></p></figure><div class="NLM_p">Compounds <b>10a</b>–<b>d</b> and <b>11a</b>–<b>d</b> containing a scaffold of 6-chloroquinazoline-2,4-diamine were synthesized as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The synthetic sequence started from 2-amino-5-chlorobenzoic acid. Intermediate <b>49</b> was obtained via cyclization with urea, which was subjected to chlorination with phosphorus oxychloride in DME to yield the key intermediate <b>50</b>. The subsequent condensation of <b>50</b> with imidazoles <b>44a</b>–<b>d</b> in the presence of <i>N</i>,<i>N</i>-diisopropylethylamine produced the corresponding product <b>51</b>. The nucleophilic aromatic substitution of intermediates <b>51a</b>–<b>d</b> with 4-(<i>N</i>-Boc-amino)-piperidine was achieved by heating under acidic conditions to provide compounds <b>52a</b>–<b>d</b>. The deprotection of <b>52a</b>–<b>d</b> under standard conditions produced the desired products <b>10a</b>–<b>d</b> in good yield. Compounds <b>11a</b>–<b>d</b> were prepared by a method similar to that for compound <b>10</b> in similar overall yields.</div><figure id="sch2" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0012.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Syntheses of 2,4-Diaminoquinazoline Derivatives (<b>10a</b>–<b>d</b> and <b>11a</b>–<b>d</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) urea, 200 °C; (b) POCl<sub>3</sub>, <i>N</i>,<i>N</i>-dimethylaniline, reflux; (c) DIEA, DMF, 0 °C, 3 h; (d) 4-(<i>N</i>-Boc-amino)-piperidine, HCl, EtOH, 120 °C, 15 h; (e) HCl–EA, DCM, rt, 12 h.</p></p></figure><div class="NLM_p">To test the structural-based hypotheses for improving PAK4 selectivity, a novel synthetic strategy was applied to rapidly generate the desired 4-aminoquinazoline-2-carboxamide derivatives (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). 2-Amino-5-chlorobenzoic acid reacted with triphosgene and ammonium hydroxide to produce compound <b>56</b>, which was acylated by ethyl oxalyl monochloride to afford compound <b>57</b>. The ethyl ester <b>57</b> was cyclized under basic conditions to give <b>58</b> in 86% yield, which was hydrolyzed in an ethanol/water solution (1:1) with an excess of NaOH (8 equiv), affording the key intermediate <b>59</b>. The acid <b>59</b> was reacted with SOCl<sub>2</sub> to give the dichlorinated product <b>60</b>, which could be condensed with various amines to the corresponding amide intermediates (<b>61</b>, <b>63</b>, and <b>65</b>) in moderate overall yields. The optimal condition was found to be 1.0 equiv of amine in anhydrous dichloromethane, stirring at −35 °C for 0.5 h. Then, the amides were converted into the intermediates by the replacement of 4-Cl in the quinazoline with the appropriate 1<i>H</i>-pyrazol-3-amines in a good yield, which were deprotected under acidic conditions to produce the final compounds (for <b>13</b>, <b>15</b>, <b>18</b>, <b>19</b>, and <b>20</b>, no deprotection was needed).</div><figure id="sch3" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0013.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Syntheses of 4-Aminoquinazoline-2-carboxamide Derivatives (<b>12</b>–<b>36</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) bis(trichloromethyl) carbonate, THF, reflux, 16 h; (ii) 1 N NH<sub>3</sub>–H<sub>2</sub>O, 65 °C, 1 h; (b) ethyl oxalyl monochloride, THF, TEA, 0 °C to rt., 3 h; (c) EtONa, EtOH, 0 °C, 3 h; (d) NaOH, H<sub>2</sub>O, EtOH, rt., 1 h; (e) SOCl<sub>2</sub>, chloroform, DMF, reflux, 3 h; (f) amine, DCM, TEA, −35 °C, 0.5 h; (g) 3-amino-5-methylpyrazole or 5-cyclopropyl-1<i>H</i>-pyrazol-3-amine, DIEA, KI, DMF, 65 °C, 8 h; (h) HCl–EA, DCM, rt, 6 h.</p></p></figure><div class="NLM_p">Compounds <b>37</b>–<b>41</b> were synthesized in a similar manner starting from the corresponding acid materials, <b>67a</b>–<b>e</b>, in a moderate overall yield (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). All compounds were characterized by <sup>1</sup>H NMR, <sup>13</sup>C NMR, and HRMS, and their purity was determined by reverse phase HPLC (see the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details).</div><figure id="sch4" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0014.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Syntheses of Compounds <b>37</b>–<b>41</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) SOCl<sub>2</sub>, chloroform, DMF, reflux, 3 h; (b) <i>tert</i>-butyl (<i>R</i>)-2-methylpiperazine-1-carboxylate, DCM, TEA, −35 °C, 0.5 h; (c) 5-cyclopropyl-1<i>H</i>-pyrazol-3-amine, DIEA, KI, DMF, 65 °C, 8 h; (d) HCl–EA, DCM, rt, 6 h.</p></p></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46628" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46628" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, starting from the 2,4-diaminoquinazoline scaffold compound <b>10a</b>, a series of novel 4-aminoquinazoline-2-carboxamide derivatives was designed and synthesized as selective and potent PAK4 inhibitors by utilizing highly efficient structure-based design strategies targeting key residues differences between PAK1 and PAK4. The optimized compound <b>31</b> demonstrates a striking selectivity for PAK4 over PAK1 (346-fold). As expected, the X-ray structure of <b>31</b> bound to PAK4 agrees well with the above-described design hypothesis. Our study also demonstrates that the selectivity and potency of PAK4 inhibitors can be improved by targeting the DFG motif and several other distinguishing features of the ATP-binding pocket. Furthermore, compound <b>31</b> possesses good kinome selectivity, favorable physicochemical properties, and moderate ADME properties, both <i>in vitro</i> and <i>in vivo</i>. Moreover, this compound inhibited the migration and invasion of A549 tumor cells by regulating the PAK4-directed downstream signaling pathways <i>in vitro</i>. Taken together, compound <b>31</b> may serve as a new lead compound for anticancer drug discovery, as well as a valuable research probe for further biological investigation on group II PAKs. The results from this study also provides molecular insights into the development of novel selective PAK4 kinase inhibitors.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62127" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62127" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General Methods for Chemistry</h3><div class="NLM_p">Commercially available chemicals were used as purchased without further purification. Solvents were purified and stored according to standard procedures. All reactions were monitored by thin-layer chromatography (TLC), and silica gel plates with fluorescence F-254 were used and visualized with UV light. Column chromatography was carried out on silica gel (100–200 mesh). The melting points were determined on a Buchi melting point B-540 apparatus and are uncorrected. The NMR spectra were measured on a Bruker Avance III-600 and/or Bruker Avance III-400 instruments (600 or 400 MHz for <sup>1</sup>H and 150 or 100 MHz for <sup>13</sup>C). Chemical shifts are expressed as δ units using tetramethylsilane as the external standard (in NMR description, s, singlet; d, doublet; t, triplet; q, quartet; m, multiplet; and br, broad peak). All coupling constants (<i>J</i>) are reported in Hertz. LC–MS analysis was performed on an Agilent 1200 LC–MS using ESI mode. High-resolution mass spectra (HRMS) were performed on an Agilent 6530 accurate-mass Q-TOF LC–MS system. All of the final compounds were purified to >95% purity, as determined by high-performance liquid chromatography (HPLC). This analysis was performed on a Waters Breeze system (Waters 1525, binary HPLC pump) equipped with an UV detector (Waters 2487, dual λ absorbance detector) and a WondaSil C-18 Superb column (5 μm, 4.6 mm × 250 mm) using a mobile phase of 60% acetonitrile in water containing 0.05% TFA. The flow-rate was 0.6 mL/min, and the injection volume was 20 μL. Peaks were detected at 254 nm. The retention times (RT) are in minutes, and the purity was calculated as percentage of total area.</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 2-(4-Aminopiperidin-1-yl)-6-chloro-<i>N</i>-(1-methyl-1<i>H</i>-imidazol-4-yl)quinazolin-4-amine Hydrochloride (<b>10a</b>)</h4><div class="NLM_p last">Two molar HCl in ethyl acetate (4 mL) was added to a solution of <b>52a</b> (152 mg, 0.33 mmol) in dichloromethane (10 mL). The mixture was stirred for 12 h at room temperature, and then, the resulting solid was filtered and dried to give the title compound (120 mg, 91% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.89 (d, <i>J</i> = 1.9 Hz, 1H), 8.41 (br, 3H), 8.38 (s, 1H), 8.19 (d, <i>J</i> = 8.8 Hz, 1H), 7.91(dd, <i>J</i> = 2.2, 9.0 Hz, 1H), 7.58 (d, <i>J</i> = 1.3 Hz, 1H), 4.75–4.72 (m, 2H), 3.85 (s, 3H), 3.42–3.41 (m, 1H), 3.36–3.30 (m, 2H), 2.14–2.11 (m, 2H), 1.71–1.63 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.2, 151.8, 140.0, 135.9, 134.3, 130.1, 129.4, 124.5, 120.4, 114.0, 111.2, 49.0, 44.4, 35.9, 29.7. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>17</sub>H<sub>21</sub>ClN<sub>7</sub> [M + H]<sup>+</sup>, 358.1541; found, 358.1547. <i>R</i><sub>f</sub> = 0.12 (DCM/MeOH, 10/1, v/v). Retention time: 3.95 min, 95.24% purity.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 2-(4-Aminopiperidin-1-yl)-6-chloro-<i>N</i>-(1-cyclopropyl-1<i>H</i>-imidazol-4-yl)quinazolin-4-amine Hydrochloride (<b>10b</b>)</h4><div class="NLM_p last">Compound <b>10b</b> was prepared in a similar manner as compound <b>10a</b>, using <b>44b</b> (R = <i>c</i>-Pr) instead of <b>44a</b> (R = Me). Light-yellow solid (yield 37%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.90 (d, <i>J</i> = 1.7 Hz, 1H), 8.45 (br, 3H), 8.43 (s, 1H), 8.24 (d, <i>J</i> = 8.9 Hz, 1H), 7.92(dd, <i>J</i> = 2.0, 8.9 Hz, 1H), 7.63 (s, 1H), 4.75 (br, 2H), 3.77–3.74 (m, 1H), 3.44–3.41 (m, 1H), 3.40–3.36 (m, 2H), 2.13–2.11 (m, 2H), 1.73–1.67 (m, 2H), 1.11–1.03 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.2, 151.7, 139.9, 135.9, 134.3, 130.7, 129.5, 124.5, 120.3, 113.5, 111.1, 49.0, 47.1, 44.3, 30.6, 29.7, 6.74. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>19</sub>H<sub>23</sub>ClN<sub>7</sub> [M + H]<sup>+</sup>, 384.1698; found, 384.1699. <i>R</i><sub>f</sub> = 0.18 (DCM/MeOH, 10/1, v/v). Retention time: 3.37 min, 97.52% purity.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 2-(4-Aminopiperidin-1-yl)-6-chloro-<i>N</i>-(1-isopropyl-1<i>H</i>-imidazol-4-yl)quinazolin-4-amine Hydrochloride (<b>10c</b>)</h4><div class="NLM_p last">Compound <b>10c</b> was prepared in a similar manner as compound <b>10a</b>, using <b>44c</b> (R = <i>i</i>-Pr) instead of <b>44a</b> (R = Me). Off-white solid (yield 33%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.91 (d, <i>J</i> = 1.9 Hz, 1H), 8.60 (s, 1H), 8.42 (br, 3H), 8.22 (d, <i>J</i> = 9.0 Hz, 1H), 7.92 (dd, <i>J</i> = 2.2, 9.0 Hz, 1H), 7.77 (d, <i>J</i> = 1.3, 1H), 4.75–4.72 (m, 2H), 4.68–4.60 (m, 1H), 3.42–3.41 (m, 1H), 3.37–3.31 (m, 2H), 2.11–2.09 (m, 2H), 1.72–1.65 (m, 2H), 1.51 (s, 3H), 1.50 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 157.5, 151.8, 140.2, 136.0, 132.4, 130.4, 129.4, 124.5, 120.5, 111.6, 111.2, 52.0, 49.0, 47.1, 44.3, 29.6, 23.0. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>19</sub>H<sub>23</sub>ClN<sub>7</sub> [M + H]<sup>+</sup>, 386.1854; found, 386.1854. <i>R</i><sub>f</sub> = 0.21 (DCM/MeOH, 10/1, v/v). Retention time: 3.84 min, 98.53% purity.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 2-(4-Aminopiperidin-1-yl)-6-chloro-<i>N</i>-(1-phenyl-1<i>H</i>-imidazol-4-yl)quinazolin-4-amine Hydrochloride (<b>10d</b>)</h4><div class="NLM_p last">Compound <b>10d</b> was prepared in a similar manner as compound <b>10a</b>, using <b>44d</b> (R = Ph) instead of <b>44a</b> (R = Me). White solid (yield 31%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.34 (br, 1H), 11.81 (br, 1H), 8.93 (d, <i>J</i> = 1.4 Hz, 1H), 8.48 (s, 1H), 8.44 (br, 3H), 8.26 (d, <i>J</i> = 8.9 Hz, 1H), 7.98 (s, 1H), 7.92 (dd, <i>J</i> = 1.8, 8.9 Hz, 1H), 7.69 (d, <i>J</i> = 7.9, 1H), 7.59 (t, <i>J</i> = 7.7, 1H), 7.45 (t, <i>J</i> = 7.4, 1H), 4.83 (br, 2H), 3.44 (m, 1H), 3.42–3.40 (m, 2H), 2.15–2.13(m, 2H), 1.76–1.70 (m, 2H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O) δ 159.0, 151.5, 136.7, 130.7, 130.4, 130.2, 123.3, 122.1, 119.1, 110.4, 105.3, 48.9, 47.5, 43.7, 28.9. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>23</sub>ClN<sub>7</sub> [M + H]<sup>+</sup>, 420.1698; found, 420.1704. <i>R</i><sub>f</sub> = 0.32 (DCM/MeOH, 10/1, v/v). Retention time: 4.04 min, 100% purity.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 2-(4-Aminopiperidin-1-yl)-6-chloro-<i>N</i>-(5-methyl-1<i>H</i>-pyrazol-3-yl)quinazolin-4-amine Hydrochloride (<b>11a</b>)</h4><div class="NLM_p last">Compound <b>11a</b> was prepared in a similar manner as compound <b>10a</b>, using <b>53a</b> (R = Me) instead of <b>44a</b>. White solid (yield 41%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.20 (br, 1H), 11.47 (s, 1H), 8.84 (d, <i>J</i> = 2.0 Hz, 1H), 8.43 (br, 3H), 8.23 (d, <i>J</i> = 9.0 Hz, 1H), 7.91 (dd, <i>J</i> = 2.1, 8.9 Hz, 1H), 6.41 (s, 1H), 4.80–4.76 (m, 2H), 3.43–3.32 (m, 3H), 2.31 (s, 3H), 2.14–2.11 (m, 2H), 1.71–1.63 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 156.7, 151.7, 145.8, 139.8, 139.6, 135.5, 129.3, 124.3, 120.2, 111.5, 98.5, 47.2, 44.2, 29.6, 11.3. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>17</sub>H<sub>21</sub>ClN<sub>7</sub> [M + H]<sup>+</sup>, 358.1541; found, 358.1542. <i>R</i><sub>f</sub> = 0.13 (DCM/MeOH, 10/1, v/v). Retention time: 3.88 min, 96.28% purity.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 2-(4-Aminopiperidin-1-yl)-6-chloro-<i>N</i>-(5-cyclopropyl-1<i>H</i>-pyrazol-3-yl)quinazolin-4-amine Hydrochloride (<b>11b</b>)</h4><div class="NLM_p last">Compound <b>11b</b> was prepared in a similar manner as compound <b>10a</b>, using <b>53b</b> (R = <i>c</i>-Pr) instead of <b>44a</b>. White solid (yield 43%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.30 (br, 1H), 11.49 (d, <i>J</i> = 2.0 Hz, 1H), 8.47 (br, 3H), 8.81 (d, 1H), 8.26 (d, <i>J</i> = 9.0 Hz, 1H), 7.89 (dd, <i>J</i> = 2.1, 8.9 Hz, 1H), 6.28 (s, 1H), 4.79–4.77 (m, 2H), 3.43–3.32 (m, 3H), 2.12–2.09 (m, 2H), 2.00–1.95 (m, 1H), 1.71–1.64 (m, 2H), 1.01–0.97 (m, 2H), 0.71–0.67 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 156.7, 151.6, 147.0, 145.8, 139.6, 135.5, 129.4, 124.3, 120.2, 111.4, 95.3, 47.2, 44.2, 29.6, 8.7, 7.3. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>19</sub>H<sub>23</sub>ClN<sub>7</sub> [M + H]<sup>+</sup>, 384.1698; found, 384.1711. <i>R</i><sub>f</sub> = 0.17 (DCM/MeOH, 10/1, v/v). Retention time: 3.99 min, 100% purity.</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 2-(4-Aminopiperidin-1-yl)-6-chloro-<i>N</i>-(5-isopropyl-1<i>H</i>-pyrazol-3-yl)quinazolin-4-amine Hydrochloride (<b>11c</b>)</h4><div class="NLM_p last">Compound <b>11c</b> was prepared in a similar manner as compound <b>10a</b>, using <b>53c</b> (R = <i>i</i>-Pr) instead of <b>44a</b>. White solid (yield 32%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.15 (br, 1H), 11.54 (s, 1H), 8.83 (d, <i>J</i> = 1.8 Hz, 1H), 8.37 (br, 3H), 8.19 (d, <i>J</i> = 8.9 Hz, 1H), 7.92 (dd, <i>J</i> = 2.0, 8.9 Hz, 1H), 6.45 (s, 1H), 4.77 (d, <i>J</i> = 12.8 Hz, 2H), 3.51–3.33 (m, 3H), 3.05–2.98 (m, 1H), 2.11–2.09 (m, 2H), 1.72–1.66 (m, 2H), 1.27(s, 3H), 1.25 (s, 3H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O) δ 156.4, 151.0, 137.7, 135.8, 130.5, 122.6, 118.8, 110.1, 48.9, 47.5, 43.7, 28.9, 25.6, 21.3. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>19</sub>H<sub>23</sub>ClN<sub>7</sub> [M + H]<sup>+</sup>, 386.1854; found, 386.1861. <i>R</i><sub>f</sub> = 0.20 (DCM/MeOH, 10/1, v/v). Retention time: 3.90 min, 96.16% purity.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 2-(4-Aminopiperidin-1-yl)-6-chloro-<i>N</i>-(5-phenyl-1<i>H</i>-pyrazol-3-yl)quinazolin-4-amine Hydrochloride (<b>11d</b>)</h4><div class="NLM_p">Compound <b>11d</b> was prepared in a similar manner as compound <b>10a</b>, using <b>53d</b> (R = Ph) instead of <b>44a</b>. White solid (yield 27%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.29 (br, 1H), 11.66 (s, 1H), 8.87 (d, <i>J</i> = 1.80 Hz, 1H), 8.40 (br, 3H), 8.25 (d, <i>J</i> = 9.0 Hz, 1H), 7.94 (dd, <i>J</i> = 2.0, 8.9 Hz, 1H), 7.78 (d, <i>J</i> = 7.4 Hz, 2H), 7.51 (t, <i>J</i> = 7.5 Hz, 2H), 7.41 (t, <i>J</i> = 7.3 Hz, 1H), 7.05 (s, 1H), 4.83–4.80 (m, 2H), 3.45–3.36 (m, 3H), 2.15–2.12 (m, 2H), 1.75–1.67 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 156.9, 151.6, 146.5, 143.2, 139.7, 135.6, 129.6, 129.4, 128.9, 125.5, 124.3, 120.2, 111.4, 96.8, 47.2, 44.3, 29.7. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>22</sub>H<sub>23</sub>ClN<sub>7</sub> [M + H]<sup>+</sup>, 420.1698; found, 420.1701. <i>R</i><sub>f</sub> = 0.23 (DCM/MeOH, 10/1, v/v). Retention time: 3.96 min, 96.67% purity.</div><div class="NLM_p last">Compounds <b>12</b>–<b>41</b> were synthesized using a similar procedure with yields in the range 12–28% (8 steps; for <b>13</b>, <b>15</b>, <b>18</b>, <b>19</b>, and <b>20</b>, no Boc deprotection was needed).</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (4-Aminopiperidin-1-yl)(6-chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)methanone Hydrochloride (<b>12</b>)</h4><div class="NLM_p last">Off-white solid (yield 27%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.73 (br, 1H), 9.05 (d, <i>J</i> = 2.0 Hz, 1H), 8.42 (br, 3H), 8.03 (dd, <i>J</i> = 2.1, 8.9 Hz, 1H), 7.92 (d, <i>J</i> = 9.0 Hz, 1H), 6.53 (s, 1H), 4.50–4.47 (m, 1H), 3.86–3.83 (m, 1H), 3.38–3.30 (m, 1H), 3.21–3.15 (m, 1H), 3.03–2.97 (m, 1H), 2.34 (s, 3H), 2.11–2.08 (m, 1H), 1.93–1.90 (m, 1H), 1.65–1.53 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.1, 157.1, 155.0, 145.5, 143.1, 141.1, 135.9, 133.0, 126.5, 124.3, 115.4, 98.7, 47.6, 44.7, 30.4, 29.5, 11.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>18</sub>H<sub>21</sub>ClN<sub>7</sub>O [M + H]<sup>+</sup>, 386.1491; found, 386.1500. <i>R</i><sub>f</sub> = 0.10 (DCM/MeOH, 5/1, v/v). Retention time: 3.91 min, 96.28% purity.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (6-Chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(4-hydroxypiperidin-1-yl)methanone (<b>13</b>)</h4><div class="NLM_p last">Off-white solid (yield 19%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.29 (s, 1H), 10.69 (s, 1H), 8.88 (s, 1H), 7.88 (dd, <i>J</i> = 1.8, 8.9 Hz, 1H), 7.79 (d, <i>J</i> = 8.8 Hz, 1H), 6.56 (s, 1H), 4.83 (d, <i>J</i> = 4.0 Hz, 1H), 4.06–4.02 (m, 1H), 3.77–3.72 (m, 1H), 3.39–3.32 (m, 1H), 3.26–3.21 (m, 1H), 3.07–3.02 (m, 1H), 2.27 (s, 3H), 1.84–1.81 (m, 1H), 1.71–1.67 (m, 1H), 1.41–1.30 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.5, 159.1, 157.1, 148.4, 138.8, 134.1, 131.3, 130.3, 123.3, 115.5, 98.3, 65.9, 44.0, 38.8, 35.0, 34.3, 11.3. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>18</sub>H<sub>20</sub>ClN<sub>6</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 387.1331; found, 387.1334. <i>R</i><sub>f</sub> = 0.60 (DCM/MeOH, 5/1, v/v). Retention time: 4.57 min, 96.49% purity.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (4-Amino-4-methylpiperidin-1-yl)(6-chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)methanone Hydrochloride (<b>14</b>)</h4><div class="NLM_p last">Off-white solid (yield 23%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.94 (br, 1H), 9.08 (d, <i>J</i> = 1.9 Hz, 1H), 8.52 (br, 3H), 8.05 (dd, <i>J</i> = 2.1, 8.9 Hz, 1H), 7.95 (d, <i>J</i> = 8.9 Hz, 1H), 6.56 (s, 1H), 4.05–4.01 (m, 1H), 3.77–3.73 (m, 1H), 3.61–3.56 (m, 1H), 3.46–3.42 (m, 1H), 2.34 (s, 3H), 1.92–1.70 (m, 4H), 1.38 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.0, 157.2, 154.9, 145.5, 142.9, 141.1, 135.9, 133.0, 126.3, 124.3, 115.4, 98.8, 52.3, 42.4, 37.8, 35.5, 34.6, 23.5, 11.3. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>19</sub>H<sub>23</sub>ClN<sub>7</sub>O [M + H]<sup>+</sup>, 400.1647; found, 400.1655. <i>R</i><sub>f</sub> = 0.13 (DCM/MeOH, 5/1, v/v). Retention time: 3.90 min, 95.57% purity.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (6-Chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(4-hydroxy-4-methylpiperidin-1-yl)methanone (<b>15</b>)</h4><div class="NLM_p last">Off-white solid (yield 16%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.28 (s, 1H), 10.68 (s, 1H), 8.88 (s, 1H), 7.88 (dd, <i>J</i> = 2.1, 8.9 Hz, 1H), 7.79 (d, <i>J</i> = 8.9 Hz, 1H), 6.55 (s, 1H), 4.46 (s, 1H), 4.11–4.08 (m, 1H), 3.24–3.16 (m, 2H), 3.09–3.04 (m, 1H), 2.27 (s, 3H), 1.60–1.38 (m, 4H), 1.16 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.5, 159.2, 157.1, 148.5, 138.9, 134.1, 131.2, 130.3, 123.2, 115.5, 98.3, 66.7, 45.8, 43.2, 39.1, 38.4, 37.8, 30.4, 11.3, 8.9. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>19</sub>H<sub>22</sub>ClN<sub>6</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 401.1487; found, 401.1501. <i>R</i><sub>f</sub> = 0.63 (DCM/MeOH, 5/1, v/v). Retention time: 4.37 min, 95.16% purity.</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (6-Chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(4-(methylamino)piperidin-1-yl)methanone Hydrochloride (<b>16</b>)</h4><div class="NLM_p last">Light-yellow solid (yield 26%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.82 (br, 2H), 9.52–9.44 (m, 2H), 9.07 (d, <i>J</i> = 1.8 Hz, 1H), 8.03 (dd, <i>J</i> = 2.0, 8.9 Hz, 1H), 7.93 (d, <i>J</i> = 8.9 Hz, 1H), 6.54 (s, 1H), 4.55–4.52 (m, 1H), 3.92–3.89 (m, 1H), 3.31–3.22 (m, 1H), 3.17–3.11 (m, 1H), 2.99–2.93 (m, 1H), 2.50 (s, 3H), 2.34 (s, 3H), 2.21–2.18 (m, 1H), 2.02–1.99 (m, 1H), 1.70–1.60 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.5, 157.1, 155.5, 145.8, 143.7, 140.8, 135.7, 132.7, 126.9, 124.1, 115.4, 98.6, 55.0, 44.7, 29.6, 28.5, 27.6, 11.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>19</sub>H<sub>23</sub>ClN<sub>7</sub> [M + H]<sup>+</sup>, 400.1647; found, 401.1650. <i>R</i><sub>f</sub> = 0.10 (DCM/MeOH, 5/1, v/v). Retention time: 3.98 min, 95.79% purity.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (6-Chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(piperazin-1-yl)methanone Hydrochloride (<b>17</b>)</h4><div class="NLM_p last">Two molar HCl in ethyl acetate (4 mL) was added to a solution of <i>tert</i>-butyl 4-(6-chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazoline-2-carbonyl)piperazine-1-carboxylate (<b>66a</b>, 145 mg, 0.35 mmol) in dichloromethane (5 mL). The mixture was stirred for 6 h at room temperature, and then, the resulting solid was filtered and dried to give the title compound (120 mg, 91% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.47 (br, 1H), 9.59 (s, 2H), 9.01 (s, 1H), 8.02–8.00 (m,1H), 7.91 (dd, <i>J</i> = 2.6, 8.8 Hz, 1H), 6.56 (s, 1H), 3.93–3.91 (m, 2H), 3.71 (s, 2H), 3.23 (s, 2H), 3.09 (s, 2H), 2.32 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.0, 157.1, 155.0, 145.7, 144.4, 140.9, 135.6, 132.8, 127.6, 124.0, 115.5, 98.6, 43.4, 43.1, 42.7, 38.8, 11.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>17</sub>H<sub>19</sub>ClN<sub>7</sub>O [M + H]<sup>+</sup>, 372.1334; found, 372.1333. <i>R</i><sub>f</sub> = 0.21 (DCM/MeOH, 5/1, v/v). Retention time: 4.82 min, 95.04% purity.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (6-Chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(morpholino)methanone (<b>18</b>)</h4><div class="NLM_p last">White solid (yield 13%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.28 (s, 1H), 10.71 (s, 1H), 8.88 (d, <i>J</i> = 1.9 Hz, 1H), 7.88 (dd, <i>J</i> = 2.1, 8.9 Hz, 1H), 7.79 (d, <i>J</i> = 8.9 Hz, 1H), 6.57 (s, 1H), 3.68–3.65 (m, 4H), 3.54–3.52 (m, 2H), 3.30–3.28 (m, 2H), 2.28 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.7, 158.4, 157.1, 148.4, 147.6, 138.8, 134.2, 131.4, 130.3, 123.2, 115.5, 98.4, 66.8, 66.5, 47.0, 41.7, 11.3. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>17</sub>H<sub>18</sub>ClN<sub>6</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 373.1174; found, 373.1176. <i>R</i><sub>f</sub> = 0.43 (DCM/MeOH, 5/1, v/v). Retention time: 4.55 min, 96.72% purity.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (6-Chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(4-methylpiperazin-1-yl)methanone (<b>19</b>)</h4><div class="NLM_p last">White solid (yield 22%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.28 (s, 1H), 10.71(s, 1H), 8.89 (d, <i>J</i> = 1.8 Hz, 1H), 7.88 (dd, <i>J</i> = 2.0, 8.9 Hz, 1H), 7.79 (d, <i>J</i> = 8.9 Hz, 1H), 6.59 (s, 1H), 3.65–3.63 (m, 2H), 3.26–3.23 (m, 2H), 3.40–3.37 (m, 2H), 2.28 (s, 3H), 2.27–2.22 (m, 2H), 2.20 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.6, 158.7, 157.1, 148.4, 147.6, 138.7, 134.1, 131.3, 130.3, 123.2, 115.5, 98.4, 55.2, 54.7, 46.3, 46.2, 41.2, 11.2. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>18</sub>H<sub>21</sub>ClN<sub>7</sub>O [M + H]<sup>+</sup>, 386.1491; found, 386.1502. <i>R</i><sub>f</sub> = 0.20 (DCM/MeOH, 5/1, v/v). Retention time: 3.91 min, 95.48% purity.</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-(6-Chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazoline-2-carbonyl)piperazin-2-one (<b>20</b>)</h4><div class="NLM_p last">Yellow solid (yield 12%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.30 (s, 1H), 10.70 (m, 1H), 8.90–8.88 (m, 1H), 8.18–8.15 (m, 1H), 7.90 (d, <i>J</i> = 8.8 Hz, 1H), 7.80 (d, <i>J</i> = 8.9 Hz, 1H), 6.56 (s, 1H), 4.16–3.90 (m, 2H), 3.85–3.45 (m, 2H), 3.32–3.17 (m, 2H), 2.27 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.3, 165.6, 158.0, 157.3, 148.3, 147.5, 138.9, 134.3, 131.6, 130.4, 123.3, 115.7, 98.4, 49.6, 45.6, 43.2, 40.9, 40.5, 38.6, 11.3. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>17</sub>H<sub>16</sub>ClN<sub>7</sub>O<sub>2</sub>, [M + H]<sup>+</sup>, 386.1127; found, 386.1118. <i>R</i><sub>f</sub> = 0.30 (DCM/MeOH, 10/1, v/v). Retention time: 4.29 min, 96.01% purity.</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 6-Chloro-<i>N</i>-methyl-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)-<i>N</i>-(piperidin-4-yl)quinazoline-2-carboxamide Hydrochloride (<b>21</b>)</h4><div class="NLM_p last">Light-yellow solid (yield 21%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.35 (br, 1H), 8.98–8.81 (m, 3H), 7.99 (t, <i>J</i> = 9.1 Hz, 1H), 7.89 (d, <i>J</i> = 7.2 Hz, 1H), 6.55–6.41 (m, 1H), 4.66–4.60 (m, 1H), 3.88–3.82 (m, 1H), 3.40–3.37 (m, 1H), 3.12–3.07 (m, 1H), 2.90–2.80 (m, 3H), 2.77–2.68 (m, 1H), 2.29 (s, 3H), 2.09–2.07(m, 2H), 1.86–1.78 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.8, 158.1, 157.0, 155.7, 145.6, 140.6, 135.8, 132.7, 126.3, 124.2, 115.5, 99.1, 52.8, 49.0, 43.0, 42.4, 31.3, 27.4, 26.2, 25.1, 11.3. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>19</sub>H<sub>23</sub>ClN<sub>7</sub>O [M + H]<sup>+</sup>, 400.1647; found, 400.1649. <i>R</i><sub>f</sub> = 0.25 (DCM/MeOH, 5/1, v/v). Retention time: 3.99 min, 97.40% purity.</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 6-Chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)-<i>N</i>-(piperidin-4-yl)quinazoline-2-carboxamide Hydrochloride (<b>22</b>)</h4><div class="NLM_p last">Off-white solid (yield 19%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.55 (br, 1H), 9.29 (s, 1H), 9.17–9.07 (m, 3H), 8.13 (dd, <i>J</i> = 2.0, 8.9 Hz, 1H), 8.06 (d, <i>J</i> = 8.9 Hz, 1H), 6.68 (s, 1H), 4.15–4.07 (m, 1H), 3.34–3.31 (m, 2H), 3.07–3.02 (m, 2H), 2.35 (s, 3H), 2.03–1.89 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 159.3, 153.5, 147.3, 146.9, 143.9, 141.0, 136.9, 134.3, 128.7, 125.4, 116.4, 98.0, 45.6, 42.4, 28.0, 11.2. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>18</sub>H<sub>21</sub>ClN<sub>7</sub>O [M + H]<sup>+</sup>, 386.1491; found, 386.1492. <i>R</i><sub>f</sub> = 0.21 (DCM/MeOH, 5/1, v/v). Retention time: 4.57 min, 95.13% purity.</div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 6-Chloro-<i>N</i>-methyl-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)-<i>N</i>-(piperidin-3-yl)quinazoline-2-carboxamide Hydrochloride (<b>23</b>)</h4><div class="NLM_p last">Light-yellow solid (yield 15%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.60 (br, 1H), 9.62–9.47 (m, 1H), 9.22–8.94 (m, 1H), 9.03 (dd, <i>J</i> = 1.6, 11.0 Hz, 1H), 8.02 (dd, <i>J</i> = 1.9, 9.0 Hz, 1H), 7.95–7.90 (m, 1H), 6.57–6.48 (m, 1H), 4.76–3.95 (m, 1H), 3.30–3.05 (m, 3H), 2.95–2.87 (m, 3H), 2.83–2.65 (m, 1H), 2.31 (s, 3H), 1.96–1.46 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.2, 157.7, 155.5, 145.5, 143.3, 141.0, 135.7, 132.8, 126.7, 124.2, 115.6, 98.8, 51.9, 48.3, 44.9, 43.8, 42.9, 42.4, 31.8, 27.6, 26.4, 25.2, 21.8, 21.3, 11.3. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>19</sub>H<sub>23</sub>ClN<sub>7</sub>O [M + H]<sup>+</sup>, 400.1647; found, 400.1650. <i>R</i><sub>f</sub> = 0.36 (DCM/MeOH, 5/1, v/v). Retention time: 3.92 min, 95.14% purity.</div></div><div id="sec5_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i>-(Azetidin-3-yl)-6-chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazoline-2-carboxamide Hydrochloride (<b>24</b>)</h4><div class="NLM_p last">Yellow solid (yield 17%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.96 (br, 1H), 9.70–9.04 (m, 3H), 8.26 (s, 1H), 8.07–7.96 (m, 2H), 6.82 (s, 1H), 4.92–4.86 (m, 1H), 4.58–4.53 (m, 1H), 4.24–4.17 (m, 2H), 3.96–3.94 (m, 1H), 3.18–3.16 (m, 1H), 2.32 (d, <i>J</i> = 2.4 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.9, 154.0, 148.7, 147.1, 145.5, 140.8, 136.2, 133.9, 129.7, 124.9, 116.6, 98.1, 51.8, 49.9, 44.9, 42.0, 11.3. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>16</sub>H<sub>17</sub>ClN<sub>7</sub>O, [M + H]<sup>+</sup>, 358.1178; found, 358.1178. <i>R</i><sub>f</sub> = 0.21 (DCM/MeOH, 5/1, v/v). Retention time: 3.90 min, 95.03% purity.</div></div><div id="sec5_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (6-Chloro-4-((5-cyclopropyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(piperazin-1-yl)methanone Hydrochloride (<b>25</b>)</h4><div class="NLM_p last">Off-white solid (yield 26%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.52 (br, 1H), 9.57 (s, 2H), 9.00 (d, <i>J</i> = 1.9 Hz, 1H), 8.01 (dd, <i>J</i> = 2.1, 8.9 Hz, 1H), 7.92 (d, <i>J</i> = 8.9 Hz, 1H), 6.46 (s, 1H), 3.93–3.91 (m, 2H), 3.75–3.72 (m, 2H), 3.23 (s, 2H), 3.10 (s, 2H), 2.04–1.97 (m, 1H), 1.02–0.97 (m, 2H), 0.80–0.76 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.0, 157.0, 153.8 148.3, 145.2, 143.0, 136.1, 133.3, 126.6, 124.4, 115.4, 95.8, 49.0, 43.5, 43.0, 42.6, 39.0, 8.8, 7.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>19</sub>H<sub>21</sub>ClN<sub>7</sub>O, [M + H]<sup>+</sup>, 398.1491; found, 398.1493. <i>R</i><sub>f</sub> = 0.38 (DCM/MeOH, 5/1, v/v). Retention time: 3.91 min, 98.79% purity.</div></div><div id="sec5_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (<i>S</i>)-(6-Chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(2-methylpiperazin-1-yl)methanone Hydrochloride (<b>26</b>)</h4><div class="NLM_p last">Light yellow solid (yield 22%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.50 (br, 1H), 9.83 (s, 1H), 9.46–9.44 (m, 1H), 9.02 (d, <i>J</i> = 1.8 Hz, 1H), 8.01 (dd, <i>J</i> = 1.8, 9.0 Hz, 1H), 7.92 (dd, <i>J</i> = 5.4, 8,8 Hz, 1H), 6.55 (d, <i>J</i> = 7.80 Hz, 1H), 4.88–4.86 (m, 1H), 4.52–4.49 (m, 1H), 4.17–3.78 (m, 1H), 3.54–2.91 (m, 1H), 2.32 (s, 3H), 1.45–1.37 (m, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.0, 157.3, 155.3, 145.8, 144.2, 140.9, 135.6, 132.8, 127.4, 124.1, 115.5, 98.6, 47.6, 46.4, 45.9, 42.8, 42.6, 38.8, 33.5, 16.8, 15.6, 11.3. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>18</sub>H<sub>21</sub>ClN<sub>7</sub>O, [M + H]<sup>+</sup>, 386.1491; found, 386.1495. <i>R</i><sub>f</sub> = 0.34 (DCM/MeOH, 5/1, v/v). Retention time: 4.03 min, 96.24% purity.</div></div><div id="sec5_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (<i>S</i>)-(6-Chloro-4-((5-cyclopropyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(2-methylpiperazin-1-yl)methanone Hydrochloride (<b>27</b>)</h4><div class="NLM_p last">Light yellow solid (yield 26%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.49 (br, 1H), 9.87 (s, 1H), 9.51–9.49 (m, 1H), 9.01 (d, <i>J</i> = 1.6 Hz, 1H), 8.00 (d, <i>J</i> = 8.7 Hz, 1H), 7.91 (d, <i>J</i> = 8.9 Hz, 1H), 6.44 (d, <i>J</i> = 8.8 Hz, 1H), 4.91–4.88 (m, 1H), 4.52–4.49 (m, 1H), 4.19–3.75 (m, 1H), 3.53–3.28 (m, 2H), 3.19–2.89 (m, 3H), 2.00–1.98 (m, 1H), 1.45–1.38 (m, 3H), 1.00–0.98 (m, 2H), 0.75–0.73 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.6, 156.9, 154.9, 148.0, 145.7, 135.7, 132.9, 127.2, 124.2, 115.5, 95.5, 47.7, 46.4, 45.9, 42.8, 42.6, 38.8, 33.6, 16.9, 15.6, 8.8, 7.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>20</sub>H<sub>23</sub>ClN<sub>7</sub>O, [M + H]<sup>+</sup>, 412.1647; found, 412.1649. <i>R</i><sub>f</sub> = 0.37 (DCM/MeOH, 5/1, v/v). Retention time: 4.10 min, 100% purity.</div></div><div id="sec5_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>R</i>)-(6-Chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(2-methylpiperazin-1-yl)methanone Hydrochloride (<b>28</b>)</h4><div class="NLM_p last">Light yellow solid (yield 19%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.21 (br, 1H), 9.69 (br, 1H), 9.28 (br, 1H), 8.96 (s, 1H), 7.97 (dd, <i>J</i> = 1.7, 8.8 Hz, 1H), 7.87 (dd, <i>J</i> = 5.4, 8,5 Hz, 1H), 6.55 (d, <i>J</i> = 9.1 Hz, 1H), 4.90–4.87 (m, 1H), 4.52–4.49 (m, 1H), 3.74–3.45 (m, 1H), 3.36–2.89 (m, 4H), 2.30 (s, 3H),1.43–1.36 (m, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.0, 157.3, 155.4, 145.8, 144.4, 140.9, 135.6, 132.8, 127.5, 124.1, 115.5, 98.6, 47.6, 46.4, 45.9, 42.8, 42.6, 38.8, 33.5, 16.8, 15.6, 11.3. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>18</sub>H<sub>21</sub>ClN<sub>7</sub>O, [M + H]<sup>+</sup>, 386.1491; found, 386.1495. <i>R</i><sub>f</sub> = 0.34 (DCM/MeOH, 5/1, v/v). Retention time: 3.99 min, 98.98% purity.</div></div><div id="sec5_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (<i>R</i>)-(6-Chloro-4-((5-cyclopropyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(2-methylpiperazin-1-yl)methanone Hydrochloride (<b>29</b>)</h4><div class="NLM_p last">Light yellow solid (yield 21%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.36 (br, 1H), 9.78 (s, 1H), 9.41–9.39 (m, 1H), 8.98 (s, 1H), 7.98 (d, <i>J</i> = 9.0 Hz, 1H), 7.89 (d, <i>J</i> = 8.9 Hz, 1H), 6.44 (d, <i>J</i> = 9.1 Hz, 1H), 4.93–4.86 (m, 1H), 4.52–4.48 (m, 1H), 3.76–3.45 (m, 1H), 3.35–2.87 (m, 4H), 2.02–1.98 (m, 1H), 1.44–1.37 (m, 3H), 1.00–0.98 (m, 2H), 0.74–0.73 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.9, 156.9, 154.9, 147.9, 145.7, 135.7, 132.8, 127.3, 124.2, 115.5, 95.5, 47.6, 46.4, 45.9, 42.8, 42.6, 38.8, 33.6, 16.8, 15.6, 8.7, 7.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>20</sub>H<sub>23</sub>ClN<sub>7</sub>O, [M + H]<sup>+</sup>, 412.1647; found, 412.1648. <i>R</i><sub>f</sub> = 0.38 (DCM/MeOH, 5/1, v/v). Retention time: 4.10 min, 100% purity.</div></div><div id="sec5_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (<i>R</i>)-(6-Chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(3-methylpiperazin-1-yl)methanone hydrochloride (<b>30</b>)</h4><div class="NLM_p last">Light yellow solid (yield 19%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.39 (br, 1H), 9.75–9.64 (m, 2H), 8.99 (s, 1H), 7.99 (d, <i>J</i> = 10.0 Hz, 1H), 7.90 (d, <i>J</i> = 8.8 Hz, 1H), 6.54 (d, <i>J</i> = 7.6 Hz, 1H), 4.47–4.43 (m, 1H), 3.92–3.83 (m, 1H), 3.54–2.98 (m, 5H), 2.31 (s, 3H), 1.36–1.12 (m, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.3, 157.3, 155.2, 145.9, 144.7, 140.8, 135.5, 132.7, 127.8, 124.0, 115.6, 98.6, 51.0, 50.5, 48.9, 44.4, 43.0, 42.6, 38.3, 15.8, 15.3, 11.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>18</sub>H<sub>21</sub>ClN<sub>7</sub>O, [M + H]<sup>+</sup>, 386.1491; found, 386.1491. <i>R</i><sub>f</sub> = 0.34 (DCM/MeOH, 5/1, v/v). Retention time: 4.01 min, 96.53% purity.</div></div><div id="sec5_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (<i>R</i>)-(6-Chloro-4-((5-cyclopropyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(3-methylpiperazin-1-yl)methanone Hydrochloride (<b>31</b>)</h4><div class="NLM_p last">Light yellow solid (yield 29%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.38 (br, 1H), 9.71–9.61 (m, 2H), 8.98 (s, 1H), 8.00 (d, <i>J</i> = 9.0 Hz, 1H), 7.90 (d, <i>J</i> = 8.8 Hz, 1H), 6.46 (d, <i>J</i> = 7.4 Hz, 1H), 4.46–4.42 (m, 1H), 3.96–3.87 (m, 1H), 3.54–2.98 (m, 5H), 2.03–1.96 (m, 1H), 1.36–1.13 (m, 3H), 1.00–0.98 (m, 2H), 0.77–0.76 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.7, 157.2, 154.4, 148.1, 145.4, 135.8, 133.0, 127.2, 124.3, 115.5, 95.8, 50.9, 50.5, 48.9, 44.4, 43.0, 42.5, 42.2, 38.4, 15.7, 15.3, 8.8, 7.5. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>20</sub>H<sub>23</sub>ClN<sub>7</sub>O, [M + H]<sup>+</sup>, 412.1647; found, 412.1648. <i>R</i><sub>f</sub> = 0.39 (DCM/MeOH, 5/1, v/v). Retention time: 3.92 min, 98.69% purity.</div></div><div id="sec5_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (<i>S</i>)-(6-Chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(3-methylpiperazin-1-yl)methanone Hydrochloride (<b>32</b>)</h4><div class="NLM_p last">Light-yellow solid (yield 23%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.42 (br, 1H), 9.77–9.66 (m, 2H), 9.00 (s, 1H), 8.00 (d, <i>J</i> = 9.0 Hz, 1H), 7.91 (d, <i>J</i> = 8.9 Hz, 1H), 6.54 (d, <i>J</i> = 7.1 Hz, 1H), 4.47–4.44 (m, 1H), 3.94–3.85 (m, 1H), 3.55–2.99 (m, 5H), 2.32 (s, 3H), 1.36–1.12 (m, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.9, 157.3, 154.8, 145.6, 144.1, 141.0, 135.7, 132.9, 127.4, 124.1, 115.5, 98.7, 50.9, 50.5, 49.0, 48.9, 44.4, 43.0, 42.6, 42.2, 38.3, 15.8, 15.3, 11.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>18</sub>H<sub>21</sub>ClN<sub>7</sub>O, [M + H]<sup>+</sup>, 386.1491; found, 386.1492. <i>R</i><sub>f</sub> = 0.34 (DCM/MeOH, 5/1, v/v). Retention time: 4.01 min, 95.35% purity.</div></div><div id="sec5_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (<i>S</i>)-(6-Chloro-4-((5-cyclopropyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(3-methylpiperazin-1-yl)methanone Hydrochloride (<b>33</b>)</h4><div class="NLM_p last">Light-yellow solid (yield 22%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.21 (br, 1H), 9.68–9.57 (m, 2H), 8.95 (d, <i>J</i> = 1.8 Hz, 1H), 7.97 (dd, <i>J</i> = 2.0, 8.9 Hz, 1H), 7.88 (dd, <i>J</i> = 1.3, 8.8 Hz, 1H), 6.44 (d, <i>J</i> = 8.4 Hz, 1H), 4.45–4.42 (m, 1H), 3.92–3.82 (m, 1H), 3.52–3.35 (m, 2H), 3.29–2.96 (m, 3H), 2.02–1.95 (m, 1H), 1.36–1.13 (m, 3H), 1.00–0.95 (m, 2H), 0.77–0.73 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.5, 157.2, 155.4, 147.4, 146.0, 145.2, 135.3, 132.6, 128.1, 123.9, 115.6, 95.9, 50.9, 50.5, 48.9, 44.3, 43.0, 42.6, 42.2, 38.3, 15.7, 15.3, 8.6, 7.5. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>20</sub>H<sub>23</sub>ClN<sub>7</sub>O, [M + H]<sup>+</sup>, 412.1647; found, 412.1644. <i>R</i><sub>f</sub> = 0.39 (DCM/MeOH, 5/1, v/v). Retention time: 3.89 min, 96.85% purity.</div></div><div id="sec5_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (6-Chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(3,5-dimethylpiperazin-1-yl)methanone Hydrochloride (<b>34</b>)</h4><div class="NLM_p last">Light-yellow solid (yield 17%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.35 (br, 1H), 9.83 (d, <i>J</i> = 9.3 Hz, 1H), 9.45–9.47 (m, 1H), 8.98 (d, <i>J</i> = 1.8 Hz, 1H), 7.99 (dd, <i>J</i> = 2.0, 8.9 Hz, 1H), 7.89 (d, <i>J</i> = 8.9 Hz, 1H), 6.52 (s, 1H), 4.60 (d, <i>J</i> = 12.8 Hz, 1H), 3.94 (d, <i>J</i> = 12.6 Hz, 1H), 3.36–3.30 (m, 2H), 3.19 (t, <i>J</i> = 13.5 Hz, 1H), 3.00 (t, <i>J</i> = 12.2 Hz, 1H), 2.32 (s, 1H), 1.36 (d, <i>J</i> = 6.4 Hz, 3H), 1.13 (d, <i>J</i> = 6.2 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.0, 157.3, 155.1, 145.9, 144.4, 140.7, 135.5, 132.7, 127.5, 124.0, 115.6, 98.6, 51.8, 51.2, 48.5, 44.0, 15.9, 15.4, 11.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>19</sub>H<sub>23</sub>ClN<sub>7</sub>O, [M + H]<sup>+</sup>, 400.1647; found, 400.1650. <i>R</i><sub>f</sub> = 0.44 (DCM/MeOH, 5/1, v/v). Retention time: 4.03 min, 96.78% purity.</div></div><div id="sec5_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (6-Chloro-4-((5-cyclopropyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(3,5-dimethylpiperazin-1-yl)methanone Hydrochloride (<b>35</b>)</h4><div class="NLM_p last">Light-yellow solid (yield 25%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.20 (br, 1H), 9.72 (d, <i>J</i> = 9.8 Hz, 1H), 9.45–9.40 (m, 1H), 8.96 (d, <i>J</i> = 1.7 Hz, 1H), 7.98 (dd, <i>J</i> = 2.0, 8.9 Hz, 1H), 7.88 (d, <i>J</i> = 8.9 Hz, 1H), 6.44 (s, 1H), 4.63–4.60 (m, 1H), 4.03 (q, <i>J</i> = 7.1 Hz, 1H), 3.95 (d, <i>J</i> = 13.3 Hz, 1H), 3.36–3.29 (m, 2H), 3.19 (t, <i>J</i> = 13.5 Hz, 1H), 2.97 (t, <i>J</i> = 13.4 Hz, 1H), 2.02–1.96 (m, 1H), 1.36 (d, <i>J</i> = 6.4 Hz, 3H), 1.14 (d, <i>J</i> = 6.2 Hz, 3H), 1.01–0.96 (m, 2H), 0.78–0.74 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.6, 157.2, 154.7, 147.8, 145.6, 143.9, 135.7, 132.9, 127.3, 124.0, 115.5, 95.9, 51.8, 51.2, 48.5, 44.1, 15.9, 15.4, 8.6, 7.5. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>21</sub>H<sub>25</sub>ClN<sub>7</sub>, [M + H]<sup>+</sup>, 426.1804; found, 426.1800. <i>R</i><sub>f</sub> = 0.37 (DCM/MeOH, 5/1, v/v). Retention time: 4.10 min, 96.76% purity.</div></div><div id="sec5_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (<i>R</i>)-(6-Chloro-4-((5-cyclopropyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(3-ethylpiperazin-1-yl)methanone Hydrochloride (<b>36</b>)</h4><div class="NLM_p last">Light-yellow solid (yield 19%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.55 (br, 1H), 9.80–9.75 (m, 2H), 9.02 (s, 1H), 8.01 (dd, <i>J</i> = 1.8, 9.0 Hz, 1H), 7.92 (dd, <i>J</i> = 5.6, 8.8 Hz, 1H), 6.47 (d, <i>J</i> = 10.4 Hz, 1H), 4.45 (d, <i>J</i> = 11.0 Hz, 1H), 4.03–3.90 (m, 1H), 3.60–3.01 (m, 5H), 2.03–1.99 (m, 1H), 1.81–1.69 (m, 1H), 1.66–1.44 (m, 1H), 1.05–0.69 (m, 7H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.8, 157.0, 154.7, 147.9, 145.6, 144.2, 135.7, 132.9, 127.5, 124.1, 115.6, 95.8, 56.3, 55.6, 47.5, 43.3, 42.8, 42.6, 38.7, 23.1, 22.8, 10.1, 9.6, 8.7, 7.5. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>21</sub>H<sub>25</sub>ClN<sub>7</sub>, [M + H]<sup>+</sup>, 426.1804; found, 426.1801. <i>R</i><sub>f</sub> = 0.39 (DCM/MeOH, 5/1, v/v). Retention time: 4.11 min, 95.17% purity.</div></div><div id="sec5_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> (<i>R</i>)-(4-((5-Cyclopropyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(3-methylpiperazin-1-yl)methanone Hydrochloride (<b>37</b>)</h4><div class="NLM_p last">Yellow solid (yield 23%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.04 (br, 1H), 9.92 (s, 2H), 8.90 (d, <i>J</i> = 8.3 Hz, 1H), 8.07 (t, <i>J</i> = 7.8 Hz, 1H), 8.00 (d, <i>J</i> = 8.4 Hz, 1H), 7.81 (t, <i>J</i> = 6.9 Hz, 1H), 6.43 (d, <i>J</i> = 4.6 Hz, 1H), 4.49–4.43 (m, 1H), 4.20–4.12 (m, 1H), 3.66–3.10 (m, 5H), 2.06–1.99 (m, 1H), 1.38–1.16 (m, 3H), 1.01–0.99 (m, 2H), 0.79–0.78 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.2, 158.2, 152.6, 147.8, 145.4, 136.5, 129.4, 125.2, 114.0, 96.4, 50.9, 50.4, 48.9, 44.7, 43.1, 42.5, 42.2, 38.7, 15.7, 15.3, 8.7, 7.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>20</sub>H<sub>24</sub>N<sub>7</sub>O, [M + H]<sup>+</sup>, 378.2037; found, 378.2039. <i>R</i><sub>f</sub> = 0.19 (DCM/MeOH, 5/1, v/v). Retention time: 4.05 min, 95.06% purity.</div></div><div id="sec5_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (<i>R</i>)-(7-Chloro-4-((5-cyclopropyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(3-methylpiperazin-1-yl)methanone Hydrochloride (<b>38</b>)</h4><div class="NLM_p last">Off-white solid (yield 24%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.70 (br, 1H), 9.86 (s, 2H), 8.88 (dd, <i>J</i> = 2.4, 9.0 Hz, 1H), 7.99 (s, 1H), 7.81 (dt, <i>J</i> = 2.1, 8.9 Hz, 1H), 6.44 (d, <i>J</i> = 5.4 Hz, 1H), 4.45–4.42 (m, 1H), 4.01–3.92 (m, 1H), 3.59–3.29 (m, 3H), 3.25–3.01 (m, 2H), 2.05–1.99 (m, 1H), 1.37–1.15 (m, 3H), 1.01–0.99 (m, 2H), 0.78–0.77 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.4, 157.6, 154.7, 148.4, 145.1, 140.3, 129.2, 127.2, 124.1, 123.9, 113.3, 95.8, 50.9, 50.4, 48.9, 44.5, 43.0, 42.6, 42.2, 38.4, 15.7, 15.3, 8.9, 7.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>20</sub>H<sub>23</sub>ClN<sub>7</sub>O, [M + H]<sup>+</sup>, 412.1647; found, 412.1645. <i>R</i><sub>f</sub> = 0.19 (DCM/MeOH, 5/1, v/v). Retention time: 4.10 min, 99.63% purity.</div></div><div id="sec5_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (<i>R</i>)-(4-((5-Cyclopropyl-1<i>H</i>-pyrazol-3-yl)amino)-6-fluoroquinazolin-2-yl)(3-methylpiperazin-1-yl)methanone Hydrochloride (<b>39</b>)</h4><div class="NLM_p last">Light yellow solid (yield 13%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.63 (br, 1H), 9.89 (s, 2H), 8.80 (d, <i>J</i> = 9.9 Hz, 1H), 8.02 (dd, <i>J</i> = 5.4, 9.2 Hz, 1H), 7.93 (t, <i>J</i> = 8.3 Hz, 1H), 6.46 (d, <i>J</i> = 5.6 Hz, 1H), 4.46–4.43 (m, 1H), 4.02–3.93 (m, 1H), 3.60–3.29 (m, 3H), 3.27–3.01 (m, 2H), 2.05–2.01 (m, 1H), 1.38–1.15 (m, 3H), 1.01–0.99 (m, 2H), 0.80–0.76 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.2, 158.2, 152.6, 147.8, 145.4, 136.5, 129.4, 125.2, 114.0, 96.4, 50.9, 50.4, 48.9, 44.7, 43.1, 42.5, 42.2, 38.7, 15.7, 15.3, 8.7, 7.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>20</sub>H<sub>23</sub>FN<sub>7</sub>O, [M + H]<sup>+</sup>, 396.1943; found, 396.1944. <i>R</i><sub>f</sub> = 0.22 (DCM/MeOH, 5/1, v/v). Retention time: 4.00 min, 96.25% purity.</div></div><div id="sec5_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (<i>R</i>)-(6-Bromo-4-((5-cyclopropyl-1<i>H</i>-pyrazol-3-yl)amino)quinazolin-2-yl)(3-methylpiperazin-1-yl)methanone Hydrochloride (<b>40</b>)</h4><div class="NLM_p last">Light yellow solid (yield 15%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.46 (br, 1H), 9.78–9.70 (m, 2H), 9.14 (s, 1H), 8.11 (d, <i>J</i> = 9.0, Hz, 1H), 7.84 (d, <i>J</i> = 8.8 Hz, 1H), 6.44 (d, <i>J</i> = 6.7 Hz, 1H), 4.46–4.42 (m, 1H), 3.96–3.88 (m, 1H), 3.56–3.28 (m, 3H), 3.26–2.99 (m, 2H), 2.04–1.97 (m, 1H), 1.37–1.14 (m, 3H), 1.00–0.98 (m, 2H), 0.78–0.76 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.5, 159.1, 157.0, 154.5, 147.9, 145.6, 138.4, 127.3, 121.3, 115.9, 95.9, 50.9, 50.4, 48.9, 44.5, 43.0, 42.6, 42.2, 38.4, 15.7, 15.3, 8.7, 7.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>20</sub>H<sub>23</sub>BrN<sub>7</sub>O, [M + H]<sup>+</sup>, 458.1122; found, 458.1123. <i>R</i><sub>f</sub> = 0.22 (DCM/MeOH, 5/1, v/v). Retention time: 4.10 min, 96.75% purity.</div></div><div id="sec5_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> (<i>R</i>)-(4-((5-Cyclopropyl-1<i>H</i>-pyrazol-3-yl)amino)-6-methoxyquinazolin-2-yl)(3-methylpiperazin-1-yl)methanone Hydrochloride (<b>41</b>)</h4><div class="NLM_p last">Light yellow solid (yield 21%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.01 (br, 1H), 9.85–9.82 (m, 2H), 8.40 (d, <i>J</i> = 2.0 Hz, 1H), 7.95 (d, <i>J</i> = 9.1 Hz, 1H), 7.70 (d, <i>J</i> = 9.1 Hz, 1H), 6.47 (d, <i>J</i> = 4.0 Hz, 1H), 4.50–4.43 (m, 1H), 4.27–4.20 (m, 1H), 3.99 (s, 3H), 3.66–3.33 (m, 3H), 3.27–3.06 (m, 2H), 2.05–1.99 (m, 1H), 1.38–1.15 (m, 3H), 1.01–0.99 (m, 2H), 0.79–0.78 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.3, 159.9, 156. 8, 149.9, 147.6, 146.0, 127.5, 125.4, 115.3, 105.0, 96.0, 57.3, 50.9, 50.4, 49.1, 45.0, 43.3, 42.5, 42.2, 38.9, 15.7, 15.3, 8.7, 7.4. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>21</sub>H<sub>26</sub>N<sub>7</sub>O<sub>2</sub>, [M + H]<sup>+</sup>, 408.2142; found, 408.2145. <i>R</i><sub>f</sub> = 0.19 (DCM/MeOH, 5/1, v/v). Retention time: 4.02 min, 96.12% purity.</div></div><div id="sec5_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 1-Methyl-4-nitro-1<i>H</i>-imidazole (<b>43</b>)</h4><div class="NLM_p last">To a solution of 4-nitro-1<i>H</i>-imidazole (<b>42</b>, 446 mg, 4.0 mmol) in CH<sub>3</sub>CN (20 mL) was added K<sub>2</sub>CO<sub>3</sub> (828 mg, 7.0 mmol) and CH<sub>3</sub>I (0.44 mL, 4.8 mmol). The mixture was stirred at 65 °C for 12 h. After cooling to room temperature (rt), diethylamine (20 mL) was added to quench the reaction. The reaction mixture was filtered. The filtrate was concentrated and purified by chromatography (20% EtOAc/hexanes) to afford <b>43</b> (361 mg, 72% yield) as an off-white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.78 (d, <i>J</i> = 1.4 Hz, 1H), 3.43 (s, 3H). MS (ESI) <i>m</i>/<i>z</i>: 127.7 [M + H]<sup>+</sup>, 149.7 [M + Na]<sup>+</sup>. <i>R</i><sub>f</sub> = 0.47 (DCM/MeOH, 20/1, v/v).</div></div><div id="sec5_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 1-Methyl-1<i>H</i>-imidazol-4-amine Hydrochloride (<b>44a</b>)</h4><div class="NLM_p">To a solution of 1-methyl-4-nitro-1<i>H</i>-imidazole (<b>43</b>, 267 mg, 2.1 mmol) in MeOH (20 mL) was added 10% wt Pd/C (40 mg). The reaction mixture was hydrogenated under hydrogen (1 atm) for 8 h. The Pd/C was removed by filtration, and the filtrate was concentrated to yield 1-methyl-1<i>H</i>-imidazol-4-amine, which was dissolved in EtOH (10 mL) and cooled to 0 °C. A saturated solution of HCl in ethanol (4 mL) was added, and the reaction was stirred at 0 °C for 1 h. The solid was filtered and washed with ethanol to provide <b>44a</b> (179 mg, 64% yield) as an off-white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.37 (d, <i>J</i> = 1.2 Hz, 1H), 7.81 (d, <i>J</i> = 0.7 Hz, 1H), 3.76 (s, 3H).</div><div class="NLM_p last">Compounds <b>44b</b>–<b>d</b> were prepared in a similar manner as compound <b>44a</b>, using <b>47a</b>, <b>45</b>, and <b>47b</b>, respectively, instead of <b>43</b>.</div></div><div id="sec5_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 1-Isopropyl-4-nitro-1<i>H</i>-imidazole (<b>45</b>)</h4><div class="NLM_p last">To a solution of 4-nitro-1<i>H</i>-imidazole (<b>42</b>, 226 mg, 2.0 mmol) in anhydrous DMF (10 mL) was added K<sub>2</sub>CO<sub>3</sub> (552 mg, 4.0 mmol) and 2-bromopropane (0.19 mL, 2.4 mmol). The mixture was stirred at 65 °C overnight. The reaction mixture was filtered, and the filtrate was concentrated and purified by chromatography (20% EtOAc/hexanes) to afford <b>45</b> as an off-white solid (239 mg, 77% yield).</div></div><div id="sec5_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 1,4-Dinitro-1<i>H</i>-imidazole (<b>46</b>)</h4><div class="NLM_p last">To a suspension of 4-nitro-1<i>H</i>-imidazole (<b>42</b>, 500 mg, 4.4 mmol) in AcOH (9 mL) was added HNO<sub>3</sub> (2.2 mL) and Ac<sub>2</sub>O (6 mL) at 0 °C. The reaction mixture was stirred at ambient temperature (<25 °C) for 1 h and poured onto ice (50 g). The reaction solution was then extracted with ethyl acetate (3 × 15 mL). The combined organic extracts were washed with water (3 × 15 mL) and 1 M NaHCO<sub>3</sub> (3 × 15 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated to yield <b>46</b> as an off-white solid (630 mg, 91% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.54 (d, <i>J</i> = 1.6 Hz, 1H), 8.40 (d, <i>J</i> = 1.6 Hz, 1H). <i>R</i><sub>f</sub> = 0.70 (DCM/MeOH, 20/1, v/v).</div></div><div id="sec5_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 1-Cyclopropyl-4-nitro-1<i>H</i>-imidazole (<b>47a</b>)</h4><div class="NLM_p last">Together, 1,4-dinitro-1<i>H</i>-imidazole (<b>46</b>, 630 mg, 4.0 mmol) and cyclopropanamine (0.30 mL, 4.4 mmol) were dissolved in water (12 mL) and MeOH (12 mL). The reaction mixture was stirred at rt for 24 h and diluted with water (30 mL). The reaction mixture was extracted with ethyl acetate (4 × 15 mL). The organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to afford <b>47a</b> as a white solid (390 mg, 64% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.76 (s, 1H), 7.47 (s, 1H), 3.45–3.40 (m, 1H), 1.12–0.99 (m, 4H).</div></div><div id="sec5_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 2,6-Dichloro-<i>N</i>-(1-methyl-1<i>H</i>-imidazol-4-yl)quinazolin-4-amine (<b>51a</b>)</h4><div class="NLM_p last">To a solution of <b>50</b> (465 mg, 2.0 mmol) in 5 mL of anhydrous DMF was added 1-methyl-1<i>H</i>-imidazol-4-amine hydrochloride (<b>44a</b>, 267 mg, 2.0 mmol) and DIEA (0.39 mL, 2.4 mmol) at 0 °C. The reaction was stirred at 0 °C for 3 h and poured into water (50 mL). The solid was filtered and dried to provide the title compound (423 mg, 72% yield) as a light-yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.08 (s, 1H), 8.94 (d, <i>J</i> = 2.2 Hz, 1H), 7.85 (dd, <i>J</i> = 2.2, 8.9 Hz, 1H), 7.70 (d, <i>J</i> = 8.9, 1H), 7.56 (d, <i>J</i> = 1.0 Hz, 1H), 7.55 (d, <i>J</i> = 1.4 Hz, 1H), 3.73 (s, 3H). <i>R</i><sub>f</sub> = 0.22 (DCM/MeOH, 20/1, v/v). MS (ESI) <i>m</i>/<i>z</i> 291.9 (M – H)<sup>−</sup>.</div></div><div id="sec5_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>tert</i>-Butyl (1-(6-Chloro-4-((1-methyl-1<i>H</i>-imidazol-4-yl)amino)quinazolin-2-yl)piperidin-4-yl)carbamate (<b>52a</b>)</h4><div class="NLM_p last">A reaction tube was charged with 2,6-dichloro-<i>N</i>-(1-methyl-1<i>H</i>-imidazol-4-yl)quinazolin-4-amine (<b>51a</b>, 196 mg, 0.67 mmol), <i>tert</i>-butyl piperidin-4-ylcarbamate (268 mg, 1.34 mmol), hydrochloric acid (HCl, 10 μL), and 5 mL of ethanol. The tube was sealed, and the mixture was heated at 120 °C for 15 h. After cooling, the reaction was concentrated and purified by silica gel chromatography (2% MeOH/DCM) to provide the title compound (221 mg, 72% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.20 (s, 1H), 8.62 (d, <i>J</i> = 2.1 Hz, 1H), 7.53–7.51 (m, 2H), 7.33 (s, 1H), 7.30 (d, <i>J</i> = 8.9 Hz, 1H), 6.85 (d, <i>J</i> = 7.8 Hz, 1H), 4.67–4.64 (m, 2H), 3.70 (s, 3H), 3.57–3.54 (m, 1H), 3.04 (t, <i>J</i> = 11.7 Hz, 2H), 1.84–1.82 (m, 2H), 1.39 (s, 9H), 1.32–1.23 (m, 2H). <i>R</i><sub>f</sub> = 0.25 (DCM/MeOH, 20/1, v/v). MS (ESI) <i>m</i>/<i>z</i> 458.3 (M + H)<sup>+</sup>.</div></div><div id="sec5_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 2-Amino-5-chlorobenzamide (<b>56</b>)</h4><div class="NLM_p last">A mixture of 2-amino-5-chlorobenzoic acid (<b>48</b>, 5.0 g, 29.1 mmol) and bis(trichloromethyl) carbonate (2.9 g, 9.9 mmol) in 50 mL of anhydrous THF was refluxed for 16 h. After cooling, the THF was removed with a vacuum. The reaction mixture was diluted with 233 mL of 1 N NH<sub>3</sub>–H<sub>2</sub>O solution. After stirring for 1 h at 65 °C, the reaction mixture was cooled to 0 °C, and the precipitated solid was collected via filtration to provide <b>56</b> (4.23 g, 85% yield) as a white solid. After a high-vacuum-dry, the product was used directly for the next step. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.84 (s, 1H), 7.59 (d, <i>J</i> = 2.5 Hz, 1H), 7.19 (s, 1H), 7.15 (dd, <i>J</i> = 2.5, 8.8 Hz, 1H), 6.71 (d, <i>J</i> = 8.8 Hz, 1H), 6.39 (br, 2H). <i>R</i><sub>f</sub> = 0.38 (DCM/MeOH, 30/1, v/v, 0.5% CH<sub>3</sub>COOH). mp 169–170 °C. MS (ESI) <i>m</i>/<i>z</i> 168.8 (M – H)<sup>−</sup>.</div></div><div id="sec5_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> Ethyl 2-((2-Carbamoyl-4-chlorophenyl)amino)-2-oxoacetate (<b>57</b>)</h4><div class="NLM_p last">To a solution of 2-amino-5-chlorobenzamide (<b>56</b>, 3.0 g, 17.6 mmol) and triethylamine (2.9 mL, 21.1 mmol) in 110 mL of anhydrous THF was added ethyl 2-chloro-2-oxoacetate (2.1 mL, 19.4 mmol) at 0 °C. The reaction mixture was stirred at rt for 3 h and diluted with water (300 mL). The solid was filtered and washed with water to provide <b>57</b> (4.46 g, 94% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.02 (s, 1H), 8.56 (d, <i>J</i> = 2.5, 1H), 8.47 (s, 1H), 7.80 (s, 1H), 7.97 (d, <i>J</i> = 2.4, 1H), 7.66 (dd, <i>J</i> = 2.4, 9.0 Hz, 1H), 4.31 (q, <i>J</i> = 7.1, 1H), 1.32 (t, <i>J</i> = 7.1, 1H). <i>R</i><sub>f</sub> = 0.41 (DCM/MeOH, 30/1, v/v, 0.5% CH<sub>3</sub>COOH). mp 201–202 °C. MS (ESI) <i>m</i>/<i>z</i> 293.0 (M + Na)<sup>+</sup>.</div></div><div id="sec5_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> Ethyl 6-Chloro-4-oxo-3,4-dihydroquinazoline-2-carboxylate (<b>58</b>)</h4><div class="NLM_p last">To a solution of ethyl 2-((2-carbamoyl-4-chlorophenyl)amino)-2-oxoacetate (<b>57</b>, 2.70 g, 10.0 mmol) in ethanol (54 mL) was added an ethanol solution (8.2 mL) of sodium ethanolate (0.82 g, 12.0 mmol) at 0 °C. The reaction mixture was stirred at 0 °C for 3 h and was acidified to pH = 3–4 with a 2 N HCl solution. The solid was filtered and washed with water to provide <b>58</b> (2.17 g, 86% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.83 (s, 1H), 8.10 (d, <i>J</i> = 2.4 Hz, 1H), 7.92 (dd, <i>J</i> = 2.4, 8.7 Hz, 1H), 7.85 (d, <i>J</i> = 8.7 Hz, 1H), 4.39 (q, <i>J</i> = 7.1 Hz, 2H), 1.35 (t, <i>J</i> = 7.1 Hz, 3H). <i>R</i><sub>f</sub> = 0.62 (DCM/MeOH, 30/1, v/v, 0.5% CH<sub>3</sub>COOH). mp 267–268 °C. MS (ESI) <i>m</i>/<i>z</i> 275.1(M + Na)<sup>+</sup>, 250.8 (M – H)<sup>−</sup>.</div></div><div id="sec5_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 6-Chloro-4-oxo-3,4-dihydroquinazoline-2-carboxylic Acid (<b>59</b>)</h4><div class="NLM_p last">To a solution of ethyl 6-chloro-4-hydroxyquinazoline-2-carboxylate (<b>58</b>, 6.0 g, 23.8 mmol) in ethanol (80 mL) and water (80 mL) was added NaOH (7.8 g, 195 mmol). The reaction mixture was stirred at 78 °C in an oil bath for 1 h. After cooling, the reaction mixture was acidified to pH = 2 with a 2 N HCl solution. The solid was filtered and washed with water to provide <b>59</b> (5.11 g, 96% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.90 (s, 1H), 7.69 (d, <i>J</i> = 9.1 Hz, 1H), 7.55 (s, 1H). <i>R</i><sub>f</sub> = 0.30 (DCM/MeOH, 30/1, v/v, 0.5% CH<sub>3</sub>COOH). MS (ESI) <i>m</i>/<i>z</i> 222.8 (M – H)<sup>−</sup>. mp 242 °C.</div></div><div id="sec5_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 4,6-Dichloroquinazoline-2-carbonyl Chloride (<b>60</b>)</h4><div class="NLM_p last">Sulfurous dichloride (0.77 mL, 10.7 mmol) was added to a solution of 6-chloro-4-hydroxyquinazoline-2-carboxylic acid (<b>59</b>, 200 mg, 0.89 mmol) and DMF (2 drop) in chloroform (4 mL). The reaction mixture was refluxed for 3 h. After the completion of the reaction as indicated by TLC, the reaction mixture was evaporated <i>in vacuo</i> to obtain a light-yellow solid (<b>60</b>, 233 mg, quantitative yield), which was used for the next step without further purification.</div></div><div id="sec5_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> <i>tert</i>-Butyl 4-(4,6-dichloroquinazoline-2-carbonyl)piperazine-1-carboxylate (<b>65a</b>)</h4><div class="NLM_p last">To a solution of freshly prepared 4,6-dichloroquinazoline-2-carbonyl chloride (<b>60</b>, 233 mg, 0.89 mmol) and triethylamine (0.37 mL, 2.67 mmol) in anhydrous dichloromethane (10 mL) was added <i>tert</i>-butyl piperazine-1-carboxylate (166 mg, 0.89 mmol) at −35 °C. The resulting solution was stirred at −35 °C for 0.5 h and quenched with water (10 mL). The reaction mixture was extracted with dichloromethane (3 × 15 mL). The combined organic extracts were collected, dried, and concentrated under reduced pressure to provide a crude mixture, which was purified by silica gel chromatography (2% MeOH/DCM) to provide the title compound (238 mg, 65% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.32 (d, <i>J</i> = 2.1 Hz, 1H), 8.08 (d, <i>J</i> = 9.0 Hz, 1H), 7.97 (dd, <i>J</i> = 2.3, 9.0 Hz, 1H), 3.85 (t, <i>J</i> = 4.9 Hz, 2H), 3.61 (t, <i>J</i> = 5.4 Hz, 2H), 3.53–3.51 (m, 2H), 3.45–3.42 (m, 2H), 1.49 (s, 9H). mp 301 °C.</div></div><div id="sec5_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> <i>tert</i>-Butyl 4-(6-chloro-4-((5-methyl-1<i>H</i>-pyrazol-3-yl)amino)quinazoline-2-carbonyl)piperazine-1-carboxylate (<b>66a</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl 4-(4,6-dichloroquinazoline-2-carbonyl)piperazine-1-carboxylate (<b>65a</b>, 145 mg, 0.35 mmol), KI (67.4 mg, 0.41 mmol), and DIEA (0.08 mL, 0.45 mmol) in 5 mL of anhydrous DMF was added 5-methyl-1<i>H</i>-pyrazol-3-amine (36.9 mg, 0.38 mmol). The resulting solution was stirred at 65 °C for 8 h and diluted with water (50 mL). The precipitated solid was collected via filtration and purified by silica gel chromatography (2% MeOH/DCM) to provide the title compound (105 mg, 63% yield) as an off-white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.28 (s, 1H), 10.71 (s, 1H), 8.89 (s, 1H), 7.89 (dd, <i>J</i> = 2.0, 8.9 Hz, 1H), 7.80 (d, <i>J</i> = 8.9 Hz, 1H), 6.57 (s, 1H), 3.65–3.62 (m, 2H), 3.49–3.40 (m, 2H), 3.29–3.28 (m, 4H), 2.27 (s, 3H), 1.41 (s, 9H). mp 290 °C.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> PAK4 Expression, Purification, and Crystallization</h3><div class="NLM_p">The cDNA encoding the kinase domain (resides 300 to 591) of human PAK4 was cloned into a modified pET28b vector with an <i>N</i>-terminal 10× His tag followed by a SUMO tag. The recombinant protein was overexpressed in <i>E. coli</i> BL21 (DE3). After overnight induction with 0.2 mM isopropyl β-<span class="smallcaps smallerCapital">d</span>-thiogalactoside (IPTG) at 16 °C in LB medium, the cells were harvested and suspended in buffer (300 mM NaCl, 20 mM Tris, pH 7.5). Then, the cells were lysed with an Emulsiflex C3 (Avestin) high-pressure homogenizer. After centrifugation at ×32,000<i>g</i>, the supernatant was applied to a HisTrap column (GE Healthcare) and subjected to ion-exchange chromatography (HiTrap Q HP). Then, the resultant protein was digested overnight by ULP1 protease. The postcleavage mixture was purified with a fresh HisTrap column to remove His-SUMO tags and further purified using a Superdex 200 gel-filtration column (GE Healthcare). The purified protein was concentrated to 8 mg/mL and stored at −80 °C for future use.</div><div class="NLM_p last">The PAK4 kinase domain and inhibitors <b>10a</b>, <b>30</b>, and <b>31</b> were mixed at a molar ratio of 1:2. After incubation at 4 °C for 1 h, crystallization screens were performed using an Art Robbins Gryphon crystallization robot by mixing equal volumes of PAK4–inhibitor complex with different screening conditions from commercial crystallization kits. Crystals were obtained under the following buffer conditions: 25% PEG3350, 0.1 M HEPES-Na, pH 7.5 for the <b>10a</b> complex; 1.0 M sodium acetate trihydrate, 0.1 M imidazole, pH 6.5 for the <b>30</b> complex; and 0.5 M magnesium formate dehydrate, 0.1 M HEPES-Na, pH 7.5 for the <b>31</b> complex, respectively.</div></div><div id="sec5_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> Data Collection and Structure Determination</h3><div class="NLM_p last">The crystals were briefly soaked in a cryo-protectant composed of reservoir solution supplemented with 20% ethylene glycol and flash frozen in liquid nitrogen for data collection at 100 K. Data collection was performed at beamline BL17U1 at the Shanghai Synchrotron Radiation Facility. The diffraction data were indexed, integrated, and merged using the HKL2000 software package (<a href="http://www.hkl-xray.com" class="extLink">http://www.hkl-xray.com</a>). The structures were solved using the molecular replacement method with the published PAK4 kinase domain structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2J0I">2J0I</a>) as a search model. Structure refinement was performed using PHENIX,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and iterative manual model building was performed using COOT.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> PAK1 and PAK4 Biochemical Assay To Measure the <i>K</i><sub>i</sub></h3><div class="NLM_p last">In a polypropylene plate, PAK1 (kinase domain) and PAK4 (kinase domain), peptide substrate (Ser/Thr19, Ser/Thr20) labeled with both coumarin (donor fluorophore) and fluorescein (acceptor fluorophore), ATP (<i>K</i><sub>m</sub>), and the test compound were incubated together in kinase reaction buffer. The 10 μL assay mixtures contained 50 mM HEPES (pH 7.5), 0.01% Brij-35, 10 mM MgCl<sub>2</sub>, 1 mM EGTA, 2 μM FRET peptide substrate, and PAK enzyme (20 pM PAK1 KD; 80 pM PAK4 KD). In all cases, each PAK biochemical assay used ATP at a final concentration equal to its <i>K</i><sub>m apparent</sub>. Incubations were carried out at 22 °C. Prior to the assay, the enzyme, FRET peptide substrate, and serially diluted test compounds were preincubated together in assay buffer (7.5 μL) for 15 min, and the assay was initiated by the addition of 2.5 μL of assay buffer containing 4× the final ATP concentration (these are the 4× concentrations: 160 μM PAK1 and 16 μM PAK4). Following a 60 min incubation, the assay mixtures were quenched by the addition of 5 μL of Z′-Lyte development reagent, and 1 h later, the emissions of coumarin (445 nm) and fluorescein (520 nm) were determined after excitation at 400 nm using an EnVision plate reader (PerkinElmer). The high ratio of coumarin/fluorescein represents a 0% phosphorylation rate, while a low ratio of coumarin/fluorescein represents 100% phosphorylation rate. The curves were fit by XLfit5 as % inhibition vs log [compound concentration] using a four-parameter logistic model, model 205. The <i>K</i><sub>i</sub> values reported in this article were calculated by converting the IC<sub>50</sub> value (generated using a nonlinear least-squares fit of the concentration–response data to a four-parameter equation) to the <i>K</i><sub>i</sub> value by employing the Cheng–Prusoff equation with the XLFIT5 software.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> Truncated PAK4 Kinase Expression</h3><div class="NLM_p last">The truncated PAK4 kinase domain was expressed and purified as reported earlier.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The purified protein was subsequently used for biophysical assay, DSF, and SPR.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> Differential Scanning Fluorimetry (DSF)</h3><div class="NLM_p">DSF screening experiments were carried out using a QuantStudio 6 Flex Real-time PCR System (Applied Biosystems). The detection filter was chosen for SYPRO Orange (λ<sub>excitation</sub> = 550 ± 11 nm, λ<sub>emission</sub> = 586 ± 10 nm), and the scanning temperature was set from 25 to 90 °C at a ramp rate of 0.5 °C/min.</div><div class="NLM_p">The truncated PAK4 protein was diluted to 0.1 mg/mL (2.7 μM) in HEPES buffer (50 mM HEPES, 150 mM NaCl, 0.02% P20, 1 mM ATP, and 5 mM MgCl<sub>2</sub>) containing (10×) SYPRO Orange (Thermo Fisher Scientific). The diluted protein solution was added to a 96-well clear PCP plate (Bio-Rad) on ice, with 25 μL of the solution per well. The inhibitor compounds were dissolved in DMSO at a concentration of 10 mM as a stock solution. Prior to use, the stock solutions were diluted to 1 mM in DMSO, and 0.5 μL (2% v/v) of the compound solution was dropped to each well for DSF assay. Final concentration of the PAK4 protein is 2.7 μM, and concentration of compound is 20 μM. DMSO was used as the reference solvent. After sealing with Optical Adhesive Film, the plate was gently vortexed, centrifuged, and analyzed in the Real-Time PCR System. The heating cycle included a 2 min prewarming step at 25 °C and subsequent gradient between 25 and 90 °C, each of 0.5 °C ramp.</div><div class="NLM_p last">All of reactions were performed in triplicate. Analysis of the data was primarily performed by ocular inspection of individual curves, and fitting <i>T</i><sub>m</sub> to sigmoidal transitions, where applicable, was performed using an Excel script.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> Surface Plasmon Resonance (SPR)</h3><div class="NLM_p">SPR biosensing experiments were performed using a four-channel Biacore T200 instrument (GE Healthcare, Sweden). Streptavidin-functionalized sensor chips (Series S SA chip, GE Healthcare) were chosen for PAK4 immobilization via a biotin–streptavidin capture-coupling protocol. HBS-N buffer (10 mM HEPES, 150 mM NaCl, pH 7.4, GE Healthcare) supplemented with 5 mM MgCl<sub>2</sub> was used as running buffer for the protein immobilization. All binding experiments were conducted in HBS-N buffer supplemented with 5 mM MgCl<sub>2</sub> and 0–5% of DMSO at 22 °C.</div><div class="NLM_p last">Briefly, the truncated biotin-PAK4 (20 μg/mL) diluted in the immobilization buffer was introduced onto a streptavidin sensor chip with a flow rate of 10 μL/min, and a response target set at 2000 RU. This procedure led to a site-specific (C-terminal), stable, and robust PAK4 layer of 1800–2200 RU, which was used for ligand–kinase protein interaction analysis. For the kinetic binding analysis, the T200 standard method LMW kinetic was chosen. The contact time was set at 60 s and dissociation time of 120–300 s at a flow rate of 30 μL/min. Data processing and analysis was performed using Biacore T200 evaluation software (GE Healthcare). Prior to the kinetic fitting, double referencing was applied by subtracting a reference signal and a blank injection signal.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Cell Viability Assays</h3><div class="NLM_p last">The viability of cells was determined using the MTT assay method as previously described.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Wound Healing Assay</h3><div class="NLM_p last">For the wound healing assay, A549 cells were cultured to confluence in six-well plates and wounded using a sterilized pipet tip to make a straight scratch. The wounded cell monolayers were washed three times with phosphate-buffered saline (PBS) and incubated in serum-free medium. Then cells were treated with compound <b>31</b> with different concentrations followed by a 48 h incubation and photographed at 24 and 48 h with an inverted microscope (AX-70; Olympus).</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> Cell Migration and Invasion Assays</h3><div class="NLM_p last">Migration and invasion assays were performed using modified Boyden chambers with a polycarbonate nucleopore membrane, as previously described.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> Western Blot Analysis</h3><div class="NLM_p last">To determine the expression levels of proteins, whole cell extracts were prepared from 1 × 10<sup>6</sup> cells in RIPA lysis buffer (50 mM Tris HCl at pH 7.4, 150 mM NaCl, 1% Nonidet P-40, 0.25% Na-deoxycholate, 1 mM EDTA, and a protease inhibitor cocktail). Equal amounts of denatured protein were separated by SDS-PAGE and transferred to a PVDF membrane (Millipore). The membrane was blocked with 5% nonfat dry milk in TBS-T (20 mM Tris at pH 7.4, 137 mM NaCl, and 0.05% Tween-20) for 3 h at room temperature, and the proteins were probed with antibodies specific to phospho-cofilin, cofilin, LIMK1, phospho-LIMK1, PAK4, phospho-PAK4/Ser474, phospho-GEF-H1, GEF-H1, and MMP-2. All PVDF membranes were detected by chemiluminescence (ECL, Pierce Technology). To ensure equal loading, membranes were stripped and reprobed with antibody against GAPDH (Shang Hai Kangchen).</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Liver Microsomal Stability Assay</h3><div class="NLM_p last">These measurements were performed as previously described.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Two parallel determinations in microsomes, with and without the NADPH regenerating system, were performed for selected compounds. Briefly, the compounds were preincubated with microsomes (human microsome, BD Gentest, lot No. 38292; rat microsome, Xenotech, Lot No. 1310030) (0.5 mg/mL) at 1 μM for 10 min at 37 °C in potassium phosphate buffer (100 mM at pH 7.4 with 10 mM MgCl<sub>2</sub>). The reactions were initiated by adding prewarmed cofactors (1 mmol NADPH). After incubation for different times (0, 5, 10, 20, 30, and 60 min) at 37 °C, cold acetonitrile was added to precipitate the protein. Then, the samples were centrifuged, and the supernatants were analyzed by LC–MS/MS.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> Stability Assay in Rat Blood Plasma</h3><div class="NLM_p last">Compound <b>31</b> was incubated in a concentration of 2 μM in SD rat blood plasma (BioreclamationIVT, batch No. RAT303151) at 37 °C. Samples were taken at the indicated time points during the incubation (0, 10, 30, 60, and 120 min in duplicate). The reactions terminated with the addition of 400 μL of cold stop solution (200 ng/mL tolbutamide and 200 ng/mL labetalol in 50% MeOH/acetonitrile). The plates were centrifuged (4000 rpm for 10 min), and the supernatants were analyzed by LC–/MS/MS.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Cytochrome P450 Inhibition Assay</h3><div class="NLM_p last">Cytochrome P450 inhibition was evaluated in human liver microsomes (0.25 mg/mL) using five specific probe substrates (CYP1A2, 10 μM phenacetin; CYP2C9, 5 μM diclofenac; CYP2C19, 30 μM <i>S</i>-mephenytoin; CYP2D6, 5 μM dextromethorphan; and CYP3A4, 2 μM midazolam) in the presence of multiple concentrations of the test compound (0.05–50 μM). After preincubation at 37 °C for 10 min, the reaction was initiated by the addition of 20 μL of NADPH to a final concentration of 10 mM. The mixture was incubated at 37 °C for 10 min, and the reaction was terminated by the addition of 400 μL of cold stop solution (200 ng/mL tolbutamide and 200 ng/mL labetalol in acetonitrile). After the reactions were terminated, the plates were centrifuged, and the supernatants were analyzed by LC–MS/MS.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> Inhibition Evaluation on the hERG K<sup>+</sup> Channel</h3><div class="NLM_p last">A detailed characterization is provided in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> Physiochemical Properties</h3><div class="NLM_p last">The physicochemical properties of the compounds were predicted using QikProp in Maestro software package. The p<i>K</i><sub>a</sub>, LogP, and LogD values and solubility profile of compound <b>31</b> were determined experimentally by Sirius Analytical (UK) via UV-metric or potentiometric (pH-metric) methods (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for full reports).</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Pharmacokinetic Assessments in SD Rats</h3><div class="NLM_p last">Single-dose pharmacokinetic studies on compound <b>31</b> were performed using male Sprague–Dawley rats (<i>n</i> = 3 per group) in full compliance with the Guide for the Care and Use of Laboratory Animals. After intravenous injection with the compound (2 mg/kg), blood samples were collected in heparin-containing tubes at predetermined time points (0.083, 0.25, 0.5, 1, 2, 4, 8, and 24 h) and centrifuged immediately at 4 °C and 4000 rpm for 20 min. Noncompartmental pharmacokinetic parameters were fitted using phoenix WinNonlin 6.3 based on the LC–MS/MS quantitation data.</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> Statistical Analysis</h3><div class="NLM_p last">Significance was analyzed with Student’s <i>t</i> test, and the results were considered statistically significant at <i>p</i> < 0.05 (notation: *, <i>p</i> < 0.05; **, <i>p</i> < 0.01; ***, <i>p</i> < 0.001).</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Molecular Modeling</h3><div class="NLM_p last">Ensemble docking was performed with AutoDock4 into the predefined kinase ATP-binding pocket of PAKs. Hydrogens were added to the modeled kinase domain (PAK4 PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X4Z">2X4Z</a>; PAK1 PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O0R">4O0R</a>), and partial atomic charges were assigned using AutoDockTools (ADT). The ligand coordinates were generated using the Corina server (<a href="http://www.mn-am.com/online_demos/corina_demo" class="extLink">www.mn-am.com/online_demos/corina_demo</a>). The ligand was placed in the kinase ATP-binding pocket and aligned manually to avoid atom clashes. A 3D grid box (dimensions = 50 × 50 × 50 units in number of grid points; grid spacing = 0.375 Å) centered at the ATP-binding pocket was created using AutoGrid4. Docking was performed using the Lamarckian genetic algorithm in AutoDock4. Each docking experiment was performed 10 times, yielding 10 docked conformations. The solutions were ranked by the calculated binding free energy. Figures were drawn using PyMOL.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i91"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01342">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58230" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58230" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01342" class="ext-link">10.1021/acs.jmedchem.7b01342</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Sequence alignment of the active site residues of PAK1–6; crystallographic parameters of all X-ray structures (cocrystal structures of <b>10a</b>, <b>30</b>, and <b>31</b> bound to PAK4); comparison of the orientation differences of the DFG Asp (Asp458<sup>PAK4</sup>/Asp407<sup>PAK1</sup>); binding mode analysis of compounds <b>12</b>, <b>17</b>, <b>18</b>, and <b>20</b>; detailed kinase selectivity data of compound <b>31</b>; <sup>1</sup>H NMR, <sup>13</sup>C NMR, and HRMS spectra of compounds <b>10a</b>–<b>d</b>, <b>11a</b>–<b>d</b>, and <b>12</b>–<b>41</b>; liver microsome stability, rat plasma stability, physiochemical properties determination, CYP450, and hERG inhibition reports of compound <b>31</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01342/suppl_file/jm7b01342_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and associated biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01342/suppl_file/jm7b01342_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01342/suppl_file/jm7b01342_si_001.pdf">jm7b01342_si_001.pdf (8.69 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01342/suppl_file/jm7b01342_si_002.csv">jm7b01342_si_002.csv (4.13 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB ID codes for <b>10a</b>, <b>30</b>, and <b>31</b> bound to PAK4 are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVF">5XVF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVA">5XVA</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVG">5XVG</a>, respectively.</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b01342" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48111" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48111" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Haitao Li</span> - <span class="hlFld-Affiliation affiliation">Tsinghua-Peking
Joint Center for Life Sciences, Beijing Advanced Innovation Center
for Structural Biology, Department of Basic Medical Sciences, School
of Medicine, Tsinghua University, Beijing 100084, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#54383c201420273d3a333c21357a3130217a373a"><span class="__cf_email__" data-cfemail="7915110d390d0a10171e110c18571c1d0c571a17">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Feng Li</span> - <span class="hlFld-Affiliation affiliation">Department
of Cell Biology, Key Laboratory of Cell Biology, Ministry of Public
Health, and Key Laboratory of Medical Cell Biology, Ministry of Education, China Medical University, Shenyang 110001, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#aec8c2c7eec3cfc7c280cdc3db80cbcadb80cdc0"><span class="__cf_email__" data-cfemail="a5c3c9cce5c8c4ccc98bc6c8d08bc0c1d08bc6cb">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dongmei Zhao</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f6929998919b939f8cdbc0c1b6c7c0c5d895999b"><span class="__cf_email__" data-cfemail="b6d2d9d8d1dbd3dfcc9b8081f687808598d5d9db">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maosheng Cheng</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#fd908e9e9598939abd8e848d9588d3989988d39e93"><span class="__cf_email__" data-cfemail="48253b2b202d262f083b3138203d662d2c3d662b26">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chenzhou Hao</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6803-7270" title="Orcid link">http://orcid.org/0000-0001-6803-7270</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fan Zhao</span> - <span class="hlFld-Affiliation affiliation">Tsinghua-Peking
Joint Center for Life Sciences, Beijing Advanced Innovation Center
for Structural Biology, Department of Basic Medical Sciences, School
of Medicine, Tsinghua University, Beijing 100084, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongyan Song</span> - <span class="hlFld-Affiliation affiliation">Institute
of Materials Research and Engineering, A*STAR
(Agency for Science, Technology and Research), 2 Fusionopolis Way, Innovis, #08-03, Singapore 138634</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Guo</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaodong Li</span> - <span class="hlFld-Affiliation affiliation">Department
of Cell Biology, Key Laboratory of Cell Biology, Ministry of Public
Health, and Key Laboratory of Medical Cell Biology, Ministry of Education, China Medical University, Shenyang 110001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaolin Jiang</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ran Huan</span> - <span class="hlFld-Affiliation affiliation">Department
of Cell Biology, Key Laboratory of Cell Biology, Ministry of Public
Health, and Key Laboratory of Medical Cell Biology, Ministry of Education, China Medical University, Shenyang 110001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shuai Song</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qiaoling Zhang</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ruifeng Wang</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kai Wang</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu Pang</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tongchao Liu</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tianqi Lu</span> - <span class="hlFld-Affiliation affiliation">Department
of Cell Biology, Key Laboratory of Cell Biology, Ministry of Public
Health, and Key Laboratory of Medical Cell Biology, Ministry of Education, China Medical University, Shenyang 110001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wanxu Huang</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Wang</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bin Lin</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Structure-Based Drug Design & Discovery of Ministry
of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhonggui He</span> - <span class="hlFld-Affiliation affiliation">School of
Pharmacy, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>C.H., F.Z., and H.S. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d78e6737-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i94">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65804" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65804" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the staff members at beamline BL17U of the Shanghai Synchrotron Radiation Facility and Dr. S. Fan at the Beijing Advanced Innovation Center for Structural Biology for their assistance in data collection. We gratefully acknowledge the financial support from the National Natural Science Foundation of China (Grant 81230077) and Program for Innovative Research Team of the Ministry of Education, and Program for Liaoning Innovative Research Team in University. H.S. is grateful to the Agency for Science, Technology and Research (A*STAR), for its financial support (JCO DP Grant: 1334k00085). H.L. is supported by the National Natural Science Foundation of China (Grant 31470720) and the Tsinghua University Initiative Scientific Research Program.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i95" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i95"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i96" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i96"> ABBREVIATIONS USED</h2><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adenosine triphosphate</p></td></tr><tr><td class="NLM_term">BRAF</td><td class="NLM_def"><p class="first last">serine/threonine-protein kinase B-raf (raf, rapidly accelerated fibrosarcoma)</p></td></tr><tr><td class="NLM_term">CHO</td><td class="NLM_def"><p class="first last">Chinese hamster ovary</p></td></tr><tr><td class="NLM_term">CL</td><td class="NLM_def"><p class="first last">clearance</p></td></tr><tr><td class="NLM_term">DFG</td><td class="NLM_def"><p class="first last">Asp-Phe-Gly</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorometry</p></td></tr><tr><td class="NLM_term">FLT3</td><td class="NLM_def"><p class="first last">fms-like tyrosine kinase</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-a-go-go related gene</p></td></tr><tr><td class="NLM_term">KD</td><td class="NLM_def"><p class="first last">kinase domain</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">LLE</td><td class="NLM_def"><p class="first last">ligand lipophilicity efficiency</p></td></tr><tr><td class="NLM_term">NAMPT</td><td class="NLM_def"><p class="first last">nicotinamide phosphoribosyl-transferase</p></td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">nonsmall cell lung cancer</p></td></tr><tr><td class="NLM_term">PAK</td><td class="NLM_def"><p class="first last">p21-activated kinase</p></td></tr><tr><td class="NLM_term">RAS</td><td class="NLM_def"><p class="first last">rat sarcoma</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i97">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10543" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10543" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 46 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Manser, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salihuddin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, L.</span><span> </span><span class="NLM_article-title">A brain serine/threonine protein kinase activated by Cdc42 and Rac1</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">367</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">46</span><span class="refDoi"> DOI: 10.1038/367040a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1038%2F367040a0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=367&publication_year=1994&pages=40-46&author=E.+Manserauthor=T.+Leungauthor=H.+Salihuddinauthor=Z.+S.+Zhaoauthor=L.+Lim&title=A+brain+serine%2Fthreonine+protein+kinase+activated+by+Cdc42+and+Rac1&doi=10.1038%2F367040a0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F367040a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F367040a0%26sid%3Dliteratum%253Aachs%26aulast%3DManser%26aufirst%3DE.%26aulast%3DLeung%26aufirst%3DT.%26aulast%3DSalihuddin%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DZ.%2BS.%26aulast%3DLim%26aufirst%3DL.%26atitle%3DA%2520brain%2520serine%252Fthreonine%2520protein%2520kinase%2520activated%2520by%2520Cdc42%2520and%2520Rac1%26jtitle%3DNature%26date%3D1994%26volume%3D367%26spage%3D40%26epage%3D46%26doi%3D10.1038%2F367040a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Radu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semenova, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosoff, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chernoff, J.</span><span> </span><span class="NLM_article-title">PAK signalling during the development and progression of cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">13</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1038/nrc3645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1038%2Fnrc3645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=24505617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOmurnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=13-25&author=M.+Raduauthor=G.+Semenovaauthor=R.+Kosoffauthor=J.+Chernoff&title=PAK+signalling+during+the+development+and+progression+of+cancer&doi=10.1038%2Fnrc3645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">PAK signalling during the development and progression of cancer</span></div><div class="casAuthors">Radu, Maria; Semenova, Galina; Kosoff, Rachelle; Chernoff, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-25</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  P21-activated kinases (PAKs) are positioned at the nexus of several oncogenic signalling pathways.  Overexpression or mutational activation of PAK isoforms frequently occurs in various human tumors, and recent data suggest that excessive PAK activity drives many of the cellular processes that are the hallmarks of cancer.  In this Review, we discuss the mechanisms of PAK activation in cancer, the key substrates that mediate the developmental and oncogenic effects of this family of kinases, and how small-mol. inhibitors of these enzymes might be best developed and deployed for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoDp4TQ1JV7bVg90H21EOLACvtfcHk0lhpDbwDhxpdEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOmurnJ&md5=52331f8a81941b5606ba25130ffef72f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc3645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3645%26sid%3Dliteratum%253Aachs%26aulast%3DRadu%26aufirst%3DM.%26aulast%3DSemenova%26aufirst%3DG.%26aulast%3DKosoff%26aufirst%3DR.%26aulast%3DChernoff%26aufirst%3DJ.%26atitle%3DPAK%2520signalling%2520during%2520the%2520development%2520and%2520progression%2520of%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D13%26epage%3D25%26doi%3D10.1038%2Fnrc3645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Arias-Romero, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chernoff, J.</span><span> </span><span class="NLM_article-title">A tale of two Paks</span> <span class="citation_source-journal">Biol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">97</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span><span class="refDoi"> DOI: 10.1042/BC20070109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1042%2FBC20070109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=18199048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD1cXosFarsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2008&pages=97-108&author=L.+E.+Arias-Romeroauthor=J.+Chernoff&title=A+tale+of+two+Paks&doi=10.1042%2FBC20070109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A tale of two Paks</span></div><div class="casAuthors">Arias-Romero, Luis E.; Chernoff, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Biology of the Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">97-108</span>CODEN:
                <span class="NLM_cas:coden">BCELDF</span>;
        ISSN:<span class="NLM_cas:issn">0248-4900</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  Paks (p21-activated kinases) are effectors for the small GTPases Cdc42 (cell division cycle 42) and Rac that play fundamental roles in a wide range of cellular processes, including cell morphol., motility, survival, gene transcription, apoptosis and hormone signalling.  These enzymes are widely expressed in numerous tissues and are activated by extracellular signals through GTPase-dependent and -independent mechanisms.  On the basis of structural and biochem. features, the Pak family members have been classified into two groups, comprising three members each.  The two Pak groups have generally been considered as two halves of a single entity, but accumulating evidence indicates that this is not so.  In this review, we discuss recent advances in our understanding of the structure, regulation and function of these kinases, emphasizing the many differences between these two groups of signalling proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJrFXNyCcF47Vg90H21EOLACvtfcHk0lhpDbwDhxpdEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXosFarsA%253D%253D&md5=9cb3271621a5d2551dfc02bec8a2759c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1042%2FBC20070109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBC20070109%26sid%3Dliteratum%253Aachs%26aulast%3DArias-Romero%26aufirst%3DL.%2BE.%26aulast%3DChernoff%26aufirst%3DJ.%26atitle%3DA%2520tale%2520of%2520two%2520Paks%26jtitle%3DBiol.%2520Cell%26date%3D2008%26volume%3D100%26spage%3D97%26epage%3D108%26doi%3D10.1042%2FBC20070109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Ye, D. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, J.</span><span> </span><span class="NLM_article-title">PAK signaling in cancer</span> <span class="citation_source-journal">Cell. Logist.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">116</span><span class="refDoi"> DOI: 10.4161/cl.21882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.4161%2Fcl.21882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=23162742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BC2sris1Wrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=105-116&author=D.+Z.+Yeauthor=J.+Field&title=PAK+signaling+in+cancer&doi=10.4161%2Fcl.21882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">PAK signaling in cancer</span></div><div class="casAuthors">Ye Diana Zi; Field Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Cellular logistics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-116</span>
        ISSN:<span class="NLM_cas:issn">2159-2780</span>.
    </div><div class="casAbstract">Transformation of a normal cell to a cancer cell is caused by mutations in genes that regulate proliferation, apoptosis, and invasion.  Small GTPases such as Ras, Rho, Rac and Cdc42 orchestrate many of the signals that are required for malignant transformation.  The p21-activated kinases (PAKs) are effectors of Rac and Cdc42.  PAKs are a family of serine/threonine protein kinases comprised of six isoforms (PAK1-6), and they play important roles in cytoskeletal dynamics, cell survival and proliferation.  They act as key signal transducers in several cancer signaling pathways, including Ras, Raf, NFκB, Akt, Bad and p53.  Although PAKs are not mutated in cancers, they are overexpressed, hyperactivated or amplified in several human tumors and their role in cell transformation make them attractive therapeutic targets.  This review discusses the evidence that PAK is important for cell transformation and some key signaling pathways it regulates.  This review primarily discusses Group I PAKs (PAK1, PAK2 and PAK3) as Group II PAKs (PAK4, PAK5 and PAK6) are discussed elsewhere in this issue (by Minden).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSGcnTbQFS8PMAKB7Fa-xrUfW6udTcc2eZKY-dq_dcF77ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sris1Wrtg%253D%253D&md5=2cf05fab271926995393ba06da3f5dbb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.4161%2Fcl.21882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcl.21882%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DD.%2BZ.%26aulast%3DField%26aufirst%3DJ.%26atitle%3DPAK%2520signaling%2520in%2520cancer%26jtitle%3DCell.%2520Logist.%26date%3D2012%26volume%3D2%26spage%3D105%26epage%3D116%26doi%3D10.4161%2Fcl.21882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Senapedis, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crochiere, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baloglu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landesman, Y.</span><span> </span><span class="NLM_article-title">Therapeutic potential of targeting PAK signaling</span> <span class="citation_source-journal">Anti-Cancer Agents Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span><span class="refDoi"> DOI: 10.2174/1871520615666150617111414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.2174%2F1871520615666150617111414" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=75-88&author=W.+Senapedisauthor=M.+Crochiereauthor=E.+Balogluauthor=Y.+Landesman&title=Therapeutic+potential+of+targeting+PAK+signaling&doi=10.2174%2F1871520615666150617111414"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2174%2F1871520615666150617111414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1871520615666150617111414%26sid%3Dliteratum%253Aachs%26aulast%3DSenapedis%26aufirst%3DW.%26aulast%3DCrochiere%26aufirst%3DM.%26aulast%3DBaloglu%26aufirst%3DE.%26aulast%3DLandesman%26aufirst%3DY.%26atitle%3DTherapeutic%2520potential%2520of%2520targeting%2520PAK%2520signaling%26jtitle%3DAnti-Cancer%2520Agents%2520Med.%2520Chem.%26date%3D2016%26volume%3D16%26spage%3D75%26epage%3D88%26doi%3D10.2174%2F1871520615666150617111414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Rudolph, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeflich, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span> </span><span class="NLM_article-title">Inhibitors of p21-activated kinases (PAKs)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span><span class="refDoi"> DOI: 10.1021/jm501613q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501613q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=111-129&author=J.+Rudolphauthor=J.+J.+Crawfordauthor=K.+P.+Hoeflichauthor=W.+Wang&title=Inhibitors+of+p21-activated+kinases+%28PAKs%29&doi=10.1021%2Fjm501613q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm501613q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501613q%26sid%3Dliteratum%253Aachs%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DInhibitors%2520of%2520p21-activated%2520kinases%2520%2528PAKs%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D111%26epage%3D129%26doi%3D10.1021%2Fjm501613q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Ha, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turk, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boggon, T. J.</span><span> </span><span class="NLM_article-title">Signaling, regulation, and specificity of the type II p21-activated kinases</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">12975</span><span class="NLM_x">–</span> <span class="NLM_lpage">12983</span><span class="refDoi"> DOI: 10.1074/jbc.R115.650416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1074%2Fjbc.R115.650416" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=12975-12983&author=B.+H.+Haauthor=E.+M.+Morseauthor=B.+E.+Turkauthor=T.+J.+Boggon&title=Signaling%2C+regulation%2C+and+specificity+of+the+type+II+p21-activated+kinases&doi=10.1074%2Fjbc.R115.650416"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R115.650416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R115.650416%26sid%3Dliteratum%253Aachs%26aulast%3DHa%26aufirst%3DB.%2BH.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26atitle%3DSignaling%252C%2520regulation%252C%2520and%2520specificity%2520of%2520the%2520type%2520II%2520p21-activated%2520kinases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D12975%26epage%3D12983%26doi%3D10.1074%2Fjbc.R115.650416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Ha, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lou, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krauthammer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halaban, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turk, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boggon, T. J.</span><span> </span><span class="NLM_article-title">Type II p21-activated kinases (PAKs) are regulated by an autoinhibitory pseudosubstrate</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">16107</span><span class="NLM_x">–</span> <span class="NLM_lpage">16112</span><span class="refDoi"> DOI: 10.1073/pnas.1214447109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1073%2Fpnas.1214447109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=22988085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGhsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16107-16112&author=B.+H.+Haauthor=M.+J.+Davisauthor=C.+Chenauthor=H.+J.+Louauthor=J.+Gaoauthor=R.+Zhangauthor=M.+Krauthammerauthor=R.+Halabanauthor=J.+Schlessingerauthor=B.+E.+Turkauthor=T.+J.+Boggon&title=Type+II+p21-activated+kinases+%28PAKs%29+are+regulated+by+an+autoinhibitory+pseudosubstrate&doi=10.1073%2Fpnas.1214447109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Type II p21-activated kinases (PAKs) are regulated by an autoinhibitory pseudosubstrate</span></div><div class="casAuthors">Ha, Byung Hak; Davis, Matthew J.; Chen, Catherine; Lou, Hua Jane; Gao, Jia; Zhang, Rong; Krauthammer, Michael; Halaban, Ruth; Schlessinger, Joseph; Turk, Benjamin E.; Boggon, Titus J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">16107-16112, S16107/1-S16107/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The type II p21-activated kinases (PAKs) are key effectors of RHO-family GTPases involved in cell motility, survival, and proliferation.  Using a structure-guided approach, we discovered that type II PAKs are regulated by an N-terminal autoinhibitory pseudosubstrate motif centered on a crit. proline residue, and that this regulation occurs independently of activation loop phosphorylation.  We detd. six X-ray crystal structures of either full-length PAK4 or its catalytic domain, that demonstrate the mol. basis for pseudosubstrate binding to the active state with phosphorylated activation loop.  We show that full-length PAK4 is constitutively autoinhibited, but mutation of the pseudo-substrate releases this inhibition and causes increased phosphorylation of the apoptotic regulation protein Bcl-2/Bcl-XL antagonist causing cell death and cellular morphol. changes.  We also find that PAK6 is regulated by the pseudosubstrate region, indicating a common type II PAK autoregulatory mechanism.  Finally, we find Src SH3, but not β-PIX SH3, can activate PAK4.  We provide a unique understanding for type II PAK regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreLSMEq29IlrVg90H21EOLACvtfcHk0li51YQQJcozfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGhsL3J&md5=65e13a76626af2ecb01509f70d36711e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1214447109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1214447109%26sid%3Dliteratum%253Aachs%26aulast%3DHa%26aufirst%3DB.%2BH.%26aulast%3DDavis%26aufirst%3DM.%2BJ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DLou%26aufirst%3DH.%2BJ.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DKrauthammer%26aufirst%3DM.%26aulast%3DHalaban%26aufirst%3DR.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26atitle%3DType%2520II%2520p21-activated%2520kinases%2520%2528PAKs%2529%2520are%2520regulated%2520by%2520an%2520autoinhibitory%2520pseudosubstrate%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D16107%26epage%3D16112%26doi%3D10.1073%2Fpnas.1214447109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Kelly, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chernoff, J.</span><span> </span><span class="NLM_article-title">Mouse models of PAK function</span> <span class="citation_source-journal">Cell. Logist.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span><span class="refDoi"> DOI: 10.4161/cl.21381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.4161%2Fcl.21381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=23162740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BC2sbpt1eltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=84-88&author=M.+L.+Kellyauthor=J.+Chernoff&title=Mouse+models+of+PAK+function&doi=10.4161%2Fcl.21381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Mouse models of PAK function</span></div><div class="casAuthors">Kelly Mollie L; Chernoff Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Cellular logistics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">84-88</span>
        ISSN:<span class="NLM_cas:issn">2159-2780</span>.
    </div><div class="casAbstract">p21-activated kinases are a family of highly conserved protein serine/threonine kinases that are increasingly recognized as playing essential roles in a variety of key signaling processes.  Genetic analyses in mice, using constitutive or regulated gene disruption, have provided important new insights into PAK function.  In this paper, we review the genetic analysis of all six PAK genes in mice.  These data address the singular and redundant functions of the various PAK genes and suggest therapeutic possibilities for small molecule PAK inhibitors or activators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQdjME4vf8KMv9pSBfyDvp7fW6udTcc2ea4djYNmPFzzLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbpt1eltA%253D%253D&md5=d6ee25015bac19af685c134474ca22d5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.4161%2Fcl.21381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcl.21381%26sid%3Dliteratum%253Aachs%26aulast%3DKelly%26aufirst%3DM.%2BL.%26aulast%3DChernoff%26aufirst%3DJ.%26atitle%3DMouse%2520models%2520of%2520PAK%2520function%26jtitle%3DCell.%2520Logist.%26date%3D2012%26volume%3D2%26spage%3D84%26epage%3D88%26doi%3D10.4161%2Fcl.21381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Tian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cammarano, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nekrasova, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minden, A.</span><span> </span><span class="NLM_article-title">Essential role for the Pak4 protein kinase in extraembryonic tissue development and vessel formation</span> <span class="citation_source-journal">Mech. Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">710</span><span class="NLM_x">–</span> <span class="NLM_lpage">720</span><span class="refDoi"> DOI: 10.1016/j.mod.2009.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2Fj.mod.2009.05.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2009&pages=710-720&author=Y.+Tianauthor=L.+Leiauthor=M.+Cammaranoauthor=T.+Nekrasovaauthor=A.+Minden&title=Essential+role+for+the+Pak4+protein+kinase+in+extraembryonic+tissue+development+and+vessel+formation&doi=10.1016%2Fj.mod.2009.05.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.mod.2009.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mod.2009.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DY.%26aulast%3DLei%26aufirst%3DL.%26aulast%3DCammarano%26aufirst%3DM.%26aulast%3DNekrasova%26aufirst%3DT.%26aulast%3DMinden%26aufirst%3DA.%26atitle%3DEssential%2520role%2520for%2520the%2520Pak4%2520protein%2520kinase%2520in%2520extraembryonic%2520tissue%2520development%2520and%2520vessel%2520formation%26jtitle%3DMech.%2520Dev.%26date%3D2009%26volume%3D126%26spage%3D710%26epage%3D720%26doi%3D10.1016%2Fj.mod.2009.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nekrasova, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suh, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minden, A.</span><span> </span><span class="NLM_article-title">The pak4 protein kinase plays a key role in cell survival and tumorigenesis in athymic mice</span> <span class="citation_source-journal">Mol. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1215</span><span class="NLM_x">–</span> <span class="NLM_lpage">1224</span><span class="refDoi"> DOI: 10.1158/1541-7786.MCR-08-0087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1158%2F1541-7786.MCR-08-0087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=18644984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslCgt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=1215-1224&author=Y.+Liuauthor=H.+Xiaoauthor=Y.+Tianauthor=T.+Nekrasovaauthor=X.+Haoauthor=H.+J.+Leeauthor=N.+Suhauthor=C.+S.+Yangauthor=A.+Minden&title=The+pak4+protein+kinase+plays+a+key+role+in+cell+survival+and+tumorigenesis+in+athymic+mice&doi=10.1158%2F1541-7786.MCR-08-0087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The Pak4 Protein Kinase Plays a Key Role in Cell Survival and Tumorigenesis in Athymic Mice</span></div><div class="casAuthors">Liu, Yingying; Xiao, Hang; Tian, Yanmei; Nekrasova, Tanya; Hao, Xingpei; Lee, Hong Jin; Suh, Nanjoo; Yang, Chung S.; Minden, Audrey</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1215-1224</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Pak4 is a member of the B group of p21-activated (Pak) kinases, originally identified as an effector protein for Cdc42.  Although Pak4 is expressed at low levels in most adult tissues, it is highly overexpressed in tumor cell lines.  Here, we show that Pak4 is also overexpressed in primary tumors, including colon, esophageal, and mammary tumors.  Overexpression of Pak4 also leads to tumor formation in athymic mice, whereas deletion of Pak4 inhibits tumorigenesis.  Although a constitutively active Pak4 mutant was previously shown to promote oncogenic transformation in cultured cells, these results are the first to show that Pak4 also promotes tumorigenesis in exptl. animals.  Furthermore, these results show for the first time that not only constitutively active Pak4, but also wild-type Pak4, is transforming, when exptl. animals are used.  These results are highly significant because wild-type Pak4, rather than activated Pak4, is overexpressed in tumor cells.  The results suggest that overexpression or activation of Pak4 is a key step in oncogenic transformation, due to its ability to promote cell survival and subsequent uncontrolled proliferation.  The finding that Pak4 is up-regulated in so many types of cancers indicates that Pak4 may play a vital role in a wide range of different types of cancer.  This makes it an attractive candidate for drug therapy for different types of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeiLYqGGfYu7Vg90H21EOLACvtfcHk0lji-u0FIdIBmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslCgt7g%253D&md5=40a0935d384470bf2cbf7a172f5730bb</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-08-0087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-08-0087%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DNekrasova%26aufirst%3DT.%26aulast%3DHao%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DSuh%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DC.%2BS.%26aulast%3DMinden%26aufirst%3DA.%26atitle%3DThe%2520pak4%2520protein%2520kinase%2520plays%2520a%2520key%2520role%2520in%2520cell%2520survival%2520and%2520tumorigenesis%2520in%2520athymic%2520mice%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2008%26volume%3D6%26spage%3D1215%26epage%3D1224%26doi%3D10.1158%2F1541-7786.MCR-08-0087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Callow, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clairvoyant, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schryver, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bischoff, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jallal, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smeal, T.</span><span> </span><span class="NLM_article-title">Requirement for PAK4 in the anchorage-independent growth of human cancer cell lines</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">550</span><span class="NLM_x">–</span> <span class="NLM_lpage">558</span><span class="refDoi"> DOI: 10.1074/jbc.M105732200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1074%2Fjbc.M105732200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=550-558&author=M.+G.+Callowauthor=F.+Clairvoyantauthor=S.+Zhuauthor=B.+Schryverauthor=D.+B.+Whyteauthor=J.+R.+Bischoffauthor=B.+Jallalauthor=T.+Smeal&title=Requirement+for+PAK4+in+the+anchorage-independent+growth+of+human+cancer+cell+lines&doi=10.1074%2Fjbc.M105732200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M105732200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M105732200%26sid%3Dliteratum%253Aachs%26aulast%3DCallow%26aufirst%3DM.%2BG.%26aulast%3DClairvoyant%26aufirst%3DF.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DSchryver%26aufirst%3DB.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DBischoff%26aufirst%3DJ.%2BR.%26aulast%3DJallal%26aufirst%3DB.%26aulast%3DSmeal%26aufirst%3DT.%26atitle%3DRequirement%2520for%2520PAK4%2520in%2520the%2520anchorage-independent%2520growth%2520of%2520human%2520cancer%2520cell%2520lines%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D550%26epage%3D558%26doi%3D10.1074%2Fjbc.M105732200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Minden, A.</span><span> </span><span class="NLM_article-title">The pak4 protein kinase in breast cancer</span> <span class="citation_source-journal">ISRN Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2012</span><span class="NLM_x">, </span> <span class="NLM_fpage">694201</span><span class="refDoi"> DOI: 10.5402/2012/694201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.5402%2F2012%2F694201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=23326684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BC3szgt1agtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2012&publication_year=2012&pages=694201&author=A.+Minden&title=The+pak4+protein+kinase+in+breast+cancer&doi=10.5402%2F2012%2F694201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The pak4 protein kinase in breast cancer</span></div><div class="casAuthors">Minden Audrey</div><div class="citationInfo"><span class="NLM_cas:title">ISRN oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2012</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">694201</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Paks4, along with Paks5, and 6 are members of the group B family of p21-activated kinases (Paks).  The Paks play multiple different roles in controlling cell morphology, cell growth, proliferation, and signaling.  Pak4 has essential roles in embryonic development (Qu et al., 2003), but in adults high levels of Pak4 are frequently associated with cancer.  Pak4 has been implicated in several types of cancer (Wells and Jones, 2010; Eswaran et al., 2009; Liu et al., 2008; and Liu et al., 2010) and it is strongly linked to breast cancer (Liu et al., 2008; Liu et al. 2010; Yu et al., 2009; Rafn et al., 2012; and So et al., 2012).  Breast tumors and breast cancer cell lines frequently have high levels of Pak4 (Liu et al., 2008), and overexpression of Pak4 in mammary epithelial cells leads to tumorigenesis in mice (Liu et al., 2010).  This paper summarizes the current work on the role of Pak4 in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR4b_ZBK4mwArFU0A1HOshhfW6udTcc2eab8FKcd2bn7Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3szgt1agtA%253D%253D&md5=4d260e310ad65144673a648ea78d22e0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.5402%2F2012%2F694201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5402%252F2012%252F694201%26sid%3Dliteratum%253Aachs%26aulast%3DMinden%26aufirst%3DA.%26atitle%3DThe%2520pak4%2520protein%2520kinase%2520in%2520breast%2520cancer%26jtitle%3DISRN%2520Oncol.%26date%3D2012%26volume%3D2012%26spage%3D694201%26doi%3D10.5402%2F2012%2F694201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Wells, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, G. E.</span><span> </span><span class="NLM_article-title">The emerging importance of group II PAKs</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">425</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">473</span><span class="refDoi"> DOI: 10.1042/BJ20091173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1042%2FBJ20091173" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2010&pages=465-473&author=C.+M.+Wellsauthor=G.+E.+Jones&title=The+emerging+importance+of+group+II+PAKs&doi=10.1042%2FBJ20091173"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1042%2FBJ20091173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20091173%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DC.%2BM.%26aulast%3DJones%26aufirst%3DG.%2BE.%26atitle%3DThe%2520emerging%2520importance%2520of%2520group%2520II%2520PAKs%26jtitle%3DBiochem.%2520J.%26date%3D2010%26volume%3D425%26spage%3D465%26epage%3D473%26doi%3D10.1042%2FBJ20091173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Rudolph, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ndubaku, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackwood, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aliagas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gazzard, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drobnick, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeflich, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Favor, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heise, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ong, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakravarty, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakubiak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Epler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vega, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cain, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, Y.</span><span> </span><span class="NLM_article-title">Chemically diverse group I p21-activated kinase (PAK) inhibitors impart acute cardiovascular toxicity with a narrow therapeutic window</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">5520</span><span class="NLM_x">–</span> <span class="NLM_lpage">5541</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00638</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00638" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslWhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5520-5541&author=J.+Rudolphauthor=L.+J.+Murrayauthor=C.+O.+Ndubakuauthor=T.+O%E2%80%99Brienauthor=E.+Blackwoodauthor=W.+Wangauthor=I.+Aliagasauthor=L.+Gazzardauthor=J.+J.+Crawfordauthor=J.+Drobnickauthor=W.+Leeauthor=X.+Zhaoauthor=K.+P.+Hoeflichauthor=D.+A.+Favorauthor=P.+Dongauthor=H.+Zhangauthor=C.+E.+Heiseauthor=A.+Ohauthor=C.+C.+Ongauthor=H.+Laauthor=P.+Chakravartyauthor=C.+Chanauthor=D.+Jakubiakauthor=J.+Eplerauthor=S.+Ramaswamyauthor=R.+Vegaauthor=G.+Cainauthor=D.+Diazauthor=Y.+Zhong&title=Chemically+diverse+group+I+p21-activated+kinase+%28PAK%29+inhibitors+impart+acute+cardiovascular+toxicity+with+a+narrow+therapeutic+window&doi=10.1021%2Facs.jmedchem.6b00638"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Diverse Group I p21-Activated Kinase (PAK) Inhibitors Impart Acute Cardiovascular Toxicity with a Narrow Therapeutic Window</span></div><div class="casAuthors">Rudolph, Joachim; Murray, Lesley J.; Ndubaku, Chudi O.; O'Brien, Thomas; Blackwood, Elizabeth; Wang, Weiru; Aliagas, Ignacio; Gazzard, Lewis; Crawford, James J.; Drobnick, Joy; Lee, Wendy; Zhao, Xianrui; Hoeflich, Klaus P.; Favor, David A.; Dong, Ping; Zhang, Haiming; Heise, Christopher E.; Oh, Angela; Ong, Christy C.; La, Hank; Chakravarty, Paroma; Chan, Connie; Jakubiak, Diana; Epler, Jennifer; Ramaswamy, Sreemathy; Vega, Roxanne; Cain, Gary; Diaz, Dolores; Zhong, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5520-5541</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P21-activated kinase 1 (PAK1) has an important role in transducing signals in several oncogenic pathways.  The concept of inhibiting this kinase has garnered significant interest over the past decade, particularly for targeting cancers assocd. with PAK1 amplification.  Animal studies with the selective group I PAK (pan-PAK1, 2, 3) inhibitor G-5555 from the pyrido[2,3-d]pyrimidin-7-one class uncovered acute toxicity with a narrow therapeutic window.  To attempt mitigating the toxicity, we introduced significant structural changes, culminating in the discovery of the potent pyridone side chain analog G-9791.  Mouse tolerability studies with this compd., other members of this series, and compds. from two structurally distinct classes revealed persistent toxicity and a correlation of min. toxic concns. and PAK1/2 mediated cellular potencies.  Broad screening of selected PAK inhibitors revealed PAK1, 2, and 3 as the only overlapping targets.  Our data suggest acute cardiovascular toxicity resulting from the inhibition of PAK2, which may be enhanced by PAK1 inhibition, and cautions against continued pursuit of pan-group I PAK inhibitors in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop35dBBsANDLVg90H21EOLACvtfcHk0lilEhN_oIVgYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslWhtLg%253D&md5=9fbab0f13e23ee9f215b32c48c617f3e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00638%26sid%3Dliteratum%253Aachs%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DMurray%26aufirst%3DL.%2BJ.%26aulast%3DNdubaku%26aufirst%3DC.%2BO.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DBlackwood%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DGazzard%26aufirst%3DL.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DDrobnick%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DFavor%26aufirst%3DD.%2BA.%26aulast%3DDong%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHeise%26aufirst%3DC.%2BE.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DOng%26aufirst%3DC.%2BC.%26aulast%3DLa%26aufirst%3DH.%26aulast%3DChakravarty%26aufirst%3DP.%26aulast%3DChan%26aufirst%3DC.%26aulast%3DJakubiak%26aufirst%3DD.%26aulast%3DEpler%26aufirst%3DJ.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DVega%26aufirst%3DR.%26aulast%3DCain%26aufirst%3DG.%26aulast%3DDiaz%26aufirst%3DD.%26aulast%3DZhong%26aufirst%3DY.%26atitle%3DChemically%2520diverse%2520group%2520I%2520p21-activated%2520kinase%2520%2528PAK%2529%2520inhibitors%2520impart%2520acute%2520cardiovascular%2520toxicity%2520with%2520a%2520narrow%2520therapeutic%2520window%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5520%26epage%3D5541%26doi%3D10.1021%2Facs.jmedchem.6b00638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The ins and outs of selective kinase inhibitor development</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">818</span><span class="NLM_x">–</span> <span class="NLM_lpage">821</span><span class="refDoi"> DOI: 10.1038/nchembio.1938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1038%2Fnchembio.1938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=26485069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=818-821&author=S.+Mullerauthor=A.+Chaikuadauthor=N.+S.+Grayauthor=S.+Knapp&title=The+ins+and+outs+of+selective+kinase+inhibitor+development&doi=10.1038%2Fnchembio.1938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The ins and outs of selective kinase inhibitor development</span></div><div class="casAuthors">Muller, Susanne; Chaikuad, Apirat; Gray, Nathanael S.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">818-821</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein kinases have emerged as one of the most successful families of drug targets.  To date, most selective kinase inhibitors have been discovered serendipitously either through broad selectivity screening or through the discovery of unique binding modes.  Here the authors discuss design strategies that could lead to a broader coverage of the kinome with selective inhibitors and to a more rational approach for developing them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQzaMrxSYx_rVg90H21EOLACvtfcHk0lim9uxyFsFODA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF&md5=f5adf0bbaa388c87b26bae23cfad0bae</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1938%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520ins%2520and%2520outs%2520of%2520selective%2520kinase%2520inhibitor%2520development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D818%26epage%3D821%26doi%3D10.1038%2Fnchembio.1938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Murray, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piraino, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westwick, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamerdin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dagostino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knighton, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loi, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zager, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraynov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popoff, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kephart, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marakovits, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergqvist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smeal, T.</span><span> </span><span class="NLM_article-title">Small-molecule p21-activated kinase inhibitor PF-3758309 is a potent inhibitor of oncogenic signaling and tumor growth</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">9446</span><span class="NLM_x">–</span> <span class="NLM_lpage">9451</span><span class="refDoi"> DOI: 10.1073/pnas.0911863107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1073%2Fpnas.0911863107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=20439741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslWqtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=9446-9451&author=B.+W.+Murrayauthor=C.+Guoauthor=J.+Pirainoauthor=J.+K.+Westwickauthor=C.+Zhangauthor=J.+Lamerdinauthor=E.+Dagostinoauthor=D.+Knightonauthor=C.+M.+Loiauthor=M.+Zagerauthor=E.+Kraynovauthor=I.+Popoffauthor=J.+G.+Christensenauthor=R.+Martinezauthor=S.+E.+Kephartauthor=J.+Marakovitsauthor=S.+Karlicekauthor=S.+Bergqvistauthor=T.+Smeal&title=Small-molecule+p21-activated+kinase+inhibitor+PF-3758309+is+a+potent+inhibitor+of+oncogenic+signaling+and+tumor+growth&doi=10.1073%2Fpnas.0911863107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule p21-activated kinase inhibitor PF3758309 is a potent inhibitor of oncogenic signaling and tumor growth</span></div><div class="casAuthors">Murray, Brion W.; Guo, Chuangxing; Piraino, Joseph; Westwick, John K.; Zhang, Cathy; Lamerdin, Jane; Dagostino, Eleanor; Knighton, Daniel; Loi, Cho-Ming; Zager, Michael; Kraynov, Eugenia; Popoff, Ian; Chrietensen, James G.; Martinez, Ricardo; Kephart, Susan E.; Marakovits, Joseph; Karlicek, Shannon; Bergqvist, Simon; Smeal, Tod</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9446-9451, S9446/1-S9446/18</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Despite abundant evidence that aberrant Rho-family GTPase activation contributes to most steps of cancer initiation and progression, there is a dearth of inhibitors of their effectors (e.g.. p21-activated kinases).  Through high-throughput screening and structure-based design, the authors identify PF-3758309, a potent (Kd = 2.7 nM), ATP-competitive, pyrrolopyrazole inhibitor of PAK4.  In cells, PF-3758309 inhibits phosphorylation of the PAK4 substrate GEF-H1 (IC50 = 1.3 nM) and anchorage-independent growth of a panel of tumor cell lines (IC50 = 4.7±3 nM).  The mol. underpinnings of PF-3758309 biol. effects were characterized using an integration of traditional and emerging technologies.  Crystallog. characterization of the PF-3758309/PAK4 complex defined determinants of potency and kinase selectivity.  Global high-content cellular anal. confirms that PF-3758309 modulates known PAK4-dependent signaling nodes and identifies unexpected links to addnl. pathways (e.g., p53).  In tumor models, PF-3758309 inhibits PAK4-dependent pathways in proteomic studies and regulates functional activities related to cell proliferation and survival.  PF-3758309 blocks the growth of multiple human tumor xenografts, with a plasma EC50 value of 0.4 nM in the most sensitive model.  This study defines PAK4-related pathways, provides addnl. support for PAK4 as a therapeutic target with a unique combination of functions (apoptotic, cytoskeletal, cell-cycle), and identifies a potent, orally available small-mol. PAK inhibitor with significant promise for the treatment of human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWI0STMKt9dbVg90H21EOLACvtfcHk0lim9uxyFsFODA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslWqtrw%253D&md5=0080659735ef51f84c00112e573b1c31</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0911863107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0911863107%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DGuo%26aufirst%3DC.%26aulast%3DPiraino%26aufirst%3DJ.%26aulast%3DWestwick%26aufirst%3DJ.%2BK.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLamerdin%26aufirst%3DJ.%26aulast%3DDagostino%26aufirst%3DE.%26aulast%3DKnighton%26aufirst%3DD.%26aulast%3DLoi%26aufirst%3DC.%2BM.%26aulast%3DZager%26aufirst%3DM.%26aulast%3DKraynov%26aufirst%3DE.%26aulast%3DPopoff%26aufirst%3DI.%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DKephart%26aufirst%3DS.%2BE.%26aulast%3DMarakovits%26aufirst%3DJ.%26aulast%3DKarlicek%26aufirst%3DS.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DSmeal%26aufirst%3DT.%26atitle%3DSmall-molecule%2520p21-activated%2520kinase%2520inhibitor%2520PF-3758309%2520is%2520a%2520potent%2520inhibitor%2520of%2520oncogenic%2520signaling%2520and%2520tumor%2520growth%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D9446%26epage%3D9451%26doi%3D10.1073%2Fpnas.0911863107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Bradshaw-Pierce, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitts, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhillips, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafson, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halsey, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kan, J. L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckhardt, S. G.</span><span> </span><span class="NLM_article-title">Tumor P-glycoprotein correlates with efficacy of PF-3758309 in in vitro and in vivo models of colorectal cancer</span> <span class="citation_source-journal">Front. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="refDoi"> DOI: 10.3389/fphar.2013.00022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.3389%2Ffphar.2013.00022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=23524533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BC3svpt1Ohsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=22&author=E.+L.+Bradshaw-Pierceauthor=T.+M.+Pittsauthor=A.+C.+Tanauthor=K.+McPhillipsauthor=M.+Westauthor=D.+L.+Gustafsonauthor=C.+Halseyauthor=L.+Nguyenauthor=N.+V.+Leeauthor=J.+L.+C.+Kanauthor=B.+W.+Murrayauthor=S.+G.+Eckhardt&title=Tumor+P-glycoprotein+correlates+with+efficacy+of+PF-3758309+in+in+vitro+and+in+vivo+models+of+colorectal+cancer&doi=10.3389%2Ffphar.2013.00022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor P-Glycoprotein Correlates with Efficacy of PF-3758309 in in vitro and in vivo Models of Colorectal Cancer</span></div><div class="casAuthors">Bradshaw-Pierce Erica Lynn; Pitts Todd M; Tan Aik-Choon; McPhillips Kelly; West Mark; Gustafson Daniel L; Halsey Charles; Nguyen Leslie; Lee Nathan V; Kan Julie L C; Murray Brion William; Eckhardt S Gail</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">22</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">P-glycoprotein (P-gp), a member of the ATP-binding cassette transporter family, is overexpressed in a number of different cancers and some studies show that P-gp overexpression can be correlated to poor prognosis or therapeutic resistance.  Here we sought to elucidate if PF-3758309 (PF-309), a novel p-21 activated kinase inhibitor, efficacy was influenced by tumor P-gp.  Based on in vitro proliferation data, a panel of colorectal cancer cell lines were ranked as sensitive or resistant and ABCB1 (P-gp) expression was evaluated by microarray for these cell lines.  P-gp expression was determined by western blot and activity determined by rhodamine efflux assay.  Knock down of P-gp and pharmacologic inhibition of P-gp to restore PF-309 activity was performed in vitro.  PF-309 activity was evaluated in vivo in cell line xenograft models and in primary patient derived tumor xenografts (PDTX).  Mice were treated with 25 mg/kg PF-309 orally, twice daily.  On the last day of treatment, tumor and plasma were collected for PF-309 analysis.  Here we show that ABCB1 gene expression correlates with resistance to PF-309 treatment in vitro and the expression and activity of P-gp was verified in a panel of resistant cells.  Furthermore, inhibition of P-gp increased the sensitivity of resistant cells, resulting in a 4-100-fold decrease in the IC50s.  Eleven cell line xenografts and 12 PDTX models were treated with PF-309.  From the cell line xenografts, we found a significant correlation between ABCB1 gene expression profiles and tumor response.  We evaluated tumor and plasma concentrations for eight tumor models (three cell line xenografts and five PDTX models) and a significant correlation was found between tumor concentration and response.  Additionally, we show that tumor concentration is approximately fourfold lower in tumors that express P-gp, verified by western blot.  Our in vitro and in vivo data strongly suggests that PF-309 efficacy is influenced by the expression of tumor P-gp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR7z7ao3-xc0c-crCu0nI_sfW6udTcc2eboMj_gegWSDbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3svpt1Ohsw%253D%253D&md5=cfcbecd18424bb7af204c2755cc87b32</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2013.00022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2013.00022%26sid%3Dliteratum%253Aachs%26aulast%3DBradshaw-Pierce%26aufirst%3DE.%2BL.%26aulast%3DPitts%26aufirst%3DT.%2BM.%26aulast%3DTan%26aufirst%3DA.%2BC.%26aulast%3DMcPhillips%26aufirst%3DK.%26aulast%3DWest%26aufirst%3DM.%26aulast%3DGustafson%26aufirst%3DD.%2BL.%26aulast%3DHalsey%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DN.%2BV.%26aulast%3DKan%26aufirst%3DJ.%2BL.%2BC.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DEckhardt%26aufirst%3DS.%2BG.%26atitle%3DTumor%2520P-glycoprotein%2520correlates%2520with%2520efficacy%2520of%2520PF-3758309%2520in%2520in%2520vitro%2520and%2520in%2520vivo%2520models%2520of%2520colorectal%2520cancer%26jtitle%3DFront.%2520Pharmacol.%26date%3D2013%26volume%3D4%26spage%3D22%26doi%3D10.3389%2Ffphar.2013.00022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Crawford, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aliagas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeflich, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouge, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heise, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mintzer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, J.</span><span> </span><span class="NLM_article-title">Structure-guided design of group I selective p21-activated kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5121</span><span class="NLM_x">–</span> <span class="NLM_lpage">5136</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5121-5136&author=J.+J.+Crawfordauthor=W.+Leeauthor=I.+Aliagasauthor=S.+Mathieuauthor=K.+P.+Hoeflichauthor=W.+Zhouauthor=W.+Wangauthor=L.+Rougeauthor=L.+Murrayauthor=H.+Laauthor=N.+Liuauthor=P.+W.+Fanauthor=J.+Cheongauthor=C.+E.+Heiseauthor=S.+Ramaswamyauthor=R.+Mintzerauthor=Y.+Liuauthor=Q.+Chaoauthor=J.+Rudolph&title=Structure-guided+design+of+group+I+selective+p21-activated+kinase+inhibitors&doi=10.1021%2Facs.jmedchem.5b00572"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00572%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DMathieu%26aufirst%3DS.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DRouge%26aufirst%3DL.%26aulast%3DMurray%26aufirst%3DL.%26aulast%3DLa%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DFan%26aufirst%3DP.%2BW.%26aulast%3DCheong%26aufirst%3DJ.%26aulast%3DHeise%26aufirst%3DC.%2BE.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DMintzer%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DChao%26aufirst%3DQ.%26aulast%3DRudolph%26aufirst%3DJ.%26atitle%3DStructure-guided%2520design%2520of%2520group%2520I%2520selective%2520p21-activated%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5121%26epage%3D5136%26doi%3D10.1021%2Facs.jmedchem.5b00572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Staben, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyle, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boggs, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burch, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chua, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiPasquale, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedman, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heise, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeppen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mintzer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouge, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeflich, K. P.</span><span> </span><span class="NLM_article-title">Back pocket flexibility provides group II p21-activated kinase (PAK) selectivity for type I 1/2 kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1033</span><span class="NLM_x">–</span> <span class="NLM_lpage">1045</span><span class="refDoi"> DOI: 10.1021/jm401768t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401768t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVGmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1033-1045&author=S.+T.+Stabenauthor=J.+A.+Fengauthor=K.+Lyleauthor=M.+Belvinauthor=J.+Boggsauthor=J.+D.+Burchauthor=C.+C.+Chuaauthor=H.+Cuiauthor=A.+G.+DiPasqualeauthor=L.+S.+Friedmanauthor=C.+Heiseauthor=H.+Koeppenauthor=A.+Koteyauthor=R.+Mintzerauthor=A.+Ohauthor=D.+A.+Robertsauthor=L.+Rougeauthor=J.+Rudolphauthor=C.+Tamauthor=W.+Wangauthor=Y.+Xiaoauthor=A.+Youngauthor=Y.+Zhangauthor=K.+P.+Hoeflich&title=Back+pocket+flexibility+provides+group+II+p21-activated+kinase+%28PAK%29+selectivity+for+type+I+1%2F2+kinase+inhibitors&doi=10.1021%2Fjm401768t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Back Pocket Flexibility Provides Group II p21-Activated Kinase (PAK) Selectivity for Type I 1/2 Kinase Inhibitors</span></div><div class="casAuthors">Staben, Steven T.; Feng, Jianwen A.; Lyle, Karen; Belvin, Marcia; Boggs, Jason; Burch, Jason D.; Chua, Ching-ching; Cui, Haifeng; DiPasquale, Antonio G.; Friedman, Lori S.; Heise, Christopher; Koeppen, Hartmut; Kotey, Adrian; Mintzer, Robert; Oh, Angela; Roberts, David Allen; Rouge, Lionel; Rudolph, Joachim; Tam, Christine; Wang, Weiru; Xiao, Yisong; Young, Amy; Zhang, Yamin; Hoeflich, Klaus P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1033-1045</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based methods were used to design a potent and highly selective group II p21-activated kinase (PAK) inhibitor with a novel binding mode, compd. I.  Hydrophobic interactions within a lipophilic pocket past the methionine gatekeeper of group II PAKs approached by these type I 1/2 binders were found to be important for improving potency.  A structure-based hypothesis and strategy for achieving selectivity over group I PAKs, and the broad kinome, based on unique flexibility of this lipophilic pocket, is presented.  A concn.-dependent decrease in tumor cell migration and invasion in two triple-neg. breast cancer cell lines was obsd. with compd. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrogBhcjJGWm7Vg90H21EOLACvtfcHk0lh1ohWb61jaPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVGmsQ%253D%253D&md5=08b11088bc4d806cc61e16c932b68cf6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm401768t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401768t%26sid%3Dliteratum%253Aachs%26aulast%3DStaben%26aufirst%3DS.%2BT.%26aulast%3DFeng%26aufirst%3DJ.%2BA.%26aulast%3DLyle%26aufirst%3DK.%26aulast%3DBelvin%26aufirst%3DM.%26aulast%3DBoggs%26aufirst%3DJ.%26aulast%3DBurch%26aufirst%3DJ.%2BD.%26aulast%3DChua%26aufirst%3DC.%2BC.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DDiPasquale%26aufirst%3DA.%2BG.%26aulast%3DFriedman%26aufirst%3DL.%2BS.%26aulast%3DHeise%26aufirst%3DC.%26aulast%3DKoeppen%26aufirst%3DH.%26aulast%3DKotey%26aufirst%3DA.%26aulast%3DMintzer%26aufirst%3DR.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DD.%2BA.%26aulast%3DRouge%26aufirst%3DL.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DTam%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DYoung%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26atitle%3DBack%2520pocket%2520flexibility%2520provides%2520group%2520II%2520p21-activated%2520kinase%2520%2528PAK%2529%2520selectivity%2520for%2520type%2520I%25201%252F2%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1033%26epage%3D1045%26doi%3D10.1021%2Fjm401768t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Abu Aboud, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senapedis, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baloglu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Argueta, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, R. H.</span><span> </span><span class="NLM_article-title">Dual and specific inhibition of NAMPT and PAK4 by KPT-9274 decreases kidney cancer growth</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">2119</span><span class="NLM_x">–</span> <span class="NLM_lpage">2129</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1158%2F1535-7163.MCT-16-0197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=27390344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVCnsLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2119-2129&author=O.+Abu+Aboudauthor=C.+H.+Chenauthor=W.+Senapedisauthor=E.+Balogluauthor=C.+Arguetaauthor=R.+H.+Weiss&title=Dual+and+specific+inhibition+of+NAMPT+and+PAK4+by+KPT-9274+decreases+kidney+cancer+growth&doi=10.1158%2F1535-7163.MCT-16-0197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Dual and Specific Inhibition of NAMPT and PAK4 By KPT-9274 Decreases Kidney Cancer Growth</span></div><div class="casAuthors">Abu Aboud, Omran; Chen, Ching-Hsien; Senapedis, William; Baloglu, Erkan; Argueta, Christian; Weiss, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2119-2129</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Kidney cancer (or renal cell carcinoma, RCC) is the sixth most common malignancy in the United States and one of the relatively few whose incidence is increasing.  Because of the near universal resistance which occurs with the use of current treatment regimens, reprogrammed metabolic pathways are being investigated as potential targets for novel therapies of this disease.  Borrowing from studies on other malignancies, we have identified the PAK4 and NAD biosynthetic pathways as being essential for RCC growth.  We now show, using the dual PAK4/NAMPT inhibitor KPT-9274, that interference with these signaling pathways results in redn. of G2-M transit as well as induction of apoptosis and decrease in cell invasion and migration in several human RCC cell lines.  Mechanistic studies demonstrate that inhibition of the PAK4 pathway by KPT-9274 attenuates nuclear β-catenin as well as the Wnt/β-catenin targets cyclin D1 and c-Myc.  Furthermore, NAPRT1 downregulation, which we show occurs in all RCC cell lines tested, makes this tumor highly dependent on NAMPT for its NAD requirements, such that inhibition of NAMPT by KPT-9274 leads to decreased survival of these rapidly proliferating cells.  When KPT-9274 was administered in vivo to a 786-O (VHL-mut) human RCC xenograft model, there was dose-dependent inhibition of tumor growth with no apparent toxicity; KPT-9274 demonstrated the expected on-target effects in this mouse model.  KPT-9274 is being evaluated in a phase I human clin. trial in solid tumors and lymphomas, which will allow this data to be rapidly translated into the clinic for the treatment of RCC.  Mol Cancer Ther; 15(9); 2119-29. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVAlkkAFgNi7Vg90H21EOLACvtfcHk0lh1ohWb61jaPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVCnsLvK&md5=043002b41043e7fda7c07f10f89fbfa0</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0197%26sid%3Dliteratum%253Aachs%26aulast%3DAbu%2BAboud%26aufirst%3DO.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DSenapedis%26aufirst%3DW.%26aulast%3DBaloglu%26aufirst%3DE.%26aulast%3DArgueta%26aufirst%3DC.%26aulast%3DWeiss%26aufirst%3DR.%2BH.%26atitle%3DDual%2520and%2520specific%2520inhibition%2520of%2520NAMPT%2520and%2520PAK4%2520by%2520KPT-9274%2520decreases%2520kidney%2520cancer%2520growth%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2119%26epage%3D2129%26doi%3D10.1158%2F1535-7163.MCT-16-0197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span> </span><span class="NLM_article-title">LCH-7749944, a novel and potent p21-activated kinase 4 inhibitor, suppresses proliferation and invasion in human gastric cancer cells</span> <span class="citation_source-journal">Cancer Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">317</span><span class="NLM_x">, </span> <span class="NLM_fpage">24</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1016/j.canlet.2011.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2Fj.canlet.2011.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=22085492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2mu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=317&publication_year=2012&pages=24-32&author=J.+Zhangauthor=J.+Wangauthor=Q.+Guoauthor=Y.+Wangauthor=Y.+Zhouauthor=H.+Pengauthor=M.+Chengauthor=D.+Zhaoauthor=F.+Li&title=LCH-7749944%2C+a+novel+and+potent+p21-activated+kinase+4+inhibitor%2C+suppresses+proliferation+and+invasion+in+human+gastric+cancer+cells&doi=10.1016%2Fj.canlet.2011.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">LCH-7749944, a novel and potent p21-activated kinase 4 inhibitor, suppresses proliferation and invasion in human gastric cancer cells</span></div><div class="casAuthors">Zhang, Jian; Wang, Jian; Guo, Qiqiang; Wang, Yu; Zhou, Ying; Peng, Huizhi; Cheng, Maosheng; Zhao, Dongmei; Li, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">317</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-32</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">P21-activated kinase 4 (PAK4), a serine/threonine protein kinase, has involved in the regulation of cytoskeletal reorganization, cell proliferation, gene transcription, oncogenic transformation and cell invasion.  Moreover, PAK4 overexpression, genetic amplification and mutations were detected in a variety of human tumors, which make it potential therapeutic target.  In this paper we found that LCH-7749944, a novel and potent PAK4 inhibitor, effectively suppressed the proliferation of human gastric cancer cells through downregulation of PAK4/c-Src/EGFR/cyclin D1 pathway.  In addn., LCH-7749944 significantly inhibited the migration and invasion of human gastric cancer cells in conjunction with concomitant blockage of PAK4/LIMK1/cofilin and PAK4/MEK-1/ERK1/2/MMP2 pathways.  Interestingly, LCH-7749944 also inhibited the formation of filopodia and induced cell elongation in SGC7901 cells.  Importantly, LCH-7749944 caused successful inhibition of EGFR activity due to its inhibitory effect on PAK4.  Taken together, these results provided novel insights into the development of PAK4 inhibitor and potential therapeutic strategies for gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaLHQjMNmVxbVg90H21EOLACvtfcHk0lj5ECNi9Nnhtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2mu7s%253D&md5=b5b8a014f7efd0247bdc5756050389a4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2011.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2011.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26atitle%3DLCH-7749944%252C%2520a%2520novel%2520and%2520potent%2520p21-activated%2520kinase%25204%2520inhibitor%252C%2520suppresses%2520proliferation%2520and%2520invasion%2520in%2520human%2520gastric%2520cancer%2520cells%26jtitle%3DCancer%2520Lett.%26date%3D2012%26volume%3D317%26spage%3D24%26epage%3D32%26doi%3D10.1016%2Fj.canlet.2011.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Hao, C. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, W. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, Z. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, M. S.</span><span> </span><span class="NLM_article-title">Development of 2, 4-diaminoquinazoline derivatives as potent PAK4 inhibitors by the core refinement strategy</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2017.02.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2Fj.ejmech.2017.02.063" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2017&pages=1-13&author=C.+Z.+Haoauthor=W.+X.+Huangauthor=X.+D.+Liauthor=J.+Guoauthor=M.+Chenauthor=Z.+Z.+Yanauthor=K.+Wangauthor=X.+L.+Jiangauthor=S.+Songauthor=J.+Wangauthor=D.+M.+Zhaoauthor=F.+Liauthor=M.+S.+Cheng&title=Development+of+2%2C+4-diaminoquinazoline+derivatives+as+potent+PAK4+inhibitors+by+the+core+refinement+strategy&doi=10.1016%2Fj.ejmech.2017.02.063"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.02.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.02.063%26sid%3Dliteratum%253Aachs%26aulast%3DHao%26aufirst%3DC.%2BZ.%26aulast%3DHuang%26aufirst%3DW.%2BX.%26aulast%3DLi%26aufirst%3DX.%2BD.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DYan%26aufirst%3DZ.%2BZ.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DJiang%26aufirst%3DX.%2BL.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DD.%2BM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DCheng%26aufirst%3DM.%2BS.%26atitle%3DDevelopment%2520of%25202%252C%25204-diaminoquinazoline%2520derivatives%2520as%2520potent%2520PAK4%2520inhibitors%2520by%2520the%2520core%2520refinement%2520strategy%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D131%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.ejmech.2017.02.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Hao, C. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, W. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, D. M.</span><span> </span><span class="NLM_article-title">Advances in the 1-phenanthryl-tetrahydroisoquinoline series of PAK4 inhibitors: potent agents restrain tumor cell growth and invasion</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">7676</span><span class="NLM_x">–</span> <span class="NLM_lpage">7690</span><span class="refDoi"> DOI: 10.1039/C6OB01072E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1039%2FC6OB01072E" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=7676-7690&author=C.+Z.+Haoauthor=X.+D.+Liauthor=S.+Songauthor=B.+Y.+Guoauthor=J.+Guoauthor=J.+Zhangauthor=Q.+L.+Zhangauthor=W.+X.+Huangauthor=J.+Wangauthor=B.+Linauthor=M.+S.+Chengauthor=F.+Liauthor=D.+M.+Zhao&title=Advances+in+the+1-phenanthryl-tetrahydroisoquinoline+series+of+PAK4+inhibitors%3A+potent+agents+restrain+tumor+cell+growth+and+invasion&doi=10.1039%2FC6OB01072E"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC6OB01072E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6OB01072E%26sid%3Dliteratum%253Aachs%26aulast%3DHao%26aufirst%3DC.%2BZ.%26aulast%3DLi%26aufirst%3DX.%2BD.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DGuo%26aufirst%3DB.%2BY.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%2BL.%26aulast%3DHuang%26aufirst%3DW.%2BX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DB.%26aulast%3DCheng%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DD.%2BM.%26atitle%3DAdvances%2520in%2520the%25201-phenanthryl-tetrahydroisoquinoline%2520series%2520of%2520PAK4%2520inhibitors%253A%2520potent%2520agents%2520restrain%2520tumor%2520cell%2520growth%2520and%2520invasion%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2016%26volume%3D14%26spage%3D7676%26epage%3D7690%26doi%3D10.1039%2FC6OB01072E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Thermal denaturation assays in chemical biology</span> <span class="citation_source-journal">Assay Drug Dev. Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">128</span><span class="NLM_x">–</span> <span class="NLM_lpage">136</span><span class="refDoi"> DOI: 10.1089/adt.2011.0390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1089%2Fadt.2011.0390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=22066913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC38Xltl2mt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=128-136&author=G.+Senisterraauthor=I.+Chauauthor=M.+Vedadi&title=Thermal+denaturation+assays+in+chemical+biology&doi=10.1089%2Fadt.2011.0390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Thermal Denaturation Assays in Chemical Biology</span></div><div class="casAuthors">Senisterra, Guillermo; Chau, Irene; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">128-136</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Thermal denaturation-based methods are becoming increasingly used to characterize protein stability and interactions.  Recent tech. advances have made these methods more suitable for high throughput screening.  Reasonable throughput and the ability to perform these screens using commonly used instruments, such as RT-PCR machines or simple plate readers equipped with heating devices, facilitate these expts. in almost any lab.  Introducing an aggregation-based monitoring approach as well as alternative fluorophores has allowed the screening of a wider range of proteins, including membrane proteins, against large chem. libraries.  Thermal denaturation-based methods are independent of protein function, which is esp. useful for the identification of orphan protein function.  Here, we review applications of thermal denaturation-based methods in characterizing protein stability and ligand binding, and also provide information on protocol modifications that may further increase throughput.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH8b9M2zSFsbVg90H21EOLACvtfcHk0li4gu7FKYCjVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xltl2mt7s%253D&md5=4631ad4237d7050f1cb460ee89490f0d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1089%2Fadt.2011.0390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2011.0390%26sid%3Dliteratum%253Aachs%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThermal%2520denaturation%2520assays%2520in%2520chemical%2520biology%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2012%26volume%3D10%26spage%3D128%26epage%3D136%26doi%3D10.1089%2Fadt.2011.0390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Homola, J.</span><span> </span><span class="NLM_article-title">Surface plasmon resonance sensors for detection of chemical and biological species</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">462</span><span class="NLM_x">–</span> <span class="NLM_lpage">493</span><span class="refDoi"> DOI: 10.1021/cr068107d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr068107d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFCiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2008&pages=462-493&author=J.+Homola&title=Surface+plasmon+resonance+sensors+for+detection+of+chemical+and+biological+species&doi=10.1021%2Fcr068107d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Surface plasmon resonance sensors for detection of chemical and biological species</span></div><div class="casAuthors">Homola, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-493</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  This paper reviews advances in surface plasmon resonance (SPR) sensor technol. and its applications since the year 2000.  The review focuses on SPR sensors employing conventional (unlocalized) surface plasmons propagating along planar structures and their applications for detection of chem. and biol. species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwiJkLw_n13rVg90H21EOLACvtfcHk0li4gu7FKYCjVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFCiu7Y%253D&md5=ab7e19092725e31728b5ff7e73c921b6</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fcr068107d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr068107d%26sid%3Dliteratum%253Aachs%26aulast%3DHomola%26aufirst%3DJ.%26atitle%3DSurface%2520plasmon%2520resonance%2520sensors%2520for%2520detection%2520of%2520chemical%2520and%2520biological%2520species%26jtitle%3DChem.%2520Rev.%26date%3D2008%26volume%3D108%26spage%3D462%26epage%3D493%26doi%3D10.1021%2Fcr068107d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Pierce, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandretto, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, G. W.</span><span> </span><span class="NLM_article-title">Kinase inhibitors and the case for CH···O hydrogen bonds in protein-ligand binding</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">567</span><span class="NLM_x">–</span> <span class="NLM_lpage">576</span><span class="refDoi"> DOI: 10.1002/prot.10259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1002%2Fprot.10259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=12402365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD38XptlGru7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2002&pages=567-576&author=A.+C.+Pierceauthor=K.+L.+Sandrettoauthor=G.+W.+Bemis&title=Kinase+inhibitors+and+the+case+for+CH%C2%B7%C2%B7%C2%B7O+hydrogen+bonds+in+protein-ligand+binding&doi=10.1002%2Fprot.10259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors and the case for CH···O hydrogen bonds in protein-ligand binding</span></div><div class="casAuthors">Pierce, Albert C.; Sandretto, Kathryn L.; Bemis, Guy W.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">567-576</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Although the hydrogen bond is known to be an important mediator of intermol. interactions, there has yet to be an anal. of the role of CH···O hydrogen bonds in protein-ligand complexes.  In this work, we present evidence for such nonstandard hydrogen bonds from a survey of arom. ligands in 184 kinase crystal structures and 358 high-resoln. structures from the Protein Data Bank.  CH groups adjacent to the pos. charged nitrogen of nicotinamide exhibit geometric preferences strongly suggestive of hydrogen bonding interactions, as do heterocyclic CH groups in kinase ligands, while other arom. CH groups do not exhibit these characteristics.  Ab initio calcns. reveal a considerable range of CH···O hydrogen bonding potentials among different arom. ring systems, with nicotinamide and heterocycles preferred in kinase inhibitors showing particularly favorable interactions.  These results provide compelling evidence for the existence of CH···O hydrogen bonds in protein-ligand interactions, as well as information on the relative strength of various arom. CH donors.  Such knowledge will be of considerable value in protein modeling, ligand design, and structure-activity anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ALx7F6vIXLVg90H21EOLACvtfcHk0ljhFd5LsGZBoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptlGru7c%253D&md5=641cc069501f9243df3ddc35cd3aa449</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fprot.10259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10259%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BC.%26aulast%3DSandretto%26aufirst%3DK.%2BL.%26aulast%3DBemis%26aufirst%3DG.%2BW.%26atitle%3DKinase%2520inhibitors%2520and%2520the%2520case%2520for%2520CH%25C2%25B7%25C2%25B7%25C2%25B7O%2520hydrogen%2520bonds%2520in%2520protein-ligand%2520binding%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2002%26volume%3D49%26spage%3D567%26epage%3D576%26doi%3D10.1002%2Fprot.10259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alex, A.</span><span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eZtnTPsPW6gHbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Springthorpe, B.</span><span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">881</span><span class="NLM_x">–</span> <span class="NLM_lpage">890</span><span class="refDoi"> DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881-890&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0ljhFd5LsGZBoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D881%26epage%3D890%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Eswaran, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debreczeni, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turnbull, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Crystal Structures of the p21-activated kinases PAK4, PAK5, and PAK6 reveal catalytic domain plasticity of active group II PAKs</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">201</span><span class="NLM_x">–</span> <span class="NLM_lpage">213</span><span class="refDoi"> DOI: 10.1016/j.str.2007.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2Fj.str.2007.01.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=201-213&author=J.+Eswaranauthor=W.+H.+Leeauthor=J.+E.+Debreczeniauthor=P.+Filippakopoulosauthor=A.+Turnbullauthor=O.+Fedorovauthor=S.+W.+Deaconauthor=J.+R.+Petersonauthor=S.+Knapp&title=Crystal+Structures+of+the+p21-activated+kinases+PAK4%2C+PAK5%2C+and+PAK6+reveal+catalytic+domain+plasticity+of+active+group+II+PAKs&doi=10.1016%2Fj.str.2007.01.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2007.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2007.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DEswaran%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DW.%2BH.%26aulast%3DDebreczeni%26aufirst%3DJ.%2BE.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DTurnbull%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DDeacon%26aufirst%3DS.%2BW.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DCrystal%2520Structures%2520of%2520the%2520p21-activated%2520kinases%2520PAK4%252C%2520PAK5%252C%2520and%2520PAK6%2520reveal%2520catalytic%2520domain%2520plasticity%2520of%2520active%2520group%2520II%2520PAKs%26jtitle%3DStructure%26date%3D2007%26volume%3D15%26spage%3D201%26epage%3D213%26doi%3D10.1016%2Fj.str.2007.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">van Linden, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">KLIFS: a knowledge-based structural database to navigate kinase-ligand interaction space</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="NLM_x">–</span> <span class="NLM_lpage">277</span><span class="refDoi"> DOI: 10.1021/jm400378w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400378w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=249-277&author=O.+P.+van+Lindenauthor=A.+J.+Kooistraauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=KLIFS%3A+a+knowledge-based+structural+database+to+navigate+kinase-ligand+interaction+space&doi=10.1021%2Fjm400378w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: A Knowledge-Based Structural Database To Navigate Kinase-Ligand Interaction Space</span></div><div class="casAuthors">van Linden, Oscar P. J.; Kooistra, Albert J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-277</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein kinases regulate the majority of signal transduction pathways in cells and have become important targets for the development of designer drugs.  We present a systematic anal. of kinase-ligand interactions in all regions of the catalytic cleft of all 1252 human kinase-ligand cocrystal structures present in the Protein Data Bank (PDB).  The kinase-ligand interaction fingerprints and structure database (KLIFS) contains a consistent alignment of 85 kinase ligand binding site residues that enables the identification of family specific interaction features and classification of ligands according to their binding modes.  We illustrate how systematic mining of kinase-ligand interaction space gives new insights into how conserved and selective kinase interaction hot spots can accommodate the large diversity of chem. scaffolds in kinase ligands.  These analyses lead to an improved understanding of the structural requirements of kinase binding that will be useful in ligand discovery and design studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4tMmT6S-vRrVg90H21EOLACvtfcHk0lieOrNnUIk9rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P&md5=8519942e1703ba04f64797e07df4b712</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm400378w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400378w%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKLIFS%253A%2520a%2520knowledge-based%2520structural%2520database%2520to%2520navigate%2520kinase-ligand%2520interaction%2520space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D249%26epage%3D277%26doi%3D10.1021%2Fjm400378w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanev, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Linden, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">KLIFS: a structural kinase-ligand interaction database</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">D365</span><span class="NLM_x">–</span> <span class="NLM_lpage">371</span><span class="refDoi"> DOI: 10.1093/nar/gkv1082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1093%2Fnar%2Fgkv1082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=26496949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D365-371&author=A.+J.+Kooistraauthor=G.+K.+Kanevauthor=O.+P.+van+Lindenauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=KLIFS%3A+a+structural+kinase-ligand+interaction+database&doi=10.1093%2Fnar%2Fgkv1082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: a structural kinase-ligand interaction database</span></div><div class="casAuthors">Kooistra, Albert J.; Kanev, Georgi K.; van Linden, Oscar P. J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D365-D371</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein kinases play a crucial role in cell signaling and are important drug targets in several therapeutic areas.  The KLIFS database contains detailed structural kinase-ligand interaction information derived from all (>2900) structures of catalytic domains of human and mouse protein kinases deposited in the Protein Data Bank in order to provide insights into the structural determinants of kinase-ligand binding and selectivity.  The kinase structures have been processed in a consistent manner by systematically analyzing the structural features and mol. interaction fingerprints (IFPs) of a predefined set of 85 binding site residues with bound ligands.  KLIFS has been completely rebuilt and extended (>65% more structures) since its first release as a data set, including: novel automated annotation methods for (i) the assessment of ligand-targeted subpockets and the anal. of (ii) DFG and (iii) αC-helix conformations; improved and automated protocols for (iv) the generation of sequence/structure alignments, (v) the curation of ligand atom and bond typing for accurate IFP anal. and (vi) weekly database updates.  KLIFS is now accessible via a website (http://klifs.vucompmedchem. nl) that provides a comprehensive visual presentation of different types of chem., biol. and structural chemogenomics data, and allows the user to easily access, compare, search and download the data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocfCaKwwgOlbVg90H21EOLACvtfcHk0lj0fmWAwIHudg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfP&md5=bc093b02225987fe791a2d41738e4d99</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1082%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DKanev%26aufirst%3DG.%2BK.%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKLIFS%253A%2520a%2520structural%2520kinase-ligand%2520interaction%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD365%26epage%3D371%26doi%3D10.1093%2Fnar%2Fgkv1082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Vulpetti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bosotti, R.</span><span> </span><span class="NLM_article-title">Sequence and structural analysis of kinase ATP pocket residues</span> <span class="citation_source-journal">Farmaco</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">759</span><span class="NLM_x">–</span> <span class="NLM_lpage">765</span><span class="refDoi"> DOI: 10.1016/j.farmac.2004.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2Fj.farmac.2004.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=15474052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlWms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2004&pages=759-765&author=A.+Vulpettiauthor=R.+Bosotti&title=Sequence+and+structural+analysis+of+kinase+ATP+pocket+residues&doi=10.1016%2Fj.farmac.2004.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence and structural analysis of kinase ATP pocket residues</span></div><div class="casAuthors">Vulpetti, Anna; Bosotti, Roberta</div><div class="citationInfo"><span class="NLM_cas:title">Farmaco</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">759-765</span>CODEN:
                <span class="NLM_cas:coden">FRMCE8</span>;
        ISSN:<span class="NLM_cas:issn">0014-827X</span>.
    
            (<span class="NLM_cas:orgname">Editions Scientifiques et Medicales Elsevier</span>)
        </div><div class="casAbstract">Protein kinases represent one of the largest known families of enzymes.  Most kinases bind ATP and most synthetic kinase inhibitors are ATP-competitive, which makes selectivity a potential problem.  However, despite the high sequence similarity in the ATP binding pocket, several groups including ours have been able to develop highly potent and selective ATP-competitive inhibitors.  To systematically aid the design of specific inhibitors in our protein kinase projects, we aligned all known three-dimensional structures and all known sequences of human protein kinases.  We identified a set of 38 residues that make up the ATP pocket and analyzed the variability among these residues.  The most variable residues in the ATP pocket are targeted to design specificity into inhibitors in our various kinase projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKrQCe0MaGh7Vg90H21EOLACvtfcHk0lj0fmWAwIHudg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlWms7o%253D&md5=d710962f752fcd893880696e34bf35a1</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.farmac.2004.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.farmac.2004.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DVulpetti%26aufirst%3DA.%26aulast%3DBosotti%26aufirst%3DR.%26atitle%3DSequence%2520and%2520structural%2520analysis%2520of%2520kinase%2520ATP%2520pocket%2520residues%26jtitle%3DFarmaco%26date%3D2004%26volume%3D59%26spage%3D759%26epage%3D765%26doi%3D10.1016%2Fj.farmac.2004.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Ruegg, U. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillian, B.</span><span> </span><span class="NLM_article-title">Staurosporine, K-252 and UCN-01: potent but nonspecific inhibitors of protein kinases</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">218</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.1016/0165-6147(89)90263-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2F0165-6147%2889%2990263-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1989&pages=218-220&author=U.+T.+Rueggauthor=B.+Gillian&title=Staurosporine%2C+K-252+and+UCN-01%3A+potent+but+nonspecific+inhibitors+of+protein+kinases&doi=10.1016%2F0165-6147%2889%2990263-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2F0165-6147%2889%2990263-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0165-6147%252889%252990263-0%26sid%3Dliteratum%253Aachs%26aulast%3DRuegg%26aufirst%3DU.%2BT.%26aulast%3DGillian%26aufirst%3DB.%26atitle%3DStaurosporine%252C%2520K-252%2520and%2520UCN-01%253A%2520potent%2520but%2520nonspecific%2520inhibitors%2520of%2520protein%2520kinases%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D1989%26volume%3D10%26spage%3D218%26epage%3D220%26doi%3D10.1016%2F0165-6147%2889%2990263-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Gilli, G. G. a. P.</span> <span class="citation_source-book">The Nature of the Hydrogen Bond</span>; <span class="NLM_publisher-name">Oxford University Press Inc.</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1093%2Facprof%3Aoso%2F9780199558964.001.0001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=G.+G.+a.+P.+Gilli&title=The+Nature+of+the+Hydrogen+Bond"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Facprof%3Aoso%2F9780199558964.001.0001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1093%252Facprof%253Aoso%252F9780199558964.001.0001%26sid%3Dliteratum%253Aachs%26aulast%3DGilli%26aufirst%3DG.%2BG.%2Ba.%2BP.%26btitle%3DThe%2520Nature%2520of%2520the%2520Hydrogen%2520Bond%26pub%3DOxford%2520University%2520Press%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Wang, R. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. M.</span><span> </span><span class="NLM_article-title">Further development and validation of empirical scoring functions for structure-based binding affinity prediction</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1023/A:1016357811882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1023%2FA%3A1016357811882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=12197663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD38XltlSqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=11-26&author=R.+X.+Wangauthor=L.+H.+Laiauthor=S.+M.+Wang&title=Further+development+and+validation+of+empirical+scoring+functions+for+structure-based+binding+affinity+prediction&doi=10.1023%2FA%3A1016357811882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Further development and validation of empirical scoring functions for structure-based binding affinity prediction</span></div><div class="casAuthors">Wang, Renxiao; Lai, Luhua; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-26</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">New empirical scoring functions have been developed to est. the binding affinity of a given protein-ligand complex with known three-dimensional structure.  These scoring functions include terms accounting for van der Waals interaction, hydrogen bonding, deformation penalty, and hydrophobic effect.  A special feature is that three different algorithms have been implemented to calc. the hydrophobic effect term, which results in three parallel scoring functions.  All three scoring functions are calibrated through multivariate regression anal. of a set of 200 protein-ligand complexes and they reproduce the binding free energies of the entire training set with std. deviations of 2.2 kcal/mol, 2.1 kcal/mol, and 2.0 kcal/mol, resp.  These three scoring functions are further combined into a consensus scoring function, X-CSCORE.  When tested on an independent set of 30 protein-ligand complexes, X-CSCORE is able to predict their binding free energies with a std. deviation of 2.2 kcal/mol.  The potential application of X-CSCORE to mol. docking is also investigated.  Our results show that this consensus scoring function improves the docking accuracy considerably when compared to the conventional force field computation used for mol. docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqULje6PqR73rVg90H21EOLACvtfcHk0li5OgWCGtSg-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltlSqs7k%253D&md5=f77fd72d168df67d7fb66a88bc3b6cd9</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1023%2FA%3A1016357811882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1016357811882%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%2BX.%26aulast%3DLai%26aufirst%3DL.%2BH.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DFurther%2520development%2520and%2520validation%2520of%2520empirical%2520scoring%2520functions%2520for%2520structure-based%2520binding%2520affinity%2520prediction%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D11%26epage%3D26%26doi%3D10.1023%2FA%3A1016357811882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Wolber, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span> </span><span class="NLM_article-title">LigandScout: 3-d pharmacophores derived from protein-bound Ligands and their use as virtual screening filters</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">160</span><span class="NLM_x">–</span> <span class="NLM_lpage">169</span><span class="refDoi"> DOI: 10.1021/ci049885e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049885e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=160-169&author=G.+Wolberauthor=T.+Langer&title=LigandScout%3A+3-d+pharmacophores+derived+from+protein-bound+Ligands+and+their+use+as+virtual+screening+filters&doi=10.1021%2Fci049885e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">LigandScout: 3-D pharmacophores derived from protein-bound ligands and their use as virtual screening filters</span></div><div class="casAuthors">Wolber Gerhard; Langer Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">160-9</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">From the historically grown archive of protein-ligand complexes in the Protein Data Bank small organic ligands are extracted and interpreted in terms of their chemical characteristics and features.  Subsequently, pharmacophores representing ligand-receptor interaction are derived from each of these small molecules and its surrounding amino acids.  Based on a defined set of only six types of chemical features and volume constraints, three-dimensional pharmacophore models are constructed, which are sufficiently selective to identify the described binding mode and are thus a useful tool for in-silico screening of large compound databases.  The algorithms for ligand extraction and interpretation as well as the pharmacophore creation technique from the automatically interpreted data are presented and applied to a rhinovirus capsid complex as application example.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX2BAiYWy6n3SfW6udTcc2eZ6R2m6PcaXzrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtg%253D%253D&md5=31f8ff7a2fa4c9411d54b001b7e1da50</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci049885e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049885e%26sid%3Dliteratum%253Aachs%26aulast%3DWolber%26aufirst%3DG.%26aulast%3DLanger%26aufirst%3DT.%26atitle%3DLigandScout%253A%25203-d%2520pharmacophores%2520derived%2520from%2520protein-bound%2520Ligands%2520and%2520their%2520use%2520as%2520virtual%2520screening%2520filters%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D160%26epage%3D169%26doi%3D10.1021%2Fci049885e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Cai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span> </span><span class="NLM_article-title">Overexpression of P21-activated kinase 4 is associated with poor prognosis in non-small cell lung cancer and promotes migration and invasion</span> <span class="citation_source-journal">J. Exp. Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">48</span><span class="refDoi"> DOI: 10.1186/s13046-015-0165-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1186%2Fs13046-015-0165-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=25975262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BC2MfktVSjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=48&author=S.+Caiauthor=Z.+Yeauthor=X.+Wangauthor=Y.+Panauthor=Y.+Wengauthor=S.+Laoauthor=H.+Weiauthor=L.+Li&title=Overexpression+of+P21-activated+kinase+4+is+associated+with+poor+prognosis+in+non-small+cell+lung+cancer+and+promotes+migration+and+invasion&doi=10.1186%2Fs13046-015-0165-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of P21-activated kinase 4 is associated with poor prognosis in non-small cell lung cancer and promotes migration and invasion</span></div><div class="casAuthors">Cai Songwang; Weng Yimin; Lao Sen; Ye Zhiqiang; Wang Xiaohong; Pan Yuhang; Wei Hongbo; Li Lian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of experimental & clinical cancer research : CR</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">48</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  P21-activated kinase 4 (PAK4), an effector of the Rho family protein Cdc42, is an important oncogene whose expression is increased in many human cancers and is generally positively correlated with advanced disease and decreased survival.  However, little is known about the expression and biological function of PAK4 in human non-small cell lung cancer (NSCLC).  METHODS:  PAK4 expression in NSCLC tissues and adjacent non-tumor tissues were assessed by immunohistochemistry, real-time PCR, and western blotting.  Prognostic value of PAK4 expression was evaluated by Kaplan-Meier analysis and Cox regression. siRNA-mediated gene silencing and protein kinase assay was applied to demonstrate the role and the mechanism of PAK4 in lung cancer cell migration, invasion.  RESULTS:  The results showed that PAK4 was overexpressed in NSCLC cell lines and human NSCLC tissues.  PAK4 expression was detected both in the membranes and cytoplasm of NSCLC cancer cells in vivo.  Moreover, increased expression of PAK4 was associated with metastasis, shorter overall survival, advanced stage of NSCLC.  Furthermore, PAK4 expression was positively correlated with phosphorylation of LIMK1 expression levels.  Knockdown of PAK4 in NSCLC cell lines led to reduce the phosphorylation of LIMK1, which resulted in decrease of the cell migration and invasion.  In addition, PAK4 bound to LIMK1 directly and activated it via phosphorylation.  CONCLUSIONS:  These data demonstrate that PAK4 mediated LIMK1 phosphorylation regulates the migration and invasion in NSCLC.  Therefore, PAK4 might be a significant prognostic marker and potential therapeutic molecular target in NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT7aG3uVGW52fcsfMJQC1ZdfW6udTcc2eZ6R2m6PcaXzrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MfktVSjuw%253D%253D&md5=c725f528e73c73e1fe9ccfe4bd80ba0e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1186%2Fs13046-015-0165-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-015-0165-2%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DS.%26aulast%3DYe%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DWeng%26aufirst%3DY.%26aulast%3DLao%26aufirst%3DS.%26aulast%3DWei%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DL.%26atitle%3DOverexpression%2520of%2520P21-activated%2520kinase%25204%2520is%2520associated%2520with%2520poor%2520prognosis%2520in%2520non-small%2520cell%2520lung%2520cancer%2520and%2520promotes%2520migration%2520and%2520invasion%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D34%26spage%3D48%26doi%3D10.1186%2Fs13046-015-0165-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Mitcheson, J. S.</span><span> </span><span class="NLM_article-title">hERG potassium channels and the structural basis of drug-induced arrhythmias</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1005</span><span class="NLM_x">–</span> <span class="NLM_lpage">1010</span><span class="refDoi"> DOI: 10.1021/tx800035b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx800035b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2008&pages=1005-1010&author=J.+S.+Mitcheson&title=hERG+potassium+channels+and+the+structural+basis+of+drug-induced+arrhythmias&doi=10.1021%2Ftx800035b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Ftx800035b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx800035b%26sid%3Dliteratum%253Aachs%26aulast%3DMitcheson%26aufirst%3DJ.%2BS.%26atitle%3DhERG%2520potassium%2520channels%2520and%2520the%2520structural%2520basis%2520of%2520drug-induced%2520arrhythmias%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26spage%3D1005%26epage%3D1010%26doi%3D10.1021%2Ftx800035b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Su, Q. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ioannidis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuaqui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almeida, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alimzhanov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bebernitz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Block, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huszar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costa, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinda, M.</span><span> </span><span class="NLM_article-title">Discovery of 1-methyl-1<i>H</i>-imidazole derivatives as potent Jak2 inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">144</span><span class="NLM_x">–</span> <span class="NLM_lpage">158</span><span class="refDoi"> DOI: 10.1021/jm401546n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401546n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWltLnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=144-158&author=Q.+B.+Suauthor=S.+Ioannidisauthor=C.+Chuaquiauthor=L.+Almeidaauthor=M.+Alimzhanovauthor=G.+Bebernitzauthor=K.+Bellauthor=M.+Blockauthor=T.+Howardauthor=S.+Huangauthor=D.+Huszarauthor=J.+A.+Readauthor=C.+R.+Costaauthor=J.+Shiauthor=M.+Suauthor=M.+W.+Yeauthor=M.+Zinda&title=Discovery+of+1-methyl-1H-imidazole+derivatives+as+potent+Jak2+inhibitors&doi=10.1021%2Fjm401546n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 1-Methyl-1H-imidazole Derivatives as Potent Jak2 Inhibitors</span></div><div class="casAuthors">Su, Qibin; Ioannidis, Stephanos; Chuaqui, Claudio; Almeida, Lynsie; Alimzhanov, Marat; Bebernitz, Geraldine; Bell, Kirsten; Block, Michael; Howard, Tina; Huang, Shan; Huszar, Dennis; Read, Jon A.; Rivard Costa, Caroline; Shi, Jie; Su, Mei; Ye, Minwei; Zinda, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">144-158</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure based design, synthesis, and biol. evaluation of a novel series of 1-methyl-1H-imidazole, as potent Jak2 inhibitors to modulate the Jak/STAT pathway, are described.  Using the C-ring fragment from our first clin. candidate AZD1480 (I), optimization of the series led to the discovery of compd. II, a potent, orally bioavailable Jak2 inhibitor.  Compd. II displayed a high level of cellular activity in hematopoietic cell lines harboring the V617F mutation and in murine BaF3 TEL-Jak2 cells.  Compd. II demonstrated significant tumor growth inhibition in a UKE-1 xenograft model within a well-tolerated dose range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqczgU5RpFv1bVg90H21EOLACvtfcHk0ljPwnJ6dZSTnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWltLnL&md5=88c57119ec77d084a91a1123fd9dea86</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm401546n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401546n%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DQ.%2BB.%26aulast%3DIoannidis%26aufirst%3DS.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DAlmeida%26aufirst%3DL.%26aulast%3DAlimzhanov%26aufirst%3DM.%26aulast%3DBebernitz%26aufirst%3DG.%26aulast%3DBell%26aufirst%3DK.%26aulast%3DBlock%26aufirst%3DM.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DHuszar%26aufirst%3DD.%26aulast%3DRead%26aufirst%3DJ.%2BA.%26aulast%3DCosta%26aufirst%3DC.%2BR.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DM.%26aulast%3DYe%26aufirst%3DM.%2BW.%26aulast%3DZinda%26aufirst%3DM.%26atitle%3DDiscovery%2520of%25201-methyl-1H-imidazole%2520derivatives%2520as%2520potent%2520Jak2%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D144%26epage%3D158%26doi%3D10.1021%2Fjm401546n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Sakamoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriya, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuge, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haga, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nonoshita, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakuraba, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirohashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanno, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwaasa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gomori, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishihara, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuroda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanatani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukami, T.</span><span> </span><span class="NLM_article-title">Novel orally active NPY Y5 receptor antagonists: synthesis and structure-activity relationship of spiroindoline class compounds</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">5015</span><span class="NLM_x">–</span> <span class="NLM_lpage">5026</span><span class="refDoi"> DOI: 10.1016/j.bmc.2009.05.064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2Fj.bmc.2009.05.064" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=5015-5026&author=T.+Sakamotoauthor=M.+Moriyaauthor=H.+Tsugeauthor=T.+Takahashiauthor=Y.+Hagaauthor=K.+Nonoshitaauthor=O.+Okamotoauthor=H.+Takahashiauthor=A.+Sakurabaauthor=T.+Hirohashiauthor=T.+Shibataauthor=T.+Kannoauthor=J.+Itoauthor=H.+Iwaasaauthor=A.+Gomoriauthor=A.+Ishiharaauthor=T.+Fukurodaauthor=A.+Kanataniauthor=T.+Fukami&title=Novel+orally+active+NPY+Y5+receptor+antagonists%3A+synthesis+and+structure-activity+relationship+of+spiroindoline+class+compounds&doi=10.1016%2Fj.bmc.2009.05.064"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2009.05.064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2009.05.064%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DT.%26aulast%3DMoriya%26aufirst%3DM.%26aulast%3DTsuge%26aufirst%3DH.%26aulast%3DTakahashi%26aufirst%3DT.%26aulast%3DHaga%26aufirst%3DY.%26aulast%3DNonoshita%26aufirst%3DK.%26aulast%3DOkamoto%26aufirst%3DO.%26aulast%3DTakahashi%26aufirst%3DH.%26aulast%3DSakuraba%26aufirst%3DA.%26aulast%3DHirohashi%26aufirst%3DT.%26aulast%3DShibata%26aufirst%3DT.%26aulast%3DKanno%26aufirst%3DT.%26aulast%3DIto%26aufirst%3DJ.%26aulast%3DIwaasa%26aufirst%3DH.%26aulast%3DGomori%26aufirst%3DA.%26aulast%3DIshihara%26aufirst%3DA.%26aulast%3DFukuroda%26aufirst%3DT.%26aulast%3DKanatani%26aufirst%3DA.%26aulast%3DFukami%26aufirst%3DT.%26atitle%3DNovel%2520orally%2520active%2520NPY%2520Y5%2520receptor%2520antagonists%253A%2520synthesis%2520and%2520structure-activity%2520relationship%2520of%2520spiroindoline%2520class%2520compounds%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D5015%26epage%3D5026%26doi%3D10.1016%2Fj.bmc.2009.05.064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljPwnJ6dZSTnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liL6BZbTeOh8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusoff, W. H.</span><span> </span><span class="NLM_article-title">Relationship between the inhibition constant (Ki) and the concentration of inhibitor which causes 50% inhibition (IC<sub>50</sub>) of an enzymatic reaction</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3099</span><span class="NLM_x">–</span> <span class="NLM_lpage">3108</span><span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=Y.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28Ki%29+and+the+concentration+of+inhibitor+which+causes+50%25+inhibition+%28IC50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0liL6BZbTeOh8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528Ki%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2525%2520inhibition%2520%2528IC50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Song, H. Y.; Smith, R. A. A.; Hao, C. Z.; Wang, J.; Dahlroth, S. K. S.; Cheng, M. S.; Fernig, D. G.; Gorelik, S.; Nurcombe, V.</span> Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Song%2C+H.+Y.%3B+Smith%2C+R.+A.+A.%3B+Hao%2C+C.+Z.%3B+Wang%2C+J.%3B+Dahlroth%2C+S.+K.+S.%3B+Cheng%2C+M.+S.%3B+Fernig%2C+D.+G.%3B+Gorelik%2C+S.%3B+Nurcombe%2C+V.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DH.%2BY." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Ma, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, J.</span><span> </span><span class="NLM_article-title">Design and optimization of a series of 1-sulfonylpyrazolo[4,3-b]pyridines as selective c-Met inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">2513</span><span class="NLM_x">–</span> <span class="NLM_lpage">2529</span><span class="refDoi"> DOI: 10.1021/jm502018y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm502018y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2513-2529&author=Y.+Maauthor=G.+Sunauthor=D.+Chenauthor=X.+Pengauthor=Y.+L.+Chenauthor=Y.+Suauthor=Y.+Jiauthor=J.+Liangauthor=X.+Wangauthor=L.+Chenauthor=J.+Dingauthor=B.+Xiongauthor=J.+Aiauthor=M.+Gengauthor=J.+Shen&title=Design+and+optimization+of+a+series+of+1-sulfonylpyrazolo%5B4%2C3-b%5Dpyridines+as+selective+c-Met+inhibitors&doi=10.1021%2Fjm502018y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm502018y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm502018y%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%2BL.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DJ.%26atitle%3DDesign%2520and%2520optimization%2520of%2520a%2520series%2520of%25201-sulfonylpyrazolo%255B4%252C3-b%255Dpyridines%2520as%2520selective%2520c-Met%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2513%26epage%3D2529%26doi%3D10.1021%2Fjm502018y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 18 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Selena Chacón Simon, Feng Wang, Lance R. Thomas, Jason Phan, Bin Zhao, Edward T. Olejniczak, Jonathan D. Macdonald, J. Grace Shaw, Caden Schlund, William Payne, Joy Creighton, Shaun R. Stauffer, Alex G. Waterson, William P. Tansey, <span class="NLM_string-name hlFld-ContribAuthor">Stephen W. Fesik</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of WD Repeat-Containing Protein 5 (WDR5)–MYC Inhibitors Using Fragment-Based Methods and Structure-Based Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (8)
                                     , 4315-4333. <a href="https://doi.org/10.1021/acs.jmedchem.0c00224" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00224</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00224%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BWD%252BRepeat-Containing%252BProtein%252B5%252B%252528WDR5%252529%2525E2%252580%252593MYC%252BInhibitors%252BUsing%252BFragment-Based%252BMethods%252Band%252BStructure-Based%252BDesign%26aulast%3DChac%25C3%25B3n%2BSimon%26aufirst%3DSelena%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06022020%26date%3D09042020%26date%3D30032020%26volume%3D63%26issue%3D8%26spage%3D4315%26epage%3D4333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ruofei  Zhu</span>, <span class="hlFld-ContribAuthor ">Kaei  Nasu</span>, <span class="hlFld-ContribAuthor ">Naoki  Hijiya</span>, <span class="hlFld-ContribAuthor ">Masato  Yoshihashi</span>, <span class="hlFld-ContribAuthor ">Tomoko  Hirakawa</span>, <span class="hlFld-ContribAuthor ">Yoko  Aoyagi</span>, <span class="hlFld-ContribAuthor ">Hisashi  Narahara</span>. </span><span class="cited-content_cbyCitation_article-title">hsa-miR-199a-3p Inhibits Motility, Invasiveness, and Contractility of Ovarian Endometriotic Stromal Cells. </span><span class="cited-content_cbyCitation_journal-name">Reproductive Sciences</span><span> <strong>2021,</strong> <em>362 </em><a href="https://doi.org/10.1007/s43032-021-00604-4" title="DOI URL">https://doi.org/10.1007/s43032-021-00604-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s43032-021-00604-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs43032-021-00604-4%26sid%3Dliteratum%253Aachs%26jtitle%3DReproductive%2520Sciences%26atitle%3Dhsa-miR-199a-3p%252BInhibits%252BMotility%25252C%252BInvasiveness%25252C%252Band%252BContractility%252Bof%252BOvarian%252BEndometriotic%252BStromal%252BCells%26aulast%3DZhu%26aufirst%3DRuofei%26date%3D2021%26date%3D2021%26volume%3D362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias R.  Bauer</span>, <span class="hlFld-ContribAuthor ">Paolo  Di Fruscia</span>, <span class="hlFld-ContribAuthor ">Simon C. C.  Lucas</span>, <span class="hlFld-ContribAuthor ">Iacovos N.  Michaelides</span>, <span class="hlFld-ContribAuthor ">Jennifer E.  Nelson</span>, <span class="hlFld-ContribAuthor ">R. Ian  Storer</span>, <span class="hlFld-ContribAuthor ">Benjamin C.  Whitehurst</span>. </span><span class="cited-content_cbyCitation_article-title">Put a ring on it: application of small aliphatic rings in medicinal chemistry. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>12 </em>
                                    (4)
                                     , 448-471. <a href="https://doi.org/10.1039/D0MD00370K" title="DOI URL">https://doi.org/10.1039/D0MD00370K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00370K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00370K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DPut%252Ba%252Bring%252Bon%252Bit%25253A%252Bapplication%252Bof%252Bsmall%252Baliphatic%252Brings%252Bin%252Bmedicinal%252Bchemistry%26aulast%3DBauer%26aufirst%3DMatthias%2BR.%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D4%26spage%3D448%26epage%3D471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Georgi K  Kanev</span>, <span class="hlFld-ContribAuthor ">Chris  de Graaf</span>, <span class="hlFld-ContribAuthor ">Bart A  Westerman</span>, <span class="hlFld-ContribAuthor ">Iwan J P  de Esch</span>, <span class="hlFld-ContribAuthor ">Albert J  Kooistra</span>. </span><span class="cited-content_cbyCitation_article-title">KLIFS: an overhaul after the first 5 years of supporting kinase research. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2021,</strong> <em>49 </em>
                                    (D1)
                                     , D562-D569. <a href="https://doi.org/10.1093/nar/gkaa895" title="DOI URL">https://doi.org/10.1093/nar/gkaa895</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkaa895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkaa895%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DKLIFS%25253A%252Ban%252Boverhaul%252Bafter%252Bthe%252Bfirst%252B5%252Byears%252Bof%252Bsupporting%252Bkinase%252Bresearch%26aulast%3DKanev%26aufirst%3DGeorgi%2BK%26date%3D2021%26date%3D2020%26volume%3D49%26issue%3DD1%26spage%3DD562%26epage%3DD569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qiaohua  Qin</span>, <span class="hlFld-ContribAuthor ">Tianxiao  Wu</span>, <span class="hlFld-ContribAuthor ">Wenbo  Yin</span>, <span class="hlFld-ContribAuthor ">Yixiang  Sun</span>, <span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Ruifeng  Wang</span>, <span class="hlFld-ContribAuthor ">Jing  Guo</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of 2,4‐diaminopyrimidine derivatives targeting p21‐activated kinase 4: Biological evaluation and docking studies. </span><span class="cited-content_cbyCitation_journal-name">Archiv der Pharmazie</span><span> <strong>2020,</strong> <em>353 </em>
                                    (10)
                                     , 2000097. <a href="https://doi.org/10.1002/ardp.202000097" title="DOI URL">https://doi.org/10.1002/ardp.202000097</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ardp.202000097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fardp.202000097%26sid%3Dliteratum%253Aachs%26jtitle%3DArchiv%2520der%2520Pharmazie%26atitle%3DDiscovery%252Bof%252B2%25252C4%2525E2%252580%252590diaminopyrimidine%252Bderivatives%252Btargeting%252Bp21%2525E2%252580%252590activated%252Bkinase%252B4%25253A%252BBiological%252Bevaluation%252Band%252Bdocking%252Bstudies%26aulast%3DQin%26aufirst%3DQiaohua%26date%3D2020%26date%3D2020%26volume%3D353%26issue%3D10%26spage%3D2000097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mingliang  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiaobao  Fang</span>, <span class="hlFld-ContribAuthor ">Cong  Wang</span>, <span class="hlFld-ContribAuthor ">Yi  Hua</span>, <span class="hlFld-ContribAuthor ">Chen  Huang</span>, <span class="hlFld-ContribAuthor ">Meng  Wang</span>, <span class="hlFld-ContribAuthor ">Lu  Zhu</span>, <span class="hlFld-ContribAuthor ">Zixu  Wang</span>, <span class="hlFld-ContribAuthor ">Yuhan  Gao</span>, <span class="hlFld-ContribAuthor ">Tianyi  Zhang</span>, <span class="hlFld-ContribAuthor ">Haichun  Liu</span>, <span class="hlFld-ContribAuthor ">Yanmin  Zhang</span>, <span class="hlFld-ContribAuthor ">Shuai  Lu</span>, <span class="hlFld-ContribAuthor ">Tao  Lu</span>, <span class="hlFld-ContribAuthor ">Yadong  Chen</span>, <span class="hlFld-ContribAuthor ">Hongmei  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of 1H-indazole-3-carboxamide derivatives as potent and selective PAK1 inhibitors with anti-tumour migration and invasion activities. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>203 </em>, 112517. <a href="https://doi.org/10.1016/j.ejmech.2020.112517" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112517</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112517%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252B1H-indazole-3-carboxamide%252Bderivatives%252Bas%252Bpotent%252Band%252Bselective%252BPAK1%252Binhibitors%252Bwith%252Banti-tumour%252Bmigration%252Band%252Binvasion%252Bactivities%26aulast%3DZhang%26aufirst%3DMingliang%26date%3D2020%26volume%3D203%26spage%3D112517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yinli  Gao</span>, <span class="hlFld-ContribAuthor ">Hanxun  Wang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">In silico studies on p21-activated kinase 4 inhibitors: comprehensive application of 3D-QSAR analysis, molecular docking, molecular dynamics simulations, and MM-GBSA calculation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2020,</strong> <em>38 </em>
                                    (14)
                                     , 4119-4133. <a href="https://doi.org/10.1080/07391102.2019.1673823" title="DOI URL">https://doi.org/10.1080/07391102.2019.1673823</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2019.1673823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2019.1673823%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DIn%252Bsilico%252Bstudies%252Bon%252Bp21-activated%252Bkinase%252B4%252Binhibitors%25253A%252Bcomprehensive%252Bapplication%252Bof%252B3D-QSAR%252Banalysis%25252C%252Bmolecular%252Bdocking%25252C%252Bmolecular%252Bdynamics%252Bsimulations%25252C%252Band%252BMM-GBSA%252Bcalculation%26aulast%3DGao%26aufirst%3DYinli%26date%3D2020%26date%3D2019%26volume%3D38%26issue%3D14%26spage%3D4119%26epage%3D4133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuan  Su</span>, <span class="hlFld-ContribAuthor ">Peilu  Song</span>, <span class="hlFld-ContribAuthor ">Hanxun  Wang</span>, <span class="hlFld-ContribAuthor ">Baichun  Hu</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Mao-Sheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Precise design of highly isoform-selective p21-activated kinase 4 inhibitors: computational insights into the selectivity mechanism through molecular dynamics simulation and binding free energy calculation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2020,</strong> <em>38 </em>
                                    (13)
                                     , 3825-3837. <a href="https://doi.org/10.1080/07391102.2019.1664330" title="DOI URL">https://doi.org/10.1080/07391102.2019.1664330</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2019.1664330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2019.1664330%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DPrecise%252Bdesign%252Bof%252Bhighly%252Bisoform-selective%252Bp21-activated%252Bkinase%252B4%252Binhibitors%25253A%252Bcomputational%252Binsights%252Binto%252Bthe%252Bselectivity%252Bmechanism%252Bthrough%252Bmolecular%252Bdynamics%252Bsimulation%252Band%252Bbinding%252Bfree%252Benergy%252Bcalculation%26aulast%3DSu%26aufirst%3DYuan%26date%3D2020%26date%3D2019%26volume%3D38%26issue%3D13%26spage%3D3825%26epage%3D3837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andreas  Kalogirou</span>, <span class="hlFld-ContribAuthor ">Christopher  Asquith</span>, <span class="hlFld-ContribAuthor ">Panayiotis  Koutentis</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of (R) and (S)-3-Chloro-5-(2,4-dimethylpiperazin-1-yl)-4H-1,2,6-thiadiazin-4-ones. </span><span class="cited-content_cbyCitation_journal-name">Molbank</span><span> <strong>2020,</strong> <em>2020 </em>
                                    (2)
                                     , M1139. <a href="https://doi.org/10.3390/M1139" title="DOI URL">https://doi.org/10.3390/M1139</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/M1139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2FM1139%26sid%3Dliteratum%253Aachs%26jtitle%3DMolbank%26atitle%3DSynthesis%252Bof%252B%252528R%252529%252Band%252B%252528S%252529-3-Chloro-5-%2525282%25252C4-dimethylpiperazin-1-yl%252529-4H-1%25252C2%25252C6-thiadiazin-4-ones%26aulast%3DKalogirou%26aufirst%3DAndreas%26date%3D2020%26date%3D2020%26volume%3D2020%26issue%3D2%26spage%3DM1139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Guo</span>, <span class="hlFld-ContribAuthor ">Tingting  Wang</span>, <span class="hlFld-ContribAuthor ">Tianxiao  Wu</span>, <span class="hlFld-ContribAuthor ">Kehan  Zhang</span>, <span class="hlFld-ContribAuthor ">Wenbo  Yin</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zhu</span>, <span class="hlFld-ContribAuthor ">Yu  Pang</span>, <span class="hlFld-ContribAuthor ">Chenzhou  Hao</span>, <span class="hlFld-ContribAuthor ">Zhonggui  He</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>, <span class="hlFld-ContribAuthor ">Yang  Liu</span>, <span class="hlFld-ContribAuthor ">Jiang  Zheng</span>, <span class="hlFld-ContribAuthor ">Jingkai  Gu</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, bioconversion, pharmacokinetic and pharmacodynamic evaluation of N-isopropyl-oxy-carbonyloxymethyl prodrugs of CZh-226, a potent and selective PAK4 inhibitor. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>186 </em>, 111878. <a href="https://doi.org/10.1016/j.ejmech.2019.111878" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111878</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111878%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%25252C%252Bbioconversion%25252C%252Bpharmacokinetic%252Band%252Bpharmacodynamic%252Bevaluation%252Bof%252BN-isopropyl-oxy-carbonyloxymethyl%252Bprodrugs%252Bof%252BCZh-226%25252C%252Ba%252Bpotent%252Band%252Bselective%252BPAK4%252Binhibitor%26aulast%3DGuo%26aufirst%3DJing%26date%3D2020%26volume%3D186%26spage%3D111878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ruifeng  Wang</span>, <span class="hlFld-ContribAuthor ">Yixuan  Chen</span>, <span class="hlFld-ContribAuthor ">Bowen  Yang</span>, <span class="hlFld-ContribAuthor ">Sijia  Yu</span>, <span class="hlFld-ContribAuthor ">Xiangxin  Zhao</span>, <span class="hlFld-ContribAuthor ">Cai  Zhang</span>, <span class="hlFld-ContribAuthor ">Chenzhou  Hao</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, biological evaluation and molecular modeling of novel 1H-pyrrolo[2,3-b]pyridine derivatives as potential anti-tumor agents. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>94 </em>, 103474. <a href="https://doi.org/10.1016/j.bioorg.2019.103474" title="DOI URL">https://doi.org/10.1016/j.bioorg.2019.103474</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2019.103474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2019.103474%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Bbiological%252Bevaluation%252Band%252Bmolecular%252Bmodeling%252Bof%252Bnovel%252B1H-pyrrolo%25255B2%25252C3-b%25255Dpyridine%252Bderivatives%252Bas%252Bpotential%252Banti-tumor%252Bagents%26aulast%3DWang%26aufirst%3DRuifeng%26date%3D2020%26volume%3D94%26spage%3D103474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hanxun  Wang</span>, <span class="hlFld-ContribAuthor ">Yinli  Gao</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Strategy Revealing the Structural Determinant of Ligand Selectivity towards Highly Similar Protein Targets. </span><span class="cited-content_cbyCitation_journal-name">Current Drug Targets</span><span> <strong>2019,</strong> <em>21 </em>
                                    (1)
                                     , 76-88. <a href="https://doi.org/10.2174/1389450120666190926113524" title="DOI URL">https://doi.org/10.2174/1389450120666190926113524</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1389450120666190926113524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1389450120666190926113524%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Drug%2520Targets%26atitle%3DComputational%252BStrategy%252BRevealing%252Bthe%252BStructural%252BDeterminant%252Bof%252BLigand%252BSelectivity%252Btowards%252BHighly%252BSimilar%252BProtein%252BTargets%26aulast%3DWang%26aufirst%3DHanxun%26date%3D2019%26volume%3D21%26issue%3D1%26spage%3D76%26epage%3D88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hanxun  Wang</span>, <span class="hlFld-ContribAuthor ">Zisen  Gao</span>, <span class="hlFld-ContribAuthor ">Peilu  Song</span>, <span class="hlFld-ContribAuthor ">Baichun  Hu</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular dynamics simulation and QM/MM calculation reveal the selectivity mechanism of type I 1/2 kinase inhibitors: the effect of intramolecular H-bonds and conformational restriction for improved selectivity. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2019,</strong> <em>21 </em>
                                    (43)
                                     , 24147-24164. <a href="https://doi.org/10.1039/C9CP04353E" title="DOI URL">https://doi.org/10.1039/C9CP04353E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9CP04353E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9CP04353E%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DMolecular%252Bdynamics%252Bsimulation%252Band%252BQM%25252FMM%252Bcalculation%252Breveal%252Bthe%252Bselectivity%252Bmechanism%252Bof%252Btype%252BI%252B1%25252F2%252Bkinase%252Binhibitors%25253A%252Bthe%252Beffect%252Bof%252Bintramolecular%252BH-bonds%252Band%252Bconformational%252Brestriction%252Bfor%252Bimproved%252Bselectivity%26aulast%3DWang%26aufirst%3DHanxun%26date%3D2019%26date%3D2019%26volume%3D21%26issue%3D43%26spage%3D24147%26epage%3D24164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pei-Lu  Song</span>, <span class="hlFld-ContribAuthor ">Gang  Wang</span>, <span class="hlFld-ContribAuthor ">Yuan  Su</span>, <span class="hlFld-ContribAuthor ">Han-Xun  Wang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Feng  Li</span>, <span class="hlFld-ContribAuthor ">Mao-Sheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Strategy and validation of a structure-based method for the discovery of selective inhibitors of PAK isoforms and the evaluation of their anti-cancer activity. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2019,</strong> <em>91 </em>, 103168. <a href="https://doi.org/10.1016/j.bioorg.2019.103168" title="DOI URL">https://doi.org/10.1016/j.bioorg.2019.103168</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2019.103168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2019.103168%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DStrategy%252Band%252Bvalidation%252Bof%252Ba%252Bstructure-based%252Bmethod%252Bfor%252Bthe%252Bdiscovery%252Bof%252Bselective%252Binhibitors%252Bof%252BPAK%252Bisoforms%252Band%252Bthe%252Bevaluation%252Bof%252Btheir%252Banti-cancer%252Bactivity%26aulast%3DSong%26aufirst%3DPei-Lu%26date%3D2019%26volume%3D91%26spage%3D103168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Debasis  Das</span>, <span class="hlFld-ContribAuthor ">Jian  Hong</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advancements of 4-aminoquinazoline derivatives as kinase inhibitors and their applications in medicinal chemistry. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>170 </em>, 55-72. <a href="https://doi.org/10.1016/j.ejmech.2019.03.004" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.03.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.03.004%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Badvancements%252Bof%252B4-aminoquinazoline%252Bderivatives%252Bas%252Bkinase%252Binhibitors%252Band%252Btheir%252Bapplications%252Bin%252Bmedicinal%252Bchemistry%26aulast%3DDas%26aufirst%3DDebasis%26date%3D2019%26volume%3D170%26spage%3D55%26epage%3D72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M. Ya.  Demakova</span>, <span class="hlFld-ContribAuthor ">R. M.  Islamova</span>, <span class="hlFld-ContribAuthor ">V. V.  Suslonov</span>. </span><span class="cited-content_cbyCitation_article-title">Palladium-Catalyzed Synthesis of 4-Aminoquinazolines from Amide Oxime Ethers and 2-Iodobenzonitrile. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of General Chemistry</span><span> <strong>2019,</strong> <em>89 </em>
                                    (4)
                                     , 668-672. <a href="https://doi.org/10.1134/S1070363219040054" title="DOI URL">https://doi.org/10.1134/S1070363219040054</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070363219040054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070363219040054%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520General%2520Chemistry%26atitle%3DPalladium-Catalyzed%252BSynthesis%252Bof%252B4-Aminoquinazolines%252Bfrom%252BAmide%252BOxime%252BEthers%252Band%252B2-Iodobenzonitrile%26aulast%3DDemakova%26aufirst%3DM.%2BYa.%26date%3D2019%26date%3D2019%26volume%3D89%26issue%3D4%26spage%3D668%26epage%3D672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Guo</span>, <span class="hlFld-ContribAuthor ">Fan  Zhao</span>, <span class="hlFld-ContribAuthor ">Wenbo  Yin</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zhu</span>, <span class="hlFld-ContribAuthor ">Chenzhou  Hao</span>, <span class="hlFld-ContribAuthor ">Yu  Pang</span>, <span class="hlFld-ContribAuthor ">Tianxiao  Wu</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>, <span class="hlFld-ContribAuthor ">Haitao  Li</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, structure-activity relationships study and X-ray crystallography of 3-substituted-indolin-2-one-5-carboxamide derivatives as PAK4 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>155 </em>, 197-209. <a href="https://doi.org/10.1016/j.ejmech.2018.05.051" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.05.051</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.05.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.05.051%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Bstructure-activity%252Brelationships%252Bstudy%252Band%252BX-ray%252Bcrystallography%252Bof%252B3-substituted-indolin-2-one-5-carboxamide%252Bderivatives%252Bas%252BPAK4%252Binhibitors%26aulast%3DGuo%26aufirst%3DJing%26date%3D2018%26volume%3D155%26spage%3D197%26epage%3D209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Namrata  Sampat</span>, <span class="hlFld-ContribAuthor ">Audrey  Minden</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of the p21 Activated Kinases. </span><span class="cited-content_cbyCitation_journal-name">Current Pharmacology Reports</span><span> <strong>2018,</strong> <em>4 </em>
                                    (3)
                                     , 238-249. <a href="https://doi.org/10.1007/s40495-018-0132-7" title="DOI URL">https://doi.org/10.1007/s40495-018-0132-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s40495-018-0132-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs40495-018-0132-7%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Pharmacology%2520Reports%26atitle%3DInhibitors%252Bof%252Bthe%252Bp21%252BActivated%252BKinases%26aulast%3DSampat%26aufirst%3DNamrata%26date%3D2018%26date%3D2018%26volume%3D4%26issue%3D3%26spage%3D238%26epage%3D249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0001.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Classification of representative PAK inhibitors by mechanism of function.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5, 6)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0002.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design and modification strategies of novel and selective PAK4 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of PAK4 in complex with <b>10a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVF">5XVF</a>). Note: Asp458 is oriented toward the NH<sub>2</sub> in the piperidine of <b>10a</b> compared to its counterpart Asp407 in PAK1 as in the structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O0R">4O0R</a> (cyan stick). The distance between the piperidine NH<sub>2</sub> of <b>10a</b> and the carboxyl group (OD2 atom) of Asp458<sup>PAK4</sup> and Asp407<sup>PAK1</sup> is 4.7 and 6.0 Å, respectively (green line). The ∠O<sub>(Asp458)</sub>–N<sub>(piperidine N)</sub>–N<sub>(piperidine NH2)</sub> angle is approximately 51.7° (cyan line).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0004.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of the ATP-binding pocket between group I and II PAKs. (A) Structure of the PAK kinase domain (generated from the PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X4Z">2X4Z</a>). Residues contributing to the binding cleft (P1–P40) are depicted as spheres. (B) Alignment (main chain atoms only) of the DFG–aspartate of PAK1 and PAK4 in the active state. An updated survey of all PDB PAK1 structures in the active DFG-in state indicated that 74% (26/35) of the DFG–aspartate side chain is pointing away from the ATP-competitive ligand. For the corresponding residue in PAK4, 71% (22/31) of the DFG–aspartate side chain is pointing toward the ligand. Figures were generated using PyMOL (The PyMOL Molecular Graphics System, Version 1.7.2 Schrödinger, LLC.). The definition of the dihedral angles χ1(chi): N–Cα–Cβ–Cγ. (C) Sequence alignment of the residues surrounding the ATP binding site in PAK1 and PAK4. Differential residues between the two kinases are highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0005.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Predicted binding mode of <b>17</b>. (A) Superimposition of structures of <b>17</b> (green) with <b>10a</b> from the crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVF">5XVF</a>). PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X4Z">2X4Z</a> was used for molecular docking simulation with Autodock4. Note: Asp458 in PAK4 overlaid with a key residue different in PAK1 (cyan, Asp407 from structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O0R">4O0R</a>). (B) Overlay of PAK4 and PAK1 with <b>17</b> (generated from the PDB structures <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X4Z">2X4Z</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O0R">4O0R</a>, respectively).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0006.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Crystal structure of PAK4 in complex with <b>30</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVA">5XVA</a>). Superimposition of the structures of <b>30</b> (green) with <b>10a</b> (cyan) from the crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVF">5XVF</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0007.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) Crystal structure of PAK4 in complex with <b>31</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVG">5XVG</a>). (B) Predicted binding mode of <b>31</b> in PAK1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O0R">4O0R</a>). A schematic representation of the contacts between <b>31</b> and the binding site residues of PAK4 (C) and PAK1 (D) was visualized by LigandScout.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Note: the piperazine ring of <b>31</b> provides charge-assisted hydrogen bond (CAHB) interactions with Asp458<sup>PAK4</sup> and Asp444<sup>PAK4</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0008.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Microenvironment of the 6-substituent of the inhibitors. The X-ray structure of <b>31</b> in complex with PAK4 (green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVG">5XVG</a>) overlaid with key residue differences with PAK1 (cyan, Tyr346 and Ala348 from structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O0R">4O0R</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0009.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Selectivity profile of compound <b>31</b> measured at a concentration of (A) 0.1 μM and (B) 1.0 μM in a panel of 54 kinases generated with the SelectScreen Profiling Service from Life Technologies (red columns denote >80%, yellow columns between 40 and 80%, and green columns <40% inhibition).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0010.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Compound <b>31</b> suppresses the migratory and invasive potential of A549 lung cancer cells via inhibition of PAK4 kinase activity and its downstream pathways. (A) MTT assay showed the antiproliferative effect of compound <b>31</b> on A549, HEK-293, and NCI-H460 cells. (B) Compound <b>31</b> inhibits migration of A549 lung cancer cells in a wound healing assay. (C–F) Transwell assay [−(D)/+(C) matrigel] was performed to show the effect of indicated concentrations of <b>31</b> on A549 lung cancer cell migration (C) and invasion (D), and migrated cells were counted, respectively (E,F). The results are presented as the mean ± standard deviation (*, <i>p</i> < 0.05; **, <i>p</i> < 0.01; ***, <i>p</i> < 0.001 vs control). (G,H) Effect of <b>31</b> on HEK-293 cell migration (G) and invasion (H). (I,J) Effect of <b>31</b> on NCI-H460 cell migration (I) and invasion (J). Scale bar, 500 μm. (K) Western blot assay showed the effect of compound <b>31</b> on the phospho-PAK4 Ser<sup>474</sup> level and on the phosphorylation levels of PAK4 downstream targets LIMK1 and cofilin. The expression of MMP-2 and the phosphorylation of GEF-H1 were also downregulated by compound <b>31</b> treatment in a dose-dependent manner. A549 cells were cultured with vehicle or the indicated concentrations of compound <b>31</b> for 24 h, and then proteins were extracted and subjected to analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0011.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of 4-Aminoimidazole Analogs<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) CH<sub>3</sub>I, K<sub>2</sub>CO<sub>3</sub>, MeCN, 65 °C, 12 h; (b) (i) H<sub>2</sub>, Pd/C, rt, 8 h; (ii) HCl–EtOH, 0 °C, 1 h; (c) <i>i-</i>PrBr, DMF, K<sub>2</sub>CO<sub>3</sub>, 65 °C, 12 h; (d) HNO<sub>3</sub>, Ac<sub>2</sub>O, H<sub>2</sub>SO<sub>4</sub>, 0 °C to rt, 1 h; (e) RNH<sub>2</sub>, MeOH, H<sub>2</sub>O, 24 h.</p></p></figure><figure data-id="sch2" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0012.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Syntheses of 2,4-Diaminoquinazoline Derivatives (<b>10a</b>–<b>d</b> and <b>11a</b>–<b>d</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) urea, 200 °C; (b) POCl<sub>3</sub>, <i>N</i>,<i>N</i>-dimethylaniline, reflux; (c) DIEA, DMF, 0 °C, 3 h; (d) 4-(<i>N</i>-Boc-amino)-piperidine, HCl, EtOH, 120 °C, 15 h; (e) HCl–EA, DCM, rt, 12 h.</p></p></figure><figure data-id="sch3" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0013.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Syntheses of 4-Aminoquinazoline-2-carboxamide Derivatives (<b>12</b>–<b>36</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) bis(trichloromethyl) carbonate, THF, reflux, 16 h; (ii) 1 N NH<sub>3</sub>–H<sub>2</sub>O, 65 °C, 1 h; (b) ethyl oxalyl monochloride, THF, TEA, 0 °C to rt., 3 h; (c) EtONa, EtOH, 0 °C, 3 h; (d) NaOH, H<sub>2</sub>O, EtOH, rt., 1 h; (e) SOCl<sub>2</sub>, chloroform, DMF, reflux, 3 h; (f) amine, DCM, TEA, −35 °C, 0.5 h; (g) 3-amino-5-methylpyrazole or 5-cyclopropyl-1<i>H</i>-pyrazol-3-amine, DIEA, KI, DMF, 65 °C, 8 h; (h) HCl–EA, DCM, rt, 6 h.</p></p></figure><figure data-id="sch4" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/medium/jm-2017-01342j_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0014.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Syntheses of Compounds <b>37</b>–<b>41</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-1/acs.jmedchem.7b01342/20180110/images/large/jm-2017-01342j_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01342&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) SOCl<sub>2</sub>, chloroform, DMF, reflux, 3 h; (b) <i>tert</i>-butyl (<i>R</i>)-2-methylpiperazine-1-carboxylate, DCM, TEA, −35 °C, 0.5 h; (c) 5-cyclopropyl-1<i>H</i>-pyrazol-3-amine, DIEA, KI, DMF, 65 °C, 8 h; (d) HCl–EA, DCM, rt, 6 h.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i97">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87158" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87158" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 46 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Manser, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salihuddin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, L.</span><span> </span><span class="NLM_article-title">A brain serine/threonine protein kinase activated by Cdc42 and Rac1</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">367</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">46</span><span class="refDoi"> DOI: 10.1038/367040a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1038%2F367040a0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=367&publication_year=1994&pages=40-46&author=E.+Manserauthor=T.+Leungauthor=H.+Salihuddinauthor=Z.+S.+Zhaoauthor=L.+Lim&title=A+brain+serine%2Fthreonine+protein+kinase+activated+by+Cdc42+and+Rac1&doi=10.1038%2F367040a0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F367040a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F367040a0%26sid%3Dliteratum%253Aachs%26aulast%3DManser%26aufirst%3DE.%26aulast%3DLeung%26aufirst%3DT.%26aulast%3DSalihuddin%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DZ.%2BS.%26aulast%3DLim%26aufirst%3DL.%26atitle%3DA%2520brain%2520serine%252Fthreonine%2520protein%2520kinase%2520activated%2520by%2520Cdc42%2520and%2520Rac1%26jtitle%3DNature%26date%3D1994%26volume%3D367%26spage%3D40%26epage%3D46%26doi%3D10.1038%2F367040a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Radu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semenova, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosoff, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chernoff, J.</span><span> </span><span class="NLM_article-title">PAK signalling during the development and progression of cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">13</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1038/nrc3645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1038%2Fnrc3645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=24505617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOmurnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=13-25&author=M.+Raduauthor=G.+Semenovaauthor=R.+Kosoffauthor=J.+Chernoff&title=PAK+signalling+during+the+development+and+progression+of+cancer&doi=10.1038%2Fnrc3645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">PAK signalling during the development and progression of cancer</span></div><div class="casAuthors">Radu, Maria; Semenova, Galina; Kosoff, Rachelle; Chernoff, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-25</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  P21-activated kinases (PAKs) are positioned at the nexus of several oncogenic signalling pathways.  Overexpression or mutational activation of PAK isoforms frequently occurs in various human tumors, and recent data suggest that excessive PAK activity drives many of the cellular processes that are the hallmarks of cancer.  In this Review, we discuss the mechanisms of PAK activation in cancer, the key substrates that mediate the developmental and oncogenic effects of this family of kinases, and how small-mol. inhibitors of these enzymes might be best developed and deployed for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoDp4TQ1JV7bVg90H21EOLACvtfcHk0lhUI0K-3z3lng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOmurnJ&md5=52331f8a81941b5606ba25130ffef72f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc3645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3645%26sid%3Dliteratum%253Aachs%26aulast%3DRadu%26aufirst%3DM.%26aulast%3DSemenova%26aufirst%3DG.%26aulast%3DKosoff%26aufirst%3DR.%26aulast%3DChernoff%26aufirst%3DJ.%26atitle%3DPAK%2520signalling%2520during%2520the%2520development%2520and%2520progression%2520of%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D13%26epage%3D25%26doi%3D10.1038%2Fnrc3645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Arias-Romero, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chernoff, J.</span><span> </span><span class="NLM_article-title">A tale of two Paks</span> <span class="citation_source-journal">Biol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">97</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span><span class="refDoi"> DOI: 10.1042/BC20070109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1042%2FBC20070109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=18199048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD1cXosFarsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2008&pages=97-108&author=L.+E.+Arias-Romeroauthor=J.+Chernoff&title=A+tale+of+two+Paks&doi=10.1042%2FBC20070109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A tale of two Paks</span></div><div class="casAuthors">Arias-Romero, Luis E.; Chernoff, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Biology of the Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">97-108</span>CODEN:
                <span class="NLM_cas:coden">BCELDF</span>;
        ISSN:<span class="NLM_cas:issn">0248-4900</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  Paks (p21-activated kinases) are effectors for the small GTPases Cdc42 (cell division cycle 42) and Rac that play fundamental roles in a wide range of cellular processes, including cell morphol., motility, survival, gene transcription, apoptosis and hormone signalling.  These enzymes are widely expressed in numerous tissues and are activated by extracellular signals through GTPase-dependent and -independent mechanisms.  On the basis of structural and biochem. features, the Pak family members have been classified into two groups, comprising three members each.  The two Pak groups have generally been considered as two halves of a single entity, but accumulating evidence indicates that this is not so.  In this review, we discuss recent advances in our understanding of the structure, regulation and function of these kinases, emphasizing the many differences between these two groups of signalling proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJrFXNyCcF47Vg90H21EOLACvtfcHk0lhUI0K-3z3lng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXosFarsA%253D%253D&md5=9cb3271621a5d2551dfc02bec8a2759c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1042%2FBC20070109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBC20070109%26sid%3Dliteratum%253Aachs%26aulast%3DArias-Romero%26aufirst%3DL.%2BE.%26aulast%3DChernoff%26aufirst%3DJ.%26atitle%3DA%2520tale%2520of%2520two%2520Paks%26jtitle%3DBiol.%2520Cell%26date%3D2008%26volume%3D100%26spage%3D97%26epage%3D108%26doi%3D10.1042%2FBC20070109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Ye, D. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, J.</span><span> </span><span class="NLM_article-title">PAK signaling in cancer</span> <span class="citation_source-journal">Cell. Logist.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">116</span><span class="refDoi"> DOI: 10.4161/cl.21882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.4161%2Fcl.21882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=23162742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BC2sris1Wrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=105-116&author=D.+Z.+Yeauthor=J.+Field&title=PAK+signaling+in+cancer&doi=10.4161%2Fcl.21882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">PAK signaling in cancer</span></div><div class="casAuthors">Ye Diana Zi; Field Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Cellular logistics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-116</span>
        ISSN:<span class="NLM_cas:issn">2159-2780</span>.
    </div><div class="casAbstract">Transformation of a normal cell to a cancer cell is caused by mutations in genes that regulate proliferation, apoptosis, and invasion.  Small GTPases such as Ras, Rho, Rac and Cdc42 orchestrate many of the signals that are required for malignant transformation.  The p21-activated kinases (PAKs) are effectors of Rac and Cdc42.  PAKs are a family of serine/threonine protein kinases comprised of six isoforms (PAK1-6), and they play important roles in cytoskeletal dynamics, cell survival and proliferation.  They act as key signal transducers in several cancer signaling pathways, including Ras, Raf, NFκB, Akt, Bad and p53.  Although PAKs are not mutated in cancers, they are overexpressed, hyperactivated or amplified in several human tumors and their role in cell transformation make them attractive therapeutic targets.  This review discusses the evidence that PAK is important for cell transformation and some key signaling pathways it regulates.  This review primarily discusses Group I PAKs (PAK1, PAK2 and PAK3) as Group II PAKs (PAK4, PAK5 and PAK6) are discussed elsewhere in this issue (by Minden).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSGcnTbQFS8PMAKB7Fa-xrUfW6udTcc2ebXF4ik6WwCNbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sris1Wrtg%253D%253D&md5=2cf05fab271926995393ba06da3f5dbb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.4161%2Fcl.21882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcl.21882%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DD.%2BZ.%26aulast%3DField%26aufirst%3DJ.%26atitle%3DPAK%2520signaling%2520in%2520cancer%26jtitle%3DCell.%2520Logist.%26date%3D2012%26volume%3D2%26spage%3D105%26epage%3D116%26doi%3D10.4161%2Fcl.21882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Senapedis, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crochiere, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baloglu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landesman, Y.</span><span> </span><span class="NLM_article-title">Therapeutic potential of targeting PAK signaling</span> <span class="citation_source-journal">Anti-Cancer Agents Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span><span class="refDoi"> DOI: 10.2174/1871520615666150617111414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.2174%2F1871520615666150617111414" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=75-88&author=W.+Senapedisauthor=M.+Crochiereauthor=E.+Balogluauthor=Y.+Landesman&title=Therapeutic+potential+of+targeting+PAK+signaling&doi=10.2174%2F1871520615666150617111414"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2174%2F1871520615666150617111414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1871520615666150617111414%26sid%3Dliteratum%253Aachs%26aulast%3DSenapedis%26aufirst%3DW.%26aulast%3DCrochiere%26aufirst%3DM.%26aulast%3DBaloglu%26aufirst%3DE.%26aulast%3DLandesman%26aufirst%3DY.%26atitle%3DTherapeutic%2520potential%2520of%2520targeting%2520PAK%2520signaling%26jtitle%3DAnti-Cancer%2520Agents%2520Med.%2520Chem.%26date%3D2016%26volume%3D16%26spage%3D75%26epage%3D88%26doi%3D10.2174%2F1871520615666150617111414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Rudolph, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeflich, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span> </span><span class="NLM_article-title">Inhibitors of p21-activated kinases (PAKs)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span><span class="refDoi"> DOI: 10.1021/jm501613q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501613q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=111-129&author=J.+Rudolphauthor=J.+J.+Crawfordauthor=K.+P.+Hoeflichauthor=W.+Wang&title=Inhibitors+of+p21-activated+kinases+%28PAKs%29&doi=10.1021%2Fjm501613q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm501613q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501613q%26sid%3Dliteratum%253Aachs%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DInhibitors%2520of%2520p21-activated%2520kinases%2520%2528PAKs%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D111%26epage%3D129%26doi%3D10.1021%2Fjm501613q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Ha, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turk, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boggon, T. J.</span><span> </span><span class="NLM_article-title">Signaling, regulation, and specificity of the type II p21-activated kinases</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">12975</span><span class="NLM_x">–</span> <span class="NLM_lpage">12983</span><span class="refDoi"> DOI: 10.1074/jbc.R115.650416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1074%2Fjbc.R115.650416" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=12975-12983&author=B.+H.+Haauthor=E.+M.+Morseauthor=B.+E.+Turkauthor=T.+J.+Boggon&title=Signaling%2C+regulation%2C+and+specificity+of+the+type+II+p21-activated+kinases&doi=10.1074%2Fjbc.R115.650416"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R115.650416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R115.650416%26sid%3Dliteratum%253Aachs%26aulast%3DHa%26aufirst%3DB.%2BH.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26atitle%3DSignaling%252C%2520regulation%252C%2520and%2520specificity%2520of%2520the%2520type%2520II%2520p21-activated%2520kinases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D12975%26epage%3D12983%26doi%3D10.1074%2Fjbc.R115.650416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Ha, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lou, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krauthammer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halaban, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turk, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boggon, T. J.</span><span> </span><span class="NLM_article-title">Type II p21-activated kinases (PAKs) are regulated by an autoinhibitory pseudosubstrate</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">16107</span><span class="NLM_x">–</span> <span class="NLM_lpage">16112</span><span class="refDoi"> DOI: 10.1073/pnas.1214447109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1073%2Fpnas.1214447109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=22988085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGhsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16107-16112&author=B.+H.+Haauthor=M.+J.+Davisauthor=C.+Chenauthor=H.+J.+Louauthor=J.+Gaoauthor=R.+Zhangauthor=M.+Krauthammerauthor=R.+Halabanauthor=J.+Schlessingerauthor=B.+E.+Turkauthor=T.+J.+Boggon&title=Type+II+p21-activated+kinases+%28PAKs%29+are+regulated+by+an+autoinhibitory+pseudosubstrate&doi=10.1073%2Fpnas.1214447109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Type II p21-activated kinases (PAKs) are regulated by an autoinhibitory pseudosubstrate</span></div><div class="casAuthors">Ha, Byung Hak; Davis, Matthew J.; Chen, Catherine; Lou, Hua Jane; Gao, Jia; Zhang, Rong; Krauthammer, Michael; Halaban, Ruth; Schlessinger, Joseph; Turk, Benjamin E.; Boggon, Titus J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">16107-16112, S16107/1-S16107/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The type II p21-activated kinases (PAKs) are key effectors of RHO-family GTPases involved in cell motility, survival, and proliferation.  Using a structure-guided approach, we discovered that type II PAKs are regulated by an N-terminal autoinhibitory pseudosubstrate motif centered on a crit. proline residue, and that this regulation occurs independently of activation loop phosphorylation.  We detd. six X-ray crystal structures of either full-length PAK4 or its catalytic domain, that demonstrate the mol. basis for pseudosubstrate binding to the active state with phosphorylated activation loop.  We show that full-length PAK4 is constitutively autoinhibited, but mutation of the pseudo-substrate releases this inhibition and causes increased phosphorylation of the apoptotic regulation protein Bcl-2/Bcl-XL antagonist causing cell death and cellular morphol. changes.  We also find that PAK6 is regulated by the pseudosubstrate region, indicating a common type II PAK autoregulatory mechanism.  Finally, we find Src SH3, but not β-PIX SH3, can activate PAK4.  We provide a unique understanding for type II PAK regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreLSMEq29IlrVg90H21EOLACvtfcHk0liMPq8EWGq6VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGhsL3J&md5=65e13a76626af2ecb01509f70d36711e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1214447109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1214447109%26sid%3Dliteratum%253Aachs%26aulast%3DHa%26aufirst%3DB.%2BH.%26aulast%3DDavis%26aufirst%3DM.%2BJ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DLou%26aufirst%3DH.%2BJ.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DKrauthammer%26aufirst%3DM.%26aulast%3DHalaban%26aufirst%3DR.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DTurk%26aufirst%3DB.%2BE.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26atitle%3DType%2520II%2520p21-activated%2520kinases%2520%2528PAKs%2529%2520are%2520regulated%2520by%2520an%2520autoinhibitory%2520pseudosubstrate%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D16107%26epage%3D16112%26doi%3D10.1073%2Fpnas.1214447109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Kelly, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chernoff, J.</span><span> </span><span class="NLM_article-title">Mouse models of PAK function</span> <span class="citation_source-journal">Cell. Logist.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span><span class="refDoi"> DOI: 10.4161/cl.21381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.4161%2Fcl.21381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=23162740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BC2sbpt1eltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=84-88&author=M.+L.+Kellyauthor=J.+Chernoff&title=Mouse+models+of+PAK+function&doi=10.4161%2Fcl.21381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Mouse models of PAK function</span></div><div class="casAuthors">Kelly Mollie L; Chernoff Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Cellular logistics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">84-88</span>
        ISSN:<span class="NLM_cas:issn">2159-2780</span>.
    </div><div class="casAbstract">p21-activated kinases are a family of highly conserved protein serine/threonine kinases that are increasingly recognized as playing essential roles in a variety of key signaling processes.  Genetic analyses in mice, using constitutive or regulated gene disruption, have provided important new insights into PAK function.  In this paper, we review the genetic analysis of all six PAK genes in mice.  These data address the singular and redundant functions of the various PAK genes and suggest therapeutic possibilities for small molecule PAK inhibitors or activators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQdjME4vf8KMv9pSBfyDvp7fW6udTcc2eY6xGnaCGrBErntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbpt1eltA%253D%253D&md5=d6ee25015bac19af685c134474ca22d5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.4161%2Fcl.21381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcl.21381%26sid%3Dliteratum%253Aachs%26aulast%3DKelly%26aufirst%3DM.%2BL.%26aulast%3DChernoff%26aufirst%3DJ.%26atitle%3DMouse%2520models%2520of%2520PAK%2520function%26jtitle%3DCell.%2520Logist.%26date%3D2012%26volume%3D2%26spage%3D84%26epage%3D88%26doi%3D10.4161%2Fcl.21381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Tian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cammarano, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nekrasova, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minden, A.</span><span> </span><span class="NLM_article-title">Essential role for the Pak4 protein kinase in extraembryonic tissue development and vessel formation</span> <span class="citation_source-journal">Mech. Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">710</span><span class="NLM_x">–</span> <span class="NLM_lpage">720</span><span class="refDoi"> DOI: 10.1016/j.mod.2009.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2Fj.mod.2009.05.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2009&pages=710-720&author=Y.+Tianauthor=L.+Leiauthor=M.+Cammaranoauthor=T.+Nekrasovaauthor=A.+Minden&title=Essential+role+for+the+Pak4+protein+kinase+in+extraembryonic+tissue+development+and+vessel+formation&doi=10.1016%2Fj.mod.2009.05.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.mod.2009.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mod.2009.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DY.%26aulast%3DLei%26aufirst%3DL.%26aulast%3DCammarano%26aufirst%3DM.%26aulast%3DNekrasova%26aufirst%3DT.%26aulast%3DMinden%26aufirst%3DA.%26atitle%3DEssential%2520role%2520for%2520the%2520Pak4%2520protein%2520kinase%2520in%2520extraembryonic%2520tissue%2520development%2520and%2520vessel%2520formation%26jtitle%3DMech.%2520Dev.%26date%3D2009%26volume%3D126%26spage%3D710%26epage%3D720%26doi%3D10.1016%2Fj.mod.2009.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nekrasova, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suh, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minden, A.</span><span> </span><span class="NLM_article-title">The pak4 protein kinase plays a key role in cell survival and tumorigenesis in athymic mice</span> <span class="citation_source-journal">Mol. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1215</span><span class="NLM_x">–</span> <span class="NLM_lpage">1224</span><span class="refDoi"> DOI: 10.1158/1541-7786.MCR-08-0087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1158%2F1541-7786.MCR-08-0087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=18644984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslCgt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=1215-1224&author=Y.+Liuauthor=H.+Xiaoauthor=Y.+Tianauthor=T.+Nekrasovaauthor=X.+Haoauthor=H.+J.+Leeauthor=N.+Suhauthor=C.+S.+Yangauthor=A.+Minden&title=The+pak4+protein+kinase+plays+a+key+role+in+cell+survival+and+tumorigenesis+in+athymic+mice&doi=10.1158%2F1541-7786.MCR-08-0087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The Pak4 Protein Kinase Plays a Key Role in Cell Survival and Tumorigenesis in Athymic Mice</span></div><div class="casAuthors">Liu, Yingying; Xiao, Hang; Tian, Yanmei; Nekrasova, Tanya; Hao, Xingpei; Lee, Hong Jin; Suh, Nanjoo; Yang, Chung S.; Minden, Audrey</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1215-1224</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Pak4 is a member of the B group of p21-activated (Pak) kinases, originally identified as an effector protein for Cdc42.  Although Pak4 is expressed at low levels in most adult tissues, it is highly overexpressed in tumor cell lines.  Here, we show that Pak4 is also overexpressed in primary tumors, including colon, esophageal, and mammary tumors.  Overexpression of Pak4 also leads to tumor formation in athymic mice, whereas deletion of Pak4 inhibits tumorigenesis.  Although a constitutively active Pak4 mutant was previously shown to promote oncogenic transformation in cultured cells, these results are the first to show that Pak4 also promotes tumorigenesis in exptl. animals.  Furthermore, these results show for the first time that not only constitutively active Pak4, but also wild-type Pak4, is transforming, when exptl. animals are used.  These results are highly significant because wild-type Pak4, rather than activated Pak4, is overexpressed in tumor cells.  The results suggest that overexpression or activation of Pak4 is a key step in oncogenic transformation, due to its ability to promote cell survival and subsequent uncontrolled proliferation.  The finding that Pak4 is up-regulated in so many types of cancers indicates that Pak4 may play a vital role in a wide range of different types of cancer.  This makes it an attractive candidate for drug therapy for different types of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeiLYqGGfYu7Vg90H21EOLACvtfcHk0lh2pIR51yYEUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslCgt7g%253D&md5=40a0935d384470bf2cbf7a172f5730bb</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-08-0087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-08-0087%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DNekrasova%26aufirst%3DT.%26aulast%3DHao%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DSuh%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DC.%2BS.%26aulast%3DMinden%26aufirst%3DA.%26atitle%3DThe%2520pak4%2520protein%2520kinase%2520plays%2520a%2520key%2520role%2520in%2520cell%2520survival%2520and%2520tumorigenesis%2520in%2520athymic%2520mice%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2008%26volume%3D6%26spage%3D1215%26epage%3D1224%26doi%3D10.1158%2F1541-7786.MCR-08-0087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Callow, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clairvoyant, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schryver, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bischoff, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jallal, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smeal, T.</span><span> </span><span class="NLM_article-title">Requirement for PAK4 in the anchorage-independent growth of human cancer cell lines</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">550</span><span class="NLM_x">–</span> <span class="NLM_lpage">558</span><span class="refDoi"> DOI: 10.1074/jbc.M105732200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1074%2Fjbc.M105732200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=550-558&author=M.+G.+Callowauthor=F.+Clairvoyantauthor=S.+Zhuauthor=B.+Schryverauthor=D.+B.+Whyteauthor=J.+R.+Bischoffauthor=B.+Jallalauthor=T.+Smeal&title=Requirement+for+PAK4+in+the+anchorage-independent+growth+of+human+cancer+cell+lines&doi=10.1074%2Fjbc.M105732200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M105732200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M105732200%26sid%3Dliteratum%253Aachs%26aulast%3DCallow%26aufirst%3DM.%2BG.%26aulast%3DClairvoyant%26aufirst%3DF.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DSchryver%26aufirst%3DB.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DBischoff%26aufirst%3DJ.%2BR.%26aulast%3DJallal%26aufirst%3DB.%26aulast%3DSmeal%26aufirst%3DT.%26atitle%3DRequirement%2520for%2520PAK4%2520in%2520the%2520anchorage-independent%2520growth%2520of%2520human%2520cancer%2520cell%2520lines%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D550%26epage%3D558%26doi%3D10.1074%2Fjbc.M105732200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Minden, A.</span><span> </span><span class="NLM_article-title">The pak4 protein kinase in breast cancer</span> <span class="citation_source-journal">ISRN Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2012</span><span class="NLM_x">, </span> <span class="NLM_fpage">694201</span><span class="refDoi"> DOI: 10.5402/2012/694201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.5402%2F2012%2F694201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=23326684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BC3szgt1agtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2012&publication_year=2012&pages=694201&author=A.+Minden&title=The+pak4+protein+kinase+in+breast+cancer&doi=10.5402%2F2012%2F694201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The pak4 protein kinase in breast cancer</span></div><div class="casAuthors">Minden Audrey</div><div class="citationInfo"><span class="NLM_cas:title">ISRN oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2012</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">694201</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Paks4, along with Paks5, and 6 are members of the group B family of p21-activated kinases (Paks).  The Paks play multiple different roles in controlling cell morphology, cell growth, proliferation, and signaling.  Pak4 has essential roles in embryonic development (Qu et al., 2003), but in adults high levels of Pak4 are frequently associated with cancer.  Pak4 has been implicated in several types of cancer (Wells and Jones, 2010; Eswaran et al., 2009; Liu et al., 2008; and Liu et al., 2010) and it is strongly linked to breast cancer (Liu et al., 2008; Liu et al. 2010; Yu et al., 2009; Rafn et al., 2012; and So et al., 2012).  Breast tumors and breast cancer cell lines frequently have high levels of Pak4 (Liu et al., 2008), and overexpression of Pak4 in mammary epithelial cells leads to tumorigenesis in mice (Liu et al., 2010).  This paper summarizes the current work on the role of Pak4 in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR4b_ZBK4mwArFU0A1HOshhfW6udTcc2eZGdADRqeG9wbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3szgt1agtA%253D%253D&md5=4d260e310ad65144673a648ea78d22e0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.5402%2F2012%2F694201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5402%252F2012%252F694201%26sid%3Dliteratum%253Aachs%26aulast%3DMinden%26aufirst%3DA.%26atitle%3DThe%2520pak4%2520protein%2520kinase%2520in%2520breast%2520cancer%26jtitle%3DISRN%2520Oncol.%26date%3D2012%26volume%3D2012%26spage%3D694201%26doi%3D10.5402%2F2012%2F694201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Wells, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, G. E.</span><span> </span><span class="NLM_article-title">The emerging importance of group II PAKs</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">425</span><span class="NLM_x">, </span> <span class="NLM_fpage">465</span><span class="NLM_x">–</span> <span class="NLM_lpage">473</span><span class="refDoi"> DOI: 10.1042/BJ20091173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1042%2FBJ20091173" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2010&pages=465-473&author=C.+M.+Wellsauthor=G.+E.+Jones&title=The+emerging+importance+of+group+II+PAKs&doi=10.1042%2FBJ20091173"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1042%2FBJ20091173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20091173%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DC.%2BM.%26aulast%3DJones%26aufirst%3DG.%2BE.%26atitle%3DThe%2520emerging%2520importance%2520of%2520group%2520II%2520PAKs%26jtitle%3DBiochem.%2520J.%26date%3D2010%26volume%3D425%26spage%3D465%26epage%3D473%26doi%3D10.1042%2FBJ20091173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Rudolph, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ndubaku, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackwood, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aliagas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gazzard, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drobnick, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeflich, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Favor, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heise, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ong, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakravarty, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakubiak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Epler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vega, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cain, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, Y.</span><span> </span><span class="NLM_article-title">Chemically diverse group I p21-activated kinase (PAK) inhibitors impart acute cardiovascular toxicity with a narrow therapeutic window</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">5520</span><span class="NLM_x">–</span> <span class="NLM_lpage">5541</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00638</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00638" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslWhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5520-5541&author=J.+Rudolphauthor=L.+J.+Murrayauthor=C.+O.+Ndubakuauthor=T.+O%E2%80%99Brienauthor=E.+Blackwoodauthor=W.+Wangauthor=I.+Aliagasauthor=L.+Gazzardauthor=J.+J.+Crawfordauthor=J.+Drobnickauthor=W.+Leeauthor=X.+Zhaoauthor=K.+P.+Hoeflichauthor=D.+A.+Favorauthor=P.+Dongauthor=H.+Zhangauthor=C.+E.+Heiseauthor=A.+Ohauthor=C.+C.+Ongauthor=H.+Laauthor=P.+Chakravartyauthor=C.+Chanauthor=D.+Jakubiakauthor=J.+Eplerauthor=S.+Ramaswamyauthor=R.+Vegaauthor=G.+Cainauthor=D.+Diazauthor=Y.+Zhong&title=Chemically+diverse+group+I+p21-activated+kinase+%28PAK%29+inhibitors+impart+acute+cardiovascular+toxicity+with+a+narrow+therapeutic+window&doi=10.1021%2Facs.jmedchem.6b00638"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Diverse Group I p21-Activated Kinase (PAK) Inhibitors Impart Acute Cardiovascular Toxicity with a Narrow Therapeutic Window</span></div><div class="casAuthors">Rudolph, Joachim; Murray, Lesley J.; Ndubaku, Chudi O.; O'Brien, Thomas; Blackwood, Elizabeth; Wang, Weiru; Aliagas, Ignacio; Gazzard, Lewis; Crawford, James J.; Drobnick, Joy; Lee, Wendy; Zhao, Xianrui; Hoeflich, Klaus P.; Favor, David A.; Dong, Ping; Zhang, Haiming; Heise, Christopher E.; Oh, Angela; Ong, Christy C.; La, Hank; Chakravarty, Paroma; Chan, Connie; Jakubiak, Diana; Epler, Jennifer; Ramaswamy, Sreemathy; Vega, Roxanne; Cain, Gary; Diaz, Dolores; Zhong, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5520-5541</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P21-activated kinase 1 (PAK1) has an important role in transducing signals in several oncogenic pathways.  The concept of inhibiting this kinase has garnered significant interest over the past decade, particularly for targeting cancers assocd. with PAK1 amplification.  Animal studies with the selective group I PAK (pan-PAK1, 2, 3) inhibitor G-5555 from the pyrido[2,3-d]pyrimidin-7-one class uncovered acute toxicity with a narrow therapeutic window.  To attempt mitigating the toxicity, we introduced significant structural changes, culminating in the discovery of the potent pyridone side chain analog G-9791.  Mouse tolerability studies with this compd., other members of this series, and compds. from two structurally distinct classes revealed persistent toxicity and a correlation of min. toxic concns. and PAK1/2 mediated cellular potencies.  Broad screening of selected PAK inhibitors revealed PAK1, 2, and 3 as the only overlapping targets.  Our data suggest acute cardiovascular toxicity resulting from the inhibition of PAK2, which may be enhanced by PAK1 inhibition, and cautions against continued pursuit of pan-group I PAK inhibitors in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop35dBBsANDLVg90H21EOLACvtfcHk0lifzG2da1UBFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslWhtLg%253D&md5=9fbab0f13e23ee9f215b32c48c617f3e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00638%26sid%3Dliteratum%253Aachs%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DMurray%26aufirst%3DL.%2BJ.%26aulast%3DNdubaku%26aufirst%3DC.%2BO.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DBlackwood%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DGazzard%26aufirst%3DL.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DDrobnick%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DFavor%26aufirst%3DD.%2BA.%26aulast%3DDong%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHeise%26aufirst%3DC.%2BE.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DOng%26aufirst%3DC.%2BC.%26aulast%3DLa%26aufirst%3DH.%26aulast%3DChakravarty%26aufirst%3DP.%26aulast%3DChan%26aufirst%3DC.%26aulast%3DJakubiak%26aufirst%3DD.%26aulast%3DEpler%26aufirst%3DJ.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DVega%26aufirst%3DR.%26aulast%3DCain%26aufirst%3DG.%26aulast%3DDiaz%26aufirst%3DD.%26aulast%3DZhong%26aufirst%3DY.%26atitle%3DChemically%2520diverse%2520group%2520I%2520p21-activated%2520kinase%2520%2528PAK%2529%2520inhibitors%2520impart%2520acute%2520cardiovascular%2520toxicity%2520with%2520a%2520narrow%2520therapeutic%2520window%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5520%26epage%3D5541%26doi%3D10.1021%2Facs.jmedchem.6b00638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The ins and outs of selective kinase inhibitor development</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">818</span><span class="NLM_x">–</span> <span class="NLM_lpage">821</span><span class="refDoi"> DOI: 10.1038/nchembio.1938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1038%2Fnchembio.1938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=26485069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=818-821&author=S.+Mullerauthor=A.+Chaikuadauthor=N.+S.+Grayauthor=S.+Knapp&title=The+ins+and+outs+of+selective+kinase+inhibitor+development&doi=10.1038%2Fnchembio.1938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The ins and outs of selective kinase inhibitor development</span></div><div class="casAuthors">Muller, Susanne; Chaikuad, Apirat; Gray, Nathanael S.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">818-821</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein kinases have emerged as one of the most successful families of drug targets.  To date, most selective kinase inhibitors have been discovered serendipitously either through broad selectivity screening or through the discovery of unique binding modes.  Here the authors discuss design strategies that could lead to a broader coverage of the kinome with selective inhibitors and to a more rational approach for developing them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQzaMrxSYx_rVg90H21EOLACvtfcHk0lifzG2da1UBFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF&md5=f5adf0bbaa388c87b26bae23cfad0bae</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1938%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520ins%2520and%2520outs%2520of%2520selective%2520kinase%2520inhibitor%2520development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D818%26epage%3D821%26doi%3D10.1038%2Fnchembio.1938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Murray, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piraino, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westwick, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamerdin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dagostino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knighton, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loi, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zager, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraynov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popoff, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kephart, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marakovits, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergqvist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smeal, T.</span><span> </span><span class="NLM_article-title">Small-molecule p21-activated kinase inhibitor PF-3758309 is a potent inhibitor of oncogenic signaling and tumor growth</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">9446</span><span class="NLM_x">–</span> <span class="NLM_lpage">9451</span><span class="refDoi"> DOI: 10.1073/pnas.0911863107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1073%2Fpnas.0911863107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=20439741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslWqtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=9446-9451&author=B.+W.+Murrayauthor=C.+Guoauthor=J.+Pirainoauthor=J.+K.+Westwickauthor=C.+Zhangauthor=J.+Lamerdinauthor=E.+Dagostinoauthor=D.+Knightonauthor=C.+M.+Loiauthor=M.+Zagerauthor=E.+Kraynovauthor=I.+Popoffauthor=J.+G.+Christensenauthor=R.+Martinezauthor=S.+E.+Kephartauthor=J.+Marakovitsauthor=S.+Karlicekauthor=S.+Bergqvistauthor=T.+Smeal&title=Small-molecule+p21-activated+kinase+inhibitor+PF-3758309+is+a+potent+inhibitor+of+oncogenic+signaling+and+tumor+growth&doi=10.1073%2Fpnas.0911863107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule p21-activated kinase inhibitor PF3758309 is a potent inhibitor of oncogenic signaling and tumor growth</span></div><div class="casAuthors">Murray, Brion W.; Guo, Chuangxing; Piraino, Joseph; Westwick, John K.; Zhang, Cathy; Lamerdin, Jane; Dagostino, Eleanor; Knighton, Daniel; Loi, Cho-Ming; Zager, Michael; Kraynov, Eugenia; Popoff, Ian; Chrietensen, James G.; Martinez, Ricardo; Kephart, Susan E.; Marakovits, Joseph; Karlicek, Shannon; Bergqvist, Simon; Smeal, Tod</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9446-9451, S9446/1-S9446/18</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Despite abundant evidence that aberrant Rho-family GTPase activation contributes to most steps of cancer initiation and progression, there is a dearth of inhibitors of their effectors (e.g.. p21-activated kinases).  Through high-throughput screening and structure-based design, the authors identify PF-3758309, a potent (Kd = 2.7 nM), ATP-competitive, pyrrolopyrazole inhibitor of PAK4.  In cells, PF-3758309 inhibits phosphorylation of the PAK4 substrate GEF-H1 (IC50 = 1.3 nM) and anchorage-independent growth of a panel of tumor cell lines (IC50 = 4.7±3 nM).  The mol. underpinnings of PF-3758309 biol. effects were characterized using an integration of traditional and emerging technologies.  Crystallog. characterization of the PF-3758309/PAK4 complex defined determinants of potency and kinase selectivity.  Global high-content cellular anal. confirms that PF-3758309 modulates known PAK4-dependent signaling nodes and identifies unexpected links to addnl. pathways (e.g., p53).  In tumor models, PF-3758309 inhibits PAK4-dependent pathways in proteomic studies and regulates functional activities related to cell proliferation and survival.  PF-3758309 blocks the growth of multiple human tumor xenografts, with a plasma EC50 value of 0.4 nM in the most sensitive model.  This study defines PAK4-related pathways, provides addnl. support for PAK4 as a therapeutic target with a unique combination of functions (apoptotic, cytoskeletal, cell-cycle), and identifies a potent, orally available small-mol. PAK inhibitor with significant promise for the treatment of human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWI0STMKt9dbVg90H21EOLACvtfcHk0lh9YoUk_5skLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslWqtrw%253D&md5=0080659735ef51f84c00112e573b1c31</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0911863107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0911863107%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DGuo%26aufirst%3DC.%26aulast%3DPiraino%26aufirst%3DJ.%26aulast%3DWestwick%26aufirst%3DJ.%2BK.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DLamerdin%26aufirst%3DJ.%26aulast%3DDagostino%26aufirst%3DE.%26aulast%3DKnighton%26aufirst%3DD.%26aulast%3DLoi%26aufirst%3DC.%2BM.%26aulast%3DZager%26aufirst%3DM.%26aulast%3DKraynov%26aufirst%3DE.%26aulast%3DPopoff%26aufirst%3DI.%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DKephart%26aufirst%3DS.%2BE.%26aulast%3DMarakovits%26aufirst%3DJ.%26aulast%3DKarlicek%26aufirst%3DS.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DSmeal%26aufirst%3DT.%26atitle%3DSmall-molecule%2520p21-activated%2520kinase%2520inhibitor%2520PF-3758309%2520is%2520a%2520potent%2520inhibitor%2520of%2520oncogenic%2520signaling%2520and%2520tumor%2520growth%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D9446%26epage%3D9451%26doi%3D10.1073%2Fpnas.0911863107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Bradshaw-Pierce, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitts, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhillips, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafson, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halsey, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, N. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kan, J. L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckhardt, S. G.</span><span> </span><span class="NLM_article-title">Tumor P-glycoprotein correlates with efficacy of PF-3758309 in in vitro and in vivo models of colorectal cancer</span> <span class="citation_source-journal">Front. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="refDoi"> DOI: 10.3389/fphar.2013.00022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.3389%2Ffphar.2013.00022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=23524533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BC3svpt1Ohsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=22&author=E.+L.+Bradshaw-Pierceauthor=T.+M.+Pittsauthor=A.+C.+Tanauthor=K.+McPhillipsauthor=M.+Westauthor=D.+L.+Gustafsonauthor=C.+Halseyauthor=L.+Nguyenauthor=N.+V.+Leeauthor=J.+L.+C.+Kanauthor=B.+W.+Murrayauthor=S.+G.+Eckhardt&title=Tumor+P-glycoprotein+correlates+with+efficacy+of+PF-3758309+in+in+vitro+and+in+vivo+models+of+colorectal+cancer&doi=10.3389%2Ffphar.2013.00022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor P-Glycoprotein Correlates with Efficacy of PF-3758309 in in vitro and in vivo Models of Colorectal Cancer</span></div><div class="casAuthors">Bradshaw-Pierce Erica Lynn; Pitts Todd M; Tan Aik-Choon; McPhillips Kelly; West Mark; Gustafson Daniel L; Halsey Charles; Nguyen Leslie; Lee Nathan V; Kan Julie L C; Murray Brion William; Eckhardt S Gail</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">22</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">P-glycoprotein (P-gp), a member of the ATP-binding cassette transporter family, is overexpressed in a number of different cancers and some studies show that P-gp overexpression can be correlated to poor prognosis or therapeutic resistance.  Here we sought to elucidate if PF-3758309 (PF-309), a novel p-21 activated kinase inhibitor, efficacy was influenced by tumor P-gp.  Based on in vitro proliferation data, a panel of colorectal cancer cell lines were ranked as sensitive or resistant and ABCB1 (P-gp) expression was evaluated by microarray for these cell lines.  P-gp expression was determined by western blot and activity determined by rhodamine efflux assay.  Knock down of P-gp and pharmacologic inhibition of P-gp to restore PF-309 activity was performed in vitro.  PF-309 activity was evaluated in vivo in cell line xenograft models and in primary patient derived tumor xenografts (PDTX).  Mice were treated with 25 mg/kg PF-309 orally, twice daily.  On the last day of treatment, tumor and plasma were collected for PF-309 analysis.  Here we show that ABCB1 gene expression correlates with resistance to PF-309 treatment in vitro and the expression and activity of P-gp was verified in a panel of resistant cells.  Furthermore, inhibition of P-gp increased the sensitivity of resistant cells, resulting in a 4-100-fold decrease in the IC50s.  Eleven cell line xenografts and 12 PDTX models were treated with PF-309.  From the cell line xenografts, we found a significant correlation between ABCB1 gene expression profiles and tumor response.  We evaluated tumor and plasma concentrations for eight tumor models (three cell line xenografts and five PDTX models) and a significant correlation was found between tumor concentration and response.  Additionally, we show that tumor concentration is approximately fourfold lower in tumors that express P-gp, verified by western blot.  Our in vitro and in vivo data strongly suggests that PF-309 efficacy is influenced by the expression of tumor P-gp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR7z7ao3-xc0c-crCu0nI_sfW6udTcc2eaw-uHOTIxwObntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3svpt1Ohsw%253D%253D&md5=cfcbecd18424bb7af204c2755cc87b32</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2013.00022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2013.00022%26sid%3Dliteratum%253Aachs%26aulast%3DBradshaw-Pierce%26aufirst%3DE.%2BL.%26aulast%3DPitts%26aufirst%3DT.%2BM.%26aulast%3DTan%26aufirst%3DA.%2BC.%26aulast%3DMcPhillips%26aufirst%3DK.%26aulast%3DWest%26aufirst%3DM.%26aulast%3DGustafson%26aufirst%3DD.%2BL.%26aulast%3DHalsey%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DN.%2BV.%26aulast%3DKan%26aufirst%3DJ.%2BL.%2BC.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DEckhardt%26aufirst%3DS.%2BG.%26atitle%3DTumor%2520P-glycoprotein%2520correlates%2520with%2520efficacy%2520of%2520PF-3758309%2520in%2520in%2520vitro%2520and%2520in%2520vivo%2520models%2520of%2520colorectal%2520cancer%26jtitle%3DFront.%2520Pharmacol.%26date%3D2013%26volume%3D4%26spage%3D22%26doi%3D10.3389%2Ffphar.2013.00022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Crawford, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aliagas, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeflich, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouge, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heise, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mintzer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, J.</span><span> </span><span class="NLM_article-title">Structure-guided design of group I selective p21-activated kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5121</span><span class="NLM_x">–</span> <span class="NLM_lpage">5136</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5121-5136&author=J.+J.+Crawfordauthor=W.+Leeauthor=I.+Aliagasauthor=S.+Mathieuauthor=K.+P.+Hoeflichauthor=W.+Zhouauthor=W.+Wangauthor=L.+Rougeauthor=L.+Murrayauthor=H.+Laauthor=N.+Liuauthor=P.+W.+Fanauthor=J.+Cheongauthor=C.+E.+Heiseauthor=S.+Ramaswamyauthor=R.+Mintzerauthor=Y.+Liuauthor=Q.+Chaoauthor=J.+Rudolph&title=Structure-guided+design+of+group+I+selective+p21-activated+kinase+inhibitors&doi=10.1021%2Facs.jmedchem.5b00572"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00572%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DMathieu%26aufirst%3DS.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DRouge%26aufirst%3DL.%26aulast%3DMurray%26aufirst%3DL.%26aulast%3DLa%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DFan%26aufirst%3DP.%2BW.%26aulast%3DCheong%26aufirst%3DJ.%26aulast%3DHeise%26aufirst%3DC.%2BE.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DMintzer%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DChao%26aufirst%3DQ.%26aulast%3DRudolph%26aufirst%3DJ.%26atitle%3DStructure-guided%2520design%2520of%2520group%2520I%2520selective%2520p21-activated%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5121%26epage%3D5136%26doi%3D10.1021%2Facs.jmedchem.5b00572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Staben, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyle, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boggs, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burch, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chua, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiPasquale, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedman, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heise, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeppen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mintzer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouge, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeflich, K. P.</span><span> </span><span class="NLM_article-title">Back pocket flexibility provides group II p21-activated kinase (PAK) selectivity for type I 1/2 kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1033</span><span class="NLM_x">–</span> <span class="NLM_lpage">1045</span><span class="refDoi"> DOI: 10.1021/jm401768t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401768t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVGmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1033-1045&author=S.+T.+Stabenauthor=J.+A.+Fengauthor=K.+Lyleauthor=M.+Belvinauthor=J.+Boggsauthor=J.+D.+Burchauthor=C.+C.+Chuaauthor=H.+Cuiauthor=A.+G.+DiPasqualeauthor=L.+S.+Friedmanauthor=C.+Heiseauthor=H.+Koeppenauthor=A.+Koteyauthor=R.+Mintzerauthor=A.+Ohauthor=D.+A.+Robertsauthor=L.+Rougeauthor=J.+Rudolphauthor=C.+Tamauthor=W.+Wangauthor=Y.+Xiaoauthor=A.+Youngauthor=Y.+Zhangauthor=K.+P.+Hoeflich&title=Back+pocket+flexibility+provides+group+II+p21-activated+kinase+%28PAK%29+selectivity+for+type+I+1%2F2+kinase+inhibitors&doi=10.1021%2Fjm401768t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Back Pocket Flexibility Provides Group II p21-Activated Kinase (PAK) Selectivity for Type I 1/2 Kinase Inhibitors</span></div><div class="casAuthors">Staben, Steven T.; Feng, Jianwen A.; Lyle, Karen; Belvin, Marcia; Boggs, Jason; Burch, Jason D.; Chua, Ching-ching; Cui, Haifeng; DiPasquale, Antonio G.; Friedman, Lori S.; Heise, Christopher; Koeppen, Hartmut; Kotey, Adrian; Mintzer, Robert; Oh, Angela; Roberts, David Allen; Rouge, Lionel; Rudolph, Joachim; Tam, Christine; Wang, Weiru; Xiao, Yisong; Young, Amy; Zhang, Yamin; Hoeflich, Klaus P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1033-1045</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based methods were used to design a potent and highly selective group II p21-activated kinase (PAK) inhibitor with a novel binding mode, compd. I.  Hydrophobic interactions within a lipophilic pocket past the methionine gatekeeper of group II PAKs approached by these type I 1/2 binders were found to be important for improving potency.  A structure-based hypothesis and strategy for achieving selectivity over group I PAKs, and the broad kinome, based on unique flexibility of this lipophilic pocket, is presented.  A concn.-dependent decrease in tumor cell migration and invasion in two triple-neg. breast cancer cell lines was obsd. with compd. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrogBhcjJGWm7Vg90H21EOLACvtfcHk0lhTNVJozzCBwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVGmsQ%253D%253D&md5=08b11088bc4d806cc61e16c932b68cf6</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm401768t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401768t%26sid%3Dliteratum%253Aachs%26aulast%3DStaben%26aufirst%3DS.%2BT.%26aulast%3DFeng%26aufirst%3DJ.%2BA.%26aulast%3DLyle%26aufirst%3DK.%26aulast%3DBelvin%26aufirst%3DM.%26aulast%3DBoggs%26aufirst%3DJ.%26aulast%3DBurch%26aufirst%3DJ.%2BD.%26aulast%3DChua%26aufirst%3DC.%2BC.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DDiPasquale%26aufirst%3DA.%2BG.%26aulast%3DFriedman%26aufirst%3DL.%2BS.%26aulast%3DHeise%26aufirst%3DC.%26aulast%3DKoeppen%26aufirst%3DH.%26aulast%3DKotey%26aufirst%3DA.%26aulast%3DMintzer%26aufirst%3DR.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DD.%2BA.%26aulast%3DRouge%26aufirst%3DL.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DTam%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DYoung%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26atitle%3DBack%2520pocket%2520flexibility%2520provides%2520group%2520II%2520p21-activated%2520kinase%2520%2528PAK%2529%2520selectivity%2520for%2520type%2520I%25201%252F2%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1033%26epage%3D1045%26doi%3D10.1021%2Fjm401768t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Abu Aboud, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senapedis, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baloglu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Argueta, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, R. H.</span><span> </span><span class="NLM_article-title">Dual and specific inhibition of NAMPT and PAK4 by KPT-9274 decreases kidney cancer growth</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">2119</span><span class="NLM_x">–</span> <span class="NLM_lpage">2129</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1158%2F1535-7163.MCT-16-0197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=27390344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVCnsLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2119-2129&author=O.+Abu+Aboudauthor=C.+H.+Chenauthor=W.+Senapedisauthor=E.+Balogluauthor=C.+Arguetaauthor=R.+H.+Weiss&title=Dual+and+specific+inhibition+of+NAMPT+and+PAK4+by+KPT-9274+decreases+kidney+cancer+growth&doi=10.1158%2F1535-7163.MCT-16-0197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Dual and Specific Inhibition of NAMPT and PAK4 By KPT-9274 Decreases Kidney Cancer Growth</span></div><div class="casAuthors">Abu Aboud, Omran; Chen, Ching-Hsien; Senapedis, William; Baloglu, Erkan; Argueta, Christian; Weiss, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2119-2129</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Kidney cancer (or renal cell carcinoma, RCC) is the sixth most common malignancy in the United States and one of the relatively few whose incidence is increasing.  Because of the near universal resistance which occurs with the use of current treatment regimens, reprogrammed metabolic pathways are being investigated as potential targets for novel therapies of this disease.  Borrowing from studies on other malignancies, we have identified the PAK4 and NAD biosynthetic pathways as being essential for RCC growth.  We now show, using the dual PAK4/NAMPT inhibitor KPT-9274, that interference with these signaling pathways results in redn. of G2-M transit as well as induction of apoptosis and decrease in cell invasion and migration in several human RCC cell lines.  Mechanistic studies demonstrate that inhibition of the PAK4 pathway by KPT-9274 attenuates nuclear β-catenin as well as the Wnt/β-catenin targets cyclin D1 and c-Myc.  Furthermore, NAPRT1 downregulation, which we show occurs in all RCC cell lines tested, makes this tumor highly dependent on NAMPT for its NAD requirements, such that inhibition of NAMPT by KPT-9274 leads to decreased survival of these rapidly proliferating cells.  When KPT-9274 was administered in vivo to a 786-O (VHL-mut) human RCC xenograft model, there was dose-dependent inhibition of tumor growth with no apparent toxicity; KPT-9274 demonstrated the expected on-target effects in this mouse model.  KPT-9274 is being evaluated in a phase I human clin. trial in solid tumors and lymphomas, which will allow this data to be rapidly translated into the clinic for the treatment of RCC.  Mol Cancer Ther; 15(9); 2119-29. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVAlkkAFgNi7Vg90H21EOLACvtfcHk0lhTNVJozzCBwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVCnsLvK&md5=043002b41043e7fda7c07f10f89fbfa0</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0197%26sid%3Dliteratum%253Aachs%26aulast%3DAbu%2BAboud%26aufirst%3DO.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DSenapedis%26aufirst%3DW.%26aulast%3DBaloglu%26aufirst%3DE.%26aulast%3DArgueta%26aufirst%3DC.%26aulast%3DWeiss%26aufirst%3DR.%2BH.%26atitle%3DDual%2520and%2520specific%2520inhibition%2520of%2520NAMPT%2520and%2520PAK4%2520by%2520KPT-9274%2520decreases%2520kidney%2520cancer%2520growth%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2119%26epage%3D2129%26doi%3D10.1158%2F1535-7163.MCT-16-0197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span> </span><span class="NLM_article-title">LCH-7749944, a novel and potent p21-activated kinase 4 inhibitor, suppresses proliferation and invasion in human gastric cancer cells</span> <span class="citation_source-journal">Cancer Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">317</span><span class="NLM_x">, </span> <span class="NLM_fpage">24</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1016/j.canlet.2011.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2Fj.canlet.2011.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=22085492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2mu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=317&publication_year=2012&pages=24-32&author=J.+Zhangauthor=J.+Wangauthor=Q.+Guoauthor=Y.+Wangauthor=Y.+Zhouauthor=H.+Pengauthor=M.+Chengauthor=D.+Zhaoauthor=F.+Li&title=LCH-7749944%2C+a+novel+and+potent+p21-activated+kinase+4+inhibitor%2C+suppresses+proliferation+and+invasion+in+human+gastric+cancer+cells&doi=10.1016%2Fj.canlet.2011.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">LCH-7749944, a novel and potent p21-activated kinase 4 inhibitor, suppresses proliferation and invasion in human gastric cancer cells</span></div><div class="casAuthors">Zhang, Jian; Wang, Jian; Guo, Qiqiang; Wang, Yu; Zhou, Ying; Peng, Huizhi; Cheng, Maosheng; Zhao, Dongmei; Li, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">317</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-32</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">P21-activated kinase 4 (PAK4), a serine/threonine protein kinase, has involved in the regulation of cytoskeletal reorganization, cell proliferation, gene transcription, oncogenic transformation and cell invasion.  Moreover, PAK4 overexpression, genetic amplification and mutations were detected in a variety of human tumors, which make it potential therapeutic target.  In this paper we found that LCH-7749944, a novel and potent PAK4 inhibitor, effectively suppressed the proliferation of human gastric cancer cells through downregulation of PAK4/c-Src/EGFR/cyclin D1 pathway.  In addn., LCH-7749944 significantly inhibited the migration and invasion of human gastric cancer cells in conjunction with concomitant blockage of PAK4/LIMK1/cofilin and PAK4/MEK-1/ERK1/2/MMP2 pathways.  Interestingly, LCH-7749944 also inhibited the formation of filopodia and induced cell elongation in SGC7901 cells.  Importantly, LCH-7749944 caused successful inhibition of EGFR activity due to its inhibitory effect on PAK4.  Taken together, these results provided novel insights into the development of PAK4 inhibitor and potential therapeutic strategies for gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaLHQjMNmVxbVg90H21EOLACvtfcHk0lgSmdri2gAzhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2mu7s%253D&md5=b5b8a014f7efd0247bdc5756050389a4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2011.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2011.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26atitle%3DLCH-7749944%252C%2520a%2520novel%2520and%2520potent%2520p21-activated%2520kinase%25204%2520inhibitor%252C%2520suppresses%2520proliferation%2520and%2520invasion%2520in%2520human%2520gastric%2520cancer%2520cells%26jtitle%3DCancer%2520Lett.%26date%3D2012%26volume%3D317%26spage%3D24%26epage%3D32%26doi%3D10.1016%2Fj.canlet.2011.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Hao, C. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, W. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, Z. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, M. S.</span><span> </span><span class="NLM_article-title">Development of 2, 4-diaminoquinazoline derivatives as potent PAK4 inhibitors by the core refinement strategy</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2017.02.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2Fj.ejmech.2017.02.063" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2017&pages=1-13&author=C.+Z.+Haoauthor=W.+X.+Huangauthor=X.+D.+Liauthor=J.+Guoauthor=M.+Chenauthor=Z.+Z.+Yanauthor=K.+Wangauthor=X.+L.+Jiangauthor=S.+Songauthor=J.+Wangauthor=D.+M.+Zhaoauthor=F.+Liauthor=M.+S.+Cheng&title=Development+of+2%2C+4-diaminoquinazoline+derivatives+as+potent+PAK4+inhibitors+by+the+core+refinement+strategy&doi=10.1016%2Fj.ejmech.2017.02.063"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.02.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.02.063%26sid%3Dliteratum%253Aachs%26aulast%3DHao%26aufirst%3DC.%2BZ.%26aulast%3DHuang%26aufirst%3DW.%2BX.%26aulast%3DLi%26aufirst%3DX.%2BD.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DYan%26aufirst%3DZ.%2BZ.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DJiang%26aufirst%3DX.%2BL.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DD.%2BM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DCheng%26aufirst%3DM.%2BS.%26atitle%3DDevelopment%2520of%25202%252C%25204-diaminoquinazoline%2520derivatives%2520as%2520potent%2520PAK4%2520inhibitors%2520by%2520the%2520core%2520refinement%2520strategy%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D131%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.ejmech.2017.02.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Hao, C. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, W. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, D. M.</span><span> </span><span class="NLM_article-title">Advances in the 1-phenanthryl-tetrahydroisoquinoline series of PAK4 inhibitors: potent agents restrain tumor cell growth and invasion</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">7676</span><span class="NLM_x">–</span> <span class="NLM_lpage">7690</span><span class="refDoi"> DOI: 10.1039/C6OB01072E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1039%2FC6OB01072E" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=7676-7690&author=C.+Z.+Haoauthor=X.+D.+Liauthor=S.+Songauthor=B.+Y.+Guoauthor=J.+Guoauthor=J.+Zhangauthor=Q.+L.+Zhangauthor=W.+X.+Huangauthor=J.+Wangauthor=B.+Linauthor=M.+S.+Chengauthor=F.+Liauthor=D.+M.+Zhao&title=Advances+in+the+1-phenanthryl-tetrahydroisoquinoline+series+of+PAK4+inhibitors%3A+potent+agents+restrain+tumor+cell+growth+and+invasion&doi=10.1039%2FC6OB01072E"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC6OB01072E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6OB01072E%26sid%3Dliteratum%253Aachs%26aulast%3DHao%26aufirst%3DC.%2BZ.%26aulast%3DLi%26aufirst%3DX.%2BD.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DGuo%26aufirst%3DB.%2BY.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%2BL.%26aulast%3DHuang%26aufirst%3DW.%2BX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DB.%26aulast%3DCheng%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DD.%2BM.%26atitle%3DAdvances%2520in%2520the%25201-phenanthryl-tetrahydroisoquinoline%2520series%2520of%2520PAK4%2520inhibitors%253A%2520potent%2520agents%2520restrain%2520tumor%2520cell%2520growth%2520and%2520invasion%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2016%26volume%3D14%26spage%3D7676%26epage%3D7690%26doi%3D10.1039%2FC6OB01072E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Thermal denaturation assays in chemical biology</span> <span class="citation_source-journal">Assay Drug Dev. Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">128</span><span class="NLM_x">–</span> <span class="NLM_lpage">136</span><span class="refDoi"> DOI: 10.1089/adt.2011.0390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1089%2Fadt.2011.0390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=22066913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC38Xltl2mt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=128-136&author=G.+Senisterraauthor=I.+Chauauthor=M.+Vedadi&title=Thermal+denaturation+assays+in+chemical+biology&doi=10.1089%2Fadt.2011.0390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Thermal Denaturation Assays in Chemical Biology</span></div><div class="casAuthors">Senisterra, Guillermo; Chau, Irene; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">128-136</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  Thermal denaturation-based methods are becoming increasingly used to characterize protein stability and interactions.  Recent tech. advances have made these methods more suitable for high throughput screening.  Reasonable throughput and the ability to perform these screens using commonly used instruments, such as RT-PCR machines or simple plate readers equipped with heating devices, facilitate these expts. in almost any lab.  Introducing an aggregation-based monitoring approach as well as alternative fluorophores has allowed the screening of a wider range of proteins, including membrane proteins, against large chem. libraries.  Thermal denaturation-based methods are independent of protein function, which is esp. useful for the identification of orphan protein function.  Here, we review applications of thermal denaturation-based methods in characterizing protein stability and ligand binding, and also provide information on protocol modifications that may further increase throughput.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH8b9M2zSFsbVg90H21EOLACvtfcHk0lgSmdri2gAzhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xltl2mt7s%253D&md5=4631ad4237d7050f1cb460ee89490f0d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1089%2Fadt.2011.0390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2011.0390%26sid%3Dliteratum%253Aachs%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThermal%2520denaturation%2520assays%2520in%2520chemical%2520biology%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2012%26volume%3D10%26spage%3D128%26epage%3D136%26doi%3D10.1089%2Fadt.2011.0390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Homola, J.</span><span> </span><span class="NLM_article-title">Surface plasmon resonance sensors for detection of chemical and biological species</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">462</span><span class="NLM_x">–</span> <span class="NLM_lpage">493</span><span class="refDoi"> DOI: 10.1021/cr068107d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr068107d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFCiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2008&pages=462-493&author=J.+Homola&title=Surface+plasmon+resonance+sensors+for+detection+of+chemical+and+biological+species&doi=10.1021%2Fcr068107d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Surface plasmon resonance sensors for detection of chemical and biological species</span></div><div class="casAuthors">Homola, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-493</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  This paper reviews advances in surface plasmon resonance (SPR) sensor technol. and its applications since the year 2000.  The review focuses on SPR sensors employing conventional (unlocalized) surface plasmons propagating along planar structures and their applications for detection of chem. and biol. species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwiJkLw_n13rVg90H21EOLACvtfcHk0ljmJho7q4DZkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFCiu7Y%253D&md5=ab7e19092725e31728b5ff7e73c921b6</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fcr068107d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr068107d%26sid%3Dliteratum%253Aachs%26aulast%3DHomola%26aufirst%3DJ.%26atitle%3DSurface%2520plasmon%2520resonance%2520sensors%2520for%2520detection%2520of%2520chemical%2520and%2520biological%2520species%26jtitle%3DChem.%2520Rev.%26date%3D2008%26volume%3D108%26spage%3D462%26epage%3D493%26doi%3D10.1021%2Fcr068107d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Pierce, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandretto, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, G. W.</span><span> </span><span class="NLM_article-title">Kinase inhibitors and the case for CH···O hydrogen bonds in protein-ligand binding</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">567</span><span class="NLM_x">–</span> <span class="NLM_lpage">576</span><span class="refDoi"> DOI: 10.1002/prot.10259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1002%2Fprot.10259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=12402365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD38XptlGru7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2002&pages=567-576&author=A.+C.+Pierceauthor=K.+L.+Sandrettoauthor=G.+W.+Bemis&title=Kinase+inhibitors+and+the+case+for+CH%C2%B7%C2%B7%C2%B7O+hydrogen+bonds+in+protein-ligand+binding&doi=10.1002%2Fprot.10259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors and the case for CH···O hydrogen bonds in protein-ligand binding</span></div><div class="casAuthors">Pierce, Albert C.; Sandretto, Kathryn L.; Bemis, Guy W.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">567-576</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Although the hydrogen bond is known to be an important mediator of intermol. interactions, there has yet to be an anal. of the role of CH···O hydrogen bonds in protein-ligand complexes.  In this work, we present evidence for such nonstandard hydrogen bonds from a survey of arom. ligands in 184 kinase crystal structures and 358 high-resoln. structures from the Protein Data Bank.  CH groups adjacent to the pos. charged nitrogen of nicotinamide exhibit geometric preferences strongly suggestive of hydrogen bonding interactions, as do heterocyclic CH groups in kinase ligands, while other arom. CH groups do not exhibit these characteristics.  Ab initio calcns. reveal a considerable range of CH···O hydrogen bonding potentials among different arom. ring systems, with nicotinamide and heterocycles preferred in kinase inhibitors showing particularly favorable interactions.  These results provide compelling evidence for the existence of CH···O hydrogen bonds in protein-ligand interactions, as well as information on the relative strength of various arom. CH donors.  Such knowledge will be of considerable value in protein modeling, ligand design, and structure-activity anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ALx7F6vIXLVg90H21EOLACvtfcHk0ljmJho7q4DZkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptlGru7c%253D&md5=641cc069501f9243df3ddc35cd3aa449</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fprot.10259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10259%26sid%3Dliteratum%253Aachs%26aulast%3DPierce%26aufirst%3DA.%2BC.%26aulast%3DSandretto%26aufirst%3DK.%2BL.%26aulast%3DBemis%26aufirst%3DG.%2BW.%26atitle%3DKinase%2520inhibitors%2520and%2520the%2520case%2520for%2520CH%25C2%25B7%25C2%25B7%25C2%25B7O%2520hydrogen%2520bonds%2520in%2520protein-ligand%2520binding%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2002%26volume%3D49%26spage%3D567%26epage%3D576%26doi%3D10.1002%2Fprot.10259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alex, A.</span><span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eY2cNeMixy2hLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Springthorpe, B.</span><span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">881</span><span class="NLM_x">–</span> <span class="NLM_lpage">890</span><span class="refDoi"> DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881-890&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0lhQng_OHa6WDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D881%26epage%3D890%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Eswaran, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debreczeni, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turnbull, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Crystal Structures of the p21-activated kinases PAK4, PAK5, and PAK6 reveal catalytic domain plasticity of active group II PAKs</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">201</span><span class="NLM_x">–</span> <span class="NLM_lpage">213</span><span class="refDoi"> DOI: 10.1016/j.str.2007.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2Fj.str.2007.01.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=201-213&author=J.+Eswaranauthor=W.+H.+Leeauthor=J.+E.+Debreczeniauthor=P.+Filippakopoulosauthor=A.+Turnbullauthor=O.+Fedorovauthor=S.+W.+Deaconauthor=J.+R.+Petersonauthor=S.+Knapp&title=Crystal+Structures+of+the+p21-activated+kinases+PAK4%2C+PAK5%2C+and+PAK6+reveal+catalytic+domain+plasticity+of+active+group+II+PAKs&doi=10.1016%2Fj.str.2007.01.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2007.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2007.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DEswaran%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DW.%2BH.%26aulast%3DDebreczeni%26aufirst%3DJ.%2BE.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DTurnbull%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DDeacon%26aufirst%3DS.%2BW.%26aulast%3DPeterson%26aufirst%3DJ.%2BR.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DCrystal%2520Structures%2520of%2520the%2520p21-activated%2520kinases%2520PAK4%252C%2520PAK5%252C%2520and%2520PAK6%2520reveal%2520catalytic%2520domain%2520plasticity%2520of%2520active%2520group%2520II%2520PAKs%26jtitle%3DStructure%26date%3D2007%26volume%3D15%26spage%3D201%26epage%3D213%26doi%3D10.1016%2Fj.str.2007.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">van Linden, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">KLIFS: a knowledge-based structural database to navigate kinase-ligand interaction space</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="NLM_x">–</span> <span class="NLM_lpage">277</span><span class="refDoi"> DOI: 10.1021/jm400378w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400378w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=249-277&author=O.+P.+van+Lindenauthor=A.+J.+Kooistraauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=KLIFS%3A+a+knowledge-based+structural+database+to+navigate+kinase-ligand+interaction+space&doi=10.1021%2Fjm400378w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: A Knowledge-Based Structural Database To Navigate Kinase-Ligand Interaction Space</span></div><div class="casAuthors">van Linden, Oscar P. J.; Kooistra, Albert J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-277</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein kinases regulate the majority of signal transduction pathways in cells and have become important targets for the development of designer drugs.  We present a systematic anal. of kinase-ligand interactions in all regions of the catalytic cleft of all 1252 human kinase-ligand cocrystal structures present in the Protein Data Bank (PDB).  The kinase-ligand interaction fingerprints and structure database (KLIFS) contains a consistent alignment of 85 kinase ligand binding site residues that enables the identification of family specific interaction features and classification of ligands according to their binding modes.  We illustrate how systematic mining of kinase-ligand interaction space gives new insights into how conserved and selective kinase interaction hot spots can accommodate the large diversity of chem. scaffolds in kinase ligands.  These analyses lead to an improved understanding of the structural requirements of kinase binding that will be useful in ligand discovery and design studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4tMmT6S-vRrVg90H21EOLACvtfcHk0lhQng_OHa6WDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P&md5=8519942e1703ba04f64797e07df4b712</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm400378w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400378w%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKLIFS%253A%2520a%2520knowledge-based%2520structural%2520database%2520to%2520navigate%2520kinase-ligand%2520interaction%2520space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D249%26epage%3D277%26doi%3D10.1021%2Fjm400378w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanev, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Linden, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">KLIFS: a structural kinase-ligand interaction database</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">D365</span><span class="NLM_x">–</span> <span class="NLM_lpage">371</span><span class="refDoi"> DOI: 10.1093/nar/gkv1082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1093%2Fnar%2Fgkv1082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=26496949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D365-371&author=A.+J.+Kooistraauthor=G.+K.+Kanevauthor=O.+P.+van+Lindenauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=KLIFS%3A+a+structural+kinase-ligand+interaction+database&doi=10.1093%2Fnar%2Fgkv1082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: a structural kinase-ligand interaction database</span></div><div class="casAuthors">Kooistra, Albert J.; Kanev, Georgi K.; van Linden, Oscar P. J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D365-D371</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein kinases play a crucial role in cell signaling and are important drug targets in several therapeutic areas.  The KLIFS database contains detailed structural kinase-ligand interaction information derived from all (>2900) structures of catalytic domains of human and mouse protein kinases deposited in the Protein Data Bank in order to provide insights into the structural determinants of kinase-ligand binding and selectivity.  The kinase structures have been processed in a consistent manner by systematically analyzing the structural features and mol. interaction fingerprints (IFPs) of a predefined set of 85 binding site residues with bound ligands.  KLIFS has been completely rebuilt and extended (>65% more structures) since its first release as a data set, including: novel automated annotation methods for (i) the assessment of ligand-targeted subpockets and the anal. of (ii) DFG and (iii) αC-helix conformations; improved and automated protocols for (iv) the generation of sequence/structure alignments, (v) the curation of ligand atom and bond typing for accurate IFP anal. and (vi) weekly database updates.  KLIFS is now accessible via a website (http://klifs.vucompmedchem. nl) that provides a comprehensive visual presentation of different types of chem., biol. and structural chemogenomics data, and allows the user to easily access, compare, search and download the data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocfCaKwwgOlbVg90H21EOLACvtfcHk0lhQng_OHa6WDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfP&md5=bc093b02225987fe791a2d41738e4d99</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1082%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DKanev%26aufirst%3DG.%2BK.%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKLIFS%253A%2520a%2520structural%2520kinase-ligand%2520interaction%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD365%26epage%3D371%26doi%3D10.1093%2Fnar%2Fgkv1082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Vulpetti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bosotti, R.</span><span> </span><span class="NLM_article-title">Sequence and structural analysis of kinase ATP pocket residues</span> <span class="citation_source-journal">Farmaco</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">759</span><span class="NLM_x">–</span> <span class="NLM_lpage">765</span><span class="refDoi"> DOI: 10.1016/j.farmac.2004.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2Fj.farmac.2004.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=15474052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlWms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2004&pages=759-765&author=A.+Vulpettiauthor=R.+Bosotti&title=Sequence+and+structural+analysis+of+kinase+ATP+pocket+residues&doi=10.1016%2Fj.farmac.2004.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence and structural analysis of kinase ATP pocket residues</span></div><div class="casAuthors">Vulpetti, Anna; Bosotti, Roberta</div><div class="citationInfo"><span class="NLM_cas:title">Farmaco</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">759-765</span>CODEN:
                <span class="NLM_cas:coden">FRMCE8</span>;
        ISSN:<span class="NLM_cas:issn">0014-827X</span>.
    
            (<span class="NLM_cas:orgname">Editions Scientifiques et Medicales Elsevier</span>)
        </div><div class="casAbstract">Protein kinases represent one of the largest known families of enzymes.  Most kinases bind ATP and most synthetic kinase inhibitors are ATP-competitive, which makes selectivity a potential problem.  However, despite the high sequence similarity in the ATP binding pocket, several groups including ours have been able to develop highly potent and selective ATP-competitive inhibitors.  To systematically aid the design of specific inhibitors in our protein kinase projects, we aligned all known three-dimensional structures and all known sequences of human protein kinases.  We identified a set of 38 residues that make up the ATP pocket and analyzed the variability among these residues.  The most variable residues in the ATP pocket are targeted to design specificity into inhibitors in our various kinase projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKrQCe0MaGh7Vg90H21EOLACvtfcHk0ljIJJswqmXjIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlWms7o%253D&md5=d710962f752fcd893880696e34bf35a1</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.farmac.2004.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.farmac.2004.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DVulpetti%26aufirst%3DA.%26aulast%3DBosotti%26aufirst%3DR.%26atitle%3DSequence%2520and%2520structural%2520analysis%2520of%2520kinase%2520ATP%2520pocket%2520residues%26jtitle%3DFarmaco%26date%3D2004%26volume%3D59%26spage%3D759%26epage%3D765%26doi%3D10.1016%2Fj.farmac.2004.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Ruegg, U. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillian, B.</span><span> </span><span class="NLM_article-title">Staurosporine, K-252 and UCN-01: potent but nonspecific inhibitors of protein kinases</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">218</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.1016/0165-6147(89)90263-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2F0165-6147%2889%2990263-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1989&pages=218-220&author=U.+T.+Rueggauthor=B.+Gillian&title=Staurosporine%2C+K-252+and+UCN-01%3A+potent+but+nonspecific+inhibitors+of+protein+kinases&doi=10.1016%2F0165-6147%2889%2990263-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2F0165-6147%2889%2990263-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0165-6147%252889%252990263-0%26sid%3Dliteratum%253Aachs%26aulast%3DRuegg%26aufirst%3DU.%2BT.%26aulast%3DGillian%26aufirst%3DB.%26atitle%3DStaurosporine%252C%2520K-252%2520and%2520UCN-01%253A%2520potent%2520but%2520nonspecific%2520inhibitors%2520of%2520protein%2520kinases%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D1989%26volume%3D10%26spage%3D218%26epage%3D220%26doi%3D10.1016%2F0165-6147%2889%2990263-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Gilli, G. G. a. P.</span> <span class="citation_source-book">The Nature of the Hydrogen Bond</span>; <span class="NLM_publisher-name">Oxford University Press Inc.</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1093%2Facprof%3Aoso%2F9780199558964.001.0001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=G.+G.+a.+P.+Gilli&title=The+Nature+of+the+Hydrogen+Bond"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Facprof%3Aoso%2F9780199558964.001.0001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1093%252Facprof%253Aoso%252F9780199558964.001.0001%26sid%3Dliteratum%253Aachs%26aulast%3DGilli%26aufirst%3DG.%2BG.%2Ba.%2BP.%26btitle%3DThe%2520Nature%2520of%2520the%2520Hydrogen%2520Bond%26pub%3DOxford%2520University%2520Press%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Wang, R. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. M.</span><span> </span><span class="NLM_article-title">Further development and validation of empirical scoring functions for structure-based binding affinity prediction</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1023/A:1016357811882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1023%2FA%3A1016357811882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=12197663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BD38XltlSqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=11-26&author=R.+X.+Wangauthor=L.+H.+Laiauthor=S.+M.+Wang&title=Further+development+and+validation+of+empirical+scoring+functions+for+structure-based+binding+affinity+prediction&doi=10.1023%2FA%3A1016357811882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Further development and validation of empirical scoring functions for structure-based binding affinity prediction</span></div><div class="casAuthors">Wang, Renxiao; Lai, Luhua; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-26</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">New empirical scoring functions have been developed to est. the binding affinity of a given protein-ligand complex with known three-dimensional structure.  These scoring functions include terms accounting for van der Waals interaction, hydrogen bonding, deformation penalty, and hydrophobic effect.  A special feature is that three different algorithms have been implemented to calc. the hydrophobic effect term, which results in three parallel scoring functions.  All three scoring functions are calibrated through multivariate regression anal. of a set of 200 protein-ligand complexes and they reproduce the binding free energies of the entire training set with std. deviations of 2.2 kcal/mol, 2.1 kcal/mol, and 2.0 kcal/mol, resp.  These three scoring functions are further combined into a consensus scoring function, X-CSCORE.  When tested on an independent set of 30 protein-ligand complexes, X-CSCORE is able to predict their binding free energies with a std. deviation of 2.2 kcal/mol.  The potential application of X-CSCORE to mol. docking is also investigated.  Our results show that this consensus scoring function improves the docking accuracy considerably when compared to the conventional force field computation used for mol. docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqULje6PqR73rVg90H21EOLACvtfcHk0ljIJJswqmXjIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltlSqs7k%253D&md5=f77fd72d168df67d7fb66a88bc3b6cd9</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1023%2FA%3A1016357811882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1016357811882%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%2BX.%26aulast%3DLai%26aufirst%3DL.%2BH.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DFurther%2520development%2520and%2520validation%2520of%2520empirical%2520scoring%2520functions%2520for%2520structure-based%2520binding%2520affinity%2520prediction%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D11%26epage%3D26%26doi%3D10.1023%2FA%3A1016357811882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Wolber, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langer, T.</span><span> </span><span class="NLM_article-title">LigandScout: 3-d pharmacophores derived from protein-bound Ligands and their use as virtual screening filters</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">160</span><span class="NLM_x">–</span> <span class="NLM_lpage">169</span><span class="refDoi"> DOI: 10.1021/ci049885e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049885e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=160-169&author=G.+Wolberauthor=T.+Langer&title=LigandScout%3A+3-d+pharmacophores+derived+from+protein-bound+Ligands+and+their+use+as+virtual+screening+filters&doi=10.1021%2Fci049885e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">LigandScout: 3-D pharmacophores derived from protein-bound ligands and their use as virtual screening filters</span></div><div class="casAuthors">Wolber Gerhard; Langer Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">160-9</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">From the historically grown archive of protein-ligand complexes in the Protein Data Bank small organic ligands are extracted and interpreted in terms of their chemical characteristics and features.  Subsequently, pharmacophores representing ligand-receptor interaction are derived from each of these small molecules and its surrounding amino acids.  Based on a defined set of only six types of chemical features and volume constraints, three-dimensional pharmacophore models are constructed, which are sufficiently selective to identify the described binding mode and are thus a useful tool for in-silico screening of large compound databases.  The algorithms for ligand extraction and interpretation as well as the pharmacophore creation technique from the automatically interpreted data are presented and applied to a rhinovirus capsid complex as application example.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX2BAiYWy6n3SfW6udTcc2eaf0xapn14DUbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtg%253D%253D&md5=31f8ff7a2fa4c9411d54b001b7e1da50</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci049885e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049885e%26sid%3Dliteratum%253Aachs%26aulast%3DWolber%26aufirst%3DG.%26aulast%3DLanger%26aufirst%3DT.%26atitle%3DLigandScout%253A%25203-d%2520pharmacophores%2520derived%2520from%2520protein-bound%2520Ligands%2520and%2520their%2520use%2520as%2520virtual%2520screening%2520filters%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D160%26epage%3D169%26doi%3D10.1021%2Fci049885e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Cai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span> </span><span class="NLM_article-title">Overexpression of P21-activated kinase 4 is associated with poor prognosis in non-small cell lung cancer and promotes migration and invasion</span> <span class="citation_source-journal">J. Exp. Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">48</span><span class="refDoi"> DOI: 10.1186/s13046-015-0165-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1186%2Fs13046-015-0165-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=25975262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A280%3ADC%252BC2MfktVSjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2015&pages=48&author=S.+Caiauthor=Z.+Yeauthor=X.+Wangauthor=Y.+Panauthor=Y.+Wengauthor=S.+Laoauthor=H.+Weiauthor=L.+Li&title=Overexpression+of+P21-activated+kinase+4+is+associated+with+poor+prognosis+in+non-small+cell+lung+cancer+and+promotes+migration+and+invasion&doi=10.1186%2Fs13046-015-0165-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of P21-activated kinase 4 is associated with poor prognosis in non-small cell lung cancer and promotes migration and invasion</span></div><div class="casAuthors">Cai Songwang; Weng Yimin; Lao Sen; Ye Zhiqiang; Wang Xiaohong; Pan Yuhang; Wei Hongbo; Li Lian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of experimental & clinical cancer research : CR</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">48</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  P21-activated kinase 4 (PAK4), an effector of the Rho family protein Cdc42, is an important oncogene whose expression is increased in many human cancers and is generally positively correlated with advanced disease and decreased survival.  However, little is known about the expression and biological function of PAK4 in human non-small cell lung cancer (NSCLC).  METHODS:  PAK4 expression in NSCLC tissues and adjacent non-tumor tissues were assessed by immunohistochemistry, real-time PCR, and western blotting.  Prognostic value of PAK4 expression was evaluated by Kaplan-Meier analysis and Cox regression. siRNA-mediated gene silencing and protein kinase assay was applied to demonstrate the role and the mechanism of PAK4 in lung cancer cell migration, invasion.  RESULTS:  The results showed that PAK4 was overexpressed in NSCLC cell lines and human NSCLC tissues.  PAK4 expression was detected both in the membranes and cytoplasm of NSCLC cancer cells in vivo.  Moreover, increased expression of PAK4 was associated with metastasis, shorter overall survival, advanced stage of NSCLC.  Furthermore, PAK4 expression was positively correlated with phosphorylation of LIMK1 expression levels.  Knockdown of PAK4 in NSCLC cell lines led to reduce the phosphorylation of LIMK1, which resulted in decrease of the cell migration and invasion.  In addition, PAK4 bound to LIMK1 directly and activated it via phosphorylation.  CONCLUSIONS:  These data demonstrate that PAK4 mediated LIMK1 phosphorylation regulates the migration and invasion in NSCLC.  Therefore, PAK4 might be a significant prognostic marker and potential therapeutic molecular target in NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT7aG3uVGW52fcsfMJQC1ZdfW6udTcc2eaf0xapn14DUbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MfktVSjuw%253D%253D&md5=c725f528e73c73e1fe9ccfe4bd80ba0e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1186%2Fs13046-015-0165-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-015-0165-2%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DS.%26aulast%3DYe%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DWeng%26aufirst%3DY.%26aulast%3DLao%26aufirst%3DS.%26aulast%3DWei%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DL.%26atitle%3DOverexpression%2520of%2520P21-activated%2520kinase%25204%2520is%2520associated%2520with%2520poor%2520prognosis%2520in%2520non-small%2520cell%2520lung%2520cancer%2520and%2520promotes%2520migration%2520and%2520invasion%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D34%26spage%3D48%26doi%3D10.1186%2Fs13046-015-0165-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Mitcheson, J. S.</span><span> </span><span class="NLM_article-title">hERG potassium channels and the structural basis of drug-induced arrhythmias</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1005</span><span class="NLM_x">–</span> <span class="NLM_lpage">1010</span><span class="refDoi"> DOI: 10.1021/tx800035b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx800035b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2008&pages=1005-1010&author=J.+S.+Mitcheson&title=hERG+potassium+channels+and+the+structural+basis+of+drug-induced+arrhythmias&doi=10.1021%2Ftx800035b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Ftx800035b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx800035b%26sid%3Dliteratum%253Aachs%26aulast%3DMitcheson%26aufirst%3DJ.%2BS.%26atitle%3DhERG%2520potassium%2520channels%2520and%2520the%2520structural%2520basis%2520of%2520drug-induced%2520arrhythmias%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26spage%3D1005%26epage%3D1010%26doi%3D10.1021%2Ftx800035b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Su, Q. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ioannidis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuaqui, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almeida, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alimzhanov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bebernitz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Block, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huszar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costa, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinda, M.</span><span> </span><span class="NLM_article-title">Discovery of 1-methyl-1<i>H</i>-imidazole derivatives as potent Jak2 inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">144</span><span class="NLM_x">–</span> <span class="NLM_lpage">158</span><span class="refDoi"> DOI: 10.1021/jm401546n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401546n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWltLnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=144-158&author=Q.+B.+Suauthor=S.+Ioannidisauthor=C.+Chuaquiauthor=L.+Almeidaauthor=M.+Alimzhanovauthor=G.+Bebernitzauthor=K.+Bellauthor=M.+Blockauthor=T.+Howardauthor=S.+Huangauthor=D.+Huszarauthor=J.+A.+Readauthor=C.+R.+Costaauthor=J.+Shiauthor=M.+Suauthor=M.+W.+Yeauthor=M.+Zinda&title=Discovery+of+1-methyl-1H-imidazole+derivatives+as+potent+Jak2+inhibitors&doi=10.1021%2Fjm401546n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 1-Methyl-1H-imidazole Derivatives as Potent Jak2 Inhibitors</span></div><div class="casAuthors">Su, Qibin; Ioannidis, Stephanos; Chuaqui, Claudio; Almeida, Lynsie; Alimzhanov, Marat; Bebernitz, Geraldine; Bell, Kirsten; Block, Michael; Howard, Tina; Huang, Shan; Huszar, Dennis; Read, Jon A.; Rivard Costa, Caroline; Shi, Jie; Su, Mei; Ye, Minwei; Zinda, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">144-158</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure based design, synthesis, and biol. evaluation of a novel series of 1-methyl-1H-imidazole, as potent Jak2 inhibitors to modulate the Jak/STAT pathway, are described.  Using the C-ring fragment from our first clin. candidate AZD1480 (I), optimization of the series led to the discovery of compd. II, a potent, orally bioavailable Jak2 inhibitor.  Compd. II displayed a high level of cellular activity in hematopoietic cell lines harboring the V617F mutation and in murine BaF3 TEL-Jak2 cells.  Compd. II demonstrated significant tumor growth inhibition in a UKE-1 xenograft model within a well-tolerated dose range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqczgU5RpFv1bVg90H21EOLACvtfcHk0lgTAe0wehY9OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWltLnL&md5=88c57119ec77d084a91a1123fd9dea86</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm401546n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401546n%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DQ.%2BB.%26aulast%3DIoannidis%26aufirst%3DS.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DAlmeida%26aufirst%3DL.%26aulast%3DAlimzhanov%26aufirst%3DM.%26aulast%3DBebernitz%26aufirst%3DG.%26aulast%3DBell%26aufirst%3DK.%26aulast%3DBlock%26aufirst%3DM.%26aulast%3DHoward%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DHuszar%26aufirst%3DD.%26aulast%3DRead%26aufirst%3DJ.%2BA.%26aulast%3DCosta%26aufirst%3DC.%2BR.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DM.%26aulast%3DYe%26aufirst%3DM.%2BW.%26aulast%3DZinda%26aufirst%3DM.%26atitle%3DDiscovery%2520of%25201-methyl-1H-imidazole%2520derivatives%2520as%2520potent%2520Jak2%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D144%26epage%3D158%26doi%3D10.1021%2Fjm401546n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Sakamoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriya, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuge, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haga, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nonoshita, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakuraba, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirohashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shibata, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanno, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwaasa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gomori, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishihara, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuroda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanatani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukami, T.</span><span> </span><span class="NLM_article-title">Novel orally active NPY Y5 receptor antagonists: synthesis and structure-activity relationship of spiroindoline class compounds</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">5015</span><span class="NLM_x">–</span> <span class="NLM_lpage">5026</span><span class="refDoi"> DOI: 10.1016/j.bmc.2009.05.064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2Fj.bmc.2009.05.064" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=5015-5026&author=T.+Sakamotoauthor=M.+Moriyaauthor=H.+Tsugeauthor=T.+Takahashiauthor=Y.+Hagaauthor=K.+Nonoshitaauthor=O.+Okamotoauthor=H.+Takahashiauthor=A.+Sakurabaauthor=T.+Hirohashiauthor=T.+Shibataauthor=T.+Kannoauthor=J.+Itoauthor=H.+Iwaasaauthor=A.+Gomoriauthor=A.+Ishiharaauthor=T.+Fukurodaauthor=A.+Kanataniauthor=T.+Fukami&title=Novel+orally+active+NPY+Y5+receptor+antagonists%3A+synthesis+and+structure-activity+relationship+of+spiroindoline+class+compounds&doi=10.1016%2Fj.bmc.2009.05.064"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2009.05.064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2009.05.064%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DT.%26aulast%3DMoriya%26aufirst%3DM.%26aulast%3DTsuge%26aufirst%3DH.%26aulast%3DTakahashi%26aufirst%3DT.%26aulast%3DHaga%26aufirst%3DY.%26aulast%3DNonoshita%26aufirst%3DK.%26aulast%3DOkamoto%26aufirst%3DO.%26aulast%3DTakahashi%26aufirst%3DH.%26aulast%3DSakuraba%26aufirst%3DA.%26aulast%3DHirohashi%26aufirst%3DT.%26aulast%3DShibata%26aufirst%3DT.%26aulast%3DKanno%26aufirst%3DT.%26aulast%3DIto%26aufirst%3DJ.%26aulast%3DIwaasa%26aufirst%3DH.%26aulast%3DGomori%26aufirst%3DA.%26aulast%3DIshihara%26aufirst%3DA.%26aulast%3DFukuroda%26aufirst%3DT.%26aulast%3DKanatani%26aufirst%3DA.%26aulast%3DFukami%26aufirst%3DT.%26atitle%3DNovel%2520orally%2520active%2520NPY%2520Y5%2520receptor%2520antagonists%253A%2520synthesis%2520and%2520structure-activity%2520relationship%2520of%2520spiroindoline%2520class%2520compounds%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D5015%26epage%3D5026%26doi%3D10.1016%2Fj.bmc.2009.05.064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljZQqmwNt6wzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljZQqmwNt6wzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusoff, W. H.</span><span> </span><span class="NLM_article-title">Relationship between the inhibition constant (Ki) and the concentration of inhibitor which causes 50% inhibition (IC<sub>50</sub>) of an enzymatic reaction</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3099</span><span class="NLM_x">–</span> <span class="NLM_lpage">3108</span><span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=Y.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28Ki%29+and+the+concentration+of+inhibitor+which+causes+50%25+inhibition+%28IC50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0lizJhuJ6JfzLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528Ki%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2525%2520inhibition%2520%2528IC50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Song, H. Y.; Smith, R. A. A.; Hao, C. Z.; Wang, J.; Dahlroth, S. K. S.; Cheng, M. S.; Fernig, D. G.; Gorelik, S.; Nurcombe, V.</span> Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Song%2C+H.+Y.%3B+Smith%2C+R.+A.+A.%3B+Hao%2C+C.+Z.%3B+Wang%2C+J.%3B+Dahlroth%2C+S.+K.+S.%3B+Cheng%2C+M.+S.%3B+Fernig%2C+D.+G.%3B+Gorelik%2C+S.%3B+Nurcombe%2C+V.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DH.%2BY." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Ma, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, J.</span><span> </span><span class="NLM_article-title">Design and optimization of a series of 1-sulfonylpyrazolo[4,3-b]pyridines as selective c-Met inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">2513</span><span class="NLM_x">–</span> <span class="NLM_lpage">2529</span><span class="refDoi"> DOI: 10.1021/jm502018y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm502018y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2513-2529&author=Y.+Maauthor=G.+Sunauthor=D.+Chenauthor=X.+Pengauthor=Y.+L.+Chenauthor=Y.+Suauthor=Y.+Jiauthor=J.+Liangauthor=X.+Wangauthor=L.+Chenauthor=J.+Dingauthor=B.+Xiongauthor=J.+Aiauthor=M.+Gengauthor=J.+Shen&title=Design+and+optimization+of+a+series+of+1-sulfonylpyrazolo%5B4%2C3-b%5Dpyridines+as+selective+c-Met+inhibitors&doi=10.1021%2Fjm502018y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm502018y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm502018y%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%2BL.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DB.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DJ.%26atitle%3DDesign%2520and%2520optimization%2520of%2520a%2520series%2520of%25201-sulfonylpyrazolo%255B4%252C3-b%255Dpyridines%2520as%2520selective%2520c-Met%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2513%26epage%3D2529%26doi%3D10.1021%2Fjm502018y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVF','PDB','5XVF'); return false;">PDB: 5XVF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O0R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O0R','PDB','4O0R'); return false;">PDB: 4O0R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X4Z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X4Z','PDB','2X4Z'); return false;">PDB: 2X4Z</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVG','PDB','5XVG'); return false;">PDB: 5XVG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVA','PDB','5XVA'); return false;">PDB: 5XVA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2J0I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2J0I','PDB','2J0I'); return false;">PDB: 2J0I</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i91"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01342">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_10817"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01342">10.1021/acs.jmedchem.7b01342</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Sequence alignment of the active site residues of PAK1–6; crystallographic parameters of all X-ray structures (cocrystal structures of <b>10a</b>, <b>30</b>, and <b>31</b> bound to PAK4); comparison of the orientation differences of the DFG Asp (Asp458<sup>PAK4</sup>/Asp407<sup>PAK1</sup>); binding mode analysis of compounds <b>12</b>, <b>17</b>, <b>18</b>, and <b>20</b>; detailed kinase selectivity data of compound <b>31</b>; <sup>1</sup>H NMR, <sup>13</sup>C NMR, and HRMS spectra of compounds <b>10a</b>–<b>d</b>, <b>11a</b>–<b>d</b>, and <b>12</b>–<b>41</b>; liver microsome stability, rat plasma stability, physiochemical properties determination, CYP450, and hERG inhibition reports of compound <b>31</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01342/suppl_file/jm7b01342_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and associated biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01342/suppl_file/jm7b01342_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01342/suppl_file/jm7b01342_si_001.pdf">jm7b01342_si_001.pdf (8.69 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01342/suppl_file/jm7b01342_si_002.csv">jm7b01342_si_002.csv (4.13 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB ID codes for <b>10a</b>, <b>30</b>, and <b>31</b> bound to PAK4 are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVF">5XVF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVA">5XVA</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XVG">5XVG</a>, respectively.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01342&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01342%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-1%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01342" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677ab6430d2f3c40","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
