
<html lang="en"     class="pb-page"  data-request-id="9d6258d3-2726-4147-a68f-e52d38bce158"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b01980;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-6"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structural Basis of Inhibitor Selectivity in the BRD7/9 Subfamily of Bromodomains" /></meta><meta name="dc.Creator" content="Rezaul Md  Karim" /></meta><meta name="dc.Creator" content="Alice  Chan" /></meta><meta name="dc.Creator" content="Jin-Yi  Zhu" /></meta><meta name="dc.Creator" content="Ernst  Schönbrunn" /></meta><meta name="dc.Description" content="Inhibition of the bromodomain containing protein 9 (BRD9) by small molecules is an attractive strategy to target mutated SWI/SNF chromatin-remodeling complexes in cancer. However, reported BRD9 inh..." /></meta><meta name="Description" content="Inhibition of the bromodomain containing protein 9 (BRD9) by small molecules is an attractive strategy to target mutated SWI/SNF chromatin-remodeling complexes in cancer. However, reported BRD9 inh..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 24, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01980" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01980" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01980" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01980" /></link>
        
    
    

<title>Structural Basis of Inhibitor Selectivity in the BRD7/9 Subfamily of Bromodomains | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01980" /></meta><meta property="og:title" content="Structural Basis of Inhibitor Selectivity in the BRD7/9 Subfamily of Bromodomains" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0007.jpeg" /></meta><meta property="og:description" content="Inhibition of the bromodomain containing protein 9 (BRD9) by small molecules is an attractive strategy to target mutated SWI/SNF chromatin-remodeling complexes in cancer. However, reported BRD9 inhibitors also inhibit the closely related bromodomain-containing protein 7 (BRD7), which has different biological functions. The structural basis for differential potency and selectivity of BRD9 inhibitors is largely unknown because of the lack of structural information on BRD7. Here, we biochemically and structurally characterized diverse inhibitors with varying degrees of potency and selectivity for BRD9 over BRD7. Novel cocrystal structures of BRD7 liganded with new and previously reported inhibitors of five different chemical scaffolds were determined alongside BRD9 and BRD4. We also report the discovery of first-in-class dual bromodomain–kinase inhibitors outside the bromodomain and extraterminal family targeting BRD7 and BRD9. Combined, the data provide a new framework for the development of BRD7/9 inhibitors with improved selectivity or additional polypharmacologic properties." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01980"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01980">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01980&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01980&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01980&amp;href=/doi/10.1021/acs.jmedchem.9b01980" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 3227-3237</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01979" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01986" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structural Basis of Inhibitor Selectivity in the BRD7/9 Subfamily of Bromodomains</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Rezaul Md Karim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rezaul Md Karim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Discovery, Moffitt Cancer Center, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div><div class="loa-info-affiliations-info">Department of Molecular Medicine, USF Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rezaul+Md++Karim">Rezaul Md Karim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alice Chan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alice Chan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Discovery, Moffitt Cancer Center, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alice++Chan">Alice Chan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jin-Yi Zhu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jin-Yi Zhu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Discovery, Moffitt Cancer Center, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jin-Yi++Zhu">Jin-Yi Zhu</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Ernst Schönbrunn</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ernst Schönbrunn</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Discovery, Moffitt Cancer Center, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div><div class="loa-info-affiliations-info">Department of Molecular Medicine, USF Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#aacfd8c4d9de84d9c9c2c5c4c8d8dfc4c4eac7c5ccccc3dede84c5d8cd"><span class="__cf_email__" data-cfemail="b8ddcad6cbcc96cbdbd0d7d6dacacdd6d6f8d5d7deded1cccc96d7cadf">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ernst++Sch%C3%B6nbrunn">Ernst Schönbrunn</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3589-3510" title="Orcid link">http://orcid.org/0000-0002-3589-3510</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01980&amp;href=/doi/10.1021%2Facs.jmedchem.9b01980" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 3227–3237</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 24, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>28 November 2019</li><li><span class="item_label"><b>Published</b> online</span>24 February 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 March 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01980" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01980</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3227%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRezaul%2BMd%2BKarim%252C%2BAlice%2BChan%252C%2BJin-Yi%2BZhu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D6%26contentID%3Dacs.jmedchem.9b01980%26title%3DStructural%2BBasis%2Bof%2BInhibitor%2BSelectivity%2Bin%2Bthe%2BBRD7%252F9%2BSubfamily%2Bof%2BBromodomains%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3237%26publicationDate%3DMarch%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01980"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1460</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01980" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structural Basis of Inhibitor Selectivity in the BRD7/9 Subfamily of Bromodomains&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Rezaul&quot;,&quot;last_name&quot;:&quot;Md Karim&quot;},{&quot;first_name&quot;:&quot;Alice&quot;,&quot;last_name&quot;:&quot;Chan&quot;},{&quot;first_name&quot;:&quot;Jin-Yi&quot;,&quot;last_name&quot;:&quot;Zhu&quot;},{&quot;first_name&quot;:&quot;Ernst&quot;,&quot;last_name&quot;:&quot;Schönbrunn&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;24&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;3227-3237&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01980&quot;},&quot;abstract&quot;:&quot;Inhibition of the bromodomain containing protein 9 (BRD9) by small molecules is an attractive strategy to target mutated SWI/SNF chromatin-remodeling complexes in cancer. However, reported BRD9 inhibitors also inhibit the closely related bromodomain-containing protein 7 (BRD7), which has different biological functions. The structural basis for differential potency and selectivity of BRD9 inhibitors is largely unknown because of the lack of structural information on BRD7. Here, we biochemically and structurally characterized diverse inhibitors with varying degrees of potency and selectivity for BRD9 over BRD7. Novel cocrystal structures of BRD7 liganded with new and previously reported inhibitors of five different chemical scaffolds were determined alongside BRD9 and BRD4. We also report the discovery of first-in-class dual bromodomain–kinase inhibitors outside the bromodomain and extraterminal family targeting BRD7 and BRD9. Combined, the data provide a new framework for the development of BRD7/9 inhibitors w&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01980&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01980" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01980&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01980" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01980&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01980" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01980&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01980&amp;href=/doi/10.1021/acs.jmedchem.9b01980" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01980" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01980" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01980%26sid%3Dliteratum%253Aachs%26pmid%3D32091206%26genre%3Darticle%26aulast%3DKarim%26date%3D2020%26atitle%3DStructural%2BBasis%2Bof%2BInhibitor%2BSelectivity%2Bin%2Bthe%2BBRD7%252F9%2BSubfamily%2Bof%2BBromodomains%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D6%26spage%3D3227%26epage%3D3237%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/jmcmar.2020.63.issue-6/20200326/jmcmar.2020.63.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Inhibition of the bromodomain containing protein 9 (BRD9) by small molecules is an attractive strategy to target mutated SWI/SNF chromatin-remodeling complexes in cancer. However, reported BRD9 inhibitors also inhibit the closely related bromodomain-containing protein 7 (BRD7), which has different biological functions. The structural basis for differential potency and selectivity of BRD9 inhibitors is largely unknown because of the lack of structural information on BRD7. Here, we biochemically and structurally characterized diverse inhibitors with varying degrees of potency and selectivity for BRD9 over BRD7. Novel cocrystal structures of BRD7 liganded with new and previously reported inhibitors of five different chemical scaffolds were determined alongside BRD9 and BRD4. We also report the discovery of first-in-class dual bromodomain–kinase inhibitors outside the bromodomain and extraterminal family targeting BRD7 and BRD9. Combined, the data provide a new framework for the development of BRD7/9 inhibitors with improved selectivity or additional polypharmacologic properties.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02332" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02332" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Bromodomains (BRDs) are evolutionary conserved epigenetic reader modules that recognize N-acetylated lysine (KAc) residues on histones and other proteins.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> BRD-containing proteins have important functions in gene transcription and chromatin remodeling, gene splicing, protein scaffolding, and signal transduction.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> BRD-containing proteins, particularly those of the bromodomain and the extraterminal (BET) family, have been linked to tumorigenesis and inflammatory diseases, and several BET inhibitors are in clinical trials for oncology and nononcology indications.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> More recently, inhibitors targeting non-BET BRDs for which the physiological functions are less understood have been the subject of intense efforts in academia and pharmaceutical industry alike.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">The BRD-containing proteins BRD7 and BRD9 recognize acetylated lysines on histone H3 (H3Ac).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Although they contain a structurally similar single BRD (62% sequence identity), the cellular functions of full-length BRD7 and BRD9 are remarkably different. BRD9 is a component of ncBAF (noncanonical BRG1/BRM-associated factor), a subtype of the BAF complex, while BRD7 is a component of the PBAF (polybromo-associated BAF) complex.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Importantly, BRD9 has an essential role in a number of cancers such as acute myeloid leukemia, synovial sarcoma, and malignant rhabdoid tumor.<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7−9)</a> In contrast, BRD7 is a potential tumor suppressor<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> with critical functions in oligodendrocytes progenitor cell differentiation<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and spermatogenesis.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p">BRD9 has attracted attention as a druggable subunit of mSWI/SNF chromatin-remodeling complexes, various subunits of which are mutated in nearly 20% of all human cancers.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> To target mSWI/SNF complexes in cancer, few small-molecule inhibitors of the acetyl-lysine binding site (KAc site) of BRD9 have been reported to date.<a onclick="showRef(event, 'ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17 ref18">(14−18)</a> These inhibitors showed varying degrees of selectivity across BRDs in general and within the BRD9/BRD7 subfamily in particular. As the role of BRD7 in human biology is not fully understood, off-target inhibition of BRD7 by nonselective BRD9 inhibitors bears the risk of unwanted pleiotropic effects. Recently, proteolysis-targeting chimeras (PROTACs), dBRD9 and VZ185, have been developed using the potent BRD7/9 inhibitor BI-7273 as a warhead. While dBRD9 induced the selective degradation of BRD9 over BRD7, VZ185 showed concomitant degradation of BRD9 and BRD7.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a></div><div class="NLM_p last">An obstacle to the rational design of inhibitors selective for BRD9 is the limited structural information on BRD7. Specifying the binding interactions of an inhibitor in closely related proteins are crucial for the development of selective inhibitors. Here, we applied a multifaceted approach toward the characterization of diverse small-molecule inhibitors with varying degrees of potency and selectivity for BRD7 and BRD9. We report the discovery of first-in-class dual bromodomain–kinase inhibitors outside the BET family targeting BRD7 and BRD9. TG003, a Cdc2-like kinase (Clk2) inhibitor,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> sunitinib, a multitargeted tyrosine kinase inhibitor,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and PF-477736, an inhibitor of cell cycle checkpoint kinase 1 (Chk1)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> were identified as moderately active BRD7/9 inhibitors with <i>K</i><sub>d</sub> values between 4 and 17 μM. Novel crystal structures of BRD7, BRD9 and BRD4 liganded with new and known inhibitors were determined, the information of which provides a structural basis for the differential potency and selectivity of BRD7/9 inhibitors. By combining, the data afford a new framework for the structure-based design of inhibitors with improved selectivity or additional polypharmacologic properties.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41576" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41576" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Structure–Activity Relationship of Reported BRD9 Inhibitors</h3><div class="NLM_p">To address inhibitor selectivity over BRD7, structure–activity relationship (SAR) studies were performed with five previously reported BRD9 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). Binding studies by differential scanning fluorimetry (DSF) and isothermal titration calorimetry (ITC) showed significant correlation between the data sets for both BRD7 and BRD9 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B,C, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Table S1, Figures S1, S2). For BRD9, the thermodynamic parameters of protein–inhibitor interactions were predominantly enthalpy driven and accompanied by entropic penalty (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). By contrast, BRD7 showed neutral or favorable entropic contributions upon interaction with all inhibitors. The isomers BI-7273 and BI-9564 have been reported as potent BRD7/9 inhibitors, BI-7273 lacking selectivity and BI-9564 being 12 times more selective for BRD9.<a onclick="showRef(event, 'ref16 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref18">(16,18)</a> Our data using crystallization-grade proteins support a different notion with BI-7273 being the most potent (<i>K</i><sub>d</sub> = 9 nM) and selective inhibitor of BRD9 among the compounds tested (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). This finding may explain the recent use of BI-7273 rather than BI-9564 as a warhead in the development of the BRD9-targeting PROTAC.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> BI-9564 shared with TP-472 similar activity and selectivity values; I-BRD9 and bromosporine were the least selective inhibitors.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding potential of inhibitors for BRD7 and BRD9. (A) Chemical structures of reported BRD9 inhibitors studied herein. (B) Melting temperature shifts of BRD7 and BRD9 in the presence of the100 μM inhibitor. (C) Δ<i>T</i><sub>m</sub> values determined by DSF and <i>K</i><sub>d</sub> values determined by ITC and data correlation for BRD9 (blue) and BRD7 (red). For statistical analysis, <i>r</i> is Pearson’s correlation coefficient; asterisks indicate the significance of the two-tailed P value (α = 0.05). For graphical expression, data were fitted to semilogarithmic lines. (D) Thermodynamic parameters of inhibitor interactions with BRD9 (upper panel) and BRD7 (lower panel). ITC conditions and thermograms are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Table S1 and Figures S1, S2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">While several co-crystal structures of BRD9 with inhibitors have been determined previously, the structure of BRD7 was described only in an unliganded state by NMR.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Recently, the Structural Genomics Consortium deposited a crystal structure of BRD7 liganded with BI-9564 in the PDB (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MQ1">5MQ1</a>), albeit without context. Although the BRDs of BRD7 and BRD9 share 62% sequence identity, BRD7 is temperature-sensitive with a melting temperature of <i>T</i><sub>m</sub> = 31 °C; whereas BRD9 shows the temperature stability typical of BET BRDs (<i>T</i><sub>m</sub> = 46 °C) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). The temperature sensitivity of BRD7 may explain the difficulties in obtaining X-ray crystals caused by protein denaturation during lengthy crystallization experiments. Therefore, purification of BRD7 was performed expeditiously, resulting in a highly homogeneous protein that readily formed X-ray grade crystals in unliganded and several liganded states under different crystallization conditions (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Tables S2 and S3). The overall structures of the BRDs of BRD7 and BRD9 are highly similar (rmsd = 0.63 Å); residues within the KAc site are identical and six differing residues are located in the vicinity of the KAc site (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Distances between the opposing flanks of the KAc site are similar for both proteins (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Differences in the overall structures of BRD7 and BRD9 affect the binding poses of BI-7273 and BI-9564. (A) Superposition of the crystal structures of BRD7 (beige) and BRD9 (cyan) in the unliganded state (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PPA">6PPA</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UZF">6UZF</a>). Differing residues (red arrows) are located outside the KAc site; the critical asparagine residue (BRD7<sup>N211</sup> and BRD9<sup>N216</sup>) is drawn in magenta. (B) Comparison of the unliganded KAc sites of BRD7 and BRD9. Distances between the flanks of the binding site are indicated as orange-dotted lines (in Å). (C) Cocrystal structure of BRD7 with BI-7273 along with a schematic presentation of the binding interactions (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1E">6V1E</a>). (D) Same as (C) for BI-7273 in BRD9 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EU1">5EU1</a>). (E) Superposition of both structures. (F) Cocrystal structure of BRD7 with BI-9564 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1F">6V1F</a>) and (G) of BRD9 with BI-9564 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5F1H">5F1H</a>). (H) Superposition of both structures. BRD7 residues are shown in beige and BRD9 residues in cyan. H-bonding interactions are indicated as black-dotted lines, vdW hydrophobic interactions as green-curved lines and π-stacking interactions as green-dotted lines. Crystallization conditions, data collection and refinement statistics are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Tables S2 and S3. 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> and <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omit electron density maps are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Figures S3 and S4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Cocrystal structures were determined for BRD7 liganded with BI-7273 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C) and BI-9564 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F) showing the same principle binding mode as previously determined in BRD9<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D,G). The naphthyridinone oxygen of both compounds establishes a single H-bond with BRD7<sup>N211</sup> or BRD9<sup>N216</sup>. However, the dimethoxyphenyl moieties of BI-7273 and BI-9564, which establish multiple hydrophobic van der Waals (vdW) interactions with KAc site residues, adopt slightly different positions (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E,H). These differences appear to be caused by steric hindrance with residue F160 of BRD9. In BRD7, the main- and side-chain conformations of the equivalent residue F155 differ from BRD9 presumably due to alanine versus glycine as preceding residues (BRD7<sup>A154</sup> and BRD9<sup>G159</sup>). In the absence of structural information on BRD7–inhibitor complexes, it was previously suggested that differences in the side-chain conformation of BRD9<sup>F163</sup> and the equivalent residue BRD7<sup>F158</sup> determine the potency and selectivity of BI-9564.<a onclick="showRef(event, 'ref16 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref18">(16,18)</a> However, comparison of BRD9 and BRD7 inhibitor complexes suggests that this residue adopts multiple conformations even in different monomers comprising the asymmetric unit. Additionally, the finding that BI-7273 is more potent and selective for BRD9 than BI-9564 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C) suggests that the hydrophobic vdW interactions observed between BRD9<sup>F163</sup> and BI-9564 are less significant.</div><div class="NLM_p">Novel cocrystal structures were also obtained for BRD7 with I-BRD9 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A), TP-472 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D) and bromosporine (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>G) as well as for BRD9 with TP-472 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). Crystal structures of BRD9 liganded with I-BRD9 and bromosporine have been determined previously,<a onclick="showRef(event, 'ref17 ref25'); return false;" href="javascript:void(0);" class="ref ref17 ref25">(17,25)</a> but we included a new data set of the BRD9–IBRD9 complex determined at higher resolution (1.5 Å) for structural comparison with BRD7 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). All three inhibitors showed the same principal interaction pattern but slightly different conformations and positioning in the respective KAc sites (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,F,I). These differences were not as pronounced as with BI-7273 and BI-9564 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E,F) and likely reflect the decreased selectivity of these compounds for BRD9 over BRD7 particularly of I-BRD9 and bromosporine (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). TP-472 undergoes a significant shift toward the critical asparagine in the KAc site of BRD9, likely strengthening H-bonding activity and explaining the observed increased selectivity for BRD9.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding modes of I-BRD9, TP-472, and bromosporine in BRD7 and BRD9. (A) Cocrystal structure of I-BRD9 in BRD7 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V17">6V17</a>) and (B) in BRD9 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1B">6V1B</a>). (C) Superposition of both structures. (D–F) Same as (A–C) for TP-472 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V16">6V16</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V14">6V14</a>). (G–I) Same as (A–C) for bromosporine (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1H">6V1H</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IGM">5IGM</a>). Except for the BRD9-bromosporine complex, all structures were determined experimentally (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Table S3). 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> and <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Figure S3 and S4</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further evaluate the SAR of BRD9 inhibitors, we attempted cocrystal structure determination with the first BRD of BRD4, a member of the evolutionary distant BET family of BRD-containing proteins. BI-7273, BI-9564, and TP-472 showed weak binding potential for the first BRD of BRD4 (BRD4-1) by DSF corresponding to <i>K</i><sub>d</sub> values above 10 μM (data not shown). Novel cocrystal structures were determined for BRD4 liganded with these inhibitors showing the same principal interactions of the respective warheads with the critical asparagine side chain in the KAc site (BRD4<sup>N140</sup>) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). However, all three inhibitors undergo significant conformational changes upon binding to BRD4, primarily caused by steric hindrance with the side chain of Trp81, which comprises the WPF shelf characteristic of all BET BRDs. The methoxyphenyl ring of BI-9564 rotates 180° to accommodate the ortho- and metamethoxy groups, while the double metaposition of the methoxy groups in BI-7273 forces the ring to rotate away from Trp81 because of steric hindrance (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B). TP-472 and bromosporine undergo the largest conformational changes upon binding to BRD4 versus BRD7 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C,D). The conformational changes of BI-7273, BI-9564, and TP-472 suggest a loss of shape complementarity with the KAc site of BRD4, resulting in a weak binding potential. In contrast, the changed binding pose of bromosporine suggests a gain in shape complementarity, reflected by a substantial increase of binding potential for BRD4-1 (<i>K</i><sub>d</sub> = 42 nM<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a>) over BRD7 (<i>K</i><sub>d</sub> = 390 nM).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding modes of BRD7/9 inhibitors in BRD4. (A) Cocrystal structure of BI-7273 with BRD4-1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1K">6V1K</a>) and superposition with the BRD7/BI-7273 complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1E">6V1E</a>). (B) Same as (A) for BI-9564 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1L">6V1L</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1F">6V1F</a>). (C) Same as (A) for TP-472 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1U">6V1U</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V16">6V16</a>). (D) Same as (A) for bromosporine (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0U">6V0U</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1H">6V1H</a>). All structures were determined experimentally. Residues of BRD4 are shown in green and of BRD7 in beige. Inhibitor liganded with BRD4 is shown in yellow and liganded with BRD7 in orange. 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> and <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Figure S5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Discovery of Dual BRD7/9-Kinase Inhibitors</h3><div class="NLM_p">Although BRD9 has an essential role in malignant transformation by modulating the functions of the BAF complex, it is not considered an oncogene. In cancer, altered BAF complexes complement driver oncogenes such as <i>PIK3CA</i>, <i>KRAS</i>, <i>c-Myc</i>, or <i>ATR</i>.<a onclick="showRef(event, 'ref8 ref15 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref8 ref15 ref26 ref27">(8,15,26,27)</a> While BRD9 inhibition alone may be insufficient to elicit robust anticancer activity, the concomitant inhibition of BRD9 and a kinase crucial for the survival of cancer cells may exert synergistic lethal effects. We explored the potential of kinase inhibitors to interact with BRD7 and BRD9 by screening a commercial library of 418 compounds against BRD9 using DSF. Three kinase inhibitors, TG003, a Clk2 inhibitor,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> sunitinib, a multitargeted tyrosine kinase inhibitor,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and PF-477736, an inhibitor of cell cycle Chk1<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> were identified as potential hits (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). These inhibitors showed small but reproducible temperature shifts against both BRD9 and BRD7 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). The highest temperature shift was 2.1 °C as compared to 13.6 °C for positive control BI-7273, suggesting relatively weak binding potential of kinase inhibitors for BRD9. ITC binding studies established that BRD7 and BRD9 interact with TG003 in the low micromolar range (<i>K</i><sub>d</sub> = 4 and 16 μM, respectively) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). Quantitative polymerase chain reaction (qPCR)-based competition assays confirmed that all three inhibitors bind to BRD7 and BRD9 with <i>K</i><sub>d</sub> values between 5 and 17 μM, except sunitinib which was significantly less active against BRD9 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Discovery of BRD7/9-kinase inhibitors. (A) Chemical structures of kinase inhibitors identified as ligands of BRD7 and BRD9. (B) DSF derivative plots for BRD7 and BRD9 in the presence of 100 μM inhibitor. (C) ITC thermograms of TG003 interaction with BRD7 and BRD9. (D) Dissociation constants determined by the qPCR assay (performed by DiscoveRx) (<i>N</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Cocrystal structures of BRD7 and BRD9 liganded with TG003 show canonical H-bonding interactions of the propanone moiety with the conserved asparagine residue (BRD7<sup>N211</sup> and BRD9<sup>N216</sup>) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B). Several vdW interactions with residues of the flanks hold the inhibitor in place of which π-stacking interactions between the ring system of TG003 and the side chain of BRD7<sup>Y217</sup> and BRD9<sup>Y222</sup> are most prominent. The only difference between the KAc sites in the unliganded and liganded states is a slight movement of this tyrosine toward the inhibitor. Superposition of the two structures revealed almost identical positioning of the inhibitor in BRD7 and BRD9 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). A cocrystal structure was also obtained for BRD9 liganded with sunitinib, which binds to the KAc site in a noncanonical manner through H-bonding of the pyrrole amine group with the main chain carbonyl oxygen of F160 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D). Thus, TG003 and sunitinib show significantly different binding modes in the KAc site of BRD9 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E). Inhibitor PF-477736 resisted cocrystallization attempts because of low aqueous solubility.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structural basis of BRD7/9 inhibition by kinase inhibitors. (A) Cocrystal structure of TG003 in BRD7 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0Q">6V0Q</a>) and (B) in BRD9 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0S">6V0S</a>). (C) Superposition of both structures. (D) Cocrystal structure of BRD9 liganded with sunitinib (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0X">6V0X</a>). (E) Superposition of TG003 (yellow) and sunitinib (orange) bound to the KAc site of BRD9. (F) Comparison of the binding mode of TG003 in the ATP site of CLK2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FYI">6FYI</a>) (left panel) and the KAc site of BRD9 (right panel). The kinase hinge region is colored in orange and other kinase residues are colored in pink. (G) Comparison of the binding mode of sunitinib in the ATP site of ITK (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MIY">3MIY</a>) (left panel) and the KAc site of BRD9 (right panel). 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> and <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Figures S3 and S4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">A comparative analysis of the binding interactions in kinases and BRDs was performed using the available cocrystal structures of TG003 in CLK2<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and sunitinib in ITK.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> TG003 and sunitinib are type I inhibitors that bind to the ATP site of kinases through H-bonding interactions with main chain atoms of the hinge region. For TG003, methoxy oxygen is the hinge-binding group in CLK2<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> while it is solvent exposed in BRD9 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>F). The propanone group which is critical for binding to the KAc site in BRD7/9 is in H-bonding distance to a lysine residue in CLK2, but this interaction is probably less significant for the kinase inhibitory activity. Thus, the functional groups primarily responsible for binding to the ATP and KAc sites are different and positioned opposite to each other. Substitution of the methoxy group likely renders the TG003 pharmacophore less active against CLK2 while maintaining activity against BRD7/9. Similarly, modifications of the propanone moiety would reduce or eliminate the binding potential for BRD7/9 while maintaining activity against kinases. Sunitinib establishes multiple H-bonding interactions with the hinge region of ITK through its indolinone and pyrrole moieties (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>G).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> As the pyrrole amine also establishes the only H-bond in the KAc site, the simultaneous inhibition of kinases and BRD7/9 is inherent to the sunitinib pharmacophore.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40127" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40127" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As with kinase inhibitors targeting the ATP site, a major challenge in the development of selective BRD inhibitors is the high similarity of the KAc site across BRD-containing proteins. Known BRD9 inhibitors also inhibit the closely related BRD7, an unwanted off-target because of its tumor suppressor properties.<a onclick="showRef(event, 'ref10 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref10 ref30 ref31">(10,30,31)</a> Previously, the structure-based design of the reported BRD9 inhibitors relied on the cocrystal structure information of BRD9 but lacked structural details on BRD7. We report the first crystal structures of BRD7 in unliganded and liganded states with inhibitors of five different chemical scaffolds. New cocrystal structures were also determined of BRD9 and BRD4. The data provide a comprehensive picture of the differences in binding interactions of the BRD9 inhibitors within and outside the BRD7/9 subfamily. Combined with biochemical data from direct binding studies, a structural basis for the potency and selectivity of diverse BRD7/9 inhibitors could be established. Although the KAc sites of BRD7 and BRD9 are composed of identical residues, small differences in the overall structures of the BRDs appear to affect the conformation and positioning of inhibitors upon binding. The observed differences in the binding pose are caused by steric hindrance with surrounding, mostly nonpolar residues. These changes may reflect loss or gain in shape complementarity of low energy conformations of an inhibitor with the KAc site, resulting in differential binding potency for BRD7 and BRD9.</div><div class="NLM_p last">Several kinase inhibitors that also inhibit BRDs have been identified for BRD4 and other BET BRDs.<a onclick="showRef(event, 'ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref34">(32−34)</a> Our discovery of first-in-class dual BRD7/9-kinase inhibitors suggests that the biological activity of kinase inhibitors in development (TG003 and PF-477736) and in clinical use (sunitinib) may in part be caused by concomitant inhibition of BRD7 and BRD9. However, the bromodomain–kinase inhibitors identified in this work are significantly more potent against the respective kinase targets than against BRD7 or BRD9. TG003 inhibits Clk1 and Clk4 with IC<sub>50</sub> values of 20 and 15 nM, respectively,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> about 250 times that of BRD7 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Sunitinib inhibits multiple tyrosine kinases with IC<sub>50</sub> values below 10 nM,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> about 1,000 times that of BRD7, and PF-477736 inhibits Chk1 with a <i>K</i><sub>i</sub> of 0.49 nM,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> about 20,000 times that of BRD7. In order to exert a meaningful synergistic action of kinase and BRD inhibition in the cell, the binding affinity of these inhibitors for BRD7 and/or BRD9 is insufficient. The structural data of this work provide a new framework for the development of more potent dual kinase-BRD7/9 inhibitors based on the chemical scaffolds identified and beyond. Recent studies on potent BRD9 inhibitors against several cancer cell lines found only weak antiproliferative effects.<a onclick="showRef(event, 'ref7 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref7 ref15 ref16">(7,15,16)</a> While inhibition of BRD9 alone may lack efficacy, the simultaneous inhibition of kinases of specific signaling events could be beneficial in anticancer therapy.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77005" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77005" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Protein Expression and Purification</h3><div class="NLM_p">The expression plasmid for the BRD9 (Uniprot ID <a href="https://Q9H8M2" class="ext-link">Q9H8M2</a>) BRD (residues 134–239) was from Addgene (plasmid 39012). DNA sequences encoding the human full length BRD7 (Uniprot ID <a href="https://Q9NPI1" class="ext-link">Q9NPI1</a>) were custom synthesized by GeneArt (Thermo Fisher Scientific), and the BRD7 BRD (residues 129–250) were cloned in-frame of a modified pET15b vector providing an N-terminal hexa-histidine tag followed by a tobacco etch virus (TEV) cleavage site. The encoded BRD proteins were expressed in the BL21 (DE3) strain of <i>Escherichia coli</i>.</div><div class="NLM_p last">Plasmids were transformed into <i>E. coli</i> cells and grown at 37 °C in the LB medium (Fisher Scientific) containing carbenicillin (0.1 mg/mL). At OD<sub>600</sub> of 0.6, the culture was cooled down to 18 °C and induced with 0.1 mM IPTG. After 18 h growth, the culture was harvested by centrifugation at 6000<i>g</i> for 25 min and stored at −80 °C. Harvested cell pellets were resuspended in 50 mM Na/K phosphate buffer (pH 7.4) containing 100 mM NaCl, 40 mM imidazole, 0.01% w/v lysozyme, and 0.01% v/v Triton X-100 at 4 °C for 1 h, subjected to sonication and the lysate was clarified by centrifugation (30,000<i>g</i> for 45 min at 4 °C). Proteins were purified by FPLC at 4 °C using columns and chromatography materials from GE Healthcare. The lysate was subjected to an immobilized Ni<sup>2+</sup> affinity chromatography column equilibrated with 50 mM Na/K phosphate buffer (pH 7.4) containing 100 mM NaCl and 40 mM imidazole using a gradient from 40 to 500 mM of imidazole. Fractions containing the target protein were combined and incubated for 2–16 h with TEV protease at 4 °C, and the cleaved His<sub>6</sub>-tag was removed by a second Ni<sup>2+</sup> affinity column. BRDs were purified to homogeneity by size exclusion chromatography using Superdex 75. The elution buffers were 50 mM HEPES/100 mM NaCl/1 mM DTT (pH 7.5) for BRD7 and 20 mM HEPES/150 mM NaCl/2 mM DTT (pH 7.5) for BRD9. BRDs eluted as monomeric proteins and were of crystallization grade quality (>95% purity as judged by SDS-PAGE). BRD-containing fractions were combined, concentrated to 10–15 mg/mL, and aliquots were flash-frozen in liquid N<sub>2</sub> and stored at −80 °C.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Isothermal Titration Calorimetry</h3><div class="NLM_p last">All experiments were conducted using an ITC200 microcalorimeter from Malvern Panalytical (Spectris PLC). BRD7 and BRD9 were buffer exchanged (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Table S1) using PD10 columns (GE life sciences) before the experiments and concentrated to 6–10 mg/mL. Experiments were carried out in the ITC buffer while stirring at 750 rpm, either in the normal titration method (BRD7) or reverse titration method (BRD9). The microsyringe (40 μL load volume) was loaded with a solution of the protein sample (150–300 μM protein in ITC buffer for BRD9) or compound sample (150–600 μM compound in ITC buffer for BRD7) and was inserted into the calorimetric cell (0.2 mL cell volume), which was filled with the solution of compounds (10–30 μM in ITC buffer) or proteins (15–50 μM in ITC buffer). All titrations were conducted using an initial control injection of 0.3 μL followed by 30 or 20 identical injections (1.58 or 2 μL per injection) with a duration of 3.16 or 4 s (per injection) and a spacing of 150 s between injections. The ratio of titrants in the titration experiments was optimized to ensure complete saturation of the titrant (protein or compounds) in the cell before the final injection thus facilitating the estimation of the baseline for each injection. The experimental data were corrected by subtracting the heats of dilution determined from independent titrations (the protein into the buffer for BRD9 or the compound into the buffer for BRD7). The collected data were analyzed using MicroCal Origin software provided with the ITC instrument to determine the enthalpies of binding (Δ<i>H</i>) and binding constants (<i>K</i><sub>B</sub>) as previously described (Wiseman et al.).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Thermodynamic parameters were calculated using the equation Δ<i>G</i> = Δ<i>H</i> – <i>T</i>Δ<i>S</i> = −<i>RT</i> ln <i>K</i><sub>B</sub>, where Δ<i>G</i>, Δ<i>H</i>, and Δ<i>S</i> are the changes in free energy, enthalpy, and entropy of binding, respectively. In all cases a single binding site model was used. Dissociation constants and thermodynamic parameters are shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C,D.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Differential Scanning Fluorimetry</h3><div class="NLM_p last">DSF experiments were carried out in Applied Biosystem QuantStudio 6 Flex (compound library screening) and a StepOnePlus (thermal-shift determination) real-time PCR system (Thermo Fisher Scientific) using sealed 384-well or 96-well format plates, assayed in quadruplicates. To obtain robust fluorescence signals, the assay was optimized regarding concentration of the protein (4 μM for screening, and 4.5 μM for thermal-shift determination) and the fluorescence dye SYPRO Orange (Invitrogen, Thermo Fisher Scientific) (5.5× for screening and 5× for thermal shift determination). For library screening, dilutions of the compound in the assay buffer (50 mM HEPES pH 7.5, 150 mM NaCl, 2 mM DTT, 1% DMSO) were prepared using a Mosquito liquid dispenser (TTP Labtech Ltd). The protein in the assay buffer including the fluorescence dye was mixed with 100 μM compound and 2% DMSO in 20 μL reaction volumes. Reaction mixtures were heated from 25 to 95 °C at 1 °C/min with fluorescence readings every 0.5 °C at 610 nm. The observed thermal shift (Δ<i>T</i><sub>m</sub>) was recorded as the difference between the <i>T</i><sub>m</sub> of the sample and DMSO reference wells.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Crystallization and X-ray Crystallography</h3><div class="NLM_p last">All crystallization experiments were performed at 18 °C. Aliquots of purified BRD7 and BRD9 were set up for crystallization using a mosquito crystallization robot (TTP Labtech). Initially, coarse screens were set up using Greiner 3-well plates at three different concentrations of the precipitant to the protein (200 + 400, 300 + 300, 400 + 200 nL) per condition. Conditions producing crystals were further optimized and scaled up for the manual set up of 2 μL drops. For cocrystallization, the compound was either premixed with protein on ice or added to the protein-reservoir solution drops to achieve final concentrations of the 1–2 mM compound and 5–10% DMSO (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Table S2). Crystals were cryoprotected using the well solution supplemented with ethylene glycol (15–30%) and flash frozen in liquid nitrogen. X-ray diffraction data were collected at −180 °C at the beamlines 22-ID and 22-BM, SER-CAT, Advanced Photon Source, Argonne National Laboratories, and in the Moffitt Chemical Biology Core using Cu Kα X-rays generated by a Rigaku Micro-Max 007-HF X-ray generator, focused by mirror optics and equipped with a Rigaku CCD Saturn 944 system. Data were reduced and scaled with XDS<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> or DIALS<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and Aimless.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> PHASER<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> was employed for molecular replacement using the PDB entry 3HME as the search model for both BRD9 and BRD7. Refinement was carried out with PHENIX<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and model building was performed with Coot.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Initial models for the small-molecule ligands were generated using MarvinSketch (ChemAxon, Cambridge, MA) with ligand restraints from the eLBOW of the PHENIX<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> suite. All structures were validated by MolProbity. Figures were prepared using PyMOL (Schrödinger, LLC). Data collection and refinement statistics are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Table S3. The coordinates and structure factors have been deposited with the PDB.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01980" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24223" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24223" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01980?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01980</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Original ITC data, crystallization conditions for protein–inhibitor complexes, X-ray data collection, refinement statistics, and electron density maps (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_001.csv">jm9b01980_si_001.csv (0.76 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf">jm9b01980_si_002.pdf (3.19 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PPA">6PPA</a> (apo), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1E">6V1E</a> (BI-7273), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1F">6V1F</a> (BI-9564), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1H">6V1H</a> (bromosporine), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V17">6V17</a> (I-BRD9), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0Q">6V0Q</a> (TG003) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V16">6V16</a> (TP-472) for BRD7. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UZF">6UZF</a> (apo), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V14">6V14</a> (TP-472), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0X">6V0X</a> (sunitinib), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1B">6V1B</a> (I-BRD9) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0S">6V0S</a> (TG003) for BRD9. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1K">6V1K</a> (BI-7273), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1L">6V1L</a> (BI-9564), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0U">6V0U</a> (bromosporine), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1U">6V1U</a> (TP-472) for BRD4-1.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01980" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88554" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88554" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ernst Schönbrunn</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Discovery, Moffitt Cancer Center, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, USF Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3589-3510" title="Orcid link">http://orcid.org/0000-0002-3589-3510</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3550475b46411b46565d5a5b5747405b5b75585a53535c41411b5a4752"><span class="__cf_email__" data-cfemail="3653445845421845555e59585444435858765b5950505f424218594451">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rezaul Md Karim</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Discovery, Moffitt Cancer Center, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, USF Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alice Chan</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Discovery, Moffitt Cancer Center, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jin-Yi Zhu</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Discovery, Moffitt Cancer Center, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>R.M.K. and A.C. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55082" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55082" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the Moffitt Chemical Biology Core for use of ITC and protein crystallography instruments (National Cancer Institute grant P30-CA076292) and the Southeast Regional Collaborative Access Team (SER-CAT, University of Georgia) for assistance with Synchrotron data collection. We thank Drs. Gunda I. Georg (University of Minnesota) and Min S. Lee (National Institute for Child Health & Human Development) for funds to initiate this project (grant HHSN275201300017C).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal domain protein</p></td></tr><tr><td class="NLM_term">BRD7</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 7</p></td></tr><tr><td class="NLM_term">BRD9</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 9</p></td></tr><tr><td class="NLM_term">Chk1</td><td class="NLM_def"><p class="first last">checkpoint kinase 1</p></td></tr><tr><td class="NLM_term">Clk2</td><td class="NLM_def"><p class="first last">Cdc2-like kinase</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">ITK</td><td class="NLM_def"><p class="first last">IL2-inducible T-cell kinase</p></td></tr><tr><td class="NLM_term">KAc</td><td class="NLM_def"><p class="first last">acetylated lysine</p></td></tr><tr><td class="NLM_term">vdW</td><td class="NLM_def"><p class="first last">van der Waals</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08280" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08280" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 40 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+Recognition+and+Large-Scale+Structural+Analysis+of+the+Human+Bromodomain+Family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lhjXMLjvbdhaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520Recognition%2520and%2520Large-Scale%2520Structural%2520Analysis%2520of%2520the%2520Human%2520Bromodomain%2520Family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-F.</span></span> <span> </span><span class="NLM_article-title">Brd4 Maintains Constitutively Active Nf-Kappab in Cancer Cells by Binding to Acetylated Rela</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">2395</span>– <span class="NLM_lpage">2404</span>, <span class="refDoi"> DOI: 10.1038/onc.2013.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fonc.2013.179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=23686307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVGktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2014&pages=2395-2404&author=Z.+Zouauthor=B.+Huangauthor=X.+Wuauthor=H.+Zhangauthor=J.+Qiauthor=J.+Bradnerauthor=S.+Nairauthor=L.-F.+Chen&title=Brd4+Maintains+Constitutively+Active+Nf-Kappab+in+Cancer+Cells+by+Binding+to+Acetylated+Rela&doi=10.1038%2Fonc.2013.179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 maintains constitutively active NF-κB in cancer cells by binding to acetylated RelA</span></div><div class="casAuthors">Zou, Z.; Huang, B.; Wu, X.; Zhang, H.; Qi, J.; Bradner, J.; Nair, S.; Chen, L.-F.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2395-2404</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acetylation of the RelA subunit of NF-κB at lysine-310 regulates the transcriptional activation of NF-κB target genes and contributes to maintaining constitutively active NF-κB in tumors.  Bromodomain-contg. factor Brd4 has been shown to bind to acetylated lysine-310 (AcLys310) and to regulate the transcriptional activity of NF-κB, but the role of this binding in maintaining constitutively active NF-κB in tumors remains elusive.  In this study, we demonstrate the structural basis for the binding of bromodomains (BDs) of bromodomain-contg. protein 4 (Brd4) to AcLys310 and identify the BD inhibitor JQ1 as an effective small mol. to block this interaction.  JQ1 suppresses TNF-α-mediated NF-κB activation and NF-κB-dependent target gene expression.  In addn., JQ1 inhibits the proliferation and transformation potential of A549 lung cancer cells and suppresses the tumorigenicity of A549 cells in severe combined immunodeficiency mice.  Furthermore, we demonstrate that depletion of Brd4 or treatment of cells with JQ1 induces the ubiquitination and degrdn. of the constitutively active nuclear form of RelA.  Our results identify a novel function of Brd4 in maintaining the persistently active form of NF-κB found in tumors, and they suggest that interference with the interaction between acetylated RelA and Brd4 could be a potential therapeutic approach for the treatment of NF-κB-driven cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreyXshjsnx27Vg90H21EOLACvtfcHk0lhjXMLjvbdhaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvVGktro%253D&md5=b2cc674e107dd84eb147594a061d2550</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fonc.2013.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2013.179%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%26aulast%3DNair%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.-F.%26atitle%3DBrd4%2520Maintains%2520Constitutively%2520Active%2520Nf-Kappab%2520in%2520Cancer%2520Cells%2520by%2520Binding%2520to%2520Acetylated%2520Rela%26jtitle%3DOncogene%26date%3D2014%26volume%3D33%26spage%3D2395%26epage%3D2404%26doi%3D10.1038%2Fonc.2013.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Functions of Bromodomain-Containing Proteins and Their Roles in Homeostasis and Cancer</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/nrm.2016.143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fnrm.2016.143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=28053347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=246-262&author=T.+Fujisawaauthor=P.+Filippakopoulos&title=Functions+of+Bromodomain-Containing+Proteins+and+Their+Roles+in+Homeostasis+and+Cancer&doi=10.1038%2Fnrm.2016.143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Functions of bromodomain-containing proteins and their roles in homeostasis and cancer</span></div><div class="casAuthors">Fujisawa, Takao; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">246-262</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Bromodomains (BRDs) are evolutionarily conserved protein-protein interaction modules that are found in a wide range of proteins with diverse catalytic and scaffolding functions and are present in most tissues.  BRDs selectively recognize and bind to acetylated Lys residues - particularly in histones - and thereby have important roles in the regulation of gene expression.  BRD-contg. proteins are frequently dysregulated in cancer, they participate in gene fusions that generate diverse, frequently oncogenic proteins, and many cancer-causing mutations have been mapped to the BRDs themselves.  Importantly, BRDs can be targeted by small-mol. inhibitors, which has stimulated many translational research projects that seek to attenuate the aberrant functions of BRD-contg. proteins in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-drLvQK1rvLVg90H21EOLACvtfcHk0lhjXMLjvbdhaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D&md5=cbccdee8423bad7a426c9cea6566151f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2016.143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2016.143%26sid%3Dliteratum%253Aachs%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DFunctions%2520of%2520Bromodomain-Containing%2520Proteins%2520and%2520Their%2520Roles%2520in%2520Homeostasis%2520and%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D246%26epage%3D262%26doi%3D10.1038%2Fnrm.2016.143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">Targeting Cancer Cells with Bet Bromodomain Inhibitors</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">a026674</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a026674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1101%2Fcshperspect.a026674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=28213432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisFKqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=a026674&author=Y.+Xuauthor=C.+R.+Vakoc&title=Targeting+Cancer+Cells+with+Bet+Bromodomain+Inhibitors&doi=10.1101%2Fcshperspect.a026674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer cells with BET bromodomain inhibitors</span></div><div class="casAuthors">Xu, Yali; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">a026674/1-a026674/18</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Cancer cells are often hypersensitive to the targeting of transcriptional regulators, which may reflect the deregulated gene expression programs that underlie malignant transformation.  One of the most prominent transcriptional vulnerabilities in human cancer to emerge in recent years is the bromodomain and extraterminal (BET) family of proteins, which are coactivators that link acetylated transcription factors and histones to the activation of RNA polymerase II.  Despite unclear mechanisms underlying the gene specificity of BET protein function, small mols. targeting these regulators preferentially suppress the transcription of cancer-promoting genes.  As a consequence, BET inhibitors elicit anticancer activity in numerous malignant contexts at doses that can be tolerated by normal tissues, a finding supported by animal studies and by phase I clin. trials in human cancer patients.  In this review, we will discuss the remarkable, and often perplexing, therapeutic effects of BET bromodomain inhibition in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgHFLVxo-LQrVg90H21EOLACvtfcHk0lh3xl-NHtDB0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisFKqtbw%253D&md5=e1bdf55b78059e868867aad8d35e737f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a026674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a026674%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DTargeting%2520Cancer%2520Cells%2520with%2520Bet%2520Bromodomain%2520Inhibitors%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2017%26volume%3D7%26spage%3Da026674%26doi%3D10.1101%2Fcshperspect.a026674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd</span></span> <span> </span><span class="NLM_article-title">Bromodomains: A New Target Class for Drug Development</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">628</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0030-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fs41573-019-0030-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=31273347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlWitbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=609-628&author=A.+G.+Cochranauthor=A.+R.+Coneryauthor=R.+J.+Sims&title=Bromodomains%3A+A+New+Target+Class+for+Drug+Development&doi=10.1038%2Fs41573-019-0030-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains: a new target class for drug development</span></div><div class="casAuthors">Cochran, Andrea G.; Conery, Andrew R.; Sims, III, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">609-628</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Less than a decade ago, it was shown that bromodomains, acetyl lysine 'reader' modules found in proteins with varied functions, were highly tractable small-mol. targets.  This is an unusual property for protein-protein or protein-peptide interaction domains, and it prompted a wave of chem. probe discovery to understand the biol. potential of new agents that targeted bromodomains.  The original examples, inhibitors of the bromodomain and extra-terminal (BET) class of bromodomains, showed enticing anti-inflammatory and anticancer activities, and several compds. have since advanced to human clin. trials.  Here, we review the current state of BET inhibitor biol. in relation to clin. development, and we discuss the next wave of bromodomain inhibitors with clin. potential in oncol. and non-oncol. indications.  The lessons learned from BET inhibitor programs should affect efforts to develop drugs that target non-BET bromodomains and other epigenetic readers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6QwxqowW9NbVg90H21EOLACvtfcHk0lh3xl-NHtDB0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlWitbnL&md5=88e56eec8edeb7c325d28156ec131ed8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0030-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0030-7%26sid%3Dliteratum%253Aachs%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26atitle%3DBromodomains%253A%2520A%2520New%2520Target%2520Class%2520for%2520Drug%2520Development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D609%26epage%3D628%26doi%3D10.1038%2Fs41573-019-0030-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mashtalir, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Avino, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zullow, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubiak, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St Pierre, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poynter, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassel, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranish, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span> <span> </span><span class="NLM_article-title">Modular Organization and Assembly of Swi/Snf Family Chromatin Remodeling Complexes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">1272</span>– <span class="NLM_lpage">1288</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.09.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1016%2Fj.cell.2018.09.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=30343899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFKjtbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2018&pages=1272-1288&author=N.+Mashtalirauthor=A.+R.+D%E2%80%99Avinoauthor=B.+C.+Michelauthor=J.+Luoauthor=J.+Panauthor=J.+E.+Ottoauthor=H.+J.+Zullowauthor=Z.+M.+McKenzieauthor=R.+L.+Kubiakauthor=R.+St+Pierreauthor=A.+M.+Valenciaauthor=S.+J.+Poynterauthor=S.+H.+Casselauthor=J.+A.+Ranishauthor=C.+Kadoch&title=Modular+Organization+and+Assembly+of+Swi%2FSnf+Family+Chromatin+Remodeling+Complexes&doi=10.1016%2Fj.cell.2018.09.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Modular organization and assembly of SWI/SNF family chromatin remodeling complexes</span></div><div class="casAuthors">Mashtalir, Nazar; D'Avino, Andrew R.; Michel, Brittany C.; Luo, Jie; Pan, Joshua; Otto, Jordan E.; Zullow, Hayley J.; McKenzie, Zachary M.; Kubiak, Rachel L.; St. Pierre, Roodolph; Valencia, Alfredo M.; Poynter, Steven J.; Cassel, Seth H.; Ranish, Jeffrey A.; Kadoch, Cigall</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1272-1288.e20</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF (mSWI/SNF) ATP-dependent chromatin remodeling complexes are multi-subunit mol. machines that play vital roles in regulating genomic architecture and are frequently disrupted in human cancer and developmental disorders.  To date, the modular organization and pathways of assembly of these chromatin regulators remain unknown, presenting a major barrier to structural and functional detn.  Here, we elucidated the architecture and assembly pathway across 3 classes of mSWI/SNF complexes [canonical BRG1/BRM-assocd. factor (BAF), polybromo-assocd. BAF (PBAF)], and newly defined ncBAF complexes-and defined the requirement of each subunit for complex formation and stability.  Using affinity purifn. of endogenous complexes from mammalian and Drosophila cells coupled with crosslinking mass spectrometry (CX-MS) and mutagenesis, we uncovered 3 distinct and evolutionarily conserved modules, their organization, and the temporal incorporation of these modules into each complete mSWI/SNF complex class.  Finally, we mapped human disease-assocd. mutations within subunits and modules, defining specific topol. regions that were affected upon subunit perturbation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0-bunBPiOqrVg90H21EOLACvtfcHk0lh3xl-NHtDB0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFKjtbnL&md5=7e5b6372e8507a346b044c12903e3d3f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.09.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.09.032%26sid%3Dliteratum%253Aachs%26aulast%3DMashtalir%26aufirst%3DN.%26aulast%3DD%25E2%2580%2599Avino%26aufirst%3DA.%2BR.%26aulast%3DMichel%26aufirst%3DB.%2BC.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DPan%26aufirst%3DJ.%26aulast%3DOtto%26aufirst%3DJ.%2BE.%26aulast%3DZullow%26aufirst%3DH.%2BJ.%26aulast%3DMcKenzie%26aufirst%3DZ.%2BM.%26aulast%3DKubiak%26aufirst%3DR.%2BL.%26aulast%3DSt%2BPierre%26aufirst%3DR.%26aulast%3DValencia%26aufirst%3DA.%2BM.%26aulast%3DPoynter%26aufirst%3DS.%2BJ.%26aulast%3DCassel%26aufirst%3DS.%2BH.%26aulast%3DRanish%26aufirst%3DJ.%2BA.%26aulast%3DKadoch%26aufirst%3DC.%26atitle%3DModular%2520Organization%2520and%2520Assembly%2520of%2520Swi%252FSnf%2520Family%2520Chromatin%2520Remodeling%2520Complexes%26jtitle%3DCell%26date%3D2018%26volume%3D175%26spage%3D1272%26epage%3D1288%26doi%3D10.1016%2Fj.cell.2018.09.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hohmann, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minder, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, J.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottschamel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">Sensitivity and Engineered Resistance of Myeloid Leukemia Cells to Brd9 Inhibition</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">679</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fnchembio.2115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=27376689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKntLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=672-679&author=A.+F.+Hohmannauthor=L.+J.+Martinauthor=J.+L.+Minderauthor=J.-S.+Roeauthor=J.+Shiauthor=S.+Steurerauthor=G.+Baderauthor=D.+McConnellauthor=M.+Pearsonauthor=T.+Gerstbergerauthor=T.+Gottschamelauthor=D.+Thompsonauthor=Y.+Suzukiauthor=M.+Koeglauthor=C.+R.+Vakoc&title=Sensitivity+and+Engineered+Resistance+of+Myeloid+Leukemia+Cells+to+Brd9+Inhibition&doi=10.1038%2Fnchembio.2115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Sensitivity and engineered resistance of myeloid leukemia cells to BRD9 inhibition</span></div><div class="casAuthors">Hohmann, Anja F.; Martin, Laetitia J.; Minder, Jessica L.; Roe, Jae-Seok; Shi, Junwei; Steurer, Steffen; Bader, Gerd; McConnell, Darryl; Pearson, Mark; Gerstberger, Thomas; Gottschamel, Teresa; Thompson, Diane; Suzuki, Yutaka; Koegl, Manfred; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">672-679</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we show that acute myeloid leukemia (AML) cells require the BRD9 subunit of the SWI-SNF chromatin-remodeling complex to sustain MYC transcription, rapid cell proliferation and a block in differentiation.  Based on these observations, we derived small-mol. inhibitors of the BRD9 bromodomain that selectively suppress the proliferation of mouse and human AML cell lines.  To establish these effects as on-target, we engineered a bromodomain-swap allele of BRD9 that retains functionality despite a radically altered bromodomain pocket.  Expression of this allele in AML cells confers resistance to the antiproliferative effects of our compd. series, thus establishing BRD9 as the relevant cellular target.  Furthermore, we used an analogous domain-swap strategy to generate an inhibitor-resistant allele of EZH2.  To our knowledge, our study provides the first evidence for a role of BRD9 in cancer and reveals a simple genetic strategy for constructing resistance alleles to demonstrate on-target activity of chem. probes in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr8rWWOXSB37Vg90H21EOLACvtfcHk0liOAH7Qb5QC5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKntLfN&md5=12c2918c7bdfacdb1c904d55bfb6afb5</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2115%26sid%3Dliteratum%253Aachs%26aulast%3DHohmann%26aufirst%3DA.%2BF.%26aulast%3DMartin%26aufirst%3DL.%2BJ.%26aulast%3DMinder%26aufirst%3DJ.%2BL.%26aulast%3DRoe%26aufirst%3DJ.-S.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DMcConnell%26aufirst%3DD.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DGottschamel%26aufirst%3DT.%26aulast%3DThompson%26aufirst%3DD.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DSensitivity%2520and%2520Engineered%2520Resistance%2520of%2520Myeloid%2520Leukemia%2520Cells%2520to%2520Brd9%2520Inhibition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D672%26epage%3D679%26doi%3D10.1038%2Fnchembio.2115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemming, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cagney, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Mierlo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltissen, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fröhling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">Targeted Degradation of Brd9 Reverses Oncogenic Gene Expression in Synovial Sarcoma</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e41305</span> <span class="refDoi"> DOI: 10.7554/elife.41305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.7554%2Felife.41305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=30431433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyqs7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=G.+L.+Brienauthor=D.+Remillardauthor=J.+Shiauthor=M.+L.+Hemmingauthor=J.+Chabonauthor=K.+Wynneauthor=E.+T.+Dillonauthor=G.+Cagneyauthor=G.+Van+Mierloauthor=M.+P.+Baltissenauthor=M.+Vermeulenauthor=J.+Qiauthor=S.+Fr%C3%B6hlingauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=C.+R.+Vakocauthor=S.+A.+Armstrong&title=Targeted+Degradation+of+Brd9+Reverses+Oncogenic+Gene+Expression+in+Synovial+Sarcoma&doi=10.7554%2Felife.41305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted degradation of BRD9 reverses oncogenic gene expression in synovial sarcoma</span></div><div class="casAuthors">Brien, Gerard L.; Remillard, David; Shi, Junwei; Hemming, Matthew L.; Chabon, Jonathon; Wynne, Kieran; Dillon, Euge Ne T.; Cagney, Gerard; Van Mierlo, Guido; Baltissen, Marijke P.; Vermeulen, Michiel; Qi, Jun; Froehling, Stefan; Gray, Nathanael S.; Bradner, James E.; Vakoc, Christopher R.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e41305/1-e41305/26</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Synovial sarcoma tumors contain a characteristic fusion protein, SS18-SSX, which drives disease development.  Targeting oncogenic fusion proteins presents an attractive therapeutic opportunity.  However, SS18-SSX has proven intractable for therapeutic intervention.  Using a domain-focused CRISPR screen we identified the bromodomain of BRD9 as a crit. functional dependency in synovial sarcoma.  BRD9 is a component of SS18-SSX contg. BAF complexes in synovial sarcoma cells; and integration of BRD9 into these complexes is crit. for cell growth.  Moreover BRD9 and SS18-SSX co-localize extensively on the synovial sarcoma genome.  Remarkably, synovial sarcoma cells are highly sensitive to a novel small mol. degrader of BRD9, while other sarcoma subtypes are unaffected.  Degrdn. of BRD9 induces downregulation of oncogenic transcriptional programs and inhibits tumor progression in vivo.  We demonstrate that BRD9 supports oncogenic mechanisms underlying the SS18-SSX fusion in synovial sarcoma and highlight targeted degrdn. of BRD9 as a potential therapeutic opportunity in this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfd6-D9YlmrrVg90H21EOLACvtfcHk0liOAH7Qb5QC5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyqs7fF&md5=cbda64457406cd0d8fbd1211678b04b3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.7554%2Felife.41305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252Felife.41305%26sid%3Dliteratum%253Aachs%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DHemming%26aufirst%3DM.%2BL.%26aulast%3DChabon%26aufirst%3DJ.%26aulast%3DWynne%26aufirst%3DK.%26aulast%3DDillon%26aufirst%3DE.%2BT.%26aulast%3DCagney%26aufirst%3DG.%26aulast%3DVan%2BMierlo%26aufirst%3DG.%26aulast%3DBaltissen%26aufirst%3DM.%2BP.%26aulast%3DVermeulen%26aufirst%3DM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DFr%25C3%25B6hling%26aufirst%3DS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DTargeted%2520Degradation%2520of%2520Brd9%2520Reverses%2520Oncogenic%2520Gene%2520Expression%2520in%2520Synovial%2520Sarcoma%26jtitle%3DElife%26date%3D2018%26volume%3D7%26doi%3D10.7554%2Felife.41305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frühwald, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerl, K.</span></span> <span> </span><span class="NLM_article-title">Brd9 Inhibition, Alone or in Combination with Cytostatic Compounds as a Therapeutic Approach in Rhabdoid Tumors</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1537</span>, <span class="refDoi"> DOI: 10.3390/ijms18071537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.3390%2Fijms18071537" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=1537&author=K.+Kr%C3%A4merauthor=N.+Morenoauthor=M.+Fr%C3%BChwaldauthor=K.+Kerl&title=Brd9+Inhibition%2C+Alone+or+in+Combination+with+Cytostatic+Compounds+as+a+Therapeutic+Approach+in+Rhabdoid+Tumors&doi=10.3390%2Fijms18071537"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3390%2Fijms18071537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18071537%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25A4mer%26aufirst%3DK.%26aulast%3DMoreno%26aufirst%3DN.%26aulast%3DFr%25C3%25BChwald%26aufirst%3DM.%26aulast%3DKerl%26aufirst%3DK.%26atitle%3DBrd9%2520Inhibition%252C%2520Alone%2520or%2520in%2520Combination%2520with%2520Cytostatic%2520Compounds%2520as%2520a%2520Therapeutic%2520Approach%2520in%2520Rhabdoid%2520Tumors%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26spage%3D1537%26doi%3D10.3390%2Fijms18071537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantovani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tocco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holstege, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerkhoven, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonkers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voorhoeve, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agami, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Sal, G.</span></span> <span> </span><span class="NLM_article-title">Brd7 Is a Candidate Tumour Suppressor Gene Required for P53 Function</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">380</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1038/ncb2038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fncb2038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=20228809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVyjurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=380-389&author=J.+Drostauthor=F.+Mantovaniauthor=F.+Toccoauthor=R.+Elkonauthor=A.+Comelauthor=H.+Holstegeauthor=R.+Kerkhovenauthor=J.+Jonkersauthor=P.+M.+Voorhoeveauthor=R.+Agamiauthor=G.+Del+Sal&title=Brd7+Is+a+Candidate+Tumour+Suppressor+Gene+Required+for+P53+Function&doi=10.1038%2Fncb2038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">BRD7 is a candidate tumour suppressor gene required for p53 function</span></div><div class="casAuthors">Drost, Jarno; Mantovani, Fiamma; Tocco, Francesca; Elkon, Ran; Comel, Anna; Holstege, Henne; Kerkhoven, Ron; Jonkers, Jos; Voorhoeve, P. Mathijs; Agami, Reuven; Del Sal, Giannino</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">380-389</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oncogene-induced senescence is a p53-dependent defense mechanism against uncontrolled proliferation.  Consequently, many human tumors harbor p53 mutations and others show a dysfunctional p53 pathway, frequently by unknown mechanisms.  Here the authors identify BRD7 (bromodomain-contg. 7) as a protein whose inhibition allows full neoplastic transformation in the presence of wild-type p53.  In human breast tumors harboring wild-type, but not mutant, p53 the BRD7 gene locus was frequently deleted and low BRD7 expression was found in a subgroup of tumors.  Functionally, BRD7 is required for efficient p53-mediated transcription of a subset of target genes.  BRD7 interacts with p53 and p300 and is recruited to target gene promoters, affecting histone acetylation, p53 acetylation and promoter activity.  Thus, BRD7 suppresses tumorigenicity by serving as a p53 cofactor required for the efficient induction of p53-dependent oncogene-induced senescence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1grCCyCF9MrVg90H21EOLACvtfcHk0ljeFT7Bp0wjhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVyjurw%253D&md5=676cb9cbc5a8d157ac85d76c42ba8007</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fncb2038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2038%26sid%3Dliteratum%253Aachs%26aulast%3DDrost%26aufirst%3DJ.%26aulast%3DMantovani%26aufirst%3DF.%26aulast%3DTocco%26aufirst%3DF.%26aulast%3DElkon%26aufirst%3DR.%26aulast%3DComel%26aufirst%3DA.%26aulast%3DHolstege%26aufirst%3DH.%26aulast%3DKerkhoven%26aufirst%3DR.%26aulast%3DJonkers%26aufirst%3DJ.%26aulast%3DVoorhoeve%26aufirst%3DP.%2BM.%26aulast%3DAgami%26aufirst%3DR.%26aulast%3DDel%2BSal%26aufirst%3DG.%26atitle%3DBrd7%2520Is%2520a%2520Candidate%2520Tumour%2520Suppressor%2520Gene%2520Required%2520for%2520P53%2520Function%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D380%26epage%3D389%26doi%3D10.1038%2Fncb2038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussenzveig, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontoura, B. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">Nucleoporin Seh1 Interacts with Olig2/Brd7 to Promote Oligodendrocyte Differentiation and Myelination</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">587</span>– <span class="NLM_lpage">601</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2019.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1016%2Fj.neuron.2019.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=30876848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVegtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2019&pages=587-601&author=Z.+Liuauthor=M.+Yanauthor=Y.+Liangauthor=M.+Liuauthor=K.+Zhangauthor=D.+Shaoauthor=R.+Jiangauthor=L.+Liauthor=C.+Wangauthor=D.+R.+Nussenzveigauthor=K.+Zhangauthor=S.+Chenauthor=C.+Zhongauthor=W.+Moauthor=B.+M.+A.+Fontouraauthor=L.+Zhang&title=Nucleoporin+Seh1+Interacts+with+Olig2%2FBrd7+to+Promote+Oligodendrocyte+Differentiation+and+Myelination&doi=10.1016%2Fj.neuron.2019.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleoporin Seh1 Interacts with Olig2/Brd7 to Promote Oligodendrocyte Differentiation and Myelination</span></div><div class="casAuthors">Liu, Zhixiong; Yan, Minbiao; Liang, Yaoji; Liu, Min; Zhang, Kun; Shao, Dandan; Jiang, Rencai; Li, Li; Wang, Chaomeng; Nussenzveig, Daniel R.; Zhang, Kunkun; Chen, Shaoxuan; Zhong, Chuanqi; Mo, Wei; Fontoura, Beatriz M. A.; Zhang, Liang</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">587-601.e7</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Nucleoporins (Nups) are involved in neural development, and alterations in Nup genes are linked to human neurol. diseases.  However, physiol. functions of specific Nups and the underlying mechanisms involved in these processes remain elusive.  Here, we show that tissue-specific depletion of the nucleoporin Seh1 causes dramatic myelination defects in the CNS.  Although proliferation is not altered in Seh1-deficient oligodendrocyte progenitor cells (OPCs), they fail to differentiate into mature oligodendrocytes, which impairs myelin prodn. and remyelination after demyelinating injury.  Genome-wide analyses show that Seh1 regulates a core myelinogenic regulatory network and establishes an accessible chromatin landscape.  Mechanistically, Seh1 regulates OPCs differentiation by assembling Olig2 and Brd7 into a transcription complex at nuclear periphery.  Together, our results reveal that Seh1 is required for oligodendrocyte differentiation and myelination by promoting assembly of an Olig2-dependent transcription complex and define a nucleoporin as a key player in the CNS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoem6E0dk5Q-rVg90H21EOLACvtfcHk0ljeFT7Bp0wjhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVegtbg%253D&md5=9020b496dc7dac04e003d67f72e7d35e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2019.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2019.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DShao%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DNussenzveig%26aufirst%3DD.%2BR.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DC.%26aulast%3DMo%26aufirst%3DW.%26aulast%3DFontoura%26aufirst%3DB.%2BM.%2BA.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DNucleoporin%2520Seh1%2520Interacts%2520with%2520Olig2%252FBrd7%2520to%2520Promote%2520Oligodendrocyte%2520Differentiation%2520and%2520Myelination%26jtitle%3DNeuron%26date%3D2019%26volume%3D102%26spage%3D587%26epage%3D601%26doi%3D10.1016%2Fj.neuron.2019.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span> <span> </span><span class="NLM_article-title">Knockout of Brd7 Results in Impaired Spermatogenesis and Male Infertility</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">21776</span>, <span class="refDoi"> DOI: 10.1038/srep21776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fsrep21776" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=26878912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC28XislOqtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=21776&author=H.+Wangauthor=R.+Zhaoauthor=C.+Guoauthor=S.+Jiangauthor=J.+Yangauthor=Y.+Xuauthor=Y.+Liuauthor=L.+Fanauthor=W.+Xiongauthor=J.+Maauthor=S.+Pengauthor=Z.+Zengauthor=Y.+Zhouauthor=X.+Liauthor=Z.+Liauthor=X.+Liauthor=D.+C.+Schmittauthor=M.+Tanauthor=G.+Liauthor=M.+Zhou&title=Knockout+of+Brd7+Results+in+Impaired+Spermatogenesis+and+Male+Infertility&doi=10.1038%2Fsrep21776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Knockout of BRD7 results in impaired spermatogenesis and male infertility</span></div><div class="casAuthors">Wang, Heran; Zhao, Ran; Guo, Chi; Jiang, Shihe; Yang, Jing; Xu, Yang; Liu, Yukun; Fan, Liqing; Xiong, Wei; Ma, Jian; Peng, Shuping; Zeng, Zhaoyang; Zhou, Yanhong; Li, Xiayu; Li, Zheng; Li, Xiaoling; Schmitt, David C.; Tan, Ming; Li, Guiyuan; Zhou, Ming</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">21776</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">BRD7 was originally identified as a novel bromodomain gene and a potential transcriptional factor.  BRD7 was found to be extensively expressed in multiple mouse tissues but was highly expressed in the testis.  Furthermore, BRD7 was located in germ cells during multiple stages of spermatogenesis, ranging from the pachytene to the round spermatid stage.  Homozygous knockout of BRD7 (BRD7-/-) resulted in complete male infertility and spermatogenesis defects, including deformed acrosomal formation, degenerative elongating spermatids and irregular head morphol. in postmeiotic germ cells in the seminiferous epithelium, which led to the complete arrest of spermatogenesis at step 13.  Moreover, a high ratio of apoptosis was detd. by TUNEL anal., which was supported by high levels of the apoptosis markers annexin V and p53 in knockout testes.  Increased expression of the DNA damage maker λH2AX was also found in BRD7-/- mice, whereas DNA damage repair genes were down-regulated.  Furthermore, no or lower expression of BRD7 was detected in the testes of azoospermia patients exhibiting spermatogenesis arrest than that in control group.  These data demonstrate that BRD7 is involved in male infertility and spermatogenesis in mice, and BRD7 defect might be assocd. with the occurrence and development of human azoospermia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmHUItFrwX3rVg90H21EOLACvtfcHk0ljeFT7Bp0wjhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XislOqtrY%253D&md5=6979ca806bfbf56c27fda6c325710cb2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsrep21776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep21776%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DXiong%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DSchmitt%26aufirst%3DD.%2BC.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DM.%26atitle%3DKnockout%2520of%2520Brd7%2520Results%2520in%2520Impaired%2520Spermatogenesis%2520and%2520Male%2520Infertility%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D21776%26doi%3D10.1038%2Fsrep21776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranish, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">Proteomic and Bioinformatic Analysis of Mammalian Swi/Snf Complexes Identifies Extensive Roles in Human Malignancy</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">592</span>– <span class="NLM_lpage">601</span>, <span class="refDoi"> DOI: 10.1038/ng.2628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fng.2628" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=23644491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=592-601&author=C.+Kadochauthor=D.+C.+Hargreavesauthor=C.+Hodgesauthor=L.+Eliasauthor=L.+Hoauthor=J.+Ranishauthor=G.+R.+Crabtree&title=Proteomic+and+Bioinformatic+Analysis+of+Mammalian+Swi%2FSnf+Complexes+Identifies+Extensive+Roles+in+Human+Malignancy&doi=10.1038%2Fng.2628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy</span></div><div class="casAuthors">Kadoch, Cigall; Hargreaves, Diana C.; Hodges, Courtney; Elias, Laura; Ho, Lena; Ranish, Jeff; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">592-601</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Subunits of mammalian SWI/SNF (mSWI/SNF or BAF) complexes have recently been implicated as tumor suppressors in human malignancies.  To understand the full extent of their involvement, we conducted a proteomic anal. of endogenous mSWI/SNF complexes, which identified several new dedicated, stable subunits not found in yeast SWI/SNF complexes, including BCL7A, BCL7B and BCL7C, BCL11A and BCL11B, BRD9 and SS18.  Incorporating these new members, we detd. mSWI/SNF subunit mutation frequency in exome and whole-genome sequencing studies of primary human tumors.  Notably, mSWI/SNF subunits are mutated in 19.6% of all human tumors reported in 44 studies.  Our anal. suggests that specific subunits protect against cancer in specific tissues.  In addn., mutations affecting more than one subunit, defined here as compd. heterozygosity, are prevalent in certain cancers.  Our studies demonstrate that mSWI/SNF is the most frequently mutated chromatin-regulatory complex (CRC) in human cancer, exhibiting a broad mutation pattern, similar to that of TP53.  Thus, proper functioning of polymorphic BAF complexes may constitute a major mechanism of tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP873DCpn7JrVg90H21EOLACvtfcHk0liA0gON1iGgWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D&md5=0f49cf3c59e7bc04b74970fe656b1e17</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fng.2628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2628%26sid%3Dliteratum%253Aachs%26aulast%3DKadoch%26aufirst%3DC.%26aulast%3DHargreaves%26aufirst%3DD.%2BC.%26aulast%3DHodges%26aufirst%3DC.%26aulast%3DElias%26aufirst%3DL.%26aulast%3DHo%26aufirst%3DL.%26aulast%3DRanish%26aufirst%3DJ.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DProteomic%2520and%2520Bioinformatic%2520Analysis%2520of%2520Mammalian%2520Swi%252FSnf%2520Complexes%2520Identifies%2520Extensive%2520Roles%2520in%2520Human%2520Malignancy%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D592%26epage%3D601%26doi%3D10.1038%2Fng.2628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieira, L. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singleton, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baronio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span> <span> </span><span class="NLM_article-title">Lp99: Discovery and Synthesis of the First Selective Brd7/9 Bromodomain Inhibitor</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6217</span>– <span class="NLM_lpage">6221</span>, <span class="refDoi"> DOI: 10.1002/anie.201501394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1002%2Fanie.201501394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlGqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=6217-6221&author=P.+G.+K.+Clarkauthor=L.+C.+C.+Vieiraauthor=C.+Tallantauthor=O.+Fedorovauthor=D.+C.+Singletonauthor=C.+M.+Rogersauthor=O.+P.+Monteiroauthor=J.+M.+Bennettauthor=R.+Baronioauthor=S.+M%C3%BCllerauthor=D.+L.+Danielsauthor=J.+M%C3%A9ndezauthor=S.+Knappauthor=P.+E.+Brennanauthor=D.+J.+Dixon&title=Lp99%3A+Discovery+and+Synthesis+of+the+First+Selective+Brd7%2F9+Bromodomain+Inhibitor&doi=10.1002%2Fanie.201501394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">LP99: Discovery and Synthesis of the First Selective BRD7/9 Bromodomain Inhibitor</span></div><div class="casAuthors">Clark, Peter G. K.; Vieira, Lucas C. C.; Tallant, Cynthia; Fedorov, Oleg; Singleton, Dean C.; Rogers, Catherine M.; Monteiro, Octovia P.; Bennett, James M.; Baronio, Roberta; Mueller, Susanne; Daniels, Danette L.; Mendez, Jacqui; Knapp, Stefan; Brennan, Paul E.; Dixon, Darren J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6217-6221</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. proteins BRD9 and BRD7 are part of the human SWI/SNF chromatin-remodeling complexes BAF and PBAF.  To date, no selective inhibitor for BRD7/9 has been reported despite its potential value as a biol. tool or as a lead for future therapeutics.  The quinolone-fused lactam LP99 (I) is now reported as the first potent and selective inhibitor of the BRD7 and BRD9 bromodomains.  Development of LP99 from a fragment hit was expedited through balancing structure-based inhibitor design and biophys. characterization against tractable chem. synthesis: Complexity-building nitro-Mannich/lactamization cascade processes allowed for early structure-activity relationship studies whereas an enantioselective organocatalytic nitro-Mannich reaction enabled the synthesis of the lead scaffold in enantioenriched form and on scale.  This epigenetic probe was shown to inhibit the assocn. of BRD7 and BRD9 to acetylated histones in vitro and in cells.  Moreover, LP99 was used to demonstrate that BRD7/9 plays a role in regulating pro-inflammatory cytokine secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplfOmbNMpdPbVg90H21EOLACvtfcHk0liA0gON1iGgWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlGqsrg%253D&md5=9dee2b352045f7ae5efcbdc83f240a3f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fanie.201501394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201501394%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DVieira%26aufirst%3DL.%2BC.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DSingleton%26aufirst%3DD.%2BC.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DBaronio%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26atitle%3DLp99%253A%2520Discovery%2520and%2520Synthesis%2520of%2520the%2520First%2520Selective%2520Brd7%252F9%2520Bromodomain%2520Inhibitor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D6217%26epage%3D6221%26doi%3D10.1002%2Fanie.201501394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vartanian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokoe, D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Bromodomain-Containing Protein 9 for the Prevention of Epigenetically-Defined Drug Resistance</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3534</span>– <span class="NLM_lpage">3541</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.05.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1016%2Fj.bmcl.2017.05.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=28606761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVSjtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=3534-3541&author=T.+D.+Crawfordauthor=S.+Vartanianauthor=A.+C%C3%B4t%C3%A9author=S.+Bellonauthor=M.+Duplessisauthor=E.+M.+Flynnauthor=M.+Hewittauthor=H.-R.+Huangauthor=J.+R.+Kieferauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+A.+Romeroauthor=P.+Sandyauthor=Y.+Tangauthor=A.+M.+Taylorauthor=V.+Tsuiauthor=J.+Wangauthor=S.+Wangauthor=L.+Zawadzkeauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=A.+G.+Cochranauthor=D.+Stokoe&title=Inhibition+of+Bromodomain-Containing+Protein+9+for+the+Prevention+of+Epigenetically-Defined+Drug+Resistance&doi=10.1016%2Fj.bmcl.2017.05.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of bromodomain-containing protein 9 for the prevention of epigenetically-defined drug resistance</span></div><div class="casAuthors">Crawford, Terry D.; Vartanian, Steffan; Cote, Alexandre; Bellon, Steve; Duplessis, Martin; Flynn, E. Megan; Hewitt, Michael; Huang, Hon-Ren; Kiefer, James R.; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Romero, F. Anthony; Sandy, Peter; Tang, Yong; Taylor, Alexander M.; Tsui, Vickie; Wang, Jian; Wang, Shumei; Zawadzke, Laura; Albrecht, Brian K.; Magnuson, Steven R.; Cochran, Andrea G.; Stokoe, David</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3534-3541</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 9 (BRD9), an epigenetic "reader" of acetylated lysines on post-translationally modified histone proteins, is upregulated in multiple cancer cell lines.  To assess the functional role of BRD9 in cancer cell lines, the authors identified a small-mol. inhibitor of the BRD9 bromodomain.  Starting from a pyrrolopyridone lead, the authors used structure-based drug design to identify a potent and highly selective in vitro tool compd. 11, (GNE-375).  While this compd. showed minimal effects in cell viability or gene expression assays, it showed remarkable potency in preventing the emergence of a drug tolerant population in EGFR mutant PC9 cells treated with EGFR inhibitors.  Such tolerance has been linked to an altered epigenetic state, and 11 decreased BRD9 binding to chromatin, and this was assocd. with decreased expression of ALDH1A1, a gene previously shown to be important in drug tolerance.  BRD9 inhibitors may therefore show utility in preventing epigenetically-defined drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWdETG4wZY9rVg90H21EOLACvtfcHk0liA0gON1iGgWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVSjtb%252FO&md5=f489e702cfb41cfce9d3d570bfde87ed</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.05.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.05.063%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DVartanian%26aufirst%3DS.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHewitt%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DStokoe%26aufirst%3DD.%26atitle%3DInhibition%2520of%2520Bromodomain-Containing%2520Protein%25209%2520for%2520the%2520Prevention%2520of%2520Epigenetically-Defined%2520Drug%2520Resistance%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D3534%26epage%3D3541%26doi%3D10.1016%2Fj.bmcl.2017.05.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockcroft, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiegen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohmann, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knesl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornigg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaaf, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of an in Vivo Active Selective Brd9 Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4462</span>– <span class="NLM_lpage">4475</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01865</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01865" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjt1Sit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4462-4475&author=L.+J.+Martinauthor=M.+Koeglauthor=G.+Baderauthor=X.-L.+Cockcroftauthor=O.+Fedorovauthor=D.+Fiegenauthor=T.+Gerstbergerauthor=M.+H.+Hofmannauthor=A.+F.+Hohmannauthor=D.+Kesslerauthor=S.+Knappauthor=P.+Kneslauthor=S.+Korniggauthor=S.+M%C3%BCllerauthor=H.+Narauthor=C.+Rogersauthor=K.+Rumpelauthor=O.+Schaafauthor=S.+Steurerauthor=C.+Tallantauthor=C.+R.+Vakocauthor=M.+Zeebauthor=A.+Zoephelauthor=M.+Pearsonauthor=G.+Boehmeltauthor=D.+McConnell&title=Structure-Based+Design+of+an+in+Vivo+Active+Selective+Brd9+Inhibitor&doi=10.1021%2Facs.jmedchem.5b01865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of an in Vivo Active Selective BRD9 Inhibitor</span></div><div class="casAuthors">Martin, Laetitia J.; Koegl, Manfred; Bader, Gerd; Cockcroft, Xiao-Ling; Fedorov, Oleg; Fiegen, Dennis; Gerstberger, Thomas; Hofmann, Marco H.; Hohmann, Anja F.; Kessler, Dirk; Knapp, Stefan; Knesl, Petr; Kornigg, Stefan; Mueller, Susanne; Nar, Herbert; Rogers, Catherine; Rumpel, Klaus; Schaaf, Otmar; Steurer, Steffen; Tallant, Cynthia; Vakoc, Christopher R.; Zeeb, Markus; Zoephel, Andreas; Pearson, Mark; Boehmelt, Guido; McConnell, Darryl</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4462-4475</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Components of the chromatin remodelling switch/sucrose nonfermentable (SWI/SNF) complex are recurrently mutated in tumors, suggesting that altering the activity of the complex plays a role in oncogenesis.  However, the role that the individual subunits play in this process is not clear.  We set out to develop an inhibitor compd. targeting the bromodomain of BRD9 in order to evaluate its function within the SWI/SNF complex.  Here, we present the discovery and development of a potent and selective BRD9 bromodomain inhibitor series based on a new pyridinone-like scaffold.  Crystallog. information on the inhibitors bound to BRD9 guided their development with respect to potency for BRD9 and selectivity against BRD4.  These compds. modulate BRD9 bromodomain cellular function and display antitumor activity in an AML xenograft model.  Two chem. probes, BI-7273 (1) and BI-9564 (2), were identified that should prove to be useful in further exploring BRD9 bromodomain biol. in both in vitro and in vivo settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWHs4hA8bijbVg90H21EOLACvtfcHk0li3zfevX6Hpzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjt1Sit7w%253D&md5=b1f4394a7a410273c800b65ee9697c7e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01865%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%2BJ.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DCockcroft%26aufirst%3DX.-L.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFiegen%26aufirst%3DD.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DHofmann%26aufirst%3DM.%2BH.%26aulast%3DHohmann%26aufirst%3DA.%2BF.%26aulast%3DKessler%26aufirst%3DD.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKnesl%26aufirst%3DP.%26aulast%3DKornigg%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DNar%26aufirst%3DH.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DSchaaf%26aufirst%3DO.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DMcConnell%26aufirst%3DD.%26atitle%3DStructure-Based%2520Design%2520of%2520an%2520in%2520Vivo%2520Active%2520Selective%2520Brd9%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4462%26epage%3D4475%26doi%3D10.1021%2Facs.jmedchem.5b01865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bit, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molnar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Discovery of I-Brd9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1439</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00256</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00256" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1425-1439&author=N.+H.+Theodoulouauthor=P.+Bamboroughauthor=A.+J.+Bannisterauthor=I.+Becherauthor=R.+A.+Bitauthor=K.+H.+Cheauthor=C.-w.+Chungauthor=A.+Dittmannauthor=G.+Drewesauthor=D.+H.+Drewryauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Leveridgeauthor=M.+Lindonauthor=A.-M.+Michonauthor=J.+Molnarauthor=S.+C.+Robsonauthor=N.+C.+O.+Tomkinsonauthor=T.+Kouzaridesauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Discovery+of+I-Brd9%2C+a+Selective+Cell+Active+Chemical+Probe+for+Bromodomain+Containing+Protein+9+Inhibition&doi=10.1021%2Facs.jmedchem.5b00256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of I-BRD9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span></div><div class="casAuthors">Theodoulou, Natalie H.; Bamborough, Paul; Bannister, Andrew J.; Becher, Isabelle; Bit, Rino A.; Che, Ka Hing; Chung, Chun-wa; Dittmann, Antje; Drewes, Gerard; Drewry, David H.; Gordon, Laurie; Grandi, Paola; Leveridge, Melanie; Lindon, Matthew; Michon, Anne-Marie; Molnar, Judit; Robson, Samuel C.; Tomkinson, Nicholas C. O.; Kouzarides, Tony; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1425-1439</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acetylation of histone lysine residues is one of the most well-studied post-translational modifications of chromatin, selectively recognized by bromodomain "reader" modules.  Inhibitors of the bromodomain and extra terminal domain (BET) family of bromodomains have shown profound anticancer and anti-inflammatory properties, generating much interest in targeting other bromodomain-contg. proteins for disease treatment.  Herein, we report the discovery of I-BRD9, the first selective cellular chem. probe for bromodomain-contg. protein 9 (BRD9).  I-BRD9 was identified through structure-based design, leading to greater than 700-fold selectivity over the BET family and 200-fold over the highly homologous bromodomain-contg. protein 7 (BRD7).  I-BRD9 was used to identify genes regulated by BRD9 in Kasumi-1 cells involved in oncol. and immune response pathways and to the best of our knowledge, represents the first selective tool compd. available to elucidate the cellular phenotype of BRD9 bromodomain inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVmjhksIDyTrVg90H21EOLACvtfcHk0li3zfevX6Hpzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D&md5=4e8f6aa664797fe54772b64c81cf2a83</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00256%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DBecher%26aufirst%3DI.%26aulast%3DBit%26aufirst%3DR.%2BA.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMolnar%26aufirst%3DJ.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DDiscovery%2520of%2520I-Brd9%252C%2520a%2520Selective%2520Cell%2520Active%2520Chemical%2520Probe%2520for%2520Bromodomain%2520Containing%2520Protein%25209%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1425%26epage%3D1439%26doi%3D10.1021%2Facs.jmedchem.5b00256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karim, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span> <span> </span><span class="NLM_article-title">An Advanced Tool to Interrogate Brd9</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4459</span>– <span class="NLM_lpage">4461</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00550</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00550" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC28XmslWgs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4459-4461&author=R.+M.+Karimauthor=E.+Sch%C3%B6nbrunn&title=An+Advanced+Tool+to+Interrogate+Brd9&doi=10.1021%2Facs.jmedchem.6b00550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">An Advanced Tool To Interrogate BRD9</span></div><div class="casAuthors">Karim, Rezaul M.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4459-4461</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective inhibitors of bromodomain-contg. protein 9 (BRD9) may have therapeutic potential in the treatment of human malignancies and inflammatory diseases.  A selective small mol. inhibitor that is well tolerated and has proper pharmacokinetic properties is required to explore the function of BRD9 in diseases.  BI-9564 (2) is a cell permeable and noncytotoxic BRD9 inhibitor provided to the scientific community to explore BRD9 biol. and det. its potential as a drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRF7iQqoEDOLVg90H21EOLACvtfcHk0ljIOrPmzkw1rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmslWgs7g%253D&md5=5f4340cb6f20c499dc6408d2ba84bf79</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00550%26sid%3Dliteratum%253Aachs%26aulast%3DKarim%26aufirst%3DR.%2BM.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DAn%2520Advanced%2520Tool%2520to%2520Interrogate%2520Brd9%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4459%26epage%3D4461%26doi%3D10.1021%2Facs.jmedchem.6b00550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wühr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Degradation of the Baf Complex Factor Brd9 by Heterobifunctional Ligands</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5743</span>, <span class="refDoi"> DOI: 10.1002/anie.201611281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1002%2Fanie.201611281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=5738-5743&author=D.+Remillardauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=G.+L.+Brienauthor=M.+Sonnettauthor=H.-S.+Seoauthor=S.+Dastjerdiauthor=M.+W%C3%BChrauthor=S.+Dhe-Paganonauthor=S.+A.+Armstrongauthor=J.+E.+Bradner&title=Degradation+of+the+Baf+Complex+Factor+Brd9+by+Heterobifunctional+Ligands&doi=10.1002%2Fanie.201611281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span></div><div class="casAuthors">Remillard, David; Buckley, Dennis L.; Paulk, Joshiawa; Brien, Gerard L.; Sonnett, Matthew; Seo, Hyuk-Soo; Dastierdi, Shiva; Wuehr, Martin; Dhe-Paganon, Sirano; Armstrong, Scott A.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5738-5743</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. protein BRD9, a subunit of the human BAF (SWI/SNF) nucleosome remodeling complex, has emerged as an attractive therapeutic target in cancer.  Despite the development of chem. probes targeting the BRD9 bromodomain, there is a limited understanding of BRD9 function beyond acetyl-lysine recognition.  We have therefore created the first BRD9-directed chem. degraders, through iterative design and testing of heterobifunctional ligands that bridge the BRD9 bromodomain and the cereblon E3 ubiquitin ligase complex.  Degraders of BRD9 exhibit markedly enhanced potency compared to parental ligands (10- to 100-fold).  Parallel study of degraders with divergent BRD9-binding chemotypes in models of acute myeloid leukemia resolves bromodomain polypharmacol. in this emerging drug class.  Together, these findings reveal the tractability of non-BET bromodomain contg. proteins to chem. degrdn., and highlight lead compd. dBRD9 as a tool for the study of BRD9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ9FU2Ipaq27Vg90H21EOLACvtfcHk0ljIOrPmzkw1rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D&md5=2eb6562cc9d976c7ae6070c9a419e9ea</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611281%26sid%3Dliteratum%253Aachs%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DSonnett%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DW%25C3%25BChr%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDegradation%2520of%2520the%2520Baf%2520Complex%2520Factor%2520Brd9%2520by%2520Heterobifunctional%2520Ligands%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5738%26epage%3D5743%26doi%3D10.1002%2Fanie.201611281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoppi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riching, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spallarossa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (Protacs) Identify Vz185 as a Potent, Fast, and Selective Von Hippel-Lindau (Vhl) Based Dual Degrader Probe of Brd9 and Brd7</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">699</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=699-726&author=V.+Zoppiauthor=S.+J.+Hughesauthor=C.+Maniaciauthor=A.+Testaauthor=T.+Gmaschitzauthor=C.+Wieshoferauthor=M.+Koeglauthor=K.+M.+Richingauthor=D.+L.+Danielsauthor=A.+Spallarossaauthor=A.+Ciulli&title=Iterative+Design+and+Optimization+of+Initially+Inactive+Proteolysis+Targeting+Chimeras+%28Protacs%29+Identify+Vz185+as+a+Potent%2C+Fast%2C+and+Selective+Von+Hippel-Lindau+%28Vhl%29+Based+Dual+Degrader+Probe+of+Brd9+and+Brd7&doi=10.1021%2Facs.jmedchem.8b01413"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (PROTACs) Identify VZ185 as a Potent, Fast, and Selective von Hippel-Lindau (VHL) Based Dual Degrader Probe of BRD9 and BRD7</span></div><div class="casAuthors">Zoppi, Vittoria; Hughes, Scott J.; Maniaci, Chiara; Testa, Andrea; Gmaschitz, Teresa; Wieshofer, Corinna; Koegl, Manfred; Riching, Kristin M.; Daniels, Danette L.; Spallarossa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">699-726</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Developing PROTACs to redirect the ubiquitination activity of E3 ligases and potently degrade a target protein within cells can be a lengthy and unpredictable process, and it remains unclear whether any combination of E3 and target might be productive for degrdn.  The authors describe a probe-quality degrader for a ligase-target pair deemed unsuitable: the von Hippel-Lindau (VHL) and BRD9, a bromodomain-contg. subunit of the SWI/SNF chromatin remodeling complex BAF.  VHL-based degraders could be optimized from suboptimal compds. in two rounds by systematically varying conjugation patterns and linkers and monitoring cellular degrdn. activities, kinetic profiles, and ubiquitination, as well as ternary complex formation thermodn.  The emerged structure-activity relations guided the discovery of VZ185, a potent, fast, and selective degrader of BRD9 and of its close homolog BRD7.  The authors' findings qualify a new chem. tool for BRD7/9 knockdown and provide a roadmap for PROTAC development against seemingly incompatible target-ligase combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiPnajotKvqLVg90H21EOLACvtfcHk0lgvEvhwW0lI4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN&md5=be7938c26807a2535b7f9b7de8ebfbb0</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01413%26sid%3Dliteratum%253Aachs%26aulast%3DZoppi%26aufirst%3DV.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRiching%26aufirst%3DK.%2BM.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSpallarossa%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DIterative%2520Design%2520and%2520Optimization%2520of%2520Initially%2520Inactive%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528Protacs%2529%2520Identify%2520Vz185%2520as%2520a%2520Potent%252C%2520Fast%252C%2520and%2520Selective%2520Von%2520Hippel-Lindau%2520%2528Vhl%2529%2520Based%2520Dual%2520Degrader%2520Probe%2520of%2520Brd9%2520and%2520Brd7%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D699%26epage%3D726%26doi%3D10.1021%2Facs.jmedchem.8b01413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muraki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohkawara, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosoya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onogi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koizumi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koizumi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sumi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yomoda, J.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuichi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibuya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krainer, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagiwara, M.</span></span> <span> </span><span class="NLM_article-title">Manipulation of Alternative Splicing by a Newly Developed Inhibitor of Clks</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">24246</span>– <span class="NLM_lpage">24254</span>, <span class="refDoi"> DOI: 10.1074/jbc.m314298200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1074%2Fjbc.M314298200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=15010457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksV2isbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=24246-24254&author=M.+Murakiauthor=B.+Ohkawaraauthor=T.+Hosoyaauthor=H.+Onogiauthor=J.+Koizumiauthor=T.+Koizumiauthor=K.+Sumiauthor=J.-i.+Yomodaauthor=M.+V.+Murrayauthor=H.+Kimuraauthor=K.+Furuichiauthor=H.+Shibuyaauthor=A.+R.+Krainerauthor=M.+Suzukiauthor=M.+Hagiwara&title=Manipulation+of+Alternative+Splicing+by+a+Newly+Developed+Inhibitor+of+Clks&doi=10.1074%2Fjbc.m314298200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Manipulation of alternative splicing by a newly developed inhibitor of Clks</span></div><div class="casAuthors">Muraki, Michiko; Ohkawara, Bisei; Hosoya, Takamitsu; Onogi, Hiroshi; Koizumi, Jun; Koizumi, Tomonobu; Sumi, Kengo; Yomoda, Jun-ichiro; Murray, Michael V.; Kimura, Hiroshi; Furuichi, Kiyoshi; Shibuya, Hiroshi; Krainer, Adrian R.; Suzuki, Masaaki; Hagiwara, Masatoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">24246-24254</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The regulation of splice site usage provides a versatile mechanism for controlling gene expression and for the generation of proteome diversity, playing an essential role in many biol. processes.  The importance of alternative splicing is further illustrated by the increasing no. of human diseases that have been attributed to mis-splicing events.  Appropriate spatial and temporal generation of splicing variants demands that alternative splicing be subjected to extensive regulation, similar to transcriptional control.  The Clk (Cdc2-like kinase) family has been implicated in splicing control and consists of at least four members.  Through extensive screening of a chem. library, we found that a benzothiazole compd., TG003, had a potent inhibitory effect on the activity of Clk1/Sty.  TG003 inhibited SF2/ASF-dependent splicing of β-globin pre-mRNA in vitro by suppression of Clk-mediated phosphorylation.  This drug also suppressed serine/arginine-rich protein phosphorylation, dissocn. of nuclear speckles, and Clk1/Sty-dependent alternative splicing in mammalian cells.  Consistently, administration of TG003 rescued the embryonic defects induced by excessive Clk activity in Xenopus.  Thus, TG003, a novel inhibitor of Clk family will be a valuable tool to dissect the regulatory mechanisms involving serine/arginine-rich protein phosphorylation signaling pathways in vivo, and may be applicable for the therapeutic manipulation of abnormal splicing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzT17psA1H8LVg90H21EOLACvtfcHk0lgvEvhwW0lI4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksV2isbg%253D&md5=6326340869863dca70799137ba67e3d8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M314298200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M314298200%26sid%3Dliteratum%253Aachs%26aulast%3DMuraki%26aufirst%3DM.%26aulast%3DOhkawara%26aufirst%3DB.%26aulast%3DHosoya%26aufirst%3DT.%26aulast%3DOnogi%26aufirst%3DH.%26aulast%3DKoizumi%26aufirst%3DJ.%26aulast%3DKoizumi%26aufirst%3DT.%26aulast%3DSumi%26aufirst%3DK.%26aulast%3DYomoda%26aufirst%3DJ.-i.%26aulast%3DMurray%26aufirst%3DM.%2BV.%26aulast%3DKimura%26aufirst%3DH.%26aulast%3DFuruichi%26aufirst%3DK.%26aulast%3DShibuya%26aufirst%3DH.%26aulast%3DKrainer%26aufirst%3DA.%2BR.%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DHagiwara%26aufirst%3DM.%26atitle%3DManipulation%2520of%2520Alternative%2520Splicing%2520by%2520a%2520Newly%2520Developed%2520Inhibitor%2520of%2520Clks%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D24246%26epage%3D24254%26doi%3D10.1074%2Fjbc.m314298200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirazian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nematalla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sistla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of 5-[5-Fluoro-2-Oxo-1,2- Dihydroindol-(3z)-Ylidenemethyl]-2,4- Dimethyl-1h-Pyrrole-3-Carboxylic Acid (2-Diethylaminoethyl)Amide, a Novel Tyrosine Kinase Inhibitor Targeting Vascular Endothelial and Platelet-Derived Growth Factor Receptor Tyrosine Kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1116</span>– <span class="NLM_lpage">1119</span>, <span class="refDoi"> DOI: 10.1021/jm0204183</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0204183" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhslSlsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=1116-1119&author=L.+Sunauthor=C.+Liangauthor=S.+Shirazianauthor=Y.+Zhouauthor=T.+Millerauthor=J.+Cuiauthor=J.+Y.+Fukudaauthor=J.-Y.+Chuauthor=A.+Nematallaauthor=X.+Wangauthor=H.+Chenauthor=A.+Sistlaauthor=T.+C.+Luuauthor=F.+Tangauthor=J.+Weiauthor=C.+Tang&title=Discovery+of+5-%5B5-Fluoro-2-Oxo-1%2C2-+Dihydroindol-%283z%29-Ylidenemethyl%5D-2%2C4-+Dimethyl-1h-Pyrrole-3-Carboxylic+Acid+%282-Diethylaminoethyl%29Amide%2C+a+Novel+Tyrosine+Kinase+Inhibitor+Targeting+Vascular+Endothelial+and+Platelet-Derived+Growth+Factor+Receptor+Tyrosine+Kinase&doi=10.1021%2Fjm0204183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 5-[5-Fluoro-2-oxo-1,2- dihydroindol-(3Z)-ylidenemethyl]-2,4- dimethyl-1H-pyrrole-3-carboxylic Acid (2-Diethylaminoethyl)amide, a Novel Tyrosine Kinase Inhibitor Targeting Vascular Endothelial and Platelet-Derived Growth Factor Receptor Tyrosine Kinase</span></div><div class="casAuthors">Sun, Li; Liang, Chris; Shirazian, Sheri; Zhou, Yong; Miller, Todd; Cui, Jean; Fukuda, Juri Y.; Chu, Ji-Yu; Nematalla, Asaad; Wang, Xueyan; Chen, Hui; Sistla, Anand; Luu, Tony C.; Tang, Flora; Wei, James; Tang, Cho</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1116-1119</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To improve the antitumor properties and optimize the pharmaceutical properties including soly. and protein binding of indolin-2-ones, a series of different basic and weakly basic pyrrolylmethylidene indolinones I [R1 = H, F, Cl, Br; R2 = Et2NCH2CH2, pyridin-4-ylmethyl, 2-(1,2,3-triazol-1-yl)ethyl, etc.] were designed and synthesized.  Indolinone I [R1 = F, R2 = Et2NCH2CH2 (II)] showed the best overall profile in terms of potency for the VEGF-R2 and PDGF-Rβ tyrosine kinase at biochem. and cellular levels, soly., protein binding, and bioavailability.  II is currently in phase I clin. trials for the treatment of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptdQJ1VSPq8bVg90H21EOLACvtfcHk0lgvEvhwW0lI4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhslSlsbg%253D&md5=f6570cfa1ae4e31703c741d90fe65382</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm0204183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0204183%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DShirazian%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DMiller%26aufirst%3DT.%26aulast%3DCui%26aufirst%3DJ.%26aulast%3DFukuda%26aufirst%3DJ.%2BY.%26aulast%3DChu%26aufirst%3DJ.-Y.%26aulast%3DNematalla%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DSistla%26aufirst%3DA.%26aulast%3DLuu%26aufirst%3DT.%2BC.%26aulast%3DTang%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DC.%26atitle%3DDiscovery%2520of%25205-%255B5-Fluoro-2-Oxo-1%252C2-%2520Dihydroindol-%25283z%2529-Ylidenemethyl%255D-2%252C4-%2520Dimethyl-1h-Pyrrole-3-Carboxylic%2520Acid%2520%25282-Diethylaminoethyl%2529Amide%252C%2520a%2520Novel%2520Tyrosine%2520Kinase%2520Inhibitor%2520Targeting%2520Vascular%2520Endothelial%2520and%2520Platelet-Derived%2520Growth%2520Factor%2520Receptor%2520Tyrosine%2520Kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D1116%26epage%3D1119%26doi%3D10.1021%2Fjm0204183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blasina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arango, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraynov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Register, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ninkovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nichols, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderes, K.</span></span> <span> </span><span class="NLM_article-title">Breaching the DNA Damage Checkpoint Via Pf-00477736, a Novel Small-Molecule Inhibitor of Checkpoint Kinase 1</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2394</span>– <span class="NLM_lpage">2404</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-07-2391</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1158%2F1535-7163.MCT-07-2391" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=18723486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVWksL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=2394-2404&author=A.+Blasinaauthor=J.+Hallinauthor=E.+Chenauthor=M.+E.+Arangoauthor=E.+Kraynovauthor=J.+Registerauthor=S.+Grantauthor=S.+Ninkovicauthor=P.+Chenauthor=T.+Nicholsauthor=P.+O%E2%80%99Connorauthor=K.+Anderes&title=Breaching+the+DNA+Damage+Checkpoint+Via+Pf-00477736%2C+a+Novel+Small-Molecule+Inhibitor+of+Checkpoint+Kinase+1&doi=10.1158%2F1535-7163.mct-07-2391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Breaching the DNA damage checkpoint via PF-00477736, a novel small-molecule inhibitor of checkpoint kinase 1</span></div><div class="casAuthors">Blasina, Alessandra; Hallin, Jill; Chen, Enhong; Arango, Maria Elena; Kraynov, Eugenia; Register, James; Grant, Stephan; Ninkovic, Sacha; Chen, Ping; Nichols, Tim; O'Connor, Patrick; Anderes, Kenna</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2394-2404</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Checkpoints are present in all phases of the cell cycle and are regarded as the gatekeepers maintaining the integrity of the genome.  Many conventional agents used to treat cancer impart damage to the genome and activate cell cycle checkpoints.  Many tumors are defective in the tumor suppressor p53 and therefore lack a functional G1 checkpoint.  In these tumors, however, the S-G2 checkpoints remain intact and, in response to DNA damage, arrest cell cycle progression allowing time for DNA repair.  Checkpoint kinase 1 (Chk1) is a key element in the DNA damage response pathway and plays a crucial role in the S-G2-phase checkpoints.  Inhibiting Chk1 represents a therapeutic strategy for creating a "synthetic lethal" response by overriding the last checkpoint defense of tumor cells against the lethal damage induced by DNA-directed chemotherapeutic agents.  Chk1 inhibition is consistent with emerging targeted therapies aiming to exploit mol. differences between normal and cancer cells.  Adding a Chk1 inhibitor to DNA-damaging cytotoxic therapy selectively targets tumors with intrinsic checkpoint defects while minimizing toxicity in checkpoint-competent normal cells.  PF-00477736 was identified as a potent, selective ATP-competitive small-mol. inhibitor that inhibits Chk1 with a Ki of 0.49 nM.  PF-00477736 abrogates cell cycle arrest induced by DNA damage and enhances cytotoxicity of clin. important chemotherapeutic agents, including gemcitabine and carboplatin.  In xenografts, PF-00477736 enhanced the antitumor activity of gemcitabine in a dose-dependent manner.  PF-00477736 combinations were well tolerated with no exacerbation of side effects commonly assocd. with cytotoxic agents. [Mol Cancer Ther 2008;7(8):2394-404].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCAlkcnouhLrVg90H21EOLACvtfcHk0lhLl6Fj3oH-lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVWksL%252FP&md5=fc3f9cf9121c207c6fa284b2090ed1c3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-07-2391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-07-2391%26sid%3Dliteratum%253Aachs%26aulast%3DBlasina%26aufirst%3DA.%26aulast%3DHallin%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DE.%26aulast%3DArango%26aufirst%3DM.%2BE.%26aulast%3DKraynov%26aufirst%3DE.%26aulast%3DRegister%26aufirst%3DJ.%26aulast%3DGrant%26aufirst%3DS.%26aulast%3DNinkovic%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DNichols%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DP.%26aulast%3DAnderes%26aufirst%3DK.%26atitle%3DBreaching%2520the%2520DNA%2520Damage%2520Checkpoint%2520Via%2520Pf-00477736%252C%2520a%2520Novel%2520Small-Molecule%2520Inhibitor%2520of%2520Checkpoint%2520Kinase%25201%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2008%26volume%3D7%26spage%3D2394%26epage%3D2404%26doi%3D10.1158%2F1535-7163.mct-07-2391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Solution Structure of Brd7 Bromodomain and Its Interaction with Acetylated Peptides from Histone H3 and H4</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2007.04.139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1016%2Fj.bbrc.2007.04.139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=17498659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsFKgtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2007&pages=435-441&author=H.+Sunauthor=J.+Liuauthor=J.+Zhangauthor=W.+Shenauthor=H.+Huangauthor=C.+Xuauthor=H.+Daiauthor=J.+Wuauthor=Y.+Shi&title=Solution+Structure+of+Brd7+Bromodomain+and+Its+Interaction+with+Acetylated+Peptides+from+Histone+H3+and+H4&doi=10.1016%2Fj.bbrc.2007.04.139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Solution structure of BRD7 bromodomain and its interaction with acetylated peptides from histone H3 and H4</span></div><div class="casAuthors">Sun, Hongbin; Liu, Jiangxin; Zhang, Jiahai; Shen, Weiqun; Huang, Hongda; Xu, Chao; Dai, Haiming; Wu, Jihui; Shi, Yunyu</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">435-441</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">BRD7 is an important protein tightly assocd. with Nasopharyngeal carcinoma (NPC).  Overexpression of BRD7 inhibits NPC cell growth and cell cycle by transcriptionally regulating the cell cycle related genes.  BRD7 contains a bromodomain that is found in many chromatin-assocd. proteins and in nearly all known nuclear histone acetyltransferases (HATs) and plays an important role in chromatin remodeling and transcriptional activation.  Here, we report the soln. structure of BRD7 bromodomain detd. by NMR spectroscopy, and its binding specificity revealed by NMR titrn. with several acetylated histone peptides.  We find that BRD7 bromodomain contains the typical left-handed four-helix bundle topol., and can bind with weak affinity to lysine-acetylated peptides derived from histone H3 with K9 or K14 acetylated and from histone H4 with K8, K12 or K16 acetylated.  Our results show that BRD7 bromodomain lacks inherent binding specificity when binding to histones in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRW-fXT-5M4rVg90H21EOLACvtfcHk0lhLl6Fj3oH-lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsFKgtL8%253D&md5=4df1fec4e7a9dc3885e176f723288703</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2007.04.139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2007.04.139%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DDai%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DSolution%2520Structure%2520of%2520Brd7%2520Bromodomain%2520and%2520Its%2520Interaction%2520with%2520Acetylated%2520Peptides%2520from%2520Histone%2520H3%2520and%2520H4%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2007%26volume%3D358%26spage%3D435%26epage%3D441%26doi%3D10.1016%2Fj.bbrc.2007.04.139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonards, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikbin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guetzoyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samsonova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Promiscuous Targeting of Bromodomains by Bromosporine Identifies Bet Proteins as Master Regulators of Primary Transcription Response in Leukemia</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e1600760</span> <span class="refDoi"> DOI: 10.1126/sciadv.1600760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1126%2Fsciadv.1600760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=27757418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&author=S.+Picaudauthor=K.+Leonardsauthor=J.-P.+Lambertauthor=O.+Doveyauthor=C.+Wellsauthor=O.+Fedorovauthor=O.+Monteiroauthor=T.+Fujisawaauthor=C.-Y.+Wangauthor=H.+Lingardauthor=C.+Tallantauthor=N.+Nikbinauthor=L.+Guetzoyanauthor=R.+Inghamauthor=S.+V.+Leyauthor=P.+Brennanauthor=S.+Mullerauthor=A.+Samsonovaauthor=A.-C.+Gingrasauthor=J.+Schwallerauthor=G.+Vassiliouauthor=S.+Knappauthor=P.+Filippakopoulos&title=Promiscuous+Targeting+of+Bromodomains+by+Bromosporine+Identifies+Bet+Proteins+as+Master+Regulators+of+Primary+Transcription+Response+in+Leukemia&doi=10.1126%2Fsciadv.1600760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span></div><div class="casAuthors">Picaud, Sarah; Leonards, Katharina; Lambert, Jean-Philippe; Dovey, Oliver; Wells, Christopher; Fedorov, Oleg; Monteiro, Octovia; Fujisawa, Takao; Wang, Chen-Yi; Lingard, Hannah; Tallant, Cynthia; Nikbin, Nikzad; Guetzoyan, Lucie; Ingham, Richard; Ley, Steven V.; Brennan, Paul; Muller, Susanne; Samsonova, Anastasia; Gingras, Anne-Claude; Schwaller, Juerg; Vassiliou, George; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1600760/1-e1600760/16</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) have emerged as compelling targets for cancer therapy.  The development of selective and potent BET (bromo and extra-terminal) inhibitors and their significant activity in diverse tumor models have rapidly translated into clin. studies and have motivated drug development efforts targeting non-BET BRDs.  However, the complexmultidomain/subunit architecture of BRD protein complexes complicates predictions of the consequences of their pharmacol. targeting.  To address this issue, we developed a promiscuous BRD inhibitor [bromosporine (BSP)] that broadly targets BRDs (including BETs) with nanomolar affinity, creating a tool for the identification of cellular processes and diseases where BRDs have a regulatory function.  As a proof of principle, we studied the effects of BSP on leukemic cell lines known to be sensitive to BET inhibition and found, as expected, strong antiproliferative activity.  Comparison of the modulation of transcriptional profiles by BSP after a short exposure to the inhibitor resulted in a BET inhibitor signature but no significant addnl. changes in transcription that could account for inhibition of other BRDs.  Thus, nonselective targeting of BRDs identified BETs, but not other BRDs, as master regulators of contextdependent primary transcription response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28artLYYsqrVg90H21EOLACvtfcHk0lhLl6Fj3oH-lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D&md5=3060e8fd66c657f70e0ec3acea273057</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1600760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1600760%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DLeonards%26aufirst%3DK.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DDovey%26aufirst%3DO.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DC.-Y.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DNikbin%26aufirst%3DN.%26aulast%3DGuetzoyan%26aufirst%3DL.%26aulast%3DIngham%26aufirst%3DR.%26aulast%3DLey%26aufirst%3DS.%2BV.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSamsonova%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DVassiliou%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DPromiscuous%2520Targeting%2520of%2520Bromodomains%2520by%2520Bromosporine%2520Identifies%2520Bet%2520Proteins%2520as%2520Master%2520Regulators%2520of%2520Primary%2520Transcription%2520Response%2520in%2520Leukemia%26jtitle%3DSci.%2520Adv.%26date%3D2016%26volume%3D2%26doi%3D10.1126%2Fsciadv.1600760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chory, E. J.</span>; <span class="NLM_string-name">Kirkland, J. G.</span>; <span class="NLM_string-name">Chang, C.-Y.</span>; <span class="NLM_string-name">D’Andrea, V. D.</span>; <span class="NLM_string-name">Gourinsankar, S.</span>; <span class="NLM_string-name">Dykhuizen, E. C.</span>; <span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">Inhibition of a Selective Swi/Snf Function Synergizes with Atr Inhibitors in Cancer Cell Killing</span>. <span class="NLM_year" style="font-weight: bold;">2019</span>, bioRxiv 660456.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=E.+J.+Chory&author=J.+G.+Kirkland&author=C.-Y.+Chang&author=V.+D.+D%E2%80%99Andrea&author=S.+Gourinsankar&author=E.+C.+Dykhuizen&author=G.+R.+Crabtree&title=Inhibition+of+a+Selective+Swi%2FSnf+Function+Synergizes+with+Atr+Inhibitors+in+Cancer+Cell+Killing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChory%26aufirst%3DE.%2BJ.%26atitle%3DInhibition%2520of%2520a%2520Selective%2520Swi%252FSnf%2520Function%2520Synergizes%2520with%2520Atr%2520Inhibitors%2520in%2520Cancer%2520Cell%2520Killing%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bell, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, P. K.</span></span> <span> </span><span class="NLM_article-title">Pik3ca Cooperates with Kras to Promote Myc Activity and Tumorigenesis Via the Bromodomain Protein Brd9</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1634</span>, <span class="refDoi"> DOI: 10.3390/cancers11111634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.3390%2Fcancers11111634" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=1634&author=C.+M.+Bellauthor=P.+Raffeinerauthor=J.+R.+Hartauthor=P.+K.+Vogt&title=Pik3ca+Cooperates+with+Kras+to+Promote+Myc+Activity+and+Tumorigenesis+Via+the+Bromodomain+Protein+Brd9&doi=10.3390%2Fcancers11111634"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3390%2Fcancers11111634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers11111634%26sid%3Dliteratum%253Aachs%26aulast%3DBell%26aufirst%3DC.%2BM.%26aulast%3DRaffeiner%26aufirst%3DP.%26aulast%3DHart%26aufirst%3DJ.%2BR.%26aulast%3DVogt%26aufirst%3DP.%2BK.%26atitle%3DPik3ca%2520Cooperates%2520with%2520Kras%2520to%2520Promote%2520Myc%2520Activity%2520and%2520Tumorigenesis%2520Via%2520the%2520Bromodomain%2520Protein%2520Brd9%26jtitle%3DCancers%26date%3D2019%26volume%3D11%26spage%3D1634%26doi%3D10.3390%2Fcancers11111634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kallen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergsdorf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnaud, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brichet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cobos-Correa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elhajouji, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freuler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galimberti, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guibourdenche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haenni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holzinger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunziker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izaac, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leder, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martus, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Matt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyakov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roethlisberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiefl, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uteng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerchner, A.</span></span> <span> </span><span class="NLM_article-title">X-Ray Structures and Feasibility Assessment of Clk2 Inhibitors for Phelan-Mcdermid Syndrome</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1997</span>– <span class="NLM_lpage">2007</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1002%2Fcmdc.201800344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=29985556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFahs7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1997-2007&author=J.+Kallenauthor=C.+Bergsdorfauthor=B.+Arnaudauthor=M.+Bernhardauthor=M.+Brichetauthor=A.+Cobos-Correaauthor=A.+Elhajoujiauthor=F.+Freulerauthor=I.+Galimbertiauthor=C.+Guibourdencheauthor=S.+Haenniauthor=S.+Holzingerauthor=J.+Hunzikerauthor=A.+Izaacauthor=M.+Kaufmannauthor=L.+Lederauthor=H.-J.+Martusauthor=P.+von+Mattauthor=V.+Polyakovauthor=P.+Roethlisbergerauthor=G.+Romaauthor=N.+Stieflauthor=M.+Utengauthor=A.+Lerchner&title=X-Ray+Structures+and+Feasibility+Assessment+of+Clk2+Inhibitors+for+Phelan-Mcdermid+Syndrome&doi=10.1002%2Fcmdc.201800344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">X-ray Structures and Feasibility Assessment of CLK2 Inhibitors for Phelan-McDermid Syndrome</span></div><div class="casAuthors">Kallen, Joerg; Bergsdorf, Christian; Arnaud, Bertrand; Bernhard, Mario; Brichet, Murielle; Cobos-Correa, Amanda; Elhajouji, Azeddine; Freuler, Felix; Galimberti, Ivan; Guibourdenche, Christel; Haenni, Simon; Holzinger, Sandra; Hunziker, Juerg; Izaac, Aude; Kaufmann, Markus; Leder, Lukas; Martus, Hans-Joerg; von Matt, Peter; Polyakov, Valery; Roethlisberger, Patrik; Roma, Guglielmo; Stiefl, Nikolaus; Uteng, Marianne; Lerchner, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1997-2007</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">CLK2 inhibition has been proposed as a potential mechanism to improve autism and neuronal functions in Phelan-McDermid syndrome (PMDS).  Herein, the discovery of a very potent indazole CLK inhibitor series and the CLK2 x-ray structure of the most potent analog are reported.  This new indazole series was identified through a biochem. CLK2 Caliper assay screen with 30k compds. selected by an in silico approach.  Novel high-resoln. x-ray structures of all CLKs, including the first CLK4 x-ray structure, bound to known CLK2 inhibitor tool compds. (e.g., TG003, CX-4945), are also shown and yield insight into inhibitor selectivity in the CLK family.  The efficacy of the new CLK2 inhibitors from the indazole series was demonstrated in the mouse brain slice assay, and potential safety concerns were investigated.  Genotoxicity findings in the human lymphocyte micronucleus test (MNT) assay are shown by using two structurally different CLK inhibitors to reveal a major concern for pan-CLK inhibition in PMDS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXWV3hjWfnELVg90H21EOLACvtfcHk0liRZnAXbAKlWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFahs7vK&md5=bd539ce78951e907dd74cc293022edb1</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800344%26sid%3Dliteratum%253Aachs%26aulast%3DKallen%26aufirst%3DJ.%26aulast%3DBergsdorf%26aufirst%3DC.%26aulast%3DArnaud%26aufirst%3DB.%26aulast%3DBernhard%26aufirst%3DM.%26aulast%3DBrichet%26aufirst%3DM.%26aulast%3DCobos-Correa%26aufirst%3DA.%26aulast%3DElhajouji%26aufirst%3DA.%26aulast%3DFreuler%26aufirst%3DF.%26aulast%3DGalimberti%26aufirst%3DI.%26aulast%3DGuibourdenche%26aufirst%3DC.%26aulast%3DHaenni%26aufirst%3DS.%26aulast%3DHolzinger%26aufirst%3DS.%26aulast%3DHunziker%26aufirst%3DJ.%26aulast%3DIzaac%26aufirst%3DA.%26aulast%3DKaufmann%26aufirst%3DM.%26aulast%3DLeder%26aufirst%3DL.%26aulast%3DMartus%26aufirst%3DH.-J.%26aulast%3Dvon%2BMatt%26aufirst%3DP.%26aulast%3DPolyakov%26aufirst%3DV.%26aulast%3DRoethlisberger%26aufirst%3DP.%26aulast%3DRoma%26aufirst%3DG.%26aulast%3DStiefl%26aufirst%3DN.%26aulast%3DUteng%26aufirst%3DM.%26aulast%3DLerchner%26aufirst%3DA.%26atitle%3DX-Ray%2520Structures%2520and%2520Feasibility%2520Assessment%2520of%2520Clk2%2520Inhibitors%2520for%2520Phelan-Mcdermid%2520Syndrome%26jtitle%3DChemMedChem%26date%3D2018%26volume%3D13%26spage%3D1997%26epage%3D2007%26doi%3D10.1002%2Fcmdc.201800344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kutach, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villaseñor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belunis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lok, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, L.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deval, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novak, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuglstatter, A.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of Il-2-Inducible T Cell Kinase Complexed with Inhibitors: Insights into Rational Drug Design and Activity Regulation</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">163</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2010.00993.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1111%2Fj.1747-0285.2010.00993.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=20545945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptleqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2010&pages=154-163&author=A.+K.+Kutachauthor=A.+G.+Villase%C3%B1orauthor=D.+Lamauthor=C.+Belunisauthor=C.+Jansonauthor=S.+Lokauthor=L.-N.+Hongauthor=C.-M.+Liuauthor=J.+Devalauthor=T.+J.+Novakauthor=J.+W.+Barnettauthor=W.+Chuauthor=D.+Shawauthor=A.+Kuglstatter&title=Crystal+Structures+of+Il-2-Inducible+T+Cell+Kinase+Complexed+with+Inhibitors%3A+Insights+into+Rational+Drug+Design+and+Activity+Regulation&doi=10.1111%2Fj.1747-0285.2010.00993.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of IL-2-inducible T cell kinase complexed with inhibitors: insights into rational drug design and activity regulation</span></div><div class="casAuthors">Kutach, Alan K.; Villasenor, Armando G.; Lam, Diana; Belunis, Charles; Janson, Cheryl; Lok, Stephen; Hong, Li-Na; Liu, Chao-Min; Deval, Jerome; Novak, Thomas J.; Barnett, Jim W.; Chu, Wei; Shaw, David; Kuglstatter, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">154-163</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">IL-2-inducible T cell kinase plays an essential role in T cell receptor signaling and is considered a drug target for the treatment of Th2-mediated inflammatory diseases.  By applying high-throughput protein engineering and crystn., we have detd. the x-ray crystal structures of IL-2-inducible T cell kinase in complex with its selective inhibitor BMS-509744 and the broad-spectrum kinase inhibitors sunitinib and RO5191614.  Sunitinib uniquely stabilizes IL-2-inducible T cell kinase in the helix C-in conformation by inducing side chain conformational changes in the ATP-binding site.  This preference of sunitinib to bind to an active kinase conformation is reflective of its broad-spectrum kinase activity.  BMS-509744 uniquely stabilizes the activation loop in a substrate-blocking inactive conformation, indicating that structural changes described for Src family kinases are also involved in the regulation of IL-2-inducible T cell kinase activity.  The obsd. BMS-509744 binding mode allows rationalization of structure-activity relationships reported for this inhibitor class and facilitates further structure-based drug design.  Sequence-based anal. of this binding mode provides guidance for the rational design of inhibitor selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXxFkkRUV7B7Vg90H21EOLACvtfcHk0lgr6_YFBRRTNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptleqs7Y%253D&md5=24b9225c52eff5cca4e295dad1c554c3</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2010.00993.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2010.00993.x%26sid%3Dliteratum%253Aachs%26aulast%3DKutach%26aufirst%3DA.%2BK.%26aulast%3DVillase%25C3%25B1or%26aufirst%3DA.%2BG.%26aulast%3DLam%26aufirst%3DD.%26aulast%3DBelunis%26aufirst%3DC.%26aulast%3DJanson%26aufirst%3DC.%26aulast%3DLok%26aufirst%3DS.%26aulast%3DHong%26aufirst%3DL.-N.%26aulast%3DLiu%26aufirst%3DC.-M.%26aulast%3DDeval%26aufirst%3DJ.%26aulast%3DNovak%26aufirst%3DT.%2BJ.%26aulast%3DBarnett%26aufirst%3DJ.%2BW.%26aulast%3DChu%26aufirst%3DW.%26aulast%3DShaw%26aufirst%3DD.%26aulast%3DKuglstatter%26aufirst%3DA.%26atitle%3DCrystal%2520Structures%2520of%2520Il-2-Inducible%2520T%2520Cell%2520Kinase%2520Complexed%2520with%2520Inhibitors%253A%2520Insights%2520into%2520Rational%2520Drug%2520Design%2520and%2520Activity%2520Regulation%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2010%26volume%3D76%26spage%3D154%26epage%3D163%26doi%3D10.1111%2Fj.1747-0285.2010.00993.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Q. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng
de, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, J. C.</span></span> <span> </span><span class="NLM_article-title">Bromodomain-Containing Protein 7 (Brd7) as a Potential Tumor Suppressor in Hepatocellular Carcinoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">16248</span>– <span class="NLM_lpage">16261</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.7637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.18632%2Foncotarget.7637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=26919247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A280%3ADC%252BC28jks1Oktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=16248-16261&author=C.+L.+Chenauthor=Y.+Wangauthor=Q.+Z.+Panauthor=Y.+Tangauthor=Q.+J.+Wangauthor=K.+Panauthor=L.+X.+Huangauthor=J.+Heauthor=J.+J.+Zhaoauthor=S.+S.+Jiangauthor=X.+F.+Zhangauthor=H.+X.+Zhangauthor=Z.+Q.+Zhouauthor=S.+Weng%0Adeauthor=J.+C.+Xia&title=Bromodomain-Containing+Protein+7+%28Brd7%29+as+a+Potential+Tumor+Suppressor+in+Hepatocellular+Carcinoma&doi=10.18632%2Foncotarget.7637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain-containing protein 7 (BRD7) as a potential tumor suppressor in hepatocellular carcinoma</span></div><div class="casAuthors">Chen Chang-Long; Wang Ying; Pan Qiu-Zhong; Tang Yan; Pan Ke; Zhao Jing-Jing; Jiang Shan-Shan; Zhang Xiao-Fei; Zhang Hong-Xia; Zhou Zi-Qi; Weng De Sheng; Xia Jian-Chuan; Chen Chang-Long; Pan Qiu-Zhong; Tang Yan; Wang Qi-Jing; Pan Ke; Huang Li-Xi; He Jia; Zhao Jing-Jing; Jiang Shan-Shan; Zhang Xiao-Fei; Zhang Hong-Xia; Zhou Zi-Qi; Weng De Sheng; Xia Jian-Chuan; Wang Ying</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">16248-61</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bromodomain-containing protein 7 (BRD7) is a subunit of the PBAF complex, which functions as a transcriptional cofactor for the tumor suppressor protein p53.  Down-regulation of BRD7 has been demonstrated in multiple types of cancer.  This study aimed to investigate BRD7 expression and its tumor suppressive effect in hepatocellular carcinoma (HCC).The expression of BRD7 was examined in clinical specimens of primary HCC and in HCC cell lines through real-time quantitative PCR, western blot and immunohistochemistry.  The prognostic value of BRD7 expression and its correlation with the clinicopathological features of HCC patients were statistically analyzed.  The effect of BRD7 on the tumorigenicity of HCC was also examined using proliferation and colony-formation assays, cell-cycle assays, migration and cell-invasion assays, and xenograft nude mouse models.  BRD7 was down-regulated in tumor tissues and HCC cell lines.  BRD7 protein expression was strongly associated with clinical stage and tumor size.  Kaplan-Meier survival curves revealed higher survival rates in patients with higher BRD7 expression levels compared to those with lower BRD7 levels.  A multivariate analysis indicated that BRD7 expression was an independent prognostic marker.  The re-introduction of BRD7 expression significantly inhibited proliferation, colony formation, migration and invasion and led to cell cycle arrest in HCC cells in vitro.  Furthermore, experiments in mice suggested that BRD7 overexpression suppresses HCC tumorigenicity in vivo.  In conclusions, our data indicated that BRD7 may serve as a tumor suppressor in HCC and may be a novel molecular target for the treatment of HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR0i0lNQTGmmJPlX7E30qdzfW6udTcc2eazSIJcPokV07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28jks1Oktw%253D%253D&md5=e9ab62dfe3a39f88b7624219c598a02d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.7637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.7637%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%2BL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DPan%26aufirst%3DQ.%2BZ.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%2BJ.%26aulast%3DPan%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DL.%2BX.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DJ.%2BJ.%26aulast%3DJiang%26aufirst%3DS.%2BS.%26aulast%3DZhang%26aufirst%3DX.%2BF.%26aulast%3DZhang%26aufirst%3DH.%2BX.%26aulast%3DZhou%26aufirst%3DZ.%2BQ.%26aulast%3DWeng%2Bde%26aufirst%3DS.%26aulast%3DXia%26aufirst%3DJ.%2BC.%26atitle%3DBromodomain-Containing%2520Protein%25207%2520%2528Brd7%2529%2520as%2520a%2520Potential%2520Tumor%2520Suppressor%2520in%2520Hepatocellular%2520Carcinoma%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D16248%26epage%3D16261%26doi%3D10.18632%2Foncotarget.7637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, H.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, D.-S.</span></span> <span> </span><span class="NLM_article-title">Tumor Suppressive Effects of Bromodomain-Containing Protein 7 (Brd7) in Epithelial Ovarian Carcinoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">565</span>– <span class="NLM_lpage">575</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-13-1271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1158%2F1078-0432.CCR-13-1271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=24198243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFWjs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=565-575&author=Y.-A.+Parkauthor=J.-W.+Leeauthor=H.-S.+Kimauthor=Y.-Y.+Leeauthor=T.-J.+Kimauthor=C.+H.+Choiauthor=J.-J.+Choiauthor=H.-K.+Jeonauthor=Y.+J.+Choauthor=J.+Y.+Ryuauthor=B.-G.+Kimauthor=D.-S.+Bae&title=Tumor+Suppressive+Effects+of+Bromodomain-Containing+Protein+7+%28Brd7%29+in+Epithelial+Ovarian+Carcinoma&doi=10.1158%2F1078-0432.ccr-13-1271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor Suppressive Effects of Bromodomain-Containing Protein 7 (BRD7) in Epithelial Ovarian Carcinoma</span></div><div class="casAuthors">Park, Young-Ae; Lee, Jeong-Won; Kim, Hye-Sun; Lee, Yoo-Young; Kim, Tae-Joong; Choi, Chel Hun; Choi, Jung-Joo; Jeon, Hye-Kyung; Cho, Young Jae; Ryu, Ji Yoon; Kim, Byoung-Gie; Bae, Duk-Soo</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">565-575</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Bromodomain-contg. protein 7 (BRD7), which is a subunit of SWI/SNF complex, has been recently suggested as a novel tumor suppressor in several cancers.  In this study, we investigated the tumor suppressive effect of BRD7 in epithelial ovarian cancer.  Exptl. Design: We analyzed the expression of BRD7 in human ovarian tissues with real-time PCR.  To investigate the functional role of BRD7, we transfected ovarian cancer cells (A2780 and SKOV3) with BRD7 plasmid and checked the cell viability, apoptosis, and invasion.  The activities of BRD7 in the signaling pathways assocd. with carcinogenesis were also tested.  In addn., we used the orthotopic mouse model for ovarian cancer to evaluate tumor growth-inhibiting effect by administration of BRD7 plasmid.  Results: The BRD7 expression was downregulated in the ovarian cancer tissues compared with normal (P < 0.05), high-grade serous cancer exhibited significantly decreased expression of BRD7 compared with low-grade (P < 0.01) serous cancer.  Transfection of BRD7 plasmid to A2780 (p53-wild) or SKOV3 (p53-null) ovarian cancer cells showed the tumor suppressive effects assessed by cell viability, apoptosis, and invasion assay and esp. significantly decreased tumor wt. in orthotopic mouse model (A2780).  Moreover, we found that tumor suppressive effects of BRD7 are independent to the presence of p53 activity in ovarian cancer cells.  BRD7 neg. regulated β-catenin pathway, resulting in decreased its accumulation in the nucleus.  Conclusions: These results suggested that BRD7 acts as a tumor suppressor in epithelial ovarian cancers independently of p53 activity, via neg. regulation of β-catenin pathway.  Clin Cancer Res; 20(3); 565-75. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEfH7-Hwr_0LVg90H21EOLACvtfcHk0liMc3IvQ-82ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFWjs78%253D&md5=243d439fc2b98b8580ad99e736aa0e7d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-13-1271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-13-1271%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DY.-A.%26aulast%3DLee%26aufirst%3DJ.-W.%26aulast%3DKim%26aufirst%3DH.-S.%26aulast%3DLee%26aufirst%3DY.-Y.%26aulast%3DKim%26aufirst%3DT.-J.%26aulast%3DChoi%26aufirst%3DC.%2BH.%26aulast%3DChoi%26aufirst%3DJ.-J.%26aulast%3DJeon%26aufirst%3DH.-K.%26aulast%3DCho%26aufirst%3DY.%2BJ.%26aulast%3DRyu%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DB.-G.%26aulast%3DBae%26aufirst%3DD.-S.%26atitle%3DTumor%2520Suppressive%2520Effects%2520of%2520Bromodomain-Containing%2520Protein%25207%2520%2528Brd7%2529%2520in%2520Epithelial%2520Ovarian%2520Carcinoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2014%26volume%3D20%26spage%3D565%26epage%3D575%26doi%3D10.1158%2F1078-0432.ccr-13-1271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasater, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Dual Kinase-Bromodomain Inhibitors for Rationally Designed Polypharmacology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+M%C3%BCllerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+Kinase-Bromodomain+Inhibitors+for+Rationally+Designed+Polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0liMc3IvQ-82ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520Kinase-Bromodomain%2520Inhibitors%2520for%2520Rationally%2520Designed%2520Polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Acetyl-Lysine Binding Site of Bromodomain-Containing Protein 4 (Brd4) Interacts with Diverse Kinase Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1021/cb500072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1160-1171&author=S.+W.+J.+Emberauthor=J.-Y.+Zhuauthor=S.+H.+Olesenauthor=M.+P.+Martinauthor=A.+Beckerauthor=N.+Berndtauthor=G.+I.+Georgauthor=E.+Sch%C3%B6nbrunn&title=Acetyl-Lysine+Binding+Site+of+Bromodomain-Containing+Protein+4+%28Brd4%29+Interacts+with+Diverse+Kinase+Inhibitors&doi=10.1021%2Fcb500072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyl-lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span></div><div class="casAuthors">Ember, Stuart W. J.; Zhu, Jin-Yi; Olesen, Sanne H.; Martin, Mathew P.; Becker, Andreas; Berndt, Norbert; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1160-1171</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the bromodomain and extra terminal (BET) family of proteins are essential for the recognition of acetylated lysine (KAc) residues in histones and have emerged as promising drug targets in cancer, inflammation, and contraception research.  In co-crystn. screening campaigns using the first bromodomain of BRD4 (BRD4-1) against kinase inhibitor libraries, we identified and characterized 14 kinase inhibitors (10 distinct chem. scaffolds) as ligands of the KAc binding site.  Among these, the PLK1 inhibitor BI2536 and the JAK2 inhibitor TG101209 displayed strongest inhibitory potential against BRD4 (IC50 = 25 nM and 130 nM, resp.) and high selectivity for BET bromodomains.  Comparative structural anal. revealed markedly different binding modes of kinase hinge-binding scaffolds in the KAc binding site, suggesting that BET proteins are potential off-targets of diverse kinase inhibitors.  Combined, these findings provide a new structural framework for the rational design of next-generation BET-selective and dual-activity BET-kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf0X1hjt7cBbVg90H21EOLACvtfcHk0liMc3IvQ-82ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D&md5=5ca37df00cdaa0ba81c0cb4a0f70ba5d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fcb500072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500072z%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DZhu%26aufirst%3DJ.-Y.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DAcetyl-Lysine%2520Binding%2520Site%2520of%2520Bromodomain-Containing%2520Protein%25204%2520%2528Brd4%2529%2520Interacts%2520with%2520Diverse%2520Kinase%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1160%26epage%3D1171%26doi%3D10.1021%2Fcb500072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Dual Kinase-Bromodomain Inhibitors in Anticancer Drug Discovery: A Structural and Pharmacological Perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9305</span>– <span class="NLM_lpage">9320</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00438</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00438" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9305-9320&author=L.+Carlinoauthor=G.+Rastelli&title=Dual+Kinase-Bromodomain+Inhibitors+in+Anticancer+Drug+Discovery%3A+A+Structural+and+Pharmacological+Perspective&doi=10.1021%2Facs.jmedchem.6b00438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Kinase-Bromodomain Inhibitors in Anticancer Drug Discovery: A Structural and Pharmacological Perspective</span></div><div class="casAuthors">Carlino, Luca; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9305-9320</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases play crucial roles in several cell transformation processes and are validated drug targets for many human diseases, including cancer.  Nevertheless, most tumors have eluded the effects of inhibition of a single kinase by activating resistance mechanisms and/or alternative pathways and escape mechanisms.  In recent years, multitarget approaches directed toward inhibition of kinases and targets of different families have received increasing attention.  In particular, co-targeting kinases and bromodomain epigenetic reader proteins has rapidly emerged as a promising approach to cancer drug development.  In this manuscript, we will review the recent discoveries that led to the identification and optimization of dual kinase/bromodomain inhibitors.  We will analyze and compare the structural features required for dual inhibition and comment on the potential of this approach in anticancer drug discovery.  Moreover, we will introduce computational approaches useful for the identification of dual kinase/bromodomain inhibitors and generate ad hoc pharmacophore and docking models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkHDSBWCGdbVg90H21EOLACvtfcHk0lg6OcWEYfqkOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP&md5=1351a235bba09782160a86ae3f904833</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00438%26sid%3Dliteratum%253Aachs%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DDual%2520Kinase-Bromodomain%2520Inhibitors%2520in%2520Anticancer%2520Drug%2520Discovery%253A%2520A%2520Structural%2520and%2520Pharmacological%2520Perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9305%26epage%3D9320%26doi%3D10.1021%2Facs.jmedchem.6b00438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williston, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandts, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-N.</span></span> <span> </span><span class="NLM_article-title">Rapid Measurement of Binding Constants and Heats of Binding Using a New Titration Calorimeter</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>179</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(89)90213-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1016%2F0003-2697%2889%2990213-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=2757186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1989&pages=131-137&author=T.+Wisemanauthor=S.+Willistonauthor=J.+F.+Brandtsauthor=L.-N.+Lin&title=Rapid+Measurement+of+Binding+Constants+and+Heats+of+Binding+Using+a+New+Titration+Calorimeter&doi=10.1016%2F0003-2697%2889%2990213-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span></div><div class="casAuthors">Wiseman, Thomas; Williston, Samuel; Brandts, John F.; Lin, Lung Nan</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-7</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A new titrn. calorimeter is described and results are presented for the binding of 2'-CMP to the active site of RNase A.  The instrument characteristics include very high sensitivity, rapid calorimetric response, and fast thermal equilibration.  Convenient software is available for instrument operation, data collection, data redn., and deconvolution to obtain least-squares ests. of binding parameters n, ΔH°, ΔS°, and the binding const. K.  Sample through-put for the instrument is high, and under favorable conditions binding consts. as large as 108 M-1 can be measured.  The bovine RNase A (RNase)/2'-CMP system was studied over a 50-fold range of RNase concn. and at 2 different temps.  The binding consts. were in the 105 to 106 M-1 range, depending on conditions, and heats of binding ∼-15,000 cal/mol.  Repeat detns. suggested errors of only a few percent in n, ΔH°, and K values over the most favorable concn. range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRWydqrfWArVg90H21EOLACvtfcHk0lg6OcWEYfqkOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D&md5=53fcdb7144fd85e57963add0bcdb6041</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2889%2990213-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252889%252990213-3%26sid%3Dliteratum%253Aachs%26aulast%3DWiseman%26aufirst%3DT.%26aulast%3DWilliston%26aufirst%3DS.%26aulast%3DBrandts%26aufirst%3DJ.%2BF.%26aulast%3DLin%26aufirst%3DL.-N.%26atitle%3DRapid%2520Measurement%2520of%2520Binding%2520Constants%2520and%2520Heats%2520of%2520Binding%2520Using%2520a%2520New%2520Titration%2520Calorimeter%26jtitle%3DAnal.%2520Biochem.%26date%3D1989%26volume%3D179%26spage%3D131%26epage%3D137%26doi%3D10.1016%2F0003-2697%2889%2990213-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, Scaling, Space-Group Assignment and Post-Refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+Scaling%2C+Space-Group+Assignment+and+Post-Refinement&doi=10.1107%2Fs0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0lg6OcWEYfqkOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520Scaling%252C%2520Space-Group%2520Assignment%2520and%2520Post-Refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2Fs0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gildea, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Montero, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michels-Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span> <span> </span><span class="NLM_article-title">Dials: Implementation and Evaluation of a New Integration Package</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1107/s2059798317017235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1107%2FS2059798317017235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=85-97&author=G.+Winterauthor=D.+G.+Watermanauthor=J.+M.+Parkhurstauthor=A.+S.+Brewsterauthor=R.+J.+Gildeaauthor=M.+Gerstelauthor=L.+Fuentes-Monteroauthor=M.+Vollmarauthor=T.+Michels-Clarkauthor=I.+D.+Youngauthor=N.+K.+Sauterauthor=G.+Evans&title=Dials%3A+Implementation+and+Evaluation+of+a+New+Integration+Package&doi=10.1107%2Fs2059798317017235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">DIALS: implementation and evaluation of a new integration package</span></div><div class="casAuthors">Winter, Graeme; Waterman, David G.; Parkhurst, James M.; Brewster, Aaron S.; Gildea, Richard J.; Gerstel, Markus; Fuentes-Montero, Luis; Vollmar, Melanie; Michels-Clark, Tara; Young, Iris D.; Sauter, Nicholas K.; Evans, Gwyndaf</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-97</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The DIALS project is a collaboration between Diamond Light Source, Lawrence Berkeley National Lab. and CCP4 to develop a new software suite for the anal. of crystallog. X-ray diffraction data, initially encompassing spot finding, indexing, refinement and integration.  The design, core algorithms and structure of the software are introduced, alongside results from the anal. of data from biol. and chem. crystallog. expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkZqQrzPvxlbVg90H21EOLACvtfcHk0lh8PMNe_upJRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D&md5=f4572135ac40643815064914216f6d4c</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS2059798317017235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317017235%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DGerstel%26aufirst%3DM.%26aulast%3DFuentes-Montero%26aufirst%3DL.%26aulast%3DVollmar%26aufirst%3DM.%26aulast%3DMichels-Clark%26aufirst%3DT.%26aulast%3DYoung%26aufirst%3DI.%2BD.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DEvans%26aufirst%3DG.%26atitle%3DDials%253A%2520Implementation%2520and%2520Evaluation%2520of%2520a%2520New%2520Integration%2520Package%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2018%26volume%3D74%26spage%3D85%26epage%3D97%26doi%3D10.1107%2Fs2059798317017235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How Good Are My Data and What Is the Resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/s0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+Good+Are+My+Data+and+What+Is+the+Resolution%3F&doi=10.1107%2Fs0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lh8PMNe_upJRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520Good%2520Are%2520My%2520Data%2520and%2520What%2520Is%2520the%2520Resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2Fs0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Phenix.Model_Vs_Data: A High-Level Tool for the Calculation of Crystallographic Model and Data Statistics</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1107/s0021889810015608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1107%2Fs0021889810015608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=20648263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVygtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=669-676&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=V.+B.+Chenauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=J.+S.+Richardsonauthor=D.+C.+Richardsonauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Phenix.Model_Vs_Data%3A+A+High-Level+Tool+for+the+Calculation+of+Crystallographic+Model+and+Data+Statistics&doi=10.1107%2Fs0021889810015608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">phenix.model vs. data: a high-level tool for the calculation of crystallographic model and data statistics</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Chen, Vincent B.; Headd, Jeffrey J.; Moriarty, Nigel W.; Richardson, Jane S.; Richardson, David C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-676</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.modelvsdata is a high-level command-line tool for the computation of crystallog. model and data statistics, and the evaluation of the fit of the model to data.  Anal. of all Protein Data Bank structures that have exptl. data available shows that in most cases the reported statistics, in particular R factors, can be reproduced within a few percentage points.  However, there are a no. of outliers where the recomputed R values are significantly different from those originally reported.  The reasons for these discrepancies are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBhMlnjsRfhLVg90H21EOLACvtfcHk0lh8PMNe_upJRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVygtr4%253D&md5=5cd851a911ecd8b518e3ec4cdd5d9dd7</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2Fs0021889810015608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889810015608%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DPhenix.Model_Vs_Data%253A%2520A%2520High-Level%2520Tool%2520for%2520the%2520Calculation%2520of%2520Crystallographic%2520Model%2520and%2520Data%2520Statistics%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D669%26epage%3D676%26doi%3D10.1107%2Fs0021889810015608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lh8PMNe_upJRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marek  Wanior</span>, <span class="hlFld-ContribAuthor ">Andreas  Krämer</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>, <span class="hlFld-ContribAuthor ">Andreas C.  Joerger</span>. </span><span class="cited-content_cbyCitation_article-title">Exploiting vulnerabilities of SWI/SNF chromatin remodelling complexes for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2021,</strong> <em>40 </em>
                                    (21)
                                     , 3637-3654. <a href="https://doi.org/10.1038/s41388-021-01781-x" title="DOI URL">https://doi.org/10.1038/s41388-021-01781-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41388-021-01781-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41388-021-01781-x%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DExploiting%252Bvulnerabilities%252Bof%252BSWI%25252FSNF%252Bchromatin%252Bremodelling%252Bcomplexes%252Bfor%252Bcancer%252Btherapy%26aulast%3DWanior%26aufirst%3DMarek%26date%3D2021%26date%3D2021%26volume%3D40%26issue%3D21%26spage%3D3637%26epage%3D3654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elisabetta  Damiani</span>, <span class="hlFld-ContribAuthor ">Munevver N.  Duran</span>, <span class="hlFld-ContribAuthor ">Nivedhitha  Mohan</span>, <span class="hlFld-ContribAuthor ">Praveen  Rajendran</span>, <span class="hlFld-ContribAuthor ">Roderick H.  Dashwood</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Epigenetic ‘Readers’ with Natural Compounds for Cancer Interception. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cancer Prevention</span><span> <strong>2020,</strong> <em>25 </em>
                                    (4)
                                     , 189-203. <a href="https://doi.org/10.15430/JCP.2020.25.4.189" title="DOI URL">https://doi.org/10.15430/JCP.2020.25.4.189</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.15430/JCP.2020.25.4.189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.15430%2FJCP.2020.25.4.189%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cancer%2520Prevention%26atitle%3DTargeting%252BEpigenetic%252B%2525E2%252580%252598Readers%2525E2%252580%252599%252Bwith%252BNatural%252BCompounds%252Bfor%252BCancer%252BInterception%26aulast%3DDamiani%26aufirst%3DElisabetta%26date%3D2020%26volume%3D25%26issue%3D4%26spage%3D189%26epage%3D203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarah M.  Innis</span>, <span class="hlFld-ContribAuthor ">Birgit  Cabot</span>. </span><span class="cited-content_cbyCitation_article-title">GBAF, a small BAF sub-complex with big implications: a systematic review. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics & Chromatin</span><span> <strong>2020,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13072-020-00370-8" title="DOI URL">https://doi.org/10.1186/s13072-020-00370-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13072-020-00370-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13072-020-00370-8%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%2520%2526%2520Chromatin%26atitle%3DGBAF%25252C%252Ba%252Bsmall%252BBAF%252Bsub-complex%252Bwith%252Bbig%252Bimplications%25253A%252Ba%252Bsystematic%252Breview%26aulast%3DInnis%26aufirst%3DSarah%2BM.%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julie M.  Garlick</span>, <span class="hlFld-ContribAuthor ">Anna K.  Mapp</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Modulation of Dynamic Protein Complexes. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2020,</strong> <em>27 </em>
                                    (8)
                                     , 986-997. <a href="https://doi.org/10.1016/j.chembiol.2020.07.019" title="DOI URL">https://doi.org/10.1016/j.chembiol.2020.07.019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2020.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2020.07.019%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DSelective%252BModulation%252Bof%252BDynamic%252BProtein%252BComplexes%26aulast%3DGarlick%26aufirst%3DJulie%2BM.%26date%3D2020%26volume%3D27%26issue%3D8%26spage%3D986%26epage%3D997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiuzuo  Zhu</span>, <span class="hlFld-ContribAuthor ">Yi  Liao</span>, <span class="hlFld-ContribAuthor ">Liling  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting BRD9 for Cancer Treatment: A New Strategy. </span><span class="cited-content_cbyCitation_journal-name">OncoTargets and Therapy</span><span> <strong>2020,</strong> <em>Volume 13 </em>, 13191-13200. <a href="https://doi.org/10.2147/OTT.S286867" title="DOI URL">https://doi.org/10.2147/OTT.S286867</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2147/OTT.S286867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2147%2FOTT.S286867%26sid%3Dliteratum%253Aachs%26jtitle%3DOncoTargets%2520and%2520Therapy%26atitle%3DTargeting%252BBRD9%252Bfor%252BCancer%252BTreatment%25253A%252BA%252BNew%252BStrategy%26aulast%3DZhu%26aufirst%3DXiuzuo%26date%3D2020%26date%3D2020%26volume%3DVolume%252013%26spage%3D13191%26epage%3D13200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding potential of inhibitors for BRD7 and BRD9. (A) Chemical structures of reported BRD9 inhibitors studied herein. (B) Melting temperature shifts of BRD7 and BRD9 in the presence of the100 μM inhibitor. (C) Δ<i>T</i><sub>m</sub> values determined by DSF and <i>K</i><sub>d</sub> values determined by ITC and data correlation for BRD9 (blue) and BRD7 (red). For statistical analysis, <i>r</i> is Pearson’s correlation coefficient; asterisks indicate the significance of the two-tailed P value (α = 0.05). For graphical expression, data were fitted to semilogarithmic lines. (D) Thermodynamic parameters of inhibitor interactions with BRD9 (upper panel) and BRD7 (lower panel). ITC conditions and thermograms are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Table S1 and Figures S1, S2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Differences in the overall structures of BRD7 and BRD9 affect the binding poses of BI-7273 and BI-9564. (A) Superposition of the crystal structures of BRD7 (beige) and BRD9 (cyan) in the unliganded state (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PPA">6PPA</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UZF">6UZF</a>). Differing residues (red arrows) are located outside the KAc site; the critical asparagine residue (BRD7<sup>N211</sup> and BRD9<sup>N216</sup>) is drawn in magenta. (B) Comparison of the unliganded KAc sites of BRD7 and BRD9. Distances between the flanks of the binding site are indicated as orange-dotted lines (in Å). (C) Cocrystal structure of BRD7 with BI-7273 along with a schematic presentation of the binding interactions (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1E">6V1E</a>). (D) Same as (C) for BI-7273 in BRD9 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EU1">5EU1</a>). (E) Superposition of both structures. (F) Cocrystal structure of BRD7 with BI-9564 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1F">6V1F</a>) and (G) of BRD9 with BI-9564 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5F1H">5F1H</a>). (H) Superposition of both structures. BRD7 residues are shown in beige and BRD9 residues in cyan. H-bonding interactions are indicated as black-dotted lines, vdW hydrophobic interactions as green-curved lines and π-stacking interactions as green-dotted lines. Crystallization conditions, data collection and refinement statistics are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Tables S2 and S3. 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> and <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omit electron density maps are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Figures S3 and S4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding modes of I-BRD9, TP-472, and bromosporine in BRD7 and BRD9. (A) Cocrystal structure of I-BRD9 in BRD7 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V17">6V17</a>) and (B) in BRD9 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1B">6V1B</a>). (C) Superposition of both structures. (D–F) Same as (A–C) for TP-472 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V16">6V16</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V14">6V14</a>). (G–I) Same as (A–C) for bromosporine (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1H">6V1H</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IGM">5IGM</a>). Except for the BRD9-bromosporine complex, all structures were determined experimentally (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Table S3). 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> and <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Figure S3 and S4</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding modes of BRD7/9 inhibitors in BRD4. (A) Cocrystal structure of BI-7273 with BRD4-1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1K">6V1K</a>) and superposition with the BRD7/BI-7273 complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1E">6V1E</a>). (B) Same as (A) for BI-9564 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1L">6V1L</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1F">6V1F</a>). (C) Same as (A) for TP-472 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1U">6V1U</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V16">6V16</a>). (D) Same as (A) for bromosporine (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0U">6V0U</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1H">6V1H</a>). All structures were determined experimentally. Residues of BRD4 are shown in green and of BRD7 in beige. Inhibitor liganded with BRD4 is shown in yellow and liganded with BRD7 in orange. 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> and <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Figure S5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Discovery of BRD7/9-kinase inhibitors. (A) Chemical structures of kinase inhibitors identified as ligands of BRD7 and BRD9. (B) DSF derivative plots for BRD7 and BRD9 in the presence of 100 μM inhibitor. (C) ITC thermograms of TG003 interaction with BRD7 and BRD9. (D) Dissociation constants determined by the qPCR assay (performed by DiscoveRx) (<i>N</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/medium/jm9b01980_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structural basis of BRD7/9 inhibition by kinase inhibitors. (A) Cocrystal structure of TG003 in BRD7 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0Q">6V0Q</a>) and (B) in BRD9 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0S">6V0S</a>). (C) Superposition of both structures. (D) Cocrystal structure of BRD9 liganded with sunitinib (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0X">6V0X</a>). (E) Superposition of TG003 (yellow) and sunitinib (orange) bound to the KAc site of BRD9. (F) Comparison of the binding mode of TG003 in the ATP site of CLK2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FYI">6FYI</a>) (left panel) and the KAc site of BRD9 (right panel). The kinase hinge region is colored in orange and other kinase residues are colored in pink. (G) Comparison of the binding mode of sunitinib in the ATP site of ITK (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MIY">3MIY</a>) (left panel) and the KAc site of BRD9 (right panel). 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> and <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf" class="ext-link">Supporting Information</a> Figures S3 and S4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b01980/20200323/images/large/jm9b01980_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01980&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i17">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50364" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50364" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 40 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+Recognition+and+Large-Scale+Structural+Analysis+of+the+Human+Bromodomain+Family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lgxnxj-WF3uzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520Recognition%2520and%2520Large-Scale%2520Structural%2520Analysis%2520of%2520the%2520Human%2520Bromodomain%2520Family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.-F.</span></span> <span> </span><span class="NLM_article-title">Brd4 Maintains Constitutively Active Nf-Kappab in Cancer Cells by Binding to Acetylated Rela</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">2395</span>– <span class="NLM_lpage">2404</span>, <span class="refDoi"> DOI: 10.1038/onc.2013.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fonc.2013.179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=23686307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVGktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2014&pages=2395-2404&author=Z.+Zouauthor=B.+Huangauthor=X.+Wuauthor=H.+Zhangauthor=J.+Qiauthor=J.+Bradnerauthor=S.+Nairauthor=L.-F.+Chen&title=Brd4+Maintains+Constitutively+Active+Nf-Kappab+in+Cancer+Cells+by+Binding+to+Acetylated+Rela&doi=10.1038%2Fonc.2013.179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 maintains constitutively active NF-κB in cancer cells by binding to acetylated RelA</span></div><div class="casAuthors">Zou, Z.; Huang, B.; Wu, X.; Zhang, H.; Qi, J.; Bradner, J.; Nair, S.; Chen, L.-F.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2395-2404</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acetylation of the RelA subunit of NF-κB at lysine-310 regulates the transcriptional activation of NF-κB target genes and contributes to maintaining constitutively active NF-κB in tumors.  Bromodomain-contg. factor Brd4 has been shown to bind to acetylated lysine-310 (AcLys310) and to regulate the transcriptional activity of NF-κB, but the role of this binding in maintaining constitutively active NF-κB in tumors remains elusive.  In this study, we demonstrate the structural basis for the binding of bromodomains (BDs) of bromodomain-contg. protein 4 (Brd4) to AcLys310 and identify the BD inhibitor JQ1 as an effective small mol. to block this interaction.  JQ1 suppresses TNF-α-mediated NF-κB activation and NF-κB-dependent target gene expression.  In addn., JQ1 inhibits the proliferation and transformation potential of A549 lung cancer cells and suppresses the tumorigenicity of A549 cells in severe combined immunodeficiency mice.  Furthermore, we demonstrate that depletion of Brd4 or treatment of cells with JQ1 induces the ubiquitination and degrdn. of the constitutively active nuclear form of RelA.  Our results identify a novel function of Brd4 in maintaining the persistently active form of NF-κB found in tumors, and they suggest that interference with the interaction between acetylated RelA and Brd4 could be a potential therapeutic approach for the treatment of NF-κB-driven cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreyXshjsnx27Vg90H21EOLACvtfcHk0lgxnxj-WF3uzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvVGktro%253D&md5=b2cc674e107dd84eb147594a061d2550</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fonc.2013.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2013.179%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%26aulast%3DNair%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.-F.%26atitle%3DBrd4%2520Maintains%2520Constitutively%2520Active%2520Nf-Kappab%2520in%2520Cancer%2520Cells%2520by%2520Binding%2520to%2520Acetylated%2520Rela%26jtitle%3DOncogene%26date%3D2014%26volume%3D33%26spage%3D2395%26epage%3D2404%26doi%3D10.1038%2Fonc.2013.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Functions of Bromodomain-Containing Proteins and Their Roles in Homeostasis and Cancer</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/nrm.2016.143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fnrm.2016.143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=28053347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=246-262&author=T.+Fujisawaauthor=P.+Filippakopoulos&title=Functions+of+Bromodomain-Containing+Proteins+and+Their+Roles+in+Homeostasis+and+Cancer&doi=10.1038%2Fnrm.2016.143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Functions of bromodomain-containing proteins and their roles in homeostasis and cancer</span></div><div class="casAuthors">Fujisawa, Takao; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">246-262</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Bromodomains (BRDs) are evolutionarily conserved protein-protein interaction modules that are found in a wide range of proteins with diverse catalytic and scaffolding functions and are present in most tissues.  BRDs selectively recognize and bind to acetylated Lys residues - particularly in histones - and thereby have important roles in the regulation of gene expression.  BRD-contg. proteins are frequently dysregulated in cancer, they participate in gene fusions that generate diverse, frequently oncogenic proteins, and many cancer-causing mutations have been mapped to the BRDs themselves.  Importantly, BRDs can be targeted by small-mol. inhibitors, which has stimulated many translational research projects that seek to attenuate the aberrant functions of BRD-contg. proteins in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-drLvQK1rvLVg90H21EOLACvtfcHk0lgjrn6he3ETrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Kiuw%253D%253D&md5=cbccdee8423bad7a426c9cea6566151f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2016.143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2016.143%26sid%3Dliteratum%253Aachs%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DFunctions%2520of%2520Bromodomain-Containing%2520Proteins%2520and%2520Their%2520Roles%2520in%2520Homeostasis%2520and%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D246%26epage%3D262%26doi%3D10.1038%2Fnrm.2016.143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">Targeting Cancer Cells with Bet Bromodomain Inhibitors</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">a026674</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a026674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1101%2Fcshperspect.a026674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=28213432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisFKqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=a026674&author=Y.+Xuauthor=C.+R.+Vakoc&title=Targeting+Cancer+Cells+with+Bet+Bromodomain+Inhibitors&doi=10.1101%2Fcshperspect.a026674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer cells with BET bromodomain inhibitors</span></div><div class="casAuthors">Xu, Yali; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">a026674/1-a026674/18</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Cancer cells are often hypersensitive to the targeting of transcriptional regulators, which may reflect the deregulated gene expression programs that underlie malignant transformation.  One of the most prominent transcriptional vulnerabilities in human cancer to emerge in recent years is the bromodomain and extraterminal (BET) family of proteins, which are coactivators that link acetylated transcription factors and histones to the activation of RNA polymerase II.  Despite unclear mechanisms underlying the gene specificity of BET protein function, small mols. targeting these regulators preferentially suppress the transcription of cancer-promoting genes.  As a consequence, BET inhibitors elicit anticancer activity in numerous malignant contexts at doses that can be tolerated by normal tissues, a finding supported by animal studies and by phase I clin. trials in human cancer patients.  In this review, we will discuss the remarkable, and often perplexing, therapeutic effects of BET bromodomain inhibition in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgHFLVxo-LQrVg90H21EOLACvtfcHk0lgjrn6he3ETrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisFKqtbw%253D&md5=e1bdf55b78059e868867aad8d35e737f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a026674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a026674%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DTargeting%2520Cancer%2520Cells%2520with%2520Bet%2520Bromodomain%2520Inhibitors%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2017%26volume%3D7%26spage%3Da026674%26doi%3D10.1101%2Fcshperspect.a026674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd</span></span> <span> </span><span class="NLM_article-title">Bromodomains: A New Target Class for Drug Development</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">628</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0030-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fs41573-019-0030-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=31273347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlWitbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=609-628&author=A.+G.+Cochranauthor=A.+R.+Coneryauthor=R.+J.+Sims&title=Bromodomains%3A+A+New+Target+Class+for+Drug+Development&doi=10.1038%2Fs41573-019-0030-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains: a new target class for drug development</span></div><div class="casAuthors">Cochran, Andrea G.; Conery, Andrew R.; Sims, III, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">609-628</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Less than a decade ago, it was shown that bromodomains, acetyl lysine 'reader' modules found in proteins with varied functions, were highly tractable small-mol. targets.  This is an unusual property for protein-protein or protein-peptide interaction domains, and it prompted a wave of chem. probe discovery to understand the biol. potential of new agents that targeted bromodomains.  The original examples, inhibitors of the bromodomain and extra-terminal (BET) class of bromodomains, showed enticing anti-inflammatory and anticancer activities, and several compds. have since advanced to human clin. trials.  Here, we review the current state of BET inhibitor biol. in relation to clin. development, and we discuss the next wave of bromodomain inhibitors with clin. potential in oncol. and non-oncol. indications.  The lessons learned from BET inhibitor programs should affect efforts to develop drugs that target non-BET bromodomains and other epigenetic readers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6QwxqowW9NbVg90H21EOLACvtfcHk0lgjrn6he3ETrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlWitbnL&md5=88e56eec8edeb7c325d28156ec131ed8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0030-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0030-7%26sid%3Dliteratum%253Aachs%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26atitle%3DBromodomains%253A%2520A%2520New%2520Target%2520Class%2520for%2520Drug%2520Development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D609%26epage%3D628%26doi%3D10.1038%2Fs41573-019-0030-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mashtalir, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Avino, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zullow, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubiak, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St Pierre, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poynter, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassel, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranish, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span> <span> </span><span class="NLM_article-title">Modular Organization and Assembly of Swi/Snf Family Chromatin Remodeling Complexes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">1272</span>– <span class="NLM_lpage">1288</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.09.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1016%2Fj.cell.2018.09.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=30343899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFKjtbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2018&pages=1272-1288&author=N.+Mashtalirauthor=A.+R.+D%E2%80%99Avinoauthor=B.+C.+Michelauthor=J.+Luoauthor=J.+Panauthor=J.+E.+Ottoauthor=H.+J.+Zullowauthor=Z.+M.+McKenzieauthor=R.+L.+Kubiakauthor=R.+St+Pierreauthor=A.+M.+Valenciaauthor=S.+J.+Poynterauthor=S.+H.+Casselauthor=J.+A.+Ranishauthor=C.+Kadoch&title=Modular+Organization+and+Assembly+of+Swi%2FSnf+Family+Chromatin+Remodeling+Complexes&doi=10.1016%2Fj.cell.2018.09.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Modular organization and assembly of SWI/SNF family chromatin remodeling complexes</span></div><div class="casAuthors">Mashtalir, Nazar; D'Avino, Andrew R.; Michel, Brittany C.; Luo, Jie; Pan, Joshua; Otto, Jordan E.; Zullow, Hayley J.; McKenzie, Zachary M.; Kubiak, Rachel L.; St. Pierre, Roodolph; Valencia, Alfredo M.; Poynter, Steven J.; Cassel, Seth H.; Ranish, Jeffrey A.; Kadoch, Cigall</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1272-1288.e20</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mammalian SWI/SNF (mSWI/SNF) ATP-dependent chromatin remodeling complexes are multi-subunit mol. machines that play vital roles in regulating genomic architecture and are frequently disrupted in human cancer and developmental disorders.  To date, the modular organization and pathways of assembly of these chromatin regulators remain unknown, presenting a major barrier to structural and functional detn.  Here, we elucidated the architecture and assembly pathway across 3 classes of mSWI/SNF complexes [canonical BRG1/BRM-assocd. factor (BAF), polybromo-assocd. BAF (PBAF)], and newly defined ncBAF complexes-and defined the requirement of each subunit for complex formation and stability.  Using affinity purifn. of endogenous complexes from mammalian and Drosophila cells coupled with crosslinking mass spectrometry (CX-MS) and mutagenesis, we uncovered 3 distinct and evolutionarily conserved modules, their organization, and the temporal incorporation of these modules into each complete mSWI/SNF complex class.  Finally, we mapped human disease-assocd. mutations within subunits and modules, defining specific topol. regions that were affected upon subunit perturbation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0-bunBPiOqrVg90H21EOLACvtfcHk0lhpnG8hUsCWtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFKjtbnL&md5=7e5b6372e8507a346b044c12903e3d3f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.09.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.09.032%26sid%3Dliteratum%253Aachs%26aulast%3DMashtalir%26aufirst%3DN.%26aulast%3DD%25E2%2580%2599Avino%26aufirst%3DA.%2BR.%26aulast%3DMichel%26aufirst%3DB.%2BC.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DPan%26aufirst%3DJ.%26aulast%3DOtto%26aufirst%3DJ.%2BE.%26aulast%3DZullow%26aufirst%3DH.%2BJ.%26aulast%3DMcKenzie%26aufirst%3DZ.%2BM.%26aulast%3DKubiak%26aufirst%3DR.%2BL.%26aulast%3DSt%2BPierre%26aufirst%3DR.%26aulast%3DValencia%26aufirst%3DA.%2BM.%26aulast%3DPoynter%26aufirst%3DS.%2BJ.%26aulast%3DCassel%26aufirst%3DS.%2BH.%26aulast%3DRanish%26aufirst%3DJ.%2BA.%26aulast%3DKadoch%26aufirst%3DC.%26atitle%3DModular%2520Organization%2520and%2520Assembly%2520of%2520Swi%252FSnf%2520Family%2520Chromatin%2520Remodeling%2520Complexes%26jtitle%3DCell%26date%3D2018%26volume%3D175%26spage%3D1272%26epage%3D1288%26doi%3D10.1016%2Fj.cell.2018.09.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hohmann, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minder, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, J.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottschamel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">Sensitivity and Engineered Resistance of Myeloid Leukemia Cells to Brd9 Inhibition</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">679</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fnchembio.2115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=27376689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKntLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=672-679&author=A.+F.+Hohmannauthor=L.+J.+Martinauthor=J.+L.+Minderauthor=J.-S.+Roeauthor=J.+Shiauthor=S.+Steurerauthor=G.+Baderauthor=D.+McConnellauthor=M.+Pearsonauthor=T.+Gerstbergerauthor=T.+Gottschamelauthor=D.+Thompsonauthor=Y.+Suzukiauthor=M.+Koeglauthor=C.+R.+Vakoc&title=Sensitivity+and+Engineered+Resistance+of+Myeloid+Leukemia+Cells+to+Brd9+Inhibition&doi=10.1038%2Fnchembio.2115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Sensitivity and engineered resistance of myeloid leukemia cells to BRD9 inhibition</span></div><div class="casAuthors">Hohmann, Anja F.; Martin, Laetitia J.; Minder, Jessica L.; Roe, Jae-Seok; Shi, Junwei; Steurer, Steffen; Bader, Gerd; McConnell, Darryl; Pearson, Mark; Gerstberger, Thomas; Gottschamel, Teresa; Thompson, Diane; Suzuki, Yutaka; Koegl, Manfred; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">672-679</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we show that acute myeloid leukemia (AML) cells require the BRD9 subunit of the SWI-SNF chromatin-remodeling complex to sustain MYC transcription, rapid cell proliferation and a block in differentiation.  Based on these observations, we derived small-mol. inhibitors of the BRD9 bromodomain that selectively suppress the proliferation of mouse and human AML cell lines.  To establish these effects as on-target, we engineered a bromodomain-swap allele of BRD9 that retains functionality despite a radically altered bromodomain pocket.  Expression of this allele in AML cells confers resistance to the antiproliferative effects of our compd. series, thus establishing BRD9 as the relevant cellular target.  Furthermore, we used an analogous domain-swap strategy to generate an inhibitor-resistant allele of EZH2.  To our knowledge, our study provides the first evidence for a role of BRD9 in cancer and reveals a simple genetic strategy for constructing resistance alleles to demonstrate on-target activity of chem. probes in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr8rWWOXSB37Vg90H21EOLACvtfcHk0lhpnG8hUsCWtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKntLfN&md5=12c2918c7bdfacdb1c904d55bfb6afb5</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2115%26sid%3Dliteratum%253Aachs%26aulast%3DHohmann%26aufirst%3DA.%2BF.%26aulast%3DMartin%26aufirst%3DL.%2BJ.%26aulast%3DMinder%26aufirst%3DJ.%2BL.%26aulast%3DRoe%26aufirst%3DJ.-S.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DMcConnell%26aufirst%3DD.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DGottschamel%26aufirst%3DT.%26aulast%3DThompson%26aufirst%3DD.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DSensitivity%2520and%2520Engineered%2520Resistance%2520of%2520Myeloid%2520Leukemia%2520Cells%2520to%2520Brd9%2520Inhibition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D672%26epage%3D679%26doi%3D10.1038%2Fnchembio.2115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemming, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cagney, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Mierlo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltissen, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fröhling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">Targeted Degradation of Brd9 Reverses Oncogenic Gene Expression in Synovial Sarcoma</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e41305</span> <span class="refDoi"> DOI: 10.7554/elife.41305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.7554%2Felife.41305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=30431433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyqs7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=G.+L.+Brienauthor=D.+Remillardauthor=J.+Shiauthor=M.+L.+Hemmingauthor=J.+Chabonauthor=K.+Wynneauthor=E.+T.+Dillonauthor=G.+Cagneyauthor=G.+Van+Mierloauthor=M.+P.+Baltissenauthor=M.+Vermeulenauthor=J.+Qiauthor=S.+Fr%C3%B6hlingauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=C.+R.+Vakocauthor=S.+A.+Armstrong&title=Targeted+Degradation+of+Brd9+Reverses+Oncogenic+Gene+Expression+in+Synovial+Sarcoma&doi=10.7554%2Felife.41305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted degradation of BRD9 reverses oncogenic gene expression in synovial sarcoma</span></div><div class="casAuthors">Brien, Gerard L.; Remillard, David; Shi, Junwei; Hemming, Matthew L.; Chabon, Jonathon; Wynne, Kieran; Dillon, Euge Ne T.; Cagney, Gerard; Van Mierlo, Guido; Baltissen, Marijke P.; Vermeulen, Michiel; Qi, Jun; Froehling, Stefan; Gray, Nathanael S.; Bradner, James E.; Vakoc, Christopher R.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e41305/1-e41305/26</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Synovial sarcoma tumors contain a characteristic fusion protein, SS18-SSX, which drives disease development.  Targeting oncogenic fusion proteins presents an attractive therapeutic opportunity.  However, SS18-SSX has proven intractable for therapeutic intervention.  Using a domain-focused CRISPR screen we identified the bromodomain of BRD9 as a crit. functional dependency in synovial sarcoma.  BRD9 is a component of SS18-SSX contg. BAF complexes in synovial sarcoma cells; and integration of BRD9 into these complexes is crit. for cell growth.  Moreover BRD9 and SS18-SSX co-localize extensively on the synovial sarcoma genome.  Remarkably, synovial sarcoma cells are highly sensitive to a novel small mol. degrader of BRD9, while other sarcoma subtypes are unaffected.  Degrdn. of BRD9 induces downregulation of oncogenic transcriptional programs and inhibits tumor progression in vivo.  We demonstrate that BRD9 supports oncogenic mechanisms underlying the SS18-SSX fusion in synovial sarcoma and highlight targeted degrdn. of BRD9 as a potential therapeutic opportunity in this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfd6-D9YlmrrVg90H21EOLACvtfcHk0lhpnG8hUsCWtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyqs7fF&md5=cbda64457406cd0d8fbd1211678b04b3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.7554%2Felife.41305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252Felife.41305%26sid%3Dliteratum%253Aachs%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DHemming%26aufirst%3DM.%2BL.%26aulast%3DChabon%26aufirst%3DJ.%26aulast%3DWynne%26aufirst%3DK.%26aulast%3DDillon%26aufirst%3DE.%2BT.%26aulast%3DCagney%26aufirst%3DG.%26aulast%3DVan%2BMierlo%26aufirst%3DG.%26aulast%3DBaltissen%26aufirst%3DM.%2BP.%26aulast%3DVermeulen%26aufirst%3DM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DFr%25C3%25B6hling%26aufirst%3DS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DTargeted%2520Degradation%2520of%2520Brd9%2520Reverses%2520Oncogenic%2520Gene%2520Expression%2520in%2520Synovial%2520Sarcoma%26jtitle%3DElife%26date%3D2018%26volume%3D7%26doi%3D10.7554%2Felife.41305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frühwald, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerl, K.</span></span> <span> </span><span class="NLM_article-title">Brd9 Inhibition, Alone or in Combination with Cytostatic Compounds as a Therapeutic Approach in Rhabdoid Tumors</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1537</span>, <span class="refDoi"> DOI: 10.3390/ijms18071537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.3390%2Fijms18071537" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=1537&author=K.+Kr%C3%A4merauthor=N.+Morenoauthor=M.+Fr%C3%BChwaldauthor=K.+Kerl&title=Brd9+Inhibition%2C+Alone+or+in+Combination+with+Cytostatic+Compounds+as+a+Therapeutic+Approach+in+Rhabdoid+Tumors&doi=10.3390%2Fijms18071537"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3390%2Fijms18071537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18071537%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25A4mer%26aufirst%3DK.%26aulast%3DMoreno%26aufirst%3DN.%26aulast%3DFr%25C3%25BChwald%26aufirst%3DM.%26aulast%3DKerl%26aufirst%3DK.%26atitle%3DBrd9%2520Inhibition%252C%2520Alone%2520or%2520in%2520Combination%2520with%2520Cytostatic%2520Compounds%2520as%2520a%2520Therapeutic%2520Approach%2520in%2520Rhabdoid%2520Tumors%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26spage%3D1537%26doi%3D10.3390%2Fijms18071537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantovani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tocco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holstege, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerkhoven, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonkers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voorhoeve, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agami, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Sal, G.</span></span> <span> </span><span class="NLM_article-title">Brd7 Is a Candidate Tumour Suppressor Gene Required for P53 Function</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">380</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1038/ncb2038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fncb2038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=20228809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVyjurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=380-389&author=J.+Drostauthor=F.+Mantovaniauthor=F.+Toccoauthor=R.+Elkonauthor=A.+Comelauthor=H.+Holstegeauthor=R.+Kerkhovenauthor=J.+Jonkersauthor=P.+M.+Voorhoeveauthor=R.+Agamiauthor=G.+Del+Sal&title=Brd7+Is+a+Candidate+Tumour+Suppressor+Gene+Required+for+P53+Function&doi=10.1038%2Fncb2038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">BRD7 is a candidate tumour suppressor gene required for p53 function</span></div><div class="casAuthors">Drost, Jarno; Mantovani, Fiamma; Tocco, Francesca; Elkon, Ran; Comel, Anna; Holstege, Henne; Kerkhoven, Ron; Jonkers, Jos; Voorhoeve, P. Mathijs; Agami, Reuven; Del Sal, Giannino</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">380-389</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oncogene-induced senescence is a p53-dependent defense mechanism against uncontrolled proliferation.  Consequently, many human tumors harbor p53 mutations and others show a dysfunctional p53 pathway, frequently by unknown mechanisms.  Here the authors identify BRD7 (bromodomain-contg. 7) as a protein whose inhibition allows full neoplastic transformation in the presence of wild-type p53.  In human breast tumors harboring wild-type, but not mutant, p53 the BRD7 gene locus was frequently deleted and low BRD7 expression was found in a subgroup of tumors.  Functionally, BRD7 is required for efficient p53-mediated transcription of a subset of target genes.  BRD7 interacts with p53 and p300 and is recruited to target gene promoters, affecting histone acetylation, p53 acetylation and promoter activity.  Thus, BRD7 suppresses tumorigenicity by serving as a p53 cofactor required for the efficient induction of p53-dependent oncogene-induced senescence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1grCCyCF9MrVg90H21EOLACvtfcHk0lgVrTJfRYpK7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVyjurw%253D&md5=676cb9cbc5a8d157ac85d76c42ba8007</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fncb2038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2038%26sid%3Dliteratum%253Aachs%26aulast%3DDrost%26aufirst%3DJ.%26aulast%3DMantovani%26aufirst%3DF.%26aulast%3DTocco%26aufirst%3DF.%26aulast%3DElkon%26aufirst%3DR.%26aulast%3DComel%26aufirst%3DA.%26aulast%3DHolstege%26aufirst%3DH.%26aulast%3DKerkhoven%26aufirst%3DR.%26aulast%3DJonkers%26aufirst%3DJ.%26aulast%3DVoorhoeve%26aufirst%3DP.%2BM.%26aulast%3DAgami%26aufirst%3DR.%26aulast%3DDel%2BSal%26aufirst%3DG.%26atitle%3DBrd7%2520Is%2520a%2520Candidate%2520Tumour%2520Suppressor%2520Gene%2520Required%2520for%2520P53%2520Function%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2010%26volume%3D12%26spage%3D380%26epage%3D389%26doi%3D10.1038%2Fncb2038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussenzveig, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontoura, B. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">Nucleoporin Seh1 Interacts with Olig2/Brd7 to Promote Oligodendrocyte Differentiation and Myelination</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">587</span>– <span class="NLM_lpage">601</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2019.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1016%2Fj.neuron.2019.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=30876848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVegtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2019&pages=587-601&author=Z.+Liuauthor=M.+Yanauthor=Y.+Liangauthor=M.+Liuauthor=K.+Zhangauthor=D.+Shaoauthor=R.+Jiangauthor=L.+Liauthor=C.+Wangauthor=D.+R.+Nussenzveigauthor=K.+Zhangauthor=S.+Chenauthor=C.+Zhongauthor=W.+Moauthor=B.+M.+A.+Fontouraauthor=L.+Zhang&title=Nucleoporin+Seh1+Interacts+with+Olig2%2FBrd7+to+Promote+Oligodendrocyte+Differentiation+and+Myelination&doi=10.1016%2Fj.neuron.2019.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleoporin Seh1 Interacts with Olig2/Brd7 to Promote Oligodendrocyte Differentiation and Myelination</span></div><div class="casAuthors">Liu, Zhixiong; Yan, Minbiao; Liang, Yaoji; Liu, Min; Zhang, Kun; Shao, Dandan; Jiang, Rencai; Li, Li; Wang, Chaomeng; Nussenzveig, Daniel R.; Zhang, Kunkun; Chen, Shaoxuan; Zhong, Chuanqi; Mo, Wei; Fontoura, Beatriz M. A.; Zhang, Liang</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">587-601.e7</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Nucleoporins (Nups) are involved in neural development, and alterations in Nup genes are linked to human neurol. diseases.  However, physiol. functions of specific Nups and the underlying mechanisms involved in these processes remain elusive.  Here, we show that tissue-specific depletion of the nucleoporin Seh1 causes dramatic myelination defects in the CNS.  Although proliferation is not altered in Seh1-deficient oligodendrocyte progenitor cells (OPCs), they fail to differentiate into mature oligodendrocytes, which impairs myelin prodn. and remyelination after demyelinating injury.  Genome-wide analyses show that Seh1 regulates a core myelinogenic regulatory network and establishes an accessible chromatin landscape.  Mechanistically, Seh1 regulates OPCs differentiation by assembling Olig2 and Brd7 into a transcription complex at nuclear periphery.  Together, our results reveal that Seh1 is required for oligodendrocyte differentiation and myelination by promoting assembly of an Olig2-dependent transcription complex and define a nucleoporin as a key player in the CNS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoem6E0dk5Q-rVg90H21EOLACvtfcHk0lgVrTJfRYpK7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVegtbg%253D&md5=9020b496dc7dac04e003d67f72e7d35e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2019.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2019.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DShao%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DNussenzveig%26aufirst%3DD.%2BR.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DC.%26aulast%3DMo%26aufirst%3DW.%26aulast%3DFontoura%26aufirst%3DB.%2BM.%2BA.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DNucleoporin%2520Seh1%2520Interacts%2520with%2520Olig2%252FBrd7%2520to%2520Promote%2520Oligodendrocyte%2520Differentiation%2520and%2520Myelination%26jtitle%3DNeuron%26date%3D2019%26volume%3D102%26spage%3D587%26epage%3D601%26doi%3D10.1016%2Fj.neuron.2019.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span> <span> </span><span class="NLM_article-title">Knockout of Brd7 Results in Impaired Spermatogenesis and Male Infertility</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">21776</span>, <span class="refDoi"> DOI: 10.1038/srep21776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fsrep21776" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=26878912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC28XislOqtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=21776&author=H.+Wangauthor=R.+Zhaoauthor=C.+Guoauthor=S.+Jiangauthor=J.+Yangauthor=Y.+Xuauthor=Y.+Liuauthor=L.+Fanauthor=W.+Xiongauthor=J.+Maauthor=S.+Pengauthor=Z.+Zengauthor=Y.+Zhouauthor=X.+Liauthor=Z.+Liauthor=X.+Liauthor=D.+C.+Schmittauthor=M.+Tanauthor=G.+Liauthor=M.+Zhou&title=Knockout+of+Brd7+Results+in+Impaired+Spermatogenesis+and+Male+Infertility&doi=10.1038%2Fsrep21776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Knockout of BRD7 results in impaired spermatogenesis and male infertility</span></div><div class="casAuthors">Wang, Heran; Zhao, Ran; Guo, Chi; Jiang, Shihe; Yang, Jing; Xu, Yang; Liu, Yukun; Fan, Liqing; Xiong, Wei; Ma, Jian; Peng, Shuping; Zeng, Zhaoyang; Zhou, Yanhong; Li, Xiayu; Li, Zheng; Li, Xiaoling; Schmitt, David C.; Tan, Ming; Li, Guiyuan; Zhou, Ming</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">21776</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">BRD7 was originally identified as a novel bromodomain gene and a potential transcriptional factor.  BRD7 was found to be extensively expressed in multiple mouse tissues but was highly expressed in the testis.  Furthermore, BRD7 was located in germ cells during multiple stages of spermatogenesis, ranging from the pachytene to the round spermatid stage.  Homozygous knockout of BRD7 (BRD7-/-) resulted in complete male infertility and spermatogenesis defects, including deformed acrosomal formation, degenerative elongating spermatids and irregular head morphol. in postmeiotic germ cells in the seminiferous epithelium, which led to the complete arrest of spermatogenesis at step 13.  Moreover, a high ratio of apoptosis was detd. by TUNEL anal., which was supported by high levels of the apoptosis markers annexin V and p53 in knockout testes.  Increased expression of the DNA damage maker λH2AX was also found in BRD7-/- mice, whereas DNA damage repair genes were down-regulated.  Furthermore, no or lower expression of BRD7 was detected in the testes of azoospermia patients exhibiting spermatogenesis arrest than that in control group.  These data demonstrate that BRD7 is involved in male infertility and spermatogenesis in mice, and BRD7 defect might be assocd. with the occurrence and development of human azoospermia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmHUItFrwX3rVg90H21EOLACvtfcHk0lhb0_0hPQhc3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XislOqtrY%253D&md5=6979ca806bfbf56c27fda6c325710cb2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsrep21776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep21776%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DXiong%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DSchmitt%26aufirst%3DD.%2BC.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DM.%26atitle%3DKnockout%2520of%2520Brd7%2520Results%2520in%2520Impaired%2520Spermatogenesis%2520and%2520Male%2520Infertility%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D21776%26doi%3D10.1038%2Fsrep21776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranish, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">Proteomic and Bioinformatic Analysis of Mammalian Swi/Snf Complexes Identifies Extensive Roles in Human Malignancy</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">592</span>– <span class="NLM_lpage">601</span>, <span class="refDoi"> DOI: 10.1038/ng.2628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fng.2628" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=23644491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=592-601&author=C.+Kadochauthor=D.+C.+Hargreavesauthor=C.+Hodgesauthor=L.+Eliasauthor=L.+Hoauthor=J.+Ranishauthor=G.+R.+Crabtree&title=Proteomic+and+Bioinformatic+Analysis+of+Mammalian+Swi%2FSnf+Complexes+Identifies+Extensive+Roles+in+Human+Malignancy&doi=10.1038%2Fng.2628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomic and bioinformatic analysis of mammalian SWI/SNF complexes identifies extensive roles in human malignancy</span></div><div class="casAuthors">Kadoch, Cigall; Hargreaves, Diana C.; Hodges, Courtney; Elias, Laura; Ho, Lena; Ranish, Jeff; Crabtree, Gerald R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">592-601</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Subunits of mammalian SWI/SNF (mSWI/SNF or BAF) complexes have recently been implicated as tumor suppressors in human malignancies.  To understand the full extent of their involvement, we conducted a proteomic anal. of endogenous mSWI/SNF complexes, which identified several new dedicated, stable subunits not found in yeast SWI/SNF complexes, including BCL7A, BCL7B and BCL7C, BCL11A and BCL11B, BRD9 and SS18.  Incorporating these new members, we detd. mSWI/SNF subunit mutation frequency in exome and whole-genome sequencing studies of primary human tumors.  Notably, mSWI/SNF subunits are mutated in 19.6% of all human tumors reported in 44 studies.  Our anal. suggests that specific subunits protect against cancer in specific tissues.  In addn., mutations affecting more than one subunit, defined here as compd. heterozygosity, are prevalent in certain cancers.  Our studies demonstrate that mSWI/SNF is the most frequently mutated chromatin-regulatory complex (CRC) in human cancer, exhibiting a broad mutation pattern, similar to that of TP53.  Thus, proper functioning of polymorphic BAF complexes may constitute a major mechanism of tumor suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP873DCpn7JrVg90H21EOLACvtfcHk0lhb0_0hPQhc3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVart7s%253D&md5=0f49cf3c59e7bc04b74970fe656b1e17</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fng.2628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2628%26sid%3Dliteratum%253Aachs%26aulast%3DKadoch%26aufirst%3DC.%26aulast%3DHargreaves%26aufirst%3DD.%2BC.%26aulast%3DHodges%26aufirst%3DC.%26aulast%3DElias%26aufirst%3DL.%26aulast%3DHo%26aufirst%3DL.%26aulast%3DRanish%26aufirst%3DJ.%26aulast%3DCrabtree%26aufirst%3DG.%2BR.%26atitle%3DProteomic%2520and%2520Bioinformatic%2520Analysis%2520of%2520Mammalian%2520Swi%252FSnf%2520Complexes%2520Identifies%2520Extensive%2520Roles%2520in%2520Human%2520Malignancy%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D592%26epage%3D601%26doi%3D10.1038%2Fng.2628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P. G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieira, L. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singleton, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baronio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. J.</span></span> <span> </span><span class="NLM_article-title">Lp99: Discovery and Synthesis of the First Selective Brd7/9 Bromodomain Inhibitor</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">6217</span>– <span class="NLM_lpage">6221</span>, <span class="refDoi"> DOI: 10.1002/anie.201501394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1002%2Fanie.201501394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlGqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=6217-6221&author=P.+G.+K.+Clarkauthor=L.+C.+C.+Vieiraauthor=C.+Tallantauthor=O.+Fedorovauthor=D.+C.+Singletonauthor=C.+M.+Rogersauthor=O.+P.+Monteiroauthor=J.+M.+Bennettauthor=R.+Baronioauthor=S.+M%C3%BCllerauthor=D.+L.+Danielsauthor=J.+M%C3%A9ndezauthor=S.+Knappauthor=P.+E.+Brennanauthor=D.+J.+Dixon&title=Lp99%3A+Discovery+and+Synthesis+of+the+First+Selective+Brd7%2F9+Bromodomain+Inhibitor&doi=10.1002%2Fanie.201501394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">LP99: Discovery and Synthesis of the First Selective BRD7/9 Bromodomain Inhibitor</span></div><div class="casAuthors">Clark, Peter G. K.; Vieira, Lucas C. C.; Tallant, Cynthia; Fedorov, Oleg; Singleton, Dean C.; Rogers, Catherine M.; Monteiro, Octovia P.; Bennett, James M.; Baronio, Roberta; Mueller, Susanne; Daniels, Danette L.; Mendez, Jacqui; Knapp, Stefan; Brennan, Paul E.; Dixon, Darren J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6217-6221</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. proteins BRD9 and BRD7 are part of the human SWI/SNF chromatin-remodeling complexes BAF and PBAF.  To date, no selective inhibitor for BRD7/9 has been reported despite its potential value as a biol. tool or as a lead for future therapeutics.  The quinolone-fused lactam LP99 (I) is now reported as the first potent and selective inhibitor of the BRD7 and BRD9 bromodomains.  Development of LP99 from a fragment hit was expedited through balancing structure-based inhibitor design and biophys. characterization against tractable chem. synthesis: Complexity-building nitro-Mannich/lactamization cascade processes allowed for early structure-activity relationship studies whereas an enantioselective organocatalytic nitro-Mannich reaction enabled the synthesis of the lead scaffold in enantioenriched form and on scale.  This epigenetic probe was shown to inhibit the assocn. of BRD7 and BRD9 to acetylated histones in vitro and in cells.  Moreover, LP99 was used to demonstrate that BRD7/9 plays a role in regulating pro-inflammatory cytokine secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplfOmbNMpdPbVg90H21EOLACvtfcHk0lhb0_0hPQhc3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlGqsrg%253D&md5=9dee2b352045f7ae5efcbdc83f240a3f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fanie.201501394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201501394%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DP.%2BG.%2BK.%26aulast%3DVieira%26aufirst%3DL.%2BC.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DSingleton%26aufirst%3DD.%2BC.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DBaronio%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DDixon%26aufirst%3DD.%2BJ.%26atitle%3DLp99%253A%2520Discovery%2520and%2520Synthesis%2520of%2520the%2520First%2520Selective%2520Brd7%252F9%2520Bromodomain%2520Inhibitor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D6217%26epage%3D6221%26doi%3D10.1002%2Fanie.201501394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vartanian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duplessis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokoe, D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Bromodomain-Containing Protein 9 for the Prevention of Epigenetically-Defined Drug Resistance</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3534</span>– <span class="NLM_lpage">3541</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.05.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1016%2Fj.bmcl.2017.05.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=28606761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVSjtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=3534-3541&author=T.+D.+Crawfordauthor=S.+Vartanianauthor=A.+C%C3%B4t%C3%A9author=S.+Bellonauthor=M.+Duplessisauthor=E.+M.+Flynnauthor=M.+Hewittauthor=H.-R.+Huangauthor=J.+R.+Kieferauthor=J.+Murrayauthor=C.+G.+Nasveschukauthor=E.+Pardoauthor=F.+A.+Romeroauthor=P.+Sandyauthor=Y.+Tangauthor=A.+M.+Taylorauthor=V.+Tsuiauthor=J.+Wangauthor=S.+Wangauthor=L.+Zawadzkeauthor=B.+K.+Albrechtauthor=S.+R.+Magnusonauthor=A.+G.+Cochranauthor=D.+Stokoe&title=Inhibition+of+Bromodomain-Containing+Protein+9+for+the+Prevention+of+Epigenetically-Defined+Drug+Resistance&doi=10.1016%2Fj.bmcl.2017.05.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of bromodomain-containing protein 9 for the prevention of epigenetically-defined drug resistance</span></div><div class="casAuthors">Crawford, Terry D.; Vartanian, Steffan; Cote, Alexandre; Bellon, Steve; Duplessis, Martin; Flynn, E. Megan; Hewitt, Michael; Huang, Hon-Ren; Kiefer, James R.; Murray, Jeremy; Nasveschuk, Christopher G.; Pardo, Eneida; Romero, F. Anthony; Sandy, Peter; Tang, Yong; Taylor, Alexander M.; Tsui, Vickie; Wang, Jian; Wang, Shumei; Zawadzke, Laura; Albrecht, Brian K.; Magnuson, Steven R.; Cochran, Andrea G.; Stokoe, David</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3534-3541</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 9 (BRD9), an epigenetic "reader" of acetylated lysines on post-translationally modified histone proteins, is upregulated in multiple cancer cell lines.  To assess the functional role of BRD9 in cancer cell lines, the authors identified a small-mol. inhibitor of the BRD9 bromodomain.  Starting from a pyrrolopyridone lead, the authors used structure-based drug design to identify a potent and highly selective in vitro tool compd. 11, (GNE-375).  While this compd. showed minimal effects in cell viability or gene expression assays, it showed remarkable potency in preventing the emergence of a drug tolerant population in EGFR mutant PC9 cells treated with EGFR inhibitors.  Such tolerance has been linked to an altered epigenetic state, and 11 decreased BRD9 binding to chromatin, and this was assocd. with decreased expression of ALDH1A1, a gene previously shown to be important in drug tolerance.  BRD9 inhibitors may therefore show utility in preventing epigenetically-defined drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWdETG4wZY9rVg90H21EOLACvtfcHk0liaYSplK6oX0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVSjtb%252FO&md5=f489e702cfb41cfce9d3d570bfde87ed</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.05.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.05.063%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DVartanian%26aufirst%3DS.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHewitt%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DH.-R.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DStokoe%26aufirst%3DD.%26atitle%3DInhibition%2520of%2520Bromodomain-Containing%2520Protein%25209%2520for%2520the%2520Prevention%2520of%2520Epigenetically-Defined%2520Drug%2520Resistance%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D3534%26epage%3D3541%26doi%3D10.1016%2Fj.bmcl.2017.05.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockcroft, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiegen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohmann, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knesl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornigg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaaf, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of an in Vivo Active Selective Brd9 Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4462</span>– <span class="NLM_lpage">4475</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01865</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01865" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjt1Sit7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4462-4475&author=L.+J.+Martinauthor=M.+Koeglauthor=G.+Baderauthor=X.-L.+Cockcroftauthor=O.+Fedorovauthor=D.+Fiegenauthor=T.+Gerstbergerauthor=M.+H.+Hofmannauthor=A.+F.+Hohmannauthor=D.+Kesslerauthor=S.+Knappauthor=P.+Kneslauthor=S.+Korniggauthor=S.+M%C3%BCllerauthor=H.+Narauthor=C.+Rogersauthor=K.+Rumpelauthor=O.+Schaafauthor=S.+Steurerauthor=C.+Tallantauthor=C.+R.+Vakocauthor=M.+Zeebauthor=A.+Zoephelauthor=M.+Pearsonauthor=G.+Boehmeltauthor=D.+McConnell&title=Structure-Based+Design+of+an+in+Vivo+Active+Selective+Brd9+Inhibitor&doi=10.1021%2Facs.jmedchem.5b01865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of an in Vivo Active Selective BRD9 Inhibitor</span></div><div class="casAuthors">Martin, Laetitia J.; Koegl, Manfred; Bader, Gerd; Cockcroft, Xiao-Ling; Fedorov, Oleg; Fiegen, Dennis; Gerstberger, Thomas; Hofmann, Marco H.; Hohmann, Anja F.; Kessler, Dirk; Knapp, Stefan; Knesl, Petr; Kornigg, Stefan; Mueller, Susanne; Nar, Herbert; Rogers, Catherine; Rumpel, Klaus; Schaaf, Otmar; Steurer, Steffen; Tallant, Cynthia; Vakoc, Christopher R.; Zeeb, Markus; Zoephel, Andreas; Pearson, Mark; Boehmelt, Guido; McConnell, Darryl</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4462-4475</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Components of the chromatin remodelling switch/sucrose nonfermentable (SWI/SNF) complex are recurrently mutated in tumors, suggesting that altering the activity of the complex plays a role in oncogenesis.  However, the role that the individual subunits play in this process is not clear.  We set out to develop an inhibitor compd. targeting the bromodomain of BRD9 in order to evaluate its function within the SWI/SNF complex.  Here, we present the discovery and development of a potent and selective BRD9 bromodomain inhibitor series based on a new pyridinone-like scaffold.  Crystallog. information on the inhibitors bound to BRD9 guided their development with respect to potency for BRD9 and selectivity against BRD4.  These compds. modulate BRD9 bromodomain cellular function and display antitumor activity in an AML xenograft model.  Two chem. probes, BI-7273 (1) and BI-9564 (2), were identified that should prove to be useful in further exploring BRD9 bromodomain biol. in both in vitro and in vivo settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWHs4hA8bijbVg90H21EOLACvtfcHk0liaYSplK6oX0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjt1Sit7w%253D&md5=b1f4394a7a410273c800b65ee9697c7e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01865%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%2BJ.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DCockcroft%26aufirst%3DX.-L.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFiegen%26aufirst%3DD.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DHofmann%26aufirst%3DM.%2BH.%26aulast%3DHohmann%26aufirst%3DA.%2BF.%26aulast%3DKessler%26aufirst%3DD.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKnesl%26aufirst%3DP.%26aulast%3DKornigg%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DNar%26aufirst%3DH.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DSchaaf%26aufirst%3DO.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DMcConnell%26aufirst%3DD.%26atitle%3DStructure-Based%2520Design%2520of%2520an%2520in%2520Vivo%2520Active%2520Selective%2520Brd9%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4462%26epage%3D4475%26doi%3D10.1021%2Facs.jmedchem.5b01865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bit, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molnar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Discovery of I-Brd9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1439</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00256</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00256" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1425-1439&author=N.+H.+Theodoulouauthor=P.+Bamboroughauthor=A.+J.+Bannisterauthor=I.+Becherauthor=R.+A.+Bitauthor=K.+H.+Cheauthor=C.-w.+Chungauthor=A.+Dittmannauthor=G.+Drewesauthor=D.+H.+Drewryauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Leveridgeauthor=M.+Lindonauthor=A.-M.+Michonauthor=J.+Molnarauthor=S.+C.+Robsonauthor=N.+C.+O.+Tomkinsonauthor=T.+Kouzaridesauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Discovery+of+I-Brd9%2C+a+Selective+Cell+Active+Chemical+Probe+for+Bromodomain+Containing+Protein+9+Inhibition&doi=10.1021%2Facs.jmedchem.5b00256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of I-BRD9, a Selective Cell Active Chemical Probe for Bromodomain Containing Protein 9 Inhibition</span></div><div class="casAuthors">Theodoulou, Natalie H.; Bamborough, Paul; Bannister, Andrew J.; Becher, Isabelle; Bit, Rino A.; Che, Ka Hing; Chung, Chun-wa; Dittmann, Antje; Drewes, Gerard; Drewry, David H.; Gordon, Laurie; Grandi, Paola; Leveridge, Melanie; Lindon, Matthew; Michon, Anne-Marie; Molnar, Judit; Robson, Samuel C.; Tomkinson, Nicholas C. O.; Kouzarides, Tony; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1425-1439</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acetylation of histone lysine residues is one of the most well-studied post-translational modifications of chromatin, selectively recognized by bromodomain "reader" modules.  Inhibitors of the bromodomain and extra terminal domain (BET) family of bromodomains have shown profound anticancer and anti-inflammatory properties, generating much interest in targeting other bromodomain-contg. proteins for disease treatment.  Herein, we report the discovery of I-BRD9, the first selective cellular chem. probe for bromodomain-contg. protein 9 (BRD9).  I-BRD9 was identified through structure-based design, leading to greater than 700-fold selectivity over the BET family and 200-fold over the highly homologous bromodomain-contg. protein 7 (BRD7).  I-BRD9 was used to identify genes regulated by BRD9 in Kasumi-1 cells involved in oncol. and immune response pathways and to the best of our knowledge, represents the first selective tool compd. available to elucidate the cellular phenotype of BRD9 bromodomain inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVmjhksIDyTrVg90H21EOLACvtfcHk0ljKDEim00D-2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlaktb8%253D&md5=4e8f6aa664797fe54772b64c81cf2a83</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00256%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DBecher%26aufirst%3DI.%26aulast%3DBit%26aufirst%3DR.%2BA.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMolnar%26aufirst%3DJ.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DDiscovery%2520of%2520I-Brd9%252C%2520a%2520Selective%2520Cell%2520Active%2520Chemical%2520Probe%2520for%2520Bromodomain%2520Containing%2520Protein%25209%2520Inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1425%26epage%3D1439%26doi%3D10.1021%2Facs.jmedchem.5b00256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karim, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span> <span> </span><span class="NLM_article-title">An Advanced Tool to Interrogate Brd9</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4459</span>– <span class="NLM_lpage">4461</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00550</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00550" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC28XmslWgs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4459-4461&author=R.+M.+Karimauthor=E.+Sch%C3%B6nbrunn&title=An+Advanced+Tool+to+Interrogate+Brd9&doi=10.1021%2Facs.jmedchem.6b00550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">An Advanced Tool To Interrogate BRD9</span></div><div class="casAuthors">Karim, Rezaul M.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4459-4461</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective inhibitors of bromodomain-contg. protein 9 (BRD9) may have therapeutic potential in the treatment of human malignancies and inflammatory diseases.  A selective small mol. inhibitor that is well tolerated and has proper pharmacokinetic properties is required to explore the function of BRD9 in diseases.  BI-9564 (2) is a cell permeable and noncytotoxic BRD9 inhibitor provided to the scientific community to explore BRD9 biol. and det. its potential as a drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRF7iQqoEDOLVg90H21EOLACvtfcHk0ljKDEim00D-2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmslWgs7g%253D&md5=5f4340cb6f20c499dc6408d2ba84bf79</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00550%26sid%3Dliteratum%253Aachs%26aulast%3DKarim%26aufirst%3DR.%2BM.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DAn%2520Advanced%2520Tool%2520to%2520Interrogate%2520Brd9%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4459%26epage%3D4461%26doi%3D10.1021%2Facs.jmedchem.6b00550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wühr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Degradation of the Baf Complex Factor Brd9 by Heterobifunctional Ligands</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5743</span>, <span class="refDoi"> DOI: 10.1002/anie.201611281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1002%2Fanie.201611281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=5738-5743&author=D.+Remillardauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=G.+L.+Brienauthor=M.+Sonnettauthor=H.-S.+Seoauthor=S.+Dastjerdiauthor=M.+W%C3%BChrauthor=S.+Dhe-Paganonauthor=S.+A.+Armstrongauthor=J.+E.+Bradner&title=Degradation+of+the+Baf+Complex+Factor+Brd9+by+Heterobifunctional+Ligands&doi=10.1002%2Fanie.201611281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span></div><div class="casAuthors">Remillard, David; Buckley, Dennis L.; Paulk, Joshiawa; Brien, Gerard L.; Sonnett, Matthew; Seo, Hyuk-Soo; Dastierdi, Shiva; Wuehr, Martin; Dhe-Paganon, Sirano; Armstrong, Scott A.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5738-5743</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. protein BRD9, a subunit of the human BAF (SWI/SNF) nucleosome remodeling complex, has emerged as an attractive therapeutic target in cancer.  Despite the development of chem. probes targeting the BRD9 bromodomain, there is a limited understanding of BRD9 function beyond acetyl-lysine recognition.  We have therefore created the first BRD9-directed chem. degraders, through iterative design and testing of heterobifunctional ligands that bridge the BRD9 bromodomain and the cereblon E3 ubiquitin ligase complex.  Degraders of BRD9 exhibit markedly enhanced potency compared to parental ligands (10- to 100-fold).  Parallel study of degraders with divergent BRD9-binding chemotypes in models of acute myeloid leukemia resolves bromodomain polypharmacol. in this emerging drug class.  Together, these findings reveal the tractability of non-BET bromodomain contg. proteins to chem. degrdn., and highlight lead compd. dBRD9 as a tool for the study of BRD9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ9FU2Ipaq27Vg90H21EOLACvtfcHk0ljKDEim00D-2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D&md5=2eb6562cc9d976c7ae6070c9a419e9ea</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611281%26sid%3Dliteratum%253Aachs%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DSonnett%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DW%25C3%25BChr%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDegradation%2520of%2520the%2520Baf%2520Complex%2520Factor%2520Brd9%2520by%2520Heterobifunctional%2520Ligands%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5738%26epage%3D5743%26doi%3D10.1002%2Fanie.201611281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoppi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riching, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spallarossa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (Protacs) Identify Vz185 as a Potent, Fast, and Selective Von Hippel-Lindau (Vhl) Based Dual Degrader Probe of Brd9 and Brd7</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">699</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=699-726&author=V.+Zoppiauthor=S.+J.+Hughesauthor=C.+Maniaciauthor=A.+Testaauthor=T.+Gmaschitzauthor=C.+Wieshoferauthor=M.+Koeglauthor=K.+M.+Richingauthor=D.+L.+Danielsauthor=A.+Spallarossaauthor=A.+Ciulli&title=Iterative+Design+and+Optimization+of+Initially+Inactive+Proteolysis+Targeting+Chimeras+%28Protacs%29+Identify+Vz185+as+a+Potent%2C+Fast%2C+and+Selective+Von+Hippel-Lindau+%28Vhl%29+Based+Dual+Degrader+Probe+of+Brd9+and+Brd7&doi=10.1021%2Facs.jmedchem.8b01413"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (PROTACs) Identify VZ185 as a Potent, Fast, and Selective von Hippel-Lindau (VHL) Based Dual Degrader Probe of BRD9 and BRD7</span></div><div class="casAuthors">Zoppi, Vittoria; Hughes, Scott J.; Maniaci, Chiara; Testa, Andrea; Gmaschitz, Teresa; Wieshofer, Corinna; Koegl, Manfred; Riching, Kristin M.; Daniels, Danette L.; Spallarossa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">699-726</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Developing PROTACs to redirect the ubiquitination activity of E3 ligases and potently degrade a target protein within cells can be a lengthy and unpredictable process, and it remains unclear whether any combination of E3 and target might be productive for degrdn.  The authors describe a probe-quality degrader for a ligase-target pair deemed unsuitable: the von Hippel-Lindau (VHL) and BRD9, a bromodomain-contg. subunit of the SWI/SNF chromatin remodeling complex BAF.  VHL-based degraders could be optimized from suboptimal compds. in two rounds by systematically varying conjugation patterns and linkers and monitoring cellular degrdn. activities, kinetic profiles, and ubiquitination, as well as ternary complex formation thermodn.  The emerged structure-activity relations guided the discovery of VZ185, a potent, fast, and selective degrader of BRD9 and of its close homolog BRD7.  The authors' findings qualify a new chem. tool for BRD7/9 knockdown and provide a roadmap for PROTAC development against seemingly incompatible target-ligase combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiPnajotKvqLVg90H21EOLACvtfcHk0liPyP9bLQFZ0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN&md5=be7938c26807a2535b7f9b7de8ebfbb0</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01413%26sid%3Dliteratum%253Aachs%26aulast%3DZoppi%26aufirst%3DV.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRiching%26aufirst%3DK.%2BM.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSpallarossa%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DIterative%2520Design%2520and%2520Optimization%2520of%2520Initially%2520Inactive%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528Protacs%2529%2520Identify%2520Vz185%2520as%2520a%2520Potent%252C%2520Fast%252C%2520and%2520Selective%2520Von%2520Hippel-Lindau%2520%2528Vhl%2529%2520Based%2520Dual%2520Degrader%2520Probe%2520of%2520Brd9%2520and%2520Brd7%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D699%26epage%3D726%26doi%3D10.1021%2Facs.jmedchem.8b01413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muraki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohkawara, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosoya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onogi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koizumi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koizumi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sumi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yomoda, J.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuichi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibuya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krainer, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagiwara, M.</span></span> <span> </span><span class="NLM_article-title">Manipulation of Alternative Splicing by a Newly Developed Inhibitor of Clks</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">24246</span>– <span class="NLM_lpage">24254</span>, <span class="refDoi"> DOI: 10.1074/jbc.m314298200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1074%2Fjbc.M314298200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=15010457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksV2isbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=24246-24254&author=M.+Murakiauthor=B.+Ohkawaraauthor=T.+Hosoyaauthor=H.+Onogiauthor=J.+Koizumiauthor=T.+Koizumiauthor=K.+Sumiauthor=J.-i.+Yomodaauthor=M.+V.+Murrayauthor=H.+Kimuraauthor=K.+Furuichiauthor=H.+Shibuyaauthor=A.+R.+Krainerauthor=M.+Suzukiauthor=M.+Hagiwara&title=Manipulation+of+Alternative+Splicing+by+a+Newly+Developed+Inhibitor+of+Clks&doi=10.1074%2Fjbc.m314298200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Manipulation of alternative splicing by a newly developed inhibitor of Clks</span></div><div class="casAuthors">Muraki, Michiko; Ohkawara, Bisei; Hosoya, Takamitsu; Onogi, Hiroshi; Koizumi, Jun; Koizumi, Tomonobu; Sumi, Kengo; Yomoda, Jun-ichiro; Murray, Michael V.; Kimura, Hiroshi; Furuichi, Kiyoshi; Shibuya, Hiroshi; Krainer, Adrian R.; Suzuki, Masaaki; Hagiwara, Masatoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">24246-24254</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The regulation of splice site usage provides a versatile mechanism for controlling gene expression and for the generation of proteome diversity, playing an essential role in many biol. processes.  The importance of alternative splicing is further illustrated by the increasing no. of human diseases that have been attributed to mis-splicing events.  Appropriate spatial and temporal generation of splicing variants demands that alternative splicing be subjected to extensive regulation, similar to transcriptional control.  The Clk (Cdc2-like kinase) family has been implicated in splicing control and consists of at least four members.  Through extensive screening of a chem. library, we found that a benzothiazole compd., TG003, had a potent inhibitory effect on the activity of Clk1/Sty.  TG003 inhibited SF2/ASF-dependent splicing of β-globin pre-mRNA in vitro by suppression of Clk-mediated phosphorylation.  This drug also suppressed serine/arginine-rich protein phosphorylation, dissocn. of nuclear speckles, and Clk1/Sty-dependent alternative splicing in mammalian cells.  Consistently, administration of TG003 rescued the embryonic defects induced by excessive Clk activity in Xenopus.  Thus, TG003, a novel inhibitor of Clk family will be a valuable tool to dissect the regulatory mechanisms involving serine/arginine-rich protein phosphorylation signaling pathways in vivo, and may be applicable for the therapeutic manipulation of abnormal splicing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzT17psA1H8LVg90H21EOLACvtfcHk0liPyP9bLQFZ0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksV2isbg%253D&md5=6326340869863dca70799137ba67e3d8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M314298200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M314298200%26sid%3Dliteratum%253Aachs%26aulast%3DMuraki%26aufirst%3DM.%26aulast%3DOhkawara%26aufirst%3DB.%26aulast%3DHosoya%26aufirst%3DT.%26aulast%3DOnogi%26aufirst%3DH.%26aulast%3DKoizumi%26aufirst%3DJ.%26aulast%3DKoizumi%26aufirst%3DT.%26aulast%3DSumi%26aufirst%3DK.%26aulast%3DYomoda%26aufirst%3DJ.-i.%26aulast%3DMurray%26aufirst%3DM.%2BV.%26aulast%3DKimura%26aufirst%3DH.%26aulast%3DFuruichi%26aufirst%3DK.%26aulast%3DShibuya%26aufirst%3DH.%26aulast%3DKrainer%26aufirst%3DA.%2BR.%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DHagiwara%26aufirst%3DM.%26atitle%3DManipulation%2520of%2520Alternative%2520Splicing%2520by%2520a%2520Newly%2520Developed%2520Inhibitor%2520of%2520Clks%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D24246%26epage%3D24254%26doi%3D10.1074%2Fjbc.m314298200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirazian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nematalla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sistla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of 5-[5-Fluoro-2-Oxo-1,2- Dihydroindol-(3z)-Ylidenemethyl]-2,4- Dimethyl-1h-Pyrrole-3-Carboxylic Acid (2-Diethylaminoethyl)Amide, a Novel Tyrosine Kinase Inhibitor Targeting Vascular Endothelial and Platelet-Derived Growth Factor Receptor Tyrosine Kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1116</span>– <span class="NLM_lpage">1119</span>, <span class="refDoi"> DOI: 10.1021/jm0204183</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0204183" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhslSlsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=1116-1119&author=L.+Sunauthor=C.+Liangauthor=S.+Shirazianauthor=Y.+Zhouauthor=T.+Millerauthor=J.+Cuiauthor=J.+Y.+Fukudaauthor=J.-Y.+Chuauthor=A.+Nematallaauthor=X.+Wangauthor=H.+Chenauthor=A.+Sistlaauthor=T.+C.+Luuauthor=F.+Tangauthor=J.+Weiauthor=C.+Tang&title=Discovery+of+5-%5B5-Fluoro-2-Oxo-1%2C2-+Dihydroindol-%283z%29-Ylidenemethyl%5D-2%2C4-+Dimethyl-1h-Pyrrole-3-Carboxylic+Acid+%282-Diethylaminoethyl%29Amide%2C+a+Novel+Tyrosine+Kinase+Inhibitor+Targeting+Vascular+Endothelial+and+Platelet-Derived+Growth+Factor+Receptor+Tyrosine+Kinase&doi=10.1021%2Fjm0204183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 5-[5-Fluoro-2-oxo-1,2- dihydroindol-(3Z)-ylidenemethyl]-2,4- dimethyl-1H-pyrrole-3-carboxylic Acid (2-Diethylaminoethyl)amide, a Novel Tyrosine Kinase Inhibitor Targeting Vascular Endothelial and Platelet-Derived Growth Factor Receptor Tyrosine Kinase</span></div><div class="casAuthors">Sun, Li; Liang, Chris; Shirazian, Sheri; Zhou, Yong; Miller, Todd; Cui, Jean; Fukuda, Juri Y.; Chu, Ji-Yu; Nematalla, Asaad; Wang, Xueyan; Chen, Hui; Sistla, Anand; Luu, Tony C.; Tang, Flora; Wei, James; Tang, Cho</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1116-1119</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To improve the antitumor properties and optimize the pharmaceutical properties including soly. and protein binding of indolin-2-ones, a series of different basic and weakly basic pyrrolylmethylidene indolinones I [R1 = H, F, Cl, Br; R2 = Et2NCH2CH2, pyridin-4-ylmethyl, 2-(1,2,3-triazol-1-yl)ethyl, etc.] were designed and synthesized.  Indolinone I [R1 = F, R2 = Et2NCH2CH2 (II)] showed the best overall profile in terms of potency for the VEGF-R2 and PDGF-Rβ tyrosine kinase at biochem. and cellular levels, soly., protein binding, and bioavailability.  II is currently in phase I clin. trials for the treatment of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptdQJ1VSPq8bVg90H21EOLACvtfcHk0liPyP9bLQFZ0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhslSlsbg%253D&md5=f6570cfa1ae4e31703c741d90fe65382</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm0204183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0204183%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DShirazian%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DMiller%26aufirst%3DT.%26aulast%3DCui%26aufirst%3DJ.%26aulast%3DFukuda%26aufirst%3DJ.%2BY.%26aulast%3DChu%26aufirst%3DJ.-Y.%26aulast%3DNematalla%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DSistla%26aufirst%3DA.%26aulast%3DLuu%26aufirst%3DT.%2BC.%26aulast%3DTang%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DC.%26atitle%3DDiscovery%2520of%25205-%255B5-Fluoro-2-Oxo-1%252C2-%2520Dihydroindol-%25283z%2529-Ylidenemethyl%255D-2%252C4-%2520Dimethyl-1h-Pyrrole-3-Carboxylic%2520Acid%2520%25282-Diethylaminoethyl%2529Amide%252C%2520a%2520Novel%2520Tyrosine%2520Kinase%2520Inhibitor%2520Targeting%2520Vascular%2520Endothelial%2520and%2520Platelet-Derived%2520Growth%2520Factor%2520Receptor%2520Tyrosine%2520Kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D1116%26epage%3D1119%26doi%3D10.1021%2Fjm0204183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blasina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arango, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraynov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Register, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ninkovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nichols, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderes, K.</span></span> <span> </span><span class="NLM_article-title">Breaching the DNA Damage Checkpoint Via Pf-00477736, a Novel Small-Molecule Inhibitor of Checkpoint Kinase 1</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2394</span>– <span class="NLM_lpage">2404</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-07-2391</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1158%2F1535-7163.MCT-07-2391" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=18723486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVWksL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=2394-2404&author=A.+Blasinaauthor=J.+Hallinauthor=E.+Chenauthor=M.+E.+Arangoauthor=E.+Kraynovauthor=J.+Registerauthor=S.+Grantauthor=S.+Ninkovicauthor=P.+Chenauthor=T.+Nicholsauthor=P.+O%E2%80%99Connorauthor=K.+Anderes&title=Breaching+the+DNA+Damage+Checkpoint+Via+Pf-00477736%2C+a+Novel+Small-Molecule+Inhibitor+of+Checkpoint+Kinase+1&doi=10.1158%2F1535-7163.mct-07-2391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Breaching the DNA damage checkpoint via PF-00477736, a novel small-molecule inhibitor of checkpoint kinase 1</span></div><div class="casAuthors">Blasina, Alessandra; Hallin, Jill; Chen, Enhong; Arango, Maria Elena; Kraynov, Eugenia; Register, James; Grant, Stephan; Ninkovic, Sacha; Chen, Ping; Nichols, Tim; O'Connor, Patrick; Anderes, Kenna</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2394-2404</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Checkpoints are present in all phases of the cell cycle and are regarded as the gatekeepers maintaining the integrity of the genome.  Many conventional agents used to treat cancer impart damage to the genome and activate cell cycle checkpoints.  Many tumors are defective in the tumor suppressor p53 and therefore lack a functional G1 checkpoint.  In these tumors, however, the S-G2 checkpoints remain intact and, in response to DNA damage, arrest cell cycle progression allowing time for DNA repair.  Checkpoint kinase 1 (Chk1) is a key element in the DNA damage response pathway and plays a crucial role in the S-G2-phase checkpoints.  Inhibiting Chk1 represents a therapeutic strategy for creating a "synthetic lethal" response by overriding the last checkpoint defense of tumor cells against the lethal damage induced by DNA-directed chemotherapeutic agents.  Chk1 inhibition is consistent with emerging targeted therapies aiming to exploit mol. differences between normal and cancer cells.  Adding a Chk1 inhibitor to DNA-damaging cytotoxic therapy selectively targets tumors with intrinsic checkpoint defects while minimizing toxicity in checkpoint-competent normal cells.  PF-00477736 was identified as a potent, selective ATP-competitive small-mol. inhibitor that inhibits Chk1 with a Ki of 0.49 nM.  PF-00477736 abrogates cell cycle arrest induced by DNA damage and enhances cytotoxicity of clin. important chemotherapeutic agents, including gemcitabine and carboplatin.  In xenografts, PF-00477736 enhanced the antitumor activity of gemcitabine in a dose-dependent manner.  PF-00477736 combinations were well tolerated with no exacerbation of side effects commonly assocd. with cytotoxic agents. [Mol Cancer Ther 2008;7(8):2394-404].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCAlkcnouhLrVg90H21EOLACvtfcHk0ljgdxBchf4aBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVWksL%252FP&md5=fc3f9cf9121c207c6fa284b2090ed1c3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-07-2391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-07-2391%26sid%3Dliteratum%253Aachs%26aulast%3DBlasina%26aufirst%3DA.%26aulast%3DHallin%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DE.%26aulast%3DArango%26aufirst%3DM.%2BE.%26aulast%3DKraynov%26aufirst%3DE.%26aulast%3DRegister%26aufirst%3DJ.%26aulast%3DGrant%26aufirst%3DS.%26aulast%3DNinkovic%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DNichols%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DP.%26aulast%3DAnderes%26aufirst%3DK.%26atitle%3DBreaching%2520the%2520DNA%2520Damage%2520Checkpoint%2520Via%2520Pf-00477736%252C%2520a%2520Novel%2520Small-Molecule%2520Inhibitor%2520of%2520Checkpoint%2520Kinase%25201%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2008%26volume%3D7%26spage%3D2394%26epage%3D2404%26doi%3D10.1158%2F1535-7163.mct-07-2391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Solution Structure of Brd7 Bromodomain and Its Interaction with Acetylated Peptides from Histone H3 and H4</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2007.04.139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1016%2Fj.bbrc.2007.04.139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=17498659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsFKgtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2007&pages=435-441&author=H.+Sunauthor=J.+Liuauthor=J.+Zhangauthor=W.+Shenauthor=H.+Huangauthor=C.+Xuauthor=H.+Daiauthor=J.+Wuauthor=Y.+Shi&title=Solution+Structure+of+Brd7+Bromodomain+and+Its+Interaction+with+Acetylated+Peptides+from+Histone+H3+and+H4&doi=10.1016%2Fj.bbrc.2007.04.139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Solution structure of BRD7 bromodomain and its interaction with acetylated peptides from histone H3 and H4</span></div><div class="casAuthors">Sun, Hongbin; Liu, Jiangxin; Zhang, Jiahai; Shen, Weiqun; Huang, Hongda; Xu, Chao; Dai, Haiming; Wu, Jihui; Shi, Yunyu</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">435-441</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">BRD7 is an important protein tightly assocd. with Nasopharyngeal carcinoma (NPC).  Overexpression of BRD7 inhibits NPC cell growth and cell cycle by transcriptionally regulating the cell cycle related genes.  BRD7 contains a bromodomain that is found in many chromatin-assocd. proteins and in nearly all known nuclear histone acetyltransferases (HATs) and plays an important role in chromatin remodeling and transcriptional activation.  Here, we report the soln. structure of BRD7 bromodomain detd. by NMR spectroscopy, and its binding specificity revealed by NMR titrn. with several acetylated histone peptides.  We find that BRD7 bromodomain contains the typical left-handed four-helix bundle topol., and can bind with weak affinity to lysine-acetylated peptides derived from histone H3 with K9 or K14 acetylated and from histone H4 with K8, K12 or K16 acetylated.  Our results show that BRD7 bromodomain lacks inherent binding specificity when binding to histones in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRW-fXT-5M4rVg90H21EOLACvtfcHk0ljgdxBchf4aBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsFKgtL8%253D&md5=4df1fec4e7a9dc3885e176f723288703</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2007.04.139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2007.04.139%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DDai%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DSolution%2520Structure%2520of%2520Brd7%2520Bromodomain%2520and%2520Its%2520Interaction%2520with%2520Acetylated%2520Peptides%2520from%2520Histone%2520H3%2520and%2520H4%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2007%26volume%3D358%26spage%3D435%26epage%3D441%26doi%3D10.1016%2Fj.bbrc.2007.04.139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonards, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikbin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guetzoyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samsonova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Promiscuous Targeting of Bromodomains by Bromosporine Identifies Bet Proteins as Master Regulators of Primary Transcription Response in Leukemia</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e1600760</span> <span class="refDoi"> DOI: 10.1126/sciadv.1600760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1126%2Fsciadv.1600760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=27757418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&author=S.+Picaudauthor=K.+Leonardsauthor=J.-P.+Lambertauthor=O.+Doveyauthor=C.+Wellsauthor=O.+Fedorovauthor=O.+Monteiroauthor=T.+Fujisawaauthor=C.-Y.+Wangauthor=H.+Lingardauthor=C.+Tallantauthor=N.+Nikbinauthor=L.+Guetzoyanauthor=R.+Inghamauthor=S.+V.+Leyauthor=P.+Brennanauthor=S.+Mullerauthor=A.+Samsonovaauthor=A.-C.+Gingrasauthor=J.+Schwallerauthor=G.+Vassiliouauthor=S.+Knappauthor=P.+Filippakopoulos&title=Promiscuous+Targeting+of+Bromodomains+by+Bromosporine+Identifies+Bet+Proteins+as+Master+Regulators+of+Primary+Transcription+Response+in+Leukemia&doi=10.1126%2Fsciadv.1600760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span></div><div class="casAuthors">Picaud, Sarah; Leonards, Katharina; Lambert, Jean-Philippe; Dovey, Oliver; Wells, Christopher; Fedorov, Oleg; Monteiro, Octovia; Fujisawa, Takao; Wang, Chen-Yi; Lingard, Hannah; Tallant, Cynthia; Nikbin, Nikzad; Guetzoyan, Lucie; Ingham, Richard; Ley, Steven V.; Brennan, Paul; Muller, Susanne; Samsonova, Anastasia; Gingras, Anne-Claude; Schwaller, Juerg; Vassiliou, George; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1600760/1-e1600760/16</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) have emerged as compelling targets for cancer therapy.  The development of selective and potent BET (bromo and extra-terminal) inhibitors and their significant activity in diverse tumor models have rapidly translated into clin. studies and have motivated drug development efforts targeting non-BET BRDs.  However, the complexmultidomain/subunit architecture of BRD protein complexes complicates predictions of the consequences of their pharmacol. targeting.  To address this issue, we developed a promiscuous BRD inhibitor [bromosporine (BSP)] that broadly targets BRDs (including BETs) with nanomolar affinity, creating a tool for the identification of cellular processes and diseases where BRDs have a regulatory function.  As a proof of principle, we studied the effects of BSP on leukemic cell lines known to be sensitive to BET inhibition and found, as expected, strong antiproliferative activity.  Comparison of the modulation of transcriptional profiles by BSP after a short exposure to the inhibitor resulted in a BET inhibitor signature but no significant addnl. changes in transcription that could account for inhibition of other BRDs.  Thus, nonselective targeting of BRDs identified BETs, but not other BRDs, as master regulators of contextdependent primary transcription response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28artLYYsqrVg90H21EOLACvtfcHk0lgZ7yyFgKvOlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D&md5=3060e8fd66c657f70e0ec3acea273057</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1600760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1600760%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DLeonards%26aufirst%3DK.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DDovey%26aufirst%3DO.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DC.-Y.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DNikbin%26aufirst%3DN.%26aulast%3DGuetzoyan%26aufirst%3DL.%26aulast%3DIngham%26aufirst%3DR.%26aulast%3DLey%26aufirst%3DS.%2BV.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSamsonova%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DVassiliou%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DPromiscuous%2520Targeting%2520of%2520Bromodomains%2520by%2520Bromosporine%2520Identifies%2520Bet%2520Proteins%2520as%2520Master%2520Regulators%2520of%2520Primary%2520Transcription%2520Response%2520in%2520Leukemia%26jtitle%3DSci.%2520Adv.%26date%3D2016%26volume%3D2%26doi%3D10.1126%2Fsciadv.1600760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chory, E. J.</span>; <span class="NLM_string-name">Kirkland, J. G.</span>; <span class="NLM_string-name">Chang, C.-Y.</span>; <span class="NLM_string-name">D’Andrea, V. D.</span>; <span class="NLM_string-name">Gourinsankar, S.</span>; <span class="NLM_string-name">Dykhuizen, E. C.</span>; <span class="NLM_string-name">Crabtree, G. R.</span></span> <span> </span><span class="NLM_article-title">Inhibition of a Selective Swi/Snf Function Synergizes with Atr Inhibitors in Cancer Cell Killing</span>. <span class="NLM_year" style="font-weight: bold;">2019</span>, bioRxiv 660456.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=E.+J.+Chory&author=J.+G.+Kirkland&author=C.-Y.+Chang&author=V.+D.+D%E2%80%99Andrea&author=S.+Gourinsankar&author=E.+C.+Dykhuizen&author=G.+R.+Crabtree&title=Inhibition+of+a+Selective+Swi%2FSnf+Function+Synergizes+with+Atr+Inhibitors+in+Cancer+Cell+Killing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChory%26aufirst%3DE.%2BJ.%26atitle%3DInhibition%2520of%2520a%2520Selective%2520Swi%252FSnf%2520Function%2520Synergizes%2520with%2520Atr%2520Inhibitors%2520in%2520Cancer%2520Cell%2520Killing%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bell, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, P. K.</span></span> <span> </span><span class="NLM_article-title">Pik3ca Cooperates with Kras to Promote Myc Activity and Tumorigenesis Via the Bromodomain Protein Brd9</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1634</span>, <span class="refDoi"> DOI: 10.3390/cancers11111634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.3390%2Fcancers11111634" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=1634&author=C.+M.+Bellauthor=P.+Raffeinerauthor=J.+R.+Hartauthor=P.+K.+Vogt&title=Pik3ca+Cooperates+with+Kras+to+Promote+Myc+Activity+and+Tumorigenesis+Via+the+Bromodomain+Protein+Brd9&doi=10.3390%2Fcancers11111634"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3390%2Fcancers11111634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers11111634%26sid%3Dliteratum%253Aachs%26aulast%3DBell%26aufirst%3DC.%2BM.%26aulast%3DRaffeiner%26aufirst%3DP.%26aulast%3DHart%26aufirst%3DJ.%2BR.%26aulast%3DVogt%26aufirst%3DP.%2BK.%26atitle%3DPik3ca%2520Cooperates%2520with%2520Kras%2520to%2520Promote%2520Myc%2520Activity%2520and%2520Tumorigenesis%2520Via%2520the%2520Bromodomain%2520Protein%2520Brd9%26jtitle%3DCancers%26date%3D2019%26volume%3D11%26spage%3D1634%26doi%3D10.3390%2Fcancers11111634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kallen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergsdorf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnaud, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brichet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cobos-Correa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elhajouji, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freuler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galimberti, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guibourdenche, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haenni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holzinger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunziker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izaac, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leder, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martus, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Matt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyakov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roethlisberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiefl, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uteng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerchner, A.</span></span> <span> </span><span class="NLM_article-title">X-Ray Structures and Feasibility Assessment of Clk2 Inhibitors for Phelan-Mcdermid Syndrome</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1997</span>– <span class="NLM_lpage">2007</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1002%2Fcmdc.201800344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=29985556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFahs7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1997-2007&author=J.+Kallenauthor=C.+Bergsdorfauthor=B.+Arnaudauthor=M.+Bernhardauthor=M.+Brichetauthor=A.+Cobos-Correaauthor=A.+Elhajoujiauthor=F.+Freulerauthor=I.+Galimbertiauthor=C.+Guibourdencheauthor=S.+Haenniauthor=S.+Holzingerauthor=J.+Hunzikerauthor=A.+Izaacauthor=M.+Kaufmannauthor=L.+Lederauthor=H.-J.+Martusauthor=P.+von+Mattauthor=V.+Polyakovauthor=P.+Roethlisbergerauthor=G.+Romaauthor=N.+Stieflauthor=M.+Utengauthor=A.+Lerchner&title=X-Ray+Structures+and+Feasibility+Assessment+of+Clk2+Inhibitors+for+Phelan-Mcdermid+Syndrome&doi=10.1002%2Fcmdc.201800344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">X-ray Structures and Feasibility Assessment of CLK2 Inhibitors for Phelan-McDermid Syndrome</span></div><div class="casAuthors">Kallen, Joerg; Bergsdorf, Christian; Arnaud, Bertrand; Bernhard, Mario; Brichet, Murielle; Cobos-Correa, Amanda; Elhajouji, Azeddine; Freuler, Felix; Galimberti, Ivan; Guibourdenche, Christel; Haenni, Simon; Holzinger, Sandra; Hunziker, Juerg; Izaac, Aude; Kaufmann, Markus; Leder, Lukas; Martus, Hans-Joerg; von Matt, Peter; Polyakov, Valery; Roethlisberger, Patrik; Roma, Guglielmo; Stiefl, Nikolaus; Uteng, Marianne; Lerchner, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1997-2007</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">CLK2 inhibition has been proposed as a potential mechanism to improve autism and neuronal functions in Phelan-McDermid syndrome (PMDS).  Herein, the discovery of a very potent indazole CLK inhibitor series and the CLK2 x-ray structure of the most potent analog are reported.  This new indazole series was identified through a biochem. CLK2 Caliper assay screen with 30k compds. selected by an in silico approach.  Novel high-resoln. x-ray structures of all CLKs, including the first CLK4 x-ray structure, bound to known CLK2 inhibitor tool compds. (e.g., TG003, CX-4945), are also shown and yield insight into inhibitor selectivity in the CLK family.  The efficacy of the new CLK2 inhibitors from the indazole series was demonstrated in the mouse brain slice assay, and potential safety concerns were investigated.  Genotoxicity findings in the human lymphocyte micronucleus test (MNT) assay are shown by using two structurally different CLK inhibitors to reveal a major concern for pan-CLK inhibition in PMDS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXWV3hjWfnELVg90H21EOLACvtfcHk0lgZ7yyFgKvOlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFahs7vK&md5=bd539ce78951e907dd74cc293022edb1</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800344%26sid%3Dliteratum%253Aachs%26aulast%3DKallen%26aufirst%3DJ.%26aulast%3DBergsdorf%26aufirst%3DC.%26aulast%3DArnaud%26aufirst%3DB.%26aulast%3DBernhard%26aufirst%3DM.%26aulast%3DBrichet%26aufirst%3DM.%26aulast%3DCobos-Correa%26aufirst%3DA.%26aulast%3DElhajouji%26aufirst%3DA.%26aulast%3DFreuler%26aufirst%3DF.%26aulast%3DGalimberti%26aufirst%3DI.%26aulast%3DGuibourdenche%26aufirst%3DC.%26aulast%3DHaenni%26aufirst%3DS.%26aulast%3DHolzinger%26aufirst%3DS.%26aulast%3DHunziker%26aufirst%3DJ.%26aulast%3DIzaac%26aufirst%3DA.%26aulast%3DKaufmann%26aufirst%3DM.%26aulast%3DLeder%26aufirst%3DL.%26aulast%3DMartus%26aufirst%3DH.-J.%26aulast%3Dvon%2BMatt%26aufirst%3DP.%26aulast%3DPolyakov%26aufirst%3DV.%26aulast%3DRoethlisberger%26aufirst%3DP.%26aulast%3DRoma%26aufirst%3DG.%26aulast%3DStiefl%26aufirst%3DN.%26aulast%3DUteng%26aufirst%3DM.%26aulast%3DLerchner%26aufirst%3DA.%26atitle%3DX-Ray%2520Structures%2520and%2520Feasibility%2520Assessment%2520of%2520Clk2%2520Inhibitors%2520for%2520Phelan-Mcdermid%2520Syndrome%26jtitle%3DChemMedChem%26date%3D2018%26volume%3D13%26spage%3D1997%26epage%3D2007%26doi%3D10.1002%2Fcmdc.201800344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kutach, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villaseñor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belunis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lok, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, L.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deval, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novak, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuglstatter, A.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of Il-2-Inducible T Cell Kinase Complexed with Inhibitors: Insights into Rational Drug Design and Activity Regulation</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">163</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2010.00993.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1111%2Fj.1747-0285.2010.00993.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=20545945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptleqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2010&pages=154-163&author=A.+K.+Kutachauthor=A.+G.+Villase%C3%B1orauthor=D.+Lamauthor=C.+Belunisauthor=C.+Jansonauthor=S.+Lokauthor=L.-N.+Hongauthor=C.-M.+Liuauthor=J.+Devalauthor=T.+J.+Novakauthor=J.+W.+Barnettauthor=W.+Chuauthor=D.+Shawauthor=A.+Kuglstatter&title=Crystal+Structures+of+Il-2-Inducible+T+Cell+Kinase+Complexed+with+Inhibitors%3A+Insights+into+Rational+Drug+Design+and+Activity+Regulation&doi=10.1111%2Fj.1747-0285.2010.00993.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of IL-2-inducible T cell kinase complexed with inhibitors: insights into rational drug design and activity regulation</span></div><div class="casAuthors">Kutach, Alan K.; Villasenor, Armando G.; Lam, Diana; Belunis, Charles; Janson, Cheryl; Lok, Stephen; Hong, Li-Na; Liu, Chao-Min; Deval, Jerome; Novak, Thomas J.; Barnett, Jim W.; Chu, Wei; Shaw, David; Kuglstatter, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">154-163</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">IL-2-inducible T cell kinase plays an essential role in T cell receptor signaling and is considered a drug target for the treatment of Th2-mediated inflammatory diseases.  By applying high-throughput protein engineering and crystn., we have detd. the x-ray crystal structures of IL-2-inducible T cell kinase in complex with its selective inhibitor BMS-509744 and the broad-spectrum kinase inhibitors sunitinib and RO5191614.  Sunitinib uniquely stabilizes IL-2-inducible T cell kinase in the helix C-in conformation by inducing side chain conformational changes in the ATP-binding site.  This preference of sunitinib to bind to an active kinase conformation is reflective of its broad-spectrum kinase activity.  BMS-509744 uniquely stabilizes the activation loop in a substrate-blocking inactive conformation, indicating that structural changes described for Src family kinases are also involved in the regulation of IL-2-inducible T cell kinase activity.  The obsd. BMS-509744 binding mode allows rationalization of structure-activity relationships reported for this inhibitor class and facilitates further structure-based drug design.  Sequence-based anal. of this binding mode provides guidance for the rational design of inhibitor selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXxFkkRUV7B7Vg90H21EOLACvtfcHk0lh50H2hHLs9NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptleqs7Y%253D&md5=24b9225c52eff5cca4e295dad1c554c3</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2010.00993.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2010.00993.x%26sid%3Dliteratum%253Aachs%26aulast%3DKutach%26aufirst%3DA.%2BK.%26aulast%3DVillase%25C3%25B1or%26aufirst%3DA.%2BG.%26aulast%3DLam%26aufirst%3DD.%26aulast%3DBelunis%26aufirst%3DC.%26aulast%3DJanson%26aufirst%3DC.%26aulast%3DLok%26aufirst%3DS.%26aulast%3DHong%26aufirst%3DL.-N.%26aulast%3DLiu%26aufirst%3DC.-M.%26aulast%3DDeval%26aufirst%3DJ.%26aulast%3DNovak%26aufirst%3DT.%2BJ.%26aulast%3DBarnett%26aufirst%3DJ.%2BW.%26aulast%3DChu%26aufirst%3DW.%26aulast%3DShaw%26aufirst%3DD.%26aulast%3DKuglstatter%26aufirst%3DA.%26atitle%3DCrystal%2520Structures%2520of%2520Il-2-Inducible%2520T%2520Cell%2520Kinase%2520Complexed%2520with%2520Inhibitors%253A%2520Insights%2520into%2520Rational%2520Drug%2520Design%2520and%2520Activity%2520Regulation%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2010%26volume%3D76%26spage%3D154%26epage%3D163%26doi%3D10.1111%2Fj.1747-0285.2010.00993.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Q. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng
de, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, J. C.</span></span> <span> </span><span class="NLM_article-title">Bromodomain-Containing Protein 7 (Brd7) as a Potential Tumor Suppressor in Hepatocellular Carcinoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">16248</span>– <span class="NLM_lpage">16261</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.7637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.18632%2Foncotarget.7637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=26919247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A280%3ADC%252BC28jks1Oktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=16248-16261&author=C.+L.+Chenauthor=Y.+Wangauthor=Q.+Z.+Panauthor=Y.+Tangauthor=Q.+J.+Wangauthor=K.+Panauthor=L.+X.+Huangauthor=J.+Heauthor=J.+J.+Zhaoauthor=S.+S.+Jiangauthor=X.+F.+Zhangauthor=H.+X.+Zhangauthor=Z.+Q.+Zhouauthor=S.+Weng%0Adeauthor=J.+C.+Xia&title=Bromodomain-Containing+Protein+7+%28Brd7%29+as+a+Potential+Tumor+Suppressor+in+Hepatocellular+Carcinoma&doi=10.18632%2Foncotarget.7637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain-containing protein 7 (BRD7) as a potential tumor suppressor in hepatocellular carcinoma</span></div><div class="casAuthors">Chen Chang-Long; Wang Ying; Pan Qiu-Zhong; Tang Yan; Pan Ke; Zhao Jing-Jing; Jiang Shan-Shan; Zhang Xiao-Fei; Zhang Hong-Xia; Zhou Zi-Qi; Weng De Sheng; Xia Jian-Chuan; Chen Chang-Long; Pan Qiu-Zhong; Tang Yan; Wang Qi-Jing; Pan Ke; Huang Li-Xi; He Jia; Zhao Jing-Jing; Jiang Shan-Shan; Zhang Xiao-Fei; Zhang Hong-Xia; Zhou Zi-Qi; Weng De Sheng; Xia Jian-Chuan; Wang Ying</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">16248-61</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bromodomain-containing protein 7 (BRD7) is a subunit of the PBAF complex, which functions as a transcriptional cofactor for the tumor suppressor protein p53.  Down-regulation of BRD7 has been demonstrated in multiple types of cancer.  This study aimed to investigate BRD7 expression and its tumor suppressive effect in hepatocellular carcinoma (HCC).The expression of BRD7 was examined in clinical specimens of primary HCC and in HCC cell lines through real-time quantitative PCR, western blot and immunohistochemistry.  The prognostic value of BRD7 expression and its correlation with the clinicopathological features of HCC patients were statistically analyzed.  The effect of BRD7 on the tumorigenicity of HCC was also examined using proliferation and colony-formation assays, cell-cycle assays, migration and cell-invasion assays, and xenograft nude mouse models.  BRD7 was down-regulated in tumor tissues and HCC cell lines.  BRD7 protein expression was strongly associated with clinical stage and tumor size.  Kaplan-Meier survival curves revealed higher survival rates in patients with higher BRD7 expression levels compared to those with lower BRD7 levels.  A multivariate analysis indicated that BRD7 expression was an independent prognostic marker.  The re-introduction of BRD7 expression significantly inhibited proliferation, colony formation, migration and invasion and led to cell cycle arrest in HCC cells in vitro.  Furthermore, experiments in mice suggested that BRD7 overexpression suppresses HCC tumorigenicity in vivo.  In conclusions, our data indicated that BRD7 may serve as a tumor suppressor in HCC and may be a novel molecular target for the treatment of HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR0i0lNQTGmmJPlX7E30qdzfW6udTcc2eYEeJWw0yuCNbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28jks1Oktw%253D%253D&md5=e9ab62dfe3a39f88b7624219c598a02d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.7637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.7637%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%2BL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DPan%26aufirst%3DQ.%2BZ.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%2BJ.%26aulast%3DPan%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DL.%2BX.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DJ.%2BJ.%26aulast%3DJiang%26aufirst%3DS.%2BS.%26aulast%3DZhang%26aufirst%3DX.%2BF.%26aulast%3DZhang%26aufirst%3DH.%2BX.%26aulast%3DZhou%26aufirst%3DZ.%2BQ.%26aulast%3DWeng%2Bde%26aufirst%3DS.%26aulast%3DXia%26aufirst%3DJ.%2BC.%26atitle%3DBromodomain-Containing%2520Protein%25207%2520%2528Brd7%2529%2520as%2520a%2520Potential%2520Tumor%2520Suppressor%2520in%2520Hepatocellular%2520Carcinoma%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D16248%26epage%3D16261%26doi%3D10.18632%2Foncotarget.7637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, H.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, D.-S.</span></span> <span> </span><span class="NLM_article-title">Tumor Suppressive Effects of Bromodomain-Containing Protein 7 (Brd7) in Epithelial Ovarian Carcinoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">565</span>– <span class="NLM_lpage">575</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-13-1271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1158%2F1078-0432.CCR-13-1271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=24198243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFWjs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=565-575&author=Y.-A.+Parkauthor=J.-W.+Leeauthor=H.-S.+Kimauthor=Y.-Y.+Leeauthor=T.-J.+Kimauthor=C.+H.+Choiauthor=J.-J.+Choiauthor=H.-K.+Jeonauthor=Y.+J.+Choauthor=J.+Y.+Ryuauthor=B.-G.+Kimauthor=D.-S.+Bae&title=Tumor+Suppressive+Effects+of+Bromodomain-Containing+Protein+7+%28Brd7%29+in+Epithelial+Ovarian+Carcinoma&doi=10.1158%2F1078-0432.ccr-13-1271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor Suppressive Effects of Bromodomain-Containing Protein 7 (BRD7) in Epithelial Ovarian Carcinoma</span></div><div class="casAuthors">Park, Young-Ae; Lee, Jeong-Won; Kim, Hye-Sun; Lee, Yoo-Young; Kim, Tae-Joong; Choi, Chel Hun; Choi, Jung-Joo; Jeon, Hye-Kyung; Cho, Young Jae; Ryu, Ji Yoon; Kim, Byoung-Gie; Bae, Duk-Soo</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">565-575</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Bromodomain-contg. protein 7 (BRD7), which is a subunit of SWI/SNF complex, has been recently suggested as a novel tumor suppressor in several cancers.  In this study, we investigated the tumor suppressive effect of BRD7 in epithelial ovarian cancer.  Exptl. Design: We analyzed the expression of BRD7 in human ovarian tissues with real-time PCR.  To investigate the functional role of BRD7, we transfected ovarian cancer cells (A2780 and SKOV3) with BRD7 plasmid and checked the cell viability, apoptosis, and invasion.  The activities of BRD7 in the signaling pathways assocd. with carcinogenesis were also tested.  In addn., we used the orthotopic mouse model for ovarian cancer to evaluate tumor growth-inhibiting effect by administration of BRD7 plasmid.  Results: The BRD7 expression was downregulated in the ovarian cancer tissues compared with normal (P < 0.05), high-grade serous cancer exhibited significantly decreased expression of BRD7 compared with low-grade (P < 0.01) serous cancer.  Transfection of BRD7 plasmid to A2780 (p53-wild) or SKOV3 (p53-null) ovarian cancer cells showed the tumor suppressive effects assessed by cell viability, apoptosis, and invasion assay and esp. significantly decreased tumor wt. in orthotopic mouse model (A2780).  Moreover, we found that tumor suppressive effects of BRD7 are independent to the presence of p53 activity in ovarian cancer cells.  BRD7 neg. regulated β-catenin pathway, resulting in decreased its accumulation in the nucleus.  Conclusions: These results suggested that BRD7 acts as a tumor suppressor in epithelial ovarian cancers independently of p53 activity, via neg. regulation of β-catenin pathway.  Clin Cancer Res; 20(3); 565-75. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEfH7-Hwr_0LVg90H21EOLACvtfcHk0lh50H2hHLs9NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFWjs78%253D&md5=243d439fc2b98b8580ad99e736aa0e7d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-13-1271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-13-1271%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DY.-A.%26aulast%3DLee%26aufirst%3DJ.-W.%26aulast%3DKim%26aufirst%3DH.-S.%26aulast%3DLee%26aufirst%3DY.-Y.%26aulast%3DKim%26aufirst%3DT.-J.%26aulast%3DChoi%26aufirst%3DC.%2BH.%26aulast%3DChoi%26aufirst%3DJ.-J.%26aulast%3DJeon%26aufirst%3DH.-K.%26aulast%3DCho%26aufirst%3DY.%2BJ.%26aulast%3DRyu%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DB.-G.%26aulast%3DBae%26aufirst%3DD.-S.%26atitle%3DTumor%2520Suppressive%2520Effects%2520of%2520Bromodomain-Containing%2520Protein%25207%2520%2528Brd7%2529%2520in%2520Epithelial%2520Ovarian%2520Carcinoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2014%26volume%3D20%26spage%3D565%26epage%3D575%26doi%3D10.1158%2F1078-0432.ccr-13-1271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasater, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Dual Kinase-Bromodomain Inhibitors for Rationally Designed Polypharmacology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+M%C3%BCllerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+Kinase-Bromodomain+Inhibitors+for+Rationally+Designed+Polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0liV0htfK7Lu_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520Kinase-Bromodomain%2520Inhibitors%2520for%2520Rationally%2520Designed%2520Polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Acetyl-Lysine Binding Site of Bromodomain-Containing Protein 4 (Brd4) Interacts with Diverse Kinase Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1021/cb500072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1160-1171&author=S.+W.+J.+Emberauthor=J.-Y.+Zhuauthor=S.+H.+Olesenauthor=M.+P.+Martinauthor=A.+Beckerauthor=N.+Berndtauthor=G.+I.+Georgauthor=E.+Sch%C3%B6nbrunn&title=Acetyl-Lysine+Binding+Site+of+Bromodomain-Containing+Protein+4+%28Brd4%29+Interacts+with+Diverse+Kinase+Inhibitors&doi=10.1021%2Fcb500072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyl-lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span></div><div class="casAuthors">Ember, Stuart W. J.; Zhu, Jin-Yi; Olesen, Sanne H.; Martin, Mathew P.; Becker, Andreas; Berndt, Norbert; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1160-1171</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the bromodomain and extra terminal (BET) family of proteins are essential for the recognition of acetylated lysine (KAc) residues in histones and have emerged as promising drug targets in cancer, inflammation, and contraception research.  In co-crystn. screening campaigns using the first bromodomain of BRD4 (BRD4-1) against kinase inhibitor libraries, we identified and characterized 14 kinase inhibitors (10 distinct chem. scaffolds) as ligands of the KAc binding site.  Among these, the PLK1 inhibitor BI2536 and the JAK2 inhibitor TG101209 displayed strongest inhibitory potential against BRD4 (IC50 = 25 nM and 130 nM, resp.) and high selectivity for BET bromodomains.  Comparative structural anal. revealed markedly different binding modes of kinase hinge-binding scaffolds in the KAc binding site, suggesting that BET proteins are potential off-targets of diverse kinase inhibitors.  Combined, these findings provide a new structural framework for the rational design of next-generation BET-selective and dual-activity BET-kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf0X1hjt7cBbVg90H21EOLACvtfcHk0ljad3Chfbs5qA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D&md5=5ca37df00cdaa0ba81c0cb4a0f70ba5d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fcb500072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500072z%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DZhu%26aufirst%3DJ.-Y.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DAcetyl-Lysine%2520Binding%2520Site%2520of%2520Bromodomain-Containing%2520Protein%25204%2520%2528Brd4%2529%2520Interacts%2520with%2520Diverse%2520Kinase%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1160%26epage%3D1171%26doi%3D10.1021%2Fcb500072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Dual Kinase-Bromodomain Inhibitors in Anticancer Drug Discovery: A Structural and Pharmacological Perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9305</span>– <span class="NLM_lpage">9320</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00438</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00438" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9305-9320&author=L.+Carlinoauthor=G.+Rastelli&title=Dual+Kinase-Bromodomain+Inhibitors+in+Anticancer+Drug+Discovery%3A+A+Structural+and+Pharmacological+Perspective&doi=10.1021%2Facs.jmedchem.6b00438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Kinase-Bromodomain Inhibitors in Anticancer Drug Discovery: A Structural and Pharmacological Perspective</span></div><div class="casAuthors">Carlino, Luca; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9305-9320</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases play crucial roles in several cell transformation processes and are validated drug targets for many human diseases, including cancer.  Nevertheless, most tumors have eluded the effects of inhibition of a single kinase by activating resistance mechanisms and/or alternative pathways and escape mechanisms.  In recent years, multitarget approaches directed toward inhibition of kinases and targets of different families have received increasing attention.  In particular, co-targeting kinases and bromodomain epigenetic reader proteins has rapidly emerged as a promising approach to cancer drug development.  In this manuscript, we will review the recent discoveries that led to the identification and optimization of dual kinase/bromodomain inhibitors.  We will analyze and compare the structural features required for dual inhibition and comment on the potential of this approach in anticancer drug discovery.  Moreover, we will introduce computational approaches useful for the identification of dual kinase/bromodomain inhibitors and generate ad hoc pharmacophore and docking models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkHDSBWCGdbVg90H21EOLACvtfcHk0ljad3Chfbs5qA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP&md5=1351a235bba09782160a86ae3f904833</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00438%26sid%3Dliteratum%253Aachs%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DDual%2520Kinase-Bromodomain%2520Inhibitors%2520in%2520Anticancer%2520Drug%2520Discovery%253A%2520A%2520Structural%2520and%2520Pharmacological%2520Perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9305%26epage%3D9320%26doi%3D10.1021%2Facs.jmedchem.6b00438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williston, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandts, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-N.</span></span> <span> </span><span class="NLM_article-title">Rapid Measurement of Binding Constants and Heats of Binding Using a New Titration Calorimeter</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>179</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(89)90213-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1016%2F0003-2697%2889%2990213-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=2757186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1989&pages=131-137&author=T.+Wisemanauthor=S.+Willistonauthor=J.+F.+Brandtsauthor=L.-N.+Lin&title=Rapid+Measurement+of+Binding+Constants+and+Heats+of+Binding+Using+a+New+Titration+Calorimeter&doi=10.1016%2F0003-2697%2889%2990213-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span></div><div class="casAuthors">Wiseman, Thomas; Williston, Samuel; Brandts, John F.; Lin, Lung Nan</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-7</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A new titrn. calorimeter is described and results are presented for the binding of 2'-CMP to the active site of RNase A.  The instrument characteristics include very high sensitivity, rapid calorimetric response, and fast thermal equilibration.  Convenient software is available for instrument operation, data collection, data redn., and deconvolution to obtain least-squares ests. of binding parameters n, ΔH°, ΔS°, and the binding const. K.  Sample through-put for the instrument is high, and under favorable conditions binding consts. as large as 108 M-1 can be measured.  The bovine RNase A (RNase)/2'-CMP system was studied over a 50-fold range of RNase concn. and at 2 different temps.  The binding consts. were in the 105 to 106 M-1 range, depending on conditions, and heats of binding ∼-15,000 cal/mol.  Repeat detns. suggested errors of only a few percent in n, ΔH°, and K values over the most favorable concn. range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRWydqrfWArVg90H21EOLACvtfcHk0ljad3Chfbs5qA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D&md5=53fcdb7144fd85e57963add0bcdb6041</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2889%2990213-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252889%252990213-3%26sid%3Dliteratum%253Aachs%26aulast%3DWiseman%26aufirst%3DT.%26aulast%3DWilliston%26aufirst%3DS.%26aulast%3DBrandts%26aufirst%3DJ.%2BF.%26aulast%3DLin%26aufirst%3DL.-N.%26atitle%3DRapid%2520Measurement%2520of%2520Binding%2520Constants%2520and%2520Heats%2520of%2520Binding%2520Using%2520a%2520New%2520Titration%2520Calorimeter%26jtitle%3DAnal.%2520Biochem.%26date%3D1989%26volume%3D179%26spage%3D131%26epage%3D137%26doi%3D10.1016%2F0003-2697%2889%2990213-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, Scaling, Space-Group Assignment and Post-Refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+Scaling%2C+Space-Group+Assignment+and+Post-Refinement&doi=10.1107%2Fs0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0lihTCjFLI8Jbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520Scaling%252C%2520Space-Group%2520Assignment%2520and%2520Post-Refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2Fs0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gildea, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Montero, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michels-Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span> <span> </span><span class="NLM_article-title">Dials: Implementation and Evaluation of a New Integration Package</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1107/s2059798317017235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1107%2FS2059798317017235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=85-97&author=G.+Winterauthor=D.+G.+Watermanauthor=J.+M.+Parkhurstauthor=A.+S.+Brewsterauthor=R.+J.+Gildeaauthor=M.+Gerstelauthor=L.+Fuentes-Monteroauthor=M.+Vollmarauthor=T.+Michels-Clarkauthor=I.+D.+Youngauthor=N.+K.+Sauterauthor=G.+Evans&title=Dials%3A+Implementation+and+Evaluation+of+a+New+Integration+Package&doi=10.1107%2Fs2059798317017235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">DIALS: implementation and evaluation of a new integration package</span></div><div class="casAuthors">Winter, Graeme; Waterman, David G.; Parkhurst, James M.; Brewster, Aaron S.; Gildea, Richard J.; Gerstel, Markus; Fuentes-Montero, Luis; Vollmar, Melanie; Michels-Clark, Tara; Young, Iris D.; Sauter, Nicholas K.; Evans, Gwyndaf</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-97</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The DIALS project is a collaboration between Diamond Light Source, Lawrence Berkeley National Lab. and CCP4 to develop a new software suite for the anal. of crystallog. X-ray diffraction data, initially encompassing spot finding, indexing, refinement and integration.  The design, core algorithms and structure of the software are introduced, alongside results from the anal. of data from biol. and chem. crystallog. expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkZqQrzPvxlbVg90H21EOLACvtfcHk0lihTCjFLI8Jbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksFarurk%253D&md5=f4572135ac40643815064914216f6d4c</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS2059798317017235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317017235%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DGerstel%26aufirst%3DM.%26aulast%3DFuentes-Montero%26aufirst%3DL.%26aulast%3DVollmar%26aufirst%3DM.%26aulast%3DMichels-Clark%26aufirst%3DT.%26aulast%3DYoung%26aufirst%3DI.%2BD.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DEvans%26aufirst%3DG.%26atitle%3DDials%253A%2520Implementation%2520and%2520Evaluation%2520of%2520a%2520New%2520Integration%2520Package%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2018%26volume%3D74%26spage%3D85%26epage%3D97%26doi%3D10.1107%2Fs2059798317017235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How Good Are My Data and What Is the Resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/s0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+Good+Are+My+Data+and+What+Is+the+Resolution%3F&doi=10.1107%2Fs0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lihTCjFLI8Jbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520Good%2520Are%2520My%2520Data%2520and%2520What%2520Is%2520the%2520Resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2Fs0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Phenix.Model_Vs_Data: A High-Level Tool for the Calculation of Crystallographic Model and Data Statistics</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1107/s0021889810015608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1107%2Fs0021889810015608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=20648263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVygtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=669-676&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=V.+B.+Chenauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=J.+S.+Richardsonauthor=D.+C.+Richardsonauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Phenix.Model_Vs_Data%3A+A+High-Level+Tool+for+the+Calculation+of+Crystallographic+Model+and+Data+Statistics&doi=10.1107%2Fs0021889810015608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">phenix.model vs. data: a high-level tool for the calculation of crystallographic model and data statistics</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Chen, Vincent B.; Headd, Jeffrey J.; Moriarty, Nigel W.; Richardson, Jane S.; Richardson, David C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-676</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.modelvsdata is a high-level command-line tool for the computation of crystallog. model and data statistics, and the evaluation of the fit of the model to data.  Anal. of all Protein Data Bank structures that have exptl. data available shows that in most cases the reported statistics, in particular R factors, can be reproduced within a few percentage points.  However, there are a no. of outliers where the recomputed R values are significantly different from those originally reported.  The reasons for these discrepancies are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBhMlnjsRfhLVg90H21EOLACvtfcHk0ljjhPbvmWlA_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVygtr4%253D&md5=5cd851a911ecd8b518e3ec4cdd5d9dd7</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2Fs0021889810015608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889810015608%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DPhenix.Model_Vs_Data%253A%2520A%2520High-Level%2520Tool%2520for%2520the%2520Calculation%2520of%2520Crystallographic%2520Model%2520and%2520Data%2520Statistics%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D669%26epage%3D676%26doi%3D10.1107%2Fs0021889810015608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljjhPbvmWlA_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MQ1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MQ1','PDB','5MQ1'); return false;">PDB: 5MQ1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PPA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PPA','PDB','6PPA'); return false;">PDB: 6PPA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UZF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UZF','PDB','6UZF'); return false;">PDB: 6UZF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1E" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1E','PDB','6V1E'); return false;">PDB: 6V1E</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EU1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EU1','PDB','5EU1'); return false;">PDB: 5EU1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1F','PDB','6V1F'); return false;">PDB: 6V1F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5F1H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5F1H','PDB','5F1H'); return false;">PDB: 5F1H</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V17" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V17','PDB','6V17'); return false;">PDB: 6V17</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1B" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1B','PDB','6V1B'); return false;">PDB: 6V1B</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V16" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V16','PDB','6V16'); return false;">PDB: 6V16</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V14" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V14','PDB','6V14'); return false;">PDB: 6V14</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1H','PDB','6V1H'); return false;">PDB: 6V1H</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IGM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IGM','PDB','5IGM'); return false;">PDB: 5IGM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1K','PDB','6V1K'); return false;">PDB: 6V1K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1L" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1L','PDB','6V1L'); return false;">PDB: 6V1L</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1U" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1U','PDB','6V1U'); return false;">PDB: 6V1U</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0U" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0U','PDB','6V0U'); return false;">PDB: 6V0U</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0Q','PDB','6V0Q'); return false;">PDB: 6V0Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0S','PDB','6V0S'); return false;">PDB: 6V0S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0X" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0X','PDB','6V0X'); return false;">PDB: 6V0X</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FYI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FYI','PDB','6FYI'); return false;">PDB: 6FYI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MIY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MIY','PDB','3MIY'); return false;">PDB: 3MIY</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i12"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01980">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_28721"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01980?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01980</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_001.csv">CSV</a>)</p></li><li><p class="inline">Original ITC data, crystallization conditions for protein–inhibitor complexes, X-ray data collection, refinement statistics, and electron density maps (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_001.csv">jm9b01980_si_001.csv (0.76 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01980/suppl_file/jm9b01980_si_002.pdf">jm9b01980_si_002.pdf (3.19 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PPA">6PPA</a> (apo), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1E">6V1E</a> (BI-7273), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1F">6V1F</a> (BI-9564), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1H">6V1H</a> (bromosporine), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V17">6V17</a> (I-BRD9), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0Q">6V0Q</a> (TG003) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V16">6V16</a> (TP-472) for BRD7. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UZF">6UZF</a> (apo), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V14">6V14</a> (TP-472), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0X">6V0X</a> (sunitinib), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1B">6V1B</a> (I-BRD9) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0S">6V0S</a> (TG003) for BRD9. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1K">6V1K</a> (BI-7273), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1L">6V1L</a> (BI-9564), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0U">6V0U</a> (bromosporine), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V1U">6V1U</a> (TP-472) for BRD4-1.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01980&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01980%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-6" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01980" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a97c9afdc2483","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
