
<html lang="en"     class="pb-page"  data-request-id="0f7af460-1eee-4e8e-820e-0ceb5ef44330"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-9;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.9b01720;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Based Design and Pharmacokinetic Optimization of Covalent Allosteric Inhibitors of the Mutant GTPase KRASG12C" /></meta><meta name="dc.Creator" content="Jason G.  Kettle" /></meta><meta name="dc.Creator" content="Sharan K.  Bagal" /></meta><meta name="dc.Creator" content="Sue  Bickerton" /></meta><meta name="dc.Creator" content="Michael S.  Bodnarchuk" /></meta><meta name="dc.Creator" content="Jason  Breed" /></meta><meta name="dc.Creator" content="Rodrigo J.  Carbajo" /></meta><meta name="dc.Creator" content="Doyle J.  Cassar" /></meta><meta name="dc.Creator" content="Atanu  Chakraborty" /></meta><meta name="dc.Creator" content="Sabina  Cosulich" /></meta><meta name="dc.Creator" content="Iain  Cumming" /></meta><meta name="dc.Creator" content="Michael  Davies" /></meta><meta name="dc.Creator" content="Andrew  Eatherton" /></meta><meta name="dc.Creator" content="Laura  Evans" /></meta><meta name="dc.Creator" content="Lyman  Feron" /></meta><meta name="dc.Creator" content="Shaun  Fillery" /></meta><meta name="dc.Creator" content="Emma S.  Gleave" /></meta><meta name="dc.Creator" content="Frederick W.  Goldberg" /></meta><meta name="dc.Creator" content="Stephanie  Harlfinger" /></meta><meta name="dc.Creator" content="Lyndsey  Hanson" /></meta><meta name="dc.Creator" content="Martin  Howard" /></meta><meta name="dc.Creator" content="Rachel  Howells" /></meta><meta name="dc.Creator" content="Anne  Jackson" /></meta><meta name="dc.Creator" content="Paul  Kemmitt" /></meta><meta name="dc.Creator" content="Jennifer K.  Kingston" /></meta><meta name="dc.Creator" content="Scott  Lamont" /></meta><meta name="dc.Creator" content="Hilary J.  Lewis" /></meta><meta name="dc.Creator" content="Songlei  Li" /></meta><meta name="dc.Creator" content="Libin  Liu" /></meta><meta name="dc.Creator" content="Derek  Ogg" /></meta><meta name="dc.Creator" content="Christopher  Phillips" /></meta><meta name="dc.Creator" content="Radek  Polanski" /></meta><meta name="dc.Creator" content="Graeme  Robb" /></meta><meta name="dc.Creator" content="David  Robinson" /></meta><meta name="dc.Creator" content="Sarah  Ross" /></meta><meta name="dc.Creator" content="James M.  Smith" /></meta><meta name="dc.Creator" content="Michael  Tonge" /></meta><meta name="dc.Creator" content="Rebecca  Whiteley" /></meta><meta name="dc.Creator" content="Junsheng  Yang" /></meta><meta name="dc.Creator" content="Longfei  Zhang" /></meta><meta name="dc.Creator" content="Xiliang  Zhao" /></meta><meta name="dc.Description" content="Attempts to directly drug the important oncogene KRAS have met with limited success despite numerous efforts across industry and academia. The KRASG12C mutant represents an “Achilles heel” and has ..." /></meta><meta name="Description" content="Attempts to directly drug the important oncogene KRAS have met with limited success despite numerous efforts across industry and academia. The KRASG12C mutant represents an “Achilles heel” and has ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 5, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01720" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01720" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01720" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01720" /></link>
        
    
    

<title>Structure-Based Design and Pharmacokinetic Optimization of Covalent Allosteric Inhibitors of the Mutant GTPase KRASG12C | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01720" /></meta><meta property="og:title" content="Structure-Based Design and Pharmacokinetic Optimization of Covalent Allosteric Inhibitors of the Mutant GTPase KRASG12C" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0016.jpeg" /></meta><meta property="og:description" content="Attempts to directly drug the important oncogene KRAS have met with limited success despite numerous efforts across industry and academia. The KRASG12C mutant represents an “Achilles heel” and has recently yielded to covalent targeting with small molecules that bind the mutant cysteine and create an allosteric pocket on GDP-bound RAS, locking it in an inactive state. A weak inhibitor at this site was optimized through conformational locking of a piperazine–quinazoline motif and linker modification. Subsequent introduction of a key methyl group to the piperazine resulted in enhancements in potency, permeability, clearance, and reactivity, leading to identification of a potent KRASG12C inhibitor with high selectivity and excellent cross-species pharmacokinetic parameters and in vivo efficacy." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01720"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01720">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01720&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01720&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01720&amp;href=/doi/10.1021/acs.jmedchem.9b01720" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4468-4483</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01622" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Drug Annotation</span><a href="/doi/10.1021/acs.jmedchem.9b01936" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Design and Pharmacokinetic Optimization of Covalent Allosteric Inhibitors of the Mutant GTPase KRAS<sup>G12C</sup></span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jason G. Kettle</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jason G. Kettle</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +44(0)1625517920. Email: <a href="/cdn-cgi/l/email-protection#dab0bba9b5b4f4b1bfaeaeb6bf9abba9aea8bba0bfb4bfb9bbf4b9b5b7"><span class="__cf_email__" data-cfemail="1d777c6e7273337678696971785d7c6e696f7c677873787e7c337e7270">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jason+G.++Kettle">Jason G. Kettle</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7373-0758" title="Orcid link">http://orcid.org/0000-0001-7373-0758</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sharan K. Bagal</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sharan K. Bagal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sharan+K.++Bagal">Sharan K. Bagal</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sue Bickerton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sue Bickerton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sue++Bickerton">Sue Bickerton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael S. Bodnarchuk</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael S. Bodnarchuk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael+S.++Bodnarchuk">Michael S. Bodnarchuk</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9172-1203" title="Orcid link">http://orcid.org/0000-0002-9172-1203</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jason Breed</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jason Breed</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jason++Breed">Jason Breed</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rodrigo J. Carbajo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rodrigo J. Carbajo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rodrigo+J.++Carbajo">Rodrigo J. Carbajo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Doyle J. Cassar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Doyle J. Cassar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Doyle+J.++Cassar">Doyle J. Cassar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Atanu Chakraborty</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Atanu Chakraborty</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Atanu++Chakraborty">Atanu Chakraborty</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sabina Cosulich</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sabina Cosulich</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sabina++Cosulich">Sabina Cosulich</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Iain Cumming</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Iain Cumming</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Iain++Cumming">Iain Cumming</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael Davies</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Davies</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Davies">Michael Davies</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew Eatherton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew Eatherton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew++Eatherton">Andrew Eatherton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laura Evans</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laura Evans</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laura++Evans">Laura Evans</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lyman Feron</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lyman Feron</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lyman++Feron">Lyman Feron</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shaun Fillery</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shaun Fillery</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shaun++Fillery">Shaun Fillery</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0314-8137" title="Orcid link">http://orcid.org/0000-0002-0314-8137</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Emma S. Gleave</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emma S. Gleave</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emma+S.++Gleave">Emma S. Gleave</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Frederick W. Goldberg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Frederick W. Goldberg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Frederick+W.++Goldberg">Frederick W. Goldberg</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4874-1700" title="Orcid link">http://orcid.org/0000-0002-4874-1700</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephanie Harlfinger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephanie Harlfinger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephanie++Harlfinger">Stephanie Harlfinger</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lyndsey Hanson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lyndsey Hanson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Alderley Park SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lyndsey++Hanson">Lyndsey Hanson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin Howard</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Howard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Howard">Martin Howard</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rachel Howells</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rachel Howells</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rachel++Howells">Rachel Howells</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anne Jackson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anne Jackson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anne++Jackson">Anne Jackson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul Kemmitt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul Kemmitt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Kemmitt">Paul Kemmitt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jennifer K. Kingston</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jennifer K. Kingston</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer+K.++Kingston">Jennifer K. Kingston</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Scott Lamont</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Scott Lamont</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Scott++Lamont">Scott Lamont</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hilary J. Lewis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hilary J. Lewis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hilary+J.++Lewis">Hilary J. Lewis</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Songlei Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Songlei Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaron Beijing Co., Ltd. 6 Taihe Road BDA, Beijing 100176 P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Songlei++Li">Songlei Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Libin Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Libin Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaron Beijing Co., Ltd. 6 Taihe Road BDA, Beijing 100176 P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Libin++Liu">Libin Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Derek Ogg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Derek Ogg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Derek++Ogg">Derek Ogg</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christopher Phillips</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christopher Phillips</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christopher++Phillips">Christopher Phillips</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Radek Polanski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Radek Polanski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Radek++Polanski">Radek Polanski</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Graeme Robb</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Graeme Robb</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Graeme++Robb">Graeme Robb</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4531-4375" title="Orcid link">http://orcid.org/0000-0002-4531-4375</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Robinson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Robinson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Robinson">David Robinson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sarah Ross</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sarah Ross</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sarah++Ross">Sarah Ross</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James M. Smith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James M. Smith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+M.++Smith">James M. Smith</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael Tonge</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Tonge</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Tonge">Michael Tonge</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rebecca Whiteley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rebecca Whiteley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rebecca++Whiteley">Rebecca Whiteley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Junsheng Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Junsheng Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaron Beijing Co., Ltd. 6 Taihe Road BDA, Beijing 100176 P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Junsheng++Yang">Junsheng Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Longfei Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Longfei Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaron Beijing Co., Ltd. 6 Taihe Road BDA, Beijing 100176 P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Longfei++Zhang">Longfei Zhang</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Xiliang Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiliang Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaron Beijing Co., Ltd. 6 Taihe Road BDA, Beijing 100176 P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiliang++Zhao">Xiliang Zhao</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01720&amp;href=/doi/10.1021%2Facs.jmedchem.9b01720" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4468–4483</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 5, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 October 2019</li><li><span class="item_label"><b>Published</b> online</span>5 February 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 May 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01720" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01720</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4468%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJason%2BG.%2BKettle%252C%2BSharan%2BK.%2BBagal%252C%2BSue%2BBickerton%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D9%26contentID%3Dacs.jmedchem.9b01720%26title%3DStructure-Based%2BDesign%2Band%2BPharmacokinetic%2BOptimization%2Bof%2BCovalent%2BAllosteric%2BInhibitors%2Bof%2Bthe%2BMutant%2BGTPase%2BKRASG12C%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4483%26publicationDate%3DMay%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01720"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">5969</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">11</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01720" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Design and Pharmacokinetic Optimization of Covalent Allosteric Inhibitors of the Mutant GTPase KRASG12C&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jason&quot;,&quot;last_name&quot;:&quot;G. Kettle&quot;},{&quot;first_name&quot;:&quot;Sharan&quot;,&quot;last_name&quot;:&quot;K. Bagal&quot;},{&quot;first_name&quot;:&quot;Sue&quot;,&quot;last_name&quot;:&quot;Bickerton&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;S. Bodnarchuk&quot;},{&quot;first_name&quot;:&quot;Jason&quot;,&quot;last_name&quot;:&quot;Breed&quot;},{&quot;first_name&quot;:&quot;Rodrigo&quot;,&quot;last_name&quot;:&quot;J. Carbajo&quot;},{&quot;first_name&quot;:&quot;Doyle&quot;,&quot;last_name&quot;:&quot;J. Cassar&quot;},{&quot;first_name&quot;:&quot;Atanu&quot;,&quot;last_name&quot;:&quot;Chakraborty&quot;},{&quot;first_name&quot;:&quot;Sabina&quot;,&quot;last_name&quot;:&quot;Cosulich&quot;},{&quot;first_name&quot;:&quot;Iain&quot;,&quot;last_name&quot;:&quot;Cumming&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Davies&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;Eatherton&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;Evans&quot;},{&quot;first_name&quot;:&quot;Lyman&quot;,&quot;last_name&quot;:&quot;Feron&quot;},{&quot;first_name&quot;:&quot;Shaun&quot;,&quot;last_name&quot;:&quot;Fillery&quot;},{&quot;first_name&quot;:&quot;Emma&quot;,&quot;last_name&quot;:&quot;S. Gleave&quot;},{&quot;first_name&quot;:&quot;Frederick&quot;,&quot;last_name&quot;:&quot;W. Goldberg&quot;},{&quot;first_name&quot;:&quot;Stephanie&quot;,&quot;last_name&quot;:&quot;Harlfinger&quot;},{&quot;first_name&quot;:&quot;Lyndsey&quot;,&quot;last_name&quot;:&quot;Hanson&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Howard&quot;},{&quot;first_name&quot;:&quot;Rachel&quot;,&quot;last_name&quot;:&quot;Howells&quot;},{&quot;first_name&quot;:&quot;Anne&quot;,&quot;last_name&quot;:&quot;Jackson&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Kemmitt&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;K. Kingston&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;Lamont&quot;},{&quot;first_name&quot;:&quot;Hilary&quot;,&quot;last_name&quot;:&quot;J. Lewis&quot;},{&quot;first_name&quot;:&quot;Songlei&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Libin&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Derek&quot;,&quot;last_name&quot;:&quot;Ogg&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;Phillips&quot;},{&quot;first_name&quot;:&quot;Radek&quot;,&quot;last_name&quot;:&quot;Polanski&quot;},{&quot;first_name&quot;:&quot;Graeme&quot;,&quot;last_name&quot;:&quot;Robb&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Robinson&quot;},{&quot;first_name&quot;:&quot;Sarah&quot;,&quot;last_name&quot;:&quot;Ross&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;M. Smith&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Tonge&quot;},{&quot;first_name&quot;:&quot;Rebecca&quot;,&quot;last_name&quot;:&quot;Whiteley&quot;},{&quot;first_name&quot;:&quot;Junsheng&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Longfei&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Xiliang&quot;,&quot;last_name&quot;:&quot;Zhao&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;4468-4483&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01720&quot;},&quot;abstract&quot;:&quot;Attempts to directly drug the important oncogene KRAS have met with limited success despite numerous efforts across industry and academia. The KRASG12C mutant represents an “Achilles heel” and has recently yielded to covalent targeting with small molecules that bind the mutant cysteine and create an allosteric pocket on GDP-bound RAS, locking it in an inactive state. A weak inhibitor at this site was optimized through conformational locking of a piperazine–quinazoline motif and linker modification. Subsequent introduction of a key methyl group to the piperazine resulted in enhancements in potency, permeability, clearance, and reactivity, leading to identification of a potent KRASG12C inhibitor with high selectivity and excellent cross-species pharmacokinetic parameters and in vivo efficacy.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01720&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01720" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01720&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01720" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01720&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01720" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01720&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01720&amp;href=/doi/10.1021/acs.jmedchem.9b01720" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01720" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01720" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01720%26sid%3Dliteratum%253Aachs%26pmid%3D32023060%26genre%3Darticle%26aulast%3DKettle%26date%3D2020%26atitle%3DStructure-Based%2BDesign%2Band%2BPharmacokinetic%2BOptimization%2Bof%2BCovalent%2BAllosteric%2BInhibitors%2Bof%2Bthe%2BMutant%2BGTPase%2BKRASG12C%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D9%26spage%3D4468%26epage%3D4483%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292504" title="Permeability">Permeability</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/jmcmar.2020.63.issue-9/20200514/jmcmar.2020.63.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Attempts to directly drug the important oncogene KRAS have met with limited success despite numerous efforts across industry and academia. The KRAS<sup>G12C</sup> mutant represents an “Achilles heel” and has recently yielded to covalent targeting with small molecules that bind the mutant cysteine and create an allosteric pocket on GDP-bound RAS, locking it in an inactive state. A weak inhibitor at this site was optimized through conformational locking of a piperazine–quinazoline motif and linker modification. Subsequent introduction of a key methyl group to the piperazine resulted in enhancements in potency, permeability, clearance, and reactivity, leading to identification of a potent KRAS<sup>G12C</sup> inhibitor with high selectivity and excellent cross-species pharmacokinetic parameters and in vivo efficacy.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60584" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60584" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">KRAS, NRAS, and HRAS are small GTPase proteins which act as molecular switches, cycling between inactive GDP-bound and active GTP-bound forms, transmitting signals from membrane bound receptors to regulate cellular processes. Nucleotide cycling is carefully controlled by guanine nucleotide exchange factors (GEFs) which promote nucleotide exchange, and GTPase activating proteins (GAPs), which stimulate GTPase activity. RAS-GTP interacts with and regulates the activity of several important cellular effector proteins, the best characterized of these being the RAF kinases, regulating the RAF–MEK–ERK pathway, and the phosphoinositide 3-kinases (PI3K), regulating the PI3K–AKT–TOR pathway. KRAS is frequently mutated or amplified in human tumors with KRAS alterations being enriched in pancreatic adenocarcinoma (PDAC) (72%), colorectal cancer (CRC) (including 69% of the genomically stable subtype), and lung adenocarcinoma (LUAD) (33%).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Oncogenic mutations of KRAS are usually found within the codons encoding glycine 12 (G12), glycine 13 (G13), or glutamine 61 (Q61), which are located within the active site of KRAS.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Mutations of these residues impair intrinsic GTPase activity, thereby increasing cellular KRAS-GTP levels and causing aberrant activation of downstream effector pathways, driving tumor cell proliferation and survival.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> Cysteine mutations at codon 12 represent an oncogenic KRAS alteration. KRAS<sup>G12C</sup> is the most common KRAS mutation found in LUAD, due to genetic aberrations driven by smoking, and is found in 13–14.5% of LUAD tumors (The Cancer Genome Atlas (TCGA) and Foundation Medicine/Flatiron data sets). Although less frequent, KRAS<sup>G12C</sup> mutations are also found in 2.6–2.9% of CRC patients (Memorial Sloan Kettering (MSK), Dana-Farber Cancer Institute (DFCI) and TCGA data sets), 1.1–2.1% of endometrial carcinomas (TCGA and MSK data sets) and 0.5–1.6% of pancreatic adenocarcinomas (Queensland Centre for Medical Genomics (QCMG) and TCGA data sets). Oncogenic KRAS including KRAS<sup>G12C</sup>, has been shown in preclinical studies to be a key driver of tumor proliferation and survival.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Moreover, recent data reported from phase I clinical trials (NCT03600883) indicates inhibitors of KRAS<sup>G12C</sup> may be effective in patients.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">Despite a recent resurgence in approaches designed to directly target the oncogene KRAS with small molecules, progress to date has been modest, consistent with the established view that RAS is an “undruggable”, featureless protein, absent of discernible targetable binding pockets. <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> shows RAS bound to its GEF Son-of-Sevenless (SOS), superimposed with ligands for which PDB depositions exist, and which therefore should be viewed as confirmed direct ligands for this important target. A wide diversity of binding sites, hit-finding techniques and potencies have been described for ligands against several RAS isoforms and include analogues which block binding of SOS to KRAS<sup>Q61H</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FA1">6FA1</a>) derived from an intracellular antibody fragment displacement screen.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Fragment-based lead generation (FBLG) has been utilized to generate ligands which bind in a similar region to this and block binding of SOS to KRAS<sup>G12D</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GJ8">6GJ8</a>),<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> which are an iteration on earlier less potent binders at this same site (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EPV">4EPV</a>).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> This pocket was in fact first reported by Genentech (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DSU">4DSU</a>)<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> following an NMR fragment screen and then later by AstraZeneca following a crystallographic cocktail soaking FBLG campaign against the HRAS–SOS complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4URZ">4URZ</a>), one which also yielded ligands directly bound to SOS (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4URU">4URU</a>).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> This same SOS binding site was subsequently observed to accommodate both agonists (PDBs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6D5G">6D5G</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EPM">6EPM</a>)<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> and antagonists of SOS (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OVI">5OVI</a>)<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> by independent groups. Large cyclic peptides have been shown to bind to inactive KRAS<sup>G12D</sup> and act to block GEF binding (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XCO">5XCO</a>),<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> thereby preventing activation of KRAS. Three distinct covalent approaches have been described, two of which are specific to the G12C mutation; ligands which bind to KRAS<sup>G12C</sup> in an allosteric pocket on the GDP bound form and are discussed below (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5V6S">5V6S</a>)<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and a covalent nucleotide mimetic approach based on modified guanine nucleotides (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KYK">5KYK</a>).<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The final covalent approach adopted reactive fragments to target C118 on HRAS (but present in all RAS isoforms), which is becomes accessible upon SOS binding to RAS (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4US2">4US2</a>).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Crystal structure overlay of the various approaches to either covalently modify RAS (shown in purple), block the interaction with the GEF SOS, or bind directly to allosteric sites within SOS (shown in gray). Annotations indicate PDB code, protein of interest, and year of disclosure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 2013 breakthrough disclosure, Shokat et al. described the elegant discovery of ligands that specifically target the codon 12 cysteine mutant of KRAS.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Crystallographic data highlighted the potential to covalently capture Cys12 of GDP-bound KRAS, with the creation of a new allosteric regulatory site under the switch II loop important for effector recognition, rendering KRAS locked in an inactive state. Reliance on this cysteine motif for binding has a number of advantages, both in the potency and subsequent selectivity that could be achieved for KRAS<sup>G12C</sup> inhibition, notably over wild-type KRAS. A disulfide fragment-based screening approach was utilized to find hits that were capable of binding reversibly and induce this novel pocket, followed by iteration with more drug-like Michael acceptor warheads. Further optimization was conducted to generate agents capable of potently inhibiting KRAS function in a cellular setting, such as ARS-853, <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> followed by subsequent scaffold hopping to quinazoline-based inhibitors such as ARS-1620, <b>2</b>, capable of delivering in vivo proof-of-concept activity for the first time.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> In more detailed mechanistic studies, it was shown that compounds <b>1</b> and <b>2</b> possess only very weak reversible binding affinity for KRAS but that the rate of inactivation by covalent modification of Cys12 is rapid. This is despite the fact the electrophilicity of the warhead is not especially high, nor that Cys12 in KRAS is particularly nucleophilic. It was suggested that Lys16 of KRAS can direct and activate the acrylamide in <b>1</b> for nucleophilic attack by Cys12, effectively stabilizing the developing negative charge upon inactivation.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Such an observation is intriguing insomuch as this upends traditional thinking about the importance of reversible affinity in covalent inhibitor design and the fine-tuning of warhead reactivity required to achieve good pharmaceutical properties.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Following the disclosure of tool compounds like <b>1</b>, new agents have subsequently been disclosed. KRAS<sup>G12C</sup> inhibitors AMG 510,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a><b>3</b> (Amgen) and MRTX849 (Mirati/Array) have now entered first-in-human clinical testing and while the structure of MRTX849 is not yet reported, an earlier, related tool compound MRTX1257, <b>4</b>, has been published.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Wellspring Biosciences recently announced the intent to develop ARS-3248, but it is unclear if this compound is related in structure to ARS-1620.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of reported KRAS<sup>G12C</sup> inhibitors. ARS-853 (<b>1</b>) and ARS-1620 (<b>2</b>) from Araxes Pharma/Wellspring Biosciences, AMG 510 (<b>3</b>) from Amgen, and MRTX1257 (<b>4</b>) from Mirati Therapeutics/Array Biopharma.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54874" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54874" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A representative synthesis of compound <b>25</b>, the result of these optimization studies, is shown in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>, with detailed experimental conditions for additional compounds available as <a href="/doi/suppl/10.1021/acs.jmedchem.9b01720/suppl_file/jm9b01720_si_002.pdf" class="ext-link">Supporting Information</a>. Accordingly, the synthesis of piperazine precursor <b>13</b> began with amide coupling of alanine methyl ester and CBz-protected serine <b>5</b> to give dipeptide <b>6</b>, which is then cyclized to diketopiperazine <b>7</b> upon removal of the protecting group (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Borane reduction of <b>7</b> gives hydroxymethyl piperazine <b>8</b>, which underwent a short sequence of protecting group manipulation to give differentially protected BOC/CBz-protected piperazine <b>10</b> via <i>mono</i>-BOC piperazine <b>9</b>. Oxidation of <b>10</b> with sodium periodate to give acid <b>11</b> was followed by esterification <b>12</b> and CBz-removal by hydrogenation to give key intermediate <b>13</b>. Separately, bromoaniline <b>14</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) was chlorinated to give a mixture of <i>ortho</i>- and <i>para</i>-chloroanilines, favoring the latter by a ratio of ∼3:1 (the mixture was carried through unresolved). Aniline <b>15</b> was then converted to quinolone <b>17</b> via high temperature cyclization of intermediate <b>16</b>, which was isolated by filtration of the precipitated mixture. Any uncyclized products arising from the <i>ortho</i>-chlorination of starting aniline <b>14</b> remained soluble and were filtered out on quinolone <b>17</b> isolation. Nitration of <b>17</b> to give 3-nitroquinolone <b>18</b> was followed by chlorination to 4-chloroquinolone <b>19</b> using phosphoryl trichloride. Piperazine <b>13</b> underwent an S<sub>N</sub>Ar reaction with chloroquinoline <b>19</b> to afford <b>20</b> and iron-mediated reduction of the nitro group resulted in direct intramolecular cyclization with the piperazine ester to give tetracyclic amide <b>21</b>. Methylation to tertiary amide <b>22</b> occurred in high yield, and cross-coupling with fluorophenol boronic acid gave 8-arylquinoline <b>23</b>. Removal of the BOC protecting group was followed by separation of the atropisomers of <b>24</b>, which was then converted to covalent inhibitor <b>25</b> by reaction with acryloyl chloride.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0009.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Piperazine <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) methyl <span class="smallcaps smallerCapital">d</span>-alaninate hydrochloride, EDC, DIPEA, DCM, 0–25 °C, 16 h, 77%; (ii) cyclohexene, 10% Pd/C, MeOH, 32 h, 88%; (iii) borane, THF, 0–70 °C, 16 h, 94%; (iv) (Boc)<sub>2</sub>O, TEA, MeOH, 0–100 °C, 34 h, then 1.5 M NaOH, EtOH, 25–100 °C, 16 h; 99%; (v) CBz-Cl, 1 M NaOH, THF, 3–25 °C, 2 h, 87%; (vi) NaIO<sub>4</sub>, RuCl<sub>3</sub>, DCM, MeCN, dimethyl carbonate, 25 °C, 3 h, 88%; (vii) MeI, K<sub>2</sub>CO<sub>3</sub>, MeCN, 0–60 °C, 2 h, 90%; (viii) H<sub>2</sub>, 10% Pd/C, MeOH, 50 °C, 16 h, 99%.</p></p></figure><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) NCS, DMF, 25–50 °C, 3 h, 48%; (ii) 5-(methoxymethylene)-2,2-dimethyl-1,3-dioxane-4,6-dione, trimethoxymethane, 85–110 °C, 3 h; (iii) Dowtherm, 210 °C, 65% 2 steps; (iv) nitric acid, propionic acid, 125 °C, 2 h, 74%; (v) POCl<sub>3</sub>, toluene, 100 °C, 16 h, 81%; (vi) <b>13</b>, DIPEA, THF, 60 °C, 3 h, 77%; %; (vii) Fe, AcOH, 80 °C, 1 h, 97%; (viii) MeI, K<sub>2</sub>CO<sub>3</sub>, acetone, 40 °C, 16 h, 65%; (ix) (2-fluoro-6-hydroxyphenyl)boronic acid, RuPhos Pd G3, K<sub>2</sub>CO<sub>3</sub>, dioxane, water, 80 °C, 1.5 h, 85%; (x) HCl, MeOH, <i>i</i>-PrOH, 25 °C, 1 h, 34%; (xi) acryloyl chloride, TEA, DCM, then 1 M NH<sub>3</sub> in MeOH, 0 °C, 1 h, 37%.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21655" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21655" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The potency of inhibitors of KRAS<sup>G12C</sup> was assessed by using a complex assay that featured multiple steps capable of measuring different modes of inhibition, specifically, compounds were incubated with GDP-bound KRAS<sup>G12C</sup> then treated with the guanine nucleotide exchange factor SOS1 in the presence of excess GTP, with any remaining activated KRAS from this step being detected by treating with a fluorescently labeled RAF binding domain (RBD). Compounds that bound to KRAS-GDP and prevented nucleotide exchange would hold KRAS in its inactive state and thus prevent recognition by the RBD. A literature inhibitor of KRAS<sup>G12C</sup><b>26</b> was synthesized<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and confirmed as a modestly potent inhibitor with an IC<sub>50</sub> of 0.496 μM in this assay format. An X-ray crystal structure of <b>26</b> in KRAS<sup>G12C</sup> was solved in-house and provided insights into the binding mode of this series of compounds, which was in good agreement with the structure of related analogues that were subsequently reported (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The acrylamide moiety forms a covalent bond with Cys12, and Lys16 additionally forms a hydrogen bond to the acrylamide carbonyl group. His95, specific to <i>Kirsten</i> variants of RAS, forms a hydrogen bond to quinazoline N1. At C7 of the quinazoline, a single atropisomer of the indazole is observed, and it is this group that causes a shift in the switch II loop, forming a direct hydrogen bond to Asp69 and a water-mediated one to Ser65. A water molecule is observed close to the quinazoline C4 position that does not appear to engage the ligand. Computational solvent analysis using Grand Canonical Monte Carlo<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> indicated that this water molecule has the potential to be displaced, potentially offering a route to improved potency.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Potency and Selected in Vitro DMPK Properties of Tethered Quinoline Analogues<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0011.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">(1) All activity data are reported in μM and are the mean of at least <i>n</i> = 2 determinations and have SEM within 0.2 log units. (2) Measured log <i>D</i> at pH<sub>7.4</sub>. (3) Rat hepatocyte Cl<sub>int</sub> (μL/min/10<sup>–6</sup> cells). (4) Human microsomal Cl<sub>int</sub> (μL/min/mg). (5) Example 40 from ref <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a>. (6) Diastereomers separated but absolute stereochemistry not elucidated.</p></div></div><div></div></div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (left) Structure of compound <b>26</b> bound to KRAS<sup>G12C</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5V">6T5V</a>). The acrylamide forms a covalent bond to Cys12, with the carbonyl forming a hydrogen bond to Lys16. The quinazoline N1 forms a hydrogen bond to His95, while the 7-indazole forms direct and water mediated interactions with Asp69 and Ser65, of the switch II loop, respectively. (right) Overlay of the crystal structures of tethered analogue <b>29</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5U">6T5U</a>) and compound <b>26</b>. Compound <b>29</b> shows a minor frame-shift relative to <b>26</b>, such that the 7-indazole now forms direct interactions with Asp69 and Ser65, with other interactions being common. Note the presence of a displaceable water, shown as a red sphere, in both crystal structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We were struck by the conformation of the piperazine relative to quinazoline, specifically the observed twist between these two groups in the active, bound state. Analysis of the predicted relative energy barriers for torsion around this bond proved informative (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Three distinct low energy conformations are available to this 4-piperazinoquinazoline, at 60°, 150–210°, and 300°. The observed bioactive torsion at 312° is close to, but not exactly coincident with, just one of these minima, leading us to hypothesize that strategies to restrict or lock out the bioactive conformation might lead to enhanced potency. Conceptually at least, this could be achieved by a strategy of tethering the piperazine ring back onto the quinazoline core, via a tetracyclic ring system, and this was realized by the synthesis of a series of tetracyclic quinoline cores tethered with a fused morpholine ring, <b>27</b>–<b>30</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Synthesis of all four possible diastereomers (from two asymmetric centers, chiral methyl and restricted rotation around the biaryl bond) revealed that one in particular, <b>29</b>, showed a reasonable ∼5-fold improvement in potency relative to the untethered reference <b>26</b>. It should be noted, however, that <b>26</b> is as a racemic mixture of its active and less active atropisomers and also that the log <i>D</i> has increased for <b>29</b> relative to <b>26</b> as a consequence of the linker and switch to quinoline from quinazoline. This increase in log <i>D</i> also elicits an erosion in in vitro Cl<sub>int</sub> values in both rat hepatocytes and human microsomal systems, which in fact are worse for this diastereomer than the other less active three compounds <b>27</b>, <b>28</b>, and <b>30</b>. An expansion of the tether to a 7-membered ring as in <b>31</b> and <b>32</b> was also investigated, although in this case both 7-membered ring enantiomers were inferior to <b>29</b> and equally suffered from metabolic liabilities. Encouraged by the gains in potency, notwithstanding the worsening properties, <b>29</b> was selected for further optimization.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Relative energy profile for driving the 4-piperazinoquinazoline torsion, highlighted in red, between 0° and 360°. Multiple local minima were observed, only one of which is close to the bioactive torsion, leading to the hypothesis that locking out one of the minima via tethering may provide an entropic boost in the binding free energy. The right-hand panel shows the various minima in green, overlaid against the bioactive conformation in orange (calculations conducted on the <i>des</i>-indazole version of compound <b>26</b> to reduce computational complexity).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Switching the indazole for a fluorophenol, as demonstrated for compound <b>33</b> had a beneficial impact on lowering both lipophilicity and in vitro Cl<sub>int</sub> values (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Unlike the indazole, however, this fluorophenol moiety existed as an interconverting mixture of atropisomers, which could be separated analytically using chiral SFC but rapidly equilibrated upon standing. To lock out the active phenol atropisomer, small substituents were explored at quinazoline C8, with the best of these being 8-fluoro <b>34</b>. Compound <b>34</b> was a potent inhibitor of KRAS<sup>G12C</sup> with an IC<sub>50</sub> of 16 nM, and although log <i>D</i> was moderately high at 3.4, metabolism in both rat and human in vitro assays proved low. An 8-methyl analogue <b>35</b> was both less potent than fluorine and more lipophilic, resulting in a slight worsening of in vitro clearances. An 8-methoxy analogue <b>36</b> showed similar potency to methyl, but erosion of Cl<sub>int</sub> values, despite a more favorable log <i>D</i>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Optimization of the C8-Substituent in Combination with a C7-Phenol Group</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0012.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char=".">KRAS<sup>G12C</sup> IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">rat heps<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">human mics<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char=".">0.211</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="center">F</td><td class="colsep0 rowsep0" align="char" char=".">0.016</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="char" char=".">17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="center">Me</td><td class="colsep0 rowsep0" align="char" char=".">0.097</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="char" char=".">28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="center">MeO</td><td class="colsep0 rowsep0" align="char" char=".">0.070</td><td class="colsep0 rowsep0" align="char" char=".">3.0</td><td class="colsep0 rowsep0" align="char" char=".">56</td><td class="colsep0 rowsep0" align="char" char=".">40</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">All activity data are reported in μM and are the geometric mean of at least <i>n</i> = 2 determinations and have SEM within 0.2 log units.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Measured log <i>D</i> at pH<sub>7.4</sub>.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Rat hepatocyte Cl<sub>int</sub> (μL/min/10<sup>–6</sup> cells).</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Human microsomal Cl<sub>int</sub> (μL/min/mg).</p></div></div></div><div class="NLM_p">Having established preference for a 6-membered tether, we then sought to further probe the nature of the new ring itself, electing to conduct this SAR exploration with a quinazoline 8-H substituent to simplify the chemistry, with the intention that an 8-fluoro substituent would subsequently be reinstalled for optimal compounds (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Switching from morpholine <b>33</b> to piperazine <b>37</b> resulted in a slight erosion of potency, although a modest reduction in log <i>D</i> saw a trend toward lowered Cl<sub>int</sub> values accompanied by an erosion in permeability likely due to the additional donor present. Methylation of the piperazine nitrogen <b>38</b> gave a boost in potency but took log <i>D</i> above 4, with rat Cl<sub>int</sub> values in particular suffering as a result. Amide linkers were also explored; secondary lactam <b>39</b> was both more potent and less lipophilic than other linkers, and in vitro clearances values were low. Despite this, efflux in Caco2 cells was substantially increased, which we attribute to the additional hydrogen bond donor. Tertiary lactam <b>40</b> proved particularly interesting, resulting in improved potency relative to the starting ether, with lower log <i>D</i> leading to low Cl<sub>int</sub> values and very high permeability with minimal efflux observed.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Optimization of the Tether</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0013.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char=".">KRAS<sup>G12C</sup> IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">rat heps<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">human mics<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">Caco2 AB (ER)<a class="ref internalNav" href="#t3fn5" aria-label="e">e</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="center">O</td><td class="colsep0 rowsep0" align="center">H, H</td><td class="colsep0 rowsep0" align="char" char=".">0.211</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="center">35.9 (0.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="center">NH</td><td class="colsep0 rowsep0" align="center">H, H</td><td class="colsep0 rowsep0" align="char" char=".">0.387</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="center">13.9 (1.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="center"><i>N</i>-Me</td><td class="colsep0 rowsep0" align="center">H, H</td><td class="colsep0 rowsep0" align="char" char=".">0.132</td><td class="colsep0 rowsep0" align="char" char=".">4.1</td><td class="colsep0 rowsep0" align="char" char=".">99</td><td class="colsep0 rowsep0" align="char" char=".">26</td><td class="colsep0 rowsep0" align="center">26.8 (1.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="center">NH</td><td class="colsep0 rowsep0" align="center">O</td><td class="colsep0 rowsep0" align="char" char=".">0.073</td><td class="colsep0 rowsep0" align="char" char=".">3.0</td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="center">8.7 (17.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="center"><i>N</i>-Me</td><td class="colsep0 rowsep0" align="center">O</td><td class="colsep0 rowsep0" align="char" char=".">0.150</td><td class="colsep0 rowsep0" align="char" char=".">3.0</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="center">55.8 (0.9)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">All activity data are reported in μM and are the geometric mean of at least <i>n</i> = 2 determinations and have SEM within 0.2 log units.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Measured log <i>D</i> at pH<sub>7.4</sub>.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Rat hepatocyte Cl<sub>int</sub> (μL/min/10<sup>–6</sup> cells).</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">Human microsomal Cl<sub>int</sub> (μL/min/mg).</p></div><div class="footnote" id="t3fn5"><sup><sup>e</sup></sup><p class="last">Permeability Caco2 <i>P</i><sub>app</sub> A to B (1 × 10<sup>–6</sup> cm/s)<sup>3</sup> (efflux ratio), permeability measured at pH<sub>6.5</sub> in the presence of specific inhibitors for Pgp, BCRP and MRP2 and efflux ratio measured at pH<sub>7.4</sub> using 10 μM solution of compound.</p></div></div></div><div class="NLM_p">We were motivated to explore the impact of further substitution on the piperazine ring carrying the acrylamide, with simple methyl groups. In vitro metabolite ID studies had indicated that glutathione addition to the acrylamide was the major route of clearance in rat for compounds such as <b>33</b>, and it was hoped that such routes of elimination might be minimized by steric crowding of the acrylamide moiety. An increase in mouse whole blood stability for compounds related to <b>26</b> has been reported using this approach.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Methylation of the methylene group opposite the tether was achieved, with <i>R</i>-enantiomer <b>41</b> showing a 4-fold enhancement in potency compared to unsubstituted <b>33</b> and <i>S</i>-enantiomer <b>42</b> showing a marked loss in potency (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). A modest increase in lipophilicity for <b>33</b> was associated with a slight increase in in vitro Cl<sub>int</sub> values. The addition of this <i>R</i>-Me group also contributed to a positive reduction in general reactivity with glutathione (416 min half-life versus 334 min) and improved passive permeability (AB <i>P</i><sub>app</sub> 47.9 versus 35.9). To help put these glutathione half-lives into context, the corresponding values for marketed reversible kinase inhibitors <i>afatinib</i><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and <i>osimertinib</i><a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> in this same in-house assay are 28 and 121 min, respectively. Both methyl and ether substituents on the piperazine of <b>41</b> sit in an axial orientation and potentially contribute to limiting access to the acrylamide from either face, with the methyl group helping to improve permeability by lipophilic shielding of the adjacent polar carbonyl group. Having earlier established that the tertiary amide tether (as in <b>40</b>) was optimal, addition of the 8-fluoro group to lock out the active atropisomer gave compound <b>43</b> which showed an excellent balance of potency that was associated with reasonable lipophilicity, low Cl<sub>int</sub> values, and high permeability. Reactivity of the acrylamide to glutathione was also further reduced (585 min half-life), which may simply be a consequence of the increased steric bulk of the amide relative to ether, blocking the lower face of the molecule more effectively. Applying the chiral methyl SAR to <b>43</b> gave compound <b>25</b>, which showed a significant increase in potency, similar gains in permeability, and further reductions in glutathione reactivity (half-life of 929 min). In this instance, the increase in lipophilicity had no material impact on in vitro Cl<sub>int</sub> values relative to <b>43</b>.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Impact of Piperazine Methylation</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0014.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char=".">KRAS<sup>G12C</sup> IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">rat heps<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">human mics<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">Caco2 AB (ER)<a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">glutathione reactivity <i>t</i><sub>1/2</sub><a class="ref internalNav" href="#t4fn6" aria-label="f">f</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>33</b></td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="char" char=".">0.211</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="center">35.9 (0.5)</td><td class="colsep0 rowsep0" align="center">334</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>41</b></td><td class="colsep0 rowsep0" align="center"><i>R</i>-Me</td><td class="colsep0 rowsep0" align="char" char=".">0.052</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char=".">42</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="center">47.9 (0.7)</td><td class="colsep0 rowsep0" align="center">416</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>42</b></td><td class="colsep0 rowsep0" align="center"><i>S</i>-Me</td><td class="colsep0 rowsep0" align="char" char=".">7.653</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">74</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">325</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>43</b></td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="char" char=".">0.018</td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="center">27.2 (1.5)</td><td class="colsep0 rowsep0" align="center">585</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>25</b></td><td class="colsep0 rowsep0" align="center"><i>R</i>-Me</td><td class="colsep0 rowsep0" align="char" char="."><0.005</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="center">48.5 (1.1)</td><td class="colsep0 rowsep0" align="center">929</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">All activity data are reported in μM and are the geometric mean of at least <i>n</i> = 2 determinations and have SEM within 0.2 log units.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Measured log <i>D</i> at pH<sub>7.4</sub>.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Rat hepatocyte Cl<sub>int</sub> (μL/min/10<sup>–6</sup> cells).</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">Human microsomal Cl<sub>int</sub> (μL/min/mg).</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last">Permeability Caco2 <i>P</i><sub>app</sub> A to B (1 × 10<sup>–6</sup> cm/s)<sup>3</sup> (efflux ratio), permeability measured at pH<sub>6.5</sub> and efflux ratio measured at pH<sub>7.4</sub> using 10 μM solution of compound.</p></div><div class="footnote" id="t4fn6"><sup><sup>f</sup></sup><p class="last">Half-life (minutes) in the presence of 4.6 mM glutathione in 0.1 M PBS at pH<sub>7.4</sub> and 37 °C.</p></div></div></div><div class="NLM_p">We had noted higher in vivo clearances for some compounds than would be predicted based on scaling from in vitro values that we attributed to extrahepatic glutathione-mediated clearance from blood and other tissues.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Glutathione <i>S</i>-transferase (GST) enzymes, which catalyze the addition of glutathione to reactive electrophiles in the body are present not just in the cytosol of hepatocytes but additionally in the gastrointestinal tract, heart, lung, skin, and blood (erythrocytes) and in certain cases may lead to additional, extrahepatic metabolism that is not accounted for simply by scaling from hepatic fractions. This extrahepatic component to clearance was particularly acute in compounds devoid of any tether such as literature compound <b>26</b> but was noted in tethered analogues to a lesser degree also. Literature inhibitor <b>44</b> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>) had high in vivo clearance in rat which is not predicted from in vitro scaling (observed 97 mL/min/kg versus 35 mL/min/kg predicted, for a notional extrahepatic contribution of 62 mL/min/kg), resulting in a low bioavailability of 24%. Exposure in nude mouse following a single high oral dose of 100 mg/kg was also low, suggestive of similar clearance issues. The simple tethered analogue <b>33</b> in comparison showed reasonable in vivo pharmacokinetics in rat, with a moderate bioavailability of 64% and clearance of 35 mL/min/kg, which is much closer to the predicted value of 28 mL/min/kg. In mouse, significantly higher exposure was also seen. It is interesting to note that compounds <b>33</b> and <b>44</b>, close in structure, of similar reactivity and metabolic fate (based on metabolite ID studies) should display such different extrahepatic liabilities. It is tempting to speculate that there may be an element of GST isoform specificity to their observed in vivo clearances because while GST-α and GST-μ are present in hepatocyte cytosol, it is the GST-π isoform that is ubiquitous in other tissues.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The impact of the piperazine chiral methyl group can be seen in the PK profile of <b>41</b>. In vivo clearance was low and well predicted from hepatocytes, and this combined with improved permeability resulted in very high bioavailability, with similar improvements in mouse oral exposure. Similar benefits were seen when comparing amides <b>43</b> and <b>25</b>. Amide <b>43</b> has low clearance which is well predicted from in vitro scaling, but methylation to give <b>25</b> resulted in further reductions in clearance and improvements to bioavailability. The impact that this methyl group conferred in this series is noteworthy, with gains in permeability, potency, reduced extrahepatic clearance, and reduced glutathione reactivity all combining to yield a potent inhibitor of KRAS<sup>G12C</sup><b>25</b> with exceptional PK properties. Compound <b>25</b> was selected for more detailed profiling.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. In Vivo Rat DMPK Data for Selected KRAS<sup>G12C</sup> Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0015.gif" alt="" id="fx5" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">bioavailability (%)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">Cl (mL/min/kg)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">predicted Cl<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">Vdss (L/kg)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">mouse AUC<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">mouse <i>C</i><sub>max</sub><a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b><a class="ref internalNav" href="#t5fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">97</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="char" char=".">3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="char" char=".">96</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td><td class="colsep0 rowsep0" align="char" char=".">65</td><td class="colsep0 rowsep0" align="char" char=".">22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="char" char=".">56</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">156.4</td><td class="colsep0 rowsep0" align="char" char=".">18.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="char" char=".">94</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">628.5</td><td class="colsep0 rowsep0" align="char" char=".">85.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">PK in Han Wistar Rat following oral (2 μmol/kg) and iv doses (1 μmol/kg).</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">Predicted clearance from scaling of rat hepatocyte Cl<sub>int</sub> using the well-stirred model.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">PK in nude mouse following a single oral dose (100 mg/kg).</p></div><div class="footnote" id="t5fn4"><sup><sup>d</sup></sup><p class="last">Example 138 from WO2015054572.</p></div></div></div><div class="NLM_p">An X-ray structure of compound <b>25</b> complexed with KRAS<sup>G12C</sup> was obtained with a resolution of 1.4 Å (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Consistent with previous observations the acrylamide moiety forms a covalent bond with Cys12, while Lys16 forms a hydrogen bond to the acrylamide carbonyl group. It is noteworthy that the amidic carbonyl on the tether forms clear water-mediated interactions back to the NH backbone of Cys12; a motif which is believed to contribute to the boost in potency relative to the ether-tethered analogues. The quinoline N1 forms an interaction to His95, while the C7-fluorophenol group engaged Asp69 on the α<sub>2</sub>-helix and formed water-mediated interactions to Ser65 on the switch II-loop.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (left) Overview of KRAS<sup>G12C</sup> complexed to GDP and compound <b>25</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5B">6T5B</a>). The switch I loop is shown in orange and the switch II loop shown in yellow. (right) Enhanced view of compound <b>25</b> bound to KRAS<sup>G12C</sup>. The acrylamide forms a covalent bond to Cys12, with the carbonyl forming a hydrogen bond to Lys16. The quinoline N1 forms a hydrogen bond to His95, while the 7-fluorophenol forms a direct interaction with Asp69 and a water mediated interaction to Ser65. The chiral <i>R</i>-Me sits axially relative to the piperazine carbonyl, resulting in the observed diaxial conformation and the displacement of the water molecule seen in the example shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. Water-mediated interactions are seen between the amidic tether and the backbone NH of Cys12.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The chiral <i>R</i>-Me was found to induce a somewhat atypical diaxial conformation around the piperazine and was found to displace the water molecule previously identified bound to <b>29</b>. Intrigued by the gain in potency for <b>25</b> in comparison to <b>43</b>, we used low-mode molecular dynamics to explore the conformational landscape of these compounds. Compound <b>43</b> is predicted to adopt an equatorial linkage to the tether approximately 20% of the time, a conformation which is incompatible with the bioactive form where the tether is axial. In comparison <b>25</b> is predicted to exclusively sit in the bioactive diaxial conformation, an observation which is consistent with free-ligand NMR analysis and suggests that <b>25</b> can bind to KRAS<sup>G12C</sup> with a minimal entropic penalty. The combination of this effect, alongside the displacement of an energetically unstable water molecule, explains the exquisite potency of <b>25</b>.</div><div class="NLM_p">One theoretical concern with any agent designed to target noncatalytic cysteine residues in proteins is that of promiscuity. The specificity of <b>25</b> by global cysteine reactivity profiling was examined to assess selectivity against the wider cysteine-containing proteome (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). NCI-H358 cells which carry the G12C mutation were treated with 4 μM of compound <b>25</b> or vehicle for 4 h. Any remaining unreacted, solvent exposed cysteine residues were then reacted with iodoacetamide desthiobiotin and proteins were subsequently digested with trypsin. The resulting biotinylated peptides were enriched with streptavidin and analyzed by MS to determine which targets <b>25</b> was capable of binding to in addition to KRAS<sup>G12C</sup>. Free cysteine profiling of compound <b>25</b> in this manner demonstrates highly specific labeling of KRAS<sup>G12C</sup>; it is the only protein with significant >2-fold protein loss compared to the DMSO control.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Free cysteine proteomic profile of compound <b>25</b>. Each data point represents a biotinylated cysteine containing peptide with log<sub>2</sub> fold change and <i>p</i>-value compared to vehicle control calculated across three biological replicates per group. A log<sub>2</sub> fold change of >2 and a <i>p</i>-value of <0.01 are indicated in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The cellular potency and selectivity of compound <b>25</b> was explored across a panel of KRAS<sup>G12C</sup>, wild-type KRAS, and non-G12C mutant KRAS cells in both two-dimensional (2D) and three-dimensional (3D) proliferation assays, as differences in KRAS dependency in monolayer and anchorage indepent growth conditions have been reported.<a onclick="showRef(event, 'ref7 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref7 ref33 ref34">(7,33,34)</a> Growth of most of the KRAS<sup>G12C</sup> cells was potently inhibited by compound <b>25</b>, with a reasonable association between sensitivity in the different assay formats. Minimal activity was observed in cell lines not carrying this specific KRAS mutation (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>; <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). Differential antiproliferative activity was observed across the panel, with LU99 and SW1573 cells showing lack of sensitivity in both 2D and 3D assay formats (2D GI<sub>50</sub> data available as <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). Differences in binding of compound <b>25</b> to KRAS<sup>G12C</sup> are unlikely to explain this differential sensitivity, as similar levels of target engagement were achieved across the cell panel and is likely to reflect the importance of other drivers in the proliferation of these cell types. Daily oral dosing of mice with 100, 50, or 20 mg/kg of compound <b>25</b> was well tolerated and caused a dose dependent regression of KRAS<sup>G12C</sup> mutant Miapaca2 xenografts (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). Biomarker analysis of Miapaca2 tumors from mice treated with compound <b>25</b> showed a robust inhibition of <i>DUSP6</i>, a well characterized transcript regulated by RAS/MAPK pathway activity<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> at 3 and 24 h postdose. Taken together, these data indicate that compound <b>25</b> is a potent and selective inhibitor of KRAS<sup>G12C</sup>.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Differential antiproliferative activity of compound <b>25</b> across KRAS<sup>G12C</sup>, KRAS<sup>WT</sup>, and KRAS<sup>non-G12Cmutant</sup> cancer cells in 3D soft agar proliferation assays. Mean GI<sub>50</sub> (μM) from a minimum of three independent experiments. Error bars represent standard error (SE). Cellular target engagement was measured by mass spectrometry following incubation of cells for 6 h with 1 μM of compound <b>25</b>. Target engagement shown as a percent relative to vehicle control treated cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0008.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Efficacy and PD of compound <b>25</b> in KRAS<sup>G12C</sup> Miapaca2 xenograft model. (left) Mice bearing Miapaca2 tumors were treated daily with vehicle control or 100, 50, and 20 mg/kg of compound <b>25</b> via oral dosing for 2 weeks followed by 3 weeks of treatment. Data are shown as geometric mean of the tumor volume and SE. (right) Mice bearing Miapaca2 tumors were treated with a single dose of vehicle control or 100, 30, and 10 mg/kg of compound <b>25</b> via oral dosing. The tumor expression of <i>DUSP6</i> mRNA were measured at 3 and 24 h postdose by qRT-PCR. The expression was normalized to <i>POLRA</i> mRNA and expressed relative to the vehicle. Data are shown as individual tumor data, treatment group geometric mean, and SE.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33500" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33500" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The discovery of compound <b>25</b>, a potent and selective KRAS<sup>G12C</sup> inhibitor, has been described. An initial key step in its discovery was the application of a conformational tether to lock the piperazine group at C4 of a quinazoline ring to derive a tetracyclic quinoline which unexpectedly also markedly diminished extrahepatic GST-mediated clearance in preclinical species observed for related literature compounds. Optimization of the series was further advanced by switching to an amide tether, and crucially, methylation of the piperazine gave simultaneous improvements in potency, permeability, and clearance and a reduction in general warhead reactivity. The potency gains obtained from this methyl group may be explained by additional conformational locking of the bioactive pose and entropic and enthalpic gains from loss of an unstable water molecule identified in initial X-ray structures. Across species, the compound demonstrated low clearance and high bioavailability. Compound <b>25</b> was highly selective for reaction with the cysteine in KRAS<sup>G12C</sup> over other cysteine containing proteins and had a selective antiproliferative phenotype in cancer cells bearing the G12C mutation that led to strong tumor regressions in a mouse xenograft disease model. More detailed studies with compound <b>25</b> will be reported in due course.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53765" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53765" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemistry</h3><div class="NLM_p">Unless otherwise stated, commercially available reagents were used as supplied. All reactions requiring anhydrous conditions were conducted in dried apparatus under an atmosphere of nitrogen. Flash column chromatography was performed on Merck Kieselgel silica (article 9385) or on reversed phase silica (Fluka silica gel 90 C18) or on Silicycle cartridges (40–63 μm silica, 4–330 g weight) or on Grace resolve cartridges (4–120 g) or on RediSep Rf 1.5 Flash columns or on RediSep Rf high performance Gold Flash columns (150–415 g weight) or on RediSep Rf Gold C18 Reversed-phase columns (20–40 μm silica) either manually or automated using an Isco CombiFlash Companion system or similar system. Preparative reverse phase HPLC was performed on a Waters instrument (600/2700 or 2525) fitted with a ZMD or ZQ ESCi mass spectrometers and a Waters X-Terra or a Waters X-Bridge or a Waters SunFire reverse-phase column (C-18, 5 μm silica, 19 mm or 50 mm diameter, 100 mm length, flow rate of 40 mL/min) by using decreasingly polar mixtures of water (containing 1% ammonia) and acetonitrile or decreasingly polar mixtures of water (containing 0.1% formic acid) and acetonitrile as eluents. NMR chemical shift values were measured on the delta scale [proton magnetic resonance spectra were determined using a Bruker Avance 500 (500 MHz) and Bruker Avance 400 (400 MHz)instruments]. Measurements were taken at ambient temperature unless otherwise specified. The following abbreviations have been used: s, singlet; d, doublet; t, triplet; q, quartet; m, multiplet; dd, doublet of doublets; ddd, doublet of doublet of doublet; dt, doublet of triplets; bs, broad signal. In general, end products were also characterized by mass spectroscopy following liquid chromatography (LCMS or UPLC); in general, reverse-phase C18 silica was used with a flow rate of 1 mL/min and detection was by electrospray mass spectrometry and by UV absorbance recording a wavelength range of 220–320 nm. Analytical UPLC was performed on CSH C18 reverse-phase silica, using a Waters XSelect CSH C18 column with dimensions 2.1 mm × 50 mm and particle size 1.7 μm. Gradient analysis was employed by using decreasingly polar mixtures as eluent, for example, decreasingly polar mixtures of water (containing 0.1% formic acid or 0.1% ammonia) as solvent A and acetonitrile as solvent B. A typical 2 min analytical UPLC method would employ a solvent gradient over 1.3 min, at approximately 1 mL per minute, from a 97:3 mixture of solvents A and B, respectively, to a 3:97 mixture of solvents A and B. The reported molecular ion corresponds to the [M + H]<sup>+</sup> unless otherwise specified; for molecules with multiple isotopic patterns (Br, Cl etc.), the reported value is the one obtained for the lowest isotope mass unless otherwise specified. Ion exchange purification was generally performed using an SCX-2 (Biotage) cartridge. Where reactions refer to the use of a microwave, one of the following microwave reactors were used: Biotage Initiator, Personal Chemistry Emrys Optimizer, Personal Chemistry Smithcreator, or CEM Explorer. Purities were assessed using LCMS by UV absorbance and <sup>1</sup>H NMR and are ≥95% unless otherwise stated.</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Methyl <i>N</i>-[(Benzyloxy)carbonyl]-<span class="smallcaps smallerCapital">l</span>-seryl-<span class="smallcaps smallerCapital">d</span>-alaninate (<b>6</b>)</h4><div class="NLM_p last">A suspension of methyl <span class="smallcaps smallerCapital">d</span>-alaninate hydrochloride (25 g, 179.11 mmol) and <i>N</i>-[(benzyloxy)carbonyl]-<span class="smallcaps smallerCapital">l</span>-serine (<b>5</b>) (42.8 g, 179.11 mmol) in DCM (786 mL) was cooled in an ice-bath to 0 °C and EDC (41.2 g, 214.93 mmol) was added. DIPEA (109 mL, 626.89 mmol) was added dropwise over 20 min, and then the resulting solution was stirred at room temperature overnight. The reaction mixture was concentrated under reduced pressure to give a colorless residue. The residue was dissolved in ethyl acetate (500 mL) and washed with 1:1 water/aqueous saturated sodium hydrogen carbonate solution (600 mL). The organic portion was collected, and the aqueous was washed with a further portion of ethyl acetate (500 mL). The combined organic extracts were washed with aqueous 2 M hydrochloric acid solution (300 mL) and brine (300 mL), dried over magnesium sulfate, filtered, and concentrated to give methyl <i>N</i>-[(benzyloxy)carbonyl]-<span class="smallcaps smallerCapital">l</span>-seryl-<span class="smallcaps smallerCapital">d</span>-alaninate (<b>6</b>) (44.9 g, 77%) as a colorless solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD, 30 °C): 1.38 (3H, d), 3.71 (3H, s), 3.76 (2H, d), 4.24 (1H, t), 4.37–4.48 (1H, m), 5.11 (2H, s), 7.21–7.47 (5H, m). <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> = 325.1.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> (3<i>S</i>,6<i>R</i>)-3-(Hydroxymethyl)-6-methylpiperazine-2,5-dione (<b>7</b>)</h4><div class="NLM_p last">Methyl <i>N</i>-[(benzyloxy)carbonyl]-<span class="smallcaps smallerCapital">l</span>-seryl-<span class="smallcaps smallerCapital">d</span>-alaninate (<b>6</b>) (44.9 g, 138.44 mmol) was dissolved in MeOH (140 mL) and cyclohexene (90 mL, 888.54 mmol). Then 10% palladium on carbon (2.25 g, 10.9 mmol) was added in one portion and the resultant mixture was heated to reflux overnight. A further portion of MeOH (120 mL) was added, and the mixture was stirred at reflux for one hour. The reaction mixture was filtered (while hot) through Celite, and the Celite was washed with hot MeOH (2 × 300 mL). The filtrate was concentrated to give a gray semisolid. The solid was triturated with acetonitrile (200 mL), filtered, and washed with acetonitrile (100 mL). The solid was dried under vacuum at 40 °C for one hour to give (3<i>S</i>,6<i>R</i>)-3-(hydroxymethyl)-6-methylpiperazine-2,5-dione (<b>7</b>) (19.22 g, 88%) as a gray solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 30 °C): 1.24 (3H, d), 3.52 (1H, ddd), 3.68 (1H, q), 3.74 (1H, ddd), 3.92 (1H, q), 5.09 (1H, t), 7.87 (1H, s), 8.06 (1H, s).</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> [(2<i>R</i>,5<i>R</i>)-5-Methylpiperazin-2-yl]methanol Dihydrochloride (<b>8</b>)</h4><div class="NLM_p last">A vessel charged with (3<i>S</i>,6<i>R</i>)-3-(hydroxymethyl)-6-methylpiperazine-2,5-dione (<b>7</b>) (10 g, 63.23 mmol) was cooled in an ice-bath, and 1 M borane in THF (474 mL, 474.21 mmol) was added slowly. On addition, the mixture was brought to room temperature and then heated to 70 °C overnight. The reaction mixture was allowed to cool to room temperature and then cooled in an ice-bath to 0 °C. MeOH (125 mL) was added dropwise, and then aqueous 5 M hydrochloric acid solution (34 mL, 170 mmol) was added dropwise. The resultant solution was heated at 70 °C for 2 h. The mixture was cooled to room temperature and then in an ice-bath. The precipitate formed was filtered off, and the filter cake was washed with THF (60 mL). The solid was dried over vacuum to give [(2<i>R</i>,5<i>R</i>)-5-methylpiperazin-2-yl]methanol (<b>8</b>) (12.11 g, 94%) as a gray solid (as the dihydrochloride salt). <sup>1</sup>H NMR (400 MHz, DMSO, 30 °C): 1.32 (3H, d), 3.11 (2H, q), 3.37–3.78 (6H, m), 5.58 (1H, s), 9.98 (4H, s).</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> <i>tert</i>-Butyl (2<i>R</i>,5<i>R</i>)-5-(Hydroxymethyl)-2-methylpiperazine-1-carboxylate (<b>9</b>)</h4><div class="NLM_p last">[(2<i>R</i>,5<i>R</i>)-5-Methylpiperazin-2-yl]methanol dihydrochloride (<b>8</b>) (19.1 g, 94.04 mmol) was suspended in MeOH (92 mL) and cooled in an ice-bath to 0 °C. Triethylamine (40.6 mL, 291.52 mmol) was added, and then a solution of di-<i>tert</i>-butyl dicarbonate (49.3 g, 225.69 mmol) in MeOH (138 mL) was added dropwise over 30 min (keeping internal reaction temperature <10 °C). The resultant solution was stirred in the ice-bath for one hour before being brought to room temperature and then heated to 50 °C overnight. The reaction mixture was cooled to room temperature and concentrated under reduced pressure. The resultant residue was dissolved in ethanol (313 mL) and aqueous 1.5 M sodium hydroxide solution (313 mL, 470.20 mmol), and the solution was heated to 100 °C overnight. The reaction mixture was cooled to room temperature and brought to pH 9 using aqueous 2 M hydrochloric acid solution. The solution was extracted with chloroform (2 × 600 mL), and the combined organic extracts were dried over magnesium sulfate, filtered, and concentrated under reduced pressure to give a pale-yellow oil. The oil was redissolved in MeOH, concentrated, and dried under vacuum overnight to give <i>tert</i>-butyl (2<i>R</i>,5<i>R</i>)-5-(hydroxymethyl)-2-methylpiperazine-1-carboxylate (<b>9</b>) (21.45 g, 99%) as an oil which solidified on standing to a pale-yellow waxy solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD, 30 °C): 1.24 (3H, d), 1.46 (9H, s), 2.57 (1H, dd), 2.93–3.02 (1H, m), 3.07 (1H, dd), 3.24 (1H, dd), 3.54 (1H, dd), 3.59–3.68 (1H, m), 3.78 (1H, dd), 4.09–4.2 (1H, m). <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> = 231.1.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 1-Benzyl 4-<i>tert</i>-Butyl (2<i>R</i>,5<i>R</i>)-2-(Hydroxymethyl)-5-methylpiperazine-1,4-dicarboxylate (<b>10</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl (2<i>R</i>,5<i>R</i>)-5-(hydroxymethyl)-2-methylpiperazine-1-carboxylate (<b>9</b>) (18.5 g, 80.33 mmol) in THF (190 mL) was cooled in an ice-bath to 0 °C. Aqueous 1 M sodium hydroxide solution (88 mL, 88.36 mmol) was added, followed by benzyl chloroformate (11.99 mL, 84.34 mmol) (internal reaction temperature kept <10 °C). The mixture was stirred in the ice-bath for 60 min before being brought to room temperature and stirred. After one hour, the phases were separated and the aqueous was washed with ethyl acetate (2 × 50 mL). The combined organics were washed with brine (100 mL), dried over magnesium sulfate, filtered, and concentrated under reduced pressure to give a pale-yellow oil. The crude product was purified by flash silica chromatography, elution gradient 0–100% ethyl acetate in heptane. Pure fractions were evaporated to dryness to afford 1-benzyl 4-<i>tert</i>-butyl (2<i>R</i>,5<i>R</i>)-2-(hydroxymethyl)-5-methylpiperazine-1,4-dicarboxylate (<b>10</b>) (25.4 g, 87%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD, 30 °C): 1.12 (3H, dd), 1.46 (9H, s), 3.1–3.29 (2H, m), 3.53–3.66 (2H, m), 3.81 (1H, d), 3.88–4.02 (1H, m), 4.15–4.38 (2H, m), 5.06–5.26 (2H, m), 7.21–7.51 (5H, m). One exchangeable proton not seen. <i>m</i>/<i>z</i>: ES+ [M-Boc] = 265.1.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (2<i>R</i>,5<i>R</i>)-1-[(Benzyloxy)carbonyl]-4-(<i>tert</i>-butoxycarbonyl)-5-methylpiperazine-2-carboxylic Acid (<b>11</b>)</h4><div class="NLM_p last">Sodium periodate (29.8 g, 139.39 mmol) was dissolved in water (174 mL) and acetonitrile/dimethyl carbonate (93 mL, 1:1 ratio), and then ruthenium(III) chloride hydrate (157 mg, 0.70 mmol) was added. The solution was cooled in an ice-bath to 0 °C, and a solution of 1-benzyl 4-<i>tert</i>-butyl (2<i>R</i>,5<i>R</i>)-2-(hydroxymethyl)-5-methylpiperazine-1,4-dicarboxylate (<b>10</b>) (25.4 g, 69.70 mmol) in acetonitrile/dimethyl carbonate (232 mL, 1:1 ratio) was added slowly over 15 min (solution precipitates). On addition, the solution was brought to room temperature and stirred. After 45 min, the reaction mixture was diluted with water (300 mL) and extracted with ethyl acetate (3 × 300 mL). The combined organics were washed with brine (500 mL) and dried over magnesium sulfate. The mixture was filtered through a short pad of Celite and concentrated under reduced pressure to give a pale-brown residue. The residue was dissolved in diethyl ether (100 mL) and slowly concentrated and dried to give a pale-brown solid. The solid was triturated with heptane (100 mL) to give a fine solid, which was filtered, washed with heptane (100 mL), and dried under vacuum at 45 °C to give (2<i>R</i>,5<i>R</i>)-1-[(benzyloxy)carbonyl]-4-(<i>tert</i>-butoxycarbonyl)-5-methylpiperazine-2-carboxylic acid (<b>11</b>) (23.3 g, 88%) as a beige solid. <sup>1</sup>H NMR (400 MHz, DMSO, 30 °C): 1.07 (3H, dd), 1.38 (9H, s), 3.17–3.4 (2H, m), 3.55–3.71 (1H, m), 4.04–4.32 (2H, m), 4.56 (1H, s), 5.01–5.23 (2H, m), 7.19–7.51 (5H, m), 13.02 (1H, s). <i>m</i>/<i>z</i>: ES = [M – H]<sup>−</sup> = 377.2.</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 1-Benzyl 4-<i>tert</i>-Butyl 2-Methyl (2<i>R</i>,5<i>R</i>)-5-Methylpiperazine-1,2,4-tricarboxylate (<b>12</b>)</h4><div class="NLM_p last">A solution of (2<i>R</i>,5<i>R</i>)-1-[(benzyloxy)carbonyl]-4-(<i>tert</i>-butoxycarbonyl)-5-methylpiperazine-2-carboxylic acid (<b>11</b>) (23.3 g, 61.57 mmol) in acetonitrile (400 mL) was placed in a water bath, and potassium carbonate (12.76 g, 92.36 mmol) was added. Iodomethane (5.75 mL, 92.36 mmol) was added dropwise, and the suspension was heated to 60 °C. After 2 h, the reaction mixture was cooled to room temperature and concentrated under reduced pressure to give a pale-yellow residue. The residue was dissolved in ethyl acetate (300 mL) and water (300 mL). The organic portion was collected, and the aqueous was extracted with ethyl acetate (300 mL). The combined organic extracts were washed with brine (300 mL), dried over magnesium sulfate, filtered, and concentrated to give a pale-brown oil (27 g). The crude product was purified by flash silica chromatography, elution gradient 0–70% ethyl acetate in heptane. Pure fractions were evaporated to dryness to afford 1-benzyl 4-<i>tert</i>-butyl 2-methyl (2<i>R</i>,5<i>R</i>)-5-methylpiperazine-1,2,4-tricarboxylate (<b>12</b>) (21.7 g, 90%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 30 °C): 1.08 (3H, d), 1.39 (9H, s), 3.30 (3H, s), 3.64 (3H, t), 4.02–4.31 (2H, m), 4.69 (1H, s), 5.02–5.23 (2H, m), 7.23–7.5 (5H, m). <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> = 393.3.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 1-<i>tert</i>-Butyl 3-Methyl (3<i>R</i>,6<i>R</i>)-6-Methylpiperazine-1,3-dicarboxylate (<b>13</b>)</h4><div class="NLM_p last">1-Benzyl 4-<i>tert</i>-butyl 2-methyl (2<i>R</i>,5<i>R</i>)-5-methylpiperazine-1,2,4-tricarboxylate (<b>12</b>) (21.7 g, 55.29 mmol) was dissolved in MeOH (276 mL), and 10% palladium on carbon (4.34 g, 40.78 mmol) was added. The mixture was stirred at room temperature under a hydrogen atmosphere (0.5 bar) overnight. The reaction mixture was filtered through a short pad of Celite, washing with MeOH (200 mL), and the filtrate was concentrated under reduced pressure to give a gray residue. The residue was dissolved in MeOH (100 mL) and passed through a syringe filter. The resultant colorless solution was concentrated under reduced pressure and dried under vacuum at 45 °C to give 1-<i>tert</i>-butyl 3-methyl (3<i>R</i>,6<i>R</i>)-6-methylpiperazine-1,3-dicarboxylate (<b>13</b>) (14.1 g, 99%) as a pale-yellow oil. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 30 °C) 1.13 (3H, d), 1.37 (9H, s), 2.46 (1H, dd), 2.66 (1H, s), 2.93 (1H, dd), 3.15 (1H, dd), 3.56 (1H, d), 3.63 (3H, s), 3.95–4.07 (1H, m), 4.07–4.16 (1H, m). <i>m</i>/<i>z</i>: ES+ [M-Boc] = 158.9.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 3-Bromo-4-chloro-2-fluoroaniline (<b>15</b>)</h4><div class="NLM_p last">To a solution of 3-bromo-2-fluoroaniline (<b>14</b>) (10 g, 52.63 mmol) in DMF (50 mL) was added 1-chloropyrrolidine-2,5-dione (7.38 g, 55.26 mmol) portionwise, and the resultant mixture stirred at room temperature for 3 h. NOTE: delayed exotherm, after 1 h temperature had risen to ca. 50 °C. Mixture was poured onto ice–water (300 mL) then extracted with ethyl acetate (200 mL), and then organics were washed with water (2 × 200 mL) then brine. Organic extracts were dried and evaporated, and the crude product purified by flash silica chromatography, elution gradient 0–5% EtOAc in heptane. Pure fractions were evaporated to dryness to afford 3-bromo-4-chloro-2-fluoroaniline (<b>15</b>) (5.69 g, 48%) as a brown oil which solidified on standing. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) 5.58 (2H, s), 6.73–6.82 (1H, m), 7.12 (1H, dd).</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 7-Bromo-6-chloro-8-fluoroquinolin-4(1<i>H</i>)-one (<b>17</b>)</h4><div class="NLM_p last">2,2-Dimethyl-1,3-dioxane-4,6-dione (4.94 g, 34.3 mmol) was added to trimethoxymethane (17.06 mL, 155.93 mmol), warmed at 110 °C (reflux) over a period of 10 min under nitrogen. The resulting solution was stirred at 85 °C for 1.5 h under nitrogen. 3-Bromo-4-chloro-2-fluoroaniline (<b>15</b>) (7 g, 31.19 mmol) was added, and the solution stirred at 85 °C for another 60 min. The mixture was cooled to room temperature, and the solid formed in the mixture was collected by filtration and washed with <i>i</i>-PrOH (2 × 1 mL) and diethyl ether (2 × 2 mL) to give a pale-orange solid. This solid was added to stirred DOWTHERM (200 mL) at 210 °C: vigorous evolution of gas. The orange solution was stirred for another 40 min at 210 °C then cooled to room temperature and stirred for 10 min. To the mixture was added heptane (200 mL), and the mixture was filtered, with the solid collected washed on the filter with heptane then diethyl ether and dried on the filter to give 7-bromo-6-chloro-8-fluoroquinolin-4(1<i>H</i>)-one (<b>17</b>) (5.6 g, 65%) as a tan solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 30 °C) 6.15 (1H, d), 7.90 (1H, d), 8.01 (1H, d), 12.11 (1H, s). <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> 276.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 7-Bromo-6-chloro-8-fluoro-3-nitroquinolin-4(1<i>H</i>)-one (<b>18</b>)</h4><div class="NLM_p last">7-Bromo-6-chloro-8-fluoroquinolin-4(1<i>H</i>)-one (<b>17</b>) (5.6 g, 20.25 mmol) was added to stirred propionic acid (55 mL, 607.62 mmol), and the mixture was heated at 125 °C with stirring. Nitric acid (fuming) (1.697 mL, 40.51 mmol) was added dropwise, and the solution was stirred for 2 h at 125 °C before being allowed to cool to room temperature. Water (50 mL) was added, and the mixture was filtered. The solid collected was washed with water (2 × 20 mL) and diethyl ether (20 mL) then dried to give 7-bromo-6-chloro-8-fluoro-3-nitroquinolin-4(1<i>H</i>)-one (<b>18</b>) (4.81 g, 74%) as a pale solid, which was not purified further. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 30 °C) 8.15 (1H, d), 8.96 (1H, s), 13.32 (1H, s). <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> 321.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 7-Bromo-4,6-dichloro-8-fluoro-3-nitroquinoline (<b>19</b>)</h4><div class="NLM_p last">Phosphoryl trichloride (1.62 mL, 17.42 mmol) was added to 7-bromo-6-chloro-8-fluoro-3-nitroquinolin-4(1<i>H</i>)-one (<b>18</b>) (1.4 g, 4.35 mmol) in toluene (25 mL) at room temperature. The mixture was heated at 100 °C with stirring, at which point DMF (0.1 mL) was added and the mixture was stirred at 105 °C overnight. The solvents were removed by evaporation. Toluene (20 mL) was added and the solvents evaporated. The residue was taken up in DCM (100 mL) and poured into ice-cold satd NaHCO<sub>3</sub> (200 mL). The mixture was extracted with DCM (3 × 100 mL). The combined organic layers were washed with brine, dried, and evaporated to give 7-bromo-4,6-dichloro-8-fluoro-3-nitroquinoline (<b>19</b>) (1.2 g, 81%) as a light-brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 30 °C) 8.51 (1H, d), 9.48 (1H, d).</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 1-<i>tert</i>-Butyl 3-Methyl (3<i>R</i>,6<i>R</i>)-4-(7-Bromo-6-chloro-8-fluoro-3-nitroquinolin-4-yl)-6-methylpiperazine-1,3-dicarboxylate (<b>20</b>)</h4><div class="NLM_p last">A solution of 1-<i>tert</i>-butyl 3-methyl (3<i>R</i>,6<i>R</i>)-6-methylpiperazine-1,3-dicarboxylate (<b>13</b>) (310 mg, 1.20 mmol) in THF (4.56 mL) and DIPEA (0.226 mL, 1.30 mmol) was degassed with nitrogen for 5 min. 7-Bromo-4,6-dichloro-8-fluoro-3-nitroquinoline (<b>19</b>) (400 mg, 1.00 mmol) was added, and the mixture was heated to 60 °C. After three hours, the reaction mixture was cooled to room temperature and concentrated in vacuo. The brown residue was dissolved in DCM (20 mL) and washed with aqueous 1 M citric acid solution (20 mL). The organic portion was collected, and the aqueous extract was washed with a further portion of DCM (20 mL). The combined organic extracts were washed with brine (10 mL), passed through a hydrophobic frit, and concentrated to give an orange solid. The solid was triturated with diethyl ether (30 mL), filtered, and dried in air over vacuum to give 1-<i>tert</i>-butyl 3-methyl (3<i>R</i>,6<i>R</i>)-4-(7-bromo-6-chloro-8-fluoro-3-nitroquinolin-4-yl)-6-methylpiperazine-1,3-dicarboxylate (<b>20</b>) (430 mg, 77%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 30 °C): 1.19 (3H, d), 1.43 (9H, s), 3.29 (1H, s), 3.58 (3H, s), 3.66 (1H, d), 3.87 (1H, dd), 4.32 (2H, d), 4.47 (1H, d), 8.36 (1H, d), 9.16 (1H, s). <sup>19</sup>F NMR (376 MHz, DMSO, 30 °C): −107.64. <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> = 560.8.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>tert</i>-Butyl (2<i>R</i>,4a<i>R</i>)-10-Bromo-11-chloro-9-fluoro-2-methyl-5-oxo-1,2,4,4a,5,6-hexahydro-3<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinoline-3-carboxylate (<b>21</b>)</h4><div class="NLM_p last">To a stirred solution of 1-<i>tert</i>-butyl 3-methyl (3<i>R</i>,6<i>R</i>)-4-(7-bromo-6-chloro-8-fluoro-3-nitroquinolin-4-yl)-6-methylpiperazine-1,3-dicarboxylate (<b>20</b>) (430 mg, 0.77 mmol) in acetic acid (7.5 mL) at room temperature was added iron powder (150 mg, 2.68 mmol). The resultant suspension was stirred at 80 °C. After 1 h, the reaction mixture was concentrated under reduced pressure to give a brown solid. The solid was suspended in DCM (30 mL), and aqueous saturated sodium hydrogen carbonate solution was added until the solution was at pH 8. The mixture was filtered through Celite (washing with DCM). The phases were separated and the aqueous was washed with DCM (30 mL). The combined organic extracts were washed with brine (30 mL), passed through a hydrophobic frit, and concentrated to give <i>tert</i>-butyl (2<i>R</i>,4a<i>R</i>)-10-bromo-11-chloro-9-fluoro-2-methyl-5-oxo-1,2,4,4a,5,6-hexahydro-3<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinoline-3-carboxylate (<b>21</b>) (371 mg, 97%) as a pale-brown solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 30 °C): 1.44 (12H, s), 2.88 (1H, dd), 2.95–3.11 (1H, m), 3.35–3.48 (1H, m), 3.77–3.92 (1H, m), 4.09–4.3 (1H, m), 4.55–4.69 (1H, m), 7.94 (1H, d), 8.64 (1H, s), 11.06 (1H, s). <sup>19</sup>F NMR (376 MHz, DMSO, 30 °C): −108.63. <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> = 500.9.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>tert</i>-Butyl (2<i>R</i>,4a<i>R</i>)-10-Bromo-11-chloro-9-fluoro-2,6-dimethyl-5-oxo-1,2,4,4a,5,6-hexahydro-3<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinoline-3-carboxylate (<b>22</b>)</h4><div class="NLM_p last">A stirred suspension of <i>tert</i>-butyl (2<i>R</i>,4a<i>R</i>)-10-bromo-11-chloro-9-fluoro-2-methyl-5-oxo-1,2,4,4a,5,6-hexahydro-3<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinoline-3-carboxylate (<b>21</b>) (371 mg, 0.74 mmol) and potassium carbonate (205 mg, 1.48 mmol) in acetone (6.96 mL) was stirred at room temperature. Iodomethane (0.462 mL, 7.42 mmol) was added, and the resultant mixture was stirred at 40 °C overnight. The reaction mixture was cooled to room temperature and evaporated to dryness. The crude residue was dissolved in DCM (20 mL), extracted with water (10 mL), and washed with brine (10 mL). The organic extract was passed through a hydrophobic frit and concentrated to give a red foam. The crude product was purified by flash silica chromatography, elution gradient 0–80% ethyl acetate in DCM. Pure fractions were evaporated to dryness to afford <i>tert</i>-butyl (2<i>R</i>,4a<i>R</i>)-10-bromo-11-chloro-9-fluoro-2,6-dimethyl-5-oxo-1,2,4,4a,5,6-hexahydro-3<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinoline-3-carboxylate (<b>22</b>) (248 mg, 65%) as a pale-yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 30 °C): 1.43–1.67 (12H, m), 2.97 (2H, s), 3.51–3.59 (5H, m), 4.51 (1H, s), 4.94 (1H, d), 8.00 (1H, d), 8.79 (1H, s). <sup>19</sup>F NMR (376 MHz, CDCl3, 30 °C): −107.72. <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> = 512.9.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>tert</i>-Butyl (2<i>R</i>,4a<i>R</i>)-11-Chloro-9-fluoro-10-(2-fluoro-6-hydroxyphenyl)-2,6-dimethyl-5-oxo-1,2,4,4a,5,6-hexahydro-3<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinoline-3-carboxylate (<b>23</b>)</h4><div class="NLM_p last">A stirred suspension of <i>tert</i>-butyl (2<i>R</i>,4a<i>R</i>)-10-bromo-11-chloro-9-fluoro-2,6-dimethyl-5-oxo-1,2,4,4a,5,6-hexahydro-3<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinoline-3-carboxylate (<b>22</b>) (248 mg, 0.48 mmol), (2-fluoro-6-hydroxyphenyl)boronic acid (151 mg, 0.97 mmol), and potassium carbonate (200 mg, 1.45 mmol) in dioxane/water (4 mL, 3:1 ratio) was degassed with nitrogen for 10 min. RuPhos Pd G3 (40.4 mg, 0.05 mmol) and RuPhos (22.52 mg, 0.05 mmol) were added, and the mixture was heated to 80 °C. After 1.5 h, the reaction mixture was cooled to room temperature and diluted with ethyl acetate (30 mL). The solution was washed with water (30 mL), and the aqueous portion was extracted with ethyl acetate (30 mL). The combined organic extracts were washed with brine (30 mL), dried over magnesium sulfate, filtered, and concentrated in vacuo to give a yellow residue. The crude product was purified by flash silica chromatography, elution gradient 0–100% ethyl acetate in heptane. Pure fractions were evaporated to dryness to afford <i>tert</i>-butyl (2<i>R</i>,4a<i>R</i>)-11-chloro-9-fluoro-10-(2-fluoro-6-hydroxyphenyl)-2,6-dimethyl-5-oxo-1,2,4,4a,5,6-hexahydro-3<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinoline-3-carboxylate (<b>23</b>) (223 mg, 85%) as a yellow dry film (as a 1:1.2 mixture of atropisomers). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 30 °C): 1.45–1.67 (12H, m), 2.86–3.08 (2H, m), 3.43–3.61 (5H, m), 4.25–4.58 (1H, m), 4.84–5.03 (1H, m), 5.74–6.28 (1H, m), 6.83 (2H, dt), 7.26–7.42 (1H, m), 7.82–8.13 (1H, m), 8.64–8.92 (1H, m). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>, 30 °C): −116.77, −116.29, −112.15, −112.03. <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> = 545.0.</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (2<i>R</i>,4a<i>R</i>)-11-Chloro-9-fluoro-10-(2-fluoro-6-hydroxyphenyl)-2,6-dimethyl-2,3,4,4a-tetrahydro-1<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinolin-5(6<i>H</i>)-one (Atropisomer 1 and Atropisomer <b>2</b>, <b>24</b>)</h4><div class="NLM_p last"><i>tert</i>-Butyl (2<i>R</i>,4a<i>R</i>)-11-chloro-9-fluoro-10-(2-fluoro-6-hydroxyphenyl)-2,6-dimethyl-5-oxo-1,2,4,4a,5,6-hexahydro-3<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinoline-3-carboxylate (<b>23</b>) (223 mg, 0.41 mmol) was dissolved in MeOH (2 mL), and hydrochloric acid (6N in 2-propanol) (2.73 mL, 16.37 mmol) was added. The resultant solution was stirred at room temperature. After 35 min, the reaction mixture was purified using ion exchange chromatography, using a SCX-2 cartridge, eluting first with MeOH and then 1 M ammonia in MeOH. Pure fractions were collected and concentrated to give (2<i>R</i>,4a<i>R</i>)-11-chloro-9-fluoro-10-(2-fluoro-6-hydroxyphenyl)-2,6-dimethyl-2,3,4,4a-tetrahydro-1<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinolin-5(6<i>H</i>)-one (162 mg) as a yellow foam (as a mixture of atropisomers). The atropisomers were separated using supercritical fluid chromatography (SFC) (Column: Phenomenex C4, 30 mm × 250 mm, 5 μm; mobile phase A, 45% MeOH (+0.1% NH<sub>3</sub>)/mobile phase B, scCO<sub>2</sub>; flow rate, 100 mL/min; BPR, 120 bar; column temperature, 40 °C). Fractions containing the desired products were evaporated to dryness to afford atropisomer 1 of (2<i>R</i>,4a<i>R</i>)-11-chloro-9-fluoro-10-(2-fluoro-6-hydroxyphenyl)-2,6-dimethyl-2,3,4,4a-tetrahydro-1<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinolin-5(6<i>H</i>)-one (56 mg, 34%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 30 °C): 1.60 (3H, d), 3.06 (2H, d), 3.24 (1H, dd), 3.47–3.6 (5H, m), 3.63–3.74 (1H, m), 6.83 (2H, t), 7.35 (1H, td), 8.03 (1H, d), 8.76 (1H, s). Two exchangeable protons not seen. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>, 30 °C): −116.58, −112.03 to −111.57. <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> = 445.0. This was followed by atropisomer 2 of (2<i>R</i>,4a<i>R</i>)-11-chloro-9-fluoro-10-(2-fluoro-6-hydroxyphenyl)-2,6-dimethyl-2,3,4,4a-tetrahydro-1<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinolin-5(6<i>H</i>)-one <b>24</b> (70 mg, 43%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 30 °C): 1.55 (3H, d), 3.06 (2H, d), 3.21 (1H, dd), 3.54 (5H, d), 3.67 (1H, dd), 6.79 (1H, t), 6.85 (1H, d), 7.27–7.38 (1H, m), 7.88 (1H, d), 8.67 (1H, s). Two exchangeable protons not seen. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>, 30 °C): −116.07, −113.08 to −112.15. <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> = 445.0.</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (2<i>R</i>,4a<i>R</i>)-11-Chloro-9-fluoro-10-(2-fluoro-6-hydroxyphenyl)-2,6-dimethyl-3-(prop-2-enoyl)-2,3,4,4a-tetrahydro-1<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinolin-5(6<i>H</i>)-one (<b>25</b>)</h4><div class="NLM_p last">A solution of (2<i>R</i>,4a<i>R</i>)-11-chloro-9-fluoro-10-(2-fluoro-6-hydroxyphenyl)-2,6-dimethyl-2,3,4,4a-tetrahydro-1<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinolin-5(6<i>H</i>)-one (<b>24</b>) (70 mg, 0.16 mmol) and triethylamine (29 μL, 0.20 mmol) in DCM (1.53 mL) was cooled in an ice-bath to 0 °C. Acryloyl chloride (14 μL, 0.17 mmol) was added, and the resultant mixture was stirred at 0 °C. After 10 min, a further portion of acryloyl chloride (14.06 μL, 0.17 mmol) was added and the mixture was stirred at 0 °C. After a further 10 min, the mixture was quenched by addition of water (5 mL) and extracted with DCM (2 × 10 mL). The combined organics were passed through a hydrophobic frit and concentrated under reduced pressure to give a yellow dry film. The film was dissolved in 1 M ammonia in MeOH (10 mL), and the suspension was stirred at room temperature. After 15 min, the reaction mixture was concentrated under reduced pressure to give a yellow dry film. The crude product was purified by preparative HPLC (Column: Waters CSH C18 OBD, 30 mm × 100 mm id, 5 μm particle size), using decreasingly polar mixtures of water (containing 1% by volume of NH<sub>4</sub>OH (28–30% in H<sub>2</sub>O)) and acetonitrile as eluents (using a shallow gradient of 20–35% acetonitrile). Fractions containing the desired compound were evaporated to dryness to afford (2<i>R</i>,4a<i>R</i>)-11-chloro-9-fluoro-10-(2-fluoro-6-hydroxyphenyl)-2,6-dimethyl-3-(prop-2-enoyl)-2,3,4,4a-tetrahydro-1<i>H</i>-pyrazino[1′,2′:4,5]pyrazino[2,3-<i>c</i>]quinolin-5(6<i>H</i>)-one (<b>25</b>) (29.1 mg, 37%) as a colorless solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, 30 °C): 1.57 (1H, s), 1.66 (3H, d), 3.01 (1H, d), 3.15 (1H, d), 3.56 (3H, s), 3.65 (1H, s), 3.81 (1H, d), 4.81 (1H, d), 5.00 (1H, s), 5.80 (1H, d), 6.36 (1H, d), 6.75–6.9 (2H, m), 6.93–7.1 (1H, m), 7.35 (1H, td), 8.02 (1H, s), 8.80 (1H, s). <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>, 30 °C): −112.04, −116.25. <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>, 27 °C) 14.8, 29.2, 37.8, 44.0, 52.0, 56.2, 107.3 (d, <i>J</i> = 22.0 Hz), 108.1 (d, <i>J</i> = 19.3 Hz), 111.9, 117.1, 117.8 (d, <i>J</i> = 17.7 Hz), 124.5, 128.1, 128.5, 130.1, 131.1 (d, <i>J</i> = 10.6 Hz), 134.2, 135.2 (d, <i>J</i> = 4.0 Hz), 137.7, 139.7, 155.4 (d, <i>J</i> = 5.9 Hz), 156.2 (d, <i>J</i> = 261.9 Hz), 160.7 (d, <i>J</i> = 246.6 Hz), 165.2, 166.7. <i>m</i>/<i>z</i> (ES+), [M + H]<sup>+</sup> = 499. HRMS (ESI) for C<sub>25</sub>H<sub>21</sub>ClF<sub>2</sub>N<sub>4</sub>O<sub>3</sub> (MH<sup>+</sup>) calculated 499.1348, found 499.1332.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01720" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63400" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63400" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01720?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01720</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Protocols for the in vitro and cell assays and in vivo experiments, synthetic methods for the remaining examples, protein synthesis and purification, X-ray crystallographic structure information (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01720/suppl_file/jm9b01720_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings with associated data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01720/suppl_file/jm9b01720_si_001.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01720/suppl_file/jm9b01720_si_001.csv">jm9b01720_si_001.csv (1.46 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01720/suppl_file/jm9b01720_si_002.pdf">jm9b01720_si_002.pdf (505.15 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB depositions are available for compounds <b>25</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5B">6T5B</a>), <b>26</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5V">6T5V</a>), and <b>29</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5U">6T5U</a>). The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01720" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00770" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00770" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jason G. Kettle</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7373-0758" title="Orcid link">http://orcid.org/0000-0001-7373-0758</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c8a2a9bba7a6e6a3adbcbca4ad88a9bbbcbaa9b2ada6adaba9e6aba7a5"><span class="__cf_email__" data-cfemail="85efe4f6eaebabeee0f1f1e9e0c5e4f6f1f7e4ffe0ebe0e6e4abe6eae8">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sharan K. Bagal</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sue Bickerton</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael S. Bodnarchuk</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9172-1203" title="Orcid link">http://orcid.org/0000-0002-9172-1203</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jason Breed</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rodrigo J. Carbajo</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Doyle J. Cassar</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Atanu Chakraborty</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sabina Cosulich</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Iain Cumming</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Davies</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew Eatherton</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura Evans</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lyman Feron</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaun Fillery</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0314-8137" title="Orcid link">http://orcid.org/0000-0002-0314-8137</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emma S. Gleave</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Frederick W. Goldberg</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4874-1700" title="Orcid link">http://orcid.org/0000-0002-4874-1700</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephanie Harlfinger</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lyndsey Hanson</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Alderley Park SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Howard</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rachel Howells</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anne Jackson</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Kemmitt</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer K. Kingston</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott Lamont</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hilary J. Lewis</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Songlei Li</span> - <span class="hlFld-Affiliation affiliation">Pharmaron
Beijing Co., Ltd. 6 Taihe
Road BDA, Beijing 100176 P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Libin Liu</span> - <span class="hlFld-Affiliation affiliation">Pharmaron
Beijing Co., Ltd. 6 Taihe
Road BDA, Beijing 100176 P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Derek Ogg</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher Phillips</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Radek Polanski</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Graeme Robb</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4531-4375" title="Orcid link">http://orcid.org/0000-0002-4531-4375</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Robinson</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sarah Ross</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James M. Smith</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Tonge</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rebecca Whiteley</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Junsheng Yang</span> - <span class="hlFld-Affiliation affiliation">Pharmaron
Beijing Co., Ltd. 6 Taihe
Road BDA, Beijing 100176 P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Longfei Zhang</span> - <span class="hlFld-Affiliation affiliation">Pharmaron
Beijing Co., Ltd. 6 Taihe
Road BDA, Beijing 100176 P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiliang Zhao</span> - <span class="hlFld-Affiliation affiliation">Pharmaron
Beijing Co., Ltd. 6 Taihe
Road BDA, Beijing 100176 P. R. China</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3581-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53572" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53572" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Paul Davey and David Longmire for assistance in running mass spectrum characterization and Matt Sanders for assistance on the preparation and analysis of chromatographic data. We thank the AstraZeneca Oncology R&D ADME team for assistance with metabolite identification and DMPK in vitro characterization.</p></div><table summary="" class="NLM_def-list" id="dl2"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">FBLG</td><td class="NLM_def"><p class="first last">fragment-based lead generation.</p></td></tr><tr><td class="NLM_term">GDP</td><td class="NLM_def"><p class="first last">guanosine diphosphate.</p></td></tr><tr><td class="NLM_term">GTP</td><td class="NLM_def"><p class="first last">guanosine triphosphate.</p></td></tr><tr><td class="NLM_term">GST</td><td class="NLM_def"><p class="first last">glutathione S-transferase.</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank.</p></td></tr><tr><td class="NLM_term">SE</td><td class="NLM_def"><p class="first last">standard error.</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of means.</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">mass spectrometry.</p></td></tr><tr><td class="NLM_term">SILAC</td><td class="NLM_def"><p class="first last">stable isotope labeling by amino acids in cell culture.</p></td></tr><tr><td class="NLM_term">RIPA</td><td class="NLM_def"><p class="first last">radioimmunoprecipitation assay</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86282" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86282" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 36 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Vega, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armenia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatila, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimitriadoy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantheti, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saghafinia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarty, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foltz, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shmulevich, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heins, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochoa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kandoth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahceci, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervishi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dogrusoz, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laird, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Way, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iavarone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bivona, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazar, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammer, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwong, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McArthur, G</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tward, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederick, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Allen, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherniack, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciriello, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sander, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, N.</span></span>; <span class="NLM_contrib-group">Cancer Genome Atlas Research Network</span> <span> </span><span class="NLM_article-title">Oncogenic Signaling Pathways in The Cancer Genome Atlas</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">321</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.03.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1016%2Fj.cell.2018.03.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=29625050" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntFahsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=321-337&author=F.+Sanchez-Vegaauthor=M.+Minaauthor=J.+Armeniaauthor=W.+K.+Chatilaauthor=A.+Lunaauthor=K.+C.+Laauthor=S.+Dimitriadoyauthor=D.+L.+Liuauthor=H.+S.+Kanthetiauthor=S.+Saghafiniaauthor=D.+Chakravartyauthor=F.+Daianauthor=Q.+Gaoauthor=M.+H.+Baileyauthor=W.+W.+Liangauthor=S.+M.+Foltzauthor=I.+Shmulevichauthor=L.+Dingauthor=Z.+Heinsauthor=A.+Ochoaauthor=B.+Grossauthor=J.+Gaoauthor=H.+Zhangauthor=R.+Kundraauthor=C.+Kandothauthor=I.+Bahceciauthor=L.+Dervishiauthor=U.+Dogrusozauthor=W.+Zhouauthor=H.+Shenauthor=P.+W.+Lairdauthor=G.+P.+Wayauthor=C.+S.+Greeneauthor=H.+Liangauthor=Y.+Xiaoauthor=C.+Wangauthor=A.+Iavaroneauthor=A.+H.+Bergerauthor=T.+G.+Bivonaauthor=A.+J.+Lazarauthor=G.+D.+Hammerauthor=T.+Giordanoauthor=L.+N.+Kwongauthor=G+McArthurauthor=C.+Huangauthor=A.+D.+Twardauthor=M.+J.+Frederickauthor=F.+McCormickauthor=M.+Meyersonauthor=E.+M.+Van+Allenauthor=A.+D.+Cherniackauthor=G.+Cirielloauthor=C.+Sanderauthor=N.+Schultzauthor=Cancer+Genome+Atlas+Research+Network&title=Oncogenic+Signaling+Pathways+in+The+Cancer+Genome+Atlas&doi=10.1016%2Fj.cell.2018.03.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic signaling pathways in the cancer genome atlas</span></div><div class="casAuthors">Sanchez-Vega, Francisco; Mina, Marco; Armenia, Joshua; Chatila, Walid K.; Luna, Augustin; La, Konnor C.; Dimitriadoy, Sofia; Liu, David L.; Kantheti, Havish S.; Saghafinia, Sadegh; Chakravarty, Debyani; Daian, Foysal; Gao, Qingsong; Bailey, Matthew H.; Liang, Wen-Wei; Foltz, Steven M.; Shmulevich, Ilya; Ding, Li; Heins, Zachary; Ochoa, Angelica; Gross, Benjamin; Gao, Jianjiong; Zhang, Hongxin; Kundra, Ritika; Kandoth, Cyriac; Bahceci, Istemi; Dervishi, Leonard; Dogrusoz, Ugur; Zhou, Wanding; Shen, Hui; Laird, Peter W.; Way, Gregory P.; Greene, Casey S.; Liang, Han; Xiao, Yonghong; Wang, Chen; Iavarone, Antonio; Berger, Alice H.; Bivona, Trever G.; Lazar, Alexander J.; Hammer, Gary D.; Giordano, Thomas; Kwong, Lawrence N.; McArthur, Grant; Huang, Chenfei; Tward, Aaron D.; Frederick, Mitchell J.; McCormick, Frank; Meyerson, Matthew; Van Allen, Eliezer M.; Cherniack, Andrew D.; Ciriello, Giovanni; Sander, Chris; Schultz, Nikolaus</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">321-337.e10</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Genetic alterations in signaling pathways that control cell-cycle progression, apoptosis, and cell growth are common hallmarks of cancer, but the extent, mechanisms, and co-occurrence of alterations in these pathways differ between individual tumors and tumor types.  Using mutations, copy-no. changes, mRNA expression, gene fusions and DNA methylation in 9125 tumors profiled by The Cancer Genome Atlas (TCGA), we analyzed the mechanisms and patterns of somatic alterations in ten canonical pathways: cell cycle, Hippo, Myc, Notch, Nrf2, PI-3-Kinase/Akt, RTK-RAS, TGFβ signaling, p53 and β-catenin/Wnt.  We charted the detailed landscape of pathway alterations in 33 cancer types, stratified into 64 subtypes, and identified patterns of co-occurrence and mutual exclusivity.  Eighty-nine percent of tumors had at least one driver alteration in these pathways, and 57% percent of tumors had at least one alteration potentially targetable by currently available drugs.  Thirty percent of tumors had multiple targetable alterations, indicating opportunities for combination therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv_ggbxuypEbVg90H21EOLACvtfcHk0lh8YDJX569I8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntFahsrs%253D&md5=3677d3a77b12767c828f48eb80d71141</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.03.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.03.035%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez-Vega%26aufirst%3DF.%26aulast%3DMina%26aufirst%3DM.%26aulast%3DArmenia%26aufirst%3DJ.%26aulast%3DChatila%26aufirst%3DW.%2BK.%26aulast%3DLuna%26aufirst%3DA.%26aulast%3DLa%26aufirst%3DK.%2BC.%26aulast%3DDimitriadoy%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DD.%2BL.%26aulast%3DKantheti%26aufirst%3DH.%2BS.%26aulast%3DSaghafinia%26aufirst%3DS.%26aulast%3DChakravarty%26aufirst%3DD.%26aulast%3DDaian%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DQ.%26aulast%3DBailey%26aufirst%3DM.%2BH.%26aulast%3DLiang%26aufirst%3DW.%2BW.%26aulast%3DFoltz%26aufirst%3DS.%2BM.%26aulast%3DShmulevich%26aufirst%3DI.%26aulast%3DDing%26aufirst%3DL.%26aulast%3DHeins%26aufirst%3DZ.%26aulast%3DOchoa%26aufirst%3DA.%26aulast%3DGross%26aufirst%3DB.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DKundra%26aufirst%3DR.%26aulast%3DKandoth%26aufirst%3DC.%26aulast%3DBahceci%26aufirst%3DI.%26aulast%3DDervishi%26aufirst%3DL.%26aulast%3DDogrusoz%26aufirst%3DU.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DLaird%26aufirst%3DP.%2BW.%26aulast%3DWay%26aufirst%3DG.%2BP.%26aulast%3DGreene%26aufirst%3DC.%2BS.%26aulast%3DLiang%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DIavarone%26aufirst%3DA.%26aulast%3DBerger%26aufirst%3DA.%2BH.%26aulast%3DBivona%26aufirst%3DT.%2BG.%26aulast%3DLazar%26aufirst%3DA.%2BJ.%26aulast%3DHammer%26aufirst%3DG.%2BD.%26aulast%3DGiordano%26aufirst%3DT.%26aulast%3DKwong%26aufirst%3DL.%2BN.%26aulast%3DMcArthur%26aufirst%3DG%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DTward%26aufirst%3DA.%2BD.%26aulast%3DFrederick%26aufirst%3DM.%2BJ.%26aulast%3DMcCormick%26aufirst%3DF.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DVan%2BAllen%26aufirst%3DE.%2BM.%26aulast%3DCherniack%26aufirst%3DA.%2BD.%26aulast%3DCiriello%26aufirst%3DG.%26aulast%3DSander%26aufirst%3DC.%26aulast%3DSchultz%26aufirst%3DN.%26aulast%3D%26atitle%3DOncogenic%2520Signaling%2520Pathways%2520in%2520The%2520Cancer%2520Genome%2520Atlas%26jtitle%3DCell%26date%3D2018%26volume%3D173%26spage%3D321%26epage%3D337%26doi%3D10.1016%2Fj.cell.2018.03.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note"><p class="first last">e10.</p></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prior, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattos, C.</span></span> <span> </span><span class="NLM_article-title">A Comprehensive Survey of Ras Mutations in Cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">2457</span>– <span class="NLM_lpage">2467</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-2612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1158%2F0008-5472.CAN-11-2612" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=22589270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvF2ntL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=2457-2467&author=I.+A.+Priorauthor=P.+D.+Lewisauthor=C.+Mattos&title=A+Comprehensive+Survey+of+Ras+Mutations+in+Cancer&doi=10.1158%2F0008-5472.CAN-11-2612"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Survey of Ras Mutations in Cancer</span></div><div class="casAuthors">Prior, Ian A.; Lewis, Paul D.; Mattos, Carla</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2457-2467</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  All mammalian cells express 3 closely related Ras proteins, termed H-Ras, K-Ras, and N-Ras, that promote oncogenesis when they are mutationally activated at codon 12, 13, or 61.  Although there is a high degree of similarity among the isoforms, K-Ras mutations are far more frequently obsd. in cancer, and each isoform displays preferential coupling to particular cancer types.  We examd. the mutational spectra of Ras isoforms curated from large-scale tumor profiling and found that each isoform exhibits surprisingly distinctive codon mutation and amino-acid substitution biases.  These findings were unexpected given that these mutations occur in regions that share 100% amino-acid sequence identity among the 3 isoforms.  Of importance, many of these mutational biases were not due to differences in exposure to mutagens, because the patterns were still evident when compared within specific cancer types.  We discuss potential genetic and epigenetic mechanisms, as well as isoform-specific differences in protein structure and signaling, that may promote these distinct mutation patterns and differential coupling to specific cancers.  Cancer Res; 72(10); 2457-67.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg2lqiicosOLVg90H21EOLACvtfcHk0lidDDPcY5wNGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvF2ntL4%253D&md5=6d1894847015468f77c8b08b57c6d62a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-2612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-2612%26sid%3Dliteratum%253Aachs%26aulast%3DPrior%26aufirst%3DI.%2BA.%26aulast%3DLewis%26aufirst%3DP.%2BD.%26aulast%3DMattos%26aufirst%3DC.%26atitle%3DA%2520Comprehensive%2520Survey%2520of%2520Ras%2520Mutations%2520in%2520Cancer%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D2457%26epage%3D2467%26doi%3D10.1158%2F0008-5472.CAN-11-2612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haigis, K. M.</span></span> <span> </span><span class="NLM_article-title"><i>KRAS</i> Alleles: The Devil is in the Detail</span>. <i>Trends Cancer.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">686</span>– <span class="NLM_lpage">697</span>, <span class="refDoi"> DOI: 10.1016/j.trecan.2017.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1016%2Fj.trecan.2017.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=28958387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1MXksVyhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=686-697&author=K.+M.+Haigis&title=KRAS+Alleles%3A+The+Devil+is+in+the+Detail&doi=10.1016%2Fj.trecan.2017.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">KRAS Alleles: The Devil Is in the Detail</span></div><div class="casAuthors">Haigis, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">686-697</span>CODEN:
                <span class="NLM_cas:coden">TCRAB5</span>;
        ISSN:<span class="NLM_cas:issn">2405-8033</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  KRAS is the most frequently mutated oncogene in cancer and KRAS mutation is commonly assocd. with poor prognosis and resistance to therapy.  Since the KRAS oncoprotein is, as yet, not directly druggable, efforts to target KRAS mutant cancers focus on identifying vulnerabilities in downstream signaling pathways or in stress response pathways that are permissive for strong oncogenic signaling.  One aspect of KRAS biol. that is not well appreciated is the potential biol. differences between the many distinct KRAS activating mutations.  This review draws upon insights from both clin. and exptl. studies to explore similarities and differences among KRAS alleles.  Historical and emerging evidence supports the notion that the specific biol. related to each allele might be exploitable for allele-specific therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjsS2Ho266RLVg90H21EOLACvtfcHk0lidDDPcY5wNGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXksVyhsg%253D%253D&md5=672e8e288211cef69fc149d658366e53</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.trecan.2017.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trecan.2017.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DHaigis%26aufirst%3DK.%2BM.%26atitle%3DKRAS%2520Alleles%253A%2520The%2520Devil%2520is%2520in%2520the%2520Detail%26jtitle%3DTrends%2520Cancer.%26date%3D2017%26volume%3D3%26spage%3D686%26epage%3D697%26doi%3D10.1016%2Fj.trecan.2017.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lito, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span> <span> </span><span class="NLM_article-title">Allele-specific Inhibitors Inactivate Mutant KRAS G12C by a Trapping Mechanism</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span>,  <span class="NLM_fpage">604</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1126/science.aad6204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1126%2Fscience.aad6204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=26841430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2jt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=604-608&author=P.+Litoauthor=M.+Solomonauthor=L.+S.+Liauthor=R.+Hansenauthor=N.+Rosen&title=Allele-specific+Inhibitors+Inactivate+Mutant+KRAS+G12C+by+a+Trapping+Mechanism&doi=10.1126%2Fscience.aad6204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Allele-specific inhibitors inactivate mutant KRAS G12C by a trapping mechanism</span></div><div class="casAuthors">Lito, Piro; Solomon, Martha; Li, Lian-Sheng; Hansen, Rasmus; Rosen, Neal</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6273</span>),
    <span class="NLM_cas:pages">604-608</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">It is thought that KRAS oncoproteins are constitutively active because their guanosine triphosphatase (GTPase) activity is disabled.  Consequently, drugs targeting the inactive or GDP-bound conformation are not expected to be effective.  We describe a mechanism that enables such drugs to inhibit KRASG12C signaling and cancer cell growth.  Inhibition requires intact GTPase activity and occurs because drug-bound KRASG12C is insusceptible to nucleotide exchange factors and thus trapped in its inactive state.  Indeed, mutants completely lacking GTPase activity and those promoting exchange reduced the potency of the drug.  Suppressing nucleotide exchange activity downstream of various tyrosine kinases enhanced KRASG12C inhibition, whereas its potentiation had the opposite effect.  These findings reveal that KRASG12C undergoes nucleotide cycling in cancer cells and provide a basis for developing effective therapies to treat KRASG12C-driven cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqK3AbqEZO6k7Vg90H21EOLACvtfcHk0lgkgKqxH1NERQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2jt78%253D&md5=22a47908ac0146822ecf19f5d0c952de</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad6204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad6204%26sid%3Dliteratum%253Aachs%26aulast%3DLito%26aufirst%3DP.%26aulast%3DSolomon%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DL.%2BS.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DAllele-specific%2520Inhibitors%2520Inactivate%2520Mutant%2520KRAS%2520G12C%2520by%2520a%2520Trapping%2520Mechanism%26jtitle%3DScience%26date%3D2016%26volume%3D351%26spage%3D604%26epage%3D608%26doi%3D10.1126%2Fscience.aad6204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fakih, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falchook, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govindan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, P. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henary, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, D. S.</span></span> <span> </span><span class="NLM_article-title">Phase 1 Study Evaluating the Safety, Tolerability, Pharmacokinetics (PK), and Efficacy of AMG 510, a Novel Small Molecule KRASG12C Inhibitor, in Advanced Solid Tumors</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">3003</span>– <span class="NLM_lpage">3003</span>, <span class="refDoi"> DOI: 10.1200/JCO.2019.37.15_suppl.3003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1200%2FJCO.2019.37.15_suppl.3003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=3003-3003&author=M.+Fakihauthor=B.+O%E2%80%99Neilauthor=T.+J.+Priceauthor=G.+S.+Falchookauthor=J.+Desaiauthor=J.+Kuoauthor=R.+Govindanauthor=E.+Rasmussenauthor=P.+K.+H.+Morrowauthor=J.+Ngangauthor=H.+A.+Henaryauthor=D.+S.+Hong&title=Phase+1+Study+Evaluating+the+Safety%2C+Tolerability%2C+Pharmacokinetics+%28PK%29%2C+and+Efficacy+of+AMG+510%2C+a+Novel+Small+Molecule+KRASG12C+Inhibitor%2C+in+Advanced+Solid+Tumors&doi=10.1200%2FJCO.2019.37.15_suppl.3003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1200%2FJCO.2019.37.15_suppl.3003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2019.37.15_suppl.3003%26sid%3Dliteratum%253Aachs%26aulast%3DFakih%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DB.%26aulast%3DPrice%26aufirst%3DT.%2BJ.%26aulast%3DFalchook%26aufirst%3DG.%2BS.%26aulast%3DDesai%26aufirst%3DJ.%26aulast%3DKuo%26aufirst%3DJ.%26aulast%3DGovindan%26aufirst%3DR.%26aulast%3DRasmussen%26aufirst%3DE.%26aulast%3DMorrow%26aufirst%3DP.%2BK.%2BH.%26aulast%3DNgang%26aufirst%3DJ.%26aulast%3DHenary%26aufirst%3DH.%2BA.%26aulast%3DHong%26aufirst%3DD.%2BS.%26atitle%3DPhase%25201%2520Study%2520Evaluating%2520the%2520Safety%252C%2520Tolerability%252C%2520Pharmacokinetics%2520%2528PK%2529%252C%2520and%2520Efficacy%2520of%2520AMG%2520510%252C%2520a%2520Novel%2520Small%2520Molecule%2520KRASG12C%2520Inhibitor%252C%2520in%2520Advanced%2520Solid%2520Tumors%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D37%26spage%3D3003%26epage%3D3003%26doi%3D10.1200%2FJCO.2019.37.15_suppl.3003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quevedo, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruz-Migoni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bery, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bataille, C. J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, L. Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallon, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tulmin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehebauer, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Fuentes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, S. E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabbitts, T. H.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors of RAS-Effector Protein Interactions Derived Using an Intracellular Antibody Fragment</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3169</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-05707-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1038%2Fs41467-018-05707-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=30093669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A280%3ADC%252BB3c7ms12mtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=3169&author=C.+E.+Quevedoauthor=A.+Cruz-Migoniauthor=N.+Beryauthor=A.+Millerauthor=T.+Tanakaauthor=D.+Petchauthor=C.+J.+R.+Batailleauthor=L.+Y.+W.+Leeauthor=P.+S.+Fallonauthor=H.+Tulminauthor=M.+T.+Ehebauerauthor=N.+Fernandez-Fuentesauthor=A.+J.+Russellauthor=S.+B.+Carrauthor=S.+E.+V.+Phillipsauthor=T.+H.+Rabbitts&title=Small+Molecule+Inhibitors+of+RAS-Effector+Protein+Interactions+Derived+Using+an+Intracellular+Antibody+Fragment&doi=10.1038%2Fs41467-018-05707-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of RAS-effector protein interactions derived using an intracellular antibody fragment</span></div><div class="casAuthors">Quevedo Camilo E; Cruz-Migoni Abimael; Bery Nicolas; Miller Ami; Tulmin Hanna; Ehebauer Matthias T; Rabbitts Terence H; Cruz-Migoni Abimael; Fernandez-Fuentes Narcis; Carr Stephen B; Phillips Simon E V; Tanaka Tomoyuki; Petch Donna; Tanaka Tomoyuki; Bataille Carole J R; Russell Angela J; Lee Lydia Y W; Fallon Phillip S; Tulmin Hanna; Ehebauer Matthias T; Fernandez-Fuentes Narcis; Carr Stephen B; Phillips Simon E V</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3169</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Targeting specific protein-protein interactions (PPIs) is an attractive concept for drug development, but hard to implement since intracellular antibodies do not penetrate cells and most small-molecule drugs are considered unsuitable for PPI inhibition.  A potential solution to these problems is to select intracellular antibody fragments to block PPIs, use these antibody fragments for target validation in disease models and finally derive small molecules overlapping the antibody-binding site.  Here, we explore this strategy using an anti-mutant RAS antibody fragment as a competitor in a small-molecule library screen for identifying RAS-binding compounds.  The initial hits are optimized by structure-based design, resulting in potent RAS-binding compounds that interact with RAS inside the cells, prevent RAS-effector interactions and inhibit endogenous RAS-dependent signalling.  Our results may aid RAS-dependent cancer drug development and demonstrate a general concept for developing small compounds to replace intracellular antibody fragments, enabling rational drug development to target validated PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSIUnXfwZaCRvxQvRxYU81tfW6udTcc2eaBWbtSSwwslLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7ms12mtQ%253D%253D&md5=79679120c60be8cd9be9fbbfe0c9fdb6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-05707-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-05707-2%26sid%3Dliteratum%253Aachs%26aulast%3DQuevedo%26aufirst%3DC.%2BE.%26aulast%3DCruz-Migoni%26aufirst%3DA.%26aulast%3DBery%26aufirst%3DN.%26aulast%3DMiller%26aufirst%3DA.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DPetch%26aufirst%3DD.%26aulast%3DBataille%26aufirst%3DC.%2BJ.%2BR.%26aulast%3DLee%26aufirst%3DL.%2BY.%2BW.%26aulast%3DFallon%26aufirst%3DP.%2BS.%26aulast%3DTulmin%26aufirst%3DH.%26aulast%3DEhebauer%26aufirst%3DM.%2BT.%26aulast%3DFernandez-Fuentes%26aufirst%3DN.%26aulast%3DRussell%26aufirst%3DA.%2BJ.%26aulast%3DCarr%26aufirst%3DS.%2BB.%26aulast%3DPhillips%26aufirst%3DS.%2BE.%2BV.%26aulast%3DRabbitts%26aufirst%3DT.%2BH.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520of%2520RAS-Effector%2520Protein%2520Interactions%2520Derived%2520Using%2520an%2520Intracellular%2520Antibody%2520Fragment%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D3169%26doi%3D10.1038%2Fs41467-018-05707-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmachl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantoulidis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gollner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häring, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hela, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Oezguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knesl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornigg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kousek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamarre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munico-Martinez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peinsipp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salamon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherbantin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipany, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrenk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siszler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolkerstorfer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span> <span> </span><span class="NLM_article-title">Drugging an Undruggable Pocket on KRAS</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">15823</span>– <span class="NLM_lpage">15829</span>, <span class="refDoi"> DOI: 10.1073/pnas.1904529116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1073%2Fpnas.1904529116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=31332011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFChs7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=15823-15829&author=D.+Kesslerauthor=M.+Gmachlauthor=A.+Mantoulidisauthor=L.+T.+Martinauthor=A.+Zoephelauthor=M.+Mayerauthor=A.+Gollnerauthor=D.+Coviniauthor=S.+Fischerauthor=T.+Gerstbergerauthor=T.+Gmaschitzauthor=C.+Goodwinauthor=P.+Grebauthor=D.+H%C3%A4ringauthor=W.+Helaauthor=J.+Hoffmannauthor=J.+Karolyi-Oezguerauthor=P.+Kneslauthor=S.+Korniggauthor=M.+Koeglauthor=R.+Kousekauthor=L.+Lamarreauthor=F.+Moserauthor=S.+Munico-Martinezauthor=C.+Peinsippauthor=J.+Phanauthor=J.+Rinnenthalauthor=J.+Saiauthor=C.+Salamonauthor=Y.+Scherbantinauthor=K.+Schipanyauthor=R.+Schnitzerauthor=A.+Schrenkauthor=B.+Sharpsauthor=G.+Siszlerauthor=Q.+Sunauthor=A.+Watersonauthor=B.+Wolkerstorferauthor=M.+Zeebauthor=M.+Pearsonauthor=S.+W.+Fesikauthor=D.+B.+McConnell&title=Drugging+an+Undruggable+Pocket+on+KRAS&doi=10.1073%2Fpnas.1904529116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging an undruggable pocket on KRAS</span></div><div class="casAuthors">Kessler, Dirk; Gmachl, Michael; Mantoulidis, Andreas; Martin, Laetitia J.; Zoephel, Andreas; Mayer, Moriz; Gollner, Andreas; Covini, David; Fischer, Silke; Gerstberger, Thomas; Gmaschitz, Teresa; Goodwin, Craig; Greb, Peter; Haring, Daniela; Hela, Wolfgang; Hoffmann, Johann; Karolyi-Oezguer, Jale; Knesl, Petr; Kornigg, Stefan; Koegl, Manfred; Kousek, Roland; Lamarre, Lyne; Moser, Franziska; Munico-Martinez, Silvia; Peinsipp, Christoph; Phan, Jason; Rinnenthal, Jorg; Sai, Jiqing; Salamon, Christian; Scherbantin, Yvonne; Schipany, Katharina; Schnitzer, Renate; Schrenk, Andreas; Sharps, Bernadette; Siszler, Gabriella; Sun, Qi; Waterson, Alex; Wolkerstorfer, Bernhard; Zeeb, Markus; Pearson, Mark; Fesik, Stephen W.; McConnell, Darryl B.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">15823-15829</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">(Imidazolylmethylindolylmethylaminomethyl)indolyl hydroxyisoindolone I, an inhibitor of both the active and inactive forms of KRAS, was discovered using fragment-based screening.  I bound to KRAS in a pocket between switches I and II of KRAS, blocking all GEF, GAP, and effector interactions with KRAS and thus inhibiting downstream signaling; I inhibited KRAS mutant cells at low micromolar concns.  Uniformly 15N-labeled KRAS bound to the stable GTP analog GCP was prepd. with yields of 10 mg/L for use in crystallog. of KRAS-inhibitor complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaMeFg74p2-rVg90H21EOLACvtfcHk0lh_SdmV8ORvpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFChs7jJ&md5=ebf443050cb2ce6cbeec1e40084129d0</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1904529116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1904529116%26sid%3Dliteratum%253Aachs%26aulast%3DKessler%26aufirst%3DD.%26aulast%3DGmachl%26aufirst%3DM.%26aulast%3DMantoulidis%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DL.%2BT.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DGollner%26aufirst%3DA.%26aulast%3DCovini%26aufirst%3DD.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DGoodwin%26aufirst%3DC.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DH%25C3%25A4ring%26aufirst%3DD.%26aulast%3DHela%26aufirst%3DW.%26aulast%3DHoffmann%26aufirst%3DJ.%26aulast%3DKarolyi-Oezguer%26aufirst%3DJ.%26aulast%3DKnesl%26aufirst%3DP.%26aulast%3DKornigg%26aufirst%3DS.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DKousek%26aufirst%3DR.%26aulast%3DLamarre%26aufirst%3DL.%26aulast%3DMoser%26aufirst%3DF.%26aulast%3DMunico-Martinez%26aufirst%3DS.%26aulast%3DPeinsipp%26aufirst%3DC.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DSai%26aufirst%3DJ.%26aulast%3DSalamon%26aufirst%3DC.%26aulast%3DScherbantin%26aufirst%3DY.%26aulast%3DSchipany%26aufirst%3DK.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DSchrenk%26aufirst%3DA.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DSiszler%26aufirst%3DG.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DWaterson%26aufirst%3DA.%26aulast%3DWolkerstorfer%26aufirst%3DB.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26atitle%3DDrugging%2520an%2520Undruggable%2520Pocket%2520on%2520KRAS%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D15823%26epage%3D15829%26doi%3D10.1073%2Fpnas.1904529116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossanese, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of Small Molecules that Bind to K-Ras and Inhibit Sos-Mediated Activation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">6140</span>– <span class="NLM_lpage">6143</span>, <span class="refDoi"> DOI: 10.1002/anie.201201358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1002%2Fanie.201201358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC38Xms1entr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=6140-6143&author=Q.+Sunauthor=J.+P.+Burkeauthor=J.+Phanauthor=M.+C.+Burnsauthor=E.+T.+Olejniczakauthor=A.+G.+Watersonauthor=T.+Leeauthor=O.+W.+Rossaneseauthor=S.+W.+Fesik&title=Discovery+of+Small+Molecules+that+Bind+to+K-Ras+and+Inhibit+Sos-Mediated+Activation&doi=10.1002%2Fanie.201201358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecules that Bind to K-Ras and Inhibit Sos-Mediated Activation</span></div><div class="casAuthors">Sun, Qi; Burke, Jason P.; Phan, Jason; Burns, Michael C.; Olejniczak, Edward T.; Waterson, Alex G.; Lee, Taekyu; Rossanese, Olivia W.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">6140-6143, S6140/1-S6140/17</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">K-Ras inhibition represents an attractive therapeutic strategy for many cancers.  We report on the discovery of novel small mols. that bind directly to K-Ras between switch I and switch II and inhibit Sos-catalyzed K-Ras activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy5Cx7BSO1pLVg90H21EOLACvtfcHk0lh_SdmV8ORvpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xms1entr4%253D&md5=7d579523462aaf7eac1890643f91fab3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fanie.201201358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201201358%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DBurke%26aufirst%3DJ.%2BP.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DBurns%26aufirst%3DM.%2BC.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DRossanese%26aufirst%3DO.%2BW.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520Small%2520Molecules%2520that%2520Bind%2520to%2520K-Ras%2520and%2520Inhibit%2520Sos-Mediated%2520Activation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D6140%26epage%3D6143%26doi%3D10.1002%2Fanie.201201358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrenton, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skelton, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fauber, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokoe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludlam, M. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giannetti, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starovasnik, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span> <span> </span><span class="NLM_article-title">Small-molecule Ligands Bind to a Distinct Pocket in Ras and Inhibit SOS-mediated Nucleotide Exchange Activity</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">5299</span>– <span class="NLM_lpage">5304</span>, <span class="refDoi"> DOI: 10.1073/pnas.1116510109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1073%2Fpnas.1116510109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=22431598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslOjsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5299-5304&author=T.+Maurerauthor=L.+S.+Garrentonauthor=A.+Ohauthor=K.+Pittsauthor=D.+J.+Andersonauthor=N.+J.+Skeltonauthor=B.+P.+Fauberauthor=B.+Panauthor=S.+Malekauthor=D.+Stokoeauthor=M.+J.+C.+Ludlamauthor=K.+K.+Bowmanauthor=J.+Wuauthor=A.+M.+Giannettiauthor=M.+A.+Starovasnikauthor=I.+Mellmanauthor=P.+K.+Jacksonauthor=J.+Rudolphauthor=W.+Wangauthor=G.+Fang&title=Small-molecule+Ligands+Bind+to+a+Distinct+Pocket+in+Ras+and+Inhibit+SOS-mediated+Nucleotide+Exchange+Activity&doi=10.1073%2Fpnas.1116510109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule ligands bind to a distinct pocket in Ras and inhibit SOS-mediated nucleotide exchange activity</span></div><div class="casAuthors">Maurer, Till; Garrenton, Lindsay S.; Oh, Angela; Pitts, Keith; Anderson, Daniel J.; Skelton, Nicholas J.; Fauber, Benjamin P.; Pan, Borlan; Malek, Shiva; Stokoe, David; Ludlam, Mary J. C.; Bowman, Krista K.; Wu, Jiansheng; Giannetti, Anthony M.; Starovasnik, Melissa A.; Mellman, Ira; Jackson, Peter K.; Rudolph, Joachim; Wang, Weiru; Fang, Guowei</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5299-5304, S5299/1-S5299/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Ras gene is frequently mutated in cancer, and mutant Ras drives tumorigenesis.  Although Ras is a central oncogene, small mols. that bind to Ras in a well-defined manner and exert inhibitory effects have not been uncovered to date.  Through an NMR-based fragment screen, we identified a group of small mols. that all bind to a common site on Ras.  High-resoln. cocrystal structures delineated a unique ligand-binding pocket on the Ras protein that is adjacent to the switch I/II regions and can be expanded upon compd. binding.  Structure anal. predicts that compd.-binding interferes with the Ras/SOS interactions.  Indeed, selected compds. inhibit SOS-mediated nucleotide exchange and prevent Ras activation by blocking the formation of intermediates of the exchange reaction.  The discovery of a small-mol. binding pocket on Ras with functional significance provides a new direction in the search of therapeutically effective inhibitors of the Ras oncoprotein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAXIzMe7yBqrVg90H21EOLACvtfcHk0lh_SdmV8ORvpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslOjsrc%253D&md5=a7210cad45154c42eb2a1faeabf88c20</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1116510109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1116510109%26sid%3Dliteratum%253Aachs%26aulast%3DMaurer%26aufirst%3DT.%26aulast%3DGarrenton%26aufirst%3DL.%2BS.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DPitts%26aufirst%3DK.%26aulast%3DAnderson%26aufirst%3DD.%2BJ.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26aulast%3DFauber%26aufirst%3DB.%2BP.%26aulast%3DPan%26aufirst%3DB.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DLudlam%26aufirst%3DM.%2BJ.%2BC.%26aulast%3DBowman%26aufirst%3DK.%2BK.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DStarovasnik%26aufirst%3DM.%2BA.%26aulast%3DMellman%26aufirst%3DI.%26aulast%3DJackson%26aufirst%3DP.%2BK.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DFang%26aufirst%3DG.%26atitle%3DSmall-molecule%2520Ligands%2520Bind%2520to%2520a%2520Distinct%2520Pocket%2520in%2520Ras%2520and%2520Inhibit%2520SOS-mediated%2520Nucleotide%2520Exchange%2520Activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D5299%26epage%3D5304%26doi%3D10.1073%2Fpnas.1116510109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, J. J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blades, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassington, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breeze, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chresta, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Embrey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairley, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faulder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finlay, M. R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kettle, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spadola, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrigley, G.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Binding Sites on the Ras:SOS Complex Can Be Exploited for Inhibition of Ras Activation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2265</span>– <span class="NLM_lpage">2274</span>, <span class="refDoi"> DOI: 10.1021/jm501660t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501660t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivF2qu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2265-2274&author=J.+J.+G.+Winterauthor=M.+Andersonauthor=K.+Bladesauthor=C.+Brassingtonauthor=A.+L.+Breezeauthor=C.+Chrestaauthor=K.+Embreyauthor=G.+Fairleyauthor=P.+Faulderauthor=M.+R.+V.+Finlayauthor=J.+G.+Kettleauthor=T.+Nowakauthor=R.+Overmanauthor=S.+J.+Patelauthor=P.+Perkinsauthor=L.+Spadolaauthor=J.+Tartauthor=J.+A.+Tuckerauthor=G.+Wrigley&title=Small+Molecule+Binding+Sites+on+the+Ras%3ASOS+Complex+Can+Be+Exploited+for+Inhibition+of+Ras+Activation&doi=10.1021%2Fjm501660t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Binding Sites on the Ras:SOS Complex Can Be Exploited for Inhibition of Ras Activation</span></div><div class="casAuthors">Winter, Jon J. G.; Anderson, Malcolm; Blades, Kevin; Brassington, Claire; Breeze, Alexander L.; Chresta, Christine; Embrey, Kevin; Fairley, Gary; Faulder, Paul; Finlay, M. Raymond V.; Kettle, Jason G.; Nowak, Thorsten; Overman, Ross; Patel, S. Joe; Perkins, Paula; Spadola, Loredana; Tart, Jonathan; Tucker, Julie A.; Wrigley, Gail</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2265-2274</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Constitutively active mutant KRas displays a reduced rate of GTP hydrolysis via both intrinsic and GTPase-activating protein-catalyzed mechanisms, resulting in the perpetual activation of Ras pathways.  We describe a fragment screening campaign using X-ray crystallog. that led to the discovery of three fragment binding sites on the Ras:SOS complex.  The identification of tool compds. binding at each of these sites allowed exploration of two new approaches to Ras pathway inhibition by stabilizing or covalently modifying the Ras:SOS complex to prevent the reloading of Ras with GTP.  Initially, we identified ligands that bound reversibly to the Ras:SOS complex in two distinct sites, but these compds. were not sufficiently potent inhibitors to validate our stabilization hypothesis.  We conclude by demonstrating that covalent modification of Cys118 on Ras leads to a novel mechanism of inhibition of the SOS-mediated interaction between Ras and Raf and is effective at inhibiting the exchange of labeled GDP in both mutant (G12C and G12V) and wild type Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfpT9XplX1v7Vg90H21EOLACvtfcHk0ljmdy_5SriSlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivF2qu7w%253D&md5=b3747c8c9bb530d39c03ad56998d90f1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm501660t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501660t%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DAnderson%26aufirst%3DM.%26aulast%3DBlades%26aufirst%3DK.%26aulast%3DBrassington%26aufirst%3DC.%26aulast%3DBreeze%26aufirst%3DA.%2BL.%26aulast%3DChresta%26aufirst%3DC.%26aulast%3DEmbrey%26aufirst%3DK.%26aulast%3DFairley%26aufirst%3DG.%26aulast%3DFaulder%26aufirst%3DP.%26aulast%3DFinlay%26aufirst%3DM.%2BR.%2BV.%26aulast%3DKettle%26aufirst%3DJ.%2BG.%26aulast%3DNowak%26aufirst%3DT.%26aulast%3DOverman%26aufirst%3DR.%26aulast%3DPatel%26aufirst%3DS.%2BJ.%26aulast%3DPerkins%26aufirst%3DP.%26aulast%3DSpadola%26aufirst%3DL.%26aulast%3DTart%26aufirst%3DJ.%26aulast%3DTucker%26aufirst%3DJ.%2BA.%26aulast%3DWrigley%26aufirst%3DG.%26atitle%3DSmall%2520Molecule%2520Binding%2520Sites%2520on%2520the%2520Ras%253ASOS%2520Complex%2520Can%2520Be%2520Exploited%2520for%2520Inhibition%2520of%2520Ras%2520Activation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2265%26epage%3D2274%26doi%3D10.1021%2Fjm501660t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hillig, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sautier, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilpmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmann, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werbeck, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boemer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mastouri, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siemeister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahmann, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegener, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhnke, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wortmann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Nussbaum, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent SOS1 Inhibitors that Block RAS Activation via Disruption of the RAS–SOS1 Interaction</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">2551</span>– <span class="NLM_lpage">2560</span>, <span class="refDoi"> DOI: 10.1073/pnas.1812963116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1073%2Fpnas.1812963116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=30683722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivVajs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=2551-2560&author=R.+C.+Hilligauthor=B.+Sautierauthor=J.+Schroederauthor=D.+Moosmayerauthor=A.+Hilpmannauthor=C.+M.+Stegmannauthor=N.+D.+Werbeckauthor=H.+Briemauthor=U.+Boemerauthor=J.+Weiskeauthor=V.+Badockauthor=J.+Mastouriauthor=K.+Petersenauthor=G.+Siemeisterauthor=J.+D.+Kahmannauthor=D.+Wegenerauthor=N.+B%C3%B6hnkeauthor=K.+Eisauthor=K.+Grahamauthor=L.+Wortmannauthor=F.+von+Nussbaumauthor=B.+Bader&title=Discovery+of+Potent+SOS1+Inhibitors+that+Block+RAS+Activation+via+Disruption+of+the+RAS%E2%80%93SOS1+Interaction&doi=10.1073%2Fpnas.1812963116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of potent SOS1 inhibitors that block RAS activation via disruption of the RAS-SOS1 interaction</span></div><div class="casAuthors">Hillig, Roman C.; Sautier, Brice; Schroeder, Jens; Moosmayer, Dieter; Hilpmann, AndrA; Stegmann, Christian M.; Werbeck, Nicolas D.; Briem, Hans; Boemer, Ulf; Weiske, Joerg; Badock, Volker; Mastouri, Julia; Petersen, Kirstin; Siemeister, Gerhard; Kahmann, Jan D.; Wegener, Dennis; Boehnke, Niels; Eis, Knut; Graham, Keith; Wortmann, Lars; von Nussbaum, Franz; Bader, Benjamin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2551-2560</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Since the late 1980s, mutations in the RAS genes have been recognized as major oncogenes with a high occurrence rate in human cancers.  Such mutations reduce the ability of the small GTPase RAS to hydrolyze GTP, keeping this mol. switch in a constitutively active GTP-bound form that drives, unchecked, oncogenic downstream signaling.  One strategy to reduce the levels of active RAS is to target guanine nucleotide exchange factors, which allow RAS to cycle from the inactive GDP-bound state to the active GTP-bound form.  Here, we describe the identification of potent and cell-active small-mol. inhibitors which efficiently disrupt the interaction between KRAS and its exchange factor SOS1, a mode of action confirmed by a series of biophys. techniques.  The binding sites, mode of action, and selectivity were elucidated using crystal structures of KRASG12C-SOS1, SOS1, and SOS2.  By preventing formation of the KRAS-SOS1 complex, these inhibitors block reloading of KRAS with GTP, leading to antiproliferative activity.  The final compd. 23(I, BAY-293) selectively inhibits the KRAS-SOS1 interaction with an IC50 of 21 nM and is a valuable chem. probe for future investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTMAcBZFHkBrVg90H21EOLACvtfcHk0ljWNUqpsKh1rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivVajs7w%253D&md5=f3150723049c15986cea1d7f93cb89af</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1812963116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1812963116%26sid%3Dliteratum%253Aachs%26aulast%3DHillig%26aufirst%3DR.%2BC.%26aulast%3DSautier%26aufirst%3DB.%26aulast%3DSchroeder%26aufirst%3DJ.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DHilpmann%26aufirst%3DA.%26aulast%3DStegmann%26aufirst%3DC.%2BM.%26aulast%3DWerbeck%26aufirst%3DN.%2BD.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DBoemer%26aufirst%3DU.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DMastouri%26aufirst%3DJ.%26aulast%3DPetersen%26aufirst%3DK.%26aulast%3DSiemeister%26aufirst%3DG.%26aulast%3DKahmann%26aufirst%3DJ.%2BD.%26aulast%3DWegener%26aufirst%3DD.%26aulast%3DB%25C3%25B6hnke%26aufirst%3DN.%26aulast%3DEis%26aufirst%3DK.%26aulast%3DGraham%26aufirst%3DK.%26aulast%3DWortmann%26aufirst%3DL.%26aulast%3Dvon%2BNussbaum%26aufirst%3DF.%26aulast%3DBader%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520Potent%2520SOS1%2520Inhibitors%2520that%2520Block%2520RAS%2520Activation%2520via%2520Disruption%2520of%2520the%2520RAS%25E2%2580%2593SOS1%2520Interaction%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D2551%26epage%3D2560%26doi%3D10.1073%2Fpnas.1812963116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbott, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Little, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salovich, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howes, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akan, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browning, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobolik, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beesetty, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadtmueller, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossanese, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovery and Structure-Based Optimization of Benzimidazole-Derived Activators of SOS1-Mediated Nucleotide Exchange on RAS</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8875</span>– <span class="NLM_lpage">8894</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01108</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01108" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs12isrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8875-8894&author=T.+R.+Hodgesauthor=J.+R.+Abbottauthor=A.+J.+Littleauthor=D.+Sarkarauthor=J.+M.+Salovichauthor=J.+E.+Howesauthor=D.+T.+Akanauthor=J.+Saiauthor=A.+L.+Arnoldauthor=C.+Browningauthor=M.+C.+Burnsauthor=T.+Sobolikauthor=Q.+Sunauthor=Y.+Beesettyauthor=J.+A.+Cokerauthor=D.+Scharnauthor=H.+Stadtmuellerauthor=O.+W.+Rossaneseauthor=J.+Phanauthor=A.+G.+Watersonauthor=D.+B.+McConnellauthor=S.+W.+Fesik&title=Discovery+and+Structure-Based+Optimization+of+Benzimidazole-Derived+Activators+of+SOS1-Mediated+Nucleotide+Exchange+on+RAS&doi=10.1021%2Facs.jmedchem.8b01108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Structure-Based Optimization of Benzimidazole-Derived Activators of SOS1-Mediated Nucleotide Exchange on RAS</span></div><div class="casAuthors">Hodges, Timothy R.; Abbott, Jason R.; Little, Andrew J.; Sarkar, Dhruba; Salovich, James M.; Howes, Jennifer E.; Akan, Denis T.; Sai, Jiqing; Arnold, Allison L.; Browning, Carrie; Burns, Michael C.; Sobolik, Tammy; Sun, Qi; Beesetty, Yugandhar; Coker, Jesse A.; Scharn, Dirk; Stadtmueller, Heinz; Rossanese, Olivia W.; Phan, Jason; Waterson, Alex G.; McConnell, Darryl B.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8875-8894</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Son of sevenless homolog 1 (SOS1) is a guanine nucleotide exchange factor that catalyzes the exchange of GDP for GTP on RAS.  In its active form, GTP-bound RAS is responsible for numerous crit. cellular processes.  Aberrant RAS activity is involved in ∼30% of all human cancers; hence, SOS1 is an attractive therapeutic target for its role in modulating RAS activation.  Here, we describe a new series of benzimidazole-derived SOS1 agonists.  Using structure-guided design, we discovered small mols. that increase nucleotide exchange on RAS in vitro at submicromolar concns., bind to SOS1 with low double-digit nanomolar affinity, rapidly enhance cellular RAS-GTP levels, and invoke biphasic signaling changes in phosphorylation of ERK 1/2.  These compds. represent the most potent series of SOS1 agonists reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnQbL1ADAtRrVg90H21EOLACvtfcHk0ljWNUqpsKh1rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs12isrnL&md5=3051827da87f015c260000ea7b2c7c34</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01108%26sid%3Dliteratum%253Aachs%26aulast%3DHodges%26aufirst%3DT.%2BR.%26aulast%3DAbbott%26aufirst%3DJ.%2BR.%26aulast%3DLittle%26aufirst%3DA.%2BJ.%26aulast%3DSarkar%26aufirst%3DD.%26aulast%3DSalovich%26aufirst%3DJ.%2BM.%26aulast%3DHowes%26aufirst%3DJ.%2BE.%26aulast%3DAkan%26aufirst%3DD.%2BT.%26aulast%3DSai%26aufirst%3DJ.%26aulast%3DArnold%26aufirst%3DA.%2BL.%26aulast%3DBrowning%26aufirst%3DC.%26aulast%3DBurns%26aufirst%3DM.%2BC.%26aulast%3DSobolik%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DBeesetty%26aufirst%3DY.%26aulast%3DCoker%26aufirst%3DJ.%2BA.%26aulast%3DScharn%26aufirst%3DD.%26aulast%3DStadtmueller%26aufirst%3DH.%26aulast%3DRossanese%26aufirst%3DO.%2BW.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520and%2520Structure-Based%2520Optimization%2520of%2520Benzimidazole-Derived%2520Activators%2520of%2520SOS1-Mediated%2520Nucleotide%2520Exchange%2520on%2520RAS%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8875%26epage%3D8894%26doi%3D10.1021%2Facs.jmedchem.8b01108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miwa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niida, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sameshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamaura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonemori, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, J-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of a Human K-Ras G12D Mutant in Complex with GDP and the Cyclic Inhibitory Peptide KRpep-2d</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00128</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00128" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVWrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=732-736&author=S.+Sogabeauthor=Y.+Kamadaauthor=M.+Miwaauthor=A.+Niidaauthor=T.+Sameshimaauthor=M.+Kamauraauthor=K.+Yonemoriauthor=S.+Sasakiauthor=J-i.+Sakamotoauthor=K.+Sakamoto&title=Crystal+Structure+of+a+Human+K-Ras+G12D+Mutant+in+Complex+with+GDP+and+the+Cyclic+Inhibitory+Peptide+KRpep-2d&doi=10.1021%2Facsmedchemlett.7b00128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of a Human K-Ras G12D Mutant in Complex with GDP and the Cyclic Inhibitory Peptide KRpep-2d</span></div><div class="casAuthors">Sogabe, Satoshi; Kamada, Yusuke; Miwa, Masanori; Niida, Ayumu; Sameshima, Tomoya; Kamaura, Masahiro; Yonemori, Kazuko; Sasaki, Shigekazu; Sakamoto, Jun-ichi; Sakamoto, Kotaro</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ras proteins play roles in cell differentiation, proliferation, and survival.  Aberrant signaling through Ras-mediated pathways in tumor cells occurs as a result of several types of mutational damage, which most frequently affects the amino acids Gly-12, Gly-13, and Gln-61.  Recently, KRpep-2d was identified as a K-Ras(G12D) selective inhibitory peptide against the G12D mutant of K-Ras, which is a key member of the Ras protein family and an attractive cancer therapeutic target.  Here, the crystal structure of the human K-Ras(G12D) mutant was detd. in complex with GDP and KRpep-2d at 1.25 Å resoln.  This structure revealed that the peptide binds near switch II and allosterically blocks protein-protein interactions with the guanine nucleotide exchange factor.  This discovery of a unique binding pocket provides valuable information that will facilitate the design of direct Ras inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomKfkAMpR_QrVg90H21EOLACvtfcHk0ljWNUqpsKh1rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVWrtL4%253D&md5=0658bfe5c7e48365e4d44b71011b374a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00128%26sid%3Dliteratum%253Aachs%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DMiwa%26aufirst%3DM.%26aulast%3DNiida%26aufirst%3DA.%26aulast%3DSameshima%26aufirst%3DT.%26aulast%3DKamaura%26aufirst%3DM.%26aulast%3DYonemori%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DS.%26aulast%3DSakamoto%26aufirst%3DJ-i.%26aulast%3DSakamoto%26aufirst%3DK.%26atitle%3DCrystal%2520Structure%2520of%2520a%2520Human%2520K-Ras%2520G12D%2520Mutant%2520in%2520Complex%2520with%2520GDP%2520and%2520the%2520Cyclic%2520Inhibitory%2520Peptide%2520KRpep-2d%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D732%26epage%3D736%26doi%3D10.1021%2Facsmedchemlett.7b00128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGregor, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerwin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Expanding the Scope of Electrophiles Capable of Targeting K-Ras Oncogenes</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3178</span>– <span class="NLM_lpage">3183</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.7b00271</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.7b00271" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVGrsbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=3178-3183&author=L.+M.+McGregorauthor=M.+L.+Jenkinsauthor=C.+Kerwinauthor=J.+E.+Burkeauthor=K.+M.+Shokat&title=Expanding+the+Scope+of+Electrophiles+Capable+of+Targeting+K-Ras+Oncogenes&doi=10.1021%2Facs.biochem.7b00271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding the Scope of Electrophiles Capable of Targeting K-Ras Oncogenes</span></div><div class="casAuthors">McGregor, Lynn M.; Jenkins, Meredith L.; Kerwin, Caitlin; Burke, John E.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">3178-3183</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is growing interest in reversible and irreversible covalent inhibitors that target noncatalytic amino acids in target proteins.  With a goal of targeting oncogenic K-Ras variants (e.g., G12D) by expanding the types of amino acids that can be targeted by covalent inhibitors, we survey a set of electrophiles for their ability to label carboxylates.  We functionalized an optimized ligand for the K-Ras switch II pocket with a set of electrophiles previously reported to react with carboxylates and characterized the ability of these compds. to react with model nucleophiles and oncogenic K-Ras proteins.  Here, we report that aziridines and stabilized diazo groups preferentially react with free carboxylates over thiols.  Although we did not identify a warhead that potently labels K-Ras G12D, we were able to study the interactions of many electrophiles with K-Ras, as most of the electrophiles rapidly label K-Ras G12C.  We characterized the resulting complexes by crystallog., hydrogen/deuterium exchange, and differential scanning fluorometry.  Our results both demonstrate the ability of a noncatalytic cysteine to react with a diverse set of electrophiles and emphasize the importance of proper spatial arrangements between a covalent inhibitor and its intended nucleophile.  We hope that these results can expand the range of electrophiles and nucleophiles of use in covalent protein modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcvKWYdfIxw7Vg90H21EOLACvtfcHk0ljvWC66EqVGuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVGrsbnI&md5=0b25b65210248a4cad2916efba90ddae</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.7b00271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.7b00271%26sid%3Dliteratum%253Aachs%26aulast%3DMcGregor%26aufirst%3DL.%2BM.%26aulast%3DJenkins%26aufirst%3DM.%2BL.%26aulast%3DKerwin%26aufirst%3DC.%26aulast%3DBurke%26aufirst%3DJ.%2BE.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DExpanding%2520the%2520Scope%2520of%2520Electrophiles%2520Capable%2520of%2520Targeting%2520K-Ras%2520Oncogenes%26jtitle%3DBiochemistry%26date%3D2017%26volume%3D56%26spage%3D3178%26epage%3D3183%26doi%3D10.1021%2Facs.biochem.7b00271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manandhar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gondi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westover, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Covalent Guanosine Mimetic Inhibitors of G12C KRAS</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFyntrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=61-66&author=Y.+Xiongauthor=J.+Luauthor=J.+Hunterauthor=L.+Liauthor=D.+Scottauthor=H.+G.+Choiauthor=S.+M.+Limauthor=A.+Manandharauthor=S.+Gondiauthor=T.+Simauthor=K.+D.+Westoverauthor=N.+S.+Gray&title=Covalent+Guanosine+Mimetic+Inhibitors+of+G12C+KRAS&doi=10.1021%2Facsmedchemlett.6b00373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Guanosine Mimetic Inhibitors of G12C KRAS</span></div><div class="casAuthors">Xiong, Yuan; Lu, Jia; Hunter, John; Li, Lianbo; Scott, David; Choi, Hwan Geun; Lim, Sang Min; Manandhar, Anuj; Gondi, Sudershan; Sim, Taebo; Westover, Kenneth D.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">61-66</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ras proteins are members of a large family of GTPase enzymes that are commonly mutated in cancer where they act as dominant oncogenes.  The authors previously developed an irreversible guanosine-derived inhibitor, SML-8-73-1, of mutant G12C RAS which forms a covalent bond with cysteine 12.  Here the authors report exploration of the structure-activity relationships (SAR) of hydrolytically stable analogs of SML-8-73-1 as covalent G12C KRAS inhibitors.  The authors report the discovery of difluoromethylene bisphosphonate analogs such as compd. I, which despite exhibiting reduced efficiency as covalent G12C KRAS inhibitors, remove the liability of the hydrolytic instability of the diphosphate moiety present in SML-8-73-1 and provide the foundation for development of prodrugs to facilitate cellular uptake.  The SAR and crystallog. results reaffirm the exquisite mol. recognition that exists in the diphosphate-region of RAS for guanosine nucleotides which must be considered in the design of nucleotide-competitive inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraZMoq5jUr_rVg90H21EOLACvtfcHk0ljvWC66EqVGuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFyntrbF&md5=b36c23e91f8a3c3f30aa9819d931870f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00373%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DHunter%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DScott%26aufirst%3DD.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DLim%26aufirst%3DS.%2BM.%26aulast%3DManandhar%26aufirst%3DA.%26aulast%3DGondi%26aufirst%3DS.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DWestover%26aufirst%3DK.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DCovalent%2520Guanosine%2520Mimetic%2520Inhibitors%2520of%2520G12C%2520KRAS%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D61%26epage%3D66%26doi%3D10.1021%2Facsmedchemlett.6b00373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ostrem, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sos, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">K-Ras(G12C) Inhibitors Allosterically Control GTP Affinity and Effector Interactions</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">548</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.1038/nature12796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1038%2Fnature12796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=24256730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=548-551&author=J.+M.+Ostremauthor=U.+Petersauthor=M.+L.+Sosauthor=J.+A.+Wellsauthor=K.+M.+Shokat&title=K-Ras%28G12C%29+Inhibitors+Allosterically+Control+GTP+Affinity+and+Effector+Interactions&doi=10.1038%2Fnature12796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span></div><div class="casAuthors">Ostrem, Jonathan M.; Peters, Ulf; Sos, Martin L.; Wells, James A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7477</span>),
    <span class="NLM_cas:pages">548-551</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Somatic mutations in the small GTPase K-Ras are the most common activating lesions found in human cancer, and are generally assocd. with poor response to std. therapies.  Efforts to target this oncogene directly have faced difficulties owing to its picomolar affinity for GTP/GDP and the absence of known allosteric regulatory sites.  Oncogenic mutations result in functional activation of Ras family proteins by impairing GTP hydrolysis.  With diminished regulation by GTPase activity, the nucleotide state of Ras becomes more dependent on relative nucleotide affinity and concn.  This gives GTP an advantage over GDP and increases the proportion of active GTP-bound Ras.  Here we report the development of small mols. that irreversibly bind to a common oncogenic mutant, K-Ras(G12C).  These compds. rely on the mutant cysteine for binding and therefore do not affect the wild-type protein.  Crystallog. studies reveal the formation of a new pocket that is not apparent in previous structures of Ras, beneath the effector binding switch-II region.  Binding of these inhibitors to K-Ras(G12C) disrupts both switch-I and switch-II, subverting the native nucleotide preference to favor GDP over GTP and impairing binding to Raf.  Our data provide structure-based validation of a new allosteric regulatory site on Ras that is targetable in a mutant-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokkgcVXuBn_rVg90H21EOLACvtfcHk0ljIz-vAO89XAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO&md5=27c1e7d8ef6b7e4b2919e9f00a073923</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature12796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12796%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DK-Ras%2528G12C%2529%2520Inhibitors%2520Allosterically%2520Control%2520GTP%2520Affinity%2520and%2520Effector%2520Interactions%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D548%26epage%3D551%26doi%3D10.1038%2Fnature12796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janes, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucharski, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ely, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firdaus, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibition of Oncogenic KRAS Output with Small Molecules Targeting the Inactive State</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-1105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1158%2F2159-8290.CD-15-1105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=26739882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVyru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=316-329&author=M.+P.+Patricelliauthor=M.+R.+Janesauthor=L.-S.+Liauthor=R.+Hansenauthor=U.+Petersauthor=L.+V.+Kesslerauthor=Y.+Chenauthor=J.+M.+Kucharskiauthor=J.+Fengauthor=T.+Elyauthor=J.+H.+Chenauthor=S.+J.+Firdausauthor=A.+Babbarauthor=P.+Renauthor=Y.+Liu&title=Selective+Inhibition+of+Oncogenic+KRAS+Output+with+Small+Molecules+Targeting+the+Inactive+State&doi=10.1158%2F2159-8290.CD-15-1105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Oncogenic KRAS Output with Small Molecules Targeting the Inactive State</span></div><div class="casAuthors">Patricelli, Matthew P.; Janes, Matthew R.; Li, Lian-Sheng; Hansen, Rasmus; Peters, Ulf; Kessler, Linda V.; Chen, Yuching; Kucharski, Jeff M.; Feng, Jun; Ely, Tess; Chen, Jeffrey H.; Firdaus, Sarah J.; Babbar, Anjali; Ren, Pingda; Liu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">316-329</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">KRAS gain-of-function mutations occur in approx. 30% of all human cancers.  Despite more than 30 years of KRAS-focused research and development efforts, no targeted therapy has been discovered for cancers with KRAS mutations.  Here, we describe ARS-853, a selective, covalent inhibitor of KRASG12C that inhibits mutant KRAS-driven signaling by binding to the GDP-bound oncoprotein and preventing activation.  Based on the rates of engagement and inhibition obsd. for ARS-853, along with a mutant-specific mass spectrometry-based assay for assessing KRAS activation status, we show that the nucleotide state of KRASG12C is in a state of dynamic flux that can be modulated by upstream signaling factors.  These studies provide convincing evidence that the KRASG12C mutation generates a "hyperexcitable" rather than a "statically active" state and that targeting the inactive, GDP-bound form is a promising approach for generating novel anti-RAS therapeutics.  Significance: A cell-active, mutant-specific, covalent inhibitor of KRASG12C is described that targets the GDP-bound, inactive state and prevents subsequent activation.  Using this novel compd., we demonstrate that KRASG12C oncoprotein rapidly cycles bound nucleotide and responds to upstream signaling inputs to maintain a highly active state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJJA-9IIHeN7Vg90H21EOLACvtfcHk0ljIz-vAO89XAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVyru7s%253D&md5=41db2f02006e8f0be7a9bb47354addcd</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-1105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-1105%26sid%3Dliteratum%253Aachs%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DLi%26aufirst%3DL.-S.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DKessler%26aufirst%3DL.%2BV.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKucharski%26aufirst%3DJ.%2BM.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DEly%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DFirdaus%26aufirst%3DS.%2BJ.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DSelective%2520Inhibition%2520of%2520Oncogenic%2520KRAS%2520Output%2520with%2520Small%2520Molecules%2520Targeting%2520the%2520Inactive%2520State%26jtitle%3DCancer%2520Discovery%26date%3D2016%26volume%3D6%26spage%3D316%26epage%3D329%26doi%3D10.1158%2F2159-8290.CD-15-1105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janes, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firdaus, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darjania, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarieh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ely, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucharski, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brehmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu-Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting KRAS Mutant Cancers with a Covalent G12C-Specific Inhibitor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">578</span>– <span class="NLM_lpage">589</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1016%2Fj.cell.2018.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=29373830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1cXislyisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2018&pages=578-589&author=M.+R.+Janesauthor=J.+Zhangauthor=L.-S.+Liauthor=R.+Hansenauthor=U.+Petersauthor=X.+Guoauthor=Y.+Chenauthor=A.+Babbarauthor=S.+J.+Firdausauthor=L.+Darjaniaauthor=J.+Fengauthor=J.+H.+Chenauthor=S.+Liauthor=S.+Liauthor=Y.+O.+Longauthor=C.+Thachauthor=Y.+Liuauthor=A.+Zariehauthor=T.+Elyauthor=J.+M.+Kucharskiauthor=L.+V.+Kesslerauthor=T.+Wuauthor=K.+Yuauthor=Y.+Wangauthor=Y.+Yaoauthor=X.+Dengauthor=P.+P.+Zarrinkarauthor=D.+Brehmerauthor=D.+Dhanakauthor=M.+V.+Lorenziauthor=D.+Hu-Loweauthor=M.+P.+Patricelliauthor=P.+Renauthor=Y.+Liu&title=Targeting+KRAS+Mutant+Cancers+with+a+Covalent+G12C-Specific+Inhibitor&doi=10.1016%2Fj.cell.2018.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting KRAS Mutant Cancers with a Covalent G12C-Specific Inhibitor</span></div><div class="casAuthors">Janes, Matthew R.; Zhang, Jingchuan; Li, Lian-Sheng; Hansen, Rasmus; Peters, Ulf; Guo, Xin; Chen, Yuching; Babbar, Anjali; Firdaus, Sarah J.; Darjania, Levan; Feng, Jun; Chen, Jeffrey H.; Li, Shuangwei; Li, Shisheng; Long, Yun O.; Thach, Carol; Liu, Yuan; Zarieh, Ata; Ely, Tess; Kucharski, Jeff M.; Kessler, Linda V.; Wu, Tao; Yu, Ke; Wang, Yi; Yao, Yvonne; Deng, Xiaohu; Zarrinkar, Patrick P.; Brehmer, Dirk; Dhanak, Dashyant; Lorenzi, Matthew V.; Hu-Lowe, Dana; Patricelli, Matthew P.; Ren, Pingda; Liu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">578-589.e17</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">KRASG12C was recently identified to be potentially druggable by allele-specific covalent targeting of Cys 12 in vicinity to an inducible allosteric switch II pocket (S-IIP).  Success of this approach requires active cycling of KRASG12C between its active-GTP and inactive-GDP conformations as accessibility of the S-IIP is restricted only to the GDP-bound state.  This strategy proved feasible for inhibiting mutant KRAS in vitro; however, it is uncertain whether this approach would translate to in vivo.  Here, the authors describe structure-based design and identification of ARS-1620, a covalent compd. with high potency and selectivity for KRASG12C.  ARS-1620 achieves rapid and sustained in vivo target occupancy to induce tumor regression.  The authors use ARS-1620 to dissect oncogenic KRAS dependency and demonstrate that monolayer culture formats significantly underestimate KRAS dependency in vivo.  This study provides in vivo evidence that mutant KRAS can be selectively targeted and reveals ARS-1620 as representing a new generation of KRASG12C-specific inhibitors with promising therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVmv4KAl4_dLVg90H21EOLACvtfcHk0lgsJRmezt5Qlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXislyisb8%253D&md5=d0883bde981b51e41a794179ffae87be</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.-S.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DGuo%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DFirdaus%26aufirst%3DS.%2BJ.%26aulast%3DDarjania%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLong%26aufirst%3DY.%2BO.%26aulast%3DThach%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZarieh%26aufirst%3DA.%26aulast%3DEly%26aufirst%3DT.%26aulast%3DKucharski%26aufirst%3DJ.%2BM.%26aulast%3DKessler%26aufirst%3DL.%2BV.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DHu-Lowe%26aufirst%3DD.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DTargeting%2520KRAS%2520Mutant%2520Cancers%2520with%2520a%2520Covalent%2520G12C-Specific%2520Inhibitor%26jtitle%3DCell%26date%3D2018%26volume%3D172%26spage%3D578%26epage%3D589%26doi%3D10.1016%2Fj.cell.2018.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janes, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">The Reactivity-driven Biochemical Mechanism of Covalent KRAS<sup>G12C</sup> Inhibitors</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">454</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.1038/s41594-018-0061-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1038%2Fs41594-018-0061-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=29760531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKqu7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=454-462&author=R.+Hansenauthor=U.+Petersauthor=A.+Babbarauthor=Y.+Chenauthor=J.+Fengauthor=M.+R.+Janesauthor=L.-S.+Liauthor=P.+Renauthor=Y.+Liuauthor=P.+P.+Zarrinkar&title=The+Reactivity-driven+Biochemical+Mechanism+of+Covalent+KRASG12C+Inhibitors&doi=10.1038%2Fs41594-018-0061-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The reactivity-driven biochemical mechanism of covalent KRASG12C inhibitors</span></div><div class="casAuthors">Hansen, Rasmus; Peters, Ulf; Babbar, Anjali; Chen, Yuching; Feng, Jun; Janes, Matthew R.; Li, Lian-Sheng; Ren, Pingda; Liu, Yi; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">454-462</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Activating mutations in KRAS are among the most common tumor driver mutations.  Until recently, KRAS had been considered undruggable with small mols.; the discovery of the covalent KRASG12C inhibitors ARS-853 and ARS-1620 has demonstrated that it is feasible to inhibit KRAS with high potency in cells and animals.  Although the biol. activity of these inhibitors has been described, the biochem. mechanism of how the compds. achieve potent inhibition remained incompletely understood.  We now show that the activity of ARS-853 and ARS-1620 is primarily driven by KRAS-mediated catalysis of the chem. reaction with Cys12 in human KRASG12C, while the reversible binding affinity is weak, in the hundreds of micromolar or higher range.  The mechanism resolves how an induced, shallow and dynamic pocket not expected to support high-affinity binding of small mols. can nevertheless be targeted with potent inhibitors and may be applicable to other targets conventionally considered undruggable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrya_A4ZYnOu7Vg90H21EOLACvtfcHk0lhv93vpDdq2oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKqu7nL&md5=b3f9198f55998306f3b881c256be22a9</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41594-018-0061-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-018-0061-5%26sid%3Dliteratum%253Aachs%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DLi%26aufirst%3DL.-S.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DThe%2520Reactivity-driven%2520Biochemical%2520Mechanism%2520of%2520Covalent%2520KRASG12C%2520Inhibitors%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D454%26epage%3D462%26doi%3D10.1038%2Fs41594-018-0061-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lonsdale, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span> <span> </span><span class="NLM_article-title">Structure-based Design of Targeted Covalent Inhibitors</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">3816</span>– <span class="NLM_lpage">3830</span>, <span class="refDoi"> DOI: 10.1039/C7CS00220C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1039%2FC7CS00220C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=29620097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntVGiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2018&pages=3816-3830&author=R.+Lonsdaleauthor=R.+A.+Ward&title=Structure-based+Design+of+Targeted+Covalent+Inhibitors&doi=10.1039%2FC7CS00220C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of targeted covalent inhibitors</span></div><div class="casAuthors">Lonsdale, Richard; Ward, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3816-3830</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Covalent inhibition is a rapidly growing discipline within drug discovery.  Many historical covalent inhibitors were discovered by serendipity, with such a mechanism of action often regarded as undesirable due to potential toxicity issues.  Recent progress has seen a major shift in this outlook, as covalent inhibition shows promise for targets where previous efforts to identify non-covalent small mol. inhibitors have failed.  Targeted covalent inhibitors (TCIs) can offer drug discovery scientists the ability to increase the potency and/or selectivity of small mol. inhibitors, by attachment of reactive functional groups designed to covalently bind to specific sites in a target.  In this tutorial review we introduce the broader concept of covalent inhibition, discuss the potential benefits and challenges of such an approach, and provide an overview of the current status of the field.  We also describe some strategies and computational tools to enable successful covalent drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnSIGMuS3vm7Vg90H21EOLACvtfcHk0lhv93vpDdq2oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntVGiurs%253D&md5=a7e31b23cf8ecfcad1397ad10ea10b7b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1039%2FC7CS00220C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CS00220C%26sid%3Dliteratum%253Aachs%26aulast%3DLonsdale%26aufirst%3DR.%26aulast%3DWard%26aufirst%3DR.%2BA.%26atitle%3DStructure-based%2520Design%2520of%2520Targeted%2520Covalent%2520Inhibitors%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2018%26volume%3D47%26spage%3D3816%26epage%3D3830%26doi%3D10.1039%2FC7CS00220C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lanman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amegadzie, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Booker, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frohn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopecky, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minatti, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickrell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegmund, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamayo, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tegley, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walton, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurz, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Achanta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartberger, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollis, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rex, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiki, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San Miguel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volak, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittington, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zech, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cee, V. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Covalent Inhibitor of KRASG12C (AMG 510) for the Treatment of Solid Tumors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">52</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01180</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01180" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlGitrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=52-65&author=B.+A.+Lanmanauthor=J.+R.+Allenauthor=J.+G.+Allenauthor=A.+K.+Amegadzieauthor=K.+S.+Ashtonauthor=S.+K.+Bookerauthor=J.+J.+Chenauthor=N.+Chenauthor=M.+J.+Frohnauthor=G.+Goodmanauthor=D.+J.+Kopeckyauthor=L.+Liuauthor=P.+Lopezauthor=J.+D.+Lowauthor=V.+Maauthor=A.+E.+Minattiauthor=T.+T.+Nguyenauthor=N.+Nishimuraauthor=A.+J.+Pickrellauthor=A.+B.+Reedauthor=Y.+Shinauthor=A.+C.+Siegmundauthor=N.+A.+Tamayoauthor=C.+M.+Tegleyauthor=M.+C.+Waltonauthor=H.-L.+Wangauthor=R.+P.+Wurzauthor=M.+Xueauthor=K.+C.+Yangauthor=P.+Achantaauthor=M.+D.+Bartbergerauthor=J.+Canonauthor=L.+S.+Hollisauthor=J.+D.+McCarterauthor=C.+Mohrauthor=K.+Rexauthor=A.+Y.+Saikiauthor=T.+San+Miguelauthor=L.+P.+Volakauthor=K.+H.+Wangauthor=D.+A.+Whittingtonauthor=S.+G.+Zechauthor=J.+R.+Lipfordauthor=V.+J.+Cee&title=Discovery+of+a+Covalent+Inhibitor+of+KRASG12C+%28AMG+510%29+for+the+Treatment+of+Solid+Tumors&doi=10.1021%2Facs.jmedchem.9b01180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Covalent Inhibitor of KRASG12C (AMG 510) for the Treatment of Solid Tumors</span></div><div class="casAuthors">Lanman, Brian A.; Allen, Jennifer R.; Allen, John G.; Amegadzie, Albert K.; Ashton, Kate S.; Booker, Shon K.; Chen, Jian Jeffrey; Chen, Ning; Frohn, Michael J.; Goodman, Guy; Kopecky, David J.; Liu, Longbin; Lopez, Patricia; Low, Jonathan D.; Ma, Vu; Minatti, Ana E.; Nguyen, Thomas T.; Nishimura, Nobuko; Pickrell, Alexander J.; Reed, Anthony B.; Shin, Youngsook; Siegmund, Aaron C.; Tamayo, Nuria A.; Tegley, Christopher M.; Walton, Mary C.; Wang, Hui-Ling; Wurz, Ryan P.; Xue, May; Yang, Kevin C.; Achanta, Pragathi; Bartberger, Michael D.; Canon, Jude; Hollis, L. Steven; McCarter, John D.; Mohr, Christopher; Rex, Karen; Saiki, Anne Y.; San Miguel, Tisha; Volak, Laurie P.; Wang, Kevin H.; Whittington, Douglas A.; Zech, Stephan G.; Lipford, J. Russell; Cee, Victor J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">52-65</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KRASG12C has emerged as a promising target in the treatment of solid tumors.  Covalent inhibitors targeting the mutant cysteine-12 residue have been shown to disrupt signaling by this long-"undruggable" target; however clin. viable inhibitors have yet to be identified.  Here, we report efforts to exploit a cryptic pocket (H95/Y96/Q99) we identified in KRASG12C to identify inhibitors suitable for clin. development.  Structure-based design efforts leading to the identification of a novel quinazolinone scaffold are described, along with optimization efforts that overcame a configurational stability issue arising from restricted rotation about an axially chiral biaryl bond.  Biopharmaceutical optimization of the resulting leads culminated in the identification of AMG 510, a highly potent, selective, and well-tolerated KRASG12C inhibitor currently in phase I clin. trials (NCT03600883).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGeWqRRXX6lbVg90H21EOLACvtfcHk0ljOywnFsjs4uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlGitrrK&md5=d973189be70bb913020edb4989f6ba7d</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01180%26sid%3Dliteratum%253Aachs%26aulast%3DLanman%26aufirst%3DB.%2BA.%26aulast%3DAllen%26aufirst%3DJ.%2BR.%26aulast%3DAllen%26aufirst%3DJ.%2BG.%26aulast%3DAmegadzie%26aufirst%3DA.%2BK.%26aulast%3DAshton%26aufirst%3DK.%2BS.%26aulast%3DBooker%26aufirst%3DS.%2BK.%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DChen%26aufirst%3DN.%26aulast%3DFrohn%26aufirst%3DM.%2BJ.%26aulast%3DGoodman%26aufirst%3DG.%26aulast%3DKopecky%26aufirst%3DD.%2BJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLopez%26aufirst%3DP.%26aulast%3DLow%26aufirst%3DJ.%2BD.%26aulast%3DMa%26aufirst%3DV.%26aulast%3DMinatti%26aufirst%3DA.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BT.%26aulast%3DNishimura%26aufirst%3DN.%26aulast%3DPickrell%26aufirst%3DA.%2BJ.%26aulast%3DReed%26aufirst%3DA.%2BB.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DSiegmund%26aufirst%3DA.%2BC.%26aulast%3DTamayo%26aufirst%3DN.%2BA.%26aulast%3DTegley%26aufirst%3DC.%2BM.%26aulast%3DWalton%26aufirst%3DM.%2BC.%26aulast%3DWang%26aufirst%3DH.-L.%26aulast%3DWurz%26aufirst%3DR.%2BP.%26aulast%3DXue%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DK.%2BC.%26aulast%3DAchanta%26aufirst%3DP.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DCanon%26aufirst%3DJ.%26aulast%3DHollis%26aufirst%3DL.%2BS.%26aulast%3DMcCarter%26aufirst%3DJ.%2BD.%26aulast%3DMohr%26aufirst%3DC.%26aulast%3DRex%26aufirst%3DK.%26aulast%3DSaiki%26aufirst%3DA.%2BY.%26aulast%3DSan%2BMiguel%26aufirst%3DT.%26aulast%3DVolak%26aufirst%3DL.%2BP.%26aulast%3DWang%26aufirst%3DK.%2BH.%26aulast%3DWhittington%26aufirst%3DD.%2BA.%26aulast%3DZech%26aufirst%3DS.%2BG.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26aulast%3DCee%26aufirst%3DV.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520Covalent%2520Inhibitor%2520of%2520KRASG12C%2520%2528AMG%2520510%2529%2520for%2520the%2520Treatment%2520of%2520Solid%2520Tumors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D52%26epage%3D65%26doi%3D10.1021%2Facs.jmedchem.9b01180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marx, M. A.</span>; <span class="NLM_string-name">Baer, B. R.</span>; <span class="NLM_string-name">Ballard, J. K.</span>; <span class="NLM_string-name">Blake, J. F.</span>; <span class="NLM_string-name">Bouhana, K.</span>; <span class="NLM_string-name">Briere, D.</span>; <span class="NLM_string-name">Burgess, L. E.</span>; <span class="NLM_string-name">Burkhard, M.</span>; <span class="NLM_string-name">Chiang, H.</span>; <span class="NLM_string-name">Chicarelli, M. J.</span>; <span class="NLM_string-name">Christensen, J. G.</span>; <span class="NLM_string-name">Fischer, J. P.</span>; <span class="NLM_string-name">Hallin, J.</span>; <span class="NLM_string-name">Mejia, M. J.</span>; <span class="NLM_string-name">Olson, P.</span>; <span class="NLM_string-name">Savechenkov, P.</span>; <span class="NLM_string-name">Sudhakar, N.</span>; <span class="NLM_string-name">Tang, T. P.</span>; <span class="NLM_string-name">Vigers, G. P.</span>; <span class="NLM_string-name">Fell, J. B.</span></span> <span> </span><span class="NLM_article-title">Structure-based Drug Discovery of MRTX1257: A Selective, Covalent KRAS G12C Inhibitor with Oral Activity in Animal Models of Cancer</span>. MEDI-0244,  <i>Abstracts of Papers, 257th ACS National Meeting & Exposition, Orlando, FL, March 31–April 4, 2019</i>; <span class="NLM_publisher-name">American Chemical Society</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=M.+A.+Marx&author=B.+R.+Baer&author=J.+K.+Ballard&author=J.+F.+Blake&author=K.+Bouhana&author=D.+Briere&author=L.+E.+Burgess&author=M.+Burkhard&author=H.+Chiang&author=M.+J.+Chicarelli&author=J.+G.+Christensen&author=J.+P.+Fischer&author=J.+Hallin&author=M.+J.+Mejia&author=P.+Olson&author=P.+Savechenkov&author=N.+Sudhakar&author=T.+P.+Tang&author=G.+P.+Vigers&author=J.+B.+Fell&title=Structure-based+Drug+Discovery+of+MRTX1257%3A+A+Selective%2C+Covalent+KRAS+G12C+Inhibitor+with+Oral+Activity+in+Animal+Models+of+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DMarx%26aufirst%3DM.%2BA.%26atitle%3DStructure-based%2520Drug%2520Discovery%2520of%2520MRTX1257%253A%2520A%2520Selective%252C%2520Covalent%2520KRAS%2520G12C%2520Inhibitor%2520with%2520Oral%2520Activity%2520in%2520Animal%2520Models%2520of%2520Cancer%26jtitle%3DAbstracts%2520of%2520Papers%252C%2520257th%2520ACS%2520National%2520Meeting%2520%2526%2520Exposition%252C%2520Orlando%252C%2520FL%252C%2520March%252031%25E2%2580%2593April%25204%252C%25202019%26pub%3DAmerican%2520Chemical%2520Society%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span>; <span class="NLM_string-name">Feng, J.</span>; <span class="NLM_string-name">Wu, T.</span>; <span class="NLM_string-name">Ren, P.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Long, O. Y.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of KRAS G12C</span>.  <i>PCT Int. Appl.</i> <span class="NLM_patent">WO2015054572</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=L.+Li&author=J.+Feng&author=T.+Wu&author=P.+Ren&author=Y.+Liu&author=Y.+Liu&author=O.+Y.+Long&title=Inhibitors+of+KRAS+G12C"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26atitle%3DInhibitors%2520of%2520KRAS%2520G12C%26jtitle%3DPCT%2520Int.%2520Appl.%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feru, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gero, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrogio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catalano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westover, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Potent and Selective Covalent Quinazoline Inhibitors of KRAS G12C</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1005</span>– <span class="NLM_lpage">1016</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.06.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1016%2Fj.chembiol.2017.06.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=28781124" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1yltr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1005-1016&author=M.+Zengauthor=J.+Luauthor=L.+Liauthor=F.+Feruauthor=C.+Quanauthor=T.+W.+Geroauthor=S.+B.+Ficarroauthor=Y.+Xiongauthor=C.+Ambrogioauthor=R.+M.+Paranalauthor=M.+Catalanoauthor=J.+Shaoauthor=K.-K.+Wongauthor=J.+A.+Martoauthor=E.+S.+Fischerauthor=P.+A.+J%C3%A4nneauthor=D.+A.+Scottauthor=K.+D.+Westoverauthor=N.+S.+Gray&title=Potent+and+Selective+Covalent+Quinazoline+Inhibitors+of+KRAS+G12C&doi=10.1016%2Fj.chembiol.2017.06.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective covalent quinazoline inhibitors of kras g12c</span></div><div class="casAuthors">Zeng, Mei; Lu, Jia; Li, Lianbo; Feru, Frederic; Quan, Chunshan; Gero, Thomas W.; Ficarro, Scott B.; Xiong, Yuan; Ambrogio, Chiara; Paranal, Raymond M.; Catalano, Marco; Shao, Jay; Wong, Kwok-Kin; Marto, Jarrod A.; Fischer, Eric S.; Janne, Pasi A.; Scott, David A.; Westover, Kenneth D.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1005-1016.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Targeted covalent small mols. have shown promise for cancers driven by KRAS G12C.  Allosteric compds. that access an inducible pocket formed by movement of a dynamic structural element in KRAS, switch II, have been reported, but these compds. require further optimization to enable their advancement into clin. development.  We demonstrate that covalent quinazoline-based switch II pocket (SIIP) compds. effectively suppress GTP loading of KRAS G12C, MAPK phosphorylation, and the growth of cancer cells harboring G12C.  Notably we find that adding an amide substituent to the quinazoline scaffold allows addnl. interactions with KRAS G12C, and remarkably increases the labeling efficiency, potency, and selectivity of KRAS G12C inhibitors.  Structural studies using X-ray crystallog. reveal a new conformation of SIIP and key interactions made by substituents located at the quinazoline 2-, 4-, and 7-positions.  Optimized lead compds. in the quinazoline series selectively inhibit KRAS G12C-dependent signaling and cancer cell growth at sub-micromolar concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxY0-7NkfIeLVg90H21EOLACvtfcHk0ljtUyudN3OLNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1yltr3N&md5=e793972cb711c5a8c0db46d8288697af</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.06.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.06.017%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DFeru%26aufirst%3DF.%26aulast%3DQuan%26aufirst%3DC.%26aulast%3DGero%26aufirst%3DT.%2BW.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DAmbrogio%26aufirst%3DC.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DCatalano%26aufirst%3DM.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DWong%26aufirst%3DK.-K.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DScott%26aufirst%3DD.%2BA.%26aulast%3DWestover%26aufirst%3DK.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DPotent%2520and%2520Selective%2520Covalent%2520Quinazoline%2520Inhibitors%2520of%2520KRAS%2520G12C%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1005%26epage%3D1016%26doi%3D10.1016%2Fj.chembiol.2017.06.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodnarchuk, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span> <span> </span><span class="NLM_article-title">Water Sites, Networks, and Free Energies with Grand Canonical Monte Carlo</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">14930</span>– <span class="NLM_lpage">14943</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b07940</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b07940" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCjurbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=14930-14943&author=G.+A.+Rossauthor=M.+S.+Bodnarchukauthor=J.+W.+Essex&title=Water+Sites%2C+Networks%2C+and+Free+Energies+with+Grand+Canonical+Monte+Carlo&doi=10.1021%2Fjacs.5b07940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Water Sites, Networks, And Free Energies with Grand Canonical Monte Carlo</span></div><div class="casAuthors">Ross, Gregory A.; Bodnarchuk, Michael S.; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">14930-14943</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water mols. play integral roles in the formation of many protein-ligand complexes, and recent computational efforts have been focused on predicting the thermodn. properties of individual waters and how they may be exploited in rational drug design.  However, when water mols. form highly coupled hydrogen-bonding networks, there is, as yet, no method that can rigorously calc. the free energy to bind the entire network or assess the degree of cooperativity between waters.  In this work, we report theor. and methodol. developments to the grand canonical Monte Carlo simulation technique.  Central to our results is a rigorous equation that can be used to calc. efficiently the binding free energies of water networks of arbitrary size and complexity.  Using a single set of simulations, our methods can locate waters, est. their binding affinities, capture the cooperativity of the water network, and evaluate the hydration free energy of entire protein binding sites.  Our techniques have been applied to multiple test systems and compare favorably to thermodn. integration simulations and exptl. data.  The implications of these methods in drug design are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8mGD8Ul3D0rVg90H21EOLACvtfcHk0lj5exck5Nq-dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCjurbJ&md5=0040dc4201610620f2b64fc6c41998a9</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b07940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b07940%26sid%3Dliteratum%253Aachs%26aulast%3DRoss%26aufirst%3DG.%2BA.%26aulast%3DBodnarchuk%26aufirst%3DM.%2BS.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DWater%2520Sites%252C%2520Networks%252C%2520and%2520Free%2520Energies%2520with%2520Grand%2520Canonical%2520Monte%2520Carlo%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D14930%26epage%3D14943%26doi%3D10.1021%2Fjacs.5b07940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bodnarchuk, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span> <span> </span><span class="NLM_article-title">Strategies to Calculate Water Binding Free Energies in Protein-Ligand Complexes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1623</span>– <span class="NLM_lpage">1633</span>, <span class="refDoi"> DOI: 10.1021/ci400674k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400674k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltF2lu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1623-1633&author=M.+S.+Bodnarchukauthor=R.+Vinerauthor=J.+Michelauthor=J.+W.+Essex&title=Strategies+to+Calculate+Water+Binding+Free+Energies+in+Protein-Ligand+Complexes&doi=10.1021%2Fci400674k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies to Calculate Water Binding Free Energies in Protein-Ligand Complexes</span></div><div class="casAuthors">Bodnarchuk, Michael S.; Viner, Russell; Michel, Julien; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1623-1633</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water mols. are commonplace in protein binding pockets, where they can typically form a complex between the protein and a ligand or become displaced upon ligand binding.  As a result, it is often of great interest to establish both the binding free energy and location of such mols.  Several approaches to predicting the location and affinity of water mols. to proteins have been proposed and utilized in the literature, although it is often unclear which method should be used under what circumstances.  We report here a comparison between three such methodologies, Just Add Water Mols. (JAWS), Grand Canonical Monte Carlo (GCMC), and double-decoupling, in the hope of understanding the advantages and limitations of each method when applied to enclosed binding sites.  As a result, we have adapted the JAWS scoring procedure, allowing the binding free energies of strongly bound water mols. to be calcd. to a high degree of accuracy, requiring significantly less computational effort than more rigorous approaches.  The combination of JAWS and GCMC offers a route to a rapid scheme capable of both locating and scoring water mols. for rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVFExfkQzj-LVg90H21EOLACvtfcHk0lj5exck5Nq-dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltF2lu7g%253D&md5=c4174f31e4c0a3e3856ac4b68213342b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci400674k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400674k%26sid%3Dliteratum%253Aachs%26aulast%3DBodnarchuk%26aufirst%3DM.%2BS.%26aulast%3DViner%26aufirst%3DR.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DStrategies%2520to%2520Calculate%2520Water%2520Binding%2520Free%2520Energies%2520in%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1623%26epage%3D1633%26doi%3D10.1021%2Fci400674k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bodnarchuk, M. S.</span></span> <span> </span><span class="NLM_article-title">Water, Water, Everywhere··· It’s Time to Stop and Think</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1139</span>– <span class="NLM_lpage">1146</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1016%2Fj.drudis.2016.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=27210724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC28XoslWltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1139-1146&author=M.+S.+Bodnarchuk&title=Water%2C+Water%2C+Everywhere%C2%B7%C2%B7%C2%B7+It%E2%80%99s+Time+to+Stop+and+Think&doi=10.1016%2Fj.drudis.2016.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Water, water, everywhere... It's time to stop and think</span></div><div class="casAuthors">Bodnarchuk, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1139-1146</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Despite the numerous methods available for predicting the location and affinity of water mols., there is still a degree of scepticism and reluctance towards using such information within a drug discovery program.  Here, I review some of the most common and popular methods to assess whether these apparent concerns are justified.  I suggest that the field is approaching maturity and that some methods are capable of giving quant. predictions, which are confirmed exptl.  This suggests that water-placement methods should be used more widely to help direct chem. efforts, although more successful examples are required to help validate the techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxfKWV5PXTkrVg90H21EOLACvtfcHk0lj5exck5Nq-dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XoslWltb4%253D&md5=dc2a72fa764bb19cd0c0ca2921fc974a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DBodnarchuk%26aufirst%3DM.%2BS.%26atitle%3DWater%252C%2520Water%252C%2520Everywhere%25C2%25B7%25C2%25B7%25C2%25B7%2520It%25E2%2580%2599s%2520Time%2520to%2520Stop%2520and%2520Think%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D1139%26epage%3D1146%26doi%3D10.1016%2Fj.drudis.2016.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span>; <span class="NLM_string-name">Feng, J.</span>; <span class="NLM_string-name">Long, O. Y.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Wu, T.</span>; <span class="NLM_string-name">Ren, P.</span>; <span class="NLM_string-name">Liu, Y.</span></span>; <span> </span><span class="NLM_article-title">Substituted Quinazoline Compounds and Methods of Use Thereof</span>.  <i>PCT Int. Appl.</i> <span class="NLM_patent">WO2016164675</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=L.+Li&author=J.+Feng&author=O.+Y.+Long&author=Y.+Liu&author=T.+Wu&author=P.+Ren&author=Y.+Liu&title=Substituted+Quinazoline+Compounds+and+Methods+of+Use+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26atitle%3DSubstituted%2520Quinazoline%2520Compounds%2520and%2520Methods%2520of%2520Use%2520Thereof%26jtitle%3DPCT%2520Int.%2520Appl.%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Solca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanderson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Himmelsbach, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haaksma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adolf, G. R.</span></span> <span> </span><span class="NLM_article-title">Afatinib, an Irreversible Inhibitor of EGFR, HER2, and ErbB4</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">350</span>, <span class="refDoi"> DOI: 10.1124/jpet.112.197756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1124%2Fjpet.112.197756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=22888144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKktbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2012&pages=342-350&author=F.+Solcaauthor=G.+Dahlauthor=A.+Zoephelauthor=G.+Baderauthor=M.+Sandersonauthor=C.+Kleinauthor=O.+Kraemerauthor=F.+Himmelsbachauthor=E.+Haaksmaauthor=G.+R.+Adolf&title=Afatinib%2C+an+Irreversible+Inhibitor+of+EGFR%2C+HER2%2C+and+ErbB4&doi=10.1124%2Fjpet.112.197756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Target binding properties and cellular activity of afatinib (BIBW 2992), an irreversible ErbB family blocker</span></div><div class="casAuthors">Solca, Flavio; Dahl, Goeran; Zoephel, Andreas; Bader, Gerd; Sanderson, Michael; Klein, Christian; Kraemer, Oliver; Himmelsbach, Frank; Haaksma, Eric; Adolf, Guenther R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">342-350</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Deregulation of the ErbB (proto-oncogene B of the avian erythroblastosis virus AEV-H strain) receptor network is well recognized as an oncogenic driver in epithelial cancers.  Several targeted drugs have been developed, including antibodies and small-mol. kinase inhibitors, each of them characterized by distinct patterns of ErbB receptor interactions.  Understanding the precise pharmacol. properties of these compds. is important for optimal use in clin. practice.  Afatinib [BIBW 2992; N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[[(3S)-tetrahydro-3-furanyl]oxy]-6-quinazolinyl]-4-(dimethylamino)-2-butenamide] is an ATP-competitive anilinoquinazoline deriv. harboring a reactive acrylamide group.  It was designed to covalently bind and irreversibly block enzymically active ErbB receptor family members.  Here, we show by X-ray crystallog. the covalent binding of afatinib to wild-type epidermal growth factor receptor (EGFR) and by mass spectrometry the covalent interaction with EGFR, EGFRL858R/T790M, human epidermal growth factor receptor 2 (HER2), and ErbB-4.  Afatinib potently inhibits the enzymatic activity of ErbB-4 (EC50 = 1 nM) and the proliferation of cancer cell lines driven by multiple ErbB receptor aberrations at concns. below 100 nM.  N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[[(3S)-tetrahydro-3-furanyl]oxy]-6-quinazolinyl]-4-(dimethylamino)-2-butanamide (BI 37781), a close analog of afatinib lacking the acrylamide group and thus incapable of covalent bond formation, had similar potency on cells driven by EGFR or EGFRL858R, but less or no detectable activity on cells expressing EGFRL858R/T790M HER2 or ErbB-4.  These results stress the importance of the acrylamide group and show that afatinib differs from approved ErbB targeting agents by irreversibly inhibiting the kinase activity of all ErbB family members.  They provide a mechanistic rationale for the distinct pharmacol. features of this compd. and explain the clin. activity seen in some patients who are resistant to antibody or kinase inhibitor therapy because of secondary mutations or ErbB receptor "reprogramming.".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofXgQqaKXv-bVg90H21EOLACvtfcHk0lgZ3sXTDTpYWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFKktbzE&md5=4a57df32abe8101f287fc6251f63f92d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1124%2Fjpet.112.197756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.112.197756%26sid%3Dliteratum%253Aachs%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DDahl%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DSanderson%26aufirst%3DM.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DKraemer%26aufirst%3DO.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DHaaksma%26aufirst%3DE.%26aulast%3DAdolf%26aufirst%3DG.%2BR.%26atitle%3DAfatinib%252C%2520an%2520Irreversible%2520Inhibitor%2520of%2520EGFR%252C%2520HER2%252C%2520and%2520ErbB4%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2012%26volume%3D343%26spage%3D342%26epage%3D350%26doi%3D10.1124%2Fjpet.112.197756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finlay, M. R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butterworth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colclough, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, D. A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currie, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grist, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassall, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinowska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarland, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orme, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richmond, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrigley, G. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Potent and Selective EGFR Inhibitor (AZD9291) of Both Sensitizing and T790M Resistance Mutations That Spares the Wild Type Form of the Receptor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8249</span>– <span class="NLM_lpage">8267</span>, <span class="refDoi"> DOI: 10.1021/jm500973a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500973a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1KhurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8249-8267&author=M.+R.+V.+Finlayauthor=M.+Andertonauthor=S.+Ashtonauthor=P.+Ballardauthor=P.+A.+Bethelauthor=M.+R.+Boxauthor=R.+H.+Bradburyauthor=S.+J.+Brownauthor=S.+Butterworthauthor=A.+Campbellauthor=C.+Chorleyauthor=N.+Colcloughauthor=D.+A.+E.+Crossauthor=G.+S.+Currieauthor=M.+Gristauthor=L.+Hassallauthor=G.+B.+Hillauthor=D.+Jamesauthor=M.+Jamesauthor=P.+Kemmittauthor=T.+Klinowskaauthor=G.+Lamontauthor=S.+G.+Lamontauthor=N.+Martinauthor=H.+L.+McFarlandauthor=M.+J.+Mellorauthor=J.+P.+Ormeauthor=D.+Perkinsauthor=P.+Perkinsauthor=G.+Richmondauthor=P.+Smithauthor=R.+A.+Wardauthor=M.+J.+Waringauthor=D.+Whittakerauthor=S.+Wellsauthor=G.+L.+Wrigley&title=Discovery+of+a+Potent+and+Selective+EGFR+Inhibitor+%28AZD9291%29+of+Both+Sensitizing+and+T790M+Resistance+Mutations+That+Spares+the+Wild+Type+Form+of+the+Receptor&doi=10.1021%2Fjm500973a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective EGFR Inhibitor (AZD9291) of Both Sensitizing and T790M Resistance Mutations That Spares the Wild Type Form of the Receptor</span></div><div class="casAuthors">Finlay, M. Raymond V.; Anderton, Mark; Ashton, Susan; Ballard, Peter; Bethel, Paul A.; Box, Matthew R.; Bradbury, Robert H.; Brown, Simon J.; Butterworth, Sam; Campbell, Andrew; Chorley, Christopher; Colclough, Nicola; Cross, Darren A. E.; Currie, Gordon S.; Grist, Matthew; Hassall, Lorraine; Hill, George B.; James, Daniel; James, Michael; Kemmitt, Paul; Klinowska, Teresa; Lamont, Gillian; Lamont, Scott G.; Martin, Nathaniel; McFarland, Heather L.; Mellor, Martine J.; Orme, Jonathon P.; Perkins, David; Perkins, Paula; Richmond, Graham; Smith, Peter; Ward, Richard A.; Waring, Michael J.; Whittaker, David; Wells, Stuart; Wrigley, Gail L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8249-8267</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) inhibitors have been used clin. in the treatment of nonsmall-cell lung cancer (NSCLC) patients harboring sensitizing (or activating) mutations for a no. of years.  Despite encouraging clin. efficacy with these agents, in many patients resistance develops leading to disease progression.  In most cases, this resistance is in the form of the T790M mutation.  In addn., EGFR wild type receptor inhibition inherent with these agents can lead to dose limiting toxicities of rash and diarrhea.  The authors describe herein the evolution of an early, mutant selective lead to the clin. candidate AZD9291 I, an irreversible inhibitor of both EGFR sensitizing (EGFRm+) and T790M resistance mutations with selectivity over the wild type form of the receptor.  Following observations of significant tumor inhibition in preclin. models, the clin. candidate was administered clin. to patients with T790M pos. EGFR-TKI resistant NSCLC and early efficacy has been obsd., accompanied by an encouraging safety profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouPAWG9N0Ro7Vg90H21EOLACvtfcHk0lgZ3sXTDTpYWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1KhurrI&md5=db9db624bb4324c2e22b1ab12035263b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm500973a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500973a%26sid%3Dliteratum%253Aachs%26aulast%3DFinlay%26aufirst%3DM.%2BR.%2BV.%26aulast%3DAnderton%26aufirst%3DM.%26aulast%3DAshton%26aufirst%3DS.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DBethel%26aufirst%3DP.%2BA.%26aulast%3DBox%26aufirst%3DM.%2BR.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DBrown%26aufirst%3DS.%2BJ.%26aulast%3DButterworth%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DA.%26aulast%3DChorley%26aufirst%3DC.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DCross%26aufirst%3DD.%2BA.%2BE.%26aulast%3DCurrie%26aufirst%3DG.%2BS.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DHassall%26aufirst%3DL.%26aulast%3DHill%26aufirst%3DG.%2BB.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DJames%26aufirst%3DM.%26aulast%3DKemmitt%26aufirst%3DP.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DLamont%26aufirst%3DG.%26aulast%3DLamont%26aufirst%3DS.%2BG.%26aulast%3DMartin%26aufirst%3DN.%26aulast%3DMcFarland%26aufirst%3DH.%2BL.%26aulast%3DMellor%26aufirst%3DM.%2BJ.%26aulast%3DOrme%26aufirst%3DJ.%2BP.%26aulast%3DPerkins%26aufirst%3DD.%26aulast%3DPerkins%26aufirst%3DP.%26aulast%3DRichmond%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWells%26aufirst%3DS.%26aulast%3DWrigley%26aufirst%3DG.%2BL.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520and%2520Selective%2520EGFR%2520Inhibitor%2520%2528AZD9291%2529%2520of%2520Both%2520Sensitizing%2520and%2520T790M%2520Resistance%2520Mutations%2520That%2520Spares%2520the%2520Wild%2520Type%2520Form%2520of%2520the%2520Receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8249%26epage%3D8267%26doi%3D10.1021%2Fjm500973a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, M.</span></span> <span> </span><span class="NLM_article-title">The Role of Extrahepatic Metabolism in the Pharmacokinetics of the Targeted Covalent Inhibitors Afatinib, Ibrutinib, and Neratinib</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.1124/dmd.114.061424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1124%2Fdmd.114.061424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=25504185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1OmsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=375-384&author=Y.+Shibataauthor=M.+Chiba&title=The+Role+of+Extrahepatic+Metabolism+in+the+Pharmacokinetics+of+the+Targeted+Covalent+Inhibitors+Afatinib%2C+Ibrutinib%2C+and+Neratinib&doi=10.1124%2Fdmd.114.061424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The role of extrahepatic metabolism in the pharmacokinetics of the targeted covalent inhibitors afatinib, ibrutinib, and neratinib</span></div><div class="casAuthors">Shibata, Yoshihiro; Chiba, Masato</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">375-384, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Despite the fact that much progress has been made recently in the development of targeted covalent inhibitors (TCIs), their pharmacokinetics (PK) have not been well characterized in the light of extrahepatic clearance (CLextH) by glutathione (GSH)/glutathione S-transferase (GST)-dependent conjugation attributable to the unique electrophilic structure (e.g., acrylamide moiety) of TCI compds.  In the present study, CLextH values were examd. in rat, dog, and monkey to predict the contribution of CLextH to the PK of the TCIs afatinib, ibrutinib, and neratinib in humans.  Afatinib and neratinib both underwent extensive conjugation with GSH in buffer and cytosol fractions of liver and kidney, whereas ibrutinib showed much lower reactivity/susceptibility to GSH/GST-dependent conjugation.  The CLextH in each species was calcd. from the difference between obsd. total body clearance and predicted hepatic clearance (CLH) in cryopreserved hepatocytes suspended in 100% serum of the corresponding species.  The power-based simple allometry relating the CLextH for the unbound compd. to animal body wt. was applicable across species for afatinib and neratinib (R2 ≥ 0.9) but not for ibrutinib (R2 = 0.04).  The predicted AUC after oral administration of afatinib and neratinib agreed reasonably closely with reported values in phase I dose-escalation studies.  Comparisons of CLextH and CLH predicted that CLextH largely detd. the PK of afatinib (>90% as a proportion of total body clearance) and neratinib (∼34%) in humans.  The present method can serve as one of the tools for the optimization of PK in humans at the discovery stage for the development of TCI candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquWC6d6e6DU7Vg90H21EOLACvtfcHk0lgZ3sXTDTpYWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1OmsrY%253D&md5=eaac67ce7cea5f2432c5144922ba3dd2</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1124%2Fdmd.114.061424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.114.061424%26sid%3Dliteratum%253Aachs%26aulast%3DShibata%26aufirst%3DY.%26aulast%3DChiba%26aufirst%3DM.%26atitle%3DThe%2520Role%2520of%2520Extrahepatic%2520Metabolism%2520in%2520the%2520Pharmacokinetics%2520of%2520the%2520Targeted%2520Covalent%2520Inhibitors%2520Afatinib%252C%2520Ibrutinib%252C%2520and%2520Neratinib%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2015%26volume%3D43%26spage%3D375%26epage%3D384%26doi%3D10.1124%2Fdmd.114.061424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allocati, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masulli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Ilio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federici, L.</span></span> <span> </span><span class="NLM_article-title">Glutathione Transferases: Substrates, Inhibitors and Pro-drugs in Cancer and Neurodegenerative Diseases</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">8</span>, <span class="refDoi"> DOI: 10.1038/s41389-017-0025-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1038%2Fs41389-017-0025-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=29362397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A280%3ADC%252BC1MvjsFGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=8&author=N.+Allocatiauthor=M.+Masulliauthor=C.+Di+Ilioauthor=L.+Federici&title=Glutathione+Transferases%3A+Substrates%2C+Inhibitors+and+Pro-drugs+in+Cancer+and+Neurodegenerative+Diseases&doi=10.1038%2Fs41389-017-0025-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione transferases: substrates, inihibitors and pro-drugs in cancer and neurodegenerative diseases</span></div><div class="casAuthors">Allocati Nerino; Masulli Michele; Di Ilio Carmine; Federici Luca; Federici Luca</div><div class="citationInfo"><span class="NLM_cas:title">Oncogenesis</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8</span>
        ISSN:<span class="NLM_cas:issn">2157-9024</span>.
    </div><div class="casAbstract">Glutathione transferase classical GSH conjugation activity plays a critical role in cellular detoxification against xenobiotics and noxious compounds as well as against oxidative stress.  However, this feature is also exploited by cancer cells to acquire drug resistance and improve their survival.  As a result, various members of the family were found overexpressed in a number of different cancers.  Moreover several GST polymorphisms, ranging from null phenotypes to point mutations, were detected in members of the family and found to correlate with the onset of neuro-degenerative diseases.  In the last decades, a great deal of research aimed at clarifying the role played by GSTs in drug resistance, at developing inhibitors to counteract this activity but also at exploiting GSTs for prodrugs specific activation in cancer cells.  Here we summarize some of the most important achievements reached in this lively area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRVakKpC8MvO5VVjGoY74BafW6udTcc2eaFE0w4yQ8TTbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvjsFGmtA%253D%253D&md5=bdaead4dda5737e83b117829a0d0ddee</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fs41389-017-0025-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41389-017-0025-3%26sid%3Dliteratum%253Aachs%26aulast%3DAllocati%26aufirst%3DN.%26aulast%3DMasulli%26aufirst%3DM.%26aulast%3DDi%2BIlio%26aufirst%3DC.%26aulast%3DFederici%26aufirst%3DL.%26atitle%3DGlutathione%2520Transferases%253A%2520Substrates%252C%2520Inhibitors%2520and%2520Pro-drugs%2520in%2520Cancer%2520and%2520Neurodegenerative%2520Diseases%26jtitle%3DOncogenesis%26date%3D2018%26volume%3D7%26spage%3D8%26doi%3D10.1038%2Fs41389-017-0025-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collisson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadanandam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibb, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinkle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakkula, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feiler, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olshen, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danenberg, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempero, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellman, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. W.</span></span> <span> </span><span class="NLM_article-title">Subtypes of Pancreatic Ductal Adenocarcinoma and their Differing Responses to Therapy</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">500</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1038/nm.2344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1038%2Fnm.2344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=21460848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFahtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=500-503&author=E.+A.+Collissonauthor=A.+Sadanandamauthor=P.+Olsonauthor=W.+J.+Gibbauthor=M.+Truittauthor=S.+Guauthor=J.+Coocauthor=J.+Weinkleauthor=G.+E.+Kimauthor=L.+Jakkulaauthor=H.+S.+Feilerauthor=A.+H.+Koauthor=A.+B.+Olshenauthor=K.+L.+Danenbergauthor=M.+A.+Temperoauthor=P.+T.+Spellmanauthor=D.+Hanahanauthor=J.+W.+Gray&title=Subtypes+of+Pancreatic+Ductal+Adenocarcinoma+and+their+Differing+Responses+to+Therapy&doi=10.1038%2Fnm.2344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Subtypes of pancreatic ductal adenocarcinoma and their differing responses to therapy</span></div><div class="casAuthors">Collisson, Eric A.; Sadanandam, Anguraj; Olson, Peter; Gibb, William J.; Truitt, Morgan; Gu, Shenda; Cooc, Janine; Weinkle, Jennifer; Kim, Grace E.; Jakkula, Lakshmi; Feiler, Heidi S.; Ko, Andrew H.; Olshen, Adam B.; Danenberg, Kathleen L.; Tempero, Margaret A.; Spellman, Paul T.; Hanahan, Douglas; Gray, Joe W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">500-503</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDA) is a lethal disease.  Overall survival is typically 6 mo from diagnosis.  Numerous phase 3 trials of agents effective in other malignancies have failed to benefit unselected PDA populations, although patients do occasionally respond.  Studies in other solid tumors have shown that heterogeneity in response is detd., in part, by mol. differences between tumors.  Furthermore, treatment outcomes are improved by targeting drugs to tumor subtypes in which they are selectively effective, with breast and lung cancers providing recent examples.  Identification of PDA mol. subtypes has been frustrated by a paucity of tumor specimens available for study.  We have overcome this problem by combined anal. of transcriptional profiles of primary PDA samples from several studies, along with human and mouse PDA cell lines.  We define three PDA subtypes: classical, quasimesenchymal and exocrine-like, and we present evidence for clin. outcome and therapeutic response differences between them.  We further define gene signatures for these subtypes that may have utility in stratifying patients for treatment and present preclin. model systems that may be used to identify new subtype specific therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-1m347gugF7Vg90H21EOLACvtfcHk0li07zexniXT9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFahtrY%253D&md5=fe6468e218cf05bb79e614a85ef01b4a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnm.2344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2344%26sid%3Dliteratum%253Aachs%26aulast%3DCollisson%26aufirst%3DE.%2BA.%26aulast%3DSadanandam%26aufirst%3DA.%26aulast%3DOlson%26aufirst%3DP.%26aulast%3DGibb%26aufirst%3DW.%2BJ.%26aulast%3DTruitt%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DS.%26aulast%3DCooc%26aufirst%3DJ.%26aulast%3DWeinkle%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DG.%2BE.%26aulast%3DJakkula%26aufirst%3DL.%26aulast%3DFeiler%26aufirst%3DH.%2BS.%26aulast%3DKo%26aufirst%3DA.%2BH.%26aulast%3DOlshen%26aufirst%3DA.%2BB.%26aulast%3DDanenberg%26aufirst%3DK.%2BL.%26aulast%3DTempero%26aufirst%3DM.%2BA.%26aulast%3DSpellman%26aufirst%3DP.%2BT.%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DGray%26aufirst%3DJ.%2BW.%26atitle%3DSubtypes%2520of%2520Pancreatic%2520Ductal%2520Adenocarcinoma%2520and%2520their%2520Differing%2520Responses%2520to%2520Therapy%26jtitle%3DNat.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D500%26epage%3D503%26doi%3D10.1038%2Fnm.2344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greninger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhodes, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koopman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Violette, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardeesy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span> <span> </span><span class="NLM_article-title">A Gene Expression Signature Associated with ″K-Ras Addiction″ Reveals Regulators of EMT and Tumor Cell Survival</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">489</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2009.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1016%2Fj.ccr.2009.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=19477428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BD1MXntlegtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=489-500&author=A.+Singhauthor=P.+Greningerauthor=D.+Rhodesauthor=L.+Koopmanauthor=S.+Violetteauthor=N.+Bardeesyauthor=J.+Settleman&title=A+Gene+Expression+Signature+Associated+with+%E2%80%B3K-Ras+Addiction%E2%80%B3+Reveals+Regulators+of+EMT+and+Tumor+Cell+Survival&doi=10.1016%2Fj.ccr.2009.03.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A gene expression signature associated with "K-ras addiction" reveals regulators of EMT and tumor cell survival</span></div><div class="casAuthors">Singh, Anurag; Greninger, Patricia; Rhodes, Daniel; Koopman, Louise; Violette, Sheila; Bardeesy, Nabeel; Settleman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">489-500</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">K-ras mutations occur frequently in epithelial cancers.  Using short hairpin RNAs to deplete K-Ras in lung and pancreatic cancer cell lines harboring K-ras mutations, two classes were identified-lines that do or do not require K-Ras to maintain viability.  Comparing these two classes of cancer cells revealed a gene expression signature in K-Ras-dependent cells, assocd. with a well-differentiated epithelial phenotype, which was also seen in primary tumors.  Several of these genes encode pharmacol. tractable proteins, such as Syk and Ron kinases and integrin β6, depletion of which induces epithelial-mesenchymal transformation (EMT) and apoptosis specifically in K-Ras-dependent cells.  These findings indicate that epithelial differentiation and tumor cell viability are assocd., and that EMT regulators in "K-Ras-addicted" cancers represent candidate therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFfjxR9oP1drVg90H21EOLACvtfcHk0li07zexniXT9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXntlegtb0%253D&md5=ebe63c5a069664f70eb88b285f1e3f6a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DGreninger%26aufirst%3DP.%26aulast%3DRhodes%26aufirst%3DD.%26aulast%3DKoopman%26aufirst%3DL.%26aulast%3DViolette%26aufirst%3DS.%26aulast%3DBardeesy%26aufirst%3DN.%26aulast%3DSettleman%26aufirst%3DJ.%26atitle%3DA%2520Gene%2520Expression%2520Signature%2520Associated%2520with%2520%25E2%2580%25B3K-Ras%2520Addiction%25E2%2580%25B3%2520Reveals%2520Regulators%2520of%2520EMT%2520and%2520Tumor%2520Cell%2520Survival%26jtitle%3DCancer%2520Cell%26date%3D2009%26volume%3D15%26spage%3D489%26epage%3D500%26doi%3D10.1016%2Fj.ccr.2009.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratilas, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Runswick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chresta, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormack, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockerill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beran, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassidy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggerty, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellison, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dering, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonazzi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravishankar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solit, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. D.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Pathway Signatures Predict MEK Addiction and Response to Selumetinib (AZD6244)</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">2264</span>– <span class="NLM_lpage">2273</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-09-1577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1158%2F0008-5472.CAN-09-1577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=20215513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFygt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=2264-2273&author=J.+R.+Dryauthor=S.+Paveyauthor=C.+A.+Pratilasauthor=C.+Harbronauthor=S.+Runswickauthor=D.+Hodgsonauthor=C.+Chrestaauthor=R.+McCormackauthor=N.+Byrneauthor=M.+Cockerillauthor=A.+Grahamauthor=G.+Beranauthor=A.+Cassidyauthor=C.+Haggertyauthor=H.+Brownauthor=G.+Ellisonauthor=J.+Deringauthor=B.+S.+Taylorauthor=M.+Starkauthor=V.+Bonazziauthor=S.+Ravishankarauthor=L.+Packerauthor=F.+Xingauthor=D.+B.+Solitauthor=R.+S.+Finnauthor=N.+Rosenauthor=N.+K.+Haywardauthor=T.+Frenchauthor=P.+D.+Smith&title=Transcriptional+Pathway+Signatures+Predict+MEK+Addiction+and+Response+to+Selumetinib+%28AZD6244%29&doi=10.1158%2F0008-5472.CAN-09-1577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional Pathway Signatures Predict MEK Addiction and Response to Selumetinib (AZD6244)</span></div><div class="casAuthors">Dry, Jonathan R.; Pavey, Sandra; Pratilas, Christine A.; Harbron, Chris; Runswick, Sarah; Hodgson, Darren; Chresta, Christine; McCormack, Rose; Byrne, Natalie; Cockerill, Mark; Graham, Alexander; Beran, Garry; Cassidy, Andrew; Haggerty, Carolyn; Brown, Helen; Ellison, Gillian; Dering, Judy; Taylor, Barry S.; Stark, Mitchell; Bonazzi, Vanessa; Ravishankar, Sugandha; Packer, Leisl; Xing, Feng; Solit, David B.; Finn, Richard S.; Rosen, Neal; Hayward, Nicholas K.; French, Tim; Smith, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2264-2273</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Selumetinib (AZD6244, ARRY-1α642886) is a selective, non-ATP-competitive inhibitor of mitogen-activated protein/extracellular signal-regulated kinase kinase (MEK)-1/2.  The range of antitumor activity seen preclinically and in patients highlights the importance of identifying determinants of response to this drug.  In large tumor cell panels of diverse lineage, we show that MEK inhibitor response does not have an abs. correlation with mutational or phospho-protein markers of BRAF/MEK, RAS, or phosphoinositide 3-kinase (PI3K) activity.  We aimed to enhance predictivity by measuring pathway output through coregulated gene networks displaying differential mRNA expression exclusive to resistant cell subsets and correlated to mutational or dynamic pathway activity.  We discovered an 18-gene signature enabling measurement of MEK functional output independent of tumor genotype.  Where the MEK pathway is activated but the cells remain resistant to selumetinib, we identified a 13-gene signature that implicates the existence of compensatory signaling from RAS effectors other than PI3K.  The ability of these signatures to stratify samples according to functional activation of MEK and/or selumetinib sensitivity was shown in multiple independent melanoma, colon, breast, and lung tumor cell lines and in xenograft models.  Furthermore, we were able to measure these signatures in fixed archival melanoma tumor samples using a single RT-qPCR-based test and found intergene correlations and assocns. with genetic markers of pathway activity to be preserved.  These signatures offer useful tools for the study of MEK biol. and clin. application of MEK inhibitors, and the novel approaches taken may benefit other targeted therapies.  Cancer Res; 70(6); 2264-73.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4oaXtcCNnKrVg90H21EOLACvtfcHk0lgw-1NqejBkzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFygt7o%253D&md5=fe1714e5398f21b3ae4a36863e598a9f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-09-1577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-09-1577%26sid%3Dliteratum%253Aachs%26aulast%3DDry%26aufirst%3DJ.%2BR.%26aulast%3DPavey%26aufirst%3DS.%26aulast%3DPratilas%26aufirst%3DC.%2BA.%26aulast%3DHarbron%26aufirst%3DC.%26aulast%3DRunswick%26aufirst%3DS.%26aulast%3DHodgson%26aufirst%3DD.%26aulast%3DChresta%26aufirst%3DC.%26aulast%3DMcCormack%26aufirst%3DR.%26aulast%3DByrne%26aufirst%3DN.%26aulast%3DCockerill%26aufirst%3DM.%26aulast%3DGraham%26aufirst%3DA.%26aulast%3DBeran%26aufirst%3DG.%26aulast%3DCassidy%26aufirst%3DA.%26aulast%3DHaggerty%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DH.%26aulast%3DEllison%26aufirst%3DG.%26aulast%3DDering%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DB.%2BS.%26aulast%3DStark%26aufirst%3DM.%26aulast%3DBonazzi%26aufirst%3DV.%26aulast%3DRavishankar%26aufirst%3DS.%26aulast%3DPacker%26aufirst%3DL.%26aulast%3DXing%26aufirst%3DF.%26aulast%3DSolit%26aufirst%3DD.%2BB.%26aulast%3DFinn%26aufirst%3DR.%2BS.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DHayward%26aufirst%3DN.%2BK.%26aulast%3DFrench%26aufirst%3DT.%26aulast%3DSmith%26aufirst%3DP.%2BD.%26atitle%3DTranscriptional%2520Pathway%2520Signatures%2520Predict%2520MEK%2520Addiction%2520and%2520Response%2520to%2520Selumetinib%2520%2528AZD6244%2529%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D2264%26epage%3D2273%26doi%3D10.1158%2F0008-5472.CAN-09-1577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ross, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellston, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staniszewska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whalley, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckett, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monia, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakey, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macleod, A. R.</span></span> <span> </span><span class="NLM_article-title">Targeting KRAS-dependent Tumors with AZD4785, a High-affinity Therapeutic Antisense Oligonucleotide Inhibitor of KRAS</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">eaal5253</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.aal5253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1126%2Fscitranslmed.aal5253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=28615361" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=eaal5253&author=S.+J.+Rossauthor=A.+S.+Revenkoauthor=L.+L.+Hansonauthor=R.+Ellstonauthor=A.+Staniszewskaauthor=N.+Whalleyauthor=S.+K.+Pandeyauthor=M.+Revillauthor=C.+Rooneyauthor=L.+K.+Buckettauthor=S.+K.+Kleinauthor=K.+Hudsonauthor=B.+P.+Moniaauthor=M.+Zindaauthor=D.+C.+Blakeyauthor=P.+D.+Lyneauthor=A.+R.+Macleod&title=Targeting+KRAS-dependent+Tumors+with+AZD4785%2C+a+High-affinity+Therapeutic+Antisense+Oligonucleotide+Inhibitor+of+KRAS&doi=10.1126%2Fscitranslmed.aal5253"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aal5253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aal5253%26sid%3Dliteratum%253Aachs%26aulast%3DRoss%26aufirst%3DS.%2BJ.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DHanson%26aufirst%3DL.%2BL.%26aulast%3DEllston%26aufirst%3DR.%26aulast%3DStaniszewska%26aufirst%3DA.%26aulast%3DWhalley%26aufirst%3DN.%26aulast%3DPandey%26aufirst%3DS.%2BK.%26aulast%3DRevill%26aufirst%3DM.%26aulast%3DRooney%26aufirst%3DC.%26aulast%3DBuckett%26aufirst%3DL.%2BK.%26aulast%3DKlein%26aufirst%3DS.%2BK.%26aulast%3DHudson%26aufirst%3DK.%26aulast%3DMonia%26aufirst%3DB.%2BP.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DBlakey%26aufirst%3DD.%2BC.%26aulast%3DLyne%26aufirst%3DP.%2BD.%26aulast%3DMacleod%26aufirst%3DA.%2BR.%26atitle%3DTargeting%2520KRAS-dependent%2520Tumors%2520with%2520AZD4785%252C%2520a%2520High-affinity%2520Therapeutic%2520Antisense%2520Oligonucleotide%2520Inhibitor%2520of%2520KRAS%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2017%26volume%3D9%26spage%3Deaal5253%26doi%3D10.1126%2Fscitranslmed.aal5253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 11 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Hao Chen, Jeff B. Smaill, Tongzheng Liu, Ke Ding, <span class="NLM_string-name hlFld-ContribAuthor">Xiaoyun Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Inhibitors Directly Targeting KRAS as Anticancer Therapeutics. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 14404-14424. <a href="https://doi.org/10.1021/acs.jmedchem.0c01312" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01312</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01312%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-Molecule%252BInhibitors%252BDirectly%252BTargeting%252BKRAS%252Bas%252BAnticancer%252BTherapeutics%26aulast%3DChen%26aufirst%3DHao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D28072020%26date%3D23112020%26volume%3D63%26issue%3D23%26spage%3D14404%26epage%3D14424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Keisuke Tokunaga, Mami Sato, Keiko Kuwata, Chizuru Miura, Hirokazu Fuchida, Naoya Matsunaga, Satoru Koyanagi, Shigehiro Ohdo, Naoya Shindo, <span class="NLM_string-name hlFld-ContribAuthor">Akio Ojida</span>. </span><span class="cited-content_cbyCitation_article-title">Bicyclobutane Carboxylic Amide as a Cysteine-Directed Strained Electrophile for Selective Targeting of Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2020,</strong> <em>142 </em>
                                    (43)
                                     , 18522-18531. <a href="https://doi.org/10.1021/jacs.0c07490" title="DOI URL">https://doi.org/10.1021/jacs.0c07490</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c07490&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c07490%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DBicyclobutane%252BCarboxylic%252BAmide%252Bas%252Ba%252BCysteine-Directed%252BStrained%252BElectrophile%252Bfor%252BSelective%252BTargeting%252Bof%252BProteins%26aulast%3DTokunaga%26aufirst%3DKeisuke%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D12072020%26date%3D13102020%26volume%3D142%26issue%3D43%26spage%3D18522%26epage%3D18531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kirsten McAulay, Emily A. Hoyt, Morgan Thomas, Marianne Schimpl, Michael S. Bodnarchuk, Hilary J. Lewis, Derek Barratt, Deepa Bhavsar, David M. Robinson, Michael J. Deery, Derek J. Ogg, Gonçalo J. L. Bernardes, Richard A. Ward, Michael J. Waring, <span class="NLM_string-name hlFld-ContribAuthor">Jason G. Kettle</span>. </span><span class="cited-content_cbyCitation_article-title">Alkynyl Benzoxazines and Dihydroquinazolines as Cysteine Targeting Covalent Warheads and Their Application in Identification of Selective Irreversible Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2020,</strong> <em>142 </em>
                                    (23)
                                     , 10358-10372. <a href="https://doi.org/10.1021/jacs.9b13391" title="DOI URL">https://doi.org/10.1021/jacs.9b13391</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.9b13391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.9b13391%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DAlkynyl%252BBenzoxazines%252Band%252BDihydroquinazolines%252Bas%252BCysteine%252BTargeting%252BCovalent%252BWarheads%252Band%252BTheir%252BApplication%252Bin%252BIdentification%252Bof%252BSelective%252BIrreversible%252BKinase%252BInhibitors%26aulast%3DMcAulay%26aufirst%3DKirsten%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D12122019%26date%3D02062020%26date%3D15052020%26volume%3D142%26issue%3D23%26spage%3D10358%26epage%3D10372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Severin  Thompson</span>, <span class="hlFld-ContribAuthor ">Andreas  Buckl</span>, <span class="hlFld-ContribAuthor ">Alexander G.  Dossetter</span>, <span class="hlFld-ContribAuthor ">Ed  Griffen</span>, <span class="hlFld-ContribAuthor ">Adrian  Gill</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecule Son of Sevenless 1 (SOS1) inhibitors: a review of the patent literature. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2021,</strong> <em> </em><a href="https://doi.org/10.1080/13543776.2021.1952984" title="DOI URL">https://doi.org/10.1080/13543776.2021.1952984</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2021.1952984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2021.1952984%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DSmall%252Bmolecule%252BSon%252Bof%252BSevenless%252B1%252B%252528SOS1%252529%252Binhibitors%25253A%252Ba%252Breview%252Bof%252Bthe%252Bpatent%252Bliterature%26aulast%3DThompson%26aufirst%3DSeverin%26date%3D2021%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel A.  Erlanson</span>, <span class="hlFld-ContribAuthor ">Kevin R.  Webster</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting mutant KRAS. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>62 </em>, 101-108. <a href="https://doi.org/10.1016/j.cbpa.2021.02.010" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.02.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.02.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.02.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DTargeting%252Bmutant%252BKRAS%26aulast%3DErlanson%26aufirst%3DDaniel%2BA.%26date%3D2021%26volume%3D62%26spage%3D101%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael S.  Bodnarchuk</span>, <span class="hlFld-ContribAuthor ">Doyle J.  Cassar</span>, <span class="hlFld-ContribAuthor ">Jason G.  Kettle</span>, <span class="hlFld-ContribAuthor ">Graeme  Robb</span>, <span class="hlFld-ContribAuthor ">Richard A.  Ward</span>. </span><span class="cited-content_cbyCitation_article-title">Drugging the undruggable: a computational chemist's view of KRAS
              G12C. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>12 </em>
                                    (4)
                                     , 609-614. <a href="https://doi.org/10.1039/D1MD00055A" title="DOI URL">https://doi.org/10.1039/D1MD00055A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1MD00055A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1MD00055A%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DDrugging%252Bthe%252Bundruggable%25253A%252Ba%252Bcomputational%252Bchemist%252527s%252Bview%252Bof%252BKRAS%252BG12C%26aulast%3DBodnarchuk%26aufirst%3DMichael%2BS.%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D4%26spage%3D609%26epage%3D614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ruth  Nussinov</span>, <span class="hlFld-ContribAuthor ">Hyunbum  Jang</span>, <span class="hlFld-ContribAuthor ">Attila  Gursoy</span>, <span class="hlFld-ContribAuthor ">Ozlem  Keskin</span>, <span class="hlFld-ContribAuthor ">Vadim  Gaponenko</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of Nonfunctional Ras. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (2)
                                     , 121-133. <a href="https://doi.org/10.1016/j.chembiol.2020.12.012" title="DOI URL">https://doi.org/10.1016/j.chembiol.2020.12.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2020.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2020.12.012%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DInhibition%252Bof%252BNonfunctional%252BRas%26aulast%3DNussinov%26aufirst%3DRuth%26date%3D2021%26volume%3D28%26issue%3D2%26spage%3D121%26epage%3D133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">György M.  Keserű</span>, <span class="hlFld-ContribAuthor ">Daniel A.  Erlanson</span>. </span><span class="cited-content_cbyCitation_article-title">The future of covalent inhibition. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 267-284. <a href="https://doi.org/10.1016/bs.armc.2020.10.003" title="DOI URL">https://doi.org/10.1016/bs.armc.2020.10.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2020.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2020.10.003%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252Bfuture%252Bof%252Bcovalent%252Binhibition%26aulast%3DKeser%25C5%25B1%26aufirst%3DGy%25C3%25B6rgy%2BM.%26date%3D2021%26spage%3D267%26epage%3D284%26pub%3DElsevier%26atitle%3DThe%252BDesign%252Bof%252BCovalent-Based%252BInhibitors%26date%3D2021%26volume%3D56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Atanu  Chakraborty</span>. </span><span class="cited-content_cbyCitation_article-title">KRASG12C inhibitor: combing for combination. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Society Transactions</span><span> <strong>2020,</strong> <em>48 </em>
                                    (6)
                                     , 2691-2701. <a href="https://doi.org/10.1042/BST20200473" title="DOI URL">https://doi.org/10.1042/BST20200473</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BST20200473&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBST20200473%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Society%2520Transactions%26atitle%3DKRASG12C%252Binhibitor%25253A%252Bcombing%252Bfor%252Bcombination%26aulast%3DChakraborty%26aufirst%3DAtanu%26date%3D2020%26date%3D2020%26volume%3D48%26issue%3D6%26spage%3D2691%26epage%3D2701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zoltán  Orgován</span>, <span class="hlFld-ContribAuthor ">György M.  Keserű</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecule inhibitors of RAS proteins with oncogenic mutations. </span><span class="cited-content_cbyCitation_journal-name">Cancer and Metastasis Reviews</span><span> <strong>2020,</strong> <em>39 </em>
                                    (4)
                                     , 1107-1126. <a href="https://doi.org/10.1007/s10555-020-09911-9" title="DOI URL">https://doi.org/10.1007/s10555-020-09911-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10555-020-09911-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10555-020-09911-9%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520and%2520Metastasis%2520Reviews%26atitle%3DSmall%252Bmolecule%252Binhibitors%252Bof%252BRAS%252Bproteins%252Bwith%252Boncogenic%252Bmutations%26aulast%3DOrgov%25C3%25A1n%26aufirst%3DZolt%25C3%25A1n%26date%3D2020%26date%3D2020%26volume%3D39%26issue%3D4%26spage%3D1107%26epage%3D1126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael  Gütschow</span>, <span class="hlFld-ContribAuthor ">Jean Jacques  Vanden Eynde</span>, <span class="hlFld-ContribAuthor ">Josef  Jampilek</span>, <span class="hlFld-ContribAuthor ">CongBao  Kang</span>, <span class="hlFld-ContribAuthor ">Arduino A.  Mangoni</span>, <span class="hlFld-ContribAuthor ">Paola  Fossa</span>, <span class="hlFld-ContribAuthor ">Rafik  Karaman</span>, <span class="hlFld-ContribAuthor ">Andrea  Trabocchi</span>, <span class="hlFld-ContribAuthor ">Peter J. H.  Scott</span>, <span class="hlFld-ContribAuthor ">Jóhannes  Reynisson</span>, <span class="hlFld-ContribAuthor ">Simona  Rapposelli</span>, <span class="hlFld-ContribAuthor ">Stefania  Galdiero</span>, <span class="hlFld-ContribAuthor ">Jean-Yves  Winum</span>, <span class="hlFld-ContribAuthor ">Chiara  Brullo</span>, <span class="hlFld-ContribAuthor ">Katalin  Prokai-Tatrai</span>, <span class="hlFld-ContribAuthor ">Arun K.  Sharma</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Yasu-Taka  Azuma</span>, <span class="hlFld-ContribAuthor ">Laura  Cerchia</span>, <span class="hlFld-ContribAuthor ">Mariana  Spetea</span>, <span class="hlFld-ContribAuthor ">Giangiacomo  Torri</span>, <span class="hlFld-ContribAuthor ">Simona  Collina</span>, <span class="hlFld-ContribAuthor ">Athina  Geronikaki</span>, <span class="hlFld-ContribAuthor ">Alfonso T.  García-Sosa</span>, <span class="hlFld-ContribAuthor ">M. Helena  Vasconcelos</span>, <span class="hlFld-ContribAuthor ">Maria Emília  Sousa</span>, <span class="hlFld-ContribAuthor ">Ivan  Kosalec</span>, <span class="hlFld-ContribAuthor ">Tiziano  Tuccinardi</span>, <span class="hlFld-ContribAuthor ">Iola F.  Duarte</span>, <span class="hlFld-ContribAuthor ">Jorge A. R.  Salvador</span>, <span class="hlFld-ContribAuthor ">Massimo  Bertinaria</span>, <span class="hlFld-ContribAuthor ">Maurizio  Pellecchia</span>, <span class="hlFld-ContribAuthor ">Jussara  Amato</span>, <span class="hlFld-ContribAuthor ">Giulio  Rastelli</span>, <span class="hlFld-ContribAuthor ">Paula A. C.  Gomes</span>, <span class="hlFld-ContribAuthor ">Rita C.  Guedes</span>, <span class="hlFld-ContribAuthor ">Jean-Marc  Sabatier</span>, <span class="hlFld-ContribAuthor ">Ana  Estévez-Braun</span>, <span class="hlFld-ContribAuthor ">Bruno  Pagano</span>, <span class="hlFld-ContribAuthor ">Stefano  Mangani</span>, <span class="hlFld-ContribAuthor ">Rino  Ragno</span>, <span class="hlFld-ContribAuthor ">George  Kokotos</span>, <span class="hlFld-ContribAuthor ">Margherita  Brindisi</span>, <span class="hlFld-ContribAuthor ">Florenci V.  González</span>, <span class="hlFld-ContribAuthor ">Fernanda  Borges</span>, <span class="hlFld-ContribAuthor ">Mariarosaria  Miloso</span>, <span class="hlFld-ContribAuthor ">Jarkko  Rautio</span>, <span class="hlFld-ContribAuthor ">Diego  Muñoz-Torrero</span>. </span><span class="cited-content_cbyCitation_article-title">Breakthroughs in Medicinal Chemistry: New Targets and Mechanisms, New Drugs, New Hopes–7. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (13)
                                     , 2968. <a href="https://doi.org/10.3390/molecules25132968" title="DOI URL">https://doi.org/10.3390/molecules25132968</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25132968&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25132968%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DBreakthroughs%252Bin%252BMedicinal%252BChemistry%25253A%252BNew%252BTargets%252Band%252BMechanisms%25252C%252BNew%252BDrugs%25252C%252BNew%252BHopes%2525E2%252580%2525937%26aulast%3DG%25C3%25BCtschow%26aufirst%3DMichael%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D13%26spage%3D2968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Crystal structure overlay of the various approaches to either covalently modify RAS (shown in purple), block the interaction with the GEF SOS, or bind directly to allosteric sites within SOS (shown in gray). Annotations indicate PDB code, protein of interest, and year of disclosure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of reported KRAS<sup>G12C</sup> inhibitors. ARS-853 (<b>1</b>) and ARS-1620 (<b>2</b>) from Araxes Pharma/Wellspring Biosciences, AMG 510 (<b>3</b>) from Amgen, and MRTX1257 (<b>4</b>) from Mirati Therapeutics/Array Biopharma.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0009.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Piperazine <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) methyl <span class="smallcaps smallerCapital">d</span>-alaninate hydrochloride, EDC, DIPEA, DCM, 0–25 °C, 16 h, 77%; (ii) cyclohexene, 10% Pd/C, MeOH, 32 h, 88%; (iii) borane, THF, 0–70 °C, 16 h, 94%; (iv) (Boc)<sub>2</sub>O, TEA, MeOH, 0–100 °C, 34 h, then 1.5 M NaOH, EtOH, 25–100 °C, 16 h; 99%; (v) CBz-Cl, 1 M NaOH, THF, 3–25 °C, 2 h, 87%; (vi) NaIO<sub>4</sub>, RuCl<sub>3</sub>, DCM, MeCN, dimethyl carbonate, 25 °C, 3 h, 88%; (vii) MeI, K<sub>2</sub>CO<sub>3</sub>, MeCN, 0–60 °C, 2 h, 90%; (viii) H<sub>2</sub>, 10% Pd/C, MeOH, 50 °C, 16 h, 99%.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) NCS, DMF, 25–50 °C, 3 h, 48%; (ii) 5-(methoxymethylene)-2,2-dimethyl-1,3-dioxane-4,6-dione, trimethoxymethane, 85–110 °C, 3 h; (iii) Dowtherm, 210 °C, 65% 2 steps; (iv) nitric acid, propionic acid, 125 °C, 2 h, 74%; (v) POCl<sub>3</sub>, toluene, 100 °C, 16 h, 81%; (vi) <b>13</b>, DIPEA, THF, 60 °C, 3 h, 77%; %; (vii) Fe, AcOH, 80 °C, 1 h, 97%; (viii) MeI, K<sub>2</sub>CO<sub>3</sub>, acetone, 40 °C, 16 h, 65%; (ix) (2-fluoro-6-hydroxyphenyl)boronic acid, RuPhos Pd G3, K<sub>2</sub>CO<sub>3</sub>, dioxane, water, 80 °C, 1.5 h, 85%; (x) HCl, MeOH, <i>i</i>-PrOH, 25 °C, 1 h, 34%; (xi) acryloyl chloride, TEA, DCM, then 1 M NH<sub>3</sub> in MeOH, 0 °C, 1 h, 37%.</p></p></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (left) Structure of compound <b>26</b> bound to KRAS<sup>G12C</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5V">6T5V</a>). The acrylamide forms a covalent bond to Cys12, with the carbonyl forming a hydrogen bond to Lys16. The quinazoline N1 forms a hydrogen bond to His95, while the 7-indazole forms direct and water mediated interactions with Asp69 and Ser65, of the switch II loop, respectively. (right) Overlay of the crystal structures of tethered analogue <b>29</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5U">6T5U</a>) and compound <b>26</b>. Compound <b>29</b> shows a minor frame-shift relative to <b>26</b>, such that the 7-indazole now forms direct interactions with Asp69 and Ser65, with other interactions being common. Note the presence of a displaceable water, shown as a red sphere, in both crystal structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Relative energy profile for driving the 4-piperazinoquinazoline torsion, highlighted in red, between 0° and 360°. Multiple local minima were observed, only one of which is close to the bioactive torsion, leading to the hypothesis that locking out one of the minima via tethering may provide an entropic boost in the binding free energy. The right-hand panel shows the various minima in green, overlaid against the bioactive conformation in orange (calculations conducted on the <i>des</i>-indazole version of compound <b>26</b> to reduce computational complexity).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (left) Overview of KRAS<sup>G12C</sup> complexed to GDP and compound <b>25</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5B">6T5B</a>). The switch I loop is shown in orange and the switch II loop shown in yellow. (right) Enhanced view of compound <b>25</b> bound to KRAS<sup>G12C</sup>. The acrylamide forms a covalent bond to Cys12, with the carbonyl forming a hydrogen bond to Lys16. The quinoline N1 forms a hydrogen bond to His95, while the 7-fluorophenol forms a direct interaction with Asp69 and a water mediated interaction to Ser65. The chiral <i>R</i>-Me sits axially relative to the piperazine carbonyl, resulting in the observed diaxial conformation and the displacement of the water molecule seen in the example shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. Water-mediated interactions are seen between the amidic tether and the backbone NH of Cys12.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Free cysteine proteomic profile of compound <b>25</b>. Each data point represents a biotinylated cysteine containing peptide with log<sub>2</sub> fold change and <i>p</i>-value compared to vehicle control calculated across three biological replicates per group. A log<sub>2</sub> fold change of >2 and a <i>p</i>-value of <0.01 are indicated in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Differential antiproliferative activity of compound <b>25</b> across KRAS<sup>G12C</sup>, KRAS<sup>WT</sup>, and KRAS<sup>non-G12Cmutant</sup> cancer cells in 3D soft agar proliferation assays. Mean GI<sub>50</sub> (μM) from a minimum of three independent experiments. Error bars represent standard error (SE). Cellular target engagement was measured by mass spectrometry following incubation of cells for 6 h with 1 μM of compound <b>25</b>. Target engagement shown as a percent relative to vehicle control treated cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/medium/jm9b01720_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0008.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Efficacy and PD of compound <b>25</b> in KRAS<sup>G12C</sup> Miapaca2 xenograft model. (left) Mice bearing Miapaca2 tumors were treated daily with vehicle control or 100, 50, and 20 mg/kg of compound <b>25</b> via oral dosing for 2 weeks followed by 3 weeks of treatment. Data are shown as geometric mean of the tumor volume and SE. (right) Mice bearing Miapaca2 tumors were treated with a single dose of vehicle control or 100, 30, and 10 mg/kg of compound <b>25</b> via oral dosing. The tumor expression of <i>DUSP6</i> mRNA were measured at 3 and 24 h postdose by qRT-PCR. The expression was normalized to <i>POLRA</i> mRNA and expressed relative to the vehicle. Data are shown as individual tumor data, treatment group geometric mean, and SE.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01720/20200507/images/large/jm9b01720_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01720&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i37">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01402" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01402" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 36 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Vega, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armenia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatila, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimitriadoy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantheti, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saghafinia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarty, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foltz, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shmulevich, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heins, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochoa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kandoth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahceci, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dervishi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dogrusoz, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laird, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Way, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iavarone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bivona, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazar, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammer, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwong, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McArthur, G</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tward, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederick, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Allen, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherniack, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciriello, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sander, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, N.</span></span>; <span class="NLM_contrib-group">Cancer Genome Atlas Research Network</span> <span> </span><span class="NLM_article-title">Oncogenic Signaling Pathways in The Cancer Genome Atlas</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">321</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.03.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1016%2Fj.cell.2018.03.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=29625050" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntFahsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=321-337&author=F.+Sanchez-Vegaauthor=M.+Minaauthor=J.+Armeniaauthor=W.+K.+Chatilaauthor=A.+Lunaauthor=K.+C.+Laauthor=S.+Dimitriadoyauthor=D.+L.+Liuauthor=H.+S.+Kanthetiauthor=S.+Saghafiniaauthor=D.+Chakravartyauthor=F.+Daianauthor=Q.+Gaoauthor=M.+H.+Baileyauthor=W.+W.+Liangauthor=S.+M.+Foltzauthor=I.+Shmulevichauthor=L.+Dingauthor=Z.+Heinsauthor=A.+Ochoaauthor=B.+Grossauthor=J.+Gaoauthor=H.+Zhangauthor=R.+Kundraauthor=C.+Kandothauthor=I.+Bahceciauthor=L.+Dervishiauthor=U.+Dogrusozauthor=W.+Zhouauthor=H.+Shenauthor=P.+W.+Lairdauthor=G.+P.+Wayauthor=C.+S.+Greeneauthor=H.+Liangauthor=Y.+Xiaoauthor=C.+Wangauthor=A.+Iavaroneauthor=A.+H.+Bergerauthor=T.+G.+Bivonaauthor=A.+J.+Lazarauthor=G.+D.+Hammerauthor=T.+Giordanoauthor=L.+N.+Kwongauthor=G+McArthurauthor=C.+Huangauthor=A.+D.+Twardauthor=M.+J.+Frederickauthor=F.+McCormickauthor=M.+Meyersonauthor=E.+M.+Van+Allenauthor=A.+D.+Cherniackauthor=G.+Cirielloauthor=C.+Sanderauthor=N.+Schultzauthor=Cancer+Genome+Atlas+Research+Network&title=Oncogenic+Signaling+Pathways+in+The+Cancer+Genome+Atlas&doi=10.1016%2Fj.cell.2018.03.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic signaling pathways in the cancer genome atlas</span></div><div class="casAuthors">Sanchez-Vega, Francisco; Mina, Marco; Armenia, Joshua; Chatila, Walid K.; Luna, Augustin; La, Konnor C.; Dimitriadoy, Sofia; Liu, David L.; Kantheti, Havish S.; Saghafinia, Sadegh; Chakravarty, Debyani; Daian, Foysal; Gao, Qingsong; Bailey, Matthew H.; Liang, Wen-Wei; Foltz, Steven M.; Shmulevich, Ilya; Ding, Li; Heins, Zachary; Ochoa, Angelica; Gross, Benjamin; Gao, Jianjiong; Zhang, Hongxin; Kundra, Ritika; Kandoth, Cyriac; Bahceci, Istemi; Dervishi, Leonard; Dogrusoz, Ugur; Zhou, Wanding; Shen, Hui; Laird, Peter W.; Way, Gregory P.; Greene, Casey S.; Liang, Han; Xiao, Yonghong; Wang, Chen; Iavarone, Antonio; Berger, Alice H.; Bivona, Trever G.; Lazar, Alexander J.; Hammer, Gary D.; Giordano, Thomas; Kwong, Lawrence N.; McArthur, Grant; Huang, Chenfei; Tward, Aaron D.; Frederick, Mitchell J.; McCormick, Frank; Meyerson, Matthew; Van Allen, Eliezer M.; Cherniack, Andrew D.; Ciriello, Giovanni; Sander, Chris; Schultz, Nikolaus</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">321-337.e10</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Genetic alterations in signaling pathways that control cell-cycle progression, apoptosis, and cell growth are common hallmarks of cancer, but the extent, mechanisms, and co-occurrence of alterations in these pathways differ between individual tumors and tumor types.  Using mutations, copy-no. changes, mRNA expression, gene fusions and DNA methylation in 9125 tumors profiled by The Cancer Genome Atlas (TCGA), we analyzed the mechanisms and patterns of somatic alterations in ten canonical pathways: cell cycle, Hippo, Myc, Notch, Nrf2, PI-3-Kinase/Akt, RTK-RAS, TGFβ signaling, p53 and β-catenin/Wnt.  We charted the detailed landscape of pathway alterations in 33 cancer types, stratified into 64 subtypes, and identified patterns of co-occurrence and mutual exclusivity.  Eighty-nine percent of tumors had at least one driver alteration in these pathways, and 57% percent of tumors had at least one alteration potentially targetable by currently available drugs.  Thirty percent of tumors had multiple targetable alterations, indicating opportunities for combination therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv_ggbxuypEbVg90H21EOLACvtfcHk0ljQQyEinC4VEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntFahsrs%253D&md5=3677d3a77b12767c828f48eb80d71141</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.03.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.03.035%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez-Vega%26aufirst%3DF.%26aulast%3DMina%26aufirst%3DM.%26aulast%3DArmenia%26aufirst%3DJ.%26aulast%3DChatila%26aufirst%3DW.%2BK.%26aulast%3DLuna%26aufirst%3DA.%26aulast%3DLa%26aufirst%3DK.%2BC.%26aulast%3DDimitriadoy%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DD.%2BL.%26aulast%3DKantheti%26aufirst%3DH.%2BS.%26aulast%3DSaghafinia%26aufirst%3DS.%26aulast%3DChakravarty%26aufirst%3DD.%26aulast%3DDaian%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DQ.%26aulast%3DBailey%26aufirst%3DM.%2BH.%26aulast%3DLiang%26aufirst%3DW.%2BW.%26aulast%3DFoltz%26aufirst%3DS.%2BM.%26aulast%3DShmulevich%26aufirst%3DI.%26aulast%3DDing%26aufirst%3DL.%26aulast%3DHeins%26aufirst%3DZ.%26aulast%3DOchoa%26aufirst%3DA.%26aulast%3DGross%26aufirst%3DB.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DKundra%26aufirst%3DR.%26aulast%3DKandoth%26aufirst%3DC.%26aulast%3DBahceci%26aufirst%3DI.%26aulast%3DDervishi%26aufirst%3DL.%26aulast%3DDogrusoz%26aufirst%3DU.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DLaird%26aufirst%3DP.%2BW.%26aulast%3DWay%26aufirst%3DG.%2BP.%26aulast%3DGreene%26aufirst%3DC.%2BS.%26aulast%3DLiang%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DIavarone%26aufirst%3DA.%26aulast%3DBerger%26aufirst%3DA.%2BH.%26aulast%3DBivona%26aufirst%3DT.%2BG.%26aulast%3DLazar%26aufirst%3DA.%2BJ.%26aulast%3DHammer%26aufirst%3DG.%2BD.%26aulast%3DGiordano%26aufirst%3DT.%26aulast%3DKwong%26aufirst%3DL.%2BN.%26aulast%3DMcArthur%26aufirst%3DG%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DTward%26aufirst%3DA.%2BD.%26aulast%3DFrederick%26aufirst%3DM.%2BJ.%26aulast%3DMcCormick%26aufirst%3DF.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DVan%2BAllen%26aufirst%3DE.%2BM.%26aulast%3DCherniack%26aufirst%3DA.%2BD.%26aulast%3DCiriello%26aufirst%3DG.%26aulast%3DSander%26aufirst%3DC.%26aulast%3DSchultz%26aufirst%3DN.%26aulast%3D%26atitle%3DOncogenic%2520Signaling%2520Pathways%2520in%2520The%2520Cancer%2520Genome%2520Atlas%26jtitle%3DCell%26date%3D2018%26volume%3D173%26spage%3D321%26epage%3D337%26doi%3D10.1016%2Fj.cell.2018.03.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note"><p class="first last">e10.</p></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prior, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattos, C.</span></span> <span> </span><span class="NLM_article-title">A Comprehensive Survey of Ras Mutations in Cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">2457</span>– <span class="NLM_lpage">2467</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-2612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1158%2F0008-5472.CAN-11-2612" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=22589270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvF2ntL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=2457-2467&author=I.+A.+Priorauthor=P.+D.+Lewisauthor=C.+Mattos&title=A+Comprehensive+Survey+of+Ras+Mutations+in+Cancer&doi=10.1158%2F0008-5472.CAN-11-2612"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Survey of Ras Mutations in Cancer</span></div><div class="casAuthors">Prior, Ian A.; Lewis, Paul D.; Mattos, Carla</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2457-2467</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  All mammalian cells express 3 closely related Ras proteins, termed H-Ras, K-Ras, and N-Ras, that promote oncogenesis when they are mutationally activated at codon 12, 13, or 61.  Although there is a high degree of similarity among the isoforms, K-Ras mutations are far more frequently obsd. in cancer, and each isoform displays preferential coupling to particular cancer types.  We examd. the mutational spectra of Ras isoforms curated from large-scale tumor profiling and found that each isoform exhibits surprisingly distinctive codon mutation and amino-acid substitution biases.  These findings were unexpected given that these mutations occur in regions that share 100% amino-acid sequence identity among the 3 isoforms.  Of importance, many of these mutational biases were not due to differences in exposure to mutagens, because the patterns were still evident when compared within specific cancer types.  We discuss potential genetic and epigenetic mechanisms, as well as isoform-specific differences in protein structure and signaling, that may promote these distinct mutation patterns and differential coupling to specific cancers.  Cancer Res; 72(10); 2457-67.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg2lqiicosOLVg90H21EOLACvtfcHk0ljUqEHH_Ljm7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvF2ntL4%253D&md5=6d1894847015468f77c8b08b57c6d62a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-2612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-2612%26sid%3Dliteratum%253Aachs%26aulast%3DPrior%26aufirst%3DI.%2BA.%26aulast%3DLewis%26aufirst%3DP.%2BD.%26aulast%3DMattos%26aufirst%3DC.%26atitle%3DA%2520Comprehensive%2520Survey%2520of%2520Ras%2520Mutations%2520in%2520Cancer%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D2457%26epage%3D2467%26doi%3D10.1158%2F0008-5472.CAN-11-2612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haigis, K. M.</span></span> <span> </span><span class="NLM_article-title"><i>KRAS</i> Alleles: The Devil is in the Detail</span>. <i>Trends Cancer.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">686</span>– <span class="NLM_lpage">697</span>, <span class="refDoi"> DOI: 10.1016/j.trecan.2017.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1016%2Fj.trecan.2017.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=28958387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1MXksVyhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=686-697&author=K.+M.+Haigis&title=KRAS+Alleles%3A+The+Devil+is+in+the+Detail&doi=10.1016%2Fj.trecan.2017.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">KRAS Alleles: The Devil Is in the Detail</span></div><div class="casAuthors">Haigis, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">686-697</span>CODEN:
                <span class="NLM_cas:coden">TCRAB5</span>;
        ISSN:<span class="NLM_cas:issn">2405-8033</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  KRAS is the most frequently mutated oncogene in cancer and KRAS mutation is commonly assocd. with poor prognosis and resistance to therapy.  Since the KRAS oncoprotein is, as yet, not directly druggable, efforts to target KRAS mutant cancers focus on identifying vulnerabilities in downstream signaling pathways or in stress response pathways that are permissive for strong oncogenic signaling.  One aspect of KRAS biol. that is not well appreciated is the potential biol. differences between the many distinct KRAS activating mutations.  This review draws upon insights from both clin. and exptl. studies to explore similarities and differences among KRAS alleles.  Historical and emerging evidence supports the notion that the specific biol. related to each allele might be exploitable for allele-specific therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjsS2Ho266RLVg90H21EOLACvtfcHk0ljUqEHH_Ljm7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXksVyhsg%253D%253D&md5=672e8e288211cef69fc149d658366e53</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.trecan.2017.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trecan.2017.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DHaigis%26aufirst%3DK.%2BM.%26atitle%3DKRAS%2520Alleles%253A%2520The%2520Devil%2520is%2520in%2520the%2520Detail%26jtitle%3DTrends%2520Cancer.%26date%3D2017%26volume%3D3%26spage%3D686%26epage%3D697%26doi%3D10.1016%2Fj.trecan.2017.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lito, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span> <span> </span><span class="NLM_article-title">Allele-specific Inhibitors Inactivate Mutant KRAS G12C by a Trapping Mechanism</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span>,  <span class="NLM_fpage">604</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1126/science.aad6204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1126%2Fscience.aad6204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=26841430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2jt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=604-608&author=P.+Litoauthor=M.+Solomonauthor=L.+S.+Liauthor=R.+Hansenauthor=N.+Rosen&title=Allele-specific+Inhibitors+Inactivate+Mutant+KRAS+G12C+by+a+Trapping+Mechanism&doi=10.1126%2Fscience.aad6204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Allele-specific inhibitors inactivate mutant KRAS G12C by a trapping mechanism</span></div><div class="casAuthors">Lito, Piro; Solomon, Martha; Li, Lian-Sheng; Hansen, Rasmus; Rosen, Neal</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6273</span>),
    <span class="NLM_cas:pages">604-608</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">It is thought that KRAS oncoproteins are constitutively active because their guanosine triphosphatase (GTPase) activity is disabled.  Consequently, drugs targeting the inactive or GDP-bound conformation are not expected to be effective.  We describe a mechanism that enables such drugs to inhibit KRASG12C signaling and cancer cell growth.  Inhibition requires intact GTPase activity and occurs because drug-bound KRASG12C is insusceptible to nucleotide exchange factors and thus trapped in its inactive state.  Indeed, mutants completely lacking GTPase activity and those promoting exchange reduced the potency of the drug.  Suppressing nucleotide exchange activity downstream of various tyrosine kinases enhanced KRASG12C inhibition, whereas its potentiation had the opposite effect.  These findings reveal that KRASG12C undergoes nucleotide cycling in cancer cells and provide a basis for developing effective therapies to treat KRASG12C-driven cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqK3AbqEZO6k7Vg90H21EOLACvtfcHk0ljUqEHH_Ljm7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2jt78%253D&md5=22a47908ac0146822ecf19f5d0c952de</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad6204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad6204%26sid%3Dliteratum%253Aachs%26aulast%3DLito%26aufirst%3DP.%26aulast%3DSolomon%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DL.%2BS.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DRosen%26aufirst%3DN.%26atitle%3DAllele-specific%2520Inhibitors%2520Inactivate%2520Mutant%2520KRAS%2520G12C%2520by%2520a%2520Trapping%2520Mechanism%26jtitle%3DScience%26date%3D2016%26volume%3D351%26spage%3D604%26epage%3D608%26doi%3D10.1126%2Fscience.aad6204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fakih, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falchook, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govindan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, P. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henary, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, D. S.</span></span> <span> </span><span class="NLM_article-title">Phase 1 Study Evaluating the Safety, Tolerability, Pharmacokinetics (PK), and Efficacy of AMG 510, a Novel Small Molecule KRASG12C Inhibitor, in Advanced Solid Tumors</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">3003</span>– <span class="NLM_lpage">3003</span>, <span class="refDoi"> DOI: 10.1200/JCO.2019.37.15_suppl.3003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1200%2FJCO.2019.37.15_suppl.3003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=3003-3003&author=M.+Fakihauthor=B.+O%E2%80%99Neilauthor=T.+J.+Priceauthor=G.+S.+Falchookauthor=J.+Desaiauthor=J.+Kuoauthor=R.+Govindanauthor=E.+Rasmussenauthor=P.+K.+H.+Morrowauthor=J.+Ngangauthor=H.+A.+Henaryauthor=D.+S.+Hong&title=Phase+1+Study+Evaluating+the+Safety%2C+Tolerability%2C+Pharmacokinetics+%28PK%29%2C+and+Efficacy+of+AMG+510%2C+a+Novel+Small+Molecule+KRASG12C+Inhibitor%2C+in+Advanced+Solid+Tumors&doi=10.1200%2FJCO.2019.37.15_suppl.3003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1200%2FJCO.2019.37.15_suppl.3003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2019.37.15_suppl.3003%26sid%3Dliteratum%253Aachs%26aulast%3DFakih%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DB.%26aulast%3DPrice%26aufirst%3DT.%2BJ.%26aulast%3DFalchook%26aufirst%3DG.%2BS.%26aulast%3DDesai%26aufirst%3DJ.%26aulast%3DKuo%26aufirst%3DJ.%26aulast%3DGovindan%26aufirst%3DR.%26aulast%3DRasmussen%26aufirst%3DE.%26aulast%3DMorrow%26aufirst%3DP.%2BK.%2BH.%26aulast%3DNgang%26aufirst%3DJ.%26aulast%3DHenary%26aufirst%3DH.%2BA.%26aulast%3DHong%26aufirst%3DD.%2BS.%26atitle%3DPhase%25201%2520Study%2520Evaluating%2520the%2520Safety%252C%2520Tolerability%252C%2520Pharmacokinetics%2520%2528PK%2529%252C%2520and%2520Efficacy%2520of%2520AMG%2520510%252C%2520a%2520Novel%2520Small%2520Molecule%2520KRASG12C%2520Inhibitor%252C%2520in%2520Advanced%2520Solid%2520Tumors%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D37%26spage%3D3003%26epage%3D3003%26doi%3D10.1200%2FJCO.2019.37.15_suppl.3003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quevedo, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruz-Migoni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bery, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bataille, C. J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, L. Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallon, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tulmin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehebauer, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Fuentes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, S. E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabbitts, T. H.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors of RAS-Effector Protein Interactions Derived Using an Intracellular Antibody Fragment</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3169</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-05707-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1038%2Fs41467-018-05707-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=30093669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A280%3ADC%252BB3c7ms12mtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=3169&author=C.+E.+Quevedoauthor=A.+Cruz-Migoniauthor=N.+Beryauthor=A.+Millerauthor=T.+Tanakaauthor=D.+Petchauthor=C.+J.+R.+Batailleauthor=L.+Y.+W.+Leeauthor=P.+S.+Fallonauthor=H.+Tulminauthor=M.+T.+Ehebauerauthor=N.+Fernandez-Fuentesauthor=A.+J.+Russellauthor=S.+B.+Carrauthor=S.+E.+V.+Phillipsauthor=T.+H.+Rabbitts&title=Small+Molecule+Inhibitors+of+RAS-Effector+Protein+Interactions+Derived+Using+an+Intracellular+Antibody+Fragment&doi=10.1038%2Fs41467-018-05707-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors of RAS-effector protein interactions derived using an intracellular antibody fragment</span></div><div class="casAuthors">Quevedo Camilo E; Cruz-Migoni Abimael; Bery Nicolas; Miller Ami; Tulmin Hanna; Ehebauer Matthias T; Rabbitts Terence H; Cruz-Migoni Abimael; Fernandez-Fuentes Narcis; Carr Stephen B; Phillips Simon E V; Tanaka Tomoyuki; Petch Donna; Tanaka Tomoyuki; Bataille Carole J R; Russell Angela J; Lee Lydia Y W; Fallon Phillip S; Tulmin Hanna; Ehebauer Matthias T; Fernandez-Fuentes Narcis; Carr Stephen B; Phillips Simon E V</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3169</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Targeting specific protein-protein interactions (PPIs) is an attractive concept for drug development, but hard to implement since intracellular antibodies do not penetrate cells and most small-molecule drugs are considered unsuitable for PPI inhibition.  A potential solution to these problems is to select intracellular antibody fragments to block PPIs, use these antibody fragments for target validation in disease models and finally derive small molecules overlapping the antibody-binding site.  Here, we explore this strategy using an anti-mutant RAS antibody fragment as a competitor in a small-molecule library screen for identifying RAS-binding compounds.  The initial hits are optimized by structure-based design, resulting in potent RAS-binding compounds that interact with RAS inside the cells, prevent RAS-effector interactions and inhibit endogenous RAS-dependent signalling.  Our results may aid RAS-dependent cancer drug development and demonstrate a general concept for developing small compounds to replace intracellular antibody fragments, enabling rational drug development to target validated PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSIUnXfwZaCRvxQvRxYU81tfW6udTcc2ebQnJhVBzg4mbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7ms12mtQ%253D%253D&md5=79679120c60be8cd9be9fbbfe0c9fdb6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-05707-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-05707-2%26sid%3Dliteratum%253Aachs%26aulast%3DQuevedo%26aufirst%3DC.%2BE.%26aulast%3DCruz-Migoni%26aufirst%3DA.%26aulast%3DBery%26aufirst%3DN.%26aulast%3DMiller%26aufirst%3DA.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DPetch%26aufirst%3DD.%26aulast%3DBataille%26aufirst%3DC.%2BJ.%2BR.%26aulast%3DLee%26aufirst%3DL.%2BY.%2BW.%26aulast%3DFallon%26aufirst%3DP.%2BS.%26aulast%3DTulmin%26aufirst%3DH.%26aulast%3DEhebauer%26aufirst%3DM.%2BT.%26aulast%3DFernandez-Fuentes%26aufirst%3DN.%26aulast%3DRussell%26aufirst%3DA.%2BJ.%26aulast%3DCarr%26aufirst%3DS.%2BB.%26aulast%3DPhillips%26aufirst%3DS.%2BE.%2BV.%26aulast%3DRabbitts%26aufirst%3DT.%2BH.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520of%2520RAS-Effector%2520Protein%2520Interactions%2520Derived%2520Using%2520an%2520Intracellular%2520Antibody%2520Fragment%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D3169%26doi%3D10.1038%2Fs41467-018-05707-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmachl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantoulidis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gollner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häring, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hela, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Oezguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knesl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornigg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kousek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamarre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munico-Martinez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peinsipp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salamon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherbantin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipany, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrenk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siszler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolkerstorfer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span> <span> </span><span class="NLM_article-title">Drugging an Undruggable Pocket on KRAS</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">15823</span>– <span class="NLM_lpage">15829</span>, <span class="refDoi"> DOI: 10.1073/pnas.1904529116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1073%2Fpnas.1904529116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=31332011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFChs7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=15823-15829&author=D.+Kesslerauthor=M.+Gmachlauthor=A.+Mantoulidisauthor=L.+T.+Martinauthor=A.+Zoephelauthor=M.+Mayerauthor=A.+Gollnerauthor=D.+Coviniauthor=S.+Fischerauthor=T.+Gerstbergerauthor=T.+Gmaschitzauthor=C.+Goodwinauthor=P.+Grebauthor=D.+H%C3%A4ringauthor=W.+Helaauthor=J.+Hoffmannauthor=J.+Karolyi-Oezguerauthor=P.+Kneslauthor=S.+Korniggauthor=M.+Koeglauthor=R.+Kousekauthor=L.+Lamarreauthor=F.+Moserauthor=S.+Munico-Martinezauthor=C.+Peinsippauthor=J.+Phanauthor=J.+Rinnenthalauthor=J.+Saiauthor=C.+Salamonauthor=Y.+Scherbantinauthor=K.+Schipanyauthor=R.+Schnitzerauthor=A.+Schrenkauthor=B.+Sharpsauthor=G.+Siszlerauthor=Q.+Sunauthor=A.+Watersonauthor=B.+Wolkerstorferauthor=M.+Zeebauthor=M.+Pearsonauthor=S.+W.+Fesikauthor=D.+B.+McConnell&title=Drugging+an+Undruggable+Pocket+on+KRAS&doi=10.1073%2Fpnas.1904529116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging an undruggable pocket on KRAS</span></div><div class="casAuthors">Kessler, Dirk; Gmachl, Michael; Mantoulidis, Andreas; Martin, Laetitia J.; Zoephel, Andreas; Mayer, Moriz; Gollner, Andreas; Covini, David; Fischer, Silke; Gerstberger, Thomas; Gmaschitz, Teresa; Goodwin, Craig; Greb, Peter; Haring, Daniela; Hela, Wolfgang; Hoffmann, Johann; Karolyi-Oezguer, Jale; Knesl, Petr; Kornigg, Stefan; Koegl, Manfred; Kousek, Roland; Lamarre, Lyne; Moser, Franziska; Munico-Martinez, Silvia; Peinsipp, Christoph; Phan, Jason; Rinnenthal, Jorg; Sai, Jiqing; Salamon, Christian; Scherbantin, Yvonne; Schipany, Katharina; Schnitzer, Renate; Schrenk, Andreas; Sharps, Bernadette; Siszler, Gabriella; Sun, Qi; Waterson, Alex; Wolkerstorfer, Bernhard; Zeeb, Markus; Pearson, Mark; Fesik, Stephen W.; McConnell, Darryl B.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">15823-15829</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">(Imidazolylmethylindolylmethylaminomethyl)indolyl hydroxyisoindolone I, an inhibitor of both the active and inactive forms of KRAS, was discovered using fragment-based screening.  I bound to KRAS in a pocket between switches I and II of KRAS, blocking all GEF, GAP, and effector interactions with KRAS and thus inhibiting downstream signaling; I inhibited KRAS mutant cells at low micromolar concns.  Uniformly 15N-labeled KRAS bound to the stable GTP analog GCP was prepd. with yields of 10 mg/L for use in crystallog. of KRAS-inhibitor complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaMeFg74p2-rVg90H21EOLACvtfcHk0livBAikg6gKdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFChs7jJ&md5=ebf443050cb2ce6cbeec1e40084129d0</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1904529116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1904529116%26sid%3Dliteratum%253Aachs%26aulast%3DKessler%26aufirst%3DD.%26aulast%3DGmachl%26aufirst%3DM.%26aulast%3DMantoulidis%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DL.%2BT.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DGollner%26aufirst%3DA.%26aulast%3DCovini%26aufirst%3DD.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DGoodwin%26aufirst%3DC.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DH%25C3%25A4ring%26aufirst%3DD.%26aulast%3DHela%26aufirst%3DW.%26aulast%3DHoffmann%26aufirst%3DJ.%26aulast%3DKarolyi-Oezguer%26aufirst%3DJ.%26aulast%3DKnesl%26aufirst%3DP.%26aulast%3DKornigg%26aufirst%3DS.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DKousek%26aufirst%3DR.%26aulast%3DLamarre%26aufirst%3DL.%26aulast%3DMoser%26aufirst%3DF.%26aulast%3DMunico-Martinez%26aufirst%3DS.%26aulast%3DPeinsipp%26aufirst%3DC.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DSai%26aufirst%3DJ.%26aulast%3DSalamon%26aufirst%3DC.%26aulast%3DScherbantin%26aufirst%3DY.%26aulast%3DSchipany%26aufirst%3DK.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DSchrenk%26aufirst%3DA.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DSiszler%26aufirst%3DG.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DWaterson%26aufirst%3DA.%26aulast%3DWolkerstorfer%26aufirst%3DB.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26atitle%3DDrugging%2520an%2520Undruggable%2520Pocket%2520on%2520KRAS%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D15823%26epage%3D15829%26doi%3D10.1073%2Fpnas.1904529116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olejniczak, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossanese, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of Small Molecules that Bind to K-Ras and Inhibit Sos-Mediated Activation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">6140</span>– <span class="NLM_lpage">6143</span>, <span class="refDoi"> DOI: 10.1002/anie.201201358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1002%2Fanie.201201358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC38Xms1entr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=6140-6143&author=Q.+Sunauthor=J.+P.+Burkeauthor=J.+Phanauthor=M.+C.+Burnsauthor=E.+T.+Olejniczakauthor=A.+G.+Watersonauthor=T.+Leeauthor=O.+W.+Rossaneseauthor=S.+W.+Fesik&title=Discovery+of+Small+Molecules+that+Bind+to+K-Ras+and+Inhibit+Sos-Mediated+Activation&doi=10.1002%2Fanie.201201358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecules that Bind to K-Ras and Inhibit Sos-Mediated Activation</span></div><div class="casAuthors">Sun, Qi; Burke, Jason P.; Phan, Jason; Burns, Michael C.; Olejniczak, Edward T.; Waterson, Alex G.; Lee, Taekyu; Rossanese, Olivia W.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">6140-6143, S6140/1-S6140/17</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">K-Ras inhibition represents an attractive therapeutic strategy for many cancers.  We report on the discovery of novel small mols. that bind directly to K-Ras between switch I and switch II and inhibit Sos-catalyzed K-Ras activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy5Cx7BSO1pLVg90H21EOLACvtfcHk0livBAikg6gKdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xms1entr4%253D&md5=7d579523462aaf7eac1890643f91fab3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fanie.201201358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201201358%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DBurke%26aufirst%3DJ.%2BP.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DBurns%26aufirst%3DM.%2BC.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DRossanese%26aufirst%3DO.%2BW.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520Small%2520Molecules%2520that%2520Bind%2520to%2520K-Ras%2520and%2520Inhibit%2520Sos-Mediated%2520Activation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D6140%26epage%3D6143%26doi%3D10.1002%2Fanie.201201358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrenton, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skelton, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fauber, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokoe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludlam, M. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giannetti, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starovasnik, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span> <span> </span><span class="NLM_article-title">Small-molecule Ligands Bind to a Distinct Pocket in Ras and Inhibit SOS-mediated Nucleotide Exchange Activity</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">5299</span>– <span class="NLM_lpage">5304</span>, <span class="refDoi"> DOI: 10.1073/pnas.1116510109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1073%2Fpnas.1116510109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=22431598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslOjsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5299-5304&author=T.+Maurerauthor=L.+S.+Garrentonauthor=A.+Ohauthor=K.+Pittsauthor=D.+J.+Andersonauthor=N.+J.+Skeltonauthor=B.+P.+Fauberauthor=B.+Panauthor=S.+Malekauthor=D.+Stokoeauthor=M.+J.+C.+Ludlamauthor=K.+K.+Bowmanauthor=J.+Wuauthor=A.+M.+Giannettiauthor=M.+A.+Starovasnikauthor=I.+Mellmanauthor=P.+K.+Jacksonauthor=J.+Rudolphauthor=W.+Wangauthor=G.+Fang&title=Small-molecule+Ligands+Bind+to+a+Distinct+Pocket+in+Ras+and+Inhibit+SOS-mediated+Nucleotide+Exchange+Activity&doi=10.1073%2Fpnas.1116510109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule ligands bind to a distinct pocket in Ras and inhibit SOS-mediated nucleotide exchange activity</span></div><div class="casAuthors">Maurer, Till; Garrenton, Lindsay S.; Oh, Angela; Pitts, Keith; Anderson, Daniel J.; Skelton, Nicholas J.; Fauber, Benjamin P.; Pan, Borlan; Malek, Shiva; Stokoe, David; Ludlam, Mary J. C.; Bowman, Krista K.; Wu, Jiansheng; Giannetti, Anthony M.; Starovasnik, Melissa A.; Mellman, Ira; Jackson, Peter K.; Rudolph, Joachim; Wang, Weiru; Fang, Guowei</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5299-5304, S5299/1-S5299/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Ras gene is frequently mutated in cancer, and mutant Ras drives tumorigenesis.  Although Ras is a central oncogene, small mols. that bind to Ras in a well-defined manner and exert inhibitory effects have not been uncovered to date.  Through an NMR-based fragment screen, we identified a group of small mols. that all bind to a common site on Ras.  High-resoln. cocrystal structures delineated a unique ligand-binding pocket on the Ras protein that is adjacent to the switch I/II regions and can be expanded upon compd. binding.  Structure anal. predicts that compd.-binding interferes with the Ras/SOS interactions.  Indeed, selected compds. inhibit SOS-mediated nucleotide exchange and prevent Ras activation by blocking the formation of intermediates of the exchange reaction.  The discovery of a small-mol. binding pocket on Ras with functional significance provides a new direction in the search of therapeutically effective inhibitors of the Ras oncoprotein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAXIzMe7yBqrVg90H21EOLACvtfcHk0liVbmsbajB4yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslOjsrc%253D&md5=a7210cad45154c42eb2a1faeabf88c20</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1116510109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1116510109%26sid%3Dliteratum%253Aachs%26aulast%3DMaurer%26aufirst%3DT.%26aulast%3DGarrenton%26aufirst%3DL.%2BS.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DPitts%26aufirst%3DK.%26aulast%3DAnderson%26aufirst%3DD.%2BJ.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26aulast%3DFauber%26aufirst%3DB.%2BP.%26aulast%3DPan%26aufirst%3DB.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DLudlam%26aufirst%3DM.%2BJ.%2BC.%26aulast%3DBowman%26aufirst%3DK.%2BK.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DStarovasnik%26aufirst%3DM.%2BA.%26aulast%3DMellman%26aufirst%3DI.%26aulast%3DJackson%26aufirst%3DP.%2BK.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DFang%26aufirst%3DG.%26atitle%3DSmall-molecule%2520Ligands%2520Bind%2520to%2520a%2520Distinct%2520Pocket%2520in%2520Ras%2520and%2520Inhibit%2520SOS-mediated%2520Nucleotide%2520Exchange%2520Activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D5299%26epage%3D5304%26doi%3D10.1073%2Fpnas.1116510109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, J. J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blades, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassington, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breeze, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chresta, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Embrey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairley, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faulder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finlay, M. R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kettle, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spadola, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrigley, G.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Binding Sites on the Ras:SOS Complex Can Be Exploited for Inhibition of Ras Activation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2265</span>– <span class="NLM_lpage">2274</span>, <span class="refDoi"> DOI: 10.1021/jm501660t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501660t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivF2qu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2265-2274&author=J.+J.+G.+Winterauthor=M.+Andersonauthor=K.+Bladesauthor=C.+Brassingtonauthor=A.+L.+Breezeauthor=C.+Chrestaauthor=K.+Embreyauthor=G.+Fairleyauthor=P.+Faulderauthor=M.+R.+V.+Finlayauthor=J.+G.+Kettleauthor=T.+Nowakauthor=R.+Overmanauthor=S.+J.+Patelauthor=P.+Perkinsauthor=L.+Spadolaauthor=J.+Tartauthor=J.+A.+Tuckerauthor=G.+Wrigley&title=Small+Molecule+Binding+Sites+on+the+Ras%3ASOS+Complex+Can+Be+Exploited+for+Inhibition+of+Ras+Activation&doi=10.1021%2Fjm501660t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Binding Sites on the Ras:SOS Complex Can Be Exploited for Inhibition of Ras Activation</span></div><div class="casAuthors">Winter, Jon J. G.; Anderson, Malcolm; Blades, Kevin; Brassington, Claire; Breeze, Alexander L.; Chresta, Christine; Embrey, Kevin; Fairley, Gary; Faulder, Paul; Finlay, M. Raymond V.; Kettle, Jason G.; Nowak, Thorsten; Overman, Ross; Patel, S. Joe; Perkins, Paula; Spadola, Loredana; Tart, Jonathan; Tucker, Julie A.; Wrigley, Gail</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2265-2274</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Constitutively active mutant KRas displays a reduced rate of GTP hydrolysis via both intrinsic and GTPase-activating protein-catalyzed mechanisms, resulting in the perpetual activation of Ras pathways.  We describe a fragment screening campaign using X-ray crystallog. that led to the discovery of three fragment binding sites on the Ras:SOS complex.  The identification of tool compds. binding at each of these sites allowed exploration of two new approaches to Ras pathway inhibition by stabilizing or covalently modifying the Ras:SOS complex to prevent the reloading of Ras with GTP.  Initially, we identified ligands that bound reversibly to the Ras:SOS complex in two distinct sites, but these compds. were not sufficiently potent inhibitors to validate our stabilization hypothesis.  We conclude by demonstrating that covalent modification of Cys118 on Ras leads to a novel mechanism of inhibition of the SOS-mediated interaction between Ras and Raf and is effective at inhibiting the exchange of labeled GDP in both mutant (G12C and G12V) and wild type Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfpT9XplX1v7Vg90H21EOLACvtfcHk0liVbmsbajB4yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivF2qu7w%253D&md5=b3747c8c9bb530d39c03ad56998d90f1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm501660t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501660t%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DJ.%2BJ.%2BG.%26aulast%3DAnderson%26aufirst%3DM.%26aulast%3DBlades%26aufirst%3DK.%26aulast%3DBrassington%26aufirst%3DC.%26aulast%3DBreeze%26aufirst%3DA.%2BL.%26aulast%3DChresta%26aufirst%3DC.%26aulast%3DEmbrey%26aufirst%3DK.%26aulast%3DFairley%26aufirst%3DG.%26aulast%3DFaulder%26aufirst%3DP.%26aulast%3DFinlay%26aufirst%3DM.%2BR.%2BV.%26aulast%3DKettle%26aufirst%3DJ.%2BG.%26aulast%3DNowak%26aufirst%3DT.%26aulast%3DOverman%26aufirst%3DR.%26aulast%3DPatel%26aufirst%3DS.%2BJ.%26aulast%3DPerkins%26aufirst%3DP.%26aulast%3DSpadola%26aufirst%3DL.%26aulast%3DTart%26aufirst%3DJ.%26aulast%3DTucker%26aufirst%3DJ.%2BA.%26aulast%3DWrigley%26aufirst%3DG.%26atitle%3DSmall%2520Molecule%2520Binding%2520Sites%2520on%2520the%2520Ras%253ASOS%2520Complex%2520Can%2520Be%2520Exploited%2520for%2520Inhibition%2520of%2520Ras%2520Activation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2265%26epage%3D2274%26doi%3D10.1021%2Fjm501660t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hillig, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sautier, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilpmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmann, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werbeck, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boemer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mastouri, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siemeister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahmann, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegener, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhnke, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wortmann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Nussbaum, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent SOS1 Inhibitors that Block RAS Activation via Disruption of the RAS–SOS1 Interaction</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">2551</span>– <span class="NLM_lpage">2560</span>, <span class="refDoi"> DOI: 10.1073/pnas.1812963116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1073%2Fpnas.1812963116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=30683722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivVajs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=2551-2560&author=R.+C.+Hilligauthor=B.+Sautierauthor=J.+Schroederauthor=D.+Moosmayerauthor=A.+Hilpmannauthor=C.+M.+Stegmannauthor=N.+D.+Werbeckauthor=H.+Briemauthor=U.+Boemerauthor=J.+Weiskeauthor=V.+Badockauthor=J.+Mastouriauthor=K.+Petersenauthor=G.+Siemeisterauthor=J.+D.+Kahmannauthor=D.+Wegenerauthor=N.+B%C3%B6hnkeauthor=K.+Eisauthor=K.+Grahamauthor=L.+Wortmannauthor=F.+von+Nussbaumauthor=B.+Bader&title=Discovery+of+Potent+SOS1+Inhibitors+that+Block+RAS+Activation+via+Disruption+of+the+RAS%E2%80%93SOS1+Interaction&doi=10.1073%2Fpnas.1812963116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of potent SOS1 inhibitors that block RAS activation via disruption of the RAS-SOS1 interaction</span></div><div class="casAuthors">Hillig, Roman C.; Sautier, Brice; Schroeder, Jens; Moosmayer, Dieter; Hilpmann, AndrA; Stegmann, Christian M.; Werbeck, Nicolas D.; Briem, Hans; Boemer, Ulf; Weiske, Joerg; Badock, Volker; Mastouri, Julia; Petersen, Kirstin; Siemeister, Gerhard; Kahmann, Jan D.; Wegener, Dennis; Boehnke, Niels; Eis, Knut; Graham, Keith; Wortmann, Lars; von Nussbaum, Franz; Bader, Benjamin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2551-2560</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Since the late 1980s, mutations in the RAS genes have been recognized as major oncogenes with a high occurrence rate in human cancers.  Such mutations reduce the ability of the small GTPase RAS to hydrolyze GTP, keeping this mol. switch in a constitutively active GTP-bound form that drives, unchecked, oncogenic downstream signaling.  One strategy to reduce the levels of active RAS is to target guanine nucleotide exchange factors, which allow RAS to cycle from the inactive GDP-bound state to the active GTP-bound form.  Here, we describe the identification of potent and cell-active small-mol. inhibitors which efficiently disrupt the interaction between KRAS and its exchange factor SOS1, a mode of action confirmed by a series of biophys. techniques.  The binding sites, mode of action, and selectivity were elucidated using crystal structures of KRASG12C-SOS1, SOS1, and SOS2.  By preventing formation of the KRAS-SOS1 complex, these inhibitors block reloading of KRAS with GTP, leading to antiproliferative activity.  The final compd. 23(I, BAY-293) selectively inhibits the KRAS-SOS1 interaction with an IC50 of 21 nM and is a valuable chem. probe for future investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTMAcBZFHkBrVg90H21EOLACvtfcHk0lhQW4j-2promg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivVajs7w%253D&md5=f3150723049c15986cea1d7f93cb89af</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1812963116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1812963116%26sid%3Dliteratum%253Aachs%26aulast%3DHillig%26aufirst%3DR.%2BC.%26aulast%3DSautier%26aufirst%3DB.%26aulast%3DSchroeder%26aufirst%3DJ.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DHilpmann%26aufirst%3DA.%26aulast%3DStegmann%26aufirst%3DC.%2BM.%26aulast%3DWerbeck%26aufirst%3DN.%2BD.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DBoemer%26aufirst%3DU.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DMastouri%26aufirst%3DJ.%26aulast%3DPetersen%26aufirst%3DK.%26aulast%3DSiemeister%26aufirst%3DG.%26aulast%3DKahmann%26aufirst%3DJ.%2BD.%26aulast%3DWegener%26aufirst%3DD.%26aulast%3DB%25C3%25B6hnke%26aufirst%3DN.%26aulast%3DEis%26aufirst%3DK.%26aulast%3DGraham%26aufirst%3DK.%26aulast%3DWortmann%26aufirst%3DL.%26aulast%3Dvon%2BNussbaum%26aufirst%3DF.%26aulast%3DBader%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520Potent%2520SOS1%2520Inhibitors%2520that%2520Block%2520RAS%2520Activation%2520via%2520Disruption%2520of%2520the%2520RAS%25E2%2580%2593SOS1%2520Interaction%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D2551%26epage%3D2560%26doi%3D10.1073%2Fpnas.1812963116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbott, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Little, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salovich, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howes, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akan, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browning, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobolik, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beesetty, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadtmueller, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossanese, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovery and Structure-Based Optimization of Benzimidazole-Derived Activators of SOS1-Mediated Nucleotide Exchange on RAS</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8875</span>– <span class="NLM_lpage">8894</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01108</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01108" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs12isrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8875-8894&author=T.+R.+Hodgesauthor=J.+R.+Abbottauthor=A.+J.+Littleauthor=D.+Sarkarauthor=J.+M.+Salovichauthor=J.+E.+Howesauthor=D.+T.+Akanauthor=J.+Saiauthor=A.+L.+Arnoldauthor=C.+Browningauthor=M.+C.+Burnsauthor=T.+Sobolikauthor=Q.+Sunauthor=Y.+Beesettyauthor=J.+A.+Cokerauthor=D.+Scharnauthor=H.+Stadtmuellerauthor=O.+W.+Rossaneseauthor=J.+Phanauthor=A.+G.+Watersonauthor=D.+B.+McConnellauthor=S.+W.+Fesik&title=Discovery+and+Structure-Based+Optimization+of+Benzimidazole-Derived+Activators+of+SOS1-Mediated+Nucleotide+Exchange+on+RAS&doi=10.1021%2Facs.jmedchem.8b01108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Structure-Based Optimization of Benzimidazole-Derived Activators of SOS1-Mediated Nucleotide Exchange on RAS</span></div><div class="casAuthors">Hodges, Timothy R.; Abbott, Jason R.; Little, Andrew J.; Sarkar, Dhruba; Salovich, James M.; Howes, Jennifer E.; Akan, Denis T.; Sai, Jiqing; Arnold, Allison L.; Browning, Carrie; Burns, Michael C.; Sobolik, Tammy; Sun, Qi; Beesetty, Yugandhar; Coker, Jesse A.; Scharn, Dirk; Stadtmueller, Heinz; Rossanese, Olivia W.; Phan, Jason; Waterson, Alex G.; McConnell, Darryl B.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8875-8894</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Son of sevenless homolog 1 (SOS1) is a guanine nucleotide exchange factor that catalyzes the exchange of GDP for GTP on RAS.  In its active form, GTP-bound RAS is responsible for numerous crit. cellular processes.  Aberrant RAS activity is involved in ∼30% of all human cancers; hence, SOS1 is an attractive therapeutic target for its role in modulating RAS activation.  Here, we describe a new series of benzimidazole-derived SOS1 agonists.  Using structure-guided design, we discovered small mols. that increase nucleotide exchange on RAS in vitro at submicromolar concns., bind to SOS1 with low double-digit nanomolar affinity, rapidly enhance cellular RAS-GTP levels, and invoke biphasic signaling changes in phosphorylation of ERK 1/2.  These compds. represent the most potent series of SOS1 agonists reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnQbL1ADAtRrVg90H21EOLACvtfcHk0lg4R42Sp5sN4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs12isrnL&md5=3051827da87f015c260000ea7b2c7c34</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01108%26sid%3Dliteratum%253Aachs%26aulast%3DHodges%26aufirst%3DT.%2BR.%26aulast%3DAbbott%26aufirst%3DJ.%2BR.%26aulast%3DLittle%26aufirst%3DA.%2BJ.%26aulast%3DSarkar%26aufirst%3DD.%26aulast%3DSalovich%26aufirst%3DJ.%2BM.%26aulast%3DHowes%26aufirst%3DJ.%2BE.%26aulast%3DAkan%26aufirst%3DD.%2BT.%26aulast%3DSai%26aufirst%3DJ.%26aulast%3DArnold%26aufirst%3DA.%2BL.%26aulast%3DBrowning%26aufirst%3DC.%26aulast%3DBurns%26aufirst%3DM.%2BC.%26aulast%3DSobolik%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DBeesetty%26aufirst%3DY.%26aulast%3DCoker%26aufirst%3DJ.%2BA.%26aulast%3DScharn%26aufirst%3DD.%26aulast%3DStadtmueller%26aufirst%3DH.%26aulast%3DRossanese%26aufirst%3DO.%2BW.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DWaterson%26aufirst%3DA.%2BG.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520and%2520Structure-Based%2520Optimization%2520of%2520Benzimidazole-Derived%2520Activators%2520of%2520SOS1-Mediated%2520Nucleotide%2520Exchange%2520on%2520RAS%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8875%26epage%3D8894%26doi%3D10.1021%2Facs.jmedchem.8b01108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miwa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niida, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sameshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamaura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonemori, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, J-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of a Human K-Ras G12D Mutant in Complex with GDP and the Cyclic Inhibitory Peptide KRpep-2d</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00128</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00128" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVWrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=732-736&author=S.+Sogabeauthor=Y.+Kamadaauthor=M.+Miwaauthor=A.+Niidaauthor=T.+Sameshimaauthor=M.+Kamauraauthor=K.+Yonemoriauthor=S.+Sasakiauthor=J-i.+Sakamotoauthor=K.+Sakamoto&title=Crystal+Structure+of+a+Human+K-Ras+G12D+Mutant+in+Complex+with+GDP+and+the+Cyclic+Inhibitory+Peptide+KRpep-2d&doi=10.1021%2Facsmedchemlett.7b00128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of a Human K-Ras G12D Mutant in Complex with GDP and the Cyclic Inhibitory Peptide KRpep-2d</span></div><div class="casAuthors">Sogabe, Satoshi; Kamada, Yusuke; Miwa, Masanori; Niida, Ayumu; Sameshima, Tomoya; Kamaura, Masahiro; Yonemori, Kazuko; Sasaki, Shigekazu; Sakamoto, Jun-ichi; Sakamoto, Kotaro</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ras proteins play roles in cell differentiation, proliferation, and survival.  Aberrant signaling through Ras-mediated pathways in tumor cells occurs as a result of several types of mutational damage, which most frequently affects the amino acids Gly-12, Gly-13, and Gln-61.  Recently, KRpep-2d was identified as a K-Ras(G12D) selective inhibitory peptide against the G12D mutant of K-Ras, which is a key member of the Ras protein family and an attractive cancer therapeutic target.  Here, the crystal structure of the human K-Ras(G12D) mutant was detd. in complex with GDP and KRpep-2d at 1.25 Å resoln.  This structure revealed that the peptide binds near switch II and allosterically blocks protein-protein interactions with the guanine nucleotide exchange factor.  This discovery of a unique binding pocket provides valuable information that will facilitate the design of direct Ras inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomKfkAMpR_QrVg90H21EOLACvtfcHk0lg4R42Sp5sN4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVWrtL4%253D&md5=0658bfe5c7e48365e4d44b71011b374a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00128%26sid%3Dliteratum%253Aachs%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DMiwa%26aufirst%3DM.%26aulast%3DNiida%26aufirst%3DA.%26aulast%3DSameshima%26aufirst%3DT.%26aulast%3DKamaura%26aufirst%3DM.%26aulast%3DYonemori%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DS.%26aulast%3DSakamoto%26aufirst%3DJ-i.%26aulast%3DSakamoto%26aufirst%3DK.%26atitle%3DCrystal%2520Structure%2520of%2520a%2520Human%2520K-Ras%2520G12D%2520Mutant%2520in%2520Complex%2520with%2520GDP%2520and%2520the%2520Cyclic%2520Inhibitory%2520Peptide%2520KRpep-2d%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D732%26epage%3D736%26doi%3D10.1021%2Facsmedchemlett.7b00128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGregor, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerwin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Expanding the Scope of Electrophiles Capable of Targeting K-Ras Oncogenes</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3178</span>– <span class="NLM_lpage">3183</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.7b00271</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.7b00271" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVGrsbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=3178-3183&author=L.+M.+McGregorauthor=M.+L.+Jenkinsauthor=C.+Kerwinauthor=J.+E.+Burkeauthor=K.+M.+Shokat&title=Expanding+the+Scope+of+Electrophiles+Capable+of+Targeting+K-Ras+Oncogenes&doi=10.1021%2Facs.biochem.7b00271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding the Scope of Electrophiles Capable of Targeting K-Ras Oncogenes</span></div><div class="casAuthors">McGregor, Lynn M.; Jenkins, Meredith L.; Kerwin, Caitlin; Burke, John E.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">3178-3183</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is growing interest in reversible and irreversible covalent inhibitors that target noncatalytic amino acids in target proteins.  With a goal of targeting oncogenic K-Ras variants (e.g., G12D) by expanding the types of amino acids that can be targeted by covalent inhibitors, we survey a set of electrophiles for their ability to label carboxylates.  We functionalized an optimized ligand for the K-Ras switch II pocket with a set of electrophiles previously reported to react with carboxylates and characterized the ability of these compds. to react with model nucleophiles and oncogenic K-Ras proteins.  Here, we report that aziridines and stabilized diazo groups preferentially react with free carboxylates over thiols.  Although we did not identify a warhead that potently labels K-Ras G12D, we were able to study the interactions of many electrophiles with K-Ras, as most of the electrophiles rapidly label K-Ras G12C.  We characterized the resulting complexes by crystallog., hydrogen/deuterium exchange, and differential scanning fluorometry.  Our results both demonstrate the ability of a noncatalytic cysteine to react with a diverse set of electrophiles and emphasize the importance of proper spatial arrangements between a covalent inhibitor and its intended nucleophile.  We hope that these results can expand the range of electrophiles and nucleophiles of use in covalent protein modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcvKWYdfIxw7Vg90H21EOLACvtfcHk0lg4R42Sp5sN4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVGrsbnI&md5=0b25b65210248a4cad2916efba90ddae</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.7b00271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.7b00271%26sid%3Dliteratum%253Aachs%26aulast%3DMcGregor%26aufirst%3DL.%2BM.%26aulast%3DJenkins%26aufirst%3DM.%2BL.%26aulast%3DKerwin%26aufirst%3DC.%26aulast%3DBurke%26aufirst%3DJ.%2BE.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DExpanding%2520the%2520Scope%2520of%2520Electrophiles%2520Capable%2520of%2520Targeting%2520K-Ras%2520Oncogenes%26jtitle%3DBiochemistry%26date%3D2017%26volume%3D56%26spage%3D3178%26epage%3D3183%26doi%3D10.1021%2Facs.biochem.7b00271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manandhar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gondi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westover, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Covalent Guanosine Mimetic Inhibitors of G12C KRAS</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFyntrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=61-66&author=Y.+Xiongauthor=J.+Luauthor=J.+Hunterauthor=L.+Liauthor=D.+Scottauthor=H.+G.+Choiauthor=S.+M.+Limauthor=A.+Manandharauthor=S.+Gondiauthor=T.+Simauthor=K.+D.+Westoverauthor=N.+S.+Gray&title=Covalent+Guanosine+Mimetic+Inhibitors+of+G12C+KRAS&doi=10.1021%2Facsmedchemlett.6b00373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Guanosine Mimetic Inhibitors of G12C KRAS</span></div><div class="casAuthors">Xiong, Yuan; Lu, Jia; Hunter, John; Li, Lianbo; Scott, David; Choi, Hwan Geun; Lim, Sang Min; Manandhar, Anuj; Gondi, Sudershan; Sim, Taebo; Westover, Kenneth D.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">61-66</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ras proteins are members of a large family of GTPase enzymes that are commonly mutated in cancer where they act as dominant oncogenes.  The authors previously developed an irreversible guanosine-derived inhibitor, SML-8-73-1, of mutant G12C RAS which forms a covalent bond with cysteine 12.  Here the authors report exploration of the structure-activity relationships (SAR) of hydrolytically stable analogs of SML-8-73-1 as covalent G12C KRAS inhibitors.  The authors report the discovery of difluoromethylene bisphosphonate analogs such as compd. I, which despite exhibiting reduced efficiency as covalent G12C KRAS inhibitors, remove the liability of the hydrolytic instability of the diphosphate moiety present in SML-8-73-1 and provide the foundation for development of prodrugs to facilitate cellular uptake.  The SAR and crystallog. results reaffirm the exquisite mol. recognition that exists in the diphosphate-region of RAS for guanosine nucleotides which must be considered in the design of nucleotide-competitive inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraZMoq5jUr_rVg90H21EOLACvtfcHk0liil0-u-dwSWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFyntrbF&md5=b36c23e91f8a3c3f30aa9819d931870f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00373%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DHunter%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DScott%26aufirst%3DD.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DLim%26aufirst%3DS.%2BM.%26aulast%3DManandhar%26aufirst%3DA.%26aulast%3DGondi%26aufirst%3DS.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DWestover%26aufirst%3DK.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DCovalent%2520Guanosine%2520Mimetic%2520Inhibitors%2520of%2520G12C%2520KRAS%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D61%26epage%3D66%26doi%3D10.1021%2Facsmedchemlett.6b00373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ostrem, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sos, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">K-Ras(G12C) Inhibitors Allosterically Control GTP Affinity and Effector Interactions</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">548</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.1038/nature12796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1038%2Fnature12796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=24256730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=548-551&author=J.+M.+Ostremauthor=U.+Petersauthor=M.+L.+Sosauthor=J.+A.+Wellsauthor=K.+M.+Shokat&title=K-Ras%28G12C%29+Inhibitors+Allosterically+Control+GTP+Affinity+and+Effector+Interactions&doi=10.1038%2Fnature12796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span></div><div class="casAuthors">Ostrem, Jonathan M.; Peters, Ulf; Sos, Martin L.; Wells, James A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7477</span>),
    <span class="NLM_cas:pages">548-551</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Somatic mutations in the small GTPase K-Ras are the most common activating lesions found in human cancer, and are generally assocd. with poor response to std. therapies.  Efforts to target this oncogene directly have faced difficulties owing to its picomolar affinity for GTP/GDP and the absence of known allosteric regulatory sites.  Oncogenic mutations result in functional activation of Ras family proteins by impairing GTP hydrolysis.  With diminished regulation by GTPase activity, the nucleotide state of Ras becomes more dependent on relative nucleotide affinity and concn.  This gives GTP an advantage over GDP and increases the proportion of active GTP-bound Ras.  Here we report the development of small mols. that irreversibly bind to a common oncogenic mutant, K-Ras(G12C).  These compds. rely on the mutant cysteine for binding and therefore do not affect the wild-type protein.  Crystallog. studies reveal the formation of a new pocket that is not apparent in previous structures of Ras, beneath the effector binding switch-II region.  Binding of these inhibitors to K-Ras(G12C) disrupts both switch-I and switch-II, subverting the native nucleotide preference to favor GDP over GTP and impairing binding to Raf.  Our data provide structure-based validation of a new allosteric regulatory site on Ras that is targetable in a mutant-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokkgcVXuBn_rVg90H21EOLACvtfcHk0liil0-u-dwSWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO&md5=27c1e7d8ef6b7e4b2919e9f00a073923</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature12796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12796%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DK-Ras%2528G12C%2529%2520Inhibitors%2520Allosterically%2520Control%2520GTP%2520Affinity%2520and%2520Effector%2520Interactions%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D548%26epage%3D551%26doi%3D10.1038%2Fnature12796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janes, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucharski, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ely, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firdaus, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibition of Oncogenic KRAS Output with Small Molecules Targeting the Inactive State</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-1105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1158%2F2159-8290.CD-15-1105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=26739882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVyru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=316-329&author=M.+P.+Patricelliauthor=M.+R.+Janesauthor=L.-S.+Liauthor=R.+Hansenauthor=U.+Petersauthor=L.+V.+Kesslerauthor=Y.+Chenauthor=J.+M.+Kucharskiauthor=J.+Fengauthor=T.+Elyauthor=J.+H.+Chenauthor=S.+J.+Firdausauthor=A.+Babbarauthor=P.+Renauthor=Y.+Liu&title=Selective+Inhibition+of+Oncogenic+KRAS+Output+with+Small+Molecules+Targeting+the+Inactive+State&doi=10.1158%2F2159-8290.CD-15-1105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Oncogenic KRAS Output with Small Molecules Targeting the Inactive State</span></div><div class="casAuthors">Patricelli, Matthew P.; Janes, Matthew R.; Li, Lian-Sheng; Hansen, Rasmus; Peters, Ulf; Kessler, Linda V.; Chen, Yuching; Kucharski, Jeff M.; Feng, Jun; Ely, Tess; Chen, Jeffrey H.; Firdaus, Sarah J.; Babbar, Anjali; Ren, Pingda; Liu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">316-329</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">KRAS gain-of-function mutations occur in approx. 30% of all human cancers.  Despite more than 30 years of KRAS-focused research and development efforts, no targeted therapy has been discovered for cancers with KRAS mutations.  Here, we describe ARS-853, a selective, covalent inhibitor of KRASG12C that inhibits mutant KRAS-driven signaling by binding to the GDP-bound oncoprotein and preventing activation.  Based on the rates of engagement and inhibition obsd. for ARS-853, along with a mutant-specific mass spectrometry-based assay for assessing KRAS activation status, we show that the nucleotide state of KRASG12C is in a state of dynamic flux that can be modulated by upstream signaling factors.  These studies provide convincing evidence that the KRASG12C mutation generates a "hyperexcitable" rather than a "statically active" state and that targeting the inactive, GDP-bound form is a promising approach for generating novel anti-RAS therapeutics.  Significance: A cell-active, mutant-specific, covalent inhibitor of KRASG12C is described that targets the GDP-bound, inactive state and prevents subsequent activation.  Using this novel compd., we demonstrate that KRASG12C oncoprotein rapidly cycles bound nucleotide and responds to upstream signaling inputs to maintain a highly active state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJJA-9IIHeN7Vg90H21EOLACvtfcHk0liUfKCGMao0Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVyru7s%253D&md5=41db2f02006e8f0be7a9bb47354addcd</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-1105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-1105%26sid%3Dliteratum%253Aachs%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DLi%26aufirst%3DL.-S.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DKessler%26aufirst%3DL.%2BV.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DKucharski%26aufirst%3DJ.%2BM.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DEly%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DFirdaus%26aufirst%3DS.%2BJ.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DSelective%2520Inhibition%2520of%2520Oncogenic%2520KRAS%2520Output%2520with%2520Small%2520Molecules%2520Targeting%2520the%2520Inactive%2520State%26jtitle%3DCancer%2520Discovery%26date%3D2016%26volume%3D6%26spage%3D316%26epage%3D329%26doi%3D10.1158%2F2159-8290.CD-15-1105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janes, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firdaus, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darjania, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarieh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ely, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucharski, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brehmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu-Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting KRAS Mutant Cancers with a Covalent G12C-Specific Inhibitor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">578</span>– <span class="NLM_lpage">589</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1016%2Fj.cell.2018.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=29373830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1cXislyisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2018&pages=578-589&author=M.+R.+Janesauthor=J.+Zhangauthor=L.-S.+Liauthor=R.+Hansenauthor=U.+Petersauthor=X.+Guoauthor=Y.+Chenauthor=A.+Babbarauthor=S.+J.+Firdausauthor=L.+Darjaniaauthor=J.+Fengauthor=J.+H.+Chenauthor=S.+Liauthor=S.+Liauthor=Y.+O.+Longauthor=C.+Thachauthor=Y.+Liuauthor=A.+Zariehauthor=T.+Elyauthor=J.+M.+Kucharskiauthor=L.+V.+Kesslerauthor=T.+Wuauthor=K.+Yuauthor=Y.+Wangauthor=Y.+Yaoauthor=X.+Dengauthor=P.+P.+Zarrinkarauthor=D.+Brehmerauthor=D.+Dhanakauthor=M.+V.+Lorenziauthor=D.+Hu-Loweauthor=M.+P.+Patricelliauthor=P.+Renauthor=Y.+Liu&title=Targeting+KRAS+Mutant+Cancers+with+a+Covalent+G12C-Specific+Inhibitor&doi=10.1016%2Fj.cell.2018.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting KRAS Mutant Cancers with a Covalent G12C-Specific Inhibitor</span></div><div class="casAuthors">Janes, Matthew R.; Zhang, Jingchuan; Li, Lian-Sheng; Hansen, Rasmus; Peters, Ulf; Guo, Xin; Chen, Yuching; Babbar, Anjali; Firdaus, Sarah J.; Darjania, Levan; Feng, Jun; Chen, Jeffrey H.; Li, Shuangwei; Li, Shisheng; Long, Yun O.; Thach, Carol; Liu, Yuan; Zarieh, Ata; Ely, Tess; Kucharski, Jeff M.; Kessler, Linda V.; Wu, Tao; Yu, Ke; Wang, Yi; Yao, Yvonne; Deng, Xiaohu; Zarrinkar, Patrick P.; Brehmer, Dirk; Dhanak, Dashyant; Lorenzi, Matthew V.; Hu-Lowe, Dana; Patricelli, Matthew P.; Ren, Pingda; Liu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">578-589.e17</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">KRASG12C was recently identified to be potentially druggable by allele-specific covalent targeting of Cys 12 in vicinity to an inducible allosteric switch II pocket (S-IIP).  Success of this approach requires active cycling of KRASG12C between its active-GTP and inactive-GDP conformations as accessibility of the S-IIP is restricted only to the GDP-bound state.  This strategy proved feasible for inhibiting mutant KRAS in vitro; however, it is uncertain whether this approach would translate to in vivo.  Here, the authors describe structure-based design and identification of ARS-1620, a covalent compd. with high potency and selectivity for KRASG12C.  ARS-1620 achieves rapid and sustained in vivo target occupancy to induce tumor regression.  The authors use ARS-1620 to dissect oncogenic KRAS dependency and demonstrate that monolayer culture formats significantly underestimate KRAS dependency in vivo.  This study provides in vivo evidence that mutant KRAS can be selectively targeted and reveals ARS-1620 as representing a new generation of KRASG12C-specific inhibitors with promising therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVmv4KAl4_dLVg90H21EOLACvtfcHk0liUfKCGMao0Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXislyisb8%253D&md5=d0883bde981b51e41a794179ffae87be</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.-S.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DGuo%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DFirdaus%26aufirst%3DS.%2BJ.%26aulast%3DDarjania%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLong%26aufirst%3DY.%2BO.%26aulast%3DThach%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZarieh%26aufirst%3DA.%26aulast%3DEly%26aufirst%3DT.%26aulast%3DKucharski%26aufirst%3DJ.%2BM.%26aulast%3DKessler%26aufirst%3DL.%2BV.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DHu-Lowe%26aufirst%3DD.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DTargeting%2520KRAS%2520Mutant%2520Cancers%2520with%2520a%2520Covalent%2520G12C-Specific%2520Inhibitor%26jtitle%3DCell%26date%3D2018%26volume%3D172%26spage%3D578%26epage%3D589%26doi%3D10.1016%2Fj.cell.2018.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janes, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">The Reactivity-driven Biochemical Mechanism of Covalent KRAS<sup>G12C</sup> Inhibitors</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">454</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.1038/s41594-018-0061-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1038%2Fs41594-018-0061-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=29760531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKqu7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=454-462&author=R.+Hansenauthor=U.+Petersauthor=A.+Babbarauthor=Y.+Chenauthor=J.+Fengauthor=M.+R.+Janesauthor=L.-S.+Liauthor=P.+Renauthor=Y.+Liuauthor=P.+P.+Zarrinkar&title=The+Reactivity-driven+Biochemical+Mechanism+of+Covalent+KRASG12C+Inhibitors&doi=10.1038%2Fs41594-018-0061-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The reactivity-driven biochemical mechanism of covalent KRASG12C inhibitors</span></div><div class="casAuthors">Hansen, Rasmus; Peters, Ulf; Babbar, Anjali; Chen, Yuching; Feng, Jun; Janes, Matthew R.; Li, Lian-Sheng; Ren, Pingda; Liu, Yi; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">454-462</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Activating mutations in KRAS are among the most common tumor driver mutations.  Until recently, KRAS had been considered undruggable with small mols.; the discovery of the covalent KRASG12C inhibitors ARS-853 and ARS-1620 has demonstrated that it is feasible to inhibit KRAS with high potency in cells and animals.  Although the biol. activity of these inhibitors has been described, the biochem. mechanism of how the compds. achieve potent inhibition remained incompletely understood.  We now show that the activity of ARS-853 and ARS-1620 is primarily driven by KRAS-mediated catalysis of the chem. reaction with Cys12 in human KRASG12C, while the reversible binding affinity is weak, in the hundreds of micromolar or higher range.  The mechanism resolves how an induced, shallow and dynamic pocket not expected to support high-affinity binding of small mols. can nevertheless be targeted with potent inhibitors and may be applicable to other targets conventionally considered undruggable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrya_A4ZYnOu7Vg90H21EOLACvtfcHk0lgmSVRthg-mcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKqu7nL&md5=b3f9198f55998306f3b881c256be22a9</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41594-018-0061-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-018-0061-5%26sid%3Dliteratum%253Aachs%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DLi%26aufirst%3DL.-S.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DThe%2520Reactivity-driven%2520Biochemical%2520Mechanism%2520of%2520Covalent%2520KRASG12C%2520Inhibitors%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D454%26epage%3D462%26doi%3D10.1038%2Fs41594-018-0061-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lonsdale, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span> <span> </span><span class="NLM_article-title">Structure-based Design of Targeted Covalent Inhibitors</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">3816</span>– <span class="NLM_lpage">3830</span>, <span class="refDoi"> DOI: 10.1039/C7CS00220C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1039%2FC7CS00220C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=29620097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntVGiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2018&pages=3816-3830&author=R.+Lonsdaleauthor=R.+A.+Ward&title=Structure-based+Design+of+Targeted+Covalent+Inhibitors&doi=10.1039%2FC7CS00220C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of targeted covalent inhibitors</span></div><div class="casAuthors">Lonsdale, Richard; Ward, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3816-3830</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Covalent inhibition is a rapidly growing discipline within drug discovery.  Many historical covalent inhibitors were discovered by serendipity, with such a mechanism of action often regarded as undesirable due to potential toxicity issues.  Recent progress has seen a major shift in this outlook, as covalent inhibition shows promise for targets where previous efforts to identify non-covalent small mol. inhibitors have failed.  Targeted covalent inhibitors (TCIs) can offer drug discovery scientists the ability to increase the potency and/or selectivity of small mol. inhibitors, by attachment of reactive functional groups designed to covalently bind to specific sites in a target.  In this tutorial review we introduce the broader concept of covalent inhibition, discuss the potential benefits and challenges of such an approach, and provide an overview of the current status of the field.  We also describe some strategies and computational tools to enable successful covalent drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnSIGMuS3vm7Vg90H21EOLACvtfcHk0lhk4Y0D2IYXZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntVGiurs%253D&md5=a7e31b23cf8ecfcad1397ad10ea10b7b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1039%2FC7CS00220C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CS00220C%26sid%3Dliteratum%253Aachs%26aulast%3DLonsdale%26aufirst%3DR.%26aulast%3DWard%26aufirst%3DR.%2BA.%26atitle%3DStructure-based%2520Design%2520of%2520Targeted%2520Covalent%2520Inhibitors%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2018%26volume%3D47%26spage%3D3816%26epage%3D3830%26doi%3D10.1039%2FC7CS00220C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lanman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amegadzie, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Booker, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frohn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopecky, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minatti, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickrell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegmund, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamayo, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tegley, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walton, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurz, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Achanta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartberger, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollis, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rex, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiki, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San Miguel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volak, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittington, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zech, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cee, V. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Covalent Inhibitor of KRASG12C (AMG 510) for the Treatment of Solid Tumors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">52</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01180</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01180" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlGitrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=52-65&author=B.+A.+Lanmanauthor=J.+R.+Allenauthor=J.+G.+Allenauthor=A.+K.+Amegadzieauthor=K.+S.+Ashtonauthor=S.+K.+Bookerauthor=J.+J.+Chenauthor=N.+Chenauthor=M.+J.+Frohnauthor=G.+Goodmanauthor=D.+J.+Kopeckyauthor=L.+Liuauthor=P.+Lopezauthor=J.+D.+Lowauthor=V.+Maauthor=A.+E.+Minattiauthor=T.+T.+Nguyenauthor=N.+Nishimuraauthor=A.+J.+Pickrellauthor=A.+B.+Reedauthor=Y.+Shinauthor=A.+C.+Siegmundauthor=N.+A.+Tamayoauthor=C.+M.+Tegleyauthor=M.+C.+Waltonauthor=H.-L.+Wangauthor=R.+P.+Wurzauthor=M.+Xueauthor=K.+C.+Yangauthor=P.+Achantaauthor=M.+D.+Bartbergerauthor=J.+Canonauthor=L.+S.+Hollisauthor=J.+D.+McCarterauthor=C.+Mohrauthor=K.+Rexauthor=A.+Y.+Saikiauthor=T.+San+Miguelauthor=L.+P.+Volakauthor=K.+H.+Wangauthor=D.+A.+Whittingtonauthor=S.+G.+Zechauthor=J.+R.+Lipfordauthor=V.+J.+Cee&title=Discovery+of+a+Covalent+Inhibitor+of+KRASG12C+%28AMG+510%29+for+the+Treatment+of+Solid+Tumors&doi=10.1021%2Facs.jmedchem.9b01180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Covalent Inhibitor of KRASG12C (AMG 510) for the Treatment of Solid Tumors</span></div><div class="casAuthors">Lanman, Brian A.; Allen, Jennifer R.; Allen, John G.; Amegadzie, Albert K.; Ashton, Kate S.; Booker, Shon K.; Chen, Jian Jeffrey; Chen, Ning; Frohn, Michael J.; Goodman, Guy; Kopecky, David J.; Liu, Longbin; Lopez, Patricia; Low, Jonathan D.; Ma, Vu; Minatti, Ana E.; Nguyen, Thomas T.; Nishimura, Nobuko; Pickrell, Alexander J.; Reed, Anthony B.; Shin, Youngsook; Siegmund, Aaron C.; Tamayo, Nuria A.; Tegley, Christopher M.; Walton, Mary C.; Wang, Hui-Ling; Wurz, Ryan P.; Xue, May; Yang, Kevin C.; Achanta, Pragathi; Bartberger, Michael D.; Canon, Jude; Hollis, L. Steven; McCarter, John D.; Mohr, Christopher; Rex, Karen; Saiki, Anne Y.; San Miguel, Tisha; Volak, Laurie P.; Wang, Kevin H.; Whittington, Douglas A.; Zech, Stephan G.; Lipford, J. Russell; Cee, Victor J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">52-65</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KRASG12C has emerged as a promising target in the treatment of solid tumors.  Covalent inhibitors targeting the mutant cysteine-12 residue have been shown to disrupt signaling by this long-"undruggable" target; however clin. viable inhibitors have yet to be identified.  Here, we report efforts to exploit a cryptic pocket (H95/Y96/Q99) we identified in KRASG12C to identify inhibitors suitable for clin. development.  Structure-based design efforts leading to the identification of a novel quinazolinone scaffold are described, along with optimization efforts that overcame a configurational stability issue arising from restricted rotation about an axially chiral biaryl bond.  Biopharmaceutical optimization of the resulting leads culminated in the identification of AMG 510, a highly potent, selective, and well-tolerated KRASG12C inhibitor currently in phase I clin. trials (NCT03600883).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGeWqRRXX6lbVg90H21EOLACvtfcHk0lg7fAhZC6RI1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlGitrrK&md5=d973189be70bb913020edb4989f6ba7d</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01180%26sid%3Dliteratum%253Aachs%26aulast%3DLanman%26aufirst%3DB.%2BA.%26aulast%3DAllen%26aufirst%3DJ.%2BR.%26aulast%3DAllen%26aufirst%3DJ.%2BG.%26aulast%3DAmegadzie%26aufirst%3DA.%2BK.%26aulast%3DAshton%26aufirst%3DK.%2BS.%26aulast%3DBooker%26aufirst%3DS.%2BK.%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DChen%26aufirst%3DN.%26aulast%3DFrohn%26aufirst%3DM.%2BJ.%26aulast%3DGoodman%26aufirst%3DG.%26aulast%3DKopecky%26aufirst%3DD.%2BJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLopez%26aufirst%3DP.%26aulast%3DLow%26aufirst%3DJ.%2BD.%26aulast%3DMa%26aufirst%3DV.%26aulast%3DMinatti%26aufirst%3DA.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BT.%26aulast%3DNishimura%26aufirst%3DN.%26aulast%3DPickrell%26aufirst%3DA.%2BJ.%26aulast%3DReed%26aufirst%3DA.%2BB.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DSiegmund%26aufirst%3DA.%2BC.%26aulast%3DTamayo%26aufirst%3DN.%2BA.%26aulast%3DTegley%26aufirst%3DC.%2BM.%26aulast%3DWalton%26aufirst%3DM.%2BC.%26aulast%3DWang%26aufirst%3DH.-L.%26aulast%3DWurz%26aufirst%3DR.%2BP.%26aulast%3DXue%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DK.%2BC.%26aulast%3DAchanta%26aufirst%3DP.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DCanon%26aufirst%3DJ.%26aulast%3DHollis%26aufirst%3DL.%2BS.%26aulast%3DMcCarter%26aufirst%3DJ.%2BD.%26aulast%3DMohr%26aufirst%3DC.%26aulast%3DRex%26aufirst%3DK.%26aulast%3DSaiki%26aufirst%3DA.%2BY.%26aulast%3DSan%2BMiguel%26aufirst%3DT.%26aulast%3DVolak%26aufirst%3DL.%2BP.%26aulast%3DWang%26aufirst%3DK.%2BH.%26aulast%3DWhittington%26aufirst%3DD.%2BA.%26aulast%3DZech%26aufirst%3DS.%2BG.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26aulast%3DCee%26aufirst%3DV.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520Covalent%2520Inhibitor%2520of%2520KRASG12C%2520%2528AMG%2520510%2529%2520for%2520the%2520Treatment%2520of%2520Solid%2520Tumors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D52%26epage%3D65%26doi%3D10.1021%2Facs.jmedchem.9b01180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marx, M. A.</span>; <span class="NLM_string-name">Baer, B. R.</span>; <span class="NLM_string-name">Ballard, J. K.</span>; <span class="NLM_string-name">Blake, J. F.</span>; <span class="NLM_string-name">Bouhana, K.</span>; <span class="NLM_string-name">Briere, D.</span>; <span class="NLM_string-name">Burgess, L. E.</span>; <span class="NLM_string-name">Burkhard, M.</span>; <span class="NLM_string-name">Chiang, H.</span>; <span class="NLM_string-name">Chicarelli, M. J.</span>; <span class="NLM_string-name">Christensen, J. G.</span>; <span class="NLM_string-name">Fischer, J. P.</span>; <span class="NLM_string-name">Hallin, J.</span>; <span class="NLM_string-name">Mejia, M. J.</span>; <span class="NLM_string-name">Olson, P.</span>; <span class="NLM_string-name">Savechenkov, P.</span>; <span class="NLM_string-name">Sudhakar, N.</span>; <span class="NLM_string-name">Tang, T. P.</span>; <span class="NLM_string-name">Vigers, G. P.</span>; <span class="NLM_string-name">Fell, J. B.</span></span> <span> </span><span class="NLM_article-title">Structure-based Drug Discovery of MRTX1257: A Selective, Covalent KRAS G12C Inhibitor with Oral Activity in Animal Models of Cancer</span>. MEDI-0244,  <i>Abstracts of Papers, 257th ACS National Meeting & Exposition, Orlando, FL, March 31–April 4, 2019</i>; <span class="NLM_publisher-name">American Chemical Society</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=M.+A.+Marx&author=B.+R.+Baer&author=J.+K.+Ballard&author=J.+F.+Blake&author=K.+Bouhana&author=D.+Briere&author=L.+E.+Burgess&author=M.+Burkhard&author=H.+Chiang&author=M.+J.+Chicarelli&author=J.+G.+Christensen&author=J.+P.+Fischer&author=J.+Hallin&author=M.+J.+Mejia&author=P.+Olson&author=P.+Savechenkov&author=N.+Sudhakar&author=T.+P.+Tang&author=G.+P.+Vigers&author=J.+B.+Fell&title=Structure-based+Drug+Discovery+of+MRTX1257%3A+A+Selective%2C+Covalent+KRAS+G12C+Inhibitor+with+Oral+Activity+in+Animal+Models+of+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DMarx%26aufirst%3DM.%2BA.%26atitle%3DStructure-based%2520Drug%2520Discovery%2520of%2520MRTX1257%253A%2520A%2520Selective%252C%2520Covalent%2520KRAS%2520G12C%2520Inhibitor%2520with%2520Oral%2520Activity%2520in%2520Animal%2520Models%2520of%2520Cancer%26jtitle%3DAbstracts%2520of%2520Papers%252C%2520257th%2520ACS%2520National%2520Meeting%2520%2526%2520Exposition%252C%2520Orlando%252C%2520FL%252C%2520March%252031%25E2%2580%2593April%25204%252C%25202019%26pub%3DAmerican%2520Chemical%2520Society%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span>; <span class="NLM_string-name">Feng, J.</span>; <span class="NLM_string-name">Wu, T.</span>; <span class="NLM_string-name">Ren, P.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Long, O. Y.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of KRAS G12C</span>.  <i>PCT Int. Appl.</i> <span class="NLM_patent">WO2015054572</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=L.+Li&author=J.+Feng&author=T.+Wu&author=P.+Ren&author=Y.+Liu&author=Y.+Liu&author=O.+Y.+Long&title=Inhibitors+of+KRAS+G12C"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26atitle%3DInhibitors%2520of%2520KRAS%2520G12C%26jtitle%3DPCT%2520Int.%2520Appl.%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feru, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gero, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrogio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catalano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westover, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Potent and Selective Covalent Quinazoline Inhibitors of KRAS G12C</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1005</span>– <span class="NLM_lpage">1016</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.06.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1016%2Fj.chembiol.2017.06.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=28781124" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1yltr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1005-1016&author=M.+Zengauthor=J.+Luauthor=L.+Liauthor=F.+Feruauthor=C.+Quanauthor=T.+W.+Geroauthor=S.+B.+Ficarroauthor=Y.+Xiongauthor=C.+Ambrogioauthor=R.+M.+Paranalauthor=M.+Catalanoauthor=J.+Shaoauthor=K.-K.+Wongauthor=J.+A.+Martoauthor=E.+S.+Fischerauthor=P.+A.+J%C3%A4nneauthor=D.+A.+Scottauthor=K.+D.+Westoverauthor=N.+S.+Gray&title=Potent+and+Selective+Covalent+Quinazoline+Inhibitors+of+KRAS+G12C&doi=10.1016%2Fj.chembiol.2017.06.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective covalent quinazoline inhibitors of kras g12c</span></div><div class="casAuthors">Zeng, Mei; Lu, Jia; Li, Lianbo; Feru, Frederic; Quan, Chunshan; Gero, Thomas W.; Ficarro, Scott B.; Xiong, Yuan; Ambrogio, Chiara; Paranal, Raymond M.; Catalano, Marco; Shao, Jay; Wong, Kwok-Kin; Marto, Jarrod A.; Fischer, Eric S.; Janne, Pasi A.; Scott, David A.; Westover, Kenneth D.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1005-1016.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Targeted covalent small mols. have shown promise for cancers driven by KRAS G12C.  Allosteric compds. that access an inducible pocket formed by movement of a dynamic structural element in KRAS, switch II, have been reported, but these compds. require further optimization to enable their advancement into clin. development.  We demonstrate that covalent quinazoline-based switch II pocket (SIIP) compds. effectively suppress GTP loading of KRAS G12C, MAPK phosphorylation, and the growth of cancer cells harboring G12C.  Notably we find that adding an amide substituent to the quinazoline scaffold allows addnl. interactions with KRAS G12C, and remarkably increases the labeling efficiency, potency, and selectivity of KRAS G12C inhibitors.  Structural studies using X-ray crystallog. reveal a new conformation of SIIP and key interactions made by substituents located at the quinazoline 2-, 4-, and 7-positions.  Optimized lead compds. in the quinazoline series selectively inhibit KRAS G12C-dependent signaling and cancer cell growth at sub-micromolar concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxY0-7NkfIeLVg90H21EOLACvtfcHk0liInqWT8DhEBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1yltr3N&md5=e793972cb711c5a8c0db46d8288697af</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.06.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.06.017%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DFeru%26aufirst%3DF.%26aulast%3DQuan%26aufirst%3DC.%26aulast%3DGero%26aufirst%3DT.%2BW.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DAmbrogio%26aufirst%3DC.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DCatalano%26aufirst%3DM.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DWong%26aufirst%3DK.-K.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DScott%26aufirst%3DD.%2BA.%26aulast%3DWestover%26aufirst%3DK.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DPotent%2520and%2520Selective%2520Covalent%2520Quinazoline%2520Inhibitors%2520of%2520KRAS%2520G12C%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1005%26epage%3D1016%26doi%3D10.1016%2Fj.chembiol.2017.06.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodnarchuk, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span> <span> </span><span class="NLM_article-title">Water Sites, Networks, and Free Energies with Grand Canonical Monte Carlo</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">14930</span>– <span class="NLM_lpage">14943</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b07940</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b07940" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCjurbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=14930-14943&author=G.+A.+Rossauthor=M.+S.+Bodnarchukauthor=J.+W.+Essex&title=Water+Sites%2C+Networks%2C+and+Free+Energies+with+Grand+Canonical+Monte+Carlo&doi=10.1021%2Fjacs.5b07940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Water Sites, Networks, And Free Energies with Grand Canonical Monte Carlo</span></div><div class="casAuthors">Ross, Gregory A.; Bodnarchuk, Michael S.; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">14930-14943</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water mols. play integral roles in the formation of many protein-ligand complexes, and recent computational efforts have been focused on predicting the thermodn. properties of individual waters and how they may be exploited in rational drug design.  However, when water mols. form highly coupled hydrogen-bonding networks, there is, as yet, no method that can rigorously calc. the free energy to bind the entire network or assess the degree of cooperativity between waters.  In this work, we report theor. and methodol. developments to the grand canonical Monte Carlo simulation technique.  Central to our results is a rigorous equation that can be used to calc. efficiently the binding free energies of water networks of arbitrary size and complexity.  Using a single set of simulations, our methods can locate waters, est. their binding affinities, capture the cooperativity of the water network, and evaluate the hydration free energy of entire protein binding sites.  Our techniques have been applied to multiple test systems and compare favorably to thermodn. integration simulations and exptl. data.  The implications of these methods in drug design are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8mGD8Ul3D0rVg90H21EOLACvtfcHk0liInqWT8DhEBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCjurbJ&md5=0040dc4201610620f2b64fc6c41998a9</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b07940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b07940%26sid%3Dliteratum%253Aachs%26aulast%3DRoss%26aufirst%3DG.%2BA.%26aulast%3DBodnarchuk%26aufirst%3DM.%2BS.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DWater%2520Sites%252C%2520Networks%252C%2520and%2520Free%2520Energies%2520with%2520Grand%2520Canonical%2520Monte%2520Carlo%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D14930%26epage%3D14943%26doi%3D10.1021%2Fjacs.5b07940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bodnarchuk, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span> <span> </span><span class="NLM_article-title">Strategies to Calculate Water Binding Free Energies in Protein-Ligand Complexes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1623</span>– <span class="NLM_lpage">1633</span>, <span class="refDoi"> DOI: 10.1021/ci400674k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci400674k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltF2lu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1623-1633&author=M.+S.+Bodnarchukauthor=R.+Vinerauthor=J.+Michelauthor=J.+W.+Essex&title=Strategies+to+Calculate+Water+Binding+Free+Energies+in+Protein-Ligand+Complexes&doi=10.1021%2Fci400674k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies to Calculate Water Binding Free Energies in Protein-Ligand Complexes</span></div><div class="casAuthors">Bodnarchuk, Michael S.; Viner, Russell; Michel, Julien; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1623-1633</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water mols. are commonplace in protein binding pockets, where they can typically form a complex between the protein and a ligand or become displaced upon ligand binding.  As a result, it is often of great interest to establish both the binding free energy and location of such mols.  Several approaches to predicting the location and affinity of water mols. to proteins have been proposed and utilized in the literature, although it is often unclear which method should be used under what circumstances.  We report here a comparison between three such methodologies, Just Add Water Mols. (JAWS), Grand Canonical Monte Carlo (GCMC), and double-decoupling, in the hope of understanding the advantages and limitations of each method when applied to enclosed binding sites.  As a result, we have adapted the JAWS scoring procedure, allowing the binding free energies of strongly bound water mols. to be calcd. to a high degree of accuracy, requiring significantly less computational effort than more rigorous approaches.  The combination of JAWS and GCMC offers a route to a rapid scheme capable of both locating and scoring water mols. for rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVFExfkQzj-LVg90H21EOLACvtfcHk0liInqWT8DhEBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltF2lu7g%253D&md5=c4174f31e4c0a3e3856ac4b68213342b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci400674k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci400674k%26sid%3Dliteratum%253Aachs%26aulast%3DBodnarchuk%26aufirst%3DM.%2BS.%26aulast%3DViner%26aufirst%3DR.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DStrategies%2520to%2520Calculate%2520Water%2520Binding%2520Free%2520Energies%2520in%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1623%26epage%3D1633%26doi%3D10.1021%2Fci400674k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bodnarchuk, M. S.</span></span> <span> </span><span class="NLM_article-title">Water, Water, Everywhere··· It’s Time to Stop and Think</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1139</span>– <span class="NLM_lpage">1146</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1016%2Fj.drudis.2016.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=27210724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC28XoslWltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1139-1146&author=M.+S.+Bodnarchuk&title=Water%2C+Water%2C+Everywhere%C2%B7%C2%B7%C2%B7+It%E2%80%99s+Time+to+Stop+and+Think&doi=10.1016%2Fj.drudis.2016.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Water, water, everywhere... It's time to stop and think</span></div><div class="casAuthors">Bodnarchuk, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1139-1146</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Despite the numerous methods available for predicting the location and affinity of water mols., there is still a degree of scepticism and reluctance towards using such information within a drug discovery program.  Here, I review some of the most common and popular methods to assess whether these apparent concerns are justified.  I suggest that the field is approaching maturity and that some methods are capable of giving quant. predictions, which are confirmed exptl.  This suggests that water-placement methods should be used more widely to help direct chem. efforts, although more successful examples are required to help validate the techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxfKWV5PXTkrVg90H21EOLACvtfcHk0linqphhEChuCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XoslWltb4%253D&md5=dc2a72fa764bb19cd0c0ca2921fc974a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DBodnarchuk%26aufirst%3DM.%2BS.%26atitle%3DWater%252C%2520Water%252C%2520Everywhere%25C2%25B7%25C2%25B7%25C2%25B7%2520It%25E2%2580%2599s%2520Time%2520to%2520Stop%2520and%2520Think%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D1139%26epage%3D1146%26doi%3D10.1016%2Fj.drudis.2016.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span>; <span class="NLM_string-name">Feng, J.</span>; <span class="NLM_string-name">Long, O. Y.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Wu, T.</span>; <span class="NLM_string-name">Ren, P.</span>; <span class="NLM_string-name">Liu, Y.</span></span>; <span> </span><span class="NLM_article-title">Substituted Quinazoline Compounds and Methods of Use Thereof</span>.  <i>PCT Int. Appl.</i> <span class="NLM_patent">WO2016164675</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=L.+Li&author=J.+Feng&author=O.+Y.+Long&author=Y.+Liu&author=T.+Wu&author=P.+Ren&author=Y.+Liu&title=Substituted+Quinazoline+Compounds+and+Methods+of+Use+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26atitle%3DSubstituted%2520Quinazoline%2520Compounds%2520and%2520Methods%2520of%2520Use%2520Thereof%26jtitle%3DPCT%2520Int.%2520Appl.%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Solca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanderson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Himmelsbach, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haaksma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adolf, G. R.</span></span> <span> </span><span class="NLM_article-title">Afatinib, an Irreversible Inhibitor of EGFR, HER2, and ErbB4</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">350</span>, <span class="refDoi"> DOI: 10.1124/jpet.112.197756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1124%2Fjpet.112.197756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=22888144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKktbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2012&pages=342-350&author=F.+Solcaauthor=G.+Dahlauthor=A.+Zoephelauthor=G.+Baderauthor=M.+Sandersonauthor=C.+Kleinauthor=O.+Kraemerauthor=F.+Himmelsbachauthor=E.+Haaksmaauthor=G.+R.+Adolf&title=Afatinib%2C+an+Irreversible+Inhibitor+of+EGFR%2C+HER2%2C+and+ErbB4&doi=10.1124%2Fjpet.112.197756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Target binding properties and cellular activity of afatinib (BIBW 2992), an irreversible ErbB family blocker</span></div><div class="casAuthors">Solca, Flavio; Dahl, Goeran; Zoephel, Andreas; Bader, Gerd; Sanderson, Michael; Klein, Christian; Kraemer, Oliver; Himmelsbach, Frank; Haaksma, Eric; Adolf, Guenther R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">342-350</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Deregulation of the ErbB (proto-oncogene B of the avian erythroblastosis virus AEV-H strain) receptor network is well recognized as an oncogenic driver in epithelial cancers.  Several targeted drugs have been developed, including antibodies and small-mol. kinase inhibitors, each of them characterized by distinct patterns of ErbB receptor interactions.  Understanding the precise pharmacol. properties of these compds. is important for optimal use in clin. practice.  Afatinib [BIBW 2992; N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[[(3S)-tetrahydro-3-furanyl]oxy]-6-quinazolinyl]-4-(dimethylamino)-2-butenamide] is an ATP-competitive anilinoquinazoline deriv. harboring a reactive acrylamide group.  It was designed to covalently bind and irreversibly block enzymically active ErbB receptor family members.  Here, we show by X-ray crystallog. the covalent binding of afatinib to wild-type epidermal growth factor receptor (EGFR) and by mass spectrometry the covalent interaction with EGFR, EGFRL858R/T790M, human epidermal growth factor receptor 2 (HER2), and ErbB-4.  Afatinib potently inhibits the enzymatic activity of ErbB-4 (EC50 = 1 nM) and the proliferation of cancer cell lines driven by multiple ErbB receptor aberrations at concns. below 100 nM.  N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[[(3S)-tetrahydro-3-furanyl]oxy]-6-quinazolinyl]-4-(dimethylamino)-2-butanamide (BI 37781), a close analog of afatinib lacking the acrylamide group and thus incapable of covalent bond formation, had similar potency on cells driven by EGFR or EGFRL858R, but less or no detectable activity on cells expressing EGFRL858R/T790M HER2 or ErbB-4.  These results stress the importance of the acrylamide group and show that afatinib differs from approved ErbB targeting agents by irreversibly inhibiting the kinase activity of all ErbB family members.  They provide a mechanistic rationale for the distinct pharmacol. features of this compd. and explain the clin. activity seen in some patients who are resistant to antibody or kinase inhibitor therapy because of secondary mutations or ErbB receptor "reprogramming.".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofXgQqaKXv-bVg90H21EOLACvtfcHk0linqphhEChuCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFKktbzE&md5=4a57df32abe8101f287fc6251f63f92d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1124%2Fjpet.112.197756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.112.197756%26sid%3Dliteratum%253Aachs%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DDahl%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DSanderson%26aufirst%3DM.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DKraemer%26aufirst%3DO.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DHaaksma%26aufirst%3DE.%26aulast%3DAdolf%26aufirst%3DG.%2BR.%26atitle%3DAfatinib%252C%2520an%2520Irreversible%2520Inhibitor%2520of%2520EGFR%252C%2520HER2%252C%2520and%2520ErbB4%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2012%26volume%3D343%26spage%3D342%26epage%3D350%26doi%3D10.1124%2Fjpet.112.197756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finlay, M. R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butterworth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colclough, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, D. A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currie, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grist, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassall, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinowska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarland, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orme, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richmond, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrigley, G. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Potent and Selective EGFR Inhibitor (AZD9291) of Both Sensitizing and T790M Resistance Mutations That Spares the Wild Type Form of the Receptor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8249</span>– <span class="NLM_lpage">8267</span>, <span class="refDoi"> DOI: 10.1021/jm500973a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500973a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1KhurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8249-8267&author=M.+R.+V.+Finlayauthor=M.+Andertonauthor=S.+Ashtonauthor=P.+Ballardauthor=P.+A.+Bethelauthor=M.+R.+Boxauthor=R.+H.+Bradburyauthor=S.+J.+Brownauthor=S.+Butterworthauthor=A.+Campbellauthor=C.+Chorleyauthor=N.+Colcloughauthor=D.+A.+E.+Crossauthor=G.+S.+Currieauthor=M.+Gristauthor=L.+Hassallauthor=G.+B.+Hillauthor=D.+Jamesauthor=M.+Jamesauthor=P.+Kemmittauthor=T.+Klinowskaauthor=G.+Lamontauthor=S.+G.+Lamontauthor=N.+Martinauthor=H.+L.+McFarlandauthor=M.+J.+Mellorauthor=J.+P.+Ormeauthor=D.+Perkinsauthor=P.+Perkinsauthor=G.+Richmondauthor=P.+Smithauthor=R.+A.+Wardauthor=M.+J.+Waringauthor=D.+Whittakerauthor=S.+Wellsauthor=G.+L.+Wrigley&title=Discovery+of+a+Potent+and+Selective+EGFR+Inhibitor+%28AZD9291%29+of+Both+Sensitizing+and+T790M+Resistance+Mutations+That+Spares+the+Wild+Type+Form+of+the+Receptor&doi=10.1021%2Fjm500973a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective EGFR Inhibitor (AZD9291) of Both Sensitizing and T790M Resistance Mutations That Spares the Wild Type Form of the Receptor</span></div><div class="casAuthors">Finlay, M. Raymond V.; Anderton, Mark; Ashton, Susan; Ballard, Peter; Bethel, Paul A.; Box, Matthew R.; Bradbury, Robert H.; Brown, Simon J.; Butterworth, Sam; Campbell, Andrew; Chorley, Christopher; Colclough, Nicola; Cross, Darren A. E.; Currie, Gordon S.; Grist, Matthew; Hassall, Lorraine; Hill, George B.; James, Daniel; James, Michael; Kemmitt, Paul; Klinowska, Teresa; Lamont, Gillian; Lamont, Scott G.; Martin, Nathaniel; McFarland, Heather L.; Mellor, Martine J.; Orme, Jonathon P.; Perkins, David; Perkins, Paula; Richmond, Graham; Smith, Peter; Ward, Richard A.; Waring, Michael J.; Whittaker, David; Wells, Stuart; Wrigley, Gail L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8249-8267</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) inhibitors have been used clin. in the treatment of nonsmall-cell lung cancer (NSCLC) patients harboring sensitizing (or activating) mutations for a no. of years.  Despite encouraging clin. efficacy with these agents, in many patients resistance develops leading to disease progression.  In most cases, this resistance is in the form of the T790M mutation.  In addn., EGFR wild type receptor inhibition inherent with these agents can lead to dose limiting toxicities of rash and diarrhea.  The authors describe herein the evolution of an early, mutant selective lead to the clin. candidate AZD9291 I, an irreversible inhibitor of both EGFR sensitizing (EGFRm+) and T790M resistance mutations with selectivity over the wild type form of the receptor.  Following observations of significant tumor inhibition in preclin. models, the clin. candidate was administered clin. to patients with T790M pos. EGFR-TKI resistant NSCLC and early efficacy has been obsd., accompanied by an encouraging safety profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouPAWG9N0Ro7Vg90H21EOLACvtfcHk0lgYMP08vyTnjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1KhurrI&md5=db9db624bb4324c2e22b1ab12035263b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm500973a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500973a%26sid%3Dliteratum%253Aachs%26aulast%3DFinlay%26aufirst%3DM.%2BR.%2BV.%26aulast%3DAnderton%26aufirst%3DM.%26aulast%3DAshton%26aufirst%3DS.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DBethel%26aufirst%3DP.%2BA.%26aulast%3DBox%26aufirst%3DM.%2BR.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DBrown%26aufirst%3DS.%2BJ.%26aulast%3DButterworth%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DA.%26aulast%3DChorley%26aufirst%3DC.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DCross%26aufirst%3DD.%2BA.%2BE.%26aulast%3DCurrie%26aufirst%3DG.%2BS.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DHassall%26aufirst%3DL.%26aulast%3DHill%26aufirst%3DG.%2BB.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DJames%26aufirst%3DM.%26aulast%3DKemmitt%26aufirst%3DP.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DLamont%26aufirst%3DG.%26aulast%3DLamont%26aufirst%3DS.%2BG.%26aulast%3DMartin%26aufirst%3DN.%26aulast%3DMcFarland%26aufirst%3DH.%2BL.%26aulast%3DMellor%26aufirst%3DM.%2BJ.%26aulast%3DOrme%26aufirst%3DJ.%2BP.%26aulast%3DPerkins%26aufirst%3DD.%26aulast%3DPerkins%26aufirst%3DP.%26aulast%3DRichmond%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWells%26aufirst%3DS.%26aulast%3DWrigley%26aufirst%3DG.%2BL.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520and%2520Selective%2520EGFR%2520Inhibitor%2520%2528AZD9291%2529%2520of%2520Both%2520Sensitizing%2520and%2520T790M%2520Resistance%2520Mutations%2520That%2520Spares%2520the%2520Wild%2520Type%2520Form%2520of%2520the%2520Receptor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8249%26epage%3D8267%26doi%3D10.1021%2Fjm500973a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, M.</span></span> <span> </span><span class="NLM_article-title">The Role of Extrahepatic Metabolism in the Pharmacokinetics of the Targeted Covalent Inhibitors Afatinib, Ibrutinib, and Neratinib</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.1124/dmd.114.061424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1124%2Fdmd.114.061424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=25504185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1OmsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=375-384&author=Y.+Shibataauthor=M.+Chiba&title=The+Role+of+Extrahepatic+Metabolism+in+the+Pharmacokinetics+of+the+Targeted+Covalent+Inhibitors+Afatinib%2C+Ibrutinib%2C+and+Neratinib&doi=10.1124%2Fdmd.114.061424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The role of extrahepatic metabolism in the pharmacokinetics of the targeted covalent inhibitors afatinib, ibrutinib, and neratinib</span></div><div class="casAuthors">Shibata, Yoshihiro; Chiba, Masato</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">375-384, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Despite the fact that much progress has been made recently in the development of targeted covalent inhibitors (TCIs), their pharmacokinetics (PK) have not been well characterized in the light of extrahepatic clearance (CLextH) by glutathione (GSH)/glutathione S-transferase (GST)-dependent conjugation attributable to the unique electrophilic structure (e.g., acrylamide moiety) of TCI compds.  In the present study, CLextH values were examd. in rat, dog, and monkey to predict the contribution of CLextH to the PK of the TCIs afatinib, ibrutinib, and neratinib in humans.  Afatinib and neratinib both underwent extensive conjugation with GSH in buffer and cytosol fractions of liver and kidney, whereas ibrutinib showed much lower reactivity/susceptibility to GSH/GST-dependent conjugation.  The CLextH in each species was calcd. from the difference between obsd. total body clearance and predicted hepatic clearance (CLH) in cryopreserved hepatocytes suspended in 100% serum of the corresponding species.  The power-based simple allometry relating the CLextH for the unbound compd. to animal body wt. was applicable across species for afatinib and neratinib (R2 ≥ 0.9) but not for ibrutinib (R2 = 0.04).  The predicted AUC after oral administration of afatinib and neratinib agreed reasonably closely with reported values in phase I dose-escalation studies.  Comparisons of CLextH and CLH predicted that CLextH largely detd. the PK of afatinib (>90% as a proportion of total body clearance) and neratinib (∼34%) in humans.  The present method can serve as one of the tools for the optimization of PK in humans at the discovery stage for the development of TCI candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquWC6d6e6DU7Vg90H21EOLACvtfcHk0lgYMP08vyTnjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1OmsrY%253D&md5=eaac67ce7cea5f2432c5144922ba3dd2</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1124%2Fdmd.114.061424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.114.061424%26sid%3Dliteratum%253Aachs%26aulast%3DShibata%26aufirst%3DY.%26aulast%3DChiba%26aufirst%3DM.%26atitle%3DThe%2520Role%2520of%2520Extrahepatic%2520Metabolism%2520in%2520the%2520Pharmacokinetics%2520of%2520the%2520Targeted%2520Covalent%2520Inhibitors%2520Afatinib%252C%2520Ibrutinib%252C%2520and%2520Neratinib%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2015%26volume%3D43%26spage%3D375%26epage%3D384%26doi%3D10.1124%2Fdmd.114.061424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allocati, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masulli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Ilio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federici, L.</span></span> <span> </span><span class="NLM_article-title">Glutathione Transferases: Substrates, Inhibitors and Pro-drugs in Cancer and Neurodegenerative Diseases</span>. <i>Oncogenesis</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">8</span>, <span class="refDoi"> DOI: 10.1038/s41389-017-0025-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1038%2Fs41389-017-0025-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=29362397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A280%3ADC%252BC1MvjsFGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=8&author=N.+Allocatiauthor=M.+Masulliauthor=C.+Di+Ilioauthor=L.+Federici&title=Glutathione+Transferases%3A+Substrates%2C+Inhibitors+and+Pro-drugs+in+Cancer+and+Neurodegenerative+Diseases&doi=10.1038%2Fs41389-017-0025-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione transferases: substrates, inihibitors and pro-drugs in cancer and neurodegenerative diseases</span></div><div class="casAuthors">Allocati Nerino; Masulli Michele; Di Ilio Carmine; Federici Luca; Federici Luca</div><div class="citationInfo"><span class="NLM_cas:title">Oncogenesis</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8</span>
        ISSN:<span class="NLM_cas:issn">2157-9024</span>.
    </div><div class="casAbstract">Glutathione transferase classical GSH conjugation activity plays a critical role in cellular detoxification against xenobiotics and noxious compounds as well as against oxidative stress.  However, this feature is also exploited by cancer cells to acquire drug resistance and improve their survival.  As a result, various members of the family were found overexpressed in a number of different cancers.  Moreover several GST polymorphisms, ranging from null phenotypes to point mutations, were detected in members of the family and found to correlate with the onset of neuro-degenerative diseases.  In the last decades, a great deal of research aimed at clarifying the role played by GSTs in drug resistance, at developing inhibitors to counteract this activity but also at exploiting GSTs for prodrugs specific activation in cancer cells.  Here we summarize some of the most important achievements reached in this lively area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRVakKpC8MvO5VVjGoY74BafW6udTcc2ebkzmPDgaWi87ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvjsFGmtA%253D%253D&md5=bdaead4dda5737e83b117829a0d0ddee</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fs41389-017-0025-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41389-017-0025-3%26sid%3Dliteratum%253Aachs%26aulast%3DAllocati%26aufirst%3DN.%26aulast%3DMasulli%26aufirst%3DM.%26aulast%3DDi%2BIlio%26aufirst%3DC.%26aulast%3DFederici%26aufirst%3DL.%26atitle%3DGlutathione%2520Transferases%253A%2520Substrates%252C%2520Inhibitors%2520and%2520Pro-drugs%2520in%2520Cancer%2520and%2520Neurodegenerative%2520Diseases%26jtitle%3DOncogenesis%26date%3D2018%26volume%3D7%26spage%3D8%26doi%3D10.1038%2Fs41389-017-0025-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collisson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadanandam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibb, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinkle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakkula, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feiler, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olshen, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danenberg, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempero, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellman, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. W.</span></span> <span> </span><span class="NLM_article-title">Subtypes of Pancreatic Ductal Adenocarcinoma and their Differing Responses to Therapy</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">500</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1038/nm.2344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1038%2Fnm.2344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=21460848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFahtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=500-503&author=E.+A.+Collissonauthor=A.+Sadanandamauthor=P.+Olsonauthor=W.+J.+Gibbauthor=M.+Truittauthor=S.+Guauthor=J.+Coocauthor=J.+Weinkleauthor=G.+E.+Kimauthor=L.+Jakkulaauthor=H.+S.+Feilerauthor=A.+H.+Koauthor=A.+B.+Olshenauthor=K.+L.+Danenbergauthor=M.+A.+Temperoauthor=P.+T.+Spellmanauthor=D.+Hanahanauthor=J.+W.+Gray&title=Subtypes+of+Pancreatic+Ductal+Adenocarcinoma+and+their+Differing+Responses+to+Therapy&doi=10.1038%2Fnm.2344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Subtypes of pancreatic ductal adenocarcinoma and their differing responses to therapy</span></div><div class="casAuthors">Collisson, Eric A.; Sadanandam, Anguraj; Olson, Peter; Gibb, William J.; Truitt, Morgan; Gu, Shenda; Cooc, Janine; Weinkle, Jennifer; Kim, Grace E.; Jakkula, Lakshmi; Feiler, Heidi S.; Ko, Andrew H.; Olshen, Adam B.; Danenberg, Kathleen L.; Tempero, Margaret A.; Spellman, Paul T.; Hanahan, Douglas; Gray, Joe W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">500-503</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDA) is a lethal disease.  Overall survival is typically 6 mo from diagnosis.  Numerous phase 3 trials of agents effective in other malignancies have failed to benefit unselected PDA populations, although patients do occasionally respond.  Studies in other solid tumors have shown that heterogeneity in response is detd., in part, by mol. differences between tumors.  Furthermore, treatment outcomes are improved by targeting drugs to tumor subtypes in which they are selectively effective, with breast and lung cancers providing recent examples.  Identification of PDA mol. subtypes has been frustrated by a paucity of tumor specimens available for study.  We have overcome this problem by combined anal. of transcriptional profiles of primary PDA samples from several studies, along with human and mouse PDA cell lines.  We define three PDA subtypes: classical, quasimesenchymal and exocrine-like, and we present evidence for clin. outcome and therapeutic response differences between them.  We further define gene signatures for these subtypes that may have utility in stratifying patients for treatment and present preclin. model systems that may be used to identify new subtype specific therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-1m347gugF7Vg90H21EOLACvtfcHk0lgpYtRG2mGpMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFahtrY%253D&md5=fe6468e218cf05bb79e614a85ef01b4a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnm.2344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2344%26sid%3Dliteratum%253Aachs%26aulast%3DCollisson%26aufirst%3DE.%2BA.%26aulast%3DSadanandam%26aufirst%3DA.%26aulast%3DOlson%26aufirst%3DP.%26aulast%3DGibb%26aufirst%3DW.%2BJ.%26aulast%3DTruitt%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DS.%26aulast%3DCooc%26aufirst%3DJ.%26aulast%3DWeinkle%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DG.%2BE.%26aulast%3DJakkula%26aufirst%3DL.%26aulast%3DFeiler%26aufirst%3DH.%2BS.%26aulast%3DKo%26aufirst%3DA.%2BH.%26aulast%3DOlshen%26aufirst%3DA.%2BB.%26aulast%3DDanenberg%26aufirst%3DK.%2BL.%26aulast%3DTempero%26aufirst%3DM.%2BA.%26aulast%3DSpellman%26aufirst%3DP.%2BT.%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DGray%26aufirst%3DJ.%2BW.%26atitle%3DSubtypes%2520of%2520Pancreatic%2520Ductal%2520Adenocarcinoma%2520and%2520their%2520Differing%2520Responses%2520to%2520Therapy%26jtitle%3DNat.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D500%26epage%3D503%26doi%3D10.1038%2Fnm.2344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greninger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhodes, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koopman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Violette, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardeesy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span> <span> </span><span class="NLM_article-title">A Gene Expression Signature Associated with ″K-Ras Addiction″ Reveals Regulators of EMT and Tumor Cell Survival</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">489</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2009.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1016%2Fj.ccr.2009.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=19477428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BD1MXntlegtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=489-500&author=A.+Singhauthor=P.+Greningerauthor=D.+Rhodesauthor=L.+Koopmanauthor=S.+Violetteauthor=N.+Bardeesyauthor=J.+Settleman&title=A+Gene+Expression+Signature+Associated+with+%E2%80%B3K-Ras+Addiction%E2%80%B3+Reveals+Regulators+of+EMT+and+Tumor+Cell+Survival&doi=10.1016%2Fj.ccr.2009.03.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A gene expression signature associated with "K-ras addiction" reveals regulators of EMT and tumor cell survival</span></div><div class="casAuthors">Singh, Anurag; Greninger, Patricia; Rhodes, Daniel; Koopman, Louise; Violette, Sheila; Bardeesy, Nabeel; Settleman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">489-500</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">K-ras mutations occur frequently in epithelial cancers.  Using short hairpin RNAs to deplete K-Ras in lung and pancreatic cancer cell lines harboring K-ras mutations, two classes were identified-lines that do or do not require K-Ras to maintain viability.  Comparing these two classes of cancer cells revealed a gene expression signature in K-Ras-dependent cells, assocd. with a well-differentiated epithelial phenotype, which was also seen in primary tumors.  Several of these genes encode pharmacol. tractable proteins, such as Syk and Ron kinases and integrin β6, depletion of which induces epithelial-mesenchymal transformation (EMT) and apoptosis specifically in K-Ras-dependent cells.  These findings indicate that epithelial differentiation and tumor cell viability are assocd., and that EMT regulators in "K-Ras-addicted" cancers represent candidate therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFfjxR9oP1drVg90H21EOLACvtfcHk0lgpYtRG2mGpMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXntlegtb0%253D&md5=ebe63c5a069664f70eb88b285f1e3f6a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DGreninger%26aufirst%3DP.%26aulast%3DRhodes%26aufirst%3DD.%26aulast%3DKoopman%26aufirst%3DL.%26aulast%3DViolette%26aufirst%3DS.%26aulast%3DBardeesy%26aufirst%3DN.%26aulast%3DSettleman%26aufirst%3DJ.%26atitle%3DA%2520Gene%2520Expression%2520Signature%2520Associated%2520with%2520%25E2%2580%25B3K-Ras%2520Addiction%25E2%2580%25B3%2520Reveals%2520Regulators%2520of%2520EMT%2520and%2520Tumor%2520Cell%2520Survival%26jtitle%3DCancer%2520Cell%26date%3D2009%26volume%3D15%26spage%3D489%26epage%3D500%26doi%3D10.1016%2Fj.ccr.2009.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratilas, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Runswick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chresta, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormack, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockerill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beran, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassidy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggerty, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellison, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dering, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonazzi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravishankar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solit, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. D.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Pathway Signatures Predict MEK Addiction and Response to Selumetinib (AZD6244)</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">2264</span>– <span class="NLM_lpage">2273</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-09-1577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1158%2F0008-5472.CAN-09-1577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=20215513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFygt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=2264-2273&author=J.+R.+Dryauthor=S.+Paveyauthor=C.+A.+Pratilasauthor=C.+Harbronauthor=S.+Runswickauthor=D.+Hodgsonauthor=C.+Chrestaauthor=R.+McCormackauthor=N.+Byrneauthor=M.+Cockerillauthor=A.+Grahamauthor=G.+Beranauthor=A.+Cassidyauthor=C.+Haggertyauthor=H.+Brownauthor=G.+Ellisonauthor=J.+Deringauthor=B.+S.+Taylorauthor=M.+Starkauthor=V.+Bonazziauthor=S.+Ravishankarauthor=L.+Packerauthor=F.+Xingauthor=D.+B.+Solitauthor=R.+S.+Finnauthor=N.+Rosenauthor=N.+K.+Haywardauthor=T.+Frenchauthor=P.+D.+Smith&title=Transcriptional+Pathway+Signatures+Predict+MEK+Addiction+and+Response+to+Selumetinib+%28AZD6244%29&doi=10.1158%2F0008-5472.CAN-09-1577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional Pathway Signatures Predict MEK Addiction and Response to Selumetinib (AZD6244)</span></div><div class="casAuthors">Dry, Jonathan R.; Pavey, Sandra; Pratilas, Christine A.; Harbron, Chris; Runswick, Sarah; Hodgson, Darren; Chresta, Christine; McCormack, Rose; Byrne, Natalie; Cockerill, Mark; Graham, Alexander; Beran, Garry; Cassidy, Andrew; Haggerty, Carolyn; Brown, Helen; Ellison, Gillian; Dering, Judy; Taylor, Barry S.; Stark, Mitchell; Bonazzi, Vanessa; Ravishankar, Sugandha; Packer, Leisl; Xing, Feng; Solit, David B.; Finn, Richard S.; Rosen, Neal; Hayward, Nicholas K.; French, Tim; Smith, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2264-2273</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Selumetinib (AZD6244, ARRY-1α642886) is a selective, non-ATP-competitive inhibitor of mitogen-activated protein/extracellular signal-regulated kinase kinase (MEK)-1/2.  The range of antitumor activity seen preclinically and in patients highlights the importance of identifying determinants of response to this drug.  In large tumor cell panels of diverse lineage, we show that MEK inhibitor response does not have an abs. correlation with mutational or phospho-protein markers of BRAF/MEK, RAS, or phosphoinositide 3-kinase (PI3K) activity.  We aimed to enhance predictivity by measuring pathway output through coregulated gene networks displaying differential mRNA expression exclusive to resistant cell subsets and correlated to mutational or dynamic pathway activity.  We discovered an 18-gene signature enabling measurement of MEK functional output independent of tumor genotype.  Where the MEK pathway is activated but the cells remain resistant to selumetinib, we identified a 13-gene signature that implicates the existence of compensatory signaling from RAS effectors other than PI3K.  The ability of these signatures to stratify samples according to functional activation of MEK and/or selumetinib sensitivity was shown in multiple independent melanoma, colon, breast, and lung tumor cell lines and in xenograft models.  Furthermore, we were able to measure these signatures in fixed archival melanoma tumor samples using a single RT-qPCR-based test and found intergene correlations and assocns. with genetic markers of pathway activity to be preserved.  These signatures offer useful tools for the study of MEK biol. and clin. application of MEK inhibitors, and the novel approaches taken may benefit other targeted therapies.  Cancer Res; 70(6); 2264-73.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4oaXtcCNnKrVg90H21EOLACvtfcHk0ljkCWGph1fgag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFygt7o%253D&md5=fe1714e5398f21b3ae4a36863e598a9f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-09-1577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-09-1577%26sid%3Dliteratum%253Aachs%26aulast%3DDry%26aufirst%3DJ.%2BR.%26aulast%3DPavey%26aufirst%3DS.%26aulast%3DPratilas%26aufirst%3DC.%2BA.%26aulast%3DHarbron%26aufirst%3DC.%26aulast%3DRunswick%26aufirst%3DS.%26aulast%3DHodgson%26aufirst%3DD.%26aulast%3DChresta%26aufirst%3DC.%26aulast%3DMcCormack%26aufirst%3DR.%26aulast%3DByrne%26aufirst%3DN.%26aulast%3DCockerill%26aufirst%3DM.%26aulast%3DGraham%26aufirst%3DA.%26aulast%3DBeran%26aufirst%3DG.%26aulast%3DCassidy%26aufirst%3DA.%26aulast%3DHaggerty%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DH.%26aulast%3DEllison%26aufirst%3DG.%26aulast%3DDering%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DB.%2BS.%26aulast%3DStark%26aufirst%3DM.%26aulast%3DBonazzi%26aufirst%3DV.%26aulast%3DRavishankar%26aufirst%3DS.%26aulast%3DPacker%26aufirst%3DL.%26aulast%3DXing%26aufirst%3DF.%26aulast%3DSolit%26aufirst%3DD.%2BB.%26aulast%3DFinn%26aufirst%3DR.%2BS.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DHayward%26aufirst%3DN.%2BK.%26aulast%3DFrench%26aufirst%3DT.%26aulast%3DSmith%26aufirst%3DP.%2BD.%26atitle%3DTranscriptional%2520Pathway%2520Signatures%2520Predict%2520MEK%2520Addiction%2520and%2520Response%2520to%2520Selumetinib%2520%2528AZD6244%2529%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D2264%26epage%3D2273%26doi%3D10.1158%2F0008-5472.CAN-09-1577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ross, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revenko, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellston, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staniszewska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whalley, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckett, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monia, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakey, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macleod, A. R.</span></span> <span> </span><span class="NLM_article-title">Targeting KRAS-dependent Tumors with AZD4785, a High-affinity Therapeutic Antisense Oligonucleotide Inhibitor of KRAS</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">eaal5253</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.aal5253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=10.1126%2Fscitranslmed.aal5253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;key=28615361" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=eaal5253&author=S.+J.+Rossauthor=A.+S.+Revenkoauthor=L.+L.+Hansonauthor=R.+Ellstonauthor=A.+Staniszewskaauthor=N.+Whalleyauthor=S.+K.+Pandeyauthor=M.+Revillauthor=C.+Rooneyauthor=L.+K.+Buckettauthor=S.+K.+Kleinauthor=K.+Hudsonauthor=B.+P.+Moniaauthor=M.+Zindaauthor=D.+C.+Blakeyauthor=P.+D.+Lyneauthor=A.+R.+Macleod&title=Targeting+KRAS-dependent+Tumors+with+AZD4785%2C+a+High-affinity+Therapeutic+Antisense+Oligonucleotide+Inhibitor+of+KRAS&doi=10.1126%2Fscitranslmed.aal5253"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aal5253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aal5253%26sid%3Dliteratum%253Aachs%26aulast%3DRoss%26aufirst%3DS.%2BJ.%26aulast%3DRevenko%26aufirst%3DA.%2BS.%26aulast%3DHanson%26aufirst%3DL.%2BL.%26aulast%3DEllston%26aufirst%3DR.%26aulast%3DStaniszewska%26aufirst%3DA.%26aulast%3DWhalley%26aufirst%3DN.%26aulast%3DPandey%26aufirst%3DS.%2BK.%26aulast%3DRevill%26aufirst%3DM.%26aulast%3DRooney%26aufirst%3DC.%26aulast%3DBuckett%26aufirst%3DL.%2BK.%26aulast%3DKlein%26aufirst%3DS.%2BK.%26aulast%3DHudson%26aufirst%3DK.%26aulast%3DMonia%26aufirst%3DB.%2BP.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DBlakey%26aufirst%3DD.%2BC.%26aulast%3DLyne%26aufirst%3DP.%2BD.%26aulast%3DMacleod%26aufirst%3DA.%2BR.%26atitle%3DTargeting%2520KRAS-dependent%2520Tumors%2520with%2520AZD4785%252C%2520a%2520High-affinity%2520Therapeutic%2520Antisense%2520Oligonucleotide%2520Inhibitor%2520of%2520KRAS%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2017%26volume%3D9%26spage%3Deaal5253%26doi%3D10.1126%2Fscitranslmed.aal5253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FA1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FA1','PDB','6FA1'); return false;">PDB: 6FA1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GJ8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GJ8','PDB','6GJ8'); return false;">PDB: 6GJ8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EPV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EPV','PDB','4EPV'); return false;">PDB: 4EPV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DSU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DSU','PDB','4DSU'); return false;">PDB: 4DSU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4URZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4URZ','PDB','4URZ'); return false;">PDB: 4URZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4URU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4URU','PDB','4URU'); return false;">PDB: 4URU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6D5G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6D5G','PDB','6D5G'); return false;">PDB: 6D5G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EPM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EPM','PDB','6EPM'); return false;">PDB: 6EPM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OVI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OVI','PDB','5OVI'); return false;">PDB: 5OVI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XCO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XCO','PDB','5XCO'); return false;">PDB: 5XCO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5V6S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5V6S','PDB','5V6S'); return false;">PDB: 5V6S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KYK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KYK','PDB','5KYK'); return false;">PDB: 5KYK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4US2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4US2','PDB','4US2'); return false;">PDB: 4US2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5V','PDB','6T5V'); return false;">PDB: 6T5V</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5U" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5U','PDB','6T5U'); return false;">PDB: 6T5U</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5B" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5B','PDB','6T5B'); return false;">PDB: 6T5B</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i33"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01720">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_34376"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01720?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01720</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Protocols for the in vitro and cell assays and in vivo experiments, synthetic methods for the remaining examples, protein synthesis and purification, X-ray crystallographic structure information (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01720/suppl_file/jm9b01720_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings with associated data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01720/suppl_file/jm9b01720_si_001.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01720/suppl_file/jm9b01720_si_001.csv">jm9b01720_si_001.csv (1.46 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01720/suppl_file/jm9b01720_si_002.pdf">jm9b01720_si_002.pdf (505.15 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB depositions are available for compounds <b>25</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5B">6T5B</a>), <b>26</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5V">6T5V</a>), and <b>29</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5U">6T5U</a>). The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01720&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-9%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01720%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01720" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a802c39ae3c58","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
