
<html lang="en"     class="pb-page"  data-request-id="8e2876ae-8f2c-40e3-afcf-791f4d786051"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.7b00366;issue:issue:10.1021/jmcmar.2017.60.issue-9;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Predicting “Hot” and “Warm” Spots for Fragment Binding" /></meta><meta name="dc.Creator" content="Prakash Chandra  Rathi" /></meta><meta name="dc.Creator" content="R. Frederick  Ludlow" /></meta><meta name="dc.Creator" content="Richard J.  Hall" /></meta><meta name="dc.Creator" content="Christopher W.  Murray" /></meta><meta name="dc.Creator" content="Paul N.  Mortenson" /></meta><meta name="dc.Creator" content="Marcel L.  Verdonk" /></meta><meta name="dc.Description" content="Computational fragment mapping methods aim to predict hotspots on protein surfaces where small fragments will bind. Such methods are popular for druggability assessment as well as structure-based d..." /></meta><meta name="Description" content="Computational fragment mapping methods aim to predict hotspots on protein surfaces where small fragments will bind. Such methods are popular for druggability assessment as well as structure-based d..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 21, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00366" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00366" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00366" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00366" /></link>
        
    
    

<title>Predicting “Hot” and “Warm” Spots for Fragment Binding | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00366" /></meta><meta property="og:title" content="Predicting “Hot” and “Warm” Spots for Fragment Binding" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0008.jpeg" /></meta><meta property="og:description" content="Computational fragment mapping methods aim to predict hotspots on protein surfaces where small fragments will bind. Such methods are popular for druggability assessment as well as structure-based design. However, to date researchers developing or using such tools have had no clear way of assessing the performance of these methods. Here, we introduce the first diverse, high quality validation set for computational fragment mapping. The set contains 52 diverse examples of fragment binding “hot” and “warm” spots from the Protein Data Bank (PDB). Additionally, we describe PLImap, a novel protocol for fragment mapping based on the Protein–Ligand Informatics force field (PLIff). We evaluate PLImap against the new fragment mapping test set, and compare its performance to that of simple shape-based algorithms and fragment docking using GOLD. PLImap is made publicly available from https://bitbucket.org/AstexUK/pli." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00366"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00366">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00366&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00366&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00366&amp;href=/doi/10.1021/acs.jmedchem.7b00366" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4036-4046</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b00357" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b00374" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Predicting “Hot” and “Warm” Spots for Fragment Binding</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Prakash+Chandra++Rathi">Prakash Chandra Rathi</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=R.+Frederick++Ludlow">R. Frederick Ludlow</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Richard+J.++Hall">Richard J. Hall</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christopher+W.++Murray">Christopher W. Murray</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paul+N.++Mortenson">Paul N. Mortenson</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marcel+L.++Verdonk">Marcel L. Verdonk</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-6484-3328" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Astex Pharmaceuticals, 436 Cambridge Science Park, Milton Road, Cambridge CB4 0QA, United Kingdom</span></div><div class="corresp-info"><strong>*</strong>Phone: +44(0)1223 226200. E-mail: <a href="/cdn-cgi/l/email-protection#4b262a39282e27653d2e392f2425200b2a383f3365282426"><span class="__cf_email__" data-cfemail="89e4e8fbeaece5a7ffecfbede6e7e2c9e8fafdf1a7eae6e4">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00366&amp;href=/doi/10.1021%2Facs.jmedchem.7b00366" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4036–4046</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 4, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 March 2017</li><li><span class="item_label"><b>Published</b> online</span>21 April 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 May 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00366" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00366</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4036%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPrakash%2BChandra%2BRathi%252C%2BR.%2BFrederick%2BLudlow%252C%2BRichard%2BJ.%2BHall%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D9%26contentID%3Dacs.jmedchem.7b00366%26title%3DPredicting%2B%25E2%2580%259CHot%25E2%2580%259D%2Band%2B%25E2%2580%259CWarm%25E2%2580%259D%2BSpots%2Bfor%2BFragment%2BBinding%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4046%26publicationDate%3DMay%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00366"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2253</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">22</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00366" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Predicting “Hot” and “Warm” Spots for Fragment Binding&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Prakash&quot;,&quot;last_name&quot;:&quot;Chandra Rathi&quot;},{&quot;first_name&quot;:&quot;R.&quot;,&quot;last_name&quot;:&quot;Frederick Ludlow&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;J. Hall&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;W. Murray&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;N. Mortenson&quot;},{&quot;first_name&quot;:&quot;Marcel&quot;,&quot;last_name&quot;:&quot;L. Verdonk&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;21&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;4036-4046&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00366&quot;},&quot;abstract&quot;:&quot;Computational fragment mapping methods aim to predict hotspots on protein surfaces where small fragments will bind. Such methods are popular for druggability assessment as well as structure-based design. However, to date researchers developing or using such tools have had no clear way of assessing the performance of these methods. Here, we introduce the first diverse, high quality validation set for computational fragment mapping. The set contains 52 diverse examples of fragment binding “hot” and “warm” spots from the Protein Data Bank (PDB). Additionally, we describe PLImap, a novel protocol for fragment mapping based on the Protein–Ligand Informatics force field (PLIff). We evaluate PLImap against the new fragment mapping test set, and compare its performance to that of simple shape-based algorithms and fragment docking using GOLD. PLImap is made publicly available from https://bitbucket.org/AstexUK/pli.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00366&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00366" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00366&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00366" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00366&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00366" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00366&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00366&amp;href=/doi/10.1021/acs.jmedchem.7b00366" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00366" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00366" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00366%26sid%3Dliteratum%253Aachs%26pmid%3D28376303%26genre%3Darticle%26aulast%3DRathi%26date%3D2017%26atitle%3DPredicting%2B%25E2%2580%259CHot%25E2%2580%259D%2Band%2B%25E2%2580%259CWarm%25E2%2580%259D%2BSpots%2Bfor%2BFragment%2BBinding%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D9%26spage%3D4036%26epage%3D4046%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292289" title="Protein identification">Protein identification</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291918" title="Gold">Gold</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/jmcmar.2017.60.issue-9/20170511/jmcmar.2017.60.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Computational fragment mapping methods aim to predict hotspots on protein surfaces where small fragments will bind. Such methods are popular for druggability assessment as well as structure-based design. However, to date researchers developing or using such tools have had no clear way of assessing the performance of these methods. Here, we introduce the first diverse, high quality validation set for computational fragment mapping. The set contains 52 diverse examples of fragment binding “hot” and “warm” spots from the Protein Data Bank (PDB). Additionally, we describe PLImap, a novel protocol for fragment mapping based on the Protein–Ligand Informatics force field (PLIff). We evaluate PLImap against the new fragment mapping test set, and compare its performance to that of simple shape-based algorithms and fragment docking using GOLD. PLImap is made publicly available from <a href="https://bitbucket.org/AstexUK/pli" class="extLink">https://bitbucket.org/AstexUK/pli</a>.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41326" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41326" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Identifying fragment binding sites on a protein target of interest is a common first step in rational fragment-based drug discovery (FBDD).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Due to the small size and low complexity of the fragments screened in an FBDD campaign, chemical space can be sampled efficiently with a relatively small library<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> and the fragment hits frequently show relatively high ligand efficiency.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> To fully leverage the advantages of FBDD, even smaller (solvent-sized) organic molecules could be used to probe the target protein. To this end, Ringe et al. pioneered the Multiple Solvent Crystal Structure (MSCS) method wherein crystals of the target protein are soaked in high concentration solutions of a variety of solvent-sized molecules.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> MSCS provides vital information on the protein of interest by identifying sites where multiple solvent-sized molecules bind. Such sites should correspond to fragment binding “hot” spots (i.e., high-affinity sites where one would also expect hits from a “regular” fragment screen).<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a> In theory, exposing a protein to such high concentrations of very small fragments should not only identify the fragment “hot” spots but could also identify lower affinity sites. Such sites, or “warm” spots, could yield useful information for a fragment-growing or general structure-based design campaign.</div><div class="NLM_p">Unfortunately, MSCS cannot be employed routinely because not all systems are amenable to the harsh conditions (e.g., 1M fragment concentrations). Additionally, many fragments are not soluble enough to achieve the required concentrations. Computational fragment mapping approaches, however, can be applied to any fragment–protein system with structural information. These approaches can be roughly divided into three categories: (i) methods that generate interaction maps of atomic probes, (ii) methods that generate individual poses for molecular probes, and (iii) simulation-based methods that generate ensembles of poses for molecular probes. Two well established programs of the first type, GRID<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8-10)</a> and SuperStar,<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> generate maps of favorable positions for a variety of atomic probes on the target protein surfaces. GRID uses a force field based energy function,<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8-10)</a> whereas SuperStar<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> uses interaction propensities derived from the Cambridge Structural Database (CSD)<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and the Protein Data Bank (PDB; <a href="http://www.rcsb.org" class="extLink">www.rcsb.org</a>).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Hot spots on protein binding sites cannot always be directly identified from the atomic interaction maps because they are generated from single atom probes and hence the context of these atoms within a larger molecule is lost. To overcome this limitation, molecular probes are used to identify sites on the protein surfaces where small molecules bind favorably. The Multiple Copy Simultaneous Search (MCSS) technique places multiple copies of a probe molecule, which do not interact with each other, on the surface of a protein and subsequently energy minimizes them simultaneously to identify the local minima of the protein-probe complex.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> Another molecular probe based approach, FTMap, identifies low energy clusters of probe molecules on protein surfaces.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> A set of 16 small molecules are used to probe the target protein surface and hotspots are then identified as sites where the majority of probe molecules bind.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> Recently, Radoux et al. described a fragment mapping approach that combines SuperStar maps with a measure of depth/burial and uses three molecular probes to identify fragment hot spots.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Finally, both Molecular Dynamics (MD) and Monte Carlo (MC) simulations are used widely to identify hotspots on protein surfaces.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24 ref25">(21-25)</a> Such molecular simulations allow the generation of thermodynamic ensembles from which one can not only infer favorable binding positions of a molecular probe but can also estimate binding free energies associated with a cluster of poses. However, such simulations are computationally too expensive to be routinely used.</div><div class="NLM_p">Despite the range of computational fragment mapping protocols described in the literature, a diverse validation set for evaluating and comparing different approaches is not available. Consequently, these methods have only been tested on a few, select fragment–protein examples. Unfortunately, there are insufficient examples of high concentration solvent soaks available in the PDB to construct a fragment mapping validation set directly. However, the PDB has a wealth of information in the form of a variety of ligands bound to the same site of a protein. This information can be used to identify hot or warm spots as sites where the same moiety is placed by multiple different ligands. For example, to date there are no structures in the PDB of the resorcinol molecule (i.e., 1,3-dihydroxybenzene) bound to the ATP site of HSP90A. However, there are dozens of examples of HSP90A ligands that contain a resorcinol moiety, and in all cases, the resorcinol group is presented to the ATP pocket in the same way.<a onclick="showRef(event, 'ref26 ref27 ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29 ref30">(26-30)</a> For the purpose of this work we have assumed that, provided such moieties are placed in the same binding subpocket by chemically diverse ligands, such binding subpockets represent hot or warm spots for the fragment (i.e., resorcinol in the example above) binding. Hence, we set out to find all examples in the PDB where multiple dissimilar ligands place the same fragment in the same binding environment of a protein. The results of this exercise were systematically filtered and prioritized to construct the first high quality, diverse fragment mapping validation set, consisting of 52 protein–fragment complexes. Finally, we characterized each entry in the set as either a hot spot or a warm spot for fragment binding, based on the frequency with which the subsite is occupied by a ligand. Prepared structures of the fragment and the protein for each entry in the set are available for download from the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> (SI).</div><div class="NLM_p last">Additionally, we introduce PLImap, a novel, standalone computer program for fragment mapping. PLImap uses a systematic search algorithm, combined with the Protein–Ligand Informatics force field (PLIff),<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and is made publicly available from <a href="https://bitbucket.org/AstexUK/pli" class="extLink">https://bitbucket.org/AstexUK/pli</a>. We test the performance of PLImap against the new fragment mapping validation set and compare the results to those obtained with two simple shape-based algorithms and fragment docking using GOLD.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> We focus here on identifying “hot” and “warm” spots within the confines of a given binding site of a protein, as this is of particular interest to us in terms of generating ideas for fragment growing and merging in FBDD projects. However, other researchers may use our set to test global search protocols for identifying fragment binding sites across an entire protein surface.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78218" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78218" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Composition of the Validation Set</h3><div class="NLM_p">A detailed description of the protocols we followed for deriving the fragment mapping validation set is given in the “Methods” section and in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>, but in brief, we searched the PDB for examples where multiple diverse ligands present the same moiety (fragment) to the same subpocket of the same protein. During the automatic filtering and manual selection of protein–fragment complexes for the validation set, emphasis was given to constructing a diverse set. No more than one example of fragments having similar chemotypes bound to the same binding site was selected. For example, out of the 40 different clusters of 22 different fragments binding to proteins belonging to the kinase superfamily, only five clusters with chemically diverse fragments were selected. Similarly, only one example from the five clusters of carbonic anhydrase II (CA2) with different aryl-sulfonamides was selected. The final set comprises 41 unique fragments bound to 45 unique proteins (all pairwise sequence identities <50%) forming 52 unique fragment–protein complexes (see <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a> and <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Moreover, these 45 unique proteins encompass 41 unique sets of Pfam domains.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The resulting set contains many examples of well-known key combinations of chemotypes and target classes, including isoxazole against a bromodomain, benzamidine against a serine protease, and pyridine against a kinase.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical diagrams of the fragments in the validation set. The charge and tautomeric states of the fragments were assigned in their bound conformation in such a way that the hydrogen bonding network with their partner proteins is optimized. Note that fragments 15 and 16 differ only in their charge states.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Fragment Mapping Validation Set</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">complex ID</th><th class="colsep0 rowsep0" align="center">fragment ID<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">cluster size<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">category<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">PDB ID<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">resolution<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">protein name</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PFJ">4PFJ</a></td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">adenosylhomocysteinase</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">warm</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2F6V">2F6V</a></td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">protein-tyrosine phosphatase 1B</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VCQ">2VCQ</a></td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">hematopoietic prostaglandin D synthase</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FGD">3FGD</a></td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">thermolysin</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GEP">3GEP</a></td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">hypoxanthine-guanine phosphoribosyltransferase</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GBA">3GBA</a></td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">ionotropic glutamate receptor</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ARA">3ARA</a></td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">deoxyuridine 5′-triphosphate nucleotidohydrolase</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AP7">4AP7</a></td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">hepatocyte growth factor receptor</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MGA">4MGA</a></td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">estrogen receptor</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">warm</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WRB">4WRB</a></td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">macrophage migration inhibitory factor</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X7C">2X7C</a></td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">kinesin Eg5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1C83">1C83</a></td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">protein-tyrosine phosphatase 1B</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4F9Y">4F9Y</a></td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">mitogen-activated protein kinase p38 alpha</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">warm</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2W0B">2W0B</a></td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">cytochrome P450 51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N8E">4N8E</a></td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">dipeptidyl peptidase 4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OGN">4OGN</a></td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">E3 ubiquitin-protein ligase Mdm2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">warm</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AXA">4AXA</a></td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">PKA-Akt chimera</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">warm</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MXC">4MXC</a></td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">Hepatocyte growth factor receptor</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JYG">4JYG</a></td><td class="colsep0 rowsep0" align="left">2.4</td><td class="colsep0 rowsep0" align="left">retinoic acid receptor beta</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">warm</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UWE">3UWE</a></td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">aldo-keto reductase family 1 member C3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KVK">3KVK</a></td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">dihydroorotate dehydrogenase</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3N5S">3N5S</a></td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">endothelial Nitric oxide synthase</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">23</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2O7N">2O7N</a></td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">integrin alpha-L</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EPV">4EPV</a></td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">GTPase Kras</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">warm</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RAL">3RAL</a></td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">cyclin-dependent kinase 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3N3J">3N3J</a></td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">carbonic anhydrase 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BAK">4BAK</a></td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">prothrombin</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FMK">3FMK</a></td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">mitogen-activated protein kinase p38 alpha</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">29</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LDO">1LDO</a></td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">avidin</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GB9">4GB9</a></td><td class="colsep0 rowsep0" align="left">2.4</td><td class="colsep0 rowsep0" align="left">phosphatidylinositol 4,5-bisphosphate 3-kinase catalytic subunit gamma isoform</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">31</td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CWO">4CWO</a></td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">Heat shock protein HSP 90-alpha</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X8Z">2X8Z</a></td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">angiotensin-converting enzyme</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">33</td><td class="colsep0 rowsep0" align="left">23</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">warm</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3E51">3E51</a></td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">RNA-directed RNA polymerase</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OP1">4OP1</a></td><td class="colsep0 rowsep0" align="left">2.4</td><td class="colsep0 rowsep0" align="left">glucokinase regulatory protein</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">35</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2J94">2J94</a></td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">coagulation factor Xa</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BW4">4BW4</a></td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">bromodomain-containing protein 4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">37</td><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HOF">4HOF</a></td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">dihydrofolate reductase</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">38</td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">warm</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VTS">2VTS</a></td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">cyclin-dependent kinase 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">39</td><td class="colsep0 rowsep0" align="left">29</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3T4Q">3T4Q</a></td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">histidine kinase 4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">warm</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MSG">4MSG</a></td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">tankyrase-1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">41</td><td class="colsep0 rowsep0" align="left">31</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">warm</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KZA">4KZA</a></td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">beta-lactamase</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">42</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CJO">3CJO</a></td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">kinesin Eg5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">43</td><td class="colsep0 rowsep0" align="left">33</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DCH">4DCH</a></td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">glucokinase</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">44</td><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OIA">3OIA</a></td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">cytochrome P450cam</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">45</td><td class="colsep0 rowsep0" align="left">35</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PCS">4PCS</a></td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">alpha-<span class="smallcaps smallerCapital">l</span>-fucosidase</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">46</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GV1">4GV1</a></td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">protein kinase B alpha</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">47</td><td class="colsep0 rowsep0" align="left">37</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1W1Y">1W1Y</a></td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">chitinase</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">48</td><td class="colsep0 rowsep0" align="left">38</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GXL">3GXL</a></td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">transforming growth factor-beta receptor type I</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">49</td><td class="colsep0 rowsep0" align="left">39</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FFI">3FFI</a></td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">HIV-1 reverse transcriptase</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FRS">4FRS</a></td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">beta-secretase 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">51</td><td class="colsep0 rowsep0" align="left">41</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">hot</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3B25">3B25</a></td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">heat shock protein HSP 90-alpha</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">52</td><td class="colsep0 rowsep0" align="left">42</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">warm</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3A4P">3A4P</a></td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">hepatocyte growth factor receptor</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Fragment ID as shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">Number of PDB complexes with chemically diverse ligands that place the fragment in the same binding subpocket.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">The category of the subsite (hot or warm) where the fragment is bound (see section “Characterizing binding sub-pockets of the fragments” for details).</p></div><div class="footnote" id="t1fn4"><sup>d</sup><p class="last">PDB ID of the representative fragment–protein complex used in the validation set. A full set of PDB IDs for each cluster is provided in Table S5 in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>.</p></div><div class="footnote" id="t1fn5"><sup>e</sup><p class="last">In Å.</p></div></div></div><div class="NLM_p">The distributions of physiochemical properties of the fragments in the validation set are shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>. It is clear that these are very small fragments with an average heavy atom count (HAC) of 8.3 (<i>s</i> = 1.7) and an average molecular weight (MW) of 120 (<i>s</i> = 25). Some of the fragments are quite lipophilic and will consequently have low aqueous solubility, which would preclude experimental fragment mapping using X-ray crystallography. We restricted the fragments to contain a maximum of one rotatable bond. As such, nine fragments in the set have one rotatable bond, and the remaining fragments have no rotatable bonds.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Distribution of physicochemical properties of the fragments in the validation set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Hot and Warm Spots</h3><div class="NLM_p">We categorized each entry in our validation set as either a “hot” or a “warm” spot (see <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). In our classification hot spots are supposed to represent high affinity sites where one might also expect to obtain hits from a “regular” fragment screen and represent key recognition motifs of ligands binding to a site. Warm spots are lower-affinity sites, but still represent important areas where specific functional groups favorably interact with the protein. These warm spots can be very interesting for structure-based design, for example, during fragment-to-lead campaigns. The position and orientation of a warm spot fragment might not necessarily be conserved when it binds independently of the rest of the ligand, as more favorable nearby hot spots might drive the fragment binding toward them. In order to classify each entry in a relatively unbiased manner, we used ligand occupancy maps for inferring whether a fragment binding site is hot or warm. A ligand occupancy map simply describes the fraction of unique ligands in the PDB that occupies each position in the binding site. Highly occupied regions (occupancy ≥ 0.8) are classified as “hot”, whereas less occupied regions are classified as “warm”. The protocols we used here are described in more detail in the “Methods” section in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>.</div><div class="NLM_p">An example of a hot spot in the validation set is the phenyl-sulfonamide binding subsite in CA2 active site (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>a,c); the subsite has ligand occupancy of 0.95 meaning 95% of the CA2 active site ligands in the PDB occupy the subsite. This example represents a high-affinity site and is a well-known recognition motif in carbonic anhydrases. Phenyl-sulfonamide has an affinity against CA2 of 430 nM<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and can easily be detected by X-ray crystallography against this target.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Examples of “hot” and “warm” spots in the validation set: (a) phenyl-sulfonamide cluster in the CA2 binding site (complex ID 26) and (b) chloro-phenyl cluster in the PKA-Akt chimera (complex ID 17). Ligand occupancy maps contoured at 0.8 (blue) and 0.5 (orange) are also shown for both examples (c and d).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">However, the chloro-phenyl binding subsite near the Gly-rich loop of PKA-Akt chimera, a surrogate for Akt, is a warm spot (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b,d); the ligand occupancy for the spot is only 0.69. In a previous study, it was shown that correctly presenting a chloro-phenyl group to this subsite can provide a ∼170-fold jump in potency.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> However, a 170-fold improvement in potency equates to only 3 kcal/mol gain in binding free energy. Hence, particularly taking the rigid-body entropic barrier to binding (approximately 4.2 kcal/mol as estimated by Murray and Verdonk<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>) and the solubility of chloro-benzene into account, it is highly unlikely that one would be able to detect the binding of this fragment in isolation. The hot spot in the PKA-Akt chimera (and Akt) binding site is the hinge-region, which preferably binds heteroaromatic rings (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b) and has a ligand occupancy of 0.99. Several experimental fragment hits have been identified binding to this hot spot region.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> PLImap</h3><div class="NLM_p">A detailed description of the PLImap protocol is provided in the “Methods” section in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>. In brief, PLImap combines the PLIff energy function<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with a systematic, grid-based search algorithm to identify favorable sites and poses for fragments binding to a protein. PLIff is routinely used at Astex for scoring protein–ligand complexes in molecular docking and generating atomic interaction fields. Recently, it has been shown that PLIff, when employed to rescore docking poses, performs at least as well as four other scoring functions including ChemPLP and GoldScore in molecular docking and virtual screening experiments on large public and internal validation sets.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> However, this is the first time that we have evaluated the performance of PLIff as a scoring function when used for both driving the search and scoring the poses in the context of fragment mapping.</div><div class="NLM_p"><a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> shows the performance of PLImap against the new fragment mapping test set. Three performance measures are listed: (i) the distance between the geometric center of the top-ranked solution and that of the X-ray binding mode; (ii) the volume-overlap-based shape similarity between the top-ranked solution and the experimental binding mode; (iii) the root-mean-square distance (RMSD) between the top-ranked PLImap solution and the experimental binding mode (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for more details). To put the PLImap results into perspective, we compare to fragment docking using GOLD (see “Supplemental methods” section in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a> for docking protocols) and to two simple shape-based pocket-finding algorithms, LIGSITE<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> AMIsite (see “Supplemental method” section in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Performance of PLImap against the Validation Set Using Three Different Measures of Success; Results Are Also Shown for Five GOLD Docking/Rescoring Protocols and Two Shape-Based Pocket Finding Algorithms</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">distance-based<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">RMSD-based<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">volume-overlap-based<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left"><i>PLImap</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PLIff</td><td class="colsep0 rowsep0" align="left">65</td><td class="colsep0 rowsep0" align="left">52</td><td class="colsep0 rowsep0" align="left">65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left"><i>GOLD Docking</i><a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ASP<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></td><td class="colsep0 rowsep0" align="left">62 ± 1</td><td class="colsep0 rowsep0" align="left">44 ± 4</td><td class="colsep0 rowsep0" align="left">63 ± 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ChemPLP<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></td><td class="colsep0 rowsep0" align="left">62 ± 1</td><td class="colsep0 rowsep0" align="left">42 ± 2</td><td class="colsep0 rowsep0" align="left">62 ± 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Chemscore<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></td><td class="colsep0 rowsep0" align="left">68 ± 1</td><td class="colsep0 rowsep0" align="left">44 ± 2</td><td class="colsep0 rowsep0" align="left">68 ± 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Goldscore<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></td><td class="colsep0 rowsep0" align="left">64 ± 2</td><td class="colsep0 rowsep0" align="left">50 ± 2</td><td class="colsep0 rowsep0" align="left">67 ± 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PLIff</td><td class="colsep0 rowsep0" align="left">64 ± 1</td><td class="colsep0 rowsep0" align="left">45 ± 3</td><td class="colsep0 rowsep0" align="left">64 ± 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left"><i>Shape-Based Methods</i></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LIGSITE<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AMIsite<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Percentage of the examples in the validation set for which the centroid of the top-ranking pose (or the pocket volume in case of shape-based methods) is within 1.5 Å of the centroid of the bound fragment.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">Percentage of the examples in the validation set for which the RMSD between the top-ranking pose and the bound fragment is within 1.5 Å.</p></div><div class="footnote" id="t2fn3"><sup>c</sup><p class="last">Percentage of the examples in the validation set for which the volume-overlap-based shape similarity between the top-ranking pose and the bound fragment is above 0.51.</p></div><div class="footnote" id="t2fn4"><sup>d</sup><p class="last">ChemPLP (the default scoring function in GOLD) was used to drive the docking. The listed scoring function was used to rescore 15 poses generated in each of the 20 repeats. Average results over the 20 repeats along with standard deviations are shown.</p></div><div class="footnote" id="t2fn5"><sup>e</sup><p class="last">See “Supplemental methods” in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a> and ref <a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">39</a> for details.</p></div></div></div><div class="NLM_p">The performance of PLImap, in terms of the three indicators recorded in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, is similar to that of the GOLD protocols listed. However, the sampling of low-PLIff-energy solutions is significantly better in the case of PLImap (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). For example, for 40 out of the 52 entries in our fragment mapping set, PLImap produces a better-scoring solution than the GOLD-PLIff protocol. A detailed analysis of the results (see “Supplemental results and discussion” section in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>) suggests that this increased sampling also leads to higher-quality binding mode predictions in the PLImap protocol. It is satisfying to see that the more sophisticated protocols (PLImap and GOLD) outperform the two shape-based methods in terms of predicting the locations of the fragment sites. As expected, all protocols perform better on the hot spots, compared to the warm spots in our new validation set (see “Supplemental results and discussion” section in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>), but this can be alleviated by considering lower-ranking poses (e.g., the top three poses).</div><div class="NLM_p">Examples of successful predictions by AMIsite, GOLD, and PLImap are shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>. AMIsite correctly identifies the hydrophobic pocket on K-Ras where the indole fragment binds (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a). An example of a successful prediction using GOLD docking driven by ChemPLP and rescored by Goldscore is the 4-oxo-3,4-dihydroquinazoline binding pose in the Tankyrase-1 binding site within 0.30 Å RMSD of the bound fragment (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b). Similarly, PLImap correctly identifies the binding mode of a benzoic acid fragment in the retinoic acid receptor beta (RARβ) binding site at the top rank (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>c).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Examples showing successful predictions from different protocols: (a) indole against K-Ras (complex ID 24) from AMIsite; (b) 4-oxo-3,4-dihydroquinazoline against Tankyrase-1 (complex ID 40) from GOLD using ChemPLP for driving the docking and Goldscore as rescoring function; (c) benzoic acid against RARβ (complex ID 19) from the PLImap protocol. The top scoring pose (or the site identified in the case of AMIsite) is shown in magenta sticks (plus signs). Ligands from the representative PDB structure are shown as sticks; carbons forming part of the fragment and in the rest of the ligand are colored green and gray, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">An advantage of the systematic sampling employed by PLImap is that it also results in a map of the binding site, which provides a complete picture of where the fragment prefers to interact. This can be exemplified by the results on benzoic acid mapping of the all-<i>trans</i> retinoic acid binding site of RARβ. RARβ antagonists are stabilized via hydrogen bonding interactions formed by the benzoic acid moiety and by extensive van der Waals contacts (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a). Several selective and nonselective RARβ antagonists present a terminal phenyl (or other aryl) moiety to a hydrophobic subpocket.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The map of a benzoic acid probe against the RARβ binding site not only shows the lowest PLIff energies where the fragment binds, but also shows favorable energies overlapping with phenyl-carbonyl and the terminal phenyl subgroups in the ligand. Similarly, in the case of endothelial nitric oxide synthase (eNOS), the map for the 4-methyl, 2-aminopyrimidine probe shows favorable energies at the centroid of the terminal 6-methyl, 2-aminopyridine in addition to the region where the fragment is bound (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>b).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Examples showing PLIff energy maps generated by the PLImap protocol for (a) RARβ (complex ID 19) and (b) eNOS (complex ID 22). Ligand and the top-ranking pose are represented in the same style as in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>. The maps show the lowest PLIff energies among all orientations of the corresponding probe molecule centered on a grid point. The maps are contoured at −3.2 (blue) and −2.8 kcal/mol (orange) in the case of RARβ and −3.0 (blue) and −2.3 kcal/mol (orange) in the case of eNOS.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Challenges in the Validation Set</h3><div class="NLM_p">There are a few factors that contributed to a low overall performance but are not related to the mapping protocols themselves. Instead, these factors represent challenges that are inherent to the examples in the validation set or to the metric used for assessing the prediction. Here we describe these challenges that resulted in failures from the PLImap and GOLD protocols. One of the challenges is the use of RMSD as a success criterion, which has inherent limitations, especially in the case of lipophilic fragments. The top-ranking PLImap pose of three such lipophilic fragments (fluoro-phenyl (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>a), 1,3-dichloro-phenyl, and adamantane), although they overlap nicely with the bound fragment, have RMSDs > 1.5 Å. These examples are successfully predicted in terms of their volume-overlap-based shape similarity with the bound fragment.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Challenging examples in the validation set: (a) fluoro-phenyl-hepatocyte growth factor receptor (complex ID 18), (b) 3,5-dimethylisoxazole-BRD4 (complex ID 36), (c) resorcinol-HSP90 (complex ID 31), and (d) chloro-phenyl-PKA-Akt chimera (complex ID 17). The top-ranking poses from the PLImap protocol (a and b) and GOLD docking with ChemPLP rescoring function (c and d) are shown as in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As a second challenge, the correct binding modes of a few fragments could not be identified at the top rank because they form water-mediated hydrogen bonding interactions with the protein. Interactions with structural waters can contribute significantly to the binding free energy for a small molecule binding to a protein.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Nevertheless, we did not include any waters in the protein structures in our set as the locations of water molecules mediating hydrogen bonds may not be known in a prospective fragment mapping exercise. Ten of the fragments in the set for which a correct binding pose was not obtained at the top rank using PLImap form at least one water mediated interaction with the protein. An example is 3,5-dimethylisoxazole bound to bromodomain-containing protein 4 (BRD4). The ring oxygen and nitrogen in the experimental structure form hydrogen bonds with the Asn140 side-chain amide nitrogen and a tightly bound water molecule, respectively (Figure S8a in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>). The top-ranking pose from PLImap perfectly overlaps with the bound fragment, but in a flipped orientation (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>b). Since we discarded all water molecules from the protein structures, PLImap predicts that the ring nitrogen, not the oxygen, forms the hydrogen bond with Asn140. This is because the PLIff potentials correctly regard the isoxazole nitrogen as a better hydrogen-bond acceptor than the oxygen (also see Figure S8b in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>).</div><div class="NLM_p">The third challenge that resulted in a low overall performance related to the choice of the representative PDB structure for the fragment–protein complex. In particular, the PDB structures chosen for the two examples of HSP90A in the set have an additional induced subpocket. This subpocket is very hydrophobic, and hence, several high-ranking poses of resorcinol (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>c) and 2-amino-4-methyl-pyrimidine occupy this subpocket.</div><div class="NLM_p last">Finally, for some examples, the top-ranking pose occupied an energetically favorable subsite other than the one where the fragment is bound as part of the ligand. This indicates that, in some cases, the binding position of the fragments as part of their parent ligands might not be conserved when they bind independently of the rest of the ligand. Kinesin-like protein KIF11, has two subpockets in the active site where aromatic rings bind; in this example, the top-ranking pose of 1,4-difluoro-phenyl occupies the “wrong” subpocket. As another example, chloro-phenyl is bound to a “warm” spot (ligand occupancy = 0.69) in the PKA-Akt chimera active site for which the top-ranking pose from GOLD docking (ChemPLP scoring function) occupies the “hot” spot where most ligands place a heteroaromatic ring (ligand occupancy = 0.99) (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>d and <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>d). Other examples of this sort include phenyl with hematopoietic prostaglandin D synthase, 1,3-chloro-4-fluoro-phenyl with hepatocyte growth factor receptor, and 2-methylpyridine with TGF-beta receptor type-1.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08288" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08288" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this article we have introduced the first high-quality, relevant, and diverse validation set for evaluating the performance of fragment mapping algorithms. The approach we adopted here was to identify examples from the PDB where the same fragment (functional group) is presented to the same subsite, by at least three chemically diverse ligands. This resulted in a set of 52 examples, representing 45 unique targets and 41 unique fragments. We have classified each example as either a fragment “hot” spot or a fragment “warm” spot. A hot spot is a high-affinity, key recognition site, where one would expect to obtain hits from a “regular” fragment screen. A warm spot is a lower affinity site that represents an opportunity for improving the affinity of an existing hit or lead.</div><div class="NLM_p">Additionally, we have introduced PLImap, a newly developed standalone fragment mapping application (code freely available from <a href="https://bitbucket.org/AstexUK/pli" class="extLink">https://bitbucket.org/AstexUK/pli</a>). We have tested the performance of PLImap against the new fragment mapping test set and have compared the results against those obtained for GOLD fragment docking and two shape-based pocket finding algorithms. In terms of raw success rates, PLImap performs equally to the GOLD protocols tested (∼50% success rate for top-ranked solution having RMSD < 1.5 Å), but the systematic search algorithm in PLImap does appear to provide slightly higher quality binding modes. PLImap and GOLD performed significantly better than the shape-based algorithms. All protocols perform better on the hot spots, compared to the warm spots in our new validation set, but this difference can be alleviated by considering additional, lower-ranking poses.</div><div class="NLM_p last">We have made the fragment mapping set freely available for download from the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a> with the intention that other researchers can test their fragment mapping protocols against it. We focused here on predicting “hot” and “warm” spots within the confines of a given binding site of a protein, as this is of particular interest to us in terms of generating ideas for fragment growing and merging in FBDD projects. However, other researchers may use the set to test their global search protocols for identifying fragment binding sites across entire protein surfaces. We believe the fragment mapping validation set we introduced here will significantly improve our understanding of the utility of fragment mapping technologies.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23310" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23310" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Construction of the Validation Set</h3><div class="NLM_p">The validation set was constructed by a ligand “shredding” approach; the approach is briefly described here and each step is explained in detail in the “Supplemental methods” section in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>. We searched the PDB for examples where multiple diverse ligands present the same moiety (fragment) to the same subpocket of the same protein (i.e., as determined by the Uniprot ID (<a href="http://www.uniprot.org" class="extLink">www.uniprot.org</a>)<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a>). Certain quality measures were used to filter the PDB structures, and the bound ligands to obtain a set of high quality structures with relevant ligands. Each ligand in the curated PDB list was recursively shredded by breaking each single bond where at least one of the participating atoms is either a carbon bonded to three heavy atoms or a heteroatom. Bonds forming rings were not considered for shredding to avoid fragmenting rings. The bonded atom that is not part of the shred was stored as a dummy atom; this helps identifying the attachment point in the shred. All shreds were generalized to their canonical tautomeric form using the MolVS tool (<a href="http://molvs.readthedocs.io" class="extLink">http://molvs.readthedocs.io</a>), which in turn uses the RDKit chemistry framework.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The shreds were filtered based on criteria described in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>. In summary, only terminal shreds with a maximum of one rotatable bond and with heavy atom counts between five and 11 were considered. The shreds were further generalized based on their chemical structure and the generic atom type of the point of attachment in the PLIff formalism.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> PLIff generalizes atoms based on their element type, number of hydrogens, and bonds the atom is involved in. This will, for example, generalize all toluene shreds that have an attachment point at any of the ring carbons to the same shred type, whereas the toluene shred with attachment point at the methylene carbon will be considered a different shred. Further filtering based on the number of times a shred occurred in PDB ligands, presence of a subshred in the set, and drug-likeness resulted in 616 unique shreds (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Shred selection workflow. HAC stands for heavy atom count.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, instances of each shred were clustered based on the protein residues the shreds are in contact with. This results in clusters of shreds in which individual member shreds interact with the same binding site of the same protein as determined by the Uniprot ID. Only one example of similar ligands placing the shred in the same protein binding site was kept. A second level of clustering was performed based on RMSD values between shreds in a cluster. All shred–protein complexes in a cluster were superimposed using KFIT, which performs a sequence alignment<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> based structure superposition,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> and pairwise RMSDs between superposed shreds were calculated. Clusters of shred–protein complexes were then obtained using an RMSD cutoff of 1.5 Å. As the final automatic filter, clusters where none of the PDB structures had structure factors for generating electron density maps were discarded. This resulted in 237 unique shred–protein clusters where at least three dissimilar ligands place the shred in the same protein environment. Next, the clusters were manually filtered during which emphasis was given to constructing a pharmaceutically relevant, high-quality, diverse validation set. No more than one example of fragments having similar chemotypes bound to the same binding site was selected. Furthermore, the number of examples for a shred with different proteins was restricted to a maximum of four to avoid over-representation of a shred in the validation set. Care was taken to ensure that the manual selection was not biased on a perceived performance of a fragment mapping exercise against the shred–protein complexes. The final set contained 52 examples involving 45 unique proteins and 41 unique shreds.</div><div class="NLM_p last">For each of the 52 clusters, protein and the bound fragment for only one representative PDB structure (best among all structures in terms of structural quality) were saved. Furthermore, the largest ligand in the cluster was also saved, which was used for defining the binding site in the fragment mapping experiments. Finally all representative structures in the set were prepared by removing waters and adding hydrogens using the Protonate3D utility<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> within the MOE<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> software with the default settings. Both the protein and the fragment were protonated together using the complex structure. All protein and fragment structures were then visually inspected and any incorrect tautomeric or charge state assignments were rectified.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Characterizing Binding Subpockets of the Fragments</h3><div class="NLM_p">Each fragment binding site in the validation set was categorized as either a “hot” spot or a “warm” spot. Ideally such a characterization should be based on binding affinities of the fragments with their targets. Alternatively, a difference between the binding affinities of the whole ligand and the ligand minus the fragment can be used as a surrogate for the binding affinity of the fragment. Unfortunately we found that these binding affinities were not available for almost all examples in the set. However, we can infer whether subsites are hot or warm by analyzing how often different ligands in the PDB place <i>any</i> atom in the subsites. We do this by generating ligand “occupancy” maps for the binding site for each of the examples in the set. All PDB structures of proteins that have a sequence similarity of at least 70% with the target protein and have any ligand bound in the fragment-binding site (not necessarily containing the fragment) were superimposed using an in-house program KFIT. Here, all ligand-bound protein structures irrespective of whether or not the ligand contains the corresponding fragment were considered. However, if multiple PDB entries have the same ligand in the same site, only one example of the ligand was kept for occupancy calculation. A 3D grid covering all aligned ligands was then placed on the active site. For each grid point, we count the number of ligands that occupy it. A grid point is considered occupied by a ligand if any atom in the ligand lies within a certain distance (2.0 Å for hydrogen bond donor or acceptor atoms; 1.5 Å for metals; van der Waals radius +0.8 Å for all other atoms). Finally, the occupancy counts are normalized by the number of the ligands used, resulting in so-called “ligand occupancy” maps. The occupancy value for each bound fragment in the validation set was then computed using the corresponding ligand occupancy maps as the average occupancy over all non-hydrogen atoms in the fragment. Here each atom in the fragment gets the occupancy value by trilinear interpolation of the occupancy map. Visually inspecting the ligand occupancy maps, we found that a cutoff of 0.8 seemed to discriminate well between “hot” and “warm” spots. All fragments for which the normalized occupancy value was below 0.8 (meaning on an average, each atom position in the fragment is occupied by less than 80% of the ligands in the site) were considered “warm”.</div><div class="NLM_p last">Defining the boundaries of a site and identifying all ligands that bind to a site for generating ligand occupancy maps is not a trivial process. There are several issues associated with this, some of which are (1) several nonoverlapping small ligands can bind to the same binding site, and (2) a very large ligand may bind to two different sites. With this in mind we clustered all ligands of a protein into sites based on the protein residues they interact with. Two ligands were clustered to the same site if they interact with at least four common residues. Only ligands with a maximum of 30 heavy atoms were considered for this first round of clustering. Ligands with more than 30 atoms were then added to the site, which is defined by the highest number of residues that the large ligand interacts with. These large ligands were added to a new site if none of the sites identified in the first round have at least four common interacting residues. This results in multiple sites on protein structures with different ligands belonging to each of the sites. One of the sites identified this way belongs to the ligand that contained the bound fragment in the validation set; all unique ligands in this site are used to generate the ligand occupancy map.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> PLImap</h3><div class="NLM_p last">We developed a method for fragment mapping using the knowledge based scoring function PLIff (Protein Ligand Informatics force field).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Since the scoring of a protein–ligand complex using PLIff is very fast, we could afford to perform a systematic search of the whole binding site for finding energetically favorable positions of the probe molecule. During the systematic search, diverse clusters of low energy probe poses are saved and written to a file at the end of the run. In more detail, we place a three-dimensional regular grid with 0.15 Å spacing covering all protein atoms that lie within 6 Å of the site ligand for sampling translational space of the probe fragment in the binding site. Grid points that are more than 6 Å away from the site ligands were masked, and the fragment probes were not placed on them; this was done to have a site definition matching as closely as possible to the one used in the shape-based methods and GOLD runs (see below and the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>). Rotational space was sampled by generating 700 different orientations of the fragment quasi-uniformly distributed on a sphere. For the fragments with a rotatable bond, multiple conformations were generated for each orientation by uniformly sampling the dihedral angle in a step size of 30°. Next, on each grid point, the PLIff score for each orientation of the probe molecule was estimated using precalculated PLIff atomic fields. These estimated scores are used to filter out poses that do not undergo a full PLIff scoring, which requires computationally more expensive Voronoi partitioning<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> and contact area calculation. Only probe orientations that have a per-atom estimated score of ≤−0.2 kcal/mol were scored using PLIff. The poses were either saved in a diverse cluster or discarded based on the clustering algorithm described in “Supplemental methods” section in the <a class="ref internalNav" href="#notes-1" aria-label="SI">SI</a>. In the end, diverse clusters are ranked based on the best score of any of the poses in the cluster, and the top five diverse clusters were written to a file along with the scores of each individual pose in the cluster.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> PLImap Validation and Comparison to Other Fragment Mapping Approaches</h3><div class="NLM_p last">PLImap was evaluated in terms of its ability to predict the positions and binding modes of the fragments in the newly created fragment mapping test set. For comparison, we also evaluated the performance of GOLD docking and two shape-based pocket-finding algorithms (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details on the protocols used for these methods). Up to three different metrics, centroid distance, RMSD, and volume-overlap-based shape similarity,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> comparing the top-ranked pose with the X-ray binding mode of the fragment, were used for evaluating the success of the different fragment mapping protocols. Volume overlap is less strict than RMSD as it will have a high value even if a fragment binds in a different orientation than the bound fragment but occupies the same space in the active site. Similarly, centroid distance is also a less strict measure for assessing success and was employed here for allowing a comparison with shape-based methods. An RMSD of ≤1.5 Å and a volume-overlap-based shape similarity of ≥0.51 between the top-ranking pose of a cluster and the bound fragment were considered successes in terms of these two metrics. The cutoff of 0.51 for the volume-overlap-based shape similarity was used as this is the minimum similarity with which two poses of a fragment can have an RMSD of 1.5 Å in our validation set. Similarly, a cutoff of 1.5 Å in terms of the distance between the centroids of the top-ranking pose in a cluster (or the pocket volume) and the bound fragment was used for assessing the prediction success.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00366">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37341" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37341" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b00366" class="ext-link">10.1021/acs.jmedchem.7b00366</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Prepared structure files for the validation set, detailed methods on the construction of the validation set and fragment mapping protocols, detailed results and discussion on the performance of different fragment mapping protocols used, binding mode and electrostatic potential surface of 3,5-dimethylisoxazole, PDB IDs of the complexes for each example in the validation set that place the fragment in the same binding subpocket (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00366/suppl_file/jm7b00366_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Validation set (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00366/suppl_file/jm7b00366_si_002.zip" class="ext-link">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00366/suppl_file/jm7b00366_si_001.pdf">jm7b00366_si_001.pdf (1.3 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00366/suppl_file/jm7b00366_si_002.zip">jm7b00366_si_002.zip (6.91 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b00366" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54836" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54836" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcel L. Verdonk</span> - <span class="hlFld-Affiliation affiliation">Astex Pharmaceuticals, 436 Cambridge Science Park, Milton Road, Cambridge CB4 0QA, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6484-3328" title="Orcid link">http://orcid.org/0000-0002-6484-3328</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#563b372435333a782033243239383d163725222e7835393b"><span class="__cf_email__" data-cfemail="b0ddd1c2d3d5dc9ec6d5c2d4dfdedbf0d1c3c4c89ed3dfdd">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Prakash Chandra Rathi</span> - <span class="hlFld-Affiliation affiliation">Astex Pharmaceuticals, 436 Cambridge Science Park, Milton Road, Cambridge CB4 0QA, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">R. Frederick Ludlow</span> - <span class="hlFld-Affiliation affiliation">Astex Pharmaceuticals, 436 Cambridge Science Park, Milton Road, Cambridge CB4 0QA, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard J. Hall</span> - <span class="hlFld-Affiliation affiliation">Astex Pharmaceuticals, 436 Cambridge Science Park, Milton Road, Cambridge CB4 0QA, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher W. Murray</span> - <span class="hlFld-Affiliation affiliation">Astex Pharmaceuticals, 436 Cambridge Science Park, Milton Road, Cambridge CB4 0QA, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul N. Mortenson</span> - <span class="hlFld-Affiliation affiliation">Astex Pharmaceuticals, 436 Cambridge Science Park, Milton Road, Cambridge CB4 0QA, United Kingdom</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d101e1701-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48026" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48026" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was funded by a postdoctoral fellowship awarded to PCR under the “Sustaining Innovation Postdoctoral Training Program” at Astex Pharmaceuticals. We thank the members of the computational chemistry and informatics department and Dr. Tom Davies at Astex for helpful discussions.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i17" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i17"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i18" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i18"> Abbreviations Used</h2><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">FBDD</td><td class="NLM_def"><p class="first last">Fragment Based Drug Discovery</p></td></tr><tr><td class="NLM_term">MSCS</td><td class="NLM_def"><p class="first last">Multiple Solvent Crystal Structure</p></td></tr><tr><td class="NLM_term">CSD</td><td class="NLM_def"><p class="first last">Cambridge Structural Database</p></td></tr><tr><td class="NLM_term">MCSS</td><td class="NLM_def"><p class="first last">Multiple Copy Simultaneous Search</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">Molecular Dynamics</p></td></tr><tr><td class="NLM_term">MC</td><td class="NLM_def"><p class="first last">Monte Carlo</p></td></tr><tr><td class="NLM_term">PLIff</td><td class="NLM_def"><p class="first last">Protein Ligand Informatics force field</p></td></tr><tr><td class="NLM_term">CA2</td><td class="NLM_def"><p class="first last">carbonic anhydrase II</p></td></tr><tr><td class="NLM_term">HAC</td><td class="NLM_def"><p class="first last">heavy atom count</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">molecular weight</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">RARβ</td><td class="NLM_def"><p class="first last">retinoic acid receptor beta</p></td></tr><tr><td class="NLM_term">eNOS</td><td class="NLM_def"><p class="first last">endothelial nitric oxide synthase</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54473" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54473" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 53 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, R.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">660</span><span class="NLM_x">–</span> <span class="NLM_lpage">672</span><span class="refDoi"> DOI: 10.1038/nrd1467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1038%2Fnrd1467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=15286733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=660-672&author=D.+C.+Reesauthor=M.+Congreveauthor=C.+W.+Murrayauthor=R.+Carr&title=Fragment-based+lead+discovery&doi=10.1038%2Fnrd1467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery</span></div><div class="casAuthors">Rees, David C.; Congreve, Miles; Murray, Christopher W.; Carr, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">660-672</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery is gaining momentum in both large pharmaceutical companies and biotechnol. labs. as a complementary approach to traditional screening.  This is because fragment-based approaches require significantly fewer compds. to be screened and synthesized, and are showing a high success rate in generating chem. series with lead-like properties.  Compared with traditional screening hits, the starting fragments have considerably lower mol. mass, and although the binding interactions of these fragments with a target protein are weak, they are structurally understood through x-ray crystallog. or NMR, and they exhibit high 'ligand efficiency'.  Here, the authors use examples from 25 different protein targets to describe chem. strategies that exploit this structural knowledge to rapidly develop fragments into high-affinity leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbKeUzFFIlhbVg90H21EOLACvtfcHk0lhtai2GQjFCdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D&md5=8d49ac7f698c9b9d573ee3c3485eced8</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd1467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1467%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCarr%26aufirst%3DR.%26atitle%3DFragment-based%2520lead%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D660%26epage%3D672%26doi%3D10.1038%2Fnrd1467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span> </span><span class="NLM_article-title">Introduction to fragment-based drug discovery</span>. In  <span class="citation_source-book">Fragment-based drug discovery and X-ray crystallography</span>; <span class="NLM_contrib-group">Davies, T. G.; Hyvönen, M.</span>, Eds.; <span class="NLM_publisher-name">Springer-Verlag</span>: <span class="NLM_publisher-loc">Berlin Heidelberg</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1007%2F128_2011_180" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=1-32&author=D.+A.+Erlansonauthor=T.+G.+Davies&author=M.+Hyv%C3%B6nen&title=Fragment-based+drug+discovery+and+X-ray+crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2F128_2011_180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F128_2011_180%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26atitle%3DIntroduction%2520to%2520fragment-based%2520drug%2520discovery%26btitle%3DFragment-based%2520drug%2520discovery%2520and%2520X-ray%2520crystallography%26aulast%3DDavies%26aufirst%3DT.%2BG.%26pub%3DSpringer-Verlag%26date%3D2011%26spage%3D1%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alex, A.</span><span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eYhcyyER7B3Qrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Allen, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellamacina, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffery, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petsko, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">An experimental approach to mapping the binding surfaces of crystalline proteins</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">2605</span><span class="NLM_x">–</span> <span class="NLM_lpage">2611</span><span class="refDoi"> DOI: 10.1021/jp952516o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp952516o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK28Xmt1GrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1996&pages=2605-2611&author=K.+N.+Allenauthor=C.+R.+Bellamacinaauthor=X.+Dingauthor=C.+J.+Jefferyauthor=C.+Mattosauthor=G.+A.+Petskoauthor=D.+Ringe&title=An+experimental+approach+to+mapping+the+binding+surfaces+of+crystalline+proteins&doi=10.1021%2Fjp952516o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">An Experimental Approach to Mapping the Binding Surfaces of Crystalline Proteins</span></div><div class="casAuthors">Allen, Karen N.; Bellamacina, Cornelia R.; Ding, Xiaochun; Jeffery, Constance J.; Mattos, Carla; Petsko, Gregory A.; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2605-11</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Porcine pancreatic elastase has been used as the model enzyme in the design and development of a crystallog. method that allows mapping of the binding surface of a protein by solving its crystal structure in a variety of org. solvents.  The ultimate goal of this method is to aid in the process of drug design, where each of the chosen org. mols. represents a given functional group in a larger inhibitor mol.  This method of multiple solvent crystal structures (MSCS) has a theor. counterpart in the method of multiply copy simultaneous search (MCSS) (Miranker, A.; Karplus, M. Proteins: Struct., Funct., Genet. 1991, 11, 29-34) and is the first exptl. method that can be used as a check to the theory.  The MSCS method is presented here with acetonitrile as the probe org. solvent.  The procedure involved does not cause significant changes in the structure of elastase as compared to the structure in aq. soln., and the positions found for the acetonitrile mols. in the active site are compared to those of similar functional groups belonging to known inhibitors bound to elastase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhPgz3NRlYqrVg90H21EOLACvtfcHk0liOerPEDa_2AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xmt1GrtQ%253D%253D&md5=c98dea1a223f30e444b9bc7faff25f0b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjp952516o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp952516o%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DBellamacina%26aufirst%3DC.%2BR.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DJeffery%26aufirst%3DC.%2BJ.%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DAn%2520experimental%2520approach%2520to%2520mapping%2520the%2520binding%2520surfaces%2520of%2520crystalline%2520proteins%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1996%26volume%3D100%26spage%3D2605%26epage%3D2611%26doi%3D10.1021%2Fjp952516o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">Locating and characterizing binding sites on proteins</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">595</span><span class="NLM_x">–</span> <span class="NLM_lpage">599</span><span class="refDoi"> DOI: 10.1038/nbt0596-595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1038%2Fnbt0596-595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=9630949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK28Xis1eksr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=595-599&author=C.+Mattosauthor=D.+Ringe&title=Locating+and+characterizing+binding+sites+on+proteins&doi=10.1038%2Fnbt0596-595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Locating and characterizing binding sites of proteins</span></div><div class="casAuthors">Mattos, Carla; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">595-9</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">A review with 38 refs.  This review article begins with a discussion of fundamental differences between substrates and inhibitors, and some of the assumptions and goals underlying the design of a new ligand to a target protein.  An overview is given of the methods currently used to locate and characterize ligand binding sites on protein surfaces, with focus on a novel approach: multiple solvent crystal structures (MSCS).  In this method, the x-ray crystal structure of the target protein is solved in a variety or org. solvents.  Each type of solvent mol. serves as a probe for complementary binding sites on the protein.  The probe distribution on the protein surface allows the location of binding sites and the characterization of the potential ligand interactions within these sites.  General aspects of the application of the MSCS method to porcine pancreatic elastase is discussed, and comparison of the results with those from x-ray crystal structures of elastase/inhibitor complexes is used to illustrate the potential of the method in aiding the process of rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDEKZ8sM6gk7Vg90H21EOLACvtfcHk0lhPqBaNCjbDeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis1eksr0%253D&md5=4facd2f240311e8937623352e7d933d3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnbt0596-595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt0596-595%26sid%3Dliteratum%253Aachs%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DLocating%2520and%2520characterizing%2520binding%2520sites%2520on%2520proteins%26jtitle%3DNat.%2520Biotechnol.%26date%3D1996%26volume%3D14%26spage%3D595%26epage%3D599%26doi%3D10.1038%2Fnbt0596-595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellamacina, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peisach, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pereira, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vitkup, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petsko, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">Multiple solvent crystal structures: probing binding sites, plasticity and hydration</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">357</span><span class="NLM_x">, </span> <span class="NLM_fpage">1471</span><span class="NLM_x">–</span> <span class="NLM_lpage">1482</span><span class="refDoi"> DOI: 10.1016/j.jmb.2006.01.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2Fj.jmb.2006.01.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=16488429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD28XivVGqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2006&pages=1471-1482&author=C.+Mattosauthor=C.+R.+Bellamacinaauthor=E.+Peisachauthor=A.+Pereiraauthor=D.+Vitkupauthor=G.+A.+Petskoauthor=D.+Ringe&title=Multiple+solvent+crystal+structures%3A+probing+binding+sites%2C+plasticity+and+hydration&doi=10.1016%2Fj.jmb.2006.01.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Solvent Crystal Structures: Probing Binding Sites, Plasticity and Hydration</span></div><div class="casAuthors">Mattos, Carla; Bellamacina, Cornelia R.; Peisach, Ezra; Pereira, Antonio; Vitkup, Dennis; Petsko, Gregory A.; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">357</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1471-1482</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Multiple solvent crystal structures (MSCS) of porcine pancreatic elastase were used to map the binding surface the enzyme.  Crystal structures of elastase in neat acetonitrile, 95% acetone, 55% DMF, 80% 5-hexene-1,2-diol, 80% isopropanol, 80% ethanol and 40% trifluoroethanol showed that the org. solvent mols. clustered in the active site, were found mostly unclustered in crystal contacts and in general did not bind elsewhere on the surface of elastase.  Mixts. of 40% benzene or 40% cyclohexane in 50% isopropanol and 10% water showed no bound benzene or cyclohexane mols., but did reveal bound isopropanol. The clusters of org. solvent probe mols. coincide with pockets occupied by known inhibitors.  MSCS also reveal the areas of plasticity within the elastase binding site and allow for the visualization of a nearly complete first hydration shell.  The pattern of org. solvent clusters detd. by MSCS for elastase is consistent with patterns for hot spots in protein-ligand interactions detd. from database anal. in general.  The MSCS method allows probing of hot spots, plasticity and hydration simultaneously, providing a powerful complementary strategy to guide computational methods currently in development for binding site detn., ligand docking and design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq66xX3ZT7wRLVg90H21EOLACvtfcHk0lhPqBaNCjbDeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivVGqtrk%253D&md5=ab7064d371e5557b2f91aeb6c262a5dc</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2006.01.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2006.01.039%26sid%3Dliteratum%253Aachs%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DBellamacina%26aufirst%3DC.%2BR.%26aulast%3DPeisach%26aufirst%3DE.%26aulast%3DPereira%26aufirst%3DA.%26aulast%3DVitkup%26aufirst%3DD.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DMultiple%2520solvent%2520crystal%2520structures%253A%2520probing%2520binding%2520sites%252C%2520plasticity%2520and%2520hydration%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D357%26spage%3D1471%26epage%3D1482%26doi%3D10.1016%2Fj.jmb.2006.01.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">English, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span> </span><span class="NLM_article-title">Experimental and computational mapping of the binding surface of a crystalline protein</span> <span class="citation_source-journal">Protein Eng., Des. Sel.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">59</span><span class="refDoi"> DOI: 10.1093/protein/14.1.47</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1093%2Fprotein%2F14.1.47" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2001&pages=47-59&author=A.+C.+Englishauthor=C.+R.+Groomauthor=R.+E.+Hubbard&title=Experimental+and+computational+mapping+of+the+binding+surface+of+a+crystalline+protein&doi=10.1093%2Fprotein%2F14.1.47"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F14.1.47&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F14.1.47%26sid%3Dliteratum%253Aachs%26aulast%3DEnglish%26aufirst%3DA.%2BC.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DExperimental%2520and%2520computational%2520mapping%2520of%2520the%2520binding%2520surface%2520of%2520a%2520crystalline%2520protein%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D2001%26volume%3D14%26spage%3D47%26epage%3D59%26doi%3D10.1093%2Fprotein%2F14.1.47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Goodford, P. J.</span><span> </span><span class="NLM_article-title">A computational procedure for determining energetically favorable binding sites on biologically important macromolecules</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">849</span><span class="NLM_x">–</span> <span class="NLM_lpage">857</span><span class="refDoi"> DOI: 10.1021/jm00145a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00145a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaL2MXktVeit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1985&pages=849-857&author=P.+J.+Goodford&title=A+computational+procedure+for+determining+energetically+favorable+binding+sites+on+biologically+important+macromolecules&doi=10.1021%2Fjm00145a002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A computational procedure for determining energetically favorable binding sites on biologically important macromolecules</span></div><div class="casAuthors">Goodford, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">849-57</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The interaction of a probe group with a protein of known structure is computed at sample positions throughout and around the macromol., giving an array of energy values.  The probe include H2O, the Me group, amine-N, carboxy-O, and OH.  Contour surfaces at appropriate energy levels are calcd. for each probe and displayed by computer graphics together with the protein structure.  Contours at neg. energy levels delineate regions of attraction between probe and protein and are found at known ligand binding clefts in particular.  The contours also enable other regions of attraction to be identified and facilitate the interpretation of protein-ligand energetics.  They may, therefore, be of value for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3tYN3Fc76dLVg90H21EOLACvtfcHk0lhPqBaNCjbDeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXktVeit7c%253D&md5=03e9d22787f16c760ef0dc29ca02aa88</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm00145a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00145a002%26sid%3Dliteratum%253Aachs%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26atitle%3DA%2520computational%2520procedure%2520for%2520determining%2520energetically%2520favorable%2520binding%2520sites%2520on%2520biologically%2520important%2520macromolecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1985%26volume%3D28%26spage%3D849%26epage%3D857%26doi%3D10.1021%2Fjm00145a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Wade, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodford, P. J.</span><span> </span><span class="NLM_article-title">Further development of hydrogen bond functions for use in determining energetically favorable binding sites on molecules of known structure. 1. Ligand probe groups with the ability to form two hydrogen bonds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">140</span><span class="NLM_x">–</span> <span class="NLM_lpage">147</span><span class="refDoi"> DOI: 10.1021/jm00053a018</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00053a018" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK3sXltFymsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1993&pages=140-147&author=R.+C.+Wadeauthor=K.+J.+Clarkauthor=P.+J.+Goodford&title=Further+development+of+hydrogen+bond+functions+for+use+in+determining+energetically+favorable+binding+sites+on+molecules+of+known+structure.+1.+Ligand+probe+groups+with+the+ability+to+form+two+hydrogen+bonds&doi=10.1021%2Fjm00053a018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Further development of hydrogen bond functions for use in determining energetically favorable binding sites on molecules of known structure.  1.  Ligand probe groups with the ability to form two hydrogen bonds</span></div><div class="casAuthors">Wade, Rebecca C.; Clark, Kevin J.; Goodford, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">140-7</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The directional properties of hydrogen bonds play a major role in detg. the specificity of intermol. interactions.  An energy function which takes explicit account of these properties was developed for use in the detn. of energetically favorable ligand binding sites on mols. of known structure by the GRID method.  In this method, the interaction energy between a target mol. and a small chem. group (a probe), which may be part of a larger ligand, was calcd. using an energy function consisting of Lennard-Jones, electrostatic, and hydrogen bond terms.  The latter term was a function of the length of the hydrogen bond, its orientation at the hydrogen-bonding atoms, and their chem. nature.  Hydrogen bond energy functions are described which take account of the spatial distribution of the hydrogen bonds made by probes with the ability to form 2 hydrogen bonds.  These functions were designed so as to model the exptl. obsd. angular dependence of the hydrogen bonds.  The procedure to locate the position and orientation of the probe at which the interaction energy is optimized, is also described..  The use of this procedure was demonstrated by examples of biol. and pharmacol. interest which show that it can produce results that are consistent with other theor. approaches and with exptl. observations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWcTTi-uqMA7Vg90H21EOLACvtfcHk0lin9NVFEGTdkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXltFymsA%253D%253D&md5=c7130820648cd5c7b94f37c4b3a29b8b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm00053a018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00053a018%26sid%3Dliteratum%253Aachs%26aulast%3DWade%26aufirst%3DR.%2BC.%26aulast%3DClark%26aufirst%3DK.%2BJ.%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26atitle%3DFurther%2520development%2520of%2520hydrogen%2520bond%2520functions%2520for%2520use%2520in%2520determining%2520energetically%2520favorable%2520binding%2520sites%2520on%2520molecules%2520of%2520known%2520structure.%25201.%2520Ligand%2520probe%2520groups%2520with%2520the%2520ability%2520to%2520form%2520two%2520hydrogen%2520bonds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1993%26volume%3D36%26spage%3D140%26epage%3D147%26doi%3D10.1021%2Fjm00053a018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Wade, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodford, P. J.</span><span> </span><span class="NLM_article-title">Further development of hydrogen bond functions for use in determining energetically favorable binding sites on molecules of known structure. 2. Ligand probe groups with the ability to form more than two hydrogen bonds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">148</span><span class="NLM_x">–</span> <span class="NLM_lpage">156</span><span class="refDoi"> DOI: 10.1021/jm00053a019</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00053a019" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK3sXjsFeqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1993&pages=148-156&author=R.+C.+Wadeauthor=P.+J.+Goodford&title=Further+development+of+hydrogen+bond+functions+for+use+in+determining+energetically+favorable+binding+sites+on+molecules+of+known+structure.+2.+Ligand+probe+groups+with+the+ability+to+form+more+than+two+hydrogen+bonds&doi=10.1021%2Fjm00053a019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Further development of hydrogen bond functions for use in determining energetically favorable binding sites on molecules of known structure.  2.  Ligand probe groups with the ability to form more than two hydrogen bonds</span></div><div class="casAuthors">Wade, Rebecca C.; Goodford, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">148-56</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The specificity of interactions between biol. macromols. and their ligands may be partially attributed to the directional properties of H bonds.  The GRID method of P. J. Goodford (1985) and of D. N. A. Boobbyer et al. (1989) for detg. energetically favorable ligand binding sites on mols. of known structure was extended to improve the treatment of groups which can make multiple H bonds.  In this method, the interaction energy between a probe (a small chem. group that may be part of a larger ligand) and a target mol. is calcd. by using an energy function that includes a H bond term that is dependent on the length of the H bond, its orientation at the H-bonding atoms, and their chem. character.  The methods described in the preceding paper (Wade, R. C. et al., 1992) for probes capable of making 2 H bonds are here extended to the following probes which have the ability to make > 2 H bonds: ammonium-NH3+, amine-NH2, sp3-hybridized hydroxyl, and water.  Use of the improved GRID procedure is demonstrated by detn. of the conformation of an amino acid side chain at the subunit interface in Hb and of the location of water binding sites in human lysozyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9XWnnyz8Wy7Vg90H21EOLACvtfcHk0lin9NVFEGTdkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXjsFeqtg%253D%253D&md5=c1ed1e964cede076d62ccaf74fec2bd6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm00053a019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00053a019%26sid%3Dliteratum%253Aachs%26aulast%3DWade%26aufirst%3DR.%2BC.%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26atitle%3DFurther%2520development%2520of%2520hydrogen%2520bond%2520functions%2520for%2520use%2520in%2520determining%2520energetically%2520favorable%2520binding%2520sites%2520on%2520molecules%2520of%2520known%2520structure.%25202.%2520Ligand%2520probe%2520groups%2520with%2520the%2520ability%2520to%2520form%2520more%2520than%2520two%2520hydrogen%2520bonds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1993%26volume%3D36%26spage%3D148%26epage%3D156%26doi%3D10.1021%2Fjm00053a019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">SuperStar: a knowledge-based approach for identifying interaction sites in proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">1093</span><span class="NLM_x">–</span> <span class="NLM_lpage">1108</span><span class="refDoi"> DOI: 10.1006/jmbi.1999.2809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1006%2Fjmbi.1999.2809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10369784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK1MXjslKhtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=1999&pages=1093-1108&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=R.+Taylor&title=SuperStar%3A+a+knowledge-based+approach+for+identifying+interaction+sites+in+proteins&doi=10.1006%2Fjmbi.1999.2809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">SuperStar: A Knowledge-based Approach for Identifying Interaction Sites in Proteins</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1093-1108</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">An empirical method for identifying interaction sites in proteins is described and validated.  The method is based entirely on exptl. information about non-bonded interactions occurring in small-mol. crystal structures.  These data are used in the form of scatterplots that show the exptl. obsd. distribution of one functional group (the "contact group" or "probe") around another.  A template mol. (e.g. a protein binding site) is broken down into structure fragments and the scatterplots, showing the distribution of a chosen probe around these structure fragments, are superimposed on the corresponding parts of the template.  The scatterplots are then translated into a three-dimensional map that shows the propensity of the probe at different positions around the template mol.  The method is illustrated for L-arabinose-binding protein, complexed with L-arabinose and with D-fucose, and for dihydrofolate reductase complexed with methotrexate.  The method is validated on 122 x-ray structures of protein-ligand complexes.  For all the binding sites of these proteins, propensity maps are generated for four different probes: a charged NH+3 nitrogen, a carbonyl oxygen, a hydroxyl oxygen and a Me carbon atom.  Next, the maps are compared with the exptl. obsd. positions of ligand atoms of these types.  For 74% of these ligand atoms (84% of the solvent-inaccessible ones) the calcd. propensity of the matching probe at the exptl. positions is higher than expected by chance.  For 68% of the atoms (82% of the solvent-inaccessible ones) the propensity of the matching probe is higher than that of the other three probes.  These results indicate that the approach generally gives good predictions for protein-ligand interactions.  The potential applications of the propensity maps range from an aid in manual docking and structure-based drug design to their use in pharmacophore development.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9QVkFGAwFt7Vg90H21EOLACvtfcHk0lin9NVFEGTdkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjslKhtrs%253D&md5=72e09c54c9a631a8271fbe72f2f431b1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1999.2809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1999.2809%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DSuperStar%253A%2520a%2520knowledge-based%2520approach%2520for%2520identifying%2520interaction%2520sites%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1999%26volume%3D289%26spage%3D1093%26epage%3D1108%26doi%3D10.1006%2Fjmbi.1999.2809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillet, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">SuperStar: improved knowledge-based interaction fields for protein binding sites</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">307</span><span class="NLM_x">, </span> <span class="NLM_fpage">841</span><span class="NLM_x">–</span> <span class="NLM_lpage">859</span><span class="refDoi"> DOI: 10.1006/jmbi.2001.4452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1006%2Fjmbi.2001.4452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=11273705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitFymur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=307&publication_year=2001&pages=841-859&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=P.+Watsonauthor=V.+Gilletauthor=P.+Willett&title=SuperStar%3A+improved+knowledge-based+interaction+fields+for+protein+binding+sites&doi=10.1006%2Fjmbi.2001.4452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">SuperStar: Improved knowledge-based interaction fields for protein binding sites</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Watson, Paul; Gillet, Val; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">307</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">841-859</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">SuperStar is an empirical method for identifying interaction sites in proteins, based entirely on exptl. information about non-bonded interactions occurring in small-mol. crystal structures, taken from the IsoStar database.  We describe recent modifications and addns. to SuperStar, validating the results on a test set of 122 x-ray structures of protein-ligand complexes.  In this validation, propensity maps are generated for all the binding sites of these proteins, using four different probes: a charged NH+3 nitrogen atom, a carbonyl oxygen atom, a hydroxyl oxygen atom and a Me carbon atom.  Next, the maps are compared with the exptl. obsd. positions of ligand atoms of these types.  A peak-searching algorithm is introduced that highlights potential interaction hot spots.  For the three hydrogen-bonding probes - NH+3 nitrogen atom, carbonyl oxygen atom and hydroxyl oxygen atom - the av. distance from the ligand atom to the nearest SuperStar peak is 1.0-1.2 Å (0.8-1.0 Å for solvent-inaccessible ligand atoms).  For the Me carbon atom probe, this distance is about 1.5 Å, probably because interactions to Me groups are much less directional. The most important addn. to SuperStar is the enabling of propensity maps around metal centers - Ca2+, Mg2+ and Zn2+ - in protein binding sites.  The results are validated on a test set of 24 protein-ligand complexes that have a metal ion in their binding site.  Coordination geometries are derived automatically, using only the protein atoms that coordinate to the metal ion.  The correct coordination geometry is derived in approx. 75 % of the cases.  If the derived geometry is assumed during the SuperStar calcn., the av. distance from a ligand atom coordinating to the metal ion to the nearest peak in the propensity map for an oxygen probe is 0.87(7) Å.  If the correct coordination geometry is imposed, this distance reduces to 0.59(7) Å.  This indicates that the SuperStar predictions around metal-binding sites are at least as good as those around other protein groups. Using clustering techniques, a non-redundant set of probes is selected from the set of probes available in the IsoStar database.  The performance in SuperStar of all these probes is tested on the test set of protein-ligand complexes.  With the exception of the "ether oxygen" probe and the "any NH+" probe, all new probes perform as well as the four probes introduced first.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCZ27rmWmxMbVg90H21EOLACvtfcHk0lgUbYjbfAMbQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitFymur0%253D&md5=026cdc7beaec6446307bbcc36a7e813f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2001.4452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2001.4452%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DWatson%26aufirst%3DP.%26aulast%3DGillet%26aufirst%3DV.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DSuperStar%253A%2520improved%2520knowledge-based%2520interaction%2520fields%2520for%2520protein%2520binding%2520sites%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2001%26volume%3D307%26spage%3D841%26epage%3D859%26doi%3D10.1006%2Fjmbi.2001.4452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Allen, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brice, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cartwright, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doubleday, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hummelink, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hummelink-Peters, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennard, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motherwell, W.</span><span> </span><span class="NLM_article-title">The Cambridge Crystallographic Data Centre: computer-based search, retrieval, analysis and display of information</span> <span class="citation_source-journal">Acta Crystallogr., Sect. B: Struct. Crystallogr. Cryst. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1979</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">2331</span><span class="NLM_x">–</span> <span class="NLM_lpage">2339</span><span class="refDoi"> DOI: 10.1107/S0567740879009249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1107%2FS0567740879009249" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1979&pages=2331-2339&author=F.+H.+Allenauthor=S.+Bellardauthor=M.+Briceauthor=B.+A.+Cartwrightauthor=A.+Doubledayauthor=H.+Higgsauthor=T.+Hummelinkauthor=B.+Hummelink-Petersauthor=O.+Kennardauthor=W.+Motherwell&title=The+Cambridge+Crystallographic+Data+Centre%3A+computer-based+search%2C+retrieval%2C+analysis+and+display+of+information&doi=10.1107%2FS0567740879009249"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1107%2FS0567740879009249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0567740879009249%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DF.%2BH.%26aulast%3DBellard%26aufirst%3DS.%26aulast%3DBrice%26aufirst%3DM.%26aulast%3DCartwright%26aufirst%3DB.%2BA.%26aulast%3DDoubleday%26aufirst%3DA.%26aulast%3DHiggs%26aufirst%3DH.%26aulast%3DHummelink%26aufirst%3DT.%26aulast%3DHummelink-Peters%26aufirst%3DB.%26aulast%3DKennard%26aufirst%3DO.%26aulast%3DMotherwell%26aufirst%3DW.%26atitle%3DThe%2520Cambridge%2520Crystallographic%2520Data%2520Centre%253A%2520computer-based%2520search%252C%2520retrieval%252C%2520analysis%2520and%2520display%2520of%2520information%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520B%253A%2520Struct.%2520Crystallogr.%2520Cryst.%2520Chem.%26date%3D1979%26volume%3D35%26spage%3D2331%26epage%3D2339%26doi%3D10.1107%2FS0567740879009249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lgUbYjbfAMbQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Caflisch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miranker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Multiple copy simultaneous search and construction of ligands in binding sites: application to inhibitors of HIV-1 aspartic proteinase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">2142</span><span class="NLM_x">–</span> <span class="NLM_lpage">2167</span><span class="refDoi"> DOI: 10.1021/jm00067a013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00067a013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK3sXkvVGnsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1993&pages=2142-2167&author=A.+Caflischauthor=A.+Mirankerauthor=M.+Karplus&title=Multiple+copy+simultaneous+search+and+construction+of+ligands+in+binding+sites%3A+application+to+inhibitors+of+HIV-1+aspartic+proteinase&doi=10.1021%2Fjm00067a013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple copy simultaneous search and construction of ligands in binding sites: application to inhibitors of HIV-1 aspartic proteinase</span></div><div class="casAuthors">Caflisch, Amedeo; Miranker, Andrew; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2142-67</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Rational ligand design is a complex problem that can be divided into three parts: the search for optimal positions and orientations of functional groups in the binding site, the connection of such positions to form candidate ligands, and the estn. of their binding consts.  Approaches for addressing the first two parts of the problem are described in the present work.  They are applied to the construction of peptide ligands in the binding site of the human immunodeficiency virus 1 (HIV-1) proteinase and are compared with known inhibitor X-ray structures.  The primary objective is to test the method by comparison of the results with the MVT-101 complex structure for which coordinates are available; the results obtained with the liganded and unliganded proteinase structure are compared to examine the utility of the latter for binding studies.  A secondary objective is to show how to find new inhibitor candidates.  The multiple copy simultaneous search (MCSS) method is utilized to search for optimal positions and orientations of a set of functional groups.  For peptide ligands, functional groups corresponding to the protein main chain (N-methylacetamide) and to protein side chains (e.g., methanol, Et guanidinium) are used.  The resulting N-methylacetamide min. are connected to form hexapeptide main chains with a simple pseudoenergy function that permits a complete search of all possible ways of connecting the min.  Side chains are added to the main-chain candidates by application of the same pseudoenergy function to the appropriate functional group min.  A set of 15 hexapeptides with the sequence of MVT-101 are then minimized by a Monte Carlo scheme, which allows for escape from local min.  Comparison of the MCSS results with the structure of MVT-101 in the HIV-1 binding site showed that all of its functional group positions correspond (within 2.4 Å) to some (usually more than one) MCSS min.  There were also many other low-energy MCSS min. which do not appear in any known inhibitors, e.g., Me ammonium min. in the neighborhood of the catalytic aspartates.  Among the 15 lowest min. are seven hexapeptides with the same main-chain orientation as the one found by X-ray crystallog. for the inhibitor MVT-101 in the binding site and eight with the main chain oriented in the opposite direction; the latter tend to be more stable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohNHWxLu8SpLVg90H21EOLACvtfcHk0lgUbYjbfAMbQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkvVGnsLc%253D&md5=9d14f5a1f3921f090cb4391d89f457db</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm00067a013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00067a013%26sid%3Dliteratum%253Aachs%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DMiranker%26aufirst%3DA.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DMultiple%2520copy%2520simultaneous%2520search%2520and%2520construction%2520of%2520ligands%2520in%2520binding%2520sites%253A%2520application%2520to%2520inhibitors%2520of%2520HIV-1%2520aspartic%2520proteinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1993%26volume%3D36%26spage%3D2142%26epage%3D2167%26doi%3D10.1021%2Fjm00067a013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Miranker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Functionality maps of binding sites: a multiple copy simultaneous search method</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">29</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1002/prot.340110104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1002%2Fprot.340110104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1961699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK38XlsFSjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=29-34&author=A.+Mirankerauthor=M.+Karplus&title=Functionality+maps+of+binding+sites%3A+a+multiple+copy+simultaneous+search+method&doi=10.1002%2Fprot.340110104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Functionality maps of binding sites:  a multiple copy simultaneous search method</span></div><div class="casAuthors">Miranker, Andrew; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Proteins:  Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-34</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    </div><div class="casAbstract">A new method is proposed for detg. energetically favorable positions and orientations for functional groups on the surface of proteins with known 3-dimensional structure.  Some 1000-5000 copies of a functional group are randomly placed in the site and subjected to simultaneous energy minimization and/or quenched mol. dynamics.  The resulting functionality maps of a protein receptor site, which can take account of its flexibility, can be used for the anal. of protein ligand interactions and rational drug design.  Application of the method to the sialic acid binding site of the influenza coat protein, hemagglutinin, yields functional group min. that correspond with those of the ligand in a cocrystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5im8BqJfF37Vg90H21EOLACvtfcHk0lh3oBg__6Ou4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlsFSjtQ%253D%253D&md5=34e0a0c36899bfeabcbdfd4790f97a1e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fprot.340110104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.340110104%26sid%3Dliteratum%253Aachs%26aulast%3DMiranker%26aufirst%3DA.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DFunctionality%2520maps%2520of%2520binding%2520sites%253A%2520a%2520multiple%2520copy%2520simultaneous%2520search%2520method%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1991%26volume%3D11%26spage%3D29%26epage%3D34%26doi%3D10.1002%2Fprot.340110104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Dennis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kortvelyesi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Computational mapping identifies the binding sites of organic solvents on proteins</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">4290</span><span class="NLM_x">–</span> <span class="NLM_lpage">4295</span><span class="refDoi"> DOI: 10.1073/pnas.062398499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1073%2Fpnas.062398499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=11904374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD38XivFShtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=4290-4295&author=S.+Dennisauthor=T.+Kortvelyesiauthor=S.+Vajda&title=Computational+mapping+identifies+the+binding+sites+of+organic+solvents+on+proteins&doi=10.1073%2Fpnas.062398499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Computational mapping identifies the binding sites of organic solvents on proteins</span></div><div class="casAuthors">Dennis, Sheldon; Kortvelyesi, Tamas; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4290-4295</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Computational mapping places mol. probes-small mols. or functional groups-on a protein surface to identify the most favorable binding positions.  Although x-ray crystallog. and NMR show that org. solvents bind to a limited no. of sites on a protein, current mapping methods result in hundreds of energy min. and do not reveal why some sites bind mols. with different sizes and polarities.  We describe a mapping algorithm that explains the origin of this phenomenon.  The algorithm has been applied to hen egg-white lysozyme and to thermolysin, interacting with eight and four different ligands, resp.  In both cases the search finds the consensus site to which all mols. bind, whereas other positions that bind only certain ligands are not necessarily found.  The consensus sites are pockets of the active site, lined with partially exposed hydrophobic residues and with a no. of polar residues toward the edge.  These sites can accommodate each ligand in a no. of rotational states, some with a hydrogen bond to one of the nearby donor/acceptor groups.  Specific substrates and/or inhibitors of hen egg-white lysozyme and thermolysin interact with the same side chains identified by the mapping, but form several hydrogen bonds and bind in unique orientations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg4sEo2VxwF7Vg90H21EOLACvtfcHk0lh3oBg__6Ou4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivFShtrw%253D&md5=90d71ffcc8860e87aefec3c5809b1127</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.062398499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.062398499%26sid%3Dliteratum%253Aachs%26aulast%3DDennis%26aufirst%3DS.%26aulast%3DKortvelyesi%26aufirst%3DT.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DComputational%2520mapping%2520identifies%2520the%2520binding%2520sites%2520of%2520organic%2520solvents%2520on%2520proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D4290%26epage%3D4295%26doi%3D10.1073%2Fpnas.062398499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grove, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohnuud, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mottarella, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">755</span><span class="refDoi"> DOI: 10.1038/nprot.2015.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1038%2Fnprot.2015.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=25855957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=733-755&author=D.+Kozakovauthor=L.+E.+Groveauthor=D.+R.+Hallauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=L.+Luoauthor=B.+Xiaauthor=D.+Beglovauthor=S.+Vajda&title=The+FTMap+family+of+web+servers+for+determining+and+characterizing+ligand-binding+hot+spots+of+proteins&doi=10.1038%2Fnprot.2015.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span></div><div class="casAuthors">Kozakov, Dima; Grove, Laurie E.; Hall, David R.; Bohnuud, Tanggis; Mottarella, Scott E.; Luo, Lingqi; Xia, Bing; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-755</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">FTMap is a computational mapping server that identifies binding hot spots of macromols., i.e., regions of the surface with major contributions to the ligand-binding free energy.  To use FTMap, users submit a protein, DNA or RNA structure in PDB (Protein Data Bank) format.  FTMap samples billions of positions of small org. mols. used as probes, and it scores the probe poses using a detailed energy expression.  Regions that bind clusters of multiple probe types identify the binding hot spots in good agreement with exptl. data.  FTMap serves as the basis for other servers, namely FTSite, which was used to predict ligand-binding sites, FTFlex, which was used to account for side chain flexibility, FTMap/param, used to parameterize addnl. probes and FTDyn, for mapping ensembles of protein structures.  Applications include detg. the druggability of proteins, identifying ligand moieties that are most important for binding, finding the most bound-like conformation in ensembles of unliganded protein structures and providing input for fragment-based drug design.  FTMap is more accurate than classical mapping methods such as GRID and MCSS, and it is much faster than the more-recent approaches to protein mapping based on mixed mol. dynamics.  By using 16 probe mols., the FTMap server finds the hot spots of an av.-size protein in <1 h.  As FTFlex performs mapping for all low-energy conformers of side chains in the binding site, its completion time is proportionately longer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnH27-sNtGLVg90H21EOLACvtfcHk0lh3oBg__6Ou4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D&md5=dbc729e9ffb47989f0df8baebaf032fa</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.043%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DThe%2520FTMap%2520family%2520of%2520web%2520servers%2520for%2520determining%2520and%2520characterizing%2520ligand-binding%2520hot%2520spots%2520of%2520proteins%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D733%26epage%3D755%26doi%3D10.1038%2Fnprot.2015.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Grove, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span> </span><span class="NLM_article-title">FTFlex: accounting for binding site flexibility to improve fragment-based identification of druggable hot spots</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1218</span><span class="NLM_x">–</span> <span class="NLM_lpage">1219</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btt102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1093%2Fbioinformatics%2Fbtt102" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=1218-1219&author=L.+E.+Groveauthor=D.+R.+Hallauthor=D.+Beglovauthor=S.+Vajdaauthor=D.+Kozakov&title=FTFlex%3A+accounting+for+binding+site+flexibility+to+improve+fragment-based+identification+of+druggable+hot+spots&doi=10.1093%2Fbioinformatics%2Fbtt102"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt102%26sid%3Dliteratum%253Aachs%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DKozakov%26aufirst%3DD.%26atitle%3DFTFlex%253A%2520accounting%2520for%2520binding%2520site%2520flexibility%2520to%2520improve%2520fragment-based%2520identification%2520of%2520druggable%2520hot%2520spots%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D1218%26epage%3D1219%26doi%3D10.1093%2Fbioinformatics%2Fbtt102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Radoux, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsson, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Identifying interactions that determine fragment binding at protein hotspots</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4314</span><span class="NLM_x">–</span> <span class="NLM_lpage">4325</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01980</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01980" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlOms7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4314-4325&author=C.+J.+Radouxauthor=T.+S.+Olssonauthor=W.+R.+Pittauthor=C.+R.+Groomauthor=T.+L.+Blundell&title=Identifying+interactions+that+determine+fragment+binding+at+protein+hotspots&doi=10.1021%2Facs.jmedchem.5b01980"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Interactions that Determine Fragment Binding at Protein Hotspots</span></div><div class="casAuthors">Radoux, Chris J.; Olsson, Tjelvar S. G.; Pitt, Will R.; Groom, Colin R.; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4314-4325</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Locating a ligand-binding site is an important first step in structure-guided drug discovery, but current methods do little to suggest which interactions within a pocket are the most important for binding.  Here the authors illustrate a method that samples at. hotspots with simple mol. probes to produce fragment hotspot maps.  These maps specifically highlight fragment-binding sites and their corresponding pharmacophores.  For ligand-bound structures, they provide an intuitive visual guide within the binding site, directing medicinal chemists where to grow the mol. and alerting them to suboptimal interactions within the original hit.  The fragment hotspot map calcn. is validated using exptl. binding positions of 21 fragments and subsequent lead mols.  The ligands are found in high scoring areas of the fragment hotspot maps, with fragment atoms having a median percentage rank of 97%.  Protein kinase B and pantothenate synthetase are examd. in detail.  In each case, the fragment hotspot maps are able to rationalize a Free-Wilson anal. of SAR data from a fragment-based drug design project.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8BzZp1bkHarVg90H21EOLACvtfcHk0lhBSOdi5duAbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlOms7Y%253D&md5=fa25eabfe291f9519f5701bcbc6c52c3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01980%26sid%3Dliteratum%253Aachs%26aulast%3DRadoux%26aufirst%3DC.%2BJ.%26aulast%3DOlsson%26aufirst%3DT.%2BS.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DIdentifying%2520interactions%2520that%2520determine%2520fragment%2520binding%2520at%2520protein%2520hotspots%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4314%26epage%3D4325%26doi%3D10.1021%2Facs.jmedchem.5b01980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Seco, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luque, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span> </span><span class="NLM_article-title">Binding site detection and druggability index from first principles</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2363</span><span class="NLM_x">–</span> <span class="NLM_lpage">2371</span><span class="refDoi"> DOI: 10.1021/jm801385d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801385d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2363-2371&author=J.+Secoauthor=F.+J.+Luqueauthor=X.+Barril&title=Binding+site+detection+and+druggability+index+from+first+principles&doi=10.1021%2Fjm801385d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Site Detection and Druggability Index from First Principles</span></div><div class="casAuthors">Seco, Jesus; Luque, F. Javier; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2363-2371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In drug discovery, it is essential to identify binding sites on protein surfaces that drug-like mols. could exploit to exert a biol. effect.  Both X-ray crystallog. and NMR expts. have demonstrated that org. solvents bind precisely at these locations.  We show that this effect is reproduced using mol. dynamics with a binary solvent.  Furthermore, anal. of the simulations give direct access to interaction free energies between the protein and small org. mols., which can be used to detect binding sites and to predict the maximal affinity that a drug-like mol. could attain for them.  On a set of pharmacol. relevant proteins, we obtain good predictions for druggable sites as well as for protein-protein and low affinity binding sites.  This is the first druggability index not based on surface descriptors and, being independent of a training set, is particularly indicated to study unconventional targets such as protein-protein interactions or allosteric binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTAgMI9Qer6rVg90H21EOLACvtfcHk0lhBSOdi5duAbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D&md5=b766cbadeba11b23a1de44b27aa794b4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm801385d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801385d%26sid%3Dliteratum%253Aachs%26aulast%3DSeco%26aufirst%3DJ.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DBinding%2520site%2520detection%2520and%2520druggability%2520index%2520from%2520first%2520principles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2363%26epage%3D2371%26doi%3D10.1021%2Fjm801385d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Clark, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarnieri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shkurko, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, J.</span><span> </span><span class="NLM_article-title">Grand canonical Monte Carlo simulation of ligand-protein binding</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1021/ci050268f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050268f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Gju7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=231-242&author=M.+Clarkauthor=F.+Guarnieriauthor=I.+Shkurkoauthor=J.+Wiseman&title=Grand+canonical+Monte+Carlo+simulation+of+ligand-protein+binding&doi=10.1021%2Fci050268f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Grand Canonical Monte Carlo Simulation of Ligand-Protein Binding</span></div><div class="casAuthors">Clark, Matthew; Guarnieri, Frank; Shkurko, Igor; Wiseman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">231-242</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new application of the grand canonical thermodn. ensemble to compute ligand-protein binding is described.  The described method is sufficiently rapid that it is practical to compute ligand-protein binding free energies for a large no. of poses over the entire protein surface, thus identifying multiple putative ligand binding sites.  In addn., the method computes binding free energies for a large no. of poses.  The method is demonstrated by the simulation of two protein-ligand systems, thermolysin and T4 lysozyme, for which there is extensive thermodn. and crystallog. data for the binding of small, rigid ligands.  These low-mol.-wt. ligands correspond to the mol. fragments used in computational fragment-based drug design.  The simulations correctly identified the exptl. binding poses and rank ordered the affinities of ligands in each of these systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD_W87xCkI47Vg90H21EOLACvtfcHk0lhBSOdi5duAbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Gju7zJ&md5=6ff1670d21b2650f0eb0a113ab38a1c5</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fci050268f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050268f%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%26aulast%3DGuarnieri%26aufirst%3DF.%26aulast%3DShkurko%26aufirst%3DI.%26aulast%3DWiseman%26aufirst%3DJ.%26atitle%3DGrand%2520canonical%2520Monte%2520Carlo%2520simulation%2520of%2520ligand-protein%2520binding%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D231%26epage%3D242%26doi%3D10.1021%2Fci050268f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Clark, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meshkat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, J. S.</span><span> </span><span class="NLM_article-title">Grand canonical free-energy calculations of protein-ligand binding</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">934</span><span class="NLM_x">–</span> <span class="NLM_lpage">943</span><span class="refDoi"> DOI: 10.1021/ci8004397</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci8004397" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsFKms7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=934-943&author=M.+Clarkauthor=S.+Meshkatauthor=J.+S.+Wiseman&title=Grand+canonical+free-energy+calculations+of+protein-ligand+binding&doi=10.1021%2Fci8004397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Grand Canonical Free-Energy Calculations of Protein-Ligand Binding</span></div><div class="casAuthors">Clark, Matthew; Meshkat, Sia; Wiseman, Jeffrey S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">934-943</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The principles behind the computation of protein-ligand binding free energies by Monte Carlo simulation in the Grand Canonical Ensemble are described in detail, and two variations of the calcn. are presented.  The computation can be performed by bathing a protein binding site either with ligand images that interact with each other or with ligand images that pass through each other.  The second method is theor. more rigorous, but we show that both methods lead to the same result, and there are distinct numeric advantages to using ligand images that interact with each other.  The Grand Canonical simulation provides gas-phase binding free energies that can be converted to aq. energies by generalized Born-surface area (GB/SA) solvation calcns. to provide values that agree with expt. within ±1.5 kcal/mol.  However, the accuracy of these simple solvation calcns. is a major limiting factor in the accuracy of the overall binding free-energy computation.  The Grand Canonical simulation has several characteristics beneficial to free-energy calcns.  One is that the no. of parameters that must be set for the simulation is small and can be detd. objectively, making the outcome more deterministic, with respect to choice of input conditions, as compared to perturbation methods.  Second, the simulation is free from assumptions about the starting pose or nature of the binding site.  A final benefit is that binding free energies are a direct outcome of the simulation, and little processing is required to det. them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmHrvGMlNIQrVg90H21EOLACvtfcHk0lh2_a0j84l5iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsFKms7s%253D&md5=ab2ced7fbc3e27709e3d6e95ed5a8461</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fci8004397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8004397%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%26aulast%3DMeshkat%26aufirst%3DS.%26aulast%3DWiseman%26aufirst%3DJ.%2BS.%26atitle%3DGrand%2520canonical%2520free-energy%2520calculations%2520of%2520protein-ligand%2520binding%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D934%26epage%3D943%26doi%3D10.1021%2Fci8004397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Ghanakota, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, H. A.</span><span> </span><span class="NLM_article-title">Driving structure-based drug discovery through cosolvent molecular dynamics</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">10383</span><span class="NLM_x">–</span> <span class="NLM_lpage">10399</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00399</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00399" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1yrsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10383-10399&author=P.+Ghanakotaauthor=H.+A.+Carlson&title=Driving+structure-based+drug+discovery+through+cosolvent+molecular+dynamics&doi=10.1021%2Facs.jmedchem.6b00399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Driving Structure-Based Drug Discovery through Cosolvent Molecular Dynamics</span></div><div class="casAuthors">Ghanakota, Phani; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10383-10399</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Identifying binding hotspots on protein surfaces is of prime interest in structure-based drug discovery, either to assess the tractability of pursuing a protein target or to drive improved potency of lead compds.  Computational approaches to detect such regions have traditionally relied on energy minimization of probe mols. onto static protein conformations in the absence of the natural water environment.  Advances in high performance computing now allow the authors to assess hotspots using mol. dynamics (MD) simulations.  MD simulations integrate protein flexibility and the complicated role of water, thereby providing a more realistic assessment of the complex kinetics and thermodn. at play.  In this review, the authors describe the evolution of various cosolvent-based MD techniques and highlight a myriad of potential applications for such technologies in computational drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWZlTX7u7cwLVg90H21EOLACvtfcHk0lh2_a0j84l5iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1yrsbzE&md5=30727d3cbb19bdfb99cdcfa544e5d82d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00399%26sid%3Dliteratum%253Aachs%26aulast%3DGhanakota%26aufirst%3DP.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DDriving%2520structure-based%2520drug%2520discovery%2520through%2520cosolvent%2520molecular%2520dynamics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10383%26epage%3D10399%26doi%3D10.1021%2Facs.jmedchem.6b00399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Tan, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spring, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, C. S.</span><span> </span><span class="NLM_article-title">The application of ligand-mapping molecular dynamics simulations to the rational design of peptidic modulators of protein-protein interactions</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">3199</span><span class="NLM_x">–</span> <span class="NLM_lpage">3210</span><span class="refDoi"> DOI: 10.1021/ct5010577</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct5010577" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps1arurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3199-3210&author=Y.+S.+Tanauthor=D.+R.+Springauthor=C.+Abellauthor=C.+S.+Verma&title=The+application+of+ligand-mapping+molecular+dynamics+simulations+to+the+rational+design+of+peptidic+modulators+of+protein-protein+interactions&doi=10.1021%2Fct5010577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The Application of Ligand-Mapping Molecular Dynamics Simulations to the Rational Design of Peptidic Modulators of Protein-Protein Interactions</span></div><div class="casAuthors">Tan, Yaw Sing; Spring, David R.; Abell, Chris; Verma, Chandra S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3199-3210</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A computational ligand-mapping approach to detect protein surface pockets that interact with hydrophobic moieties is presented.  In this method, we incorporated benzene mols. into explicit solvent mol. dynamics simulations of various protein targets.  The benzene mols. successfully identified the binding locations of hydrophobic hot-spot residues and all-hydrocarbon cross-links from known peptidic ligands.  They also unveiled cryptic binding sites that are occluded by side chains and the protein backbone.  Our results demonstrate that ligand-mapping mol. dynamics simulations hold immense promise to guide the rational design of peptidic modulators of protein-protein interactions, including that of stapled peptides, which show promise as an exciting new class of cell-penetrating therapeutic mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3jTNpkoiL7rVg90H21EOLACvtfcHk0lh2_a0j84l5iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps1arurk%253D&md5=f28bcf448035402f8976142b89d093b3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fct5010577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct5010577%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DY.%2BS.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DVerma%26aufirst%3DC.%2BS.%26atitle%3DThe%2520application%2520of%2520ligand-mapping%2520molecular%2520dynamics%2520simulations%2520to%2520the%2520rational%2520design%2520of%2520peptidic%2520modulators%2520of%2520protein-protein%2520interactions%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3199%26epage%3D3210%26doi%3D10.1021%2Fct5010577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Vallée, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilorge, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dupuy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parent, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fassy, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamberton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrebola, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guerif, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rohaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Certal, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruxer, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delorme, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jouanen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grépin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Combeau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goulaouic, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dereu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikol, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mailliet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minoux, H.</span><span> </span><span class="NLM_article-title">Tricyclic series of heat shock protein 90 (Hsp90) inhibitors part i: Discovery of tricyclic imidazo[4,5-c]pyridines as potent inhibitors of the Hsp90 molecular chaperone</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">7206</span><span class="NLM_x">–</span> <span class="NLM_lpage">7219</span><span class="refDoi"> DOI: 10.1021/jm200784m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200784m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7206-7219&author=F.+Vall%C3%A9eauthor=C.+Carrezauthor=F.+Pilorgeauthor=A.+Dupuyauthor=A.+Parentauthor=L.+Bertinauthor=F.+Thompsonauthor=P.+Ferrariauthor=F.+Fassyauthor=A.+Lambertonauthor=A.+Thomasauthor=R.+Arrebolaauthor=S.+Guerifauthor=A.+Rohautauthor=V.+Certalauthor=J.-M.+Ruxerauthor=C.+Delormeauthor=A.+Jouanenauthor=J.+Dumasauthor=C.+Gr%C3%A9pinauthor=C.+Combeauauthor=H.+Goulaouicauthor=N.+Dereuauthor=V.+Mikolauthor=P.+Maillietauthor=H.+Minoux&title=Tricyclic+series+of+heat+shock+protein+90+%28Hsp90%29+inhibitors+part+i%3A+Discovery+of+tricyclic+imidazo%5B4%2C5-c%5Dpyridines+as+potent+inhibitors+of+the+Hsp90+molecular+chaperone&doi=10.1021%2Fjm200784m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm200784m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200784m%26sid%3Dliteratum%253Aachs%26aulast%3DVall%25C3%25A9e%26aufirst%3DF.%26aulast%3DCarrez%26aufirst%3DC.%26aulast%3DPilorge%26aufirst%3DF.%26aulast%3DDupuy%26aufirst%3DA.%26aulast%3DParent%26aufirst%3DA.%26aulast%3DBertin%26aufirst%3DL.%26aulast%3DThompson%26aufirst%3DF.%26aulast%3DFerrari%26aufirst%3DP.%26aulast%3DFassy%26aufirst%3DF.%26aulast%3DLamberton%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DA.%26aulast%3DArrebola%26aufirst%3DR.%26aulast%3DGuerif%26aufirst%3DS.%26aulast%3DRohaut%26aufirst%3DA.%26aulast%3DCertal%26aufirst%3DV.%26aulast%3DRuxer%26aufirst%3DJ.-M.%26aulast%3DDelorme%26aufirst%3DC.%26aulast%3DJouanen%26aufirst%3DA.%26aulast%3DDumas%26aufirst%3DJ.%26aulast%3DGr%25C3%25A9pin%26aufirst%3DC.%26aulast%3DCombeau%26aufirst%3DC.%26aulast%3DGoulaouic%26aufirst%3DH.%26aulast%3DDereu%26aufirst%3DN.%26aulast%3DMikol%26aufirst%3DV.%26aulast%3DMailliet%26aufirst%3DP.%26aulast%3DMinoux%26aufirst%3DH.%26atitle%3DTricyclic%2520series%2520of%2520heat%2520shock%2520protein%252090%2520%2528Hsp90%2529%2520inhibitors%2520part%2520i%253A%2520Discovery%2520of%2520tricyclic%2520imidazo%255B4%252C5-c%255Dpyridines%2520as%2520potent%2520inhibitors%2520of%2520the%2520Hsp90%2520molecular%2520chaperone%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7206%26epage%3D7219%26doi%3D10.1021%2Fjm200784m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Kung, P.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Funk, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catherine Johnson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, M.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodgers, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayman, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smeal, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maegley, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehring, M. R.</span><span> </span><span class="NLM_article-title">Dihydroxylphenyl amides as inhibitors of the Hsp90 molecular chaperone</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">6273</span><span class="NLM_x">–</span> <span class="NLM_lpage">6278</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.09.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2Fj.bmcl.2008.09.081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=18929486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOhtrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=6273-6278&author=P.-P.+Kungauthor=L.+Funkauthor=J.+Mengauthor=M.+Collinsauthor=J.+Z.+Zhouauthor=M.+Catherine+Johnsonauthor=A.+Ekkerauthor=J.+Wangauthor=P.+Mehtaauthor=M.-J.+Yinauthor=C.+Rodgersauthor=J.+F.+Daviesauthor=E.+Baymanauthor=T.+Smealauthor=K.+A.+Maegleyauthor=M.+R.+Gehring&title=Dihydroxylphenyl+amides+as+inhibitors+of+the+Hsp90+molecular+chaperone&doi=10.1016%2Fj.bmcl.2008.09.081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Dihydroxylphenyl amides as inhibitors of the Hsp90 molecular chaperone</span></div><div class="casAuthors">Kung, Pei-Pei; Funk, Lee; Meng, Jerry; Collins, Michael; Zhou, Joe Zhongxiang; Catherine Johnson, M.; Ekker, Anne; Wang, Jeff; Mehta, Pramod; Yin, Min-Jean; Rodgers, Caroline; Davies, Jay F., II; Bayman, Eileen; Smeal, Tod; Maegley, Karen A.; Gehring, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6273-6278</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Information from X-ray crystal structures were used to optimize the potency of a HTS hit in a Hsp90 competitive binding assay.  A class of novel and potent small mol. Hsp90 inhibitors were thereby identified.  Enantio-pure compds. I [n = 1, 2] were potent in PGA-based competitive binding assay and inhibited proliferation of various human cancer cell lines in vitro, with IC50 values averaging 20 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraBHD3PPbhDbVg90H21EOLACvtfcHk0ljUpvF2EE6SOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOhtrzE&md5=36eb69d7ffb65443bc5328f74806fa0b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.09.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.09.081%26sid%3Dliteratum%253Aachs%26aulast%3DKung%26aufirst%3DP.-P.%26aulast%3DFunk%26aufirst%3DL.%26aulast%3DMeng%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DJ.%2BZ.%26aulast%3DCatherine%2BJohnson%26aufirst%3DM.%26aulast%3DEkker%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DMehta%26aufirst%3DP.%26aulast%3DYin%26aufirst%3DM.-J.%26aulast%3DRodgers%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DJ.%2BF.%26aulast%3DBayman%26aufirst%3DE.%26aulast%3DSmeal%26aufirst%3DT.%26aulast%3DMaegley%26aufirst%3DK.%2BA.%26aulast%3DGehring%26aufirst%3DM.%2BR.%26atitle%3DDihydroxylphenyl%2520amides%2520as%2520inhibitors%2520of%2520the%2520Hsp90%2520molecular%2520chaperone%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D6273%26epage%3D6278%26doi%3D10.1016%2Fj.bmcl.2008.09.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Kung, P.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Digits, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skaptason, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neul, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zientek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elleraas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, M.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gajiwala, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodgers, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehring, M. R.</span><span> </span><span class="NLM_article-title">Dihydroxyphenylisoindoline amides as orally bioavailable inhibitors of the heat shock protein 90 (hsp90) molecular chaperone</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">499</span><span class="NLM_x">–</span> <span class="NLM_lpage">503</span><span class="refDoi"> DOI: 10.1021/jm901209q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901209q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=499-503&author=P.-P.+Kungauthor=B.+Huangauthor=G.+Zhangauthor=J.+Z.+Zhouauthor=J.+Wangauthor=J.+A.+Digitsauthor=J.+Skaptasonauthor=S.+Yamazakiauthor=D.+Neulauthor=M.+Zientekauthor=J.+Elleraasauthor=P.+Mehtaauthor=M.-J.+Yinauthor=M.+J.+Hickeyauthor=K.+S.+Gajiwalaauthor=C.+Rodgersauthor=J.+F.+Daviesauthor=M.+R.+Gehring&title=Dihydroxyphenylisoindoline+amides+as+orally+bioavailable+inhibitors+of+the+heat+shock+protein+90+%28hsp90%29+molecular+chaperone&doi=10.1021%2Fjm901209q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm901209q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901209q%26sid%3Dliteratum%253Aachs%26aulast%3DKung%26aufirst%3DP.-P.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DJ.%2BZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDigits%26aufirst%3DJ.%2BA.%26aulast%3DSkaptason%26aufirst%3DJ.%26aulast%3DYamazaki%26aufirst%3DS.%26aulast%3DNeul%26aufirst%3DD.%26aulast%3DZientek%26aufirst%3DM.%26aulast%3DElleraas%26aufirst%3DJ.%26aulast%3DMehta%26aufirst%3DP.%26aulast%3DYin%26aufirst%3DM.-J.%26aulast%3DHickey%26aufirst%3DM.%2BJ.%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DRodgers%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DJ.%2BF.%26aulast%3DGehring%26aufirst%3DM.%2BR.%26atitle%3DDihydroxyphenylisoindoline%2520amides%2520as%2520orally%2520bioavailable%2520inhibitors%2520of%2520the%2520heat%2520shock%2520protein%252090%2520%2528hsp90%2529%2520molecular%2520chaperone%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D499%26epage%3D503%26doi%3D10.1021%2Fjm901209q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Bruncko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tahir, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judge, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madar, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, C.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elmore, S. W.</span><span> </span><span class="NLM_article-title">N-Aryl-benzimidazolones as novel small molecule HSP90 inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">7503</span><span class="NLM_x">–</span> <span class="NLM_lpage">7506</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2Fj.bmcl.2010.10.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=7503-7506&author=M.+Brunckoauthor=S.+K.+Tahirauthor=X.+Songauthor=J.+Chenauthor=H.+Dingauthor=J.+R.+Huthauthor=S.+Jinauthor=R.+A.+Judgeauthor=D.+J.+Madarauthor=C.+H.+Parkauthor=C.-M.+Parkauthor=A.+M.+Petrosauthor=C.+Tseauthor=S.+H.+Rosenbergauthor=S.+W.+Elmore&title=N-Aryl-benzimidazolones+as+novel+small+molecule+HSP90+inhibitors&doi=10.1016%2Fj.bmcl.2010.10.010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DBruncko%26aufirst%3DM.%26aulast%3DTahir%26aufirst%3DS.%2BK.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DJudge%26aufirst%3DR.%2BA.%26aulast%3DMadar%26aufirst%3DD.%2BJ.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DPark%26aufirst%3DC.-M.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26atitle%3DN-Aryl-benzimidazolones%2520as%2520novel%2520small%2520molecule%2520HSP90%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D7503%26epage%3D7506%26doi%3D10.1016%2Fj.bmcl.2010.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Casale, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amboldi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brasca, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caronni, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felder, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fogliatto, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isacchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polucci, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riceputi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sola, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Visco, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuccotto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casuscelli, F.</span><span> </span><span class="NLM_article-title">Fragment-based hit discovery and structure-based optimization of aminotriazoloquinazolines as novel Hsp90 inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4135</span><span class="NLM_x">–</span> <span class="NLM_lpage">4150</span><span class="refDoi"> DOI: 10.1016/j.bmc.2014.05.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2Fj.bmc.2014.05.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=24980703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVKhtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=4135-4150&author=E.+Casaleauthor=N.+Amboldiauthor=M.+G.+Brascaauthor=D.+Caronniauthor=N.+Colomboauthor=C.+Dalvitauthor=E.+R.+Felderauthor=G.+Fogliattoauthor=A.+Galvaniauthor=A.+Isacchiauthor=P.+Polucciauthor=L.+Riceputiauthor=F.+Solaauthor=C.+Viscoauthor=F.+Zuccottoauthor=F.+Casuscelli&title=Fragment-based+hit+discovery+and+structure-based+optimization+of+aminotriazoloquinazolines+as+novel+Hsp90+inhibitors&doi=10.1016%2Fj.bmc.2014.05.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based hit discovery and structure-based optimization of aminotriazoloquinazolines as novel Hsp90 inhibitors</span></div><div class="casAuthors">Casale, Elena; Amboldi, Nadia; Brasca, Maria Gabriella; Caronni, Dannica; Colombo, Nicoletta; Dalvit, Claudio; Felder, Eduard R.; Fogliatto, Gianpaolo; Galvani, Arturo; Isacchi, Antonella; Polucci, Paolo; Riceputi, Laura; Sola, Francesco; Visco, Carlo; Zuccotto, Fabio; Casuscelli, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4135-4150</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In the last decade the heat shock protein 90 (Hsp90) has emerged as a major therapeutic target and many efforts have been dedicated to the discovery of Hsp90 inhibitors as new potent anticancer agents.  Here the authors report the identification of a novel class of Hsp90 inhibitors by means of a biophys. FAXS-NMR based screening of a library of fragments.  The use of x-ray structure information combined with modeling studies enabled the fragment evolution of the initial triazoloquinazoline hit to a class of compds. with nanomolar potency and drug-like properties suited for further lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8pZK-hye807Vg90H21EOLACvtfcHk0liQSGbbVfMERg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVKhtbfK&md5=7d3d910e96249f6d5550cecc2363af9c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.05.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.05.056%26sid%3Dliteratum%253Aachs%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DAmboldi%26aufirst%3DN.%26aulast%3DBrasca%26aufirst%3DM.%2BG.%26aulast%3DCaronni%26aufirst%3DD.%26aulast%3DColombo%26aufirst%3DN.%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DFelder%26aufirst%3DE.%2BR.%26aulast%3DFogliatto%26aufirst%3DG.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DIsacchi%26aufirst%3DA.%26aulast%3DPolucci%26aufirst%3DP.%26aulast%3DRiceputi%26aufirst%3DL.%26aulast%3DSola%26aufirst%3DF.%26aulast%3DVisco%26aufirst%3DC.%26aulast%3DZuccotto%26aufirst%3DF.%26aulast%3DCasuscelli%26aufirst%3DF.%26atitle%3DFragment-based%2520hit%2520discovery%2520and%2520structure-based%2520optimization%2520of%2520aminotriazoloquinazolines%2520as%2520novel%2520Hsp90%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D4135%26epage%3D4150%26doi%3D10.1016%2Fj.bmc.2014.05.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludlow, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giangreco, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rathi, P. C.</span><span> </span><span class="NLM_article-title">Protein-Ligand-Informatics force field (PLIff): towards a fully knowledge driven “force field” for biomolecular interactions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">6891</span><span class="NLM_x">–</span> <span class="NLM_lpage">6902</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00716</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00716" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6891-6902&author=M.+L.+Verdonkauthor=R.+F.+Ludlowauthor=I.+Giangrecoauthor=P.+C.+Rathi&title=Protein-Ligand-Informatics+force+field+%28PLIff%29%3A+towards+a+fully+knowledge+driven+%E2%80%9Cforce+field%E2%80%9D+for+biomolecular+interactions&doi=10.1021%2Facs.jmedchem.6b00716"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00716%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DGiangreco%26aufirst%3DI.%26aulast%3DRathi%26aufirst%3DP.%2BC.%26atitle%3DProtein-Ligand-Informatics%2520force%2520field%2520%2528PLIff%2529%253A%2520towards%2520a%2520fully%2520knowledge%2520driven%2520%25E2%2580%259Cforce%2520field%25E2%2580%259D%2520for%2520biomolecular%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6891%26epage%3D6902%26doi%3D10.1021%2Facs.jmedchem.6b00716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Jones, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">748</span><span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0liQSGbbVfMERg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Bateman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durbin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollich, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffiths-Jones, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khanna, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moxon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonnhammer, E. L.</span><span> </span><span class="NLM_article-title">The Pfam protein families database</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">D138</span><span class="NLM_x">–</span> <span class="NLM_lpage">D141</span><span class="refDoi"> DOI: 10.1093/nar/gkh121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1093%2Fnar%2Fgkh121" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=D138-D141&author=A.+Batemanauthor=L.+Coinauthor=R.+Durbinauthor=R.+D.+Finnauthor=V.+Hollichauthor=S.+Griffiths-Jonesauthor=A.+Khannaauthor=M.+Marshallauthor=S.+Moxonauthor=E.+L.+Sonnhammer&title=The+Pfam+protein+families+database&doi=10.1093%2Fnar%2Fgkh121"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh121%26sid%3Dliteratum%253Aachs%26aulast%3DBateman%26aufirst%3DA.%26aulast%3DCoin%26aufirst%3DL.%26aulast%3DDurbin%26aufirst%3DR.%26aulast%3DFinn%26aufirst%3DR.%2BD.%26aulast%3DHollich%26aufirst%3DV.%26aulast%3DGriffiths-Jones%26aufirst%3DS.%26aulast%3DKhanna%26aufirst%3DA.%26aulast%3DMarshall%26aufirst%3DM.%26aulast%3DMoxon%26aufirst%3DS.%26aulast%3DSonnhammer%26aufirst%3DE.%2BL.%26atitle%3DThe%2520Pfam%2520protein%2520families%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DD138%26epage%3DD141%26doi%3D10.1093%2Fnar%2Fgkh121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Balaydın, H. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Şentürk, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Menzek, A.</span><span> </span><span class="NLM_article-title">Synthesis and carbonic anhydrase inhibitory properties of novel cyclohexanonyl bromophenol derivatives</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1352</span><span class="NLM_x">–</span> <span class="NLM_lpage">1357</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.12.069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2Fj.bmcl.2011.12.069" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=1352-1357&author=H.+T.+Balayd%C4%B1nauthor=M.+%C5%9Eent%C3%BCrkauthor=A.+Menzek&title=Synthesis+and+carbonic+anhydrase+inhibitory+properties+of+novel+cyclohexanonyl+bromophenol+derivatives&doi=10.1016%2Fj.bmcl.2011.12.069"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.12.069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.12.069%26sid%3Dliteratum%253Aachs%26aulast%3DBalayd%25C4%25B1n%26aufirst%3DH.%2BT.%26aulast%3D%25C5%259Eent%25C3%25BCrk%26aufirst%3DM.%26aulast%3DMenzek%26aufirst%3DA.%26atitle%3DSynthesis%2520and%2520carbonic%2520anhydrase%2520inhibitory%2520properties%2520of%2520novel%2520cyclohexanonyl%2520bromophenol%2520derivatives%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D1352%26epage%3D1357%26doi%3D10.1016%2Fj.bmcl.2011.12.069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Saxty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyle, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barford, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Downham, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrett, M. D.</span><span> </span><span class="NLM_article-title">Identification of inhibitors of protein kinase B using fragment-based lead discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2293</span><span class="NLM_x">–</span> <span class="NLM_lpage">2296</span><span class="refDoi"> DOI: 10.1021/jm070091b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070091b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVKjur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2293-2296&author=G.+Saxtyauthor=S.+J.+Woodheadauthor=V.+Berdiniauthor=T.+G.+Daviesauthor=M.+L.+Verdonkauthor=P.+G.+Wyattauthor=R.+G.+Boyleauthor=D.+Barfordauthor=R.+Downhamauthor=M.+D.+Garrett&title=Identification+of+inhibitors+of+protein+kinase+B+using+fragment-based+lead+discovery&doi=10.1021%2Fjm070091b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Inhibitors of Protein Kinase B Using Fragment-Based Lead Discovery</span></div><div class="casAuthors">Saxty, Gordon; Woodhead, Steven J.; Berdini, Valerio; Davies, Thomas G.; Verdonk, Marcel L.; Wyatt, Paul G.; Boyle, Robert G.; Barford, David; Downham, Robert; Garrett, Michelle D.; Carr, Robin A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2293-2296</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using fragment-based screening techniques, 5-methyl-4-phenyl-1H-pyrazole (IC50 80 μM) was identified as a novel, low mol. wt. inhibitor of protein kinase B (PKB).  Herein we describe the rapid elaboration of highly potent and ligand efficient analogs using a fragment growing approach.  Iterative structure-based design was supported by protein-ligand structure detns. using a PKA-PKB "chimera" and a final protein-ligand structure of a lead compd. in PKBβ itself.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlCYaIT8Xn87Vg90H21EOLACvtfcHk0lg7FQ6r2bLykA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVKjur8%253D&md5=2a09e4bbaca1493be8e215927e7e72e1</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm070091b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070091b%26sid%3Dliteratum%253Aachs%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWoodhead%26aufirst%3DS.%2BJ.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26aulast%3DBoyle%26aufirst%3DR.%2BG.%26aulast%3DBarford%26aufirst%3DD.%26aulast%3DDownham%26aufirst%3DR.%26aulast%3DGarrett%26aufirst%3DM.%2BD.%26atitle%3DIdentification%2520of%2520inhibitors%2520of%2520protein%2520kinase%2520B%2520using%2520fragment-based%2520lead%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2293%26epage%3D2296%26doi%3D10.1021%2Fjm070091b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span> </span><span class="NLM_article-title">The consequences of translational and rotational entropy lost by small molecules on binding to proteins</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">741</span><span class="NLM_x">–</span> <span class="NLM_lpage">753</span><span class="refDoi"> DOI: 10.1023/A:1022446720849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1023%2FA%3A1022446720849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=12650591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD3sXht1Wktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=741-753&author=C.+W.+Murrayauthor=M.+L.+Verdonk&title=The+consequences+of+translational+and+rotational+entropy+lost+by+small+molecules+on+binding+to+proteins&doi=10.1023%2FA%3A1022446720849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The consequences of translational and rotational entropy lost by small molecules on binding to proteins</span></div><div class="casAuthors">Murray, Christopher W.; Verdonk, Marcel L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">741-753</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A review.  When a small mol. binds to a protein, it loses a significant amt. of rigid body translational and rotational entropy.  Ests. of the assocd. energy barrier vary widely in the literature yet accurate ests. are important in the interpretation of results from fragment-based drug discovery techniques.  This paper describes an anal. that allows the estn. of the rigid body entropy barrier from the increase in binding affinities that results when two fragments of known affinity and known binding mode are joined together.  The paper reviews the relatively rare no. of examples where good quality data is available.  From the anal. of this data, we est. that the barrier to binding, due to the loss of rigid-body entropy, is 15-20 kJ/mol, i.e. around 3 orders of magnitude in affinity at 298 K.  This large barrier explains why it is comparatively rare to observe multiple fragments binding to non-overlapping adjacent sites in enzymes.  The barrier is also consistent with medicinal chem. experience where small changes in the crit. binding regions of ligands are often poorly tolerated by enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6KId6AyrMCLVg90H21EOLACvtfcHk0lg7FQ6r2bLykA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXht1Wktrw%253D&md5=4ad6aef74de3cb86302fe16f7e61c1c6</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1023%2FA%3A1022446720849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1022446720849%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26atitle%3DThe%2520consequences%2520of%2520translational%2520and%2520rotational%2520entropy%2520lost%2520by%2520small%2520molecules%2520on%2520binding%2520to%2520proteins%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D741%26epage%3D753%26doi%3D10.1023%2FA%3A1022446720849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Davies, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span> </span><span class="NLM_article-title">Fragment-based discovery of inhibitors of protein kinase B</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1705</span><span class="NLM_x">–</span> <span class="NLM_lpage">1717</span><span class="refDoi"> DOI: 10.2174/156802609790102419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.2174%2F156802609790102419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=19929839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFeitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=1705-1717&author=T.+G.+Daviesauthor=S.+J.+Woodheadauthor=I.+Collins&title=Fragment-based+discovery+of+inhibitors+of+protein+kinase+B&doi=10.2174%2F156802609790102419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based discovery of inhibitors of protein kinase B</span></div><div class="casAuthors">Davies, Thomas G.; Woodhead, Steven J.; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1705-1717</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Multiple ligand efficient fragment inhibitors of protein kinase B were identified through a combined in silico compd. screen and high-throughput crystallog. anal. of protein-ligand structures.  A well-validated apo-PKB-PKA chimeric protein provided a convenient platform for high-throughput crystallog. by soaking of inhibitors, and a method for the detn. of PKB-ligand structures was developed to support inhibitor development.  Pyrazole and azaindole fragment hits with micromolar activity were rapidly progressed to nanomolar inhibitors of PKB with activity in cells using crystallog. anal. of inhibitor binding modes to guide medicinal chem.  Compds. with selectivity for PKB over PKA and other kinases were identified by this approach, resulting in potent inhibitors with in vivo activity through both oral and systemic administration, and suitable for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc9x00Q3BGobVg90H21EOLACvtfcHk0lhe6I4dkVvavg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFeitr8%253D&md5=b06ab1b9f05c336eec0b68d63622907c</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.2174%2F156802609790102419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802609790102419%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DWoodhead%26aufirst%3DS.%2BJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DFragment-based%2520discovery%2520of%2520inhibitors%2520of%2520protein%2520kinase%2520B%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2009%26volume%3D9%26spage%3D1705%26epage%3D1717%26doi%3D10.2174%2F156802609790102419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Hendlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnickel, G.</span><span> </span><span class="NLM_article-title">LIGSITE: automatic and efficient detection of potential small molecule-binding sites in proteins</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">359</span><span class="NLM_x">–</span> <span class="NLM_lpage">363</span><span class="refDoi"> DOI: 10.1016/S1093-3263(98)00002-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2FS1093-3263%2898%2900002-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=9704298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A280%3ADyaK1czmvF2hsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1997&pages=359-363&author=M.+Hendlichauthor=F.+Rippmannauthor=G.+Barnickel&title=LIGSITE%3A+automatic+and+efficient+detection+of+potential+small+molecule-binding+sites+in+proteins&doi=10.1016%2FS1093-3263%2898%2900002-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">LIGSITE: automatic and efficient detection of potential small molecule-binding sites in proteins</span></div><div class="casAuthors">Hendlich M; Rippmann F; Barnickel G</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular graphics & modelling</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">359-63, 389</span>
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    </div><div class="casAbstract">LIGSITE is a new program for the automatic and time-efficient detection of pockets on the surface of proteins that may act as binding sites for small molecule ligands.  Pockets are identified with a series of simple operations on a cubic grid.  Using a set of receptor-ligand complexes we show that LIGSITE is able to identify the binding sites of small molecule ligands with high precision.  The main advantage of LIGSITE is its speed.  Typical search times are in the range of 5 to 20 s for medium-sized proteins.  LIGSITE is therefore well suited for identification of pockets in large sets of proteins (e.g., protein families) for comparative studies.  For graphical display LIGSITE produces VRML representations of the protein-ligand complex and the binding site for display with a VRML viewer such as WebSpace from SGI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRvgmm5GX6HIgaK-_ZudimnfW6udTcc2eak8STgG-Usdbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1czmvF2hsw%253D%253D&md5=b7174632bdeaebcfbedec66824ec5cb6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2898%2900002-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252898%252900002-3%26sid%3Dliteratum%253Aachs%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DBarnickel%26aufirst%3DG.%26atitle%3DLIGSITE%253A%2520automatic%2520and%2520efficient%2520detection%2520of%2520potential%2520small%2520molecule-binding%2520sites%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D1997%26volume%3D15%26spage%3D359%26epage%3D363%26doi%3D10.1016%2FS1093-3263%2898%2900002-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Ludlow, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saini, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tickle, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Detection of secondary binding sites in proteins using fragment screening</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">15910</span><span class="NLM_x">–</span> <span class="NLM_lpage">15915</span><span class="refDoi"> DOI: 10.1073/pnas.1518946112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1073%2Fpnas.1518946112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=26655740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=15910-15915&author=R.+F.+Ludlowauthor=M.+L.+Verdonkauthor=H.+K.+Sainiauthor=I.+J.+Tickleauthor=H.+Jhoti&title=Detection+of+secondary+binding+sites+in+proteins+using+fragment+screening&doi=10.1073%2Fpnas.1518946112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of secondary binding sites in proteins using fragment screening</span></div><div class="casAuthors">Ludlow, R. Frederick; Verdonk, Marcel L.; Saini, Harpreet K.; Tickle, Ian J.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">15910-15915</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Proteins need to be tightly regulated as they control biol. processes in most normal cellular functions.  The precise mechanisms of regulation are rarely completely understood but can involve binding of endogenous ligands and/or partner proteins at specific locations on a protein that can modulate function.  Often, these addnl. secondary binding sites appear sep. to the primary binding site, which, e.g., for an enzyme may bind a substrate.  In previous work, the authors uncovered several examples in which secondary binding sites were discovered on proteins using fragment screening approaches.  In each case, the authors were able to establish that the newly identified secondary binding site was biol. relevant as it was able to modulate function by the binding of a small mol.  Here, the authors investigated how often secondary binding sites are located on proteins by analyzing 24 protein targets for which they performed a fragment screen using x-ray crystallog.  The anal. showed that, surprisingly, the majority of proteins contained secondary binding sites based on their ability to bind fragments.  Furthermore, sequence anal. of these previously unknown sites indicated high conservation, which suggested that they may have a biol. function, perhaps via an allosteric mechanism.  Comparing the physicochem. properties of the secondary sites with known primary ligand-binding sites also showed broad similarities indicating that many of the secondary sites may be druggable in nature with small mols. that could provide new opportunities to modulate potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ITF1w230fbVg90H21EOLACvtfcHk0lhe6I4dkVvavg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP&md5=9e5f15595263dc049c713d5a3f5912a1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1518946112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1518946112%26sid%3Dliteratum%253Aachs%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DSaini%26aufirst%3DH.%2BK.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DDetection%2520of%2520secondary%2520binding%2520sites%2520in%2520proteins%2520using%2520fragment%2520screening%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D15910%26epage%3D15915%26doi%3D10.1073%2Fpnas.1518946112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Mooij, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span> </span><span class="NLM_article-title">General and targeted statistical potentials for protein–ligand interactions</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">272</span><span class="NLM_x">–</span> <span class="NLM_lpage">287</span><span class="refDoi"> DOI: 10.1002/prot.20588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1002%2Fprot.20588" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=272-287&author=W.+Mooijauthor=M.+L.+Verdonk&title=General+and+targeted+statistical+potentials+for+protein%E2%80%93ligand+interactions&doi=10.1002%2Fprot.20588"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fprot.20588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20588%26sid%3Dliteratum%253Aachs%26aulast%3DMooij%26aufirst%3DW.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26atitle%3DGeneral%2520and%2520targeted%2520statistical%2520potentials%2520for%2520protein%25E2%2580%2593ligand%2520interactions%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2005%26volume%3D61%26spage%3D272%26epage%3D287%26doi%3D10.1002%2Fprot.20588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Korb, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stutzle, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Exner, T. E.</span><span> </span><span class="NLM_article-title">Empirical scoring functions for advanced protein– ligand docking with PLANTS</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1021/ci800298z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800298z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=84-96&author=O.+Korbauthor=T.+Stutzleauthor=T.+E.+Exner&title=Empirical+scoring+functions+for+advanced+protein%E2%80%93+ligand+docking+with+PLANTS&doi=10.1021%2Fci800298z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical Scoring Functions for Advanced Protein-Ligand Docking with PLANTS</span></div><div class="casAuthors">Korb, Oliver; Stuetzle, Thomas; Exner, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-96</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper we present two empirical scoring functions, PLANTSCHEMPLP and PLANTSPLP, designed for our docking algorithm PLANTS (Protein-Ligand ANT System), which is based on ant colony optimization (ACO).  They are related, regarding their functional form, to parts of already published scoring functions and force fields.  The parametrization procedure described here was able to identify several parameter settings showing an excellent performance for the task of pose prediction on two test sets comprising 298 complexes in total.  Up to 87% of the complexes of the Astex diverse set and 77% of the CCDC/Astex clean listnc (noncovalently bound complexes of the clean list) could be reproduced with root-mean-square deviations of less than 2 Å with respect to the exptl. detd. structures.  A comparison with the state-of-the-art docking tool GOLD clearly shows that this is, esp. for the druglike Astex diverse set, an improvement in pose prediction performance.  Addnl., optimized parameter settings for the search algorithm were identified, which can be used to balance pose prediction reliability and search speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_kyR7CGk9mbVg90H21EOLACvtfcHk0lhP7GLnjqbUmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVSquw%253D%253D&md5=fce960b75b9f8906354b7c97ff84c092</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fci800298z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800298z%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DStutzle%26aufirst%3DT.%26aulast%3DExner%26aufirst%3DT.%2BE.%26atitle%3DEmpirical%2520scoring%2520functions%2520for%2520advanced%2520protein%25E2%2580%2593%2520ligand%2520docking%2520with%2520PLANTS%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D84%26epage%3D96%26doi%3D10.1021%2Fci800298z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Eldridge, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auton, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paolini, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mee, R. P.</span><span> </span><span class="NLM_article-title">Empirical scoring functions: I. The development of a fast empirical scoring function to estimate the binding affinity of ligands in receptor complexes</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">425</span><span class="NLM_x">–</span> <span class="NLM_lpage">445</span><span class="refDoi"> DOI: 10.1023/A:1007996124545</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1023%2FA%3A1007996124545" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=9385547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK2sXnsV2it7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1997&pages=425-445&author=M.+D.+Eldridgeauthor=C.+W.+Murrayauthor=T.+R.+Autonauthor=G.+V.+Paoliniauthor=R.+P.+Mee&title=Empirical+scoring+functions%3A+I.+The+development+of+a+fast+empirical+scoring+function+to+estimate+the+binding+affinity+of+ligands+in+receptor+complexes&doi=10.1023%2FA%3A1007996124545"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical scoring functions: I. The development of a fast empirical scoring function to estimate the binding affinity of ligands in receptor complexes</span></div><div class="casAuthors">Eldridge, Matthew D.; Murray, Christopher W.; Auton, Timothy R.; Paolini, Gaia V.; Mee, Roger P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">425-445</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">This paper describes the development of a simple empirical scoring function designed to est. the free energy of binding for a protein-ligand complex when the 3D structure of the complex is known or can be approximated.  The function uses simple contact terms to est. lipophilic and metal-ligand binding contributions, a simple explicit form for hydrogen bonds and a term which penalizes flexibility.  The coeffs. of each term are obtained using a regression based on 82 ligand-receptor complexes for which the binding affinity is known.  The function reproduces the binding affinity of the complexes with a cross-validated error of 8.68 kJ/mol.  Tests on internal consistency indicate that the coeffs. obtained are stable to changes in the compn. of the training set.  The function is also tested on two test sets contg. a further 20 and 10 complexes, resp.  The deficiencies of this type of function are discussed and it is compared to approaches by other workers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquJZ-k95NG1LVg90H21EOLACvtfcHk0lhP7GLnjqbUmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsV2it7o%253D&md5=ece4b881e181f3d5c1287af55efaf6b4</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1023%2FA%3A1007996124545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1007996124545%26sid%3Dliteratum%253Aachs%26aulast%3DEldridge%26aufirst%3DM.%2BD.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DAuton%26aufirst%3DT.%2BR.%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DMee%26aufirst%3DR.%2BP.%26atitle%3DEmpirical%2520scoring%2520functions%253A%2520I.%2520The%2520development%2520of%2520a%2520fast%2520empirical%2520scoring%2520function%2520to%2520estimate%2520the%2520binding%2520affinity%2520of%2520ligands%2520in%2520receptor%2520complexes%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1997%26volume%3D11%26spage%3D425%26epage%3D445%26doi%3D10.1023%2FA%3A1007996124545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Jones, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span> </span><span class="NLM_article-title">Molecular recognition of receptor sites using a genetic algorithm with a description of desolvation</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">245</span><span class="NLM_x">, </span> <span class="NLM_fpage">43</span><span class="NLM_x">–</span> <span class="NLM_lpage">53</span><span class="refDoi"> DOI: 10.1016/S0022-2836(95)80037-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2FS0022-2836%2895%2980037-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=7823319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK2MXjtVKlsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=245&publication_year=1995&pages=43-53&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glen&title=Molecular+recognition+of+receptor+sites+using+a+genetic+algorithm+with+a+description+of+desolvation&doi=10.1016%2FS0022-2836%2895%2980037-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of a receptor sites using a genetic algorithm with a description of desolvation</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">245</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-53</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Understanding the principles whereby macromol. biol. receptors can recognize small mol. substrates or inhibitors is the subject of a major effort.  This is of paramount importance in rational drug design where the receptor structure is known (the "docking" problem).  Current theor. approaches utilize models of the steric and electrostatic interaction of bound ligands and recently conformational flexibility has been incorporated.  The authors report results based on software using a genetic algorithm that uses an evolutionary strategy in exploring the full conformational flexibility of the ligand with partial flexibility of the protein, and which satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Results are reported on five test systems showing excellent agreement with exptl. data.  The design of the algorithm offers insight into the mol. recognition mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Xcn7nQyVRLVg90H21EOLACvtfcHk0lhP7GLnjqbUmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjtVKlsbc%253D&md5=bf45edafb986a73c7c1d0ac55e070f28</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2895%2980037-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252895%252980037-9%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26atitle%3DMolecular%2520recognition%2520of%2520receptor%2520sites%2520using%2520a%2520genetic%2520algorithm%2520with%2520a%2520description%2520of%2520desolvation%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1995%26volume%3D245%26spage%3D43%26epage%3D53%26doi%3D10.1016%2FS0022-2836%2895%2980037-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Nadendla, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teyssier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delfosse, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vivat, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnasamy, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gronemeyer, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourguet, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germain, P.</span><span> </span><span class="NLM_article-title">An unexpected mode of binding defines BMS948 as a full retinoic acid receptor β (RARβ, NR1B2) selective agonist</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0123195</span><span class="refDoi"> DOI: 10.1371/journal.pone.0123195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1371%2Fjournal.pone.0123195" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0123195&author=E.+Nadendlaauthor=C.+Teyssierauthor=V.+Delfosseauthor=V.+Vivatauthor=G.+Krishnasamyauthor=H.+Gronemeyerauthor=W.+Bourguetauthor=P.+Germain&title=An+unexpected+mode+of+binding+defines+BMS948+as+a+full+retinoic+acid+receptor+%CE%B2+%28RAR%CE%B2%2C+NR1B2%29+selective+agonist&doi=10.1371%2Fjournal.pone.0123195"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0123195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0123195%26sid%3Dliteratum%253Aachs%26aulast%3DNadendla%26aufirst%3DE.%26aulast%3DTeyssier%26aufirst%3DC.%26aulast%3DDelfosse%26aufirst%3DV.%26aulast%3DVivat%26aufirst%3DV.%26aulast%3DKrishnasamy%26aufirst%3DG.%26aulast%3DGronemeyer%26aufirst%3DH.%26aulast%3DBourguet%26aufirst%3DW.%26aulast%3DGermain%26aufirst%3DP.%26atitle%3DAn%2520unexpected%2520mode%2520of%2520binding%2520defines%2520BMS948%2520as%2520a%2520full%2520retinoic%2520acid%2520receptor%2520%25CE%25B2%2520%2528RAR%25CE%25B2%252C%2520NR1B2%2529%2520selective%2520agonist%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0123195%26doi%3D10.1371%2Fjournal.pone.0123195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">García-Sosa, A. T.</span><span> </span><span class="NLM_article-title">Hydration properties of ligands and drugs in protein binding sites: Tightly-bound, bridging water molecules and their effects and consequences on molecular design strategies</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1388</span><span class="NLM_x">–</span> <span class="NLM_lpage">1405</span><span class="refDoi"> DOI: 10.1021/ci3005786</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3005786" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsVeqt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=1388-1405&author=A.+T.+Garc%C3%ADa-Sosa&title=Hydration+properties+of+ligands+and+drugs+in+protein+binding+sites%3A+Tightly-bound%2C+bridging+water+molecules+and+their+effects+and+consequences+on+molecular+design+strategies&doi=10.1021%2Fci3005786"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Hydration Properties of Ligands and Drugs in Protein Binding Sites: Tightly-Bound, Bridging Water Molecules and Their Effects and Consequences on Molecular Design Strategies</span></div><div class="casAuthors">Garcia-Sosa, Alfonso T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1388-1405</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Some water mols. in binding sites are important for intermol. interactions and stability.  The way binding site explicit water mols. are dealt with affects the diversity and nature of designed ligand chem. structures and properties.  The strategies commonly employed frequently assume that a gain in binding affinity will be achieved by their targeting or neglect.  However, in the present work, 2332 high-resoln. X-ray crystal structures of hydrated and non-hydrated, drug and nondrug compds. in biomol. complexes with reported Ki or Kd show that compds. that use tightly bound, bridging water mols. are as potent as those that do not.  The distribution of their energies, physicochem. properties, and ligand efficiency indexes were compared for statistical significance, and the results were confirmed using 2000 permutation runs.  Ligand cases were also split into agonists and antagonists, and crystal structure pairs with differing tightly bound water mols. were also compared.  In addn., agonists and antagonists that use tightly bound water bridges are smaller, less lipophilic, and less planar; have deeper ligand efficiency indexes; and in general, possess better physicochem. properties for further development.  Therefore, tightly bound, bridging water mols. may in some cases be replaced and targeted as a strategy, though sometimes keeping them as bridges may be better from a pharmacodynamic perspective.  The results suggest general indications on tightly hydrated and non-tightly hydrated compds. in binding sites and practical considerations to adopt a strategy in drug and mol. design when faced with this special type of water mols.  There are also benefits of lower log P and better developability for tightly hydrated compds., while stronger potency is not always required or beneficial.  The hydrated binding site may be one of the many structure conformations available to the receptor, and different ligands will have a different ability to select either hydrated or non-hydrated receptor binding site conformations.  Compds. may thus be designed, and if a tightly bound, bridging water mol. is obsd. in the binding site, attempts to replace it should only be made if the subsequent ligand modification would improve also its ligand efficiency, enthalpy, specificity, and pharmacokinetic properties.  If the modification does succeed in replacing the tightly bound, bridging water mol., it will have at least achieved benefits for ligand optimization and development independently of either pos. or neg. change in binding affinity outcome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWcKH47-JHsLVg90H21EOLACvtfcHk0ljNTsttdGIdHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsVeqt7w%253D&md5=3bbc8769c80c60d1377eb22826647f80</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fci3005786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3005786%26sid%3Dliteratum%253Aachs%26aulast%3DGarc%25C3%25ADa-Sosa%26aufirst%3DA.%2BT.%26atitle%3DHydration%2520properties%2520of%2520ligands%2520and%2520drugs%2520in%2520protein%2520binding%2520sites%253A%2520Tightly-bound%252C%2520bridging%2520water%2520molecules%2520and%2520their%2520effects%2520and%2520consequences%2520on%2520molecular%2520design%2520strategies%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D1388%26epage%3D1405%26doi%3D10.1021%2Fci3005786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">The UniProt Consortium.</span><span> </span><span class="NLM_article-title">UniProt: the universal protein knowledgebase</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">D158</span><span class="NLM_x">–</span> <span class="NLM_lpage">D169</span><span class="refDoi"> DOI: 10.1093/nar/gkw1099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1093%2Fnar%2Fgkw1099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=27899622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D158-D169&author=The+UniProt+Consortium.&title=UniProt%3A+the+universal+protein+knowledgebase&doi=10.1093%2Fnar%2Fgkw1099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1099%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniProt%253A%2520the%2520universal%2520protein%2520knowledgebase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD158%26epage%3DD169%26doi%3D10.1093%2Fnar%2Fgkw1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span>RDKit, Open-source chemoinformatics. <a href="http://www.rdkit.org" class="extLink">http://www.rdkit.org</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=RDKit%2C+Open-source+chemoinformatics.+http%3A%2F%2Fwww.rdkit.org."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Edgar, R. C.</span><span> </span><span class="NLM_article-title">MUSCLE: a multiple sequence alignment method with reduced time and space complexity</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="refDoi"> DOI: 10.1186/1471-2105-5-113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1186%2F1471-2105-5-113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=14706121" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=1&author=R.+C.+Edgar&title=MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity&doi=10.1186%2F1471-2105-5-113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-5-113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-5-113%26sid%3Dliteratum%253Aachs%26aulast%3DEdgar%26aufirst%3DR.%2BC.%26atitle%3DMUSCLE%253A%2520a%2520multiple%2520sequence%2520alignment%2520method%2520with%2520reduced%2520time%2520and%2520space%2520complexity%26jtitle%3DBMC%2520Bioinf.%26date%3D2004%26volume%3D5%26spage%3D1%26doi%3D10.1186%2F1471-2105-5-113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Kearsley, S. K.</span><span> </span><span class="NLM_article-title">On the orthogonal transformation used for structural comparisons</span> <span class="citation_source-journal">Acta Crystallogr., Sect. A: Found. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">208</span><span class="NLM_x">–</span> <span class="NLM_lpage">210</span><span class="refDoi"> DOI: 10.1107/S0108767388010128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1107%2FS0108767388010128" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=1989&pages=208-210&author=S.+K.+Kearsley&title=On+the+orthogonal+transformation+used+for+structural+comparisons&doi=10.1107%2FS0108767388010128"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0108767388010128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108767388010128%26sid%3Dliteratum%253Aachs%26aulast%3DKearsley%26aufirst%3DS.%2BK.%26atitle%3DOn%2520the%2520orthogonal%2520transformation%2520used%2520for%2520structural%2520comparisons%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Found.%2520Crystallogr.%26date%3D1989%26volume%3D45%26spage%3D208%26epage%3D210%26doi%3D10.1107%2FS0108767388010128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Labute, P.</span><span> </span><span class="NLM_article-title">Protonate3D: assignment of ionization states and hydrogen coordinates to macromolecular structures</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">205</span><span class="refDoi"> DOI: 10.1002/prot.22234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1002%2Fprot.22234" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=18814299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisV2isrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2009&pages=187-205&author=P.+Labute&title=Protonate3D%3A+assignment+of+ionization+states+and+hydrogen+coordinates+to+macromolecular+structures&doi=10.1002%2Fprot.22234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Protonate3D: assignment of ionization states and hydrogen coordinates to macromolecular structures</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">187-205</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A new method, called Protonate3D, is presented for the automated prediction of hydrogen coordinates given the 3D coordinates of the heavy atoms of a macromol. structure.  Protonate3D considers side-chain "flip," rotamer, tautomer, and ionization states of all chem. groups, ligands, and solvent, provided suitable templates are available in a parameter file.  The energy model includes van der Waals, Coulomb, solvation, rotamer, tautomer, and titrn. effects.  The results of computational validation expts. suggest that Protonate3D can accurately predict the location of hydrogen atoms in macromol. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmnIRzxpxpCrVg90H21EOLACvtfcHk0ljNTsttdGIdHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisV2isrs%253D&md5=89803fe2028fb9257eb09854a27ef2dd</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fprot.22234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22234%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DProtonate3D%253A%2520assignment%2520of%2520ionization%2520states%2520and%2520hydrogen%2520coordinates%2520to%2520macromolecular%2520structures%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2009%26volume%3D75%26spage%3D187%26epage%3D205%26doi%3D10.1002%2Fprot.22234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span> <span class="citation_source-book">Molecular Operating Environment (MOE), 2013.08</span>; <span class="NLM_publisher-name">Chemical Computing Group Inc.</span>: <span class="NLM_publisher-loc">Montreal, QC, Canada</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Molecular+Operating+Environment+%28MOE%29%2C+2013.08%3B+Chemical+Computing+Group+Inc.%3A+Montreal%2C+QC%2C+Canada%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%252C%25202013.08%26pub%3DChemical%2520Computing%2520Group%2520Inc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">McConkey, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sobolev, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edelman, M.</span><span> </span><span class="NLM_article-title">Quantification of protein surfaces, volumes and atom–atom contacts using a constrained Voronoi procedure</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1365</span><span class="NLM_x">–</span> <span class="NLM_lpage">1373</span><span class="refDoi"> DOI: 10.1093/bioinformatics/18.10.1365</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1093%2Fbioinformatics%2F18.10.1365" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2002&pages=1365-1373&author=B.+J.+McConkeyauthor=V.+Sobolevauthor=M.+Edelman&title=Quantification+of+protein+surfaces%2C+volumes+and+atom%E2%80%93atom+contacts+using+a+constrained+Voronoi+procedure&doi=10.1093%2Fbioinformatics%2F18.10.1365"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2F18.10.1365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252F18.10.1365%26sid%3Dliteratum%253Aachs%26aulast%3DMcConkey%26aufirst%3DB.%2BJ.%26aulast%3DSobolev%26aufirst%3DV.%26aulast%3DEdelman%26aufirst%3DM.%26atitle%3DQuantification%2520of%2520protein%2520surfaces%252C%2520volumes%2520and%2520atom%25E2%2580%2593atom%2520contacts%2520using%2520a%2520constrained%2520Voronoi%2520procedure%26jtitle%3DBioinformatics%26date%3D2002%26volume%3D18%26spage%3D1365%26epage%3D1373%26doi%3D10.1093%2Fbioinformatics%2F18.10.1365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Sastry, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span> </span><span class="NLM_article-title">Rapid shape-based ligand alignment and virtual screening method based on atom/feature-pair similarities and volume overlap scoring</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2455</span><span class="NLM_x">–</span> <span class="NLM_lpage">2466</span><span class="refDoi"> DOI: 10.1021/ci2002704</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2002704" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKnt73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2455-2466&author=G.+M.+Sastryauthor=S.+L.+Dixonauthor=W.+Sherman&title=Rapid+shape-based+ligand+alignment+and+virtual+screening+method+based+on+atom%2Ffeature-pair+similarities+and+volume+overlap+scoring&doi=10.1021%2Fci2002704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Shape-Based Ligand Alignment and Virtual Screening Method Based on Atom/Feature-Pair Similarities and Volume Overlap Scoring</span></div><div class="casAuthors">Sastry, G. Madhavi; Dixon, Steven L.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2455-2466</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Shape-based methods for aligning and scoring ligands have proven to be valuable in the field of computer-aided drug design.  Here, we describe a new shape-based flexible ligand superposition and virtual screening method, Phase Shape, which is shown to rapidly produce accurate 3D ligand alignments and efficiently enrich actives in virtual screening.  We describe the methodol., which is based on the principle of atom distribution triplets to rapidly define trial alignments, followed by refinement of top alignments to maximize the vol. overlap.  The method can be run in a shape-only mode or it can include atom types or pharmacophore feature encoding, the latter consistently producing the best results for database screening.  We apply Phase Shape to flexibly align mols. that bind to the same target and show that the method consistently produces correct alignments when compared with crystal structures.  We then illustrate the effectiveness of the method for identifying active compds. in virtual screening of eleven diverse targets.  Multiple parameters are explored, including atom typing, query structure conformation, and the database conformer generation protocol.  We show that Phase Shape performs well in database screening calcns. when compared with other shape-based methods using a common set of actives and decoys from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM-u23DvDpQ7Vg90H21EOLACvtfcHk0lislzMSBnKRHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKnt73K&md5=4234abb2dd7e6364c377e92047b65aba</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fci2002704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2002704%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DRapid%2520shape-based%2520ligand%2520alignment%2520and%2520virtual%2520screening%2520method%2520based%2520on%2520atom%252Ffeature-pair%2520similarities%2520and%2520volume%2520overlap%2520scoring%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2455%26epage%3D2466%26doi%3D10.1021%2Fci2002704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 22 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Amanda E. Wakefield, Christine Yueh, Dmitri Beglov, Marcelo S. Castilho, Dima Kozakov, György M. Keserű, Adrian Whitty, <span class="NLM_string-name hlFld-ContribAuthor">Sandor Vajda</span>. </span><span class="cited-content_cbyCitation_article-title">Benchmark Sets for Binding Hot Spot Identification in Fragment-Based Ligand Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6612-6623. <a href="https://doi.org/10.1021/acs.jcim.0c00877" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00877</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00877%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBenchmark%252BSets%252Bfor%252BBinding%252BHot%252BSpot%252BIdentification%252Bin%252BFragment-Based%252BLigand%252BDiscovery%26aulast%3DWakefield%26aufirst%3DAmanda%2BE.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D31072020%26date%3D08122020%26volume%3D60%26issue%3D12%26spage%3D6612%26epage%3D6623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kenneth Goossens, Berthold Wroblowski, Cassiano Langini, Herman van Vlijmen, Amedeo Caflisch, <span class="NLM_string-name hlFld-ContribAuthor">Hans De Winter</span>. </span><span class="cited-content_cbyCitation_article-title">Assessment of the Fragment Docking Program SEED. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (10)
                                     , 4881-4893. <a href="https://doi.org/10.1021/acs.jcim.0c00556" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00556</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00556&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00556%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAssessment%252Bof%252Bthe%252BFragment%252BDocking%252BProgram%252BSEED%26aulast%3DGoossens%26aufirst%3DKenneth%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D19052020%26date%3D31082020%26date%3D21082020%26volume%3D60%26issue%3D10%26spage%3D4881%26epage%3D4893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter R. Curran, Chris J. Radoux, Mihaela D. Smilova, Richard A. Sykes, Alicia P. Higueruelo, Anthony R. Bradley, Brian D. Marsden, David R. Spring, Tom L. Blundell, Andrew R. Leach, William R. Pitt, <span class="NLM_string-name hlFld-ContribAuthor">Jason C. Cole</span>. </span><span class="cited-content_cbyCitation_article-title">Hotspots API: A Python Package for the Detection of Small Molecule Binding Hotspots and Application to Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (4)
                                     , 1911-1916. <a href="https://doi.org/10.1021/acs.jcim.9b00996" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00996</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00996%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHotspots%252BAPI%25253A%252BA%252BPython%252BPackage%252Bfor%252Bthe%252BDetection%252Bof%252BSmall%252BMolecule%252BBinding%252BHotspots%252Band%252BApplication%252Bto%252BStructure-Based%252BDrug%252BDesign%26aulast%3DCurran%26aufirst%3DPeter%2BR.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D28102019%26date%3D02042020%26date%3D24032020%26volume%3D60%26issue%3D4%26spage%3D1911%26epage%3D1916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yang Zhang, Jun Biao Ying, Jia Jun Hong, Feng Cheng Li, Ting Ting Fu, Feng Yuan Yang, Guo Xun Zheng, Xiao Jun Yao, Yan Lou, Yunqing Qiu, Wei Wei Xue, <span class="NLM_string-name hlFld-ContribAuthor">Feng Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">How Does Chirality Determine the Selective Inhibition of Histone Deacetylase 6? A Lesson from Trichostatin A Enantiomers Based on Molecular Dynamics. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2019,</strong> <em>10 </em>
                                    (5)
                                     , 2467-2480. <a href="https://doi.org/10.1021/acschemneuro.8b00729" title="DOI URL">https://doi.org/10.1021/acschemneuro.8b00729</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.8b00729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.8b00729%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DHow%252BDoes%252BChirality%252BDetermine%252Bthe%252BSelective%252BInhibition%252Bof%252BHistone%252BDeacetylase%252B6%25253F%252BA%252BLesson%252Bfrom%252BTrichostatin%252BA%252BEnantiomers%252BBased%252Bon%252BMolecular%252BDynamics%26aulast%3DZhang%26aufirst%3DYang%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D24122018%26date%3D20022019%26date%3D26022019%26date%3D20022019%26volume%3D10%26issue%3D5%26spage%3D2467%26epage%3D2480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Tom D. Heightman, James F. Callahan, Elisabetta Chiarparin, Joseph E. Coyle, Charlotte Griffiths-Jones, Ami S. Lakdawala, Rachel McMenamin, Paul N. Mortenson, David Norton, Torren M. Peakman, Sharna J. Rich, Caroline Richardson, William L. Rumsey, Yolanda Sanchez, Gordon Saxty, Henriëtte
M. G. Willems, Lawrence Wolfe, III, Alison J.-A. Woolford, Zining Wu, Hongxing Yan, Jeffrey K. Kerns, <span class="NLM_string-name hlFld-ContribAuthor">Thomas G. Davies</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–Activity and Structure–Conformation Relationships of Aryl Propionic Acid Inhibitors of the Kelch-like ECH-Associated Protein 1/Nuclear Factor Erythroid 2-Related Factor 2 (KEAP1/NRF2) Protein–Protein Interaction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (9)
                                     , 4683-4702. <a href="https://doi.org/10.1021/acs.jmedchem.9b00279" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00279</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00279%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure%2525E2%252580%252593Activity%252Band%252BStructure%2525E2%252580%252593Conformation%252BRelationships%252Bof%252BAryl%252BPropionic%252BAcid%252BInhibitors%252Bof%252Bthe%252BKelch-like%252BECH-Associated%252BProtein%252B1%25252FNuclear%252BFactor%252BErythroid%252B2-Related%252BFactor%252B2%252B%252528KEAP1%25252FNRF2%252529%252BProtein%2525E2%252580%252593Protein%252BInteraction%26aulast%3DHeightman%26aufirst%3DTom%2BD.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D12022019%26date%3D25042019%26date%3D11042019%26volume%3D62%26issue%3D9%26spage%3D4683%26epage%3D4702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Daniel
J. Wood, J. Daniel Lopez-Fernandez, Leanne E. Knight, Islam Al-Khawaldeh, Conghao Gai, Shengying Lin, Mathew P. Martin, Duncan C. Miller, Céline Cano, Jane A. Endicott, Ian R. Hardcastle, Martin E. M. Noble, <span class="NLM_string-name hlFld-ContribAuthor">Michael J. Waring</span>. </span><span class="cited-content_cbyCitation_article-title">FragLites—Minimal, Halogenated Fragments Displaying Pharmacophore Doublets. An Efficient Approach to Druggability Assessment and Hit Generation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (7)
                                     , 3741-3752. <a href="https://doi.org/10.1021/acs.jmedchem.9b00304" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00304</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00304%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragLites%2525E2%252580%252594Minimal%25252C%252BHalogenated%252BFragments%252BDisplaying%252BPharmacophore%252BDoublets.%252BAn%252BEfficient%252BApproach%252Bto%252BDruggability%252BAssessment%252Band%252BHit%252BGeneration%26aulast%3DWood%26aufirst%3DDaniel%2BJ.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D15022019%26date%3D26032019%26date%3D12032019%26volume%3D62%26issue%3D7%26spage%3D3741%26epage%3D3752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Malgorzata
N. Drwal, Guillaume Bret, Carlos Perez, Célien Jacquemard, Jérémy Desaphy, <span class="NLM_string-name hlFld-ContribAuthor">Esther Kellenberger</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Insights on Fragment Binding Mode Conservation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (14)
                                     , 5963-5973. <a href="https://doi.org/10.1021/acs.jmedchem.8b00256" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00256%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructural%252BInsights%252Bon%252BFragment%252BBinding%252BMode%252BConservation%26aulast%3DDrwal%26aufirst%3DMalgorzata%2BN.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D15022018%26date%3D03072018%26date%3D15062018%26volume%3D61%26issue%3D14%26spage%3D5963%26epage%3D5973" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhi-Zheng  Wang</span>, <span class="hlFld-ContribAuthor ">Xing-Xing  Shi</span>, <span class="hlFld-ContribAuthor ">Guang-Yi  Huang</span>, <span class="hlFld-ContribAuthor ">Ge-Fei  Hao</span>, <span class="hlFld-ContribAuthor ">Guang-Fu  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-based drug design facilitates selective kinase inhibitor discovery. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2021,</strong> <em>42 </em>
                                    (7)
                                     , 551-565. <a href="https://doi.org/10.1016/j.tips.2021.04.001" title="DOI URL">https://doi.org/10.1016/j.tips.2021.04.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2021.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2021.04.001%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DFragment-based%252Bdrug%252Bdesign%252Bfacilitates%252Bselective%252Bkinase%252Binhibitor%252Bdiscovery%26aulast%3DWang%26aufirst%3DZhi-Zheng%26date%3D2021%26volume%3D42%26issue%3D7%26spage%3D551%26epage%3D565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maicol  Bissaro</span>, <span class="hlFld-ContribAuthor ">Mattia  Sturlese</span>, <span class="hlFld-ContribAuthor ">Stefano  Moro</span>. </span><span class="cited-content_cbyCitation_article-title">The rise of molecular simulations in fragment-based drug design (FBDD): an overview. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (9)
                                     , 1693-1701. <a href="https://doi.org/10.1016/j.drudis.2020.06.023" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.06.023</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.06.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.06.023%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DThe%252Brise%252Bof%252Bmolecular%252Bsimulations%252Bin%252Bfragment-based%252Bdrug%252Bdesign%252B%252528FBDD%252529%25253A%252Ban%252Boverview%26aulast%3DBissaro%26aufirst%3DMaicol%26date%3D2020%26volume%3D25%26issue%3D9%26spage%3D1693%26epage%3D1701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Clement  Agoni</span>, <span class="hlFld-ContribAuthor ">Fisayo A.  Olotu</span>, <span class="hlFld-ContribAuthor ">Pritika  Ramharack</span>, <span class="hlFld-ContribAuthor ">Mahmoud E.  Soliman</span>. </span><span class="cited-content_cbyCitation_article-title">Druggability and drug-likeness concepts in drug design: are biomodelling and predictive tools having their say?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2020,</strong> <em>26 </em>
                                    (6)
                                     <a href="https://doi.org/10.1007/s00894-020-04385-6" title="DOI URL">https://doi.org/10.1007/s00894-020-04385-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-020-04385-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-020-04385-6%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DDruggability%252Band%252Bdrug-likeness%252Bconcepts%252Bin%252Bdrug%252Bdesign%25253A%252Bare%252Bbiomodelling%252Band%252Bpredictive%252Btools%252Bhaving%252Btheir%252Bsay%25253F%26aulast%3DAgoni%26aufirst%3DClement%26date%3D2020%26date%3D2020%26volume%3D26%26issue%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander D.  MacKerell</span>, <span class="hlFld-ContribAuthor ">Sunhwan  Jo</span>, <span class="hlFld-ContribAuthor ">Sirish Kaushik  Lakkaraju</span>, <span class="hlFld-ContribAuthor ">Christoffer  Lind</span>, <span class="hlFld-ContribAuthor ">Wenbo  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and characterization of fragment binding sites for allosteric ligand design using the site identification by ligand competitive saturation hotspots approach (SILCS-Hotspots). </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - General Subjects</span><span> <strong>2020,</strong> <em>1864 </em>
                                    (4)
                                     , 129519. <a href="https://doi.org/10.1016/j.bbagen.2020.129519" title="DOI URL">https://doi.org/10.1016/j.bbagen.2020.129519</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagen.2020.129519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagen.2020.129519%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520General%2520Subjects%26atitle%3DIdentification%252Band%252Bcharacterization%252Bof%252Bfragment%252Bbinding%252Bsites%252Bfor%252Ballosteric%252Bligand%252Bdesign%252Busing%252Bthe%252Bsite%252Bidentification%252Bby%252Bligand%252Bcompetitive%252Bsaturation%252Bhotspots%252Bapproach%252B%252528SILCS-Hotspots%252529%26aulast%3DMacKerell%26aufirst%3DAlexander%2BD.%26date%3D2020%26volume%3D1864%26issue%3D4%26spage%3D129519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fernando D.  Prieto-Martínez</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Current advances on the development of BET inhibitors: insights from computational methods. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 127-180. <a href="https://doi.org/10.1016/bs.apcsb.2020.06.002" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2020.06.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2020.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2020.06.002%26sid%3Dliteratum%253Aachs%26atitle%3DCurrent%252Badvances%252Bon%252Bthe%252Bdevelopment%252Bof%252BBET%252Binhibitors%25253A%252Binsights%252Bfrom%252Bcomputational%252Bmethods%26aulast%3DPrieto-Mart%25C3%25ADnez%26aufirst%3DFernando%2BD.%26date%3D2020%26spage%3D127%26epage%3D180%26pub%3DElsevier%26date%3D2020%26volume%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrés Felipe  Vásquez</span>, <span class="hlFld-ContribAuthor ">Andrés  González Barrios</span>. </span><span class="cited-content_cbyCitation_article-title">Pushing the Ligand Efficiency Metrics: Relative Group Contribution (RGC) Model as a Helpful Strategy to Promote a Fragment “Rescue” Effect. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2019,</strong> <em>7 </em><a href="https://doi.org/10.3389/fchem.2019.00564" title="DOI URL">https://doi.org/10.3389/fchem.2019.00564</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2019.00564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2019.00564%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DPushing%252Bthe%252BLigand%252BEfficiency%252BMetrics%25253A%252BRelative%252BGroup%252BContribution%252B%252528RGC%252529%252BModel%252Bas%252Ba%252BHelpful%252BStrategy%252Bto%252BPromote%252Ba%252BFragment%252B%2525E2%252580%25259CRescue%2525E2%252580%25259D%252BEffect%26aulast%3DV%25C3%25A1squez%26aufirst%3DAndr%25C3%25A9s%2BFelipe%26date%3D2019%26date%3D2019%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marc  O’Reilly</span>, <span class="hlFld-ContribAuthor ">Anne  Cleasby</span>, <span class="hlFld-ContribAuthor ">Thomas G.  Davies</span>, <span class="hlFld-ContribAuthor ">Richard J.  Hall</span>, <span class="hlFld-ContribAuthor ">R. Frederick  Ludlow</span>, <span class="hlFld-ContribAuthor ">Christopher W.  Murray</span>, <span class="hlFld-ContribAuthor ">Dominic  Tisi</span>, <span class="hlFld-ContribAuthor ">Harren  Jhoti</span>. </span><span class="cited-content_cbyCitation_article-title">Crystallographic screening using ultra-low-molecular-weight ligands to guide drug design. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2019,</strong> <em>24 </em>
                                    (5)
                                     , 1081-1086. <a href="https://doi.org/10.1016/j.drudis.2019.03.009" title="DOI URL">https://doi.org/10.1016/j.drudis.2019.03.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2019.03.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2019.03.009%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DCrystallographic%252Bscreening%252Busing%252Bultra-low-molecular-weight%252Bligands%252Bto%252Bguide%252Bdrug%252Bdesign%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DMarc%26date%3D2019%26volume%3D24%26issue%3D5%26spage%3D1081%26epage%3D1086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kai  Sommer</span>, <span class="hlFld-ContribAuthor ">Florian  Flachsenberg</span>, <span class="hlFld-ContribAuthor ">Matthias  Rarey</span>. </span><span class="cited-content_cbyCitation_article-title">NAOMInext – Synthetically feasible fragment growing in a structure-based design context. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>163 </em>, 747-762. <a href="https://doi.org/10.1016/j.ejmech.2018.11.075" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.11.075</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.11.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.11.075%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNAOMInext%252B%2525E2%252580%252593%252BSynthetically%252Bfeasible%252Bfragment%252Bgrowing%252Bin%252Ba%252Bstructure-based%252Bdesign%252Bcontext%26aulast%3DSommer%26aufirst%3DKai%26date%3D2019%26volume%3D163%26spage%3D747%26epage%3D762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fernando D.  Prieto-Martínez</span>, <span class="hlFld-ContribAuthor ">Ulf  Norinder</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Cheminformatics Explorations of Natural Products. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 1-35. <a href="https://doi.org/10.1007/978-3-030-14632-0_1" title="DOI URL">https://doi.org/10.1007/978-3-030-14632-0_1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-14632-0_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-14632-0_1%26sid%3Dliteratum%253Aachs%26atitle%3DCheminformatics%252BExplorations%252Bof%252BNatural%252BProducts%26aulast%3DPrieto-Mart%25C3%25ADnez%26aufirst%3DFernando%2BD.%26date%3D2019%26date%3D2019%26spage%3D1%26epage%3D35%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DProgress%252Bin%252Bthe%252BChemistry%252Bof%252BOrganic%252BNatural%252BProducts%252B110%26aulast%3DKinghorn%26aufirst%3DA.%2BDouglas%26date%3D2019%26volume%3D110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lucas  Defelipe</span>, <span class="hlFld-ContribAuthor ">Juan  Arcon</span>, <span class="hlFld-ContribAuthor ">Carlos  Modenutti</span>, <span class="hlFld-ContribAuthor ">Marcelo  Marti</span>, <span class="hlFld-ContribAuthor ">Adrián  Turjanski</span>, <span class="hlFld-ContribAuthor ">Xavier  Barril</span>. </span><span class="cited-content_cbyCitation_article-title">Solvents to Fragments to Drugs: MD Applications in Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (12)
                                     , 3269. <a href="https://doi.org/10.3390/molecules23123269" title="DOI URL">https://doi.org/10.3390/molecules23123269</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23123269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23123269%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DSolvents%252Bto%252BFragments%252Bto%252BDrugs%25253A%252BMD%252BApplications%252Bin%252BDrug%252BDesign%26aulast%3DDefelipe%26aufirst%3DLucas%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D12%26spage%3D3269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guoxun  Zheng</span>, <span class="hlFld-ContribAuthor ">Fengyuan  Yang</span>, <span class="hlFld-ContribAuthor ">Tingting  Fu</span>, <span class="hlFld-ContribAuthor ">Gao  Tu</span>, <span class="hlFld-ContribAuthor ">Yuzong  Chen</span>, <span class="hlFld-ContribAuthor ">Xiaojun  Yao</span>, <span class="hlFld-ContribAuthor ">Weiwei  Xue</span>, <span class="hlFld-ContribAuthor ">Feng  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Computational characterization of the selective inhibition of human norepinephrine and serotonin transporters by an escitalopram scaffold. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2018,</strong> <em>20 </em>
                                    (46)
                                     , 29513-29527. <a href="https://doi.org/10.1039/C8CP06232C" title="DOI URL">https://doi.org/10.1039/C8CP06232C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8CP06232C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8CP06232C%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DComputational%252Bcharacterization%252Bof%252Bthe%252Bselective%252Binhibition%252Bof%252Bhuman%252Bnorepinephrine%252Band%252Bserotonin%252Btransporters%252Bby%252Ban%252Bescitalopram%252Bscaffold%26aulast%3DZheng%26aufirst%3DGuoxun%26date%3D2018%26date%3D2018%26volume%3D20%26issue%3D46%26spage%3D29513%26epage%3D29527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sinisa  Vukovic</span>, <span class="hlFld-ContribAuthor ">David J.  Huggins</span>. </span><span class="cited-content_cbyCitation_article-title">Quantitative metrics for drug–target ligandability. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2018,</strong> <em>23 </em>
                                    (6)
                                     , 1258-1266. <a href="https://doi.org/10.1016/j.drudis.2018.02.015" title="DOI URL">https://doi.org/10.1016/j.drudis.2018.02.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2018.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2018.02.015%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DQuantitative%252Bmetrics%252Bfor%252Bdrug%2525E2%252580%252593target%252Bligandability%26aulast%3DVukovic%26aufirst%3DSinisa%26date%3D2018%26volume%3D23%26issue%3D6%26spage%3D1258%26epage%3D1266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lucy J  Colwell</span>. </span><span class="cited-content_cbyCitation_article-title">Statistical and machine learning approaches to predicting protein–ligand interactions. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2018,</strong> <em>49 </em>, 123-128. <a href="https://doi.org/10.1016/j.sbi.2018.01.006" title="DOI URL">https://doi.org/10.1016/j.sbi.2018.01.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2018.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2018.01.006%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DStatistical%252Band%252Bmachine%252Blearning%252Bapproaches%252Bto%252Bpredicting%252Bprotein%2525E2%252580%252593ligand%252Binteractions%26aulast%3DColwell%26aufirst%3DLucy%2BJ%26date%3D2018%26volume%3D49%26spage%3D123%26epage%3D128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">, , <span class="hlFld-ContribAuthor ">Honorine  Lebraud</span>, <span class="hlFld-ContribAuthor ">Tom D.  Heightman</span>. </span><span class="cited-content_cbyCitation_article-title">Protein degradation: a validated therapeutic strategy with exciting prospects. </span><span class="cited-content_cbyCitation_journal-name">Essays in Biochemistry</span><span> <strong>2017,</strong> <em>61 </em>
                                    (5)
                                     , 517-527. <a href="https://doi.org/10.1042/EBC20170030" title="DOI URL">https://doi.org/10.1042/EBC20170030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/EBC20170030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FEBC20170030%26sid%3Dliteratum%253Aachs%26jtitle%3DEssays%2520in%2520Biochemistry%26atitle%3DProtein%252Bdegradation%25253A%252Ba%252Bvalidated%252Btherapeutic%252Bstrategy%252Bwith%252Bexciting%252Bprospects%26aulast%3Dvan%2BMontfort%26aufirst%3DRob%2BL.M.%26date%3D2017%26date%3D2017%26volume%3D61%26issue%3D5%26spage%3D517%26epage%3D527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">György M  Keserű </span>, <span class="hlFld-ContribAuthor ">Michael M  Hann</span>. </span><span class="cited-content_cbyCitation_article-title">What is the future for fragment-based drug discovery?. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2017,</strong> <em>9 </em>
                                    (13)
                                     , 1457-1460. <a href="https://doi.org/10.4155/fmc-2017-0106" title="DOI URL">https://doi.org/10.4155/fmc-2017-0106</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2017-0106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2017-0106%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DWhat%252Bis%252Bthe%252Bfuture%252Bfor%252Bfragment-based%252Bdrug%252Bdiscovery%25253F%26aulast%3DKeser%25C5%25B1%25E2%2580%258A%26aufirst%3DGy%25C3%25B6rgy%2BM%26date%3D2017%26volume%3D9%26issue%3D13%26spage%3D1457%26epage%3D1460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical diagrams of the fragments in the validation set. The charge and tautomeric states of the fragments were assigned in their bound conformation in such a way that the hydrogen bonding network with their partner proteins is optimized. Note that fragments 15 and 16 differ only in their charge states.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Distribution of physicochemical properties of the fragments in the validation set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Examples of “hot” and “warm” spots in the validation set: (a) phenyl-sulfonamide cluster in the CA2 binding site (complex ID 26) and (b) chloro-phenyl cluster in the PKA-Akt chimera (complex ID 17). Ligand occupancy maps contoured at 0.8 (blue) and 0.5 (orange) are also shown for both examples (c and d).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Examples showing successful predictions from different protocols: (a) indole against K-Ras (complex ID 24) from AMIsite; (b) 4-oxo-3,4-dihydroquinazoline against Tankyrase-1 (complex ID 40) from GOLD using ChemPLP for driving the docking and Goldscore as rescoring function; (c) benzoic acid against RARβ (complex ID 19) from the PLImap protocol. The top scoring pose (or the site identified in the case of AMIsite) is shown in magenta sticks (plus signs). Ligands from the representative PDB structure are shown as sticks; carbons forming part of the fragment and in the rest of the ligand are colored green and gray, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Examples showing PLIff energy maps generated by the PLImap protocol for (a) RARβ (complex ID 19) and (b) eNOS (complex ID 22). Ligand and the top-ranking pose are represented in the same style as in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>. The maps show the lowest PLIff energies among all orientations of the corresponding probe molecule centered on a grid point. The maps are contoured at −3.2 (blue) and −2.8 kcal/mol (orange) in the case of RARβ and −3.0 (blue) and −2.3 kcal/mol (orange) in the case of eNOS.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Challenging examples in the validation set: (a) fluoro-phenyl-hepatocyte growth factor receptor (complex ID 18), (b) 3,5-dimethylisoxazole-BRD4 (complex ID 36), (c) resorcinol-HSP90 (complex ID 31), and (d) chloro-phenyl-PKA-Akt chimera (complex ID 17). The top-ranking poses from the PLImap protocol (a and b) and GOLD docking with ChemPLP rescoring function (c and d) are shown as in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/medium/jm-2017-00366t_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Shred selection workflow. HAC stands for heavy atom count.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00366/20170505/images/large/jm-2017-00366t_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00366&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i19">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66045" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66045" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 53 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, R.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">660</span><span class="NLM_x">–</span> <span class="NLM_lpage">672</span><span class="refDoi"> DOI: 10.1038/nrd1467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1038%2Fnrd1467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=15286733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=660-672&author=D.+C.+Reesauthor=M.+Congreveauthor=C.+W.+Murrayauthor=R.+Carr&title=Fragment-based+lead+discovery&doi=10.1038%2Fnrd1467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery</span></div><div class="casAuthors">Rees, David C.; Congreve, Miles; Murray, Christopher W.; Carr, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">660-672</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery is gaining momentum in both large pharmaceutical companies and biotechnol. labs. as a complementary approach to traditional screening.  This is because fragment-based approaches require significantly fewer compds. to be screened and synthesized, and are showing a high success rate in generating chem. series with lead-like properties.  Compared with traditional screening hits, the starting fragments have considerably lower mol. mass, and although the binding interactions of these fragments with a target protein are weak, they are structurally understood through x-ray crystallog. or NMR, and they exhibit high 'ligand efficiency'.  Here, the authors use examples from 25 different protein targets to describe chem. strategies that exploit this structural knowledge to rapidly develop fragments into high-affinity leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbKeUzFFIlhbVg90H21EOLACvtfcHk0lgtKWZzgSwh1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D&md5=8d49ac7f698c9b9d573ee3c3485eced8</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd1467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1467%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCarr%26aufirst%3DR.%26atitle%3DFragment-based%2520lead%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D660%26epage%3D672%26doi%3D10.1038%2Fnrd1467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span> </span><span class="NLM_article-title">Introduction to fragment-based drug discovery</span>. In  <span class="citation_source-book">Fragment-based drug discovery and X-ray crystallography</span>; <span class="NLM_contrib-group">Davies, T. G.; Hyvönen, M.</span>, Eds.; <span class="NLM_publisher-name">Springer-Verlag</span>: <span class="NLM_publisher-loc">Berlin Heidelberg</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1007%2F128_2011_180" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=1-32&author=D.+A.+Erlansonauthor=T.+G.+Davies&author=M.+Hyv%C3%B6nen&title=Fragment-based+drug+discovery+and+X-ray+crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2F128_2011_180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F128_2011_180%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26atitle%3DIntroduction%2520to%2520fragment-based%2520drug%2520discovery%26btitle%3DFragment-based%2520drug%2520discovery%2520and%2520X-ray%2520crystallography%26aulast%3DDavies%26aufirst%3DT.%2BG.%26pub%3DSpringer-Verlag%26date%3D2011%26spage%3D1%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alex, A.</span><span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2ebMt54QVjxsfLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Allen, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellamacina, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffery, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petsko, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">An experimental approach to mapping the binding surfaces of crystalline proteins</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">2605</span><span class="NLM_x">–</span> <span class="NLM_lpage">2611</span><span class="refDoi"> DOI: 10.1021/jp952516o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp952516o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK28Xmt1GrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1996&pages=2605-2611&author=K.+N.+Allenauthor=C.+R.+Bellamacinaauthor=X.+Dingauthor=C.+J.+Jefferyauthor=C.+Mattosauthor=G.+A.+Petskoauthor=D.+Ringe&title=An+experimental+approach+to+mapping+the+binding+surfaces+of+crystalline+proteins&doi=10.1021%2Fjp952516o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">An Experimental Approach to Mapping the Binding Surfaces of Crystalline Proteins</span></div><div class="casAuthors">Allen, Karen N.; Bellamacina, Cornelia R.; Ding, Xiaochun; Jeffery, Constance J.; Mattos, Carla; Petsko, Gregory A.; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2605-11</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Porcine pancreatic elastase has been used as the model enzyme in the design and development of a crystallog. method that allows mapping of the binding surface of a protein by solving its crystal structure in a variety of org. solvents.  The ultimate goal of this method is to aid in the process of drug design, where each of the chosen org. mols. represents a given functional group in a larger inhibitor mol.  This method of multiple solvent crystal structures (MSCS) has a theor. counterpart in the method of multiply copy simultaneous search (MCSS) (Miranker, A.; Karplus, M. Proteins: Struct., Funct., Genet. 1991, 11, 29-34) and is the first exptl. method that can be used as a check to the theory.  The MSCS method is presented here with acetonitrile as the probe org. solvent.  The procedure involved does not cause significant changes in the structure of elastase as compared to the structure in aq. soln., and the positions found for the acetonitrile mols. in the active site are compared to those of similar functional groups belonging to known inhibitors bound to elastase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhPgz3NRlYqrVg90H21EOLACvtfcHk0ljNfK-3Ubbjog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xmt1GrtQ%253D%253D&md5=c98dea1a223f30e444b9bc7faff25f0b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjp952516o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp952516o%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DBellamacina%26aufirst%3DC.%2BR.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DJeffery%26aufirst%3DC.%2BJ.%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DAn%2520experimental%2520approach%2520to%2520mapping%2520the%2520binding%2520surfaces%2520of%2520crystalline%2520proteins%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1996%26volume%3D100%26spage%3D2605%26epage%3D2611%26doi%3D10.1021%2Fjp952516o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">Locating and characterizing binding sites on proteins</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">595</span><span class="NLM_x">–</span> <span class="NLM_lpage">599</span><span class="refDoi"> DOI: 10.1038/nbt0596-595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1038%2Fnbt0596-595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=9630949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK28Xis1eksr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=595-599&author=C.+Mattosauthor=D.+Ringe&title=Locating+and+characterizing+binding+sites+on+proteins&doi=10.1038%2Fnbt0596-595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Locating and characterizing binding sites of proteins</span></div><div class="casAuthors">Mattos, Carla; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">595-9</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">A review with 38 refs.  This review article begins with a discussion of fundamental differences between substrates and inhibitors, and some of the assumptions and goals underlying the design of a new ligand to a target protein.  An overview is given of the methods currently used to locate and characterize ligand binding sites on protein surfaces, with focus on a novel approach: multiple solvent crystal structures (MSCS).  In this method, the x-ray crystal structure of the target protein is solved in a variety or org. solvents.  Each type of solvent mol. serves as a probe for complementary binding sites on the protein.  The probe distribution on the protein surface allows the location of binding sites and the characterization of the potential ligand interactions within these sites.  General aspects of the application of the MSCS method to porcine pancreatic elastase is discussed, and comparison of the results with those from x-ray crystal structures of elastase/inhibitor complexes is used to illustrate the potential of the method in aiding the process of rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDEKZ8sM6gk7Vg90H21EOLACvtfcHk0ljNfK-3Ubbjog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis1eksr0%253D&md5=4facd2f240311e8937623352e7d933d3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnbt0596-595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt0596-595%26sid%3Dliteratum%253Aachs%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DLocating%2520and%2520characterizing%2520binding%2520sites%2520on%2520proteins%26jtitle%3DNat.%2520Biotechnol.%26date%3D1996%26volume%3D14%26spage%3D595%26epage%3D599%26doi%3D10.1038%2Fnbt0596-595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Mattos, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellamacina, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peisach, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pereira, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vitkup, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petsko, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span> </span><span class="NLM_article-title">Multiple solvent crystal structures: probing binding sites, plasticity and hydration</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">357</span><span class="NLM_x">, </span> <span class="NLM_fpage">1471</span><span class="NLM_x">–</span> <span class="NLM_lpage">1482</span><span class="refDoi"> DOI: 10.1016/j.jmb.2006.01.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2Fj.jmb.2006.01.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=16488429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD28XivVGqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2006&pages=1471-1482&author=C.+Mattosauthor=C.+R.+Bellamacinaauthor=E.+Peisachauthor=A.+Pereiraauthor=D.+Vitkupauthor=G.+A.+Petskoauthor=D.+Ringe&title=Multiple+solvent+crystal+structures%3A+probing+binding+sites%2C+plasticity+and+hydration&doi=10.1016%2Fj.jmb.2006.01.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Solvent Crystal Structures: Probing Binding Sites, Plasticity and Hydration</span></div><div class="casAuthors">Mattos, Carla; Bellamacina, Cornelia R.; Peisach, Ezra; Pereira, Antonio; Vitkup, Dennis; Petsko, Gregory A.; Ringe, Dagmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">357</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1471-1482</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Multiple solvent crystal structures (MSCS) of porcine pancreatic elastase were used to map the binding surface the enzyme.  Crystal structures of elastase in neat acetonitrile, 95% acetone, 55% DMF, 80% 5-hexene-1,2-diol, 80% isopropanol, 80% ethanol and 40% trifluoroethanol showed that the org. solvent mols. clustered in the active site, were found mostly unclustered in crystal contacts and in general did not bind elsewhere on the surface of elastase.  Mixts. of 40% benzene or 40% cyclohexane in 50% isopropanol and 10% water showed no bound benzene or cyclohexane mols., but did reveal bound isopropanol. The clusters of org. solvent probe mols. coincide with pockets occupied by known inhibitors.  MSCS also reveal the areas of plasticity within the elastase binding site and allow for the visualization of a nearly complete first hydration shell.  The pattern of org. solvent clusters detd. by MSCS for elastase is consistent with patterns for hot spots in protein-ligand interactions detd. from database anal. in general.  The MSCS method allows probing of hot spots, plasticity and hydration simultaneously, providing a powerful complementary strategy to guide computational methods currently in development for binding site detn., ligand docking and design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq66xX3ZT7wRLVg90H21EOLACvtfcHk0libwW2LhzcgCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivVGqtrk%253D&md5=ab7064d371e5557b2f91aeb6c262a5dc</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2006.01.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2006.01.039%26sid%3Dliteratum%253Aachs%26aulast%3DMattos%26aufirst%3DC.%26aulast%3DBellamacina%26aufirst%3DC.%2BR.%26aulast%3DPeisach%26aufirst%3DE.%26aulast%3DPereira%26aufirst%3DA.%26aulast%3DVitkup%26aufirst%3DD.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26atitle%3DMultiple%2520solvent%2520crystal%2520structures%253A%2520probing%2520binding%2520sites%252C%2520plasticity%2520and%2520hydration%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D357%26spage%3D1471%26epage%3D1482%26doi%3D10.1016%2Fj.jmb.2006.01.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">English, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span> </span><span class="NLM_article-title">Experimental and computational mapping of the binding surface of a crystalline protein</span> <span class="citation_source-journal">Protein Eng., Des. Sel.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">59</span><span class="refDoi"> DOI: 10.1093/protein/14.1.47</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1093%2Fprotein%2F14.1.47" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2001&pages=47-59&author=A.+C.+Englishauthor=C.+R.+Groomauthor=R.+E.+Hubbard&title=Experimental+and+computational+mapping+of+the+binding+surface+of+a+crystalline+protein&doi=10.1093%2Fprotein%2F14.1.47"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F14.1.47&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F14.1.47%26sid%3Dliteratum%253Aachs%26aulast%3DEnglish%26aufirst%3DA.%2BC.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DExperimental%2520and%2520computational%2520mapping%2520of%2520the%2520binding%2520surface%2520of%2520a%2520crystalline%2520protein%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D2001%26volume%3D14%26spage%3D47%26epage%3D59%26doi%3D10.1093%2Fprotein%2F14.1.47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Goodford, P. J.</span><span> </span><span class="NLM_article-title">A computational procedure for determining energetically favorable binding sites on biologically important macromolecules</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">849</span><span class="NLM_x">–</span> <span class="NLM_lpage">857</span><span class="refDoi"> DOI: 10.1021/jm00145a002</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00145a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaL2MXktVeit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1985&pages=849-857&author=P.+J.+Goodford&title=A+computational+procedure+for+determining+energetically+favorable+binding+sites+on+biologically+important+macromolecules&doi=10.1021%2Fjm00145a002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A computational procedure for determining energetically favorable binding sites on biologically important macromolecules</span></div><div class="casAuthors">Goodford, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">849-57</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The interaction of a probe group with a protein of known structure is computed at sample positions throughout and around the macromol., giving an array of energy values.  The probe include H2O, the Me group, amine-N, carboxy-O, and OH.  Contour surfaces at appropriate energy levels are calcd. for each probe and displayed by computer graphics together with the protein structure.  Contours at neg. energy levels delineate regions of attraction between probe and protein and are found at known ligand binding clefts in particular.  The contours also enable other regions of attraction to be identified and facilitate the interpretation of protein-ligand energetics.  They may, therefore, be of value for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3tYN3Fc76dLVg90H21EOLACvtfcHk0libwW2LhzcgCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXktVeit7c%253D&md5=03e9d22787f16c760ef0dc29ca02aa88</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm00145a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00145a002%26sid%3Dliteratum%253Aachs%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26atitle%3DA%2520computational%2520procedure%2520for%2520determining%2520energetically%2520favorable%2520binding%2520sites%2520on%2520biologically%2520important%2520macromolecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1985%26volume%3D28%26spage%3D849%26epage%3D857%26doi%3D10.1021%2Fjm00145a002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Wade, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodford, P. J.</span><span> </span><span class="NLM_article-title">Further development of hydrogen bond functions for use in determining energetically favorable binding sites on molecules of known structure. 1. Ligand probe groups with the ability to form two hydrogen bonds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">140</span><span class="NLM_x">–</span> <span class="NLM_lpage">147</span><span class="refDoi"> DOI: 10.1021/jm00053a018</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00053a018" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK3sXltFymsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1993&pages=140-147&author=R.+C.+Wadeauthor=K.+J.+Clarkauthor=P.+J.+Goodford&title=Further+development+of+hydrogen+bond+functions+for+use+in+determining+energetically+favorable+binding+sites+on+molecules+of+known+structure.+1.+Ligand+probe+groups+with+the+ability+to+form+two+hydrogen+bonds&doi=10.1021%2Fjm00053a018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Further development of hydrogen bond functions for use in determining energetically favorable binding sites on molecules of known structure.  1.  Ligand probe groups with the ability to form two hydrogen bonds</span></div><div class="casAuthors">Wade, Rebecca C.; Clark, Kevin J.; Goodford, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">140-7</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The directional properties of hydrogen bonds play a major role in detg. the specificity of intermol. interactions.  An energy function which takes explicit account of these properties was developed for use in the detn. of energetically favorable ligand binding sites on mols. of known structure by the GRID method.  In this method, the interaction energy between a target mol. and a small chem. group (a probe), which may be part of a larger ligand, was calcd. using an energy function consisting of Lennard-Jones, electrostatic, and hydrogen bond terms.  The latter term was a function of the length of the hydrogen bond, its orientation at the hydrogen-bonding atoms, and their chem. nature.  Hydrogen bond energy functions are described which take account of the spatial distribution of the hydrogen bonds made by probes with the ability to form 2 hydrogen bonds.  These functions were designed so as to model the exptl. obsd. angular dependence of the hydrogen bonds.  The procedure to locate the position and orientation of the probe at which the interaction energy is optimized, is also described..  The use of this procedure was demonstrated by examples of biol. and pharmacol. interest which show that it can produce results that are consistent with other theor. approaches and with exptl. observations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWcTTi-uqMA7Vg90H21EOLACvtfcHk0libwW2LhzcgCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXltFymsA%253D%253D&md5=c7130820648cd5c7b94f37c4b3a29b8b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm00053a018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00053a018%26sid%3Dliteratum%253Aachs%26aulast%3DWade%26aufirst%3DR.%2BC.%26aulast%3DClark%26aufirst%3DK.%2BJ.%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26atitle%3DFurther%2520development%2520of%2520hydrogen%2520bond%2520functions%2520for%2520use%2520in%2520determining%2520energetically%2520favorable%2520binding%2520sites%2520on%2520molecules%2520of%2520known%2520structure.%25201.%2520Ligand%2520probe%2520groups%2520with%2520the%2520ability%2520to%2520form%2520two%2520hydrogen%2520bonds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1993%26volume%3D36%26spage%3D140%26epage%3D147%26doi%3D10.1021%2Fjm00053a018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Wade, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodford, P. J.</span><span> </span><span class="NLM_article-title">Further development of hydrogen bond functions for use in determining energetically favorable binding sites on molecules of known structure. 2. Ligand probe groups with the ability to form more than two hydrogen bonds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">148</span><span class="NLM_x">–</span> <span class="NLM_lpage">156</span><span class="refDoi"> DOI: 10.1021/jm00053a019</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00053a019" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK3sXjsFeqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1993&pages=148-156&author=R.+C.+Wadeauthor=P.+J.+Goodford&title=Further+development+of+hydrogen+bond+functions+for+use+in+determining+energetically+favorable+binding+sites+on+molecules+of+known+structure.+2.+Ligand+probe+groups+with+the+ability+to+form+more+than+two+hydrogen+bonds&doi=10.1021%2Fjm00053a019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Further development of hydrogen bond functions for use in determining energetically favorable binding sites on molecules of known structure.  2.  Ligand probe groups with the ability to form more than two hydrogen bonds</span></div><div class="casAuthors">Wade, Rebecca C.; Goodford, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">148-56</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The specificity of interactions between biol. macromols. and their ligands may be partially attributed to the directional properties of H bonds.  The GRID method of P. J. Goodford (1985) and of D. N. A. Boobbyer et al. (1989) for detg. energetically favorable ligand binding sites on mols. of known structure was extended to improve the treatment of groups which can make multiple H bonds.  In this method, the interaction energy between a probe (a small chem. group that may be part of a larger ligand) and a target mol. is calcd. by using an energy function that includes a H bond term that is dependent on the length of the H bond, its orientation at the H-bonding atoms, and their chem. character.  The methods described in the preceding paper (Wade, R. C. et al., 1992) for probes capable of making 2 H bonds are here extended to the following probes which have the ability to make > 2 H bonds: ammonium-NH3+, amine-NH2, sp3-hybridized hydroxyl, and water.  Use of the improved GRID procedure is demonstrated by detn. of the conformation of an amino acid side chain at the subunit interface in Hb and of the location of water binding sites in human lysozyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9XWnnyz8Wy7Vg90H21EOLACvtfcHk0lhiqHTtduxSpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXjsFeqtg%253D%253D&md5=c1ed1e964cede076d62ccaf74fec2bd6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm00053a019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00053a019%26sid%3Dliteratum%253Aachs%26aulast%3DWade%26aufirst%3DR.%2BC.%26aulast%3DGoodford%26aufirst%3DP.%2BJ.%26atitle%3DFurther%2520development%2520of%2520hydrogen%2520bond%2520functions%2520for%2520use%2520in%2520determining%2520energetically%2520favorable%2520binding%2520sites%2520on%2520molecules%2520of%2520known%2520structure.%25202.%2520Ligand%2520probe%2520groups%2520with%2520the%2520ability%2520to%2520form%2520more%2520than%2520two%2520hydrogen%2520bonds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1993%26volume%3D36%26spage%3D148%26epage%3D156%26doi%3D10.1021%2Fjm00053a019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">SuperStar: a knowledge-based approach for identifying interaction sites in proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">1093</span><span class="NLM_x">–</span> <span class="NLM_lpage">1108</span><span class="refDoi"> DOI: 10.1006/jmbi.1999.2809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1006%2Fjmbi.1999.2809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10369784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK1MXjslKhtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=1999&pages=1093-1108&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=R.+Taylor&title=SuperStar%3A+a+knowledge-based+approach+for+identifying+interaction+sites+in+proteins&doi=10.1006%2Fjmbi.1999.2809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">SuperStar: A Knowledge-based Approach for Identifying Interaction Sites in Proteins</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1093-1108</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">An empirical method for identifying interaction sites in proteins is described and validated.  The method is based entirely on exptl. information about non-bonded interactions occurring in small-mol. crystal structures.  These data are used in the form of scatterplots that show the exptl. obsd. distribution of one functional group (the "contact group" or "probe") around another.  A template mol. (e.g. a protein binding site) is broken down into structure fragments and the scatterplots, showing the distribution of a chosen probe around these structure fragments, are superimposed on the corresponding parts of the template.  The scatterplots are then translated into a three-dimensional map that shows the propensity of the probe at different positions around the template mol.  The method is illustrated for L-arabinose-binding protein, complexed with L-arabinose and with D-fucose, and for dihydrofolate reductase complexed with methotrexate.  The method is validated on 122 x-ray structures of protein-ligand complexes.  For all the binding sites of these proteins, propensity maps are generated for four different probes: a charged NH+3 nitrogen, a carbonyl oxygen, a hydroxyl oxygen and a Me carbon atom.  Next, the maps are compared with the exptl. obsd. positions of ligand atoms of these types.  For 74% of these ligand atoms (84% of the solvent-inaccessible ones) the calcd. propensity of the matching probe at the exptl. positions is higher than expected by chance.  For 68% of the atoms (82% of the solvent-inaccessible ones) the propensity of the matching probe is higher than that of the other three probes.  These results indicate that the approach generally gives good predictions for protein-ligand interactions.  The potential applications of the propensity maps range from an aid in manual docking and structure-based drug design to their use in pharmacophore development.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9QVkFGAwFt7Vg90H21EOLACvtfcHk0lhiqHTtduxSpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjslKhtrs%253D&md5=72e09c54c9a631a8271fbe72f2f431b1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1999.2809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1999.2809%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DSuperStar%253A%2520a%2520knowledge-based%2520approach%2520for%2520identifying%2520interaction%2520sites%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1999%26volume%3D289%26spage%3D1093%26epage%3D1108%26doi%3D10.1006%2Fjmbi.1999.2809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillet, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">SuperStar: improved knowledge-based interaction fields for protein binding sites</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">307</span><span class="NLM_x">, </span> <span class="NLM_fpage">841</span><span class="NLM_x">–</span> <span class="NLM_lpage">859</span><span class="refDoi"> DOI: 10.1006/jmbi.2001.4452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1006%2Fjmbi.2001.4452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=11273705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitFymur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=307&publication_year=2001&pages=841-859&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=P.+Watsonauthor=V.+Gilletauthor=P.+Willett&title=SuperStar%3A+improved+knowledge-based+interaction+fields+for+protein+binding+sites&doi=10.1006%2Fjmbi.2001.4452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">SuperStar: Improved knowledge-based interaction fields for protein binding sites</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Watson, Paul; Gillet, Val; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">307</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">841-859</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">SuperStar is an empirical method for identifying interaction sites in proteins, based entirely on exptl. information about non-bonded interactions occurring in small-mol. crystal structures, taken from the IsoStar database.  We describe recent modifications and addns. to SuperStar, validating the results on a test set of 122 x-ray structures of protein-ligand complexes.  In this validation, propensity maps are generated for all the binding sites of these proteins, using four different probes: a charged NH+3 nitrogen atom, a carbonyl oxygen atom, a hydroxyl oxygen atom and a Me carbon atom.  Next, the maps are compared with the exptl. obsd. positions of ligand atoms of these types.  A peak-searching algorithm is introduced that highlights potential interaction hot spots.  For the three hydrogen-bonding probes - NH+3 nitrogen atom, carbonyl oxygen atom and hydroxyl oxygen atom - the av. distance from the ligand atom to the nearest SuperStar peak is 1.0-1.2 Å (0.8-1.0 Å for solvent-inaccessible ligand atoms).  For the Me carbon atom probe, this distance is about 1.5 Å, probably because interactions to Me groups are much less directional. The most important addn. to SuperStar is the enabling of propensity maps around metal centers - Ca2+, Mg2+ and Zn2+ - in protein binding sites.  The results are validated on a test set of 24 protein-ligand complexes that have a metal ion in their binding site.  Coordination geometries are derived automatically, using only the protein atoms that coordinate to the metal ion.  The correct coordination geometry is derived in approx. 75 % of the cases.  If the derived geometry is assumed during the SuperStar calcn., the av. distance from a ligand atom coordinating to the metal ion to the nearest peak in the propensity map for an oxygen probe is 0.87(7) Å.  If the correct coordination geometry is imposed, this distance reduces to 0.59(7) Å.  This indicates that the SuperStar predictions around metal-binding sites are at least as good as those around other protein groups. Using clustering techniques, a non-redundant set of probes is selected from the set of probes available in the IsoStar database.  The performance in SuperStar of all these probes is tested on the test set of protein-ligand complexes.  With the exception of the "ether oxygen" probe and the "any NH+" probe, all new probes perform as well as the four probes introduced first.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCZ27rmWmxMbVg90H21EOLACvtfcHk0lh7vVxhJElLKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitFymur0%253D&md5=026cdc7beaec6446307bbcc36a7e813f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2001.4452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2001.4452%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DWatson%26aufirst%3DP.%26aulast%3DGillet%26aufirst%3DV.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DSuperStar%253A%2520improved%2520knowledge-based%2520interaction%2520fields%2520for%2520protein%2520binding%2520sites%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2001%26volume%3D307%26spage%3D841%26epage%3D859%26doi%3D10.1006%2Fjmbi.2001.4452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Allen, F. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brice, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cartwright, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doubleday, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgs, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hummelink, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hummelink-Peters, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennard, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motherwell, W.</span><span> </span><span class="NLM_article-title">The Cambridge Crystallographic Data Centre: computer-based search, retrieval, analysis and display of information</span> <span class="citation_source-journal">Acta Crystallogr., Sect. B: Struct. Crystallogr. Cryst. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1979</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">2331</span><span class="NLM_x">–</span> <span class="NLM_lpage">2339</span><span class="refDoi"> DOI: 10.1107/S0567740879009249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1107%2FS0567740879009249" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1979&pages=2331-2339&author=F.+H.+Allenauthor=S.+Bellardauthor=M.+Briceauthor=B.+A.+Cartwrightauthor=A.+Doubledayauthor=H.+Higgsauthor=T.+Hummelinkauthor=B.+Hummelink-Petersauthor=O.+Kennardauthor=W.+Motherwell&title=The+Cambridge+Crystallographic+Data+Centre%3A+computer-based+search%2C+retrieval%2C+analysis+and+display+of+information&doi=10.1107%2FS0567740879009249"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1107%2FS0567740879009249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0567740879009249%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DF.%2BH.%26aulast%3DBellard%26aufirst%3DS.%26aulast%3DBrice%26aufirst%3DM.%26aulast%3DCartwright%26aufirst%3DB.%2BA.%26aulast%3DDoubleday%26aufirst%3DA.%26aulast%3DHiggs%26aufirst%3DH.%26aulast%3DHummelink%26aufirst%3DT.%26aulast%3DHummelink-Peters%26aufirst%3DB.%26aulast%3DKennard%26aufirst%3DO.%26aulast%3DMotherwell%26aufirst%3DW.%26atitle%3DThe%2520Cambridge%2520Crystallographic%2520Data%2520Centre%253A%2520computer-based%2520search%252C%2520retrieval%252C%2520analysis%2520and%2520display%2520of%2520information%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520B%253A%2520Struct.%2520Crystallogr.%2520Cryst.%2520Chem.%26date%3D1979%26volume%3D35%26spage%3D2331%26epage%3D2339%26doi%3D10.1107%2FS0567740879009249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lh7vVxhJElLKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Caflisch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miranker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Multiple copy simultaneous search and construction of ligands in binding sites: application to inhibitors of HIV-1 aspartic proteinase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">2142</span><span class="NLM_x">–</span> <span class="NLM_lpage">2167</span><span class="refDoi"> DOI: 10.1021/jm00067a013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00067a013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK3sXkvVGnsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1993&pages=2142-2167&author=A.+Caflischauthor=A.+Mirankerauthor=M.+Karplus&title=Multiple+copy+simultaneous+search+and+construction+of+ligands+in+binding+sites%3A+application+to+inhibitors+of+HIV-1+aspartic+proteinase&doi=10.1021%2Fjm00067a013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple copy simultaneous search and construction of ligands in binding sites: application to inhibitors of HIV-1 aspartic proteinase</span></div><div class="casAuthors">Caflisch, Amedeo; Miranker, Andrew; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2142-67</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Rational ligand design is a complex problem that can be divided into three parts: the search for optimal positions and orientations of functional groups in the binding site, the connection of such positions to form candidate ligands, and the estn. of their binding consts.  Approaches for addressing the first two parts of the problem are described in the present work.  They are applied to the construction of peptide ligands in the binding site of the human immunodeficiency virus 1 (HIV-1) proteinase and are compared with known inhibitor X-ray structures.  The primary objective is to test the method by comparison of the results with the MVT-101 complex structure for which coordinates are available; the results obtained with the liganded and unliganded proteinase structure are compared to examine the utility of the latter for binding studies.  A secondary objective is to show how to find new inhibitor candidates.  The multiple copy simultaneous search (MCSS) method is utilized to search for optimal positions and orientations of a set of functional groups.  For peptide ligands, functional groups corresponding to the protein main chain (N-methylacetamide) and to protein side chains (e.g., methanol, Et guanidinium) are used.  The resulting N-methylacetamide min. are connected to form hexapeptide main chains with a simple pseudoenergy function that permits a complete search of all possible ways of connecting the min.  Side chains are added to the main-chain candidates by application of the same pseudoenergy function to the appropriate functional group min.  A set of 15 hexapeptides with the sequence of MVT-101 are then minimized by a Monte Carlo scheme, which allows for escape from local min.  Comparison of the MCSS results with the structure of MVT-101 in the HIV-1 binding site showed that all of its functional group positions correspond (within 2.4 Å) to some (usually more than one) MCSS min.  There were also many other low-energy MCSS min. which do not appear in any known inhibitors, e.g., Me ammonium min. in the neighborhood of the catalytic aspartates.  Among the 15 lowest min. are seven hexapeptides with the same main-chain orientation as the one found by X-ray crystallog. for the inhibitor MVT-101 in the binding site and eight with the main chain oriented in the opposite direction; the latter tend to be more stable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohNHWxLu8SpLVg90H21EOLACvtfcHk0ljzKSZa9kCmxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkvVGnsLc%253D&md5=9d14f5a1f3921f090cb4391d89f457db</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm00067a013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00067a013%26sid%3Dliteratum%253Aachs%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DMiranker%26aufirst%3DA.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DMultiple%2520copy%2520simultaneous%2520search%2520and%2520construction%2520of%2520ligands%2520in%2520binding%2520sites%253A%2520application%2520to%2520inhibitors%2520of%2520HIV-1%2520aspartic%2520proteinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1993%26volume%3D36%26spage%3D2142%26epage%3D2167%26doi%3D10.1021%2Fjm00067a013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Miranker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karplus, M.</span><span> </span><span class="NLM_article-title">Functionality maps of binding sites: a multiple copy simultaneous search method</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">29</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1002/prot.340110104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1002%2Fprot.340110104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1961699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK38XlsFSjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1991&pages=29-34&author=A.+Mirankerauthor=M.+Karplus&title=Functionality+maps+of+binding+sites%3A+a+multiple+copy+simultaneous+search+method&doi=10.1002%2Fprot.340110104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Functionality maps of binding sites:  a multiple copy simultaneous search method</span></div><div class="casAuthors">Miranker, Andrew; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Proteins:  Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-34</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    </div><div class="casAbstract">A new method is proposed for detg. energetically favorable positions and orientations for functional groups on the surface of proteins with known 3-dimensional structure.  Some 1000-5000 copies of a functional group are randomly placed in the site and subjected to simultaneous energy minimization and/or quenched mol. dynamics.  The resulting functionality maps of a protein receptor site, which can take account of its flexibility, can be used for the anal. of protein ligand interactions and rational drug design.  Application of the method to the sialic acid binding site of the influenza coat protein, hemagglutinin, yields functional group min. that correspond with those of the ligand in a cocrystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5im8BqJfF37Vg90H21EOLACvtfcHk0ljzKSZa9kCmxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlsFSjtQ%253D%253D&md5=34e0a0c36899bfeabcbdfd4790f97a1e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fprot.340110104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.340110104%26sid%3Dliteratum%253Aachs%26aulast%3DMiranker%26aufirst%3DA.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DFunctionality%2520maps%2520of%2520binding%2520sites%253A%2520a%2520multiple%2520copy%2520simultaneous%2520search%2520method%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1991%26volume%3D11%26spage%3D29%26epage%3D34%26doi%3D10.1002%2Fprot.340110104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Dennis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kortvelyesi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">Computational mapping identifies the binding sites of organic solvents on proteins</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">4290</span><span class="NLM_x">–</span> <span class="NLM_lpage">4295</span><span class="refDoi"> DOI: 10.1073/pnas.062398499</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1073%2Fpnas.062398499" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=11904374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD38XivFShtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=4290-4295&author=S.+Dennisauthor=T.+Kortvelyesiauthor=S.+Vajda&title=Computational+mapping+identifies+the+binding+sites+of+organic+solvents+on+proteins&doi=10.1073%2Fpnas.062398499"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Computational mapping identifies the binding sites of organic solvents on proteins</span></div><div class="casAuthors">Dennis, Sheldon; Kortvelyesi, Tamas; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4290-4295</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Computational mapping places mol. probes-small mols. or functional groups-on a protein surface to identify the most favorable binding positions.  Although x-ray crystallog. and NMR show that org. solvents bind to a limited no. of sites on a protein, current mapping methods result in hundreds of energy min. and do not reveal why some sites bind mols. with different sizes and polarities.  We describe a mapping algorithm that explains the origin of this phenomenon.  The algorithm has been applied to hen egg-white lysozyme and to thermolysin, interacting with eight and four different ligands, resp.  In both cases the search finds the consensus site to which all mols. bind, whereas other positions that bind only certain ligands are not necessarily found.  The consensus sites are pockets of the active site, lined with partially exposed hydrophobic residues and with a no. of polar residues toward the edge.  These sites can accommodate each ligand in a no. of rotational states, some with a hydrogen bond to one of the nearby donor/acceptor groups.  Specific substrates and/or inhibitors of hen egg-white lysozyme and thermolysin interact with the same side chains identified by the mapping, but form several hydrogen bonds and bind in unique orientations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg4sEo2VxwF7Vg90H21EOLACvtfcHk0ljzKSZa9kCmxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XivFShtrw%253D&md5=90d71ffcc8860e87aefec3c5809b1127</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.062398499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.062398499%26sid%3Dliteratum%253Aachs%26aulast%3DDennis%26aufirst%3DS.%26aulast%3DKortvelyesi%26aufirst%3DT.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DComputational%2520mapping%2520identifies%2520the%2520binding%2520sites%2520of%2520organic%2520solvents%2520on%2520proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D4290%26epage%3D4295%26doi%3D10.1073%2Fpnas.062398499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Kozakov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grove, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohnuud, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mottarella, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span> </span><span class="NLM_article-title">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">755</span><span class="refDoi"> DOI: 10.1038/nprot.2015.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1038%2Fnprot.2015.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=25855957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=733-755&author=D.+Kozakovauthor=L.+E.+Groveauthor=D.+R.+Hallauthor=T.+Bohnuudauthor=S.+E.+Mottarellaauthor=L.+Luoauthor=B.+Xiaauthor=D.+Beglovauthor=S.+Vajda&title=The+FTMap+family+of+web+servers+for+determining+and+characterizing+ligand-binding+hot+spots+of+proteins&doi=10.1038%2Fnprot.2015.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The FTMap family of web servers for determining and characterizing ligand-binding hot spots of proteins</span></div><div class="casAuthors">Kozakov, Dima; Grove, Laurie E.; Hall, David R.; Bohnuud, Tanggis; Mottarella, Scott E.; Luo, Lingqi; Xia, Bing; Beglov, Dmitri; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-755</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">FTMap is a computational mapping server that identifies binding hot spots of macromols., i.e., regions of the surface with major contributions to the ligand-binding free energy.  To use FTMap, users submit a protein, DNA or RNA structure in PDB (Protein Data Bank) format.  FTMap samples billions of positions of small org. mols. used as probes, and it scores the probe poses using a detailed energy expression.  Regions that bind clusters of multiple probe types identify the binding hot spots in good agreement with exptl. data.  FTMap serves as the basis for other servers, namely FTSite, which was used to predict ligand-binding sites, FTFlex, which was used to account for side chain flexibility, FTMap/param, used to parameterize addnl. probes and FTDyn, for mapping ensembles of protein structures.  Applications include detg. the druggability of proteins, identifying ligand moieties that are most important for binding, finding the most bound-like conformation in ensembles of unliganded protein structures and providing input for fragment-based drug design.  FTMap is more accurate than classical mapping methods such as GRID and MCSS, and it is much faster than the more-recent approaches to protein mapping based on mixed mol. dynamics.  By using 16 probe mols., the FTMap server finds the hot spots of an av.-size protein in <1 h.  As FTFlex performs mapping for all low-energy conformers of side chains in the binding site, its completion time is proportionately longer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnH27-sNtGLVg90H21EOLACvtfcHk0lhFq6UO2TvneA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Smt74%253D&md5=dbc729e9ffb47989f0df8baebaf032fa</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2015.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2015.043%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBohnuud%26aufirst%3DT.%26aulast%3DMottarella%26aufirst%3DS.%2BE.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DThe%2520FTMap%2520family%2520of%2520web%2520servers%2520for%2520determining%2520and%2520characterizing%2520ligand-binding%2520hot%2520spots%2520of%2520proteins%26jtitle%3DNat.%2520Protoc.%26date%3D2015%26volume%3D10%26spage%3D733%26epage%3D755%26doi%3D10.1038%2Fnprot.2015.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Grove, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beglov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozakov, D.</span><span> </span><span class="NLM_article-title">FTFlex: accounting for binding site flexibility to improve fragment-based identification of druggable hot spots</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1218</span><span class="NLM_x">–</span> <span class="NLM_lpage">1219</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btt102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1093%2Fbioinformatics%2Fbtt102" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=1218-1219&author=L.+E.+Groveauthor=D.+R.+Hallauthor=D.+Beglovauthor=S.+Vajdaauthor=D.+Kozakov&title=FTFlex%3A+accounting+for+binding+site+flexibility+to+improve+fragment-based+identification+of+druggable+hot+spots&doi=10.1093%2Fbioinformatics%2Fbtt102"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt102%26sid%3Dliteratum%253Aachs%26aulast%3DGrove%26aufirst%3DL.%2BE.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DKozakov%26aufirst%3DD.%26atitle%3DFTFlex%253A%2520accounting%2520for%2520binding%2520site%2520flexibility%2520to%2520improve%2520fragment-based%2520identification%2520of%2520druggable%2520hot%2520spots%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D1218%26epage%3D1219%26doi%3D10.1093%2Fbioinformatics%2Fbtt102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Radoux, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsson, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Identifying interactions that determine fragment binding at protein hotspots</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4314</span><span class="NLM_x">–</span> <span class="NLM_lpage">4325</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01980</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01980" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlOms7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4314-4325&author=C.+J.+Radouxauthor=T.+S.+Olssonauthor=W.+R.+Pittauthor=C.+R.+Groomauthor=T.+L.+Blundell&title=Identifying+interactions+that+determine+fragment+binding+at+protein+hotspots&doi=10.1021%2Facs.jmedchem.5b01980"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Interactions that Determine Fragment Binding at Protein Hotspots</span></div><div class="casAuthors">Radoux, Chris J.; Olsson, Tjelvar S. G.; Pitt, Will R.; Groom, Colin R.; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4314-4325</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Locating a ligand-binding site is an important first step in structure-guided drug discovery, but current methods do little to suggest which interactions within a pocket are the most important for binding.  Here the authors illustrate a method that samples at. hotspots with simple mol. probes to produce fragment hotspot maps.  These maps specifically highlight fragment-binding sites and their corresponding pharmacophores.  For ligand-bound structures, they provide an intuitive visual guide within the binding site, directing medicinal chemists where to grow the mol. and alerting them to suboptimal interactions within the original hit.  The fragment hotspot map calcn. is validated using exptl. binding positions of 21 fragments and subsequent lead mols.  The ligands are found in high scoring areas of the fragment hotspot maps, with fragment atoms having a median percentage rank of 97%.  Protein kinase B and pantothenate synthetase are examd. in detail.  In each case, the fragment hotspot maps are able to rationalize a Free-Wilson anal. of SAR data from a fragment-based drug design project.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8BzZp1bkHarVg90H21EOLACvtfcHk0lhFq6UO2TvneA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlOms7Y%253D&md5=fa25eabfe291f9519f5701bcbc6c52c3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01980%26sid%3Dliteratum%253Aachs%26aulast%3DRadoux%26aufirst%3DC.%2BJ.%26aulast%3DOlsson%26aufirst%3DT.%2BS.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DIdentifying%2520interactions%2520that%2520determine%2520fragment%2520binding%2520at%2520protein%2520hotspots%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4314%26epage%3D4325%26doi%3D10.1021%2Facs.jmedchem.5b01980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Seco, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luque, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span> </span><span class="NLM_article-title">Binding site detection and druggability index from first principles</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2363</span><span class="NLM_x">–</span> <span class="NLM_lpage">2371</span><span class="refDoi"> DOI: 10.1021/jm801385d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801385d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2363-2371&author=J.+Secoauthor=F.+J.+Luqueauthor=X.+Barril&title=Binding+site+detection+and+druggability+index+from+first+principles&doi=10.1021%2Fjm801385d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Site Detection and Druggability Index from First Principles</span></div><div class="casAuthors">Seco, Jesus; Luque, F. Javier; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2363-2371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In drug discovery, it is essential to identify binding sites on protein surfaces that drug-like mols. could exploit to exert a biol. effect.  Both X-ray crystallog. and NMR expts. have demonstrated that org. solvents bind precisely at these locations.  We show that this effect is reproduced using mol. dynamics with a binary solvent.  Furthermore, anal. of the simulations give direct access to interaction free energies between the protein and small org. mols., which can be used to detect binding sites and to predict the maximal affinity that a drug-like mol. could attain for them.  On a set of pharmacol. relevant proteins, we obtain good predictions for druggable sites as well as for protein-protein and low affinity binding sites.  This is the first druggability index not based on surface descriptors and, being independent of a training set, is particularly indicated to study unconventional targets such as protein-protein interactions or allosteric binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTAgMI9Qer6rVg90H21EOLACvtfcHk0lgTCvETc_ppEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtlOhu70%253D&md5=b766cbadeba11b23a1de44b27aa794b4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm801385d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801385d%26sid%3Dliteratum%253Aachs%26aulast%3DSeco%26aufirst%3DJ.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DBinding%2520site%2520detection%2520and%2520druggability%2520index%2520from%2520first%2520principles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2363%26epage%3D2371%26doi%3D10.1021%2Fjm801385d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Clark, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarnieri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shkurko, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, J.</span><span> </span><span class="NLM_article-title">Grand canonical Monte Carlo simulation of ligand-protein binding</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1021/ci050268f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci050268f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Gju7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=231-242&author=M.+Clarkauthor=F.+Guarnieriauthor=I.+Shkurkoauthor=J.+Wiseman&title=Grand+canonical+Monte+Carlo+simulation+of+ligand-protein+binding&doi=10.1021%2Fci050268f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Grand Canonical Monte Carlo Simulation of Ligand-Protein Binding</span></div><div class="casAuthors">Clark, Matthew; Guarnieri, Frank; Shkurko, Igor; Wiseman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">231-242</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new application of the grand canonical thermodn. ensemble to compute ligand-protein binding is described.  The described method is sufficiently rapid that it is practical to compute ligand-protein binding free energies for a large no. of poses over the entire protein surface, thus identifying multiple putative ligand binding sites.  In addn., the method computes binding free energies for a large no. of poses.  The method is demonstrated by the simulation of two protein-ligand systems, thermolysin and T4 lysozyme, for which there is extensive thermodn. and crystallog. data for the binding of small, rigid ligands.  These low-mol.-wt. ligands correspond to the mol. fragments used in computational fragment-based drug design.  The simulations correctly identified the exptl. binding poses and rank ordered the affinities of ligands in each of these systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD_W87xCkI47Vg90H21EOLACvtfcHk0lgTCvETc_ppEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Gju7zJ&md5=6ff1670d21b2650f0eb0a113ab38a1c5</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fci050268f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050268f%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%26aulast%3DGuarnieri%26aufirst%3DF.%26aulast%3DShkurko%26aufirst%3DI.%26aulast%3DWiseman%26aufirst%3DJ.%26atitle%3DGrand%2520canonical%2520Monte%2520Carlo%2520simulation%2520of%2520ligand-protein%2520binding%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D231%26epage%3D242%26doi%3D10.1021%2Fci050268f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Clark, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meshkat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, J. S.</span><span> </span><span class="NLM_article-title">Grand canonical free-energy calculations of protein-ligand binding</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">934</span><span class="NLM_x">–</span> <span class="NLM_lpage">943</span><span class="refDoi"> DOI: 10.1021/ci8004397</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci8004397" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsFKms7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=934-943&author=M.+Clarkauthor=S.+Meshkatauthor=J.+S.+Wiseman&title=Grand+canonical+free-energy+calculations+of+protein-ligand+binding&doi=10.1021%2Fci8004397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Grand Canonical Free-Energy Calculations of Protein-Ligand Binding</span></div><div class="casAuthors">Clark, Matthew; Meshkat, Sia; Wiseman, Jeffrey S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">934-943</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The principles behind the computation of protein-ligand binding free energies by Monte Carlo simulation in the Grand Canonical Ensemble are described in detail, and two variations of the calcn. are presented.  The computation can be performed by bathing a protein binding site either with ligand images that interact with each other or with ligand images that pass through each other.  The second method is theor. more rigorous, but we show that both methods lead to the same result, and there are distinct numeric advantages to using ligand images that interact with each other.  The Grand Canonical simulation provides gas-phase binding free energies that can be converted to aq. energies by generalized Born-surface area (GB/SA) solvation calcns. to provide values that agree with expt. within ±1.5 kcal/mol.  However, the accuracy of these simple solvation calcns. is a major limiting factor in the accuracy of the overall binding free-energy computation.  The Grand Canonical simulation has several characteristics beneficial to free-energy calcns.  One is that the no. of parameters that must be set for the simulation is small and can be detd. objectively, making the outcome more deterministic, with respect to choice of input conditions, as compared to perturbation methods.  Second, the simulation is free from assumptions about the starting pose or nature of the binding site.  A final benefit is that binding free energies are a direct outcome of the simulation, and little processing is required to det. them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmHrvGMlNIQrVg90H21EOLACvtfcHk0lgTCvETc_ppEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsFKms7s%253D&md5=ab2ced7fbc3e27709e3d6e95ed5a8461</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fci8004397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8004397%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%26aulast%3DMeshkat%26aufirst%3DS.%26aulast%3DWiseman%26aufirst%3DJ.%2BS.%26atitle%3DGrand%2520canonical%2520free-energy%2520calculations%2520of%2520protein-ligand%2520binding%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D934%26epage%3D943%26doi%3D10.1021%2Fci8004397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Ghanakota, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, H. A.</span><span> </span><span class="NLM_article-title">Driving structure-based drug discovery through cosolvent molecular dynamics</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">10383</span><span class="NLM_x">–</span> <span class="NLM_lpage">10399</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00399</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00399" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1yrsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10383-10399&author=P.+Ghanakotaauthor=H.+A.+Carlson&title=Driving+structure-based+drug+discovery+through+cosolvent+molecular+dynamics&doi=10.1021%2Facs.jmedchem.6b00399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Driving Structure-Based Drug Discovery through Cosolvent Molecular Dynamics</span></div><div class="casAuthors">Ghanakota, Phani; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10383-10399</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Identifying binding hotspots on protein surfaces is of prime interest in structure-based drug discovery, either to assess the tractability of pursuing a protein target or to drive improved potency of lead compds.  Computational approaches to detect such regions have traditionally relied on energy minimization of probe mols. onto static protein conformations in the absence of the natural water environment.  Advances in high performance computing now allow the authors to assess hotspots using mol. dynamics (MD) simulations.  MD simulations integrate protein flexibility and the complicated role of water, thereby providing a more realistic assessment of the complex kinetics and thermodn. at play.  In this review, the authors describe the evolution of various cosolvent-based MD techniques and highlight a myriad of potential applications for such technologies in computational drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWZlTX7u7cwLVg90H21EOLACvtfcHk0lhkw14o_3CQiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1yrsbzE&md5=30727d3cbb19bdfb99cdcfa544e5d82d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00399%26sid%3Dliteratum%253Aachs%26aulast%3DGhanakota%26aufirst%3DP.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DDriving%2520structure-based%2520drug%2520discovery%2520through%2520cosolvent%2520molecular%2520dynamics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10383%26epage%3D10399%26doi%3D10.1021%2Facs.jmedchem.6b00399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Tan, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spring, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, C. S.</span><span> </span><span class="NLM_article-title">The application of ligand-mapping molecular dynamics simulations to the rational design of peptidic modulators of protein-protein interactions</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">3199</span><span class="NLM_x">–</span> <span class="NLM_lpage">3210</span><span class="refDoi"> DOI: 10.1021/ct5010577</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct5010577" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps1arurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=3199-3210&author=Y.+S.+Tanauthor=D.+R.+Springauthor=C.+Abellauthor=C.+S.+Verma&title=The+application+of+ligand-mapping+molecular+dynamics+simulations+to+the+rational+design+of+peptidic+modulators+of+protein-protein+interactions&doi=10.1021%2Fct5010577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The Application of Ligand-Mapping Molecular Dynamics Simulations to the Rational Design of Peptidic Modulators of Protein-Protein Interactions</span></div><div class="casAuthors">Tan, Yaw Sing; Spring, David R.; Abell, Chris; Verma, Chandra S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3199-3210</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A computational ligand-mapping approach to detect protein surface pockets that interact with hydrophobic moieties is presented.  In this method, we incorporated benzene mols. into explicit solvent mol. dynamics simulations of various protein targets.  The benzene mols. successfully identified the binding locations of hydrophobic hot-spot residues and all-hydrocarbon cross-links from known peptidic ligands.  They also unveiled cryptic binding sites that are occluded by side chains and the protein backbone.  Our results demonstrate that ligand-mapping mol. dynamics simulations hold immense promise to guide the rational design of peptidic modulators of protein-protein interactions, including that of stapled peptides, which show promise as an exciting new class of cell-penetrating therapeutic mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3jTNpkoiL7rVg90H21EOLACvtfcHk0lhkw14o_3CQiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps1arurk%253D&md5=f28bcf448035402f8976142b89d093b3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fct5010577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct5010577%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DY.%2BS.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DVerma%26aufirst%3DC.%2BS.%26atitle%3DThe%2520application%2520of%2520ligand-mapping%2520molecular%2520dynamics%2520simulations%2520to%2520the%2520rational%2520design%2520of%2520peptidic%2520modulators%2520of%2520protein-protein%2520interactions%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D3199%26epage%3D3210%26doi%3D10.1021%2Fct5010577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Vallée, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilorge, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dupuy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parent, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fassy, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamberton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrebola, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guerif, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rohaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Certal, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruxer, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delorme, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jouanen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grépin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Combeau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goulaouic, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dereu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikol, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mailliet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minoux, H.</span><span> </span><span class="NLM_article-title">Tricyclic series of heat shock protein 90 (Hsp90) inhibitors part i: Discovery of tricyclic imidazo[4,5-c]pyridines as potent inhibitors of the Hsp90 molecular chaperone</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">7206</span><span class="NLM_x">–</span> <span class="NLM_lpage">7219</span><span class="refDoi"> DOI: 10.1021/jm200784m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200784m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7206-7219&author=F.+Vall%C3%A9eauthor=C.+Carrezauthor=F.+Pilorgeauthor=A.+Dupuyauthor=A.+Parentauthor=L.+Bertinauthor=F.+Thompsonauthor=P.+Ferrariauthor=F.+Fassyauthor=A.+Lambertonauthor=A.+Thomasauthor=R.+Arrebolaauthor=S.+Guerifauthor=A.+Rohautauthor=V.+Certalauthor=J.-M.+Ruxerauthor=C.+Delormeauthor=A.+Jouanenauthor=J.+Dumasauthor=C.+Gr%C3%A9pinauthor=C.+Combeauauthor=H.+Goulaouicauthor=N.+Dereuauthor=V.+Mikolauthor=P.+Maillietauthor=H.+Minoux&title=Tricyclic+series+of+heat+shock+protein+90+%28Hsp90%29+inhibitors+part+i%3A+Discovery+of+tricyclic+imidazo%5B4%2C5-c%5Dpyridines+as+potent+inhibitors+of+the+Hsp90+molecular+chaperone&doi=10.1021%2Fjm200784m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm200784m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200784m%26sid%3Dliteratum%253Aachs%26aulast%3DVall%25C3%25A9e%26aufirst%3DF.%26aulast%3DCarrez%26aufirst%3DC.%26aulast%3DPilorge%26aufirst%3DF.%26aulast%3DDupuy%26aufirst%3DA.%26aulast%3DParent%26aufirst%3DA.%26aulast%3DBertin%26aufirst%3DL.%26aulast%3DThompson%26aufirst%3DF.%26aulast%3DFerrari%26aufirst%3DP.%26aulast%3DFassy%26aufirst%3DF.%26aulast%3DLamberton%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DA.%26aulast%3DArrebola%26aufirst%3DR.%26aulast%3DGuerif%26aufirst%3DS.%26aulast%3DRohaut%26aufirst%3DA.%26aulast%3DCertal%26aufirst%3DV.%26aulast%3DRuxer%26aufirst%3DJ.-M.%26aulast%3DDelorme%26aufirst%3DC.%26aulast%3DJouanen%26aufirst%3DA.%26aulast%3DDumas%26aufirst%3DJ.%26aulast%3DGr%25C3%25A9pin%26aufirst%3DC.%26aulast%3DCombeau%26aufirst%3DC.%26aulast%3DGoulaouic%26aufirst%3DH.%26aulast%3DDereu%26aufirst%3DN.%26aulast%3DMikol%26aufirst%3DV.%26aulast%3DMailliet%26aufirst%3DP.%26aulast%3DMinoux%26aufirst%3DH.%26atitle%3DTricyclic%2520series%2520of%2520heat%2520shock%2520protein%252090%2520%2528Hsp90%2529%2520inhibitors%2520part%2520i%253A%2520Discovery%2520of%2520tricyclic%2520imidazo%255B4%252C5-c%255Dpyridines%2520as%2520potent%2520inhibitors%2520of%2520the%2520Hsp90%2520molecular%2520chaperone%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7206%26epage%3D7219%26doi%3D10.1021%2Fjm200784m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Kung, P.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Funk, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catherine Johnson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, M.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodgers, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayman, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smeal, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maegley, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehring, M. R.</span><span> </span><span class="NLM_article-title">Dihydroxylphenyl amides as inhibitors of the Hsp90 molecular chaperone</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">6273</span><span class="NLM_x">–</span> <span class="NLM_lpage">6278</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.09.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2Fj.bmcl.2008.09.081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=18929486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOhtrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=6273-6278&author=P.-P.+Kungauthor=L.+Funkauthor=J.+Mengauthor=M.+Collinsauthor=J.+Z.+Zhouauthor=M.+Catherine+Johnsonauthor=A.+Ekkerauthor=J.+Wangauthor=P.+Mehtaauthor=M.-J.+Yinauthor=C.+Rodgersauthor=J.+F.+Daviesauthor=E.+Baymanauthor=T.+Smealauthor=K.+A.+Maegleyauthor=M.+R.+Gehring&title=Dihydroxylphenyl+amides+as+inhibitors+of+the+Hsp90+molecular+chaperone&doi=10.1016%2Fj.bmcl.2008.09.081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Dihydroxylphenyl amides as inhibitors of the Hsp90 molecular chaperone</span></div><div class="casAuthors">Kung, Pei-Pei; Funk, Lee; Meng, Jerry; Collins, Michael; Zhou, Joe Zhongxiang; Catherine Johnson, M.; Ekker, Anne; Wang, Jeff; Mehta, Pramod; Yin, Min-Jean; Rodgers, Caroline; Davies, Jay F., II; Bayman, Eileen; Smeal, Tod; Maegley, Karen A.; Gehring, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6273-6278</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Information from X-ray crystal structures were used to optimize the potency of a HTS hit in a Hsp90 competitive binding assay.  A class of novel and potent small mol. Hsp90 inhibitors were thereby identified.  Enantio-pure compds. I [n = 1, 2] were potent in PGA-based competitive binding assay and inhibited proliferation of various human cancer cell lines in vitro, with IC50 values averaging 20 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraBHD3PPbhDbVg90H21EOLACvtfcHk0ljn5V-d3IH6EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOhtrzE&md5=36eb69d7ffb65443bc5328f74806fa0b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.09.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.09.081%26sid%3Dliteratum%253Aachs%26aulast%3DKung%26aufirst%3DP.-P.%26aulast%3DFunk%26aufirst%3DL.%26aulast%3DMeng%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DJ.%2BZ.%26aulast%3DCatherine%2BJohnson%26aufirst%3DM.%26aulast%3DEkker%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DMehta%26aufirst%3DP.%26aulast%3DYin%26aufirst%3DM.-J.%26aulast%3DRodgers%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DJ.%2BF.%26aulast%3DBayman%26aufirst%3DE.%26aulast%3DSmeal%26aufirst%3DT.%26aulast%3DMaegley%26aufirst%3DK.%2BA.%26aulast%3DGehring%26aufirst%3DM.%2BR.%26atitle%3DDihydroxylphenyl%2520amides%2520as%2520inhibitors%2520of%2520the%2520Hsp90%2520molecular%2520chaperone%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D6273%26epage%3D6278%26doi%3D10.1016%2Fj.bmcl.2008.09.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Kung, P.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Digits, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skaptason, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neul, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zientek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elleraas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehta, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, M.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickey, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gajiwala, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodgers, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehring, M. R.</span><span> </span><span class="NLM_article-title">Dihydroxyphenylisoindoline amides as orally bioavailable inhibitors of the heat shock protein 90 (hsp90) molecular chaperone</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">499</span><span class="NLM_x">–</span> <span class="NLM_lpage">503</span><span class="refDoi"> DOI: 10.1021/jm901209q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901209q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=499-503&author=P.-P.+Kungauthor=B.+Huangauthor=G.+Zhangauthor=J.+Z.+Zhouauthor=J.+Wangauthor=J.+A.+Digitsauthor=J.+Skaptasonauthor=S.+Yamazakiauthor=D.+Neulauthor=M.+Zientekauthor=J.+Elleraasauthor=P.+Mehtaauthor=M.-J.+Yinauthor=M.+J.+Hickeyauthor=K.+S.+Gajiwalaauthor=C.+Rodgersauthor=J.+F.+Daviesauthor=M.+R.+Gehring&title=Dihydroxyphenylisoindoline+amides+as+orally+bioavailable+inhibitors+of+the+heat+shock+protein+90+%28hsp90%29+molecular+chaperone&doi=10.1021%2Fjm901209q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm901209q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901209q%26sid%3Dliteratum%253Aachs%26aulast%3DKung%26aufirst%3DP.-P.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DJ.%2BZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDigits%26aufirst%3DJ.%2BA.%26aulast%3DSkaptason%26aufirst%3DJ.%26aulast%3DYamazaki%26aufirst%3DS.%26aulast%3DNeul%26aufirst%3DD.%26aulast%3DZientek%26aufirst%3DM.%26aulast%3DElleraas%26aufirst%3DJ.%26aulast%3DMehta%26aufirst%3DP.%26aulast%3DYin%26aufirst%3DM.-J.%26aulast%3DHickey%26aufirst%3DM.%2BJ.%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DRodgers%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DJ.%2BF.%26aulast%3DGehring%26aufirst%3DM.%2BR.%26atitle%3DDihydroxyphenylisoindoline%2520amides%2520as%2520orally%2520bioavailable%2520inhibitors%2520of%2520the%2520heat%2520shock%2520protein%252090%2520%2528hsp90%2529%2520molecular%2520chaperone%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D499%26epage%3D503%26doi%3D10.1021%2Fjm901209q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Bruncko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tahir, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huth, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Judge, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madar, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, C.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elmore, S. W.</span><span> </span><span class="NLM_article-title">N-Aryl-benzimidazolones as novel small molecule HSP90 inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">7503</span><span class="NLM_x">–</span> <span class="NLM_lpage">7506</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2Fj.bmcl.2010.10.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=7503-7506&author=M.+Brunckoauthor=S.+K.+Tahirauthor=X.+Songauthor=J.+Chenauthor=H.+Dingauthor=J.+R.+Huthauthor=S.+Jinauthor=R.+A.+Judgeauthor=D.+J.+Madarauthor=C.+H.+Parkauthor=C.-M.+Parkauthor=A.+M.+Petrosauthor=C.+Tseauthor=S.+H.+Rosenbergauthor=S.+W.+Elmore&title=N-Aryl-benzimidazolones+as+novel+small+molecule+HSP90+inhibitors&doi=10.1016%2Fj.bmcl.2010.10.010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DBruncko%26aufirst%3DM.%26aulast%3DTahir%26aufirst%3DS.%2BK.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DHuth%26aufirst%3DJ.%2BR.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DJudge%26aufirst%3DR.%2BA.%26aulast%3DMadar%26aufirst%3DD.%2BJ.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DPark%26aufirst%3DC.-M.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26atitle%3DN-Aryl-benzimidazolones%2520as%2520novel%2520small%2520molecule%2520HSP90%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D7503%26epage%3D7506%26doi%3D10.1016%2Fj.bmcl.2010.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Casale, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amboldi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brasca, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caronni, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colombo, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felder, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fogliatto, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galvani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isacchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polucci, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riceputi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sola, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Visco, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuccotto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casuscelli, F.</span><span> </span><span class="NLM_article-title">Fragment-based hit discovery and structure-based optimization of aminotriazoloquinazolines as novel Hsp90 inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4135</span><span class="NLM_x">–</span> <span class="NLM_lpage">4150</span><span class="refDoi"> DOI: 10.1016/j.bmc.2014.05.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2Fj.bmc.2014.05.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=24980703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVKhtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=4135-4150&author=E.+Casaleauthor=N.+Amboldiauthor=M.+G.+Brascaauthor=D.+Caronniauthor=N.+Colomboauthor=C.+Dalvitauthor=E.+R.+Felderauthor=G.+Fogliattoauthor=A.+Galvaniauthor=A.+Isacchiauthor=P.+Polucciauthor=L.+Riceputiauthor=F.+Solaauthor=C.+Viscoauthor=F.+Zuccottoauthor=F.+Casuscelli&title=Fragment-based+hit+discovery+and+structure-based+optimization+of+aminotriazoloquinazolines+as+novel+Hsp90+inhibitors&doi=10.1016%2Fj.bmc.2014.05.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based hit discovery and structure-based optimization of aminotriazoloquinazolines as novel Hsp90 inhibitors</span></div><div class="casAuthors">Casale, Elena; Amboldi, Nadia; Brasca, Maria Gabriella; Caronni, Dannica; Colombo, Nicoletta; Dalvit, Claudio; Felder, Eduard R.; Fogliatto, Gianpaolo; Galvani, Arturo; Isacchi, Antonella; Polucci, Paolo; Riceputi, Laura; Sola, Francesco; Visco, Carlo; Zuccotto, Fabio; Casuscelli, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4135-4150</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In the last decade the heat shock protein 90 (Hsp90) has emerged as a major therapeutic target and many efforts have been dedicated to the discovery of Hsp90 inhibitors as new potent anticancer agents.  Here the authors report the identification of a novel class of Hsp90 inhibitors by means of a biophys. FAXS-NMR based screening of a library of fragments.  The use of x-ray structure information combined with modeling studies enabled the fragment evolution of the initial triazoloquinazoline hit to a class of compds. with nanomolar potency and drug-like properties suited for further lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8pZK-hye807Vg90H21EOLACvtfcHk0lj-Ii7AOti19g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVKhtbfK&md5=7d3d910e96249f6d5550cecc2363af9c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.05.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.05.056%26sid%3Dliteratum%253Aachs%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DAmboldi%26aufirst%3DN.%26aulast%3DBrasca%26aufirst%3DM.%2BG.%26aulast%3DCaronni%26aufirst%3DD.%26aulast%3DColombo%26aufirst%3DN.%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DFelder%26aufirst%3DE.%2BR.%26aulast%3DFogliatto%26aufirst%3DG.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DIsacchi%26aufirst%3DA.%26aulast%3DPolucci%26aufirst%3DP.%26aulast%3DRiceputi%26aufirst%3DL.%26aulast%3DSola%26aufirst%3DF.%26aulast%3DVisco%26aufirst%3DC.%26aulast%3DZuccotto%26aufirst%3DF.%26aulast%3DCasuscelli%26aufirst%3DF.%26atitle%3DFragment-based%2520hit%2520discovery%2520and%2520structure-based%2520optimization%2520of%2520aminotriazoloquinazolines%2520as%2520novel%2520Hsp90%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D4135%26epage%3D4150%26doi%3D10.1016%2Fj.bmc.2014.05.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludlow, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giangreco, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rathi, P. C.</span><span> </span><span class="NLM_article-title">Protein-Ligand-Informatics force field (PLIff): towards a fully knowledge driven “force field” for biomolecular interactions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">6891</span><span class="NLM_x">–</span> <span class="NLM_lpage">6902</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00716</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00716" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6891-6902&author=M.+L.+Verdonkauthor=R.+F.+Ludlowauthor=I.+Giangrecoauthor=P.+C.+Rathi&title=Protein-Ligand-Informatics+force+field+%28PLIff%29%3A+towards+a+fully+knowledge+driven+%E2%80%9Cforce+field%E2%80%9D+for+biomolecular+interactions&doi=10.1021%2Facs.jmedchem.6b00716"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00716%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DGiangreco%26aufirst%3DI.%26aulast%3DRathi%26aufirst%3DP.%2BC.%26atitle%3DProtein-Ligand-Informatics%2520force%2520field%2520%2528PLIff%2529%253A%2520towards%2520a%2520fully%2520knowledge%2520driven%2520%25E2%2580%259Cforce%2520field%25E2%2580%259D%2520for%2520biomolecular%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6891%26epage%3D6902%26doi%3D10.1021%2Facs.jmedchem.6b00716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Jones, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">267</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">748</span><span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0limCQBrqpIm5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Bateman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durbin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollich, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffiths-Jones, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khanna, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moxon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonnhammer, E. L.</span><span> </span><span class="NLM_article-title">The Pfam protein families database</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">D138</span><span class="NLM_x">–</span> <span class="NLM_lpage">D141</span><span class="refDoi"> DOI: 10.1093/nar/gkh121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1093%2Fnar%2Fgkh121" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=D138-D141&author=A.+Batemanauthor=L.+Coinauthor=R.+Durbinauthor=R.+D.+Finnauthor=V.+Hollichauthor=S.+Griffiths-Jonesauthor=A.+Khannaauthor=M.+Marshallauthor=S.+Moxonauthor=E.+L.+Sonnhammer&title=The+Pfam+protein+families+database&doi=10.1093%2Fnar%2Fgkh121"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh121%26sid%3Dliteratum%253Aachs%26aulast%3DBateman%26aufirst%3DA.%26aulast%3DCoin%26aufirst%3DL.%26aulast%3DDurbin%26aufirst%3DR.%26aulast%3DFinn%26aufirst%3DR.%2BD.%26aulast%3DHollich%26aufirst%3DV.%26aulast%3DGriffiths-Jones%26aufirst%3DS.%26aulast%3DKhanna%26aufirst%3DA.%26aulast%3DMarshall%26aufirst%3DM.%26aulast%3DMoxon%26aufirst%3DS.%26aulast%3DSonnhammer%26aufirst%3DE.%2BL.%26atitle%3DThe%2520Pfam%2520protein%2520families%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DD138%26epage%3DD141%26doi%3D10.1093%2Fnar%2Fgkh121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Balaydın, H. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Şentürk, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Menzek, A.</span><span> </span><span class="NLM_article-title">Synthesis and carbonic anhydrase inhibitory properties of novel cyclohexanonyl bromophenol derivatives</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1352</span><span class="NLM_x">–</span> <span class="NLM_lpage">1357</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.12.069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2Fj.bmcl.2011.12.069" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=1352-1357&author=H.+T.+Balayd%C4%B1nauthor=M.+%C5%9Eent%C3%BCrkauthor=A.+Menzek&title=Synthesis+and+carbonic+anhydrase+inhibitory+properties+of+novel+cyclohexanonyl+bromophenol+derivatives&doi=10.1016%2Fj.bmcl.2011.12.069"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.12.069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.12.069%26sid%3Dliteratum%253Aachs%26aulast%3DBalayd%25C4%25B1n%26aufirst%3DH.%2BT.%26aulast%3D%25C5%259Eent%25C3%25BCrk%26aufirst%3DM.%26aulast%3DMenzek%26aufirst%3DA.%26atitle%3DSynthesis%2520and%2520carbonic%2520anhydrase%2520inhibitory%2520properties%2520of%2520novel%2520cyclohexanonyl%2520bromophenol%2520derivatives%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D1352%26epage%3D1357%26doi%3D10.1016%2Fj.bmcl.2011.12.069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Saxty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyle, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barford, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Downham, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garrett, M. D.</span><span> </span><span class="NLM_article-title">Identification of inhibitors of protein kinase B using fragment-based lead discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2293</span><span class="NLM_x">–</span> <span class="NLM_lpage">2296</span><span class="refDoi"> DOI: 10.1021/jm070091b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070091b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVKjur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2293-2296&author=G.+Saxtyauthor=S.+J.+Woodheadauthor=V.+Berdiniauthor=T.+G.+Daviesauthor=M.+L.+Verdonkauthor=P.+G.+Wyattauthor=R.+G.+Boyleauthor=D.+Barfordauthor=R.+Downhamauthor=M.+D.+Garrett&title=Identification+of+inhibitors+of+protein+kinase+B+using+fragment-based+lead+discovery&doi=10.1021%2Fjm070091b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Inhibitors of Protein Kinase B Using Fragment-Based Lead Discovery</span></div><div class="casAuthors">Saxty, Gordon; Woodhead, Steven J.; Berdini, Valerio; Davies, Thomas G.; Verdonk, Marcel L.; Wyatt, Paul G.; Boyle, Robert G.; Barford, David; Downham, Robert; Garrett, Michelle D.; Carr, Robin A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2293-2296</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using fragment-based screening techniques, 5-methyl-4-phenyl-1H-pyrazole (IC50 80 μM) was identified as a novel, low mol. wt. inhibitor of protein kinase B (PKB).  Herein we describe the rapid elaboration of highly potent and ligand efficient analogs using a fragment growing approach.  Iterative structure-based design was supported by protein-ligand structure detns. using a PKA-PKB "chimera" and a final protein-ligand structure of a lead compd. in PKBβ itself.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlCYaIT8Xn87Vg90H21EOLACvtfcHk0ljMxFzyfOSUvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVKjur8%253D&md5=2a09e4bbaca1493be8e215927e7e72e1</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm070091b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070091b%26sid%3Dliteratum%253Aachs%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DWoodhead%26aufirst%3DS.%2BJ.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26aulast%3DBoyle%26aufirst%3DR.%2BG.%26aulast%3DBarford%26aufirst%3DD.%26aulast%3DDownham%26aufirst%3DR.%26aulast%3DGarrett%26aufirst%3DM.%2BD.%26atitle%3DIdentification%2520of%2520inhibitors%2520of%2520protein%2520kinase%2520B%2520using%2520fragment-based%2520lead%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2293%26epage%3D2296%26doi%3D10.1021%2Fjm070091b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span> </span><span class="NLM_article-title">The consequences of translational and rotational entropy lost by small molecules on binding to proteins</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">741</span><span class="NLM_x">–</span> <span class="NLM_lpage">753</span><span class="refDoi"> DOI: 10.1023/A:1022446720849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1023%2FA%3A1022446720849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=12650591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD3sXht1Wktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=741-753&author=C.+W.+Murrayauthor=M.+L.+Verdonk&title=The+consequences+of+translational+and+rotational+entropy+lost+by+small+molecules+on+binding+to+proteins&doi=10.1023%2FA%3A1022446720849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The consequences of translational and rotational entropy lost by small molecules on binding to proteins</span></div><div class="casAuthors">Murray, Christopher W.; Verdonk, Marcel L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">741-753</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A review.  When a small mol. binds to a protein, it loses a significant amt. of rigid body translational and rotational entropy.  Ests. of the assocd. energy barrier vary widely in the literature yet accurate ests. are important in the interpretation of results from fragment-based drug discovery techniques.  This paper describes an anal. that allows the estn. of the rigid body entropy barrier from the increase in binding affinities that results when two fragments of known affinity and known binding mode are joined together.  The paper reviews the relatively rare no. of examples where good quality data is available.  From the anal. of this data, we est. that the barrier to binding, due to the loss of rigid-body entropy, is 15-20 kJ/mol, i.e. around 3 orders of magnitude in affinity at 298 K.  This large barrier explains why it is comparatively rare to observe multiple fragments binding to non-overlapping adjacent sites in enzymes.  The barrier is also consistent with medicinal chem. experience where small changes in the crit. binding regions of ligands are often poorly tolerated by enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6KId6AyrMCLVg90H21EOLACvtfcHk0ljMxFzyfOSUvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXht1Wktrw%253D&md5=4ad6aef74de3cb86302fe16f7e61c1c6</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1023%2FA%3A1022446720849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1022446720849%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26atitle%3DThe%2520consequences%2520of%2520translational%2520and%2520rotational%2520entropy%2520lost%2520by%2520small%2520molecules%2520on%2520binding%2520to%2520proteins%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D741%26epage%3D753%26doi%3D10.1023%2FA%3A1022446720849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Davies, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span> </span><span class="NLM_article-title">Fragment-based discovery of inhibitors of protein kinase B</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1705</span><span class="NLM_x">–</span> <span class="NLM_lpage">1717</span><span class="refDoi"> DOI: 10.2174/156802609790102419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.2174%2F156802609790102419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=19929839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFeitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=1705-1717&author=T.+G.+Daviesauthor=S.+J.+Woodheadauthor=I.+Collins&title=Fragment-based+discovery+of+inhibitors+of+protein+kinase+B&doi=10.2174%2F156802609790102419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based discovery of inhibitors of protein kinase B</span></div><div class="casAuthors">Davies, Thomas G.; Woodhead, Steven J.; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1705-1717</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Multiple ligand efficient fragment inhibitors of protein kinase B were identified through a combined in silico compd. screen and high-throughput crystallog. anal. of protein-ligand structures.  A well-validated apo-PKB-PKA chimeric protein provided a convenient platform for high-throughput crystallog. by soaking of inhibitors, and a method for the detn. of PKB-ligand structures was developed to support inhibitor development.  Pyrazole and azaindole fragment hits with micromolar activity were rapidly progressed to nanomolar inhibitors of PKB with activity in cells using crystallog. anal. of inhibitor binding modes to guide medicinal chem.  Compds. with selectivity for PKB over PKA and other kinases were identified by this approach, resulting in potent inhibitors with in vivo activity through both oral and systemic administration, and suitable for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc9x00Q3BGobVg90H21EOLACvtfcHk0lgg-BAzruKyhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFeitr8%253D&md5=b06ab1b9f05c336eec0b68d63622907c</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.2174%2F156802609790102419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802609790102419%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26aulast%3DWoodhead%26aufirst%3DS.%2BJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DFragment-based%2520discovery%2520of%2520inhibitors%2520of%2520protein%2520kinase%2520B%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2009%26volume%3D9%26spage%3D1705%26epage%3D1717%26doi%3D10.2174%2F156802609790102419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Hendlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnickel, G.</span><span> </span><span class="NLM_article-title">LIGSITE: automatic and efficient detection of potential small molecule-binding sites in proteins</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">359</span><span class="NLM_x">–</span> <span class="NLM_lpage">363</span><span class="refDoi"> DOI: 10.1016/S1093-3263(98)00002-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2FS1093-3263%2898%2900002-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=9704298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A280%3ADyaK1czmvF2hsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1997&pages=359-363&author=M.+Hendlichauthor=F.+Rippmannauthor=G.+Barnickel&title=LIGSITE%3A+automatic+and+efficient+detection+of+potential+small+molecule-binding+sites+in+proteins&doi=10.1016%2FS1093-3263%2898%2900002-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">LIGSITE: automatic and efficient detection of potential small molecule-binding sites in proteins</span></div><div class="casAuthors">Hendlich M; Rippmann F; Barnickel G</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular graphics & modelling</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">359-63, 389</span>
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    </div><div class="casAbstract">LIGSITE is a new program for the automatic and time-efficient detection of pockets on the surface of proteins that may act as binding sites for small molecule ligands.  Pockets are identified with a series of simple operations on a cubic grid.  Using a set of receptor-ligand complexes we show that LIGSITE is able to identify the binding sites of small molecule ligands with high precision.  The main advantage of LIGSITE is its speed.  Typical search times are in the range of 5 to 20 s for medium-sized proteins.  LIGSITE is therefore well suited for identification of pockets in large sets of proteins (e.g., protein families) for comparative studies.  For graphical display LIGSITE produces VRML representations of the protein-ligand complex and the binding site for display with a VRML viewer such as WebSpace from SGI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRvgmm5GX6HIgaK-_ZudimnfW6udTcc2ea5DDzkCnmGlbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1czmvF2hsw%253D%253D&md5=b7174632bdeaebcfbedec66824ec5cb6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2898%2900002-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252898%252900002-3%26sid%3Dliteratum%253Aachs%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DBarnickel%26aufirst%3DG.%26atitle%3DLIGSITE%253A%2520automatic%2520and%2520efficient%2520detection%2520of%2520potential%2520small%2520molecule-binding%2520sites%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D1997%26volume%3D15%26spage%3D359%26epage%3D363%26doi%3D10.1016%2FS1093-3263%2898%2900002-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Ludlow, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saini, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tickle, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Detection of secondary binding sites in proteins using fragment screening</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">15910</span><span class="NLM_x">–</span> <span class="NLM_lpage">15915</span><span class="refDoi"> DOI: 10.1073/pnas.1518946112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1073%2Fpnas.1518946112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=26655740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=15910-15915&author=R.+F.+Ludlowauthor=M.+L.+Verdonkauthor=H.+K.+Sainiauthor=I.+J.+Tickleauthor=H.+Jhoti&title=Detection+of+secondary+binding+sites+in+proteins+using+fragment+screening&doi=10.1073%2Fpnas.1518946112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of secondary binding sites in proteins using fragment screening</span></div><div class="casAuthors">Ludlow, R. Frederick; Verdonk, Marcel L.; Saini, Harpreet K.; Tickle, Ian J.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">15910-15915</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Proteins need to be tightly regulated as they control biol. processes in most normal cellular functions.  The precise mechanisms of regulation are rarely completely understood but can involve binding of endogenous ligands and/or partner proteins at specific locations on a protein that can modulate function.  Often, these addnl. secondary binding sites appear sep. to the primary binding site, which, e.g., for an enzyme may bind a substrate.  In previous work, the authors uncovered several examples in which secondary binding sites were discovered on proteins using fragment screening approaches.  In each case, the authors were able to establish that the newly identified secondary binding site was biol. relevant as it was able to modulate function by the binding of a small mol.  Here, the authors investigated how often secondary binding sites are located on proteins by analyzing 24 protein targets for which they performed a fragment screen using x-ray crystallog.  The anal. showed that, surprisingly, the majority of proteins contained secondary binding sites based on their ability to bind fragments.  Furthermore, sequence anal. of these previously unknown sites indicated high conservation, which suggested that they may have a biol. function, perhaps via an allosteric mechanism.  Comparing the physicochem. properties of the secondary sites with known primary ligand-binding sites also showed broad similarities indicating that many of the secondary sites may be druggable in nature with small mols. that could provide new opportunities to modulate potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ITF1w230fbVg90H21EOLACvtfcHk0lgfKk6VSyU3Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP&md5=9e5f15595263dc049c713d5a3f5912a1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1518946112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1518946112%26sid%3Dliteratum%253Aachs%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DSaini%26aufirst%3DH.%2BK.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DDetection%2520of%2520secondary%2520binding%2520sites%2520in%2520proteins%2520using%2520fragment%2520screening%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D15910%26epage%3D15915%26doi%3D10.1073%2Fpnas.1518946112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Mooij, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span> </span><span class="NLM_article-title">General and targeted statistical potentials for protein–ligand interactions</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">272</span><span class="NLM_x">–</span> <span class="NLM_lpage">287</span><span class="refDoi"> DOI: 10.1002/prot.20588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1002%2Fprot.20588" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=272-287&author=W.+Mooijauthor=M.+L.+Verdonk&title=General+and+targeted+statistical+potentials+for+protein%E2%80%93ligand+interactions&doi=10.1002%2Fprot.20588"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fprot.20588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20588%26sid%3Dliteratum%253Aachs%26aulast%3DMooij%26aufirst%3DW.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26atitle%3DGeneral%2520and%2520targeted%2520statistical%2520potentials%2520for%2520protein%25E2%2580%2593ligand%2520interactions%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2005%26volume%3D61%26spage%3D272%26epage%3D287%26doi%3D10.1002%2Fprot.20588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Korb, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stutzle, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Exner, T. E.</span><span> </span><span class="NLM_article-title">Empirical scoring functions for advanced protein– ligand docking with PLANTS</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1021/ci800298z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800298z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=84-96&author=O.+Korbauthor=T.+Stutzleauthor=T.+E.+Exner&title=Empirical+scoring+functions+for+advanced+protein%E2%80%93+ligand+docking+with+PLANTS&doi=10.1021%2Fci800298z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical Scoring Functions for Advanced Protein-Ligand Docking with PLANTS</span></div><div class="casAuthors">Korb, Oliver; Stuetzle, Thomas; Exner, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-96</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper we present two empirical scoring functions, PLANTSCHEMPLP and PLANTSPLP, designed for our docking algorithm PLANTS (Protein-Ligand ANT System), which is based on ant colony optimization (ACO).  They are related, regarding their functional form, to parts of already published scoring functions and force fields.  The parametrization procedure described here was able to identify several parameter settings showing an excellent performance for the task of pose prediction on two test sets comprising 298 complexes in total.  Up to 87% of the complexes of the Astex diverse set and 77% of the CCDC/Astex clean listnc (noncovalently bound complexes of the clean list) could be reproduced with root-mean-square deviations of less than 2 Å with respect to the exptl. detd. structures.  A comparison with the state-of-the-art docking tool GOLD clearly shows that this is, esp. for the druglike Astex diverse set, an improvement in pose prediction performance.  Addnl., optimized parameter settings for the search algorithm were identified, which can be used to balance pose prediction reliability and search speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_kyR7CGk9mbVg90H21EOLACvtfcHk0lgfKk6VSyU3Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVSquw%253D%253D&md5=fce960b75b9f8906354b7c97ff84c092</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fci800298z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800298z%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DStutzle%26aufirst%3DT.%26aulast%3DExner%26aufirst%3DT.%2BE.%26atitle%3DEmpirical%2520scoring%2520functions%2520for%2520advanced%2520protein%25E2%2580%2593%2520ligand%2520docking%2520with%2520PLANTS%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D84%26epage%3D96%26doi%3D10.1021%2Fci800298z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Eldridge, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auton, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paolini, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mee, R. P.</span><span> </span><span class="NLM_article-title">Empirical scoring functions: I. The development of a fast empirical scoring function to estimate the binding affinity of ligands in receptor complexes</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">425</span><span class="NLM_x">–</span> <span class="NLM_lpage">445</span><span class="refDoi"> DOI: 10.1023/A:1007996124545</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1023%2FA%3A1007996124545" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=9385547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK2sXnsV2it7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1997&pages=425-445&author=M.+D.+Eldridgeauthor=C.+W.+Murrayauthor=T.+R.+Autonauthor=G.+V.+Paoliniauthor=R.+P.+Mee&title=Empirical+scoring+functions%3A+I.+The+development+of+a+fast+empirical+scoring+function+to+estimate+the+binding+affinity+of+ligands+in+receptor+complexes&doi=10.1023%2FA%3A1007996124545"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical scoring functions: I. The development of a fast empirical scoring function to estimate the binding affinity of ligands in receptor complexes</span></div><div class="casAuthors">Eldridge, Matthew D.; Murray, Christopher W.; Auton, Timothy R.; Paolini, Gaia V.; Mee, Roger P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">425-445</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">This paper describes the development of a simple empirical scoring function designed to est. the free energy of binding for a protein-ligand complex when the 3D structure of the complex is known or can be approximated.  The function uses simple contact terms to est. lipophilic and metal-ligand binding contributions, a simple explicit form for hydrogen bonds and a term which penalizes flexibility.  The coeffs. of each term are obtained using a regression based on 82 ligand-receptor complexes for which the binding affinity is known.  The function reproduces the binding affinity of the complexes with a cross-validated error of 8.68 kJ/mol.  Tests on internal consistency indicate that the coeffs. obtained are stable to changes in the compn. of the training set.  The function is also tested on two test sets contg. a further 20 and 10 complexes, resp.  The deficiencies of this type of function are discussed and it is compared to approaches by other workers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquJZ-k95NG1LVg90H21EOLACvtfcHk0ljCmU6HK6ra1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsV2it7o%253D&md5=ece4b881e181f3d5c1287af55efaf6b4</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1023%2FA%3A1007996124545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1007996124545%26sid%3Dliteratum%253Aachs%26aulast%3DEldridge%26aufirst%3DM.%2BD.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DAuton%26aufirst%3DT.%2BR.%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DMee%26aufirst%3DR.%2BP.%26atitle%3DEmpirical%2520scoring%2520functions%253A%2520I.%2520The%2520development%2520of%2520a%2520fast%2520empirical%2520scoring%2520function%2520to%2520estimate%2520the%2520binding%2520affinity%2520of%2520ligands%2520in%2520receptor%2520complexes%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1997%26volume%3D11%26spage%3D425%26epage%3D445%26doi%3D10.1023%2FA%3A1007996124545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Jones, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span> </span><span class="NLM_article-title">Molecular recognition of receptor sites using a genetic algorithm with a description of desolvation</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">245</span><span class="NLM_x">, </span> <span class="NLM_fpage">43</span><span class="NLM_x">–</span> <span class="NLM_lpage">53</span><span class="refDoi"> DOI: 10.1016/S0022-2836(95)80037-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1016%2FS0022-2836%2895%2980037-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=7823319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADyaK2MXjtVKlsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=245&publication_year=1995&pages=43-53&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glen&title=Molecular+recognition+of+receptor+sites+using+a+genetic+algorithm+with+a+description+of+desolvation&doi=10.1016%2FS0022-2836%2895%2980037-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of a receptor sites using a genetic algorithm with a description of desolvation</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">245</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-53</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Understanding the principles whereby macromol. biol. receptors can recognize small mol. substrates or inhibitors is the subject of a major effort.  This is of paramount importance in rational drug design where the receptor structure is known (the "docking" problem).  Current theor. approaches utilize models of the steric and electrostatic interaction of bound ligands and recently conformational flexibility has been incorporated.  The authors report results based on software using a genetic algorithm that uses an evolutionary strategy in exploring the full conformational flexibility of the ligand with partial flexibility of the protein, and which satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Results are reported on five test systems showing excellent agreement with exptl. data.  The design of the algorithm offers insight into the mol. recognition mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Xcn7nQyVRLVg90H21EOLACvtfcHk0ljCmU6HK6ra1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjtVKlsbc%253D&md5=bf45edafb986a73c7c1d0ac55e070f28</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2895%2980037-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252895%252980037-9%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26atitle%3DMolecular%2520recognition%2520of%2520receptor%2520sites%2520using%2520a%2520genetic%2520algorithm%2520with%2520a%2520description%2520of%2520desolvation%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1995%26volume%3D245%26spage%3D43%26epage%3D53%26doi%3D10.1016%2FS0022-2836%2895%2980037-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Nadendla, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teyssier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delfosse, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vivat, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnasamy, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gronemeyer, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourguet, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Germain, P.</span><span> </span><span class="NLM_article-title">An unexpected mode of binding defines BMS948 as a full retinoic acid receptor β (RARβ, NR1B2) selective agonist</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0123195</span><span class="refDoi"> DOI: 10.1371/journal.pone.0123195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1371%2Fjournal.pone.0123195" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0123195&author=E.+Nadendlaauthor=C.+Teyssierauthor=V.+Delfosseauthor=V.+Vivatauthor=G.+Krishnasamyauthor=H.+Gronemeyerauthor=W.+Bourguetauthor=P.+Germain&title=An+unexpected+mode+of+binding+defines+BMS948+as+a+full+retinoic+acid+receptor+%CE%B2+%28RAR%CE%B2%2C+NR1B2%29+selective+agonist&doi=10.1371%2Fjournal.pone.0123195"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0123195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0123195%26sid%3Dliteratum%253Aachs%26aulast%3DNadendla%26aufirst%3DE.%26aulast%3DTeyssier%26aufirst%3DC.%26aulast%3DDelfosse%26aufirst%3DV.%26aulast%3DVivat%26aufirst%3DV.%26aulast%3DKrishnasamy%26aufirst%3DG.%26aulast%3DGronemeyer%26aufirst%3DH.%26aulast%3DBourguet%26aufirst%3DW.%26aulast%3DGermain%26aufirst%3DP.%26atitle%3DAn%2520unexpected%2520mode%2520of%2520binding%2520defines%2520BMS948%2520as%2520a%2520full%2520retinoic%2520acid%2520receptor%2520%25CE%25B2%2520%2528RAR%25CE%25B2%252C%2520NR1B2%2529%2520selective%2520agonist%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0123195%26doi%3D10.1371%2Fjournal.pone.0123195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">García-Sosa, A. T.</span><span> </span><span class="NLM_article-title">Hydration properties of ligands and drugs in protein binding sites: Tightly-bound, bridging water molecules and their effects and consequences on molecular design strategies</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1388</span><span class="NLM_x">–</span> <span class="NLM_lpage">1405</span><span class="refDoi"> DOI: 10.1021/ci3005786</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3005786" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsVeqt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=1388-1405&author=A.+T.+Garc%C3%ADa-Sosa&title=Hydration+properties+of+ligands+and+drugs+in+protein+binding+sites%3A+Tightly-bound%2C+bridging+water+molecules+and+their+effects+and+consequences+on+molecular+design+strategies&doi=10.1021%2Fci3005786"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Hydration Properties of Ligands and Drugs in Protein Binding Sites: Tightly-Bound, Bridging Water Molecules and Their Effects and Consequences on Molecular Design Strategies</span></div><div class="casAuthors">Garcia-Sosa, Alfonso T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1388-1405</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Some water mols. in binding sites are important for intermol. interactions and stability.  The way binding site explicit water mols. are dealt with affects the diversity and nature of designed ligand chem. structures and properties.  The strategies commonly employed frequently assume that a gain in binding affinity will be achieved by their targeting or neglect.  However, in the present work, 2332 high-resoln. X-ray crystal structures of hydrated and non-hydrated, drug and nondrug compds. in biomol. complexes with reported Ki or Kd show that compds. that use tightly bound, bridging water mols. are as potent as those that do not.  The distribution of their energies, physicochem. properties, and ligand efficiency indexes were compared for statistical significance, and the results were confirmed using 2000 permutation runs.  Ligand cases were also split into agonists and antagonists, and crystal structure pairs with differing tightly bound water mols. were also compared.  In addn., agonists and antagonists that use tightly bound water bridges are smaller, less lipophilic, and less planar; have deeper ligand efficiency indexes; and in general, possess better physicochem. properties for further development.  Therefore, tightly bound, bridging water mols. may in some cases be replaced and targeted as a strategy, though sometimes keeping them as bridges may be better from a pharmacodynamic perspective.  The results suggest general indications on tightly hydrated and non-tightly hydrated compds. in binding sites and practical considerations to adopt a strategy in drug and mol. design when faced with this special type of water mols.  There are also benefits of lower log P and better developability for tightly hydrated compds., while stronger potency is not always required or beneficial.  The hydrated binding site may be one of the many structure conformations available to the receptor, and different ligands will have a different ability to select either hydrated or non-hydrated receptor binding site conformations.  Compds. may thus be designed, and if a tightly bound, bridging water mol. is obsd. in the binding site, attempts to replace it should only be made if the subsequent ligand modification would improve also its ligand efficiency, enthalpy, specificity, and pharmacokinetic properties.  If the modification does succeed in replacing the tightly bound, bridging water mol., it will have at least achieved benefits for ligand optimization and development independently of either pos. or neg. change in binding affinity outcome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWcKH47-JHsLVg90H21EOLACvtfcHk0lis1N-Kdq2Mqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsVeqt7w%253D&md5=3bbc8769c80c60d1377eb22826647f80</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fci3005786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3005786%26sid%3Dliteratum%253Aachs%26aulast%3DGarc%25C3%25ADa-Sosa%26aufirst%3DA.%2BT.%26atitle%3DHydration%2520properties%2520of%2520ligands%2520and%2520drugs%2520in%2520protein%2520binding%2520sites%253A%2520Tightly-bound%252C%2520bridging%2520water%2520molecules%2520and%2520their%2520effects%2520and%2520consequences%2520on%2520molecular%2520design%2520strategies%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D1388%26epage%3D1405%26doi%3D10.1021%2Fci3005786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">The UniProt Consortium.</span><span> </span><span class="NLM_article-title">UniProt: the universal protein knowledgebase</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">D158</span><span class="NLM_x">–</span> <span class="NLM_lpage">D169</span><span class="refDoi"> DOI: 10.1093/nar/gkw1099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1093%2Fnar%2Fgkw1099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=27899622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D158-D169&author=The+UniProt+Consortium.&title=UniProt%3A+the+universal+protein+knowledgebase&doi=10.1093%2Fnar%2Fgkw1099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1099%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniProt%253A%2520the%2520universal%2520protein%2520knowledgebase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD158%26epage%3DD169%26doi%3D10.1093%2Fnar%2Fgkw1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span>RDKit, Open-source chemoinformatics. <a href="http://www.rdkit.org" class="extLink">http://www.rdkit.org</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=RDKit%2C+Open-source+chemoinformatics.+http%3A%2F%2Fwww.rdkit.org."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Edgar, R. C.</span><span> </span><span class="NLM_article-title">MUSCLE: a multiple sequence alignment method with reduced time and space complexity</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="refDoi"> DOI: 10.1186/1471-2105-5-113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1186%2F1471-2105-5-113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=14706121" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=1&author=R.+C.+Edgar&title=MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity&doi=10.1186%2F1471-2105-5-113"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-5-113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-5-113%26sid%3Dliteratum%253Aachs%26aulast%3DEdgar%26aufirst%3DR.%2BC.%26atitle%3DMUSCLE%253A%2520a%2520multiple%2520sequence%2520alignment%2520method%2520with%2520reduced%2520time%2520and%2520space%2520complexity%26jtitle%3DBMC%2520Bioinf.%26date%3D2004%26volume%3D5%26spage%3D1%26doi%3D10.1186%2F1471-2105-5-113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Kearsley, S. K.</span><span> </span><span class="NLM_article-title">On the orthogonal transformation used for structural comparisons</span> <span class="citation_source-journal">Acta Crystallogr., Sect. A: Found. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">208</span><span class="NLM_x">–</span> <span class="NLM_lpage">210</span><span class="refDoi"> DOI: 10.1107/S0108767388010128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1107%2FS0108767388010128" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=1989&pages=208-210&author=S.+K.+Kearsley&title=On+the+orthogonal+transformation+used+for+structural+comparisons&doi=10.1107%2FS0108767388010128"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0108767388010128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108767388010128%26sid%3Dliteratum%253Aachs%26aulast%3DKearsley%26aufirst%3DS.%2BK.%26atitle%3DOn%2520the%2520orthogonal%2520transformation%2520used%2520for%2520structural%2520comparisons%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Found.%2520Crystallogr.%26date%3D1989%26volume%3D45%26spage%3D208%26epage%3D210%26doi%3D10.1107%2FS0108767388010128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Labute, P.</span><span> </span><span class="NLM_article-title">Protonate3D: assignment of ionization states and hydrogen coordinates to macromolecular structures</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">205</span><span class="refDoi"> DOI: 10.1002/prot.22234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1002%2Fprot.22234" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=18814299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisV2isrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2009&pages=187-205&author=P.+Labute&title=Protonate3D%3A+assignment+of+ionization+states+and+hydrogen+coordinates+to+macromolecular+structures&doi=10.1002%2Fprot.22234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Protonate3D: assignment of ionization states and hydrogen coordinates to macromolecular structures</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">187-205</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A new method, called Protonate3D, is presented for the automated prediction of hydrogen coordinates given the 3D coordinates of the heavy atoms of a macromol. structure.  Protonate3D considers side-chain "flip," rotamer, tautomer, and ionization states of all chem. groups, ligands, and solvent, provided suitable templates are available in a parameter file.  The energy model includes van der Waals, Coulomb, solvation, rotamer, tautomer, and titrn. effects.  The results of computational validation expts. suggest that Protonate3D can accurately predict the location of hydrogen atoms in macromol. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmnIRzxpxpCrVg90H21EOLACvtfcHk0lhYhnDPTtMKyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisV2isrs%253D&md5=89803fe2028fb9257eb09854a27ef2dd</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fprot.22234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22234%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DProtonate3D%253A%2520assignment%2520of%2520ionization%2520states%2520and%2520hydrogen%2520coordinates%2520to%2520macromolecular%2520structures%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2009%26volume%3D75%26spage%3D187%26epage%3D205%26doi%3D10.1002%2Fprot.22234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span> <span class="citation_source-book">Molecular Operating Environment (MOE), 2013.08</span>; <span class="NLM_publisher-name">Chemical Computing Group Inc.</span>: <span class="NLM_publisher-loc">Montreal, QC, Canada</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Molecular+Operating+Environment+%28MOE%29%2C+2013.08%3B+Chemical+Computing+Group+Inc.%3A+Montreal%2C+QC%2C+Canada%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%252C%25202013.08%26pub%3DChemical%2520Computing%2520Group%2520Inc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">McConkey, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sobolev, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edelman, M.</span><span> </span><span class="NLM_article-title">Quantification of protein surfaces, volumes and atom–atom contacts using a constrained Voronoi procedure</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1365</span><span class="NLM_x">–</span> <span class="NLM_lpage">1373</span><span class="refDoi"> DOI: 10.1093/bioinformatics/18.10.1365</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=10.1093%2Fbioinformatics%2F18.10.1365" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2002&pages=1365-1373&author=B.+J.+McConkeyauthor=V.+Sobolevauthor=M.+Edelman&title=Quantification+of+protein+surfaces%2C+volumes+and+atom%E2%80%93atom+contacts+using+a+constrained+Voronoi+procedure&doi=10.1093%2Fbioinformatics%2F18.10.1365"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2F18.10.1365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252F18.10.1365%26sid%3Dliteratum%253Aachs%26aulast%3DMcConkey%26aufirst%3DB.%2BJ.%26aulast%3DSobolev%26aufirst%3DV.%26aulast%3DEdelman%26aufirst%3DM.%26atitle%3DQuantification%2520of%2520protein%2520surfaces%252C%2520volumes%2520and%2520atom%25E2%2580%2593atom%2520contacts%2520using%2520a%2520constrained%2520Voronoi%2520procedure%26jtitle%3DBioinformatics%26date%3D2002%26volume%3D18%26spage%3D1365%26epage%3D1373%26doi%3D10.1093%2Fbioinformatics%2F18.10.1365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Sastry, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherman, W.</span><span> </span><span class="NLM_article-title">Rapid shape-based ligand alignment and virtual screening method based on atom/feature-pair similarities and volume overlap scoring</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2455</span><span class="NLM_x">–</span> <span class="NLM_lpage">2466</span><span class="refDoi"> DOI: 10.1021/ci2002704</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2002704" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKnt73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2455-2466&author=G.+M.+Sastryauthor=S.+L.+Dixonauthor=W.+Sherman&title=Rapid+shape-based+ligand+alignment+and+virtual+screening+method+based+on+atom%2Ffeature-pair+similarities+and+volume+overlap+scoring&doi=10.1021%2Fci2002704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Shape-Based Ligand Alignment and Virtual Screening Method Based on Atom/Feature-Pair Similarities and Volume Overlap Scoring</span></div><div class="casAuthors">Sastry, G. Madhavi; Dixon, Steven L.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2455-2466</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Shape-based methods for aligning and scoring ligands have proven to be valuable in the field of computer-aided drug design.  Here, we describe a new shape-based flexible ligand superposition and virtual screening method, Phase Shape, which is shown to rapidly produce accurate 3D ligand alignments and efficiently enrich actives in virtual screening.  We describe the methodol., which is based on the principle of atom distribution triplets to rapidly define trial alignments, followed by refinement of top alignments to maximize the vol. overlap.  The method can be run in a shape-only mode or it can include atom types or pharmacophore feature encoding, the latter consistently producing the best results for database screening.  We apply Phase Shape to flexibly align mols. that bind to the same target and show that the method consistently produces correct alignments when compared with crystal structures.  We then illustrate the effectiveness of the method for identifying active compds. in virtual screening of eleven diverse targets.  Multiple parameters are explored, including atom typing, query structure conformation, and the database conformer generation protocol.  We show that Phase Shape performs well in database screening calcns. when compared with other shape-based methods using a common set of actives and decoys from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM-u23DvDpQ7Vg90H21EOLACvtfcHk0lhHgVd7O6CDhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKnt73K&md5=4234abb2dd7e6364c377e92047b65aba</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fci2002704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2002704%26sid%3Dliteratum%253Aachs%26aulast%3DSastry%26aufirst%3DG.%2BM.%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DRapid%2520shape-based%2520ligand%2520alignment%2520and%2520virtual%2520screening%2520method%2520based%2520on%2520atom%252Ffeature-pair%2520similarities%2520and%2520volume%2520overlap%2520scoring%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2455%26epage%3D2466%26doi%3D10.1021%2Fci2002704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PFJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PFJ','PDB','4PFJ'); return false;">PDB: 4PFJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2F6V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2F6V','PDB','2F6V'); return false;">PDB: 2F6V</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VCQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VCQ','PDB','2VCQ'); return false;">PDB: 2VCQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FGD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FGD','PDB','3FGD'); return false;">PDB: 3FGD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GEP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GEP','PDB','3GEP'); return false;">PDB: 3GEP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GBA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GBA','PDB','3GBA'); return false;">PDB: 3GBA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ARA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ARA','PDB','3ARA'); return false;">PDB: 3ARA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AP7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AP7','PDB','4AP7'); return false;">PDB: 4AP7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MGA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MGA','PDB','4MGA'); return false;">PDB: 4MGA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WRB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WRB','PDB','4WRB'); return false;">PDB: 4WRB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X7C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X7C','PDB','2X7C'); return false;">PDB: 2X7C</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1C83" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1C83','PDB','1C83'); return false;">PDB: 1C83</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4F9Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4F9Y','PDB','4F9Y'); return false;">PDB: 4F9Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2W0B" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2W0B','PDB','2W0B'); return false;">PDB: 2W0B</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N8E" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N8E','PDB','4N8E'); return false;">PDB: 4N8E</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OGN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OGN','PDB','4OGN'); return false;">PDB: 4OGN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AXA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AXA','PDB','4AXA'); return false;">PDB: 4AXA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MXC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MXC','PDB','4MXC'); return false;">PDB: 4MXC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JYG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JYG','PDB','4JYG'); return false;">PDB: 4JYG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UWE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UWE','PDB','3UWE'); return false;">PDB: 3UWE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KVK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KVK','PDB','3KVK'); return false;">PDB: 3KVK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3N5S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3N5S','PDB','3N5S'); return false;">PDB: 3N5S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2O7N" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2O7N','PDB','2O7N'); return false;">PDB: 2O7N</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EPV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EPV','PDB','4EPV'); return false;">PDB: 4EPV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RAL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RAL','PDB','3RAL'); return false;">PDB: 3RAL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3N3J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3N3J','PDB','3N3J'); return false;">PDB: 3N3J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BAK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BAK','PDB','4BAK'); return false;">PDB: 4BAK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FMK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FMK','PDB','3FMK'); return false;">PDB: 3FMK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LDO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LDO','PDB','1LDO'); return false;">PDB: 1LDO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GB9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GB9','PDB','4GB9'); return false;">PDB: 4GB9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CWO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CWO','PDB','4CWO'); return false;">PDB: 4CWO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X8Z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X8Z','PDB','2X8Z'); return false;">PDB: 2X8Z</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3E51" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3E51','PDB','3E51'); return false;">PDB: 3E51</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OP1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OP1','PDB','4OP1'); return false;">PDB: 4OP1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2J94" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2J94','PDB','2J94'); return false;">PDB: 2J94</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BW4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BW4','PDB','4BW4'); return false;">PDB: 4BW4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HOF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HOF','PDB','4HOF'); return false;">PDB: 4HOF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VTS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VTS','PDB','2VTS'); return false;">PDB: 2VTS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3T4Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3T4Q','PDB','3T4Q'); return false;">PDB: 3T4Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MSG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MSG','PDB','4MSG'); return false;">PDB: 4MSG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KZA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KZA','PDB','4KZA'); return false;">PDB: 4KZA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CJO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CJO','PDB','3CJO'); return false;">PDB: 3CJO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DCH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DCH','PDB','4DCH'); return false;">PDB: 4DCH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OIA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OIA','PDB','3OIA'); return false;">PDB: 3OIA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PCS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PCS','PDB','4PCS'); return false;">PDB: 4PCS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GV1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GV1','PDB','4GV1'); return false;">PDB: 4GV1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1W1Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1W1Y','PDB','1W1Y'); return false;">PDB: 1W1Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GXL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GXL','PDB','3GXL'); return false;">PDB: 3GXL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FFI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FFI','PDB','3FFI'); return false;">PDB: 3FFI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FRS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FRS','PDB','4FRS'); return false;">PDB: 4FRS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3B25" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3B25','PDB','3B25'); return false;">PDB: 3B25</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3A4P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3A4P','PDB','3A4P'); return false;">PDB: 3A4P</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i15"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00366">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_37252"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b00366">10.1021/acs.jmedchem.7b00366</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Prepared structure files for the validation set, detailed methods on the construction of the validation set and fragment mapping protocols, detailed results and discussion on the performance of different fragment mapping protocols used, binding mode and electrostatic potential surface of 3,5-dimethylisoxazole, PDB IDs of the complexes for each example in the validation set that place the fragment in the same binding subpocket (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00366/suppl_file/jm7b00366_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Validation set (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00366/suppl_file/jm7b00366_si_002.zip">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00366/suppl_file/jm7b00366_si_001.pdf">jm7b00366_si_001.pdf (1.3 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00366/suppl_file/jm7b00366_si_002.zip">jm7b00366_si_002.zip (6.91 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00366%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-9%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00366" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a9ea1eae53d60","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
