
<html lang="en"     class="pb-page"  data-request-id="0020283f-0364-4192-adce-84401e871bf1"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b01927;issue:issue:10.1021/jmcmar.2020.63.issue-17;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="How We Think about Targeting RNA with Small Molecules" /></meta><meta name="dc.Creator" content="Matthew G.  Costales" /></meta><meta name="dc.Creator" content="Jessica L.  Childs-Disney" /></meta><meta name="dc.Creator" content="Hafeez S.  Haniff" /></meta><meta name="dc.Creator" content="Matthew D.  Disney" /></meta><meta name="dc.Description" content="RNA offers nearly unlimited potential as a target for small molecule chemical probes and lead medicines. Many RNAs fold into structures that can be selectively targeted with small molecules. This P..." /></meta><meta name="Description" content="RNA offers nearly unlimited potential as a target for small molecule chemical probes and lead medicines. Many RNAs fold into structures that can be selectively targeted with small molecules. This P..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 26, 2020" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01927" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01927" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01927" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01927" /></link>
        
    
    

<title>How We Think about Targeting RNA with Small Molecules | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01927" /></meta><meta property="og:title" content="How We Think about Targeting RNA with Small Molecules" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0010.jpeg" /></meta><meta property="og:description" content="RNA offers nearly unlimited potential as a target for small molecule chemical probes and lead medicines. Many RNAs fold into structures that can be selectively targeted with small molecules. This Perspective discusses molecular recognition of RNA by small molecules and highlights key enabling technologies and properties of bioactive interactions. Sequence-based design of ligands targeting RNA has established rules for affecting RNA targets and provided a potentially general platform for the discovery of bioactive small molecules. The RNA targets that contain preferred small molecule binding sites can be identified from sequence, allowing identification of off-targets and prediction of bioactive interactions by nature of ligand recognition of functional sites. Small molecule targeted degradation of RNA targets (ribonuclease-targeted chimeras, RIBOTACs) and direct cleavage by small molecules have also been developed. These growing technologies suggest that the time is right to provide small molecule chemical probes to target functionally relevant RNAs throughout the human transcriptome." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01927"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01927">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01927&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01927&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01927&amp;href=/doi/10.1021/acs.jmedchem.9b01927" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 8880-8900</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01314" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.9b02021" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">How We Think about Targeting RNA with Small Molecules</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Matthew G. Costales</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew G. Costales</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+G.++Costales">Matthew G. Costales</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jessica L. Childs-Disney</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jessica L. Childs-Disney</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jessica+L.++Childs-Disney">Jessica L. Childs-Disney</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hafeez S. Haniff</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hafeez S. Haniff</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hafeez+S.++Haniff">Hafeez S. Haniff</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Matthew D. Disney</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew D. Disney</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#6a0e0319040f132a190918031a1a19440f0e1f"><span class="__cf_email__" data-cfemail="016568726f647841726273687171722f646574">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+D.++Disney">Matthew D. Disney</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath4">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8486-1796" title="Orcid link">http://orcid.org/0000-0001-8486-1796</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01927&amp;href=/doi/10.1021%2Facs.jmedchem.9b01927" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 8880–8900</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 26, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 November 2019</li><li><span class="item_label"><b>Published</b> online</span>26 March 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01927" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01927</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8880%26pageCount%3D21%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMatthew%2BG.%2BCostales%252C%2BJessica%2BL.%2BChilds-Disney%252C%2BHafeez%2BS.%2BHaniff%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D17%26contentID%3Dacs.jmedchem.9b01927%26title%3DHow%2BWe%2BThink%2Babout%2BTargeting%2BRNA%2Bwith%2BSmall%2BMolecules%26numPages%3D21%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8900%26publicationDate%3DSeptember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01927"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">7007</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">29</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01927" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;How We Think about Targeting RNA with Small Molecules&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;G. Costales&quot;},{&quot;first_name&quot;:&quot;Jessica&quot;,&quot;last_name&quot;:&quot;L. Childs-Disney&quot;},{&quot;first_name&quot;:&quot;Hafeez&quot;,&quot;last_name&quot;:&quot;S. Haniff&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;D. Disney&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;26&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;8880-8900&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01927&quot;},&quot;abstract&quot;:&quot;RNA offers nearly unlimited potential as a target for small molecule chemical probes and lead medicines. Many RNAs fold into structures that can be selectively targeted with small molecules. This Perspective discusses molecular recognition of RNA by small molecules and highlights key enabling technologies and properties of bioactive interactions. Sequence-based design of ligands targeting RNA has established rules for affecting RNA targets and provided a potentially general platform for the discovery of bioactive small molecules. The RNA targets that contain preferred small molecule binding sites can be identified from sequence, allowing identification of off-targets and prediction of bioactive interactions by nature of ligand recognition of functional sites. Small molecule targeted degradation of RNA targets (ribonuclease-targeted chimeras, RIBOTACs) and direct cleavage by small molecules have also been developed. These growing technologies suggest that the time is right to provide small molecule chemical pr&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01927&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01927" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01927&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01927" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01927&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01927" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01927&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01927&amp;href=/doi/10.1021/acs.jmedchem.9b01927" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01927" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01927" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01927&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01927%26sid%3Dliteratum%253Aachs%26pmid%3D32212706%26genre%3Darticle%26aulast%3DCostales%26date%3D2020%26atitle%3DHow%2BWe%2BThink%2Babout%2BTargeting%2BRNA%2Bwith%2BSmall%2BMolecules%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D17%26spage%3D8880%26epage%3D8900%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292544" title="Targeting">Targeting</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292276" title="Small molecules">Small molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/jmcmar.2020.63.issue-17/20200910/jmcmar.2020.63.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">RNA offers nearly unlimited potential as a target for small molecule chemical probes and lead medicines. Many RNAs fold into structures that can be selectively targeted with small molecules. This Perspective discusses molecular recognition of RNA by small molecules and highlights key enabling technologies and properties of bioactive interactions. Sequence-based design of ligands targeting RNA has established rules for affecting RNA targets and provided a potentially general platform for the discovery of bioactive small molecules. The RNA targets that contain preferred small molecule binding sites can be identified from sequence, allowing identification of off-targets and prediction of bioactive interactions by nature of ligand recognition of functional sites. Small molecule targeted degradation of RNA targets (ribonuclease-targeted chimeras, RIBOTACs) and direct cleavage by small molecules have also been developed. These growing technologies suggest that the time is right to provide small molecule chemical probes to target functionally relevant RNAs throughout the human transcriptome.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Drug Discovery in the “-omics” Era</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20334" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20334" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In 2001, the dissemination of the draft sequence of the human genome was noted as a crowning scientific achievement.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Genomics inspired “omics” for nearly every biomolecule such as the proteome, the glycome, and the transcriptome.<a onclick="showRef(event, 'ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5">(2−5)</a> Many of these studies have been aimed at associating differential biomolecule expression to disease states.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Ideally, being armed with an encyclopedia of genome sequence, one could readily identify the basis for understudied diseases and rapidly develop effective, orally bioavailable small molecule treatments.</div><div class="NLM_p">Because of the focus of using genomics to inform medicinal development, one of the first genome-wide analyses determined the number of open reading frames (ORFs). Surprisingly, these annotations, culminating in the Encyclopedia of DNA Elements (ENCODE) project,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> revealed that canonical ORFs are present in only <sup>1</sup>/<sub>3</sub> of the genes that humans were previously predicted to have. Additionally, only 2% of human DNA is translated into protein, the most well studied small molecule drug target (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Interestingly, it was found that 90% of the genome is transcribed into RNA, with the vast majority of these having non-protein-coding functions.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> Noncoding RNAs have been subsequently found to have many different functions, and analysis of the noncoding RNA between organisms follows their complexity, in contrast to differences in their respective protein-coding genes (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> Historically, protein is the predominant biomolecule considered for small molecule drug targeting. Thus, druggability, or whether a target protein is a member of a family that has been previously targeted with a small molecule, was initially assessed across the genome for protein-coding genes. This analysis revealed that 15% of proteins are considered “druggable”, i.e., within genes that have been targeted with a small molecule (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> Recent studies suggest that this notion may need reassessment, however. Perhaps the language should be changed to “undrugged”, as a wide variety of difficult to target proteins have been targeted by small molecules, including the mutant Kirsten rat sarcoma (KRAS) proto-oncogene.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. RNA as a viable drug target. (A) The conventional binary approach to small molecule drugs is their molecular recognition of proteins. Among the ∼20 000 proteins that comprise the proteome, only about 15% are in traditional “druggable” protein families. In turn, this only represents a fraction of the genome that is transcribed, leaving much of the transcriptome underexploited as therapeutic targets. (B) Noncoding genes relate to the complexity of the organism, as the relative number of coding bases remains similar, while the relative number of long noncoding RNAs (lncRNA) significantly increases, indicating that much of the intricacies of human biology and disease are represented among noncoding regions. (C) Due to the importance of coding and noncoding RNA to biology, small molecules interacting with RNA can act on the transcriptome, resulting in varied downstream effects. Importantly, validated activities for small molecules that target human RNA include: (i) changing gene expression by modulating the stability of mRNA by direct binding; (ii) affecting its noncoding RNA effectors; (iii) affecting the epitranscriptome; or (iv) influencing alternative splicing. Affecting the transcriptome with small molecule drugs can rescue disease by modulating the translation of beneficial or detrimental proteins.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A dogma for the development of small molecules against this fraction of the proteome has been centered on some tenants that perhaps need re-evaluation. First, binding sites for a bioactive ligand must be in an active site or an allosteric site to affect function. Second, the occupancy-driven view of pharmacology dictates that high target occupancy is necessary for effective target inhibition. Thus, maintained exposure to the target is the major driver of pharmacodynamic profiles. Third, the proteome is targetable at defined three-dimensional clefts that allow for high surface area recognition of ligands.</div><div class="NLM_p">Many groups have worked tirelessly to challenge these assertions. While there are no hard and fast rules in drug discovery, pragmatic solutions to important biomedical problems always win out. For example, recent developments on targeting KRAS have shown that ligands can be developed to target important proteins that do not have the traditionally desired ligand binding pockets.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> There are a variety of new ligands in the clinic that affect these previously perceived as impossible-to-drug targets. Additionally, work in the targeted protein degradation, or proteolysis targeting chimeras (PROTACs), area has challenged notions of occupancy-driven pharmacology, as well as the necessity for bioactive ligands targeting allosteric or active, i.e., functional, sites.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> For example, the ability to recruit E3-ubiquitin ligases to selectively degrade target proteins bound by selective ligands suggests that small molecules do not require occupancy to enable a pharmacological effect. Additionally, ligand occupancy or “ligandability” has been studied across the proteome and has provided an encyclopedia of information on proteome-wide ligand occupancy.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a></div><div class="NLM_p last">RNA could be advantageous as a small molecule drug target, following the paradigm of occupancy-driven inhibition of functional sites with defined three-dimensional structure employed for enzymes and receptors. RNA is also an attractive target as it is causative and/or upstream of pathological mechanisms related to disease states (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> Importantly, RNAs can be analyzed from tissue or liquid biopsies more easily than proteins because nucleic acids can be amplified and sequenced. This Perspective focuses on our current understanding of the molecular recognition of RNA by small molecules, the design of <i>bioactive</i> small molecules, and tools used to study RNA target validation, engagement, and selectivity. There are many excellent reviews on RNA chemical biology and drug discovery that cover topics outside the scope of this Perspective.<a onclick="showRef(event, 'ref24 ref25 ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27 ref28 ref29">(24−29)</a></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Assessing RNA as a Drug Target</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60431" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60431" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Since the beginning of modern medicine, RNA played an essential role as a small molecule drug target (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Streptomycin was found to target the bacterial ribosome and shortly after its discovery, became a first-in-line treatment for <i>Mycobacterium tuberculosis</i>. Prior to this breakthrough discovery, tuberculosis was a devastating disease, and those affected were housed in sanitariums until they succumbed to the infection. Other natural products isolated from species of <i>Streptomyces</i> bacteria are also tried and true antibacterials such as other aminoglycoside antibiotics like neomycin B, the active ingredient in Neosporin. Studies of these compounds and their derivatives quickly expanded to various other aminoglycosides-based antibiotics, which are still in wide use today.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> While these aminoglycosides are effective as broad-spectrum antibiotics, they tend to be promiscuous binders, leading to their limited use unless they are modified.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> However, between their discovery during WWII and now, there have been few ligands that target RNA that have progressed to the clinic (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Timeline of major developments in the RNA-targeting field. The history of drugging RNA is tied closely with molecular biology discoveries (DNA/RNA structural determination). Antibacterials that targeted RNA preceded the first investigations into antisense oligonucleotides. However, FDA approvals of antisense oligonucleotides increased upon completion of the human genome project. Recent developments, such as the use of rational design-based approaches, the initiation of clinical trials for small molecule drugs treating spinal muscular atrophy (SMA), and the first report of ribonuclease targeting chimeras (RIBOTACs), demonstrate the rapid development of small molecules targeting RNA. These advancements provide a convincing argument to turn our focus to the druggable transcriptome.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Phenotypic screens have been broadly used in the drug discovery industry beyond screening bacterial lysates to find active antibacterials. These approaches provide compounds that affect a wide variety of pathways. The challenge with phenotypic screens is identifying the engaged targets to understand the mechanism of action, compounded by the fact that target validation tools for these assays were generally designed to probe protein-centric pathways. For example, pooled CRISPR and shRNA hairpin libraries knock down ORFs (protein) or forced expression of the ORF can be used to track down a target for a small molecule.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a></div><div class="NLM_p">Although these phenotypic screens do not traditionally consider RNA as a potential target, they have indeed discovered compounds that modulate RNA. Unfortunately, these screens do not necessitate interaction with the target RNA of interest to modulate downstream RNA biology. Discovered serendipitously through phenotypic screening of small molecules interfering with the bacterial riboflavin biosynthetic pathway, Merck reported ribocil, which acts to inhibit the flavin mononucleotide (FMN) riboswitch and subsequently disrupt translation of the downstream mRNA.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> After demonstrating that riboflavin genes (<i>ribA</i> and <i>ribB</i>) are essential for <i>Escherichia coli</i> growth, ∼57 000 compounds were screened for inhibition of bacterial growth that could be reversed by riboflavin supplementation. This screen identified that ribocil inhibited cellular production of riboflavin and its metabolites. Compound-resistant <i>E. coli</i> strains were then sequenced, revealing that all mutants contained base pair mutations affecting the aptamer region of the FMN riboswitch, which controls <i>ribB</i> expression. Additional binding and reporter gene assays indicated that ribocil was a competitive inhibitor of FMN and that inhibition was ablated in FMN mutant constructs, validating this RNA as the compound’s target. Although ribocil selectively binds the FMN riboswitch and is structurally distinct from the endogenous FMN riboswitch ligand, the rapid development of bacterial resistance in various species precluded it from progression as a clinical candidate. Overall, the development of ribocil has proven that phenotypic screens can provide small molecules that bind RNA, encouraging investigators to consider RNA modulation as a mechanism of action.</div><div class="NLM_p">An early adaptor of these phenotypic studies was PTC Therapeutics. Their programs include compounds that perturb readthrough of stop codons and affect pre-mRNA splicing outcomes. Phenotypic screening identified ataluren, a clinical candidate for treatment of Duchenne’s muscular dystrophy (DMD). DMD is caused by a premature stop codon in the dystrophin mRNA, and ataluren is thought to stimulate its readthrough. Although ataluren was conditionally approved for treatment of DMD in Europe, the FDA declined to accept it as a drug based on a clinical trial in which ataluren missed its primary end point.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> While exact details of its mechanism are not known, studies revealed that ataluren directly binds and stabilizes firefly luciferase, thus allowing for its identification in the luciferase-based nonsense codon suppression assay.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> Therefore, in order to avoid false positives, target activity observed from phenotypic screens must also consider potential interference with downstream assay signals.</div><div class="NLM_p">Risdaplam is a second compound that emerged from these screens and affects pre-mRNA splicing as a treatment for spinomuscular atrophy (SMA). This autosomal recessive neurodegenerative disease is caused by deficiencies in survival motor neuron (SMN) protein, which is caused by deletion or loss in function of the <i>SMN1</i> gene.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> SMA is classified by disease severity, which scales with the reduction of SMN protein levels. The loss of functional SMN protein can be compensated by its expression through the <i>SMN2</i> gene. However, <i>SMN2</i> typically produces shorter and nonfunctional SMN protein due to a C to T single nucleotide change that causes biased exon 7 skipping, decreasing the half-life of the resulting protein. Phenotypic screens led by Roche/PTC identified compounds that increased inclusion of exon 7 in a luciferase-based cellular assay. Following efficacy studies in patient-derived cells and mouse models and RNA-Seq analysis to verify selectivity, the orally active Risdaplam was identified to convert the SMN2-encoding pre-mRNA into a longer lived version by modifying its splicing outcome.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> In a similar report, phenotypic screening at Novartis identified a compound, Branaplam, that similarly increased production of functional SMN protein through splicing modulation, which is now progressing through clinical development.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div><div class="NLM_p">Due to their identification through phenotypic screens, the mechanisms of action of these splicing modulators have not been completely defined.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Multiple studies have proposed that compounds in the Risdiplam series can achieve such exceptional molecular recognition and specificity by formation of a higher order complex comprising the exonic splicing enhancer (ESE) sequence, the 5′-splice site (5′-ss), and other regulatory splicing proteins.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Wang and others used several approaches, including chemical cross-linking and isolation by pull-down (Chem-CLIP), in order to determine direct target engagement and interactions of these compounds with RNA.<a onclick="showRef(event, 'ref42 ref44'); return false;" href="javascript:void(0);" class="ref ref42 ref44">(42,44)</a> These Chem-CLIP studies, in addition to a series of genomic and proteomic experiments of this class of compounds, revealed their direct binding to the AGGAAG motif present in exon 7 of the SMN2 pre-mRNA. Small molecule binding promotes conformational changes that result in increased binding of splicing modulators (FUBP1, KHSRP) to enhance SMN2 splicing.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> These studies suggest that compounds can directly bind and alter <i>SMN2</i> RNA structure, thereby affecting interactions with other splicing modifiers.<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a> Unfortunately, the specificity afforded by these compounds in the context of interacting with specific splicing proteins and RNA motifs in SMA may not be available for all RNA targets.</div><div class="NLM_p last">Collectively, none of these efforts purposefully targeted RNA. While they validate that small molecules can affect several different RNA functions, purposeful targeting of RNA is altogether different. Tools such as Chem-CLIP will therefore by key for exploiting RNA as a drug target, providing direct evidence of target engagement and hence compound mode of action.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Rich Structure of RNA</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72347" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72347" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Mammalian RNA encodes both genetic and structural information, exemplified by noncoding transfer (t)RNA, which was the first nucleic acid sequenced (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The discovery that the stable cloverleaf structure of tRNA interacts with the codon of messenger (m)RNA represents that RNA–RNA interactions function as a physical template for protein synthesis, setting the stage for the central dogma of biology.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Beyond their fundamental roles in protein synthesis, noncoding RNAs also play essential regulatory roles. Recent research even points to the fact that organismal complexity is directly associated with the number of noncoding RNAs, rather than the size of the genome (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> As an early example, the Steitz group proved in seminal work that noncoding U1 small nuclear RNA (U1 snRNA) recognizes 5′ splice sites in pre-mRNA,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> demonstrating that RNA structural recognition is necessary for accurate splicing and that RNA adopts intricate structures that influence genetic processes.</div><div class="NLM_p">RNA folds into elaborate structures that enable its essential functions in diverse applications in biology, encompassing regulation of gene expression, ligand sensing, catalysis, and others.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> In order to minimize its free energy, single stranded RNA forms fully base-paired as well as noncanonically paired regions (hairpins, internal loops, bulges, etc.) that determine higher order (tertiary) folding patterns (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> These secondary and tertiary RNA structural elements are highly dynamic and dependent on their environment (protein, salt concentration, small molecules, etc.), which can greatly affect function. Within structured RNAs, over 50% of residues are confined in Watson–Crick base-paired helices, and emerging studies show that subtle changes to secondary structure, or variation in Watson–Crick base pair interactions within RNA helices, significantly impact the formation of tertiary structures.<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49,50)</a> Notably, not all human RNAs contain long-range interactions between secondary structural elements, or tertiary structure (unlike RNase P RNA, (t)RNAs, and ribosomal (r)RNAs that display well-defined tertiary structures), including coaxial stacking and pseudoknots, among others. Recent investigations into RNA tertiary folding energetics have also suggested that tertiary structure can be defined through a dynamic thermodynamic ensemble of assembled secondary structural elements based on RNA sequence.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> That is, secondary structure limits the number of tertiary interactions available and hence constrains tertiary structure.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Modular RNA secondary structure motifs form three-dimensional structures. (A) Hierarchical assembly of RNA structure from sequence to secondary structure. Many of these secondary structures can form modular RNA motifs that can allow for small molecule recognition. (B) Structural schematic of microRNA processing. Primary transcripts (pri-miRNAs) are processed by the Drosha endonuclease to yield precursor hairpins (pre-miRNAs), which are exported to the cytoplasm and subsequently processed by the Dicer endonuclease to liberate a mature miRNA. One of the mature strands is then loaded into the argonaute/RNA-induced silencing complex (AGO/RISC), whereupon it acts on RNAs to modulate gene expression. Aberrant miRNA expression can be causative of disease phenotypes. (C) One rational design approach to target RNA is to understand the molecular recognition of structural elements by small molecules, those elements that are preferred by the small molecule and those that are discriminated against. Inforna compares structural elements within an RNA target to a database of these preferred interactions to afford lead small molecules. For example, binding to the pri- or pre-miRNAs at functional Drosha or Dicer sites can prevent their processing to the active, mature strand, thus allowing the rescue of disease-associated phenotypes through the inhibition of biogenesis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As understanding RNA folding is foundational to deducing its overall structure, much effort has been dedicated toward accurately predicting RNA secondary structure from sequence, including phylogenetic comparison, free energy minimization, and combinations thereof.<a onclick="showRef(event, 'ref52 ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref52 ref53 ref54">(52−54)</a> For example, entire kingdoms of life have been determined through the conservation of RNA secondary structure and phylogenetic comparison.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> In cases where limited RNA sequences are available, secondary structure prediction through free energy minimization is commonly used.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> These free energy calculations are able to provide both optimal and suboptimal structures.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> This technique is more powerful when incorporating experimental constraints from structural probing, allowing accurate determination of more complicated RNA structures (rRNA, viral RNAs, etc.).<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Use of dimethyl sulfate (DMS) chemical probing has also allowed global monitoring of RNA structure <i>in vivo</i> with single nucleotide resolution, showing that cellular mRNAs are predominantly unfolded.<a onclick="showRef(event, 'ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref59 ref60">(59,60)</a> Optimal integration of chemical probing experimental data and computational tools remains an ongoing challenge in the field of RNA structural determination, as recent quantitative tools to measure covariations in RNA alignments have disputed evidence of proposed secondary structures of various long noncoding RNAs.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Recently, advanced bioinformatics scanning window models (ScanFold) have allowed generation of high likelihood functional RNA structures.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> These resultant RNA structuromes provide evidence for hubs of structured regions within viral and mammalian RNA.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div><div class="NLM_p last">RNAs play essential roles in cellular processes; thus their dysregulated expression or misfolding can be causative of disease pathologies.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> For example, overexpression of regulatory noncoding RNAs, such as microRNAs (miRNAs) that have defined secondary structural elements but lack tertiary structure, can result in oncogenic phenotypes.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> Improper folding of RNA can also be responsible for disease pathology, as is commonly seen with short tandem repeat, or microsatellite, RNA expansions.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Due to their misfolding, several pathological mechanisms can cause a wide variety of disease mechanisms. Examples include (i) dysregulation of RNA splicing due to single nucleotide polymorphisms (SNPs), as seen in Tau neurodegenerative disorders; (ii) sequestration of essential proteins that disrupts their normal function, as observed in the sequestration of splicing factors by expanded r(CUG) or r(CCUG) repeats in myotonic dystrophy type 1 (DM1) and type 2 (DM2), respectively; and (iii) the production of toxic or nonfunctional proteins, as observed in r(G<sub>4</sub>C<sub>2</sub>) repeat expansions observed in frontotemporal dementia and amyotrophic lateral sclerosis (FTD/ALS).<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Overall, the molecular basis for many diseases is rooted in deregulated RNA function, which is intimately tied to its proper structural folds.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Basics of Targeting RNA</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75140" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75140" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As aberrantly folded RNA structures contribute to disease, it is of utmost importance to provide compounds to correct their dysfunction or to facilitate their study in healthy and disease states. Beyond simple molecular recognition, however, targeting RNA is also a function of a variety of interrelated factors (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). First of all, drugs to target RNA must be able to access the structural elements present in the target RNA, which may be in prohibitive structural interactions with other biomolecules [i.e., competition with RNA-binding proteins (RBPs)], or the RNA itself may be difficult to access (e.g., blood–brain penetrance). Additionally, selective compounds must be able to disrupt the on-target RNA, while not binding to off-target transcripts, within acceptable limits. Binding to a specific transcript is also related to the expression levels of competing RNA motifs, which is intimately connected to tissue-specific or cellular compartment-specific expression. Furthermore, compounds must be able to elicit observable biological modulations; that is, compounds must interact with RNA at a functional site (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Factors that influence bioactivity and selectivity of RNA-binding ligands. (A) 2DCS selection and HiT-StARTS analysis identifies the top binding RNA motifs and nonbinders to targapremir-210 (TGP-210). Z-Score is a calculated value that represents fitness of the RNA-small molecule interaction. (B) Fitness plot of the top 25 binding motifs are shown. One of the highest fitness RNA motif interactions with TGP-210 is the 5′ A<u class="uu">C</u>U/3′ U<u class="uu">C</u>A 3 × 3 internal loop (blue box) found in the functional Dicer site of miR-210. The 5′ A<u class="uu">G</u>C/3′ U<u class="uu">A</u>G RNA motif is a lower affinity interaction (purple box). (C) Structure of TGP-210. (D, E) The bioactivity of TGP-210 to selectively inhibit miR-210 biogenesis is a function of multiple factors, including the fitness of the RNA motif-small molecule interaction, the expression and turnover of the target RNA(s), whether or not occupancy of the target site will result in a functional interaction (i.e., occupying a Dicer/Drosha site), and the accessibility of TGP-210 to off-target sites.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A common approach to affect RNA function is sequence complementarity through antisense oligonucleotide (ASO) hybridization with a target strand. The resultant antiparallel base-paired duplex can either affect the natural folding of the target RNA, thus disrupting its interactions with other biomolecules, or recruit endogenous cellular machinery to cleave the RNA.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> These interactions, however, are dependent on the thermodynamic and kinetic energy barriers necessary for the folding/unfolding/hybridization of the native conformations of both the target RNA and ASO, which can limit their effectiveness against highly structured RNAs.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Thus, the antisense-based strategy is best applied to weakly structured RNAs. Beyond these difficulties to provide meaningful molecular interactions, oligonucleotides have been known to have inadequate <i>in vivo</i> properties (limited delivery strategies, biodistribution, and tissue penetrance) and exhibit various side effects that include eliciting of an immune response, thrombocytopenia, and hepatotoxicity in patients that have caused clinical trials to be halted.<a onclick="showRef(event, 'ref65 ref66 ref67 ref68'); return false;" href="javascript:void(0);" class="ref ref65 ref66 ref67 ref68">(65−68)</a></div><div class="NLM_p">As the previous FDA standards indicate, ASO drugs are assumed to be selective. Intriguingly, the selectivity of past FDA-approved ASOs have not been thoroughly studied transcriptome-wide or proteome-wide, despite the increasing stringency to provide mode-of-action centric, selective drugs at the molecular level.<a onclick="showRef(event, 'ref66 ref69'); return false;" href="javascript:void(0);" class="ref ref66 ref69">(66,69)</a> Indeed, investigations into the specificity and selectivity of ASOs are more nuanced than previously thought.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Naively, it might be assumed that selectivity can be enhanced by simply lengthening the oligonucleotide. Herschlag and co-workers found, however, that there is an optimal length of oligonucleotide to achieve selectivity.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> Oligonucleotides that are too long can form thermodynamically stable duplexes containing mismatches with off-targets. Indeed, the sources of hybridization of oligonucleotides with unintended targets are well-studied, and developments in analytical bioinformatics and algorithms have allowed computational screening to determine the maximal on-target transcript effects with an acceptable tolerance of off-targets.<a onclick="showRef(event, 'ref66 ref71'); return false;" href="javascript:void(0);" class="ref ref66 ref71">(66,71)</a> Still, the experimental determination of on-/off-targets through transcriptome-wide profiling of ASOs is still necessary to measure selectivity. Interestingly, there are multiple demonstrations that small molecules that target RNA can rival the selectivity of ASOs or even exhibit enhanced selectivity in certain cases.<a onclick="showRef(event, 'ref72 ref73 ref74'); return false;" href="javascript:void(0);" class="ref ref72 ref73 ref74">(72−74)</a> Thus, small molecule targeting of RNA structure may overcome the various liabilities of ASOs.</div><div class="NLM_p">An emerging strategy to selectively target RNA with small molecules is to focus on the molecular recognition of structural elements.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> One approach to rapidly define selective small molecule binders of RNA structural motifs is through a selection-based method that screens a library of RNA motifs vs a library of small molecules embedded in an agarose microarray (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C).<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> Dubbed two-dimensional combinatorial screening (2DCS), the small molecule microarray is incubated with a labeled RNA library that displays discrete structural elements in a randomized region. Weak binders and interactions with elements common to all library members (i.e., outside the randomized region) are competed off with an excess of oligonucleotide competitors (including DNA and regions that mimic the cassette displaying the randomized motif). Upon excision, amplification, and RNA-seq analysis of the bound RNA, the motifs preferred by each small molecule are defined by a statistical method named “high throughput structure–activity relationships through sequencing” (HiT-StARTS).<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> HiT-StARTS determines the statistical significance of the enrichment of a given motif in the 2DCS selection vs the starting library. The higher the statistical confidence in this enrichment, the more privileged, or fit, the interaction is. Normalization of the statistical confidence affords a fitness score for all members of an RNA library for a given small molecule. By creating a database of the results from 2DCS and HiT-StARTS, we developed a lead identification strategy for RNA targets, dubbed Inforna (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C).<a onclick="showRef(event, 'ref78 ref79'); return false;" href="javascript:void(0);" class="ref ref78 ref79">(78,79)</a> That is, Inforna searches RNA targets for structural elements preferred by small molecules, which are chemical leads. Similar to the Watson–Crick base-pairing rules that govern oligonucleotide selectivity to RNA sequence, Inforna provides the small molecule equivalent of experimentally derived RNA fold-small molecule interactions for selective recognition of RNA structure.</div><div class="NLM_p last">While structure-based targeting shows great promise as detailed below, it does have limitations. For example, not all structural elements are functional; thus, even if a small molecule binder is identified, it likely will have no biological consequence. Alternatively, a ligand may not yet have been discovered for a particular functional structure. We have recently developed a strategy dubbed ribonuclease targeting chimeras (RIBOTACs) to overcome these limitations (discussed below). Furthermore, not all structures may be accessible for ligand binding, for example, due to the presence of long-range tertiary interactions or its interaction with RBPs.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Selective Small Molecule Approaches to Purposefully Affect microRNA Biology</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86116" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86116" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Much of the transcriptome exists in defined hubs of structured RNA folds, and Inforna’s foundation is in querying these highly probable regions of structured RNA elements against experimentally identified and characterized RNA 3D fold-small molecule interactions. Built into Inforna are several strategies to rationally design compounds to purposefully target RNA and affect downstream biology. As selectivity remains the largest concern for small molecule targeting of RNA, Inforna has the ability to predict on- and off-target transcripts (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) (that is, RNAs with 3D-folds that are optimal (high fitness) or suboptimal for small molecule targeting).<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> Additionally, these RNA 3D folds can be limited to parts of RNA that are important for bioactivity, such as the Dicer and Drosha processing sites in miRNAs. By use of the highest fitness and most selective binders from Inforna, small molecules that target oncogenic miRNA precursors have been designed and have shown promising preclinical data <i>in vivo</i>.<a onclick="showRef(event, 'ref75 ref77'); return false;" href="javascript:void(0);" class="ref ref75 ref77">(75,77)</a></div><div class="NLM_p">Studies have determined that Inforna-defined RNA-small molecule interactions can inform selective targeting of miRNA precursors. These studies have formulated guidelines for targeting RNA with small molecules. The oncogenic miR-210 provides a prototypical example. Aberrant expression of miR-210 is observed in cancer cells that are in low oxygen, hypoxic environments, such as in solid breast cancer tumors.<a onclick="showRef(event, 'ref81 ref82'); return false;" href="javascript:void(0);" class="ref ref81 ref82">(81,82)</a> Inforna identified a compound, Targapremir-210 (TGP-210), that selectively inhibits miR-210 biogenesis by binding to the C/C internal loop displayed in its Dicer processing site.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> Inhibition of miR-210 disrupted the hypoxic circuit, resulting in stimulation of apoptosis in cellular and <i>in vivo</i> models.<a onclick="showRef(event, 'ref80 ref83'); return false;" href="javascript:void(0);" class="ref ref80 ref83">(80,83)</a></div><div class="NLM_p">Inhibition of miR-210 as a result of selective on-target engagement was confirmed with a Chem-CLIP probe in which TGP-210 was appended to a cross-linking (chlorambucil) module and a purification (biotin) module. Chem-CLIP studies revealed that miR-210 was the most enriched transcript among highly abundant RNAs (rRNAs, tRNAs, mRNAs, etc.) but also among hypoxia-associated miRNAs and miRNAs that contain suboptimal binding sites (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). As the less expressed miR-497 contained the same C/C internal loop as miR-210, it was also pulled down in the enriched Chem-CLIP fraction. Transcripts like miR-497, or other RNAs that contain predicted interactions with the lead compound, are termed RNA isoforms. Interestingly, TGP-210 bound to pre-miR-497 in a nonfunctional site; thus miR-497 biogenesis remained unaffected.</div><div class="NLM_p">Competitive Chem-CLIP (C-Chem-CLIP) was also applied by co-treating cells with the TGP-210 Chem-CLIP probe and the parent compound. Here, levels of miR-210 were depleted in the pulled down fraction due to competition for occupancy of target binding sites with TGP-210. These thorough studies elucidated various rules necessary for small molecules targeting RNA structure to selectively affect RNA biology. Importantly, the presence of the C/C internal loop in the functionally relevant Dicer processing site enabled TGP-210 to modulate miR-210’s downstream biology. Despite Chem-CLIP studies indicating target engagement with off-target miR-497, due to its interaction with TGP-210 in a nonfunctional site, its binding remained “silent”. Additionally, abundance of the target plays an important role, as miR-497 expression is 10-fold lower than the overexpressed miR-210 in hypoxia, potentially explaining its lower occupancy by TGP-210 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Giving confidence to target expression being necessary for compound bioactivity is the fact that apoptosis was not triggered in normoxic cells, i.e., in a regular oxygen environment, that do not overexpress miR-210. Overall, target expression levels and occupancy of a biologically relevant site are necessary variables to consider for small molecules that target structured RNAs. Furthermore, target engagement techniques, such as Chem-CLIP, can be used to verify on- and off-targets.</div><div class="NLM_p">In the case of miR-210, Inforna identified lead RNA motif-small molecule interactions that are uniquely displayed in highly expressed, functional sites. Given the high concentration of bystander RNAs (rRNA, tRNA) and RNA isoforms, there are small molecules that bind motifs found in multiple RNAs.<a onclick="showRef(event, 'ref84 ref85'); return false;" href="javascript:void(0);" class="ref ref84 ref85">(84,85)</a> As an example, Inforna identified a small molecule with overlapping affinity for the functional Drosha sites present in both pri-miR-515 and pri-miR-885.<a onclick="showRef(event, 'ref78 ref86'); return false;" href="javascript:void(0);" class="ref ref78 ref86">(78,86)</a> Selective inhibition of pri-miR-515 was further compounded by the ∼2.5-fold increased expression of miR-885 relative to miR-515 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). In order to discriminate between these two miRNAs, Inforna queried adjacent structured regions for a lead small molecule, enabling optimization through a modular approach in which multiple motifs can be targeted with the same molecule.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> While many transcripts may contain the specific RNA motif targeted by a small molecule, not all of them will be presented in functional sites, while even fewer RNAs will contain two targetable sites separated by defined, specific distances (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). By screening a library of RNA-binding modules separated by different linker lengths,<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> a dimeric compound, Targaprimir-515 (TGP-515), was developed to selectively inhibit production of miR-515 vs miR-885.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Treatment with TGP-515 in a nonmigratory breast cancer cell line resulted in the increased biosynthesis of sphingosine 1-phosphate (S1P) through the derepression of sphingosine kinase 1 (SK1), triggering a migratory phenotype. This multivalent approach has been also been applied to target other specific RNAs successfully.<a onclick="showRef(event, 'ref88 ref89'); return false;" href="javascript:void(0);" class="ref ref88 ref89">(88,89)</a></div><div class="NLM_p">The selectivity of TGP-515 to inhibit miR-515 biogenesis and modulate downstream biology was validated through several approaches. In a similar manner to the miR-210 studies, a Chem-CLIP probe was synthesized, and competition with TGP-515 indicated selective occupancy of only pri-miR-515, and not pri-miR-885. The selectivity of the dimer compound was rigorously confirmed through quantitative polymerase chain reaction (qPCR) profiling, RNA-seq, and global neo-protein studies, demonstrating on-target pathway activation and limited off-target effects in MCF-7 cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Interestingly, proteomic studies indicated that human epidermal growth factor receptor 2 (HER2) was the most upregulated protein, which is normally not highly abundant in basal MCF-7 cells. Pretreatment with TGP-515 in various HER2-negative cell lines sensitized cells to the anti-HER2 therapies, Herceptin and Kadcyla, proving the hypothesis that increased HER2 production upon TGP-515 treatment can render insensitive cells sensitive to targeted therapies. Generally, the selectivity of a small molecule for its target is one of the first considerations when designing chemical probes. In this study, potential off-targets were identified computationally through Inforna, and suboptimal selectivity was overcome through the use of multivalency. Ultimately, validation and selectivity studies were able to identify a potential precision medicine approach for difficult-to-treat cancers.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Tools to assess target engagement and selectivity of small molecules targeting RNA. Developing small molecules against RNA starts with identification of a hit, whether through Inforna (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) or screening approaches (target-based, phenotypic, fragment-based, DNA-encoded, etc.). Considering the factors from <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, the hit must then be validated and optimized, including for <i>in vitro</i> binding affinity to the RNA structural element over RNAs that do not contain the motif and other abundant RNA/DNAs. Further validation <i>in vitro</i> and in cells can be accomplished with target engagement approaches that use chemical probing methods that measure RNA enrichment (Chem-CLIP) or RNA depletion (Competitive (C)-Chem-CLIP), among others. Comprehensively evaluating cellular selectivity on a transcriptome- and proteome-wide scale is also part of the workflow to validate a small molecule RNA target. After demonstration of selective on-target effects, the compound’s functional effect must then be validated in more advanced models, including the effect of a gain or loss in expression of the target.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a similar case to miR-515, a monomeric compound inhibited the biogenesis of both miR-377 and miR-421 by binding to their common Dicer functional sites.<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a> Fortuitously, Inforna identified a lead compound that bound to an adjacent RNA motif in the pre-miR-377 hairpin that was not present in pre-miR-421. Upon optimization of the linker spacing between the RNA-binding modules that bound the two sites in pre-miR-377, the resulting dimeric compound selectively inhibited pre-miR-377, effecting angiogenesis through modulation of VEGFA protein.</div><div class="NLM_p last">Thus, Inforna has proven to be broadly applicable for the purposeful targeting of functional sites in human RNAs with selective, bioactive small molecules, while also providing comprehensive data sets defining RNA landscapes that are key for selective targeting and for identifying chemotypes with RNA-binding capacities. In addition, Inforna can be applied to develop multivalent ligands with improved selectivity and potency for RNAs that display unique targetable RNA motifs.</div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Exploiting RNA Structure for Chemical Probe Design</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78306" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78306" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Many proteins are difficult to drug directly since they lack traditionally targeted small molecule binding pockets; that is, they have intrinsically disordered domains. Therefore, affecting the coding mRNA upstream of protein production may present a more viable therapeutic option to decrease aberrant protein levels. One aspect of protein folding only recently receiving attention for its possible functional roles is intrinsically disordered regions.<a onclick="showRef(event, 'ref91 ref92'); return false;" href="javascript:void(0);" class="ref ref91 ref92">(91,92)</a> Indeed, only a small fraction of protein crystal structures recorded in the Protein Data Bank (PDB) do not possess disordered regions. These regions could represent a protein–protein interface or potential allosteric site, the folding of which is induced by binding of another protein, metabolite, or small molecule drug. Indeed, the study of molecular recognition of these protein structures by small molecules has provided well-informed paths toward drug design.<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a></div><div class="NLM_p">One clinically relevant intrinsically disordered protein is α-synuclein, which is causative of neurodegeneration in Parkinson’s disease due to its aggregation in Lewy bodies.<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> While α-synuclein protein is intrinsically disordered, the <i>SNCA</i> mRNA that encodes this pathogenic protein has a highly structured iron responsive element (IRE) in its 5′ untranslated region (UTR) that controls its translation.<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> Inforna identified a small molecule, Synucleozid, that directly binds an A bulge in the IRE, inhibiting translation in cells and providing a cytoprotective effect.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> Direct molecular recognition of the RNA by Synucleozid was validated using antisense oligonucleotide ligand binding site mapping (ASO-Bind-Map), a small molecule application of a previously developed technique.<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> ASO-Bind-Map profiles binding sites of small molecules <i>in vitro</i> and in cells as the small molecule binds and stabilizes the target RNA’s structural element, thus interfering with ASO binding and subsequent RNase H-mediated cleavage. Targeting α-synuclein upstream of its pathogenic protein demonstrates that targeting RNA elements with extensive structure can inhibit canonical translation. Importantly, affecting disease-causing intrinsically disordered proteins encoded by structured RNAs may not be limited to mRNAs with IRE and could be broadly applied to other proteins that lack binding sites for traditional “drug-like” small molecules.</div><div class="NLM_p">The biology of RNA repeat expansion disorders presents another notable target for small molecules that bind to structured regions. From a small molecule targeting standpoint, these repeat expansions form stable repeating structural units that can be exploited for targeting, especially with on-site compound synthesis using the disease-causing RNA as a catalyst for inhibitor synthesis.<a onclick="showRef(event, 'ref89 ref98'); return false;" href="javascript:void(0);" class="ref ref89 ref98">(89,98)</a> These repeats are potentially difficult to treat with ASOs that have to disrupt these stable RNA structures. Repeating transcripts are causative of >30 incurable diseases including ALS/FTD, fragile X-associated tremor ataxia syndrome (FXTAS), and DM1 and DM2.<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> Disease-causing RNA repeats can be present within coding or noncoding regions, which can affect their downstream biology. In FXTAS and DM, repeats are present in the 5′ and 3′ UTRs, respectively. ASOs sterically blocking protein interactions within the RNA repeat expansions can therefore modulate downstream biology. In the case of FXTAS, however, ASOs can also decrease downstream translation, suggesting that targeting r(CGG) repeat expansions with ASOs could worsen, rather than alleviate, disease. The repeating units, however, are amenable for small molecule targeting, as they can allow for increased specificity through the use of multivalent ligands.</div><div class="NLM_p last">In the case of myotonic dystrophy type 1, the r(CUG) repeats trigger disease by sequestering muscle-blind like protein 1 (MBNL1), which regulates the alternative splicing of various transcripts, including the insulin receptor (<i>IR</i>) and muscle-specific chloride ion channel (<i>Clnc1</i>). This gain of function by the repeats thus results in aberrant splicing products and hence DM1 disease biology. In addition to spliceopathy, other phenotypic complications arise from the expanded repeat such as aggregation of mutant transcripts with RBPs in nuclear foci, activation of stress pathways, haploinsufficiency, alteration of neighboring genes, and dysregulation of translation.<a onclick="showRef(event, 'ref100 ref101 ref102'); return false;" href="javascript:void(0);" class="ref ref100 ref101 ref102">(100−102)</a> The binding affinity of MBNL1 for r(CUG) repeats has been measured in the nM range (∼3–200 nM).<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a> Therefore, multivalent small molecules that bind the repeats with binding affinities also in the nM range were developed that can compete for binding with MBNL1, freeing it to resume its normal splicing functions. Thus, directly binding RNA to compete with RBPs presents a viable strategy to treat repeat expansion disorders. This strategy is in contrast to small molecules that form a ternary complex with the RNA and protein to treat SMA, as described above.</div></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Tools to Assess Target Engagement and Selectivity of RNA-Binding Small Molecules</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25203" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25203" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A significant amount of weight in the drug discovery process for any biomolecule is placed on <i>in vitro</i> binding affinity. While important, depending too heavily on binding studies can also be problematic, especially for dynamic RNA targets whose structure can vary <i>in vitro</i> vs <i>in situ</i>. Indeed, studies have generated ligands that interact with RNA with high affinity but display little to no bioactivity. This mismatch between optimized binding affinity and bioactivity could be due to many factors, including <i>in vitro</i> conditions significantly differing from <i>in situ</i> conditions (concentration/expression level, presence of other biomolecules, metal ions, etc.), <i>in vitro</i> models adopting different structures than the cellular RNA, the binding method itself, poor cellular permeability, localization of the small molecule to a cellular compartment where the target is not located, etc. Thus additional methods of analysis are required to validate target engagement (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>).</div><div class="NLM_p">Studying the <i>in vitro</i> selectivity of small molecules has been more or less standardized. Typically, binding affinities of the compound to the target RNA are quantified using various biophysical techniques (biolayer interferometry [BLI], surface plasmon resonance [SPR], NMR spectroscopy, microscale thermophoresis [MST], ESI-LC/MS, electrophoretic mobility shift assays [EMSA], etc.), that have been previously used for other biomolecules.<a onclick="showRef(event, 'ref104 ref105 ref106'); return false;" href="javascript:void(0);" class="ref ref104 ref105 ref106">(104−106)</a> In addition, studying RNA-small molecule interactions has also made the use of ligand or RNA-based fluorescence and fluorescence-based displacement assays.<a onclick="showRef(event, 'ref106 ref107'); return false;" href="javascript:void(0);" class="ref ref106 ref107">(106,107)</a> In these <i>in vitro</i> studies, the selectivity of ligands to the target RNA are measured relative to several controls, including point mutants of the target RNA that change its three-dimensional structure (mutating an internal loop into a base pair, for example), highly expressed RNAs (yeast tRNAs), or bulk DNA (fish sperm DNA). These initial studies provide a starting point to inform the on- and off-rate of the small molecule to bind to a specific region of the RNA in a general sense; however these assays may simplify the complexity of RNA structural dynamics in the cell, which typically exist across a population of states rather than a single explicit structure and where stability of RNA structures can be influenced by RBPs or post-transcriptional modifications.<a onclick="showRef(event, 'ref48 ref54'); return false;" href="javascript:void(0);" class="ref ref48 ref54">(48,54)</a></div><div class="NLM_p"><i>If we are indeed to develop guidelines for the selective recognition of RNA by small molecules, it is imperative to trace bioactivity to direct engagement of the target.</i> Indeed, even among well-developed protein targeting modalities that have reached clinical trials, the compound action and mechanism of action remains not well-characterized, which can result in off-target effects.<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> Until recently, target engagement methods had not been developed for RNA targets. These new methods, which employ cross-linking or cleavage, have been reviewed previously.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> In addition to target engagement, many profiling techniques are available to assess selectivity of small molecules targeting RNA, each with their own experimental advantages and liabilities.<a onclick="showRef(event, 'ref109 ref110'); return false;" href="javascript:void(0);" class="ref ref109 ref110">(109,110)</a> Below, we summarize three such methods.</div><div class="NLM_p">Quantifying expression differences of RNA transcripts has traditionally been performed through real-time reverse transcription quantitative polymerase chain reaction (RT-qPCR). Indeed, new instrumentation and technologies have allowed for increased throughput while maintaining high sensitivity and resolution, even at the single-cell level.<a onclick="showRef(event, 'ref111 ref112'); return false;" href="javascript:void(0);" class="ref ref111 ref112">(111,112)</a> The use of RT-qPCR has been commonly used to measure differential miRNA expression (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). RT-qPCR miRNA profiling revealed that small molecule hits, defined by Inforna, that target pri-miR-96 or pri-miR-515 had comparable selectivity to ASOs targeting mature miR-96-5p or mature miR-515-5p, respectively (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). Potential liabilities with measuring RNA levels with RT-qPCR include the potential biases in melting temperatures and amplification efficiency, and the labor-intensive process can be prone to user error. RT-qPCR profiling as a tool to measure the transcriptome is still useful to measure a large quantity of genes, but it is used more importantly for validation of known pathways. The method’s reliance on gene specific primers potentially precludes its use as a discovery tool.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Quantitatively evaluating the selectivity of small molecules targeting RNA. (A) Data from profiling experiments can be used to quantify compound selectivity by calculating a Gini coefficient (GC). GC analysis of kinase inhibitors showed that promiscuous compounds (staurosporine) are characterized by values close to 0, while highly selective compounds exhibit Gini coefficient values close to 1 (PD184352; 0.91), with selective compounds being defined as >0.6. (B) GC analyses can be applied to profiling data (left), such as miRNA qPCR profiling data between small molecules (TGP-515; blue) and ASOs (515-ASO; green). Applying this analysis to various small molecule ligands targeting RNA <i>structure</i> indicates that they demonstrate high selectivity for their targets. Antisense oligonucleotides targeting RNA <i>sequence</i> are also selective for their targets. Shown are the structures and GC values of TGP-515 (blue) and Vivo-Morpholino ASO targeting miR-515 (green), in addition to other small molecule/ASO GC analyses. Overall, Gini coefficients provide a metric to quantitatively define compound selectivity. When applied to RNA targeting, GC analyses demonstrate that small molecules that recognize RNA structure can rival or exceed the selectivity of ASOs designed to bind via Watson–Crick base pairing.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Whole transcriptome, RNA sequencing (RNA-Seq) is the emerging technology to measure transcript abundance in gene expression studies. As RNA-Seq is based on counting reads aligned to reference sequences and does not require specific probe sequences for expression measurements, it overcomes limitations encountered by RT-qPCR profiling. Additionally, the dynamic range of RNA-Seq is only limited by the read count and can be performed with a limited amount of transcript material. A potential limitation with RNA-Seq has been standards for data processing and methods for normalization and statistical analysis, where best practices are still being developed and must be completed on a case-by-case basis. Differential expression analysis from RNA-Seq can also be used to quantitatively define on- and off-targets, as well as to study the downstream effects of compound treatment, i.e., downstream pathway analysis. A recent example includes utilizing RNA-Seq at an early time point in breast cancer cells treated with a compound that inhibited miR-515 biogenesis. This analysis showed the compound’s limited off-target effects on the transcriptome but also the upregulation of downstream proliferative pathways.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a></div><div class="NLM_p">Studying the effects on the downstream proteins takes investigations into small molecule selectivity beyond the transcriptome and can define how the modulation of RNA biology affects the proteome. Although compounds are affecting the transcriptome, proteomics can be used as a readout to measure the change in the protein landscape as a result of affecting RNA. Global proteomics, commonly used to study compounds selectively targeting proteins, or neo-protein synthesis, measuring proteins that are up- or down-regulated after compound treatment, can reveal novel biology. Indeed, the downstream neo-protein synthesis upon inhibition of miR-515 with TGP-515 revealed compound enhancement of HER2 protein expression in a normally HER2 negative cell line. Upon this realization, the levels of HER2 were able to be tuned in order to sensitize their treatment with anti-HER2 precision medicines.</div><div class="NLM_p last">Functional assays are important to verify that the compound is functioning through the predicted RNA-centric mode of action. With TGP-515, important loss- and gain-of-function assays were performed to validate the sensitivity of HER2 negative cell lines affecting miR-515 and SK1 pathways. For example, an siRNA or chemical knockdown of SK1 ablated the migratory phenotype (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) and sensitivity to HER2 therapies. Similarly, overexpression of SK1 resulted in a similar sensitivity, indicating that derepression of SK1 by miR-515 inhibition was responsible for increased sensitivity to HER2-targeted therapies. In addition to confirmation of target engagement and selectivity through techniques such as Chem-CLIP and profiling, functional assays are necessary to ascertain that the compound is operating through the hypothesized mode of action.</div></div><div id="sec8" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Quantification of Selectivity</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88407" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88407" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Quantification of compound selectivity has previously been measured with kinase inhibitors.<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> One particular metric is called a Gini coefficient, which was originally developed as a measure of statistical dispersion of wealth distribution to measure inequality.<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a> More recently, the Gini coefficient has been used to measure selectivity of kinase inhibitors as it expresses a frequency distribution of differential inhibition among a population of kinases. In brief, Gini coefficients range from 0, indicating a nonselective compound, to 1, indicating selectivity for a single target. For example, a kinase inhibitor that lacks selectivity (inhibits 78/85 kinases tested) had a Gini coefficient of 0.15 while inhibitors considered selective had Gini coefficients ranging from 0.69 to 0.91 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).<a onclick="showRef(event, 'ref114 ref115'); return false;" href="javascript:void(0);" class="ref ref114 ref115">(114,115)</a> Applying such an analysis to RT-qPCR profiling data from miRNAs expressed in MCF-7 cells indicated similar selectivity of a small molecule that affects pri-miR-515 and a Vivo-Morpholino modified ASO that targets miR-515-5p (Gini coefficients of 0.75 and 0.72, respectively) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). In another analysis, the small molecule and ASO targeting miR-96 displayed comparable levels of selectivity with Gini coefficient values of 0.71 and 0.62, respectively (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Besides qPCR profiling data, Gini coefficient analysis can be applied to larger data sets. For example, an analysis on a transcriptome-wide microarray experiment highlighted the broad selectivity of both a small molecule targeting miR-544 (Gini coefficient = 0.73) and an ASO targeting miR-544 (Gini coefficient = 0.70).<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a> Interestingly, the monomeric miR-544 small molecule has highly drug-like properties, supporting the idea that selective small molecules targeting structured RNA motifs also occupy traditional “drug-like” chemical space. Overall, in order to be able to objectively measure the selectivity of small molecules interacting with RNA, metrics such as the Gini coefficient are useful tools to compare compound selectivity. Importantly, Gini coefficient analyses have shown that the selectivity of small molecules can rival or exceed that of ASOs.</div></div><div id="sec9" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Druglikeness and beyond the Rule of 5</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56523" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56523" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Drug discovery has long used a set of physicochemical parameters to define the chemical space that is most likely to become an orally bioavailable drug. As RNA has previously been considered an “undruggable” target, then perhaps the physicochemical profiles of orally bioactive drugs may not necessarily be the best measure for RNA-binding ligands. Indeed, not all drugs are created for the same purpose, which may inaccurately bias trends toward an average that is not necessarily true for the specific applications of lead compounds.<a onclick="showRef(event, 'ref117'); return false;" href="javascript:void(0);" class="ref ref117">(117)</a> For example, many small molecules differ in clearance rates and exposure to the tissue type of interest; i.e., blood–brain barrier penetrant compounds are different from systemic broad spectrum antibiotics. Therefore, the commonly used parameters that make drugs particularly attractive for oral delivery in protein targeting applications may not be appropriate for targeting RNA. In fact, there is an increasing trend of compounds that exist in a space that is extended from the Rule of 5 (eRo5) or even beyond the Rule of 5 (bRo5).<a onclick="showRef(event, 'ref118'); return false;" href="javascript:void(0);" class="ref ref118">(118)</a> Indeed, as the methods to analyze the selectivity of compounds has progressively become more rigorous (full proteomics and transcriptomics profiles) and with the greater rigor for compounds with defined modes of action, the landscape of what is considered a “drug” that selectively hits a defined drug target has significantly changed.<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> Traditional ASOs, and other biologics such as antibody-drug conjugates, have also seen a rapid rise in approvals as FDA-approved drugs, suggesting that compounds that do not fit into traditional medicinal chemistry parameters are also viable as effective therapeutics.<a onclick="showRef(event, 'ref119'); return false;" href="javascript:void(0);" class="ref ref119">(119)</a></div><div class="NLM_p">In pursuit of more selective and potent compounds, a recent trend has become known as “compound bloat”. That is, drugs and potential drug candidates have been increasing in molecular weight and changing the classic idea of drug-like compounds. As mentioned by others, this “compound bloat” has been in response to decreasing off-target effects that were not previously measurable with technologies available at the time the drug was developed (CRISPR screens, RNA-Seq, proteomics).<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> Indeed, targeting multiple “hot sites” of the same target or using binding sites between two different surfaces (i.e., protein–protein interactions (PPI)) has now been implemented.<a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a> However, these innovative techniques have required increased molecular weight and polar surface area, along with an increased number of hydrogen bond donors and acceptors.<a onclick="showRef(event, 'ref119'); return false;" href="javascript:void(0);" class="ref ref119">(119)</a></div><div class="NLM_p">While many drug-like compounds are heavily tested <i>in vitro</i> before <i>in cellulis</i> evaluation, binding contacts are not the only factor that influences bioactivity when concerned with drug-like properties. Furthermore, bioactivity does not necessarily indicate on-target effects, as without proper target engagement and validation, the compound may work in a different mechanism than hypothesized. Indeed, focusing on optimal physicochemical properties is important toward discovering privileged RNA space, but more innovative and novel approaches to the problem of targeting RNA with small molecules may result in compounds that look more “undrug-like” than traditional small molecule therapeutics. More provocatively, chemical matter that affects RNA space most effectively may differ significantly from traditional guidelines used to identify, design, and optimize ligands that affect other biomolecules.</div><div class="NLM_p">Insight into the physicochemical properties of RNA-binding small molecules can begin to be gleaned from interactions housed in Inforna<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> and a recently created catalog named R-BIND.<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a> Analysis of the compounds in both databases revealed highly similar chemical properties to FDA approved drugs in DrugBank (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A).<a onclick="showRef(event, 'ref122'); return false;" href="javascript:void(0);" class="ref ref122">(122)</a> However, differences are indeed observed. For example, RNA binders on average have more aromatic rings (3.9 ± 1.9 vs 1.4 ± 1.4 in approved drugs) and more H bond donor (5.2 ± 4.5 vs 2.4 ± 0.5.0) and acceptor groups (8.5 ± 6.2 vs 5.1 ± 7.4).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Physiochemical properties of RNA binders and common RNA-binding scaffolds. (A) Physiochemical properties of cataloged RNA binders contained within Inforna and R-BIND show convergence and correlate with properties of FDA approved drugs. FDA approved drugs were taken from DrugBank.<a onclick="showRef(event, 'ref122'); return false;" href="javascript:void(0);" class="ref ref122">(122)</a> (B) Properties that are enriched within RNA binders include greater positive charge at pH 7.4, number of H-bond donor and acceptor counts, and total polar surface area (TPSA). RNA binders also exhibit fewer chiral centers, aliphatic atoms, and rotational bonds compared to nonbinders. (C) Scaffolds contained within Inforna and R-BIND that exhibit RNA binding. These include oligosaccharides, benzimidazoles, purines, naphthalenes, quinolines, quinazolines, aminopyrimidines, and quinidine thiazoles. Interestingly, while most data generally point to planar molecules as RNA binders, some sterically rich compounds have been found to bind RNA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In one large screen to identify novel compounds that bind RNA, a comparison of binding and nonbinding molecules showed that binders had on average 1.5-fold higher cLogP values, 1.3-fold more ring systems, 1.7-fold fewer aliphatic atoms, 1.5-fold more H-bond donors, 2.4-fold more H-bond acceptors, 10-fold fewer chiral centers, and slightly greater positive charge (δ<sup>+</sup> of 0.4 vs 0.3 at pH 7.4) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B).<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a> These data indicate a propensity for RNA binders to be structurally rigid and planar, likely driving binding of the RNA via π–π stacking and hydrogen bonding interactions with the nucleobases and/or the phosphate backbone.<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a></div><div class="NLM_p last">Scaffolds commonly associated with RNA binding that have yielded bioactive ligands include benzimidazoles, <i>bis</i>-benzimidazoles and phenylbenzimidazoles, aminopyrimidines, and quinazolines (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). Other scaffolds identified from Inforna and R-BIND include oligosaccharides, naphthalenes, and purines. While the above physiochemical data and previous works suggest a preference for planar molecules, sterically rich compounds have also been identified such as oligosaccharides and those found in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C including spirocycles and fused ring spirocycles.</div></div><div id="sec10" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Emerging Modalities for Affecting RNA Biology</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62686" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62686" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As RNA presents a novel and challenging target, new approaches are needed to affect its downstream biology through its direct targeting. Recent work has demonstrated the use of other modalities to enable the targeted degradation of RNA, such as bleomycin conjugates (reviewed more in-depth previously),<a onclick="showRef(event, 'ref69 ref99 ref119 ref123'); return false;" href="javascript:void(0);" class="ref ref69 ref99 ref119 ref123">(69,99,119,123)</a> outside the traditional “drug-like” physicochemical properties.<a onclick="showRef(event, 'ref124'); return false;" href="javascript:void(0);" class="ref ref124">(124)</a> Many compounds (described above) have rescued disease-associated phenotypes through a binding mode of action; however, it may be advantageous to expand the mode of action to cleave the RNA target, whether directly or by recruiting endogenous nucleases (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). These “RNA degraders” can potentially ameliorate disease pathologies in a more potent and selective manner than simple binding compounds.<a onclick="showRef(event, 'ref72 ref89'); return false;" href="javascript:void(0);" class="ref ref72 ref89">(72,89)</a></div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Properties of bioactive therapeutic modalities. Current and emerging strategies to affect downstream biology include antibodies (rituximab, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KAQ">4KAQ</a>), ASOs (Nusinersen), small molecules (TGP-210), and targeted degradation approaches (TGP-210 RIBOTAC). Each targets a unique space, but RIBOTACs can affect bioactivity of RNA without binding to a functional site. RIBOTACs also degrade their targets in a catalytic and substoichiometric fashion, thus allowing greater potency. Small-molecule-based modalities are advantageous as their physicochemical properties can be potentially medicinally optimized. ASO, small molecule, and targeted degradation models were made using the Online SMILES Generator (National Cancer Institute).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">RNA degraders have been applied extensively to RNA repeat expansion disorders, as many of these highly stable structures can function as scaffolds to sequester proteins or to trigger repeat associated non-ATG (RAN) translation. Recently, Angelbello et al. demonstrated that RNA-degraders rescue DM1 disease pathology through intraperitoneal delivery of Cugamycin (a dimeric r(CUG)-binding compound appended to bleomycin), which selectively cleaves the repeat expansion in a preclinical mouse model.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> In addition to its therapeutic potential, these degrader molecules can also function as effective chemical probes, as described through the development of small molecule nucleic acid profiling by cleavage applied to RNA (RiboSNAP). RiboSNAP uses the cleavage activity of bleomycin to deplete the RNA target’s levels in patient-derived cells, thus demonstrating target engagement. RiboSNAP has also been applied with other noncoding RNAs to map small molecule binding sites,<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> thus establishing RNA degraders and RiboSNAP as a new potential class of targeted therapeutic and chemical probing technique.</div><div class="NLM_p">Recently, our lab has designed small molecules that can recruit a nuclease to a specific transcript, triggering its degradation. RIBOTACs, akin to PROTACs,<a onclick="showRef(event, 'ref125'); return false;" href="javascript:void(0);" class="ref ref125">(125)</a> recruit nucleases for the targeted degradation of structured RNA sites (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). Specifically, RIBOTACs locally recruit endogenous latent ribonuclease (RNase L) to a specific transcript, allowing for assembly of dimeric, active RNase L to selectively degrade the RNA target. This strategy is akin to a small molecule form of CRISPR, without the need for transfection of foreign guide RNA transcripts or proteins. In contrast, RIBOTACs exploit a cell’s endogenous machinery for nucleic acid disposal by bringing together all the “players” in a ternary complex (target RNA:RIBOTAC:RNase L). By locally eliminating aberrantly functioning RNA, rather than simply binding and inhibiting it, RIBOTACs offer specific advantages:</div><div id="sec10_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Occupancy vs Event Driven Processes</h3><div class="NLM_p">Current pharmacological paradigms require small molecules to modulate downstream function by occupying enzyme active sites or blocking receptors, which requires high concentrations of drug at the correct position. Not only can maintaining high concentrations of drug lead to off-target effects, this occupancy-driven model requires RNA with accessible, “druggable” active sites, which may be difficult to target selectively. By eliminating the aberrant RNA, rather than binding and inhibiting them, RIBOTACs circumvent the occupancy-driven model and instead are event-driven: binding enables activation of ribonucleases to initiate degradation (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a> and <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>).</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Ribonuclease targeting chimeras (RIBOTACs) as heterobifunctional degraders of RNA. (A) Taking cues from PROTACs and RNase H-based antisense oligonucleotide approaches, RIBOTACs are heterobifunctional compounds that recruit endogenous nucleases to degrade a targeted transcript. RIBOTACs can potentially increase potency of small molecules as they can catalytically and substoichiometrically degrade an RNA target. These RIBOTACs simply need to bind the target (not necessarily at a functional site) and use endogenous ribonuclease pathways to remove the RNA via targeted degradation, which also rids the RNA of any potential scaffolding functions with RBPs. Formation of the ternary complex may also increase selectivity as only meaningful interactions between the RNA:RIBOTAC:RNase L will result in cleavage. While this approach is potentially broadly applicable, development and optimization of both RNA binders and RNase-recruiting modules remain time-consuming, especially as there are a limited number of known RNase activators. Additionally, RIBOTACs that function through RNase L can have less pronounced effects on nuclear RNA, as RNase L is primarily cytoplasmic. (B) Advantages provided by the RIBOTAC approach. (C) Demonstration of increased selectivity of different RNA-binding modules, as indicated by GC analysis. The monomeric RNA-binding module that binds a single functional site on an RNA is less selective than the multivalent ligand targeting the same RNA. Adding an RNase L recruitment module to convert the dimeric compound into a RIBOTAC allows for increased selectivity, potentially due to the requirement of effective ternary complex formation between the RNA, RIBOTAC, and RNase L.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec10_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Increased Potency and Catalytic Activity</h3><div class="NLM_p last">Low levels of occupancy of RIBOTACs may be able to maintain a rate of RNA degradation that can provide the desired pharmacological effect. Previous small molecule catalytic activity was seen with Ru(bypy) photocatalysts as singlet oxygen generators,<a onclick="showRef(event, 'ref126'); return false;" href="javascript:void(0);" class="ref ref126">(126)</a> and RIBOTACs have demonstrated a similar level of substoichiometric degradation of their targets.<a onclick="showRef(event, 'ref127'); return false;" href="javascript:void(0);" class="ref ref127">(127)</a> This catalytic effect also suggests that RIBOTAC concentrations required for degradation may be much lower than those required for levels of inhibition by simple binding. As a catalytic degrader, RIBOTAC action also allows for greater RNA depletion over time; that is, low exposures can lead to meaningful knockdown, reducing potential off-target toxicity.</div></div><div id="sec10_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Avoiding Target Accumulation</h3><div class="NLM_p last">Drug binding can potentially stabilize RNA, thereby increasing its half-life. Once drug exposure drops below a certain inhibitory level, the disease-causing RNA will then persist. Degradation avoids this dilemma entirely, as it eliminates the target RNA completely. Additionally, degradation rather than binding also rids the targeted RNA of their potential disease-causing interactions with other biomolecules, such as RBPs like MBNL1 with r(CUG) expanded repeats. Accumulation of mutations in the drug target can also occur under selective pressure, which may negatively affect drug binding and occupancy, thereby leading to a decrease in efficacy, as observed with ribocil. Interestingly, RIBOTACs do not necessarily have to target an active site to form a ternary complex and degrade the target, as even an allosteric binder can be used. Degradation of the RNA also leads to a lower risk of the target complex evolving a resistant form, but upon formation of resistance, the modularity of RIBOTACs can potentially lead to the simple solution of using an alternative target binder. As ribonucleases are highly conserved and ubiquitous among cells, resistance to RIBOTAC-induced degradation may be much lower than observed for simple binding compounds.</div></div><div id="sec10_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Gain of Specificity</h3><div class="NLM_p">Achieving good selectivity over off-targets still presents a significant challenge among small molecule inhibitors, especially in RNA targets that contain highly homologous binding sites. RIBOTACs achieve targeted degradation in two steps, first from binding of the RIBOTAC to the target and, second, upon nuclease recruitment to the target RNA (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). While the first step is mainly controlled by the affinity of the RIBOTAC to form binary complexes with its target RNA, the selectivity of the latter step can be appropriately adjusted for the RNA of interest via linker length or even by modulating the RNase recruiter. Additionally, as described above, compounds typically require binding to active sites within an RNA target to be effective; however RIBOTACs can be designed to bind structural motifs unique to a specific target RNA, thus enabling greater selectivity. Therefore, selectivity can be engineered into each component of the RIBOTAC to modulate the intrinsic binding affinity of the lone RNA-binding module to off-target RNAs. One important factor to consider for future RIBOTACs may be the role of binding kinetics (<i>k</i><sub>on</sub>/<i>k</i><sub>off</sub> rates). Indeed, with the advent of newer biophysical methods to analyze kinetics, recent studies have shown greater appreciation for on-/off-rates for drugs binding to their targets.<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> These properties are further complicated upon accounting for the optimal presentation/rate of recruitment of the RNase recruitment module and the complexities of ternary complex formation with the RNA target and RNA-binding module in the cell, which are required for RIBOTACs to propagate cleavage.</div><div class="NLM_p last">Collectively, our inaugural studies suggest (i) RIBOTACs could be a general strategy broadly applicable to imbue RNA-targeted small molecules with cleavage capability, extending mode of action beyond simple binding and (ii) selectivity, as compared to the RNA-binding compound alone, can be enhanced by conjugation of a degradation module, as also observed with nonspecific kinase inhibitors converted to a PROTAC.<a onclick="showRef(event, 'ref129'); return false;" href="javascript:void(0);" class="ref ref129">(129)</a> Additionally, RIBOTACs can use guiding principles from studies into ternary complex formation and the role of kinetics for optimal activity in the PROTACs space when considering RNA-targeted degradation.<a onclick="showRef(event, 'ref130 ref131'); return false;" href="javascript:void(0);" class="ref ref130 ref131">(130,131)</a> Bearing this in mind, future studies to apply the RIBOTAC technology for the druggable transcriptome may require (i) medicinal chemistry of RNA-binding small molecule, linker, and RNase L recruitment modules, especially regarding on-/off-rates; (ii) studies toward the recruitment of other ribonuclease modalities, preferably with favorable physicochemical properties; (iii) new linker chemistries to optimize presentation of the bifunctional ends; and (iv) application of RIBOTACs to <i>in vivo</i> models. Indeed, a recent study has shown that a RIBOTAC using a heterocyclic recruiter of RNase L was able to substoichiometrically degrade pre-miR-21 to impede metastasis of breast cancer to the lung in a mouse model.<a onclick="showRef(event, 'ref132'); return false;" href="javascript:void(0);" class="ref ref132">(132)</a> Interestingly, selectivity and Gini coefficient analysis indicated that the RIBOTAC targeting miR-21 was more selective than the parent binding compounds, rivaling or exceeding that of protein-targeted drugs. Further downstream transcriptome and proteome-wide studies demonstrated broad inhibition of oncogenic pathways, as expected upon miR-21 depletion.<a onclick="showRef(event, 'ref132'); return false;" href="javascript:void(0);" class="ref ref132">(132)</a></div></div></div><div id="sec11" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58026" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58026" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As controlled RNA expression manages nearly every function in the human body, modulation of these phenomena represents an integral goal in biomedical research. As a result of chemical probes and small molecule drugs being developed for a tiny fraction of the proteome, a call to develop chemical probes (small molecules, degraders, biologics, etc.) for the entirety of the human proteome was recently announced (Target 2035).<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> Indeed, this ambitious aspiration will require many groups willing to work on proteins in the “dark proteome” and the advancement and optimization of new and current technologies (DNA-encoded libraries, massively parallel high throughput screening, functional assays for understudied proteins, etc.).<a onclick="showRef(event, 'ref134 ref135'); return false;" href="javascript:void(0);" class="ref ref134 ref135">(134,135)</a> Another approach to increase coverage of chemical probes would be to study compounds that modulate protein levels by affecting them at the RNA level. Not only could compounds that target RNA effectively inhibit protein expression by binding to their precursor mRNA transcript, but they also could potentially activate proteins by modifying their associated regulatory RNAs (miRNAs, lncRNAs, etc.), thus providing a bilateral approach to affect protein biology.</div><div class="NLM_p">While effective control of RNA expression using antisense technologies has been demonstrated, these modalities are still limited in their own unique ways, leading researchers to examine if the same level of control and selectivity can be achieved using small molecules. Highlights from this Perspective have illustrated that the selective molecular recognition of RNA with small molecules is possible through the interaction of ligands with structured RNA motif regions. Bioactivity of small molecule ligands requires more than just optimizing for high affinity <i>in vitro</i> binding interactions. Furthermore, the analysis of transcriptome- and downstream proteome-wide selectivity can determine how structural recognition can be exploited for specific RNA modulation. Importantly, target engagement and validation studies are essential to verify RNA-centric modes of action and to rescue disease-associated phenotypes through an expanding toolbox of techniques. Finally, RNA is indeed druggable, although the compounds used may not look like traditional drugs, such as heterobifunctional chimeric compounds that have demonstrated efficacy in selective RNA degradation.</div><div class="NLM_p last">As opportunities arise and challenges are met for RNA-targeting small molecules, it is an exciting time to propose a druggable transcriptome project in order to provide chemical probes for functional RNAs on a transcriptome-wide scale. Developing technologies, such as RIBOTACs, and currently existing methodologies, such as Inforna, can be used in conjunction to study and optimize small molecules targeting all functionally relevant RNAs broadly. As advancements and innovations in the design and discovery small molecules interacting with RNA continue, the use of these techniques can bring about a new paradigm in chemical biology and potentially launch research into new therapeutic modalities and viable medicines.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01927" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56713" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56713" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew D. Disney</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, The
Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8486-1796" title="Orcid link">http://orcid.org/0000-0001-8486-1796</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#583c312b363d21182b3b2a3128282b763d3c2d"><span class="__cf_email__" data-cfemail="67030e1409021e271404150e17171449020312">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew G. Costales</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, The
Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jessica L. Childs-Disney</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, The
Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hafeez S. Haniff</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, The
Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): M.D.D. is a founder of Expansion Therapeutics.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15213" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15213" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Matthew G. Costales</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=BIO-d7e1378-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Matthew G. Costales</b> completed his B.S. in Chemistry at the University of Maryland, College Park. He spent the following years as an Oak Ridge Institute for Science and Education Fellow in the Immunobiology Group in the Center for Food Safety and Applied Nutrition at the Food and Drug Administration. He then received his Ph.D. in Chemical Biology under the supervision of Prof. Matthew D. Disney at the Scripps Research Institute in Florida for his dissertation research on strategies to selectively target RNA with small molecules. Currently, he is working in the Neuro Discovery group at Dicerna Pharmaceuticals.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Jessica L. Childs-Disney</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=BIO-d7e1383-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Jessica L. Childs-Disney</b> attended Messiah College where she received her B.S. in Chemistry. She then completed her Ph.D. in Biophysical Chemistry, studying how to induce RNA misfolding under the tutelage of Prof. Douglas H. Turner, and postdoctoral studies at the Swiss Federal Institute of Technology (ETH) Zurich. After spending 4 years as an Assistant Professor at Canisius College, she moved to The Scripps Research Institute in Florida where she is a senior staff scientist.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Hafeez S. Haniff</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=BIO-d7e1388-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Hafeez S. Haniff</b> finished his B.S. in Chemistry at the University of Central Florida. He is currently completing his Ph.D. in the lab of Prof. Matthew D. Disney at the Scripps Research Institute in Florida. His dissertation research is focused on the development and application of RNA-focused small molecules libraries to modulate the function of RNA-based diseases for regenerative medicine.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Matthew D. Disney</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=BIO-d7e1393-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Matthew D. Disney</b> received his B.S. in Chemistry from the University of Maryland, College Park. He then completed his M.S. and Ph.D. in Biophysical Chemistry with Prof. Douglas H. Turner. He completed postdoctoral studies in the lab of Prof. Peter H. Seeberger at the Massachusetts Institute of Technology and the Swiss Federal Institute of Technology (ETH) Zurich. He started his independent research career and eventually joined the faculty in the bicoastal Department of Chemistry at The Scripps Research Institute in Florida, where he is Full Professor. His group focuses on developing small molecules to selectively drug human RNAs. His research has also led to the founding of Expansion Therapeutics.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58658" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58658" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the members of our laboratory that have helped advance the science of small molecules interacting with RNA, especially human RNAs, over the past 15 years. We thank the funding agencies (National Institutes of Health Grants R01 GM97455, R01 CA249180, P01 NS099114, and DP1 NS096898 to M.D.D.; the Department of Defense Peer Reviewed Medical Research Program Grant W81XWH-18-0718 to M.D.D.; ACS Medicinal Chemistry Predoctoral Fellowship to M.G.C.) that have supported our work in this area.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">2′-5′ A</td><td class="NLM_def"><p class="first last">2′-5′ linked oligoadenylate</p></td></tr><tr><td class="NLM_term">2DCS</td><td class="NLM_def"><p class="first last">two-dimensional combinatorial screening</p></td></tr><tr><td class="NLM_term">AGO/RISC</td><td class="NLM_def"><p class="first last">argonaute/RNA-induced silencing complex</p></td></tr><tr><td class="NLM_term">ASO</td><td class="NLM_def"><p class="first last">antisense oligonucleotide</p></td></tr><tr><td class="NLM_term">ASO-Bind-Map</td><td class="NLM_def"><p class="first last">antisense oligonucleotide ligand binding site mapping</p></td></tr><tr><td class="NLM_term">BLI</td><td class="NLM_def"><p class="first last">biolayer interferometry</p></td></tr><tr><td class="NLM_term">bRo5</td><td class="NLM_def"><p class="first last">beyond rule of 5</p></td></tr><tr><td class="NLM_term">C-Chem-CLIP</td><td class="NLM_def"><p class="first last">competitive cross-linking and isolation by pulldown</p></td></tr><tr><td class="NLM_term">Chem-CLIP</td><td class="NLM_def"><p class="first last">chemical cross-linking and isolation by pulldown</p></td></tr><tr><td class="NLM_term">CRISPR</td><td class="NLM_def"><p class="first last">clustered regularly interspaced short palindromic repeats</p></td></tr><tr><td class="NLM_term">DM1/DM2</td><td class="NLM_def"><p class="first last">myotonic dystrophy type I/II</p></td></tr><tr><td class="NLM_term">DMD</td><td class="NLM_def"><p class="first last">Duchenne’s muscular dystrophy</p></td></tr><tr><td class="NLM_term">EMSA</td><td class="NLM_def"><p class="first last">electrophoretic mobility shift assay</p></td></tr><tr><td class="NLM_term">ENCODE</td><td class="NLM_def"><p class="first last">Encyclopedia of DNA Elements</p></td></tr><tr><td class="NLM_term">eRo5</td><td class="NLM_def"><p class="first last">extended rule of 5</p></td></tr><tr><td class="NLM_term">ESE</td><td class="NLM_def"><p class="first last">exonic splicing enhancer</p></td></tr><tr><td class="NLM_term">ESI-LC/MS</td><td class="NLM_def"><p class="first last">electrospray ionization liquid chromatography/mass spectrometry</p></td></tr><tr><td class="NLM_term">FMN</td><td class="NLM_def"><p class="first last">flavin mononucleotide</p></td></tr><tr><td class="NLM_term">FTD/ALS</td><td class="NLM_def"><p class="first last">frontotemporal dementia/amyotrophic lateral sclerosis</p></td></tr><tr><td class="NLM_term">FXTAS</td><td class="NLM_def"><p class="first last">fragile X-associated tremor ataxia syndrome</p></td></tr><tr><td class="NLM_term">GC</td><td class="NLM_def"><p class="first last">Gini coefficient</p></td></tr><tr><td class="NLM_term">HER2</td><td class="NLM_def"><p class="first last">human epidermal growth factor receptor 2</p></td></tr><tr><td class="NLM_term">HiT-StARTS</td><td class="NLM_def"><p class="first last">high throughput structure–activity relationships through sequencing</p></td></tr><tr><td class="NLM_term">IDP</td><td class="NLM_def"><p class="first last">intrinsically disordered protein</p></td></tr><tr><td class="NLM_term">IRE</td><td class="NLM_def"><p class="first last">iron responsive element</p></td></tr><tr><td class="NLM_term">IRP</td><td class="NLM_def"><p class="first last">iron responsive protein</p></td></tr><tr><td class="NLM_term">KRAS</td><td class="NLM_def"><p class="first last">Kirsten rat sarcoma proto-oncogene</p></td></tr><tr><td class="NLM_term">lncRNA</td><td class="NLM_def"><p class="first last">long noncoding RNA</p></td></tr><tr><td class="NLM_term">MBNL1</td><td class="NLM_def"><p class="first last">muscle-blind like protein 1</p></td></tr><tr><td class="NLM_term">miR</td><td class="NLM_def"><p class="first last">microRNA</p></td></tr><tr><td class="NLM_term">MST</td><td class="NLM_def"><p class="first last">microscale thermophoresis</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interactions</p></td></tr><tr><td class="NLM_term">pre-miR</td><td class="NLM_def"><p class="first last">precursor microRNA hairpin</p></td></tr><tr><td class="NLM_term">pre-mRNA</td><td class="NLM_def"><p class="first last">precursor messenger RNA</p></td></tr><tr><td class="NLM_term">pri-miR</td><td class="NLM_def"><p class="first last">primary microRNA transcript</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">RiboSNAP</td><td class="NLM_def"><p class="first last">small-molecule nucleic acid profiling by cleavage applied to RNA</p></td></tr><tr><td class="NLM_term">RIBOTAC</td><td class="NLM_def"><p class="first last">ribonuclease targeting chimera</p></td></tr><tr><td class="NLM_term">RNase</td><td class="NLM_def"><p class="first last">ribonuclease</p></td></tr><tr><td class="NLM_term">RNA-Seq</td><td class="NLM_def"><p class="first last">RNA sequencing</p></td></tr><tr><td class="NLM_term">RT-qPCR</td><td class="NLM_def"><p class="first last">real time reverse transcription quantitative polymerase chain reaction</p></td></tr><tr><td class="NLM_term">S1P</td><td class="NLM_def"><p class="first last">sphingosine 1-phosphate</p></td></tr><tr><td class="NLM_term">shRNA</td><td class="NLM_def"><p class="first last">short hairpin RNA</p></td></tr><tr><td class="NLM_term">siRNA</td><td class="NLM_def"><p class="first last">short interfering RNA</p></td></tr><tr><td class="NLM_term">SK1</td><td class="NLM_def"><p class="first last">sphingosine kinase 1</p></td></tr><tr><td class="NLM_term">SMA</td><td class="NLM_def"><p class="first last">spinomuscular atrophy</p></td></tr><tr><td class="NLM_term">SMN</td><td class="NLM_def"><p class="first last">survival motor neuron protein</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">TGP-210</td><td class="NLM_def"><p class="first last">targapremir-210</p></td></tr><tr><td class="NLM_term">TGP-515</td><td class="NLM_def"><p class="first last">targaprimir-515</p></td></tr><tr><td class="NLM_term">U1 snRNA</td><td class="NLM_def"><p class="first last">U1 small nuclear RNA</p></td></tr><tr><td class="NLM_term">UTR</td><td class="NLM_def"><p class="first last">untranslated region</p></td></tr><tr><td class="NLM_term">VEGFA</td><td class="NLM_def"><p class="first last">vascular endothelial growth factor A</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21071" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21071" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 135 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linton, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nusbaum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zody, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FitzHugh, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gage, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heaford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehoczky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeVine, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEwan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKernan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meldrim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesirov, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raymond, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sougnez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stange-Thomann, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stojanovic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulston, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ainscough, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bentley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deadman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deloukas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunham, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunham, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durbin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grafham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphray, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullikin, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mungall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plumb, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shownkeen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterston, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillier, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPherson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mardis, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulton, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinwalla, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pepin, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gish, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chissoe, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendl, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delehaunty, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miner, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delehaunty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minx, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clifton, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branscomb, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Predki, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenning, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slezak, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doggett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uberbacher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frazier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbs, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muzny, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherer, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouck, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sodergren, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worley, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rives, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorrell, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzker, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucherlapati, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstock, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyoda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawagoe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissenbach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heilig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saurin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artiguenave, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brottier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruls, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelletier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wincker, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette-Stamm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubenfield, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstock, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubois, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platzer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyakatura, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taudien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rump, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federspiel, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abola, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proctor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmutz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimwood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaul, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raymond, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimizu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawasaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minoshima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasiou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehrach, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhardt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCombie, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Bastide, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedhia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blocker, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornischer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordsiek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aravind, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bateman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batzoglou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bork, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burge, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerutti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Church, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clamp, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copley, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doerks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddy, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichler, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furey, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galagan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashizaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haussler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermjakob, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hokamp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaspryzk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitts, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koonin, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korf, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLysaght, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikkelsen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulder, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollara, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponting, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupka, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szustakowki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thierry-Mieg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thierry-Mieg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, Y. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guyer, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felsenfeld, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetterstrand, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patrinos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jong, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catanese, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osoegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shizuya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szustakowki, J.</span></span> <span> </span><span class="NLM_article-title">Initial sequencing and analysis of the human genome</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>409</i></span>,  <span class="NLM_fpage">860</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1038/35057062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2F35057062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=11237011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhsFCjtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=409&publication_year=2001&pages=860-921&author=E.+S.+Landerauthor=L.+M.+Lintonauthor=B.+Birrenauthor=C.+Nusbaumauthor=M.+C.+Zodyauthor=J.+Baldwinauthor=K.+Devonauthor=K.+Dewarauthor=M.+Doyleauthor=W.+FitzHughauthor=R.+Funkeauthor=D.+Gageauthor=K.+Harrisauthor=A.+Heafordauthor=J.+Howlandauthor=L.+Kannauthor=J.+Lehoczkyauthor=R.+LeVineauthor=P.+McEwanauthor=K.+McKernanauthor=J.+Meldrimauthor=J.+P.+Mesirovauthor=C.+Mirandaauthor=W.+Morrisauthor=J.+Naylorauthor=C.+Raymondauthor=M.+Rosettiauthor=R.+Santosauthor=A.+Sheridanauthor=C.+Sougnezauthor=Y.+Stange-Thomannauthor=N.+Stojanovicauthor=A.+Subramanianauthor=D.+Wymanauthor=J.+Rogersauthor=J.+Sulstonauthor=R.+Ainscoughauthor=S.+Beckauthor=D.+Bentleyauthor=J.+Burtonauthor=C.+Cleeauthor=N.+Carterauthor=A.+Coulsonauthor=R.+Deadmanauthor=P.+Deloukasauthor=A.+Dunhamauthor=I.+Dunhamauthor=R.+Durbinauthor=L.+Frenchauthor=D.+Grafhamauthor=S.+Gregoryauthor=T.+Hubbardauthor=S.+Humphrayauthor=A.+Huntauthor=M.+Jonesauthor=C.+Lloydauthor=A.+McMurrayauthor=L.+Matthewsauthor=S.+Mercerauthor=S.+Milneauthor=J.+C.+Mullikinauthor=A.+Mungallauthor=R.+Plumbauthor=M.+Rossauthor=R.+Shownkeenauthor=S.+Simsauthor=R.+H.+Waterstonauthor=R.+K.+Wilsonauthor=L.+W.+Hillierauthor=J.+D.+McPhersonauthor=M.+A.+Marraauthor=E.+R.+Mardisauthor=L.+A.+Fultonauthor=A.+T.+Chinwallaauthor=K.+H.+Pepinauthor=W.+R.+Gishauthor=S.+L.+Chissoeauthor=M.+C.+Wendlauthor=K.+D.+Delehauntyauthor=T.+L.+Minerauthor=A.+Delehauntyauthor=J.+B.+Kramerauthor=L.+L.+Cookauthor=R.+S.+Fultonauthor=D.+L.+Johnsonauthor=P.+J.+Minxauthor=S.+W.+Cliftonauthor=T.+Hawkinsauthor=E.+Branscombauthor=P.+Predkiauthor=P.+Richardsonauthor=S.+Wenningauthor=T.+Slezakauthor=N.+Doggettauthor=J.+F.+Chengauthor=A.+Olsenauthor=S.+Lucasauthor=C.+Elkinauthor=E.+Uberbacherauthor=M.+Frazierauthor=R.+A.+Gibbsauthor=D.+M.+Muznyauthor=S.+E.+Schererauthor=J.+B.+Bouckauthor=E.+J.+Sodergrenauthor=K.+C.+Worleyauthor=C.+M.+Rivesauthor=J.+H.+Gorrellauthor=M.+L.+Metzkerauthor=S.+L.+Naylorauthor=R.+S.+Kucherlapatiauthor=D.+L.+Nelsonauthor=G.+M.+Weinstockauthor=Y.+Sakakiauthor=A.+Fujiyamaauthor=M.+Hattoriauthor=T.+Yadaauthor=A.+Toyodaauthor=T.+Itohauthor=C.+Kawagoeauthor=H.+Watanabeauthor=Y.+Totokiauthor=T.+Taylorauthor=J.+Weissenbachauthor=R.+Heiligauthor=W.+Saurinauthor=F.+Artiguenaveauthor=P.+Brottierauthor=T.+Brulsauthor=E.+Pelletierauthor=C.+Robertauthor=P.+Winckerauthor=D.+R.+Smithauthor=L.+Doucette-Stammauthor=M.+Rubenfieldauthor=K.+Weinstockauthor=H.+M.+Leeauthor=J.+Duboisauthor=A.+Rosenthalauthor=M.+Platzerauthor=G.+Nyakaturaauthor=S.+Taudienauthor=A.+Rumpauthor=H.+Yangauthor=J.+Yuauthor=J.+Wangauthor=G.+Huangauthor=J.+Guauthor=L.+Hoodauthor=L.+Rowenauthor=A.+Madanauthor=S.+Qinauthor=R.+W.+Davisauthor=N.+A.+Federspielauthor=A.+P.+Abolaauthor=M.+J.+Proctorauthor=R.+M.+Myersauthor=J.+Schmutzauthor=M.+Dicksonauthor=J.+Grimwoodauthor=D.+R.+Coxauthor=M.+V.+Olsonauthor=R.+Kaulauthor=C.+Raymondauthor=N.+Shimizuauthor=K.+Kawasakiauthor=S.+Minoshimaauthor=G.+A.+Evansauthor=M.+Athanasiouauthor=R.+Schultzauthor=B.+A.+Roeauthor=F.+Chenauthor=H.+Panauthor=J.+Ramserauthor=H.+Lehrachauthor=R.+Reinhardtauthor=W.+R.+McCombieauthor=M.+de+la+Bastideauthor=N.+Dedhiaauthor=H.+Blockerauthor=K.+Hornischerauthor=G.+Nordsiekauthor=R.+Agarwalaauthor=L.+Aravindauthor=J.+A.+Baileyauthor=A.+Batemanauthor=S.+Batzoglouauthor=E.+Birneyauthor=P.+Borkauthor=D.+G.+Brownauthor=C.+B.+Burgeauthor=L.+Ceruttiauthor=H.+C.+Chenauthor=D.+Churchauthor=M.+Clampauthor=R.+R.+Copleyauthor=T.+Doerksauthor=S.+R.+Eddyauthor=E.+E.+Eichlerauthor=T.+S.+Fureyauthor=J.+Galaganauthor=J.+G.+Gilbertauthor=C.+Harmonauthor=Y.+Hayashizakiauthor=D.+Hausslerauthor=H.+Hermjakobauthor=K.+Hokampauthor=W.+Jangauthor=L.+S.+Johnsonauthor=T.+A.+Jonesauthor=S.+Kasifauthor=A.+Kaspryzkauthor=S.+Kennedyauthor=W.+J.+Kentauthor=P.+Kittsauthor=E.+V.+Kooninauthor=I.+Korfauthor=D.+Kulpauthor=D.+Lancetauthor=T.+M.+Loweauthor=A.+McLysaghtauthor=T.+Mikkelsenauthor=J.+V.+Moranauthor=N.+Mulderauthor=V.+J.+Pollaraauthor=C.+P.+Pontingauthor=G.+Schulerauthor=J.+Schultzauthor=G.+Slaterauthor=A.+F.+Smitauthor=E.+Stupkaauthor=J.+Szustakowkiauthor=D.+Thierry-Miegauthor=J.+Thierry-Miegauthor=L.+Wagnerauthor=J.+Wallisauthor=R.+Wheelerauthor=A.+Williamsauthor=Y.+I.+Wolfauthor=K.+H.+Wolfeauthor=S.+P.+Yangauthor=R.+F.+Yehauthor=F.+Collinsauthor=M.+S.+Guyerauthor=J.+Petersonauthor=A.+Felsenfeldauthor=K.+A.+Wetterstrandauthor=A.+Patrinosauthor=M.+J.+Morganauthor=P.+de+Jongauthor=J.+J.+Cataneseauthor=K.+Osoegawaauthor=H.+Shizuyaauthor=S.+Choiauthor=Y.+J.+Chenauthor=J.+Szustakowki&title=Initial+sequencing+and+analysis+of+the+human+genome&doi=10.1038%2F35057062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Initial sequencing and analysis of the human genome</span></div><div class="casAuthors">Lander, Eric S.; Linton, Lauren M.; Birren, Bruce; Nusbaum, Chad; Zody, Michael C.; Baldwin, Jennifer; Devon, Keri; Dewar, Ken; Doyle, Michael; FitzHugh, William; Funke, Roel; Gage, Diane; Harris, Katrina; Heaford, Andrew; Howland, John; Kann, Lisa; Lehoczky, Jessica; LeVine, Rosie; McEwan, Paul; McKernan, Kevin; Meldrim, James; Mesirov, Jill P.; Miranda, Cher; Morris, William; Naylor, Jerome; Raymond, Christina; Rosetti, Mark; Santos, Ralph; Sheridan, Andrew; Sougnez, Carrie; Stange-Thomann, Nicole; Stojanovic, Nikola; Subramanian, Aravind; Wyman, Dudley; Sulston, John; Ainscough, Rachael; Beck, Stephan; Bentley, David; Burton, John; Clee, Christopher; Carter, Nigel; Coulson, Alan; Deadman, Rebecca; Deloukas, Panos; Dunham, Andrew; Dunham, Ian; Durbin, Richard; French, Lisa; Grafham, Darren; Gregory, Simon; Hubbard, Tim; Humphray, Sean; Hunt, Adrienne; Jones, Matthew; Lloyd, Christine; McMurray, Amanda; Matthews, Lucy; Mercer, Simon; Milne, Sarah; Mullikin, James C.; Mungall, Andrew; Plumb, Robert; Ross, Mark; Shownkeen, Ratna; Sims, Sarah; Waterston, Robert H.; Wilson, Richard K.; Hillier, LaDeana W.; McPherson, John D.; Marra, Marco A.; Mardis, Elaine R.; Fulton, Lucinda A.; Chinwalla, Asif T.; Pepin, Kymberlie H.; Gish, Warren R.; Chissoe, Stephanie L.; Wendl, Michael C.; Delehaunty, Kim D.; Miner, Tracie L.; Delehaunty, Andrew; Kramer, Jason B.; Cook, Lisa L.; Fulton, Robert S.; Johnson, Douglas L.; Minx, Patrick J.; Clifton, Sandra W.; Harkins, Trevor; Branscomb, Elbert; Predki, Paul; Richardson, Paul; Wenning, Sarah; Slezak, Tom; Doggett, Norman; Cheng, Jan-Fang; Olsen, Anne; Lucas, Susan; Elkin, Christopher; Uberbacher, Edward; Frazier, Marvin; Gibbs, Richard A.; Muzny, Donna M.; Scherer, Steven E.; Bouck, John B.; Sodergren, Erica J.; Worley, Kim C.; Rives, Catherine M.; Gorrell, James H.; Metzker, Michael L.; Naylor, Susan L.; Kurcherlapati, Raju S.; Nelson, David L.; Weinstock, George M.; Sakaki, Yoshiyuki; Fujiyama, Asao; Hattori, Masahira; Yada, Tetsushi; Toyoda, Atsushi; Itoh, Takehiko; Kawagoe, Chiharu; Watanabe, Hidemi; Totoki, Yasushi; Taylor, Todd; Weissenbach, Jean; Heilig, Roland; Saurin, William; Artiguenave, Francois; Brottier, Philippe; Bruls, Thomas; Pelletier, Eric; Robert, Catherine; Wincker, Patrick; Smith, Douglas R.; Doucette-Stamm, Lynn; Rubenfield, Marc; Weinstock, Keith; Lee, Hong Mei; Dubois, JoAnn; Rosenthal, Andre; Platzer, Matthias; Nyakatura, Gerald; Taudien, Stefan; Rump, Andreas; Yang, Huanming; Yu, Jun; Wang, Jian; Huang, Guyang; Gu, Ju; Hood, Leroy; Rowen, Lee; Madan, Anup; Qin, Shizen; Davis, Ronald W.; Federspiel, Nancy A.; Pia Abola, A.; Proctor, Michael J.; Schmutz, Jeremy; Dickson, Mark; Grimwood, Jane; Cox, David R.; Olson, Maynard V.; Kaul, Rajinder; Raymond, Christopher; Shimizu, Nobuyoshi; Kawasaki, Kazuhiko; Minoshima, Shinsei; Evans, Glen A.; Athanasiou, Maria; Schutlz, Roger; Roe, Bruce A.; Chen, Feng; Pan, Hauqin; Ramser, Juilane; Lehrach, Hans; Reinhardt, Richard; McCombie, W. Richard; de la Bastide, Melissa; Dedhia, Neilay; Blocker, Helmut; Hornischer, Klaus; Nordsiek, Gabriele; Agarwala, Richa; Aravind, L.; Bailey, Jeffrey A.; Bateman, Alex; Batzoglou, Serafim; Birney, Ewan; Bork, Peer; Brown, Daniel G.; Burge, Christopher B.; Cerutti, Lorenzo; Chen, Hsiu-Chuan; Church, Deanna; Clamp, Michele; Copley, Richard R.; Doerks, Tobias; Eddy, Sean R.; Eichler, Evan E.; Furey, Terrence S.; Galagan, James; Gilbert, James G. R.; Harmon, Cyrus; Hayashizaki, Yoshihide; Haussler, David; Hermjakob, Henning; Hokamp, Karsten; Jang, Wonhee; Johnson, L. Steven; Jones, Thomas A.; Kasit, Simon; Kaspryzk, Arek; Kennedy, Scot; Kent, W. James; Kitts, Paul; Koonin, Eugene V.; Kort, Ian; Kulp, David; Lancet, Doron; Lowe, Todd M.; McLysaght, Aoife; Mikkelsen, Tarjei; Moran, John V.; Mulder, Nicola; Pollara, Victor J.; Ponting, Chris P.; Schuler, Greg; Schultz, Jorg; Slater, Guy; Smit, Arian; Stupka, Eila; Szustakowki, Joseph; Thierry-Mieg, Danielle; Thierry-Mieg, Jean; Wagner, Lukas; Wallis, John; Wheeler, Raymond; Williams, Alan; Wolf, Yuri I.; Wolfe, Kenneth H.; Yang, Shiaw-Pyng; Yeh, Ru-Fang; Collins, Francis; Guyer, Mark S.; Peterson, Jane; Felsenfeld, Adam; Wetterstrand, Kris A.; Patrinos, Aristides; Morgan, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">409</span>
        (<span class="NLM_cas:issue">6822</span>),
    <span class="NLM_cas:pages">860-921</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The human genome holds an extraordinary trove of information about human development, physiol., medicine, and evolution.  The results of an international collaboration to produce and make freely available a draft sequence of the human genome are now reported.  An initial anal. of the data is also presented, describing some of the insights that can be gleaned from the sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBLwD5_sOlWrVg90H21EOLACvtfcHk0licC_b78j4Z2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhsFCjtLc%253D&md5=b5554a573a36709f185f5e23bb46ad1f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F35057062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35057062%26sid%3Dliteratum%253Aachs%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DLinton%26aufirst%3DL.%2BM.%26aulast%3DBirren%26aufirst%3DB.%26aulast%3DNusbaum%26aufirst%3DC.%26aulast%3DZody%26aufirst%3DM.%2BC.%26aulast%3DBaldwin%26aufirst%3DJ.%26aulast%3DDevon%26aufirst%3DK.%26aulast%3DDewar%26aufirst%3DK.%26aulast%3DDoyle%26aufirst%3DM.%26aulast%3DFitzHugh%26aufirst%3DW.%26aulast%3DFunke%26aufirst%3DR.%26aulast%3DGage%26aufirst%3DD.%26aulast%3DHarris%26aufirst%3DK.%26aulast%3DHeaford%26aufirst%3DA.%26aulast%3DHowland%26aufirst%3DJ.%26aulast%3DKann%26aufirst%3DL.%26aulast%3DLehoczky%26aufirst%3DJ.%26aulast%3DLeVine%26aufirst%3DR.%26aulast%3DMcEwan%26aufirst%3DP.%26aulast%3DMcKernan%26aufirst%3DK.%26aulast%3DMeldrim%26aufirst%3DJ.%26aulast%3DMesirov%26aufirst%3DJ.%2BP.%26aulast%3DMiranda%26aufirst%3DC.%26aulast%3DMorris%26aufirst%3DW.%26aulast%3DNaylor%26aufirst%3DJ.%26aulast%3DRaymond%26aufirst%3DC.%26aulast%3DRosetti%26aufirst%3DM.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DSheridan%26aufirst%3DA.%26aulast%3DSougnez%26aufirst%3DC.%26aulast%3DStange-Thomann%26aufirst%3DY.%26aulast%3DStojanovic%26aufirst%3DN.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DWyman%26aufirst%3DD.%26aulast%3DRogers%26aufirst%3DJ.%26aulast%3DSulston%26aufirst%3DJ.%26aulast%3DAinscough%26aufirst%3DR.%26aulast%3DBeck%26aufirst%3DS.%26aulast%3DBentley%26aufirst%3DD.%26aulast%3DBurton%26aufirst%3DJ.%26aulast%3DClee%26aufirst%3DC.%26aulast%3DCarter%26aufirst%3DN.%26aulast%3DCoulson%26aufirst%3DA.%26aulast%3DDeadman%26aufirst%3DR.%26aulast%3DDeloukas%26aufirst%3DP.%26aulast%3DDunham%26aufirst%3DA.%26aulast%3DDunham%26aufirst%3DI.%26aulast%3DDurbin%26aufirst%3DR.%26aulast%3DFrench%26aufirst%3DL.%26aulast%3DGrafham%26aufirst%3DD.%26aulast%3DGregory%26aufirst%3DS.%26aulast%3DHubbard%26aufirst%3DT.%26aulast%3DHumphray%26aufirst%3DS.%26aulast%3DHunt%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DLloyd%26aufirst%3DC.%26aulast%3DMcMurray%26aufirst%3DA.%26aulast%3DMatthews%26aufirst%3DL.%26aulast%3DMercer%26aufirst%3DS.%26aulast%3DMilne%26aufirst%3DS.%26aulast%3DMullikin%26aufirst%3DJ.%2BC.%26aulast%3DMungall%26aufirst%3DA.%26aulast%3DPlumb%26aufirst%3DR.%26aulast%3DRoss%26aufirst%3DM.%26aulast%3DShownkeen%26aufirst%3DR.%26aulast%3DSims%26aufirst%3DS.%26aulast%3DWaterston%26aufirst%3DR.%2BH.%26aulast%3DWilson%26aufirst%3DR.%2BK.%26aulast%3DHillier%26aufirst%3DL.%2BW.%26aulast%3DMcPherson%26aufirst%3DJ.%2BD.%26aulast%3DMarra%26aufirst%3DM.%2BA.%26aulast%3DMardis%26aufirst%3DE.%2BR.%26aulast%3DFulton%26aufirst%3DL.%2BA.%26aulast%3DChinwalla%26aufirst%3DA.%2BT.%26aulast%3DPepin%26aufirst%3DK.%2BH.%26aulast%3DGish%26aufirst%3DW.%2BR.%26aulast%3DChissoe%26aufirst%3DS.%2BL.%26aulast%3DWendl%26aufirst%3DM.%2BC.%26aulast%3DDelehaunty%26aufirst%3DK.%2BD.%26aulast%3DMiner%26aufirst%3DT.%2BL.%26aulast%3DDelehaunty%26aufirst%3DA.%26aulast%3DKramer%26aufirst%3DJ.%2BB.%26aulast%3DCook%26aufirst%3DL.%2BL.%26aulast%3DFulton%26aufirst%3DR.%2BS.%26aulast%3DJohnson%26aufirst%3DD.%2BL.%26aulast%3DMinx%26aufirst%3DP.%2BJ.%26aulast%3DClifton%26aufirst%3DS.%2BW.%26aulast%3DHawkins%26aufirst%3DT.%26aulast%3DBranscomb%26aufirst%3DE.%26aulast%3DPredki%26aufirst%3DP.%26aulast%3DRichardson%26aufirst%3DP.%26aulast%3DWenning%26aufirst%3DS.%26aulast%3DSlezak%26aufirst%3DT.%26aulast%3DDoggett%26aufirst%3DN.%26aulast%3DCheng%26aufirst%3DJ.%2BF.%26aulast%3DOlsen%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DS.%26aulast%3DElkin%26aufirst%3DC.%26aulast%3DUberbacher%26aufirst%3DE.%26aulast%3DFrazier%26aufirst%3DM.%26aulast%3DGibbs%26aufirst%3DR.%2BA.%26aulast%3DMuzny%26aufirst%3DD.%2BM.%26aulast%3DScherer%26aufirst%3DS.%2BE.%26aulast%3DBouck%26aufirst%3DJ.%2BB.%26aulast%3DSodergren%26aufirst%3DE.%2BJ.%26aulast%3DWorley%26aufirst%3DK.%2BC.%26aulast%3DRives%26aufirst%3DC.%2BM.%26aulast%3DGorrell%26aufirst%3DJ.%2BH.%26aulast%3DMetzker%26aufirst%3DM.%2BL.%26aulast%3DNaylor%26aufirst%3DS.%2BL.%26aulast%3DKucherlapati%26aufirst%3DR.%2BS.%26aulast%3DNelson%26aufirst%3DD.%2BL.%26aulast%3DWeinstock%26aufirst%3DG.%2BM.%26aulast%3DSakaki%26aufirst%3DY.%26aulast%3DFujiyama%26aufirst%3DA.%26aulast%3DHattori%26aufirst%3DM.%26aulast%3DYada%26aufirst%3DT.%26aulast%3DToyoda%26aufirst%3DA.%26aulast%3DItoh%26aufirst%3DT.%26aulast%3DKawagoe%26aufirst%3DC.%26aulast%3DWatanabe%26aufirst%3DH.%26aulast%3DTotoki%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DT.%26aulast%3DWeissenbach%26aufirst%3DJ.%26aulast%3DHeilig%26aufirst%3DR.%26aulast%3DSaurin%26aufirst%3DW.%26aulast%3DArtiguenave%26aufirst%3DF.%26aulast%3DBrottier%26aufirst%3DP.%26aulast%3DBruls%26aufirst%3DT.%26aulast%3DPelletier%26aufirst%3DE.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DWincker%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DD.%2BR.%26aulast%3DDoucette-Stamm%26aufirst%3DL.%26aulast%3DRubenfield%26aufirst%3DM.%26aulast%3DWeinstock%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DH.%2BM.%26aulast%3DDubois%26aufirst%3DJ.%26aulast%3DRosenthal%26aufirst%3DA.%26aulast%3DPlatzer%26aufirst%3DM.%26aulast%3DNyakatura%26aufirst%3DG.%26aulast%3DTaudien%26aufirst%3DS.%26aulast%3DRump%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DHood%26aufirst%3DL.%26aulast%3DRowen%26aufirst%3DL.%26aulast%3DMadan%26aufirst%3DA.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DDavis%26aufirst%3DR.%2BW.%26aulast%3DFederspiel%26aufirst%3DN.%2BA.%26aulast%3DAbola%26aufirst%3DA.%2BP.%26aulast%3DProctor%26aufirst%3DM.%2BJ.%26aulast%3DMyers%26aufirst%3DR.%2BM.%26aulast%3DSchmutz%26aufirst%3DJ.%26aulast%3DDickson%26aufirst%3DM.%26aulast%3DGrimwood%26aufirst%3DJ.%26aulast%3DCox%26aufirst%3DD.%2BR.%26aulast%3DOlson%26aufirst%3DM.%2BV.%26aulast%3DKaul%26aufirst%3DR.%26aulast%3DRaymond%26aufirst%3DC.%26aulast%3DShimizu%26aufirst%3DN.%26aulast%3DKawasaki%26aufirst%3DK.%26aulast%3DMinoshima%26aufirst%3DS.%26aulast%3DEvans%26aufirst%3DG.%2BA.%26aulast%3DAthanasiou%26aufirst%3DM.%26aulast%3DSchultz%26aufirst%3DR.%26aulast%3DRoe%26aufirst%3DB.%2BA.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DRamser%26aufirst%3DJ.%26aulast%3DLehrach%26aufirst%3DH.%26aulast%3DReinhardt%26aufirst%3DR.%26aulast%3DMcCombie%26aufirst%3DW.%2BR.%26aulast%3Dde%2Bla%2BBastide%26aufirst%3DM.%26aulast%3DDedhia%26aufirst%3DN.%26aulast%3DBlocker%26aufirst%3DH.%26aulast%3DHornischer%26aufirst%3DK.%26aulast%3DNordsiek%26aufirst%3DG.%26aulast%3DAgarwala%26aufirst%3DR.%26aulast%3DAravind%26aufirst%3DL.%26aulast%3DBailey%26aufirst%3DJ.%2BA.%26aulast%3DBateman%26aufirst%3DA.%26aulast%3DBatzoglou%26aufirst%3DS.%26aulast%3DBirney%26aufirst%3DE.%26aulast%3DBork%26aufirst%3DP.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DBurge%26aufirst%3DC.%2BB.%26aulast%3DCerutti%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DH.%2BC.%26aulast%3DChurch%26aufirst%3DD.%26aulast%3DClamp%26aufirst%3DM.%26aulast%3DCopley%26aufirst%3DR.%2BR.%26aulast%3DDoerks%26aufirst%3DT.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26aulast%3DEichler%26aufirst%3DE.%2BE.%26aulast%3DFurey%26aufirst%3DT.%2BS.%26aulast%3DGalagan%26aufirst%3DJ.%26aulast%3DGilbert%26aufirst%3DJ.%2BG.%26aulast%3DHarmon%26aufirst%3DC.%26aulast%3DHayashizaki%26aufirst%3DY.%26aulast%3DHaussler%26aufirst%3DD.%26aulast%3DHermjakob%26aufirst%3DH.%26aulast%3DHokamp%26aufirst%3DK.%26aulast%3DJang%26aufirst%3DW.%26aulast%3DJohnson%26aufirst%3DL.%2BS.%26aulast%3DJones%26aufirst%3DT.%2BA.%26aulast%3DKasif%26aufirst%3DS.%26aulast%3DKaspryzk%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DKent%26aufirst%3DW.%2BJ.%26aulast%3DKitts%26aufirst%3DP.%26aulast%3DKoonin%26aufirst%3DE.%2BV.%26aulast%3DKorf%26aufirst%3DI.%26aulast%3DKulp%26aufirst%3DD.%26aulast%3DLancet%26aufirst%3DD.%26aulast%3DLowe%26aufirst%3DT.%2BM.%26aulast%3DMcLysaght%26aufirst%3DA.%26aulast%3DMikkelsen%26aufirst%3DT.%26aulast%3DMoran%26aufirst%3DJ.%2BV.%26aulast%3DMulder%26aufirst%3DN.%26aulast%3DPollara%26aufirst%3DV.%2BJ.%26aulast%3DPonting%26aufirst%3DC.%2BP.%26aulast%3DSchuler%26aufirst%3DG.%26aulast%3DSchultz%26aufirst%3DJ.%26aulast%3DSlater%26aufirst%3DG.%26aulast%3DSmit%26aufirst%3DA.%2BF.%26aulast%3DStupka%26aufirst%3DE.%26aulast%3DSzustakowki%26aufirst%3DJ.%26aulast%3DThierry-Mieg%26aufirst%3DD.%26aulast%3DThierry-Mieg%26aufirst%3DJ.%26aulast%3DWagner%26aufirst%3DL.%26aulast%3DWallis%26aufirst%3DJ.%26aulast%3DWheeler%26aufirst%3DR.%26aulast%3DWilliams%26aufirst%3DA.%26aulast%3DWolf%26aufirst%3DY.%2BI.%26aulast%3DWolfe%26aufirst%3DK.%2BH.%26aulast%3DYang%26aufirst%3DS.%2BP.%26aulast%3DYeh%26aufirst%3DR.%2BF.%26aulast%3DCollins%26aufirst%3DF.%26aulast%3DGuyer%26aufirst%3DM.%2BS.%26aulast%3DPeterson%26aufirst%3DJ.%26aulast%3DFelsenfeld%26aufirst%3DA.%26aulast%3DWetterstrand%26aufirst%3DK.%2BA.%26aulast%3DPatrinos%26aufirst%3DA.%26aulast%3DMorgan%26aufirst%3DM.%2BJ.%26aulast%3Dde%2BJong%26aufirst%3DP.%26aulast%3DCatanese%26aufirst%3DJ.%2BJ.%26aulast%3DOsoegawa%26aufirst%3DK.%26aulast%3DShizuya%26aufirst%3DH.%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%2BJ.%26aulast%3DSzustakowki%26aufirst%3DJ.%26atitle%3DInitial%2520sequencing%2520and%2520analysis%2520of%2520the%2520human%2520genome%26jtitle%3DNature%26date%3D2001%26volume%3D409%26spage%3D860%26epage%3D921%26doi%3D10.1038%2F35057062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, N. G.</span></span> <span> </span><span class="NLM_article-title">Proteome and proteomics: new technologies, new concepts, and new words</span>. <i>Electrophoresis</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1853</span>– <span class="NLM_lpage">1861</span>, <span class="refDoi"> DOI: 10.1002/elps.1150191103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1002%2Felps.1150191103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=9740045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaK1cXmtlGrt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1998&pages=1853-1861&author=N.+L.+Andersonauthor=N.+G.+Anderson&title=Proteome+and+proteomics%3A+new+technologies%2C+new+concepts%2C+and+new+words&doi=10.1002%2Felps.1150191103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome and proteomics. New technologies, new concepts, and new words</span></div><div class="casAuthors">Anderson, N. Leigh; Anderson, Norman G.</div><div class="citationInfo"><span class="NLM_cas:title">Electrophoresis</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1853-1861</span>CODEN:
                <span class="NLM_cas:coden">ELCTDN</span>;
        ISSN:<span class="NLM_cas:issn">0173-0835</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">A review with 41 refs.  The goal of proteomics is a comprehensive, quant. description of protein expression and its changes under the influence of biol. perturbations such as disease or drug treatment.  Quant. anal. of protein expression data obtained by high-throughput methods has led us to define the concept of regulatory homol. and use it to begin to elucidate the basic structure of gene expression control in vivo.  Such investigations lay the groundwork for construction of comprehensive databases of mechanisms (cataloguing possible biol. outcomes), the next logical step after the soon to be completed cataloguing of genes and gene products.  Mechanism databases provide a roadmap towards effective therapeutic intervention that is more direct than that offered by conventional genomics approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqkBUnV8ugtbVg90H21EOLACvtfcHk0lgB3WjC9mE-bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmtlGrt7o%253D&md5=f401c7bf42c69a996722b7c5deb78bb7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Felps.1150191103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Felps.1150191103%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DN.%2BL.%26aulast%3DAnderson%26aufirst%3DN.%2BG.%26atitle%3DProteome%2520and%2520proteomics%253A%2520new%2520technologies%252C%2520new%2520concepts%252C%2520and%2520new%2520words%26jtitle%3DElectrophoresis%26date%3D1998%26volume%3D19%26spage%3D1853%26epage%3D1861%26doi%3D10.1002%2Felps.1150191103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bertozzi, C. R.</span>; <span class="NLM_string-name">Sasisekharan, R.</span></span> <span> </span><span class="NLM_article-title">Glycomics</span>. In  <i>Essentials of Glycobiology</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Varki, A.</span>, <span class="NLM_string-name">Cummings, R. D.</span>, <span class="NLM_string-name">Esko, J. D.</span>, <span class="NLM_string-name">Freeze, H. H.</span>, <span class="NLM_string-name">Stanley, P.</span>, <span class="NLM_string-name">Bertozzi, C. R.</span>, <span class="NLM_string-name">Hart, G. W.</span>, <span class="NLM_string-name">Etzler, M. E.</span></span>, Eds.; <span class="NLM_publisher-name">Cold Spring Harbor Laboratory Press</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=C.+R.+Bertozzi&author=R.+Sasisekharanauthor=A.+Varki&author=R.+D.+Cummings&author=J.+D.+Esko&author=H.+H.+Freeze&author=P.+Stanley&author=C.+R.+Bertozzi&author=G.+W.+Hart&author=M.+E.+Etzler&title=Essentials+of+Glycobiology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBertozzi%26aufirst%3DC.%2BR.%26atitle%3DGlycomics%26btitle%3DEssentials%2520of%2520Glycobiology%26aulast%3DVarki%26aufirst%3DA.%26pub%3DCold%2520Spring%2520Harbor%2520Laboratory%2520Press%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ejsing, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampaio, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surendranath, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duchoslav, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekroos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klemm, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shevchenko, A.</span></span> <span> </span><span class="NLM_article-title">Global analysis of the yeast lipidome by quantitative shotgun mass spectrometry</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">2136</span>– <span class="NLM_lpage">2141</span>, <span class="refDoi"> DOI: 10.1073/pnas.0811700106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.0811700106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19174513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlaqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=2136-2141&author=C.+S.+Ejsingauthor=J.+L.+Sampaioauthor=V.+Surendranathauthor=E.+Duchoslavauthor=K.+Ekroosauthor=R.+W.+Klemmauthor=K.+Simonsauthor=A.+Shevchenko&title=Global+analysis+of+the+yeast+lipidome+by+quantitative+shotgun+mass+spectrometry&doi=10.1073%2Fpnas.0811700106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Global analysis of the yeast lipidome by quantitative shotgun mass spectrometry</span></div><div class="casAuthors">Ejsing, Christer S.; Sampaio, Julio L.; Surendranath, Vineeth; Duchoslav, Eva; Ekroos, Kim; Klemm, Robin W.; Simons, Kai; Shevchenko, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2136-2141</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Although the transcriptome, proteome, and interactome of several eukaryotic model organisms have been described in detail, lipidomes remain relatively uncharacterized.  Using Saccharomyces cerevisiae as an example, the authors demonstrate that automated shotgun lipidomics anal. enabled lipidome-wide abs. quantification of individual mol. lipid species by streamlined processing of a single sample of only 2 million yeast cells.  By comparative lipidomics, the authors achieved the abs. quantification of 250 mol. lipid species covering 21 major lipid classes.  This anal. provided ≈95% coverage of the yeast lipidome achieved with 125-fold improvement in sensitivity compared with previous approaches.  Comparative lipidomics demonstrated that growth temp. and defects in lipid biosynthesis induce ripple effects throughout the mol. compn. of the yeast lipidome.  This work serves as a resource for mol. characterization of eukaryotic lipidomes, and establishes shotgun lipidomics as a powerful platform for complementing biochem. studies and other systems-level approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxNQmv7RPMGbVg90H21EOLACvtfcHk0lgB3WjC9mE-bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlaqtr4%253D&md5=07d1c452da5f08731ade905cde007e80</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0811700106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0811700106%26sid%3Dliteratum%253Aachs%26aulast%3DEjsing%26aufirst%3DC.%2BS.%26aulast%3DSampaio%26aufirst%3DJ.%2BL.%26aulast%3DSurendranath%26aufirst%3DV.%26aulast%3DDuchoslav%26aufirst%3DE.%26aulast%3DEkroos%26aufirst%3DK.%26aulast%3DKlemm%26aufirst%3DR.%2BW.%26aulast%3DSimons%26aufirst%3DK.%26aulast%3DShevchenko%26aufirst%3DA.%26atitle%3DGlobal%2520analysis%2520of%2520the%2520yeast%2520lipidome%2520by%2520quantitative%2520shotgun%2520mass%2520spectrometry%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D2136%26epage%3D2141%26doi%3D10.1073%2Fpnas.0811700106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGettigan, P. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptomics in the RNA-seq era</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2012.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.cbpa.2012.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=23290152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2013&pages=4-11&author=P.+A.+McGettigan&title=Transcriptomics+in+the+RNA-seq+era&doi=10.1016%2Fj.cbpa.2012.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptomics in the RNA-seq era</span></div><div class="casAuthors">McGettigan, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-11</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The transcriptomics field has developed rapidly with the advent of next-generation sequencing technologies.  RNA-seq has now displaced microarrays as the preferred method for gene expression profiling.The comprehensive nature of the data generated has been a boon in terms of transcript identification but anal. challenges remain.  Key among these problems is the development of suitable expression metrics for expression level comparisons and methods for identification of differentially expressed genes (and exons).  Several approaches were developed but as yet no consensus exists on the best pipeline to use.  De novo transcriptome approaches are increasingly viable for organisms lacking a sequenced genome.  The redn. in starting RNA required has enabled the development of new applications such as single cell transcriptomics.  The emerging picture of mammalian transcription is complex with further refinement expected with the integration of epigenomic data generated by projects such as ENCODE.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgBaCUjBafcLVg90H21EOLACvtfcHk0lgB5b1bZWb6wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWqsw%253D%253D&md5=8249af313bf4f611dec1d167ef6c7097</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2012.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2012.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DMcGettigan%26aufirst%3DP.%2BA.%26atitle%3DTranscriptomics%2520in%2520the%2520RNA-seq%2520era%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D17%26spage%3D4%26epage%3D11%26doi%3D10.1016%2Fj.cbpa.2012.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adelstein, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassis, A. I.</span></span> <span> </span><span class="NLM_article-title">Target discovery from data mining approaches</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">S16</span>– <span class="NLM_lpage">S23</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.drudis.2011.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=22178890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns12htQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=S16-S23&author=Y.+Yangauthor=S.+J.+Adelsteinauthor=A.+I.+Kassis&title=Target+discovery+from+data+mining+approaches&doi=10.1016%2Fj.drudis.2011.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Target discovery from data mining approaches</span></div><div class="casAuthors">Yang, Yongliang; Adelstein, S. James; Kassis, Amin I.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">Suppl.</span>),
    <span class="NLM_cas:pages">S16-S23</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Data mining of available biomedical data and information has greatly boosted target discovery in the omics' era.  Target discovery is the key step in the biomarker and drug discovery pipeline to diagnose and fight human diseases.  In biomedical science, the target' is a broad concept ranging from mol. entities (such as genes, proteins and miRNAs) to biol. phenomena (such as mol. functions, pathways and phenotypes).  Within the context of biomedical science, data mining refers to a bioinformatics approach that combines biol. concepts with computer tools or statistical methods that are mainly used to discover, select and prioritize targets.  In response to the huge demand of data mining for target discovery in the omics' era, this review explicates various data mining approaches and their applications to target discovery with emphasis on text and microarray data anal.  Two emerging data mining approaches, chemogenomic data mining and proteomic data mining, are briefly introduced.  Also discussed are the limitations of various data mining approaches found in the level of database integration, the quality of data annotation, sample heterogeneity and the performance of anal. and mining tools.  Tentative strategies of integrating different data sources for target discovery, such as integrated text mining with high-throughput data anal. and integrated mining with pathway databases, are introduced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhbUNWD288SrVg90H21EOLACvtfcHk0lgB5b1bZWb6wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns12htQ%253D%253D&md5=329803c354df96e8b274e3e3bdc716e3</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DAdelstein%26aufirst%3DS.%2BJ.%26aulast%3DKassis%26aufirst%3DA.%2BI.%26atitle%3DTarget%2520discovery%2520from%2520data%2520mining%2520approaches%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26spage%3DS16%26epage%3DS23%26doi%3D10.1016%2Fj.drudis.2011.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaughn, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McManus, M. T.</span></span> <span> </span><span class="NLM_article-title">Pervasive transcription of the human genome produces thousands of previously unidentified long intergenic noncoding RNAs</span>. <i>PLoS Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e1003569</span> <span class="refDoi"> DOI: 10.1371/journal.pgen.1003569</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1371%2Fjournal.pgen.1003569" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=23818866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFersLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&author=M.+J.+Hangauerauthor=I.+W.+Vaughnauthor=M.+T.+McManus&title=Pervasive+transcription+of+the+human+genome+produces+thousands+of+previously+unidentified+long+intergenic+noncoding+RNAs&doi=10.1371%2Fjournal.pgen.1003569"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Pervasive transcription of the human genome produces thousands of previously unidentified long intergenic noncoding RNAs</span></div><div class="casAuthors">Hangauer, Matthew J.; Vaughn, Ian W.; McManus, Michael T.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1003569</span>CODEN:
                <span class="NLM_cas:coden">PGLEB5</span>;
        ISSN:<span class="NLM_cas:issn">1553-7404</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Known protein coding gene exons compose less than 3% of the human genome.  The remaining 97% is largely uncharted territory, with only a small fraction characterized.  The recent observation of transcription in this intergenic territory has stimulated debate about the extent of intergenic transcription and whether these intergenic RNAs are functional.  Here the authors directly obsd. with a large set of RNA-seq data covering a wide array of human tissue types that the majority of the genome is indeed transcribed, corroborating recent observations by the ENCODE project.  Furthermore, using de novo transcriptome assembly of this RNA-seq data, the authors found that intergenic regions encode far more long intergenic noncoding RNAs (lincRNAs) than previously described, helping to resolve the discrepancy between the vast amt. of obsd. intergenic transcription and the limited no. of previously known lincRNAs.  In total, the authors identified tens of thousands of putative lincRNAs expressed at a min. of one copy per cell, significantly expanding upon prior lincRNA annotation sets.  These lincRNAs are specifically regulated and conserved rather than being the product of transcriptional noise.  In addn., lincRNAs are strongly enriched for trait-assocd. SNPs suggesting a new mechanism by which intergenic trait-assocd. regions may function.  These findings will enable the discovery and interrogation of novel intergenic functional elements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu0Xih8vOO7Vg90H21EOLACvtfcHk0lgB5b1bZWb6wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFersLvL&md5=843cea216c8f8397e22e49c9616aa1cf</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1003569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1003569%26sid%3Dliteratum%253Aachs%26aulast%3DHangauer%26aufirst%3DM.%2BJ.%26aulast%3DVaughn%26aufirst%3DI.%2BW.%26aulast%3DMcManus%26aufirst%3DM.%2BT.%26atitle%3DPervasive%2520transcription%2520of%2520the%2520human%2520genome%2520produces%2520thousands%2520of%2520previously%2520unidentified%2520long%2520intergenic%2520noncoding%2520RNAs%26jtitle%3DPLoS%2520Genet.%26date%3D2013%26volume%3D9%26doi%3D10.1371%2Fjournal.pgen.1003569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Djebali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortazavi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagarde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlesinger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinov, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khatun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaleski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozowsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Röder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokocinski, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdelhamid, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alioto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antoshechkin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baer, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bar, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batut, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabortty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrast, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derrien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drenkow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumais, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumais, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duttagupta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falconnet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fastuca, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fejes-Toth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foissac, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fullwood, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunawardena, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howald, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapranov, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingswood, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persaud, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preall, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeca, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risk, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robyr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sammeth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">See, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahab, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skancke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tilgner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trout, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrobel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashizaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonarakis, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giddings, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carninci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guigó, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingeras, T. R.</span></span> <span> </span><span class="NLM_article-title">Landscape of transcription in human cells</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>489</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">108</span>, <span class="refDoi"> DOI: 10.1038/nature11233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnature11233" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=22955620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlGnt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=101-108&author=S.+Djebaliauthor=C.+A.+Davisauthor=A.+Merkelauthor=A.+Dobinauthor=T.+Lassmannauthor=A.+Mortazaviauthor=A.+Tanzerauthor=J.+Lagardeauthor=W.+Linauthor=F.+Schlesingerauthor=C.+Xueauthor=G.+K.+Marinovauthor=J.+Khatunauthor=B.+A.+Williamsauthor=C.+Zaleskiauthor=J.+Rozowskyauthor=M.+R%C3%B6derauthor=F.+Kokocinskiauthor=R.+F.+Abdelhamidauthor=T.+Aliotoauthor=I.+Antoshechkinauthor=M.+T.+Baerauthor=N.+S.+Barauthor=P.+Batutauthor=K.+Bellauthor=I.+Bellauthor=S.+Chakraborttyauthor=X.+Chenauthor=J.+Chrastauthor=J.+Curadoauthor=T.+Derrienauthor=J.+Drenkowauthor=E.+Dumaisauthor=J.+Dumaisauthor=R.+Duttaguptaauthor=E.+Falconnetauthor=M.+Fastucaauthor=K.+Fejes-Tothauthor=P.+Ferreiraauthor=S.+Foissacauthor=M.+J.+Fullwoodauthor=H.+Gaoauthor=D.+Gonzalezauthor=A.+Gordonauthor=H.+Gunawardenaauthor=C.+Howaldauthor=S.+Jhaauthor=R.+Johnsonauthor=P.+Kapranovauthor=B.+Kingauthor=C.+Kingswoodauthor=O.+J.+Luoauthor=E.+Parkauthor=K.+Persaudauthor=J.+B.+Preallauthor=P.+Ribecaauthor=B.+Riskauthor=D.+Robyrauthor=M.+Sammethauthor=L.+Schafferauthor=L.-H.+Seeauthor=A.+Shahabauthor=J.+Skanckeauthor=A.+M.+Suzukiauthor=H.+Takahashiauthor=H.+Tilgnerauthor=D.+Troutauthor=N.+Waltersauthor=H.+Wangauthor=J.+Wrobelauthor=Y.+Yuauthor=X.+Ruanauthor=Y.+Hayashizakiauthor=J.+Harrowauthor=M.+Gersteinauthor=T.+Hubbardauthor=A.+Reymondauthor=S.+E.+Antonarakisauthor=G.+Hannonauthor=M.+C.+Giddingsauthor=Y.+Ruanauthor=B.+Woldauthor=P.+Carninciauthor=R.+Guig%C3%B3author=T.+R.+Gingeras&title=Landscape+of+transcription+in+human+cells&doi=10.1038%2Fnature11233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Landscape of transcription in human cells</span></div><div class="casAuthors">Djebali, Sarah; Davis, Carrie A.; Merkel, Angelika; Dobin, Alex; Lassmann, Timo; Mortazavi, Ali; Tanzer, Andrea; Lagarde, Julien; Lin, Wei; Schlesinger, Felix; Xue, Chenghai; Marinov, Georgi K.; Khatun, Jainab; Williams, Brian A.; Zaleski, Chris; Rozowsky, Joel; Roeder, Maik; Kokocinski, Felix; Abdelhamid, Rehab F.; Alioto, Tyler; Antoshechkin, Igor; Baer, Michael T.; Bar, Nadav S.; Batut, Philippe; Bell, Kimberly; Bell, Ian; Chakrabortty, Sudipto; Chen, Xian; Chrast, Jacqueline; Curado, Joao; Derrien, Thomas; Drenkow, Jorg; Dumais, Erica; Dumais, Jacqueline; Duttagupta, Radha; Falconnet, Emilie; Fastuca, Meagan; Fejes-Toth, Kata; Ferreira, Pedro; Foissac, Sylvain; Fullwood, Melissa J.; Gao, Hui; Gonzalez, David; Gordon, Assaf; Gunawardena, Harsha; Howald, Cedric; Jha, Sonali; Johnson, Rory; Kapranov, Philipp; King, Brandon; Kingswood, Colin; Luo, Oscar J.; Park, Eddie; Persaud, Kimberly; Preall, Jonathan B.; Ribeca, Paolo; Risk, Brian; Robyr, Daniel; Sammeth, Michael; Schaffer, Lorian; See, Lei-Hoon; Shahab, Atif; Skancke, Jorgen; Suzuki, Ana Maria; Takahashi, Hazuki; Tilgner, Hagen; Trout, Diane; Walters, Nathalie; Wang, Huaien; Wrobel, John; Yu, Yanbao; Ruan, Xiaoan; Hayashizaki, Yoshihide; Harrow, Jennifer; Gerstein, Mark; Hubbard, Tim; Reymond, Alexandre; Antonarakis, Stylianos E.; Hannon, Gregory; Giddings, Morgan C.; Ruan, Yijun; Wold, Barbara; Carninci, Piero; Guigo, Roderic; Gingeras, Thomas R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">489</span>
        (<span class="NLM_cas:issue">7414</span>),
    <span class="NLM_cas:pages">101-108</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Eukaryotic cells make many types of primary and processed RNAs that are found either in specific subcellular compartments or throughout the cells.  A complete catalog of these RNAs is not yet available and their characteristic subcellular localizations are also poorly understood.  Because RNA represents the direct output of the genetic information encoded by genomes and a significant proportion of a cell's regulatory capabilities are focused on its synthesis, processing, transport, modification and translation, the generation of such a catalog is crucial for understanding genome function.  Here we report evidence that three-quarters of the human genome is capable of being transcribed, as well as observations about the range and levels of expression, localization, processing fates, regulatory regions and modifications of almost all currently annotated and thousands of previously unannotated RNAs.  These observations, taken together, prompt a redefinition of the concept of a gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQJgBaBiE8o7Vg90H21EOLACvtfcHk0liDcA64onctSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlGnt73M&md5=8f8ee2a1189ced1010dec5e45a305acf</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature11233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11233%26sid%3Dliteratum%253Aachs%26aulast%3DDjebali%26aufirst%3DS.%26aulast%3DDavis%26aufirst%3DC.%2BA.%26aulast%3DMerkel%26aufirst%3DA.%26aulast%3DDobin%26aufirst%3DA.%26aulast%3DLassmann%26aufirst%3DT.%26aulast%3DMortazavi%26aufirst%3DA.%26aulast%3DTanzer%26aufirst%3DA.%26aulast%3DLagarde%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DW.%26aulast%3DSchlesinger%26aufirst%3DF.%26aulast%3DXue%26aufirst%3DC.%26aulast%3DMarinov%26aufirst%3DG.%2BK.%26aulast%3DKhatun%26aufirst%3DJ.%26aulast%3DWilliams%26aufirst%3DB.%2BA.%26aulast%3DZaleski%26aufirst%3DC.%26aulast%3DRozowsky%26aufirst%3DJ.%26aulast%3DR%25C3%25B6der%26aufirst%3DM.%26aulast%3DKokocinski%26aufirst%3DF.%26aulast%3DAbdelhamid%26aufirst%3DR.%2BF.%26aulast%3DAlioto%26aufirst%3DT.%26aulast%3DAntoshechkin%26aufirst%3DI.%26aulast%3DBaer%26aufirst%3DM.%2BT.%26aulast%3DBar%26aufirst%3DN.%2BS.%26aulast%3DBatut%26aufirst%3DP.%26aulast%3DBell%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DI.%26aulast%3DChakrabortty%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChrast%26aufirst%3DJ.%26aulast%3DCurado%26aufirst%3DJ.%26aulast%3DDerrien%26aufirst%3DT.%26aulast%3DDrenkow%26aufirst%3DJ.%26aulast%3DDumais%26aufirst%3DE.%26aulast%3DDumais%26aufirst%3DJ.%26aulast%3DDuttagupta%26aufirst%3DR.%26aulast%3DFalconnet%26aufirst%3DE.%26aulast%3DFastuca%26aufirst%3DM.%26aulast%3DFejes-Toth%26aufirst%3DK.%26aulast%3DFerreira%26aufirst%3DP.%26aulast%3DFoissac%26aufirst%3DS.%26aulast%3DFullwood%26aufirst%3DM.%2BJ.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DGonzalez%26aufirst%3DD.%26aulast%3DGordon%26aufirst%3DA.%26aulast%3DGunawardena%26aufirst%3DH.%26aulast%3DHowald%26aufirst%3DC.%26aulast%3DJha%26aufirst%3DS.%26aulast%3DJohnson%26aufirst%3DR.%26aulast%3DKapranov%26aufirst%3DP.%26aulast%3DKing%26aufirst%3DB.%26aulast%3DKingswood%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DO.%2BJ.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DPersaud%26aufirst%3DK.%26aulast%3DPreall%26aufirst%3DJ.%2BB.%26aulast%3DRibeca%26aufirst%3DP.%26aulast%3DRisk%26aufirst%3DB.%26aulast%3DRobyr%26aufirst%3DD.%26aulast%3DSammeth%26aufirst%3DM.%26aulast%3DSchaffer%26aufirst%3DL.%26aulast%3DSee%26aufirst%3DL.-H.%26aulast%3DShahab%26aufirst%3DA.%26aulast%3DSkancke%26aufirst%3DJ.%26aulast%3DSuzuki%26aufirst%3DA.%2BM.%26aulast%3DTakahashi%26aufirst%3DH.%26aulast%3DTilgner%26aufirst%3DH.%26aulast%3DTrout%26aufirst%3DD.%26aulast%3DWalters%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWrobel%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DRuan%26aufirst%3DX.%26aulast%3DHayashizaki%26aufirst%3DY.%26aulast%3DHarrow%26aufirst%3DJ.%26aulast%3DGerstein%26aufirst%3DM.%26aulast%3DHubbard%26aufirst%3DT.%26aulast%3DReymond%26aufirst%3DA.%26aulast%3DAntonarakis%26aufirst%3DS.%2BE.%26aulast%3DHannon%26aufirst%3DG.%26aulast%3DGiddings%26aufirst%3DM.%2BC.%26aulast%3DRuan%26aufirst%3DY.%26aulast%3DWold%26aufirst%3DB.%26aulast%3DCarninci%26aufirst%3DP.%26aulast%3DGuig%25C3%25B3%26aufirst%3DR.%26aulast%3DGingeras%26aufirst%3DT.%2BR.%26atitle%3DLandscape%2520of%2520transcription%2520in%2520human%2520cells%26jtitle%3DNature%26date%3D2012%26volume%3D489%26spage%3D101%26epage%3D108%26doi%3D10.1038%2Fnature11233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wade, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grainger, D. C.</span></span> <span> </span><span class="NLM_article-title">Pervasive transcription: illuminating the dark matter of bacterial transcriptomes</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">653</span>, <span class="refDoi"> DOI: 10.1038/nrmicro3316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrmicro3316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=25069631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1amsLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=647-653&author=J.+T.+Wadeauthor=D.+C.+Grainger&title=Pervasive+transcription%3A+illuminating+the+dark+matter+of+bacterial+transcriptomes&doi=10.1038%2Fnrmicro3316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pervasive transcription: illuminating the dark matter of bacterial transcriptomes</span></div><div class="casAuthors">Wade, Joseph T.; Grainger, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">647-653</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The conventional view of transcription posits that mRNAs are generated from the coding DNA strand and are delineated by gene boundaries; however, recent reports have mapped transcription start sites to unexpected locations in bacterial genomes, including the non-coding strand.  The resultant RNAs were previously dismissed as artifacts, but models that describe such events as 'pervasive transcription' are now gaining support.  In this Opinion article, we discuss our current understanding of pervasive transcription, its genetic origin and its regulation.  On the basis of existing observations, we propose that RNAs that result from pervasive transcription are more than 'transcriptional noise' and have important functions in gene regulation and genome evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0npPgSYlBiLVg90H21EOLACvtfcHk0liDcA64onctSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1amsLzL&md5=8f2faa39c9b77d4c5a7fcbf989d14668</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro3316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro3316%26sid%3Dliteratum%253Aachs%26aulast%3DWade%26aufirst%3DJ.%2BT.%26aulast%3DGrainger%26aufirst%3DD.%2BC.%26atitle%3DPervasive%2520transcription%253A%2520illuminating%2520the%2520dark%2520matter%2520of%2520bacterial%2520transcriptomes%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2014%26volume%3D12%26spage%3D647%26epage%3D653%26doi%3D10.1038%2Fnrmicro3316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattick, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taft, R. J.</span></span> <span> </span><span class="NLM_article-title">A meta-analysis of the genomic and transcriptomic composition of complex life</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2061</span>– <span class="NLM_lpage">2072</span>, <span class="refDoi"> DOI: 10.4161/cc.25134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.4161%2Fcc.25134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=23759593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVOk" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=2061-2072&author=G.+Liuauthor=J.+S.+Mattickauthor=R.+J.+Taft&title=A+meta-analysis+of+the+genomic+and+transcriptomic+composition+of+complex+life&doi=10.4161%2Fcc.25134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A meta-analysis of the genomic and transcriptomic composition of complex life</span></div><div class="casAuthors">Liu, Ganqiang; Mattick, John S.; Taft, Ryan J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2061-2072</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">It is now clear that animal genomes are predominantly non-protein-coding, and that these sequences encode a wide array of RNA transcripts and other regulatory elements that are fundamental to the development of complex life.  We have previously argued that the proportion of an animal genome that is non-protein-coding DNA (ncDNA) correlates well with its apparent biol. complexity.  Here we extend on that work and, using data from a total of 1,627 prokaryotic and 153 eukaryotic complete and annotated genomes, show that the proportion of ncDNA per haploid genome is significantly pos. correlated with a previously published proxy of biol. complexity, the no. of distinct cell types.  This is in contrast to the amt. of the genome that encodes proteins, which we show is essentially unchanged across Metazoa.  Furthermore, using a total of 179 RNA-seq data sets from nematode (47), fruit fly (72), zebrafish (20) and human (42), we show, consistent with other recent reports, that the vast majority of ncDNA in animals is transcribed.  This includes more than 60 human loci previously considered "gene deserts," many of which are expressed tissue-specifically and assocd. with previously reported GWAS SNPs.  These results suggest that ncDNA, and the ncRNAs encoded within it, may be intimately involved in the evolution, maintenance and development of complex life.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGossTGtUcknC7Vg90H21EOLACvtfcHk0lh-0aN8Ty8Rsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVOk&md5=c20682cb9543b36ec1b2377ce862e5e7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.4161%2Fcc.25134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.25134%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DMattick%26aufirst%3DJ.%2BS.%26aulast%3DTaft%26aufirst%3DR.%2BJ.%26atitle%3DA%2520meta-analysis%2520of%2520the%2520genomic%2520and%2520transcriptomic%2520composition%2520of%2520complex%2520life%26jtitle%3DCell%2520Cycle%26date%3D2013%26volume%3D12%26spage%3D2061%26epage%3D2072%26doi%3D10.4161%2Fcc.25134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Necsulea, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soumillon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warnefors, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liechti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daish, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grützner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaessmann, H.</span></span> <span> </span><span class="NLM_article-title">The evolution of lncRNA repertoires and expression patterns in tetrapods</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>505</i></span>,  <span class="NLM_fpage">635</span>– <span class="NLM_lpage">640</span>, <span class="refDoi"> DOI: 10.1038/nature12943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnature12943" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=24463510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFemtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=505&publication_year=2014&pages=635-640&author=A.+Necsuleaauthor=M.+Soumillonauthor=M.+Warneforsauthor=A.+Liechtiauthor=T.+Daishauthor=U.+Zellerauthor=J.+C.+Bakerauthor=F.+Gr%C3%BCtznerauthor=H.+Kaessmann&title=The+evolution+of+lncRNA+repertoires+and+expression+patterns+in+tetrapods&doi=10.1038%2Fnature12943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The evolution of lncRNA repertoires and expression patterns in tetrapods</span></div><div class="casAuthors">Necsulea, Anamaria; Soumillon, Magali; Warnefors, Maria; Liechti, Angelica; Daish, Tasman; Zeller, Ulrich; Baker, Julie C.; Gruetzner, Frank; Kaessmann, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">505</span>
        (<span class="NLM_cas:issue">7485</span>),
    <span class="NLM_cas:pages">635-640</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Only a very small fraction of long noncoding RNAs (lncRNAs) are well characterized.  The evolutionary history of lncRNAs can provide insights into their functionality, but the absence of lncRNA annotations in non-model organisms has precluded comparative analyses.  Here we present a large-scale evolutionary study of lncRNA repertoires and expression patterns, in 11 tetrapod species.  We identify approx. 11,000 primate-specific lncRNAs and 2500 highly conserved lncRNAs, including approx. 400 genes that are likely to have originated more than 300 million years ago.  We find that lncRNAs, in particular ancient ones, are in general actively regulated and may function predominantly in embryonic development.  Most lncRNAs evolve rapidly in terms of sequence and expression levels, but tissue specificities are often conserved.  We compared expression patterns of homologous lncRNA and protein-coding families across tetrapods to reconstruct an evolutionarily conserved co-expression network.  This network suggests potential functions for lncRNAs in fundamental processes such as spermatogenesis and synaptic transmission, but also in more specific mechanisms such as placenta development through microRNA prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry4ZkSNvvosbVg90H21EOLACvtfcHk0lh-0aN8Ty8Rsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFemtLg%253D&md5=7eeda95062d8f96267952e8d452d5fb6</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature12943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12943%26sid%3Dliteratum%253Aachs%26aulast%3DNecsulea%26aufirst%3DA.%26aulast%3DSoumillon%26aufirst%3DM.%26aulast%3DWarnefors%26aufirst%3DM.%26aulast%3DLiechti%26aufirst%3DA.%26aulast%3DDaish%26aufirst%3DT.%26aulast%3DZeller%26aufirst%3DU.%26aulast%3DBaker%26aufirst%3DJ.%2BC.%26aulast%3DGr%25C3%25BCtzner%26aufirst%3DF.%26aulast%3DKaessmann%26aufirst%3DH.%26atitle%3DThe%2520evolution%2520of%2520lncRNA%2520repertoires%2520and%2520expression%2520patterns%2520in%2520tetrapods%26jtitle%3DNature%26date%3D2014%26volume%3D505%26spage%3D635%26epage%3D640%26doi%3D10.1038%2Fnature12943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Makley, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Expanding the number of “druggable” targets: non-enzymes and protein-protein interactions</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1111%2Fcbdd.12066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=23253128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1ymtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=22-32&author=L.+N.+Makleyauthor=J.+E.+Gestwicki&title=Expanding+the+number+of+%E2%80%9Cdruggable%E2%80%9D+targets%3A+non-enzymes+and+protein-protein+interactions&doi=10.1111%2Fcbdd.12066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding the number of "druggable" targets: non-enzymes and protein-protein interactions</span></div><div class="casAuthors">Makley, Leah N.; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-32</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Following sequencing and assembly of the human genome, the preferred methods for identification of new drug targets have changed dramatically.  Modern tactics such as genome-wide assocn. studies (GWAS) and deep sequencing are fundamentally different from the pharmacol.-guided approaches used previously, in which knowledge of small mol. ligands acting at their cellular targets was the primary discovery engine.  A consequence of the "target-first, pharmacol.-second" strategy is that many predicted drug targets are non-enzymes, such as scaffolding, regulatory or structural proteins, and their activities are often dependent on protein-protein interactions (PPIs).  These types of targets create unique challenges to drug discovery efforts because enzymic turnover cannot be used as a convenient surrogate for compd. potency.  Moreover, it is often challenging to predict how ligand binding to non-enzymes might affect changes in protein function and/or pathobiol.  Thus, in the postgenomic era, targets might be strongly implicated by mol. biol.-based methods, yet they often later earn the designation of "undruggable".  Can the scope of available targets be widened to include these promising, but challenging, non-enzymes.  In this review, we discuss advances in high-throughput screening (HTS) technol. and chem. library design that are emerging to deal with these challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMtMz2zhY_vLVg90H21EOLACvtfcHk0lh-0aN8Ty8Rsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1ymtQ%253D%253D&md5=9e8a15432b02550dca5c9e4668d99038</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12066%26sid%3Dliteratum%253Aachs%26aulast%3DMakley%26aufirst%3DL.%2BN.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DExpanding%2520the%2520number%2520of%2520%25E2%2580%259Cdruggable%25E2%2580%259D%2520targets%253A%2520non-enzymes%2520and%2520protein-protein%2520interactions%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2013%26volume%3D81%26spage%3D22%26epage%3D32%26doi%3D10.1111%2Fcbdd.12066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Limbird, L. E.</span></span> <span> </span><span class="NLM_article-title">The receptor concept: a continuing evolution</span>. <i>Mol. Interventions</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1124/mi.4.6.6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1124%2Fmi.4.6.6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=15616162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnt1ejtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=326-336&author=L.+E.+Limbird&title=The+receptor+concept%3A+a+continuing+evolution&doi=10.1124%2Fmi.4.6.6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The receptor concept: A continuing evolution</span></div><div class="casAuthors">Limbird, Lee E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Interventions</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">326-336</span>CODEN:
                <span class="NLM_cas:coden">MIONAR</span>;
        ISSN:<span class="NLM_cas:issn">1534-0384</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  The author briefly summarizes the development of the receptor concept, the identification receptors based 1st on biol. response data and subsequently on radioligand binding properties, and the biol. and physiol. understandings that these approaches have made possible.  The development of receptor characterization began with receptors that ultimately were discovered to mediate response by coupling to G-binding proteins, also known as G protein-coupled receptors (GPCRs).  Consequently, many if not all of the examples in this overview describe studies characterizing GPCRs in general, and adrenergic receptors in particular.  The purpose of this review, however, is not a detailed chronol. account of a huge literature, but rather an overview of the fundamental questions posed and answered by these studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8s65Hqwvj1bVg90H21EOLACvtfcHk0li5IkC2vkCWMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnt1ejtA%253D%253D&md5=b3ca1691f239fe1baac632f328332bb5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1124%2Fmi.4.6.6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmi.4.6.6%26sid%3Dliteratum%253Aachs%26aulast%3DLimbird%26aufirst%3DL.%2BE.%26atitle%3DThe%2520receptor%2520concept%253A%2520a%2520continuing%2520evolution%26jtitle%3DMol.%2520Interventions%26date%3D2004%26volume%3D4%26spage%3D326%26epage%3D336%26doi%3D10.1124%2Fmi.4.6.6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazo, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharlow, E. R.</span></span> <span> </span><span class="NLM_article-title">Drugging undruggable molecular cancer targets</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1146%2Fannurev-pharmtox-010715-103440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=26527069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGlsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=23-40&author=J.+S.+Lazoauthor=E.+R.+Sharlow&title=Drugging+undruggable+molecular+cancer+targets&doi=10.1146%2Fannurev-pharmtox-010715-103440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging Undruggable Molecular Cancer Targets</span></div><div class="casAuthors">Lazo, John S.; Sharlow, Elizabeth R.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23-40</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Cancer, more than any other human disease, now has a surfeit of potential mol. targets poised for therapeutic exploitation.  Currently, a no. of attractive and validated cancer targets remain outside of the reach of pharmacol. regulation.  Some have been described as undruggable, at least by traditional strategies.  In this article, we outline the basis for the undruggable moniker, propose a reclassification of these targets as undrugged, and highlight three general classes of this imposing group as exemplars with some attendant strategies currently being explored to reclassify them.  Expanding the spectrum of disease-relevant targets to pharmacol. manipulation is central to reducing cancer morbidity and mortality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpny-ev1lJKqbVg90H21EOLACvtfcHk0li5IkC2vkCWMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGlsr7O&md5=5ab0518d4155a00e2a898a6f68d551d8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103440%26sid%3Dliteratum%253Aachs%26aulast%3DLazo%26aufirst%3DJ.%2BS.%26aulast%3DSharlow%26aufirst%3DE.%2BR.%26atitle%3DDrugging%2520undruggable%2520molecular%2520cancer%2520targets%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2016%26volume%3D56%26spage%3D23%26epage%3D40%26doi%3D10.1146%2Fannurev-pharmtox-010715-103440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Drugging the ‘undruggable’ cancer targets</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrc.2017.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28643779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=502-508&author=C.+V.+Dangauthor=E.+P.+Reddyauthor=K.+M.+Shokatauthor=L.+Soucek&title=Drugging+the+%E2%80%98undruggable%E2%80%99+cancer+targets&doi=10.1038%2Fnrc.2017.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the 'undruggable' cancer targets</span></div><div class="casAuthors">Dang, Chi V.; Reddy, E. Premkumar; Shokat, Kevan M.; Soucek, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">502-508</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The term 'undruggable' was coined to describe proteins that could not be targeted pharmacol.  However, progress is being made to 'drug' many of these targets, and therefore more appropriate terms might be 'difficult to drug' or 'yet to be drugged'.  Many desirable targets in cancer fall into this category, including the RAS and MYC oncogenes, and pharmacol. targeting these intractable proteins is now a key challenge in cancer research that requires innovation and the development of new technologies.  In this Viewpoint article, we asked four scientists working in this field for their opinions on the most crucial advances, as well as the challenges and what the future holds for this important area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoElcMCmqGx_rVg90H21EOLACvtfcHk0li5IkC2vkCWMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP&md5=774da7ed04931a3fd3689eb9acea594a</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.36%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DReddy%26aufirst%3DE.%2BP.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DDrugging%2520the%2520%25E2%2580%2598undruggable%25E2%2580%2599%2520cancer%2520targets%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D502%26epage%3D508%26doi%3D10.1038%2Fnrc.2017.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ostrem, J. M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Direct small-molecule inhibitors of KRAS: from structural insights to mechanism-based design</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">771</span>– <span class="NLM_lpage">785</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrd.2016.139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27469033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=771-785&author=J.+M.+L.+Ostremauthor=K.+M.+Shokat&title=Direct+small-molecule+inhibitors+of+KRAS%3A+from+structural+insights+to+mechanism-based+design&doi=10.1038%2Fnrd.2016.139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Direct small-molecule inhibitors of KRAS: from structural insights to mechanism-based design</span></div><div class="casAuthors">Ostrem, Jonathan M. L.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">771-785</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">KRAS is the most frequently mutated oncogene in human cancer.  In addn. to holding this distinction, unsuccessful attempts to target this protein have led to the characterization of RAS as 'undruggable'.  However, recent advances in technol. and novel approaches to drug discovery have renewed hope that a direct KRAS inhibitor may be on the horizon.  In this Review, we provide an in-depth anal. of the structure, dynamics, mutational activation and inactivation, and signalling mechanisms of RAS.  From this perspective, we then consider potential mechanisms of action for effective RAS inhibitors.  Finally, we examine each of the many recent reports of direct RAS inhibitors and discuss promising avenues for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBobrhZLF3V7Vg90H21EOLACvtfcHk0lgkiJeofXSBzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FK&md5=b0832c7ef45cc0555e11c59baccc5abd</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.139%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%2BL.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DDirect%2520small-molecule%2520inhibitors%2520of%2520KRAS%253A%2520from%2520structural%2520insights%2520to%2520mechanism-based%2520design%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D771%26epage%3D785%26doi%3D10.1038%2Fnrd.2016.139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmachl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantoulidis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gollner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häring, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hela, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Oezguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knesl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornigg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kousek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamarre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munico-Martinez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peinsipp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salamon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherbantin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipany, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrenk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siszler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolkerstorfer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span> <span> </span><span class="NLM_article-title">Drugging an undruggable pocket on KRAS</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">15823</span>– <span class="NLM_lpage">15829</span>, <span class="refDoi"> DOI: 10.1073/pnas.1904529116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1904529116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31332011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFChs7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=15823-15829&author=D.+Kesslerauthor=M.+Gmachlauthor=A.+Mantoulidisauthor=L.+J.+Martinauthor=A.+Zoephelauthor=M.+Mayerauthor=A.+Gollnerauthor=D.+Coviniauthor=S.+Fischerauthor=T.+Gerstbergerauthor=T.+Gmaschitzauthor=C.+Goodwinauthor=P.+Grebauthor=D.+H%C3%A4ringauthor=W.+Helaauthor=J.+Hoffmannauthor=J.+Karolyi-Oezguerauthor=P.+Kneslauthor=S.+Korniggauthor=M.+Koeglauthor=R.+Kousekauthor=L.+Lamarreauthor=F.+Moserauthor=S.+Munico-Martinezauthor=C.+Peinsippauthor=J.+Phanauthor=J.+Rinnenthalauthor=J.+Saiauthor=C.+Salamonauthor=Y.+Scherbantinauthor=K.+Schipanyauthor=R.+Schnitzerauthor=A.+Schrenkauthor=B.+Sharpsauthor=G.+Siszlerauthor=Q.+Sunauthor=A.+Watersonauthor=B.+Wolkerstorferauthor=M.+Zeebauthor=M.+Pearsonauthor=S.+W.+Fesikauthor=D.+B.+McConnell&title=Drugging+an+undruggable+pocket+on+KRAS&doi=10.1073%2Fpnas.1904529116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging an undruggable pocket on KRAS</span></div><div class="casAuthors">Kessler, Dirk; Gmachl, Michael; Mantoulidis, Andreas; Martin, Laetitia J.; Zoephel, Andreas; Mayer, Moriz; Gollner, Andreas; Covini, David; Fischer, Silke; Gerstberger, Thomas; Gmaschitz, Teresa; Goodwin, Craig; Greb, Peter; Haring, Daniela; Hela, Wolfgang; Hoffmann, Johann; Karolyi-Oezguer, Jale; Knesl, Petr; Kornigg, Stefan; Koegl, Manfred; Kousek, Roland; Lamarre, Lyne; Moser, Franziska; Munico-Martinez, Silvia; Peinsipp, Christoph; Phan, Jason; Rinnenthal, Jorg; Sai, Jiqing; Salamon, Christian; Scherbantin, Yvonne; Schipany, Katharina; Schnitzer, Renate; Schrenk, Andreas; Sharps, Bernadette; Siszler, Gabriella; Sun, Qi; Waterson, Alex; Wolkerstorfer, Bernhard; Zeeb, Markus; Pearson, Mark; Fesik, Stephen W.; McConnell, Darryl B.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">15823-15829</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">(Imidazolylmethylindolylmethylaminomethyl)indolyl hydroxyisoindolone I, an inhibitor of both the active and inactive forms of KRAS, was discovered using fragment-based screening.  I bound to KRAS in a pocket between switches I and II of KRAS, blocking all GEF, GAP, and effector interactions with KRAS and thus inhibiting downstream signaling; I inhibited KRAS mutant cells at low micromolar concns.  Uniformly 15N-labeled KRAS bound to the stable GTP analog GCP was prepd. with yields of 10 mg/L for use in crystallog. of KRAS-inhibitor complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaMeFg74p2-rVg90H21EOLACvtfcHk0lgkiJeofXSBzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFChs7jJ&md5=ebf443050cb2ce6cbeec1e40084129d0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1904529116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1904529116%26sid%3Dliteratum%253Aachs%26aulast%3DKessler%26aufirst%3DD.%26aulast%3DGmachl%26aufirst%3DM.%26aulast%3DMantoulidis%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DL.%2BJ.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DGollner%26aufirst%3DA.%26aulast%3DCovini%26aufirst%3DD.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DGoodwin%26aufirst%3DC.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DH%25C3%25A4ring%26aufirst%3DD.%26aulast%3DHela%26aufirst%3DW.%26aulast%3DHoffmann%26aufirst%3DJ.%26aulast%3DKarolyi-Oezguer%26aufirst%3DJ.%26aulast%3DKnesl%26aufirst%3DP.%26aulast%3DKornigg%26aufirst%3DS.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DKousek%26aufirst%3DR.%26aulast%3DLamarre%26aufirst%3DL.%26aulast%3DMoser%26aufirst%3DF.%26aulast%3DMunico-Martinez%26aufirst%3DS.%26aulast%3DPeinsipp%26aufirst%3DC.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DSai%26aufirst%3DJ.%26aulast%3DSalamon%26aufirst%3DC.%26aulast%3DScherbantin%26aufirst%3DY.%26aulast%3DSchipany%26aufirst%3DK.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DSchrenk%26aufirst%3DA.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DSiszler%26aufirst%3DG.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DWaterson%26aufirst%3DA.%26aulast%3DWolkerstorfer%26aufirst%3DB.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26atitle%3DDrugging%2520an%2520undruggable%2520pocket%2520on%2520KRAS%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D15823%26epage%3D15829%26doi%3D10.1073%2Fpnas.1904529116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yogodzinski, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shkedi, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, A. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaney, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, L. A.</span></span> <span> </span><span class="NLM_article-title">KRASG12C inhibition produces a driver-limited state revealing collateral dependencies</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">eaaw9450</span> <span class="refDoi"> DOI: 10.1126/scisignal.aaw9450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1126%2Fscisignal.aaw9450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31138768" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&author=K.+Louauthor=V.+Steriauthor=A.+Y.+Geauthor=Y.+C.+Hwangauthor=C.+H.+Yogodzinskiauthor=A.+R.+Shkediauthor=A.+L.+M.+Choiauthor=D.+C.+Mitchellauthor=D.+L.+Swaneyauthor=B.+Hannauthor=J.+D.+Gordanauthor=K.+M.+Shokatauthor=L.+A.+Gilbert&title=KRASG12C+inhibition+produces+a+driver-limited+state+revealing+collateral+dependencies&doi=10.1126%2Fscisignal.aaw9450"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.aaw9450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.aaw9450%26sid%3Dliteratum%253Aachs%26aulast%3DLou%26aufirst%3DK.%26aulast%3DSteri%26aufirst%3DV.%26aulast%3DGe%26aufirst%3DA.%2BY.%26aulast%3DHwang%26aufirst%3DY.%2BC.%26aulast%3DYogodzinski%26aufirst%3DC.%2BH.%26aulast%3DShkedi%26aufirst%3DA.%2BR.%26aulast%3DChoi%26aufirst%3DA.%2BL.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BC.%26aulast%3DSwaney%26aufirst%3DD.%2BL.%26aulast%3DHann%26aufirst%3DB.%26aulast%3DGordan%26aufirst%3DJ.%2BD.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DGilbert%26aufirst%3DL.%2BA.%26atitle%3DKRASG12C%2520inhibition%2520produces%2520a%2520driver-limited%2520state%2520revealing%2520collateral%2520dependencies%26jtitle%3DSci.%2520Signaling%26date%3D2019%26volume%3D12%26doi%3D10.1126%2Fscisignal.aaw9450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROteolysis TArgeting Chimeras (PROTACs)—past, present and future</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.ddtec.2019.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31200855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=15-27&author=M.+Petterssonauthor=C.+M.+Crews&title=PROteolysis+TArgeting+Chimeras+%28PROTACs%29%E2%80%94past%2C+present+and+future&doi=10.1016%2Fj.ddtec.2019.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">PROteolysis TArgeting Chimeras (PROTACs) - Past, present and future</span></div><div class="casAuthors">Pettersson Mariell; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The majority of currently used therapeutics are small molecule-based and utilize occupancy-driven pharmacology as the mode of action (MOA), in which the protein function is modulated via temporary inhibition.  New modalities that operate using alternative MOAs are essential for tapping into the "undruggable" proteome.  The PROteolysis Targeting Chimera (PROTAC) technology provides an attractive new approach that utilizes an event-driven MOA.  Small molecule-based heterobifunctional PROTACs modulate protein target levels by hijacking the ubiquitin-proteasome system to induce degradation of the target.  Here, we address important milestones in the development of the PROTAC technology, as well as emphasize key findings from this previous year and highlight future directions of this promising drug discovery modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRfKb6ELK8t0fW6udTcc2eY28G9AbDKrJLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D&md5=c629f3128321e136f1f247e05ee6f182</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROteolysis%2520TArgeting%2520Chimeras%2520%2528PROTACs%2529%25E2%2580%2594past%252C%2520present%2520and%2520future%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.ddtec.2019.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Backus, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horning, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Paez, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teijaro, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolan, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Proteome-wide covalent ligand discovery in native biological systems</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>534</i></span>,  <span class="NLM_fpage">570</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1038/nature18002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnature18002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27309814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSksbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=570-574&author=K.+M.+Backusauthor=B.+E.+Correiaauthor=K.+M.+Lumauthor=S.+Forliauthor=B.+D.+Horningauthor=G.+E.+Gonzalez-Paezauthor=S.+Chatterjeeauthor=B.+R.+Lanningauthor=J.+R.+Teijaroauthor=A.+J.+Olsonauthor=D.+W.+Wolanauthor=B.+F.+Cravatt&title=Proteome-wide+covalent+ligand+discovery+in+native+biological+systems&doi=10.1038%2Fnature18002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-wide covalent ligand discovery in native biological systems</span></div><div class="casAuthors">Backus, Keriann M.; Correia, Bruno E.; Lum, Kenneth M.; Forli, Stefano; Horning, Benjamin D.; Gonzalez-Paez, Gonzalo E.; Chatterjee, Sandip; Lanning, Bryan R.; Teijaro, John R.; Olson, Arthur J.; Wolan, Dennis W.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7608</span>),
    <span class="NLM_cas:pages">570-574</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small mols. are powerful tools for investigating protein function and can serve as leads for new therapeutics.  Most human proteins, however, lack small-mol. ligands, and entire protein classes are considered 'undruggable'.  Fragment-based ligand discovery can identify small-mol. probes for proteins that have proven difficult to target using high-throughput screening of complex compd. libraries.  Although reversibly binding ligands are commonly pursued, covalent fragments provide an alternative route to small-mol. probes, including those that can access regions of proteins that are difficult to target through binding affinity alone.  Here we report a quant. anal. of cysteine-reactive small-mol. fragments screened against thousands of proteins in human proteomes and cells.  Covalent ligands were identified for >700 cysteines found in both druggable proteins and proteins deficient in chem. probes, including transcription factors, adaptor/scaffolding proteins, and uncharacterized proteins.  Among the atypical ligand-protein interactions discovered were compds. that react preferentially with pro- (inactive) caspases.  We used these ligands to distinguish extrinsic apoptosis pathways in human cell lines vs. primary human T cells, showing that the former is largely mediated by caspase-8 while the latter depends on both caspase-8 and -10.  Fragment-based covalent ligand discovery provides a greatly expanded portrait of the ligandable proteome and furnishes compds. that can illuminate protein functions in native biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgNJ88KALeSLVg90H21EOLACvtfcHk0lg4qymiPGOnyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSksbnN&md5=6fab087c042de735f8a5931e9f5b946c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature18002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18002%26sid%3Dliteratum%253Aachs%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DLum%26aufirst%3DK.%2BM.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DGonzalez-Paez%26aufirst%3DG.%2BE.%26aulast%3DChatterjee%26aufirst%3DS.%26aulast%3DLanning%26aufirst%3DB.%2BR.%26aulast%3DTeijaro%26aufirst%3DJ.%2BR.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DWolan%26aufirst%3DD.%2BW.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DProteome-wide%2520covalent%2520ligand%2520discovery%2520in%2520native%2520biological%2520systems%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D570%26epage%3D574%26doi%3D10.1038%2Fnature18002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galmozzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joslyn, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavallaro, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narvaiza, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Ligand and target discovery by fragment-based screening in human cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>168</i></span>,  <span class="NLM_fpage">527</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.12.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.cell.2016.12.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28111073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVeiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=527-541&author=C.+G.+Parkerauthor=A.+Galmozziauthor=Y.+Wangauthor=B.+E.+Correiaauthor=K.+Sasakiauthor=C.+M.+Joslynauthor=A.+S.+Kimauthor=C.+L.+Cavallaroauthor=R.+M.+Lawrenceauthor=S.+R.+Johnsonauthor=I.+Narvaizaauthor=E.+Saezauthor=B.+F.+Cravatt&title=Ligand+and+target+discovery+by+fragment-based+screening+in+human+cells&doi=10.1016%2Fj.cell.2016.12.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand and Target Discovery by Fragment-Based Screening in Human Cells</span></div><div class="casAuthors">Parker, Christopher G.; Galmozzi, Andrea; Wang, Yujia; Correia, Bruno E.; Sasaki, Kenji; Joslyn, Christopher M.; Kim, Arthur S.; Cavallaro, Cullen L.; Lawrence, R. Michael; Johnson, Stephen R.; Narvaiza, Inigo; Saez, Enrique; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">527-541.e29</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Advances in the synthesis and screening of small-mol. libraries have accelerated the discovery of chem. probes for studying biol. processes.  Still, only a small fraction of the human proteome has chem. ligands.  Here, we describe a platform that marries fragment-based ligand discovery with quant. chem. proteomics to map thousands of reversible small mol.-protein interactions directly in human cells, many of which can be site-specifically detd.  We show that fragment hits can be advanced to furnish selective ligands that affect the activity of proteins heretofore lacking chem. probes.  We further combine fragment-based chem. proteomics with phenotypic screening to identify small mols. that promote adipocyte differentiation by engaging the poorly characterized membrane protein PGRMC2.  Fragment-based screening in human cells thus provides an extensive proteome-wide map of protein ligandability and facilitates the coordinated discovery of bioactive small mols. and their mol. targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPk-YJwFlDwbVg90H21EOLACvtfcHk0lgEaul1hdbxaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVeiu7Y%253D&md5=835264291015037ab9b7e446992caba3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.12.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.12.029%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DC.%2BG.%26aulast%3DGalmozzi%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DSasaki%26aufirst%3DK.%26aulast%3DJoslyn%26aufirst%3DC.%2BM.%26aulast%3DKim%26aufirst%3DA.%2BS.%26aulast%3DCavallaro%26aufirst%3DC.%2BL.%26aulast%3DLawrence%26aufirst%3DR.%2BM.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DNarvaiza%26aufirst%3DI.%26aulast%3DSaez%26aufirst%3DE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DLigand%2520and%2520target%2520discovery%2520by%2520fragment-based%2520screening%2520in%2520human%2520cells%26jtitle%3DCell%26date%3D2017%26volume%3D168%26spage%3D527%26epage%3D541%26doi%3D10.1016%2Fj.cell.2016.12.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Rourke, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, M. S.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of RNA-mediated disease</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">7419</span>– <span class="NLM_lpage">7423</span>, <span class="refDoi"> DOI: 10.1074/jbc.R800025200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1074%2Fjbc.R800025200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=18957432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtVSmsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=7419-7423&author=J.+R.+O%E2%80%99Rourkeauthor=M.+S.+Swanson&title=Mechanisms+of+RNA-mediated+disease&doi=10.1074%2Fjbc.R800025200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of RNA-mediated Disease</span></div><div class="casAuthors">O'Rourke, Jason R.; Swanson, Maurice S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">7419-7423</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Recent mapping of functional sequence elements in the human genome has led to the realization that transcription is pervasive and that noncoding RNAs compose a significant portion of the transcriptome.  Some dominantly inherited neurol. disorders are assocd. with the expansion of microsatellite repeats in noncoding regions that result in the synthesis of pathogenic RNAs.  Here, we review RNA gain-of-function mechanisms underlying three of these microsatellite expansion disorders to illustrate how some mutant RNAs cause disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp32ch1_mjp-7Vg90H21EOLACvtfcHk0lgEaul1hdbxaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtVSmsrk%253D&md5=7a66790b67e63d47b710246b13b2f6ed</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R800025200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R800025200%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DJ.%2BR.%26aulast%3DSwanson%26aufirst%3DM.%2BS.%26atitle%3DMechanisms%2520of%2520RNA-mediated%2520disease%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D7419%26epage%3D7423%26doi%3D10.1074%2Fjbc.R800025200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreyfuss, G.</span></span> <span> </span><span class="NLM_article-title">RNA and disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">777</span>– <span class="NLM_lpage">793</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2009.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.cell.2009.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19239895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvFGktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=777-793&author=T.+A.+Cooperauthor=L.+Wanauthor=G.+Dreyfuss&title=RNA+and+disease&doi=10.1016%2Fj.cell.2009.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">RNA and disease</span></div><div class="casAuthors">Cooper, Thomas A.; Wan, Lili; Dreyfuss, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">777-793</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Cellular functions depend on numerous protein-coding and noncoding RNAs and the RNA-binding proteins assocd. with them, which form ribonucleoprotein complexes (RNPs).  Mutations that disrupt either the RNA or protein components of RNPs or the factors required for their assembly can be deleterious.  Alternative splicing provides cells with an exquisite capacity to fine-tune their transcriptome and proteome in response to cues.  Splicing depends on a complex code, numerous RNA-binding proteins, and an enormously intricate network of interactions among them, increasing the opportunity for exposure to mutations and misregulation that cause disease.  The discovery of disease-causing mutations in RNAs is yielding a wealth of new therapeutic targets, and the growing understanding of RNA biol. and chem. is providing new RNA-based tools for developing therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotOMwPdFvVpLVg90H21EOLACvtfcHk0lgEaul1hdbxaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvFGktrw%253D&md5=f165df2b33f97df6e895045b81e853d4</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DCooper%26aufirst%3DT.%2BA.%26aulast%3DWan%26aufirst%3DL.%26aulast%3DDreyfuss%26aufirst%3DG.%26atitle%3DRNA%2520and%2520disease%26jtitle%3DCell%26date%3D2009%26volume%3D136%26spage%3D777%26epage%3D793%26doi%3D10.1016%2Fj.cell.2009.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sztuba-Solinska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavez-Calvillo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cline, S. E.</span></span> <span> </span><span class="NLM_article-title">Unveiling the druggable RNA targets and small molecule therapeutics</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2149</span>– <span class="NLM_lpage">2165</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2019.03.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.bmc.2019.03.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30981606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntFCiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=2149-2165&author=J.+Sztuba-Solinskaauthor=G.+Chavez-Calvilloauthor=S.+E.+Cline&title=Unveiling+the+druggable+RNA+targets+and+small+molecule+therapeutics&doi=10.1016%2Fj.bmc.2019.03.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Unveiling the druggable RNA targets and small molecule therapeutics</span></div><div class="casAuthors">Sztuba-Solinska, Joanna; Chavez-Calvillo, Gabriela; Cline, Sabrina Elizabeth</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2149-2165</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The increasing appreciation for the crucial roles of RNAs in infectious and non-infectious human diseases makes them attractive therapeutic targets.  Coding and non-coding RNAs frequently fold into complex conformations which, if effectively targeted, offer opportunities to therapeutically modulate numerous cellular processes, including those linked to undruggable protein targets.  Despite the considerable skepticism as to whether RNAs can be targeted with small mol. therapeutics, overwhelming evidence suggests the challenges we are currently facing are not outside the realm of possibility.  In this review, we highlight the most recent advances in mol. techniques that have sparked a revolution in understanding the RNA structure-to-function relationship.  We bring attention to the application of these modern techniques to identify druggable RNA targets and to assess small mol. binding specificity.  Finally, we discuss novel screening methodologies that support RNA drug discovery and present examples of therapeutically valuable RNA targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ_ZttV3TINLVg90H21EOLACvtfcHk0lgEaul1hdbxaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntFCiu74%253D&md5=5c2197acf2f0a04f6fda7711fb3cc0a9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.03.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.03.057%26sid%3Dliteratum%253Aachs%26aulast%3DSztuba-Solinska%26aufirst%3DJ.%26aulast%3DChavez-Calvillo%26aufirst%3DG.%26aulast%3DCline%26aufirst%3DS.%2BE.%26atitle%3DUnveiling%2520the%2520druggable%2520RNA%2520targets%2520and%2520small%2520molecule%2520therapeutics%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D2149%26epage%3D2165%26doi%3D10.1016%2Fj.bmc.2019.03.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ursu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vezina-Dawod, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Methods to identify and optimize small molecules interacting with RNA (SMIRNAs)</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2002</span>– <span class="NLM_lpage">2016</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2019.06.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.drudis.2019.06.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31356880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFWjt7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=2002-2016&author=A.+Ursuauthor=S.+Vezina-Dawodauthor=M.+D.+Disney&title=Methods+to+identify+and+optimize+small+molecules+interacting+with+RNA+%28SMIRNAs%29&doi=10.1016%2Fj.drudis.2019.06.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Methods to identify and optimize small molecules interacting with RNA (SMIRNAs)</span></div><div class="casAuthors">Ursu, Andrei; Vezina-Dawod, Simon; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2002-2016</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  RNAs, particularly noncoding RNAs (ncRNAs), are becoming increasingly important therapeutic targets, because they are causative and antagonists of human disease.  Indeed, aberrant RNA structural elements and expression deregulate biol. processes.  In this review, describe methodologies to discover and optimize small mols. interacting with RNA (SMIRNAs), including the evaluation of direct target engagement and the rescue of RNA-mediated phenotypes in vitro and in vivo.  Such studies are essential to fully characterize the mode of action of SMIRNAs and advance our understanding of rationally and efficiently drugging RNAs for therapeutic benefit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRFrcfvOwE9bVg90H21EOLACvtfcHk0lhztxmnbTrUrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFWjt7nP&md5=8f5a62ec3259cf0f1577c35b1243633b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2019.06.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2019.06.019%26sid%3Dliteratum%253Aachs%26aulast%3DUrsu%26aufirst%3DA.%26aulast%3DVezina-Dawod%26aufirst%3DS.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DMethods%2520to%2520identify%2520and%2520optimize%2520small%2520molecules%2520interacting%2520with%2520RNA%2520%2528SMIRNAs%2529%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26spage%3D2002%26epage%3D2016%26doi%3D10.1016%2Fj.drudis.2019.06.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rizvi, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. F.</span></span> <span> </span><span class="NLM_article-title">RNA as a small molecule druggable target</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">5083</span>– <span class="NLM_lpage">5088</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.10.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.bmcl.2017.10.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29097169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslemtLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=5083-5088&author=N.+F.+Rizviauthor=G.+F.+Smith&title=RNA+as+a+small+molecule+druggable+target&doi=10.1016%2Fj.bmcl.2017.10.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">RNA as a small molecule druggable target</span></div><div class="casAuthors">Rizvi, Noreen F.; Smith, Graham F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5083-5088</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Small mol. drugs have readily been developed against many proteins in the human proteome, but RNA has remained an elusive target for drug discovery.  Increasingly, the authors see that RNA, and to a lesser extent DNA elements, show a persistent tertiary structure responsible for many diverse and complex cellular functions.  In this digest, the authors have summarized recent advances in screening approaches for RNA targets and outlined the discovery of novel, drug-like small mols. against RNA targets from various classes and therapeutic areas.  The link of structure, function, and small-mol. Druggability validates now for the first time that RNA can be the targets of therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQHvz_K9pGLVg90H21EOLACvtfcHk0lhztxmnbTrUrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslemtLfF&md5=ba213f6e8954c97239ccc23f7096c13e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.10.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.10.052%26sid%3Dliteratum%253Aachs%26aulast%3DRizvi%26aufirst%3DN.%2BF.%26aulast%3DSmith%26aufirst%3DG.%2BF.%26atitle%3DRNA%2520as%2520a%2520small%2520molecule%2520druggable%2520target%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D5083%26epage%3D5088%26doi%3D10.1016%2Fj.bmcl.2017.10.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Giorgio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span> <span> </span><span class="NLM_article-title">Synthetic small-molecule RNA ligands: future prospects as therapeutic agents</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1242</span>– <span class="NLM_lpage">1255</span>, <span class="refDoi"> DOI: 10.1039/C9MD00195F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1039%2FC9MD00195F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31534649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1ShurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1242-1255&author=A.+Di%0AGiorgioauthor=M.+Duca&title=Synthetic+small-molecule+RNA+ligands%3A+future+prospects+as+therapeutic+agents&doi=10.1039%2FC9MD00195F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic small-molecule RNA ligands: future prospects as therapeutic agents</span></div><div class="casAuthors">Di Giorgio, A.; Duca, M.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1242-1255</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">RNA is one of the most intriguing and promising biol. targets for the discovery of innovative drugs in many pathologies and various biol. relevant RNAs that could serve as drug targets have already been identified.  Among the most important ones, one can mention prokaryotic rRNA which is the target of several marketed antibiotics, viral RNAs or oncogenic microRNAs that are tightly involved in the development and progression of various cancers.  Oligonucleotides are efficient and specific RNA targeting agents but suffer from poor pharmacodynamic and pharmacokinetic properties.  For this reason, a no. of synthetic small-mol. ligands have been identified and studied upon screening of chem. libraries or focused design of RNA binders.  In this review, we report the most relevant examples of synthetic compds. bearing sufficient selectivity to envisage clin. studies and future therapeutic applications with a particular attention for the main strategies that can be undertaken toward the improvement of selectivity and biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-wwNyCsBkOLVg90H21EOLACvtfcHk0lhztxmnbTrUrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1ShurY%253D&md5=60f47cc3df704517d5c308ebbe2cae6b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC9MD00195F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00195F%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BGiorgio%26aufirst%3DA.%26aulast%3DDuca%26aufirst%3DM.%26atitle%3DSynthetic%2520small-molecule%2520RNA%2520ligands%253A%2520future%2520prospects%2520as%2520therapeutic%2520agents%26jtitle%3DMedChemComm%26date%3D2019%26volume%3D10%26spage%3D1242%26epage%3D1255%26doi%3D10.1039%2FC9MD00195F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forte, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargrove, A. E.</span></span> <span> </span><span class="NLM_article-title">Insights into the development of chemical probes for RNA</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">8025</span>– <span class="NLM_lpage">8037</span>, <span class="refDoi"> DOI: 10.1093/nar/gky718</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1093%2Fnar%2Fgky718" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30102391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=8025-8037&author=B.+S.+Morganauthor=J.+E.+Forteauthor=A.+E.+Hargrove&title=Insights+into+the+development+of+chemical+probes+for+RNA&doi=10.1093%2Fnar%2Fgky718"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the development of chemical probes for RNA</span></div><div class="casAuthors">Morgan, Brittany S.; Forte, Jordan E.; Hargrove, Amanda E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8025-8037</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Over the past decade, the RNA revolution has revealed thousands of non-coding RNAs that are essential for cellular regulation and are misregulated in disease.  While the development of methods and tools to study these RNAs has been challenging, the power and promise of small mol. chem. probes is increasingly recognized.  To harness existing knowledge, we compiled a list of 116 ligands with reported activity against RNA targets in biol. systems (R-BIND).  In this survey, we examine the RNA targets, design and discovery strategies, and chem. probe characterization techniques of these ligands.  We discuss the applicability of current tools to identify and evaluate RNA-targeted chem. probes, suggest criteria to assess the quality of RNA chem. probes and targets, and propose areas where new tools are particularly needed.  We anticipate that this knowledge will expedite the discovery of RNA-targeted ligands and the next phase of the RNA revolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBbTlYBW7qtrVg90H21EOLACvtfcHk0li0fb4viA6BEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosFansLk%253D&md5=69a67e0be1045cd8731a23b6e9d353f7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky718%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DB.%2BS.%26aulast%3DForte%26aufirst%3DJ.%2BE.%26aulast%3DHargrove%26aufirst%3DA.%2BE.%26atitle%3DInsights%2520into%2520the%2520development%2520of%2520chemical%2520probes%2520for%2520RNA%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3D8025%26epage%3D8037%26doi%3D10.1093%2Fnar%2Fgky718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warner, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajdin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weeks, K. M.</span></span> <span> </span><span class="NLM_article-title">Principles for targeting RNA with drug-like small molecules</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrd.2018.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29977051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1OntLnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=547-558&author=K.+D.+Warnerauthor=C.+E.+Hajdinauthor=K.+M.+Weeks&title=Principles+for+targeting+RNA+with+drug-like+small+molecules&doi=10.1038%2Fnrd.2018.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Principles for targeting RNA with drug-like small molecules</span></div><div class="casAuthors">Warner, Katherine Deigan; Hajdin, Christine E.; Weeks, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">547-558</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">RNA mols. are essential for cellular information transfer and gene regulation, and RNAs have been implicated in many human diseases.  Messenger and non-coding RNAs contain highly structured elements, and evidence suggests that many of these structures are important for function.  Targeting these RNAs with small mols. offers opportunities to therapeutically modulate numerous cellular processes, including those linked to undruggable protein targets.  Despite this promise, there is currently only a single class of human-designed small mols. that target RNA used clin. - the linezolid antibiotics.  However, a growing no. of small-mol. RNA ligands are being identified, leading to burgeoning interest in the field.  Here, we discuss principles for discovering small-mol. drugs that target RNA and argue that the overarching challenge is to identify appropriate target structures - namely, in disease-causing RNAs that have high information content and, consequently, appropriate ligand-binding pockets.  If focus is placed on such druggable binding sites in RNA, extensive knowledge of the typical physicochem. properties of drug-like small mols. could then enable small-mol. drug discovery for RNA targets to become (only) roughly as difficult as for protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYl_OguWzasrVg90H21EOLACvtfcHk0li0fb4viA6BEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1OntLnL&md5=df01185ded43db426749b9c656a04f64</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.93%26sid%3Dliteratum%253Aachs%26aulast%3DWarner%26aufirst%3DK.%2BD.%26aulast%3DHajdin%26aufirst%3DC.%2BE.%26aulast%3DWeeks%26aufirst%3DK.%2BM.%26atitle%3DPrinciples%2520for%2520targeting%2520RNA%2520with%2520drug-like%2520small%2520molecules%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D547%26epage%3D558%26doi%3D10.1038%2Fnrd.2018.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Begg, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barclay, M. L.</span></span> <span> </span><span class="NLM_article-title">Aminoglycosides - 50 years on</span>. <i>Br. J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">597</span>– <span class="NLM_lpage">603</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1111%2Fj.1365-2125.1995.tb05719.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=7654476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADyaK2MzotlSktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1995&pages=597-603&author=E.+J.+Beggauthor=M.+L.+Barclay&title=Aminoglycosides+-+50+years+on"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Aminoglycosides--50 years on</span></div><div class="casAuthors">Begg E J; Barclay M L</div><div class="citationInfo"><span class="NLM_cas:title">British journal of clinical pharmacology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">597-603</span>
        ISSN:<span class="NLM_cas:issn">0306-5251</span>.
    </div><div class="casAbstract">1.  The aminoglycoside antibiotics are 50 years old.  Their success and continuing use can be attributed to various factors including rapid concentration-dependent bactericidal effect, synergism with beta-lactam antibiotics, clinical effectiveness, a low rate of true resistance and low cost. 2.  The aminoglycosides remain drugs of choice in many circumstances including septicaemia, other serious infections due to Gram negative bacilli, and bacterial endocarditis. 3.  Nephrotoxicity and ototoxicity have been the main drawbacks clinically for the aminoglycosides. 4.  There has been an evolution in dosing strategies largely aimed at reducing toxicity.  Therapeutic drug monitoring has been used extensively to assist dosing, and target concentrations have been advocated, such as peak concentrations of between 6 and 10 mg l-1 and through concentrations of < 2 mg l-1 for gentamicin, tobramycin and netilmicin. 5.  Recently there has been a minor revolution in the approach to aminoglycoside dosing, with a change to larger doses, given less frequently.  In its most convenient form this is 'Once-daily aminoglycoside dosing'.  It offers the hope of better efficacy, less toxicity, and easier administration and monitoring. 6.  This article summarises the background of aminoglycoside usage, leading up to the recent changes in dosing strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR1Gxp5vfmgRW52AMpI4GK2fW6udTcc2eae2PgYtKMn_bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2MzotlSktw%253D%253D&md5=3e122c24a54f582a7ea1a26972519f4e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2125.1995.tb05719.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2125.1995.tb05719.x%26sid%3Dliteratum%253Aachs%26aulast%3DBegg%26aufirst%3DE.%2BJ.%26aulast%3DBarclay%26aufirst%3DM.%2BL.%26atitle%3DAminoglycosides%2520-%252050%2520years%2520on%26jtitle%3DBr.%2520J.%2520Clin.%2520Pharmacol.%26date%3D1995%26volume%3D39%26spage%3D597%26epage%3D603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labuda, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poplawski, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pushechnikov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span> <span> </span><span class="NLM_article-title">Two-dimensional combinatorial screening identifies specific aminoglycoside-RNA internal loop partners</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">11185</span>– <span class="NLM_lpage">11194</span>, <span class="refDoi"> DOI: 10.1021/ja803234t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja803234t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovFygu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=11185-11194&author=M.+D.+Disneyauthor=L.+P.+Labudaauthor=D.+J.+Paulauthor=S.+G.+Poplawskiauthor=A.+Pushechnikovauthor=T.+Tranauthor=S.+P.+Velagapudiauthor=M.+Wuauthor=J.+L.+Childs-Disney&title=Two-dimensional+combinatorial+screening+identifies+specific+aminoglycoside-RNA+internal+loop+partners&doi=10.1021%2Fja803234t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Two-Dimensional Combinatorial Screening Identifies Specific Aminoglycoside-RNA Internal Loop Partners</span></div><div class="casAuthors">Disney, Matthew D.; Labuda, Lucas P.; Paul, Dustin J.; Poplawski, Shane G.; Pushechnikov, Alexei; Tran, Tuan; Velagapudi, Sai P.; Wu, Meilan; Childs-Disney, Jessica L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">11185-11194</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein is described the identification of RNA internal loops that bind to derivs. of neomycin B, neamine, tobramycin, and kanamycin A.  RNA loop-ligand partners were identified by a two-dimensional combinatorial screening (2DCS) platform that probes RNA and chem. spaces simultaneously.  In 2DCS, an aminoglycoside library immobilized onto an agarose microarray was probed for binding to a 3 × 3 nucleotide RNA internal loop library (81 920 interactions probed in duplicate in a single expt.).  RNAs that bound aminoglycosides were harvested from the array via gel excision.  RNA internal loop preferences for three aminoglycosides were identified from statistical anal. of selected structures.  This provides consensus RNA internal loops that bind these structures and include: loops with potential GA pairs for the neomycin deriv., loops with potential GG pairs for the tobramycin deriv., and pyrimidine-rich loops for the kanamycin A deriv.  Results with the neamine deriv. show that it binds a variety of loops, including loops that contain potential GA pairs that also recognize the neomycin B deriv.  All studied selected internal loops are specific for the aminoglycoside that they were selected to bind.  Specificity was quantified for 16 selected internal loops by studying their binding to each of the arrayed aminoglycosides.  Specificities ranged from 2- to 80-fold with an av. specificity of 20-fold.  These studies show that 2DCS is a unique platform to probe RNA and chem. space simultaneously to identify specific RNA motif-ligand interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDfkvgnM2Yh7Vg90H21EOLACvtfcHk0ljJG_m9L28EvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovFygu7Y%253D&md5=9910fb8215768c6ccda8a771a5d98ed4</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja803234t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja803234t%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DLabuda%26aufirst%3DL.%2BP.%26aulast%3DPaul%26aufirst%3DD.%2BJ.%26aulast%3DPoplawski%26aufirst%3DS.%2BG.%26aulast%3DPushechnikov%26aufirst%3DA.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26atitle%3DTwo-dimensional%2520combinatorial%2520screening%2520identifies%2520specific%2520aminoglycoside-RNA%2520internal%2520loop%2520partners%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D11185%26epage%3D11194%26doi%3D10.1021%2Fja803234t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Datlinger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendeiro, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krausgruber, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klughammer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alpar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, C.</span></span> <span> </span><span class="NLM_article-title">Pooled CRISPR screening with single-cell transcriptome readout</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1038/nmeth.4177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnmeth.4177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28099430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKjsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=297-301&author=P.+Datlingerauthor=A.+F.+Rendeiroauthor=C.+Schmidlauthor=T.+Krausgruberauthor=P.+Traxlerauthor=J.+Klughammerauthor=L.+C.+Schusterauthor=A.+Kuchlerauthor=D.+Alparauthor=C.+Bock&title=Pooled+CRISPR+screening+with+single-cell+transcriptome+readout&doi=10.1038%2Fnmeth.4177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Pooled CRISPR screening with single-cell transcriptome readout</span></div><div class="casAuthors">Datlinger, Paul; Rendeiro, Andre F.; Schmidl, Christian; Krausgruber, Thomas; Traxler, Peter; Klughammer, Johanna; Schuster, Linda C.; Kuchler, Amelie; Alpar, Donat; Bock, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">297-301</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">CRISPR-based genetic screens are accelerating biol. discovery, but current methods have inherent limitations.  Widely used pooled screens are restricted to simple readouts including cell proliferation and sortable marker proteins.  Arrayed screens allow for comprehensive mol. readouts such as transcriptome profiling, but at much lower throughput.  Here we combine pooled CRISPR screening with single-cell RNA sequencing into a broadly applicable workflow, directly linking guide RNA expression to transcriptome responses in thousands of individual cells.  Our method for CRISPR droplet sequencing (CROP-seq) enables pooled CRISPR screens with single-cell transcriptome resoln., which will facilitate high-throughput functional dissection of complex regulatory mechanisms and heterogeneous cell populations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGg_xnPKp1wbVg90H21EOLACvtfcHk0ljJG_m9L28EvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKjsrk%253D&md5=5c32085acdcee491f18b47149713a43d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.4177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.4177%26sid%3Dliteratum%253Aachs%26aulast%3DDatlinger%26aufirst%3DP.%26aulast%3DRendeiro%26aufirst%3DA.%2BF.%26aulast%3DSchmidl%26aufirst%3DC.%26aulast%3DKrausgruber%26aufirst%3DT.%26aulast%3DTraxler%26aufirst%3DP.%26aulast%3DKlughammer%26aufirst%3DJ.%26aulast%3DSchuster%26aufirst%3DL.%2BC.%26aulast%3DKuchler%26aufirst%3DA.%26aulast%3DAlpar%26aufirst%3DD.%26aulast%3DBock%26aufirst%3DC.%26atitle%3DPooled%2520CRISPR%2520screening%2520with%2520single-cell%2520transcriptome%2520readout%26jtitle%3DNat.%2520Methods%26date%3D2017%26volume%3D14%26spage%3D297%26epage%3D301%26doi%3D10.1038%2Fnmeth.4177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">Building mammalian signalling pathways with RNAi screens</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1038/nrm1860</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrm1860" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=16496020" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhs1Kisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=177-187&author=J.+Moffatauthor=D.+M.+Sabatini&title=Building+mammalian+signalling+pathways+with+RNAi+screens&doi=10.1038%2Fnrm1860"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Building mammalian signalling pathways with RNAi screens</span></div><div class="casAuthors">Moffat, Jason; Sabatini, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">177-187</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Technol. advances in mammalian systems are providing new tools to identify the mol. components of signaling pathways.  Foremost among these tools is the ability to knock down gene function through the use of RNA interference (RNAi).  The fact that RNAi can be scaled up for use in high-throughput techniques has motivated the creation of genome-wide RNAi reagents.  We are now at the brink of being able to harness the power of RNAi for large-scale functional discovery in mammalian cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFu-CHw0OiGrVg90H21EOLACvtfcHk0ljJG_m9L28EvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhs1Kisrg%253D&md5=f042bfc5a7c6091d14ed54a25e9d7cf7</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnrm1860&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1860%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DBuilding%2520mammalian%2520signalling%2520pathways%2520with%2520RNAi%2520screens%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2006%26volume%3D7%26spage%3D177%26epage%3D187%26doi%3D10.1038%2Fnrm1860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howe, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmann, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balibar, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galgoci, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malinverni, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayhood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villafania, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahvi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murgolo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbieri, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Painter, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherborne, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jesus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotkin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rindgen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlisi, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheth, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roemer, T.</span></span> <span> </span><span class="NLM_article-title">Selective small-molecule inhibition of an RNA structural element</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>526</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">677</span>, <span class="refDoi"> DOI: 10.1038/nature15542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnature15542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=26416753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=526&publication_year=2015&pages=672-677&author=J.+A.+Howeauthor=H.+Wangauthor=T.+O.+Fischmannauthor=C.+J.+Balibarauthor=L.+Xiaoauthor=A.+M.+Galgociauthor=J.+C.+Malinverniauthor=T.+Mayhoodauthor=A.+Villafaniaauthor=A.+Nahviauthor=N.+Murgoloauthor=C.+M.+Barbieriauthor=P.+A.+Mannauthor=D.+Carrauthor=E.+Xiaauthor=P.+Zuckauthor=D.+Rileyauthor=R.+E.+Painterauthor=S.+S.+Walkerauthor=B.+Sherborneauthor=R.+de+Jesusauthor=W.+Panauthor=M.+A.+Plotkinauthor=J.+Wuauthor=D.+Rindgenauthor=J.+Cummingsauthor=C.+G.+Garlisiauthor=R.+Zhangauthor=P.+R.+Shethauthor=C.+J.+Gillauthor=H.+Tangauthor=T.+Roemer&title=Selective+small-molecule+inhibition+of+an+RNA+structural+element&doi=10.1038%2Fnature15542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-molecule inhibition of an RNA structural element</span></div><div class="casAuthors">Howe, John A.; Wang, Hao; Fischmann, Thierry O.; Balibar, Carl J.; Xiao, Li; Galgoci, Andrew M.; Malinverni, Juliana C.; Mayhood, Todd; Villafania, Artjohn; Nahvi, Ali; Murgolo, Nicholas; Barbieri, Christopher M.; Mann, Paul A.; Carr, Donna; Xia, Ellen; Zuck, Paul; Riley, Dan; Painter, Ronald E.; Walker, Scott S.; Sherborne, Brad; de Jesus, Reynalda; Pan, Weidong; Plotkin, Michael A.; Wu, Jin; Rindgen, Diane; Cummings, John; Garlisi, Charles G.; Zhang, Rumin; Sheth, Payal R.; Gill, Charles J.; Tang, Haifeng; Roemer, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">526</span>
        (<span class="NLM_cas:issue">7575</span>),
    <span class="NLM_cas:pages">672-677</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Riboswitches are non-coding RNA structures located in mRNAs that bind endogenous ligands, such as a specific metabolite or ion, to regulate gene expression.  As such, riboswitches serve as a novel, yet largely unexploited, class of emerging drug targets.  Demonstrating this potential, however, has proven difficult and is restricted to structurally similar antimetabolites and semi-synthetic analogs of their cognate ligand, thus greatly restricting the chem. space and selectivity sought for such inhibitors.  Here we report the discovery and characterization of ribocil, a highly selective chem. modulator of bacterial riboflavin riboswitches, which was identified in a phenotypic screen and acts as a structurally distinct synthetic mimic of the natural ligand, FMN, to repress riboswitch-mediated ribB gene expression and inhibit bacterial cell growth.  Our findings indicate that non-coding RNA structural elements may be more broadly targeted by synthetic small mols. than previously expected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWptaVLK3njLVg90H21EOLACvtfcHk0liZ7HVKxvYpQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7%252FL&md5=c0f8443348c10f69eb47f05a2ac5e893</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnature15542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15542%26sid%3Dliteratum%253Aachs%26aulast%3DHowe%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DFischmann%26aufirst%3DT.%2BO.%26aulast%3DBalibar%26aufirst%3DC.%2BJ.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DGalgoci%26aufirst%3DA.%2BM.%26aulast%3DMalinverni%26aufirst%3DJ.%2BC.%26aulast%3DMayhood%26aufirst%3DT.%26aulast%3DVillafania%26aufirst%3DA.%26aulast%3DNahvi%26aufirst%3DA.%26aulast%3DMurgolo%26aufirst%3DN.%26aulast%3DBarbieri%26aufirst%3DC.%2BM.%26aulast%3DMann%26aufirst%3DP.%2BA.%26aulast%3DCarr%26aufirst%3DD.%26aulast%3DXia%26aufirst%3DE.%26aulast%3DZuck%26aufirst%3DP.%26aulast%3DRiley%26aufirst%3DD.%26aulast%3DPainter%26aufirst%3DR.%2BE.%26aulast%3DWalker%26aufirst%3DS.%2BS.%26aulast%3DSherborne%26aufirst%3DB.%26aulast%3Dde%2BJesus%26aufirst%3DR.%26aulast%3DPan%26aufirst%3DW.%26aulast%3DPlotkin%26aufirst%3DM.%2BA.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DRindgen%26aufirst%3DD.%26aulast%3DCummings%26aufirst%3DJ.%26aulast%3DGarlisi%26aufirst%3DC.%2BG.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DSheth%26aufirst%3DP.%2BR.%26aulast%3DGill%26aufirst%3DC.%2BJ.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DRoemer%26aufirst%3DT.%26atitle%3DSelective%2520small-molecule%2520inhibition%2520of%2520an%2520RNA%2520structural%2520element%26jtitle%3DNature%26date%3D2015%26volume%3D526%26spage%3D672%26epage%3D677%26doi%3D10.1038%2Fnature15542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeFrancesco, L.</span></span> <span> </span><span class="NLM_article-title">Drug pipeline: 1Q17</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">400</span>, <span class="refDoi"> DOI: 10.1038/nbt.3874</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnbt.3874" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28486449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC1crlsFSnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=400&author=L.+DeFrancesco&title=Drug+pipeline%3A+1Q17&doi=10.1038%2Fnbt.3874"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Drug pipeline: 1Q17</span></div><div class="casAuthors">DeFrancesco Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature biotechnology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">400</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTg0zjb7i0zCHqj_yg7twvpfW6udTcc2ebJ3IykwyrJpbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crlsFSnsA%253D%253D&md5=0bf4b76fe97b1316e0312ba39455813c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3874&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3874%26sid%3Dliteratum%253Aachs%26aulast%3DDeFrancesco%26aufirst%3DL.%26atitle%3DDrug%2520pipeline%253A%25201Q17%26jtitle%3DNat.%2520Biotechnol.%26date%3D2017%26volume%3D35%26spage%3D400%26doi%3D10.1038%2Fnbt.3874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torricelli, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkel, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanigan, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goemans, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heydemann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osorio, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schara, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sejersen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shieh, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweeney, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topaloglu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tulinius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilchez, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elfring, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntosh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riebling, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiegel, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peltz, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercuri, E.</span></span> <span> </span><span class="NLM_article-title">Ataluren in patients with nonsense mutation Duchenne muscular dystrophy (ACT DMD): a multicentre, randomised, double-blind, placebo-controlled, phase 3 trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">1489</span>– <span class="NLM_lpage">1498</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(17)31611-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2FS0140-6736%2817%2931611-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28728956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2hsL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2017&pages=1489-1498&author=C.+M.+McDonaldauthor=C.+Campbellauthor=R.+E.+Torricelliauthor=R.+S.+Finkelauthor=K.+M.+Flaniganauthor=N.+Goemansauthor=P.+Heydemannauthor=A.+Kaminskaauthor=J.+Kirschnerauthor=F.+Muntoniauthor=A.+N.+Osorioauthor=U.+Scharaauthor=T.+Sejersenauthor=P.+B.+Shiehauthor=H.+L.+Sweeneyauthor=H.+Topalogluauthor=M.+Tuliniusauthor=J.+J.+Vilchezauthor=T.+Voitauthor=B.+Wongauthor=G.+Elfringauthor=H.+Krogerauthor=X.+Luoauthor=J.+McIntoshauthor=T.+Ongauthor=P.+Rieblingauthor=M.+Souzaauthor=R.+J.+Spiegelauthor=S.+W.+Peltzauthor=E.+Mercuri&title=Ataluren+in+patients+with+nonsense+mutation+Duchenne+muscular+dystrophy+%28ACT+DMD%29%3A+a+multicentre%2C+randomised%2C+double-blind%2C+placebo-controlled%2C+phase+3+trial&doi=10.1016%2FS0140-6736%2817%2931611-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Ataluren in patients with nonsense mutation Duchenne muscular dystrophy (ACT DMD): a multicentre, randomised, double-blind, placebo-controlled, phase 3 trial</span></div><div class="casAuthors">McDonald, Craig M.; Campbell, Craig; Torricelli, Ricardo Erazo; Finkel, Richard S.; Flanigan, Kevin M.; Goemans, Nathalie; Heydemann, Peter; Kaminska, Anna; Kirschner, Janbernd; Muntoni, Francesco; Osorio, Andres Nascimento; Schara, Ulrike; Sejersen, Thomas; Shieh, Perry B.; Sweeney, H. Lee; Topaloglu, Haluk; Tulinius, Mar; Vilchez, Juan J.; Voit, Thomas; Wong, Brenda; Elfring, Gary; Kroger, Hans; Luo, Xiaohui; McIntosh, Joseph; Ong, Tuyen; Riebling, Peter; Souza, Marcio; Spiegel, Robert J.; Peltz, Stuart W.; Mercuri, Eugenio</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">10101</span>),
    <span class="NLM_cas:pages">1489-1498</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Duchenne muscular dystrophy (DMD) is a severe, progressive, and rare neuromuscular, X-linked recessive disease.  Dystrophin deficiency is the underlying cause of disease; therefore, mutation-specific therapies aimed at restoring dystrophin protein prodn. are being explored.  We aimed to assess the efficacy and safety of ataluren in ambulatory boys with nonsense mutation DMD.  We did this multicentre, randomised, double-blind, placebo-controlled, phase 3 trial at 54 sites in 18 countries located in North America, Europe, the Asia-Pacific region, and Latin America.  Boys aged 7-16 years with nonsense mutation DMD and a baseline 6-min walk distance (6MWD) of 150 m or more and 80% or less of the predicted normal value for age and height were randomly assigned (1:1), via permuted block randomisation (block size of four) using an interactive voice-response or web-response system, to receive ataluren orally three times daily (40 mg/kg per day) or matching placebo.  Randomisation was stratified by age (<9 years vs ≥9 years), duration of previous corticosteroid use (6 mo to <12 mo vs ≥12 mo), and baseline 6MWD (<350 m vs ≥350 m).  Patients, parents and caregivers, investigational site personnel, PTC Therapeutics employees, and all other study personnel were masked to group allocation until after database lock.  The primary endpoint was change in 6MWD from baseline to week 48.  We addnl. did a prespecified subgroup anal. of the primary endpoint, based on baseline 6MWD, which is reflective of anticipated rates of disease progression over 1 yr.  The primary anal. was by intention to treat.  This study is registered with ClinicalTrials.gov, no. NCT01826487.  Between March 26, 2013, and Aug 26, 2014, we randomly assigned 230 patients to receive ataluren (n=115) or placebo (n=115); 228 patients comprised the intention-to-treat population.  The least-squares mean change in 6MWD from baseline to week 48 was -47·7 m (SE 9·3) for ataluren-treated patients and -60·7 m (9·3) for placebo-treated patients (difference 13·0 m [SE 10·4], 95% CI -7·4 to 33·4; p=0·213).  The least-squares mean change for ataluren vs. placebo in the prespecified subgroups was -7·7 m (SE 24·1, 95% CI -54·9 to 39·5; p=0·749) in the group with a 6MWD of less than 300 m, 42·9 m (15·9, 11·8-74·0; p=0·007) in the group with a 6MWD of 300 m or more to less than 400 m, and -9·5 m (17·2, -43·2 to 24·2; p=0·580) in the group with a 6MWD of 400 m or more.  Ataluren was generally well tolerated and most treatment-emergent adverse events were mild to moderate in severity.  Eight (3%) patients (n=4 per group) reported serious adverse events; all except one event in the placebo group (abnormal hepatic function deemed possibly related to treatment) were deemed unrelated to treatment.  Change in 6MWD did not differ significantly between patients in the ataluren group and those in the placebo group, neither in the intention-to-treat population nor in the prespecified subgroups with a baseline 6MWD of less than 300 m or 400 m or more.  However, we recorded a significant effect of ataluren in the prespecified subgroup of patients with a baseline 6MWD of 300 m or more to less than 400 m.  Baseline 6MWD values within this range were assocd. with a more predictable rate of decline over 1 yr; this finding has implications for the design of future DMD trials with the 6-min walk test as the endpoint.PTC Therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrdEeegEaHJLVg90H21EOLACvtfcHk0lj9kdJ2aN-suA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2hsL3N&md5=25ae69e359455aca9d716e964aa09015</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2817%2931611-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252817%252931611-2%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DC.%2BM.%26aulast%3DCampbell%26aufirst%3DC.%26aulast%3DTorricelli%26aufirst%3DR.%2BE.%26aulast%3DFinkel%26aufirst%3DR.%2BS.%26aulast%3DFlanigan%26aufirst%3DK.%2BM.%26aulast%3DGoemans%26aufirst%3DN.%26aulast%3DHeydemann%26aufirst%3DP.%26aulast%3DKaminska%26aufirst%3DA.%26aulast%3DKirschner%26aufirst%3DJ.%26aulast%3DMuntoni%26aufirst%3DF.%26aulast%3DOsorio%26aufirst%3DA.%2BN.%26aulast%3DSchara%26aufirst%3DU.%26aulast%3DSejersen%26aufirst%3DT.%26aulast%3DShieh%26aufirst%3DP.%2BB.%26aulast%3DSweeney%26aufirst%3DH.%2BL.%26aulast%3DTopaloglu%26aufirst%3DH.%26aulast%3DTulinius%26aufirst%3DM.%26aulast%3DVilchez%26aufirst%3DJ.%2BJ.%26aulast%3DVoit%26aufirst%3DT.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DElfring%26aufirst%3DG.%26aulast%3DKroger%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DMcIntosh%26aufirst%3DJ.%26aulast%3DOng%26aufirst%3DT.%26aulast%3DRiebling%26aufirst%3DP.%26aulast%3DSouza%26aufirst%3DM.%26aulast%3DSpiegel%26aufirst%3DR.%2BJ.%26aulast%3DPeltz%26aufirst%3DS.%2BW.%26aulast%3DMercuri%26aufirst%3DE.%26atitle%3DAtaluren%2520in%2520patients%2520with%2520nonsense%2520mutation%2520Duchenne%2520muscular%2520dystrophy%2520%2528ACT%2520DMD%2529%253A%2520a%2520multicentre%252C%2520randomised%252C%2520double-blind%252C%2520placebo-controlled%252C%2520phase%25203%2520trial%26jtitle%3DLancet%26date%3D2017%26volume%3D390%26spage%3D1489%26epage%3D1498%26doi%3D10.1016%2FS0140-6736%2817%2931611-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span> <span> </span><span class="NLM_article-title">Mechanism of PTC124 activity in cell-based luciferase assays of nonsense codon suppression</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">3585</span>– <span class="NLM_lpage">3590</span>, <span class="refDoi"> DOI: 10.1073/pnas.0813345106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.0813345106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19208811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivFGqt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=3585-3590&author=D.+S.+Auldauthor=N.+Thorneauthor=W.+F.+Maguireauthor=J.+Inglese&title=Mechanism+of+PTC124+activity+in+cell-based+luciferase+assays+of+nonsense+codon+suppression&doi=10.1073%2Fpnas.0813345106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of PTC124 activity in cell-based lucifierase assays of nonsense codon suppression</span></div><div class="casAuthors">Auld, Douglas S.; Thorne, Natasha; Maguire, William F.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3585-3590</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) assays used in drug discovery frequently use reporter enzymes such as firefly luciferase (FLuc) as indicators of target activity.  An important caveat to consider, however, is that compds. can directly affect the reporter, leading to nonspecific but highly reproducible assay signal modulation.  In rare cases, this activity appears counterintuitive; for example, some FLuc inhibitors, acting through posttranslational Fluc reporter stabilization, appear to activate gene expression.  Previous efforts to characterize mols. that influence luciferase activity identified a subset of 3,5-diaryl-oxadiazole-contg. compds. as FLuc inhibitors.  Here, we evaluate a no. of compds. with this structural motif for activity against FLuc.  One such compd. is PTC124 {3-[5-(2-fluorophenyl)-1,2,4-oxadiazol-3-yl]benzoic acid}, a mol. originally identified in a cell-based FLuc assay as having nonsense codon suppression activity.  We find that the potency of FLuc inhibition for the tested compds. strictly correlates with their activity in a FLuc reporter cell-based nonsense codon assay, with PTC124 emerging as the most potent FLuc inhibitor (IC50 = 7 ± 1 nM).  However, these compds., including PTC124, fail to show nonsense codon suppression activity when Renilla reniformis luciferase (RLuc) is used as a reporter and are inactive against the RLuc enzyme.  This suggests that the initial discovery of PTC124 may have been biased by its direct effect on the FLuc reporter, implicating firefly luciferase as a mol. target of PTC124.  Our results demonstrate the value of understanding potential interactions between reporter enzymes and chem. compds. and emphasize the importance of implementing the appropriate control assays before interpreting HTS results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVYgPZRfbQR7Vg90H21EOLACvtfcHk0lj9kdJ2aN-suA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivFGqt74%253D&md5=8fd46b333035a4454e61ae600271fba8</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0813345106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0813345106%26sid%3Dliteratum%253Aachs%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DMaguire%26aufirst%3DW.%2BF.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DMechanism%2520of%2520PTC124%2520activity%2520in%2520cell-based%2520luciferase%2520assays%2520of%2520nonsense%2520codon%2520suppression%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D3585%26epage%3D3590%26doi%3D10.1073%2Fpnas.0813345106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lea, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanzlik, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for the high-affinity binding and stabilization of firefly luciferase by PTC124</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">4878</span>– <span class="NLM_lpage">4883</span>, <span class="refDoi"> DOI: 10.1073/pnas.0909141107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.0909141107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=20194791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFaqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=4878-4883&author=D.+S.+Auldauthor=S.+Lovellauthor=N.+Thorneauthor=W.+A.+Leaauthor=D.+J.+Maloneyauthor=M.+Shenauthor=G.+Raiauthor=K.+P.+Battaileauthor=C.+J.+Thomasauthor=A.+Simeonovauthor=R.+P.+Hanzlikauthor=J.+Inglese&title=Molecular+basis+for+the+high-affinity+binding+and+stabilization+of+firefly+luciferase+by+PTC124&doi=10.1073%2Fpnas.0909141107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for the high-affinity binding and stabilization of firefly luciferase by PTC124</span></div><div class="casAuthors">Auld, Douglas S.; Lovell, Scott; Thorne, Natasha; Lea, Wendy A.; Maloney, David J.; Shen, Min; Rai, Ganesha; Battaile, Kevin P.; Thomas, Craig J.; Simeonov, Anton; Hanzlik, Robert P.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4878-4883, S4878/1-S4878/14</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Firefly luciferase (FLuc), an ATP-dependent bioluminescent reporter enzyme, is broadly used in chem. biol. and drug discovery assays.  PTC124 (Ataluren; (3-[5-(2-fluorophenyl)-1,2,4-oxadiazol-3-yl]benzoic acid)) discovered in an FLuc-based assay targeting nonsense codon suppression, is an unusually potent FLuc-inhibitor.  Paradoxically, PTC124 and related analogs increase cellular FLuc activity levels by posttranslational stabilization.  In this study, we show that FLuc inhibition and stabilization is the result of an inhibitory product formed during the FLuc-catalyzed reaction between its natural substrate, ATP, and PTC124.  A 2.0 Å cocrystal structure revealed the inhibitor to be the acyl-AMP mixed-anhydride adduct PTC124-AMP, which was subsequently synthesized and shown to be a high-affinity multisubstrate adduct inhibitor (MAI; KD = 120 pM) of FLuc.  Biochem. assays, liq. chromatog./mass spectrometry, and near-attack conformer modeling demonstrate that formation of this novel MAI is absolutely dependent upon the precise positioning and reactivity of a key meta-carboxylate of PTC124 within the FLuc active site.  We also demonstrate that the inhibitory activity of PTC124-AMP is relieved by free CoA, a component present at high concns. in luciferase detection reagents used for cell-based assays.  This explains why PTC124 can appear to increase, instead of inhibit, FLuc activity in cell-based reporter gene assays.  To our knowledge, this is an unusual example in which the "off-target" effect of a small mol. is mediated by an MAI mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIPfC1eBndl7Vg90H21EOLACvtfcHk0lj9kdJ2aN-suA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFaqsbo%253D&md5=e22180fb96672f3fa7cdc907b7ada4fe</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0909141107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0909141107%26sid%3Dliteratum%253Aachs%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DLea%26aufirst%3DW.%2BA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHanzlik%26aufirst%3DR.%2BP.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DMolecular%2520basis%2520for%2520the%2520high-affinity%2520binding%2520and%2520stabilization%2520of%2520firefly%2520luciferase%2520by%2520PTC124%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D4878%26epage%3D4883%26doi%3D10.1073%2Fpnas.0909141107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroth, M.</span></span> <span> </span><span class="NLM_article-title">Gene therapy for spinal muscular atrophy: an emerging treatment option for a devastating disease</span>. <i>J. Manag. Care Spec. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">S3</span>– <span class="NLM_lpage">S16</span>, <span class="refDoi"> DOI: 10.18553/jmcp.2018.24.12-a.s3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.18553%2Fjmcp.2018.24.12-a.s3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30582825" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=S3-S16&author=V.+K.+Raoauthor=D.+Kappauthor=M.+Schroth&title=Gene+therapy+for+spinal+muscular+atrophy%3A+an+emerging+treatment+option+for+a+devastating+disease&doi=10.18553%2Fjmcp.2018.24.12-a.s3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.18553%2Fjmcp.2018.24.12-a.s3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18553%252Fjmcp.2018.24.12-a.s3%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DV.%2BK.%26aulast%3DKapp%26aufirst%3DD.%26aulast%3DSchroth%26aufirst%3DM.%26atitle%3DGene%2520therapy%2520for%2520spinal%2520muscular%2520atrophy%253A%2520an%2520emerging%2520treatment%2520option%2520for%2520a%2520devastating%2520disease%26jtitle%3DJ.%2520Manag.%2520Care%2520Spec.%2520Pharm.%26date%3D2018%26volume%3D24%26spage%3DS3%26epage%3DS16%26doi%3D10.18553%2Fjmcp.2018.24.12-a.s3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axford, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cody, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridrich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmaster, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoosear, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servais, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivasankaran, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dales, N. A.</span></span> <span> </span><span class="NLM_article-title">Discovery of small molecule splicing modulators of survival motor neuron-2 (SMN2) for the treatment of spinal muscular atrophy (SMA)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">11021</span>– <span class="NLM_lpage">11036</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01291</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01291" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFeju7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=11021-11036&author=A.+K.+Cheungauthor=B.+Hurleyauthor=R.+Kerriganauthor=L.+Shuauthor=D.+N.+Chinauthor=Y.+Shenauthor=G.+O%E2%80%99Brienauthor=M.+J.+Sungauthor=Y.+Houauthor=J.+Axfordauthor=E.+Codyauthor=R.+Sunauthor=A.+Fazalauthor=C.+Fridrichauthor=C.+C.+Sanchezauthor=R.+C.+Tomlinsonauthor=M.+Jainauthor=L.+Dengauthor=K.+Hoffmasterauthor=C.+Songauthor=M.+Van+Hoosearauthor=Y.+Shinauthor=R.+Servaisauthor=C.+Towlerauthor=M.+Hildauthor=D.+Curtisauthor=W.+F.+Dietrichauthor=L.+G.+Hamannauthor=K.+Brinerauthor=K.+S.+Chenauthor=D.+Kobayashiauthor=R.+Sivasankaranauthor=N.+A.+Dales&title=Discovery+of+small+molecule+splicing+modulators+of+survival+motor+neuron-2+%28SMN2%29+for+the+treatment+of+spinal+muscular+atrophy+%28SMA%29&doi=10.1021%2Facs.jmedchem.8b01291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule Splicing Modulators of Survival Motor Neuron-2 (SMN2) for the Treatment of Spinal Muscular Atrophy (SMA)</span></div><div class="casAuthors">Cheung, Atwood K.; Hurley, Brian; Kerrigan, Ryan; Shu, Lei; Chin, Donovan N.; Shen, Yiping; OBrien, Gary; Sung, Moo Je; Hou, Ying; Axford, Jake; Cody, Emma; Sun, Robert; Fazal, Aleem; Tomlinson, Ronald C.; Jain, Monish; Deng, Lin; Hoffmaster, Keith; Song, Cheng; Van Hoosear, Mailin; Shin, Youngah; Servais, Rebecca; Towler, Christopher; Hild, Marc; Curtis, Daniel; Dietrich, William F.; Hamann, Lawrence G.; Briner, Karin; Chen, Karen S.; Kobayashi, Dione; Sivasankaran, Rajeev; Dales, Natalie A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11021-11036</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA), a rare neuromuscular disorder, is the leading genetic cause of death in infants and toddlers.  SMA is caused by the deletion or a loss of function mutation of the survival motor neuron 1 (SMN1) gene.  In humans, a second closely related gene SMN2 exists, however it codes for a less stable SMN protein.  In recent years, significant progress has been made toward disease modifying treatments for SMA by modulating SMN2 pre-mRNA splicing.  Herein, we describe the discovery of LMI070/branaplam, a small mol. that stabilizes the interaction between the spliceosome and SMN2 pre-mRNA.  Branaplam (1) originated from a high-throughput phenotypic screening hit, pyridazine 2, and evolved via multi-parameter lead optimization.  In a severe mouse SMA model, branaplam treatment increased full-length SMN RNA and protein levels, and extended survival.  Currently, branaplam is in clin. studies for SMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprrHPkSEhGhrVg90H21EOLACvtfcHk0lh90XS1abapGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFeju7nL&md5=c564332b680a6ef0ad06bc9e5a4db16e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01291%26sid%3Dliteratum%253Aachs%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DHurley%26aufirst%3DB.%26aulast%3DKerrigan%26aufirst%3DR.%26aulast%3DShu%26aufirst%3DL.%26aulast%3DChin%26aufirst%3DD.%2BN.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DG.%26aulast%3DSung%26aufirst%3DM.%2BJ.%26aulast%3DHou%26aufirst%3DY.%26aulast%3DAxford%26aufirst%3DJ.%26aulast%3DCody%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DR.%26aulast%3DFazal%26aufirst%3DA.%26aulast%3DFridrich%26aufirst%3DC.%26aulast%3DSanchez%26aufirst%3DC.%2BC.%26aulast%3DTomlinson%26aufirst%3DR.%2BC.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DHoffmaster%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DVan%2BHoosear%26aufirst%3DM.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DServais%26aufirst%3DR.%26aulast%3DTowler%26aufirst%3DC.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DCurtis%26aufirst%3DD.%26aulast%3DDietrich%26aufirst%3DW.%2BF.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DBriner%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DK.%2BS.%26aulast%3DKobayashi%26aufirst%3DD.%26aulast%3DSivasankaran%26aufirst%3DR.%26aulast%3DDales%26aufirst%3DN.%2BA.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520splicing%2520modulators%2520of%2520survival%2520motor%2520neuron-2%2520%2528SMN2%2529%2520for%2520the%2520treatment%2520of%2520spinal%2520muscular%2520atrophy%2520%2528SMA%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D11021%26epage%3D11036%26doi%3D10.1021%2Facs.jmedchem.8b01291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charnas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voltz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berghs-Clairmont, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Praestgaard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Raspide, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deconinck, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Born, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranello, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schara, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goemans, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roubenoff, R.</span></span> <span> </span><span class="NLM_article-title">Safety and efficacy findings in the first-in-human trial (FIH) of the oral splice modulator branaplam in type 1 spinal muscular atrophy (SMA): interim results</span>. <i>Neuromuscul. Disord.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">S207</span>– <span class="NLM_lpage">S208</span>, <span class="refDoi"> DOI: 10.1016/j.nmd.2017.06.411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.nmd.2017.06.411" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=S207-S208&author=L.+Charnasauthor=E.+Voltzauthor=C.+Pfisterauthor=T.+Petersauthor=A.+Hartmannauthor=C.+Berghs-Clairmontauthor=J.+Praestgaardauthor=M.+de+Raspideauthor=N.+Deconinckauthor=A.+Bornauthor=G.+Baranelloauthor=E.+Bertiniauthor=U.+Scharaauthor=N.+Goemansauthor=R.+Roubenoff&title=Safety+and+efficacy+findings+in+the+first-in-human+trial+%28FIH%29+of+the+oral+splice+modulator+branaplam+in+type+1+spinal+muscular+atrophy+%28SMA%29%3A+interim+results&doi=10.1016%2Fj.nmd.2017.06.411"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.nmd.2017.06.411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nmd.2017.06.411%26sid%3Dliteratum%253Aachs%26aulast%3DCharnas%26aufirst%3DL.%26aulast%3DVoltz%26aufirst%3DE.%26aulast%3DPfister%26aufirst%3DC.%26aulast%3DPeters%26aufirst%3DT.%26aulast%3DHartmann%26aufirst%3DA.%26aulast%3DBerghs-Clairmont%26aufirst%3DC.%26aulast%3DPraestgaard%26aufirst%3DJ.%26aulast%3Dde%2BRaspide%26aufirst%3DM.%26aulast%3DDeconinck%26aufirst%3DN.%26aulast%3DBorn%26aufirst%3DA.%26aulast%3DBaranello%26aufirst%3DG.%26aulast%3DBertini%26aufirst%3DE.%26aulast%3DSchara%26aufirst%3DU.%26aulast%3DGoemans%26aufirst%3DN.%26aulast%3DRoubenoff%26aufirst%3DR.%26atitle%3DSafety%2520and%2520efficacy%2520findings%2520in%2520the%2520first-in-human%2520trial%2520%2528FIH%2529%2520of%2520the%2520oral%2520splice%2520modulator%2520branaplam%2520in%2520type%25201%2520spinal%2520muscular%2520atrophy%2520%2528SMA%2529%253A%2520interim%2520results%26jtitle%3DNeuromuscul.%2520Disord.%26date%3D2017%26volume%3D27%26spage%3DS207%26epage%3DS208%26doi%3D10.1016%2Fj.nmd.2017.06.411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, K. A.</span></span> <span> </span><span class="NLM_article-title">Mechanistic studies of a small-molecule modulator of SMN2 splicing</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">E4604</span>– <span class="NLM_lpage">E4612</span>, <span class="refDoi"> DOI: 10.1073/pnas.1800260115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1800260115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29712837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVGhtLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=E4604-E4612&author=J.+Wangauthor=P.+G.+Schultzauthor=K.+A.+Johnson&title=Mechanistic+studies+of+a+small-molecule+modulator+of+SMN2+splicing&doi=10.1073%2Fpnas.1800260115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic studies of a small-molecule modulator of SMN2 splicing</span></div><div class="casAuthors">Wang, Jingxin; Schultz, Peter G.; Johnson, Kristen A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">E4604-E4612</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">RG-7916 is a first-in-class drug candidate for the treatment of spinal muscular atrophy (SMA) that functions by modulating pre-mRNA splicing of the SMN2 gene, resulting in a 2.5-fold increase in survival of motor neuron (SMN) protein level, a key protein lacking in SMA patients.  RG-7916 is currently in three interventional phase 2 clin. trials for various types of SMA.  In this report, we show that SMN-C2 and -C3, close analogs of RG-7916, act as selective RNA-binding ligands that modulate pre-mRNA splicing.  Chem. proteomic and genomic techniques reveal that SMN-C2 directly binds to the AGGAAG motif on exon 7 of the SMN2 pre-mRNA, and promotes a conformational change in two to three unpaired nucleotides at the junction of intron 6 and exon 7 in both in vitro and in-cell models.  This change creates a new functional binding surface that increases binding of the splicing modulators, far upstream element binding protein 1 (FUBP1) and its homolog, KH-type splicing regulatory protein (KHSRP), to the SMN-C2/C3-SMN2 pre-mRNA complex and enhances SMN2 splicing.  These findings underscore the potential of small-mol. drugs to selectively bind RNA and modulate pre-mRNA splicing as an approach to the treatment of human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh37iFocFbJbVg90H21EOLACvtfcHk0ljhQ1UvlDnh1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVGhtLjM&md5=989d9a1be671358c31a9c957768272df</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1800260115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1800260115%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DJohnson%26aufirst%3DK.%2BA.%26atitle%3DMechanistic%2520studies%2520of%2520a%2520small-molecule%2520modulator%2520of%2520SMN2%2520splicing%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2018%26volume%3D115%26spage%3DE4604%26epage%3DE4612%26doi%3D10.1073%2Fpnas.1800260115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sivaramakrishnan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campagne, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heckel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meistermann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hug, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birrer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moursy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khawaja, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmucki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berntenis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giroud, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzouros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banfai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duran-Pacheco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamerz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiu Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luebbers, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebeling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cléry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paushkin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krainer, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, F. H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, F.</span></span> <span> </span><span class="NLM_article-title">Binding to SMN2 pre-mRNA-protein complex elicits specificity for small molecule splicing modifiers</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1476</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01559-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fs41467-017-01559-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29133793" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC1M3gvVagtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1476&author=M.+Sivaramakrishnanauthor=K.+D.+McCarthyauthor=S.+Campagneauthor=S.+Huberauthor=S.+Meierauthor=A.+Augustinauthor=T.+Heckelauthor=H.+Meistermannauthor=M.+N.+Hugauthor=P.+Birrerauthor=A.+Moursyauthor=S.+Khawajaauthor=R.+Schmuckiauthor=N.+Berntenisauthor=N.+Giroudauthor=S.+Gollingauthor=M.+Tzourosauthor=B.+Banfaiauthor=G.+Duran-Pachecoauthor=J.+Lamerzauthor=Y.+Hsiu+Liuauthor=T.+Luebbersauthor=H.+Ratniauthor=M.+Ebelingauthor=A.+Cl%C3%A9ryauthor=S.+Paushkinauthor=A.+R.+Krainerauthor=F.+H.+T.+Allainauthor=F.+Metzger&title=Binding+to+SMN2+pre-mRNA-protein+complex+elicits+specificity+for+small+molecule+splicing+modifiers&doi=10.1038%2Fs41467-017-01559-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Binding to SMN2 pre-mRNA-protein complex elicits specificity for small molecule splicing modifiers</span></div><div class="casAuthors">Sivaramakrishnan Manaswini; McCarthy Kathleen D; Huber Sylwia; Meier Sonja; Augustin Angelique; Heckel Tobias; Meistermann Helene; Hug Melanie N; Birrer Pascale; Schmucki Roland; Berntenis Nikos; Giroud Nicolas; Golling Sabrina; Tzouros Manuel; Banfai Balazs; Duran-Pacheco Gonzalo; Lamerz Jens; Luebbers Thomas; Ratni Hasane; Ebeling Martin; Metzger Friedrich; Sivaramakrishnan Manaswini; Meier Sonja; Hsiu Liu Ying; Krainer Adrian R; Campagne Sebastien; Moursy Ahmed; Khawaja Sarah; Clery Antoine; Allain Frederic H-T; Paushkin Sergey</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1476</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Small molecule splicing modifiers have been previously described that target the general splicing machinery and thus have low specificity for individual genes.  Several potent molecules correcting the splicing deficit of the SMN2 (survival of motor neuron 2) gene have been identified and these molecules are moving towards a potential therapy for spinal muscular atrophy (SMA).  Here by using a combination of RNA splicing, transcription, and protein chemistry techniques, we show that these molecules directly bind to two distinct sites of the SMN2 pre-mRNA, thereby stabilizing a yet unidentified ribonucleoprotein (RNP) complex that is critical to the specificity of these small molecules for SMN2 over other genes.  In addition to the therapeutic potential of these molecules for treatment of SMA, our work has wide-ranging implications in understanding how small molecules can interact with specific quaternary RNA structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR8ACsymX9-A2BAiYWy6n3SfW6udTcc2eYyddGTwtAcqLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3gvVagtA%253D%253D&md5=b739bbdecf61fd3f3ac20966ac3798fe</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01559-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01559-4%26sid%3Dliteratum%253Aachs%26aulast%3DSivaramakrishnan%26aufirst%3DM.%26aulast%3DMcCarthy%26aufirst%3DK.%2BD.%26aulast%3DCampagne%26aufirst%3DS.%26aulast%3DHuber%26aufirst%3DS.%26aulast%3DMeier%26aufirst%3DS.%26aulast%3DAugustin%26aufirst%3DA.%26aulast%3DHeckel%26aufirst%3DT.%26aulast%3DMeistermann%26aufirst%3DH.%26aulast%3DHug%26aufirst%3DM.%2BN.%26aulast%3DBirrer%26aufirst%3DP.%26aulast%3DMoursy%26aufirst%3DA.%26aulast%3DKhawaja%26aufirst%3DS.%26aulast%3DSchmucki%26aufirst%3DR.%26aulast%3DBerntenis%26aufirst%3DN.%26aulast%3DGiroud%26aufirst%3DN.%26aulast%3DGolling%26aufirst%3DS.%26aulast%3DTzouros%26aufirst%3DM.%26aulast%3DBanfai%26aufirst%3DB.%26aulast%3DDuran-Pacheco%26aufirst%3DG.%26aulast%3DLamerz%26aufirst%3DJ.%26aulast%3DHsiu%2BLiu%26aufirst%3DY.%26aulast%3DLuebbers%26aufirst%3DT.%26aulast%3DRatni%26aufirst%3DH.%26aulast%3DEbeling%26aufirst%3DM.%26aulast%3DCl%25C3%25A9ry%26aufirst%3DA.%26aulast%3DPaushkin%26aufirst%3DS.%26aulast%3DKrainer%26aufirst%3DA.%2BR.%26aulast%3DAllain%26aufirst%3DF.%2BH.%2BT.%26aulast%3DMetzger%26aufirst%3DF.%26atitle%3DBinding%2520to%2520SMN2%2520pre-mRNA-protein%2520complex%2520elicits%2520specificity%2520for%2520small%2520molecule%2520splicing%2520modifiers%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1476%26doi%3D10.1038%2Fs41467-017-01559-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of non-ATG translational events in cells via covalent small molecules targeting RNA</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">5336</span>– <span class="NLM_lpage">5345</span>, <span class="refDoi"> DOI: 10.1021/ja507448y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja507448y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsFGnurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=5336-5345&author=W.-Y.+Yangauthor=H.+D.+Wilsonauthor=S.+P.+Velagapudiauthor=M.+D.+Disney&title=Inhibition+of+non-ATG+translational+events+in+cells+via+covalent+small+molecules+targeting+RNA&doi=10.1021%2Fja507448y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Non-ATG Translational Events in Cells via Covalent Small Molecules Targeting RNA</span></div><div class="casAuthors">Yang, Wang-Yong; Wilson, Henry D.; Velagapudi, Sai Pradeep; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5336-5345</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One major class of disease-causing RNAs is expanded repeating transcripts.  These RNAs cause diseases via multiple mechanisms, including: (i) gain-of-function, in which repeating RNAs bind and sequester proteins involved in RNA biogenesis and (ii) repeat assocd. non-ATG (RAN) translation, in which repeating transcripts are translated into toxic proteins without use of a canonical, AUG, start codon.  Herein, the authors develop and study chem. probes that bind and react with an expanded r(CGG) repeat (r(CGG)exp) present in a 5' untranslated region that causes fragile X-assocd. tremor/ataxia syndrome (FXTAS).  Reactive compds. bind to r(CGG)expin cellulo as shown with Chem-CLIP-Map, an approach to map small mol. binding sites within RNAs in cells.  Compds. also potently improve FXTAS-assocd. premRNA splicing and RAN translational defects, while not affecting translation of the downstream open reading frame.  In contrast, oligonucleotides affect both RAN and canonical translation when they bind to r(CGG)exp, which is mechanistically traced to a decrease in polysome loading.  Thus, designer small mols. that react with RNA targets can be used to profile the RNAs to which they bind in cells, including identification of binding sites, and can modulate several aspects of RNA-mediated disease pathol. in a manner that may be more beneficial than oligonucleotides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2Of8BkHmwpbVg90H21EOLACvtfcHk0ljhQ1UvlDnh1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsFGnurY%253D&md5=432505c32da2493c2e77960be352d806</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fja507448y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja507448y%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.-Y.%26aulast%3DWilson%26aufirst%3DH.%2BD.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DInhibition%2520of%2520non-ATG%2520translational%2520events%2520in%2520cells%2520via%2520covalent%2520small%2520molecules%2520targeting%2520RNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D5336%26epage%3D5345%26doi%3D10.1021%2Fja507448y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holley, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apgar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Everett, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madison, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquisee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merrill, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penswick, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamir, A.</span></span> <span> </span><span class="NLM_article-title">Structure of a ribonucleic acid</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1965</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">1462</span>– <span class="NLM_lpage">1465</span>, <span class="refDoi"> DOI: 10.1126/science.147.3664.1462</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1126%2Fscience.147.3664.1462" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=14263761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaF2MXotlOlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=1965&pages=1462-1465&author=R.+W.+Holleyauthor=J.+Apgarauthor=G.+A.+Everettauthor=J.+T.+Madisonauthor=M.+Marquiseeauthor=S.+H.+Merrillauthor=J.+R.+Penswickauthor=A.+Zamir&title=Structure+of+a+ribonucleic+acid&doi=10.1126%2Fscience.147.3664.1462"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a ribonucleic acid</span></div><div class="casAuthors">Holley, Robert W.; Apgar, Jean; Everett, George A.; Madison, James T.; Marquisee, Mark; Merrill, Susan H.; Penswick, John Robert; Zamir, Ada</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1965</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">3664</span>),
    <span class="NLM_cas:pages">1462-5</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">Structure was detd. for alanyl RNA with pancreatic RNase and Taka-Diastase RNase Tl to digest small fragments followed by successively larger fragments.  The RNA is described in terms of 16 oligonucleotide sequences.  The presence of a 5'-phosphate on 1 sequence establishes it as the left end of the mol., while a 3'-hydroxyl establishes the right end.  The sequence of the 16 oligonucleotides is not known.  There is a total of 77 nucleotide residues to give a mol. wt. of 26,600 as the Na salt.  There is no pattern in the distribution of the minor residues; they are scattered throughout most of the mol. though none is near the amino acid-acceptor end.  There are no long nucleotide sequences so that double-stranded regions are either short or contain imperfections in base pairing.  A detailed description of each sequence and the method of detn. is included.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrysckjLwWNQ7Vg90H21EOLACvtfcHk0liYU6DyJvg2YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2MXotlOlug%253D%253D&md5=9f12aded94c0643eb0ee8a9330813b0d</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fscience.147.3664.1462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.147.3664.1462%26sid%3Dliteratum%253Aachs%26aulast%3DHolley%26aufirst%3DR.%2BW.%26aulast%3DApgar%26aufirst%3DJ.%26aulast%3DEverett%26aufirst%3DG.%2BA.%26aulast%3DMadison%26aufirst%3DJ.%2BT.%26aulast%3DMarquisee%26aufirst%3DM.%26aulast%3DMerrill%26aufirst%3DS.%2BH.%26aulast%3DPenswick%26aufirst%3DJ.%2BR.%26aulast%3DZamir%26aufirst%3DA.%26atitle%3DStructure%2520of%2520a%2520ribonucleic%2520acid%26jtitle%3DScience%26date%3D1965%26volume%3D147%26spage%3D1462%26epage%3D1465%26doi%3D10.1126%2Fscience.147.3664.1462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RajBhandary, U. L.</span></span> <span> </span><span class="NLM_article-title">Transfer RNA: molecular structure, sequence, and properties</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">860</span>, <span class="refDoi"> DOI: 10.1146/annurev.bi.45.070176.004105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1146%2Fannurev.bi.45.070176.004105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=60910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaE28XkvVylsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=1976&pages=805-860&author=A.+Richauthor=U.+L.+RajBhandary&title=Transfer+RNA%3A+molecular+structure%2C+sequence%2C+and+properties&doi=10.1146%2Fannurev.bi.45.070176.004105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Transfer RNA:  molecular structure, sequence, and properties</span></div><div class="casAuthors">Rich, Alexander; Bhandary, U. L. Raj</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">805-60</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    </div><div class="casAbstract">A review with 320 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF4j8tnvuc9rVg90H21EOLACvtfcHk0liYU6DyJvg2YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XkvVylsbg%253D&md5=4ad97acf33f3344229b5a4b9f5bed322</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1146%2Fannurev.bi.45.070176.004105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.bi.45.070176.004105%26sid%3Dliteratum%253Aachs%26aulast%3DRich%26aufirst%3DA.%26aulast%3DRajBhandary%26aufirst%3DU.%2BL.%26atitle%3DTransfer%2520RNA%253A%2520molecular%2520structure%252C%2520sequence%252C%2520and%2520properties%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D1976%26volume%3D45%26spage%3D805%26epage%3D860%26doi%3D10.1146%2Fannurev.bi.45.070176.004105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyle, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mount, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steitz, J. A.</span></span> <span> </span><span class="NLM_article-title">Are snRNPs involved in splicing?</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">220</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1038/283220a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2F283220a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=7350545" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaL3cXhsF2jtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=1980&pages=220-224&author=M.+R.+Lernerauthor=J.+A.+Boyleauthor=S.+M.+Mountauthor=S.+L.+Wolinauthor=J.+A.+Steitz&title=Are+snRNPs+involved+in+splicing%3F&doi=10.1038%2F283220a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Are snRNPs involved in splicing?</span></div><div class="casAuthors">Lerner, Michael R.; Boyle, John A.; Mount, Stephen M.; Wolin, Sandra L.; Steitz, Joan A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">5743</span>),
    <span class="NLM_cas:pages">220-4</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">A comparison of small nuclear ribonucleoproteins (snRNPs) from HeLa, Ehrlich ascites, and armyworm cells indicated that antigenic snRNPs were highly conserved from man to insects.  Liver cells which actively formed mRNA had both higher amts. and a slightly different population of snRNAs than the cryptic nuclei of biosynthetically inactive erythrocytes.  Antigenic snRNPs contg. snRNAs U1, U2, U3, U4, U5, and U6 from Ehrlich ascites cells sedimented in the 30 S region of the sucrose gradient, whereas RNA-labeled U1a* appeared only in the 10 S region where the free snRNP particles sedimented.  Fingerprint anal. showed that U1a* was identical to U1a except that it lacked the 5'-trimethylated cap moiety plus 6 addnl. nucleotides from the 5' end of the U1a sequence.  Several of the residues missing in U1a* fell within a region of U1a which was extensively complementary to sequences in splice junctions in hnRNPs (heterogeneous nuclear ribonucleoproteins).  Thus, snRNPs may be directly involved in the splicing of hnRNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIeFSm-3VWlbVg90H21EOLACvtfcHk0liYU6DyJvg2YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXhsF2jtr8%253D&md5=a05e9914f26d348216e5c04d4b9cc038</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2F283220a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F283220a0%26sid%3Dliteratum%253Aachs%26aulast%3DLerner%26aufirst%3DM.%2BR.%26aulast%3DBoyle%26aufirst%3DJ.%2BA.%26aulast%3DMount%26aufirst%3DS.%2BM.%26aulast%3DWolin%26aufirst%3DS.%2BL.%26aulast%3DSteitz%26aufirst%3DJ.%2BA.%26atitle%3DAre%2520snRNPs%2520involved%2520in%2520splicing%253F%26jtitle%3DNature%26date%3D1980%26volume%3D283%26spage%3D220%26epage%3D224%26doi%3D10.1038%2F283220a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ganser, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herschlag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hashimi, H. M.</span></span> <span> </span><span class="NLM_article-title">The roles of structural dynamics in the cellular functions of RNAs</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">474</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1038/s41580-019-0136-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fs41580-019-0136-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31182864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=474-489&author=L.+R.+Ganserauthor=M.+L.+Kellyauthor=D.+Herschlagauthor=H.+M.+Al-Hashimi&title=The+roles+of+structural+dynamics+in+the+cellular+functions+of+RNAs&doi=10.1038%2Fs41580-019-0136-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The roles of structural dynamics in the cellular functions of RNAs</span></div><div class="casAuthors">Ganser, Laura R.; Kelly, Megan L.; Herschlag, Daniel; Al-Hashimi, Hashim M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">474-489</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  RNAs fold into 3D structures that range from simple helical elements to complex tertiary structures and quaternary ribonucleoprotein assemblies.  The functions of many regulatory RNAs depend on how their 3D structure changes in response to a diverse array of cellular conditions.  In this Review, we examine how the structural characterization of RNA as dynamic ensembles of conformations, which form with different probabilities and at different timescales, is improving our understanding of RNA function in cells.  We discuss the mechanisms of gene regulation by microRNAs, riboswitches, ribozymes, post-transcriptional RNA modifications and RNA-binding proteins, and how the cellular environment and processes such as liq.-liq. phase sepn. may affect RNA folding and activity.  The emerging RNA-ensemble-function paradigm is changing our perspective and understanding of RNA regulation, from in vitro to in vivo and from descriptive to predictive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6pHr5mgMApLVg90H21EOLACvtfcHk0lgFCoQu9HwN5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtb7K&md5=83a97268a18fb90f821ce6627b89af5a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fs41580-019-0136-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41580-019-0136-0%26sid%3Dliteratum%253Aachs%26aulast%3DGanser%26aufirst%3DL.%2BR.%26aulast%3DKelly%26aufirst%3DM.%2BL.%26aulast%3DHerschlag%26aufirst%3DD.%26aulast%3DAl-Hashimi%26aufirst%3DH.%2BM.%26atitle%3DThe%2520roles%2520of%2520structural%2520dynamics%2520in%2520the%2520cellular%2520functions%2520of%2520RNAs%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2019%26volume%3D20%26spage%3D474%26epage%3D489%26doi%3D10.1038%2Fs41580-019-0136-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stombaugh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zirbel, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westhof, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leontis, N. B.</span></span> <span> </span><span class="NLM_article-title">Frequency and isostericity of RNA base pairs</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">2294</span>– <span class="NLM_lpage">2312</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1093%2Fnar%2Fgkp011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19240142" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltFWktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=2294-2312&author=J.+Stombaughauthor=C.+L.+Zirbelauthor=E.+Westhofauthor=N.+B.+Leontis&title=Frequency+and+isostericity+of+RNA+base+pairs&doi=10.1093%2Fnar%2Fgkp011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Frequency and isostericity of RNA base pairs</span></div><div class="casAuthors">Stombaugh, Jesse; Zirbel, Craig L.; Westhof, Eric; Leontis, Neocles B.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2294-2312</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Most of the hairpin, internal and junction loops that appear single-stranded in std. RNA secondary structures form recurrent 3D motifs, where non-Watson-Crick base pairs play a central role.  Non-Watson-Crick base pairs also play crucial roles in tertiary contacts in structured RNA mols.  We previously classified RNA base pairs geometrically to group together those base pairs that are structurally similar (isosteric) and therefore able to substitute for each other by mutation without disrupting the 3D structure.  Here, we introduce a quant. measure of base pair isostericity, the IsoDiscrepancy Index (IDI), to more accurately det. which base pair substitutions can potentially occur in conserved motifs.  We ext. and classify base pairs from a reduced-redundancy set of RNA 3D structures from the Protein Data Bank (PDB) and calc. centroids (exemplars) for each base combination and geometric base pair type (family).  We use the exemplars and IDI values to update our online Basepair Catalog and the Isostericity Matrixes (IM) for each base pair family.  From the database of base pairs obsd. in 3D structures we derive base pair occurrence frequencies for each of the 12 geometric base pair families.  To improve the statistics from the 3D structures, we also derive base pair occurrence frequencies from rRNA sequence alignments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmiA1N4lBbPLVg90H21EOLACvtfcHk0lgFCoQu9HwN5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltFWktrc%253D&md5=7dd0cc423532908b8f18dfd4bccba582</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp011%26sid%3Dliteratum%253Aachs%26aulast%3DStombaugh%26aufirst%3DJ.%26aulast%3DZirbel%26aufirst%3DC.%2BL.%26aulast%3DWesthof%26aufirst%3DE.%26aulast%3DLeontis%26aufirst%3DN.%2BB.%26atitle%3DFrequency%2520and%2520isostericity%2520of%2520RNA%2520base%2520pairs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26spage%3D2294%26epage%3D2312%26doi%3D10.1093%2Fnar%2Fgkp011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yesselman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisaria, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herschlag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenleaf, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, R.</span></span> <span> </span><span class="NLM_article-title">Sequence-dependent RNA helix conformational preferences predictably impact tertiary structure formation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">16847</span>– <span class="NLM_lpage">16855</span>, <span class="refDoi"> DOI: 10.1073/pnas.1901530116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1901530116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31375637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1alsL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=16847-16855&author=J.+D.+Yesselmanauthor=S.+K.+Dennyauthor=N.+Bisariaauthor=D.+Herschlagauthor=W.+J.+Greenleafauthor=R.+Das&title=Sequence-dependent+RNA+helix+conformational+preferences+predictably+impact+tertiary+structure+formation&doi=10.1073%2Fpnas.1901530116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence-dependent RNA helix conformational preferences predictably impact tertiary structure formation</span></div><div class="casAuthors">Yesselman, Joseph D.; Denny, Sarah K.; Bisaria, Namita; Herschlag, Daniel; Greenleaf, William J.; Das, Rhiju</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">16847-16855</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Structured RNAs and RNA complexes underlie biol. processes ranging from control of gene expression to protein translation.  Approx. 50% of nucleotides within known structured RNAs are folded into Watson-Crick (WC) base pairs, and sequence changes that preserve these pairs are typically assumed to preserve higher-order RNA structure and binding of macromol. partners.  Here, we report that indirect effects of the helix sequence on RNA tertiary stability are, in fact, significant but are nevertheless predictable from a simple computational model called RNAMake-δδG.  When tested through the RNA on a massively parallel array (RNA-MaP) exptl. platform, blind predictions for>1500 variants of the tectoRNA heterodimer model system achieve high accuracy (rmsd 0.34 and 0.77 kcal/mol for sequence and length changes, resp.).  Detailed comparison of predictions to expts. support a microscopic picture of how helix sequence changes subtly modulate conformational fluctuations at each base-pair step, which accumulate to impact RNA tertiary structure stability.  Our study reveals a previously overlooked phenomenon in RNA structure formation and provides a framework of computation and expt. for understanding helix conformational preferences and their impact across biol. RNA and RNA-protein assemblies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBas5Lz0S4fbVg90H21EOLACvtfcHk0lgFCoQu9HwN5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1alsL%252FI&md5=2028e40755bcc5d65b78014b8ecb2464</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1901530116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1901530116%26sid%3Dliteratum%253Aachs%26aulast%3DYesselman%26aufirst%3DJ.%2BD.%26aulast%3DDenny%26aufirst%3DS.%2BK.%26aulast%3DBisaria%26aufirst%3DN.%26aulast%3DHerschlag%26aufirst%3DD.%26aulast%3DGreenleaf%26aufirst%3DW.%2BJ.%26aulast%3DDas%26aufirst%3DR.%26atitle%3DSequence-dependent%2520RNA%2520helix%2520conformational%2520preferences%2520predictably%2520impact%2520tertiary%2520structure%2520formation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D16847%26epage%3D16855%26doi%3D10.1073%2Fpnas.1901530116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Denny, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisaria, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yesselman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herschlag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenleaf, W. J.</span></span> <span> </span><span class="NLM_article-title">High-throughput investigation of diverse junction elements in RNA tertiary folding</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">390</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.05.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.cell.2018.05.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29961580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Cis7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2018&pages=377-390&author=S.+K.+Dennyauthor=N.+Bisariaauthor=J.+D.+Yesselmanauthor=R.+Dasauthor=D.+Herschlagauthor=W.+J.+Greenleaf&title=High-throughput+investigation+of+diverse+junction+elements+in+RNA+tertiary+folding&doi=10.1016%2Fj.cell.2018.05.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput investigation of diverse junction elements in RNA tertiary folding</span></div><div class="casAuthors">Denny, Sarah Knight; Bisaria, Namita; Yesselman, Joseph David; Das, Rhiju; Herschlag, Daniel; Greenleaf, William James</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-390.e20</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">RNAs fold into defined tertiary structures to function in crit. biol. processes.  While quant. models can predict RNA secondary structure stability, we are still unable to predict the thermodn. stability of RNA tertiary structure.  Here, we probed conformational preferences of diverse RNA two-way junctions to develop a predictive model for the formation of RNA tertiary structure.  We quant. measured tertiary assembly energetics of >1000 RNA junctions inserted in multiple structural scaffolds to generate a "thermodn. fingerprint" for each junction.  Thermodn. fingerprints enabled comparison of junction conformational preferences, revealing principles for how sequence influences 3-dimensional conformations.  Utilizing fingerprints of junctions with known crystal structures, we generated ensembles for related junctions that predicted their thermodn. effects on assembly formation.  This work revealed sequence-structure-energetic relations in RNA, demonstrated the capacity for diverse compensation strategies within tertiary structures, and provided a path to quant. modeling of RNA folding energetics based on "ensemble modularity.".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_6bGWqromibVg90H21EOLACvtfcHk0lgyBMP-2GRiCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Cis7vO&md5=30f8c56d8b28e7bacc0ccad937a12e60</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.05.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.05.038%26sid%3Dliteratum%253Aachs%26aulast%3DDenny%26aufirst%3DS.%2BK.%26aulast%3DBisaria%26aufirst%3DN.%26aulast%3DYesselman%26aufirst%3DJ.%2BD.%26aulast%3DDas%26aufirst%3DR.%26aulast%3DHerschlag%26aufirst%3DD.%26aulast%3DGreenleaf%26aufirst%3DW.%2BJ.%26atitle%3DHigh-throughput%2520investigation%2520of%2520diverse%2520junction%2520elements%2520in%2520RNA%2520tertiary%2520folding%26jtitle%3DCell%26date%3D2018%26volume%3D174%26spage%3D377%26epage%3D390%26doi%3D10.1016%2Fj.cell.2018.05.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assmann, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevilacqua, P. C.</span></span> <span> </span><span class="NLM_article-title">The RNA structurome: transcriptome-wide structure probing with next-generation sequencing</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">232</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2015.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.tibs.2015.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=25797096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvVGgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2015&pages=221-232&author=C.+K.+Kwokauthor=Y.+Tangauthor=S.+M.+Assmannauthor=P.+C.+Bevilacqua&title=The+RNA+structurome%3A+transcriptome-wide+structure+probing+with+next-generation+sequencing&doi=10.1016%2Fj.tibs.2015.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The RNA structurome: transcriptome-wide structure probing with next-generation sequencing</span></div><div class="casAuthors">Kwok, Chun Kit; Tang, Yin; Assmann, Sarah M.; Bevilacqua, Philip C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">221-232</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  RNA folds into intricate structures that enable its pivotal roles in biol., ranging from regulation of gene expression to ligand sensing and enzymic functions.  Therefore, elucidating RNA structure can provide profound insights into living systems.  A recent marriage between in vivo RNA structure probing and next-generation sequencing (NGS) has revolutionized the RNA field by enabling transcriptome-wide structure detn. in vivo, which has been applied to date to human cells, yeast cells, and Arabidopsis seedlings.  Anal. of resultant in vivo 'RNA structuromes' provides new and important information regarding myriad cellular processes, including control of translation, alternative splicing, alternative polyadenylation, energy-dependent unfolding of mRNA, and effects of proteins on RNA structure.  An emerging view suggests potential links between RNA structure and stress and disease physiol. across the tree of life.  As we discuss here, these exciting findings open new frontiers into RNA biol., genome biol., and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI9z6BEOwvmrVg90H21EOLACvtfcHk0lgyBMP-2GRiCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvVGgsLc%253D&md5=288aa8be796db4c452b6bcfa27012424</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2015.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2015.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKwok%26aufirst%3DC.%2BK.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DAssmann%26aufirst%3DS.%2BM.%26aulast%3DBevilacqua%26aufirst%3DP.%2BC.%26atitle%3DThe%2520RNA%2520structurome%253A%2520transcriptome-wide%2520structure%2520probing%2520with%2520next-generation%2520sequencing%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2015%26volume%3D40%26spage%3D221%26epage%3D232%26doi%3D10.1016%2Fj.tibs.2015.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fallmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Will, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grüning, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backofen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, P. F.</span></span> <span> </span><span class="NLM_article-title">Recent advances in RNA folding</span>. <i>J. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>261</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">104</span>, <span class="refDoi"> DOI: 10.1016/j.jbiotec.2017.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.jbiotec.2017.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28690134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Sls73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=2017&pages=97-104&author=J.+Fallmannauthor=S.+Willauthor=J.+Engelhardtauthor=B.+Gr%C3%BCningauthor=R.+Backofenauthor=P.+F.+Stadler&title=Recent+advances+in+RNA+folding&doi=10.1016%2Fj.jbiotec.2017.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in RNA folding</span></div><div class="casAuthors">Fallmann, Joerg; Will, Sebastian; Engelhardt, Jan; Gruening, Bjoern; Backofen, Rolf; Stadler, Peter F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biotechnology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">97-104</span>CODEN:
                <span class="NLM_cas:coden">JBITD4</span>;
        ISSN:<span class="NLM_cas:issn">0168-1656</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In the realm of nucleic acid structures, secondary structure forms a conceptually important intermediate level of description and explains the dominating part of the free energy of structure formation.  Secondary structures are well conserved over evolutionary time-scales and for many classes of RNAs evolve slower than the underlying primary sequences.  Given the close link between structure and function, secondary structure is routinely used as a basis to explain exptl. findings.  Recent technol. advances, finally, have made it possible to assay secondary structure directly using high throughput methods.  From a computational biol. point of view, secondary structures have a special role because they can be computed efficiently using exact dynamic programming algorithms.  In this contribution we provide a short overview of RNA folding algorithms, recent addns. and variations and address methods to align, compare, and cluster RNA structures, followed by a tabular summary of the most important software suites in the fields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjwBWQrP9xYLVg90H21EOLACvtfcHk0lgyBMP-2GRiCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Sls73E&md5=ffd726b00ff26ee6fd6cc1e952b3a97b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.jbiotec.2017.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbiotec.2017.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DFallmann%26aufirst%3DJ.%26aulast%3DWill%26aufirst%3DS.%26aulast%3DEngelhardt%26aufirst%3DJ.%26aulast%3DGr%25C3%25BCning%26aufirst%3DB.%26aulast%3DBackofen%26aufirst%3DR.%26aulast%3DStadler%26aufirst%3DP.%2BF.%26atitle%3DRecent%2520advances%2520in%2520RNA%2520folding%26jtitle%3DJ.%2520Biotechnol.%26date%3D2017%26volume%3D261%26spage%3D97%26epage%3D104%26doi%3D10.1016%2Fj.jbiotec.2017.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bevilacqua, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritchey, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assmann, S. M.</span></span> <span> </span><span class="NLM_article-title">Genome-wide analysis of RNA secondary structure</span>. <i>Annu. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1146/annurev-genet-120215-035034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1146%2Fannurev-genet-120215-035034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27648642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFemtrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2016&pages=235-266&author=P.+C.+Bevilacquaauthor=L.+E.+Ritcheyauthor=Z.+Suauthor=S.+M.+Assmann&title=Genome-wide+analysis+of+RNA+secondary+structure&doi=10.1146%2Fannurev-genet-120215-035034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-Wide Analysis of RNA Secondary Structure</span></div><div class="casAuthors">Bevilacqua, Philip C.; Ritchey, Laura E.; Su, Zhao; Assmann, Sarah M.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Genetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">235-266</span>CODEN:
                <span class="NLM_cas:coden">ARVGB7</span>;
        ISSN:<span class="NLM_cas:issn">0066-4197</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  Single-stranded RNA mols. fold into extraordinarily complicated secondary and tertiary structures as a result of intramol. base pairing.  In vivo, these RNA structures are not static.  Instead, they are remodeled in response to changes in the prevailing physicochem. environment of the cell and as a result of intermol. base pairing and interactions with RNA-binding proteins.  Remarkable tech. advances now allow us to probe RNA secondary structure at single-nucleotide resoln. and genome-wide, both in vitro and in vivo.  These data sets provide new glimpses into the RNA universe.  Analyses of RNA structuromes in HIV, yeast, Arabidopsis, and mammalian cells and tissues have revealed regulatory effects of RNA structure on mRNA (mRNA) polyadenylation, splicing, translation, and turnover.  Application of new methods for genome-wide identification of mRNA modifications, particularly methylation and pseudouridylation, has shown that the RNA "epitranscriptome" both influences and is influenced by RNA structure.  In this review, we describe newly developed genome-wide RNA structure-probing methods and synthesize the information emerging from their application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptXBfgVqAC-rVg90H21EOLACvtfcHk0lhQGZe1FRsGaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFemtrzM&md5=6213387ba5787285eeaabd1b624787a3</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1146%2Fannurev-genet-120215-035034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-genet-120215-035034%26sid%3Dliteratum%253Aachs%26aulast%3DBevilacqua%26aufirst%3DP.%2BC.%26aulast%3DRitchey%26aufirst%3DL.%2BE.%26aulast%3DSu%26aufirst%3DZ.%26aulast%3DAssmann%26aufirst%3DS.%2BM.%26atitle%3DGenome-wide%2520analysis%2520of%2520RNA%2520secondary%2520structure%26jtitle%3DAnnu.%2520Rev.%2520Genet.%26date%3D2016%26volume%3D50%26spage%3D235%26epage%3D266%26doi%3D10.1146%2Fannurev-genet-120215-035034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iida, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, T.-a.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takaiwa, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwabuchi, M.</span></span> <span> </span><span class="NLM_article-title">Phylogenetic relationships among eukaryotic kingdoms inferred from ribosomal RNA sequences</span>. <i>J. Mol. Evol.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1007/BF02105802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1007%2FBF02105802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=3932662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaL2MXltFygu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1985&pages=32-38&author=M.+Hasegawaauthor=Y.+Iidaauthor=T.-a.+Yanoauthor=F.+Takaiwaauthor=M.+Iwabuchi&title=Phylogenetic+relationships+among+eukaryotic+kingdoms+inferred+from+ribosomal+RNA+sequences&doi=10.1007%2FBF02105802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Phylogenetic relationships among eukaryotic kingdoms inferred from ribosomal RNA sequences</span></div><div class="casAuthors">Hasegawa, Masami; Iida, Yoichi; Yano, Takaaki; Takaiwa, Fumio; Iwabuchi, Masaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Evolution</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-8</span>CODEN:
                <span class="NLM_cas:coden">JMEVAU</span>;
        ISSN:<span class="NLM_cas:issn">0022-2844</span>.
    </div><div class="casAbstract">Phylogenetic trees among eukaryotic kingdoms were inferred for large- and small-subunit rRNAs by using a max.-likelihood method developed by J. Felsenstein (1981).  Although Felsenstein's method assumes equal evolutionary rates for transitions and transversions, this is apparently not the case for these data.  Therefore, only transversion-type substitutions were taken into account.  The mols. used were large-subunit rRNAs from Xenopus laevis (Animalia), rice (Plantae), Saccharomyces cerevisiae (Fungi), Dictyostelium discoideum (Protista), and Physarum polycephalum (Protista), and small-subunit rRNAs from maize (Plantae), S. cerevisiae, X. laevis, rat (Animalia), and D. discoideum.  Only conservative regions of the nucleotide sequences were considered for this study.  In the max.-likelihood trees for both large- and small-subunit rRNAs, Animalia and Fungi were the most closely related eukaryotic kingdoms, and Plantae was the next most closely related kingdom, although other branching orders among Plantae, Animalia, and Fungi were not excluded by this work.  These 3 eukaryotic kingdoms apparently shared a common ancestor after the divergence of the 2 species of Protista, D. discoideum and P. polycephalum.  These 2 species of Protista do not form a clade, and P. polycephalum diverged 1st and D. discoideum 2nd from the line leading to the common ancestor of Plantae, Animalia, and Fungi.  The sequence data indicate that a drastic change occurred in the nucleotide sequences of rRNAs during the evolutionary sepn. between prokaryote and eukaryote.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgdQQQ9ZpMZLVg90H21EOLACvtfcHk0lhQGZe1FRsGaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXltFygu7o%253D&md5=17434f208c671f54566aa086a643e533</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1007%2FBF02105802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02105802%26sid%3Dliteratum%253Aachs%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DIida%26aufirst%3DY.%26aulast%3DYano%26aufirst%3DT.-a.%26aulast%3DTakaiwa%26aufirst%3DF.%26aulast%3DIwabuchi%26aufirst%3DM.%26atitle%3DPhylogenetic%2520relationships%2520among%2520eukaryotic%2520kingdoms%2520inferred%2520from%2520ribosomal%2520RNA%2520sequences%26jtitle%3DJ.%2520Mol.%2520Evol.%26date%3D1985%26volume%3D22%26spage%3D32%26epage%3D38%26doi%3D10.1007%2FBF02105802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turner, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, D. H.</span></span> <span> </span><span class="NLM_article-title">NNDB: the nearest neighbor parameter database for predicting stability of nucleic acid secondary structure</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">D280</span>– <span class="NLM_lpage">D282</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1093%2Fnar%2Fgkp892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19880381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktlOqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=D280-D282&author=D.+H.+Turnerauthor=D.+H.+Mathews&title=NNDB%3A+the+nearest+neighbor+parameter+database+for+predicting+stability+of+nucleic+acid+secondary+structure&doi=10.1093%2Fnar%2Fgkp892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">NNDB: the nearest neighbor parameter database for predicting stability of nucleic acid secondary structure</span></div><div class="casAuthors">Turner, Douglas H.; Mathews, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D280-D282</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Nearest Neighbor Database (NNDB, http://rna.urmc.rochester.edu/NNDB) is a web-based resource for disseminating parameter sets for predicting nucleic acid secondary structure stabilities.  For each set of parameters, the database includes the set of rules with descriptive text, sequence-dependent parameters in plain text and html, literature refs. to expts. and usage tutorials.  The initial release covers parameters for predicting RNA folding free energy and enthalpy changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg3uUXrM-0_bVg90H21EOLACvtfcHk0lhQGZe1FRsGaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktlOqtA%253D%253D&md5=7171efa8b463e554cc893f4cc241dd15</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp892%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DD.%2BH.%26aulast%3DMathews%26aufirst%3DD.%2BH.%26atitle%3DNNDB%253A%2520the%2520nearest%2520neighbor%2520parameter%2520database%2520for%2520predicting%2520stability%2520of%2520nucleic%2520acid%2520secondary%2520structure%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DD280%26epage%3DD282%26doi%3D10.1093%2Fnar%2Fgkp892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuker, M.</span></span> <span> </span><span class="NLM_article-title">On finding all suboptimal foldings of an RNA molecule</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>244</i></span>,  <span class="NLM_fpage">48</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1126/science.2468181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1126%2Fscience.2468181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=2468181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaL1MXkt1SnsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=244&publication_year=1989&pages=48-52&author=M.+Zuker&title=On+finding+all+suboptimal+foldings+of+an+RNA+molecule&doi=10.1126%2Fscience.2468181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">On finding all suboptimal foldings of an RNA molecule</span></div><div class="casAuthors">Zuker, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">244</span>
        (<span class="NLM_cas:issue">4900</span>),
    <span class="NLM_cas:pages">48-52</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">An algorithm and a computer program were prepd. for detg. RNA secondary structures within any prescribed increment of the computed global min. free energy.  The math. problem of detg. how well defined a min. energy folding is can now be solved.  All predicted base pairs that can participate in suboptimal structures may be displayed and analyzed graphically.  Representative suboptimal foldings are generated by selecting these base pairs one at a time and computing the best foldings that contain them.  A distance criterion that ensures that no 2 structures are too close is used to avoid multiple generation of similar structures.  Thermodn. parameters, including free energy increments for single-base stacking at the ends of helices and for terminal mismatched pairs in interior and hairpin loops, are incorporated into the underlying folding model of the above algorithm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRk6_lhTFKLbVg90H21EOLACvtfcHk0lhQGZe1FRsGaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXkt1SnsbY%253D&md5=727f88c4e1f5bda392240e981db256a3</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1126%2Fscience.2468181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2468181%26sid%3Dliteratum%253Aachs%26aulast%3DZuker%26aufirst%3DM.%26atitle%3DOn%2520finding%2520all%2520suboptimal%2520foldings%2520of%2520an%2520RNA%2520molecule%26jtitle%3DScience%26date%3D1989%26volume%3D244%26spage%3D48%26epage%3D52%26doi%3D10.1126%2Fscience.2468181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, D. H.</span></span> <span> </span><span class="NLM_article-title">Incorporating chemical modification constraints into a dynamic programming algorithm for prediction of RNA secondary structure</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">7287</span>– <span class="NLM_lpage">7292</span>, <span class="refDoi"> DOI: 10.1073/pnas.0401799101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.0401799101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=15123812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFaks7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=7287-7292&author=D.+H.+Mathewsauthor=M.+D.+Disneyauthor=J.+L.+Childsauthor=S.+J.+Schroederauthor=M.+Zukerauthor=D.+H.+Turner&title=Incorporating+chemical+modification+constraints+into+a+dynamic+programming+algorithm+for+prediction+of+RNA+secondary+structure&doi=10.1073%2Fpnas.0401799101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Incorporating chemical modification constraints into a dynamic programming algorithm for prediction of RNA secondary structure</span></div><div class="casAuthors">Mathews, David H.; Disney, Matthew D.; Childs, Jessica L.; Schroeder, Susan J.; Zuker, Michael; Turner, Douglas H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7287-7292</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A dynamic programming algorithm for prediction of RNA secondary structure has been revised to accommodate folding constraints detd. by chem. modification and to include free energy increments for coaxial stacking of helixes when they are either adjacent or sepd. by a single mismatch.  Furthermore, free energy parameters are revised to account for recent exptl. results for terminal mismatches and hairpin, bulge, internal, and multibranch loops.  To demonstrate the applicability of this method, in vivo modification was performed on 5S rRNA in both Escherichia coli and Candida albicans with 1-cyclohexyl-3-(2-morpholinoethyl) carbodiimide metho-p-toluene sulfonate, di-Me sulfate, and kethoxal.  The percentage of known base pairs in the predicted structure increased from 26.3% to 86.8% for the E. coli sequence by using modification constraints.  For C. albicans, the accuracy remained 87.5% both with and without modification data.  On av., for these sequences and a set of 14 sequences with known secondary structure and chem. modification data taken from the literature, accuracy improves from 67% to 76%.  This enhancement primarily reflects improvement for three sequences that are predicted with <40% accuracy on the basis of energetics alone.  For these sequences, inclusion of chem. modification constraints improves the av. accuracy from 28% to 78%.  For the 11 sequences with <6% pseudoknotted base pairs, structures predicted with constraints from chem. modification contain on av. 84% of known canonical base pairs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqzYqTEJBW1bVg90H21EOLACvtfcHk0li_ybpSjqG10A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFaks7s%253D&md5=cbe3a05189b963f2bee130567335a72d</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0401799101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0401799101%26sid%3Dliteratum%253Aachs%26aulast%3DMathews%26aufirst%3DD.%2BH.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DChilds%26aufirst%3DJ.%2BL.%26aulast%3DSchroeder%26aufirst%3DS.%2BJ.%26aulast%3DZuker%26aufirst%3DM.%26aulast%3DTurner%26aufirst%3DD.%2BH.%26atitle%3DIncorporating%2520chemical%2520modification%2520constraints%2520into%2520a%2520dynamic%2520programming%2520algorithm%2520for%2520prediction%2520of%2520RNA%2520secondary%2520structure%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D7287%26epage%3D7292%26doi%3D10.1073%2Fpnas.0401799101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouskin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zubradt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washietl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span> <span> </span><span class="NLM_article-title">Genome-wide probing of RNA structure reveals active unfolding of mRNA structures in vivo</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>505</i></span>,  <span class="NLM_fpage">701</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1038/nature12894</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnature12894" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=24336214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=505&publication_year=2014&pages=701-705&author=S.+Rouskinauthor=M.+Zubradtauthor=S.+Washietlauthor=M.+Kellisauthor=J.+S.+Weissman&title=Genome-wide+probing+of+RNA+structure+reveals+active+unfolding+of+mRNA+structures+in+vivo&doi=10.1038%2Fnature12894"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide probing of RNA structure reveals active unfolding of mRNA structures in vivo</span></div><div class="casAuthors">Rouskin, Silvi; Zubradt, Meghan; Washietl, Stefan; Kellis, Manolis; Weissman, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">505</span>
        (<span class="NLM_cas:issue">7485</span>),
    <span class="NLM_cas:pages">701-705</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA has a dual role as an informational mol. and a direct effector of biol. tasks.  The latter function is enabled by RNA's ability to adopt complex secondary and tertiary folds and thus has motivated extensive computational and exptl. efforts for detg. RNA structures.  Existing approaches for evaluating RNA structure have been largely limited to in vitro systems, yet the thermodn. forces which drive RNA folding in vitro may not be sufficient to predict stable RNA structures in vivo.  Indeed, the presence of RNA-binding proteins and ATP-dependent helicases can influence which structures are present inside cells.  Here we present an approach for globally monitoring RNA structure in native conditions in vivo with single-nucleotide precision.  This method is based on in vivo modification with di-Me sulfate (DMS), which reacts with unpaired adenine and cytosine residues, followed by deep sequencing to monitor modifications.  Our data from yeast and mammalian cells are in excellent agreement with known mRNA structures and with the high-resoln. crystal structure of the Saccharomyces cerevisiae ribosome.  Comparison between in vivo and in vitro data reveals that in rapidly dividing cells there are vastly fewer structured mRNA regions in vivo than in vitro.  Even thermostable RNA structures are often denatured in cells, highlighting the importance of cellular processes in regulating RNA structure.  Indeed, anal. of mRNA structure under ATP-depleted conditions in yeast shows that energy-dependent processes strongly contribute to the predominantly unfolded state of mRNAs inside cells.  Our studies broadly enable the functional anal. of physiol. RNA structures and reveal that, in contrast to the Anfinsen view of protein folding whereby the structure formed is the most thermodynamically favorable, thermodn. have an incomplete role in detg. mRNA structure in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquJ_RYbQmU7bVg90H21EOLACvtfcHk0li_ybpSjqG10A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFentbc%253D&md5=f04acd7ce0fa8ed94e1cd428c769bbc6</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnature12894&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12894%26sid%3Dliteratum%253Aachs%26aulast%3DRouskin%26aufirst%3DS.%26aulast%3DZubradt%26aufirst%3DM.%26aulast%3DWashietl%26aufirst%3DS.%26aulast%3DKellis%26aufirst%3DM.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26atitle%3DGenome-wide%2520probing%2520of%2520RNA%2520structure%2520reveals%2520active%2520unfolding%2520of%2520mRNA%2520structures%2520in%2520vivo%26jtitle%3DNature%26date%3D2014%26volume%3D505%26spage%3D701%26epage%3D705%26doi%3D10.1038%2Fnature12894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zubradt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambowitz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouskin, S.</span></span> <span> </span><span class="NLM_article-title">DMS-MaPseq for genome-wide or targeted RNA structure probing in vivo</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1038/nmeth.4057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnmeth.4057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27819661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSiurfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=75-82&author=M.+Zubradtauthor=P.+Guptaauthor=S.+Persadauthor=A.+M.+Lambowitzauthor=J.+S.+Weissmanauthor=S.+Rouskin&title=DMS-MaPseq+for+genome-wide+or+targeted+RNA+structure+probing+in+vivo&doi=10.1038%2Fnmeth.4057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">DMS-MaPseq for genome-wide or targeted RNA structure probing in vivo</span></div><div class="casAuthors">Zubradt, Meghan; Gupta, Paromita; Persad, Sitara; Lambowitz, Alan M.; Weissman, Jonathan S.; Rouskin, Silvi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-82</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Coupling of structure-specific in vivo chem. modification to next-generation sequencing is transforming RNA secondary structure studies in living cells.  The dominant strategy for detecting in vivo chem. modifications uses reverse transcriptase truncation products, which introduce biases and necessitate population-av. assessments of RNA structure.  Here we present di-Me sulfate (DMS) mutational profiling with sequencing (DMS-MaPseq), which encodes DMS modifications as mismatches using a thermostable group II intron reverse transcriptase.  DMS-MaPseq yields a high signal-to-noise ratio, can report multiple structural features per mol., and allows both genome-wide studies and focused in vivo investigations of even low-abundance RNAs.  We apply DMS-MaPseq for the first anal. of RNA structure within an animal tissue and to identify a functional structure involved in noncanonical translation initiation.  Addnl., we use DMS-MaPseq to compare the in vivo structure of pre-mRNAs with their mature isoforms.  These applications illustrate DMS-MaPseq's capacity to dramatically expand in vivo anal. of RNA structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY8vORTX4NyrVg90H21EOLACvtfcHk0li_ybpSjqG10A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSiurfE&md5=bbc171334f1930859ea327c7d5744f76</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.4057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.4057%26sid%3Dliteratum%253Aachs%26aulast%3DZubradt%26aufirst%3DM.%26aulast%3DGupta%26aufirst%3DP.%26aulast%3DPersad%26aufirst%3DS.%26aulast%3DLambowitz%26aufirst%3DA.%2BM.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26aulast%3DRouskin%26aufirst%3DS.%26atitle%3DDMS-MaPseq%2520for%2520genome-wide%2520or%2520targeted%2520RNA%2520structure%2520probing%2520in%2520vivo%26jtitle%3DNat.%2520Methods%26date%3D2017%26volume%3D14%26spage%3D75%26epage%3D82%26doi%3D10.1038%2Fnmeth.4057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rivas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clements, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddy, S. R.</span></span> <span> </span><span class="NLM_article-title">A statistical test for conserved RNA structure shows lack of evidence for structure in lncRNAs</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1038/nmeth.4066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnmeth.4066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27819659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSiu77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=45-48&author=E.+Rivasauthor=J.+Clementsauthor=S.+R.+Eddy&title=A+statistical+test+for+conserved+RNA+structure+shows+lack+of+evidence+for+structure+in+lncRNAs&doi=10.1038%2Fnmeth.4066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">A statistical test for conserved RNA structure shows lack of evidence for structure in lncRNAs</span></div><div class="casAuthors">Rivas, Elena; Clements, Jody; Eddy, Sean R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-48</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Many functional RNAs have an evolutionarily conserved secondary structure.  Conservation of RNA base pairing induces pairwise covariations in sequence alignments.  We developed a computational method, R-scape (RNA Structural Covariation Above Phylogenetic Expectation), that quant. tests whether covariation anal. supports the presence of a conserved RNA secondary structure.  R-scape anal. finds no statistically significant support for proposed secondary structures of the long noncoding RNAs HOTAIR, SRA, and Xist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnFROMK2OjrVg90H21EOLACvtfcHk0lgxQ2ZQevFlmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSiu77J&md5=689c629a3940f4a64c8f43a6b2ba8155</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.4066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.4066%26sid%3Dliteratum%253Aachs%26aulast%3DRivas%26aufirst%3DE.%26aulast%3DClements%26aufirst%3DJ.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26atitle%3DA%2520statistical%2520test%2520for%2520conserved%2520RNA%2520structure%2520shows%2520lack%2520of%2520evidence%2520for%2520structure%2520in%2520lncRNAs%26jtitle%3DNat.%2520Methods%26date%3D2017%26volume%3D14%26spage%3D45%26epage%3D48%26doi%3D10.1038%2Fnmeth.4066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span> <span> </span><span class="NLM_article-title">ScanFold: an approach for genome-wide discovery of local RNA structural elements-applications to Zika virus and HIV</span>. <i>PeerJ</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e6136</span> <span class="refDoi"> DOI: 10.7717/peerj.6136</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.7717%2Fpeerj.6136" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30627482" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&author=R.+J.+Andrewsauthor=J.+Rocheauthor=W.+N.+Moss&title=ScanFold%3A+an+approach+for+genome-wide+discovery+of+local+RNA+structural+elements-applications+to+Zika+virus+and+HIV&doi=10.7717%2Fpeerj.6136"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.7717%2Fpeerj.6136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7717%252Fpeerj.6136%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DR.%2BJ.%26aulast%3DRoche%26aufirst%3DJ.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26atitle%3DScanFold%253A%2520an%2520approach%2520for%2520genome-wide%2520discovery%2520of%2520local%2520RNA%2520structural%2520elements-applications%2520to%2520Zika%2520virus%2520and%2520HIV%26jtitle%3DPeerJ%26date%3D2018%26volume%3D6%26doi%3D10.7717%2Fpeerj.6136" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, G. J.</span></span> <span> </span><span class="NLM_article-title">MicroRNAs: small RNAs with a big role in gene regulation</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">522</span>– <span class="NLM_lpage">531</span>, <span class="refDoi"> DOI: 10.1038/nrg1379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrg1379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=15211354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD2cXltVSmsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=522-531&author=L.+Heauthor=G.+J.+Hannon&title=MicroRNAs%3A+small+RNAs+with+a+big+role+in+gene+regulation&doi=10.1038%2Fnrg1379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNAs: small RNAs with a big role in gene regulation</span></div><div class="casAuthors">He, Lin; Hannon, Gregory J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">522-531</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  MicroRNAs are a family of small, non-coding RNAs that regulate gene expression in a sequence-specific manner.  The two founding members of the microRNA family were originally identified in Caenorhabditis elegans as genes that were required for the timed regulation of developmental events.  Since then, hundreds of microRNAs have been identified in almost all metazoan genomes, including worms, flies, plants and mammals.  MicroRNAs have diverse expression patterns and might regulate various developmental and physiol. processes.  Their discovery adds a new dimension to our understanding of complex gene regulatory networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp1kiZyCfuZ7Vg90H21EOLACvtfcHk0lgxQ2ZQevFlmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXltVSmsbw%253D&md5=d4d40468a3dab2b1142be9f4d6735980</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnrg1379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg1379%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DL.%26aulast%3DHannon%26aufirst%3DG.%2BJ.%26atitle%3DMicroRNAs%253A%2520small%2520RNAs%2520with%2520a%2520big%2520role%2520in%2520gene%2520regulation%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2004%26volume%3D5%26spage%3D522%26epage%3D531%26doi%3D10.1038%2Fnrg1379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">RNA structures as mediators of neurological diseases and as drug targets</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2015.06.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.neuron.2015.06.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=26139368" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WksrzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2015&pages=28-46&author=V.+Bernatauthor=M.+D.+Disney&title=RNA+structures+as+mediators+of+neurological+diseases+and+as+drug+targets&doi=10.1016%2Fj.neuron.2015.06.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">RNA structures as mediators of neurological diseases and as drug targets</span></div><div class="casAuthors">Bernat, Viachaslau; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-46</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">RNAs adopt diverse folded structures that are essential for function and thus play crit. roles in cellular biol.  A striking example of this is the ribosome, a complex, three-dimensionally folded macromol. machine that orchestrates protein synthesis.  Advances in RNA biochem., structural and mol. biol., and bioinformatics have revealed other non-coding RNAs whose functions are dictated by their structure.  It is not surprising that aberrantly folded RNA structures contribute to disease.  In this Review, we provide a brief introduction into RNA structural biol. and then describe how RNA structures function in cells and cause or contribute to neurol. disease.  Finally, we highlight successful applications of rational design principles to provide chem. probes and lead compds. targeting structured RNAs.  Based on several examples of well-characterized RNA-driven neurol. disorders, we demonstrate how designed small mols. can facilitate the study of RNA dysfunction, elucidating previously unknown roles for RNA in disease, and provide lead therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIX63rZ0lwjbVg90H21EOLACvtfcHk0lgesAcyRY_SiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WksrzJ&md5=5b93b94ae980c7ac0a0a8b99e76ebe40</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2015.06.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2015.06.012%26sid%3Dliteratum%253Aachs%26aulast%3DBernat%26aufirst%3DV.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DRNA%2520structures%2520as%2520mediators%2520of%2520neurological%2520diseases%2520and%2520as%2520drug%2520targets%26jtitle%3DNeuron%26date%3D2015%26volume%3D87%26spage%3D28%26epage%3D46%26doi%3D10.1016%2Fj.neuron.2015.06.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C. F.</span></span> <span> </span><span class="NLM_article-title">Therapeutic antisense oligonucleotides are coming of age</span>. <i>Annu. Rev. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">321</span>, <span class="refDoi"> DOI: 10.1146/annurev-med-041217-010829</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1146%2Fannurev-med-041217-010829" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30691367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvF2lsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2019&pages=307-321&author=C.+F.+Bennett&title=Therapeutic+antisense+oligonucleotides+are+coming+of+age&doi=10.1146%2Fannurev-med-041217-010829"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Antisense Oligonucleotides Are Coming of Age</span></div><div class="casAuthors">Bennett, C. Frank</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">307-321</span>CODEN:
                <span class="NLM_cas:coden">ARMCAH</span>;
        ISSN:<span class="NLM_cas:issn">0066-4219</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The first published description of therapeutic applications of antisense oligonucleotide (ASO) technol. occurred in the late 1970s and was followed by the founding of com. companies focused on developing antisense therapeutics in the late 1980s.  Since the late 1980s, there has been steady progress in improving the technol. platform, taking advantage of advances in oligonucleotide chem. and formulations as well as increased understanding of the distribution and safety of ASOs.  There are several approved ASO drugs and a broad pipeline in development.  In addn., advances in understanding human disease, including the genetic basis for most monogenic diseases and the availability of the full human genome sequence, have created numerous therapeutic applications for the technol.  I summarize the state of the technol. and highlight how advances in the technol. position ASOs to be an important contributor to future medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQKzgrxHUp1LVg90H21EOLACvtfcHk0lgesAcyRY_SiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvF2lsbs%253D&md5=6c7f4a07724b43b769c8014ca3108e6a</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1146%2Fannurev-med-041217-010829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-med-041217-010829%26sid%3Dliteratum%253Aachs%26aulast%3DBennett%26aufirst%3DC.%2BF.%26atitle%3DTherapeutic%2520antisense%2520oligonucleotides%2520are%2520coming%2520of%2520age%26jtitle%3DAnnu.%2520Rev.%2520Med.%26date%3D2019%26volume%3D70%26spage%3D307%26epage%3D321%26doi%3D10.1146%2Fannurev-med-041217-010829" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagedorn, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindow, M.</span></span> <span> </span><span class="NLM_article-title">Managing the sequence-specificity of antisense oligonucleotides in drug discovery</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2262</span>– <span class="NLM_lpage">2282</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1093%2Fnar%2Fgkx056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28426096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFygs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=2262-2282&author=P.+H.+Hagedornauthor=B.+R.+Hansenauthor=T.+Kochauthor=M.+Lindow&title=Managing+the+sequence-specificity+of+antisense+oligonucleotides+in+drug+discovery&doi=10.1093%2Fnar%2Fgkx056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Managing the sequence-specificity of antisense oligonucleotides in drug discovery</span></div><div class="casAuthors">Hagedorn, Peter H.; Hansen, Bo R.; Koch, Troels; Lindow, Morten</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2262-2282</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  All drugs perturb the expression of many genes in the cells that are exposed to them.  These gene expression changes can be divided into effects resulting from engaging the intended target and effects resulting from engaging unintended targets.  For antisense oligonucleotides, developments in bioinformatics algorithms, and the quality of sequence databases, allow oligonucleotide sequences to be analyzed computationally, in terms of the predictability of their interactions with intended and unintended RNA targets.  Applying these tools enables selection of sequence-specific oligonucleotides where no- or only few unintended RNA targets are expected.  To evaluate oligonucleotide sequence-specificity exptl., we recommend a transcriptomics protocol where two or more oligonucleotides targeting the same RNA mol., but with entirely different sequences, are evaluated together.  This helps to clarify which changes in cellular RNA levels result from downstream processes of engaging the intended target, and which are likely to be related to engaging unintended targets.  As required for all classes of drugs, the toxic potential of oligonucleotides must be evaluated in cell- and animal models before clin. testing.  Since potential adverse effects related to unintended targeting are sequence-dependent and therefore species-specific, in vitro toxicol. assays in human cells are esp. relevant in oligonucleotide drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq77hjabvSVxLVg90H21EOLACvtfcHk0lgesAcyRY_SiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFygs7g%253D&md5=ab9bc075d7a76d0d7a837c8af00899d7</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx056%26sid%3Dliteratum%253Aachs%26aulast%3DHagedorn%26aufirst%3DP.%2BH.%26aulast%3DHansen%26aufirst%3DB.%2BR.%26aulast%3DKoch%26aufirst%3DT.%26aulast%3DLindow%26aufirst%3DM.%26atitle%3DManaging%2520the%2520sequence-specificity%2520of%2520antisense%2520oligonucleotides%2520in%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D2262%26epage%3D2282%26doi%3D10.1093%2Fnar%2Fgkx056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papoian, T.</span></span> <span> </span><span class="NLM_article-title">Safety of antisense oligonucleotide and siRNA-based therapeutics</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">823</span>– <span class="NLM_lpage">833</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.drudis.2017.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28159625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1eqs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=823-833&author=X.+Chiauthor=P.+Gattiauthor=T.+Papoian&title=Safety+of+antisense+oligonucleotide+and+siRNA-based+therapeutics&doi=10.1016%2Fj.drudis.2017.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Safety of antisense oligonucleotide and siRNA-based therapeutics</span></div><div class="casAuthors">Chi, Xuan; Gatti, Philip; Papoian, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">823-833</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Oligonucleotide-based therapy is an active area of drug development designed to treat a variety of gene-specific diseases.  Two of the more promising platforms are the antisense oligonucleotides (ASOs) and short interfering RNAs (siRNAs), both of which are often directed against similar targets.  In light of recent reports on clin. trials of severe thrombocytopenia with two different ASO drugs and increased peripheral neuropathy with an siRNA drug, we compared and contrasted the specific safety characteristics of these two classes of oligonucleotide therapeutic.  The objectives were to assess factors that could contribute to the specific toxicities obsd. with these two classes of promising drugs, and get a better understanding of the potential mechanism(s) responsible for these rare, but serious, adverse events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Ej7iOK40PbVg90H21EOLACvtfcHk0lgftWQEldUIiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1eqs70%253D&md5=40ef4591957b37ab501c95f53b6aa07b</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DChi%26aufirst%3DX.%26aulast%3DGatti%26aufirst%3DP.%26aulast%3DPapoian%26aufirst%3DT.%26atitle%3DSafety%2520of%2520antisense%2520oligonucleotide%2520and%2520siRNA-based%2520therapeutics%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26spage%3D823%26epage%3D833%26doi%3D10.1016%2Fj.drudis.2017.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frazier, K. S.</span></span> <span> </span><span class="NLM_article-title">Antisense oligonucleotide therapies: the promise and the challenges from a toxicologic pathologist’s perspective</span>. <i>Toxicol. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1177/0192623314551840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1177%2F0192623314551840" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=25385330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislWktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=78-89&author=K.+S.+Frazier&title=Antisense+oligonucleotide+therapies%3A+the+promise+and+the+challenges+from+a+toxicologic+pathologist%E2%80%99s+perspective&doi=10.1177%2F0192623314551840"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Antisense oligonucleotide therapies: the promise and the challenges from a toxicologic pathologist's perspective</span></div><div class="casAuthors">Frazier, Kendall S.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicologic Pathology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-89/1-78-89/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">TOPADD</span>;
        ISSN:<span class="NLM_cas:issn">0192-6233</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Many antisense oligonucleotides (ASOs) from several classes of mols. are currently in drug development.  Despite over 20 years of pharmaceutical research, few ASOs have been marketed due to problems with clin. efficacy or preclin. toxicol. challenges.  However, a no. of recent developments have renewed interest in this class including the registration of mipomersen, the advent of successful screening strategies to eliminate more toxic mols., and new understanding of the risks of off-target nucleotide binding and mitigation of potential off-target effects.  Recent advances in backbone chemistries, conjugation to other moieties, and new delivery systems have allowed better tissue penetration, enhanced intracellular targeting, and less frequent dosing, resulting in fewer toxicities.  While these new developments provide invigorated interest in these platforms, a few lingering challenges and preclin./clin. toxicity issues remain to be completely resolved, including: (1) proinflammatory effects (vasculitis/inflammatory infiltrates); (2) nephrotoxicity and hepatotoxicity unrelated to lysosomal accumulation; and (3) thrombocytopenia.  Recent investigative work by several labs. have helped elucidate mechanisms for these issues, allowing a better understanding of the clin. relevance and implications of particular toxicities.  It is important for toxicologists, pathologists, and regulatory reviewers to be familiar with new developments in the ASO field and their implications, as a greater no. of new types of antisense mols. undergo preclin. toxicity testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsxbnyuLQ_MbVg90H21EOLACvtfcHk0lgftWQEldUIiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislWktbc%253D&md5=08881cd4d57355fcc3a53dd6bad5368f</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1177%2F0192623314551840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0192623314551840%26sid%3Dliteratum%253Aachs%26aulast%3DFrazier%26aufirst%3DK.%2BS.%26atitle%3DAntisense%2520oligonucleotide%2520therapies%253A%2520the%2520promise%2520and%2520the%2520challenges%2520from%2520a%2520toxicologic%2520pathologist%25E2%2580%2599s%2520perspective%26jtitle%3DToxicol.%2520Pathol.%26date%3D2015%26volume%3D43%26spage%3D78%26epage%3D89%26doi%3D10.1177%2F0192623314551840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimonet, P.</span></span> <span> </span><span class="NLM_article-title">New modalities, technologies, and partnerships in probe and lead generation: Enabling a mode-of-action centric paradigm</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9004</span>– <span class="NLM_lpage">9029</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00378</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00378" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVGjs77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9004-9029&author=E.+Valeurauthor=P.+Jimonet&title=New+modalities%2C+technologies%2C+and+partnerships+in+probe+and+lead+generation%3A+Enabling+a+mode-of-action+centric+paradigm&doi=10.1021%2Facs.jmedchem.8b00378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">New Modalities, Technologies, and Partnerships in Probe and Lead Generation: Enabling a Mode-of-Action Centric Paradigm</span></div><div class="casAuthors">Valeur, Eric; Jimonet, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9004-9029</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the rise of novel biol. and high potential target identification technologies originating from advances in genomics, medicinal chemists are progressively facing targets of increasing complexity and often unprecedented.  Novel hit finding technologies, combined with a wider choice of drug modalities, has resulted in a unique repertoire of options to address these challenging targets and to identify suitable starting points for optimization.  Furthermore, innovative solns. originating from a range of academic groups and biotech companies require new types of collaborative models to leverage and integrate them in the drug discovery process.  This perspective provides a guide for medicinal chemists covering contemporary probe and lead generation approaches and discusses the strengths and limitations of each strategy.  Moreover, the expansion of strategies to modulate proteins creates the opportunity of a modality-agnostic and mode-of-action centric hit finding paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS8l5d3NOahrVg90H21EOLACvtfcHk0ljs3NyGyXY3_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVGjs77P&md5=62e768509dac90567f3f021ed83b1df1</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00378%26sid%3Dliteratum%253Aachs%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DJimonet%26aufirst%3DP.%26atitle%3DNew%2520modalities%252C%2520technologies%252C%2520and%2520partnerships%2520in%2520probe%2520and%2520lead%2520generation%253A%2520Enabling%2520a%2520mode-of-action%2520centric%2520paradigm%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9004%26epage%3D9029%26doi%3D10.1021%2Facs.jmedchem.8b00378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herschlag, D.</span></span> <span> </span><span class="NLM_article-title">Implications of ribozyme kinetics for targeting the cleavage of specific RNA molecules in vivo: more isn’t always better</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">6921</span>– <span class="NLM_lpage">6925</span>, <span class="refDoi"> DOI: 10.1073/pnas.88.16.6921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.88.16.6921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1871108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaK3MXlsFekt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=1991&pages=6921-6925&author=D.+Herschlag&title=Implications+of+ribozyme+kinetics+for+targeting+the+cleavage+of+specific+RNA+molecules+in+vivo%3A+more+isn%E2%80%99t+always+better&doi=10.1073%2Fpnas.88.16.6921"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Implications of ribozyme kinetics for targeting the cleavage of specific RNA molecules in vivo:  more is not always better</span></div><div class="casAuthors">Herschlag, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6921-5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Kinetic and thermodn. factors that det. specificity of RNA cleavage by ribozymes are illustrated with examples from recent work with a ribozyme derived from the group I intron of Tetrahymena thermophila pre-rRNA.  The conclusions also apply to other ribozymes, to antisense oligonucleotide expts., and to RNA and DNA cleavage agents that can recognize a single-stranded or double-stranded region of variable length.  At first, adding bases to a ribozyme's recognition sequence is expected to increase cleavage of the target RNA relative to cleavage of other RNAs.  However, adding more bases ultimately reduces this discrimination, as cleavage occurs essentially every time the target RNA or a mismatched RNA binds the ribozyme.  This occurs despite the weaker binding of the mismatched RNA because dissocn. becomes too slow (binding is too strong) to allow the ribozyme to choose between cleavage of the target RNA and a mismatched RNA.  In summary, more (base pairing) is not always better, because maximal discrimination requires equil. binding prior to cleavage.  The max. discrimination that can be obtained is expected to be greater with an A + U-rich recognition sequence than with a G + C-rich recognition sequence.  This is because the weaker A·U base pairs (relative to G·C base pairs) allow recognition to be spread over a larger no. of bases while preventing binding that is too strong.  Finally, creating an A-rich ribozyme rather than a U-rich ribozyme avoids the loss in discrimination expected with U-rich ribozymes from the formation of U·G wobble pairs in addn. to the targeted Watson-Crick U·A pair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEhIwC2M4ydbVg90H21EOLACvtfcHk0ljs3NyGyXY3_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlsFekt7s%253D&md5=df0058648a09c0094f03276cb3cd42de</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1073%2Fpnas.88.16.6921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.88.16.6921%26sid%3Dliteratum%253Aachs%26aulast%3DHerschlag%26aufirst%3DD.%26atitle%3DImplications%2520of%2520ribozyme%2520kinetics%2520for%2520targeting%2520the%2520cleavage%2520of%2520specific%2520RNA%2520molecules%2520in%2520vivo%253A%2520more%2520isn%25E2%2580%2599t%2520always%2520better%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1991%26volume%3D88%26spage%3D6921%26epage%3D6925%26doi%3D10.1073%2Fpnas.88.16.6921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagedorn, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindholm, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindow, M.</span></span> <span> </span><span class="NLM_article-title">A kinetic model explains why shorter and less affine enzyme-recruiting oligonucleotides can be more potent</span>. <i>Mol. Ther.--Nucleic Acids</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e149</span> <span class="refDoi"> DOI: 10.1038/mtna.2013.72</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fmtna.2013.72" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=24549300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1agtr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&author=L.+Pedersenauthor=P.+H.+Hagedornauthor=M.+W.+Lindholmauthor=M.+Lindow&title=A+kinetic+model+explains+why+shorter+and+less+affine+enzyme-recruiting+oligonucleotides+can+be+more+potent&doi=10.1038%2Fmtna.2013.72"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">A Kinetic Model Explains Why Shorter and Less Affine Enzyme-recruiting Oligonucleotides Can Be More Potent</span></div><div class="casAuthors">Pedersen, Lykke; Hagedorn, Peter H.; Lindholm, Marie Wickstrom; Lindow, Morten</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy--Nucleic Acids</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">Feb.</span>),
    <span class="NLM_cas:pages">e149</span>CODEN:
                <span class="NLM_cas:coden">MTAOC5</span>;
        ISSN:<span class="NLM_cas:issn">2162-2531</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Antisense oligonucleotides complementary to RNA targets promise generality and ease of drug design.  The first systemically administered antisense drug was recently approved for treatment and others are in clin. development.  Chem. modifications that increase the hybridization affinity of oligonucleotides are reasoned to confer higher potency, [ital: null], modified oligonucleotides can be dosed at lower concns. to achieve the same effect.  Surprisingly, shorter and less affine oligonucleotides sometimes display increased potency.  To explain this apparent contradiction, increased uptake or decreased propensity to form structures have been suggested as possible mechanisms.  Here, we provide an alternative explanation that invokes only the kinetics behind oligonucleotide-mediated cleavage of RNA targets.  A model based on the law of mass action predicts, and expts. support, the existence of an optimal binding affinity.  Exaggerated affinity, and not length [ital: null], is detrimental to potency.  This finding clarifies how to optimally apply high-affinity modifications in the discovery of potent antisense oligonucleotide drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8YJl-hKnoIrVg90H21EOLACvtfcHk0ljs3NyGyXY3_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1agtr7O&md5=511e8a57529fb19beb8626416a277fa9</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1038%2Fmtna.2013.72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmtna.2013.72%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DL.%26aulast%3DHagedorn%26aufirst%3DP.%2BH.%26aulast%3DLindholm%26aufirst%3DM.%2BW.%26aulast%3DLindow%26aufirst%3DM.%26atitle%3DA%2520kinetic%2520model%2520explains%2520why%2520shorter%2520and%2520less%2520affine%2520enzyme-recruiting%2520oligonucleotides%2520can%2520be%2520more%2520potent%26jtitle%3DMol.%2520Ther.--Nucleic%2520Acids%26date%3D2014%26volume%3D3%26doi%3D10.1038%2Fmtna.2013.72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mckee, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olafson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Precise small molecule cleavage of a r(CUG) repeat expansion in a myotonic dystrophy mouse model</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">7799</span>– <span class="NLM_lpage">7804</span>, <span class="refDoi"> DOI: 10.1073/pnas.1901484116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1901484116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30926669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVeks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=7799-7804&author=A.+J.+Angelbelloauthor=S.+G.+Rzuczekauthor=K.+K.+Mckeeauthor=J.+L.+Chenauthor=H.+Olafsonauthor=M.+D.+Cameronauthor=W.+N.+Mossauthor=E.+T.+Wangauthor=M.+D.+Disney&title=Precise+small+molecule+cleavage+of+a+r%28CUG%29+repeat+expansion+in+a+myotonic+dystrophy+mouse+model&doi=10.1073%2Fpnas.1901484116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Precise small-molecule cleavage of an r(CUG) repeat expansion in a myotonic dystrophy mouse model</span></div><div class="casAuthors">Angelbello, Alicia J.; Rzuczek, Suzanne G.; Mckee, Kendra K.; Chen, Jonathan L.; Olafson, Hailey; Cameron, Michael D.; Moss, Walter N.; Wang, Eric T.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7799-7804</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 1 (DM1) is an incurable neuromuscular disorder caused by an expanded CTG repeat that is transcribed into r(CUG)exp.  The RNA repeat expansion sequesters regulatory proteins such as Muscleblind-like protein 1 (MBNL1), which causes pre-mRNA splicing defects.  The disease-causing r(CUG)exp has been targeted by antisense oligonucleotides, CRISPR-based approaches, and RNA-targeting small mols.  Herein, we describe a designer small mol., Cugamycin, that recognizes the structure of r(CUG)exp and cleaves it in both DM1 patient-derived myotubes and a DM1 mouse model, leaving short repeats of r(CUG) untouched.  In contrast, oligonucleotides that recognize r(CUG) sequence rather than structure cleave both long and short r(CUG)-contg. transcripts.  Transcriptomic, histol., and phenotypic studies demonstrate that Cugamycin broadly and specifically relieves DM1-assocd. defects in vivo without detectable off-targets.  Thus, small mols. that bind and cleave RNA have utility as lead chem. probes and medicines and can selectively target disease-causing RNA structures to broadly improve defects in preclin. animal models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg0da_Q8Pd37Vg90H21EOLACvtfcHk0ljWIi6b_6EqyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVeks70%253D&md5=9dfa637039d1b532cbcbbaaccaed40b1</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1901484116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1901484116%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DMckee%26aufirst%3DK.%2BK.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DOlafson%26aufirst%3DH.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26aulast%3DWang%26aufirst%3DE.%2BT.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DPrecise%2520small%2520molecule%2520cleavage%2520of%2520a%2520r%2528CUG%2529%2520repeat%2520expansion%2520in%2520a%2520myotonic%2520dystrophy%2520mouse%2520model%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D7799%26epage%3D7804%26doi%3D10.1073%2Fpnas.1901484116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">A designed small molecule inhibitor of a non-coding RNA sensitizes HER2 negative cancer to Herceptin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">2960</span>– <span class="NLM_lpage">2974</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b10558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b10558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVGmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=2960-2974&author=M.+G.+Costalesauthor=D.+G.+Hochauthor=D.+Abeggauthor=J.+L.+Childs-Disneyauthor=S.+P.+Velagapudiauthor=A.+Adibekianauthor=M.+D.+Disney&title=A+designed+small+molecule+inhibitor+of+a+non-coding+RNA+sensitizes+HER2+negative+cancer+to+Herceptin&doi=10.1021%2Fjacs.8b10558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">A designed small molecule inhibitor of a non-coding RNA sensitizes HER2 negative cancers to herceptin</span></div><div class="casAuthors">Costales, Matthew G.; Hoch, Dominic G.; Abegg, Daniel; Childs-Disney, Jessica L.; Velagapudi, Sai Pradeep; Adibekian, Alexander; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2960-2974</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A small mol. (1) with overlapping affinity for two microRNA (miRNA) precursors was used to inform design of a dimeric compd. (2 (I)) selective for one of the miRNAs.  In particular, I selectively targets the microRNA(miR)-515 hairpin precursor to inhibit prodn. of miR-515 that represses sphingosine kinase 1 (SK1), a key enzyme in the biosynthesis of sphingosine 1-phosphate (S1P).  Application of I to breast cancer cells enhanced SK1 and S1P levels, triggering a migratory phenotype.  Knockout of SK1, forced overexpression of miR-515, and application of a small mol. SK1 inhibitor all ablated I's effect on phenotype, consistent with its designed mode of action.  Target profiling studies via Chem-CLIP showed that I bound selectively to the miR-515 hairpin precursor in cells.  Global neoprotein synthesis upon addn. of I to MCF-7 breast cancer cells demonstrated I's selectivity and upregulation of cancer-assocd. proteins regulated by S1P.  The most upregulated protein was human epidermal growth factor receptor 2 (ERBB2/HER2), which is regulated by the SK1/S1P pathway and is normally not expressed in MCF-7 cells.  Like triple neg. breast cancer (TNBC) cells, the lack of HER2 renders them insusceptible to Herceptin and its antibody-drug conjugate Kadcyla.  In addn. to proteomics, an RNA-seq study supports that I has limited off target effects and other studies support that I is more selective than an oligonucleotide.  We therefore hypothesized that I could sensitize MCF-7 cells to anti-HER2 therapies.  Indeed, application of I sensitized cells to Herceptin.  These results were confirmed in two other cell lines that express miR-515 and are HER2-, the hepatocellular carcinoma cell line HepG2 and the TNBC line MDA-MB-231.  Importantly, normal breast epithelial cells (MCF-10A) that do not express miR-515 are not affected by I.  These observations suggest a precision medicine approach to sensitize HER2- cancers to approved anticancer medicines.  This study has implications for broadening the therapeutic utility of known targeted cancer therapeutics by using a secondary targeted approach to render otherwise insensitive cells, sensitive to a targeted therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaWGyvVjeHDrVg90H21EOLACvtfcHk0ljWIi6b_6EqyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVGmtrY%253D&md5=1e2f57da2a12fe57d6f60fa44e3e79e2</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b10558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b10558%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DA%2520designed%2520small%2520molecule%2520inhibitor%2520of%2520a%2520non-coding%2520RNA%2520sensitizes%2520HER2%2520negative%2520cancer%2520to%2520Herceptin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D2960%26epage%3D2974%26doi%3D10.1021%2Fjacs.8b10558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Comparison of small molecules and oligonucleotides that target a toxic, non-coding RNA</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2605</span>– <span class="NLM_lpage">2609</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.04.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.bmcl.2016.04.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27117425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVCmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=2605-2609&author=M.+G.+Costalesauthor=S.+G.+Rzuczekauthor=M.+D.+Disney&title=Comparison+of+small+molecules+and+oligonucleotides+that+target+a+toxic%2C+non-coding+RNA&doi=10.1016%2Fj.bmcl.2016.04.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of small molecules and oligonucleotides that target a toxic, non-coding RNA</span></div><div class="casAuthors">Costales, Matthew G.; Rzuczek, Suzanne G.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2605-2609</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Potential RNA targets for chem. probes and therapeutic modalities are pervasive in the transcriptome.  Oligonucleotide-based therapeutics are commonly used to target RNA sequence.  Small mols. are emerging as a modality to target RNA structures selectively, but their development is still in its infancy.  The authors compare the activity of oligonucleotides and several classes of small mols. that target the non-coding r(CCUG) repeat expansion (r(CCUG)exp) that causes myotonic dystrophy type 2 (DM2), an incurable disease that is the second-most common cause of adult onset muscular dystrophy.  Small mol. types investigated include monomers, dimers, and multivalent compds. synthesized on-site by using RNA-templated click chem.  Oligonucleotides investigated include phosphorothioates that cleave their target and vivo-morpholinos that modulate target RNA activity via binding.  The authors show that compds. assembled on-site that recognize structure have the highest potencies amongst small mols. and are similar in potency to a vivo-morpholino modified oligonucleotide that targets sequence.  These studies are likely to impact the design of therapeutic modalities targeting other repeats expansions that cause fragile X syndrome and amyotrophic lateral sclerosis, for example.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3hWVZ6e6s27Vg90H21EOLACvtfcHk0ljWIi6b_6EqyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVCmtrY%253D&md5=540a6ffdd24b0205cfe7fdf702832174</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.04.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.04.025%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DComparison%2520of%2520small%2520molecules%2520and%2520oligonucleotides%2520that%2520target%2520a%2520toxic%252C%2520non-coding%2520RNA%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D2605%26epage%3D2609%26doi%3D10.1016%2Fj.bmcl.2016.04.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span> <span> </span><span class="NLM_article-title">Identifying and validating small molecules interacting with RNA (SMIRNAs)</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>623</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/bs.mie.2019.04.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fbs.mie.2019.04.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31239057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVSkt7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=623&publication_year=2019&pages=45-66&author=M.+D.+Disneyauthor=S.+P.+Velagapudiauthor=Y.+Liauthor=M.+G.+Costalesauthor=J.+L.+Childs-Disney&title=Identifying+and+validating+small+molecules+interacting+with+RNA+%28SMIRNAs%29&doi=10.1016%2Fbs.mie.2019.04.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and validating small molecules interacting with RNA (SMIRNAs)</span></div><div class="casAuthors">Disney, Matthew D.; Velagapudi, Sai Pradeep; Li, Yue; Costales, Matthew G.; Childs-Disney, Jessica L.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">623</span>
        (<span class="NLM_cas:issue">RNA Recognition</span>),
    <span class="NLM_cas:pages">45-66</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">High throughput sequencing has revolutionized our ability to identify aberrant RNA expression and mutations that cause or contribute to disease.  These data can be used directly to design oligonucleotide-based modalities using Watson-Crick pairing to target unstructured regions in an RNA.  A complementary, although more difficult, strategy to deactivate a malfunctioning RNA is to target highly structured regions with small mols.  Indeed, RNA structures are directly causative of disease.  Herein, we discuss emerging strategies to design high affinity, selective, bioactive ligands targeting RNA, or small mols. interacting with RNA (SMIRNAs), and target validation and profiling methods.  An exptl. foundation is required for a lead identification strateqy for RNA structures, constructed from a library-vs.-library screen that probes vast libraries of small mols. for binding RNA three dimensional folds.  Dubbed 2-dimensional combinatorial screening (2DCS), the resulting data can be mined against transcriptomes or the composite of RNAs :hat are produced in an organism to define folded RNA structures that can be targeted.  By applying SMIRNAs to cells and using target validation tools such as Chem. Crosslinking and Isolation by Pull-down (Chem-CLIP) and Small Mol. Nucleic Acid Profiling by Cleavage Applied to RNA (RiboSNAP), all targets engaged in cells can be defined, along with rules for mol. recognition to affect RNA biol.  This chapter will describe lessons learned in applying these approaches in vitro, in cells, and in pre-dinical animal models of disease, enabling SMIRNAs to capture opportunities in chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDS1IOjQJ36LVg90H21EOLACvtfcHk0lho_iTOHEyPeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVSkt7vJ&md5=e43c245d539353958b0595e20876d6e7</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fbs.mie.2019.04.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.mie.2019.04.027%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26atitle%3DIdentifying%2520and%2520validating%2520small%2520molecules%2520interacting%2520with%2520RNA%2520%2528SMIRNAs%2529%26jtitle%3DMethods%2520Enzymol.%26date%3D2019%26volume%3D623%26spage%3D45%26epage%3D66%26doi%3D10.1016%2Fbs.mie.2019.04.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vummidi, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Approved anti-cancer drugs target oncogenic non-coding RNAs</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1086</span>– <span class="NLM_lpage">1094</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.05.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.chembiol.2018.05.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30251629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Cis7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=1086-1094&author=S.+P.+Velagapudiauthor=M.+G.+Costalesauthor=B.+R.+Vummidiauthor=Y.+Nakaiauthor=A.+J.+Angelbelloauthor=T.+Tranauthor=H.+S.+Haniffauthor=Y.+Matsumotoauthor=Z.+F.+Wangauthor=A.+K.+Chatterjeeauthor=J.+L.+Childs-Disneyauthor=M.+D.+Disney&title=Approved+anti-cancer+drugs+target+oncogenic+non-coding+RNAs&doi=10.1016%2Fj.chembiol.2018.05.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Approved Anti-cancer Drugs Target Oncogenic Non-coding RNAs</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Costales, Matthew G.; Vummidi, Balayeshwanth R.; Nakai, Yoshio; Angelbello, Alicia J.; Tran, Tuan; Haniff, Hafeez S.; Matsumoto, Yasumasa; Wang, Zi Fu; Chatterjee, Arnab K.; Childs-Disney, Jessica L.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1086-1094.e7</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Potential RNA drug targets for small mols. are found throughout the human transcriptome, yet small mols. known to elicit a pharmacol. response by directly targeting RNA are limited to antibacterials.  Herein, we describe AbsorbArray, a small mol. microarray-based approach that allows for unmodified compds., including FDA-approved drugs, to be probed for binding to RNA motif libraries in a massively parallel format.  Several drug classes bind RNA including kinase and topoisomerase inhibitors.  The latter avidly bound the motif found in the Dicer site of oncogenic microRNA (miR)-21 and inhibited its processing both in vitro and in cells.  The most potent compd. de-repressed a downstream protein target and inhibited a miR-21-mediated invasive phenotype.  The compd.'s activity was ablated upon overexpression of pre-miR-21.  Target validation via chem. crosslinking and isolation by pull-down showed direct engagement of pre-miR-21 by the small mol. in cells, demonstrating that RNAs should indeed be considered druggable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopr-LYm-CTeLVg90H21EOLACvtfcHk0lho_iTOHEyPeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Cis7bM&md5=ea6d403eaef61fff07d168a750caa9e7</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.05.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.05.015%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DVummidi%26aufirst%3DB.%2BR.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DMatsumoto%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.%2BF.%26aulast%3DChatterjee%26aufirst%3DA.%2BK.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DApproved%2520anti-cancer%2520drugs%2520target%2520oncogenic%2520non-coding%2520RNAs%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D1086%26epage%3D1094%26doi%3D10.1016%2Fj.chembiol.2018.05.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallahi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Defining RNA-small molecule affinity landscapes enables design of a small molecule inhibitor of an oncogenic noncoding RNA</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">216</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.7b00009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.7b00009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1GqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=205-216&author=S.+P.+Velagapudiauthor=Y.+Luoauthor=T.+Tranauthor=H.+S.+Haniffauthor=Y.+Nakaiauthor=M.+Fallahiauthor=G.+J.+Martinezauthor=J.+L.+Childs-Disneyauthor=M.+D.+Disney&title=Defining+RNA-small+molecule+affinity+landscapes+enables+design+of+a+small+molecule+inhibitor+of+an+oncogenic+noncoding+RNA&doi=10.1021%2Facscentsci.7b00009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Defining RNA-Small Molecule Affinity Landscapes Enables Design of a Small Molecule Inhibitor of an Oncogenic Noncoding RNA</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Luo, Yiling; Tran, Tuan; Haniff, Hafeez S.; Nakai, Yoshio; Fallahi, Mohammad; Martinez, Gustavo J.; Childs-Disney, Jessica L.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">205-216</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RNA drug targets are pervasive in cells but methods to design small mols. that target them are sparse.  Herein, the authors report a general approach to score the affinity and selectivity of RNA motif-small mol. interactions identified via selection.  Named High Throughput Structure-Activity Relationships Through Sequencing (HiT-StARTS), HiT-StARTS is statistical in nature and compares input nucleic acid sequences to selected library members that bind a ligand via high throughput sequencing.  The approach allowed facile definition of the fitness landscape of hundreds of thousands of RNA motif-small mol. binding partners.  These results were mined against folded RNAs in the human transcriptome and identified an avid interaction between a small mol. and the Dicer nuclease-processing site in the oncogenic microRNA (miR)-18a hairpin precursor, which is a member of the miR-17-92 cluster.  Application of the small mol., Targapremir-18a, to prostate cancer cells inhibited prodn. of miR-18a from the cluster, derepressed serine/threonine protein kinase 4 protein (STK4), and triggered apoptosis.  Profiling the cellular targets of Targapremir-18a via Chem. Cross Linking and Isolation by Pull Down (Chem-CLIP), a covalent small mol.-RNA cellular profiling approach, and other studies showed specific binding of the compd. to the miR-18a precursor, revealing broadly applicable factors that govern small mol. drugging of noncoding RNAs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPmptGpjN_4rVg90H21EOLACvtfcHk0lho_iTOHEyPeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1GqsLs%253D&md5=fefd2fe6e9d637c4aa04fd1e25fb1446</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.7b00009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.7b00009%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DMartinez%26aufirst%3DG.%2BJ.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDefining%2520RNA-small%2520molecule%2520affinity%2520landscapes%2520enables%2520design%2520of%2520a%2520small%2520molecule%2520inhibitor%2520of%2520an%2520oncogenic%2520noncoding%2520RNA%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2017%26volume%3D3%26spage%3D205%26epage%3D216%26doi%3D10.1021%2Facscentsci.7b00009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallo, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Sequence-based design of bioactive small molecules that target precursor microRNAs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnchembio.1452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=24509821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFaksbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=291-297&author=S.+P.+Velagapudiauthor=S.+M.+Galloauthor=M.+D.+Disney&title=Sequence-based+design+of+bioactive+small+molecules+that+target+precursor+microRNAs&doi=10.1038%2Fnchembio.1452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence-based design of bioactive small molecules that target precursor microRNAs</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Gallo, Steven M.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">291-297</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oligonucleotides are designed to target RNA using base pairing rules, but they can be hampered by poor cellular delivery and nonspecific stimulation of the immune system.  Small mols. are preferred as lead drugs or probes but cannot be designed from sequence.  Herein, we describe an approach termed Inforna that designs lead small mols. for RNA from solely sequence.  Inforna was applied to all human microRNA hairpin precursors, and it identified bioactive small mols. that inhibit biogenesis by binding nuclease-processing sites (44% hit rate).  Among 27 lead interactions, the most avid interaction is between a benzimidazole (1) and precursor microRNA-96.  Compd. 1 selectively inhibits biogenesis of microRNA-96, upregulating a protein target (FOXO1) and inducing apoptosis in cancer cells.  Apoptosis is ablated when FOXO1 mRNA expression is knocked down by an siRNA, validating compd. selectivity.  Markedly, microRNA profiling shows that 1 only affects microRNA-96 biogenesis and is at least as selective as an oligonucleotide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy1YQeIYltbrVg90H21EOLACvtfcHk0liCSeNy9f3OZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFaksbo%253D&md5=d57f6a2ca7d6784017e2adff9ddb2f55</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1452%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DGallo%26aufirst%3DS.%2BM.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSequence-based%2520design%2520of%2520bioactive%2520small%2520molecules%2520that%2520target%2520precursor%2520microRNAs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D291%26epage%3D297%26doi%3D10.1038%2Fnchembio.1452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkelsas, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallahi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span> <span> </span><span class="NLM_article-title">Inforna 2.0: A platform for the sequence-based design of small molecules targeting structured RNAs</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1720</span>– <span class="NLM_lpage">1728</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmt1Cgsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1720-1728&author=M.+D.+Disneyauthor=A.+M.+Winkelsasauthor=S.+P.+Velagapudiauthor=M.+Southernauthor=M.+Fallahiauthor=J.+L.+Childs-Disney&title=Inforna+2.0%3A+A+platform+for+the+sequence-based+design+of+small+molecules+targeting+structured+RNAs&doi=10.1021%2Facschembio.6b00001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Inforna 2.0: A Platform for the Sequence-Based Design of Small Molecules Targeting Structured RNAs</span></div><div class="casAuthors">Disney, Matthew D.; Winkelsas, Audrey M.; Velagapudi, Sai Pradeep; Southern, Mark; Fallahi, Mohammad; Childs-Disney, Jessica L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1720-1728</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of small mols. that target RNA is challenging yet, if successful, could advance the development of chem. probes to study RNA function or precision therapeutics to treat RNA-mediated disease.  Previously, we described Inforna, an approach that can mine motifs (secondary structures) within target RNAs, which is deduced from the RNA sequence, and compare them to a database of known RNA motif-small mol. binding partners.  Output generated by Inforna includes the motif found in both the database and the desired RNA target, lead small mols. for that target, and other related meta-data.  Lead small mols. can then be tested for binding and affecting cellular (dys)function.  Herein, we describe Inforna 2.0, which incorporates all known RNA motif-small mol. binding partners reported in the scientific literature, a chem. similarity searching feature, and an improved user interface and is freely available via an online web server.  By incorporation of interactions identified by other labs., the database has been doubled, contg. 1936 RNA motif-small mol. interactions, including 244 unique small mols. and 1331 motifs.  Interestingly, chemotype anal. of the compds. that bind RNA in the database reveals features in small mol. chemotypes that are privileged for binding.  Further, this updated database expanded the no. of cellular RNAs to which lead compds. can be identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHmJfcxIk9yrVg90H21EOLACvtfcHk0liCSeNy9f3OZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmt1Cgsrw%253D&md5=6086bc48066565e519f7ee9e9cf2542a</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00001%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DWinkelsas%26aufirst%3DA.%2BM.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DSouthern%26aufirst%3DM.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26atitle%3DInforna%25202.0%253A%2520A%2520platform%2520for%2520the%2520sequence-based%2520design%2520of%2520small%2520molecules%2520targeting%2520structured%2520RNAs%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1720%26epage%3D1728%26doi%3D10.1021%2Facschembio.6b00001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phinney, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of microRNA-210 reprograms an oncogenic hypoxic circuit</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">3446</span>– <span class="NLM_lpage">3455</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b11273</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b11273" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlCjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=3446-3455&author=M.+G.+Costalesauthor=C.+L.+Hagaauthor=S.+P.+Velagapudiauthor=J.+L.+Childs-Disneyauthor=D.+G.+Phinneyauthor=M.+D.+Disney&title=Small+molecule+inhibition+of+microRNA-210+reprograms+an+oncogenic+hypoxic+circuit&doi=10.1021%2Fjacs.6b11273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibition of microRNA-210 Reprograms an Oncogenic Hypoxic Circuit</span></div><div class="casAuthors">Costales, Matthew G.; Haga, Christopher L.; Velagapudi, Sai Pradeep; Childs-Disney, Jessica L.; Phinney, Donald G.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3446-3455</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A hypoxic state is crit. to the metastatic and invasive characteristics of cancer.  Numerous pathways play crit. roles in cancer maintenance, many of which include noncoding RNAs such as microRNA (miR)-210 that regulates hypoxia inducible factors (HIFs).  Herein, we describe the identification of a small mol. named Targapremir-210 that binds to the Dicer site of the miR-210 hairpin precursor.  This interaction inhibits prodn. of the mature miRNA, derepresses glycerol-3-phosphate dehydrogenase 1-like enzyme (GPD1L), a hypoxia-assocd. protein neg. regulated by miR-210, decreases HIF-1α, and triggers apoptosis of triple neg. breast cancer cells only under hypoxic conditions.  Further, Targapremir-210 inhibits tumorigenesis in a mouse xenograft model of hypoxic triple neg. breast cancer.  Many factors govern mol. recognition of biol. targets by small mols.  For protein, chemoproteomics and activity-based protein profiling are invaluable tools to study small mol. target engagement and selectivity in cells.  Such approaches are lacking for RNA, leaving a void in the understanding of its druggability.  We applied Chem. Crosslinking and Isolation by Pull Down (Chem-CLIP) to study the cellular selectivity and the on- and off-targets of Targapremir-210.  Targapremir-210 selectively recognizes the miR-210 precursor and can differentially recognize RNAs in cells that have the same target motif but have different expression levels, revealing this important feature for selectively drugging RNAs for the first time.  These studies show that small mols. can be rapidly designed to selectively target RNAs and affect cellular responses to environmental conditions, resulting in favorable benefits against cancer.  Further, they help define rules for identifying druggable targets in the transcriptome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAY_YQQyzKvLVg90H21EOLACvtfcHk0liCSeNy9f3OZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlCjt7w%253D&md5=c6725837e6f1713facb507b518998a8f</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b11273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b11273%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DHaga%26aufirst%3DC.%2BL.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DPhinney%26aufirst%3DD.%2BG.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520microRNA-210%2520reprograms%2520an%2520oncogenic%2520hypoxic%2520circuit%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D3446%26epage%3D3455%26doi%3D10.1021%2Fjacs.6b11273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennewith, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welford, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Story, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, Q.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giaccia, A. J.</span></span> <span> </span><span class="NLM_article-title">Hypoxia-inducible mir-210 regulates normoxic gene expression involved in tumor initiation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">856</span>– <span class="NLM_lpage">867</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2009.09.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.molcel.2009.09.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19782034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVChsrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2009&pages=856-867&author=X.+Huangauthor=L.+Dingauthor=K.+L.+Bennewithauthor=R.+T.+Tongauthor=S.+M.+Welfordauthor=K.+K.+Angauthor=M.+Storyauthor=Q.-T.+Leauthor=A.+J.+Giaccia&title=Hypoxia-inducible+mir-210+regulates+normoxic+gene+expression+involved+in+tumor+initiation&doi=10.1016%2Fj.molcel.2009.09.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Hypoxia-inducible mir-210 regulates normoxic gene expression involved in tumor initiation</span></div><div class="casAuthors">Huang, Xin; Ding, Lianghao; Bennewith, Kevin L.; Tong, Ricky T.; Welford, Scott M.; Ang, K. Kian; Story, Michael; Le, Quynh-Thu; Giaccia, Amato J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">856-867</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Previous studies have suggested that the HIF transcription factors can both activate and inhibit gene expression.  Here we show that HIF1 regulates the expression of mir-210 in a variety of tumor types through a hypoxia-responsive element.  Expression anal. in primary head and neck tumor samples indicates that mir-210 may serve as an in vivo marker for tumor hypoxia.  By Argonaute protein immunopptn., we identified 50 potential mir-210 targets and validated randomly selected ones.  The majority of these 50 genes are not classical hypoxia-inducible genes, suggesting mir-210 represses genes expressed under normoxia that are no longer necessary to adapt and survive in a hypoxic environment.  When human head and neck or pancreatic tumor cells ectopically expressing mir-210 were implanted into immunodeficient mice, mir-210 repressed initiation of tumor growth.  Taken together, these data implicate an important role for mir-210 in regulating the hypoxic response of tumor cells and tumor growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-QNBn2_n7KbVg90H21EOLACvtfcHk0liDHBRN-N5t7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVChsrzI&md5=98acfaab2f9f3a97e16c91801197209c</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2009.09.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2009.09.006%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DL.%26aulast%3DBennewith%26aufirst%3DK.%2BL.%26aulast%3DTong%26aufirst%3DR.%2BT.%26aulast%3DWelford%26aufirst%3DS.%2BM.%26aulast%3DAng%26aufirst%3DK.%2BK.%26aulast%3DStory%26aufirst%3DM.%26aulast%3DLe%26aufirst%3DQ.-T.%26aulast%3DGiaccia%26aufirst%3DA.%2BJ.%26atitle%3DHypoxia-inducible%2520mir-210%2520regulates%2520normoxic%2520gene%2520expression%2520involved%2520in%2520tumor%2520initiation%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D35%26spage%3D856%26epage%3D867%26doi%3D10.1016%2Fj.molcel.2009.09.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baosheng, L.</span></span> <span> </span><span class="NLM_article-title">Multiple functions of hypoxia-regulated miR-210 in cancer</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">50</span>, <span class="refDoi"> DOI: 10.1186/1756-9966-33-50</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1186%2F1756-9966-33-50" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=24909053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFShu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2014&pages=50&author=Q.+Qinauthor=W.+Furongauthor=L.+Baosheng&title=Multiple+functions+of+hypoxia-regulated+miR-210+in+cancer&doi=10.1186%2F1756-9966-33-50"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple functions of hypoxia-regulated miR-210 in cancer</span></div><div class="casAuthors">Qin, Qin; Wei, Furong; Li, Baosheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental & Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">50/1-50/10</span>CODEN:
                <span class="NLM_cas:coden">JECRDN</span>;
        ISSN:<span class="NLM_cas:issn">1756-9966</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  MicroRNAs (miRNAs) are small noncoding RNAs that regulate gene expression post-transcriptionally.  MiRNAs can be induced by a variety of stresses such as hypoxia, and are involved in diverse biol. processes including differentiation, cell proliferation, cell death, and tumorigenesis.  Hypoxia, a common feature of tumor microenvironment, can induce a no. of miRNAs expression. miRNA-210 (miR-210) is one of the hypoxia-regulated-miRNAs, which has been investigated extensively in cancer.  However, paradoxically opposing results were documented regarding whether it is an oncogene or a tumor suppressor, and whether it is a pos. or neg. prognostic biomarker.  In the present review, we focus on the following investigations of miR-210: (1) its functions of as an oncogene, (2) its functions as a tumor suppressor, (3) its functions in mitochondrial metab., and finally, the diagnostic and prognostic value of miR-210 in cancer researches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGU4V5lXKPCbVg90H21EOLACvtfcHk0liDHBRN-N5t7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFShu78%253D&md5=07067621ddb3082065991fad0ee7f157</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1186%2F1756-9966-33-50&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-9966-33-50%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DQ.%26aulast%3DFurong%26aufirst%3DW.%26aulast%3DBaosheng%26aufirst%3DL.%26atitle%3DMultiple%2520functions%2520of%2520hypoxia-regulated%2520miR-210%2520in%2520cancer%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2014%26volume%3D33%26spage%3D50%26doi%3D10.1186%2F1756-9966-33-50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puigserver, P.</span></span> <span> </span><span class="NLM_article-title">A hypoxia-induced positive feedback loop promotes hypoxia-inducible factor 1alpha stability through miR-210 suppression of glycerol-3-phosphate dehydrogenase 1-like</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">2696</span>– <span class="NLM_lpage">2706</span>, <span class="refDoi"> DOI: 10.1128/MCB.01242-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1128%2FMCB.01242-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=21555452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFKjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2011&pages=2696-2706&author=T.+J.+Kellyauthor=A.+L.+Souzaauthor=C.+B.+Clishauthor=P.+Puigserver&title=A+hypoxia-induced+positive+feedback+loop+promotes+hypoxia-inducible+factor+1alpha+stability+through+miR-210+suppression+of+glycerol-3-phosphate+dehydrogenase+1-like&doi=10.1128%2FMCB.01242-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">A hypoxia-induced positive feedback loop promotes hypoxia-inducible factor 1α stability through miR-210 suppression of glycerol-3-phosphate dehydrogenase 1-like</span></div><div class="casAuthors">Kelly, Timothy J.; Souza, Amanda L.; Clish, Clary B.; Puigserver, Pere</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2696-2706</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Oxygen-dependent regulation of the transcription factor HIF-1α relies on a family of prolyl hydroxylases (PHDs) that hydroxylate hypoxia-inducible factor 1α (HIF-1α) protein at two prolines during normal oxygen conditions, resulting in degrdn. by the proteasome.  During low-oxygen conditions, these prolines are no longer hydroxylated and HIF-1α degrdn. is blocked.  Hypoxia-induced miRNA-210 (miR-210) is a direct transcriptional target of HIF-1α, but its complete role and targets during hypoxia are not well understood.  Here, we identify the enzyme glycerol-3-phosphate dehydrogenase 1-like (GPD1L) as a novel regulator of HIF-1α stability and a direct target of miR-210.  Expression of miR-210 results in stabilization of HIF-1α due to decreased levels of GPD1L resulting in an increase in HIF-1α target genes.  Altering GPD1L levels by overexpression or knockdown results in a decrease or increase in HIF-1α stability, resp.  GPD1L-mediated decreases in HIF-1α stability can be reversed by pharmacol. inhibition of the proteasome or PHD activity.  When rescued from degrdn. by proteasome inhibition, elevated amts. of GPD1L cause hyperhydroxylation of HIF-1α, suggesting increases in PHD activity.  Importantly, expression of GPD1L attenuates the hypoxic response, preventing complete HIF-1α induction.  We propose a model in which hypoxia-induced miR-210 represses GPD1L, contributing to suppression of PHD activity, and increases of HIF-1α protein levels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKHol8fw_kW7Vg90H21EOLACvtfcHk0liDHBRN-N5t7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFKjs7c%253D&md5=af0e9747e80cb309199e0c5181d0a97c</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1128%2FMCB.01242-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01242-10%26sid%3Dliteratum%253Aachs%26aulast%3DKelly%26aufirst%3DT.%2BJ.%26aulast%3DSouza%26aufirst%3DA.%2BL.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DPuigserver%26aufirst%3DP.%26atitle%3DA%2520hypoxia-induced%2520positive%2520feedback%2520loop%2520promotes%2520hypoxia-inducible%2520factor%25201alpha%2520stability%2520through%2520miR-210%2520suppression%2520of%2520glycerol-3-phosphate%2520dehydrogenase%25201-like%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2011%26volume%3D31%26spage%3D2696%26epage%3D2706%26doi%3D10.1128%2FMCB.01242-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abelson, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, H.</span></span> <span> </span><span class="NLM_article-title">The relative amounts of the cytoplasmic RNA species in normal, transformed and senescent cultured cell lines</span>. <i>J. Cell. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">470</span>, <span class="refDoi"> DOI: 10.1002/jcp.1040900310</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1002%2Fjcp.1040900310" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=192738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaE2sXhvVOgsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1977&pages=465-470&author=L.+F.+Johnsonauthor=H.+T.+Abelsonauthor=S.+Penmanauthor=H.+Green&title=The+relative+amounts+of+the+cytoplasmic+RNA+species+in+normal%2C+transformed+and+senescent+cultured+cell+lines&doi=10.1002%2Fjcp.1040900310"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">The relative amounts of the cytoplasmic RNA species in normal, transformed and senescent cultured cell lines</span></div><div class="casAuthors">Johnson, Lee F.; Abelson, Herbert T.; Penman, Sheldon; Green, Howard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Physiology</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">465-70</span>CODEN:
                <span class="NLM_cas:coden">JCLLAX</span>;
        ISSN:<span class="NLM_cas:issn">0021-9541</span>.
    </div><div class="casAbstract">The relative quantities of 18S and 28S rRNA, 4S, RNA, and poly(A) + mRNA were examd. in mouse fibroblast lines 3T3 and 3T6 in the resting (contact inhibited) and growing (sparse) states, 3T3 clones transformed with SV40 (SV3T3) and with both SV40 and polyoma (SV-Py 3T3) viruses hamster lung fibroblasts (V79), human cervical carcinoma cells (HeLa), and human diploid fibroblasts at early and late passage.  The ratio of mRNA-to-RNA varied 1.1-2.7% in the different cell lines, the more rapidly growing cell lines usually giving a higher ratio.  In cells experiencing growth limitation either by contact inhibition or due to senescence, the ratio of mRNA-to-rRNA was ∼30% lower than in the corresponding cells in the growing state.  In most cell lines the ratio of 4S RNA-to-18S rRNA was 0.8-1.2, but in senescent fibroblasts, this ratio increased to >1.7.  Senescent fibroblasts also contained much more total RNA per unit of DNA than the same cells at early passage or than 3T6 or 3T3 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonjmUBnEcv5LVg90H21EOLACvtfcHk0lh4pkrA4nyPTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXhvVOgsL0%253D&md5=9f81b7e26960b0aac7af8705355bc40c</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1002%2Fjcp.1040900310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcp.1040900310%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DL.%2BF.%26aulast%3DAbelson%26aufirst%3DH.%2BT.%26aulast%3DPenman%26aufirst%3DS.%26aulast%3DGreen%26aufirst%3DH.%26atitle%3DThe%2520relative%2520amounts%2520of%2520the%2520cytoplasmic%2520RNA%2520species%2520in%2520normal%252C%2520transformed%2520and%2520senescent%2520cultured%2520cell%2520lines%26jtitle%3DJ.%2520Cell.%2520Physiol.%26date%3D1977%26volume%3D90%26spage%3D465%26epage%3D470%26doi%3D10.1002%2Fjcp.1040900310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Znosko, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallahi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallo, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Analysis of secondary structural elements in human microRNA hairpin precursors</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">112</span>, <span class="refDoi"> DOI: 10.1186/s12859-016-0960-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1186%2Fs12859-016-0960-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=26928172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisFWntrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=112&author=B.+Liuauthor=J.+L.+Childs-Disneyauthor=B.+M.+Znoskoauthor=D.+Wangauthor=M.+Fallahiauthor=S.+M.+Galloauthor=M.+D.+Disney&title=Analysis+of+secondary+structural+elements+in+human+microRNA+hairpin+precursors&doi=10.1186%2Fs12859-016-0960-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of secondary structural elements in human microRNA hairpin precursors</span></div><div class="casAuthors">Liu, Biao; Childs-Disney, Jessica L.; Znosko, Brent M.; Wang, Dan; Fallahi, Mohammad; Gallo, Steven M.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112/1-112/11</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: MicroRNAs (miRNAs) regulate gene expression by targeting complementary mRNAs for destruction or translational repression.  Aberrant expression of miRNAs has been assocd. with various diseases including cancer, thus making them interesting therapeutic targets.  The composite of secondary structural elements that comprise miRNAs could aid the design of small mols. that modulate their function.  Results: We analyzed the secondary structural elements, or motifs, present in all human miRNA hairpin precursors and compared them to highly expressed human RNAs with known structures and other RNAs from various organisms.  Amongst human miRNAs, there are 3808 are unique motifs, many residing in processing sites.  Further, we identified motifs in miRNAs that are not present in other highly expressed human RNAs, desirable targets for small mols.  MiRNA motifs were incorporated into a searchable database that is freely available.  We also analyzed the most frequently occurring bulges and internal loops for each RNA class and found that the smallest loops possible prevail.  However, the distribution of loops and the preferred closing base pairs were unique to each class.  Conclusions: Collectively, we have completed a broad survey of motifs found in human miRNA precursors, highly expressed human RNAs, and RNAs from other organisms.  Interestingly, unique motifs were identified in human miRNA processing sites, binding to which could inhibit miRNA maturation and hence function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry3o_xOUoujrVg90H21EOLACvtfcHk0lh4pkrA4nyPTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisFWntrs%253D&md5=9855e1cb5c01973e01f40a8d3a59f310</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1186%2Fs12859-016-0960-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-016-0960-6%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DZnosko%26aufirst%3DB.%2BM.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DGallo%26aufirst%3DS.%2BM.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DAnalysis%2520of%2520secondary%2520structural%2520elements%2520in%2520human%2520microRNA%2520hairpin%2520precursors%26jtitle%3DBMC%2520Bioinf.%26date%3D2016%26volume%3D17%26spage%3D112%26doi%3D10.1186%2Fs12859-016-0960-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seedhouse, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Defining the RNA internal loops preferred by benzimidazole derivatives via 2D combinatorial screening and computational analysis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">10111</span>– <span class="NLM_lpage">10118</span>, <span class="refDoi"> DOI: 10.1021/ja200212b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja200212b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnt1agtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=10111-10118&author=S.+P.+Velagapudiauthor=S.+J.+Seedhouseauthor=J.+Frenchauthor=M.+D.+Disney&title=Defining+the+RNA+internal+loops+preferred+by+benzimidazole+derivatives+via+2D+combinatorial+screening+and+computational+analysis&doi=10.1021%2Fja200212b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Defining the RNA Internal Loops Preferred by Benzimidazole Derivatives via 2D Combinatorial Screening and Computational Analysis</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Seedhouse, Steven J.; French, Jonathan; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10111-10118</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RNA is an important therapeutic target; however, RNA targets are generally under-exploited due to a lack of understanding of the small mols. that bind RNA and the RNA motifs that bind small mols.  Herein, we describe the identification of the RNA internal loops derived from a 4096 member 3×3 nucleotide loop library that are the most specific and highest affinity binders to a series of four designer, drug-like benzimidazoles.  These studies establish a potentially general protocol to define the highest affinity and most specific RNA motif targets for heterocyclic small mols.  Such information could be used to target functionally important RNAs in genomic sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1erYl2X0GarVg90H21EOLACvtfcHk0lh4pkrA4nyPTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnt1agtbk%253D&md5=5b8ea0838113f73f2ad3f1f54f21749e</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1021%2Fja200212b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja200212b%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DSeedhouse%26aufirst%3DS.%2BJ.%26aulast%3DFrench%26aufirst%3DJ.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDefining%2520the%2520RNA%2520internal%2520loops%2520preferred%2520by%2520benzimidazole%2520derivatives%2520via%25202D%2520combinatorial%2520screening%2520and%2520computational%2520analysis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D10111%26epage%3D10118%26doi%3D10.1021%2Fja200212b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pushechnikov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span> <span> </span><span class="NLM_article-title">The role of flexibility in the rational design of modularly assembled ligands targeting the RNAs that cause the myotonic dystrophies</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">382</span>, <span class="refDoi"> DOI: 10.1002/cbic.200900716</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1002%2Fcbic.200900716" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=20058255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGhu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=375-382&author=M.+D.+Disneyauthor=M.+M.+Leeauthor=A.+Pushechnikovauthor=J.+L.+Childs-Disney&title=The+role+of+flexibility+in+the+rational+design+of+modularly+assembled+ligands+targeting+the+RNAs+that+cause+the+myotonic+dystrophies&doi=10.1002%2Fcbic.200900716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Flexibility in the Rational Design of Modularly Assembled Ligands Targeting the RNAs that Cause the Myotonic Dystrophies</span></div><div class="casAuthors">Disney, Matthew D.; Lee, Melissa M.; Pushechnikov, Alexei; Childs-Disney, Jessica L.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">375-382</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Modularly assembled ligands were designed to target the RNAs that cause two currently untreatable neuromuscular disorders, myotonic dystrophy types 1 (DM1) and 2 (DM2).  DM1 is caused by an expanded repeating sequence of CUG, and DM2 is caused by expanded CCUG repeats.  Both are present in noncoding regions and fold into hairpins with either repeating 1×1 nucleotide UU (DM1) or 2×2 nucleotide 5'-CU/3'-UC (DM2) internal loops sepd. by two GC pairs.  The repeats are toxic because they sequester the RNA splicing regulator muscleblind-like 1 protein (MBNL1).  Rational design of ligands targeting these RNAs was enabled by a database of RNA motif-ligand partners compiled by using two-dimensional combinatorial screening (2DCS).  One 2DCS study found that the 6''-azido-kanamycin A module binds internal loops similar to those found in DM1 and DM2.  In order to further enhance affinity and specificity, the ligand was assembled on a peptoid backbone to precisely control valency and the distance between ligand modules.  Designed compds. are more potent and specific binders to the toxic RNAs than MBNL1 and inhibit the formation of the RNA-protein complexes with nanomolar IC50 values.  This study shows that three important factors govern potent inhibition: (1) the surface area sequestered by the assembled ligands; (2) the spacing between ligand modules since a longer distance is required to target DM2 RNAs than DM1 RNAs; and (3) flexibility in the modular assembly scaffold used to display the RNA-binding module.  These results have impacts on the general design of assembled ligands targeting RNAs present in genomic sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-f73j_3RGI7Vg90H21EOLACvtfcHk0lh4pkrA4nyPTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGhu7g%253D&md5=6c241e8fdb9de0cad47ba0ed5153e061</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200900716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200900716%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DLee%26aufirst%3DM.%2BM.%26aulast%3DPushechnikov%26aufirst%3DA.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26atitle%3DThe%2520role%2520of%2520flexibility%2520in%2520the%2520rational%2520design%2520of%2520modularly%2520assembled%2520ligands%2520targeting%2520the%2520RNAs%2520that%2520cause%2520the%2520myotonic%2520dystrophies%26jtitle%3DChemBioChem%26date%3D2010%26volume%3D11%26spage%3D375%26epage%3D382%26doi%3D10.1002%2Fcbic.200900716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafitte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duckett, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phinney, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Design of a small molecule against an oncogenic noncoding RNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">5898</span>– <span class="NLM_lpage">5903</span>, <span class="refDoi"> DOI: 10.1073/pnas.1523975113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1523975113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27170187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1ejurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=5898-5903&author=S.+P.+Velagapudiauthor=M.+D.+Cameronauthor=C.+L.+Hagaauthor=L.+H.+Rosenbergauthor=M.+Lafitteauthor=D.+R.+Duckettauthor=D.+G.+Phinneyauthor=M.+D.+Disney&title=Design+of+a+small+molecule+against+an+oncogenic+noncoding+RNA&doi=10.1073%2Fpnas.1523975113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a small molecule against an oncogenic noncoding RNA</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Cameron, Michael D.; Haga, Christopher L.; Rosenberg, Laura H.; Lafitte, Marie; Duckett, Derek R.; Phinney, Donald G.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5898-5903</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The design of precision, preclin. therapeutics from sequence is difficult, but advances in this area, particularly those focused on rational design, could quickly transform the sequence of disease-causing gene products into lead modalities.  Herein, we describe the use of Inforna, a computational approach that enables the rational design of small mols. targeting RNA to quickly provide a potent modulator of oncogenic microRNA-96 (miR-96).  We mined the secondary structure of primary microRNA-96 (pri-miR-96) hairpin precursor against a database of RNA motif-small mol. interactions, which identified modules that bound RNA motifs nearby and in the Drosha processing site.  Precise linking of these modules together provided Targaprimir-96 (3), which selectively modulates miR-96 prodn. in cancer cells and triggers apoptosis.  Importantly, the compd. is ineffective on healthy breast cells, and exogenous overexpression of pri-miR-96 reduced compd. potency in breast cancer cells.  Chem. Crosslinking and Isolation by Pull-Down (Chem-CLIP), a small-mol. RNA target validation approach, shows that 3 directly engages pri-miR-96 in breast cancer cells.  In vivo, 3 has a favorable pharmacokinetic profile and decreases tumor burden in a mouse model of triple-neg. breast cancer.  Thus, rational design can quickly produce precision, in vivo bioactive lead small mols. against hard-to-treat cancers by targeting oncogenic noncoding RNAs, advancing a disease-to-gene-to-drug paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlNk7Nj8xTxrVg90H21EOLACvtfcHk0lj8WH7084hyRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1ejurw%253D&md5=9254bf5bef8ae21aa59ae0bca5f47f37</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1523975113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1523975113%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DHaga%26aufirst%3DC.%2BL.%26aulast%3DRosenberg%26aufirst%3DL.%2BH.%26aulast%3DLafitte%26aufirst%3DM.%26aulast%3DDuckett%26aufirst%3DD.%2BR.%26aulast%3DPhinney%26aufirst%3DD.%2BG.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDesign%2520of%2520a%2520small%2520molecule%2520against%2520an%2520oncogenic%2520noncoding%2520RNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D5898%26epage%3D5903%26doi%3D10.1073%2Fpnas.1523975113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colgan, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Precise small-molecule recognition of a toxic CUG RNA repeat expansion</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">193</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnchembio.2251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27941760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVSrurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=188-193&author=S.+G.+Rzuczekauthor=L.+A.+Colganauthor=Y.+Nakaiauthor=M.+D.+Cameronauthor=D.+Furlingauthor=R.+Yasudaauthor=M.+D.+Disney&title=Precise+small-molecule+recognition+of+a+toxic+CUG+RNA+repeat+expansion&doi=10.1038%2Fnchembio.2251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Precise small-molecule recognition of a toxic CUG RNA repeat expansion</span></div><div class="casAuthors">Rzuczek, Suzanne G.; Colgan, Lesley A.; Nakai, Yoshio; Cameron, Michael D.; Furling, Denis; Yasuda, Ryohei; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">188-193</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Excluding the ribosome and riboswitches, developing small mols. that selectively target RNA is a longstanding problem in chem. biol.  A typical cellular RNA is difficult to target because it has little tertiary, but abundant secondary structure.  We designed allele-selective compds. that target such an RNA, the toxic noncoding repeat expansion (r(CUG)exp) that causes myotonic dystrophy type 1 (DM1).  We developed several strategies to generate allele-selective small mols., including non-covalent binding, covalent binding, cleavage and on-site probe synthesis.  Covalent binding and cleavage enabled target profiling in cells derived from individuals with DM1, showing precise recognition of r(CUG)exp.  In the on-site probe synthesis approach, small mols. bound adjacent sites in r(CUG)exp and reacted to afford picomolar inhibitors via a proximity-based click reaction only in DM1-affected cells.  We expanded this approach to image r(CUG)exp in its natural context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC5kTu0Wkj9bVg90H21EOLACvtfcHk0lj8WH7084hyRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVSrurbM&md5=bb30d79fe6e8d513a4956d13ee20f69c</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2251%26sid%3Dliteratum%253Aachs%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DColgan%26aufirst%3DL.%2BA.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DFurling%26aufirst%3DD.%26aulast%3DYasuda%26aufirst%3DR.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DPrecise%2520small-molecule%2520recognition%2520of%2520a%2520toxic%2520CUG%2520RNA%2520repeat%2520expansion%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D188%26epage%3D193%26doi%3D10.1038%2Fnchembio.2251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crynen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boström, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lekah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camerson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yildirm, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemurell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Design of a small molecule that stimulates VEGFA informed from an expanded encyclopedia of RNA fold-small molecule interactions</span>. <i>ChemRxiv</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.26434/chemrxiv.7967291.v1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.26434%2Fchemrxiv.7967291.v1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=H.+S.+Haniffauthor=L.+Knerrauthor=X.+Liuauthor=G.+Crynenauthor=J.+Bostr%C3%B6mauthor=D.+Abeggauthor=A.+Adibekianauthor=E.+Lekahauthor=K.-W.+Wangauthor=M.+D.+Camersonauthor=I.+Yildirmauthor=M.+Lemurellauthor=M.+D.+Disney&title=Design+of+a+small+molecule+that+stimulates+VEGFA+informed+from+an+expanded+encyclopedia+of+RNA+fold-small+molecule+interactions&doi=10.26434%2Fchemrxiv.7967291.v1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.26434%2Fchemrxiv.7967291.v1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.26434%252Fchemrxiv.7967291.v1%26sid%3Dliteratum%253Aachs%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DKnerr%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DCrynen%26aufirst%3DG.%26aulast%3DBostr%25C3%25B6m%26aufirst%3DJ.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DLekah%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DK.-W.%26aulast%3DCamerson%26aufirst%3DM.%2BD.%26aulast%3DYildirm%26aufirst%3DI.%26aulast%3DLemurell%26aufirst%3DM.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDesign%2520of%2520a%2520small%2520molecule%2520that%2520stimulates%2520VEGFA%2520informed%2520from%2520an%2520expanded%2520encyclopedia%2520of%2520RNA%2520fold-small%2520molecule%2520interactions%26jtitle%3DChemRxiv%26date%3D2020%26doi%3D10.26434%2Fchemrxiv.7967291.v1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
der Lee, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buljan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatheritt, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daughdrill, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunker, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuxreiter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gsponer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kriwacki, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldfield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappu, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tompa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uversky, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babu, M. M.</span></span> <span> </span><span class="NLM_article-title">Classification of intrinsically disordered regions and proteins</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">6589</span>– <span class="NLM_lpage">6631</span>, <span class="refDoi"> DOI: 10.1021/cr400525m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr400525m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFChur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=6589-6631&author=R.+van%0Ader+Leeauthor=M.+Buljanauthor=B.+Langauthor=R.+J.+Weatherittauthor=G.+W.+Daughdrillauthor=A.+K.+Dunkerauthor=M.+Fuxreiterauthor=J.+Goughauthor=J.+Gsponerauthor=D.+T.+Jonesauthor=P.+M.+Kimauthor=R.+W.+Kriwackiauthor=C.+J.+Oldfieldauthor=R.+V.+Pappuauthor=P.+Tompaauthor=V.+N.+Uverskyauthor=P.+E.+Wrightauthor=M.+M.+Babu&title=Classification+of+intrinsically+disordered+regions+and+proteins&doi=10.1021%2Fcr400525m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of intrinsically disordered regions and proteins</span></div><div class="casAuthors">van der Lee, Robin; Buljan, Marija; Lang, Benjamin; Weatheritt, Robert J.; Daughdrill, Gary W.; Dunker, A. Keith; Fuxreiter, Monika; Gough, Julian; Gsponer, Joerg; Jones, David T.; Kim, Philip M.; Kriwacki, Richard W.; Oldfield, Christopher J.; Pappu, Rohit V.; Tompa, Peter; Uversky, Vladimir N.; Wright, Peter E.; Babu, M. Madan</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6589-6631</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  While many proteins need to adopt a well-defined structure to carry out their function, a large fraction of the proteome of any organism consists of polypeptide segments that are not likely to form a defined 3-dimensional structure, but are nevertheless functional.  These protein segments are referred to as intrinsically disordered regions (IDRs).  Proteins without IDRs are called structured proteins, and proteins with entirely disordered sequences that do not adopt any tertiary structure are referred to as intrinsically disordered proteins (IDPs).  IDRs and IDPs are prevalent in eukaryotic genomes, with 44% of human protein-coding genes contg. disordered segments of >30 residues in length.  Here, the authors discuss classification approaches based on function, functional elements, sequence, protein interactions, evolution, regulation, and biophys. properties.  The authors also discuss resources that are currently available for gaining insight into IDR function, suggest areas where increased efforts are likely to advance the understanding of the functions of protein disorder, and speculate how combinations of multiple existing classification schemes could achieve high quality function prediction for IDRs, which should ultimately lead to improved function coverage and a deeper understanding of protein function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZKPkqA5jhibVg90H21EOLACvtfcHk0lj3QbhvrbSUWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFChur0%253D&md5=436cf71d9b77f6f3b176233a6abd9628</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1021%2Fcr400525m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr400525m%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BLee%26aufirst%3DR.%26aulast%3DBuljan%26aufirst%3DM.%26aulast%3DLang%26aufirst%3DB.%26aulast%3DWeatheritt%26aufirst%3DR.%2BJ.%26aulast%3DDaughdrill%26aufirst%3DG.%2BW.%26aulast%3DDunker%26aufirst%3DA.%2BK.%26aulast%3DFuxreiter%26aufirst%3DM.%26aulast%3DGough%26aufirst%3DJ.%26aulast%3DGsponer%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DD.%2BT.%26aulast%3DKim%26aufirst%3DP.%2BM.%26aulast%3DKriwacki%26aufirst%3DR.%2BW.%26aulast%3DOldfield%26aufirst%3DC.%2BJ.%26aulast%3DPappu%26aufirst%3DR.%2BV.%26aulast%3DTompa%26aufirst%3DP.%26aulast%3DUversky%26aufirst%3DV.%2BN.%26aulast%3DWright%26aufirst%3DP.%2BE.%26aulast%3DBabu%26aufirst%3DM.%2BM.%26atitle%3DClassification%2520of%2520intrinsically%2520disordered%2520regions%2520and%2520proteins%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26spage%3D6589%26epage%3D6631%26doi%3D10.1021%2Fcr400525m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uversky, V. N.</span></span> <span> </span><span class="NLM_article-title">Intrinsically disordered proteins and their “mysterious” (meta)physics</span>. <i>Front. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">10</span>, <span class="refDoi"> DOI: 10.3389/fphy.2019.00010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.3389%2Ffphy.2019.00010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=10&author=V.+N.+Uversky&title=Intrinsically+disordered+proteins+and+their+%E2%80%9Cmysterious%E2%80%9D+%28meta%29physics&doi=10.3389%2Ffphy.2019.00010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.3389%2Ffphy.2019.00010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphy.2019.00010%26sid%3Dliteratum%253Aachs%26aulast%3DUversky%26aufirst%3DV.%2BN.%26atitle%3DIntrinsically%2520disordered%2520proteins%2520and%2520their%2520%25E2%2580%259Cmysterious%25E2%2580%259D%2520%2528meta%2529physics%26jtitle%3DFront.%2520Phys.%26date%3D2019%26volume%3D7%26spage%3D10%26doi%3D10.3389%2Ffphy.2019.00010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Targeting intrinsically disordered proteins at the edge of chaos</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">217</span>– <span class="NLM_lpage">227</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2018.09.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.drudis.2018.09.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30278223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVKkt7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=217-227&author=H.+Ruanauthor=Q.+Sunauthor=W.+Zhangauthor=Y.+Liuauthor=L.+Lai&title=Targeting+intrinsically+disordered+proteins+at+the+edge+of+chaos&doi=10.1016%2Fj.drudis.2018.09.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting intrinsically disordered proteins at the edge of chaos</span></div><div class="casAuthors">Ruan, Hao; Sun, Qi; Zhang, Weilin; Liu, Ying; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">217-227</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Intrinsically disordered proteins or intrinsically disordered regions (IDPs or IDRs) are those that do not fold into defined tertiary structures under physiol. conditions.  Given their prevalence in various diseases, IDPs are attractive therapeutic targets.  However, because of the dynamic nature of the IDP structure, conventional structure-based drug design methods cannot be directly applied.  Thanks to recent progress in understanding the mechanisms underlying IDP and ligand interactions, computational strategies for IDP-targeted rational drug discovery are emerging.  Here, we summarize recent developments in computational IDP drug design strategies and their successful applications, analyze the typical properties of reported IDP-binding compds. (iIDPs), and discuss the major challenges ahead as well as possible solns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom2VnnFE1np7Vg90H21EOLACvtfcHk0lj3QbhvrbSUWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVKkt7nM&md5=affd85fe8eb5642d69f1cb6e703be36d</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.09.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.09.017%26sid%3Dliteratum%253Aachs%26aulast%3DRuan%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DTargeting%2520intrinsically%2520disordered%2520proteins%2520at%2520the%2520edge%2520of%2520chaos%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26spage%3D217%26epage%3D227%26doi%3D10.1016%2Fj.drudis.2018.09.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kågedal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halliday, G. M.</span></span> <span> </span><span class="NLM_article-title">Alpha-synuclein biology in Lewy body diseases</span>. <i>Alzheimer’s Res. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">73</span>, <span class="refDoi"> DOI: 10.1186/s13195-014-0073-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1186%2Fs13195-014-0073-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=25580161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFCmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=73&author=W.+S.+Kimauthor=K.+K%C3%A5gedalauthor=G.+M.+Halliday&title=Alpha-synuclein+biology+in+Lewy+body+diseases&doi=10.1186%2Fs13195-014-0073-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Alpha-synuclein biology in Lewy body diseases</span></div><div class="casAuthors">Kim, Woojin Scott; Kaagedal, Katarina; Halliday, Glenda M.</div><div class="citationInfo"><span class="NLM_cas:title">Alzheimer's Research & Therapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">73/1-73/9</span>CODEN:
                <span class="NLM_cas:coden">ARTLCD</span>;
        ISSN:<span class="NLM_cas:issn">1758-9193</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">α-Synuclein is an abundantly expressed neuronal protein that is at the center of focus in understanding a group of neurodegenerative disorders called α-synucleinopathies, which are characterized by the presence of aggregated ´a-synuclein intracellularly.  Primary-synucleinopathies include Parkinson's disease (PD), dementia with Lewy bodies and multiple system atrophy, with α-synuclein also found secondarily in a no. of other diseases, including Alzheimer's disease.  Understanding how α-synuclein aggregates form in these different disorders is important for the understanding of its pathogenesis in Lewy body diseases.  PD is the most prevalent of the α-synucleinopathies and much of the initial research on α-synuclein Lewy body pathol. was based on PD but is also relevant to Lewy bodies in other diseases (dementia with Lewy bodies and Alzheimerαs disease).  Polymorphism and mutation studies of SNCA, the gene that encodes α-synuclein, provide much evidence for a causal link between α-synuclein and PD.  Among the primary α-synucleinopathies, multiple system atrophy is unique in that α-synuclein deposition occurs in oligodendrocytes rather than neurons.  It is unclear whether α-synuclein originates from oligodendrocytes or whether it is transmitted somehow from neurons. α-Synuclein exists as a natively unfolded monomer in the cytosol, but in the presence of lipid membranes it is thought to undergo a conformational change to a folded α-helical secondary structure that is prone to forming dimers and oligomers.  Posttranslational modification of α-synuclein, such as phosphorylation, ubiquitination and nitration, has been widely implicated in α-synuclein aggregation process and neurotoxicity.  Recent studies using animal and cell models, as well as autopsy studies of patients with neuron transplants, provided compelling evidence for prion-like propagation of α-synuclein.  This observation has implications for therapeutic strategies, and much recent effort is focused on developing antibodies that target extracellular α-synuclein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJE-_Hg3c0xLVg90H21EOLACvtfcHk0lj3QbhvrbSUWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFCmtg%253D%253D&md5=f1bf48c62287cf69fc540ce86d741ded</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1186%2Fs13195-014-0073-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13195-014-0073-2%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DW.%2BS.%26aulast%3DK%25C3%25A5gedal%26aufirst%3DK.%26aulast%3DHalliday%26aufirst%3DG.%2BM.%26atitle%3DAlpha-synuclein%2520biology%2520in%2520Lewy%2520body%2520diseases%26jtitle%3DAlzheimer%25E2%2580%2599s%2520Res.%2520Ther.%26date%3D2014%26volume%3D6%26spage%3D73%26doi%3D10.1186%2Fs13195-014-0073-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cahill, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahiri, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, J. T.</span></span> <span> </span><span class="NLM_article-title">Amyloid precursor protein and alpha synuclein translation, implications for iron and inflammation in neurodegenerative diseases</span>. <i>Biochim. Biophys. Acta, Gen. Subj.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1790</i></span>,  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">628</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2008.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.bbagen.2008.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19166904" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1yjtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1790&publication_year=2009&pages=615-628&author=C.+M.+Cahillauthor=D.+K.+Lahiriauthor=X.+Huangauthor=J.+T.+Rogers&title=Amyloid+precursor+protein+and+alpha+synuclein+translation%2C+implications+for+iron+and+inflammation+in+neurodegenerative+diseases&doi=10.1016%2Fj.bbagen.2008.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Amyloid precursor protein and alpha synuclein translation, implications for iron and inflammation in neurodegenerative diseases</span></div><div class="casAuthors">Cahill, Catherine M.; Lahiri, Debomoy K.; Huang, Xudong; Rogers, Jack T.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1790</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">615-628</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Recent studies that alleles in the hemochromatosis gene may accelerate the onset of Alzheimer's disease by five years have validated interest in the model in which metals (particularly iron) accelerate disease course.  Biochem. and biophys. measurements demonstrated the presence of elevated levels of neurotoxic copper, zinc and iron in the brains of AD patients.  Intracellular levels of APP holoprotein were shown to be modulated by iron by a mechanism that is similar to the translation control of the ferritin L- and H mRNAs by iron-responsive element (IRE) RNA stem loops in their 5' untranslated regions (5'UTRs).  More recently a putative IRE-like sequence was hypothesized present in the Parkinsons's alpha synuclein (ASYN) transcript (see [A.L. Friedlich, R.E.Tanzi, J.T. Rogers, The 5'-untranslated region of Parkinson's disease alpha-synuclein mRNA contains a predicted iron responsive element, Mol. Psychiatry 12 (2007) 222-223. [6]]).  Together with the demonstration of metal dependent translation of APP mRNA, the involvement of metals in the plaque of AD patients and of increased iron in striatal neurons in the substantia nigra (SN) of Parkinson's disease patients have stimulated the development of metal attenuating agents and iron chelators as a major new therapeutic strategy for the treatment of these neurodegenerative diseases.  In the case of AD, metal based therapeutics may ultimately prove more cost effective than the use of an amyloid vaccine as the preferred anti-amyloid therapeutic strategy to ameliorate the cognitive decline of AD patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5db9ufEacxbVg90H21EOLACvtfcHk0lj9rSRN3Dnhkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1yjtLY%253D&md5=54ca57f06dbfe2a2278b8d820ccefa3b</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2008.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2008.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DCahill%26aufirst%3DC.%2BM.%26aulast%3DLahiri%26aufirst%3DD.%2BK.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DRogers%26aufirst%3DJ.%2BT.%26atitle%3DAmyloid%2520precursor%2520protein%2520and%2520alpha%2520synuclein%2520translation%252C%2520implications%2520for%2520iron%2520and%2520inflammation%2520in%2520neurodegenerative%2520diseases%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2009%26volume%3D1790%26spage%3D615%26epage%3D628%26doi%3D10.1016%2Fj.bbagen.2008.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishnu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouradian, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Translation of the intrinsically disordered protein α-synuclein is inhibited by a small molecule targeting its structured mRNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1457</span>– <span class="NLM_lpage">1467</span>, <span class="refDoi"> DOI: 10.1073/pnas.1905057117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1905057117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31900363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFKktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=1457-1467&author=P.+Zhangauthor=H.-J.+Parkauthor=J.+Zhangauthor=E.+Junnauthor=R.+J.+Andrewsauthor=S.+P.+Velagapudiauthor=D.+Abeggauthor=K.+Vishnuauthor=M.+G.+Costalesauthor=J.+L.+Childs-Disneyauthor=A.+Adibekianauthor=W.+N.+Mossauthor=M.+M.+Mouradianauthor=M.+D.+Disney&title=Translation+of+the+intrinsically+disordered+protein+%CE%B1-synuclein+is+inhibited+by+a+small+molecule+targeting+its+structured+mRNA&doi=10.1073%2Fpnas.1905057117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Translation of the intrinsically disordered protein α-synuclein is inhibited by a small molecule targeting its structured mRNA</span></div><div class="casAuthors">Zhang, Peiyuan; Park, Hye-Jin; Zhang, Jie; Junn, Eunsung; Andrews, Ryan J.; Velagapudi, Sai Pradeep; Abegg, Daniel; Vishnu, Kamalakannan; Costales, Matthew G.; Childs-Disney, Jessica L.; Adibekian, Alexander; Moss, Walter N.; Mouradian, M. Maral; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1457-1467</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Many proteins are refractory to targeting because they lack small-mol. binding pockets.  An alternative to drugging these proteins directly is to target the messenger (m)RNA that encodes them, thereby reducing protein levels.  We describe such an approach for the difficult-to-target protein α-synuclein encoded by the SNCA gene.  Multiplication of the SNCA gene locus causes dominantly inherited Parkinson's disease (PD), and α-synuclein protein aggregates in Lewy bodies and Lewy neurites in sporadic PD.  Thus, reducing the expression of α-synuclein protein is expected to have therapeutic value.  Fortuitously, the SNCA mRNA has a structured iron-responsive element (IRE) in its 5' untranslated region (5' UTR) that controls its translation.  Using sequence-based design, we discovered small mols. that target the IRE structure and inhibit SNCA translation in cells, the most potent of which is named Synucleozid.  Both in vitro and cellular profiling studies showed Synucleozid directly targets the α-synuclein mRNA 5' UTR at the designed site.α.  Mechanistic studies revealed that Synucleozid reduces α-synuclein protein levels by decreasing the amt. of SNCA mRNA loaded into polysomes, mechanistically providing a cytoprotective effect in cells.  Proteome- and transcriptome-wide studies showed that the compd.'s selectivity makes Synucleozid suitable for further development.  Importantly, transcriptome-wide anal. of mRNAs that encode intrinsically disordered proteins revealed that each has structured regions that could be targeted with small mols.  These findings demonstrate the potential for targeting undruggable proteins at the level of their coding mRNAs.  This approach, as applied to SNCA, is a promising disease-modifying therapeutic strategy for PD and other α-synucleinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvPjL_9G6xS7Vg90H21EOLACvtfcHk0lj9rSRN3Dnhkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFKktbc%253D&md5=4f2a60d2c0fe09ea99f05291621ad4ef</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1905057117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1905057117%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DPark%26aufirst%3DH.-J.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DJunn%26aufirst%3DE.%26aulast%3DAndrews%26aufirst%3DR.%2BJ.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DVishnu%26aufirst%3DK.%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26aulast%3DMouradian%26aufirst%3DM.%2BM.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DTranslation%2520of%2520the%2520intrinsically%2520disordered%2520protein%2520%25CE%25B1-synuclein%2520is%2520inhibited%2520by%2520a%2520small%2520molecule%2520targeting%2520its%2520structured%2520mRNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D1457%26epage%3D1467%26doi%3D10.1073%2Fpnas.1905057117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, J. R.</span></span> <span> </span><span class="NLM_article-title">Kinetic intermediates in RNA folding</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>265</i></span>,  <span class="NLM_fpage">918</span>– <span class="NLM_lpage">924</span>, <span class="refDoi"> DOI: 10.1126/science.8052848</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1126%2Fscience.8052848" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=8052848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaK2cXmslClsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1994&pages=918-924&author=P.+P.+Zarrinkarauthor=J.+R.+Williamson&title=Kinetic+intermediates+in+RNA+folding&doi=10.1126%2Fscience.8052848"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic intermediates in RNA folding</span></div><div class="casAuthors">Zarrinkar, Patrick P.; Williamson, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">5174</span>),
    <span class="NLM_cas:pages">918-24</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The folding pathways of large, highly structured RNA mols. are largely unexplored.  Insight into both the kinetics of folding and the presence of intermediates was provided in a study of the Mg2+-induced folding of the Tetrahymena ribozyme by hybridization of complementary oligodeoxynucleotide probes.  This RNA folds via a complex mechanism involving both Mg2+-dependent and Mg2+-independent steps.  A hierarchical model for the folding pathway is proposed in which formation of one helical domain (P4-P6) precedes that of a second helical domain (P3-P7).  The overall rate-limiting step is formation of P3-P7, and takes place with an obsd. rate const. of 0.72 min-1.  The folding mechanism of large RNAs appears similar to that of many multidomain proteins in that formation of independently stable substructures precedes their assocn. into the final conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrty3NoGILqxbVg90H21EOLACvtfcHk0lj9rSRN3Dnhkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmslClsLk%253D&md5=db9be5dab3438ebadad715ac5e1461e3</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1126%2Fscience.8052848&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8052848%26sid%3Dliteratum%253Aachs%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DWilliamson%26aufirst%3DJ.%2BR.%26atitle%3DKinetic%2520intermediates%2520in%2520RNA%2520folding%26jtitle%3DScience%26date%3D1994%26volume%3D265%26spage%3D918%26epage%3D924%26doi%3D10.1126%2Fscience.8052848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">A toxic RNA catalyzes the in cellulo synthesis of its own inhibitor</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">10956</span>– <span class="NLM_lpage">10959</span>, <span class="refDoi"> DOI: 10.1002/anie.201406465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1002%2Fanie.201406465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=25164984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC2M%252FlsFKnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=10956-10959&author=S.+G.+Rzuczekauthor=H.+Parkauthor=M.+D.+Disney&title=A+toxic+RNA+catalyzes+the+in+cellulo+synthesis+of+its+own+inhibitor&doi=10.1002%2Fanie.201406465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">A toxic RNA catalyzes the in cellulo synthesis of its own inhibitor</span></div><div class="casAuthors">Rzuczek Suzanne G; Park HaJeung; Disney Matthew D</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">10956-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Potent modulators of RNA function can be assembled in cellulo by using the cell as a reaction vessel and a disease-causing RNA as a catalyst.  When designing small molecule effectors of function, a balance between permeability and potency must be struck.  Low molecular weight compounds are more permeable whereas higher molecular weight compounds are more potent.  The advantages of both types of compounds could be synergized if low molecular weight molecules could be transformed into potent, multivalent ligands by a reaction that is catalyzed by binding to a target in cells expressing a genetic defect.  It was shown that this approach is indeed viable in cellulo.  Small molecule modules with precisely positioned alkyne and azide moieties bind adjacent internal loops in r(CCUG)(exp), the causative agent of myotonic dystrophy type 2 (DM2), and are transformed into oligomeric, potent inhibitors of DM2 RNA dysfunction by a Huisgen 1,3-dipolar cycloaddition reaction, a variant of click chemistry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQTJ9u4F-oycFQKX3ooHxWjfW6udTcc2eZ6HAlNpLTgSLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M%252FlsFKnsw%253D%253D&md5=2fb96c08ea5546457c9cd4d050fb0201</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1002%2Fanie.201406465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201406465%26sid%3Dliteratum%253Aachs%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DA%2520toxic%2520RNA%2520catalyzes%2520the%2520in%2520cellulo%2520synthesis%2520of%2520its%2520own%2520inhibitor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2014%26volume%3D53%26spage%3D10956%26epage%3D10959%26doi%3D10.1002%2Fanie.201406465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Using genome sequence to enable the design of medicines and chemical probes</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1663</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00504</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00504" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlvFGqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=1599-1663&author=A.+J.+Angelbelloauthor=J.+L.+Chenauthor=J.+L.+Childs-Disneyauthor=P.+Zhangauthor=Z.-F.+Wangauthor=M.+D.+Disney&title=Using+genome+sequence+to+enable+the+design+of+medicines+and+chemical+probes&doi=10.1021%2Facs.chemrev.7b00504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Using Genome Sequence to Enable the Design of Medicines and Chemical Probes</span></div><div class="casAuthors">Angelbello, Alicia J.; Chen, Jonathan L.; Childs-Disney, Jessica L.; Zhang, Peiyuan; Wang, Zi-Fu; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1599-1663</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Rapid progress in genome sequencing technol. has put us firmly into a postgenomic era.  A key challenge in biomedical research is harnessing genome sequence to fulfill the promise of personalized medicine.  This Review describes how genome sequencing has enabled the identification of disease-causing biomols. and how these data have been converted into chem. probes of function, preclin. lead modalities, and ultimately U.S.  Food and Drug Administration (FDA)-approved drugs.  In particular, the authors focus on the use of oligonucleotide-based modalities to target disease-causing RNAs; small mols. that target DNA, RNA, or protein; the rational repurposing of known therapeutic modalities; and the advantages of pharmacogenetics.  Lastly, the authors discuss the remaining challenges and opportunities in the direct utilization of genome sequence to enable design of medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreyX8o6MEWwLVg90H21EOLACvtfcHk0lhhroh2a5FOGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlvFGqsQ%253D%253D&md5=6cf4911da55d4304d7c818f0f7a4a6a0</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00504%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DZ.-F.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DUsing%2520genome%2520sequence%2520to%2520enable%2520the%2520design%2520of%2520medicines%2520and%2520chemical%2520probes%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D1599%26epage%3D1663%26doi%3D10.1021%2Facs.chemrev.7b00504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mateos-Aierdi, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goicoechea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiastui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Torrón, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Puga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matheu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López de Munain, A.</span></span> <span> </span><span class="NLM_article-title">Muscle wasting in myotonic dystrophies: a model of premature aging</span>. <i>Front. Aging Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">125</span>, <span class="refDoi"> DOI: 10.3389/fnagi.2015.00125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.3389%2Ffnagi.2015.00125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=26217220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC28%252FnsVOgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=125&author=A.+J.+Mateos-Aierdiauthor=M.+Goicoecheaauthor=A.+Aiastuiauthor=R.+Fern%C3%A1ndez-Torr%C3%B3nauthor=M.+Garcia-Pugaauthor=A.+Matheuauthor=A.+L%C3%B3pez+de+Munain&title=Muscle+wasting+in+myotonic+dystrophies%3A+a+model+of+premature+aging&doi=10.3389%2Ffnagi.2015.00125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Muscle wasting in myotonic dystrophies: a model of premature aging</span></div><div class="casAuthors">Mateos-Aierdi Alba Judith; Goicoechea Maria; Aiastui Ana; Fernandez-Torron Roberto; Garcia-Puga Mikel; Matheu Ander; Lopez de Munain Adolfo</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in aging neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">125</span>
        ISSN:<span class="NLM_cas:issn">1663-4365</span>.
    </div><div class="casAbstract">Myotonic dystrophy type 1 (DM1 or Steinert's disease) and type 2 (DM2) are multisystem disorders of genetic origin.  Progressive muscular weakness, atrophy and myotonia are the most prominent neuromuscular features of these diseases, while other clinical manifestations such as cardiomyopathy, insulin resistance and cataracts are also common.  From a clinical perspective, most DM symptoms are interpreted as a result of an accelerated aging (cataracts, muscular weakness and atrophy, cognitive decline, metabolic dysfunction, etc.), including an increased risk of developing tumors.  From this point of view, DM1 could be described as a progeroid syndrome since a notable age-dependent dysfunction of all systems occurs.  The underlying molecular disorder in DM1 consists of the existence of a pathological (CTG) triplet expansion in the 3' untranslated region (UTR) of the Dystrophia Myotonica Protein Kinase (DMPK) gene, whereas (CCTG)n repeats in the first intron of the Cellular Nucleic acid Binding Protein/Zinc Finger Protein 9 (CNBP/ZNF9) gene cause DM2.  The expansions are transcribed into (CUG)n and (CCUG)n-containing RNA, respectively, which form secondary structures and sequester RNA-binding proteins, such as the splicing factor muscleblind-like protein (MBNL), forming nuclear aggregates known as foci.  Other splicing factors, such as CUGBP, are also disrupted, leading to a spliceopathy of a large number of downstream genes linked to the clinical features of these diseases.  Skeletal muscle regeneration relies on muscle progenitor cells, known as satellite cells, which are activated after muscle damage, and which proliferate and differentiate to muscle cells, thus regenerating the damaged tissue.  Satellite cell dysfunction seems to be a common feature of both age-dependent muscle degeneration (sarcopenia) and muscle wasting in DM and other muscle degenerative diseases.  This review aims to describe the cellular, molecular and macrostructural processes involved in the muscular degeneration seen in DM patients, highlighting the similarities found with muscle aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSw12Ob59ewiLxanA7vODO6fW6udTcc2eZ6HAlNpLTgSLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252FnsVOgtA%253D%253D&md5=fa6238e374e69abf9ae9cf9dda9be4ae</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.3389%2Ffnagi.2015.00125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnagi.2015.00125%26sid%3Dliteratum%253Aachs%26aulast%3DMateos-Aierdi%26aufirst%3DA.%2BJ.%26aulast%3DGoicoechea%26aufirst%3DM.%26aulast%3DAiastui%26aufirst%3DA.%26aulast%3DFern%25C3%25A1ndez-Torr%25C3%25B3n%26aufirst%3DR.%26aulast%3DGarcia-Puga%26aufirst%3DM.%26aulast%3DMatheu%26aufirst%3DA.%26aulast%3DL%25C3%25B3pez%2Bde%2BMunain%26aufirst%3DA.%26atitle%3DMuscle%2520wasting%2520in%2520myotonic%2520dystrophies%253A%2520a%2520model%2520of%2520premature%2520aging%26jtitle%3DFront.%2520Aging%2520Neurosci.%26date%3D2015%26volume%3D7%26spage%3D125%26doi%3D10.3389%2Ffnagi.2015.00125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meola, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardani, R.</span></span> <span> </span><span class="NLM_article-title">Myotonic dystrophies: An update on clinical aspects, genetic, pathology, and molecular pathomechanisms</span>. <i>Biochim. Biophys. Acta, Mol. Basis Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1852</i></span>,  <span class="NLM_fpage">594</span>– <span class="NLM_lpage">606</span>, <span class="refDoi"> DOI: 10.1016/j.bbadis.2014.05.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.bbadis.2014.05.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslKktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1852&publication_year=2015&pages=594-606&author=G.+Meolaauthor=R.+Cardani&title=Myotonic+dystrophies%3A+An+update+on+clinical+aspects%2C+genetic%2C+pathology%2C+and+molecular+pathomechanisms&doi=10.1016%2Fj.bbadis.2014.05.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Myotonic dystrophies: An update on clinical aspects, genetic, pathology, and molecular pathomechanisms</span></div><div class="casAuthors">Meola, Giovanni; Cardani, Rosanna</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Basis of Disease</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1852</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">594-606</span>CODEN:
                <span class="NLM_cas:coden">BBADEX</span>;
        ISSN:<span class="NLM_cas:issn">0925-4439</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  Myotonic dystrophy (DM) is the most common adult muscular dystrophy, characterized by autosomal dominant progressive myopathy, myotonia and multiorgan involvement.  To date two distinct forms caused by similar mutations have been identified.  Myotonic dystrophy type 1 (DM1, Steinert's disease) is caused by a (CTG)n expansion in DMPK, while myotonic dystrophy type 2 (DM2) is caused by a (CCTG)n expansion in ZNF9/CNBP.  When transcribed into CUG/CCUG-contg. RNA, mutant transcripts aggregate as nuclear foci that sequester RNA-binding proteins, resulting in spliceopathy of downstream effector genes.  However, it is now clear that addnl. pathogenic mechanism like changes in gene expression, protein translation and micro-RNA metab. may also contribute to disease pathol.  Despite clin. and genetic similarities, DM1 and DM2 are distinct disorders requiring different diagnostic and management strategies.  This review is an update on the recent advances in the understanding of the mol. mechanisms behind myotonic dystrophies.  This article is part of a Special Issue entitled: Neuromuscular Diseases: Pathol. and Mol. Pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB007xIEAaQrVg90H21EOLACvtfcHk0ljX6w3Z7WvrfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslKktbc%253D&md5=1c3446d7430fcfd9222b460925e910db</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1016%2Fj.bbadis.2014.05.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbadis.2014.05.019%26sid%3Dliteratum%253Aachs%26aulast%3DMeola%26aufirst%3DG.%26aulast%3DCardani%26aufirst%3DR.%26atitle%3DMyotonic%2520dystrophies%253A%2520An%2520update%2520on%2520clinical%2520aspects%252C%2520genetic%252C%2520pathology%252C%2520and%2520molecular%2520pathomechanisms%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Basis%2520Dis.%26date%3D2015%26volume%3D1852%26spage%3D594%26epage%3D606%26doi%3D10.1016%2Fj.bbadis.2014.05.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Udd, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krahe, R.</span></span> <span> </span><span class="NLM_article-title">The myotonic dystrophies: molecular, clinical, and therapeutic challenges</span>. <i>Lancet Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">891</span>– <span class="NLM_lpage">905</span>, <span class="refDoi"> DOI: 10.1016/S1474-4422(12)70204-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2FS1474-4422%2812%2970204-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=22995693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVSns7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=891-905&author=B.+Uddauthor=R.+Krahe&title=The+myotonic+dystrophies%3A+molecular%2C+clinical%2C+and+therapeutic+challenges&doi=10.1016%2FS1474-4422%2812%2970204-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">The myotonic dystrophies: molecular, clinical, and therapeutic challenges</span></div><div class="casAuthors">Udd, Bjarne; Krahe, Ralf</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Neurology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">891-905</span>CODEN:
                <span class="NLM_cas:coden">LNAEAM</span>;
        ISSN:<span class="NLM_cas:issn">1474-4422</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Summary: Myotonic dystrophy is the most common type of muscular dystrophy in adults and is characterized by progressive myopathy, myotonia, and multiorgan involvement.  Two genetically distinct entities have been identified.  Myotonic dystrophy type 1 (also known as Steinert's disease) was first described more than 100 years ago, whereas myotonic dystrophy type 2 was identified only 18 years ago, after genetic testing for type 1 disease could be applied.  Both diseases are caused by autosomal dominant nucleotide repeat expansions.  In patients with myotonic dystrophy type 1, a (CTG)n expansion is present in DMPK, whereas in patients with type 2 disease, there is a (CCTG)n expansion in CNBP.  When transcribed into CUG-contg. RNA, mutant transcripts aggregate as nuclear foci that sequester RNA-binding proteins, resulting in a spliceopathy of downstream effector genes.  The prevailing paradigm therefore is that both disorders are toxic RNA diseases.  However, research indicates several addnl. pathogenic effects take place with respect to protein translation and turnover.  Despite clin. and genetic similarities, myotonic dystrophy type 1 and type 2 are distinct disorders requiring different diagnostic and management strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqElKJ28_rHObVg90H21EOLACvtfcHk0ljX6w3Z7WvrfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVSns7fK&md5=40f7d2204cc59403b438ce8840d37498</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1016%2FS1474-4422%2812%2970204-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1474-4422%252812%252970204-1%26sid%3Dliteratum%253Aachs%26aulast%3DUdd%26aufirst%3DB.%26aulast%3DKrahe%26aufirst%3DR.%26atitle%3DThe%2520myotonic%2520dystrophies%253A%2520molecular%252C%2520clinical%252C%2520and%2520therapeutic%2520challenges%26jtitle%3DLancet%2520Neurol.%26date%3D2012%26volume%3D11%26spage%3D891%26epage%3D905%26doi%3D10.1016%2FS1474-4422%2812%2970204-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Features of modularly assembled compounds that impart bioactivity against an RNA target</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2321</span>, <span class="refDoi"> DOI: 10.1021/cb400265y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400265y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVCgtL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2312-2321&author=S.+G.+Rzuczekauthor=Y.+Gaoauthor=Z.-Z.+Tangauthor=C.+A.+Thorntonauthor=T.+Kodadekauthor=M.+D.+Disney&title=Features+of+modularly+assembled+compounds+that+impart+bioactivity+against+an+RNA+target&doi=10.1021%2Fcb400265y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Features of Modularly Assembled Compounds That Impart Bioactivity Against an RNA Target</span></div><div class="casAuthors">Rzuczek, Suzanne G.; Gao, Yu; Tang, Zhen-Zhi; Thornton, Charles A.; Kodadek, Thomas; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2312-2321</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Transcriptomes provide a myriad of potential RNAs that could be the targets of therapeutics or chem. genetic probes of function.  Cell-permeable small mols., however, generally do not exploit these targets, owing to the difficulty in the design of high affinity, specific small mols. targeting RNA.  As part of a general program to study RNA function using small mols., we designed bioactive, modularly assembled small mols. that target the noncoding expanded RNA repeat that causes myotonic dystrophy type 1 (DM1), r-(CUG)exp.  Herein, we present a rigorous study to elucidate features in modularly assembled compds. that afford bioactivity.  Different modular assembly scaffolds were investigated, including polyamines, α-peptides, β-peptides, and peptide tertiary amides (PTAs).  On the basis of activity as assessed by improvement of DM1-assocd. defects, stability against proteases, cellular permeability, and toxicity, we discovered that constrained backbones, namely, PTAs, are optimal.  Notably, we detd. that r-(CUG)exp is the target of the optimal PTA in cellular models and that the optimal PTA improves DM1-assocd. defects in a mouse model.  Biophys. analyses were employed to investigate potential sources of bioactivity.  These investigations show that modularly assembled compds. have increased residence times on their targets and faster on rates than the RNA-binding modules from which they were derived.  Moreover, they have faster on rates than the protein that binds r-(CUG)exp, the inactivation of which gives rise to DM1-assocd. defects.  These studies provide information about features of small mols. that are programmable for targeting RNA, allowing for the facile optimization of therapeutics or chem. probes against other cellular RNA targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiiHXEm7vakrVg90H21EOLACvtfcHk0ljX6w3Z7WvrfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVCgtL%252FL&md5=89a6385bd74e38e54bde583c339c23cc</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1021%2Fcb400265y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400265y%26sid%3Dliteratum%253Aachs%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DZ.-Z.%26aulast%3DThornton%26aufirst%3DC.%2BA.%26aulast%3DKodadek%26aufirst%3DT.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DFeatures%2520of%2520modularly%2520assembled%2520compounds%2520that%2520impart%2520bioactivity%2520against%2520an%2520RNA%2520target%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2312%26epage%3D2321%26doi%3D10.1021%2Fcb400265y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moon, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilimire, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneekloth, J. S.</span></span> <span> </span><span class="NLM_article-title">Measuring RNA-ligand interactions with microscale thermophoresis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4638</span>– <span class="NLM_lpage">4643</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.7b01141</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.7b01141" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1Slsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=4638-4643&author=M.+H.+Moonauthor=T.+A.+Hilimireauthor=A.+M.+Sandersauthor=J.+S.+Schneekloth&title=Measuring+RNA-ligand+interactions+with+microscale+thermophoresis&doi=10.1021%2Facs.biochem.7b01141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Measuring RNA-Ligand Interactions with Microscale Thermophoresis</span></div><div class="casAuthors">Moon, Michelle H.; Hilimire, Thomas A.; Sanders, Allix M.; Schneekloth, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">4638-4643</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the past twenty years, there has been a dramatic increase in the study of RNA.  RNA has gone from being known as an intermediate in the central dogma of mol. biol. to a mol. with a large diversity of structure and function that is involved in all aspects of biol.  As new functions are rapidly discovered, it has become clear that there is a need for RNA-targeting small mol. probes to investigate RNA biol. and clarify the potential for therapeutics based on RNA/small mol. interactions.  While a host of techniques exist to measure small mol./RNA interactions, many of these have drawbacks that make them intractable for routine use and are often not broadly applicable.  A newer technol. called microscale thermophoresis (MST), which measures the directed migration of a mol. and/or mol. ligand complex along a temp. gradient, can be used to measure binding affinities using very small amts. of sample, greatly reducing cost of materials.  Here, we demonstrate how MST can be used to study a range of RNA interactions, including peptide/RNA interactions, RNA/small mol. interactions, and displacement of an RNA bound peptide by a small mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUVOWHOSVjTLVg90H21EOLACvtfcHk0li5DjHZCvW71A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1Slsg%253D%253D&md5=0aad61e97e0a21d3004e7f99c3b57c31</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.7b01141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.7b01141%26sid%3Dliteratum%253Aachs%26aulast%3DMoon%26aufirst%3DM.%2BH.%26aulast%3DHilimire%26aufirst%3DT.%2BA.%26aulast%3DSanders%26aufirst%3DA.%2BM.%26aulast%3DSchneekloth%26aufirst%3DJ.%2BS.%26atitle%3DMeasuring%2520RNA-ligand%2520interactions%2520with%2520microscale%2520thermophoresis%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26spage%3D4638%26epage%3D4643%26doi%3D10.1021%2Facs.biochem.7b01141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Umuhire Juru, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patwardhan, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargrove, A. E.</span></span> <span> </span><span class="NLM_article-title">Understanding the contributions of conformational changes, thermodynamics, and kinetics of RNA-small molecule interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">824</span>– <span class="NLM_lpage">838</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00945</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00945" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosFWntbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=824-838&author=A.+Umuhire+Juruauthor=N.+N.+Patwardhanauthor=A.+E.+Hargrove&title=Understanding+the+contributions+of+conformational+changes%2C+thermodynamics%2C+and+kinetics+of+RNA-small+molecule+interactions&doi=10.1021%2Facschembio.8b00945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding the Contributions of Conformational Changes, Thermodynamics, and Kinetics of RNA-Small Molecule Interactions</span></div><div class="casAuthors">Umuhire Juru, Aline; Patwardhan, Neeraj N.; Hargrove, Amanda E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">824-838</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The implication of RNA in multiple cellular processes beyond protein coding has revitalized interest in the development of small mols. for therapeutically targeting RNA and for further probing its cellular biol.  However, the process of rationally designing such small mol. probes is hampered by the paucity of information about fundamental mol. recognition principles of RNA.  In this Review, the authors summarize two important and often underappreciated aspects of RNA-small mol. recognition: RNA conformational dynamics and the biophys. properties of interactions of small mols. with RNA, specifically thermodn. and kinetics.  While conformational flexibility is often said to impede RNA ligand development, the ability of small mols. to influence the RNA conformational landscape can have a significant effect on the cellular functions of RNA.  An anal. of the conformational landscape of RNA and the interactions of individual conformations with ligands can thus guide the development of new small mol. probes, which needs to be investigated further.  Addnl., while it is common practice to quantify the binding affinities (Ka or Kd) of small mols. for biomacromols. as a measure of their activity, further biophys. characterization of their interaction can provide a deeper understanding.  Studies that focus on the thermodn. and kinetic parameters for interaction between RNA and ligands are next discussed.  Finally, this Review provides the reader with a perspective on how such in-depth anal. of biophys. characteristics of the interaction of RNA and small mols. can impact the understanding of these interactions and how they will benefit the future design of small mol. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9o1fhVs7EBbVg90H21EOLACvtfcHk0li5DjHZCvW71A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosFWntbs%253D&md5=c8986916178a2acc361c64f5c18327ee</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00945%26sid%3Dliteratum%253Aachs%26aulast%3DUmuhire%2BJuru%26aufirst%3DA.%26aulast%3DPatwardhan%26aufirst%3DN.%2BN.%26aulast%3DHargrove%26aufirst%3DA.%2BE.%26atitle%3DUnderstanding%2520the%2520contributions%2520of%2520conformational%2520changes%252C%2520thermodynamics%252C%2520and%2520kinetics%2520of%2520RNA-small%2520molecule%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D824%26epage%3D838%26doi%3D10.1021%2Facschembio.8b00945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blakeley, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePorter, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolbert, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNaughton, B. R.</span></span> <span> </span><span class="NLM_article-title">Methods for identifying and characterizing interactions involving RNA</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">8837</span>– <span class="NLM_lpage">8855</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2012.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.tet.2012.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVyjsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=8837-8855&author=B.+D.+Blakeleyauthor=S.+M.+DePorterauthor=U.+Mohanauthor=R.+Buraiauthor=B.+S.+Tolbertauthor=B.+R.+McNaughton&title=Methods+for+identifying+and+characterizing+interactions+involving+RNA&doi=10.1016%2Fj.tet.2012.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Methods for identifying and characterizing interactions involving RNA</span></div><div class="casAuthors">Blakeley, Brett D.; DePorter, Sandra M.; Mohan, Utpal; Burai, Ritwik; Tolbert, Blanton S.; McNaughton, Brian R.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">8837-8855</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Topics discussed include fluorescence-based methods, microarrays, selection-based methods, and low-throughput methods for measuring ligand-RNA interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc9bjJ4rDA_LVg90H21EOLACvtfcHk0li5DjHZCvW71A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVyjsLnP&md5=ecbfe5736239b30b9e615d3b51dc78ed</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2012.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2012.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DBlakeley%26aufirst%3DB.%2BD.%26aulast%3DDePorter%26aufirst%3DS.%2BM.%26aulast%3DMohan%26aufirst%3DU.%26aulast%3DBurai%26aufirst%3DR.%26aulast%3DTolbert%26aufirst%3DB.%2BS.%26aulast%3DMcNaughton%26aufirst%3DB.%2BR.%26atitle%3DMethods%2520for%2520identifying%2520and%2520characterizing%2520interactions%2520involving%2520RNA%26jtitle%3DTetrahedron%26date%3D2012%26volume%3D68%26spage%3D8837%26epage%3D8855%26doi%3D10.1016%2Fj.tet.2012.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wicks, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargrove, A. E.</span></span> <span> </span><span class="NLM_article-title">Fluorescent indicator displacement assays to identify and characterize small molecule interactions with RNA</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2019.04.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.ymeth.2019.04.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31051253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpsVOqs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2019&pages=3-14&author=S.+L.+Wicksauthor=A.+E.+Hargrove&title=Fluorescent+indicator+displacement+assays+to+identify+and+characterize+small+molecule+interactions+with+RNA&doi=10.1016%2Fj.ymeth.2019.04.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescent indicator displacement assays to identify and characterize small molecule interactions with RNA</span></div><div class="casAuthors">Wicks, Sarah L.; Hargrove, Amanda E.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3-14</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Fluorescent indicator displacement (FID) assays are an advantageous approach to convert receptors into optical sensors that can detect binding of various ligands.  In particular, the identification of ligands that bind to RNA receptors has become of increasing interest as the roles of RNA in cellular processes and disease pathogenesis continue to be discovered.  Small mols. have been validated as tools to elucidate unknown RNA functions, underscoring the crit. need to rapidly identify and quant. characterize RNA:small mol. interactions for the development of chem. probes.  The successful application of FID assays to evaluate interactions between diverse RNA receptors and small mols. has been facilitated by the characterization of distinct fluorescent indicators that reversibly bind RNA and modulate the fluorescence signal.  The utility of RNA-based FID assays to both academia and industry has been demonstrated through numerous uses in high-throughput screening efforts, structure-activity relationship studies, and in vitro target engagement studies.  Furthermore, the development, optimization, and validation of a variety of RNA-based FID assays has led to general guidelines that can be utilized for facile implementation of the method with new or underexplored RNA receptors.  Altogether, the use of RNA-based FID assays as a general anal. tool has provided valuable insights into small mol. affinity and selectivity, furthering the fundamental understanding of RNA:small mol. recognition.  In this review, we will summarize efforts to employ FID assays using RNA receptors and describe the significant contributions of the method towards the development of chem. probes to reveal unknown RNA functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZAGIlueOLvrVg90H21EOLACvtfcHk0ljiD0vshwS31w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpsVOqs7g%253D&md5=c6955af7de2b7beafd3e7c3879c35e60</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2019.04.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2019.04.018%26sid%3Dliteratum%253Aachs%26aulast%3DWicks%26aufirst%3DS.%2BL.%26aulast%3DHargrove%26aufirst%3DA.%2BE.%26atitle%3DFluorescent%2520indicator%2520displacement%2520assays%2520to%2520identify%2520and%2520characterize%2520small%2520molecule%2520interactions%2520with%2520RNA%26jtitle%3DMethods%26date%3D2019%26volume%3D167%26spage%3D3%26epage%3D14%26doi%3D10.1016%2Fj.ymeth.2019.04.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuliano, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palladino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">John, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowicz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sausville, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukow, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chait, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galluzzo, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheltzer, J. M.</span></span> <span> </span><span class="NLM_article-title">Off-target toxicity is a common mechanism of action of cancer drugs undergoing clinical trials</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">eaaw8412</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.aaw8412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1126%2Fscitranslmed.aaw8412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31511426" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&author=A.+Linauthor=C.+J.+Giulianoauthor=A.+Palladinoauthor=K.+M.+Johnauthor=C.+Abramowiczauthor=M.+L.+Yuanauthor=E.+L.+Sausvilleauthor=D.+A.+Lukowauthor=L.+Liuauthor=A.+R.+Chaitauthor=Z.+C.+Galluzzoauthor=C.+Tuckerauthor=J.+M.+Sheltzer&title=Off-target+toxicity+is+a+common+mechanism+of+action+of+cancer+drugs+undergoing+clinical+trials&doi=10.1126%2Fscitranslmed.aaw8412"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aaw8412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aaw8412%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DA.%26aulast%3DGiuliano%26aufirst%3DC.%2BJ.%26aulast%3DPalladino%26aufirst%3DA.%26aulast%3DJohn%26aufirst%3DK.%2BM.%26aulast%3DAbramowicz%26aufirst%3DC.%26aulast%3DYuan%26aufirst%3DM.%2BL.%26aulast%3DSausville%26aufirst%3DE.%2BL.%26aulast%3DLukow%26aufirst%3DD.%2BA.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChait%26aufirst%3DA.%2BR.%26aulast%3DGalluzzo%26aufirst%3DZ.%2BC.%26aulast%3DTucker%26aufirst%3DC.%26aulast%3DSheltzer%26aufirst%3DJ.%2BM.%26atitle%3DOff-target%2520toxicity%2520is%2520a%2520common%2520mechanism%2520of%2520action%2520of%2520cancer%2520drugs%2520undergoing%2520clinical%2520trials%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2019%26volume%3D11%26doi%3D10.1126%2Fscitranslmed.aaw8412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirley, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bleackley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shafee, T.</span></span> <span> </span><span class="NLM_article-title">Transcriptomics technologies</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e1005457</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005457</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1371%2Fjournal.pcbi.1005457" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28545146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1CitrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&author=R.+Loweauthor=N.+Shirleyauthor=M.+Bleackleyauthor=S.+Dolanauthor=T.+Shafee&title=Transcriptomics+technologies&doi=10.1371%2Fjournal.pcbi.1005457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptomics technologies</span></div><div class="casAuthors">Lowe, Rohan; Shirley, Neil; Bleackley, Mark; Dolan, Stephen; Shafee, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e1005457/1-e1005457/23</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Transcriptomics technologies are the techniques used to study an organism's transcriptome, the sum of all of its RNA transcripts.  The information content of an organism is recorded in the DNA of its genome and expressed through transcription.  Here, mRNA serves as a transient intermediary mol. in the information network, while noncoding RNAs perform addnl. diverse functions.  A transcriptome captures a snapshot in time of the total transcripts present in a cell.  The first attempts to study the whole transcriptome began in the early 1990s, and technol. advances since the late 1990s have made transcriptomics a widespread discipline.  Transcriptomics has been defined by repeated technol. innovations that transform the field.  There are two key contemporary techniques in the field: microarrays, which quantify a set of predetd. sequences, and RNA sequencing (RNA-Seq), which uses high-throughput sequencing to capture all sequences.  Measuring the expression of an organism's genes in different tissues, conditions, or time points gives information on how genes are regulated and reveals details of an organism's biol.  It can also help to infer the functions of previously unannotated genes.  Transcriptomic anal. has enabled the study of how gene expression changes in different organisms and has been instrumental in the understanding of human disease.  An anal. of gene expression in its entirety allows detection of broad coordinated trends which cannot be discerned by more targeted assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ8OybmUCAlbVg90H21EOLACvtfcHk0ljiD0vshwS31w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1CitrbM&md5=9e38118fab964a734cb224d1c5c08156</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005457%26sid%3Dliteratum%253Aachs%26aulast%3DLowe%26aufirst%3DR.%26aulast%3DShirley%26aufirst%3DN.%26aulast%3DBleackley%26aufirst%3DM.%26aulast%3DDolan%26aufirst%3DS.%26aulast%3DShafee%26aufirst%3DT.%26atitle%3DTranscriptomics%2520technologies%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2017%26volume%3D13%26doi%3D10.1371%2Fjournal.pcbi.1005457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Git, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvinge, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmon-Divon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadfield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertone, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldas, C.</span></span> <span> </span><span class="NLM_article-title">Systematic comparison of microarray profiling, real-time PCR, and next-generation sequencing technologies for measuring differential microRNA expression</span>. <i>RNA</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">991</span>– <span class="NLM_lpage">1006</span>, <span class="refDoi"> DOI: 10.1261/rna.1947110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1261%2Frna.1947110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=20360395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlslOmu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=991-1006&author=A.+Gitauthor=H.+Dvingeauthor=M.+Salmon-Divonauthor=M.+Osborneauthor=C.+Kutterauthor=J.+Hadfieldauthor=P.+Bertoneauthor=C.+Caldas&title=Systematic+comparison+of+microarray+profiling%2C+real-time+PCR%2C+and+next-generation+sequencing+technologies+for+measuring+differential+microRNA+expression&doi=10.1261%2Frna.1947110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic comparison of microarray profiling, real-time PCR, and next-generation sequencing technologies for measuring differential microRNA expression</span></div><div class="casAuthors">Git, Anna; Dvinge, Heidi; Salmon-Divon, Mali; Osborne, Michelle; Kutter, Claudia; Hadfield, James; Bertone, Paul; Caldas, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">RNA</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">991-1006</span>CODEN:
                <span class="NLM_cas:coden">RNARFU</span>;
        ISSN:<span class="NLM_cas:issn">1355-8382</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">RNA abundance and DNA copy no. are routinely measured in high-throughput using microarray and next-generation sequencing (NGS) technologies, and the attributes of different platforms have been extensively analyzed.  Recently, the application of both microarrays and NGS has expanded to include microRNAs (miRNAs), but the relative performance of these methods has not been rigorously characterized.  We analyzed three biol. samples across six miRNA microarray platforms and compared their hybridization performance.  We examd. the utility of these platforms, as well as NGS, for the detection of differentially expressed miRNAs.  We then validated the results for 89 miRNAs by real-time RT-PCR and challenged the use of this assay as a "gold std.".  Finally, we implemented a novel method to evaluate false-pos. and false-neg. rates for all methods in the absence of a ref. method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqK5DxDMY2fNLVg90H21EOLACvtfcHk0ljiD0vshwS31w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlslOmu7s%253D&md5=d7e5d56b792ccf2ad5f8d3e4552afd27</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1261%2Frna.1947110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.1947110%26sid%3Dliteratum%253Aachs%26aulast%3DGit%26aufirst%3DA.%26aulast%3DDvinge%26aufirst%3DH.%26aulast%3DSalmon-Divon%26aufirst%3DM.%26aulast%3DOsborne%26aufirst%3DM.%26aulast%3DKutter%26aufirst%3DC.%26aulast%3DHadfield%26aufirst%3DJ.%26aulast%3DBertone%26aufirst%3DP.%26aulast%3DCaldas%26aufirst%3DC.%26atitle%3DSystematic%2520comparison%2520of%2520microarray%2520profiling%252C%2520real-time%2520PCR%252C%2520and%2520next-generation%2520sequencing%2520technologies%2520for%2520measuring%2520differential%2520microRNA%2520expression%26jtitle%3DRNA%26date%3D2010%26volume%3D16%26spage%3D991%26epage%3D1006%26doi%3D10.1261%2Frna.1947110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bittker, J. A.</span></span> <span> </span><span class="NLM_article-title">High-throughput RT-PCR for small-molecule screening assays</span>. <i>Curr. Protoc. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1002/9780470559277.ch110204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1002%2F9780470559277.ch110204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=23487248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC2srmvFGqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=49-63&author=J.+A.+Bittker&title=High-throughput+RT-PCR+for+small-molecule+screening+assays&doi=10.1002%2F9780470559277.ch110204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput RT-PCR for small-molecule screening assays</span></div><div class="casAuthors">Bittker Joshua A</div><div class="citationInfo"><span class="NLM_cas:title">Current protocols in chemical biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Quantitative measurement of the levels of mRNA expression using real-time reverse transcription polymerase chain reaction (RT-PCR) has long been used for analyzing expression differences in tissue or cell lines of interest.  This method has been used somewhat less frequently to measure the changes in gene expression due to perturbagens such as small molecules or siRNA.  The availability of new instrumentation for liquid handling and real-time PCR analysis as well as the commercial availability of start-to-finish kits for RT-PCR has enabled the use of this method for high-throughput small-molecule screening on a scale comparable to traditional high-throughput screening (HTS) assays.  This protocol focuses on the special considerations necessary for using quantitative RT-PCR as a primary small-molecule screening assay, including the different methods available for mRNA isolation and analysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT8tsd63zvM2ZotVjB3IRIyfW6udTcc2ea6ooDEEBMyUrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srmvFGqtw%253D%253D&md5=d580a34d032303110167269f4bc5626d</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1002%2F9780470559277.ch110204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9780470559277.ch110204%26sid%3Dliteratum%253Aachs%26aulast%3DBittker%26aufirst%3DJ.%2BA.%26atitle%3DHigh-throughput%2520RT-PCR%2520for%2520small-molecule%2520screening%2520assays%26jtitle%3DCurr.%2520Protoc.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D4%26spage%3D49%26epage%3D63%26doi%3D10.1002%2F9780470559277.ch110204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Citri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Südhof, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wernig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malenka, R. C.</span></span> <span> </span><span class="NLM_article-title">Comprehensive qPCR profiling of gene expression in single neuronal cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">127</span>, <span class="refDoi"> DOI: 10.1038/nprot.2011.430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnprot.2011.430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Kitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=118-127&author=A.+Citriauthor=Z.+P.+Pangauthor=T.+C.+S%C3%BCdhofauthor=M.+Wernigauthor=R.+C.+Malenka&title=Comprehensive+qPCR+profiling+of+gene+expression+in+single+neuronal+cells&doi=10.1038%2Fnprot.2011.430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive qPCR profiling of gene expression in single neuronal cells</span></div><div class="casAuthors">Citri, Ami; Pang, Zhiping P.; Sudhof, Thomas C.; Wernig, Marius; Malenka, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">118-127</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A major challenge in neuronal stem cell biol. lies in characterization of lineage-specific reprogrammed human neuronal cells, a process that necessitates the use of an assay sensitive to the single-cell level.  Single-cell gene profiling can provide definitive evidence regarding the conversion of one cell type into another at a high level of resoln.  The protocol we describe uses Fluidigm Biomark dynamic arrays for high-throughput expression profiling from single neuronal cells, assaying up to 96 independent samples with up to 96 quant. PCR (qPCR) probes (equiv. to 9,216 reactions) in a single expt., which can be completed within 2-3 d.  The protocol enables simple and cost-effective profiling of several hundred transcripts from a single cell, and it could have numerous utilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW5UaKD-acu7Vg90H21EOLACvtfcHk0lgs5bHW0OUBMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Kitb0%253D&md5=a5211923274be519f9aceb1f3de49996</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2011.430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2011.430%26sid%3Dliteratum%253Aachs%26aulast%3DCitri%26aufirst%3DA.%26aulast%3DPang%26aufirst%3DZ.%2BP.%26aulast%3DS%25C3%25BCdhof%26aufirst%3DT.%2BC.%26aulast%3DWernig%26aufirst%3DM.%26aulast%3DMalenka%26aufirst%3DR.%2BC.%26atitle%3DComprehensive%2520qPCR%2520profiling%2520of%2520gene%2520expression%2520in%2520single%2520neuronal%2520cells%26jtitle%3DNat.%2520Protoc.%26date%3D2012%26volume%3D7%26spage%3D118%26epage%3D127%26doi%3D10.1038%2Fnprot.2011.430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bosc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, P.</span></span> <span> </span><span class="NLM_article-title">The use of novel selectivity metrics in kinase research</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">17</span>, <span class="refDoi"> DOI: 10.1186/s12859-016-1413-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1186%2Fs12859-016-1413-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28056771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC1c7is1Orsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=17&author=N.+Boscauthor=C.+Meyerauthor=P.+Bonnet&title=The+use+of+novel+selectivity+metrics+in+kinase+research&doi=10.1186%2Fs12859-016-1413-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">The use of novel selectivity metrics in kinase research</span></div><div class="casAuthors">Bosc Nicolas; Bonnet Pascal; Meyer Christophe</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Compound selectivity is an important issue when developing a new drug.  In many instances, a lack of selectivity can translate to increased toxicity.  Protein kinases are particularly concerned with this issue because they share high sequence and structural similarity.  However, selectivity may be assessed early on using data generated from protein kinase profiling panels.  RESULTS:  To guide lead optimization in drug discovery projects, we propose herein two new selectivity metrics, namely window score (WS) and ranking score (RS).  These metrics can be applied to standard in vitro data-including intrinsic enzyme activity/affinity (Ki, IC50 or percentage of inhibition), cell-based potency (percentage of effect, EC50) or even kinetics data (Kd, Kon and Koff).  They are both easy to compute and offer different viewpoints from which to consider compound selectivity.  CONCLUSIONS:  We performed a comparative analysis of their respective performance on several data sets against already published selectivity metrics and analyzed how they might influence compound selection.  Our results showed that the two new metrics bring additional information to prioritize compound selection.  Two novel metrics were developed to better estimate selectivity of compounds screened on multiple proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQBArFY-xzq77BItaL1WaVAfW6udTcc2ea6ooDEEBMyUrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c7is1Orsg%253D%253D&md5=c8bb088288d1fc3fedc2628303ddeb46</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1186%2Fs12859-016-1413-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-016-1413-y%26sid%3Dliteratum%253Aachs%26aulast%3DBosc%26aufirst%3DN.%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DBonnet%26aufirst%3DP.%26atitle%3DThe%2520use%2520of%2520novel%2520selectivity%2520metrics%2520in%2520kinase%2520research%26jtitle%3DBMC%2520Bioinf.%26date%3D2017%26volume%3D18%26spage%3D17%26doi%3D10.1186%2Fs12859-016-1413-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Graczyk, P. P.</span></span> <span> </span><span class="NLM_article-title">Gini coefficient: a new way to express selectivity of kinase inhibitors against a family of kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">5773</span>– <span class="NLM_lpage">5779</span>, <span class="refDoi"> DOI: 10.1021/jm070562u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070562u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFyhu7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=5773-5779&author=P.+P.+Graczyk&title=Gini+coefficient%3A+a+new+way+to+express+selectivity+of+kinase+inhibitors+against+a+family+of+kinases&doi=10.1021%2Fjm070562u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Gini Coefficient: A New Way To Express Selectivity of Kinase Inhibitors against a Family of Kinases</span></div><div class="casAuthors">Graczyk, Piotr P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5773-5779</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel application of the Gini coeff. for expressing selectivity of kinase inhibitors against a panel of kinases is proposed.  This has been illustrated using single-point inhibition data for 40 com. available kinase inhibitors screened against 85 kinases.  Nonselective inhibitors are characterized by Gini values close to zero (Staurosporine, Gini 0.150).  Highly selective compds. exhibit Gini values close to 1 (PD184352 Gini 0.905).  The relative selectivity of inhibitors does not depend on the ATP concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqUHpmF-C9u7Vg90H21EOLACvtfcHk0liSKex9ihwzAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFyhu7rL&md5=51f4902e49196dfef8e135c4a09182d7</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1021%2Fjm070562u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070562u%26sid%3Dliteratum%253Aachs%26aulast%3DGraczyk%26aufirst%3DP.%2BP.%26atitle%3DGini%2520coefficient%253A%2520a%2520new%2520way%2520to%2520express%2520selectivity%2520of%2520kinase%2520inhibitors%2520against%2520a%2520family%2520of%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D5773%26epage%3D5779%26doi%3D10.1021%2Fjm070562u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ali, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axtman, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuzzi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luecking, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saikatendu, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span> <span> </span><span class="NLM_article-title">Progress towards a public chemogenomic set for protein kinases and a call for contributions</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0181585</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0181585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1371%2Fjournal.pone.0181585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28767711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFWktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=D.+H.+Drewryauthor=C.+I.+Wellsauthor=D.+M.+Andrewsauthor=R.+Angellauthor=H.+Al-Aliauthor=A.+D.+Axtmanauthor=S.+J.+Capuzziauthor=J.+M.+Elkinsauthor=P.+Ettmayerauthor=M.+Frederiksenauthor=O.+Gileadiauthor=N.+Grayauthor=A.+Hooperauthor=S.+Knappauthor=S.+Lauferauthor=U.+Lueckingauthor=M.+Michaelidesauthor=S.+Mullerauthor=E.+Muratovauthor=R.+A.+Dennyauthor=K.+S.+Saikatenduauthor=D.+K.+Treiberauthor=W.+J.+Zuercherauthor=T.+M.+Willson&title=Progress+towards+a+public+chemogenomic+set+for+protein+kinases+and+a+call+for+contributions&doi=10.1371%2Fjournal.pone.0181585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Progress towards a public chemogenomic set for protein kinases and a call for contributions</span></div><div class="casAuthors">Drewry, David H.; Wells, Carrow I.; Andrews, David M.; Angell, Richard; Al-Ali, Hassan; Axtman, Alison D.; Capuzzi, Stephen J.; Elkins, Jonathan M.; Ettmayer, Peter; Frederiksen, Mathias; Gileadi, Opher; Gray, Nathanael; Hooper, Alice; Knapp, Stefan; Laufer, Stefan; Luecking, Ulrich; Michaelides, Michael; Muller, Susanne; Muratov, Eugene; Aldrin, Denny R.; Saikatendu, Kumar S.; Treiber, Daniel K.; Zuercher, William J.; Willson, Timothy M.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e0181585/1-e0181585/20</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein kinases are highly tractable targets for drug discovery.  However, the biol. function and therapeutic potential of the majority of the 500+ human protein kinases remains unknown.  We have developed phys. and virtual collections of small mol. inhibitors, which we call chemogenomic sets, that are designed to inhibit the catalytic function of almost half the human protein kinases.  In this manuscript we share our progress towards generation of a comprehensive kinase chemogenomic set (KCGS), release kinome profiling data of a large inhibitor set (Published Kinase Inhibitor Set 2 (PKIS2)), and outline a process through which the community can openly collaborate to create a KCGS that probes the full complement of human protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpozOiNZl51XbVg90H21EOLACvtfcHk0liSKex9ihwzAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFWktL0%253D&md5=9c27ba5cd21da43d30ce485ffa0e1675</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0181585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0181585%26sid%3Dliteratum%253Aachs%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DAndrews%26aufirst%3DD.%2BM.%26aulast%3DAngell%26aufirst%3DR.%26aulast%3DAl-Ali%26aufirst%3DH.%26aulast%3DAxtman%26aufirst%3DA.%2BD.%26aulast%3DCapuzzi%26aufirst%3DS.%2BJ.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DHooper%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLaufer%26aufirst%3DS.%26aulast%3DLuecking%26aufirst%3DU.%26aulast%3DMichaelides%26aufirst%3DM.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DSaikatendu%26aufirst%3DK.%2BS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26atitle%3DProgress%2520towards%2520a%2520public%2520chemogenomic%2520set%2520for%2520protein%2520kinases%2520and%2520a%2520call%2520for%2520contributions%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0181585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strivelli, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phinney, D. G.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of miR-544 biogenesis disrupts adaptive responses to hypoxia by modulating ATM-mTOR signaling</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2267</span>– <span class="NLM_lpage">2276</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00265</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00265" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFOlsLvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2267-2276&author=C.+L.+Hagaauthor=S.+P.+Velagapudiauthor=J.+R.+Strivelliauthor=W.+Y.+Yangauthor=M.+D.+Disneyauthor=D.+G.+Phinney&title=Small+molecule+inhibition+of+miR-544+biogenesis+disrupts+adaptive+responses+to+hypoxia+by+modulating+ATM-mTOR+signaling&doi=10.1021%2Facschembio.5b00265"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibition of miR-544 Biogenesis Disrupts Adaptive Responses to Hypoxia by Modulating ATM-mTOR Signaling</span></div><div class="casAuthors">Haga, Christopher L.; Velagapudi, Sai Pradeep; Strivelli, Jacqueline R.; Yang, Wang-Yong; Disney, Matthew D.; Phinney, Donald G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2267-2276</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hypoxia induces a complex circuit of gene expression that drives tumor progression and increases drug resistance.  Defining these changes allows for an understanding of how hypoxia alters tumor biol. and informs design of lead therapeutics.  We probed the role of microRNA-544 (miR-544), which silences mammalian target of rapamycin (mTOR), in a hypoxic breast cancer model by using a small mol. (1) that selectively impedes the microRNA's biogenesis.  Application of 1 to hypoxic tumor cells selectively inhibited prodn. of the mature microRNA, sensitized cells to 5-fluorouracil, and derepressed mRNAs affected by miR-544 in cellulo and in vivo, including boosting mTOR expression.  Thus, small mol. inhibition of miR-544 reverses a tumor cell's physiol. response to hypoxia.  Importantly, 1 sensitized tumor cells to hypoxia-assocd. apoptosis at a 25-fold lower concn. than a 2'-O-Me RNA antagomir and was as selective.  Further, the apoptotic effect of 1 was suppressed by treatment of cell with rapamycin, a well-known inhibitor of the mTOR signaling pathway, illustrating the selectivity of the compd.  Thus, RNA-directed chem. probes, which could also serve as lead therapeutics, enable interrogation of complex cellular networks in cells and animals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrV19uI0H2fLVg90H21EOLACvtfcHk0liSKex9ihwzAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOlsLvE&md5=32efb5ba1d1712026f1d158be1390fbe</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00265%26sid%3Dliteratum%253Aachs%26aulast%3DHaga%26aufirst%3DC.%2BL.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DStrivelli%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DW.%2BY.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DPhinney%26aufirst%3DD.%2BG.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520miR-544%2520biogenesis%2520disrupts%2520adaptive%2520responses%2520to%2520hypoxia%2520by%2520modulating%2520ATM-mTOR%2520signaling%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D2267%26epage%3D2276%26doi%3D10.1021%2Facschembio.5b00265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenny, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montanari, C. A.</span></span> <span> </span><span class="NLM_article-title">Inflation of correlation in the pursuit of drug-likeness</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1007/s10822-012-9631-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1007%2Fs10822-012-9631-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=23306465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1SrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=1-13&author=P.+W.+Kennyauthor=C.+A.+Montanari&title=Inflation+of+correlation+in+the+pursuit+of+drug-likeness&doi=10.1007%2Fs10822-012-9631-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Inflation of correlation in the pursuit of drug-likeness</span></div><div class="casAuthors">Kenny, Peter W.; Montanari, Carlos A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Drug-likeness is a frequently invoked, although not always precisely defined, concept in drug discovery.  Opinions on drug-likeness are to a large extent shaped by the relationships that are obsd. between surrogate measures of drug-likeness (e.g. aq. soly.; permeability; pharmacol. promiscuity) and fundamental physicochem. properties (e.g. lipophilicity; mol. size).  This article draws on examples from the literature to highlight approaches to data anal. that exaggerate trends in data and the term correlation inflation is introduced in the context of drug discovery.  Averaging groups of data points prior to anal. is a common cause of correlation inflation and results from anal. of binned continuous data should always be treated with caution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreGcOTh1EJCrVg90H21EOLACvtfcHk0ljJfE8BaX-e4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1SrtbY%253D&md5=e8542e2402b11cfb2fd98ae8a5185f46</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9631-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9631-5%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DMontanari%26aufirst%3DC.%2BA.%26atitle%3DInflation%2520of%2520correlation%2520in%2520the%2520pursuit%2520of%2520drug-likeness%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D1%26epage%3D13%26doi%3D10.1007%2Fs10822-012-9631-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span> <span> </span><span class="NLM_article-title">Two decades under the influence of the Rule of Five and the changing properties of approved oral drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1714</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00686</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00686" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslait73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1701-1714&author=M.+D.+Shultz&title=Two+decades+under+the+influence+of+the+Rule+of+Five+and+the+changing+properties+of+approved+oral+drugs&doi=10.1021%2Facs.jmedchem.8b00686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Two Decades under the Influence of the Rule of Five and the Changing Properties of Approved Oral Drugs</span></div><div class="casAuthors">Shultz, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1701-1714</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two decades have passed since the rule of five ushered in the concept of "drug-like" properties.  Attempts to quantify, correlate, and categorize mols. based on Ro5 parameters evolved into the introduction of efficiency metrics with far reaching consequences in decision making by industry leaders and scientists seeking to discover new medicines.  Examn. of oral drug parameters approved before and after the original Ro5 anal. demonstrates that some parameters such as clogP and HBD remained const. while the cutoffs for parameters such as mol. wt. and HBA have increased substantially over the past 20 years.  The time dependent increase in the mol. wt. of oral drugs during the past 20 years provides compelling evidence to disprove the hypothesis that mol. wt. is a "drug-like" property.  This anal. does not validate parameters that have not changed as being "drug-like" but instead calls into question the entire hypothesis that "drug-like" properties exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnoxpQsv2kbVg90H21EOLACvtfcHk0ljJfE8BaX-e4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslait73K&md5=d3c8b1107133dc0b7b277d22fa029597</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00686%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26atitle%3DTwo%2520decades%2520under%2520the%2520influence%2520of%2520the%2520Rule%2520of%2520Five%2520and%2520the%2520changing%2520properties%2520of%2520approved%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1701%26epage%3D1714%26doi%3D10.1021%2Facs.jmedchem.8b00686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gueret, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adihou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemurell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span> <span> </span><span class="NLM_article-title">New modalities for challenging targets in drug discovery</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">10294</span>– <span class="NLM_lpage">10323</span>, <span class="refDoi"> DOI: 10.1002/anie.201611914</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1002%2Fanie.201611914" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28186380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC1c3ls1Wruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=10294-10323&author=E.+Valeurauthor=S.+M.+Gueretauthor=H.+Adihouauthor=R.+Gopalakrishnanauthor=M.+Lemurellauthor=H.+Waldmannauthor=T.+N.+Grossmannauthor=A.+T.+Plowright&title=New+modalities+for+challenging+targets+in+drug+discovery&doi=10.1002%2Fanie.201611914"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">New Modalities for Challenging Targets in Drug Discovery</span></div><div class="casAuthors">Valeur Eric; Gueret Stephanie M; Adihou Helene; Gopalakrishnan Ranganath; Lemurell Malin; Plowright Alleyn T; Gueret Stephanie M; Adihou Helene; Gopalakrishnan Ranganath; Waldmann Herbert; Waldmann Herbert; Grossmann Tom N; Grossmann Tom N</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">10294-10323</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Our ever-increasing understanding of biological systems is providing a range of exciting novel biological targets, whose modulation may enable novel therapeutic options for many diseases.  These targets include protein-protein and protein-nucleic acid interactions, which are, however, often refractory to classical small-molecule approaches.  Other types of molecules, or modalities, are therefore required to address these targets, which has led several academic research groups and pharmaceutical companies to increasingly use the concept of so-called "new modalities".  This Review defines for the first time the scope of this term, which includes novel peptidic scaffolds, oligonucleotides, hybrids, molecular conjugates, as well as new uses of classical small molecules.  We provide the most representative examples of these modalities to target large binding surface areas such as those found in protein-protein interactions and for biological processes at the center of cell regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQq7SOpoq1Cja91pGlMMoT5fW6udTcc2eZmq3xhi7V0DLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c3ls1Wruw%253D%253D&md5=569b3b4a194d0682b6c81603b29f9b8a</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611914&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611914%26sid%3Dliteratum%253Aachs%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DGueret%26aufirst%3DS.%2BM.%26aulast%3DAdihou%26aufirst%3DH.%26aulast%3DGopalakrishnan%26aufirst%3DR.%26aulast%3DLemurell%26aufirst%3DM.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26atitle%3DNew%2520modalities%2520for%2520challenging%2520targets%2520in%2520drug%2520discovery%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2017%26volume%3D56%26spage%3D10294%26epage%3D10323%26doi%3D10.1002%2Fanie.201611914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagorce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douguet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miteva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villoutreix, B. O.</span></span> <span> </span><span class="NLM_article-title">Computational analysis of calculated physicochemical and ADMET properties of protein-protein interaction inhibitors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">46277</span>, <span class="refDoi"> DOI: 10.1038/srep46277</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fsrep46277" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28397808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVenurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=46277&author=D.+Lagorceauthor=D.+Douguetauthor=M.+A.+Mitevaauthor=B.+O.+Villoutreix&title=Computational+analysis+of+calculated+physicochemical+and+ADMET+properties+of+protein-protein+interaction+inhibitors&doi=10.1038%2Fsrep46277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Computational analysis of calculated physicochemical and ADMET properties of protein-protein interaction inhibitors</span></div><div class="casAuthors">Lagorce, David; Douguet, Dominique; Miteva, Maria A.; Villoutreix, Bruno O.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">46277</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The modulation of PPIs by low mol. wt. chem. compds., particularly by orally bioavailable mols., would be very valuable in numerous disease indications.  However, it is known that PPI inhibitors (iPPIs) tend to have properties that are linked to poor Absorption, Distribution, Metab., Excretion and Toxicity (ADMET) and in some cases to poor clin. outcomes.  Previously reported in silico analyses of iPPIs have essentially focused on physicochem. properties but several other ADMET parameters would be important to assess.  In order to gain new insights into the ADMET properties of iPPIs, computations were carried out on eight datasets collected from several databases.  These datasets involve compds. targeting enzymes, GPCRs, ion channels, nuclear receptors, allosteric modulators, oral marketed drugs, oral natural product-derived marketed drugs and iPPIs.  Several trends are reported that should assist the design and optimization of future PPI inhibitors, either for drug discovery endeavors or for chem. biol. projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw3PmL5znBCLVg90H21EOLACvtfcHk0ljD4lgXcel48A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVenurk%253D&md5=162ddf6594d20c770c1016313182245c</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1038%2Fsrep46277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep46277%26sid%3Dliteratum%253Aachs%26aulast%3DLagorce%26aufirst%3DD.%26aulast%3DDouguet%26aufirst%3DD.%26aulast%3DMiteva%26aufirst%3DM.%2BA.%26aulast%3DVilloutreix%26aufirst%3DB.%2BO.%26atitle%3DComputational%2520analysis%2520of%2520calculated%2520physicochemical%2520and%2520ADMET%2520properties%2520of%2520protein-protein%2520interaction%2520inhibitors%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D46277%26doi%3D10.1038%2Fsrep46277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref121"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref121'); return false;" data-citation="" class="refNumLink">121</a></strong><div class="NLM_citation" id="cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanaba, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donlic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karloff, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forte, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargrove, A. E.</span></span> <span> </span><span class="NLM_article-title">R-BIND: an interactive database for exploring and developing RNA-targeted chemical probes</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2691</span>– <span class="NLM_lpage">2700</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFWlsbfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=2691-2700&author=B.+S.+Morganauthor=B.+G.+Sanabaauthor=A.+Donlicauthor=D.+B.+Karloffauthor=J.+E.+Forteauthor=Y.+Zhangauthor=A.+E.+Hargrove&title=R-BIND%3A+an+interactive+database+for+exploring+and+developing+RNA-targeted+chemical+probes&doi=10.1021%2Facschembio.9b00631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">R-BIND: An Interactive Database for Exploring and Developing RNA-Targeted Chemical Probes</span></div><div class="casAuthors">Morgan, Brittany S.; Sanaba, Bilva G.; Donlic, Anita; Karloff, Diane B.; Forte, Jordan E.; Zhang, Yuqi; Hargrove, Amanda E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2691-2700</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">While the opportunities available for targeting RNA with small mols. have been widely appreciated, the challenges assocd. with achieving specific RNA recognition in biol. systems have hindered progress and prevented many researchers from entering the field.  To facilitate the discovery of RNA-targeted chem. probes and their subsequent applications, the authors curated the RNA-targeted BIoactive ligaNd Database (R-BIND).  This collection contains an array of information on reported chem. probes that target nonrRNA and have biol. activity, and anal. has led to the discovery of RNA-privileged properties.  Herein, the authors developed an online platform to make this information freely available to the community, offering search options, a suite of tools for probe development, and an updated R-BIND data set with detailed exptl. information for each probe.  The authors repeated the previous cheminformatics anal. on the updated R-BIND list and found that the distinguishing physicochem., structural, and spatial properties remained unchanged, despite an almost 50% increase in the database size.  Further, the authors developed several user-friendly tools, including queries based on cheminformatic parameters, exptl. details, functional groups, and substructures.  In addn., a nearest neighbors algorithm can assess the similarity of user-uploaded mols. to R-BIND ligands.  These tools and resources can be used to design small mol. libraries, optimize lead ligands, or select targets, probes, assays, and control expts.  Chem. probes are crit. to the study and discovery of novel functions for RNA, and the authors expect this resource to greatly assist researchers in exploring and developing successful RNA-targeted probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPoJQXYLWkH7Vg90H21EOLACvtfcHk0ljD4lgXcel48A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFWlsbfL&md5=02d2d41ec793396243f2935514c6b572</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00631%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DB.%2BS.%26aulast%3DSanaba%26aufirst%3DB.%2BG.%26aulast%3DDonlic%26aufirst%3DA.%26aulast%3DKarloff%26aufirst%3DD.%2BB.%26aulast%3DForte%26aufirst%3DJ.%2BE.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DHargrove%26aufirst%3DA.%2BE.%26atitle%3DR-BIND%253A%2520an%2520interactive%2520database%2520for%2520exploring%2520and%2520developing%2520RNA-targeted%2520chemical%2520probes%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D2691%26epage%3D2700%26doi%3D10.1021%2Facschembio.9b00631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref122"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref122'); return false;" data-citation="" class="refNumLink">122</a></strong><div class="NLM_citation" id="cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrivastava, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gautam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassanali, M.</span></span> <span> </span><span class="NLM_article-title">DrugBank: a knowledgebase for drugs, drug actions and drug targets</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">D901</span>– <span class="NLM_lpage">D906</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1093%2Fnar%2Fgkm958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=18048412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVSrs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D901-D906&author=D.+S.+Wishartauthor=C.+Knoxauthor=A.+C.+Guoauthor=D.+Chengauthor=S.+Shrivastavaauthor=D.+Tzurauthor=B.+Gautamauthor=M.+Hassanali&title=DrugBank%3A+a+knowledgebase+for+drugs%2C+drug+actions+and+drug+targets&doi=10.1093%2Fnar%2Fgkm958"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank: a knowledgebase for drugs, drug actions and drug targets</span></div><div class="casAuthors">Wishart, David S.; Knox, Craig; Guo, An Chi; Cheng, Dean; Shrivastava, Savita; Tzur, Dan; Gautam, Bijaya; Hassanali, Murtaza</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D901-D906</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a richly annotated resource that combines detailed drug data with comprehensive drug target and drug action information.  Since its first release in 2006, DrugBank has been widely used to facilitate in silico drug target discovery, drug design, drug docking or screening, drug metab. prediction, drug interaction prediction and general pharmaceutical education.  The latest version of DrugBank (release 2.0) has been expanded significantly over the previous release.  With ∼4900 drug entries, it now contains 60% more FDA-approved small mol. and biotech drugs including 10% more exptl.' drugs.  Significantly, more protein target data has also been added to the database, with the latest version of DrugBank contg. three times as many non-redundant protein or drug target sequences as before (1565 vs. 524).  Each DrugCard entry now contains more than 100 data fields with half of the information being devoted to drug/chem. data and the other half devoted to pharmacol., pharmacogenomic and mol. biol. data.  A no. of new data fields, including food-drug interactions, drug-drug interactions and exptl. ADME data have been added in response to numerous user requests.  DrugBank has also significantly improved the power and simplicity of its structure query and text query searches.  DrugBank is available at http://www.drugbank.ca.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonqKjSWZe1ULVg90H21EOLACvtfcHk0ljD4lgXcel48A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVSrs7Y%253D&md5=f8f8ca15f248b8c8b9c6d3bd0e85eec8</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm958%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DShrivastava%26aufirst%3DS.%26aulast%3DTzur%26aufirst%3DD.%26aulast%3DGautam%26aufirst%3DB.%26aulast%3DHassanali%26aufirst%3DM.%26atitle%3DDrugBank%253A%2520a%2520knowledgebase%2520for%2520drugs%252C%2520drug%2520actions%2520and%2520drug%2520targets%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3DD901%26epage%3DD906%26doi%3D10.1093%2Fnar%2Fgkm958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref123"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref123'); return false;" data-citation="" class="refNumLink">123</a></strong><div class="NLM_citation" id="cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narjes, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span> <span> </span><span class="NLM_article-title">Emerging modes-of-action in drug discovery</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1550</span>– <span class="NLM_lpage">1568</span>, <span class="refDoi"> DOI: 10.1039/C9MD00263D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1039%2FC9MD00263D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31673315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1ersbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1550-1568&author=E.+Valeurauthor=F.+Narjesauthor=C.+Ottmannauthor=A.+T.+Plowright&title=Emerging+modes-of-action+in+drug+discovery&doi=10.1039%2FC9MD00263D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging modes-of-action in drug discovery</span></div><div class="casAuthors">Valeur, Eric; Narjes, Frank; Ottmann, Christian; Plowright, Alleyn T.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1550-1568</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">An increasing focus on complex biol. to cure diseases rather than merely treat symptoms has transformed how drug discovery can be approached.  Instead of activating or blocking protein function, a growing repertoire of drug modalities can be leveraged or engineered to hijack cellular processes, such as translational regulation or degrdn. mechanisms.  Drug hunters can therefore access a wider arsenal of modes-of-action to modulate biol. processes and this review summarises these emerging strategies by highlighting the most representative examples of these approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraPh2j99fLKrVg90H21EOLACvtfcHk0lhPWUt8wySciQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1ersbvJ&md5=396fd707054d3ecdaf51220b7b3c6dcb</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1039%2FC9MD00263D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00263D%26sid%3Dliteratum%253Aachs%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DNarjes%26aufirst%3DF.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26atitle%3DEmerging%2520modes-of-action%2520in%2520drug%2520discovery%26jtitle%3DMedChemComm%26date%3D2019%26volume%3D10%26spage%3D1550%26epage%3D1568%26doi%3D10.1039%2FC9MD00263D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref124"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref124'); return false;" data-citation="" class="refNumLink">124</a></strong><div class="NLM_citation" id="cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span> <span> </span><span class="NLM_article-title">Lead- and drug-like compounds: the rule-of-five revolution</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2004.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.ddtec.2004.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=24981612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=337-341&author=C.+A.+Lipinski&title=Lead-+and+drug-like+compounds%3A+the+rule-of-five+revolution&doi=10.1016%2Fj.ddtec.2004.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">Lead- and drug-like compounds: the rule-of-five revolution</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-341</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Citations in CAS SciFinder to the rule-of-five (RO5) publication will exceed 1000 by year-end 2004.  Trends in the RO5 literature explosion that can be discerned are the further definitions of drug-like.  This topic is explored in terms of drug-like physicochem. features, drug-like structural features, a comparison of drug-like and non-drug-like in drug discovery and a discussion of how drug-like features relate to clin. success.  Physicochem. features of CNS drugs and features related to CNS blood-brain transporter affinity are briefly reviewed.  Recent literature on features of non-oral drugs is reviewed and how features of lead-like compds. differ from those of drug-like compds. is discussed.  Most recently, partly driven by NIH roadmap initiatives, considerations have arisen as to what tool-like means in the search for chem. tools to probe biol. space.  All these topics frame the scope of this short review/perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnjiyP3CY7lLVg90H21EOLACvtfcHk0lhPWUt8wySciQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D&md5=952c29b47003b884116d790ae9e4721f</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DLead-%2520and%2520drug-like%2520compounds%253A%2520the%2520rule-of-five%2520revolution%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D337%26epage%3D341%26doi%3D10.1016%2Fj.ddtec.2004.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref125"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref125'); return false;" data-citation="" class="refNumLink">125</a></strong><div class="NLM_citation" id="cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-induced protein degradation in drug discovery: Breaking the rules or just making new ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-induced+protein+degradation+in+drug+discovery%3A+Breaking+the+rules+or+just+making+new+ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0lhPWUt8wySciQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-induced%2520protein%2520degradation%2520in%2520drug%2520discovery%253A%2520Breaking%2520the%2520rules%2520or%2520just%2520making%2520new%2520ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref126"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref126'); return false;" data-citation="" class="refNumLink">126</a></strong><div class="NLM_citation" id="cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udugamasooriya, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">Potent and selective photo-inactivation of proteins with peptoid-ruthenium conjugates</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">258</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1038/nchembio.333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnchembio.333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=20228793" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFynuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=258-260&author=J.+Leeauthor=D.+G.+Udugamasooriyaauthor=H.-S.+Limauthor=T.+Kodadek&title=Potent+and+selective+photo-inactivation+of+proteins+with+peptoid-ruthenium+conjugates&doi=10.1038%2Fnchembio.333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective photo-inactivation of proteins with peptoid-ruthenium conjugates</span></div><div class="casAuthors">Lee, Jiyong; Udugamasooriya, D. Gomika; Lim, Hyun-Suk; Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">258-260</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Advances in high-throughput screening now enable the rapid discovery of bioactive small mols., but these primary hits almost always exhibit modest potency.  We report a strategy for the transformation of these hits into much more potent inhibitors without compd. optimization.  Appending a deriv. of Ru(II)(tris-bipyridyl)2+, an efficient photosensitizer of singlet oxygen prodn., to synthetic protein-binding compds. results in highly potent and specific target protein inactivation upon irradn. with visible light.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLrAtKe0wa1bVg90H21EOLACvtfcHk0lhPWUt8wySciQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFynuro%253D&md5=b88e635815cbfa3fdd2ba94142352a71</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.333%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DUdugamasooriya%26aufirst%3DD.%2BG.%26aulast%3DLim%26aufirst%3DH.-S.%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DPotent%2520and%2520selective%2520photo-inactivation%2520of%2520proteins%2520with%2520peptoid-ruthenium%2520conjugates%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D258%26epage%3D260%26doi%3D10.1038%2Fnchembio.333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref127"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref127'); return false;" data-citation="" class="refNumLink">127</a></strong><div class="NLM_citation" id="cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Small molecule targeted recruitment of a nuclease to RNA</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">6741</span>– <span class="NLM_lpage">6744</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b01233</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b01233" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpvFKktbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=6741-6744&author=M.+G.+Costalesauthor=Y.+Matsumotoauthor=S.+P.+Velagapudiauthor=M.+D.+Disney&title=Small+molecule+targeted+recruitment+of+a+nuclease+to+RNA&doi=10.1021%2Fjacs.8b01233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Targeted Recruitment of a Nuclease to RNA</span></div><div class="casAuthors">Costales, Matthew G.; Matsumoto, Yasumasa; Velagapudi, Sai Pradeep; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6741-6744</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The choreog. between RNA synthesis and degrdn. is a key determinant in biol.  Engineered systems such as CRISPR have been developed to rid a cell of RNAs.  Here, we show that a small mol. can recruit a nuclease to a specific transcript, triggering its destruction.  A small mol. that selectively binds the oncogenic microRNA(miR)-96 hairpin precursor was appended with a short 2'-5' poly(A) oligonucleotide.  The conjugate locally activated endogenous, latent RNase (RNase L), which selectively cleaved the miR-96 precursor in cancer cells in a catalytic fashion.  Silencing miR-96 de-repressed pro-apoptotic FOXO1 transcription factor, triggering apoptosis in breast cancer, but not healthy breast, cells.  These results demonstrate that small mols. can be programmed to selectively cleave RNA and has broad implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBE-TmSLM__LVg90H21EOLACvtfcHk0livjj4c-GKtKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpvFKktbs%253D&md5=7d4ccaad0cba3da84f924f244b8bcdbf</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b01233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b01233%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DMatsumoto%26aufirst%3DY.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSmall%2520molecule%2520targeted%2520recruitment%2520of%2520a%2520nuclease%2520to%2520RNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D6741%26epage%3D6744%26doi%3D10.1021%2Fjacs.8b01233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref128"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref128'); return false;" data-citation="" class="refNumLink">128</a></strong><div class="NLM_citation" id="cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vauquelin, G.</span></span> <span> </span><span class="NLM_article-title">Effects of target binding kinetics on in vivo drug efficacy: k<sub>off</sub>, k<sub>on</sub> and rebinding</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">2319</span>– <span class="NLM_lpage">2334</span>, <span class="refDoi"> DOI: 10.1111/bph.13504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1111%2Fbph.13504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27129075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSiurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2016&pages=2319-2334&author=G.+Vauquelin&title=Effects+of+target+binding+kinetics+on+in+vivo+drug+efficacy%3A+koff%2C+kon+and+rebinding&doi=10.1111%2Fbph.13504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of target binding kinetics on in vivo drug efficacy: koff, kon and rebinding</span></div><div class="casAuthors">Vauquelin, Georges</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2319-2334</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Background and Purpose : Optimal drug therapy often requires continuing high levels of target occupancy.  Besides the traditional pharmacokinetic contribution, target binding kinetics is increasingly considered to play an important role as well.  While most attention has been focused on the dissocn. rate of the complex, recent reports expressed doubt about the unreserved translatability of this pharmacodynamic property into clin. efficacy. 'Micro'-pharmacokinetic mechanisms like drug rebinding and partitioning into the cell membrane may constitute a potential fix.  Exptl. Approach : Simulations were based on solving differential equations.  Key Results : Based on a selected range of assocn. and dissocn. rate consts., kon and koff, and rebinding potencies of the drugs as variables, their effects on the temporal in vivo occupancy profile of their targets, after one or multiple repetitive dosings, have here been simulated.  Conclusions and Implications : Most strikingly, the simulations show that, when rebinding is also taken into account, increasing kon may produce closely the same outcome as decreasing koff when dosing is performed in accordance with the therapeutically most relevant const. [Lmax]/KD ratio paradigm.  Also, under certain conditions, rebinding may produce closely the same outcome as invoking slow diffusion of the drug between the plasma compartment and a target-contg. 'effect' compartment.  Although the present simulations should only be regarded as a 'proof of principle', these findings may help pharmacologists and medicinal chemists to devise ex vivo and in vitro binding kinetic assays that are more relevant and translatable to in vivo settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5-dbZNJetv7Vg90H21EOLACvtfcHk0livjj4c-GKtKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSiurvI&md5=5d1b13ef5786a0c661118ce9be503d50</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1111%2Fbph.13504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.13504%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26atitle%3DEffects%2520of%2520target%2520binding%2520kinetics%2520on%2520in%2520vivo%2520drug%2520efficacy%253A%2520koff%252C%2520kon%2520and%2520rebinding%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2016%26volume%3D173%26spage%3D2319%26epage%3D2334%26doi%3D10.1111%2Fbph.13504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref129"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref129'); return false;" data-citation="" class="refNumLink">129</a></strong><div class="NLM_citation" id="cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0livjj4c-GKtKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref130"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref130'); return false;" data-citation="" class="refNumLink">130</a></strong><div class="NLM_citation" id="cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">SPR-measured dissociation kinetics of PROTAC ternary complexes influence target degradation rate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=361-368&author=M.+J.+Royauthor=S.+Winklerauthor=S.+J.+Hughesauthor=C.+Whitworthauthor=M.+Galantauthor=W.+Farnabyauthor=K.+Rumpelauthor=A.+Ciulli&title=SPR-measured+dissociation+kinetics+of+PROTAC+ternary+complexes+influence+target+degradation+rate&doi=10.1021%2Facschembio.9b00092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">SPR-Measured Dissociation Kinetics of PROTAC Ternary Complexes Influence Target Degradation Rate</span></div><div class="casAuthors">Roy, Michael J.; Winkler, Sandra; Hughes, Scott J.; Whitworth, Claire; Galant, Michael; Farnaby, William; Rumpel, Klaus; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-368</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bifunctional degrader mols., known as proteolysis-targeting chimeras (PROTACs), function by recruiting a target to an E3 ligase, forming a target/PROTAC/ligase ternary complex.  Despite the importance of this key intermediate species, no detailed validation of a method to directly det. binding parameters for ternary complex kinetics has been reported, and it remains to be addressed whether tuning the kinetics of PROTAC ternary complexes may be an effective strategy to improve the efficiency of targeted protein degrdn.  Here, we develop an SPR-based assay to quantify the stability of PROTAC-induced ternary complexes by measuring for the first time the kinetics of their formation and dissocn. in vitro using purified proteins.  We benchmark our assay using four PROTACs that target the bromodomains (BDs) of bromodomain and extraterminal domain proteins Brd2, Brd3, and Brd4 to the von Hippel-Lindau E3 ligase (VHL).  We reveal marked differences in ternary complex off-rates for different PROTACs that exhibit either pos. or neg. cooperativity for ternary complex formation relative to binary binding.  The pos. cooperative degrader MZ1 forms comparatively stable and long-lived ternary complexes with either Brd4BD2 or Brd2BD2 and VHL.  Equivalent complexes with Brd3BD2 are destabilized due to a single amino acid difference (Glu/Gly swap) present in the bromodomain.  We observe that this difference in ternary complex dissociative half-life correlates to a greater initial rate of intracellular degrdn. of Brd2 and Brd4 relative to Brd3.  These findings establish a novel assay to measure the kinetics of PROTAC ternary complexes and elucidate the important kinetic parameters that drive effective target degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFhvuKYztcbVg90H21EOLACvtfcHk0ljN4YBWT2nKgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D&md5=629d1acbd218b668a2696b727946962b</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00092%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWinkler%26aufirst%3DS.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSPR-measured%2520dissociation%2520kinetics%2520of%2520PROTAC%2520ternary%2520complexes%2520influence%2520target%2520degradation%2520rate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D361%26epage%3D368%26doi%3D10.1021%2Facschembio.9b00092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref131"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref131'); return false;" data-citation="" class="refNumLink">131</a></strong><div class="NLM_citation" id="cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">The drug-target residence time model: a 10-year retrospective</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">95</span>, <span class="refDoi"> DOI: 10.1038/nrd.2015.18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrd.2015.18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=26678621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVCrurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=87-95&author=R.+A.+Copeland&title=The+drug-target+residence+time+model%3A+a+10-year+retrospective&doi=10.1038%2Fnrd.2015.18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">The drug-target residence time model: a 10-year retrospective</span></div><div class="casAuthors">Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">87-95</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The drug-target residence time model was first introduced in 2006 and has been broadly adopted across the chem. biol., biotechnol. and pharmaceutical communities.  While traditional in vitro methods view drug-target interactions exclusively in terms of equil. affinity, the residence time model takes into account the conformational dynamics of target macromols. that affect drug binding and dissocn.  The key tenet of this model is that the lifetime (or residence time) of the binary drug-target complex, and not the binding affinity per se, dictates much of the in vivo pharmacol. activity.  Here, this model is revisited and key applications of it over the past 10 years are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwNrhp9U8kwLVg90H21EOLACvtfcHk0ljN4YBWT2nKgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVCrurnE&md5=23f4a6973d77804292ff0af6c81358c0</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2015.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2015.18%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DThe%2520drug-target%2520residence%2520time%2520model%253A%2520a%252010-year%2520retrospective%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D87%26epage%3D95%26doi%3D10.1038%2Fnrd.2015.18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref132"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref132'); return false;" data-citation="" class="refNumLink">132</a></strong><div class="NLM_citation" id="cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeep
Velagapudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yildirim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Small-molecule targeted recruitment of a nuclease to cleave an oncogenic RNA in a mouse model of metastatic cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">2406</span>– <span class="NLM_lpage">2411</span>, <span class="refDoi"> DOI: 10.1073/pnas.1914286117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1914286117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31964809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVWisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=2406-2411&author=M.+G.+Costalesauthor=H.+Aikawaauthor=Y.+Liauthor=J.+L.+Childs-Disneyauthor=D.+Abeggauthor=D.+G.+Hochauthor=S.+Pradeep%0AVelagapudiauthor=Y.+Nakaiauthor=T.+Khanauthor=K.+W.+Wangauthor=I.+Yildirimauthor=A.+Adibekianauthor=E.+T.+Wangauthor=M.+D.+Disney&title=Small-molecule+targeted+recruitment+of+a+nuclease+to+cleave+an+oncogenic+RNA+in+a+mouse+model+of+metastatic+cancer&doi=10.1073%2Fpnas.1914286117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule targeted recruitment of a nuclease to cleave an oncogenic RNA in a mouse model of metastatic cancer</span></div><div class="casAuthors">Costales, Matthew G.; Aikawa, Haruo; Li, Yue; Childs-Disney, Jessica L.; Abegg, Daniel; Hoch, Dominic G.; Velagapudi, Sai Pradeep; Nakai, Yoshio; Khan, Tanya; Wang, Kye Won; Yildirim, Ilyas; Adibekian, Alexander; Wang, Eric T.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2406-2411</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the area of small mols. interacting with RNA advances, general routes to provide bioactive compds. are needed as ligands can bind RNA avidly to sites that will not affect function.  Small-mol. targeted RNA degrdn. will thus provide a general route to affect RNA biol.  A non-oligonucleotide-contg. compd. was designed from sequence to target the precursor to oncogenic microRNA-21 (pre-miR-21) for enzymic destruction with selectivity that can exceed that for protein-targeted medicines.  The compd. specifically binds the target and contains a heterocycle that recruits and activates a RNase to pre-miR-21 to substoichiometrically effect its cleavage and subsequently impede metastasis of breast cancer to lung in a mouse model.  Transcriptomic and proteomic analyses demonstrate that the compd. is potent and selective, specifically modulating oncogenic pathways.  Thus, small mols. can be designed from sequence to have all of the functional repertoire of oligonucleotides, including inducing enzymic degrdn., and to selectively and potently modulate RNA function in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolzufxJ0-LLLVg90H21EOLACvtfcHk0ljN4YBWT2nKgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVWisL0%253D&md5=450996be5aadaa9e13acb5c2cfb03c62</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1914286117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1914286117%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DAikawa%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DPradeep%2BVelagapudi%26aufirst%3DS.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DKhan%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DK.%2BW.%26aulast%3DYildirim%26aufirst%3DI.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DE.%2BT.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSmall-molecule%2520targeted%2520recruitment%2520of%2520a%2520nuclease%2520to%2520cleave%2520an%2520oncogenic%2520RNA%2520in%2520a%2520mouse%2520model%2520of%2520metastatic%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D2406%26epage%3D2411%26doi%3D10.1073%2Fpnas.1914286117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref133"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref133'); return false;" data-citation="" class="refNumLink">133</a></strong><div class="NLM_citation" id="cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M.</span></span> <span> </span><span class="NLM_article-title">Target 2035: probing the human proteome</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2111</span>– <span class="NLM_lpage">2115</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2019.06.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.drudis.2019.06.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31278990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlahtrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=2111-2115&issue=11&author=A.+J.+Carterauthor=O.+Kraemerauthor=M.+Zwickauthor=A.+Mueller-Fahrnowauthor=C.+H.+Arrowsmithauthor=A.+M.+Edwards&title=Target+2035%3A+probing+the+human+proteome&doi=10.1016%2Fj.drudis.2019.06.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Target 2035: probing the human proteome</span></div><div class="casAuthors">Carter, Adrian J.; Kraemer, Oliver; Zwick, Matthias; Mueller-Fahrnow, Anke; Arrowsmith, Cheryl H.; Edwards, Aled M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2111-2115</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Biomedical scientists tend to focus on only a small fraction of the proteins encoded by the human genome despite overwhelming genetic evidence that many understudied proteins are important for human disease.  One of the best ways to interrogate the function of a protein and to det. its relevance as a drug target is by using a pharmacol. modulator, such as a chem. probe or an antibody.  If these tools were available for most human proteins, it should be possible to translate the tremendous advances in genomics into a greater understanding of human health and disease, and catalyze the creation of innovative new medicines.  Target 2035 is a global federation for developing and applying new technologies with the goal of creating chemogenomic libraries, chem. probes, and/or functional antibodies for the entire proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSY5b9ZXh0ALVg90H21EOLACvtfcHk0lj37-hKNKK6pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlahtrnN&md5=212e832238c8860439b343cbeef17b74</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2019.06.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2019.06.020%26sid%3Dliteratum%253Aachs%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DKraemer%26aufirst%3DO.%26aulast%3DZwick%26aufirst%3DM.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26atitle%3DTarget%25202035%253A%2520probing%2520the%2520human%2520proteome%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26issue%3D11%26spage%3D2111%26epage%3D2115%26doi%3D10.1016%2Fj.drudis.2019.06.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref134"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref134'); return false;" data-citation="" class="refNumLink">134</a></strong><div class="NLM_citation" id="cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kehe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulesa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackerman, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakku, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuehn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gore, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blainey, P. C.</span></span> <span> </span><span class="NLM_article-title">Massively parallel screening of synthetic microbial communities</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">12804</span>– <span class="NLM_lpage">12809</span>, <span class="refDoi"> DOI: 10.1073/pnas.1900102116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1900102116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31186361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1ersrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=12804-12809&author=J.+Keheauthor=A.+Kulesaauthor=A.+Ortizauthor=C.+M.+Ackermanauthor=S.+G.+Thakkuauthor=D.+Sellersauthor=S.+Kuehnauthor=J.+Goreauthor=J.+Friedmanauthor=P.+C.+Blainey&title=Massively+parallel+screening+of+synthetic+microbial+communities&doi=10.1073%2Fpnas.1900102116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Massively parallel screening of synthetic microbial communities</span></div><div class="casAuthors">Kehe, Jared; Kulesa, Anthony; Ortiz, Anthony; Ackerman, Cheri M.; Thakku, Sri Gowtham; Sellers, Daniel; Kuehn, Seppe; Gore, Jeff; Friedman, Jonathan; Blainey, Paul C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">12804-12809</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Microbial communities have numerous potential applications in biotechnol., agriculture, and medicine.  Nevertheless, the limited accuracy with which we can predict interspecies interactions and environmental dependencies hinders efforts to rationally engineer beneficial consortia.  Empirical screening is a complementary approach wherein synthetic communities are combinatorially constructed and assayed in high throughput.  However, assembling many combinations of microbes is logistically complex and difficult to achieve on a timescale commensurate with microbial growth.  Here, we introduce the kChip, a droplets-based platform that performs rapid, massively parallel, bottom-up construction and screening of synthetic microbial communities.  We first show that the kChip enables phenotypic characterization of microbes across environmental conditions.  Next, in a screen of a$$Hat$$ 1/4100,000 multispecies communities comprising up to 19 soil isolates, we identified sets that promote the growth of the model plant symbiont Herbaspirillum frisingense in a manner robust to carbon source variation and the presence of addnl. species.  Broadly, kChip screening can identify multispecies consortia possessing any optically assayable function, including facilitation of biocontrol agents, suppression of pathogens, degrdn. of recalcitrant substrates, and robustness of these functions to perturbation, with many applications across basic and applied microbial ecol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNU77NEWb3TrVg90H21EOLACvtfcHk0lj37-hKNKK6pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1ersrnF&md5=38e41c9eed2b1246c90ee5c81f1fc47e</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1900102116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1900102116%26sid%3Dliteratum%253Aachs%26aulast%3DKehe%26aufirst%3DJ.%26aulast%3DKulesa%26aufirst%3DA.%26aulast%3DOrtiz%26aufirst%3DA.%26aulast%3DAckerman%26aufirst%3DC.%2BM.%26aulast%3DThakku%26aufirst%3DS.%2BG.%26aulast%3DSellers%26aufirst%3DD.%26aulast%3DKuehn%26aufirst%3DS.%26aulast%3DGore%26aufirst%3DJ.%26aulast%3DFriedman%26aufirst%3DJ.%26aulast%3DBlainey%26aufirst%3DP.%2BC.%26atitle%3DMassively%2520parallel%2520screening%2520of%2520synthetic%2520microbial%2520communities%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D12804%26epage%3D12809%26doi%3D10.1073%2Fpnas.1900102116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref135"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref135'); return false;" data-citation="" class="refNumLink">135</a></strong><div class="NLM_citation" id="cit135"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">DNA-Encoded Chemical Libraries: A selection system based on endowing organic compounds with amplifiable information</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">479</span>– <span class="NLM_lpage">502</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-062917-012550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1146%2Fannurev-biochem-062917-012550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29328784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVCrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2018&pages=479-502&author=D.+Neriauthor=R.+A.+Lerner&title=DNA-Encoded+Chemical+Libraries%3A+A+selection+system+based+on+endowing+organic+compounds+with+amplifiable+information&doi=10.1146%2Fannurev-biochem-062917-012550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Chemical Libraries: A Selection System Based on Endowing Organic Compounds with Amplifiable Information</span></div><div class="casAuthors">Neri, Dario; Lerner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">479-502</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The discovery of org. ligands that bind specifically to proteins is a central problem in chem., biol., and the biomedical sciences.  The encoding of individual org. mols. with distinctive DNA tags, serving as amplifiable identification bar codes, allows the construction and screening of combinatorial libraries of unprecedented size, thus facilitating the discovery of ligands to many different protein targets.  Fundamentally, one links powers of genetics and chem. synthesis.  After the initial description of DNA-encoded chem. libraries in 1992, several exptl. embodiments of the technol. have been reduced to practice.  This review provides a historical account of important milestones in the development of DNA-encoded chem. libraries, a survey of relevant ongoing research activities, and a glimpse into the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAAjUib3HfEbVg90H21EOLACvtfcHk0lj37-hKNKK6pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVCrsg%253D%253D&md5=559fefad70a7b9949319ac8ad8066004</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-062917-012550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-062917-012550%26sid%3Dliteratum%253Aachs%26aulast%3DNeri%26aufirst%3DD.%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DDNA-Encoded%2520Chemical%2520Libraries%253A%2520A%2520selection%2520system%2520based%2520on%2520endowing%2520organic%2520compounds%2520with%2520amplifiable%2520information%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2018%26volume%3D87%26spage%3D479%26epage%3D502%26doi%3D10.1146%2Fannurev-biochem-062917-012550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120'],'ref121':['cit121'],'ref122':['cit122'],'ref123':['cit123'],'ref124':['cit124'],'ref125':['cit125'],'ref126':['cit126'],'ref127':['cit127'],'ref128':['cit128'],'ref129':['cit129'],'ref130':['cit130'],'ref131':['cit131'],'ref132':['cit132'],'ref133':['cit133'],'ref134':['cit134'],'ref135':['cit135']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 29 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jessica A. Bush, Christopher C. Williams, Samantha M. Meyer, Yuquan Tong, Hafeez S. Haniff, Jessica L. Childs-Disney, <span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Disney</span>. </span><span class="cited-content_cbyCitation_article-title">Systematically Studying the Effect of Small Molecules Interacting with RNA in Cellular and Preclinical Models. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (7)
                                     , 1111-1127. <a href="https://doi.org/10.1021/acschembio.1c00014" title="DOI URL">https://doi.org/10.1021/acschembio.1c00014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.1c00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.1c00014%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DSystematically%252BStudying%252Bthe%252BEffect%252Bof%252BSmall%252BMolecules%252BInteracting%252Bwith%252BRNA%252Bin%252BCellular%252Band%252BPreclinical%252BModels%26aulast%3DBush%26aufirst%3DJessica%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D06012021%26date%3D07062021%26date%3D24062021%26volume%3D16%26issue%3D7%26spage%3D1111%26epage%3D1127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shiying Mao, Chengyu Wu, Yanqing Gao, Jin Hao, Xiaohua He, Pan Tao, Jian Li, Shibin Shang, Zhanqian Song, <span class="NLM_string-name hlFld-ContribAuthor">Jie Song</span>. </span><span class="cited-content_cbyCitation_article-title">Pine Rosin as a Valuable Natural Resource in the Synthesis of Fungicide Candidates for Controlling Fusarium oxysporum on Cucumber. </span><span class="cited-content_cbyCitation_journal-name">Journal of Agricultural and Food Chemistry</span><span> <strong>2021,</strong> <em>69 </em>
                                    (23)
                                     , 6475-6484. <a href="https://doi.org/10.1021/acs.jafc.1c01887" title="DOI URL">https://doi.org/10.1021/acs.jafc.1c01887</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jafc.1c01887&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jafc.1c01887%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Agricultural%2520and%2520Food%2520Chemistry%26atitle%3DPine%252BRosin%252Bas%252Ba%252BValuable%252BNatural%252BResource%252Bin%252Bthe%252BSynthesis%252Bof%252BFungicide%252BCandidates%252Bfor%252BControlling%252BFusarium%252Boxysporum%252Bon%252BCucumber%26aulast%3DMao%26aufirst%3DShiying%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D01042021%26date%3D21052021%26date%3D18052021%26date%3D02062021%26volume%3D69%26issue%3D23%26spage%3D6475%26epage%3D6484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Farzad Zamani, <span class="NLM_string-name hlFld-ContribAuthor">Takayoshi Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Synthetic RNA Modulators in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (11)
                                     , 7110-7155. <a href="https://doi.org/10.1021/acs.jmedchem.1c00154" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00154</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00154%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthetic%252BRNA%252BModulators%252Bin%252BDrug%252BDiscovery%26aulast%3DZamani%26aufirst%3DFarzad%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D27012021%26date%3D01062021%26volume%3D64%26issue%3D11%26spage%3D7110%26epage%3D7155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hasane Ratni, Renata S. Scalco, <span class="NLM_string-name hlFld-ContribAuthor">Alexander H. Stephan</span>. </span><span class="cited-content_cbyCitation_article-title">Risdiplam, the First Approved Small Molecule Splicing Modifier Drug as a Blueprint for Future Transformative Medicines. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (6)
                                     , 874-877. <a href="https://doi.org/10.1021/acsmedchemlett.0c00659" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00659</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00659%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DRisdiplam%25252C%252Bthe%252BFirst%252BApproved%252BSmall%252BMolecule%252BSplicing%252BModifier%252BDrug%252Bas%252Ba%252BBlueprint%252Bfor%252BFuture%252BTransformative%252BMedicines%26aulast%3DRatni%26aufirst%3DHasane%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D18122020%26date%3D19012021%26date%3D28012021%26volume%3D12%26issue%3D6%26spage%3D874%26epage%3D877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Andrew W. Robertson, Jorge Sandoval, Osama G. Mohamed, Yihao Zhuang, Erin E. Gallagher, Jennifer Schmidt, Lisa Caratelli, Arya Menon, Pamela J. Schultz, Rachel M. Torrez, Catherine L. Hay, Bailey A. Bell, Paul A. Price, Amanda L. Garner, <span class="NLM_string-name hlFld-ContribAuthor">Ashootosh Tripathi</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Surfactins as Inhibitors of MicroRNA Processing Using Cat-ELCCA. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (6)
                                     , 878-886. <a href="https://doi.org/10.1021/acsmedchemlett.1c00046" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00046</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00046%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BSurfactins%252Bas%252BInhibitors%252Bof%252BMicroRNA%252BProcessing%252BUsing%252BCat-ELCCA%26aulast%3DRobertson%26aufirst%3DAndrew%2BW.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D22012021%26date%3D22032021%26date%3D02042021%26volume%3D12%26issue%3D6%26spage%3D878%26epage%3D886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sandro Bottaro, Giovanni Bussi, <span class="NLM_string-name hlFld-ContribAuthor">Kresten Lindorff-Larsen</span>. </span><span class="cited-content_cbyCitation_article-title">Conformational Ensembles of Noncoding Elements in the SARS-CoV-2 Genome from Molecular Dynamics Simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (22)
                                     , 8333-8343. <a href="https://doi.org/10.1021/jacs.1c01094" title="DOI URL">https://doi.org/10.1021/jacs.1c01094</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.1c01094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.1c01094%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DConformational%252BEnsembles%252Bof%252BNoncoding%252BElements%252Bin%252Bthe%252BSARS-CoV-2%252BGenome%252Bfrom%252BMolecular%252BDynamics%252BSimulations%26aulast%3DBottaro%26aufirst%3DSandro%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D28012021%26date%3D27052021%26volume%3D143%26issue%3D22%26spage%3D8333%26epage%3D8343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Amanda L. Garner, <span class="NLM_string-name hlFld-ContribAuthor">Stevan W. Djuric</span>. </span><span class="cited-content_cbyCitation_article-title">RNA: Opening New Doors in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (9)
                                     , 1659-1660. <a href="https://doi.org/10.1021/acsmedchemlett.0c00421" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00421</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00421%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DRNA%25253A%252BOpening%252BNew%252BDoors%252Bin%252BMedicinal%252BChemistry%26aulast%3DGarner%26aufirst%3DAmanda%2BL.%26date%3D2020%26date%3D2020%26date%3D19082020%26volume%3D11%26issue%3D9%26spage%3D1659%26epage%3D1660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mattias  Bood</span>, <span class="hlFld-ContribAuthor ">Anna Wypijewska  del Nogal</span>, <span class="hlFld-ContribAuthor ">Jesper R.  Nilsson</span>, <span class="hlFld-ContribAuthor ">Fredrik  Edfeldt</span>, <span class="hlFld-ContribAuthor ">Anders  Dahlén</span>, <span class="hlFld-ContribAuthor ">Malin  Lemurell</span>, <span class="hlFld-ContribAuthor ">L. Marcus  Wilhelmsson</span>, <span class="hlFld-ContribAuthor ">Morten  Grøtli</span>. </span><span class="cited-content_cbyCitation_article-title">Interbase-FRET binding assay for pre-microRNAs. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-021-88922-0" title="DOI URL">https://doi.org/10.1038/s41598-021-88922-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-021-88922-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-021-88922-0%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DInterbase-FRET%252Bbinding%252Bassay%252Bfor%252Bpre-microRNAs%26aulast%3DBood%26aufirst%3DMattias%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huachun  Liu</span>, <span class="hlFld-ContribAuthor ">Simone  Rauch</span>, <span class="hlFld-ContribAuthor ">Bryan C.  Dickinson</span>. </span><span class="cited-content_cbyCitation_article-title">Programmable technologies to manipulate gene expression at the RNA level. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>64 </em>, 27-37. <a href="https://doi.org/10.1016/j.cbpa.2021.03.004" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.03.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.03.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DProgrammable%252Btechnologies%252Bto%252Bmanipulate%252Bgene%252Bexpression%252Bat%252Bthe%252BRNA%252Blevel%26aulast%3DLiu%26aufirst%3DHuachun%26date%3D2021%26volume%3D64%26spage%3D27%26epage%3D37" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lluís  Guijarro</span>, <span class="hlFld-ContribAuthor ">Álvaro  Martínez-Camarena</span>, <span class="hlFld-ContribAuthor ">Javier U.  Chicote</span>, <span class="hlFld-ContribAuthor ">Antonio  García-España</span>, <span class="hlFld-ContribAuthor ">Enrique  García-España</span>, <span class="hlFld-ContribAuthor ">Mario  Inclán</span>, <span class="hlFld-ContribAuthor ">Begoña  Verdejo</span>, <span class="hlFld-ContribAuthor ">Jorge  González-García</span>. </span><span class="cited-content_cbyCitation_article-title">Ditopic Aza-Scorpiand Ligands Interact Selectively with ds-RNA and Modulate the Interaction upon Formation of Zn2+ Complexes. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (13)
                                     , 3957. <a href="https://doi.org/10.3390/molecules26133957" title="DOI URL">https://doi.org/10.3390/molecules26133957</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26133957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26133957%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DDitopic%252BAza-Scorpiand%252BLigands%252BInteract%252BSelectively%252Bwith%252Bds-RNA%252Band%252BModulate%252Bthe%252BInteraction%252Bupon%252BFormation%252Bof%252BZn2%25252B%252BComplexes%26aulast%3DGuijarro%26aufirst%3DLlu%25C3%25ADs%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D13%26spage%3D3957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hafeez S.  Haniff</span>, <span class="hlFld-ContribAuthor ">Xiaohui  Liu</span>, <span class="hlFld-ContribAuthor ">Yuquan  Tong</span>, <span class="hlFld-ContribAuthor ">Samantha M.  Meyer</span>, <span class="hlFld-ContribAuthor ">Laurent  Knerr</span>, <span class="hlFld-ContribAuthor ">Malin  Lemurell</span>, <span class="hlFld-ContribAuthor ">Daniel  Abegg</span>, <span class="hlFld-ContribAuthor ">Haruo  Aikawa</span>, <span class="hlFld-ContribAuthor ">Alexander  Adibekian</span>, <span class="hlFld-ContribAuthor ">Matthew D.  Disney</span>. </span><span class="cited-content_cbyCitation_article-title">A structure-specific small molecule inhibits a miRNA-200 family member precursor and reverses a type 2 diabetes phenotype. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>4 </em><a href="https://doi.org/10.1016/j.chembiol.2021.07.006" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.07.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.07.006%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DA%252Bstructure-specific%252Bsmall%252Bmolecule%252Binhibits%252Ba%252BmiRNA-200%252Bfamily%252Bmember%252Bprecursor%252Band%252Breverses%252Ba%252Btype%252B2%252Bdiabetes%252Bphenotype%26aulast%3DHaniff%26aufirst%3DHafeez%2BS.%26date%3D2021%26volume%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dagmara  Baraniak</span>, <span class="hlFld-ContribAuthor ">Jerzy  Boryski</span>. </span><span class="cited-content_cbyCitation_article-title">Triazole-Modified Nucleic Acids for the Application in Bioorganic and Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Biomedicines</span><span> <strong>2021,</strong> <em>9 </em>
                                    (6)
                                     , 628. <a href="https://doi.org/10.3390/biomedicines9060628" title="DOI URL">https://doi.org/10.3390/biomedicines9060628</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biomedicines9060628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiomedicines9060628%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicines%26atitle%3DTriazole-Modified%252BNucleic%252BAcids%252Bfor%252Bthe%252BApplication%252Bin%252BBioorganic%252Band%252BMedicinal%252BChemistry%26aulast%3DBaraniak%26aufirst%3DDagmara%26date%3D2021%26date%3D2021%26volume%3D9%26issue%3D6%26spage%3D628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martina  Zafferani</span>, <span class="hlFld-ContribAuthor ">Amanda E.  Hargrove</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecule targeting of biologically relevant RNA tertiary and quaternary structures. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (5)
                                     , 594-609. <a href="https://doi.org/10.1016/j.chembiol.2021.03.003" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.03.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.03.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DSmall%252Bmolecule%252Btargeting%252Bof%252Bbiologically%252Brelevant%252BRNA%252Btertiary%252Band%252Bquaternary%252Bstructures%26aulast%3DZafferani%26aufirst%3DMartina%26date%3D2021%26volume%3D28%26issue%3D5%26spage%3D594%26epage%3D609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Muthana  Al Abo</span>, <span class="hlFld-ContribAuthor ">Terry  Hyslop</span>, <span class="hlFld-ContribAuthor ">Xiaodi  Qin</span>, <span class="hlFld-ContribAuthor ">Kouros  Owzar</span>, <span class="hlFld-ContribAuthor ">Daniel J.  George</span>, <span class="hlFld-ContribAuthor ">Steven R.  Patierno</span>, <span class="hlFld-ContribAuthor ">Jennifer A.  Freedman</span>. </span><span class="cited-content_cbyCitation_article-title">Differential alternative RNA splicing and transcription events between tumors from African American and White patients in The Cancer Genome Atlas. </span><span class="cited-content_cbyCitation_journal-name">Genomics</span><span> <strong>2021,</strong> <em>113 </em>
                                    (3)
                                     , 1234-1246. <a href="https://doi.org/10.1016/j.ygeno.2021.02.020" title="DOI URL">https://doi.org/10.1016/j.ygeno.2021.02.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ygeno.2021.02.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ygeno.2021.02.020%26sid%3Dliteratum%253Aachs%26jtitle%3DGenomics%26atitle%3DDifferential%252Balternative%252BRNA%252Bsplicing%252Band%252Btranscription%252Bevents%252Bbetween%252Btumors%252Bfrom%252BAfrican%252BAmerican%252Band%252BWhite%252Bpatients%252Bin%252BThe%252BCancer%252BGenome%252BAtlas%26aulast%3DAl%2BAbo%26aufirst%3DMuthana%26date%3D2021%26volume%3D113%26issue%3D3%26spage%3D1234%26epage%3D1246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ramya  Rangan</span>, <span class="hlFld-ContribAuthor ">Andrew M  Watkins</span>, <span class="hlFld-ContribAuthor ">Jose  Chacon</span>, <span class="hlFld-ContribAuthor ">Rachael  Kretsch</span>, <span class="hlFld-ContribAuthor ">Wipapat  Kladwang</span>, <span class="hlFld-ContribAuthor ">Ivan N  Zheludev</span>, <span class="hlFld-ContribAuthor ">Jill  Townley</span>, <span class="hlFld-ContribAuthor ">Mats  Rynge</span>, <span class="hlFld-ContribAuthor ">Gregory  Thain</span>, <span class="hlFld-ContribAuthor ">Rhiju  Das</span>. </span><span class="cited-content_cbyCitation_article-title">De novo
              3D models of SARS-CoV-2 RNA elements from consensus experimental secondary structures. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2021,</strong> <em>49 </em>
                                    (6)
                                     , 3092-3108. <a href="https://doi.org/10.1093/nar/gkab119" title="DOI URL">https://doi.org/10.1093/nar/gkab119</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkab119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkab119%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DDe%252Bnovo%252B3D%252Bmodels%252Bof%252BSARS-CoV-2%252BRNA%252Belements%252Bfrom%252Bconsensus%252Bexperimental%252Bsecondary%252Bstructures%26aulast%3DRangan%26aufirst%3DRamya%26date%3D2021%26date%3D2021%26volume%3D49%26issue%3D6%26spage%3D3092%26epage%3D3108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria Teresa  Di Martino</span>, <span class="hlFld-ContribAuthor ">Caterina  Riillo</span>, <span class="hlFld-ContribAuthor ">Francesca  Scionti</span>, <span class="hlFld-ContribAuthor ">Katia  Grillone</span>, <span class="hlFld-ContribAuthor ">Nicoletta  Polerà</span>, <span class="hlFld-ContribAuthor ">Daniele  Caracciolo</span>, <span class="hlFld-ContribAuthor ">Mariamena  Arbitrio</span>, <span class="hlFld-ContribAuthor ">Pierosandro  Tagliaferri</span>, <span class="hlFld-ContribAuthor ">Pierfrancesco  Tassone</span>. </span><span class="cited-content_cbyCitation_article-title">miRNAs and lncRNAs as Novel Therapeutic Targets to Improve Cancer Immunotherapy. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (7)
                                     , 1587. <a href="https://doi.org/10.3390/cancers13071587" title="DOI URL">https://doi.org/10.3390/cancers13071587</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13071587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13071587%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DmiRNAs%252Band%252BlncRNAs%252Bas%252BNovel%252BTherapeutic%252BTargets%252Bto%252BImprove%252BCancer%252BImmunotherapy%26aulast%3DDi%2BMartino%26aufirst%3DMaria%2BTeresa%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D7%26spage%3D1587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Siyuan  Luan</span>, <span class="hlFld-ContribAuthor ">Yushang  Yang</span>, <span class="hlFld-ContribAuthor ">Yuxin  Zhou</span>, <span class="hlFld-ContribAuthor ">Xiaoxi  Zeng</span>, <span class="hlFld-ContribAuthor ">Xin  Xiao</span>, <span class="hlFld-ContribAuthor ">Bo  Liu</span>, <span class="hlFld-ContribAuthor ">Yong  Yuan</span>. </span><span class="cited-content_cbyCitation_article-title">The emerging role of long noncoding RNAs in esophageal carcinoma: from underlying mechanisms to clinical implications. </span><span class="cited-content_cbyCitation_journal-name">Cellular and Molecular Life Sciences</span><span> <strong>2021,</strong> <em>78 </em>
                                    (7)
                                     , 3403-3422. <a href="https://doi.org/10.1007/s00018-020-03751-0" title="DOI URL">https://doi.org/10.1007/s00018-020-03751-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00018-020-03751-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00018-020-03751-0%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520and%2520Molecular%2520Life%2520Sciences%26atitle%3DThe%252Bemerging%252Brole%252Bof%252Blong%252Bnoncoding%252BRNAs%252Bin%252Besophageal%252Bcarcinoma%25253A%252Bfrom%252Bunderlying%252Bmechanisms%252Bto%252Bclinical%252Bimplications%26aulast%3DLuan%26aufirst%3DSiyuan%26date%3D2021%26date%3D2021%26volume%3D78%26issue%3D7%26spage%3D3403%26epage%3D3422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sagar  Satpathi</span>, <span class="hlFld-ContribAuthor ">Tamaki  Endoh</span>, <span class="hlFld-ContribAuthor ">Peter  Podbevšek</span>, <span class="hlFld-ContribAuthor ">Janez  Plavec</span>, <span class="hlFld-ContribAuthor ">Naoki  Sugimoto</span>. </span><span class="cited-content_cbyCitation_article-title">Transcriptome screening followed by integrated physicochemical and structural analyses for investigating RNA-mediated berberine activity. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2021,</strong> <em>11 </em><a href="https://doi.org/10.1093/nar/gkab189" title="DOI URL">https://doi.org/10.1093/nar/gkab189</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkab189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkab189%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DTranscriptome%252Bscreening%252Bfollowed%252Bby%252Bintegrated%252Bphysicochemical%252Band%252Bstructural%252Banalyses%252Bfor%252Binvestigating%252BRNA-mediated%252Bberberine%252Bactivity%26aulast%3DSatpathi%26aufirst%3DSagar%26date%3D2021%26date%3D2021%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David Z  Huang</span>, <span class="hlFld-ContribAuthor ">J. Christian  Baber</span>, <span class="hlFld-ContribAuthor ">Sogole Sami  Bahmanyar</span>. </span><span class="cited-content_cbyCitation_article-title">The challenges of generalizability in artificial intelligence for ADME/Tox endpoint and activity prediction. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2021,</strong> <em>576 </em>, 1-12. <a href="https://doi.org/10.1080/17460441.2021.1901685" title="DOI URL">https://doi.org/10.1080/17460441.2021.1901685</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2021.1901685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2021.1901685%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DThe%252Bchallenges%252Bof%252Bgeneralizability%252Bin%252Bartificial%252Bintelligence%252Bfor%252BADME%25252FTox%252Bendpoint%252Band%252Bactivity%252Bprediction%26aulast%3DHuang%26aufirst%3DDavid%2BZ%26date%3D2021%26date%3D2021%26volume%3D576%26spage%3D1%26epage%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dejun  Liu</span>, <span class="hlFld-ContribAuthor ">Xinqiang  Wan</span>, <span class="hlFld-ContribAuthor ">Xiangxiang  Shan</span>, <span class="hlFld-ContribAuthor ">Rengen  Fan</span>, <span class="hlFld-ContribAuthor ">Wenzhang  Zha</span>. </span><span class="cited-content_cbyCitation_article-title">Drugging the “undruggable” microRNAs. </span><span class="cited-content_cbyCitation_journal-name">Cellular and Molecular Life Sciences</span><span> <strong>2021,</strong> <em>78 </em>
                                    (5)
                                     , 1861-1871. <a href="https://doi.org/10.1007/s00018-020-03676-8" title="DOI URL">https://doi.org/10.1007/s00018-020-03676-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00018-020-03676-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00018-020-03676-8%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520and%2520Molecular%2520Life%2520Sciences%26atitle%3DDrugging%252Bthe%252B%2525E2%252580%25259Cundruggable%2525E2%252580%25259D%252BmicroRNAs%26aulast%3DLiu%26aufirst%3DDejun%26date%3D2021%26date%3D2020%26volume%3D78%26issue%3D5%26spage%3D1861%26epage%3D1871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">James P.  Falese</span>, <span class="hlFld-ContribAuthor ">Anita  Donlic</span>, <span class="hlFld-ContribAuthor ">Amanda E.  Hargrove</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting RNA with small molecules: from fundamental principles towards the clinic. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2021,</strong> <em>50 </em>
                                    (4)
                                     , 2224-2243. <a href="https://doi.org/10.1039/D0CS01261K" title="DOI URL">https://doi.org/10.1039/D0CS01261K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CS01261K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CS01261K%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DTargeting%252BRNA%252Bwith%252Bsmall%252Bmolecules%25253A%252Bfrom%252Bfundamental%252Bprinciples%252Btowards%252Bthe%252Bclinic%26aulast%3DFalese%26aufirst%3DJames%2BP.%26date%3D2021%26date%3D2021%26volume%3D50%26issue%3D4%26spage%3D2224%26epage%3D2243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yangwei  Jiang</span>, <span class="hlFld-ContribAuthor ">Shi-Jie  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">RLDOCK method for predicting RNA-small molecule binding modes. </span><span class="cited-content_cbyCitation_journal-name">Methods</span><span> <strong>2021,</strong> <em>289 </em><a href="https://doi.org/10.1016/j.ymeth.2021.01.009" title="DOI URL">https://doi.org/10.1016/j.ymeth.2021.01.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ymeth.2021.01.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ymeth.2021.01.009%26sid%3Dliteratum%253Aachs%26jtitle%3DMethods%26atitle%3DRLDOCK%252Bmethod%252Bfor%252Bpredicting%252BRNA-small%252Bmolecule%252Bbinding%252Bmodes%26aulast%3DJiang%26aufirst%3DYangwei%26date%3D2021%26volume%3D289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Timothy L.  Foley</span>, <span class="hlFld-ContribAuthor ">Woodrow  Burchett</span>, <span class="hlFld-ContribAuthor ">Qiuxia  Chen</span>, <span class="hlFld-ContribAuthor ">Mark E.  Flanagan</span>, <span class="hlFld-ContribAuthor ">Brendon  Kapinos</span>, <span class="hlFld-ContribAuthor ">Xianyang  Li</span>, <span class="hlFld-ContribAuthor ">Justin I.  Montgomery</span>, <span class="hlFld-ContribAuthor ">Anokha S.  Ratnayake</span>, <span class="hlFld-ContribAuthor ">Hongyao  Zhu</span>, <span class="hlFld-ContribAuthor ">Marie-Claire  Peakman</span>. </span><span class="cited-content_cbyCitation_article-title">Selecting Approaches for Hit Identification and Increasing Options by Building the Efficient Discovery of Actionable Chemical Matter from DNA-Encoded Libraries. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2021,</strong> <em>26 </em>
                                    (2)
                                     , 263-280. <a href="https://doi.org/10.1177/2472555220979589" title="DOI URL">https://doi.org/10.1177/2472555220979589</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555220979589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555220979589%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DSelecting%252BApproaches%252Bfor%252BHit%252BIdentification%252Band%252BIncreasing%252BOptions%252Bby%252BBuilding%252Bthe%252BEfficient%252BDiscovery%252Bof%252BActionable%252BChemical%252BMatter%252Bfrom%252BDNA-Encoded%252BLibraries%26aulast%3DFoley%26aufirst%3DTimothy%2BL.%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D2%26spage%3D263%26epage%3D280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenfei  Li</span>, <span class="hlFld-ContribAuthor ">Stacey Tsai-Lan  Chang</span>, <span class="hlFld-ContribAuthor ">Fred. R.  Ward</span>, <span class="hlFld-ContribAuthor ">Jamie H. D.  Cate</span>. </span><span class="cited-content_cbyCitation_article-title">Selective inhibition of human translation termination by a drug-like compound. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2020,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-020-18765-2" title="DOI URL">https://doi.org/10.1038/s41467-020-18765-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-020-18765-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-020-18765-2%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DSelective%252Binhibition%252Bof%252Bhuman%252Btranslation%252Btermination%252Bby%252Ba%252Bdrug-like%252Bcompound%26aulast%3DLi%26aufirst%3DWenfei%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yonghan  He</span>, <span class="hlFld-ContribAuthor ">Sajid  Khan</span>, <span class="hlFld-ContribAuthor ">Zhiguang  Huo</span>, <span class="hlFld-ContribAuthor ">Dongwen  Lv</span>, <span class="hlFld-ContribAuthor ">Xuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Xingui  Liu</span>, <span class="hlFld-ContribAuthor ">Yaxia  Yuan</span>, <span class="hlFld-ContribAuthor ">Robert  Hromas</span>, <span class="hlFld-ContribAuthor ">Mingjiang  Xu</span>, <span class="hlFld-ContribAuthor ">Guangrong  Zheng</span>, <span class="hlFld-ContribAuthor ">Daohong  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimeras (PROTACs) are emerging therapeutics for hematologic malignancies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2020,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-020-00924-z" title="DOI URL">https://doi.org/10.1186/s13045-020-00924-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-020-00924-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-020-00924-z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DProteolysis%252Btargeting%252Bchimeras%252B%252528PROTACs%252529%252Bare%252Bemerging%252Btherapeutics%252Bfor%252Bhematologic%252Bmalignancies%26aulast%3DHe%26aufirst%3DYonghan%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samantha M.  Meyer</span>, <span class="hlFld-ContribAuthor ">Christopher C.  Williams</span>, <span class="hlFld-ContribAuthor ">Yoshihiro  Akahori</span>, <span class="hlFld-ContribAuthor ">Toru  Tanaka</span>, <span class="hlFld-ContribAuthor ">Haruo  Aikawa</span>, <span class="hlFld-ContribAuthor ">Yuquan  Tong</span>, <span class="hlFld-ContribAuthor ">Jessica L.  Childs-Disney</span>, <span class="hlFld-ContribAuthor ">Matthew D.  Disney</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecule recognition of disease-relevant RNA structures. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2020,</strong> <em>49 </em>
                                    (19)
                                     , 7167-7199. <a href="https://doi.org/10.1039/D0CS00560F" title="DOI URL">https://doi.org/10.1039/D0CS00560F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CS00560F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CS00560F%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DSmall%252Bmolecule%252Brecognition%252Bof%252Bdisease-relevant%252BRNA%252Bstructures%26aulast%3DMeyer%26aufirst%3DSamantha%2BM.%26date%3D2020%26date%3D2020%26volume%3D49%26issue%3D19%26spage%3D7167%26epage%3D7199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jun  Li</span>, <span class="hlFld-ContribAuthor ">Man  Wang</span>, <span class="hlFld-ContribAuthor ">Yao-Yao  Fan</span>, <span class="hlFld-ContribAuthor ">Hui-Ling  Duan</span>, <span class="hlFld-ContribAuthor ">Xu  Deng</span>, <span class="hlFld-ContribAuthor ">Zhi-Qi  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand-RNA interaction assay based on size-selective fluorescence core-shell nanocomposite. </span><span class="cited-content_cbyCitation_journal-name">Analytical and Bioanalytical Chemistry</span><span> <strong>2020,</strong> <em>412 </em>
                                    (26)
                                     , 7349-7356. <a href="https://doi.org/10.1007/s00216-020-02869-1" title="DOI URL">https://doi.org/10.1007/s00216-020-02869-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00216-020-02869-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00216-020-02869-1%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520and%2520Bioanalytical%2520Chemistry%26atitle%3DLigand-RNA%252Binteraction%252Bassay%252Bbased%252Bon%252Bsize-selective%252Bfluorescence%252Bcore-shell%252Bnanocomposite%26aulast%3DLi%26aufirst%3DJun%26date%3D2020%26date%3D2020%26volume%3D412%26issue%3D26%26spage%3D7349%26epage%3D7356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ai-Ming  Yu</span>, <span class="hlFld-ContribAuthor ">Young Hee  Choi</span>, <span class="hlFld-ContribAuthor ">Mei-Juan  Tu</span>, . </span><span class="cited-content_cbyCitation_article-title">RNA Drugs and RNA Targets for Small Molecules: Principles, Progress, and Challenges. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Reviews</span><span> <strong>2020,</strong> <em>72 </em>
                                    (4)
                                     , 862-898. <a href="https://doi.org/10.1124/pr.120.019554" title="DOI URL">https://doi.org/10.1124/pr.120.019554</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/pr.120.019554&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fpr.120.019554%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Reviews%26atitle%3DRNA%252BDrugs%252Band%252BRNA%252BTargets%252Bfor%252BSmall%252BMolecules%25253A%252BPrinciples%25252C%252BProgress%25252C%252Band%252BChallenges%26aulast%3DYu%26aufirst%3DAi-Ming%26date%3D2020%26date%3D2020%26volume%3D72%26issue%3D4%26spage%3D862%26epage%3D898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sajid  Khan</span>, <span class="hlFld-ContribAuthor ">Yonghan  He</span>, <span class="hlFld-ContribAuthor ">Xuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Yaxia  Yuan</span>, <span class="hlFld-ContribAuthor ">Shaoyan  Pu</span>, <span class="hlFld-ContribAuthor ">Qingpeng  Kong</span>, <span class="hlFld-ContribAuthor ">Guangrong  Zheng</span>, <span class="hlFld-ContribAuthor ">Daohong  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">PROteolysis TArgeting Chimeras (PROTACs) as emerging anticancer therapeutics. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2020,</strong> <em>39 </em>
                                    (26)
                                     , 4909-4924. <a href="https://doi.org/10.1038/s41388-020-1336-y" title="DOI URL">https://doi.org/10.1038/s41388-020-1336-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41388-020-1336-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41388-020-1336-y%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DPROteolysis%252BTArgeting%252BChimeras%252B%252528PROTACs%252529%252Bas%252Bemerging%252Banticancer%252Btherapeutics%26aulast%3DKhan%26aufirst%3DSajid%26date%3D2020%26date%3D2020%26volume%3D39%26issue%3D26%26spage%3D4909%26epage%3D4924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. RNA as a viable drug target. (A) The conventional binary approach to small molecule drugs is their molecular recognition of proteins. Among the ∼20 000 proteins that comprise the proteome, only about 15% are in traditional “druggable” protein families. In turn, this only represents a fraction of the genome that is transcribed, leaving much of the transcriptome underexploited as therapeutic targets. (B) Noncoding genes relate to the complexity of the organism, as the relative number of coding bases remains similar, while the relative number of long noncoding RNAs (lncRNA) significantly increases, indicating that much of the intricacies of human biology and disease are represented among noncoding regions. (C) Due to the importance of coding and noncoding RNA to biology, small molecules interacting with RNA can act on the transcriptome, resulting in varied downstream effects. Importantly, validated activities for small molecules that target human RNA include: (i) changing gene expression by modulating the stability of mRNA by direct binding; (ii) affecting its noncoding RNA effectors; (iii) affecting the epitranscriptome; or (iv) influencing alternative splicing. Affecting the transcriptome with small molecule drugs can rescue disease by modulating the translation of beneficial or detrimental proteins.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Timeline of major developments in the RNA-targeting field. The history of drugging RNA is tied closely with molecular biology discoveries (DNA/RNA structural determination). Antibacterials that targeted RNA preceded the first investigations into antisense oligonucleotides. However, FDA approvals of antisense oligonucleotides increased upon completion of the human genome project. Recent developments, such as the use of rational design-based approaches, the initiation of clinical trials for small molecule drugs treating spinal muscular atrophy (SMA), and the first report of ribonuclease targeting chimeras (RIBOTACs), demonstrate the rapid development of small molecules targeting RNA. These advancements provide a convincing argument to turn our focus to the druggable transcriptome.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Modular RNA secondary structure motifs form three-dimensional structures. (A) Hierarchical assembly of RNA structure from sequence to secondary structure. Many of these secondary structures can form modular RNA motifs that can allow for small molecule recognition. (B) Structural schematic of microRNA processing. Primary transcripts (pri-miRNAs) are processed by the Drosha endonuclease to yield precursor hairpins (pre-miRNAs), which are exported to the cytoplasm and subsequently processed by the Dicer endonuclease to liberate a mature miRNA. One of the mature strands is then loaded into the argonaute/RNA-induced silencing complex (AGO/RISC), whereupon it acts on RNAs to modulate gene expression. Aberrant miRNA expression can be causative of disease phenotypes. (C) One rational design approach to target RNA is to understand the molecular recognition of structural elements by small molecules, those elements that are preferred by the small molecule and those that are discriminated against. Inforna compares structural elements within an RNA target to a database of these preferred interactions to afford lead small molecules. For example, binding to the pri- or pre-miRNAs at functional Drosha or Dicer sites can prevent their processing to the active, mature strand, thus allowing the rescue of disease-associated phenotypes through the inhibition of biogenesis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Factors that influence bioactivity and selectivity of RNA-binding ligands. (A) 2DCS selection and HiT-StARTS analysis identifies the top binding RNA motifs and nonbinders to targapremir-210 (TGP-210). Z-Score is a calculated value that represents fitness of the RNA-small molecule interaction. (B) Fitness plot of the top 25 binding motifs are shown. One of the highest fitness RNA motif interactions with TGP-210 is the 5′ A<u class="uu">C</u>U/3′ U<u class="uu">C</u>A 3 × 3 internal loop (blue box) found in the functional Dicer site of miR-210. The 5′ A<u class="uu">G</u>C/3′ U<u class="uu">A</u>G RNA motif is a lower affinity interaction (purple box). (C) Structure of TGP-210. (D, E) The bioactivity of TGP-210 to selectively inhibit miR-210 biogenesis is a function of multiple factors, including the fitness of the RNA motif-small molecule interaction, the expression and turnover of the target RNA(s), whether or not occupancy of the target site will result in a functional interaction (i.e., occupying a Dicer/Drosha site), and the accessibility of TGP-210 to off-target sites.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Tools to assess target engagement and selectivity of small molecules targeting RNA. Developing small molecules against RNA starts with identification of a hit, whether through Inforna (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) or screening approaches (target-based, phenotypic, fragment-based, DNA-encoded, etc.). Considering the factors from <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, the hit must then be validated and optimized, including for <i>in vitro</i> binding affinity to the RNA structural element over RNAs that do not contain the motif and other abundant RNA/DNAs. Further validation <i>in vitro</i> and in cells can be accomplished with target engagement approaches that use chemical probing methods that measure RNA enrichment (Chem-CLIP) or RNA depletion (Competitive (C)-Chem-CLIP), among others. Comprehensively evaluating cellular selectivity on a transcriptome- and proteome-wide scale is also part of the workflow to validate a small molecule RNA target. After demonstration of selective on-target effects, the compound’s functional effect must then be validated in more advanced models, including the effect of a gain or loss in expression of the target.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Quantitatively evaluating the selectivity of small molecules targeting RNA. (A) Data from profiling experiments can be used to quantify compound selectivity by calculating a Gini coefficient (GC). GC analysis of kinase inhibitors showed that promiscuous compounds (staurosporine) are characterized by values close to 0, while highly selective compounds exhibit Gini coefficient values close to 1 (PD184352; 0.91), with selective compounds being defined as >0.6. (B) GC analyses can be applied to profiling data (left), such as miRNA qPCR profiling data between small molecules (TGP-515; blue) and ASOs (515-ASO; green). Applying this analysis to various small molecule ligands targeting RNA <i>structure</i> indicates that they demonstrate high selectivity for their targets. Antisense oligonucleotides targeting RNA <i>sequence</i> are also selective for their targets. Shown are the structures and GC values of TGP-515 (blue) and Vivo-Morpholino ASO targeting miR-515 (green), in addition to other small molecule/ASO GC analyses. Overall, Gini coefficients provide a metric to quantitatively define compound selectivity. When applied to RNA targeting, GC analyses demonstrate that small molecules that recognize RNA structure can rival or exceed the selectivity of ASOs designed to bind via Watson–Crick base pairing.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Physiochemical properties of RNA binders and common RNA-binding scaffolds. (A) Physiochemical properties of cataloged RNA binders contained within Inforna and R-BIND show convergence and correlate with properties of FDA approved drugs. FDA approved drugs were taken from DrugBank.<a onclick="showRef(event, 'ref122'); return false;" href="javascript:void(0);" class="ref ref122">(122)</a> (B) Properties that are enriched within RNA binders include greater positive charge at pH 7.4, number of H-bond donor and acceptor counts, and total polar surface area (TPSA). RNA binders also exhibit fewer chiral centers, aliphatic atoms, and rotational bonds compared to nonbinders. (C) Scaffolds contained within Inforna and R-BIND that exhibit RNA binding. These include oligosaccharides, benzimidazoles, purines, naphthalenes, quinolines, quinazolines, aminopyrimidines, and quinidine thiazoles. Interestingly, while most data generally point to planar molecules as RNA binders, some sterically rich compounds have been found to bind RNA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Properties of bioactive therapeutic modalities. Current and emerging strategies to affect downstream biology include antibodies (rituximab, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KAQ">4KAQ</a>), ASOs (Nusinersen), small molecules (TGP-210), and targeted degradation approaches (TGP-210 RIBOTAC). Each targets a unique space, but RIBOTACs can affect bioactivity of RNA without binding to a functional site. RIBOTACs also degrade their targets in a catalytic and substoichiometric fashion, thus allowing greater potency. Small-molecule-based modalities are advantageous as their physicochemical properties can be potentially medicinally optimized. ASO, small molecule, and targeted degradation models were made using the Online SMILES Generator (National Cancer Institute).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/medium/jm9b01927_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Ribonuclease targeting chimeras (RIBOTACs) as heterobifunctional degraders of RNA. (A) Taking cues from PROTACs and RNase H-based antisense oligonucleotide approaches, RIBOTACs are heterobifunctional compounds that recruit endogenous nucleases to degrade a targeted transcript. RIBOTACs can potentially increase potency of small molecules as they can catalytically and substoichiometrically degrade an RNA target. These RIBOTACs simply need to bind the target (not necessarily at a functional site) and use endogenous ribonuclease pathways to remove the RNA via targeted degradation, which also rids the RNA of any potential scaffolding functions with RBPs. Formation of the ternary complex may also increase selectivity as only meaningful interactions between the RNA:RIBOTAC:RNase L will result in cleavage. While this approach is potentially broadly applicable, development and optimization of both RNA binders and RNase-recruiting modules remain time-consuming, especially as there are a limited number of known RNase activators. Additionally, RIBOTACs that function through RNase L can have less pronounced effects on nuclear RNA, as RNase L is primarily cytoplasmic. (B) Advantages provided by the RIBOTAC approach. (C) Demonstration of increased selectivity of different RNA-binding modules, as indicated by GC analysis. The monomeric RNA-binding module that binds a single functional site on an RNA is less selective than the multivalent ligand targeting the same RNA. Adding an RNase L recruitment module to convert the dimeric compound into a RIBOTAC allows for increased selectivity, potentially due to the requirement of effective ternary complex formation between the RNA, RIBOTAC, and RNase L.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.9b01927/20200903/images/large/jm9b01927_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01927&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52552" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52552" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 135 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linton, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nusbaum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zody, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FitzHugh, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gage, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heaford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehoczky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeVine, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEwan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKernan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meldrim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesirov, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raymond, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sougnez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stange-Thomann, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stojanovic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulston, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ainscough, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bentley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deadman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deloukas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunham, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunham, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durbin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grafham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphray, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullikin, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mungall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plumb, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shownkeen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterston, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillier, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPherson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mardis, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulton, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinwalla, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pepin, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gish, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chissoe, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendl, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delehaunty, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miner, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delehaunty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minx, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clifton, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branscomb, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Predki, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenning, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slezak, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doggett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uberbacher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frazier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbs, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muzny, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherer, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouck, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sodergren, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worley, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rives, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorrell, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzker, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucherlapati, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstock, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyoda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawagoe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissenbach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heilig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saurin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artiguenave, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brottier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruls, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelletier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wincker, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette-Stamm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubenfield, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstock, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubois, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platzer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyakatura, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taudien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rump, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federspiel, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abola, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proctor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmutz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimwood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaul, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raymond, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimizu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawasaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minoshima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasiou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehrach, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhardt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCombie, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Bastide, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedhia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blocker, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornischer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordsiek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aravind, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bateman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batzoglou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bork, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burge, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerutti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Church, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clamp, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copley, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doerks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddy, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichler, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furey, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galagan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashizaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haussler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermjakob, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hokamp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaspryzk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitts, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koonin, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korf, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLysaght, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikkelsen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulder, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollara, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponting, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smit, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupka, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szustakowki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thierry-Mieg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thierry-Mieg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, Y. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guyer, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felsenfeld, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetterstrand, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patrinos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jong, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catanese, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osoegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shizuya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szustakowki, J.</span></span> <span> </span><span class="NLM_article-title">Initial sequencing and analysis of the human genome</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>409</i></span>,  <span class="NLM_fpage">860</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1038/35057062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2F35057062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=11237011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhsFCjtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=409&publication_year=2001&pages=860-921&author=E.+S.+Landerauthor=L.+M.+Lintonauthor=B.+Birrenauthor=C.+Nusbaumauthor=M.+C.+Zodyauthor=J.+Baldwinauthor=K.+Devonauthor=K.+Dewarauthor=M.+Doyleauthor=W.+FitzHughauthor=R.+Funkeauthor=D.+Gageauthor=K.+Harrisauthor=A.+Heafordauthor=J.+Howlandauthor=L.+Kannauthor=J.+Lehoczkyauthor=R.+LeVineauthor=P.+McEwanauthor=K.+McKernanauthor=J.+Meldrimauthor=J.+P.+Mesirovauthor=C.+Mirandaauthor=W.+Morrisauthor=J.+Naylorauthor=C.+Raymondauthor=M.+Rosettiauthor=R.+Santosauthor=A.+Sheridanauthor=C.+Sougnezauthor=Y.+Stange-Thomannauthor=N.+Stojanovicauthor=A.+Subramanianauthor=D.+Wymanauthor=J.+Rogersauthor=J.+Sulstonauthor=R.+Ainscoughauthor=S.+Beckauthor=D.+Bentleyauthor=J.+Burtonauthor=C.+Cleeauthor=N.+Carterauthor=A.+Coulsonauthor=R.+Deadmanauthor=P.+Deloukasauthor=A.+Dunhamauthor=I.+Dunhamauthor=R.+Durbinauthor=L.+Frenchauthor=D.+Grafhamauthor=S.+Gregoryauthor=T.+Hubbardauthor=S.+Humphrayauthor=A.+Huntauthor=M.+Jonesauthor=C.+Lloydauthor=A.+McMurrayauthor=L.+Matthewsauthor=S.+Mercerauthor=S.+Milneauthor=J.+C.+Mullikinauthor=A.+Mungallauthor=R.+Plumbauthor=M.+Rossauthor=R.+Shownkeenauthor=S.+Simsauthor=R.+H.+Waterstonauthor=R.+K.+Wilsonauthor=L.+W.+Hillierauthor=J.+D.+McPhersonauthor=M.+A.+Marraauthor=E.+R.+Mardisauthor=L.+A.+Fultonauthor=A.+T.+Chinwallaauthor=K.+H.+Pepinauthor=W.+R.+Gishauthor=S.+L.+Chissoeauthor=M.+C.+Wendlauthor=K.+D.+Delehauntyauthor=T.+L.+Minerauthor=A.+Delehauntyauthor=J.+B.+Kramerauthor=L.+L.+Cookauthor=R.+S.+Fultonauthor=D.+L.+Johnsonauthor=P.+J.+Minxauthor=S.+W.+Cliftonauthor=T.+Hawkinsauthor=E.+Branscombauthor=P.+Predkiauthor=P.+Richardsonauthor=S.+Wenningauthor=T.+Slezakauthor=N.+Doggettauthor=J.+F.+Chengauthor=A.+Olsenauthor=S.+Lucasauthor=C.+Elkinauthor=E.+Uberbacherauthor=M.+Frazierauthor=R.+A.+Gibbsauthor=D.+M.+Muznyauthor=S.+E.+Schererauthor=J.+B.+Bouckauthor=E.+J.+Sodergrenauthor=K.+C.+Worleyauthor=C.+M.+Rivesauthor=J.+H.+Gorrellauthor=M.+L.+Metzkerauthor=S.+L.+Naylorauthor=R.+S.+Kucherlapatiauthor=D.+L.+Nelsonauthor=G.+M.+Weinstockauthor=Y.+Sakakiauthor=A.+Fujiyamaauthor=M.+Hattoriauthor=T.+Yadaauthor=A.+Toyodaauthor=T.+Itohauthor=C.+Kawagoeauthor=H.+Watanabeauthor=Y.+Totokiauthor=T.+Taylorauthor=J.+Weissenbachauthor=R.+Heiligauthor=W.+Saurinauthor=F.+Artiguenaveauthor=P.+Brottierauthor=T.+Brulsauthor=E.+Pelletierauthor=C.+Robertauthor=P.+Winckerauthor=D.+R.+Smithauthor=L.+Doucette-Stammauthor=M.+Rubenfieldauthor=K.+Weinstockauthor=H.+M.+Leeauthor=J.+Duboisauthor=A.+Rosenthalauthor=M.+Platzerauthor=G.+Nyakaturaauthor=S.+Taudienauthor=A.+Rumpauthor=H.+Yangauthor=J.+Yuauthor=J.+Wangauthor=G.+Huangauthor=J.+Guauthor=L.+Hoodauthor=L.+Rowenauthor=A.+Madanauthor=S.+Qinauthor=R.+W.+Davisauthor=N.+A.+Federspielauthor=A.+P.+Abolaauthor=M.+J.+Proctorauthor=R.+M.+Myersauthor=J.+Schmutzauthor=M.+Dicksonauthor=J.+Grimwoodauthor=D.+R.+Coxauthor=M.+V.+Olsonauthor=R.+Kaulauthor=C.+Raymondauthor=N.+Shimizuauthor=K.+Kawasakiauthor=S.+Minoshimaauthor=G.+A.+Evansauthor=M.+Athanasiouauthor=R.+Schultzauthor=B.+A.+Roeauthor=F.+Chenauthor=H.+Panauthor=J.+Ramserauthor=H.+Lehrachauthor=R.+Reinhardtauthor=W.+R.+McCombieauthor=M.+de+la+Bastideauthor=N.+Dedhiaauthor=H.+Blockerauthor=K.+Hornischerauthor=G.+Nordsiekauthor=R.+Agarwalaauthor=L.+Aravindauthor=J.+A.+Baileyauthor=A.+Batemanauthor=S.+Batzoglouauthor=E.+Birneyauthor=P.+Borkauthor=D.+G.+Brownauthor=C.+B.+Burgeauthor=L.+Ceruttiauthor=H.+C.+Chenauthor=D.+Churchauthor=M.+Clampauthor=R.+R.+Copleyauthor=T.+Doerksauthor=S.+R.+Eddyauthor=E.+E.+Eichlerauthor=T.+S.+Fureyauthor=J.+Galaganauthor=J.+G.+Gilbertauthor=C.+Harmonauthor=Y.+Hayashizakiauthor=D.+Hausslerauthor=H.+Hermjakobauthor=K.+Hokampauthor=W.+Jangauthor=L.+S.+Johnsonauthor=T.+A.+Jonesauthor=S.+Kasifauthor=A.+Kaspryzkauthor=S.+Kennedyauthor=W.+J.+Kentauthor=P.+Kittsauthor=E.+V.+Kooninauthor=I.+Korfauthor=D.+Kulpauthor=D.+Lancetauthor=T.+M.+Loweauthor=A.+McLysaghtauthor=T.+Mikkelsenauthor=J.+V.+Moranauthor=N.+Mulderauthor=V.+J.+Pollaraauthor=C.+P.+Pontingauthor=G.+Schulerauthor=J.+Schultzauthor=G.+Slaterauthor=A.+F.+Smitauthor=E.+Stupkaauthor=J.+Szustakowkiauthor=D.+Thierry-Miegauthor=J.+Thierry-Miegauthor=L.+Wagnerauthor=J.+Wallisauthor=R.+Wheelerauthor=A.+Williamsauthor=Y.+I.+Wolfauthor=K.+H.+Wolfeauthor=S.+P.+Yangauthor=R.+F.+Yehauthor=F.+Collinsauthor=M.+S.+Guyerauthor=J.+Petersonauthor=A.+Felsenfeldauthor=K.+A.+Wetterstrandauthor=A.+Patrinosauthor=M.+J.+Morganauthor=P.+de+Jongauthor=J.+J.+Cataneseauthor=K.+Osoegawaauthor=H.+Shizuyaauthor=S.+Choiauthor=Y.+J.+Chenauthor=J.+Szustakowki&title=Initial+sequencing+and+analysis+of+the+human+genome&doi=10.1038%2F35057062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Initial sequencing and analysis of the human genome</span></div><div class="casAuthors">Lander, Eric S.; Linton, Lauren M.; Birren, Bruce; Nusbaum, Chad; Zody, Michael C.; Baldwin, Jennifer; Devon, Keri; Dewar, Ken; Doyle, Michael; FitzHugh, William; Funke, Roel; Gage, Diane; Harris, Katrina; Heaford, Andrew; Howland, John; Kann, Lisa; Lehoczky, Jessica; LeVine, Rosie; McEwan, Paul; McKernan, Kevin; Meldrim, James; Mesirov, Jill P.; Miranda, Cher; Morris, William; Naylor, Jerome; Raymond, Christina; Rosetti, Mark; Santos, Ralph; Sheridan, Andrew; Sougnez, Carrie; Stange-Thomann, Nicole; Stojanovic, Nikola; Subramanian, Aravind; Wyman, Dudley; Sulston, John; Ainscough, Rachael; Beck, Stephan; Bentley, David; Burton, John; Clee, Christopher; Carter, Nigel; Coulson, Alan; Deadman, Rebecca; Deloukas, Panos; Dunham, Andrew; Dunham, Ian; Durbin, Richard; French, Lisa; Grafham, Darren; Gregory, Simon; Hubbard, Tim; Humphray, Sean; Hunt, Adrienne; Jones, Matthew; Lloyd, Christine; McMurray, Amanda; Matthews, Lucy; Mercer, Simon; Milne, Sarah; Mullikin, James C.; Mungall, Andrew; Plumb, Robert; Ross, Mark; Shownkeen, Ratna; Sims, Sarah; Waterston, Robert H.; Wilson, Richard K.; Hillier, LaDeana W.; McPherson, John D.; Marra, Marco A.; Mardis, Elaine R.; Fulton, Lucinda A.; Chinwalla, Asif T.; Pepin, Kymberlie H.; Gish, Warren R.; Chissoe, Stephanie L.; Wendl, Michael C.; Delehaunty, Kim D.; Miner, Tracie L.; Delehaunty, Andrew; Kramer, Jason B.; Cook, Lisa L.; Fulton, Robert S.; Johnson, Douglas L.; Minx, Patrick J.; Clifton, Sandra W.; Harkins, Trevor; Branscomb, Elbert; Predki, Paul; Richardson, Paul; Wenning, Sarah; Slezak, Tom; Doggett, Norman; Cheng, Jan-Fang; Olsen, Anne; Lucas, Susan; Elkin, Christopher; Uberbacher, Edward; Frazier, Marvin; Gibbs, Richard A.; Muzny, Donna M.; Scherer, Steven E.; Bouck, John B.; Sodergren, Erica J.; Worley, Kim C.; Rives, Catherine M.; Gorrell, James H.; Metzker, Michael L.; Naylor, Susan L.; Kurcherlapati, Raju S.; Nelson, David L.; Weinstock, George M.; Sakaki, Yoshiyuki; Fujiyama, Asao; Hattori, Masahira; Yada, Tetsushi; Toyoda, Atsushi; Itoh, Takehiko; Kawagoe, Chiharu; Watanabe, Hidemi; Totoki, Yasushi; Taylor, Todd; Weissenbach, Jean; Heilig, Roland; Saurin, William; Artiguenave, Francois; Brottier, Philippe; Bruls, Thomas; Pelletier, Eric; Robert, Catherine; Wincker, Patrick; Smith, Douglas R.; Doucette-Stamm, Lynn; Rubenfield, Marc; Weinstock, Keith; Lee, Hong Mei; Dubois, JoAnn; Rosenthal, Andre; Platzer, Matthias; Nyakatura, Gerald; Taudien, Stefan; Rump, Andreas; Yang, Huanming; Yu, Jun; Wang, Jian; Huang, Guyang; Gu, Ju; Hood, Leroy; Rowen, Lee; Madan, Anup; Qin, Shizen; Davis, Ronald W.; Federspiel, Nancy A.; Pia Abola, A.; Proctor, Michael J.; Schmutz, Jeremy; Dickson, Mark; Grimwood, Jane; Cox, David R.; Olson, Maynard V.; Kaul, Rajinder; Raymond, Christopher; Shimizu, Nobuyoshi; Kawasaki, Kazuhiko; Minoshima, Shinsei; Evans, Glen A.; Athanasiou, Maria; Schutlz, Roger; Roe, Bruce A.; Chen, Feng; Pan, Hauqin; Ramser, Juilane; Lehrach, Hans; Reinhardt, Richard; McCombie, W. Richard; de la Bastide, Melissa; Dedhia, Neilay; Blocker, Helmut; Hornischer, Klaus; Nordsiek, Gabriele; Agarwala, Richa; Aravind, L.; Bailey, Jeffrey A.; Bateman, Alex; Batzoglou, Serafim; Birney, Ewan; Bork, Peer; Brown, Daniel G.; Burge, Christopher B.; Cerutti, Lorenzo; Chen, Hsiu-Chuan; Church, Deanna; Clamp, Michele; Copley, Richard R.; Doerks, Tobias; Eddy, Sean R.; Eichler, Evan E.; Furey, Terrence S.; Galagan, James; Gilbert, James G. R.; Harmon, Cyrus; Hayashizaki, Yoshihide; Haussler, David; Hermjakob, Henning; Hokamp, Karsten; Jang, Wonhee; Johnson, L. Steven; Jones, Thomas A.; Kasit, Simon; Kaspryzk, Arek; Kennedy, Scot; Kent, W. James; Kitts, Paul; Koonin, Eugene V.; Kort, Ian; Kulp, David; Lancet, Doron; Lowe, Todd M.; McLysaght, Aoife; Mikkelsen, Tarjei; Moran, John V.; Mulder, Nicola; Pollara, Victor J.; Ponting, Chris P.; Schuler, Greg; Schultz, Jorg; Slater, Guy; Smit, Arian; Stupka, Eila; Szustakowki, Joseph; Thierry-Mieg, Danielle; Thierry-Mieg, Jean; Wagner, Lukas; Wallis, John; Wheeler, Raymond; Williams, Alan; Wolf, Yuri I.; Wolfe, Kenneth H.; Yang, Shiaw-Pyng; Yeh, Ru-Fang; Collins, Francis; Guyer, Mark S.; Peterson, Jane; Felsenfeld, Adam; Wetterstrand, Kris A.; Patrinos, Aristides; Morgan, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">409</span>
        (<span class="NLM_cas:issue">6822</span>),
    <span class="NLM_cas:pages">860-921</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The human genome holds an extraordinary trove of information about human development, physiol., medicine, and evolution.  The results of an international collaboration to produce and make freely available a draft sequence of the human genome are now reported.  An initial anal. of the data is also presented, describing some of the insights that can be gleaned from the sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBLwD5_sOlWrVg90H21EOLACvtfcHk0lgtngc3k25COw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhsFCjtLc%253D&md5=b5554a573a36709f185f5e23bb46ad1f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F35057062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35057062%26sid%3Dliteratum%253Aachs%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DLinton%26aufirst%3DL.%2BM.%26aulast%3DBirren%26aufirst%3DB.%26aulast%3DNusbaum%26aufirst%3DC.%26aulast%3DZody%26aufirst%3DM.%2BC.%26aulast%3DBaldwin%26aufirst%3DJ.%26aulast%3DDevon%26aufirst%3DK.%26aulast%3DDewar%26aufirst%3DK.%26aulast%3DDoyle%26aufirst%3DM.%26aulast%3DFitzHugh%26aufirst%3DW.%26aulast%3DFunke%26aufirst%3DR.%26aulast%3DGage%26aufirst%3DD.%26aulast%3DHarris%26aufirst%3DK.%26aulast%3DHeaford%26aufirst%3DA.%26aulast%3DHowland%26aufirst%3DJ.%26aulast%3DKann%26aufirst%3DL.%26aulast%3DLehoczky%26aufirst%3DJ.%26aulast%3DLeVine%26aufirst%3DR.%26aulast%3DMcEwan%26aufirst%3DP.%26aulast%3DMcKernan%26aufirst%3DK.%26aulast%3DMeldrim%26aufirst%3DJ.%26aulast%3DMesirov%26aufirst%3DJ.%2BP.%26aulast%3DMiranda%26aufirst%3DC.%26aulast%3DMorris%26aufirst%3DW.%26aulast%3DNaylor%26aufirst%3DJ.%26aulast%3DRaymond%26aufirst%3DC.%26aulast%3DRosetti%26aufirst%3DM.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DSheridan%26aufirst%3DA.%26aulast%3DSougnez%26aufirst%3DC.%26aulast%3DStange-Thomann%26aufirst%3DY.%26aulast%3DStojanovic%26aufirst%3DN.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DWyman%26aufirst%3DD.%26aulast%3DRogers%26aufirst%3DJ.%26aulast%3DSulston%26aufirst%3DJ.%26aulast%3DAinscough%26aufirst%3DR.%26aulast%3DBeck%26aufirst%3DS.%26aulast%3DBentley%26aufirst%3DD.%26aulast%3DBurton%26aufirst%3DJ.%26aulast%3DClee%26aufirst%3DC.%26aulast%3DCarter%26aufirst%3DN.%26aulast%3DCoulson%26aufirst%3DA.%26aulast%3DDeadman%26aufirst%3DR.%26aulast%3DDeloukas%26aufirst%3DP.%26aulast%3DDunham%26aufirst%3DA.%26aulast%3DDunham%26aufirst%3DI.%26aulast%3DDurbin%26aufirst%3DR.%26aulast%3DFrench%26aufirst%3DL.%26aulast%3DGrafham%26aufirst%3DD.%26aulast%3DGregory%26aufirst%3DS.%26aulast%3DHubbard%26aufirst%3DT.%26aulast%3DHumphray%26aufirst%3DS.%26aulast%3DHunt%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DLloyd%26aufirst%3DC.%26aulast%3DMcMurray%26aufirst%3DA.%26aulast%3DMatthews%26aufirst%3DL.%26aulast%3DMercer%26aufirst%3DS.%26aulast%3DMilne%26aufirst%3DS.%26aulast%3DMullikin%26aufirst%3DJ.%2BC.%26aulast%3DMungall%26aufirst%3DA.%26aulast%3DPlumb%26aufirst%3DR.%26aulast%3DRoss%26aufirst%3DM.%26aulast%3DShownkeen%26aufirst%3DR.%26aulast%3DSims%26aufirst%3DS.%26aulast%3DWaterston%26aufirst%3DR.%2BH.%26aulast%3DWilson%26aufirst%3DR.%2BK.%26aulast%3DHillier%26aufirst%3DL.%2BW.%26aulast%3DMcPherson%26aufirst%3DJ.%2BD.%26aulast%3DMarra%26aufirst%3DM.%2BA.%26aulast%3DMardis%26aufirst%3DE.%2BR.%26aulast%3DFulton%26aufirst%3DL.%2BA.%26aulast%3DChinwalla%26aufirst%3DA.%2BT.%26aulast%3DPepin%26aufirst%3DK.%2BH.%26aulast%3DGish%26aufirst%3DW.%2BR.%26aulast%3DChissoe%26aufirst%3DS.%2BL.%26aulast%3DWendl%26aufirst%3DM.%2BC.%26aulast%3DDelehaunty%26aufirst%3DK.%2BD.%26aulast%3DMiner%26aufirst%3DT.%2BL.%26aulast%3DDelehaunty%26aufirst%3DA.%26aulast%3DKramer%26aufirst%3DJ.%2BB.%26aulast%3DCook%26aufirst%3DL.%2BL.%26aulast%3DFulton%26aufirst%3DR.%2BS.%26aulast%3DJohnson%26aufirst%3DD.%2BL.%26aulast%3DMinx%26aufirst%3DP.%2BJ.%26aulast%3DClifton%26aufirst%3DS.%2BW.%26aulast%3DHawkins%26aufirst%3DT.%26aulast%3DBranscomb%26aufirst%3DE.%26aulast%3DPredki%26aufirst%3DP.%26aulast%3DRichardson%26aufirst%3DP.%26aulast%3DWenning%26aufirst%3DS.%26aulast%3DSlezak%26aufirst%3DT.%26aulast%3DDoggett%26aufirst%3DN.%26aulast%3DCheng%26aufirst%3DJ.%2BF.%26aulast%3DOlsen%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DS.%26aulast%3DElkin%26aufirst%3DC.%26aulast%3DUberbacher%26aufirst%3DE.%26aulast%3DFrazier%26aufirst%3DM.%26aulast%3DGibbs%26aufirst%3DR.%2BA.%26aulast%3DMuzny%26aufirst%3DD.%2BM.%26aulast%3DScherer%26aufirst%3DS.%2BE.%26aulast%3DBouck%26aufirst%3DJ.%2BB.%26aulast%3DSodergren%26aufirst%3DE.%2BJ.%26aulast%3DWorley%26aufirst%3DK.%2BC.%26aulast%3DRives%26aufirst%3DC.%2BM.%26aulast%3DGorrell%26aufirst%3DJ.%2BH.%26aulast%3DMetzker%26aufirst%3DM.%2BL.%26aulast%3DNaylor%26aufirst%3DS.%2BL.%26aulast%3DKucherlapati%26aufirst%3DR.%2BS.%26aulast%3DNelson%26aufirst%3DD.%2BL.%26aulast%3DWeinstock%26aufirst%3DG.%2BM.%26aulast%3DSakaki%26aufirst%3DY.%26aulast%3DFujiyama%26aufirst%3DA.%26aulast%3DHattori%26aufirst%3DM.%26aulast%3DYada%26aufirst%3DT.%26aulast%3DToyoda%26aufirst%3DA.%26aulast%3DItoh%26aufirst%3DT.%26aulast%3DKawagoe%26aufirst%3DC.%26aulast%3DWatanabe%26aufirst%3DH.%26aulast%3DTotoki%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DT.%26aulast%3DWeissenbach%26aufirst%3DJ.%26aulast%3DHeilig%26aufirst%3DR.%26aulast%3DSaurin%26aufirst%3DW.%26aulast%3DArtiguenave%26aufirst%3DF.%26aulast%3DBrottier%26aufirst%3DP.%26aulast%3DBruls%26aufirst%3DT.%26aulast%3DPelletier%26aufirst%3DE.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DWincker%26aufirst%3DP.%26aulast%3DSmith%26aufirst%3DD.%2BR.%26aulast%3DDoucette-Stamm%26aufirst%3DL.%26aulast%3DRubenfield%26aufirst%3DM.%26aulast%3DWeinstock%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DH.%2BM.%26aulast%3DDubois%26aufirst%3DJ.%26aulast%3DRosenthal%26aufirst%3DA.%26aulast%3DPlatzer%26aufirst%3DM.%26aulast%3DNyakatura%26aufirst%3DG.%26aulast%3DTaudien%26aufirst%3DS.%26aulast%3DRump%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DHood%26aufirst%3DL.%26aulast%3DRowen%26aufirst%3DL.%26aulast%3DMadan%26aufirst%3DA.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DDavis%26aufirst%3DR.%2BW.%26aulast%3DFederspiel%26aufirst%3DN.%2BA.%26aulast%3DAbola%26aufirst%3DA.%2BP.%26aulast%3DProctor%26aufirst%3DM.%2BJ.%26aulast%3DMyers%26aufirst%3DR.%2BM.%26aulast%3DSchmutz%26aufirst%3DJ.%26aulast%3DDickson%26aufirst%3DM.%26aulast%3DGrimwood%26aufirst%3DJ.%26aulast%3DCox%26aufirst%3DD.%2BR.%26aulast%3DOlson%26aufirst%3DM.%2BV.%26aulast%3DKaul%26aufirst%3DR.%26aulast%3DRaymond%26aufirst%3DC.%26aulast%3DShimizu%26aufirst%3DN.%26aulast%3DKawasaki%26aufirst%3DK.%26aulast%3DMinoshima%26aufirst%3DS.%26aulast%3DEvans%26aufirst%3DG.%2BA.%26aulast%3DAthanasiou%26aufirst%3DM.%26aulast%3DSchultz%26aufirst%3DR.%26aulast%3DRoe%26aufirst%3DB.%2BA.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DRamser%26aufirst%3DJ.%26aulast%3DLehrach%26aufirst%3DH.%26aulast%3DReinhardt%26aufirst%3DR.%26aulast%3DMcCombie%26aufirst%3DW.%2BR.%26aulast%3Dde%2Bla%2BBastide%26aufirst%3DM.%26aulast%3DDedhia%26aufirst%3DN.%26aulast%3DBlocker%26aufirst%3DH.%26aulast%3DHornischer%26aufirst%3DK.%26aulast%3DNordsiek%26aufirst%3DG.%26aulast%3DAgarwala%26aufirst%3DR.%26aulast%3DAravind%26aufirst%3DL.%26aulast%3DBailey%26aufirst%3DJ.%2BA.%26aulast%3DBateman%26aufirst%3DA.%26aulast%3DBatzoglou%26aufirst%3DS.%26aulast%3DBirney%26aufirst%3DE.%26aulast%3DBork%26aufirst%3DP.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DBurge%26aufirst%3DC.%2BB.%26aulast%3DCerutti%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DH.%2BC.%26aulast%3DChurch%26aufirst%3DD.%26aulast%3DClamp%26aufirst%3DM.%26aulast%3DCopley%26aufirst%3DR.%2BR.%26aulast%3DDoerks%26aufirst%3DT.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26aulast%3DEichler%26aufirst%3DE.%2BE.%26aulast%3DFurey%26aufirst%3DT.%2BS.%26aulast%3DGalagan%26aufirst%3DJ.%26aulast%3DGilbert%26aufirst%3DJ.%2BG.%26aulast%3DHarmon%26aufirst%3DC.%26aulast%3DHayashizaki%26aufirst%3DY.%26aulast%3DHaussler%26aufirst%3DD.%26aulast%3DHermjakob%26aufirst%3DH.%26aulast%3DHokamp%26aufirst%3DK.%26aulast%3DJang%26aufirst%3DW.%26aulast%3DJohnson%26aufirst%3DL.%2BS.%26aulast%3DJones%26aufirst%3DT.%2BA.%26aulast%3DKasif%26aufirst%3DS.%26aulast%3DKaspryzk%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DKent%26aufirst%3DW.%2BJ.%26aulast%3DKitts%26aufirst%3DP.%26aulast%3DKoonin%26aufirst%3DE.%2BV.%26aulast%3DKorf%26aufirst%3DI.%26aulast%3DKulp%26aufirst%3DD.%26aulast%3DLancet%26aufirst%3DD.%26aulast%3DLowe%26aufirst%3DT.%2BM.%26aulast%3DMcLysaght%26aufirst%3DA.%26aulast%3DMikkelsen%26aufirst%3DT.%26aulast%3DMoran%26aufirst%3DJ.%2BV.%26aulast%3DMulder%26aufirst%3DN.%26aulast%3DPollara%26aufirst%3DV.%2BJ.%26aulast%3DPonting%26aufirst%3DC.%2BP.%26aulast%3DSchuler%26aufirst%3DG.%26aulast%3DSchultz%26aufirst%3DJ.%26aulast%3DSlater%26aufirst%3DG.%26aulast%3DSmit%26aufirst%3DA.%2BF.%26aulast%3DStupka%26aufirst%3DE.%26aulast%3DSzustakowki%26aufirst%3DJ.%26aulast%3DThierry-Mieg%26aufirst%3DD.%26aulast%3DThierry-Mieg%26aufirst%3DJ.%26aulast%3DWagner%26aufirst%3DL.%26aulast%3DWallis%26aufirst%3DJ.%26aulast%3DWheeler%26aufirst%3DR.%26aulast%3DWilliams%26aufirst%3DA.%26aulast%3DWolf%26aufirst%3DY.%2BI.%26aulast%3DWolfe%26aufirst%3DK.%2BH.%26aulast%3DYang%26aufirst%3DS.%2BP.%26aulast%3DYeh%26aufirst%3DR.%2BF.%26aulast%3DCollins%26aufirst%3DF.%26aulast%3DGuyer%26aufirst%3DM.%2BS.%26aulast%3DPeterson%26aufirst%3DJ.%26aulast%3DFelsenfeld%26aufirst%3DA.%26aulast%3DWetterstrand%26aufirst%3DK.%2BA.%26aulast%3DPatrinos%26aufirst%3DA.%26aulast%3DMorgan%26aufirst%3DM.%2BJ.%26aulast%3Dde%2BJong%26aufirst%3DP.%26aulast%3DCatanese%26aufirst%3DJ.%2BJ.%26aulast%3DOsoegawa%26aufirst%3DK.%26aulast%3DShizuya%26aufirst%3DH.%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%2BJ.%26aulast%3DSzustakowki%26aufirst%3DJ.%26atitle%3DInitial%2520sequencing%2520and%2520analysis%2520of%2520the%2520human%2520genome%26jtitle%3DNature%26date%3D2001%26volume%3D409%26spage%3D860%26epage%3D921%26doi%3D10.1038%2F35057062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, N. G.</span></span> <span> </span><span class="NLM_article-title">Proteome and proteomics: new technologies, new concepts, and new words</span>. <i>Electrophoresis</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1853</span>– <span class="NLM_lpage">1861</span>, <span class="refDoi"> DOI: 10.1002/elps.1150191103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1002%2Felps.1150191103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=9740045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaK1cXmtlGrt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1998&pages=1853-1861&author=N.+L.+Andersonauthor=N.+G.+Anderson&title=Proteome+and+proteomics%3A+new+technologies%2C+new+concepts%2C+and+new+words&doi=10.1002%2Felps.1150191103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome and proteomics. New technologies, new concepts, and new words</span></div><div class="casAuthors">Anderson, N. Leigh; Anderson, Norman G.</div><div class="citationInfo"><span class="NLM_cas:title">Electrophoresis</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1853-1861</span>CODEN:
                <span class="NLM_cas:coden">ELCTDN</span>;
        ISSN:<span class="NLM_cas:issn">0173-0835</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">A review with 41 refs.  The goal of proteomics is a comprehensive, quant. description of protein expression and its changes under the influence of biol. perturbations such as disease or drug treatment.  Quant. anal. of protein expression data obtained by high-throughput methods has led us to define the concept of regulatory homol. and use it to begin to elucidate the basic structure of gene expression control in vivo.  Such investigations lay the groundwork for construction of comprehensive databases of mechanisms (cataloguing possible biol. outcomes), the next logical step after the soon to be completed cataloguing of genes and gene products.  Mechanism databases provide a roadmap towards effective therapeutic intervention that is more direct than that offered by conventional genomics approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqkBUnV8ugtbVg90H21EOLACvtfcHk0ljX09a78GWqTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmtlGrt7o%253D&md5=f401c7bf42c69a996722b7c5deb78bb7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Felps.1150191103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Felps.1150191103%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DN.%2BL.%26aulast%3DAnderson%26aufirst%3DN.%2BG.%26atitle%3DProteome%2520and%2520proteomics%253A%2520new%2520technologies%252C%2520new%2520concepts%252C%2520and%2520new%2520words%26jtitle%3DElectrophoresis%26date%3D1998%26volume%3D19%26spage%3D1853%26epage%3D1861%26doi%3D10.1002%2Felps.1150191103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bertozzi, C. R.</span>; <span class="NLM_string-name">Sasisekharan, R.</span></span> <span> </span><span class="NLM_article-title">Glycomics</span>. In  <i>Essentials of Glycobiology</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Varki, A.</span>, <span class="NLM_string-name">Cummings, R. D.</span>, <span class="NLM_string-name">Esko, J. D.</span>, <span class="NLM_string-name">Freeze, H. H.</span>, <span class="NLM_string-name">Stanley, P.</span>, <span class="NLM_string-name">Bertozzi, C. R.</span>, <span class="NLM_string-name">Hart, G. W.</span>, <span class="NLM_string-name">Etzler, M. E.</span></span>, Eds.; <span class="NLM_publisher-name">Cold Spring Harbor Laboratory Press</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=C.+R.+Bertozzi&author=R.+Sasisekharanauthor=A.+Varki&author=R.+D.+Cummings&author=J.+D.+Esko&author=H.+H.+Freeze&author=P.+Stanley&author=C.+R.+Bertozzi&author=G.+W.+Hart&author=M.+E.+Etzler&title=Essentials+of+Glycobiology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBertozzi%26aufirst%3DC.%2BR.%26atitle%3DGlycomics%26btitle%3DEssentials%2520of%2520Glycobiology%26aulast%3DVarki%26aufirst%3DA.%26pub%3DCold%2520Spring%2520Harbor%2520Laboratory%2520Press%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ejsing, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampaio, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surendranath, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duchoslav, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekroos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klemm, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shevchenko, A.</span></span> <span> </span><span class="NLM_article-title">Global analysis of the yeast lipidome by quantitative shotgun mass spectrometry</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">2136</span>– <span class="NLM_lpage">2141</span>, <span class="refDoi"> DOI: 10.1073/pnas.0811700106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.0811700106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19174513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlaqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=2136-2141&author=C.+S.+Ejsingauthor=J.+L.+Sampaioauthor=V.+Surendranathauthor=E.+Duchoslavauthor=K.+Ekroosauthor=R.+W.+Klemmauthor=K.+Simonsauthor=A.+Shevchenko&title=Global+analysis+of+the+yeast+lipidome+by+quantitative+shotgun+mass+spectrometry&doi=10.1073%2Fpnas.0811700106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Global analysis of the yeast lipidome by quantitative shotgun mass spectrometry</span></div><div class="casAuthors">Ejsing, Christer S.; Sampaio, Julio L.; Surendranath, Vineeth; Duchoslav, Eva; Ekroos, Kim; Klemm, Robin W.; Simons, Kai; Shevchenko, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2136-2141</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Although the transcriptome, proteome, and interactome of several eukaryotic model organisms have been described in detail, lipidomes remain relatively uncharacterized.  Using Saccharomyces cerevisiae as an example, the authors demonstrate that automated shotgun lipidomics anal. enabled lipidome-wide abs. quantification of individual mol. lipid species by streamlined processing of a single sample of only 2 million yeast cells.  By comparative lipidomics, the authors achieved the abs. quantification of 250 mol. lipid species covering 21 major lipid classes.  This anal. provided ≈95% coverage of the yeast lipidome achieved with 125-fold improvement in sensitivity compared with previous approaches.  Comparative lipidomics demonstrated that growth temp. and defects in lipid biosynthesis induce ripple effects throughout the mol. compn. of the yeast lipidome.  This work serves as a resource for mol. characterization of eukaryotic lipidomes, and establishes shotgun lipidomics as a powerful platform for complementing biochem. studies and other systems-level approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxNQmv7RPMGbVg90H21EOLACvtfcHk0lhtHQi2IsrCiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlaqtr4%253D&md5=07d1c452da5f08731ade905cde007e80</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0811700106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0811700106%26sid%3Dliteratum%253Aachs%26aulast%3DEjsing%26aufirst%3DC.%2BS.%26aulast%3DSampaio%26aufirst%3DJ.%2BL.%26aulast%3DSurendranath%26aufirst%3DV.%26aulast%3DDuchoslav%26aufirst%3DE.%26aulast%3DEkroos%26aufirst%3DK.%26aulast%3DKlemm%26aufirst%3DR.%2BW.%26aulast%3DSimons%26aufirst%3DK.%26aulast%3DShevchenko%26aufirst%3DA.%26atitle%3DGlobal%2520analysis%2520of%2520the%2520yeast%2520lipidome%2520by%2520quantitative%2520shotgun%2520mass%2520spectrometry%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D2136%26epage%3D2141%26doi%3D10.1073%2Fpnas.0811700106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGettigan, P. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptomics in the RNA-seq era</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2012.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.cbpa.2012.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=23290152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2013&pages=4-11&author=P.+A.+McGettigan&title=Transcriptomics+in+the+RNA-seq+era&doi=10.1016%2Fj.cbpa.2012.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptomics in the RNA-seq era</span></div><div class="casAuthors">McGettigan, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-11</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The transcriptomics field has developed rapidly with the advent of next-generation sequencing technologies.  RNA-seq has now displaced microarrays as the preferred method for gene expression profiling.The comprehensive nature of the data generated has been a boon in terms of transcript identification but anal. challenges remain.  Key among these problems is the development of suitable expression metrics for expression level comparisons and methods for identification of differentially expressed genes (and exons).  Several approaches were developed but as yet no consensus exists on the best pipeline to use.  De novo transcriptome approaches are increasingly viable for organisms lacking a sequenced genome.  The redn. in starting RNA required has enabled the development of new applications such as single cell transcriptomics.  The emerging picture of mammalian transcription is complex with further refinement expected with the integration of epigenomic data generated by projects such as ENCODE.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgBaCUjBafcLVg90H21EOLACvtfcHk0lhtHQi2IsrCiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWqsw%253D%253D&md5=8249af313bf4f611dec1d167ef6c7097</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2012.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2012.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DMcGettigan%26aufirst%3DP.%2BA.%26atitle%3DTranscriptomics%2520in%2520the%2520RNA-seq%2520era%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D17%26spage%3D4%26epage%3D11%26doi%3D10.1016%2Fj.cbpa.2012.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adelstein, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassis, A. I.</span></span> <span> </span><span class="NLM_article-title">Target discovery from data mining approaches</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">S16</span>– <span class="NLM_lpage">S23</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.drudis.2011.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=22178890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns12htQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=S16-S23&author=Y.+Yangauthor=S.+J.+Adelsteinauthor=A.+I.+Kassis&title=Target+discovery+from+data+mining+approaches&doi=10.1016%2Fj.drudis.2011.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Target discovery from data mining approaches</span></div><div class="casAuthors">Yang, Yongliang; Adelstein, S. James; Kassis, Amin I.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">Suppl.</span>),
    <span class="NLM_cas:pages">S16-S23</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Data mining of available biomedical data and information has greatly boosted target discovery in the omics' era.  Target discovery is the key step in the biomarker and drug discovery pipeline to diagnose and fight human diseases.  In biomedical science, the target' is a broad concept ranging from mol. entities (such as genes, proteins and miRNAs) to biol. phenomena (such as mol. functions, pathways and phenotypes).  Within the context of biomedical science, data mining refers to a bioinformatics approach that combines biol. concepts with computer tools or statistical methods that are mainly used to discover, select and prioritize targets.  In response to the huge demand of data mining for target discovery in the omics' era, this review explicates various data mining approaches and their applications to target discovery with emphasis on text and microarray data anal.  Two emerging data mining approaches, chemogenomic data mining and proteomic data mining, are briefly introduced.  Also discussed are the limitations of various data mining approaches found in the level of database integration, the quality of data annotation, sample heterogeneity and the performance of anal. and mining tools.  Tentative strategies of integrating different data sources for target discovery, such as integrated text mining with high-throughput data anal. and integrated mining with pathway databases, are introduced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhbUNWD288SrVg90H21EOLACvtfcHk0lhtHQi2IsrCiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns12htQ%253D%253D&md5=329803c354df96e8b274e3e3bdc716e3</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DAdelstein%26aufirst%3DS.%2BJ.%26aulast%3DKassis%26aufirst%3DA.%2BI.%26atitle%3DTarget%2520discovery%2520from%2520data%2520mining%2520approaches%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26spage%3DS16%26epage%3DS23%26doi%3D10.1016%2Fj.drudis.2011.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaughn, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McManus, M. T.</span></span> <span> </span><span class="NLM_article-title">Pervasive transcription of the human genome produces thousands of previously unidentified long intergenic noncoding RNAs</span>. <i>PLoS Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e1003569</span> <span class="refDoi"> DOI: 10.1371/journal.pgen.1003569</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1371%2Fjournal.pgen.1003569" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=23818866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFersLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&author=M.+J.+Hangauerauthor=I.+W.+Vaughnauthor=M.+T.+McManus&title=Pervasive+transcription+of+the+human+genome+produces+thousands+of+previously+unidentified+long+intergenic+noncoding+RNAs&doi=10.1371%2Fjournal.pgen.1003569"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Pervasive transcription of the human genome produces thousands of previously unidentified long intergenic noncoding RNAs</span></div><div class="casAuthors">Hangauer, Matthew J.; Vaughn, Ian W.; McManus, Michael T.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1003569</span>CODEN:
                <span class="NLM_cas:coden">PGLEB5</span>;
        ISSN:<span class="NLM_cas:issn">1553-7404</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Known protein coding gene exons compose less than 3% of the human genome.  The remaining 97% is largely uncharted territory, with only a small fraction characterized.  The recent observation of transcription in this intergenic territory has stimulated debate about the extent of intergenic transcription and whether these intergenic RNAs are functional.  Here the authors directly obsd. with a large set of RNA-seq data covering a wide array of human tissue types that the majority of the genome is indeed transcribed, corroborating recent observations by the ENCODE project.  Furthermore, using de novo transcriptome assembly of this RNA-seq data, the authors found that intergenic regions encode far more long intergenic noncoding RNAs (lincRNAs) than previously described, helping to resolve the discrepancy between the vast amt. of obsd. intergenic transcription and the limited no. of previously known lincRNAs.  In total, the authors identified tens of thousands of putative lincRNAs expressed at a min. of one copy per cell, significantly expanding upon prior lincRNA annotation sets.  These lincRNAs are specifically regulated and conserved rather than being the product of transcriptional noise.  In addn., lincRNAs are strongly enriched for trait-assocd. SNPs suggesting a new mechanism by which intergenic trait-assocd. regions may function.  These findings will enable the discovery and interrogation of novel intergenic functional elements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu0Xih8vOO7Vg90H21EOLACvtfcHk0lgruN7OnK781Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFersLvL&md5=843cea216c8f8397e22e49c9616aa1cf</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1003569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1003569%26sid%3Dliteratum%253Aachs%26aulast%3DHangauer%26aufirst%3DM.%2BJ.%26aulast%3DVaughn%26aufirst%3DI.%2BW.%26aulast%3DMcManus%26aufirst%3DM.%2BT.%26atitle%3DPervasive%2520transcription%2520of%2520the%2520human%2520genome%2520produces%2520thousands%2520of%2520previously%2520unidentified%2520long%2520intergenic%2520noncoding%2520RNAs%26jtitle%3DPLoS%2520Genet.%26date%3D2013%26volume%3D9%26doi%3D10.1371%2Fjournal.pgen.1003569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Djebali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortazavi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagarde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlesinger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinov, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khatun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaleski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozowsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Röder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokocinski, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdelhamid, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alioto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antoshechkin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baer, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bar, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batut, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabortty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrast, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derrien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drenkow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumais, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumais, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duttagupta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falconnet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fastuca, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fejes-Toth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foissac, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fullwood, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunawardena, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howald, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapranov, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingswood, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persaud, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preall, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeca, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risk, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robyr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sammeth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">See, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahab, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skancke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tilgner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trout, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrobel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashizaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonarakis, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giddings, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carninci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guigó, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingeras, T. R.</span></span> <span> </span><span class="NLM_article-title">Landscape of transcription in human cells</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>489</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">108</span>, <span class="refDoi"> DOI: 10.1038/nature11233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnature11233" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=22955620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlGnt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=101-108&author=S.+Djebaliauthor=C.+A.+Davisauthor=A.+Merkelauthor=A.+Dobinauthor=T.+Lassmannauthor=A.+Mortazaviauthor=A.+Tanzerauthor=J.+Lagardeauthor=W.+Linauthor=F.+Schlesingerauthor=C.+Xueauthor=G.+K.+Marinovauthor=J.+Khatunauthor=B.+A.+Williamsauthor=C.+Zaleskiauthor=J.+Rozowskyauthor=M.+R%C3%B6derauthor=F.+Kokocinskiauthor=R.+F.+Abdelhamidauthor=T.+Aliotoauthor=I.+Antoshechkinauthor=M.+T.+Baerauthor=N.+S.+Barauthor=P.+Batutauthor=K.+Bellauthor=I.+Bellauthor=S.+Chakraborttyauthor=X.+Chenauthor=J.+Chrastauthor=J.+Curadoauthor=T.+Derrienauthor=J.+Drenkowauthor=E.+Dumaisauthor=J.+Dumaisauthor=R.+Duttaguptaauthor=E.+Falconnetauthor=M.+Fastucaauthor=K.+Fejes-Tothauthor=P.+Ferreiraauthor=S.+Foissacauthor=M.+J.+Fullwoodauthor=H.+Gaoauthor=D.+Gonzalezauthor=A.+Gordonauthor=H.+Gunawardenaauthor=C.+Howaldauthor=S.+Jhaauthor=R.+Johnsonauthor=P.+Kapranovauthor=B.+Kingauthor=C.+Kingswoodauthor=O.+J.+Luoauthor=E.+Parkauthor=K.+Persaudauthor=J.+B.+Preallauthor=P.+Ribecaauthor=B.+Riskauthor=D.+Robyrauthor=M.+Sammethauthor=L.+Schafferauthor=L.-H.+Seeauthor=A.+Shahabauthor=J.+Skanckeauthor=A.+M.+Suzukiauthor=H.+Takahashiauthor=H.+Tilgnerauthor=D.+Troutauthor=N.+Waltersauthor=H.+Wangauthor=J.+Wrobelauthor=Y.+Yuauthor=X.+Ruanauthor=Y.+Hayashizakiauthor=J.+Harrowauthor=M.+Gersteinauthor=T.+Hubbardauthor=A.+Reymondauthor=S.+E.+Antonarakisauthor=G.+Hannonauthor=M.+C.+Giddingsauthor=Y.+Ruanauthor=B.+Woldauthor=P.+Carninciauthor=R.+Guig%C3%B3author=T.+R.+Gingeras&title=Landscape+of+transcription+in+human+cells&doi=10.1038%2Fnature11233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Landscape of transcription in human cells</span></div><div class="casAuthors">Djebali, Sarah; Davis, Carrie A.; Merkel, Angelika; Dobin, Alex; Lassmann, Timo; Mortazavi, Ali; Tanzer, Andrea; Lagarde, Julien; Lin, Wei; Schlesinger, Felix; Xue, Chenghai; Marinov, Georgi K.; Khatun, Jainab; Williams, Brian A.; Zaleski, Chris; Rozowsky, Joel; Roeder, Maik; Kokocinski, Felix; Abdelhamid, Rehab F.; Alioto, Tyler; Antoshechkin, Igor; Baer, Michael T.; Bar, Nadav S.; Batut, Philippe; Bell, Kimberly; Bell, Ian; Chakrabortty, Sudipto; Chen, Xian; Chrast, Jacqueline; Curado, Joao; Derrien, Thomas; Drenkow, Jorg; Dumais, Erica; Dumais, Jacqueline; Duttagupta, Radha; Falconnet, Emilie; Fastuca, Meagan; Fejes-Toth, Kata; Ferreira, Pedro; Foissac, Sylvain; Fullwood, Melissa J.; Gao, Hui; Gonzalez, David; Gordon, Assaf; Gunawardena, Harsha; Howald, Cedric; Jha, Sonali; Johnson, Rory; Kapranov, Philipp; King, Brandon; Kingswood, Colin; Luo, Oscar J.; Park, Eddie; Persaud, Kimberly; Preall, Jonathan B.; Ribeca, Paolo; Risk, Brian; Robyr, Daniel; Sammeth, Michael; Schaffer, Lorian; See, Lei-Hoon; Shahab, Atif; Skancke, Jorgen; Suzuki, Ana Maria; Takahashi, Hazuki; Tilgner, Hagen; Trout, Diane; Walters, Nathalie; Wang, Huaien; Wrobel, John; Yu, Yanbao; Ruan, Xiaoan; Hayashizaki, Yoshihide; Harrow, Jennifer; Gerstein, Mark; Hubbard, Tim; Reymond, Alexandre; Antonarakis, Stylianos E.; Hannon, Gregory; Giddings, Morgan C.; Ruan, Yijun; Wold, Barbara; Carninci, Piero; Guigo, Roderic; Gingeras, Thomas R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">489</span>
        (<span class="NLM_cas:issue">7414</span>),
    <span class="NLM_cas:pages">101-108</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Eukaryotic cells make many types of primary and processed RNAs that are found either in specific subcellular compartments or throughout the cells.  A complete catalog of these RNAs is not yet available and their characteristic subcellular localizations are also poorly understood.  Because RNA represents the direct output of the genetic information encoded by genomes and a significant proportion of a cell's regulatory capabilities are focused on its synthesis, processing, transport, modification and translation, the generation of such a catalog is crucial for understanding genome function.  Here we report evidence that three-quarters of the human genome is capable of being transcribed, as well as observations about the range and levels of expression, localization, processing fates, regulatory regions and modifications of almost all currently annotated and thousands of previously unannotated RNAs.  These observations, taken together, prompt a redefinition of the concept of a gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQJgBaBiE8o7Vg90H21EOLACvtfcHk0lgruN7OnK781Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlGnt73M&md5=8f8ee2a1189ced1010dec5e45a305acf</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature11233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11233%26sid%3Dliteratum%253Aachs%26aulast%3DDjebali%26aufirst%3DS.%26aulast%3DDavis%26aufirst%3DC.%2BA.%26aulast%3DMerkel%26aufirst%3DA.%26aulast%3DDobin%26aufirst%3DA.%26aulast%3DLassmann%26aufirst%3DT.%26aulast%3DMortazavi%26aufirst%3DA.%26aulast%3DTanzer%26aufirst%3DA.%26aulast%3DLagarde%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DW.%26aulast%3DSchlesinger%26aufirst%3DF.%26aulast%3DXue%26aufirst%3DC.%26aulast%3DMarinov%26aufirst%3DG.%2BK.%26aulast%3DKhatun%26aufirst%3DJ.%26aulast%3DWilliams%26aufirst%3DB.%2BA.%26aulast%3DZaleski%26aufirst%3DC.%26aulast%3DRozowsky%26aufirst%3DJ.%26aulast%3DR%25C3%25B6der%26aufirst%3DM.%26aulast%3DKokocinski%26aufirst%3DF.%26aulast%3DAbdelhamid%26aufirst%3DR.%2BF.%26aulast%3DAlioto%26aufirst%3DT.%26aulast%3DAntoshechkin%26aufirst%3DI.%26aulast%3DBaer%26aufirst%3DM.%2BT.%26aulast%3DBar%26aufirst%3DN.%2BS.%26aulast%3DBatut%26aufirst%3DP.%26aulast%3DBell%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DI.%26aulast%3DChakrabortty%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChrast%26aufirst%3DJ.%26aulast%3DCurado%26aufirst%3DJ.%26aulast%3DDerrien%26aufirst%3DT.%26aulast%3DDrenkow%26aufirst%3DJ.%26aulast%3DDumais%26aufirst%3DE.%26aulast%3DDumais%26aufirst%3DJ.%26aulast%3DDuttagupta%26aufirst%3DR.%26aulast%3DFalconnet%26aufirst%3DE.%26aulast%3DFastuca%26aufirst%3DM.%26aulast%3DFejes-Toth%26aufirst%3DK.%26aulast%3DFerreira%26aufirst%3DP.%26aulast%3DFoissac%26aufirst%3DS.%26aulast%3DFullwood%26aufirst%3DM.%2BJ.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DGonzalez%26aufirst%3DD.%26aulast%3DGordon%26aufirst%3DA.%26aulast%3DGunawardena%26aufirst%3DH.%26aulast%3DHowald%26aufirst%3DC.%26aulast%3DJha%26aufirst%3DS.%26aulast%3DJohnson%26aufirst%3DR.%26aulast%3DKapranov%26aufirst%3DP.%26aulast%3DKing%26aufirst%3DB.%26aulast%3DKingswood%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DO.%2BJ.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DPersaud%26aufirst%3DK.%26aulast%3DPreall%26aufirst%3DJ.%2BB.%26aulast%3DRibeca%26aufirst%3DP.%26aulast%3DRisk%26aufirst%3DB.%26aulast%3DRobyr%26aufirst%3DD.%26aulast%3DSammeth%26aufirst%3DM.%26aulast%3DSchaffer%26aufirst%3DL.%26aulast%3DSee%26aufirst%3DL.-H.%26aulast%3DShahab%26aufirst%3DA.%26aulast%3DSkancke%26aufirst%3DJ.%26aulast%3DSuzuki%26aufirst%3DA.%2BM.%26aulast%3DTakahashi%26aufirst%3DH.%26aulast%3DTilgner%26aufirst%3DH.%26aulast%3DTrout%26aufirst%3DD.%26aulast%3DWalters%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWrobel%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DRuan%26aufirst%3DX.%26aulast%3DHayashizaki%26aufirst%3DY.%26aulast%3DHarrow%26aufirst%3DJ.%26aulast%3DGerstein%26aufirst%3DM.%26aulast%3DHubbard%26aufirst%3DT.%26aulast%3DReymond%26aufirst%3DA.%26aulast%3DAntonarakis%26aufirst%3DS.%2BE.%26aulast%3DHannon%26aufirst%3DG.%26aulast%3DGiddings%26aufirst%3DM.%2BC.%26aulast%3DRuan%26aufirst%3DY.%26aulast%3DWold%26aufirst%3DB.%26aulast%3DCarninci%26aufirst%3DP.%26aulast%3DGuig%25C3%25B3%26aufirst%3DR.%26aulast%3DGingeras%26aufirst%3DT.%2BR.%26atitle%3DLandscape%2520of%2520transcription%2520in%2520human%2520cells%26jtitle%3DNature%26date%3D2012%26volume%3D489%26spage%3D101%26epage%3D108%26doi%3D10.1038%2Fnature11233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wade, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grainger, D. C.</span></span> <span> </span><span class="NLM_article-title">Pervasive transcription: illuminating the dark matter of bacterial transcriptomes</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">653</span>, <span class="refDoi"> DOI: 10.1038/nrmicro3316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrmicro3316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=25069631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1amsLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=647-653&author=J.+T.+Wadeauthor=D.+C.+Grainger&title=Pervasive+transcription%3A+illuminating+the+dark+matter+of+bacterial+transcriptomes&doi=10.1038%2Fnrmicro3316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pervasive transcription: illuminating the dark matter of bacterial transcriptomes</span></div><div class="casAuthors">Wade, Joseph T.; Grainger, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">647-653</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The conventional view of transcription posits that mRNAs are generated from the coding DNA strand and are delineated by gene boundaries; however, recent reports have mapped transcription start sites to unexpected locations in bacterial genomes, including the non-coding strand.  The resultant RNAs were previously dismissed as artifacts, but models that describe such events as 'pervasive transcription' are now gaining support.  In this Opinion article, we discuss our current understanding of pervasive transcription, its genetic origin and its regulation.  On the basis of existing observations, we propose that RNAs that result from pervasive transcription are more than 'transcriptional noise' and have important functions in gene regulation and genome evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0npPgSYlBiLVg90H21EOLACvtfcHk0liY5G-3elombg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1amsLzL&md5=8f2faa39c9b77d4c5a7fcbf989d14668</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro3316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro3316%26sid%3Dliteratum%253Aachs%26aulast%3DWade%26aufirst%3DJ.%2BT.%26aulast%3DGrainger%26aufirst%3DD.%2BC.%26atitle%3DPervasive%2520transcription%253A%2520illuminating%2520the%2520dark%2520matter%2520of%2520bacterial%2520transcriptomes%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2014%26volume%3D12%26spage%3D647%26epage%3D653%26doi%3D10.1038%2Fnrmicro3316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattick, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taft, R. J.</span></span> <span> </span><span class="NLM_article-title">A meta-analysis of the genomic and transcriptomic composition of complex life</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2061</span>– <span class="NLM_lpage">2072</span>, <span class="refDoi"> DOI: 10.4161/cc.25134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.4161%2Fcc.25134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=23759593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVOk" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=2061-2072&author=G.+Liuauthor=J.+S.+Mattickauthor=R.+J.+Taft&title=A+meta-analysis+of+the+genomic+and+transcriptomic+composition+of+complex+life&doi=10.4161%2Fcc.25134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A meta-analysis of the genomic and transcriptomic composition of complex life</span></div><div class="casAuthors">Liu, Ganqiang; Mattick, John S.; Taft, Ryan J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2061-2072</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">It is now clear that animal genomes are predominantly non-protein-coding, and that these sequences encode a wide array of RNA transcripts and other regulatory elements that are fundamental to the development of complex life.  We have previously argued that the proportion of an animal genome that is non-protein-coding DNA (ncDNA) correlates well with its apparent biol. complexity.  Here we extend on that work and, using data from a total of 1,627 prokaryotic and 153 eukaryotic complete and annotated genomes, show that the proportion of ncDNA per haploid genome is significantly pos. correlated with a previously published proxy of biol. complexity, the no. of distinct cell types.  This is in contrast to the amt. of the genome that encodes proteins, which we show is essentially unchanged across Metazoa.  Furthermore, using a total of 179 RNA-seq data sets from nematode (47), fruit fly (72), zebrafish (20) and human (42), we show, consistent with other recent reports, that the vast majority of ncDNA in animals is transcribed.  This includes more than 60 human loci previously considered "gene deserts," many of which are expressed tissue-specifically and assocd. with previously reported GWAS SNPs.  These results suggest that ncDNA, and the ncRNAs encoded within it, may be intimately involved in the evolution, maintenance and development of complex life.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGossTGtUcknC7Vg90H21EOLACvtfcHk0liY5G-3elombg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVOk&md5=c20682cb9543b36ec1b2377ce862e5e7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.4161%2Fcc.25134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.25134%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DMattick%26aufirst%3DJ.%2BS.%26aulast%3DTaft%26aufirst%3DR.%2BJ.%26atitle%3DA%2520meta-analysis%2520of%2520the%2520genomic%2520and%2520transcriptomic%2520composition%2520of%2520complex%2520life%26jtitle%3DCell%2520Cycle%26date%3D2013%26volume%3D12%26spage%3D2061%26epage%3D2072%26doi%3D10.4161%2Fcc.25134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Necsulea, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soumillon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warnefors, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liechti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daish, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grützner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaessmann, H.</span></span> <span> </span><span class="NLM_article-title">The evolution of lncRNA repertoires and expression patterns in tetrapods</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>505</i></span>,  <span class="NLM_fpage">635</span>– <span class="NLM_lpage">640</span>, <span class="refDoi"> DOI: 10.1038/nature12943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnature12943" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=24463510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFemtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=505&publication_year=2014&pages=635-640&author=A.+Necsuleaauthor=M.+Soumillonauthor=M.+Warneforsauthor=A.+Liechtiauthor=T.+Daishauthor=U.+Zellerauthor=J.+C.+Bakerauthor=F.+Gr%C3%BCtznerauthor=H.+Kaessmann&title=The+evolution+of+lncRNA+repertoires+and+expression+patterns+in+tetrapods&doi=10.1038%2Fnature12943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The evolution of lncRNA repertoires and expression patterns in tetrapods</span></div><div class="casAuthors">Necsulea, Anamaria; Soumillon, Magali; Warnefors, Maria; Liechti, Angelica; Daish, Tasman; Zeller, Ulrich; Baker, Julie C.; Gruetzner, Frank; Kaessmann, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">505</span>
        (<span class="NLM_cas:issue">7485</span>),
    <span class="NLM_cas:pages">635-640</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Only a very small fraction of long noncoding RNAs (lncRNAs) are well characterized.  The evolutionary history of lncRNAs can provide insights into their functionality, but the absence of lncRNA annotations in non-model organisms has precluded comparative analyses.  Here we present a large-scale evolutionary study of lncRNA repertoires and expression patterns, in 11 tetrapod species.  We identify approx. 11,000 primate-specific lncRNAs and 2500 highly conserved lncRNAs, including approx. 400 genes that are likely to have originated more than 300 million years ago.  We find that lncRNAs, in particular ancient ones, are in general actively regulated and may function predominantly in embryonic development.  Most lncRNAs evolve rapidly in terms of sequence and expression levels, but tissue specificities are often conserved.  We compared expression patterns of homologous lncRNA and protein-coding families across tetrapods to reconstruct an evolutionarily conserved co-expression network.  This network suggests potential functions for lncRNAs in fundamental processes such as spermatogenesis and synaptic transmission, but also in more specific mechanisms such as placenta development through microRNA prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry4ZkSNvvosbVg90H21EOLACvtfcHk0li_Ow7RmIgDUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFemtLg%253D&md5=7eeda95062d8f96267952e8d452d5fb6</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature12943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12943%26sid%3Dliteratum%253Aachs%26aulast%3DNecsulea%26aufirst%3DA.%26aulast%3DSoumillon%26aufirst%3DM.%26aulast%3DWarnefors%26aufirst%3DM.%26aulast%3DLiechti%26aufirst%3DA.%26aulast%3DDaish%26aufirst%3DT.%26aulast%3DZeller%26aufirst%3DU.%26aulast%3DBaker%26aufirst%3DJ.%2BC.%26aulast%3DGr%25C3%25BCtzner%26aufirst%3DF.%26aulast%3DKaessmann%26aufirst%3DH.%26atitle%3DThe%2520evolution%2520of%2520lncRNA%2520repertoires%2520and%2520expression%2520patterns%2520in%2520tetrapods%26jtitle%3DNature%26date%3D2014%26volume%3D505%26spage%3D635%26epage%3D640%26doi%3D10.1038%2Fnature12943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Makley, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Expanding the number of “druggable” targets: non-enzymes and protein-protein interactions</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1111%2Fcbdd.12066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=23253128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1ymtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=22-32&author=L.+N.+Makleyauthor=J.+E.+Gestwicki&title=Expanding+the+number+of+%E2%80%9Cdruggable%E2%80%9D+targets%3A+non-enzymes+and+protein-protein+interactions&doi=10.1111%2Fcbdd.12066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding the number of "druggable" targets: non-enzymes and protein-protein interactions</span></div><div class="casAuthors">Makley, Leah N.; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-32</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Following sequencing and assembly of the human genome, the preferred methods for identification of new drug targets have changed dramatically.  Modern tactics such as genome-wide assocn. studies (GWAS) and deep sequencing are fundamentally different from the pharmacol.-guided approaches used previously, in which knowledge of small mol. ligands acting at their cellular targets was the primary discovery engine.  A consequence of the "target-first, pharmacol.-second" strategy is that many predicted drug targets are non-enzymes, such as scaffolding, regulatory or structural proteins, and their activities are often dependent on protein-protein interactions (PPIs).  These types of targets create unique challenges to drug discovery efforts because enzymic turnover cannot be used as a convenient surrogate for compd. potency.  Moreover, it is often challenging to predict how ligand binding to non-enzymes might affect changes in protein function and/or pathobiol.  Thus, in the postgenomic era, targets might be strongly implicated by mol. biol.-based methods, yet they often later earn the designation of "undruggable".  Can the scope of available targets be widened to include these promising, but challenging, non-enzymes.  In this review, we discuss advances in high-throughput screening (HTS) technol. and chem. library design that are emerging to deal with these challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMtMz2zhY_vLVg90H21EOLACvtfcHk0li_Ow7RmIgDUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1ymtQ%253D%253D&md5=9e8a15432b02550dca5c9e4668d99038</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12066%26sid%3Dliteratum%253Aachs%26aulast%3DMakley%26aufirst%3DL.%2BN.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DExpanding%2520the%2520number%2520of%2520%25E2%2580%259Cdruggable%25E2%2580%259D%2520targets%253A%2520non-enzymes%2520and%2520protein-protein%2520interactions%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2013%26volume%3D81%26spage%3D22%26epage%3D32%26doi%3D10.1111%2Fcbdd.12066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Limbird, L. E.</span></span> <span> </span><span class="NLM_article-title">The receptor concept: a continuing evolution</span>. <i>Mol. Interventions</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1124/mi.4.6.6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1124%2Fmi.4.6.6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=15616162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnt1ejtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=326-336&author=L.+E.+Limbird&title=The+receptor+concept%3A+a+continuing+evolution&doi=10.1124%2Fmi.4.6.6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The receptor concept: A continuing evolution</span></div><div class="casAuthors">Limbird, Lee E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Interventions</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">326-336</span>CODEN:
                <span class="NLM_cas:coden">MIONAR</span>;
        ISSN:<span class="NLM_cas:issn">1534-0384</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  The author briefly summarizes the development of the receptor concept, the identification receptors based 1st on biol. response data and subsequently on radioligand binding properties, and the biol. and physiol. understandings that these approaches have made possible.  The development of receptor characterization began with receptors that ultimately were discovered to mediate response by coupling to G-binding proteins, also known as G protein-coupled receptors (GPCRs).  Consequently, many if not all of the examples in this overview describe studies characterizing GPCRs in general, and adrenergic receptors in particular.  The purpose of this review, however, is not a detailed chronol. account of a huge literature, but rather an overview of the fundamental questions posed and answered by these studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8s65Hqwvj1bVg90H21EOLACvtfcHk0lh3SuQNi2laQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnt1ejtA%253D%253D&md5=b3ca1691f239fe1baac632f328332bb5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1124%2Fmi.4.6.6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmi.4.6.6%26sid%3Dliteratum%253Aachs%26aulast%3DLimbird%26aufirst%3DL.%2BE.%26atitle%3DThe%2520receptor%2520concept%253A%2520a%2520continuing%2520evolution%26jtitle%3DMol.%2520Interventions%26date%3D2004%26volume%3D4%26spage%3D326%26epage%3D336%26doi%3D10.1124%2Fmi.4.6.6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazo, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharlow, E. R.</span></span> <span> </span><span class="NLM_article-title">Drugging undruggable molecular cancer targets</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1146%2Fannurev-pharmtox-010715-103440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=26527069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGlsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=23-40&author=J.+S.+Lazoauthor=E.+R.+Sharlow&title=Drugging+undruggable+molecular+cancer+targets&doi=10.1146%2Fannurev-pharmtox-010715-103440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging Undruggable Molecular Cancer Targets</span></div><div class="casAuthors">Lazo, John S.; Sharlow, Elizabeth R.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23-40</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Cancer, more than any other human disease, now has a surfeit of potential mol. targets poised for therapeutic exploitation.  Currently, a no. of attractive and validated cancer targets remain outside of the reach of pharmacol. regulation.  Some have been described as undruggable, at least by traditional strategies.  In this article, we outline the basis for the undruggable moniker, propose a reclassification of these targets as undrugged, and highlight three general classes of this imposing group as exemplars with some attendant strategies currently being explored to reclassify them.  Expanding the spectrum of disease-relevant targets to pharmacol. manipulation is central to reducing cancer morbidity and mortality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpny-ev1lJKqbVg90H21EOLACvtfcHk0lh3SuQNi2laQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGlsr7O&md5=5ab0518d4155a00e2a898a6f68d551d8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103440%26sid%3Dliteratum%253Aachs%26aulast%3DLazo%26aufirst%3DJ.%2BS.%26aulast%3DSharlow%26aufirst%3DE.%2BR.%26atitle%3DDrugging%2520undruggable%2520molecular%2520cancer%2520targets%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2016%26volume%3D56%26spage%3D23%26epage%3D40%26doi%3D10.1146%2Fannurev-pharmtox-010715-103440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Drugging the ‘undruggable’ cancer targets</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrc.2017.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28643779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=502-508&author=C.+V.+Dangauthor=E.+P.+Reddyauthor=K.+M.+Shokatauthor=L.+Soucek&title=Drugging+the+%E2%80%98undruggable%E2%80%99+cancer+targets&doi=10.1038%2Fnrc.2017.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the 'undruggable' cancer targets</span></div><div class="casAuthors">Dang, Chi V.; Reddy, E. Premkumar; Shokat, Kevan M.; Soucek, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">502-508</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The term 'undruggable' was coined to describe proteins that could not be targeted pharmacol.  However, progress is being made to 'drug' many of these targets, and therefore more appropriate terms might be 'difficult to drug' or 'yet to be drugged'.  Many desirable targets in cancer fall into this category, including the RAS and MYC oncogenes, and pharmacol. targeting these intractable proteins is now a key challenge in cancer research that requires innovation and the development of new technologies.  In this Viewpoint article, we asked four scientists working in this field for their opinions on the most crucial advances, as well as the challenges and what the future holds for this important area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoElcMCmqGx_rVg90H21EOLACvtfcHk0ljCJ5Zzj2NFZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP&md5=774da7ed04931a3fd3689eb9acea594a</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.36%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DReddy%26aufirst%3DE.%2BP.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DDrugging%2520the%2520%25E2%2580%2598undruggable%25E2%2580%2599%2520cancer%2520targets%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D502%26epage%3D508%26doi%3D10.1038%2Fnrc.2017.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ostrem, J. M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Direct small-molecule inhibitors of KRAS: from structural insights to mechanism-based design</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">771</span>– <span class="NLM_lpage">785</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrd.2016.139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27469033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=771-785&author=J.+M.+L.+Ostremauthor=K.+M.+Shokat&title=Direct+small-molecule+inhibitors+of+KRAS%3A+from+structural+insights+to+mechanism-based+design&doi=10.1038%2Fnrd.2016.139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Direct small-molecule inhibitors of KRAS: from structural insights to mechanism-based design</span></div><div class="casAuthors">Ostrem, Jonathan M. L.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">771-785</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">KRAS is the most frequently mutated oncogene in human cancer.  In addn. to holding this distinction, unsuccessful attempts to target this protein have led to the characterization of RAS as 'undruggable'.  However, recent advances in technol. and novel approaches to drug discovery have renewed hope that a direct KRAS inhibitor may be on the horizon.  In this Review, we provide an in-depth anal. of the structure, dynamics, mutational activation and inactivation, and signalling mechanisms of RAS.  From this perspective, we then consider potential mechanisms of action for effective RAS inhibitors.  Finally, we examine each of the many recent reports of direct RAS inhibitors and discuss promising avenues for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBobrhZLF3V7Vg90H21EOLACvtfcHk0ljCJ5Zzj2NFZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FK&md5=b0832c7ef45cc0555e11c59baccc5abd</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.139%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%2BL.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DDirect%2520small-molecule%2520inhibitors%2520of%2520KRAS%253A%2520from%2520structural%2520insights%2520to%2520mechanism-based%2520design%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D771%26epage%3D785%26doi%3D10.1038%2Fnrd.2016.139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmachl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantoulidis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gollner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häring, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hela, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Oezguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knesl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornigg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kousek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamarre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munico-Martinez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peinsipp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salamon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherbantin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipany, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrenk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siszler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolkerstorfer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span> <span> </span><span class="NLM_article-title">Drugging an undruggable pocket on KRAS</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">15823</span>– <span class="NLM_lpage">15829</span>, <span class="refDoi"> DOI: 10.1073/pnas.1904529116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1904529116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31332011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFChs7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=15823-15829&author=D.+Kesslerauthor=M.+Gmachlauthor=A.+Mantoulidisauthor=L.+J.+Martinauthor=A.+Zoephelauthor=M.+Mayerauthor=A.+Gollnerauthor=D.+Coviniauthor=S.+Fischerauthor=T.+Gerstbergerauthor=T.+Gmaschitzauthor=C.+Goodwinauthor=P.+Grebauthor=D.+H%C3%A4ringauthor=W.+Helaauthor=J.+Hoffmannauthor=J.+Karolyi-Oezguerauthor=P.+Kneslauthor=S.+Korniggauthor=M.+Koeglauthor=R.+Kousekauthor=L.+Lamarreauthor=F.+Moserauthor=S.+Munico-Martinezauthor=C.+Peinsippauthor=J.+Phanauthor=J.+Rinnenthalauthor=J.+Saiauthor=C.+Salamonauthor=Y.+Scherbantinauthor=K.+Schipanyauthor=R.+Schnitzerauthor=A.+Schrenkauthor=B.+Sharpsauthor=G.+Siszlerauthor=Q.+Sunauthor=A.+Watersonauthor=B.+Wolkerstorferauthor=M.+Zeebauthor=M.+Pearsonauthor=S.+W.+Fesikauthor=D.+B.+McConnell&title=Drugging+an+undruggable+pocket+on+KRAS&doi=10.1073%2Fpnas.1904529116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging an undruggable pocket on KRAS</span></div><div class="casAuthors">Kessler, Dirk; Gmachl, Michael; Mantoulidis, Andreas; Martin, Laetitia J.; Zoephel, Andreas; Mayer, Moriz; Gollner, Andreas; Covini, David; Fischer, Silke; Gerstberger, Thomas; Gmaschitz, Teresa; Goodwin, Craig; Greb, Peter; Haring, Daniela; Hela, Wolfgang; Hoffmann, Johann; Karolyi-Oezguer, Jale; Knesl, Petr; Kornigg, Stefan; Koegl, Manfred; Kousek, Roland; Lamarre, Lyne; Moser, Franziska; Munico-Martinez, Silvia; Peinsipp, Christoph; Phan, Jason; Rinnenthal, Jorg; Sai, Jiqing; Salamon, Christian; Scherbantin, Yvonne; Schipany, Katharina; Schnitzer, Renate; Schrenk, Andreas; Sharps, Bernadette; Siszler, Gabriella; Sun, Qi; Waterson, Alex; Wolkerstorfer, Bernhard; Zeeb, Markus; Pearson, Mark; Fesik, Stephen W.; McConnell, Darryl B.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">15823-15829</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">(Imidazolylmethylindolylmethylaminomethyl)indolyl hydroxyisoindolone I, an inhibitor of both the active and inactive forms of KRAS, was discovered using fragment-based screening.  I bound to KRAS in a pocket between switches I and II of KRAS, blocking all GEF, GAP, and effector interactions with KRAS and thus inhibiting downstream signaling; I inhibited KRAS mutant cells at low micromolar concns.  Uniformly 15N-labeled KRAS bound to the stable GTP analog GCP was prepd. with yields of 10 mg/L for use in crystallog. of KRAS-inhibitor complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaMeFg74p2-rVg90H21EOLACvtfcHk0lj9Cd47XqNd0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFChs7jJ&md5=ebf443050cb2ce6cbeec1e40084129d0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1904529116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1904529116%26sid%3Dliteratum%253Aachs%26aulast%3DKessler%26aufirst%3DD.%26aulast%3DGmachl%26aufirst%3DM.%26aulast%3DMantoulidis%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DL.%2BJ.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DGollner%26aufirst%3DA.%26aulast%3DCovini%26aufirst%3DD.%26aulast%3DFischer%26aufirst%3DS.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DGoodwin%26aufirst%3DC.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DH%25C3%25A4ring%26aufirst%3DD.%26aulast%3DHela%26aufirst%3DW.%26aulast%3DHoffmann%26aufirst%3DJ.%26aulast%3DKarolyi-Oezguer%26aufirst%3DJ.%26aulast%3DKnesl%26aufirst%3DP.%26aulast%3DKornigg%26aufirst%3DS.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DKousek%26aufirst%3DR.%26aulast%3DLamarre%26aufirst%3DL.%26aulast%3DMoser%26aufirst%3DF.%26aulast%3DMunico-Martinez%26aufirst%3DS.%26aulast%3DPeinsipp%26aufirst%3DC.%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DSai%26aufirst%3DJ.%26aulast%3DSalamon%26aufirst%3DC.%26aulast%3DScherbantin%26aufirst%3DY.%26aulast%3DSchipany%26aufirst%3DK.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DSchrenk%26aufirst%3DA.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DSiszler%26aufirst%3DG.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DWaterson%26aufirst%3DA.%26aulast%3DWolkerstorfer%26aufirst%3DB.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26atitle%3DDrugging%2520an%2520undruggable%2520pocket%2520on%2520KRAS%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D15823%26epage%3D15829%26doi%3D10.1073%2Fpnas.1904529116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yogodzinski, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shkedi, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, A. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaney, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, L. A.</span></span> <span> </span><span class="NLM_article-title">KRASG12C inhibition produces a driver-limited state revealing collateral dependencies</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">eaaw9450</span> <span class="refDoi"> DOI: 10.1126/scisignal.aaw9450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1126%2Fscisignal.aaw9450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31138768" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&author=K.+Louauthor=V.+Steriauthor=A.+Y.+Geauthor=Y.+C.+Hwangauthor=C.+H.+Yogodzinskiauthor=A.+R.+Shkediauthor=A.+L.+M.+Choiauthor=D.+C.+Mitchellauthor=D.+L.+Swaneyauthor=B.+Hannauthor=J.+D.+Gordanauthor=K.+M.+Shokatauthor=L.+A.+Gilbert&title=KRASG12C+inhibition+produces+a+driver-limited+state+revealing+collateral+dependencies&doi=10.1126%2Fscisignal.aaw9450"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.aaw9450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.aaw9450%26sid%3Dliteratum%253Aachs%26aulast%3DLou%26aufirst%3DK.%26aulast%3DSteri%26aufirst%3DV.%26aulast%3DGe%26aufirst%3DA.%2BY.%26aulast%3DHwang%26aufirst%3DY.%2BC.%26aulast%3DYogodzinski%26aufirst%3DC.%2BH.%26aulast%3DShkedi%26aufirst%3DA.%2BR.%26aulast%3DChoi%26aufirst%3DA.%2BL.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BC.%26aulast%3DSwaney%26aufirst%3DD.%2BL.%26aulast%3DHann%26aufirst%3DB.%26aulast%3DGordan%26aufirst%3DJ.%2BD.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DGilbert%26aufirst%3DL.%2BA.%26atitle%3DKRASG12C%2520inhibition%2520produces%2520a%2520driver-limited%2520state%2520revealing%2520collateral%2520dependencies%26jtitle%3DSci.%2520Signaling%26date%3D2019%26volume%3D12%26doi%3D10.1126%2Fscisignal.aaw9450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROteolysis TArgeting Chimeras (PROTACs)—past, present and future</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.ddtec.2019.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31200855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=15-27&author=M.+Petterssonauthor=C.+M.+Crews&title=PROteolysis+TArgeting+Chimeras+%28PROTACs%29%E2%80%94past%2C+present+and+future&doi=10.1016%2Fj.ddtec.2019.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">PROteolysis TArgeting Chimeras (PROTACs) - Past, present and future</span></div><div class="casAuthors">Pettersson Mariell; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The majority of currently used therapeutics are small molecule-based and utilize occupancy-driven pharmacology as the mode of action (MOA), in which the protein function is modulated via temporary inhibition.  New modalities that operate using alternative MOAs are essential for tapping into the "undruggable" proteome.  The PROteolysis Targeting Chimera (PROTAC) technology provides an attractive new approach that utilizes an event-driven MOA.  Small molecule-based heterobifunctional PROTACs modulate protein target levels by hijacking the ubiquitin-proteasome system to induce degradation of the target.  Here, we address important milestones in the development of the PROTAC technology, as well as emphasize key findings from this previous year and highlight future directions of this promising drug discovery modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRfKb6ELK8t0fW6udTcc2eYex1kdH378Hrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D&md5=c629f3128321e136f1f247e05ee6f182</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROteolysis%2520TArgeting%2520Chimeras%2520%2528PROTACs%2529%25E2%2580%2594past%252C%2520present%2520and%2520future%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.ddtec.2019.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Backus, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horning, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Paez, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanning, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teijaro, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolan, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Proteome-wide covalent ligand discovery in native biological systems</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>534</i></span>,  <span class="NLM_fpage">570</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1038/nature18002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnature18002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27309814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSksbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=570-574&author=K.+M.+Backusauthor=B.+E.+Correiaauthor=K.+M.+Lumauthor=S.+Forliauthor=B.+D.+Horningauthor=G.+E.+Gonzalez-Paezauthor=S.+Chatterjeeauthor=B.+R.+Lanningauthor=J.+R.+Teijaroauthor=A.+J.+Olsonauthor=D.+W.+Wolanauthor=B.+F.+Cravatt&title=Proteome-wide+covalent+ligand+discovery+in+native+biological+systems&doi=10.1038%2Fnature18002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-wide covalent ligand discovery in native biological systems</span></div><div class="casAuthors">Backus, Keriann M.; Correia, Bruno E.; Lum, Kenneth M.; Forli, Stefano; Horning, Benjamin D.; Gonzalez-Paez, Gonzalo E.; Chatterjee, Sandip; Lanning, Bryan R.; Teijaro, John R.; Olson, Arthur J.; Wolan, Dennis W.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7608</span>),
    <span class="NLM_cas:pages">570-574</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small mols. are powerful tools for investigating protein function and can serve as leads for new therapeutics.  Most human proteins, however, lack small-mol. ligands, and entire protein classes are considered 'undruggable'.  Fragment-based ligand discovery can identify small-mol. probes for proteins that have proven difficult to target using high-throughput screening of complex compd. libraries.  Although reversibly binding ligands are commonly pursued, covalent fragments provide an alternative route to small-mol. probes, including those that can access regions of proteins that are difficult to target through binding affinity alone.  Here we report a quant. anal. of cysteine-reactive small-mol. fragments screened against thousands of proteins in human proteomes and cells.  Covalent ligands were identified for >700 cysteines found in both druggable proteins and proteins deficient in chem. probes, including transcription factors, adaptor/scaffolding proteins, and uncharacterized proteins.  Among the atypical ligand-protein interactions discovered were compds. that react preferentially with pro- (inactive) caspases.  We used these ligands to distinguish extrinsic apoptosis pathways in human cell lines vs. primary human T cells, showing that the former is largely mediated by caspase-8 while the latter depends on both caspase-8 and -10.  Fragment-based covalent ligand discovery provides a greatly expanded portrait of the ligandable proteome and furnishes compds. that can illuminate protein functions in native biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgNJ88KALeSLVg90H21EOLACvtfcHk0ljez8yLGJOVxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSksbnN&md5=6fab087c042de735f8a5931e9f5b946c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature18002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18002%26sid%3Dliteratum%253Aachs%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DLum%26aufirst%3DK.%2BM.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DGonzalez-Paez%26aufirst%3DG.%2BE.%26aulast%3DChatterjee%26aufirst%3DS.%26aulast%3DLanning%26aufirst%3DB.%2BR.%26aulast%3DTeijaro%26aufirst%3DJ.%2BR.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DWolan%26aufirst%3DD.%2BW.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DProteome-wide%2520covalent%2520ligand%2520discovery%2520in%2520native%2520biological%2520systems%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D570%26epage%3D574%26doi%3D10.1038%2Fnature18002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galmozzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joslyn, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavallaro, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narvaiza, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Ligand and target discovery by fragment-based screening in human cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>168</i></span>,  <span class="NLM_fpage">527</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.12.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.cell.2016.12.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28111073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVeiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=527-541&author=C.+G.+Parkerauthor=A.+Galmozziauthor=Y.+Wangauthor=B.+E.+Correiaauthor=K.+Sasakiauthor=C.+M.+Joslynauthor=A.+S.+Kimauthor=C.+L.+Cavallaroauthor=R.+M.+Lawrenceauthor=S.+R.+Johnsonauthor=I.+Narvaizaauthor=E.+Saezauthor=B.+F.+Cravatt&title=Ligand+and+target+discovery+by+fragment-based+screening+in+human+cells&doi=10.1016%2Fj.cell.2016.12.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand and Target Discovery by Fragment-Based Screening in Human Cells</span></div><div class="casAuthors">Parker, Christopher G.; Galmozzi, Andrea; Wang, Yujia; Correia, Bruno E.; Sasaki, Kenji; Joslyn, Christopher M.; Kim, Arthur S.; Cavallaro, Cullen L.; Lawrence, R. Michael; Johnson, Stephen R.; Narvaiza, Inigo; Saez, Enrique; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">527-541.e29</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Advances in the synthesis and screening of small-mol. libraries have accelerated the discovery of chem. probes for studying biol. processes.  Still, only a small fraction of the human proteome has chem. ligands.  Here, we describe a platform that marries fragment-based ligand discovery with quant. chem. proteomics to map thousands of reversible small mol.-protein interactions directly in human cells, many of which can be site-specifically detd.  We show that fragment hits can be advanced to furnish selective ligands that affect the activity of proteins heretofore lacking chem. probes.  We further combine fragment-based chem. proteomics with phenotypic screening to identify small mols. that promote adipocyte differentiation by engaging the poorly characterized membrane protein PGRMC2.  Fragment-based screening in human cells thus provides an extensive proteome-wide map of protein ligandability and facilitates the coordinated discovery of bioactive small mols. and their mol. targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPk-YJwFlDwbVg90H21EOLACvtfcHk0lggkOe6QSTUOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVeiu7Y%253D&md5=835264291015037ab9b7e446992caba3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.12.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.12.029%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DC.%2BG.%26aulast%3DGalmozzi%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DSasaki%26aufirst%3DK.%26aulast%3DJoslyn%26aufirst%3DC.%2BM.%26aulast%3DKim%26aufirst%3DA.%2BS.%26aulast%3DCavallaro%26aufirst%3DC.%2BL.%26aulast%3DLawrence%26aufirst%3DR.%2BM.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DNarvaiza%26aufirst%3DI.%26aulast%3DSaez%26aufirst%3DE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DLigand%2520and%2520target%2520discovery%2520by%2520fragment-based%2520screening%2520in%2520human%2520cells%26jtitle%3DCell%26date%3D2017%26volume%3D168%26spage%3D527%26epage%3D541%26doi%3D10.1016%2Fj.cell.2016.12.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Rourke, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, M. S.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of RNA-mediated disease</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">7419</span>– <span class="NLM_lpage">7423</span>, <span class="refDoi"> DOI: 10.1074/jbc.R800025200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1074%2Fjbc.R800025200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=18957432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtVSmsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=7419-7423&author=J.+R.+O%E2%80%99Rourkeauthor=M.+S.+Swanson&title=Mechanisms+of+RNA-mediated+disease&doi=10.1074%2Fjbc.R800025200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of RNA-mediated Disease</span></div><div class="casAuthors">O'Rourke, Jason R.; Swanson, Maurice S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">7419-7423</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Recent mapping of functional sequence elements in the human genome has led to the realization that transcription is pervasive and that noncoding RNAs compose a significant portion of the transcriptome.  Some dominantly inherited neurol. disorders are assocd. with the expansion of microsatellite repeats in noncoding regions that result in the synthesis of pathogenic RNAs.  Here, we review RNA gain-of-function mechanisms underlying three of these microsatellite expansion disorders to illustrate how some mutant RNAs cause disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp32ch1_mjp-7Vg90H21EOLACvtfcHk0lggkOe6QSTUOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtVSmsrk%253D&md5=7a66790b67e63d47b710246b13b2f6ed</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R800025200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R800025200%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DJ.%2BR.%26aulast%3DSwanson%26aufirst%3DM.%2BS.%26atitle%3DMechanisms%2520of%2520RNA-mediated%2520disease%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D7419%26epage%3D7423%26doi%3D10.1074%2Fjbc.R800025200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreyfuss, G.</span></span> <span> </span><span class="NLM_article-title">RNA and disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">777</span>– <span class="NLM_lpage">793</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2009.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.cell.2009.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19239895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvFGktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=777-793&author=T.+A.+Cooperauthor=L.+Wanauthor=G.+Dreyfuss&title=RNA+and+disease&doi=10.1016%2Fj.cell.2009.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">RNA and disease</span></div><div class="casAuthors">Cooper, Thomas A.; Wan, Lili; Dreyfuss, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">777-793</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Cellular functions depend on numerous protein-coding and noncoding RNAs and the RNA-binding proteins assocd. with them, which form ribonucleoprotein complexes (RNPs).  Mutations that disrupt either the RNA or protein components of RNPs or the factors required for their assembly can be deleterious.  Alternative splicing provides cells with an exquisite capacity to fine-tune their transcriptome and proteome in response to cues.  Splicing depends on a complex code, numerous RNA-binding proteins, and an enormously intricate network of interactions among them, increasing the opportunity for exposure to mutations and misregulation that cause disease.  The discovery of disease-causing mutations in RNAs is yielding a wealth of new therapeutic targets, and the growing understanding of RNA biol. and chem. is providing new RNA-based tools for developing therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotOMwPdFvVpLVg90H21EOLACvtfcHk0lgfDQ7cuTAH-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvFGktrw%253D&md5=f165df2b33f97df6e895045b81e853d4</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DCooper%26aufirst%3DT.%2BA.%26aulast%3DWan%26aufirst%3DL.%26aulast%3DDreyfuss%26aufirst%3DG.%26atitle%3DRNA%2520and%2520disease%26jtitle%3DCell%26date%3D2009%26volume%3D136%26spage%3D777%26epage%3D793%26doi%3D10.1016%2Fj.cell.2009.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sztuba-Solinska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavez-Calvillo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cline, S. E.</span></span> <span> </span><span class="NLM_article-title">Unveiling the druggable RNA targets and small molecule therapeutics</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2149</span>– <span class="NLM_lpage">2165</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2019.03.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.bmc.2019.03.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30981606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntFCiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=2149-2165&author=J.+Sztuba-Solinskaauthor=G.+Chavez-Calvilloauthor=S.+E.+Cline&title=Unveiling+the+druggable+RNA+targets+and+small+molecule+therapeutics&doi=10.1016%2Fj.bmc.2019.03.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Unveiling the druggable RNA targets and small molecule therapeutics</span></div><div class="casAuthors">Sztuba-Solinska, Joanna; Chavez-Calvillo, Gabriela; Cline, Sabrina Elizabeth</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2149-2165</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The increasing appreciation for the crucial roles of RNAs in infectious and non-infectious human diseases makes them attractive therapeutic targets.  Coding and non-coding RNAs frequently fold into complex conformations which, if effectively targeted, offer opportunities to therapeutically modulate numerous cellular processes, including those linked to undruggable protein targets.  Despite the considerable skepticism as to whether RNAs can be targeted with small mol. therapeutics, overwhelming evidence suggests the challenges we are currently facing are not outside the realm of possibility.  In this review, we highlight the most recent advances in mol. techniques that have sparked a revolution in understanding the RNA structure-to-function relationship.  We bring attention to the application of these modern techniques to identify druggable RNA targets and to assess small mol. binding specificity.  Finally, we discuss novel screening methodologies that support RNA drug discovery and present examples of therapeutically valuable RNA targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ_ZttV3TINLVg90H21EOLACvtfcHk0lgfDQ7cuTAH-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntFCiu74%253D&md5=5c2197acf2f0a04f6fda7711fb3cc0a9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.03.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.03.057%26sid%3Dliteratum%253Aachs%26aulast%3DSztuba-Solinska%26aufirst%3DJ.%26aulast%3DChavez-Calvillo%26aufirst%3DG.%26aulast%3DCline%26aufirst%3DS.%2BE.%26atitle%3DUnveiling%2520the%2520druggable%2520RNA%2520targets%2520and%2520small%2520molecule%2520therapeutics%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D2149%26epage%3D2165%26doi%3D10.1016%2Fj.bmc.2019.03.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ursu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vezina-Dawod, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Methods to identify and optimize small molecules interacting with RNA (SMIRNAs)</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2002</span>– <span class="NLM_lpage">2016</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2019.06.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.drudis.2019.06.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31356880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFWjt7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=2002-2016&author=A.+Ursuauthor=S.+Vezina-Dawodauthor=M.+D.+Disney&title=Methods+to+identify+and+optimize+small+molecules+interacting+with+RNA+%28SMIRNAs%29&doi=10.1016%2Fj.drudis.2019.06.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Methods to identify and optimize small molecules interacting with RNA (SMIRNAs)</span></div><div class="casAuthors">Ursu, Andrei; Vezina-Dawod, Simon; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2002-2016</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  RNAs, particularly noncoding RNAs (ncRNAs), are becoming increasingly important therapeutic targets, because they are causative and antagonists of human disease.  Indeed, aberrant RNA structural elements and expression deregulate biol. processes.  In this review, describe methodologies to discover and optimize small mols. interacting with RNA (SMIRNAs), including the evaluation of direct target engagement and the rescue of RNA-mediated phenotypes in vitro and in vivo.  Such studies are essential to fully characterize the mode of action of SMIRNAs and advance our understanding of rationally and efficiently drugging RNAs for therapeutic benefit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRFrcfvOwE9bVg90H21EOLACvtfcHk0ljCeNzPpe8xUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFWjt7nP&md5=8f5a62ec3259cf0f1577c35b1243633b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2019.06.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2019.06.019%26sid%3Dliteratum%253Aachs%26aulast%3DUrsu%26aufirst%3DA.%26aulast%3DVezina-Dawod%26aufirst%3DS.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DMethods%2520to%2520identify%2520and%2520optimize%2520small%2520molecules%2520interacting%2520with%2520RNA%2520%2528SMIRNAs%2529%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26spage%3D2002%26epage%3D2016%26doi%3D10.1016%2Fj.drudis.2019.06.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rizvi, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. F.</span></span> <span> </span><span class="NLM_article-title">RNA as a small molecule druggable target</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">5083</span>– <span class="NLM_lpage">5088</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.10.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.bmcl.2017.10.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29097169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslemtLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=5083-5088&author=N.+F.+Rizviauthor=G.+F.+Smith&title=RNA+as+a+small+molecule+druggable+target&doi=10.1016%2Fj.bmcl.2017.10.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">RNA as a small molecule druggable target</span></div><div class="casAuthors">Rizvi, Noreen F.; Smith, Graham F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5083-5088</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Small mol. drugs have readily been developed against many proteins in the human proteome, but RNA has remained an elusive target for drug discovery.  Increasingly, the authors see that RNA, and to a lesser extent DNA elements, show a persistent tertiary structure responsible for many diverse and complex cellular functions.  In this digest, the authors have summarized recent advances in screening approaches for RNA targets and outlined the discovery of novel, drug-like small mols. against RNA targets from various classes and therapeutic areas.  The link of structure, function, and small-mol. Druggability validates now for the first time that RNA can be the targets of therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQHvz_K9pGLVg90H21EOLACvtfcHk0ljCeNzPpe8xUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslemtLfF&md5=ba213f6e8954c97239ccc23f7096c13e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.10.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.10.052%26sid%3Dliteratum%253Aachs%26aulast%3DRizvi%26aufirst%3DN.%2BF.%26aulast%3DSmith%26aufirst%3DG.%2BF.%26atitle%3DRNA%2520as%2520a%2520small%2520molecule%2520druggable%2520target%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D5083%26epage%3D5088%26doi%3D10.1016%2Fj.bmcl.2017.10.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Giorgio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duca, M.</span></span> <span> </span><span class="NLM_article-title">Synthetic small-molecule RNA ligands: future prospects as therapeutic agents</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1242</span>– <span class="NLM_lpage">1255</span>, <span class="refDoi"> DOI: 10.1039/C9MD00195F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1039%2FC9MD00195F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31534649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1ShurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1242-1255&author=A.+Di%0AGiorgioauthor=M.+Duca&title=Synthetic+small-molecule+RNA+ligands%3A+future+prospects+as+therapeutic+agents&doi=10.1039%2FC9MD00195F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic small-molecule RNA ligands: future prospects as therapeutic agents</span></div><div class="casAuthors">Di Giorgio, A.; Duca, M.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1242-1255</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">RNA is one of the most intriguing and promising biol. targets for the discovery of innovative drugs in many pathologies and various biol. relevant RNAs that could serve as drug targets have already been identified.  Among the most important ones, one can mention prokaryotic rRNA which is the target of several marketed antibiotics, viral RNAs or oncogenic microRNAs that are tightly involved in the development and progression of various cancers.  Oligonucleotides are efficient and specific RNA targeting agents but suffer from poor pharmacodynamic and pharmacokinetic properties.  For this reason, a no. of synthetic small-mol. ligands have been identified and studied upon screening of chem. libraries or focused design of RNA binders.  In this review, we report the most relevant examples of synthetic compds. bearing sufficient selectivity to envisage clin. studies and future therapeutic applications with a particular attention for the main strategies that can be undertaken toward the improvement of selectivity and biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-wwNyCsBkOLVg90H21EOLACvtfcHk0ljCeNzPpe8xUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1ShurY%253D&md5=60f47cc3df704517d5c308ebbe2cae6b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC9MD00195F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00195F%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BGiorgio%26aufirst%3DA.%26aulast%3DDuca%26aufirst%3DM.%26atitle%3DSynthetic%2520small-molecule%2520RNA%2520ligands%253A%2520future%2520prospects%2520as%2520therapeutic%2520agents%26jtitle%3DMedChemComm%26date%3D2019%26volume%3D10%26spage%3D1242%26epage%3D1255%26doi%3D10.1039%2FC9MD00195F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forte, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargrove, A. E.</span></span> <span> </span><span class="NLM_article-title">Insights into the development of chemical probes for RNA</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">8025</span>– <span class="NLM_lpage">8037</span>, <span class="refDoi"> DOI: 10.1093/nar/gky718</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1093%2Fnar%2Fgky718" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30102391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=8025-8037&author=B.+S.+Morganauthor=J.+E.+Forteauthor=A.+E.+Hargrove&title=Insights+into+the+development+of+chemical+probes+for+RNA&doi=10.1093%2Fnar%2Fgky718"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the development of chemical probes for RNA</span></div><div class="casAuthors">Morgan, Brittany S.; Forte, Jordan E.; Hargrove, Amanda E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8025-8037</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Over the past decade, the RNA revolution has revealed thousands of non-coding RNAs that are essential for cellular regulation and are misregulated in disease.  While the development of methods and tools to study these RNAs has been challenging, the power and promise of small mol. chem. probes is increasingly recognized.  To harness existing knowledge, we compiled a list of 116 ligands with reported activity against RNA targets in biol. systems (R-BIND).  In this survey, we examine the RNA targets, design and discovery strategies, and chem. probe characterization techniques of these ligands.  We discuss the applicability of current tools to identify and evaluate RNA-targeted chem. probes, suggest criteria to assess the quality of RNA chem. probes and targets, and propose areas where new tools are particularly needed.  We anticipate that this knowledge will expedite the discovery of RNA-targeted ligands and the next phase of the RNA revolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBbTlYBW7qtrVg90H21EOLACvtfcHk0lgx4P9zCFcboQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosFansLk%253D&md5=69a67e0be1045cd8731a23b6e9d353f7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky718%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DB.%2BS.%26aulast%3DForte%26aufirst%3DJ.%2BE.%26aulast%3DHargrove%26aufirst%3DA.%2BE.%26atitle%3DInsights%2520into%2520the%2520development%2520of%2520chemical%2520probes%2520for%2520RNA%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3D8025%26epage%3D8037%26doi%3D10.1093%2Fnar%2Fgky718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warner, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajdin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weeks, K. M.</span></span> <span> </span><span class="NLM_article-title">Principles for targeting RNA with drug-like small molecules</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrd.2018.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29977051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1OntLnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=547-558&author=K.+D.+Warnerauthor=C.+E.+Hajdinauthor=K.+M.+Weeks&title=Principles+for+targeting+RNA+with+drug-like+small+molecules&doi=10.1038%2Fnrd.2018.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Principles for targeting RNA with drug-like small molecules</span></div><div class="casAuthors">Warner, Katherine Deigan; Hajdin, Christine E.; Weeks, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">547-558</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">RNA mols. are essential for cellular information transfer and gene regulation, and RNAs have been implicated in many human diseases.  Messenger and non-coding RNAs contain highly structured elements, and evidence suggests that many of these structures are important for function.  Targeting these RNAs with small mols. offers opportunities to therapeutically modulate numerous cellular processes, including those linked to undruggable protein targets.  Despite this promise, there is currently only a single class of human-designed small mols. that target RNA used clin. - the linezolid antibiotics.  However, a growing no. of small-mol. RNA ligands are being identified, leading to burgeoning interest in the field.  Here, we discuss principles for discovering small-mol. drugs that target RNA and argue that the overarching challenge is to identify appropriate target structures - namely, in disease-causing RNAs that have high information content and, consequently, appropriate ligand-binding pockets.  If focus is placed on such druggable binding sites in RNA, extensive knowledge of the typical physicochem. properties of drug-like small mols. could then enable small-mol. drug discovery for RNA targets to become (only) roughly as difficult as for protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYl_OguWzasrVg90H21EOLACvtfcHk0lgx4P9zCFcboQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1OntLnL&md5=df01185ded43db426749b9c656a04f64</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.93%26sid%3Dliteratum%253Aachs%26aulast%3DWarner%26aufirst%3DK.%2BD.%26aulast%3DHajdin%26aufirst%3DC.%2BE.%26aulast%3DWeeks%26aufirst%3DK.%2BM.%26atitle%3DPrinciples%2520for%2520targeting%2520RNA%2520with%2520drug-like%2520small%2520molecules%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D547%26epage%3D558%26doi%3D10.1038%2Fnrd.2018.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Begg, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barclay, M. L.</span></span> <span> </span><span class="NLM_article-title">Aminoglycosides - 50 years on</span>. <i>Br. J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">597</span>– <span class="NLM_lpage">603</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1111%2Fj.1365-2125.1995.tb05719.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=7654476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADyaK2MzotlSktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1995&pages=597-603&author=E.+J.+Beggauthor=M.+L.+Barclay&title=Aminoglycosides+-+50+years+on"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Aminoglycosides--50 years on</span></div><div class="casAuthors">Begg E J; Barclay M L</div><div class="citationInfo"><span class="NLM_cas:title">British journal of clinical pharmacology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">597-603</span>
        ISSN:<span class="NLM_cas:issn">0306-5251</span>.
    </div><div class="casAbstract">1.  The aminoglycoside antibiotics are 50 years old.  Their success and continuing use can be attributed to various factors including rapid concentration-dependent bactericidal effect, synergism with beta-lactam antibiotics, clinical effectiveness, a low rate of true resistance and low cost. 2.  The aminoglycosides remain drugs of choice in many circumstances including septicaemia, other serious infections due to Gram negative bacilli, and bacterial endocarditis. 3.  Nephrotoxicity and ototoxicity have been the main drawbacks clinically for the aminoglycosides. 4.  There has been an evolution in dosing strategies largely aimed at reducing toxicity.  Therapeutic drug monitoring has been used extensively to assist dosing, and target concentrations have been advocated, such as peak concentrations of between 6 and 10 mg l-1 and through concentrations of < 2 mg l-1 for gentamicin, tobramycin and netilmicin. 5.  Recently there has been a minor revolution in the approach to aminoglycoside dosing, with a change to larger doses, given less frequently.  In its most convenient form this is 'Once-daily aminoglycoside dosing'.  It offers the hope of better efficacy, less toxicity, and easier administration and monitoring. 6.  This article summarises the background of aminoglycoside usage, leading up to the recent changes in dosing strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR1Gxp5vfmgRW52AMpI4GK2fW6udTcc2eYs2j7M-XYexrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2MzotlSktw%253D%253D&md5=3e122c24a54f582a7ea1a26972519f4e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2125.1995.tb05719.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2125.1995.tb05719.x%26sid%3Dliteratum%253Aachs%26aulast%3DBegg%26aufirst%3DE.%2BJ.%26aulast%3DBarclay%26aufirst%3DM.%2BL.%26atitle%3DAminoglycosides%2520-%252050%2520years%2520on%26jtitle%3DBr.%2520J.%2520Clin.%2520Pharmacol.%26date%3D1995%26volume%3D39%26spage%3D597%26epage%3D603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labuda, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poplawski, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pushechnikov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span> <span> </span><span class="NLM_article-title">Two-dimensional combinatorial screening identifies specific aminoglycoside-RNA internal loop partners</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">11185</span>– <span class="NLM_lpage">11194</span>, <span class="refDoi"> DOI: 10.1021/ja803234t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja803234t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovFygu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=11185-11194&author=M.+D.+Disneyauthor=L.+P.+Labudaauthor=D.+J.+Paulauthor=S.+G.+Poplawskiauthor=A.+Pushechnikovauthor=T.+Tranauthor=S.+P.+Velagapudiauthor=M.+Wuauthor=J.+L.+Childs-Disney&title=Two-dimensional+combinatorial+screening+identifies+specific+aminoglycoside-RNA+internal+loop+partners&doi=10.1021%2Fja803234t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Two-Dimensional Combinatorial Screening Identifies Specific Aminoglycoside-RNA Internal Loop Partners</span></div><div class="casAuthors">Disney, Matthew D.; Labuda, Lucas P.; Paul, Dustin J.; Poplawski, Shane G.; Pushechnikov, Alexei; Tran, Tuan; Velagapudi, Sai P.; Wu, Meilan; Childs-Disney, Jessica L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">11185-11194</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein is described the identification of RNA internal loops that bind to derivs. of neomycin B, neamine, tobramycin, and kanamycin A.  RNA loop-ligand partners were identified by a two-dimensional combinatorial screening (2DCS) platform that probes RNA and chem. spaces simultaneously.  In 2DCS, an aminoglycoside library immobilized onto an agarose microarray was probed for binding to a 3 × 3 nucleotide RNA internal loop library (81 920 interactions probed in duplicate in a single expt.).  RNAs that bound aminoglycosides were harvested from the array via gel excision.  RNA internal loop preferences for three aminoglycosides were identified from statistical anal. of selected structures.  This provides consensus RNA internal loops that bind these structures and include: loops with potential GA pairs for the neomycin deriv., loops with potential GG pairs for the tobramycin deriv., and pyrimidine-rich loops for the kanamycin A deriv.  Results with the neamine deriv. show that it binds a variety of loops, including loops that contain potential GA pairs that also recognize the neomycin B deriv.  All studied selected internal loops are specific for the aminoglycoside that they were selected to bind.  Specificity was quantified for 16 selected internal loops by studying their binding to each of the arrayed aminoglycosides.  Specificities ranged from 2- to 80-fold with an av. specificity of 20-fold.  These studies show that 2DCS is a unique platform to probe RNA and chem. space simultaneously to identify specific RNA motif-ligand interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDfkvgnM2Yh7Vg90H21EOLACvtfcHk0lhbQlvIcLLkPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovFygu7Y%253D&md5=9910fb8215768c6ccda8a771a5d98ed4</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja803234t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja803234t%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DLabuda%26aufirst%3DL.%2BP.%26aulast%3DPaul%26aufirst%3DD.%2BJ.%26aulast%3DPoplawski%26aufirst%3DS.%2BG.%26aulast%3DPushechnikov%26aufirst%3DA.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26atitle%3DTwo-dimensional%2520combinatorial%2520screening%2520identifies%2520specific%2520aminoglycoside-RNA%2520internal%2520loop%2520partners%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D11185%26epage%3D11194%26doi%3D10.1021%2Fja803234t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Datlinger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendeiro, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krausgruber, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klughammer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alpar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, C.</span></span> <span> </span><span class="NLM_article-title">Pooled CRISPR screening with single-cell transcriptome readout</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1038/nmeth.4177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnmeth.4177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28099430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKjsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=297-301&author=P.+Datlingerauthor=A.+F.+Rendeiroauthor=C.+Schmidlauthor=T.+Krausgruberauthor=P.+Traxlerauthor=J.+Klughammerauthor=L.+C.+Schusterauthor=A.+Kuchlerauthor=D.+Alparauthor=C.+Bock&title=Pooled+CRISPR+screening+with+single-cell+transcriptome+readout&doi=10.1038%2Fnmeth.4177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Pooled CRISPR screening with single-cell transcriptome readout</span></div><div class="casAuthors">Datlinger, Paul; Rendeiro, Andre F.; Schmidl, Christian; Krausgruber, Thomas; Traxler, Peter; Klughammer, Johanna; Schuster, Linda C.; Kuchler, Amelie; Alpar, Donat; Bock, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">297-301</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">CRISPR-based genetic screens are accelerating biol. discovery, but current methods have inherent limitations.  Widely used pooled screens are restricted to simple readouts including cell proliferation and sortable marker proteins.  Arrayed screens allow for comprehensive mol. readouts such as transcriptome profiling, but at much lower throughput.  Here we combine pooled CRISPR screening with single-cell RNA sequencing into a broadly applicable workflow, directly linking guide RNA expression to transcriptome responses in thousands of individual cells.  Our method for CRISPR droplet sequencing (CROP-seq) enables pooled CRISPR screens with single-cell transcriptome resoln., which will facilitate high-throughput functional dissection of complex regulatory mechanisms and heterogeneous cell populations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGg_xnPKp1wbVg90H21EOLACvtfcHk0liT1jncMVtOIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKjsrk%253D&md5=5c32085acdcee491f18b47149713a43d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.4177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.4177%26sid%3Dliteratum%253Aachs%26aulast%3DDatlinger%26aufirst%3DP.%26aulast%3DRendeiro%26aufirst%3DA.%2BF.%26aulast%3DSchmidl%26aufirst%3DC.%26aulast%3DKrausgruber%26aufirst%3DT.%26aulast%3DTraxler%26aufirst%3DP.%26aulast%3DKlughammer%26aufirst%3DJ.%26aulast%3DSchuster%26aufirst%3DL.%2BC.%26aulast%3DKuchler%26aufirst%3DA.%26aulast%3DAlpar%26aufirst%3DD.%26aulast%3DBock%26aufirst%3DC.%26atitle%3DPooled%2520CRISPR%2520screening%2520with%2520single-cell%2520transcriptome%2520readout%26jtitle%3DNat.%2520Methods%26date%3D2017%26volume%3D14%26spage%3D297%26epage%3D301%26doi%3D10.1038%2Fnmeth.4177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">Building mammalian signalling pathways with RNAi screens</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1038/nrm1860</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrm1860" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=16496020" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhs1Kisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=177-187&author=J.+Moffatauthor=D.+M.+Sabatini&title=Building+mammalian+signalling+pathways+with+RNAi+screens&doi=10.1038%2Fnrm1860"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Building mammalian signalling pathways with RNAi screens</span></div><div class="casAuthors">Moffat, Jason; Sabatini, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">177-187</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Technol. advances in mammalian systems are providing new tools to identify the mol. components of signaling pathways.  Foremost among these tools is the ability to knock down gene function through the use of RNA interference (RNAi).  The fact that RNAi can be scaled up for use in high-throughput techniques has motivated the creation of genome-wide RNAi reagents.  We are now at the brink of being able to harness the power of RNAi for large-scale functional discovery in mammalian cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFu-CHw0OiGrVg90H21EOLACvtfcHk0liT1jncMVtOIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhs1Kisrg%253D&md5=f042bfc5a7c6091d14ed54a25e9d7cf7</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnrm1860&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1860%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DBuilding%2520mammalian%2520signalling%2520pathways%2520with%2520RNAi%2520screens%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2006%26volume%3D7%26spage%3D177%26epage%3D187%26doi%3D10.1038%2Fnrm1860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howe, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmann, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balibar, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galgoci, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malinverni, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayhood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villafania, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahvi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murgolo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbieri, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Painter, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherborne, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jesus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotkin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rindgen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlisi, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheth, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roemer, T.</span></span> <span> </span><span class="NLM_article-title">Selective small-molecule inhibition of an RNA structural element</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>526</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">677</span>, <span class="refDoi"> DOI: 10.1038/nature15542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnature15542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=26416753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=526&publication_year=2015&pages=672-677&author=J.+A.+Howeauthor=H.+Wangauthor=T.+O.+Fischmannauthor=C.+J.+Balibarauthor=L.+Xiaoauthor=A.+M.+Galgociauthor=J.+C.+Malinverniauthor=T.+Mayhoodauthor=A.+Villafaniaauthor=A.+Nahviauthor=N.+Murgoloauthor=C.+M.+Barbieriauthor=P.+A.+Mannauthor=D.+Carrauthor=E.+Xiaauthor=P.+Zuckauthor=D.+Rileyauthor=R.+E.+Painterauthor=S.+S.+Walkerauthor=B.+Sherborneauthor=R.+de+Jesusauthor=W.+Panauthor=M.+A.+Plotkinauthor=J.+Wuauthor=D.+Rindgenauthor=J.+Cummingsauthor=C.+G.+Garlisiauthor=R.+Zhangauthor=P.+R.+Shethauthor=C.+J.+Gillauthor=H.+Tangauthor=T.+Roemer&title=Selective+small-molecule+inhibition+of+an+RNA+structural+element&doi=10.1038%2Fnature15542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-molecule inhibition of an RNA structural element</span></div><div class="casAuthors">Howe, John A.; Wang, Hao; Fischmann, Thierry O.; Balibar, Carl J.; Xiao, Li; Galgoci, Andrew M.; Malinverni, Juliana C.; Mayhood, Todd; Villafania, Artjohn; Nahvi, Ali; Murgolo, Nicholas; Barbieri, Christopher M.; Mann, Paul A.; Carr, Donna; Xia, Ellen; Zuck, Paul; Riley, Dan; Painter, Ronald E.; Walker, Scott S.; Sherborne, Brad; de Jesus, Reynalda; Pan, Weidong; Plotkin, Michael A.; Wu, Jin; Rindgen, Diane; Cummings, John; Garlisi, Charles G.; Zhang, Rumin; Sheth, Payal R.; Gill, Charles J.; Tang, Haifeng; Roemer, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">526</span>
        (<span class="NLM_cas:issue">7575</span>),
    <span class="NLM_cas:pages">672-677</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Riboswitches are non-coding RNA structures located in mRNAs that bind endogenous ligands, such as a specific metabolite or ion, to regulate gene expression.  As such, riboswitches serve as a novel, yet largely unexploited, class of emerging drug targets.  Demonstrating this potential, however, has proven difficult and is restricted to structurally similar antimetabolites and semi-synthetic analogs of their cognate ligand, thus greatly restricting the chem. space and selectivity sought for such inhibitors.  Here we report the discovery and characterization of ribocil, a highly selective chem. modulator of bacterial riboflavin riboswitches, which was identified in a phenotypic screen and acts as a structurally distinct synthetic mimic of the natural ligand, FMN, to repress riboswitch-mediated ribB gene expression and inhibit bacterial cell growth.  Our findings indicate that non-coding RNA structural elements may be more broadly targeted by synthetic small mols. than previously expected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWptaVLK3njLVg90H21EOLACvtfcHk0lgnmDK6Y9nddg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7%252FL&md5=c0f8443348c10f69eb47f05a2ac5e893</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnature15542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15542%26sid%3Dliteratum%253Aachs%26aulast%3DHowe%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DFischmann%26aufirst%3DT.%2BO.%26aulast%3DBalibar%26aufirst%3DC.%2BJ.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DGalgoci%26aufirst%3DA.%2BM.%26aulast%3DMalinverni%26aufirst%3DJ.%2BC.%26aulast%3DMayhood%26aufirst%3DT.%26aulast%3DVillafania%26aufirst%3DA.%26aulast%3DNahvi%26aufirst%3DA.%26aulast%3DMurgolo%26aufirst%3DN.%26aulast%3DBarbieri%26aufirst%3DC.%2BM.%26aulast%3DMann%26aufirst%3DP.%2BA.%26aulast%3DCarr%26aufirst%3DD.%26aulast%3DXia%26aufirst%3DE.%26aulast%3DZuck%26aufirst%3DP.%26aulast%3DRiley%26aufirst%3DD.%26aulast%3DPainter%26aufirst%3DR.%2BE.%26aulast%3DWalker%26aufirst%3DS.%2BS.%26aulast%3DSherborne%26aufirst%3DB.%26aulast%3Dde%2BJesus%26aufirst%3DR.%26aulast%3DPan%26aufirst%3DW.%26aulast%3DPlotkin%26aufirst%3DM.%2BA.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DRindgen%26aufirst%3DD.%26aulast%3DCummings%26aufirst%3DJ.%26aulast%3DGarlisi%26aufirst%3DC.%2BG.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DSheth%26aufirst%3DP.%2BR.%26aulast%3DGill%26aufirst%3DC.%2BJ.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DRoemer%26aufirst%3DT.%26atitle%3DSelective%2520small-molecule%2520inhibition%2520of%2520an%2520RNA%2520structural%2520element%26jtitle%3DNature%26date%3D2015%26volume%3D526%26spage%3D672%26epage%3D677%26doi%3D10.1038%2Fnature15542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeFrancesco, L.</span></span> <span> </span><span class="NLM_article-title">Drug pipeline: 1Q17</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">400</span>, <span class="refDoi"> DOI: 10.1038/nbt.3874</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnbt.3874" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28486449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC1crlsFSnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=400&author=L.+DeFrancesco&title=Drug+pipeline%3A+1Q17&doi=10.1038%2Fnbt.3874"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Drug pipeline: 1Q17</span></div><div class="casAuthors">DeFrancesco Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature biotechnology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">400</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTg0zjb7i0zCHqj_yg7twvpfW6udTcc2eb893UBFRvvlrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crlsFSnsA%253D%253D&md5=0bf4b76fe97b1316e0312ba39455813c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3874&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3874%26sid%3Dliteratum%253Aachs%26aulast%3DDeFrancesco%26aufirst%3DL.%26atitle%3DDrug%2520pipeline%253A%25201Q17%26jtitle%3DNat.%2520Biotechnol.%26date%3D2017%26volume%3D35%26spage%3D400%26doi%3D10.1038%2Fnbt.3874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torricelli, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkel, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanigan, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goemans, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heydemann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muntoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osorio, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schara, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sejersen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shieh, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweeney, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topaloglu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tulinius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilchez, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elfring, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntosh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riebling, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiegel, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peltz, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercuri, E.</span></span> <span> </span><span class="NLM_article-title">Ataluren in patients with nonsense mutation Duchenne muscular dystrophy (ACT DMD): a multicentre, randomised, double-blind, placebo-controlled, phase 3 trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">1489</span>– <span class="NLM_lpage">1498</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(17)31611-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2FS0140-6736%2817%2931611-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28728956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2hsL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2017&pages=1489-1498&author=C.+M.+McDonaldauthor=C.+Campbellauthor=R.+E.+Torricelliauthor=R.+S.+Finkelauthor=K.+M.+Flaniganauthor=N.+Goemansauthor=P.+Heydemannauthor=A.+Kaminskaauthor=J.+Kirschnerauthor=F.+Muntoniauthor=A.+N.+Osorioauthor=U.+Scharaauthor=T.+Sejersenauthor=P.+B.+Shiehauthor=H.+L.+Sweeneyauthor=H.+Topalogluauthor=M.+Tuliniusauthor=J.+J.+Vilchezauthor=T.+Voitauthor=B.+Wongauthor=G.+Elfringauthor=H.+Krogerauthor=X.+Luoauthor=J.+McIntoshauthor=T.+Ongauthor=P.+Rieblingauthor=M.+Souzaauthor=R.+J.+Spiegelauthor=S.+W.+Peltzauthor=E.+Mercuri&title=Ataluren+in+patients+with+nonsense+mutation+Duchenne+muscular+dystrophy+%28ACT+DMD%29%3A+a+multicentre%2C+randomised%2C+double-blind%2C+placebo-controlled%2C+phase+3+trial&doi=10.1016%2FS0140-6736%2817%2931611-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Ataluren in patients with nonsense mutation Duchenne muscular dystrophy (ACT DMD): a multicentre, randomised, double-blind, placebo-controlled, phase 3 trial</span></div><div class="casAuthors">McDonald, Craig M.; Campbell, Craig; Torricelli, Ricardo Erazo; Finkel, Richard S.; Flanigan, Kevin M.; Goemans, Nathalie; Heydemann, Peter; Kaminska, Anna; Kirschner, Janbernd; Muntoni, Francesco; Osorio, Andres Nascimento; Schara, Ulrike; Sejersen, Thomas; Shieh, Perry B.; Sweeney, H. Lee; Topaloglu, Haluk; Tulinius, Mar; Vilchez, Juan J.; Voit, Thomas; Wong, Brenda; Elfring, Gary; Kroger, Hans; Luo, Xiaohui; McIntosh, Joseph; Ong, Tuyen; Riebling, Peter; Souza, Marcio; Spiegel, Robert J.; Peltz, Stuart W.; Mercuri, Eugenio</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">10101</span>),
    <span class="NLM_cas:pages">1489-1498</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Duchenne muscular dystrophy (DMD) is a severe, progressive, and rare neuromuscular, X-linked recessive disease.  Dystrophin deficiency is the underlying cause of disease; therefore, mutation-specific therapies aimed at restoring dystrophin protein prodn. are being explored.  We aimed to assess the efficacy and safety of ataluren in ambulatory boys with nonsense mutation DMD.  We did this multicentre, randomised, double-blind, placebo-controlled, phase 3 trial at 54 sites in 18 countries located in North America, Europe, the Asia-Pacific region, and Latin America.  Boys aged 7-16 years with nonsense mutation DMD and a baseline 6-min walk distance (6MWD) of 150 m or more and 80% or less of the predicted normal value for age and height were randomly assigned (1:1), via permuted block randomisation (block size of four) using an interactive voice-response or web-response system, to receive ataluren orally three times daily (40 mg/kg per day) or matching placebo.  Randomisation was stratified by age (<9 years vs ≥9 years), duration of previous corticosteroid use (6 mo to <12 mo vs ≥12 mo), and baseline 6MWD (<350 m vs ≥350 m).  Patients, parents and caregivers, investigational site personnel, PTC Therapeutics employees, and all other study personnel were masked to group allocation until after database lock.  The primary endpoint was change in 6MWD from baseline to week 48.  We addnl. did a prespecified subgroup anal. of the primary endpoint, based on baseline 6MWD, which is reflective of anticipated rates of disease progression over 1 yr.  The primary anal. was by intention to treat.  This study is registered with ClinicalTrials.gov, no. NCT01826487.  Between March 26, 2013, and Aug 26, 2014, we randomly assigned 230 patients to receive ataluren (n=115) or placebo (n=115); 228 patients comprised the intention-to-treat population.  The least-squares mean change in 6MWD from baseline to week 48 was -47·7 m (SE 9·3) for ataluren-treated patients and -60·7 m (9·3) for placebo-treated patients (difference 13·0 m [SE 10·4], 95% CI -7·4 to 33·4; p=0·213).  The least-squares mean change for ataluren vs. placebo in the prespecified subgroups was -7·7 m (SE 24·1, 95% CI -54·9 to 39·5; p=0·749) in the group with a 6MWD of less than 300 m, 42·9 m (15·9, 11·8-74·0; p=0·007) in the group with a 6MWD of 300 m or more to less than 400 m, and -9·5 m (17·2, -43·2 to 24·2; p=0·580) in the group with a 6MWD of 400 m or more.  Ataluren was generally well tolerated and most treatment-emergent adverse events were mild to moderate in severity.  Eight (3%) patients (n=4 per group) reported serious adverse events; all except one event in the placebo group (abnormal hepatic function deemed possibly related to treatment) were deemed unrelated to treatment.  Change in 6MWD did not differ significantly between patients in the ataluren group and those in the placebo group, neither in the intention-to-treat population nor in the prespecified subgroups with a baseline 6MWD of less than 300 m or 400 m or more.  However, we recorded a significant effect of ataluren in the prespecified subgroup of patients with a baseline 6MWD of 300 m or more to less than 400 m.  Baseline 6MWD values within this range were assocd. with a more predictable rate of decline over 1 yr; this finding has implications for the design of future DMD trials with the 6-min walk test as the endpoint.PTC Therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrdEeegEaHJLVg90H21EOLACvtfcHk0lg54dsZ5LVZiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2hsL3N&md5=25ae69e359455aca9d716e964aa09015</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2817%2931611-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252817%252931611-2%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DC.%2BM.%26aulast%3DCampbell%26aufirst%3DC.%26aulast%3DTorricelli%26aufirst%3DR.%2BE.%26aulast%3DFinkel%26aufirst%3DR.%2BS.%26aulast%3DFlanigan%26aufirst%3DK.%2BM.%26aulast%3DGoemans%26aufirst%3DN.%26aulast%3DHeydemann%26aufirst%3DP.%26aulast%3DKaminska%26aufirst%3DA.%26aulast%3DKirschner%26aufirst%3DJ.%26aulast%3DMuntoni%26aufirst%3DF.%26aulast%3DOsorio%26aufirst%3DA.%2BN.%26aulast%3DSchara%26aufirst%3DU.%26aulast%3DSejersen%26aufirst%3DT.%26aulast%3DShieh%26aufirst%3DP.%2BB.%26aulast%3DSweeney%26aufirst%3DH.%2BL.%26aulast%3DTopaloglu%26aufirst%3DH.%26aulast%3DTulinius%26aufirst%3DM.%26aulast%3DVilchez%26aufirst%3DJ.%2BJ.%26aulast%3DVoit%26aufirst%3DT.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DElfring%26aufirst%3DG.%26aulast%3DKroger%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DMcIntosh%26aufirst%3DJ.%26aulast%3DOng%26aufirst%3DT.%26aulast%3DRiebling%26aufirst%3DP.%26aulast%3DSouza%26aufirst%3DM.%26aulast%3DSpiegel%26aufirst%3DR.%2BJ.%26aulast%3DPeltz%26aufirst%3DS.%2BW.%26aulast%3DMercuri%26aufirst%3DE.%26atitle%3DAtaluren%2520in%2520patients%2520with%2520nonsense%2520mutation%2520Duchenne%2520muscular%2520dystrophy%2520%2528ACT%2520DMD%2529%253A%2520a%2520multicentre%252C%2520randomised%252C%2520double-blind%252C%2520placebo-controlled%252C%2520phase%25203%2520trial%26jtitle%3DLancet%26date%3D2017%26volume%3D390%26spage%3D1489%26epage%3D1498%26doi%3D10.1016%2FS0140-6736%2817%2931611-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span> <span> </span><span class="NLM_article-title">Mechanism of PTC124 activity in cell-based luciferase assays of nonsense codon suppression</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">3585</span>– <span class="NLM_lpage">3590</span>, <span class="refDoi"> DOI: 10.1073/pnas.0813345106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.0813345106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19208811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivFGqt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=3585-3590&author=D.+S.+Auldauthor=N.+Thorneauthor=W.+F.+Maguireauthor=J.+Inglese&title=Mechanism+of+PTC124+activity+in+cell-based+luciferase+assays+of+nonsense+codon+suppression&doi=10.1073%2Fpnas.0813345106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of PTC124 activity in cell-based lucifierase assays of nonsense codon suppression</span></div><div class="casAuthors">Auld, Douglas S.; Thorne, Natasha; Maguire, William F.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3585-3590</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) assays used in drug discovery frequently use reporter enzymes such as firefly luciferase (FLuc) as indicators of target activity.  An important caveat to consider, however, is that compds. can directly affect the reporter, leading to nonspecific but highly reproducible assay signal modulation.  In rare cases, this activity appears counterintuitive; for example, some FLuc inhibitors, acting through posttranslational Fluc reporter stabilization, appear to activate gene expression.  Previous efforts to characterize mols. that influence luciferase activity identified a subset of 3,5-diaryl-oxadiazole-contg. compds. as FLuc inhibitors.  Here, we evaluate a no. of compds. with this structural motif for activity against FLuc.  One such compd. is PTC124 {3-[5-(2-fluorophenyl)-1,2,4-oxadiazol-3-yl]benzoic acid}, a mol. originally identified in a cell-based FLuc assay as having nonsense codon suppression activity.  We find that the potency of FLuc inhibition for the tested compds. strictly correlates with their activity in a FLuc reporter cell-based nonsense codon assay, with PTC124 emerging as the most potent FLuc inhibitor (IC50 = 7 ± 1 nM).  However, these compds., including PTC124, fail to show nonsense codon suppression activity when Renilla reniformis luciferase (RLuc) is used as a reporter and are inactive against the RLuc enzyme.  This suggests that the initial discovery of PTC124 may have been biased by its direct effect on the FLuc reporter, implicating firefly luciferase as a mol. target of PTC124.  Our results demonstrate the value of understanding potential interactions between reporter enzymes and chem. compds. and emphasize the importance of implementing the appropriate control assays before interpreting HTS results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVYgPZRfbQR7Vg90H21EOLACvtfcHk0lhhCuPUdpQQtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivFGqt74%253D&md5=8fd46b333035a4454e61ae600271fba8</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0813345106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0813345106%26sid%3Dliteratum%253Aachs%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DMaguire%26aufirst%3DW.%2BF.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DMechanism%2520of%2520PTC124%2520activity%2520in%2520cell-based%2520luciferase%2520assays%2520of%2520nonsense%2520codon%2520suppression%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D3585%26epage%3D3590%26doi%3D10.1073%2Fpnas.0813345106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lea, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanzlik, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for the high-affinity binding and stabilization of firefly luciferase by PTC124</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">4878</span>– <span class="NLM_lpage">4883</span>, <span class="refDoi"> DOI: 10.1073/pnas.0909141107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.0909141107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=20194791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFaqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=4878-4883&author=D.+S.+Auldauthor=S.+Lovellauthor=N.+Thorneauthor=W.+A.+Leaauthor=D.+J.+Maloneyauthor=M.+Shenauthor=G.+Raiauthor=K.+P.+Battaileauthor=C.+J.+Thomasauthor=A.+Simeonovauthor=R.+P.+Hanzlikauthor=J.+Inglese&title=Molecular+basis+for+the+high-affinity+binding+and+stabilization+of+firefly+luciferase+by+PTC124&doi=10.1073%2Fpnas.0909141107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for the high-affinity binding and stabilization of firefly luciferase by PTC124</span></div><div class="casAuthors">Auld, Douglas S.; Lovell, Scott; Thorne, Natasha; Lea, Wendy A.; Maloney, David J.; Shen, Min; Rai, Ganesha; Battaile, Kevin P.; Thomas, Craig J.; Simeonov, Anton; Hanzlik, Robert P.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4878-4883, S4878/1-S4878/14</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Firefly luciferase (FLuc), an ATP-dependent bioluminescent reporter enzyme, is broadly used in chem. biol. and drug discovery assays.  PTC124 (Ataluren; (3-[5-(2-fluorophenyl)-1,2,4-oxadiazol-3-yl]benzoic acid)) discovered in an FLuc-based assay targeting nonsense codon suppression, is an unusually potent FLuc-inhibitor.  Paradoxically, PTC124 and related analogs increase cellular FLuc activity levels by posttranslational stabilization.  In this study, we show that FLuc inhibition and stabilization is the result of an inhibitory product formed during the FLuc-catalyzed reaction between its natural substrate, ATP, and PTC124.  A 2.0 Å cocrystal structure revealed the inhibitor to be the acyl-AMP mixed-anhydride adduct PTC124-AMP, which was subsequently synthesized and shown to be a high-affinity multisubstrate adduct inhibitor (MAI; KD = 120 pM) of FLuc.  Biochem. assays, liq. chromatog./mass spectrometry, and near-attack conformer modeling demonstrate that formation of this novel MAI is absolutely dependent upon the precise positioning and reactivity of a key meta-carboxylate of PTC124 within the FLuc active site.  We also demonstrate that the inhibitory activity of PTC124-AMP is relieved by free CoA, a component present at high concns. in luciferase detection reagents used for cell-based assays.  This explains why PTC124 can appear to increase, instead of inhibit, FLuc activity in cell-based reporter gene assays.  To our knowledge, this is an unusual example in which the "off-target" effect of a small mol. is mediated by an MAI mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIPfC1eBndl7Vg90H21EOLACvtfcHk0lhhCuPUdpQQtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFaqsbo%253D&md5=e22180fb96672f3fa7cdc907b7ada4fe</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0909141107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0909141107%26sid%3Dliteratum%253Aachs%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DLea%26aufirst%3DW.%2BA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHanzlik%26aufirst%3DR.%2BP.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DMolecular%2520basis%2520for%2520the%2520high-affinity%2520binding%2520and%2520stabilization%2520of%2520firefly%2520luciferase%2520by%2520PTC124%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D4878%26epage%3D4883%26doi%3D10.1073%2Fpnas.0909141107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroth, M.</span></span> <span> </span><span class="NLM_article-title">Gene therapy for spinal muscular atrophy: an emerging treatment option for a devastating disease</span>. <i>J. Manag. Care Spec. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">S3</span>– <span class="NLM_lpage">S16</span>, <span class="refDoi"> DOI: 10.18553/jmcp.2018.24.12-a.s3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.18553%2Fjmcp.2018.24.12-a.s3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30582825" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=S3-S16&author=V.+K.+Raoauthor=D.+Kappauthor=M.+Schroth&title=Gene+therapy+for+spinal+muscular+atrophy%3A+an+emerging+treatment+option+for+a+devastating+disease&doi=10.18553%2Fjmcp.2018.24.12-a.s3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.18553%2Fjmcp.2018.24.12-a.s3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18553%252Fjmcp.2018.24.12-a.s3%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DV.%2BK.%26aulast%3DKapp%26aufirst%3DD.%26aulast%3DSchroth%26aufirst%3DM.%26atitle%3DGene%2520therapy%2520for%2520spinal%2520muscular%2520atrophy%253A%2520an%2520emerging%2520treatment%2520option%2520for%2520a%2520devastating%2520disease%26jtitle%3DJ.%2520Manag.%2520Care%2520Spec.%2520Pharm.%26date%3D2018%26volume%3D24%26spage%3DS3%26epage%3DS16%26doi%3D10.18553%2Fjmcp.2018.24.12-a.s3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerrigan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axford, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cody, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridrich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmaster, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoosear, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servais, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivasankaran, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dales, N. A.</span></span> <span> </span><span class="NLM_article-title">Discovery of small molecule splicing modulators of survival motor neuron-2 (SMN2) for the treatment of spinal muscular atrophy (SMA)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">11021</span>– <span class="NLM_lpage">11036</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01291</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01291" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFeju7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=11021-11036&author=A.+K.+Cheungauthor=B.+Hurleyauthor=R.+Kerriganauthor=L.+Shuauthor=D.+N.+Chinauthor=Y.+Shenauthor=G.+O%E2%80%99Brienauthor=M.+J.+Sungauthor=Y.+Houauthor=J.+Axfordauthor=E.+Codyauthor=R.+Sunauthor=A.+Fazalauthor=C.+Fridrichauthor=C.+C.+Sanchezauthor=R.+C.+Tomlinsonauthor=M.+Jainauthor=L.+Dengauthor=K.+Hoffmasterauthor=C.+Songauthor=M.+Van+Hoosearauthor=Y.+Shinauthor=R.+Servaisauthor=C.+Towlerauthor=M.+Hildauthor=D.+Curtisauthor=W.+F.+Dietrichauthor=L.+G.+Hamannauthor=K.+Brinerauthor=K.+S.+Chenauthor=D.+Kobayashiauthor=R.+Sivasankaranauthor=N.+A.+Dales&title=Discovery+of+small+molecule+splicing+modulators+of+survival+motor+neuron-2+%28SMN2%29+for+the+treatment+of+spinal+muscular+atrophy+%28SMA%29&doi=10.1021%2Facs.jmedchem.8b01291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule Splicing Modulators of Survival Motor Neuron-2 (SMN2) for the Treatment of Spinal Muscular Atrophy (SMA)</span></div><div class="casAuthors">Cheung, Atwood K.; Hurley, Brian; Kerrigan, Ryan; Shu, Lei; Chin, Donovan N.; Shen, Yiping; OBrien, Gary; Sung, Moo Je; Hou, Ying; Axford, Jake; Cody, Emma; Sun, Robert; Fazal, Aleem; Tomlinson, Ronald C.; Jain, Monish; Deng, Lin; Hoffmaster, Keith; Song, Cheng; Van Hoosear, Mailin; Shin, Youngah; Servais, Rebecca; Towler, Christopher; Hild, Marc; Curtis, Daniel; Dietrich, William F.; Hamann, Lawrence G.; Briner, Karin; Chen, Karen S.; Kobayashi, Dione; Sivasankaran, Rajeev; Dales, Natalie A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11021-11036</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA), a rare neuromuscular disorder, is the leading genetic cause of death in infants and toddlers.  SMA is caused by the deletion or a loss of function mutation of the survival motor neuron 1 (SMN1) gene.  In humans, a second closely related gene SMN2 exists, however it codes for a less stable SMN protein.  In recent years, significant progress has been made toward disease modifying treatments for SMA by modulating SMN2 pre-mRNA splicing.  Herein, we describe the discovery of LMI070/branaplam, a small mol. that stabilizes the interaction between the spliceosome and SMN2 pre-mRNA.  Branaplam (1) originated from a high-throughput phenotypic screening hit, pyridazine 2, and evolved via multi-parameter lead optimization.  In a severe mouse SMA model, branaplam treatment increased full-length SMN RNA and protein levels, and extended survival.  Currently, branaplam is in clin. studies for SMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprrHPkSEhGhrVg90H21EOLACvtfcHk0ljg4nVpX9qk5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFeju7nL&md5=c564332b680a6ef0ad06bc9e5a4db16e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01291%26sid%3Dliteratum%253Aachs%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DHurley%26aufirst%3DB.%26aulast%3DKerrigan%26aufirst%3DR.%26aulast%3DShu%26aufirst%3DL.%26aulast%3DChin%26aufirst%3DD.%2BN.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DG.%26aulast%3DSung%26aufirst%3DM.%2BJ.%26aulast%3DHou%26aufirst%3DY.%26aulast%3DAxford%26aufirst%3DJ.%26aulast%3DCody%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DR.%26aulast%3DFazal%26aufirst%3DA.%26aulast%3DFridrich%26aufirst%3DC.%26aulast%3DSanchez%26aufirst%3DC.%2BC.%26aulast%3DTomlinson%26aufirst%3DR.%2BC.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DHoffmaster%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DVan%2BHoosear%26aufirst%3DM.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DServais%26aufirst%3DR.%26aulast%3DTowler%26aufirst%3DC.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DCurtis%26aufirst%3DD.%26aulast%3DDietrich%26aufirst%3DW.%2BF.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DBriner%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DK.%2BS.%26aulast%3DKobayashi%26aufirst%3DD.%26aulast%3DSivasankaran%26aufirst%3DR.%26aulast%3DDales%26aufirst%3DN.%2BA.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520splicing%2520modulators%2520of%2520survival%2520motor%2520neuron-2%2520%2528SMN2%2529%2520for%2520the%2520treatment%2520of%2520spinal%2520muscular%2520atrophy%2520%2528SMA%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D11021%26epage%3D11036%26doi%3D10.1021%2Facs.jmedchem.8b01291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charnas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voltz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berghs-Clairmont, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Praestgaard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Raspide, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deconinck, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Born, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranello, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schara, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goemans, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roubenoff, R.</span></span> <span> </span><span class="NLM_article-title">Safety and efficacy findings in the first-in-human trial (FIH) of the oral splice modulator branaplam in type 1 spinal muscular atrophy (SMA): interim results</span>. <i>Neuromuscul. Disord.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">S207</span>– <span class="NLM_lpage">S208</span>, <span class="refDoi"> DOI: 10.1016/j.nmd.2017.06.411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.nmd.2017.06.411" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=S207-S208&author=L.+Charnasauthor=E.+Voltzauthor=C.+Pfisterauthor=T.+Petersauthor=A.+Hartmannauthor=C.+Berghs-Clairmontauthor=J.+Praestgaardauthor=M.+de+Raspideauthor=N.+Deconinckauthor=A.+Bornauthor=G.+Baranelloauthor=E.+Bertiniauthor=U.+Scharaauthor=N.+Goemansauthor=R.+Roubenoff&title=Safety+and+efficacy+findings+in+the+first-in-human+trial+%28FIH%29+of+the+oral+splice+modulator+branaplam+in+type+1+spinal+muscular+atrophy+%28SMA%29%3A+interim+results&doi=10.1016%2Fj.nmd.2017.06.411"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.nmd.2017.06.411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nmd.2017.06.411%26sid%3Dliteratum%253Aachs%26aulast%3DCharnas%26aufirst%3DL.%26aulast%3DVoltz%26aufirst%3DE.%26aulast%3DPfister%26aufirst%3DC.%26aulast%3DPeters%26aufirst%3DT.%26aulast%3DHartmann%26aufirst%3DA.%26aulast%3DBerghs-Clairmont%26aufirst%3DC.%26aulast%3DPraestgaard%26aufirst%3DJ.%26aulast%3Dde%2BRaspide%26aufirst%3DM.%26aulast%3DDeconinck%26aufirst%3DN.%26aulast%3DBorn%26aufirst%3DA.%26aulast%3DBaranello%26aufirst%3DG.%26aulast%3DBertini%26aufirst%3DE.%26aulast%3DSchara%26aufirst%3DU.%26aulast%3DGoemans%26aufirst%3DN.%26aulast%3DRoubenoff%26aufirst%3DR.%26atitle%3DSafety%2520and%2520efficacy%2520findings%2520in%2520the%2520first-in-human%2520trial%2520%2528FIH%2529%2520of%2520the%2520oral%2520splice%2520modulator%2520branaplam%2520in%2520type%25201%2520spinal%2520muscular%2520atrophy%2520%2528SMA%2529%253A%2520interim%2520results%26jtitle%3DNeuromuscul.%2520Disord.%26date%3D2017%26volume%3D27%26spage%3DS207%26epage%3DS208%26doi%3D10.1016%2Fj.nmd.2017.06.411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, K. A.</span></span> <span> </span><span class="NLM_article-title">Mechanistic studies of a small-molecule modulator of SMN2 splicing</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">E4604</span>– <span class="NLM_lpage">E4612</span>, <span class="refDoi"> DOI: 10.1073/pnas.1800260115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1800260115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29712837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVGhtLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=E4604-E4612&author=J.+Wangauthor=P.+G.+Schultzauthor=K.+A.+Johnson&title=Mechanistic+studies+of+a+small-molecule+modulator+of+SMN2+splicing&doi=10.1073%2Fpnas.1800260115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic studies of a small-molecule modulator of SMN2 splicing</span></div><div class="casAuthors">Wang, Jingxin; Schultz, Peter G.; Johnson, Kristen A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">E4604-E4612</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">RG-7916 is a first-in-class drug candidate for the treatment of spinal muscular atrophy (SMA) that functions by modulating pre-mRNA splicing of the SMN2 gene, resulting in a 2.5-fold increase in survival of motor neuron (SMN) protein level, a key protein lacking in SMA patients.  RG-7916 is currently in three interventional phase 2 clin. trials for various types of SMA.  In this report, we show that SMN-C2 and -C3, close analogs of RG-7916, act as selective RNA-binding ligands that modulate pre-mRNA splicing.  Chem. proteomic and genomic techniques reveal that SMN-C2 directly binds to the AGGAAG motif on exon 7 of the SMN2 pre-mRNA, and promotes a conformational change in two to three unpaired nucleotides at the junction of intron 6 and exon 7 in both in vitro and in-cell models.  This change creates a new functional binding surface that increases binding of the splicing modulators, far upstream element binding protein 1 (FUBP1) and its homolog, KH-type splicing regulatory protein (KHSRP), to the SMN-C2/C3-SMN2 pre-mRNA complex and enhances SMN2 splicing.  These findings underscore the potential of small-mol. drugs to selectively bind RNA and modulate pre-mRNA splicing as an approach to the treatment of human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh37iFocFbJbVg90H21EOLACvtfcHk0lggvT8HLe07lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVGhtLjM&md5=989d9a1be671358c31a9c957768272df</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1800260115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1800260115%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DJohnson%26aufirst%3DK.%2BA.%26atitle%3DMechanistic%2520studies%2520of%2520a%2520small-molecule%2520modulator%2520of%2520SMN2%2520splicing%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2018%26volume%3D115%26spage%3DE4604%26epage%3DE4612%26doi%3D10.1073%2Fpnas.1800260115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sivaramakrishnan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campagne, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heckel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meistermann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hug, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birrer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moursy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khawaja, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmucki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berntenis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giroud, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzouros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banfai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duran-Pacheco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamerz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiu Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luebbers, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebeling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cléry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paushkin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krainer, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, F. H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, F.</span></span> <span> </span><span class="NLM_article-title">Binding to SMN2 pre-mRNA-protein complex elicits specificity for small molecule splicing modifiers</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1476</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01559-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fs41467-017-01559-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29133793" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC1M3gvVagtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1476&author=M.+Sivaramakrishnanauthor=K.+D.+McCarthyauthor=S.+Campagneauthor=S.+Huberauthor=S.+Meierauthor=A.+Augustinauthor=T.+Heckelauthor=H.+Meistermannauthor=M.+N.+Hugauthor=P.+Birrerauthor=A.+Moursyauthor=S.+Khawajaauthor=R.+Schmuckiauthor=N.+Berntenisauthor=N.+Giroudauthor=S.+Gollingauthor=M.+Tzourosauthor=B.+Banfaiauthor=G.+Duran-Pachecoauthor=J.+Lamerzauthor=Y.+Hsiu+Liuauthor=T.+Luebbersauthor=H.+Ratniauthor=M.+Ebelingauthor=A.+Cl%C3%A9ryauthor=S.+Paushkinauthor=A.+R.+Krainerauthor=F.+H.+T.+Allainauthor=F.+Metzger&title=Binding+to+SMN2+pre-mRNA-protein+complex+elicits+specificity+for+small+molecule+splicing+modifiers&doi=10.1038%2Fs41467-017-01559-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Binding to SMN2 pre-mRNA-protein complex elicits specificity for small molecule splicing modifiers</span></div><div class="casAuthors">Sivaramakrishnan Manaswini; McCarthy Kathleen D; Huber Sylwia; Meier Sonja; Augustin Angelique; Heckel Tobias; Meistermann Helene; Hug Melanie N; Birrer Pascale; Schmucki Roland; Berntenis Nikos; Giroud Nicolas; Golling Sabrina; Tzouros Manuel; Banfai Balazs; Duran-Pacheco Gonzalo; Lamerz Jens; Luebbers Thomas; Ratni Hasane; Ebeling Martin; Metzger Friedrich; Sivaramakrishnan Manaswini; Meier Sonja; Hsiu Liu Ying; Krainer Adrian R; Campagne Sebastien; Moursy Ahmed; Khawaja Sarah; Clery Antoine; Allain Frederic H-T; Paushkin Sergey</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1476</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Small molecule splicing modifiers have been previously described that target the general splicing machinery and thus have low specificity for individual genes.  Several potent molecules correcting the splicing deficit of the SMN2 (survival of motor neuron 2) gene have been identified and these molecules are moving towards a potential therapy for spinal muscular atrophy (SMA).  Here by using a combination of RNA splicing, transcription, and protein chemistry techniques, we show that these molecules directly bind to two distinct sites of the SMN2 pre-mRNA, thereby stabilizing a yet unidentified ribonucleoprotein (RNP) complex that is critical to the specificity of these small molecules for SMN2 over other genes.  In addition to the therapeutic potential of these molecules for treatment of SMA, our work has wide-ranging implications in understanding how small molecules can interact with specific quaternary RNA structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR8ACsymX9-A2BAiYWy6n3SfW6udTcc2eap_t_6JOMg2rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3gvVagtA%253D%253D&md5=b739bbdecf61fd3f3ac20966ac3798fe</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01559-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01559-4%26sid%3Dliteratum%253Aachs%26aulast%3DSivaramakrishnan%26aufirst%3DM.%26aulast%3DMcCarthy%26aufirst%3DK.%2BD.%26aulast%3DCampagne%26aufirst%3DS.%26aulast%3DHuber%26aufirst%3DS.%26aulast%3DMeier%26aufirst%3DS.%26aulast%3DAugustin%26aufirst%3DA.%26aulast%3DHeckel%26aufirst%3DT.%26aulast%3DMeistermann%26aufirst%3DH.%26aulast%3DHug%26aufirst%3DM.%2BN.%26aulast%3DBirrer%26aufirst%3DP.%26aulast%3DMoursy%26aufirst%3DA.%26aulast%3DKhawaja%26aufirst%3DS.%26aulast%3DSchmucki%26aufirst%3DR.%26aulast%3DBerntenis%26aufirst%3DN.%26aulast%3DGiroud%26aufirst%3DN.%26aulast%3DGolling%26aufirst%3DS.%26aulast%3DTzouros%26aufirst%3DM.%26aulast%3DBanfai%26aufirst%3DB.%26aulast%3DDuran-Pacheco%26aufirst%3DG.%26aulast%3DLamerz%26aufirst%3DJ.%26aulast%3DHsiu%2BLiu%26aufirst%3DY.%26aulast%3DLuebbers%26aufirst%3DT.%26aulast%3DRatni%26aufirst%3DH.%26aulast%3DEbeling%26aufirst%3DM.%26aulast%3DCl%25C3%25A9ry%26aufirst%3DA.%26aulast%3DPaushkin%26aufirst%3DS.%26aulast%3DKrainer%26aufirst%3DA.%2BR.%26aulast%3DAllain%26aufirst%3DF.%2BH.%2BT.%26aulast%3DMetzger%26aufirst%3DF.%26atitle%3DBinding%2520to%2520SMN2%2520pre-mRNA-protein%2520complex%2520elicits%2520specificity%2520for%2520small%2520molecule%2520splicing%2520modifiers%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1476%26doi%3D10.1038%2Fs41467-017-01559-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of non-ATG translational events in cells via covalent small molecules targeting RNA</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">5336</span>– <span class="NLM_lpage">5345</span>, <span class="refDoi"> DOI: 10.1021/ja507448y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja507448y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsFGnurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=5336-5345&author=W.-Y.+Yangauthor=H.+D.+Wilsonauthor=S.+P.+Velagapudiauthor=M.+D.+Disney&title=Inhibition+of+non-ATG+translational+events+in+cells+via+covalent+small+molecules+targeting+RNA&doi=10.1021%2Fja507448y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Non-ATG Translational Events in Cells via Covalent Small Molecules Targeting RNA</span></div><div class="casAuthors">Yang, Wang-Yong; Wilson, Henry D.; Velagapudi, Sai Pradeep; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5336-5345</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One major class of disease-causing RNAs is expanded repeating transcripts.  These RNAs cause diseases via multiple mechanisms, including: (i) gain-of-function, in which repeating RNAs bind and sequester proteins involved in RNA biogenesis and (ii) repeat assocd. non-ATG (RAN) translation, in which repeating transcripts are translated into toxic proteins without use of a canonical, AUG, start codon.  Herein, the authors develop and study chem. probes that bind and react with an expanded r(CGG) repeat (r(CGG)exp) present in a 5' untranslated region that causes fragile X-assocd. tremor/ataxia syndrome (FXTAS).  Reactive compds. bind to r(CGG)expin cellulo as shown with Chem-CLIP-Map, an approach to map small mol. binding sites within RNAs in cells.  Compds. also potently improve FXTAS-assocd. premRNA splicing and RAN translational defects, while not affecting translation of the downstream open reading frame.  In contrast, oligonucleotides affect both RAN and canonical translation when they bind to r(CGG)exp, which is mechanistically traced to a decrease in polysome loading.  Thus, designer small mols. that react with RNA targets can be used to profile the RNAs to which they bind in cells, including identification of binding sites, and can modulate several aspects of RNA-mediated disease pathol. in a manner that may be more beneficial than oligonucleotides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2Of8BkHmwpbVg90H21EOLACvtfcHk0ljpI2rrc9me-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsFGnurY%253D&md5=432505c32da2493c2e77960be352d806</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fja507448y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja507448y%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.-Y.%26aulast%3DWilson%26aufirst%3DH.%2BD.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DInhibition%2520of%2520non-ATG%2520translational%2520events%2520in%2520cells%2520via%2520covalent%2520small%2520molecules%2520targeting%2520RNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D5336%26epage%3D5345%26doi%3D10.1021%2Fja507448y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holley, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apgar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Everett, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madison, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquisee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merrill, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penswick, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamir, A.</span></span> <span> </span><span class="NLM_article-title">Structure of a ribonucleic acid</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1965</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">1462</span>– <span class="NLM_lpage">1465</span>, <span class="refDoi"> DOI: 10.1126/science.147.3664.1462</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1126%2Fscience.147.3664.1462" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=14263761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaF2MXotlOlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=1965&pages=1462-1465&author=R.+W.+Holleyauthor=J.+Apgarauthor=G.+A.+Everettauthor=J.+T.+Madisonauthor=M.+Marquiseeauthor=S.+H.+Merrillauthor=J.+R.+Penswickauthor=A.+Zamir&title=Structure+of+a+ribonucleic+acid&doi=10.1126%2Fscience.147.3664.1462"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a ribonucleic acid</span></div><div class="casAuthors">Holley, Robert W.; Apgar, Jean; Everett, George A.; Madison, James T.; Marquisee, Mark; Merrill, Susan H.; Penswick, John Robert; Zamir, Ada</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1965</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">3664</span>),
    <span class="NLM_cas:pages">1462-5</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">Structure was detd. for alanyl RNA with pancreatic RNase and Taka-Diastase RNase Tl to digest small fragments followed by successively larger fragments.  The RNA is described in terms of 16 oligonucleotide sequences.  The presence of a 5'-phosphate on 1 sequence establishes it as the left end of the mol., while a 3'-hydroxyl establishes the right end.  The sequence of the 16 oligonucleotides is not known.  There is a total of 77 nucleotide residues to give a mol. wt. of 26,600 as the Na salt.  There is no pattern in the distribution of the minor residues; they are scattered throughout most of the mol. though none is near the amino acid-acceptor end.  There are no long nucleotide sequences so that double-stranded regions are either short or contain imperfections in base pairing.  A detailed description of each sequence and the method of detn. is included.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrysckjLwWNQ7Vg90H21EOLACvtfcHk0lg4qdwHERVjRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2MXotlOlug%253D%253D&md5=9f12aded94c0643eb0ee8a9330813b0d</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fscience.147.3664.1462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.147.3664.1462%26sid%3Dliteratum%253Aachs%26aulast%3DHolley%26aufirst%3DR.%2BW.%26aulast%3DApgar%26aufirst%3DJ.%26aulast%3DEverett%26aufirst%3DG.%2BA.%26aulast%3DMadison%26aufirst%3DJ.%2BT.%26aulast%3DMarquisee%26aufirst%3DM.%26aulast%3DMerrill%26aufirst%3DS.%2BH.%26aulast%3DPenswick%26aufirst%3DJ.%2BR.%26aulast%3DZamir%26aufirst%3DA.%26atitle%3DStructure%2520of%2520a%2520ribonucleic%2520acid%26jtitle%3DScience%26date%3D1965%26volume%3D147%26spage%3D1462%26epage%3D1465%26doi%3D10.1126%2Fscience.147.3664.1462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RajBhandary, U. L.</span></span> <span> </span><span class="NLM_article-title">Transfer RNA: molecular structure, sequence, and properties</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">860</span>, <span class="refDoi"> DOI: 10.1146/annurev.bi.45.070176.004105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1146%2Fannurev.bi.45.070176.004105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=60910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaE28XkvVylsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=1976&pages=805-860&author=A.+Richauthor=U.+L.+RajBhandary&title=Transfer+RNA%3A+molecular+structure%2C+sequence%2C+and+properties&doi=10.1146%2Fannurev.bi.45.070176.004105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Transfer RNA:  molecular structure, sequence, and properties</span></div><div class="casAuthors">Rich, Alexander; Bhandary, U. L. Raj</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">805-60</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    </div><div class="casAbstract">A review with 320 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF4j8tnvuc9rVg90H21EOLACvtfcHk0lg4qdwHERVjRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XkvVylsbg%253D&md5=4ad97acf33f3344229b5a4b9f5bed322</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1146%2Fannurev.bi.45.070176.004105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.bi.45.070176.004105%26sid%3Dliteratum%253Aachs%26aulast%3DRich%26aufirst%3DA.%26aulast%3DRajBhandary%26aufirst%3DU.%2BL.%26atitle%3DTransfer%2520RNA%253A%2520molecular%2520structure%252C%2520sequence%252C%2520and%2520properties%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D1976%26volume%3D45%26spage%3D805%26epage%3D860%26doi%3D10.1146%2Fannurev.bi.45.070176.004105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyle, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mount, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steitz, J. A.</span></span> <span> </span><span class="NLM_article-title">Are snRNPs involved in splicing?</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">220</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1038/283220a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2F283220a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=7350545" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaL3cXhsF2jtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=1980&pages=220-224&author=M.+R.+Lernerauthor=J.+A.+Boyleauthor=S.+M.+Mountauthor=S.+L.+Wolinauthor=J.+A.+Steitz&title=Are+snRNPs+involved+in+splicing%3F&doi=10.1038%2F283220a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Are snRNPs involved in splicing?</span></div><div class="casAuthors">Lerner, Michael R.; Boyle, John A.; Mount, Stephen M.; Wolin, Sandra L.; Steitz, Joan A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">5743</span>),
    <span class="NLM_cas:pages">220-4</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">A comparison of small nuclear ribonucleoproteins (snRNPs) from HeLa, Ehrlich ascites, and armyworm cells indicated that antigenic snRNPs were highly conserved from man to insects.  Liver cells which actively formed mRNA had both higher amts. and a slightly different population of snRNAs than the cryptic nuclei of biosynthetically inactive erythrocytes.  Antigenic snRNPs contg. snRNAs U1, U2, U3, U4, U5, and U6 from Ehrlich ascites cells sedimented in the 30 S region of the sucrose gradient, whereas RNA-labeled U1a* appeared only in the 10 S region where the free snRNP particles sedimented.  Fingerprint anal. showed that U1a* was identical to U1a except that it lacked the 5'-trimethylated cap moiety plus 6 addnl. nucleotides from the 5' end of the U1a sequence.  Several of the residues missing in U1a* fell within a region of U1a which was extensively complementary to sequences in splice junctions in hnRNPs (heterogeneous nuclear ribonucleoproteins).  Thus, snRNPs may be directly involved in the splicing of hnRNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIeFSm-3VWlbVg90H21EOLACvtfcHk0lg0HczjiMC7IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXhsF2jtr8%253D&md5=a05e9914f26d348216e5c04d4b9cc038</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2F283220a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F283220a0%26sid%3Dliteratum%253Aachs%26aulast%3DLerner%26aufirst%3DM.%2BR.%26aulast%3DBoyle%26aufirst%3DJ.%2BA.%26aulast%3DMount%26aufirst%3DS.%2BM.%26aulast%3DWolin%26aufirst%3DS.%2BL.%26aulast%3DSteitz%26aufirst%3DJ.%2BA.%26atitle%3DAre%2520snRNPs%2520involved%2520in%2520splicing%253F%26jtitle%3DNature%26date%3D1980%26volume%3D283%26spage%3D220%26epage%3D224%26doi%3D10.1038%2F283220a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ganser, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herschlag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hashimi, H. M.</span></span> <span> </span><span class="NLM_article-title">The roles of structural dynamics in the cellular functions of RNAs</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">474</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1038/s41580-019-0136-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fs41580-019-0136-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31182864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=474-489&author=L.+R.+Ganserauthor=M.+L.+Kellyauthor=D.+Herschlagauthor=H.+M.+Al-Hashimi&title=The+roles+of+structural+dynamics+in+the+cellular+functions+of+RNAs&doi=10.1038%2Fs41580-019-0136-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The roles of structural dynamics in the cellular functions of RNAs</span></div><div class="casAuthors">Ganser, Laura R.; Kelly, Megan L.; Herschlag, Daniel; Al-Hashimi, Hashim M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">474-489</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  RNAs fold into 3D structures that range from simple helical elements to complex tertiary structures and quaternary ribonucleoprotein assemblies.  The functions of many regulatory RNAs depend on how their 3D structure changes in response to a diverse array of cellular conditions.  In this Review, we examine how the structural characterization of RNA as dynamic ensembles of conformations, which form with different probabilities and at different timescales, is improving our understanding of RNA function in cells.  We discuss the mechanisms of gene regulation by microRNAs, riboswitches, ribozymes, post-transcriptional RNA modifications and RNA-binding proteins, and how the cellular environment and processes such as liq.-liq. phase sepn. may affect RNA folding and activity.  The emerging RNA-ensemble-function paradigm is changing our perspective and understanding of RNA regulation, from in vitro to in vivo and from descriptive to predictive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6pHr5mgMApLVg90H21EOLACvtfcHk0lg0HczjiMC7IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtb7K&md5=83a97268a18fb90f821ce6627b89af5a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fs41580-019-0136-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41580-019-0136-0%26sid%3Dliteratum%253Aachs%26aulast%3DGanser%26aufirst%3DL.%2BR.%26aulast%3DKelly%26aufirst%3DM.%2BL.%26aulast%3DHerschlag%26aufirst%3DD.%26aulast%3DAl-Hashimi%26aufirst%3DH.%2BM.%26atitle%3DThe%2520roles%2520of%2520structural%2520dynamics%2520in%2520the%2520cellular%2520functions%2520of%2520RNAs%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2019%26volume%3D20%26spage%3D474%26epage%3D489%26doi%3D10.1038%2Fs41580-019-0136-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stombaugh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zirbel, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westhof, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leontis, N. B.</span></span> <span> </span><span class="NLM_article-title">Frequency and isostericity of RNA base pairs</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">2294</span>– <span class="NLM_lpage">2312</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1093%2Fnar%2Fgkp011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19240142" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltFWktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=2294-2312&author=J.+Stombaughauthor=C.+L.+Zirbelauthor=E.+Westhofauthor=N.+B.+Leontis&title=Frequency+and+isostericity+of+RNA+base+pairs&doi=10.1093%2Fnar%2Fgkp011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Frequency and isostericity of RNA base pairs</span></div><div class="casAuthors">Stombaugh, Jesse; Zirbel, Craig L.; Westhof, Eric; Leontis, Neocles B.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2294-2312</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Most of the hairpin, internal and junction loops that appear single-stranded in std. RNA secondary structures form recurrent 3D motifs, where non-Watson-Crick base pairs play a central role.  Non-Watson-Crick base pairs also play crucial roles in tertiary contacts in structured RNA mols.  We previously classified RNA base pairs geometrically to group together those base pairs that are structurally similar (isosteric) and therefore able to substitute for each other by mutation without disrupting the 3D structure.  Here, we introduce a quant. measure of base pair isostericity, the IsoDiscrepancy Index (IDI), to more accurately det. which base pair substitutions can potentially occur in conserved motifs.  We ext. and classify base pairs from a reduced-redundancy set of RNA 3D structures from the Protein Data Bank (PDB) and calc. centroids (exemplars) for each base combination and geometric base pair type (family).  We use the exemplars and IDI values to update our online Basepair Catalog and the Isostericity Matrixes (IM) for each base pair family.  From the database of base pairs obsd. in 3D structures we derive base pair occurrence frequencies for each of the 12 geometric base pair families.  To improve the statistics from the 3D structures, we also derive base pair occurrence frequencies from rRNA sequence alignments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmiA1N4lBbPLVg90H21EOLACvtfcHk0liRfeyR_4ocoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltFWktrc%253D&md5=7dd0cc423532908b8f18dfd4bccba582</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp011%26sid%3Dliteratum%253Aachs%26aulast%3DStombaugh%26aufirst%3DJ.%26aulast%3DZirbel%26aufirst%3DC.%2BL.%26aulast%3DWesthof%26aufirst%3DE.%26aulast%3DLeontis%26aufirst%3DN.%2BB.%26atitle%3DFrequency%2520and%2520isostericity%2520of%2520RNA%2520base%2520pairs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26spage%3D2294%26epage%3D2312%26doi%3D10.1093%2Fnar%2Fgkp011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yesselman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisaria, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herschlag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenleaf, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, R.</span></span> <span> </span><span class="NLM_article-title">Sequence-dependent RNA helix conformational preferences predictably impact tertiary structure formation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">16847</span>– <span class="NLM_lpage">16855</span>, <span class="refDoi"> DOI: 10.1073/pnas.1901530116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1901530116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31375637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1alsL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=16847-16855&author=J.+D.+Yesselmanauthor=S.+K.+Dennyauthor=N.+Bisariaauthor=D.+Herschlagauthor=W.+J.+Greenleafauthor=R.+Das&title=Sequence-dependent+RNA+helix+conformational+preferences+predictably+impact+tertiary+structure+formation&doi=10.1073%2Fpnas.1901530116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence-dependent RNA helix conformational preferences predictably impact tertiary structure formation</span></div><div class="casAuthors">Yesselman, Joseph D.; Denny, Sarah K.; Bisaria, Namita; Herschlag, Daniel; Greenleaf, William J.; Das, Rhiju</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">16847-16855</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Structured RNAs and RNA complexes underlie biol. processes ranging from control of gene expression to protein translation.  Approx. 50% of nucleotides within known structured RNAs are folded into Watson-Crick (WC) base pairs, and sequence changes that preserve these pairs are typically assumed to preserve higher-order RNA structure and binding of macromol. partners.  Here, we report that indirect effects of the helix sequence on RNA tertiary stability are, in fact, significant but are nevertheless predictable from a simple computational model called RNAMake-δδG.  When tested through the RNA on a massively parallel array (RNA-MaP) exptl. platform, blind predictions for>1500 variants of the tectoRNA heterodimer model system achieve high accuracy (rmsd 0.34 and 0.77 kcal/mol for sequence and length changes, resp.).  Detailed comparison of predictions to expts. support a microscopic picture of how helix sequence changes subtly modulate conformational fluctuations at each base-pair step, which accumulate to impact RNA tertiary structure stability.  Our study reveals a previously overlooked phenomenon in RNA structure formation and provides a framework of computation and expt. for understanding helix conformational preferences and their impact across biol. RNA and RNA-protein assemblies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBas5Lz0S4fbVg90H21EOLACvtfcHk0liRfeyR_4ocoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1alsL%252FI&md5=2028e40755bcc5d65b78014b8ecb2464</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1901530116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1901530116%26sid%3Dliteratum%253Aachs%26aulast%3DYesselman%26aufirst%3DJ.%2BD.%26aulast%3DDenny%26aufirst%3DS.%2BK.%26aulast%3DBisaria%26aufirst%3DN.%26aulast%3DHerschlag%26aufirst%3DD.%26aulast%3DGreenleaf%26aufirst%3DW.%2BJ.%26aulast%3DDas%26aufirst%3DR.%26atitle%3DSequence-dependent%2520RNA%2520helix%2520conformational%2520preferences%2520predictably%2520impact%2520tertiary%2520structure%2520formation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D16847%26epage%3D16855%26doi%3D10.1073%2Fpnas.1901530116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Denny, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisaria, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yesselman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herschlag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenleaf, W. J.</span></span> <span> </span><span class="NLM_article-title">High-throughput investigation of diverse junction elements in RNA tertiary folding</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">390</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.05.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.cell.2018.05.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29961580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Cis7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2018&pages=377-390&author=S.+K.+Dennyauthor=N.+Bisariaauthor=J.+D.+Yesselmanauthor=R.+Dasauthor=D.+Herschlagauthor=W.+J.+Greenleaf&title=High-throughput+investigation+of+diverse+junction+elements+in+RNA+tertiary+folding&doi=10.1016%2Fj.cell.2018.05.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput investigation of diverse junction elements in RNA tertiary folding</span></div><div class="casAuthors">Denny, Sarah Knight; Bisaria, Namita; Yesselman, Joseph David; Das, Rhiju; Herschlag, Daniel; Greenleaf, William James</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-390.e20</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">RNAs fold into defined tertiary structures to function in crit. biol. processes.  While quant. models can predict RNA secondary structure stability, we are still unable to predict the thermodn. stability of RNA tertiary structure.  Here, we probed conformational preferences of diverse RNA two-way junctions to develop a predictive model for the formation of RNA tertiary structure.  We quant. measured tertiary assembly energetics of >1000 RNA junctions inserted in multiple structural scaffolds to generate a "thermodn. fingerprint" for each junction.  Thermodn. fingerprints enabled comparison of junction conformational preferences, revealing principles for how sequence influences 3-dimensional conformations.  Utilizing fingerprints of junctions with known crystal structures, we generated ensembles for related junctions that predicted their thermodn. effects on assembly formation.  This work revealed sequence-structure-energetic relations in RNA, demonstrated the capacity for diverse compensation strategies within tertiary structures, and provided a path to quant. modeling of RNA folding energetics based on "ensemble modularity.".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_6bGWqromibVg90H21EOLACvtfcHk0ljrLHHwwOqOFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Cis7vO&md5=30f8c56d8b28e7bacc0ccad937a12e60</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.05.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.05.038%26sid%3Dliteratum%253Aachs%26aulast%3DDenny%26aufirst%3DS.%2BK.%26aulast%3DBisaria%26aufirst%3DN.%26aulast%3DYesselman%26aufirst%3DJ.%2BD.%26aulast%3DDas%26aufirst%3DR.%26aulast%3DHerschlag%26aufirst%3DD.%26aulast%3DGreenleaf%26aufirst%3DW.%2BJ.%26atitle%3DHigh-throughput%2520investigation%2520of%2520diverse%2520junction%2520elements%2520in%2520RNA%2520tertiary%2520folding%26jtitle%3DCell%26date%3D2018%26volume%3D174%26spage%3D377%26epage%3D390%26doi%3D10.1016%2Fj.cell.2018.05.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assmann, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevilacqua, P. C.</span></span> <span> </span><span class="NLM_article-title">The RNA structurome: transcriptome-wide structure probing with next-generation sequencing</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">232</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2015.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.tibs.2015.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=25797096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvVGgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2015&pages=221-232&author=C.+K.+Kwokauthor=Y.+Tangauthor=S.+M.+Assmannauthor=P.+C.+Bevilacqua&title=The+RNA+structurome%3A+transcriptome-wide+structure+probing+with+next-generation+sequencing&doi=10.1016%2Fj.tibs.2015.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The RNA structurome: transcriptome-wide structure probing with next-generation sequencing</span></div><div class="casAuthors">Kwok, Chun Kit; Tang, Yin; Assmann, Sarah M.; Bevilacqua, Philip C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">221-232</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  RNA folds into intricate structures that enable its pivotal roles in biol., ranging from regulation of gene expression to ligand sensing and enzymic functions.  Therefore, elucidating RNA structure can provide profound insights into living systems.  A recent marriage between in vivo RNA structure probing and next-generation sequencing (NGS) has revolutionized the RNA field by enabling transcriptome-wide structure detn. in vivo, which has been applied to date to human cells, yeast cells, and Arabidopsis seedlings.  Anal. of resultant in vivo 'RNA structuromes' provides new and important information regarding myriad cellular processes, including control of translation, alternative splicing, alternative polyadenylation, energy-dependent unfolding of mRNA, and effects of proteins on RNA structure.  An emerging view suggests potential links between RNA structure and stress and disease physiol. across the tree of life.  As we discuss here, these exciting findings open new frontiers into RNA biol., genome biol., and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrI9z6BEOwvmrVg90H21EOLACvtfcHk0ljrLHHwwOqOFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvVGgsLc%253D&md5=288aa8be796db4c452b6bcfa27012424</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2015.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2015.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKwok%26aufirst%3DC.%2BK.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DAssmann%26aufirst%3DS.%2BM.%26aulast%3DBevilacqua%26aufirst%3DP.%2BC.%26atitle%3DThe%2520RNA%2520structurome%253A%2520transcriptome-wide%2520structure%2520probing%2520with%2520next-generation%2520sequencing%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2015%26volume%3D40%26spage%3D221%26epage%3D232%26doi%3D10.1016%2Fj.tibs.2015.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fallmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Will, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grüning, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backofen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, P. F.</span></span> <span> </span><span class="NLM_article-title">Recent advances in RNA folding</span>. <i>J. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>261</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">104</span>, <span class="refDoi"> DOI: 10.1016/j.jbiotec.2017.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.jbiotec.2017.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28690134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Sls73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=2017&pages=97-104&author=J.+Fallmannauthor=S.+Willauthor=J.+Engelhardtauthor=B.+Gr%C3%BCningauthor=R.+Backofenauthor=P.+F.+Stadler&title=Recent+advances+in+RNA+folding&doi=10.1016%2Fj.jbiotec.2017.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in RNA folding</span></div><div class="casAuthors">Fallmann, Joerg; Will, Sebastian; Engelhardt, Jan; Gruening, Bjoern; Backofen, Rolf; Stadler, Peter F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biotechnology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">97-104</span>CODEN:
                <span class="NLM_cas:coden">JBITD4</span>;
        ISSN:<span class="NLM_cas:issn">0168-1656</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In the realm of nucleic acid structures, secondary structure forms a conceptually important intermediate level of description and explains the dominating part of the free energy of structure formation.  Secondary structures are well conserved over evolutionary time-scales and for many classes of RNAs evolve slower than the underlying primary sequences.  Given the close link between structure and function, secondary structure is routinely used as a basis to explain exptl. findings.  Recent technol. advances, finally, have made it possible to assay secondary structure directly using high throughput methods.  From a computational biol. point of view, secondary structures have a special role because they can be computed efficiently using exact dynamic programming algorithms.  In this contribution we provide a short overview of RNA folding algorithms, recent addns. and variations and address methods to align, compare, and cluster RNA structures, followed by a tabular summary of the most important software suites in the fields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjwBWQrP9xYLVg90H21EOLACvtfcHk0ljrLHHwwOqOFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Sls73E&md5=ffd726b00ff26ee6fd6cc1e952b3a97b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.jbiotec.2017.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jbiotec.2017.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DFallmann%26aufirst%3DJ.%26aulast%3DWill%26aufirst%3DS.%26aulast%3DEngelhardt%26aufirst%3DJ.%26aulast%3DGr%25C3%25BCning%26aufirst%3DB.%26aulast%3DBackofen%26aufirst%3DR.%26aulast%3DStadler%26aufirst%3DP.%2BF.%26atitle%3DRecent%2520advances%2520in%2520RNA%2520folding%26jtitle%3DJ.%2520Biotechnol.%26date%3D2017%26volume%3D261%26spage%3D97%26epage%3D104%26doi%3D10.1016%2Fj.jbiotec.2017.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bevilacqua, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritchey, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assmann, S. M.</span></span> <span> </span><span class="NLM_article-title">Genome-wide analysis of RNA secondary structure</span>. <i>Annu. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1146/annurev-genet-120215-035034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1146%2Fannurev-genet-120215-035034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27648642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFemtrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2016&pages=235-266&author=P.+C.+Bevilacquaauthor=L.+E.+Ritcheyauthor=Z.+Suauthor=S.+M.+Assmann&title=Genome-wide+analysis+of+RNA+secondary+structure&doi=10.1146%2Fannurev-genet-120215-035034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-Wide Analysis of RNA Secondary Structure</span></div><div class="casAuthors">Bevilacqua, Philip C.; Ritchey, Laura E.; Su, Zhao; Assmann, Sarah M.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Genetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">235-266</span>CODEN:
                <span class="NLM_cas:coden">ARVGB7</span>;
        ISSN:<span class="NLM_cas:issn">0066-4197</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  Single-stranded RNA mols. fold into extraordinarily complicated secondary and tertiary structures as a result of intramol. base pairing.  In vivo, these RNA structures are not static.  Instead, they are remodeled in response to changes in the prevailing physicochem. environment of the cell and as a result of intermol. base pairing and interactions with RNA-binding proteins.  Remarkable tech. advances now allow us to probe RNA secondary structure at single-nucleotide resoln. and genome-wide, both in vitro and in vivo.  These data sets provide new glimpses into the RNA universe.  Analyses of RNA structuromes in HIV, yeast, Arabidopsis, and mammalian cells and tissues have revealed regulatory effects of RNA structure on mRNA (mRNA) polyadenylation, splicing, translation, and turnover.  Application of new methods for genome-wide identification of mRNA modifications, particularly methylation and pseudouridylation, has shown that the RNA "epitranscriptome" both influences and is influenced by RNA structure.  In this review, we describe newly developed genome-wide RNA structure-probing methods and synthesize the information emerging from their application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptXBfgVqAC-rVg90H21EOLACvtfcHk0ljp2zCnS5S2-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFemtrzM&md5=6213387ba5787285eeaabd1b624787a3</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1146%2Fannurev-genet-120215-035034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-genet-120215-035034%26sid%3Dliteratum%253Aachs%26aulast%3DBevilacqua%26aufirst%3DP.%2BC.%26aulast%3DRitchey%26aufirst%3DL.%2BE.%26aulast%3DSu%26aufirst%3DZ.%26aulast%3DAssmann%26aufirst%3DS.%2BM.%26atitle%3DGenome-wide%2520analysis%2520of%2520RNA%2520secondary%2520structure%26jtitle%3DAnnu.%2520Rev.%2520Genet.%26date%3D2016%26volume%3D50%26spage%3D235%26epage%3D266%26doi%3D10.1146%2Fannurev-genet-120215-035034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iida, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, T.-a.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takaiwa, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwabuchi, M.</span></span> <span> </span><span class="NLM_article-title">Phylogenetic relationships among eukaryotic kingdoms inferred from ribosomal RNA sequences</span>. <i>J. Mol. Evol.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1007/BF02105802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1007%2FBF02105802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=3932662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaL2MXltFygu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1985&pages=32-38&author=M.+Hasegawaauthor=Y.+Iidaauthor=T.-a.+Yanoauthor=F.+Takaiwaauthor=M.+Iwabuchi&title=Phylogenetic+relationships+among+eukaryotic+kingdoms+inferred+from+ribosomal+RNA+sequences&doi=10.1007%2FBF02105802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Phylogenetic relationships among eukaryotic kingdoms inferred from ribosomal RNA sequences</span></div><div class="casAuthors">Hasegawa, Masami; Iida, Yoichi; Yano, Takaaki; Takaiwa, Fumio; Iwabuchi, Masaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Evolution</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-8</span>CODEN:
                <span class="NLM_cas:coden">JMEVAU</span>;
        ISSN:<span class="NLM_cas:issn">0022-2844</span>.
    </div><div class="casAbstract">Phylogenetic trees among eukaryotic kingdoms were inferred for large- and small-subunit rRNAs by using a max.-likelihood method developed by J. Felsenstein (1981).  Although Felsenstein's method assumes equal evolutionary rates for transitions and transversions, this is apparently not the case for these data.  Therefore, only transversion-type substitutions were taken into account.  The mols. used were large-subunit rRNAs from Xenopus laevis (Animalia), rice (Plantae), Saccharomyces cerevisiae (Fungi), Dictyostelium discoideum (Protista), and Physarum polycephalum (Protista), and small-subunit rRNAs from maize (Plantae), S. cerevisiae, X. laevis, rat (Animalia), and D. discoideum.  Only conservative regions of the nucleotide sequences were considered for this study.  In the max.-likelihood trees for both large- and small-subunit rRNAs, Animalia and Fungi were the most closely related eukaryotic kingdoms, and Plantae was the next most closely related kingdom, although other branching orders among Plantae, Animalia, and Fungi were not excluded by this work.  These 3 eukaryotic kingdoms apparently shared a common ancestor after the divergence of the 2 species of Protista, D. discoideum and P. polycephalum.  These 2 species of Protista do not form a clade, and P. polycephalum diverged 1st and D. discoideum 2nd from the line leading to the common ancestor of Plantae, Animalia, and Fungi.  The sequence data indicate that a drastic change occurred in the nucleotide sequences of rRNAs during the evolutionary sepn. between prokaryote and eukaryote.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgdQQQ9ZpMZLVg90H21EOLACvtfcHk0ljp2zCnS5S2-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXltFygu7o%253D&md5=17434f208c671f54566aa086a643e533</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1007%2FBF02105802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02105802%26sid%3Dliteratum%253Aachs%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DIida%26aufirst%3DY.%26aulast%3DYano%26aufirst%3DT.-a.%26aulast%3DTakaiwa%26aufirst%3DF.%26aulast%3DIwabuchi%26aufirst%3DM.%26atitle%3DPhylogenetic%2520relationships%2520among%2520eukaryotic%2520kingdoms%2520inferred%2520from%2520ribosomal%2520RNA%2520sequences%26jtitle%3DJ.%2520Mol.%2520Evol.%26date%3D1985%26volume%3D22%26spage%3D32%26epage%3D38%26doi%3D10.1007%2FBF02105802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turner, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, D. H.</span></span> <span> </span><span class="NLM_article-title">NNDB: the nearest neighbor parameter database for predicting stability of nucleic acid secondary structure</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">D280</span>– <span class="NLM_lpage">D282</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1093%2Fnar%2Fgkp892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19880381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktlOqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=D280-D282&author=D.+H.+Turnerauthor=D.+H.+Mathews&title=NNDB%3A+the+nearest+neighbor+parameter+database+for+predicting+stability+of+nucleic+acid+secondary+structure&doi=10.1093%2Fnar%2Fgkp892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">NNDB: the nearest neighbor parameter database for predicting stability of nucleic acid secondary structure</span></div><div class="casAuthors">Turner, Douglas H.; Mathews, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D280-D282</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Nearest Neighbor Database (NNDB, http://rna.urmc.rochester.edu/NNDB) is a web-based resource for disseminating parameter sets for predicting nucleic acid secondary structure stabilities.  For each set of parameters, the database includes the set of rules with descriptive text, sequence-dependent parameters in plain text and html, literature refs. to expts. and usage tutorials.  The initial release covers parameters for predicting RNA folding free energy and enthalpy changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg3uUXrM-0_bVg90H21EOLACvtfcHk0lhha7e2G0ILtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktlOqtA%253D%253D&md5=7171efa8b463e554cc893f4cc241dd15</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp892%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DD.%2BH.%26aulast%3DMathews%26aufirst%3DD.%2BH.%26atitle%3DNNDB%253A%2520the%2520nearest%2520neighbor%2520parameter%2520database%2520for%2520predicting%2520stability%2520of%2520nucleic%2520acid%2520secondary%2520structure%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DD280%26epage%3DD282%26doi%3D10.1093%2Fnar%2Fgkp892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuker, M.</span></span> <span> </span><span class="NLM_article-title">On finding all suboptimal foldings of an RNA molecule</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>244</i></span>,  <span class="NLM_fpage">48</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1126/science.2468181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1126%2Fscience.2468181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=2468181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaL1MXkt1SnsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=244&publication_year=1989&pages=48-52&author=M.+Zuker&title=On+finding+all+suboptimal+foldings+of+an+RNA+molecule&doi=10.1126%2Fscience.2468181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">On finding all suboptimal foldings of an RNA molecule</span></div><div class="casAuthors">Zuker, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">244</span>
        (<span class="NLM_cas:issue">4900</span>),
    <span class="NLM_cas:pages">48-52</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">An algorithm and a computer program were prepd. for detg. RNA secondary structures within any prescribed increment of the computed global min. free energy.  The math. problem of detg. how well defined a min. energy folding is can now be solved.  All predicted base pairs that can participate in suboptimal structures may be displayed and analyzed graphically.  Representative suboptimal foldings are generated by selecting these base pairs one at a time and computing the best foldings that contain them.  A distance criterion that ensures that no 2 structures are too close is used to avoid multiple generation of similar structures.  Thermodn. parameters, including free energy increments for single-base stacking at the ends of helices and for terminal mismatched pairs in interior and hairpin loops, are incorporated into the underlying folding model of the above algorithm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRk6_lhTFKLbVg90H21EOLACvtfcHk0lhha7e2G0ILtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXkt1SnsbY%253D&md5=727f88c4e1f5bda392240e981db256a3</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1126%2Fscience.2468181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2468181%26sid%3Dliteratum%253Aachs%26aulast%3DZuker%26aufirst%3DM.%26atitle%3DOn%2520finding%2520all%2520suboptimal%2520foldings%2520of%2520an%2520RNA%2520molecule%26jtitle%3DScience%26date%3D1989%26volume%3D244%26spage%3D48%26epage%3D52%26doi%3D10.1126%2Fscience.2468181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, D. H.</span></span> <span> </span><span class="NLM_article-title">Incorporating chemical modification constraints into a dynamic programming algorithm for prediction of RNA secondary structure</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">7287</span>– <span class="NLM_lpage">7292</span>, <span class="refDoi"> DOI: 10.1073/pnas.0401799101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.0401799101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=15123812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFaks7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=7287-7292&author=D.+H.+Mathewsauthor=M.+D.+Disneyauthor=J.+L.+Childsauthor=S.+J.+Schroederauthor=M.+Zukerauthor=D.+H.+Turner&title=Incorporating+chemical+modification+constraints+into+a+dynamic+programming+algorithm+for+prediction+of+RNA+secondary+structure&doi=10.1073%2Fpnas.0401799101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Incorporating chemical modification constraints into a dynamic programming algorithm for prediction of RNA secondary structure</span></div><div class="casAuthors">Mathews, David H.; Disney, Matthew D.; Childs, Jessica L.; Schroeder, Susan J.; Zuker, Michael; Turner, Douglas H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7287-7292</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A dynamic programming algorithm for prediction of RNA secondary structure has been revised to accommodate folding constraints detd. by chem. modification and to include free energy increments for coaxial stacking of helixes when they are either adjacent or sepd. by a single mismatch.  Furthermore, free energy parameters are revised to account for recent exptl. results for terminal mismatches and hairpin, bulge, internal, and multibranch loops.  To demonstrate the applicability of this method, in vivo modification was performed on 5S rRNA in both Escherichia coli and Candida albicans with 1-cyclohexyl-3-(2-morpholinoethyl) carbodiimide metho-p-toluene sulfonate, di-Me sulfate, and kethoxal.  The percentage of known base pairs in the predicted structure increased from 26.3% to 86.8% for the E. coli sequence by using modification constraints.  For C. albicans, the accuracy remained 87.5% both with and without modification data.  On av., for these sequences and a set of 14 sequences with known secondary structure and chem. modification data taken from the literature, accuracy improves from 67% to 76%.  This enhancement primarily reflects improvement for three sequences that are predicted with <40% accuracy on the basis of energetics alone.  For these sequences, inclusion of chem. modification constraints improves the av. accuracy from 28% to 78%.  For the 11 sequences with <6% pseudoknotted base pairs, structures predicted with constraints from chem. modification contain on av. 84% of known canonical base pairs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqzYqTEJBW1bVg90H21EOLACvtfcHk0li--3wIv0ruww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFaks7s%253D&md5=cbe3a05189b963f2bee130567335a72d</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0401799101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0401799101%26sid%3Dliteratum%253Aachs%26aulast%3DMathews%26aufirst%3DD.%2BH.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DChilds%26aufirst%3DJ.%2BL.%26aulast%3DSchroeder%26aufirst%3DS.%2BJ.%26aulast%3DZuker%26aufirst%3DM.%26aulast%3DTurner%26aufirst%3DD.%2BH.%26atitle%3DIncorporating%2520chemical%2520modification%2520constraints%2520into%2520a%2520dynamic%2520programming%2520algorithm%2520for%2520prediction%2520of%2520RNA%2520secondary%2520structure%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D7287%26epage%3D7292%26doi%3D10.1073%2Fpnas.0401799101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouskin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zubradt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washietl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span> <span> </span><span class="NLM_article-title">Genome-wide probing of RNA structure reveals active unfolding of mRNA structures in vivo</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>505</i></span>,  <span class="NLM_fpage">701</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1038/nature12894</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnature12894" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=24336214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=505&publication_year=2014&pages=701-705&author=S.+Rouskinauthor=M.+Zubradtauthor=S.+Washietlauthor=M.+Kellisauthor=J.+S.+Weissman&title=Genome-wide+probing+of+RNA+structure+reveals+active+unfolding+of+mRNA+structures+in+vivo&doi=10.1038%2Fnature12894"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide probing of RNA structure reveals active unfolding of mRNA structures in vivo</span></div><div class="casAuthors">Rouskin, Silvi; Zubradt, Meghan; Washietl, Stefan; Kellis, Manolis; Weissman, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">505</span>
        (<span class="NLM_cas:issue">7485</span>),
    <span class="NLM_cas:pages">701-705</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA has a dual role as an informational mol. and a direct effector of biol. tasks.  The latter function is enabled by RNA's ability to adopt complex secondary and tertiary folds and thus has motivated extensive computational and exptl. efforts for detg. RNA structures.  Existing approaches for evaluating RNA structure have been largely limited to in vitro systems, yet the thermodn. forces which drive RNA folding in vitro may not be sufficient to predict stable RNA structures in vivo.  Indeed, the presence of RNA-binding proteins and ATP-dependent helicases can influence which structures are present inside cells.  Here we present an approach for globally monitoring RNA structure in native conditions in vivo with single-nucleotide precision.  This method is based on in vivo modification with di-Me sulfate (DMS), which reacts with unpaired adenine and cytosine residues, followed by deep sequencing to monitor modifications.  Our data from yeast and mammalian cells are in excellent agreement with known mRNA structures and with the high-resoln. crystal structure of the Saccharomyces cerevisiae ribosome.  Comparison between in vivo and in vitro data reveals that in rapidly dividing cells there are vastly fewer structured mRNA regions in vivo than in vitro.  Even thermostable RNA structures are often denatured in cells, highlighting the importance of cellular processes in regulating RNA structure.  Indeed, anal. of mRNA structure under ATP-depleted conditions in yeast shows that energy-dependent processes strongly contribute to the predominantly unfolded state of mRNAs inside cells.  Our studies broadly enable the functional anal. of physiol. RNA structures and reveal that, in contrast to the Anfinsen view of protein folding whereby the structure formed is the most thermodynamically favorable, thermodn. have an incomplete role in detg. mRNA structure in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquJ_RYbQmU7bVg90H21EOLACvtfcHk0li--3wIv0ruww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFentbc%253D&md5=f04acd7ce0fa8ed94e1cd428c769bbc6</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnature12894&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12894%26sid%3Dliteratum%253Aachs%26aulast%3DRouskin%26aufirst%3DS.%26aulast%3DZubradt%26aufirst%3DM.%26aulast%3DWashietl%26aufirst%3DS.%26aulast%3DKellis%26aufirst%3DM.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26atitle%3DGenome-wide%2520probing%2520of%2520RNA%2520structure%2520reveals%2520active%2520unfolding%2520of%2520mRNA%2520structures%2520in%2520vivo%26jtitle%3DNature%26date%3D2014%26volume%3D505%26spage%3D701%26epage%3D705%26doi%3D10.1038%2Fnature12894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zubradt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambowitz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouskin, S.</span></span> <span> </span><span class="NLM_article-title">DMS-MaPseq for genome-wide or targeted RNA structure probing in vivo</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1038/nmeth.4057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnmeth.4057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27819661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSiurfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=75-82&author=M.+Zubradtauthor=P.+Guptaauthor=S.+Persadauthor=A.+M.+Lambowitzauthor=J.+S.+Weissmanauthor=S.+Rouskin&title=DMS-MaPseq+for+genome-wide+or+targeted+RNA+structure+probing+in+vivo&doi=10.1038%2Fnmeth.4057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">DMS-MaPseq for genome-wide or targeted RNA structure probing in vivo</span></div><div class="casAuthors">Zubradt, Meghan; Gupta, Paromita; Persad, Sitara; Lambowitz, Alan M.; Weissman, Jonathan S.; Rouskin, Silvi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-82</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Coupling of structure-specific in vivo chem. modification to next-generation sequencing is transforming RNA secondary structure studies in living cells.  The dominant strategy for detecting in vivo chem. modifications uses reverse transcriptase truncation products, which introduce biases and necessitate population-av. assessments of RNA structure.  Here we present di-Me sulfate (DMS) mutational profiling with sequencing (DMS-MaPseq), which encodes DMS modifications as mismatches using a thermostable group II intron reverse transcriptase.  DMS-MaPseq yields a high signal-to-noise ratio, can report multiple structural features per mol., and allows both genome-wide studies and focused in vivo investigations of even low-abundance RNAs.  We apply DMS-MaPseq for the first anal. of RNA structure within an animal tissue and to identify a functional structure involved in noncanonical translation initiation.  Addnl., we use DMS-MaPseq to compare the in vivo structure of pre-mRNAs with their mature isoforms.  These applications illustrate DMS-MaPseq's capacity to dramatically expand in vivo anal. of RNA structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY8vORTX4NyrVg90H21EOLACvtfcHk0ljfkccfVCnybQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSiurfE&md5=bbc171334f1930859ea327c7d5744f76</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.4057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.4057%26sid%3Dliteratum%253Aachs%26aulast%3DZubradt%26aufirst%3DM.%26aulast%3DGupta%26aufirst%3DP.%26aulast%3DPersad%26aufirst%3DS.%26aulast%3DLambowitz%26aufirst%3DA.%2BM.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26aulast%3DRouskin%26aufirst%3DS.%26atitle%3DDMS-MaPseq%2520for%2520genome-wide%2520or%2520targeted%2520RNA%2520structure%2520probing%2520in%2520vivo%26jtitle%3DNat.%2520Methods%26date%3D2017%26volume%3D14%26spage%3D75%26epage%3D82%26doi%3D10.1038%2Fnmeth.4057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rivas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clements, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddy, S. R.</span></span> <span> </span><span class="NLM_article-title">A statistical test for conserved RNA structure shows lack of evidence for structure in lncRNAs</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1038/nmeth.4066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnmeth.4066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27819659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSiu77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=45-48&author=E.+Rivasauthor=J.+Clementsauthor=S.+R.+Eddy&title=A+statistical+test+for+conserved+RNA+structure+shows+lack+of+evidence+for+structure+in+lncRNAs&doi=10.1038%2Fnmeth.4066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">A statistical test for conserved RNA structure shows lack of evidence for structure in lncRNAs</span></div><div class="casAuthors">Rivas, Elena; Clements, Jody; Eddy, Sean R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-48</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Many functional RNAs have an evolutionarily conserved secondary structure.  Conservation of RNA base pairing induces pairwise covariations in sequence alignments.  We developed a computational method, R-scape (RNA Structural Covariation Above Phylogenetic Expectation), that quant. tests whether covariation anal. supports the presence of a conserved RNA secondary structure.  R-scape anal. finds no statistically significant support for proposed secondary structures of the long noncoding RNAs HOTAIR, SRA, and Xist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnFROMK2OjrVg90H21EOLACvtfcHk0ljfkccfVCnybQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSiu77J&md5=689c629a3940f4a64c8f43a6b2ba8155</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.4066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.4066%26sid%3Dliteratum%253Aachs%26aulast%3DRivas%26aufirst%3DE.%26aulast%3DClements%26aufirst%3DJ.%26aulast%3DEddy%26aufirst%3DS.%2BR.%26atitle%3DA%2520statistical%2520test%2520for%2520conserved%2520RNA%2520structure%2520shows%2520lack%2520of%2520evidence%2520for%2520structure%2520in%2520lncRNAs%26jtitle%3DNat.%2520Methods%26date%3D2017%26volume%3D14%26spage%3D45%26epage%3D48%26doi%3D10.1038%2Fnmeth.4066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span> <span> </span><span class="NLM_article-title">ScanFold: an approach for genome-wide discovery of local RNA structural elements-applications to Zika virus and HIV</span>. <i>PeerJ</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e6136</span> <span class="refDoi"> DOI: 10.7717/peerj.6136</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.7717%2Fpeerj.6136" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30627482" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&author=R.+J.+Andrewsauthor=J.+Rocheauthor=W.+N.+Moss&title=ScanFold%3A+an+approach+for+genome-wide+discovery+of+local+RNA+structural+elements-applications+to+Zika+virus+and+HIV&doi=10.7717%2Fpeerj.6136"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.7717%2Fpeerj.6136&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7717%252Fpeerj.6136%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DR.%2BJ.%26aulast%3DRoche%26aufirst%3DJ.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26atitle%3DScanFold%253A%2520an%2520approach%2520for%2520genome-wide%2520discovery%2520of%2520local%2520RNA%2520structural%2520elements-applications%2520to%2520Zika%2520virus%2520and%2520HIV%26jtitle%3DPeerJ%26date%3D2018%26volume%3D6%26doi%3D10.7717%2Fpeerj.6136" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, G. J.</span></span> <span> </span><span class="NLM_article-title">MicroRNAs: small RNAs with a big role in gene regulation</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">522</span>– <span class="NLM_lpage">531</span>, <span class="refDoi"> DOI: 10.1038/nrg1379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrg1379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=15211354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD2cXltVSmsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=522-531&author=L.+Heauthor=G.+J.+Hannon&title=MicroRNAs%3A+small+RNAs+with+a+big+role+in+gene+regulation&doi=10.1038%2Fnrg1379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNAs: small RNAs with a big role in gene regulation</span></div><div class="casAuthors">He, Lin; Hannon, Gregory J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">522-531</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  MicroRNAs are a family of small, non-coding RNAs that regulate gene expression in a sequence-specific manner.  The two founding members of the microRNA family were originally identified in Caenorhabditis elegans as genes that were required for the timed regulation of developmental events.  Since then, hundreds of microRNAs have been identified in almost all metazoan genomes, including worms, flies, plants and mammals.  MicroRNAs have diverse expression patterns and might regulate various developmental and physiol. processes.  Their discovery adds a new dimension to our understanding of complex gene regulatory networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp1kiZyCfuZ7Vg90H21EOLACvtfcHk0li1Xq8E1e9DdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXltVSmsbw%253D&md5=d4d40468a3dab2b1142be9f4d6735980</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnrg1379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg1379%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DL.%26aulast%3DHannon%26aufirst%3DG.%2BJ.%26atitle%3DMicroRNAs%253A%2520small%2520RNAs%2520with%2520a%2520big%2520role%2520in%2520gene%2520regulation%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2004%26volume%3D5%26spage%3D522%26epage%3D531%26doi%3D10.1038%2Fnrg1379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">RNA structures as mediators of neurological diseases and as drug targets</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2015.06.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.neuron.2015.06.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=26139368" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WksrzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2015&pages=28-46&author=V.+Bernatauthor=M.+D.+Disney&title=RNA+structures+as+mediators+of+neurological+diseases+and+as+drug+targets&doi=10.1016%2Fj.neuron.2015.06.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">RNA structures as mediators of neurological diseases and as drug targets</span></div><div class="casAuthors">Bernat, Viachaslau; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-46</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">RNAs adopt diverse folded structures that are essential for function and thus play crit. roles in cellular biol.  A striking example of this is the ribosome, a complex, three-dimensionally folded macromol. machine that orchestrates protein synthesis.  Advances in RNA biochem., structural and mol. biol., and bioinformatics have revealed other non-coding RNAs whose functions are dictated by their structure.  It is not surprising that aberrantly folded RNA structures contribute to disease.  In this Review, we provide a brief introduction into RNA structural biol. and then describe how RNA structures function in cells and cause or contribute to neurol. disease.  Finally, we highlight successful applications of rational design principles to provide chem. probes and lead compds. targeting structured RNAs.  Based on several examples of well-characterized RNA-driven neurol. disorders, we demonstrate how designed small mols. can facilitate the study of RNA dysfunction, elucidating previously unknown roles for RNA in disease, and provide lead therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIX63rZ0lwjbVg90H21EOLACvtfcHk0li1Xq8E1e9DdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WksrzJ&md5=5b93b94ae980c7ac0a0a8b99e76ebe40</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2015.06.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2015.06.012%26sid%3Dliteratum%253Aachs%26aulast%3DBernat%26aufirst%3DV.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DRNA%2520structures%2520as%2520mediators%2520of%2520neurological%2520diseases%2520and%2520as%2520drug%2520targets%26jtitle%3DNeuron%26date%3D2015%26volume%3D87%26spage%3D28%26epage%3D46%26doi%3D10.1016%2Fj.neuron.2015.06.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C. F.</span></span> <span> </span><span class="NLM_article-title">Therapeutic antisense oligonucleotides are coming of age</span>. <i>Annu. Rev. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">321</span>, <span class="refDoi"> DOI: 10.1146/annurev-med-041217-010829</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1146%2Fannurev-med-041217-010829" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30691367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvF2lsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2019&pages=307-321&author=C.+F.+Bennett&title=Therapeutic+antisense+oligonucleotides+are+coming+of+age&doi=10.1146%2Fannurev-med-041217-010829"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Antisense Oligonucleotides Are Coming of Age</span></div><div class="casAuthors">Bennett, C. Frank</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">307-321</span>CODEN:
                <span class="NLM_cas:coden">ARMCAH</span>;
        ISSN:<span class="NLM_cas:issn">0066-4219</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The first published description of therapeutic applications of antisense oligonucleotide (ASO) technol. occurred in the late 1970s and was followed by the founding of com. companies focused on developing antisense therapeutics in the late 1980s.  Since the late 1980s, there has been steady progress in improving the technol. platform, taking advantage of advances in oligonucleotide chem. and formulations as well as increased understanding of the distribution and safety of ASOs.  There are several approved ASO drugs and a broad pipeline in development.  In addn., advances in understanding human disease, including the genetic basis for most monogenic diseases and the availability of the full human genome sequence, have created numerous therapeutic applications for the technol.  I summarize the state of the technol. and highlight how advances in the technol. position ASOs to be an important contributor to future medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQKzgrxHUp1LVg90H21EOLACvtfcHk0liqdLwAOy7CcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvF2lsbs%253D&md5=6c7f4a07724b43b769c8014ca3108e6a</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1146%2Fannurev-med-041217-010829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-med-041217-010829%26sid%3Dliteratum%253Aachs%26aulast%3DBennett%26aufirst%3DC.%2BF.%26atitle%3DTherapeutic%2520antisense%2520oligonucleotides%2520are%2520coming%2520of%2520age%26jtitle%3DAnnu.%2520Rev.%2520Med.%26date%3D2019%26volume%3D70%26spage%3D307%26epage%3D321%26doi%3D10.1146%2Fannurev-med-041217-010829" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagedorn, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindow, M.</span></span> <span> </span><span class="NLM_article-title">Managing the sequence-specificity of antisense oligonucleotides in drug discovery</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2262</span>– <span class="NLM_lpage">2282</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1093%2Fnar%2Fgkx056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28426096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFygs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=2262-2282&author=P.+H.+Hagedornauthor=B.+R.+Hansenauthor=T.+Kochauthor=M.+Lindow&title=Managing+the+sequence-specificity+of+antisense+oligonucleotides+in+drug+discovery&doi=10.1093%2Fnar%2Fgkx056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Managing the sequence-specificity of antisense oligonucleotides in drug discovery</span></div><div class="casAuthors">Hagedorn, Peter H.; Hansen, Bo R.; Koch, Troels; Lindow, Morten</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2262-2282</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  All drugs perturb the expression of many genes in the cells that are exposed to them.  These gene expression changes can be divided into effects resulting from engaging the intended target and effects resulting from engaging unintended targets.  For antisense oligonucleotides, developments in bioinformatics algorithms, and the quality of sequence databases, allow oligonucleotide sequences to be analyzed computationally, in terms of the predictability of their interactions with intended and unintended RNA targets.  Applying these tools enables selection of sequence-specific oligonucleotides where no- or only few unintended RNA targets are expected.  To evaluate oligonucleotide sequence-specificity exptl., we recommend a transcriptomics protocol where two or more oligonucleotides targeting the same RNA mol., but with entirely different sequences, are evaluated together.  This helps to clarify which changes in cellular RNA levels result from downstream processes of engaging the intended target, and which are likely to be related to engaging unintended targets.  As required for all classes of drugs, the toxic potential of oligonucleotides must be evaluated in cell- and animal models before clin. testing.  Since potential adverse effects related to unintended targeting are sequence-dependent and therefore species-specific, in vitro toxicol. assays in human cells are esp. relevant in oligonucleotide drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq77hjabvSVxLVg90H21EOLACvtfcHk0liqdLwAOy7CcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFygs7g%253D&md5=ab9bc075d7a76d0d7a837c8af00899d7</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx056%26sid%3Dliteratum%253Aachs%26aulast%3DHagedorn%26aufirst%3DP.%2BH.%26aulast%3DHansen%26aufirst%3DB.%2BR.%26aulast%3DKoch%26aufirst%3DT.%26aulast%3DLindow%26aufirst%3DM.%26atitle%3DManaging%2520the%2520sequence-specificity%2520of%2520antisense%2520oligonucleotides%2520in%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3D2262%26epage%3D2282%26doi%3D10.1093%2Fnar%2Fgkx056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papoian, T.</span></span> <span> </span><span class="NLM_article-title">Safety of antisense oligonucleotide and siRNA-based therapeutics</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">823</span>– <span class="NLM_lpage">833</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.drudis.2017.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28159625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1eqs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=823-833&author=X.+Chiauthor=P.+Gattiauthor=T.+Papoian&title=Safety+of+antisense+oligonucleotide+and+siRNA-based+therapeutics&doi=10.1016%2Fj.drudis.2017.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Safety of antisense oligonucleotide and siRNA-based therapeutics</span></div><div class="casAuthors">Chi, Xuan; Gatti, Philip; Papoian, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">823-833</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Oligonucleotide-based therapy is an active area of drug development designed to treat a variety of gene-specific diseases.  Two of the more promising platforms are the antisense oligonucleotides (ASOs) and short interfering RNAs (siRNAs), both of which are often directed against similar targets.  In light of recent reports on clin. trials of severe thrombocytopenia with two different ASO drugs and increased peripheral neuropathy with an siRNA drug, we compared and contrasted the specific safety characteristics of these two classes of oligonucleotide therapeutic.  The objectives were to assess factors that could contribute to the specific toxicities obsd. with these two classes of promising drugs, and get a better understanding of the potential mechanism(s) responsible for these rare, but serious, adverse events.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Ej7iOK40PbVg90H21EOLACvtfcHk0lib7DB7G-NPcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1eqs70%253D&md5=40ef4591957b37ab501c95f53b6aa07b</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DChi%26aufirst%3DX.%26aulast%3DGatti%26aufirst%3DP.%26aulast%3DPapoian%26aufirst%3DT.%26atitle%3DSafety%2520of%2520antisense%2520oligonucleotide%2520and%2520siRNA-based%2520therapeutics%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26spage%3D823%26epage%3D833%26doi%3D10.1016%2Fj.drudis.2017.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frazier, K. S.</span></span> <span> </span><span class="NLM_article-title">Antisense oligonucleotide therapies: the promise and the challenges from a toxicologic pathologist’s perspective</span>. <i>Toxicol. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1177/0192623314551840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1177%2F0192623314551840" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=25385330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislWktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=78-89&author=K.+S.+Frazier&title=Antisense+oligonucleotide+therapies%3A+the+promise+and+the+challenges+from+a+toxicologic+pathologist%E2%80%99s+perspective&doi=10.1177%2F0192623314551840"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Antisense oligonucleotide therapies: the promise and the challenges from a toxicologic pathologist's perspective</span></div><div class="casAuthors">Frazier, Kendall S.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicologic Pathology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-89/1-78-89/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">TOPADD</span>;
        ISSN:<span class="NLM_cas:issn">0192-6233</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Many antisense oligonucleotides (ASOs) from several classes of mols. are currently in drug development.  Despite over 20 years of pharmaceutical research, few ASOs have been marketed due to problems with clin. efficacy or preclin. toxicol. challenges.  However, a no. of recent developments have renewed interest in this class including the registration of mipomersen, the advent of successful screening strategies to eliminate more toxic mols., and new understanding of the risks of off-target nucleotide binding and mitigation of potential off-target effects.  Recent advances in backbone chemistries, conjugation to other moieties, and new delivery systems have allowed better tissue penetration, enhanced intracellular targeting, and less frequent dosing, resulting in fewer toxicities.  While these new developments provide invigorated interest in these platforms, a few lingering challenges and preclin./clin. toxicity issues remain to be completely resolved, including: (1) proinflammatory effects (vasculitis/inflammatory infiltrates); (2) nephrotoxicity and hepatotoxicity unrelated to lysosomal accumulation; and (3) thrombocytopenia.  Recent investigative work by several labs. have helped elucidate mechanisms for these issues, allowing a better understanding of the clin. relevance and implications of particular toxicities.  It is important for toxicologists, pathologists, and regulatory reviewers to be familiar with new developments in the ASO field and their implications, as a greater no. of new types of antisense mols. undergo preclin. toxicity testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsxbnyuLQ_MbVg90H21EOLACvtfcHk0lib7DB7G-NPcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislWktbc%253D&md5=08881cd4d57355fcc3a53dd6bad5368f</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1177%2F0192623314551840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0192623314551840%26sid%3Dliteratum%253Aachs%26aulast%3DFrazier%26aufirst%3DK.%2BS.%26atitle%3DAntisense%2520oligonucleotide%2520therapies%253A%2520the%2520promise%2520and%2520the%2520challenges%2520from%2520a%2520toxicologic%2520pathologist%25E2%2580%2599s%2520perspective%26jtitle%3DToxicol.%2520Pathol.%26date%3D2015%26volume%3D43%26spage%3D78%26epage%3D89%26doi%3D10.1177%2F0192623314551840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimonet, P.</span></span> <span> </span><span class="NLM_article-title">New modalities, technologies, and partnerships in probe and lead generation: Enabling a mode-of-action centric paradigm</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9004</span>– <span class="NLM_lpage">9029</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00378</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00378" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVGjs77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9004-9029&author=E.+Valeurauthor=P.+Jimonet&title=New+modalities%2C+technologies%2C+and+partnerships+in+probe+and+lead+generation%3A+Enabling+a+mode-of-action+centric+paradigm&doi=10.1021%2Facs.jmedchem.8b00378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">New Modalities, Technologies, and Partnerships in Probe and Lead Generation: Enabling a Mode-of-Action Centric Paradigm</span></div><div class="casAuthors">Valeur, Eric; Jimonet, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9004-9029</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the rise of novel biol. and high potential target identification technologies originating from advances in genomics, medicinal chemists are progressively facing targets of increasing complexity and often unprecedented.  Novel hit finding technologies, combined with a wider choice of drug modalities, has resulted in a unique repertoire of options to address these challenging targets and to identify suitable starting points for optimization.  Furthermore, innovative solns. originating from a range of academic groups and biotech companies require new types of collaborative models to leverage and integrate them in the drug discovery process.  This perspective provides a guide for medicinal chemists covering contemporary probe and lead generation approaches and discusses the strengths and limitations of each strategy.  Moreover, the expansion of strategies to modulate proteins creates the opportunity of a modality-agnostic and mode-of-action centric hit finding paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS8l5d3NOahrVg90H21EOLACvtfcHk0lib7DB7G-NPcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVGjs77P&md5=62e768509dac90567f3f021ed83b1df1</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00378%26sid%3Dliteratum%253Aachs%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DJimonet%26aufirst%3DP.%26atitle%3DNew%2520modalities%252C%2520technologies%252C%2520and%2520partnerships%2520in%2520probe%2520and%2520lead%2520generation%253A%2520Enabling%2520a%2520mode-of-action%2520centric%2520paradigm%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9004%26epage%3D9029%26doi%3D10.1021%2Facs.jmedchem.8b00378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herschlag, D.</span></span> <span> </span><span class="NLM_article-title">Implications of ribozyme kinetics for targeting the cleavage of specific RNA molecules in vivo: more isn’t always better</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">6921</span>– <span class="NLM_lpage">6925</span>, <span class="refDoi"> DOI: 10.1073/pnas.88.16.6921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.88.16.6921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1871108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaK3MXlsFekt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=1991&pages=6921-6925&author=D.+Herschlag&title=Implications+of+ribozyme+kinetics+for+targeting+the+cleavage+of+specific+RNA+molecules+in+vivo%3A+more+isn%E2%80%99t+always+better&doi=10.1073%2Fpnas.88.16.6921"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Implications of ribozyme kinetics for targeting the cleavage of specific RNA molecules in vivo:  more is not always better</span></div><div class="casAuthors">Herschlag, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6921-5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Kinetic and thermodn. factors that det. specificity of RNA cleavage by ribozymes are illustrated with examples from recent work with a ribozyme derived from the group I intron of Tetrahymena thermophila pre-rRNA.  The conclusions also apply to other ribozymes, to antisense oligonucleotide expts., and to RNA and DNA cleavage agents that can recognize a single-stranded or double-stranded region of variable length.  At first, adding bases to a ribozyme's recognition sequence is expected to increase cleavage of the target RNA relative to cleavage of other RNAs.  However, adding more bases ultimately reduces this discrimination, as cleavage occurs essentially every time the target RNA or a mismatched RNA binds the ribozyme.  This occurs despite the weaker binding of the mismatched RNA because dissocn. becomes too slow (binding is too strong) to allow the ribozyme to choose between cleavage of the target RNA and a mismatched RNA.  In summary, more (base pairing) is not always better, because maximal discrimination requires equil. binding prior to cleavage.  The max. discrimination that can be obtained is expected to be greater with an A + U-rich recognition sequence than with a G + C-rich recognition sequence.  This is because the weaker A·U base pairs (relative to G·C base pairs) allow recognition to be spread over a larger no. of bases while preventing binding that is too strong.  Finally, creating an A-rich ribozyme rather than a U-rich ribozyme avoids the loss in discrimination expected with U-rich ribozymes from the formation of U·G wobble pairs in addn. to the targeted Watson-Crick U·A pair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEhIwC2M4ydbVg90H21EOLACvtfcHk0lj4gNzJVLq_JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlsFekt7s%253D&md5=df0058648a09c0094f03276cb3cd42de</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1073%2Fpnas.88.16.6921&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.88.16.6921%26sid%3Dliteratum%253Aachs%26aulast%3DHerschlag%26aufirst%3DD.%26atitle%3DImplications%2520of%2520ribozyme%2520kinetics%2520for%2520targeting%2520the%2520cleavage%2520of%2520specific%2520RNA%2520molecules%2520in%2520vivo%253A%2520more%2520isn%25E2%2580%2599t%2520always%2520better%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1991%26volume%3D88%26spage%3D6921%26epage%3D6925%26doi%3D10.1073%2Fpnas.88.16.6921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagedorn, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindholm, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindow, M.</span></span> <span> </span><span class="NLM_article-title">A kinetic model explains why shorter and less affine enzyme-recruiting oligonucleotides can be more potent</span>. <i>Mol. Ther.--Nucleic Acids</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e149</span> <span class="refDoi"> DOI: 10.1038/mtna.2013.72</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fmtna.2013.72" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=24549300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1agtr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&author=L.+Pedersenauthor=P.+H.+Hagedornauthor=M.+W.+Lindholmauthor=M.+Lindow&title=A+kinetic+model+explains+why+shorter+and+less+affine+enzyme-recruiting+oligonucleotides+can+be+more+potent&doi=10.1038%2Fmtna.2013.72"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">A Kinetic Model Explains Why Shorter and Less Affine Enzyme-recruiting Oligonucleotides Can Be More Potent</span></div><div class="casAuthors">Pedersen, Lykke; Hagedorn, Peter H.; Lindholm, Marie Wickstrom; Lindow, Morten</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy--Nucleic Acids</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">Feb.</span>),
    <span class="NLM_cas:pages">e149</span>CODEN:
                <span class="NLM_cas:coden">MTAOC5</span>;
        ISSN:<span class="NLM_cas:issn">2162-2531</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Antisense oligonucleotides complementary to RNA targets promise generality and ease of drug design.  The first systemically administered antisense drug was recently approved for treatment and others are in clin. development.  Chem. modifications that increase the hybridization affinity of oligonucleotides are reasoned to confer higher potency, [ital: null], modified oligonucleotides can be dosed at lower concns. to achieve the same effect.  Surprisingly, shorter and less affine oligonucleotides sometimes display increased potency.  To explain this apparent contradiction, increased uptake or decreased propensity to form structures have been suggested as possible mechanisms.  Here, we provide an alternative explanation that invokes only the kinetics behind oligonucleotide-mediated cleavage of RNA targets.  A model based on the law of mass action predicts, and expts. support, the existence of an optimal binding affinity.  Exaggerated affinity, and not length [ital: null], is detrimental to potency.  This finding clarifies how to optimally apply high-affinity modifications in the discovery of potent antisense oligonucleotide drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8YJl-hKnoIrVg90H21EOLACvtfcHk0lj4gNzJVLq_JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1agtr7O&md5=511e8a57529fb19beb8626416a277fa9</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1038%2Fmtna.2013.72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmtna.2013.72%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DL.%26aulast%3DHagedorn%26aufirst%3DP.%2BH.%26aulast%3DLindholm%26aufirst%3DM.%2BW.%26aulast%3DLindow%26aufirst%3DM.%26atitle%3DA%2520kinetic%2520model%2520explains%2520why%2520shorter%2520and%2520less%2520affine%2520enzyme-recruiting%2520oligonucleotides%2520can%2520be%2520more%2520potent%26jtitle%3DMol.%2520Ther.--Nucleic%2520Acids%26date%3D2014%26volume%3D3%26doi%3D10.1038%2Fmtna.2013.72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mckee, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olafson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Precise small molecule cleavage of a r(CUG) repeat expansion in a myotonic dystrophy mouse model</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">7799</span>– <span class="NLM_lpage">7804</span>, <span class="refDoi"> DOI: 10.1073/pnas.1901484116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1901484116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30926669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVeks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=7799-7804&author=A.+J.+Angelbelloauthor=S.+G.+Rzuczekauthor=K.+K.+Mckeeauthor=J.+L.+Chenauthor=H.+Olafsonauthor=M.+D.+Cameronauthor=W.+N.+Mossauthor=E.+T.+Wangauthor=M.+D.+Disney&title=Precise+small+molecule+cleavage+of+a+r%28CUG%29+repeat+expansion+in+a+myotonic+dystrophy+mouse+model&doi=10.1073%2Fpnas.1901484116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Precise small-molecule cleavage of an r(CUG) repeat expansion in a myotonic dystrophy mouse model</span></div><div class="casAuthors">Angelbello, Alicia J.; Rzuczek, Suzanne G.; Mckee, Kendra K.; Chen, Jonathan L.; Olafson, Hailey; Cameron, Michael D.; Moss, Walter N.; Wang, Eric T.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7799-7804</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 1 (DM1) is an incurable neuromuscular disorder caused by an expanded CTG repeat that is transcribed into r(CUG)exp.  The RNA repeat expansion sequesters regulatory proteins such as Muscleblind-like protein 1 (MBNL1), which causes pre-mRNA splicing defects.  The disease-causing r(CUG)exp has been targeted by antisense oligonucleotides, CRISPR-based approaches, and RNA-targeting small mols.  Herein, we describe a designer small mol., Cugamycin, that recognizes the structure of r(CUG)exp and cleaves it in both DM1 patient-derived myotubes and a DM1 mouse model, leaving short repeats of r(CUG) untouched.  In contrast, oligonucleotides that recognize r(CUG) sequence rather than structure cleave both long and short r(CUG)-contg. transcripts.  Transcriptomic, histol., and phenotypic studies demonstrate that Cugamycin broadly and specifically relieves DM1-assocd. defects in vivo without detectable off-targets.  Thus, small mols. that bind and cleave RNA have utility as lead chem. probes and medicines and can selectively target disease-causing RNA structures to broadly improve defects in preclin. animal models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg0da_Q8Pd37Vg90H21EOLACvtfcHk0ljVG7CInWtvng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVeks70%253D&md5=9dfa637039d1b532cbcbbaaccaed40b1</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1901484116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1901484116%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DMckee%26aufirst%3DK.%2BK.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DOlafson%26aufirst%3DH.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26aulast%3DWang%26aufirst%3DE.%2BT.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DPrecise%2520small%2520molecule%2520cleavage%2520of%2520a%2520r%2528CUG%2529%2520repeat%2520expansion%2520in%2520a%2520myotonic%2520dystrophy%2520mouse%2520model%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D7799%26epage%3D7804%26doi%3D10.1073%2Fpnas.1901484116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">A designed small molecule inhibitor of a non-coding RNA sensitizes HER2 negative cancer to Herceptin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">2960</span>– <span class="NLM_lpage">2974</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b10558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b10558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVGmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=2960-2974&author=M.+G.+Costalesauthor=D.+G.+Hochauthor=D.+Abeggauthor=J.+L.+Childs-Disneyauthor=S.+P.+Velagapudiauthor=A.+Adibekianauthor=M.+D.+Disney&title=A+designed+small+molecule+inhibitor+of+a+non-coding+RNA+sensitizes+HER2+negative+cancer+to+Herceptin&doi=10.1021%2Fjacs.8b10558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">A designed small molecule inhibitor of a non-coding RNA sensitizes HER2 negative cancers to herceptin</span></div><div class="casAuthors">Costales, Matthew G.; Hoch, Dominic G.; Abegg, Daniel; Childs-Disney, Jessica L.; Velagapudi, Sai Pradeep; Adibekian, Alexander; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2960-2974</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A small mol. (1) with overlapping affinity for two microRNA (miRNA) precursors was used to inform design of a dimeric compd. (2 (I)) selective for one of the miRNAs.  In particular, I selectively targets the microRNA(miR)-515 hairpin precursor to inhibit prodn. of miR-515 that represses sphingosine kinase 1 (SK1), a key enzyme in the biosynthesis of sphingosine 1-phosphate (S1P).  Application of I to breast cancer cells enhanced SK1 and S1P levels, triggering a migratory phenotype.  Knockout of SK1, forced overexpression of miR-515, and application of a small mol. SK1 inhibitor all ablated I's effect on phenotype, consistent with its designed mode of action.  Target profiling studies via Chem-CLIP showed that I bound selectively to the miR-515 hairpin precursor in cells.  Global neoprotein synthesis upon addn. of I to MCF-7 breast cancer cells demonstrated I's selectivity and upregulation of cancer-assocd. proteins regulated by S1P.  The most upregulated protein was human epidermal growth factor receptor 2 (ERBB2/HER2), which is regulated by the SK1/S1P pathway and is normally not expressed in MCF-7 cells.  Like triple neg. breast cancer (TNBC) cells, the lack of HER2 renders them insusceptible to Herceptin and its antibody-drug conjugate Kadcyla.  In addn. to proteomics, an RNA-seq study supports that I has limited off target effects and other studies support that I is more selective than an oligonucleotide.  We therefore hypothesized that I could sensitize MCF-7 cells to anti-HER2 therapies.  Indeed, application of I sensitized cells to Herceptin.  These results were confirmed in two other cell lines that express miR-515 and are HER2-, the hepatocellular carcinoma cell line HepG2 and the TNBC line MDA-MB-231.  Importantly, normal breast epithelial cells (MCF-10A) that do not express miR-515 are not affected by I.  These observations suggest a precision medicine approach to sensitize HER2- cancers to approved anticancer medicines.  This study has implications for broadening the therapeutic utility of known targeted cancer therapeutics by using a secondary targeted approach to render otherwise insensitive cells, sensitive to a targeted therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaWGyvVjeHDrVg90H21EOLACvtfcHk0ljVG7CInWtvng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVGmtrY%253D&md5=1e2f57da2a12fe57d6f60fa44e3e79e2</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b10558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b10558%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DA%2520designed%2520small%2520molecule%2520inhibitor%2520of%2520a%2520non-coding%2520RNA%2520sensitizes%2520HER2%2520negative%2520cancer%2520to%2520Herceptin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D2960%26epage%3D2974%26doi%3D10.1021%2Fjacs.8b10558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Comparison of small molecules and oligonucleotides that target a toxic, non-coding RNA</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2605</span>– <span class="NLM_lpage">2609</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.04.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.bmcl.2016.04.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27117425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVCmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=2605-2609&author=M.+G.+Costalesauthor=S.+G.+Rzuczekauthor=M.+D.+Disney&title=Comparison+of+small+molecules+and+oligonucleotides+that+target+a+toxic%2C+non-coding+RNA&doi=10.1016%2Fj.bmcl.2016.04.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of small molecules and oligonucleotides that target a toxic, non-coding RNA</span></div><div class="casAuthors">Costales, Matthew G.; Rzuczek, Suzanne G.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2605-2609</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Potential RNA targets for chem. probes and therapeutic modalities are pervasive in the transcriptome.  Oligonucleotide-based therapeutics are commonly used to target RNA sequence.  Small mols. are emerging as a modality to target RNA structures selectively, but their development is still in its infancy.  The authors compare the activity of oligonucleotides and several classes of small mols. that target the non-coding r(CCUG) repeat expansion (r(CCUG)exp) that causes myotonic dystrophy type 2 (DM2), an incurable disease that is the second-most common cause of adult onset muscular dystrophy.  Small mol. types investigated include monomers, dimers, and multivalent compds. synthesized on-site by using RNA-templated click chem.  Oligonucleotides investigated include phosphorothioates that cleave their target and vivo-morpholinos that modulate target RNA activity via binding.  The authors show that compds. assembled on-site that recognize structure have the highest potencies amongst small mols. and are similar in potency to a vivo-morpholino modified oligonucleotide that targets sequence.  These studies are likely to impact the design of therapeutic modalities targeting other repeats expansions that cause fragile X syndrome and amyotrophic lateral sclerosis, for example.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3hWVZ6e6s27Vg90H21EOLACvtfcHk0ljVG7CInWtvng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVCmtrY%253D&md5=540a6ffdd24b0205cfe7fdf702832174</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.04.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.04.025%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DComparison%2520of%2520small%2520molecules%2520and%2520oligonucleotides%2520that%2520target%2520a%2520toxic%252C%2520non-coding%2520RNA%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D2605%26epage%3D2609%26doi%3D10.1016%2Fj.bmcl.2016.04.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span> <span> </span><span class="NLM_article-title">Identifying and validating small molecules interacting with RNA (SMIRNAs)</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>623</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/bs.mie.2019.04.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fbs.mie.2019.04.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31239057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVSkt7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=623&publication_year=2019&pages=45-66&author=M.+D.+Disneyauthor=S.+P.+Velagapudiauthor=Y.+Liauthor=M.+G.+Costalesauthor=J.+L.+Childs-Disney&title=Identifying+and+validating+small+molecules+interacting+with+RNA+%28SMIRNAs%29&doi=10.1016%2Fbs.mie.2019.04.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and validating small molecules interacting with RNA (SMIRNAs)</span></div><div class="casAuthors">Disney, Matthew D.; Velagapudi, Sai Pradeep; Li, Yue; Costales, Matthew G.; Childs-Disney, Jessica L.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">623</span>
        (<span class="NLM_cas:issue">RNA Recognition</span>),
    <span class="NLM_cas:pages">45-66</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">High throughput sequencing has revolutionized our ability to identify aberrant RNA expression and mutations that cause or contribute to disease.  These data can be used directly to design oligonucleotide-based modalities using Watson-Crick pairing to target unstructured regions in an RNA.  A complementary, although more difficult, strategy to deactivate a malfunctioning RNA is to target highly structured regions with small mols.  Indeed, RNA structures are directly causative of disease.  Herein, we discuss emerging strategies to design high affinity, selective, bioactive ligands targeting RNA, or small mols. interacting with RNA (SMIRNAs), and target validation and profiling methods.  An exptl. foundation is required for a lead identification strateqy for RNA structures, constructed from a library-vs.-library screen that probes vast libraries of small mols. for binding RNA three dimensional folds.  Dubbed 2-dimensional combinatorial screening (2DCS), the resulting data can be mined against transcriptomes or the composite of RNAs :hat are produced in an organism to define folded RNA structures that can be targeted.  By applying SMIRNAs to cells and using target validation tools such as Chem. Crosslinking and Isolation by Pull-down (Chem-CLIP) and Small Mol. Nucleic Acid Profiling by Cleavage Applied to RNA (RiboSNAP), all targets engaged in cells can be defined, along with rules for mol. recognition to affect RNA biol.  This chapter will describe lessons learned in applying these approaches in vitro, in cells, and in pre-dinical animal models of disease, enabling SMIRNAs to capture opportunities in chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDS1IOjQJ36LVg90H21EOLACvtfcHk0ljeqY5yjrNTzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVSkt7vJ&md5=e43c245d539353958b0595e20876d6e7</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fbs.mie.2019.04.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.mie.2019.04.027%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26atitle%3DIdentifying%2520and%2520validating%2520small%2520molecules%2520interacting%2520with%2520RNA%2520%2528SMIRNAs%2529%26jtitle%3DMethods%2520Enzymol.%26date%3D2019%26volume%3D623%26spage%3D45%26epage%3D66%26doi%3D10.1016%2Fbs.mie.2019.04.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vummidi, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Approved anti-cancer drugs target oncogenic non-coding RNAs</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1086</span>– <span class="NLM_lpage">1094</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.05.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.chembiol.2018.05.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30251629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Cis7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=1086-1094&author=S.+P.+Velagapudiauthor=M.+G.+Costalesauthor=B.+R.+Vummidiauthor=Y.+Nakaiauthor=A.+J.+Angelbelloauthor=T.+Tranauthor=H.+S.+Haniffauthor=Y.+Matsumotoauthor=Z.+F.+Wangauthor=A.+K.+Chatterjeeauthor=J.+L.+Childs-Disneyauthor=M.+D.+Disney&title=Approved+anti-cancer+drugs+target+oncogenic+non-coding+RNAs&doi=10.1016%2Fj.chembiol.2018.05.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Approved Anti-cancer Drugs Target Oncogenic Non-coding RNAs</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Costales, Matthew G.; Vummidi, Balayeshwanth R.; Nakai, Yoshio; Angelbello, Alicia J.; Tran, Tuan; Haniff, Hafeez S.; Matsumoto, Yasumasa; Wang, Zi Fu; Chatterjee, Arnab K.; Childs-Disney, Jessica L.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1086-1094.e7</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Potential RNA drug targets for small mols. are found throughout the human transcriptome, yet small mols. known to elicit a pharmacol. response by directly targeting RNA are limited to antibacterials.  Herein, we describe AbsorbArray, a small mol. microarray-based approach that allows for unmodified compds., including FDA-approved drugs, to be probed for binding to RNA motif libraries in a massively parallel format.  Several drug classes bind RNA including kinase and topoisomerase inhibitors.  The latter avidly bound the motif found in the Dicer site of oncogenic microRNA (miR)-21 and inhibited its processing both in vitro and in cells.  The most potent compd. de-repressed a downstream protein target and inhibited a miR-21-mediated invasive phenotype.  The compd.'s activity was ablated upon overexpression of pre-miR-21.  Target validation via chem. crosslinking and isolation by pull-down showed direct engagement of pre-miR-21 by the small mol. in cells, demonstrating that RNAs should indeed be considered druggable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopr-LYm-CTeLVg90H21EOLACvtfcHk0ljeqY5yjrNTzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Cis7bM&md5=ea6d403eaef61fff07d168a750caa9e7</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.05.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.05.015%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DVummidi%26aufirst%3DB.%2BR.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DMatsumoto%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.%2BF.%26aulast%3DChatterjee%26aufirst%3DA.%2BK.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DApproved%2520anti-cancer%2520drugs%2520target%2520oncogenic%2520non-coding%2520RNAs%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D1086%26epage%3D1094%26doi%3D10.1016%2Fj.chembiol.2018.05.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallahi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Defining RNA-small molecule affinity landscapes enables design of a small molecule inhibitor of an oncogenic noncoding RNA</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">216</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.7b00009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.7b00009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1GqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=205-216&author=S.+P.+Velagapudiauthor=Y.+Luoauthor=T.+Tranauthor=H.+S.+Haniffauthor=Y.+Nakaiauthor=M.+Fallahiauthor=G.+J.+Martinezauthor=J.+L.+Childs-Disneyauthor=M.+D.+Disney&title=Defining+RNA-small+molecule+affinity+landscapes+enables+design+of+a+small+molecule+inhibitor+of+an+oncogenic+noncoding+RNA&doi=10.1021%2Facscentsci.7b00009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Defining RNA-Small Molecule Affinity Landscapes Enables Design of a Small Molecule Inhibitor of an Oncogenic Noncoding RNA</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Luo, Yiling; Tran, Tuan; Haniff, Hafeez S.; Nakai, Yoshio; Fallahi, Mohammad; Martinez, Gustavo J.; Childs-Disney, Jessica L.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">205-216</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RNA drug targets are pervasive in cells but methods to design small mols. that target them are sparse.  Herein, the authors report a general approach to score the affinity and selectivity of RNA motif-small mol. interactions identified via selection.  Named High Throughput Structure-Activity Relationships Through Sequencing (HiT-StARTS), HiT-StARTS is statistical in nature and compares input nucleic acid sequences to selected library members that bind a ligand via high throughput sequencing.  The approach allowed facile definition of the fitness landscape of hundreds of thousands of RNA motif-small mol. binding partners.  These results were mined against folded RNAs in the human transcriptome and identified an avid interaction between a small mol. and the Dicer nuclease-processing site in the oncogenic microRNA (miR)-18a hairpin precursor, which is a member of the miR-17-92 cluster.  Application of the small mol., Targapremir-18a, to prostate cancer cells inhibited prodn. of miR-18a from the cluster, derepressed serine/threonine protein kinase 4 protein (STK4), and triggered apoptosis.  Profiling the cellular targets of Targapremir-18a via Chem. Cross Linking and Isolation by Pull Down (Chem-CLIP), a covalent small mol.-RNA cellular profiling approach, and other studies showed specific binding of the compd. to the miR-18a precursor, revealing broadly applicable factors that govern small mol. drugging of noncoding RNAs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPmptGpjN_4rVg90H21EOLACvtfcHk0ljeqY5yjrNTzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1GqsLs%253D&md5=fefd2fe6e9d637c4aa04fd1e25fb1446</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.7b00009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.7b00009%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DMartinez%26aufirst%3DG.%2BJ.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDefining%2520RNA-small%2520molecule%2520affinity%2520landscapes%2520enables%2520design%2520of%2520a%2520small%2520molecule%2520inhibitor%2520of%2520an%2520oncogenic%2520noncoding%2520RNA%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2017%26volume%3D3%26spage%3D205%26epage%3D216%26doi%3D10.1021%2Facscentsci.7b00009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallo, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Sequence-based design of bioactive small molecules that target precursor microRNAs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnchembio.1452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=24509821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFaksbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=291-297&author=S.+P.+Velagapudiauthor=S.+M.+Galloauthor=M.+D.+Disney&title=Sequence-based+design+of+bioactive+small+molecules+that+target+precursor+microRNAs&doi=10.1038%2Fnchembio.1452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence-based design of bioactive small molecules that target precursor microRNAs</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Gallo, Steven M.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">291-297</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Oligonucleotides are designed to target RNA using base pairing rules, but they can be hampered by poor cellular delivery and nonspecific stimulation of the immune system.  Small mols. are preferred as lead drugs or probes but cannot be designed from sequence.  Herein, we describe an approach termed Inforna that designs lead small mols. for RNA from solely sequence.  Inforna was applied to all human microRNA hairpin precursors, and it identified bioactive small mols. that inhibit biogenesis by binding nuclease-processing sites (44% hit rate).  Among 27 lead interactions, the most avid interaction is between a benzimidazole (1) and precursor microRNA-96.  Compd. 1 selectively inhibits biogenesis of microRNA-96, upregulating a protein target (FOXO1) and inducing apoptosis in cancer cells.  Apoptosis is ablated when FOXO1 mRNA expression is knocked down by an siRNA, validating compd. selectivity.  Markedly, microRNA profiling shows that 1 only affects microRNA-96 biogenesis and is at least as selective as an oligonucleotide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy1YQeIYltbrVg90H21EOLACvtfcHk0li4XrMgEOix0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFaksbo%253D&md5=d57f6a2ca7d6784017e2adff9ddb2f55</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1452%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DGallo%26aufirst%3DS.%2BM.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSequence-based%2520design%2520of%2520bioactive%2520small%2520molecules%2520that%2520target%2520precursor%2520microRNAs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D291%26epage%3D297%26doi%3D10.1038%2Fnchembio.1452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkelsas, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallahi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span> <span> </span><span class="NLM_article-title">Inforna 2.0: A platform for the sequence-based design of small molecules targeting structured RNAs</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1720</span>– <span class="NLM_lpage">1728</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmt1Cgsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1720-1728&author=M.+D.+Disneyauthor=A.+M.+Winkelsasauthor=S.+P.+Velagapudiauthor=M.+Southernauthor=M.+Fallahiauthor=J.+L.+Childs-Disney&title=Inforna+2.0%3A+A+platform+for+the+sequence-based+design+of+small+molecules+targeting+structured+RNAs&doi=10.1021%2Facschembio.6b00001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Inforna 2.0: A Platform for the Sequence-Based Design of Small Molecules Targeting Structured RNAs</span></div><div class="casAuthors">Disney, Matthew D.; Winkelsas, Audrey M.; Velagapudi, Sai Pradeep; Southern, Mark; Fallahi, Mohammad; Childs-Disney, Jessica L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1720-1728</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of small mols. that target RNA is challenging yet, if successful, could advance the development of chem. probes to study RNA function or precision therapeutics to treat RNA-mediated disease.  Previously, we described Inforna, an approach that can mine motifs (secondary structures) within target RNAs, which is deduced from the RNA sequence, and compare them to a database of known RNA motif-small mol. binding partners.  Output generated by Inforna includes the motif found in both the database and the desired RNA target, lead small mols. for that target, and other related meta-data.  Lead small mols. can then be tested for binding and affecting cellular (dys)function.  Herein, we describe Inforna 2.0, which incorporates all known RNA motif-small mol. binding partners reported in the scientific literature, a chem. similarity searching feature, and an improved user interface and is freely available via an online web server.  By incorporation of interactions identified by other labs., the database has been doubled, contg. 1936 RNA motif-small mol. interactions, including 244 unique small mols. and 1331 motifs.  Interestingly, chemotype anal. of the compds. that bind RNA in the database reveals features in small mol. chemotypes that are privileged for binding.  Further, this updated database expanded the no. of cellular RNAs to which lead compds. can be identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHmJfcxIk9yrVg90H21EOLACvtfcHk0li4XrMgEOix0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmt1Cgsrw%253D&md5=6086bc48066565e519f7ee9e9cf2542a</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00001%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DWinkelsas%26aufirst%3DA.%2BM.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DSouthern%26aufirst%3DM.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26atitle%3DInforna%25202.0%253A%2520A%2520platform%2520for%2520the%2520sequence-based%2520design%2520of%2520small%2520molecules%2520targeting%2520structured%2520RNAs%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1720%26epage%3D1728%26doi%3D10.1021%2Facschembio.6b00001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phinney, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of microRNA-210 reprograms an oncogenic hypoxic circuit</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">3446</span>– <span class="NLM_lpage">3455</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b11273</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b11273" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlCjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=3446-3455&author=M.+G.+Costalesauthor=C.+L.+Hagaauthor=S.+P.+Velagapudiauthor=J.+L.+Childs-Disneyauthor=D.+G.+Phinneyauthor=M.+D.+Disney&title=Small+molecule+inhibition+of+microRNA-210+reprograms+an+oncogenic+hypoxic+circuit&doi=10.1021%2Fjacs.6b11273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibition of microRNA-210 Reprograms an Oncogenic Hypoxic Circuit</span></div><div class="casAuthors">Costales, Matthew G.; Haga, Christopher L.; Velagapudi, Sai Pradeep; Childs-Disney, Jessica L.; Phinney, Donald G.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3446-3455</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A hypoxic state is crit. to the metastatic and invasive characteristics of cancer.  Numerous pathways play crit. roles in cancer maintenance, many of which include noncoding RNAs such as microRNA (miR)-210 that regulates hypoxia inducible factors (HIFs).  Herein, we describe the identification of a small mol. named Targapremir-210 that binds to the Dicer site of the miR-210 hairpin precursor.  This interaction inhibits prodn. of the mature miRNA, derepresses glycerol-3-phosphate dehydrogenase 1-like enzyme (GPD1L), a hypoxia-assocd. protein neg. regulated by miR-210, decreases HIF-1α, and triggers apoptosis of triple neg. breast cancer cells only under hypoxic conditions.  Further, Targapremir-210 inhibits tumorigenesis in a mouse xenograft model of hypoxic triple neg. breast cancer.  Many factors govern mol. recognition of biol. targets by small mols.  For protein, chemoproteomics and activity-based protein profiling are invaluable tools to study small mol. target engagement and selectivity in cells.  Such approaches are lacking for RNA, leaving a void in the understanding of its druggability.  We applied Chem. Crosslinking and Isolation by Pull Down (Chem-CLIP) to study the cellular selectivity and the on- and off-targets of Targapremir-210.  Targapremir-210 selectively recognizes the miR-210 precursor and can differentially recognize RNAs in cells that have the same target motif but have different expression levels, revealing this important feature for selectively drugging RNAs for the first time.  These studies show that small mols. can be rapidly designed to selectively target RNAs and affect cellular responses to environmental conditions, resulting in favorable benefits against cancer.  Further, they help define rules for identifying druggable targets in the transcriptome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAY_YQQyzKvLVg90H21EOLACvtfcHk0li4XrMgEOix0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlCjt7w%253D&md5=c6725837e6f1713facb507b518998a8f</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b11273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b11273%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DHaga%26aufirst%3DC.%2BL.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DPhinney%26aufirst%3DD.%2BG.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520microRNA-210%2520reprograms%2520an%2520oncogenic%2520hypoxic%2520circuit%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D3446%26epage%3D3455%26doi%3D10.1021%2Fjacs.6b11273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennewith, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welford, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Story, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, Q.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giaccia, A. J.</span></span> <span> </span><span class="NLM_article-title">Hypoxia-inducible mir-210 regulates normoxic gene expression involved in tumor initiation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">856</span>– <span class="NLM_lpage">867</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2009.09.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.molcel.2009.09.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19782034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVChsrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2009&pages=856-867&author=X.+Huangauthor=L.+Dingauthor=K.+L.+Bennewithauthor=R.+T.+Tongauthor=S.+M.+Welfordauthor=K.+K.+Angauthor=M.+Storyauthor=Q.-T.+Leauthor=A.+J.+Giaccia&title=Hypoxia-inducible+mir-210+regulates+normoxic+gene+expression+involved+in+tumor+initiation&doi=10.1016%2Fj.molcel.2009.09.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Hypoxia-inducible mir-210 regulates normoxic gene expression involved in tumor initiation</span></div><div class="casAuthors">Huang, Xin; Ding, Lianghao; Bennewith, Kevin L.; Tong, Ricky T.; Welford, Scott M.; Ang, K. Kian; Story, Michael; Le, Quynh-Thu; Giaccia, Amato J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">856-867</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Previous studies have suggested that the HIF transcription factors can both activate and inhibit gene expression.  Here we show that HIF1 regulates the expression of mir-210 in a variety of tumor types through a hypoxia-responsive element.  Expression anal. in primary head and neck tumor samples indicates that mir-210 may serve as an in vivo marker for tumor hypoxia.  By Argonaute protein immunopptn., we identified 50 potential mir-210 targets and validated randomly selected ones.  The majority of these 50 genes are not classical hypoxia-inducible genes, suggesting mir-210 represses genes expressed under normoxia that are no longer necessary to adapt and survive in a hypoxic environment.  When human head and neck or pancreatic tumor cells ectopically expressing mir-210 were implanted into immunodeficient mice, mir-210 repressed initiation of tumor growth.  Taken together, these data implicate an important role for mir-210 in regulating the hypoxic response of tumor cells and tumor growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-QNBn2_n7KbVg90H21EOLACvtfcHk0liC0XzexSviKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVChsrzI&md5=98acfaab2f9f3a97e16c91801197209c</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2009.09.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2009.09.006%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DL.%26aulast%3DBennewith%26aufirst%3DK.%2BL.%26aulast%3DTong%26aufirst%3DR.%2BT.%26aulast%3DWelford%26aufirst%3DS.%2BM.%26aulast%3DAng%26aufirst%3DK.%2BK.%26aulast%3DStory%26aufirst%3DM.%26aulast%3DLe%26aufirst%3DQ.-T.%26aulast%3DGiaccia%26aufirst%3DA.%2BJ.%26atitle%3DHypoxia-inducible%2520mir-210%2520regulates%2520normoxic%2520gene%2520expression%2520involved%2520in%2520tumor%2520initiation%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D35%26spage%3D856%26epage%3D867%26doi%3D10.1016%2Fj.molcel.2009.09.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baosheng, L.</span></span> <span> </span><span class="NLM_article-title">Multiple functions of hypoxia-regulated miR-210 in cancer</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">50</span>, <span class="refDoi"> DOI: 10.1186/1756-9966-33-50</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1186%2F1756-9966-33-50" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=24909053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFShu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2014&pages=50&author=Q.+Qinauthor=W.+Furongauthor=L.+Baosheng&title=Multiple+functions+of+hypoxia-regulated+miR-210+in+cancer&doi=10.1186%2F1756-9966-33-50"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple functions of hypoxia-regulated miR-210 in cancer</span></div><div class="casAuthors">Qin, Qin; Wei, Furong; Li, Baosheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental & Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">50/1-50/10</span>CODEN:
                <span class="NLM_cas:coden">JECRDN</span>;
        ISSN:<span class="NLM_cas:issn">1756-9966</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  MicroRNAs (miRNAs) are small noncoding RNAs that regulate gene expression post-transcriptionally.  MiRNAs can be induced by a variety of stresses such as hypoxia, and are involved in diverse biol. processes including differentiation, cell proliferation, cell death, and tumorigenesis.  Hypoxia, a common feature of tumor microenvironment, can induce a no. of miRNAs expression. miRNA-210 (miR-210) is one of the hypoxia-regulated-miRNAs, which has been investigated extensively in cancer.  However, paradoxically opposing results were documented regarding whether it is an oncogene or a tumor suppressor, and whether it is a pos. or neg. prognostic biomarker.  In the present review, we focus on the following investigations of miR-210: (1) its functions of as an oncogene, (2) its functions as a tumor suppressor, (3) its functions in mitochondrial metab., and finally, the diagnostic and prognostic value of miR-210 in cancer researches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGU4V5lXKPCbVg90H21EOLACvtfcHk0liC0XzexSviKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFShu78%253D&md5=07067621ddb3082065991fad0ee7f157</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1186%2F1756-9966-33-50&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-9966-33-50%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DQ.%26aulast%3DFurong%26aufirst%3DW.%26aulast%3DBaosheng%26aufirst%3DL.%26atitle%3DMultiple%2520functions%2520of%2520hypoxia-regulated%2520miR-210%2520in%2520cancer%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2014%26volume%3D33%26spage%3D50%26doi%3D10.1186%2F1756-9966-33-50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puigserver, P.</span></span> <span> </span><span class="NLM_article-title">A hypoxia-induced positive feedback loop promotes hypoxia-inducible factor 1alpha stability through miR-210 suppression of glycerol-3-phosphate dehydrogenase 1-like</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">2696</span>– <span class="NLM_lpage">2706</span>, <span class="refDoi"> DOI: 10.1128/MCB.01242-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1128%2FMCB.01242-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=21555452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFKjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2011&pages=2696-2706&author=T.+J.+Kellyauthor=A.+L.+Souzaauthor=C.+B.+Clishauthor=P.+Puigserver&title=A+hypoxia-induced+positive+feedback+loop+promotes+hypoxia-inducible+factor+1alpha+stability+through+miR-210+suppression+of+glycerol-3-phosphate+dehydrogenase+1-like&doi=10.1128%2FMCB.01242-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">A hypoxia-induced positive feedback loop promotes hypoxia-inducible factor 1α stability through miR-210 suppression of glycerol-3-phosphate dehydrogenase 1-like</span></div><div class="casAuthors">Kelly, Timothy J.; Souza, Amanda L.; Clish, Clary B.; Puigserver, Pere</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2696-2706</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Oxygen-dependent regulation of the transcription factor HIF-1α relies on a family of prolyl hydroxylases (PHDs) that hydroxylate hypoxia-inducible factor 1α (HIF-1α) protein at two prolines during normal oxygen conditions, resulting in degrdn. by the proteasome.  During low-oxygen conditions, these prolines are no longer hydroxylated and HIF-1α degrdn. is blocked.  Hypoxia-induced miRNA-210 (miR-210) is a direct transcriptional target of HIF-1α, but its complete role and targets during hypoxia are not well understood.  Here, we identify the enzyme glycerol-3-phosphate dehydrogenase 1-like (GPD1L) as a novel regulator of HIF-1α stability and a direct target of miR-210.  Expression of miR-210 results in stabilization of HIF-1α due to decreased levels of GPD1L resulting in an increase in HIF-1α target genes.  Altering GPD1L levels by overexpression or knockdown results in a decrease or increase in HIF-1α stability, resp.  GPD1L-mediated decreases in HIF-1α stability can be reversed by pharmacol. inhibition of the proteasome or PHD activity.  When rescued from degrdn. by proteasome inhibition, elevated amts. of GPD1L cause hyperhydroxylation of HIF-1α, suggesting increases in PHD activity.  Importantly, expression of GPD1L attenuates the hypoxic response, preventing complete HIF-1α induction.  We propose a model in which hypoxia-induced miR-210 represses GPD1L, contributing to suppression of PHD activity, and increases of HIF-1α protein levels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKHol8fw_kW7Vg90H21EOLACvtfcHk0liC0XzexSviKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFKjs7c%253D&md5=af0e9747e80cb309199e0c5181d0a97c</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1128%2FMCB.01242-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01242-10%26sid%3Dliteratum%253Aachs%26aulast%3DKelly%26aufirst%3DT.%2BJ.%26aulast%3DSouza%26aufirst%3DA.%2BL.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DPuigserver%26aufirst%3DP.%26atitle%3DA%2520hypoxia-induced%2520positive%2520feedback%2520loop%2520promotes%2520hypoxia-inducible%2520factor%25201alpha%2520stability%2520through%2520miR-210%2520suppression%2520of%2520glycerol-3-phosphate%2520dehydrogenase%25201-like%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2011%26volume%3D31%26spage%3D2696%26epage%3D2706%26doi%3D10.1128%2FMCB.01242-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abelson, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, H.</span></span> <span> </span><span class="NLM_article-title">The relative amounts of the cytoplasmic RNA species in normal, transformed and senescent cultured cell lines</span>. <i>J. Cell. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">470</span>, <span class="refDoi"> DOI: 10.1002/jcp.1040900310</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1002%2Fjcp.1040900310" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=192738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaE2sXhvVOgsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1977&pages=465-470&author=L.+F.+Johnsonauthor=H.+T.+Abelsonauthor=S.+Penmanauthor=H.+Green&title=The+relative+amounts+of+the+cytoplasmic+RNA+species+in+normal%2C+transformed+and+senescent+cultured+cell+lines&doi=10.1002%2Fjcp.1040900310"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">The relative amounts of the cytoplasmic RNA species in normal, transformed and senescent cultured cell lines</span></div><div class="casAuthors">Johnson, Lee F.; Abelson, Herbert T.; Penman, Sheldon; Green, Howard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Physiology</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">465-70</span>CODEN:
                <span class="NLM_cas:coden">JCLLAX</span>;
        ISSN:<span class="NLM_cas:issn">0021-9541</span>.
    </div><div class="casAbstract">The relative quantities of 18S and 28S rRNA, 4S, RNA, and poly(A) + mRNA were examd. in mouse fibroblast lines 3T3 and 3T6 in the resting (contact inhibited) and growing (sparse) states, 3T3 clones transformed with SV40 (SV3T3) and with both SV40 and polyoma (SV-Py 3T3) viruses hamster lung fibroblasts (V79), human cervical carcinoma cells (HeLa), and human diploid fibroblasts at early and late passage.  The ratio of mRNA-to-RNA varied 1.1-2.7% in the different cell lines, the more rapidly growing cell lines usually giving a higher ratio.  In cells experiencing growth limitation either by contact inhibition or due to senescence, the ratio of mRNA-to-rRNA was ∼30% lower than in the corresponding cells in the growing state.  In most cell lines the ratio of 4S RNA-to-18S rRNA was 0.8-1.2, but in senescent fibroblasts, this ratio increased to >1.7.  Senescent fibroblasts also contained much more total RNA per unit of DNA than the same cells at early passage or than 3T6 or 3T3 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonjmUBnEcv5LVg90H21EOLACvtfcHk0liC0XzexSviKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXhvVOgsL0%253D&md5=9f81b7e26960b0aac7af8705355bc40c</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1002%2Fjcp.1040900310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcp.1040900310%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DL.%2BF.%26aulast%3DAbelson%26aufirst%3DH.%2BT.%26aulast%3DPenman%26aufirst%3DS.%26aulast%3DGreen%26aufirst%3DH.%26atitle%3DThe%2520relative%2520amounts%2520of%2520the%2520cytoplasmic%2520RNA%2520species%2520in%2520normal%252C%2520transformed%2520and%2520senescent%2520cultured%2520cell%2520lines%26jtitle%3DJ.%2520Cell.%2520Physiol.%26date%3D1977%26volume%3D90%26spage%3D465%26epage%3D470%26doi%3D10.1002%2Fjcp.1040900310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Znosko, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallahi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallo, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Analysis of secondary structural elements in human microRNA hairpin precursors</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">112</span>, <span class="refDoi"> DOI: 10.1186/s12859-016-0960-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1186%2Fs12859-016-0960-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=26928172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisFWntrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=112&author=B.+Liuauthor=J.+L.+Childs-Disneyauthor=B.+M.+Znoskoauthor=D.+Wangauthor=M.+Fallahiauthor=S.+M.+Galloauthor=M.+D.+Disney&title=Analysis+of+secondary+structural+elements+in+human+microRNA+hairpin+precursors&doi=10.1186%2Fs12859-016-0960-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of secondary structural elements in human microRNA hairpin precursors</span></div><div class="casAuthors">Liu, Biao; Childs-Disney, Jessica L.; Znosko, Brent M.; Wang, Dan; Fallahi, Mohammad; Gallo, Steven M.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112/1-112/11</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: MicroRNAs (miRNAs) regulate gene expression by targeting complementary mRNAs for destruction or translational repression.  Aberrant expression of miRNAs has been assocd. with various diseases including cancer, thus making them interesting therapeutic targets.  The composite of secondary structural elements that comprise miRNAs could aid the design of small mols. that modulate their function.  Results: We analyzed the secondary structural elements, or motifs, present in all human miRNA hairpin precursors and compared them to highly expressed human RNAs with known structures and other RNAs from various organisms.  Amongst human miRNAs, there are 3808 are unique motifs, many residing in processing sites.  Further, we identified motifs in miRNAs that are not present in other highly expressed human RNAs, desirable targets for small mols.  MiRNA motifs were incorporated into a searchable database that is freely available.  We also analyzed the most frequently occurring bulges and internal loops for each RNA class and found that the smallest loops possible prevail.  However, the distribution of loops and the preferred closing base pairs were unique to each class.  Conclusions: Collectively, we have completed a broad survey of motifs found in human miRNA precursors, highly expressed human RNAs, and RNAs from other organisms.  Interestingly, unique motifs were identified in human miRNA processing sites, binding to which could inhibit miRNA maturation and hence function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry3o_xOUoujrVg90H21EOLACvtfcHk0lhvgyei4MsCzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisFWntrs%253D&md5=9855e1cb5c01973e01f40a8d3a59f310</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1186%2Fs12859-016-0960-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-016-0960-6%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DZnosko%26aufirst%3DB.%2BM.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DGallo%26aufirst%3DS.%2BM.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DAnalysis%2520of%2520secondary%2520structural%2520elements%2520in%2520human%2520microRNA%2520hairpin%2520precursors%26jtitle%3DBMC%2520Bioinf.%26date%3D2016%26volume%3D17%26spage%3D112%26doi%3D10.1186%2Fs12859-016-0960-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seedhouse, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Defining the RNA internal loops preferred by benzimidazole derivatives via 2D combinatorial screening and computational analysis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">10111</span>– <span class="NLM_lpage">10118</span>, <span class="refDoi"> DOI: 10.1021/ja200212b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja200212b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnt1agtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=10111-10118&author=S.+P.+Velagapudiauthor=S.+J.+Seedhouseauthor=J.+Frenchauthor=M.+D.+Disney&title=Defining+the+RNA+internal+loops+preferred+by+benzimidazole+derivatives+via+2D+combinatorial+screening+and+computational+analysis&doi=10.1021%2Fja200212b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Defining the RNA Internal Loops Preferred by Benzimidazole Derivatives via 2D Combinatorial Screening and Computational Analysis</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Seedhouse, Steven J.; French, Jonathan; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10111-10118</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RNA is an important therapeutic target; however, RNA targets are generally under-exploited due to a lack of understanding of the small mols. that bind RNA and the RNA motifs that bind small mols.  Herein, we describe the identification of the RNA internal loops derived from a 4096 member 3×3 nucleotide loop library that are the most specific and highest affinity binders to a series of four designer, drug-like benzimidazoles.  These studies establish a potentially general protocol to define the highest affinity and most specific RNA motif targets for heterocyclic small mols.  Such information could be used to target functionally important RNAs in genomic sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1erYl2X0GarVg90H21EOLACvtfcHk0lhvgyei4MsCzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnt1agtbk%253D&md5=5b8ea0838113f73f2ad3f1f54f21749e</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1021%2Fja200212b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja200212b%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DSeedhouse%26aufirst%3DS.%2BJ.%26aulast%3DFrench%26aufirst%3DJ.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDefining%2520the%2520RNA%2520internal%2520loops%2520preferred%2520by%2520benzimidazole%2520derivatives%2520via%25202D%2520combinatorial%2520screening%2520and%2520computational%2520analysis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D10111%26epage%3D10118%26doi%3D10.1021%2Fja200212b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pushechnikov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span> <span> </span><span class="NLM_article-title">The role of flexibility in the rational design of modularly assembled ligands targeting the RNAs that cause the myotonic dystrophies</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">382</span>, <span class="refDoi"> DOI: 10.1002/cbic.200900716</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1002%2Fcbic.200900716" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=20058255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGhu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=375-382&author=M.+D.+Disneyauthor=M.+M.+Leeauthor=A.+Pushechnikovauthor=J.+L.+Childs-Disney&title=The+role+of+flexibility+in+the+rational+design+of+modularly+assembled+ligands+targeting+the+RNAs+that+cause+the+myotonic+dystrophies&doi=10.1002%2Fcbic.200900716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Flexibility in the Rational Design of Modularly Assembled Ligands Targeting the RNAs that Cause the Myotonic Dystrophies</span></div><div class="casAuthors">Disney, Matthew D.; Lee, Melissa M.; Pushechnikov, Alexei; Childs-Disney, Jessica L.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">375-382</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Modularly assembled ligands were designed to target the RNAs that cause two currently untreatable neuromuscular disorders, myotonic dystrophy types 1 (DM1) and 2 (DM2).  DM1 is caused by an expanded repeating sequence of CUG, and DM2 is caused by expanded CCUG repeats.  Both are present in noncoding regions and fold into hairpins with either repeating 1×1 nucleotide UU (DM1) or 2×2 nucleotide 5'-CU/3'-UC (DM2) internal loops sepd. by two GC pairs.  The repeats are toxic because they sequester the RNA splicing regulator muscleblind-like 1 protein (MBNL1).  Rational design of ligands targeting these RNAs was enabled by a database of RNA motif-ligand partners compiled by using two-dimensional combinatorial screening (2DCS).  One 2DCS study found that the 6''-azido-kanamycin A module binds internal loops similar to those found in DM1 and DM2.  In order to further enhance affinity and specificity, the ligand was assembled on a peptoid backbone to precisely control valency and the distance between ligand modules.  Designed compds. are more potent and specific binders to the toxic RNAs than MBNL1 and inhibit the formation of the RNA-protein complexes with nanomolar IC50 values.  This study shows that three important factors govern potent inhibition: (1) the surface area sequestered by the assembled ligands; (2) the spacing between ligand modules since a longer distance is required to target DM2 RNAs than DM1 RNAs; and (3) flexibility in the modular assembly scaffold used to display the RNA-binding module.  These results have impacts on the general design of assembled ligands targeting RNAs present in genomic sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-f73j_3RGI7Vg90H21EOLACvtfcHk0livOWuPsa_Wag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGhu7g%253D&md5=6c241e8fdb9de0cad47ba0ed5153e061</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200900716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200900716%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DLee%26aufirst%3DM.%2BM.%26aulast%3DPushechnikov%26aufirst%3DA.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26atitle%3DThe%2520role%2520of%2520flexibility%2520in%2520the%2520rational%2520design%2520of%2520modularly%2520assembled%2520ligands%2520targeting%2520the%2520RNAs%2520that%2520cause%2520the%2520myotonic%2520dystrophies%26jtitle%3DChemBioChem%26date%3D2010%26volume%3D11%26spage%3D375%26epage%3D382%26doi%3D10.1002%2Fcbic.200900716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafitte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duckett, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phinney, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Design of a small molecule against an oncogenic noncoding RNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">5898</span>– <span class="NLM_lpage">5903</span>, <span class="refDoi"> DOI: 10.1073/pnas.1523975113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1523975113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27170187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1ejurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=5898-5903&author=S.+P.+Velagapudiauthor=M.+D.+Cameronauthor=C.+L.+Hagaauthor=L.+H.+Rosenbergauthor=M.+Lafitteauthor=D.+R.+Duckettauthor=D.+G.+Phinneyauthor=M.+D.+Disney&title=Design+of+a+small+molecule+against+an+oncogenic+noncoding+RNA&doi=10.1073%2Fpnas.1523975113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a small molecule against an oncogenic noncoding RNA</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Cameron, Michael D.; Haga, Christopher L.; Rosenberg, Laura H.; Lafitte, Marie; Duckett, Derek R.; Phinney, Donald G.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5898-5903</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The design of precision, preclin. therapeutics from sequence is difficult, but advances in this area, particularly those focused on rational design, could quickly transform the sequence of disease-causing gene products into lead modalities.  Herein, we describe the use of Inforna, a computational approach that enables the rational design of small mols. targeting RNA to quickly provide a potent modulator of oncogenic microRNA-96 (miR-96).  We mined the secondary structure of primary microRNA-96 (pri-miR-96) hairpin precursor against a database of RNA motif-small mol. interactions, which identified modules that bound RNA motifs nearby and in the Drosha processing site.  Precise linking of these modules together provided Targaprimir-96 (3), which selectively modulates miR-96 prodn. in cancer cells and triggers apoptosis.  Importantly, the compd. is ineffective on healthy breast cells, and exogenous overexpression of pri-miR-96 reduced compd. potency in breast cancer cells.  Chem. Crosslinking and Isolation by Pull-Down (Chem-CLIP), a small-mol. RNA target validation approach, shows that 3 directly engages pri-miR-96 in breast cancer cells.  In vivo, 3 has a favorable pharmacokinetic profile and decreases tumor burden in a mouse model of triple-neg. breast cancer.  Thus, rational design can quickly produce precision, in vivo bioactive lead small mols. against hard-to-treat cancers by targeting oncogenic noncoding RNAs, advancing a disease-to-gene-to-drug paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlNk7Nj8xTxrVg90H21EOLACvtfcHk0livOWuPsa_Wag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1ejurw%253D&md5=9254bf5bef8ae21aa59ae0bca5f47f37</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1523975113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1523975113%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DHaga%26aufirst%3DC.%2BL.%26aulast%3DRosenberg%26aufirst%3DL.%2BH.%26aulast%3DLafitte%26aufirst%3DM.%26aulast%3DDuckett%26aufirst%3DD.%2BR.%26aulast%3DPhinney%26aufirst%3DD.%2BG.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDesign%2520of%2520a%2520small%2520molecule%2520against%2520an%2520oncogenic%2520noncoding%2520RNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D5898%26epage%3D5903%26doi%3D10.1073%2Fpnas.1523975113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colgan, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Precise small-molecule recognition of a toxic CUG RNA repeat expansion</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">193</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnchembio.2251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27941760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVSrurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=188-193&author=S.+G.+Rzuczekauthor=L.+A.+Colganauthor=Y.+Nakaiauthor=M.+D.+Cameronauthor=D.+Furlingauthor=R.+Yasudaauthor=M.+D.+Disney&title=Precise+small-molecule+recognition+of+a+toxic+CUG+RNA+repeat+expansion&doi=10.1038%2Fnchembio.2251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Precise small-molecule recognition of a toxic CUG RNA repeat expansion</span></div><div class="casAuthors">Rzuczek, Suzanne G.; Colgan, Lesley A.; Nakai, Yoshio; Cameron, Michael D.; Furling, Denis; Yasuda, Ryohei; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">188-193</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Excluding the ribosome and riboswitches, developing small mols. that selectively target RNA is a longstanding problem in chem. biol.  A typical cellular RNA is difficult to target because it has little tertiary, but abundant secondary structure.  We designed allele-selective compds. that target such an RNA, the toxic noncoding repeat expansion (r(CUG)exp) that causes myotonic dystrophy type 1 (DM1).  We developed several strategies to generate allele-selective small mols., including non-covalent binding, covalent binding, cleavage and on-site probe synthesis.  Covalent binding and cleavage enabled target profiling in cells derived from individuals with DM1, showing precise recognition of r(CUG)exp.  In the on-site probe synthesis approach, small mols. bound adjacent sites in r(CUG)exp and reacted to afford picomolar inhibitors via a proximity-based click reaction only in DM1-affected cells.  We expanded this approach to image r(CUG)exp in its natural context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC5kTu0Wkj9bVg90H21EOLACvtfcHk0livOWuPsa_Wag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVSrurbM&md5=bb30d79fe6e8d513a4956d13ee20f69c</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2251%26sid%3Dliteratum%253Aachs%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DColgan%26aufirst%3DL.%2BA.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DFurling%26aufirst%3DD.%26aulast%3DYasuda%26aufirst%3DR.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DPrecise%2520small-molecule%2520recognition%2520of%2520a%2520toxic%2520CUG%2520RNA%2520repeat%2520expansion%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D188%26epage%3D193%26doi%3D10.1038%2Fnchembio.2251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crynen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boström, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lekah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camerson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yildirm, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemurell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Design of a small molecule that stimulates VEGFA informed from an expanded encyclopedia of RNA fold-small molecule interactions</span>. <i>ChemRxiv</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.26434/chemrxiv.7967291.v1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.26434%2Fchemrxiv.7967291.v1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=H.+S.+Haniffauthor=L.+Knerrauthor=X.+Liuauthor=G.+Crynenauthor=J.+Bostr%C3%B6mauthor=D.+Abeggauthor=A.+Adibekianauthor=E.+Lekahauthor=K.-W.+Wangauthor=M.+D.+Camersonauthor=I.+Yildirmauthor=M.+Lemurellauthor=M.+D.+Disney&title=Design+of+a+small+molecule+that+stimulates+VEGFA+informed+from+an+expanded+encyclopedia+of+RNA+fold-small+molecule+interactions&doi=10.26434%2Fchemrxiv.7967291.v1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.26434%2Fchemrxiv.7967291.v1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.26434%252Fchemrxiv.7967291.v1%26sid%3Dliteratum%253Aachs%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DKnerr%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DCrynen%26aufirst%3DG.%26aulast%3DBostr%25C3%25B6m%26aufirst%3DJ.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DLekah%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DK.-W.%26aulast%3DCamerson%26aufirst%3DM.%2BD.%26aulast%3DYildirm%26aufirst%3DI.%26aulast%3DLemurell%26aufirst%3DM.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDesign%2520of%2520a%2520small%2520molecule%2520that%2520stimulates%2520VEGFA%2520informed%2520from%2520an%2520expanded%2520encyclopedia%2520of%2520RNA%2520fold-small%2520molecule%2520interactions%26jtitle%3DChemRxiv%26date%3D2020%26doi%3D10.26434%2Fchemrxiv.7967291.v1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
der Lee, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buljan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatheritt, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daughdrill, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunker, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuxreiter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gsponer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kriwacki, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldfield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappu, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tompa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uversky, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babu, M. M.</span></span> <span> </span><span class="NLM_article-title">Classification of intrinsically disordered regions and proteins</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">6589</span>– <span class="NLM_lpage">6631</span>, <span class="refDoi"> DOI: 10.1021/cr400525m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr400525m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFChur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=6589-6631&author=R.+van%0Ader+Leeauthor=M.+Buljanauthor=B.+Langauthor=R.+J.+Weatherittauthor=G.+W.+Daughdrillauthor=A.+K.+Dunkerauthor=M.+Fuxreiterauthor=J.+Goughauthor=J.+Gsponerauthor=D.+T.+Jonesauthor=P.+M.+Kimauthor=R.+W.+Kriwackiauthor=C.+J.+Oldfieldauthor=R.+V.+Pappuauthor=P.+Tompaauthor=V.+N.+Uverskyauthor=P.+E.+Wrightauthor=M.+M.+Babu&title=Classification+of+intrinsically+disordered+regions+and+proteins&doi=10.1021%2Fcr400525m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of intrinsically disordered regions and proteins</span></div><div class="casAuthors">van der Lee, Robin; Buljan, Marija; Lang, Benjamin; Weatheritt, Robert J.; Daughdrill, Gary W.; Dunker, A. Keith; Fuxreiter, Monika; Gough, Julian; Gsponer, Joerg; Jones, David T.; Kim, Philip M.; Kriwacki, Richard W.; Oldfield, Christopher J.; Pappu, Rohit V.; Tompa, Peter; Uversky, Vladimir N.; Wright, Peter E.; Babu, M. Madan</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6589-6631</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  While many proteins need to adopt a well-defined structure to carry out their function, a large fraction of the proteome of any organism consists of polypeptide segments that are not likely to form a defined 3-dimensional structure, but are nevertheless functional.  These protein segments are referred to as intrinsically disordered regions (IDRs).  Proteins without IDRs are called structured proteins, and proteins with entirely disordered sequences that do not adopt any tertiary structure are referred to as intrinsically disordered proteins (IDPs).  IDRs and IDPs are prevalent in eukaryotic genomes, with 44% of human protein-coding genes contg. disordered segments of >30 residues in length.  Here, the authors discuss classification approaches based on function, functional elements, sequence, protein interactions, evolution, regulation, and biophys. properties.  The authors also discuss resources that are currently available for gaining insight into IDR function, suggest areas where increased efforts are likely to advance the understanding of the functions of protein disorder, and speculate how combinations of multiple existing classification schemes could achieve high quality function prediction for IDRs, which should ultimately lead to improved function coverage and a deeper understanding of protein function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZKPkqA5jhibVg90H21EOLACvtfcHk0lhzEHIB4ISfPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFChur0%253D&md5=436cf71d9b77f6f3b176233a6abd9628</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1021%2Fcr400525m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr400525m%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BLee%26aufirst%3DR.%26aulast%3DBuljan%26aufirst%3DM.%26aulast%3DLang%26aufirst%3DB.%26aulast%3DWeatheritt%26aufirst%3DR.%2BJ.%26aulast%3DDaughdrill%26aufirst%3DG.%2BW.%26aulast%3DDunker%26aufirst%3DA.%2BK.%26aulast%3DFuxreiter%26aufirst%3DM.%26aulast%3DGough%26aufirst%3DJ.%26aulast%3DGsponer%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DD.%2BT.%26aulast%3DKim%26aufirst%3DP.%2BM.%26aulast%3DKriwacki%26aufirst%3DR.%2BW.%26aulast%3DOldfield%26aufirst%3DC.%2BJ.%26aulast%3DPappu%26aufirst%3DR.%2BV.%26aulast%3DTompa%26aufirst%3DP.%26aulast%3DUversky%26aufirst%3DV.%2BN.%26aulast%3DWright%26aufirst%3DP.%2BE.%26aulast%3DBabu%26aufirst%3DM.%2BM.%26atitle%3DClassification%2520of%2520intrinsically%2520disordered%2520regions%2520and%2520proteins%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26spage%3D6589%26epage%3D6631%26doi%3D10.1021%2Fcr400525m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uversky, V. N.</span></span> <span> </span><span class="NLM_article-title">Intrinsically disordered proteins and their “mysterious” (meta)physics</span>. <i>Front. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">10</span>, <span class="refDoi"> DOI: 10.3389/fphy.2019.00010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.3389%2Ffphy.2019.00010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=10&author=V.+N.+Uversky&title=Intrinsically+disordered+proteins+and+their+%E2%80%9Cmysterious%E2%80%9D+%28meta%29physics&doi=10.3389%2Ffphy.2019.00010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.3389%2Ffphy.2019.00010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphy.2019.00010%26sid%3Dliteratum%253Aachs%26aulast%3DUversky%26aufirst%3DV.%2BN.%26atitle%3DIntrinsically%2520disordered%2520proteins%2520and%2520their%2520%25E2%2580%259Cmysterious%25E2%2580%259D%2520%2528meta%2529physics%26jtitle%3DFront.%2520Phys.%26date%3D2019%26volume%3D7%26spage%3D10%26doi%3D10.3389%2Ffphy.2019.00010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Targeting intrinsically disordered proteins at the edge of chaos</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">217</span>– <span class="NLM_lpage">227</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2018.09.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.drudis.2018.09.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=30278223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVKkt7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=217-227&author=H.+Ruanauthor=Q.+Sunauthor=W.+Zhangauthor=Y.+Liuauthor=L.+Lai&title=Targeting+intrinsically+disordered+proteins+at+the+edge+of+chaos&doi=10.1016%2Fj.drudis.2018.09.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting intrinsically disordered proteins at the edge of chaos</span></div><div class="casAuthors">Ruan, Hao; Sun, Qi; Zhang, Weilin; Liu, Ying; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">217-227</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Intrinsically disordered proteins or intrinsically disordered regions (IDPs or IDRs) are those that do not fold into defined tertiary structures under physiol. conditions.  Given their prevalence in various diseases, IDPs are attractive therapeutic targets.  However, because of the dynamic nature of the IDP structure, conventional structure-based drug design methods cannot be directly applied.  Thanks to recent progress in understanding the mechanisms underlying IDP and ligand interactions, computational strategies for IDP-targeted rational drug discovery are emerging.  Here, we summarize recent developments in computational IDP drug design strategies and their successful applications, analyze the typical properties of reported IDP-binding compds. (iIDPs), and discuss the major challenges ahead as well as possible solns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom2VnnFE1np7Vg90H21EOLACvtfcHk0lhzEHIB4ISfPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVKkt7nM&md5=affd85fe8eb5642d69f1cb6e703be36d</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.09.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.09.017%26sid%3Dliteratum%253Aachs%26aulast%3DRuan%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DTargeting%2520intrinsically%2520disordered%2520proteins%2520at%2520the%2520edge%2520of%2520chaos%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26spage%3D217%26epage%3D227%26doi%3D10.1016%2Fj.drudis.2018.09.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kågedal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halliday, G. M.</span></span> <span> </span><span class="NLM_article-title">Alpha-synuclein biology in Lewy body diseases</span>. <i>Alzheimer’s Res. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">73</span>, <span class="refDoi"> DOI: 10.1186/s13195-014-0073-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1186%2Fs13195-014-0073-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=25580161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFCmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=73&author=W.+S.+Kimauthor=K.+K%C3%A5gedalauthor=G.+M.+Halliday&title=Alpha-synuclein+biology+in+Lewy+body+diseases&doi=10.1186%2Fs13195-014-0073-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Alpha-synuclein biology in Lewy body diseases</span></div><div class="casAuthors">Kim, Woojin Scott; Kaagedal, Katarina; Halliday, Glenda M.</div><div class="citationInfo"><span class="NLM_cas:title">Alzheimer's Research & Therapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">73/1-73/9</span>CODEN:
                <span class="NLM_cas:coden">ARTLCD</span>;
        ISSN:<span class="NLM_cas:issn">1758-9193</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">α-Synuclein is an abundantly expressed neuronal protein that is at the center of focus in understanding a group of neurodegenerative disorders called α-synucleinopathies, which are characterized by the presence of aggregated ´a-synuclein intracellularly.  Primary-synucleinopathies include Parkinson's disease (PD), dementia with Lewy bodies and multiple system atrophy, with α-synuclein also found secondarily in a no. of other diseases, including Alzheimer's disease.  Understanding how α-synuclein aggregates form in these different disorders is important for the understanding of its pathogenesis in Lewy body diseases.  PD is the most prevalent of the α-synucleinopathies and much of the initial research on α-synuclein Lewy body pathol. was based on PD but is also relevant to Lewy bodies in other diseases (dementia with Lewy bodies and Alzheimerαs disease).  Polymorphism and mutation studies of SNCA, the gene that encodes α-synuclein, provide much evidence for a causal link between α-synuclein and PD.  Among the primary α-synucleinopathies, multiple system atrophy is unique in that α-synuclein deposition occurs in oligodendrocytes rather than neurons.  It is unclear whether α-synuclein originates from oligodendrocytes or whether it is transmitted somehow from neurons. α-Synuclein exists as a natively unfolded monomer in the cytosol, but in the presence of lipid membranes it is thought to undergo a conformational change to a folded α-helical secondary structure that is prone to forming dimers and oligomers.  Posttranslational modification of α-synuclein, such as phosphorylation, ubiquitination and nitration, has been widely implicated in α-synuclein aggregation process and neurotoxicity.  Recent studies using animal and cell models, as well as autopsy studies of patients with neuron transplants, provided compelling evidence for prion-like propagation of α-synuclein.  This observation has implications for therapeutic strategies, and much recent effort is focused on developing antibodies that target extracellular α-synuclein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJE-_Hg3c0xLVg90H21EOLACvtfcHk0lgP8wHR4VlAww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFCmtg%253D%253D&md5=f1bf48c62287cf69fc540ce86d741ded</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1186%2Fs13195-014-0073-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13195-014-0073-2%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DW.%2BS.%26aulast%3DK%25C3%25A5gedal%26aufirst%3DK.%26aulast%3DHalliday%26aufirst%3DG.%2BM.%26atitle%3DAlpha-synuclein%2520biology%2520in%2520Lewy%2520body%2520diseases%26jtitle%3DAlzheimer%25E2%2580%2599s%2520Res.%2520Ther.%26date%3D2014%26volume%3D6%26spage%3D73%26doi%3D10.1186%2Fs13195-014-0073-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cahill, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahiri, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, J. T.</span></span> <span> </span><span class="NLM_article-title">Amyloid precursor protein and alpha synuclein translation, implications for iron and inflammation in neurodegenerative diseases</span>. <i>Biochim. Biophys. Acta, Gen. Subj.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1790</i></span>,  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">628</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2008.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.bbagen.2008.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=19166904" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1yjtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1790&publication_year=2009&pages=615-628&author=C.+M.+Cahillauthor=D.+K.+Lahiriauthor=X.+Huangauthor=J.+T.+Rogers&title=Amyloid+precursor+protein+and+alpha+synuclein+translation%2C+implications+for+iron+and+inflammation+in+neurodegenerative+diseases&doi=10.1016%2Fj.bbagen.2008.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Amyloid precursor protein and alpha synuclein translation, implications for iron and inflammation in neurodegenerative diseases</span></div><div class="casAuthors">Cahill, Catherine M.; Lahiri, Debomoy K.; Huang, Xudong; Rogers, Jack T.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1790</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">615-628</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Recent studies that alleles in the hemochromatosis gene may accelerate the onset of Alzheimer's disease by five years have validated interest in the model in which metals (particularly iron) accelerate disease course.  Biochem. and biophys. measurements demonstrated the presence of elevated levels of neurotoxic copper, zinc and iron in the brains of AD patients.  Intracellular levels of APP holoprotein were shown to be modulated by iron by a mechanism that is similar to the translation control of the ferritin L- and H mRNAs by iron-responsive element (IRE) RNA stem loops in their 5' untranslated regions (5'UTRs).  More recently a putative IRE-like sequence was hypothesized present in the Parkinsons's alpha synuclein (ASYN) transcript (see [A.L. Friedlich, R.E.Tanzi, J.T. Rogers, The 5'-untranslated region of Parkinson's disease alpha-synuclein mRNA contains a predicted iron responsive element, Mol. Psychiatry 12 (2007) 222-223. [6]]).  Together with the demonstration of metal dependent translation of APP mRNA, the involvement of metals in the plaque of AD patients and of increased iron in striatal neurons in the substantia nigra (SN) of Parkinson's disease patients have stimulated the development of metal attenuating agents and iron chelators as a major new therapeutic strategy for the treatment of these neurodegenerative diseases.  In the case of AD, metal based therapeutics may ultimately prove more cost effective than the use of an amyloid vaccine as the preferred anti-amyloid therapeutic strategy to ameliorate the cognitive decline of AD patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5db9ufEacxbVg90H21EOLACvtfcHk0lgP8wHR4VlAww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1yjtLY%253D&md5=54ca57f06dbfe2a2278b8d820ccefa3b</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2008.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2008.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DCahill%26aufirst%3DC.%2BM.%26aulast%3DLahiri%26aufirst%3DD.%2BK.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DRogers%26aufirst%3DJ.%2BT.%26atitle%3DAmyloid%2520precursor%2520protein%2520and%2520alpha%2520synuclein%2520translation%252C%2520implications%2520for%2520iron%2520and%2520inflammation%2520in%2520neurodegenerative%2520diseases%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2009%26volume%3D1790%26spage%3D615%26epage%3D628%26doi%3D10.1016%2Fj.bbagen.2008.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishnu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouradian, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Translation of the intrinsically disordered protein α-synuclein is inhibited by a small molecule targeting its structured mRNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1457</span>– <span class="NLM_lpage">1467</span>, <span class="refDoi"> DOI: 10.1073/pnas.1905057117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1905057117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31900363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFKktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=1457-1467&author=P.+Zhangauthor=H.-J.+Parkauthor=J.+Zhangauthor=E.+Junnauthor=R.+J.+Andrewsauthor=S.+P.+Velagapudiauthor=D.+Abeggauthor=K.+Vishnuauthor=M.+G.+Costalesauthor=J.+L.+Childs-Disneyauthor=A.+Adibekianauthor=W.+N.+Mossauthor=M.+M.+Mouradianauthor=M.+D.+Disney&title=Translation+of+the+intrinsically+disordered+protein+%CE%B1-synuclein+is+inhibited+by+a+small+molecule+targeting+its+structured+mRNA&doi=10.1073%2Fpnas.1905057117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Translation of the intrinsically disordered protein α-synuclein is inhibited by a small molecule targeting its structured mRNA</span></div><div class="casAuthors">Zhang, Peiyuan; Park, Hye-Jin; Zhang, Jie; Junn, Eunsung; Andrews, Ryan J.; Velagapudi, Sai Pradeep; Abegg, Daniel; Vishnu, Kamalakannan; Costales, Matthew G.; Childs-Disney, Jessica L.; Adibekian, Alexander; Moss, Walter N.; Mouradian, M. Maral; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1457-1467</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Many proteins are refractory to targeting because they lack small-mol. binding pockets.  An alternative to drugging these proteins directly is to target the messenger (m)RNA that encodes them, thereby reducing protein levels.  We describe such an approach for the difficult-to-target protein α-synuclein encoded by the SNCA gene.  Multiplication of the SNCA gene locus causes dominantly inherited Parkinson's disease (PD), and α-synuclein protein aggregates in Lewy bodies and Lewy neurites in sporadic PD.  Thus, reducing the expression of α-synuclein protein is expected to have therapeutic value.  Fortuitously, the SNCA mRNA has a structured iron-responsive element (IRE) in its 5' untranslated region (5' UTR) that controls its translation.  Using sequence-based design, we discovered small mols. that target the IRE structure and inhibit SNCA translation in cells, the most potent of which is named Synucleozid.  Both in vitro and cellular profiling studies showed Synucleozid directly targets the α-synuclein mRNA 5' UTR at the designed site.α.  Mechanistic studies revealed that Synucleozid reduces α-synuclein protein levels by decreasing the amt. of SNCA mRNA loaded into polysomes, mechanistically providing a cytoprotective effect in cells.  Proteome- and transcriptome-wide studies showed that the compd.'s selectivity makes Synucleozid suitable for further development.  Importantly, transcriptome-wide anal. of mRNAs that encode intrinsically disordered proteins revealed that each has structured regions that could be targeted with small mols.  These findings demonstrate the potential for targeting undruggable proteins at the level of their coding mRNAs.  This approach, as applied to SNCA, is a promising disease-modifying therapeutic strategy for PD and other α-synucleinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvPjL_9G6xS7Vg90H21EOLACvtfcHk0lgP8wHR4VlAww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFKktbc%253D&md5=4f2a60d2c0fe09ea99f05291621ad4ef</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1905057117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1905057117%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DPark%26aufirst%3DH.-J.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DJunn%26aufirst%3DE.%26aulast%3DAndrews%26aufirst%3DR.%2BJ.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DVishnu%26aufirst%3DK.%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26aulast%3DMouradian%26aufirst%3DM.%2BM.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DTranslation%2520of%2520the%2520intrinsically%2520disordered%2520protein%2520%25CE%25B1-synuclein%2520is%2520inhibited%2520by%2520a%2520small%2520molecule%2520targeting%2520its%2520structured%2520mRNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D1457%26epage%3D1467%26doi%3D10.1073%2Fpnas.1905057117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, J. R.</span></span> <span> </span><span class="NLM_article-title">Kinetic intermediates in RNA folding</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>265</i></span>,  <span class="NLM_fpage">918</span>– <span class="NLM_lpage">924</span>, <span class="refDoi"> DOI: 10.1126/science.8052848</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1126%2Fscience.8052848" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=8052848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADyaK2cXmslClsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1994&pages=918-924&author=P.+P.+Zarrinkarauthor=J.+R.+Williamson&title=Kinetic+intermediates+in+RNA+folding&doi=10.1126%2Fscience.8052848"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic intermediates in RNA folding</span></div><div class="casAuthors">Zarrinkar, Patrick P.; Williamson, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">5174</span>),
    <span class="NLM_cas:pages">918-24</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The folding pathways of large, highly structured RNA mols. are largely unexplored.  Insight into both the kinetics of folding and the presence of intermediates was provided in a study of the Mg2+-induced folding of the Tetrahymena ribozyme by hybridization of complementary oligodeoxynucleotide probes.  This RNA folds via a complex mechanism involving both Mg2+-dependent and Mg2+-independent steps.  A hierarchical model for the folding pathway is proposed in which formation of one helical domain (P4-P6) precedes that of a second helical domain (P3-P7).  The overall rate-limiting step is formation of P3-P7, and takes place with an obsd. rate const. of 0.72 min-1.  The folding mechanism of large RNAs appears similar to that of many multidomain proteins in that formation of independently stable substructures precedes their assocn. into the final conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrty3NoGILqxbVg90H21EOLACvtfcHk0lj1afaOIPQY4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmslClsLk%253D&md5=db9be5dab3438ebadad715ac5e1461e3</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1126%2Fscience.8052848&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8052848%26sid%3Dliteratum%253Aachs%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DWilliamson%26aufirst%3DJ.%2BR.%26atitle%3DKinetic%2520intermediates%2520in%2520RNA%2520folding%26jtitle%3DScience%26date%3D1994%26volume%3D265%26spage%3D918%26epage%3D924%26doi%3D10.1126%2Fscience.8052848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">A toxic RNA catalyzes the in cellulo synthesis of its own inhibitor</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">10956</span>– <span class="NLM_lpage">10959</span>, <span class="refDoi"> DOI: 10.1002/anie.201406465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1002%2Fanie.201406465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=25164984" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC2M%252FlsFKnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=10956-10959&author=S.+G.+Rzuczekauthor=H.+Parkauthor=M.+D.+Disney&title=A+toxic+RNA+catalyzes+the+in+cellulo+synthesis+of+its+own+inhibitor&doi=10.1002%2Fanie.201406465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">A toxic RNA catalyzes the in cellulo synthesis of its own inhibitor</span></div><div class="casAuthors">Rzuczek Suzanne G; Park HaJeung; Disney Matthew D</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">10956-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Potent modulators of RNA function can be assembled in cellulo by using the cell as a reaction vessel and a disease-causing RNA as a catalyst.  When designing small molecule effectors of function, a balance between permeability and potency must be struck.  Low molecular weight compounds are more permeable whereas higher molecular weight compounds are more potent.  The advantages of both types of compounds could be synergized if low molecular weight molecules could be transformed into potent, multivalent ligands by a reaction that is catalyzed by binding to a target in cells expressing a genetic defect.  It was shown that this approach is indeed viable in cellulo.  Small molecule modules with precisely positioned alkyne and azide moieties bind adjacent internal loops in r(CCUG)(exp), the causative agent of myotonic dystrophy type 2 (DM2), and are transformed into oligomeric, potent inhibitors of DM2 RNA dysfunction by a Huisgen 1,3-dipolar cycloaddition reaction, a variant of click chemistry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQTJ9u4F-oycFQKX3ooHxWjfW6udTcc2eYgXRM5Q6MO1Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M%252FlsFKnsw%253D%253D&md5=2fb96c08ea5546457c9cd4d050fb0201</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1002%2Fanie.201406465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201406465%26sid%3Dliteratum%253Aachs%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DA%2520toxic%2520RNA%2520catalyzes%2520the%2520in%2520cellulo%2520synthesis%2520of%2520its%2520own%2520inhibitor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2014%26volume%3D53%26spage%3D10956%26epage%3D10959%26doi%3D10.1002%2Fanie.201406465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Using genome sequence to enable the design of medicines and chemical probes</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1663</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00504</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00504" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlvFGqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=1599-1663&author=A.+J.+Angelbelloauthor=J.+L.+Chenauthor=J.+L.+Childs-Disneyauthor=P.+Zhangauthor=Z.-F.+Wangauthor=M.+D.+Disney&title=Using+genome+sequence+to+enable+the+design+of+medicines+and+chemical+probes&doi=10.1021%2Facs.chemrev.7b00504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Using Genome Sequence to Enable the Design of Medicines and Chemical Probes</span></div><div class="casAuthors">Angelbello, Alicia J.; Chen, Jonathan L.; Childs-Disney, Jessica L.; Zhang, Peiyuan; Wang, Zi-Fu; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1599-1663</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Rapid progress in genome sequencing technol. has put us firmly into a postgenomic era.  A key challenge in biomedical research is harnessing genome sequence to fulfill the promise of personalized medicine.  This Review describes how genome sequencing has enabled the identification of disease-causing biomols. and how these data have been converted into chem. probes of function, preclin. lead modalities, and ultimately U.S.  Food and Drug Administration (FDA)-approved drugs.  In particular, the authors focus on the use of oligonucleotide-based modalities to target disease-causing RNAs; small mols. that target DNA, RNA, or protein; the rational repurposing of known therapeutic modalities; and the advantages of pharmacogenetics.  Lastly, the authors discuss the remaining challenges and opportunities in the direct utilization of genome sequence to enable design of medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreyX8o6MEWwLVg90H21EOLACvtfcHk0lgaz7-rC6Fr3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlvFGqsQ%253D%253D&md5=6cf4911da55d4304d7c818f0f7a4a6a0</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00504%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DZ.-F.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DUsing%2520genome%2520sequence%2520to%2520enable%2520the%2520design%2520of%2520medicines%2520and%2520chemical%2520probes%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D1599%26epage%3D1663%26doi%3D10.1021%2Facs.chemrev.7b00504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mateos-Aierdi, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goicoechea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiastui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Torrón, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Puga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matheu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López de Munain, A.</span></span> <span> </span><span class="NLM_article-title">Muscle wasting in myotonic dystrophies: a model of premature aging</span>. <i>Front. Aging Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">125</span>, <span class="refDoi"> DOI: 10.3389/fnagi.2015.00125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.3389%2Ffnagi.2015.00125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=26217220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC28%252FnsVOgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=125&author=A.+J.+Mateos-Aierdiauthor=M.+Goicoecheaauthor=A.+Aiastuiauthor=R.+Fern%C3%A1ndez-Torr%C3%B3nauthor=M.+Garcia-Pugaauthor=A.+Matheuauthor=A.+L%C3%B3pez+de+Munain&title=Muscle+wasting+in+myotonic+dystrophies%3A+a+model+of+premature+aging&doi=10.3389%2Ffnagi.2015.00125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Muscle wasting in myotonic dystrophies: a model of premature aging</span></div><div class="casAuthors">Mateos-Aierdi Alba Judith; Goicoechea Maria; Aiastui Ana; Fernandez-Torron Roberto; Garcia-Puga Mikel; Matheu Ander; Lopez de Munain Adolfo</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in aging neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">125</span>
        ISSN:<span class="NLM_cas:issn">1663-4365</span>.
    </div><div class="casAbstract">Myotonic dystrophy type 1 (DM1 or Steinert's disease) and type 2 (DM2) are multisystem disorders of genetic origin.  Progressive muscular weakness, atrophy and myotonia are the most prominent neuromuscular features of these diseases, while other clinical manifestations such as cardiomyopathy, insulin resistance and cataracts are also common.  From a clinical perspective, most DM symptoms are interpreted as a result of an accelerated aging (cataracts, muscular weakness and atrophy, cognitive decline, metabolic dysfunction, etc.), including an increased risk of developing tumors.  From this point of view, DM1 could be described as a progeroid syndrome since a notable age-dependent dysfunction of all systems occurs.  The underlying molecular disorder in DM1 consists of the existence of a pathological (CTG) triplet expansion in the 3' untranslated region (UTR) of the Dystrophia Myotonica Protein Kinase (DMPK) gene, whereas (CCTG)n repeats in the first intron of the Cellular Nucleic acid Binding Protein/Zinc Finger Protein 9 (CNBP/ZNF9) gene cause DM2.  The expansions are transcribed into (CUG)n and (CCUG)n-containing RNA, respectively, which form secondary structures and sequester RNA-binding proteins, such as the splicing factor muscleblind-like protein (MBNL), forming nuclear aggregates known as foci.  Other splicing factors, such as CUGBP, are also disrupted, leading to a spliceopathy of a large number of downstream genes linked to the clinical features of these diseases.  Skeletal muscle regeneration relies on muscle progenitor cells, known as satellite cells, which are activated after muscle damage, and which proliferate and differentiate to muscle cells, thus regenerating the damaged tissue.  Satellite cell dysfunction seems to be a common feature of both age-dependent muscle degeneration (sarcopenia) and muscle wasting in DM and other muscle degenerative diseases.  This review aims to describe the cellular, molecular and macrostructural processes involved in the muscular degeneration seen in DM patients, highlighting the similarities found with muscle aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSw12Ob59ewiLxanA7vODO6fW6udTcc2eYi8ANqsPqWt7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252FnsVOgtA%253D%253D&md5=fa6238e374e69abf9ae9cf9dda9be4ae</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.3389%2Ffnagi.2015.00125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnagi.2015.00125%26sid%3Dliteratum%253Aachs%26aulast%3DMateos-Aierdi%26aufirst%3DA.%2BJ.%26aulast%3DGoicoechea%26aufirst%3DM.%26aulast%3DAiastui%26aufirst%3DA.%26aulast%3DFern%25C3%25A1ndez-Torr%25C3%25B3n%26aufirst%3DR.%26aulast%3DGarcia-Puga%26aufirst%3DM.%26aulast%3DMatheu%26aufirst%3DA.%26aulast%3DL%25C3%25B3pez%2Bde%2BMunain%26aufirst%3DA.%26atitle%3DMuscle%2520wasting%2520in%2520myotonic%2520dystrophies%253A%2520a%2520model%2520of%2520premature%2520aging%26jtitle%3DFront.%2520Aging%2520Neurosci.%26date%3D2015%26volume%3D7%26spage%3D125%26doi%3D10.3389%2Ffnagi.2015.00125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meola, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardani, R.</span></span> <span> </span><span class="NLM_article-title">Myotonic dystrophies: An update on clinical aspects, genetic, pathology, and molecular pathomechanisms</span>. <i>Biochim. Biophys. Acta, Mol. Basis Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1852</i></span>,  <span class="NLM_fpage">594</span>– <span class="NLM_lpage">606</span>, <span class="refDoi"> DOI: 10.1016/j.bbadis.2014.05.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.bbadis.2014.05.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslKktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1852&publication_year=2015&pages=594-606&author=G.+Meolaauthor=R.+Cardani&title=Myotonic+dystrophies%3A+An+update+on+clinical+aspects%2C+genetic%2C+pathology%2C+and+molecular+pathomechanisms&doi=10.1016%2Fj.bbadis.2014.05.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Myotonic dystrophies: An update on clinical aspects, genetic, pathology, and molecular pathomechanisms</span></div><div class="casAuthors">Meola, Giovanni; Cardani, Rosanna</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Basis of Disease</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1852</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">594-606</span>CODEN:
                <span class="NLM_cas:coden">BBADEX</span>;
        ISSN:<span class="NLM_cas:issn">0925-4439</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  Myotonic dystrophy (DM) is the most common adult muscular dystrophy, characterized by autosomal dominant progressive myopathy, myotonia and multiorgan involvement.  To date two distinct forms caused by similar mutations have been identified.  Myotonic dystrophy type 1 (DM1, Steinert's disease) is caused by a (CTG)n expansion in DMPK, while myotonic dystrophy type 2 (DM2) is caused by a (CCTG)n expansion in ZNF9/CNBP.  When transcribed into CUG/CCUG-contg. RNA, mutant transcripts aggregate as nuclear foci that sequester RNA-binding proteins, resulting in spliceopathy of downstream effector genes.  However, it is now clear that addnl. pathogenic mechanism like changes in gene expression, protein translation and micro-RNA metab. may also contribute to disease pathol.  Despite clin. and genetic similarities, DM1 and DM2 are distinct disorders requiring different diagnostic and management strategies.  This review is an update on the recent advances in the understanding of the mol. mechanisms behind myotonic dystrophies.  This article is part of a Special Issue entitled: Neuromuscular Diseases: Pathol. and Mol. Pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB007xIEAaQrVg90H21EOLACvtfcHk0lgaz7-rC6Fr3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslKktbc%253D&md5=1c3446d7430fcfd9222b460925e910db</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1016%2Fj.bbadis.2014.05.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbadis.2014.05.019%26sid%3Dliteratum%253Aachs%26aulast%3DMeola%26aufirst%3DG.%26aulast%3DCardani%26aufirst%3DR.%26atitle%3DMyotonic%2520dystrophies%253A%2520An%2520update%2520on%2520clinical%2520aspects%252C%2520genetic%252C%2520pathology%252C%2520and%2520molecular%2520pathomechanisms%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Basis%2520Dis.%26date%3D2015%26volume%3D1852%26spage%3D594%26epage%3D606%26doi%3D10.1016%2Fj.bbadis.2014.05.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Udd, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krahe, R.</span></span> <span> </span><span class="NLM_article-title">The myotonic dystrophies: molecular, clinical, and therapeutic challenges</span>. <i>Lancet Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">891</span>– <span class="NLM_lpage">905</span>, <span class="refDoi"> DOI: 10.1016/S1474-4422(12)70204-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2FS1474-4422%2812%2970204-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=22995693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVSns7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=891-905&author=B.+Uddauthor=R.+Krahe&title=The+myotonic+dystrophies%3A+molecular%2C+clinical%2C+and+therapeutic+challenges&doi=10.1016%2FS1474-4422%2812%2970204-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">The myotonic dystrophies: molecular, clinical, and therapeutic challenges</span></div><div class="casAuthors">Udd, Bjarne; Krahe, Ralf</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Neurology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">891-905</span>CODEN:
                <span class="NLM_cas:coden">LNAEAM</span>;
        ISSN:<span class="NLM_cas:issn">1474-4422</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Summary: Myotonic dystrophy is the most common type of muscular dystrophy in adults and is characterized by progressive myopathy, myotonia, and multiorgan involvement.  Two genetically distinct entities have been identified.  Myotonic dystrophy type 1 (also known as Steinert's disease) was first described more than 100 years ago, whereas myotonic dystrophy type 2 was identified only 18 years ago, after genetic testing for type 1 disease could be applied.  Both diseases are caused by autosomal dominant nucleotide repeat expansions.  In patients with myotonic dystrophy type 1, a (CTG)n expansion is present in DMPK, whereas in patients with type 2 disease, there is a (CCTG)n expansion in CNBP.  When transcribed into CUG-contg. RNA, mutant transcripts aggregate as nuclear foci that sequester RNA-binding proteins, resulting in a spliceopathy of downstream effector genes.  The prevailing paradigm therefore is that both disorders are toxic RNA diseases.  However, research indicates several addnl. pathogenic effects take place with respect to protein translation and turnover.  Despite clin. and genetic similarities, myotonic dystrophy type 1 and type 2 are distinct disorders requiring different diagnostic and management strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqElKJ28_rHObVg90H21EOLACvtfcHk0lgaz7-rC6Fr3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVSns7fK&md5=40f7d2204cc59403b438ce8840d37498</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1016%2FS1474-4422%2812%2970204-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1474-4422%252812%252970204-1%26sid%3Dliteratum%253Aachs%26aulast%3DUdd%26aufirst%3DB.%26aulast%3DKrahe%26aufirst%3DR.%26atitle%3DThe%2520myotonic%2520dystrophies%253A%2520molecular%252C%2520clinical%252C%2520and%2520therapeutic%2520challenges%26jtitle%3DLancet%2520Neurol.%26date%3D2012%26volume%3D11%26spage%3D891%26epage%3D905%26doi%3D10.1016%2FS1474-4422%2812%2970204-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Features of modularly assembled compounds that impart bioactivity against an RNA target</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2321</span>, <span class="refDoi"> DOI: 10.1021/cb400265y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400265y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVCgtL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2312-2321&author=S.+G.+Rzuczekauthor=Y.+Gaoauthor=Z.-Z.+Tangauthor=C.+A.+Thorntonauthor=T.+Kodadekauthor=M.+D.+Disney&title=Features+of+modularly+assembled+compounds+that+impart+bioactivity+against+an+RNA+target&doi=10.1021%2Fcb400265y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Features of Modularly Assembled Compounds That Impart Bioactivity Against an RNA Target</span></div><div class="casAuthors">Rzuczek, Suzanne G.; Gao, Yu; Tang, Zhen-Zhi; Thornton, Charles A.; Kodadek, Thomas; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2312-2321</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Transcriptomes provide a myriad of potential RNAs that could be the targets of therapeutics or chem. genetic probes of function.  Cell-permeable small mols., however, generally do not exploit these targets, owing to the difficulty in the design of high affinity, specific small mols. targeting RNA.  As part of a general program to study RNA function using small mols., we designed bioactive, modularly assembled small mols. that target the noncoding expanded RNA repeat that causes myotonic dystrophy type 1 (DM1), r-(CUG)exp.  Herein, we present a rigorous study to elucidate features in modularly assembled compds. that afford bioactivity.  Different modular assembly scaffolds were investigated, including polyamines, α-peptides, β-peptides, and peptide tertiary amides (PTAs).  On the basis of activity as assessed by improvement of DM1-assocd. defects, stability against proteases, cellular permeability, and toxicity, we discovered that constrained backbones, namely, PTAs, are optimal.  Notably, we detd. that r-(CUG)exp is the target of the optimal PTA in cellular models and that the optimal PTA improves DM1-assocd. defects in a mouse model.  Biophys. analyses were employed to investigate potential sources of bioactivity.  These investigations show that modularly assembled compds. have increased residence times on their targets and faster on rates than the RNA-binding modules from which they were derived.  Moreover, they have faster on rates than the protein that binds r-(CUG)exp, the inactivation of which gives rise to DM1-assocd. defects.  These studies provide information about features of small mols. that are programmable for targeting RNA, allowing for the facile optimization of therapeutics or chem. probes against other cellular RNA targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiiHXEm7vakrVg90H21EOLACvtfcHk0lg5W0qswQKZZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVCgtL%252FL&md5=89a6385bd74e38e54bde583c339c23cc</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1021%2Fcb400265y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400265y%26sid%3Dliteratum%253Aachs%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DZ.-Z.%26aulast%3DThornton%26aufirst%3DC.%2BA.%26aulast%3DKodadek%26aufirst%3DT.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DFeatures%2520of%2520modularly%2520assembled%2520compounds%2520that%2520impart%2520bioactivity%2520against%2520an%2520RNA%2520target%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2312%26epage%3D2321%26doi%3D10.1021%2Fcb400265y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moon, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilimire, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneekloth, J. S.</span></span> <span> </span><span class="NLM_article-title">Measuring RNA-ligand interactions with microscale thermophoresis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4638</span>– <span class="NLM_lpage">4643</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.7b01141</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.7b01141" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1Slsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=4638-4643&author=M.+H.+Moonauthor=T.+A.+Hilimireauthor=A.+M.+Sandersauthor=J.+S.+Schneekloth&title=Measuring+RNA-ligand+interactions+with+microscale+thermophoresis&doi=10.1021%2Facs.biochem.7b01141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Measuring RNA-Ligand Interactions with Microscale Thermophoresis</span></div><div class="casAuthors">Moon, Michelle H.; Hilimire, Thomas A.; Sanders, Allix M.; Schneekloth, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">4638-4643</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the past twenty years, there has been a dramatic increase in the study of RNA.  RNA has gone from being known as an intermediate in the central dogma of mol. biol. to a mol. with a large diversity of structure and function that is involved in all aspects of biol.  As new functions are rapidly discovered, it has become clear that there is a need for RNA-targeting small mol. probes to investigate RNA biol. and clarify the potential for therapeutics based on RNA/small mol. interactions.  While a host of techniques exist to measure small mol./RNA interactions, many of these have drawbacks that make them intractable for routine use and are often not broadly applicable.  A newer technol. called microscale thermophoresis (MST), which measures the directed migration of a mol. and/or mol. ligand complex along a temp. gradient, can be used to measure binding affinities using very small amts. of sample, greatly reducing cost of materials.  Here, we demonstrate how MST can be used to study a range of RNA interactions, including peptide/RNA interactions, RNA/small mol. interactions, and displacement of an RNA bound peptide by a small mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUVOWHOSVjTLVg90H21EOLACvtfcHk0lg5W0qswQKZZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1Slsg%253D%253D&md5=0aad61e97e0a21d3004e7f99c3b57c31</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.7b01141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.7b01141%26sid%3Dliteratum%253Aachs%26aulast%3DMoon%26aufirst%3DM.%2BH.%26aulast%3DHilimire%26aufirst%3DT.%2BA.%26aulast%3DSanders%26aufirst%3DA.%2BM.%26aulast%3DSchneekloth%26aufirst%3DJ.%2BS.%26atitle%3DMeasuring%2520RNA-ligand%2520interactions%2520with%2520microscale%2520thermophoresis%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26spage%3D4638%26epage%3D4643%26doi%3D10.1021%2Facs.biochem.7b01141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Umuhire Juru, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patwardhan, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargrove, A. E.</span></span> <span> </span><span class="NLM_article-title">Understanding the contributions of conformational changes, thermodynamics, and kinetics of RNA-small molecule interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">824</span>– <span class="NLM_lpage">838</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00945</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00945" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosFWntbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=824-838&author=A.+Umuhire+Juruauthor=N.+N.+Patwardhanauthor=A.+E.+Hargrove&title=Understanding+the+contributions+of+conformational+changes%2C+thermodynamics%2C+and+kinetics+of+RNA-small+molecule+interactions&doi=10.1021%2Facschembio.8b00945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding the Contributions of Conformational Changes, Thermodynamics, and Kinetics of RNA-Small Molecule Interactions</span></div><div class="casAuthors">Umuhire Juru, Aline; Patwardhan, Neeraj N.; Hargrove, Amanda E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">824-838</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The implication of RNA in multiple cellular processes beyond protein coding has revitalized interest in the development of small mols. for therapeutically targeting RNA and for further probing its cellular biol.  However, the process of rationally designing such small mol. probes is hampered by the paucity of information about fundamental mol. recognition principles of RNA.  In this Review, the authors summarize two important and often underappreciated aspects of RNA-small mol. recognition: RNA conformational dynamics and the biophys. properties of interactions of small mols. with RNA, specifically thermodn. and kinetics.  While conformational flexibility is often said to impede RNA ligand development, the ability of small mols. to influence the RNA conformational landscape can have a significant effect on the cellular functions of RNA.  An anal. of the conformational landscape of RNA and the interactions of individual conformations with ligands can thus guide the development of new small mol. probes, which needs to be investigated further.  Addnl., while it is common practice to quantify the binding affinities (Ka or Kd) of small mols. for biomacromols. as a measure of their activity, further biophys. characterization of their interaction can provide a deeper understanding.  Studies that focus on the thermodn. and kinetic parameters for interaction between RNA and ligands are next discussed.  Finally, this Review provides the reader with a perspective on how such in-depth anal. of biophys. characteristics of the interaction of RNA and small mols. can impact the understanding of these interactions and how they will benefit the future design of small mol. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9o1fhVs7EBbVg90H21EOLACvtfcHk0ljFkpnPppcUTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosFWntbs%253D&md5=c8986916178a2acc361c64f5c18327ee</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00945%26sid%3Dliteratum%253Aachs%26aulast%3DUmuhire%2BJuru%26aufirst%3DA.%26aulast%3DPatwardhan%26aufirst%3DN.%2BN.%26aulast%3DHargrove%26aufirst%3DA.%2BE.%26atitle%3DUnderstanding%2520the%2520contributions%2520of%2520conformational%2520changes%252C%2520thermodynamics%252C%2520and%2520kinetics%2520of%2520RNA-small%2520molecule%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D824%26epage%3D838%26doi%3D10.1021%2Facschembio.8b00945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blakeley, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePorter, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolbert, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNaughton, B. R.</span></span> <span> </span><span class="NLM_article-title">Methods for identifying and characterizing interactions involving RNA</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">8837</span>– <span class="NLM_lpage">8855</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2012.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.tet.2012.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVyjsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=8837-8855&author=B.+D.+Blakeleyauthor=S.+M.+DePorterauthor=U.+Mohanauthor=R.+Buraiauthor=B.+S.+Tolbertauthor=B.+R.+McNaughton&title=Methods+for+identifying+and+characterizing+interactions+involving+RNA&doi=10.1016%2Fj.tet.2012.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Methods for identifying and characterizing interactions involving RNA</span></div><div class="casAuthors">Blakeley, Brett D.; DePorter, Sandra M.; Mohan, Utpal; Burai, Ritwik; Tolbert, Blanton S.; McNaughton, Brian R.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">8837-8855</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Topics discussed include fluorescence-based methods, microarrays, selection-based methods, and low-throughput methods for measuring ligand-RNA interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc9bjJ4rDA_LVg90H21EOLACvtfcHk0ljFkpnPppcUTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVyjsLnP&md5=ecbfe5736239b30b9e615d3b51dc78ed</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2012.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2012.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DBlakeley%26aufirst%3DB.%2BD.%26aulast%3DDePorter%26aufirst%3DS.%2BM.%26aulast%3DMohan%26aufirst%3DU.%26aulast%3DBurai%26aufirst%3DR.%26aulast%3DTolbert%26aufirst%3DB.%2BS.%26aulast%3DMcNaughton%26aufirst%3DB.%2BR.%26atitle%3DMethods%2520for%2520identifying%2520and%2520characterizing%2520interactions%2520involving%2520RNA%26jtitle%3DTetrahedron%26date%3D2012%26volume%3D68%26spage%3D8837%26epage%3D8855%26doi%3D10.1016%2Fj.tet.2012.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wicks, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargrove, A. E.</span></span> <span> </span><span class="NLM_article-title">Fluorescent indicator displacement assays to identify and characterize small molecule interactions with RNA</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2019.04.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.ymeth.2019.04.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31051253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpsVOqs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2019&pages=3-14&author=S.+L.+Wicksauthor=A.+E.+Hargrove&title=Fluorescent+indicator+displacement+assays+to+identify+and+characterize+small+molecule+interactions+with+RNA&doi=10.1016%2Fj.ymeth.2019.04.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescent indicator displacement assays to identify and characterize small molecule interactions with RNA</span></div><div class="casAuthors">Wicks, Sarah L.; Hargrove, Amanda E.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3-14</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Fluorescent indicator displacement (FID) assays are an advantageous approach to convert receptors into optical sensors that can detect binding of various ligands.  In particular, the identification of ligands that bind to RNA receptors has become of increasing interest as the roles of RNA in cellular processes and disease pathogenesis continue to be discovered.  Small mols. have been validated as tools to elucidate unknown RNA functions, underscoring the crit. need to rapidly identify and quant. characterize RNA:small mol. interactions for the development of chem. probes.  The successful application of FID assays to evaluate interactions between diverse RNA receptors and small mols. has been facilitated by the characterization of distinct fluorescent indicators that reversibly bind RNA and modulate the fluorescence signal.  The utility of RNA-based FID assays to both academia and industry has been demonstrated through numerous uses in high-throughput screening efforts, structure-activity relationship studies, and in vitro target engagement studies.  Furthermore, the development, optimization, and validation of a variety of RNA-based FID assays has led to general guidelines that can be utilized for facile implementation of the method with new or underexplored RNA receptors.  Altogether, the use of RNA-based FID assays as a general anal. tool has provided valuable insights into small mol. affinity and selectivity, furthering the fundamental understanding of RNA:small mol. recognition.  In this review, we will summarize efforts to employ FID assays using RNA receptors and describe the significant contributions of the method towards the development of chem. probes to reveal unknown RNA functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZAGIlueOLvrVg90H21EOLACvtfcHk0ljFkpnPppcUTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpsVOqs7g%253D&md5=c6955af7de2b7beafd3e7c3879c35e60</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2019.04.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2019.04.018%26sid%3Dliteratum%253Aachs%26aulast%3DWicks%26aufirst%3DS.%2BL.%26aulast%3DHargrove%26aufirst%3DA.%2BE.%26atitle%3DFluorescent%2520indicator%2520displacement%2520assays%2520to%2520identify%2520and%2520characterize%2520small%2520molecule%2520interactions%2520with%2520RNA%26jtitle%3DMethods%26date%3D2019%26volume%3D167%26spage%3D3%26epage%3D14%26doi%3D10.1016%2Fj.ymeth.2019.04.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuliano, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palladino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">John, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowicz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sausville, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukow, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chait, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galluzzo, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheltzer, J. M.</span></span> <span> </span><span class="NLM_article-title">Off-target toxicity is a common mechanism of action of cancer drugs undergoing clinical trials</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">eaaw8412</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.aaw8412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1126%2Fscitranslmed.aaw8412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31511426" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&author=A.+Linauthor=C.+J.+Giulianoauthor=A.+Palladinoauthor=K.+M.+Johnauthor=C.+Abramowiczauthor=M.+L.+Yuanauthor=E.+L.+Sausvilleauthor=D.+A.+Lukowauthor=L.+Liuauthor=A.+R.+Chaitauthor=Z.+C.+Galluzzoauthor=C.+Tuckerauthor=J.+M.+Sheltzer&title=Off-target+toxicity+is+a+common+mechanism+of+action+of+cancer+drugs+undergoing+clinical+trials&doi=10.1126%2Fscitranslmed.aaw8412"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aaw8412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aaw8412%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DA.%26aulast%3DGiuliano%26aufirst%3DC.%2BJ.%26aulast%3DPalladino%26aufirst%3DA.%26aulast%3DJohn%26aufirst%3DK.%2BM.%26aulast%3DAbramowicz%26aufirst%3DC.%26aulast%3DYuan%26aufirst%3DM.%2BL.%26aulast%3DSausville%26aufirst%3DE.%2BL.%26aulast%3DLukow%26aufirst%3DD.%2BA.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChait%26aufirst%3DA.%2BR.%26aulast%3DGalluzzo%26aufirst%3DZ.%2BC.%26aulast%3DTucker%26aufirst%3DC.%26aulast%3DSheltzer%26aufirst%3DJ.%2BM.%26atitle%3DOff-target%2520toxicity%2520is%2520a%2520common%2520mechanism%2520of%2520action%2520of%2520cancer%2520drugs%2520undergoing%2520clinical%2520trials%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2019%26volume%3D11%26doi%3D10.1126%2Fscitranslmed.aaw8412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirley, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bleackley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shafee, T.</span></span> <span> </span><span class="NLM_article-title">Transcriptomics technologies</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e1005457</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005457</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1371%2Fjournal.pcbi.1005457" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28545146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1CitrbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&author=R.+Loweauthor=N.+Shirleyauthor=M.+Bleackleyauthor=S.+Dolanauthor=T.+Shafee&title=Transcriptomics+technologies&doi=10.1371%2Fjournal.pcbi.1005457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptomics technologies</span></div><div class="casAuthors">Lowe, Rohan; Shirley, Neil; Bleackley, Mark; Dolan, Stephen; Shafee, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e1005457/1-e1005457/23</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Transcriptomics technologies are the techniques used to study an organism's transcriptome, the sum of all of its RNA transcripts.  The information content of an organism is recorded in the DNA of its genome and expressed through transcription.  Here, mRNA serves as a transient intermediary mol. in the information network, while noncoding RNAs perform addnl. diverse functions.  A transcriptome captures a snapshot in time of the total transcripts present in a cell.  The first attempts to study the whole transcriptome began in the early 1990s, and technol. advances since the late 1990s have made transcriptomics a widespread discipline.  Transcriptomics has been defined by repeated technol. innovations that transform the field.  There are two key contemporary techniques in the field: microarrays, which quantify a set of predetd. sequences, and RNA sequencing (RNA-Seq), which uses high-throughput sequencing to capture all sequences.  Measuring the expression of an organism's genes in different tissues, conditions, or time points gives information on how genes are regulated and reveals details of an organism's biol.  It can also help to infer the functions of previously unannotated genes.  Transcriptomic anal. has enabled the study of how gene expression changes in different organisms and has been instrumental in the understanding of human disease.  An anal. of gene expression in its entirety allows detection of broad coordinated trends which cannot be discerned by more targeted assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ8OybmUCAlbVg90H21EOLACvtfcHk0liP7xpu9ufhcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1CitrbM&md5=9e38118fab964a734cb224d1c5c08156</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005457%26sid%3Dliteratum%253Aachs%26aulast%3DLowe%26aufirst%3DR.%26aulast%3DShirley%26aufirst%3DN.%26aulast%3DBleackley%26aufirst%3DM.%26aulast%3DDolan%26aufirst%3DS.%26aulast%3DShafee%26aufirst%3DT.%26atitle%3DTranscriptomics%2520technologies%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2017%26volume%3D13%26doi%3D10.1371%2Fjournal.pcbi.1005457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Git, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvinge, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmon-Divon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadfield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertone, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldas, C.</span></span> <span> </span><span class="NLM_article-title">Systematic comparison of microarray profiling, real-time PCR, and next-generation sequencing technologies for measuring differential microRNA expression</span>. <i>RNA</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">991</span>– <span class="NLM_lpage">1006</span>, <span class="refDoi"> DOI: 10.1261/rna.1947110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1261%2Frna.1947110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=20360395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlslOmu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=991-1006&author=A.+Gitauthor=H.+Dvingeauthor=M.+Salmon-Divonauthor=M.+Osborneauthor=C.+Kutterauthor=J.+Hadfieldauthor=P.+Bertoneauthor=C.+Caldas&title=Systematic+comparison+of+microarray+profiling%2C+real-time+PCR%2C+and+next-generation+sequencing+technologies+for+measuring+differential+microRNA+expression&doi=10.1261%2Frna.1947110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic comparison of microarray profiling, real-time PCR, and next-generation sequencing technologies for measuring differential microRNA expression</span></div><div class="casAuthors">Git, Anna; Dvinge, Heidi; Salmon-Divon, Mali; Osborne, Michelle; Kutter, Claudia; Hadfield, James; Bertone, Paul; Caldas, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">RNA</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">991-1006</span>CODEN:
                <span class="NLM_cas:coden">RNARFU</span>;
        ISSN:<span class="NLM_cas:issn">1355-8382</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">RNA abundance and DNA copy no. are routinely measured in high-throughput using microarray and next-generation sequencing (NGS) technologies, and the attributes of different platforms have been extensively analyzed.  Recently, the application of both microarrays and NGS has expanded to include microRNAs (miRNAs), but the relative performance of these methods has not been rigorously characterized.  We analyzed three biol. samples across six miRNA microarray platforms and compared their hybridization performance.  We examd. the utility of these platforms, as well as NGS, for the detection of differentially expressed miRNAs.  We then validated the results for 89 miRNAs by real-time RT-PCR and challenged the use of this assay as a "gold std.".  Finally, we implemented a novel method to evaluate false-pos. and false-neg. rates for all methods in the absence of a ref. method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqK5DxDMY2fNLVg90H21EOLACvtfcHk0lh0x0CPj-NmyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlslOmu7s%253D&md5=d7e5d56b792ccf2ad5f8d3e4552afd27</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1261%2Frna.1947110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.1947110%26sid%3Dliteratum%253Aachs%26aulast%3DGit%26aufirst%3DA.%26aulast%3DDvinge%26aufirst%3DH.%26aulast%3DSalmon-Divon%26aufirst%3DM.%26aulast%3DOsborne%26aufirst%3DM.%26aulast%3DKutter%26aufirst%3DC.%26aulast%3DHadfield%26aufirst%3DJ.%26aulast%3DBertone%26aufirst%3DP.%26aulast%3DCaldas%26aufirst%3DC.%26atitle%3DSystematic%2520comparison%2520of%2520microarray%2520profiling%252C%2520real-time%2520PCR%252C%2520and%2520next-generation%2520sequencing%2520technologies%2520for%2520measuring%2520differential%2520microRNA%2520expression%26jtitle%3DRNA%26date%3D2010%26volume%3D16%26spage%3D991%26epage%3D1006%26doi%3D10.1261%2Frna.1947110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bittker, J. A.</span></span> <span> </span><span class="NLM_article-title">High-throughput RT-PCR for small-molecule screening assays</span>. <i>Curr. Protoc. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1002/9780470559277.ch110204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1002%2F9780470559277.ch110204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=23487248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC2srmvFGqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=49-63&author=J.+A.+Bittker&title=High-throughput+RT-PCR+for+small-molecule+screening+assays&doi=10.1002%2F9780470559277.ch110204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput RT-PCR for small-molecule screening assays</span></div><div class="casAuthors">Bittker Joshua A</div><div class="citationInfo"><span class="NLM_cas:title">Current protocols in chemical biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Quantitative measurement of the levels of mRNA expression using real-time reverse transcription polymerase chain reaction (RT-PCR) has long been used for analyzing expression differences in tissue or cell lines of interest.  This method has been used somewhat less frequently to measure the changes in gene expression due to perturbagens such as small molecules or siRNA.  The availability of new instrumentation for liquid handling and real-time PCR analysis as well as the commercial availability of start-to-finish kits for RT-PCR has enabled the use of this method for high-throughput small-molecule screening on a scale comparable to traditional high-throughput screening (HTS) assays.  This protocol focuses on the special considerations necessary for using quantitative RT-PCR as a primary small-molecule screening assay, including the different methods available for mRNA isolation and analysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT8tsd63zvM2ZotVjB3IRIyfW6udTcc2eaebqfeo2J9Wbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srmvFGqtw%253D%253D&md5=d580a34d032303110167269f4bc5626d</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1002%2F9780470559277.ch110204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9780470559277.ch110204%26sid%3Dliteratum%253Aachs%26aulast%3DBittker%26aufirst%3DJ.%2BA.%26atitle%3DHigh-throughput%2520RT-PCR%2520for%2520small-molecule%2520screening%2520assays%26jtitle%3DCurr.%2520Protoc.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D4%26spage%3D49%26epage%3D63%26doi%3D10.1002%2F9780470559277.ch110204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Citri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Südhof, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wernig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malenka, R. C.</span></span> <span> </span><span class="NLM_article-title">Comprehensive qPCR profiling of gene expression in single neuronal cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">127</span>, <span class="refDoi"> DOI: 10.1038/nprot.2011.430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnprot.2011.430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Kitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=118-127&author=A.+Citriauthor=Z.+P.+Pangauthor=T.+C.+S%C3%BCdhofauthor=M.+Wernigauthor=R.+C.+Malenka&title=Comprehensive+qPCR+profiling+of+gene+expression+in+single+neuronal+cells&doi=10.1038%2Fnprot.2011.430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive qPCR profiling of gene expression in single neuronal cells</span></div><div class="casAuthors">Citri, Ami; Pang, Zhiping P.; Sudhof, Thomas C.; Wernig, Marius; Malenka, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">118-127</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A major challenge in neuronal stem cell biol. lies in characterization of lineage-specific reprogrammed human neuronal cells, a process that necessitates the use of an assay sensitive to the single-cell level.  Single-cell gene profiling can provide definitive evidence regarding the conversion of one cell type into another at a high level of resoln.  The protocol we describe uses Fluidigm Biomark dynamic arrays for high-throughput expression profiling from single neuronal cells, assaying up to 96 independent samples with up to 96 quant. PCR (qPCR) probes (equiv. to 9,216 reactions) in a single expt., which can be completed within 2-3 d.  The protocol enables simple and cost-effective profiling of several hundred transcripts from a single cell, and it could have numerous utilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW5UaKD-acu7Vg90H21EOLACvtfcHk0lj3tLjNpU0iwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Kitb0%253D&md5=a5211923274be519f9aceb1f3de49996</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2011.430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2011.430%26sid%3Dliteratum%253Aachs%26aulast%3DCitri%26aufirst%3DA.%26aulast%3DPang%26aufirst%3DZ.%2BP.%26aulast%3DS%25C3%25BCdhof%26aufirst%3DT.%2BC.%26aulast%3DWernig%26aufirst%3DM.%26aulast%3DMalenka%26aufirst%3DR.%2BC.%26atitle%3DComprehensive%2520qPCR%2520profiling%2520of%2520gene%2520expression%2520in%2520single%2520neuronal%2520cells%26jtitle%3DNat.%2520Protoc.%26date%3D2012%26volume%3D7%26spage%3D118%26epage%3D127%26doi%3D10.1038%2Fnprot.2011.430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bosc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, P.</span></span> <span> </span><span class="NLM_article-title">The use of novel selectivity metrics in kinase research</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">17</span>, <span class="refDoi"> DOI: 10.1186/s12859-016-1413-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1186%2Fs12859-016-1413-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28056771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC1c7is1Orsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=17&author=N.+Boscauthor=C.+Meyerauthor=P.+Bonnet&title=The+use+of+novel+selectivity+metrics+in+kinase+research&doi=10.1186%2Fs12859-016-1413-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">The use of novel selectivity metrics in kinase research</span></div><div class="casAuthors">Bosc Nicolas; Bonnet Pascal; Meyer Christophe</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Compound selectivity is an important issue when developing a new drug.  In many instances, a lack of selectivity can translate to increased toxicity.  Protein kinases are particularly concerned with this issue because they share high sequence and structural similarity.  However, selectivity may be assessed early on using data generated from protein kinase profiling panels.  RESULTS:  To guide lead optimization in drug discovery projects, we propose herein two new selectivity metrics, namely window score (WS) and ranking score (RS).  These metrics can be applied to standard in vitro data-including intrinsic enzyme activity/affinity (Ki, IC50 or percentage of inhibition), cell-based potency (percentage of effect, EC50) or even kinetics data (Kd, Kon and Koff).  They are both easy to compute and offer different viewpoints from which to consider compound selectivity.  CONCLUSIONS:  We performed a comparative analysis of their respective performance on several data sets against already published selectivity metrics and analyzed how they might influence compound selection.  Our results showed that the two new metrics bring additional information to prioritize compound selection.  Two novel metrics were developed to better estimate selectivity of compounds screened on multiple proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQBArFY-xzq77BItaL1WaVAfW6udTcc2eaQuHaKOlH247ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c7is1Orsg%253D%253D&md5=c8bb088288d1fc3fedc2628303ddeb46</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1186%2Fs12859-016-1413-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-016-1413-y%26sid%3Dliteratum%253Aachs%26aulast%3DBosc%26aufirst%3DN.%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DBonnet%26aufirst%3DP.%26atitle%3DThe%2520use%2520of%2520novel%2520selectivity%2520metrics%2520in%2520kinase%2520research%26jtitle%3DBMC%2520Bioinf.%26date%3D2017%26volume%3D18%26spage%3D17%26doi%3D10.1186%2Fs12859-016-1413-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Graczyk, P. P.</span></span> <span> </span><span class="NLM_article-title">Gini coefficient: a new way to express selectivity of kinase inhibitors against a family of kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">5773</span>– <span class="NLM_lpage">5779</span>, <span class="refDoi"> DOI: 10.1021/jm070562u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070562u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFyhu7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=5773-5779&author=P.+P.+Graczyk&title=Gini+coefficient%3A+a+new+way+to+express+selectivity+of+kinase+inhibitors+against+a+family+of+kinases&doi=10.1021%2Fjm070562u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Gini Coefficient: A New Way To Express Selectivity of Kinase Inhibitors against a Family of Kinases</span></div><div class="casAuthors">Graczyk, Piotr P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5773-5779</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel application of the Gini coeff. for expressing selectivity of kinase inhibitors against a panel of kinases is proposed.  This has been illustrated using single-point inhibition data for 40 com. available kinase inhibitors screened against 85 kinases.  Nonselective inhibitors are characterized by Gini values close to zero (Staurosporine, Gini 0.150).  Highly selective compds. exhibit Gini values close to 1 (PD184352 Gini 0.905).  The relative selectivity of inhibitors does not depend on the ATP concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqUHpmF-C9u7Vg90H21EOLACvtfcHk0ljRjXEiAl-bTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFyhu7rL&md5=51f4902e49196dfef8e135c4a09182d7</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1021%2Fjm070562u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070562u%26sid%3Dliteratum%253Aachs%26aulast%3DGraczyk%26aufirst%3DP.%2BP.%26atitle%3DGini%2520coefficient%253A%2520a%2520new%2520way%2520to%2520express%2520selectivity%2520of%2520kinase%2520inhibitors%2520against%2520a%2520family%2520of%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D5773%26epage%3D5779%26doi%3D10.1021%2Fjm070562u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ali, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axtman, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuzzi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luecking, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saikatendu, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span> <span> </span><span class="NLM_article-title">Progress towards a public chemogenomic set for protein kinases and a call for contributions</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0181585</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0181585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1371%2Fjournal.pone.0181585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28767711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFWktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=D.+H.+Drewryauthor=C.+I.+Wellsauthor=D.+M.+Andrewsauthor=R.+Angellauthor=H.+Al-Aliauthor=A.+D.+Axtmanauthor=S.+J.+Capuzziauthor=J.+M.+Elkinsauthor=P.+Ettmayerauthor=M.+Frederiksenauthor=O.+Gileadiauthor=N.+Grayauthor=A.+Hooperauthor=S.+Knappauthor=S.+Lauferauthor=U.+Lueckingauthor=M.+Michaelidesauthor=S.+Mullerauthor=E.+Muratovauthor=R.+A.+Dennyauthor=K.+S.+Saikatenduauthor=D.+K.+Treiberauthor=W.+J.+Zuercherauthor=T.+M.+Willson&title=Progress+towards+a+public+chemogenomic+set+for+protein+kinases+and+a+call+for+contributions&doi=10.1371%2Fjournal.pone.0181585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Progress towards a public chemogenomic set for protein kinases and a call for contributions</span></div><div class="casAuthors">Drewry, David H.; Wells, Carrow I.; Andrews, David M.; Angell, Richard; Al-Ali, Hassan; Axtman, Alison D.; Capuzzi, Stephen J.; Elkins, Jonathan M.; Ettmayer, Peter; Frederiksen, Mathias; Gileadi, Opher; Gray, Nathanael; Hooper, Alice; Knapp, Stefan; Laufer, Stefan; Luecking, Ulrich; Michaelides, Michael; Muller, Susanne; Muratov, Eugene; Aldrin, Denny R.; Saikatendu, Kumar S.; Treiber, Daniel K.; Zuercher, William J.; Willson, Timothy M.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e0181585/1-e0181585/20</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein kinases are highly tractable targets for drug discovery.  However, the biol. function and therapeutic potential of the majority of the 500+ human protein kinases remains unknown.  We have developed phys. and virtual collections of small mol. inhibitors, which we call chemogenomic sets, that are designed to inhibit the catalytic function of almost half the human protein kinases.  In this manuscript we share our progress towards generation of a comprehensive kinase chemogenomic set (KCGS), release kinome profiling data of a large inhibitor set (Published Kinase Inhibitor Set 2 (PKIS2)), and outline a process through which the community can openly collaborate to create a KCGS that probes the full complement of human protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpozOiNZl51XbVg90H21EOLACvtfcHk0ljRjXEiAl-bTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFWktL0%253D&md5=9c27ba5cd21da43d30ce485ffa0e1675</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0181585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0181585%26sid%3Dliteratum%253Aachs%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DAndrews%26aufirst%3DD.%2BM.%26aulast%3DAngell%26aufirst%3DR.%26aulast%3DAl-Ali%26aufirst%3DH.%26aulast%3DAxtman%26aufirst%3DA.%2BD.%26aulast%3DCapuzzi%26aufirst%3DS.%2BJ.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DHooper%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLaufer%26aufirst%3DS.%26aulast%3DLuecking%26aufirst%3DU.%26aulast%3DMichaelides%26aufirst%3DM.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DSaikatendu%26aufirst%3DK.%2BS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26atitle%3DProgress%2520towards%2520a%2520public%2520chemogenomic%2520set%2520for%2520protein%2520kinases%2520and%2520a%2520call%2520for%2520contributions%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0181585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strivelli, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phinney, D. G.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of miR-544 biogenesis disrupts adaptive responses to hypoxia by modulating ATM-mTOR signaling</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2267</span>– <span class="NLM_lpage">2276</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00265</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00265" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFOlsLvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2267-2276&author=C.+L.+Hagaauthor=S.+P.+Velagapudiauthor=J.+R.+Strivelliauthor=W.+Y.+Yangauthor=M.+D.+Disneyauthor=D.+G.+Phinney&title=Small+molecule+inhibition+of+miR-544+biogenesis+disrupts+adaptive+responses+to+hypoxia+by+modulating+ATM-mTOR+signaling&doi=10.1021%2Facschembio.5b00265"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibition of miR-544 Biogenesis Disrupts Adaptive Responses to Hypoxia by Modulating ATM-mTOR Signaling</span></div><div class="casAuthors">Haga, Christopher L.; Velagapudi, Sai Pradeep; Strivelli, Jacqueline R.; Yang, Wang-Yong; Disney, Matthew D.; Phinney, Donald G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2267-2276</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hypoxia induces a complex circuit of gene expression that drives tumor progression and increases drug resistance.  Defining these changes allows for an understanding of how hypoxia alters tumor biol. and informs design of lead therapeutics.  We probed the role of microRNA-544 (miR-544), which silences mammalian target of rapamycin (mTOR), in a hypoxic breast cancer model by using a small mol. (1) that selectively impedes the microRNA's biogenesis.  Application of 1 to hypoxic tumor cells selectively inhibited prodn. of the mature microRNA, sensitized cells to 5-fluorouracil, and derepressed mRNAs affected by miR-544 in cellulo and in vivo, including boosting mTOR expression.  Thus, small mol. inhibition of miR-544 reverses a tumor cell's physiol. response to hypoxia.  Importantly, 1 sensitized tumor cells to hypoxia-assocd. apoptosis at a 25-fold lower concn. than a 2'-O-Me RNA antagomir and was as selective.  Further, the apoptotic effect of 1 was suppressed by treatment of cell with rapamycin, a well-known inhibitor of the mTOR signaling pathway, illustrating the selectivity of the compd.  Thus, RNA-directed chem. probes, which could also serve as lead therapeutics, enable interrogation of complex cellular networks in cells and animals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrV19uI0H2fLVg90H21EOLACvtfcHk0lidRGLxfR-q7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOlsLvE&md5=32efb5ba1d1712026f1d158be1390fbe</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00265%26sid%3Dliteratum%253Aachs%26aulast%3DHaga%26aufirst%3DC.%2BL.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DStrivelli%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DW.%2BY.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DPhinney%26aufirst%3DD.%2BG.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520miR-544%2520biogenesis%2520disrupts%2520adaptive%2520responses%2520to%2520hypoxia%2520by%2520modulating%2520ATM-mTOR%2520signaling%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D2267%26epage%3D2276%26doi%3D10.1021%2Facschembio.5b00265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenny, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montanari, C. A.</span></span> <span> </span><span class="NLM_article-title">Inflation of correlation in the pursuit of drug-likeness</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1007/s10822-012-9631-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1007%2Fs10822-012-9631-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=23306465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1SrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=1-13&author=P.+W.+Kennyauthor=C.+A.+Montanari&title=Inflation+of+correlation+in+the+pursuit+of+drug-likeness&doi=10.1007%2Fs10822-012-9631-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Inflation of correlation in the pursuit of drug-likeness</span></div><div class="casAuthors">Kenny, Peter W.; Montanari, Carlos A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Drug-likeness is a frequently invoked, although not always precisely defined, concept in drug discovery.  Opinions on drug-likeness are to a large extent shaped by the relationships that are obsd. between surrogate measures of drug-likeness (e.g. aq. soly.; permeability; pharmacol. promiscuity) and fundamental physicochem. properties (e.g. lipophilicity; mol. size).  This article draws on examples from the literature to highlight approaches to data anal. that exaggerate trends in data and the term correlation inflation is introduced in the context of drug discovery.  Averaging groups of data points prior to anal. is a common cause of correlation inflation and results from anal. of binned continuous data should always be treated with caution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreGcOTh1EJCrVg90H21EOLACvtfcHk0lidRGLxfR-q7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1SrtbY%253D&md5=e8542e2402b11cfb2fd98ae8a5185f46</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9631-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9631-5%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DMontanari%26aufirst%3DC.%2BA.%26atitle%3DInflation%2520of%2520correlation%2520in%2520the%2520pursuit%2520of%2520drug-likeness%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D1%26epage%3D13%26doi%3D10.1007%2Fs10822-012-9631-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span> <span> </span><span class="NLM_article-title">Two decades under the influence of the Rule of Five and the changing properties of approved oral drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1714</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00686</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00686" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslait73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1701-1714&author=M.+D.+Shultz&title=Two+decades+under+the+influence+of+the+Rule+of+Five+and+the+changing+properties+of+approved+oral+drugs&doi=10.1021%2Facs.jmedchem.8b00686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Two Decades under the Influence of the Rule of Five and the Changing Properties of Approved Oral Drugs</span></div><div class="casAuthors">Shultz, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1701-1714</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two decades have passed since the rule of five ushered in the concept of "drug-like" properties.  Attempts to quantify, correlate, and categorize mols. based on Ro5 parameters evolved into the introduction of efficiency metrics with far reaching consequences in decision making by industry leaders and scientists seeking to discover new medicines.  Examn. of oral drug parameters approved before and after the original Ro5 anal. demonstrates that some parameters such as clogP and HBD remained const. while the cutoffs for parameters such as mol. wt. and HBA have increased substantially over the past 20 years.  The time dependent increase in the mol. wt. of oral drugs during the past 20 years provides compelling evidence to disprove the hypothesis that mol. wt. is a "drug-like" property.  This anal. does not validate parameters that have not changed as being "drug-like" but instead calls into question the entire hypothesis that "drug-like" properties exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnoxpQsv2kbVg90H21EOLACvtfcHk0lgZxwg3KUv_Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslait73K&md5=d3c8b1107133dc0b7b277d22fa029597</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00686%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26atitle%3DTwo%2520decades%2520under%2520the%2520influence%2520of%2520the%2520Rule%2520of%2520Five%2520and%2520the%2520changing%2520properties%2520of%2520approved%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1701%26epage%3D1714%26doi%3D10.1021%2Facs.jmedchem.8b00686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gueret, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adihou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemurell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span> <span> </span><span class="NLM_article-title">New modalities for challenging targets in drug discovery</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">10294</span>– <span class="NLM_lpage">10323</span>, <span class="refDoi"> DOI: 10.1002/anie.201611914</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1002%2Fanie.201611914" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28186380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A280%3ADC%252BC1c3ls1Wruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=10294-10323&author=E.+Valeurauthor=S.+M.+Gueretauthor=H.+Adihouauthor=R.+Gopalakrishnanauthor=M.+Lemurellauthor=H.+Waldmannauthor=T.+N.+Grossmannauthor=A.+T.+Plowright&title=New+modalities+for+challenging+targets+in+drug+discovery&doi=10.1002%2Fanie.201611914"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">New Modalities for Challenging Targets in Drug Discovery</span></div><div class="casAuthors">Valeur Eric; Gueret Stephanie M; Adihou Helene; Gopalakrishnan Ranganath; Lemurell Malin; Plowright Alleyn T; Gueret Stephanie M; Adihou Helene; Gopalakrishnan Ranganath; Waldmann Herbert; Waldmann Herbert; Grossmann Tom N; Grossmann Tom N</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">10294-10323</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Our ever-increasing understanding of biological systems is providing a range of exciting novel biological targets, whose modulation may enable novel therapeutic options for many diseases.  These targets include protein-protein and protein-nucleic acid interactions, which are, however, often refractory to classical small-molecule approaches.  Other types of molecules, or modalities, are therefore required to address these targets, which has led several academic research groups and pharmaceutical companies to increasingly use the concept of so-called "new modalities".  This Review defines for the first time the scope of this term, which includes novel peptidic scaffolds, oligonucleotides, hybrids, molecular conjugates, as well as new uses of classical small molecules.  We provide the most representative examples of these modalities to target large binding surface areas such as those found in protein-protein interactions and for biological processes at the center of cell regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQq7SOpoq1Cja91pGlMMoT5fW6udTcc2ebGOzUSV5THc7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c3ls1Wruw%253D%253D&md5=569b3b4a194d0682b6c81603b29f9b8a</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611914&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611914%26sid%3Dliteratum%253Aachs%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DGueret%26aufirst%3DS.%2BM.%26aulast%3DAdihou%26aufirst%3DH.%26aulast%3DGopalakrishnan%26aufirst%3DR.%26aulast%3DLemurell%26aufirst%3DM.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26atitle%3DNew%2520modalities%2520for%2520challenging%2520targets%2520in%2520drug%2520discovery%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2017%26volume%3D56%26spage%3D10294%26epage%3D10323%26doi%3D10.1002%2Fanie.201611914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagorce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douguet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miteva, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villoutreix, B. O.</span></span> <span> </span><span class="NLM_article-title">Computational analysis of calculated physicochemical and ADMET properties of protein-protein interaction inhibitors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">46277</span>, <span class="refDoi"> DOI: 10.1038/srep46277</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fsrep46277" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=28397808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVenurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=46277&author=D.+Lagorceauthor=D.+Douguetauthor=M.+A.+Mitevaauthor=B.+O.+Villoutreix&title=Computational+analysis+of+calculated+physicochemical+and+ADMET+properties+of+protein-protein+interaction+inhibitors&doi=10.1038%2Fsrep46277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Computational analysis of calculated physicochemical and ADMET properties of protein-protein interaction inhibitors</span></div><div class="casAuthors">Lagorce, David; Douguet, Dominique; Miteva, Maria A.; Villoutreix, Bruno O.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">46277</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The modulation of PPIs by low mol. wt. chem. compds., particularly by orally bioavailable mols., would be very valuable in numerous disease indications.  However, it is known that PPI inhibitors (iPPIs) tend to have properties that are linked to poor Absorption, Distribution, Metab., Excretion and Toxicity (ADMET) and in some cases to poor clin. outcomes.  Previously reported in silico analyses of iPPIs have essentially focused on physicochem. properties but several other ADMET parameters would be important to assess.  In order to gain new insights into the ADMET properties of iPPIs, computations were carried out on eight datasets collected from several databases.  These datasets involve compds. targeting enzymes, GPCRs, ion channels, nuclear receptors, allosteric modulators, oral marketed drugs, oral natural product-derived marketed drugs and iPPIs.  Several trends are reported that should assist the design and optimization of future PPI inhibitors, either for drug discovery endeavors or for chem. biol. projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw3PmL5znBCLVg90H21EOLACvtfcHk0lhtBSfyCs4jLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVenurk%253D&md5=162ddf6594d20c770c1016313182245c</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1038%2Fsrep46277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep46277%26sid%3Dliteratum%253Aachs%26aulast%3DLagorce%26aufirst%3DD.%26aulast%3DDouguet%26aufirst%3DD.%26aulast%3DMiteva%26aufirst%3DM.%2BA.%26aulast%3DVilloutreix%26aufirst%3DB.%2BO.%26atitle%3DComputational%2520analysis%2520of%2520calculated%2520physicochemical%2520and%2520ADMET%2520properties%2520of%2520protein-protein%2520interaction%2520inhibitors%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D46277%26doi%3D10.1038%2Fsrep46277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref121"><div class="reference"><strong class="refLabel"><a href="#ref121" class="rightTabRefNumLink">121</a></strong><div class="NLM_citation" id="rightTab-cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanaba, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donlic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karloff, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forte, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargrove, A. E.</span></span> <span> </span><span class="NLM_article-title">R-BIND: an interactive database for exploring and developing RNA-targeted chemical probes</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2691</span>– <span class="NLM_lpage">2700</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFWlsbfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=2691-2700&author=B.+S.+Morganauthor=B.+G.+Sanabaauthor=A.+Donlicauthor=D.+B.+Karloffauthor=J.+E.+Forteauthor=Y.+Zhangauthor=A.+E.+Hargrove&title=R-BIND%3A+an+interactive+database+for+exploring+and+developing+RNA-targeted+chemical+probes&doi=10.1021%2Facschembio.9b00631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">R-BIND: An Interactive Database for Exploring and Developing RNA-Targeted Chemical Probes</span></div><div class="casAuthors">Morgan, Brittany S.; Sanaba, Bilva G.; Donlic, Anita; Karloff, Diane B.; Forte, Jordan E.; Zhang, Yuqi; Hargrove, Amanda E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2691-2700</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">While the opportunities available for targeting RNA with small mols. have been widely appreciated, the challenges assocd. with achieving specific RNA recognition in biol. systems have hindered progress and prevented many researchers from entering the field.  To facilitate the discovery of RNA-targeted chem. probes and their subsequent applications, the authors curated the RNA-targeted BIoactive ligaNd Database (R-BIND).  This collection contains an array of information on reported chem. probes that target nonrRNA and have biol. activity, and anal. has led to the discovery of RNA-privileged properties.  Herein, the authors developed an online platform to make this information freely available to the community, offering search options, a suite of tools for probe development, and an updated R-BIND data set with detailed exptl. information for each probe.  The authors repeated the previous cheminformatics anal. on the updated R-BIND list and found that the distinguishing physicochem., structural, and spatial properties remained unchanged, despite an almost 50% increase in the database size.  Further, the authors developed several user-friendly tools, including queries based on cheminformatic parameters, exptl. details, functional groups, and substructures.  In addn., a nearest neighbors algorithm can assess the similarity of user-uploaded mols. to R-BIND ligands.  These tools and resources can be used to design small mol. libraries, optimize lead ligands, or select targets, probes, assays, and control expts.  Chem. probes are crit. to the study and discovery of novel functions for RNA, and the authors expect this resource to greatly assist researchers in exploring and developing successful RNA-targeted probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPoJQXYLWkH7Vg90H21EOLACvtfcHk0lhtBSfyCs4jLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFWlsbfL&md5=02d2d41ec793396243f2935514c6b572</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00631%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DB.%2BS.%26aulast%3DSanaba%26aufirst%3DB.%2BG.%26aulast%3DDonlic%26aufirst%3DA.%26aulast%3DKarloff%26aufirst%3DD.%2BB.%26aulast%3DForte%26aufirst%3DJ.%2BE.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DHargrove%26aufirst%3DA.%2BE.%26atitle%3DR-BIND%253A%2520an%2520interactive%2520database%2520for%2520exploring%2520and%2520developing%2520RNA-targeted%2520chemical%2520probes%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D2691%26epage%3D2700%26doi%3D10.1021%2Facschembio.9b00631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref122"><div class="reference"><strong class="refLabel"><a href="#ref122" class="rightTabRefNumLink">122</a></strong><div class="NLM_citation" id="rightTab-cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrivastava, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gautam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassanali, M.</span></span> <span> </span><span class="NLM_article-title">DrugBank: a knowledgebase for drugs, drug actions and drug targets</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">D901</span>– <span class="NLM_lpage">D906</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1093%2Fnar%2Fgkm958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=18048412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVSrs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D901-D906&author=D.+S.+Wishartauthor=C.+Knoxauthor=A.+C.+Guoauthor=D.+Chengauthor=S.+Shrivastavaauthor=D.+Tzurauthor=B.+Gautamauthor=M.+Hassanali&title=DrugBank%3A+a+knowledgebase+for+drugs%2C+drug+actions+and+drug+targets&doi=10.1093%2Fnar%2Fgkm958"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank: a knowledgebase for drugs, drug actions and drug targets</span></div><div class="casAuthors">Wishart, David S.; Knox, Craig; Guo, An Chi; Cheng, Dean; Shrivastava, Savita; Tzur, Dan; Gautam, Bijaya; Hassanali, Murtaza</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D901-D906</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a richly annotated resource that combines detailed drug data with comprehensive drug target and drug action information.  Since its first release in 2006, DrugBank has been widely used to facilitate in silico drug target discovery, drug design, drug docking or screening, drug metab. prediction, drug interaction prediction and general pharmaceutical education.  The latest version of DrugBank (release 2.0) has been expanded significantly over the previous release.  With ∼4900 drug entries, it now contains 60% more FDA-approved small mol. and biotech drugs including 10% more exptl.' drugs.  Significantly, more protein target data has also been added to the database, with the latest version of DrugBank contg. three times as many non-redundant protein or drug target sequences as before (1565 vs. 524).  Each DrugCard entry now contains more than 100 data fields with half of the information being devoted to drug/chem. data and the other half devoted to pharmacol., pharmacogenomic and mol. biol. data.  A no. of new data fields, including food-drug interactions, drug-drug interactions and exptl. ADME data have been added in response to numerous user requests.  DrugBank has also significantly improved the power and simplicity of its structure query and text query searches.  DrugBank is available at http://www.drugbank.ca.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonqKjSWZe1ULVg90H21EOLACvtfcHk0lgm_VyoMxi4Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVSrs7Y%253D&md5=f8f8ca15f248b8c8b9c6d3bd0e85eec8</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm958%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DShrivastava%26aufirst%3DS.%26aulast%3DTzur%26aufirst%3DD.%26aulast%3DGautam%26aufirst%3DB.%26aulast%3DHassanali%26aufirst%3DM.%26atitle%3DDrugBank%253A%2520a%2520knowledgebase%2520for%2520drugs%252C%2520drug%2520actions%2520and%2520drug%2520targets%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3DD901%26epage%3DD906%26doi%3D10.1093%2Fnar%2Fgkm958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref123"><div class="reference"><strong class="refLabel"><a href="#ref123" class="rightTabRefNumLink">123</a></strong><div class="NLM_citation" id="rightTab-cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narjes, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span> <span> </span><span class="NLM_article-title">Emerging modes-of-action in drug discovery</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1550</span>– <span class="NLM_lpage">1568</span>, <span class="refDoi"> DOI: 10.1039/C9MD00263D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1039%2FC9MD00263D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31673315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1ersbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1550-1568&author=E.+Valeurauthor=F.+Narjesauthor=C.+Ottmannauthor=A.+T.+Plowright&title=Emerging+modes-of-action+in+drug+discovery&doi=10.1039%2FC9MD00263D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging modes-of-action in drug discovery</span></div><div class="casAuthors">Valeur, Eric; Narjes, Frank; Ottmann, Christian; Plowright, Alleyn T.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1550-1568</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">An increasing focus on complex biol. to cure diseases rather than merely treat symptoms has transformed how drug discovery can be approached.  Instead of activating or blocking protein function, a growing repertoire of drug modalities can be leveraged or engineered to hijack cellular processes, such as translational regulation or degrdn. mechanisms.  Drug hunters can therefore access a wider arsenal of modes-of-action to modulate biol. processes and this review summarises these emerging strategies by highlighting the most representative examples of these approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraPh2j99fLKrVg90H21EOLACvtfcHk0lgm_VyoMxi4Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1ersbvJ&md5=396fd707054d3ecdaf51220b7b3c6dcb</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1039%2FC9MD00263D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00263D%26sid%3Dliteratum%253Aachs%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DNarjes%26aufirst%3DF.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26atitle%3DEmerging%2520modes-of-action%2520in%2520drug%2520discovery%26jtitle%3DMedChemComm%26date%3D2019%26volume%3D10%26spage%3D1550%26epage%3D1568%26doi%3D10.1039%2FC9MD00263D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref124"><div class="reference"><strong class="refLabel"><a href="#ref124" class="rightTabRefNumLink">124</a></strong><div class="NLM_citation" id="rightTab-cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span> <span> </span><span class="NLM_article-title">Lead- and drug-like compounds: the rule-of-five revolution</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2004.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.ddtec.2004.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=24981612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2004&pages=337-341&author=C.+A.+Lipinski&title=Lead-+and+drug-like+compounds%3A+the+rule-of-five+revolution&doi=10.1016%2Fj.ddtec.2004.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">Lead- and drug-like compounds: the rule-of-five revolution</span></div><div class="casAuthors">Lipinski, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-341</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Citations in CAS SciFinder to the rule-of-five (RO5) publication will exceed 1000 by year-end 2004.  Trends in the RO5 literature explosion that can be discerned are the further definitions of drug-like.  This topic is explored in terms of drug-like physicochem. features, drug-like structural features, a comparison of drug-like and non-drug-like in drug discovery and a discussion of how drug-like features relate to clin. success.  Physicochem. features of CNS drugs and features related to CNS blood-brain transporter affinity are briefly reviewed.  Recent literature on features of non-oral drugs is reviewed and how features of lead-like compds. differ from those of drug-like compds. is discussed.  Most recently, partly driven by NIH roadmap initiatives, considerations have arisen as to what tool-like means in the search for chem. tools to probe biol. space.  All these topics frame the scope of this short review/perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnjiyP3CY7lLVg90H21EOLACvtfcHk0lgm_VyoMxi4Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSqsrg%253D&md5=952c29b47003b884116d790ae9e4721f</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2004.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2004.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DLead-%2520and%2520drug-like%2520compounds%253A%2520the%2520rule-of-five%2520revolution%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2004%26volume%3D1%26spage%3D337%26epage%3D341%26doi%3D10.1016%2Fj.ddtec.2004.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref125"><div class="reference"><strong class="refLabel"><a href="#ref125" class="rightTabRefNumLink">125</a></strong><div class="NLM_citation" id="rightTab-cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-induced protein degradation in drug discovery: Breaking the rules or just making new ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-induced+protein+degradation+in+drug+discovery%3A+Breaking+the+rules+or+just+making+new+ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0lg6a9_sdKnWxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-induced%2520protein%2520degradation%2520in%2520drug%2520discovery%253A%2520Breaking%2520the%2520rules%2520or%2520just%2520making%2520new%2520ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref126"><div class="reference"><strong class="refLabel"><a href="#ref126" class="rightTabRefNumLink">126</a></strong><div class="NLM_citation" id="rightTab-cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udugamasooriya, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">Potent and selective photo-inactivation of proteins with peptoid-ruthenium conjugates</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">258</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1038/nchembio.333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnchembio.333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=20228793" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFynuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=258-260&author=J.+Leeauthor=D.+G.+Udugamasooriyaauthor=H.-S.+Limauthor=T.+Kodadek&title=Potent+and+selective+photo-inactivation+of+proteins+with+peptoid-ruthenium+conjugates&doi=10.1038%2Fnchembio.333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective photo-inactivation of proteins with peptoid-ruthenium conjugates</span></div><div class="casAuthors">Lee, Jiyong; Udugamasooriya, D. Gomika; Lim, Hyun-Suk; Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">258-260</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Advances in high-throughput screening now enable the rapid discovery of bioactive small mols., but these primary hits almost always exhibit modest potency.  We report a strategy for the transformation of these hits into much more potent inhibitors without compd. optimization.  Appending a deriv. of Ru(II)(tris-bipyridyl)2+, an efficient photosensitizer of singlet oxygen prodn., to synthetic protein-binding compds. results in highly potent and specific target protein inactivation upon irradn. with visible light.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLrAtKe0wa1bVg90H21EOLACvtfcHk0lg6a9_sdKnWxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFynuro%253D&md5=b88e635815cbfa3fdd2ba94142352a71</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.333%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DUdugamasooriya%26aufirst%3DD.%2BG.%26aulast%3DLim%26aufirst%3DH.-S.%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DPotent%2520and%2520selective%2520photo-inactivation%2520of%2520proteins%2520with%2520peptoid-ruthenium%2520conjugates%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D258%26epage%3D260%26doi%3D10.1038%2Fnchembio.333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref127"><div class="reference"><strong class="refLabel"><a href="#ref127" class="rightTabRefNumLink">127</a></strong><div class="NLM_citation" id="rightTab-cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Small molecule targeted recruitment of a nuclease to RNA</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">6741</span>– <span class="NLM_lpage">6744</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b01233</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b01233" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpvFKktbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=6741-6744&author=M.+G.+Costalesauthor=Y.+Matsumotoauthor=S.+P.+Velagapudiauthor=M.+D.+Disney&title=Small+molecule+targeted+recruitment+of+a+nuclease+to+RNA&doi=10.1021%2Fjacs.8b01233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Targeted Recruitment of a Nuclease to RNA</span></div><div class="casAuthors">Costales, Matthew G.; Matsumoto, Yasumasa; Velagapudi, Sai Pradeep; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6741-6744</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The choreog. between RNA synthesis and degrdn. is a key determinant in biol.  Engineered systems such as CRISPR have been developed to rid a cell of RNAs.  Here, we show that a small mol. can recruit a nuclease to a specific transcript, triggering its destruction.  A small mol. that selectively binds the oncogenic microRNA(miR)-96 hairpin precursor was appended with a short 2'-5' poly(A) oligonucleotide.  The conjugate locally activated endogenous, latent RNase (RNase L), which selectively cleaved the miR-96 precursor in cancer cells in a catalytic fashion.  Silencing miR-96 de-repressed pro-apoptotic FOXO1 transcription factor, triggering apoptosis in breast cancer, but not healthy breast, cells.  These results demonstrate that small mols. can be programmed to selectively cleave RNA and has broad implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBE-TmSLM__LVg90H21EOLACvtfcHk0ljjvsx8uT2hJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpvFKktbs%253D&md5=7d4ccaad0cba3da84f924f244b8bcdbf</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b01233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b01233%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DMatsumoto%26aufirst%3DY.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSmall%2520molecule%2520targeted%2520recruitment%2520of%2520a%2520nuclease%2520to%2520RNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D6741%26epage%3D6744%26doi%3D10.1021%2Fjacs.8b01233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref128"><div class="reference"><strong class="refLabel"><a href="#ref128" class="rightTabRefNumLink">128</a></strong><div class="NLM_citation" id="rightTab-cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vauquelin, G.</span></span> <span> </span><span class="NLM_article-title">Effects of target binding kinetics on in vivo drug efficacy: k<sub>off</sub>, k<sub>on</sub> and rebinding</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">2319</span>– <span class="NLM_lpage">2334</span>, <span class="refDoi"> DOI: 10.1111/bph.13504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1111%2Fbph.13504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=27129075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSiurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2016&pages=2319-2334&author=G.+Vauquelin&title=Effects+of+target+binding+kinetics+on+in+vivo+drug+efficacy%3A+koff%2C+kon+and+rebinding&doi=10.1111%2Fbph.13504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of target binding kinetics on in vivo drug efficacy: koff, kon and rebinding</span></div><div class="casAuthors">Vauquelin, Georges</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2319-2334</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Background and Purpose : Optimal drug therapy often requires continuing high levels of target occupancy.  Besides the traditional pharmacokinetic contribution, target binding kinetics is increasingly considered to play an important role as well.  While most attention has been focused on the dissocn. rate of the complex, recent reports expressed doubt about the unreserved translatability of this pharmacodynamic property into clin. efficacy. 'Micro'-pharmacokinetic mechanisms like drug rebinding and partitioning into the cell membrane may constitute a potential fix.  Exptl. Approach : Simulations were based on solving differential equations.  Key Results : Based on a selected range of assocn. and dissocn. rate consts., kon and koff, and rebinding potencies of the drugs as variables, their effects on the temporal in vivo occupancy profile of their targets, after one or multiple repetitive dosings, have here been simulated.  Conclusions and Implications : Most strikingly, the simulations show that, when rebinding is also taken into account, increasing kon may produce closely the same outcome as decreasing koff when dosing is performed in accordance with the therapeutically most relevant const. [Lmax]/KD ratio paradigm.  Also, under certain conditions, rebinding may produce closely the same outcome as invoking slow diffusion of the drug between the plasma compartment and a target-contg. 'effect' compartment.  Although the present simulations should only be regarded as a 'proof of principle', these findings may help pharmacologists and medicinal chemists to devise ex vivo and in vitro binding kinetic assays that are more relevant and translatable to in vivo settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5-dbZNJetv7Vg90H21EOLACvtfcHk0ljjvsx8uT2hJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSiurvI&md5=5d1b13ef5786a0c661118ce9be503d50</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1111%2Fbph.13504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.13504%26sid%3Dliteratum%253Aachs%26aulast%3DVauquelin%26aufirst%3DG.%26atitle%3DEffects%2520of%2520target%2520binding%2520kinetics%2520on%2520in%2520vivo%2520drug%2520efficacy%253A%2520koff%252C%2520kon%2520and%2520rebinding%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2016%26volume%3D173%26spage%3D2319%26epage%3D2334%26doi%3D10.1111%2Fbph.13504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref129"><div class="reference"><strong class="refLabel"><a href="#ref129" class="rightTabRefNumLink">129</a></strong><div class="NLM_citation" id="rightTab-cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0ljtnmIJx1no0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref130"><div class="reference"><strong class="refLabel"><a href="#ref130" class="rightTabRefNumLink">130</a></strong><div class="NLM_citation" id="rightTab-cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">SPR-measured dissociation kinetics of PROTAC ternary complexes influence target degradation rate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=361-368&author=M.+J.+Royauthor=S.+Winklerauthor=S.+J.+Hughesauthor=C.+Whitworthauthor=M.+Galantauthor=W.+Farnabyauthor=K.+Rumpelauthor=A.+Ciulli&title=SPR-measured+dissociation+kinetics+of+PROTAC+ternary+complexes+influence+target+degradation+rate&doi=10.1021%2Facschembio.9b00092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">SPR-Measured Dissociation Kinetics of PROTAC Ternary Complexes Influence Target Degradation Rate</span></div><div class="casAuthors">Roy, Michael J.; Winkler, Sandra; Hughes, Scott J.; Whitworth, Claire; Galant, Michael; Farnaby, William; Rumpel, Klaus; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-368</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bifunctional degrader mols., known as proteolysis-targeting chimeras (PROTACs), function by recruiting a target to an E3 ligase, forming a target/PROTAC/ligase ternary complex.  Despite the importance of this key intermediate species, no detailed validation of a method to directly det. binding parameters for ternary complex kinetics has been reported, and it remains to be addressed whether tuning the kinetics of PROTAC ternary complexes may be an effective strategy to improve the efficiency of targeted protein degrdn.  Here, we develop an SPR-based assay to quantify the stability of PROTAC-induced ternary complexes by measuring for the first time the kinetics of their formation and dissocn. in vitro using purified proteins.  We benchmark our assay using four PROTACs that target the bromodomains (BDs) of bromodomain and extraterminal domain proteins Brd2, Brd3, and Brd4 to the von Hippel-Lindau E3 ligase (VHL).  We reveal marked differences in ternary complex off-rates for different PROTACs that exhibit either pos. or neg. cooperativity for ternary complex formation relative to binary binding.  The pos. cooperative degrader MZ1 forms comparatively stable and long-lived ternary complexes with either Brd4BD2 or Brd2BD2 and VHL.  Equivalent complexes with Brd3BD2 are destabilized due to a single amino acid difference (Glu/Gly swap) present in the bromodomain.  We observe that this difference in ternary complex dissociative half-life correlates to a greater initial rate of intracellular degrdn. of Brd2 and Brd4 relative to Brd3.  These findings establish a novel assay to measure the kinetics of PROTAC ternary complexes and elucidate the important kinetic parameters that drive effective target degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFhvuKYztcbVg90H21EOLACvtfcHk0ljtnmIJx1no0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D&md5=629d1acbd218b668a2696b727946962b</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00092%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWinkler%26aufirst%3DS.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSPR-measured%2520dissociation%2520kinetics%2520of%2520PROTAC%2520ternary%2520complexes%2520influence%2520target%2520degradation%2520rate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D361%26epage%3D368%26doi%3D10.1021%2Facschembio.9b00092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref131"><div class="reference"><strong class="refLabel"><a href="#ref131" class="rightTabRefNumLink">131</a></strong><div class="NLM_citation" id="rightTab-cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">The drug-target residence time model: a 10-year retrospective</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">95</span>, <span class="refDoi"> DOI: 10.1038/nrd.2015.18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1038%2Fnrd.2015.18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=26678621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVCrurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=87-95&author=R.+A.+Copeland&title=The+drug-target+residence+time+model%3A+a+10-year+retrospective&doi=10.1038%2Fnrd.2015.18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">The drug-target residence time model: a 10-year retrospective</span></div><div class="casAuthors">Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">87-95</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The drug-target residence time model was first introduced in 2006 and has been broadly adopted across the chem. biol., biotechnol. and pharmaceutical communities.  While traditional in vitro methods view drug-target interactions exclusively in terms of equil. affinity, the residence time model takes into account the conformational dynamics of target macromols. that affect drug binding and dissocn.  The key tenet of this model is that the lifetime (or residence time) of the binary drug-target complex, and not the binding affinity per se, dictates much of the in vivo pharmacol. activity.  Here, this model is revisited and key applications of it over the past 10 years are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwNrhp9U8kwLVg90H21EOLACvtfcHk0lhvxwBV924WgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVCrurnE&md5=23f4a6973d77804292ff0af6c81358c0</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2015.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2015.18%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DThe%2520drug-target%2520residence%2520time%2520model%253A%2520a%252010-year%2520retrospective%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D87%26epage%3D95%26doi%3D10.1038%2Fnrd.2015.18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref132"><div class="reference"><strong class="refLabel"><a href="#ref132" class="rightTabRefNumLink">132</a></strong><div class="NLM_citation" id="rightTab-cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeep
Velagapudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yildirim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Small-molecule targeted recruitment of a nuclease to cleave an oncogenic RNA in a mouse model of metastatic cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">2406</span>– <span class="NLM_lpage">2411</span>, <span class="refDoi"> DOI: 10.1073/pnas.1914286117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1914286117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31964809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVWisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=2406-2411&author=M.+G.+Costalesauthor=H.+Aikawaauthor=Y.+Liauthor=J.+L.+Childs-Disneyauthor=D.+Abeggauthor=D.+G.+Hochauthor=S.+Pradeep%0AVelagapudiauthor=Y.+Nakaiauthor=T.+Khanauthor=K.+W.+Wangauthor=I.+Yildirimauthor=A.+Adibekianauthor=E.+T.+Wangauthor=M.+D.+Disney&title=Small-molecule+targeted+recruitment+of+a+nuclease+to+cleave+an+oncogenic+RNA+in+a+mouse+model+of+metastatic+cancer&doi=10.1073%2Fpnas.1914286117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule targeted recruitment of a nuclease to cleave an oncogenic RNA in a mouse model of metastatic cancer</span></div><div class="casAuthors">Costales, Matthew G.; Aikawa, Haruo; Li, Yue; Childs-Disney, Jessica L.; Abegg, Daniel; Hoch, Dominic G.; Velagapudi, Sai Pradeep; Nakai, Yoshio; Khan, Tanya; Wang, Kye Won; Yildirim, Ilyas; Adibekian, Alexander; Wang, Eric T.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2406-2411</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the area of small mols. interacting with RNA advances, general routes to provide bioactive compds. are needed as ligands can bind RNA avidly to sites that will not affect function.  Small-mol. targeted RNA degrdn. will thus provide a general route to affect RNA biol.  A non-oligonucleotide-contg. compd. was designed from sequence to target the precursor to oncogenic microRNA-21 (pre-miR-21) for enzymic destruction with selectivity that can exceed that for protein-targeted medicines.  The compd. specifically binds the target and contains a heterocycle that recruits and activates a RNase to pre-miR-21 to substoichiometrically effect its cleavage and subsequently impede metastasis of breast cancer to lung in a mouse model.  Transcriptomic and proteomic analyses demonstrate that the compd. is potent and selective, specifically modulating oncogenic pathways.  Thus, small mols. can be designed from sequence to have all of the functional repertoire of oligonucleotides, including inducing enzymic degrdn., and to selectively and potently modulate RNA function in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolzufxJ0-LLLVg90H21EOLACvtfcHk0lhvxwBV924WgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVWisL0%253D&md5=450996be5aadaa9e13acb5c2cfb03c62</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1914286117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1914286117%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DAikawa%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DPradeep%2BVelagapudi%26aufirst%3DS.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DKhan%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DK.%2BW.%26aulast%3DYildirim%26aufirst%3DI.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DE.%2BT.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSmall-molecule%2520targeted%2520recruitment%2520of%2520a%2520nuclease%2520to%2520cleave%2520an%2520oncogenic%2520RNA%2520in%2520a%2520mouse%2520model%2520of%2520metastatic%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D2406%26epage%3D2411%26doi%3D10.1073%2Fpnas.1914286117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref133"><div class="reference"><strong class="refLabel"><a href="#ref133" class="rightTabRefNumLink">133</a></strong><div class="NLM_citation" id="rightTab-cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M.</span></span> <span> </span><span class="NLM_article-title">Target 2035: probing the human proteome</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2111</span>– <span class="NLM_lpage">2115</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2019.06.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1016%2Fj.drudis.2019.06.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31278990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlahtrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=2111-2115&issue=11&author=A.+J.+Carterauthor=O.+Kraemerauthor=M.+Zwickauthor=A.+Mueller-Fahrnowauthor=C.+H.+Arrowsmithauthor=A.+M.+Edwards&title=Target+2035%3A+probing+the+human+proteome&doi=10.1016%2Fj.drudis.2019.06.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Target 2035: probing the human proteome</span></div><div class="casAuthors">Carter, Adrian J.; Kraemer, Oliver; Zwick, Matthias; Mueller-Fahrnow, Anke; Arrowsmith, Cheryl H.; Edwards, Aled M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2111-2115</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Biomedical scientists tend to focus on only a small fraction of the proteins encoded by the human genome despite overwhelming genetic evidence that many understudied proteins are important for human disease.  One of the best ways to interrogate the function of a protein and to det. its relevance as a drug target is by using a pharmacol. modulator, such as a chem. probe or an antibody.  If these tools were available for most human proteins, it should be possible to translate the tremendous advances in genomics into a greater understanding of human health and disease, and catalyze the creation of innovative new medicines.  Target 2035 is a global federation for developing and applying new technologies with the goal of creating chemogenomic libraries, chem. probes, and/or functional antibodies for the entire proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSY5b9ZXh0ALVg90H21EOLACvtfcHk0lh-atvxgMfisQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlahtrnN&md5=212e832238c8860439b343cbeef17b74</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2019.06.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2019.06.020%26sid%3Dliteratum%253Aachs%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DKraemer%26aufirst%3DO.%26aulast%3DZwick%26aufirst%3DM.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26atitle%3DTarget%25202035%253A%2520probing%2520the%2520human%2520proteome%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26issue%3D11%26spage%3D2111%26epage%3D2115%26doi%3D10.1016%2Fj.drudis.2019.06.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref134"><div class="reference"><strong class="refLabel"><a href="#ref134" class="rightTabRefNumLink">134</a></strong><div class="NLM_citation" id="rightTab-cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kehe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulesa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackerman, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakku, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuehn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gore, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blainey, P. C.</span></span> <span> </span><span class="NLM_article-title">Massively parallel screening of synthetic microbial communities</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">12804</span>– <span class="NLM_lpage">12809</span>, <span class="refDoi"> DOI: 10.1073/pnas.1900102116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1073%2Fpnas.1900102116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=31186361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1ersrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=12804-12809&author=J.+Keheauthor=A.+Kulesaauthor=A.+Ortizauthor=C.+M.+Ackermanauthor=S.+G.+Thakkuauthor=D.+Sellersauthor=S.+Kuehnauthor=J.+Goreauthor=J.+Friedmanauthor=P.+C.+Blainey&title=Massively+parallel+screening+of+synthetic+microbial+communities&doi=10.1073%2Fpnas.1900102116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Massively parallel screening of synthetic microbial communities</span></div><div class="casAuthors">Kehe, Jared; Kulesa, Anthony; Ortiz, Anthony; Ackerman, Cheri M.; Thakku, Sri Gowtham; Sellers, Daniel; Kuehn, Seppe; Gore, Jeff; Friedman, Jonathan; Blainey, Paul C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">12804-12809</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Microbial communities have numerous potential applications in biotechnol., agriculture, and medicine.  Nevertheless, the limited accuracy with which we can predict interspecies interactions and environmental dependencies hinders efforts to rationally engineer beneficial consortia.  Empirical screening is a complementary approach wherein synthetic communities are combinatorially constructed and assayed in high throughput.  However, assembling many combinations of microbes is logistically complex and difficult to achieve on a timescale commensurate with microbial growth.  Here, we introduce the kChip, a droplets-based platform that performs rapid, massively parallel, bottom-up construction and screening of synthetic microbial communities.  We first show that the kChip enables phenotypic characterization of microbes across environmental conditions.  Next, in a screen of a$$Hat$$ 1/4100,000 multispecies communities comprising up to 19 soil isolates, we identified sets that promote the growth of the model plant symbiont Herbaspirillum frisingense in a manner robust to carbon source variation and the presence of addnl. species.  Broadly, kChip screening can identify multispecies consortia possessing any optically assayable function, including facilitation of biocontrol agents, suppression of pathogens, degrdn. of recalcitrant substrates, and robustness of these functions to perturbation, with many applications across basic and applied microbial ecol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNU77NEWb3TrVg90H21EOLACvtfcHk0lh-atvxgMfisQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1ersrnF&md5=38e41c9eed2b1246c90ee5c81f1fc47e</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1900102116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1900102116%26sid%3Dliteratum%253Aachs%26aulast%3DKehe%26aufirst%3DJ.%26aulast%3DKulesa%26aufirst%3DA.%26aulast%3DOrtiz%26aufirst%3DA.%26aulast%3DAckerman%26aufirst%3DC.%2BM.%26aulast%3DThakku%26aufirst%3DS.%2BG.%26aulast%3DSellers%26aufirst%3DD.%26aulast%3DKuehn%26aufirst%3DS.%26aulast%3DGore%26aufirst%3DJ.%26aulast%3DFriedman%26aufirst%3DJ.%26aulast%3DBlainey%26aufirst%3DP.%2BC.%26atitle%3DMassively%2520parallel%2520screening%2520of%2520synthetic%2520microbial%2520communities%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D12804%26epage%3D12809%26doi%3D10.1073%2Fpnas.1900102116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref135"><div class="reference"><strong class="refLabel"><a href="#ref135" class="rightTabRefNumLink">135</a></strong><div class="NLM_citation" id="rightTab-cit135"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">DNA-Encoded Chemical Libraries: A selection system based on endowing organic compounds with amplifiable information</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">479</span>– <span class="NLM_lpage">502</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-062917-012550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=10.1146%2Fannurev-biochem-062917-012550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=29328784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVCrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2018&pages=479-502&author=D.+Neriauthor=R.+A.+Lerner&title=DNA-Encoded+Chemical+Libraries%3A+A+selection+system+based+on+endowing+organic+compounds+with+amplifiable+information&doi=10.1146%2Fannurev-biochem-062917-012550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Chemical Libraries: A Selection System Based on Endowing Organic Compounds with Amplifiable Information</span></div><div class="casAuthors">Neri, Dario; Lerner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">479-502</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The discovery of org. ligands that bind specifically to proteins is a central problem in chem., biol., and the biomedical sciences.  The encoding of individual org. mols. with distinctive DNA tags, serving as amplifiable identification bar codes, allows the construction and screening of combinatorial libraries of unprecedented size, thus facilitating the discovery of ligands to many different protein targets.  Fundamentally, one links powers of genetics and chem. synthesis.  After the initial description of DNA-encoded chem. libraries in 1992, several exptl. embodiments of the technol. have been reduced to practice.  This review provides a historical account of important milestones in the development of DNA-encoded chem. libraries, a survey of relevant ongoing research activities, and a glimpse into the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAAjUib3HfEbVg90H21EOLACvtfcHk0lgf4Uqp5cAhUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVCrsg%253D%253D&md5=559fefad70a7b9949319ac8ad8066004</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-062917-012550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-062917-012550%26sid%3Dliteratum%253Aachs%26aulast%3DNeri%26aufirst%3DD.%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DDNA-Encoded%2520Chemical%2520Libraries%253A%2520A%2520selection%2520system%2520based%2520on%2520endowing%2520organic%2520compounds%2520with%2520amplifiable%2520information%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2018%26volume%3D87%26spage%3D479%26epage%3D502%26doi%3D10.1146%2Fannurev-biochem-062917-012550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KAQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KAQ','PDB','4KAQ'); return false;">PDB: 4KAQ</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01927&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01927%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-17%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01927" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677a96068f612362","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
