
<html lang="en"     class="pb-page"  data-request-id="3ab41ce9-3575-460d-b89f-c4fce33cb0e5"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2021.64.issue-3;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.0c01897"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of a Napabucasin PROTAC as an Effective Degrader of the E3 Ligase ZFP91" /></meta><meta name="dc.Creator" content="Maha  Hanafi" /></meta><meta name="dc.Creator" content="Xinde  Chen" /></meta><meta name="dc.Creator" content="Nouri  Neamati" /></meta><meta name="dc.Description" content="Napabucasin, undergoing multiple clinical trials, was reported to inhibit the signal transducer and transcription factor 3 (STAT3). To better elucidate its mechanism of action, we designed a napabu..." /></meta><meta name="Description" content="Napabucasin, undergoing multiple clinical trials, was reported to inhibit the signal transducer and transcription factor 3 (STAT3). To better elucidate its mechanism of action, we designed a napabu..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 28, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01897" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01897" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01897" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01897" /></link>
        
    
    

<title>Discovery of a Napabucasin PROTAC as an Effective Degrader of the E3 Ligase ZFP91 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01897" /></meta><meta property="og:title" content="Discovery of a Napabucasin PROTAC as an Effective Degrader of the E3 Ligase ZFP91" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0026.jpeg" /></meta><meta property="og:description" content="Napabucasin, undergoing multiple clinical trials, was reported to inhibit the signal transducer and transcription factor 3 (STAT3). To better elucidate its mechanism of action, we designed a napabucasin-based proteolysis targeting chimera (PROTAC), XD2-149 that resulted in inhibition of STAT3 signaling in pancreatic cancer cell lines without inducing proteasome-dependent degradation of STAT3. Proteomics analysis of XD2-149 revealed the downregulation of the E3 ubiquitin-protein ligase ZFP91. XD2-149 degrades ZFP91 with DC50 values in the nanomolar range. The cytotoxicity of XD2-149 was significantly, but not fully, reduced with ZFP91 knockdown providing evidence for its multi-targeted mechanism of action. The NQO1 inhibitor, dicoumarol, rescued the cytotoxicity of XD2-149 but not ZFP91 degradation, suggesting that the NQO1-induced cell death is independent of ZFP91. ZFP91 plays a role in tumorigenesis and is involved in multiple oncogenic pathways including NF-κB and HIF-1α." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01897"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01897">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-14/achre4.2021.54.issue-14/20210720/achre4.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-29/aamick.2021.13.issue-29/20210728/aamick.2021.13.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-14/acncdm.2021.12.issue-14/20210721/acncdm.2021.12.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-29/acsodf.2021.6.issue-29/20210727/acsodf.2021.6.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-29/ascecg.2021.9.issue-29/20210726/ascecg.2021.9.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-29/ancham.2021.93.issue-29/20210727/ancham.2021.93.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-29/bichaw.2021.60.issue-29/20210727/bichaw.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-27/cgeabj.2021.99.issue-27/20210726-01/cgeabj.2021.99.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-14/esthag.2021.55.issue-14/20210720/esthag.2021.55.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-29/iecred.2021.60.issue-29/20210728/iecred.2021.60.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-14/inocaj.2021.60.issue-14/20210719/inocaj.2021.60.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-29/jafcau.2021.69.issue-29/20210728/jafcau.2021.69.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-29/jacsat.2021.143.issue-29/20210728/jacsat.2021.143.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-29/langd5.2021.37.issue-29/20210727/langd5.2021.37.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01897&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01897&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01897&amp;href=/doi/10.1021/acs.jmedchem.0c01897" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 1626-1648</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01809" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01900" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a Napabucasin PROTAC as an Effective Degrader of the E3 Ligase ZFP91</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Maha Hanafi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maha Hanafi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy and the Rogel Cancer Center, North Campus Research Complex, 1600 Huron Parkway, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry, Faculty of Pharmacy, Cairo University, Cairo 11562, Egypt</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maha++Hanafi">Maha Hanafi</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5389-4656" title="Orcid link">http://orcid.org/0000-0002-5389-4656</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xinde Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xinde Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy and the Rogel Cancer Center, North Campus Research Complex, 1600 Huron Parkway, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xinde++Chen">Xinde Chen</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Nouri Neamati</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nouri Neamati</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy and the Rogel Cancer Center, North Campus Research Complex, 1600 Huron Parkway, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#dbb5bebab6baafb29baeb6b2b8b3f5bebfae"><span class="__cf_email__" data-cfemail="0e606b6f636f7a674e7b63676d66206b6a7b">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nouri++Neamati">Nouri Neamati</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3291-7131" title="Orcid link">http://orcid.org/0000-0003-3291-7131</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01897&amp;href=/doi/10.1021%2Facs.jmedchem.0c01897" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 1626–1648</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 28, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>3 November 2020</li><li><span class="item_label"><b>Published</b> online</span>28 January 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 February 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01897" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01897</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1626%26pageCount%3D23%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMaha%2BHanafi%252C%2BXinde%2BChen%252C%2BNouri%2BNeamati%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D3%26contentID%3Dacs.jmedchem.0c01897%26title%3DDiscovery%2Bof%2Ba%2BNapabucasin%2BPROTAC%2Bas%2Ban%2BEffective%2BDegrader%2Bof%2Bthe%2BE3%2BLigase%2BZFP91%26numPages%3D23%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1648%26publicationDate%3DFebruary%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01897"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3061</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01897" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a Napabucasin PROTAC as an Effective Degrader of the E3 Ligase ZFP91&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Maha&quot;,&quot;last_name&quot;:&quot;Hanafi&quot;},{&quot;first_name&quot;:&quot;Xinde&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Nouri&quot;,&quot;last_name&quot;:&quot;Neamati&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;28&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;1626-1648&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01897&quot;},&quot;abstract&quot;:&quot;Napabucasin, undergoing multiple clinical trials, was reported to inhibit the signal transducer and transcription factor 3 (STAT3). To better elucidate its mechanism of action, we designed a napabucasin-based proteolysis targeting chimera (PROTAC), XD2-149 that resulted in inhibition of STAT3 signaling in pancreatic cancer cell lines without inducing proteasome-dependent degradation of STAT3. Proteomics analysis of XD2-149 revealed the downregulation of the E3 ubiquitin-protein ligase ZFP91. XD2-149 degrades ZFP91 with DC50 values in the nanomolar range. The cytotoxicity of XD2-149 was significantly, but not fully, reduced with ZFP91 knockdown providing evidence for its multi-targeted mechanism of action. The NQO1 inhibitor, dicoumarol, rescued the cytotoxicity of XD2-149 but not ZFP91 degradation, suggesting that the NQO1-induced cell death is independent of ZFP91. ZFP91 plays a role in tumorigenesis and is involved in multiple oncogenic pathways including NF-κB and HIF-1α.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01897&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01897" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01897&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01897" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01897&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01897" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01897&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01897&amp;href=/doi/10.1021/acs.jmedchem.0c01897" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01897" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01897" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01897%26sid%3Dliteratum%253Aachs%26pmid%3D33506674%26genre%3Darticle%26aulast%3DHanafi%26date%3D2021%26atitle%3DDiscovery%2Bof%2Ba%2BNapabucasin%2BPROTAC%2Bas%2Ban%2BEffective%2BDegrader%2Bof%2Bthe%2BE3%2BLigase%2BZFP91%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D3%26spage%3D1626%26epage%3D1648%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=290808" title="Toxicity">Toxicity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/jmcmar.2021.64.issue-3/20210211/jmcmar.2021.64.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0026.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Napabucasin, undergoing multiple clinical trials, was reported to inhibit the signal transducer and transcription factor 3 (STAT3). To better elucidate its mechanism of action, we designed a napabucasin-based proteolysis targeting chimera (PROTAC), <b>XD2-149</b> that resulted in inhibition of STAT3 signaling in pancreatic cancer cell lines without inducing proteasome-dependent degradation of STAT3. Proteomics analysis of <b>XD2-149</b> revealed the downregulation of the E3 ubiquitin-protein ligase ZFP91. <b>XD2-149</b> degrades ZFP91 with DC<sub>50</sub> values in the nanomolar range. The cytotoxicity of <b>XD2-149</b> was significantly, but not fully, reduced with ZFP91 knockdown providing evidence for its multi-targeted mechanism of action. The NQO1 inhibitor, dicoumarol, rescued the cytotoxicity of <b>XD2-149</b> but not ZFP91 degradation, suggesting that the NQO1-induced cell death is independent of ZFP91. ZFP91 plays a role in tumorigenesis and is involved in multiple oncogenic pathways including NF-κB and HIF-1α.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37074" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37074" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Naphthoquinone 2-acetylbenzo[<i>f</i>][1]benzofuran-4,9-dione (NPQ) was first discovered in 1982 as a plant-derived compound with anticancer activity.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In 1998, NPQ was reported to have antipsoriatic, antibacterial, and antifungal activities.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Despite the promising preliminary data on NPQ, there was no interest in its development for almost a decade until 2009 when it was re-discovered by Boston Biomedical, Inc. and given the name napabucasin, also known as BBI608. Napabucasin was dubbed as the first-in-class cancer stemness inhibitor through inhibition of the signal transducer and transcription factor 3 (STAT3) signaling pathway.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Significant efforts were directed toward the development of napabucasin that resulted in clinical trials in several cancers as a single agent and in combination with chemotherapy (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Table S1</a>). Napabucasin was also formulated in polymeric vesicles that resulted in a decrease in the pancreatic cancer cell viability by 70%.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Promising data from clinical trials accelerated the designation of napabucasin to orphan drug status for gastric/gastroesophageal and pancreatic cancer in 2016 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Discovery timeline of napabucasin highlighting select milestones. (B) Reported mechanisms of action for napabucasin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Pancreatic cancer is a devastating disease and is one of the leading causes of cancer-related deaths worldwide; therefore, novel therapies are highly sought after.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a><i>In vivo</i> studies have shown that napabucasin is capable of blocking cancer metastasis and its relapse in pancreatic and colon cancers through inhibition of cancer stemness.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Combination with nab-paclitaxel and gemcitabine in a phase I/II clinical trial showed that napabucasin is well tolerated and displays activity against metastatic pancreatic adenocarcinoma (mPDAC).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> However, a phase III study investigating the efficacy of napabucasin when given with the standard-of-care chemotherapy in patients with mPDAC was discontinued due to futility.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a></div><div class="NLM_p">Several analogs of napabucasin were synthesized to improve its efficacy and/or physicochemical properties. Introducing a (piperidinyl)ethylamino substitution enhanced the activity 10-fold and resulted in improved solubility.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Another compound (LD-19) that possesses a methyl piperazine group displayed enhanced <i>in vitro</i> cytotoxicity in liver cancer cells and improved water solubility.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> However, none of the optimized analogs have been advanced to in-depth pre-clinical studies. While napabucasin is facing a lack of success in the clinic, there is still considerable interest in understanding its mechanism of action since it is reported as a STAT3 inhibitor.</div><div class="NLM_p">STAT3 is a transcription factor that is implicated in the majority of human cancers and is associated with poor prognosis.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> The constitutive activation of STAT3 in cancer cells is highly advantageous for proliferative and metastatic phenotypes. This master regulatory role allows for selective cancer cell killing, making it an attractive target for cancer treatment.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Over the past 30 years, significant efforts were directed toward understanding the biological function of STAT3 and developing an effective inhibitor capable of blocking STAT3 activity. However, very few compounds including napabucasin made it to clinical trials. Most of these studies displayed insufficient efficacy and/or intolerable side effects.</div><div class="NLM_p">Although napabucasin has been reported as a STAT3 inhibitor, its exact mechanism of action is not clearly understood. Napabucasin has been claimed to directly bind STAT3 by preventing its proteolytic degradation.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Furthermore, a series of napabucasin analogs were reported to bind to STAT3 using surface plasmon resonance (SPR) analysis with a <i>K</i><sub>D</sub> of 110 nM for the most potent analog.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> It is claimed that napabucasin inhibits STAT3 by binding to its SH2 domain and preventing its dimerization and subsequent activation but there is no crystallographic evidence to support that claim.<a onclick="showRef(event, 'ref11 ref12 ref16'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref16">(11,12,16)</a> Napabucasin is also reported to act on multiple other oncogenic pathways including WNT/β-catenin pathway that is constitutively activated in tumor tissues.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Napabucasin is a substrate for NAD(P)H:quinone oxidoreductase 1 (NQO1), and its activity is dependent on NQO1 levels.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> This enzyme catalyzes the reduction of a quinone to hydroquinone resulting in the generation of reactive oxygen species (ROS), DNA alkylation, and subsequent cancer cell death.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> This suggests that part of napabucasin’s mechanism of action could be a result of ROS induction and/or DNA alkylation.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Recently, we showed that napabucasin induces ROS, has similar transcriptomic signature as H<sub>2</sub>O<sub>2</sub>, and induces cancer cell death via necroptosis.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Napabucasin was also reported to act through inhibition of protein synthesis via regulating the eukaryotic initiation factor 4E (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><div class="NLM_p">In an attempt to better understand napabucasin’s mechanism of action and its connection to STAT3, we designed a series of novel napabucasin-derived proteolysis targeting chimeras (PROTACs) and assessed their cytotoxicity in pancreatic cancer cell lines. PROTACs are heterobifunctional molecules that simultaneously bind an E3 ligase and a target protein, driving target protein ubiquitination by the E3 ligase complex leading to recognition and degradation by the 26S proteasome.<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> The first-in-class STAT3 degrader, SD-36, was reported earlier this year by applying a cereblon (CRBN)-based PROTAC targeting the SH2 domain of STAT3.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> SD-36 degraded STAT3 with nanomolar DC<sub>50</sub> values in leukemia and lymphoma cell lines and elicited complete tumor regression within 2 weeks.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> This study validates the use of PROTACs in effective and selective targeting of STAT3. As napabucasin is also reported to target the SH2 domain of STAT3, we hypothesized that napabucasin-based PROTACs would be a novel and effective strategy to degrade STAT3. Herein, we describe the synthesis and evaluation of a series of napabucasin-PROTACs with cellular potency in pancreatic cancer cell lines. Proteomics and mechanistic studies did not reveal proteasome-dependent degradation of STAT3. Instead, we observed a significant degradation of ZFP91 with the napabucasin-PROTAC, <b>3-6d</b> (<b>XD2-149</b>) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Our study provides further insight into the mechanism of action of napabucasin and for the first time, linking napabucasin to the oncogenic protein ZFP91.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures of napabucasin, the designed PROTACs, and their respective negative controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13614" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13614" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design of PROTACs Using Napabucasin as the Targeting Warhead</h3><div class="NLM_p">In order to develop an effective PROTAC that can bind to and degrade the potential target of napabucasin, we designed a series of compounds applying three major optimization approaches. The design followed a systematic strategy that included the optimization of the linker length/composition, the position of the linker attachment to napabucasin, and the choice of the E3 ligase ligand. It is critical to adjust for the distance between the two partner proteins to allow for efficient binding and degradation. CRBN and Von Hippel Lindau (VHL) E3 ligases have been successfully employed in the design of the PROTACs for degrading different proteins.<a onclick="showRef(event, 'ref27 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29 ref30 ref31">(27−31)</a> CRBN E3 ligase ligands are composed of the low-molecular-weight immunomodulatory drugs (IMiDs) including pomalidomide, thalidomide, and lenalidomide that make them ideal for the design of PROTACs with good physicochemical properties.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Therefore, we focused our studies on the design of CRBN-based PROTACs with the exploration of the effect of the VHL E3 ligase ligand. Pomalidomide has a higher cellular stability compared to lenalidomide and thalidomide,<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> so we initially designed pomalidomide-based compounds for studying the choice of the optimal linker. The synthesized compounds were evaluated for their cytotoxicity and protein degradation in the two pancreatic cancer cell lines, BxPC-3 and MIA PaCa-2. We used BxPC-3 to guide our structure–activity relationship studies (SAR) due to the improved activity of the compounds in immunoblotting experiments, in addition to its higher expression and activation of STAT3 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Summary for PROTAC optimization including linker length and composition, point of linker attachment to napabucasin, and the choice of the E3 ligase ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to obtain the optimum PROTAC, it is important to consider the stability of the ternary complex in which the linker plays an important role. With pomalidomide as the CRBN handle and napabucasin as the targeting warhead, we designed a series of PROTACs applying various linker lengths and compositions (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>). We synthesized compounds with alkyl and PEG linkers of different lengths (2–20 atoms) and evaluated their cytotoxicity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Compound <b>2-5a</b> with the 2-atom spacer displayed low micromolar cytotoxicity, while compounds with linkers ranging between 3 and 11 atoms were 2-fold less potent than <b>2-5a</b>. Increasing the linker length >11 atoms in <b>2-5c</b> and <b>2-5j</b> resulted in decreased cytotoxicity. Our results show that the optimal linker length for the cytotoxicity of compounds is ≤11 atoms and that there is no significant difference between the hydrophobic and hydrophilic linkers.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>2</b>-<b>5a–j</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOAc, HOAc, 100 °C; (b) DIPEA, DMF, 80 °C; (c) TFA, DCM, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>3-6a–d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DIPEA, DMF, 80 °C; (b) TFA, DCM, rt; (c) NaBH(OAc)<sub>3</sub>, DCE, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Cytotoxicity of CRBN-Based PROTACs in Pancreatic Cancer Cell Lines</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0022.gif" alt="" id="gr21" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0023.gif" alt="" id="gr22" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Napabucasin.</p></div></div><div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> <b>2-5g</b> (<b>XD97</b>) Reduces STAT3 Expression in a Proteasome-Independent Manner</h3><div class="NLM_p">We tested a selection of the compounds in a panel of cell lines outside of pancreatic cancer to determine if there is a specific cell line in which the compounds are selective (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Tables S2 and S3</a>). <b>XD97</b> with an IC<sub>50</sub> of ∼0.3 μM in OVCAR-3 cells was the most cytotoxic. To determine whether the cytotoxicity of these compounds is STAT3-dependent, we used a pair of OVCAR-3 wild type (WT) and STAT3 knockout (KO) cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Table S3</a>).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Overall, the compounds were more potent in the WT cell line, among which <b>XD97</b> displayed significant selectivity over the STAT3 KO cell line (>6-fold). These data suggest that STAT3 is involved in the mechanism of cell death. Since the cytotoxicity was STAT3-dependent, we tested whether the napabucasin PROTAC <b>XD97</b> degraded STAT3. <b>XD97</b> at different concentrations and times caused near complete depletion of STAT3 after 12 h, while a decrease in p-STAT3 level was observed after 6 h (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Figure S1B</a>).</div><div class="NLM_p">To investigate if the degradation of STAT3 was proteasome-dependent, we employed three approaches: treatment in the presence of a proteasome inhibitor, competition with the E3 ligase ligand, and synthesis of a negative control incapable of binding CRBN. Upon treatment with the proteasome inhibitor MG132, we observed no blockade of <b>XD97</b>-induced decrease in STAT3 expression, concluding that this decrease is not dependent on the proteasome (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Figure S1C</a>). In fact, MG132 enhanced the activity of <b>XD97</b> resulting in a more prominent reduction in STAT3 protein levels. It is worth noting that proteasome inhibitors like bortezomib have been reported to synergize with STAT3 inhibitors to promote cell death.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> However, we did not observe synergy between napabucasin and proteasome inhibitors in cell proliferation assays (data not shown). Similarly, competition experiments in the presence of pomalidomide did not antagonize the effect of <b>XD97</b>, indicating that it is not a substrate of CRBN (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Figure S1D</a>). We synthesized <b>10-5</b> (<b>XD171</b>) as a negative control by replacing the glutarimide ring in pomalidomide by a δ-lactam ring that abrogates its binding to CRBN (<a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Figure S1A</a>).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a><b>XD171</b> displayed comparable activity to <b>XD97</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Figure S1E</a>). Among the STAT family members, STAT1 has a high structural similarity to STAT3.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Surprisingly, we observed a decrease in STAT1 levels that is even more prominent than that observed with STAT3 at the same concentrations of <b>XD97</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Figure S1B</a>). Taken together, the mechanism of action of <b>XD97</b> is independent of the proteasome and the observed effects are possibly due to inhibitory activity or changes on the transcriptional level. We also conclude that the activity of <b>XD97</b> is not specific to STAT3.</div><div class="NLM_p">We further tested <b>XD97</b> for its effect on STAT3 expression in the pancreatic cancer cell line BxPC-3 to determine if it possessed a different mechanism of action and possibly act as a degrader. Similar to OVCAR-3, <b>XD97</b> reduced the STAT3 level after 12 h, but the effect in BxPC-3 was observed at a lower concentration (DC<sub>50</sub> = 7.8 μM) despite of the greater cytotoxicity of <b>XD97</b> in OVCAR-3 (IC<sub>50</sub> of ∼0.2 and 4.3 μM in OVCAR-3 and BxPC-3, respectively) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Figure S2A,B</a>). This encouraged us to test <b>XD97</b> in combination with the proteasome inhibitor bortezomib. Unfortunately, bortezomib did not block the activity of <b>XD97</b>, thereby proving that its STAT3 inhibitory activity is independent of the proteasome-mediated degradation (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Figure S2C</a>). Interestingly, we observed reduced STAT3 expression with napabucasin (DC<sub>50</sub> = 2.1 μM) that was not observed in OVCAR-3 (Figure S2B). Moreover, napabucasin and <b>XD97</b> both reduced NQO1 levels (Figure S2D). It is possible that napabucasin has multiple targets or acts through interfering with the transcription/translation of target proteins.</div><div class="NLM_p last">Although PROTACs have the advantage of selectively degrading the target protein, recruitment of other neo-substrates can be a challenge. The surface of CRBN can change upon binding of IMiDs like pomalidomide and allow recruitment of other proteins such as the translation termination factor GSPT1, which has been previously reported as an off-target substrate for CRBN-targeted PROTACs.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> This change in the surface structure leads to creation of hotspots that cause the PROTAC molecule to act as a “molecular glue” and bind non-specific proteins.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> The protein level of GSPT1 was not affected by <b>XD97</b> or napabucasin treatment (Figure S2D). This provides evidence that napabucasin does not non-specifically cause degradation of proteins and that it is mechanism-based.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Optimization of Napabucasin-Based PROTACs</h3><div class="NLM_p">Moving forward with the linker optimization, we synthesized compounds with cyclized linkers using piperidine. The piperidine linkers displayed improved activity over the alkyl/PEG linkers (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Piperidine groups have been previously incorporated into PROTACs to improve water solubility.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a><b>3-6c</b> and <b>3-6d</b> (<b>XD2-149</b>) showed comparable potency with IC<sub>50</sub> < 1 μM in BxPC-3. Compound <b>3-6a</b> with a piperazine ring resulted in significant loss of activity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Our data shows that piperidine-containing linkers display the best cytotoxicity and are superior to napabucasin, suggesting that these compounds might act as effective degraders. Therefore, <b>XD2-149</b> was selected as a representative PROTAC for further mechanistic studies.</div><div class="NLM_p">The small molecule <b>1-6</b> is a previously reported analog of napabucasin that displayed greater cytotoxicity (>10-fold) and was used to investigate the effect of the position of linker attachment (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Scheme S1 and Table S4</a>).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Attaching the linker to the position 7 of dihydronaptho[2,3-<i>b</i>]furan position resulted in <b>13-11</b> did not result in improved cytotoxicity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Scheme S2 and Table S5</a>). Changing the position of the linker attachment to the E3 ligase ligand from position 4 of the isoindoline-1,3-dione ring to position 5 (<b>12-5</b>) reduced the cytotoxicity as compared to <b>XD97</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Scheme S3 and Table S5</a>).</div><div class="NLM_p">To examine the CRBN ligand portion, another set of compounds was designed using thalidomide as the CRBN E3 ligase ligand employing various linkers (<a class="ref internalNav" href="#sch3" aria-label="Schemes 3">Schemes 3</a>–<named-content content-type="anchor" rid="sch4" type="simple"></named-content><a class="ref internalNav" href="#sch5" aria-label="5">5</a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Amide-containing linkers displayed a significant decrease in activity with IC<sub>50</sub> > 10 μM except for <b>6-6b</b> with a linker that extends to >13 atoms contributing to its enhanced activity. Compound <b>5-5b</b> with a 9-atom PEG linker had an IC<sub>50</sub> ≈ 8 μM, while <b>5-5a</b> with a 4-atom alkyl linker was almost 2-fold more potent. Two other compounds with alkyl and PEG linkers conjugated to lenalidomide showed moderate cytotoxicity (IC<sub>50</sub> range 6–14 μM) (<a class="ref internalNav" href="#sch6" aria-label="Schemes 6">Schemes 6</a> and 7 and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>4-5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOAc, HOAc, 120 °C; (b) Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, TEA, DMF, 80 °C; (c) TFA, DCM, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0015.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>5-5a</b> and <b>5-5b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOAc, HOAc, 110 °C; (b) KHCO<sub>3</sub>, KI, DMF, 70 °C; (c) TFA, DCM, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><figure id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0016.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>6-6a–c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, DMF, rt; (b) TFA, rt; (c) HATU, DIPEA, DMF, rt; (d) TFA, 50 °C.</p></p></figure><figure id="sch6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0017.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compound <b>7-4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DIPEA, DMF, 80 °C; (b) TFA, DCM, rt; (c) HATU, DIPEA, DMF, rt.</p></p></figure><div class="NLM_p">Our synthetic efforts extended to testing compounds with VHL as the E3 ligase ligand. Interestingly, the VHL-based compounds (<a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>) were inactive with IC<sub>50</sub> > 30 μM, suggesting that VHL is not optimal for the design of napabucasin-based PROTACs (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Cytotoxicity of VHL-Based PROTACs in Pancreatic Cancer Cell Lines</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0024.gif" alt="" id="gr23" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0025.gif" alt="" id="gr24" /></img><div></div></div><div class="NLM_p last">In conclusion, CRBN E3 ligases were more effective in the design of PROTACs compared to VHL. Thalidomide- and lenalidomide-based compounds were less cytotoxic than pomalidomide-based conjugates, suggesting that pomalidomide provides a more stable ternary complex with napabucasin and CRBN.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> <b>3-6d</b> (<b>XD2-149</b>) Is Cytotoxic in Pancreatic Cancer Cell Lines</h3><div class="NLM_p">Previous studies on napabucasin have proven its activity against pancreatic cancer for which it was approved as an orphan drug. BxPC-3 cells have high protein levels of STAT3 and pSTAT3.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> We tested our compounds for their ability to inhibit cell growth in MIA PaCa-2 and BxPC-3 cell lines using MTT assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Table S5</a>). Compound <b>XD2-149</b> was among the most active compounds in both cell lines with an IC<sub>50</sub> of ∼1 μM. We synthesized <b>11-7</b> (<b>XD2-162</b>) as a negative control by replacing the NH of the glutarimide ring in pomalidomide with an <i>N</i>-methyl group (<a class="ref internalNav" href="#sch10" aria-label="Scheme 10">Scheme 10</a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). The N-methylated pomalidomide lacks the key H-bond donor involved in an interaction with His378 of CRBN, in addition to the steric hindrance created by the methyl in the binding pocket.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a><b>XD2-149</b> is significantly more potent than <b>XD2-162</b> in the MTT assay, suggesting that the cell growth inhibition involves target degradation (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B). The colony formation assay (CFA), a cell survival assay that assesses the ability of the cells to survive and form colonies, was also used to test the cytotoxicity of the compounds. Lead compound inhibits colony formation in different pancreatic cancer cells lines. <b>XD2-149</b> displayed a greater inhibitory activity than napabucasin in the BxPC-3 cell line (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C,D).</div><figure id="fig4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. <b>XD2-149</b> inhibits cell proliferation and colony formation in pancreatic cancer cell lines. (A) Design of <b>XD2-162</b>, the negative control for the PROTAC <b>XD2-149</b>. IC<sub>50</sub> values correspond to the cytotoxicity of the compounds in the MTT assay. (B) Dose–response curves of the cytotoxicity of <b>XD2-149</b> and <b>XD2-162</b> in the MTT assay after 72 h. (C) Colony formation assay for napabucasin (NP), <b>XD2-149</b>, and <b>XD2-162</b> in pancreatic cancer cell lines. Cells were treated for 7–10 days. (D) Quantification for the colony formation assay was done by dissolving the colonies using Sorensen’s buffer and reading the absorbance at 570 nm. All data presented was generated from three independent experiments and is presented as mean ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> <b>XD2-149</b> Inhibits IL6-Dependent STAT3 Signaling</h3><div class="NLM_p">To determine whether our compounds have an effect on the transcriptional activity of STAT3, we screened the compounds at 2 and 10 μM in the STAT3 luciferase assay using the HEK-293 cell line. The luciferase reporter assay is commonly used in the discovery and evaluation of STAT3 inhibitors.<a onclick="showRef(event, 'ref26 ref46'); return false;" href="javascript:void(0);" class="ref ref26 ref46">(26,46)</a> Following IL6-induced STAT3 activation, the cells were treated with the compounds for 24 h. The majority of the compound resulted in a reduced signal, indicating inhibition of STAT3-driven transcription (Figure S3). All compounds were more potent in the luciferase assay than a 24-h MTT assay, indicating the signal loss was not due to cytotoxicity and cell death. Among the tested compounds, <b>XD2-149</b> was the most efficient in the inhibition of STAT3-driven transcription with an IC<sub>50</sub> of ∼1 μM, which is almost 7-fold greater than its activity in the MTT assay (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><figure id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. <b>XD2-149</b> inhibits STAT3-driven gene transcription. (A) Diagram representing the principal of the STAT3 luciferase reporter assay. Transfected HEK-293 cell lines that express Renilla luciferase reporter under the transcriptional control of STAT3 were used. (B) IC<sub>50</sub> values of the tested compounds in the luciferase assay after treatment for 24 h compared to their IC<sub>50</sub> in the MTT assay at the same treatment time point. For the luciferase assay, cells were pretreated with 100 ng/mL IL6 for 24 h followed by treatment with the compounds for 24 h. The luciferin substrate was then added and the luminescence signal was measured. (C) IC<sub>50</sub> values of napabucasin (NP) and <b>XD2-149</b> in MTT and luciferase (LUC) assays after a 24 h treatment. Data was generated from three independent experiments and is presented as mean ± SD. ** denotes <i>p</i> ≤ 0.01, and *** denotes <i>p</i> ≤ 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> <b>XD2-149</b> Reduces STAT3 and Relevant Proteins’ Expression Independent of the Proteasome</h3><div class="NLM_p">To evaluate the ability of <b>XD2-149</b> to degrade STAT3, we treated MIA PaCa-2 and BxPC-3 cells with the compound at a range of concentrations up to 5 μM (∼5X IC<sub>50</sub>). In BxPC-3, after a 16 h treatment, we observed a decrease in the STAT3 protein expression at 5 μM, while a significant decrease in the pSTAT3 level was achieved at 1 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). In contrast, <b>XD2-162</b> did not have any effect on STAT3 expression but resulted in a decrease in pSTAT3 expression at 5 μM. This offset in concentration-dependent protein downregulation correlates well with the differences in MTT IC<sub>50</sub> values. Interestingly, napabucasin alone was able to reduce the total STAT3 protein levels in addition to decreasing pSTAT3. The fact that the mechanism of action of the parent inhibitor involves reduction of STAT3 protein expression makes it challenging to study napabucasin PROTACs. The small-molecule analog of napabucasin <b>1-6</b> was also tested for its structural similarity to our compounds. Compound <b>1-6</b> did not have any effect on STAT3 expression at 0.5 μM (5X IC<sub>50</sub>) with a reduced pSTAT3 expression at the same concentration. As previously discussed, napabucasin is reported as a substrate of NQO1. Therefore, we tested the effect of <b>XD2-149</b> on NQO1 as another possible target for degradation. We observed similar results to STAT3, which suggests that the activity of the compounds is not selective to STAT3 and the possibility of NQO1 degradation by the PROTAC given that napabucasin is a substrate of the enzyme. A reduction in STAT3 activation can result from increased ROS levels generated from NQO1-induced bioactivation of napabucasin.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><figure id="fig6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <b>XD2-149</b> reduces the expression of STAT3, pSTAT3, and NQO1 in a proteasome-independent manner. (A) <b>XD2-149</b> reduces the expression of STAT3, pSTAT3, and NQO1 proteins, while <b>XD2-162</b> showed an effect only on pSTAT3. The small molecules <b>1-6</b> and napabucasin (NP) displayed similar effects to <b>XD2-149</b>. BxPC-3 and MIA PaCa-2 cells were treated with <b>XD2-149</b>, <b>XD2-162</b>, <b>1-6</b>, or NP at the indicated concentrations for 16 h. (B) The effect of <b>XD2-149</b> on STAT3/pSTAT3 expression is independent of the proteasome. BxPC-3 cells were pretreated with 10 μM MG132 for 2 h then with <b>XD2-149</b>, <b>XD2-162</b> or NP for 12 h. (C) <b>XD2-149</b> is possibly interfering with the protein synthesis of STAT3 and NQO1. BxPC-3 cells were pretreated with cycloheximide (CX) for 1 h followed by NP or <b>XD2-149</b> for 16 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We repeated the experiment in MIA PaCa-2 cells; however, we observed less prominent effects on the proteins’ expression than in BxPC-3 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). Notably, <b>XD2-149</b> was less effective in inhibiting the phosphorylation of STAT3 in MIA PaCa-2 with no effect on the NQO1 level. It is worth noting that, in MIA PaCa-2 cells, we were able to test napabucasin and <b>XD2-149</b> only up to 1 μM due to their high cytotoxicity and narrow therapeutic window. Napabucasin did not have a significant effect on the STAT3 protein expression but completely depleted the pSTAT3 levels at 1 μM.</div><div class="NLM_p">Among the tested cell lines, BxPC-3 displayed the best activity for <b>XD2-149</b> in terms of cell growth inhibition and reduced STAT3 expression and activation. Therefore, further experiments were conducted in BxPC-3. To determine whether the reduction of STAT3 expression is due to proteasome-dependent degradation, we treated the cells with <b>XD2-149</b> in the presence of the proteasome inhibitor MG132. Proteasome inhibition did not block the <b>XD2-149</b>-induced decrease in STAT3 protein (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). In contrast, we observed an enhanced reduction in NQO1 expression with <b>XD2-149</b> treatment in the presence of MG132.</div><div class="NLM_p">To determine if these effects occur at the translational level, we tested the effect of <b>XD2-149</b> on STAT3 and NQO1 in the presence of the protein synthesis inhibitor, cycloheximide. We observed an enhanced cycloheximide-mediated protein synthesis inhibition of STAT3 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). Similar results were previously reported on napabucasin in osteosarcoma cells where the study demonstrated the inhibitory effect of napabucasin on protein synthesis as a result of STAT3 inhibition.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> On the other hand, the reduced NQO1 expression caused by <b>XD2-149</b> was blocked in the presence of cycloheximide, suggesting that the compound-induced downregulation of NQO1 is dependent on the <i>de novo</i> protein synthesis. Moreover, we sought to determine whether napabucasin engages its downregulated proteins by performing cellular thermal shift assay (CETSA). We tested the effect of napabucasin on the thermal stability of STAT3, NQO1, and an irrelevant protein (PDI) and observed a slight shift in the melting curve with all tested proteins except for the control protein GAPDH (Figure S4).</div><div class="NLM_p last">Collectively, even though <b>XD2-149</b> displayed pronounced cytotoxicity in various cell lines compared to the negative control, it proved to not degrade the STAT3 protein through the proteasome machinery. It interferes with the protein synthesis of NQO1 resulting in reduced NQO1 levels, which was restored in the presence of a protein synthesis inhibitor.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> <b>XD2-149</b> Is a <i>Bona Fide</i> Degrader of ZFP91</h3><div class="NLM_p">To further study the effect of <b>XD2-149</b> at the protein level and determine proteins selectively altered upon treatment, we performed proteomics analysis in BxPC-3. We evaluated proteome changes after 16 h treatment with either <b>XD2-149</b>, napabucasin, or <b>XD2-162</b>. The negative control was included in the study for comparison and elimination of nonspecific proteins or proteins whose expression is affected as a result of the inhibitor only or other transcriptional changes. A total of 136 proteins were dysregulated (>1.5 FC, <i>p</i> ≤ 0.05) by <b>XD2-149</b>, out of which 84 proteins were downregulated (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). The top 25 upregulated and downregulated proteins for each treatment are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Tables S6–S11</a>. Comparative analysis with napabucasin and <b>XD2-162</b> showed 40 proteins that were exclusively downregulated by <b>XD2-149</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). Interestingly, STAT3 was among the proteins exclusive for <b>XD2-149</b> with a log<sub>2</sub>FC of −0.69. This downregulation was consistent with our western blot results.</div><figure id="fig7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Transcriptomic profile of <b>XD2-149</b> shows the exclusive downregulation of ZFP91 compared to napabucasin and the negative control in BxPC-3 cells with the enriched hallmark gene sets presented. BxPC-3 cells were treated with <b>XD2-149</b> (2 μM), napabucasin (2 μM), and <b>XD2-162</b> (4 μM) for 16 h. Cell lysates were analyzed by multiplexed proteomics analysis (<b>XD2-149</b>: <i>n</i> = 3, napabucasin and <b>XD2-162</b>: <i>n</i> = 2). (A) Volcano plot for <b>XD2-149</b> proteins. The significant upregulated and downregulated proteins are colored in pink (fold change (FC) > 1.5 and <i>p</i>-value ≤0.05). (B) Venn diagram showing the overlap of common proteins between the three treatments and the exclusive proteins for each. Significant proteins >1.5 fold change and <i>p</i>-value ≤0.05 were used to generate the diagram. (C) Upregulated and downregulated enriched hallmark gene sets from GSEA analysis of <b>XD2-149</b>. (D) Select enrichment plots for the top upregulated and downregulated enriched gene sets from GSEA analysis of <b>XD2-149</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Downregulated Proteins Exclusive to <b>XD2-149</b> Treatment in BxPC-3 Cells</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">gene symbol</th><th class="colsep0 rowsep0" align="center">name</th><th class="colsep0 rowsep0" align="center" char=".">Log<sub>2</sub>FC<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."><i>p</i>-value<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">TMPRSS4</td><td class="colsep0 rowsep0" align="left">transmembrane protease serine 4</td><td class="colsep0 rowsep0" align="char" char=".">–1.27</td><td class="colsep0 rowsep0" align="char" char=".">0.028</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">PLK1</td><td class="colsep0 rowsep0" align="left">serine/threonine-protein kinase PLK1</td><td class="colsep0 rowsep0" align="char" char=".">–1.25</td><td class="colsep0 rowsep0" align="char" char=".">0.020</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">TP53BP2</td><td class="colsep0 rowsep0" align="left">apoptosis-stimulating of p53 protein 2</td><td class="colsep0 rowsep0" align="char" char=".">–1.02</td><td class="colsep0 rowsep0" align="char" char=".">0.005</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">DYNLT1</td><td class="colsep0 rowsep0" align="left">dynein light chain Tctex-type 1</td><td class="colsep0 rowsep0" align="char" char=".">–0.97</td><td class="colsep0 rowsep0" align="char" char=".">0.014</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">DTX3L</td><td class="colsep0 rowsep0" align="left">E3 ubiquitin-protein ligase DTX3L</td><td class="colsep0 rowsep0" align="char" char=".">–0.96</td><td class="colsep0 rowsep0" align="char" char=".">0.042</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">MAGED1</td><td class="colsep0 rowsep0" align="left">melanoma-associated antigen D1</td><td class="colsep0 rowsep0" align="char" char=".">–0.9</td><td class="colsep0 rowsep0" align="char" char=".">0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">STAT2</td><td class="colsep0 rowsep0" align="left">signal transducer and activator of transcription 2</td><td class="colsep0 rowsep0" align="char" char=".">–0.9</td><td class="colsep0 rowsep0" align="char" char=".">0.010</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">CBR1; SETD4</td><td class="colsep0 rowsep0" align="left">carbonyl reductase [NADPH] 1</td><td class="colsep0 rowsep0" align="char" char=".">–0.86</td><td class="colsep0 rowsep0" align="char" char="."><0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">SPINT1</td><td class="colsep0 rowsep0" align="left">Kunitz-type protease inhibitor 1</td><td class="colsep0 rowsep0" align="char" char=".">–0.85</td><td class="colsep0 rowsep0" align="char" char=".">0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">CCNB1</td><td class="colsep0 rowsep0" align="left">G2/mitotic-specific cyclin-B1</td><td class="colsep0 rowsep0" align="char" char=".">–0.84</td><td class="colsep0 rowsep0" align="char" char=".">0.031</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">EDF1</td><td class="colsep0 rowsep0" align="left">endothelial differentiation-related factor 1</td><td class="colsep0 rowsep0" align="char" char=".">–0.82</td><td class="colsep0 rowsep0" align="char" char="."><0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">CST3</td><td class="colsep0 rowsep0" align="left">cystatin C</td><td class="colsep0 rowsep0" align="char" char=".">–0.81</td><td class="colsep0 rowsep0" align="char" char=".">0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">NABP2</td><td class="colsep0 rowsep0" align="left">SOSS complex subunit B1</td><td class="colsep0 rowsep0" align="char" char=".">–0.78</td><td class="colsep0 rowsep0" align="char" char=".">0.008</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">WDFY4</td><td class="colsep0 rowsep0" align="left">WD repeat- and FYVE domain-containing protein 4</td><td class="colsep0 rowsep0" align="char" char=".">–0.78</td><td class="colsep0 rowsep0" align="char" char=".">0.009</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">CSDE1</td><td class="colsep0 rowsep0" align="left">cold shock domain-containing protein E1</td><td class="colsep0 rowsep0" align="char" char=".">–0.75</td><td class="colsep0 rowsep0" align="char" char="."><0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">SHPRH</td><td class="colsep0 rowsep0" align="left">E3 ubiquitin-protein ligase SHPRH</td><td class="colsep0 rowsep0" align="char" char=".">–0.74</td><td class="colsep0 rowsep0" align="char" char=".">0.007</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">FAM3C; WNT16</td><td class="colsep0 rowsep0" align="left">protein FAM3C</td><td class="colsep0 rowsep0" align="char" char=".">–0.73</td><td class="colsep0 rowsep0" align="char" char="."><0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">DDR1</td><td class="colsep0 rowsep0" align="left">epithelial discoidin domain-containing receptor 1</td><td class="colsep0 rowsep0" align="char" char=".">–0.73</td><td class="colsep0 rowsep0" align="char" char="."><0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">DSP</td><td class="colsep0 rowsep0" align="left">desmoplakin</td><td class="colsep0 rowsep0" align="char" char=".">–0.70</td><td class="colsep0 rowsep0" align="char" char=".">0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">GPR56; ADGRG1</td><td class="colsep0 rowsep0" align="left">adhesion G-protein coupled receptor G1</td><td class="colsep0 rowsep0" align="char" char=".">–0.69</td><td class="colsep0 rowsep0" align="char" char=".">0.004</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">STAT3</td><td class="colsep0 rowsep0" align="left">signal transducer and activator of transcription 3</td><td class="colsep0 rowsep0" align="char" char=".">–0.69</td><td class="colsep0 rowsep0" align="char" char=".">0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">LTV1</td><td class="colsep0 rowsep0" align="left">protein LTV1 homolog</td><td class="colsep0 rowsep0" align="char" char=".">–0.69</td><td class="colsep0 rowsep0" align="char" char=".">0.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">23</td><td class="colsep0 rowsep0" align="left">AGRN</td><td class="colsep0 rowsep0" align="left">Agrin</td><td class="colsep0 rowsep0" align="char" char=".">–0.68</td><td class="colsep0 rowsep0" align="char" char=".">0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">SDF4</td><td class="colsep0 rowsep0" align="left">45 kDa calcium-binding protein</td><td class="colsep0 rowsep0" align="char" char=".">–0.67</td><td class="colsep0 rowsep0" align="char" char=".">0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">LAMB1</td><td class="colsep0 rowsep0" align="left">laminin subunit beta-1</td><td class="colsep0 rowsep0" align="char" char=".">–0.67</td><td class="colsep0 rowsep0" align="char" char="."><0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left">UBE2T</td><td class="colsep0 rowsep0" align="left">ubiquitin-conjugating enzyme E2 T</td><td class="colsep0 rowsep0" align="char" char=".">–0.66</td><td class="colsep0 rowsep0" align="char" char=".">0.029</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left">PHLDB1</td><td class="colsep0 rowsep0" align="left">pleckstrin homology-like domain family B member 1</td><td class="colsep0 rowsep0" align="char" char=".">–0.64</td><td class="colsep0 rowsep0" align="char" char=".">0.009</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">CHST14</td><td class="colsep0 rowsep0" align="left">carbohydrate sulfotransferase 14</td><td class="colsep0 rowsep0" align="char" char=".">–0.64</td><td class="colsep0 rowsep0" align="char" char=".">0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">29</td><td class="colsep0 rowsep0" align="left">PLOD2</td><td class="colsep0 rowsep0" align="left">procollagen-lysine,2-oxoglutarate 5-dioxygenase 2</td><td class="colsep0 rowsep0" align="char" char=".">–0.63</td><td class="colsep0 rowsep0" align="char" char=".">0.019</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left">PVRL1; NECTIN1</td><td class="colsep0 rowsep0" align="left">nectin-1</td><td class="colsep0 rowsep0" align="char" char=".">–0.63</td><td class="colsep0 rowsep0" align="char" char=".">0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">31</td><td class="colsep0 rowsep0" align="left">PRRC2B</td><td class="colsep0 rowsep0" align="left">protein PRRC2B</td><td class="colsep0 rowsep0" align="char" char=".">–0.63</td><td class="colsep0 rowsep0" align="char" char=".">0.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">MGMT</td><td class="colsep0 rowsep0" align="left">methylated-DNA--protein-cysteine methyltransferase</td><td class="colsep0 rowsep0" align="char" char=".">–0.63</td><td class="colsep0 rowsep0" align="char" char=".">0.004</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">33</td><td class="colsep0 rowsep0" align="left">NSMCE2</td><td class="colsep0 rowsep0" align="left">E3 SUMO-protein ligase NSE2</td><td class="colsep0 rowsep0" align="char" char=".">–0.63</td><td class="colsep0 rowsep0" align="char" char=".">0.012</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">ZFP91</td><td class="colsep0 rowsep0" align="left">E3 ubiquitin-protein ligase ZFP91</td><td class="colsep0 rowsep0" align="char" char=".">–0.62</td><td class="colsep0 rowsep0" align="char" char=".">0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">35</td><td class="colsep0 rowsep0" align="left">SERF2</td><td class="colsep0 rowsep0" align="left">small EDRK-rich factor 2</td><td class="colsep0 rowsep0" align="char" char=".">–0.61</td><td class="colsep0 rowsep0" align="char" char=".">0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">POLD4</td><td class="colsep0 rowsep0" align="left">DNA polymerase delta subunit 4</td><td class="colsep0 rowsep0" align="char" char=".">–0.61</td><td class="colsep0 rowsep0" align="char" char=".">0.017</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">37</td><td class="colsep0 rowsep0" align="left">PRDM16</td><td class="colsep0 rowsep0" align="left">histone-lysine <i>N</i>-methyltransferase PRDM16</td><td class="colsep0 rowsep0" align="char" char=".">–0.6</td><td class="colsep0 rowsep0" align="char" char=".">0.033</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">38</td><td class="colsep0 rowsep0" align="left">CIAPIN1</td><td class="colsep0 rowsep0" align="left">anamorsin</td><td class="colsep0 rowsep0" align="char" char=".">–0.59</td><td class="colsep0 rowsep0" align="char" char="."><0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">39</td><td class="colsep0 rowsep0" align="left">CCNDBP1</td><td class="colsep0 rowsep0" align="left">cyclin-D1-binding protein 1</td><td class="colsep0 rowsep0" align="char" char=".">–0.59</td><td class="colsep0 rowsep0" align="char" char=".">0.005</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">USP8</td><td class="colsep0 rowsep0" align="left">ubiquitin carboxyl-terminal hydrolase 8</td><td class="colsep0 rowsep0" align="char" char=".">–0.59</td><td class="colsep0 rowsep0" align="char" char=".">0.008</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">FC > 1.5.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last"><i>p</i>-value ≤ 0.05.</p></div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Upregulated Proteins Exclusive to <b>XD2-149</b> Treatment in BxPC-3 Cells</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">gene symbol</th><th class="colsep0 rowsep0" align="center">name</th><th class="colsep0 rowsep0" align="center" char=".">Log<sub>2</sub>FC<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."><i>p</i>-value<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">JUN</td><td class="colsep0 rowsep0" align="left">transcription factor AP-1</td><td class="colsep0 rowsep0" align="char" char=".">1.58</td><td class="colsep0 rowsep0" align="char" char="."><0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">ARX</td><td class="colsep0 rowsep0" align="left">homeobox protein ARX</td><td class="colsep0 rowsep0" align="char" char=".">1.54</td><td class="colsep0 rowsep0" align="char" char=".">0.043</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">DNAJB9</td><td class="colsep0 rowsep0" align="left">DnaJ homolog subfamily B member 9</td><td class="colsep0 rowsep0" align="char" char=".">1.35</td><td class="colsep0 rowsep0" align="char" char="."><0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">IL1RL1</td><td class="colsep0 rowsep0" align="left">interleukin-1 receptor-like 1</td><td class="colsep0 rowsep0" align="char" char=".">1.24</td><td class="colsep0 rowsep0" align="char" char=".">0.010</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">SYNC</td><td class="colsep0 rowsep0" align="left">syncoilin</td><td class="colsep0 rowsep0" align="char" char=".">1.04</td><td class="colsep0 rowsep0" align="char" char=".">0.045</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">RND3</td><td class="colsep0 rowsep0" align="left">Rho-related GTP-binding protein RhoE</td><td class="colsep0 rowsep0" align="char" char=".">0.98</td><td class="colsep0 rowsep0" align="char" char=".">0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">HMGCR</td><td class="colsep0 rowsep0" align="left">3-hydroxy-3-methylglutaryl-coenzyme A reductase</td><td class="colsep0 rowsep0" align="char" char=".">0.83</td><td class="colsep0 rowsep0" align="char" char=".">0.009</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">ZNF430</td><td class="colsep0 rowsep0" align="left">zinc finger protein 430</td><td class="colsep0 rowsep0" align="char" char=".">0.77</td><td class="colsep0 rowsep0" align="char" char=".">0.039</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">ECM1</td><td class="colsep0 rowsep0" align="left">extracellular matrix protein 1</td><td class="colsep0 rowsep0" align="char" char=".">0.76</td><td class="colsep0 rowsep0" align="char" char=".">0.026</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">RHOB</td><td class="colsep0 rowsep0" align="left">Rho-related GTP-binding protein RhoB</td><td class="colsep0 rowsep0" align="char" char=".">0.73</td><td class="colsep0 rowsep0" align="char" char=".">0.035</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">GDF15</td><td class="colsep0 rowsep0" align="left">growth/differentiation factor 15</td><td class="colsep0 rowsep0" align="char" char=".">0.69</td><td class="colsep0 rowsep0" align="char" char=".">0.010</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">ABCB6</td><td class="colsep0 rowsep0" align="left">ATP-binding cassette sub-family B member 6, mitochondrial</td><td class="colsep0 rowsep0" align="char" char=".">0.68</td><td class="colsep0 rowsep0" align="char" char=".">0.025</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">AFF1</td><td class="colsep0 rowsep0" align="left">AF4/FMR2 family member 1</td><td class="colsep0 rowsep0" align="char" char=".">0.67</td><td class="colsep0 rowsep0" align="char" char=".">0.049</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">MPV17</td><td class="colsep0 rowsep0" align="left">protein Mpv17</td><td class="colsep0 rowsep0" align="char" char=".">0.66</td><td class="colsep0 rowsep0" align="char" char=".">0.049</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">EPHB3</td><td class="colsep0 rowsep0" align="left">ephrin type-B receptor 3</td><td class="colsep0 rowsep0" align="char" char=".">0.65</td><td class="colsep0 rowsep0" align="char" char=".">0.024</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">UGCG</td><td class="colsep0 rowsep0" align="left">ceramide glucosyltransferase</td><td class="colsep0 rowsep0" align="char" char=".">0.62</td><td class="colsep0 rowsep0" align="char" char=".">0.032</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">AKR1C2</td><td class="colsep0 rowsep0" align="left">aldo-keto reductase family 1 member C2</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td><td class="colsep0 rowsep0" align="char" char=".">0.004</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">BACH1</td><td class="colsep0 rowsep0" align="left">transcription regulator protein BACH1</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td><td class="colsep0 rowsep0" align="char" char=".">0.004</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">FC > 1.5.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last"><i>p</i>-value ≤ 0.05.</p></div></div></div><div class="NLM_p">Among the downregulated proteins is the E3 ubiquitin-protein ligase ZFP91 that has been reported as a neo-substrate that can be recruited by CRBN-based PROTACs independent of the target protein.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Previous studies show that the nonspecific recruitment of ZFP91 was observed with CRBN ligase but not VHL possibly due to differences in the protein surface that allows CRBN to create high-affinity binding interfaces.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> A pSILAC mass spectrometric study was performed for target identification of small molecules that can induce protein degradation.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The study identified ZFP91 as a substrate of lenalidomide-induced CRBN exploring the possibility of IMiD-induced repurposing of CRBN to target new proteins for degradation. The structural basis and mechanism for lenalidomide-induced CRBN-mediated degradation of proteins were reported to prove that protein recruitment is dependent on the presence of the IMiD that remodels the CRBN surface for binding of these proteins.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> This intrigued us to test for the degradation of ZFP91, especially that <b>XD2-149</b> did not result in proteasome-dependent degradation of STAT3, the intended target protein. We performed western blot experiments treating BxPC-3 cells with <b>XD2-149</b> and napabucasin for 16 h. Interestingly, ZFP91 protein was significantly downregulated by <b>XD2-149</b> at 1 μM while we observed no effect with napabucasin up to 3 μM (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A). To determine whether this effect occurs at an earlier time point, we tested the effect of <b>XD2-149</b> after 4, 16, and 24 h. A significant decrease in the expression of ZFP91 was observed at a comparable DC<sub>50</sub> for the 16 and 24 h treatments, while the earlier time point showed no effect (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B). Next, we repeated the experiment in the presence of the proteasome inhibitor carfilzomib to determine if the downregulation is a result of proteasome-dependent degradation. Gratifyingly, the results showed a restoration in the ZFP91 protein levels due to the blockade of the <b>XD2-149</b>-induced degradation (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C). We obtained similar results in MIA PaCa-2 cells (Figure S5). This confirms that our PROTAC results in proteasome-dependent degradation of ZFP91 and provides a possible explanation for the lack of degradation effect on the STAT3 protein. Combining napabucasin with <b>XD2-149</b> did not affect the activity of the PROTAC, suggesting that it does not compete with napabucasin for its binding site (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D).</div><figure id="fig8" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. <b>XD2-149</b> degrades ZFP91 protein in a proteasome-dependent manner. (A) <b>XD2-149</b> reduces ZFP91 expression, while napabucasin (NP) has no effect. BxPC-3 cells were treated with NP or <b>XD2-149</b> at the indicated concentrations for 16 h. (B) <b>XD2-149</b> degrades ZFP91 at 16 and 24 h with no effect at 4 h. Dose–response curves of ZFP91 degradation by <b>XD2-149</b> are presented for the 16 and 24 h time points. (C) The effect of <b>XD2-149</b> on ZFP91 is proteasome-dependent. BxPC-3 cells were pretreated with 1 μM proteasome inhibitor carfilzomib for 1 h followed by <b>XD2-149</b> or <b>XD2-162</b> treatment for 16 h. (D) NP does not abolish the effect of <b>XD2-149</b>. BxPC-3 cells were treated with <b>XD2-149</b> in presence or absence of NP for 16 h. (E) Pomalidomide (POM) induces ZFP91 degradation after 16 h. (F) <b>XD2-149</b> have no effect on the expression of GSPT. BxPC-3 cells were treated with NP or <b>XD2-149</b> for 16 h. Immunoblots were quantified using ImageJ, and the DC<sub>50</sub> values were calculated using GraphPad Prism 8. * denotes <i>p</i> ≤ 0.05, ** denotes <i>p</i> ≤ 0.01, and *** denotes <i>p</i> ≤ 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Pomalidomide has been reported to induce the degradation of CRBN neo-substrates like ZFP91.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> We sought to test its effect on ZFP91 in BxPC-3 cells to better understand the activity of our PROTAC. Pomalidomide displayed a DC<sub>50</sub> of 0.42 μM, which is 5-fold less potent than the PROTAC (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>E). This result shows that although the pomalidomide moiety contributes to ZFP91-induced degradation, the PROTAC might be involved in a more stable ternary complex that results in more effective degradation. Moreover, pomalidomide does not show any cytotoxicity in the MTT assay up to 30 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Table S4</a>). We also tested for other targets like GSPT1 that might be recruited for proteasome-dependent degradation. However, <b>XD2-149</b> did not degrade GSPT, excluding the possibility that our PROTAC is non-selectively targeting CRBN neo-substrates (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>F). Other proteins that have also been reported as CRBN neo-substrates including IKZF1 and IKZF3,<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49,50)</a> CK1α,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> and ZBTB16<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> were not found among the proteins exclusively downregulated with <b>XD2-149</b> treatment.</div><div class="NLM_p">ZFP91 is involved in various biological processes, and among its oncogenic properties is the regulation of NF-κB signaling pathway and HIF-1α promoting tumorigenesis.<a onclick="showRef(event, 'ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref52 ref53">(52,53)</a> It is also reported to play a role in ubiquitination and destabilization of FOXA1 to promote cancer cell survival.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> In-depth understanding of the exact role of ZFP91 is lacking. However, the published data so far suggests that its inhibition or degradation could be beneficial as an anticancer strategy.</div><div class="NLM_p">Consistent with the napabucasin reported mechanism of action, proteomics results for <b>XD2-149</b> show differential expression of proteins involved in cancer stemness and STAT3 signaling including downregulation of WNT16, DDR1, DSP, CST3, PLK1, DTX3L, and CCNB1 (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) and upregulation of JUN, IL1RL1, RND3, ECM1, RHOB, EPHB3, UCUG, and AKR1C2 (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). As previously discussed, <b>XD2-149</b> inhibits STAT3-driven transcription in the luciferase assay, indicating that it acts through inhibition of the IL6-STAT3 pathway. Altered proteins in common between napabucasin, <b>XD2-149</b>, and <b>XD2-162</b> include 10 downregulated proteins and 14 upregulated proteins presented in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Tables S12 and S13</a>.</div><div class="NLM_p last">We performed gene set enrichment analysis (GSEA) to gain insight into the signaling pathways involved in the mechanism of action of <b>XD2-149</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Tables S23–S32</a>). Enriched gene sets for napabucasin and <b>XD2-162</b> are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">Tables S14–S22 and S33–42</a>, respectively. GSEA analysis for <b>XD2-149</b> displayed nine upregulated and five downregulated enriched Hallmark gene sets (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C,D). Comparison with napabucasin and <b>XD2-162</b> revealed enrichment of eight common upregulated Hallmark gene sets including TNFA signaling via NF-κB, hypoxia, IL2/STAT5 signaling, KRAS signaling, unfolded protein response, inflammatory response, apoptosis, and coagulation (Figure S6). These pathways are commonly altered with STAT3 inhibition, suggesting that napabucasin and its PROTAC work through inhibition of STAT3 signaling. The altered pathways are also possibly cumulative as a result of multiple actions. We mentioned earlier that ZFP91 is involved in the regulation of NF-κB and HIF-1α pathways and our GSEA data shows upregulation of both pathways, suggesting that these effects can be secondary to degradation of ZFP91. On the other hand, there are seven downregulated gene sets in common including E2F targets, DNA replication, mismatch repair, and telomere maintenance via semiconservative replication (Figure S6). STAT3 plays a role in regulation of cell cycle through activation of E2F targets including Cyclin D1, Cyclin B1, and Cdc2.<a onclick="showRef(event, 'ref15 ref55'); return false;" href="javascript:void(0);" class="ref ref15 ref55">(15,55)</a> This explains the observed downregulation in E2F targets that can result from STAT3 downregulation caused by napabucasin and <b>XD2-149</b> treatment. STAT3 is known to play an important role in cell proliferation through mechanisms that include suppression of ATR and modulation of DNA repair.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a><b>XD2-149</b> was also found to downregulate the hallmark interferon-alpha response gene set (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D). IFN-α engages with its receptor and triggers the activation of STAT3. Studies have shown that STAT3 itself plays a role in regulating the IFN-mediated response.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> This data is in accord with the reported experiments that STAT3 signaling is a major target pathway in the mechanism of action of napabucasin.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> <b>XD2-149</b> Cytotoxicity Is Dependent on ZFP91</h3><div class="NLM_p">Until now, there are no reported ZFP91-targeted anticancer therapies. A previous study on the kinase inhibitor foretinib demonstrated that GSPT1 and ZFP91 proteins are neo-substrates for the CRBN-based PROTAC but did not investigate their contribution to its anticancer activity.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> ZFP91 has been validated as an oncogenic target that regulates different pathways involved in tumorigenesis, and its targeting will produce novel class of anticancer agents.<a onclick="showRef(event, 'ref53 ref54 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref53 ref54 ref58 ref59">(53,54,58,59)</a> ZFP91 is highly expressed in pancreatic cancer cells and is associated with lower overall survival (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A).<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Knockdown studies of ZFP91 in pancreatic cancer cells demonstrated reduced cell growth.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> To determine whether the cytotoxicity of <b>XD2-149</b> is dependent on ZFP91, we knocked down the protein in BxPC-3 using siRNA (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B). This resulted in a significant reduction (<i>p</i> < 0.01) in the cytotoxicity of <b>XD2-149</b> after a 48 h treatment in MTT, providing evidence to the importance of ZFP91 for its anticancer activity (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>C).</div><figure id="fig9" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Cytotoxicity of <b>XD2-149</b> is partially dependent on ZFP91. (A) Expression levels of ZFP91 in normal (gray) and pancreatic adenocarcinoma (PAAD) (red) tissues and the overall survival with low vs high ZFP91 expression in PAAD. The data was obtained from the GEPIA2 web server that is based on TCGA and GTEx databases. (B) siRNA knockdown of ZFP91 in BxPC-3 cells after 48 and 72 h. (C) IC<sub>50</sub> values of <b>XD2-149</b> show significant reduction in its cytotoxicity with ZFP91 knockdown compared to the siRNA control cells (scrRNA). BxPC-3 cells were treated with siZFP91 and NP or <b>XD2-149</b> for 48 h in the MTT assay. ** denotes <i>p</i> ≤ 0.01.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> NQO1-Mediated Cytotoxicity of <b>XD2-149</b> Is Independent of ZFP91</h3><div class="NLM_p">To further study the cell death mechanisms that contribute to the cytotoxicity of <b>XD2-149</b>, we examined the mechanism induced by napabucasin to determine if they kill the cells similarly. As previously mentioned, NQO1 activates napabucasin to induce ROS and subsequently kill the cancer cells.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In the presence of the NQO1 inhibitor, dicoumarol (DIC), napabucasin and <b>XD2-149</b> displayed reduced colony formation in BxPC-3 and MIA PaCa-2 cells (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A). These results suggest that the cell death mechanism of <b>XD2-149</b> is similar to napabucasin and is dependent on NQO1. To determine whether the NQO1-induced cytotoxicity is associated with ZFP91, we tested for the ZFP91 levels when treated with the PROTAC in the presence of DIC. The presence of DIC did not alter the ZFP91 levels, suggesting a ZFP91-independent cell death mechanism (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>B).</div><figure id="fig10" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. NQO1-induced cytotoxicity of <b>XD2-149</b> is independent of ZFP91. (A) Dicoumarol partially rescues the cell death caused by <b>XD2-149</b> in CFA. BxPC-3 and MIA PaCa-2 cells were treated with <b>XD2-149</b> or napabucasin for 7–10 days. (B) Dicoumarol (DIC) effect is independent of ZFP91. BxPC-3 cells were treated with <b>XD2-149</b> in the absence or presence of DIC for 16 h. The proteasome inhibitor carfilzomib was added 1 h prior to the treatments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21710" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21710" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We synthesized and tested a series of napabucasin-based PROTACs for targeting STAT3. We optimized the linker and the E3 ligase ligand to produce the lead compound <b>XD2-149</b> that showed significant cytotoxicity in several cancer cell lines. Although <b>XD2-149</b> resulted in inhibition of IL6/STAT3 pathway signaling, its effect was independent of the proteasome-mediated STAT3 degradation. GSEA of the proteomics data revealed several gene sets that indicated the STAT3 and other signaling pathways were targeted. Proteomics and western blot analyses identified and validated a proteasome-dependent degradation of ZFP91 by <b>XD2-149</b>. The PROTAC was more effective in degrading ZFP91 than the IMiD pomalidomide. Furthermore, we confirmed that the cytotoxicity of <b>XD2-149</b> is partially dependent on ZFP91. We also demonstrated that NQO1 contributes to the cytotoxicity of <b>XD2-149</b> independent of ZFP91, which suggests that the PROTAC kills the cancer cells via multiple cell death mechanisms. ZFP91 is an oncogenic protein that has been studied for its potential as an anticancer target, and further optimization may lead to the development of potent degraders of ZFP91 for anticancer therapy. Studies are in progress to validate other downregulated proteins as potential targets of our napabucasin-based PROTAC.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50845" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50845" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Generation of pomalidomide-conjugated napabucasin PROTACS with varying linear linkers (<b>5a–j</b>) is summarized in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Intermediate <b>2-2</b> was obtained by condensation of <b>2-1</b> with 3-aminopiperidine-2,6-dione hydrochloride. A nucleophilic substitution reaction between <b>2-2</b> and corresponding amines in the presence of DIPEA yielded intermediates <b>2-3a–j</b>. Boc deprotection of <b>2-3a–j</b> using TFA yielded <b>2-4a–j</b>, which were converted to <b>2-5a–j</b> by reacting with <b>1-5</b> (Scheme S1) in the presence of HATU and DIPEA.</div><div class="NLM_p">Syntheses of pomalidomide-conjugated PROTACS with varying cyclized linkers (<b>3-6a–d</b>) are outlined in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. A nucleophilic substitution reaction between <b>2-2</b> and corresponding amines in the presence of DIPEA yielded intermediates <b>3-2a–d</b>, which were converted to <b>3-3a–d</b> by deprotection of the Boc group. Intermediates <b>3-4a–d</b> were obtained through a reductive amination reaction using NaBH(OAc)<sub>3</sub>. Further removal of the Boc group yielded amines <b>3-5a–d</b>. Compounds <b>3-6a–c and 3-6d</b> (<b>XD2-149</b>) were obtained by reacting <b>5a–d</b> with <b>1-5</b> in the presence of HATU and DIPEA.</div><div class="NLM_p">The synthesis of compound <b>4-5</b> is outlined in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Condensation of <b>4-1</b> with 3-aminopiperidine-2,6-dione hydrochloride gave intermediate <b>4-2</b>, which was converted to <b>4-3</b> through a Sonogashira reaction. Deprotection of the Boc group of <b>4-3</b> yielded amine <b>4-4</b>, which was converted to <b>4-5</b> by reacting with <b>1-5</b> in the presence of HATU and DIPEA.</div><div class="NLM_p">The synthesis of compounds <b>5-5a</b> and <b>5-5b</b> are outlined in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. Condensation of <b>5-1</b> with 3-aminopiperidine-2,6-dione hydrochloride gave intermediate <b>5-2</b>, which was converted to <b>5-3a–b</b> through a nucleophilic substitution. Deprotection of Boc groups of <b>5-3a–b</b> yielded amines <b>5-4a–b</b>, which were converted to <b>5-5a</b> and <b>5-5b</b> by reacting with <b>1-5</b> in the presence of HATU and DIPEA.</div><div class="NLM_p">The synthesis of compounds <b>6-6a–c</b> is outlined in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. A substitution reaction between <b>5-1</b> and <i>t</i>-butyl bromoacetate gave intermediate <b>6-2</b>, which was converted to <b>6-3</b> using TFA. Condensation reaction between <b>6-3</b> and corresponding amines yielded <b>6-4a–c</b>, which was converted to <b>6-5a–c</b> by deprotection of the Boc group. Compounds <b>6-6a–c</b> were obtained by reacting <b>6-5a–c</b> with <b>1-5</b> in the presence of HATU and DIPEA.</div><div class="NLM_p">The synthesis of compound <b>7-4</b> is summarized in <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>. A nucleophilic substitution reaction between <b>7-1</b> and the corresponding amine in the presence of DIPEA yielded intermediate <b>7-2</b>. Boc deprotection of <b>7-2</b> using TFA yielded <b>7-3</b>, which was converted to <b>XD182</b> by reacting with <b>1-5</b> in the presence of HATU and DIPEA.</div><div class="NLM_p">The synthesis of compound <b>8-1</b> is summarized in <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>. <b>8-1</b> was obtained by reacting <b>8</b> with <b>1-5</b> in the presence of HATU and DIPEA.</div><figure id="sch7" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0018.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compound <b>8-1</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU, DIPEA, DMF, rt.</p></p></figure><div class="NLM_p">The synthesis of compounds <b>9-4a–b</b> is outlined in <a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>. A condensation reaction between <b>1-5</b> and corresponding amines gave intermediates <b>9-2a–b</b>, which was converted to <b>9-3a–b</b> using TFA. A condensation reaction between <b>9-3a–b</b> and (<i>S</i>,<i>R</i>,<i>S</i>)-AHPC hydrochloride yielded <b>9-4a–b</b>.</div><figure id="sch8" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0019.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Compounds <b>9-4a–b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU, DIPEA, DMF, rt; (b) TFA, DCM, rt; (c) (<i>S</i>,<i>R</i>,<i>S</i>)-AHPC hydrochloride, HATU, DIPEA, DMF, rt.</p></p></figure><div class="NLM_p">The synthesis of compound <b>10-5</b> (<b>XD171</b>) is summarized in <a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>. Intermediate <b>10-2</b> was obtained by condensation of <b>2-1</b> with 3-aminopiperidine-2,6-dione hydrochloride. A nucleophilic substitution reaction between <b>10-2</b> and corresponding amine in the presence of DIPEA yielded intermediate <b>10-3</b>. Boc deprotection of <b>10-3</b> using TFA yielded <b>10-4</b>, which was converted to <b>XD171</b> by reacting with <b>1-5</b> in the presence of HATU and DIPEA.</div><figure id="sch9" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0020.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of Compound <b>10-5</b> (<b>XD171</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOAc, HOAc, 100 °C; (b) DIPEA, DMF, 80 °C; (c) TFA, DCM, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><div class="NLM_p">The synthesis of compound <b>11-7</b> (<b>XD2-162</b>) is outlined in <a class="ref internalNav" href="#sch10" aria-label="Scheme 10">Scheme 10</a>. A nucleophilic substitution reaction between <b>11-2</b> and the corresponding amine in the presence of DIPEA yielded intermediate <b>11-3</b>, which was converted to <b>11-4</b> by deprotection of the Boc group. Intermediate <b>11-5</b> was obtained through a reductive amination reaction using NaBH(OAc)<sub>3</sub>. Further removal of the Boc group yielded amine <b>11-6</b>. Compound <b>XD2-162</b> was obtained by reacting <b>11-6</b> with <b>1-5</b> in the presence of HATU and DIPEA.</div><figure id="sch10" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0021.jpeg" id="gr20" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Synthesis of Compound <b>11-7</b> (<b>XD2-162</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOAc, HOAc, 100 °C; (b) DIPEA, DMF, 80 °C; (c) TFA, DCM, rt; (d) NaBH(OAc)<sub>3</sub>, DCE, rt; (e) HATU, DIPEA, DMF, rt.</p></p></figure></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26721" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26721" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> General Methods</h3><div class="NLM_p">Reagents and anhydrous solvents were purchased from commercial sources and used without further purification. Reaction progress was monitored by UV absorbance using thin-layer chromatography (TLC) on aluminum-backed precoated silica plates from Silicycle (SiliaPlate, 200 μm thickness, F254). Glassware for reactions were oven-dried in preparation, and reactions were performed under a nitrogen or argon atmosphere using standard inert conditions. <sup>1</sup>H NMR spectra were obtained using a Bruker (300 or 400 MHz) instrument. Spectral data are reported using the following abbreviations: s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, dd = doublet of doublets, ddd = doublet of doublet of doublets, dt = doublet of triplets, ddt = doublet of doublet of triplets, dtd = doublet of triplet of doublets, and coupling constants are reported in Hz followed by integration. Purifications using flash chromatography were performed using a Biotage Isolera chromatography system (25 μM spherical silica). Column chromatography was performed on silica gel (200–300 mesh), and preparative TLC was performed on UV 254 (0.5 mm thickness, Sigma-Aldrich). A Shimadzu LCMS 20-20 system was utilized for generating HPLC traces, obtaining mass spectrometry data, and evaluating purity. The system is equipped with a PDA UV detector and Kinetex 2.6 μm, XB-C18 100 Å, 75 mm × 4.6 mm column, which was used at room temperature. The HPLC gradient method utilized a 1% to 90% MeCN in H<sub>2</sub>O with 0.01% formic acid over 15 min with a 0.50 mL/min flow rate. Purity of the final compounds is ≥95% and was assessed at 254 nm using the described column and method.</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 2-Hydroxy-3-(prop-1-en-1-yl)naphthalene-1,4-dione (<b>1<i>-</i>2</b>)</h4><div class="NLM_p last">To a solution of reagent <b>1-1</b> (10.00 g, 54.41 mmol) in 170 mL of HOAc and 10 mL of concentrated HCl was added propionaldehyde (20.81 mL, 287.09 mmol) at 80 °C, and the reaction mixture was stirred at 80 °C for 1.5 h. After the reaction was completed, the reaction solution was poured into ice water and extracted with ethyl acetate (150 mL × 3). The organic layers were combined and extracted with saturated K<sub>2</sub>CO<sub>3</sub> aqueous solution (150 mL × 6), the combined aqueous layer was acidized using concentrated HCl, and the red solid formed was collected by filtration, which was used into next step without further purification (2.90 g). LCMS (ESI) 213 [M – H]<sup>−</sup>.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 2-Methylnaphtho[2,3-<i>b</i>]furan-4,9-dione (<b>1<i>-</i>3</b>)</h4><div class="NLM_p last">A mixture of <b>1-2</b> (2.20 g, 10.28 mmol) and Hg(OAc)<sub>2</sub> (6.45 g, 20.56 mmol) in 250 mL of HOAc was stirred at rt for 3 h and then filtered, and the filtrate was concentrated under reduced pressure to obtain a red solid, which was dissolved in 40 mL of HOAc and 40 mL of concentrated HCl; the resulting solution was stirred at 80 °C for 1 h. After the reaction was completed, the reaction solution was diluted with water at 0 °C, and the yellow solid formed was collected by filtration (2.30 g, 82%). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.23–8.12 (m, 2H), 7.78–7.67 (m, 2H), 6.61 (q, <i>J</i> = 0.9 Hz, 1H), 2.52 (d, <i>J</i> = 0.9 Hz, 3H).</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 4,9-Dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carbaldehyde (<b>1-4</b>)</h4><div class="NLM_p last">A mixture of intermediate <b>1-3</b> (1.80 g, 8.49 mmol), SeO<sub>2</sub> (4.70 g, 67.92 mmol), and SiO<sub>2</sub> (18.00 g) was dissolved in DCM (100 mL), then the solvent was removed under reduced pressure, and the residue obtained was stirred at 150 °C for 6 h. The crude product was purified with column chromatography (hexane/ethyl acetate = 4:1) to yield intermediate <b>1-4</b> as a yellow solid (1.10 g, 57%). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 9.98 (s, 1H), 8.36–8.19 (m, 2H), 7.91–7.76 (m, 2H), 7.69 (s, 1H).</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 4,9-Dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxylic Acid (<b>1-5</b>)</h4><div class="NLM_p last">To a solution of intermediate <b>1-4</b> in HOAc (60 mL) was added H<sub>2</sub>O<sub>2</sub> (22 mL) at 75 °C, and the resulting mixture was stirred at 75 °C for 2 h. After cooling down, water (50 mL) was added, the solution was concentrated under reduced pressure, and the solid formed was collected by filtration and washed with water (5 mL × 3). Intermediate <b>1-5</b> was obtained as a yellow solid, which was used into next step without further purification (470 mg, 73%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.25–8.04 (m, 2H), 7.97–7.82 (m, 2H), 7.68 (s, 1H).</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 2-(2,6-Dioxopiperidin-3-yl)-4-fluoroisoindoline-1,3-dione (<b>2-2</b>)</h4><div class="NLM_p last">A mixture of <b>2-1</b> (2.20, 13.25 mmol), 3-aminopiperidine-2,6-dione hydrochloride (2.17, 13.25 mmol), and NaOAc (3.30, 40.24 mmol) was dissolved in HOAc (50 mL), and the resulting mixture was stirred at 100 °C for 16 h. After the reaction was completed, the solution was concentrated under reduced pressure to obtain a residue, which was diluted with NaHCO<sub>3</sub> aqueous solution and extracted with ethyl acetate (150 mL × 3), and the combined organic phase was washed with NaHCO<sub>3</sub> aqueous solution (150 mL) and brine (150 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to yield a yellow solid, which was used in the next step without further purification (2.30 g, 63%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.16 (s, 1H), 7.94 (ddd, <i>J</i> = 8.4, 7.3, 4.6 Hz, 1H), 7.79 (d, <i>J</i> = 7.3 Hz, 1H), 7.73 (dd, <i>J</i> = 9.3, 8.4, 1H), 5.16 (dd, <i>J</i> = 12.9, 5.4 Hz, 1H), 2.89 (m, 1H), 2.67–2.50 (m, 2H), 2.14–2.00 (m, 1H).</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> General Procedure for the Preparation of <b>2-3a</b>–<b>j</b></h4><div class="NLM_p">A mixture of intermediate <b>2-2</b> (1 equiv), <i>tert</i>-butyl (2-aminoethyl)carbamate (1 equiv), DIPEA (2 equiv), and DMF (2 mL) was stirred at 80 °C. After 12 h, the reaction mixture was diluted with water (10 mL), extracted with ethyl acetate (2 x 15 mL), washed with brine, dried over Na<sub>2</sub>SO<sub><span class="strike">4</span></sub>, filtered, and concentrated. The residue obtained was purified with preparative TLC (DCM/MeOH = 15:1) to obtain intermediate <b>2-3a–j</b> or was used in the next step without further purification.</div><div id="sec5_1_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> <i>tert</i>-Butyl (2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)carbamate (<b>2-3a</b>)</h5><div class="NLM_p last">Yellow solid (20 mg, 27%). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.43 (s, 1H), 7.49 (t, <i>J</i> = 7.9 Hz, 1H), 7.10 (d, <i>J</i> = 7.1 Hz, 1H), 6.97 (d, <i>J</i> = 8.5 Hz, 1H), 6.39 (t, <i>J</i> = 5.5 Hz, 1H), 5.09–4.80 (m, 2H), 3.52–3.24 (m, 4H), 2.90–2.71 (m, 3H), 2.19–2.05 (m, 1H), 1.43 (s, 9H).</div></div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> General Procedure for the Preparation of <b>2-4a</b>–<b>j</b></h4><div class="NLM_p last">Intermediate <b>2-3</b> was dissolved in DCM (2 mL), TFA (1 mL) was added to the reaction mixture, and the mixture was stirred at rt. After 2 h, the reaction mixture was concentrated under reduced pressure. The residue thus obtained was used in the next step without further purification.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>N</i>-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>2-5a</b>)</h4><div class="NLM_p last">To a stirred solution of intermediate <b>1-5</b> (12 mg, 0.048 mmol) in DMF (2 mL) was added HATU (27 mg, 0.071 mmol), DIPEA (16 mg, 0.15 mmol), and intermediate <b>2-4a</b> (20 mg, 0.048 mmol), and the reaction mixture was stirred at rt for 12 h. The reaction solution was diluted with water (10 mL), extracted with ethyl acetate (15 mL × 2), washed with brine, dried over Na<sub>2</sub>SO<sub><span class="strike">4</span></sub>, filtered, and concentrated. The residue obtained was purified with preparative TLC (DCM/MeOH = 25:1) to yield compound <b>2-5a</b> as a yellow solid (5 mg, 18%). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.63 (s, 1H), 8.26–8.12 (m, 2H), 7.84–7.70 (m, 2H), 7.62 (s, H), 7.58 (s, 1H), 7.48 (t, <i>J</i> = 7.9 Hz, 1H), 7.08 (d, <i>J</i> = 7.0 Hz, 1H), 6.98 (d, <i>J</i> = 8.2 Hz, 1H), 6.49 (s, 1H), 5.17–4.92 (m, 1H), 3.79–3.63 (m, 2H), 3.63–3.47 (m, 2H), 2.97–2.73 (m, 3H), 2.21–2.06 (m, 1H). LCMS (ESI) 541 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>N</i>-(3-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)propyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>2-5b</b>)</h4><div class="NLM_p last">Compound <b>2-5b</b> was obtained as a yellow solid using a method similar to that described for compound <b>2-5a</b>. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.27–8.18 (m, 2H), 8.02 (s, 1H), 7.83–7.78 (m, 2H), 7.62 (s, 1H), 7.57–7.45 (m, 1H), 7.11 (d, <i>J</i> = 7.0 Hz, 1H), 6.91 (d, <i>J</i> = 8.5 Hz, 1H), 6.48 (s, 1H), 3.71–3.53 (m, 2H), 3.49–3.36 (m, 3H), 3.01–2.56 (m, 2H), 2.20–1.93 (m, 4H). LCMS (ESI) 555 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i>-(4-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)butyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>2-5c</b>)</h4><div class="NLM_p last">Compound <b>2-5c</b> was obtained as a yellow solid using a method similar to that described for compound <b>2-5a</b>. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.29–8.16 (m, 2H), 8.02 (s, 1H), 7.85–7.75 (m, 2H), 7.60 (s, 1H), 7.50 (dd, <i>J</i> = 8.4, 7.3 Hz, 1H), 7.09 (d, <i>J</i> = 7.0 Hz, 1H), 6.92–6.81 (m, 2H), 6.26 (s, 1H), 5.04–4.78 (m, 1H), 3.65–3.47 (m, 2H), 3.42–3.24 (m, 2H), 2.97–2.67 (m, 3H), 2.27–2.06 (m, 1H), 1.83–1.74 (m, 4H). LCMS (ESI) 569 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>N</i>-(5-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)pentyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>2-5d</b>)</h4><div class="NLM_p last">Compound <b>2-5d</b> was obtained as a yellow solid using a method similar to that described for compound <b>2-5a</b>. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.32–8.14 (m, 2H), 8.05 (s, 1H), 7.89–7.71 (m, 2H), 7.60 (s, 1H), 7.50 (dd, <i>J</i> = 8.5, 7.2 Hz, 1H), 7.09 (d, <i>J</i> = 7.1 Hz, 1H), 6.89 (d, <i>J</i> = 8.5 Hz, 2H), 5.03–4.75 (m, 1H), 3.51 (q, <i>J</i> = 7.3 Hz, 2H), 3.38–3.15 (m, 2H), 3.00–2.65 (m, 3H), 2.22–2.01 (m, 1H), 1.81–1.67 (m, 4H), 1.08–0.66 (m, 2H). LCMS (ESI) 583 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> <i>N</i>-(6-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)hexyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>2-5e</b>)</h4><div class="NLM_p last">Compound <b>2-5e</b> was obtained as a yellow solid using a method similar to that described for compound <b>2-5a</b>. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.29–8.13 (m, 2H), 8.09 (s, 1H), 7.88–7.71 (m, 2H), 7.59 (s, 1H), 7.55–7.41 (m, 1H), 7.08 (d, <i>J</i> = 7.1 Hz, 1H), 6.88 (d, <i>J</i> = 8.3 Hz, 1H), 6.87–6.75 (m, 2H), 6.24 (s, 1H), 5.00–4.79 (m, 1H), 3.47 (q, <i>J</i> = 6.3 Hz, 2H), 3.28 (q, <i>J</i> = 6.8 Hz, 2H), 2.98–2.65 (m, 3H), 2.22–2.07 (m, 1H), 1.68 (dd, <i>J</i> = 13.9, 6.5 Hz, 4H), 1.51–1.40 (m, 4H). LCMS (ESI) 597 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>N</i>-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>2-5f</b>)</h4><div class="NLM_p last">Compound <b>2-5f</b> was obtained as a yellow solid using a method similar to that described for compound <b>2-5a</b>. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.29–8.08 (m, 3H), 7.85–7.74 (m, 2H), 7.61 (s, 1H), 7.49 (t, <i>J</i> = 7.9 Hz, 1H), 7.21 (s, 1H), 7.06 (d, <i>J</i> = 7.1 Hz, 1H), 6.91 (d, <i>J</i> = 8.5 Hz, 1H), 6.55 (s, 1H), 5.09–4.94 (m, 1H), 3.88–3.62 (m, 6H), 3.60–3.34 (m, 2H), 2.96–2.77 (m, 3H), 2.24–2.08 (m, 1H). LCMS (ESI) 585 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>N</i>-(2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethoxy)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>2-5g</b>, <b>XD97</b>)</h4><div class="NLM_p last">Compound <b>XD97</b> was obtained as a yellow solid using a method similar to that described for compound <b>2-5a</b>. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.47 (s, 1H), 8.27–8.12 (m, 2H), 7.85–7.68 (m, 2H), 7.52 (s, 1H), 7.33 (t, <i>J</i> = 7.7 Hz, 2H), 6.97 (d, <i>J</i> = 7.1 Hz, 1H), 6.83 (d, <i>J</i> = 8.5 Hz, 1H), 6.52 (s, 1H), 4.99–4.83 (m, 1H), 3.81 (t, <i>J</i> = 5.2 Hz, 2H), 3.76–3.59 (m, 8H), 3.48 (q, <i>J</i> = 5.4 Hz, 2H), 2.93–2.72 (m, 3H), 2.25–2.10 (m, 1H). LCMS (ESI) 629 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>N</i>-(2-(2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethoxy)ethoxy)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>2-5h</b>)</h4><div class="NLM_p last">Compound <b>2-5h</b> was obtained as a yellow solid using a method similar to that described for compound <b>2-5a</b>. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.27 (s, 1H), 8.23–8.14 (m, 2H), 7.83–7.75 (m, 2H), 7.56 (s, 1H), 7.49–7.38 (m, 2H), 7.05 (d, <i>J</i> = 7.2 Hz, 1H), 6.87 (d, <i>J</i> = 8.4 Hz, 1H), 6.46 (d, <i>J</i> = 5.1 Hz, 1H), 5.04–4.80 (m, 1H), 3.75–3.64 (m, 14H), 3.51–3.39 (m, 2H), 3.01–2.60 (m, 3H), 2.24–2.09 (m, 1H). LCMS (ESI) 673 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>N</i>-(14-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)-3,6,9,12-tetraoxatetradecyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>2-5i</b>)</h4><div class="NLM_p last">Compound <b>2-5i</b> was obtained as a yellow solid using a method similar to that described for compound <b>2-5a</b>. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.46 (s, 1H), 8.28–8.16 (m, 2H), 7.83–7.72 (m, 2H), 7.58 (s, 2H), 7.46 (t, <i>J</i> = 8.0 Hz, 1H), 7.07 (d, <i>J</i> = 7.2 Hz, 1H), 6.87 (d, <i>J</i> = 8.6 Hz, 1H), 6.47 (s, 1H), 5.03–4.86 (m, 1H), 3.78–3.63 (m, 18H), 3.50–3.38 (m, 2H), 2.99–2.71 (m, 3H), 2.26–2.05 (m, 1H). LCMS (ESI) 717 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <i>N</i>-(20-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)-3,6,9,12,15,18-hexaoxaicosyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>2-5j</b>)</h4><div class="NLM_p last">Compound <b>2-5j</b> was obtained as a yellow solid using a method similar to that described for compound <b>2-5a</b>. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.51 (s, 1H), 8.29–8.12 (m, 2H), 7.85–7.72 (m, 2H), 7.58 (s, 1H), 7.48 (q, <i>J</i> = 8.3, 7.3 Hz, 2H), 7.07 (d, <i>J</i> = 7.1 Hz, 1H), 6.89 (d, <i>J</i> = 8.6 Hz, 1H), 6.48 (t, <i>J</i> = 6.5 Hz, 1H), 5.01–4.86 (m, 1H), 3.70–3.62 (m, 22H), 3.50–3.38 (m, 2H), 2.96–2.66 (m, 3H), 2.19–2.04 (m, 1H). LCMS (ESI) 805 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>tert</i>-Butyl 4-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)piperazine-1-carboxylate (<b>3-2a</b>)</h4><div class="NLM_p last">A mixture of <b>2-2</b> (200 mg, 0.72 mmol), <i>tert</i>-butyl 4-(2-aminoethyl)piperazine-1-carboxylate (174 mg, 0.76 mmol), and DIPEA (187 mg, 1.45 mmol) in DMF (5 mL) was stirred at 80 °C for 12 h. The reaction mixture was diluted with water (20 mL) and extracted with ethyl acetate (25 mL × 3). The organic layers were combined and washed with brine, dried over Na<sub>2</sub>SO<sub><span class="strike">4</span></sub>, filtered, and concentrated. The residue obtained was purified with column chromatography (DCM/MeOH = 15:1) to yield intermediate <b>3-2a</b> as a yellow solid (80 mg, 23%). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 9.11 (s, 1H), 7.45 (t, <i>J</i> = 7.9 Hz, 1H), 7.05 (d, <i>J</i> = 7.2 Hz, 1H), 6.83 (d, <i>J</i> = 8.4 Hz, 1H), 6.65 (s, 1H), 4.99–4.80 (m, 1H), 3.48–3.40 (m, 4H), 3.37–3.27 (m, 2H), 2.87–2.51 (m, 5H), 2.46–2.38 (m, 4H), 2.16–2.03 (m, 1H), 1.41 (s, 9H).</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 2-(2,6-Dioxopiperidin-3-yl)-4-((2-(piperazin-1-yl)ethyl)amino)isoindoline-1,3-dione Trifluoroacetate (<b>3-3a</b>)</h4><div class="NLM_p last">Intermediate <b>3-2a</b> (60 mg, 0.12 mmol) was dissolved in DCM (2 mL), TFA (1 mL) was added, and the mixture was stirred at rt for 2 h. The reaction solution was concentrated under reduced pressure to obtain <b>3-3a</b> as a yellow solid, which was used into next step without further purification (60 mg, 100%).</div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>tert</i>-Butyl (2-(4-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)piperazin-1-yl)ethyl)carbamate (<b>3-4a</b>)</h4><div class="NLM_p last">Intermediate <b>3-3a</b> (60 mg, 0.12 mmol) and <i>tert</i>-butyl (2-oxoethyl)carbamate (40 mg, 0.24 mmol) were dissolved in DCE (10 mL), the mixture was stirred at rt for 0.5 h, then NaBH(OAc)<sub>3</sub> (106 mg, 0.50 mmol) was added, and the mixture was stirred at rt for 12 h. The reaction mixture was diluted with water (20 mL) and extracted with DCM (15 mL × 3). The organic layers were combined and washed with brine, dried over Na<sub>2</sub>SO<sub><span class="strike">4</span></sub>, filtered, and concentrated. The residue obtained was purified with column chromatography (DCM/MeOH = 10:1) to yield intermediate <b>3-4a</b> as a yellow solid (60 mg, 91%). <sup>1</sup>H NMR (300 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.55 (t, <i>J</i> = 8.0 Hz, 1H), 7.05 (d, <i>J</i> = 7.8 Hz, 2H), 5.06 (dd, <i>J</i> = 12.3, 5.2 Hz, 1H), 3.69–3.52 (m, 4H), 3.41 (dt, <i>J</i> = 14.7, 6.5 Hz, 4H), 3.16 (q, <i>J</i> = 6.9 Hz, 4H), 3.05–2.84 (m, 4H), 2.83–2.59 (m, 3H), 2.15–2.10 (m, 1H), 1.44 (s, 9H).</div></div><div id="sec5_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 4-((2-(4-(2-Aminoethyl)piperazin-1-yl)ethyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione Trifluoroacetate (<b>3-5a</b>)</h4><div class="NLM_p last">Intermediate <b>3-4a</b> (40 mg, 0.076 mmol) was dissolved in DCM (2 mL), TFA (1 mL) was added, and the mixture was stirred at rt for 2 h. The reaction solution was concentrated under reduced pressure to obtain <b>3-5a</b> as a yellow solid, which was used in the next step without further purification (48 mg, 100%).</div></div><div id="sec5_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> <i>N</i>-(2-(4-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)piperazin-1-yl)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>3-6a</b>)</h4><div class="NLM_p last">To a stirred solution of intermediate <b>1-5</b> (37 mg, 0.015 mmol) in DMF (2 mL) was added HATU (88 mg, 0.23 mmol), DIPEA (60 mg, 0.46 mmol), and intermediate <b>3-5a</b> (48 mg, 0.077 mmol), and the reaction mixture was stirred at rt for 12 h. The reaction solution was diluted with water (10 mL), extracted with ethyl acetate (15 mL × 2), washed with brine, dried over Na<sub>2</sub>SO<sub><span class="strike">4</span></sub>, filtered, and concentrated. The residue obtained was purified with preparative TLC (DCM/MeOH = 15:1) to yield compound <b>3-6a</b> as a yellow solid (12 mg, 24%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.08 (s, 1H), 8.85 (t, <i>J</i> = 5.8 Hz, 1H), 8.18–8.07 (m, 2H), 7.94–7.87 (m, 2H), 7.62 (s, 1H), 7.60–7.54 (m, 1H), 7.09 (d, <i>J</i> = 8.5 Hz, 1H), 7.02 (d, <i>J</i> = 7.0 Hz, 1H), 6.75 (t, <i>J</i> = 5.1 Hz, 1H), 5.07 (dd, <i>J</i> = 12.9, 5.5 Hz, 1H), 3.39 (dt, <i>J</i> = 18.3, 6.1 Hz, 4H), 2.88 (ddd, <i>J</i> = 17.4, 14.0, 5.3 Hz, 1H), 2.64–2.52 (m, 4H), 2.50–2.35 (m, 10H), 2.07–1.99 (m, 1H). LCMS (ESI) 653 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> <i>N</i>-(2-(4-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethyl)piperidin-1-yl)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>3-6b</b>)</h4><div class="NLM_p last">Compound <b>3-6b</b> was obtained as a yellow solid using a method similar to that described for compound <b>3-6a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.08 (s, 1H), 8.84 (t, <i>J</i> = 5.7 Hz, 1H), 8.25–8.06 (m, 2H), 8.03–7.85 (m, 2H), 7.62 (s, 1H), 7.58 (dd, <i>J</i> = 8.6, 7.1 Hz, 1H), 7.09 (d, <i>J</i> = 8.6 Hz, 1H), 7.02 (d, <i>J</i> = 7.0 Hz, 1H), 6.49 (t, <i>J</i> = 5.8 Hz, 1H), 5.05 (dd, <i>J</i> = 12.9, 5.4 Hz, 1H), 3.40 (q, <i>J</i> = 6.4 Hz, 2H), 3.36–3.32 (m, 2H), 2.96–2.76 (m, 3H), 2.63–2.53 (m, 2H), 2.49–2.43 (m, 2H), 2.07–1.99 (m, 1H), 2.00–1.90 (m, 2H), 1.69 (d, <i>J</i> = 12.5 Hz, 2H), 1.57–1.45 (m, 2H), 1.41–1.27 (m, 1H), 1.21–1.16 (m, 2H). LCMS (ESI) 652 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>N</i>-(2-(4-(3-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)propyl)piperidin-1-yl)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>3-6c</b>)</h4><div class="NLM_p last">Compound <b>3-6c</b> was obtained as a yellow solid using a method similar to that described for compound <b>3-6a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.08 (s, 1H), 8.83 (t, <i>J</i> = 5.7 Hz, 1H), 8.19–8.05 (m, 2H), 7.94–7.86 (m, 2H), 7.62 (s, 1H), 7.57 (dd, <i>J</i> = 8.6, 7.0 Hz, 1H), 7.08 (d, <i>J</i> = 8.6 Hz, 1H), 7.01 (d, <i>J</i> = 7.0 Hz, 1H), 6.52 (t, <i>J</i> = 5.9 Hz, 1H), 5.05 (dd, <i>J</i> = 12.9, 5.4 Hz, 1H), 3.39 (q, <i>J</i> = 6.4 Hz, 2H), 3.29–3.23 (m, 2H), 2.95–2.81 (m, 3H), 2.64–2.52 (m, 2H), 2.46 (t, <i>J</i> = 6.8 Hz, 2H), 2.07–2.00 (m, 1H), 1.97–1.86 (m, 2H), 1.67–1.52 (m, 4H), 1.31–1.20 (m, 3H), 1.16–1.05 (m, 2H). LCMS (ESI) 666 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>N</i>-(2-(4-(5-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)pentyl)piperidin-1-yl)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>3-6d</b>, <b>XD2-149</b>)</h4><div class="NLM_p last">Compound <b>XD2-149</b> was obtained as a yellow solid using a method similar to that described for compound <b>3-6a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.10 (s, 1H), 8.86 (t, <i>J</i> = 5.7 Hz, 1H), 8.18–8.06 (m, 2H), 7.90 (ddt, <i>J</i> = 6.8, 4.7, 3.4 Hz, 2H), 7.62 (s, 1H), 7.57 (dd, <i>J</i> = 8.5, 7.0 Hz, 1H), 7.09 (d, <i>J</i> = 8.6 Hz, 1H), 7.01 (d, <i>J</i> = 7.0 Hz, 1H), 6.53 (t, <i>J</i> = 6.1 Hz, 1H), 5.10–4.99 (m, 1H), 3.58–3.34 (m, 10H), 2.94–2.76 (m, 3H), 2.63–2.55 (m, 1H), 2.05–1.98 (m, 1H), 1.92 (d, <i>J</i> = 10.2 Hz, 1H), 1.58 (q, <i>J</i> = 9.5, 6.5 Hz, 4H), 1.38–1.24 (m, 6H). LCMS (ESI) 694 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 4-Bromo-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>4-2</b>)</h4><div class="NLM_p last">A mixture of <b>4-1</b> (2.00 g, 8.81 mmol), 3-aminopiperidine-2,6-dione hydrochloride (1.44 g, 8.81 mmol), and NaOAc (2.17, 26.46 mmol) was dissolved in HOAc (50 mL), and the resulting mixture was stirred at 120 °C for 24 h. After the reaction was completed, the solution was concentrated under reduced pressure to obtain a residue, which was diluted with water (100 mL), and the gray solid formed was collected by filtration, which was used into next step without further purification (2.20 g, 74%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.17 (s, 1H), 8.06 (d, <i>J</i> = 8.0 Hz, 1H), 7.93 (d, <i>J</i> = 7.3 Hz, 1H), 7.77 (t, <i>J</i> = 7.8 Hz, 1H), 5.17 (dd, <i>J</i> = 12.6, 4.8 Hz, 1H), 3.02–2.78 (m, 1H), 2.67–2.51 (m, 2H), 2.17–1.96 (m, 1H).</div></div><div id="sec5_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>tert</i>-Butyl (5-(2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)pent-4-yn-1-yl)carbamate (<b>4-3</b>)</h4><div class="NLM_p last">A mixture of <b>4-2</b> (80 mg, 0.24 mmol), <i>tert</i>-butyl pent-4-yn-1-ylcarbamate (44 mg, 0.24 mmol), Pd(PPh3)<sub>2</sub>Cl<sub>2</sub> (17 mg, 0.024 mmol), and CuI (9 mg, 0.048 mmol) in DMF/TEA (2 mL/1 mL) was purged with nitrogen several times and then stirred at 80 °C for 12 h under a nitrogen atmosphere. The reaction mixture was then filtered, and the filtrate was concentrated under reduced pressure to obtain a residue, which was diluted with water (10 mL) and extracted with ethyl acetate (15 mL × 3). The organic layers were combined and washed with brine, dried over Na<sub>2</sub>SO<sub><span class="strike">4</span></sub>, filtered, and concentrated. The residue obtained was purified with column chromatography (DCM/MeOH = 25:1) to yield intermediate <b>4-3</b> as a yellow solid (14 mg, 13%). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.25 (s, 1H), 7.86 (t, <i>J</i> = 7.7 Hz, 1H), 7.78 (d, <i>J</i> = 6.8 Hz, 1H), 7.64 (d, <i>J</i> = 8.3 Hz, 1H), 5.06–4.89 (m, 2H), 3.42–3.27 (m, 2H), 2.90–2.75 (m, 3H), 2.57 (t, <i>J</i> = 6.8 Hz, 2H), 2.22–2.07 (m, 1H), 1.95–1.78 (m, 2H), 1.41 (s, 9H).</div></div><div id="sec5_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 4-(5-Aminopent-1-yn-1-yl)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione Trifluoroacetate (<b>4-4</b>)</h4><div class="NLM_p last">Intermediate <b>4-4</b> was obtained as a yellow solid from <b>4-3</b> using a method similar as that described for intermediate <b>2-4a</b> in quantitative yield.</div></div><div id="sec5_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>N</i>-(5-(2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)pent-4-yn-1-yl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>4-5</b>)</h4><div class="NLM_p last">To a stirred solution of intermediate <b>1-5</b> (8 mg, 0.033 mmol) in DMF (2 mL) was added HATU (18 mg, 0.047 mmol), DIPEA (16 mg, 0.12 mmol), and intermediate <b>4-4</b> (14 mg, 0.032 mmol), the reaction mixture was stirred at rt for 12 h. The reaction solution was diluted with water (10 mL), extracted with ethyl acetate (15 mL × 2), washed with brine, dried over Na<sub>2</sub>SO<sub><span class="strike">4</span></sub>, filtered, and concentrated. The residue obtained was purified with preparative TLC (DCM/MeOH = 20:1) to yield compound <b>4-5</b> as a yellow solid (8 mg, 44%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 (s, 1H), 9.02 (t, <i>J</i> = 5.7 Hz, 1H), 8.16–8.08 (m, 2H), 7.94–7.87 (m, 2H), 7.86–7.75 (m, 3H), 7.64 (s, 1H), 5.14 (dd, <i>J</i> = 12.9, 5.4 Hz, 1H), 3.49 (q, <i>J</i> = 6.5 Hz, 2H), 2.89 (ddd, <i>J</i> = 17.1, 13.9, 5.3 Hz, 1H), 2.65–2.51 (m, 4H), 2.07 (ddd, <i>J</i> = 10.5, 5.4, 3.1 Hz, 1H), 1.89 (p, <i>J</i> = 6.9 Hz, 2H). LCMS (ESI) 564 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 2-(2,6-Dioxopiperidin-3-yl)-4-hydroxyisoindoline-1,3-dione (<b>5-2</b>)</h4><div class="NLM_p last">A mixture of <b>5-1</b> (3.00 g, 18.29 mmol), 3-aminopiperidine-2,6-dione hydrochloride (3.00 g, 18.29 mmol), and NaOAc (4.50, 54.88 mmol) was dissolved in HOAc (50 mL), and the resulting mixture was stirred at 110 °C for 16 h. After the reaction was complete, the solution was concentrated under reduced pressure to obtain a residue, which was diluted with water (100 mL), and the gray solid formed was collected by filtration, which was used in the next step without further purification (3.70 g, 74%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.19 (s, 1H), 11.10 (s, 1H), 7.65 (dd, <i>J</i> = 8.4, 7.2 Hz, 1H), 7.32 (d, <i>J</i> = 7.1 Hz, 1H), 7.25 (dd, <i>J</i> = 8.4, 0.8 Hz, 1H), 5.07 (dd, <i>J</i> = 12.9, 5.4 Hz, 1H), 2.89 (ddd, <i>J</i> = 18.0, 14.0, 5.5 Hz, 1H), 2.67–2.49 (m, 2H), 2.10–1.92 (m, 1H).</div></div><div id="sec5_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> <i>tert</i>-Butyl (4-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)butyl)carbamate (<b>5-3</b>)</h4><div class="NLM_p last">A suspension of <b>5-2</b> (100 mg, 0.36 mmol), <i>tert</i>-butyl (4-bromobutyl)carbamate (101 mg, 0.40 mmol), KHCO<sub>3</sub> (110 mg, 1.10 mmol), and a catalytic amount of KI in DMF (5 mL) was stirred at 70 °C for 12 h. The reaction mixture was diluted with water (20 mL) and extracted with ethyl acetate (15 mL × 3). The organic layers were combined and washed with brine, dried over Na<sub>2</sub>SO<sub><span class="strike">4</span></sub>, filtered, and concentrated. The residue obtained was purified with column chromatography (DCM/MeOH = 15:1) to yield intermediate <b>5-3</b> as a brown solid (60 mg, 37%). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.68 (s, 1H), 7.64 (t, <i>J</i> = 7.9 Hz, 1H), 7.41 (d, <i>J</i> = 7.3 Hz, 1H), 7.19 (d, <i>J</i> = 8.5 Hz, 1H), 5.14–4.85 (m, 2H), 4.16 (t, <i>J</i> = 6.1 Hz, 2H), 3.30–3.08 (m, 2H), 2.92–2.66 (m, 3H), 2.18–2.02 (m, 1H), 1.89 (p, <i>J</i> = 6.5 Hz, 2H), 1.70 (p, <i>J</i> = 6.5 Hz, 2H), 1.40 (s, 9H).</div></div><div id="sec5_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 4-(4-Aminobutoxy)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione Trifluoroacetate (<b>5-4</b>)</h4><div class="NLM_p last">Intermediate <b>5-4</b> was obtained as a yellow solid from <b>5-3</b> using a method similar as that described for intermediate <b>2-4a</b> in quantitative yield.</div></div><div id="sec5_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> <i>N</i>-(4-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)butyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>5-5a</b>)</h4><div class="NLM_p last">To a stirred solution of intermediate <b>1-5</b> (20 mg, 0.083 mmol) in DMF (2 mL) was added HATU (39 mg, 0.10 mmol), DIPEA (27 mg, 0.21 mmol), and intermediate <b>5-4</b> (30 mg, 0.068 mmol), and the reaction mixture was stirred at rt for 12 h. The reaction solution was diluted with water (10 mL), extracted with ethyl acetate (15 mL × 2), washed with brine, dried over Na<sub>2</sub>SO<sub><span class="strike">4</span></sub>, filtered, and concentrated. The residue obtained was purified with preparative TLC (DCM/MeOH = 15:1) to yield compound <b>5-5a</b> as a white solid (12 mg, 31%). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.39 (s, 1H), 8.28–8.15 (m, 2H), 8.18–8.08 (m, 1H), 7.83–7.73 (m, 2H), 7.71 (t, <i>J</i> = 8.0 Hz, 1H), 7.60 (s, 1H), 7.48 (d, <i>J</i> = 7.3 Hz, 1H), 7.21 (d, <i>J</i> = 8.5 Hz, 1H), 5.45–5.20 (m, 1H), 4.24 (s, 2H), 3.78–3.44 (m, 2H), 22.88 (s, 2H), 2.38–2.19 (m, 1H), 2.12–1.86 (m, 4H). LCMS (ESI) 570 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>N</i>-(2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)ethoxy)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>5-5b</b>)</h4><div class="NLM_p last">Compound <b>5-5b</b> was obtained as a yellow solid using a method similar to that described for compound <b>5-5a</b>. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.43 (s, 1H), 8.19 (dd, <i>J</i> = 8.9, 3.8 Hz, 2H), 7.78 (t, <i>J</i> = 4.5 Hz, 2H), 7.58 (t, <i>J</i> = 7.9 Hz, 1H), 7.54 (s, 1H), 7.48–7.34 (m, 2H), 7.23 (d, <i>J</i> = 9.1 Hz, 1H), 5.05–4.86 (m, 1H), 4.39 (s, 2H), 4.01 (s, 2H), 3.82 (s, 2H), 3.75–3.60 (m, 6H), 2.95–2.71 (m, 3H), 2.21–2.07 (m, 1H). LCMS (ESI) 630 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> <i>tert</i>-Butyl 2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetate (<b>6-2</b>)</h4><div class="NLM_p last">A mixture of <b>5-1</b> (1.50 g, 5.47 mmol), <i>tert</i>-butyl 2-bromoacetate (1.12 g, 5.74 mmol), and K<sub>2</sub>CO<sub>3</sub> (1.13 g, 8.19 mmol) in DMF (20 mL) was stirred at rt for 12 h. Water was added slowly to the reaction solution, and the precipitate formed was collected by filtration and washed with water (15 mL × 2). Intermediate <b>6-2</b> was obtained as a white solid (600 mg, 29%). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 7.98 (s, 1H), 7.67 (t, <i>J</i> = 8.3 Hz, 1H), 7.51 (d, <i>J</i> = 7.4 Hz, 1H), 7.10 (d, <i>J</i> = 8.5 Hz, 1H), 5.03–4.88 (m, 1H), 4.79 (s, 2H), 2.99–2.63 (m, 3H), 2.22–2.03 (m, 1H), 1.48 (s, 9H).</div></div><div id="sec5_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetic Acid (<b>6-3</b>)</h4><div class="NLM_p last">Intermediate <b>6-2</b> (500 mg, 1.29 mmol) was dissolved in TFA (4 mL), the solution was stirred at rt for 3 h, and then the reaction was diluted with DCM (10 mL) and concentrated under reduced pressure to obtain a white solid, which was used into next step without further purification (500 mg, crude).</div></div><div id="sec5_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>tert</i>-Butyl (2-(2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)ethoxy)ethoxy)ethyl)carbamate (<b>6-4a</b>)</h4><div class="NLM_p last">To a stirred solution of <b>6-3</b> (86 mg, 0.27 mmol) in DMF (2 mL) was added HATU (152 mg, 0.40 mmol), DIPEA (0.80 mmol), and <i>tert</i>-butyl (2-(2-(2-aminoethoxy)ethoxy)ethyl)carbamate (69 mg, 0.28 mmol), and the reaction was stirred at rt for 16 h. The reaction solution was diluted with water (10 mL) and extracted with ethyl acetate (15 mL × 3). The organic layers were combined and washed with brine, dried over Na<sub>2</sub>SO<sub><span class="strike">4</span></sub>, filtered, and concentrated. The residue obtained was purified with column chromatography (DCM/MeOH = 15:1) to yield intermediate <b>6-4a</b> as a yellow solid (50 mg, 53%). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.91 (s, 1H), 7.72 (t, <i>J</i> = 7.9 Hz, 1H), 7.62 (s, 1H), 7.53 (d, <i>J</i> = 7.3 Hz, 1H), 7.18 (d, <i>J</i> = 8.4 Hz, 1H), 5.16 (s, 1H), 5.05–4.90 (m, 1H), 4.65 (s, 2H), 3.66–3.46 (m, 10H), 3.28 (d, <i>J</i> = 5.6 Hz, 2H), 2.91–2.71 (m, 3H), 2.20–2.06 (m, 1H), 1.42 (s, 9H).</div></div><div id="sec5_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>N</i>-(2-(2-(2-Aminoethoxy)ethoxy)ethyl)-2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamide Trifluoroacetate (<b>6-5a</b>)</h4><div class="NLM_p last">Intermediate <b>6-4a</b> (40 mg, 0.072 mmol) was dissolved in TFA (2 mL), and the mixture was stirred at 50 °C for 1 h. The reaction solution was diluted with DCM (5 mL) and concentrated under reduced pressure to obtain <b>6-5a</b> as a yellow oil, which was used in the next step without further purification (40 mg, 100%).</div></div><div id="sec5_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> <i>N</i>-(2-(2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)ethoxy)ethoxy)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>6-6a</b>)</h4><div class="NLM_p last">To a stirred solution of intermediate <b>1-5</b> (17 mg, 0.070 mmol) in DMF (2 mL) was added HATU (41 mg, 0.11 mmol), DIPEA (27 mg, 0.21 mmol), and intermediate <b>6-5a</b> (40 mg, 0.072 mmol), and the reaction mixture was stirred at rt for 12 h. The reaction solution was diluted with water (10 mL), extracted with ethyl acetate (15 mL × 2), washed with brine, dried over Na<sub>2</sub>SO<sub><span class="strike">4</span></sub>, filtered, and concentrated. The residue obtained was purified with preparative TLC (DCM/MeOH = 15:1) to yield compound <b>6-6a</b> as a white solid (14 mg, 29%). <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.83 (s, 1H), 8.29–8.10 (m, 2H), 7.78 (t, <i>J</i> = 4.5 Hz, 2H), 7.70 (q, <i>J</i> = 11.2, 9.6 Hz, 3H), 7.58–7.45 (m, 2H), 7.17 (d, <i>J</i> = 8.4 Hz, 1H), 5.15–4.95 (m, 1H), 4.64 (s, 2H), 3.81–3.49 (m, 12H), 3.02–2.64 (m, 3H), 2.26–2.13 (m, 1H). LCMS (ESI) 687 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> <i>N</i>-(1-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)-2-oxo-6,9,12-trioxa-3-azatetradecan-14-yl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>6-6b</b>)</h4><div class="NLM_p last">Compound <b>6-6b</b> was obtained as a yellow solid using a method similar to that described for compound <b>6-6a</b>. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.21 (s, 2H), 7.79 (dd, <i>J</i> = 5.7, 3.4 Hz, 2H), 7.70 (s, 1H), 7.58 (s, 1H), 7.53 (d, <i>J</i> = 7.4 Hz, 1H), 7.20 (s, 1H), 4.64 (s, 2H), 3.71 (q, <i>J</i> = 6.3 Hz, 16H), 3.18 (d, <i>J</i> = 7.4 Hz, 4H). LCMS (ESI) 731 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> <i>N</i>-(2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-5-yl)amino)ethoxy)ethoxy)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>6-6c</b>)</h4><div class="NLM_p last">Compound <b>6-6c</b> was obtained as a yellow solid using a method similar to that described for compound <b>6-6a</b>. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.12 (s, 1H), 9.03–8.86 (m, 1H), 8.20–8.07 (m, 2H), 8.08–7.95 (m, 1H), 7.96–7.86 (m, 2H), 7.81 (t, <i>J</i> = 7.9 Hz, 1H), 7.63 (s, 1H), 7.48 (d, <i>J</i> = 7.4 Hz, 1H), 7.38 (d, <i>J</i> = 8.6 Hz, 1H), 5.11 (dd, <i>J</i> = 12.9, 5.4 Hz, 1H), 4.78 (s, 2H), 3.28 (d, <i>J</i> = 7.5 Hz, 4H), 3.18 (d, <i>J</i> = 6.4 Hz, 2H), 3.01–2.74 (m, 1H), 2.68–2.52 (m, 2H), 2.12–1.93 (m, 1H), 1.61–1.37 (m, 4H). LCMS (ESI) 649 [M + Na]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> <i>N</i>-(2-(2-(2-((2-(2,6-Dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)amino)ethoxy)ethoxy)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>7-4</b>)</h4><div class="NLM_p last">Compound <b>7-4</b> was obtained as a yellow solid using a method similar to that described for compound <b>2-5a</b> by using 3-(4-fluoro-1-oxoisoindolin-2-yl)piperidine-2,6-dione as the starting material. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.59 (s, 1H), 8.22–8.12 (m, 2H), 7.82–7.73 (m, 2H), 7.53 (s, 1H), 7.38 (t, <i>J</i> = 5.1 Hz, 1H), 7.23 (d, <i>J</i> = 7.7 Hz, 1H), 7.16 (dd, <i>J</i> = 7.6, 0.9 Hz, 1H), 6.73 (dd, <i>J</i> = 8.0, 0.9 Hz, 1H), 5.24 (dd, <i>J</i> = 13.2, 5.2 Hz, 1H), 4.32 (d, <i>J</i> = 15.6 Hz, 1H), 4.13 (d, <i>J</i> = 15.7 Hz, 1H), 3.77 (tt, <i>J</i> = 7.1, 3.7 Hz, 2H), 3.70–3.53 (m, 8H), 3.39 (q, <i>J</i> = 4.7 Hz, 2H), 2.94–2.77 (m, 2H), 2.34 (qd, <i>J</i> = 12.8, 5.7 Hz, 1H), 2.20 (dtd, <i>J</i> = 13.0, 5.1, 2.9 Hz, 1H). LCMS (ESI) 615 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> <i>N</i>-(5-(2-(2,6-Dioxopiperidin-3-yl)-1-oxoisoindolin-4-yl)pentyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>8-1</b>)</h4><div class="NLM_p last">Compound <b>8-1</b> was obtained as a yellow solid using a method similar to that described for compound <b>4-5</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.99 (s, 1H), 8.93 (t, <i>J</i> = 5.7 Hz, 1H), 8.19–8.09 (m, 2H), 7.95–7.87 (m, 2H), 7.61 (s, 1H), 7.55 (dd, <i>J</i> = 6.4, 2.2 Hz, 1H), 7.49–7.41 (m, 2H), 5.12 (dd, <i>J</i> = 13.3, 5.1 Hz, 1H), 4.47 (d, <i>J</i> = 17.2 Hz, 1H), 4.31 (d, <i>J</i> = 17.1 Hz, 1H), 3.27 (d, <i>J</i> = 6.8 Hz, 2H), 2.99–2.84 (m, 1H), 2.64 (d, <i>J</i> = 8.0 Hz, 2H), 2.48–2.36 (m, 2H), 2.07–1.98 (m, 1H), 1.61 (dq, <i>J</i> = 21.6, 7.4 Hz, 4H), 1.37 (p, <i>J</i> = 7.8 Hz, 2H). LCMS (ESI) 554 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> <i>tert</i>-Butyl 2-(2-(2-(4,9-Dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamido)ethoxy)ethoxy)acetate (<b>9-2a</b>)</h4><div class="NLM_p last">To a stirred solution of intermediate <b>1-5</b> (30 mg, 0.12 mmol) in DMF (2 mL) was added HATU (71 mg, 0.19 mmol), DIPEA (48 mg, 0.37 mmol), and <i>tert</i>-butyl 2-(2-(2-aminoethoxy)ethoxy)acetate (30 mg, 0.14 mmol), and the reaction mixture was stirred at rt for 12 h. The reaction solution was diluted with water (10 mL), extracted with ethyl acetate (15 mL × 2), washed with brine, dried over Na<sub>2</sub>SO<sub><span class="strike">4</span></sub>, filtered, and concentrated. The residue obtained was purified with column chromatography (DCM/MeOH = 15:1) to yield compound <b>9-2a</b> as a yellow solid (28 mg, 51%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.23–8.16 (m, 2H), 7.81–7.73 (m, 2H), 7.56 (s, 1H), 7.40 (s, 1H), 4.09 (s, 2H), 3.76–3.63 (m, 8H), 1.44 (s, 9H).</div></div><div id="sec5_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 2-(2-(2-(4,9-Dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamido)ethoxy)ethoxy)acetic Acid (<b>9-3a</b>)</h4><div class="NLM_p last">Intermediate <b>9-2a</b> (28 mg, 0.063 mmol) was dissolved in DCM (2 mL), TFA (2 mL) was added, and the mixture was stirred at rt for 2 h. The reaction solution was concentrated under reduced pressure to obtain <b>9-3a</b> as a yellow oil, which was used into the next step without further purification (30 mg, crude).</div></div><div id="sec5_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-12-(<i>tert</i>-Butyl)-1-(4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-yl)-1,10-dioxo-5,8-dioxa-2,11-diazatridecan-13-oyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>9-4a</b>)</h4><div class="NLM_p last">To a stirred solution of intermediate <b>9-3a</b> (15 mg, 0.039 mmol) in DMF (2 mL) was added HATU (22 mg, 0.58 mmol), DIPEA (15 mg, 0.12 mmol), and (<i>S</i>,<i>R</i>,<i>S</i>)-AHPC hydrochloride (17 mg, 0.039 mmol), and the reaction mixture was stirred at rt for 12 h. The reaction solution was diluted with water (10 mL), extracted with ethyl acetate (15 mL × 2), washed with brine, dried over Na<sub>2</sub>SO<sub><span class="strike">4</span></sub>, filtered, and concentrated. The residue obtained was purified with preparative TLC (DCM/MeOH = 15:1) to yield compound <b>9-4a</b> as a yellow solid (8 mg, 26%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.97 (t, <i>J</i> = 5.7 Hz, 1H), 8.94 (s, 1H), 8.55 (t, <i>J</i> = 6.0 Hz, 1H), 8.18–8.06 (m, 2H), 7.97–7.84 (m, 2H), 7.64 (s, 1H), 7.44 (d, <i>J</i> = 9.8 Hz, 1H), 7.38 (s, 4H), 5.14 (d, <i>J</i> = 3.5 Hz, 1H), 4.57 (d, <i>J</i> = 9.5 Hz, 1H), 4.48–4.31 (m, 3H), 4.24 (dd, <i>J</i> = 15.7, 5.7 Hz, 1H), 3.98 (s, 2H), 3.69–3.56 (m, 8H), 3.50–3.40 (m, 2H), 2.42 (s, 3H), 2.09–2.01 (m, 1H), 1.96–1.84 (m, 1H), 0.94 (s, 9H). LCMS (ESI) 800 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> (4<i>R</i>)-1-((<i>S</i>)-2-(2-(2-(4,9-Dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamido)ethoxy)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>9-4b</b>)</h4><div class="NLM_p last">Compound <b>9-4b</b> was obtained as a yellow solid using a method similar to that described for compound <b>9-4a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.12 (t, <i>J</i> = 5.8 Hz, 1H), 8.96 (s, 1H), 8.55 (t, <i>J</i> = 6.0 Hz, 1H), 8.19–8.06 (m, 2H), 7.94–7.86 (m, 2H), 7.66 (d, <i>J</i> = 1.3 Hz, 1H), 7.53 (d, <i>J</i> = 9.5 Hz, 1H), 7.39 (d, <i>J</i> = 1.3 Hz, 4H), 5.13 (d, <i>J</i> = 3.4 Hz, 1H), 4.57 (d, <i>J</i> = 9.4 Hz, 1H), 4.48–4.31 (m, 3H), 4.24 (dd, <i>J</i> = 15.7, 5.7 Hz, 1H), 4.02 (s, 2H), 3.71–3.58 (m, 4H), 3.55–3.46 (m, 2H), 2.43 (d, <i>J</i> = 1.4 Hz, 3H), 2.10–2.00 (m, 1H), 1.96–1.84 (m, 1H), 0.93 (s, 9H). LCMS (ESI) 756 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> <i>N</i>-(2-(2-(2-((1,3-Dioxo-2-(2-oxopiperidin-3-yl)isoindolin-4-yl)amino)ethoxy)ethoxy)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>10-5</b>, <b>XD171</b>)</h4><div class="NLM_p last">Compound <b>XD171</b> was obtained as a yellow solid using a method similar to that described for compound <b>2-5a</b>. <sup>1</sup>H NMR (300 MHz, chloroform-<i>d</i>) δ 8.17 (t, <i>J</i> = 4.7 Hz, 2H), 7.82–7.63 (m, 3H), 7.51 (s, 1H), 7.32 (t, <i>J</i> = 7.8 Hz, 1H), 6.95 (d, <i>J</i> = 7.1 Hz, 1H), 6.81 (d, <i>J</i> = 8.5 Hz, 1H), 6.61 (s, 1H), 6.53 (s, 1H), 4.68 (dd, <i>J</i> = 12.1, 6.1 Hz, 1H), 3.77 (t, <i>J</i> = 5.4 Hz, 2H), 3.70–3.43 (m, 13H), 2.38 (q, <i>J</i> = 12.0 Hz, 1H), 2.09 (s, 1H), 1.94 (s, 1H). LCMS (ESI) 615 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div><div id="sec5_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> <i>N</i>-(2-(4-(5-((2-(1-Methyl-2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)pentyl)piperidin-1-yl)ethyl)-4,9-dioxo-4,9-dihydronaphtho[2,3-<i>b</i>]furan-2-carboxamide (<b>11-7</b>, <b>XD2<i>-</i>162</b>)</h4><div class="NLM_p last">Compound <b>XD2-162</b> was obtained as a yellow solid using a method similar to that described for compound <b>3-6a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.85 (t, <i>J</i> = 5.8 Hz, 1H), 8.18–8.08 (m, 2H), 7.96–7.85 (m, 2H), 7.62 (s, 1H), 7.58 (dd, <i>J</i> = 8.6, 7.0 Hz, 1H), 7.09 (d, <i>J</i> = 8.6 Hz, 1H), 7.01 (d, <i>J</i> = 7.1 Hz, 1H), 6.53 (t, <i>J</i> = 5.9 Hz, 1H), 5.19–5.01 (m, 1H), 3.42–3.34 (m, 10H), 3.01 (s, 3H), 2.94–2.70 (m, 3H), 2.44 (d, <i>J</i> = 6.7 Hz, 1H), 2.04 (dtd, <i>J</i> = 13.2, 5.5, 2.8 Hz, 1H), 1.91 (d, <i>J</i> = 2.1 Hz, 1H), 1.58 (t, <i>J</i> = 14.0 Hz, 4H), 1.21–1.01 (m, 6H). LCMS (ESI) 708 [M + H]<sup>+</sup>. HPLC purity at 254 nm.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> Cell Culture</h3><div class="NLM_p last">Cell lines were maintained in RPMI-1640 or DMEM supplemented with 10% fetal bovine serum (FBS) (ThermoFisher Scientific). The cells were grown as monolayer cultures at 37 °C in a humidified atmosphere of 5% CO<sub>2</sub> and tested for <i>Mycoplasma</i> contamination with the <i>Mycoplasma</i> detection kit, PlasmaTest (InvivoGen). The BxPC-3 cell line was authenticated with STR DNS profiling (University of Michigan Sequencing Core) and matched to reference profiles from the ATCC database. The STAT3-knockout cell lines were generated in our laboratory using CRISPR Cas9.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> Cytotoxicity Assays</h3><div class="NLM_p last">For MTT assay, the experiments were carried out in 96-well plates where cells (3000–6000 cells/well) were seeded in the medium and incubated overnight. The compounds or the vehicle (DMSO) were added to the cells the following day and incubated for 3 days. MTT solution (0.3 mg/mL) was added to the wells, and plates were incubated for 3 h at 37 °C after which MTT was discarded and cells were dissolved in DMSO. The absorbance of the formazan crystals was read by a microplate reader (Molecular Devices) at 570 nm, and the data were analyzed using GraphPad Prism 8 software. For colony formation assays, the cells (200–500 cells/well) were seeded in 96-well plates overnight and then treated with DMSO or the compound for 7–10 days until 80% confluency. After colony formation, the medium was removed and the cells were stained with 0.05% crystal violet, washed with water, and imaged with iBright imaging system (Invitrogen). The colonies were dissolved in Sorensen buffer (0.1 M sodium citrate, pH 4.2) for colorimetric quantification.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Western Blot Analysis</h3><div class="NLM_p last">Cells were seeded (OVCAR-3: 500 × 10<sup>3</sup>, MIA PaCa-2: 200 × 10<sup>3</sup>, BxPC-3: 500 × 10<sup>3</sup>cells/well) in 6-well plates. Following treatment, cells were lysed at 4 °C by sonication with lysis buffer (25 mM tris(hydroxymethyl)aminomethane, 150 mM NaCl, 17 mM Triton X-100, and 3.5 mM SDS, pH 7.4) supplemented with protease inhibitor and phosphatase inhibitor. The cells were then spun at 12,000 rpm at 4 °C for 10 min and collected for determining the protein concentration with the BCA assay (ThermoFisher Scientific, Waltham, MO). Samples were then prepared and loaded onto 10 or 12% acrylamide (BioRad, Hercules, CA) gels followed by the transfer of the proteins onto PVDF membranes (EMD Millipore, La Jolla, CA). The membranes were blocked for 1 h prior to incubation with the primary antibodies using Odyssey blocking buffer (LI-COR Biosciences). Membranes were then probed for STAT3/pSTAT3/STAT1/NQO1 (Cell Signaling, Danvers, MA, 1:1000), GAPDH (Cell Signaling, Danvers, MA, 1:4000), GSPT (Cell Signaling, 1:1000), and ZFP91 (Bethyl, 1:1000). Following overnight incubation with the primary antibodies at 4 °C, the membranes were incubated with the secondary antibodies (anti-rabbit, Cell Signaling, 1:7500 or anti-mouse, Cell Signaling, 1:5000) for 1 h and imaged with the Odyssey imaging system (LI-COR Biosciences).</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> STAT3 Luciferase Reporter Assay</h3><div class="NLM_p last">The GloResponse SIE-<i>luc2P</i> HEK-293 cells (10,000 cells/well) were seeded in a 96-well plate (white, opaque bottom) and incubated overnight. On day 2, IL6 (100 ng/mL) was added to the cells for 24 h to induce activation of the STAT3 pathway. The compounds were then added for 24 h after which the Bio-Glo luciferase assay reagent (Promega #G7940) was added to each well. The luminescence was measured after 5 min using a plate reader. An MTT assay in HEK-293 was performed in parallel treating the cells with the compounds for 24 h.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Protein Identification and Relative Quantitation by TMT Labeling and LC-Tandem MS</h3><div class="NLM_p last">BxPC-3 cells were treated with DMSO (control), napabucasin (2 μM), <b>XD2-149</b> (2 μM), or <b>XD2-162</b> (4 μM) for 16 h then collected and lysed in RIPA buffer (Pierce, #89900) at 4 °C. The control and <b>XD2-149</b> treatments were carried out in triplicate, while napabucasin and <b>XD2-162</b> were carried out in duplicate. Protein samples were then collected, and the concentrations were determined using the BCA assay (ThermoFisher Scientific). Samples were prepared in a concentration of 75 μg and submitted to the Proteomics Core in the Department of Pathology at the University of Michigan. Tandem mass tag (TMT) labeling was performed using the TMT 10plex isobaric labeling kit (ThermoFisher Scientific, #90110) according to the manufacturer’s protocol with minor modifications. This involves reduction of the samples with DTT (1 h, 55 °C) followed by alkylation with 2-chloroacetamide (30 min, rt). Proteins were precipitated using cold acetone overnight. Next, the proteins were pelleted and resuspended in TEAB to which trypsin (Promega, V5113) was added for digestion (overnight, 37 °C). TMT reagents were reconstituted in anhydrous acetonitrile, and the digested peptides were transferred to TMT reagent vials and incubated at rt for 1 h. The reaction was quenched with 5% hydroxylamine for 15 min, and then samples were combined and dried. Prior to MS analysis, two-dimensional separation of the sample was performed. For the first dimension, an offline fractionation of an aliquot of each sample into 10 fractions was performed following the manufacturer’s protocol (Pierce, #84868). Fractions were then dried and reconstituted in loading buffer (0.1% formic acid and 2% acetonitrile). For quantitation, the MultiNotch-MS3 method was employed.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> MS was performed on the Orbitrap Fusion Tribrid with ETD (ThermoFisher Scientific) equipped with the nano-LC system (Dionex RSLC-nano). Proteome discoverer (v2.1, ThermoFisher Scientific) was used for data analysis. MS2 spectra were searched against the SwissProt human protein database. The abundance ratio data sets were transformed to log2FC values, and the whole protein list was used for GSEA analysis (GSEA 4.0.3). For ranking, the proteins were filtered applying a fold change (FC) cutoff of 1.5 and <i>p</i>-value ≤ 0.05. All proteins have an experimental <i>q</i>-value of <0.1.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> siRNA Knockdown of ZFP91</h3><div class="NLM_p last">ZFP91 siRNA (hs.Ri.ZFP91.13.1, Integrated DNA Technologies) or scrambled negative control siRNA (#51-01-19-09, Integrated DNA Technologies) with Lipofectamine RNAiMAX transfection reagent (ThermoFisher Scientific) was added to BxPC-3 cells in Opti-MEM medium (ThermoFisher Scientific) for 48 and 72 h after which the cells were harvested for immunoblotting. The siRNA transfection reagent complex was prepared following the manufacturer’s instructions. Briefly, 30 pmol of siRNA in 150 μL of Opti-MEM medium was added to 9 μL of the Lipofectamine reagent in 150 μL of Opti-MEM medium. After 5 min incubation at room temperature, the siRNA transfection complex was added to each well in a 6-well plate. For MTT assay, 5 pmol of siRNA in 25 μL of Opti-MEM medium was added to 1.5 μL of the Lipofectamine reagent in 25 μL of Opti-MEM medium. After 5 min incubation at room temperature, 10 μL of the siRNA transfection complex was added to each well in a 96-well plate. <b>XD2-149</b> or napabucasin was added to the cells simultaneously with the siRNA transfection reagent complex for 48 h to determine their cytotoxicity.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> Statistical Analysis</h3><div class="NLM_p last">Significance levels for assays and immunoblots were calculated using unpaired Student’s <i>t</i>-test in GraphPad Prism. Results are shown as mean ± standard deviation from three independent experiments.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i88"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01897" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27852" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27852" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01897?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01897</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional figures described in the text; tables including clinical trials, activity of the compounds and proteomics results; additional synthetic schemes; additional methods; and NMR and LC/MS charts (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and the associated biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf">jm0c01897_si_001.pdf (8.85 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_002.csv">jm0c01897_si_002.csv (3.45 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01897" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61602" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61602" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nouri Neamati</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, College of Pharmacy and the Rogel Cancer Center,
North Campus Research Complex, 1600 Huron Parkway, University of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3291-7131" title="Orcid link">http://orcid.org/0000-0003-3291-7131</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6b050e0a060a1f022b1e06020803450e0f1e"><span class="__cf_email__" data-cfemail="9ef0fbfff3ffeaf7deebf3f7fdf6b0fbfaeb">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maha Hanafi</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, College of Pharmacy and the Rogel Cancer Center,
North Campus Research Complex, 1600 Huron Parkway, University of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Pharmacy, Cairo University, Cairo 11562, Egypt</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5389-4656" title="Orcid link">http://orcid.org/0000-0002-5389-4656</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xinde Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, College of Pharmacy and the Rogel Cancer Center,
North Campus Research Complex, 1600 Huron Parkway, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.H. and X.C. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i90">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83673" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83673" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by NIH grant R01 CA188252 and a grant from the University of Michigan Forbes Institute for Cancer Discovery. M.H. received a scholarship from the Egyptian government (Ministry of Higher Education and Scientific Research). We would also like to thank our colleagues Christine Cuthbertson, Daulat Khadka, and Essam Osman for reading the manuscript and providing helpful suggestions.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVIATIONS Used</td></tr><tr><td class="NLM_term">CK1α</td><td class="NLM_def"><p class="first last">casein Kinase 1α</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">CX</td><td class="NLM_def"><p class="first last">cycloheximide</p></td></tr><tr><td class="NLM_term">DIC</td><td class="NLM_def"><p class="first last">dicoumarol</p></td></tr><tr><td class="NLM_term">FC</td><td class="NLM_def"><p class="first last">fold change</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">GSEA</td><td class="NLM_def"><p class="first last">gene set enrichment analysis</p></td></tr><tr><td class="NLM_term">GSPT</td><td class="NLM_def"><p class="first last">G1 to S phase transition protein</p></td></tr><tr><td class="NLM_term">HIF-1α</td><td class="NLM_def"><p class="first last">hypoxia inducible factor 1 subunit alpha</p></td></tr><tr><td class="NLM_term">IKZF1</td><td class="NLM_def"><p class="first last">IKAROS Family Zinc Finger 1</p></td></tr><tr><td class="NLM_term">IKZF3</td><td class="NLM_def"><p class="first last">IKAROS Family Zinc Finger 3</p></td></tr><tr><td class="NLM_term">IMiDs</td><td class="NLM_def"><p class="first last">immunomodulatory drugs</p></td></tr><tr><td class="NLM_term">mPDAC</td><td class="NLM_def"><p class="first last">metastatic pancreatic adenocarcinoma</p></td></tr><tr><td class="NLM_term">MTT</td><td class="NLM_def"><p class="first last">3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide</p></td></tr><tr><td class="NLM_term">NF-κB</td><td class="NLM_def"><p class="first last">nuclear factor kappa B</p></td></tr><tr><td class="NLM_term">NP</td><td class="NLM_def"><p class="first last">napabucasin</p></td></tr><tr><td class="NLM_term">NQO1</td><td class="NLM_def"><p class="first last">NAD(P)H Quinone Dehydrogenase 1</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">STAT3</td><td class="NLM_def"><p class="first last">signaling transducer and activator of transcription 3</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">Von Hippel–Lindau</p></td></tr><tr><td class="NLM_term">ZBTB16</td><td class="NLM_def"><p class="first last">Zinc Finger and BTB Domain Containing 16</p></td></tr><tr><td class="NLM_term">ZFP91</td><td class="NLM_def"><p class="first last">zinc finger protein 91 homolog</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i92">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88831" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88831" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 61 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingston, D. G. I.</span></span> <span> </span><span class="NLM_article-title">Plant anticancer agents. XII. Isolation and structure elucidation of new cytotoxic quinones from Tabebuia cassinoides</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">600</span>– <span class="NLM_lpage">604</span>, <span class="refDoi"> DOI: 10.1021/np50023a014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np50023a014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADyaL3sXhsVemuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=1982&pages=600-604&author=M.+M.+Raoauthor=D.+G.+I.+Kingston&title=Plant+anticancer+agents.+XII.+Isolation+and+structure+elucidation+of+new+cytotoxic+quinones+from+Tabebuia+cassinoides&doi=10.1021%2Fnp50023a014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Plant anticancer agents.  XII.  Isolation and structure elucidation of new cytotoxic quinones from Tabebuia cassinoides</span></div><div class="casAuthors">Rao, M. Madhusudana; Kingston, David G. I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">600-4</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    </div><div class="casAbstract">Isolation of the 3 new cytotoxic furonaphthoquinones I, II, and III (or IV) from T. cassinoides is described.  The compds. were characterized by 1H NMR, UV, IR, and mass spectra.  The synthesis of I is also reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3HhDIxx1hoLVg90H21EOLACvtfcHk0lhqhr5BVAkOUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhsVemuw%253D%253D&md5=4586f58fa31a9b5eb46b77c988c3d4c3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fnp50023a014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp50023a014%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DM.%2BM.%26aulast%3DKingston%26aufirst%3DD.%2BG.%2BI.%26atitle%3DPlant%2520anticancer%2520agents.%2520XII.%2520Isolation%2520and%2520structure%2520elucidation%2520of%2520new%2520cytotoxic%2520quinones%2520from%2520Tabebuia%2520cassinoides%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D1982%26volume%3D45%26spage%3D600%26epage%3D604%26doi%3D10.1021%2Fnp50023a014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirai, K.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial activity of novel furanonaphthoquinone analogs</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">700</span>– <span class="NLM_lpage">702</span>, <span class="refDoi"> DOI: 10.1128/AAC.42.3.700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1128%2FAAC.42.3.700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=9517956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADyaK1cXhvV2kur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1998&pages=700-702&author=K.+Nagataauthor=K.-I.+Hiraiauthor=J.+Koyamaauthor=Y.+Wadaauthor=T.+Tamura&title=Antimicrobial+activity+of+novel+furanonaphthoquinone+analogs&doi=10.1128%2FAAC.42.3.700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial activity of novel furanonaphthoquinone analogs</span></div><div class="casAuthors">Nagata, Kumiko; Hirai, Kei-Ichi; Koyama, Junko; Wada, Yasunao; Tamura, Toshihide</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">700-702</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Analogs of furanonaphthoquinone (FNQ) from Tecoma ipe Mart had MICs ranging from 1.56 to 25 μg/mL against Gram-pos. bacteria.  FNQ showed significantly lower MICs against methicillin-resistant Staphylococcus aureus than against methicillin-sensitive S. aureus.  FNQ inhibited Helicobacter pylori with an MIC of 0.1 μg/mL.  Fungi, including pathogenic species, were sensitive to FNQ with MICs similar to those of amphotericin B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB7nhFqCqbd7Vg90H21EOLACvtfcHk0lhqhr5BVAkOUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhvV2kur8%253D&md5=26cb45ea436344e5bb35678d2be3c0c8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2FAAC.42.3.700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.42.3.700%26sid%3Dliteratum%253Aachs%26aulast%3DNagata%26aufirst%3DK.%26aulast%3DHirai%26aufirst%3DK.-I.%26aulast%3DKoyama%26aufirst%3DJ.%26aulast%3DWada%26aufirst%3DY.%26aulast%3DTamura%26aufirst%3DT.%26atitle%3DAntimicrobial%2520activity%2520of%2520novel%2520furanonaphthoquinone%2520analogs%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1998%26volume%3D42%26spage%3D700%26epage%3D702%26doi%3D10.1128%2FAAC.42.3.700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogoff, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murikipudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikule, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leggett, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardee, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. J.</span></span> <span> </span><span class="NLM_article-title">Suppression of cancer relapse and metastasis by inhibiting cancer stemness</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">1839</span>– <span class="NLM_lpage">1844</span>, <span class="refDoi"> DOI: 10.1073/pnas.1424171112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1073%2Fpnas.1424171112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=25605917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFKht7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=1839-1844&author=Y.+Liauthor=H.+A.+Rogoffauthor=S.+Keatesauthor=Y.+Gaoauthor=S.+Murikipudiauthor=K.+Mikuleauthor=D.+Leggettauthor=W.+Liauthor=A.+B.+Pardeeauthor=C.+J.+Li&title=Suppression+of+cancer+relapse+and+metastasis+by+inhibiting+cancer+stemness&doi=10.1073%2Fpnas.1424171112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of cancer relapse and metastasis by inhibiting cancer stemness</span></div><div class="casAuthors">Li, Youzhi; Rogoff, Harry A.; Keates, Sarah; Gao, Yuan; Murikipudi, Sylaja; Mikule, Keith; Leggett, David; Li, Wei; Pardee, Arthur B.; Li, Chiang J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1839-1844</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Partial or even complete cancer regression can be achieved in some patients with current cancer treatments.  However, such initial responses are almost always followed by relapse, with the recurrent cancer being resistant to further treatments.  The discovery of therapeutic approaches that counteract relapse is, therefore, essential for advancing cancer medicine.  Cancer cells are extremely heterogeneous, even in each individual patient, in terms of their malignant potential, drug sensitivity, and their potential to metastasize and cause relapse.  Indeed, hypermalignant cancer cells, termed cancer stem cells or stemness-high cancer cells, that are highly tumorigenic and metastatic have been isolated from cancer patients with a variety of tumor types.  Moreover, such stemness-high cancer cells are resistant to conventional chemotherapy and radiation.  Here we show that BBI608, a small mol. identified by its ability to inhibit gene transcription driven by Stat3 and cancer stemness properties, can inhibit stemness gene expression and block spherogenesis of or kill stemness-high cancer cells isolated from a variety of cancer types.  Moreover, cancer relapse and metastasis were effectively blocked by BBI608 in mice.  These data demonstrate targeting cancer stemness as a novel approach to develop the next generation of cancer therapeutics to suppress cancer relapse and metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBRIs4CUCKMrVg90H21EOLACvtfcHk0lgBo6kzCP8iww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFKht7k%253D&md5=7f77ee7a484e6f161a0d958743a5fa12</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1424171112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1424171112%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DRogoff%26aufirst%3DH.%2BA.%26aulast%3DKeates%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DMurikipudi%26aufirst%3DS.%26aulast%3DMikule%26aufirst%3DK.%26aulast%3DLeggett%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DPardee%26aufirst%3DA.%2BB.%26aulast%3DLi%26aufirst%3DC.%2BJ.%26atitle%3DSuppression%2520of%2520cancer%2520relapse%2520and%2520metastasis%2520by%2520inhibiting%2520cancer%2520stemness%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D1839%26epage%3D1844%26doi%3D10.1073%2Fpnas.1424171112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. L.</span>, <span class="NLM_string-name">Chiang, J.</span></span>, ; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> ; <span class="NLM_contrib-group"><span class="NLM_string-name">Leggett, D.</span></span> <span> </span><span class="NLM_article-title">Novel STAT3 Pathway Inhibitors and Cancer Stem Cell Inhibitors</span>. <span class="NLM_patent">US2,017,019,793A1</span>, July 2, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=Z.+L.+Jiang&author=J.+Chiangauthor=W.+Liauthor=D.+Leggett&title=Novel+STAT3+Pathway+Inhibitors+and+Cancer+Stem+Cell+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%2BL.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLeggett%26aufirst%3DD.%26atitle%3DNovel%2520STAT3%2520Pathway%2520Inhibitors%2520and%2520Cancer%2520Stem%2520Cell%2520Inhibitors%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karandish, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamnoon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gange, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallik, S.</span></span> <span> </span><span class="NLM_article-title">Nucleus-targeted, echogenic polymersomes for delivering a cancer stemness inhibitor to pancreatic cancer cells</span>. <i>Biomacromolecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4122</span>– <span class="NLM_lpage">4132</span>, <span class="refDoi"> DOI: 10.1021/acs.biomac.8b01133</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biomac.8b01133" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1WksbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=4122-4132&author=F.+Karandishauthor=B.+Mamnoonauthor=L.+Fengauthor=M.+K.+Haldarauthor=L.+Xiaauthor=K.+N.+Gangeauthor=S.+Youauthor=Y.+Choiauthor=K.+Sarkarauthor=S.+Mallik&title=Nucleus-targeted%2C+echogenic+polymersomes+for+delivering+a+cancer+stemness+inhibitor+to+pancreatic+cancer+cells&doi=10.1021%2Facs.biomac.8b01133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleus-Targeted, Echogenic Polymersomes for Delivering a Cancer Stemness Inhibitor to Pancreatic Cancer Cells</span></div><div class="casAuthors">Karandish, Fataneh; Mamnoon, Babak; Feng, Li; Haldar, Manas K.; Xia, Lang; Gange, Kara N.; You, Seungyong; Choi, Yongki; Sarkar, Kausik; Mallik, Sanku</div><div class="citationInfo"><span class="NLM_cas:title">Biomacromolecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4122-4132</span>CODEN:
                <span class="NLM_cas:coden">BOMAF6</span>;
        ISSN:<span class="NLM_cas:issn">1525-7797</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chemotherapeutic agents for treating cancers show considerable side effects, toxicity, and drug resistance.  To mitigate the problems, we designed nucleus-targeted, echogenic, stimuli-responsive polymeric vesicles (polymersomes) to transport and subsequently release the encapsulated anticancer drugs within the nuclei of pancreatic cancer cells.  We synthesized an alkyne-dexamethasone deriv. and conjugated it to N3-polyethylene glycol (PEG)-polylactic acid (PLA) copolymer employing the Cu2+ catalyzed "Click" reaction.  We prepd. polymersomes from the dexamethasone-PEG-PLA conjugate along with a synthesized stimuli-responsive polymer PEG-S-S-PLA.  The dexamethasone group dilates the nuclear pore complexes and transports the vesicles to the nuclei.  We designed the polymersomes to release the encapsulated drugs in the presence of a high concn. of reducing agents in the nuclei of pancreatic cancer cells.  We obsd. that the nucleus-targeted, stimuli-responsive polymersomes released 70% of encapsulated contents in the nucleus-mimicking environment in 80 min.  We encapsulated the cancer stemness inhibitor BBI608 in the vesicles and obsd. that the BBI608 encapsulated polymersomes reduced the viability of the BxPC3 cells to 43% in three-dimensional spheroid cultures.  The polymersomes were prepd. following a special protocol so that they scatter ultrasound, allowing imaging by a medical ultrasound scanner.  Therefore, these echogenic, targeted, stimuli-responsive, and drug-encapsulated polymersomes have the potential for trackable, targeted carrier of chemotherapeutic drugs to cancer cell nuclei.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIrfwt25qtC7Vg90H21EOLACvtfcHk0lgBo6kzCP8iww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1WksbfN&md5=8cbd99021cd2df174002181ba59c8274</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facs.biomac.8b01133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biomac.8b01133%26sid%3Dliteratum%253Aachs%26aulast%3DKarandish%26aufirst%3DF.%26aulast%3DMamnoon%26aufirst%3DB.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DHaldar%26aufirst%3DM.%2BK.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DGange%26aufirst%3DK.%2BN.%26aulast%3DYou%26aufirst%3DS.%26aulast%3DChoi%26aufirst%3DY.%26aulast%3DSarkar%26aufirst%3DK.%26aulast%3DMallik%26aufirst%3DS.%26atitle%3DNucleus-targeted%252C%2520echogenic%2520polymersomes%2520for%2520delivering%2520a%2520cancer%2520stemness%2520inhibitor%2520to%2520pancreatic%2520cancer%2520cells%26jtitle%3DBiomacromolecules%26date%3D2018%26volume%3D19%26spage%3D4122%26epage%3D4132%26doi%3D10.1021%2Facs.biomac.8b01133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span> <i>Search Orphan Drug Designations
and Approvals</i>; . (accessed October 10, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Search+Orphan+Drug+Designations%0Aand+Approvals%3B+.+%28accessed+October+10%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26jtitle%3DSearch%2520Orphan%2520Drug%2520Designations%250Aand%2520Approvals" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neoptolemos, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleeff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costello, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenhalf, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, D. H.</span></span> <span> </span><span class="NLM_article-title">Therapeutic developments in pancreatic cancer: current and future perspectives</span>. <i>Nat. Rev. Gastroenterol. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">348</span>, <span class="refDoi"> DOI: 10.1038/s41575-018-0005-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fs41575-018-0005-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=29717230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BC1MjovVOguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=333-348&author=J.+P.+Neoptolemosauthor=J.+Kleeffauthor=P.+Michlauthor=E.+Costelloauthor=W.+Greenhalfauthor=D.+H.+Palmer&title=Therapeutic+developments+in+pancreatic+cancer%3A+current+and+future+perspectives&doi=10.1038%2Fs41575-018-0005-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic developments in pancreatic cancer: current and future perspectives</span></div><div class="casAuthors">Neoptolemos John P; Kleeff Jorg; Kleeff Jorg; Costello Eithne; Greenhalf William; Palmer Daniel H; Michl Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Nature reviews. Gastroenterology & hepatology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">333-348</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The overall 5-year survival for pancreatic cancer has changed little over the past few decades, and pancreatic cancer is predicted to be the second leading cause of cancer-related mortality in the next decade in Western countries.  The past few years, however, have seen improvements in first-line and second-line palliative therapies and considerable progress in increasing survival with adjuvant treatment.  The use of biomarkers to help define treatment and the potential of neoadjuvant therapies also offer opportunities to improve outcomes.  This Review brings together information on achievements to date, what is working currently and where successes are likely to be achieved in the future.  Furthermore, we address the questions of how we should approach the development of pancreatic cancer treatments, including those for patients with metastatic, locally advanced and borderline resectable pancreatic cancer, as well as for patients with resected tumours.  In addition to embracing newer strategies comprising genomics, stromal therapies and immunotherapies, conventional approaches using chemotherapy and radiotherapy still offer considerable prospects for greater traction and synergy with evolving concepts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcStGgd4rHkEDz69zv1FsJFMfW6udTcc2eYVDFa1izFKg7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjovVOguw%253D%253D&md5=d1b4ab24b72f95c7077798a8258ccf58</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fs41575-018-0005-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41575-018-0005-x%26sid%3Dliteratum%253Aachs%26aulast%3DNeoptolemos%26aufirst%3DJ.%2BP.%26aulast%3DKleeff%26aufirst%3DJ.%26aulast%3DMichl%26aufirst%3DP.%26aulast%3DCostello%26aufirst%3DE.%26aulast%3DGreenhalf%26aufirst%3DW.%26aulast%3DPalmer%26aufirst%3DD.%2BH.%26atitle%3DTherapeutic%2520developments%2520in%2520pancreatic%2520cancer%253A%2520current%2520and%2520future%2520perspectives%26jtitle%3DNat.%2520Rev.%2520Gastroenterol.%2520Hepatol.%26date%3D2018%26volume%3D15%26spage%3D333%26epage%3D348%26doi%3D10.1038%2Fs41575-018-0005-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bekaii-Saab, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starodub, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Rayes, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaib, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanna, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikhail, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neki, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borodyansky, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span> <span> </span><span class="NLM_article-title">Phase 1b/2 trial of cancer stemness inhibitor napabucasin (NAPA) plus nab-paclitaxel (nPTX) and gemcitabine (Gem) in metastatic pancreatic adenocarcinoma (mPDAC)</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">4110</span>– <span class="NLM_lpage">4110</span>, <span class="refDoi"> DOI: 10.1200/JCO.2018.36.15_suppl.4110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1200%2FJCO.2018.36.15_suppl.4110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=4110-4110&author=T.+S.+Bekaii-Saabauthor=A.+Starodubauthor=B.+F.+El-Rayesauthor=S.+Shahdaauthor=B.+H.+O%E2%80%99Neilauthor=A.+M.+Noonanauthor=A.+M.+Noonanauthor=W.+L.+Shaibauthor=W.+T.+Hannaauthor=S.+Mikhailauthor=A.+S.+Nekiauthor=Y.+Changauthor=X.+Daiauthor=W.+Liauthor=E.+Brooksauthor=C.+Ohauthor=L.+Borodyanskyauthor=C.+Li&title=Phase+1b%2F2+trial+of+cancer+stemness+inhibitor+napabucasin+%28NAPA%29+plus+nab-paclitaxel+%28nPTX%29+and+gemcitabine+%28Gem%29+in+metastatic+pancreatic+adenocarcinoma+%28mPDAC%29&doi=10.1200%2FJCO.2018.36.15_suppl.4110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1200%2FJCO.2018.36.15_suppl.4110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2018.36.15_suppl.4110%26sid%3Dliteratum%253Aachs%26aulast%3DBekaii-Saab%26aufirst%3DT.%2BS.%26aulast%3DStarodub%26aufirst%3DA.%26aulast%3DEl-Rayes%26aufirst%3DB.%2BF.%26aulast%3DShahda%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DB.%2BH.%26aulast%3DNoonan%26aufirst%3DA.%2BM.%26aulast%3DNoonan%26aufirst%3DA.%2BM.%26aulast%3DShaib%26aufirst%3DW.%2BL.%26aulast%3DHanna%26aufirst%3DW.%2BT.%26aulast%3DMikhail%26aufirst%3DS.%26aulast%3DNeki%26aufirst%3DA.%2BS.%26aulast%3DChang%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DBrooks%26aufirst%3DE.%26aulast%3DOh%26aufirst%3DC.%26aulast%3DBorodyansky%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%26atitle%3DPhase%25201b%252F2%2520trial%2520of%2520cancer%2520stemness%2520inhibitor%2520napabucasin%2520%2528NAPA%2529%2520plus%2520nab-paclitaxel%2520%2528nPTX%2529%2520and%2520gemcitabine%2520%2528Gem%2529%2520in%2520metastatic%2520pancreatic%2520adenocarcinoma%2520%2528mPDAC%2529%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D36%26spage%3D4110%26epage%3D4110%26doi%3D10.1200%2FJCO.2018.36.15_suppl.4110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sonbol, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okusaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabernero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarulla, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Cutsem, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekaii-Saab, T.</span></span> <span> </span><span class="NLM_article-title">CanStem111P trial: a Phase III study of napabucasin plus nab-paclitaxel with gemcitabine</span>. <i>Future Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1295</span>– <span class="NLM_lpage">1302</span>, <span class="refDoi"> DOI: 10.2217/fon-2018-0903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.2217%2Ffon-2018-0903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=30768369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXotlWmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=1295-1302&author=M.+B.+Sonbolauthor=D.+H.+Ahnauthor=D.+Goldsteinauthor=T.+Okusakaauthor=J.+Taberneroauthor=T.+Macarullaauthor=M.+Reniauthor=C.+P.+Liauthor=B.+O%E2%80%99Neilauthor=E.+Van+Cutsemauthor=T.+Bekaii-Saab&title=CanStem111P+trial%3A+a+Phase+III+study+of+napabucasin+plus+nab-paclitaxel+with+gemcitabine&doi=10.2217%2Ffon-2018-0903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">CanStem111P trial: a Phase III study of napabucasin plus nab-paclitaxel with gemcitabine</span></div><div class="casAuthors">Sonbol, Mohamad Bassam; Ahn, Daniel H.; Goldstein, David; Okusaka, Takuji; Tabernero, Josep; Macarulla, Teresa; Reni, Michele; Li, Chung-Pin; O'Neil, Bert; Van Cutsem, Eric; Bekaii-Saab, Tanios</div><div class="citationInfo"><span class="NLM_cas:title">Future Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1295-1302</span>CODEN:
                <span class="NLM_cas:coden">FOUNBN</span>;
        ISSN:<span class="NLM_cas:issn">1744-8301</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Napabucasin (also known as BBI-608 or BBI608) is an investigational, oral agent hypothesized to inhibit multiple oncogenic pathways.  In this article, we describe the design and rationale for the CanStem111P clin. trial, a multicenter, randomized, open-label, Phase III study designed to det. the efficacy and safety of combining napabucasin with nab-paclitaxel and gemcitabine for first-line treatment of patients with metastatic pancreatic adenocarcinoma (NCT02993731).  Patients were randomized in a 1:1 fashion to receive weekly gemcitabine and nab-paclitaxel with or without napabucasin.  The results of this study will help define the role of this novel agent in the management of advanced pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY0Bk1A2JAWrVg90H21EOLACvtfcHk0lhBz5S9ZVlRCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXotlWmsbg%253D&md5=65a65f835bd6c32a16722fa919f9b5d9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2217%2Ffon-2018-0903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Ffon-2018-0903%26sid%3Dliteratum%253Aachs%26aulast%3DSonbol%26aufirst%3DM.%2BB.%26aulast%3DAhn%26aufirst%3DD.%2BH.%26aulast%3DGoldstein%26aufirst%3DD.%26aulast%3DOkusaka%26aufirst%3DT.%26aulast%3DTabernero%26aufirst%3DJ.%26aulast%3DMacarulla%26aufirst%3DT.%26aulast%3DReni%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DC.%2BP.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DB.%26aulast%3DVan%2BCutsem%26aufirst%3DE.%26aulast%3DBekaii-Saab%26aufirst%3DT.%26atitle%3DCanStem111P%2520trial%253A%2520a%2520Phase%2520III%2520study%2520of%2520napabucasin%2520plus%2520nab-paclitaxel%2520with%2520gemcitabine%26jtitle%3DFuture%2520Oncol.%26date%3D2019%26volume%3D15%26spage%3D1295%26epage%3D1302%26doi%3D10.2217%2Ffon-2018-0903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span> <i>Meet Sumitomo Dainippon
Pharma Oncology</i>; . (accessed August 1, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Meet+Sumitomo+Dainippon%0APharma+Oncology%3B+.+%28accessed+August+1%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26jtitle%3DMeet%2520Sumitomo%2520Dainippon%250APharma%2520Oncology" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span> <span> </span><span class="NLM_article-title">A novel series of napabucasin derivatives as orally active inhibitors of signal transducer and activator of transcription 3 (STAT3)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">543</span>– <span class="NLM_lpage">554</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.10.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.ejmech.2018.10.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=30472602" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit12ltLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2019&pages=543-554&author=C.+Liauthor=C.+Chenauthor=Q.+Anauthor=T.+Yangauthor=Z.+Sangauthor=Y.+Yangauthor=Y.+Juauthor=A.+Tongauthor=Y.+Luo&title=A+novel+series+of+napabucasin+derivatives+as+orally+active+inhibitors+of+signal+transducer+and+activator+of+transcription+3+%28STAT3%29&doi=10.1016%2Fj.ejmech.2018.10.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A novel series of napabucasin derivatives as orally active inhibitors of signal transducer and activator of transcription 3 (STAT3)</span></div><div class="casAuthors">Li, Chungen; Chen, Caili; An, Qi; Yang, Tao; Sang, Zitai; Yang, Yang; Ju, Yuan; Tong, Aiping; Luo, Youfu</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">543-554</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The transcription factor STAT3 is an attractive target for a variety of cancers therapy.  Napabucasin, applied in phase III clin. trials for the treatment of a variety of cancers, was regarded as one of the most promising anticancer drug by targeting STAT3.  Herein, a novel series of napabucasin derivs. were designed and synthesized, which presented a potent inhibitory activity on a variety of cancers cells.  Among the derivs. compd. 8q exhibited potent inhibitory activity on U251, HepG2, HT29 and CT26 cells with the IC50 values of 0.22, 0.49, 0.07 and 0.14 μM, resp., which was over 10-fold more potent than napabucasin.  Treatment with compd. 8q decreased protein expression level of total STAT3 and p-STAT3Y705in vitro.  The binding of compd. 8q with STAT3 were further validated by electrophoretic mobility shift assay and surface plasmon resonance anal.  Compd. 8q has a KD of 110.2 nM for full-length STAT3 recombinant protein.  Moreover, the aq. soly. of 8q was over 4.5-fold than that of napabucasin.  In addn., compd. 8q in vivo significantly reduced tumor growth compared to untreated mice, and exhibited good safety profile, indicating its great potential as an efficacious drug candidate for oncotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdjaMcoHkmCbVg90H21EOLACvtfcHk0lhP_DwkIXWuxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit12ltLrE&md5=4b95a9d078c485681ef7cba49e51acec</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.10.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.10.067%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DAn%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DSang%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DJu%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DA.%26aulast%3DLuo%26aufirst%3DY.%26atitle%3DA%2520novel%2520series%2520of%2520napabucasin%2520derivatives%2520as%2520orally%2520active%2520inhibitors%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%2520%2528STAT3%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D162%26spage%3D543%26epage%3D554%26doi%3D10.1016%2Fj.ejmech.2018.10.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and activity of BBI608 derivatives targeting on stem cells</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.ejmech.2018.03.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=29604543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmslKgtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=39-50&author=Q.+Zhouauthor=C.+Pengauthor=F.+Duauthor=L.+Zhouauthor=Y.+Shiauthor=Y.+Duauthor=D.+Liuauthor=W.+Sunauthor=M.+Zhangauthor=G.+Chen&title=Design%2C+synthesis+and+activity+of+BBI608+derivatives+targeting+on+stem+cells&doi=10.1016%2Fj.ejmech.2018.03.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and activity of BBI608 derivatives targeting on stem cells</span></div><div class="casAuthors">Zhou, Qifan; Peng, Chen; Du, Fangyu; Zhou, Linbo; Shi, Yajie; Du, Yang; Liu, Dongdong; Sun, Wenjiao; Zhang, Meixia; Chen, Guoliang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39-50</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">STAT3 plays a vital role in maintaining the self-renewal of tumor stem cells.  BBI608, a small mol. identified by its ability to inhibit gene transcription driven by STAT3 and cancer stemness properties, can inhibit stemness gene expression and kill stemness-high cancer cells isolated from a variety of cancer types.  In order to improve the pharmacokinetic properties of BBI608 and the antitumor activity, a series of BBI608 derivs. were designed and synthesized here.  Most of these compds. were more potent than BBI608 on HepG2 cells, compd. I had the most potent inhibitory activity among them and was 5.4-fold more potent than BBI608 (IC50=11.2μM), but had considerable activity on normal liver cells L-02.  Compds. II (IC50=3.5μM) and III (IC50=2.9μM) were found to possess significant inhibitory activities and good selectivity.  The results showed that compd. LD-19 was worthy to investigate further as a lead compd. according to its potent inhibitory activity, ideal ClogP value and better water soly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMlEGtqRVLqrVg90H21EOLACvtfcHk0lhP_DwkIXWuxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmslKgtbc%253D&md5=3bd9949d032ee24e433b2e34b3ca9369</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.054%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DDu%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DG.%26atitle%3DDesign%252C%2520synthesis%2520and%2520activity%2520of%2520BBI608%2520derivatives%2520targeting%2520on%2520stem%2520cells%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D39%26epage%3D50%26doi%3D10.1016%2Fj.ejmech.2018.03.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frank, D. A.</span></span> <span> </span><span class="NLM_article-title">STAT3 as a central mediator of neoplastic cellular transformation</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>251</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2006.10.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.canlet.2006.10.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=17129668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkvFGrsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=251&publication_year=2007&pages=199-210&author=D.+A.+Frank&title=STAT3+as+a+central+mediator+of+neoplastic+cellular+transformation&doi=10.1016%2Fj.canlet.2006.10.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">STAT3 as a central mediator of neoplastic cellular transformation</span></div><div class="casAuthors">Frank, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">251</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">199-210</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Much of the focus in understanding the mol. pathogenesis of tumors has centered on kinases that are activated in cancer.  However, cancers driven by a diversity of activated kinases may have very similar pathol. and clin. properties.  This likely relates to the fact that the biol. characteristics of a tumor are driven by the pattern of gene expression in that tumor, and that a wide spectrum of activating events at the cell surface and in the cytoplasm converge on a relatively small no. of transcription factors that regulate the expression of key target genes.  One transcription factor that has been found to be activated inappropriately in a wide range of human cancers is STAT3.  STAT3 target genes are involved in fundamental events of tumor development including proliferation, survival, self-renewal, invasion, and angiogenesis.  Furthermore, there is strong evidence that STAT3 is crit. for these processes, in that inhibition of STAT3 by a variety of means can exert an anti-cancer effect.  Since normal cells are relatively tolerant of interruption in STAT3 signaling, these findings suggest that STAT3 may also be an excellent target for the mol. therapy of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo20EchEwndc7Vg90H21EOLACvtfcHk0lhP_DwkIXWuxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkvFGrsLk%253D&md5=f97845a4bd961469b5f9cecc5b02df93</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2006.10.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2006.10.017%26sid%3Dliteratum%253Aachs%26aulast%3DFrank%26aufirst%3DD.%2BA.%26atitle%3DSTAT3%2520as%2520a%2520central%2520mediator%2520of%2520neoplastic%2520cellular%2520transformation%26jtitle%3DCancer%2520Lett.%26date%3D2007%26volume%3D251%26spage%3D199%26epage%3D210%26doi%3D10.1016%2Fj.canlet.2006.10.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">STAT proteins: novel molecular targets for cancer drug discovery</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6613</span>– <span class="NLM_lpage">6626</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1204086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fsj.onc.1204086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=11426647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhsF2rsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=6613-6626&author=J.+Turksonauthor=R.+Jove&title=STAT+proteins%3A+novel+molecular+targets+for+cancer+drug+discovery&doi=10.1038%2Fsj.onc.1204086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">STAT proteins: novel molecular targets for cancer drug discovery</span></div><div class="casAuthors">Turkson, James; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">56</span>),
    <span class="NLM_cas:pages">6613-6626</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with 101 refs.  Signal Transducers and Activators of Transcription (STATs) are a family of cytoplasmic proteins with roles as signal messengers and transcription factors that participate in normal cellular responses to cytokines and growth factors.  Frequently, however, abnormal activity of certain STAT family members, particularly Stat3 and Stat5, is assocd. with a wide variety of human malignancies, including hematol., breast, head and neck, and prostate cancers.  Application of mol. biol. and pharmacol. tools in disease-relevant models has confirmed Stat3 as having a causal role in oncogenesis, and provided validation of Stat3 as a target for cancer drug discovery and therapeutic intervention.  Furthermore, a constitutively-active mutant form of Stat3 is sufficient to induce oncogenic transformation of cells, which form tumors in vivo.  Constitutive activation of Stat3 signaling is accompanied by upregulation of cyclin D1, c-Myc, and Bcl-x, changes consistent with subversion of normal cellular growth and survival control mechanisms.  Block of constitutive Stat3 signaling results in growth inhibition and apoptosis of Stat3-pos. tumor cells in vitro and in vivo.  The obsd. dependence of certain tumors on constitutive Stat3 signaling for growth and survival has wide implications for cancer therapy, offering the potential for preferential tumor cell killing.  This review evaluates constitutive Stat3 activation as a "cancer-causing" factor, and proposes a no. of mol. strategies for targeting Stat3 signaling for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoffTICt_aLhLVg90H21EOLACvtfcHk0lhP_DwkIXWuxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhsF2rsro%253D&md5=2011fb621ef70547ddbbef2ed253df98</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1204086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1204086%26sid%3Dliteratum%253Aachs%26aulast%3DTurkson%26aufirst%3DJ.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DSTAT%2520proteins%253A%2520novel%2520molecular%2520targets%2520for%2520cancer%2520drug%2520discovery%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D6613%26epage%3D6626%26doi%3D10.1038%2Fsj.onc.1204086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bromberg, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrzeszczynska, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestell, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanese, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Stat3 as an oncogene</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">303</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)81959-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2FS0092-8674%2800%2981959-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10458605" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADyaK1MXlt1Cgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1999&pages=295-303&author=J.+F.+Brombergauthor=M.+H.+Wrzeszczynskaauthor=G.+Devganauthor=Y.+Zhaoauthor=R.+G.+Pestellauthor=C.+Albaneseauthor=J.+E.+Darnell&title=Stat3+as+an+oncogene&doi=10.1016%2FS0092-8674%2800%2981959-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Stat3 as an oncogene</span></div><div class="casAuthors">Bromberg, Jacqueline F.; Wrzeszczynska, Melissa H.; Devgan, Geeta; Zhao, Yanxiang; Pestell, Richard G.; Albanese, Chris; Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">295-303</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">STATs are latent transcription factors that mediate cytokine- and growth factor-directed transcription.  In many human cancers and transformed cell lines, Stat3 is persistently activated, and in cell culture, active Stat3 is either required for transformation, enhances transformation, or blocks apoptosis.  Here, the substitution of two cysteine residues within the C-terminal loop of the SH2 domain of Stat3 produced a mol. that dimerizes spontaneously, binds to DNA, and activates transcription.  The Stat3-C mol. in immortalized fibroblasts causes cellular transformation scored by colony formation in soft agar and tumor formation in nude mice.  Thus, the activated Stat3 mol. by itself can mediate cellular transformation and the expts. focus attention on the importance of constitutive Stat3 activation in human tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOPJVvUEeTbVg90H21EOLACvtfcHk0ljLTIdXX7scCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlt1Cgt7s%253D&md5=f2e69e246bbfae15a3c5ee7b94340300</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981959-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981959-5%26sid%3Dliteratum%253Aachs%26aulast%3DBromberg%26aufirst%3DJ.%2BF.%26aulast%3DWrzeszczynska%26aufirst%3DM.%2BH.%26aulast%3DDevgan%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DPestell%26aufirst%3DR.%2BG.%26aulast%3DAlbanese%26aufirst%3DC.%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DStat3%2520as%2520an%2520oncogene%26jtitle%3DCell%26date%3D1999%26volume%3D98%26spage%3D295%26epage%3D303%26doi%3D10.1016%2FS0092-8674%2800%2981959-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Löcken, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clamor, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, K.</span></span> <span> </span><span class="NLM_article-title">Napabucasin and related heterocycle-fused naphthoquinones as STAT3 inhibitors with antiproliferative activity against cancer cells</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">1636</span>– <span class="NLM_lpage">1644</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.8b00247</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.8b00247" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FnslKisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=1636-1644&author=H.+L%C3%B6ckenauthor=C.+Clamorauthor=K.+M%C3%BCller&title=Napabucasin+and+related+heterocycle-fused+naphthoquinones+as+STAT3+inhibitors+with+antiproliferative+activity+against+cancer+cells&doi=10.1021%2Facs.jnatprod.8b00247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Napabucasin and Related Heterocycle-Fused Naphthoquinones as STAT3 Inhibitors with Antiproliferative Activity against Cancer Cells</span></div><div class="casAuthors">Locken Hauke; Clamor Cinzia; Muller Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of natural products</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1636-1644</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Napabucasin (6) and its angularly anellated isomer (7), for which the synthesis is described, together with related plant-derived naphthoquinones, were evaluated in vitro against human breast cancer (MDA-MB-231) and chronic myelogenous leukemia (K562) cells.  As observed for β-lapachone (3), the active naphthoquinones all induced apoptosis in a cell-cycle-independent fashion.  In contrast to the pyran-fused β-lapachone (3), however, the most potent furan-fused naphthoquinones were able to redox cycle and generate superoxide in cell-based assays, which was independent of NAD(P)H:quinone oxido-reductase 1.  In a homogeneous time-resolved fluorescence (HTRF) assays with MDA-MB-231 cells, both napabucasin (6) and isonapabucasin (7) were identified as targeting STAT3 phosphorylation.  In addition, drug affinity responsive target stability assays were performed to validate a direct interaction of the naphthoquinones with STAT3.  Isonapabucasin (7) turned out to be twice as potent against STAT3 as napabucasin (6) in the HTRF assay, with an EC50 in the submicromolar range, which was in excellent agreement with the potency of both agents to inhibit the growth of MDA-MB-231 cells.  Moreover, molecular docking experiments predicted different binding modes to the STAT3 SH2 domain for the linearly anellated napabucasin (6) and its angularly anellated isomer (7).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTSQACUhahgsEEgZCMqtRBifW6udTcc2eah5hqG_ebXXrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FnslKisw%253D%253D&md5=f7b8eca5ba22354ee3c13457409262f4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.8b00247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.8b00247%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B6cken%26aufirst%3DH.%26aulast%3DClamor%26aufirst%3DC.%26aulast%3DM%25C3%25BCller%26aufirst%3DK.%26atitle%3DNapabucasin%2520and%2520related%2520heterocycle-fused%2520naphthoquinones%2520as%2520STAT3%2520inhibitors%2520with%2520antiproliferative%2520activity%2520against%2520cancer%2520cells%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2018%26volume%3D81%26spage%3D1636%26epage%3D1644%26doi%3D10.1021%2Facs.jnatprod.8b00247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rindtorff, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutros, M.</span></span> <span> </span><span class="NLM_article-title">Wnt signaling in cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1461</span>– <span class="NLM_lpage">1473</span>, <span class="refDoi"> DOI: 10.1038/onc.2016.304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fonc.2016.304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=27617575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2gsL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1461-1473&author=T.+Zhanauthor=N.+Rindtorffauthor=M.+Boutros&title=Wnt+signaling+in+cancer&doi=10.1038%2Fonc.2016.304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Wnt signaling in cancer</span></div><div class="casAuthors">Zhan, T.; Rindtorff, N.; Boutros, M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1461-1473</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Wnt signaling is one of the key cascades regulating development and stemness, and has also been tightly assocd. with cancer.  The role of Wnt signaling in carcinogenesis has most prominently been described for colorectal cancer, but aberrant Wnt signaling is obsd. in many more cancer entities.  Here, we review current insights into novel components of Wnt pathways and describe their impact on cancer development.  Furthermore, we highlight expanding functions of Wnt signaling for both solid and liq. tumors.  We also describe current findings how Wnt signaling affects maintenance of cancer stem cells, metastasis and immune control.  Finally, we provide an overview of current strategies to antagonize Wnt signaling in cancer and challenges that are assocd. with such approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5DnHKAd0M3rVg90H21EOLACvtfcHk0limGKdlSX54hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2gsL3O&md5=abef967a4e295e8d997ea2f241b035bc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.304%26sid%3Dliteratum%253Aachs%26aulast%3DZhan%26aufirst%3DT.%26aulast%3DRindtorff%26aufirst%3DN.%26aulast%3DBoutros%26aufirst%3DM.%26atitle%3DWnt%2520signaling%2520in%2520cancer%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D1461%26epage%3D1473%26doi%3D10.1038%2Fonc.2016.304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, A.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogoff, H. A.</span></span> <span> </span><span class="NLM_article-title">Evaluation of tumor cell-tumor microenvironment component interactions as potential predictors of patient response to napabucasin</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1429</span>– <span class="NLM_lpage">1434</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-18-1242</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1158%2F1541-7786.MCR-18-1242" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31043490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1KjsL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2019&pages=1429-1434&author=A.-Y.+Changauthor=E.+Hsuauthor=J.+Patelauthor=Y.+Liauthor=M.+Zhangauthor=H.+Iguchiauthor=H.+A.+Rogoff&title=Evaluation+of+tumor+cell-tumor+microenvironment+component+interactions+as+potential+predictors+of+patient+response+to+napabucasin&doi=10.1158%2F1541-7786.MCR-18-1242"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of tumor cell-tumor microenvironment component interactions as potential predictors of patient response to napabucasin</span></div><div class="casAuthors">Chang, An-Yun; Hsu, Eric; Patel, Jaimin; Li, Yiqun; Zhang, Minjie; Iguchi, Haruhisa; Rogoff, Harry A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1429-1434</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Napabucasin is an NAD(P)H:quinone oxidoreductase 1 (NQO1)-bioactivatable small mol. hypothesized to affect multiple oncogenic pathways.  In a prespecified, retrospective anal. of the napabucasin phase III CO.23 study, overall survival was longer for napabucasin vs. placebo in patients expressing phosphorylated STAT3 (pSTAT3) in tumor cells and cells of the tumor microenvironment (TME).  We hypothesized that a connection may exist between NQO1 expression in cancer cells and pSTAT3 in tumor cells and the TME.  In 3D spheroid cocultures of cancer cells and cancer-assocd. fibroblasts, the antitumor activity of napabucasin was NQO1 dependent.  The levels of cytokines such as IL6, CXCL10, and GM-CSF were higher in NQO1-pos. vs. NQO1-deleted cocultures.  These differentially secreted cytokines promoted STAT3 phosphorylation in tumor cells and the TME.  NQO1-expressing, napabucasin-sensitive tumor cells can modify tumor cells and the TME to promote STAT3 phosphorylation, suggesting that pSTAT3 may be used to identify a subpopulation of patients who would likely respond to napabucasin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRjLHe-89V9bVg90H21EOLACvtfcHk0limGKdlSX54hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1KjsL3O&md5=2788fe3d367fd97f4e1efb447be5d225</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-18-1242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-18-1242%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DA.-Y.%26aulast%3DHsu%26aufirst%3DE.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DIguchi%26aufirst%3DH.%26aulast%3DRogoff%26aufirst%3DH.%2BA.%26atitle%3DEvaluation%2520of%2520tumor%2520cell-tumor%2520microenvironment%2520component%2520interactions%2520as%2520potential%2520predictors%2520of%2520patient%2520response%2520to%2520napabucasin%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2019%26volume%3D17%26spage%3D1429%26epage%3D1434%26doi%3D10.1158%2F1541-7786.MCR-18-1242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parkinson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Deoxynyboquinones as NQO1-activated cancer therapeutics</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2715</span>– <span class="NLM_lpage">2723</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.5b00365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.5b00365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wqs7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=2715-2723&author=E.+I.+Parkinsonauthor=P.+J.+Hergenrother&title=Deoxynyboquinones+as+NQO1-activated+cancer+therapeutics&doi=10.1021%2Facs.accounts.5b00365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Deoxynyboquinones as NQO1-Activated Cancer Therapeutics</span></div><div class="casAuthors">Parkinson, Elizabeth I.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2715-2723</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the major goals of cancer therapy is the selective targeting of cancer cells over normal cells.  Unfortunately, even with recent advances, the majority of chemotherapeutics still indiscriminately kill all rapidly dividing cells.  Although these drugs are effective in certain settings, their inability to specifically target cancer results in significant dose-limiting toxicities.  One way to avoid such toxicities is to target an aspect of the cancer cell that is not shared by normal cells.  A potential cancer-specific target is the enzyme NAD(P)H quinone oxidoreductase 1 (NQO1).  NQO1 is a 2-electron reductase responsible for the detoxification of quinones.  Its expression is typically quite low in normal tissue, but it has been found to be greatly overexpressed in many types of solid tumors, including lung, breast, pancreatic, and colon cancers.  This overexpression is thought to be in response to the higher oxidative stress of the cancer cell, and it is possible that NQO1 contributes to tumor progression.  The overexpression of NQO1 and its correlation with poor patient outcome make it an intriguing target.  Although some have explored inhibiting NQO1 as an anticancer strategy, this has generally been unsuccessful.  A more promising strategy is to utilize NQO1 substrates that are activated upon redn. by NQO1.  For example, in principle, redn. of a quinone can result in a hydroquinone that is a DNA alkylator, protein inhibitor, or redn.-oxidn. cycler.  Although there are many proposed NQO1 substrates, head-to-head assays reveal only two classes of compds. that convincingly induce cancer cell death through NQO1-mediated activation.  The authors describe the discovery and development of one of these compds., the natural product deoxynyboquinone (DNQ), an excellent NQO1 substrate and anticancer agent.  A modular synthesis of DNQ was developed that enabled access to the large compd. quantities needed to conduct extensive mechanistic evaluations and animal expts.  During these evaluations, the authors found that DNQ is an outstanding NQO1 substrate that is processed much more efficiently than other putative NQO1 substrates.  Importantly, its anticancer activity is strictly dependent on the overexpression of active NQO1.  Using previous crystal structures of NQO1, novel DNQ derivs. were designed that are also excellent NQO1 substrates and possess properties that make them more attractive than the parent natural product for translational development.  Given their selectivity, potency, outstanding pharmacokinetic properties, and the ready availability of diagnostics to assess NQO1 in patients, DNQ and its derivs. have considerable potential as personalized medicines for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW3MluuGsMA7Vg90H21EOLACvtfcHk0limGKdlSX54hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wqs7fI&md5=9395e54798f6a41a31b47a5ad078c9c1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.5b00365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.5b00365%26sid%3Dliteratum%253Aachs%26aulast%3DParkinson%26aufirst%3DE.%2BI.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DDeoxynyboquinones%2520as%2520NQO1-activated%2520cancer%2520therapeutics%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26spage%3D2715%26epage%3D2723%26doi%3D10.1021%2Facs.accounts.5b00365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Froeling, F. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swamynathan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deschênes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chio, I. I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosnan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alagesan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, A.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trotman, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belleau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogoff, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuveson, D. A.</span></span> <span> </span><span class="NLM_article-title">Bioactivation of napabucasin triggers reactive oxygen species-mediated cancer cell death</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">7162</span>– <span class="NLM_lpage">7174</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-19-0302</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1158%2F1078-0432.CCR-19-0302" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31527169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWitLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=7162-7174&author=F.+E.+M.+Froelingauthor=M.+M.+Swamynathanauthor=A.+Desch%C3%AAnesauthor=I.+I.+C.+Chioauthor=E.+Brosnanauthor=M.+A.+Yaoauthor=P.+Alagesanauthor=M.+Lucitoauthor=J.+Y.+Liauthor=A.-Y.+Changauthor=L.+C.+Trotmanauthor=P.+Belleauauthor=Y.+Parkauthor=H.+A.+Rogoffauthor=J.+D.+Watsonauthor=D.+A.+Tuveson&title=Bioactivation+of+napabucasin+triggers+reactive+oxygen+species-mediated+cancer+cell+death&doi=10.1158%2F1078-0432.CCR-19-0302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactivation of napabucasin triggers reactive oxygen species-mediated cancer cell death</span></div><div class="casAuthors">Froeling, Fieke E. M.; Swamynathan, Manojit Mosur; Deschenes, Astrid; Chio, Iok In Christine; Brosnan, Erin; Yao, Melissa A.; Alagesan, Priya; Lucito, Matthew; Li, Juying; Chang, An-Yun; Trotman, Lloyd C.; Belleau, Pascal; Park, Youngkyu; Rogoff, Harry A.; Watson, James D.; Tuveson, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7162-7174</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Napabucasin (2-acetylfuro-1,4-naphthoquinone or BBI-608) is a small mol. currently being clin. evaluated in various cancer types.  It has mostly been recognized for its ability to inhibit STAT3 signaling.  However, based on its chem. structure, we hypothesized that napabucasin is a substrate for intracellular oxidoreductases and therefore may exert its anticancer effect through redox cycling, resulting in reactive oxygen species (ROS) prodn. and cell death.  Exptl. Design: Binding of napabucasin to NAD(P)H:quinone oxidoreductase-1 (NQO1), and other oxidoreductases, was measured.  Pancreatic cancer cell lines were treated with napabucasin, and cell survival, ROS generation, DNA damage, transcriptomic changes, and alterations in STAT3 activation were assayed in vitro and in vivo.  Genetic knockout or pharmacol. inhibition with dicoumarol was used to evaluate the dependency on NQO1.  Results: Napabucasin was found to bind with high affinity to NQO1 and to a lesser degree to cytochrome P 450 oxidoreductase (POR).  Treatment resulted in marked induction of ROS and DNA damage with an NQO1- and ROS-dependent decrease in STAT3 phosphorylation.  Differential cytotoxic effects were obsd., where NQO1-expressing cells generating cytotoxic levels of ROS at low napabucasin concns. were more sensitive.  Cells with low or no baseline NQO1 expression also produced ROS in response to napabucasin, albeit to a lesser extent, through the one-electron reductase POR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobkyO7Js1b7LVg90H21EOLACvtfcHk0liaWfZkwWEwYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWitLvN&md5=f2bbfdc3e8673c095f1118e361e6f81d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-19-0302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-19-0302%26sid%3Dliteratum%253Aachs%26aulast%3DFroeling%26aufirst%3DF.%2BE.%2BM.%26aulast%3DSwamynathan%26aufirst%3DM.%2BM.%26aulast%3DDesch%25C3%25AAnes%26aufirst%3DA.%26aulast%3DChio%26aufirst%3DI.%2BI.%2BC.%26aulast%3DBrosnan%26aufirst%3DE.%26aulast%3DYao%26aufirst%3DM.%2BA.%26aulast%3DAlagesan%26aufirst%3DP.%26aulast%3DLucito%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%2BY.%26aulast%3DChang%26aufirst%3DA.-Y.%26aulast%3DTrotman%26aufirst%3DL.%2BC.%26aulast%3DBelleau%26aufirst%3DP.%26aulast%3DPark%26aufirst%3DY.%26aulast%3DRogoff%26aufirst%3DH.%2BA.%26aulast%3DWatson%26aufirst%3DJ.%2BD.%26aulast%3DTuveson%26aufirst%3DD.%2BA.%26atitle%3DBioactivation%2520of%2520napabucasin%2520triggers%2520reactive%2520oxygen%2520species-mediated%2520cancer%2520cell%2520death%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2019%26volume%3D25%26spage%3D7162%26epage%3D7174%26doi%3D10.1158%2F1078-0432.CCR-19-0302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sechi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">A novel redox modulator induces a GPX4-mediated cell death that is dependent on iron and reactive oxygen species</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9838</span>– <span class="NLM_lpage">9855</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01016</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1aqsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9838-9855&author=S.+Huauthor=M.+Sechiauthor=P.+K.+Singhauthor=L.+Daiauthor=S.+McCannauthor=D.+Sunauthor=M.+Ljungmanauthor=N.+Neamati&title=A+novel+redox+modulator+induces+a+GPX4-mediated+cell+death+that+is+dependent+on+iron+and+reactive+oxygen+species&doi=10.1021%2Facs.jmedchem.0c01016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Redox Modulator Induces a GPX4-Mediated Cell Death That Is Dependent on Iron and Reactive Oxygen Species</span></div><div class="casAuthors">Hu, Shuai; Sechi, Mario; Singh, Pankaj Kumar; Dai, Lipeng; McCann, Sean; Sun, Duxin; Ljungman, Mats; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9838-9855</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthesis and biol. evaluation of a quinazoline-dione derivs. I [R = 4-cyclohexylphenyl, 9H-fluoren-2-yl, biphenyl-3-yl, etc.; R1 = H, Me] as reactive oxygen species (ROS) inducer and showing significant cytotoxicity in pancreatic cancer cells was reported.  Compd. I [R = 4-(4-methylpiperazin-1-yl)phenyl; R1 = H (II)] shared a highly similar transcriptomic profile to napabucasin, inhibited STAT3 phosphorylation, increased ROS levels and induced DNA damage.  However, while napabucasin caused necroptosis, compd. II induced an iron-, ROS-dependent and GPX4-mediated cell death.  Importantly, compd. II decreased the expression of LRPPRC and PNPT1; two proteins involved in mitochondrial RNA catabolic processes and both neg. correlated with the overall survival of pancreatic cancer patients.  Pharmacokinetics-guided lead optimization resulted in compd. I [R = 4-(4-methylpiperazin-1-yl)phenyl; R1 = Me (III)], which showed improved plasma stability and reduced toxicity in mice compared to compd. II.  Overall, compds. II and III represent novel ROS-inducing drug-like compds. warranting further development for the treatment of pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp25T74AYyvlbVg90H21EOLACvtfcHk0liaWfZkwWEwYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1aqsLfF&md5=7b206e886439efb9c0b045de5b60621a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01016%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DS.%26aulast%3DSechi%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DP.%2BK.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DMcCann%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DLjungman%26aufirst%3DM.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DA%2520novel%2520redox%2520modulator%2520induces%2520a%2520GPX4-mediated%2520cell%2520death%2520that%2520is%2520dependent%2520on%2520iron%2520and%2520reactive%2520oxygen%2520species%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9838%26epage%3D9855%26doi%3D10.1021%2Facs.jmedchem.0c01016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shogren, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jewison, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waletzki, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, A. L.,  II</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaszemski, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maran, A.</span></span> <span> </span><span class="NLM_article-title">Inhibition of STAT3 blocks protein synthesis and tumor metastasis in osteosarcoma cells</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">244</span>, <span class="refDoi"> DOI: 10.1186/s13046-018-0914-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1186%2Fs13046-018-0914-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=30286779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjvVCjtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2018&pages=244&author=D.+Zuoauthor=K.+L.+Shogrenauthor=J.+Zangauthor=D.+E.+Jewisonauthor=B.+E.+Waletzkiauthor=A.+L.+Millerauthor=S.+H.+Okunoauthor=Z.+Caiauthor=M.+J.+Yaszemskiauthor=A.+Maran&title=Inhibition+of+STAT3+blocks+protein+synthesis+and+tumor+metastasis+in+osteosarcoma+cells&doi=10.1186%2Fs13046-018-0914-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of STAT3 blocks protein synthesis and tumor metastasis in osteosarcoma cells</span></div><div class="casAuthors">Zuo, Dongqing; Shogren, Kristen L.; Zang, Jie; Jewison, Donna E.; Waletzki, Brian E.; Miller, Alan L, II; Okuno, Scott H.; Cai, Zhengdong; Yaszemski, Michael J.; Maran, Avudaiappan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental & Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">244/1-244/11</span>CODEN:
                <span class="NLM_cas:coden">JECRDN</span>;
        ISSN:<span class="NLM_cas:issn">1756-9966</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Osteosarcoma is the most common bone cancer.  Despite advances, mol. mechanisms assocd. with osteosarcoma have not been fully understood.  Hence, an effective treatment for osteosarcoma has yet to be developed.  Even though signal transducer and activator of transcription3 (STAT3) has been implicated, its role in pathogenesis of osteosarcoma is not fully detd.  In this study, we investigated the antitumor effect of napabucasin (NP) (BBI608), an inhibitor of STAT3 on osteosarcoma in vitro and in vivo and studied the underlying mol. mechanism.  Methods: Cell viability, colony formation, apoptosis, tumor growth and metastasis assays were performed to examine the effect of NP on osteosarcoma in vitro and in vivo.  Real-time RT-PCR, western anal., immunofluorescence and reporter assays were used to monitor the expression and activity of proteins and underlying mol. pathways.  Protein synthesis, co-immunopptn. and CAP binding assays were carried out to understand NP-mediated mechanism of actions in osteosarcoma cells.  Results: Our results show that NP treatment decreases cell viability and induces apoptosis in several osteosarcoma cell lines.  NP treatment suppresses both expression and phosphorylation of STAT3 in addn. to blocking STAT3-mediated transcription and downstream target proteins in osteosarcoma cells.  Furthermore, NP inhibits protein synthesis through regulation of the eukaryotic initiation factor 4E (eIF4E) and eIF4E-binding protein 1 (4E-BP1).  NP also inhibits the progression of osteosarcoma tumors and metastasis in vivo in an orthotopic tibial model of osteosarcoma.  Conclusions: Taken together, our investigation reveals that NP acts through a novel mechanism and inhibits osteosarcoma growth and metastasis, and could be investigated clin. for treating osteosarcoma patients alone or in combination with other drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov_lXwQAlLabVg90H21EOLACvtfcHk0liWNsJ1gDSRQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjvVCjtr4%253D&md5=05bb3251ce5c49c4f64a2983f747d6e5</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1186%2Fs13046-018-0914-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-018-0914-0%26sid%3Dliteratum%253Aachs%26aulast%3DZuo%26aufirst%3DD.%26aulast%3DShogren%26aufirst%3DK.%2BL.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DJewison%26aufirst%3DD.%2BE.%26aulast%3DWaletzki%26aufirst%3DB.%2BE.%26aulast%3DMiller%26aufirst%3DA.%2BL.%26aulast%3DOkuno%26aufirst%3DS.%2BH.%26aulast%3DCai%26aufirst%3DZ.%26aulast%3DYaszemski%26aufirst%3DM.%2BJ.%26aulast%3DMaran%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520STAT3%2520blocks%2520protein%2520synthesis%2520and%2520tumor%2520metastasis%2520in%2520osteosarcoma%2520cells%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2018%26volume%3D37%26spage%3D244%26doi%3D10.1186%2Fs13046-018-0914-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.-H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0liWNsJ1gDSRQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule control of intracellular protein levels through modulation of the ubiquitin proteasome system</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2330</span>, <span class="refDoi"> DOI: 10.1002/anie.201307761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1002%2Fanie.201307761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlegsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=2312-2330&author=D.+L.+Buckleyauthor=C.+M.+Crews&title=Small-molecule+control+of+intracellular+protein+levels+through+modulation+of+the+ubiquitin+proteasome+system&doi=10.1002%2Fanie.201307761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Control of Intracellular Protein Levels through Modulation of the Ubiquitin Proteasome System</span></div><div class="casAuthors">Buckley, Dennis L.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2312-2330</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Traditionally, biol. probes and drugs have targeted the activities of proteins (such as enzymes and receptors) that can be readily controlled by small mols.  The remaining majority of the proteome has been deemed "undruggable".  By using small-mol. modulators of the ubiquitin proteasome, protein levels, rather than protein activity, can be targeted instead, thus increasing the no. of druggable targets.  Whereas targeting of the proteasome itself can lead to a global increase in protein levels, the targeting of other components of the UPS (e.g., the E3 ubiquitin ligases) can lead to an increase in protein levels in a more targeted fashion.  Alternatively, multiple strategies for inducing protein degrdn. with small-mol. probes are emerging.  With the ability to induce and inhibit the degrdn. of targeted proteins, small-mol. modulators of the UPS have the potential to significantly expand the druggable portion of the proteome beyond traditional targets, such as enzymes and receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUSadJB1NNJ7Vg90H21EOLACvtfcHk0liWNsJ1gDSRQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlegsLo%253D&md5=4d88e9c3f2dbf92c9db4aa81bd8cdac1</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201307761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201307761%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520control%2520of%2520intracellular%2520protein%2520levels%2520through%2520modulation%2520of%2520the%2520ubiquitin%2520proteasome%2520system%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D2312%26epage%3D2330%26doi%3D10.1002%2Fanie.201307761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: from chemical biology to drug discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.chembiol.2017.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=28648379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1181-1190&author=P.+M.+Crommauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+from+chemical+biology+to+drug+discovery&doi=10.1016%2Fj.chembiol.2017.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span></div><div class="casAuthors">Cromm, Philipp M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Traditional pharmaceutical drug discovery is almost exclusively focused on directly controlling protein activity to cure diseases.  Modulators of protein activity, esp. inhibitors, are developed and applied at high concn. to achieve maximal effects.  Thereby, reduced bioavailability and off-target effects can hamper compd. efficacy.  Nucleic acid-based strategies that control protein function by affecting expression have emerged as an alternative.  However, metabolic stability and broad bioavailability represent development hurdles that remain to be overcome for these approaches.  More recently, utilizing the cell's own protein destruction machinery for selective degrdn. of essential drivers of human disorders has opened up a new and exciting area of drug discovery.  Small-mol.-induced proteolysis of selected substrates offers the potential of reaching beyond the limitations of the current pharmaceutical paradigm to expand the druggable target space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrosuhmnPE0RrVg90H21EOLACvtfcHk0lgPlDAO8K80bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO&md5=3c8c6d9488396d5eb08ca51cee20bbf4</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520from%2520chemical%2520biology%2520to%2520drug%2520discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1181%26epage%3D1190%26doi%3D10.1016%2Fj.chembiol.2017.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A potent and selective small-molecule degrader of STAT3 achieves complete tumor regression in vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">511.e17</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.ccell.2019.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31715132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=498-511.e17&author=L.+Baiauthor=H.+Zhouauthor=R.+Xuauthor=Y.+Zhaoauthor=K.+Chinnaswamyauthor=D.+McEachernauthor=J.+Chenauthor=C.-Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=H.+Jiangauthor=B.+Wenauthor=P.+Kumarauthor=J.+L.+Meagherauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=A+potent+and+selective+small-molecule+degrader+of+STAT3+achieves+complete+tumor+regression+in+vivo&doi=10.1016%2Fj.ccell.2019.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Haibin; Xu, Renqi; Zhao, Yujun; Chinnaswamy, Krishnapriya; McEachern, Donna; Chen, Jianyong; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Jiang, Hui; Wen, Bo; Kumar, Praveen; Meagher, Jennifer L.; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">498-511.e17</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is an attractive cancer therapeutic target.  Here we report the discovery of SD-36(I), a small-mol. degrader of STAT3.  SD-36 potently induces the degrdn. of STAT3 protein in vitro and in vivo and demonstrates high selectivity over other STAT members.  Induced degrdn. of STAT3 results in a strong suppression of its transcription network in leukemia and lymphoma cells.  SD-36 inhibits the growth of a subset of acute myeloid leukemia and anaplastic large-cell lymphoma cell lines by inducing cell-cycle arrest and/or apoptosis.  SD-36 achieves complete and long-lasting tumor regression in multiple xenograft mouse models at well-tolerated dose schedules.  Degrdn. of STAT3 protein, therefore, is a promising cancer therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUaP49Ap9DxLVg90H21EOLACvtfcHk0lgPlDAO8K80bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO&md5=71252c17d50a1a8be829352942175bd6</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520potent%2520and%2520selective%2520small-molecule%2520degrader%2520of%2520STAT3%2520achieves%2520complete%2520tumor%2520regression%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D498%26epage%3D511.e17%26doi%3D10.1016%2Fj.ccell.2019.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of SD-36 as a potent, selective, and efficacious PROTAC degrader of STAT3 protein</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11280</span>– <span class="NLM_lpage">11300</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01530</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01530" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11280-11300&author=H.+Zhouauthor=L.+Baiauthor=R.+Xuauthor=Y.+Zhaoauthor=J.+Chenauthor=D.+McEachernauthor=K.+Chinnaswamyauthor=B.+Wenauthor=L.+Daiauthor=P.+Kumarauthor=C.-Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=J.+L.+Meagherauthor=H.+Yiauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-based+discovery+of+SD-36+as+a+potent%2C+selective%2C+and+efficacious+PROTAC+degrader+of+STAT3+protein&doi=10.1021%2Facs.jmedchem.9b01530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein</span></div><div class="casAuthors">Zhou, Haibin; Bai, Longchuan; Xu, Renqi; Zhao, Yujun; Chen, Jianyong; McEachern, Donna; Chinnaswamy, Krishnapriya; Wen, Bo; Dai, Lipeng; Kumar, Praveen; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Meagher, Jennifer L.; Yi, Han; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11280-11300</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is a transcription factor and an attractive therapeutic target for cancer and other human diseases.  Despite twenty-years of persistent research efforts, targeting STAT3 has been very challenging.  The authors report herein the structure-based discovery of potent small-mol. STAT3 degraders based upon the proteolysis targeting chimera (PROTAC) concept.  The authors first designed SI-109 as a potent, small-mol. inhibitor of the STAT3 SH2 domain.  Employing ligands for cereblon/Cullin 4A E3 ligase and SI-109, the authors obtained a series of potent PROTAC STAT3 degraders, exemplified by SD-36.  SD-36 induces rapid STAT3 degrdn. at low nanomolar concns. in cells and fails to degrader other STAT proteins.  SD-36 achieves nanomolar cell growth inhibitory activity in leukemia and lymphoma cell lines with high levels of phosphorylated STAT3.  A single dose of SD-36 results in complete STAT3 protein degrdn. in xenograft tumor tissue and normal mouse tissues.  SD-36 achieves complete and long-lasting tumor regression in the Molm-16 xenograft tumor model at well-tolerated dose-schedules.  SD-36 is a potent, selective and efficacious STAT3 degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIAWd1hiuqy7Vg90H21EOLACvtfcHk0lgPlDAO8K80bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J&md5=7d0c656a2d96196a3b71829c9efe715a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01530%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520discovery%2520of%2520SD-36%2520as%2520a%2520potent%252C%2520selective%252C%2520and%2520efficacious%2520PROTAC%2520degrader%2520of%2520STAT3%2520protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11280%26epage%3D11300%26doi%3D10.1021%2Facs.jmedchem.9b01530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alabi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitale, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Androgen receptor degradation by the proteolysis-targeting chimera ARCC-4 outperforms enzalutamide in cellular models of prostate cancer drug resistance</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">100</span>, <span class="refDoi"> DOI: 10.1038/s42003-018-0105-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fs42003-018-0105-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=30271980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BB3czksVCksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=100&author=J.+Salamiauthor=S.+Alabiauthor=R.+R.+Willardauthor=N.+J.+Vitaleauthor=J.+Wangauthor=H.+Dongauthor=M.+Jinauthor=D.+P.+McDonnellauthor=A.+P.+Crewauthor=T.+K.+Neklesaauthor=C.+M.+Crews&title=Androgen+receptor+degradation+by+the+proteolysis-targeting+chimera+ARCC-4+outperforms+enzalutamide+in+cellular+models+of+prostate+cancer+drug+resistance&doi=10.1038%2Fs42003-018-0105-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Androgen receptor degradation by the proteolysis-targeting chimera ARCC-4 outperforms enzalutamide in cellular models of prostate cancer drug resistance</span></div><div class="casAuthors">Salami Jemilat; Crews Craig M; Alabi Shanique; Crews Craig M; Willard Ryan R; Vitale Nick J; Wang Jing; Dong Hanqing; Jin Meizhong; Crew Andrew P; Neklesa Taavi K; McDonnell Donald P; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Communications biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The androgen receptor is a major driver of prostate cancer and inhibition of its transcriptional activity using competitive antagonists, such as enzalutamide remains a frontline therapy for prostate cancer management.  However, the majority of patients eventually develop drug resistance.  We propose that targeting the androgen receptor for degradation via Proteolysis Targeting Chimeras (PROTACs) will be a better therapeutic strategy for targeting androgen receptor signaling in prostate cancer cells.  Here we perform a head-to-head comparison between a currently approved androgen receptor antagonist enzalutamide, and its PROTAC derivative, ARCC-4, across different cellular models of prostate cancer drug resistance.  ARCC-4 is a low-nanomolar androgen receptor degrader able to degrade about 95% of cellular androgen receptors.  ARCC-4 inhibits prostate tumor cell proliferation, degrades clinically relevant androgen receptor point mutants and unlike enzalutamide, retains antiproliferative effect in a high androgen environment.  Thus, ARCC-4 exemplifies how protein degradation can address the drug resistance hurdles of enzalutamide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQilKiHIro9_JI8NvIb08rbfW6udTcc2eY23w6L7aYVs7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czksVCksA%253D%253D&md5=b8b7a81226dcf37d9ff430244b21f952</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs42003-018-0105-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-018-0105-8%26sid%3Dliteratum%253Aachs%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DAlabi%26aufirst%3DS.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DVitale%26aufirst%3DN.%2BJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DM.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAndrogen%2520receptor%2520degradation%2520by%2520the%2520proteolysis-targeting%2520chimera%2520ARCC-4%2520outperforms%2520enzalutamide%2520in%2520cellular%2520models%2520of%2520prostate%2520cancer%2520drug%2520resistance%26jtitle%3DCommun.%2520Biol.%26date%3D2018%26volume%3D1%26spage%3D100%26doi%3D10.1038%2Fs42003-018-0105-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of MD-224 as a first-in-class, highly potent, and efficacious proteolysis targeting chimera murine double minute 2 degrader capable of achieving complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00909</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00909" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=448-466&author=Y.+Liauthor=J.+Yangauthor=A.+Aguilarauthor=D.+McEachernauthor=S.+Przybranowskiauthor=L.+Liuauthor=C.-Y.+Yangauthor=M.+Wangauthor=X.+Hanauthor=S.+Wang&title=Discovery+of+MD-224+as+a+first-in-class%2C+highly+potent%2C+and+efficacious+proteolysis+targeting+chimera+murine+double+minute+2+degrader+capable+of+achieving+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of MD-224 as a First-in-Class, Highly Potent, and Efficacious Proteolysis Targeting Chimera Murine Double Minute 2 Degrader Capable of Achieving Complete and Durable Tumor Regression</span></div><div class="casAuthors">Li, Yangbing; Yang, Jiuling; Aguilar, Angelo; McEachern, Donna; Przybranowski, Sally; Liu, Liu; Yang, Chao-Yie; Wang, Mi; Han, Xin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">448-466</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human murine double minute 2 (MDM2) protein is a primary endogenous cellular inhibitor of the tumor suppressor p53 and has been pursued as an attractive cancer therapeutic target.  Several potent, nonpeptide, small-mol. inhibitors of MDM2 are currently in clin. development.  In this paper, we report our design, synthesis, and evaluation of small-mol. MDM2 degraders based on the proteolysis targeting chimera (PROTAC) concept.  The most promising compd. (MD-224, I) effectively induces rapid degrdn. of MDM2 at concns. <1 nM in human leukemia cells.  It achieves an IC50 value of 1.5 nM in inhibition of growth of RS4;11 cells and also low nanomolar IC50 values in a panel of leukemia cell lines.  MD-224 (I) achieves complete and durable tumor regression in vivo in the RS4;11 xenograft tumor model in mice at well-tolerated dose schedules.  MD-224 (I) is thus a highly potent and efficacious MDM2 degrader and warrants extensive evaluations as a new class of anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfVzK-tVEdk7Vg90H21EOLACvtfcHk0lh7VFAV_T5z2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK&md5=3eaa40a8b0ae15c0172ef44a2fd1b902</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00909%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520MD-224%2520as%2520a%2520first-in-class%252C%2520highly%2520potent%252C%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520murine%2520double%2520minute%25202%2520degrader%2520capable%2520of%2520achieving%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D448%26epage%3D466%26doi%3D10.1021%2Facs.jmedchem.8b00909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras for the selective degradation of Mcl-1/Bcl-2 derived from nonselective target binding ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8152</span>– <span class="NLM_lpage">8163</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00919</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00919" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFemtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8152-8163&author=Z.+Wangauthor=N.+Heauthor=Z.+Guoauthor=C.+Niuauthor=T.+Songauthor=Y.+Guoauthor=K.+Caoauthor=A.+Wangauthor=J.+Zhuauthor=X.+Zhangauthor=Z.+Zhang&title=Proteolysis+targeting+chimeras+for+the+selective+degradation+of+Mcl-1%2FBcl-2+derived+from+nonselective+target+binding+ligands&doi=10.1021%2Facs.jmedchem.9b00919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras for the Selective Degradation of Mcl-1/Bcl-2 Derived from Nonselective Target Binding Ligands</span></div><div class="casAuthors">Wang, Ziqian; He, Nianzhe; Guo, Zongwei; Niu, Cuili; Song, Ting; Guo, Yafei; Cao, Keke; Wang, Anhui; Zhu, Junjie; Zhang, Xiaodong; Zhang, Zhichao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8152-8163</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) recruits an E3 ligase to a target protein to induce its ubiquitination and subsequent degrdn.  We reported the success in the development of two PROTACs (C3(I) and C5(II)) that potently and selectively induce the degrdn. of Mcl-1 and Bcl-2 (DC50 = 0.7 and 3.0 μM), resp., by introducing the E3 ligase cereblon (CRBN)-binding ligand pomalidomide to Mcl-1/Bcl-2 dual inhibitors S1-6 and Nap-1 with μM-range affinity.  I-induced Mcl-1 ubiquitination translated into much more lethality in Mcl-1-dependent H23 cells than the most potent Mcl-1 occupancy-based inhibitor A-1210477 with nM-range affinity.  Moreover, structure-activity relationship (SAR) anal. and mol. dynamic (MD) simulations discovered the structural basis for turning nonselective or promiscuous Bcl-2 family ligands into selective PROTACs.  I and II exhibited reversible depletion in living cells, which provides a new potent toolkit for gain-of-function studies to probe the dynamic roles of Bcl-2 and Mcl-1 in apoptosis networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9YwjuYRj58LVg90H21EOLACvtfcHk0lh7VFAV_T5z2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFemtLvN&md5=f962278d69b69117c501bc7c0c31ebfd</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00919%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DHe%26aufirst%3DN.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DNiu%26aufirst%3DC.%26aulast%3DSong%26aufirst%3DT.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520for%2520the%2520selective%2520degradation%2520of%2520Mcl-1%252FBcl-2%2520derived%2520from%2520nonselective%2520target%2520binding%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8152%26epage%3D8163%26doi%3D10.1021%2Facs.jmedchem.9b00919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lhibGaBYqswvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B. E.</span></span> <span> </span><span class="NLM_article-title">Cereblon modulators: Low molecular weight inducers of protein degradation</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.ddtec.2019.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31200856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=29-34&author=P.+P.+Chamberlainauthor=B.+E.+Cathers&title=Cereblon+modulators%3A+Low+molecular+weight+inducers+of+protein+degradation&doi=10.1016%2Fj.ddtec.2019.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Cereblon modulators: Low molecular weight inducers of protein degradation</span></div><div class="casAuthors">Chamberlain Philip P; Cathers Brian E</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">29-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Targeted protein degradation has become an exciting new paradigm in drug discovery with the potential to target new protein families for therapeutic intervention.  In 2010, Hiroshi Handa and colleagues discovered that the drug thalidomide binds to the protein cereblon, a component of the CRL4(CRBN) E3 ubiquitin ligase.  In contrast to the heterobifunctional small molecule degraders reported in the literature, thalidomide is of very low molecular weight (∼258Da) with molecular properties (solubility, metabolic stability, permeability etc) that readily support pharmaceutical dosing.  It was subsequently shown that thalidomide and the analogues lenalidomide and pomalidomide are able to degrade the transcription factors Ikaros and Aiolos.  CK1α and GSPT1 were subsequently identified as substrates for specific ligands, indicating that this molecular class could be tuned for selective protein degradation.  Structural studies showed that the thalidomide analogues bind to a shallow hydrophobic pocket on the surface of cereblon, and scaffold a protein-protein interaction with target proteins.  Target proteins do not need any affinity for the cereblon modulators, and as such undruggable, or even unligandable, proteins can be targeted for degradation.  A similar mechanism of action was subsequently identified for the clinical molecule indisulam, indicating that low molecular weight degraders are not unique to cereblon.  The groundbreaking work on cereblon represents a case study for the discovery and characterization of low molecular weight protein degraders for other ligases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXobBItaL1WaVAfW6udTcc2eblu1r5IIwkQbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsg%253D%253D&md5=ab46406dc0664c8efe3d2fbbf9bd061b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26aulast%3DCathers%26aufirst%3DB.%2BE.%26atitle%3DCereblon%2520modulators%253A%2520Low%2520molecular%2520weight%2520inducers%2520of%2520protein%2520degradation%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D29%26epage%3D34%26doi%3D10.1016%2Fj.ddtec.2019.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Girardini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Cereblon versus VHL: Hijacking E3 ligases against each other using PROTACs</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2466</span>– <span class="NLM_lpage">2479</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2019.02.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.bmc.2019.02.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=30826187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXktV2rsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=2466-2479&author=M.+Girardiniauthor=C.+Maniaciauthor=S.+J.+Hughesauthor=A.+Testaauthor=A.+Ciulli&title=Cereblon+versus+VHL%3A+Hijacking+E3+ligases+against+each+other+using+PROTACs&doi=10.1016%2Fj.bmc.2019.02.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Cereblon versus VHL: Hijacking E3 ligases against each other using PROTACs</span></div><div class="casAuthors">Girardini, Miriam; Maniaci, Chiara; Hughes, Scott J.; Testa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2466-2479</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The von Hippel-Lindau (VHL) and cereblon (CRBN) proteins are substrate recognition subunits of two ubiquitously expressed and biol. important Cullin RING E3 ubiquitin ligase complexes.  VHL and CRBN are also the two most popular E3 ligases being recruited by bifunctional Proteolysis-targeting chimeras (PROTACs) to induce ubiquitination and subsequent proteasomal degrdn. of a target protein.  Using homo-PROTACs, VHL and CRBN have been independently dimerized to induce their own degrdn.  Here we report the design, synthesis and cellular activity of VHL-CRBN hetero-dimerizing PROTACs featuring diverse conjugation patterns.  We found that the most active compd. 14a induced potent, rapid and profound preferential degrdn. of CRBN over VHL in cancer cell lines.  At lower concns., weaker degrdn. of VHL was instead obsd.  This work demonstrates proof of concept of designing PROTACs to hijack different E3 ligases against each other, and highlights a powerful and generalizable proximity-induced strategy to achieve E3 ligase knockdown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGops5qv3m7tibVg90H21EOLACvtfcHk0lhibGaBYqswvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXktV2rsb8%253D&md5=ef00788b6428899cf4dfe1c8d357dd64</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.02.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.02.048%26sid%3Dliteratum%253Aachs%26aulast%3DGirardini%26aufirst%3DM.%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DCereblon%2520versus%2520VHL%253A%2520Hijacking%2520E3%2520ligases%2520against%2520each%2520other%2520using%2520PROTACs%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D2466%26epage%3D2479%26doi%3D10.1016%2Fj.bmc.2019.02.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shahbazi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polizzotto, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uldrick, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyvill, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeldis, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarchoan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figg, W. D.</span></span> <span> </span><span class="NLM_article-title">A sensitive and robust HPLC assay with fluorescence detection for the quantification of pomalidomide in human plasma for pharmacokinetic analyses</span>. <i>J. Pharm. Biomed. Anal.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/j.jpba.2014.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.jpba.2014.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=24486861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlSktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2014&pages=63-68&author=S.+Shahbaziauthor=C.+J.+Peerauthor=M.+N.+Polizzottoauthor=T.+S.+Uldrickauthor=J.+Rothauthor=K.+M.+Wyvillauthor=K.+Alemanauthor=J.+B.+Zeldisauthor=R.+Yarchoanauthor=W.+D.+Figg&title=A+sensitive+and+robust+HPLC+assay+with+fluorescence+detection+for+the+quantification+of+pomalidomide+in+human+plasma+for+pharmacokinetic+analyses&doi=10.1016%2Fj.jpba.2014.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A sensitive and robust HPLC assay with fluorescence detection for the quantification of pomalidomide in human plasma for pharmacokinetic analyses</span></div><div class="casAuthors">Shahbazi, Shandiz; Peer, Cody J.; Polizzotto, Mark N.; Uldrick, Thomas S.; Roth, Jeffrey; Wyvill, Kathleen M.; Aleman, Karen; Zeldis, Jerome B.; Yarchoan, Robert; Figg, William D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical and Biomedical Analysis</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">63-68</span>CODEN:
                <span class="NLM_cas:coden">JPBADA</span>;
        ISSN:<span class="NLM_cas:issn">0731-7085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Pomalidomide is a second generation IMiD (immunomodulatory agent) that has recently been granted approval by the Food and Drug Administration for treatment of relapsed multiple myeloma after prior treatment with two antimyeloma agents, including lenalidomide and bortezomib.  A simple and robust HPLC assay with fluorescence detection for pomalidomide over the range of 1-500 ng/mL has been developed for application to pharmacokinetic studies in ongoing clin. trials in various other malignancies.  A liq.-liq. extn. from human plasma alone or pre-stabilized with 0.1% HCl was performed, using Pr paraben as the internal std.  From plasma either pre-stabilized with 0.1% HCl or not, the assay was shown to be selective, sensitive, accurate, precise, and have minimal matrix effects (<20%).  Pomalidomide was stable in plasma through 4 freeze-thaw cycles (<12% change), in plasma at room temp. for up to 2 h for samples not pre-stabilized with 0.1% HCl and up to 8 h in samples pre-stabilized with 0.1% HCl, 24 h post-prepn. at 4 °C (<2% change), and showed excellent extn. recovery (∼90%).  This is the first reported description of the freeze/thaw and plasma stability of pomalidomide in plasma either pre-stabilized with 0.1% HCl or not.  The information presented in this manuscript is important when performing pharmacokinetic analyses.  The method was used to analyze clin. pharmacokinetics samples obtained after a 5 mg oral dose of pomalidomide.  This relatively simple HPLC-FL assay allows a broader range of labs. to measure pomalidomide for application to clin. pharmacokinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrTvTW3k7k5LVg90H21EOLACvtfcHk0ljAQlZYTcF0ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlSktb4%253D&md5=ab14a2918ff157d138b5191738ce3590</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.jpba.2014.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jpba.2014.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DShahbazi%26aufirst%3DS.%26aulast%3DPeer%26aufirst%3DC.%2BJ.%26aulast%3DPolizzotto%26aufirst%3DM.%2BN.%26aulast%3DUldrick%26aufirst%3DT.%2BS.%26aulast%3DRoth%26aufirst%3DJ.%26aulast%3DWyvill%26aufirst%3DK.%2BM.%26aulast%3DAleman%26aufirst%3DK.%26aulast%3DZeldis%26aufirst%3DJ.%2BB.%26aulast%3DYarchoan%26aufirst%3DR.%26aulast%3DFigg%26aufirst%3DW.%2BD.%26atitle%3DA%2520sensitive%2520and%2520robust%2520HPLC%2520assay%2520with%2520fluorescence%2520detection%2520for%2520the%2520quantification%2520of%2520pomalidomide%2520in%2520human%2520plasma%2520for%2520pharmacokinetic%2520analyses%26jtitle%3DJ.%2520Pharm.%2520Biomed.%2520Anal.%26date%3D2014%26volume%3D92%26spage%3D63%26epage%3D68%26doi%3D10.1016%2Fj.jpba.2014.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibitors of signal transducer and activator of transcription 3 (Stat3) protein</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6645</span>– <span class="NLM_lpage">6668</span>, <span class="refDoi"> DOI: 10.1021/jm300207s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300207s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnsl2msrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6645-6668&author=B.+Debnathauthor=S.+Xuauthor=N.+Neamati&title=Small+molecule+inhibitors+of+signal+transducer+and+activator+of+transcription+3+%28Stat3%29+protein&doi=10.1021%2Fjm300207s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors of Signal Transducer and Activator of Transcription 3 (Stat3) Protein</span></div><div class="casAuthors">Debnath, Bikash; Xu, Shili; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6645-6668</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Signal transducer and activator of transcription 3 (Stat3) is a transcription factor belonging to the STAT protein family, members of which regulate a wide variety of biol. processes, including cell growth, proliferation, survival, differentiation, development, movement and inflammation in response to extracellular signaling mols. such as cytokines and growth factors.  Receptor assocd. tyrosine kinases phosphorylate Stat3 protein at Tyr705 and enable it to dimerize and subsequently enter the nucleus where it activates the transcription of its target genes.  Accumulating evidence has established Stat3's role in oncogenesis, cardiovascular diseases, rheumatoid arthritis, fibrotic, and Alzheimer's diseases.  Thus, Stat3 inhibition is a promising therapeutic approach not only for cancer, but also for other diseases.  Although Stat3 has been considered as an important therapeutic target, progress in discovery and development of effective and specific Stat3 inhibitors remains slow.  In this article we summarize Stat3 inhibitors published between 2006 and 2011.  New strategies along with novel technologies are required to achieve breakthrough in the development of Stat3 inhibitors that can be translated into clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWWQl9gV1-ZLVg90H21EOLACvtfcHk0ljAQlZYTcF0ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnsl2msrc%253D&md5=f5f3ac9f8fc2326c75edd2a9c136275b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm300207s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300207s%26sid%3Dliteratum%253Aachs%26aulast%3DDebnath%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%2520%2528Stat3%2529%2520protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6645%26epage%3D6668%26doi%3D10.1021%2Fjm300207s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bankhead, A.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Multi-omics profiling reveals key signaling pathways in ovarian cancer controlled by STAT3</span>. <i>Theranostics</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">5478</span>– <span class="NLM_lpage">5496</span>, <span class="refDoi"> DOI: 10.7150/thno.33444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.7150%2Fthno.33444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31534498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvVSgtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=5478-5496&author=T.+Luauthor=A.+Bankheadauthor=M.+Ljungmanauthor=N.+Neamati&title=Multi-omics+profiling+reveals+key+signaling+pathways+in+ovarian+cancer+controlled+by+STAT3&doi=10.7150%2Fthno.33444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-omics profiling reveals key signaling pathways in ovarian cancer controlled by STAT3</span></div><div class="casAuthors">Lu, Tiangong; Bankhead, Armand, III; Ljungman, Mats; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5478-5496</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">Inhibiting STAT3 signaling reduces tumor progression, metastasis and chemoresistance, however the precise mol. mechanism has not been fully delineated in ovarian cancer.  In this study, we generated STAT3 knockout (KO) ovarian cancer cell lines.  Effects of STAT3 KO on cell proliferation, migration and spheroid formation were assessed in vitro and effects on in vivo tumor growth were tested using several tumor xenograft models.  We used multi-omic genome-wide profiling to identify multi-level (Bru-Seq, RNA-Seq, and MS Proteomic) expression signatures of STAT3 KO ovarian cancer cells.  We obsd. that deletion of STAT3 blocked cell proliferation and migration in vitro and suppressed tumor growth in mice.  Deletion of STAT3 transcriptionally suppressed key genes involved in EMT, cell cycle progression, E2F signaling, and altered stemness markers.  Notably, KO of STAT3 resulted in modulation of the expression of other STAT family members.  Our study presents a rich, multi-faceted summary of the mol. mechanisms impacted by STAT3 deletion and provides new insight for STAT3's potential as a therapeutic target in ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTvEerWSP_tbVg90H21EOLACvtfcHk0ljAQlZYTcF0ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvVSgtLs%253D&md5=0685567a4fff07940bc7dddcca2dc1df</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.7150%2Fthno.33444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.33444%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DT.%26aulast%3DBankhead%26aufirst%3DA.%26aulast%3DLjungman%26aufirst%3DM.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DMulti-omics%2520profiling%2520reveals%2520key%2520signaling%2520pathways%2520in%2520ovarian%2520cancer%2520controlled%2520by%2520STAT3%26jtitle%3DTheranostics%26date%3D2019%26volume%3D9%26spage%3D5478%26epage%3D5496%26doi%3D10.7150%2Fthno.33444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeman-Neill, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. E.</span></span> <span> </span><span class="NLM_article-title">Bortezomib up-regulates activated signal transducer and activator of transcription-3 and synergizes with inhibitors of signal transducer and activator of transcription-3 to promote head and neck squamous cell carcinoma cell death</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2211</span>– <span class="NLM_lpage">2220</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-09-0327</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1158%2F1535-7163.MCT-09-0327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=19638453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFCnu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=2211-2220&author=C.+Liauthor=Y.+Zangauthor=M.+Senauthor=R.+J.+Leeman-Neillauthor=D.+S.+Manauthor=J.+R.+Grandisauthor=D.+E.+Johnson&title=Bortezomib+up-regulates+activated+signal+transducer+and+activator+of+transcription-3+and+synergizes+with+inhibitors+of+signal+transducer+and+activator+of+transcription-3+to+promote+head+and+neck+squamous+cell+carcinoma+cell+death&doi=10.1158%2F1535-7163.MCT-09-0327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Bortezomib up-regulates activated signal transducer and activator of transcription-3 and synergizes with inhibitors of signal transducer and activator of transcription-3 to promote head and neck squamous cell carcinoma cell death</span></div><div class="casAuthors">Li, Changyou; Zang, Yan; Sen, Malabika; Leeman-Neill, Rebecca J.; Man, David S. K.; Grandis, Jennifer R.; Johnson, Daniel E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2211-2220</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Head and neck squamous cell carcinomas (HNSCC) are commonly resistant to conventional chemotherapy drugs and exhibit overexpression of signal transducer and activator of transcription 3 (STAT3).  STAT3 promotes both the proliferation and survival of HNSCC cells.  Recent studies have shown that the proteasome inhibitor bortezomib shows cytotoxic activity against HNSCC in vitro and in vivo.  We report that treatment of HNSCC cells with bortezomib led to up-regulation of total STAT3 protein and the phosphorylated/activated form of STAT3, as well as an increase in cellular STAT3 activity.  This suggested that the ability of bortezomib to kill HNSCC cells may be blunted due to induction of STAT3, and inhibition of STAT3 may be a useful means for improving bortezomib efficacy.  Indeed, forced expression of dominant-active STAT3 inhibited bortezomib-induced cell death, whereas expression of dominant-neg. STAT3 served to enhance killing by this compd.  In addn., specific inhibition of STAT3 with the use of a STAT3 decoy oligonucleotide resulted in enhancement of bortezomib-induced apoptosis signaling and loss of clonogenic survival.  Cotreatment of HNSCC cells with bortezomib and guggulsterone, a naturally occurring compd. known to inhibit STAT3 activation, led to synergistic activation of cell death and loss of clonogenic survival.  In summary, these studies show that bortezomib induces the expression of active STAT3, a key growth- promoting protein in HNSCC cells.  Furthermore, our findings suggest that the therapeutic activity of bortezomib against HNSCC may be markedly improved by cotreatment with mol. targeting agents against STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAVkhXr3kma7Vg90H21EOLACvtfcHk0lgVEaC6D92qWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFCnu7s%253D&md5=b755af9decfbc6ab534da714d847ec33</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-09-0327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-09-0327%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZang%26aufirst%3DY.%26aulast%3DSen%26aufirst%3DM.%26aulast%3DLeeman-Neill%26aufirst%3DR.%2BJ.%26aulast%3DMan%26aufirst%3DD.%2BS.%26aulast%3DGrandis%26aufirst%3DJ.%2BR.%26aulast%3DJohnson%26aufirst%3DD.%2BE.%26atitle%3DBortezomib%2520up-regulates%2520activated%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription-3%2520and%2520synergizes%2520with%2520inhibitors%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription-3%2520to%2520promote%2520head%2520and%2520neck%2520squamous%2520cell%2520carcinoma%2520cell%2520death%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2009%26volume%3D8%26spage%3D2211%26epage%3D2220%26doi%3D10.1158%2F1535-7163.MCT-09-0327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A chemoproteomic approach to query the degradable kinome using a multi-kinase degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">99.e6</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.chembiol.2017.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=29129717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=88-99.e6&author=H.-T.+Huangauthor=D.+Dobrovolskyauthor=J.+Paulkauthor=G.+Yangauthor=E.+L.+Weisbergauthor=Z.+M.+Doctorauthor=D.+L.+Buckleyauthor=J.+H.+Choauthor=E.+Koauthor=J.+Jangauthor=K.+Shiauthor=H.+G.+Choiauthor=J.+D.+Griffinauthor=Y.+Liauthor=S.+P.+Treonauthor=E.+S.+Fischerauthor=J.+E.+Bradnerauthor=L.+Tanauthor=N.+S.+Gray&title=A+chemoproteomic+approach+to+query+the+degradable+kinome+using+a+multi-kinase+degrader&doi=10.1016%2Fj.chembiol.2017.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-kinase Degrader</span></div><div class="casAuthors">Huang, Hai-Tsang; Dobrovolsky, Dennis; Paulk, Joshiawa; Yang, Guang; Weisberg, Ellen L.; Doctor, Zainab M.; Buckley, Dennis L.; Cho, Joong-Heui; Ko, Eunhwa; Jang, Jaebong; Shi, Kun; Choi, Hwan Geun; Griffin, James D.; Li, Ying; Treon, Steven P.; Fischer, Eric S.; Bradner, James E.; Tan, Li; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-99.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heterobifunctional mols. that recruit E3 ubiquitin ligases, such as cereblon, for targeted protein degrdn. represent an emerging pharmacol. strategy.  A major unanswered question is how generally applicable this strategy is to all protein targets.  In this study, we designed a multi-kinase degrader by conjugating a highly promiscuous kinase inhibitor with a cereblon-binding ligand, and used quant. proteomics to discover 28 kinases, including BTK, PTK2, PTK2B, FLT3, AURKA, AURKB, TEC, ULK1, ITK, and nine members of the CDK family, as degradable.  This set of kinases is only a fraction of the intracellular targets bound by the degrader, demonstrating that successful degrdn. requires more than target engagement.  The results guided us to develop selective degraders for FLT3 and BTK, with potentials to improve disease treatment.  Together, this study demonstrates an efficient approach to triage a gene family of interest to identify readily degradable targets for further studies and pre-clin. developments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYNxhoPV3127Vg90H21EOLACvtfcHk0lgVEaC6D92qWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO&md5=703cfbaca3edd24176472158563f895f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.-T.%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520chemoproteomic%2520approach%2520to%2520query%2520the%2520degradable%2520kinome%2520using%2520a%2520multi-kinase%2520degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D88%26epage%3D99.e6%26doi%3D10.1016%2Fj.chembiol.2017.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horne, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaidehi, N.</span></span> <span> </span><span class="NLM_article-title">Molecular dynamics simulations of the conformational changes in signal transducers and activators of transcription, Stat1 and Stat3</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2009.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.jmgm.2009.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=19781967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlCmt7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=347-356&author=J.+Linauthor=R.+Buettnerauthor=Y.+C.+Yuanauthor=R.+Yipauthor=D.+Horneauthor=R.+Joveauthor=N.+Vaidehi&title=Molecular+dynamics+simulations+of+the+conformational+changes+in+signal+transducers+and+activators+of+transcription%2C+Stat1+and+Stat3&doi=10.1016%2Fj.jmgm.2009.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations of the conformational changes in signal transducers and activators of transcription, Stat1 and Stat3</span></div><div class="casAuthors">Lin, Jianping; Buettner, Ralf; Yuan, Yate-Ching; Yip, Richard; Horne, David; Jove, Richard; Vaidehi, Nagarajan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">347-356</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">All signal transducers and activators of transcription (STAT) factors are a family of cytoplasmic transcription factors that mediate the signal response to cytokines, growth factors, and hormonal factors.  The phosphorylation and subsequent activation of Stat3, a member of the STAT family, has been found to be elevated in a large no. of diverse human cancers.  An understanding of the dynamics of the Stat3 dimer interface is pertinent to designing small mol. inhibitors to activated Stat3 dimer.  To this end, the authors performed mol. dynamics simulations in explicit water of the activated Stat3 homodimer, and also its closely related member of the STAT family, activated Stat1 homodimer.  The authors obsd. a large-scale domain motion in the Stat3 dimer while the structure of the monomer remained intact.  The driving force for this conformational change was enhanced binding of the Stat3 dimer to DNA, thereby regulating gene expression.  The model showed that the C-terminus of one monomer wrapped around the core of the SH2 domain of the other monomer, and that this region that made up the dimer interface remained intact during the dynamics.  Water diffused into a cavity under this dimer interface, thus expanding a pre-existing cavity that gets gated and closed by the loops in the SH2 domain.  This cavity could serve as a potential binding pocket for inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1eGdndUM1ArVg90H21EOLACvtfcHk0lgVEaC6D92qWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlCmt7vL&md5=d32ef6835f4f199f6fc14d58e51d9881</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2009.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2009.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DYuan%26aufirst%3DY.%2BC.%26aulast%3DYip%26aufirst%3DR.%26aulast%3DHorne%26aufirst%3DD.%26aulast%3DJove%26aufirst%3DR.%26aulast%3DVaidehi%26aufirst%3DN.%26atitle%3DMolecular%2520dynamics%2520simulations%2520of%2520the%2520conformational%2520changes%2520in%2520signal%2520transducers%2520and%2520activators%2520of%2520transcription%252C%2520Stat1%2520and%2520Stat3%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2009%26volume%3D28%26spage%3D347%26epage%3D356%26doi%3D10.1016%2Fj.jmgm.2009.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parapatics, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ecker, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Translation termination factor GSPT1 Is a phenotypically relevant off-target of heterobifunctional phthalimide degraders</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00969</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00969" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=553-560&author=M.+Ishoeyauthor=S.+Chornauthor=N.+Singhauthor=M.+G.+Jaegerauthor=M.+Brandauthor=J.+Paulkauthor=S.+Bauerauthor=M.+A.+Erbauthor=K.+Parapaticsauthor=A.+C.+M%C3%BCllerauthor=K.+L.+Bennettauthor=G.+F.+Eckerauthor=J.+E.+Bradnerauthor=G.+E.+Winter&title=Translation+termination+factor+GSPT1+Is+a+phenotypically+relevant+off-target+of+heterobifunctional+phthalimide+degraders&doi=10.1021%2Facschembio.7b00969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Translation Termination Factor GSPT1 Is a Phenotypically Relevant Off-Target of Heterobifunctional Phthalimide Degraders</span></div><div class="casAuthors">Ishoey, Mette; Chorn, Someth; Singh, Natesh; Jaeger, Martin G.; Brand, Matthias; Paulk, Joshiawa; Bauer, Sophie; Erb, Michael A.; Parapatics, Katja; Muller, Andre C.; Bennett, Keiryn L.; Ecker, Gerhard F.; Bradner, James E.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">553-560</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein degrdn. is an emerging therapeutic strategy with a unique mol. pharmacol. that enables the disruption of all functions assocd. with a target.  This is particularly relevant for proteins depending on mol. scaffolding, such as transcription factors or receptor tyrosine kinases (RTKs).  To address tractability of multiple RTKs for chem. degrdn. by the E3 ligase CUL4-RBX1-DDB1-CRBN (CRL4CRBN), we synthesized a series of phthalimide degraders based on the promiscuous kinase inhibitors sunitinib and PHA665752.  While both series failed to induce degrdn. of their consensus targets, individual mols. displayed pronounced efficacy in leukemia cell lines.  Orthogonal target identification supported by mol. docking led us to identify the translation termination factor G1 to S phase transition 1 (GSPT1) as a converging off-target, resulting from inadvertent E3 ligase modulation.  This research highlights the importance of monitoring degrdn. events that are independent of the resp. targeting ligand as a unique feature of small-mol. degraders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_ZxnB5rKvvLVg90H21EOLACvtfcHk0lhueCRFW6HGyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D&md5=02a70959e8a8929a3c70846c1ed0200f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00969%26sid%3Dliteratum%253Aachs%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DChorn%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DJaeger%26aufirst%3DM.%2BG.%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DParapatics%26aufirst%3DK.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DTranslation%2520termination%2520factor%2520GSPT1%2520Is%2520a%2520phenotypically%2520relevant%2520off-target%2520of%2520heterobifunctional%2520phthalimide%2520degraders%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D553%26epage%3D560%26doi%3D10.1021%2Facschembio.7b00969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matyskiela, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagarigan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houston, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nosaka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaidarova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchelman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span> <span> </span><span class="NLM_article-title">A novel cereblon modulator recruits GSPT1 to the CRL4<sup>CRBN</sup> ubiquitin ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>535</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1038/nature18611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fnature18611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=27338790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=252-257&author=M.+E.+Matyskielaauthor=G.+Luauthor=T.+Itoauthor=B.+Pagariganauthor=C.-C.+Luauthor=K.+Millerauthor=W.+Fangauthor=N.-Y.+Wangauthor=D.+Nguyenauthor=J.+Houstonauthor=G.+Carmelauthor=T.+Tranauthor=M.+Rileyauthor=L.+Nosakaauthor=G.+C.+Landerauthor=S.+Gaidarovaauthor=S.+Xuauthor=A.+L.+Ruchelmanauthor=H.+Handaauthor=J.+Carmichaelauthor=T.+O.+Danielauthor=B.+E.+Cathersauthor=A.+Lopez-Gironaauthor=P.+P.+Chamberlain&title=A+novel+cereblon+modulator+recruits+GSPT1+to+the+CRL4CRBN+ubiquitin+ligase&doi=10.1038%2Fnature18611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A novel cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Matyskiela, Mary E.; Lu, Gang; Ito, Takumi; Pagarigan, Barbra; Lu, Chin-Chun; Miller, Karen; Fang, Wei; Wang, Nai-Yu; Nguyen, Derek; Houston, Jack; Carmel, Gilles; Tran, Tam; Riley, Mariko; Nosaka, Lyn'Al; Lander, Gabriel C.; Gaidarova, Svetlana; Xu, Shuichan; Ruchelman, Alexander L.; Handa, Hiroshi; Carmichael, James; Daniel, Thomas O.; Cathers, Brian E.; Lopez-Girona, Antonia; Chamberlain, Philip P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7611</span>),
    <span class="NLM_cas:pages">252-257</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Immunomodulatory drugs bind to cereblon (CRBN) to confer differentiated substrate specificity on the CRL4CRBN E3 ubiquitin ligase.  Here we report the identification of a new cereblon modulator, CC-885, with potent anti-tumor activity.  The anti-tumor activity of CC-885 is mediated through the cereblon-dependent ubiquitination and degrdn. of the translation termination factor GSPT1.  Patient-derived acute myeloid leukemia tumor cells exhibit high sensitivity to CC-885, indicating the clin. potential of this mechanism.  Crystallog. studies of the CRBN-DDB1-CC-885-GSPT1 complex reveal that GSPT1 binds to cereblon through a surface turn contg. a glycine residue at a key position, interacting with both CC-885 and a 'hotspot' on the cereblon surface.  Although GSPT1 possesses no obvious structural, sequence or functional homol. to previously known cereblon substrates, mutational anal. and modeling indicate that the cereblon substrate Ikaros uses a similar structural feature to bind cereblon, suggesting a common motif for substrate recruitment.  These findings define a structural degron underlying cereblon 'neosubstrate' selectivity, and identify an anti-tumor target rendered druggable by cereblon modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgAWiXdlJgYLVg90H21EOLACvtfcHk0lhueCRFW6HGyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI&md5=1d4e340416aa0b08fc0f123faf638c3f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnature18611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18611%26sid%3Dliteratum%253Aachs%26aulast%3DMatyskiela%26aufirst%3DM.%2BE.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DPagarigan%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DC.-C.%26aulast%3DMiller%26aufirst%3DK.%26aulast%3DFang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DN.-Y.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DHouston%26aufirst%3DJ.%26aulast%3DCarmel%26aufirst%3DG.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DRiley%26aufirst%3DM.%26aulast%3DNosaka%26aufirst%3DL.%26aulast%3DLander%26aufirst%3DG.%2BC.%26aulast%3DGaidarova%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DRuchelman%26aufirst%3DA.%2BL.%26aulast%3DHanda%26aufirst%3DH.%26aulast%3DCarmichael%26aufirst%3DJ.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DCathers%26aufirst%3DB.%2BE.%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26atitle%3DA%2520novel%2520cereblon%2520modulator%2520recruits%2520GSPT1%2520to%2520the%2520CRL4CRBN%2520ubiquitin%2520ligase%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D252%26epage%3D257%26doi%3D10.1038%2Fnature18611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Simple structural modifications converting a bona fide MDM2 PROTAC degrader into a molecular glue molecule: A cautionary tale in the design of PROTAC degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9471</span>– <span class="NLM_lpage">9487</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00846</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00846" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVGgtb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9471-9487&author=J.+Yangauthor=Y.+Liauthor=A.+Aguilarauthor=Z.+Liuauthor=C.+Y.+Yangauthor=S.+Wang&title=Simple+structural+modifications+converting+a+bona+fide+MDM2+PROTAC+degrader+into+a+molecular+glue+molecule%3A+A+cautionary+tale+in+the+design+of+PROTAC+degraders&doi=10.1021%2Facs.jmedchem.9b00846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Simple Structural Modifications Converting a Bona fide MDM2 PROTAC Degrader into a Molecular Glue Molecule: A Cautionary Tale in the Design of PROTAC Degraders</span></div><div class="casAuthors">Yang, Jiuling; Li, Yangbing; Aguilar, Angelo; Liu, Zhaomin; Yang, Chao-Yie; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9471-9487</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inducing protein degrdn. by proteolysis targeting chimeras (PROTACs) has gained tremendous momentum for its promise to discover and develop new therapies.  Based upon our previously reported PROTAC MDM2 degraders, we have designed and synthesized addnl. analogs.  Surprisingly, we found that simple structural modifications of MD-222, a bona fide MDM2 PROTAC degrader, converts it into a "mol. glue", as exemplified by MG-277.  MG-277 induces only moderate MDM2 degrdn. and fails to activate wild-type p53 but is highly potent in inhibition of tumor cell growth in a p53-independent manner.  Our mechanistic investigation established that MG-277 is not a PROTAC MDM2 degrader but instead works as a mol. glue, inducing degrdn. of a translation termination factor, GSPT1 to achieve its potent anticancer activity.  Our study provides the first example that simple structural modifications can convert a bona fide PROTAC degrader into a mol. glue compd., which has a completely different mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH5W2xSSGs6rVg90H21EOLACvtfcHk0li_xiwEYYwbIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVGgtb7K&md5=8a3e36a4fc3110c798d379b70d6203fc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00846%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DSimple%2520structural%2520modifications%2520converting%2520a%2520bona%2520fide%2520MDM2%2520PROTAC%2520degrader%2520into%2520a%2520molecular%2520glue%2520molecule%253A%2520A%2520cautionary%2520tale%2520in%2520the%2520design%2520of%2520PROTAC%2520degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9471%26epage%3D9487%26doi%3D10.1021%2Facs.jmedchem.9b00846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=941-964&author=X.+Hanauthor=C.+Wangauthor=C.+Qinauthor=W.+Xiangauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=M.+Wangauthor=L.+Zhaoauthor=T.+Xuauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ARD-69+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+androgen+receptor+%28AR%29+for+the+treatment+of+prostate+cancer&doi=10.1021%2Facs.jmedchem.8b01631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span></div><div class="casAuthors">Han, Xin; Wang, Chao; Qin, Chong; Xiang, Weiguo; Fernandez-Salas, Ester; Yang, Chao-Yie; Wang, Mi; Zhao, Lijie; Xu, Tianfeng; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">941-964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the discovery of highly potent PROTAC degraders of androgen receptor (AR), as exemplified by IARD-69.  I induces degrdn. of AR protein in AR-pos. prostate cancer cell lines in a dose- and time-dependent manner.  I achieves DC50 values of 0.86, 0.76, and 10.4 nM in LNCaP, VCaP, and 22Rv1 AR+ prostate cancer cell lines, resp.  I is capable of reducing the AR protein level by >95% in these prostate cancer cell lines and effectively suppressing AR-regulated gene expression.  I potently inhibits cell growth in these AR-pos. prostate cancer cell lines and is >100 times more potent than AR antagonists.  A single dose of I effectively reduces the level of AR protein in xenograft tumor tissue in mice.  Further optimization of I may ultimately lead to a new therapy for AR+, castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTlOD69tfhtbVg90H21EOLACvtfcHk0li_xiwEYYwbIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D&md5=2376573fb1a1e19c517d64c70199700a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01631%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ARD-69%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520androgen%2520receptor%2520%2528AR%2529%2520for%2520the%2520treatment%2520of%2520prostate%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D941%26epage%3D964%26doi%3D10.1021%2Facs.jmedchem.8b01631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akuffo, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alontaga, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beatty, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hesterberg, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodheart, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunawan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karim, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orobello, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoder, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajadhyaksha, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epling-Burnette, P. K.</span></span> <span> </span><span class="NLM_article-title">Ligand-mediated protein degradation reveals functional conservation among sequence variants of the CUL4-type E3 ligase substrate receptor cereblon</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">6187</span>– <span class="NLM_lpage">6200</span>, <span class="refDoi"> DOI: 10.1074/jbc.M117.816868</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1074%2Fjbc.M117.816868" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=29449372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotFSqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=6187-6200&author=A.+A.+Akuffoauthor=A.+Y.+Alontagaauthor=R.+Metcalfauthor=M.+S.+Beattyauthor=A.+Beckerauthor=J.+M.+McDanielauthor=R.+S.+Hesterbergauthor=W.+E.+Goodheartauthor=S.+Gunawanauthor=M.+Ayazauthor=Y.+Yangauthor=M.+R.+Karimauthor=M.+E.+Orobelloauthor=K.+Danielauthor=W.+Guidaauthor=J.+A.+Yoderauthor=A.+M.+Rajadhyakshaauthor=E.+Schonbrunnauthor=H.+R.+Lawrenceauthor=N.+J.+Lawrenceauthor=P.+K.+Epling-Burnette&title=Ligand-mediated+protein+degradation+reveals+functional+conservation+among+sequence+variants+of+the+CUL4-type+E3+ligase+substrate+receptor+cereblon&doi=10.1074%2Fjbc.M117.816868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-mediated protein degradation reveals functional conservation among sequence variants of the CUL4-type E3 ligase substrate receptor cereblon</span></div><div class="casAuthors">Akuffo, Afua A.; Alontaga, Aileen Y.; Metcalf, Rainer; Beatty, Matthew S.; Becker, Andreas; McDaniel, Jessica M.; Hesterberg, Rebecca S.; Goodheart, William E.; Gunawan, Steven; Ayaz, Muhammad; Yang, Yan; Karim, Md. Rezaul; Orobello, Morgan E.; Daniel, Kenyon; Guida, Wayne; Yoder, Jeffrey A.; Rajadhyaksha, Anjali M.; Schonbrunn, Ernst; Lawrence, Harshani R.; Lawrence, Nicholas J.; Epling-Burnette, Pearlie K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6187-6200</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Upon binding to thalidomide and other immunomodulatory drugs, the E3 ligase substrate receptor cereblon (CRBN) promotes proteosomal destruction by engaging the DDB1-CUL4A-Roc1-RBX1 E3 ubiquitin ligase in human cells but not in mouse cells, suggesting that sequence variations in CRBN may cause its inactivation.  Therapeutically, CRBN engagers have the potential for broad applications in cancer and immune therapy by specifically reducing protein expression through targeted ubiquitin-mediated degrdn.  To examine the effects of defined sequence changes on CRBN's activity, we performed a comprehensive study using complementary theor., biophys., and biol. assays aimed at understanding CRBN's nonprimate sequence variations.  With a series of recombinant thalidomide-binding domain (TBD) proteins, we show that CRBN sequence variants retain their drug-binding properties to both classical immunomodulatory drugs and dBET1, a chem. compd. and targeting ligand designed to degrade bromodomain-contg. 4 (BRD4) via a CRBN-dependent mechanism.  We further show that dBET1 stimulates CRBN's E3 ubiquitin-conjugating function and degrades BRD4 in both mouse and human cells.  This insight paves the way for studies of CRBN-dependent proteasome-targeting mols. in nonprimate models and provides a new understanding of CRBN's substrate-recruiting function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIqHjqVpk5xrVg90H21EOLACvtfcHk0li_xiwEYYwbIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotFSqtb8%253D&md5=16bc835472884d8b173e96fccdc3019a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M117.816868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M117.816868%26sid%3Dliteratum%253Aachs%26aulast%3DAkuffo%26aufirst%3DA.%2BA.%26aulast%3DAlontaga%26aufirst%3DA.%2BY.%26aulast%3DMetcalf%26aufirst%3DR.%26aulast%3DBeatty%26aufirst%3DM.%2BS.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DMcDaniel%26aufirst%3DJ.%2BM.%26aulast%3DHesterberg%26aufirst%3DR.%2BS.%26aulast%3DGoodheart%26aufirst%3DW.%2BE.%26aulast%3DGunawan%26aufirst%3DS.%26aulast%3DAyaz%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DKarim%26aufirst%3DM.%2BR.%26aulast%3DOrobello%26aufirst%3DM.%2BE.%26aulast%3DDaniel%26aufirst%3DK.%26aulast%3DGuida%26aufirst%3DW.%26aulast%3DYoder%26aufirst%3DJ.%2BA.%26aulast%3DRajadhyaksha%26aufirst%3DA.%2BM.%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DLawrence%26aufirst%3DH.%2BR.%26aulast%3DLawrence%26aufirst%3DN.%2BJ.%26aulast%3DEpling-Burnette%26aufirst%3DP.%2BK.%26atitle%3DLigand-mediated%2520protein%2520degradation%2520reveals%2520functional%2520conservation%2520among%2520sequence%2520variants%2520of%2520the%2520CUL4-type%2520E3%2520ligase%2520substrate%2520receptor%2520cereblon%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D6187%26epage%3D6200%26doi%3D10.1074%2Fjbc.M117.816868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87.e5</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87.e5&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lg-QkVX1DUxEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87.e5%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chau, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, P. P.</span></span> <span> </span><span class="NLM_article-title">Development of a STAT3 reporter prostate cancer cell line for high throughput screening of STAT3 activators and inhibitors</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>377</i></span>,  <span class="NLM_fpage">627</span>– <span class="NLM_lpage">631</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2008.10.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.bbrc.2008.10.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=18930029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlykt7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=377&publication_year=2008&pages=627-631&author=M.+N.+Chauauthor=P.+P.+Banerjee&title=Development+of+a+STAT3+reporter+prostate+cancer+cell+line+for+high+throughput+screening+of+STAT3+activators+and+inhibitors&doi=10.1016%2Fj.bbrc.2008.10.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a STAT3 reporter prostate cancer cell line for high throughput screening of STAT3 activators and inhibitors</span></div><div class="casAuthors">Chau, My N.; Banerjee, Partha P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">377</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">627-631</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">STAT3 is constitutively activated in several cancers, including prostate cancer, and is therefore, a potential target for cancer therapy.  DU-145 prostate cancer cells were stably co-transfected with STAT3 reporter and puromycin resistant plasmids to create a stable STAT3 reporter cell line that can be used for high throughput screening of STAT3 modulators.  The applicability of this cell line was tested with two known activators and inhibitors of STAT3.  As expected, EGF and IL-6 increased STAT3 reporter activity and enhanced the nuclear localization of phosphorylated STAT3 (pSTAT3); whereas Cucurbitacin I and AG490 decreased STAT3 reporter activity dose and time-dependently and reduced the localization of pSTAT3 in the nuclei of prostate cancer cells.  Given the importance of STAT3 in cancer initiation and progression, the development of a stable STAT3 reporter cell line in prostate cancer cells provides a rapid, sensitive, and cost effective method for the screening of potential STAT3 modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjSY8FsvDnt7Vg90H21EOLACvtfcHk0lg-QkVX1DUxEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlykt7nE&md5=dcdbd3c066f54ef2c3fd66a1de526229</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2008.10.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2008.10.025%26sid%3Dliteratum%253Aachs%26aulast%3DChau%26aufirst%3DM.%2BN.%26aulast%3DBanerjee%26aufirst%3DP.%2BP.%26atitle%3DDevelopment%2520of%2520a%2520STAT3%2520reporter%2520prostate%2520cancer%2520cell%2520line%2520for%2520high%2520throughput%2520screening%2520of%2520STAT3%2520activators%2520and%2520inhibitors%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2008%26volume%3D377%26spage%3D627%26epage%3D631%26doi%3D10.1016%2Fj.bbrc.2008.10.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponthier, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sack, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seebacher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">pSILAC mass spectrometry reveals ZFP91 as IMiD-dependent substrate of the CRL4<sup>CRBN</sup> ubiquitin ligase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">15398</span>, <span class="refDoi"> DOI: 10.1038/ncomms15398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fncomms15398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=28530236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2sXot1yqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=15398&author=J.+Anauthor=C.+M.+Ponthierauthor=R.+Sackauthor=J.+Seebacherauthor=M.+B.+Stadlerauthor=K.+A.+Donovanauthor=E.+S.+Fischer&title=pSILAC+mass+spectrometry+reveals+ZFP91+as+IMiD-dependent+substrate+of+the+CRL4CRBN+ubiquitin+ligase&doi=10.1038%2Fncomms15398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">pSILAC mass spectrometry reveals ZFP91 as IMiD-dependent substrate of the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">An, Jian; Ponthier, Charles M.; Sack, Ragna; Seebacher, Jan; Stadler, Michael B.; Donovan, Katherine A.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15398</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thalidomide and its derivs. lenalidomide and pomalidomide (IMiDs) are effective treatments of haematol. malignancies.  It was shown that IMiDs impart gain-of-function properties to the CUL4-RBX1-DDB1-CRBN (CRL4CRBN) ubiquitin ligase that enable binding, ubiquitination and degrdn. of key therapeutic targets such as IKZF1, IKZF3 and CSNK1A1.  While these substrates have been implicated as efficacy targets in multiple myeloma (MM) and 5q deletion assocd. myelodysplastic syndrome (del(5q)-MDS), other targets likely exist.  Using a pulse-chase SILAC mass spectrometry-based proteomics approach, we demonstrate that lenalidomide induces the ubiquitination and degrdn. of ZFP91.  We establish ZFP91 as a bona fide IMiD-dependent CRL4CRBN substrate and further show that ZFP91 harbours a zinc finger (ZnF) motif, related to the IKZF1/3 ZnF, crit. for IMiD-dependent CRBN binding.  These findings demonstrate that single time point pulse-chase SILAC mass spectrometry-based proteomics (pSILAC MS) is a sensitive approach for target identification of small mols. inducing selective protein degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA_4DdmHNeC7Vg90H21EOLACvtfcHk0lg-QkVX1DUxEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXot1yqtbY%253D&md5=253c73df7a064a5f88f184ff782ef222</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fncomms15398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms15398%26sid%3Dliteratum%253Aachs%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DPonthier%26aufirst%3DC.%2BM.%26aulast%3DSack%26aufirst%3DR.%26aulast%3DSeebacher%26aufirst%3DJ.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DpSILAC%2520mass%2520spectrometry%2520reveals%2520ZFP91%2520as%2520IMiD-dependent%2520substrate%2520of%2520the%2520CRL4CRBN%2520ubiquitin%2520ligase%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D15398%26doi%3D10.1038%2Fncomms15398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structural basis of lenalidomide-induced CK1α degradation by the CRL4<sup>CRBN</sup> ubiquitin ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>532</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">130</span>, <span class="refDoi"> DOI: 10.1038/nature16979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fnature16979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=26909574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtFOis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=532&publication_year=2016&pages=127-130&author=G.+Petzoldauthor=E.+S.+Fischerauthor=N.+H.+Thom%C3%A4&title=Structural+basis+of+lenalidomide-induced+CK1%CE%B1+degradation+by+the+CRL4CRBN+ubiquitin+ligase&doi=10.1038%2Fnature16979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of lenalidomide-induced CK1α degradation by the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Petzold, Georg; Fischer, Eric S.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">532</span>
        (<span class="NLM_cas:issue">7597</span>),
    <span class="NLM_cas:pages">127-130</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thalidomide and its derivs., lenalidomide and pomalidomide, are immune modulatory drugs (IMiDs) used in the treatment of hematol. malignancies.  IMiDs bind CRBN, the substrate receptor of the CUL4-RBX1-DDB1-CRBN (also known as CRL4CRBN) E3 ubiquitin ligase, and inhibit ubiquitination of endogenous CRL4CRBN substrates.  Unexpectedly, IMiDs also repurpose the ligase to target new proteins for degrdn.  Lenalidomide induces degrdn. of the lymphoid transcription factors Ikaros and Aiolos (also known as IKZF1 and IKZF3), and casein kinase 1α (CK1α), which contributes to its clin. efficacy in the treatment of multiple myeloma and 5q-deletion assocd. myelodysplastic syndrome (del(5q) MDS), resp.  How lenalidomide alters the specificity of the ligase to degrade these proteins remains elusive.  Here we present the 2.45 Å crystal structure of DDB1-CRBN bound to lenalidomide and CK1α.  CRBN and lenalidomide jointly provide the binding interface for a CK1α β-hairpin-loop located in the kinase N-lobe.  We show that CK1α binding to CRL4CRBN is strictly dependent on the presence of an IMiD.  Binding of IKZF1 to CRBN similarly requires the compd. and both, IKZF1 and CK1α, use a related binding mode.  Our study provides a mechanistic explanation for the selective efficacy of lenalidomide in del(5q) MDS therapy.  We anticipate that high-affinity protein-protein interactions induced by small mols. will provide opportunities for drug development, particularly for targeted protein degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs4FxWriI1ObVg90H21EOLACvtfcHk0ligteZOgPGznw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtFOis78%253D&md5=320fc6ff811da335fa8e02feab2d702d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnature16979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16979%26sid%3Dliteratum%253Aachs%26aulast%3DPetzold%26aufirst%3DG.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructural%2520basis%2520of%2520lenalidomide-induced%2520CK1%25CE%25B1%2520degradation%2520by%2520the%2520CRL4CRBN%2520ubiquitin%2520ligase%26jtitle%3DNature%26date%3D2016%26volume%3D532%26spage%3D127%26epage%3D130%26doi%3D10.1038%2Fnature16979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kronke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udeshi, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grauman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConkey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svinkina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heckl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciarlo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span> <span> </span><span class="NLM_article-title">Lenalidomide causes selective degradation of IKZF1 and IKZF3 in multiple myeloma cells</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">305</span>, <span class="refDoi"> DOI: 10.1126/science.1244851</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1126%2Fscience.1244851" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=24292625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BC2c3itFajsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2014&pages=301-305&author=J.+Kronkeauthor=N.+D.+Udeshiauthor=A.+Narlaauthor=P.+Graumanauthor=S.+N.+Hurstauthor=M.+McConkeyauthor=T.+Svinkinaauthor=D.+Hecklauthor=E.+Comerauthor=X.+Liauthor=C.+Ciarloauthor=E.+Hartmanauthor=N.+Munshiauthor=M.+Schenoneauthor=S.+L.+Schreiberauthor=S.+A.+Carrauthor=B.+L.+Ebert&title=Lenalidomide+causes+selective+degradation+of+IKZF1+and+IKZF3+in+multiple+myeloma+cells&doi=10.1126%2Fscience.1244851"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Lenalidomide causes selective degradation of IKZF1 and IKZF3 in multiple myeloma cells</span></div><div class="casAuthors">Kronke Jan; Udeshi Namrata D; Narla Anupama; Grauman Peter; Hurst Slater N; McConkey Marie; Svinkina Tanya; Heckl Dirk; Comer Eamon; Li Xiaoyu; Ciarlo Christie; Hartman Emily; Munshi Nikhil; Schenone Monica; Schreiber Stuart L; Carr Steven A; Ebert Benjamin L</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">6168</span>),
    <span class="NLM_cas:pages">301-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lenalidomide is a drug with clinical efficacy in multiple myeloma and other B cell neoplasms, but its mechanism of action is unknown.  Using quantitative proteomics, we found that lenalidomide causes selective ubiquitination and degradation of two lymphoid transcription factors, IKZF1 and IKZF3, by the CRBN-CRL4 ubiquitin ligase.  IKZF1 and IKZF3 are essential transcription factors in multiple myeloma.  A single amino acid substitution of IKZF3 conferred resistance to lenalidomide-induced degradation and rescued lenalidomide-induced inhibition of cell growth.  Similarly, we found that lenalidomide-induced interleukin-2 production in T cells is due to depletion of IKZF1 and IKZF3.  These findings reveal a previously unknown mechanism of action for a therapeutic agent: alteration of the activity of an E3 ubiquitin ligase, leading to selective degradation of specific targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQiiGR0Yx7nuYFEOhA_Im3cfW6udTcc2ebj6XIzmug-ILntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3itFajsQ%253D%253D&md5=8cae3e554399308493aa38f0d05ed610</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1126%2Fscience.1244851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1244851%26sid%3Dliteratum%253Aachs%26aulast%3DKronke%26aufirst%3DJ.%26aulast%3DUdeshi%26aufirst%3DN.%2BD.%26aulast%3DNarla%26aufirst%3DA.%26aulast%3DGrauman%26aufirst%3DP.%26aulast%3DHurst%26aufirst%3DS.%2BN.%26aulast%3DMcConkey%26aufirst%3DM.%26aulast%3DSvinkina%26aufirst%3DT.%26aulast%3DHeckl%26aufirst%3DD.%26aulast%3DComer%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DCiarlo%26aufirst%3DC.%26aulast%3DHartman%26aufirst%3DE.%26aulast%3DMunshi%26aufirst%3DN.%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26atitle%3DLenalidomide%2520causes%2520selective%2520degradation%2520of%2520IKZF1%2520and%2520IKZF3%2520in%2520multiple%2520myeloma%2520cells%26jtitle%3DScience%26date%3D2014%26volume%3D343%26spage%3D301%26epage%3D305%26doi%3D10.1126%2Fscience.1244851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Havens, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conklin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldman, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakurta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klippel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span> <span> </span><span class="NLM_article-title">Immunomodulatory agents lenalidomide and pomalidomide co-stimulate T cells by inducing degradation of T cell repressors Ikaros and Aiolos via modulation of the E3 ubiquitin ligase complex CRL4(CRBN.)</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>164</i></span>,  <span class="NLM_fpage">811</span>– <span class="NLM_lpage">821</span>, <span class="refDoi"> DOI: 10.1111/bjh.12708</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1111%2Fbjh.12708" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=24328678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjslWmu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=164&publication_year=2014&pages=811-821&author=A.+K.+Gandhiauthor=J.+Kangauthor=C.+G.+Havensauthor=T.+Conklinauthor=Y.+Ningauthor=L.+Wuauthor=T.+Itoauthor=H.+Andoauthor=M.+F.+Waldmanauthor=A.+Thakurtaauthor=A.+Klippelauthor=H.+Handaauthor=T.+O.+Danielauthor=P.+H.+Schaferauthor=R.+Chopra&title=Immunomodulatory+agents+lenalidomide+and+pomalidomide+co-stimulate+T+cells+by+inducing+degradation+of+T+cell+repressors+Ikaros+and+Aiolos+via+modulation+of+the+E3+ubiquitin+ligase+complex+CRL4%28CRBN.%29&doi=10.1111%2Fbjh.12708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Immunomodulatory agents lenalidomide and pomalidomide co-stimulate T cells by inducing degradation of T cell repressors Ikaros and Aiolos via modulation of the E3 ubiquitin ligase complex CRL4CRBN</span></div><div class="casAuthors">Gandhi, Anita K.; Kang, Jian; Havens, Courtney G.; Conklin, Thomas; Ning, Yuhong; Wu, Lei; Ito, Takumi; Ando, Hideki; Waldman, Michelle F.; Thakurta, Anjan; Klippel, Anke; Handa, Hiroshi; Daniel, Thomas O.; Schafer, Peter H.; Chopra, Rajesh</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">164</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">811-821</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Cereblon (CRBN), the mol. target of lenalidomide and pomalidomide, is a substrate receptor of the cullin ring E3 ubiquitin ligase complex, CRL4CRBN.  T cell co-stimulation by lenalidomide or pomalidomide is cereblon dependent: however, the CRL4CRBN substrates responsible for T cell co-stimulation have yet to be identified.  Here we demonstrate that interaction of the transcription factors Ikaros (IKZF1, encoded by the IKZF1 gene) and Aiolos (IKZF3, encoded by the IKZF3 gene) with CRL4CRBN is induced by lenalidomide or pomalidomide.  Each agent promotes Aiolos and Ikaros binding to CRL4CRBN with enhanced ubiquitination leading to cereblon-dependent proteosomal degrdn. in T lymphocytes.  We confirm that Aiolos and Ikaros are transcriptional repressors of interleukin-2 expression.  The findings link lenalidomide- or pomalidomide-induced degrdn. of these transcriptional suppressors to well documented T cell activation.  Importantly, Aiolos could serve as a proximal pharmacodynamic marker for lenalidomide and pomalidomide, as healthy human subjects administered lenalidomide demonstrated Aiolos degrdn. in their peripheral T cells.  In conclusion, we present a mol. model in which drug binding to cereblon results in the interaction of Ikaros and Aiolos to CRL4CRBN, leading to their ubiquitination, subsequent proteasomal degrdn. and T cell activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNm80s9sQNeLVg90H21EOLACvtfcHk0ligteZOgPGznw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjslWmu74%253D&md5=8ddf00ec1e3c1b1176603f22e8832f58</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1111%2Fbjh.12708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbjh.12708%26sid%3Dliteratum%253Aachs%26aulast%3DGandhi%26aufirst%3DA.%2BK.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DHavens%26aufirst%3DC.%2BG.%26aulast%3DConklin%26aufirst%3DT.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DAndo%26aufirst%3DH.%26aulast%3DWaldman%26aufirst%3DM.%2BF.%26aulast%3DThakurta%26aufirst%3DA.%26aulast%3DKlippel%26aufirst%3DA.%26aulast%3DHanda%26aufirst%3DH.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DSchafer%26aufirst%3DP.%2BH.%26aulast%3DChopra%26aufirst%3DR.%26atitle%3DImmunomodulatory%2520agents%2520lenalidomide%2520and%2520pomalidomide%2520co-stimulate%2520T%2520cells%2520by%2520inducing%2520degradation%2520of%2520T%2520cell%2520repressors%2520Ikaros%2520and%2520Aiolos%2520via%2520modulation%2520of%2520the%2520E3%2520ubiquitin%2520ligase%2520complex%2520CRL4%2528CRBN.%2529%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2014%26volume%3D164%26spage%3D811%26epage%3D821%26doi%3D10.1111%2Fbjh.12708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matyskiela, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baughman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clayton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slade, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBrun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. W.</span></span> <span> </span><span class="NLM_article-title">Cereblon Modulators Target ZBTB16 and Its Oncogenic Fusion Partners for Degradation via Distinct Structural Degrons</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">3149</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00674</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00674" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlGqsL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=3149&author=M.+E.+Matyskielaauthor=J.+Zhuauthor=J.+M.+Baughmanauthor=T.+Claytonauthor=M.+Sladeauthor=H.+K.+Wongauthor=K.+Dangaauthor=X.+Zhengauthor=M.+Labowauthor=L.+LeBrunauthor=G.+Luauthor=P.+P.+Chamberlainauthor=J.+W.+Thompson&title=Cereblon+Modulators+Target+ZBTB16+and+Its+Oncogenic+Fusion+Partners+for+Degradation+via+Distinct+Structural+Degrons&doi=10.1021%2Facschembio.0c00674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Cereblon Modulators Target ZBTB16 and Its Oncogenic Fusion Partners for Degradation via Distinct Structural Degrons</span></div><div class="casAuthors">Matyskiela, Mary E.; Zhu, Jinyi; Baughman, Joshua M.; Clayton, Thomas; Slade, Michelle; Wong, Hon Kit; Danga, Kristina; Zheng, Xinde; Labow, Mark; LeBrun, Laurie; Lu, Gang; Chamberlain, Philip P.; Thompson, Joel W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3149-3158</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is a growing interest in using targeted protein degrdn. as a therapeutic modality in view of its potential to expand the druggable proteome.  One avenue to using this modality is via mol. glue based Cereblon E3 Ligase Modulating Drug compds.  Here, we report the identification of the transcription factor ZBTB16 as a Cereblon neosubstrate.  We also report two new Cereblon modulators, CC-3060 and CC-647, that promote ZBTB16 degrdn.  Unexpectedly, CC-3060 and CC-647 target ZBTB16 for degrdn. by primarily engaging distinct structural degrons on different zinc finger domains.  The reciprocal fusion proteins, ZBTB16-RARα and RARα-ZBTB16, which cause a rare acute promyelocytic leukemia, contain these same structural degrons and can be targeted for proteasomal degrdn. with Cereblon modulator treatment.  Thus, a targeted protein degrdn. approach via Cereblon modulators may represent a novel therapeutic strategy in acute promyelocytic leukemia where ZBTB16/RARA rearrangements are crit. disease drivers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBTWDqkgT1g7Vg90H21EOLACvtfcHk0li_81-fUg057g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlGqsL3K&md5=6c13b47b19522cce5466f5a100b68f57</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00674%26sid%3Dliteratum%253Aachs%26aulast%3DMatyskiela%26aufirst%3DM.%2BE.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DBaughman%26aufirst%3DJ.%2BM.%26aulast%3DClayton%26aufirst%3DT.%26aulast%3DSlade%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DH.%2BK.%26aulast%3DDanga%26aufirst%3DK.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLabow%26aufirst%3DM.%26aulast%3DLeBrun%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26aulast%3DThompson%26aufirst%3DJ.%2BW.%26atitle%3DCereblon%2520Modulators%2520Target%2520ZBTB16%2520and%2520Its%2520Oncogenic%2520Fusion%2520Partners%2520for%2520Degradation%2520via%2520Distinct%2520Structural%2520Degrons%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26spage%3D3149%26doi%3D10.1021%2Facschembio.0c00674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. J.</span></span> <span> </span><span class="NLM_article-title">An atypical E3 ligase zinc finger protein 91 stabilizes and activates NF-κB-inducing kinase via Lys63-linked ubiquitination</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">30539</span>– <span class="NLM_lpage">30547</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.129551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1074%2Fjbc.M110.129551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=20682767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1SlsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=30539-30547&author=X.+Jinauthor=H.+R.+Jinauthor=H.+S.+Jungauthor=S.+J.+Leeauthor=J.+H.+Leeauthor=J.+J.+Lee&title=An+atypical+E3+ligase+zinc+finger+protein+91+stabilizes+and+activates+NF-%CE%BAB-inducing+kinase+via+Lys63-linked+ubiquitination&doi=10.1074%2Fjbc.M110.129551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">An Atypical E3 Ligase Zinc Finger Protein 91 Stabilizes and Activates NF-κB-inducing Kinase via Lys63-linked Ubiquitination</span></div><div class="casAuthors">Jin, Xuejun; Jin, Hong Ri; Jung, Haeng Sun; Lee, Se Jeong; Lee, Jeong-Hyung; Lee, Jung Joon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">30539-30547</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The NF-κB transcription factors control many physiol. processes, including inflammation, immunity, and apoptosis.  Its activity contributes to the development of various cell malignancies.  NF-κB-inducing kinase (NIK) plays a pivotal role in NF-κB activation.  However, the mol. mechanism to stabilize and activate NIK remains elusive, although it is known that cIAP1/2 (cellular inhibitor of apoptosis 1 and 2) ubiquitinate NIK for degrdn.  Here, the authors report a novel NF-κB-related zinc finger protein 91 (ZFP91) that stabilizes and activates NIK in a ubiquitination-dependent manner.  The authors show that ZFP91 interacts with and promotes the Lys63-linked ubiquitination of NIK and subsequent processing of p100 to p52.  The results of in vitro biochem. assays indicate that ZFP91 functions as an E3 ligase directly to NIK.  Remarkably, the ubiquitination of NIK coincides with its Thr559 phosphorylation.  Furthermore, knockdown of ZFP91 expression by RNA interference inhibits the CD40 ligation-induced activation of NIK and p100 processing as well as the expression of noncanonical NF-κB target genes.  These data clearly indicate that ZFP91 is an important regulator of the noncanonical NF-κB pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmSf7n4uBsTbVg90H21EOLACvtfcHk0li_81-fUg057g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1SlsrrN&md5=a07919aab27d42b626a349faadf0be34</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.129551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.129551%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DH.%2BR.%26aulast%3DJung%26aufirst%3DH.%2BS.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BJ.%26atitle%3DAn%2520atypical%2520E3%2520ligase%2520zinc%2520finger%2520protein%252091%2520stabilizes%2520and%2520activates%2520NF-%25CE%25BAB-inducing%2520kinase%2520via%2520Lys63-linked%2520ubiquitination%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D30539%26epage%3D30547%26doi%3D10.1074%2Fjbc.M110.129551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X.</span></span> <span> </span><span class="NLM_article-title">Zinc finger protein 91 (ZFP91) activates HIF-1α via NF-κB/p65 to promote proliferation and tumorigenesis of colon cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">36551</span>– <span class="NLM_lpage">36562</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.9070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.18632%2Foncotarget.9070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=27144516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BC28blslOltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=36551-36562&author=J.+Maauthor=C.+Miauthor=K.+S.+Wangauthor=J.+J.+Leeauthor=X.+Jin&title=Zinc+finger+protein+91+%28ZFP91%29+activates+HIF-1%CE%B1+via+NF-%CE%BAB%2Fp65+to+promote+proliferation+and+tumorigenesis+of+colon+cancer&doi=10.18632%2Foncotarget.9070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Zinc finger protein 91 (ZFP91) activates HIF-1α via NF-κB/p65 to promote proliferation and tumorigenesis of colon cancer</span></div><div class="casAuthors">Ma Juan; Mi Chunliu; Wang Ke Si; Lee Jung Joon; Jin Xuejun</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">36551-36562</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Zinc finger protein 91 (ZFP91) has been reported to be involved in various biological processes.  However, the clinical significance and biological role of ZFP91 in colon cancer remains unknown.  Here, we show that ZFP91 expression is upregulated in patients with colon cancer.  We found that ZFP91 upregulated HIF-1α at the levels of promoter and protein in colon cancer cells.  Using chromatin immunoprecipitation, electrophoretic mobility shift assay and luciferase reporter gene assay, we found that NF-κB/p65 is required for the binding of ZFP91 to the HIF-1α promoter at -197/-188 base pairs and for the transcriptional activation of HIF-1α gene mediated by ZFP91.  Flow cytometry, 5-ethynyl-2'-deoxyuridine (EdU) incorporation and tumor xenograft assay demonstrated that ZFP91 enhanced cell proliferation of colon cancer through upregulating HIF-1α in vitro and in vivo.  Furthermore, ZFP91 is positively associated with HIF-1α in human colon cancer.  Thus, we concluded that ZFP91 activates transcriptional coregulatory protein HIF-1α through transcription factor NF-κB/p65 in the promotion of proliferation and tumorigenesis in colon cancer cell.  ZFP91 may serve as a driver gene to activate HIF-1α transcription in the development of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTvqyrzGHsbP1WTCLT8lX9SfW6udTcc2ea3awuZoBtSsbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28blslOltw%253D%253D&md5=319c7bf77a2d38d41b27284b1538a7de</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.9070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.9070%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DMi%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DK.%2BS.%26aulast%3DLee%26aufirst%3DJ.%2BJ.%26aulast%3DJin%26aufirst%3DX.%26atitle%3DZinc%2520finger%2520protein%252091%2520%2528ZFP91%2529%2520activates%2520HIF-1%25CE%25B1%2520via%2520NF-%25CE%25BAB%252Fp65%2520to%2520promote%2520proliferation%2520and%2520tumorigenesis%2520of%2520colon%2520cancer%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D36551%26epage%3D36562%26doi%3D10.18632%2Foncotarget.9070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, D.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.-H.</span></span> <span> </span><span class="NLM_article-title">The ubiquitinase ZFP91 promotes tumor cell survival and confers chemoresistance through FOXA1 destabilization</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgz085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1093%2Fcarcin%2Fbgz085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31046116" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWntr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2020&pages=56-66&author=D.-E.+Tangauthor=Y.+Daiauthor=Y.+Xuauthor=L.-W.+Linauthor=D.-Z.+Liuauthor=X.-P.+Hongauthor=M.-L.+Ouauthor=H.+W.+Jiangauthor=S.-H.+Xu&title=The+ubiquitinase+ZFP91+promotes+tumor+cell+survival+and+confers+chemoresistance+through+FOXA1+destabilization&doi=10.1093%2Fcarcin%2Fbgz085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitinase ZFP91 promotes tumor cell survival and confers chemoresistance through FOXA1 destabilization</span></div><div class="casAuthors">Tang, Dong-E.; Dai, Yong; Xu, Yong; Lin, Lie-Wen; Liu, Dong-Zhou; Hong, Xiao-Ping; Ou, Ming-Lin; Jiang, Hao-Wu; Xu, Song-Hui</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-66</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">1460-2180</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The forkhead box A1 (FOXA1), one of the forkhead class of DNA-binding proteins, functions as a transcription factor and plays a vital role in cellular control of embryonic development and cancer progression.  Downregulation of FOXA1 has reported in several types of cancer, which contributes to cancer cell survival and chemoresistance.  However, the mechanism for FOXA1 downregulation in cancer remains unclear.  Here, we report that the ubiquitination enzyme zinc figer protein 91 (ZFP91) ubiquitinates and destabilizes FOXA1, which promotes cancer cell growth.  High level of ZFP91 expression correlates with low level of FOXA1 protein in human gastric cancer (GC) cell lines and patient samples.  Furthermore, ZFP91 knockdown reduces FOXA1 polyubiquitination, which decreases FOXA1 turnover and enhances cellular sensitivity to chemotherapy.  Taken together, our fidings reveal ZFP91-FOXA1 axis plays an important role in promoting GC progression and provides us a potential therapeutic intervention in the treatment of GC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm01VlO3OLPbVg90H21EOLACvtfcHk0lg0TqR2tciN3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWntr7J&md5=32142868ef12f3f7c0301564d7b1aad2</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgz085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgz085%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DD.-E.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DL.-W.%26aulast%3DLiu%26aufirst%3DD.-Z.%26aulast%3DHong%26aufirst%3DX.-P.%26aulast%3DOu%26aufirst%3DM.-L.%26aulast%3DJiang%26aufirst%3DH.%2BW.%26aulast%3DXu%26aufirst%3DS.-H.%26atitle%3DThe%2520ubiquitinase%2520ZFP91%2520promotes%2520tumor%2520cell%2520survival%2520and%2520confers%2520chemoresistance%2520through%2520FOXA1%2520destabilization%26jtitle%3DCarcinogenesis%26date%3D2020%26volume%3D41%26spage%3D56%26epage%3D66%26doi%3D10.1093%2Fcarcin%2Fbgz085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span> <span> </span><span class="NLM_article-title">E2F is required for STAT3-mediated upregulation of cyclin B1 and Cdc2 expressions and contributes to G2-M phase transition</span>. <i>Acta. Biochim. Biophys. Sin.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">322</span>, <span class="refDoi"> DOI: 10.1093/abbs/gmy174</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1093%2Fabbs%2Fgmy174" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=30726872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVCmu7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=313-322&author=J.+Sunauthor=Y.+Duauthor=Q.+Songauthor=J.+Nanauthor=P.+Guanauthor=J.+Guoauthor=X.+Wangauthor=J.+Yangauthor=C.+Zhao&title=E2F+is+required+for+STAT3-mediated+upregulation+of+cyclin+B1+and+Cdc2+expressions+and+contributes+to+G2-M+phase+transition&doi=10.1093%2Fabbs%2Fgmy174"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">E2F is required for STAT3-mediated upregulation of cyclin B1 and Cdc2 expressions and contributes to G2-M phase transition</span></div><div class="casAuthors">Sun, Jingjie; Du, Yuping; Song, Qiaoling; Nan, Jing; Guan, Peizhu; Guo, Jihui; Wang, Xiao; Yang, Jinbo; Zhao, Chenyang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Biochimica et Biophysica Sinica</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">313-322</span>CODEN:
                <span class="NLM_cas:coden">ABBSC2</span>;
        ISSN:<span class="NLM_cas:issn">1745-7270</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Activation of transcription factor STAT3 is involved in cell proliferation, differentiation, and cell survival.  Constitutive activation of STAT3 pathway has been assocd. with the oncogenesis of various types of cancers.  It has been reported that STAT3 plays a key role in the G1 to S phase cell cycle transition induced by the cytokine receptor subunit gp130, through the upregulation of cyclins D1, D2, D3, A, and Cdc25A and the concomitant downregulation of p21 and p27.  However, its role in mediating G2-M phase transition has not been studied.  The cyclin B1/Cdc2 complex is widely accepted as the trigger of mitosis in all organisms and is believed to be necessary for progression through S phase and keep active during the G2-M transition and progression.  In the present study, we found that activation of STAT3 stimulates cyclin B1 and Cdc2 expressions.  Deletion and site-directed mutations on cyclin B1 and Cdc2 promoters indicated that E2F element mediates the upregulation of these two promoters in a STAT3-dependent manner.  The findings reported here demonstrated that STAT3 participates in modulating G2-M phase checkpoint by regulating gene expressions of cyclin B1 and Cdc2 via E2F.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWSSYKmdNxOrVg90H21EOLACvtfcHk0lg0TqR2tciN3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVCmu7fJ&md5=a5aa475fbed3f2122538f7d0225ba1ac</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fabbs%2Fgmy174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fabbs%252Fgmy174%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DQ.%26aulast%3DNan%26aufirst%3DJ.%26aulast%3DGuan%26aufirst%3DP.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DC.%26atitle%3DE2F%2520is%2520required%2520for%2520STAT3-mediated%2520upregulation%2520of%2520cyclin%2520B1%2520and%2520Cdc2%2520expressions%2520and%2520contributes%2520to%2520G2-M%2520phase%2520transition%26jtitle%3DActa.%2520Biochim.%2520Biophys.%2520Sin.%26date%3D2019%26volume%3D51%26spage%3D313%26epage%3D322%26doi%3D10.1093%2Fabbs%2Fgmy174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koganti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui-Yuen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grasser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palendira, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tangye, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhaduri-McIntosh, S.</span></span> <span> </span><span class="NLM_article-title">STAT3 interrupts ATR-Chk1 signaling to allow oncovirus-mediated cell proliferation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">4946</span>– <span class="NLM_lpage">4951</span>, <span class="refDoi"> DOI: 10.1073/pnas.1400683111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1073%2Fpnas.1400683111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=24639502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksVSjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=4946-4951&author=S.+Kogantiauthor=J.+Hui-Yuenauthor=S.+McAllisterauthor=B.+Gardnerauthor=F.+Grasserauthor=U.+Palendiraauthor=S.+G.+Tangyeauthor=A.+F.+Freemanauthor=S.+Bhaduri-McIntosh&title=STAT3+interrupts+ATR-Chk1+signaling+to+allow+oncovirus-mediated+cell+proliferation&doi=10.1073%2Fpnas.1400683111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">STAT3 interrupts ATR-Chk1 signaling to allow oncovirus-mediated cell proliferation</span></div><div class="casAuthors">Koganti, Siva; Hui-Yuen, Joyce; McAllister, Shane; Gardner, Benjamin; Grasser, Friedrich; Palendira, Umaimainthan; Tangye, Stuart G.; Freeman, Alexandra F.; Bhaduri-McIntosh, Sumita</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4946-4951</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">DNA damage response (DDR) is a signaling network that senses DNA damage and activates response pathways to coordinate cell-cycle progression and DNA repair.  Thus, DDR is crit. for maintenance of genome stability, and presents a powerful defense against tumorigenesis.  Therefore, to drive cell-proliferation and transformation, viral and cellular oncogenes need to circumvent DDR-induced cell-cycle checkpoints.  Unlike in hereditary cancers, mechanisms that attenuate DDR and disrupt cell-cycle checkpoints in sporadic cancers are not well understood.  Using Epstein-Barr virus (EBV) as a source of oncogenes, we have previously shown that EBV-driven cell proliferation requires the cellular transcription factor STAT3.  EBV infection is rapidly followed by activation and increased expression of STAT3, which mediates relaxation of the intra-S phase cell-cycle checkpoint; this facilitates viral oncogene-driven cell proliferation.  We now show that replication stress-assocd. DNA damage, which results from EBV infection, is detected by DDR.  However, signaling downstream of ATR is impaired by STAT3, leading to relaxation of the intra-S phase checkpoint.  We find that STAT3 interrupts ATR-to-Chk1 signaling by promoting loss of Claspin, a protein that assists ATR to phosphorylate Chk1.  This loss of Claspin which ultimately facilitates cell proliferation is mediated by caspase 7, a protein that typically promotes cell death.  Our findings demonstrate how STAT3, which is constitutively active in many human cancers, suppresses DDR, fundamental to tumorigenesis.  This newly recognized role for STAT3 in attenuation of DDR, discovered in the context of EBV infection, is of broad interest as the biol. of cell proliferation is central to both health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzwtYHixWYErVg90H21EOLACvtfcHk0ljbHF1bYXgJXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksVSjs7k%253D&md5=6f8ba71080289ac8ab8c04ae36fb6686</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1400683111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1400683111%26sid%3Dliteratum%253Aachs%26aulast%3DKoganti%26aufirst%3DS.%26aulast%3DHui-Yuen%26aufirst%3DJ.%26aulast%3DMcAllister%26aufirst%3DS.%26aulast%3DGardner%26aufirst%3DB.%26aulast%3DGrasser%26aufirst%3DF.%26aulast%3DPalendira%26aufirst%3DU.%26aulast%3DTangye%26aufirst%3DS.%2BG.%26aulast%3DFreeman%26aufirst%3DA.%2BF.%26aulast%3DBhaduri-McIntosh%26aufirst%3DS.%26atitle%3DSTAT3%2520interrupts%2520ATR-Chk1%2520signaling%2520to%2520allow%2520oncovirus-mediated%2520cell%2520proliferation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D4946%26epage%3D4951%26doi%3D10.1073%2Fpnas.1400683111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-K.</span></span> <span> </span><span class="NLM_article-title">STAT3 negatively regulates antiviral responses through suppression of TLR and type I IFN-mediated responses</span>. <i>Cytokine+</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">126</span>– <span class="NLM_lpage">126</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=126-126&author=W.-B.+Wangauthor=H.+K.+Dengauthor=D.+E.+Levyauthor=C.-K.+Lee&title=STAT3+negatively+regulates+antiviral+responses+through+suppression+of+TLR+and+type+I+IFN-mediated+responses"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.-B.%26aulast%3DDeng%26aufirst%3DH.%2BK.%26aulast%3DLevy%26aufirst%3DD.%2BE.%26aulast%3DLee%26aufirst%3DC.-K.%26atitle%3DSTAT3%2520negatively%2520regulates%2520antiviral%2520responses%2520through%2520suppression%2520of%2520TLR%2520and%2520type%2520I%2520IFN-mediated%2520responses%26jtitle%3DCytokine%252B%26date%3D2009%26volume%3D48%26spage%3D126%26epage%3D126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G.-R.</span></span> <span> </span><span class="NLM_article-title">ECD promotes gastric cancer metastasis by blocking E3 ligase ZFP91-mediated hnRNP F ubiquitination and degradation</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">479</span>, <span class="refDoi"> DOI: 10.1038/s41419-018-0525-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fs41419-018-0525-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=29706618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BC1MjnvVWmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=479&author=S.-H.+Xuauthor=S.+Zhuauthor=Y.+Wangauthor=J.-Z.+Huangauthor=M.+Chenauthor=Q.-X.+Wuauthor=Y.-T.+Heauthor=D.+Chenauthor=G.-R.+Yan&title=ECD+promotes+gastric+cancer+metastasis+by+blocking+E3+ligase+ZFP91-mediated+hnRNP+F+ubiquitination+and+degradation&doi=10.1038%2Fs41419-018-0525-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">ECD promotes gastric cancer metastasis by blocking E3 ligase ZFP91-mediated hnRNP F ubiquitination and degradation</span></div><div class="casAuthors">Xu Song-Hui; Zhu Song; Wang Yanjie; Huang Jin-Zhou; Chen Min; Wu Qing-Xia; He Yu-Tian; Chen De; Yan Guang-Rong; Xu Song-Hui; Zhu Song; Chen Min; Chen De; Yan Guang-Rong</div><div class="citationInfo"><span class="NLM_cas:title">Cell death & disease</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">479</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The human ortholog of the Drosophila ecdysoneless gene (ECD) is required for embryonic development and cell-cycle progression; however, its role in cancer progression and metastasis remains unclear.  Here, we found that ECD is frequently overexpressed in gastric cancer (GC), especially in metastatic GC, and is correlated with poor clinical outcomes in GC patients.  Silencing ECD inhibited GC migration and invasion in vitro and metastasis in vivo, while ECD overexpression promoted GC migration and invasion.  ECD promoted GC invasion and metastasis by protecting hnRNP F from ubiquitination and degradation.  We identified ZFP91 as the E3 ubiquitin ligase that is responsible for hnRNP F ubiquitination at Lys 185 and proteasomal degradation.  ECD competitively bound to hnRNP F via the N-terminal STG1 domain (13-383aa), preventing hnRNP F from interacting with ZFP91, thus preventing ZFP91-mediated hnRNP F ubiquitination and proteasomal degradation.  Collectively, our findings indicate that ECD promotes cancer invasion and metastasis by preventing E3 ligase ZFP91-mediated hnRNP F ubiquitination and degradation, suggesting that ECD may be a marker for poor prognosis and a potential therapeutic target for GC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT9Q4NhHtDg2ysZ1x6zOs0vfW6udTcc2eafjqGc3uy_ebntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjnvVWmtQ%253D%253D&md5=ea4b31d786a3eb7236234ce4aaf633bb</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fs41419-018-0525-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41419-018-0525-x%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.-H.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DJ.-Z.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DQ.-X.%26aulast%3DHe%26aufirst%3DY.-T.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DYan%26aufirst%3DG.-R.%26atitle%3DECD%2520promotes%2520gastric%2520cancer%2520metastasis%2520by%2520blocking%2520E3%2520ligase%2520ZFP91-mediated%2520hnRNP%2520F%2520ubiquitination%2520and%2520degradation%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2018%26volume%3D9%26spage%3D479%26doi%3D10.1038%2Fs41419-018-0525-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Inhibitory effect of RNA-mediated knockdown of zinc finger protein 91 pseudogene on pancreatic cancer cell growth and invasion</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1343</span>– <span class="NLM_lpage">1348</span>, <span class="refDoi"> DOI: 10.3892/ol.2016.4794</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.3892%2Fol.2016.4794" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=27446435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsFynurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1343-1348&author=W.+Huangauthor=N.+Liauthor=J.+Huauthor=L.+Wang&title=Inhibitory+effect+of+RNA-mediated+knockdown+of+zinc+finger+protein+91+pseudogene+on+pancreatic+cancer+cell+growth+and+invasion&doi=10.3892%2Fol.2016.4794"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitory effect of RNA-mediated knockdown of zinc finger protein 91 pseudogene on pancreatic cancer cell growth and invasion</span></div><div class="casAuthors">Huang, Weiyi; Li, Ning; Hu, Jiong; Wang, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1343-1348</span>CODEN:
                <span class="NLM_cas:coden">OLNEB5</span>;
        ISSN:<span class="NLM_cas:issn">1792-1082</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Worldwide, human pancreatic cancer is a rare malignancy with a poor prognosis.  Long non-coding RNAs (lncRNAs) are known to have a crucial role in cancer occurrence and progression; however, the role of pseudogene-expressed lncRNAs, a major type of lncRNA, have not been thoroughly analyzed in cancer.  Therefore, the present study focused on zinc finger protein 91 pseudogene (ZFP91-P).  ZFP91-P expression was initially detected in two pancreatic cancer cell lines by reverse transcription-quant. polymerase chain reaction (RT-qPCR) and the highest expression of ZFP91-P was found in the BXPC-3-H cell line.  Subsequently, BXPC-3-H cells were transfected with ZFP91-P short hairpin RNA (shRNA) using a plasmid vector and termed shZFP91-P.  Cells transfected with neg. control plasmid vector were termed shCon.  MTT and Transwell assays were performed to analyze the proliferation and migration of BXPC-3-H cells, resp., and western blotting was used to detect epithelial-mesenchymal transition markers, including vimentin and β-catenin.  The present study showed that depletion of ZFP91-P markedly decreased pancreatic cancer cell proliferation and inhibited cell migration capacity.  In addn., the expression of β-catenin increased while vimentin expression decreased.  The current findings suggest that high expression of ZFP91-P promotes the migration of BXPC-3-H cells and may be a novel marker for early diagnosis for pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnj3hLFXvTGbVg90H21EOLACvtfcHk0ljbHF1bYXgJXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsFynurk%253D&md5=3b4e0004a453cf004b91375b3aac1e0e</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.3892%2Fol.2016.4794&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2016.4794%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DInhibitory%2520effect%2520of%2520RNA-mediated%2520knockdown%2520of%2520zinc%2520finger%2520protein%252091%2520pseudogene%2520on%2520pancreatic%2520cancer%2520cell%2520growth%2520and%2520invasion%26jtitle%3DOncol.%2520Lett.%26date%3D2016%26volume%3D12%26spage%3D1343%26epage%3D1348%26doi%3D10.3892%2Fol.2016.4794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span> <span> </span><span class="NLM_article-title">GEPIA2: an enhanced web server for large-scale expression profiling and interactive analysis</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">W556</span>– <span class="NLM_lpage">W560</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1093%2Fnar%2Fgkz430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31114875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVCmur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W556-W560&author=Z.+Tangauthor=B.+Kangauthor=C.+Liauthor=T.+Chenauthor=Z.+Zhang&title=GEPIA2%3A+an+enhanced+web+server+for+large-scale+expression+profiling+and+interactive+analysis&doi=10.1093%2Fnar%2Fgkz430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">GEPIA2: an enhanced web server for large-scale expression profiling and interactive analysis</span></div><div class="casAuthors">Tang, Zefang; Kang, Boxi; Li, Chenwei; Chen, Tianxiang; Zhang, Zemin</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W556-W560</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Introduced in 2017, the GEPIA (Gene Expression Profiling Interactive Anal.) web server has been a valuable and highly cited resource for gene expression anal. based on tumor and normal samples from the TCGA and the GTEx databases.  Here, we present GEPIA2, an updated and enhanced version to provide insights with higher resoln. and more functionalities.  Featuring 198 619 isoforms and 84 cancer subtypes, GEPIA2 has extended gene expression quantification from the gene level to the transcript level, and supports anal. of a specific cancer subtype, and comparison between subtypes.  In addn., GEPIA2 has adopted new anal. techniques of gene signature quantification inspired by single-cell sequencing studies, and provides customized anal. where users can upload their own RNA-seq data and compare them with TCGA and GTEx samples.  We also offer an API for batch process and easy retrieval of the anal. results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6aL3UepcbfrVg90H21EOLACvtfcHk0lhQaXReVlVeNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVCmur0%253D&md5=d6b03c2e91117df1d75e26f5beb58195</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz430%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DKang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DGEPIA2%253A%2520an%2520enhanced%2520web%2520server%2520for%2520large-scale%2520expression%2520profiling%2520and%2520interactive%2520analysis%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3DW556%26epage%3DW560%26doi%3D10.1093%2Fnar%2Fgkz430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McAlister, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nusinow, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrychowski, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wühr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huttlin, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gygi, S. P.</span></span> <span> </span><span class="NLM_article-title">MultiNotch MS3 enables accurate, sensitive, and multiplexed detection of differential expression across cancer cell line proteomes</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">7150</span>– <span class="NLM_lpage">7158</span>, <span class="refDoi"> DOI: 10.1021/ac502040v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac502040v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1Kmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=7150-7158&author=G.+C.+McAlisterauthor=D.+P.+Nusinowauthor=M.+P.+Jedrychowskiauthor=M.+W%C3%BChrauthor=E.+L.+Huttlinauthor=B.+K.+Ericksonauthor=R.+Radauthor=W.+Haasauthor=S.+P.+Gygi&title=MultiNotch+MS3+enables+accurate%2C+sensitive%2C+and+multiplexed+detection+of+differential+expression+across+cancer+cell+line+proteomes&doi=10.1021%2Fac502040v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">MultiNotch MS3 Enables Accurate, Sensitive, and Multiplexed Detection of Differential Expression across Cancer Cell Line Proteomes</span></div><div class="casAuthors">McAlister, Graeme C.; Nusinow, David P.; Jedrychowski, Mark P.; Wuhr, Martin; Huttlin, Edward L.; Erickson, Brian K.; Rad, Ramin; Haas, Wilhelm; Gygi, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7150-7158</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multiplexed quantitation via isobaric chem. tags (e.g., tandem mass tags (TMT) and isobaric tags for relative and abs. quantitation (iTRAQ)) has the potential to revolutionize quant. proteomics.  However, until recently the utility of these tags was questionable due to reporter ion ratio distortion resulting from fragmentation of coisolated interfering species.  These interfering signals can be negated through addnl. gas-phase manipulations (e.g., MS/MS/MS (MS3) and proton-transfer reactions (PTR)).  These methods, however, have a significant sensitivity penalty.  Using isolation waveforms with multiple frequency notches (i.e., synchronous precursor selection, SPS), the authors coisolated and cofragmented multiple MS2 fragment ions, thereby increasing the no. of reporter ions in the MS3 spectrum 10-fold over the std. MS3 method (i.e., MultiNotch MS3).  By increasing the reporter ion signals, this method improves the dynamic range of reporter ion quantitation, reduces reporter ion signal variance, and ultimately produces more high-quality quant. measurements.  To demonstrate utility, the authors analyzed biol. triplicates of eight colon cancer cell lines using the MultiNotch MS3 method.  Across all the replicates the authors quantified 8378 proteins in union and 6168 proteins in common.  Taking into account that each of these quantified proteins contains eight distinct cell-line measurements, this data set encompasses 174,704 quant. ratios each measured in triplicate across the biol. replicates.  Herein, the MultiNotch MS3 method uniquely combines multiplexing capacity with quant. sensitivity and accuracy, drastically increasing the informational value obtainable from proteomic expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonIBGSGUtKzbVg90H21EOLACvtfcHk0lg1slPnlk7itw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1Kmur8%253D&md5=bda847612d4a16030b7921bc8ae6f5c6</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fac502040v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac502040v%26sid%3Dliteratum%253Aachs%26aulast%3DMcAlister%26aufirst%3DG.%2BC.%26aulast%3DNusinow%26aufirst%3DD.%2BP.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DW%25C3%25BChr%26aufirst%3DM.%26aulast%3DHuttlin%26aufirst%3DE.%2BL.%26aulast%3DErickson%26aufirst%3DB.%2BK.%26aulast%3DRad%26aufirst%3DR.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26atitle%3DMultiNotch%2520MS3%2520enables%2520accurate%252C%2520sensitive%252C%2520and%2520multiplexed%2520detection%2520of%2520differential%2520expression%2520across%2520cancer%2520cell%2520line%2520proteomes%26jtitle%3DAnal.%2520Chem.%26date%3D2014%26volume%3D86%26spage%3D7150%26epage%3D7158%26doi%3D10.1021%2Fac502040v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jinyun Dong, Xiang-Dong Cheng, Wei-Dong Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Jiang-Jiang Qin</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Update on Development of Small-Molecule STAT3 Inhibitors for Cancer Therapy: From Phosphorylation Inhibition to Protein Degradation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (13)
                                     , 8884-8915. <a href="https://doi.org/10.1021/acs.jmedchem.1c00629" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00629</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00629%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BUpdate%252Bon%252BDevelopment%252Bof%252BSmall-Molecule%252BSTAT3%252BInhibitors%252Bfor%252BCancer%252BTherapy%25253A%252BFrom%252BPhosphorylation%252BInhibition%252Bto%252BProtein%252BDegradation%26aulast%3DDong%26aufirst%3DJinyun%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D06042021%26date%3D25062021%26volume%3D64%26issue%3D13%26spage%3D8884%26epage%3D8915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martin  Reynders</span>, <span class="hlFld-ContribAuthor ">Dirk  Trauner</span>. </span><span class="cited-content_cbyCitation_article-title">Optical control of targeted protein degradation. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (7)
                                     , 969-986. <a href="https://doi.org/10.1016/j.chembiol.2021.05.010" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.05.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.05.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DOptical%252Bcontrol%252Bof%252Btargeted%252Bprotein%252Bdegradation%26aulast%3DReynders%26aufirst%3DMartin%26date%3D2021%26volume%3D28%26issue%3D7%26spage%3D969%26epage%3D986" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0026.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Discovery timeline of napabucasin highlighting select milestones. (B) Reported mechanisms of action for napabucasin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures of napabucasin, the designed PROTACs, and their respective negative controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Summary for PROTAC optimization including linker length and composition, point of linker attachment to napabucasin, and the choice of the E3 ligase ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>2</b>-<b>5a–j</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOAc, HOAc, 100 °C; (b) DIPEA, DMF, 80 °C; (c) TFA, DCM, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>3-6a–d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DIPEA, DMF, 80 °C; (b) TFA, DCM, rt; (c) NaBH(OAc)<sub>3</sub>, DCE, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>4-5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOAc, HOAc, 120 °C; (b) Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, TEA, DMF, 80 °C; (c) TFA, DCM, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0015.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>5-5a</b> and <b>5-5b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOAc, HOAc, 110 °C; (b) KHCO<sub>3</sub>, KI, DMF, 70 °C; (c) TFA, DCM, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><figure data-id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0016.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>6-6a–c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, DMF, rt; (b) TFA, rt; (c) HATU, DIPEA, DMF, rt; (d) TFA, 50 °C.</p></p></figure><figure data-id="sch6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0017.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compound <b>7-4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DIPEA, DMF, 80 °C; (b) TFA, DCM, rt; (c) HATU, DIPEA, DMF, rt.</p></p></figure><figure data-id="fig4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. <b>XD2-149</b> inhibits cell proliferation and colony formation in pancreatic cancer cell lines. (A) Design of <b>XD2-162</b>, the negative control for the PROTAC <b>XD2-149</b>. IC<sub>50</sub> values correspond to the cytotoxicity of the compounds in the MTT assay. (B) Dose–response curves of the cytotoxicity of <b>XD2-149</b> and <b>XD2-162</b> in the MTT assay after 72 h. (C) Colony formation assay for napabucasin (NP), <b>XD2-149</b>, and <b>XD2-162</b> in pancreatic cancer cell lines. Cells were treated for 7–10 days. (D) Quantification for the colony formation assay was done by dissolving the colonies using Sorensen’s buffer and reading the absorbance at 570 nm. All data presented was generated from three independent experiments and is presented as mean ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. <b>XD2-149</b> inhibits STAT3-driven gene transcription. (A) Diagram representing the principal of the STAT3 luciferase reporter assay. Transfected HEK-293 cell lines that express Renilla luciferase reporter under the transcriptional control of STAT3 were used. (B) IC<sub>50</sub> values of the tested compounds in the luciferase assay after treatment for 24 h compared to their IC<sub>50</sub> in the MTT assay at the same treatment time point. For the luciferase assay, cells were pretreated with 100 ng/mL IL6 for 24 h followed by treatment with the compounds for 24 h. The luciferin substrate was then added and the luminescence signal was measured. (C) IC<sub>50</sub> values of napabucasin (NP) and <b>XD2-149</b> in MTT and luciferase (LUC) assays after a 24 h treatment. Data was generated from three independent experiments and is presented as mean ± SD. ** denotes <i>p</i> ≤ 0.01, and *** denotes <i>p</i> ≤ 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <b>XD2-149</b> reduces the expression of STAT3, pSTAT3, and NQO1 in a proteasome-independent manner. (A) <b>XD2-149</b> reduces the expression of STAT3, pSTAT3, and NQO1 proteins, while <b>XD2-162</b> showed an effect only on pSTAT3. The small molecules <b>1-6</b> and napabucasin (NP) displayed similar effects to <b>XD2-149</b>. BxPC-3 and MIA PaCa-2 cells were treated with <b>XD2-149</b>, <b>XD2-162</b>, <b>1-6</b>, or NP at the indicated concentrations for 16 h. (B) The effect of <b>XD2-149</b> on STAT3/pSTAT3 expression is independent of the proteasome. BxPC-3 cells were pretreated with 10 μM MG132 for 2 h then with <b>XD2-149</b>, <b>XD2-162</b> or NP for 12 h. (C) <b>XD2-149</b> is possibly interfering with the protein synthesis of STAT3 and NQO1. BxPC-3 cells were pretreated with cycloheximide (CX) for 1 h followed by NP or <b>XD2-149</b> for 16 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Transcriptomic profile of <b>XD2-149</b> shows the exclusive downregulation of ZFP91 compared to napabucasin and the negative control in BxPC-3 cells with the enriched hallmark gene sets presented. BxPC-3 cells were treated with <b>XD2-149</b> (2 μM), napabucasin (2 μM), and <b>XD2-162</b> (4 μM) for 16 h. Cell lysates were analyzed by multiplexed proteomics analysis (<b>XD2-149</b>: <i>n</i> = 3, napabucasin and <b>XD2-162</b>: <i>n</i> = 2). (A) Volcano plot for <b>XD2-149</b> proteins. The significant upregulated and downregulated proteins are colored in pink (fold change (FC) > 1.5 and <i>p</i>-value ≤0.05). (B) Venn diagram showing the overlap of common proteins between the three treatments and the exclusive proteins for each. Significant proteins >1.5 fold change and <i>p</i>-value ≤0.05 were used to generate the diagram. (C) Upregulated and downregulated enriched hallmark gene sets from GSEA analysis of <b>XD2-149</b>. (D) Select enrichment plots for the top upregulated and downregulated enriched gene sets from GSEA analysis of <b>XD2-149</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. <b>XD2-149</b> degrades ZFP91 protein in a proteasome-dependent manner. (A) <b>XD2-149</b> reduces ZFP91 expression, while napabucasin (NP) has no effect. BxPC-3 cells were treated with NP or <b>XD2-149</b> at the indicated concentrations for 16 h. (B) <b>XD2-149</b> degrades ZFP91 at 16 and 24 h with no effect at 4 h. Dose–response curves of ZFP91 degradation by <b>XD2-149</b> are presented for the 16 and 24 h time points. (C) The effect of <b>XD2-149</b> on ZFP91 is proteasome-dependent. BxPC-3 cells were pretreated with 1 μM proteasome inhibitor carfilzomib for 1 h followed by <b>XD2-149</b> or <b>XD2-162</b> treatment for 16 h. (D) NP does not abolish the effect of <b>XD2-149</b>. BxPC-3 cells were treated with <b>XD2-149</b> in presence or absence of NP for 16 h. (E) Pomalidomide (POM) induces ZFP91 degradation after 16 h. (F) <b>XD2-149</b> have no effect on the expression of GSPT. BxPC-3 cells were treated with NP or <b>XD2-149</b> for 16 h. Immunoblots were quantified using ImageJ, and the DC<sub>50</sub> values were calculated using GraphPad Prism 8. * denotes <i>p</i> ≤ 0.05, ** denotes <i>p</i> ≤ 0.01, and *** denotes <i>p</i> ≤ 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Cytotoxicity of <b>XD2-149</b> is partially dependent on ZFP91. (A) Expression levels of ZFP91 in normal (gray) and pancreatic adenocarcinoma (PAAD) (red) tissues and the overall survival with low vs high ZFP91 expression in PAAD. The data was obtained from the GEPIA2 web server that is based on TCGA and GTEx databases. (B) siRNA knockdown of ZFP91 in BxPC-3 cells after 48 and 72 h. (C) IC<sub>50</sub> values of <b>XD2-149</b> show significant reduction in its cytotoxicity with ZFP91 knockdown compared to the siRNA control cells (scrRNA). BxPC-3 cells were treated with siZFP91 and NP or <b>XD2-149</b> for 48 h in the MTT assay. ** denotes <i>p</i> ≤ 0.01.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. NQO1-induced cytotoxicity of <b>XD2-149</b> is independent of ZFP91. (A) Dicoumarol partially rescues the cell death caused by <b>XD2-149</b> in CFA. BxPC-3 and MIA PaCa-2 cells were treated with <b>XD2-149</b> or napabucasin for 7–10 days. (B) Dicoumarol (DIC) effect is independent of ZFP91. BxPC-3 cells were treated with <b>XD2-149</b> in the absence or presence of DIC for 16 h. The proteasome inhibitor carfilzomib was added 1 h prior to the treatments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch7" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0018.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Compound <b>8-1</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU, DIPEA, DMF, rt.</p></p></figure><figure data-id="sch8" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0019.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Compounds <b>9-4a–b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU, DIPEA, DMF, rt; (b) TFA, DCM, rt; (c) (<i>S</i>,<i>R</i>,<i>S</i>)-AHPC hydrochloride, HATU, DIPEA, DMF, rt.</p></p></figure><figure data-id="sch9" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0020.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of Compound <b>10-5</b> (<b>XD171</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOAc, HOAc, 100 °C; (b) DIPEA, DMF, 80 °C; (c) TFA, DCM, rt; (d) HATU, DIPEA, DMF, rt.</p></p></figure><figure data-id="sch10" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/medium/jm0c01897_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0021.jpeg" id="rightTab-gr20" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Synthesis of Compound <b>11-7</b> (<b>XD2-162</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-3/acs.jmedchem.0c01897/20210204/images/large/jm0c01897_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01897&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOAc, HOAc, 100 °C; (b) DIPEA, DMF, 80 °C; (c) TFA, DCM, rt; (d) NaBH(OAc)<sub>3</sub>, DCE, rt; (e) HATU, DIPEA, DMF, rt.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i92">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63512" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63512" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 61 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingston, D. G. I.</span></span> <span> </span><span class="NLM_article-title">Plant anticancer agents. XII. Isolation and structure elucidation of new cytotoxic quinones from Tabebuia cassinoides</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">600</span>– <span class="NLM_lpage">604</span>, <span class="refDoi"> DOI: 10.1021/np50023a014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np50023a014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADyaL3sXhsVemuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=1982&pages=600-604&author=M.+M.+Raoauthor=D.+G.+I.+Kingston&title=Plant+anticancer+agents.+XII.+Isolation+and+structure+elucidation+of+new+cytotoxic+quinones+from+Tabebuia+cassinoides&doi=10.1021%2Fnp50023a014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Plant anticancer agents.  XII.  Isolation and structure elucidation of new cytotoxic quinones from Tabebuia cassinoides</span></div><div class="casAuthors">Rao, M. Madhusudana; Kingston, David G. I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">600-4</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    </div><div class="casAbstract">Isolation of the 3 new cytotoxic furonaphthoquinones I, II, and III (or IV) from T. cassinoides is described.  The compds. were characterized by 1H NMR, UV, IR, and mass spectra.  The synthesis of I is also reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3HhDIxx1hoLVg90H21EOLACvtfcHk0lgtGsLhi1GYhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhsVemuw%253D%253D&md5=4586f58fa31a9b5eb46b77c988c3d4c3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fnp50023a014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp50023a014%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DM.%2BM.%26aulast%3DKingston%26aufirst%3DD.%2BG.%2BI.%26atitle%3DPlant%2520anticancer%2520agents.%2520XII.%2520Isolation%2520and%2520structure%2520elucidation%2520of%2520new%2520cytotoxic%2520quinones%2520from%2520Tabebuia%2520cassinoides%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D1982%26volume%3D45%26spage%3D600%26epage%3D604%26doi%3D10.1021%2Fnp50023a014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirai, K.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial activity of novel furanonaphthoquinone analogs</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">700</span>– <span class="NLM_lpage">702</span>, <span class="refDoi"> DOI: 10.1128/AAC.42.3.700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1128%2FAAC.42.3.700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=9517956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADyaK1cXhvV2kur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1998&pages=700-702&author=K.+Nagataauthor=K.-I.+Hiraiauthor=J.+Koyamaauthor=Y.+Wadaauthor=T.+Tamura&title=Antimicrobial+activity+of+novel+furanonaphthoquinone+analogs&doi=10.1128%2FAAC.42.3.700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial activity of novel furanonaphthoquinone analogs</span></div><div class="casAuthors">Nagata, Kumiko; Hirai, Kei-Ichi; Koyama, Junko; Wada, Yasunao; Tamura, Toshihide</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">700-702</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Analogs of furanonaphthoquinone (FNQ) from Tecoma ipe Mart had MICs ranging from 1.56 to 25 μg/mL against Gram-pos. bacteria.  FNQ showed significantly lower MICs against methicillin-resistant Staphylococcus aureus than against methicillin-sensitive S. aureus.  FNQ inhibited Helicobacter pylori with an MIC of 0.1 μg/mL.  Fungi, including pathogenic species, were sensitive to FNQ with MICs similar to those of amphotericin B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB7nhFqCqbd7Vg90H21EOLACvtfcHk0lh87w8I-n89MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhvV2kur8%253D&md5=26cb45ea436344e5bb35678d2be3c0c8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1128%2FAAC.42.3.700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.42.3.700%26sid%3Dliteratum%253Aachs%26aulast%3DNagata%26aufirst%3DK.%26aulast%3DHirai%26aufirst%3DK.-I.%26aulast%3DKoyama%26aufirst%3DJ.%26aulast%3DWada%26aufirst%3DY.%26aulast%3DTamura%26aufirst%3DT.%26atitle%3DAntimicrobial%2520activity%2520of%2520novel%2520furanonaphthoquinone%2520analogs%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1998%26volume%3D42%26spage%3D700%26epage%3D702%26doi%3D10.1128%2FAAC.42.3.700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogoff, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murikipudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikule, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leggett, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardee, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. J.</span></span> <span> </span><span class="NLM_article-title">Suppression of cancer relapse and metastasis by inhibiting cancer stemness</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">1839</span>– <span class="NLM_lpage">1844</span>, <span class="refDoi"> DOI: 10.1073/pnas.1424171112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1073%2Fpnas.1424171112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=25605917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFKht7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=1839-1844&author=Y.+Liauthor=H.+A.+Rogoffauthor=S.+Keatesauthor=Y.+Gaoauthor=S.+Murikipudiauthor=K.+Mikuleauthor=D.+Leggettauthor=W.+Liauthor=A.+B.+Pardeeauthor=C.+J.+Li&title=Suppression+of+cancer+relapse+and+metastasis+by+inhibiting+cancer+stemness&doi=10.1073%2Fpnas.1424171112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of cancer relapse and metastasis by inhibiting cancer stemness</span></div><div class="casAuthors">Li, Youzhi; Rogoff, Harry A.; Keates, Sarah; Gao, Yuan; Murikipudi, Sylaja; Mikule, Keith; Leggett, David; Li, Wei; Pardee, Arthur B.; Li, Chiang J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1839-1844</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Partial or even complete cancer regression can be achieved in some patients with current cancer treatments.  However, such initial responses are almost always followed by relapse, with the recurrent cancer being resistant to further treatments.  The discovery of therapeutic approaches that counteract relapse is, therefore, essential for advancing cancer medicine.  Cancer cells are extremely heterogeneous, even in each individual patient, in terms of their malignant potential, drug sensitivity, and their potential to metastasize and cause relapse.  Indeed, hypermalignant cancer cells, termed cancer stem cells or stemness-high cancer cells, that are highly tumorigenic and metastatic have been isolated from cancer patients with a variety of tumor types.  Moreover, such stemness-high cancer cells are resistant to conventional chemotherapy and radiation.  Here we show that BBI608, a small mol. identified by its ability to inhibit gene transcription driven by Stat3 and cancer stemness properties, can inhibit stemness gene expression and block spherogenesis of or kill stemness-high cancer cells isolated from a variety of cancer types.  Moreover, cancer relapse and metastasis were effectively blocked by BBI608 in mice.  These data demonstrate targeting cancer stemness as a novel approach to develop the next generation of cancer therapeutics to suppress cancer relapse and metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBRIs4CUCKMrVg90H21EOLACvtfcHk0lh87w8I-n89MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFKht7k%253D&md5=7f77ee7a484e6f161a0d958743a5fa12</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1424171112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1424171112%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DRogoff%26aufirst%3DH.%2BA.%26aulast%3DKeates%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DMurikipudi%26aufirst%3DS.%26aulast%3DMikule%26aufirst%3DK.%26aulast%3DLeggett%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DPardee%26aufirst%3DA.%2BB.%26aulast%3DLi%26aufirst%3DC.%2BJ.%26atitle%3DSuppression%2520of%2520cancer%2520relapse%2520and%2520metastasis%2520by%2520inhibiting%2520cancer%2520stemness%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D1839%26epage%3D1844%26doi%3D10.1073%2Fpnas.1424171112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z. L.</span>, <span class="NLM_string-name">Chiang, J.</span></span>, ; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> ; <span class="NLM_contrib-group"><span class="NLM_string-name">Leggett, D.</span></span> <span> </span><span class="NLM_article-title">Novel STAT3 Pathway Inhibitors and Cancer Stem Cell Inhibitors</span>. <span class="NLM_patent">US2,017,019,793A1</span>, July 2, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=Z.+L.+Jiang&author=J.+Chiangauthor=W.+Liauthor=D.+Leggett&title=Novel+STAT3+Pathway+Inhibitors+and+Cancer+Stem+Cell+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DZ.%2BL.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLeggett%26aufirst%3DD.%26atitle%3DNovel%2520STAT3%2520Pathway%2520Inhibitors%2520and%2520Cancer%2520Stem%2520Cell%2520Inhibitors%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karandish, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamnoon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gange, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallik, S.</span></span> <span> </span><span class="NLM_article-title">Nucleus-targeted, echogenic polymersomes for delivering a cancer stemness inhibitor to pancreatic cancer cells</span>. <i>Biomacromolecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4122</span>– <span class="NLM_lpage">4132</span>, <span class="refDoi"> DOI: 10.1021/acs.biomac.8b01133</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biomac.8b01133" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1WksbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=4122-4132&author=F.+Karandishauthor=B.+Mamnoonauthor=L.+Fengauthor=M.+K.+Haldarauthor=L.+Xiaauthor=K.+N.+Gangeauthor=S.+Youauthor=Y.+Choiauthor=K.+Sarkarauthor=S.+Mallik&title=Nucleus-targeted%2C+echogenic+polymersomes+for+delivering+a+cancer+stemness+inhibitor+to+pancreatic+cancer+cells&doi=10.1021%2Facs.biomac.8b01133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleus-Targeted, Echogenic Polymersomes for Delivering a Cancer Stemness Inhibitor to Pancreatic Cancer Cells</span></div><div class="casAuthors">Karandish, Fataneh; Mamnoon, Babak; Feng, Li; Haldar, Manas K.; Xia, Lang; Gange, Kara N.; You, Seungyong; Choi, Yongki; Sarkar, Kausik; Mallik, Sanku</div><div class="citationInfo"><span class="NLM_cas:title">Biomacromolecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4122-4132</span>CODEN:
                <span class="NLM_cas:coden">BOMAF6</span>;
        ISSN:<span class="NLM_cas:issn">1525-7797</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chemotherapeutic agents for treating cancers show considerable side effects, toxicity, and drug resistance.  To mitigate the problems, we designed nucleus-targeted, echogenic, stimuli-responsive polymeric vesicles (polymersomes) to transport and subsequently release the encapsulated anticancer drugs within the nuclei of pancreatic cancer cells.  We synthesized an alkyne-dexamethasone deriv. and conjugated it to N3-polyethylene glycol (PEG)-polylactic acid (PLA) copolymer employing the Cu2+ catalyzed "Click" reaction.  We prepd. polymersomes from the dexamethasone-PEG-PLA conjugate along with a synthesized stimuli-responsive polymer PEG-S-S-PLA.  The dexamethasone group dilates the nuclear pore complexes and transports the vesicles to the nuclei.  We designed the polymersomes to release the encapsulated drugs in the presence of a high concn. of reducing agents in the nuclei of pancreatic cancer cells.  We obsd. that the nucleus-targeted, stimuli-responsive polymersomes released 70% of encapsulated contents in the nucleus-mimicking environment in 80 min.  We encapsulated the cancer stemness inhibitor BBI608 in the vesicles and obsd. that the BBI608 encapsulated polymersomes reduced the viability of the BxPC3 cells to 43% in three-dimensional spheroid cultures.  The polymersomes were prepd. following a special protocol so that they scatter ultrasound, allowing imaging by a medical ultrasound scanner.  Therefore, these echogenic, targeted, stimuli-responsive, and drug-encapsulated polymersomes have the potential for trackable, targeted carrier of chemotherapeutic drugs to cancer cell nuclei.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIrfwt25qtC7Vg90H21EOLACvtfcHk0lj6hfH3AoxJoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1WksbfN&md5=8cbd99021cd2df174002181ba59c8274</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facs.biomac.8b01133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biomac.8b01133%26sid%3Dliteratum%253Aachs%26aulast%3DKarandish%26aufirst%3DF.%26aulast%3DMamnoon%26aufirst%3DB.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DHaldar%26aufirst%3DM.%2BK.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DGange%26aufirst%3DK.%2BN.%26aulast%3DYou%26aufirst%3DS.%26aulast%3DChoi%26aufirst%3DY.%26aulast%3DSarkar%26aufirst%3DK.%26aulast%3DMallik%26aufirst%3DS.%26atitle%3DNucleus-targeted%252C%2520echogenic%2520polymersomes%2520for%2520delivering%2520a%2520cancer%2520stemness%2520inhibitor%2520to%2520pancreatic%2520cancer%2520cells%26jtitle%3DBiomacromolecules%26date%3D2018%26volume%3D19%26spage%3D4122%26epage%3D4132%26doi%3D10.1021%2Facs.biomac.8b01133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span> <i>Search Orphan Drug Designations
and Approvals</i>; . (accessed October 10, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Search+Orphan+Drug+Designations%0Aand+Approvals%3B+.+%28accessed+October+10%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26jtitle%3DSearch%2520Orphan%2520Drug%2520Designations%250Aand%2520Approvals" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neoptolemos, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleeff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costello, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenhalf, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, D. H.</span></span> <span> </span><span class="NLM_article-title">Therapeutic developments in pancreatic cancer: current and future perspectives</span>. <i>Nat. Rev. Gastroenterol. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">348</span>, <span class="refDoi"> DOI: 10.1038/s41575-018-0005-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fs41575-018-0005-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=29717230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BC1MjovVOguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=333-348&author=J.+P.+Neoptolemosauthor=J.+Kleeffauthor=P.+Michlauthor=E.+Costelloauthor=W.+Greenhalfauthor=D.+H.+Palmer&title=Therapeutic+developments+in+pancreatic+cancer%3A+current+and+future+perspectives&doi=10.1038%2Fs41575-018-0005-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic developments in pancreatic cancer: current and future perspectives</span></div><div class="casAuthors">Neoptolemos John P; Kleeff Jorg; Kleeff Jorg; Costello Eithne; Greenhalf William; Palmer Daniel H; Michl Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Nature reviews. Gastroenterology & hepatology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">333-348</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The overall 5-year survival for pancreatic cancer has changed little over the past few decades, and pancreatic cancer is predicted to be the second leading cause of cancer-related mortality in the next decade in Western countries.  The past few years, however, have seen improvements in first-line and second-line palliative therapies and considerable progress in increasing survival with adjuvant treatment.  The use of biomarkers to help define treatment and the potential of neoadjuvant therapies also offer opportunities to improve outcomes.  This Review brings together information on achievements to date, what is working currently and where successes are likely to be achieved in the future.  Furthermore, we address the questions of how we should approach the development of pancreatic cancer treatments, including those for patients with metastatic, locally advanced and borderline resectable pancreatic cancer, as well as for patients with resected tumours.  In addition to embracing newer strategies comprising genomics, stromal therapies and immunotherapies, conventional approaches using chemotherapy and radiotherapy still offer considerable prospects for greater traction and synergy with evolving concepts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcStGgd4rHkEDz69zv1FsJFMfW6udTcc2eb2YuOl68PgWrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjovVOguw%253D%253D&md5=d1b4ab24b72f95c7077798a8258ccf58</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fs41575-018-0005-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41575-018-0005-x%26sid%3Dliteratum%253Aachs%26aulast%3DNeoptolemos%26aufirst%3DJ.%2BP.%26aulast%3DKleeff%26aufirst%3DJ.%26aulast%3DMichl%26aufirst%3DP.%26aulast%3DCostello%26aufirst%3DE.%26aulast%3DGreenhalf%26aufirst%3DW.%26aulast%3DPalmer%26aufirst%3DD.%2BH.%26atitle%3DTherapeutic%2520developments%2520in%2520pancreatic%2520cancer%253A%2520current%2520and%2520future%2520perspectives%26jtitle%3DNat.%2520Rev.%2520Gastroenterol.%2520Hepatol.%26date%3D2018%26volume%3D15%26spage%3D333%26epage%3D348%26doi%3D10.1038%2Fs41575-018-0005-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bekaii-Saab, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starodub, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Rayes, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaib, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanna, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikhail, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neki, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borodyansky, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span> <span> </span><span class="NLM_article-title">Phase 1b/2 trial of cancer stemness inhibitor napabucasin (NAPA) plus nab-paclitaxel (nPTX) and gemcitabine (Gem) in metastatic pancreatic adenocarcinoma (mPDAC)</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">4110</span>– <span class="NLM_lpage">4110</span>, <span class="refDoi"> DOI: 10.1200/JCO.2018.36.15_suppl.4110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1200%2FJCO.2018.36.15_suppl.4110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=4110-4110&author=T.+S.+Bekaii-Saabauthor=A.+Starodubauthor=B.+F.+El-Rayesauthor=S.+Shahdaauthor=B.+H.+O%E2%80%99Neilauthor=A.+M.+Noonanauthor=A.+M.+Noonanauthor=W.+L.+Shaibauthor=W.+T.+Hannaauthor=S.+Mikhailauthor=A.+S.+Nekiauthor=Y.+Changauthor=X.+Daiauthor=W.+Liauthor=E.+Brooksauthor=C.+Ohauthor=L.+Borodyanskyauthor=C.+Li&title=Phase+1b%2F2+trial+of+cancer+stemness+inhibitor+napabucasin+%28NAPA%29+plus+nab-paclitaxel+%28nPTX%29+and+gemcitabine+%28Gem%29+in+metastatic+pancreatic+adenocarcinoma+%28mPDAC%29&doi=10.1200%2FJCO.2018.36.15_suppl.4110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1200%2FJCO.2018.36.15_suppl.4110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2018.36.15_suppl.4110%26sid%3Dliteratum%253Aachs%26aulast%3DBekaii-Saab%26aufirst%3DT.%2BS.%26aulast%3DStarodub%26aufirst%3DA.%26aulast%3DEl-Rayes%26aufirst%3DB.%2BF.%26aulast%3DShahda%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DB.%2BH.%26aulast%3DNoonan%26aufirst%3DA.%2BM.%26aulast%3DNoonan%26aufirst%3DA.%2BM.%26aulast%3DShaib%26aufirst%3DW.%2BL.%26aulast%3DHanna%26aufirst%3DW.%2BT.%26aulast%3DMikhail%26aufirst%3DS.%26aulast%3DNeki%26aufirst%3DA.%2BS.%26aulast%3DChang%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DBrooks%26aufirst%3DE.%26aulast%3DOh%26aufirst%3DC.%26aulast%3DBorodyansky%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%26atitle%3DPhase%25201b%252F2%2520trial%2520of%2520cancer%2520stemness%2520inhibitor%2520napabucasin%2520%2528NAPA%2529%2520plus%2520nab-paclitaxel%2520%2528nPTX%2529%2520and%2520gemcitabine%2520%2528Gem%2529%2520in%2520metastatic%2520pancreatic%2520adenocarcinoma%2520%2528mPDAC%2529%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D36%26spage%3D4110%26epage%3D4110%26doi%3D10.1200%2FJCO.2018.36.15_suppl.4110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sonbol, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okusaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabernero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarulla, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Cutsem, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekaii-Saab, T.</span></span> <span> </span><span class="NLM_article-title">CanStem111P trial: a Phase III study of napabucasin plus nab-paclitaxel with gemcitabine</span>. <i>Future Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1295</span>– <span class="NLM_lpage">1302</span>, <span class="refDoi"> DOI: 10.2217/fon-2018-0903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.2217%2Ffon-2018-0903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=30768369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXotlWmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=1295-1302&author=M.+B.+Sonbolauthor=D.+H.+Ahnauthor=D.+Goldsteinauthor=T.+Okusakaauthor=J.+Taberneroauthor=T.+Macarullaauthor=M.+Reniauthor=C.+P.+Liauthor=B.+O%E2%80%99Neilauthor=E.+Van+Cutsemauthor=T.+Bekaii-Saab&title=CanStem111P+trial%3A+a+Phase+III+study+of+napabucasin+plus+nab-paclitaxel+with+gemcitabine&doi=10.2217%2Ffon-2018-0903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">CanStem111P trial: a Phase III study of napabucasin plus nab-paclitaxel with gemcitabine</span></div><div class="casAuthors">Sonbol, Mohamad Bassam; Ahn, Daniel H.; Goldstein, David; Okusaka, Takuji; Tabernero, Josep; Macarulla, Teresa; Reni, Michele; Li, Chung-Pin; O'Neil, Bert; Van Cutsem, Eric; Bekaii-Saab, Tanios</div><div class="citationInfo"><span class="NLM_cas:title">Future Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1295-1302</span>CODEN:
                <span class="NLM_cas:coden">FOUNBN</span>;
        ISSN:<span class="NLM_cas:issn">1744-8301</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Napabucasin (also known as BBI-608 or BBI608) is an investigational, oral agent hypothesized to inhibit multiple oncogenic pathways.  In this article, we describe the design and rationale for the CanStem111P clin. trial, a multicenter, randomized, open-label, Phase III study designed to det. the efficacy and safety of combining napabucasin with nab-paclitaxel and gemcitabine for first-line treatment of patients with metastatic pancreatic adenocarcinoma (NCT02993731).  Patients were randomized in a 1:1 fashion to receive weekly gemcitabine and nab-paclitaxel with or without napabucasin.  The results of this study will help define the role of this novel agent in the management of advanced pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY0Bk1A2JAWrVg90H21EOLACvtfcHk0lh5Hr01G8CEqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXotlWmsbg%253D&md5=65a65f835bd6c32a16722fa919f9b5d9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2217%2Ffon-2018-0903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Ffon-2018-0903%26sid%3Dliteratum%253Aachs%26aulast%3DSonbol%26aufirst%3DM.%2BB.%26aulast%3DAhn%26aufirst%3DD.%2BH.%26aulast%3DGoldstein%26aufirst%3DD.%26aulast%3DOkusaka%26aufirst%3DT.%26aulast%3DTabernero%26aufirst%3DJ.%26aulast%3DMacarulla%26aufirst%3DT.%26aulast%3DReni%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DC.%2BP.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DB.%26aulast%3DVan%2BCutsem%26aufirst%3DE.%26aulast%3DBekaii-Saab%26aufirst%3DT.%26atitle%3DCanStem111P%2520trial%253A%2520a%2520Phase%2520III%2520study%2520of%2520napabucasin%2520plus%2520nab-paclitaxel%2520with%2520gemcitabine%26jtitle%3DFuture%2520Oncol.%26date%3D2019%26volume%3D15%26spage%3D1295%26epage%3D1302%26doi%3D10.2217%2Ffon-2018-0903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span> <i>Meet Sumitomo Dainippon
Pharma Oncology</i>; . (accessed August 1, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Meet+Sumitomo+Dainippon%0APharma+Oncology%3B+.+%28accessed+August+1%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26jtitle%3DMeet%2520Sumitomo%2520Dainippon%250APharma%2520Oncology" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span> <span> </span><span class="NLM_article-title">A novel series of napabucasin derivatives as orally active inhibitors of signal transducer and activator of transcription 3 (STAT3)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">543</span>– <span class="NLM_lpage">554</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.10.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.ejmech.2018.10.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=30472602" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit12ltLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2019&pages=543-554&author=C.+Liauthor=C.+Chenauthor=Q.+Anauthor=T.+Yangauthor=Z.+Sangauthor=Y.+Yangauthor=Y.+Juauthor=A.+Tongauthor=Y.+Luo&title=A+novel+series+of+napabucasin+derivatives+as+orally+active+inhibitors+of+signal+transducer+and+activator+of+transcription+3+%28STAT3%29&doi=10.1016%2Fj.ejmech.2018.10.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A novel series of napabucasin derivatives as orally active inhibitors of signal transducer and activator of transcription 3 (STAT3)</span></div><div class="casAuthors">Li, Chungen; Chen, Caili; An, Qi; Yang, Tao; Sang, Zitai; Yang, Yang; Ju, Yuan; Tong, Aiping; Luo, Youfu</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">543-554</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The transcription factor STAT3 is an attractive target for a variety of cancers therapy.  Napabucasin, applied in phase III clin. trials for the treatment of a variety of cancers, was regarded as one of the most promising anticancer drug by targeting STAT3.  Herein, a novel series of napabucasin derivs. were designed and synthesized, which presented a potent inhibitory activity on a variety of cancers cells.  Among the derivs. compd. 8q exhibited potent inhibitory activity on U251, HepG2, HT29 and CT26 cells with the IC50 values of 0.22, 0.49, 0.07 and 0.14 μM, resp., which was over 10-fold more potent than napabucasin.  Treatment with compd. 8q decreased protein expression level of total STAT3 and p-STAT3Y705in vitro.  The binding of compd. 8q with STAT3 were further validated by electrophoretic mobility shift assay and surface plasmon resonance anal.  Compd. 8q has a KD of 110.2 nM for full-length STAT3 recombinant protein.  Moreover, the aq. soly. of 8q was over 4.5-fold than that of napabucasin.  In addn., compd. 8q in vivo significantly reduced tumor growth compared to untreated mice, and exhibited good safety profile, indicating its great potential as an efficacious drug candidate for oncotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdjaMcoHkmCbVg90H21EOLACvtfcHk0lh5Hr01G8CEqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit12ltLrE&md5=4b95a9d078c485681ef7cba49e51acec</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.10.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.10.067%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DAn%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DSang%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DJu%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DA.%26aulast%3DLuo%26aufirst%3DY.%26atitle%3DA%2520novel%2520series%2520of%2520napabucasin%2520derivatives%2520as%2520orally%2520active%2520inhibitors%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%2520%2528STAT3%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D162%26spage%3D543%26epage%3D554%26doi%3D10.1016%2Fj.ejmech.2018.10.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and activity of BBI608 derivatives targeting on stem cells</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.ejmech.2018.03.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=29604543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmslKgtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=39-50&author=Q.+Zhouauthor=C.+Pengauthor=F.+Duauthor=L.+Zhouauthor=Y.+Shiauthor=Y.+Duauthor=D.+Liuauthor=W.+Sunauthor=M.+Zhangauthor=G.+Chen&title=Design%2C+synthesis+and+activity+of+BBI608+derivatives+targeting+on+stem+cells&doi=10.1016%2Fj.ejmech.2018.03.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and activity of BBI608 derivatives targeting on stem cells</span></div><div class="casAuthors">Zhou, Qifan; Peng, Chen; Du, Fangyu; Zhou, Linbo; Shi, Yajie; Du, Yang; Liu, Dongdong; Sun, Wenjiao; Zhang, Meixia; Chen, Guoliang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39-50</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">STAT3 plays a vital role in maintaining the self-renewal of tumor stem cells.  BBI608, a small mol. identified by its ability to inhibit gene transcription driven by STAT3 and cancer stemness properties, can inhibit stemness gene expression and kill stemness-high cancer cells isolated from a variety of cancer types.  In order to improve the pharmacokinetic properties of BBI608 and the antitumor activity, a series of BBI608 derivs. were designed and synthesized here.  Most of these compds. were more potent than BBI608 on HepG2 cells, compd. I had the most potent inhibitory activity among them and was 5.4-fold more potent than BBI608 (IC50=11.2μM), but had considerable activity on normal liver cells L-02.  Compds. II (IC50=3.5μM) and III (IC50=2.9μM) were found to possess significant inhibitory activities and good selectivity.  The results showed that compd. LD-19 was worthy to investigate further as a lead compd. according to its potent inhibitory activity, ideal ClogP value and better water soly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMlEGtqRVLqrVg90H21EOLACvtfcHk0lg1UvwRCkeL9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmslKgtbc%253D&md5=3bd9949d032ee24e433b2e34b3ca9369</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.054%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DDu%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DG.%26atitle%3DDesign%252C%2520synthesis%2520and%2520activity%2520of%2520BBI608%2520derivatives%2520targeting%2520on%2520stem%2520cells%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D39%26epage%3D50%26doi%3D10.1016%2Fj.ejmech.2018.03.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frank, D. A.</span></span> <span> </span><span class="NLM_article-title">STAT3 as a central mediator of neoplastic cellular transformation</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>251</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2006.10.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.canlet.2006.10.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=17129668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkvFGrsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=251&publication_year=2007&pages=199-210&author=D.+A.+Frank&title=STAT3+as+a+central+mediator+of+neoplastic+cellular+transformation&doi=10.1016%2Fj.canlet.2006.10.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">STAT3 as a central mediator of neoplastic cellular transformation</span></div><div class="casAuthors">Frank, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">251</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">199-210</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Much of the focus in understanding the mol. pathogenesis of tumors has centered on kinases that are activated in cancer.  However, cancers driven by a diversity of activated kinases may have very similar pathol. and clin. properties.  This likely relates to the fact that the biol. characteristics of a tumor are driven by the pattern of gene expression in that tumor, and that a wide spectrum of activating events at the cell surface and in the cytoplasm converge on a relatively small no. of transcription factors that regulate the expression of key target genes.  One transcription factor that has been found to be activated inappropriately in a wide range of human cancers is STAT3.  STAT3 target genes are involved in fundamental events of tumor development including proliferation, survival, self-renewal, invasion, and angiogenesis.  Furthermore, there is strong evidence that STAT3 is crit. for these processes, in that inhibition of STAT3 by a variety of means can exert an anti-cancer effect.  Since normal cells are relatively tolerant of interruption in STAT3 signaling, these findings suggest that STAT3 may also be an excellent target for the mol. therapy of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo20EchEwndc7Vg90H21EOLACvtfcHk0lg1UvwRCkeL9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkvFGrsLk%253D&md5=f97845a4bd961469b5f9cecc5b02df93</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2006.10.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2006.10.017%26sid%3Dliteratum%253Aachs%26aulast%3DFrank%26aufirst%3DD.%2BA.%26atitle%3DSTAT3%2520as%2520a%2520central%2520mediator%2520of%2520neoplastic%2520cellular%2520transformation%26jtitle%3DCancer%2520Lett.%26date%3D2007%26volume%3D251%26spage%3D199%26epage%3D210%26doi%3D10.1016%2Fj.canlet.2006.10.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span> <span> </span><span class="NLM_article-title">STAT proteins: novel molecular targets for cancer drug discovery</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6613</span>– <span class="NLM_lpage">6626</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1204086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fsj.onc.1204086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=11426647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhsF2rsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=6613-6626&author=J.+Turksonauthor=R.+Jove&title=STAT+proteins%3A+novel+molecular+targets+for+cancer+drug+discovery&doi=10.1038%2Fsj.onc.1204086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">STAT proteins: novel molecular targets for cancer drug discovery</span></div><div class="casAuthors">Turkson, James; Jove, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">56</span>),
    <span class="NLM_cas:pages">6613-6626</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with 101 refs.  Signal Transducers and Activators of Transcription (STATs) are a family of cytoplasmic proteins with roles as signal messengers and transcription factors that participate in normal cellular responses to cytokines and growth factors.  Frequently, however, abnormal activity of certain STAT family members, particularly Stat3 and Stat5, is assocd. with a wide variety of human malignancies, including hematol., breast, head and neck, and prostate cancers.  Application of mol. biol. and pharmacol. tools in disease-relevant models has confirmed Stat3 as having a causal role in oncogenesis, and provided validation of Stat3 as a target for cancer drug discovery and therapeutic intervention.  Furthermore, a constitutively-active mutant form of Stat3 is sufficient to induce oncogenic transformation of cells, which form tumors in vivo.  Constitutive activation of Stat3 signaling is accompanied by upregulation of cyclin D1, c-Myc, and Bcl-x, changes consistent with subversion of normal cellular growth and survival control mechanisms.  Block of constitutive Stat3 signaling results in growth inhibition and apoptosis of Stat3-pos. tumor cells in vitro and in vivo.  The obsd. dependence of certain tumors on constitutive Stat3 signaling for growth and survival has wide implications for cancer therapy, offering the potential for preferential tumor cell killing.  This review evaluates constitutive Stat3 activation as a "cancer-causing" factor, and proposes a no. of mol. strategies for targeting Stat3 signaling for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoffTICt_aLhLVg90H21EOLACvtfcHk0lg1UvwRCkeL9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhsF2rsro%253D&md5=2011fb621ef70547ddbbef2ed253df98</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1204086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1204086%26sid%3Dliteratum%253Aachs%26aulast%3DTurkson%26aufirst%3DJ.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DSTAT%2520proteins%253A%2520novel%2520molecular%2520targets%2520for%2520cancer%2520drug%2520discovery%26jtitle%3DOncogene%26date%3D2000%26volume%3D19%26spage%3D6613%26epage%3D6626%26doi%3D10.1038%2Fsj.onc.1204086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bromberg, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrzeszczynska, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pestell, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanese, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Stat3 as an oncogene</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">303</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)81959-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2FS0092-8674%2800%2981959-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10458605" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADyaK1MXlt1Cgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1999&pages=295-303&author=J.+F.+Brombergauthor=M.+H.+Wrzeszczynskaauthor=G.+Devganauthor=Y.+Zhaoauthor=R.+G.+Pestellauthor=C.+Albaneseauthor=J.+E.+Darnell&title=Stat3+as+an+oncogene&doi=10.1016%2FS0092-8674%2800%2981959-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Stat3 as an oncogene</span></div><div class="casAuthors">Bromberg, Jacqueline F.; Wrzeszczynska, Melissa H.; Devgan, Geeta; Zhao, Yanxiang; Pestell, Richard G.; Albanese, Chris; Darnell, James E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">295-303</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">STATs are latent transcription factors that mediate cytokine- and growth factor-directed transcription.  In many human cancers and transformed cell lines, Stat3 is persistently activated, and in cell culture, active Stat3 is either required for transformation, enhances transformation, or blocks apoptosis.  Here, the substitution of two cysteine residues within the C-terminal loop of the SH2 domain of Stat3 produced a mol. that dimerizes spontaneously, binds to DNA, and activates transcription.  The Stat3-C mol. in immortalized fibroblasts causes cellular transformation scored by colony formation in soft agar and tumor formation in nude mice.  Thus, the activated Stat3 mol. by itself can mediate cellular transformation and the expts. focus attention on the importance of constitutive Stat3 activation in human tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOPJVvUEeTbVg90H21EOLACvtfcHk0livPZE6tH0xBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlt1Cgt7s%253D&md5=f2e69e246bbfae15a3c5ee7b94340300</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981959-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981959-5%26sid%3Dliteratum%253Aachs%26aulast%3DBromberg%26aufirst%3DJ.%2BF.%26aulast%3DWrzeszczynska%26aufirst%3DM.%2BH.%26aulast%3DDevgan%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DPestell%26aufirst%3DR.%2BG.%26aulast%3DAlbanese%26aufirst%3DC.%26aulast%3DDarnell%26aufirst%3DJ.%2BE.%26atitle%3DStat3%2520as%2520an%2520oncogene%26jtitle%3DCell%26date%3D1999%26volume%3D98%26spage%3D295%26epage%3D303%26doi%3D10.1016%2FS0092-8674%2800%2981959-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Löcken, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clamor, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, K.</span></span> <span> </span><span class="NLM_article-title">Napabucasin and related heterocycle-fused naphthoquinones as STAT3 inhibitors with antiproliferative activity against cancer cells</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">1636</span>– <span class="NLM_lpage">1644</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.8b00247</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.8b00247" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FnslKisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2018&pages=1636-1644&author=H.+L%C3%B6ckenauthor=C.+Clamorauthor=K.+M%C3%BCller&title=Napabucasin+and+related+heterocycle-fused+naphthoquinones+as+STAT3+inhibitors+with+antiproliferative+activity+against+cancer+cells&doi=10.1021%2Facs.jnatprod.8b00247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Napabucasin and Related Heterocycle-Fused Naphthoquinones as STAT3 Inhibitors with Antiproliferative Activity against Cancer Cells</span></div><div class="casAuthors">Locken Hauke; Clamor Cinzia; Muller Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of natural products</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1636-1644</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Napabucasin (6) and its angularly anellated isomer (7), for which the synthesis is described, together with related plant-derived naphthoquinones, were evaluated in vitro against human breast cancer (MDA-MB-231) and chronic myelogenous leukemia (K562) cells.  As observed for β-lapachone (3), the active naphthoquinones all induced apoptosis in a cell-cycle-independent fashion.  In contrast to the pyran-fused β-lapachone (3), however, the most potent furan-fused naphthoquinones were able to redox cycle and generate superoxide in cell-based assays, which was independent of NAD(P)H:quinone oxido-reductase 1.  In a homogeneous time-resolved fluorescence (HTRF) assays with MDA-MB-231 cells, both napabucasin (6) and isonapabucasin (7) were identified as targeting STAT3 phosphorylation.  In addition, drug affinity responsive target stability assays were performed to validate a direct interaction of the naphthoquinones with STAT3.  Isonapabucasin (7) turned out to be twice as potent against STAT3 as napabucasin (6) in the HTRF assay, with an EC50 in the submicromolar range, which was in excellent agreement with the potency of both agents to inhibit the growth of MDA-MB-231 cells.  Moreover, molecular docking experiments predicted different binding modes to the STAT3 SH2 domain for the linearly anellated napabucasin (6) and its angularly anellated isomer (7).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTSQACUhahgsEEgZCMqtRBifW6udTcc2eaY52n_UKGoBbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FnslKisw%253D%253D&md5=f7b8eca5ba22354ee3c13457409262f4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.8b00247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.8b00247%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25B6cken%26aufirst%3DH.%26aulast%3DClamor%26aufirst%3DC.%26aulast%3DM%25C3%25BCller%26aufirst%3DK.%26atitle%3DNapabucasin%2520and%2520related%2520heterocycle-fused%2520naphthoquinones%2520as%2520STAT3%2520inhibitors%2520with%2520antiproliferative%2520activity%2520against%2520cancer%2520cells%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2018%26volume%3D81%26spage%3D1636%26epage%3D1644%26doi%3D10.1021%2Facs.jnatprod.8b00247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rindtorff, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutros, M.</span></span> <span> </span><span class="NLM_article-title">Wnt signaling in cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1461</span>– <span class="NLM_lpage">1473</span>, <span class="refDoi"> DOI: 10.1038/onc.2016.304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fonc.2016.304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=27617575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2gsL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1461-1473&author=T.+Zhanauthor=N.+Rindtorffauthor=M.+Boutros&title=Wnt+signaling+in+cancer&doi=10.1038%2Fonc.2016.304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Wnt signaling in cancer</span></div><div class="casAuthors">Zhan, T.; Rindtorff, N.; Boutros, M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1461-1473</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Wnt signaling is one of the key cascades regulating development and stemness, and has also been tightly assocd. with cancer.  The role of Wnt signaling in carcinogenesis has most prominently been described for colorectal cancer, but aberrant Wnt signaling is obsd. in many more cancer entities.  Here, we review current insights into novel components of Wnt pathways and describe their impact on cancer development.  Furthermore, we highlight expanding functions of Wnt signaling for both solid and liq. tumors.  We also describe current findings how Wnt signaling affects maintenance of cancer stem cells, metastasis and immune control.  Finally, we provide an overview of current strategies to antagonize Wnt signaling in cancer and challenges that are assocd. with such approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5DnHKAd0M3rVg90H21EOLACvtfcHk0livPZE6tH0xBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2gsL3O&md5=abef967a4e295e8d997ea2f241b035bc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.304%26sid%3Dliteratum%253Aachs%26aulast%3DZhan%26aufirst%3DT.%26aulast%3DRindtorff%26aufirst%3DN.%26aulast%3DBoutros%26aufirst%3DM.%26atitle%3DWnt%2520signaling%2520in%2520cancer%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D1461%26epage%3D1473%26doi%3D10.1038%2Fonc.2016.304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, A.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogoff, H. A.</span></span> <span> </span><span class="NLM_article-title">Evaluation of tumor cell-tumor microenvironment component interactions as potential predictors of patient response to napabucasin</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1429</span>– <span class="NLM_lpage">1434</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-18-1242</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1158%2F1541-7786.MCR-18-1242" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31043490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1KjsL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2019&pages=1429-1434&author=A.-Y.+Changauthor=E.+Hsuauthor=J.+Patelauthor=Y.+Liauthor=M.+Zhangauthor=H.+Iguchiauthor=H.+A.+Rogoff&title=Evaluation+of+tumor+cell-tumor+microenvironment+component+interactions+as+potential+predictors+of+patient+response+to+napabucasin&doi=10.1158%2F1541-7786.MCR-18-1242"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of tumor cell-tumor microenvironment component interactions as potential predictors of patient response to napabucasin</span></div><div class="casAuthors">Chang, An-Yun; Hsu, Eric; Patel, Jaimin; Li, Yiqun; Zhang, Minjie; Iguchi, Haruhisa; Rogoff, Harry A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1429-1434</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Napabucasin is an NAD(P)H:quinone oxidoreductase 1 (NQO1)-bioactivatable small mol. hypothesized to affect multiple oncogenic pathways.  In a prespecified, retrospective anal. of the napabucasin phase III CO.23 study, overall survival was longer for napabucasin vs. placebo in patients expressing phosphorylated STAT3 (pSTAT3) in tumor cells and cells of the tumor microenvironment (TME).  We hypothesized that a connection may exist between NQO1 expression in cancer cells and pSTAT3 in tumor cells and the TME.  In 3D spheroid cocultures of cancer cells and cancer-assocd. fibroblasts, the antitumor activity of napabucasin was NQO1 dependent.  The levels of cytokines such as IL6, CXCL10, and GM-CSF were higher in NQO1-pos. vs. NQO1-deleted cocultures.  These differentially secreted cytokines promoted STAT3 phosphorylation in tumor cells and the TME.  NQO1-expressing, napabucasin-sensitive tumor cells can modify tumor cells and the TME to promote STAT3 phosphorylation, suggesting that pSTAT3 may be used to identify a subpopulation of patients who would likely respond to napabucasin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRjLHe-89V9bVg90H21EOLACvtfcHk0lhQvOYu_SK0Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1KjsL3O&md5=2788fe3d367fd97f4e1efb447be5d225</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-18-1242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-18-1242%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DA.-Y.%26aulast%3DHsu%26aufirst%3DE.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DIguchi%26aufirst%3DH.%26aulast%3DRogoff%26aufirst%3DH.%2BA.%26atitle%3DEvaluation%2520of%2520tumor%2520cell-tumor%2520microenvironment%2520component%2520interactions%2520as%2520potential%2520predictors%2520of%2520patient%2520response%2520to%2520napabucasin%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2019%26volume%3D17%26spage%3D1429%26epage%3D1434%26doi%3D10.1158%2F1541-7786.MCR-18-1242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parkinson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Deoxynyboquinones as NQO1-activated cancer therapeutics</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2715</span>– <span class="NLM_lpage">2723</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.5b00365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.5b00365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wqs7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=2715-2723&author=E.+I.+Parkinsonauthor=P.+J.+Hergenrother&title=Deoxynyboquinones+as+NQO1-activated+cancer+therapeutics&doi=10.1021%2Facs.accounts.5b00365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Deoxynyboquinones as NQO1-Activated Cancer Therapeutics</span></div><div class="casAuthors">Parkinson, Elizabeth I.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2715-2723</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the major goals of cancer therapy is the selective targeting of cancer cells over normal cells.  Unfortunately, even with recent advances, the majority of chemotherapeutics still indiscriminately kill all rapidly dividing cells.  Although these drugs are effective in certain settings, their inability to specifically target cancer results in significant dose-limiting toxicities.  One way to avoid such toxicities is to target an aspect of the cancer cell that is not shared by normal cells.  A potential cancer-specific target is the enzyme NAD(P)H quinone oxidoreductase 1 (NQO1).  NQO1 is a 2-electron reductase responsible for the detoxification of quinones.  Its expression is typically quite low in normal tissue, but it has been found to be greatly overexpressed in many types of solid tumors, including lung, breast, pancreatic, and colon cancers.  This overexpression is thought to be in response to the higher oxidative stress of the cancer cell, and it is possible that NQO1 contributes to tumor progression.  The overexpression of NQO1 and its correlation with poor patient outcome make it an intriguing target.  Although some have explored inhibiting NQO1 as an anticancer strategy, this has generally been unsuccessful.  A more promising strategy is to utilize NQO1 substrates that are activated upon redn. by NQO1.  For example, in principle, redn. of a quinone can result in a hydroquinone that is a DNA alkylator, protein inhibitor, or redn.-oxidn. cycler.  Although there are many proposed NQO1 substrates, head-to-head assays reveal only two classes of compds. that convincingly induce cancer cell death through NQO1-mediated activation.  The authors describe the discovery and development of one of these compds., the natural product deoxynyboquinone (DNQ), an excellent NQO1 substrate and anticancer agent.  A modular synthesis of DNQ was developed that enabled access to the large compd. quantities needed to conduct extensive mechanistic evaluations and animal expts.  During these evaluations, the authors found that DNQ is an outstanding NQO1 substrate that is processed much more efficiently than other putative NQO1 substrates.  Importantly, its anticancer activity is strictly dependent on the overexpression of active NQO1.  Using previous crystal structures of NQO1, novel DNQ derivs. were designed that are also excellent NQO1 substrates and possess properties that make them more attractive than the parent natural product for translational development.  Given their selectivity, potency, outstanding pharmacokinetic properties, and the ready availability of diagnostics to assess NQO1 in patients, DNQ and its derivs. have considerable potential as personalized medicines for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW3MluuGsMA7Vg90H21EOLACvtfcHk0lhQvOYu_SK0Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wqs7fI&md5=9395e54798f6a41a31b47a5ad078c9c1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.5b00365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.5b00365%26sid%3Dliteratum%253Aachs%26aulast%3DParkinson%26aufirst%3DE.%2BI.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DDeoxynyboquinones%2520as%2520NQO1-activated%2520cancer%2520therapeutics%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26spage%3D2715%26epage%3D2723%26doi%3D10.1021%2Facs.accounts.5b00365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Froeling, F. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swamynathan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deschênes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chio, I. I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosnan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alagesan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, A.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trotman, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belleau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogoff, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuveson, D. A.</span></span> <span> </span><span class="NLM_article-title">Bioactivation of napabucasin triggers reactive oxygen species-mediated cancer cell death</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">7162</span>– <span class="NLM_lpage">7174</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-19-0302</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1158%2F1078-0432.CCR-19-0302" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31527169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWitLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=7162-7174&author=F.+E.+M.+Froelingauthor=M.+M.+Swamynathanauthor=A.+Desch%C3%AAnesauthor=I.+I.+C.+Chioauthor=E.+Brosnanauthor=M.+A.+Yaoauthor=P.+Alagesanauthor=M.+Lucitoauthor=J.+Y.+Liauthor=A.-Y.+Changauthor=L.+C.+Trotmanauthor=P.+Belleauauthor=Y.+Parkauthor=H.+A.+Rogoffauthor=J.+D.+Watsonauthor=D.+A.+Tuveson&title=Bioactivation+of+napabucasin+triggers+reactive+oxygen+species-mediated+cancer+cell+death&doi=10.1158%2F1078-0432.CCR-19-0302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactivation of napabucasin triggers reactive oxygen species-mediated cancer cell death</span></div><div class="casAuthors">Froeling, Fieke E. M.; Swamynathan, Manojit Mosur; Deschenes, Astrid; Chio, Iok In Christine; Brosnan, Erin; Yao, Melissa A.; Alagesan, Priya; Lucito, Matthew; Li, Juying; Chang, An-Yun; Trotman, Lloyd C.; Belleau, Pascal; Park, Youngkyu; Rogoff, Harry A.; Watson, James D.; Tuveson, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7162-7174</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Napabucasin (2-acetylfuro-1,4-naphthoquinone or BBI-608) is a small mol. currently being clin. evaluated in various cancer types.  It has mostly been recognized for its ability to inhibit STAT3 signaling.  However, based on its chem. structure, we hypothesized that napabucasin is a substrate for intracellular oxidoreductases and therefore may exert its anticancer effect through redox cycling, resulting in reactive oxygen species (ROS) prodn. and cell death.  Exptl. Design: Binding of napabucasin to NAD(P)H:quinone oxidoreductase-1 (NQO1), and other oxidoreductases, was measured.  Pancreatic cancer cell lines were treated with napabucasin, and cell survival, ROS generation, DNA damage, transcriptomic changes, and alterations in STAT3 activation were assayed in vitro and in vivo.  Genetic knockout or pharmacol. inhibition with dicoumarol was used to evaluate the dependency on NQO1.  Results: Napabucasin was found to bind with high affinity to NQO1 and to a lesser degree to cytochrome P 450 oxidoreductase (POR).  Treatment resulted in marked induction of ROS and DNA damage with an NQO1- and ROS-dependent decrease in STAT3 phosphorylation.  Differential cytotoxic effects were obsd., where NQO1-expressing cells generating cytotoxic levels of ROS at low napabucasin concns. were more sensitive.  Cells with low or no baseline NQO1 expression also produced ROS in response to napabucasin, albeit to a lesser extent, through the one-electron reductase POR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobkyO7Js1b7LVg90H21EOLACvtfcHk0lhQvOYu_SK0Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWitLvN&md5=f2bbfdc3e8673c095f1118e361e6f81d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-19-0302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-19-0302%26sid%3Dliteratum%253Aachs%26aulast%3DFroeling%26aufirst%3DF.%2BE.%2BM.%26aulast%3DSwamynathan%26aufirst%3DM.%2BM.%26aulast%3DDesch%25C3%25AAnes%26aufirst%3DA.%26aulast%3DChio%26aufirst%3DI.%2BI.%2BC.%26aulast%3DBrosnan%26aufirst%3DE.%26aulast%3DYao%26aufirst%3DM.%2BA.%26aulast%3DAlagesan%26aufirst%3DP.%26aulast%3DLucito%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%2BY.%26aulast%3DChang%26aufirst%3DA.-Y.%26aulast%3DTrotman%26aufirst%3DL.%2BC.%26aulast%3DBelleau%26aufirst%3DP.%26aulast%3DPark%26aufirst%3DY.%26aulast%3DRogoff%26aufirst%3DH.%2BA.%26aulast%3DWatson%26aufirst%3DJ.%2BD.%26aulast%3DTuveson%26aufirst%3DD.%2BA.%26atitle%3DBioactivation%2520of%2520napabucasin%2520triggers%2520reactive%2520oxygen%2520species-mediated%2520cancer%2520cell%2520death%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2019%26volume%3D25%26spage%3D7162%26epage%3D7174%26doi%3D10.1158%2F1078-0432.CCR-19-0302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sechi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">A novel redox modulator induces a GPX4-mediated cell death that is dependent on iron and reactive oxygen species</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9838</span>– <span class="NLM_lpage">9855</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01016</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01016" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1aqsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9838-9855&author=S.+Huauthor=M.+Sechiauthor=P.+K.+Singhauthor=L.+Daiauthor=S.+McCannauthor=D.+Sunauthor=M.+Ljungmanauthor=N.+Neamati&title=A+novel+redox+modulator+induces+a+GPX4-mediated+cell+death+that+is+dependent+on+iron+and+reactive+oxygen+species&doi=10.1021%2Facs.jmedchem.0c01016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Redox Modulator Induces a GPX4-Mediated Cell Death That Is Dependent on Iron and Reactive Oxygen Species</span></div><div class="casAuthors">Hu, Shuai; Sechi, Mario; Singh, Pankaj Kumar; Dai, Lipeng; McCann, Sean; Sun, Duxin; Ljungman, Mats; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9838-9855</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthesis and biol. evaluation of a quinazoline-dione derivs. I [R = 4-cyclohexylphenyl, 9H-fluoren-2-yl, biphenyl-3-yl, etc.; R1 = H, Me] as reactive oxygen species (ROS) inducer and showing significant cytotoxicity in pancreatic cancer cells was reported.  Compd. I [R = 4-(4-methylpiperazin-1-yl)phenyl; R1 = H (II)] shared a highly similar transcriptomic profile to napabucasin, inhibited STAT3 phosphorylation, increased ROS levels and induced DNA damage.  However, while napabucasin caused necroptosis, compd. II induced an iron-, ROS-dependent and GPX4-mediated cell death.  Importantly, compd. II decreased the expression of LRPPRC and PNPT1; two proteins involved in mitochondrial RNA catabolic processes and both neg. correlated with the overall survival of pancreatic cancer patients.  Pharmacokinetics-guided lead optimization resulted in compd. I [R = 4-(4-methylpiperazin-1-yl)phenyl; R1 = Me (III)], which showed improved plasma stability and reduced toxicity in mice compared to compd. II.  Overall, compds. II and III represent novel ROS-inducing drug-like compds. warranting further development for the treatment of pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp25T74AYyvlbVg90H21EOLACvtfcHk0ljB6FJ3QKN0Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1aqsLfF&md5=7b206e886439efb9c0b045de5b60621a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01016%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DS.%26aulast%3DSechi%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DP.%2BK.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DMcCann%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DLjungman%26aufirst%3DM.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DA%2520novel%2520redox%2520modulator%2520induces%2520a%2520GPX4-mediated%2520cell%2520death%2520that%2520is%2520dependent%2520on%2520iron%2520and%2520reactive%2520oxygen%2520species%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9838%26epage%3D9855%26doi%3D10.1021%2Facs.jmedchem.0c01016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shogren, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jewison, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waletzki, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, A. L.,  II</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaszemski, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maran, A.</span></span> <span> </span><span class="NLM_article-title">Inhibition of STAT3 blocks protein synthesis and tumor metastasis in osteosarcoma cells</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">244</span>, <span class="refDoi"> DOI: 10.1186/s13046-018-0914-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1186%2Fs13046-018-0914-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=30286779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjvVCjtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2018&pages=244&author=D.+Zuoauthor=K.+L.+Shogrenauthor=J.+Zangauthor=D.+E.+Jewisonauthor=B.+E.+Waletzkiauthor=A.+L.+Millerauthor=S.+H.+Okunoauthor=Z.+Caiauthor=M.+J.+Yaszemskiauthor=A.+Maran&title=Inhibition+of+STAT3+blocks+protein+synthesis+and+tumor+metastasis+in+osteosarcoma+cells&doi=10.1186%2Fs13046-018-0914-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of STAT3 blocks protein synthesis and tumor metastasis in osteosarcoma cells</span></div><div class="casAuthors">Zuo, Dongqing; Shogren, Kristen L.; Zang, Jie; Jewison, Donna E.; Waletzki, Brian E.; Miller, Alan L, II; Okuno, Scott H.; Cai, Zhengdong; Yaszemski, Michael J.; Maran, Avudaiappan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental & Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">244/1-244/11</span>CODEN:
                <span class="NLM_cas:coden">JECRDN</span>;
        ISSN:<span class="NLM_cas:issn">1756-9966</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Osteosarcoma is the most common bone cancer.  Despite advances, mol. mechanisms assocd. with osteosarcoma have not been fully understood.  Hence, an effective treatment for osteosarcoma has yet to be developed.  Even though signal transducer and activator of transcription3 (STAT3) has been implicated, its role in pathogenesis of osteosarcoma is not fully detd.  In this study, we investigated the antitumor effect of napabucasin (NP) (BBI608), an inhibitor of STAT3 on osteosarcoma in vitro and in vivo and studied the underlying mol. mechanism.  Methods: Cell viability, colony formation, apoptosis, tumor growth and metastasis assays were performed to examine the effect of NP on osteosarcoma in vitro and in vivo.  Real-time RT-PCR, western anal., immunofluorescence and reporter assays were used to monitor the expression and activity of proteins and underlying mol. pathways.  Protein synthesis, co-immunopptn. and CAP binding assays were carried out to understand NP-mediated mechanism of actions in osteosarcoma cells.  Results: Our results show that NP treatment decreases cell viability and induces apoptosis in several osteosarcoma cell lines.  NP treatment suppresses both expression and phosphorylation of STAT3 in addn. to blocking STAT3-mediated transcription and downstream target proteins in osteosarcoma cells.  Furthermore, NP inhibits protein synthesis through regulation of the eukaryotic initiation factor 4E (eIF4E) and eIF4E-binding protein 1 (4E-BP1).  NP also inhibits the progression of osteosarcoma tumors and metastasis in vivo in an orthotopic tibial model of osteosarcoma.  Conclusions: Taken together, our investigation reveals that NP acts through a novel mechanism and inhibits osteosarcoma growth and metastasis, and could be investigated clin. for treating osteosarcoma patients alone or in combination with other drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov_lXwQAlLabVg90H21EOLACvtfcHk0ljB6FJ3QKN0Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjvVCjtr4%253D&md5=05bb3251ce5c49c4f64a2983f747d6e5</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1186%2Fs13046-018-0914-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13046-018-0914-0%26sid%3Dliteratum%253Aachs%26aulast%3DZuo%26aufirst%3DD.%26aulast%3DShogren%26aufirst%3DK.%2BL.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DJewison%26aufirst%3DD.%2BE.%26aulast%3DWaletzki%26aufirst%3DB.%2BE.%26aulast%3DMiller%26aufirst%3DA.%2BL.%26aulast%3DOkuno%26aufirst%3DS.%2BH.%26aulast%3DCai%26aufirst%3DZ.%26aulast%3DYaszemski%26aufirst%3DM.%2BJ.%26aulast%3DMaran%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520STAT3%2520blocks%2520protein%2520synthesis%2520and%2520tumor%2520metastasis%2520in%2520osteosarcoma%2520cells%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2018%26volume%3D37%26spage%3D244%26doi%3D10.1186%2Fs13046-018-0914-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.-H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0ljB6FJ3QKN0Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule control of intracellular protein levels through modulation of the ubiquitin proteasome system</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2330</span>, <span class="refDoi"> DOI: 10.1002/anie.201307761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1002%2Fanie.201307761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlegsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=2312-2330&author=D.+L.+Buckleyauthor=C.+M.+Crews&title=Small-molecule+control+of+intracellular+protein+levels+through+modulation+of+the+ubiquitin+proteasome+system&doi=10.1002%2Fanie.201307761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Control of Intracellular Protein Levels through Modulation of the Ubiquitin Proteasome System</span></div><div class="casAuthors">Buckley, Dennis L.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2312-2330</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Traditionally, biol. probes and drugs have targeted the activities of proteins (such as enzymes and receptors) that can be readily controlled by small mols.  The remaining majority of the proteome has been deemed "undruggable".  By using small-mol. modulators of the ubiquitin proteasome, protein levels, rather than protein activity, can be targeted instead, thus increasing the no. of druggable targets.  Whereas targeting of the proteasome itself can lead to a global increase in protein levels, the targeting of other components of the UPS (e.g., the E3 ubiquitin ligases) can lead to an increase in protein levels in a more targeted fashion.  Alternatively, multiple strategies for inducing protein degrdn. with small-mol. probes are emerging.  With the ability to induce and inhibit the degrdn. of targeted proteins, small-mol. modulators of the UPS have the potential to significantly expand the druggable portion of the proteome beyond traditional targets, such as enzymes and receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUSadJB1NNJ7Vg90H21EOLACvtfcHk0liIMdrzT4bf0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlegsLo%253D&md5=4d88e9c3f2dbf92c9db4aa81bd8cdac1</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201307761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201307761%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520control%2520of%2520intracellular%2520protein%2520levels%2520through%2520modulation%2520of%2520the%2520ubiquitin%2520proteasome%2520system%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D2312%26epage%3D2330%26doi%3D10.1002%2Fanie.201307761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: from chemical biology to drug discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.chembiol.2017.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=28648379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1181-1190&author=P.+M.+Crommauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+from+chemical+biology+to+drug+discovery&doi=10.1016%2Fj.chembiol.2017.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span></div><div class="casAuthors">Cromm, Philipp M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Traditional pharmaceutical drug discovery is almost exclusively focused on directly controlling protein activity to cure diseases.  Modulators of protein activity, esp. inhibitors, are developed and applied at high concn. to achieve maximal effects.  Thereby, reduced bioavailability and off-target effects can hamper compd. efficacy.  Nucleic acid-based strategies that control protein function by affecting expression have emerged as an alternative.  However, metabolic stability and broad bioavailability represent development hurdles that remain to be overcome for these approaches.  More recently, utilizing the cell's own protein destruction machinery for selective degrdn. of essential drivers of human disorders has opened up a new and exciting area of drug discovery.  Small-mol.-induced proteolysis of selected substrates offers the potential of reaching beyond the limitations of the current pharmaceutical paradigm to expand the druggable target space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrosuhmnPE0RrVg90H21EOLACvtfcHk0liIMdrzT4bf0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO&md5=3c8c6d9488396d5eb08ca51cee20bbf4</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520from%2520chemical%2520biology%2520to%2520drug%2520discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1181%26epage%3D1190%26doi%3D10.1016%2Fj.chembiol.2017.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A potent and selective small-molecule degrader of STAT3 achieves complete tumor regression in vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">511.e17</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.ccell.2019.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31715132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=498-511.e17&author=L.+Baiauthor=H.+Zhouauthor=R.+Xuauthor=Y.+Zhaoauthor=K.+Chinnaswamyauthor=D.+McEachernauthor=J.+Chenauthor=C.-Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=H.+Jiangauthor=B.+Wenauthor=P.+Kumarauthor=J.+L.+Meagherauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=A+potent+and+selective+small-molecule+degrader+of+STAT3+achieves+complete+tumor+regression+in+vivo&doi=10.1016%2Fj.ccell.2019.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Haibin; Xu, Renqi; Zhao, Yujun; Chinnaswamy, Krishnapriya; McEachern, Donna; Chen, Jianyong; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Jiang, Hui; Wen, Bo; Kumar, Praveen; Meagher, Jennifer L.; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">498-511.e17</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is an attractive cancer therapeutic target.  Here we report the discovery of SD-36(I), a small-mol. degrader of STAT3.  SD-36 potently induces the degrdn. of STAT3 protein in vitro and in vivo and demonstrates high selectivity over other STAT members.  Induced degrdn. of STAT3 results in a strong suppression of its transcription network in leukemia and lymphoma cells.  SD-36 inhibits the growth of a subset of acute myeloid leukemia and anaplastic large-cell lymphoma cell lines by inducing cell-cycle arrest and/or apoptosis.  SD-36 achieves complete and long-lasting tumor regression in multiple xenograft mouse models at well-tolerated dose schedules.  Degrdn. of STAT3 protein, therefore, is a promising cancer therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUaP49Ap9DxLVg90H21EOLACvtfcHk0liIMdrzT4bf0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO&md5=71252c17d50a1a8be829352942175bd6</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520potent%2520and%2520selective%2520small-molecule%2520degrader%2520of%2520STAT3%2520achieves%2520complete%2520tumor%2520regression%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D498%26epage%3D511.e17%26doi%3D10.1016%2Fj.ccell.2019.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of SD-36 as a potent, selective, and efficacious PROTAC degrader of STAT3 protein</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11280</span>– <span class="NLM_lpage">11300</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01530</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01530" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11280-11300&author=H.+Zhouauthor=L.+Baiauthor=R.+Xuauthor=Y.+Zhaoauthor=J.+Chenauthor=D.+McEachernauthor=K.+Chinnaswamyauthor=B.+Wenauthor=L.+Daiauthor=P.+Kumarauthor=C.-Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=J.+L.+Meagherauthor=H.+Yiauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-based+discovery+of+SD-36+as+a+potent%2C+selective%2C+and+efficacious+PROTAC+degrader+of+STAT3+protein&doi=10.1021%2Facs.jmedchem.9b01530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of SD-36 as a Potent, Selective, and Efficacious PROTAC Degrader of STAT3 Protein</span></div><div class="casAuthors">Zhou, Haibin; Bai, Longchuan; Xu, Renqi; Zhao, Yujun; Chen, Jianyong; McEachern, Donna; Chinnaswamy, Krishnapriya; Wen, Bo; Dai, Lipeng; Kumar, Praveen; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Meagher, Jennifer L.; Yi, Han; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11280-11300</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is a transcription factor and an attractive therapeutic target for cancer and other human diseases.  Despite twenty-years of persistent research efforts, targeting STAT3 has been very challenging.  The authors report herein the structure-based discovery of potent small-mol. STAT3 degraders based upon the proteolysis targeting chimera (PROTAC) concept.  The authors first designed SI-109 as a potent, small-mol. inhibitor of the STAT3 SH2 domain.  Employing ligands for cereblon/Cullin 4A E3 ligase and SI-109, the authors obtained a series of potent PROTAC STAT3 degraders, exemplified by SD-36.  SD-36 induces rapid STAT3 degrdn. at low nanomolar concns. in cells and fails to degrader other STAT proteins.  SD-36 achieves nanomolar cell growth inhibitory activity in leukemia and lymphoma cell lines with high levels of phosphorylated STAT3.  A single dose of SD-36 results in complete STAT3 protein degrdn. in xenograft tumor tissue and normal mouse tissues.  SD-36 achieves complete and long-lasting tumor regression in the Molm-16 xenograft tumor model at well-tolerated dose-schedules.  SD-36 is a potent, selective and efficacious STAT3 degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIAWd1hiuqy7Vg90H21EOLACvtfcHk0lgS010Vt8CkKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1SnsL3J&md5=7d0c656a2d96196a3b71829c9efe715a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01530%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-based%2520discovery%2520of%2520SD-36%2520as%2520a%2520potent%252C%2520selective%252C%2520and%2520efficacious%2520PROTAC%2520degrader%2520of%2520STAT3%2520protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11280%26epage%3D11300%26doi%3D10.1021%2Facs.jmedchem.9b01530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alabi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitale, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Androgen receptor degradation by the proteolysis-targeting chimera ARCC-4 outperforms enzalutamide in cellular models of prostate cancer drug resistance</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">100</span>, <span class="refDoi"> DOI: 10.1038/s42003-018-0105-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fs42003-018-0105-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=30271980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BB3czksVCksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=100&author=J.+Salamiauthor=S.+Alabiauthor=R.+R.+Willardauthor=N.+J.+Vitaleauthor=J.+Wangauthor=H.+Dongauthor=M.+Jinauthor=D.+P.+McDonnellauthor=A.+P.+Crewauthor=T.+K.+Neklesaauthor=C.+M.+Crews&title=Androgen+receptor+degradation+by+the+proteolysis-targeting+chimera+ARCC-4+outperforms+enzalutamide+in+cellular+models+of+prostate+cancer+drug+resistance&doi=10.1038%2Fs42003-018-0105-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Androgen receptor degradation by the proteolysis-targeting chimera ARCC-4 outperforms enzalutamide in cellular models of prostate cancer drug resistance</span></div><div class="casAuthors">Salami Jemilat; Crews Craig M; Alabi Shanique; Crews Craig M; Willard Ryan R; Vitale Nick J; Wang Jing; Dong Hanqing; Jin Meizhong; Crew Andrew P; Neklesa Taavi K; McDonnell Donald P; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Communications biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The androgen receptor is a major driver of prostate cancer and inhibition of its transcriptional activity using competitive antagonists, such as enzalutamide remains a frontline therapy for prostate cancer management.  However, the majority of patients eventually develop drug resistance.  We propose that targeting the androgen receptor for degradation via Proteolysis Targeting Chimeras (PROTACs) will be a better therapeutic strategy for targeting androgen receptor signaling in prostate cancer cells.  Here we perform a head-to-head comparison between a currently approved androgen receptor antagonist enzalutamide, and its PROTAC derivative, ARCC-4, across different cellular models of prostate cancer drug resistance.  ARCC-4 is a low-nanomolar androgen receptor degrader able to degrade about 95% of cellular androgen receptors.  ARCC-4 inhibits prostate tumor cell proliferation, degrades clinically relevant androgen receptor point mutants and unlike enzalutamide, retains antiproliferative effect in a high androgen environment.  Thus, ARCC-4 exemplifies how protein degradation can address the drug resistance hurdles of enzalutamide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQilKiHIro9_JI8NvIb08rbfW6udTcc2eaklDcAX3Dxrbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czksVCksA%253D%253D&md5=b8b7a81226dcf37d9ff430244b21f952</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs42003-018-0105-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-018-0105-8%26sid%3Dliteratum%253Aachs%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DAlabi%26aufirst%3DS.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DVitale%26aufirst%3DN.%2BJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DM.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAndrogen%2520receptor%2520degradation%2520by%2520the%2520proteolysis-targeting%2520chimera%2520ARCC-4%2520outperforms%2520enzalutamide%2520in%2520cellular%2520models%2520of%2520prostate%2520cancer%2520drug%2520resistance%26jtitle%3DCommun.%2520Biol.%26date%3D2018%26volume%3D1%26spage%3D100%26doi%3D10.1038%2Fs42003-018-0105-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of MD-224 as a first-in-class, highly potent, and efficacious proteolysis targeting chimera murine double minute 2 degrader capable of achieving complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00909</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00909" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=448-466&author=Y.+Liauthor=J.+Yangauthor=A.+Aguilarauthor=D.+McEachernauthor=S.+Przybranowskiauthor=L.+Liuauthor=C.-Y.+Yangauthor=M.+Wangauthor=X.+Hanauthor=S.+Wang&title=Discovery+of+MD-224+as+a+first-in-class%2C+highly+potent%2C+and+efficacious+proteolysis+targeting+chimera+murine+double+minute+2+degrader+capable+of+achieving+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of MD-224 as a First-in-Class, Highly Potent, and Efficacious Proteolysis Targeting Chimera Murine Double Minute 2 Degrader Capable of Achieving Complete and Durable Tumor Regression</span></div><div class="casAuthors">Li, Yangbing; Yang, Jiuling; Aguilar, Angelo; McEachern, Donna; Przybranowski, Sally; Liu, Liu; Yang, Chao-Yie; Wang, Mi; Han, Xin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">448-466</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human murine double minute 2 (MDM2) protein is a primary endogenous cellular inhibitor of the tumor suppressor p53 and has been pursued as an attractive cancer therapeutic target.  Several potent, nonpeptide, small-mol. inhibitors of MDM2 are currently in clin. development.  In this paper, we report our design, synthesis, and evaluation of small-mol. MDM2 degraders based on the proteolysis targeting chimera (PROTAC) concept.  The most promising compd. (MD-224, I) effectively induces rapid degrdn. of MDM2 at concns. <1 nM in human leukemia cells.  It achieves an IC50 value of 1.5 nM in inhibition of growth of RS4;11 cells and also low nanomolar IC50 values in a panel of leukemia cell lines.  MD-224 (I) achieves complete and durable tumor regression in vivo in the RS4;11 xenograft tumor model in mice at well-tolerated dose schedules.  MD-224 (I) is thus a highly potent and efficacious MDM2 degrader and warrants extensive evaluations as a new class of anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfVzK-tVEdk7Vg90H21EOLACvtfcHk0lgS010Vt8CkKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVChtrfK&md5=3eaa40a8b0ae15c0172ef44a2fd1b902</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00909%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520MD-224%2520as%2520a%2520first-in-class%252C%2520highly%2520potent%252C%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520murine%2520double%2520minute%25202%2520degrader%2520capable%2520of%2520achieving%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D448%26epage%3D466%26doi%3D10.1021%2Facs.jmedchem.8b00909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras for the selective degradation of Mcl-1/Bcl-2 derived from nonselective target binding ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8152</span>– <span class="NLM_lpage">8163</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00919</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00919" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFemtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8152-8163&author=Z.+Wangauthor=N.+Heauthor=Z.+Guoauthor=C.+Niuauthor=T.+Songauthor=Y.+Guoauthor=K.+Caoauthor=A.+Wangauthor=J.+Zhuauthor=X.+Zhangauthor=Z.+Zhang&title=Proteolysis+targeting+chimeras+for+the+selective+degradation+of+Mcl-1%2FBcl-2+derived+from+nonselective+target+binding+ligands&doi=10.1021%2Facs.jmedchem.9b00919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras for the Selective Degradation of Mcl-1/Bcl-2 Derived from Nonselective Target Binding Ligands</span></div><div class="casAuthors">Wang, Ziqian; He, Nianzhe; Guo, Zongwei; Niu, Cuili; Song, Ting; Guo, Yafei; Cao, Keke; Wang, Anhui; Zhu, Junjie; Zhang, Xiaodong; Zhang, Zhichao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8152-8163</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) recruits an E3 ligase to a target protein to induce its ubiquitination and subsequent degrdn.  We reported the success in the development of two PROTACs (C3(I) and C5(II)) that potently and selectively induce the degrdn. of Mcl-1 and Bcl-2 (DC50 = 0.7 and 3.0 μM), resp., by introducing the E3 ligase cereblon (CRBN)-binding ligand pomalidomide to Mcl-1/Bcl-2 dual inhibitors S1-6 and Nap-1 with μM-range affinity.  I-induced Mcl-1 ubiquitination translated into much more lethality in Mcl-1-dependent H23 cells than the most potent Mcl-1 occupancy-based inhibitor A-1210477 with nM-range affinity.  Moreover, structure-activity relationship (SAR) anal. and mol. dynamic (MD) simulations discovered the structural basis for turning nonselective or promiscuous Bcl-2 family ligands into selective PROTACs.  I and II exhibited reversible depletion in living cells, which provides a new potent toolkit for gain-of-function studies to probe the dynamic roles of Bcl-2 and Mcl-1 in apoptosis networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9YwjuYRj58LVg90H21EOLACvtfcHk0lh8Vc3PgWq1Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFemtLvN&md5=f962278d69b69117c501bc7c0c31ebfd</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00919%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DHe%26aufirst%3DN.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DNiu%26aufirst%3DC.%26aulast%3DSong%26aufirst%3DT.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520for%2520the%2520selective%2520degradation%2520of%2520Mcl-1%252FBcl-2%2520derived%2520from%2520nonselective%2520target%2520binding%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8152%26epage%3D8163%26doi%3D10.1021%2Facs.jmedchem.9b00919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lh8Vc3PgWq1Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B. E.</span></span> <span> </span><span class="NLM_article-title">Cereblon modulators: Low molecular weight inducers of protein degradation</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.ddtec.2019.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31200856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=29-34&author=P.+P.+Chamberlainauthor=B.+E.+Cathers&title=Cereblon+modulators%3A+Low+molecular+weight+inducers+of+protein+degradation&doi=10.1016%2Fj.ddtec.2019.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Cereblon modulators: Low molecular weight inducers of protein degradation</span></div><div class="casAuthors">Chamberlain Philip P; Cathers Brian E</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">29-34</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Targeted protein degradation has become an exciting new paradigm in drug discovery with the potential to target new protein families for therapeutic intervention.  In 2010, Hiroshi Handa and colleagues discovered that the drug thalidomide binds to the protein cereblon, a component of the CRL4(CRBN) E3 ubiquitin ligase.  In contrast to the heterobifunctional small molecule degraders reported in the literature, thalidomide is of very low molecular weight (∼258Da) with molecular properties (solubility, metabolic stability, permeability etc) that readily support pharmaceutical dosing.  It was subsequently shown that thalidomide and the analogues lenalidomide and pomalidomide are able to degrade the transcription factors Ikaros and Aiolos.  CK1α and GSPT1 were subsequently identified as substrates for specific ligands, indicating that this molecular class could be tuned for selective protein degradation.  Structural studies showed that the thalidomide analogues bind to a shallow hydrophobic pocket on the surface of cereblon, and scaffold a protein-protein interaction with target proteins.  Target proteins do not need any affinity for the cereblon modulators, and as such undruggable, or even unligandable, proteins can be targeted for degradation.  A similar mechanism of action was subsequently identified for the clinical molecule indisulam, indicating that low molecular weight degraders are not unique to cereblon.  The groundbreaking work on cereblon represents a case study for the discovery and characterization of low molecular weight protein degraders for other ligases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXobBItaL1WaVAfW6udTcc2eYpF-clmTADiLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsg%253D%253D&md5=ab46406dc0664c8efe3d2fbbf9bd061b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26aulast%3DCathers%26aufirst%3DB.%2BE.%26atitle%3DCereblon%2520modulators%253A%2520Low%2520molecular%2520weight%2520inducers%2520of%2520protein%2520degradation%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D29%26epage%3D34%26doi%3D10.1016%2Fj.ddtec.2019.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Girardini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Cereblon versus VHL: Hijacking E3 ligases against each other using PROTACs</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2466</span>– <span class="NLM_lpage">2479</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2019.02.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.bmc.2019.02.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=30826187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXktV2rsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=2466-2479&author=M.+Girardiniauthor=C.+Maniaciauthor=S.+J.+Hughesauthor=A.+Testaauthor=A.+Ciulli&title=Cereblon+versus+VHL%3A+Hijacking+E3+ligases+against+each+other+using+PROTACs&doi=10.1016%2Fj.bmc.2019.02.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Cereblon versus VHL: Hijacking E3 ligases against each other using PROTACs</span></div><div class="casAuthors">Girardini, Miriam; Maniaci, Chiara; Hughes, Scott J.; Testa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2466-2479</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The von Hippel-Lindau (VHL) and cereblon (CRBN) proteins are substrate recognition subunits of two ubiquitously expressed and biol. important Cullin RING E3 ubiquitin ligase complexes.  VHL and CRBN are also the two most popular E3 ligases being recruited by bifunctional Proteolysis-targeting chimeras (PROTACs) to induce ubiquitination and subsequent proteasomal degrdn. of a target protein.  Using homo-PROTACs, VHL and CRBN have been independently dimerized to induce their own degrdn.  Here we report the design, synthesis and cellular activity of VHL-CRBN hetero-dimerizing PROTACs featuring diverse conjugation patterns.  We found that the most active compd. 14a induced potent, rapid and profound preferential degrdn. of CRBN over VHL in cancer cell lines.  At lower concns., weaker degrdn. of VHL was instead obsd.  This work demonstrates proof of concept of designing PROTACs to hijack different E3 ligases against each other, and highlights a powerful and generalizable proximity-induced strategy to achieve E3 ligase knockdown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGops5qv3m7tibVg90H21EOLACvtfcHk0liKPanAY3KlCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXktV2rsb8%253D&md5=ef00788b6428899cf4dfe1c8d357dd64</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.02.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.02.048%26sid%3Dliteratum%253Aachs%26aulast%3DGirardini%26aufirst%3DM.%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DCereblon%2520versus%2520VHL%253A%2520Hijacking%2520E3%2520ligases%2520against%2520each%2520other%2520using%2520PROTACs%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D2466%26epage%3D2479%26doi%3D10.1016%2Fj.bmc.2019.02.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shahbazi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polizzotto, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uldrick, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyvill, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeldis, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarchoan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figg, W. D.</span></span> <span> </span><span class="NLM_article-title">A sensitive and robust HPLC assay with fluorescence detection for the quantification of pomalidomide in human plasma for pharmacokinetic analyses</span>. <i>J. Pharm. Biomed. Anal.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/j.jpba.2014.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.jpba.2014.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=24486861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlSktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2014&pages=63-68&author=S.+Shahbaziauthor=C.+J.+Peerauthor=M.+N.+Polizzottoauthor=T.+S.+Uldrickauthor=J.+Rothauthor=K.+M.+Wyvillauthor=K.+Alemanauthor=J.+B.+Zeldisauthor=R.+Yarchoanauthor=W.+D.+Figg&title=A+sensitive+and+robust+HPLC+assay+with+fluorescence+detection+for+the+quantification+of+pomalidomide+in+human+plasma+for+pharmacokinetic+analyses&doi=10.1016%2Fj.jpba.2014.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A sensitive and robust HPLC assay with fluorescence detection for the quantification of pomalidomide in human plasma for pharmacokinetic analyses</span></div><div class="casAuthors">Shahbazi, Shandiz; Peer, Cody J.; Polizzotto, Mark N.; Uldrick, Thomas S.; Roth, Jeffrey; Wyvill, Kathleen M.; Aleman, Karen; Zeldis, Jerome B.; Yarchoan, Robert; Figg, William D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical and Biomedical Analysis</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">63-68</span>CODEN:
                <span class="NLM_cas:coden">JPBADA</span>;
        ISSN:<span class="NLM_cas:issn">0731-7085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Pomalidomide is a second generation IMiD (immunomodulatory agent) that has recently been granted approval by the Food and Drug Administration for treatment of relapsed multiple myeloma after prior treatment with two antimyeloma agents, including lenalidomide and bortezomib.  A simple and robust HPLC assay with fluorescence detection for pomalidomide over the range of 1-500 ng/mL has been developed for application to pharmacokinetic studies in ongoing clin. trials in various other malignancies.  A liq.-liq. extn. from human plasma alone or pre-stabilized with 0.1% HCl was performed, using Pr paraben as the internal std.  From plasma either pre-stabilized with 0.1% HCl or not, the assay was shown to be selective, sensitive, accurate, precise, and have minimal matrix effects (<20%).  Pomalidomide was stable in plasma through 4 freeze-thaw cycles (<12% change), in plasma at room temp. for up to 2 h for samples not pre-stabilized with 0.1% HCl and up to 8 h in samples pre-stabilized with 0.1% HCl, 24 h post-prepn. at 4 °C (<2% change), and showed excellent extn. recovery (∼90%).  This is the first reported description of the freeze/thaw and plasma stability of pomalidomide in plasma either pre-stabilized with 0.1% HCl or not.  The information presented in this manuscript is important when performing pharmacokinetic analyses.  The method was used to analyze clin. pharmacokinetics samples obtained after a 5 mg oral dose of pomalidomide.  This relatively simple HPLC-FL assay allows a broader range of labs. to measure pomalidomide for application to clin. pharmacokinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrTvTW3k7k5LVg90H21EOLACvtfcHk0liKPanAY3KlCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlSktb4%253D&md5=ab14a2918ff157d138b5191738ce3590</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.jpba.2014.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jpba.2014.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DShahbazi%26aufirst%3DS.%26aulast%3DPeer%26aufirst%3DC.%2BJ.%26aulast%3DPolizzotto%26aufirst%3DM.%2BN.%26aulast%3DUldrick%26aufirst%3DT.%2BS.%26aulast%3DRoth%26aufirst%3DJ.%26aulast%3DWyvill%26aufirst%3DK.%2BM.%26aulast%3DAleman%26aufirst%3DK.%26aulast%3DZeldis%26aufirst%3DJ.%2BB.%26aulast%3DYarchoan%26aufirst%3DR.%26aulast%3DFigg%26aufirst%3DW.%2BD.%26atitle%3DA%2520sensitive%2520and%2520robust%2520HPLC%2520assay%2520with%2520fluorescence%2520detection%2520for%2520the%2520quantification%2520of%2520pomalidomide%2520in%2520human%2520plasma%2520for%2520pharmacokinetic%2520analyses%26jtitle%3DJ.%2520Pharm.%2520Biomed.%2520Anal.%26date%3D2014%26volume%3D92%26spage%3D63%26epage%3D68%26doi%3D10.1016%2Fj.jpba.2014.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibitors of signal transducer and activator of transcription 3 (Stat3) protein</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6645</span>– <span class="NLM_lpage">6668</span>, <span class="refDoi"> DOI: 10.1021/jm300207s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300207s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnsl2msrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6645-6668&author=B.+Debnathauthor=S.+Xuauthor=N.+Neamati&title=Small+molecule+inhibitors+of+signal+transducer+and+activator+of+transcription+3+%28Stat3%29+protein&doi=10.1021%2Fjm300207s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors of Signal Transducer and Activator of Transcription 3 (Stat3) Protein</span></div><div class="casAuthors">Debnath, Bikash; Xu, Shili; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6645-6668</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Signal transducer and activator of transcription 3 (Stat3) is a transcription factor belonging to the STAT protein family, members of which regulate a wide variety of biol. processes, including cell growth, proliferation, survival, differentiation, development, movement and inflammation in response to extracellular signaling mols. such as cytokines and growth factors.  Receptor assocd. tyrosine kinases phosphorylate Stat3 protein at Tyr705 and enable it to dimerize and subsequently enter the nucleus where it activates the transcription of its target genes.  Accumulating evidence has established Stat3's role in oncogenesis, cardiovascular diseases, rheumatoid arthritis, fibrotic, and Alzheimer's diseases.  Thus, Stat3 inhibition is a promising therapeutic approach not only for cancer, but also for other diseases.  Although Stat3 has been considered as an important therapeutic target, progress in discovery and development of effective and specific Stat3 inhibitors remains slow.  In this article we summarize Stat3 inhibitors published between 2006 and 2011.  New strategies along with novel technologies are required to achieve breakthrough in the development of Stat3 inhibitors that can be translated into clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWWQl9gV1-ZLVg90H21EOLACvtfcHk0liAc4-ffsCAXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnsl2msrc%253D&md5=f5f3ac9f8fc2326c75edd2a9c136275b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm300207s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300207s%26sid%3Dliteratum%253Aachs%26aulast%3DDebnath%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DSmall%2520molecule%2520inhibitors%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%2520%2528Stat3%2529%2520protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6645%26epage%3D6668%26doi%3D10.1021%2Fjm300207s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bankhead, A.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Multi-omics profiling reveals key signaling pathways in ovarian cancer controlled by STAT3</span>. <i>Theranostics</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">5478</span>– <span class="NLM_lpage">5496</span>, <span class="refDoi"> DOI: 10.7150/thno.33444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.7150%2Fthno.33444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31534498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvVSgtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=5478-5496&author=T.+Luauthor=A.+Bankheadauthor=M.+Ljungmanauthor=N.+Neamati&title=Multi-omics+profiling+reveals+key+signaling+pathways+in+ovarian+cancer+controlled+by+STAT3&doi=10.7150%2Fthno.33444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-omics profiling reveals key signaling pathways in ovarian cancer controlled by STAT3</span></div><div class="casAuthors">Lu, Tiangong; Bankhead, Armand, III; Ljungman, Mats; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5478-5496</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">Inhibiting STAT3 signaling reduces tumor progression, metastasis and chemoresistance, however the precise mol. mechanism has not been fully delineated in ovarian cancer.  In this study, we generated STAT3 knockout (KO) ovarian cancer cell lines.  Effects of STAT3 KO on cell proliferation, migration and spheroid formation were assessed in vitro and effects on in vivo tumor growth were tested using several tumor xenograft models.  We used multi-omic genome-wide profiling to identify multi-level (Bru-Seq, RNA-Seq, and MS Proteomic) expression signatures of STAT3 KO ovarian cancer cells.  We obsd. that deletion of STAT3 blocked cell proliferation and migration in vitro and suppressed tumor growth in mice.  Deletion of STAT3 transcriptionally suppressed key genes involved in EMT, cell cycle progression, E2F signaling, and altered stemness markers.  Notably, KO of STAT3 resulted in modulation of the expression of other STAT family members.  Our study presents a rich, multi-faceted summary of the mol. mechanisms impacted by STAT3 deletion and provides new insight for STAT3's potential as a therapeutic target in ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTvEerWSP_tbVg90H21EOLACvtfcHk0liAc4-ffsCAXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvVSgtLs%253D&md5=0685567a4fff07940bc7dddcca2dc1df</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.7150%2Fthno.33444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.33444%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DT.%26aulast%3DBankhead%26aufirst%3DA.%26aulast%3DLjungman%26aufirst%3DM.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DMulti-omics%2520profiling%2520reveals%2520key%2520signaling%2520pathways%2520in%2520ovarian%2520cancer%2520controlled%2520by%2520STAT3%26jtitle%3DTheranostics%26date%3D2019%26volume%3D9%26spage%3D5478%26epage%3D5496%26doi%3D10.7150%2Fthno.33444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeman-Neill, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D. E.</span></span> <span> </span><span class="NLM_article-title">Bortezomib up-regulates activated signal transducer and activator of transcription-3 and synergizes with inhibitors of signal transducer and activator of transcription-3 to promote head and neck squamous cell carcinoma cell death</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2211</span>– <span class="NLM_lpage">2220</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-09-0327</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1158%2F1535-7163.MCT-09-0327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=19638453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFCnu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=2211-2220&author=C.+Liauthor=Y.+Zangauthor=M.+Senauthor=R.+J.+Leeman-Neillauthor=D.+S.+Manauthor=J.+R.+Grandisauthor=D.+E.+Johnson&title=Bortezomib+up-regulates+activated+signal+transducer+and+activator+of+transcription-3+and+synergizes+with+inhibitors+of+signal+transducer+and+activator+of+transcription-3+to+promote+head+and+neck+squamous+cell+carcinoma+cell+death&doi=10.1158%2F1535-7163.MCT-09-0327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Bortezomib up-regulates activated signal transducer and activator of transcription-3 and synergizes with inhibitors of signal transducer and activator of transcription-3 to promote head and neck squamous cell carcinoma cell death</span></div><div class="casAuthors">Li, Changyou; Zang, Yan; Sen, Malabika; Leeman-Neill, Rebecca J.; Man, David S. K.; Grandis, Jennifer R.; Johnson, Daniel E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2211-2220</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Head and neck squamous cell carcinomas (HNSCC) are commonly resistant to conventional chemotherapy drugs and exhibit overexpression of signal transducer and activator of transcription 3 (STAT3).  STAT3 promotes both the proliferation and survival of HNSCC cells.  Recent studies have shown that the proteasome inhibitor bortezomib shows cytotoxic activity against HNSCC in vitro and in vivo.  We report that treatment of HNSCC cells with bortezomib led to up-regulation of total STAT3 protein and the phosphorylated/activated form of STAT3, as well as an increase in cellular STAT3 activity.  This suggested that the ability of bortezomib to kill HNSCC cells may be blunted due to induction of STAT3, and inhibition of STAT3 may be a useful means for improving bortezomib efficacy.  Indeed, forced expression of dominant-active STAT3 inhibited bortezomib-induced cell death, whereas expression of dominant-neg. STAT3 served to enhance killing by this compd.  In addn., specific inhibition of STAT3 with the use of a STAT3 decoy oligonucleotide resulted in enhancement of bortezomib-induced apoptosis signaling and loss of clonogenic survival.  Cotreatment of HNSCC cells with bortezomib and guggulsterone, a naturally occurring compd. known to inhibit STAT3 activation, led to synergistic activation of cell death and loss of clonogenic survival.  In summary, these studies show that bortezomib induces the expression of active STAT3, a key growth- promoting protein in HNSCC cells.  Furthermore, our findings suggest that the therapeutic activity of bortezomib against HNSCC may be markedly improved by cotreatment with mol. targeting agents against STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAVkhXr3kma7Vg90H21EOLACvtfcHk0liAc4-ffsCAXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFCnu7s%253D&md5=b755af9decfbc6ab534da714d847ec33</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-09-0327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-09-0327%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZang%26aufirst%3DY.%26aulast%3DSen%26aufirst%3DM.%26aulast%3DLeeman-Neill%26aufirst%3DR.%2BJ.%26aulast%3DMan%26aufirst%3DD.%2BS.%26aulast%3DGrandis%26aufirst%3DJ.%2BR.%26aulast%3DJohnson%26aufirst%3DD.%2BE.%26atitle%3DBortezomib%2520up-regulates%2520activated%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription-3%2520and%2520synergizes%2520with%2520inhibitors%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription-3%2520to%2520promote%2520head%2520and%2520neck%2520squamous%2520cell%2520carcinoma%2520cell%2520death%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2009%26volume%3D8%26spage%3D2211%26epage%3D2220%26doi%3D10.1158%2F1535-7163.MCT-09-0327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A chemoproteomic approach to query the degradable kinome using a multi-kinase degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">99.e6</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.chembiol.2017.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=29129717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=88-99.e6&author=H.-T.+Huangauthor=D.+Dobrovolskyauthor=J.+Paulkauthor=G.+Yangauthor=E.+L.+Weisbergauthor=Z.+M.+Doctorauthor=D.+L.+Buckleyauthor=J.+H.+Choauthor=E.+Koauthor=J.+Jangauthor=K.+Shiauthor=H.+G.+Choiauthor=J.+D.+Griffinauthor=Y.+Liauthor=S.+P.+Treonauthor=E.+S.+Fischerauthor=J.+E.+Bradnerauthor=L.+Tanauthor=N.+S.+Gray&title=A+chemoproteomic+approach+to+query+the+degradable+kinome+using+a+multi-kinase+degrader&doi=10.1016%2Fj.chembiol.2017.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-kinase Degrader</span></div><div class="casAuthors">Huang, Hai-Tsang; Dobrovolsky, Dennis; Paulk, Joshiawa; Yang, Guang; Weisberg, Ellen L.; Doctor, Zainab M.; Buckley, Dennis L.; Cho, Joong-Heui; Ko, Eunhwa; Jang, Jaebong; Shi, Kun; Choi, Hwan Geun; Griffin, James D.; Li, Ying; Treon, Steven P.; Fischer, Eric S.; Bradner, James E.; Tan, Li; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-99.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heterobifunctional mols. that recruit E3 ubiquitin ligases, such as cereblon, for targeted protein degrdn. represent an emerging pharmacol. strategy.  A major unanswered question is how generally applicable this strategy is to all protein targets.  In this study, we designed a multi-kinase degrader by conjugating a highly promiscuous kinase inhibitor with a cereblon-binding ligand, and used quant. proteomics to discover 28 kinases, including BTK, PTK2, PTK2B, FLT3, AURKA, AURKB, TEC, ULK1, ITK, and nine members of the CDK family, as degradable.  This set of kinases is only a fraction of the intracellular targets bound by the degrader, demonstrating that successful degrdn. requires more than target engagement.  The results guided us to develop selective degraders for FLT3 and BTK, with potentials to improve disease treatment.  Together, this study demonstrates an efficient approach to triage a gene family of interest to identify readily degradable targets for further studies and pre-clin. developments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYNxhoPV3127Vg90H21EOLACvtfcHk0liy3K4oG9p1hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO&md5=703cfbaca3edd24176472158563f895f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.-T.%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520chemoproteomic%2520approach%2520to%2520query%2520the%2520degradable%2520kinome%2520using%2520a%2520multi-kinase%2520degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D88%26epage%3D99.e6%26doi%3D10.1016%2Fj.chembiol.2017.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horne, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jove, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaidehi, N.</span></span> <span> </span><span class="NLM_article-title">Molecular dynamics simulations of the conformational changes in signal transducers and activators of transcription, Stat1 and Stat3</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2009.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.jmgm.2009.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=19781967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlCmt7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=347-356&author=J.+Linauthor=R.+Buettnerauthor=Y.+C.+Yuanauthor=R.+Yipauthor=D.+Horneauthor=R.+Joveauthor=N.+Vaidehi&title=Molecular+dynamics+simulations+of+the+conformational+changes+in+signal+transducers+and+activators+of+transcription%2C+Stat1+and+Stat3&doi=10.1016%2Fj.jmgm.2009.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations of the conformational changes in signal transducers and activators of transcription, Stat1 and Stat3</span></div><div class="casAuthors">Lin, Jianping; Buettner, Ralf; Yuan, Yate-Ching; Yip, Richard; Horne, David; Jove, Richard; Vaidehi, Nagarajan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">347-356</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">All signal transducers and activators of transcription (STAT) factors are a family of cytoplasmic transcription factors that mediate the signal response to cytokines, growth factors, and hormonal factors.  The phosphorylation and subsequent activation of Stat3, a member of the STAT family, has been found to be elevated in a large no. of diverse human cancers.  An understanding of the dynamics of the Stat3 dimer interface is pertinent to designing small mol. inhibitors to activated Stat3 dimer.  To this end, the authors performed mol. dynamics simulations in explicit water of the activated Stat3 homodimer, and also its closely related member of the STAT family, activated Stat1 homodimer.  The authors obsd. a large-scale domain motion in the Stat3 dimer while the structure of the monomer remained intact.  The driving force for this conformational change was enhanced binding of the Stat3 dimer to DNA, thereby regulating gene expression.  The model showed that the C-terminus of one monomer wrapped around the core of the SH2 domain of the other monomer, and that this region that made up the dimer interface remained intact during the dynamics.  Water diffused into a cavity under this dimer interface, thus expanding a pre-existing cavity that gets gated and closed by the loops in the SH2 domain.  This cavity could serve as a potential binding pocket for inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1eGdndUM1ArVg90H21EOLACvtfcHk0liy3K4oG9p1hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlCmt7vL&md5=d32ef6835f4f199f6fc14d58e51d9881</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2009.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2009.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DYuan%26aufirst%3DY.%2BC.%26aulast%3DYip%26aufirst%3DR.%26aulast%3DHorne%26aufirst%3DD.%26aulast%3DJove%26aufirst%3DR.%26aulast%3DVaidehi%26aufirst%3DN.%26atitle%3DMolecular%2520dynamics%2520simulations%2520of%2520the%2520conformational%2520changes%2520in%2520signal%2520transducers%2520and%2520activators%2520of%2520transcription%252C%2520Stat1%2520and%2520Stat3%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2009%26volume%3D28%26spage%3D347%26epage%3D356%26doi%3D10.1016%2Fj.jmgm.2009.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parapatics, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ecker, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Translation termination factor GSPT1 Is a phenotypically relevant off-target of heterobifunctional phthalimide degraders</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00969</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00969" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=553-560&author=M.+Ishoeyauthor=S.+Chornauthor=N.+Singhauthor=M.+G.+Jaegerauthor=M.+Brandauthor=J.+Paulkauthor=S.+Bauerauthor=M.+A.+Erbauthor=K.+Parapaticsauthor=A.+C.+M%C3%BCllerauthor=K.+L.+Bennettauthor=G.+F.+Eckerauthor=J.+E.+Bradnerauthor=G.+E.+Winter&title=Translation+termination+factor+GSPT1+Is+a+phenotypically+relevant+off-target+of+heterobifunctional+phthalimide+degraders&doi=10.1021%2Facschembio.7b00969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Translation Termination Factor GSPT1 Is a Phenotypically Relevant Off-Target of Heterobifunctional Phthalimide Degraders</span></div><div class="casAuthors">Ishoey, Mette; Chorn, Someth; Singh, Natesh; Jaeger, Martin G.; Brand, Matthias; Paulk, Joshiawa; Bauer, Sophie; Erb, Michael A.; Parapatics, Katja; Muller, Andre C.; Bennett, Keiryn L.; Ecker, Gerhard F.; Bradner, James E.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">553-560</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein degrdn. is an emerging therapeutic strategy with a unique mol. pharmacol. that enables the disruption of all functions assocd. with a target.  This is particularly relevant for proteins depending on mol. scaffolding, such as transcription factors or receptor tyrosine kinases (RTKs).  To address tractability of multiple RTKs for chem. degrdn. by the E3 ligase CUL4-RBX1-DDB1-CRBN (CRL4CRBN), we synthesized a series of phthalimide degraders based on the promiscuous kinase inhibitors sunitinib and PHA665752.  While both series failed to induce degrdn. of their consensus targets, individual mols. displayed pronounced efficacy in leukemia cell lines.  Orthogonal target identification supported by mol. docking led us to identify the translation termination factor G1 to S phase transition 1 (GSPT1) as a converging off-target, resulting from inadvertent E3 ligase modulation.  This research highlights the importance of monitoring degrdn. events that are independent of the resp. targeting ligand as a unique feature of small-mol. degraders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_ZxnB5rKvvLVg90H21EOLACvtfcHk0liy3K4oG9p1hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D&md5=02a70959e8a8929a3c70846c1ed0200f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00969%26sid%3Dliteratum%253Aachs%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DChorn%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DJaeger%26aufirst%3DM.%2BG.%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DParapatics%26aufirst%3DK.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DTranslation%2520termination%2520factor%2520GSPT1%2520Is%2520a%2520phenotypically%2520relevant%2520off-target%2520of%2520heterobifunctional%2520phthalimide%2520degraders%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D553%26epage%3D560%26doi%3D10.1021%2Facschembio.7b00969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matyskiela, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagarigan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houston, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nosaka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaidarova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchelman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span> <span> </span><span class="NLM_article-title">A novel cereblon modulator recruits GSPT1 to the CRL4<sup>CRBN</sup> ubiquitin ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>535</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1038/nature18611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fnature18611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=27338790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=252-257&author=M.+E.+Matyskielaauthor=G.+Luauthor=T.+Itoauthor=B.+Pagariganauthor=C.-C.+Luauthor=K.+Millerauthor=W.+Fangauthor=N.-Y.+Wangauthor=D.+Nguyenauthor=J.+Houstonauthor=G.+Carmelauthor=T.+Tranauthor=M.+Rileyauthor=L.+Nosakaauthor=G.+C.+Landerauthor=S.+Gaidarovaauthor=S.+Xuauthor=A.+L.+Ruchelmanauthor=H.+Handaauthor=J.+Carmichaelauthor=T.+O.+Danielauthor=B.+E.+Cathersauthor=A.+Lopez-Gironaauthor=P.+P.+Chamberlain&title=A+novel+cereblon+modulator+recruits+GSPT1+to+the+CRL4CRBN+ubiquitin+ligase&doi=10.1038%2Fnature18611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A novel cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Matyskiela, Mary E.; Lu, Gang; Ito, Takumi; Pagarigan, Barbra; Lu, Chin-Chun; Miller, Karen; Fang, Wei; Wang, Nai-Yu; Nguyen, Derek; Houston, Jack; Carmel, Gilles; Tran, Tam; Riley, Mariko; Nosaka, Lyn'Al; Lander, Gabriel C.; Gaidarova, Svetlana; Xu, Shuichan; Ruchelman, Alexander L.; Handa, Hiroshi; Carmichael, James; Daniel, Thomas O.; Cathers, Brian E.; Lopez-Girona, Antonia; Chamberlain, Philip P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7611</span>),
    <span class="NLM_cas:pages">252-257</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Immunomodulatory drugs bind to cereblon (CRBN) to confer differentiated substrate specificity on the CRL4CRBN E3 ubiquitin ligase.  Here we report the identification of a new cereblon modulator, CC-885, with potent anti-tumor activity.  The anti-tumor activity of CC-885 is mediated through the cereblon-dependent ubiquitination and degrdn. of the translation termination factor GSPT1.  Patient-derived acute myeloid leukemia tumor cells exhibit high sensitivity to CC-885, indicating the clin. potential of this mechanism.  Crystallog. studies of the CRBN-DDB1-CC-885-GSPT1 complex reveal that GSPT1 binds to cereblon through a surface turn contg. a glycine residue at a key position, interacting with both CC-885 and a 'hotspot' on the cereblon surface.  Although GSPT1 possesses no obvious structural, sequence or functional homol. to previously known cereblon substrates, mutational anal. and modeling indicate that the cereblon substrate Ikaros uses a similar structural feature to bind cereblon, suggesting a common motif for substrate recruitment.  These findings define a structural degron underlying cereblon 'neosubstrate' selectivity, and identify an anti-tumor target rendered druggable by cereblon modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgAWiXdlJgYLVg90H21EOLACvtfcHk0lgYhILZa6Hnog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI&md5=1d4e340416aa0b08fc0f123faf638c3f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnature18611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18611%26sid%3Dliteratum%253Aachs%26aulast%3DMatyskiela%26aufirst%3DM.%2BE.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DPagarigan%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DC.-C.%26aulast%3DMiller%26aufirst%3DK.%26aulast%3DFang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DN.-Y.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DHouston%26aufirst%3DJ.%26aulast%3DCarmel%26aufirst%3DG.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DRiley%26aufirst%3DM.%26aulast%3DNosaka%26aufirst%3DL.%26aulast%3DLander%26aufirst%3DG.%2BC.%26aulast%3DGaidarova%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DRuchelman%26aufirst%3DA.%2BL.%26aulast%3DHanda%26aufirst%3DH.%26aulast%3DCarmichael%26aufirst%3DJ.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DCathers%26aufirst%3DB.%2BE.%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26atitle%3DA%2520novel%2520cereblon%2520modulator%2520recruits%2520GSPT1%2520to%2520the%2520CRL4CRBN%2520ubiquitin%2520ligase%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D252%26epage%3D257%26doi%3D10.1038%2Fnature18611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Simple structural modifications converting a bona fide MDM2 PROTAC degrader into a molecular glue molecule: A cautionary tale in the design of PROTAC degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9471</span>– <span class="NLM_lpage">9487</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00846</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00846" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVGgtb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9471-9487&author=J.+Yangauthor=Y.+Liauthor=A.+Aguilarauthor=Z.+Liuauthor=C.+Y.+Yangauthor=S.+Wang&title=Simple+structural+modifications+converting+a+bona+fide+MDM2+PROTAC+degrader+into+a+molecular+glue+molecule%3A+A+cautionary+tale+in+the+design+of+PROTAC+degraders&doi=10.1021%2Facs.jmedchem.9b00846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Simple Structural Modifications Converting a Bona fide MDM2 PROTAC Degrader into a Molecular Glue Molecule: A Cautionary Tale in the Design of PROTAC Degraders</span></div><div class="casAuthors">Yang, Jiuling; Li, Yangbing; Aguilar, Angelo; Liu, Zhaomin; Yang, Chao-Yie; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9471-9487</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inducing protein degrdn. by proteolysis targeting chimeras (PROTACs) has gained tremendous momentum for its promise to discover and develop new therapies.  Based upon our previously reported PROTAC MDM2 degraders, we have designed and synthesized addnl. analogs.  Surprisingly, we found that simple structural modifications of MD-222, a bona fide MDM2 PROTAC degrader, converts it into a "mol. glue", as exemplified by MG-277.  MG-277 induces only moderate MDM2 degrdn. and fails to activate wild-type p53 but is highly potent in inhibition of tumor cell growth in a p53-independent manner.  Our mechanistic investigation established that MG-277 is not a PROTAC MDM2 degrader but instead works as a mol. glue, inducing degrdn. of a translation termination factor, GSPT1 to achieve its potent anticancer activity.  Our study provides the first example that simple structural modifications can convert a bona fide PROTAC degrader into a mol. glue compd., which has a completely different mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH5W2xSSGs6rVg90H21EOLACvtfcHk0lgYhILZa6Hnog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVGgtb7K&md5=8a3e36a4fc3110c798d379b70d6203fc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00846%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DSimple%2520structural%2520modifications%2520converting%2520a%2520bona%2520fide%2520MDM2%2520PROTAC%2520degrader%2520into%2520a%2520molecular%2520glue%2520molecule%253A%2520A%2520cautionary%2520tale%2520in%2520the%2520design%2520of%2520PROTAC%2520degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9471%26epage%3D9487%26doi%3D10.1021%2Facs.jmedchem.9b00846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=941-964&author=X.+Hanauthor=C.+Wangauthor=C.+Qinauthor=W.+Xiangauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=M.+Wangauthor=L.+Zhaoauthor=T.+Xuauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ARD-69+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+androgen+receptor+%28AR%29+for+the+treatment+of+prostate+cancer&doi=10.1021%2Facs.jmedchem.8b01631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span></div><div class="casAuthors">Han, Xin; Wang, Chao; Qin, Chong; Xiang, Weiguo; Fernandez-Salas, Ester; Yang, Chao-Yie; Wang, Mi; Zhao, Lijie; Xu, Tianfeng; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">941-964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the discovery of highly potent PROTAC degraders of androgen receptor (AR), as exemplified by IARD-69.  I induces degrdn. of AR protein in AR-pos. prostate cancer cell lines in a dose- and time-dependent manner.  I achieves DC50 values of 0.86, 0.76, and 10.4 nM in LNCaP, VCaP, and 22Rv1 AR+ prostate cancer cell lines, resp.  I is capable of reducing the AR protein level by >95% in these prostate cancer cell lines and effectively suppressing AR-regulated gene expression.  I potently inhibits cell growth in these AR-pos. prostate cancer cell lines and is >100 times more potent than AR antagonists.  A single dose of I effectively reduces the level of AR protein in xenograft tumor tissue in mice.  Further optimization of I may ultimately lead to a new therapy for AR+, castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTlOD69tfhtbVg90H21EOLACvtfcHk0lgYhILZa6Hnog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D&md5=2376573fb1a1e19c517d64c70199700a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01631%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ARD-69%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520androgen%2520receptor%2520%2528AR%2529%2520for%2520the%2520treatment%2520of%2520prostate%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D941%26epage%3D964%26doi%3D10.1021%2Facs.jmedchem.8b01631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akuffo, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alontaga, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beatty, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hesterberg, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodheart, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunawan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karim, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orobello, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoder, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajadhyaksha, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epling-Burnette, P. K.</span></span> <span> </span><span class="NLM_article-title">Ligand-mediated protein degradation reveals functional conservation among sequence variants of the CUL4-type E3 ligase substrate receptor cereblon</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">6187</span>– <span class="NLM_lpage">6200</span>, <span class="refDoi"> DOI: 10.1074/jbc.M117.816868</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1074%2Fjbc.M117.816868" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=29449372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotFSqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=6187-6200&author=A.+A.+Akuffoauthor=A.+Y.+Alontagaauthor=R.+Metcalfauthor=M.+S.+Beattyauthor=A.+Beckerauthor=J.+M.+McDanielauthor=R.+S.+Hesterbergauthor=W.+E.+Goodheartauthor=S.+Gunawanauthor=M.+Ayazauthor=Y.+Yangauthor=M.+R.+Karimauthor=M.+E.+Orobelloauthor=K.+Danielauthor=W.+Guidaauthor=J.+A.+Yoderauthor=A.+M.+Rajadhyakshaauthor=E.+Schonbrunnauthor=H.+R.+Lawrenceauthor=N.+J.+Lawrenceauthor=P.+K.+Epling-Burnette&title=Ligand-mediated+protein+degradation+reveals+functional+conservation+among+sequence+variants+of+the+CUL4-type+E3+ligase+substrate+receptor+cereblon&doi=10.1074%2Fjbc.M117.816868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-mediated protein degradation reveals functional conservation among sequence variants of the CUL4-type E3 ligase substrate receptor cereblon</span></div><div class="casAuthors">Akuffo, Afua A.; Alontaga, Aileen Y.; Metcalf, Rainer; Beatty, Matthew S.; Becker, Andreas; McDaniel, Jessica M.; Hesterberg, Rebecca S.; Goodheart, William E.; Gunawan, Steven; Ayaz, Muhammad; Yang, Yan; Karim, Md. Rezaul; Orobello, Morgan E.; Daniel, Kenyon; Guida, Wayne; Yoder, Jeffrey A.; Rajadhyaksha, Anjali M.; Schonbrunn, Ernst; Lawrence, Harshani R.; Lawrence, Nicholas J.; Epling-Burnette, Pearlie K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6187-6200</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Upon binding to thalidomide and other immunomodulatory drugs, the E3 ligase substrate receptor cereblon (CRBN) promotes proteosomal destruction by engaging the DDB1-CUL4A-Roc1-RBX1 E3 ubiquitin ligase in human cells but not in mouse cells, suggesting that sequence variations in CRBN may cause its inactivation.  Therapeutically, CRBN engagers have the potential for broad applications in cancer and immune therapy by specifically reducing protein expression through targeted ubiquitin-mediated degrdn.  To examine the effects of defined sequence changes on CRBN's activity, we performed a comprehensive study using complementary theor., biophys., and biol. assays aimed at understanding CRBN's nonprimate sequence variations.  With a series of recombinant thalidomide-binding domain (TBD) proteins, we show that CRBN sequence variants retain their drug-binding properties to both classical immunomodulatory drugs and dBET1, a chem. compd. and targeting ligand designed to degrade bromodomain-contg. 4 (BRD4) via a CRBN-dependent mechanism.  We further show that dBET1 stimulates CRBN's E3 ubiquitin-conjugating function and degrades BRD4 in both mouse and human cells.  This insight paves the way for studies of CRBN-dependent proteasome-targeting mols. in nonprimate models and provides a new understanding of CRBN's substrate-recruiting function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIqHjqVpk5xrVg90H21EOLACvtfcHk0lhhB3wzP_8M6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotFSqtb8%253D&md5=16bc835472884d8b173e96fccdc3019a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M117.816868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M117.816868%26sid%3Dliteratum%253Aachs%26aulast%3DAkuffo%26aufirst%3DA.%2BA.%26aulast%3DAlontaga%26aufirst%3DA.%2BY.%26aulast%3DMetcalf%26aufirst%3DR.%26aulast%3DBeatty%26aufirst%3DM.%2BS.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DMcDaniel%26aufirst%3DJ.%2BM.%26aulast%3DHesterberg%26aufirst%3DR.%2BS.%26aulast%3DGoodheart%26aufirst%3DW.%2BE.%26aulast%3DGunawan%26aufirst%3DS.%26aulast%3DAyaz%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DKarim%26aufirst%3DM.%2BR.%26aulast%3DOrobello%26aufirst%3DM.%2BE.%26aulast%3DDaniel%26aufirst%3DK.%26aulast%3DGuida%26aufirst%3DW.%26aulast%3DYoder%26aufirst%3DJ.%2BA.%26aulast%3DRajadhyaksha%26aufirst%3DA.%2BM.%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DLawrence%26aufirst%3DH.%2BR.%26aulast%3DLawrence%26aufirst%3DN.%2BJ.%26aulast%3DEpling-Burnette%26aufirst%3DP.%2BK.%26atitle%3DLigand-mediated%2520protein%2520degradation%2520reveals%2520functional%2520conservation%2520among%2520sequence%2520variants%2520of%2520the%2520CUL4-type%2520E3%2520ligase%2520substrate%2520receptor%2520cereblon%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D6187%26epage%3D6200%26doi%3D10.1074%2Fjbc.M117.816868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87.e5</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87.e5&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lhhB3wzP_8M6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87.e5%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chau, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, P. P.</span></span> <span> </span><span class="NLM_article-title">Development of a STAT3 reporter prostate cancer cell line for high throughput screening of STAT3 activators and inhibitors</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>377</i></span>,  <span class="NLM_fpage">627</span>– <span class="NLM_lpage">631</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2008.10.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1016%2Fj.bbrc.2008.10.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=18930029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlykt7nE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=377&publication_year=2008&pages=627-631&author=M.+N.+Chauauthor=P.+P.+Banerjee&title=Development+of+a+STAT3+reporter+prostate+cancer+cell+line+for+high+throughput+screening+of+STAT3+activators+and+inhibitors&doi=10.1016%2Fj.bbrc.2008.10.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a STAT3 reporter prostate cancer cell line for high throughput screening of STAT3 activators and inhibitors</span></div><div class="casAuthors">Chau, My N.; Banerjee, Partha P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">377</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">627-631</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">STAT3 is constitutively activated in several cancers, including prostate cancer, and is therefore, a potential target for cancer therapy.  DU-145 prostate cancer cells were stably co-transfected with STAT3 reporter and puromycin resistant plasmids to create a stable STAT3 reporter cell line that can be used for high throughput screening of STAT3 modulators.  The applicability of this cell line was tested with two known activators and inhibitors of STAT3.  As expected, EGF and IL-6 increased STAT3 reporter activity and enhanced the nuclear localization of phosphorylated STAT3 (pSTAT3); whereas Cucurbitacin I and AG490 decreased STAT3 reporter activity dose and time-dependently and reduced the localization of pSTAT3 in the nuclei of prostate cancer cells.  Given the importance of STAT3 in cancer initiation and progression, the development of a stable STAT3 reporter cell line in prostate cancer cells provides a rapid, sensitive, and cost effective method for the screening of potential STAT3 modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjSY8FsvDnt7Vg90H21EOLACvtfcHk0liB0NK2LKu9YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlykt7nE&md5=dcdbd3c066f54ef2c3fd66a1de526229</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2008.10.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2008.10.025%26sid%3Dliteratum%253Aachs%26aulast%3DChau%26aufirst%3DM.%2BN.%26aulast%3DBanerjee%26aufirst%3DP.%2BP.%26atitle%3DDevelopment%2520of%2520a%2520STAT3%2520reporter%2520prostate%2520cancer%2520cell%2520line%2520for%2520high%2520throughput%2520screening%2520of%2520STAT3%2520activators%2520and%2520inhibitors%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2008%26volume%3D377%26spage%3D627%26epage%3D631%26doi%3D10.1016%2Fj.bbrc.2008.10.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponthier, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sack, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seebacher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">pSILAC mass spectrometry reveals ZFP91 as IMiD-dependent substrate of the CRL4<sup>CRBN</sup> ubiquitin ligase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">15398</span>, <span class="refDoi"> DOI: 10.1038/ncomms15398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fncomms15398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=28530236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2sXot1yqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=15398&author=J.+Anauthor=C.+M.+Ponthierauthor=R.+Sackauthor=J.+Seebacherauthor=M.+B.+Stadlerauthor=K.+A.+Donovanauthor=E.+S.+Fischer&title=pSILAC+mass+spectrometry+reveals+ZFP91+as+IMiD-dependent+substrate+of+the+CRL4CRBN+ubiquitin+ligase&doi=10.1038%2Fncomms15398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">pSILAC mass spectrometry reveals ZFP91 as IMiD-dependent substrate of the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">An, Jian; Ponthier, Charles M.; Sack, Ragna; Seebacher, Jan; Stadler, Michael B.; Donovan, Katherine A.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15398</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thalidomide and its derivs. lenalidomide and pomalidomide (IMiDs) are effective treatments of haematol. malignancies.  It was shown that IMiDs impart gain-of-function properties to the CUL4-RBX1-DDB1-CRBN (CRL4CRBN) ubiquitin ligase that enable binding, ubiquitination and degrdn. of key therapeutic targets such as IKZF1, IKZF3 and CSNK1A1.  While these substrates have been implicated as efficacy targets in multiple myeloma (MM) and 5q deletion assocd. myelodysplastic syndrome (del(5q)-MDS), other targets likely exist.  Using a pulse-chase SILAC mass spectrometry-based proteomics approach, we demonstrate that lenalidomide induces the ubiquitination and degrdn. of ZFP91.  We establish ZFP91 as a bona fide IMiD-dependent CRL4CRBN substrate and further show that ZFP91 harbours a zinc finger (ZnF) motif, related to the IKZF1/3 ZnF, crit. for IMiD-dependent CRBN binding.  These findings demonstrate that single time point pulse-chase SILAC mass spectrometry-based proteomics (pSILAC MS) is a sensitive approach for target identification of small mols. inducing selective protein degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA_4DdmHNeC7Vg90H21EOLACvtfcHk0liB0NK2LKu9YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXot1yqtbY%253D&md5=253c73df7a064a5f88f184ff782ef222</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fncomms15398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms15398%26sid%3Dliteratum%253Aachs%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DPonthier%26aufirst%3DC.%2BM.%26aulast%3DSack%26aufirst%3DR.%26aulast%3DSeebacher%26aufirst%3DJ.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DpSILAC%2520mass%2520spectrometry%2520reveals%2520ZFP91%2520as%2520IMiD-dependent%2520substrate%2520of%2520the%2520CRL4CRBN%2520ubiquitin%2520ligase%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D15398%26doi%3D10.1038%2Fncomms15398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structural basis of lenalidomide-induced CK1α degradation by the CRL4<sup>CRBN</sup> ubiquitin ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>532</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">130</span>, <span class="refDoi"> DOI: 10.1038/nature16979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fnature16979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=26909574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtFOis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=532&publication_year=2016&pages=127-130&author=G.+Petzoldauthor=E.+S.+Fischerauthor=N.+H.+Thom%C3%A4&title=Structural+basis+of+lenalidomide-induced+CK1%CE%B1+degradation+by+the+CRL4CRBN+ubiquitin+ligase&doi=10.1038%2Fnature16979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of lenalidomide-induced CK1α degradation by the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Petzold, Georg; Fischer, Eric S.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">532</span>
        (<span class="NLM_cas:issue">7597</span>),
    <span class="NLM_cas:pages">127-130</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thalidomide and its derivs., lenalidomide and pomalidomide, are immune modulatory drugs (IMiDs) used in the treatment of hematol. malignancies.  IMiDs bind CRBN, the substrate receptor of the CUL4-RBX1-DDB1-CRBN (also known as CRL4CRBN) E3 ubiquitin ligase, and inhibit ubiquitination of endogenous CRL4CRBN substrates.  Unexpectedly, IMiDs also repurpose the ligase to target new proteins for degrdn.  Lenalidomide induces degrdn. of the lymphoid transcription factors Ikaros and Aiolos (also known as IKZF1 and IKZF3), and casein kinase 1α (CK1α), which contributes to its clin. efficacy in the treatment of multiple myeloma and 5q-deletion assocd. myelodysplastic syndrome (del(5q) MDS), resp.  How lenalidomide alters the specificity of the ligase to degrade these proteins remains elusive.  Here we present the 2.45 Å crystal structure of DDB1-CRBN bound to lenalidomide and CK1α.  CRBN and lenalidomide jointly provide the binding interface for a CK1α β-hairpin-loop located in the kinase N-lobe.  We show that CK1α binding to CRL4CRBN is strictly dependent on the presence of an IMiD.  Binding of IKZF1 to CRBN similarly requires the compd. and both, IKZF1 and CK1α, use a related binding mode.  Our study provides a mechanistic explanation for the selective efficacy of lenalidomide in del(5q) MDS therapy.  We anticipate that high-affinity protein-protein interactions induced by small mols. will provide opportunities for drug development, particularly for targeted protein degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs4FxWriI1ObVg90H21EOLACvtfcHk0liB0NK2LKu9YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtFOis78%253D&md5=320fc6ff811da335fa8e02feab2d702d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnature16979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16979%26sid%3Dliteratum%253Aachs%26aulast%3DPetzold%26aufirst%3DG.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructural%2520basis%2520of%2520lenalidomide-induced%2520CK1%25CE%25B1%2520degradation%2520by%2520the%2520CRL4CRBN%2520ubiquitin%2520ligase%26jtitle%3DNature%26date%3D2016%26volume%3D532%26spage%3D127%26epage%3D130%26doi%3D10.1038%2Fnature16979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kronke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udeshi, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grauman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConkey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svinkina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heckl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciarlo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span> <span> </span><span class="NLM_article-title">Lenalidomide causes selective degradation of IKZF1 and IKZF3 in multiple myeloma cells</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">305</span>, <span class="refDoi"> DOI: 10.1126/science.1244851</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1126%2Fscience.1244851" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=24292625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BC2c3itFajsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2014&pages=301-305&author=J.+Kronkeauthor=N.+D.+Udeshiauthor=A.+Narlaauthor=P.+Graumanauthor=S.+N.+Hurstauthor=M.+McConkeyauthor=T.+Svinkinaauthor=D.+Hecklauthor=E.+Comerauthor=X.+Liauthor=C.+Ciarloauthor=E.+Hartmanauthor=N.+Munshiauthor=M.+Schenoneauthor=S.+L.+Schreiberauthor=S.+A.+Carrauthor=B.+L.+Ebert&title=Lenalidomide+causes+selective+degradation+of+IKZF1+and+IKZF3+in+multiple+myeloma+cells&doi=10.1126%2Fscience.1244851"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Lenalidomide causes selective degradation of IKZF1 and IKZF3 in multiple myeloma cells</span></div><div class="casAuthors">Kronke Jan; Udeshi Namrata D; Narla Anupama; Grauman Peter; Hurst Slater N; McConkey Marie; Svinkina Tanya; Heckl Dirk; Comer Eamon; Li Xiaoyu; Ciarlo Christie; Hartman Emily; Munshi Nikhil; Schenone Monica; Schreiber Stuart L; Carr Steven A; Ebert Benjamin L</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">6168</span>),
    <span class="NLM_cas:pages">301-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lenalidomide is a drug with clinical efficacy in multiple myeloma and other B cell neoplasms, but its mechanism of action is unknown.  Using quantitative proteomics, we found that lenalidomide causes selective ubiquitination and degradation of two lymphoid transcription factors, IKZF1 and IKZF3, by the CRBN-CRL4 ubiquitin ligase.  IKZF1 and IKZF3 are essential transcription factors in multiple myeloma.  A single amino acid substitution of IKZF3 conferred resistance to lenalidomide-induced degradation and rescued lenalidomide-induced inhibition of cell growth.  Similarly, we found that lenalidomide-induced interleukin-2 production in T cells is due to depletion of IKZF1 and IKZF3.  These findings reveal a previously unknown mechanism of action for a therapeutic agent: alteration of the activity of an E3 ubiquitin ligase, leading to selective degradation of specific targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQiiGR0Yx7nuYFEOhA_Im3cfW6udTcc2eZwu8n_tCvMp7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3itFajsQ%253D%253D&md5=8cae3e554399308493aa38f0d05ed610</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1126%2Fscience.1244851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1244851%26sid%3Dliteratum%253Aachs%26aulast%3DKronke%26aufirst%3DJ.%26aulast%3DUdeshi%26aufirst%3DN.%2BD.%26aulast%3DNarla%26aufirst%3DA.%26aulast%3DGrauman%26aufirst%3DP.%26aulast%3DHurst%26aufirst%3DS.%2BN.%26aulast%3DMcConkey%26aufirst%3DM.%26aulast%3DSvinkina%26aufirst%3DT.%26aulast%3DHeckl%26aufirst%3DD.%26aulast%3DComer%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DCiarlo%26aufirst%3DC.%26aulast%3DHartman%26aufirst%3DE.%26aulast%3DMunshi%26aufirst%3DN.%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26atitle%3DLenalidomide%2520causes%2520selective%2520degradation%2520of%2520IKZF1%2520and%2520IKZF3%2520in%2520multiple%2520myeloma%2520cells%26jtitle%3DScience%26date%3D2014%26volume%3D343%26spage%3D301%26epage%3D305%26doi%3D10.1126%2Fscience.1244851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Havens, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conklin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldman, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakurta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klippel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span> <span> </span><span class="NLM_article-title">Immunomodulatory agents lenalidomide and pomalidomide co-stimulate T cells by inducing degradation of T cell repressors Ikaros and Aiolos via modulation of the E3 ubiquitin ligase complex CRL4(CRBN.)</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>164</i></span>,  <span class="NLM_fpage">811</span>– <span class="NLM_lpage">821</span>, <span class="refDoi"> DOI: 10.1111/bjh.12708</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1111%2Fbjh.12708" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=24328678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjslWmu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=164&publication_year=2014&pages=811-821&author=A.+K.+Gandhiauthor=J.+Kangauthor=C.+G.+Havensauthor=T.+Conklinauthor=Y.+Ningauthor=L.+Wuauthor=T.+Itoauthor=H.+Andoauthor=M.+F.+Waldmanauthor=A.+Thakurtaauthor=A.+Klippelauthor=H.+Handaauthor=T.+O.+Danielauthor=P.+H.+Schaferauthor=R.+Chopra&title=Immunomodulatory+agents+lenalidomide+and+pomalidomide+co-stimulate+T+cells+by+inducing+degradation+of+T+cell+repressors+Ikaros+and+Aiolos+via+modulation+of+the+E3+ubiquitin+ligase+complex+CRL4%28CRBN.%29&doi=10.1111%2Fbjh.12708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Immunomodulatory agents lenalidomide and pomalidomide co-stimulate T cells by inducing degradation of T cell repressors Ikaros and Aiolos via modulation of the E3 ubiquitin ligase complex CRL4CRBN</span></div><div class="casAuthors">Gandhi, Anita K.; Kang, Jian; Havens, Courtney G.; Conklin, Thomas; Ning, Yuhong; Wu, Lei; Ito, Takumi; Ando, Hideki; Waldman, Michelle F.; Thakurta, Anjan; Klippel, Anke; Handa, Hiroshi; Daniel, Thomas O.; Schafer, Peter H.; Chopra, Rajesh</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">164</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">811-821</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Cereblon (CRBN), the mol. target of lenalidomide and pomalidomide, is a substrate receptor of the cullin ring E3 ubiquitin ligase complex, CRL4CRBN.  T cell co-stimulation by lenalidomide or pomalidomide is cereblon dependent: however, the CRL4CRBN substrates responsible for T cell co-stimulation have yet to be identified.  Here we demonstrate that interaction of the transcription factors Ikaros (IKZF1, encoded by the IKZF1 gene) and Aiolos (IKZF3, encoded by the IKZF3 gene) with CRL4CRBN is induced by lenalidomide or pomalidomide.  Each agent promotes Aiolos and Ikaros binding to CRL4CRBN with enhanced ubiquitination leading to cereblon-dependent proteosomal degrdn. in T lymphocytes.  We confirm that Aiolos and Ikaros are transcriptional repressors of interleukin-2 expression.  The findings link lenalidomide- or pomalidomide-induced degrdn. of these transcriptional suppressors to well documented T cell activation.  Importantly, Aiolos could serve as a proximal pharmacodynamic marker for lenalidomide and pomalidomide, as healthy human subjects administered lenalidomide demonstrated Aiolos degrdn. in their peripheral T cells.  In conclusion, we present a mol. model in which drug binding to cereblon results in the interaction of Ikaros and Aiolos to CRL4CRBN, leading to their ubiquitination, subsequent proteasomal degrdn. and T cell activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNm80s9sQNeLVg90H21EOLACvtfcHk0lhTmqwkWvayHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjslWmu74%253D&md5=8ddf00ec1e3c1b1176603f22e8832f58</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1111%2Fbjh.12708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbjh.12708%26sid%3Dliteratum%253Aachs%26aulast%3DGandhi%26aufirst%3DA.%2BK.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DHavens%26aufirst%3DC.%2BG.%26aulast%3DConklin%26aufirst%3DT.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DAndo%26aufirst%3DH.%26aulast%3DWaldman%26aufirst%3DM.%2BF.%26aulast%3DThakurta%26aufirst%3DA.%26aulast%3DKlippel%26aufirst%3DA.%26aulast%3DHanda%26aufirst%3DH.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DSchafer%26aufirst%3DP.%2BH.%26aulast%3DChopra%26aufirst%3DR.%26atitle%3DImmunomodulatory%2520agents%2520lenalidomide%2520and%2520pomalidomide%2520co-stimulate%2520T%2520cells%2520by%2520inducing%2520degradation%2520of%2520T%2520cell%2520repressors%2520Ikaros%2520and%2520Aiolos%2520via%2520modulation%2520of%2520the%2520E3%2520ubiquitin%2520ligase%2520complex%2520CRL4%2528CRBN.%2529%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2014%26volume%3D164%26spage%3D811%26epage%3D821%26doi%3D10.1111%2Fbjh.12708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matyskiela, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baughman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clayton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slade, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBrun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. W.</span></span> <span> </span><span class="NLM_article-title">Cereblon Modulators Target ZBTB16 and Its Oncogenic Fusion Partners for Degradation via Distinct Structural Degrons</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">3149</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00674</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00674" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlGqsL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=3149&author=M.+E.+Matyskielaauthor=J.+Zhuauthor=J.+M.+Baughmanauthor=T.+Claytonauthor=M.+Sladeauthor=H.+K.+Wongauthor=K.+Dangaauthor=X.+Zhengauthor=M.+Labowauthor=L.+LeBrunauthor=G.+Luauthor=P.+P.+Chamberlainauthor=J.+W.+Thompson&title=Cereblon+Modulators+Target+ZBTB16+and+Its+Oncogenic+Fusion+Partners+for+Degradation+via+Distinct+Structural+Degrons&doi=10.1021%2Facschembio.0c00674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Cereblon Modulators Target ZBTB16 and Its Oncogenic Fusion Partners for Degradation via Distinct Structural Degrons</span></div><div class="casAuthors">Matyskiela, Mary E.; Zhu, Jinyi; Baughman, Joshua M.; Clayton, Thomas; Slade, Michelle; Wong, Hon Kit; Danga, Kristina; Zheng, Xinde; Labow, Mark; LeBrun, Laurie; Lu, Gang; Chamberlain, Philip P.; Thompson, Joel W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3149-3158</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is a growing interest in using targeted protein degrdn. as a therapeutic modality in view of its potential to expand the druggable proteome.  One avenue to using this modality is via mol. glue based Cereblon E3 Ligase Modulating Drug compds.  Here, we report the identification of the transcription factor ZBTB16 as a Cereblon neosubstrate.  We also report two new Cereblon modulators, CC-3060 and CC-647, that promote ZBTB16 degrdn.  Unexpectedly, CC-3060 and CC-647 target ZBTB16 for degrdn. by primarily engaging distinct structural degrons on different zinc finger domains.  The reciprocal fusion proteins, ZBTB16-RARα and RARα-ZBTB16, which cause a rare acute promyelocytic leukemia, contain these same structural degrons and can be targeted for proteasomal degrdn. with Cereblon modulator treatment.  Thus, a targeted protein degrdn. approach via Cereblon modulators may represent a novel therapeutic strategy in acute promyelocytic leukemia where ZBTB16/RARA rearrangements are crit. disease drivers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBTWDqkgT1g7Vg90H21EOLACvtfcHk0lhTmqwkWvayHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlGqsL3K&md5=6c13b47b19522cce5466f5a100b68f57</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00674%26sid%3Dliteratum%253Aachs%26aulast%3DMatyskiela%26aufirst%3DM.%2BE.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DBaughman%26aufirst%3DJ.%2BM.%26aulast%3DClayton%26aufirst%3DT.%26aulast%3DSlade%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DH.%2BK.%26aulast%3DDanga%26aufirst%3DK.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLabow%26aufirst%3DM.%26aulast%3DLeBrun%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26aulast%3DThompson%26aufirst%3DJ.%2BW.%26atitle%3DCereblon%2520Modulators%2520Target%2520ZBTB16%2520and%2520Its%2520Oncogenic%2520Fusion%2520Partners%2520for%2520Degradation%2520via%2520Distinct%2520Structural%2520Degrons%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26spage%3D3149%26doi%3D10.1021%2Facschembio.0c00674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. J.</span></span> <span> </span><span class="NLM_article-title">An atypical E3 ligase zinc finger protein 91 stabilizes and activates NF-κB-inducing kinase via Lys63-linked ubiquitination</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">30539</span>– <span class="NLM_lpage">30547</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.129551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1074%2Fjbc.M110.129551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=20682767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1SlsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=30539-30547&author=X.+Jinauthor=H.+R.+Jinauthor=H.+S.+Jungauthor=S.+J.+Leeauthor=J.+H.+Leeauthor=J.+J.+Lee&title=An+atypical+E3+ligase+zinc+finger+protein+91+stabilizes+and+activates+NF-%CE%BAB-inducing+kinase+via+Lys63-linked+ubiquitination&doi=10.1074%2Fjbc.M110.129551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">An Atypical E3 Ligase Zinc Finger Protein 91 Stabilizes and Activates NF-κB-inducing Kinase via Lys63-linked Ubiquitination</span></div><div class="casAuthors">Jin, Xuejun; Jin, Hong Ri; Jung, Haeng Sun; Lee, Se Jeong; Lee, Jeong-Hyung; Lee, Jung Joon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">30539-30547</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The NF-κB transcription factors control many physiol. processes, including inflammation, immunity, and apoptosis.  Its activity contributes to the development of various cell malignancies.  NF-κB-inducing kinase (NIK) plays a pivotal role in NF-κB activation.  However, the mol. mechanism to stabilize and activate NIK remains elusive, although it is known that cIAP1/2 (cellular inhibitor of apoptosis 1 and 2) ubiquitinate NIK for degrdn.  Here, the authors report a novel NF-κB-related zinc finger protein 91 (ZFP91) that stabilizes and activates NIK in a ubiquitination-dependent manner.  The authors show that ZFP91 interacts with and promotes the Lys63-linked ubiquitination of NIK and subsequent processing of p100 to p52.  The results of in vitro biochem. assays indicate that ZFP91 functions as an E3 ligase directly to NIK.  Remarkably, the ubiquitination of NIK coincides with its Thr559 phosphorylation.  Furthermore, knockdown of ZFP91 expression by RNA interference inhibits the CD40 ligation-induced activation of NIK and p100 processing as well as the expression of noncanonical NF-κB target genes.  These data clearly indicate that ZFP91 is an important regulator of the noncanonical NF-κB pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmSf7n4uBsTbVg90H21EOLACvtfcHk0liOZGBjptZlmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1SlsrrN&md5=a07919aab27d42b626a349faadf0be34</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.129551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.129551%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DH.%2BR.%26aulast%3DJung%26aufirst%3DH.%2BS.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BJ.%26atitle%3DAn%2520atypical%2520E3%2520ligase%2520zinc%2520finger%2520protein%252091%2520stabilizes%2520and%2520activates%2520NF-%25CE%25BAB-inducing%2520kinase%2520via%2520Lys63-linked%2520ubiquitination%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D30539%26epage%3D30547%26doi%3D10.1074%2Fjbc.M110.129551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X.</span></span> <span> </span><span class="NLM_article-title">Zinc finger protein 91 (ZFP91) activates HIF-1α via NF-κB/p65 to promote proliferation and tumorigenesis of colon cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">36551</span>– <span class="NLM_lpage">36562</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.9070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.18632%2Foncotarget.9070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=27144516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BC28blslOltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=36551-36562&author=J.+Maauthor=C.+Miauthor=K.+S.+Wangauthor=J.+J.+Leeauthor=X.+Jin&title=Zinc+finger+protein+91+%28ZFP91%29+activates+HIF-1%CE%B1+via+NF-%CE%BAB%2Fp65+to+promote+proliferation+and+tumorigenesis+of+colon+cancer&doi=10.18632%2Foncotarget.9070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Zinc finger protein 91 (ZFP91) activates HIF-1α via NF-κB/p65 to promote proliferation and tumorigenesis of colon cancer</span></div><div class="casAuthors">Ma Juan; Mi Chunliu; Wang Ke Si; Lee Jung Joon; Jin Xuejun</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">36551-36562</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Zinc finger protein 91 (ZFP91) has been reported to be involved in various biological processes.  However, the clinical significance and biological role of ZFP91 in colon cancer remains unknown.  Here, we show that ZFP91 expression is upregulated in patients with colon cancer.  We found that ZFP91 upregulated HIF-1α at the levels of promoter and protein in colon cancer cells.  Using chromatin immunoprecipitation, electrophoretic mobility shift assay and luciferase reporter gene assay, we found that NF-κB/p65 is required for the binding of ZFP91 to the HIF-1α promoter at -197/-188 base pairs and for the transcriptional activation of HIF-1α gene mediated by ZFP91.  Flow cytometry, 5-ethynyl-2'-deoxyuridine (EdU) incorporation and tumor xenograft assay demonstrated that ZFP91 enhanced cell proliferation of colon cancer through upregulating HIF-1α in vitro and in vivo.  Furthermore, ZFP91 is positively associated with HIF-1α in human colon cancer.  Thus, we concluded that ZFP91 activates transcriptional coregulatory protein HIF-1α through transcription factor NF-κB/p65 in the promotion of proliferation and tumorigenesis in colon cancer cell.  ZFP91 may serve as a driver gene to activate HIF-1α transcription in the development of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTvqyrzGHsbP1WTCLT8lX9SfW6udTcc2ebo_VnY6k9aZ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28blslOltw%253D%253D&md5=319c7bf77a2d38d41b27284b1538a7de</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.9070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.9070%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DMi%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DK.%2BS.%26aulast%3DLee%26aufirst%3DJ.%2BJ.%26aulast%3DJin%26aufirst%3DX.%26atitle%3DZinc%2520finger%2520protein%252091%2520%2528ZFP91%2529%2520activates%2520HIF-1%25CE%25B1%2520via%2520NF-%25CE%25BAB%252Fp65%2520to%2520promote%2520proliferation%2520and%2520tumorigenesis%2520of%2520colon%2520cancer%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D36551%26epage%3D36562%26doi%3D10.18632%2Foncotarget.9070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, D.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.-H.</span></span> <span> </span><span class="NLM_article-title">The ubiquitinase ZFP91 promotes tumor cell survival and confers chemoresistance through FOXA1 destabilization</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1093/carcin/bgz085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1093%2Fcarcin%2Fbgz085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31046116" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWntr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2020&pages=56-66&author=D.-E.+Tangauthor=Y.+Daiauthor=Y.+Xuauthor=L.-W.+Linauthor=D.-Z.+Liuauthor=X.-P.+Hongauthor=M.-L.+Ouauthor=H.+W.+Jiangauthor=S.-H.+Xu&title=The+ubiquitinase+ZFP91+promotes+tumor+cell+survival+and+confers+chemoresistance+through+FOXA1+destabilization&doi=10.1093%2Fcarcin%2Fbgz085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitinase ZFP91 promotes tumor cell survival and confers chemoresistance through FOXA1 destabilization</span></div><div class="casAuthors">Tang, Dong-E.; Dai, Yong; Xu, Yong; Lin, Lie-Wen; Liu, Dong-Zhou; Hong, Xiao-Ping; Ou, Ming-Lin; Jiang, Hao-Wu; Xu, Song-Hui</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-66</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">1460-2180</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The forkhead box A1 (FOXA1), one of the forkhead class of DNA-binding proteins, functions as a transcription factor and plays a vital role in cellular control of embryonic development and cancer progression.  Downregulation of FOXA1 has reported in several types of cancer, which contributes to cancer cell survival and chemoresistance.  However, the mechanism for FOXA1 downregulation in cancer remains unclear.  Here, we report that the ubiquitination enzyme zinc figer protein 91 (ZFP91) ubiquitinates and destabilizes FOXA1, which promotes cancer cell growth.  High level of ZFP91 expression correlates with low level of FOXA1 protein in human gastric cancer (GC) cell lines and patient samples.  Furthermore, ZFP91 knockdown reduces FOXA1 polyubiquitination, which decreases FOXA1 turnover and enhances cellular sensitivity to chemotherapy.  Taken together, our fidings reveal ZFP91-FOXA1 axis plays an important role in promoting GC progression and provides us a potential therapeutic intervention in the treatment of GC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm01VlO3OLPbVg90H21EOLACvtfcHk0liOZGBjptZlmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWntr7J&md5=32142868ef12f3f7c0301564d7b1aad2</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgz085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgz085%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DD.-E.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DL.-W.%26aulast%3DLiu%26aufirst%3DD.-Z.%26aulast%3DHong%26aufirst%3DX.-P.%26aulast%3DOu%26aufirst%3DM.-L.%26aulast%3DJiang%26aufirst%3DH.%2BW.%26aulast%3DXu%26aufirst%3DS.-H.%26atitle%3DThe%2520ubiquitinase%2520ZFP91%2520promotes%2520tumor%2520cell%2520survival%2520and%2520confers%2520chemoresistance%2520through%2520FOXA1%2520destabilization%26jtitle%3DCarcinogenesis%26date%3D2020%26volume%3D41%26spage%3D56%26epage%3D66%26doi%3D10.1093%2Fcarcin%2Fbgz085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span> <span> </span><span class="NLM_article-title">E2F is required for STAT3-mediated upregulation of cyclin B1 and Cdc2 expressions and contributes to G2-M phase transition</span>. <i>Acta. Biochim. Biophys. Sin.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">322</span>, <span class="refDoi"> DOI: 10.1093/abbs/gmy174</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1093%2Fabbs%2Fgmy174" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=30726872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVCmu7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=313-322&author=J.+Sunauthor=Y.+Duauthor=Q.+Songauthor=J.+Nanauthor=P.+Guanauthor=J.+Guoauthor=X.+Wangauthor=J.+Yangauthor=C.+Zhao&title=E2F+is+required+for+STAT3-mediated+upregulation+of+cyclin+B1+and+Cdc2+expressions+and+contributes+to+G2-M+phase+transition&doi=10.1093%2Fabbs%2Fgmy174"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">E2F is required for STAT3-mediated upregulation of cyclin B1 and Cdc2 expressions and contributes to G2-M phase transition</span></div><div class="casAuthors">Sun, Jingjie; Du, Yuping; Song, Qiaoling; Nan, Jing; Guan, Peizhu; Guo, Jihui; Wang, Xiao; Yang, Jinbo; Zhao, Chenyang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Biochimica et Biophysica Sinica</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">313-322</span>CODEN:
                <span class="NLM_cas:coden">ABBSC2</span>;
        ISSN:<span class="NLM_cas:issn">1745-7270</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Activation of transcription factor STAT3 is involved in cell proliferation, differentiation, and cell survival.  Constitutive activation of STAT3 pathway has been assocd. with the oncogenesis of various types of cancers.  It has been reported that STAT3 plays a key role in the G1 to S phase cell cycle transition induced by the cytokine receptor subunit gp130, through the upregulation of cyclins D1, D2, D3, A, and Cdc25A and the concomitant downregulation of p21 and p27.  However, its role in mediating G2-M phase transition has not been studied.  The cyclin B1/Cdc2 complex is widely accepted as the trigger of mitosis in all organisms and is believed to be necessary for progression through S phase and keep active during the G2-M transition and progression.  In the present study, we found that activation of STAT3 stimulates cyclin B1 and Cdc2 expressions.  Deletion and site-directed mutations on cyclin B1 and Cdc2 promoters indicated that E2F element mediates the upregulation of these two promoters in a STAT3-dependent manner.  The findings reported here demonstrated that STAT3 participates in modulating G2-M phase checkpoint by regulating gene expressions of cyclin B1 and Cdc2 via E2F.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWSSYKmdNxOrVg90H21EOLACvtfcHk0lju3oXicV8PXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVCmu7fJ&md5=a5aa475fbed3f2122538f7d0225ba1ac</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fabbs%2Fgmy174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fabbs%252Fgmy174%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DQ.%26aulast%3DNan%26aufirst%3DJ.%26aulast%3DGuan%26aufirst%3DP.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DC.%26atitle%3DE2F%2520is%2520required%2520for%2520STAT3-mediated%2520upregulation%2520of%2520cyclin%2520B1%2520and%2520Cdc2%2520expressions%2520and%2520contributes%2520to%2520G2-M%2520phase%2520transition%26jtitle%3DActa.%2520Biochim.%2520Biophys.%2520Sin.%26date%3D2019%26volume%3D51%26spage%3D313%26epage%3D322%26doi%3D10.1093%2Fabbs%2Fgmy174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koganti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui-Yuen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grasser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palendira, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tangye, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhaduri-McIntosh, S.</span></span> <span> </span><span class="NLM_article-title">STAT3 interrupts ATR-Chk1 signaling to allow oncovirus-mediated cell proliferation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">4946</span>– <span class="NLM_lpage">4951</span>, <span class="refDoi"> DOI: 10.1073/pnas.1400683111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1073%2Fpnas.1400683111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=24639502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksVSjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=4946-4951&author=S.+Kogantiauthor=J.+Hui-Yuenauthor=S.+McAllisterauthor=B.+Gardnerauthor=F.+Grasserauthor=U.+Palendiraauthor=S.+G.+Tangyeauthor=A.+F.+Freemanauthor=S.+Bhaduri-McIntosh&title=STAT3+interrupts+ATR-Chk1+signaling+to+allow+oncovirus-mediated+cell+proliferation&doi=10.1073%2Fpnas.1400683111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">STAT3 interrupts ATR-Chk1 signaling to allow oncovirus-mediated cell proliferation</span></div><div class="casAuthors">Koganti, Siva; Hui-Yuen, Joyce; McAllister, Shane; Gardner, Benjamin; Grasser, Friedrich; Palendira, Umaimainthan; Tangye, Stuart G.; Freeman, Alexandra F.; Bhaduri-McIntosh, Sumita</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4946-4951</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">DNA damage response (DDR) is a signaling network that senses DNA damage and activates response pathways to coordinate cell-cycle progression and DNA repair.  Thus, DDR is crit. for maintenance of genome stability, and presents a powerful defense against tumorigenesis.  Therefore, to drive cell-proliferation and transformation, viral and cellular oncogenes need to circumvent DDR-induced cell-cycle checkpoints.  Unlike in hereditary cancers, mechanisms that attenuate DDR and disrupt cell-cycle checkpoints in sporadic cancers are not well understood.  Using Epstein-Barr virus (EBV) as a source of oncogenes, we have previously shown that EBV-driven cell proliferation requires the cellular transcription factor STAT3.  EBV infection is rapidly followed by activation and increased expression of STAT3, which mediates relaxation of the intra-S phase cell-cycle checkpoint; this facilitates viral oncogene-driven cell proliferation.  We now show that replication stress-assocd. DNA damage, which results from EBV infection, is detected by DDR.  However, signaling downstream of ATR is impaired by STAT3, leading to relaxation of the intra-S phase checkpoint.  We find that STAT3 interrupts ATR-to-Chk1 signaling by promoting loss of Claspin, a protein that assists ATR to phosphorylate Chk1.  This loss of Claspin which ultimately facilitates cell proliferation is mediated by caspase 7, a protein that typically promotes cell death.  Our findings demonstrate how STAT3, which is constitutively active in many human cancers, suppresses DDR, fundamental to tumorigenesis.  This newly recognized role for STAT3 in attenuation of DDR, discovered in the context of EBV infection, is of broad interest as the biol. of cell proliferation is central to both health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzwtYHixWYErVg90H21EOLACvtfcHk0lju3oXicV8PXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksVSjs7k%253D&md5=6f8ba71080289ac8ab8c04ae36fb6686</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1400683111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1400683111%26sid%3Dliteratum%253Aachs%26aulast%3DKoganti%26aufirst%3DS.%26aulast%3DHui-Yuen%26aufirst%3DJ.%26aulast%3DMcAllister%26aufirst%3DS.%26aulast%3DGardner%26aufirst%3DB.%26aulast%3DGrasser%26aufirst%3DF.%26aulast%3DPalendira%26aufirst%3DU.%26aulast%3DTangye%26aufirst%3DS.%2BG.%26aulast%3DFreeman%26aufirst%3DA.%2BF.%26aulast%3DBhaduri-McIntosh%26aufirst%3DS.%26atitle%3DSTAT3%2520interrupts%2520ATR-Chk1%2520signaling%2520to%2520allow%2520oncovirus-mediated%2520cell%2520proliferation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D4946%26epage%3D4951%26doi%3D10.1073%2Fpnas.1400683111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-K.</span></span> <span> </span><span class="NLM_article-title">STAT3 negatively regulates antiviral responses through suppression of TLR and type I IFN-mediated responses</span>. <i>Cytokine+</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">126</span>– <span class="NLM_lpage">126</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=126-126&author=W.-B.+Wangauthor=H.+K.+Dengauthor=D.+E.+Levyauthor=C.-K.+Lee&title=STAT3+negatively+regulates+antiviral+responses+through+suppression+of+TLR+and+type+I+IFN-mediated+responses"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.-B.%26aulast%3DDeng%26aufirst%3DH.%2BK.%26aulast%3DLevy%26aufirst%3DD.%2BE.%26aulast%3DLee%26aufirst%3DC.-K.%26atitle%3DSTAT3%2520negatively%2520regulates%2520antiviral%2520responses%2520through%2520suppression%2520of%2520TLR%2520and%2520type%2520I%2520IFN-mediated%2520responses%26jtitle%3DCytokine%252B%26date%3D2009%26volume%3D48%26spage%3D126%26epage%3D126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G.-R.</span></span> <span> </span><span class="NLM_article-title">ECD promotes gastric cancer metastasis by blocking E3 ligase ZFP91-mediated hnRNP F ubiquitination and degradation</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">479</span>, <span class="refDoi"> DOI: 10.1038/s41419-018-0525-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1038%2Fs41419-018-0525-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=29706618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A280%3ADC%252BC1MjnvVWmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=479&author=S.-H.+Xuauthor=S.+Zhuauthor=Y.+Wangauthor=J.-Z.+Huangauthor=M.+Chenauthor=Q.-X.+Wuauthor=Y.-T.+Heauthor=D.+Chenauthor=G.-R.+Yan&title=ECD+promotes+gastric+cancer+metastasis+by+blocking+E3+ligase+ZFP91-mediated+hnRNP+F+ubiquitination+and+degradation&doi=10.1038%2Fs41419-018-0525-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">ECD promotes gastric cancer metastasis by blocking E3 ligase ZFP91-mediated hnRNP F ubiquitination and degradation</span></div><div class="casAuthors">Xu Song-Hui; Zhu Song; Wang Yanjie; Huang Jin-Zhou; Chen Min; Wu Qing-Xia; He Yu-Tian; Chen De; Yan Guang-Rong; Xu Song-Hui; Zhu Song; Chen Min; Chen De; Yan Guang-Rong</div><div class="citationInfo"><span class="NLM_cas:title">Cell death & disease</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">479</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The human ortholog of the Drosophila ecdysoneless gene (ECD) is required for embryonic development and cell-cycle progression; however, its role in cancer progression and metastasis remains unclear.  Here, we found that ECD is frequently overexpressed in gastric cancer (GC), especially in metastatic GC, and is correlated with poor clinical outcomes in GC patients.  Silencing ECD inhibited GC migration and invasion in vitro and metastasis in vivo, while ECD overexpression promoted GC migration and invasion.  ECD promoted GC invasion and metastasis by protecting hnRNP F from ubiquitination and degradation.  We identified ZFP91 as the E3 ubiquitin ligase that is responsible for hnRNP F ubiquitination at Lys 185 and proteasomal degradation.  ECD competitively bound to hnRNP F via the N-terminal STG1 domain (13-383aa), preventing hnRNP F from interacting with ZFP91, thus preventing ZFP91-mediated hnRNP F ubiquitination and proteasomal degradation.  Collectively, our findings indicate that ECD promotes cancer invasion and metastasis by preventing E3 ligase ZFP91-mediated hnRNP F ubiquitination and degradation, suggesting that ECD may be a marker for poor prognosis and a potential therapeutic target for GC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT9Q4NhHtDg2ysZ1x6zOs0vfW6udTcc2ebsfvJpLIHFArntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjnvVWmtQ%253D%253D&md5=ea4b31d786a3eb7236234ce4aaf633bb</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fs41419-018-0525-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41419-018-0525-x%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.-H.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DJ.-Z.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DQ.-X.%26aulast%3DHe%26aufirst%3DY.-T.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DYan%26aufirst%3DG.-R.%26atitle%3DECD%2520promotes%2520gastric%2520cancer%2520metastasis%2520by%2520blocking%2520E3%2520ligase%2520ZFP91-mediated%2520hnRNP%2520F%2520ubiquitination%2520and%2520degradation%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2018%26volume%3D9%26spage%3D479%26doi%3D10.1038%2Fs41419-018-0525-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Inhibitory effect of RNA-mediated knockdown of zinc finger protein 91 pseudogene on pancreatic cancer cell growth and invasion</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1343</span>– <span class="NLM_lpage">1348</span>, <span class="refDoi"> DOI: 10.3892/ol.2016.4794</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.3892%2Fol.2016.4794" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=27446435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsFynurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1343-1348&author=W.+Huangauthor=N.+Liauthor=J.+Huauthor=L.+Wang&title=Inhibitory+effect+of+RNA-mediated+knockdown+of+zinc+finger+protein+91+pseudogene+on+pancreatic+cancer+cell+growth+and+invasion&doi=10.3892%2Fol.2016.4794"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitory effect of RNA-mediated knockdown of zinc finger protein 91 pseudogene on pancreatic cancer cell growth and invasion</span></div><div class="casAuthors">Huang, Weiyi; Li, Ning; Hu, Jiong; Wang, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1343-1348</span>CODEN:
                <span class="NLM_cas:coden">OLNEB5</span>;
        ISSN:<span class="NLM_cas:issn">1792-1082</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Worldwide, human pancreatic cancer is a rare malignancy with a poor prognosis.  Long non-coding RNAs (lncRNAs) are known to have a crucial role in cancer occurrence and progression; however, the role of pseudogene-expressed lncRNAs, a major type of lncRNA, have not been thoroughly analyzed in cancer.  Therefore, the present study focused on zinc finger protein 91 pseudogene (ZFP91-P).  ZFP91-P expression was initially detected in two pancreatic cancer cell lines by reverse transcription-quant. polymerase chain reaction (RT-qPCR) and the highest expression of ZFP91-P was found in the BXPC-3-H cell line.  Subsequently, BXPC-3-H cells were transfected with ZFP91-P short hairpin RNA (shRNA) using a plasmid vector and termed shZFP91-P.  Cells transfected with neg. control plasmid vector were termed shCon.  MTT and Transwell assays were performed to analyze the proliferation and migration of BXPC-3-H cells, resp., and western blotting was used to detect epithelial-mesenchymal transition markers, including vimentin and β-catenin.  The present study showed that depletion of ZFP91-P markedly decreased pancreatic cancer cell proliferation and inhibited cell migration capacity.  In addn., the expression of β-catenin increased while vimentin expression decreased.  The current findings suggest that high expression of ZFP91-P promotes the migration of BXPC-3-H cells and may be a novel marker for early diagnosis for pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnj3hLFXvTGbVg90H21EOLACvtfcHk0lgOLp7ewGs_-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsFynurk%253D&md5=3b4e0004a453cf004b91375b3aac1e0e</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.3892%2Fol.2016.4794&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2016.4794%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DInhibitory%2520effect%2520of%2520RNA-mediated%2520knockdown%2520of%2520zinc%2520finger%2520protein%252091%2520pseudogene%2520on%2520pancreatic%2520cancer%2520cell%2520growth%2520and%2520invasion%26jtitle%3DOncol.%2520Lett.%26date%3D2016%26volume%3D12%26spage%3D1343%26epage%3D1348%26doi%3D10.3892%2Fol.2016.4794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span> <span> </span><span class="NLM_article-title">GEPIA2: an enhanced web server for large-scale expression profiling and interactive analysis</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">W556</span>– <span class="NLM_lpage">W560</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=10.1093%2Fnar%2Fgkz430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=31114875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVCmur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W556-W560&author=Z.+Tangauthor=B.+Kangauthor=C.+Liauthor=T.+Chenauthor=Z.+Zhang&title=GEPIA2%3A+an+enhanced+web+server+for+large-scale+expression+profiling+and+interactive+analysis&doi=10.1093%2Fnar%2Fgkz430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">GEPIA2: an enhanced web server for large-scale expression profiling and interactive analysis</span></div><div class="casAuthors">Tang, Zefang; Kang, Boxi; Li, Chenwei; Chen, Tianxiang; Zhang, Zemin</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W556-W560</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Introduced in 2017, the GEPIA (Gene Expression Profiling Interactive Anal.) web server has been a valuable and highly cited resource for gene expression anal. based on tumor and normal samples from the TCGA and the GTEx databases.  Here, we present GEPIA2, an updated and enhanced version to provide insights with higher resoln. and more functionalities.  Featuring 198 619 isoforms and 84 cancer subtypes, GEPIA2 has extended gene expression quantification from the gene level to the transcript level, and supports anal. of a specific cancer subtype, and comparison between subtypes.  In addn., GEPIA2 has adopted new anal. techniques of gene signature quantification inspired by single-cell sequencing studies, and provides customized anal. where users can upload their own RNA-seq data and compare them with TCGA and GTEx samples.  We also offer an API for batch process and easy retrieval of the anal. results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6aL3UepcbfrVg90H21EOLACvtfcHk0lgOLp7ewGs_-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVCmur0%253D&md5=d6b03c2e91117df1d75e26f5beb58195</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz430%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DKang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DGEPIA2%253A%2520an%2520enhanced%2520web%2520server%2520for%2520large-scale%2520expression%2520profiling%2520and%2520interactive%2520analysis%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3DW556%26epage%3DW560%26doi%3D10.1093%2Fnar%2Fgkz430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McAlister, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nusinow, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrychowski, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wühr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huttlin, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rad, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gygi, S. P.</span></span> <span> </span><span class="NLM_article-title">MultiNotch MS3 enables accurate, sensitive, and multiplexed detection of differential expression across cancer cell line proteomes</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">7150</span>– <span class="NLM_lpage">7158</span>, <span class="refDoi"> DOI: 10.1021/ac502040v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac502040v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1Kmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=7150-7158&author=G.+C.+McAlisterauthor=D.+P.+Nusinowauthor=M.+P.+Jedrychowskiauthor=M.+W%C3%BChrauthor=E.+L.+Huttlinauthor=B.+K.+Ericksonauthor=R.+Radauthor=W.+Haasauthor=S.+P.+Gygi&title=MultiNotch+MS3+enables+accurate%2C+sensitive%2C+and+multiplexed+detection+of+differential+expression+across+cancer+cell+line+proteomes&doi=10.1021%2Fac502040v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">MultiNotch MS3 Enables Accurate, Sensitive, and Multiplexed Detection of Differential Expression across Cancer Cell Line Proteomes</span></div><div class="casAuthors">McAlister, Graeme C.; Nusinow, David P.; Jedrychowski, Mark P.; Wuhr, Martin; Huttlin, Edward L.; Erickson, Brian K.; Rad, Ramin; Haas, Wilhelm; Gygi, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7150-7158</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multiplexed quantitation via isobaric chem. tags (e.g., tandem mass tags (TMT) and isobaric tags for relative and abs. quantitation (iTRAQ)) has the potential to revolutionize quant. proteomics.  However, until recently the utility of these tags was questionable due to reporter ion ratio distortion resulting from fragmentation of coisolated interfering species.  These interfering signals can be negated through addnl. gas-phase manipulations (e.g., MS/MS/MS (MS3) and proton-transfer reactions (PTR)).  These methods, however, have a significant sensitivity penalty.  Using isolation waveforms with multiple frequency notches (i.e., synchronous precursor selection, SPS), the authors coisolated and cofragmented multiple MS2 fragment ions, thereby increasing the no. of reporter ions in the MS3 spectrum 10-fold over the std. MS3 method (i.e., MultiNotch MS3).  By increasing the reporter ion signals, this method improves the dynamic range of reporter ion quantitation, reduces reporter ion signal variance, and ultimately produces more high-quality quant. measurements.  To demonstrate utility, the authors analyzed biol. triplicates of eight colon cancer cell lines using the MultiNotch MS3 method.  Across all the replicates the authors quantified 8378 proteins in union and 6168 proteins in common.  Taking into account that each of these quantified proteins contains eight distinct cell-line measurements, this data set encompasses 174,704 quant. ratios each measured in triplicate across the biol. replicates.  Herein, the MultiNotch MS3 method uniquely combines multiplexing capacity with quant. sensitivity and accuracy, drastically increasing the informational value obtainable from proteomic expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonIBGSGUtKzbVg90H21EOLACvtfcHk0lgOLp7ewGs_-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1Kmur8%253D&md5=bda847612d4a16030b7921bc8ae6f5c6</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fac502040v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac502040v%26sid%3Dliteratum%253Aachs%26aulast%3DMcAlister%26aufirst%3DG.%2BC.%26aulast%3DNusinow%26aufirst%3DD.%2BP.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DW%25C3%25BChr%26aufirst%3DM.%26aulast%3DHuttlin%26aufirst%3DE.%2BL.%26aulast%3DErickson%26aufirst%3DB.%2BK.%26aulast%3DRad%26aufirst%3DR.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26atitle%3DMultiNotch%2520MS3%2520enables%2520accurate%252C%2520sensitive%252C%2520and%2520multiplexed%2520detection%2520of%2520differential%2520expression%2520across%2520cancer%2520cell%2520line%2520proteomes%26jtitle%3DAnal.%2520Chem.%26date%3D2014%26volume%3D86%26spage%3D7150%26epage%3D7158%26doi%3D10.1021%2Fac502040v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i88"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01897">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_53831"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01897?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01897</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional figures described in the text; tables including clinical trials, activity of the compounds and proteomics results; additional synthetic schemes; additional methods; and NMR and LC/MS charts (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and the associated biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_001.pdf">jm0c01897_si_001.pdf (8.85 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01897/suppl_file/jm0c01897_si_002.csv">jm0c01897_si_002.csv (3.45 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01897&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-3%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01897" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01897" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"677ab4dfbdcb32c1","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
